
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Genetic architecture of anthocyanin pigment traits and purple spot (Stemphylium vesicarium) resistance in an F1 pseudo‐testcross population of asparagus - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="37B2C3FD7FFFA56306C3FD0000D91D02.m_1" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11981981/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Genetic architecture of anthocyanin pigment traits and purple spot (Stemphylium vesicarium) resistance in an F1 pseudo‐testcross population of asparagus">
<meta name="citation_author" content="Suman Parajuli">
<meta name="citation_author_institution" content="Department of Plant Agriculture, University of Guelph, Guelph, Ontario, Canada">
<meta name="citation_author" content="Mary Ruth McDonald">
<meta name="citation_author_institution" content="Department of Plant Agriculture, University of Guelph, Guelph, Ontario, Canada">
<meta name="citation_author" content="Laxman Adhikari">
<meta name="citation_author_institution" content="Plant Science Program, Biological and Environmental Science and Engineering Division, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia">
<meta name="citation_author" content="David J Wolyn">
<meta name="citation_author_institution" content="Department of Plant Agriculture, University of Guelph, Guelph, Ontario, Canada">
<meta name="citation_publication_date" content="2025 Apr 9">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="2">
<meta name="citation_firstpage" content="e70028">
<meta name="citation_doi" content="10.1002/tpg2.70028">
<meta name="citation_pmid" content="40204667">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11981981/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11981981/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11981981/pdf/TPG2-18-e70028.pdf">
<meta name="description" content="Stemphylium vesicarium (Wallr.) Simmons is a plant pathogenic fungus causing purple spot in both fern and spears of asparagus (Asparagus officinalis L.). Although the fern can be sprayed with fungicides to control the disease, pesticide applications ...">
<meta name="og:title" content="Genetic architecture of anthocyanin pigment traits and purple spot (Stemphylium vesicarium) resistance in an F1 pseudo‐testcross population of asparagus">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Stemphylium vesicarium (Wallr.) Simmons is a plant pathogenic fungus causing purple spot in both fern and spears of asparagus (Asparagus officinalis L.). Although the fern can be sprayed with fungicides to control the disease, pesticide applications ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11981981/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11981981">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70028"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70028.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11981981%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11981981/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11981981/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11981981/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70028" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Apr 9;18(2):e70028. doi: <a href="https://doi.org/10.1002/tpg2.70028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70028</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Genetic architecture of anthocyanin pigment traits and purple spot (<em>Stemphylium vesicarium</em>) resistance in an F<sub>1</sub> pseudo‐testcross population of asparagus</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Parajuli%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Suman Parajuli</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Suman Parajuli</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Agriculture, University of Guelph, Guelph, Ontario, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Parajuli%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Suman Parajuli</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McDonald%20MR%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Mary Ruth McDonald</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Mary Ruth McDonald</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Agriculture, University of Guelph, Guelph, Ontario, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McDonald%20MR%22%5BAuthor%5D" class="usa-link"><span class="name western">Mary Ruth McDonald</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Adhikari%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Laxman Adhikari</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Laxman Adhikari</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Plant Science Program, Biological and Environmental Science and Engineering Division, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Adhikari%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Laxman Adhikari</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wolyn%20DJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">David J Wolyn</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">David J Wolyn</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Plant Agriculture, University of Guelph, Guelph, Ontario, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wolyn%20DJ%22%5BAuthor%5D" class="usa-link"><span class="name western">David J Wolyn</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270028-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Plant Agriculture, University of Guelph, Guelph, Ontario, Canada</div>
<div id="tpg270028-aff-0002">
<sup>
<sup>2</sup>
</sup>Plant Science Program, Biological and Environmental Science and Engineering Division, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, David J. Wolyn, Department of Plant Agriculture, University of Guelph, Guelph, Ontario, NIG 2W1, Canada. Email: <span>dwolyn@uguelph.ca</span>.
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Feb 27; Received 2024 Oct 13; Accepted 2025 Mar 5; Issue date 2025 Jun.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11981981  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40204667/" class="usa-link">40204667</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>
<em>Stemphylium vesicarium</em> (Wallr.) Simmons is a plant pathogenic fungus causing purple spot in both fern and spears of asparagus (<em>Asparagus officinalis</em> L.). Although the fern can be sprayed with fungicides to control the disease, pesticide applications during spear harvest are restricted. Infected spears can develop prominent pigmentation at lesion sites, reducing marketable yield. Breeding resistant asparagus cultivars with decreased lesion numbers and reduced purpling at the site of infection is considered the most economical and sustainable approach to combat this disease. The objectives of this study were to determine the genetic architectures of, and relationships among, anthocyanin pigment expression in spear scale leaves (ALS) and spear lesions (APS) and purple spot levels in spears (NPS) and fern (PSF). Traits were phenotyped over 2 years under natural conditions in an F<sub>1</sub> pseudo‐testcross population, and quantitative trait loci (QTL) were mapped. ALS, APS, NPS, and PSF were not correlated, suggesting independent regulation of the anthocyanin pathway in scale leaves and lesions and no relationship between pigment and disease. Segregation, 3 red:1 purple and 3 red:13 purple, was observed in scale leaves and lesions, respectively. Two stable QTL for each of ASL, APS, and NPS, one tentative QTL for ASL, four tentative QTL for APS, two tentative QTL for NPS, and three tentative QTL for PSF were identified. Candidate genes were found for four loci. This study advances the genetic understanding of anthocyanin pigmentation at a tissue‐specific level, and purple spot disease severity in spears and fern, supporting future breeding efforts.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270028-list-10001" class="list" style="list-style-type:disc">
<li><p>One‐ and two‐gene models governed the inheritance of anthocyanin color in scale leaves and purple spot lesions.</p></li>
<li><p>Anthocyanin levels in scale leaves and purple spot lesions were not linked to disease resistance in spear and fern.</p></li>
<li><p>Six stable and 10 tentative quantitative trait loci (QTL) were identified for anthocyanin levels and purple spot resistance traits.</p></li>
<li><p>Five candidate genes were mapped within the QTL regions.</p></li>
</ul></section><section id="tpg270028-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>
<em>Asparagus officinalis</em> L. is an herbaceous, dioecious perennial in the Asparagaceae family with a chromosome number of 2<em>n</em> = 2x = 20 and a haploid genome size of 1323 Mb (Bennett &amp; Leitch, <a href="#tpg270028-bib-0006" class="usa-link" aria-describedby="tpg270028-bib-0006">2011</a>). The cultivation of asparagus is challenged by various biotic and abiotic stresses that reduce growth and yield. Among these, purple spot, caused by the necrotrophic fungus <em>Stemphylium vesicarium</em> (Wallr.) Simmons, represents a major biotic stress that is a substantial threat to asparagus production worldwide (Han et al., <a href="#tpg270028-bib-0025" class="usa-link" aria-describedby="tpg270028-bib-0025">2019</a>). This pathogen infects asparagus spears in the spring and produces lesions and anthocyanin at the infection sites. The infected spears have low visual appeal and are mostly unmarketable, thus reducing yield by up to 50% (Menzies et al., <a href="#tpg270028-bib-0042" class="usa-link" aria-describedby="tpg270028-bib-0042">1992</a>). In the summer, the disease also manifests as purple spot lesions on the fern, damaging stems, branches, and cladophylls, which reduces the rate of photosynthesis, inhibits the flow of carbohydrates to the roots, and decreases spear yield in the subsequent year (Elena, <a href="#tpg270028-bib-0013" class="usa-link" aria-describedby="tpg270028-bib-0013">2007</a>; Hausbeck et al., <a href="#tpg270028-bib-0026" class="usa-link" aria-describedby="tpg270028-bib-0026">1999</a>).</p>
<p>In response to purple spot infection, the asparagus plant produces anthocyanins at the lesion sites. Anthocyanins can also accumulate in spear scale leaves and on the entire spear in certain cultivars. Six unique anthocyanin pigments have been identified: cyanidin, peonidin, malvidin, delphinidin, petunidin, and pelargonidin (Dong et al., <a href="#tpg270028-bib-0012" class="usa-link" aria-describedby="tpg270028-bib-0012">2019</a>; Liang et al., <a href="#tpg270028-bib-0039" class="usa-link" aria-describedby="tpg270028-bib-0039">2022</a>). Among these, cyanidin tri‐glycoside and peonidin‐3‐glucoside are the most abundant and provide purple and red color to spears, respectively. Their visible coloration is influenced by pH and co‐pigmentation (Khoo et al., <a href="#tpg270028-bib-0034" class="usa-link" aria-describedby="tpg270028-bib-0034">2017</a>). Acidic conditions usually intensify the red color, neutral pH results in purple or violet color, and alkaline conditions shift the color toward blue, while co‐pigmentation with flavonoids or phenolic compounds enhances color intensity and stability (Boulton, <a href="#tpg270028-bib-0007" class="usa-link" aria-describedby="tpg270028-bib-0007">2001</a>). In addition to their role in pigmentation, anthocyanins contribute to disease resistance via antioxidant activities (Sicilia et al., <a href="#tpg270028-bib-0051" class="usa-link" aria-describedby="tpg270028-bib-0051">2021</a>), activation of defense‐related genes, synthesis of antimicrobial phytoalexins, and formation of physical barriers at infection sites to restrain pathogen invasion (Ibraheem et al., <a href="#tpg270028-bib-0029" class="usa-link" aria-describedby="tpg270028-bib-0029">2010</a>; Khusnutdinov et al., <a href="#tpg270028-bib-0035" class="usa-link" aria-describedby="tpg270028-bib-0035">2021</a>). Despite their role in plant defense response, anthocyanin production can sometimes increase plant susceptibility to diseases due to resource allocation trade‐offs (Figuero et al., <a href="#tpg270028-bib-0017" class="usa-link" aria-describedby="tpg270028-bib-0017">2021</a>; Tanaka et al., <a href="#tpg270028-bib-0053" class="usa-link" aria-describedby="tpg270028-bib-0053">2014</a>).</p>
<p>The inheritance, biosynthesis, and regulation of anthocyanin pigments in plants is a complex process influenced by several factors, including multiple genes, their interactions, and the environment (Gould et al., <a href="#tpg270028-bib-0019" class="usa-link" aria-describedby="tpg270028-bib-0019">2000</a>; Lee &amp; Collins, <a href="#tpg270028-bib-0037" class="usa-link" aria-describedby="tpg270028-bib-0037">2001</a>). In asparagus, dominant alleles can enhance, or suppress anthocyanin production (Helper et al., <a href="#tpg270028-bib-0028" class="usa-link" aria-describedby="tpg270028-bib-0028">1957</a>; Peirce, <a href="#tpg270028-bib-0044" class="usa-link" aria-describedby="tpg270028-bib-0044">1982</a>; Sakaguchi et al., <a href="#tpg270028-bib-0048" class="usa-link" aria-describedby="tpg270028-bib-0048">2008</a>). A quantitative effect, increasing pigment intensity by the cumulative action of alleles at two loci, has also been demonstrated (Janick &amp; Topoleski, <a href="#tpg270028-bib-0030" class="usa-link" aria-describedby="tpg270028-bib-0030">1963</a>), as well as modifier genes affecting color and intensity (Peirce, <a href="#tpg270028-bib-0044" class="usa-link" aria-describedby="tpg270028-bib-0044">1982</a>).</p>
<p>Preliminary observations suggested cultivar‐specific differences among ‘Guelph Millennium’ (GM), ‘Guelph Eclipse’ (GE), and ‘Jersey Giant’ (JG) for disease severity and anthocyanin pigment production in response to <em>S. vesicarium</em> spear infection. GM showed less intense pigment development at lesion sites and fewer lesions than GE and JG (unpublished data). Although both cultivars produce anthocyanin pigments in their spear scale leaves and cladodes, GE and JG respond more strongly than GM to cool weather, increasing pigment production. Preliminary studies on cultivars GE and GM indicated a potential link between natural pigment intensities in scale leaves and pigmentation at lesion sites (unpublished data). However, these associations require further validation in segregating populations, as relying on comparisons among a limited number of genotypes can be biased by genetic background effects and sample size.</p>
<p>Previous efforts to breed asparagus cultivars resistant to purple spot in fern identified significant genotypic variation for disease traits, particularly within an F<sub>1</sub> segregating population (Moreno‐Pinel et al., <a href="#tpg270028-bib-0043" class="usa-link" aria-describedby="tpg270028-bib-0043">2021</a>). Greenhouse screening of progeny from a plant resistant to purple spot in fern showed higher disease tolerance than cultivar JG, a standard for commercial production at the time (Falloon et al., <a href="#tpg270028-bib-0015" class="usa-link" aria-describedby="tpg270028-bib-0015">1987</a>). Varying purple spot lesion numbers per spear were also observed among cultivars under controlled infections (Austin, <a href="#tpg270028-bib-0003" class="usa-link" aria-describedby="tpg270028-bib-0003">2023</a>); however, no asparagus cultivar displayed complete resistance.</p>
<p>Although variation for anthocyanin pigment color, red or purple, and concentration in both scale leaves and purple spot lesions have been observed, the relationships between these factors have not been studied to determine if similar or independent pathways for pigment production exist. Additionally, associations between disease levels and anthocyanin traits have also not been assessed. To address these gaps, three hypotheses are proposed: (1) Pigment levels in scale leaves and lesions are correlated, with overlapping loci controlling tissue‐specific anthocyanin accumulation; (2) purple spot severity in spear and fern are not correlated; and (3) disease severity and pigment levels are not correlated, with distinct genetic loci controlling each trait. The objectives were to develop a segregating F<sub>1</sub> pseudo‐testcross population to assess purple spot severity in spear and fern, quantify anthocyanin pigments in scale leaves and lesions, and map quantitative trait loci (QTL) for these traits to understand tissue‐specific regulation of anthocyanin pigments and relationships with purple spot resistance. By dissecting the genetic architecture underlying tissue‐specific anthocyanin production and its relationship with disease resistance, this study aims to provide valuable insights for breeding asparagus cultivars with improved disease resistance and desirable color attributes.</p></section><section id="tpg270028-sec-0020"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270028-sec-0030"><h3 class="pmc_sec_title">2.1. Plant material</h3>
<p>Due to the dioecious nature of asparagus, where male and female plants occur in separate plants, selfing usually does not occur. To facilitate genetic mapping, an F<sub>1</sub> pseudo‐testcross population was developed by crossing two heterozygous parents: a female clone derived from cultivar Viking 2K and a supermale derived from cultivar JG. The female parent had light red pigmentation in the scale leaves, whereas the supermale had dark purple. The female also had fewer purple spot lesions in the spear and less intensity of pigmentation at the infection site than the male parent.</p>
<p>A pseudo‐testcross mapping strategy (Grattapaglia &amp; Sederoff, <a href="#tpg270028-bib-0021" class="usa-link" aria-describedby="tpg270028-bib-0021">1994</a>) is commonly used in outcrossing species like asparagus and involves crossing two heterozygous parents to produce an F<sub>1</sub> population. This strategy focuses on markers that are heterozygous in one parent and homozygous in the other, simulating a Mendelian testcross. Separate genetic maps are created for each parent, which can be integrated with markers heterozygous in both parents that segregate 1:2:1.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270028-sec-0040"><h4 class="pmc_sec_title">Core Ideas</h4>
<ul id="tpg270028-list-0001" class="list" style="list-style-type:disc">
<li><p>One‐ and two‐gene models governed the inheritance of anthocyanin color in scale leaves and purple spot lesions.</p></li>
<li><p>Anthocyanin levels in scale leaves and purple spot lesions were not linked to disease resistance in spear and fern.</p></li>
<li><p>Six stable and 10 tentative quantitative trait loci (QTL) were identified for anthocyanin levels and purple spot resistance traits.</p></li>
<li><p>Five candidate genes were mapped within the QTL regions.</p></li>
</ul></section></section></section><section id="tpg270028-sec-0050"><h3 class="pmc_sec_title">2.2. Plant establishment</h3>
<p>Seeds of the F<sub>1</sub> pseudo‐testcross population, along with control cultivars GM and GE, were initially sown in 288‐cell plug trays using a potting mix (Sunshine L.C. 4, Sun Gro Horticulture) in the greenhouse during March 2018 and 2019. After 40 days, seedlings were transferred to 50‐cell plug trays filled with the same potting mix and grown in the greenhouse at 25/20°C (day/night) under natural irradiance with a 12 h photoperiod supplement with high‐pressure sodium lamps (∼80 µmol m<sup>2</sup> s<sup>−1</sup>). Plants were fertilized weekly with 20N:3.50P:16.60K (1.25 g L<sup>−1</sup>; Plant Products Limited). In June 2018 and 2019, 173 and 243 10‐week‐old seedlings, as well as control, were transplanted across two sites at the Simcoe Research Station in Simcoe, Ontario (Lat. 42°51′ N; Long. 80°16′ W, elevation 240.50 m), respectively. All seedlings, including controls, were spaced 0.30‐m in 20‐cm deep trenches, with rows spaced 1.25 m. Crowns were initially covered with 5 cm of soil, and trenches were gradually filled with soil over the summer. Guard rows were planted around the entire field to minimize the edge effect.</p></section><section id="tpg270028-sec-0060"><h3 class="pmc_sec_title">2.3. Experimental design</h3>
<p>The 416 individuals from the F<sub>1</sub> pseudo‐testcross population and controls were phenotyped at two sites for two replicate years, and data were analyzed in an augmented randomized complete block design (Federer, <a href="#tpg270028-bib-0016" class="usa-link" aria-describedby="tpg270028-bib-0016">1956</a>) structured in 28 blocks. Each block consisted of twenty genotypes, 18 of which were test entries (test genotypes), and the remaining two were control entries. Test genotypes were randomly assigned within each block in a row, while the control plants, either GM or GE, were alternately planted after every fifth and tenth genotype in a block. These seed‐propagated check cultivars were chosen for their extreme variation in anthocyanin pigment and purple spot traits in lieu of the parents of the cross, which would have required tissue culture micropropagation to achieve the required numbers for the experiment.</p></section><section id="tpg270028-sec-0070"><h3 class="pmc_sec_title">2.4. Phenotyping</h3>
<p>Anthocyanin pigments in the scale leaves and at the sites of infection were assessed using two methods. First, a nominal scale was used for phenotyping the presence and absence of purple and red anthocyanin pigment in the scale leaves and sites of infection (Figure <a href="#tpg270028-fig-0001" class="usa-link">1</a>). Second, a 1–9 ordinal scale was employed to quantify anthocyanin pigment levels for the ASL (Figure <a href="#tpg270028-fig-0002" class="usa-link">2</a>) and APS (Figure <a href="#tpg270028-fig-0003" class="usa-link">3</a>) traits. The number of purple spot lesions on individual spears (NPS) was counted in response to natural purple spot infection. Data for all traits were collected on two spears per genotype (20 ± 5 cm in length), once weekly for 3 weeks in early May for two consecutive years, 2021 and 2022, from the same plants of the F<sub>1</sub> pseudo‐testcross population and controls. As asparagus is a perennial crop, the F<sub>1</sub> pseudo‐testcross population remained genetically stable across both years for evaluation.</p>
<figure class="fig xbox font-sm" id="tpg270028-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e5b/11981981/54ef8d2d8c2e/TPG2-18-e70028-g003.jpg" loading="lazy" id="jats-graphic-1" height="489" width="400" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270028-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Visual rating used to evaluate anthocyanin pigment phenotype in the scale leaves (A and B) and purple spot lesions (C and D) in an asparagus F<sub>1</sub> pseudo‐testcross population as red (A and C) and purple (B and D).</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270028-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e5b/11981981/f436a002b118/TPG2-18-e70028-g004.jpg" loading="lazy" id="jats-graphic-3" height="827" width="600" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270028-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Variation of natural anthocyanin pigment in scale leaves in an asparagus F<sub>1</sub> pseudo‐testcross population where 1 and 9 indicate the phenotypic classes with the lowest and highest pigment levels, respectively.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270028-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e5b/11981981/d5f842779989/TPG2-18-e70028-g001.jpg" loading="lazy" id="jats-graphic-5" height="849" width="600" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270028-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Variation of anthocyanin pigment levels in response to purple spot infection in an asparagus F<sub>1</sub> pseudo‐testcross population where 1 and 9 indicate the phenotypic classes with the highest and lowest pigment levels, respectively.</p></figcaption></figure><p>Plants were rated for anthocyanin pigment by three individuals, and the average of three ratings was computed to minimize the potential bias or error from a single rater. Spears of each genotype were also photographed in the field under natural light. The visually scored pigment data for all genotypes were validated using digital image review. Purple spot severity in fern (PSF) was evaluated in the last week of September and the first week of October in both 2021 and 2022 from natural infection after fungicide sprays ceased in late August. Disease severity in fern was scored using an ordinal scale (Figure <a href="#tpg270028-supinfo-0001" class="usa-link">S1</a>). Phenotypic datasets collected using nominal, ordinal (ASL, APS, and PSF), and quantitative (NPS counts) were used for QTL analyses.</p></section><section id="tpg270028-sec-0080"><h3 class="pmc_sec_title">2.5. Extraction and quantification of anthocyanin</h3>
<p>Total anthocyanin concentration was estimated following the method described by Sims and Gamon (<a href="#tpg270028-bib-0052" class="usa-link" aria-describedby="tpg270028-bib-0052">2002</a>). Five spears representing each phenotypic category (1–9 scale) for anthocyanin pigment in the scale leaves and purple spot lesions were collected from the field in early May 2021 and stored in a cooler during transport to the laboratory. Scale leaves were removed from the spears, and 1.0 × 0.5 cm sections of epidermal tissue at lesion sites containing four to five lesions of consistent size were dissected from each spear. Tissues were pooled for each phenotypic category of scale leaves and lesions, frozen with liquid nitrogen, and stored at −80°C until extraction. The freeze‐dried samples were ground into a fine powder using a mortar and pestle in the presence of liquid nitrogen. From the powdered samples, 100 mg was transferred to an Eppendorf tube containing 2 mL of acidified methanol (90:1:1 methanol:HCL:water, v/v/v) and mixed using a vortex. The homogenate was then centrifuged at 14,000 <em>g</em> for 10 min. From the supernatant, 1 mL was placed in a cuvette, and absorbance was measured at both 529 nm and 650 nm. Total anthocyanin was calculated using the following formula:
</p>
<table class="disp-formula p" id="tpg270028-disp-0001"><tr>
<td class="formula"><math id="jats-math-1" display="block"><mrow><mtable displaystyle="true"><mtr><mtd></mtd><mtd></mtd><mtd columnalign="left"><mrow><mrow><mi>Anthocyanin</mi><mspace width="0.28em"></mspace></mrow><mfenced separators="" open="(" close=")"><mfrac><mi>mg</mi><mrow><mn>100</mn><mrow><mspace width="0.28em"></mspace><mi mathvariant="normal">g</mi></mrow></mrow></mfrac></mfenced></mrow></mtd></mtr><mtr><mtd></mtd><mtd></mtd><mtd columnalign="left"><mrow><mo linebreak="badbreak">=</mo><mfrac><mfenced separators="" open="[" close="]"><mrow><mfenced separators="" open="(" close=")"><mrow><mi>A</mi><mn>529</mn><mo>−</mo><mn>0.288</mn><mo>×</mo><mi>A</mi><mn>650</mn></mrow></mfenced><mspace width="0.28em"></mspace><mo>×</mo><mspace width="0.28em"></mspace><mi mathvariant="normal">D</mi><mo>.</mo><mrow><mi mathvariant="normal">F</mi><mspace width="0.28em"></mspace></mrow><mo>×</mo><mrow><mspace width="0.28em"></mspace><mi mathvariant="normal">M</mi></mrow><mo>.</mo><mrow><mi mathvariant="normal">W</mi><mspace width="0.28em"></mspace></mrow><mo>×</mo><mi>V</mi><mo>×</mo><mn>1000</mn></mrow></mfenced><mfenced separators="" open="(" close=")"><mrow><mi mathvariant="normal">ε</mi><mo>×</mo><mi>P</mi><mo>×</mo><mi>X</mi></mrow></mfenced></mfrac></mrow></mtd></mtr><mtr><mtd></mtd><mtd></mtd><mtd columnalign="left"><mrow><mspace width="0.16em"></mspace><mo linebreak="badbreak">×</mo><mn>100</mn></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>where (A529 − 0.288 × A650) is anthocyanin absorbance corrected for chlorophyll absorbance; D.F is the dilution factor (set as 1, the volume of samples remains constant); M.W is the molecular weight (449.2 for cyanidin‐3‐glucoside; the dominant anthocyanin in asparagus); <em>V</em> is the final volume (L) of anthocyanin containing supernatant; <em>X</em> is the weight (g) of the asparagus powder extracted; <em>ɛ</em> is the molar absorbance coefficient for anthocyanin (26,900), and <em>P</em> is the cuvette pathlength (1 cm).</p>
<p>In early May 2021, during the peak period of anthocyanin pigment expression, 10 asparagus spears (25 ± 5 cm) representing each phenotypic category (1–9) for scale leaves and lesions were collected from the field. Scale leaves and lesion tissue samples were handled and processed to produce powder, as described above. From the powder samples, 100 mg was added to an Eppendorf tube with 2 mL of 70% aqueous methanol. The mixture was incubated overnight at 4°C, then centrifuged at 14,000 <em>g</em> for 10 min. The samples were filtered through a 0.45 µm chromatography filter and then transferred to high‐performance liquid chromatography vials (National Scientific; model no. B7999‐1) with each vial containing a total volume of 500 µL. For analysis, 2 µL were analyzed with the Vanquish Flex Binary ultra‐high performance liquid chromatography system interfaced with a Thermo Scientific Q‐Exactive Orbitrap mass spectrometer utilizing a column C18 (Kinetex XB‐C18 100A) for chromatography separation, following the method outlined by Cox et al. (<a href="#tpg270028-bib-0011" class="usa-link" aria-describedby="tpg270028-bib-0011">2021</a>).</p></section><section id="tpg270028-sec-0090"><h3 class="pmc_sec_title">2.6. Genotyping</h3>
<p>For DNA extraction, green fern tissues were collected from the 416 individuals of the F<sub>1</sub> pseudo‐testcross population and parents in 2021 in 15 mL Falcon Tubes (Thermo Fisher Scientific) and placed on dry ice. Samples were then frozen at −80°C and freeze‐dried using a Savant ModulyoD Thermoquest (Savant Instruments). Genomic DNA was extracted from 30 to 40 mg of dry fern per genotype using a NucleoSpin Plant II kit (Macherey‐Nagel) per the manufacturer's instructions. The quality of DNA was verified using a NanoDrop spectrophotometer (Thermo Fisher Scientific). DNA concentration was measured using a Qubit 2.0 fluorometer (Invitrogen), and samples (10 ng/µL) were sent to Plate‐forme D'analyses Génomiques at Université Laval (Laval) for genotyping‐by‐sequencing (GBS). The restriction enzymes <em>MspI</em> and <em>Pst</em>I were used to generate the sequence library, following the methods outlined by Mascher et al. (<a href="#tpg270028-bib-0040" class="usa-link" aria-describedby="tpg270028-bib-0040">2013</a>). Sequencing was conducted on an Ion Proton sequencing machine (Life Technologies) by Université Laval, Québec, Canada. A total of 746 million reads, each ranging in length from 50 to 150 base pairs, were generated with a GC content of 49.91% and an average Q30 ratio of 89.36%, indicating the high quality of sequencing data.</p>
<p>The initial data preparation, genotyping, and filtration were executed using STACKS v2.62 (<a href="https://catchenlab.life.illinois.edu/stacks/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://catchenlab.life.illinois.edu/stacks/</a>) and stacks workflow v2.62 (<a href="https://github.com/enormandeau/stacks_workflow" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/enormandeau/stacks_workflow</a>) following the methods described by Rochette and Catchen (<a href="#tpg270028-bib-0047" class="usa-link" aria-describedby="tpg270028-bib-0047">2017</a>). The demultiplexed reads were aligned to the reference genome using the Burrows–Wheeler Aligner (BWA v0.7.17‐r1188) with parameters ‐k 19 ‐c 500 ‐O 0,0 ‐E 2,2 ‐T 0 (Li &amp; Durbin et al., <a href="#tpg270028-bib-0038" class="usa-link" aria-describedby="tpg270028-bib-0038">2008</a>). These were further processed with SAM tools v1.8 with the flags ‐Sb ‐q 1 ‐F 4 ‐F 256 ‐F 2048, which converted data into the binary format while also filtering based on mapping quality and read types (Li &amp; Durbin et al., <a href="#tpg270028-bib-0038" class="usa-link" aria-describedby="tpg270028-bib-0038">2008</a>). To conclude the procedure, the population module from STACKS was run with parameters ‐p 1 ‐r 0.5 –ordered‐export –fasta‐loci –vcf to execute the last step of the STACKS pipeline, calculating population genetics statistics, exporting data in various formats, and conducting genotyping.</p>
<p>After genotyping, using STACKS steps, filtering of identified SNPs was performed using an in‐house Python script, 05_filter_vcf_fast.py, with parameters 3, 60, 0, and 1 so that all genotypes had a minimum coverage of 3, and all samples combined had at most 40% missing SNP data. This filtering process facilitated the generation of a high‐quality dataset suitable for downstream genetic analysis.</p></section><section id="tpg270028-sec-0100"><h3 class="pmc_sec_title">2.7. Linkage map construction, QTL analysis, and candidate gene identification</h3>
<p>The filtered SNPs belonged to three distinct groups of marker segregation: (a) maternal testcross markers that were heterozygous in the female parent and homozygous in the male parent segregating as in the classical backcross (lm × ll) at a 1:1 ratio; (b) paternal testcross markers that were heterozygous in male parent and homozygous in the female parent (nn × np) segregating at a 1:1 ratio; (c) inter‐cross markers, which were heterozygous in both parents (hk × hk) and segregating in 1:2:1 ratio. For each parental map construction, only single‐dose markers were used. Markers segregating at distorted Mendelian ratios were removed. The single‐dose markers from the maternal and paternal parents were analyzed separately using JoinMap 5.11 (Van Ooijen, <a href="#tpg270028-bib-0057" class="usa-link" aria-describedby="tpg270028-bib-0057">2018</a>), and the outcross‐pollinated family was selected as the population type. The linkage groups (LGs) were built using a regression mapping algorithm, with a minimum logarithm of odds (LOD) value at 4 for marker grouping and a maximum recombination frequency of 0.40. The distance between the markers was generated using the Kosambi map functions (Kosambi, <a href="#tpg270028-bib-0036" class="usa-link" aria-describedby="tpg270028-bib-0036">2016</a>). LGs were drawn with MapChart (Voorrips, <a href="#tpg270028-bib-0058" class="usa-link" aria-describedby="tpg270028-bib-0058">2002</a>). Heterozygous markers in both parents (hk × hk) were selected for integrated map construction after those with segregation distortion were eliminated.</p>
<p>For each genotype, means were estimated over the multiple sampling dates in each of the 2 years, then over the 2 years. To account for spatial heterogeneity within each block, individual yearly means and the combined 2‐year mean for each genotype were independently adjusted based on the average values of the replicated check treatments (GM and GE) among and within each block using a linear mixed model. The negative values from this adjustment were set to zero for correlation and QTL analysis, as negative means are not biologically feasible. An adjusted genotype mean was obtained using the formula:</p>
<table class="disp-formula p" id="tpg270028-disp-0002"><tr>
<td class="formula"><math id="jats-math-2" display="block"><mrow><mtable displaystyle="true"><mtr><mtd></mtd><mtd></mtd><mtd columnalign="left"><mrow><mrow><mi>Adjusted</mi><mspace width="0.28em"></mspace><mi>genotype</mi><mspace width="0.28em"></mspace><mi>mean</mi></mrow><mo linebreak="badbreak">=</mo><mrow><mi>Genotype</mi><mspace width="0.28em"></mspace><mi>mean</mi></mrow></mrow></mtd></mtr><mtr><mtd></mtd><mtd></mtd><mtd columnalign="left"><mrow><mo linebreak="badbreak">−</mo><mspace width="0.16em"></mspace><mfenced separators="" open="(" close=""><mrow><mi>mean</mi><mspace width="0.28em"></mspace><mi>of</mi><mspace width="0.28em"></mspace><mi>GM</mi><mspace width="0.28em"></mspace><mi>and</mi><mspace width="0.28em"></mspace><mi>GE</mi><mspace width="0.28em"></mspace><mi>within</mi><mspace width="0.28em"></mspace><mi mathvariant="normal">a</mi><mspace width="0.28em"></mspace><mi>block</mi></mrow></mfenced></mrow></mtd></mtr><mtr><mtd></mtd><mtd></mtd><mtd columnalign="left"><mfenced separators="" open="" close=")"><mo linebreak="badbreak">−</mo><mspace width="0.16em"></mspace><mrow><mi>grand</mi><mspace width="0.28em"></mspace><mi>mean</mi><mspace width="0.28em"></mspace><mi>of</mi><mspace width="0.28em"></mspace><mi>GM</mi><mspace width="0.28em"></mspace><mi>and</mi><mspace width="0.28em"></mspace><mi>GE</mi><mspace width="0.28em"></mspace><mi>across</mi><mspace width="0.28em"></mspace><mi>all</mi><mspace width="0.28em"></mspace><mi>blocks</mi></mrow></mfenced></mtd></mtr></mtable></mrow></math></td>
<td class="label">(2)</td>
</tr></table>
<p>QTL analysis was conducted using two methods based on the nature of the traits. For nominal data, such as the presence and absence of purple and red anthocyanin pigment in the scale leaves and lesions, QTL analysis was performed using R/qtl (Broman et al., <a href="#tpg270028-bib-0008" class="usa-link" aria-describedby="tpg270028-bib-0008">2003</a>). For ordinal data (ASL, APS, PSF, and NPS), QTL were detected using composite interval mapping in WinQTL Cart 2.5 (Wang et al., <a href="#tpg270028-bib-0059" class="usa-link" aria-describedby="tpg270028-bib-0059">2012</a>) with a walking speed of 1 cM. For determining the cofactors, a forward and backward regression method with a probability threshold of 0.1 and a window size of 10 cM was used. The significance of detected QTL was determined using an LOD threshold based on a 1000 permutation test for each dataset with a significance level of <em>p</em> ≤ 0.05 (Churchill &amp; Doerge, <a href="#tpg270028-bib-0010" class="usa-link" aria-describedby="tpg270028-bib-0010">1994</a>). A 95% confidence interval for each detected QTL was estimated using a 1‐LOD support interval (LSI) (Van Ooijen, <a href="#tpg270028-bib-0056" class="usa-link" aria-describedby="tpg270028-bib-0056">1992</a>). Adjacent QTL on the same chromosome for the same trait were considered separate if their support intervals did not overlap (Haggard et al., <a href="#tpg270028-bib-0023" class="usa-link" aria-describedby="tpg270028-bib-0023">2015</a>). The percentage of phenotypic variation (<em>R</em>
<sup>2</sup>) explained by each QTL in proportion to the total phenotypic variation was estimated. The QTL were named according to McCouch et al. (<a href="#tpg270028-bib-0041" class="usa-link" aria-describedby="tpg270028-bib-0041">1997</a>) and labeled as qVARx.y where VAR is a trait, <em>x</em> is chromosome number, and <em>y</em> is QTL number on a given chromosome.</p>
<p>The identified QTL were classified as stable or tentative. Those detected across multiple years of phenotypic data were considered stable, whereas QTL detected in only 1 year were classified as tentative. QTL explaining 15% or more of the phenotypic variation are considered major, whereas those explaining &lt;15% are minor. The regions of colocalized QTL were searched using adjusted least square means to identify chromosomal regions containing putative pleiotropic genes controlling multiple traits. Finally, MapChart 2.3 (Voorrips, <a href="#tpg270028-bib-0058" class="usa-link" aria-describedby="tpg270028-bib-0058">2002</a>) was used to visually represent the positions of identified QTL on each linkage map.</p>
<p>Flanking sequences (300–600 bp) of SNPs associated with each QTL from GBS analysis were aligned with the asparagus reference genome v1.1 (<span class="text-underline">reference genome v1.1</span>) using the Basic Local Alignment Search Tool. Candidate genes were identified within the LSI of QTL regions for the traits ASL, APS, NPS, and PSF based on gene function.</p></section><section id="tpg270028-sec-0110"><h3 class="pmc_sec_title">2.8. Statistical analysis</h3>
<p>All statistical analyses were performed in R (R Core Team, <a href="#tpg270028-bib-0045" class="usa-link" aria-describedby="tpg270028-bib-0045">2022</a>). Analysis of variance (ANOVA) and adjusted means were estimated using the package ‘augmentedRCBD’ (Aravind et al., <a href="#tpg270028-bib-0001" class="usa-link" aria-describedby="tpg270028-bib-0001">2021</a>). A linear mixed model was employed for the ANOVA:
</p>
<table class="disp-formula p" id="tpg270028-disp-0003"><tr>
<td class="formula"><math id="jats-math-3" display="block"><mrow><mrow><msub><mi>Y</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi></mrow></msub><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><msub><mi>G</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>B</mi><mi>j</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>E</mi><mrow><mi>i</mi><mi>j</mi><mi>k</mi></mrow></msub></mrow></mrow></math></td>
<td class="label">(3)</td>
</tr></table>
<p>where <em>Y<sub>ijk</sub>
</em> is the measurement recorded in a plot, <em>μ</em> is the overall mean of the plot, <em>G<sub>i</sub>
</em> is the fixed effect of the <em>i</em>th genotype (which includes both test and check genotype), <em>B<sub>j</sub>
</em> is the random effect of the <em>j</em>th block, and <em>E<sub>ijk</sub>
</em> is the experimental error.</p>
<p>The normality of data was assessed with the Shapiro–Wilk test using the shapiro.test() function in the “stats” package. Bartlett's (Bartlett, <a href="#tpg270028-bib-0005" class="usa-link" aria-describedby="tpg270028-bib-0005">1937</a>) tests for homogeneity of variance across years were estimated using the “car” package. Pearson correlation coefficients were estimated using the “psych” package (Revelle, <a href="#tpg270028-bib-0046" class="usa-link" aria-describedby="tpg270028-bib-0046">2023</a>) for all traits. Grubbs’ outlier test (Grubbs, <a href="#tpg270028-bib-0022" class="usa-link" aria-describedby="tpg270028-bib-0022">1950</a>) was conducted to detect outliers with a package “outlier.”</p></section></section><section id="tpg270028-sec-0120"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270028-sec-0130"><h3 class="pmc_sec_title">3.1. Inheritance of tissue‐specific anthocyanin pigment color, red or purple, in spear scale leaves and purple spot lesions</h3>
<p>Pigment color segregated 3 red: 1 purple in scale leaves and 13 purple: 3 red in purple spot lesions in the F<sub>1</sub> pseudo‐testcross population (Table <a href="#tpg270028-tbl-0001" class="usa-link">1</a>), suggesting one‐ and two‐gene models and unique genetic regulation of the two traits. However, these data are considered preliminary and require validation with additional crosses. No QTL for color, red and purple were identified, likely due to insufficient marker density on the male/female integrated genetic map (Tables <a href="#tpg270028-supinfo-0001" class="usa-link">S1–S5</a>; Figures <a href="#tpg270028-supinfo-0001" class="usa-link">S2–S5</a>).</p>
<section class="tw xbox font-sm" id="tpg270028-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Segregation of red and purple phenotypes for scale leaves and purple spot lesions in an asparagus F<sub>1</sub> pseudo‐testcross population.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead>
<tr>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">F<sub>1</sub> Phenotype</th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">Year</th>
<th align="left" rowspan="1" colspan="1">Observed red (#)</th>
<th align="left" rowspan="1" colspan="1">Observed purple (#)</th>
<th align="left" rowspan="1" colspan="1">Expected ratio</th>
<th align="left" rowspan="1" colspan="1">Chi‐square</th>
<th align="left" rowspan="1" colspan="1">
<em>p</em>‐value</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="2" align="left" colspan="1">Scale leaves</td>
<td align="left" rowspan="1" colspan="1">2021</td>
<td align="left" rowspan="1" colspan="1">296</td>
<td align="left" rowspan="1" colspan="1">120</td>
<td align="left" rowspan="1" colspan="1">3:1</td>
<td align="left" rowspan="1" colspan="1">3.28</td>
<td align="left" rowspan="1" colspan="1">0.07</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2022</td>
<td align="left" rowspan="1" colspan="1">300</td>
<td align="left" rowspan="1" colspan="1">116</td>
<td align="left" rowspan="1" colspan="1">3:1</td>
<td align="left" rowspan="1" colspan="1">1.84</td>
<td align="left" rowspan="1" colspan="1">0.17</td>
</tr>
<tr>
<td rowspan="2" align="left" colspan="1">Purple spot</td>
<td align="left" rowspan="1" colspan="1">2021</td>
<td align="left" rowspan="1" colspan="1">73</td>
<td align="left" rowspan="1" colspan="1">343</td>
<td align="left" rowspan="1" colspan="1">13:3</td>
<td align="left" rowspan="1" colspan="1">0.52</td>
<td align="left" rowspan="1" colspan="1">0.39</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2022</td>
<td align="left" rowspan="1" colspan="1">75</td>
<td align="left" rowspan="1" colspan="1">341</td>
<td align="left" rowspan="1" colspan="1">13:3</td>
<td align="left" rowspan="1" colspan="1">0.14</td>
<td align="left" rowspan="1" colspan="1">0.70</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270028-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="tpg270028-sec-0140"><h3 class="pmc_sec_title">3.2. Variation of anthocyanin composition in scale leaves and spear purple spot lesions</h3>
<p>Cyanidin and peonidin anthocyanins, including cyanidin rhamnoside hexoside‐hexoside, cyanidin hexoside rhamnoside, peonidin rhamnoside hexoside‐hexoside, peonidin rhamnoside hexoside, and cyanidin glucoside, were identified using ultra‐high performance liquid chromatography coupled with mass spectrometry (LC/MS) in the segregating population (Table <a href="#tpg270028-supinfo-0001" class="usa-link">S1</a>). Each pigment demonstrated unique physicochemical properties, including retention time, absorbance at UV‐520 nm, molecular weight (m/z), and a molecular formula.</p>
<p>The relative quantitative analysis of anthocyanin pigments in the red and purple scale leaves revealed significant concentration differences. Cyanidin (C) and peonidin (P) were notably more elevated in purple scale leaves compared to red scale leaves, with cyanidin being the most prominent (Table <a href="#tpg270028-supinfo-0001" class="usa-link">S2</a>). Intriguingly, the C:P ratio differed between red and purple color classes, with a ratio of 26.81 observed in red scale leaves compared to 11.08 in purple scale leaves (Table <a href="#tpg270028-tbl-0002" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="tpg270028-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Quantification of anthocyanin pigment types in asparagus scale leaves using liquid chromatography coupled with mass spectrometry (LC/MS), visually rated as purple or red in an F<sub>1</sub> pseudo‐testcross population.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead>
<tr>
<th rowspan="2" align="left" colspan="1">Compounds</th>
<th colspan="4" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Relative anthocyanin concentration<a href="#tpg270028-tbl2-note-0001" class="usa-link">
<sup>a</sup>
</a>
</th>
</tr>
<tr>
<th colspan="2" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Scale leaves</th>
<th colspan="2" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Purple spot</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">Red</th>
<th align="left" rowspan="1" colspan="1">Purple</th>
<th align="left" rowspan="1" colspan="1">Red</th>
<th align="left" rowspan="1" colspan="1">Purple</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Cya‐rham‐hexoside‐hexoside (C)</td>
<td align="left" rowspan="1" colspan="1">2.10E+08</td>
<td align="left" rowspan="1" colspan="1">1.90E+09</td>
<td align="left" rowspan="1" colspan="1">6.15E+08</td>
<td align="left" rowspan="1" colspan="1">1.63E+09</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Cya‐hexoside‐rhamnoside (C)</td>
<td align="left" rowspan="1" colspan="1">1.37E+08</td>
<td align="left" rowspan="1" colspan="1">9.53E+08</td>
<td align="left" rowspan="1" colspan="1">6.02E+08</td>
<td align="left" rowspan="1" colspan="1">1.63E+09</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Peo‐rham‐hex‐hex (P)</td>
<td align="left" rowspan="1" colspan="1">8.10E+06</td>
<td align="left" rowspan="1" colspan="1">1.64E+08</td>
<td align="left" rowspan="1" colspan="1">4.57E+07</td>
<td align="left" rowspan="1" colspan="1">1.24E+08</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Peo‐rham‐hex (P)</td>
<td align="left" rowspan="1" colspan="1">5.06E+06</td>
<td align="left" rowspan="1" colspan="1">9.87E+07</td>
<td align="left" rowspan="1" colspan="1">5.50E+07</td>
<td align="left" rowspan="1" colspan="1">1.28E+08</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Cya‐glucoside (C)</td>
<td align="left" rowspan="1" colspan="1">6.66E+05</td>
<td align="left" rowspan="1" colspan="1">5.66E+07</td>
<td align="left" rowspan="1" colspan="1">1.45E+07</td>
<td align="left" rowspan="1" colspan="1">4.99E+07</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Cyanidin total</td>
<td align="left" rowspan="1" colspan="1">3.53E+08</td>
<td align="left" rowspan="1" colspan="1">2.90E+09</td>
<td align="left" rowspan="1" colspan="1">2.12E+09</td>
<td align="left" rowspan="1" colspan="1">2.55E+09</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Peonidin total</td>
<td align="left" rowspan="1" colspan="1">1.31E+07</td>
<td align="left" rowspan="1" colspan="1">2.62E+08</td>
<td align="left" rowspan="1" colspan="1">1.23E+09</td>
<td align="left" rowspan="1" colspan="1">3.30E+09</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">C total/P total</td>
<td align="left" rowspan="1" colspan="1">26.79</td>
<td align="left" rowspan="1" colspan="1">11.07</td>
<td align="left" rowspan="1" colspan="1">12.24</td>
<td align="left" rowspan="1" colspan="1">13.14</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270028-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270028-tbl2-note-0001">
<sup>
<sup>a</sup>
</sup><p class="display-inline">Values represent the average relative anthocyanin concentration units as measured by LC/MS in scale leaves and purple spot lesions.</p>
</div></div></section><p>The comparative quantitative analysis of anthocyanin pigments in the red and purple spot lesions indicated that both C and P total concentrations were significantly elevated in purple lesions compared to red lesions, and C was the most dominant (Table <a href="#tpg270028-supinfo-0001" class="usa-link">S2</a>). The C total:P total ratio was relatively consistent across both color types, 12.23 in red and 13.14 in purple lesions (Table <a href="#tpg270028-tbl-0002" class="usa-link">2</a>).</p></section><section id="tpg270028-sec-0150"><h3 class="pmc_sec_title">3.3. Validation of visual anthocyanin ratings in scale leaves and purple spot lesions</h3>
<p>Visual anthocyanin phenotypic ratings (1–9) based on pigment intensity, irrespective of red or purple color, in scale leaves and purple spot lesions were highly correlated with total pigment concentrations as determined by LC/MS (Table <a href="#tpg270028-supinfo-0001" class="usa-link">S2</a>) or spectrophotometry (Table <a href="#tpg270028-supinfo-0001" class="usa-link">S3</a>). For the scale leaves, the coefficient of determination (<em>R</em>
<sup>2</sup>) between the visual pigment ratings and the spectrophotometer measurements was 0.99, whereas that for LC/MS analysis was 0.88. For visual pigment ratings in spear lesions, the <em>R</em>
<sup>2</sup> values with spectrophotometer measurement and LC/MS analysis were 0.98 and 0.97, respectively. The high <em>R</em>
<sup>2</sup> values indicate that the visual assessments are reliable for determining total anthocyanin concentration in scale leaves and lesions.</p></section><section id="tpg270028-sec-0160"><h3 class="pmc_sec_title">3.4. Phenotypic assessment of the segregating F<sub>1</sub> pseudo‐testcross population</h3>
<p>The ANOVA indicated significant differences (<em>p</em> &lt; 0.05) for traits ASL, APS, NPS, and PSF among 416 individuals in the F<sub>1</sub> pseudo‐testcross population (Table <a href="#tpg270028-supinfo-0001" class="usa-link">S4</a>). All traits showed normal to near‐normal distributions (Figure <a href="#tpg270028-supinfo-0001" class="usa-link">S2</a>), suggesting they are most likely quantitative, involving multiple loci. Correlations between replicate years, 2021 and 2022, were high for ASL (0.78), moderate for NPS (0.50) and PSF (0.47), and low for APS (0.20). APS and ASL were poorly correlated within each of the years (0.21–0.33), as well as APS with NPS (0.22–0.29) (Table <a href="#tpg270028-tbl-0003" class="usa-link">3</a>). Near zero correlations were observed for APS with PSF, ASL with NPS or PSF, and NPS with PSF.</p>
<section class="tw xbox font-sm" id="tpg270028-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>Within‐year Pearson correlation coefficients among natural levels of anthocyanin in scale leaves (ASL)<a href="#tpg270028-tbl3-note-0001" class="usa-link">
<sup>a</sup>
</a>, and spear purple spot lesions (APS)<a href="#tpg270028-tbl3-note-0002" class="usa-link">
<sup>b</sup>
</a>, number of purple spot lesions in spears (NPS), and purple spot severity for fern (PSF)<a href="#tpg270028-tbl3-note-0003" class="usa-link">
<sup>c</sup>
</a> in an asparagus F<sub>1</sub> pseudo‐testcross population evaluated in 2021 and 2022.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Phenotype</th>
<th align="left" rowspan="1" colspan="1">APS 2021</th>
<th align="left" rowspan="1" colspan="1">ASL 2021</th>
<th align="left" rowspan="1" colspan="1">NPS 2021</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">ASL 2021</td>
<td align="center" rowspan="1" colspan="1">0.21***</td>
<td align="center" rowspan="1" colspan="1"></td>
<td align="center" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">NPS 2021</td>
<td align="center" rowspan="1" colspan="1">0.29***</td>
<td align="center" rowspan="1" colspan="1">0.06</td>
<td align="center" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PSF 2021</td>
<td align="center" rowspan="1" colspan="1">−0.06</td>
<td align="center" rowspan="1" colspan="1">0.02</td>
<td align="center" rowspan="1" colspan="1">0.01</td>
</tr>
</tbody>
</table></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Phenotype</th>
<th align="left" rowspan="1" colspan="1">APS 2022</th>
<th align="left" rowspan="1" colspan="1">ASL 2022</th>
<th align="left" rowspan="1" colspan="1">NPS 2022</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">ASL 2022</td>
<td align="center" rowspan="1" colspan="1">0.33***</td>
<td align="center" rowspan="1" colspan="1"></td>
<td align="center" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">NPS 2022</td>
<td align="center" rowspan="1" colspan="1">0.22***</td>
<td align="center" rowspan="1" colspan="1">0.11*</td>
<td align="center" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PSF 2022</td>
<td align="center" rowspan="1" colspan="1">−0.02</td>
<td align="center" rowspan="1" colspan="1">0.05</td>
<td align="center" rowspan="1" colspan="1">−0.10*</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270028-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270028-tbl3-note-0001">
<sup>
<sup>a</sup>
</sup><p class="display-inline">Anthocyanin pigment phenotypic rating in spear (1 = low pigment level, 9 = high pigment level).</p>
</div>
<div class="fn" id="tpg270028-tbl3-note-0002">
<sup>
<sup>b</sup>
</sup><p class="display-inline">Anthocyanin pigment phenotypic rating for spear purple spot lesion (1 = low pigment level, 9 = high pigment level).</p>
</div>
<div class="fn" id="tpg270028-tbl3-note-0003">
<sup>
<sup>c</sup>
</sup><p class="display-inline">Purple spot severity for fern (0 = highly resistant genotype, 9 = highly susceptible genotype).</p>
</div>
<div class="fn" id="tpg270028-tbl3-note-0004"><p>*<em>p</em> &lt; 0.05,**<em>p</em> &lt; 0.01,***<em>p</em> &lt; 0.001.</p></div>
</div></section></section><section id="tpg270028-sec-0170"><h3 class="pmc_sec_title">3.5. Genetic map, QTL analysis, and candidate gene discovery</h3>
<p>The genetic maps for the maternal and paternal parents resolved into 10 LGs (Table <a href="#tpg270028-supinfo-0001" class="usa-link">S5</a>; Figure <a href="#tpg270028-fig-0004" class="usa-link">4</a>, Figures <a href="#tpg270028-supinfo-0001" class="usa-link">S3–S5</a>). Initially, 5786 SNP markers were detected across 416 F<sub>1</sub> pseudo‐testcross genotypes. After filtration, 871 markers specific to maternal test crosses were identified; 288 showed segregation distortion and were excluded, leaving 583 for mapping. The female parent had a total genetic map covering 1543.93 cM, with individual chromosomes ranging from 106.11 to 203.65 cM. The average inter‐marker distance was 4.12 cM, and the number of SNPs per LG ranged from 13 to 88. For the paternal testcross, 846 markers were identified, and 67% were excluded due to segregation distortion, leaving 279 SNPs to construct the male genetic map. The male genetic map comprised a total distance of 826 cM, with individual chromosomes ranging from 18.44 to 134.16 cM. The average inter‐marker distance was 4.86 cM, and the number of SNPs per LG ranged from 2 to 77. For the integrated male and female maps, 189 of 339 markers were used, covering 974.11 cM. The lengths of the LGs varied between 53.23 cM and 147.46 cM, and the average inter‐marker distance was 6.70 cM.</p>
<figure class="fig xbox font-sm" id="tpg270028-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e5b/11981981/65d49290be2c/TPG2-18-e70028-g002.jpg" loading="lazy" id="jats-graphic-7" height="624" width="798" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270028-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Quantitative trait loci (QTL) detected in the female parent of an asparagus F<sub>1</sub> pseudo‐testcross population for natural levels of anthocyanin pigment at scale leaves (ASL), anthocyanin pigment levels at spear purple spot lesions (APS), number of purple spots on spears (NPS), and purple spot severity in fern (PSF); black bar: detected only in 2021; red bar: detected only in 2022; green bar: detected based on the mean of data from 2 years. For QTL, the bar represents a 1‐LOD (where LOD is logarithm of odds) support interval, and the line represents a 2‐LOD support interval.</p></figcaption></figure><p>QTL were identified only with the female genetic map; no QTL were detected in the male or integrated maps due to insufficient resolution caused by low marker density and high segregation distortion (Table <a href="#tpg270028-tbl-0004" class="usa-link">4</a>; Figure <a href="#tpg270028-fig-0004" class="usa-link">4</a>). For ASL, two stable QTL (qASL4.1, and qASL8.1), one in each of LG 4 and 8, were observed, which together accounted for 8.01% of the phenotypic variation. One tentative QTL (qASL1.1), also identified in LG 1, explained 7.27% of the phenotypic variance. Allelic effects for the three QTL ranged from 0.73 to 1.00, relative to the 1–9 rating scale; all three were positive. Two stable QTL for APS (qAPS4.1 and qAPS10.1) in LGs 4 and 10 explained 7.38% of phenotypic variance. Four tentative QTL (qAPS4.2, qAPS4.3, qAPS6.1, and qAPS8.1) were also identified, explaining 15.58% of the phenotypic variance. For APS, the allelic effects of the six QTL ranged from 0.47 to 0.76, relative to the 1 to 9 rating scale; four were positive, and two were negative. For NPS, two stable QTL (qNPS7.1 and qNPS8.1) in LGs 7 and 8 explained 10.73% of the observed phenotypic variance. Two tentative QTL (qNPS2.1 and qNPS4.1) were identified in LGs 2 and 4, contributing 14.27% of the phenotypic variance. The allelic effects of the four QTL ranged from 2.38 to 6.01, and all were positive. Three tentative QTL (qPSF 4.1, qPSF 7.1, and qPSF 8.1) were identified for PSF in LGs 4, 7, and 8, contributing to 9.98% of observed phenotypic variance. The allelic effects of the three QTL ranged from 0.51 to 0.76, relative to the 0–9 rating; two were positive, and one was negative. Notably, there was an overlap of one QTL for each of the PSF and APS, suggesting shared genetic determinants may be influencing the two traits.</p>
<section class="tw xbox font-sm" id="tpg270028-tbl-0004"><h4 class="obj_head">TABLE 4.</h4>
<div class="caption p"><p>Quantitative trait loci (QTL) for natural levels of anthocyanin pigment in spear scale leaves (ASL), anthocyanin pigment levels at spear purple spot lesions (APS), number of purple spot lesions in spears (NPS), and purple spot severity in fern (PSF) based on a 1–9 visual phenotypic scale detected in the female parent of an asparagus F<sub>1</sub> pseudo‐testcross population assessed across 2 years, 2021 and 2022.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">QTL code</th>
<th align="left" rowspan="1" colspan="1">Year<a href="#tpg270028-tbl4-note-0001" class="usa-link">
<sup>a</sup>
</a>
</th>
<th align="left" rowspan="1" colspan="1">Chr.</th>
<th align="left" rowspan="1" colspan="1">Position (cM)</th>
<th align="left" rowspan="1" colspan="1">Peak LOD</th>
<th align="left" rowspan="1" colspan="1">Allele effect</th>
<th align="left" rowspan="1" colspan="1">
<em>R</em>
<sup>2</sup>
</th>
<th align="left" rowspan="1" colspan="1">LSI</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">qASL1.1</td>
<td align="left" rowspan="1" colspan="1">2021</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">33.41</td>
<td align="left" rowspan="1" colspan="1">3.15</td>
<td align="left" rowspan="1" colspan="1">1.01(+)</td>
<td align="left" rowspan="1" colspan="1">0.0727</td>
<td align="left" rowspan="1" colspan="1">32.40–34.90</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">qASL4.1</td>
<td align="left" rowspan="1" colspan="1">2021, 2021–2022</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">104.11</td>
<td align="left" rowspan="1" colspan="1">3.91</td>
<td align="left" rowspan="1" colspan="1">0.85(+)</td>
<td align="left" rowspan="1" colspan="1">0.0404</td>
<td align="left" rowspan="1" colspan="1">103.60–107.40</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">qASL8.1</td>
<td align="left" rowspan="1" colspan="1">2022, 2021–22</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">32.81</td>
<td align="left" rowspan="1" colspan="1">3.82</td>
<td align="left" rowspan="1" colspan="1">0.73(+)</td>
<td align="left" rowspan="1" colspan="1">0.0397</td>
<td align="left" rowspan="1" colspan="1">32.70–33.30</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">qAPS4.1</td>
<td align="left" rowspan="1" colspan="1">2021, 2022</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">3.01</td>
<td align="left" rowspan="1" colspan="1">2.91</td>
<td align="left" rowspan="1" colspan="1">0.72(+)</td>
<td align="left" rowspan="1" colspan="1">0.0390</td>
<td align="left" rowspan="1" colspan="1">0–10</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">qAPS4.2</td>
<td align="left" rowspan="1" colspan="1">2022</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">33.11</td>
<td align="left" rowspan="1" colspan="1">2.95</td>
<td align="left" rowspan="1" colspan="1">0.47(−)</td>
<td align="left" rowspan="1" colspan="1">0.0412</td>
<td align="left" rowspan="1" colspan="1">33.10–35</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">qAPS4.3</td>
<td align="left" rowspan="1" colspan="1">2021</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">135.41</td>
<td align="left" rowspan="1" colspan="1">3.29</td>
<td align="left" rowspan="1" colspan="1">0.76(+)</td>
<td align="left" rowspan="1" colspan="1">0.0332</td>
<td align="left" rowspan="1" colspan="1">134.30–142</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">qAPS6.1</td>
<td align="left" rowspan="1" colspan="1">2022</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">37.01</td>
<td align="left" rowspan="1" colspan="1">2.98</td>
<td align="left" rowspan="1" colspan="1">0.47(+)</td>
<td align="left" rowspan="1" colspan="1">0.0412</td>
<td align="left" rowspan="1" colspan="1">35.60–38</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">qAPS8.1</td>
<td align="left" rowspan="1" colspan="1">2021</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">142.51</td>
<td align="left" rowspan="1" colspan="1">3.96</td>
<td align="left" rowspan="1" colspan="1">0.73(+)</td>
<td align="left" rowspan="1" colspan="1">0.0402</td>
<td align="left" rowspan="1" colspan="1">139.7–143</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">qAPS10.1</td>
<td align="left" rowspan="1" colspan="1">2021, 2022</td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">0.01</td>
<td align="left" rowspan="1" colspan="1">3.28</td>
<td align="left" rowspan="1" colspan="1">0.66(−)</td>
<td align="left" rowspan="1" colspan="1">0.0348</td>
<td align="left" rowspan="1" colspan="1">0–8</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">qNPS2.1</td>
<td align="left" rowspan="1" colspan="1">2022</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">61.41</td>
<td align="left" rowspan="1" colspan="1">3.86</td>
<td align="left" rowspan="1" colspan="1">6.01(+)</td>
<td align="left" rowspan="1" colspan="1">0.1127</td>
<td align="left" rowspan="1" colspan="1">59.30–63.90</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">qNPS4.1</td>
<td align="left" rowspan="1" colspan="1">2021</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">37.91</td>
<td align="left" rowspan="1" colspan="1">2.82</td>
<td align="left" rowspan="1" colspan="1">2.38(+)</td>
<td align="left" rowspan="1" colspan="1">0.030</td>
<td align="left" rowspan="1" colspan="1">45.80–48.70</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">qNPS7.1</td>
<td align="left" rowspan="1" colspan="1">2021, 2022</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">119.91</td>
<td align="left" rowspan="1" colspan="1">3.26</td>
<td align="left" rowspan="1" colspan="1">3.71 (+)</td>
<td align="left" rowspan="1" colspan="1">0.0662</td>
<td align="left" rowspan="1" colspan="1">118.80–122</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">qNPS8.1</td>
<td align="left" rowspan="1" colspan="1">2021, 2022</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">85.51</td>
<td align="left" rowspan="1" colspan="1">3.94</td>
<td align="left" rowspan="1" colspan="1">3.38 (+)</td>
<td align="left" rowspan="1" colspan="1">0.0411</td>
<td align="left" rowspan="1" colspan="1">84.50–86.30</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">qPSF4.1</td>
<td align="left" rowspan="1" colspan="1">2021‐22</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">137.01</td>
<td align="left" rowspan="1" colspan="1">3.11</td>
<td align="left" rowspan="1" colspan="1">0.76(+)</td>
<td align="left" rowspan="1" colspan="1">0.0332</td>
<td align="left" rowspan="1" colspan="1">133.60–143.70</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">qPSF7.1</td>
<td align="left" rowspan="1" colspan="1">2021</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">72.31</td>
<td align="left" rowspan="1" colspan="1">2.75</td>
<td align="left" rowspan="1" colspan="1">0.51(+)</td>
<td align="left" rowspan="1" colspan="1">0.0304</td>
<td align="left" rowspan="1" colspan="1">71.50–73.70</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">qPSF8.1</td>
<td align="left" rowspan="1" colspan="1">2022</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">34.71</td>
<td align="left" rowspan="1" colspan="1">3.25</td>
<td align="left" rowspan="1" colspan="1">0.59(−)</td>
<td align="left" rowspan="1" colspan="1">0.0362</td>
<td align="left" rowspan="1" colspan="1">34–34.90</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270028-tbl-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tpg270028-tbl4-note-0001">
<sup>
<sup>a</sup>
</sup><p class="display-inline">Individual years; 2021–2022 = mean over 2 years when QTL detected.</p>
</div>
<div class="fn" id="tpg270028-tbl4-note-0002"><p>Abbreviations: Chr., chromosome number; LOD, logarithm of odds; LSI, 1‐LOD support interval in cM; <em>R</em>
<sup>2</sup>, coefficient of determination.</p></div>
</div></section><p>Candidate genes identified for QTL included crocetin glucosyltransferase chloroplast‐like (qAPS4.2) for APS, V‐type proton ATPase subunit e1‐like (qNPS4.1), and GDSL esterase/lipase EXL3‐like (qNPS7.1) for NPS, and homeobox‐leucine zipper HOX1‐like (qPSF 8.1), and ubiquitin‐conjugating E2 22‐like (qPSF 8.1) for PSF (Table <a href="#tpg270028-tbl-0005" class="usa-link">5</a>; Table <a href="#tpg270028-supinfo-0001" class="usa-link">S6</a>). No candidate genes were found for ASL.</p>
<section class="tw xbox font-sm" id="tpg270028-tbl-0005"><h4 class="obj_head">TABLE 5.</h4>
<div class="caption p"><p>Identification and characterization of asparagus candidate genes for anthocyanin pigment levels at spear purple spot lesions (APS), number of purple spot lesions in spear (NPS), and purple spot severity in fern (PSF) via NCBI Basic Local Alignment Search Tool (BLAST) search against the asparagus reference genome.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">QTL</th>
<th align="left" rowspan="1" colspan="1">Candidate gene name</th>
<th align="left" rowspan="1" colspan="1">NCBI gene I.D.</th>
<th align="left" rowspan="1" colspan="1">Gene type</th>
<th align="left" rowspan="1" colspan="1">
<em>E</em>‐value</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">APS</td>
<td align="left" rowspan="1" colspan="1">qAPS4.2</td>
<td align="left" rowspan="1" colspan="1">
<em>Crocetin glucosyltransferase, chloroplastic‐like</em>
</td>
<td align="left" rowspan="1" colspan="1">109,836,841</td>
<td align="left" rowspan="1" colspan="1">Protein coding</td>
<td align="left" rowspan="1" colspan="1">3E‐153</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">NPS</td>
<td align="left" rowspan="1" colspan="1">qNPS4.1</td>
<td align="left" rowspan="1" colspan="1">
<em>V‐type proton ATPase subunit e1‐like</em>
</td>
<td align="left" rowspan="1" colspan="1">109,846,756</td>
<td align="left" rowspan="1" colspan="1">Protein coding</td>
<td align="left" rowspan="1" colspan="1">8E‐39</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">NPS</td>
<td align="left" rowspan="1" colspan="1">qNPS7.1</td>
<td align="left" rowspan="1" colspan="1">
<em>GDSL esterase/lipase EXL3‐like</em>
</td>
<td align="left" rowspan="1" colspan="1">109,837,676</td>
<td align="left" rowspan="1" colspan="1">Protein coding</td>
<td align="left" rowspan="1" colspan="1">2E‐93</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PSF</td>
<td align="left" rowspan="1" colspan="1">qPSF 8.1</td>
<td align="left" rowspan="1" colspan="1">
<em>Homeobox‐leucine zipper HOX1‐like</em>
</td>
<td align="left" rowspan="1" colspan="1">109,843,823</td>
<td align="left" rowspan="1" colspan="1">Protein coding</td>
<td align="left" rowspan="1" colspan="1">4E‐32</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PSF</td>
<td align="left" rowspan="1" colspan="1">qPSF 8.1</td>
<td align="left" rowspan="1" colspan="1">
<em>Ubiquitin‐conjugating E2 22‐like</em>
</td>
<td align="left" rowspan="1" colspan="1">109,841,370</td>
<td align="left" rowspan="1" colspan="1">Protein coding</td>
<td align="left" rowspan="1" colspan="1">5E‐46</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270028-tbl-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270028-tbl5-note-0001"><p>Abbreviation: QTL, quantitative trait loci.</p></div></div></section></section></section><section id="tpg270028-sec-0180"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>Distinct inheritance patterns were observed for red:purple segregation in asparagus spear scale leaves and purple spot lesions, with ratios of 3:1 and 3:13, respectively. In addition, the two tissues exhibited different cyanidin:peonidin ratios for the two phenotypic classes. Low to near‐zero correlations were observed between ASL and APS, NPS, and PSF, and NPS and APS. Distinct stable and tentative QTL were identified for ASL, APS, NPS, and PSF, except one for each of PSF and APS were colocalized. The data suggested that independent genetic regulatory pathways are likely responsible for pigment produced in scale leaves and purple spot lesions, as well as resistance mechanisms for purple spot infection in spear and fern. Candidate genes were identified for APS, NPS, and PSF.</p>
<section id="tpg270028-sec-0190"><h3 class="pmc_sec_title">4.1. Anthocyanin regulation in scale leaves and purple spot lesions</h3>
<p>The observed 3 red: 1 purple segregation in scale leaves and 3 red: 13 purple segregation in purple spot lesions suggested distinct genetic mechanisms: a single gene regulating pigmentation in the scale leaves and two epistatically interacting genes controlling pigmentation in purple spot lesions. Although additional crosses are required to validate the genetic models, these results strongly indicate tissue‐specific genetic regulation, which is plausible and not exceptional. Previous studies in carrot (<em>Daucus carota</em> L.) (Bannoud et al., <a href="#tpg270028-bib-0004" class="usa-link" aria-describedby="tpg270028-bib-0004">2021</a>), grapes (<em>Vitis vinifera</em> L.) (Xie et al., <a href="#tpg270028-bib-0061" class="usa-link" aria-describedby="tpg270028-bib-0061">2015</a>), maize (<em>Zea mays</em> L.) (Chatham &amp; Juvik, <a href="#tpg270028-bib-0009" class="usa-link" aria-describedby="tpg270028-bib-0009">2021</a>), and Arabidopsis (<em>Arabidopsis thaliana</em> (L.) Heynh.) (Falcone Ferreyra et al., <a href="#tpg270028-bib-0014" class="usa-link" aria-describedby="tpg270028-bib-0014">2012</a>) have similarly documented differential regulation of anthocyanin biosynthetic genes across specific tissues. Additionally, tissue‐specific anthocyanin regulation in lesions and scale leaves of asparagus spears is further supported by C:P ratios, which were similar in red and purple lesions but varied between the color classes in scale leaves. Tissue‐specific variation in cellular pH, which affects pigment hue (Khoo et al., <a href="#tpg270028-bib-0034" class="usa-link" aria-describedby="tpg270028-bib-0034">2017</a>), and co‐pigmentation with flavonoids, which can modify and intensify colors (Asen et al., <a href="#tpg270028-bib-0002" class="usa-link" aria-describedby="tpg270028-bib-0002">1972</a>; Boulton, <a href="#tpg270028-bib-0007" class="usa-link" aria-describedby="tpg270028-bib-0007">2001</a>), may explain anomalies with C:P ratios.</p></section><section id="tpg270028-sec-0200"><h3 class="pmc_sec_title">4.2. Purple spot resistance in spears versus fern</h3>
<p>A low correlation between NPS and PSF suggests that purple spot resistance mechanisms are genetically distinct in fern and spear. These findings are consistent with those of Foster et al. (<a href="#tpg270028-bib-0018" class="usa-link" aria-describedby="tpg270028-bib-0018">2019</a>), who observed similar results in an analysis of asparagus hybrids, which parallel those of other species. In potato (<em>Solanum tuberosum</em> L.), the quantitative resistance of foliage to late blight, caused by <em>Phytophthora infestans</em> (Mont.) de Bary, was weakly correlated with tuber resistance, suggesting involvement of tissue‐specific gene or differential expression (Vale et al., <a href="#tpg270028-bib-0055" class="usa-link" aria-describedby="tpg270028-bib-0055">2001</a>). Similarly, Kawuki et al. (<a href="#tpg270028-bib-0033" class="usa-link" aria-describedby="tpg270028-bib-0033">2019</a>) reported a consistently low correlation between disease severities of cassava (<em>Manihot esculenta</em> Crantz) brown streak disease on leaves and roots, further supporting the concept of tissue‐specific resistance mechanisms. Consequently, selection for resistance in one tissue will not simultaneously enhance levels in the other, and the breeding program would need to address asparagus spear and fern resistance separately.</p></section><section id="tpg270028-sec-0210"><h3 class="pmc_sec_title">4.3. Relationship between anthocyanin pigment levels and purple spot resistance in spears and fern</h3>
<p>The low correlations observed between ASL and APS with NPS or PSF suggested anthocyanin plays no role in disease susceptibility or resistance in asparagus. These results add to the ongoing debate within the scientific literature about the role of anthocyanins in plant defense, which appears to be highly crop‐ and disease‐specific. For example, studies in wheat (<em>Triticum aestivum</em> L.) demonstrated that anthocyanins can enhance resistance to Fusarium head blight (<em>Fusarium graminearum</em> Schwabe.) (Gozzi et al., <a href="#tpg270028-bib-0020" class="usa-link" aria-describedby="tpg270028-bib-0020">2023</a>), whereas research on apples (<em>Malus domestica</em> Borkh.) has shown that anthocyanins may have no effect or even increase susceptibility to certain pathogens (Hajnajari et al., <a href="#tpg270028-bib-0024" class="usa-link" aria-describedby="tpg270028-bib-0024">2012</a>). Further analyses focusing on individual anthocyanin compounds, rather than total anthocyanin content, are warranted to understand if specific associations with purple spot resistance occur in asparagus, which can enhance breeding efforts.</p></section><section id="tpg270028-sec-0220"><h3 class="pmc_sec_title">4.4. QTL mapping for anthocyanin level and purple spot resistance in spears and fern</h3>
<p>Multiple stable and tentative QTL were identified for APS and ASL, which did not overlap, and phenotypic correlations between traits were low (0.21–0.33), although significant, further supporting tissue‐specific regulation of anthocyanin production. Multiple QTL have been identified regulating pigment in carrot xylem and phloem (Bannoud et al., <a href="#tpg270028-bib-0004" class="usa-link" aria-describedby="tpg270028-bib-0004">2021</a>), and 17 QTL controlled anthocyanin‐related traits in eggplant (<em>Solanum melongena</em> L.) (Toppino et al., <a href="#tpg270028-bib-0054" class="usa-link" aria-describedby="tpg270028-bib-0054">2020</a>), indicating a complex genetic network. Multiple stable and tentative QTL were observed for NPS and PSF. Loci for the two traits did not overlap, suggesting independent resistance mechanisms in the two tissues, which was also supported by low phenotypic correlations between traits.</p>
<p>No major QTL, explaining 15% or greater of the phenotypic variance, were observed in this study, which could partly be explained by limited marker density. A moderate number of markers from the female parent, 67%, could be mapped to the reference genome. A similar percentage of markers were eliminated from the male map due to segregation distortion. Improved genomics tools to map increased numbers of markers to a reference genome and enhanced understanding of high segregation distortion will support precise QTL mapping in the future.</p></section><section id="tpg270028-sec-0230"><h3 class="pmc_sec_title">4.5. Candidate genes analysis for anthocyanin levels and purple spot resistance in spear and fern</h3>
<p>Five candidate genes that may be involved in controlling disease resistance and anthocyanin accumulation in asparagus were identified within the confidence intervals of some QTL. The gene encoding crocetin glucosyltransferase chloroplast‐like protein was identified as a candidate for controlling APS due to its role in crocetin glycosylation, which stabilizes anthocyanins and enhances their solubility in the plant's cellular environment (Kapoor et al., <a href="#tpg270028-bib-0032" class="usa-link" aria-describedby="tpg270028-bib-0032">2022</a>). Two genes were identified as candidates for controlling NPS: the V‐type proton ATPase subunit e1‐like protein, which regulates critical cellular processes such as homeostasis, hypersensitivity, stress response, and programmed cell death (Seidel, <a href="#tpg270028-bib-0050" class="usa-link" aria-describedby="tpg270028-bib-0050">2022</a>; Z. Zhang et al., <a href="#tpg270028-bib-0063" class="usa-link" aria-describedby="tpg270028-bib-0063">2008</a>), and the GDSL esterase/lipase EXL3‐like protein, which mediates cell wall fortification and defense signaling during pathogen attacks (Ji et al., <a href="#tpg270028-bib-0031" class="usa-link" aria-describedby="tpg270028-bib-0031">2023</a>). Moreover, the genes for homeobox‐leucine zipper HOX1‐like protein and ubiquitin‐conjugating E2 22‐like protein (UBC22‐like) were associated with PSF. Homeobox‐leucine zipper HOX1‐like protein plays a vital role in disease resistance by activating defense‐related genes for synthesizing antimicrobial compounds, pathogenesis‐related proteins, and defense signal molecules, as reported in rice (<em>Oryza sativa</em> L.) and cotton (<em>Gossypium hirsutum</em> L.) (He et al., <a href="#tpg270028-bib-0027" class="usa-link" aria-describedby="tpg270028-bib-0027">2020</a>; Salvador‐Guirao et al., <a href="#tpg270028-bib-0049" class="usa-link" aria-describedby="tpg270028-bib-0049">2018</a>). The ubiquitin‐conjugating E2 22‐like protein facilitates post‐translational modification by attaching ubiquitin molecules to target proteins, thereby regulating their stability and localization (Wang &amp; Wang, <a href="#tpg270028-bib-0060" class="usa-link" aria-describedby="tpg270028-bib-0060">2021</a>). The ubiquitin‐conjugating E2 22‐like protein may also play a role in eliminating pathogen effectors, as evidenced by the studies on wheat and potatoes (S. Zhang et al., <a href="#tpg270028-bib-0062" class="usa-link" aria-describedby="tpg270028-bib-0062">2022</a>; Zhou &amp; Zeng, <a href="#tpg270028-bib-0064" class="usa-link" aria-describedby="tpg270028-bib-0064">2017</a>). These candidate genes provide valuable targets for functional validation through gene overexpression or knockout studies. Incorporating important loci into the breeding program via marker‐assisted selection (MAS) could significantly improve resistance while maintaining desirable pigment traits.</p></section></section><section id="tpg270028-sec-0240"><h2 class="pmc_sec_title">5. CONCLUSIONS</h2>
<p>This study provides important insights into the genetic basis of purple spot resistance and anthocyanin pigment variation in asparagus through genetic analysis. Tissue‐specific genetic regulation was observed for color, red versus purple, in spear scale leaves and purple spot lesions. QTL were identified for ASL, APS, NPS, and PSF. Complex and distinct inheritance patterns were observed for ASL and APS, indicating the involvement of different genetic pathways. NPS and PSF were not correlated, and QTL for the two traits did not colocalize, indicating that the selection for resistance in the spears does not increase or decrease resistance in the fern. The identification of candidate genes provides a foundation for future genetic and molecular studies, providing pathways for developing asparagus cultivars with improved resistance to purple spot and regulated anthocyanin content, potentially increasing the marketability of the crop.</p></section><section id="tpg270028-sec-0250"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Suman Parajuli</strong>: Data curation; formal analysis; investigation; methodology; software; visualization; writing—original draft. <strong>Mary Ruth McDonald</strong>: Methodology; resources; writing—review and editing. <strong>Laxman Adhikari</strong>: Formal analysis; methodology; software. <strong>David J. Wolyn</strong>: Conceptualization; funding acquisition; methodology; writing—review and editing.</p></section><section id="tpg270028-sec-0270"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec27"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270028-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270028-supitem-0001"><div class="caption p">
<p>
<strong>Supplementary Table S1</strong>: Anthocyanin pigments detected by LC/MS in asparagus scale leaves and purple spot lesions in an F<sub>1</sub> pseudo‐test cross population.</p>
<p>
<strong>Supplementary Table S2</strong>: Quantification of anthocyanin pigment types in asparagus scale leaves and purple spot lesions using LC/MS, visually rated as purple or red in an F<sub>1</sub> pseudo‐testcross population.</p>
<p>
<strong>Supplementary Table S3</strong>: Total anthocyanin pigment concentrations, as determined by spectrophotometry, for nine asparagus scale leaf pigment phenotypes based on visual ratings of an F<sub>1</sub> pseudo‐testcross population.</p>
<p>
<strong>Supplementary Table S4</strong>: Analysis of variance of the natural levels of anthocyanin pigment at scale leaves (ASL), anthocyanin pigment levels at spear purple spot lesions (APS), number of purple spot lesions in spears (NPS), and purple spot severity in fern (PSF) for an asparagus F<sub>1</sub> pseudo‐testcross population across two years (2021 and 2022) and average over years.</p>
<p>
<strong>Supplementary Table S5</strong>
<span class="text-underline">:</span> Summary of genetic maps constructed for an asparagus F<sub>1</sub> pseudo‐testcross population (n = 416).</p>
<p>
<strong>Supplementary Table S6</strong>
<span class="text-underline">:</span> Quantitative trait loci (QTL) characterization for anthocyanin pigment levels at spear purple spot lesions (APS), number of purple spot lesions in spears (NPS) and purple spot severity in fern (PSF) for asparagus.</p>
<p>
<strong>Supplementary Figure S1</strong>: Different levels of purple spot infection in an asparagus F<sub>1</sub> pseudo‐testcross population. 0: Highly resistant‐ fern displayed no signs of disease; 1: Resistant‐fern remained predominantly green with minimal cladophyll drop and no noticeable symptoms; 3: Moderately resistant‐fern retained green coloration, but with some cladophyll drop and few instances of purple leaf spot; 5: Moderately susceptible‐ fern showed a noticeable number of cladophylls dropped and purple leaf spot, yet without any stem desiccation; 7: Susceptible fern with significant cladophyll dropping and purple spot presence, accompanied by partial stem dryness and yellowing; 9: Highly susceptible fern with complete yellowing and extensive stem desiccation.</p>
<p>
<strong>Supplementary Figure S2</strong>: Distribution of anthocyanin phenotypic classes (1 = low pigment level, 9 = high pigment level) in scale leaf (A, B, and C) and purple spot lesions (D, E, and F); numbers of purple spot lesions in spears (G, H, and I) and purple spot severity in fern J, K, and L) in an asparagus F<sub>1</sub> pseudo‐testcross population in 2021 (A, D, G, J), 2022 (B, E, H, K), and average of 2021–2022 data (C, F, I, L). The green and red vertical lines represent two control cultivars, Guelph Millenium, and Guelph Eclipse, respectively.</p>
<p>
<strong>Supplementary Figure S3</strong>: Genetic linkage map of the female parent for an asparagus F<sub>1</sub> pseudo‐testcross population. The chromosome number is represented as the first number (e.g., 1.1, 2.1), following underscore are the position of markers in the reference genome. The scale bar indicates the distance in centimorgans (cM).</p>
<p>
<strong>Supplementary Figure S4</strong>: Genetic linkage map of asparagus male parent. The chromosome number is represented as the first number (e.g., 1.1, 2.1), following underscore are the position of markers in the reference genome. The scale bar indicates the distance in centimorgans (cM).</p>
<p>
<strong>Supplementary Figure S5</strong>: Integrated genetic linkage map of asparagus. The chromosome number is represented as the first number (e.g., 1.1, 2.1), following underscore are the position of markers in the reference genome. The scale bar indicates the distance in centimorgans (cM).</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11981981/bin/TPG2-18-e70028-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70028-s001.docx</a><sup> (3.7MB, docx) </sup>
</div></div></section></section><section id="tpg270028-sec-0260" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>This work was supported by the Canadian Agri‐science cluster for Horticulture 3, in cooperation with Agriculture and Agri‐food Canada's Agriscience Program, a Canadian Agricultural Partnership Initiative, the Fruit &amp; Vegetable Growers of Canada, the Asparagus Growers of Ontario, and the Ontario Ministry of Agriculture, Food and Rural Affairs. We express our gratitude to Dr. Lili Mats, Guelph Research and Development Centre, Agriculture and Agri‐food Canada laboratory, for conducting the LC/MS analysis of anthocyanins.</p></section><section id="notes1"><p>


Parajuli, S.
, 
McDonald, M. R.
, 
Adhikari, L.
, &amp; 
Wolyn, D. J.
 (2025). Genetic architecture of anthocyanin pigment traits and purple spot (<em>Stemphylium vesicarium</em>) resistance in an F<sub>1</sub> pseudo‐testcross population of asparagus. The Plant Genome, 18, e70028. 10.1002/tpg2.70028


</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270028-note-0001"><p>Assigned to Associate Editor Alexander Lipka.</p></div></div></section></section><section id="tpg270028-sec-0290"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>Phenotypic and genotypic data used in this study are available on Dryad at <a href="https://doi.org/10.5061/dryad.nzs7h452b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5061/dryad.nzs7h452b</a>. The raw sequencing data supporting this study have been deposited in the NCBI (SRA) under the BioProject accession number PRJNA1169105, available at <a href="https://www.ncbi.nlm.nih.gov/sra/PRJNA1169105" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/sra/PRJNA1169105</a>.</p></section><section id="tpg270028-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270028-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270028-bib-0001">
<cite>

Aravind, J.
, 
Mukesh Sankar, S.
, 
Wankhede, D.
, &amp; 
Kaur, V.
 (2021). 
<em>augmentedRCBD: Analysis of augmented randomised complete block designs</em> (Version 0.1.2) [R package]. Comprehensive R Archive Network (CRAN). <a href="https://aravind-j.github.io/augmentedRCBD/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://aravind‐j.github.io/augmentedRCBD/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?Aravind,%20J.%20,%20Mukesh%20Sankar,%20S.%20,%20Wankhede,%20D.%20,%20&amp;%20Kaur,%20V.%20(2021).%20augmentedRCBD:%20Analysis%20of%20augmented%20randomised%20complete%20block%20designs%20(Version%200.1.2)%20%5BR%20package%5D.%20Comprehensive%20R%20Archive%20Network%20(CRAN).%20https://aravind%E2%80%90j.github.io/augmentedRCBD/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0002">
<cite>

Asen, S.
, 
Stewart, R.
, &amp; 
Norris, K.
 (1972). Co‐pigmentation of anthocyanins in plant tissues and its effect on color. Phytochemistry, 11(3), 1139–1144. 10.1016/S0031-9422(00)88467-8
</cite> [<a href="https://doi.org/10.1016/S0031-9422(00)88467-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phytochemistry&amp;title=Co%E2%80%90pigmentation%20of%20anthocyanins%20in%20plant%20tissues%20and%20its%20effect%20on%20color&amp;author=S.%20Asen&amp;author=R.%20Stewart&amp;author=K.%20Norris&amp;volume=11&amp;issue=3&amp;publication_year=1972&amp;pages=1139-1144&amp;doi=10.1016/S0031-9422(00)88467-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0003">
<cite>

Austin, G.
 (2023). A bioassay for asparagus (Asparagus officinalis L.) resistance to Stemphylium vesicarium [Master's thesis, University of Guelph]. <a href="https://atrium.lib.uoguelph.ca/items/28574f99-6357-43c4-afd8-fde53ad0c0e2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://atrium.lib.uoguelph.ca/items/28574f99‐6357‐43c4‐afd8‐fde53ad0c0e2</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=A%20bioassay%20for%20asparagus%20(Asparagus%20officinalis%20L.)%20resistance%20to%20Stemphylium%20vesicarium&amp;author=G.%20Austin&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0004">
<cite>

Bannoud, F.
, 
Carvajal, S.
, 
Ellison, S.
, 
Senalik, D.
, 
Gomez Talquenca, S.
, 
Iorizzo, M.
, 
Simon, P. W.
, &amp; 
Cavagnaro, P. F.
 (2021). Genetic and transcription profile analysis of tissue‐specific anthocyanin pigmentation in carrot root phloem. Genes, 12(10), 1464. 10.3390/genes12101464

</cite> [<a href="https://doi.org/10.3390/genes12101464" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8535259/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34680859/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes&amp;title=Genetic%20and%20transcription%20profile%20analysis%20of%20tissue%E2%80%90specific%20anthocyanin%20pigmentation%20in%20carrot%20root%20phloem&amp;author=F.%20Bannoud&amp;author=S.%20Carvajal&amp;author=S.%20Ellison&amp;author=D.%20Senalik&amp;author=S.%20Gomez%20Talquenca&amp;volume=12&amp;issue=10&amp;publication_year=2021&amp;pages=1464&amp;pmid=34680859&amp;doi=10.3390/genes12101464&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0005">
<cite>

Bartlett, M. S.
 (1937). Properties of sufficiency and statistical test. Proceedings of the Royal Society A, 160(901), 268–282. 10.1098/rspa.1937.0109
</cite> [<a href="https://doi.org/10.1098/rspa.1937.0109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20Royal%20Society%20A&amp;title=Properties%20of%20sufficiency%20and%20statistical%20test&amp;author=M.%20S.%20Bartlett&amp;volume=160&amp;issue=901&amp;publication_year=1937&amp;pages=268-282&amp;doi=10.1098/rspa.1937.0109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0006">
<cite>

Bennett, M. D.
, &amp; 
Leitch, I. J.
 (2011). Nuclear DNA amounts in angiosperms: Targets, trends and tomorrow. Annals of Botany, 107(3), 467–590. 10.1093/aob/mcq258

</cite> [<a href="https://doi.org/10.1093/aob/mcq258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3043933/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21257716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Botany&amp;title=Nuclear%20DNA%20amounts%20in%20angiosperms:%20Targets,%20trends%20and%20tomorrow&amp;author=M.%20D.%20Bennett&amp;author=I.%20J.%20Leitch&amp;volume=107&amp;issue=3&amp;publication_year=2011&amp;pages=467-590&amp;pmid=21257716&amp;doi=10.1093/aob/mcq258&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0007">
<cite>

Boulton, R.
 (2001). The copigmentation of anthocyanins and its role in the color of red wine: A critical review. American Journal of Enology and Viticulture, 52(2), 67–87. 10.5344/ajev.2001.52.2.67
</cite> [<a href="https://doi.org/10.5344/ajev.2001.52.2.67" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Enology%20and%20Viticulture&amp;title=The%20copigmentation%20of%20anthocyanins%20and%20its%20role%20in%20the%20color%20of%20red%20wine:%20A%20critical%20review&amp;author=R.%20Boulton&amp;volume=52&amp;issue=2&amp;publication_year=2001&amp;pages=67-87&amp;doi=10.5344/ajev.2001.52.2.67&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0008">
<cite>

Broman, K. W.
, 
Wu, H.
, 
Sen, Ś.
, &amp; 
Churchill, G. A.
 (2003). R/qtl: QTL mapping in experimental crosses. Bioinformatics, 19(7), 889–890. 10.1093/bioinformatics/btg112

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btg112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12724300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=R/qtl:%20QTL%20mapping%20in%20experimental%20crosses&amp;author=K.%20W.%20Broman&amp;author=H.%20Wu&amp;author=%C5%9A.%20Sen&amp;author=G.%20A.%20Churchill&amp;volume=19&amp;issue=7&amp;publication_year=2003&amp;pages=889-890&amp;pmid=12724300&amp;doi=10.1093/bioinformatics/btg112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0009">
<cite>

Chatham, L. A.
, &amp; 
Juvik, J. A.
 (2021). Linking anthocyanin diversity, hue, and genetics in purple corn. G3 Genes|Genomes|Genetics, 11(2), jkaa062. 10.1093/g3journal/jkaa062

</cite> [<a href="https://doi.org/10.1093/g3journal/jkaa062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8022952/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33585872/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Linking%20anthocyanin%20diversity,%20hue,%20and%20genetics%20in%20purple%20corn&amp;author=L.%20A.%20Chatham&amp;author=J.%20A.%20Juvik&amp;volume=11&amp;issue=2&amp;publication_year=2021&amp;pages=jkaa062&amp;pmid=33585872&amp;doi=10.1093/g3journal/jkaa062&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0010">
<cite>

Churchill, G. A.
, &amp; 
Doerge, R.
 (1994). Empirical threshold values for quantitative trait mapping. Genetics, 138(3), 963–971. 10.1093/genetics/138.3.963

</cite> [<a href="https://doi.org/10.1093/genetics/138.3.963" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1206241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7851788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Empirical%20threshold%20values%20for%20quantitative%20trait%20mapping&amp;author=G.%20A.%20Churchill&amp;author=R.%20Doerge&amp;volume=138&amp;issue=3&amp;publication_year=1994&amp;pages=963-971&amp;pmid=7851788&amp;doi=10.1093/genetics/138.3.963&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0011">
<cite>

Cox, L. D.
, 
Munholland, S.
, 
Mats, L.
, 
Zhu, H.
, 
Crosby, W. L.
, 
Lukens, L.
, 
Pauls, K. P.
, &amp; 
Bozzo, G. G.
 (2021). The induction of the isoflavone biosynthesis pathway is associated with resistance to common bacterial blight in <em>Phaseolus vulgaris</em> L. Metabolites, 11(7), 433. 10.3390/metabo11070433

</cite> [<a href="https://doi.org/10.3390/metabo11070433" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8306140/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34357327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Metabolites&amp;title=The%20induction%20of%20the%20isoflavone%20biosynthesis%20pathway%20is%20associated%20with%20resistance%20to%20common%20bacterial%20blight%20in%20Phaseolus%20vulgaris%20L&amp;author=L.%20D.%20Cox&amp;author=S.%20Munholland&amp;author=L.%20Mats&amp;author=H.%20Zhu&amp;author=W.%20L.%20Crosby&amp;volume=11&amp;issue=7&amp;publication_year=2021&amp;pages=433&amp;pmid=34357327&amp;doi=10.3390/metabo11070433&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0012">
<cite>

Dong, T.
, 
Han, R.
, 
Yu, J.
, 
Zhu, M.
, 
Zhang, Y.
, 
Gong, Y.
, &amp; 
Li, Z.
 (2019). Anthocyanins accumulation and molecular analysis of correlated genes by metabolome and transcriptome in green and purple asparaguses (<em>Asparagus officinalis</em>, L.). Food Chemistry, 271, 18–28. 10.1016/j.foodchem.2018.07.120

</cite> [<a href="https://doi.org/10.1016/j.foodchem.2018.07.120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30236664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Chemistry&amp;title=Anthocyanins%20accumulation%20and%20molecular%20analysis%20of%20correlated%20genes%20by%20metabolome%20and%20transcriptome%20in%20green%20and%20purple%20asparaguses%20(Asparagus%20officinalis,%20L.)&amp;author=T.%20Dong&amp;author=R.%20Han&amp;author=J.%20Yu&amp;author=M.%20Zhu&amp;author=Y.%20Zhang&amp;volume=271&amp;publication_year=2019&amp;pages=18-28&amp;pmid=30236664&amp;doi=10.1016/j.foodchem.2018.07.120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0013">
<cite>

Elena, K.
 (2007). Asparagus diseases. European Journal of Plant Science and Biotechnology, 1, 76–83. <a href="https://cir.nii.ac.jp/crid/1370569646446993024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cir.nii.ac.jp/crid/1370569646446993024</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Plant%20Science%20and%20Biotechnology&amp;title=Asparagus%20diseases&amp;author=K.%20Elena&amp;volume=1&amp;publication_year=2007&amp;pages=76-83&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0014">
<cite>

Falcone Ferreyra, M. L.
, 
Rius, S. P.
, &amp; 
Casati, P.
 (2012). Flavonoids: Biosynthesis, biological functions, and biotechnological applications. Frontiers in Plant Science, 3, 222. 10.3389/fpls.2012.00222

</cite> [<a href="https://doi.org/10.3389/fpls.2012.00222" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3460232/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23060891/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Flavonoids:%20Biosynthesis,%20biological%20functions,%20and%20biotechnological%20applications&amp;author=M.%20L.%20Falcone%20Ferreyra&amp;author=S.%20P.%20Rius&amp;author=P.%20Casati&amp;volume=3&amp;publication_year=2012&amp;pages=222&amp;pmid=23060891&amp;doi=10.3389/fpls.2012.00222&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0015">
<cite>

Falloon, P. G.
 (1987). Etiology and epidemiology of Stemphylium leaf spot and purple spot. Phytopathology, 77, 407–413. 10.1094/Phyto-77-407
</cite> [<a href="https://doi.org/10.1094/Phyto-77-407" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phytopathology&amp;title=Etiology%20and%20epidemiology%20of%20Stemphylium%20leaf%20spot%20and%20purple%20spot&amp;author=P.%20G.%20Falloon&amp;volume=77&amp;publication_year=1987&amp;pages=407-413&amp;doi=10.1094/Phyto-77-407&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0016">
<cite>

Federer, W. T.
 (1956). Augmented (or hoonuiaku) designs. The Hawaiian Planters' Record, 55(2), 191–208. <a href="https://ecommons.cornell.edu/handle/1813/32841" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ecommons.cornell.edu/handle/1813/32841</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Hawaiian%20Planters'%20Record&amp;title=Augmented%20(or%20hoonuiaku)%20designs&amp;author=W.%20T.%20Federer&amp;volume=55&amp;issue=2&amp;publication_year=1956&amp;pages=191-208&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0017">
<cite>

Figueroa‐Macías, J. P.
, 
García, Y. C.
, 
Núñez, M.
, 
Díaz, K.
, 
Olea, A. F.
, &amp; 
Espinoza, L.
 (2021). Plant growth‐defense trade‐offs: Molecular processes leading to physiological changes. International Journal of Molecular Sciences, 22(2), 693. 10.3390/ijms22020693

</cite> [<a href="https://doi.org/10.3390/ijms22020693" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7828132/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33445665/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Plant%20growth%E2%80%90defense%20trade%E2%80%90offs:%20Molecular%20processes%20leading%20to%20physiological%20changes&amp;author=J.%20P.%20Figueroa%E2%80%90Mac%C3%ADas&amp;author=Y.%20C.%20Garc%C3%ADa&amp;author=M.%20N%C3%BA%C3%B1ez&amp;author=K.%20D%C3%ADaz&amp;author=A.%20F.%20Olea&amp;volume=22&amp;issue=2&amp;publication_year=2021&amp;pages=693&amp;pmid=33445665&amp;doi=10.3390/ijms22020693&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0018">
<cite>

Foster, J. M.
, 
Tayviah, C. S.
, 
Stricker, S. M.
, 
Gossen, B. D.
, &amp; 
McDonald, M. R.
 (2019). Susceptibility to <em>Stemphylium vesicarium</em> of asparagus, onion, pear, and rye in Canada. Canadian Journal of Plant Pathology, 41(2), 228–241. 10.1080/07060661.2019.1574901
</cite> [<a href="https://doi.org/10.1080/07060661.2019.1574901" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Canadian%20Journal%20of%20Plant%20Pathology&amp;title=Susceptibility%20to%20Stemphylium%20vesicarium%20of%20asparagus,%20onion,%20pear,%20and%20rye%20in%20Canada&amp;author=J.%20M.%20Foster&amp;author=C.%20S.%20Tayviah&amp;author=S.%20M.%20Stricker&amp;author=B.%20D.%20Gossen&amp;author=M.%20R.%20McDonald&amp;volume=41&amp;issue=2&amp;publication_year=2019&amp;pages=228-241&amp;doi=10.1080/07060661.2019.1574901&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0019">
<cite>

Gould, K. S.
, 
Markham, K. R.
, 
Smith, R. H.
, &amp; 
Goris, J. J.
 (2000). Functional role of anthocyanins in the leaves of <em>Quintinia serrata</em> A. Cunn. Journal of Experimental Botany, 51(347), 1107–1115. 10.1093/jexbot/51.347.1107

</cite> [<a href="https://doi.org/10.1093/jexbot/51.347.1107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10948238/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Functional%20role%20of%20anthocyanins%20in%20the%20leaves%20of%20Quintinia%20serrata%20A.%20Cunn&amp;author=K.%20S.%20Gould&amp;author=K.%20R.%20Markham&amp;author=R.%20H.%20Smith&amp;author=J.%20J.%20Goris&amp;volume=51&amp;issue=347&amp;publication_year=2000&amp;pages=1107-1115&amp;pmid=10948238&amp;doi=10.1093/jexbot/51.347.1107&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0020">
<cite>

Gozzi, M.
, 
Blandino, M.
, 
Dall'Asta, C.
, 
Martinek, P.
, 
Bruni, R.
, &amp; 
Righetti, L.
 (2023). Anthocyanin content and <em>Fusarium</em> mycotoxins in pigmented wheat (<em>Triticum aestivum</em> L. spp. <em>aestivum</em>): An open field evaluation. Plants, 12(4), 693. 10.3390/plants12040693

</cite> [<a href="https://doi.org/10.3390/plants12040693" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9965368/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36840042/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Anthocyanin%20content%20and%20Fusarium%20mycotoxins%20in%20pigmented%20wheat%20(Triticum%20aestivum%20L.%20spp.%20aestivum):%20An%20open%20field%20evaluation&amp;author=M.%20Gozzi&amp;author=M.%20Blandino&amp;author=C.%20Dall'Asta&amp;author=P.%20Martinek&amp;author=R.%20Bruni&amp;volume=12&amp;issue=4&amp;publication_year=2023&amp;pages=693&amp;pmid=36840042&amp;doi=10.3390/plants12040693&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0021">
<cite>

Grattapaglia, D.
, &amp; 
Sederoff, R.
 (1994). Genetic linkage maps of <em>Eucalyptus grandis</em> and <em>Eucalyptus urophylla</em> using a pseudo‐testcross mapping strategy and RAPD markers. Genetics, 137(4), 1121–1137. 10.1093/genetics/137.4.1121

</cite> [<a href="https://doi.org/10.1093/genetics/137.4.1121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1206059/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7982566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genetic%20linkage%20maps%20of%20Eucalyptus%20grandis%20and%20Eucalyptus%20urophylla%20using%20a%20pseudo%E2%80%90testcross%20mapping%20strategy%20and%20RAPD%20markers&amp;author=D.%20Grattapaglia&amp;author=R.%20Sederoff&amp;volume=137&amp;issue=4&amp;publication_year=1994&amp;pages=1121-1137&amp;pmid=7982566&amp;doi=10.1093/genetics/137.4.1121&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0022">
<cite>

Grubbs, F. E.
 (1950). Sample criteria for testing outlying observations. The Annals of Mathematical Statistics, 21(1), 27–58. <a href="http://www.jstor.org/stable/2236553" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.jstor.org/stable/2236553</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Annals%20of%20Mathematical%20Statistics&amp;title=Sample%20criteria%20for%20testing%20outlying%20observations&amp;author=F.%20E.%20Grubbs&amp;volume=21&amp;issue=1&amp;publication_year=1950&amp;pages=27-58&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0023">
<cite>

Haggard, J. E.
, 
Johnson, E. B.
, &amp; 
St Clair, D. A.
 (2015). Multiple QTL for horticultural traits and quantitative resistance to <em>Phytophthora infestans</em> linked on <em>Solanum habrochaites</em> chromosome 11. G3 Genes|Genomes|Genetics, 5(2), 219–233. 10.1534/g3.114.014654
</cite> [<a href="https://doi.org/10.1534/g3.114.014654" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4321030/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25504736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Multiple%20QTL%20for%20horticultural%20traits%20and%20quantitative%20resistance%20to%20Phytophthora%20infestans%20linked%20on%20Solanum%20habrochaites%20chromosome%2011&amp;author=J.%20E.%20Haggard&amp;author=E.%20B.%20Johnson&amp;author=D.%20A.%20St%20Clair&amp;volume=5&amp;issue=2&amp;publication_year=2015&amp;pages=219-233&amp;pmid=25504736&amp;doi=10.1534/g3.114.014654&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0024">
<cite>

Hajnajari, H.
, 
Soroori, S.
, 
Rezaei, S.
, &amp; 
Zamani Zadeh, H.
 (2012). Correlation between anthocyanins and tolerance to crown rot disease in five different half sib populations of apple. In Donnelly D. J., &amp; Folta K. M., Eds., Acta horticulturae: XXVIII International Horticultural Congress on Science and Horticulture for People (IHC2010): International symposium on environmental, edaphic, and genetic factors affecting plants, seeds and turfgrass (Vol. 938, pp. 517–525). International Society for Horticultural Science. 10.17660/ActaHortic.2012.938.68
</cite> [<a href="https://doi.org/10.17660/ActaHortic.2012.938.68" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Acta%20horticulturae:%20XXVIII%20International%20Horticultural%20Congress%20on%20Science%20and%20Horticulture%20for%20People%20(IHC2010):%20International%20symposium%20on%20environmental,%20edaphic,%20and%20genetic%20factors%20affecting%20plants,%20seeds%20and%20turfgrass&amp;author=H.%20Hajnajari&amp;author=S.%20Soroori&amp;author=S.%20Rezaei&amp;author=H.%20Zamani%20Zadeh&amp;publication_year=2012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0025">
<cite>

Han, J.‐H.
, 
Shin, J.‐H.
, 
Fu, T.
, &amp; 
Kim, K. S.
 (2019). A new record and characterization of asparagus purple spot caused by <em>Stemphylium vesicarium</em> in Korea. Mycobiology, 47(1), 120–125. 10.1080/12298093.2018.1561238

</cite> [<a href="https://doi.org/10.1080/12298093.2018.1561238" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6450512/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30988995/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mycobiology&amp;title=A%20new%20record%20and%20characterization%20of%20asparagus%20purple%20spot%20caused%20by%20Stemphylium%20vesicarium%20in%20Korea&amp;author=J.%E2%80%90H.%20Han&amp;author=J.%E2%80%90H.%20Shin&amp;author=T.%20Fu&amp;author=K.%20S.%20Kim&amp;volume=47&amp;issue=1&amp;publication_year=2019&amp;pages=120-125&amp;pmid=30988995&amp;doi=10.1080/12298093.2018.1561238&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0026">
<cite>

Hausbeck, M.
, 
Hartwell, J.
, &amp; 
Byrne, J.
 (1999). Epidemiology of Stemphylium leaf spot and purple spot in no‐till asparagus. In Krug J. (Ed.), Acta Horticulturae: IX International Asparagus Symposium (Vol. 479, pp. 205–210). International Society for Horticultural Science. 10.17660/ActaHortic.1999.479.28
</cite> [<a href="https://doi.org/10.17660/ActaHortic.1999.479.28" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Acta%20Horticulturae:%20IX%20International%20Asparagus%20Symposium&amp;author=M.%20Hausbeck&amp;author=J.%20Hartwell&amp;author=J.%20Byrne&amp;publication_year=1999&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0027">
<cite>

He, Q.
, 
Lu, Q.
, 
He, Y.
, 
Wang, Y.
, 
Zhang, N.
, 
Zhao, W.
, &amp; 
Zhang, L.
 (2020). Dynamic changes of the anthocyanin biosynthesis mechanism during the development of heading Chinese cabbage (<em>Brassica rapa</em> L.) and Arabidopsis under the control of <em>BrMYB2</em>
. Frontiers in Plant Science, 11, 593766. 10.3389/fpls.2020.593766

</cite> [<a href="https://doi.org/10.3389/fpls.2020.593766" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7785979/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33424889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Dynamic%20changes%20of%20the%20anthocyanin%20biosynthesis%20mechanism%20during%20the%20development%20of%20heading%20Chinese%20cabbage%20(Brassica%20rapa%20L.)%20and%20Arabidopsis%20under%20the%20control%20of%20BrMYB2&amp;author=Q.%20He&amp;author=Q.%20Lu&amp;author=Y.%20He&amp;author=Y.%20Wang&amp;author=N.%20Zhang&amp;volume=11&amp;publication_year=2020&amp;pages=593766&amp;pmid=33424889&amp;doi=10.3389/fpls.2020.593766&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0028">
<cite>

Hepler, P. R.
, 
McCollum, J. P.
, &amp; 
Thompson, A. E.
 (1957). Inheritance of resistance to asparagus rust: Results of recent investigations in Illinois. University of Illinois Agricultural Experiment Station Bulletin. <a href="https://hdl.handle.net/2142/13294" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://hdl.handle.net/2142/13294</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Inheritance%20of%20resistance%20to%20asparagus%20rust:%20Results%20of%20recent%20investigations%20in%20Illinois&amp;author=P.%20R.%20Hepler&amp;author=J.%20P.%20McCollum&amp;author=A.%20E.%20Thompson&amp;publication_year=1957&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0029">
<cite>

Ibraheem, F.
, 
Gaffoor, I.
, &amp; 
Chopra, S.
 (2010). Flavonoid phytoalexin‐dependent resistance to anthracnose leaf blight requires a functional <em>yellow seed1</em> in <em>Sorghum bicolor</em>
. Genetics, 184(4), 915–926. 10.1534/genetics.109.111831

</cite> [<a href="https://doi.org/10.1534/genetics.109.111831" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2865927/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20083611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Flavonoid%20phytoalexin%E2%80%90dependent%20resistance%20to%20anthracnose%20leaf%20blight%20requires%20a%20functional%20yellow%20seed1%20in%20Sorghum%20bicolor&amp;author=F.%20Ibraheem&amp;author=I.%20Gaffoor&amp;author=S.%20Chopra&amp;volume=184&amp;issue=4&amp;publication_year=2010&amp;pages=915-926&amp;pmid=20083611&amp;doi=10.1534/genetics.109.111831&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0030">
<cite>

Janick, J.
, &amp; 
Topoleski, L. D.
 (1963). Inheritance of fruit color in eggplant (<em>Solanum melongena</em> L.). Proceedings of the American Society for Horticultural Science, 83, 547–558. <a href="https://www.cabidigitallibrary.org/doi/full/10.5555/19640304974" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cabidigitallibrary.org/doi/full/10.5555/19640304974</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20American%20Society%20for%20Horticultural%20Science&amp;title=Inheritance%20of%20fruit%20color%20in%20eggplant%20(Solanum%20melongena%20L.)&amp;author=J.%20Janick&amp;author=L.%20D.%20Topoleski&amp;volume=83&amp;publication_year=1963&amp;pages=547-558&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0031">
<cite>

Ji, Z.
, 
Wang, M.
, 
Zhang, S.
, 
Du, Y.
, 
Cong, J.
, 
Yan, H.
, 
Guo, H.
, 
Xu, B.
, &amp; 
Zhou, Z.
 (2023). GDSL esterase/lipase <em>GELP1</em> involved in the defense of apple leaves against <em>Colletotrichum gloeosporioides</em> infection. International Journal of Molecular Sciences, 24(12), 10343. 10.3390/ijms241210343

</cite> [<a href="https://doi.org/10.3390/ijms241210343" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10299637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37373491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=GDSL%20esterase/lipase%20GELP1%20involved%20in%20the%20defense%20of%20apple%20leaves%20against%20Colletotrichum%20gloeosporioides%20infection&amp;author=Z.%20Ji&amp;author=M.%20Wang&amp;author=S.%20Zhang&amp;author=Y.%20Du&amp;author=J.%20Cong&amp;volume=24&amp;issue=12&amp;publication_year=2023&amp;pages=10343&amp;pmid=37373491&amp;doi=10.3390/ijms241210343&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0032">
<cite>

Kapoor, L.
, 
Simkin, A. J.
, 
George Priya Doss, C.
, &amp; 
Siva, R.
 (2022). Fruit ripening: Dynamics and integrated analysis of carotenoids and anthocyanins. BMC Plant Biology, 22, Article 27. 10.1186/s12870-021-03411-w

</cite> [<a href="https://doi.org/10.1186/s12870-021-03411-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8750800/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35016620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Fruit%20ripening:%20Dynamics%20and%20integrated%20analysis%20of%20carotenoids%20and%20anthocyanins&amp;author=L.%20Kapoor&amp;author=A.%20J.%20Simkin&amp;author=C.%20George%20Priya%20Doss&amp;author=R.%20Siva&amp;volume=22&amp;publication_year=2022&amp;pages=27&amp;pmid=35016620&amp;doi=10.1186/s12870-021-03411-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0033">
<cite>

Kawuki, R. S.
, 
Esuma, W.
, 
Ozimati, A.
, 
Kayondo, I. S.
, 
Nandudu, L.
, &amp; 
Wolfe, M.
 (2019). Alternative approaches for assessing cassava brown streak root necrosis to guide resistance breeding and selection. Frontiers in Plant Science, 10, 1461. 10.3389/fpls.2019.01461

</cite> [<a href="https://doi.org/10.3389/fpls.2019.01461" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6872967/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31803205/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Alternative%20approaches%20for%20assessing%20cassava%20brown%20streak%20root%20necrosis%20to%20guide%20resistance%20breeding%20and%20selection&amp;author=R.%20S.%20Kawuki&amp;author=W.%20Esuma&amp;author=A.%20Ozimati&amp;author=I.%20S.%20Kayondo&amp;author=L.%20Nandudu&amp;volume=10&amp;publication_year=2019&amp;pages=1461&amp;pmid=31803205&amp;doi=10.3389/fpls.2019.01461&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0034">
<cite>

Khoo, H. E.
, 
Azlan, A.
, 
Tang, S. T.
, &amp; 
Lim, S. M.
 (2017). Anthocyanidins and anthocyanins: Colored pigments as food, pharmaceutical ingredients, and the potential health benefits. Food &amp; Nutrition Research, 61, 1361779. 10.1080/16546628.2017.1361779

</cite> [<a href="https://doi.org/10.1080/16546628.2017.1361779" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5613902/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28970777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20&amp;%20Nutrition%20Research&amp;title=Anthocyanidins%20and%20anthocyanins:%20Colored%20pigments%20as%20food,%20pharmaceutical%20ingredients,%20and%20the%20potential%20health%20benefits&amp;author=H.%20E.%20Khoo&amp;author=A.%20Azlan&amp;author=S.%20T.%20Tang&amp;author=S.%20M.%20Lim&amp;volume=61&amp;publication_year=2017&amp;pages=1361779&amp;pmid=28970777&amp;doi=10.1080/16546628.2017.1361779&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0035">
<cite>

Khusnutdinov, E.
, 
Sukhareva, A.
, 
Panfilova, M.
, &amp; 
Mikhaylova, E.
 (2021). Anthocyanin biosynthesis genes as model genes for genome editing in plants. International Journal of Molecular Sciences, 22(16), 8752. 10.3390/ijms22168752

</cite> [<a href="https://doi.org/10.3390/ijms22168752" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8395717/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34445458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Anthocyanin%20biosynthesis%20genes%20as%20model%20genes%20for%20genome%20editing%20in%20plants&amp;author=E.%20Khusnutdinov&amp;author=A.%20Sukhareva&amp;author=M.%20Panfilova&amp;author=E.%20Mikhaylova&amp;volume=22&amp;issue=16&amp;publication_year=2021&amp;pages=8752&amp;pmid=34445458&amp;doi=10.3390/ijms22168752&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0036">
<cite>

Kosambi, D. D.
 (2016). The estimation of map distances from recombination values. In Ramaswamy R. (Ed.), D.D. Kosambi: Selected works in mathematics and statistics (pp. 125–130). Springer. <a href="https://link.springer.com/chapter/10.1007/978-81-322-3676-4_16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1007/978‐81‐322‐3676‐4_16</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=D.D.%20Kosambi:%20Selected%20works%20in%20mathematics%20and%20statistics&amp;author=D.%20D.%20Kosambi&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0037">
<cite>

Lee, D. W.
, &amp; 
Collins, T. M.
 (2001). Phylogenetic and ontogenetic influences on the distribution of anthocyanins and betacyanins in leaves of tropical plants. International Journal of Plant Sciences, 162(5), 1141–1153. 10.1086/321926
</cite> [<a href="https://doi.org/10.1086/321926" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Plant%20Sciences&amp;title=Phylogenetic%20and%20ontogenetic%20influences%20on%20the%20distribution%20of%20anthocyanins%20and%20betacyanins%20in%20leaves%20of%20tropical%20plants&amp;author=D.%20W.%20Lee&amp;author=T.%20M.%20Collins&amp;volume=162&amp;issue=5&amp;publication_year=2001&amp;pages=1141-1153&amp;doi=10.1086/321926&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0038">
<cite>

Li, H.
, 
Ruan, J.
, &amp; 
Durbin, R.
 (2008). Mapping short DNA sequencing reads and calling variants using mapping quality scores. Genome Research, 18(11), 1851–1858. 10.1101/gr.078212.108

</cite> [<a href="https://doi.org/10.1101/gr.078212.108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2577856/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18714091/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=Mapping%20short%20DNA%20sequencing%20reads%20and%20calling%20variants%20using%20mapping%20quality%20scores&amp;author=H.%20Li&amp;author=J.%20Ruan&amp;author=R.%20Durbin&amp;volume=18&amp;issue=11&amp;publication_year=2008&amp;pages=1851-1858&amp;pmid=18714091&amp;doi=10.1101/gr.078212.108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0039">
<cite>

Liang, Y.
, 
Han, C.
, 
Yun, L.
, 
Yang, Y.
, &amp; 
Cao, Y.
 (2022). Transcriptomic and metabolomic analysis of the mechanism of temperature‐regulated anthocyanin biosynthesis in purple asparagus spears. Scientia Horticulturae, 295, 110858. 10.1016/j.scienta.2021.110858
</cite> [<a href="https://doi.org/10.1016/j.scienta.2021.110858" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientia%20Horticulturae&amp;title=Transcriptomic%20and%20metabolomic%20analysis%20of%20the%20mechanism%20of%20temperature%E2%80%90regulated%20anthocyanin%20biosynthesis%20in%20purple%20asparagus%20spears&amp;author=Y.%20Liang&amp;author=C.%20Han&amp;author=L.%20Yun&amp;author=Y.%20Yang&amp;author=Y.%20Cao&amp;volume=295&amp;publication_year=2022&amp;pages=110858&amp;doi=10.1016/j.scienta.2021.110858&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0040">
<cite>

Mascher, M.
, 
Wu, S.
, 
Amand, P. S.
, 
Stein, N.
, &amp; 
Poland, J.
 (2013). Application of genotyping‐by‐sequencing on semiconductor sequencing platforms: A comparison of genetic and reference‐based marker ordering in barley. PLoS One, 8(10), e76925. 10.1371/journal.pone.0076925

</cite> [<a href="https://doi.org/10.1371/journal.pone.0076925" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3789676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24098570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Application%20of%20genotyping%E2%80%90by%E2%80%90sequencing%20on%20semiconductor%20sequencing%20platforms:%20A%20comparison%20of%20genetic%20and%20reference%E2%80%90based%20marker%20ordering%20in%20barley&amp;author=M.%20Mascher&amp;author=S.%20Wu&amp;author=P.%20S.%20Amand&amp;author=N.%20Stein&amp;author=J.%20Poland&amp;volume=8&amp;issue=10&amp;publication_year=2013&amp;pages=e76925&amp;pmid=24098570&amp;doi=10.1371/journal.pone.0076925&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0041">
<cite>

McCouch, S. R.
, 
Chen, X.
, 
Panaud, O.
, 
Temnykh, S.
, 
Xu, Y.
, 
Cho, Y. G.
, 
Huang, N.
, 
Ishii, T.
, &amp; 
Blair, M.
 (1997). Microsatellite marker development, mapping and applications in rice genetics and breeding. Oryza: From Molecule to Plant, 35(1–2), 89–99. 10.1007/978-94-011-5794-0_9
</cite> [<a href="https://doi.org/10.1007/978-94-011-5794-0_9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9291963/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oryza:%20From%20Molecule%20to%20Plant&amp;title=Microsatellite%20marker%20development,%20mapping%20and%20applications%20in%20rice%20genetics%20and%20breeding&amp;author=S.%20R.%20McCouch&amp;author=X.%20Chen&amp;author=O.%20Panaud&amp;author=S.%20Temnykh&amp;author=Y.%20Xu&amp;volume=35&amp;issue=1%E2%80%932&amp;publication_year=1997&amp;pages=89-99&amp;pmid=9291963&amp;doi=10.1007/978-94-011-5794-0_9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0042">
<cite>

Menzies, S.
, 
Broadhurst, P.
, &amp; 
Triggs, C.
 (1992). Stemphylium disease of asparagus (<em>Asparagus officinalis</em> L.) in New Zealand. New Zealand Journal of Crop and Horticultural Science, 20(4), 427–433. 10.1080/01140671.1992.10418060
</cite> [<a href="https://doi.org/10.1080/01140671.1992.10418060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Zealand%20Journal%20of%20Crop%20and%20Horticultural%20Science&amp;title=Stemphylium%20disease%20of%20asparagus%20(Asparagus%20officinalis%20L.)%20in%20New%20Zealand&amp;author=S.%20Menzies&amp;author=P.%20Broadhurst&amp;author=C.%20Triggs&amp;volume=20&amp;issue=4&amp;publication_year=1992&amp;pages=427-433&amp;doi=10.1080/01140671.1992.10418060&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0043">
<cite>

Moreno‐Pinel, R.
, 
Castro‐López, P.
, 
Die‐Ramón, J. V.
, &amp; 
Gil‐Ligero, J.
 (2021). Asparagus (<em>Asparagus officinalis</em> L.): Breeding and Genomics. In Al‐Khayri J. M., Jain S. M., &amp; Johnson D. V. (Eds.), Advances in plant breeding strategies: Vegetable crops (Vol. 9, pp. 425–469). Springer. 10.1007/978-3-030-66961-4_12
</cite> [<a href="https://doi.org/10.1007/978-3-030-66961-4_12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20plant%20breeding%20strategies:%20Vegetable%20crops&amp;author=R.%20Moreno%E2%80%90Pinel&amp;author=P.%20Castro%E2%80%90L%C3%B3pez&amp;author=J.%20V.%20Die%E2%80%90Ram%C3%B3n&amp;author=J.%20Gil%E2%80%90Ligero&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0044">
<cite>

Peirce, L.
 (1982). Inheritance of a green stem mutant in asparagus. HortScience, 17(3), 389–391. 10.21273/hortsci.17.3.389
</cite> [<a href="https://doi.org/10.21273/hortsci.17.3.389" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=HortScience&amp;title=Inheritance%20of%20a%20green%20stem%20mutant%20in%20asparagus&amp;author=L.%20Peirce&amp;volume=17&amp;issue=3&amp;publication_year=1982&amp;pages=389-391&amp;doi=10.21273/hortsci.17.3.389&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0045">
<cite>
R Core Team
. (2022). R: A language and environment for statistical computing. R Foundation for Statistical Computing. <a href="https://www.R-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.R‐project.org/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=R:%20A%20language%20and%20environment%20for%20statistical%20computing&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0046">
<cite>

Revelle, W.
 (2023). 
<em>psych: Procedures for psychological, psychometric, and personality research</em> (Version 2.3.9) [R package]. Comprehensive R Archive Network (CRAN). 10.32614/CRAN.package.psych
</cite> [<a href="https://doi.org/10.32614/CRAN.package.psych" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Revelle,%20W.%20(2023).%20psych:%20Procedures%20for%20psychological,%20psychometric,%20and%20personality%20research%20(Version%202.3.9)%20%5BR%20package%5D.%20Comprehensive%20R%20Archive%20Network%20(CRAN).%2010.32614/CRAN.package.psych" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0047">
<cite>

Rochette, N. C.
, &amp; 
Catchen, J. M.
 (2017). Deriving genotypes from RAD‐seq short‐read data using Stacks. Nature Protocols, 12(12), 2640–2659. 10.1038/nprot.2017.123

</cite> [<a href="https://doi.org/10.1038/nprot.2017.123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29189774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Protocols&amp;title=Deriving%20genotypes%20from%20RAD%E2%80%90seq%20short%E2%80%90read%20data%20using%20Stacks&amp;author=N.%20C.%20Rochette&amp;author=J.%20M.%20Catchen&amp;volume=12&amp;issue=12&amp;publication_year=2017&amp;pages=2640-2659&amp;pmid=29189774&amp;doi=10.1038/nprot.2017.123&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0048">
<cite>

Sakaguchi, Y.
, 
Ozaki, Y.
, 
Miyajima, I.
, 
Yamaguchi, M.
, 
Fukui, Y.
, 
Iwasa, K.
, 
Motoki, S.
, 
Suzuki, T.
, &amp; 
Okubo, H.
 (2008). Major anthocyanins from purple asparagus (<em>Asparagus officinalis</em>). Phytochemistry, 69(8), 1763–1766. 10.1016/j.phytochem.2008.02.021

</cite> [<a href="https://doi.org/10.1016/j.phytochem.2008.02.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18406435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phytochemistry&amp;title=Major%20anthocyanins%20from%20purple%20asparagus%20(Asparagus%20officinalis)&amp;author=Y.%20Sakaguchi&amp;author=Y.%20Ozaki&amp;author=I.%20Miyajima&amp;author=M.%20Yamaguchi&amp;author=Y.%20Fukui&amp;volume=69&amp;issue=8&amp;publication_year=2008&amp;pages=1763-1766&amp;pmid=18406435&amp;doi=10.1016/j.phytochem.2008.02.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0049">
<cite>

Salvador‐Guirao, R.
, 
Hsing, Y.‐I.
, &amp; 
San Segundo, B.
 (2018). The polycistronic miR166k‐166 h positively regulates rice immunity via post‐transcriptional control of <em>EIN2</em>
. Frontiers in Plant Science, 9, 337. 10.3389/fpls.2018.00337

</cite> [<a href="https://doi.org/10.3389/fpls.2018.00337" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5869255/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29616057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=The%20polycistronic%20miR166k%E2%80%90166%C2%A0h%20positively%20regulates%20rice%20immunity%20via%20post%E2%80%90transcriptional%20control%20of%20EIN2&amp;author=R.%20Salvador%E2%80%90Guirao&amp;author=Y.%E2%80%90I.%20Hsing&amp;author=B.%20San%20Segundo&amp;volume=9&amp;publication_year=2018&amp;pages=337&amp;pmid=29616057&amp;doi=10.3389/fpls.2018.00337&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0050">
<cite>

Seidel, T.
 (2022). The plant V‐ATPase. Frontiers in Plant Science, 13, 931777. 10.3389/fpls.2022.931777

</cite> [<a href="https://doi.org/10.3389/fpls.2022.931777" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9280200/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35845650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=The%20plant%20V%E2%80%90ATPase&amp;author=T.%20Seidel&amp;volume=13&amp;publication_year=2022&amp;pages=931777&amp;pmid=35845650&amp;doi=10.3389/fpls.2022.931777&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0051">
<cite>

Sicilia, A.
, 
Catara, V.
, 
Scialò, E.
, &amp; 
Lo Piero, A. R.
 (2021). Fungal infection induces anthocyanin biosynthesis and changes in DNA methylation configuration of blood orange [<em>Citrus sinensis</em> L. (Osbeck)]. Plants, 10(2), 244. 10.3390/plants10020244

</cite> [<a href="https://doi.org/10.3390/plants10020244" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7910907/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33513740/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Fungal%20infection%20induces%20anthocyanin%20biosynthesis%20and%20changes%20in%20DNA%20methylation%20configuration%20of%20blood%20orange%20%5BCitrus%20sinensis%20L.%20(Osbeck)%5D&amp;author=A.%20Sicilia&amp;author=V.%20Catara&amp;author=E.%20Scial%C3%B2&amp;author=A.%20R.%20Lo%20Piero&amp;volume=10&amp;issue=2&amp;publication_year=2021&amp;pages=244&amp;pmid=33513740&amp;doi=10.3390/plants10020244&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0052">
<cite>

Sims, D. A.
, &amp; 
Gamon, J. A.
 (2002). Relationships between leaf pigment content and spectral reflectance across a wide range of species, leaf structures and developmental stages. Remote Sensing of Environment, 81(2–3), 337–354. 10.1016/S0034-4257(02)00010-X
</cite> [<a href="https://doi.org/10.1016/S0034-4257(02)00010-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Remote%20Sensing%20of%20Environment&amp;title=Relationships%20between%20leaf%20pigment%20content%20and%20spectral%20reflectance%20across%20a%20wide%20range%20of%20species,%20leaf%20structures%20and%20developmental%20stages&amp;author=D.%20A.%20Sims&amp;author=J.%20A.%20Gamon&amp;volume=81&amp;issue=2%E2%80%933&amp;publication_year=2002&amp;pages=337-354&amp;doi=10.1016/S0034-4257(02)00010-X&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0053">
<cite>

Tanaka, S.
, 
Brefort, T.
, 
Neidig, N.
, 
Djamei, A.
, 
Kahnt, J.
, 
Vermerris, W.
, 
Koenig, S.
, 
Feussner, K.
, 
Feussner, I.
, &amp; 
Kahmann, R.
 (2014). A secreted <em>Ustilago maydis</em> effector promotes virulence by targeting anthocyanin biosynthesis in maize. Elife, 3, e01355. 10.7554/eLife.01355

</cite> [<a href="https://doi.org/10.7554/eLife.01355" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3904489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24473076/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=A%20secreted%20Ustilago%20maydis%20effector%20promotes%20virulence%20by%20targeting%20anthocyanin%20biosynthesis%20in%20maize&amp;author=S.%20Tanaka&amp;author=T.%20Brefort&amp;author=N.%20Neidig&amp;author=A.%20Djamei&amp;author=J.%20Kahnt&amp;volume=3&amp;publication_year=2014&amp;pages=e01355&amp;pmid=24473076&amp;doi=10.7554/eLife.01355&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0054">
<cite>

Toppino, L.
, 
Barchi, L.
, 
Mercati, F.
, 
Acciarri, N.
, 
Perrone, D.
, 
Martina, M.
, 
Gattolin, S.
, 
Sala, T.
, 
Fadda, S.
, 
Mauceri, A.
, 
Ciriaci, T.
, 
Carimi, F.
, 
Portis, E.
, 
Sunseri, F.
, 
Lanteri, S.
, &amp; 
Rotino, G. L.
 (2020). A new intra‐specific and high‐resolution genetic map of eggplant based on a RIL population, and location of QTLs related to plant anthocyanin pigmentation and seed vigour. Genes, 11(7), 745. 10.3390/genes11070745

</cite> [<a href="https://doi.org/10.3390/genes11070745" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7397344/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32635424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes&amp;title=A%20new%20intra%E2%80%90specific%20and%20high%E2%80%90resolution%20genetic%20map%20of%20eggplant%20based%20on%20a%20RIL%20population,%20and%20location%20of%20QTLs%20related%20to%20plant%20anthocyanin%20pigmentation%20and%20seed%20vigour&amp;author=L.%20Toppino&amp;author=L.%20Barchi&amp;author=F.%20Mercati&amp;author=N.%20Acciarri&amp;author=D.%20Perrone&amp;volume=11&amp;issue=7&amp;publication_year=2020&amp;pages=745&amp;pmid=32635424&amp;doi=10.3390/genes11070745&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0055">
<cite>

Vale, F. X. R.
, 
Parlevliet, J.
, &amp; 
Zambolim, L.
 (2001). Concepts in plant disease resistance. Fitopatologia Brasileira, 26(3), 577–589. 10.1590/S0100-41582001000300001
</cite> [<a href="https://doi.org/10.1590/S0100-41582001000300001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Fitopatologia%20Brasileira&amp;title=Concepts%20in%20plant%20disease%20resistance&amp;author=F.%20X.%20R.%20Vale&amp;author=J.%20Parlevliet&amp;author=L.%20Zambolim&amp;volume=26&amp;issue=3&amp;publication_year=2001&amp;pages=577-589&amp;doi=10.1590/S0100-41582001000300001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0056">
<cite>

Van Ooijen, J. W.
 (1992). Accuracy of mapping quantitative trait loci in autogamous species. Theoretical and Applied Ggenetics, 84, 803–811. 10.1007/BF00227388
</cite> [<a href="https://doi.org/10.1007/BF00227388" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24201478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Ggenetics&amp;title=Accuracy%20of%20mapping%20quantitative%20trait%20loci%20in%20autogamous%20species&amp;author=J.%20W.%20Van%20Ooijen&amp;volume=84&amp;publication_year=1992&amp;pages=803-811&amp;pmid=24201478&amp;doi=10.1007/BF00227388&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0057">
<cite>

Van Ooijen, J. W.
 (2018). JoinMap® 5, Software for the calculation of genetic linkage maps in experimental populations of diploid species. Kyazma BV. <a href="https://www.kyazma.nl/index.php/JoinMap/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.kyazma.nl/index.php/JoinMap/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=JoinMap%C2%AE%205,%20Software%20for%20the%20calculation%20of%20genetic%20linkage%20maps%20in%20experimental%20populations%20of%20diploid%20species&amp;author=J.%20W.%20Van%20Ooijen&amp;publication_year=2018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0058">
<cite>

Voorrips, R. E.
 (2002). MapChart: Software for the graphical presentation of linkage maps and QTLs. Journal of Heredity, 93(1), 77–78. 10.1093/jhered/93.1.77

</cite> [<a href="https://doi.org/10.1093/jhered/93.1.77" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12011185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Heredity&amp;title=MapChart:%20Software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&amp;author=R.%20E.%20Voorrips&amp;volume=93&amp;issue=1&amp;publication_year=2002&amp;pages=77-78&amp;pmid=12011185&amp;doi=10.1093/jhered/93.1.77&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0059">
<cite>

Wang, S.
, 
Basten, C. J.
, &amp; 
Zeng, Z.‐B.
 (2012). Windows QTL cartographer 2.5. Department of statistics. North Carolina State University. <a href="http://statgen.ncsu.edu/qtlcart/WQTLCart.htm" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://statgen.ncsu.edu/qtlcart/WQTLCart.htm</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Windows%20QTL%20cartographer%202.5.%20Department%20of%20statistics&amp;author=S.%20Wang&amp;author=C.%20J.%20Basten&amp;author=Z.%E2%80%90B.%20Zeng&amp;publication_year=2012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0060">
<cite>

Wang, Y.
, &amp; 
Wang, F.
 (2021). Post‐translational modifications of deubiquitinating enzymes: Expanding the ubiquitin code. Frontiers in Pharmacology, 12, 685011. 10.3389/fphar.2021.685011

</cite> [<a href="https://doi.org/10.3389/fphar.2021.685011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8224227/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34177595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Pharmacology&amp;title=Post%E2%80%90translational%20modifications%20of%20deubiquitinating%20enzymes:%20Expanding%20the%20ubiquitin%20code&amp;author=Y.%20Wang&amp;author=F.%20Wang&amp;volume=12&amp;publication_year=2021&amp;pages=685011&amp;pmid=34177595&amp;doi=10.3389/fphar.2021.685011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0061">
<cite>

Xie, S.
, 
Song, C.
, 
Wang, X.
, 
Liu, M.
, 
Zhang, Z.
, &amp; 
Xi, Z.
 (2015). Tissue‐specific expression analysis of anthocyanin biosynthetic genes in white‐and red‐fleshed grape cultivars. Molecules, 20(12), 22767–22780. 10.3390/molecules201219883

</cite> [<a href="https://doi.org/10.3390/molecules201219883" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6331842/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26703539/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Tissue%E2%80%90specific%20expression%20analysis%20of%20anthocyanin%20biosynthetic%20genes%20in%20white%E2%80%90and%20red%E2%80%90fleshed%20grape%20cultivars&amp;author=S.%20Xie&amp;author=C.%20Song&amp;author=X.%20Wang&amp;author=M.%20Liu&amp;author=Z.%20Zhang&amp;volume=20&amp;issue=12&amp;publication_year=2015&amp;pages=22767-22780&amp;pmid=26703539&amp;doi=10.3390/molecules201219883&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0062">
<cite>

Zhang, S.
, 
Li, C.
, 
Si, J.
, 
Han, Z.
, &amp; 
Chen, D.
 (2022). Action mechanisms of effectors in plant‐pathogen interaction. International Journal of Molecular Sciences, 23(12), 6758. 10.3390/ijms23126758

</cite> [<a href="https://doi.org/10.3390/ijms23126758" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9224169/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35743201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Action%20mechanisms%20of%20effectors%20in%20plant%E2%80%90pathogen%20interaction&amp;author=S.%20Zhang&amp;author=C.%20Li&amp;author=J.%20Si&amp;author=Z.%20Han&amp;author=D.%20Chen&amp;volume=23&amp;issue=12&amp;publication_year=2022&amp;pages=6758&amp;pmid=35743201&amp;doi=10.3390/ijms23126758&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0063">
<cite>

Zhang, Z.
, 
Zheng, Y.
, 
Mazon, H.
, 
Milgrom, E.
, 
Kitagawa, N.
, 
Kish‐Trier, E.
, 
Heck, A. J.
, 
Kane, P. M.
, &amp; 
Wilkens, S.
 (2008). Structure of the yeast vacuolar ATPase. Journal of Biological Chemistry, 283(51), 35983–35995. 10.1074/jbc.M805345200

</cite> [<a href="https://doi.org/10.1074/jbc.M805345200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2602884/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18955482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=Structure%20of%20the%20yeast%20vacuolar%20ATPase&amp;author=Z.%20Zhang&amp;author=Y.%20Zheng&amp;author=H.%20Mazon&amp;author=E.%20Milgrom&amp;author=N.%20Kitagawa&amp;volume=283&amp;issue=51&amp;publication_year=2008&amp;pages=35983-35995&amp;pmid=18955482&amp;doi=10.1074/jbc.M805345200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270028-bib-0064">
<cite>

Zhou, B.
, &amp; 
Zeng, L.
 (2017). Conventional and unconventional ubiquitination in plant immunity. Molecular Plant Pathology, 18(9), 1313–1330. 10.1111/mpp.12521

</cite> [<a href="https://doi.org/10.1111/mpp.12521" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6638253/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27925369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant%20Pathology&amp;title=Conventional%20and%20unconventional%20ubiquitination%20in%20plant%20immunity&amp;author=B.%20Zhou&amp;author=L.%20Zeng&amp;volume=18&amp;issue=9&amp;publication_year=2017&amp;pages=1313-1330&amp;pmid=27925369&amp;doi=10.1111/mpp.12521&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<p>
<strong>Supplementary Table S1</strong>: Anthocyanin pigments detected by LC/MS in asparagus scale leaves and purple spot lesions in an F<sub>1</sub> pseudo‐test cross population.</p>
<p>
<strong>Supplementary Table S2</strong>: Quantification of anthocyanin pigment types in asparagus scale leaves and purple spot lesions using LC/MS, visually rated as purple or red in an F<sub>1</sub> pseudo‐testcross population.</p>
<p>
<strong>Supplementary Table S3</strong>: Total anthocyanin pigment concentrations, as determined by spectrophotometry, for nine asparagus scale leaf pigment phenotypes based on visual ratings of an F<sub>1</sub> pseudo‐testcross population.</p>
<p>
<strong>Supplementary Table S4</strong>: Analysis of variance of the natural levels of anthocyanin pigment at scale leaves (ASL), anthocyanin pigment levels at spear purple spot lesions (APS), number of purple spot lesions in spears (NPS), and purple spot severity in fern (PSF) for an asparagus F<sub>1</sub> pseudo‐testcross population across two years (2021 and 2022) and average over years.</p>
<p>
<strong>Supplementary Table S5</strong>
<span class="text-underline">:</span> Summary of genetic maps constructed for an asparagus F<sub>1</sub> pseudo‐testcross population (n = 416).</p>
<p>
<strong>Supplementary Table S6</strong>
<span class="text-underline">:</span> Quantitative trait loci (QTL) characterization for anthocyanin pigment levels at spear purple spot lesions (APS), number of purple spot lesions in spears (NPS) and purple spot severity in fern (PSF) for asparagus.</p>
<p>
<strong>Supplementary Figure S1</strong>: Different levels of purple spot infection in an asparagus F<sub>1</sub> pseudo‐testcross population. 0: Highly resistant‐ fern displayed no signs of disease; 1: Resistant‐fern remained predominantly green with minimal cladophyll drop and no noticeable symptoms; 3: Moderately resistant‐fern retained green coloration, but with some cladophyll drop and few instances of purple leaf spot; 5: Moderately susceptible‐ fern showed a noticeable number of cladophylls dropped and purple leaf spot, yet without any stem desiccation; 7: Susceptible fern with significant cladophyll dropping and purple spot presence, accompanied by partial stem dryness and yellowing; 9: Highly susceptible fern with complete yellowing and extensive stem desiccation.</p>
<p>
<strong>Supplementary Figure S2</strong>: Distribution of anthocyanin phenotypic classes (1 = low pigment level, 9 = high pigment level) in scale leaf (A, B, and C) and purple spot lesions (D, E, and F); numbers of purple spot lesions in spears (G, H, and I) and purple spot severity in fern J, K, and L) in an asparagus F<sub>1</sub> pseudo‐testcross population in 2021 (A, D, G, J), 2022 (B, E, H, K), and average of 2021–2022 data (C, F, I, L). The green and red vertical lines represent two control cultivars, Guelph Millenium, and Guelph Eclipse, respectively.</p>
<p>
<strong>Supplementary Figure S3</strong>: Genetic linkage map of the female parent for an asparagus F<sub>1</sub> pseudo‐testcross population. The chromosome number is represented as the first number (e.g., 1.1, 2.1), following underscore are the position of markers in the reference genome. The scale bar indicates the distance in centimorgans (cM).</p>
<p>
<strong>Supplementary Figure S4</strong>: Genetic linkage map of asparagus male parent. The chromosome number is represented as the first number (e.g., 1.1, 2.1), following underscore are the position of markers in the reference genome. The scale bar indicates the distance in centimorgans (cM).</p>
<p>
<strong>Supplementary Figure S5</strong>: Integrated genetic linkage map of asparagus. The chromosome number is represented as the first number (e.g., 1.1, 2.1), following underscore are the position of markers in the reference genome. The scale bar indicates the distance in centimorgans (cM).</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11981981/bin/TPG2-18-e70028-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70028-s001.docx</a><sup> (3.7MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Phenotypic and genotypic data used in this study are available on Dryad at <a href="https://doi.org/10.5061/dryad.nzs7h452b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5061/dryad.nzs7h452b</a>. The raw sequencing data supporting this study have been deposited in the NCBI (SRA) under the BioProject accession number PRJNA1169105, available at <a href="https://www.ncbi.nlm.nih.gov/sra/PRJNA1169105" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/sra/PRJNA1169105</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70028"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70028.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11981981/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11981981/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11981981%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11981981/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11981981/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11981981/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40204667/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11981981/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40204667/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11981981/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11981981/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="y5aeqG2utmejXF9dg2dR72SPMFcKVnE1zzDkJ9ssdgQOAurYVHw0JzQuQtINrRVt">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
