
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            The full-length BEND2 protein is dispensable for spermatogenesis but required for setting the ovarian reserve in mice - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4AB8C8AF2C62305AB8C0055D3ED2E.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="elife">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367297/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="eLife">
<meta name="citation_title" content="The full-length BEND2 protein is dispensable for spermatogenesis but required for setting the ovarian reserve in mice">
<meta name="citation_author" content="Yan Huang">
<meta name="citation_author_institution" content="Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_author_institution" content="Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_author" content="Nina Bucevic">
<meta name="citation_author_institution" content="Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_author_institution" content="Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_author" content="Carmen Coves">
<meta name="citation_author_institution" content="Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_author_institution" content="Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_author" content="Natalia Felipe-Medina">
<meta name="citation_author_institution" content="Molecular Mechanisms Program, Centro de Investigación del Cáncer and Instituto de Biología Molecular y Celular del Cáncer (CSIC-Universidad de Salamanca), Salamanca, Spain">
<meta name="citation_author" content="Marina Marcet-Ortega">
<meta name="citation_author_institution" content="Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_author_institution" content="Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_author" content="Nikoleta Nikou">
<meta name="citation_author_institution" content="Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_author_institution" content="Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_author" content="Cristina Madrid-Sandín">
<meta name="citation_author_institution" content="Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_author_institution" content="Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_author" content="Maria López-Panadés">
<meta name="citation_author_institution" content="Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_author_institution" content="Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_author" content="Carolina Buza">
<meta name="citation_author_institution" content="Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_author_institution" content="Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_author" content="Neus Ferrer Miralles">
<meta name="citation_author_institution" content="Institut de Biotecnologia i de Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallés, Spain">
<meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Bioingeniería, Biomateriales y Nanomedicina, Instituto de Salud Carlos III, Cerdanyola del Vallès, Spain">
<meta name="citation_author_institution" content="Departament de Genètica i de Microbiologia, Universitat Autònoma de Barcelona, Cerdanyola del Vallés, Spain">
<meta name="citation_author" content="Antoni Iborra">
<meta name="citation_author_institution" content="Servei de Cultius Cel·lulars, Producció d&#x27;Anticossos i Citometria, Universitat Autònoma de Barcelona, Cerdanyola del Vallés, Spain">
<meta name="citation_author" content="Anna Pujol">
<meta name="citation_author_institution" content="Transgenic Animal Unit, Center of Animal Biotechnology and Gene Therapy, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_author" content="Alberto M Pendás">
<meta name="citation_author_institution" content="Molecular Mechanisms Program, Centro de Investigación del Cáncer and Instituto de Biología Molecular y Celular del Cáncer (CSIC-Universidad de Salamanca), Salamanca, Spain">
<meta name="citation_author" content="Ignasi Roig">
<meta name="citation_author_institution" content="Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_author_institution" content="Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="13">
<meta name="citation_firstpage" content="RP96052">
<meta name="citation_doi" content="10.7554/eLife.96052">
<meta name="citation_pmid" content="40833257">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367297/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367297/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367297/pdf/elife-96052.pdf">
<meta name="description" content="Infertility affects up to 12% of couples globally, with genetic factors contributing to nearly half of the cases. Advances in genomic technologies have led to the discovery of genes like Bend2, which play a crucial role in gametogenesis. In the ...">
<meta name="og:title" content="The full-length BEND2 protein is dispensable for spermatogenesis but required for setting the ovarian reserve in mice">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Infertility affects up to 12% of couples globally, with genetic factors contributing to nearly half of the cases. Advances in genomic technologies have led to the discovery of genes like Bend2, which play a crucial role in gametogenesis. In the ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367297/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12367297">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.7554/eLife.96052"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/elife-96052.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367297%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12367297/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12367297/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367297/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-elife.png" alt="eLife logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to eLife" title="Link to eLife" shape="default" href="https://doi.org/10.7554/eLife.96052" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">eLife</button></div>. 2025 Aug 20;13:RP96052. doi: <a href="https://doi.org/10.7554/eLife.96052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.96052</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22eLife%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22eLife%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22eLife%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22eLife%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>The full-length BEND2 protein is dispensable for spermatogenesis but required for setting the ovarian reserve in mice</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Yan Huang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yan Huang</span></h3>
<div class="p">
<sup>1</sup>Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">
<sup>2</sup>Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yan Huang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bucevic%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Nina Bucevic</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Nina Bucevic</span></h3>
<div class="p">
<sup>1</sup>Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">
<sup>2</sup>Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bucevic%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nina Bucevic</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Coves%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Carmen Coves</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Carmen Coves</span></h3>
<div class="p">
<sup>1</sup>Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">
<sup>2</sup>Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Coves%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Carmen Coves</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Felipe-Medina%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Natalia Felipe-Medina</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Natalia Felipe-Medina</span></h3>
<div class="p">
<sup>3</sup>Molecular Mechanisms Program, Centro de Investigación del Cáncer and Instituto de Biología Molecular y Celular del Cáncer (CSIC-Universidad de Salamanca), Salamanca, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Felipe-Medina%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Natalia Felipe-Medina</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marcet-Ortega%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Marina Marcet-Ortega</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Marina Marcet-Ortega</span></h3>
<div class="p">
<sup>1</sup>Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">
<sup>2</sup>Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marcet-Ortega%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marina Marcet-Ortega</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nikou%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Nikoleta Nikou</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Nikoleta Nikou</span></h3>
<div class="p">
<sup>1</sup>Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">
<sup>2</sup>Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nikou%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nikoleta Nikou</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Madrid-Sand%C3%ADn%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Cristina Madrid-Sandín</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Cristina Madrid-Sandín</span></h3>
<div class="p">
<sup>1</sup>Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">
<sup>2</sup>Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Madrid-Sand%C3%ADn%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Cristina Madrid-Sandín</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22L%C3%B3pez-Panad%C3%A9s%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Maria López-Panadés</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Maria López-Panadés</span></h3>
<div class="p">
<sup>1</sup>Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">
<sup>2</sup>Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22L%C3%B3pez-Panad%C3%A9s%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maria López-Panadés</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Buza%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Carolina Buza</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Carolina Buza</span></h3>
<div class="p">
<sup>1</sup>Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">
<sup>2</sup>Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Buza%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Carolina Buza</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferrer%20Miralles%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Neus Ferrer Miralles</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Neus Ferrer Miralles</span></h3>
<div class="p">
<sup>4</sup>Institut de Biotecnologia i de Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallés, Spain</div>
<div class="p">
<sup>5</sup>Centro de Investigación Biomédica en Red de Bioingeniería, Biomateriales y Nanomedicina, Instituto de Salud Carlos III, Cerdanyola del Vallès, Spain</div>
<div class="p">
<sup>6</sup>Departament de Genètica i de Microbiologia, Universitat Autònoma de Barcelona, Cerdanyola del Vallés, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferrer%20Miralles%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Neus Ferrer Miralles</span></a>
</div>
</div>
<sup>4,</sup><sup>5,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Iborra%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Antoni Iborra</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Antoni Iborra</span></h3>
<div class="p">
<sup>7</sup>Servei de Cultius Cel·lulars, Producció d'Anticossos i Citometria, Universitat Autònoma de Barcelona, Cerdanyola del Vallés, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Iborra%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Antoni Iborra</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pujol%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Anna Pujol</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Anna Pujol</span></h3>
<div class="p">
<sup>8</sup>Transgenic Animal Unit, Center of Animal Biotechnology and Gene Therapy, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pujol%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anna Pujol</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pend%C3%A1s%20AM%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Alberto M Pendás</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Alberto M Pendás</span></h3>
<div class="p">
<sup>3</sup>Molecular Mechanisms Program, Centro de Investigación del Cáncer and Instituto de Biología Molecular y Celular del Cáncer (CSIC-Universidad de Salamanca), Salamanca, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pend%C3%A1s%20AM%22%5BAuthor%5D" class="usa-link"><span class="name western">Alberto M Pendás</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roig%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Ignasi Roig</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Ignasi Roig</span></h3>
<div class="p">
<sup>1</sup>Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">
<sup>2</sup>Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roig%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ignasi Roig</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<div class="cg p">Editors: <span class="name western">Carmen J Williams</span><sup>9</sup>, <span class="name western">Wei Yan</span><sup>10</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>Genome Integrity and Instability Group, Institut de Biotecnologia i Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div id="aff2">
<sup>2</sup>Department of Cell Biology, Physiology, and Immunology, Cytology and Histology Unit, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div id="aff3">
<sup>3</sup>Molecular Mechanisms Program, Centro de Investigación del Cáncer and Instituto de Biología Molecular y Celular del Cáncer (CSIC-Universidad de Salamanca), Salamanca, Spain</div>
<div id="aff4">
<sup>4</sup>Institut de Biotecnologia i de Biomedicina, Universitat Autònoma de Barcelona, Cerdanyola del Vallés, Spain</div>
<div id="aff5">
<sup>5</sup>Centro de Investigación Biomédica en Red de Bioingeniería, Biomateriales y Nanomedicina, Instituto de Salud Carlos III, Cerdanyola del Vallès, Spain</div>
<div id="aff6">
<sup>6</sup>Departament de Genètica i de Microbiologia, Universitat Autònoma de Barcelona, Cerdanyola del Vallés, Spain</div>
<div id="aff7">
<sup>7</sup>Servei de Cultius Cel·lulars, Producció d'Anticossos i Citometria, Universitat Autònoma de Barcelona, Cerdanyola del Vallés, Spain</div>
<div id="aff8">
<sup>8</sup>Transgenic Animal Unit, Center of Animal Biotechnology and Gene Therapy, Universitat Autònoma de Barcelona, Cerdanyola del Vallès, Spain</div>
<div id="aff9">
<sup>9</sup>National Institute of Environmental Health Sciences, United States</div>
<div id="aff10">
<sup>10</sup>Washington State University, United States</div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Carmen J Williams</span></strong>: <span class="role">Reviewing Editor</span>
</div>
<div>
<strong class="contrib"><span class="name western">Wei Yan</span></strong>: <span class="role">Senior Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2024, Huang et al</div>
<p>This article is distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use and redistribution provided that the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12367297  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40833257/" class="usa-link">40833257</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article">
<div>
<strong>Previous version available:</strong> This article is based on a previously available preprint with doi: <a href="https://doi.org/10.1101/2024.02.06.579114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1101/2024.02.06.579114</a>.</div>
<div>
<strong>Previous version available:</strong> This article is based on a previously available preprint with doi: <a href="https://doi.org/10.7554/eLife.96052.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.7554/eLife.96052.1</a>.</div>
<div>
<strong>Previous version available:</strong> This article is based on a previously available preprint with doi: <a href="https://doi.org/10.7554/eLife.96052.2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.7554/eLife.96052.2</a>.</div>
<div>
<strong>Previous version available:</strong> This article is based on a previously available preprint with doi: <a href="https://doi.org/10.7554/eLife.96052.3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.7554/eLife.96052.3</a>.</div>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Infertility affects up to 12% of couples globally, with genetic factors contributing to nearly half of the cases. Advances in genomic technologies have led to the discovery of genes like <em>Bend2</em>, which play a crucial role in gametogenesis. In the testis, <em>Bend2</em> expresses two protein isoforms: full-length and a smaller one. Ablation of both proteins results in an arrested spermatogenesis. Because the <em>Bend2</em> locus is on the X chromosome, and the <em>Bend2<sup>-/y</sup></em> mutants are sterile, BEND2’s role in oogenesis remained elusive. In this study, we employed a novel <em>Bend2</em> mutation that blocks the expression of the full-length BEND2 protein but allows the expression of the smaller BEND2 isoform. Interestingly, this mutation does not confer male sterility and mildly affects spermatogenesis. Thus, it allowed us to study the role of BEND2 in oogenesis. Our findings demonstrate that full-length BEND2 is dispensable for male fertility, and its ablation leads to a reduced establishment of the ovarian reserve. These results reveal a critical role for full-length BEND2 in oogenesis and provide insights into the mechanisms underlying the establishment of the ovarian reserve. Furthermore, these findings hold relevance for the diagnostic landscape of human infertility.</p>
<section id="kwd-group2" class="kwd-group"><p><strong>Research organism:</strong> Mouse</p></section></section><section id="s1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Infertility is a complex and multifaceted issue affecting approximately 8–12% of couples worldwide, with genetic factors playing a significant role in up to 50% of cases (<a href="#bib12" class="usa-link" aria-describedby="bib12">Ding and Schimenti, 2021</a>). The intricate process of human reproduction involves a delicate interplay of numerous genes and biological pathways, making the genetic landscape of infertility both diverse and challenging to unravel.</p>
<p>Advancements in genomic technologies and high-throughput sequencing have facilitated the identification of novel genes implicated in reproductive processes. Among these discoveries, <em>Bend2</em> has recently emerged as a novel gene involved in mammalian gametogenesis (<a href="#bib25" class="usa-link" aria-describedby="bib25">Malcher et al., 2022</a>). BEND2 is a member of the BEN domain-containing protein family, whose members play diverse roles in cellular processes such as transcriptional regulation, chromatin remodeling, and protein-protein interactions (<a href="#bib1" class="usa-link" aria-describedby="bib1">Abhiman et al., 2008</a>).</p>
<p>Mouse models have been instrumental in identifying novel genes involved in mammalian fertility. This approach has significantly expanded our understanding of the genetic factors underlying reproductive processes in both males and females (<a href="#bib14" class="usa-link" aria-describedby="bib14">Garretson et al., 2023</a>). A recent study reported that BEND2 is required to complete spermatogenesis since it is a crucial regulator of mammalian meiosis in spermatocytes (<a href="#bib24" class="usa-link" aria-describedby="bib24">Ma et al., 2022</a>). Click or tap here to enter text. BEND2 is specifically expressed in spermatogenic cells shortly before and during the prophase of meiosis I. In the testis, <em>Bend2</em> expresses two protein isoforms: full-length, named p140, and smaller one, p80. BEND2 is essential for the transition from zygonema to pachynema, as its knockout in male mice arrests meiosis at this stage. Its absence leads to disrupted synapsis and induced non-homologous chromosomal pairing. BEND2 interacts with multiple chromatin-associated proteins, including components of transcription-repressor complexes. Additionally, BEND2 regulates chromatin accessibility and modifies H3K4me3, influencing the overall chromatin state during spermatogenesis. These functions collectively establish BEND2 as a key regulator of meiosis, gene expression, and chromatin state in mouse male germ cells. Nonetheless, its role in oogenesis remains unexplored since the <em>Bend2</em> locus is on the X chromosome, and mutant males are sterile.</p>
<p>In this study, we aimed to elucidate the role of <em>Bend2</em> in oogenesis using a novel mutation of <em>Bend2</em> that does not confer male sterility. Using this novel <em>Bend2</em> mutation, we have investigated the effects of full-length BEND2 ablation on meiosis, oocyte quality, and follicular dynamics. Furthermore, as this mutation does not cause male infertility, it allowed us to investigate the role of BEND2 in mammalian meiosis in more detail. Our findings contribute to expanding our understanding of the genetic factors influencing mammalian fertility and provide potential avenues for diagnosing genetic determinants of human infertility.</p></section><section id="s2"><h2 class="pmc_sec_title">Results</h2>
<section id="s2-1"><h3 class="pmc_sec_title">BEND2 is highly expressed in spermatogenic nuclei before meiosis initiation and during early meiotic prophase I, independent of meiotic recombination</h3>
<p>Taking advantage of several unannotated transcripts detected in 14 dpp wild-type mouse testis from our previous RNA sequencing analysis (<a href="#bib28" class="usa-link" aria-describedby="bib28">Marcet-Ortega and Roig, 2016</a>), we found a novel splice variant of the then annotated <em>Gm15262</em> gene, which is an X-linked gene containing two BEN domains (<a href="#fig1" class="usa-link">Figure 1A</a>). This variant is specifically expressed in mouse testes and fetal ovaries containing germ cells undergoing meiotic prophase I and shows a nuclear localization at the heterochromatin of spermatocytes when the GFP-tagged splice variant is electroporated to the testes of young mice (<a href="#fig1s1" class="usa-link">Figure 1—figure supplement 1A–B</a>). Thus, we hypothesize that this gene could be essential for spermatocyte and oocyte development. Based on sequence homology, we named this <em>Gm15262</em> gene as the mouse homolog of the human <em>Bend2</em>, and thus, we renamed it <em>Bend2</em>, as also proposed by others recently (<a href="#bib24" class="usa-link" aria-describedby="bib24">Ma et al., 2022</a>).</p>
<figure class="fig xbox font-sm" id="fig1"><h4 class="obj_head">Figure 1. Expression of BEND2 during spermatogenesis.</h4>
<figcaption><p>(<strong>A</strong>) Schematic representation of mouse <em>Gm15262</em>/<em>Bend2</em> and its novel splice variant <em>255</em> p1. The exons are shown as purple boxes. The predicted domains are labeled below the exons. SAP130_C: C-terminal domain of histone deacetylase complex subunit SAP130; BEN: BEN domain. A pair of gRNAs (orange arrows) target exon 11 of <em>Bend2</em> within the first BEN domain to generate the <em>Bend2<sup>D11</sup></em> mutation by CRISPR/Cas9. The <em>in-house</em> BEND2 antibody is generated using full-length 255P1 sequence as immunogen. (<strong>B</strong>) BEND2 localization in wild-type mouse testis. Staging of the seminiferous epithelium is based on the localization of spermatocytes (indicated by the expression and localization of YH2AX (green)) and spermatids (indicated by DAPI). The tubule stage is shown in uppercase Roman numerals. Scale bar, 20 µm. (<strong>C</strong>) Magnification of BEND2-positive cells from testis sections. Expression of BEND2 was characterized in cells along spermatogenesis from spermatogonia to late diplotene spermatocyte. Scale bar, 10 µm. (<strong>D</strong>) BEND2 localization in SPO11- and DMC1-deficient testis. In these cases, SYCP3 (green) was used to identify spermatocytes. Scale bar, 50 µm. Spg: spermatogonia; B: B-type spermatogonia; PL: pre-leptotene spermatocyte; L: leptotene spermatocyte; Z: zygotene spermatocyte; Z-like: zygotene-like spermatocyte; EP: early pachytene spermatocyte; LP: late pachytene spermatocyte; P-like: pachytene-like spermatocyte; D: diplotene spermatocyte.</p></figcaption><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367297_elife-96052-fig1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e0f/12367297/ed7519b66735/elife-96052-fig1.jpg" loading="lazy" height="1010" width="735" alt="Figure 1."></a></p>
<div class="p"><figure class="fig xbox font-sm" id="fig1s1"><h5 class="obj_head">Figure 1—figure supplement 1. Expression and localization of <em>Bend2</em> (<strong>A</strong>) Expression of 255P1 in mouse gonads and somatic tissues by RT-PCR.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367297_elife-96052-fig1-figsupp1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e0f/12367297/3ab593a23a3b/elife-96052-fig1-figsupp1.jpg" loading="lazy" height="978" width="735" alt="Figure 1—figure supplement 1."></a></p>
<figcaption><div class="p">Actin-400bp; 255P1-576bp.(<strong>B</strong>) Expression pattern of EGFP-255P1 in mouse testis. Representative images of spermatocyte squash and spread stained against SYCP3 and GFP. DNA is counterstained with DAPI (blue). Squash: patch-like signals at the heterochromatic region. Spread 1: cell exhibiting clear foci signals at telomeres along with patch-like signals. Spread 2: cell exhibiting foci signals at the chromosomal telomeres and axes. Signal accumulation at XY chromosomes (white square). Scale bar, 10 µm. (<strong>C</strong>) BEND2 localization in 16 dpc wild-type mouse ovary. 16 dpc ovary sections were treated with antigen retrieval using Tris-EDTA buffer and stained with BEND2 (red) antibody and YH2AX (green) antibody. Strong BEND2 signals were occasionally found in early meiotic prophase oocytes (upper) and oogonia (lower) before DSBs appeared. Scale bar, 20 µm.</div>
<div class="p">
<section class="sm xbox font-sm" id="fig1s1sdata1"><div class="caption p"><span>Figure 1—figure supplement 1—source data 1. Source images for <a href="#fig1s1" class="usa-link">Figure 1—figure supplement 1A</a>.</span></div>
<div class="media p" id="d67e703"><div class="caption">
<a href="/articles/instance/12367297/bin/elife-96052-fig1-figsupp1-data1.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-96052-fig1-figsupp1-data1.zip</a><sup> (427.9KB, zip) </sup>
</div></div></section>
</div>
<div class="p">
<section class="sm xbox font-sm" id="fig1s1sdata2"><div class="caption p"><span>Figure 1—figure supplement 1—source data 2. Source images for <a href="#fig1s1" class="usa-link">Figure 1—figure supplement 1A</a>.</span></div>
<div class="media p" id="d67e715"><div class="caption">
<a href="/articles/instance/12367297/bin/elife-96052-fig1-figsupp1-data2.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-96052-fig1-figsupp1-data2.zip</a><sup> (91.5KB, zip) </sup>
</div></div></section>
</div></figcaption></figure></div>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>We generated a polyclonal antibody against BEND2 using the novel splice variant of <em>Bend2</em> as an immunogen (see validation in <a href="#fig2" class="usa-link">Figure 2</a>). To examine BEND2 expression during spermatogenesis, we immunostained PFA-fixed testis sections against BEND2 and the DNA damage marker γH2AX to allow the identification of spermatocytes. BEND2 was abundantly detected in the nuclei of the peripheral cells of tubules from stage V until XII-I (<a href="#fig1" class="usa-link">Figure 1B</a>). This nuclear staining first appeared in spermatogonia before DSBs were generated. BEND2 staining was highly present in spermatocytes from pre-leptotene when meiosis initiates and persisted until early pachytene. In late pachytene and diplotene cells, little BEND2 remained in the nuclei of spermatocytes (<a href="#fig1" class="usa-link">Figure 1C</a>). Notably, this expression pattern of BEND2 is reminiscent of the one reported before (<a href="#bib24" class="usa-link" aria-describedby="bib24">Ma et al., 2022</a>).</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">Figure 2. Disruption of BEND2 in <em>Bend2<sup>D11/y</sup></em> mice.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367297_elife-96052-fig2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e0f/12367297/ac19542675ff/elife-96052-fig2.jpg" loading="lazy" height="648" width="735" alt="Figure 2."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Mouse appearance and testis size. Scale bar, 5 mm. (<strong>B</strong>) <em>Bend2</em> expression in testis by RT-PCR. The expected size of the wild-type (WT) allele was 576 bp. Sanger sequencing results showed that the amplified DNA in the mutated allele (372 bp, green arrowhead) was from an mRNA that resulted from skipping exon 11 of <em>Bend2</em>. (<strong>C</strong>) Detection of BEND2 by WB from testis protein extracts. The orange arrowhead indicates the full-length BEND2 protein band. An extra protein band ~75 kDa (black arrowhead) was also detected in both wild-type and mutant testes. (<strong>D</strong>) Detection of BEND2 by IF. Testis sections were treated with antigen retrieval using Tris-EDTA buffer before staining against BEND2 (red) and SYCP3 (green). The BEND2 staining observed in the <em>Bend2</em> mutants closely resembles background staining (not shown), suggesting that the cytoplasmic signal might be nonspecific. Scale bar, 10 µm.</p>
<div class="p">
<section class="sm xbox font-sm" id="fig2sdata1"><div class="caption p"><span>Figure 2—source data 1. Source images for <a href="#fig2" class="usa-link">Figure 2B</a>.</span></div>
<div class="media p" id="d67e774"><div class="caption">
<a href="/articles/instance/12367297/bin/elife-96052-fig2-data1.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-96052-fig2-data1.zip</a><sup> (459.8KB, zip) </sup>
</div></div></section>
</div>
<div class="p">
<section class="sm xbox font-sm" id="fig2sdata2"><div class="caption p"><span>Figure 2—source data 2. Source images for <a href="#fig2" class="usa-link">Figure 2B</a>.</span></div>
<div class="media p" id="d67e786"><div class="caption">
<a href="/articles/instance/12367297/bin/elife-96052-fig2-data2.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-96052-fig2-data2.zip</a><sup> (200KB, zip) </sup>
</div></div></section>
</div></figcaption></figure><p>Furthermore, to reveal if the localization of BEND2 in germ cells could have originated as a response to critical events of the meiotic prophase, we examined BEND2 expression in testis sections from recombination-defective mice lacking SPO11. In SPO11-deficient testis, where no DSBs are formed at the onset of meiosis, spermatocytes cannot initiate meiotic recombination, thereby failing to synapse and entering apoptosis (<a href="#bib4" class="usa-link" aria-describedby="bib4">Barchi et al., 2005</a>; <a href="#bib5" class="usa-link" aria-describedby="bib5">Baudat et al., 2000</a>; <a href="#bib33" class="usa-link" aria-describedby="bib33">Pacheco et al., 2015</a>). Interestingly, we found BEND2 was extensively present in nuclei of spermatogonia, leptotene, zygotene, and zygotene-like spermatocytes, as in wild-type mice (<a href="#fig1" class="usa-link">Figure 1D</a>). Very occasionally, a few BEND2 signals could be observed in some SPO11-deficient spermatocytes, presumably at a more advanced pachytene-like stage. This is consistent with its reduced expression after early pachytene in wild-type mice. Similarly, BEND2 expression was not altered in <em>Dmc1</em> mutant spermatocytes (<a href="#fig1" class="usa-link">Figure 1D</a>), which cannot complete meiotic recombination (<a href="#bib4" class="usa-link" aria-describedby="bib4">Barchi et al., 2005</a>; <a href="#bib35" class="usa-link" aria-describedby="bib35">Pittman et al., 1998</a>). Altogether, these results indicate that although BEND2 is highly expressed during early meiotic prophase I when meiotic recombination initiates and progresses, its expression starts as early as in spermatogonia before meiosis begins and is independent of either DSB formation or completion of recombination.</p>
<p>We also examined BEND2 expression in 16 dpc fetal ovary sections by IF and found it localizing at nuclei of some zygotene stage oocytes (<a href="#fig1s1" class="usa-link">Figure 1—figure supplement 1C</a>). However, due to scarce samples and technical obstacles, we could not characterize BEND2 expression in oocytes at earlier stages from 12 to 14 dpc ovary sections.</p></section><section id="s2-2"><h3 class="pmc_sec_title">Full-length BEND2 deficiency causes increased apoptosis and persistent unrepaired DSBs in spermatocytes</h3>
<p>To address the germ cell functions of BEND2 in mice, we generated BEND2-deficient mice by CRISPR/Cas9. Part of exon 11 of <em>Bend2</em> was targeted to be removed to disrupt the first BEN domain (<a href="#fig1" class="usa-link">Figure 1A</a>). <em>Bend2Δ<sup>11/y</sup></em> mice developed into adults without apparent differences in general physical appearance compared to their littermates. Male <em>Bend2<sup>D11/y</sup></em> mice were fertile. The size and weight of <em>Bend2<sup>D11/y</sup></em> testes were comparable to that of wild-type testes (<a href="#fig2" class="usa-link">Figure 2A</a> and <a href="#supp1" class="usa-link">Supplementary file 1</a>). Similarly to <a href="#bib24" class="usa-link" aria-describedby="bib24">Ma et al., 2022</a>, using our BEND2 antibody, we detected two BEND2-related proteins in wild-type testes by WB: one was ~130 kDa, and another was ~75 kDa (<a href="#fig2" class="usa-link">Figure 2C</a>). Although the predicted size of full-length BEND2 is 80.9 kDa, based on previous reports (<a href="#bib24" class="usa-link" aria-describedby="bib24">Ma et al., 2022</a>), the 130 kDa protein corresponds to the full-length BEND2, which displays a slower electrophoretic mobility due to its unusual sequence/structure. The 75 kDa protein corresponds to a smaller version of BEND2 lacking the N-terminus, p80 (<a href="#bib24" class="usa-link" aria-describedby="bib24">Ma et al., 2022</a>). Interestingly, we did not detect any BEND2 protein in wild-type testes of 48.3 kDa, which would be expected from the novel transcript 255p1 we identified.</p>
<p>We were unable to detect the full-length BEND2 in <em>Bend2<sup>D11/y</sup></em> mouse testis extracts by WB (<a href="#fig2" class="usa-link">Figure 2C</a>). The staining of BEND2 observed in wild-type cells was not present in mutant cells (<a href="#fig2" class="usa-link">Figure 2D</a>). However, the p80 BEND2-related protein was still present in <em>Bend2<sup>D11/y</sup></em> mutant testes (<a href="#fig2" class="usa-link">Figure 2C</a>). Moreover, an alternative <em>Bend2</em> transcript skipping exon 11 was detected in <em>Bend2<sup>D11/y</sup></em> testis by RT-PCR and sequencing (<a href="#fig2" class="usa-link">Figure 2B</a>).</p>
<p>We attempted to test if this exon 11-skipped transcript was also present in wild-type mouse testis by RT-PCR and if it represented the p80 BEND2-related protein detected in both wild-type and <em>Bend2<sup>D11/y</sup></em> testis by immunoprecipitation (IP) coupled to peptide mass fingerprinting analysis. Unfortunately, only the full-length <em>Bend2</em>-202 transcript was consistently amplified in our RT-PCR analysis, and our <em>in-house</em> BEND2 antibody did not give satisfactory IP results (<em>Data was not shown</em>). Overall, these results suggest that the full-length BEND2 was successfully eliminated from mutant mouse testis. Also, the p80 BEND2 protein, likely corresponding to an exon 11, skipped transcript, is present and might be functional in our mutant testis, based on the observed phenotype (see below).</p>
<p>Histological analysis of <em>Bend2<sup>D11/y</sup></em> testes revealed the presence of cells at all the stages of spermatogenesis (<a href="#fig3" class="usa-link">Figure 3A</a>). However, a significant increase in apoptotic cells was detected in <em>Bend2<sup>D11/y</sup></em> testes via TUNEL assays (p&lt;0.0001 t-test, <a href="#fig3" class="usa-link">Figure 3B–C</a>). Further staging analysis showed that <em>Bend2<sup>D11/y</sup></em> mice presented a comparable number of apoptotic spermatocytes (66.8 ± 9.7%, mean ± SD, N=4) than wild-type mice (52.5 ± 9.5%, mean ± SD, N=4, p&gt;0.05 One-Way ANOVA, <a href="#fig3" class="usa-link">Figure 3D</a>). Therefore, the full-length BEND2 might be dispensable for completing spermatogenesis, and its absence only caused a slight defect in spermatogenesis.</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">Figure 3. The full-length BEND2 is dispensable to complete spermatogenesis in mice.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367297_elife-96052-fig3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e0f/12367297/a8eb67d59ab7/elife-96052-fig3.jpg" loading="lazy" height="744" width="735" alt="Figure 3."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Representative PAS-H stained mouse testis sections. The square in the left image shows the zoomed images on the right. Scale bar on the right images, 20 µm. (<strong>B</strong>) Apoptosis detection on testis sections by TUNEL assay. Scale bar, 20 µm. (<strong>C</strong>) Quantification of TUNEL-positive cells. The horizontal lines represent the mean ± SD. N=1,125 for <em>Bend2<sup>+/y</sup></em>; N=1195 for <em>Bend2<sup>D11/y</sup></em>, ****p&lt;0.0001 t-test. (<strong>D</strong>) Classification of TUNEL-positive cells. The columns and error lines indicate the mean ± SD. N=4, p&gt;0.05 One-Way ANOVA.</p></figcaption></figure><p>To further explore the causes of the increased apoptosis in BEND2–deficient testes, we assessed chromosome synapsis and meiotic recombination progress in spermatocytes by IF. In wild-type spermatocytes, SYCP3 began to form each developing chromosome axis at leptotene. At zygotene, synapsis is initiated as SYCP1 appears at the synapsed region of the homologs. At pachytene, synapsis was completed as SYCP3 and SYCP1 completely colocalized. At diplotene, SCs disassembled, and SYCP1 was lost from separated SYCP3-labelled axes, but homologous chromosomes remained held together by chiasmata.</p>
<p>Synapsis progressed similarly to wild-type spermatocytes in <em>Bend2<sup>D11/y</sup></em> spermatocytes (<a href="#fig4" class="usa-link">Figure 4A</a>). However, the fraction of diplotene spermatocytes was significantly increased (p=0.019, One-Way ANOVA). Additionally, the fraction of pachytene spermatocytes (48.6 ± 6.8%, mean ± SD, N=4) seemed to decrease in <em>Bend2<sup>D11/y</sup></em> mice, compared to wild-type mice (57.3 ± 4.5%, mean ± SD, N=4, p=0.078 One-Way ANOVA). This accumulation of <em>Bend2<sup>D11/y</sup></em> spermatocytes at the diplotene stage could be explained if there was a later block of the meiotic progression or if <em>Bend2<sup>D11/y</sup></em> spermatocytes might exit pachytene faster than wild-type spermatocytes. Taken together, these results demonstrated that in the absence of the full-length BEND2, spermatocytes could complete synapsis and progress through meiotic prophase but following a slightly altered timeline.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">Figure 4. <em>Bend2<sup>D11/y</sup></em> males display minor recombination defects.</h4>
<figcaption><p>(<strong>A</strong>) Chromosomal synapsis in spermatocytes. Representative images of SYCP3 and SYCP1 staining in spermatocyte nuclei from the stages shown (left). Meiotic prophase staging of spermatocytes (right). L: leptotene, Z: zygotene, P: pachytene, D: diplotene. The columns and lines indicate the mean and SD. N=4, *p=0.019 One-Way ANOVA. (<strong>B</strong>) Examination of DSBs in <em>Bend2<sup>D11/y</sup></em> spermatocytes. Representative images of YH2AX staining in spermatocyte nuclei along meiotic prophase (left). Increased YH2AX signals are detected during late prophase in <em>Bend2<sup>D11/y</sup></em> spermatocyte (white arrowhead). Quantification of YH2AX patches in spermatocyte nuclei per sub-stages (right). The patches were counted manually using the same method for every pair of control and mutant mice. From left to right, N=76/74, 86/83, 73/75, and 78/81, ***p=0.0001 (late pachytene) and 0.0007 (early diplotene), *p=0.0222 t-test. Analysis of replication protein A (RPA) (<strong>C</strong>) and RAD51 (D) foci counts in control and <em>Bend2<sup>D11/y</sup></em> spermatocytes. RPA and RAD51 foci were counted using ImageJ with the same method for every <em>Bend2<sup>+/y</sup></em> and <em>Bend2<sup>D11/y</sup></em> mice pair. From left to right, N=44/43, 51/43, 59/46, 45/39, 81/77, and 48/43, **p=0.0015 for RPA quantification; N=52/43, 42/46, 44/46, 51/45, 77/73, and 45/49, *p=0.0428 and ***p=0.0002 for RAD51 quantification; t-test. (<strong>C</strong>) Examination of CO formation in <em>Bend2<sup>D11/y</sup></em> spermatocytes. Representative images of MLH1 in control and mutant spermatocyte nuclei (left). Quantification of MLH1 foci in spermatocyte nuclei (right). Only spermatocytes containing ≥19 MLH1 foci/nucleus were counted. N=74 for <em>Bend2<sup>+/y</sup></em> and 68 for <em>Bend2<sup>D11/y</sup></em>, p&gt;0.05 Mann-Whitney test. The horizontal lines represent mean ± SD (<strong>B–E</strong>). Scale bar, 10 µm.</p></figcaption><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367297_elife-96052-fig4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e0f/12367297/c4bbd1f54235/elife-96052-fig4.jpg" loading="lazy" height="1070" width="735" alt="Figure 4."></a></p>
<div class="p"><figure class="fig xbox font-sm" id="fig4s1"><h5 class="obj_head">Figure 4—figure supplement 1. Examination of the non-homologous end joining (NHEJ) pathway activity in <em>Bend2<sup>D11/</sup></em><strong><sup>y</sup></strong> mice.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367297_elife-96052-fig4-figsupp1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e0f/12367297/6a35ba15f680/elife-96052-fig4-figsupp1.jpg" loading="lazy" height="909" width="735" alt="Figure 4—figure supplement 1."></a></p>
<figcaption><div class="p">(<strong>A</strong>) Representative images of Ku70 expression pattern in wild-type and <em>Bend2<sup>D11/y</sup></em> testis. Scale bar, 40 μm. Yellow arrows indicate the Ku70-positive sex body, and red arrows are two examples of Ku70 foci. (<strong>B</strong>) Quantification of Ku70 foci in pachytene and diplotene spermatocytes. Foci colocalizing on SYCP3 were counted manually from at least 40 cells per animal. The columns and lines indicate the mean and SD (N=2). (<strong>C</strong>) Western blot analysis of Ku70 and LINE-1 proteins. (<strong>D</strong>) Quantification of relative Ku70 protein levels by WB. The columns and lines indicate the mean and SD (N=3).</div>
<div class="p">
<section class="sm xbox font-sm" id="fig4s1sdata1"><div class="caption p"><span>Figure 4—figure supplement 1—source data 1. Source images for <a href="#fig4" class="usa-link">Figure 4D</a>.</span></div>
<div class="media p" id="d67e1068"><div class="caption">
<a href="/articles/instance/12367297/bin/elife-96052-fig4-figsupp1-data1.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-96052-fig4-figsupp1-data1.zip</a><sup> (964.8KB, zip) </sup>
</div></div></section>
</div>
<div class="p">
<section class="sm xbox font-sm" id="fig4s1sdata2"><div class="caption p"><span>Figure 4—figure supplement 1—source data 2. Source images for <a href="#fig4" class="usa-link">Figure 4D</a>.</span></div>
<div class="media p" id="d67e1080"><div class="caption">
<a href="/articles/instance/12367297/bin/elife-96052-fig4-figsupp1-data2.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-96052-fig4-figsupp1-data2.zip</a><sup> (102.2KB, zip) </sup>
</div></div></section>
</div></figcaption></figure></div>
<div class="p"><figure class="fig xbox font-sm" id="fig4s2"><h5 class="obj_head">Figure 4—figure supplement 2. Analysis of LINE-1 retrotransposon expression in <em>Bend2<sup>D11/</sup></em><strong><sup>y</sup></strong> mice.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367297_elife-96052-fig4-figsupp2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e0f/12367297/efe256e677e7/elife-96052-fig4-figsupp2.jpg" loading="lazy" height="746" width="735" alt="Figure 4—figure supplement 2."></a></p>
<figcaption><div class="p">(<strong>A</strong>) Representative images of LINE-1 expression pattern in wild-type and <em>Bend2<sup>D11/y</sup></em> testis. Scale bar, 40 μm. Quantification of LINE-1 positive tubules (<strong>B</strong>) and relative protein levels in <em>Bend2<sup>D11/y</sup></em> animals from Figure S2 (<strong>C</strong>). The columns and lines indicate the mean and SD (N=3). *p=0.046 (<strong>B</strong>) and *p=0.02 (<strong>C</strong>) t-test.</div></figcaption></figure></div>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>During early meiosis, recombination initiates with SPO11-mediated DSB formation, leading to the phosphorylation of the histone H2AX by ATM and thus triggering a series of DSB repair responses in the meiotic prophase (<a href="#bib18" class="usa-link" aria-describedby="bib18">Huang and Roig, 2023</a>). In wild-type mice, ϒH2AX progressively disappeared from the autosomes while DSBs were repaired as prophase progressed. By pachytene, most ϒH2AX was associated with the sex body. Few ϒH2AX patches could be observed on the autosomes from late pachytene, presumably corresponding to unrepaired DSBs (<a href="#fig4" class="usa-link">Figure 4B</a>).</p>
<p>Interestingly, a marked increase in the number of ϒH2AX patches was detected from early pachytene until late diplotene in <em>Bend2<sup>D11/y</sup></em> spermatocytes, indicating a possible delay in DSB repair (<a href="#fig4" class="usa-link">Figure 4B</a>). Alternatively, the same results could be observed if more DSBs were formed during prophase in <em>Bend2<sup>D11/y</sup></em> spermatocytes.</p>
<p>Once DSBs are resected, replication protein A (RPA) transiently binds to the nascent ssDNA overhangs, promoting the assembly of the recombinases RAD51 and DMC1 at DSB sites (<a href="#bib31" class="usa-link" aria-describedby="bib31">Moens et al., 2002</a>). RAD51 and DMC1 replace RPA and form nucleoprotein filaments with the ssDNA, directing homology search and strand invasion to proceed along the DSB repair pathway (<a href="#bib6" class="usa-link" aria-describedby="bib6">Brown and Bishop, 2015</a>; <a href="#bib17" class="usa-link" aria-describedby="bib17">Hinch et al., 2020</a>).</p>
<p>In wild-type spermatocytes, the number of RPA foci peaked around early zygotene and progressively diminished as recombination proceeded; on the other hand, there were many RAD51 foci since leptotene, and the number started to decline from early zygotene (<a href="#fig4" class="usa-link">Figure 4C–D</a>), consistent with previous studies (<a href="#bib31" class="usa-link" aria-describedby="bib31">Moens et al., 2002</a>; <a href="#bib33" class="usa-link" aria-describedby="bib33">Pacheco et al., 2015</a>). In <em>Bend2<sup>D11/y</sup></em> males, the spermatocytes had similar initial RPA and RAD51 foci numbers, suggesting that DSB formation was not affected by the loss of BEND2 (<a href="#fig4" class="usa-link">Figure 4C–D</a>). However, <em>Bend2<sup>D11/y</sup></em> cells tended to accumulate more RPA foci and lose RAD51 foci as they progressed along the meiotic prophase. The differences became significant for RPA at late leptotene (p=0.0015 t-test, <a href="#fig4" class="usa-link">Figure 4C</a>) and RAD51 at late zygotene and pachytene (p=0.0428 and 0.0002, respectively t-test, <a href="#fig4" class="usa-link">Figure 4D</a>)<em>,</em> indicating a subtle defect in the replacement of RPA by RAD51.</p>
<p>Because of meiotic recombination, at least one crossover per bivalent is generated to ensure accurate chromosome segregation at the first meiotic division (<a href="#bib51" class="usa-link" aria-describedby="bib51">Zickler and Kleckner, 1999</a>). We examined MLH1 foci, which become apparent at mid-late pachytene and mark most crossover-designated sites (<a href="#bib3" class="usa-link" aria-describedby="bib3">Anderson et al., 1999</a>). We did not detect significant differences in the number of MLH1 foci in <em>Bend2<sup>D11/y</sup></em> spermatocytes (22.0 ± 2.3, mean ± SD, N=68) compared to wild-type cells (22.5 ± 2.9, mean ± SD, N=74, p=0.2543, t-test, <a href="#fig4" class="usa-link">Figure 4E</a>), suggesting that BEND2 deficiency did not affect crossover formation in spermatocytes.</p>
<p>Homologous recombination is the principal DSB repair pathway in meiosis responding to SPO11-dependent DSBs. But in late spermatocytes (late pachytene and diplotene), other DSB repair pathways, such as non-homologous end joining (NHEJ) or inter-sister homologous recombination (IS-HR), take over (<a href="#bib2" class="usa-link" aria-describedby="bib2">Ahmed et al., 2010</a>; <a href="#bib13" class="usa-link" aria-describedby="bib13">Enguita-Marruedo et al., 2019</a>; <a href="#bib16" class="usa-link" aria-describedby="bib16">Goedecke et al., 1999</a>). Moreover, during pachytene, new DSBs could be formed by SPO11-independent mechanisms (<a href="#bib8" class="usa-link" aria-describedby="bib8">Carofiglio et al., 2013</a>). Therefore, the increased presence of ϒH2AX patches observed in <em>Bend2</em> mutant mice might be explained by the inactivation of these alternative DSB repair pathways during the late meiotic prophase or an increase of SPO11-independent DSBs during pachytene.</p>
<p>To gain insight into these hypotheses, we studied if the NHEJ pathway activity is reduced during meiosis in <em>Bend2<sup>D11/y</sup></em> mice by examining the localization and expression of its constitutive protein, Ku70, on testes sections. As anticipated, Ku70 was observed as a patch-like signal on sex bodies in pachytene and diplotene spermatocytes. A punctuated signal, marking presumably DSB sites in the nucleus, was seen in cells of all spermatogenesis stages besides round and elongated spermatids (<a href="#fig4s1" class="usa-link">Figure 4—figure supplement 1A</a>). The number of Ku70 foci colocalizing with SYCP3 in spermatocytes in pachytene and diplotene stages was similar between control and mutant samples (<a href="#fig4s1" class="usa-link">Figure 4—figure supplement 1B</a>). No differences were observed in the proportion of tubules expressing Ku70 either. These results were further confirmed with Western blot analysis (<a href="#fig4s1" class="usa-link">Figure 4—figure supplement 1C–D</a>), thus suggesting that <em>Bend2</em> depletion does not affect NHEJ activation.</p></section><section id="s2-3"><h3 class="pmc_sec_title">BEND2 deficiency leads to LINE-1 retrotransposon suppression</h3>
<p>The formation of SPO11-independent DSBs during meiotic prophase has been linked to the LINE-1 retrotransposon activation (<a href="#bib8" class="usa-link" aria-describedby="bib8">Carofiglio et al., 2013</a>; <a href="#bib26" class="usa-link" aria-describedby="bib26">Malki et al., 2014</a>; <a href="#bib45" class="usa-link" aria-describedby="bib45">Soper et al., 2008</a>). To evaluate if the LINE-1 activity was increased in <em>Bend2</em> mutant mice, we examined LINE1 expression in mouse testes by IF and WB.</p>
<p>As expected, the most prominent LINE-1 staining localized to the cytoplasm of leptotene, zygotene, and pachytene spermatocytes (<a href="#fig4s2" class="usa-link">Figure 4—figure supplement 2A</a>). The signal intensity varied greatly between seminiferous tubules of the same sample in both wild-type and mutant animals. In <em>Bend2<sup>D11/y</sup></em> mice, spermatocytes showed similar signal intensity compared to control mice. Contrary to the expected, a statistically significant lower number of LINE-1 positive tubules was observed in mutant mice. An average of 2.8% and 0.5% positive tubules was observed in control and <em>Bend2<sup>D11/y</sup></em> mice, respectively (p=0.02, t-test) (<a href="#fig4s2" class="usa-link">Figure 4—figure supplement 2B</a>). Western blot analysis validated the lower LINE-1 protein expression in <em>Bend2<sup>D11/y</sup></em> testes sections (<a href="#fig4s1" class="usa-link">Figure 4—figure supplement 1B</a>). Indeed, a significant decrease in LINE-1 protein levels was found in <em>Bend2<sup>D11/y</sup></em> mice (p=0.0469, t-test, <a href="#fig4s2" class="usa-link">Figure 4—figure supplement 2C</a>), thus confirming the IF results. Defects in the demethylation efficiency of LINE-1 in <em>Bend2</em> mutant mice during the prophase may explain these results. On the other hand, the signal intensity was equal in wild-type and mutant mice, suggesting that some tubules successfully complete LINE-1 demethylation, thus resulting in normal LINE-1 expression.</p></section><section id="s2-4"><h3 class="pmc_sec_title">BEND2 deficiency results in a reduced ovarian reserve</h3>
<p>A more robust phenotype was observed in females when BEND2 was disrupted compared to males. Female <em>Bend2<sup>D11/D11</sup></em> mice were fertile. However, the litter size was significantly smaller than that in wild-type females (<a href="#fig5" class="usa-link">Figure 5A</a>), and noticeably smaller ovaries were present in mutant females (<a href="#fig5" class="usa-link">Figure 5B</a>).</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">Figure 5. Oogenesis is altered in <em>Bend2</em> mutant females.</h4>
<figcaption><p>(<strong>A</strong>) Fertility evaluation of <em>Bend2<sup>D11/D11</sup></em> females. Two-month-old <em>Bend2<sup>D11/D11</sup></em> females were crossed with wild-type males for 5 months; litter size data of four animals per genotype were collected for analysis. *p=0.0069, t-test. (<strong>B</strong>) DDX4-stained and Periodic Acid Schiff-Hematoxylin (PAS-H) stained histological ovary sections from females at different ages. Scale bars, 50 µm for Immunofluorescent images and 0.5 mm for the PAS-H stained ones. (<strong>C</strong>) Quantification of the number of oocytes found in the analyzed ovaries of newborn, one, three, 15–20, and 40 weeks old mice. The columns and lines indicate the mean and SD. **p=0.0075, 0.0012, and 0.0040; t-test.</p></figcaption><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367297_elife-96052-fig5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e0f/12367297/f2692f065299/elife-96052-fig5.jpg" loading="lazy" height="1014" width="735" alt="Figure 5."></a></p>
<div class="p"><figure class="fig xbox font-sm" id="fig5s1"><h5 class="obj_head">Figure 5—figure supplement 1. Classification of the type of follicles found in control and <em>Bend2<sup>D11/D11</sup></em> ovaries.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367297_elife-96052-fig5-figsupp1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e0f/12367297/5b566460533f/elife-96052-fig5-figsupp1.jpg" loading="lazy" height="652" width="735" alt="Figure 5—figure supplement 1."></a></p>
<figcaption><div class="p">Classification of analyzed follicles from ovaries at the age of 1 week (<strong>A</strong>), 3 weeks (<strong>B</strong>), 15–20 weeks (<strong>C</strong>), and 40 weeks (<strong>D</strong>). The number of ovaries analyzed per genotype and age is four, except for 15–20 weeks, where eight ovaries were analyzed per genotype. The columns and lines indicate the mean and SD. ***p=0.0007 (<strong>A</strong>); **p=0.0024, ****p&lt;0.0001 (<strong>C</strong>); t-test.</div></figcaption></figure></div>
<div class="p"><figure class="fig xbox font-sm" id="fig5s2"><h5 class="obj_head">Figure 5—figure supplement 2. Analysis of LINE-1 retrotransposon expression in <em>Bend2<sup>D11/D11</sup></em> mice.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367297_elife-96052-fig5-figsupp2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e0f/12367297/3d0c05a4fa3b/elife-96052-fig5-figsupp2.jpg" loading="lazy" height="799" width="735" alt="Figure 5—figure supplement 2."></a></p>
<figcaption><div class="p">(<strong>A</strong>) Representative images of LINE-1 expression pattern in wild-type and <em>Bend2<sup>D11/D11</sup></em> ovaries. Scale bar, 40 μm. (<strong>B</strong>) Quantification of LINE-1 positive oocytes per section. The columns and lines indicate the mean and SD (N=5), p=0.1851 t-test.</div></figcaption></figure></div>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>To investigate if the loss of BEND2 affected oogenesis, a more detailed histological analysis of whole ovaries was performed in young (1-week-old), adolescent (3-weeks-old), adult (15–20-weeks-old), and aged (40-weeks-old) <em>Bend2<sup>D11/D11</sup></em> and <em>Bend2<sup>+/+</sup></em> mice. At 1 week, <em>Bend2<sup>D11/D11</sup></em> females exhibited a significantly reduced number of oocytes (p=0.0012, t-test; <a href="#fig5" class="usa-link">Figure 5C</a>), mainly due to a reduced number of primordial follicles (p=0.0007, t-test; <a href="#fig5s1" class="usa-link">Figure 5—figure supplement 1A</a>). Growing follicles were found at all stages, with no statistically significant differences between <em>Bend2<sup>D11/D11</sup></em> and control mice (<a href="#fig5s1" class="usa-link">Figure 5—figure supplement 1A</a>). This tendency continued in prepubertal 3-week-old animals. At this stage, Bend2 mutant ovaries seemed to contain fewer oocytes (p=0.0834, t-test; <a href="#fig5" class="usa-link">Figure 5C</a>) and primordial follicles (p=0.0704, t-test; <a href="#fig5s1" class="usa-link">Figure 5—figure supplement 1B</a>). At an adult stage of 15–20 weeks, the mutant ovaries presented significantly fewer oocytes (p=0.0040, t-test; <a href="#fig5" class="usa-link">Figure 5C</a>), again due to a reduction in the number of primordial follicles (p=0.0024, t-test; <a href="#fig5s1" class="usa-link">Figure 5—figure supplement 1C</a>). Interestingly, at this stage, we also noticed a significant reduction in the number of antral follicles in mutant mice (p&lt;0.0001, t-test; <a href="#fig5s1" class="usa-link">Figure 5—figure supplement 1C</a>), which could partly explain the reduced fertility of these females. Finally, at 40 weeks, there tended to be fewer oocytes and primordial follicles in <em>Bend2<sup>D11/D11</sup></em> ovaries than in <em>Bend2<sup>+/+</sup></em> ovaries (p=0.1171 and p=0.1383, t-test; <a href="#fig5s1" class="usa-link">Figure 5—figure supplement 1D</a>). These results suggest a reduction in the oocyte pool established at birth and the possible occurrence of premature ovarian insufficiency in <em>Bend2</em> mutant females.</p>
<p>To certify that BEND2 was required to establish the ovarian reserve, we counted the oocyte population present in newborn <em>Bend2<sup>+/+</sup></em> and <em>Bend2<sup>D11/D11</sup></em> mutant mice by staining ovarian sections with the germ-cell marker DDX4 (<a href="#fig5" class="usa-link">Figure 5B</a>). <em>Bend2</em> mutant ovaries contained almost 40% fewer oocytes than control ovaries (p=0.0037, t-test; <a href="#fig5" class="usa-link">Figure 5C</a>), evidencing that the absence of the full-length BEND2 resulted in a reduced ovarian reserve.</p></section><section id="s2-5"><h3 class="pmc_sec_title">Mutant <em>Bend2</em> oocytes show persistent unrepaired DSBs and impaired crossover formation</h3>
<p>To address the cause of this reduced ovarian reserve, we first analyze the expression of LINE-1, which has been associated with increased perinatal oocyte death in mice (<a href="#bib27" class="usa-link" aria-describedby="bib27">Malki et al., 2019</a>). However, we found a similar percentage of oocytes expressing LINE-1 in <em>Bend2<sup>+/+</sup></em> and <em>Bend2<sup>D11/D11</sup></em> ovarian sections (20.2±0.6; 16.8±2.3, mean ± SEM; p=0.1851, t-test; <a href="#fig5s2" class="usa-link">Figure 5—figure supplement 2</a>). These data suggest that the reduction in the number of oocytes present in <em>Bend2</em> mutant mice might not be related to an increased expression of LINE-1.</p>
<p>Next, we examined if errors during the meiotic prophase could be responsible for the observed reduced ovarian reserve in <em>Bend2<sup>D11/D11</sup></em> mice. Thus, we studied chromosome synapsis and meiotic recombination progress in fetal oocytes by IF. In wild-type ovaries, most oocytes at 18 dpc were at the pachytene stage, a small fraction was still at the zygotene stage, and a minority at the leptotene stage. At 1 dpp, almost all oocytes were at the pachytene and diplotene stages (<a href="#fig6" class="usa-link">Figure 6A</a>). In <em>Bend2<sup>D11/D11</sup></em> ovaries, zygotene, and pachytene oocytes undergoing normal synapsis and diplotene oocytes undergoing desynapsis were observed at 18 dpc and 1 dpp (<a href="#fig6" class="usa-link">Figure 6A</a>).</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">Figure 6. Synapsis and recombination in <em>Bend2<sup>D11/D11</sup></em> females.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12367297_elife-96052-fig6.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e0f/12367297/2d987d5e5c1b/elife-96052-fig6.jpg" loading="lazy" height="898" width="735" alt="Figure 6."></a></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Chromosomal synapsis in oocytes. Representative images of SYCP3 and SYCP1 staining in 18 dpc and 1 dpp oocyte nuclei are shown (left). Meiotic prophase staging of 18 dpc and 1 dpp oocytes (right). L: leptotene, Z: zygotene, P: pachytene, D: diplotene. The columns and lines indicate the mean and SD. The number of animals analyzed per genotype, 2 for 18 dpc and 3 for 1 dpp. p&gt;0.5 for all the comparisons One-Way ANOVA. (<strong>B</strong>) Examination of DSBs in <em>Bend2<sup>D11/D11</sup></em> oocytes. Representative images of ϒH2AX staining in 18 dpc and 1 dpp oocyte nuclei at the pachytene and diplotene stage (left). Quantification of ϒH2AX patches in oocyte nuclei at sub-stages (right). The mean intensity of ϒH2AX staining in 18 dpc oocyte nuclei was measured by Image J. The patches of ϒH2AX in 1 dpp oocyte nuclei were counted manually. The horizontal lines represent the mean ± SD. From left to right, the number of analyzed nuclei per genotype: 40/30, 64/48, 81/27, and 57/28. *p=0.0288, ****p&lt;0.0001 t-test. (<strong>C</strong>) Analysis of replication protein A (RPA) (<strong>C</strong>) and RAD5 (<strong>D</strong>) foci in control and <em>Bend2<sup>D11/D11</sup></em> oocytes. Representative images of RPA staining in 18 dpc and 1 dpp oocyte nuclei from zygotene to diplotene stage (left). Quantification of RPA foci present in oocyte nuclei at sub-stages (right). Zygotene and early pachytene nuclei analyzed were from 18 dpc ovaries, and late pachytene and diplotene nuclei analyzed were from 1 dpp ovaries. Foci were counted manually. The horizontal lines represent the mean ± SD. From left to right, number of analyzed nuclei: 12/13, 27/17, 41/42, and 65/49 for RPA, and 8/11, 27/23, 38/46, and 62/55 for RAD51. p&gt;0.5 for all the comparisons t-test. (<strong>E</strong>) Examination of CO formation in control and <em>Bend2<sup>D11/D11</sup></em> oocytes. Representative images of MLH1 in 1 dpp oocyte nuclei (left). Quantification of MLH1 foci in 1 dpp oocyte nuclei (right). Only oocytes containing ≥20 MLH1 foci/nucleus were counted. The horizontal lines represent the mean ± SD. N=120 for <em>Bend2<sup>+/+</sup></em>, 95 for <em>Bend2<sup>D11/D11</sup></em>; **p=0.0057 Mann-Whitney test. Scale bar, 10 µm.</p></figcaption></figure><p>Consistent with the observation in <em>Bend2</em> mutant males, we also found increased levels of ϒH2AX in late prophase <em>Bend2<sup>D11/D11</sup></em> oocytes (<a href="#fig6" class="usa-link">Figure 6B</a>). In wild-type females, ϒH2AX patches could be detected along axes in most pachytene oocytes and many diplotene oocytes (<a href="#fig6" class="usa-link">Figure 6B</a>). This is different from wild-type males, in which ϒH2AX can only be detected as very few patches in late pachytene cells and is almost undetectable in most diplotene cells (<a href="#fig4" class="usa-link">Figure 4B</a>), presumably because synapsis progresses faster than recombination in females than in males (<a href="#bib38" class="usa-link" aria-describedby="bib38">Roig et al., 2004</a>).</p>
<p>In <em>Bend2<sup>D11/D11</sup></em> oocytes, higher levels of ϒH2AX were observed at all stages analyzed (early pachytene, late pachytene, and diplotene, <a href="#fig6" class="usa-link">Figure 6B</a>). Many early pachytene <em>Bend2<sup>D11/D11</sup></em> oocytes exhibited an overall intense ϒH2AX signal reminiscent of wild-type zygotene cells (<a href="#fig6" class="usa-link">Figure 6B</a>). To quantitatively compare these differences, we measured the ϒH2AX signal intensity in early pachytene oocytes and counted the ϒH2AX patches in late pachytene and diplotene oocytes. As expected, a significant increase of ϒH2AX was present in <em>Bend2<sup>D11/D11</sup></em> oocytes at all the tested stages (p=0.0288 for early pachytene; p&lt;0.0001 for late pachytene, early diplotene and late diplotene, t-test, <a href="#fig6" class="usa-link">Figure 6B</a>). Thus, like in <em>Bend2</em> mutant males, DSB repair was impaired, or more DSBs were formed in <em>Bend2<sup>D11/D11</sup></em> females.</p>
<p>In females, no differences in the number of RPA and RAD51 foci were found between wild-type and <em>Bend2<sup>D11/D11</sup></em> oocytes, apart from a slight decrease in RPA foci in <em>Bend2<sup>D11/D11</sup></em> oocytes (<a href="#fig6" class="usa-link">Figure 6C</a>) and a slight increase in RAD51 foci in pachytene <em>Bend2<sup>D11/D11</sup></em> oocytes (<a href="#fig6" class="usa-link">Figure 6D</a>). These results suggest that the increased presence of ϒH2AX in <em>Bend2<sup>D11/D11</sup></em> oocytes may not be related to defective meiotic recombination but to other causes, like meiotic silencing of unsynapsed chromosomes (MSUC) or alternative DNA repair pathways.</p>
<p>Interestingly, the number of MLH1 foci in late prophase oocytes was significantly lower in <em>Bend2<sup>D11/D11</sup></em> females than in control females (p=0.006 for <em>Bend2<sup>D11/D11</sup></em>, t-test, <a href="#fig6" class="usa-link">Figure 6E</a>). Thus, we concluded that the depletion of BEND2 caused a reduced crossover formation in oocytes, probably partly causing the reduced litter size in <em>Bend2</em> mutant females.</p>
<p>Finally, to check if these alterations affected <em>Bend2<sup>D11/D11</sup></em> oocyte quality, we performed ovarian stimulation on 7-month-old <em>Bend2<sup>+/D11</sup></em> and <em>Bend2<sup>D11/D11</sup></em> mice. On average, we obtained a similar number of oocytes from the ovarian stimulation in control and mutant mice (10.7±4.1 vs 10.0±1.2; mean ± SEM; p=0.8852, t-test; <a href="#table1" class="usa-link">Table 1</a>). Remarkably, the ability of <em>Bend2<sup>D11/D11</sup></em> mutant oocytes to be fertilized and develop into the blastocyst stage did not seem to be compromised (<a href="#table1" class="usa-link">Table 1</a>), suggesting that the absence of the full-length BEND2 did not compromise oocyte quality, at least to be fertilized and develop up to the blastocyst stage.</p>
<section class="tw xbox font-sm" id="table1"><h4 class="obj_head">Table 1. <em>Bend2<sup>D11/D11</sup></em> oocytes fertilize and develop into blastocyst stage similar to control oocytes.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" valign="top" rowspan="1" colspan="1">Group</th>
<th align="left" valign="top" rowspan="1" colspan="1">Genotype</th>
<th align="left" valign="top" rowspan="1" colspan="1"># mouse</th>
<th align="left" valign="top" rowspan="1" colspan="1">Age (months)</th>
<th align="left" valign="top" rowspan="1" colspan="1"># oocyte</th>
<th align="left" valign="top" rowspan="1" colspan="1"># 2 cell embryo</th>
<th align="left" valign="top" rowspan="1" colspan="1">Fertilization (%)</th>
<th align="left" valign="top" rowspan="1" colspan="1">Blastocyst (%)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Control</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<em>Bend2<sup>+/D11</sup></em>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">3</td>
<td align="left" valign="top" rowspan="1" colspan="1">7</td>
<td align="left" valign="top" rowspan="1" colspan="1">10.7±7.2</td>
<td align="left" valign="top" rowspan="1" colspan="1">7.7±3.8</td>
<td align="left" valign="top" rowspan="1" colspan="1">78.0±19.5</td>
<td align="left" valign="top" rowspan="1" colspan="1">23.3±25.2</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Mutant</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<em>Bend2<sup>D11/D11</sup></em>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">3</td>
<td align="left" valign="top" rowspan="1" colspan="1">7</td>
<td align="left" valign="top" rowspan="1" colspan="1">10.0±2.0</td>
<td align="left" valign="top" rowspan="1" colspan="1">8.3±3.2</td>
<td align="left" valign="top" rowspan="1" colspan="1">82.7±20.5</td>
<td align="left" valign="top" rowspan="1" colspan="1">46.8±12.2</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/table1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section></section><section id="s3"><h2 class="pmc_sec_title">Discussion</h2>
<p>In mammals, the establishment and gradual depletion of the ovarian reserve defines a finite female reproductive life span. In this study, we demonstrated that BEND2, a new player in meiosis (<a href="#bib24" class="usa-link" aria-describedby="bib24">Ma et al., 2022</a>), is essential for the primordial follicle pool setup. The depletion of the full-length BEND2 causes severe defects in female oogenesis with a decreased ovarian reserve and subfertility. However, these defects may not be directly related to meiotic recombination progression or synapsis, although a high level of unrepaired DSBs persists during late meiotic prophase. We also showed that BEND2 has a role in regulating LINE-1 retrotransposon activity in spermatocytes. Our work expands our knowledge of the genetic factors influencing the ovarian reserve, potentially benefiting the genetic diagnosis of human infertility.</p>
<p>BEND2 has two C-terminal BEN domains, marked by α-helical structure, and is conserved in a range of metazoan and viral proteins. BEN domain is suggested to mediate protein–DNA and protein-protein interactions during chromatin organization and transcription (<a href="#bib1" class="usa-link" aria-describedby="bib1">Abhiman et al., 2008</a>). So far, only a few BEN domain-containing proteins have been described. All of their functions are linked to transcriptional repression regardless of whether the proteins contain other characterized domains, including mammalian BANP/SMAR1, NAC1, BEND3 and RBB and <em>Drosophila</em> mod(mdg4), BEND1, and BEND5 (<a href="#bib10" class="usa-link" aria-describedby="bib10">Dai et al., 2015</a>; <a href="#bib9" class="usa-link" aria-describedby="bib9">Dai et al., 2013</a>; <a href="#bib15" class="usa-link" aria-describedby="bib15">Gerasimova et al., 1995</a>; <a href="#bib20" class="usa-link" aria-describedby="bib20">Kaul-Ghanekar et al., 2004</a>; <a href="#bib22" class="usa-link" aria-describedby="bib22">Korutla et al., 2007</a>; <a href="#bib21" class="usa-link" aria-describedby="bib21">Korutla et al., 2005</a>; <a href="#bib32" class="usa-link" aria-describedby="bib32">Nègre et al., 2010</a>; <a href="#bib36" class="usa-link" aria-describedby="bib36">Rampalli et al., 2005</a>; <a href="#bib42" class="usa-link" aria-describedby="bib42">Sathyan et al., 2011</a>; <a href="#bib50" class="usa-link" aria-describedby="bib50">Xuan et al., 2013</a>). BEND1, BEND5, and RBB are revealed to bind DNA through sequence-specific recognition (<a href="#bib10" class="usa-link" aria-describedby="bib10">Dai et al., 2015</a>; <a href="#bib9" class="usa-link" aria-describedby="bib9">Dai et al., 2013</a>; <a href="#bib50" class="usa-link" aria-describedby="bib50">Xuan et al., 2013</a>), and BEND3 specifically binds heterochromatin (<a href="#bib42" class="usa-link" aria-describedby="bib42">Sathyan et al., 2011</a>). Additionally, several human BEND2 fusion proteins have been identified in tumors, and the activation of BEND2 is suggested to promote oncogenic activity (<a href="#bib7" class="usa-link" aria-describedby="bib7">Burford et al., 2018</a>; <a href="#bib43" class="usa-link" aria-describedby="bib43">Scarpa et al., 2017</a>; <a href="#bib46" class="usa-link" aria-describedby="bib46">Sturm et al., 2016</a>; <a href="#bib49" class="usa-link" aria-describedby="bib49">Williamson et al., 2019</a>). These data suggest that BEND2 might have a role in regulating chromatin and transcriptional gene expression. A recent study showed that BEND2 is a crucial regulator of meiosis gene expression and chromatin state during mouse spermatogenesis (<a href="#bib24" class="usa-link" aria-describedby="bib24">Ma et al., 2022</a>).</p>
<p>In the study of <a href="#bib24" class="usa-link" aria-describedby="bib24">Ma et al., 2022</a>, two BEND2-related proteins (p140 and p80) were found in mouse testis, using an antibody against a short polypeptide from the C-terminus of BEND2. They demonstrated that p140 is the full-length version of BEND2, displaying slower electrophoretic mobility due to its unusual sequence/structure at the N-terminus. They also reported that BEND2 was expressed in the nuclei of spermatogenic cells around meiosis initiation. Similarly, using our antibody against the whole C-terminus of the protein, we also found two BEND2-related proteins expressed in mouse testis and a highly consistent expression pattern of BEND2 during spermatogenesis. These two proteins likely correspond to full-length and smaller isoform from <a href="#bib24" class="usa-link" aria-describedby="bib24">Ma et al., 2022</a>.</p>
<p><a href="#bib24" class="usa-link" aria-describedby="bib24">Ma et al., 2022</a> disrupted <em>Bend2</em> by deleting exons 12 and 13, which resulted in the loss of both the full-length (p140) and the smaller isoform (p80) of BEND2. In their study, <em>Bend2</em> mutant mice were sterile and exhibited severe meiotic recombination and synapsis defects. In contrast, our study targeted exon 11, specifically ablating the full-length p140 isoform while preserving the expression of the smaller p80 isoform in the testis. Interestingly, <em>Bend2<sup>Δ11/y</sup></em> mutant males remain fertile, with essentially normal meiotic recombination and synapsis, suggesting that the full-length BEND2 (p140) is not essential for the repair of SPO11-induced double-strand breaks or for synapsis during meiosis.</p>
<p>This observation strongly implies that p80 is sufficient to fulfill the essential roles of BEND2 in meiosis and spermatogenesis. The preserved fertility and proper progression of meiosis in our <em>Bend2<sup>Δ11/y</sup></em> mutants suggest that the smaller p80 isoform is the primary driver of BEND2’s function in these processes. Conversely, the sterility and severe meiotic defects observed in the study by Ma et al. can be attributed to the loss of p80 and p140. These findings highlight a functional redundancy or compensatory mechanism provided by p80 that allows meiosis and spermatogenesis to proceed effectively in the absence of the full-length isoform.</p>
<p>Thus, we propose that p80 is the indispensable isoform of BEND2 required for meiotic progression. Its presence in our model likely prevents the spermatogenesis arrest seen in the Ma et al. study. This hypothesis provides a compelling explanation for why our <em>Bend2<sup>Δ11/y</sup></em> males exhibit a less severe phenotype and remain fertile despite the absence of full-length BEND2. These results underscore the importance of considering isoform-specific functions when investigating the roles of multi-isoform genes in complex biological processes like meiosis and gametogenesis.</p>
<p>In our study, one common defect in mutant spermatocytes and oocytes was a significant increase in unrepaired DSBs during late prophase. Examination of the chromosomal dynamics and meiotic recombination demonstrated that the full-length BEND2 is not essential for SPO11-induced DSB repair. Since all meiotic recombination markers studied, apart from gH2AX, were mostly indistinguishable in mutant and control cells, we hypothesized that the observed unrepaired DSBs might be SPO11-independent and further explored the LINE-1 retrotransposon activation, which is known to cause DSBs during meiosis (<a href="#bib45" class="usa-link" aria-describedby="bib45">Soper et al., 2008</a>; <a href="#bib8" class="usa-link" aria-describedby="bib8">Carofiglio et al., 2013</a>; <a href="#bib26" class="usa-link" aria-describedby="bib26">Malki et al., 2014</a>) and it has been implicated in the regulation of the ovarian reserve (<a href="#bib30" class="usa-link" aria-describedby="bib30">Martínez-Marchal et al., 2020</a>; <a href="#bib47" class="usa-link" aria-describedby="bib47">Tharp et al., 2020</a>). Unexpectedly, we found a significant reduction in LINE-1 expression levels in mutant testis and although it was not statistically significant, mutant oocytes seemed to have a similar trend.</p>
<p>Repetitive transposon elements, including LINE-1, occupy almost half of the human genome and are extremely important for genome evolution (<a href="#bib48" class="usa-link" aria-describedby="bib48">Wang, 2017</a>). Transposon-mediated insertional mutagenesis was linked to a multitude of genetic diseases. Even though LINE-1 is active mainly during the genome-wide demethylation, transient DNA demethylation occurring at the onset of meiosis was found to trigger a persisting LINE-1 expression until mid-pachytene (<a href="#bib45" class="usa-link" aria-describedby="bib45">Soper et al., 2008</a>). Silencing of LINE-1 retrotransposons is achieved with multiple mechanisms such as DNA methylation, histone modifications, and piRNAs (piwi-interacting RNAs) (<a href="#bib11" class="usa-link" aria-describedby="bib11">Di Giacomo et al., 2013</a>). <a href="#bib24" class="usa-link" aria-describedby="bib24">Ma et al., 2022</a> described that in <em>Bend2</em> KO male mice, there is an up-regulation of PIWIL2 expression at the onset of meiosis. PIWIL2 belongs to the PIWI subfamily of proteins that bind to piRNAs, which are required for several biological processes, including silencing retrotransposons during meiosis (<a href="#bib11" class="usa-link" aria-describedby="bib11">Di Giacomo et al., 2013</a>). Thus, it is likely that <em>Bend2</em> depletion caused an up-regulation of the piRNA pathway, resulting in lower LINE-1 expression in our mouse mutant spermatocytes.</p>
<p>Alternatively, as BEND2 was found to bind to repetitive sequences (<a href="#bib24" class="usa-link" aria-describedby="bib24">Ma et al., 2022</a>), it could interact directly with LINE-1 to promote its expression. The lower LINE-1 expression might result straightforwardly from less binding of BEND2. Finally, <em>Bend2</em> mutation could affect chromatin state and higher-order structures (<a href="#bib24" class="usa-link" aria-describedby="bib24">Ma et al., 2022</a>), which could alter LINE-1 expression and hinder DNA repair. Further studies focusing on the regulation of PIWIL2 by BEND2 will clarify these aspects of the phenotype.</p>
<p>Due to the sterility of hemizygous mutant males, female homozygous mutants can't be produced in the study of <a href="#bib24" class="usa-link" aria-describedby="bib24">Ma et al., 2022</a>. So, our female mutant model facilitates further exploration of BEND2’s roles in oogenesis.</p>
<p>Interestingly, our mutant females <em>Bend2<sup>D11/D11</sup></em> exhibited a more severe phenotype with a reduced ovarian reserve and subfertility, which had not been described before. Mouse females establish the pool of primordial follicles around birth. As folliculogenesis begins, the pool is gradually depleted throughout reproductive life (<a href="#bib19" class="usa-link" aria-describedby="bib19">Hunter, 2017</a>). By examining the dynamics of the oocyte pool in female mice at different ages, we showed that compared to wild-type females, BEND2-deficient females have a significantly smaller ovarian reserve. As a result, fewer follicles develop during folliculogenesis from pre-puberty throughout adulthood. Consistent with these, BEND2-deficient females produced reduced litter size, which indicates subfertility. Interestingly, our mutant oocyte quality analysis suggests that mature oocytes from mutant females are equally competent to develop into a blastocyst as control ones. These data suggest that the subfertility observed in <em>Bend2</em> mutants may be due to errors in later developmental stages, such as implantation or organogenesis.</p>
<p>Thus, BEND2, apart from its roles in male meiosis (<a href="#bib24" class="usa-link" aria-describedby="bib24">Ma et al., 2022</a>), is also required to establish the ovarian reserve during oogenesis. The full-length BEND2 may take on this role in females. However, future investigation of BEND2 expression in fetal ovaries is necessary to validate this. Genes discovered to have disease-causing roles in mouse models often offer a panel of strong candidate genes for screening human infertility factors (<a href="#bib18" class="usa-link" aria-describedby="bib18">Huang and Roig, 2023</a>; <a href="#bib37" class="usa-link" aria-describedby="bib37">Riera-Escamilla et al., 2019</a>). Our data shows that the depletion of BEND2 may lead to premature ovarian insufficiency (POI), which is also a significant cause of female infertility in humans, and its underlying genetic causes are mainly unknown (<a href="#bib40" class="usa-link" aria-describedby="bib40">Rossetti et al., 2017</a>). Thus, identifying BEND2’s requirement for normal oogenesis is also important to recognizing the genetic determinants of human POI.</p></section><section id="s4"><h2 class="pmc_sec_title">Materials and methods</h2>
<section class="tw xbox font-sm" id="keyresource"><h3 class="obj_head">Key resources table.</h3>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" valign="top" rowspan="1" colspan="1">Reagent type (species) or resource</th>
<th align="left" valign="top" rowspan="1" colspan="1">Designation</th>
<th align="left" valign="top" rowspan="1" colspan="1">Source or reference</th>
<th align="left" valign="top" rowspan="1" colspan="1">Identifiers</th>
<th align="left" valign="top" rowspan="1" colspan="1">Additional information</th>
</tr></thead>
<tbody>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Gene (<em>M. musculus</em>)</td>
<td align="left" valign="top" rowspan="1" colspan="1">Bend2</td>
<td align="left" valign="top" rowspan="1" colspan="1">Ensembl</td>
<td align="left" valign="top" rowspan="1" colspan="1">ENSMUSG00000108981</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<em>Gm15262</em>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Genetic reagent (<em>M. musculus</em>)</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<em>Bend2<sup>D11/D11</sup> or Bend2<sup>D11/y</sup></em>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1">Available from the authors upon request<br>Dr. Ignasi Roig<br>Ignasi.Roig@uab.cat</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sequence-based reagent</td>
<td align="left" valign="top" rowspan="1" colspan="1">gRNA 1-antisense</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper<br>(Sigma)</td>
<td align="left" valign="top" rowspan="1" colspan="1">gRNA</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<em>AGTAGCAGGCTGCATAAGT</em> GGG</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sequence-based reagent</td>
<td align="left" valign="top" rowspan="1" colspan="1">gRNA 2-sense</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper<br>(Sigma)</td>
<td align="left" valign="top" rowspan="1" colspan="1">gRNA</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<em>AGACCAGCCTTATTGACCA</em> TGG</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Strain, strain background (<em>Escherichia coli</em>)</td>
<td align="left" valign="top" rowspan="1" colspan="1">5-alpha competent <em>E. coli</em> cells</td>
<td align="left" valign="top" rowspan="1" colspan="1">New England Biolabs.</td>
<td align="left" valign="top" rowspan="1" colspan="1">C2987H</td>
<td align="left" valign="top" rowspan="1" colspan="1">competent cells</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Strain, strain background (<em>Escherichia coli</em>)</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<em>E. coli</em> BL21(DE3)</td>
<td align="left" valign="top" rowspan="1" colspan="1">NCBI</td>
<td align="left" valign="top" rowspan="1" colspan="1">469008</td>
<td align="left" valign="top" rowspan="1" colspan="1">competent cells</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Cell line (<em>H. sapiens</em>)</td>
<td align="left" valign="top" rowspan="1" colspan="1">HEK 293T</td>
<td align="left" valign="top" rowspan="1" colspan="1">Cellosaurus</td>
<td align="left" valign="top" rowspan="1" colspan="1">CVCL_0063</td>
<td align="left" valign="top" rowspan="1" colspan="1">Cell line maintained in SCAC, Institut de Biotecnologia i de Biomedicina</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Antibody</td>
<td align="left" valign="top" rowspan="1" colspan="1">anti-BEND2 Rb</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1">IF(1:100), WB (3.2 μg/ml)<br>Available from the authors upon request<br>Dr. Ignasi Roig<br>Ignasi.Roig@uab.cat</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Antibody</td>
<td align="left" valign="top" rowspan="1" colspan="1">anti-Ku70 Rb</td>
<td align="left" valign="top" rowspan="1" colspan="1">Abcam</td>
<td align="left" valign="top" rowspan="1" colspan="1">ab92450</td>
<td align="left" valign="top" rowspan="1" colspan="1">IF (1:100)<br>WB (1:2000)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Antibody</td>
<td align="left" valign="top" rowspan="1" colspan="1">anti-GAPDH Rb</td>
<td align="left" valign="top" rowspan="1" colspan="1">Abcam</td>
<td align="left" valign="top" rowspan="1" colspan="1">ab37168</td>
<td align="left" valign="top" rowspan="1" colspan="1">WB (1:2000)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Antibody</td>
<td align="left" valign="top" rowspan="1" colspan="1">anti- SYCP3 Ms or Rb</td>
<td align="left" valign="top" rowspan="1" colspan="1">Abcam</td>
<td align="left" valign="top" rowspan="1" colspan="1">ab97672 or ab15093</td>
<td align="left" valign="top" rowspan="1" colspan="1">IF (1:200)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Antibody</td>
<td align="left" valign="top" rowspan="1" colspan="1">anti- SYCP1 Rb</td>
<td align="left" valign="top" rowspan="1" colspan="1">Abcam</td>
<td align="left" valign="top" rowspan="1" colspan="1">ab15090</td>
<td align="left" valign="top" rowspan="1" colspan="1">IF (1:200)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Antibody</td>
<td align="left" valign="top" rowspan="1" colspan="1">anti-phospho-Histone H2A.X Ms</td>
<td align="left" valign="top" rowspan="1" colspan="1">Millipore</td>
<td align="left" valign="top" rowspan="1" colspan="1">05–636</td>
<td align="left" valign="top" rowspan="1" colspan="1">IF (1:400)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Antibody</td>
<td align="left" valign="top" rowspan="1" colspan="1">anti-MLH1 Ms</td>
<td align="left" valign="top" rowspan="1" colspan="1">BD Biosciences</td>
<td align="left" valign="top" rowspan="1" colspan="1">51-1327GR</td>
<td align="left" valign="top" rowspan="1" colspan="1">IF (1:50)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Antibody</td>
<td align="left" valign="top" rowspan="1" colspan="1">RPA32 (4E4) Rat</td>
<td align="left" valign="top" rowspan="1" colspan="1">Cell Signalling</td>
<td align="left" valign="top" rowspan="1" colspan="1">2208 S</td>
<td align="left" valign="top" rowspan="1" colspan="1">IF (1:100)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Antibody</td>
<td align="left" valign="top" rowspan="1" colspan="1">anti-RAD51 (ab-1) Rb</td>
<td align="left" valign="top" rowspan="1" colspan="1">Millipore</td>
<td align="left" valign="top" rowspan="1" colspan="1">PC130</td>
<td align="left" valign="top" rowspan="1" colspan="1">IF (1:100)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Antibody</td>
<td align="left" valign="top" rowspan="1" colspan="1">anti-GFP Rb</td>
<td align="left" valign="top" rowspan="1" colspan="1">Thermo Fisher Scientific</td>
<td align="left" valign="top" rowspan="1" colspan="1">A-11122</td>
<td align="left" valign="top" rowspan="1" colspan="1">IF (1:200)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Antibody</td>
<td align="left" valign="top" rowspan="1" colspan="1">anti-LINE1 ORF1p Rb</td>
<td align="left" valign="top" rowspan="1" colspan="1">Abcam</td>
<td align="left" valign="top" rowspan="1" colspan="1">ab216324</td>
<td align="left" valign="top" rowspan="1" colspan="1">IF (1:100)<br>WB (1:2000)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Antibody</td>
<td align="left" valign="top" rowspan="1" colspan="1">anti-rabbit HPR</td>
<td align="left" valign="top" rowspan="1" colspan="1">Bio-Rad</td>
<td align="left" valign="top" rowspan="1" colspan="1">170–6515</td>
<td align="left" valign="top" rowspan="1" colspan="1">WB (1:5000)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Recombinant DNA reagent</td>
<td align="left" valign="top" rowspan="1" colspan="1">pEGFP-C1</td>
<td align="left" valign="top" rowspan="1" colspan="1">Clontech</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog:6084–1</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Recombinant DNA reagent</td>
<td align="left" valign="top" rowspan="1" colspan="1">pEGFP-N1</td>
<td align="left" valign="top" rowspan="1" colspan="1">Clontech</td>
<td align="left" valign="top" rowspan="1" colspan="1">Catalog:6085–1</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Recombinant DNA reagent</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1">Available from the authors upon request<br>Dr. Ignasi Roig<br>Ignasi.Roig@uab.cat</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Recombinant DNA reagent</td>
<td align="left" valign="top" rowspan="1" colspan="1">pEGFP-N1-BEND2</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1">Available from the authors upon request<br>Dr. Ignasi Roig<br>Ignasi.Roig@uab.cat</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Recombinant DNA reagent</td>
<td align="left" valign="top" rowspan="1" colspan="1">pET-28a (+)-TEV</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1">Dr. Neus Ferrer Miralles<br>Institut de Biotecnologia i de Biomedicina,</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sequence-based reagent</td>
<td align="left" valign="top" rowspan="1" colspan="1">forward</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper</td>
<td align="left" valign="top" rowspan="1" colspan="1">PCR primers</td>
<td align="left" valign="top" rowspan="1" colspan="1">BEND2 genotyping;<br><em>TTGCCAGTGGGGTATTACGA</em>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sequence-based reagent</td>
<td align="left" valign="top" rowspan="1" colspan="1">reverse</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper</td>
<td align="left" valign="top" rowspan="1" colspan="1">PCR primers</td>
<td align="left" valign="top" rowspan="1" colspan="1">BEND2 genotyping;<br><em>CAGGGCATTTGCACCCCATGCC</em>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sequence-based reagent</td>
<td align="left" valign="top" rowspan="1" colspan="1">forward</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper</td>
<td align="left" valign="top" rowspan="1" colspan="1">PCR primers</td>
<td align="left" valign="top" rowspan="1" colspan="1">BEND2 female genotyping;<br><em>TTTGCTCCACTGTTTCACGC</em>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sequence-based reagent</td>
<td align="left" valign="top" rowspan="1" colspan="1">reverse</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper</td>
<td align="left" valign="top" rowspan="1" colspan="1">PCR primers</td>
<td align="left" valign="top" rowspan="1" colspan="1">BEND2 female genotyping;<br><em>TCCCTTTAAACTGCCAACAACA</em>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sequence-based reagent</td>
<td align="left" valign="top" rowspan="1" colspan="1">255P1 forward</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper</td>
<td align="left" valign="top" rowspan="1" colspan="1">PCR primers</td>
<td align="left" valign="top" rowspan="1" colspan="1">255P1 Gene specific primers for RT-PCR;<br><em>TAGGGACCAAGAACCTGCTG</em>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sequence-based reagent</td>
<td align="left" valign="top" rowspan="1" colspan="1">reverse</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper</td>
<td align="left" valign="top" rowspan="1" colspan="1">PCR primers</td>
<td align="left" valign="top" rowspan="1" colspan="1">255P1 Gene specific primers for RT-PCR; <em>TCCTGAAGCCACTGAGAAGG</em>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sequence-based reagent</td>
<td align="left" valign="top" rowspan="1" colspan="1">forward</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper</td>
<td align="left" valign="top" rowspan="1" colspan="1">PCR primers</td>
<td align="left" valign="top" rowspan="1" colspan="1">β-actin for RT-PCR;<br><em>AGGTCTTTACGGATGTCAACG</em>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sequence-based reagent</td>
<td align="left" valign="top" rowspan="1" colspan="1">reverse</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper</td>
<td align="left" valign="top" rowspan="1" colspan="1">PCR primers</td>
<td align="left" valign="top" rowspan="1" colspan="1">β-actin primers for RT-PCR; <em>ATCTACGAGGGCTATGCTCTC</em>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sequence-based reagent</td>
<td align="left" valign="top" rowspan="1" colspan="1">forward</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper</td>
<td align="left" valign="top" rowspan="1" colspan="1">PCR primers</td>
<td align="left" valign="top" rowspan="1" colspan="1">BEND2 cloning primer; agaaATGCCAGGAAAAACTGAAG</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Sequence-based reagent</td>
<td align="left" valign="top" rowspan="1" colspan="1">reverse</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper</td>
<td align="left" valign="top" rowspan="1" colspan="1">PCR primers</td>
<td align="left" valign="top" rowspan="1" colspan="1">BEND2 cloning primer; <em>TTAAGCTATTGCATTCCTTGGG</em> for pEGFP-C1 vector;gAGCTATTGCATTCCTTGGGC for pEGFP-N1 vectors</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Peptide, recombinant protein</td>
<td align="left" valign="top" rowspan="1" colspan="1">His-tagged BEND2 protein</td>
<td align="left" valign="top" rowspan="1" colspan="1">This paper</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1">Available from the authors upon request<br>Dr. Ignasi Roig<br>Ignasi.Roig@uab.cat</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Commercial assay or kit</td>
<td align="left" valign="top" rowspan="1" colspan="1">jetPEI DNA transfection reagent</td>
<td align="left" valign="top" rowspan="1" colspan="1">Polyplus Transfection</td>
<td align="left" valign="top" rowspan="1" colspan="1">101–40 N</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Commercial assay or kit</td>
<td align="left" valign="top" rowspan="1" colspan="1">HisTrap HP column</td>
<td align="left" valign="top" rowspan="1" colspan="1">Cytiva</td>
<td align="left" valign="top" rowspan="1" colspan="1">29051021 or 17524801</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Commercial assay or kit</td>
<td align="left" valign="top" rowspan="1" colspan="1">TUNEL reaction mixture</td>
<td align="left" valign="top" rowspan="1" colspan="1">Roche Diagnostics</td>
<td align="left" valign="top" rowspan="1" colspan="1">11684795910</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="#fig3" class="usa-link">Figure 3</a>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Software, algorithm</td>
<td align="left" valign="top" rowspan="1" colspan="1">Image J</td>
<td align="left" valign="top" rowspan="1" colspan="1">Image J</td>
<td align="left" valign="top" rowspan="1" colspan="1">RRID:<a href="https://identifiers.org/RRID:SCR_003070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">SCR_003070</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Software, algorithm</td>
<td align="left" valign="top" rowspan="1" colspan="1">GraphPad Prism 8</td>
<td align="left" valign="top" rowspan="1" colspan="1">GraphPad</td>
<td align="left" valign="top" rowspan="1" colspan="1">RRID:<a href="https://identifiers.org/RRID:SCR_002798" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">SCR_002798</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Software, algorithm</td>
<td align="left" valign="top" rowspan="1" colspan="1">Adobe Photoshop</td>
<td align="left" valign="top" rowspan="1" colspan="1">Adobe</td>
<td align="left" valign="top" rowspan="1" colspan="1">RRID:<a href="https://identifiers.org/RRID:SCR_014199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">SCR_014199</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Other</td>
<td align="left" valign="top" rowspan="1" colspan="1">DAPI stain</td>
<td align="left" valign="top" rowspan="1" colspan="1">Invitrogen</td>
<td align="left" valign="top" rowspan="1" colspan="1">D1306</td>
<td align="left" valign="top" rowspan="1" colspan="1">0.1 µg/ml</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Other</td>
<td align="left" valign="top" rowspan="1" colspan="1">Vectashield antifade mounting medium</td>
<td align="left" valign="top" rowspan="1" colspan="1">Vector Laboratories</td>
<td align="left" valign="top" rowspan="1" colspan="1">H1000</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/keyresource/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section id="s4-1"><h3 class="pmc_sec_title">Mice</h3>
<p><em>Bend2</em> mutant mice were generated by using the CRISPR/Cas9 system. A pair of gRNAs with minimum off-target and maximum on-target activity were designed and selected to specifically target sequences that encode essential protein domains of the <em>Bend2</em> gene using CRISPR DESIGN TOOLS (Millipore Sigma) (gRNA1-antisense: <em>AGTAGCAGGCTGCATAAGT</em> GGG; gRNA2-sense: <em>AGACCAGCCTTATTGACCA</em> TGG). SgRNAs were synthesized by Sigma and microinjected with Cas9 protein into the pronucleus of C57BL/6JOlaHsd zygotes. Edited founders were identified by PCR with primers flanking the targeted region and HincII digest. PCR products were further purified and determined by Sanger sequencing. Five out of 17 F0 mice were identified to carry desired mutations, including two homozygous males and three heterozygous females. Each of them was crossed with wild-type C57BL/6JOlaHsd to eliminate possible off-target mutations to generate pure heterozygotes. Female heterozygotes (<em>Bend2<sup>+/D11</sup></em>) were crossed with wild-type males (<em>Bend2<sup>+/y</sup></em>) to obtain wild-type females (<em>Bend2<sup>+/+</sup></em>), heterozygous females (<em>Bend2<sup>+/D11</sup></em>), wild-type male (<em>Bend2<sup>+/y</sup></em>), and mutant males (<em>Bend2<sup>D11/y</sup></em>) offsprings. Female heterozygotes (<em>Bend2<sup>+/D11</sup></em>) were crossed with mutant males (<em>Bend2<sup>D11/y</sup></em>) to obtain mutant females (<em>Bend2<sup>D11/D11</sup></em>), heterozygous females (<em>Bend2<sup>+/D11</sup></em>), wild-type males (<em>Bend2<sup>+/y</sup></em>) and mutant males (<em>Bend2<sup>+/y</sup></em>). Mice from at least F2 generation were analyzed for their phenotyping. All experiments used at least three animals from each genotype (unless mentioned in the text). Mutant males were compared with their wild-type littermates. Testes from 2 to 6-months-old mice were collected and processed for adult mice. Female mutants were compared to wild-type mice from other litters of the same age and from animals of closely related parents.</p>
<p>For genotyping, genomic DNA was extracted from mouse tails by overnight incubation at 56℃ in lysis buffer (0.1 M Tris-HCl pH 8.5–9, 0.2 M NaCl, 0.2% SDS, 5 mM EDTA, and 0.4 mg/ml proteinase K), followed by precipitation with isopropanol and washes in cold 70% ethanol and subjected to PCR using NZYTaq II 2 x Green Master Mix employing primer pair (forward: <em>TTGCCAGTGGGGTATTACGA</em>, and reverse: <em>CTGGAAGGCAGGAAGTTTAACA</em>). For female genotyping, the identified possible homozygous BEND2 females were subjected to an extra PCR using the primer pair (forward: <em>TTTGCTCCACTGTTTCACGC</em>, and reverse: <em>TCCCTTTAAACTGCCAACAACA</em>) to confirm the homozygosity.</p>
<p>The experiments performed in this study complied with EU regulations and were approved by the Ethics Committee of the UAB and the Catalan Government (5322-CEEA-UAB).</p></section><section id="s4-2"><h3 class="pmc_sec_title">Ovarian stimulation and in vitro fertilization</h3>
<p>Intra-peritoneal injection of 15 IU pregnant mare’s serum gonadotrophin per female was administered to three <em>Bend2<sup>+/D11</sup></em> and three <em>Bend2<sup>D11/D11</sup></em> 7-month-old mice. After 49.5 hr, 10 IU human chorionic gonadotrophin (hCG) per female was injected and 13.5 hr later mice were sacrificed. Oviducts of treated females were dissected under stereo-microscope and cumulus masses were released into a drop of fertilization medium (0.25 mM GSH in HTF medium). Oocytes released from each female mice were counted. Time of ovulation was counted as 0.5 dpc.</p>
<p>For in vitro fertilization, fresh sperm from a proven fertile, C57BL/6 N wild-type male of 4 months of age were used and poured into a dish containing oocytes in fertilization medium. After 3 hr of fertilization, oocytes were washed and O/N cultured in HTF medium. Assessment of fertilized, unfertilized oocytes was done under stereo-microscope. From 2 cell-stage until blastocysts, embryos were cultured in KSOM medium.</p></section><section id="s4-3"><h3 class="pmc_sec_title">Total RNA purification and RT-PCR</h3>
<p>Total RNA was purified from mouse tissues using the RNeasy Plus Mini Kit (Qiagen) and then transcribed into cDNA using the iScript cDNA Synthesis Kit (Bio-Rad), following the manufacturer’s instructions. cDNA was amplified using NZYTaq II 2 x Green Master Mix with gene-specific primers (255P1 forward: <em>TAGGGACCAAGAACCTGCTG</em>, and reverse: <em>TCCTGAAGCCACTGAGAAGG</em>) and β-actin primers (forward: <em>AGGTCTTTACGGATGTCAACG</em>, and reverse: <em>ATCTACGAGGGCTATGCTCTC</em>) as control. A minus Reverse Transcription control (RT-) containing all the reaction components except the reverse transcriptase was included for testing for contaminating DNA.</p></section><section id="s4-4"><h3 class="pmc_sec_title">Gene cloning and transfection</h3>
<p>cDNA from wild-type mouse testis was subject to PCR using Phusion High-Fidelity DNA Polymerase (Thermo Fisher Scientific) with a pair of specific primers (forward: <em>agaaATGCCAGGAAAAACTGAAG</em>, and reverse: <em>TTAAGCTATTGCATTCCTTGGG</em> for pEGFP-C1 vector and <em>gAGCTATTGCATTCCTTGGGC</em> for pEGFP-N1 vectors) targeting at both ends of the coding sequence to amplify the full-length of <em>255p1 (Bend2</em> novel splice variant). DNA purified from PCR reaction mix was phosphorylated and inserted into dephosphorylated pEGFP-C1 and pEGFP-N1 vectors. Plasmids were transformed to 5-alpha competent <em>E. coli</em> cells (NEB) and identified by colony PCR followed by Sanger sequencing.</p>
<p>pEGFP-C1-BEND2 or pEGFP-N1-BEND2 were transfected into the human embryonic kidney cell line (HEK 293T, obtained from the Cell Culture, Antibody Production and Cytometry Service (SCAC) from UAB, its identity was confirmed by STR profiling and were mycoplasma negative) by using jetPEI DNA transfection reagent (Polyplus Transfection) and into 16–18 dpp wild-type live mouse testis by electroporation (<a href="#bib44" class="usa-link" aria-describedby="bib44">Shibuya et al., 2014</a>). Mice were sacrificed 24–72 hr post-electroporation and used for experiments.</p></section><section id="s4-5"><h3 class="pmc_sec_title">Antibody generation</h3>
<p>To generate <em>in-house</em> BEND2 polyclonal antibody, the full-length of 255p1 (Bend2 novel splice variant) was amplified from pEGFP-C1-BEND2 and cloned into modified bacterial expression vector pET-28a (+)-TEV. The recombinant His-tagged BEND2 protein was expressed in <em>E. coli</em> BL21(DE3) competent cells by IPTG induction and then purified by affinity chromatography using HisTrap HP column (Cytiva). Purified His-tagged BEND2 protein was treated with TEV protease, followed by another affinity purification, to remove the His-tag. His-tag removed BEND2 protein was used to immunize rabbits. After immunization, the immune response was checked by ELISA. Antibodies were purified from rabbit serum by affinity chromatography using Affi-Prep protein A resin cartridge (Bio-Rad).</p></section><section id="s4-6"><h3 class="pmc_sec_title">Western blot</h3>
<p>Total protein was extracted from mouse testis using RIPA lysis buffer (50 mM Tris pH 8, 1% Triton X-100, 0.1% SD, 150 mM NaCl, 1 mM EDTA, 0.5% Sodium Deoxycholate, 10 mM NaF, 1x Protease Inhibitor). Testis tissue was disrupted and homogenized thoroughly with a pestle in RIPA lysis buffer, followed by 10 min incubation at 95℃. Each lysate sample’s protein concentration was determined using Pierce BCA Protein Assay Kit (Thermo Scientific). 50 μg of total protein was loaded per well, and samples were separated by TGX-PAGE gel electrophoresis in Tris-Glycine-SDS buffer. Proteins were transferred to PVDF membranes (Bio-Rad) by wet electroblotting in Tris/glycine buffer. Membranes were blocked in 5% non-fat milk in PBS for 2 h at room temperature, followed by incubation of primary antibody diluted in blocking buffer overnight at 4 °C. The next day, after three washes of PBST (PBS containing 0.05% Tween-20), membranes were incubated in anti-rabbit HPR conjugate antibody (1:5000, 170–6515, Bio-Rad) in PBS for 1 hr at room temperature. ECL substrate (Bio-Rad) was used for chemiluminescent detection. Imaging was performed on ChemiDoc Touch Imaging system (Bio-Rad), and images were analyzed by Imagelab software (Bio-Rad). Primary antibodies used in WB: anti-BEND2 Rb (<em>in-house</em>, 3.2 μg/ml), anti-Ku70 Rb (abcam, 1:2000), anti-LINE1 ORF1p Rb (abcam, 1:2000), and anti-GAPDH Rb (abcam, 1:2000).</p></section><section id="s4-7"><h3 class="pmc_sec_title">Nuclei spreading and immunofluorescence</h3>
<p>Spermatocyte nuclei spreading was prepared using frozen testes. Protocol was adapted from <a href="#bib23" class="usa-link" aria-describedby="bib23">Liebe et al., 2004</a>: briefly, a small portion of testis was cut and minced thoroughly by a sterile blade in cold PBS (pH 7.4) containing 1x protease inhibitor, PI (Roche Diagnostics) on a petri dish; cell mixture was transferred to a sterile Eppendorf and sat for 15 min to sediment; 25 µl of supernatant cell suspension was spread onto a glass slide and incubated with 80 µl of 1% Lipsol containing 1x PI, for 15–20 min, followed by fixation in 150 µl of the PFA fixative solution (1% Paraformaldehyde pH 9.2–9.4, 15% Triton X-100, 1 x PI) for 2 hr at room temperature in a closed humid chamber. Oocyte nuclei spreading was prepared from fresh fetal and perinatal ovaries. Briefly, one pair of ovaries were dissected from each female under a stereomicroscope (Nikon SMZ-1); ovaries were first incubated in 500 µl of M2 medium (Sigma-Aldrich) containing 2.5 mg/ml collagenase (Sigma-Aldrich) for 30 min at 37℃, and then incubated in 500 µl of hypotonic buffer (30 mM Tris-HCl pH 8.2, 50 mM Sucrose, 17 mM Sodium Citrate, 5 mM EDTA, 0.5 mM DTT, 1 x PI) for 30 min at room temperature; finally, single-cell suspension was prepared by disaggregating ovaries in 100 mM sucrose by pipetting under the stereo microscope. Each 10 µl of the cell suspension was spread onto a glass slide, followed by fixation in 40 µl of the PFA fixative solution (1% PFA, 5 mM Sodium Borate, 0.15% Triton X-100, 3 mM DTT, 1 x PI, pH 9.2) for 2 hr at room temperature in a closed humid chamber. Slides were dried under a fume hood and then washed in 0.4% Photoflo (Kodak) solution four times.</p>
<p>For immunofluorescence staining, slides of spermatocyte or oocyte nuclei spreading were blocked in freshly made blocking solution (0.2% BSA, 0.2% gelatin, 0.05% Tween-20 in PBS) at room temperature, followed by incubation of primary antibody diluted in blocking solution in a humid chamber overnight at 4℃; the next day, slides were washed four times in blocking solution, and incubated in secondary antibody diluted in blocking solution in a humid chamber for 1 hr at 37℃, followed by another four washes in blocking solutions. Drained slides were mounted with 0.1 µg/ml DAPI in Vectashield antifade mounting medium and analyzed with an epifluorescence microscope (Zeiss Axioskop). Primary antibodies used for IF: anti-SYCP3 Ms or Rb (abcam, 1:200), anti-SYCP1 Rb (abcam, 1:200), anti-phospho-Histone H2A.X Ms (Millipore, 1:400), anti-MLH1 Ms (BD Biosciences, 1:50), RPA32 (4E4) Rat (cell signaling, 1:100), anti-RAD51 (ab-1) Rb (Millipore, 1:100), and anti-GFP Rb (Thermo Fisher Scientific, 1:200).</p></section><section id="s4-8"><h3 class="pmc_sec_title">PAS-Hematoxylin, immunohistochemical, and TUNEL staining</h3>
<p>Fresh mouse tissues were fixed overnight at 4℃ with freshly made 4% PFA (0.4 g paraformaldehyde dissolved in 10 ml PBS, pH 7.4) for immunohistochemical (IHC)/TUNEL assay or with Bouin’s fixative for the use of PAS (Periodic Acid Schiff) staining. After fixation, tissues were washed in PBS, dehydrated in a series of ethanol with increasing concentration, cleared by histoclear and infiltrated by paraffin in sequence; embedded tissues were cut into thin slices (6–7 µm for testes; 4 µm for ovaries) using a microtome, which were mounted on poly-L-lysine coated slides and dry overnight at 37℃. Before any staining procedures, slides were deparaffinized in xylene and a series of ethanol with decreasing concentration in sequence and rehydrated in distilled water.</p>
<p>For IHC, an antigen retrieval step was performed to expose the epitopes masked by PFA fixation: incubating the slides in sodium citrate buffer (10 mM Sodium citrate, 0.05% Tween 20, pH 6.0) or Tris-EDTA buffer (10 mM Tris, 1 mM EDTA, 0.05% Tween 20, pH 9.0) for 30 min at 95–100° C. Subsequently, an immunofluorescence was performed as described above and the used primary antibodies were anti-DDX4 (Abcam, 1:100) and anti-LINE-1 (Abcam, 1:100).</p>
<p>For TUNEL assay, tissue section samples were permeabilized in 0.5% Triton X-100 in PBS for 15 min followed by two washes in PBS, incubated in background reducing solution (Dako), followed by a rinse in PBS, incubated in TUNEL reaction mixture (10% TdT enzyme solution, Roche Diagnostics) for 1 hr at 37℃ in a humid chamber, followed by three washes in PBS. Slides were mounted with 15 µl DAPI (0.1 µg/ml in Vectashield antifade mounting medium) and analyzed with an epifluorescence microscope (Zeiss Axioskop).</p>
<p>For PAS-Hematoxylin (PAS-H) staining, tissue sections were oxidized by 1% Periodic Acid solution for 10 min, followed by two washes in distilled water, stained by Schiff’s reagent for 30 min in darkness, followed by two washes in sulfurous water (10% Potassium metabisulfite, 0.1 M HCl in Milli-Q water) and two washes in distilled water; counterstained in Mayer’s Hematoxylin for 1 min and rinsed in running tap water; dehydrated in a series of ethanal with increasing concentrations and xylene in sequence; mounted with DPX mounting medium and analyzed with an optical microscope. Images were captured using a Zeiss Axioskop microscope.</p></section><section id="s4-9"><h3 class="pmc_sec_title">Follicle count and classification</h3>
<p>The whole ovaries were sectioned for follicle quantification in newborn, young, adolescent, and adult mice. Eight consecutive sections were mounted on each slide. Every third section was counted per slide to avoid counting follicles more than once. Five alternate slides per animal were counted (both ovaries). Follicles were only counted if the nucleus of the oocyte was visible. They were classified as primordial if they contained one single layer of plane squamous granulosa cells or as primary if they displayed one layer of cuboidal granulosa cells. Furthermore, follicles containing both plane and cuboidal cells were classified based on the predominant granulosa cell morphology. Follicles were classified as secondary if they showed more than one layer of cuboidal granulosa cells with no visible antrum. All follicles displaying an antral cavity of any size were classified as antral. Counting and classification were performed under a bright-field Zeiss Axioskop microscope.</p></section><section id="s4-10"><h3 class="pmc_sec_title">Image processing, analysis, and statistical analysis</h3>
<p>All the images were processed by Adobe Photoshop. Fluorescence intensity was quantified by ImageJ; fluorescence signals were counted manually or by ImageJ. Data analysis and statistical inference were performed using GraphPad Prism 8 software. The sample size was determined based on our previous experience analyzing other mutant phenotypes and the variability observed within each type of analysis (<a href="#bib29" class="usa-link" aria-describedby="bib29">Marcet-Ortega et al., 2017</a>,<a href="#bib30" class="usa-link" aria-describedby="bib30">Martínez-Marchal et al., 2020</a>; <a href="#bib34" class="usa-link" aria-describedby="bib34">Pacheco et al., 2018</a>; <a href="#bib39" class="usa-link" aria-describedby="bib39">Roig et al., 2010</a>; <a href="#bib41" class="usa-link" aria-describedby="bib41">Ruth et al., 2021</a>). Statistical methods and p-values are presented in each graph or in the main text or figure legends. Statistical significances were determined using t-tests for normally distributed or Mann-Whitney U-tests for non-normally distributed continuous data. One-way ANOVA test was used for analysis of homogeneity of variances. For all tests, statistical significance was considered for p&lt;0.05.</p></section></section><section id="ack" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was supported by Spanish Ministerio de Ciencia, Innovación e Universidades grants (I.R: PID 2019-107082RB-I00, PID2022-138905OB-I00; A.M.P.: PID2020–120326RB-I00), Junta de Castilla y León grants (A.M.P.: CSI017P23 and CSI017P23), and a grant from La Fundació Marató de TV3 (I.R.: 677 /U/2021). Yan Huang is a recipient of a fellowship from the China Scholarship Council (201607040048). Cristina Madrid-Sandín is the recipient of an FPU fellowship from the Spanish Ministerio de Ciencia, Innovación e Universidades (FPU19/02885). Nikoleta Nikou and Carolina Buza-Tarna are supported by an FPI fellowship from the Spanish Ministerio de Ciencia, Innovación e Universidades (PRE2020-094355 and PREP2022-000758, respectively). Maria López-Panadés is supported by a FI-SDUR fellowship from the Catalan Government (2023 FISDU 00058). Moreover, we would like to acknowledge the rest of the members of the Roig Lab for their support, comments, and discussions about the project.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.</p></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Ignasi Roig, Email: ignasi.roig@uab.cat.</p>
<p>Carmen J Williams, National Institute of Environmental Health Sciences, United States.</p>
<p>Wei Yan, Washington State University, United States.</p></section><section id="sec20"><h2 class="pmc_sec_title">Funding Information</h2>
<p>This paper was supported by the following grants:</p>
<ul class="list" style="list-style-type:disc">
<li><p>
Ministerio de Ciencia, Innovación y Universidades

PID2019-107082RB-I00 to Ignasi Roig.</p></li>
<li><p>
Ministerio de Ciencia, Innovación y Universidades

PID2022-138905OB-I00 to Ignasi Roig.</p></li>
<li><p>
Ministerio de Ciencia, Innovación y Universidades

PID2020-120326RB-I00 to Alberto M Pendás.</p></li>
<li><p>
Fundació la Marató de TV3

677/U/2021 to Ignasi Roig.</p></li>
<li><p>
China Scholarship Council

201607040048 to Yan Huang.</p></li>
<li><p>
Ministerio de Ciencia, Innovación y Universidades

FPU19/02885 to Cristina Madrid-Sandín.</p></li>
<li><p>
Ministerio de Ciencia, Innovación y Universidades

PRE2020-094355 to Nikoleta Nikou.</p></li>
<li><p>
Ministerio de Ciencia, Innovación y Universidades

PREP2022-000758 to Carolina Buza.</p></li>
<li><p>
Agència de Gestió d'Ajuts Universitaris i de Recerca

2023 FISDU 00058 to Maria López-Panadés.</p></li>
<li><p>
Junta de Castilla y León

CSI017P23 to Alberto M Pendás.</p></li>
</ul></section><section id="s5"><h2 class="pmc_sec_title">Additional information</h2>
<section id="fn-group1" class="fn-group"><h3 class="pmc_sec_title">Competing interests</h3>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="conf1"><p>No competing interests declared.</p></div></div></section><section id="fn-group2" class="fn-group"><h3 class="pmc_sec_title">Author contributions</h3>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="con1"><p>Conceptualization, Data curation, Formal analysis, Investigation, Visualization, Methodology, Writing – original draft, Writing – review and editing.</p></div>
<div class="fn p" id="con2"><p>Data curation, Formal analysis, Investigation, Visualization, Methodology.</p></div>
<div class="fn p" id="con3"><p>Data curation, Formal analysis, Investigation, Visualization, Methodology.</p></div>
<div class="fn p" id="con4"><p>Data curation, Formal analysis, Investigation, Visualization, Methodology.</p></div>
<div class="fn p" id="con5"><p>Data curation, Formal analysis, Investigation, Visualization, Methodology.</p></div>
<div class="fn p" id="con6"><p>Data curation, Formal analysis, Investigation, Visualization, Methodology, Writing – review and editing.</p></div>
<div class="fn p" id="con7"><p>Data curation, Formal analysis, Investigation, Visualization, Methodology, Writing – review and editing.</p></div>
<div class="fn p" id="con8"><p>Data curation, Formal analysis, Investigation, Visualization, Methodology, Writing – review and editing.</p></div>
<div class="fn p" id="con9"><p>Data curation, Formal analysis, Investigation, Visualization, Methodology, Writing – review and editing.</p></div>
<div class="fn p" id="con10"><p>Conceptualization, Validation, Investigation, Visualization, Methodology, Writing – review and editing.</p></div>
<div class="fn p" id="con11"><p>Conceptualization, Formal analysis, Validation, Investigation, Visualization, Methodology, Writing – review and editing.</p></div>
<div class="fn p" id="con12"><p>Conceptualization, Formal analysis, Investigation, Visualization, Methodology, Writing – review and editing.</p></div>
<div class="fn p" id="con13"><p>Conceptualization, Supervision, Methodology, Writing – review and editing.</p></div>
<div class="fn p" id="con14"><p>Conceptualization, Data curation, Formal analysis, Supervision, Funding acquisition, Validation, Investigation, Methodology, Writing – original draft, Project administration, Writing – review and editing.</p></div>
</div></section><section id="fn-group3" class="fn-group"><h3 class="pmc_sec_title">Ethics</h3>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn16"><p>The experiments performed in this study complied with EU regulations and were approved by the Ethics Committee of the UAB and the Catalan Government (5322-CEEA-UAB).</p></div></div></section></section><section id="s6"><h2 class="pmc_sec_title">Additional files</h2>
<section class="sm xbox font-sm" id="supp1"><div class="caption p"><span>Supplementary file 1. Testis and body weight (TW/BW) ratio of male mice.</span></div>
<div class="media p" id="d67e2868"><div class="caption">
<a href="/articles/instance/12367297/bin/elife-96052-supp1.docx" data-ga-action="click_feat_suppl" class="usa-link">elife-96052-supp1.docx</a><sup> (16.3KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="mdar"><div class="caption p"><span>MDAR checklist</span></div>
<div class="media p" id="d67e2871"><div class="caption">
<a href="/articles/instance/12367297/bin/elife-96052-mdarchecklist1.docx" data-ga-action="click_feat_suppl" class="usa-link">elife-96052-mdarchecklist1.docx</a><sup> (61KB, docx) </sup>
</div></div></section></section><section id="s7"><h2 class="pmc_sec_title">Data availability</h2>
<p>All the data analyzed in this study are available within the manuscript and its supporting materials.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="bib1">
<cite>Abhiman S, Iyer LM, Aravind L. BEN: a novel domain in chromatin factors and DNA viral proteins. Bioinformatics. 2008;24:458–461. doi: 10.1093/bioinformatics/btn007.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btn007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2477736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18203771/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=BEN:%20a%20novel%20domain%20in%20chromatin%20factors%20and%20DNA%20viral%20proteins&amp;author=S%20Abhiman&amp;author=LM%20Iyer&amp;author=L%20Aravind&amp;volume=24&amp;publication_year=2008&amp;pages=458-461&amp;pmid=18203771&amp;doi=10.1093/bioinformatics/btn007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib2">
<cite>Ahmed EA, Philippens MEP, Kal HB, de Rooij DG, de Boer P. Genetic probing of homologous recombination and non-homologous end joining during meiotic prophase in irradiated mouse spermatocytes. Mutation Research. 2010;688:12–18. doi: 10.1016/j.mrfmmm.2010.02.004.</cite> [<a href="https://doi.org/10.1016/j.mrfmmm.2010.02.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20167225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mutation%20Research&amp;title=Genetic%20probing%20of%20homologous%20recombination%20and%20non-homologous%20end%20joining%20during%20meiotic%20prophase%20in%20irradiated%20mouse%20spermatocytes&amp;author=EA%20Ahmed&amp;author=MEP%20Philippens&amp;author=HB%20Kal&amp;author=DG%20de%20Rooij&amp;author=P%20de%20Boer&amp;volume=688&amp;publication_year=2010&amp;pages=12-18&amp;pmid=20167225&amp;doi=10.1016/j.mrfmmm.2010.02.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib3">
<cite>Anderson LK, Reeves A, Webb LM, Ashley T. Distribution of crossing over on mouse synaptonemal complexes using immunofluorescent localization of MLH1 protein. Genetics. 1999;151:1569–1579. doi: 10.1093/genetics/151.4.1569.</cite> [<a href="https://doi.org/10.1093/genetics/151.4.1569" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1460565/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10101178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Distribution%20of%20crossing%20over%20on%20mouse%20synaptonemal%20complexes%20using%20immunofluorescent%20localization%20of%20MLH1%20protein&amp;author=LK%20Anderson&amp;author=A%20Reeves&amp;author=LM%20Webb&amp;author=T%20Ashley&amp;volume=151&amp;publication_year=1999&amp;pages=1569-1579&amp;pmid=10101178&amp;doi=10.1093/genetics/151.4.1569&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib4">
<cite>Barchi M, Mahadevaiah S, Di Giacomo M, Baudat F, de Rooij DG, Burgoyne PS, Jasin M, Keeney S. Surveillance of different recombination defects in mouse spermatocytes yields distinct responses despite elimination at an identical developmental stage. Molecular and Cellular Biology. 2005;25:7203–7215. doi: 10.1128/MCB.25.16.7203-7215.2005.</cite> [<a href="https://doi.org/10.1128/MCB.25.16.7203-7215.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1190256/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16055729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20and%20Cellular%20Biology&amp;title=Surveillance%20of%20different%20recombination%20defects%20in%20mouse%20spermatocytes%20yields%20distinct%20responses%20despite%20elimination%20at%20an%20identical%20developmental%20stage&amp;author=M%20Barchi&amp;author=S%20Mahadevaiah&amp;author=M%20Di%20Giacomo&amp;author=F%20Baudat&amp;author=DG%20de%20Rooij&amp;volume=25&amp;publication_year=2005&amp;pages=7203-7215&amp;pmid=16055729&amp;doi=10.1128/MCB.25.16.7203-7215.2005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib5">
<cite>Baudat F, Manova K, Yuen JP, Jasin M, Keeney S. Chromosome synapsis defects and sexually dimorphic meiotic progression in mice lacking Spo11. Molecular Cell. 2000;6:989–998. doi: 10.1016/s1097-2765(00)00098-8.</cite> [<a href="https://doi.org/10.1016/s1097-2765(00)00098-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11106739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cell&amp;title=Chromosome%20synapsis%20defects%20and%20sexually%20dimorphic%20meiotic%20progression%20in%20mice%20lacking%20Spo11&amp;author=F%20Baudat&amp;author=K%20Manova&amp;author=JP%20Yuen&amp;author=M%20Jasin&amp;author=S%20Keeney&amp;volume=6&amp;publication_year=2000&amp;pages=989-998&amp;pmid=11106739&amp;doi=10.1016/s1097-2765(00)00098-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib6">
<cite>Brown MS, Bishop DK. DNA strand exchange and RecA homologs in meiosis. Cold Spring Harbor Perspectives in Biology. 2015;7:a016659.  doi: 10.1101/cshperspect.a016659.</cite> [<a href="https://doi.org/10.1101/cshperspect.a016659" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4292170/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25475089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harbor%20Perspectives%20in%20Biology&amp;title=DNA%20strand%20exchange%20and%20RecA%20homologs%20in%20meiosis&amp;author=MS%20Brown&amp;author=DK%20Bishop&amp;volume=7&amp;publication_year=2015&amp;pages=a016659&amp;pmid=25475089&amp;doi=10.1101/cshperspect.a016659&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib7">
<cite>Burford A, Mackay A, Popov S, Vinci M, Carvalho D, Clarke M, Izquierdo E, Avery A, Jacques TS, Ingram WJ, Moore AS, Frawley K, Hassall TE, Robertson T, Jones C. The ten-year evolutionary trajectory of a highly recurrent paediatric high grade neuroepithelial tumour with MN1:BEND2 fusion. Scientific Reports. 2018;8:1032.  doi: 10.1038/s41598-018-19389-9.</cite> [<a href="https://doi.org/10.1038/s41598-018-19389-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5773598/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29348602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=The%20ten-year%20evolutionary%20trajectory%20of%20a%20highly%20recurrent%20paediatric%20high%20grade%20neuroepithelial%20tumour%20with%20MN1:BEND2%20fusion&amp;author=A%20Burford&amp;author=A%20Mackay&amp;author=S%20Popov&amp;author=M%20Vinci&amp;author=D%20Carvalho&amp;volume=8&amp;publication_year=2018&amp;pages=1032&amp;pmid=29348602&amp;doi=10.1038/s41598-018-19389-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib8">
<cite>Carofiglio F, Inagaki A, de Vries S, Wassenaar E, Schoenmakers S, Vermeulen C, van Cappellen WA, Sleddens-Linkels E, Grootegoed JA, Te Riele HPJ, de Massy B, Baarends WM. SPO11-independent DNA repair foci and their role in meiotic silencing. PLOS Genetics. 2013;9:e1003538.  doi: 10.1371/journal.pgen.1003538.</cite> [<a href="https://doi.org/10.1371/journal.pgen.1003538" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3675022/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23754961/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genetics&amp;title=SPO11-independent%20DNA%20repair%20foci%20and%20their%20role%20in%20meiotic%20silencing&amp;author=F%20Carofiglio&amp;author=A%20Inagaki&amp;author=S%20de%20Vries&amp;author=E%20Wassenaar&amp;author=S%20Schoenmakers&amp;volume=9&amp;publication_year=2013&amp;pages=e1003538&amp;pmid=23754961&amp;doi=10.1371/journal.pgen.1003538&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib9">
<cite>Dai Q, Ren A, Westholm JO, Serganov AA, Patel DJ, Lai EC. The BEN domain is a novel sequence-specific DNA-binding domain conserved in neural transcriptional repressors. Genes &amp; Development. 2013;27:602–614. doi: 10.1101/gad.213314.113.</cite> [<a href="https://doi.org/10.1101/gad.213314.113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3613608/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23468431/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20&amp;%20Development&amp;title=The%20BEN%20domain%20is%20a%20novel%20sequence-specific%20DNA-binding%20domain%20conserved%20in%20neural%20transcriptional%20repressors&amp;author=Q%20Dai&amp;author=A%20Ren&amp;author=JO%20Westholm&amp;author=AA%20Serganov&amp;author=DJ%20Patel&amp;volume=27&amp;publication_year=2013&amp;pages=602-614&amp;pmid=23468431&amp;doi=10.1101/gad.213314.113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib10">
<cite>Dai Qi, Ren A, Westholm JO, Duan H, Patel DJ, Lai EC. Common and distinct DNA-binding and regulatory activities of the BEN-solo transcription factor family. Genes &amp; Development. 2015;29:48–62. doi: 10.1101/gad.252122.114.</cite> [<a href="https://doi.org/10.1101/gad.252122.114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4281564/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25561495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20&amp;%20Development&amp;title=Common%20and%20distinct%20DNA-binding%20and%20regulatory%20activities%20of%20the%20BEN-solo%20transcription%20factor%20family&amp;author=Qi%20Dai&amp;author=A%20Ren&amp;author=JO%20Westholm&amp;author=H%20Duan&amp;author=DJ%20Patel&amp;volume=29&amp;publication_year=2015&amp;pages=48-62&amp;pmid=25561495&amp;doi=10.1101/gad.252122.114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib11">
<cite>Di Giacomo M, Comazzetto S, Saini H, De Fazio S, Carrieri C, Morgan M, Vasiliauskaite L, Benes V, Enright AJ, O’Carroll D. Multiple epigenetic mechanisms and the piRNA pathway enforce LINE1 silencing during adult spermatogenesis. Molecular Cell. 2013;50:601–608. doi: 10.1016/j.molcel.2013.04.026.</cite> [<a href="https://doi.org/10.1016/j.molcel.2013.04.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23706823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cell&amp;title=Multiple%20epigenetic%20mechanisms%20and%20the%20piRNA%20pathway%20enforce%20LINE1%20silencing%20during%20adult%20spermatogenesis&amp;author=M%20Di%20Giacomo&amp;author=S%20Comazzetto&amp;author=H%20Saini&amp;author=S%20De%20Fazio&amp;author=C%20Carrieri&amp;volume=50&amp;publication_year=2013&amp;pages=601-608&amp;pmid=23706823&amp;doi=10.1016/j.molcel.2013.04.026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib12">
<cite>Ding X, Schimenti JC. Strategies to identify genetic variants causing infertility. Trends in Molecular Medicine. 2021;27:792–806. doi: 10.1016/j.molmed.2020.12.008.</cite> [<a href="https://doi.org/10.1016/j.molmed.2020.12.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8263794/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33431240/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Molecular%20Medicine&amp;title=Strategies%20to%20identify%20genetic%20variants%20causing%20infertility&amp;author=X%20Ding&amp;author=JC%20Schimenti&amp;volume=27&amp;publication_year=2021&amp;pages=792-806&amp;pmid=33431240&amp;doi=10.1016/j.molmed.2020.12.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib13">
<cite>Enguita-Marruedo A, Martín-Ruiz M, García E, Gil-Fernández A, Parra MT, Viera A, Rufas JS, Page J. Transition from a meiotic to a somatic-like DNA damage response during the pachytene stage in mouse meiosis. PLOS Genetics. 2019;15:e1007439.  doi: 10.1371/journal.pgen.1007439.</cite> [<a href="https://doi.org/10.1371/journal.pgen.1007439" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6358097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30668564/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genetics&amp;title=Transition%20from%20a%20meiotic%20to%20a%20somatic-like%20DNA%20damage%20response%20during%20the%20pachytene%20stage%20in%20mouse%20meiosis&amp;author=A%20Enguita-Marruedo&amp;author=M%20Mart%C3%ADn-Ruiz&amp;author=E%20Garc%C3%ADa&amp;author=A%20Gil-Fern%C3%A1ndez&amp;author=MT%20Parra&amp;volume=15&amp;publication_year=2019&amp;pages=e1007439&amp;pmid=30668564&amp;doi=10.1371/journal.pgen.1007439&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib14">
<cite>Garretson A, Dumont BL, Handel MA. Reproductive genomics of the mouse: implications for human fertility and infertility. Development. 2023;150:dev201313.  doi: 10.1242/dev.201313.</cite> [<a href="https://doi.org/10.1242/dev.201313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10836652/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36779988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Reproductive%20genomics%20of%20the%20mouse:%20implications%20for%20human%20fertility%20and%20infertility&amp;author=A%20Garretson&amp;author=BL%20Dumont&amp;author=MA%20Handel&amp;volume=150&amp;publication_year=2023&amp;pages=dev201313&amp;pmid=36779988&amp;doi=10.1242/dev.201313&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib15">
<cite>Gerasimova TI, Gdula DA, Gerasimov DV, Simonova O, Corces VG. A Drosophila protein that imparts directionality on a chromatin insulator is an enhancer of position-effect variegation. Cell. 1995;82:587–597. doi: 10.1016/0092-8674(95)90031-4.</cite> [<a href="https://doi.org/10.1016/0092-8674(95)90031-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7664338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20Drosophila%20protein%20that%20imparts%20directionality%20on%20a%20chromatin%20insulator%20is%20an%20enhancer%20of%20position-effect%20variegation&amp;author=TI%20Gerasimova&amp;author=DA%20Gdula&amp;author=DV%20Gerasimov&amp;author=O%20Simonova&amp;author=VG%20Corces&amp;volume=82&amp;publication_year=1995&amp;pages=587-597&amp;pmid=7664338&amp;doi=10.1016/0092-8674(95)90031-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib16">
<cite>Goedecke W, Eijpe M, Offenberg HH, van Aalderen M, Heyting C. Mre11 and Ku70 interact in somatic cells, but are differentially expressed in early meiosis. Nature Genetics. 1999;23:194–198. doi: 10.1038/13821.</cite> [<a href="https://doi.org/10.1038/13821" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10508516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=Mre11%20and%20Ku70%20interact%20in%20somatic%20cells,%20but%20are%20differentially%20expressed%20in%20early%20meiosis&amp;author=W%20Goedecke&amp;author=M%20Eijpe&amp;author=HH%20Offenberg&amp;author=M%20van%20Aalderen&amp;author=C%20Heyting&amp;volume=23&amp;publication_year=1999&amp;pages=194-198&amp;pmid=10508516&amp;doi=10.1038/13821&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib17">
<cite>Hinch AG, Becker PW, Li T, Moralli D, Zhang G, Bycroft C, Green C, Keeney S, Shi Q, Davies B, Donnelly P. The Configuration of RPA, RAD51, and DMC1 binding in meiosis reveals the nature of critical recombination intermediates. Molecular Cell. 2020;79:689–701. doi: 10.1016/j.molcel.2020.06.015.</cite> [<a href="https://doi.org/10.1016/j.molcel.2020.06.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7447979/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32610038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cell&amp;title=The%20Configuration%20of%20RPA,%20RAD51,%20and%20DMC1%20binding%20in%20meiosis%20reveals%20the%20nature%20of%20critical%20recombination%20intermediates&amp;author=AG%20Hinch&amp;author=PW%20Becker&amp;author=T%20Li&amp;author=D%20Moralli&amp;author=G%20Zhang&amp;volume=79&amp;publication_year=2020&amp;pages=689-701&amp;pmid=32610038&amp;doi=10.1016/j.molcel.2020.06.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib18">
<cite>Huang Y, Roig I. Genetic control of meiosis surveillance mechanisms in mammals. Frontiers in Cell and Developmental Biology. 2023;11:1127440.  doi: 10.3389/fcell.2023.1127440.</cite> [<a href="https://doi.org/10.3389/fcell.2023.1127440" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9996228/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36910159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Cell%20and%20Developmental%20Biology&amp;title=Genetic%20control%20of%20meiosis%20surveillance%20mechanisms%20in%20mammals&amp;author=Y%20Huang&amp;author=I%20Roig&amp;volume=11&amp;publication_year=2023&amp;pages=1127440&amp;pmid=36910159&amp;doi=10.3389/fcell.2023.1127440&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib19">
<cite>Hunter N. Oocyte quality control: causes, mechanisms, and consequences. Cold Spring Harbor Symposia on Quantitative Biology. 2017;82:235–247. doi: 10.1101/sqb.2017.82.035394.</cite> [<a href="https://doi.org/10.1101/sqb.2017.82.035394" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29743337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harbor%20Symposia%20on%20Quantitative%20Biology&amp;title=Oocyte%20quality%20control:%20causes,%20mechanisms,%20and%20consequences&amp;author=N%20Hunter&amp;volume=82&amp;publication_year=2017&amp;pages=235-247&amp;pmid=29743337&amp;doi=10.1101/sqb.2017.82.035394&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib20">
<cite>Kaul-Ghanekar R, Jalota A, Pavithra L, Tucker P, Chattopadhyay S. SMAR1 and Cux/CDP modulate chromatin and act as negative regulators of the TCRbeta enhancer (Ebeta) Nucleic Acids Research. 2004;32:4862–4875. doi: 10.1093/nar/gkh807.</cite> [<a href="https://doi.org/10.1093/nar/gkh807" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC519105/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15371550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=SMAR1%20and%20Cux/CDP%20modulate%20chromatin%20and%20act%20as%20negative%20regulators%20of%20the%20TCRbeta%20enhancer%20(Ebeta)&amp;author=R%20Kaul-Ghanekar&amp;author=A%20Jalota&amp;author=L%20Pavithra&amp;author=P%20Tucker&amp;author=S%20Chattopadhyay&amp;volume=32&amp;publication_year=2004&amp;pages=4862-4875&amp;pmid=15371550&amp;doi=10.1093/nar/gkh807&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib21">
<cite>Korutla L, Wang PJ, Mackler SA. The POZ/BTB protein NAC1 interacts with two different histone deacetylases in neuronal-like cultures. Journal of Neurochemistry. 2005;94:786–793. doi: 10.1111/j.1471-4159.2005.03206.x.</cite> [<a href="https://doi.org/10.1111/j.1471-4159.2005.03206.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16033423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Neurochemistry&amp;title=The%20POZ/BTB%20protein%20NAC1%20interacts%20with%20two%20different%20histone%20deacetylases%20in%20neuronal-like%20cultures&amp;author=L%20Korutla&amp;author=PJ%20Wang&amp;author=SA%20Mackler&amp;volume=94&amp;publication_year=2005&amp;pages=786-793&amp;pmid=16033423&amp;doi=10.1111/j.1471-4159.2005.03206.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib22">
<cite>Korutla L, Degnan R, Wang P, Mackler SA. NAC1, a cocaine-regulated POZ/BTB protein interacts with CoREST. Journal of Neurochemistry. 2007;101:611–618. doi: 10.1111/j.1471-4159.2006.04387.x.</cite> [<a href="https://doi.org/10.1111/j.1471-4159.2006.04387.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17254023/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Neurochemistry&amp;title=NAC1,%20a%20cocaine-regulated%20POZ/BTB%20protein%20interacts%20with%20CoREST&amp;author=L%20Korutla&amp;author=R%20Degnan&amp;author=P%20Wang&amp;author=SA%20Mackler&amp;volume=101&amp;publication_year=2007&amp;pages=611-618&amp;pmid=17254023&amp;doi=10.1111/j.1471-4159.2006.04387.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib23">
<cite>Liebe B, Alsheimer M, Höög C, Benavente R, Scherthan H. Telomere attachment, meiotic chromosome condensation, pairing, and bouquet stage duration are modified in spermatocytes lacking axial elements. Molecular Biology of the Cell. 2004;15:827–837. doi: 10.1091/mbc.e03-07-0524.</cite> [<a href="https://doi.org/10.1091/mbc.e03-07-0524" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC329396/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14657244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20of%20the%20Cell&amp;title=Telomere%20attachment,%20meiotic%20chromosome%20condensation,%20pairing,%20and%20bouquet%20stage%20duration%20are%20modified%20in%20spermatocytes%20lacking%20axial%20elements&amp;author=B%20Liebe&amp;author=M%20Alsheimer&amp;author=C%20H%C3%B6%C3%B6g&amp;author=R%20Benavente&amp;author=H%20Scherthan&amp;volume=15&amp;publication_year=2004&amp;pages=827-837&amp;pmid=14657244&amp;doi=10.1091/mbc.e03-07-0524&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib24">
<cite>Ma L, Xie D, Luo M, Lin X, Nie H, Chen J, Gao C, Duo S, Han C. Identification and characterization of BEND2 as a key regulator of meiosis during mouse spermatogenesis. Science Advances. 2022;8:1606.  doi: 10.1126/sciadv.abn1606.</cite> [<a href="https://doi.org/10.1126/sciadv.abn1606" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9132480/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35613276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20Advances&amp;title=Identification%20and%20characterization%20of%20BEND2%20as%20a%20key%20regulator%20of%20meiosis%20during%20mouse%20spermatogenesis&amp;author=L%20Ma&amp;author=D%20Xie&amp;author=M%20Luo&amp;author=X%20Lin&amp;author=H%20Nie&amp;volume=8&amp;publication_year=2022&amp;pages=1606&amp;pmid=35613276&amp;doi=10.1126/sciadv.abn1606&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib25">
<cite>Malcher A, Stokowy T, Berman A, Olszewska M, Jedrzejczak P, Sielski D, Nowakowski A, Rozwadowska N, Yatsenko AN, Kurpisz MK. Whole-genome sequencing identifies new candidate genes for nonobstructive azoospermia. Andrology. 2022;10:1605–1624. doi: 10.1111/andr.13269.</cite> [<a href="https://doi.org/10.1111/andr.13269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9826517/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36017582/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Andrology&amp;title=Whole-genome%20sequencing%20identifies%20new%20candidate%20genes%20for%20nonobstructive%20azoospermia&amp;author=A%20Malcher&amp;author=T%20Stokowy&amp;author=A%20Berman&amp;author=M%20Olszewska&amp;author=P%20Jedrzejczak&amp;volume=10&amp;publication_year=2022&amp;pages=1605-1624&amp;pmid=36017582&amp;doi=10.1111/andr.13269&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib26">
<cite>Malki S, van der Heijden GW, O’Donnell KA, Martin SL, Bortvin A. A role for retrotransposon LINE-1 in fetal oocyte attrition in mice. Developmental Cell. 2014;29:521–533. doi: 10.1016/j.devcel.2014.04.027.</cite> [<a href="https://doi.org/10.1016/j.devcel.2014.04.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4056315/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24882376/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Developmental%20Cell&amp;title=A%20role%20for%20retrotransposon%20LINE-1%20in%20fetal%20oocyte%20attrition%20in%20mice&amp;author=S%20Malki&amp;author=GW%20van%20der%20Heijden&amp;author=KA%20O%E2%80%99Donnell&amp;author=SL%20Martin&amp;author=A%20Bortvin&amp;volume=29&amp;publication_year=2014&amp;pages=521-533&amp;pmid=24882376&amp;doi=10.1016/j.devcel.2014.04.027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib27">
<cite>Malki S, van der Heijden GW, O’Donnell KA, Martin SL, Bortvin A. A role for retrotransposon LINE-1 in fetal oocyte attrition in mice. Developmental Cell. 2019;51:658.  doi: 10.1016/j.devcel.2019.11.011.</cite> [<a href="https://doi.org/10.1016/j.devcel.2019.11.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31794719/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Developmental%20Cell&amp;title=A%20role%20for%20retrotransposon%20LINE-1%20in%20fetal%20oocyte%20attrition%20in%20mice&amp;author=S%20Malki&amp;author=GW%20van%20der%20Heijden&amp;author=KA%20O%E2%80%99Donnell&amp;author=SL%20Martin&amp;author=A%20Bortvin&amp;volume=51&amp;publication_year=2019&amp;pages=658&amp;pmid=31794719&amp;doi=10.1016/j.devcel.2019.11.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib28">
<cite>Marcet-Ortega M, Roig I.  Surveillance Mechanisms in Mammalian Meiosis. Universitat Autònoma de Barcelona; 2016. </cite> [<a href="https://scholar.google.com/scholar_lookup?title=Surveillance%20Mechanisms%20in%20Mammalian%20Meiosis&amp;author=M%20Marcet-Ortega&amp;author=I%20Roig&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib29">
<cite>Marcet-Ortega M, Pacheco S, Martínez-Marchal A, Castillo H, Flores E, Jasin M, Keeney S, Roig I. p53 and TAp63 participate in the recombination-dependent pachytene arrest in mouse spermatocytes. PLOS Genetics. 2017;13:e1006845.  doi: 10.1371/journal.pgen.1006845.</cite> [<a href="https://doi.org/10.1371/journal.pgen.1006845" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5491309/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28617799/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genetics&amp;title=p53%20and%20TAp63%20participate%20in%20the%20recombination-dependent%20pachytene%20arrest%20in%20mouse%20spermatocytes&amp;author=M%20Marcet-Ortega&amp;author=S%20Pacheco&amp;author=A%20Mart%C3%ADnez-Marchal&amp;author=H%20Castillo&amp;author=E%20Flores&amp;volume=13&amp;publication_year=2017&amp;pages=e1006845&amp;pmid=28617799&amp;doi=10.1371/journal.pgen.1006845&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib30">
<cite>Martínez-Marchal A, Huang Y, Guillot-Ferriols MT, Ferrer-Roda M, Guixé A, Garcia-Caldés M, Roig I. The DNA damage response is required for oocyte cyst breakdown and follicle formation in mice. PLOS Genetics. 2020;16:e1009067.  doi: 10.1371/journal.pgen.1009067.</cite> [<a href="https://doi.org/10.1371/journal.pgen.1009067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7710113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33206637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genetics&amp;title=The%20DNA%20damage%20response%20is%20required%20for%20oocyte%20cyst%20breakdown%20and%20follicle%20formation%20in%20mice&amp;author=A%20Mart%C3%ADnez-Marchal&amp;author=Y%20Huang&amp;author=MT%20Guillot-Ferriols&amp;author=M%20Ferrer-Roda&amp;author=A%20Guix%C3%A9&amp;volume=16&amp;publication_year=2020&amp;pages=e1009067&amp;pmid=33206637&amp;doi=10.1371/journal.pgen.1009067&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib31">
<cite>Moens PB, Kolas NK, Tarsounas M, Marcon E, Cohen PE, Spyropoulos B. The time course and chromosomal localization of recombination-related proteins at meiosis in the mouse are compatible with models that can resolve the early DNA-DNA interactions without reciprocal recombination. Journal of Cell Science. 2002;115:1611–1622. doi: 10.1242/jcs.115.8.1611.</cite> [<a href="https://doi.org/10.1242/jcs.115.8.1611" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11950880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cell%20Science&amp;title=The%20time%20course%20and%20chromosomal%20localization%20of%20recombination-related%20proteins%20at%20meiosis%20in%20the%20mouse%20are%20compatible%20with%20models%20that%20can%20resolve%20the%20early%20DNA-DNA%20interactions%20without%20reciprocal%20recombination&amp;author=PB%20Moens&amp;author=NK%20Kolas&amp;author=M%20Tarsounas&amp;author=E%20Marcon&amp;author=PE%20Cohen&amp;volume=115&amp;publication_year=2002&amp;pages=1611-1622&amp;pmid=11950880&amp;doi=10.1242/jcs.115.8.1611&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib32">
<cite>Nègre N, Brown CD, Shah PK, Kheradpour P, Morrison CA, Henikoff JG, Feng X, Ahmad K, Russell S, White RAH, Stein L, Henikoff S, Kellis M, White KP. A comprehensive map of insulator elements for the Drosophila genome. PLOS Genetics. 2010;6:e1000814.  doi: 10.1371/journal.pgen.1000814.</cite> [<a href="https://doi.org/10.1371/journal.pgen.1000814" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2797089/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20084099/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genetics&amp;title=A%20comprehensive%20map%20of%20insulator%20elements%20for%20the%20Drosophila%20genome&amp;author=N%20N%C3%A8gre&amp;author=CD%20Brown&amp;author=PK%20Shah&amp;author=P%20Kheradpour&amp;author=CA%20Morrison&amp;volume=6&amp;publication_year=2010&amp;pages=e1000814&amp;pmid=20084099&amp;doi=10.1371/journal.pgen.1000814&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib33">
<cite>Pacheco S, Marcet-Ortega M, Lange J, Jasin M, Keeney S, Roig I. The ATM signaling cascade promotes recombination-dependent pachytene arrest in mouse spermatocytes. PLOS Genetics. 2015;11:e1005017.  doi: 10.1371/journal.pgen.1005017.</cite> [<a href="https://doi.org/10.1371/journal.pgen.1005017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4358828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25768017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genetics&amp;title=The%20ATM%20signaling%20cascade%20promotes%20recombination-dependent%20pachytene%20arrest%20in%20mouse%20spermatocytes&amp;author=S%20Pacheco&amp;author=M%20Marcet-Ortega&amp;author=J%20Lange&amp;author=M%20Jasin&amp;author=S%20Keeney&amp;volume=11&amp;publication_year=2015&amp;pages=e1005017&amp;pmid=25768017&amp;doi=10.1371/journal.pgen.1005017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib34">
<cite>Pacheco S, Maldonado-Linares A, Marcet-Ortega M, Rojas C, Martínez-Marchal A, Fuentes-Lazaro J, Lange J, Jasin M, Keeney S, Fernández-Capetillo O, Garcia-Caldés M, Roig I. ATR is required to complete meiotic recombination in mice. Nature Communications. 2018;9:2622.  doi: 10.1038/s41467-018-04851-z.</cite> [<a href="https://doi.org/10.1038/s41467-018-04851-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6033890/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29977027/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=ATR%20is%20required%20to%20complete%20meiotic%20recombination%20in%20mice&amp;author=S%20Pacheco&amp;author=A%20Maldonado-Linares&amp;author=M%20Marcet-Ortega&amp;author=C%20Rojas&amp;author=A%20Mart%C3%ADnez-Marchal&amp;volume=9&amp;publication_year=2018&amp;pages=2622&amp;pmid=29977027&amp;doi=10.1038/s41467-018-04851-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib35">
<cite>Pittman DL, Cobb J, Schimenti KJ, Wilson LA, Cooper DM, Brignull E, Handel MA, Schimenti JC. Meiotic prophase arrest with failure of chromosome synapsis in mice deficient for Dmc1, a germline-specific RecA homolog. Molecular Cell. 1998;1:697–705. doi: 10.1016/s1097-2765(00)80069-6.</cite> [<a href="https://doi.org/10.1016/s1097-2765(00)80069-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9660953/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cell&amp;title=Meiotic%20prophase%20arrest%20with%20failure%20of%20chromosome%20synapsis%20in%20mice%20deficient%20for%20Dmc1,%20a%20germline-specific%20RecA%20homolog&amp;author=DL%20Pittman&amp;author=J%20Cobb&amp;author=KJ%20Schimenti&amp;author=LA%20Wilson&amp;author=DM%20Cooper&amp;volume=1&amp;publication_year=1998&amp;pages=697-705&amp;pmid=9660953&amp;doi=10.1016/s1097-2765(00)80069-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib36">
<cite>Rampalli S, Pavithra L, Bhatt A, Kundu TK, Chattopadhyay S. Tumor suppressor SMAR1 mediates cyclin D1 repression by recruitment of the SIN3/histone deacetylase 1 complex. Molecular and Cellular Biology. 2005;25:8415–8429. doi: 10.1128/MCB.25.19.8415-8429.2005.</cite> [<a href="https://doi.org/10.1128/MCB.25.19.8415-8429.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1265755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16166625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20and%20Cellular%20Biology&amp;title=Tumor%20suppressor%20SMAR1%20mediates%20cyclin%20D1%20repression%20by%20recruitment%20of%20the%20SIN3/histone%20deacetylase%201%20complex&amp;author=S%20Rampalli&amp;author=L%20Pavithra&amp;author=A%20Bhatt&amp;author=TK%20Kundu&amp;author=S%20Chattopadhyay&amp;volume=25&amp;publication_year=2005&amp;pages=8415-8429&amp;pmid=16166625&amp;doi=10.1128/MCB.25.19.8415-8429.2005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib37">
<cite>Riera-Escamilla A, Enguita-Marruedo A, Moreno-Mendoza D, Chianese C, Sleddens-Linkels E, Contini E, Benelli M, Natali A, Colpi GM, Ruiz-Castañé E, Maggi M, Baarends WM, Krausz C. Sequencing of a “mouse azoospermia” gene panel in azoospermic men: identification of RNF212 and STAG3 mutations as novel genetic causes of meiotic arrest. Human Reproduction. 2019;34:978–988. doi: 10.1093/humrep/dez042.</cite> [<a href="https://doi.org/10.1093/humrep/dez042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31125047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Human%20Reproduction&amp;title=Sequencing%20of%20a%20%E2%80%9Cmouse%20azoospermia%E2%80%9D%20gene%20panel%20in%20azoospermic%20men:%20identification%20of%20RNF212%20and%20STAG3%20mutations%20as%20novel%20genetic%20causes%20of%20meiotic%20arrest&amp;author=A%20Riera-Escamilla&amp;author=A%20Enguita-Marruedo&amp;author=D%20Moreno-Mendoza&amp;author=C%20Chianese&amp;author=E%20Sleddens-Linkels&amp;volume=34&amp;publication_year=2019&amp;pages=978-988&amp;pmid=31125047&amp;doi=10.1093/humrep/dez042&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib38">
<cite>Roig I, Liebe B, Egozcue J, Cabero L, Garcia M, Scherthan H. Female-specific features of recombinational double-stranded DNA repair in relation to synapsis and telomere dynamics in human oocytes. Chromosoma. 2004;113:22–33. doi: 10.1007/s00412-004-0290-8.</cite> [<a href="https://doi.org/10.1007/s00412-004-0290-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15235794/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chromosoma&amp;title=Female-specific%20features%20of%20recombinational%20double-stranded%20DNA%20repair%20in%20relation%20to%20synapsis%20and%20telomere%20dynamics%20in%20human%20oocytes&amp;author=I%20Roig&amp;author=B%20Liebe&amp;author=J%20Egozcue&amp;author=L%20Cabero&amp;author=M%20Garcia&amp;volume=113&amp;publication_year=2004&amp;pages=22-33&amp;pmid=15235794&amp;doi=10.1007/s00412-004-0290-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib39">
<cite>Roig I, Dowdle JA, Toth A, de Rooij DG, Jasin M, Keeney S. Mouse TRIP13/PCH2 is required for recombination and normal higher-order chromosome structure during meiosis. PLOS Genetics. 2010;6:e1001062.  doi: 10.1371/journal.pgen.1001062.</cite> [<a href="https://doi.org/10.1371/journal.pgen.1001062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2920839/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20711356/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genetics&amp;title=Mouse%20TRIP13/PCH2%20is%20required%20for%20recombination%20and%20normal%20higher-order%20chromosome%20structure%20during%20meiosis&amp;author=I%20Roig&amp;author=JA%20Dowdle&amp;author=A%20Toth&amp;author=DG%20de%20Rooij&amp;author=M%20Jasin&amp;volume=6&amp;publication_year=2010&amp;pages=e1001062&amp;pmid=20711356&amp;doi=10.1371/journal.pgen.1001062&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib40">
<cite>Rossetti R, Ferrari I, Bonomi M, Persani L. Genetics of primary ovarian insufficiency. Clinical Genetics. 2017;91:183–198. doi: 10.1111/cge.12921.</cite> [<a href="https://doi.org/10.1111/cge.12921" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27861765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clinical%20Genetics&amp;title=Genetics%20of%20primary%20ovarian%20insufficiency&amp;author=R%20Rossetti&amp;author=I%20Ferrari&amp;author=M%20Bonomi&amp;author=L%20Persani&amp;volume=91&amp;publication_year=2017&amp;pages=183-198&amp;pmid=27861765&amp;doi=10.1111/cge.12921&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib41">
<cite>Ruth KS, Day FR, Hussain J, Martínez-Marchal A, Aiken CE, Azad A, Thompson DJ, Knoblochova L, Abe H, Tarry-Adkins JL, Gonzalez JM, Fontanillas P, Claringbould A, Bakker OB, Sulem P, Walters RG, Terao C, Turon S, Horikoshi M, Lin K, Onland-Moret NC, Sankar A, Hertz EPT, Timshel PN, Shukla V, Borup R, Olsen KW, Aguilera P, Ferrer-Roda M, Huang Y, Stankovic S, Timmers P, Ahearn TU, Alizadeh BZ, Naderi E, Andrulis IL, Arnold AM, Aronson KJ, Augustinsson A, Bandinelli S, Barbieri CM, Beaumont RN, Becher H, Beckmann MW, Benonisdottir S, Bergmann S, Bochud M, Boerwinkle E, Bojesen SE, Bolla MK, Boomsma DI, Bowker N, Brody JA, Broer L, Buring JE, Campbell A, Campbell H, Castelao JE, Catamo E, Chanock SJ, Chenevix-Trench G, Ciullo M, Corre T, Couch FJ, Cox A, Crisponi L, Cross SS, Cucca F, Czene K, Smith GD, de Geus E, de Mutsert R, De Vivo I, Demerath EW, Dennis J, Dunning AM, Dwek M, Eriksson M, Esko T, Fasching PA, Faul JD, Ferrucci L, Franceschini N, Frayling TM, Gago-Dominguez M, Mezzavilla M, García-Closas M, Gieger C, Giles GG, Grallert H, Gudbjartsson DF, Gudnason V, Guénel P, Haiman CA, Håkansson N, Hall P, Hayward C, He C, He W, Heiss G, Høffding MK, Hopper JL, Hottenga JJ, Hu F, Hunter D, Ikram MA, Jackson RD, Joaquim MDR, John EM, Joshi PK, Karasik D, Kardia SLR, Kartsonaki C, Karlsson R, Kitahara CM, Kolcic I, Kooperberg C, Kraft P, Kurian AW, Kutalik Z, La Bianca M, LaChance G, Langenberg C, Launer LJ, Laven JSE, Lawlor DA, Le Marchand L, Li J, Lindblom A, Lindstrom S, Lindstrom T, Linet M, Liu Y, Liu S, Luan J, Mägi R, Magnusson PKE, Mangino M, Mannermaa A, Marco B, Marten J, Martin NG, Mbarek H, McKnight B, Medland SE, Meisinger C, Meitinger T, Menni C, Metspalu A, Milani L, Milne RL, Montgomery GW, Mook-Kanamori DO, Mulas A, Mulligan AM, Murray A, Nalls MA, Newman A, Noordam R, Nutile T, Nyholt DR, Olshan AF, Olsson H, Painter JN, Patel AV, Pedersen NL, Perjakova N, Peters A, Peters U, Pharoah PDP, Polasek O, Porcu E, Psaty BM, Rahman I, Rennert G, Rennert HS, Ridker PM, Ring SM, Robino A, Rose LM, Rosendaal FR, Rossouw J, Rudan I, Rueedi R, Ruggiero D, Sala CF, Saloustros E, Sandler DP, Sanna S, Sawyer EJ, Sarnowski C, Schlessinger D, Schmidt MK, Schoemaker MJ, Schraut KE, Scott C, Shekari S, Shrikhande A, Smith AV, Smith BH, Smith JA, Sorice R, Southey MC, Spector TD, Spinelli JJ, Stampfer M, Stöckl D, van Meurs JBJ, Strauch K, Styrkarsdottir U, Swerdlow AJ, Tanaka T, Teras LR, Teumer A, Þorsteinsdottir U, Timpson NJ, Toniolo D, Traglia M, Troester MA, Truong T, Tyrrell J, Uitterlinden AG, Ulivi S, Vachon CM, Vitart V, Völker U, Vollenweider P, Völzke H, Wang Q, Wareham NJ, Weinberg CR, Weir DR, Wilcox AN, van Dijk KW, Willemsen G, Wilson JF, Wolffenbuttel BHR, Wolk A, Wood AR, Zhao W, Zygmunt M, Chen Z, Li L, Franke L, Burgess S, Deelen P, Pers TH, Grøndahl ML, Andersen CY, Pujol A, Lopez-Contreras AJ, Daniel JA, Stefansson K, Chang-Claude J, van der Schouw YT, Lunetta KL, Chasman DI, Easton DF, Visser JA, Ozanne SE, Namekawa SH, Solc P, Murabito JM, Ong KK, Hoffmann ER, Murray A, Roig I, Perry JRB, Biobank-based Integrative Omics Study Consortium. eQTLGen Consortium. Biobank Japan Project. China Kadoorie Biobank Collaborative Group. kConFab Investigators. LifeLines Cohort Study. InterAct consortium. 23andMe Research Team  Genetic insights into biological mechanisms governing human ovarian ageing. Nature. 2021;596:393–397. doi: 10.1038/s41586-021-03779-7.</cite> [<a href="https://doi.org/10.1038/s41586-021-03779-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7611832/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34349265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetic%20insights%20into%20biological%20mechanisms%20governing%20human%20ovarian%20ageing&amp;author=KS%20Ruth&amp;author=FR%20Day&amp;author=J%20Hussain&amp;author=A%20Mart%C3%ADnez-Marchal&amp;author=CE%20Aiken&amp;volume=596&amp;publication_year=2021&amp;pages=393-397&amp;pmid=34349265&amp;doi=10.1038/s41586-021-03779-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib42">
<cite>Sathyan KM, Shen Z, Tripathi V, Prasanth KV, Prasanth SG. A BEN-domain-containing protein associates with heterochromatin and represses transcription. Journal of Cell Science. 2011;124:3149–3163. doi: 10.1242/jcs.086603.</cite> [<a href="https://doi.org/10.1242/jcs.086603" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4074281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21914818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cell%20Science&amp;title=A%20BEN-domain-containing%20protein%20associates%20with%20heterochromatin%20and%20represses%20transcription&amp;author=KM%20Sathyan&amp;author=Z%20Shen&amp;author=V%20Tripathi&amp;author=KV%20Prasanth&amp;author=SG%20Prasanth&amp;volume=124&amp;publication_year=2011&amp;pages=3149-3163&amp;pmid=21914818&amp;doi=10.1242/jcs.086603&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib43">
<cite>Scarpa A, Chang DK, Nones K, Corbo V, Patch AM, Bailey P, Lawlor RT, Johns AL, Miller DK, Mafficini A, Rusev B, Scardoni M, Antonello D, Barbi S, Sikora KO, Cingarlini S, Vicentini C, McKay S, Quinn MCJ, Bruxner TJC, Christ AN, Harliwong I, Idrisoglu S, McLean S, Nourse C, Nourbakhsh E, Wilson PJ, Anderson MJ, Fink JL, Newell F, Waddell N, Holmes O, Kazakoff SH, Leonard C, Wood S, Xu Q, Nagaraj SH, Amato E, Dalai I, Bersani S, Cataldo I, Dei Tos AP, Capelli P, Davì MV, Landoni L, Malpaga A, Miotto M, Whitehall VLJ, Leggett BA, Harris JL, Harris J, Jones MD, Humphris J, Chantrill LA, Chin V, Nagrial AM, Pajic M, Scarlett CJ, Pinho A, Rooman I, Toon C, Wu J, Pinese M, Cowley M, Barbour A, Mawson A, Humphrey ES, Colvin EK, Chou A, Lovell JA, Jamieson NB, Duthie F, Gingras MC, Fisher WE, Dagg RA, Lau LMS, Lee M, Pickett HA, Reddel RR, Samra JS, Kench JG, Merrett ND, Epari K, Nguyen NQ, Zeps N, Falconi M, Simbolo M, Butturini G, Van Buren G, Partelli S, Fassan M, Khanna KK, Gill AJ, Wheeler DA, Gibbs RA, Musgrove EA, Bassi C, Tortora G, Pederzoli P, Pearson JV, Waddell N, Biankin AV, Grimmond SM, Australian Pancreatic Cancer Genome Initiative  Whole-genome landscape of pancreatic neuroendocrine tumours. Nature. 2017;543:65–71. doi: 10.1038/nature21063.</cite> [<a href="https://doi.org/10.1038/nature21063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28199314/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Whole-genome%20landscape%20of%20pancreatic%20neuroendocrine%20tumours&amp;author=A%20Scarpa&amp;author=DK%20Chang&amp;author=K%20Nones&amp;author=V%20Corbo&amp;author=AM%20Patch&amp;volume=543&amp;publication_year=2017&amp;pages=65-71&amp;pmid=28199314&amp;doi=10.1038/nature21063&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib44">
<cite>Shibuya H, Morimoto A, Watanabe Y. The dissection of meiotic chromosome movement in mice using an in vivo electroporation technique. PLOS Genetics. 2014;10:e1004821.  doi: 10.1371/journal.pgen.1004821.</cite> [<a href="https://doi.org/10.1371/journal.pgen.1004821" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4263375/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25502938/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genetics&amp;title=The%20dissection%20of%20meiotic%20chromosome%20movement%20in%20mice%20using%20an%20in%20vivo%20electroporation%20technique&amp;author=H%20Shibuya&amp;author=A%20Morimoto&amp;author=Y%20Watanabe&amp;volume=10&amp;publication_year=2014&amp;pages=e1004821&amp;pmid=25502938&amp;doi=10.1371/journal.pgen.1004821&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib45">
<cite>Soper SFC, van der Heijden GW, Hardiman TC, Goodheart M, Martin SL, de Boer P, Bortvin A. Mouse maelstrom, a component of nuage, is essential for spermatogenesis and transposon repression in meiosis. Developmental Cell. 2008;15:285–297. doi: 10.1016/j.devcel.2008.05.015.</cite> [<a href="https://doi.org/10.1016/j.devcel.2008.05.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2546488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18694567/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Developmental%20Cell&amp;title=Mouse%20maelstrom,%20a%20component%20of%20nuage,%20is%20essential%20for%20spermatogenesis%20and%20transposon%20repression%20in%20meiosis&amp;author=SFC%20Soper&amp;author=GW%20van%20der%20Heijden&amp;author=TC%20Hardiman&amp;author=M%20Goodheart&amp;author=SL%20Martin&amp;volume=15&amp;publication_year=2008&amp;pages=285-297&amp;pmid=18694567&amp;doi=10.1016/j.devcel.2008.05.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib46">
<cite>Sturm D, Orr BA, Toprak UH, Hovestadt V, Jones DTW, Capper D, Sill M, Buchhalter I, Northcott PA, Leis I, Ryzhova M, Koelsche C, Pfaff E, Allen SJ, Balasubramanian G, Worst BC, Pajtler KW, Brabetz S, Johann PD, Sahm F, Reimand J, Mackay A, Carvalho DM, Remke M, Phillips JJ, Perry A, Cowdrey C, Drissi R, Fouladi M, Giangaspero F, Łastowska M, Grajkowska W, Scheurlen W, Pietsch T, Hagel C, Gojo J, Lötsch D, Berger W, Slavc I, Haberler C, Jouvet A, Holm S, Hofer S, Prinz M, Keohane C, Fried I, Mawrin C, Scheie D, Mobley BC, Schniederjan MJ, Santi M, Buccoliero AM, Dahiya S, Kramm CM, von Bueren AO, von Hoff K, Rutkowski S, Herold-Mende C, Frühwald MC, Milde T, Hasselblatt M, Wesseling P, Rößler J, Schüller U, Ebinger M, Schittenhelm J, Frank S, Grobholz R, Vajtai I, Hans V, Schneppenheim R, Zitterbart K, Collins VP, Aronica E, Varlet P, Puget S, Dufour C, Grill J, Figarella-Branger D, Wolter M, Schuhmann MU, Shalaby T, Grotzer M, van Meter T, Monoranu C-M, Felsberg J, Reifenberger G, Snuderl M, Forrester LA, Koster J, Versteeg R, Volckmann R, van Sluis P, Wolf S, Mikkelsen T, Gajjar A, Aldape K, Moore AS, Taylor MD, Jones C, Jabado N, Karajannis MA, Eils R, Schlesner M, Lichter P, von Deimling A, Pfister SM, Ellison DW, Korshunov A, Kool M. New brain tumor entities emerge from molecular classification of CNS-PNETs. Cell. 2016;164:1060–1072. doi: 10.1016/j.cell.2016.01.015.</cite> [<a href="https://doi.org/10.1016/j.cell.2016.01.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5139621/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26919435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=New%20brain%20tumor%20entities%20emerge%20from%20molecular%20classification%20of%20CNS-PNETs&amp;author=D%20Sturm&amp;author=BA%20Orr&amp;author=UH%20Toprak&amp;author=V%20Hovestadt&amp;author=DTW%20Jones&amp;volume=164&amp;publication_year=2016&amp;pages=1060-1072&amp;pmid=26919435&amp;doi=10.1016/j.cell.2016.01.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib47">
<cite>Tharp ME, Malki S, Bortvin A. Maximizing the ovarian reserve in mice by evading LINE-1 genotoxicity. Nature Communications. 2020;11:330.  doi: 10.1038/s41467-019-14055-8.</cite> [<a href="https://doi.org/10.1038/s41467-019-14055-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6965193/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31949138/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Maximizing%20the%20ovarian%20reserve%20in%20mice%20by%20evading%20LINE-1%20genotoxicity&amp;author=ME%20Tharp&amp;author=S%20Malki&amp;author=A%20Bortvin&amp;volume=11&amp;publication_year=2020&amp;pages=330&amp;pmid=31949138&amp;doi=10.1038/s41467-019-14055-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib48">
<cite>Wang PJ. Tracking LINE1 retrotransposition in the germline. PNAS. 2017;114:7194–7196. doi: 10.1073/pnas.1709067114.</cite> [<a href="https://doi.org/10.1073/pnas.1709067114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5514774/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28663337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=Tracking%20LINE1%20retrotransposition%20in%20the%20germline&amp;author=PJ%20Wang&amp;volume=114&amp;publication_year=2017&amp;pages=7194-7196&amp;pmid=28663337&amp;doi=10.1073/pnas.1709067114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib49">
<cite>Williamson LM, Steel M, Grewal JK, Thibodeau ML, Zhao EY, Loree JM, Yang KC, Gorski SM, Mungall AJ, Mungall KL, Moore RA, Marra MA, Laskin J, Renouf DJ, Schaeffer DF, Jones SJM. Genomic characterization of a well-differentiated grade 3 pancreatic neuroendocrine tumor. Cold Spring Harbor Molecular Case Studies. 2019;5:1–14. doi: 10.1101/mcs.a003814.</cite> [<a href="https://doi.org/10.1101/mcs.a003814" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6549554/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31160355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harbor%20Molecular%20Case%20Studies&amp;title=Genomic%20characterization%20of%20a%20well-differentiated%20grade%203%20pancreatic%20neuroendocrine%20tumor&amp;author=LM%20Williamson&amp;author=M%20Steel&amp;author=JK%20Grewal&amp;author=ML%20Thibodeau&amp;author=EY%20Zhao&amp;volume=5&amp;publication_year=2019&amp;pages=1-14&amp;pmid=31160355&amp;doi=10.1101/mcs.a003814&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib50">
<cite>Xuan C, Wang Q, Han X, Duan Y, Li L, Shi L, Wang Y, Shan L, Yao Z, Shang Y. RBB, a novel transcription repressor, represses the transcription of HDM2 oncogene. Oncogene. 2013;32:3711–3721. doi: 10.1038/onc.2012.386.</cite> [<a href="https://doi.org/10.1038/onc.2012.386" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22926524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=RBB,%20a%20novel%20transcription%20repressor,%20represses%20the%20transcription%20of%20HDM2%20oncogene&amp;author=C%20Xuan&amp;author=Q%20Wang&amp;author=X%20Han&amp;author=Y%20Duan&amp;author=L%20Li&amp;volume=32&amp;publication_year=2013&amp;pages=3711-3721&amp;pmid=22926524&amp;doi=10.1038/onc.2012.386&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="bib51">
<cite>Zickler D, Kleckner N. Meiotic chromosomes: integrating structure and function. Annual Review of Genetics. 1999;33:603–754. doi: 10.1146/annurev.genet.33.1.603.</cite> [<a href="https://doi.org/10.1146/annurev.genet.33.1.603" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10690419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Genetics&amp;title=Meiotic%20chromosomes:%20integrating%20structure%20and%20function&amp;author=D%20Zickler&amp;author=N%20Kleckner&amp;volume=33&amp;publication_year=1999&amp;pages=603-754&amp;pmid=10690419&amp;doi=10.1146/annurev.genet.33.1.603&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section></section><article class="sub-article" id="sa0"><section class="pmc-layout__citation font-secondary font-xs"><div>eLife. doi: <a href="https://doi.org/10.7554/eLife.96052.4.sa0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.96052.4.sa0</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>eLife Assessment</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Williams%20CJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Carmen J Williams</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Carmen J Williams</span></h3>
<div class="p">
<sup>1</sup>National Institute of Environmental Health Sciences, United States</div>
<div>Reviewing Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Williams%20CJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Carmen J Williams</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.c" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a.c" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.c" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.c" class="d-panel p" style="display: none">
<div class="p" id="aff11">
<sup>1</sup>National Institute of Environmental Health Sciences, United States</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Carmen J Williams</span></strong>: <span class="role">Reviewing Editor</span>
</div>
</div>
<div id="anp_a.c" class="d-panel p" style="display: none"><div class="notes p">
<section id="kwd-group3" class="kwd-group"><p><strong>Keywords:</strong> Solid</p></section><section id="kwd-group4" class="kwd-group"><p><strong>Keywords:</strong> Valuable</p></section>
</div></div>
<div id="clp_a.c" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>This study provides <strong>valuable</strong> information on a novel gene that regulates meiotic progression in both male and female meiosis. The evidence supporting the conclusions of the authors is <strong>solid</strong>. This study will be of interest to developmental and reproductive biologists.</p></section></article><article class="sub-article" id="sa1"><section class="pmc-layout__citation font-secondary font-xs"><div>eLife. doi: <a href="https://doi.org/10.7554/eLife.96052.4.sa1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.96052.4.sa1</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Reviewer #1 (Public review):</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name given-only">Anonymous</span></a><div hidden="hidden" id="id16">
<h3><span class="name given-only">Anonymous</span></h3>
<div>Reviewer</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link"><span class="name given-only">Anonymous</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.d" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.d" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.d" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name given-only">Anonymous</span></strong>: <span class="role">Reviewer</span>
</div>
</div>
<div id="clp_a.d" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>Summary:</p>
<p>In this manuscript, the authors investigate the role of BEND2, a novel regulator of meiosis, in both male and female fertility. Huang et al have created a mouse model where the full-length BEND2 transcript is depleted but the truncated BEND2 version remains. This mouse model is fertile, and the authors used it to study the role of BEND2 on both male and female meiosis. Overall, the full-length BEND2 appears dispensable for male meiosis. The more interesting phenotype was observed in females. Females exhibit a lower ovarian reserve suggesting that full-length BEND2 is involved in the establishment of the primordial follicle pool.</p>
<p>Strengths:</p>
<p>The authors generated a mouse model that enabled them to study the role of BEND2 in meiosis. The role of BEND2 in female fertility is novel and enhances our knowledge of genes involved in the establishment of the primordial follicle pool.</p>
<p>Weaknesses highlighted previously:</p>
<p>The manuscript extensively explores the role of BEND2 in male meiosis; however, a more interesting result was obtained from the study of female mice.</p></section></article><article class="sub-article" id="sa2"><section class="pmc-layout__citation font-secondary font-xs"><div>eLife. doi: <a href="https://doi.org/10.7554/eLife.96052.4.sa2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.96052.4.sa2</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Reviewer #2 (Public review):</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name given-only">Anonymous</span></a><div hidden="hidden" id="id17">
<h3><span class="name given-only">Anonymous</span></h3>
<div>Reviewer</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link"><span class="name given-only">Anonymous</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.e" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.e" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.e" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name given-only">Anonymous</span></strong>: <span class="role">Reviewer</span>
</div>
</div>
<div id="clp_a.e" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>In their manuscript entitled "BEND2 is a crucial player in oogenesis and reproductive aging", the authors present their findings that full-length BEND2 is important for repair of meiotic double strand break repair in spermatocytes, regulation of LINE-1 elements in spermatocytes, and proper oocyte meiosis and folliculogenesis in females. The manuscript utilizes an elegant system to specifically ablate the full-length form of BEND2 which has been historically difficult to study due to its location on the X chromosome and male sterility of global knockout animals.</p>
<p>The authors have been extremely responsive to reviewer critiques and have presented strong data and appropriate conclusions, making it an excellent addition to the field.</p></section></article><article class="sub-article" id="sa3"><section class="pmc-layout__citation font-secondary font-xs"><div>eLife. doi: <a href="https://doi.org/10.7554/eLife.96052.4.sa3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.96052.4.sa3</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Reviewer #3 (Public review):</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name given-only">Anonymous</span></a><div hidden="hidden" id="id18">
<h3><span class="name given-only">Anonymous</span></h3>
<div>Reviewer</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anonymous%22%5BAuthor%5D" class="usa-link"><span class="name given-only">Anonymous</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.f" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.f" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.f" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name given-only">Anonymous</span></strong>: <span class="role">Reviewer</span>
</div>
</div>
<div id="clp_a.f" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>Huang et al. investigated the phenotype of Bend2 mutant mice which expressed truncated isoform. Bend2 deletion in male showed fertility and this enabled them to analyze the BEND2 function in females. They showed that Bend2 deletion in females showed decreasing follicle number which may lead to loss of ovarian reserve.</p>
<p>Strengths:</p>
<p>They found the truncated isoform of Bend2 and the depletion of this isoform showed decreasing follicle number at birth.</p>
<p>Weaknesses highlighted previously:</p>
<p>The authors showed novel factors that impact ovarian reserve. Although the number of follicles and conception rate are reduced in mutant mice, the in vitro fertilization rate is normal and follicles remain at 40 weeks of age. It is difficult to know how critical this is when applied to the human case.</p>
<p>[Editors' note: We thank the authors for considering the previous recommendations and suggested corrections.]</p></section></article><article class="sub-article" id="sa4"><section class="pmc-layout__citation font-secondary font-xs"><div>eLife. 2025 Aug 20;13:RP96052. doi: <a href="https://doi.org/10.7554/eLife.96052.4.sa4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.7554/eLife.96052.4.sa4</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Author response</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Yan Huang</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Yan Huang</span></h3>
<div class="p">
<sup>1</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yan Huang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bucevic%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Nina Bucevic</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Nina Bucevic</span></h3>
<div class="p">
<sup>2</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bucevic%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nina Bucevic</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Coves%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id21"><span class="name western">Carmen Coves</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Carmen Coves</span></h3>
<div class="p">
<sup>3</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Coves%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Carmen Coves</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Felipe-Medina%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id22"><span class="name western">Natalia Felipe-Medina</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Natalia Felipe-Medina</span></h3>
<div class="p">
<sup>4</sup>Spanish National Research Council, Salamanca, Spain</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Felipe-Medina%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Natalia Felipe-Medina</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marcet-Ortega%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id23"><span class="name western">Marina Marcet-Ortega</span></a><div hidden="hidden" id="id23">
<h3><span class="name western">Marina Marcet-Ortega</span></h3>
<div class="p">
<sup>5</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marcet-Ortega%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marina Marcet-Ortega</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nikou%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id24"><span class="name western">Nikoleta Nikou</span></a><div hidden="hidden" id="id24">
<h3><span class="name western">Nikoleta Nikou</span></h3>
<div class="p">
<sup>6</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nikou%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nikoleta Nikou</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Madrid-Sand%C3%ADn%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id25"><span class="name western">Cristina Madrid-Sandín</span></a><div hidden="hidden" id="id25">
<h3><span class="name western">Cristina Madrid-Sandín</span></h3>
<div class="p">
<sup>7</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Madrid-Sand%C3%ADn%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Cristina Madrid-Sandín</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22L%C3%B3pez-Panad%C3%A9s%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id26"><span class="name western">Maria López-Panadés</span></a><div hidden="hidden" id="id26">
<h3><span class="name western">Maria López-Panadés</span></h3>
<div class="p">
<sup>8</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22L%C3%B3pez-Panad%C3%A9s%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maria López-Panadés</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Buza%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id27"><span class="name western">Carolina Buza</span></a><div hidden="hidden" id="id27">
<h3><span class="name western">Carolina Buza</span></h3>
<div class="p">
<sup>9</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Buza%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Carolina Buza</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferrer%20Miralles%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id28"><span class="name western">Neus Ferrer Miralles</span></a><div hidden="hidden" id="id28">
<h3><span class="name western">Neus Ferrer Miralles</span></h3>
<div class="p">
<sup>10</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferrer%20Miralles%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Neus Ferrer Miralles</span></a>
</div>
</div>
<sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Iborra%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id29"><span class="name western">Antoni Iborra</span></a><div hidden="hidden" id="id29">
<h3><span class="name western">Antoni Iborra</span></h3>
<div class="p">
<sup>11</sup>SCAC, Universitat Autònoma de Barcelona (UAB), Bellaterra, Spain, Barcelona, Spain</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Iborra%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Antoni Iborra</span></a>
</div>
</div>
<sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pujol%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id30"><span class="name western">Anna Pujol</span></a><div hidden="hidden" id="id30">
<h3><span class="name western">Anna Pujol</span></h3>
<div class="p">
<sup>12</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pujol%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anna Pujol</span></a>
</div>
</div>
<sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pend%C3%A1s%20AM%22%5BAuthor%5D" class="usa-link" aria-describedby="id31"><span class="name western">Alberto M Pendás</span></a><div hidden="hidden" id="id31">
<h3><span class="name western">Alberto M Pendás</span></h3>
<div class="p">
<sup>13</sup>Centro de Investigacion del Cácner, Salamanca, Spain</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pend%C3%A1s%20AM%22%5BAuthor%5D" class="usa-link"><span class="name western">Alberto M Pendás</span></a>
</div>
</div>
<sup>13</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roig%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id32"><span class="name western">Ignasi Roig</span></a><div hidden="hidden" id="id32">
<h3><span class="name western">Ignasi Roig</span></h3>
<div class="p">
<sup>14</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div>Author</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Roig%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ignasi Roig</span></a>
</div>
</div>
<sup>14</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.g" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a.g" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.g" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.g" class="d-panel p" style="display: none">
<div class="p" id="aff12">
<sup>1</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div id="aff13">
<sup>2</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div id="aff14">
<sup>3</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div id="aff15">
<sup>4</sup>Spanish National Research Council, Salamanca, Spain</div>
<div id="aff16">
<sup>5</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div id="aff17">
<sup>6</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div id="aff18">
<sup>7</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div id="aff19">
<sup>8</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div id="aff20">
<sup>9</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div id="aff21">
<sup>10</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div id="aff22">
<sup>11</sup>SCAC, Universitat Autònoma de Barcelona (UAB), Bellaterra, Spain, Barcelona, Spain</div>
<div id="aff23">
<sup>12</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<div id="aff24">
<sup>13</sup>Centro de Investigacion del Cácner, Salamanca, Spain</div>
<div id="aff25">
<sup>14</sup>Autonomous University of Barcelona, Cerdanyola del Vallès, Spain</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Yan Huang</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Nina Bucevic</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Carmen Coves</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Natalia Felipe-Medina</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Marina Marcet-Ortega</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Nikoleta Nikou</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Cristina Madrid-Sandín</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Maria López-Panadés</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Carolina Buza</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Neus Ferrer Miralles</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Antoni Iborra</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Anna Pujol</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Alberto M Pendás</span></strong>: <span class="role">Author</span>
</div>
<div>
<strong class="contrib"><span class="name western">Ignasi Roig</span></strong>: <span class="role">Author</span>
</div>
</div>
<div id="anp_a.g" class="d-panel p" style="display: none"><div class="notes p"><section id="historyfront-stub5" class="history"><p>Collection date 2025.</p></section></div></div>
<div id="clp_a.g" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>The following is the authors’ response to the previous reviews.</p>
<blockquote class="text-italic">
<p>
<strong>Public Reviews:</strong>
</p>
<p>
<strong>Reviewer #1 (Public review):</strong>
</p>
<p>Summary:</p>
<p>In this manuscript, the authors investigate the role of BEND2, a novel regulator of meiosis, in both male and female fertility. Huang et al have created a mouse model where the full-length BEND2 transcript is depleted but the truncated BEND2 version remains. This mouse model is fertile, and the authors used it to study the role of BEND2 on both male and female meiosis. Overall, the full-length BEND2 appears dispensable for male meiosis. The more interesting phenotype was observed in females. Females exhibit a lower ovarian reserve suggesting that full-length BEND2 is involved in the establishment of the primordial follicle pool.</p>
<p>Strengths:</p>
<p>The authors generated a mouse model that enabled them to study the role of BEND2 in meiosis. The role of BEND2 in female fertility is novel and enhances our knowledge of genes involved in the establishment of the primordial follicle pool.</p>
<p>Weaknesses:</p>
<p>The manuscript extensively explores the role of BEND2 in male meiosis; however, a more interesting result was obtained from the study of female mice.</p>
</blockquote>
<p>We sincerely appreciate the reviewer’s thoughtful evaluation of our work and recognition of the strengths of our study. We are especially grateful for the acknowledgment of the novelty of our findings regarding the role of BEND2 in female fertility. While we extensively characterized the e ects of BEND2 depletion in male meiosis, we agree that the phenotype observed in females provides particularly interesting insights into the establishment of the primordial follicle pool.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #2 (Public review):</strong>
</p>
<p>In their manuscript entitled "BEND2 is a crucial player in oogenesis and reproductive aging", the authors present their findings that full-length BEND2 is important for repair of meiotic double strand break repair in spermatocytes, regulation of LINE-1 elements in spermatocytes, and proper oocyte meiosis and folliculogenesis in females. The manuscript utilizes an elegant system to specifically ablate the full-length form of BEND2 which has been historically di icult to study due to its location on the X chromosome and male sterility of global knockout animals.</p>
<p>The authors have been extremely responsive to reviewer critiques and have presented strong data and appropriate conclusions, making it an excellent addition to the field.</p>
</blockquote>
<p>We are truly grateful for the reviewer’s thoughtful review and recognition of the key contributions of our study. We appreciate the acknowledgment of how our model overcomes the challenges in studying BEND2 and the importance of our findings in both male and female meiosis. We also value the reviewer’s encouraging comments on our responsiveness to their feedback and the quality of our data and conclusions.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #3 (Public review):</strong>
</p>
<p>Huang et al. investigated the phenotype of Bend2 mutant mice which expressed truncated isoform. Bend2 deletion in male showed fertility and this enabled them to analyze the BEND2 function in females. They showed that Bend2 deletion in females showed decreasing follicle number which may lead to loss of ovarian reserve.</p>
<p>Strengths:</p>
<p>They found the truncated isoform of Bend2 and the depletion of this isoform showed decreasing follicle number at birth.</p>
<p>Weaknesses:</p>
<p>The authors showed novel factors that impact ovarian reserve. Although the number of follicles and conception rate are reduced in mutant mice, the in vitro fertilization rate is normal and follicles remain at 40 weeks of age. It is difficult to know how critical this is when applied to the human case.</p>
</blockquote>
<p>We greatly appreciate the reviewer’s comments and recognition of the strengths of our work. We are grateful for their acknowledgment of our findings related to the truncated isoform of Bend2 and its e ect on ovarian reserve. We also agree that, although our study provides important insights, we are still far from directly applying these results to human clinical scenarios. There is much further research needed before these findings can be translated.</p>
<blockquote class="text-italic">
<p>
<strong>Recommendations for the authors:</strong>
</p>
<p>
<strong>Reviewer #1 (Recommendations for the authors)::</strong>
</p>
<p>The authors have addressed all concerns both editorially and experimentally. This is a very nice manuscript, and I congratulate the authors on their work.</p>
</blockquote>
<p>We sincerely appreciate your kind words and thoughtful review. Your feedback has been invaluable in improving our manuscript, and we are grateful for your time and effort. Thank you for your support and encouragement!</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #2 (Recommendations for the authors)::</strong>
</p>
<p>In Figure 3, graphs in panels C &amp; D have typos in the early zygotene column where it reads "zyotene".</p>
</blockquote>
<p>We appreciate your careful review and for pointing out the typos in Figure 4, which has been corrected in the new version of the manuscript.</p>
<blockquote class="text-italic">
<p>
<strong>Reviewer #3 (Recommendations for the authors):</strong>
</p>
<p>・Since there are two isoforms of Bend2, and the authors depleted one isoform, this is not suitable to use "full length" in the titles and in the manuscripts.</p>
</blockquote>
<p>We respectfully disagree with the reviewer’s comment. In our mouse model, we specifically remove the full-length isoform of Bend2. Therefore, we consider it appropriate to refer to it as such in the manuscript. Our results indicate that the full-length isoform is not required to complete meiotic prophase in males but is indispensable for setting up the ovarian reserve in females. We appreciate the reviewer’s input and are happy to clarify this point further if needed.</p>
<blockquote class="text-italic"><p>・Is there any reason why authors used 7 month old females for in vitro fertilization? It may not be recognized as aged mice but it seems a bit old to perform IVF especially when the ovarian reserve in mutant mice is decreased. If there is any reason, please clarify it. In addition, since the authors added IVF data, which showed similar fertilization ratio between control and mutant, the authors need to discuss why the litter size was decreased in mutant mice. It may be to strong to conclude "subfertility".</p></blockquote>
<p>We used 7-month-old females for IVF because this falls within the age range of the samples analyzed for ovarian reserve, with the oldest females being 8 months old. Regarding the apparent discrepancy between IVF results and litter size, we addressed this in the discussion section of the manuscript: 'Interestingly, our mutant oocyte quality analysis suggests that mature oocytes from mutant females are equally competent to develop into a blastocyst as control ones. These data suggest that the subfertility observed in Bend2 mutants may be due to errors in later developmental stages, such as implantation or organogenesis.' We appreciate the reviewer’s feedback and hope this clarification helps.</p></section></article><article class="sub-article" id="_ad93_"><section class="pmc-layout__citation font-secondary font-xs"><div></div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Associated Data</h1></hgroup><ul class="d-buttons inline-list"></ul>
<div class="d-panels font-secondary-light"></div>
<div></div>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
</div></section><section class="body sub-article-body"><section id="_adsm93_" lang="en" class="supplementary-materials"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>Figure 1—figure supplement 1—source data 1. Source images for <a href="#fig1s1" class="usa-link">Figure 1—figure supplement 1A</a>.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367297/bin/elife-96052-fig1-figsupp1-data1.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-96052-fig1-figsupp1-data1.zip</a><sup> (427.9KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><span>Figure 1—figure supplement 1—source data 2. Source images for <a href="#fig1s1" class="usa-link">Figure 1—figure supplement 1A</a>.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367297/bin/elife-96052-fig1-figsupp1-data2.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-96052-fig1-figsupp1-data2.zip</a><sup> (91.5KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><span>Figure 2—source data 1. Source images for <a href="#fig2" class="usa-link">Figure 2B</a>.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367297/bin/elife-96052-fig2-data1.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-96052-fig2-data1.zip</a><sup> (459.8KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p"><span>Figure 2—source data 2. Source images for <a href="#fig2" class="usa-link">Figure 2B</a>.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367297/bin/elife-96052-fig2-data2.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-96052-fig2-data2.zip</a><sup> (200KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p"><span>Figure 4—figure supplement 1—source data 1. Source images for <a href="#fig4" class="usa-link">Figure 4D</a>.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367297/bin/elife-96052-fig4-figsupp1-data1.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-96052-fig4-figsupp1-data1.zip</a><sup> (964.8KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p"><span>Figure 4—figure supplement 1—source data 2. Source images for <a href="#fig4" class="usa-link">Figure 4D</a>.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367297/bin/elife-96052-fig4-figsupp1-data2.zip" data-ga-action="click_feat_suppl" class="usa-link">elife-96052-fig4-figsupp1-data2.zip</a><sup> (102.2KB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p"><span>Supplementary file 1. Testis and body weight (TW/BW) ratio of male mice.</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367297/bin/elife-96052-supp1.docx" data-ga-action="click_feat_suppl" class="usa-link">elife-96052-supp1.docx</a><sup> (16.3KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="caption p"><span>MDAR checklist</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367297/bin/elife-96052-mdarchecklist1.docx" data-ga-action="click_feat_suppl" class="usa-link">elife-96052-mdarchecklist1.docx</a><sup> (61KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>All the data analyzed in this study are available within the manuscript and its supporting materials.</p></section></section></article><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from eLife are provided here courtesy of <strong>eLife Sciences Publications, Ltd</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.7554/eLife.96052"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/elife-96052.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (15.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12367297/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12367297/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367297%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367297/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12367297/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12367297/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40833257/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12367297/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40833257/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12367297/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12367297/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="RwlmhwtqgAOIznc7Q4dNisD2hv6Ly766q4TxPOzuMahK76HOXmJHVl8o2mAJXnqE">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
