
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Recovery of phenotypically sorted cells using droplet-digital microfluidics - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4B7BC8AF3DF7305B7BC004B1DDEA2.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="rscsd">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12281281/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Lab on a Chip">
<meta name="citation_title" content="Recovery of phenotypically sorted cells using droplet-digital microfluidics">
<meta name="citation_author" content="Zhiyang Deng">
<meta name="citation_author_institution" content="Department of Electrical and Computer Engineering, Concordia University, Montréal, Canada, Email: steve.shih@concordia.ca">
<meta name="citation_author_institution" content="Centre for Applied Synthetic Biology, Concordia University, Montréal, Canada">
<meta name="citation_author" content="James M Perry">
<meta name="citation_author_institution" content="Centre for Applied Synthetic Biology, Concordia University, Montréal, Canada">
<meta name="citation_author" content="Marian Weiss">
<meta name="citation_author_institution" content="VERAXA Biotech GmbH, Heidelberg, Germany">
<meta name="citation_author" content="Robert Genth">
<meta name="citation_author_institution" content="VERAXA Biotech GmbH, Heidelberg, Germany">
<meta name="citation_author" content="Alexis Autour">
<meta name="citation_author_institution" content="Institute of Bioengineering, School of Engineering, École Polytechnique Fédérale de Lausanne (EPFL), Lausanne, Switzerland, Email: christoph.merten@epfl.ch">
<meta name="citation_author" content="Christoph A Merten">
<meta name="citation_author_institution" content="Institute of Bioengineering, School of Engineering, École Polytechnique Fédérale de Lausanne (EPFL), Lausanne, Switzerland, Email: christoph.merten@epfl.ch">
<meta name="citation_author" content="Steve C C Shih">
<meta name="citation_author_institution" content="Department of Electrical and Computer Engineering, Concordia University, Montréal, Canada, Email: steve.shih@concordia.ca">
<meta name="citation_author_institution" content="Centre for Applied Synthetic Biology, Concordia University, Montréal, Canada">
<meta name="citation_publication_date" content="2025 Jul 22">
<meta name="citation_volume" content="25">
<meta name="citation_issue" content="17">
<meta name="citation_firstpage" content="4410">
<meta name="citation_doi" content="10.1039/d5lc00415b">
<meta name="citation_pmid" content="40693295">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12281281/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12281281/">
<meta name="description" content="Droplet microfluidics has become a ubiquitous and powerful tool for high-throughput phenotypic screening at the single-cell level. Large numbers of cells can be sorted for a variety of functions, including the secretion of antibodies with tailored ...">
<meta name="og:title" content="Recovery of phenotypically sorted cells using droplet-digital microfluidics">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Droplet microfluidics has become a ubiquitous and powerful tool for high-throughput phenotypic screening at the single-cell level. Large numbers of cells can be sorted for a variety of functions, including the secretion of antibodies with tailored ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12281281/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12281281">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1039/d5lc00415b"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12281281%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12281281/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12281281/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12281281/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-rscsd.png" alt="Royal Society of Chemistry logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Royal Society of Chemistry" title="Link to Royal Society of Chemistry" shape="default" href="https://doi.org/10.1039/d5lc00415b" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Lab Chip</button></div>. 2025 Jul 22;25(17):4410–4421. doi: <a href="https://doi.org/10.1039/d5lc00415b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1039/d5lc00415b</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Lab%20Chip%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lab%20Chip%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Lab%20Chip%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Lab%20Chip%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Recovery of phenotypically sorted cells using droplet-digital microfluidics<sup><a href="#fn1" class="usa-link">†</a></sup>
</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deng%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Zhiyang Deng</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Zhiyang Deng</span></h3>
<div class="p">
<sup>a</sup>Department of Electrical and Computer Engineering, Concordia University, Montréal, Canada, Email: steve.shih@concordia.ca</div>
<div class="p">
<sup>b</sup>Centre for Applied Synthetic Biology, Concordia University, Montréal, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deng%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhiyang Deng</span></a>
</div>
</div>
<sup>a,</sup><sup>b</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Perry%20JM%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">James M Perry</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">James M Perry</span></h3>
<div class="p">
<sup>b</sup>Centre for Applied Synthetic Biology, Concordia University, Montréal, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Perry%20JM%22%5BAuthor%5D" class="usa-link"><span class="name western">James M Perry</span></a>
</div>
</div>
<sup>b</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weiss%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Marian Weiss</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Marian Weiss</span></h3>
<div class="p">
<sup>c</sup>VERAXA Biotech GmbH, Heidelberg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weiss%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marian Weiss</span></a>
</div>
</div>
<sup>c</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Genth%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Robert Genth</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Robert Genth</span></h3>
<div class="p">
<sup>c</sup>VERAXA Biotech GmbH, Heidelberg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Genth%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Robert Genth</span></a>
</div>
</div>
<sup>c</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Autour%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Alexis Autour</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Alexis Autour</span></h3>
<div class="p">
<sup>d</sup>Institute of Bioengineering, School of Engineering, École Polytechnique Fédérale de Lausanne (EPFL), Lausanne, Switzerland, Email: christoph.merten@epfl.ch</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Autour%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexis Autour</span></a>
</div>
</div>
<sup>d</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Merten%20CA%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Christoph A Merten</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Christoph A Merten</span></h3>
<div class="p">
<sup>d</sup>Institute of Bioengineering, School of Engineering, École Polytechnique Fédérale de Lausanne (EPFL), Lausanne, Switzerland, Email: christoph.merten@epfl.ch</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Merten%20CA%22%5BAuthor%5D" class="usa-link"><span class="name western">Christoph A Merten</span></a>
</div>
</div>
<sup>d,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shih%20SCC%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Steve C C Shih</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Steve C C Shih</span></h3>
<div class="p">
<sup>a</sup>Department of Electrical and Computer Engineering, Concordia University, Montréal, Canada, Email: steve.shih@concordia.ca</div>
<div class="p">
<sup>b</sup>Centre for Applied Synthetic Biology, Concordia University, Montréal, Canada</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shih%20SCC%22%5BAuthor%5D" class="usa-link"><span class="name western">Steve C C Shih</span></a>
</div>
</div>
<sup>a,</sup><sup>b,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="affa">
<sup>a</sup>Department of Electrical and Computer Engineering, Concordia University, Montréal, Canada, Email: steve.shih@concordia.ca</div>
<div id="affb">
<sup>b</sup>Centre for Applied Synthetic Biology, Concordia University, Montréal, Canada</div>
<div id="affc">
<sup>c</sup>VERAXA Biotech GmbH, Heidelberg, Germany</div>
<div id="affd">
<sup>d</sup>Institute of Bioengineering, School of Engineering, École Polytechnique Fédérale de Lausanne (EPFL), Lausanne, Switzerland, Email: christoph.merten@epfl.ch</div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 28; Accepted 2025 Jul 7; Collection date 2025 Aug 19.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>This journal is © The Royal Society of Chemistry</div>
<p>This article is licensed under a <a href="https://creativecommons.org/licenses/by-nc/3.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution-Non Commercial 3.0 Unported Licence</a>. You can use material from this article in other publications without requesting further permissions from the RSC, provided that the correct acknowledgement is given and it is not used for commercial purposes.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12281281  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40693295/" class="usa-link">40693295</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Droplet microfluidics has become a ubiquitous and powerful tool for high-throughput phenotypic screening at the single-cell level. Large numbers of cells can be sorted for a variety of functions, including the secretion of antibodies with tailored properties. The recovery of cells from sorted droplets is still very poor compared to droplet sorting, usually being limited to around 50% of all sorted hits. Here, we present a fully integrated droplet-digital microfluidic platform for the isolation and the recovery of rare single cells and applied our system to antibody discovery. From our work, we have achieved an 18-fold increase in the recovery rate of individual cells and beads from droplets, as compared to conventional methods. We believe that the combination of high-throughput droplet generation with the on-demand control features of digital microfluidics will improve the number of characterized hits in single-cell -omics, antibody screens, directed evolution of enzymes, and beyond.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p>We introduce a droplet-digital microfluidic recovery system to collect single cells from pooled culture and transfer these droplets into a well-plate. The platform is shown for the recovery of single beads and rare antibody secreting cells.<a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12281281_d5lc00415b-ga.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ed3/12281281/4597e5d1178c/d5lc00415b-ga.jpg" loading="lazy" id="ga" height="378" width="698" alt="graphic file with name d5lc00415b-ga.jpg"></a></p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Single cell analysis has become a frequently used method for assessing genomes, transcriptomes, or proteomes which leads to important advances in developing cancer therapies,<sup><a href="#cit1" class="usa-link" aria-describedby="cit1">1–3</a></sup> antibody discovery,<sup><a href="#cit4" class="usa-link" aria-describedby="cit4">4,5</a></sup> and deciphering neural pathways and brain disorders.<sup><a href="#cit6" class="usa-link" aria-describedby="cit6">6,7</a></sup> Droplet microfluidics has gained increasing popularity for such studies,<sup><a href="#cit8" class="usa-link" aria-describedby="cit8">8,9</a></sup> given its high-throughput and multi-parameter measurement capabilities. Fluorescence-activated droplet sorting (FADS) has been frequently used as a high-throughput screening tool for single-cell studies.<sup><a href="#cit10" class="usa-link" aria-describedby="cit10">10–12</a></sup> FADS is known to apply less stress to the sorted cells, enabling higher cell viability for post-processing.<sup><a href="#cit13" class="usa-link" aria-describedby="cit13">13</a></sup> Additionally, encapsulating secreted analytes in droplets has been crucial for many genotype–phenotype applications like enzyme-based strain development and antibody discovery.<sup><a href="#cit12" class="usa-link" aria-describedby="cit12">12,14</a></sup> However, droplet-based systems can face a critical challenge – sample loss during the recovery of the contents from processed droplets.<sup><a href="#cit15" class="usa-link" aria-describedby="cit15">15–19</a></sup> Conventionally, recovering contents from droplets is a complex multi-step process. Clausell-Tormos <em>et al.</em> pooled cell-containing droplets in a centrifuge tube, broke the emulsion, and recovered the contents from those droplets. In this standard droplet recovery protocol, a recovery rate of 20–30% was reported after 24-hour incubation of the droplets.<sup><a href="#cit15" class="usa-link" aria-describedby="cit15">15</a></sup> Sun <em>et al.</em> reported an efficiency of 50% while sorting and collecting positive hits. Furthermore, only 60% of the collected droplets can be recovered for downstream analysis, contributing to an overall workflow efficiency of only 10–20%.<sup><a href="#cit17" class="usa-link" aria-describedby="cit17">17</a></sup> Gaa <em>et al.</em> used a commercial system to recover contents from sorted droplets into well plates and were only able to achieve a recovery rate of less than 53%.<sup><a href="#cit19" class="usa-link" aria-describedby="cit19">19</a></sup> The sample loss issue can be more daunting when working with sensitive and rare targets such as antibody-secreting B cells or circulating tumor cells (CTCs). Having a critical role in immunotherapy, B cells make up 10% to 20% of all lymphocytes. However, only 0.1% to 2% of isolated B cells can produce functional antigen-specific antibodies.<sup><a href="#cit20" class="usa-link" aria-describedby="cit20">20</a></sup> CTCs are informative for cancer studies; however, they are extremely rare, often less than 100 CTCs can be found per 10<sup>7</sup> leukocytes and 5 × 10<sup>9</sup> erythrocytes in 1 mL of whole blood.<sup><a href="#cit21" class="usa-link" aria-describedby="cit21">21</a></sup> Furthermore, since applications like clonal expansion,<sup><a href="#cit22" class="usa-link" aria-describedby="cit22">22</a></sup> single-cell profiling,<sup><a href="#cit23" class="usa-link" aria-describedby="cit23">23</a></sup> and cytokine detection<sup><a href="#cit24" class="usa-link" aria-describedby="cit24">24</a></sup> are downstream of cell sorting and recovery, the sample loss during collection can lead to loss of crucial data or significant time of experimental work. Hence, new methods for single-cell isolation and recovery of viable cells from processed droplets are in demand for advancing the field of single-cell analysis using microfluidics.</p>
<p>There are currently two methods for sample recovery, recovery from bulk (pooled) droplets<sup><a href="#cit25" class="usa-link" aria-describedby="cit25">25–27</a></sup> or from individual (single) droplets.<sup><a href="#cit28" class="usa-link" aria-describedby="cit28">28–30</a></sup> For pooled recovery, droplets are usually collected in a tube, followed by a break of emulsions to extract the contents.<sup><a href="#cit27" class="usa-link" aria-describedby="cit27">27</a></sup> This method is useful for recovering a large, processed population; however, there is a loss of resolution such that deconvoluting hits from different droplets is challenging and there is a risk of cross-contamination between droplets. Single-cell screens, such as antibody discovery campaigns,<sup><a href="#cit12" class="usa-link" aria-describedby="cit12">12</a></sup> could greatly benefit from droplet recovery, to directly match a droplet phenotype (<em>e.g.</em> strongest binding or functional effect) with a particular gene sequence of the corresponding hit. For example, Cole <em>et al.</em> have designed a continuous and programmable system integrating a droplet sorter with a cell printer for precise dispensing of single droplets, termed printed droplet microfluidics (PDM).<sup><a href="#cit28" class="usa-link" aria-describedby="cit28">28</a></sup> This system uses dielectrophoretic trapping to position individual droplets into nanowell arrays, facilitating high-throughput droplet collection. While the structured printing substrate facilitates high-throughput collection of cells, the fixed nanowell configuration creates limitations on subsequent cell culture and analysis. Another approach by Nan <em>et al.</em>, developed an on-demand droplet collection (ODC),<sup><a href="#cit29" class="usa-link" aria-describedby="cit29">29</a></sup> that enables continuous and quantitative collection of target droplets into microtubes mounted on three different rotary platforms. However, this platform is technically complex and still does not allow easy transfer of individual hits into any reservoir of choice, including <em>e.g.</em> industrially used microtiter plates. Josephides <em>et al.</em> reported the use of the Cyto-Mine droplet microfluidics, single-cell sorting and dispensing system.<sup><a href="#cit31" class="usa-link" aria-describedby="cit31">31</a></sup> This platform achieved high performance, with 98.93% accuracy in dispensing sorted droplets into microtiter plates and high cell viability after recovery. However, despite its effectiveness, Cyto-Mine is a closed, proprietary system with limited flexibility, making it difficult to modify or customize for specific applications or alternative workflows. Here, we introduce a newly constructed droplet-digital (D<sup>2</sup>) microfluidic platform for the isolation and recovery of rare single cells from processed droplets. In this method, single-plate digital microfluidics (DMF) is used in combination with a droplet channel layer. By integrating droplet-based microfluidics with DMF, droplets can be generated and manipulated in an on-demand and addressable manner.<sup><a href="#cit14" class="usa-link" aria-describedby="cit14">14</a></sup> Using the open system, we are able to address any part of the workflow where it is difficult to recover rare single cells from individual droplets and transfer them into conventional culture plates. The workflow consists of generating droplets and sorting the droplets based on fluorescence intensity, followed by collecting the contents of the sorted droplets with a μL-scale volume recovery buffer, in a controllable, precise, and automated manner. To demonstrate the utility of our platform, we have performed recovery experiments with single beads, single HEK 293 cells, and single specific antibody-secreting hybridoma cells at typical and low ratios<sup><a href="#cit20" class="usa-link" aria-describedby="cit20">20</a></sup> to model rare events, establishing it as a versatile tool for a range of single-cell studies.</p></section><section id="sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="sec3"><h3 class="pmc_sec_title">Reagents, materials, and equipment</h3>
<section id="sec4"><h4 class="pmc_sec_title">Microfabrication materials</h4>
<p>Microfluidic device fabrication materials included a high resolution 25 400 dpi transparent photomask (Artnet Pro Inc., Bandon, OR, USA), AZ-1500 positive photoresist coated chromium glass substrates (Telic, Valencia, CA, USA), Si wafers (Silicon Valley Microelectronics Inc., Santa Clara, CA, USA), MF-321 developer, CR-4 chromium etchant (OM Group, Cleveland, OH, USA), AZ-300 T photoresist stripper (Integrated Micro Materials, Argyle, TX, USA), SU-82075 photoresists and SU-8 developer (Kayaku Advanced Materials, Westborough, MA, USA), polydimethylsiloxane (PDMS) (DOW Silicones Corporation, Midland, MI, USA), chlorotrimethylsilane (Sigma-Aldrich, Oakville, ON, Canada) and polylactic acid (PLA) (Shop3D, Mississauga, ON, CA). General-use solvents and chemicals used for microfluidic chip fabrication, including acetone (cleanroom lab grade) and isopropanol (IPA, cleanroom lab grade), were purchased from Sigma-Aldrich (Oakville, ON, Canada). De-ionized (DI) water has a resistivity of 18 MΩ cm at 25 °C.</p>
<p>PDMS surface treatment and device operation reagents included 3 M Novec HFE 7500 engineering fluid (M.G. Chemicals, Burlington, ON, Canada), 008-fluoro-surfactant dissolved in HFE7500 (20 g of 5% wt) (RAN Biotechnologies, Beverly, MA, USA), Aquapel (Amazon, Canada) and OptiPrep density gradient medium (Sigma-Aldrich, Canada).</p></section><section id="sec5"><h4 class="pmc_sec_title">Microfabrication equipment</h4>
<p>Microfabrication equipment included a Tergeo plasma cleaner (PIE Scientific LLC, Union City, CA, USA), a Quintel Q-4000 mask aligner (Neutronix Quintel, Morgan Hill, CA, USA), a Laurell spin coater (model WS-650Mz-8NPPB, Laurell Technologies Corporation, North Wales, PA, USA), and a UV-KUB 3 mask aligner (KLOÉ, Montpellier, FRA).</p></section><section id="sec6"><h4 class="pmc_sec_title">Cell culture and antibody screening reagents</h4>
<p>The cell culture medium was purchased from Wisent Inc. (Saint-Jean-Baptiste, QC, Canada). Cell culture reagents include Iscove's modified Dulbecco's media (IMDM) for culturing hybridoma cells, IMDM with no phenol red, Dulbecco's modified Eagle's medium (DMEM) for the target cells, fetal bovine serum (FBS), recombinant mouse IL-6 (carrier-free), Geneticin (G418), penicillin/streptomycin and phosphate buffer saline (PBS) (Ca<sup>2+</sup>/Mg<sup>2+</sup> free). FluoSpheres™ polystyrene microspheres (10 μm, blue-green fluorescent (430/465)), Alexa Fluor™ 488 goat anti-mouse IgG (H+L) and CellTrace™ calcein red-orange (AM) dyes, and SYTOX™ blue dead cell stain were purchased from Thermo Fisher Scientific (Mississauga, ON, Canada). The cell dissociation solution (non-enzymatic, 1×) was purchased from Sigma-Aldrich (Oakville, ON, Canada).</p></section><section id="sec7"><h4 class="pmc_sec_title">Fluidic system</h4>
<p>Gastight 500 μL and 2.5 mL glass syringes were purchased from Hamilton (Reno, NV, USA). Fittings and tubing: 1/4–28 × 10–32 PEEK adapter, 10–32 PEEK union assembly, finger tight micro ferrule 10–32 coned for 1/32′′ OD and PEEK tubing (1/32′′ diameter) were purchased from IDEX Corporation (Lake Forest, Illinois, USA). BD 1 mL and 3 mL syringes were purchased from Fisher Scientific (Waltham, MA, USA). Tygon tubing was purchased from VWR (Radnor, PA, USA). Stainless steel connection needles for the BD plastic syringes were purchased from McMaster-CARR (Elmhurst, Illinois, USA). A low-pressure Nemesys pump system (Cetoni, Korbussen, DE) was used for flow control.</p></section><section id="sec8"><h4 class="pmc_sec_title">Optical setup, data acquisition and control system</h4>
<p>The construction of the electrode control system has been previously published by our lab<sup><a href="#cit10" class="usa-link" aria-describedby="cit10">10,14</a></sup> and others.<sup><a href="#cit32" class="usa-link" aria-describedby="cit32">32</a></sup> Briefly, the hardware requirements contain a function generator (33210A, Keysight Technologies, Santa Rosa, CA, USA), an amplifier (PZD700A, Advanced Energy Inc, Denver, CO, USA), a DC power supply (GPE-4323, GW Instek America Corp, Montclair, CA, USA), a stacked control board of optocoupler switches, and a microcontroller (Teensy 4.1, Sherwood, OR, USA). The microcontroller was connected to a port expander that delivers the potential to the contact pads of the device through optocoupler switches. Droplet manipulations were achieved by application of AC potentials (140 V<sub>rms</sub>) at 15 kHz. The optical setup for sorting consisted of an Olympus IX73 inverted microscope (Evident Scientific, Mississauga, ON, Canada) mounted on a vibration-dampening bench (Thorlabs, Newton, NJ, USA), a photon counting head, a filter block, a SMA connector (Hamamatsu Photonics <a href="https://www.ncbi.nlm.nih.gov/nuccore/H11890" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">H11890</a>, Township, NJ, USA), an X-Cite XLED1 multi-triggering LED illumination system (Excelitas Technologies Corp., Waltham, MA, USA), bandpass filters, an attenuator filter, fiber optics (Thorlabs, Newton, NJ, USA), and a Hamamatsu ORCA-Flash 4.0 camera (Township, NJ, USA). Fluorescence images of the single cells in well plates were taken using an EVOS M5000 imaging system (Thermo Fisher Scientific, Waltham, MA, USA). Spectrometry data acquisition, PMT control, and droplet actuation were performed through a USB interface using our Python-based graphical user interface, as described in previous work.<sup><a href="#cit14" class="usa-link" aria-describedby="cit14">14</a></sup> The software is open-source and available to download at <a href="https://bitbucket.org/shihmicrolab/uflowcontrol" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bitbucket.org/shihmicrolab/uflowcontrol</a>. ESI<a href="#fn1" class="usa-link">†</a> Movie S1 shows the software interface presenting the operation of the software and live sorting of a single cell droplet.</p></section></section><section id="sec9"><h3 class="pmc_sec_title">Microfluidic device fabrication</h3>
<p>The fabrication of a droplet-digital (D<sup>2</sup>) microfluidic device followed a similar protocol to that in Ahmadi <em>et al.</em><sup><a href="#cit14" class="usa-link" aria-describedby="cit14">14</a></sup> The device consisted of three layers: a Cr-electrode layer, a dielectric layer, and a PDMS channel layer that were patterned by photolithography and soft-lithography methods. The first layer followed typical protocols for electrode fabrication.<sup><a href="#cit10" class="usa-link" aria-describedby="cit10">10,14,32,33</a></sup> Briefly, chromium-coated 50 × 75 mm glass slides (Telic, Valencia, CA, USA) with an S1811 positive resist were UV exposed (7 s at 38–50 mW cm<sup>−2</sup>), then developed in MF-321 developer, etched with a CR-4 chromium etchant, and stripped with an AZ-300 T photoresist stripper. For the dielectric layer, 1 mL of PDMS (1 : 10 w/w ratio, curing agent to prepolymer) was poured at the center of the Cr-patterned glass substrate, and the device was spin-coated at 4500 rpm for 90 s. Chromium contact pads were covered with Scotch 3 M tape prior to spin coating for easy removal of the excessive PDMS. The tape was removed after spin-coating and the device was baked at 90 °C for 1 h, resulting in a dielectric layer of ∼7 μm.</p>
<p>For the sorter PDMS channel layer, we used our previous fabrication protocol.<sup><a href="#cit10" class="usa-link" aria-describedby="cit10">10</a></sup> SU-8 2075 was spin-coated onto a 100 mm silicon wafer (10 s – 500 rpm, 30 s – 2000 rpm, 10 s – 500 rpm) to obtain a 100 μm thick layer, followed by a baking and exposure cycle according to the manufacturer's datasheet. A second layer (for the optical fibers) containing SU-8 2075 was spin-coated on top of the non-developed first layer (10 s – 500 rpm, 30 s – 2000 rpm, 10 s – 500 rpm) to obtain a 100 μm thick layer. After pre-exposure baking, the second layer mask was aligned and exposed (UV-KUB 3, KLOÉ, Montpellier, France), followed by baking and development according to the manufacturer's datasheet. The master mold was exposed to chlorotrimethylsilane vapour that was situated in a desiccator for 45 min, to prevent PDMS from permanently sticking to the mold. Next, PDMS (1 : 10 w/w ratio curing agent to prepolymer) was degassed and poured over the mold. To prevent PDMS shrinkage, the PDMS was incubated at room temperature to cure for at least 48 h. PDMS parts were cut to the designed size with a razor blade. Inlets for oil and aqueous phases were made using a 1.5 mm biopsy puncher (World Precision Instruments, FL, USA). The withdraw/waste outlet and recovery buffer inlet were made using a 0.75 mm biopsy puncher. The droplet-digital collection site was made with a 3 mm biopsy puncher and a semicircle was excised with a razor blade. 1.5 mm inlets were fitted with 0.06-inch OD Tygon transparent tubing and 0.75 mm inlets were fitted with 1/32′′ OD PEEK tubing. The PDMS was cleaned with Scotch tape to remove dust before assembly. The PDMS channel layer and the substrate were treated with oxygen plasma for 15 s, bonded together, and baked at 60 °C for 30 min. The PDMS channel containing the sorter channel was manually aligned with the dielectric-coated electrodes under a microscope. Device channels were then treated with Aquapel for 1 min, and then the device was air-dried and baked at 85 °C for 20 min. A flat, cleaved, multi-mode optical fiber (200 μm core, 0.49 N.A.) was prepared for droplet detection (Thorlabs, NJ, US). The fiber was fixed with Scotch tape on the microscope stage.</p></section><section id="sec10"><h3 class="pmc_sec_title">Experimental sample preparation</h3>
<section id="sec11"><h4 class="pmc_sec_title">HEK293 cell culture and preparation</h4>
<p>Two HEK293 cell lines, wild type (WT) and GFP-expressing variant, were donated by Dr. Elena Kuzmin and Dr. Alisa Piekny lab and cultured in DMEM supplemented with 10% heat-inactivated FBS, 1% penicillin–streptomycin (P/S) in a cell incubator at 37 °C and in 5% CO<sub>2</sub>. For cell maintenance, cells were passaged every two days at 80% confluency by washing with PBS and trypsinization, and then added to a final volume of 15 mL DMEM medium, in T-75 treated flasks.</p>
<p>For HEK cell experiments, the supernatant was discarded and cells were washed two times with 5 mL PBS. 2 mL of trypsin was added to the cell flask followed by 5 min incubation in a cell incubator in 5% CO<sub>2</sub> and at 37 °C. Next, 8 mL of DMEM (+ 10% FBS, 1% P/S) was added to the flask to inactivate trypsin and to wash the remaining cells off the bottom of the flask. 5 mL of the cell suspension was transferred to a 15 mL tube for centrifugation (1000 rpm, 5 min). Lastly, the supernatant in the tube was discarded and the cell pellet was resuspended in 5 mL of PBS and placed into a syringe.</p>
<p>For “spiked ratio” experiments, the GFP-expressing HEK cells and WT HEK cells were both adjusted to a final concentration of 1 × 10<sup>6</sup> cells per mL. The cell lines were mixed in two ratios (1 : 400 and 1 : 1000 – GFP : WT), at a total volume of 1 mL per tube. Next, to mimic low-volume rare cell inputs, tubes with 10 000 cells and tubes with 100 000 cells were prepared for different ratios and topped up to 400 μL with PBS. Lastly, 80 μL of OptiPrep was added to each 400 μL cell suspension tube, followed by a gentle vortex. For microfluidic sorting, the different ratios of cell suspensions (480 μL) were loaded into 1 mL BD plastic syringes. 600 μL of GFP-expressing and 600 μL of WT cells were loaded into 1 mL syringes as the positive and negative controls.</p></section><section id="sec12"><h4 class="pmc_sec_title">Cell culture and preparation of hybridoma and target cells</h4>
<p>Two hybridoma cell lines were provided by the National Research Council Canada (NRC) and were cultured in IMDM supplemented with 10% heat-inactivated FBS, mouse interleukin-6 (1 ng mL<sup>−1</sup>), and 1% penicillin–streptomycin in a cell incubator at 37 °C and in 5% CO<sub>2</sub>. Cells were passaged every three days at a final concentration of 8 × 10<sup>5</sup> cells per mL in 15 mL of medium in T-25 treated flasks. Cells were used for experiments when the cells were in the mid-logarithmic and under passage 12.</p>
<p>Target cells were provided by the NRC. These cells were cultured in DMEM (+ 10% FBS) and G418 (0.4 mg mL<sup>−1</sup>) in a cell incubator in 5% CO<sub>2</sub> and at 37 °C. Cells were passaged every three days at 80% confluency. Passaging and preparation followed the same protocol as above for hybridoma cells, except that a pre-warmed (37 °C) non-enzymatic cell-dissociation solution was used in lieu of trypsin to preserve the expression of the surface receptor antigen.</p>
<p>For sorting experiments, the cells were resuspended in the IMDM (no phenol red) cell culture medium containing 10% FBS and mouse interleukin-6 (1 ng mL<sup>−1</sup>) without antibiotics at a final concentration of 1 × 10<sup>6</sup> cells per mL. Specific-binding-antibody hybridoma producers were stained with CellTrace™ calcein-AM red-orange following the manufacturer's protocol. The target cells were resuspended in the IMDM (no phenol red) medium containing 10% FBS, mouse interleukin-6 (1 ng mL<sup>−1</sup>), and Alexa-488-conjugated goat-anti-mouse antibodies (2.5 μg mL<sup>−1</sup>) – <em>i.e.</em>, no antibiotic – at a final concentration of 3 × 10<sup>6</sup> cells per mL.</p></section><section id="sec13"><h4 class="pmc_sec_title">Preparation of fluorescent beads</h4>
<p>For experiments using fluorescent beads (blue-green fluorescent, 10 μm diameter), the beads were washed three times with PBS and diluted in PBS to 1 × 10<sup>6</sup> beads per mL before loading into a 1 mL syringe. 16% Optiprep was added to prevent beads from clumping.</p></section></section><section id="sec14"><h3 class="pmc_sec_title">D<sup>2</sup> microfluidic platform setup</h3>
<section id="sec15"><h4 class="pmc_sec_title">Bead and HEK cell experiments</h4>
<p>A D<sup>2</sup> device was used to perform bead and cell encapsulation, sorting, collection, and recovery. The chip included two inlets for the flow-focusing droplet generator, a pair of sorting electrodes, four pairs of collection electrodes, a fiber channel connected to a PMT, a withdraw/waste channel, a recovery medium channel, a waste channel, and a collection channel for sorted droplets. A syringe (2.5 mL gastight) was connected to a withdrawal port and another for the recovery buffer injection (500 μL gastight). Both were fitted with 1/32′′ OD, 0.127 mm ID PEEK tubing. A syringe with HFE oil (2.5 mL gastight) and with beads/cells (1 mL BD plastic) was fitted with 0.06-inch OD and 0.02-inch ID Tygon tubing. Syringes were installed on a low-pressure neMESYS pump system (Cetoni, Korbussen, DE) for automation control.</p>
<p>The optical fiber was inserted into the channel for the fiber on the device and was fixed in a 3D printed holder, and clamped in place with a pogo pin PCB providing contact with the electrode pads. The holder base plate was custom-made to fit on the scanning stage (XYZ Tango, Marzhauser, Wetzlar, DE) of an inverted epi-fluorescence microscope (Olympus IX73, Olympus, Montreal, QC, Canada). The fluorescent excitation light (<em>λ</em><sub>ex</sub> = 470 nm) was produced by an X-cite XLED (Ocean Optics, Ottawa, ON, Canada). The SMA end of the detection fiber was coupled to a photon counting head (Hamamatsu Photonics <a href="https://www.ncbi.nlm.nih.gov/nuccore/H11890" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">H11890</a>, Township, NJ, USA). The flow inside the microfluidic channel was observed under a 4× or 10× objective under bright-field illumination. The spectrometer (photon counting head), fluid flow, and electrode actuation were controlled using an in-house automation system and graphical user interface.<sup><a href="#cit14" class="usa-link" aria-describedby="cit14">14</a></sup></p></section><section id="sec16"><h4 class="pmc_sec_title">Hybridoma and target cell experiments</h4>
<p>A co-encapsulation chip (Fig. S1<a href="#fn1" class="usa-link">†</a>) and a D<sup>2</sup> microfluidic device were used to perform an antibody-producing hybridoma cell screening and recovery assay. The co-encapsulation device consisted of a flow-focusing generator with two cell inlets for hybridoma cells and target cells, respectively. For co-encapsulation, a syringe for withdrawal (3 mL BD plastic, for droplet collection and off-chip incubation), a syringe filled with HFE oil (2.5 mL gastight) and two syringes with cells (1 mL BD plastic) were fitted with 0.06-inch OD and 0.02-inch ID Tygon tubing. Syringes were installed on a low-pressure neMESYS pump system (Cetoni, Korbussen, DE). For sorting, collection and recovery, the setup was the same as described in the above section.</p></section></section><section id="sec17"><h3 class="pmc_sec_title">D<sup>2</sup> microfluidic operation for bead/cell recovery</h3>
<section id="sec18"><h4 class="pmc_sec_title">General workflow</h4>
<p>The D<sup>2</sup> microfluidic operation included droplet generation, sorting, collection and recovery. All the steps were performed with the in-house sorting software (uflow<sup><a href="#cit14" class="usa-link" aria-describedby="cit14">14</a></sup>). For bead and HEK cell recovery experiments, fluidic operation on the device followed four steps: first, a 2.5 mL syringe containing 2% 008-fluoro-surfactant HFE oil was connected to an oil inlet and was injected at a flow rate of 0.15 μL s<sup>−1</sup>. Second, after priming all the channels with oil and to ensure that all the droplets travel into the waste channel by default, a syringe for withdrawal was connected to the waste channel outlet and maintained at −0.15 μL s<sup>−1</sup>. Third, beads or cells were loaded into a 1 mL BD plastic syringe and introduced to the aqueous inlet of the chip at a flow rate of 0.05 μL s<sup>−1</sup>. Fourth, 300 μL recovery medium (PBS for beads, DMEM for HEK cells and IMDM for antibody-secreting cells) was loaded into a 500 μL gastight syringe, and the syringe was connected to the recovery medium channel inlet and injected at a flow rate of 1 μL s<sup>−1</sup> to make a 3 μL recovery medium.</p>
<p>After droplets were generated at ∼50 Hz, 470 nm wavelength light was cast at the detection fiber area. All the droplets prior to sorting were excited by the light and bead/cell-containing droplets emitted green fluorescence signals (∼520 nm); counts were recorded using the photon counting head. For single bead experiments, the sorting threshold was characterized and set between 7 × 10<sup>3</sup> counts and 2 × 10<sup>4</sup> counts (thresholding strategy shown in Fig. S2<a href="#fn1" class="usa-link">†</a>). For single-cell experiments, the sorting threshold was set between 1 × 10<sup>3</sup> counts and 2.5 × 10<sup>3</sup> counts. When the single bead/cell droplet passed the fiber region, a fluorescence peak signal was measured by the PMT system and shown on the uflowcontrol user interface (Fig. S2<a href="#fn1" class="usa-link">†</a>). If the fluorescence counts were above the threshold of droplets containing one target and below the threshold line of droplets with two or more targets, our automation system triggered the electrodes at the sorting junction to sort the single-bead/cell droplets. After each actuation of the sorting electrode, a 3 μL recovery medium was generated at a flow rate of 1 μL s<sup>−1</sup> and ‘parked’ at the sorting channel outlet. After a 7 s delay post-sorting, the sorted 1 nL droplet arrived at the sorting channel outlet and was in contact with the recovery medium <em>via</em> actuation by the software. The mixture was transferred into a 96-well plate using a P20 pipette, and the pipette was set to 5 μL to collect all the media. Each manual transfer operation, from the device to the well, took ∼10 seconds. The tubing method was similar to the D<sup>2</sup> workflow except that, after sorting, 6 cm Tygon tubing was connected to the sorting outlet and the end of the outlet tubing was directly guided to a well of a 96-well plate. For our device design, we incorporated six pairs of electrodes in the sorting channel to arrest the sorted droplets <em>via</em> actuation. ESI<a href="#fn1" class="usa-link">†</a> Movie S2 shows a sorted single-cell containing a droplet holding and merging with the recovery medium.</p></section><section id="sec19"><h4 class="pmc_sec_title">Fluorescent bead recovery</h4>
<p>Before the experiment, the well plate was filled with 100 μL per well of recovery medium (PBS). D<sup>2</sup> and tubing methods both followed the general workflow described above. For single recovery, 50 sorted droplets were collected and recovered into different individual wells. In the process of recovering 50 droplets into a well plate, 50 pipetting operations after merging the sorted droplet and the buffer were required using the D<sup>2</sup> method. For the tubing method, we manually moved the plate and guided the tubing outlet to individual wells after each sorting event. For pooled recovery, 50 sorted droplets were collected and recovered into one well together, followed by a limiting dilution to achieve one bead/cell per well.<sup><a href="#cit34" class="usa-link" aria-describedby="cit34">34</a></sup> In the process of recovering 50 droplets to a well plate, one pipetting operation was required using the D<sup>2</sup> method; for the tubing method, removing the tubing from the device outlet and aligning it vertically to allow constituents containing the droplets (and oil) to flow into a well were required. After experiments, the well plates were stored in a mammalian cell incubator until downstream analysis.</p></section><section id="sec20"><h4 class="pmc_sec_title">GFP HEK-293 cell recovery</h4>
<p>Before the experiment, the well plate was filled with 100 μL per well of recovery medium (DMEM). The D<sup>2</sup> method followed the general workflow and single recovery process described above. After experiments, the well plates were stored in a mammalian cell incubator until downstream analysis. For the viability test, 70 single cells were sorted/recovered into individual wells of a 96-well plate. A conditioned culture medium (150 μL, 30% supernatant from spent eGFP HEK cell culture filtered with a 0.22 μm filter and mixed with 70% fresh DMEM, 10% FBS and 1% penicillin–streptomycin added) was used for the first 24 hours post sorting, and a fresh medium (50 μL, 10% FBS and 1% penicillin–streptomycin added) was added every 48 hours to maintain the cells. From the starting time point (0 hour), after every 24 hours (within 7 days), SYTOX blue dead cell stain was added to 10 wells as per the manufacturer's protocol. The viability was calculated as the number of wells with non-blue cells per 10 wells. A dead cell stained with blue dye is shown in Fig. S3.<a href="#fn1" class="usa-link">†</a> The proliferation images were taken every 24 hours using an EVOS M5000 imaging system.</p></section></section><section id="sec21"><h3 class="pmc_sec_title">Antibody-secreting hybridoma cell screening and recovery</h3>
<p>Prior to experiments, cells were washed three times with neat IMDM to remove FBS from the media. Before cell encapsulation, we prepared a hybridoma cell mixture consisting of two cell lines: secreting monoclonal antibodies (mAbs) that bind to the receptor antigen expressed on the membrane of target cells and the other producing non-binding mAbs in 1 : 400 and 1 : 1000 ratios. The specific mAbs producing hybridoma cell and target cell lines were stained with calcein-AM red-orange (shown as red) and CellTrace blue (shown as blue), respectively, as per the manufacturer's instructions. The target cells (3 × 10<sup>6</sup> cells per mL<sup>−1</sup>) and goat anti-mouse Alexa Fluor 488 secondary antibodies (2.5 μ mL<sup>−1</sup>) along with Optiprep (11%, w/v) were introduced from one inlet in the co-encapsulation chip at a flow rate of 0.3 μL s<sup>−1</sup>. The hybridoma cell mixture (1 × 10<sup>6</sup> cells per mL) along with Optiprep (11%, w/v) was introduced from another inlet in the co-encapsulation chip at a flow rate of 0.3 μL s<sup>−1</sup>. HFE 7500 oil with 2% 008-fluoro-surfactant (RAN Biotechnologies) was used to produce droplets at a flow rate of 0.8 μL s<sup>−1</sup>. A withdrawing syringe (BD 1 mL plastic) was connected to the chip outlet and set at a flow rate of 1.4 μL s<sup>−1</sup> to collect droplets, and the syringe was incubated for 2 hours in a cell incubator at 37 °C and in 5% CO<sub>2</sub>. The droplets were re-injected into the sorting and recovery chip using our D<sup>2</sup> method at a flow rate of 0.02 μL s<sup>−1</sup>, the spacing oil was set at a flow rate of 0.15 μL s<sup>−1</sup> and the waste syringe was set at −0.12 μL s<sup>−1</sup>. The sorting and recovery were performed using in-house software (uflow) and the sorting threshold was set between 1.8 × 10<sup>4</sup> counts and 3 × 10<sup>4</sup> counts for the specific antibody-binding assay. The throughput of the sorting was ∼50 Hz with a monodisperse droplet size of ∼1 nL. After each sorting event, a 3 μL IMDM (10% FBS, 1 ng mL<sup>−1</sup> IL-6) recovery medium was generated at the collection channel outlet and a delay (7–9 seconds) was set before actuating the electrodes three times (500 ms per actuation) to merge the samples. The sorted droplet and recovery medium mixtures were individually transferred using a pipette to a 96-well plate for downstream analysis. The well plate was preloaded with 100 μL of IMDM (10% FBS, 1 ng mL<sup>−1</sup> IL-6, 1% penicillin–streptomycin) and immediately moved to an incubator after the experiment. The well plate was imaged using an EVOS M5000 imaging system.</p></section><section id="sec22"><h3 class="pmc_sec_title">Data processing and analysis</h3>
<p>Data and statistical analyses were performed using GraphPad Prism 10. The image of the outlet tubing was taken using a phone camera. The images of ‘stuck’ droplets were taken using an inverted microscope (Olympus IX73) with brightfield illumination. The images of single beads and cells were taken using an EVOS M5000 imaging system. Bead/cell counting was done using Fiji (Image J) in the following steps: 1) open the image in Fiji software, 2) adjust the brightness &amp; contrast by clicking ‘auto’, 3) apply a median filter to filter out noises (radius 5.0 pixels), and 4) find the maxima and output the number of the counts. The recovery rate was calculated by dividing the recovered (maxima) counts (Fiji) by the sorted counts (uflow):<a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12281281_d5lc00415b-t1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ed3/12281281/502b9c9bd1b3/d5lc00415b-t1.jpg" loading="lazy" id="ugt1" height="104" width="670" alt="graphic file with name d5lc00415b-t1.jpg"></a>The applied electrical signal was measured using an oscilloscope. All in-house codes were previously described by our group.<sup><a href="#cit14" class="usa-link" aria-describedby="cit14">14</a></sup></p></section><section id="sec23"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p>Statistical analysis was performed using descriptive statistics (mean, standard deviation) and an unpaired <em>t</em>-test (<em>p</em>-value) with Prism V10.1.1 (GraphPad) for <a href="#fig2" class="usa-link">Fig. 2–4</a>.</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">Fig. 2. Comparison between D<sup>2</sup> and tubing for multi-bead recovery. (A) Recovery rate of recovering <em>X</em> number of beads into one well after <em>X</em> sorting events. (<em>X</em> = 5, 10, 20, 50). (B) Images of the tubing outlet indicating that the sorted droplets were ‘trapped’ at the tubing ‘arc’. (C) An illustration of the PDMS–tubing interface (top) and an image of sorted droplets being ‘trapped’ at the connection junction (bottom). All error bars represent the ± range; ns indicates no significant difference. Statistical analysis was performed using an unpaired <em>t</em>-test.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12281281_d5lc00415b-f2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ed3/12281281/2512618d28bc/d5lc00415b-f2.jpg" loading="lazy" height="437" width="766" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">Fig. 3. Comparison between D<sup>2</sup> and traditional tubing technique for single-bead recovery. (A) Recovery rate of isolating/recovering 50 beads into 50 individual wells, recovering one droplet at a time to one well (single recovery). All error bars represent ± SD; *, **, ***, and **** represent <em>p</em>-values below 0.05, 0.01, 0.001 and 0.0001, respectively. Statistical analysis was performed using an unpaired <em>t</em>-test. (B) Images of nine wells filled with single beads from a 96-well plate, scale bar: 200 μm. Zoomed-in views of the recovered beads are shown in the white boxes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12281281_d5lc00415b-f3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ed3/12281281/0bd8bcdc73b9/d5lc00415b-f3.jpg" loading="lazy" height="281" width="767" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">Fig. 4. Low-input rare single-cell recovery using D<sup>2</sup>. (A) Recovery rate of D<sup>2</sup>, using different spike-in ratios. (B) The graph (left) shows the D<sup>2</sup> sorting threshold (green line) and sorting conditions (total cells, spike-in ratios). Images of six recovered single cells in a well plate (right), scale bar: 20 μm. (C) Single HEK cell viability over 7 days using D<sup>2</sup> (left). Bright-field and fluorescence images of 7-day single HEK cell proliferation, sorted and recovered by the D<sup>2</sup> system (right), scale bar: 100 μm.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12281281_d5lc00415b-f4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ed3/12281281/64f05dbd83ed/d5lc00415b-f4.jpg" loading="lazy" height="385" width="756" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section></section><section id="sec24"><h2 class="pmc_sec_title">Results and discussion</h2>
<section id="sec25"><h3 class="pmc_sec_title">Droplet-digital based microfluidics for sample collection</h3>
<p>Droplet-based microfluidics has been frequently used for single-cell studies; however, such technologies are challenging with collection and recovery, resulting in unwanted sample loss.<sup><a href="#cit29" class="usa-link" aria-describedby="cit29">29</a></sup> To address this, we devised a new droplet-digital (D<sup>2</sup>) microfluidic strategy which has been reported as an efficient platform for on-demand merging, mixing and sorting of the droplets – operations that were challenging with standard droplet-in-channel microfluidics.<sup><a href="#cit14" class="usa-link" aria-describedby="cit14">14</a></sup> Many cell-based workflows generally require sorting, and fluorescence-activated droplet sorting was reported as a powerful tool for this purpose<sup><a href="#cit35" class="usa-link" aria-describedby="cit35">35</a></sup> (<a href="#fig1" class="usa-link">Fig. 1A</a>). There are commercial systems (Cyto-Mine,<sup><a href="#cit31" class="usa-link" aria-describedby="cit31">31</a></sup> LiveDrop,<sup><a href="#cit36" class="usa-link" aria-describedby="cit36">36</a></sup> Atrandi Biosciences) that are effective in sorting and recovery; however, these are closed systems and are difficult to use for low-hit, rare, cell ratios (Table S1<a href="#fn1" class="usa-link">†</a>). The D<sup>2</sup> system can overcome these limitations using an open design for the recovery region, enabling direct pipetting of individually sorted droplets from the device to well plates.</p>
<figure class="fig xbox font-sm" id="fig1"><h4 class="obj_head">Fig. 1. Traditional system (FADS) and D<sup>2</sup> device for droplet sorting and recovery workflows. (A) Traditional droplet microfluidic system for sorting/recovering single-cell targets from a pooled cell population. All droplets fulfilling the sorting criteria are sorted into the same reservoir of hits, making their separation for individual analysis challenging and prone to sample loss. (B) D<sup>2</sup> device. The contact pads are connected to the in-house automation control system (top view). A zoomed-in top view of the semicircular outlet with electrodes for droplet collection (left) and side view of the device (right). Droplet collection starts with D<sup>2</sup> sorting (i) followed by D<sup>2</sup> droplet recovery (ii). The dashed line around the single cell indicates that the sorted droplet has been merged with the recovery media. These operations are shown as images from a movie: droplet generator (iii), DMF sorter (iv) and droplet merger between the single cell droplet with the buffer droplet (v).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12281281_d5lc00415b-f1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ed3/12281281/a02531bd9648/d5lc00415b-f1.jpg" loading="lazy" height="939" width="787" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>We have previously described a droplet-to-digital platform that uses two devices connected by a capillary to transfer droplets from the generator to the digital part of the platform for screening cells.<sup><a href="#cit37" class="usa-link" aria-describedby="cit37">37</a></sup> This technique, however, suffers from the requirement of tubing to transfer droplets, which can cause droplet losses and create difficulty to track a particular droplet. In addition, there is a mismatch in droplet numbers from generation (1000s) to droplet screening (only a few droplets at a time).</p>
<p>Here, we report a fully integrated droplet-digital screening platform, also being able to alleviate the droplet mismatch between the two systems. As shown in <a href="#fig1" class="usa-link">Fig. 1B(i and ii)</a>, are the steps for the droplet recovery. The D<sup>2</sup> microfluidic device is controlled by our in-house automation system (uflow, top view) and the device consists of three layers (side view). To ensure reliable droplet collection using the D<sup>2</sup> method, we started with (iii) encapsulation of cells/beads in ∼1 nL droplets, (iv) sorting of single cell/bead droplets using our fiber-based detection,<sup><a href="#cit10" class="usa-link" aria-describedby="cit10">10</a></sup> and (v) on-demand control to merge and to recover the sorted single-cell/bead-containing droplet. For the collection, the outlet was made with a 3 mm puncher and cut to a semicircular shape with one side opening as shown in the top view, offering direct access to pipetting samples from the collection channel. Sorted droplets were guided to the merging area by the oil flow of ∼0.05 μL s<sup>−1</sup>. Two 200 × 200 μm merging electrodes were patterned at the interface between the sorting channel and the merging area. The merging area contains a 3 μL recovery buffer that can merge with the sorted 1 nL droplet. After merging, these droplets were collected <em>via</em> manual pipetting and directly transferred to a well plate for downstream analysis. A key difference from previous droplet-digital interface work<sup><a href="#cit37" class="usa-link" aria-describedby="cit37">37</a></sup> is that our system reported here maximizes the capabilities of the droplet sorter and digital microfluidics to control the collection.</p></section><section id="sec26"><h3 class="pmc_sec_title">Comparison between D<sup>2</sup> and the conventional method for fluorescent bead recovery</h3>
<p>Pooling droplets allows for the collection and grouping of desired droplet samples for many different applications.<sup><a href="#cit10" class="usa-link" aria-describedby="cit10">10,12,18,25,38</a></sup> For example, Debs <em>et al.</em> used a pooling strategy to recover sorted droplets for functional single-cell hybridoma screening, and after sorting, the authors pooled cell-containing droplets and performed droplet breakup for secreted antibody measurements.<sup><a href="#cit25" class="usa-link" aria-describedby="cit25">25</a></sup> As an initial test, we compared the traditional collection with D<sup>2</sup> collection of fluorescent beads by pooling the sorted droplets. Droplet generation and sorting are the same for both techniques, except that the processed droplets are pooled and then transferred to a centrifuge tube or a well plate through tubing or using the on-demand D<sup>2</sup> collector. To maximize droplet collection with the traditional techniques, we followed other studies for collection in which protocols such as removing the tubing from the device after sorting and orienting the tubing to vertically drain the liquid to the collection tube by gravity.<sup><a href="#cit35" class="usa-link" aria-describedby="cit35">35</a></sup> As shown in <a href="#fig2" class="usa-link">Fig. 2A</a> is the recovery rate comparing the two different methods. We measured the number of pooled beads that can be recovered in the well-plate after the same number of sorting events. The D<sup>2</sup> method achieved an average collection recovery rate of ∼93.5% compared to 78.5% with the traditional tubing method. Moreover, the significant difference is in the variation of the recovery (as shown by the error bars). The traditional case showed a large range of recovery rates, especially for the higher number of beads, while the D<sup>2</sup> case showed more reproducible collection regardless of the number of events. The tubing-based collection method showed a high standard deviation of 26.1%, reflecting substantial inconsistencies in recovery, as opposed to the D<sup>2</sup> method, which showed a lower standard deviation of 5.29% (Table S2<a href="#fn1" class="usa-link">†</a>). These results describe that the D<sup>2</sup> system effectively eliminates the variability introduced by tubing collection. Particularly when collecting 50 beads, the D<sup>2</sup> system achieved a fivefold reduction in standard deviation, highlighting its robustness and consistency in sample handling. We speculate that the tubing configuration, as depicted in <a href="#fig2" class="usa-link">Fig. 2B and C</a>, creates an “arc” for transferring processed droplets downstream, leading to a static area where droplets can accumulate. The commonly used HFE oil is heavier than the aqueous droplets, and thus the droplets tend to remain at the top region of the tubing (<a href="#fig2" class="usa-link">Fig. 2B</a>). Additionally, in a conventional tubing setup for pooled transfer, the sorted droplets are in contact with each other, which occasionally results in droplet coalescence or cross-talk. In the current system, the droplets are isolated or “parked” individually, preventing any fusion or diffusive exchange. Moreover, when the number of sorted droplets is low, the flow rate is insufficient to displace them out of the tubing, leading to retention within the arc. Retention is also a problem at the interface between the PDMS and the tubing (<a href="#fig2" class="usa-link">Fig. 2C</a>). Since commonly used tubing materials (Tygon or PEEK) have a tubing wall thickness greater than the droplet diameter (&lt;100 μm), sorted droplets became trapped along the tubing wall. However, in the D<sup>2</sup> system, since no tubing is required, the above two causes of sample loss will be mitigated. The new system is capable of easily collecting the pooled droplets and transferring them to well-plates with high fidelity.</p>
<p>Instead of pooling our droplet samples, we evaluated similar steps for collecting and transferring contents from individual cell-containing droplets. Individual cell-containing droplet collection is commonly required for enrichment of desired products or mutants from a mixed population.<sup><a href="#cit39" class="usa-link" aria-describedby="cit39">39</a></sup> For single recovery, we recovered the contents from droplets after each sorting event using tubing or D<sup>2</sup> collection and transferred them into individual wells of a 96-well plate, resulting in one bead per well. Interestingly, we observed a significantly higher mean recovery rate (∼87%) than the tubing method (∼5%) for single bead recovery (<a href="#fig3" class="usa-link">Fig. 3A</a>). After collection, we acquired images of single cells in the well plate, confirming that we are able to transfer the single beads with our new method (<a href="#fig3" class="usa-link">Fig. 3B</a>).</p></section><section id="sec27"><h3 class="pmc_sec_title">Low-input rare single-cell recovery using the D<sup>2</sup> platform</h3>
<p>The recovery of low and rare cells is interesting for many applications,<sup><a href="#cit40" class="usa-link" aria-describedby="cit40">40</a></sup> but the collection of such cells has always been challenging.<sup><a href="#cit29" class="usa-link" aria-describedby="cit29">29</a></sup> We constructed a novel microfluidic system for high recovery rates, high viability cell sorting and recovery. Therefore, we conducted a series of experiments using our platform to sort cell-containing droplets and collect single cells in multi-well plates to assess the D<sup>2</sup> performance. By evaluating the recovery rates and post-sorting viability of GFP-expressing HEK 293 cells, we aimed to determine whether the D<sup>2</sup> method can recover the rare population.</p>
<p>For this experiment, GFP-expressing HEK 293 cells were prepared at two different cell numbers, 10<sup>4</sup> and 10<sup>5</sup> cells, and in each set, different ratios (1 : 400 and 1 : 1000) of GFP to WT cells were spiked. Briefly, we performed cell sorting and collection/recovery into a 96-well plate, and validated the recovery rates and viability changes within 7 days post sorting/recovery for the D<sup>2</sup> system. The comparative data describing the recovery rate for the GFP cells using the D<sup>2</sup> method are shown in <a href="#fig4" class="usa-link">Fig. 4A</a>. Compared to previously reported recovery rates from 20 to 30%,<sup><a href="#cit15" class="usa-link" aria-describedby="cit15">15</a></sup> the D<sup>2</sup> method shows excellent recovery rates of over 80% for both cell inputs (10<sup>4</sup>, 10<sup>5</sup>) and cell ratios (1 : 400, 1 : 1000). As shown in <a href="#fig4" class="usa-link">Fig. 4B</a> is an example spectrometer plot in the D<sup>2</sup> workflow; the green line indicates the sorting threshold and peaks above the threshold were sorted, counted, recovered and transferred to a well plate. Here, using our 1 : 1000 dilution ratio, six GFP target cells were sorted, counted and recovered during ∼1 h of operation. Images of recovered GFP cells in the well plates suggest that the D<sup>2</sup> method is suitable for collecting single cells for downstream analysis.</p>
<p>A unique strength of the D<sup>2</sup> method is that it uses low electric fields to sort and collect the cells, which minimizes the electric field-induced damage to the cells. Instead of using conventional dielectrophoresis (DEP) with injected electrodes,<sup><a href="#cit12" class="usa-link" aria-describedby="cit12">12,41</a></sup> we used the D<sup>2</sup> binary sorter by patterning electrodes beneath the channel, which requires lower operation voltage for sorting.<sup><a href="#cit42" class="usa-link" aria-describedby="cit42">42</a></sup> The DEP sorting systems usually require voltages ranging from 200 V to 1200 V,<sup><a href="#cit43" class="usa-link" aria-describedby="cit43">43,44</a></sup> while the D<sup>2</sup> sorter uses a lower voltage of less than 140 V that can preserve the cell's health. To test this, single HEK cells were sorted, recovered, and cultured in well plates. As shown in <a href="#fig4" class="usa-link">Fig. 4C</a>, we compared our results to previously reported post-sorting viability (∼95% for MCF-7 cells,<sup><a href="#cit45" class="usa-link" aria-describedby="cit45">45</a></sup> ∼79% for Jurkat cells and ∼90% for HEK293T cells<sup><a href="#cit15" class="usa-link" aria-describedby="cit15">15</a></sup>), and found that the cells that were collected with D<sup>2</sup> are similar in terms of viability (∼90%) and also showed proliferation over seven days. In addition, the cells collected by the D<sup>2</sup> method exhibit spreading, a positive indicator of viability, suggesting that the platform effectively preserves cell health post-isolation and is well-suited for downstream functional assays.</p></section><section id="sec28"><h3 class="pmc_sec_title">Application: rare antibody-secreting cell recovery</h3>
<p>Droplet-based microfluidics has been reported as a useful screening tool for finding specific antibody-secreting cells.<sup><a href="#cit12" class="usa-link" aria-describedby="cit12">12,19</a></sup> Hybridoma technology is one of the most common methods to produce monoclonal antibodies. It fuses antibody-secreting B cells with immortal myeloma cell lines to create a hybrid cell line that can be expanded to produce monoclonal antibodies.<sup><a href="#cit46" class="usa-link" aria-describedby="cit46">46</a></sup> However, the antigen-specific B cells in the peripheral blood of immunized human donors are very rare, and the ratio of such cells is usually in the range of 0.1–2%.<sup><a href="#cit20" class="usa-link" aria-describedby="cit20">20</a></sup> Therefore, recovery of these rare hits is critical, yet challenging. There is a droplet-based platform for directly identifying and isolating antibody-encoding genes from rare screening hits, but this setup does not allow recovery of live cells for downstream expansion.<sup><a href="#cit47" class="usa-link" aria-describedby="cit47">47</a></sup></p>
<p>To showcase the D<sup>2</sup> system's capability of recovering sorted rare targets, we performed a specific antibody binding assay. We prepared hybridoma cell line mixtures consisting of two cell lines: one producing specific-binding monoclonal antibodies (mAbs) that bind to the cell-surface-antigen-overexpressing target cell line, and the other producing non-specific mAbs which will not bind to the surface of the target cells. In these experiments, we mixed Alexa Fluor 488 secondary antibodies with a plain IMDM solution containing target cells to be able to investigate specific mAbs binding through fluorescence intensity detection. The specific and non-specific antibody-secreting hybridoma cells were mixed in ratios of 1 : 400 (0.25%) and 1 : 1000 (0.1%) to match the low ratio of specific antibody-secreting B cells. Only the specific hybridoma cells were stained with calcein AM red and the target cells were stained with CellTrace blue. The hybridoma mixture and target cells were then introduced into a co-encapsulation device to form double-cell droplets (5% to 10%, based on Poisson statistics), as shown in <a href="#fig5" class="usa-link">Fig. 5A</a>. Images of recovered cells in well plates and their corresponding green fluorescence intensities are shown in <a href="#fig5" class="usa-link">Fig. 5B</a>. In this workflow, we sorted based on the green fluorescence intensity. All droplets contain the same amount of fluorescent secondary antibodies. In the presence of a specific-antibody-secreting hybridoma cell and a target cell, there will be an accumulation of fluorescent secondary antibodies on the target cell surface, resulting in a sharp, high green fluorescence intensity peak within the droplet. Droplets with only antibody-secreting cells emitted medium fluorescence peaks and droplets with only target cells emitted background fluorescence intensity. The fluorescence threshold for the green channel was set at 18 000 photon counts, which correlates to the peaks generated by the specific antibody binding to the surface of target cells. Using this threshold, we were able to obtain specific-mAb-secreting hybridoma and target cell doublet combinations. We verified the number of recovered specific antibody-secreting hybridoma–target cell combinations in well plates with the number of sorted droplet counts in our software. The recovery rates of 1 : 400 and 1 : 1000 spike-in ratios are shown in <a href="#fig5" class="usa-link">Fig. 5C</a>. For the 1 : 400 ratio, we have a mean recovery rate of around 88%, and for the 1 : 1000 ratio, the mean recovery rate is around 78% and is generally better than the recovery rates of ∼30% for recovering cells from droplets reported in the literature.<sup><a href="#cit15" class="usa-link" aria-describedby="cit15">15</a></sup> The enhanced performance can be attributed to the on-demand, selective collection of droplets based on phenotypic or fluorescence readouts, which significantly reduces false positives and minimizes sample loss due to nonspecific or bulk collection. In contrast, typical commercial systems like Cyto-Mine rely on pre-set gating, passive flow valve-based mechanisms, which are inherently limited in collection precision and are susceptible to high levels of stochastic loss, particularly during droplet transfer <em>via</em> tubing or plate deposition.</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">Fig. 5. Rare specific-antibody-secreting cell recovery. (A) An illustration of the co-encapsulation chip. (B) Schematic showing the different cell combinations in droplets that correlate to the detected fluorescence signal (see arrows). A spectrometer plot showing different fluorescence intensity peaks. Sorting was triggered based on the threshold shown as the green line. Microscopy images of one hybridoma cell with one target cell, one hybridoma cell, and one target cell were taken at four different color channels to show the cell staining strategy: hybridoma Ab-producer (red), target cell (blue), and secondary antibody (green). Scale bars: 10 μm. (C) Plot showing the droplet recovery rate (%) for spiked ‘rare’ 1 : 400 and 1 : 1000 ratios (specific antibody-secreting cells : non-specific antibody-secreting cells). All error bars represent ± SD.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12281281_d5lc00415b-f5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8ed3/12281281/aa141bc312a3/d5lc00415b-f5.jpg" loading="lazy" height="482" width="756" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section></section><section id="sec29"><h2 class="pmc_sec_title">Conclusion</h2>
<p>We have reported a droplet-digital microfluidic platform for the efficient isolation of rare single cells in phenotypic assays. The platform can generate droplets, sort them based on fluorescence intensity, and collect the contents of the sorted droplets with a μL-scale volume recovery buffer, in a controllable and automated manner. To demonstrate the utility of our platform, we have performed recovery experiments with single fluorescent beads, single HEK 293 cells, and single specific antibody-secreting hybridoma cells at low ratios (1 : 400 to 1 : 1000). For these experiments, we were able to obtain significantly higher recovery rates (&gt;80%), as compared to previously published systems,<sup><a href="#cit15" class="usa-link" aria-describedby="cit15">15,17–19,48</a></sup> establishing it as a versatile tool for typical applications in single-cell analysis, especially for rare cell isolation/expansion and single-cell antibody screens. Lastly, the design of the recovery region can potentially be adapted to any droplet microfluidic devices where precise recovery of processed droplets is required, making it a convenient module for different droplet-based systems.</p></section><section id="sec30"><h2 class="pmc_sec_title">Author contributions</h2>
<p>CAM and SCCS envisaged the project and coordinated the study. ZD, CAM, and SCCS designed the experiments and analyzed the data. JMP, MW and RG helped with the design of the microfluidic device (droplet generator, sorter geometry). ZD designed (sorter electrodes, recovery region) and fabricated all devices, and carried out all experiments except for the automation software (uflow) established previously. MW and AA helped with troubleshooting the antibody assay. ZD, CAM, and SCCS wrote the manuscript which was revised and reviewed by all authors.</p></section><section id="sec31"><h2 class="pmc_sec_title">Conflicts of interest</h2>
<p>There are no conflicts to declare.</p></section><section id="sec32"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="LC-025-D5LC00415B-s001"><div class="caption p"><span>LC-025-D5LC00415B-s001</span></div>
<div class="media p" id="d67e993"><div class="caption">
<a href="/articles/instance/12281281/bin/LC-025-D5LC00415B-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">LC-025-D5LC00415B-s001.pdf</a><sup> (876KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="LC-025-D5LC00415B-s002"><div class="caption p"><span>LC-025-D5LC00415B-s002</span></div>
<div class="media p" id="d67e996"><div class="caption">
<a href="/articles/instance/12281281/bin/LC-025-D5LC00415B-s002.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (17.4MB, mp4) </sup>
</div></div></section><section class="sm xbox font-sm" id="LC-025-D5LC00415B-s003"><div class="caption p"><span>LC-025-D5LC00415B-s003</span></div>
<div class="media p" id="d67e999"><div class="caption">
<a href="/articles/instance/12281281/bin/LC-025-D5LC00415B-s003.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (18.4MB, mp4) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors would like to thank the Human Health Therapeutics Research team at National Research Council Canada for providing cell lines. ZD thanks VERAXA Biotech GmbH for funding and technical support when designing the device and antibody assay. The authors thank the Natural Sciences and Engineering Research Council (NSERC; RGPIN-2021-04101), the Canadian Foundation of Innovation (CFI JELF; 35649), and the Swiss National Science Foundation (SNSF; 222922) for funding. ZD thanks the Department of Electrical and Computer Engineering, Concordia University for FRS funding. SCCS thanks Concordia University for a Research Chair.</p></section><section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<sup>†</sup><p class="display-inline">Electronic supplementary information (ESI) available. See DOI: <a href="https://doi.org/10.1039/d5lc00415b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1039/d5lc00415b</a></p>
</div></div></section><section id="sec33"><h2 class="pmc_sec_title">Data availability</h2>
<p>The automated script and software for sorting cells were used to support this manuscript and are available on our shared bitbucket website (see the Methods section). We have included the CAD files for the devices on our Open Science Framework website <a href="https://osf.io/jmxc6/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://osf.io/jmxc6/</a>. The raw data associated with this manuscript are available from the corresponding author upon request.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="cit1">
<cite>Gohil S. H. Iorgulescu J. B. Braun D. A. Keskin D. B. Livak K. J. Nat. Rev. Clin. Oncol. 2021;18(4):244–256. doi: 10.1038/s41571-020-00449-x.</cite> [<a href="https://doi.org/10.1038/s41571-020-00449-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8415132/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33277626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Clin.%20Oncol.&amp;author=S.%20H.%20Gohil&amp;author=J.%20B.%20Iorgulescu&amp;author=D.%20A.%20Braun&amp;author=D.%20B.%20Keskin&amp;author=K.%20J.%20Livak&amp;volume=18&amp;issue=4&amp;publication_year=2021&amp;pages=244-256&amp;pmid=33277626&amp;doi=10.1038/s41571-020-00449-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit2">
<cite>Rebuffet L. Melsen J. E. Escalière B. Basurto-Lozada D. Bhandoola A. Björkström N. K. Bryceson Y. T. Castriconi R. Cichocki F. Colonna M. Davis D. M. Diefenbach A. Ding Y. Haniffa M. Horowitz A. Lanier L. L. Malmberg K.-J. Miller J. S. Moretta L. Narni-Mancinelli E. O'Neill L. A. J. Romagnani C. Ryan D. G. Sivori S. Sun D. Vagne C. Vivier E. Nat. Immunol. 2024;25(8):1474–1488. doi: 10.1038/s41590-024-01883-0.</cite> [<a href="https://doi.org/10.1038/s41590-024-01883-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11291291/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38956378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Immunol.&amp;author=L.%20Rebuffet&amp;author=J.%20E.%20Melsen&amp;author=B.%20Escali%C3%A8re&amp;author=D.%20Basurto-Lozada&amp;author=A.%20Bhandoola&amp;volume=25&amp;issue=8&amp;publication_year=2024&amp;pages=1474-1488&amp;pmid=38956378&amp;doi=10.1038/s41590-024-01883-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit3">
<cite>Li P.-H. Kong X.-Y. He Y.-Z. Liu Y. Peng X. Li Z.-H. Xu H. Luo H. Park J. Mil. Med. Res. 2022;9(1):52. doi: 10.1186/s40779-022-00414-y.</cite> [<a href="https://doi.org/10.1186/s40779-022-00414-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9511789/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36154923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mil.%20Med.%20Res.&amp;author=P.-H.%20Li&amp;author=X.-Y.%20Kong&amp;author=Y.-Z.%20He&amp;author=Y.%20Liu&amp;author=X.%20Peng&amp;volume=9&amp;issue=1&amp;publication_year=2022&amp;pages=52&amp;pmid=36154923&amp;doi=10.1186/s40779-022-00414-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit4">
<cite>Pedrioli A. Oxenius A. Trends Immunol. 2021;42(12):1143–1158. doi: 10.1016/j.it.2021.10.008.</cite> [<a href="https://doi.org/10.1016/j.it.2021.10.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34743921/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Immunol.&amp;author=A.%20Pedrioli&amp;author=A.%20Oxenius&amp;volume=42&amp;issue=12&amp;publication_year=2021&amp;pages=1143-1158&amp;pmid=34743921&amp;doi=10.1016/j.it.2021.10.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit5">
<cite>Van Lent J. Breukers J. Ven K. Ampofo L. Horta S. Pollet F. Imbrechts M. Geukens N. Vanhoorelbeke K. Declerck P. Lammertyn J. Lab Chip. 2021;21(19):3627–3654. doi: 10.1039/D1LC00243K.</cite> [<a href="https://doi.org/10.1039/D1LC00243K" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34505611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lab%20Chip&amp;author=J.%20Van%20Lent&amp;author=J.%20Breukers&amp;author=K.%20Ven&amp;author=L.%20Ampofo&amp;author=S.%20Horta&amp;volume=21&amp;issue=19&amp;publication_year=2021&amp;pages=3627-3654&amp;pmid=34505611&amp;doi=10.1039/D1LC00243K&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit6">
<cite>Piwecka M. Rajewsky N. Rybak-Wolf A. Nat. Rev. Neurol. 2023;19(6):346–362. doi: 10.1038/s41582-023-00809-y.</cite> [<a href="https://doi.org/10.1038/s41582-023-00809-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10191412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37198436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Neurol.&amp;author=M.%20Piwecka&amp;author=N.%20Rajewsky&amp;author=A.%20Rybak-Wolf&amp;volume=19&amp;issue=6&amp;publication_year=2023&amp;pages=346-362&amp;pmid=37198436&amp;doi=10.1038/s41582-023-00809-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit7">
<cite>Walter T. J. Suter R. K. Ayad N. G. Neurobiol. Dis. 2023;184:106201. doi: 10.1016/j.nbd.2023.106201.</cite> [<a href="https://doi.org/10.1016/j.nbd.2023.106201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10470823/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37321420/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neurobiol.%20Dis.&amp;author=T.%20J.%20Walter&amp;author=R.%20K.%20Suter&amp;author=N.%20G.%20Ayad&amp;volume=184&amp;publication_year=2023&amp;pages=106201&amp;pmid=37321420&amp;doi=10.1016/j.nbd.2023.106201&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit8">
<cite>Matuła K. Rivello F. Huck W. T. S. Adv. Biosyst. 2020;4(1):1900188. doi: 10.1002/adbi.201900188.</cite> [<a href="https://doi.org/10.1002/adbi.201900188" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32293129/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Biosyst.&amp;author=K.%20Matu%C5%82a&amp;author=F.%20Rivello&amp;author=W.%20T.%20S.%20Huck&amp;volume=4&amp;issue=1&amp;publication_year=2020&amp;pages=1900188&amp;pmid=32293129&amp;doi=10.1002/adbi.201900188&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit9">
<cite>Xu X. Wang J. Wu L. Guo J. Song Y. Tian T. Wang W. Zhu Z. Yang C. Small. 2020;16(9):1903905. doi: 10.1002/smll.201903905.</cite> [<a href="https://doi.org/10.1002/smll.201903905" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31544338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Small&amp;author=X.%20Xu&amp;author=J.%20Wang&amp;author=L.%20Wu&amp;author=J.%20Guo&amp;author=Y.%20Song&amp;volume=16&amp;issue=9&amp;publication_year=2020&amp;pages=1903905&amp;pmid=31544338&amp;doi=10.1002/smll.201903905&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit10">
<cite>Leal-Alves C. Dumont S. Deng Z. Alkhaldi S. Leung Z. Oeser M. Shih S. C. C. Adv. Mater. Technol. 2025;10(6):2401209. doi: 10.1002/admt.202401209.</cite> [<a href="https://doi.org/10.1002/admt.202401209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Mater.%20Technol.&amp;author=C.%20Leal-Alves&amp;author=S.%20Dumont&amp;author=Z.%20Deng&amp;author=S.%20Alkhaldi&amp;author=Z.%20Leung&amp;volume=10&amp;issue=6&amp;publication_year=2025&amp;pages=2401209&amp;doi=10.1002/admt.202401209&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit11">
<cite>Abate A. R. Hung T. Mary P. Agresti J. J. Weitz D. A. Proc. Natl. Acad. Sci. U. S. A. 2010;107(45):19163–19166. doi: 10.1073/pnas.1006888107.</cite> [<a href="https://doi.org/10.1073/pnas.1006888107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2984161/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20962271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&amp;author=A.%20R.%20Abate&amp;author=T.%20Hung&amp;author=P.%20Mary&amp;author=J.%20J.%20Agresti&amp;author=D.%20A.%20Weitz&amp;volume=107&amp;issue=45&amp;publication_year=2010&amp;pages=19163-19166&amp;pmid=20962271&amp;doi=10.1073/pnas.1006888107&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit12">
<cite>Shembekar N. Hu H. Eustace D. Merten C. A. Cell Rep. 2018;22(8):2206–2215. doi: 10.1016/j.celrep.2018.01.071.</cite> [<a href="https://doi.org/10.1016/j.celrep.2018.01.071" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5842027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29466744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&amp;author=N.%20Shembekar&amp;author=H.%20Hu&amp;author=D.%20Eustace&amp;author=C.%20A.%20Merten&amp;volume=22&amp;issue=8&amp;publication_year=2018&amp;pages=2206-2215&amp;pmid=29466744&amp;doi=10.1016/j.celrep.2018.01.071&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit13">
<cite>Verbist W. Breukers J. Sharma S. Rutten I. Gerstmans H. Coelmont L. Dal Dosso F. Dallmeier K. Lammertyn J. Lab Chip. 2024;24(7):2107–2121. doi: 10.1039/D3LC01075A.</cite> [<a href="https://doi.org/10.1039/D3LC01075A" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38450543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lab%20Chip&amp;author=W.%20Verbist&amp;author=J.%20Breukers&amp;author=S.%20Sharma&amp;author=I.%20Rutten&amp;author=H.%20Gerstmans&amp;volume=24&amp;issue=7&amp;publication_year=2024&amp;pages=2107-2121&amp;pmid=38450543&amp;doi=10.1039/D3LC01075A&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit14">
<cite>Ahmadi F. Tran H. Letourneau N. Little S. R. Fortin A. Moraitis A. N. Shih S. C. C. Small. 2024;20(26):2308950. doi: 10.1002/smll.202308950.</cite> [<a href="https://doi.org/10.1002/smll.202308950" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38441226/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Small&amp;author=F.%20Ahmadi&amp;author=H.%20Tran&amp;author=N.%20Letourneau&amp;author=S.%20R.%20Little&amp;author=A.%20Fortin&amp;volume=20&amp;issue=26&amp;publication_year=2024&amp;pages=2308950&amp;pmid=38441226&amp;doi=10.1002/smll.202308950&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit15">
<cite>Clausell-Tormos J. Lieber D. Baret J.-C. El-Harrak A. Miller O. J. Frenz L. Blouwolff J. Humphry K. J. Köster S. Duan H. Holtze C. Weitz D. A. Griffiths A. D. Merten C. A. Chem. Biol. 2008;15(5):427–437. doi: 10.1016/j.chembiol.2008.04.004.</cite> [<a href="https://doi.org/10.1016/j.chembiol.2008.04.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18482695/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Biol.&amp;author=J.%20Clausell-Tormos&amp;author=D.%20Lieber&amp;author=J.-C.%20Baret&amp;author=A.%20El-Harrak&amp;author=O.%20J.%20Miller&amp;volume=15&amp;issue=5&amp;publication_year=2008&amp;pages=427-437&amp;pmid=18482695&amp;doi=10.1016/j.chembiol.2008.04.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit16">
<cite>Murugan R. Imkeller K. Busse C. E. Wardemann H. Eur. J. Immunol. 2015;45(9):2698–2700. doi: 10.1002/eji.201545526.</cite> [<a href="https://doi.org/10.1002/eji.201545526" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5008140/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26138551/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Immunol.&amp;author=R.%20Murugan&amp;author=K.%20Imkeller&amp;author=C.%20E.%20Busse&amp;author=H.%20Wardemann&amp;volume=45&amp;issue=9&amp;publication_year=2015&amp;pages=2698-2700&amp;pmid=26138551&amp;doi=10.1002/eji.201545526&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit17">
<cite>Sun C. Liu L. Pérez L. Li X. Liu Y. Xu P. Boritz E. A. Mullins J. I. Abate A. R. Nat. Biomed. Eng. 2022;6(8):1004–1012. doi: 10.1038/s41551-022-00864-8.</cite> [<a href="https://doi.org/10.1038/s41551-022-00864-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9398922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35347274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biomed.%20Eng.&amp;author=C.%20Sun&amp;author=L.%20Liu&amp;author=L.%20P%C3%A9rez&amp;author=X.%20Li&amp;author=Y.%20Liu&amp;volume=6&amp;issue=8&amp;publication_year=2022&amp;pages=1004-1012&amp;pmid=35347274&amp;doi=10.1038/s41551-022-00864-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit18">
<cite>Ding R. Hung K. C. Mitra A. Ung L. W. Lightwood D. Tu R. Starkie D. Cai L. Mazutis L. Chong S. Weitz D. A. Heyman J. A. RSC Adv. 2020;10(45):27006–27013. doi: 10.1039/D0RA04328A.</cite> [<a href="https://doi.org/10.1039/D0RA04328A" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9055518/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35515810/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Adv.&amp;author=R.%20Ding&amp;author=K.%20C.%20Hung&amp;author=A.%20Mitra&amp;author=L.%20W.%20Ung&amp;author=D.%20Lightwood&amp;volume=10&amp;issue=45&amp;publication_year=2020&amp;pages=27006-27013&amp;pmid=35515810&amp;doi=10.1039/D0RA04328A&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit19">
<cite>Gaa R. Menang-Ndi E. Pratapa S. Nguyen C. Kumar S. Doerner A. mAbs. 2021;13(1):1978130. doi: 10.1080/19420862.2021.1978130.</cite> [<a href="https://doi.org/10.1080/19420862.2021.1978130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8489958/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34586015/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mAbs&amp;author=R.%20Gaa&amp;author=E.%20Menang-Ndi&amp;author=S.%20Pratapa&amp;author=C.%20Nguyen&amp;author=S.%20Kumar&amp;volume=13&amp;issue=1&amp;publication_year=2021&amp;pages=1978130&amp;pmid=34586015&amp;doi=10.1080/19420862.2021.1978130&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit20">
<cite>Kodituwakku A. P. Jessup C. Zola H. Roberton D. M. Immunol. Cell Biol. 2003;81(3):163–170. doi: 10.1046/j.1440-1711.2003.01152.x.</cite> [<a href="https://doi.org/10.1046/j.1440-1711.2003.01152.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12752679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunol.%20Cell%20Biol.&amp;author=A.%20P.%20Kodituwakku&amp;author=C.%20Jessup&amp;author=H.%20Zola&amp;author=D.%20M.%20Roberton&amp;volume=81&amp;issue=3&amp;publication_year=2003&amp;pages=163-170&amp;pmid=12752679&amp;doi=10.1046/j.1440-1711.2003.01152.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit21">
<cite>Paterlini-Brechot P. Benali N. L. Cancer Lett. 2007;253(2):180–204. doi: 10.1016/j.canlet.2006.12.014.</cite> [<a href="https://doi.org/10.1016/j.canlet.2006.12.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17314005/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Lett.&amp;author=P.%20Paterlini-Brechot&amp;author=N.%20L.%20Benali&amp;volume=253&amp;issue=2&amp;publication_year=2007&amp;pages=180-204&amp;pmid=17314005&amp;doi=10.1016/j.canlet.2006.12.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit22">
<cite>Parks D. R. Bryan V. M. Oi V. T. Herzenberg L. A. Proc. Natl. Acad. Sci. U. S. A. 1979;76(4):1962–1966. doi: 10.1073/pnas.76.4.1962.</cite> [<a href="https://doi.org/10.1073/pnas.76.4.1962" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC383513/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/287037/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&amp;author=D.%20R.%20Parks&amp;author=V.%20M.%20Bryan&amp;author=V.%20T.%20Oi&amp;author=L.%20A.%20Herzenberg&amp;volume=76&amp;issue=4&amp;publication_year=1979&amp;pages=1962-1966&amp;pmid=287037&amp;doi=10.1073/pnas.76.4.1962&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit23">
<cite>Ma S. Zhang B. LaFave L. M. Earl A. S. Chiang Z. Hu Y. Ding J. Brack A. Kartha V. K. Tay T. Law T. Lareau C. Hsu Y.-C. Regev A. Buenrostro J. D. Cell. 2020;183(4):1103–1116.e1120. doi: 10.1016/j.cell.2020.09.056.</cite> [<a href="https://doi.org/10.1016/j.cell.2020.09.056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7669735/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33098772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;author=S.%20Ma&amp;author=B.%20Zhang&amp;author=L.%20M.%20LaFave&amp;author=A.%20S.%20Earl&amp;author=Z.%20Chiang&amp;volume=183&amp;issue=4&amp;publication_year=2020&amp;pages=1103-1116.e1120&amp;pmid=33098772&amp;doi=10.1016/j.cell.2020.09.056&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit24">
<cite>Portmann K. Linder A. Oelgarth N. Eyer K. Cells Rep. Methods. 2023;3(7):100502. doi: 10.1016/j.crmeth.2023.100502.</cite> [<a href="https://doi.org/10.1016/j.crmeth.2023.100502" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10391336/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37533643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cells%20Rep.%20Methods&amp;author=K.%20Portmann&amp;author=A.%20Linder&amp;author=N.%20Oelgarth&amp;author=K.%20Eyer&amp;volume=3&amp;issue=7&amp;publication_year=2023&amp;pages=100502&amp;pmid=37533643&amp;doi=10.1016/j.crmeth.2023.100502&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit25">
<cite>Debs B. E. Utharala R. Balyasnikova I. V. Griffiths A. D. Merten C. A. Proc. Natl. Acad. Sci. U. S. A. 2012;109(29):11570–11575. doi: 10.1073/pnas.1204514109.</cite> [<a href="https://doi.org/10.1073/pnas.1204514109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3406880/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22753519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&amp;author=B.%20E.%20Debs&amp;author=R.%20Utharala&amp;author=I.%20V.%20Balyasnikova&amp;author=A.%20D.%20Griffiths&amp;author=C.%20A.%20Merten&amp;volume=109&amp;issue=29&amp;publication_year=2012&amp;pages=11570-11575&amp;pmid=22753519&amp;doi=10.1073/pnas.1204514109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit26">
<cite>Karbaschi M. Shahi P. Abate A. R. Biomicrofluidics. 2017;11(4):044107. doi: 10.1063/1.4995479.</cite> [<a href="https://doi.org/10.1063/1.4995479" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5519397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28794817/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomicrofluidics&amp;author=M.%20Karbaschi&amp;author=P.%20Shahi&amp;author=A.%20R.%20Abate&amp;volume=11&amp;issue=4&amp;publication_year=2017&amp;pages=044107&amp;pmid=28794817&amp;doi=10.1063/1.4995479&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit27">
<cite>Zilionis R. Nainys J. Veres A. Savova V. Zemmour D. Klein A. M. Mazutis L. Nat. Protoc. 2017;12(1):44–73. doi: 10.1038/nprot.2016.154.</cite> [<a href="https://doi.org/10.1038/nprot.2016.154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27929523/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;author=R.%20Zilionis&amp;author=J.%20Nainys&amp;author=A.%20Veres&amp;author=V.%20Savova&amp;author=D.%20Zemmour&amp;volume=12&amp;issue=1&amp;publication_year=2017&amp;pages=44-73&amp;pmid=27929523&amp;doi=10.1038/nprot.2016.154&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit28">
<cite>Cole R. H. Tang S.-Y. Siltanen C. A. Shahi P. Zhang J. Q. Poust S. Gartner Z. J. Abate A. R. Proc. Natl. Acad. Sci. U. S. A. 2017;114(33):8728–8733. doi: 10.1073/pnas.1704020114.</cite> [<a href="https://doi.org/10.1073/pnas.1704020114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5565430/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28760972/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&amp;author=R.%20H.%20Cole&amp;author=S.-Y.%20Tang&amp;author=C.%20A.%20Siltanen&amp;author=P.%20Shahi&amp;author=J.%20Q.%20Zhang&amp;volume=114&amp;issue=33&amp;publication_year=2017&amp;pages=8728-8733&amp;pmid=28760972&amp;doi=10.1073/pnas.1704020114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit29">
<cite>Nan L. Lai M. Y. A. Tang M. Y. H. Chan Y. K. Poon L. L. M. Shum H. C. Small. 2020;16(9):e1902889. doi: 10.1002/smll.201902889.</cite> [<a href="https://doi.org/10.1002/smll.201902889" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31448532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Small&amp;author=L.%20Nan&amp;author=M.%20Y.%20A.%20Lai&amp;author=M.%20Y.%20H.%20Tang&amp;author=Y.%20K.%20Chan&amp;author=L.%20L.%20M.%20Poon&amp;volume=16&amp;issue=9&amp;publication_year=2020&amp;pages=e1902889&amp;pmid=31448532&amp;doi=10.1002/smll.201902889&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit30">
<cite>Weiss M. Hasan S. Genth R. Mollah M. Robert E. Gil A. Hufnagel L. Lab Chip. 2025;25(4):600–612. doi: 10.1039/D4LC00536H.</cite> [<a href="https://doi.org/10.1039/D4LC00536H" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39834322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lab%20Chip&amp;author=M.%20Weiss&amp;author=S.%20Hasan&amp;author=R.%20Genth&amp;author=M.%20Mollah&amp;author=E.%20Robert&amp;volume=25&amp;issue=4&amp;publication_year=2025&amp;pages=600-612&amp;pmid=39834322&amp;doi=10.1039/D4LC00536H&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit31">
<cite>Josephides D. Davoli S. Whitley W. Ruis R. Salter R. Gokkaya S. Vallet M. Matthews D. Benazzi G. Shvets E. Gesellchen F. Geere D. Liu X. Li X. Mackworth B. Young W. Owen Z. Smith C. Starkie D. White J. Sweeney B. Hinchliffe M. Tickle S. Lightwood D. J. Rehak M. Craig F. F. Holmes D. SLAS Technol. 2020;25(2):177–189. doi: 10.1177/2472630319892571.</cite> [<a href="https://doi.org/10.1177/2472630319892571" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31941402/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=SLAS%20Technol.&amp;author=D.%20Josephides&amp;author=S.%20Davoli&amp;author=W.%20Whitley&amp;author=R.%20Ruis&amp;author=R.%20Salter&amp;volume=25&amp;issue=2&amp;publication_year=2020&amp;pages=177-189&amp;pmid=31941402&amp;doi=10.1177/2472630319892571&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit32">
<cite>Fobel R. Fobel C. Wheeler A. R. Appl. Phys. Lett. 2013;102(19):193513. doi: 10.1063/1.4807118.</cite> [<a href="https://doi.org/10.1063/1.4807118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Phys.%20Lett.&amp;author=R.%20Fobel&amp;author=C.%20Fobel&amp;author=A.%20R.%20Wheeler&amp;volume=102&amp;issue=19&amp;publication_year=2013&amp;pages=193513&amp;doi=10.1063/1.4807118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit33">
<cite>Abdelgawad M. Watson M. W. L. Wheeler A. R. Lab Chip. 2009;9(8):1046–1051. doi: 10.1039/B820682A.</cite> [<a href="https://doi.org/10.1039/B820682A" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19350085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lab%20Chip&amp;author=M.%20Abdelgawad&amp;author=M.%20W.%20L.%20Watson&amp;author=A.%20R.%20Wheeler&amp;volume=9&amp;issue=8&amp;publication_year=2009&amp;pages=1046-1051&amp;pmid=19350085&amp;doi=10.1039/B820682A&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit34">
<cite>Strijbosch L. W. G. Buurman W. A. Does R. J. M. M. Zinken P. H. Groenewegen G. J. Immunol. Methods. 1987;97(1):133–140. doi: 10.1016/0022-1759(87)90115-3.</cite> [<a href="https://doi.org/10.1016/0022-1759(87)90115-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3819436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.%20Methods&amp;author=L.%20W.%20G.%20Strijbosch&amp;author=W.%20A.%20Buurman&amp;author=R.%20J.%20M.%20M.%20Does&amp;author=P.%20H.%20Zinken&amp;author=G.%20Groenewegen&amp;volume=97&amp;issue=1&amp;publication_year=1987&amp;pages=133-140&amp;pmid=3819436&amp;doi=10.1016/0022-1759(87)90115-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit35">
<cite>Baret J.-C. Miller O. J. Taly V. Ryckelynck M. El-Harrak A. Frenz L. Rick C. Samuels M. L. Hutchison J. B. Agresti J. J. Link D. R. Weitz D. A. Griffiths A. D. Lab Chip. 2009;9(13):1850–1858. doi: 10.1039/B902504A.</cite> [<a href="https://doi.org/10.1039/B902504A" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19532959/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lab%20Chip&amp;author=J.-C.%20Baret&amp;author=O.%20J.%20Miller&amp;author=V.%20Taly&amp;author=M.%20Ryckelynck&amp;author=A.%20El-Harrak&amp;volume=9&amp;issue=13&amp;publication_year=2009&amp;pages=1850-1858&amp;pmid=19532959&amp;doi=10.1039/B902504A&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit36"><cite>
Pernod K., Coppieters N., Graillet Q., Wagner A., Arnaud C. and van Loo S., D-Sire &amp; ModaFlow for High-Throughput Antibody Discovery, 2025</cite></li>
<li id="cit37">
<cite>Shih S. C. C. Gach P. C. Sustarich J. Simmons B. A. Adams P. D. Singh S. Singh A. K. Lab Chip. 2015;15(1):225–236. doi: 10.1039/C4LC00794H.</cite> [<a href="https://doi.org/10.1039/C4LC00794H" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25354549/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lab%20Chip&amp;author=S.%20C.%20C.%20Shih&amp;author=P.%20C.%20Gach&amp;author=J.%20Sustarich&amp;author=B.%20A.%20Simmons&amp;author=P.%20D.%20Adams&amp;volume=15&amp;issue=1&amp;publication_year=2015&amp;pages=225-236&amp;pmid=25354549&amp;doi=10.1039/C4LC00794H&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit38">
<cite>Lan F. Demaree B. Ahmed N. Abate A. R. Nat. Biotechnol. 2017;35(7):640–646. doi: 10.1038/nbt.3880.</cite> [<a href="https://doi.org/10.1038/nbt.3880" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5531050/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28553940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&amp;author=F.%20Lan&amp;author=B.%20Demaree&amp;author=N.%20Ahmed&amp;author=A.%20R.%20Abate&amp;volume=35&amp;issue=7&amp;publication_year=2017&amp;pages=640-646&amp;pmid=28553940&amp;doi=10.1038/nbt.3880&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit39">
<cite>Hu P. Zhang W. Xin H. Deng G. Front. Cell Dev. Biol. 2016;4:116. doi: 10.3389/fcell.2016.00116.</cite> [<a href="https://doi.org/10.3389/fcell.2016.00116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5078503/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27826548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Cell%20Dev.%20Biol.&amp;author=P.%20Hu&amp;author=W.%20Zhang&amp;author=H.%20Xin&amp;author=G.%20Deng&amp;volume=4&amp;publication_year=2016&amp;pages=116&amp;pmid=27826548&amp;doi=10.3389/fcell.2016.00116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit40">
<cite>Chen Y. Li P. Huang P. H. Xie Y. Mai J. D. Wang L. Nguyen N. T. Huang T. J. Lab Chip. 2014;14(4):626–645. doi: 10.1039/C3LC90136J.</cite> [<a href="https://doi.org/10.1039/C3LC90136J" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3991782/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24406985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lab%20Chip&amp;author=Y.%20Chen&amp;author=P.%20Li&amp;author=P.%20H.%20Huang&amp;author=Y.%20Xie&amp;author=J.%20D.%20Mai&amp;volume=14&amp;issue=4&amp;publication_year=2014&amp;pages=626-645&amp;pmid=24406985&amp;doi=10.1039/C3LC90136J&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit41">
<cite>Mazutis L. Gilbert J. Ung W. L. Weitz D. A. Griffiths A. D. Heyman J. A. Nat. Protoc. 2013;8(5):870–891. doi: 10.1038/nprot.2013.046.</cite> [<a href="https://doi.org/10.1038/nprot.2013.046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4128248/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23558786/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;author=L.%20Mazutis&amp;author=J.%20Gilbert&amp;author=W.%20L.%20Ung&amp;author=D.%20A.%20Weitz&amp;author=A.%20D.%20Griffiths&amp;volume=8&amp;issue=5&amp;publication_year=2013&amp;pages=870-891&amp;pmid=23558786&amp;doi=10.1038/nprot.2013.046&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit42">
<cite>Samlali K. Alves C. L. Jezernik M. Shih S. C. C. Microsyst. Nanoeng. 2022;8(1):123. doi: 10.1038/s41378-022-00456-1.</cite> [<a href="https://doi.org/10.1038/s41378-022-00456-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9681769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36438986/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microsyst.%20Nanoeng.&amp;author=K.%20Samlali&amp;author=C.%20L.%20Alves&amp;author=M.%20Jezernik&amp;author=S.%20C.%20C.%20Shih&amp;volume=8&amp;issue=1&amp;publication_year=2022&amp;pages=123&amp;pmid=36438986&amp;doi=10.1038/s41378-022-00456-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit43">
<cite>Schütz S. S. Beneyton T. Baret J.-C. Schneider T. M. Lab Chip. 2019;19(13):2220–2232. doi: 10.1039/C9LC00149B.</cite> [<a href="https://doi.org/10.1039/C9LC00149B" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31157806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lab%20Chip&amp;author=S.%20S.%20Sch%C3%BCtz&amp;author=T.%20Beneyton&amp;author=J.-C.%20Baret&amp;author=T.%20M.%20Schneider&amp;volume=19&amp;issue=13&amp;publication_year=2019&amp;pages=2220-2232&amp;pmid=31157806&amp;doi=10.1039/C9LC00149B&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit44">
<cite>Caen O. Schütz S. Jammalamadaka M. S. S. Vrignon J. Nizard P. Schneider T. M. Baret J.-C. Taly V. Microsyst. Nanoeng. 2018;4(1):33. doi: 10.1038/s41378-018-0033-2.</cite> [<a href="https://doi.org/10.1038/s41378-018-0033-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6220162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31057921/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microsyst.%20Nanoeng.&amp;author=O.%20Caen&amp;author=S.%20Sch%C3%BCtz&amp;author=M.%20S.%20S.%20Jammalamadaka&amp;author=J.%20Vrignon&amp;author=P.%20Nizard&amp;volume=4&amp;issue=1&amp;publication_year=2018&amp;pages=33&amp;pmid=31057921&amp;doi=10.1038/s41378-018-0033-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit45">
<cite>Chen L. Xu Y. Zhou L. Ma D. Zhang R. Liu Y. Mi X. Microsyst. Nanoeng. 2025;11(1):10. doi: 10.1038/s41378-024-00861-8.</cite> [<a href="https://doi.org/10.1038/s41378-024-00861-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11739583/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39819845/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microsyst.%20Nanoeng.&amp;author=L.%20Chen&amp;author=Y.%20Xu&amp;author=L.%20Zhou&amp;author=D.%20Ma&amp;author=R.%20Zhang&amp;volume=11&amp;issue=1&amp;publication_year=2025&amp;pages=10&amp;pmid=39819845&amp;doi=10.1038/s41378-024-00861-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit46">
<cite>Mitra S. Tomar P. C. J. Genet. Eng. Biotechnol. 2021;19(1):159. doi: 10.1186/s43141-021-00264-6.</cite> [<a href="https://doi.org/10.1186/s43141-021-00264-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8521504/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34661773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Genet.%20Eng.%20Biotechnol.&amp;author=S.%20Mitra&amp;author=P.%20C.%20Tomar&amp;volume=19&amp;issue=1&amp;publication_year=2021&amp;pages=159&amp;pmid=34661773&amp;doi=10.1186/s43141-021-00264-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit47">
<cite>Autour A. Merten C. A. Proc. Natl. Acad. Sci. U. S. A. 2024;121(37):e2405342121. doi: 10.1073/pnas.2405342121.</cite> [<a href="https://doi.org/10.1073/pnas.2405342121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11406258/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39240970/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&amp;author=A.%20Autour&amp;author=C.%20A.%20Merten&amp;volume=121&amp;issue=37&amp;publication_year=2024&amp;pages=e2405342121&amp;pmid=39240970&amp;doi=10.1073/pnas.2405342121&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit48">
<cite>Murugan R. Imkeller K. Busse C. E. Wardemann H. Eur. J. Immunol. 2015;45(9):2698–2700. doi: 10.1002/eji.201545526.</cite> [<a href="https://doi.org/10.1002/eji.201545526" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5008140/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26138551/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Immunol.&amp;author=R.%20Murugan&amp;author=K.%20Imkeller&amp;author=C.%20E.%20Busse&amp;author=H.%20Wardemann&amp;volume=45&amp;issue=9&amp;publication_year=2015&amp;pages=2698-2700&amp;pmid=26138551&amp;doi=10.1002/eji.201545526&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>LC-025-D5LC00415B-s001</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12281281/bin/LC-025-D5LC00415B-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">LC-025-D5LC00415B-s001.pdf</a><sup> (876KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><span>LC-025-D5LC00415B-s002</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12281281/bin/LC-025-D5LC00415B-s002.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (17.4MB, mp4) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><span>LC-025-D5LC00415B-s003</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12281281/bin/LC-025-D5LC00415B-s003.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (18.4MB, mp4) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The automated script and software for sorting cells were used to support this manuscript and are available on our shared bitbucket website (see the Methods section). We have included the CAD files for the devices on our Open Science Framework website <a href="https://osf.io/jmxc6/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://osf.io/jmxc6/</a>. The raw data associated with this manuscript are available from the corresponding author upon request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Lab on a Chip are provided here courtesy of <strong>Royal Society of Chemistry</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1039/d5lc00415b"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12281281/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12281281/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12281281%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12281281/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12281281/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12281281/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40693295/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12281281/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40693295/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12281281/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12281281/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="exieCF1mten2DhkTsoDpF3dblrKSxwqB9tz682OUSA2YNwrnSfgBIgDbi3srHzAk">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
