
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Reproducible Boolean model analyses and simulations with the CoLoMoTo software suite: a tutorial - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532F4F8AF22A43052F4F00033B4953.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="interfacefoc">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371346/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Interface Focus">
<meta name="citation_title" content="Reproducible Boolean model analyses and simulations with the CoLoMoTo software suite: a tutorial">
<meta name="citation_author" content="Vincent Noël">
<meta name="citation_author_institution" content="Institut Curie, Université PSL, 75005 Paris, France">
<meta name="citation_author_institution" content="INSERM U1331, 75005 Paris, France">
<meta name="citation_author_institution" content="Mines ParisTech, Université PSL, 75005 Paris, France">
<meta name="citation_author" content="Aurélien Naldi">
<meta name="citation_author_institution" content="Département de Biologie, Ecole Normale Supérieure, PSL Université, 75005 Paris, France">
<meta name="citation_author" content="Laurence Calzone">
<meta name="citation_author_institution" content="Institut Curie, Université PSL, 75005 Paris, France">
<meta name="citation_author_institution" content="INSERM U1331, 75005 Paris, France">
<meta name="citation_author_institution" content="Mines ParisTech, Université PSL, 75005 Paris, France">
<meta name="citation_author" content="Loic Paulevé">
<meta name="citation_author_institution" content="LaBRI, CNRS UMR 5800, INP, Université de Bordeaux, 33522 Talence, France">
<meta name="citation_author" content="Denis Thieffry">
<meta name="citation_author_institution" content="Institut Curie, Université PSL, 75005 Paris, France">
<meta name="citation_author_institution" content="INSERM U1331, 75005 Paris, France">
<meta name="citation_author_institution" content="Mines ParisTech, Université PSL, 75005 Paris, France">
<meta name="citation_author_institution" content="Département de Biologie, Ecole Normale Supérieure, PSL Université, 75005 Paris, France">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="15">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="20250002">
<meta name="citation_doi" content="10.1098/rsfs.2025.0002">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371346/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371346/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371346/pdf/rsfs.2025.0002.pdf">
<meta name="description" content="This tutorial provides stepwise instructions to install over 20 tools, written in multiple languages. Their integration in the CoLoMoTo software suite makes them accessible with a single popular language (Python), thereby enabling reproducible and ...">
<meta name="og:title" content="Reproducible Boolean model analyses and simulations with the CoLoMoTo software suite: a tutorial">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="This tutorial provides stepwise instructions to install over 20 tools, written in multiple languages. Their integration in the CoLoMoTo software suite makes them accessible with a single popular language (Python), thereby enabling reproducible and ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371346/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371346">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1098/rsfs.2025.0002"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/rsfs.2025.0002.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371346%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371346/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371346/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371346/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-interfacefoc.gif" alt="Interface Focus logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Interface Focus" title="Link to Interface Focus" shape="default" href="http://rsfs.royalsocietypublishing.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Interface Focus</button></div>. 2025 Aug 22;15(3):20250002. doi: <a href="https://doi.org/10.1098/rsfs.2025.0002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1098/rsfs.2025.0002</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Interface%20Focus%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Interface%20Focus%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Interface%20Focus%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Interface%20Focus%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Reproducible Boolean model analyses and simulations with the CoLoMoTo software suite: a tutorial</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22No%C3%ABl%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Vincent Noël</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Vincent Noël</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Institut Curie, Université PSL, 75005 Paris, France</div>
<div class="p">
<sup><sup>2</sup></sup>INSERM U1331, 75005 Paris, France</div>
<div class="p">
<sup><sup>3</sup></sup>Mines ParisTech, Université PSL, 75005 Paris, France</div>
<div>Conceptualization, Formal analysis, Investigation, Methodology, Software, Validation, Writing – original draft, Writing – review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22No%C3%ABl%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Vincent Noël</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Naldi%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Aurélien Naldi</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Aurélien Naldi</span></h3>
<div class="p">
<sup><sup>4</sup></sup>Département de Biologie, Ecole Normale Supérieure, PSL Université, 75005 Paris, France</div>
<div>Formal analysis, Investigation, Methodology, Software, Writing – original draft, Writing – review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Naldi%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Aurélien Naldi</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Calzone%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Laurence Calzone</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Laurence Calzone</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Institut Curie, Université PSL, 75005 Paris, France</div>
<div class="p">
<sup><sup>2</sup></sup>INSERM U1331, 75005 Paris, France</div>
<div class="p">
<sup><sup>3</sup></sup>Mines ParisTech, Université PSL, 75005 Paris, France</div>
<div>Conceptualization, Formal analysis, Funding acquisition, Methodology, Software, Supervision, Validation, Visualization, Writing – original draft, Writing – review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Calzone%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Laurence Calzone</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Paulev%C3%A9%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Loic Paulevé</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Loic Paulevé</span></h3>
<div class="p">
<sup><sup>5</sup></sup>LaBRI, CNRS UMR 5800, INP, Université de Bordeaux, 33522 Talence, France</div>
<div>Conceptualization, Formal analysis, Investigation, Methodology, Software, Supervision, Validation, Writing – original draft, Writing – review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Paulev%C3%A9%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Loic Paulevé</span></a>
</div>
</div>
<sup>5,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thieffry%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Denis Thieffry</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Denis Thieffry</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Institut Curie, Université PSL, 75005 Paris, France</div>
<div class="p">
<sup><sup>2</sup></sup>INSERM U1331, 75005 Paris, France</div>
<div class="p">
<sup><sup>3</sup></sup>Mines ParisTech, Université PSL, 75005 Paris, France</div>
<div class="p">
<sup><sup>4</sup></sup>Département de Biologie, Ecole Normale Supérieure, PSL Université, 75005 Paris, France</div>
<div>Conceptualization, Formal analysis, Investigation, Methodology, Software, Supervision, Validation, Visualization, Writing – original draft, Writing – review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thieffry%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Denis Thieffry</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup><sup>1</sup></sup>Institut Curie, Université PSL, 75005 Paris, France</div>
<div id="aff2">
<sup><sup>2</sup></sup>INSERM U1331, 75005 Paris, France</div>
<div id="aff3">
<sup><sup>3</sup></sup>Mines ParisTech, Université PSL, 75005 Paris, France</div>
<div id="aff4">
<sup><sup>4</sup></sup>Département de Biologie, Ecole Normale Supérieure, PSL Université, 75005 Paris, France</div>
<div id="aff5">
<sup><sup>5</sup></sup>LaBRI, CNRS UMR 5800, INP, Université de Bordeaux, 33522 Talence, France</div>
<div class="author-notes p">
<div class="fn" id="fn1"><p>Electronic supplementary material is available online at <a href="https://doi.org/10.6084/m9.figshare.c.7888734" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.6084/m9.figshare.c.7888734</a>.</p></div>
<div class="fn" id="fn2"><p>One contribution of 6 to a theme issue 'Combinatorial models for evolving representations of dynamical behaviours in biological networks'.</p></div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Vincent Noël</span></strong>: <span class="role">Conceptualization, Formal analysis, Investigation, Methodology, Software, Validation, Writing – original draft, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Aurélien Naldi</span></strong>: <span class="role">Formal analysis, Investigation, Methodology, Software, Writing – original draft, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Laurence Calzone</span></strong>: <span class="role">Conceptualization, Formal analysis, Funding acquisition, Methodology, Software, Supervision, Validation, Visualization, Writing – original draft, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Loic Paulevé</span></strong>: <span class="role">Conceptualization, Formal analysis, Investigation, Methodology, Software, Supervision, Validation, Writing – original draft, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Denis Thieffry</span></strong>: <span class="role">Conceptualization, Formal analysis, Investigation, Methodology, Software, Supervision, Validation, Visualization, Writing – original draft, Writing – review and editing</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 22; Revised 2025 May 4; Accepted 2025 Jun 2; Collection date 2025 Aug 22.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors.</div>
<p>Published by the Royal Society under the terms of the Creative Commons Attribution License <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>, which permits unrestricted use, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371346</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>This tutorial provides stepwise instructions to install over 20 tools, written in multiple languages. Their integration in the <em>CoLoMoTo</em> software suite makes them accessible with a single popular language (<em>Python</em>), thereby enabling reproducible and sophisticated dynamical analyses of logical models of complex cellular networks. The tutorial specifically focuses on the analysis of a previously published model of the regulatory network controlling mammalian cell proliferation. It includes chunks of <em>Python</em> code to reproduce several of the results and figures published in the original article, and further extends these results with the help of selected tools included in the <em>CoLoMoTo</em> suite. The tutorial covers the visualization of the network with the tool <em>GINsim</em>, an attractor analysis with <em>bioLQM</em>, the computation of synchronous attractors with <em>BNS</em>, the extraction of modules from the full model, stochastic simulations of the wild-type model and of selected perturbations with <em>MaBoSS</em> and finally the delineation of compressed probabilistic state transition graphs. The integration of all these analyses in an executable <em>Jupyter Notebook</em> greatly eases their reproducibility, as well as the inclusion of further extensions. The notebook provided along with this tutorial further constitutes a template, which can be enriched with other <em>ColoMoTo</em> tools, to develop comprehensive dynamical analyses of various biological network models.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> reproducible, Boolean, models, simulations, <em>CoLoMoTo</em>, tutorial</p></section></section><section id="s1"><h2 class="pmc_sec_title">1.  Introduction</h2>
<p>Since the seminal studies from Sugita [<a href="#B1" class="usa-link" aria-describedby="B1">1</a>], Kauffman [<a href="#B2" class="usa-link" aria-describedby="B2">2</a>] and Thomas [<a href="#B3" class="usa-link" aria-describedby="B3">3</a>], Boolean models have been increasingly applied to biological signalling and regulatory networks [<a href="#B4" class="usa-link" aria-describedby="B4">4</a>–<a href="#B7" class="usa-link" aria-describedby="B7">7</a>]. With the rise of high-throughput functional genomic methods and the development of molecular pathway knowledge databases, modellers are now coping with networks encompassing hundreds of regulatory components controlling cell fate decisions for normal and pathological conditions (e.g. [<a href="#B8" class="usa-link" aria-describedby="B8">8</a>]).</p>
<p>In parallel, dozens of computational tools have been developed to infer, edit and analyse the dynamical properties of Boolean networks, using different programming languages and model formats (e.g. [<a href="#B9" class="usa-link" aria-describedby="B9">9</a>] and references therein). However, like experimental biologists, modellers are facing serious reproducibility challenges, which have been stressed in recent years (see [<a href="#B10" class="usa-link" aria-describedby="B10">10</a>], focusing on modelling studies using ordinary differential equations).</p>
<p>About a decade ago, several international teams working on the development of Boolean modelling tools gathered efforts to address these challenges. A first effort focused on the definition of a common standard format to foster the exchange and reuse of qualitative models between different software tools, which took the form of a dedicated extension of a new, modular release of the popular <em>SBML</em> format, known as <em>sbml-qual</em> [<a href="#B11" class="usa-link" aria-describedby="B11">11</a>].</p>
<p>A second, recurrent effort is devoted to the integration of a growing set of tools in a <em>Docker</em> container to ease their installation and articulation in flexible analysis workflows. The first release of the resulting <em>Common Logical Modelling Toolbox</em> (or <em>CoLoMoTo</em>) encompassed six complementary software tools [<a href="#B12" class="usa-link" aria-describedby="B12">12</a>]. Since then, the number of tools integrated into the <em>CoLoMoTo</em> suite has been constantly increasing, reaching a total of 24 in 2025. The tools currently integrated in the suite are listed in <a href="#F1" class="usa-link">figure 1</a>, with key characteristics.</p>
<figure class="fig xbox font-sm" id="F1" title="Tools currently integrated in the CoLoMoTo suite."><h3 class="obj_head">Figure 1. </h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371346_rsfs.2025.0002.f001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/400ae1560a49/rsfs.2025.0002.f001.jpg" loading="lazy" height="723" width="760" alt="Tools currently integrated in the CoLoMoTo suite."></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Tools currently integrated in the <em>CoLoMoTo</em> suite. Names in bold refer to the <em>CoLoMoTo</em> tools used in this tutorial.</p></figcaption></figure><p>This tutorial aims to help modellers take advantage of the multiple tools integrated into the <em>CoLoMoTo</em> software suite and perform sophisticated Boolean model analyses of complex signalling/regulatory networks. This tutorial relies on a comprehensive model of key signalling pathways and a core regulatory network controlling mammalian cell proliferation, published by Sizek <em>et al</em>. [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>]. The proposed tasks performed with selected tools aim at comparing the model outcomes with expected experimental behaviours (e.g. existence of cell states in response to receptor activation, cell differentiation, etc.), predicting the results of potential perturbations (such as mutations in patients) or anticipating the effect of treatments.</p>
<p>As we shall see, the tutorial includes chunks of <em>Python</em> code to reproduce several of the results and figures published in the original article, and further extends these results with the help of several tools included in the <em>CoLoMoTo</em> software suite, in particular taking advantage of the <em>MaBoSS</em> package to perform stochastic simulations for the wild-type (WT) and various mutant backgrounds. This tutorial specifically aims to demonstrate the added value of combining different tools to perform reproducible dynamic analyses, and shows the effectiveness of using <em>Python</em> to orchestrate these different analyses.</p></section><section id="s2"><h2 class="pmc_sec_title">2.  Materials and methods</h2>
<section id="s2-1"><h3 class="pmc_sec_title">2.1.  Installation</h3>
<p>The notebook can be run on a recent personal computer with at least 16 Go of RAM and about 10 Go of available disk space.</p>
<p>The <em>CoLoMoTo</em> suite can be installed either as a <em>Docker</em> container or as individual <em>conda</em> packages.</p>
<section id="s2-1-1"><h4 class="pmc_sec_title">2.1.1.  Installation of the <em>CoLoMoTo Docker</em> container</h4>
<p><em>Docker</em> allows packaging a complete pre-installed software environment, so that applications can run consistently across different machines and operating systems. As such, it is well adapted to reproduce complex computational workflows, relying on numerous software.</p>
<p><em>Docker</em> and <em>Python</em> need to be pre-installed. Go to your folder and then type in a terminal:</p>
<figure class="fig xbox font-sm" id="d67e598"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/9f0b02dd3ee5/rsfs.2025.0002.inline001.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline001.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e598/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>where <kbd>$</kbd> stands for the terminal prompt.</p></section><section id="s2-1-2"><h4 class="pmc_sec_title">2.1.2.  Installation of <em>CoLoMoTo conda</em> packages</h4>
<p>As an alternative to using <em>Docker</em>, one can rely on <em>conda</em> to create a software environment able to reproduce the notebook on <em>Linux</em> and <em>macOS</em> computers by typing the following commands in a terminal:</p>
<figure class="fig xbox font-sm" id="d67e628"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/9883c4b6f689/rsfs.2025.0002.inline002.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline002.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e628/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>To launch this notebook, type the command:</p>
<figure class="fig xbox font-sm" id="d67e631"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/a5a57cc80861/rsfs.2025.0002.inline003.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline003.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e631/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Because <em>BNS</em> and <em>boolSim</em> executables are not available for <em>Windows<sup>®</sup></em> operating systems, the complete notebook can be executed only within a <em>Docker</em> environment on these systems.</p>
<p>More generally, as the versions of package dependencies cannot be fully controlled with the aforementioned <em>conda</em> command, reproducibility is better ensured by using the <em>Docker</em> container.</p></section></section><section id="s2-2"><h3 class="pmc_sec_title">2.2.  Model</h3>
<p>The model published by Sizek <em>et al.</em> [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>] has been imported and edited with the <em>GINsim</em> software [<a href="#B14" class="usa-link" aria-describedby="B14">14</a>]. It is publicly available in the GINsim <em>zginml</em> model format (including the layout, to be open with <em>GINsim</em>, version ≥ 3.0), as well as in the <em>sbml-qual</em> format, in the <em>GINsim</em> model repository (<a href="http://ginsim.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://ginsim.org/</a>). Hence, the model can be directly loaded from the notebook using the link to the corresponding <em>GINsim</em> model repository entry. Alternatively, the model can be downloaded locally and the loading command modified accordingly, thereby enabling offline work.</p>
<p>The model was initially conceived as the combination of five main signalling/regulatory modules contributing to the decision between cell proliferation versus death. A first module accounts for the orchestration of the cell cycle, resulting in the successive activation of the main markers of the cell division process, the cyclins. A second module accounts for the control of apoptosis, represented by the activation of Casp3 and the influence of the TRAIL death signal. A third module represents the regulation of growth via the MAPK and PI3k/AKT pathways and their activation by environmental factors. A fourth module represents the G1 restriction point. A fifth module represents the regulation of the origin of replication. Connecting all these modules (displayed with different colours in figure 2), the model can account for many different cellular situations, including genetic alterations common in cancer, and can successfully reproduce observed phenotypes.</p></section></section><section id="s3"><h2 class="pmc_sec_title">3.  Model analysis</h2>
<p>The following sections of the tutorial cover the following steps:</p>
<ul id="list1" class="list" style="list-style-type:none">
<li>
<span class="label">—</span><p class="display-inline">Loading of the packages required for the tutorial.</p>
</li>
<li>
<span class="label">—</span><p class="display-inline">Loading of the model and visualization of the network with the software <em>GINsim</em>.</p>
</li>
<li>
<span class="label">—</span><p class="display-inline">Attractor analysis with the software <em>bioLQM</em>.</p>
</li>
<li>
<span class="label">—</span><p class="display-inline">Computation of synchronous attractors with <em>BNS</em>.</p>
</li>
<li>
<span class="label">—</span><p class="display-inline">Analysis of the attractors of the model modules using <em>bioLQM</em>, <em>BNS</em> and <em>boolSim</em>.</p>
</li>
<li>
<span class="label">—</span><p class="display-inline">Stochastic dynamical simulations of the WT and mutant models with <em>MaBoSS</em>.</p>
</li>
<li>
<span class="label">—</span><p class="display-inline">The construction of compressed probabilistic transition graphs with <em>MaBoSS</em>.</p>
</li>
</ul>
<p>This chaining of steps constitutes a coherent analysis workflow, starting with the identification of attractors (stable states or cycles), considering different updating schemes, before performing more precise and complex stochastic simulations, for the WT case, as well as for selected mutant backgrounds. Each step is performed using one selected software tool, but other <em>CoLoMoTo</em> tools could be used to perform similar analyses (cf. <a href="#F1" class="usa-link">figure 1</a>).</p>
<section id="s3-1"><h3 class="pmc_sec_title">3.1.  Loading required packages</h3>
<p>Before running the analyses, the packages to be used need to be loaded. For this tutorial, only the necessary <em>CoLoMoTo</em> packages are selected. The other packages listed in <a href="#F1" class="usa-link">figure 1</a> can be loaded according to the instructions provided at <a href="https://colomoto.github.io/colomoto-docker/README.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://colomoto.github.io/colomoto-docker/.html</a>. Note that these packages can also be loaded later on, just before their use.</p>
<p>Pieces of code will be shown in this tutorial in boxes as follows. They represent code cells extracted from the <em>Python Jupyter</em> notebook.</p>
<figure class="fig xbox font-sm" id="d67e785"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/edf3a4fe7664/rsfs.2025.0002.inline004.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline004.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e785/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The execution of this cell outputs a few lines specifying what docker image and how many computer resources were used:</p>
<figure class="fig xbox font-sm" id="d67e788"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/87d5a96836e6/rsfs.2025.0002.inline005.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline005.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e788/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="s3-2"><h3 class="pmc_sec_title">3.2.  Loading and visualization of the model with <em>GINsim</em>
</h3>
<p>Prior to this tutorial, the model was first imported using <em>GINsim</em>, in order to redesign its layout and to associate extensive annotations with each regulatory node. The resulting model has the same logical rules as the original one and can be found in the <em>GINsim</em> repository at: <a href="http://ginsim.org/node/258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://ginsim.org/node/258</a>.</p>
<p>To load the model and visualize its <em>regulatory graph</em>, we run the following code cell:</p>
<figure class="fig xbox font-sm" id="d67e815"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/60ed09fc676f/rsfs.2025.0002.inline006.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline006.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e815/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p><a href="#F2" class="usa-link">Figure 2</a> displays the corresponding regulatory graph.</p>
<figure class="fig xbox font-sm" id="F2" title="Boolean model published by Sizek et al"><h4 class="obj_head">Figure 2. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371346_rsfs.2025.0002.f002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/ad9e2aeadffd/rsfs.2025.0002.f002.jpg" loading="lazy" height="840" width="786" alt="Boolean model published by Sizek et al"></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Boolean model published by Sizek <em>et al</em>. [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>]. The corresponding regulatory graph encompasses 87 nodes, representing different molecular species and cellular processes involved in the regulation of cell proliferation and apoptosis in a generic human cell. Green: growth signalling module; red: apoptosis module; yellow: origin of replication licensing module; blue: restriction point module; pink: cell phase control module; orange: nodes connecting different modules/processes; white: environmental factors. Green arrows and red blunt arcs denote activatory and inhibitory interactions, respectively.</p></figcaption></figure><p>The regulatory graph shown in <a href="#F2" class="usa-link">figure 2</a> encompasses 87 nodes representing different molecular species and cellular processes involved in the regulation of cell proliferation and apoptosis in a generic human cell. The authors conceived this complex model as an association of five main functional modules:</p>
<ul id="list2" class="list" style="list-style-type:none">
<li>
<span class="label">—</span><p class="display-inline">The growth signalling module (green nodes) incorporates growth signalling pathways driving cell cycle commitment, responsible for modelling the dynamics of PI3K, AKT1, MAPK and mTORC.</p>
</li>
<li>
<span class="label">—</span><p class="display-inline">The restriction switch module (blue nodes) is responsible for commitment to DNA synthesis.</p>
</li>
<li>
<span class="label">—</span><p class="display-inline">The origin licensing switch module (yellow nodes) controls licensing and firing of replication origins.</p>
</li>
<li>
<span class="label">—</span><p class="display-inline">The phase switch module (pink nodes) controls cell cycle progression (G1 → S → G2 → M), taking into account the role of Polo-like kinase 1 (Plk1) during mitosis.</p>
</li>
<li>
<span class="label">—</span><p class="display-inline">The apoptosis switch module (red nodes) implements the decision between cell survival versus apoptosis.</p>
</li>
</ul>
<p>In the regulatory graph, green arrows denote activatory interactions, while red blunt arcs represent inhibitions. As many model nodes are regulated by multiple nodes, their responses to regulatory input node levels are encoded in Boolean rules, which combine literals (i.e. node ids) with the classical Boolean operators NOT (written ‘!’ in <em>GINsim</em>), OR (‘|’) and AND (‘&amp;’). The rules defined in the initial publication were encoded in the <em>zginml</em> file available in the <em>GINsim</em> model repository.</p></section><section id="s3-3"><h3 class="pmc_sec_title">3.3.  Attractor analysis with <em>bioLQM</em>
</h3>
<p><em>Attractors</em> represent the long-term (<em>asymptotic</em>) behaviours of the model. In the case of Boolean models, attractors can correspond to stable states (with each of the components frozen at level 0 or 1) or to periodic/cyclic behaviour (with a subset of components switching between values 0 or 1). Attractors are often associated with cellular phenotypes, cell fates (e.g. cell proliferation or cell death), or cellular types in the case of cell differentiation models. Their analysis allows the modeller to verify that the model reproduces the dynamical properties expected for the biological system under study, for normal or perturbed conditions.</p>
<p>To obtain a first overview of the asymptotic properties of the model, we can use the software tool <em>bioLQM</em> [<a href="#B15" class="usa-link" aria-describedby="B15">15</a>], which includes an efficient algorithm to identify all stable states [<a href="#B16" class="usa-link" aria-describedby="B16">16</a>]. The model is first converted into the <em>bioLQM</em> format, which can be achieved with the following code cell:</p>
<figure class="fig xbox font-sm" id="d67e899"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/27e082cc8ce7/rsfs.2025.0002.inline007.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline007.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e899/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Using <em>bioLQM</em>, the <em>stable states</em> of the model can be computed by the following code cell:</p>
<figure class="fig xbox font-sm" id="d67e909"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/37f04997fa61/rsfs.2025.0002.inline008.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline008.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e909/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The execution of this code cell takes less than a second and outputs a table listing the eight stable states (also called <em>fixed points</em>) of the model (shown in the notebook), matching those reported by Sizek <em>et al</em>. in the supplementary table S2 of their article.</p>
<p>To better visualize any of these stable states, the corresponding values can be projected on the model regulatory graph with a simple code line, using <em>GINsim</em>:</p>
<figure class="fig xbox font-sm" id="d67e924"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/103d57bd8bb3/rsfs.2025.0002.inline009.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline009.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e924/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Such a visualization highlights the pathways/modules that are active in one particular stable state. An example of such stable state projection is provided in <a href="#F3" class="usa-link">figure 3</a>.</p>
<figure class="fig xbox font-sm" id="F3" title="Graphical representation of a stable state of the Boolean model published by Sizek et al."><h4 class="obj_head">Figure 3. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371346_rsfs.2025.0002.f003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/ddbc500aa955/rsfs.2025.0002.f003.jpg" loading="lazy" height="840" width="786" alt="Graphical representation of a stable state of the Boolean model published by Sizek et al."></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Graphical representation of a stable state of the Boolean model published by Sizek <em>et al</em>. [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>] corresponding to a cell death phenotype. Active nodes and their outgoing edges are displayed in blue.</p></figcaption></figure></section><section id="s3-4"><h3 class="pmc_sec_title">3.4.  Computation of synchronous attractors with <em>BNS</em>
</h3>
<p>In their publication, Sizek <em>et al</em>. report the existence of a cyclic behaviour when a synchronous simulation method is used. Synchronous simulations simultaneously update all the components at a given state, according to the logical rules, leading to a unique successor state. This is a strong assumption, as in reality, timing plays an important role in biology and concurrent events barely occur simultaneously.</p>
<p>In contrast, asynchronous simulations typically update only one component at each time step, either selecting randomly one component to update, or yet considering separately all single update calls, potentially giving rise to more complex, non-deterministic dynamics.</p>
<p>Of note, for a given Boolean model, the stable states are not affected by the choice of updating, as stable states are devoid of update calls.</p>
<p>Cyclic attractors correspond to a circular sequence of events, such as the activation of cyclins, markers of the cell cycle. Cyclic attractors may depend on the choice of updating, and their identification is more complex in the asynchronous updating case.</p>
<p><em>BNS</em> can be used to compute the synchronous attractors of the model (i.e. fixed points, as well as simple cyclic trajectories in the synchronous case). Although <em>BNS</em> can search for attractors of any length, this search fails with such a complex model. To overcome this limitation, a range of cycle lengths is defined, and attractors for the corresponding lengths are explored sequentially.</p>
<p>In the following code cell, we consider a maximum length of 32, and use <em>BNS</em> to compute all cycles of lengths between 1 to this maximum length. Of note, an attractor of length k is also an attractor of length 2k, 3k, etc. In particular, fixed points are detected as attractors of any length. Hence, in the following code cell, we further filter the attractors that amount to iterations of smaller ones.</p>
<figure class="fig xbox font-sm" id="d67e977"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/d25c382f83df/rsfs.2025.0002.inline010.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline010.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e977/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>In addition to the eight fixed points already identified (corresponding to <em>cyclic attractors of length 1</em>), <em>BNS</em> returns four attractors of length 10, and one attractor of length 21, which correspond to the cyclic attractor reported by Sizek <em>et al</em>. in electronic supplementary material, table S2, supposedly representing the successive phases of the cell cycle: G1 from step 1 to 5, S for steps 6 and 7, G2 from step 8 to 13, prometaphase for step 14, metaphase from step 15 to 18, anaphase for step 19, telophase for step 20 and cytokinesis for step 21.</p>
<p>However, the interpretation of cyclic attractors under the synchronous updating strategy is not that straightforward. Indeed, the cyclic attractors of length 10 might be artefactual, as synchronous updating is known to give rise to spurious cycles.</p>
<p>In the following section, we further characterize the attractors of the five main constitutive regulatory modules, as well as the presence of coupling between these modules.</p>
<p>In §3.7, we will compute the probability of cyclic attractors under an asynchronous mode to follow the successive activations of the cyclins, each representing a phase of the cell cycle.</p></section><section id="s3-5"><h3 class="pmc_sec_title">3.5.  Analysis of the model constitutive modules</h3>
<p>To generate their complex model, Sizek e<em>t al</em>. initially developed smaller models for different functional modules (cf. §2). These modules correspond to processes such as DNA damage, cell cycle, MAPK pathway or apoptosis and encompass the main genes that are known to be involved in these processes. They are defined by the authors in electronic supplementary material. The following code cell retrieves the information on the node composition of each module, counts the total number of nodes belonging to each of these modules, together with the number of nodes that belong to the rest of the network and that regulate nodes belonging to the module (also known as module <em>inputs</em>). It then computes the number of stable states, the number of terminal trap spaces (i.e. sub-hypercubes closed by the logical rules, comprising the stable states, but potentially also approximations to cyclic attractors), as well as the numbers of synchronous and asynchronous attractors for each of these modules.</p>
<figure class="fig xbox font-sm" id="d67e1008"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/5a946a6b89f9/rsfs.2025.0002.inline011.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline011.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The execution of this code cell takes less than a second to generate the results reported in <a href="#T1" class="usa-link">table 1</a>.</p>
<section class="tw xbox font-sm" id="T1"><h4 class="obj_head">Table 1. </h4>
<div class="caption p"><p>Some properties of the five modules constitutive of the model of Sizek <em>et al</em>. [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>].</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="left" valign="bottom" rowspan="1" colspan="1"></th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<p>restriction switch</p>
</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<p>apoptotic switch</p>
</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<p>PI3K</p>
</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<p>origin licensing switch</p>
</th>
<th align="left" valign="bottom" rowspan="1" colspan="1">
<p>phase switch</p>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>number of nodes</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>7</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>16</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>8</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>4</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>15</p>
</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>number of input nodes</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>14</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>13</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>11</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>11</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>16</p>
</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>number of stable states</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>2</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>2</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>0</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>2</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>3</p>
</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>number of terminal traps</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>2</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>2</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>1</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>2</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>3</p>
</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>dynchronous attractors</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>2</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>2</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>1</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>2</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>3</p>
</td>
</tr>
<tr>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>asynchronous attractors</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>2</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>2</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>1</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>2</p>
</td>
<td align="left" valign="bottom" rowspan="1" colspan="1">
<p>3</p>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>We can observe that the modules are highly interconnected, as they all have high numbers of inputs relative to their sizes, due to regulations exerted by nodes belonging to other modules.</p>
<p>Furthermore, all these modules but one display multistability, but no cyclic behaviour. The PI3K module is the exception, as it has a unique, cyclic attractor, regardless of the use of synchronous or asynchronous updating. The cyclic behaviour of the PI3K module is indeed reported and discussed by Sizek <em>et al.</em> [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>].</p>
<p>In conclusion, the attractors found for the different modules do not help us understand the mechanisms underlying the cyclic behaviour of the whole model. More precisely, at this point, it becomes apparent that the global cyclic behaviour corresponding to the cell cycle (see §3.4) arises from the interconnections of multiple modules.</p></section><section id="s3-6"><h3 class="pmc_sec_title">3.6.  Characterization of synchronous cyclic properties with <em>bioLQM</em>
</h3>
<p>To interpret the meaning of a cyclic attractor, it is important to check which variables are fixed in this attractor and which ones switch values.</p>
<p>To visualize the 21 states composing the synchronous cyclic attractor, <em>bioLQM</em> can be used to generate a trace from one of the cycle states reported by Sizek <em>et al.</em> [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>], considering a number of steps somewhat greater than 21. First, we define an initial state belonging to the cyclic attractor, according to Sizek <em>et al.</em> [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>], with the following code cell:</p>
<figure class="fig xbox font-sm" id="d67e1248"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/2dc6fd007b45/rsfs.2025.0002.inline012.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline012.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1248/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Next, we compute a synchronous updating trace, starting with this initial state and considering a maximum of 50 steps, with the following code cell:</p>
<figure class="fig xbox font-sm" id="d67e1251"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/f96f65c343ba/rsfs.2025.0002.inline013.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline013.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1251/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The execution of this cell takes less than a second and generates a table encompassing 22 rows (cf. notebook), each corresponding to one state belonging to a cyclic sequence. We can easily check that the first and last states are indeed identical with the following code cell:</p>
<figure class="fig xbox font-sm" id="d67e1254"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/4c435bd06cda/rsfs.2025.0002.inline014.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline014.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1254/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The execution of this cell confirms that we have identified the cyclic attractor of length 21 mentioned above. To ease the comparison of this sequence of states with the periodic pattern reported by Sizek <em>et al.</em> [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>] in fig. 4, we can plot it with similar graphical conventions, using the following code cell:</p>
<figure class="fig xbox font-sm" id="d67e1264"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/4bf774a1214c/rsfs.2025.0002.inline015.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline015.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1264/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The resulting heatmap is shown in <a href="#F4" class="usa-link">figure 4</a>, exhibiting the full states sequence of the aforementioned cyclic attractor.</p>
<figure class="fig xbox font-sm" id="F4" title="Heat map displaying the sequence of 22 states (columns) forming the synchronous cycle of length 21, with the initial state (first column, index 0) matching the last one (column with index 21)"><h4 class="obj_head">Figure 4. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/3deb91e5e5cf/rsfs.2025.0002.f004.jpg" loading="lazy" height="1041" width="594" alt="Heat map displaying the sequence of 22 states (columns) forming the synchronous cycle of length 21, with the initial state (first column, index 0) matching the last one (column with index 21)"></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Heat map displaying the sequence of 22 states (columns) forming the synchronous cycle of length 21, with the initial state (first column, index 0) matching the last one (column with index 21). Orange cells denote the active components (rows) at the corresponding cycle state (column), whereas the blue cells denote the suppressed components.</p></figcaption></figure><p>Note that some nodes are never activated along the cycle, while others are always activated. In particular, the apoptotic pathway (BIM, BAX, BAK, CytC, Casp3, Casp9, etc.) is inactive in this sequence, while the cell cycle is activated by the MAPK pathway (RTK, Ras, Raf, PI3K, etc.).</p>
<p>Hence, using different tools, we could also identify the synchronous attractor reported by Sizek <em>et al.</em> [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>], which recapitulates the typical sequence of (in)activation of cell components, including cyclins. However, as this synchronous cyclic attractor is not expected to precisely reproduce the biological cyclic behaviour, a deeper analysis of the timing of cyclin (in)activations will be considered using asynchronous updating in §3.7.</p>
<p>Of note, Sizek <em>et al.</em> [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>] reported that the observed cyclic behaviour is sensitive to the updating strategy used. To further characterize this sensitivity, <em>bioLQM</em> can be further used to compute traces starting from the same initial state but using random asynchronous updating, i.e. selecting one node among all nodes called to be updated at each state, with identical probabilities assigned to all single node transitions. This can be achieved using the following code cell:</p>
<figure class="fig xbox font-sm" id="d67e1301"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/210b52bd74ac/rsfs.2025.0002.inline016.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline016.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1301/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>This cell takes a few seconds to execute. Note that a longer maximal number of steps (500) is considered here because we do not know if the trace will end up in an asynchronous cycle, or how long such a cycle might be. A typical result is shown in <a href="#F5" class="usa-link">figure 5</a>.</p>
<figure class="fig xbox font-sm" id="F5" title="Heat map displaying a sequence of 500 states (columns) resulting from a random asynchronous updating of the model, starting with the same initial state shown in figure 4"><h4 class="obj_head">Figure 5. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/74d37903c403/rsfs.2025.0002.f005.jpg" loading="lazy" height="1050" width="596" alt="Heat map displaying a sequence of 500 states (columns) resulting from a random asynchronous updating of the model, starting with the same initial state shown in figure 4"></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Heat map displaying a sequence of 500 states (columns) resulting from a random asynchronous updating of the model, starting with the same initial state shown in <a href="#F4" class="usa-link">figure 4</a>. Orange cells denote the active components (rows) at the corresponding cycle state (column), whereas the blue cells denote the suppressed components.</p></figcaption></figure><p>In the heatmap shown in <a href="#F5" class="usa-link">figure 5</a>, we can observe a transient oscillating pattern, but this pattern gets progressively degraded along the simulation. This is due to the random choice of transitions whenever several components are called to switch their values at a given state, according to their logical rules. Of note, if we repeat the same random asynchronous simulation, we typically end up with a different transient oscillating pattern.</p>
<p>Interestingly, when large enough maximal step numbers are considered, simulations ultimately lead to a stable state corresponding to an apoptotic fate. To visualize this, in the notebook, we provide the code for a simulation using a maximum number of steps set to 2000, which needs to be uncommented to be run.</p>
<p>In their paper, the authors defined a biased transition order to enforce a more robust cyclic behaviour. Hereafter, we propose an alternative approach to deepen our understanding of the asynchronous behaviour of the model, taking advantage of the software <em>MaBoSS</em> [<a href="#B17" class="usa-link" aria-describedby="B17">17</a>] to integrate over a large number of potentially different traces.</p></section><section id="s3-7"><h3 class="pmc_sec_title">3.7.  Using <em>MaBoSS</em> software to perform stochastic simulations of the wild-type model</h3>
<p>It is difficult to build the asynchronous state transition graph and analyse the transient dynamics of such complex models. To cope with this issue, we use <em>MaBoSS</em>, a tool performing stochastic simulations over Boolean networks based on the continuous-time <em>Markov chain</em> formalism, relying on the <em>Gillespie algorithm</em> [<a href="#B17" class="usa-link" aria-describedby="B17">17</a>]. The software enables the generation of <em>time plots</em> showing the evolution of mean component values (or patterns thereof), as well as of <em>pie charts</em> showing the probabilities of the different model states reached at the end of the simulation.</p>
<p>The <em>bioLQM</em> model can be converted into the format required by <em>MaBoSS</em> with the following code cell:</p>
<figure class="fig xbox font-sm" id="d67e1375"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/e7445cb56506/rsfs.2025.0002.inline017.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline017.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1375/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>After loading the model, simulation parameters can be modified with the following code cell:</p>
<figure class="fig xbox font-sm" id="d67e1378"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/bded0f37a599/rsfs.2025.0002.inline018.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline018.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1378/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>This code cell defines the following simulation parameters:</p>
<ul id="list3" class="list" style="list-style-type:none">
<li>
<span class="label">—</span><p class="display-inline">Number of simulations: 5000.</p>
</li>
<li>
<span class="label">—</span><p class="display-inline">Maximum length of the simulation (in arbitrary units): 200.</p>
</li>
<li>
<span class="label">—</span><p class="display-inline">Interval between two time points: 0.5.</p>
</li>
</ul>
<p>Note that <em>MaBoSS</em> uses continuous-time Boolean modelling and requires the specification of a maximum simulation length and an interval between time points. All the stochastic simulations reported in the notebook use these same simulation parameters.</p>
<p>Next, we define a WT version of the model, called <em>WT</em>, and we select the <em>external variables</em>, i.e. the variables for which the values will be reported during the simulation. This is crucial to avoid combinatorial explosion.</p>
<figure class="fig xbox font-sm" id="d67e1405"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/09c4f582defc/rsfs.2025.0002.inline019.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline019.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1405/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Note that when exporting a model to <em>MaBoSS</em> with <em>bioLQM</em> or <em>GINsim</em>, all the initial values are set to 0 by default. Hereafter, we assign to the model the initial conditions defined by Sizek <em>et al.</em> [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>] in their fig. 6.</p>
<figure class="fig xbox font-sm" id="d67e1424"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/cb426ccc564a/rsfs.2025.0002.inline020.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline020.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1424/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p><em>MaBoSS</em> simulation results can be graphically displayed as <em>time plots</em> showing the probabilities of active nodes or model states (vectors of active nodes) over time, or as <em>pie charts</em> showing the probabilities of the final states (i.e. the states reached at the end of the simulations). In brief, the trajectories plotted on these graphs represent mean trajectories of 5000 single trajectories focusing on a subset of seven selected nodes. It is possible to visualize the activity of each of these nodes over time (<a href="#F6" class="usa-link">figure 6</a>, upper left), or the evolution of model states defined as configurations of active nodes (<a href="#F6" class="usa-link">figure 6</a>, upper right), which can also be displayed as a pie chart for the last time point considered (<a href="#F6" class="usa-link">figure 6</a>, lower left).</p>
<figure class="fig xbox font-sm" id="F6" title="MaBoSS simulations of the WT model."><h4 class="obj_head">Figure 6. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371346_rsfs.2025.0002.f006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/ccfa51c92c2e/rsfs.2025.0002.f006.jpg" loading="lazy" height="570" width="769" alt="MaBoSS simulations of the WT model."></a></p>
<div class="p text-right font-secondary"><a href="figure/F6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><em>MaBoSS</em> simulations of the WT model. Upper left panel: trajectories for a subset of nodes. Upper right panel: trajectories for a subset of model states (legend in the lower left panel). Lower left panel: pie chart representing the probabilities for the model states at maximum time (time = 200). Lower right panel: entropy (H) and transition entropy (TH) time plots.</p></figcaption></figure><p>It is also possible to compute the <em>state entropy</em> (H) and the <em>transition entropy</em> (TH) over time (<a href="#F6" class="usa-link">figure 6</a>, lower right), which can be interpreted as signatures for the stable states or cyclic attractors. These curves provide a means to check the nature of the attractors of the model. Entropy measures the ‘disorder’ of the system studied. A very low transition entropy at the end of a simulation thus suggests that the system has reached an equilibrium, while a positive state entropy may suggest the existence of a cyclic attractor or the occurrence of multiple stable states (for more details, see [<a href="#B18" class="usa-link" aria-describedby="B18">18</a>]).</p>
<p>The following code cell launches a simulation of the WT model with <em>MaBoSS</em>, using the parameters defined above:</p>
<figure class="fig xbox font-sm" id="d67e1476"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/755800554e4a/rsfs.2025.0002.inline021.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline021.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1476/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The execution of this code cell takes less than a minute. To graphically visualize the results of this simulation in terms of node or model states trajectories, of final model states repartition, and of entropy curves, we can use the following code cell:</p>
<figure class="fig xbox font-sm" id="d67e1479"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/bd7002d4eb24/rsfs.2025.0002.inline022.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline022.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1479/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The resulting graphs are shown in <a href="#F6" class="usa-link">figure 6</a>.</p>
<p>The simulations reveal a transient cyclic-like behaviour, which is eventually lost due to the spontaneous Casp3 activation, thereby revealing the progressive dominance of the apoptotic stable state.</p>
<p>The state entropy initially rises abruptly, and later slowly decreases. In parallel, the transition entropy curve displays a little peak before remaining close to zero, with some noise. Together, these two curves are suggestive of a transient periodic behaviour, which ultimately collapses as more and more cells trigger apoptosis.</p>
<p>Next, we can use <em>MaBoSS</em> to study the impact of changes in the initial conditions on the dynamic behaviour of the model. In the preceding simulation, all death signals were initially OFF. Hereafter, we assess the effect of activating TRAIL, a ligand binding to death receptors of tumour cells and initiating the death pathways, while keeping the rest of the other initial conditions unchanged. Note that TRAIL is an input of the death module and thus not regulated. Such inputs are set to either 0 or 1 in the initial conditions, thereby conveying the status of the environment. The following code cell defines a copy of the WT model, triggers TRAIL level to 1 at the initial state, performs the simulation and finally displays the results:</p>
<figure class="fig xbox font-sm" id="d67e1494"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/48411d01658e/rsfs.2025.0002.inline023.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline023.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1494/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The resulting graphs are displayed in <a href="#F7" class="usa-link">figure 7</a>.</p>
<figure class="fig xbox font-sm" id="F7" title="MaBoSS simulations for the WT condition with death receptor signals active. Upper left panel: Trajectories for a subset of nodes."><h4 class="obj_head">Figure 7. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371346_rsfs.2025.0002.f007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/88aef721783d/rsfs.2025.0002.f007.jpg" loading="lazy" height="567" width="766" alt="MaBoSS simulations for the WT condition with death receptor signals active. Upper left panel: Trajectories for a subset of nodes."></a></p>
<div class="p text-right font-secondary"><a href="figure/F7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><em>MaBoSS</em> simulations for the WT condition with death receptor signals active. Upper left panel: trajectories for a subset of nodes. Upper right panel: trajectories for a subset of model states. Lower left panel: pie chart displaying the probabilities for the model states at maximum time (time = 200). Lower right panel: entropy (H) and transition entropy (TH) time plots.</p></figcaption></figure><p>Looking at the graphs of <a href="#F7" class="usa-link">figure 7</a>, we can see that the activation of TRAIL at the initial state enables a faster rise of Casp3 and thus of the apoptotic fate. In parallel, the state entropy drops much faster, while the transition entropy stabilizes around zero, suggesting a stable state situation.</p></section><section id="s3-8"><h3 class="pmc_sec_title">3.8.  Mutant simulations with <em>MaBoSS</em>
</h3>
<p>In their article, Sizek <em>et al.</em> report a sophisticated analysis of the role and dynamical properties of the PI3K/AKT1 pathway, and recapitulate deleterious effects of alterations thereof. They further analyse the impact of the blocking of the Polo-like kinase 1 (Plk1, a mitotic driver and chemotherapy target) on cell cycle progression.</p>
<p>Hereafter, we focus on the analysis of the impact of the perturbations of other model components, using the <em>MaBoSS</em> software. Such perturbations can be understood as mutations that patients bear, or as targets of drugs that would suppress proliferation or increase apoptosis.</p>
<section id="s3-8-1"><h4 class="pmc_sec_title">3.8.1.  Stochastic simulation of ectopic Casp8 activity</h4>
<p>Casp8 is a caspase initiating the apoptotic cascade. By changing its activity (ectopic activity or knockdown), we can explore its role in triggering cell death. Using the following code cell, we first simulate the impact of a constant activation of Casp8 and display the resulting time plot and pie chart.</p>
<figure class="fig xbox font-sm" id="d67e1544"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/49810fccefbe/rsfs.2025.0002.inline024.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline024.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1544/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p><a href="#F8" class="usa-link">Figure 8</a> (left) shows the resulting plots. Based on these results, we can conclude that the ectopic expression of Casp8 drives the cells into apoptosis faster, whether the death signal (TRAIL) is ON or OFF. This shows that death is not dependent on external signals in this context: the cell becomes independent from the death receptor engagement and apoptosis is constantly activated.</p>
<figure class="fig xbox font-sm" id="F8" title="MaBoSS simulations for Casp 8 mutants. Left panels: Node trajectories and pie chart showing the asymptotic solutions for Casp8 overexpression."><h5 class="obj_head">Figure 8. </h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371346_rsfs.2025.0002.f008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/69deb2701389/rsfs.2025.0002.f008.jpg" loading="lazy" height="551" width="787" alt="MaBoSS simulations for Casp 8 mutants. Left panels: Node trajectories and pie chart showing the asymptotic solutions for Casp8 overexpression."></a></p>
<div class="p text-right font-secondary"><a href="figure/F8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><em>MaBoSS</em> simulations for Casp8 mutants. Left panels: node trajectories and pie chart showing the asymptotic solutions for Casp8 overexpression. Right panels: node trajectories and pie chart representing the probabilities for the model states at the end of simulations for Casp8 knockdown.</p></figcaption></figure></section><section id="s3-8-2"><h4 class="pmc_sec_title">3.8.2.  Stochastic simulation of Casp8 knockdown</h4>
<p>We can also check the impact of a full knockdown of Casp8 by executing the following code cell:</p>
<figure class="fig xbox font-sm" id="d67e1569"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/eacd1590f4fb/rsfs.2025.0002.inline025.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline025.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1569/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>As shown in <a href="#F8" class="usa-link">figure 8</a> (right), Casp3 is initially blocked but becomes ultimately activated, like in the WT case, independently of the status of Casp8. Hence, the model predicts that apoptosis (represented by Casp3 activity) can ultimately occur even in the absence of Casp8 activity. The knockout of Casp8 does not affect the model in the absence or presence of TRAIL.</p>
<p>Of note, it is possible to knock out a specific interaction with <em>bioLQM</em>, using the syntax: ‘target:regulator%0’, denoting a blockade of the interaction exerted by the regulator onto the target:</p>
<figure class="fig xbox font-sm" id="d67e1581"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/f2bb2bda0d08/rsfs.2025.0002.inline026.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline026.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1581/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Using a bit of <em>Python</em> code, it is further possible to systematically explore a series of mutants, and to filter the results. In this respect, the notebook provides an example of such code, considering 10 mutant simulations (inhibition and activation of five different nodes), and filtering those enabling a blockade of apoptosis, denoted by the suppression of Casp3 activation, while keeping the cell cycle active. This would correspond to a highly proliferative phenotype, with some alterations correlating with cancer patient conditions.</p>
<p>Overall, this analysis shows the important but not necessary role of Casp8 in triggering apoptosis in this model. The gradual activation of Casp3, even in the absence of Casp8, may be an artefact of the model, or could be interpreted as an intrinsic propensity of cells to die after multiple rounds of divisions. In the latter case, the model would benefit from the inclusion of more details regarding the mechanisms triggering cell death.</p></section></section><section id="s3-9"><h3 class="pmc_sec_title">3.9.  Deeper analysis of the quasi-cyclic synchronous attractor</h3>
<p>In this section, we explore the dynamics of the cell cycle by studying the succession of cyclin activations and inactivations. In continuous frameworks, i.e. models based on ordinary differential equations, it is possible to follow the activations of the cyclins that are crucial to ensure a proper cell cycle. In the WT situation, we expect that the cyclins get activated in the following order: CyclinE, followed by CyclinA, then CyclinB, and get inactivated in the same order. This sequence is not based on a simple cascade, but on complex regulation where new phases are controlled by checkpoints.</p>
<p>In the case of stochastic Boolean simulations, the verification of this sequence of cyclin activations is more challenging. In this respect, we have developed a functionality reporting the probabilities of occurrences of specific node level transitions, thereby enabling the delineation of highly probable orders of activations and inactivations of cyclins over time.</p>
<section id="s3-9-1"><h4 class="pmc_sec_title">3.9.1.  Verification of the order of activations of cyclins in the wild-type situation by plotting a compressed state transition graph</h4>
<p>The following code cell defines a simulation of the WT model focusing on cyclin dynamics:</p>
<figure class="fig xbox font-sm" id="d67e1604"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/810660c23f5d/rsfs.2025.0002.inline027.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline027.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1604/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The resulting time plot is shown in <a href="#F9" class="usa-link">figure 9</a> (top left). We can see that the cyclins follow the expected order with the proper sequence of peaks: first CyclinE around <em>t</em> = 5, then CyclinA around <em>t</em> = 10, and finally CyclinB around <em>t</em> = 15. Note that as <em>MaBoSS</em> reports mean probabilities over time, these oscillations are rapidly damped.</p>
<figure class="fig xbox font-sm" id="F9" title="Left: MaBoSS simulation of the WT condition, showing the mean probabilities of cyclin activations and inactivations (upper left panel), along with a compressed state transition graph (lower left panel)."><h5 class="obj_head">Figure 9. </h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371346_rsfs.2025.0002.f009.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/7a990cb7a7cb/rsfs.2025.0002.f009.jpg" loading="lazy" height="368" width="779" alt="Left: MaBoSS simulation of the WT condition, showing the mean probabilities of cyclin activations and inactivations (upper left panel), along with a compressed state transition graph (lower left panel)."></a></p>
<div class="p text-right font-secondary"><a href="figure/F9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Left: <em>MaBoSS</em> simulation of the WT condition, showing the mean probabilities of cyclin activations and inactivations (upper left panel), along with a compressed state transition graph (lower left panel). Middle: <em>MaBoSS</em> simulation of the WT condition, showing the mean probabilities of cyclin and Casp3 activations and inactivations (upper middle panel), along with a compressed state transition graph (lower middle panel). Right: <em>MaBoSS</em> simulation of the p110++mutant, showing the mean probabilities of cyclin and Casp3 activations and inactivations (upper right panel), along with a compressed state transition graph (lower right panel). In the compressed state transition graphs, each node represents a set of states characterized by a specific combination of activated markers (i.e. cyclins A, B and E, plus Casp3 in the middle lower panel), whereas the colours of the edges denote the probabilities of transitions between pairs of sets of states (light blue: low probability; dark blue: high probability).</p></figcaption></figure><p>We can further compute the frequencies of cyclin activations and inactivations over all individual trajectories of the simulation. To do this, we use a new functionality of <em>MaBoSS</em>, estimating the frequencies of transitions between <em>activity patterns</em> involving selected nodes, and plot them as a graph. To prevent the explosion of the size of this graph, we only use the three main cyclins responsible for the cyclic attractor. The following code cell computes such transition frequencies and displays the results as a <em>compressed state transition graph</em>, i.e. a projection of the transition frequencies computed over the full state space onto the cyclin activity patterns of interest.</p>
<figure class="fig xbox font-sm" id="d67e1634"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/5e58526e3534/rsfs.2025.0002.inline028.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline028.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1634/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The resulting transition graph is shown in <a href="#F9" class="usa-link">figure 9</a> (bottom left).</p>
<p>In this graph, the transitions denoted by darker blue arrows have a higher probability of occurring. This representation enables us to visualize the most probable sequence of cyclin activations and inactivations. Starting from the state ‘&lt;nil&gt;’ (corresponding to G0), the most probable sequence of state transitions successively involves CyclinE activation, CyclinA activation, CyclinE inactivation, CyclinB activation, CyclinA inactivation and finally CyclinB inactivation, thereby completing the cycle. Note that other cycles can be followed with lower probabilities.</p></section><section id="s3-9-2"><h4 class="pmc_sec_title">3.9.2.  Analysis of the apoptosis triggering timing along the cell cycle</h4>
<p>Here, we study the condition leading to apoptotic death, focusing on the point in the cell cycle where Casp3 is activated. We can achieve this using the following code cell:</p>
<figure class="fig xbox font-sm" id="d67e1668"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/11e2dfbe2bc8/rsfs.2025.0002.inline029.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline029.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1668/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The resulting graphs are shown in <a href="#F9" class="usa-link">figure 9</a> (middle). In the time plot (<a href="#F9" class="usa-link">figure 9</a>, middle top), we can observe that cells initially follow the expected cycle, but quickly activate Casp3 and are thus led to apoptosis.</p>
<p>Turning to the compressed transition graph (<a href="#F9" class="usa-link">figure 9</a>, middle bottom), we used a minimum probability threshold (2%) to discard unlikely edges during plotting. We see that there are two main ways to activate apoptosis: from the node ‘&lt;nil&gt;’ (with no active cyclins), which presumably represents the G0 state, and from the node ‘cyclinB’ (with only cyclin B active), which tentatively represents the G2/M phase.</p></section><section id="s3-9-3"><h4 class="pmc_sec_title">3.9.3.  Analysis of cyclin triggering timing in presence of ectopic p110_H activity</h4>
<p>In their article, Sizek <em>et al.</em> [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>] reported that p110 activates both PI3K and AKT, which in turn inhibits cell death. In the mutant analyses performed by these authors, p110 played a key role in the interaction between the MAPK module and the apoptotic module. To verify the impact of a forced p110_H activation, we can execute the following code cell:</p>
<figure class="fig xbox font-sm" id="d67e1697"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8744/12371346/7aaec5d4cd15/rsfs.2025.0002.inline030.jpg" loading="lazy" alt="graphic file with name rsfs.2025.0002.inline030.jpg"></p>
<div class="p text-right font-secondary"><a href="figure/d67e1697/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>As can be seen in the resulting graph (<a href="#F9" class="usa-link">figure 9</a>, right), the ectopic activity of p110_H completely disables the activation of apoptosis, enabling the maintenance of the oscillatory behaviour.</p></section></section></section><section id="s4"><h2 class="pmc_sec_title">4.  Conclusions</h2>
<p>The <em>CoLoMoTo</em> software suite integrates over 20 tools written in multiple languages, and makes them accessible with a single popular language (<em>Python</em>). After providing the instructions to install the suite, this tutorial focused on the analysis of a published model of the regulatory network controlling mammalian cell proliferation [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>], which can be further used as a template for the analysis of any other Boolean model encoded in one of the multiple supported formats (e.g. <em>sbml-qual</em>).</p>
<p>The tutorial includes <em>Python</em> code enabling the reproduction of several of the results reported by the initial publication [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>], as well as to further extend these results with new stochastic simulation results for WT cells, as well as for various single and double mutants. Using selected tools included in the <em>CoLoMoTo</em> suite, we showed how to compute the model attractors with synchronous or asynchronous updating. Although stable states are consistently preserved, our analysis emphasizes the sensitivity of synchronous cyclic attractors to the update mode.</p>
<p>We further showed how the use of a probabilistic simulation framework, implemented in the <em>MaBoSS</em> software, can provide interesting information on the transient behaviour of such a complex model, for WT conditions, as well as for different types of model perturbations.</p>
<p>Integrating all these analyses into an executable <em>Jupyter Notebook</em> greatly facilitates their reproducibility, as well as extensions. The notebook can also be used as a template for encoding completely new model analyses.</p>
<p>Due to space constraints, this notebook explicitly covers the use of only a fraction of the tools available in the <em>CoLoMoTo</em> suite. However, all integrated tools are accompanied by short specific tutorials available in the <em>CoLoMoTo Docker</em> container, as well as on the <em>CoLoMoTo</em> website (<a href="https://colomoto.github.io/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://colomoto.github.io/</a>). The analysis presented in this tutorial can be easily modified or extended by taking advantage of other tools included in the <em>CoLoMoTo</em> suite. For example, simulations with different updating modes can also be performed with <em>mpbn</em> or <em>BooleanNet</em>; <em>BooN</em> can be used for stable state analysis; <em>Pint</em> can be used to perform automatic search of mutations enforcing or impeding the reachability of a specific attractor; models can be imported from other databases, such as <em>Cell Collective</em>, in different formats, such as <em>BoolSim</em> or <em>BoolNet</em> formats.</p></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We are grateful to Claudine Chaouiya and Pedro Monteiro for their key contributions to the development of <em>GINsim</em> and to our many colleagues who contributed tools integrated in the <em>CoLoMoTo</em> suite.</p></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Vincent Noël, Email: vincent.noel@curie.fr.</p>
<p>Aurélien Naldi, Email: aurelien.naldi@gmail.com.</p>
<p>Laurence Calzone, Email: laurence.calzone@curie.fr.</p>
<p>Loic Paulevé, Email: loic.pauleve@labri.fr.</p>
<p>Denis Thieffry, Email: denis.thieffry@ens.fr.</p></section><section id="s5"><h2 class="pmc_sec_title">Ethics</h2>
<p>This work did not require ethical approval from a human subject or animal welfare committee.</p></section><section id="s6"><h2 class="pmc_sec_title">Data accessibility</h2>
<p>The notebook and companion files are available at Zenodo [<a href="#B19" class="usa-link" aria-describedby="B19">19</a>]. The source notebook file (with an ipynb extension) can be uploaded and executed within the Jupyter interface of the CoLoMoTo notebook, using the Docker image colomoto/colomoto-docker:2025-03-01.</p>
<p>Supplementary material is available online [<a href="#B20" class="usa-link" aria-describedby="B20">20</a>].</p></section><section id="s7"><h2 class="pmc_sec_title">Declaration of AI use</h2>
<p>We have not used AI-assisted technologies in creating this article.</p></section><section id="s8"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>V.N.: conceptualization, formal analysis, investigation, methodology, software, validation, writing—original draft, writing—review and editing; A.N.: formal analysis, investigation, methodology, software, writing—original draft, writing—review and editing; L.C.: conceptualization, formal analysis, funding acquisition, methodology, software, supervision, validation, visualization, writing—original draft, writing—review and editing; L.P.: conceptualization, formal analysis, investigation, methodology, software, supervision, validation, writing—original draft, writing—review and editing; D.T.: conceptualization, formal analysis, investigation, methodology, software, supervision, validation, visualization, writing—original draft, writing—review and editing.</p>
<p>All authors gave final approval for publication and agreed to be held accountable for the work performed therein.</p></section><section id="s9"><h2 class="pmc_sec_title">Conflict of interest declaration</h2>
<p>We declare we have no competing interests.</p></section><section id="s10"><h2 class="pmc_sec_title">Funding</h2>
<p>This work has been supported by the French Plan Cancer-MIC ITMO, project MoDICeD (grant 20CM114-00). L.C. and L.P. were partly supported by the French ANR, project RD2BOOL (ANR-23-CE45-0014). L.P. was partly supported by France 2030 project ‘AI4scMED’ (grant ANR-22-PESN-0002).</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1">
<span class="label">1.</span><cite>
Sugita M. 1963. 
Functional analysis of chemical systems in vivo using a logical circuit equivalent. II. The idea of a molecular automation. J. Theor. Biol.
4, 179–184. ( 10.1016/0022-5193(63)90027-4)
</cite> [<a href="https://doi.org/10.1016/0022-5193(63)90027-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/5875160/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Theor.%20Biol.&amp;title=Functional%20analysis%20of%20chemical%20systems%20in%20vivo%20using%20a%20logical%20circuit%20equivalent.%20II.%20The%20idea%20of%20a%20molecular%20automation&amp;volume=4&amp;publication_year=1963&amp;pages=179-184&amp;pmid=5875160&amp;doi=10.1016/0022-5193(63)90027-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B2">
<span class="label">2.</span><cite>
Kauffman SA. 1969. 
Metabolic stability and epigenesis in randomly constructed genetic nets. J. Theor. Biol.
22, 437–467. ( 10.1016/0022-5193(69)90015-0)
</cite> [<a href="https://doi.org/10.1016/0022-5193(69)90015-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/5803332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Theor.%20Biol.&amp;title=Metabolic%20stability%20and%20epigenesis%20in%20randomly%20constructed%20genetic%20nets&amp;volume=22&amp;publication_year=1969&amp;pages=437-467&amp;pmid=5803332&amp;doi=10.1016/0022-5193(69)90015-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B3">
<span class="label">3.</span><cite>
Thomas R. 1973. 
Boolean formalization of genetic control circuits. J. Theor. Biol.
42, 563–585. ( 10.1016/0022-5193(73)90247-6)
</cite> [<a href="https://doi.org/10.1016/0022-5193(73)90247-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4588055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Theor.%20Biol.&amp;title=Boolean%20formalization%20of%20genetic%20control%20circuits&amp;volume=42&amp;publication_year=1973&amp;pages=563-585&amp;pmid=4588055&amp;doi=10.1016/0022-5193(73)90247-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B4">
<span class="label">4.</span><cite>
Albert R, Thakar J. 2014. 
Boolean modeling: a logic‐based dynamic approach for understanding signaling and regulatory networks and for making useful predictions. WIREs Syst. Biol. Med.
6, 353–369. ( 10.1002/wsbm.1273)</cite> [<a href="https://doi.org/10.1002/wsbm.1273" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25269159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=WIREs%20Syst.%20Biol.%20Med.&amp;title=Boolean%20modeling:%20a%20logic%E2%80%90based%20dynamic%20approach%20for%20understanding%20signaling%20and%20regulatory%20networks%20and%20for%20making%20useful%20predictions&amp;volume=6&amp;publication_year=2014&amp;pages=353-369&amp;pmid=25269159&amp;doi=10.1002/wsbm.1273&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B5">
<span class="label">5.</span><cite>
Abou-Jaoudé W, Traynard P, Monteiro P, Saez-Rodriguez J, Helikar T, Thieffry D, Chaouiya C. 2016. 
Logical modeling and dynamical analysis of cellular networks. Front. Genet.
7, 94. ( 10.3389/fgene.2016.00094)
</cite> [<a href="https://doi.org/10.3389/fgene.2016.00094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4885885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27303434/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Genet.&amp;title=Logical%20modeling%20and%20dynamical%20analysis%20of%20cellular%20networks&amp;volume=7&amp;publication_year=2016&amp;pages=94&amp;pmid=27303434&amp;doi=10.3389/fgene.2016.00094&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B6">
<span class="label">6.</span><cite>
Schwab JD, Kühlwein SD, Ikonomi N, Kühl M, Kestler HA. 2020. 
Concepts in Boolean network modeling: What do they all mean?
Comput. Struct. Biotechnol. J.
18, 571–582. ( 10.1016/j.csbj.2020.03.001)
</cite> [<a href="https://doi.org/10.1016/j.csbj.2020.03.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7096748/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32257043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput.%20Struct.%20Biotechnol.%20J.&amp;title=Concepts%20in%20Boolean%20network%20modeling:%20What%20do%20they%20all%20mean?&amp;volume=18&amp;publication_year=2020&amp;pages=571-582&amp;pmid=32257043&amp;doi=10.1016/j.csbj.2020.03.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B7">
<span class="label">7.</span><cite>
Thakar J. 2024. 
Pillars of biology: Boolean modeling of gene-regulatory networks. J. Theor. Biol.
578, 111682. ( 10.1016/j.jtbi.2023.111682)
</cite> [<a href="https://doi.org/10.1016/j.jtbi.2023.111682" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38008156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Theor.%20Biol.&amp;title=Pillars%20of%20biology:%20Boolean%20modeling%20of%20gene-regulatory%20networks&amp;volume=578&amp;publication_year=2024&amp;pages=111682&amp;pmid=38008156&amp;doi=10.1016/j.jtbi.2023.111682&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B8">
<span class="label">8.</span><cite>
Zerrouk N, Augé F, Niarakis A. 2024. 
Building a modular and multi-cellular virtual twin of the synovial joint in rheumatoid arthritis. NPJ Digit. Med.
7, 379. ( 10.1038/s41746-024-01396-y)
</cite> [<a href="https://doi.org/10.1038/s41746-024-01396-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11668869/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39719524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=NPJ%20Digit.%20Med.&amp;title=Building%20a%20modular%20and%20multi-cellular%20virtual%20twin%20of%20the%20synovial%20joint%20in%20rheumatoid%20arthritis&amp;volume=7&amp;publication_year=2024&amp;pages=379&amp;pmid=39719524&amp;doi=10.1038/s41746-024-01396-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B9">
<span class="label">9.</span><cite>
Hemedan AA, Niarakis A, Schneider R, Ostaszewski M. 2022. 
Boolean modelling as a logic-based dynamic approach in systems medicine. Comput. Struct. Biotechnol. J.
20, 3161–3172. ( 10.1016/j.csbj.2022.06.035)
</cite> [<a href="https://doi.org/10.1016/j.csbj.2022.06.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9234349/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35782730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput.%20Struct.%20Biotechnol.%20J.&amp;title=Boolean%20modelling%20as%20a%20logic-based%20dynamic%20approach%20in%20systems%20medicine&amp;volume=20&amp;publication_year=2022&amp;pages=3161-3172&amp;pmid=35782730&amp;doi=10.1016/j.csbj.2022.06.035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B10">
<span class="label">10.</span><cite>
Tiwari K, et al. 2021. 
Reproducibility in systems biology modelling. Mol. Syst. Biol.
17, e9982. ( 10.15252/msb.20209982)
</cite> [<a href="https://doi.org/10.15252/msb.20209982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7901289/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33620773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Syst.%20Biol.&amp;title=Reproducibility%20in%20systems%20biology%20modelling&amp;volume=17&amp;publication_year=2021&amp;pages=e9982&amp;pmid=33620773&amp;doi=10.15252/msb.20209982&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B11">
<span class="label">11.</span><cite>
Chaouiya C, et al. 2013. 
SBML qualitative models: a model representation format and infrastructure to foster interactions between qualitative modelling formalisms and tools. BMC Syst. Biol.
7, 135. ( 10.1186/1752-0509-7-135)
</cite> [<a href="https://doi.org/10.1186/1752-0509-7-135" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3892043/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24321545/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Syst.%20Biol.&amp;title=SBML%20qualitative%20models:%20a%20model%20representation%20format%20and%20infrastructure%20to%20foster%20interactions%20between%20qualitative%20modelling%20formalisms%20and%20tools&amp;volume=7&amp;publication_year=2013&amp;pages=135&amp;pmid=24321545&amp;doi=10.1186/1752-0509-7-135&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B12">
<span class="label">12.</span><cite>
Naldi A, et al. 2018. 
The CoLoMoTo interactive notebook: accessible and reproducible computational analyses for qualitative biological networks. Front. Physiol.
9, 680. ( 10.3389/fphys.2018.00680)
</cite> [<a href="https://doi.org/10.3389/fphys.2018.00680" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6018415/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29971009/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Physiol.&amp;title=The%20CoLoMoTo%20interactive%20notebook:%20accessible%20and%20reproducible%20computational%20analyses%20for%20qualitative%20biological%20networks&amp;volume=9&amp;publication_year=2018&amp;pages=680&amp;pmid=29971009&amp;doi=10.3389/fphys.2018.00680&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B13">
<span class="label">13.</span><cite>
Sizek H, Hamel A, Deritei D, Campbell S, Ravasz Regan ER. 2019. 
Boolean model of growth signaling, cell cycle and apoptosis predicts the molecular mechanism of aberrant cell cycle progression driven by hyperactive PI3K. PLoS Comput. Biol.
15, e1006402. ( 10.1371/journal.pcbi.1006402)
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1006402" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6436762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30875364/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput.%20Biol.&amp;title=Boolean%20model%20of%20growth%20signaling,%20cell%20cycle%20and%20apoptosis%20predicts%20the%20molecular%20mechanism%20of%20aberrant%20cell%20cycle%20progression%20driven%20by%20hyperactive%20PI3K&amp;volume=15&amp;publication_year=2019&amp;pages=e1006402&amp;pmid=30875364&amp;doi=10.1371/journal.pcbi.1006402&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B14">
<span class="label">14.</span><cite>
Naldi A, Hernandez C, Abou-Jaoudé W, Monteiro PT, Chaouiya C, Thieffry D. 2018. 
Logical modeling and analysis of cellular regulatory networks with GINsim 3.0. Front. Physiol.
9, 646. ( 10.3389/fphys.2018.00646)
</cite> [<a href="https://doi.org/10.3389/fphys.2018.00646" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6018412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29971008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Physiol.&amp;title=Logical%20modeling%20and%20analysis%20of%20cellular%20regulatory%20networks%20with%20GINsim%203.0&amp;volume=9&amp;publication_year=2018&amp;pages=646&amp;pmid=29971008&amp;doi=10.3389/fphys.2018.00646&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B15">
<span class="label">15.</span><cite>
Naldi A. 2018. 
BioLQM: a Java toolkit for the manipulation and conversion of logical qualitative models of biological networks. Front. Physiol.
9, 1605. ( 10.3389/fphys.2018.01605)
</cite> [<a href="https://doi.org/10.3389/fphys.2018.01605" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6254088/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30510517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Physiol.&amp;title=BioLQM:%20a%20Java%20toolkit%20for%20the%20manipulation%20and%20conversion%20of%20logical%20qualitative%20models%20of%20biological%20networks&amp;volume=9&amp;publication_year=2018&amp;pages=1605&amp;pmid=30510517&amp;doi=10.3389/fphys.2018.01605&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B16">
<span class="label">16.</span><cite>
Naldi A, Remy E, Thieffry D, Chaouiya C. 2011. 
Dynamically consistent reduction of logical regulatory graphs. Theor. Comput. Sci.
412, 2207–2218. ( 10.1016/j.tcs.2010.10.021)</cite> [<a href="https://doi.org/10.1016/j.tcs.2010.10.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theor.%20Comput.%20Sci.&amp;title=Dynamically%20consistent%20reduction%20of%20logical%20regulatory%20graphs&amp;volume=412&amp;publication_year=2011&amp;pages=2207-2218&amp;doi=10.1016/j.tcs.2010.10.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B17">
<span class="label">17.</span><cite>
Stoll G, Caron B, Viara E, Dugourd A, Zinovyev A, Naldi A, Kroemer G, Barillot E, Calzone L. 2017. 
MaBoSS 2.0: an environment for stochastic Boolean modeling. Bioinformatics
33, 2226–2228. ( 10.1093/bioinformatics/btx123)
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28881959/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=MaBoSS%202.0:%20an%20environment%20for%20stochastic%20Boolean%20modeling&amp;volume=33&amp;publication_year=2017&amp;pages=2226-2228&amp;pmid=28881959&amp;doi=10.1093/bioinformatics/btx123&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B18">
<span class="label">18.</span><cite>
Stoll G, Viara E, Barillot E, Calzone L. 2012. 
Continuous time Boolean modeling for biological signaling: application of Gillespie algorithm. BMC Syst. Biol.
6, 116. ( 10.1186/1752-0509-6-116)
</cite> [<a href="https://doi.org/10.1186/1752-0509-6-116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3517402/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22932419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Syst.%20Biol.&amp;title=Continuous%20time%20Boolean%20modeling%20for%20biological%20signaling:%20application%20of%20Gillespie%20algorithm&amp;volume=6&amp;publication_year=2012&amp;pages=116&amp;pmid=22932419&amp;doi=10.1186/1752-0509-6-116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B19">
<span class="label">19.</span><cite>
Noël V, Naldi A, Calzone L, Paulevé L, Thieffry D. 2025. 
Notebook and data for: Reproducible Boolean model analyses and simulations with the CoLoMoTo software suite: a tutorial. Zenodo. ( 10.5281/zenodo.15602428)</cite> [<a href="https://doi.org/10.5281/zenodo.15602428" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="B20">
<span class="label">20.</span><cite>
Noël V, Naldi A, Clazone L, Paulevé L, Thieffry D. 2025. 
Supplementary material from: Reproducible Boolean model analyses and simulations with the CoLoMoTo software suite: a tutorial. Figshare. ( 10.6084/m9.figshare.c.7888734)</cite> [<a href="https://doi.org/10.6084/m9.figshare.c.7888734" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The notebook and companion files are available at Zenodo [<a href="#B19" class="usa-link" aria-describedby="B19">19</a>]. The source notebook file (with an ipynb extension) can be uploaded and executed within the Jupyter interface of the CoLoMoTo notebook, using the Docker image colomoto/colomoto-docker:2025-03-01.</p>
<p>Supplementary material is available online [<a href="#B20" class="usa-link" aria-describedby="B20">20</a>].</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Interface Focus are provided here courtesy of <strong>The Royal Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1098/rsfs.2025.0002"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/rsfs.2025.0002.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371346/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371346/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371346%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371346/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371346/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371346/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371346/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371346/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371346/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="lx4yxALvh3DjdWM4xyZUzfZdEmA16JG8LQjkL6g7C8aFfJnNPsxGuA2GxJzzkl9n">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
