
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Adaptive Tolerance of Listeria monocytogenes to Chemical Oxidants: Comparative Analysis of Transcriptomic Studies - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15251A158AEECE03051A150014187BAD.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371860/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Comprehensive Reviews in Food Science and Food Safety">
<meta name="citation_title" content="Adaptive Tolerance of Listeria monocytogenes to Chemical Oxidants: Comparative Analysis of Transcriptomic Studies">
<meta name="citation_author" content="Nagendran Rajalingam">
<meta name="citation_author_institution" content="Center for Food Biotechnology and Microbiology, Ghent University Global Campus, Incheon, Republic of Korea">
<meta name="citation_author" content="Sam Van Haute">
<meta name="citation_author_institution" content="Center for Food Biotechnology and Microbiology, Ghent University Global Campus, Incheon, Republic of Korea">
<meta name="citation_author_institution" content="Department of Food Technology, Safety and Health, Faculty of Bioscience Engineering, Ghent University, Ghent, Belgium">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="24">
<meta name="citation_issue" content="5">
<meta name="citation_firstpage" content="e70260">
<meta name="citation_doi" content="10.1111/1541-4337.70260">
<meta name="citation_pmid" content="40843964">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371860/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371860/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371860/pdf/CRF3-24-e70260.pdf">
<meta name="description" content="Listeria monocytogenes is a foodborne pathogen that poses significant challenges to food safety and public health due to its ability to adapt to harsh environments, particularly those found in food processing facilities. This review explores the ...">
<meta name="og:title" content="Adaptive Tolerance of Listeria monocytogenes to Chemical Oxidants: Comparative Analysis of Transcriptomic Studies">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Listeria monocytogenes is a foodborne pathogen that poses significant challenges to food safety and public health due to its ability to adapt to harsh environments, particularly those found in food processing facilities. This review explores the ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371860/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371860">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1111/1541-4337.70260"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/CRF3-24-e70260.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371860%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371860/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371860/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371860/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1111/1541-4337.70260" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Compr Rev Food Sci Food Saf</button></div>. 2025 Aug 22;24(5):e70260. doi: <a href="https://doi.org/10.1111/1541-4337.70260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1111/1541-4337.70260</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Compr%20Rev%20Food%20Sci%20Food%20Saf%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Compr%20Rev%20Food%20Sci%20Food%20Saf%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Compr%20Rev%20Food%20Sci%20Food%20Saf%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Compr%20Rev%20Food%20Sci%20Food%20Saf%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Adaptive Tolerance of <em>Listeria monocytogenes</em> to Chemical Oxidants: Comparative Analysis of Transcriptomic Studies</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rajalingam%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Nagendran Rajalingam</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Nagendran Rajalingam</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Center for Food Biotechnology and Microbiology, Ghent University Global Campus, Incheon, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rajalingam%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nagendran Rajalingam</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Van%20Haute%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Sam Van Haute</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Sam Van Haute</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Center for Food Biotechnology and Microbiology, Ghent University Global Campus, Incheon, Republic of Korea</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Food Technology, Safety and Health, Faculty of Bioscience Engineering, Ghent University, Ghent, Belgium</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Van%20Haute%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sam Van Haute</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="crf370260-aff-0001">
<sup>
<sup>1</sup>
</sup>Center for Food Biotechnology and Microbiology, Ghent University Global Campus, Incheon, Republic of Korea</div>
<div id="crf370260-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Food Technology, Safety and Health, Faculty of Bioscience Engineering, Ghent University, Ghent, Belgium</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>: Sam Van Haute (<span>sam.vanhaute@ghent.ac.kr</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jul 23; Received 2025 Apr 21; Accepted 2025 Aug 3; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Comprehensive Reviews in Food Science and Food Safety</em> published by Wiley Periodicals LLC on behalf of Institute of Food Technologists.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371860  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40843964/" class="usa-link">40843964</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>
<em>Listeria monocytogenes</em> is a foodborne pathogen that poses significant challenges to food safety and public health due to its ability to adapt to harsh environments, particularly those found in food processing facilities. This review explores the global transcriptional responses of <em>L. monocytogenes</em> to various chemical oxidants, including hydrogen peroxide, chlorine dioxide, ozone, and plasma‐activated water. By comparing the transcriptomic data of multiple studies, we identified the differentially expressed genes associated with key cellular processes, including oxidative stress responses, cell envelope biosynthesis, metabolic adaptation, efflux mechanisms, and virulence regulation. This review demonstrates that <em>L. monocytogenes</em> employs distinct gene expression patterns to resist disinfectant stress, primarily by upregulating efflux pumps, reactive oxygen species detoxification mechanisms, and DNA repair pathways as well as modulating central metabolism. Several disinfection treatments commonly affect the key genes related to peptidoglycan biosynthesis, cell envelope, cell division, glycolysis, oxidative stress response, and chemotaxis. Although oxidizing agents induce widely conserved gene expression patterns, other treatments trigger unique responses. However, interpretation of different study findings is restricted by methodological inconsistencies, including variations in treatment conditions, media, bacterial states, and transcriptomic techniques. These variations and nonuniform fold change thresholds for differentially expressed genes complicate the comparison of different studies. Therefore, standardized frameworks are necessary to elucidate the adaptive responses of <em>L. monocytogenes</em> and refine its disinfection methods in food processing.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> chemical oxidants, ClO<sub>2</sub>
, H<sub>2</sub>O<sub>2</sub>
, <em>Listeria monocytogenes</em>, O<sub>3</sub>
, plasma‐activated water, transcriptomic analysis</p></section></section><section id="crf370260-sec-0010"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>
<em>Listeria monocytogenes</em>, a resilient foodborne pathogen commonly found in soil, water, vegetation, livestock, and excrement, can contaminate food across the supply chain (Rodriguez et al. <a href="#crf370260-bib-0166" class="usa-link" aria-describedby="crf370260-bib-0166">2021</a>). Its ability to thrive under extreme acidic, alkaline, cold, and dry conditions makes its control challenging. This pathogen transitions from a saprophytic lifestyle at 20–25°C to a pathogenic state at 37°C by repressing flagellar gene expression (Pizarro‐Cerdá and Cossart <a href="#crf370260-bib-0147" class="usa-link" aria-describedby="crf370260-bib-0147">2019</a>). It also forms biofilms and adheres to diverse surfaces, which contributes to its persistence in food and clinical settings (Bechtel and Gibbons <a href="#crf370260-bib-0014" class="usa-link" aria-describedby="crf370260-bib-0014">2021</a>). Surveillance studies have reported its presence in dairy products, seafood, meat, and produce (Gérard et al. <a href="#crf370260-bib-0066" class="usa-link" aria-describedby="crf370260-bib-0066">2020</a>; Y. Liu et al. <a href="#crf370260-bib-0116" class="usa-link" aria-describedby="crf370260-bib-0116">2020</a>; Zakrzewski et al. <a href="#crf370260-bib-0227" class="usa-link" aria-describedby="crf370260-bib-0227">2024</a>).</p>
<p>Food industry uses chemical antimicrobials, such as chlorine, ozone (O<sub>3</sub>), and peracetic acid (Simpson and Mitch <a href="#crf370260-bib-0192" class="usa-link" aria-describedby="crf370260-bib-0192">2022</a>), typically in combination with alkaline or acidic cleaning processes (Simpson and Mitch <a href="#crf370260-bib-0192" class="usa-link" aria-describedby="crf370260-bib-0192">2022</a>) to prevent contamination. Additionally, physical methods, such as ultraviolet and high‐pressure processing methods, are used to decrease the microbial load (Sharma et al. <a href="#crf370260-bib-0185" class="usa-link" aria-describedby="crf370260-bib-0185">2022</a>). However, bacteria develop tolerance to these measures. Antibiotic resistance remains a major public health concern, as multidrug‐resistant “superbugs” are emerging faster than new antibiotics can be developed (Fatima et al. <a href="#crf370260-bib-0056" class="usa-link" aria-describedby="crf370260-bib-0056">2023</a>; Salam et al. <a href="#crf370260-bib-0176" class="usa-link" aria-describedby="crf370260-bib-0176">2023</a>). Acquired tolerance is particularly concerning, as it enables previously susceptible bacteria to resist treatment via horizontal gene transfer and chromosomal gene mutations. These changes often reduce the antibiotic uptake by altering or downregulating the porin proteins (Mancuso et al. <a href="#crf370260-bib-0122" class="usa-link" aria-describedby="crf370260-bib-0122">2021</a>).</p>
<p>Chemical oxidants differ from antibiotics in that they inflict widespread damage rather than targeting specific sites (Griffiths <a href="#crf370260-bib-0070" class="usa-link" aria-describedby="crf370260-bib-0070">2005</a>). Resistance to chemical oxidants is better understood as increased tolerance. Exposure conditions are crucial for examining tolerance. <em>L. monocytogenes</em> count is reduced by 1000‐fold (3 log reduction) by 0.5 mg/L hypochlorous acid in 10 s in the absence of chlorine interference (Gongora et al. <a href="#crf370260-bib-0069" class="usa-link" aria-describedby="crf370260-bib-0069">2024</a>). However, for vegetable wash water with 800 mg/L chemical oxygen demand (indicating oxidizable matter), a 2‐min contact with 50 mg/L hypochlorous acid is essential to achieve the same <em>L. monocytogenes</em> reduction (Van Haute et al. <a href="#crf370260-bib-0206" class="usa-link" aria-describedby="crf370260-bib-0206">2013</a>). Comparatively, organic materials produce less potent chloramines that only cause sublethal stress, potentially inducing adaptive responses (Pfaller et al. <a href="#crf370260-bib-0241" class="usa-link" aria-describedby="crf370260-bib-0241">2021</a>).</p>
<p>Sublethal exposure to oxidants affects antimicrobial resistance. In <em>Pseudomonas</em> spp., such exposure is associated with increased antibiotic resistance (Tong et al. <a href="#crf370260-bib-0200" class="usa-link" aria-describedby="crf370260-bib-0200">2021</a>). Chlorine dioxide (ClO<sub>2</sub>)‐treated wastewater shows a high prevalence of bacteria with tolerance genes (S.‐S. Liu et al. <a href="#crf370260-bib-0115" class="usa-link" aria-describedby="crf370260-bib-0115">2018</a>), possibly due to enhanced horizontal gene transfer via natural transformation (Jin et al. <a href="#crf370260-bib-0087" class="usa-link" aria-describedby="crf370260-bib-0087">2020</a>). Chlorination increases the membrane permeability and reactive oxygen species (ROS) levels and activates the salt overly sensitive (SOS) responses, facilitating the mutation and spread of tolerance genes (W. Zhao et al. <a href="#crf370260-bib-0232" class="usa-link" aria-describedby="crf370260-bib-0232">2025</a>). Despite the widespread use of oxidants, <em>L. monocytogenes</em> frequently survives in food processing environments. Its resilience is due to its ability to colonize microniches and exhibit temporary adaptive tolerance via biofilm formation, decreased metabolism, and increased efflux pump activity (Tong et al. <a href="#crf370260-bib-0200" class="usa-link" aria-describedby="crf370260-bib-0200">2021</a>; van Dijk and Verbrugh <a href="#crf370260-bib-0205" class="usa-link" aria-describedby="crf370260-bib-0205">2022</a>). However, the specific molecular mechanisms underlying adaptive tolerance remain unclear. Recent transcriptomic studies have revealed the <em>L. monocytogenes</em> responses to oxidative stress at the gene level (Fernández‐Gómez et al. <a href="#crf370260-bib-0058" class="usa-link" aria-describedby="crf370260-bib-0058">2023</a>; Kastbjerg et al. <a href="#crf370260-bib-0098" class="usa-link" aria-describedby="crf370260-bib-0098">2010</a>; Siderakou et al. <a href="#crf370260-bib-0190" class="usa-link" aria-describedby="crf370260-bib-0190">2022</a>).</p>
<p>Proper <em>L. monocytogenes</em> contamination control is crucial to prevent foodborne outbreaks and serious health consequences in the food industry. In healthcare, <em>L. monocytogenes</em> infects individuals with weak immune systems, including expectant mothers, necessitating the development of effective preventive strategies. This review discusses previous reports on the global transcriptional responses of <em>L. monocytogenes</em> to chemical oxidants to understand the molecular mechanisms underlying its adaptation and survival. Additionally, this review highlights the <em>L. monocytogenes</em> survival strategies under chemical oxidant stress to facilitate the development of suitable strategies to prevent <em>Listeria</em> persistence in various industries, including food processing.</p></section><section id="crf370260-sec-0020"><h2 class="pmc_sec_title">2. Selection of Disinfection Treatments for Review</h2>
<p>We conducted a thorough, albeit nonsystematic, literature search to identify the studies on <em>L. monocytogenes</em> transcriptomic responses to chemical oxidants. Our search spanned Google Scholar, PubMed, Scopus, and Web of Science using combinations of keywords, such as “<em>Listeria monocytogenes</em>,” “transcriptome,” “RNA‐seq,” “disinfectants,” “chemical oxidants,” “oxidative stress,” “hydrogen peroxide,” “ozone,” and “chlorine.” We carefully reviewed the titles, abstracts, and full‐texts of the articles to identify relevant studies on bacterial responses to disinfectants. We extracted gene expression data as reported by the original authors without reprocessing the raw data from the main text, supplementary materials, figures, and tables and consulted raw datasets, when available, to support our findings. Although no formal systematic review protocol was followed, our approach was designed to be as thorough and focused as possible within the scope of this review.</p>
<p>We selected disinfection treatments based on the availability of published transcriptome data for <em>L. monocytogenes</em>. This review focuses on the global transcriptional responses of <em>L. monocytogenes</em> to various disinfectants, particularly oxidizing agents such as hydrogen peroxide (H<sub>2</sub>O<sub>2</sub>), ClO<sub>2</sub>, ozone (O<sub>3</sub>), and plasma‐activated water (PAW). We also discuss additional treatments, including benzethonium chloride (BZT), lactic acid (LA), pulsed magnetic fields (PMF), and high‐pressure processing (HPP). While our primary focus is on oxidizing agents, we also provide a comparative analysis of nonoxidizing chemical disinfectants (e.g., BZT and LA) and physical treatments (e.g., PMF and HPP). This comparative perspective is crucial for distinguishing the transcriptional responses specific to oxidative stress from general stress responses triggered by various treatment types. Details regarding the strains, treatment conditions, and transcriptome analysis methods used in the reviewed studies are summarized in Table <a href="#crf370260-tbl-0001" class="usa-link">1</a>. Further discussion of the transcriptomic effects of BZT, LA, PMF, and HPP is included in the <a href="#crf370260-supinfo-0001" class="usa-link">Supporting Information Data</a>. By establishing a comparative baseline, this review aims to clarify the unique adaptive mechanisms that <em>L. monocytogenes</em> employs to survive oxidative stress, thereby enhancing our understanding of how this pathogen responds to different disinfection strategies.</p>
<section class="tw xbox font-sm" id="crf370260-tbl-0001"><h3 class="obj_head">TABLE 1.</h3>
<div class="caption p"><p>Disinfection treatments used to compare the transcriptomic responses of <em>Listeria monocytogenes</em> in this review.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Disinfection treatments</th>
<th align="center" rowspan="1" colspan="1">Treatment conditions</th>
<th align="center" rowspan="1" colspan="1">
<em>L. mono</em>. strains</th>
<th align="center" rowspan="1" colspan="1">Bacterial reduction</th>
<th align="center" rowspan="1" colspan="1">Transcriptome method</th>
<th align="center" rowspan="1" colspan="1">Total DEGs [fold change (FC)]</th>
<th align="center" rowspan="1" colspan="1">Reference</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="2" align="left" colspan="1">Benzethonium chloride (BZT)</td>
<td rowspan="2" align="center" colspan="1">
<em>L. monocytogenes</em> was grown in the tryptic soy broth (TSB) in the presence of 4 ppm BZT at 14°C until the early stationary phase</td>
<td rowspan="3" align="center" colspan="1">6179, serotype 1/2a</td>
<td align="center" rowspan="1" colspan="1">0.16 Abs 610 nm (0.49 to 0.33)</td>
<td align="center" rowspan="1" colspan="1">Microarray</td>
<td align="center" rowspan="1" colspan="1">92 (FC ≥7.5)</td>
<td align="center" rowspan="1" colspan="1">Fox et al. (<a href="#crf370260-bib-0060" class="usa-link" aria-describedby="crf370260-bib-0060">2011</a>)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Not recorded</td>
<td align="center" rowspan="1" colspan="1">RNA‐seq</td>
<td align="center" rowspan="1" colspan="1">600 (FC ≥ 4)</td>
<td align="center" rowspan="1" colspan="1">Casey et al. (<a href="#crf370260-bib-0035" class="usa-link" aria-describedby="crf370260-bib-0035">2014</a>)</td>
</tr>
<tr>
<td rowspan="2" align="left" colspan="1">Lactic acid (LA)</td>
<td rowspan="2" align="center" colspan="1">
<em>L. monocytogenes</em> strains were exposed to 1% lactic acid (pH 3.4) in TSB at 20°C for 30 min</td>
<td align="center" rowspan="1" colspan="1">Not recorded</td>
<td align="center" rowspan="1" colspan="1">RNA‐seq</td>
<td align="center" rowspan="1" colspan="1">2280 (log<sub>2</sub> FC &gt; 1)</td>
<td rowspan="2" align="center" colspan="1">Cortes et al. (<a href="#crf370260-bib-0242" class="usa-link" aria-describedby="crf370260-bib-0242">2020</a>)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">R479a, serotype 1/2a</td>
<td align="center" rowspan="1" colspan="1">Not recorded</td>
<td align="center" rowspan="1" colspan="1">RNA‐seq</td>
<td align="center" rowspan="1" colspan="1">2151 (log<sub>2</sub> FC &gt;1)</td>
</tr>
<tr>
<td rowspan="2" align="left" colspan="1">Hydrogen peroxide (H<sub>2</sub>O<sub>2</sub>)</td>
<td rowspan="2" align="center" colspan="1">
<em>L. monocytogenes</em> strains were exposed to 0.01% hydrogen peroxide in TSB at 20°C for 30 min</td>
<td align="center" rowspan="1" colspan="1">6179, serotype 1/2a</td>
<td align="center" rowspan="1" colspan="1">Not recorded</td>
<td align="center" rowspan="1" colspan="1">RNA‐seq</td>
<td align="center" rowspan="1" colspan="1">2280 (log<sub>2</sub> FC &gt; 1)</td>
<td rowspan="2" align="center" colspan="1">Cortes et al. (<a href="#crf370260-bib-0242" class="usa-link" aria-describedby="crf370260-bib-0242">2020</a>)</td>
</tr>
<tr>
<td rowspan="2" align="center" colspan="1">R479a, serotype 1/2a</td>
<td align="center" rowspan="1" colspan="1">Not recorded</td>
<td align="center" rowspan="1" colspan="1">RNA‐seq</td>
<td align="center" rowspan="1" colspan="1">2151 (log<sub>2</sub> FC &gt;1)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Chlorine dioxide (ClO<sub>2</sub>)</td>
<td align="center" rowspan="1" colspan="1">
<em>L. monocytogenes</em> in the brain heart infusion (BHI) broth was exposed to 300 mg/L aqueous ClO<sub>2</sub> at 37°C and 230 rpm for 15 min</td>
<td align="center" rowspan="1" colspan="1">0.33 ± 0.37 log CFU/mL reduction</td>
<td align="center" rowspan="1" colspan="1">Microarray</td>
<td align="center" rowspan="1" colspan="1">340 (FC ≥ 1.5)</td>
<td align="center" rowspan="1" colspan="1">Pleitner et al. (<a href="#crf370260-bib-0148" class="usa-link" aria-describedby="crf370260-bib-0148">2014</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Ozone (O<sub>3</sub>)</td>
<td align="center" rowspan="1" colspan="1">
<em>L. monocytogenes</em> (250 µL; &gt; 6.5 log CFU/g) was inoculated on the surface of tomato fruits and treated with O<sub>3</sub> at 2 µg/g tomato fruits (moderate dose) at 23°C for 1 h</td>
<td align="center" rowspan="1" colspan="1">ATCC 19115, serotype 4b</td>
<td align="center" rowspan="1" colspan="1">1.075 ± 0.3 log reduction</td>
<td align="center" rowspan="1" colspan="1">RNA‐seq</td>
<td align="center" rowspan="1" colspan="1">283 (PCC ≥ 0.9999)<a href="#crf370260-tbl1-note-0001" class="usa-link">
<sup>a</sup>
</a>
</td>
<td align="center" rowspan="1" colspan="1">Shu et al. (<a href="#crf370260-bib-0188" class="usa-link" aria-describedby="crf370260-bib-0188">2021</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Plasma‐activated water (PAW)</td>
<td align="center" rowspan="1" colspan="1">Stationary‐phase <em>L. monocytogenes</em> cells were suspended in Ringer solution and treated for 5 min at room temperature with PAW generated using a plasma power of 36 W and an activation time of 30 min, and stored for 24 h</td>
<td align="center" rowspan="1" colspan="1">ULE1265, serotype 1/2c</td>
<td align="center" rowspan="1" colspan="1">2.2 ± 0.1 log<sub>10</sub> CFU/mL reduction</td>
<td align="center" rowspan="1" colspan="1">RNA‐seq</td>
<td align="center" rowspan="1" colspan="1">399 (log<sub>2</sub> FC &gt; 1)</td>
<td align="center" rowspan="1" colspan="1">Fernández‐Gómez et al. (<a href="#crf370260-bib-0058" class="usa-link" aria-describedby="crf370260-bib-0058">2023</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Pulsed magnetic field (PMF)</td>
<td align="center" rowspan="1" colspan="1">
<em>L. monocytogenes</em> in phosphate‐buffered saline (PBS) was exposed to PMF at 8 T intensity with 20 pulses at room temperature</td>
<td align="center" rowspan="1" colspan="1">ATCC 19111, serotype 1/2a</td>
<td align="center" rowspan="1" colspan="1">9.6% (90.4% reduction)</td>
<td align="center" rowspan="1" colspan="1">RNA‐seq</td>
<td align="center" rowspan="1" colspan="1">588 (FC &gt;2)</td>
<td align="center" rowspan="1" colspan="1">Qian et al. (<a href="#crf370260-bib-0156" class="usa-link" aria-describedby="crf370260-bib-0156">2020</a>)</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">High hydrostatic pressure processing (HPP)</td>
<td align="center" rowspan="1" colspan="1">
<em>L. monocytogenes</em> in the tryptone soy yeast extract (TSYE) broth was subjected to HPP at 400 and 600 MPa and 15°C for 5 min</td>
<td align="center" rowspan="1" colspan="1">S2542, serotype 4b</td>
<td align="center" rowspan="1" colspan="1">&gt; 3 log CFU/mL reduction</td>
<td align="center" rowspan="1" colspan="1">Microarray</td>
<td align="center" rowspan="1" colspan="1">NA<a href="#crf370260-tbl1-note-0002" class="usa-link">
<sup>b</sup>
</a>
</td>
<td align="center" rowspan="1" colspan="1">Bowman et al. (<a href="#crf370260-bib-0261" class="usa-link" aria-describedby="crf370260-bib-0261">2010</a>)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/crf370260-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="crf370260-tbl1-note-0001">
<sup>
<sup>a</sup>
</sup><p class="display-inline">PCC, Pearson's correlation coefficient.</p>
</div>
<div class="fn" id="crf370260-tbl1-note-0002">
<sup>
<sup>b</sup>
</sup><p class="display-inline">NA, not available.</p>
</div>
</div></section><section id="crf370260-sec-0030"><h3 class="pmc_sec_title">2.1. Differentially Expressed Gene Identification Under the Selected Disinfection Treatments</h3>
<p>Differentially expressed genes (DEGs) were meticulously identified and extracted from the article text, tables, and supplementary figures of the respective published research articles on BZT (Fox et al. <a href="#crf370260-bib-0060" class="usa-link" aria-describedby="crf370260-bib-0060">2011</a>; Casey et al. <a href="#crf370260-bib-0035" class="usa-link" aria-describedby="crf370260-bib-0035">2014</a>), LA, H<sub>2</sub>O<sub>2</sub> (Cortes et al. <a href="#crf370260-bib-0242" class="usa-link" aria-describedby="crf370260-bib-0242">2020</a>), ClO<sub>2</sub> (Pleitner et al. <a href="#crf370260-bib-0148" class="usa-link" aria-describedby="crf370260-bib-0148">2014</a>), O<sub>3</sub> (Shu et al. <a href="#crf370260-bib-0188" class="usa-link" aria-describedby="crf370260-bib-0188">2021</a>), PAW (Fernández‐Gómez et al. <a href="#crf370260-bib-0058" class="usa-link" aria-describedby="crf370260-bib-0058">2023</a>), PMF (Qian et al. <a href="#crf370260-bib-0156" class="usa-link" aria-describedby="crf370260-bib-0156">2020</a>), and HPP (Bowman et al. <a href="#crf370260-bib-0022" class="usa-link" aria-describedby="crf370260-bib-0022">2008</a>). Each gene upregulated or downregulated by the selected disinfection treatments was logged into a Microsoft Excel file (Microsoft Corporation, Redmond, WA, USA) and organized accordingly. Transcriptome data of the <em>L. monocytogenes</em> American Type Culture Collection (ATCC) 19115 strain under O<sub>3</sub> treatment were aligned and annotated against the reference genome of <em>L. monocytogenes</em> serovar 1/2a (strain ATCC BAA‐679/EGD‐e) using the Geneious Prime 2024.0.5 software (Kearse et al. <a href="#crf370260-bib-0099" class="usa-link" aria-describedby="crf370260-bib-0099">2012</a>). Functional categories of each gene were determined using the UniProt Gene Ontology annotation tool with the prokaryotic slimming set (<a href="https://www.uniprot.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.uniprot.org/</a>), with <em>L. monocytogenes</em> strain ATCC BAA‐679/EGD‐e as the reference organism (Tables <a href="#crf370260-supinfo-0001" class="usa-link">S1–S9</a>; Figure <a href="#crf370260-fig-0001" class="usa-link">1</a>). An online tool for Venn diagrams (<a href="https://www.interactivenn.net" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.interactivenn.net</a>) (Heberle et al. <a href="#crf370260-bib-0077" class="usa-link" aria-describedby="crf370260-bib-0077">2015</a>) was used to merge the gene lists for individual disinfection treatments and identify the genes affected by multiple disinfection treatments (Figure <a href="#crf370260-fig-0002" class="usa-link">2</a>). Furthermore, a heat map was generated using Microsoft Excel to illustrate the effects of various disinfection methods on specific genes (Figure <a href="#crf370260-supinfo-0001" class="usa-link">S1</a>).</p>
<figure class="fig xbox font-sm" id="crf370260-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ceb8/12371860/d5247f7750ee/CRF3-24-e70260-g002.jpg" loading="lazy" id="jats-graphic-1" height="501" width="546" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/crf370260-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Differentially expressed gene (DEG) distribution among different role categories. Total number of upregulated (↑) and downregulated (↓) genes in each role category is indicated. Each cell represents the number of <em>Listeria monocytogenes</em> genes differentially expressed in response to various disinfection treatments: benzethonium chloride (BZT), chlorine dioxide (ClO<sub>2</sub>), hydrogen peroxide (H<sub>2</sub>O<sub>2</sub>), hydrostatic pressure processing (HPP), lactic acid (LA), ozone (O<sub>3</sub>), plasma‐activated water (PAW), and pulsed magnetic field (PMF).</p></figcaption></figure><figure class="fig xbox font-sm" id="crf370260-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ceb8/12371860/482a38df73cc/CRF3-24-e70260-g001.jpg" loading="lazy" id="jats-graphic-3" height="318" width="700" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/crf370260-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparative transcriptome profiles of <em>L. monocytogenes</em> under various disinfection treatments. A Venn diagram illustrates the total number of DEGs in <em>L. monocytogenes</em> under various disinfection treatments. (A) Oxidizing agents (hydrogen peroxide [H<sub>2</sub>O<sub>2</sub>], chlorine dioxide [ClO<sub>2</sub>], ozone [O<sub>3</sub>], and plasma‐activated water [PAW]). (B) Other disinfectants (benzethonium chloride [BZT], lactic acid [LA], pulsed magnetic field [PMF], and hydrostatic pressure processing [HPP]).</p></figcaption></figure></section></section><section id="crf370260-sec-0040"><h2 class="pmc_sec_title">3. Transcriptional Responses of <em>L. monocytogenes</em> to Various Disinfection Treatments</h2>
<p>As mentioned above, the common bactericidal mechanism of most disinfection treatments involves the destruction of the cell membrane, resulting in the leakage of intracellular substances and DNA damage in <em>L. monocytogenes</em>. Additionally, disinfectants disrupt the intracellular ROS balance in <em>L. monocytogenes</em> by inhibiting the antioxidant enzyme activity, leading to bacterial inactivation (H. Li et al. <a href="#crf370260-bib-0112" class="usa-link" aria-describedby="crf370260-bib-0112">2021</a>). However, their effects on the genes and pathways in <em>L. monocytogenes</em> vary substantially, depending on the specific disinfectant used and pathogen responses. Repeated disinfectant exposure in the food industry induces selective pressure, enhancing the tolerance to these compounds, triggering the development of antibiotic resistance, and increasing the fitness of <em>L. monocytogenes</em> (Bland et al. <a href="#crf370260-bib-0018" class="usa-link" aria-describedby="crf370260-bib-0018">2022</a>). Genetic mechanisms underlying the adaptive tolerance to disinfectants involve changes in the expression levels of various regulatory genes and pathways necessary for stress adaptation and tolerance (Schulz et al. <a href="#crf370260-bib-0243" class="usa-link" aria-describedby="crf370260-bib-0243">2023</a>). Therefore, a deeper understanding of the transcriptional responses of <em>L. monocytogenes</em> to various disinfectants is essential for the development of effective disinfection approaches.</p>
<section id="crf370260-sec-0050"><h3 class="pmc_sec_title">3.1. Chlorine Dioxide (ClO<sub>2</sub>)</h3>
<p>ClO<sub>2</sub> is a powerful and rapid oxidizing agent. It is used in both gaseous and aqueous food systems to effectively reduce the counts of foodborne pathogens, including <em>L. monocytogenes</em> (Contreras‐Soto et al. <a href="#crf370260-bib-0045" class="usa-link" aria-describedby="crf370260-bib-0045">2021</a>; Pleitner et al. <a href="#crf370260-bib-0148" class="usa-link" aria-describedby="crf370260-bib-0148">2014</a>). This low‐molecular‐weight oxidative disinfectant targets almost every biomolecule, including nucleic acids, amino acids, peptides, proteins, and lipids (Fukuzaki <a href="#crf370260-bib-0064" class="usa-link" aria-describedby="crf370260-bib-0064">2006</a>), and multiple cell wall regions, causing cytoplasmic membrane disruption and peroxidation, enzyme inhibition (Maillard and Pascoe <a href="#crf370260-bib-0120" class="usa-link" aria-describedby="crf370260-bib-0120">2024</a>), bacterial genome destabilization, reduced energy production, protein synthesis disruption, and eventual cell death (Imlay <a href="#crf370260-bib-0080" class="usa-link" aria-describedby="crf370260-bib-0080">2003</a>; Kohanski et al. <a href="#crf370260-bib-0104" class="usa-link" aria-describedby="crf370260-bib-0104">2010</a>; Sazykin and Sazykina <a href="#crf370260-bib-0178" class="usa-link" aria-describedby="crf370260-bib-0178">2023</a>).</p>
<p>Pleitner et al. (<a href="#crf370260-bib-0148" class="usa-link" aria-describedby="crf370260-bib-0148">2014</a>) conducted whole‐genome microarray analysis to characterize the transcriptional changes in <em>L. monocytogenes</em> 10403S when vegetative cells in the brain heart infusion broth were exposed to 300 mg/L ClO<sub>2</sub> for 15 min. Along with a 0.33 ± 0.37 log CFU/mL reduction in <em>L. monocytogenes</em> count, ClO<sub>2</sub> exposure resulted in the differential expression of 340 genes, of which 113 and 16 genes belonged to <em>sigB</em> and <em>CtsR</em> regulons, respectively, implicating both the general and protein‐specific stress response pathways. In total, levels of 223 genes related to carbohydrate and glycerol metabolism, protein quality control, hemostasis, phosphotransferase system (PTS), quorum sensing (QS), and several stress response genes (mainly oxidative stress response genes) were upregulated. In contrast, chemotaxis and flagellar gene levels were downregulated (Figure <a href="#crf370260-fig-0001" class="usa-link">1</a>). Oxidative stress responses of <em>L. monocytogenes</em> to ClO<sub>2</sub> involve the increased expression levels of several redox‐associated genes, including superoxide dismutase A (<em>sodA</em>), organic hydroperoxide resistance (<em>ohr</em>)‐<em>A</em>, <em>lmo2230</em>, and global stress regulator (<em>spxA</em> and <em>spxH</em>) genes. These genes are crucial to maintain the redox balance and manage oxidative damage. Additionally, upregulation of the DNA repair (e.g., <em>fri</em> and <em>deoC</em>), protein quality control (<em>clpB</em>, <em>clpC</em>, <em>clpP</em>, and <em>clpX</em>), and heat shock repressor (<em>ctsR</em>) gene levels indicates that the bacteria actively repair the damaged proteins to reduce the cellular stress. Taken together, these findings suggest that <em>L. monocytogenes</em> combats ClO<sub>2</sub>‐induced oxidative stress by enhancing the systems involved in redox homeostasis, DNA repair, and protein maintenance (Pleitner et al. <a href="#crf370260-bib-0148" class="usa-link" aria-describedby="crf370260-bib-0148">2014</a>).</p></section><section id="crf370260-sec-0060"><h3 class="pmc_sec_title">3.2. Hydrogen Peroxide</h3>
<p>Oxidizing agents, reactants involved in redox reactions, gain electrons and oxidize other substances. H<sub>2</sub>O<sub>2</sub> is an oxidizing agent generating hydroxyl radicals (•OH) through various processes, such as the Fenton reaction that involves H<sub>2</sub>O<sub>2</sub> decomposition into oxygen and water, accompanied by considerable heat release (Andrés et al. <a href="#crf370260-bib-0008" class="usa-link" aria-describedby="crf370260-bib-0008">2022</a>; Manso et al. <a href="#crf370260-bib-0123" class="usa-link" aria-describedby="crf370260-bib-0123">2020</a>). H<sub>2</sub>O<sub>2</sub> is generally recognized as safe (21 CFR 184.1366) for use as a food disinfectant, provided that any residual H<sub>2</sub>O<sub>2</sub> is eliminated using suitable physical and chemical methods during food processing (Robinson and D'Amico <a href="#crf370260-bib-0164" class="usa-link" aria-describedby="crf370260-bib-0164">2021</a>). H<sub>2</sub>O<sub>2</sub> inactivates bacteria by generating radicals, causing oxidative damage to their cells, targeting the peptidoglycan wall and cell membrane, disrupting the nucleic acid structure, denaturing the proteins and lipids, and ultimately leading to cell death (Harter et al. <a href="#crf370260-bib-0074" class="usa-link" aria-describedby="crf370260-bib-0074">2017</a>; Manso et al. <a href="#crf370260-bib-0123" class="usa-link" aria-describedby="crf370260-bib-0123">2020</a>).</p>
<p>Along with evaluating the responses to LA, Cortes et al (<a href="#crf370260-bib-0242" class="usa-link" aria-describedby="crf370260-bib-0242">2020</a>) recorded the transcriptional responses of pathogens to oxidative stress by exposing <em>L. monocytogenes</em> strains 6179 and R479a to 0.01% H<sub>2</sub>O<sub>2</sub> in tryptic soy broth (TSB) at 20°C for 30 min. Only a few genes (154 and 73 DEGs in 6179 and R479a, respectively) showed differential expression under H<sub>2</sub>O<sub>2</sub> conditions compared to those showing differential expression under LA conditions. Levels of several genes related to the oxidative stress response and DNA repair were upregulated, whereas those of genes involved in cell wall and glycolytic processes were downregulated (Figure <a href="#crf370260-fig-0001" class="usa-link">1</a>). At the molecular level, <em>L. monocytogenes</em> relies on conserved antioxidant defense mechanisms to mitigate H<sub>2</sub>O<sub>2</sub>‐induced oxidative damage. These include <em>ohrA</em>, <em>ohrR</em>, and <em>lmo2230</em>, which contribute to the detoxification of peroxides. Gene <em>sodA</em> converts superoxide radicals into H<sub>2</sub>O<sub>2,</sub> which is neutralized by catalases and peroxidases (Winterbourn <a href="#crf370260-bib-0220" class="usa-link" aria-describedby="crf370260-bib-0220">2020</a>). Levels of several DNA repair genes (<em>dinB</em>, <em>uvrA</em>, <em>uvrB</em>, <em>recA</em>, <em>exoA</em>, <em>addA</em>, and <em>addB</em>) are upregulated in response to oxidative DNA damage. This repair activity is partially coordinated by the LexA‐regulated SOS response, in which DNA damage induces RecA‐mediated LexA autoproteolysis, thereby increasing SOS gene repression to promote DNA repair and cell survival (Cory et al. <a href="#crf370260-bib-0046" class="usa-link" aria-describedby="crf370260-bib-0046">2024</a>). A low H<sub>2</sub>O<sub>2</sub> concentration and interaction with TSB components may possibly suppress gene expression changes, as noted by Cortes et al. (<a href="#crf370260-bib-0242" class="usa-link" aria-describedby="crf370260-bib-0242">2020</a>). Nevertheless, upregulation of oxidative stress gene levels suggests a conserved bacterial strategy for redox detoxification and DNA damage repair.</p></section><section id="crf370260-sec-0070"><h3 class="pmc_sec_title">3.3. Ozone (O<sub>3</sub>)</h3>
<p>O<sub>3</sub>, an allotrope of oxygen, appears colorless to pale blue and is soluble in water (Chuwa et al. <a href="#crf370260-bib-0042" class="usa-link" aria-describedby="crf370260-bib-0042">2020</a>). It is a potent oxidizing agent with a chlorine‐like odor, extensively used as a disinfectant in the food industry (Botondi et al. <a href="#crf370260-bib-0021" class="usa-link" aria-describedby="crf370260-bib-0021">2021</a>). Molecular O<sub>3</sub> and the free radicals (O<sup>•─</sup>, O<sub>2</sub>
<sup>•─</sup>, O<sub>3</sub>
<sup>•─</sup>, <sup>•</sup>OH, <sup>•</sup>HO<sub>2</sub>, HO<sub>3</sub>
<sup>•</sup>, and H<sub>2</sub>O<sub>2</sub>) generated by its decomposition inactivate bacteria through complex processes targeting multiple cellular components (Epelle et al. <a href="#crf370260-bib-0053" class="usa-link" aria-describedby="crf370260-bib-0053">2023</a>). These include cell membrane lysis, resulting from the presence of unsaturated lipids and peptidoglycans in the cell envelope, and leakage of cellular components, such as nucleic acids, proteins, and enzymes, from the cytoplasm (Khadre et al. <a href="#crf370260-bib-0101" class="usa-link" aria-describedby="crf370260-bib-0101">2001</a>). Gaseous and aqueous forms of O<sub>3</sub> effectively inactivate <em>L. monocytogenes</em> in various foods, including lettuce (Ölmez and Akbas <a href="#crf370260-bib-0142" class="usa-link" aria-describedby="crf370260-bib-0142">2009</a>), spinach (Wani et al. <a href="#crf370260-bib-0215" class="usa-link" aria-describedby="crf370260-bib-0215">2015</a>), apple juice (Song et al. <a href="#crf370260-bib-0196" class="usa-link" aria-describedby="crf370260-bib-0196">2015</a>), apples, strawberries, and cantaloupes (Rodgers et al. <a href="#crf370260-bib-0165" class="usa-link" aria-describedby="crf370260-bib-0165">2004</a>).</p>
<p>Shu et al. (<a href="#crf370260-bib-0188" class="usa-link" aria-describedby="crf370260-bib-0188">2021</a>) evaluated the antimicrobial efficacy of O<sub>3</sub> at different doses (1, 2, and 3 µg O<sub>3</sub>/g of fruit) and exposure periods (1, 2, and 3 h) against <em>L. monocytogenes</em> on the surface of tomato fruits. They found that a moderate O<sub>3</sub> dose (2 µg O<sub>3</sub>/g of fruit) effectively reduced the <em>L. monocytogenes</em> count (1.1 ± 0.3 log reduction) within a short exposure time of 1 h. Transcriptional profile of <em>L. monocytogenes</em> at this O<sub>3</sub> dose revealed the differential expression of 283 genes, including 121 upregulated and 161 downregulated genes. Levels of genes related to general and oxidative stress responses, cell division, virulence, and chemotaxis were upregulated. In contrast, levels of several genes with unknown functions and those related to metabolism and transport systems were downregulated (Figure <a href="#crf370260-fig-0001" class="usa-link">1</a>). O<sub>3</sub> exposure triggered complex transcriptional responses, including the upregulation and downregulation of stress‐associated genes, in <em>L. monocytogenes</em>. Levels of the oxidative stress gene <em>sodA</em> were upregulated, whereas those of <em>trxB</em> (thioredoxin reductase) and <em>prli42</em> (membrane‐associated stress protein) were downregulated. This indicates the selective modulation of pathways related to redox and membrane stress. Furthermore, the upregulation of cold shock protein (<em>Csp</em>)‐<em>D</em> and <em>osmC</em> levels indicated a response to cold shock and osmotic stress. Increased <em>cheR, flgE</em>, and <em>mogR</em> levels indicated enhanced chemotaxis and flagellar regulation, supporting adaptive motility under oxidative stress conditions. These gene activation and repression patterns suggest that <em>L. monocytogenes</em> prioritizes responses that enhance stress tolerance during transport and metabolic processes in response to moderate O<sub>3</sub> doses (Shu et al. <a href="#crf370260-bib-0188" class="usa-link" aria-describedby="crf370260-bib-0188">2021</a>).</p></section><section id="crf370260-sec-0080"><h3 class="pmc_sec_title">3.4. Plasma‐Activated Water</h3>
<p>Nonthermal plasma, particularly PAW, effectively disinfects <em>L. monocytogenes</em> (Jyung et al. <a href="#crf370260-bib-0094" class="usa-link" aria-describedby="crf370260-bib-0094">2023</a>; Rothwell et al. <a href="#crf370260-bib-0172" class="usa-link" aria-describedby="crf370260-bib-0172">2022</a>). PAW is generated by treating water with a plasma discharge (ionized gas stream) that produces reactive species (Y.‐M. Zhao et al. <a href="#crf370260-bib-0233" class="usa-link" aria-describedby="crf370260-bib-0233">2020</a>). ROS generated in PAW contain potent oxidizing agents, including <sup>•</sup>OH, H<sub>2</sub>O<sub>2</sub>, atomic oxygen (O), superoxide anions (O<sub>2</sub>
<sup>•─</sup>), and O<sub>3</sub>. ROS in PAW induce oxidative stress in microbes and inactivate them by damaging the redox state of the antioxidants (Z. Zhang et al. <a href="#crf370260-bib-0231" class="usa-link" aria-describedby="crf370260-bib-0231">2013</a>), penetrating the cell membrane by promoting lipid oxidation (Joshi et al. <a href="#crf370260-bib-0092" class="usa-link" aria-describedby="crf370260-bib-0092">2011</a>), damaging the cell structure by destroying the intramolecular bonds of the peptidoglycan layer (Yusupov et al. <a href="#crf370260-bib-0226" class="usa-link" aria-describedby="crf370260-bib-0226">2013</a>), interfering with metabolic activities, and eventually causing cell death (Wong et al. <a href="#crf370260-bib-0221" class="usa-link" aria-describedby="crf370260-bib-0221">2023</a>). Fernández‐Gómez et al. (<a href="#crf370260-bib-0058" class="usa-link" aria-describedby="crf370260-bib-0058">2023</a>) analyzed the transcriptomic responses of <em>L. monocytogenes</em> ULE1265 (serotype 1/2c), which were harvested from a stationary‐phase culture, resuspended in Ringer solution, treated with PAW (generated with 36 W discharge power and 30‐min activation time) for 5 min, and stored for 24 h. This PAW disinfection treatment for 5 min resulted in a 2.2 ± 0.1 log<sub>10</sub> CFU/mL reduction in <em>L. monocytogenes</em> count. Moreover, 399 genes were differentially expressed, with 178 being upregulated and 221 being downregulated. Expression levels of metabolism‐related genes, especially the cobalamin‐dependent gene cluster (CDGC) associated with ethanolamine (EA) and 1,2‐propanediol (PD) utilization, internal surface protein, chemotaxis, and QS genes, were increased. In contrast, levels of genes related to the cell envelope and PTS were decreased (Figure <a href="#crf370260-fig-0001" class="usa-link">1</a>). PAW treatment altered the expression levels of several genes related to virulence and general stress responses, many of which were regulated by the alternative sigma factor, SigB. Notably, the glutamate decarboxylase (GAD) system, crucial for acid stress responses, was upregulated. However, no significant changes in the genes typically associated with the oxidative stress responses were observed under the tested conditions. These results suggest that <em>L. monocytogenes</em> responds to PAW by remodeling carbon metabolism, PTS, and CDGC (Fernández‐Gómez et al. <a href="#crf370260-bib-0058" class="usa-link" aria-describedby="crf370260-bib-0058">2023</a>).</p></section></section><section id="crf370260-sec-0090"><h2 class="pmc_sec_title">4. Comparative Analysis of <em>L. monocytogenes</em> Transcriptional Profiles Under Different Disinfection Treatment Conditions</h2>
<p>Comparison of the gene expression patterns of <em>L. monocytogenes</em> under different treatment conditions provides insights into the behavioral mechanisms of the pathogen under various stresses, improving our understanding of the gene regulatory networks and pathways involved in specific biological processes (Q. Wang et al. <a href="#crf370260-bib-0213" class="usa-link" aria-describedby="crf370260-bib-0213">2022</a>). Comparative transcriptome analysis revealed the distinct effects of different disinfection treatments on the genes responsible for key biological processes in <em>L. monocytogenes</em> (Figure <a href="#crf370260-fig-0002" class="usa-link">2</a>). Most gene regulation mechanisms were specific to a particular treatment or shared by only two treatments. Few genes associated with peptidoglycan biosynthesis (<em>murA</em>, <em>murE</em>, <em>mreB</em>, and <em>glmS</em>), cell division (filamentous temperature‐sensitive [<em>fts</em>]‐<em>E</em>, <em>ftsX</em>, <em>ftsW</em>, and <em>actA</em>), glycolysis (<em>gpmA</em> and <em>gpmI</em>), glycerol metabolism (<em>glpD</em> and <em>glpK</em>), nucleic acid metabolism (<em>guaA</em>), protein synthesis (<em>rpsP</em> and <em>rpiB</em>), PTS (<em>bglP</em>), cold shock response (<em>cspD</em>), oxidative stress response (<em>sodA</em>), chemotaxis (<em>cheR</em>), flagellar assembly (<em>fliN</em> and <em>flgE</em>), and virulence (<em>inlA</em>) were commonly affected by at least three different disinfection treatments, as shown in Figure <a href="#crf370260-fig-0002" class="usa-link">2</a> and Figure <a href="#crf370260-supinfo-0001" class="usa-link">S1</a>.</p>
<section id="crf370260-sec-0100"><h3 class="pmc_sec_title">4.1. Cell Wall/Membrane Synthesis and Integrity</h3>
<p>
<em>L. monocytogenes</em> exhibits a cell wall composed of cross‐linked peptidoglycans with multiple layers of teichoic and lipoteichoic acids (Vollmer and Seligman <a href="#crf370260-bib-0208" class="usa-link" aria-describedby="crf370260-bib-0208">2010</a>). The cell wall plays a crucial role in maintaining the integrity of the bacterium, with high internal osmotic pressure defining the cell shape, protecting against mechanical stress, and serving as an anchor for various proteins (Pucciarelli et al. <a href="#crf370260-bib-0154" class="usa-link" aria-describedby="crf370260-bib-0154">2007</a>). Most disinfectants disrupt the bacterial cell membrane, which is closely associated with the cell wall, thereby compromising the cell wall integrity (Brauge et al. <a href="#crf370260-bib-0024" class="usa-link" aria-describedby="crf370260-bib-0024">2020</a>). Transcriptome analysis of <em>L. monocytogenes</em> exposed to various disinfection treatments revealed that these treatments affect multiple genes associated with peptidoglycan and wall teichoic acid (WTA) biosynthesis, cell envelope, and cell division.</p>
<section id="crf370260-sec-0110"><h4 class="pmc_sec_title">4.1.1. Peptidoglycan Biosynthesis</h4>
<p>The bacterial cell wall is a primary disinfection target. The peptidoglycan biosynthesis pathway is essential to maintain the cell wall integrity and is tightly regulated in <em>L. monocytogenes</em> (Fox et al. <a href="#crf370260-bib-0060" class="usa-link" aria-describedby="crf370260-bib-0060">2011</a>). Alterations in this pathway affect the cell surface properties, cell wall integrity, stress tolerance, and overall fitness of <em>L. monocytogenes</em>. Oxidizing agents H<sub>2</sub>O<sub>2</sub> (<em>khpA</em>), O<sub>3</sub> (<em>glmS</em> and <em>murO</em>), and ClO<sub>2</sub> (<em>glmU, murE</em>, and <em>mreB</em>) inhibit the regulation of several peptidoglycan pathway genes (Figure <a href="#crf370260-supinfo-0001" class="usa-link">S1</a>). The <em>murA</em> levels are upregulated in <em>L. monocytogenes</em> following O<sub>3</sub> treatment. Peptidoglycan biosynthesis occurs in the cytoplasm by converting UDP‐<em>N</em>‐acetylglucosamine to UDP‐<em>N</em>‐acetylmuramic acid and lipids I and II through the actions of MurA, MurB, MurCDE, MurF, MraY, and MurG (Schulz et al. <a href="#crf370260-bib-0183" class="usa-link" aria-describedby="crf370260-bib-0183">2022</a>). GlmS, GlmM, GlmU, and GlmR catalyze a four‐step process to convert fructose‐6‐phosphate to UDP‐<em>N</em>‐acetylglucosamine, which is essential for peptidoglycan biosynthesis (Schulz et al. <a href="#crf370260-bib-0183" class="usa-link" aria-describedby="crf370260-bib-0183">2022</a>).</p></section><section id="crf370260-sec-0120"><h4 class="pmc_sec_title">4.1.2. WTA Biosynthesis</h4>
<p>WTAs are anionic glycopolymers covalently attached to the peptidoglycan layers of Gram‐positive bacteria (S. Brown et al. <a href="#crf370260-bib-0028" class="usa-link" aria-describedby="crf370260-bib-0028">2013</a>). These polymers play crucial roles in determining the cell shape, regulating cell division, and influencing other essential physiological features of Gram‐positive bacteria. Teichoic acid glycerol (<em>tag</em> genes) encodes the proteins involved in WTA biosynthesis. Levels of genes involved in teichoic acid biosynthesis, namely <em>gtcA</em>, <em>dltA</em>, <em>dltB</em>, <em>dltC</em>, and <em>dltD</em>, are downregulated after ClO<sub>2</sub> and H<sub>2</sub>O<sub>2</sub> treatments (Table <a href="#crf370260-supinfo-0001" class="usa-link">S1</a>).</p></section><section id="crf370260-sec-0130"><h4 class="pmc_sec_title">4.1.3. Cell Division Genes</h4>
<p>Cell division is a crucial process in living organisms. Bacteria use binary fission to generate offspring identical to the parent cell (Casiraghi et al. <a href="#crf370260-bib-0036" class="usa-link" aria-describedby="crf370260-bib-0036">2020</a>). To ensure correct cell division, parent cells coordinate various processes, including overall biomass increase, membrane synthesis, chromosome duplication, and cell wall production (Silber et al. <a href="#crf370260-bib-0191" class="usa-link" aria-describedby="crf370260-bib-0191">2020</a>). These processes are mediated by a protein complex, known as the divisome, which regulates cell division (Mahone and Goley <a href="#crf370260-bib-0119" class="usa-link" aria-describedby="crf370260-bib-0119">2020</a>). <em>fts</em> genes in <em>L. monocytogenes</em> show different responses to various disinfection treatments. ClO<sub>2</sub> downregulates <em>ftsE, ftsX, ftsW</em>, and <em>ftsL</em> levels, but not <em>ftsH</em> levels. O<sub>3</sub> and H<sub>2</sub>O<sub>2</sub> suppress the regulation of <em>sepF, LMOf2365_2563</em>, and <em>ftsW</em>. Moreover, levels of <em>yneA</em>, which encodes the cell division suppressor protein YneA, are upregulated in response to H<sub>2</sub>O<sub>2</sub>. The products of these <em>fts</em> genes are essential for creating the septum and initiating cell division in bacteria (Han et al. <a href="#crf370260-bib-0073" class="usa-link" aria-describedby="crf370260-bib-0073">2021</a>).</p>
<p>
<em>L. monocytogenes</em> modifies its cell wall composition and thickness in response to stress, which affects its susceptibility to disinfectants (Cotter and Hill <a href="#crf370260-bib-0244" class="usa-link" aria-describedby="crf370260-bib-0244">2003</a>). Downregulation of peptidoglycan (<em>mur</em> and<em> glm</em>) and WTA (<em>dlt</em>) biosynthesis gene levels in <em>L. monocytogenes</em> upon exposure to ClO<sub>2</sub>, H<sub>2</sub>O<sub>2</sub>, and O<sub>3</sub> indicates a stress response during which the bacterium conserves energy by limiting cell wall synthesis. This results in weakened cell walls and increased membrane permeability, making the pathogen more susceptible to oxidizing agents (Schulz et al. <a href="#crf370260-bib-0183" class="usa-link" aria-describedby="crf370260-bib-0183">2022</a>).</p></section></section><section id="crf370260-sec-0140"><h3 class="pmc_sec_title">4.2. Metabolic Pathway Genes</h3>
<p>Central metabolic pathways actively produce stored biological energy and generate vital metabolic precursors necessary to build the fundamental cellular components of all living organisms (Romano and Conway <a href="#crf370260-bib-0168" class="usa-link" aria-describedby="crf370260-bib-0168">1996</a>). Gene transcription operates in conjunction with cell metabolism, with both mutually regulating each other (Bian et al. <a href="#crf370260-bib-0016" class="usa-link" aria-describedby="crf370260-bib-0016">2022</a>). Bacterial metabolic processes are the primary targets of disinfection treatments that cause <em>L. monocytogenes</em> to adjust its metabolism to survive under the disinfection stress (Bland et al. <a href="#crf370260-bib-0018" class="usa-link" aria-describedby="crf370260-bib-0018">2022</a>). Disinfection treatments affect various metabolic pathways, including respiration, energy, carbohydrate, lipid, nucleic acid, and amino acid metabolism pathways.</p>
<section id="crf370260-sec-0150"><h4 class="pmc_sec_title">4.2.1. Respiration and Energy Metabolism</h4>
<p>Cellular respiration is crucial for generating energy and maintaining the electron balance in cells, particularly under stress. Aerobic organisms generate energy via cellular respiration, which involves oxidizing an electron donor, such as glucose, and reducing an electron acceptor, such as oxygen, resulting in the production of carbon dioxide. Electron exchange occurs via the electron transport chain to produce ATP, the primary energy currency for cells (Fuhrmann <a href="#crf370260-bib-0063" class="usa-link" aria-describedby="crf370260-bib-0063">2021</a>). <em>L. monocytogenes</em> exhibits flexible respirofermentative metabolism, allowing it to thrive in diverse environments using various carbon sources (Rivera‐Lugo et al. <a href="#crf370260-bib-0163" class="usa-link" aria-describedby="crf370260-bib-0163">2022</a>). Cellular respiration maintains the redox balance, regenerates nicotinamide adenine dinucleotide (NAD<sup>+</sup>), and produces a proton motive force. The Menaquinone (MK) biosynthetic pathway is essential for both aerobic and anaerobic respiration (Smith et al. <a href="#crf370260-bib-0194" class="usa-link" aria-describedby="crf370260-bib-0194">2023</a>). Various genes involved in the electron transport chain, NAD<sup>+</sup> regeneration, and ATP synthesis are differentially regulated under ClO<sub>2</sub> and O<sub>3</sub> treatments (Table <a href="#crf370260-supinfo-0001" class="usa-link">S2</a>; Supporting Information: Data <a href="#crf370260-supinfo-0001" class="usa-link">S2.2.1</a>), suggesting that energy metabolic pathways are responsive to oxidative stress.</p>
<section id="crf370260-sec-0160"><h5 class="pmc_sec_title">4.2.1.1. NAD<sup>+</sup> Regeneration and ATP Synthesis</h5>
<p>NAD<sup>+</sup> is a crucial small molecule facilitating electron and proton transfer during enzymatic reactions (Chen et al. <a href="#crf370260-bib-0039" class="usa-link" aria-describedby="crf370260-bib-0039">2024</a>). It also plays a vital role in cellular energy metabolism by reducing NAD<sup>+</sup> to NADH (reduced form of NAD) in multiple metabolic signaling pathways, including glycolysis and tricarboxylic acid (TCA) cycle (Xiao et al. <a href="#crf370260-bib-0223" class="usa-link" aria-describedby="crf370260-bib-0223">2018</a>). NAD<sup>+</sup> acts as a coenzyme in vital processes, including glycolysis, oxidative phosphorylation, protein deacetylation, DNA repair, and Ca<sup>2+</sup> activation. Expression of the NADPH dehydrogenase gene <em>namA</em> and the glutamine‐dependent NAD<sup>+</sup> synthetase gene <em>nadE</em> was suppressed by O<sub>3</sub>. Exposure to ClO<sub>2</sub> increases the expression of <em>atpC</em> in <em>L. monocytogenes</em>. ATP synthases are multi‐subunit enzyme complexes found in bacterial cell membranes (Guo et al. <a href="#crf370260-bib-0072" class="usa-link" aria-describedby="crf370260-bib-0072">2019</a>). They use transmembrane electrochemical proton motive force to generate ATP from ADP and inorganic phosphate (Pi) (Meyrat and von Ballmoos <a href="#crf370260-bib-0130" class="usa-link" aria-describedby="crf370260-bib-0130">2019</a>). Downregulation of NADPH dehydrogenase and NAD<sup>+</sup> synthetase gene levels by O<sub>3</sub> suggests a potential reduction in NADPH and ATP synthesis. This causes an imbalance in the NADH/NAD<sup>+</sup> ratio within the cells, affecting central carbon and energy metabolism (Zhang et al. <a href="#crf370260-bib-0245" class="usa-link" aria-describedby="crf370260-bib-0245">2014</a>). These expression patterns indicate the stress‐associated shifts in redox‐related pathways.</p></section></section><section id="crf370260-sec-0170"><h4 class="pmc_sec_title">4.2.2. Carbohydrate Metabolism</h4>
<p>Carbohydrate metabolism encompasses all biochemical processes responsible for the formation, breakdown, and conversion of carbohydrates to provide a steady energy source for living cells (Chandel <a href="#crf370260-bib-0037" class="usa-link" aria-describedby="crf370260-bib-0037">2021</a>). Among the other carbohydrates, glucose, fructose, and galactose play vital roles in several metabolic pathways. The disinfection treatments differentially expressed several carbohydrate metabolic pathways, including glycolysis, the pentose phosphate pathway (PPP), and genes associated with acetoin biosynthesis, pyruvate fermentation, rhamnose and lactose metabolism, as well as glycosidase activity (Table <a href="#crf370260-supinfo-0001" class="usa-link">S2</a>; Supporting Information: Data <a href="#crf370260-supinfo-0001" class="usa-link">S2.2.2</a>).</p>
<section id="crf370260-sec-0180"><h5 class="pmc_sec_title">4.2.2.1. Glycolysis</h5>
<p>Glycolysis is a central metabolic pathway that converts glucose to pyruvate or lactate, producing ATP, NADH, or water through various enzymatic reactions to generate energy (Chaudhry and Varacallo <a href="#crf370260-bib-0038" class="usa-link" aria-describedby="crf370260-bib-0038">2024</a>). Among the disinfection treatments against <em>L. monocytogenes</em>, ClO<sub>2</sub> and PAW treatments upregulated several glycolytic pathway gene levels, whereas O<sub>3</sub> treatment downregulated them (Supporting Information: Data <a href="#crf370260-supinfo-0001" class="usa-link">S2.2.2</a>). The major glycolytic genes upregulated by disinfection treatments included <em>gpmA</em> (ClO<sub>2</sub>), <em>ccpN</em> (H<sub>2</sub>O<sub>2</sub>), <em>cggR</em> (PAW), <em>eno</em> (PAW), and <em>pgk</em> (PAW), whereas the downregulated genes included <em>gpmA</em> (O<sub>3</sub>), <em>gpmI</em> (O<sub>3</sub>), <em>pgi</em> (ClO<sub>2</sub>), and <em>pgmB</em> (O<sub>3</sub>). In addition to generating ATP and NADH, glycolysis produces precursor metabolites for several metabolic pathways, including the peptidoglycan biosynthesis pathway, which is essential for cell wall integrity and tolerance to cell wall‐targeting antimicrobial agents (Fuhrmann <a href="#crf370260-bib-0063" class="usa-link" aria-describedby="crf370260-bib-0063">2021</a>; Reed et al. <a href="#crf370260-bib-0159" class="usa-link" aria-describedby="crf370260-bib-0159">2015</a>). Upregulation of glycolytic gene levels by disinfection treatments allows <em>L. monocytogenes</em> to efficiently utilize the available carbohydrates, ensuring a constant supply of energy and metabolic intermediates necessary for cell wall regeneration, energy production, and other stress response mechanisms. However, glycolysis disruption by ClO<sub>2</sub> treatment possibly induces cell death (Ranjbar et al. <a href="#crf370260-bib-0157" class="usa-link" aria-describedby="crf370260-bib-0157">2020</a>).</p></section><section id="crf370260-sec-0190"><h5 class="pmc_sec_title">4.2.2.2. Pentose Phosphate Pathway</h5>
<p>PPP is a glucose oxidation pathway operating alongside glycolysis (Wamelink et al. <a href="#crf370260-bib-0211" class="usa-link" aria-describedby="crf370260-bib-0211">2008</a>). It produces ribose 5‐phosphate and NADPH, redirecting carbon to the glycolytic or gluconeogenic pathways. Ribose 5‐phosphate is used to create nucleotides, whereas NADPH maintains the redox balance and facilitates biosynthetic activities, including the production of tetrahydrofolate, deoxyribonucleotides, proline, and fatty acids (Kim et al. <a href="#crf370260-bib-0103" class="usa-link" aria-describedby="crf370260-bib-0103">2023</a>). Transcript levels of PPP enzymes, including <em>tal1</em>, <em>lmo2674</em>, and <em>lmo0342</em>, are increased in <em>L. monocytogenes</em> in response to ClO<sub>2</sub> treatment. O<sub>3</sub> treatment also increases the expression levels of the PPP enzyme <em>fsa</em> and decreases those of <em>tkt</em> and <em>tktC</em> in <em>L. monocytogenes</em>. PPP plays crucial roles in maintaining the balance of cellular redox levels and supporting biosynthesis (TeSlaa et al. <a href="#crf370260-bib-0198" class="usa-link" aria-describedby="crf370260-bib-0198">2023</a>). These transcriptional changes suggest treatment‐specific effects on carbon allocation via PPP. PPP interruption influences the cell wall, alters the oxygen radical sensitivity, reduces nucleotide synthesis, and affects biofilm formation (Kim et al. <a href="#crf370260-bib-0103" class="usa-link" aria-describedby="crf370260-bib-0103">2023</a>).</p></section><section id="crf370260-sec-0200"><h5 class="pmc_sec_title">4.2.2.3. TCA Cycle</h5>
<p>
<em>L. monocytogenes</em> has a distinct and specialized metabolism because of its incomplete TCA cycle (Schauer et al. <a href="#crf370260-bib-0180" class="usa-link" aria-describedby="crf370260-bib-0180">2010</a>). It lacks the α‐ketoglutarate dehydrogenase enzyme, essential for converting α‐ketoglutarate to succinyl‐coenzyme A (CoA). This absence restricts the pathogen's capacity to fully utilize the TCA cycle for energy production and biosynthesis (Whiteley et al. <a href="#crf370260-bib-0217" class="usa-link" aria-describedby="crf370260-bib-0217">2017</a>). <em>L. monocytogenes</em> compensates for this by using alternative pathways, including glycolysis, PPP, and pyruvate carboxylation pathways, to produce the intermediates necessary for metabolism and energy production (Schär et al. <a href="#crf370260-bib-0179" class="usa-link" aria-describedby="crf370260-bib-0179">2010</a>; Schauer et al. <a href="#crf370260-bib-0180" class="usa-link" aria-describedby="crf370260-bib-0180">2010</a>). This metabolic flexibility is crucial for the survival and virulence of bacteria within host cells (Fuchs et al. <a href="#crf370260-bib-0062" class="usa-link" aria-describedby="crf370260-bib-0062">2012</a>). H<sub>2</sub>O<sub>2</sub> treatment upregulates the expression levels of <em>citB</em>, <em>citC</em>, and <em>citZ</em> genes in <em>L. monocytogenes</em> (Table <a href="#crf370260-supinfo-0001" class="usa-link">S2</a>). Genes <em>citB</em>, <em>citC</em>, and <em>citZ</em>, which encode aconitase, isocitrate dehydrogenase, and citrate synthetase, respectively, are associated with the regulation of the TCA cycle and central metabolism. Oxidative stress due to H<sub>2</sub>O<sub>2</sub> induces metabolic changes in bacteria, including increased pyruvate production and reduced pyruvate dehydrogenase (PDH) activity, resulting in altered metabolic flux and elevated pyruvate levels under stress (Bignucolo et al. <a href="#crf370260-bib-0017" class="usa-link" aria-describedby="crf370260-bib-0017">2013</a>).</p></section><section id="crf370260-sec-0210"><h5 class="pmc_sec_title">4.2.2.4. Pyruvate Metabolism</h5>
<p>Pyruvate is an essential metabolite that connects critical metabolic pathways, including glycolysis, gluconeogenesis, TCA cycle, amino acid biosynthesis, and fatty acid biosynthesis (FAB) (Luo et al. <a href="#crf370260-bib-0118" class="usa-link" aria-describedby="crf370260-bib-0118">2023</a>). The initial substrate for the gluconeogenic pathway is pyruvate, which can be utilized to produce glucose (Brissac et al. <a href="#crf370260-bib-0026" class="usa-link" aria-describedby="crf370260-bib-0026">2015</a>). H<sub>2</sub>O<sub>2</sub> and ClO<sub>2</sub> disinfection treatments exert different effects on the expression levels of genes involved in pyruvate metabolism. Levels of genes associated with PDH complexes, <em>pdhA, pdhB, pdhC</em>, and <em>pdhD</em>, and the pyruvate formate‐lyase gene <em>pflA</em> are downregulated by H<sub>2</sub>O<sub>2</sub> treatment. In contrast, ClO<sub>2</sub> upregulates <em>pdhC, pdhD</em>, and pyruvate carboxylase (PYC) gene <em>pycA</em> levels. Pyruvate metabolism, primarily through the function of the PDH complex and PYC, plays a vital role in carbon metabolism in <em>L. monocytogenes</em>. The PDH complex catalyzes the oxidative decarboxylation of pyruvate to acetyl‐CoA, which connects glycolysis to the TCA cycle (Fuchs et al. <a href="#crf370260-bib-0062" class="usa-link" aria-describedby="crf370260-bib-0062">2012</a>). <em>L. monocytogenes</em> exhibits an incomplete TCA cycle, making PYC‐catalyzed carboxylation of pyruvate essential for oxaloacetate production (Schär et al. <a href="#crf370260-bib-0179" class="usa-link" aria-describedby="crf370260-bib-0179">2010</a>). These observations suggest potential shifts in the carbon flux under oxidative stress.</p></section><section id="crf370260-sec-0220"><h5 class="pmc_sec_title">4.2.2.5. Other Carbohydrate Metabolic Processes</h5>
<p>Disinfection treatments further altered the expression of <em>L. monocytogenes</em> genes involved in other carbohydrate metabolic processes, such as fructose metabolism (<em>fba</em> upregulated by O<sub>3</sub>; <em>fruA</em> upregulated by PAW), lactose metabolism (<em>ldh</em> downregulated by PAW), rhamnose metabolism (<em>lmo1082</em> downregulated by ClO<sub>2</sub>), and chitin degradation (<em>lmo1883</em> upregulated by ClO<sub>2</sub>; <em>chbG</em> downregulated by O<sub>3</sub>). During disinfection, <em>L. monocytogenes</em> cells enter a state of stress, requiring increased intake of carbohydrates as catabolic products for cell repair. Downregulation of genes related to carbohydrate metabolism suggests that disinfection methods directly restrict the ability of bacteria to absorb carbohydrates from their surroundings. This further affects the ability of bacteria to use their breakdown products for cellular maintenance, potentially limiting access to metabolic intermediates necessary for their survival (Qian et al. <a href="#crf370260-bib-0156" class="usa-link" aria-describedby="crf370260-bib-0156">2020</a>).</p></section></section><section id="crf370260-sec-0230"><h4 class="pmc_sec_title">4.2.3. Lipid Metabolism</h4>
<p>In addition to carbohydrate metabolism, the disinfection treatment affected several genes related to FAB and glycerol metabolism (Table <a href="#crf370260-supinfo-0001" class="usa-link">S2</a>). Lipids are the primary components of the bacterial cytoplasmic membrane. They form a phospholipid bilayer structure and can be categorized into several chemical classes varying in molecular size and isomeric form (Strahl and Errington <a href="#crf370260-bib-0246" class="usa-link" aria-describedby="crf370260-bib-0246">2017</a>). The precursor molecules of lipids are primarily obtained from central metabolism. Lipid metabolism is crucial for the ability of a bacterium to survive and multiply within host cells. Disinfection with H<sub>2</sub>O<sub>2</sub>, ClO<sub>2</sub>, O<sub>3</sub>, and PAW affected FAB and glycerol metabolism differently (Table <a href="#crf370260-supinfo-0001" class="usa-link">S2</a>; Supporting Information: Data <a href="#crf370260-supinfo-0001" class="usa-link">S2.2.3</a>).</p>
<section id="crf370260-sec-0240"><h5 class="pmc_sec_title">4.2.3.1. Fatty Acid Biosynthesis</h5>
<p>The FAB pathway is a potential target for disinfection treatments, including antibiotics, because it plays a vital role in creating phospholipid membranes, lipoproteins, and lipopolysaccharides (Y.‐M. Zhang et al. <a href="#crf370260-bib-0230" class="usa-link" aria-describedby="crf370260-bib-0230">2006</a>). Phospholipids are key components of the membrane lipids that form lipid bilayers. They create a protective barrier for cells and facilitate cellular processes within subcellular compartments (Dai et al. <a href="#crf370260-bib-0048" class="usa-link" aria-describedby="crf370260-bib-0048">2021</a>). Phosphatidic acid is produced through two consecutive acylation reactions of glycerol‐3‐phosphate using acyl‐acyl carrier protein (ACP) produced by the bacterial type II fatty acid synthesis system (FASII). This substance is a precursor of all phospholipid species (Yao and Rock <a href="#crf370260-bib-0224" class="usa-link" aria-describedby="crf370260-bib-0224">2017</a>). <em>Listeria</em> spp. initiate the first reaction of the FASII pathway by combining fatty acid thioesters with different acyl chain lengths with a malonyl‐ACP (Soares da Costa et al. <a href="#crf370260-bib-0195" class="usa-link" aria-describedby="crf370260-bib-0195">2017</a>). Exposure of <em>L. monocytogenes</em> to PAW upregulates the levels of FASII pathway genes <em>accB</em> and <em>fabI</em>. In contrast, ClO<sub>2</sub> downregulates the levels of FAB gene <em>acpA, fabG</em>, <em>fabH</em>, and <em>lmo0970</em> in <em>L. monocytogenes</em>. These transcriptional changes indicate alterations in membrane lipid synthesis in response to oxidizing agents. FASII initiation, elongation, and acyltransferase modules work together to synthesize fatty acids in appropriate quantities and structures (Parsons and Rock <a href="#crf370260-bib-0144" class="usa-link" aria-describedby="crf370260-bib-0144">2013</a>).</p></section><section id="crf370260-sec-0250"><h5 class="pmc_sec_title">4.2.3.2. Glycerol Metabolism</h5>
<p>Glycerol plays a crucial role in <em>L. monocytogenes</em> virulence and biofilm formation, serving as a significant carbon source that is metabolized through pathways involving glycerol kinase (GlpK) and glycerol‐3‐phosphate dehydrogenase (GlpD) (Joseph et al. <a href="#crf370260-bib-0090" class="usa-link" aria-describedby="crf370260-bib-0090">2008</a>). Bacteria use glycerol dehydrogenase to convert glycerol into dihydroxyacetone (Dha). <em>L. monocytogenes</em> possesses two Dha kinase systems, which are part of a PEP‐dependent phosphorylation system for Dha utilization and are connected to the PPP (Monniot et al. <a href="#crf370260-bib-0133" class="usa-link" aria-describedby="crf370260-bib-0133">2012</a>). Expression levels of genes involved in glycerol metabolism in <em>L. monocytogenes</em> are upregulated (<em>isp</em> and <em>LMOf2365_1834</em>) and downregulated (<em>ispE</em>, <em>ispD</em>, <em>dhaK1</em>, <em>LMOf2365_1310</em>, <em>LMOf2365_0375</em>, <em>LMOf2365_2202</em>, <em>LMOf2365_2487</em>, and <em>LMOf2365_2806</em>) after O<sub>3</sub> treatment. Other chemical oxidants, H<sub>2</sub>O<sub>2</sub> (<em>dhaI</em>, <em>dhaK</em>, and <em>dhaM</em>) and ClO<sub>2</sub> (<em>glpD</em>, <em>glpK</em>, and <em>lmo0347</em>), upregulated, whereas PAW (<em>glpD</em> and <em>glpK</em>) downregulated glycerol metabolism gene levels in <em>L. monocytogenes</em>. These changes in gene expression suggest that glycerol utilization is differentially regulated in response to varying levels of oxidative stress.</p></section></section><section id="crf370260-sec-0260"><h4 class="pmc_sec_title">4.2.4. Nucleic Acid Metabolism</h4>
<p>Disinfection also significantly affected the expression of genes involved in nucleotide biosynthesis. Nucleotides are crucial for various cellular processes such as DNA replication, energy storage, virulence, and signaling and are continuously synthesized de novo in all cells (Goncheva et al. <a href="#crf370260-bib-0068" class="usa-link" aria-describedby="crf370260-bib-0068">2022</a>). Treatment with H<sub>2</sub>O<sub>2</sub> increased the expression of the <em>xpt</em> gene, whereas ClO<sub>2</sub> decreased xpt expression in <em>L. monocytogenes</em>. The <em>xpt</em> gene encodes xanthine phosphoribosyltransferase, which is involved in purine salvage pathways (Glaser et al. <a href="#crf370260-bib-0067" class="usa-link" aria-describedby="crf370260-bib-0067">2001</a>). This enzyme converts xanthine, a nucleic acid breakdown product, into xanthosine 5'‐monophosphate, which allows the pathogen to reuse xanthine for RNA or DNA synthesis. Levels of the <em>guaA</em> gene, which encodes a guanosine monophosphate (GMP) synthase, are decreased after ClO<sub>2</sub> treatment (Table <a href="#crf370260-supinfo-0001" class="usa-link">S2</a>). This gene is responsible for converting inosinic acid to GMP (Bennett et al. <a href="#crf370260-bib-0015" class="usa-link" aria-describedby="crf370260-bib-0015">2007</a>). Similar to glycerol metabolism, levels of several genes related to nucleic acid metabolism in <em>L. monocytogenes</em> are upregulated (<em>purA</em>, <em>mrnC</em>, <em>nrdD</em>, <em>nudL</em>, <em>xerC</em>, <em>gmk</em>, <em>carB</em>, and <em>tenA</em>) or downregulated (<em>purL</em>, <em>purE</em>, <em>recF</em>, <em>adaB</em>, <em>dinB</em>, <em>gyrB</em>, <em>holB</em>, <em>cysK</em>, <em>rnhC</em>, <em>dprA</em>, <em>LMOf2365_1336</em>, <em>LMOf2365_1861</em>, <em>LMOf2365_2183</em>, <em>LMOf2365_2186</em>, <em>LMOf2365_2656</em>, and <em>LMOf2365_1822</em>) by O<sub>3</sub> treatment. These transcriptional shifts indicate treatment‐specific effects on nucleotide synthesis, salvage, and DNA repair under stress conditions.</p></section><section id="crf370260-sec-0270"><h4 class="pmc_sec_title">4.2.5. Amino Acid Metabolism</h4>
<p>Amino acids are essential for protein synthesis in microorganisms and play crucial roles in regulating peptidoglycan metabolism, maintaining cell structure, promoting bacterial growth, facilitating biofilm formation, and protecting against environmental hazards (Miyamoto and Homma <a href="#crf370260-bib-0132" class="usa-link" aria-describedby="crf370260-bib-0132">2021</a>). ClO<sub>2</sub> treatment upregulates the expression levels of <em>gcvT</em>, <em>gcvH</em>, and <em>gcvPB</em> (Table <a href="#crf370260-supinfo-0001" class="usa-link">S2</a>). Excess glycine hinders the growth of several bacteria (Minami et al. <a href="#crf370260-bib-0131" class="usa-link" aria-describedby="crf370260-bib-0131">2004</a>). The glycine cleavage system is a multienzyme system, catalyzing the oxidative cleavage of glycine to form 5,10‐methylene‐tetrahydrofolate, a crucial one‐carbon precursor for various biosynthetic processes, along with serine, thymidine, and purines (M. J. Brown et al. <a href="#crf370260-bib-0027" class="usa-link" aria-describedby="crf370260-bib-0027">2014</a>). The GCS pathway also contributes to energy metabolism in bacterial cells by generating NADH (Kikuchi et al. <a href="#crf370260-bib-0102" class="usa-link" aria-describedby="crf370260-bib-0102">2008</a>). Upregulation of GCS gene levels by ClO<sub>2</sub> treatment increases the metabolic fitness of <em>L. monocytogenes</em> (M. J. Brown et al. <a href="#crf370260-bib-0027" class="usa-link" aria-describedby="crf370260-bib-0027">2014</a>).</p>
<p>Some genes associated with the GAD pathway are differentially expressed in <em>L. monocytogenes</em> in response to disinfection. ClO<sub>2</sub> and PAW treatments upregulate the expression levels of <em>gadB</em> and <em>gadC</em> in <em>L. monocytogenes</em>. The GAD pathway functions as a central mechanism for acid tolerance, facilitating microbial survival at low pH (Sezgin and Tekin <a href="#crf370260-bib-0184" class="usa-link" aria-describedby="crf370260-bib-0184">2023</a>). This pathway transports extracellular glutamate into the cell and converts it to γ‐aminobutyrate (GABA), thereby increasing the intracellular pH under acid stress to maintain the intracellular pH homeostasis (C. Wang et al. <a href="#crf370260-bib-0212" class="usa-link" aria-describedby="crf370260-bib-0212">2018</a>). GAD pathway gene expression changes in <em>L. monocytogenes</em> by ClO<sub>2</sub> and PAW treatments primarily affect its acid tolerance and intracellular pH homeostasis, further impacting metabolism, stress response, and survival (Feehily and Karatzas <a href="#crf370260-bib-0057" class="usa-link" aria-describedby="crf370260-bib-0057">2013</a>).</p></section></section><section id="crf370260-sec-0280"><h3 class="pmc_sec_title">4.3. Metabolic Enzymes</h3>
<p>
<em>L. monocytogenes</em> resists disinfectants via biofilm formation (Skowron et al. <a href="#crf370260-bib-0193" class="usa-link" aria-describedby="crf370260-bib-0193">2018</a>), detoxification using efflux pumps (Jiang et al. <a href="#crf370260-bib-0086" class="usa-link" aria-describedby="crf370260-bib-0086">2016</a>), oxidative stress responses (Ogunleye et al. <a href="#crf370260-bib-0140" class="usa-link" aria-describedby="crf370260-bib-0140">2024</a>), and adaptive metabolic flexibility (Wiktorczyk‐Kapischke et al. <a href="#crf370260-bib-0218" class="usa-link" aria-describedby="crf370260-bib-0218">2021</a>). Metabolic flexibility allows <em>L. monocytogenes</em> to use various carbon sources (e.g., such as glycerol and amino acids), depending on the environmental conditions (Schauer et al. <a href="#crf370260-bib-0180" class="usa-link" aria-describedby="crf370260-bib-0180">2010</a>). Metabolic enzymes facilitate this adaptability, enabling the bacterium to switch between metabolic pathways, thereby enhancing its survival upon disinfectant exposure. Understanding these enzymes can aid in the development of novel methods to control <em>L. monocytogenes</em> infections in the food industry. The major metabolic enzymes exhibiting differential expression in response to various disinfection treatments (H<sub>2</sub>O<sub>2</sub>, ClO<sub>2</sub>, O<sub>3</sub>, and PAW) are oxidoreductases, peptidases, transferases, and hydrolases (Table <a href="#crf370260-supinfo-0001" class="usa-link">S3</a>; Supporting Information: Data <a href="#crf370260-supinfo-0001" class="usa-link">S2.3</a>).</p>
<section id="crf370260-sec-0290"><h4 class="pmc_sec_title">4.3.1. Oxidoreductases</h4>
<p>Oxidoreductases are a class of enzymes facilitating oxidation–reduction or redox reactions by transferring electrons between molecules (Nord <a href="#crf370260-bib-0139" class="usa-link" aria-describedby="crf370260-bib-0139">2009</a>). In these reactions, one substrate is oxidized and loses electrons, whereas the other is reduced and gains electrons. Common oxidoreductases include dehydrogenases, which remove hydrogen atoms (and thus electrons) from substrates, oxidases, which transfer electrons to oxygen molecules, and reductases, which catalyze the reduction of a substrate by adding electrons (Cárdenas‐Moreno et al. <a href="#crf370260-bib-0033" class="usa-link" aria-describedby="crf370260-bib-0033">2023</a>). These enzymes play critical roles in various biological processes, including cellular respiration, metabolism, and ROS detoxification. Oxidizing agents H<sub>2</sub>O<sub>2</sub> (<em>ywnB</em>), ClO<sub>2</sub> (<em>lmo0521</em>, <em>lmo1789</em>, <em>lmo0669</em>, <em>lmo0613</em>, <em>lmo2572</em>, <em>lmo0761</em>, <em>lmo1433</em>, <em>lmo2426</em>, and <em>lmo2573</em>), and O<sub>3</sub> (<em>morA</em> and <em>LMOf2365_0583</em>) upregulate the expression levels of oxidoreductase activity genes. However, O<sub>3</sub> (<em>ssuE</em> and <em>bsaA</em>) and PAW (<em>adhE</em>) decrease the expression levels of specific oxidoreductase genes in <em>L. monocytogenes</em>. Increased expression of oxidoreductases protects the cells against damage from ROS and other oxidizing agents (Ingram <a href="#crf370260-bib-0082" class="usa-link" aria-describedby="crf370260-bib-0082">2018</a>). These enzymes detoxify harmful oxidants and maintain the redox balance within the cells (Cheng et al. <a href="#crf370260-bib-0040" class="usa-link" aria-describedby="crf370260-bib-0040">2017</a>).</p></section><section id="crf370260-sec-0300"><h4 class="pmc_sec_title">4.3.2. Transferases</h4>
<p>Transferases facilitate the transfer of functional groups from donors to acceptor molecules. These transferases can be categorized into several subclasses based on their functional groups, including methyltransferases, <em>N</em>‐acetyltransferases, glutathione <em>S</em>‐transferases, sulfur transferases, transaminases, phosphotransferases, acyltransferases, glycosyltransferases, selenotransferases, and molybdenum‐ or tungsten‐transferases based on the functional groups they transfer (McDonald and Tipton <a href="#crf370260-bib-0128" class="usa-link" aria-describedby="crf370260-bib-0128">2023</a>). Transferases in bacteria play essential roles in various functions, including cellular detoxification (Allocati et al. <a href="#crf370260-bib-0006" class="usa-link" aria-describedby="crf370260-bib-0006">2009</a>), biosynthesis pathways of disaccharides, oligosaccharides, and polysaccharides; protein glycosylation (J. Schmid et al. <a href="#crf370260-bib-0182" class="usa-link" aria-describedby="crf370260-bib-0182">2016</a>), antimicrobial drug resistance; and bacterial capsule polymerization (Litschko et al. <a href="#crf370260-bib-0114" class="usa-link" aria-describedby="crf370260-bib-0114">2024</a>). H<sub>2</sub>O<sub>2</sub> treatment increases the expression of specific transferase genes, including <em>lmo0602</em> and <em>lmo0652</em>. In contrast, O<sub>3</sub> treatment resulted in both an increase (<em>LMOf2365_0152</em>) and a decrease (<em>ackA</em>, <em>dacA</em>, and <em>LMOf2365_0447</em>) in the expression of some transferase genes. Some transferases in the bacterial antioxidant defense system protect against oxidative stress by neutralizing ROS and other oxidizing agents (Allocati et al. <a href="#crf370260-bib-0006" class="usa-link" aria-describedby="crf370260-bib-0006">2009</a>). The elevated expression of transferase genes induced by the oxidizing agents H<sub>2</sub>O<sub>2</sub> and O<sub>3</sub> suggests that the bacteria actively attempt to neutralize or modify the disinfectant molecules. These alterations in expression may indicate a shift in the cellular processing of metabolites or detoxification responses to oxidative stress.</p></section><section id="crf370260-sec-0310"><h4 class="pmc_sec_title">4.3.3. Hydrolases</h4>
<p>Hydrolases are enzymes that catalyze the cleavage of chemical bonds (e.g., C–C, C–O, C–N, and C–S) by adding –H or –OH groups from H<sub>2</sub>O across the bond (Kermasha and Eskin <a href="#crf370260-bib-0100" class="usa-link" aria-describedby="crf370260-bib-0100">2021</a>). These hydrolytic enzymes participate in various cellular processes, including DNA repair, RNA processing, protein degradation, cell wall growth and remodeling, cell division, autolysis, peptidoglycan recycling, detoxification, and regulation of cell signaling pathways (Do et al. <a href="#crf370260-bib-0051" class="usa-link" aria-describedby="crf370260-bib-0051">2020</a>; Nardini et al. <a href="#crf370260-bib-0136" class="usa-link" aria-describedby="crf370260-bib-0136">1999</a>; Qi et al. <a href="#crf370260-bib-0155" class="usa-link" aria-describedby="crf370260-bib-0155">2022</a>). Expression levels of specific hydrolase activity genes (<em>yocD</em>, <em>bds1</em>, and <em>lmo2266</em>) in <em>L. monocytogenes</em> are upregulated in response to H<sub>2</sub>O<sub>2</sub>. ClO<sub>2</sub> and O<sub>3</sub> treatments both upregulate (ClO<sub>2</sub>: <em>lmo1577</em>, <em>lmo0869</em>, <em>lmo2603</em>, <em>lmo0268</em>, <em>lmo0907</em>, <em>lmo1254</em>, <em>lmo0764</em>, <em>lmo0261</em>, and <em>lmo1728</em>; O<sub>3</sub>: <em>aguA</em>) and downregulate (ClO<sub>2</sub>: <em>lmo0998</em>, <em>lmo2450</em>, and <em>lmo1240</em>; O<sub>3</sub>: <em>LMOf2365_0277</em>, <em>LMOf2365_0523</em>, and <em>LMOf2365_0341</em>) specific hydrolase activity gene levels in <em>L. monocytogenes</em>. Peptidoglycan hydrolases play a role in cell‐wall remodeling and turnover. These enzymes may help to modify the cell wall structure to reduce disinfectant penetration during stress (Lee and Huang <a href="#crf370260-bib-0107" class="usa-link" aria-describedby="crf370260-bib-0107">2013</a>). Some hydrolases may directly inactivate or modify disinfectant molecules, helping bacteria detoxify and survive during treatment. These changes suggest potential alterations in the cell wall or detoxification response to oxidative stress (F. Zhang and Cheng <a href="#crf370260-bib-0229" class="usa-link" aria-describedby="crf370260-bib-0229">2022</a>).</p>
<p>Peptidases, also known as proteases, catalyze the breakdown of peptide bonds through a process called hydrolysis. By cleaving these bonds, peptidases convert long protein molecules into smaller peptides or individual amino acids, which are essential for various biological processes, such as protein maturation, recycling, and cellular regulation (Potempa and Pike <a href="#crf370260-bib-0151" class="usa-link" aria-describedby="crf370260-bib-0151">2004</a>). They are crucial for bacterial growth, metabolism, and adaptation to various environments. These proteolytic enzymes have diverse structures, perform different biological roles, and can be found in the cytoplasm, attached to the cell membrane, or released into the surroundings (Nguyen et al. <a href="#crf370260-bib-0137" class="usa-link" aria-describedby="crf370260-bib-0137">2019</a>). The O<sub>3</sub> and PAW treatments showed varied effects on the expression of specific peptidase activity genes in <em>L. monocytogenes</em>, with some genes being upregulated (O<sub>3</sub>: <em>pepT</em>; PAW: <em>iap</em>) and others being downregulated (O<sub>3</sub>: <em>srtA</em>, <em>LMOf2365_0284</em>, <em>LMOf2365_1945</em>, and <em>LMOf2365_0569</em>; PAW: <em>lspA</em>). In contrast, ClO<sub>2</sub> (<em>lmo1354</em>, <em>lmo1578</em>, <em>lmo1611</em>, <em>LMOf6854_0307</em>, and <em>LMOf6854_0278</em>) upregulated the expression of specific genes. As peptidases may play an active role in breaking down and utilizing proteins, the differential expression of genes encoding these enzymes following disinfection indicates that bacteria may be attempting to manage damaged or misfolded proteins under challenging conditions.</p></section></section><section id="crf370260-sec-0320"><h3 class="pmc_sec_title">4.4. Small Molecule Biosynthesis</h3>
<p>Synthesis of biomolecules is a multistage process involving enzymatic conversion or combination of simple compounds to produce macromolecules or more intricate products in living organisms (Abdel‐Aziz et al. <a href="#crf370260-bib-0002" class="usa-link" aria-describedby="crf370260-bib-0002">2017</a>). This process of small molecule biosynthesis is crucial for bacteria as it allows the synthesis of various compounds necessary for their survival, growth, and interactions with the environment. Bacteria synthesize many small molecules, including vitamins, minerals, cofactors, organic acids, ketones, and alcohols (Coupat et al. <a href="#crf370260-bib-0047" class="usa-link" aria-describedby="crf370260-bib-0047">2008</a>; França et al. <a href="#crf370260-bib-0061" class="usa-link" aria-describedby="crf370260-bib-0061">2014</a>). Disinfection treatments (H<sub>2</sub>O<sub>2</sub>, ClO<sub>2</sub>, O<sub>3</sub>, and PAW) affect various small molecule biosynthesis pathways, such as cobalamin, EA, and PD utilization, pantothenate, MK, molybdenum cofactor, and heme biosynthesis, and myo‐inositol catabolic pathways (Table <a href="#crf370260-supinfo-0001" class="usa-link">S4</a>; Supporting Information: Data <a href="#crf370260-supinfo-0001" class="usa-link">S2.4</a>).</p>
<section id="crf370260-sec-0330"><h4 class="pmc_sec_title">4.4.1. Cobalamin Biosynthesis and EA and PD Utilization</h4>
<p>Cobalamin (vitamin B12) serves as a cofactor for enzymes crucial for various metabolic processes, such as methylation reactions in amino acid metabolism (e.g., methionine synthase) (Matthews <a href="#crf370260-bib-0127" class="usa-link" aria-describedby="crf370260-bib-0127">2009</a>), rearrangements in carbon metabolism (e.g., methylmalonyl‐CoA mutase) (Roth et al. <a href="#crf370260-bib-0171" class="usa-link" aria-describedby="crf370260-bib-0171">1996</a>), reduction reactions in nucleotide synthesis (e.g., ribonucleotide reductase) (Booker and Stubbe <a href="#crf370260-bib-0020" class="usa-link" aria-describedby="crf370260-bib-0020">1993</a>), and fermentation reactions during anaerobic fermentation (e.g., ethanol to acetate) (Buckel and Thauer <a href="#crf370260-bib-0029" class="usa-link" aria-describedby="crf370260-bib-0029">2013</a>). Cobalamin biosynthesis is a complex and tightly regulated process involving approximately 30 enzyme‐mediated steps. It is also influenced by environmental factors, such as oxygen and cobalt availability (Raux et al. <a href="#crf370260-bib-0158" class="usa-link" aria-describedby="crf370260-bib-0158">2000</a>). Levels of cobalamin metabolism gene cluster <em>cob‐cbi</em> and EA and PD utilization operons (<em>eut</em> and <em>pdu</em>, respectively), which are part of CDGC, were upregulated by PAW. Levels of PD gene <em>pduX</em>, cobalamin biosynthesis genes <em>cbiD</em> and <em>cbiH</em>, and <em>eut</em> genes <em>eutB</em>, <em>eutC</em>, <em>eutE</em>, <em>eutH</em>, and <em>eutT</em> were also upregulated. In contrast, except for <em>cbiG</em> levels, <em>cobC</em>, <em>cobJ</em>, <em>cobK</em>, <em>cobA</em>, and <em>LMOf2365_1212</em> levels were downregulated by O<sub>3</sub>. The <em>eut</em> operon uses EA as the carbon and nitrogen source, whereas the <em>pdu</em> operon is involved in PD metabolism. The enzymes involved in these pathways rely on cofactors derived from cobalamin, which are produced de novo through <em>cob</em> and <em>cbi</em> (Anast et al. <a href="#crf370260-bib-0007" class="usa-link" aria-describedby="crf370260-bib-0007">2020</a>). CDGC activation enhances competitive fitness against commensal bacteria in the gastrointestinal tract and other environments. Sublethal exposure to specific disinfection treatments activates the CDGC signaling pathways, potentially increasing the expression levels of operons linked to host cell internalization and stress adaptation. This response further increases the virulence of persistent bacterial strains (Fernández‐Gómez et al. <a href="#crf370260-bib-0058" class="usa-link" aria-describedby="crf370260-bib-0058">2023</a>).</p></section><section id="crf370260-sec-0340"><h4 class="pmc_sec_title">4.4.2. Pantothenate Biosynthesis</h4>
<p>Pantothenate (vitamin B5) is a precursor of CoA, essential for many metabolic processes in bacteria (Leonardi and Jackowski <a href="#crf370260-bib-0110" class="usa-link" aria-describedby="crf370260-bib-0110">2007</a>). CoA is a vital cofactor for cell growth and integral to numerous metabolic processes, including phospholipid metabolism, FAS and fatty acid breakdown, nonribosomal peptide biosynthesis, and the TCA cycle (Spry et al. <a href="#crf370260-bib-0197" class="usa-link" aria-describedby="crf370260-bib-0197">2008</a>). Levels of <em>panB, the coax</em> gene required for pantothenate biosynthesis and CoA, are downregulated by O<sub>3</sub> treatment (Table <a href="#crf370260-supinfo-0001" class="usa-link">S3</a>). Pantothenate biosynthesis is linked to other metabolic pathways, including branched‐chain amino acid biosynthesis and nucleotide metabolism pathways (Leonardi and Jackowski <a href="#crf370260-bib-0110" class="usa-link" aria-describedby="crf370260-bib-0110">2007</a>; Primerano and Burns <a href="#crf370260-bib-0153" class="usa-link" aria-describedby="crf370260-bib-0153">1983</a>). Downregulation of several amino acid and nucleotide metabolism gene levels in <em>L. monocytogenes</em> by O<sub>3</sub> disinfection is correlated with the downregulation of the levels of genes involved in pantothenate and CoA biosynthesis (Tables <a href="#crf370260-supinfo-0001" class="usa-link">S2 and S3</a>). This suggests that the oxidizing agent O<sub>3</sub> inhibits numerous metabolic processes essential for the survival of <em>L. monocytogenes</em> through chain reactions.</p></section><section id="crf370260-sec-0350"><h4 class="pmc_sec_title">4.4.3. MK Biosynthesis</h4>
<p>MK (vitamin K2) plays a vital role in prokaryotes by facilitating electron transfer, ATP production, oxidative phosphorylation, and endospore formation (Das et al. <a href="#crf370260-bib-0049" class="usa-link" aria-describedby="crf370260-bib-0049">1989</a>; Hiratsuka et al. <a href="#crf370260-bib-0078" class="usa-link" aria-describedby="crf370260-bib-0078">2008</a>). MK is the primary component of the bacterial cell membrane and plays a crucial role in the aerobic and anaerobic respiration of many Gram‐positive bacteria. It facilitates the transfer of electrons between the enzymes, NADH dehydrogenase I and II, succinate dehydrogenase, cytochrome bc1–aa3 supercomplex, F<sub>0</sub>F<sub>1</sub> ATP synthase, cytochrome <em>bd</em> oxidase, nitrate, and fumarate reductase (Boersch et al. <a href="#crf370260-bib-0019" class="usa-link" aria-describedby="crf370260-bib-0019">2018</a>; Kurosu and Begari <a href="#crf370260-bib-0105" class="usa-link" aria-describedby="crf370260-bib-0105">2010</a>). The <em>L. monocytogenes</em> genes involved in MK biosynthesis, <em>menD</em> and <em>menH</em>, were upregulated by O<sub>3</sub>, whereas <em>menB</em> and <em>menG</em> were downregulated by O<sub>3</sub> and PAW, respectively. These mixed responses indicate complex regulation of the respiratory chain during oxidative stress. As MK plays a central role in redox balance, changes in its biosynthesis affect electron transport and cellular energy status. MK biosynthesis is crucial for the survival of Gram‐positive bacteria (Truglio et al. <a href="#crf370260-bib-0202" class="usa-link" aria-describedby="crf370260-bib-0202">2003</a>), and inhibition of MK production leads to bactericidal effects, irrespective of the growth phase (Boersch et al. <a href="#crf370260-bib-0019" class="usa-link" aria-describedby="crf370260-bib-0019">2018</a>).</p></section><section id="crf370260-sec-0360"><h4 class="pmc_sec_title">4.4.4. Molybdenum Cofactor Biosynthesis</h4>
<p>Mo is a trace element found in molybdoenzymes and plays a crucial role in essential cellular functions, including energy generation, detoxification reactions, and anaerobic fermentation (Zhong et al. <a href="#crf370260-bib-0235" class="usa-link" aria-describedby="crf370260-bib-0235">2020</a>). The molybdenum cofactor (Moco) biosynthesis pathway is conserved in both prokaryotes and eukaryotes (Leimkühler <a href="#crf370260-bib-0108" class="usa-link" aria-describedby="crf370260-bib-0108">2020</a>). In bacteria, the pathway consists of four main steps: (i) formation of cyclic pyranopterin monophosphate (cPMP) from GTP, (ii) insertion of two sulfur atoms into cPMP to form molybdopterin (MPT), (iii) insertion of molybdenum into MPT to form Moco, and (iv) additional modification of Moco to form dinucleotide variants (specific to bacteria) (Mendel and Leimkühler <a href="#crf370260-bib-0129" class="usa-link" aria-describedby="crf370260-bib-0129">2015</a>). This process involves the interactions of multiple proteins and requires iron, ATP, and copper. The key enzymes/proteins involved include MoaA and MoaC for cPMP formation; MoaD, MoaE, and MoeB for MPT synthesis; and MogA and MoeA for molybdenum insertion (Hasnat et al. <a href="#crf370260-bib-0075" class="usa-link" aria-describedby="crf370260-bib-0075">2024</a>). MPT synthase catalytic subunit <em>moeE</em> and MPT molybdenum transferase <em>moeA</em> were upregulated and downregulated by O<sub>3</sub>, respectively. Molybdoenzymes in bacteria are primarily regulated by anaerobiosis and require iron. The transcription factor fumarate and nitrate reduction (FNR) requires a [4Fe‐4S] cluster for binding (Hasnat et al. <a href="#crf370260-bib-0075" class="usa-link" aria-describedby="crf370260-bib-0075">2024</a>). MoaA contains a [4Fe‐4S] cluster and directly connects the Moco biosynthesis pathway to the formation of iron‐sulfur (Fe‐S) clusters (Yokoyama and Leimkühler <a href="#crf370260-bib-0225" class="usa-link" aria-describedby="crf370260-bib-0225">2015</a>). A feedback regulation mechanism controls the transcription of the <em>moa</em> operon genes. When MPT accumulates in the cells, it triggers a molybdenum cofactor riboswitch. This mechanism allows the bacteria to adapt to environmental changes (Regulski et al. <a href="#crf370260-bib-0160" class="usa-link" aria-describedby="crf370260-bib-0160">2008</a>). The increased <em>moeE</em> expression in <em>L. monocytogenes</em> in response to O<sub>3</sub> potentially enhances its tolerance to oxidative stress (Cai et al. <a href="#crf370260-bib-0032" class="usa-link" aria-describedby="crf370260-bib-0032">2023</a>).</p></section><section id="crf370260-sec-0370"><h4 class="pmc_sec_title">4.4.5. Heme Catabolism</h4>
<p>Heme is a complex ring composed of porphyrins and iron. It is a redox‐active component essential for the function of several cellular proteins (Choby and Skaar <a href="#crf370260-bib-0041" class="usa-link" aria-describedby="crf370260-bib-0041">2016</a>). Hemoproteins rely on heme and are involved in energy production and immune responses. Heme plays a crucial role in cellular respiration and acts as an electron carrier in the electron transport chain (Shimizu et al. <a href="#crf370260-bib-0187" class="usa-link" aria-describedby="crf370260-bib-0187">2019</a>). It is also essential for several cellular functions, including signaling, cellular differentiation, and miRNA processing (Choby and Skaar <a href="#crf370260-bib-0041" class="usa-link" aria-describedby="crf370260-bib-0041">2016</a>; Faller et al. <a href="#crf370260-bib-0055" class="usa-link" aria-describedby="crf370260-bib-0055">2007</a>). Levels of <em>lmo2133</em>, which encodes an isomer‐type heme‐degrading monooxygenase, are upregulated by ClO<sub>2</sub>. In contrast, levels of the <em>isdG</em> gene (iron‐containing steroidogenic enzyme) are downregulated by O<sub>3</sub>. Heme oxygenase catalyzes heme degradation to form biliverdin in the presence of an electron donor, releasing iron, which allows <em>L. monocytogenes</em> to utilize the host heme as an iron source (Duong et al. <a href="#crf370260-bib-0052" class="usa-link" aria-describedby="crf370260-bib-0052">2014</a>). Heme oxygenase plays crucial roles in maintaining iron homeostasis and protecting against oxidative stress by breaking down heme (Alavi et al. <a href="#crf370260-bib-0004" class="usa-link" aria-describedby="crf370260-bib-0004">2018</a>).</p></section></section><section id="crf370260-sec-0380"><h3 class="pmc_sec_title">4.5. Transcription and Translation</h3>
<p>Transcription is the fundamental process of copying genetic information from DNA to RNA, producing proteins essential for biological functions (Boyle <a href="#crf370260-bib-0023" class="usa-link" aria-describedby="crf370260-bib-0023">2005</a>). RNA polymerase (RNAP) is the key enzyme responsible for this process, which involves initiation, elongation, and termination. In bacteria, a single RNAP enzyme performs all transcriptional functions, tightly regulating the process to allow the cells to adapt to environmental changes and effectively manage gene expression (Wilson et al. <a href="#crf370260-bib-0219" class="usa-link" aria-describedby="crf370260-bib-0219">2018</a>). O<sub>3</sub> treatment of <em>L. monocytogenes</em> downregulated the levels of RNAP subunit <em>rpoB</em>, transcription factors <em>tsf</em> and <em>LMOf2365_1316</em>, and other components of the transcription machinery, including <em>trmL</em>, <em>rny</em>, and <em>codY</em> (Table <a href="#crf370260-supinfo-0001" class="usa-link">S5</a>).</p>
<p>Bacteria use mRNA transcribed from DNA to produce proteins through a process known as translation. This process occurs in the ribosome and involves several key steps, including initiation, elongation, termination, and recycling (Arenz and Wilson <a href="#crf370260-bib-0010" class="usa-link" aria-describedby="crf370260-bib-0010">2016</a>). Ribosomal biogenesis is an intricate process involving ribosome assembly that is crucial for protein synthesis (Davis and Williamson <a href="#crf370260-bib-0050" class="usa-link" aria-describedby="crf370260-bib-0050">2017</a>). Disinfection treatments differentially regulate the genes associated with ribosomal biogenesis and translation. O<sub>3</sub> and ClO<sub>2</sub> treatments differentially affect the expression levels of translational machinery genes in <em>L. monocytogenes</em>, upregulating some genes (O<sub>3</sub>: <em>rpmI</em>, <em>rpiB</em>, and <em>rnaJ</em>; ClO<sub>2</sub>: <em>rplY</em>, <em>def</em>, <em>mecA</em>, <em>groEL</em>, <em>groES</em>, <em>rsgA1</em>, <em>lmo2511</em>, <em>lmo1502</em>, <em>lmo1218</em>, and <em>lmo1703</em>) and downregulating others (O<sub>3</sub>: <em>rsmE</em>, <em>rpmG</em>, <em>rpsB</em>, and <em>mecA</em>; ClO<sub>2</sub>: <em>rplM</em>, <em>rpmF2</em>, <em>rpmG2</em>, <em>rpmH</em>, <em>rpsB</em>, <em>rpsJ</em>, <em>rsmG</em>, <em>bipA</em>, <em>ychF</em>, <em>gatA</em>, <em>lepA</em>, <em>metG</em>, and <em>mnmE</em>). In contrast, H<sub>2</sub>O<sub>2</sub> and PAW treatments downregulated the levels of specific genes (<em>rpsP</em> and <em>rnjB</em>, respectively).</p>
<p>Ribosomal biogenesis and translation are crucial processes ensuring the synthesis of proteins essential for the cellular functions and survival of bacteria. Roller et al. (<a href="#crf370260-bib-0167" class="usa-link" aria-describedby="crf370260-bib-0167">2016</a>) demonstrated the correlations between the number of ribosomal RNA operons (rrn) and growth rate and stability. They reported that, when the rrn copy number doubles, the maximum reproductive rate of bacteria also doubles, with the rate and yield of protein production reflecting the general trend in maximum growth rate and efficiency (Roller et al. <a href="#crf370260-bib-0167" class="usa-link" aria-describedby="crf370260-bib-0167">2016</a>). Increased expression levels of ribosomal genes in <em>L. monocytogenes</em> following disinfection lead to enhanced ribosome production. This is crucial for protein synthesis and cellular repair, enabling the pathogen to rapidly produce proteins vital for stress responses. Transcriptional changes in ribosomal genes possibly reflect the cellular responses to stress, including protein synthesis adjustments, to support <em>L. monocytogenes</em> survival and repair under disinfection stress.</p></section><section id="crf370260-sec-0390"><h3 class="pmc_sec_title">4.6. Transport Genes</h3>
<p>
<em>L. monocytogenes</em> harbors several transporters playing crucial roles in various cellular processes and facilitating its adaptation to different environments, thereby contributing to its pathogenicity and survival. The major transporters affected by disinfection treatments include multidrug efflux transporters, phosphoenolpyruvate‐dependent PTS, metal ion transporters, and amino acid and protein transporters. Among these transporters, several genes belonging to the PTS system were differentially expressed in response to the reviewed disinfection treatments (Table <a href="#crf370260-supinfo-0001" class="usa-link">S6</a>; Supporting Information: Data <a href="#crf370260-supinfo-0001" class="usa-link">S2.6</a>).</p>
<section id="crf370260-sec-0400"><h4 class="pmc_sec_title">4.6.1. Phosphotransferase System</h4>
<p>PTS, a sugar‐specific transport system that simultaneously phosphorylates and transports various sugars into the cell, is crucial for carbohydrate transport and metabolism (Casey et al. <a href="#crf370260-bib-0035" class="usa-link" aria-describedby="crf370260-bib-0035">2014</a>). Levels of <em>lmo0781</em>, <em>lmo0915</em>, <em>lmo1255</em>, <em>lmo2000</em>, <em>lmo2373</em>, <em>lmo2650</em>, and <em>lmo1254</em>, and several other genes belonging to the IIA component of the mannose/fructose/sorbose family PTS are upregulated by ClO<sub>2</sub> treatment. In contrast, H<sub>2</sub>O<sub>2</sub> (<em>bglC</em>, <em>bglH</em>, and <em>lmo0785</em>) and O<sub>3</sub> (<em>celB</em>, <em>LMOf2365_0537</em>, <em>LMOf2365_0660</em>, <em>LMOf2365_0798</em>, <em>LMOf2365_0937</em>, <em>LMOf2365_2023</em>, <em>LMOf2365_2344</em>, <em>LMOf2365_2621</em>, <em>LMOf2365_2664</em>, and <em>LMOf2365_2774</em>) reduce the expression levels of several PTS system‐related genes. PAW treatment decreases the expression levels of PTS‐type galactitol (<em>gatA</em>, <em>gatB</em>, and <em>gatC</em>), cellobiose (<em>catA</em>, <em>catB</em>, and <em>catC</em>), and mannose (<em>manY</em> and <em>manR</em>) genes. Carbon sources fructose, galactitol (sugar alcohol derived from galactose hydrogenation), and cellobiose (disaccharide produced by cellulose hydrolysis) are essential precursors for cell wall biosynthesis in <em>L. monocytogenes</em> (Popowska et al. <a href="#crf370260-bib-0150" class="usa-link" aria-describedby="crf370260-bib-0150">2012</a>). PTS downregulation under PAW treatment restricts the absorption and use of these carbohydrates, allowing the pathogen to conserve energy and resources for other stress response mechanisms.</p></section><section id="crf370260-sec-0410"><h4 class="pmc_sec_title">4.6.2. Multidrug Efflux Transporters</h4>
<p>Bacterial multidrug efflux transporters contribute to the ability of bacteria to resist and expel toxic compounds, including disinfectants, antibiotics, and heavy metals. Tolerance to quaternary ammonium compounds (QACs) and antibiotics mediated by multidrug efflux transporters has been reported in <em>L. monocytogenes</em> (Mata et al. <a href="#crf370260-bib-0126" class="usa-link" aria-describedby="crf370260-bib-0126">2000</a>; Romanova et al. <a href="#crf370260-bib-0169" class="usa-link" aria-describedby="crf370260-bib-0169">2006</a>). Levels of <em>ebrA</em> and <em>qacC</em> are upregulated in response to PAW treatment. QacC is a small multidrug resistance family efflux transporter that is a part of the <em>bcrABC</em> cassette, which primarily confers tolerance to QACs in <em>L. monocytogenes</em> (He et al. <a href="#crf370260-bib-0076" class="usa-link" aria-describedby="crf370260-bib-0076">2022</a>). Upregulation of multidrug efflux transporter gene (<em>qacH</em>, <em>qacC</em>, and <em>ebrA</em>) levels by BZT indicates an adaptive mechanism to actively expel the disinfectant molecules from bacterial cells, thereby reducing their intracellular accumulation and toxicity.</p></section><section id="crf370260-sec-0420"><h4 class="pmc_sec_title">4.6.3. Metal Ion Transporters</h4>
<p>Metal ion transporters in bacteria play crucial roles in various cellular processes, including the maintenance of intracellular metal ion homeostasis, regulation of gene expression, response to oxidative stress, and virulence (Abrantes et al. <a href="#crf370260-bib-0003" class="usa-link" aria-describedby="crf370260-bib-0003">2011</a>). Both <em>mntB</em> and <em>mntR</em> levels are upregulated in response to PAW treatment. H<sub>2</sub>O<sub>2</sub> downregulates the expression of <em>mntH</em>. Manganese is involved in ROS detoxification and is essential for the oxidative stress responses in bacteria (Juttukonda and Skaar <a href="#crf370260-bib-0093" class="usa-link" aria-describedby="crf370260-bib-0093">2015</a>). Mn transporters regulate the levels of Mn inside cells and shield the bacteria from oxidative stress (Jakubovics and Jenkinson <a href="#crf370260-bib-0085" class="usa-link" aria-describedby="crf370260-bib-0085">2001</a>). The increased expression of manganese transporter genes in <em>L. monocytogenes</em> following PAW treatment indicates enhanced manganese uptake, which is crucial for the oxidative stress response, acid tolerance, and other stress adaptation mechanisms <em>in L. monocytogenes</em> (Juttukonda and Skaar <a href="#crf370260-bib-0093" class="usa-link" aria-describedby="crf370260-bib-0093">2015</a>).</p></section><section id="crf370260-sec-0430"><h4 class="pmc_sec_title">4.6.4. Amino Acid and Protein Transporters</h4>
<p>Bacteria use different transport mechanisms to uptake amino acids and oligopeptides. These include ATP‐binding cassette (ABC) transporters, which are divided into subfamilies, such as polar and hydrophobic amino acid transport families (Hosie and Poole <a href="#crf370260-bib-0079" class="usa-link" aria-describedby="crf370260-bib-0079">2001</a>), and sodium‐dependent and independent systems (Landick et al. <a href="#crf370260-bib-0106" class="usa-link" aria-describedby="crf370260-bib-0106">1985</a>). ABC transporters use ATP hydrolysis to transport solutes across cellular membranes. Recent studies have identified six transporters facilitating the uptake of amino acids and oligopeptides in <em>L. monocytogenes</em>, thereby aiding the survival and replication of this bacterium (Joseph et al. <a href="#crf370260-bib-0091" class="usa-link" aria-describedby="crf370260-bib-0091">2006</a>; Schauer et al. <a href="#crf370260-bib-0180" class="usa-link" aria-describedby="crf370260-bib-0180">2010</a>). In <em>L. monocytogenes</em>, H<sub>2</sub>O<sub>2</sub> increases the expression levels of <em>bilEA</em>, <em>bilEB</em>, and <em>pbuX</em>. Simultaneously, <em>secA1</em> and <em>gadC</em> levels were elevated by ClO<sub>2</sub> and PAW, respectively. Levels of <em>yrhG</em> and <em>dtpT</em> decreased in response to H<sub>2</sub>O<sub>2</sub> and PAW, respectively. <em>L. monocytogenes</em> BilE protein, encoded by <em>bilEA</em> and <em>bilEB</em>, resembles an ABC importer, and contributes to virulence and stress responses by transporting QACs (Ruiz et al. <a href="#crf370260-bib-0174" class="usa-link" aria-describedby="crf370260-bib-0174">2016</a>). PAW treatment decreases the pH levels, indicating increased acidity due to the reactive oxygen and nitrogen species interacting with water molecules to generate protons (H+) (Jirešová et al. <a href="#crf370260-bib-0088" class="usa-link" aria-describedby="crf370260-bib-0088">2022</a>). The <em>gadC</em> gene encodes a glutamate/GABA antiporter in bacteria, which is essential for the glutamate‐dependent acid resistance system (Tsai et al. <a href="#crf370260-bib-0203" class="usa-link" aria-describedby="crf370260-bib-0203">2013</a>). GadC promotes the exchange of deprotonated glutamate (Glu<sup>–</sup>) for protonated GABA<sup>+</sup> across the cell membrane, effectively expelling protons and preventing their influx into the cell. Through the upregulation of <em>gadC</em> levels by PAW, <em>L. monocytogenes</em> enhances its ability to thrive under acidic conditions by maintaining an intracellular pH balance.</p></section></section><section id="crf370260-sec-0440"><h3 class="pmc_sec_title">4.7. Stress Responses</h3>
<p>Stress refers to the genomic, proteomic, and environmental changes reducing the growth rate and survival of an organism (Vorob'eva <a href="#crf370260-bib-0209" class="usa-link" aria-describedby="crf370260-bib-0209">2004</a>). Sensing and responding to environmental signals are crucial for the adaptability and survival of bacterial pathogens (Guerreiro et al. <a href="#crf370260-bib-0071" class="usa-link" aria-describedby="crf370260-bib-0071">2020</a>). Foodborne pathogens adjust their cellular processes to survive and grow under stress conditions (Sibanda and Buys <a href="#crf370260-bib-0189" class="usa-link" aria-describedby="crf370260-bib-0189">2022</a>). <em>L. monocytogenes</em> encounters various physical and chemical challenges impeding its growth and survival throughout the food chain. This pathogen is primarily known for its ability to persist and thrive in food production environments (Sibanda and Buys <a href="#crf370260-bib-0189" class="usa-link" aria-describedby="crf370260-bib-0189">2022</a>). Notably, enhanced QAC tolerance is observed in <em>L. monocytogenes</em> isolates persisting for over 4 years in such environments (Aase et al. <a href="#crf370260-bib-0001" class="usa-link" aria-describedby="crf370260-bib-0001">2000</a>). Many genes involved in stress responses, including those encoding the general stress response proteins, cold shock proteins (Csps), and heat shock proteins (Hsps), and those related to temperature, osmotic, cell envelope stress, oxidative stress, antibiotics, radiation, DNA damage, signal peptide processing, protein quality control, and homeostasis, were differentially regulated by the reviewed disinfection treatments (Table <a href="#crf370260-supinfo-0001" class="usa-link">S7</a>; Supporting Information: Data <a href="#crf370260-supinfo-0001" class="usa-link">S2.7</a>).</p>
<section id="crf370260-sec-0450"><h4 class="pmc_sec_title">4.7.1. General Stress Responses</h4>
<p>General stress response proteins comprise a group of proteins that allow the bacteria to withstand various types of stress, including oxidative, acidic, and osmotic stress. Stressosome is a large bacterial protein complex that detects environmental stress and activates a general stress response (Martinez‐Bond et al. <a href="#crf370260-bib-0125" class="usa-link" aria-describedby="crf370260-bib-0125">2025</a>). Upon stressosome activation, RsbT is released, which activates σB, a crucial regulator of the stress response managing over 300 genes related to stress resilience and virulence (Guerreiro et al. <a href="#crf370260-bib-0071" class="usa-link" aria-describedby="crf370260-bib-0071">2020</a>). O<sub>3</sub> treatment reduces <em>prli42</em> gene expression, which encodes the stressosome‐associated protein Prli42, in <em>L. monocytogenes</em>. This membrane‐anchored miniprotein is crucial for σB activation and essential for bacterial growth and survival under oxidative stress. Prli42 conveys the environmental stress signals to the stressomes (Impens et al. <a href="#crf370260-bib-0081" class="usa-link" aria-describedby="crf370260-bib-0081">2017</a>).</p></section><section id="crf370260-sec-0460"><h4 class="pmc_sec_title">4.7.2. Temperature Stress Responses</h4>
<p>Disinfection treatments regulate several cold‐ and heat‐shock proteins in different ways. Csps are highly conserved small nucleic acid‐binding proteins involved in stress tolerance mechanisms (Muchaamba et al. <a href="#crf370260-bib-0135" class="usa-link" aria-describedby="crf370260-bib-0135">2021</a>). Csp genes <em>cspA, cspB</em>, and <em>cspD</em> are induced upon exposure to various stresses, including cold, osmotic, oxidative, and desiccation. Csp gene levels in <em>L. monocytogenes</em> are upregulated in response to ClO<sub>2</sub> (<em>cspB</em>, <em>cspD</em>, and <em>ltrC</em>) and O<sub>3</sub> (<em>cspD</em>) treatment. Csps possibly counteract the harmful effects of stress by preserving cellular functions, such as DNA replication, transcription, and translation, thereby ensuring cell survival and growth (B. Schmid et al. <a href="#crf370260-bib-0181" class="usa-link" aria-describedby="crf370260-bib-0181">2009</a>). Csps also play a role in biofilm formation and motility, which are critical for the persistence and transmission of <em>L. monocytogenes</em> in food‐processing environments (Eshwar et al. <a href="#crf370260-bib-0054" class="usa-link" aria-describedby="crf370260-bib-0054">2017</a>). Similarly, heat shock response is a crucial process allowing the bacteria to survive and adapt to harsh environmental conditions. In response to temperature changes, heat shock response triggers the production of several proteins, known as Hsps (Maleki et al. <a href="#crf370260-bib-0121" class="usa-link" aria-describedby="crf370260-bib-0121">2016</a>). Hsps primarily assist in proper protein folding and participate in the repair or degradation of misfolded proteins and harmful aggregates. When temperatures suddenly increase, Hsps rapidly accumulate in the cells. After the organism adapts, these protein levels decrease but remain elevated above their original baseline, facilitating bacterial growth even under suboptimal conditions (Roncarati and Scarlato <a href="#crf370260-bib-0170" class="usa-link" aria-describedby="crf370260-bib-0170">2017</a>). ClO<sub>2</sub> increases the expression levels of Hsp‐related genes <em>dnaK</em> and <em>grpE</em>. Upregulation of Csp and Hsp levels in <em>L. monocytogenes</em> represents a stress adaptation mechanism that enhances the ability of the pathogen to survive and grow under the specific treatment conditions (B. Schmid et al. <a href="#crf370260-bib-0181" class="usa-link" aria-describedby="crf370260-bib-0181">2009</a>).</p></section><section id="crf370260-sec-0470"><h4 class="pmc_sec_title">4.7.3. Oxidative Stress Responses</h4>
<p>Disinfectants often generate ROS and activate genes to combat oxidative stress, such as superoxide dismutase (SOD) and catalase. Increased expression of <em>sigB</em>‐dependent genes by oxidizing agents H<sub>2</sub>O<sub>2,</sub> O<sub>3,</sub> and ClO<sub>2</sub> enables <em>L. monocytogenes</em> to activate genes protecting it from acid and oxidative stress. For instance, ClO<sub>2</sub> exposure increases the expression levels of oxidative stress response genes <em>sodA</em>, <em>ohrA</em>, <em>spxA</em>, <em>spxH</em>, <em>lmo0906</em>, and <em>lmo0983</em> in <em>L. monocytogenes</em>, enabling it to combat oxidative stress. Other disinfection treatments, such as H<sub>2</sub>O<sub>2</sub> (<em>ohrR</em>, <em>ohrA</em>, <em>sodA</em>, and <em>gpo</em>) and O<sub>3</sub> (<em>sodA</em>), also increase the levels of oxidative response genes. Levels of the <em>sodA</em> gene, which encodes SOD, are consistently upregulated by all oxidizing agents. SOD is an antioxidant enzyme playing an essential role in protecting cells from oxidative damage (Zheng et al. <a href="#crf370260-bib-0234" class="usa-link" aria-describedby="crf370260-bib-0234">2023</a>). SOD catalyzes the conversion of superoxide (O<sub>2</sub>•−) into oxygen and H<sub>2</sub>O<sub>2</sub>, transforming the highly reactive superoxide radical into less harmful substances (Manso et al. <a href="#crf370260-bib-0123" class="usa-link" aria-describedby="crf370260-bib-0123">2020</a>). The increased expression of oxidative stress response genes enables <em>L. monocytogenes</em> to counteract the detrimental effects of oxidative damage caused by ROS (Bucur et al. <a href="#crf370260-bib-0030" class="usa-link" aria-describedby="crf370260-bib-0030">2018</a>) from disinfectants, such as H<sub>2</sub>O<sub>2</sub>, O<sub>3</sub>, and ClO<sub>2</sub>, thereby improving its survival under oxidative stress.</p></section><section id="crf370260-sec-0480"><h4 class="pmc_sec_title">4.7.4. Responses to Toxic Substances</h4>
<p>Bacteria can adapt to chemical stress induced by toxic substances. Excessive or improper disinfectant application decreases their susceptibility and increases their tolerance (Rozman et al. <a href="#crf370260-bib-0173" class="usa-link" aria-describedby="crf370260-bib-0173">2021</a>; van Dijk and Verbrugh <a href="#crf370260-bib-0205" class="usa-link" aria-describedby="crf370260-bib-0205">2022</a>). Bacterial regulatory networks typically exhibit distinct changes in response to physical or chemical stresses, such as insufficient nutrient availability, elevated temperatures, disinfectants, and antibiotics (LeRoux et al. <a href="#crf370260-bib-0111" class="usa-link" aria-describedby="crf370260-bib-0111">2015</a>). Danger sensing is the ability to identify external substances posing a threat to the organism. Levels of phosphate regulator gene <em>phoU</em> (ClO<sub>2</sub>), arsenic resistance gene <em>lmo2230</em> (H<sub>2</sub>O<sub>2</sub> and ClO<sub>2</sub>), and toxin resistance genes <em>lmo2570</em> (ClO<sub>2</sub>) and <em>lmo1708</em> (ClO<sub>2</sub>) are upregulated in <em>L. monocytogenes</em> in response to disinfection. In contrast, PAW downregulates the <em>phoU</em> and toxin resistance gene <em>telA1</em> levels. The two‐component regulatory system is crucial for signal transduction, allowing bacteria to sense and respond to changes in various environmental conditions (Casado et al. <a href="#crf370260-bib-0034" class="usa-link" aria-describedby="crf370260-bib-0034">2022</a>). In bacteria, the two‐component regulatory system governs the phosphate (Pho) regulon, with PhoU as its key negative regulator (Santos‐Beneit <a href="#crf370260-bib-0177" class="usa-link" aria-describedby="crf370260-bib-0177">2015</a>). PhoU is a crucial regulator of persister cell development. Its inactivation leads to the loss of persisters, resulting in increased sensitivity to antibiotics and stress (Y. Li and Zhang <a href="#crf370260-bib-0113" class="usa-link" aria-describedby="crf370260-bib-0113">2007</a>). <em>Escherichia coli</em> PhoU mutant cannot suppress the metabolic processes necessary for persister formation, thereby showing increased susceptibility (Y. Li and Zhang <a href="#crf370260-bib-0113" class="usa-link" aria-describedby="crf370260-bib-0113">2007</a>).</p></section><section id="crf370260-sec-0490"><h4 class="pmc_sec_title">4.7.5. DNA Damage (DNA Repair) Responses</h4>
<p>Bacteria respond to agents causing DNA damage by activating the SOS response, which contributes to DNA repair (Arcari et al. <a href="#crf370260-bib-0009" class="usa-link" aria-describedby="crf370260-bib-0009">2020</a>). Disinfection treatments H<sub>2</sub>O<sub>2</sub> (<em>dinB</em>, <em>uvrA</em>, <em>uvrB</em>, <em>recA</em>, <em>lexA</em>, <em>exoA</em>, <em>addA</em>, <em>addB</em>, <em>lmo0157</em>, <em>lmo2222</em>, <em>lmo2050</em>, and <em>lmo1639</em>) and ClO<sub>2</sub> (<em>fri</em>, <em>deoC</em>, and <em>lmo1514</em>). RecA plays a crucial role in DNA repair and acts as a key activator of the SOS response in <em>L. monocytogenes</em> (Van Der Veen et al. <a href="#crf370260-bib-0204" class="usa-link" aria-describedby="crf370260-bib-0204">2010</a>). RecA filaments bind to single‐stranded DNA, triggering the self‐cleavage of LexA, an SOS regulon repressor. Cleaved LexA is further processed by ClpXP bacterial proteases, which activate the SOS pathway (Cohn et al. <a href="#crf370260-bib-0044" class="usa-link" aria-describedby="crf370260-bib-0044">2011</a>). This leads to a temporary halt in cell division, allowing DNA repair (Walker <a href="#crf370260-bib-0210" class="usa-link" aria-describedby="crf370260-bib-0210">1996</a>). Nucleotide excision repair (NER) is another highly conserved multistep DNA recognition and repair process mediated by the UvrABC repair system (Truglio et al. <a href="#crf370260-bib-0201" class="usa-link" aria-describedby="crf370260-bib-0201">2004</a>). UvrA detects damage and forms the heterotrimeric complex, UvrA2UvrB, with UvrB (Theis et al. <a href="#crf370260-bib-0199" class="usa-link" aria-describedby="crf370260-bib-0199">2000</a>). UvrB is a central component of the bacterial NER system, participating in damage recognition, strand excision, and repair synthesis (Verhoeven et al. <a href="#crf370260-bib-0207" class="usa-link" aria-describedby="crf370260-bib-0207">2002</a>). LexA and DnaA proteins regulate the transcription of <em>uvrB</em> (NER system), <em>dinJ </em>(antitoxin component of a type II toxin‐antitoxin system), and <em>recN</em> (DNA double‐strand break repair) in <em>E. coli</em>, thereby influencing the SOS regulon (Wurihan et al. <a href="#crf370260-bib-0222" class="usa-link" aria-describedby="crf370260-bib-0222">2018</a>). SOS response of <em>L. monocytogenes</em> plays a role in its survival under various stressors, further contributing to its persistence in the environment (Van Der Veen et al. <a href="#crf370260-bib-0204" class="usa-link" aria-describedby="crf370260-bib-0204">2010</a>).</p></section><section id="crf370260-sec-0500"><h4 class="pmc_sec_title">4.7.6. Protein Quality Control and Homeostasis</h4>
<p>In addition to inducing DNA damage, disinfection stress leads to the accumulation of damaged proteins, which must be eliminated by the cellular proteolytic systems. <em>L. monocytogenes</em> uses stress regulators and response genes to manage oxidative stress. Most stress response genes are regulated by SigB factors, including <em>ctsR</em>, which represses the caseinolytic protease (clp) operon under optimal conditions. CLP proteins, including both chaperones and proteases, are activated as a survival strategy under temperature and stress conditions (Manso et al. <a href="#crf370260-bib-0123" class="usa-link" aria-describedby="crf370260-bib-0123">2020</a>). Clp ATPases are a large family of related proteins playing key housekeeping roles, including in protein reactivation, remodeling, and targeting of specific proteins for degradation by ClpP peptidase (Lemos and Burne <a href="#crf370260-bib-0109" class="usa-link" aria-describedby="crf370260-bib-0109">2002</a>). ClO<sub>2</sub> (<em>clpB</em>, <em>clpC</em>, <em>clpP</em>, <em>clpX</em>, and <em>ctsR</em>) and PAW (<em>clpP</em>) increase the expression levels of Clp protease genes. ClpP is a tetradecameric peptidase degrading misfolded proteins aided by chaperones, such as ClpC and ClpX (Baker and Sauer <a href="#crf370260-bib-0011" class="usa-link" aria-describedby="crf370260-bib-0011">2012</a>). The ClpXP complex plays crucial roles in protein quality control, SOS responses, and virulence in <em>L. monocytogenes</em> (Balogh et al. <a href="#crf370260-bib-0012" class="usa-link" aria-describedby="crf370260-bib-0012">2022</a>). ClpX, a hexameric ATPase associated with diverse cellular activities (AAA+) and ATP‐driven chaperone, identifies protein substrates and channels unfolded peptide chains into the tetradecameric barrel of the ClpP serine protease for breakdown (Baker and Sauer <a href="#crf370260-bib-0011" class="usa-link" aria-describedby="crf370260-bib-0011">2012</a>). <em>L. monocytogenes</em> possesses two ClpP isoforms, ClpP1 and ClpP2, which form a heterooligomeric complex with enhanced proteolytic activity. ClpP1 enhances degradation efficiency, whereas ClpP2 facilitates ClpX binding and functions as a substrate gatekeeper (Balogh et al. <a href="#crf370260-bib-0012" class="usa-link" aria-describedby="crf370260-bib-0012">2022</a>). Regulation of <em>clp</em> genes is mediated by CtsR, which represses stress response gene expression (Wiktorczyk‐Kapischke et al. <a href="#crf370260-bib-0218" class="usa-link" aria-describedby="crf370260-bib-0218">2021</a>).</p></section></section><section id="crf370260-sec-0510"><h3 class="pmc_sec_title">4.8. Chemotaxis and Motility</h3>
<p>Bacterial chemotaxis is a phenomenon by which bacteria sense changes in the chemical composition of their environment via chemoreceptors and migrate toward a favorable environment (Bren and Eisenbach <a href="#crf370260-bib-0025" class="usa-link" aria-describedby="crf370260-bib-0025">2000</a>). Motility is a crucial feature allowing the bacteria to search for nutrients and avoid toxic compounds. Bacteria have to adapt to various stressors, including antibiotics and disinfectants, to survive (Piskovsky and Oliveira <a href="#crf370260-bib-0146" class="usa-link" aria-describedby="crf370260-bib-0146">2023</a>). Particularly, disinfection significantly alters the regulation of several key genes involved in chemotaxis and flagellar biosynthesis (Table <a href="#crf370260-supinfo-0001" class="usa-link">S8</a>; Supporting Information: Data <a href="#crf370260-supinfo-0001" class="usa-link">S2.8</a>).</p>
<section id="crf370260-sec-0520"><h4 class="pmc_sec_title">4.8.1. Chemotaxis</h4>
<p>Chemotaxis plays vital roles in various biological processes, including stress responses, biofilm formation, QS, bacterial pathogenesis, and host infection (Karmakar <a href="#crf370260-bib-0097" class="usa-link" aria-describedby="crf370260-bib-0097">2021</a>). Methyltransferase enzyme CheR plays a crucial role in chemotaxis, enabling bacteria to sense and respond to their environment by modifying the chemotaxis receptors. It regulates the chemotaxis pathway by modulating the methylation state of methyl‐accepting chemotaxis proteins in response to chemotactic gradients. O<sub>3</sub> and PAW treatments increase the expression levels of <em>cheR</em> in <em>L. monocytogenes</em>. Upregulation of chemotactic gene levels by O<sub>3</sub> and PAW treatment helps <em>L. monocytogenes</em> to navigate toward more favorable environments for survival. In contrast, ClO<sub>2</sub> treatment decreases the expression levels of chemotactic genes <em>lmo1699</em> and <em>lmo0693</em> in <em>L. monocytogenes</em>, limiting its potential to sense and respond to the stress caused by these treatments.</p></section><section id="crf370260-sec-0530"><h4 class="pmc_sec_title">4.8.2. Bacterial Flagellum Biogenesis</h4>
<p>Flagella are motile chemotactic devices facilitating bacterial survival, adaptation, and pathogenesis (O'Neil and Marquis <a href="#crf370260-bib-0143" class="usa-link" aria-describedby="crf370260-bib-0143">2006</a>). O<sub>3</sub> (<em>flgE</em> and <em>mogR</em>) and PAW (<em>fliH</em>, <em>flgE</em>, and <em>fliI</em>) treatments increase the expression levels of genes responsible for flagellar assembly. Conversely, ClO<sub>2</sub> (<em>flgD</em>, <em>fliK</em>, and <em>fliQ</em>) and H<sub>2</sub>O<sub>2</sub> (<em>fliP</em>) treatments decrease the expression levels of flagellar genes. O<sub>3</sub> treatment also suppresses the expression of specific genes (<em>motB</em>, <em>fliN</em>, <em>flgB</em>, <em>fliE</em>, and <em>fliF</em>) in <em>L. monocytogenes</em>. Many chemotaxis genes, such as <em>cheA</em>, <em>cheY</em>, and methyl‐accepting chemotaxis protein genes, coordinate with the flagellar assembly genes to support bacterial motility. Y. Li and Zhang (<a href="#crf370260-bib-0113" class="usa-link" aria-describedby="crf370260-bib-0113">2007</a>) identified a correlation between the increased expression levels of several genes related to flagella and chemotaxis and the absence of the negative regulator, PhoU, in <em>E. coli</em>. Consistently, downregulation of <em>phoU</em> levels by PAW treatment increased the expression levels of chemotaxis and flagellar genes. Conversely, upregulation of <em>phoU</em> genes by ClO<sub>2</sub> decreased these gene levels. These patterns suggest regulatory interactions between PhoU and motility genes that affect bacterial adaptation and energy expenditure under stress. Y. Li and Zhang (<a href="#crf370260-bib-0113" class="usa-link" aria-describedby="crf370260-bib-0113">2007</a>) further proposed that the loss of the negative regulator PhoU increases cellular metabolic activity, thereby enhancing susceptibility to various antibiotics and stressors.</p>
<p>Motility and flagellar gene expression levels in <em>L. monocytogenes</em> are strongly regulated by temperature. These gene levels are generally upregulated at environmental temperatures (≤ 30°C) and repressed at 37°C by the transcriptional repressor MogR (Ivy et al. <a href="#crf370260-bib-0084" class="usa-link" aria-describedby="crf370260-bib-0084">2010</a>; Shen and Higgins <a href="#crf370260-bib-0186" class="usa-link" aria-describedby="crf370260-bib-0186">2006</a>). This repression is halted at low temperatures by the anti‐repressor, GmaR, which acts as a thermosensor but becomes inactive or aggregates at high temperatures (Kamp and Higgins <a href="#crf370260-bib-0095" class="usa-link" aria-describedby="crf370260-bib-0095">2009</a>, <a href="#crf370260-bib-0096" class="usa-link" aria-describedby="crf370260-bib-0096">2011</a>). In this review, we compared disinfection treatments across various temperatures, ranging from 20°C (H<sub>2</sub>O<sub>2</sub>) to 37°C (ClO<sub>2</sub>). Therefore, the variations in flagellar gene expression patterns indicated here reflect both the disinfection treatment and baseline regulatory influence of the incubation temperature.</p></section></section><section id="crf370260-sec-0540"><h3 class="pmc_sec_title">4.9. QS and Virulence‐Related Genes</h3>
<p>QS is a bacterial signaling mechanism enabling cell–cell communication within and between species by producing, detecting, and responding to the extracellular signaling molecules, known as autoinducers (AIs) (Ismail et al. <a href="#crf370260-bib-0083" class="usa-link" aria-describedby="crf370260-bib-0083">2016</a>). This mechanism enables bacteria to regulate their gene expression, metabolic activity, and behaviors in response to variations in population density, thereby inducing biofilm formation (Preda and Săndulescu <a href="#crf370260-bib-0152" class="usa-link" aria-describedby="crf370260-bib-0152">2019</a>; Rutherford and Bassler <a href="#crf370260-bib-0175" class="usa-link" aria-describedby="crf370260-bib-0175">2012</a>). Expression levels of virulence factors, such as adhesins, internalins, surface proteins (e.g., ActA), siderophores, hemolysins (e.g., listeriolysin O), phospholipases, and toxins, and survival within host cells significantly influence the severity of <em>L. monocytogenes</em> infection (Coelho et al. <a href="#crf370260-bib-0043" class="usa-link" aria-describedby="crf370260-bib-0043">2018</a>). Disinfection procedures influence multiple QS‐ and virulence‐related genes (Table <a href="#crf370260-supinfo-0001" class="usa-link">S9</a>; Supporting Information: Data <a href="#crf370260-supinfo-0001" class="usa-link">S2.9</a>), indicating their potential effects on bacterial communication and pathogenic traits.</p>
<section id="crf370260-sec-0550"><h4 class="pmc_sec_title">4.9.1. Quorum Sensing</h4>
<p>QS is an intercellular communication mechanism regulating gene expression across cellular processes (Waters and Bassler <a href="#crf370260-bib-0216" class="usa-link" aria-describedby="crf370260-bib-0216">2005</a>). It is essential for bacterial adaptation and survival under different environmental conditions during infection, processing, and transmission (Banerji et al. <a href="#crf370260-bib-0013" class="usa-link" aria-describedby="crf370260-bib-0013">2022</a>). PAW increases the levels of accessory gene regulator (<em>agr</em>)‐<em>B</em>, whereas ClO<sub>2</sub> upregulates the levels of <em>luxS</em>, which synthesizes AI‐2. The <em>agr</em> system is crucial for both biofilm formation and virulence and facilitates adaptation to changing environments (Zetzmann et al. <a href="#crf370260-bib-0228" class="usa-link" aria-describedby="crf370260-bib-0228">2016</a>). In <em>L. monocytogenes</em>, <em>agrB</em> is essential for processing the precursor peptide from <em>agrD</em> to the mature autoinducing peptide (Rieu et al. <a href="#crf370260-bib-0162" class="usa-link" aria-describedby="crf370260-bib-0162">2007</a>). Autoinducing peptide activates histidine kinase AgrC, which phosphorylates and activates response regulator AgrA, thereby regulating the genes associated with biofilm formation and virulence (Riedel et al. <a href="#crf370260-bib-0161" class="usa-link" aria-describedby="crf370260-bib-0161">2009</a>). The <em>agrBDCA</em> operon is autoregulated, exhibiting higher expression levels during biofilm growth than during the planktonic or early attachment phases (Zetzmann et al. <a href="#crf370260-bib-0228" class="usa-link" aria-describedby="crf370260-bib-0228">2016</a>). <em>luxS</em> encodes the LuxS enzyme, which is involved in the synthesis of AI‐2, a universal signaling molecule sensed by various bacterial species, facilitating interspecies communication. Communication enables bacteria to coordinate behaviors, such as biofilm formation, virulence, and antibiotic resistance (Wang et al. <a href="#crf370260-bib-0214" class="usa-link" aria-describedby="crf370260-bib-0214">2019</a>). PAW and ClO<sub>2</sub> treatments upregulate the QS gene (<em>agrB</em> and <em>luxS</em>) levels, enhancing biofilm formation. This biofilm protects <em>L. monocytogenes</em> from disinfectants by forming a physical barrier and modifying the internal microenvironment.</p></section><section id="crf370260-sec-0560"><h4 class="pmc_sec_title">4.9.2. Virulence‐Related Genes</h4>
<p>The <em>prfA</em> virulence gene cluster (pVGC) is the primary pathogenicity island in <em>L. monocytogenes</em>, comprising <em>prfA</em>, <em>plcA</em>, <em>hly</em>, <em>mpl</em>, <em>actA</em>, and <em>plcB</em>. Levels of pVGC and other virulence factor genes are decreased by O<sub>3</sub> (<em>comGB</em>, ESAT‐6‐like secretion system [<em>ess</em>]‐<em>A</em>, <em>essC</em>, <em>lmaC</em>, <em>plcA</em>, <em>actA</em>, <em>LMOf2365_0374</em>, <em>LMOf2365_0909</em>, <em>LMOf2365_1977</em>, and <em>LMOf2365_2034</em>) and PAW (<em>actA</em>). In contrast, H<sub>2</sub>O<sub>2</sub> (<em>lhrC</em> and <em>lisK</em>), PAW (<em>inlA</em> and <em>iap</em>), and O<sub>3</sub> (<em>lysR</em>, <em>lmaD</em>, and <em>epsC</em>) increase specific gene levels in <em>L. monocytogenes</em>. <em>essA</em> and <em>essC</em> are part of the ESS or type VII secretion system in Gram‐positive bacteria. This system is crucial for the identification and secretion of proteins enhancing virulence and bacterial competition (Zoltner et al. <a href="#crf370260-bib-0236" class="usa-link" aria-describedby="crf370260-bib-0236">2016</a>; Burts et al. <a href="#crf370260-bib-0031" class="usa-link" aria-describedby="crf370260-bib-0031">2005</a>). The ActA protein of <em>L. monocytogenes</em> plays a crucial role as a virulence factor. Actin polymerization is initiated on the bacterial surface, forming an actin tail that enables bacteria to move within and between the host cells (Pillich et al. <a href="#crf370260-bib-0145" class="usa-link" aria-describedby="crf370260-bib-0145">2017</a>). <em>lisK</em> encodes the LisK protein, a histidine kinase in the LisRK two‐component system, in <em>L. monocytogenes</em>. This protein is crucial for the adaptation of organisms to environmental stresses, such as ethanol and oxidative stress (Alejandro‐Navarreto and Freitag <a href="#crf370260-bib-0005" class="usa-link" aria-describedby="crf370260-bib-0005">2024</a>). Virulence gene expression changes enhance disinfection resistance by improving the stress tolerance, suppressing biofilm formation, and altering the cell surface properties in Gram‐positive pathogens (Frees et al. <a href="#crf370260-bib-0248" class="usa-link" aria-describedby="crf370260-bib-0248">2004</a>; Rouhi et al. <a href="#crf370260-bib-0249" class="usa-link" aria-describedby="crf370260-bib-0249">2024</a>; Kastbjerg et al. <a href="#crf370260-bib-0098" class="usa-link" aria-describedby="crf370260-bib-0098">2010</a>). These varying transcriptional responses show that disinfection treatments either inhibit or stimulate virulence‐related pathways, influencing the ability of <em>L. monocytogenes</em> to persist under, adapt to, or withstand stress, depending on the environmental conditions.</p>
<p>Virulence gene expression in <em>L. monocytogenes</em> is regulated by temperature‐dependent mechanisms. Levels of genes within pVGC, including <em>actA</em>, <em>plcA</em>, and <em>hly</em>, are generally upregulated at 37°C but remain low at low temperatures. This control is mediated by a thermosensitive RNA structure in the 5′‐untranslated region of <em>prfA</em>, which unfolds at 37°C to allow translation (Johansson et al. <a href="#crf370260-bib-0089" class="usa-link" aria-describedby="crf370260-bib-0089">2002</a>; Loh et al. <a href="#crf370260-bib-0117" class="usa-link" aria-describedby="crf370260-bib-0117">2012</a>). Consequently, ClO<sub>2</sub> treatment at 37°C induces a greater baseline expression of virulence genes compared to that at low temperatures (e.g., H<sub>2</sub>O<sub>2</sub> at 20°C; PAW at room temperature). This partially explains the variability in virulence gene expression under different treatments.</p></section></section></section><section id="crf370260-sec-0570"><h2 class="pmc_sec_title">5. Limitations and Future Considerations</h2>
<p>This review summarizes the overall genetic activity of <em>L. monocytogenes</em> under different disinfection treatments. Gene expression changes in <em>L. monocytogenes</em> in response to different disinfectants provide valuable insights into the cellular mechanisms of this foodborne pathogen under stress conditions. However, the limited number of studies on <em>L. monocytogenes</em> responses to disinfection treatments makes it difficult to conclude that this pathogen responds solely in the discussed manner. Moreover, this review article has several limitations resulting from the types of disinfection treatments selected for comparison. One major issue is the variability in the treatment conditions (Table <a href="#crf370260-tbl-0001" class="usa-link">1</a>) applied to <em>L. monocytogenes</em>, which complicates the interpretation of results and identification of specific patterns related to particular conditions. Additionally, most discussed studies did not document the nature of the bacteria (bactericidal, bacteriostatic, or exhibiting reduced growth) during RNA isolation for transcriptomic analyses. For instance, populations of <em>L. monocytogenes</em> after treatment with BZT (Casey et al. <a href="#crf370260-bib-0035" class="usa-link" aria-describedby="crf370260-bib-0035">2014</a>), LA, and H<sub>2</sub>O<sub>2</sub> (Cortes et al. <a href="#crf370260-bib-0242" class="usa-link" aria-describedby="crf370260-bib-0242">2020</a>) were not recorded. Moreover, the substrate or medium used for <em>L. monocytogenes</em> disinfection varied among the selected studies. For example, O<sub>3</sub> was applied to the surface of tomatoes, whereas BZT, LA, H<sub>2</sub>O<sub>2</sub>, and HPP were applied to <em>L. monocytogenes</em> suspended in TSB or the tryptone soy yeast extract broth. ClO<sub>2</sub> and PMF were applied to <em>L. monocytogenes</em> suspended in the brain heart infusion broth and phosphate‐buffered saline, respectively.</p>
<p>Each experiment included in our analysis was conducted under different temperature conditions, which may have influenced the growth kinetics and stress responses of <em>L. monocytogenes</em> (Poimenidou et al. <a href="#crf370260-bib-0149" class="usa-link" aria-describedby="crf370260-bib-0149">2023</a>). Temperature is a critical factor affecting both the efficacy of chemical oxidants and the physiological resilience of bacterial cells (Gélinas et al. <a href="#crf370260-bib-0065" class="usa-link" aria-describedby="crf370260-bib-0065">1984</a>; Manso et al. <a href="#crf370260-bib-0123" class="usa-link" aria-describedby="crf370260-bib-0123">2020</a>). Additionally, different <em>L. monocytogenes</em> strains from multiple serotypes, such as 1/2a, 1/2c, and 4b, have been used in various studies (Table <a href="#crf370260-tbl-0001" class="usa-link">1</a>). Considering the well‐documented genetic and phenotypic diversity of <em>L. monocytogenes</em> strains, including differences in their stress tolerance and virulence, this variability further complicates the interpretation of the results (Nightingale et al. <a href="#crf370260-bib-0138" class="usa-link" aria-describedby="crf370260-bib-0138">2005</a>; Muchaamba et al. <a href="#crf370260-bib-0134" class="usa-link" aria-describedby="crf370260-bib-0134">2022</a>). However, it is important to note that only a small number of strains were evaluated in the studies discussed, which limits how well the findings apply to the broader <em>L. monocytogenes</em> population. Collectively, these differences make it difficult to draw definitive conclusions regarding the typical responses of <em>L. monocytogenes</em> to the chemical oxidants discussed in this review.</p>
<p>In addition to the disinfection treatments and bacterial conditions, the methods used for transcriptome analysis, such as RNA‐sequencing and microarray analysis, also varied among the included studies. Moreover, the fold change (FC) threshold, an important factor to accurately compare the transcriptomic responses of <em>L. monocytogenes</em> under different treatment conditions, varied across studies. For example, DEGs in <em>L. monocytogenes</em> treated with BZT were analyzed using FC thresholds ≥ 7.5 (Fox et al. <a href="#crf370260-bib-0060" class="usa-link" aria-describedby="crf370260-bib-0060">2011</a>) and ≥ 4 (Casey et al. <a href="#crf370260-bib-0035" class="usa-link" aria-describedby="crf370260-bib-0035">2014</a>). In contrast, with LA, H<sub>2</sub>O<sub>2</sub>, and PAW treatments, DEGs were identified using log2 FC &gt; 1. Studies using low FC thresholds identified many DEGs, including those showing minor expression changes. Conversely, studies with high FC thresholds only identified the genes exhibiting significant expression changes. These varying FC cutoff points affected the results, impacting their accurate comparison and interpretation.</p>
<p>Results interpretation under different experimental conditions requires careful analysis and assessment. Investigation of a specific technique under bactericidal, bacteriostatic, and reduced growth conditions can reveal the differences in gene expression. Thorough analysis is necessary when significant variations are observed. For instance, comparative analysis of their behaviors in biofilm‐forming environments and suspensions can provide insights into bacterial adaptive responses. Such meticulous analysis is crucial for understanding complex microbial behaviors and guiding future research on bacterial growth control.</p>
<p>Another limitation is that most transcriptomic studies of <em>L. monocytogenes</em> under oxidative stress conditions, as mentioned in this paper, use short exposure times—often ranging from 5 min to 1 h, which primarily capture the bacterium's initial adaptive responses rather than its fully developed physiological states and do not address the persistence of these transcriptional changes. These early responses involve the rapid upregulation of general stress response genes, such as those controlled by alternative sigma factors (e.g., <em>sigB</em>), chaperones, and noncoding RNAs, which help reduce immediate damage and prepare the cell for survival (Cortes et al. <a href="#crf370260-bib-0242" class="usa-link" aria-describedby="crf370260-bib-0242">2020</a>; Anast and Schmitz‐Esser <a href="#crf370260-bib-0250" class="usa-link" aria-describedby="crf370260-bib-0250">2020</a>). However, these transcriptional changes are usually temporary and might not accurately reflect the long‐term regulatory adjustments, metabolic reprogramming, or structural changes needed for sustained stress tolerance (Bren et al. <a href="#crf370260-bib-0251" class="usa-link" aria-describedby="crf370260-bib-0251">2023</a>; Kratz and Banerjee <a href="#crf370260-bib-0252" class="usa-link" aria-describedby="crf370260-bib-0252">2024</a>). Furthermore, the transient activation of these stress responses could lead to cross‐protection, where sublethal exposure to an oxidant preadapts the bacterium to tolerate subsequent, unrelated stresses (e.g., acid, osmotic, or thermal stress) that it encounters later (Ferreira et al. <a href="#crf370260-bib-0253" class="usa-link" aria-describedby="crf370260-bib-0253">2003</a>; Lou and Yousef <a href="#crf370260-bib-0254" class="usa-link" aria-describedby="crf370260-bib-0254">1997</a>). Additionally, many stress‐induced gene expression changes are reversible; the final physiological state of the cell depends on the duration and intensity of the stress, as well as post‐transcriptional, translational, and post‐translational regulatory layers (Roemhild et al. <a href="#crf370260-bib-0255" class="usa-link" aria-describedby="crf370260-bib-0255">2022</a>; Papadimitriou et al. <a href="#crf370260-bib-0256" class="usa-link" aria-describedby="crf370260-bib-0256">2016</a>). Therefore, relying on short‐term transcriptome data may overemphasize early responses while underestimating the complexity of ongoing adaptation mechanisms, highlighting the need for integrated time‐course and multiomics studies to fully understand the stress physiology of <em>L. monocytogenes</em>.</p>
<p>Furthermore, it is essential to acknowledge that transcriptomic data alone do not necessarily confirm the functional significance of gene expression changes. Although stress triggers rapid and widespread changes in gene transcription (e.g., via SigB), the actual protein levels or subsequent metabolic activity often do not correlate with mRNA abundance due to post‐transcriptional, translational, and post‐translational regulation (Sibanda and Buys <a href="#crf370260-bib-0189" class="usa-link" aria-describedby="crf370260-bib-0189">2022</a>; Chaturongakul et al. <a href="#crf370260-bib-0257" class="usa-link" aria-describedby="crf370260-bib-0257">2008</a>). While proteomic studies on <em>L. monocytogenes</em> under oxidative stress are currently limited, some research suggests that there may be a poor correlation between mRNA levels and corresponding protein expression under stress conditions (Johansson and Freitag <a href="#crf370260-bib-0258" class="usa-link" aria-describedby="crf370260-bib-0258">2019</a>). This includes regulation by RNA‐binding proteins (such as Hfq), small RNAs, mRNA stability, translation efficiency, and protein degradation rates (Seixas et al. <a href="#crf370260-bib-0259" class="usa-link" aria-describedby="crf370260-bib-0259">2025</a>). Additionally, extrinsic biological noise and environmental fluctuations further contribute to the disconnect between transcript and protein levels. As a result, relying solely on transcriptomic data may misrepresent the actual physiological state of <em>L. monocytogenes</em> under stress. This underscores the importance of integrating proteomic analyses to achieve a comprehensive understanding of bacterial adaptation and survival mechanisms (Bowman et al. <a href="#crf370260-bib-0260" class="usa-link" aria-describedby="crf370260-bib-0260">2012</a>; Chaturongakul et al. <a href="#crf370260-bib-0250" class="usa-link" aria-describedby="crf370260-bib-0250">2008</a>). Combining data from transcriptomic studies with proteomic and functional assays would offer a more accurate depiction of the pathogen's survival strategies.</p></section><section id="crf370260-sec-0580"><h2 class="pmc_sec_title">6. Conclusion</h2>
<p>
<em>L. monocytogenes</em> can adapt to and survive under disinfection stress, posing significant challenges for the food industry. This review highlights its complex transcriptional responses to various chemical oxidants. Oxidizing agents (e.g., ClO<sub>2</sub>, H<sub>2</sub>O<sub>2</sub>, and O<sub>3</sub>) induce strong oxidative stress responses (e.g., <em>sodA</em> and <em>ohrA</em>) and activate DNA repair systems (<em>recA</em> and <em>uvrABC</em>) in <em>L. monocytogenes</em>. Nonoxidizing treatments (e.g., BZT and HPP) upregulate the cell wall biosynthesis gene (<em>murA</em> and <em>glmS</em>) levels, hindering disinfectant penetration. Interestingly, sublethal exposure to some disinfectants unintentionally increases the virulence (<em>inlA</em> and <em>lisK</em>) and QS (<em>agrB</em> and <em>luxS</em>) gene levels, enhancing the adaptive tolerance of <em>L. monocytogenes</em>. Changes in efflux pumps (<em>qacC</em> and <em>ebrA</em>) and metabolic shifts (e.g., activation of PPP and glycerol utilization) further highlight the adaptability of this pathogen. However, variations in experimental conditions (e.g., disinfectant concentration and bacterial physiological state), transcriptomic analysis methods, and DEG thresholds across studies limit the comparability of the data. To enhance the disinfection efficacy, future studies should use standardized methodologies to investigate the pathogen responses under different conditions and states, including biofilm‐forming environments and the planktonic (suspended) state. A deeper understanding of the underlying molecular mechanisms is critical to develop effective control measures against <em>L. monocytogenes</em> contamination in food processing environments and prevent listeriosis.</p></section><section id="crf370260-sec-0590"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>
<strong>Nagendran Rajalingam</strong>: conceptualization, investigation, methodology, writing–original draft, software, formal analysis, data curation, writing–review and editing. <strong>Sam Van Haute</strong>: funding acquisition, writing–review and editing, validation, visualization, project administration, supervision, resources, formal analysis.</p></section><section id="crf370260-sec-0610"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec61"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="crf370260-supinfo-0001"></section><section class="sm xbox font-sm" id="crf370260-supitem-0001"><div class="caption p"><p>
<strong>Supporting Material</strong>: crf370260‐sup‐0001‐SuppMat.docx</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12371860/bin/CRF3-24-e70260-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">CRF3-24-e70260-s002.docx</a><sup> (95.6KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="crf370260-supitem-0002"><div class="caption p"><p>
<strong>Supporting Table and FIgure</strong>: crf370260‐sup‐0002‐SuppMat.docx</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12371860/bin/CRF3-24-e70260-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">CRF3-24-e70260-s001.docx</a><sup> (121.3KB, docx) </sup>
</div></div></section></section><section id="crf370260-sec-0600" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>This work was financially supported by the Ghent University Global Campus.</p></section><section id="notes1"><p>


Rajalingam, N.
, and Van Haute S.. 2025. “Adaptive Tolerance of <em>Listeria monocytogenes</em> to Chemical Oxidants: Comparative Analysis of Transcriptomic Studies.” Comprehensive Reviews in Food Science and Food Safety
24, no. 5: 24, e70260. 10.1111/1541-4337.70260

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="crf370260-note-0001"><p>
<strong>Funding</strong>: This research was funded by Ghent University Global Campus, the extended campus of Ghent University (Belgium) in South Korea.</p></div></div></section></section><section id="crf370260-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="crf370260-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="crf370260-bib-0001">
<cite>

Aase, B.
, Sundheim G., Langsrud S., and Rørvik L. M.. 2000. “Occurrence of and a Possible Mechanism for Resistance to a Quaternary Ammonium Compound in <em>Listeria monocytogenes</em>
.” International Journal of Food Microbiology
62, no. 1–2: 57–63. 10.1016/S0168-1605(00)00357-3.
</cite> [<a href="https://doi.org/10.1016/S0168-1605(00)00357-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11139022/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Food%20Microbiology&amp;title=Occurrence%20of%20and%20a%20Possible%20Mechanism%20for%20Resistance%20to%20a%20Quaternary%20Ammonium%20Compound%20in%20Listeria%20monocytogenes&amp;author=B.%20Aase&amp;author=G.%20Sundheim&amp;author=S.%20Langsrud&amp;author=L.%20M.%20R%C3%B8rvik&amp;volume=62&amp;issue=1%E2%80%932&amp;publication_year=2000&amp;pages=57-63&amp;pmid=11139022&amp;doi=10.1016/S0168-1605(00)00357-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0002">
<cite>

Abdel‐Aziz, S. M.
, Abo Elsoud M. M., and Anise A. A. H.. 2017. “Chapter 2 ‐ Microbial Biosynthesis: A Repertory of Vital Natural Products.” In Food Biosynthesis, edited by Grumezescu Alexandru Mihai and Maria Holban Alina, 25–54. Handbook of Food Bioengineering, Volume 1. Academic Press. 10.1016/B978-0-12-811372-1.00003-8.</cite> [<a href="https://doi.org/10.1016/B978-0-12-811372-1.00003-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Food%20Biosynthesis&amp;author=S.%20M.%20Abdel%E2%80%90Aziz&amp;author=M.%20M.%20Abo%20Elsoud&amp;author=A.%20A.%20H.%20Anise&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0003">
<cite>

Abrantes, M. C.
, Lopes M. d. F., and Kok J.. 2011. “Impact of Manganese, Copper and Zinc Ions on the Transcriptome of the Nosocomial Pathogen Enterococcus Faecalis V583.” PLoS ONE
6, no. 10: e26519. 10.1371/journal.pone.0026519.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0026519" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3203883/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22053193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Impact%20of%20Manganese,%20Copper%20and%20Zinc%20Ions%20on%20the%20Transcriptome%20of%20the%20Nosocomial%20Pathogen%20Enterococcus%20Faecalis%20V583&amp;author=M.%20C.%20Abrantes&amp;author=M.%20d.%20F.%20Lopes&amp;author=J.%20Kok&amp;volume=6&amp;issue=10&amp;publication_year=2011&amp;pages=e26519&amp;pmid=22053193&amp;doi=10.1371/journal.pone.0026519&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0004">
<cite>

Alavi, F. S.
, Gheidi M., Zahedi M., Safari N., and Ryde U.. 2018. “A Novel Mechanism of Heme Degradation to Biliverdin Studied by QM/MM and QM Calculations.” Dalton Transactions
47, no. 25: 8283–8291. 10.1039/C8DT00064F.
</cite> [<a href="https://doi.org/10.1039/C8DT00064F" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29892759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dalton%20Transactions&amp;title=A%20Novel%20Mechanism%20of%20Heme%20Degradation%20to%20Biliverdin%20Studied%20by%20QM/MM%20and%20QM%20Calculations&amp;author=F.%20S.%20Alavi&amp;author=M.%20Gheidi&amp;author=M.%20Zahedi&amp;author=N.%20Safari&amp;author=U.%20Ryde&amp;volume=47&amp;issue=25&amp;publication_year=2018&amp;pages=8283-8291&amp;pmid=29892759&amp;doi=10.1039/C8DT00064F&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0005">
<cite>

Alejandro‐Navarreto, X.
, and Freitag N. E.. 2024. “Revisiting Old Friends: Updates on the Role of Two‐Component Signaling Systems in <em>Listeria monocytogenes</em> Survival and Pathogenesis.” Infection and Immunity
92, no. 4: e00345–23. 10.1128/iai.00345-23.</cite> [<a href="https://doi.org/10.1128/iai.00345-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11003226/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38591895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infection%20and%20Immunity&amp;title=Revisiting%20Old%20Friends:%20Updates%20on%20the%20Role%20of%20Two%E2%80%90Component%20Signaling%20Systems%20in%20Listeria%20monocytogenes%20Survival%20and%20Pathogenesis&amp;author=X.%20Alejandro%E2%80%90Navarreto&amp;author=N.%20E.%20Freitag&amp;volume=92&amp;issue=4&amp;publication_year=2024&amp;pages=e00345-23&amp;pmid=38591895&amp;doi=10.1128/iai.00345-23&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0006">
<cite>

Allocati, N.
, Federici L., Masulli M., and Di Ilio C.. 2009. “Glutathione Transferases in Bacteria.” The FEBS Journal
276, no. 1: 58–75. 10.1111/j.1742-4658.2008.06743.x.
</cite> [<a href="https://doi.org/10.1111/j.1742-4658.2008.06743.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19016852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20FEBS%20Journal&amp;title=Glutathione%20Transferases%20in%20Bacteria&amp;author=N.%20Allocati&amp;author=L.%20Federici&amp;author=M.%20Masulli&amp;author=C.%20Di%20Ilio&amp;volume=276&amp;issue=1&amp;publication_year=2009&amp;pages=58-75&amp;pmid=19016852&amp;doi=10.1111/j.1742-4658.2008.06743.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0007">
<cite>

Anast, J. M.
, Bobik T. A., and Schmitz‐Esser S.. 2020. “The Cobalamin‐Dependent Gene Cluster of <em>Listeria monocytogenes</em>: Implications for Virulence, Stress Response, and Food Safety.” Frontiers in Microbiology
11, no. November: 601816. 10.3389/fmicb.2020.601816.
</cite> [<a href="https://doi.org/10.3389/fmicb.2020.601816" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7677406/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33240255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=The%20Cobalamin%E2%80%90Dependent%20Gene%20Cluster%20of%20Listeria%20monocytogenes:%20Implications%20for%20Virulence,%20Stress%20Response,%20and%20Food%20Safety&amp;author=J.%20M.%20Anast&amp;author=T.%20A.%20Bobik&amp;author=S.%20Schmitz%E2%80%90Esser&amp;volume=11&amp;issue=November&amp;publication_year=2020&amp;pages=601816&amp;pmid=33240255&amp;doi=10.3389/fmicb.2020.601816&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0250">
<cite>

Anast, J. M.
, and Schmitz‐Esser S.. 2020. “The Transcriptome of Listeria monocytogenes During Co‐Cultivation With Cheese Rind Bacteria Suggests Adaptation by Induction of Ethanolamine and 1,2‐Propanediol Catabolism Pathway Genes.” PLoS ONE
15, no. 7: e0233945. 10.1371/journal.pone.0233945.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0233945" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7377500/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32701964/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=The%20Transcriptome%20of%20Listeria%20monocytogenes%20During%20Co%E2%80%90Cultivation%20With%20Cheese%20Rind%20Bacteria%20Suggests%20Adaptation%20by%20Induction%20of%20Ethanolamine%20and%201,2%E2%80%90Propanediol%20Catabolism%20Pathway%20Genes&amp;author=J.%20M.%20Anast&amp;author=S.%20Schmitz%E2%80%90Esser&amp;volume=15&amp;issue=7&amp;publication_year=2020&amp;pages=e0233945&amp;pmid=32701964&amp;doi=10.1371/journal.pone.0233945&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0008">
<cite>

Andrés, C. M. C.
, Lastra J. M. P. d. l., Juan C. A., Plou F. J., and Pérez‐Lebeña E.. 2022. “Chemistry of Hydrogen Peroxide Formation and Elimination in Mammalian Cells, and Its Role in Various Pathologies.” Stresses
2, no. 3: 256–274. 10.3390/stresses2030019.</cite> [<a href="https://doi.org/10.3390/stresses2030019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Stresses&amp;title=Chemistry%20of%20Hydrogen%20Peroxide%20Formation%20and%20Elimination%20in%20Mammalian%20Cells,%20and%20Its%20Role%20in%20Various%20Pathologies&amp;author=C.%20M.%20C.%20Andr%C3%A9s&amp;author=J.%20M.%20P.%20d.%20l.%20Lastra&amp;author=C.%20A.%20Juan&amp;author=F.%20J.%20Plou&amp;author=E.%20P%C3%A9rez%E2%80%90Lebe%C3%B1a&amp;volume=2&amp;issue=3&amp;publication_year=2022&amp;pages=256-274&amp;doi=10.3390/stresses2030019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0009">
<cite>

Arcari, T.
, Feger M.‐L., Guerreiro D. N., Wu J., and O'Byrne C. P.. 2020. “Comparative Review of the Responses of <em>Listeria monocytogenes</em> and <em>Escherichia coli</em> to Low pH Stress.” Genes
11, no. 11: 1330. 10.3390/genes11111330.
</cite> [<a href="https://doi.org/10.3390/genes11111330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7698193/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33187233/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes&amp;title=Comparative%20Review%20of%20the%20Responses%20of%20Listeria%20monocytogenes%20and%20Escherichia%20coli%20to%20Low%20pH%20Stress&amp;author=T.%20Arcari&amp;author=M.%E2%80%90L.%20Feger&amp;author=D.%20N.%20Guerreiro&amp;author=J.%20Wu&amp;author=C.%20P.%20O'Byrne&amp;volume=11&amp;issue=11&amp;publication_year=2020&amp;pages=1330&amp;pmid=33187233&amp;doi=10.3390/genes11111330&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0010">
<cite>

Arenz, S.
, and Wilson D. N.. 2016. “Bacterial Protein Synthesis as a Target for Antibiotic Inhibition.” Cold Spring Harbor Perspectives in Medicine
6, no. 9: a025361. 10.1101/cshperspect.a025361.
</cite> [<a href="https://doi.org/10.1101/cshperspect.a025361" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5008061/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27481773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harbor%20Perspectives%20in%20Medicine&amp;title=Bacterial%20Protein%20Synthesis%20as%20a%20Target%20for%20Antibiotic%20Inhibition&amp;author=S.%20Arenz&amp;author=D.%20N.%20Wilson&amp;volume=6&amp;issue=9&amp;publication_year=2016&amp;pages=a025361&amp;pmid=27481773&amp;doi=10.1101/cshperspect.a025361&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0011">
<cite>

Baker, T. A.
, and Sauer R. T.. 2012. “ClpXP, an ATP‐Powered Unfolding and Protein‐Degradation Machine.” Biochimica et Biophysica Acta (BBA)—Molecular Cell Research
1823, no. 1: 15–28. Special issue: AAA ATPases: structure and function. 10.1016/j.bbamcr.2011.06.007.
</cite> [<a href="https://doi.org/10.1016/j.bbamcr.2011.06.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3209554/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21736903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta%20(BBA)%E2%80%94Molecular%20Cell%20Research&amp;title=ClpXP,%20an%20ATP%E2%80%90Powered%20Unfolding%20and%20Protein%E2%80%90Degradation%20Machine&amp;author=T.%20A.%20Baker&amp;author=R.%20T.%20Sauer&amp;volume=1823&amp;issue=1&amp;publication_year=2012&amp;pages=15-28&amp;pmid=21736903&amp;doi=10.1016/j.bbamcr.2011.06.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0012">
<cite>

Balogh, D.
, Eckel K., Fetzer C., and Sieber S. A.. 2022. “
<em>Listeria monocytogenes</em> Utilizes the ClpP1/2 Proteolytic Machinery for Fine‐Tuned Substrate Degradation at Elevated Temperatures.” RSC Chemical Biology
3, no. 7: 955–971. 10.1039/D2CB00077F.
</cite> [<a href="https://doi.org/10.1039/D2CB00077F" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9257651/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35866172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Chemical%20Biology&amp;title=Listeria%20monocytogenes%20Utilizes%20the%20ClpP1/2%20Proteolytic%20Machinery%20for%20Fine%E2%80%90Tuned%20Substrate%20Degradation%20at%20Elevated%20Temperatures&amp;author=D.%20Balogh&amp;author=K.%20Eckel&amp;author=C.%20Fetzer&amp;author=S.%20A.%20Sieber&amp;volume=3&amp;issue=7&amp;publication_year=2022&amp;pages=955-971&amp;pmid=35866172&amp;doi=10.1039/D2CB00077F&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0013">
<cite>

Banerji, R.
, Karkee A., Kanojiya P., Patil A., and Saroj S. D.. 2022. “Bacterial Communication in the Regulation of Stress Response in <em>Listeria monocytogenes</em>
.” LWT
154, no. January: 112703. 10.1016/j.lwt.2021.112703.</cite> [<a href="https://doi.org/10.1016/j.lwt.2021.112703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=LWT&amp;title=Bacterial%20Communication%20in%20the%20Regulation%20of%20Stress%20Response%20in%20Listeria%20monocytogenes&amp;author=R.%20Banerji&amp;author=A.%20Karkee&amp;author=P.%20Kanojiya&amp;author=A.%20Patil&amp;author=S.%20D.%20Saroj&amp;volume=154&amp;issue=January&amp;publication_year=2022&amp;pages=112703&amp;doi=10.1016/j.lwt.2021.112703&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0014">
<cite>

Bechtel, T. D.
, and Gibbons J. G.. 2021. “Population Genomic Analysis of <em>Listeria monocytogenes</em> From Food Reveals Substrate‐Specific Genome Variation.” Frontiers in Microbiology
12, no. February:. 10.3389/fmicb.2021.620033.</cite> [<a href="https://doi.org/10.3389/fmicb.2021.620033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7902062/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33633707/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Population%20Genomic%20Analysis%20of%20Listeria%20monocytogenes%20From%20Food%20Reveals%20Substrate%E2%80%90Specific%20Genome%20Variation&amp;author=T.%20D.%20Bechtel&amp;author=J.%20G.%20Gibbons&amp;volume=12&amp;issue=February&amp;publication_year=2021&amp;pmid=33633707&amp;doi=10.3389/fmicb.2021.620033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0015">
<cite>

Bennett, H. J.
, Pearce D. M., Glenn S., et al. 2007. “Characterization of relA and codY Mutants of <em>Listeria monocytogenes</em>: Identification of the CodY Regulon and Its Role in Virulence.” Molecular Microbiology
63, no. 5: 1453–1467. 10.1111/j.1365-2958.2007.05597.x.
</cite> [<a href="https://doi.org/10.1111/j.1365-2958.2007.05597.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17302820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Microbiology&amp;title=Characterization%20of%20relA%20and%20codY%20Mutants%20of%20Listeria%20monocytogenes:%20Identification%20of%20the%20CodY%20Regulon%20and%20Its%20Role%20in%20Virulence&amp;author=H.%20J.%20Bennett&amp;author=D.%20M.%20Pearce&amp;author=S.%20Glenn&amp;volume=63&amp;issue=5&amp;publication_year=2007&amp;pages=1453-1467&amp;pmid=17302820&amp;doi=10.1111/j.1365-2958.2007.05597.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0016">
<cite>

Bian, X.
, Jiang H., Meng Y., Li Y.‐p., Fang J., and Lu Z.. 2022. “Regulation of Gene Expression by Glycolytic and Gluconeogenic Enzymes.” Trends in Cell Biology
32, no. 9: 786–799. 10.1016/j.tcb.2022.02.003.
</cite> [<a href="https://doi.org/10.1016/j.tcb.2022.02.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35300892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Cell%20Biology&amp;title=Regulation%20of%20Gene%20Expression%20by%20Glycolytic%20and%20Gluconeogenic%20Enzymes&amp;author=X.%20Bian&amp;author=H.%20Jiang&amp;author=Y.%20Meng&amp;author=Y.%E2%80%90p.%20Li&amp;author=J.%20Fang&amp;volume=32&amp;issue=9&amp;publication_year=2022&amp;pages=786-799&amp;pmid=35300892&amp;doi=10.1016/j.tcb.2022.02.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0017">
<cite>

Bignucolo, A.
, Appanna V. P., Thomas S. C., et al. 2013. “Hydrogen Peroxide Stress Provokes a Metabolic Reprogramming in <em>Pseudomonas Fluorescens</em>: Enhanced Production of Pyruvate.” Journal of Biotechnology
167, no. 3: 309–315. 10.1016/j.jbiotec.2013.07.002.
</cite> [<a href="https://doi.org/10.1016/j.jbiotec.2013.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23871654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biotechnology&amp;title=Hydrogen%20Peroxide%20Stress%20Provokes%20a%20Metabolic%20Reprogramming%20in%20Pseudomonas%20Fluorescens:%20Enhanced%20Production%20of%20Pyruvate&amp;author=A.%20Bignucolo&amp;author=V.%20P.%20Appanna&amp;author=S.%20C.%20Thomas&amp;volume=167&amp;issue=3&amp;publication_year=2013&amp;pages=309-315&amp;pmid=23871654&amp;doi=10.1016/j.jbiotec.2013.07.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0018">
<cite>

Bland, R.
, Brown S. R. B., Waite‐Cusic J., and Kovacevic J.. 2022. “Probing Antimicrobial Resistance and Sanitizer Tolerance Themes and Their Implications for the Food Industry Through the <em>Listeria monocytogenes</em> Lens.” Comprehensive Reviews in Food Science and Food Safety
21, no. 2: 1777–1802. 10.1111/1541-4337.12910.
</cite> [<a href="https://doi.org/10.1111/1541-4337.12910" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35212132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comprehensive%20Reviews%20in%20Food%20Science%20and%20Food%20Safety&amp;title=Probing%20Antimicrobial%20Resistance%20and%20Sanitizer%20Tolerance%20Themes%20and%20Their%20Implications%20for%20the%20Food%20Industry%20Through%20the%20Listeria%20monocytogenes%20Lens&amp;author=R.%20Bland&amp;author=S.%20R.%20B.%20Brown&amp;author=J.%20Waite%E2%80%90Cusic&amp;author=J.%20Kovacevic&amp;volume=21&amp;issue=2&amp;publication_year=2022&amp;pages=1777-1802&amp;pmid=35212132&amp;doi=10.1111/1541-4337.12910&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0019">
<cite>

Boersch, M.
, Rudrawar S., Grant G., and Zunk M.. 2018. “Menaquinone Biosynthesis Inhibition: A Review of Advancements Toward a New Antibiotic Mechanism.” RSC Advances
8, no. 10: 5099–5105. 10.1039/C7RA12950E.
</cite> [<a href="https://doi.org/10.1039/C7RA12950E" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9078190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35542397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Advances&amp;title=Menaquinone%20Biosynthesis%20Inhibition:%20A%20Review%20of%20Advancements%20Toward%20a%20New%20Antibiotic%20Mechanism&amp;author=M.%20Boersch&amp;author=S.%20Rudrawar&amp;author=G.%20Grant&amp;author=M.%20Zunk&amp;volume=8&amp;issue=10&amp;publication_year=2018&amp;pages=5099-5105&amp;pmid=35542397&amp;doi=10.1039/C7RA12950E&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0020">
<cite>

Booker, S.
, and Stubbe J.. 1993. “Cloning, Sequencing, and Expression of the Adenosylcobalamin‐Dependent Ribonucleotide Reductase From <em>Lactobacillus leichmannii</em>
.” Proceedings of the National Academy of Sciences of the United States of America
90, no. 18: 8352. 10.1073/pnas.90.18.8352.
</cite> [<a href="https://doi.org/10.1073/pnas.90.18.8352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC47354/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8397403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Cloning,%20Sequencing,%20and%20Expression%20of%20the%20Adenosylcobalamin%E2%80%90Dependent%20Ribonucleotide%20Reductase%20From%20Lactobacillus%20leichmannii&amp;author=S.%20Booker&amp;author=J.%20Stubbe&amp;volume=90&amp;issue=18&amp;publication_year=1993&amp;pages=8352&amp;pmid=8397403&amp;doi=10.1073/pnas.90.18.8352&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0021">
<cite>

Botondi, R.
, Barone M., and Grasso C.. 2021. “A Review Into the Effectiveness of Ozone Technology for Improving the Safety and Preserving the Quality of Fresh‐Cut Fruits and Vegetables.” Foods
10, no. 4: 748. 10.3390/foods10040748.
</cite> [<a href="https://doi.org/10.3390/foods10040748" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8065486/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33915979/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Foods&amp;title=A%20Review%20Into%20the%20Effectiveness%20of%20Ozone%20Technology%20for%20Improving%20the%20Safety%20and%20Preserving%20the%20Quality%20of%20Fresh%E2%80%90Cut%20Fruits%20and%20Vegetables&amp;author=R.%20Botondi&amp;author=M.%20Barone&amp;author=C.%20Grasso&amp;volume=10&amp;issue=4&amp;publication_year=2021&amp;pages=748&amp;pmid=33915979&amp;doi=10.3390/foods10040748&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0022">
<cite>

Bowman, J. P.
, Bittencourt C. R., and Ross T.. 2008. “Differential Gene Expression of <em>Listeria monocytogenes</em> During High Hydrostatic Pressure Processing.” Microbiology (Reading, England)
154, no. 2: 462–475. 10.1099/mic.0.2007/010314-0.
</cite> [<a href="https://doi.org/10.1099/mic.0.2007/010314-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18227250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology%20(Reading,%20England)&amp;title=Differential%20Gene%20Expression%20of%20Listeria%20monocytogenes%20During%20High%20Hydrostatic%20Pressure%20Processing&amp;author=J.%20P.%20Bowman&amp;author=C.%20R.%20Bittencourt&amp;author=T.%20Ross&amp;volume=154&amp;issue=2&amp;publication_year=2008&amp;pages=462-475&amp;pmid=18227250&amp;doi=10.1099/mic.0.2007/010314-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0260">
<cite>

Bowman, J. P.
, Hages E., Nilsson R. E., Kocharunchitt C., and Ross T.. 2012. “Investigation of the Listeria Monocytogenes Scott A Acid Tolerance Response and Associated Physiological and Phenotypic Features via Whole Proteome Analysis.” Journal of Proteome Research
11, no. 4: 2409–2426. 10.1021/pr201137c.
</cite> [<a href="https://doi.org/10.1021/pr201137c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22372944/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteome%20Research&amp;title=Investigation%20of%20the%20Listeria%20Monocytogenes%20Scott%20A%20Acid%20Tolerance%20Response%20and%20Associated%20Physiological%20and%20Phenotypic%20Features%20via%20Whole%20Proteome%20Analysis&amp;author=J.%20P.%20Bowman&amp;author=E.%20Hages&amp;author=R.%20E.%20Nilsson&amp;author=C.%20Kocharunchitt&amp;author=T.%20Ross&amp;volume=11&amp;issue=4&amp;publication_year=2012&amp;pages=2409-2426&amp;pmid=22372944&amp;doi=10.1021/pr201137c&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0261">
<cite>

Bowman, J. P.
, Chang K. J. L., Pinfold T., and Ross T.. 2010. “Transcriptomic and Phenotypic Responses of Listeria Monocytogenes Strains Possessing Different Growth Efficiencies Under Acidic Conditions.” Applied and Environmental Microbiology
76, no. 14: 4836–4850. 10.1128/AEM.00315-10.
</cite> [<a href="https://doi.org/10.1128/AEM.00315-10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2901733/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20511423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=Transcriptomic%20and%20Phenotypic%20Responses%20of%20Listeria%20Monocytogenes%20Strains%20Possessing%20Different%20Growth%20Efficiencies%20Under%20Acidic%20Conditions&amp;author=J.%20P.%20Bowman&amp;author=K.%20J.%20L.%20Chang&amp;author=T.%20Pinfold&amp;author=T.%20Ross&amp;volume=76&amp;issue=14&amp;publication_year=2010&amp;pages=4836-4850&amp;pmid=20511423&amp;doi=10.1128/AEM.00315-10&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0023">
<cite>

Boyle, J.

2005. “Lehninger Principles of Biochemistry (4th Ed.): Nelson, D., and Cox, M.” Biochemistry and Molecular Biology Education
33, no. 1: 74–75. 10.1002/bmb.2005.494033010419.</cite> [<a href="https://doi.org/10.1002/bmb.2005.494033010419" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry%20and%20Molecular%20Biology%20Education&amp;title=Lehninger%20Principles%20of%20Biochemistry%20(4th%20Ed.):%20Nelson,%20D.,%20and%20Cox,%20M&amp;author=J.%20Boyle&amp;volume=33&amp;issue=1&amp;publication_year=2005&amp;pages=74-75&amp;doi=10.1002/bmb.2005.494033010419&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0024">
<cite>

Brauge, T.
, Faille C., Leleu G., Denis C., Hanin A., and Midelet G.. 2020. “Treatment With Disinfectants May Induce an Increase in Viable but Non‐Culturable Populations of <em>Listeria monocytogenes</em> in Biofilms Formed in Smoked Salmon Processing Environments.” Food Microbiology
92: 103548. 10.1016/j.fm.2020.103548.
</cite> [<a href="https://doi.org/10.1016/j.fm.2020.103548" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32950145/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Microbiology&amp;title=Treatment%20With%20Disinfectants%20May%20Induce%20an%20Increase%20in%20Viable%20but%20Non%E2%80%90Culturable%20Populations%20of%20Listeria%20monocytogenes%20in%20Biofilms%20Formed%20in%20Smoked%20Salmon%20Processing%20Environments&amp;author=T.%20Brauge&amp;author=C.%20Faille&amp;author=G.%20Leleu&amp;author=C.%20Denis&amp;author=A.%20Hanin&amp;volume=92&amp;publication_year=2020&amp;pages=103548&amp;pmid=32950145&amp;doi=10.1016/j.fm.2020.103548&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0025">
<cite>

Bren, A.
, and Eisenbach M.. 2000. “How Signals Are Heard During Bacterial Chemotaxis: Protein‐Protein Interactions in Sensory Signal Propagation.” Journal of Bacteriology
182, no. 24: 6865–6873.
</cite> [<a href="https://doi.org/10.1128/jb.182.24.6865-6873.2000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC94809/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11092844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Bacteriology&amp;title=How%20Signals%20Are%20Heard%20During%20Bacterial%20Chemotaxis:%20Protein%E2%80%90Protein%20Interactions%20in%20Sensory%20Signal%20Propagation&amp;author=A.%20Bren&amp;author=M.%20Eisenbach&amp;volume=182&amp;issue=24&amp;publication_year=2000&amp;pages=6865-6873&amp;pmid=11092844&amp;doi=10.1128/jb.182.24.6865-6873.2000&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0251">
<cite>

Bren, A.
, Glass D. S., Kohanim Y. K., Mayo A., and Alon U.. 2023. “Tradeoffs in Bacterial Physiology Determine the Efficiency of Antibiotic Killing.” Proceedings of the National Academy of Sciences of the United States of America
120, no. 51: e2312651120. 10.1073/pnas.2312651120.
</cite> [<a href="https://doi.org/10.1073/pnas.2312651120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10742385/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38096408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Tradeoffs%20in%20Bacterial%20Physiology%20Determine%20the%20Efficiency%20of%20Antibiotic%20Killing&amp;author=A.%20Bren&amp;author=D.%20S.%20Glass&amp;author=Y.%20K.%20Kohanim&amp;author=A.%20Mayo&amp;author=U.%20Alon&amp;volume=120&amp;issue=51&amp;publication_year=2023&amp;pages=e2312651120&amp;pmid=38096408&amp;doi=10.1073/pnas.2312651120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0026">
<cite>

Brissac, T.
, Ziveri J., Ramond E., et al. 2015. “Gluconeogenesis, an Essential Metabolic Pathway for Pathogenic <em>Rancisella</em>
.” Molecular Microbiology
98, no. 3: 518–534. 10.1111/mmi.13139.
</cite> [<a href="https://doi.org/10.1111/mmi.13139" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26192619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Microbiology&amp;title=Gluconeogenesis,%20an%20Essential%20Metabolic%20Pathway%20for%20Pathogenic%20Rancisella&amp;author=T.%20Brissac&amp;author=J.%20Ziveri&amp;author=E.%20Ramond&amp;volume=98&amp;issue=3&amp;publication_year=2015&amp;pages=518-534&amp;pmid=26192619&amp;doi=10.1111/mmi.13139&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0027">
<cite>

Brown, M. J.
, Russo B. C., O'Dee D. M., Schmitt D. M., and Nau G. J.. 2014. “The Contribution of the Glycine Cleavage System to the Pathogenesis of <em>Francisella tularensis</em>
.” Microbes and Infection /Institut Pasteur
16, no. 4: 300–309. 10.1016/j.micinf.2013.12.003.</cite> [<a href="https://doi.org/10.1016/j.micinf.2013.12.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4089098/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24374051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbes%20and%20Infection%20/Institut%20Pasteur&amp;title=The%20Contribution%20of%20the%20Glycine%20Cleavage%20System%20to%20the%20Pathogenesis%20of%20Francisella%20tularensis&amp;author=M.%20J.%20Brown&amp;author=B.%20C.%20Russo&amp;author=D.%20M.%20O'Dee&amp;author=D.%20M.%20Schmitt&amp;author=G.%20J.%20Nau&amp;volume=16&amp;issue=4&amp;publication_year=2014&amp;pages=300-309&amp;pmid=24374051&amp;doi=10.1016/j.micinf.2013.12.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0028">
<cite>

Brown, S.
, Santa Maria J. P., and Walker S.. 2013. “Wall Teichoic Acids of Gram‐Positive Bacteria.” Annual Review of Microbiology
67: 313–336. 10.1146/annurev-micro-092412-155620.</cite> [<a href="https://doi.org/10.1146/annurev-micro-092412-155620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3883102/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24024634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Microbiology&amp;title=Wall%20Teichoic%20Acids%20of%20Gram%E2%80%90Positive%20Bacteria&amp;author=S.%20Brown&amp;author=J.%20P.%20Santa%20Maria&amp;author=S.%20Walker&amp;volume=67&amp;publication_year=2013&amp;pages=313-336&amp;pmid=24024634&amp;doi=10.1146/annurev-micro-092412-155620&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0029">
<cite>

Buckel, W.
, and Thauer R. K.. 2013. “Energy Conservation via Electron Bifurcating Ferredoxin Reduction and Proton/Na+ Translocating Ferredoxin Oxidation.” Biochimica et Biophysica Acta (BBA)—Bioenergetics
1827, no. 2: 94–113. Special issue: The evolutionary aspects of bioenergetic systems. 10.1016/j.bbabio.2012.07.002.
</cite> [<a href="https://doi.org/10.1016/j.bbabio.2012.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22800682/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta%20(BBA)%E2%80%94Bioenergetics&amp;title=Energy%20Conservation%20via%20Electron%20Bifurcating%20Ferredoxin%20Reduction%20and%20Proton/Na+%20Translocating%20Ferredoxin%20Oxidation&amp;author=W.%20Buckel&amp;author=R.%20K.%20Thauer&amp;volume=1827&amp;issue=2&amp;publication_year=2013&amp;pages=94-113&amp;pmid=22800682&amp;doi=10.1016/j.bbabio.2012.07.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0030">
<cite>

Bucur, F. I.
, Grigore‐Gurgu L., Crauwels P., Riedel C. U., and Nicolau A. I.. 2018. “Resistance of <em>Listeria monocytogenes</em> to Stress Conditions Encountered in Food and Food Processing Environments.” Frontiers in Microbiology
9, no. November: 2700. 10.3389/fmicb.2018.02700.
</cite> [<a href="https://doi.org/10.3389/fmicb.2018.02700" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6282059/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30555426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Resistance%20of%20Listeria%20monocytogenes%20to%20Stress%20Conditions%20Encountered%20in%20Food%20and%20Food%20Processing%20Environments&amp;author=F.%20I.%20Bucur&amp;author=L.%20Grigore%E2%80%90Gurgu&amp;author=P.%20Crauwels&amp;author=C.%20U.%20Riedel&amp;author=A.%20I.%20Nicolau&amp;volume=9&amp;issue=November&amp;publication_year=2018&amp;pages=2700&amp;pmid=30555426&amp;doi=10.3389/fmicb.2018.02700&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0031">
<cite>

Burts, M. L.
, Williams W. A., DeBord K., and Missiakas D. M.. 2005. “EsxA and EsxB Are Secreted by an ESAT‐6‐Like System That Is Required for the Pathogenesis of <em>Staphylococcus aureus</em> Infections.” Proceedings of the National Academy of Sciences
102, no. 4: 1169–1174. 10.1073/pnas.0405620102.</cite> [<a href="https://doi.org/10.1073/pnas.0405620102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC545836/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15657139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=EsxA%20and%20EsxB%20Are%20Secreted%20by%20an%20ESAT%E2%80%906%E2%80%90Like%20System%20That%20Is%20Required%20for%20the%20Pathogenesis%20of%20Staphylococcus%20aureus%20Infections&amp;author=M.%20L.%20Burts&amp;author=W.%20A.%20Williams&amp;author=K.%20DeBord&amp;author=D.%20M.%20Missiakas&amp;volume=102&amp;issue=4&amp;publication_year=2005&amp;pages=1169-1174&amp;pmid=15657139&amp;doi=10.1073/pnas.0405620102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0033">
<cite>

Cárdenas‐Moreno, Y.
, González‐Bacerio J., Arellano H. G., and Del Monte‐Martínez A.. 2023. “Oxidoreductase Enzymes: Characteristics, Applications, and Challenges as a Biocatalyst.” Biotechnology and Applied Biochemistry
70, no. 6: 2108–2135. 10.1002/bab.2513.
</cite> [<a href="https://doi.org/10.1002/bab.2513" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37753743/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnology%20and%20Applied%20Biochemistry&amp;title=Oxidoreductase%20Enzymes:%20Characteristics,%20Applications,%20and%20Challenges%20as%20a%20Biocatalyst&amp;author=Y.%20C%C3%A1rdenas%E2%80%90Moreno&amp;author=J.%20Gonz%C3%A1lez%E2%80%90Bacerio&amp;author=H.%20G.%20Arellano&amp;author=A.%20Del%20Monte%E2%80%90Mart%C3%ADnez&amp;volume=70&amp;issue=6&amp;publication_year=2023&amp;pages=2108-2135&amp;pmid=37753743&amp;doi=10.1002/bab.2513&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0032">
<cite>

Cai, J.
, Zhang M., Chen Z., et al. 2023. “MoaE Is Involved in Response to Oxidative Stress in <em>Deinococcus radiodurans</em>
.” International Journal of Molecular Sciences
24, no. 3: 2441. 10.3390/ijms24032441.
</cite> [<a href="https://doi.org/10.3390/ijms24032441" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9916421/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36768763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=MoaE%20Is%20Involved%20in%20Response%20to%20Oxidative%20Stress%20in%20Deinococcus%20radiodurans&amp;author=J.%20Cai&amp;author=M.%20Zhang&amp;author=Z.%20Chen&amp;volume=24&amp;issue=3&amp;publication_year=2023&amp;pages=2441&amp;pmid=36768763&amp;doi=10.3390/ijms24032441&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0034">
<cite>

Casado, J.
, Lanas Á. n., and González A.. 2022. “Two‐Component Regulatory Systems in <em>Helicobacter pylori</em> and <em>Campylobacter jejuni</em>: Attractive Targets for Novel Antibacterial Drugs.” Frontiers in Cellular and Infection Microbiology
12, no. August. 10.3389/fcimb.2022.977944.</cite> [<a href="https://doi.org/10.3389/fcimb.2022.977944" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9449129/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36093179/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Cellular%20and%20Infection%20Microbiology&amp;title=Two%E2%80%90Component%20Regulatory%20Systems%20in%20Helicobacter%20pylori%20and%20Campylobacter%20jejuni:%20Attractive%20Targets%20for%20Novel%20Antibacterial%20Drugs&amp;author=J.%20Casado&amp;author=%C3%81.%20n.%20Lanas&amp;author=A.%20Gonz%C3%A1lez&amp;volume=12&amp;issue=August&amp;publication_year=2022&amp;pmid=36093179&amp;doi=10.3389/fcimb.2022.977944&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0035">
<cite>

Casey, A.
, Fox E. M., Schmitz‐Esser S., Coffey A., McAuliffe O., and Jordan K.. 2014. “Transcriptome Analysis of <em>Listeria monocytogenes</em> Exposed to Biocide Stress Reveals a Multi‐System Response Involving Cell Wall Synthesis, Sugar Uptake, and Motility.” Frontiers in Microbiology
5: 68. 10.3389/fmicb.2014.00068.</cite> [<a href="https://doi.org/10.3389/fmicb.2014.00068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3937556/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24616718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Transcriptome%20Analysis%20of%20Listeria%20monocytogenes%20Exposed%20to%20Biocide%20Stress%20Reveals%20a%20Multi%E2%80%90System%20Response%20Involving%20Cell%20Wall%20Synthesis,%20Sugar%20Uptake,%20and%20Motility&amp;author=A.%20Casey&amp;author=E.%20M.%20Fox&amp;author=S.%20Schmitz%E2%80%90Esser&amp;author=A.%20Coffey&amp;author=O.%20McAuliffe&amp;volume=5&amp;publication_year=2014&amp;pmid=24616718&amp;doi=10.3389/fmicb.2014.00068&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0036">
<cite>

Casiraghi, A.
, Suigo L., Valoti E., and Straniero V.. 2020. “Targeting Bacterial Cell Division: A Binding Site‐Centered Approach to the Most Promising Inhibitors of the Essential Protein FtsZ.” Antibiotics
9, no. 2: 69. 10.3390/antibiotics9020069.
</cite> [<a href="https://doi.org/10.3390/antibiotics9020069" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7167804/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32046082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antibiotics&amp;title=Targeting%20Bacterial%20Cell%20Division:%20A%20Binding%20Site%E2%80%90Centered%20Approach%20to%20the%20Most%20Promising%20Inhibitors%20of%20the%20Essential%20Protein%20FtsZ&amp;author=A.%20Casiraghi&amp;author=L.%20Suigo&amp;author=E.%20Valoti&amp;author=V.%20Straniero&amp;volume=9&amp;issue=2&amp;publication_year=2020&amp;pages=69&amp;pmid=32046082&amp;doi=10.3390/antibiotics9020069&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0037">
<cite>

Chandel, N. S.

2021. “Carbohydrate Metabolism.” Cold Spring Harbor Perspectives in Biology
13, no. 1: a040568. 10.1101/cshperspect.a040568.
</cite> [<a href="https://doi.org/10.1101/cshperspect.a040568" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7778149/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33397651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harbor%20Perspectives%20in%20Biology&amp;title=Carbohydrate%20Metabolism&amp;author=N.%20S.%20Chandel&amp;volume=13&amp;issue=1&amp;publication_year=2021&amp;pages=a040568&amp;pmid=33397651&amp;doi=10.1101/cshperspect.a040568&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0257">
<cite>

Chaturongakul, S.
, Raengpradub S., Wiedmann M., and Boor K. J.. 2008. “Modulation of Stress and Virulence in Listeria Monocytogenes.” Trends in Microbiology
16, no. 8: 388–396. 10.1016/j.tim.2008.05.006.
</cite> [<a href="https://doi.org/10.1016/j.tim.2008.05.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3400534/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18619843/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Microbiology&amp;title=Modulation%20of%20Stress%20and%20Virulence%20in%20Listeria%20Monocytogenes&amp;author=S.%20Chaturongakul&amp;author=S.%20Raengpradub&amp;author=M.%20Wiedmann&amp;author=K.%20J.%20Boor&amp;volume=16&amp;issue=8&amp;publication_year=2008&amp;pages=388-396&amp;pmid=18619843&amp;doi=10.1016/j.tim.2008.05.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0038">
<cite>

Chaudhry, R.
, and Varacallo M.. 2024. “Biochemistry, Glycolysis.” In StatPearls. StatPearls Publishing. <a href="http://www.ncbi.nlm.nih.gov/books/NBK482303/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.ncbi.nlm.nih.gov/books/NBK482303/</a>.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/29493928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=StatPearls&amp;author=R.%20Chaudhry&amp;author=M.%20Varacallo&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0039">
<cite>

Chen, Y.
, Ying Y., Lalsiamthara J., et al. 2024. “From Bacteria to Biomedicine: Developing Therapies Exploiting NAD+ Metabolism.” Bioorganic Chemistry
142, no. January: 106974. 10.1016/j.bioorg.2023.106974.
</cite> [<a href="https://doi.org/10.1016/j.bioorg.2023.106974" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37984103/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorganic%20Chemistry&amp;title=From%20Bacteria%20to%20Biomedicine:%20Developing%20Therapies%20Exploiting%20NAD+%20Metabolism&amp;author=Y.%20Chen&amp;author=Y.%20Ying&amp;author=J.%20Lalsiamthara&amp;volume=142&amp;issue=January&amp;publication_year=2024&amp;pages=106974&amp;pmid=37984103&amp;doi=10.1016/j.bioorg.2023.106974&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0040">
<cite>

Cheng, C.
, Dong Z., Han X., et al. 2017. “Thioredoxin A Is Essential for Motility and Contributes to Host Infection of <em>Listeria monocytogenes</em> via Redox Interactions.” Frontiers in Cellular and Infection Microbiology
7, no. June: 287. 10.3389/fcimb.2017.00287.</cite> [<a href="https://doi.org/10.3389/fcimb.2017.00287" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5487381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28702378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Cellular%20and%20Infection%20Microbiology&amp;title=Thioredoxin%20A%20Is%20Essential%20for%20Motility%20and%20Contributes%20to%20Host%20Infection%20of%20Listeria%20monocytogenes%20via%20Redox%20Interactions&amp;author=C.%20Cheng&amp;author=Z.%20Dong&amp;author=X.%20Han&amp;volume=7&amp;issue=June&amp;publication_year=2017&amp;pmid=28702378&amp;doi=10.3389/fcimb.2017.00287&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0041">
<cite>

Choby, J. E.
, and Skaar E. P.. 2016. “Heme Synthesis and Acquisition in Bacterial Pathogens.” Journal of Molecular Biology
428, no. 17: 3408. 10.1016/j.jmb.2016.03.018.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2016.03.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5125930/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27019298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Molecular%20Biology&amp;title=Heme%20Synthesis%20and%20Acquisition%20in%20Bacterial%20Pathogens&amp;author=J.%20E.%20Choby&amp;author=E.%20P.%20Skaar&amp;volume=428&amp;issue=17&amp;publication_year=2016&amp;pages=3408&amp;pmid=27019298&amp;doi=10.1016/j.jmb.2016.03.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0042">
<cite>

Chuwa, C.
, Vaidya D., Kathuria D., Gautam S., Sharma S., and Sharma B.. 2020. “Ozone (O<sub>3</sub>): An Emerging Technology in the Food Industry.” Food &amp; Nutrition Journal
5: 224.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20&amp;%20Nutrition%20Journal&amp;title=Ozone%20(O3):%20An%20Emerging%20Technology%20in%20the%20Food%20Industry&amp;author=C.%20Chuwa&amp;author=D.%20Vaidya&amp;author=D.%20Kathuria&amp;author=S.%20Gautam&amp;author=S.%20Sharma&amp;volume=5&amp;publication_year=2020&amp;pages=224&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0043">
<cite>

Coelho, C.
, Brown L., Maryam M., et al. 2018. “
<em>Listeria monocytogenes</em> Virulence Factors, Including Listeriolysin O, Are Secreted in Biologically Active Extracellular Vesicles.” The Journal of Biological Chemistry
294, no. 4: 1202. 10.1074/jbc.RA118.006472.
</cite> [<a href="https://doi.org/10.1074/jbc.RA118.006472" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6349127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30504226/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Biological%20Chemistry&amp;title=Listeria%20monocytogenes%20Virulence%20Factors,%20Including%20Listeriolysin%20O,%20Are%20Secreted%20in%20Biologically%20Active%20Extracellular%20Vesicles&amp;author=C.%20Coelho&amp;author=L.%20Brown&amp;author=M.%20Maryam&amp;volume=294&amp;issue=4&amp;publication_year=2018&amp;pages=1202&amp;pmid=30504226&amp;doi=10.1074/jbc.RA118.006472&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0044">
<cite>

Cohn, M. T.
, Kjelgaard P., Frees D., Penadés J. R., and Ingmer H.. 2011. “Clp‐Dependent Proteolysis of the LexA N‐Terminal Domain in <em>Staphylococcus aureus</em>
.” Microbiology (Reading, England)
157, no. 3: 677–684. 10.1099/mic.0.043794-0.
</cite> [<a href="https://doi.org/10.1099/mic.0.043794-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21183573/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology%20(Reading,%20England)&amp;title=Clp%E2%80%90Dependent%20Proteolysis%20of%20the%20LexA%20N%E2%80%90Terminal%20Domain%20in%20Staphylococcus%20aureus&amp;author=M.%20T.%20Cohn&amp;author=P.%20Kjelgaard&amp;author=D.%20Frees&amp;author=J.%20R.%20Penad%C3%A9s&amp;author=H.%20Ingmer&amp;volume=157&amp;issue=3&amp;publication_year=2011&amp;pages=677-684&amp;pmid=21183573&amp;doi=10.1099/mic.0.043794-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0045">
<cite>

Contreras‐Soto, M.
, Medrano‐Félix J., Valdez‐Torres B., Chaidez C., and Castro‐del Campo N.. 2021. “Chlorine Dioxide: An Evaluation Based on a Microbial Decay Approach During Mango Packing Process.” International Journal of Environmental Health Research
31, no. 5: 518–529. 10.1080/09603123.2019.1670785.
</cite> [<a href="https://doi.org/10.1080/09603123.2019.1670785" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31569961/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Environmental%20Health%20Research&amp;title=Chlorine%20Dioxide:%20An%20Evaluation%20Based%20on%20a%20Microbial%20Decay%20Approach%20During%20Mango%20Packing%20Process&amp;author=M.%20Contreras%E2%80%90Soto&amp;author=J.%20Medrano%E2%80%90F%C3%A9lix&amp;author=B.%20Valdez%E2%80%90Torres&amp;author=C.%20Chaidez&amp;author=N.%20Castro%E2%80%90del%20Campo&amp;volume=31&amp;issue=5&amp;publication_year=2021&amp;pages=518-529&amp;pmid=31569961&amp;doi=10.1080/09603123.2019.1670785&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0242">
<cite>

Cortes, B. W.
, Naditz A. L., Anast J. M., and Schmitz‐Esser S.. 2020. “Transcriptome Sequencing of Listeria Monocytogenes Reveals Major Gene Expression Changes in Response to Lactic Acid Stress Exposure but a Less Pronounced Response to Oxidative Stress.” Frontiers in Microbiology
10, no. January: 3110. 10.3389/fmicb.2019.03110.
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.03110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6985202/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32038553/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Transcriptome%20Sequencing%20of%20Listeria%20Monocytogenes%20Reveals%20Major%20Gene%20Expression%20Changes%20in%20Response%20to%20Lactic%20Acid%20Stress%20Exposure%20but%20a%20Less%20Pronounced%20Response%20to%20Oxidative%20Stress&amp;author=B.%20W.%20Cortes&amp;author=A.%20L.%20Naditz&amp;author=J.%20M.%20Anast&amp;author=S.%20Schmitz%E2%80%90Esser&amp;volume=10&amp;issue=January&amp;publication_year=2020&amp;pages=3110&amp;pmid=32038553&amp;doi=10.3389/fmicb.2019.03110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0046">
<cite>

Cory, M. B.
, Li A., Hurley C. M., et al. 2024. “The LexA‐RecA* Structure Reveals a Cryptic Lock‐and‐Key Mechanism for SOS Activation.” Nature Structural &amp; Molecular Biology
31, no. 10: 1522–1531. 10.1038/s41594-024-01317-3.</cite> [<a href="https://doi.org/10.1038/s41594-024-01317-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11521096/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38755298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Structural%20&amp;%20Molecular%20Biology&amp;title=The%20LexA%E2%80%90RecA*%20Structure%20Reveals%20a%20Cryptic%20Lock%E2%80%90and%E2%80%90Key%20Mechanism%20for%20SOS%20Activation&amp;author=M.%20B.%20Cory&amp;author=A.%20Li&amp;author=C.%20M.%20Hurley&amp;volume=31&amp;issue=10&amp;publication_year=2024&amp;pages=1522-1531&amp;pmid=38755298&amp;doi=10.1038/s41594-024-01317-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0244">
<cite>

Cotter, P. D.
, and Hill C.. 2003. “Surviving the Acid Test: Responses of Gram‐Positive Bacteria to Low pH.” Microbiology and Molecular Biology Reviews: MMBR
67, no. 3: 429–453, table of contents. 10.1128/MMBR.67.3.429-453.2003.
</cite> [<a href="https://doi.org/10.1128/MMBR.67.3.429-453.2003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC193868/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12966143/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology%20and%20Molecular%20Biology%20Reviews:%20MMBR&amp;title=Surviving%20the%20Acid%20Test:%20Responses%20of%20Gram%E2%80%90Positive%20Bacteria%20to%20Low%20pH&amp;author=P.%20D.%20Cotter&amp;author=C.%20Hill&amp;volume=67&amp;issue=3&amp;publication_year=2003&amp;pages=429-453&amp;pmid=12966143&amp;doi=10.1128/MMBR.67.3.429-453.2003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0047">
<cite>

Coupat, B.
, Chaumeille‐Dole F., Fall S., et al. 2008. “Natural Transformation in the <em>Ralstonia solanacearum</em> Species Complex: Number and Size of DNA That Can Be Transferred.” FEMS Microbiology Ecology
66, no. 1: 14–24. 10.1111/j.1574-6941.2008.00552.x.
</cite> [<a href="https://doi.org/10.1111/j.1574-6941.2008.00552.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18662313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiology%20Ecology&amp;title=Natural%20Transformation%20in%20the%20Ralstonia%20solanacearum%20Species%20Complex:%20Number%20and%20Size%20of%20DNA%20That%20Can%20Be%20Transferred&amp;author=B.%20Coupat&amp;author=F.%20Chaumeille%E2%80%90Dole&amp;author=S.%20Fall&amp;volume=66&amp;issue=1&amp;publication_year=2008&amp;pages=14-24&amp;pmid=18662313&amp;doi=10.1111/j.1574-6941.2008.00552.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0048">
<cite>

Dai, Y.
, Tang H., and Pang S.. 2021. “The Crucial Roles of Phospholipids in Aging and Lifespan Regulation.” Frontiers in Physiology
12, no. November: 775648. 10.3389/fphys.2021.775648.
</cite> [<a href="https://doi.org/10.3389/fphys.2021.775648" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8650052/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34887779/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Physiology&amp;title=The%20Crucial%20Roles%20of%20Phospholipids%20in%20Aging%20and%20Lifespan%20Regulation&amp;author=Y.%20Dai&amp;author=H.%20Tang&amp;author=S.%20Pang&amp;volume=12&amp;issue=November&amp;publication_year=2021&amp;pages=775648&amp;pmid=34887779&amp;doi=10.3389/fphys.2021.775648&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0049">
<cite>

Das, A.
, Hugenholtz J., Van Halbeek H., and Ljungdahl L. G.. 1989. “Structure and Function of a Menaquinone Involved in Electron Transport in Membranes of <em>Clostridium thermoautotrophicum</em> and <em>Clostridium thermoaceticum</em>
.” Journal of Bacteriology
171, no. 11: 5823–5829. 10.1128/jb.171.11.5823-5829.1989.
</cite> [<a href="https://doi.org/10.1128/jb.171.11.5823-5829.1989" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC210442/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2808299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Bacteriology&amp;title=Structure%20and%20Function%20of%20a%20Menaquinone%20Involved%20in%20Electron%20Transport%20in%20Membranes%20of%20Clostridium%20thermoautotrophicum%20and%20Clostridium%20thermoaceticum&amp;author=A.%20Das&amp;author=J.%20Hugenholtz&amp;author=H.%20Van%20Halbeek&amp;author=L.%20G.%20Ljungdahl&amp;volume=171&amp;issue=11&amp;publication_year=1989&amp;pages=5823-5829&amp;pmid=2808299&amp;doi=10.1128/jb.171.11.5823-5829.1989&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0050">
<cite>

Davis, J. H.
, and Williamson J. R.. 2017. “Structure and Dynamics of Bacterial Ribosome Biogenesis.” Philosophical Transactions of the Royal Society B: Biological Sciences
372, no. 1716: 20160181. 10.1098/rstb.2016.0181.</cite> [<a href="https://doi.org/10.1098/rstb.2016.0181" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5311925/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28138067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Philosophical%20Transactions%20of%20the%20Royal%20Society%20B:%20Biological%20Sciences&amp;title=Structure%20and%20Dynamics%20of%20Bacterial%20Ribosome%20Biogenesis&amp;author=J.%20H.%20Davis&amp;author=J.%20R.%20Williamson&amp;volume=372&amp;issue=1716&amp;publication_year=2017&amp;pages=20160181&amp;pmid=28138067&amp;doi=10.1098/rstb.2016.0181&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0205">
<cite>

van Dijk, H. F. G.
, and Verbrugh H. A.. 2022. “Resisting Disinfectants.” Communications Medicine
2, no. 1: 1–5. 10.1038/s43856-021-00070-8.
</cite> [<a href="https://doi.org/10.1038/s43856-021-00070-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9053202/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35603291/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Communications%20Medicine&amp;title=Resisting%20Disinfectants&amp;author=H.%20F.%20G.%20van%20Dijk&amp;author=H.%20A.%20Verbrugh&amp;volume=2&amp;issue=1&amp;publication_year=2022&amp;pages=1-5&amp;pmid=35603291&amp;doi=10.1038/s43856-021-00070-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0051">
<cite>

Do, T.
, Page J. E., and Walker S.. 2020. “Uncovering the Activities, Biological Roles, and Regulation of Bacterial Cell Wall Hydrolases and Tailoring Enzymes.” The Journal of Biological Chemistry
295, no. 10: 3347. 10.1074/jbc.REV119.010155.
</cite> [<a href="https://doi.org/10.1074/jbc.REV119.010155" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7062177/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31974163/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Biological%20Chemistry&amp;title=Uncovering%20the%20Activities,%20Biological%20Roles,%20and%20Regulation%20of%20Bacterial%20Cell%20Wall%20Hydrolases%20and%20Tailoring%20Enzymes&amp;author=T.%20Do&amp;author=J.%20E.%20Page&amp;author=S.%20Walker&amp;volume=295&amp;issue=10&amp;publication_year=2020&amp;pages=3347&amp;pmid=31974163&amp;doi=10.1074/jbc.REV119.010155&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0052">
<cite>

Duong, T.
, Park K., Kim T., et al. 2014. “Structural and Functional Characterization of an Isd‐Type Haem‐Degradation Enzyme From <em>Listeria monocytogenes</em>
.” Acta Crystallographica, Section D, Biological Crystallography
70, no. Pt 3: 615–626. 10.1107/S1399004713030794.
</cite> [<a href="https://doi.org/10.1107/S1399004713030794" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24598731/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallographica,%20Section%20D,%20Biological%20Crystallography&amp;title=Structural%20and%20Functional%20Characterization%20of%20an%20Isd%E2%80%90Type%20Haem%E2%80%90Degradation%20Enzyme%20From%20Listeria%20monocytogenes&amp;author=T.%20Duong&amp;author=K.%20Park&amp;author=T.%20Kim&amp;volume=70&amp;issue=Pt%203&amp;publication_year=2014&amp;pages=615-626&amp;pmid=24598731&amp;doi=10.1107/S1399004713030794&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0053">
<cite>

Epelle, E. I.
, Macfarlane A., Cusack M., et al. 2023. “Ozone Application in Different Industries: A Review of Recent Developments.” Chemical Engineering Journal
454, no. February: 140188. 10.1016/j.cej.2022.140188.
</cite> [<a href="https://doi.org/10.1016/j.cej.2022.140188" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9637394/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36373160/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20Engineering%20Journal&amp;title=Ozone%20Application%20in%20Different%20Industries:%20A%20Review%20of%20Recent%20Developments&amp;author=E.%20I.%20Epelle&amp;author=A.%20Macfarlane&amp;author=M.%20Cusack&amp;volume=454&amp;issue=February&amp;publication_year=2023&amp;pages=140188&amp;pmid=36373160&amp;doi=10.1016/j.cej.2022.140188&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0054">
<cite>

Eshwar, A. K.
, Guldimann C., Oevermann A., and Tasara T.. 2017. “Cold‐Shock Domain Family Proteins (Csps) Are Involved in Regulation of Virulence, Cellular Aggregation, and Flagella‐Based Motility in <em>Listeria monocytogenes</em>
.” Frontiers in Cellular and Infection Microbiology
7, no. October: 453. 10.3389/fcimb.2017.00453.
</cite> [<a href="https://doi.org/10.3389/fcimb.2017.00453" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5662587/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29124040/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Cellular%20and%20Infection%20Microbiology&amp;title=Cold%E2%80%90Shock%20Domain%20Family%20Proteins%20(Csps)%20Are%20Involved%20in%20Regulation%20of%20Virulence,%20Cellular%20Aggregation,%20and%20Flagella%E2%80%90Based%20Motility%20in%20Listeria%20monocytogenes&amp;author=A.%20K.%20Eshwar&amp;author=C.%20Guldimann&amp;author=A.%20Oevermann&amp;author=T.%20Tasara&amp;volume=7&amp;issue=October&amp;publication_year=2017&amp;pages=453&amp;pmid=29124040&amp;doi=10.3389/fcimb.2017.00453&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0055">
<cite>

Faller, M.
, Matsunaga M., Yin S., Loo J. A., and Guo F.. 2007. “Heme Is Involved in microRNA Processing.” Nature Structural &amp; Molecular Biology
14, no. 1: 23–29. 10.1038/nsmb1182.</cite> [<a href="https://doi.org/10.1038/nsmb1182" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17159994/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Structural%20&amp;%20Molecular%20Biology&amp;title=Heme%20Is%20Involved%20in%20microRNA%20Processing&amp;author=M.%20Faller&amp;author=M.%20Matsunaga&amp;author=S.%20Yin&amp;author=J.%20A.%20Loo&amp;author=F.%20Guo&amp;volume=14&amp;issue=1&amp;publication_year=2007&amp;pages=23-29&amp;pmid=17159994&amp;doi=10.1038/nsmb1182&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0056">
<cite>

Fatima, Z.
, Purkait D., Rehman S., Rai S., and Hameed S.. 2023. “Chapter 11 ‐ Multidrug Resistance: A Threat to Antibiotic Era.” In Biological and Environmental Hazards, Risks, and Disasters (Second Edition), edited by Sivanpillai Ramesh and Shroder John F., 197–220. Hazards and Disasters Series. Elsevier. 10.1016/B978-0-12-820509-9.00014-9.</cite> [<a href="https://doi.org/10.1016/B978-0-12-820509-9.00014-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Biological%20and%20Environmental%20Hazards,%20Risks,%20and%20Disasters&amp;author=Z.%20Fatima&amp;author=D.%20Purkait&amp;author=S.%20Rehman&amp;author=S.%20Rai&amp;author=S.%20Hameed&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0057">
<cite>

Feehily, C.
, and Karatzas K. A. G.. 2013. “Role of Glutamate Metabolism in Bacterial Responses towards Acid and Other Stresses.” Journal of Applied Microbiology
114, no. 1: 11–24. 10.1111/j.1365-2672.2012.05434.x.
</cite> [<a href="https://doi.org/10.1111/j.1365-2672.2012.05434.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22924898/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Applied%20Microbiology&amp;title=Role%20of%20Glutamate%20Metabolism%20in%20Bacterial%20Responses%20towards%20Acid%20and%20Other%20Stresses&amp;author=C.%20Feehily&amp;author=K.%20A.%20G.%20Karatzas&amp;volume=114&amp;issue=1&amp;publication_year=2013&amp;pages=11-24&amp;pmid=22924898&amp;doi=10.1111/j.1365-2672.2012.05434.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0058">
<cite>

Fernández‐Gómez, P.
, Cobo‐Díaz J. F., Oliveira M., et al. 2023. “Susceptibility and Transcriptomic Response to Plasma‐Activated Water of <em>Listeria monocytogenes</em> Planktonic and Sessile Cells.” Food Microbiology
113, no. August: 104252. 10.1016/j.fm.2023.104252.
</cite> [<a href="https://doi.org/10.1016/j.fm.2023.104252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37098419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Microbiology&amp;title=Susceptibility%20and%20Transcriptomic%20Response%20to%20Plasma%E2%80%90Activated%20Water%20of%20Listeria%20monocytogenes%20Planktonic%20and%20Sessile%20Cells&amp;author=P.%20Fern%C3%A1ndez%E2%80%90G%C3%B3mez&amp;author=J.%20F.%20Cobo%E2%80%90D%C3%ADaz&amp;author=M.%20Oliveira&amp;volume=113&amp;issue=August&amp;publication_year=2023&amp;pages=104252&amp;pmid=37098419&amp;doi=10.1016/j.fm.2023.104252&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0253">
<cite>

Ferreira, A.
, Sue D., O'Byrne C. P., and Boor K. J.. 2003. “Role of Listeria Monocytogenes s B in Survival of Lethal Acidic Conditions and in the Acquired Acid Tolerance Response.” Applied and Environmental Microbiology
69, no. 5: 2692–2698. 10.1128/AEM.69.5.2692-2698.2003.
</cite> [<a href="https://doi.org/10.1128/AEM.69.5.2692-2698.2003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC154505/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12732538/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=Role%20of%20Listeria%20Monocytogenes%20s%20B%20in%20Survival%20of%20Lethal%20Acidic%20Conditions%20and%20in%20the%20Acquired%20Acid%20Tolerance%20Response&amp;author=A.%20Ferreira&amp;author=D.%20Sue&amp;author=C.%20P.%20O'Byrne&amp;author=K.%20J.%20Boor&amp;volume=69&amp;issue=5&amp;publication_year=2003&amp;pages=2692-2698&amp;pmid=12732538&amp;doi=10.1128/AEM.69.5.2692-2698.2003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0060">
<cite>

Fox, E. M.
, Leonard N., and Jordan K.. 2011. “Physiological and Transcriptional Characterization of Persistent and Nonpersistent <em>Listeria monocytogenes</em> Isolates.” Applied and Environmental Microbiology
77, no. 18: 6559–6569. 10.1128/AEM.05529-11.
</cite> [<a href="https://doi.org/10.1128/AEM.05529-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3187160/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21764947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=Physiological%20and%20Transcriptional%20Characterization%20of%20Persistent%20and%20Nonpersistent%20Listeria%20monocytogenes%20Isolates&amp;author=E.%20M.%20Fox&amp;author=N.%20Leonard&amp;author=K.%20Jordan&amp;volume=77&amp;issue=18&amp;publication_year=2011&amp;pages=6559-6569&amp;pmid=21764947&amp;doi=10.1128/AEM.05529-11&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0061">
<cite>

França, A.
, Carvalhais V., Maira‐Litrán T., Vilanova M., Cerca N., and Pier G.. 2014. “Alterations in the <em>Staphylococcus epidermidis</em> Biofilm Transcriptome Following Interaction With Whole Human Blood.” Pathogens and Disease
70, no. 3: 444–448. 10.1111/2049-632X.12130.
</cite> [<a href="https://doi.org/10.1111/2049-632X.12130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24391077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pathogens%20and%20Disease&amp;title=Alterations%20in%20the%20Staphylococcus%20epidermidis%20Biofilm%20Transcriptome%20Following%20Interaction%20With%20Whole%20Human%20Blood&amp;author=A.%20Fran%C3%A7a&amp;author=V.%20Carvalhais&amp;author=T.%20Maira%E2%80%90Litr%C3%A1n&amp;author=M.%20Vilanova&amp;author=N.%20Cerca&amp;volume=70&amp;issue=3&amp;publication_year=2014&amp;pages=444-448&amp;pmid=24391077&amp;doi=10.1111/2049-632X.12130&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0248">
<cite>

Frees, D.
, Chastanet A., Qazi S., et al. 2004. “Clp ATPases Are Required for Stress Tolerance, Intracellular Replication and Biofilm Formation in Staphylococcus Aureus.” Molecular Microbiology
54, no. 5: 1445–1462. 10.1111/j.1365-2958.2004.04368.x.
</cite> [<a href="https://doi.org/10.1111/j.1365-2958.2004.04368.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15554981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Microbiology&amp;title=Clp%20ATPases%20Are%20Required%20for%20Stress%20Tolerance,%20Intracellular%20Replication%20and%20Biofilm%20Formation%20in%20Staphylococcus%20Aureus&amp;author=D.%20Frees&amp;author=A.%20Chastanet&amp;author=S.%20Qazi&amp;volume=54&amp;issue=5&amp;publication_year=2004&amp;pages=1445-1462&amp;pmid=15554981&amp;doi=10.1111/j.1365-2958.2004.04368.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0062">
<cite>

Fuchs, T. M.
, Eisenreich W., Kern T., and Dandekar T.. 2012. “Toward a Systemic Understanding of <em>Listeria monocytogenes</em> Metabolism During Infection.” Frontiers in Microbiology
3, no. February: 23. 10.3389/fmicb.2012.00023.</cite> [<a href="https://doi.org/10.3389/fmicb.2012.00023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3271275/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22347216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Toward%20a%20Systemic%20Understanding%20of%20Listeria%20monocytogenes%20Metabolism%20During%20Infection&amp;author=T.%20M.%20Fuchs&amp;author=W.%20Eisenreich&amp;author=T.%20Kern&amp;author=T.%20Dandekar&amp;volume=3&amp;issue=February&amp;publication_year=2012&amp;pmid=22347216&amp;doi=10.3389/fmicb.2012.00023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0063">
<cite>

Fuhrmann, J. J.

2021. “3 ‐ Microbial Metabolism.” In Principles and Applications of Soil Microbiology (Third Edition), edited by Gentry Terry J., Fuhrmann Jeffry J., and Zuberer David A., 57–87. Elsevier. 10.1016/B978-0-12-820202-9.00003-4.</cite> [<a href="https://doi.org/10.1016/B978-0-12-820202-9.00003-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Principles%20and%20Applications%20of%20Soil%20Microbiology&amp;author=J.%20J.%20Fuhrmann&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0064">
<cite>

Fukuzaki, S.

2006. “Mechanisms of Actions of Sodium Hypochlorite in Cleaning and Disinfection Processes.” Biocontrol Science
11, no. 4: 147–157. 10.4265/bio.11.147.
</cite> [<a href="https://doi.org/10.4265/bio.11.147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17190269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biocontrol%20Science&amp;title=Mechanisms%20of%20Actions%20of%20Sodium%20Hypochlorite%20in%20Cleaning%20and%20Disinfection%20Processes&amp;author=S.%20Fukuzaki&amp;volume=11&amp;issue=4&amp;publication_year=2006&amp;pages=147-157&amp;pmid=17190269&amp;doi=10.4265/bio.11.147&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0065">
<cite>

Gélinas, P.
, Goulet J., Tastayre G. M., and Picard G. A.. 1984. “Effect of Temperature and Contact Time on the Activity of Eight Disinfectants—A Classification.” Journal of Food Protection
47, no. 11: 841–847. 10.4315/0362-028X-47.11.841.
</cite> [<a href="https://doi.org/10.4315/0362-028X-47.11.841" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30934436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Food%20Protection&amp;title=Effect%20of%20Temperature%20and%20Contact%20Time%20on%20the%20Activity%20of%20Eight%20Disinfectants%E2%80%94A%20Classification&amp;author=P.%20G%C3%A9linas&amp;author=J.%20Goulet&amp;author=G.%20M.%20Tastayre&amp;author=G.%20A.%20Picard&amp;volume=47&amp;issue=11&amp;publication_year=1984&amp;pages=841-847&amp;pmid=30934436&amp;doi=10.4315/0362-028X-47.11.841&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0066">
<cite>

Gérard, A.
, El‐Hajjaji S., Van Coillie E., Bentaïb A., Daube G., and Sindic M.. 2020. “Determination of the Growth Potential of <em>Listeria monocytogenes</em> in Various Types of Belgian Artisanal Cheeses by Challenge Tests.” Food Microbiology
92, no. December: 103582. 10.1016/j.fm.2020.103582.
</cite> [<a href="https://doi.org/10.1016/j.fm.2020.103582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32950166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Microbiology&amp;title=Determination%20of%20the%20Growth%20Potential%20of%20Listeria%20monocytogenes%20in%20Various%20Types%20of%20Belgian%20Artisanal%20Cheeses%20by%20Challenge%20Tests&amp;author=A.%20G%C3%A9rard&amp;author=S.%20El%E2%80%90Hajjaji&amp;author=E.%20Van%20Coillie&amp;author=A.%20Benta%C3%AFb&amp;author=G.%20Daube&amp;volume=92&amp;issue=December&amp;publication_year=2020&amp;pages=103582&amp;pmid=32950166&amp;doi=10.1016/j.fm.2020.103582&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0067">
<cite>

Glaser, P.
, Frangeul L., Buchrieser C., et al. 2001. “Comparative Genomics of Listeria Species.” Science (New York, N.Y.)
294, no. 5543: 849–852. 10.1126/science.1063447.
</cite> [<a href="https://doi.org/10.1126/science.1063447" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11679669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20(New%20York,%20N.Y.)&amp;title=Comparative%20Genomics%20of%20Listeria%20Species&amp;author=P.%20Glaser&amp;author=L.%20Frangeul&amp;author=C.%20Buchrieser&amp;volume=294&amp;issue=5543&amp;publication_year=2001&amp;pages=849-852&amp;pmid=11679669&amp;doi=10.1126/science.1063447&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0068">
<cite>

Goncheva, M. I.
, Chin D., and Heinrichs D. E.. 2022. “Nucleotide Biosynthesis: The Base of Bacterial Pathogenesis.” Trends in Microbiology
30, no. 8: 793–804. 10.1016/j.tim.2021.12.007.
</cite> [<a href="https://doi.org/10.1016/j.tim.2021.12.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35074276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Microbiology&amp;title=Nucleotide%20Biosynthesis:%20The%20Base%20of%20Bacterial%20Pathogenesis&amp;author=M.%20I.%20Goncheva&amp;author=D.%20Chin&amp;author=D.%20E.%20Heinrichs&amp;volume=30&amp;issue=8&amp;publication_year=2022&amp;pages=793-804&amp;pmid=35074276&amp;doi=10.1016/j.tim.2021.12.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0069">
<cite>

Gongora, K.
, Vankerschaver J., Sampers I., and Van Haute S.. 2024. “The Influence of pH on the Efficacy of Oxidation‐Reduction Potential (ORP) to Predict Chlorine Disinfection of Surrogate Bacteria, <em>Escherichia coli</em> O157:H7 and <em>Listeria monocytogenes</em> in Oxidant Demand Free Conditions and Fresh Produce Wash Water.” Food Microbiology
121, no. August: 104516. 10.1016/j.fm.2024.104516.
</cite> [<a href="https://doi.org/10.1016/j.fm.2024.104516" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38637078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Microbiology&amp;title=The%20Influence%20of%20pH%20on%20the%20Efficacy%20of%20Oxidation%E2%80%90Reduction%20Potential%20(ORP)%20to%20Predict%20Chlorine%20Disinfection%20of%20Surrogate%20Bacteria,%20Escherichia%20coli%20O157:H7%20and%20Listeria%20monocytogenes%20in%20Oxidant%20Demand%20Free%20Conditions%20and%20Fresh%20Produce%20Wash%20Water&amp;author=K.%20Gongora&amp;author=J.%20Vankerschaver&amp;author=I.%20Sampers&amp;author=S.%20Van%20Haute&amp;volume=121&amp;issue=August&amp;publication_year=2024&amp;pages=104516&amp;pmid=38637078&amp;doi=10.1016/j.fm.2024.104516&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0070">
<cite>

Griffiths, H. R.

2005. “Chemical Modifications of Biomolecules by Oxidants.” In Reactions, Processes: Oxidants and Antioxidant Defense Systems, edited by Grune Tilman, 33–62. Springer. 10.1007/b101145.</cite> [<a href="https://doi.org/10.1007/b101145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Reactions,%20Processes:%20Oxidants%20and%20Antioxidant%20Defense%20Systems&amp;author=H.%20R.%20Griffiths&amp;publication_year=2005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0071">
<cite>

Guerreiro, D. N.
, Arcari T., and O'Byrne C. P.. 2020. “The σB‐Mediated General Stress Response of <em>Listeria monocytogenes</em>: Life and Death Decision Making in a Pathogen.” Frontiers in Microbiology
11: 1505. 10.3389/fmicb.2020.01505.
</cite> [<a href="https://doi.org/10.3389/fmicb.2020.01505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7358398/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32733414/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=The%20%CF%83B%E2%80%90Mediated%20General%20Stress%20Response%20of%20Listeria%20monocytogenes:%20Life%20and%20Death%20Decision%20Making%20in%20a%20Pathogen&amp;author=D.%20N.%20Guerreiro&amp;author=T.%20Arcari&amp;author=C.%20P.%20O'Byrne&amp;volume=11&amp;publication_year=2020&amp;pages=1505&amp;pmid=32733414&amp;doi=10.3389/fmicb.2020.01505&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0072">
<cite>

Guo, H.
, Suzuki T., and Rubinstein J. L.. 2019. “Structure of a Bacterial ATP Synthase.” Elife
8: e43128. 10.7554/eLife.43128.
</cite> [<a href="https://doi.org/10.7554/eLife.43128" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6377231/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30724163/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=Structure%20of%20a%20Bacterial%20ATP%20Synthase&amp;author=H.%20Guo&amp;author=T.%20Suzuki&amp;author=J.%20L.%20Rubinstein&amp;volume=8&amp;publication_year=2019&amp;pages=e43128&amp;pmid=30724163&amp;doi=10.7554/eLife.43128&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0073">
<cite>

Han, H.
, Wang Z., Li T., et al. 2021. “Recent Progress of Bacterial FtsZ Inhibitors With a Focus on Peptides.” The FEBS Journal
288, no. 4: 1091–1106. 10.1111/febs.15489.
</cite> [<a href="https://doi.org/10.1111/febs.15489" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32681661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20FEBS%20Journal&amp;title=Recent%20Progress%20of%20Bacterial%20FtsZ%20Inhibitors%20With%20a%20Focus%20on%20Peptides&amp;author=H.%20Han&amp;author=Z.%20Wang&amp;author=T.%20Li&amp;volume=288&amp;issue=4&amp;publication_year=2021&amp;pages=1091-1106&amp;pmid=32681661&amp;doi=10.1111/febs.15489&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0074">
<cite>

Harter, E.
, Wagner E. M., Zaiser A., Halecker S., Wagner M., and Rychli K.. 2017. “Stress Survival Islet 2, Predominantly Present in <em>Listeria monocytogenes</em> Strains of Sequence Type 121, Is Involved in the Alkaline and Oxidative Stress Responses.” Applied and Environmental Microbiology
83, no. 16: e00827–17. 10.1128/AEM.00827-17.
</cite> [<a href="https://doi.org/10.1128/AEM.00827-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5541211/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28625982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=Stress%20Survival%20Islet%202,%20Predominantly%20Present%20in%20Listeria%20monocytogenes%20Strains%20of%20Sequence%20Type%20121,%20Is%20Involved%20in%20the%20Alkaline%20and%20Oxidative%20Stress%20Responses&amp;author=E.%20Harter&amp;author=E.%20M.%20Wagner&amp;author=A.%20Zaiser&amp;author=S.%20Halecker&amp;author=M.%20Wagner&amp;volume=83&amp;issue=16&amp;publication_year=2017&amp;pages=e00827-17&amp;pmid=28625982&amp;doi=10.1128/AEM.00827-17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0075">
<cite>

Hasnat, M. A.
, Zupok A., Gorka M., et al. 2024. “Iron Limitation Indirectly Reduces the <em>Escherichia coli</em> torCAD Operon Expression by a Reduction of Molybdenum Cofactor Availability.” Microbiology Spectrum
12, no. 2: e03480–23. 10.1128/spectrum.03480-23.
</cite> [<a href="https://doi.org/10.1128/spectrum.03480-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10845959/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38193660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology%20Spectrum&amp;title=Iron%20Limitation%20Indirectly%20Reduces%20the%20Escherichia%20coli%20torCAD%20Operon%20Expression%20by%20a%20Reduction%20of%20Molybdenum%20Cofactor%20Availability&amp;author=M.%20A.%20Hasnat&amp;author=A.%20Zupok&amp;author=M.%20Gorka&amp;volume=12&amp;issue=2&amp;publication_year=2024&amp;pages=e03480-23&amp;pmid=38193660&amp;doi=10.1128/spectrum.03480-23&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0076">
<cite>

He, Y.
, Xu T., Li S., et al. 2022. “Integrative Assessment of Reduced <em>Listeria monocytogenes</em> Susceptibility to Benzalkonium Chloride in Produce Processing Environments.” Applied and Environmental Microbiology
88, no. 21: e01269–22. 10.1128/aem.01269-22.
</cite> [<a href="https://doi.org/10.1128/aem.01269-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9642021/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36226965/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=Integrative%20Assessment%20of%20Reduced%20Listeria%20monocytogenes%20Susceptibility%20to%20Benzalkonium%20Chloride%20in%20Produce%20Processing%20Environments&amp;author=Y.%20He&amp;author=T.%20Xu&amp;author=S.%20Li&amp;volume=88&amp;issue=21&amp;publication_year=2022&amp;pages=e01269-22&amp;pmid=36226965&amp;doi=10.1128/aem.01269-22&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0077">
<cite>

Heberle, H.
, Vaz Meirelles G., da Silva F. R., Telles G. P., and Minghim R.. 2015. “InteractiVenn: A Web‐Based Tool for the Analysis of Sets Through Venn Diagrams.” BMC Bioinformatics [Electronic Resource]
16, no. 1: 169. 10.1186/s12859-015-0611-3.
</cite> [<a href="https://doi.org/10.1186/s12859-015-0611-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4455604/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25994840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics%20%5BElectronic%20Resource%5D&amp;title=InteractiVenn:%20A%20Web%E2%80%90Based%20Tool%20for%20the%20Analysis%20of%20Sets%20Through%20Venn%20Diagrams&amp;author=H.%20Heberle&amp;author=G.%20Vaz%20Meirelles&amp;author=F.%20R.%20da%20Silva&amp;author=G.%20P.%20Telles&amp;author=R.%20Minghim&amp;volume=16&amp;issue=1&amp;publication_year=2015&amp;pages=169&amp;pmid=25994840&amp;doi=10.1186/s12859-015-0611-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0078">
<cite>

Hiratsuka, T.
, Furihata K., Ishikawa J., et al. 2008. “An Alternative Menaquinone Biosynthetic Pathway Operating in Microorganisms.” Science
321, no. 5896: 1670–1673.
</cite> [<a href="https://doi.org/10.1126/science.1160446" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18801996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=An%20Alternative%20Menaquinone%20Biosynthetic%20Pathway%20Operating%20in%20Microorganisms&amp;author=T.%20Hiratsuka&amp;author=K.%20Furihata&amp;author=J.%20Ishikawa&amp;volume=321&amp;issue=5896&amp;publication_year=2008&amp;pages=1670-1673&amp;pmid=18801996&amp;doi=10.1126/science.1160446&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0079">
<cite>

Hosie, A. H.
, and Poole P. S.. 2001. “Bacterial ABC Transporters of Amino Acids.” Research in Microbiology
152, no. 3–4: 259–270. 10.1016/s0923-2508(01)01197-4.
</cite> [<a href="https://doi.org/10.1016/s0923-2508(01)01197-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11421273/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Research%20in%20Microbiology&amp;title=Bacterial%20ABC%20Transporters%20of%20Amino%20Acids&amp;author=A.%20H.%20Hosie&amp;author=P.%20S.%20Poole&amp;volume=152&amp;issue=3%E2%80%934&amp;publication_year=2001&amp;pages=259-270&amp;pmid=11421273&amp;doi=10.1016/s0923-2508(01)01197-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0080">
<cite>

Imlay, J. A.

2003. “Pathways of Oxidative Damage.” Annual Review of Microbiology
57, no. 1: 395–418. 10.1146/annurev.micro.57.030502.090938.</cite> [<a href="https://doi.org/10.1146/annurev.micro.57.030502.090938" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14527285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Microbiology&amp;title=Pathways%20of%20Oxidative%20Damage&amp;author=J.%20A.%20Imlay&amp;volume=57&amp;issue=1&amp;publication_year=2003&amp;pages=395-418&amp;pmid=14527285&amp;doi=10.1146/annurev.micro.57.030502.090938&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0081">
<cite>

Impens, F.
, Rolhion N., Radoshevich L., et al. 2017. “N‐Terminomics Identifies Prli42 as a Membrane Miniprotein Conserved in Firmicutes and Critical for Stressosome Activation in <em>Listeria monocytogenes</em>
.” Nature Microbiology
2, no. February: 17005. 10.1038/nmicrobiol.2017.5.</cite> [<a href="https://doi.org/10.1038/nmicrobiol.2017.5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5802382/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28191904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Microbiology&amp;title=N%E2%80%90Terminomics%20Identifies%20Prli42%20as%20a%20Membrane%20Miniprotein%20Conserved%20in%20Firmicutes%20and%20Critical%20for%20Stressosome%20Activation%20in%20Listeria%20monocytogenes&amp;author=F.%20Impens&amp;author=N.%20Rolhion&amp;author=L.%20Radoshevich&amp;volume=2&amp;issue=February&amp;publication_year=2017&amp;pages=17005&amp;pmid=28191904&amp;doi=10.1038/nmicrobiol.2017.5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0082"><cite>

Ingram, S. L.

2018. “Regulationof Oxidoreductase Enzymes During Inflammation”. A Thesis Submitted to theUniversity of Brighton and the University of Sussex for a Programme of StudyUndertaken at the Brighton and Sussex Medical School for the Degree of Doctorof Philosophy.</cite></li>
<li id="crf370260-bib-0083">
<cite>

Ismail, A. S.
, Valastyan J. S., and Bassler B. L.. 2016. “A Host‐Produced Autoinducer‐2 Mimic Activates Bacterial Quorum Sensing.” Cell Host &amp; Microbe
19, no. 4: 470–480. 10.1016/j.chom.2016.02.020.
</cite> [<a href="https://doi.org/10.1016/j.chom.2016.02.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4869860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26996306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Host%20&amp;%20Microbe&amp;title=A%20Host%E2%80%90Produced%20Autoinducer%E2%80%902%20Mimic%20Activates%20Bacterial%20Quorum%20Sensing&amp;author=A.%20S.%20Ismail&amp;author=J.%20S.%20Valastyan&amp;author=B.%20L.%20Bassler&amp;volume=19&amp;issue=4&amp;publication_year=2016&amp;pages=470-480&amp;pmid=26996306&amp;doi=10.1016/j.chom.2016.02.020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0084">
<cite>

Ivy, R. A.
, Chan Y. C., Bowen B. M., Boor K. J., and Wiedmann M.. 2010. “Growth Temperature‐Dependent Contributions of Response Regulators, σB, PrfA, and Motility Factors to <em>Listeria monocytogenes</em> Invasion of Caco‐2 Cells.” Foodborne Pathogens and Disease
7, no. 11: 1337–1349. 10.1089/fpd.2010.0563.
</cite> [<a href="https://doi.org/10.1089/fpd.2010.0563" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3079390/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20707735/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Foodborne%20Pathogens%20and%20Disease&amp;title=Growth%20Temperature%E2%80%90Dependent%20Contributions%20of%20Response%20Regulators,%20%CF%83B,%20PrfA,%20and%20Motility%20Factors%20to%20Listeria%20monocytogenes%20Invasion%20of%20Caco%E2%80%902%20Cells&amp;author=R.%20A.%20Ivy&amp;author=Y.%20C.%20Chan&amp;author=B.%20M.%20Bowen&amp;author=K.%20J.%20Boor&amp;author=M.%20Wiedmann&amp;volume=7&amp;issue=11&amp;publication_year=2010&amp;pages=1337-1349&amp;pmid=20707735&amp;doi=10.1089/fpd.2010.0563&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0085">
<cite>

Jakubovics, N. S.
, and Jenkinson H. F.. 2001. “Out of the Iron Age: New Insights Into the Critical Role of Manganese Homeostasis in Bacteria.” Microbiology (Reading, England)
147, no. 7: 1709–1718. 10.1099/00221287-147-7-1709.
</cite> [<a href="https://doi.org/10.1099/00221287-147-7-1709" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11429449/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology%20(Reading,%20England)&amp;title=Out%20of%20the%20Iron%20Age:%20New%20Insights%20Into%20the%20Critical%20Role%20of%20Manganese%20Homeostasis%20in%20Bacteria&amp;author=N.%20S.%20Jakubovics&amp;author=H.%20F.%20Jenkinson&amp;volume=147&amp;issue=7&amp;publication_year=2001&amp;pages=1709-1718&amp;pmid=11429449&amp;doi=10.1099/00221287-147-7-1709&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0086">
<cite>

Jiang, X.
, Yu T., Liang Y., et al. 2016. “Efflux Pump‐Mediated Benzalkonium Chloride Resistance in <em>Listeria monocytogenes</em> Isolated From Retail Food.” International Journal of Food Microbiology
217, no. January: 141–145. 10.1016/j.ijfoodmicro.2015.10.022.
</cite> [<a href="https://doi.org/10.1016/j.ijfoodmicro.2015.10.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26513255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Food%20Microbiology&amp;title=Efflux%20Pump%E2%80%90Mediated%20Benzalkonium%20Chloride%20Resistance%20in%20Listeria%20monocytogenes%20Isolated%20From%20Retail%20Food&amp;author=X.%20Jiang&amp;author=T.%20Yu&amp;author=Y.%20Liang&amp;volume=217&amp;issue=January&amp;publication_year=2016&amp;pages=141-145&amp;pmid=26513255&amp;doi=10.1016/j.ijfoodmicro.2015.10.022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0087">
<cite>

Jin, M.
, Liu L., Wang D.‐n., et al. 2020. “Chlorine Disinfection Promotes the Exchange of Antibiotic Resistance Genes Across Bacterial Genera by Natural Transformation.” The ISME Journal
14, no. 7: 1847–1856. 10.1038/s41396-020-0656-9.
</cite> [<a href="https://doi.org/10.1038/s41396-020-0656-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7305130/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32327733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20ISME%20Journal&amp;title=Chlorine%20Disinfection%20Promotes%20the%20Exchange%20of%20Antibiotic%20Resistance%20Genes%20Across%20Bacterial%20Genera%20by%20Natural%20Transformation&amp;author=M.%20Jin&amp;author=L.%20Liu&amp;author=D.%E2%80%90n.%20Wang&amp;volume=14&amp;issue=7&amp;publication_year=2020&amp;pages=1847-1856&amp;pmid=32327733&amp;doi=10.1038/s41396-020-0656-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0088">
<cite>

Jirešová, J.
, Scholtz V., Julák J., and Šerá B.. 2022. “Comparison of the Effect of Plasma‐Activated Water and Artificially Prepared Plasma‐Activated Water on Wheat Grain Properties.” Plants
11, no. 11: 1471. 10.3390/plants11111471.
</cite> [<a href="https://doi.org/10.3390/plants11111471" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9183031/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35684244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Comparison%20of%20the%20Effect%20of%20Plasma%E2%80%90Activated%20Water%20and%20Artificially%20Prepared%20Plasma%E2%80%90Activated%20Water%20on%20Wheat%20Grain%20Properties&amp;author=J.%20Jire%C5%A1ov%C3%A1&amp;author=V.%20Scholtz&amp;author=J.%20Jul%C3%A1k&amp;author=B.%20%C5%A0er%C3%A1&amp;volume=11&amp;issue=11&amp;publication_year=2022&amp;pages=1471&amp;pmid=35684244&amp;doi=10.3390/plants11111471&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0089">
<cite>

Johansson, J.
, Mandin P., Renzoni A., Chiaruttini C., Springer M., and Cossart P.. 2002. “An RNA Thermosensor Controls Expression of Virulence Genes in <em>Listeria monocytogenes</em>
.” Cell
110, no. 5: 551–561. 10.1016/s0092-8674(02)00905-4.
</cite> [<a href="https://doi.org/10.1016/s0092-8674(02)00905-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12230973/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=An%20RNA%20Thermosensor%20Controls%20Expression%20of%20Virulence%20Genes%20in%20Listeria%20monocytogenes&amp;author=J.%20Johansson&amp;author=P.%20Mandin&amp;author=A.%20Renzoni&amp;author=C.%20Chiaruttini&amp;author=M.%20Springer&amp;volume=110&amp;issue=5&amp;publication_year=2002&amp;pages=551-561&amp;pmid=12230973&amp;doi=10.1016/s0092-8674(02)00905-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0258">
<cite>

Johansson, J.
, and Freitag N. E.. 2019. “Regulation of Listeria Monocytogenes Virulence.” Microbiology Spectrum
7, no. 4: 10.1128/microbiolspec.gpp3-0064-2019.</cite> [<a href="https://doi.org/10.1128/microbiolspec.gpp3-0064-2019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10957223/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31441398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology%20Spectrum&amp;title=Regulation%20of%20Listeria%20Monocytogenes%20Virulence&amp;author=J.%20Johansson&amp;author=N.%20E.%20Freitag&amp;volume=7&amp;issue=4&amp;publication_year=2019&amp;pmid=31441398&amp;doi=10.1128/microbiolspec.gpp3-0064-2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0090">
<cite>

Joseph, B.
, Mertins S., Stoll R., et al. 2008. “Glycerol Metabolism and PrfA Activity in <em>Listeria monocytogenes</em>
.” Journal of Bacteriology
190, no. 15: 5412–5430. 10.1128/JB.00259-08.
</cite> [<a href="https://doi.org/10.1128/JB.00259-08" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2493282/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18502850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Bacteriology&amp;title=Glycerol%20Metabolism%20and%20PrfA%20Activity%20in%20Listeria%20monocytogenes&amp;author=B.%20Joseph&amp;author=S.%20Mertins&amp;author=R.%20Stoll&amp;volume=190&amp;issue=15&amp;publication_year=2008&amp;pages=5412-5430&amp;pmid=18502850&amp;doi=10.1128/JB.00259-08&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0091">
<cite>

Joseph, B.
, Przybilla K., Stühler C., et al. 2006. “Identification of <em>Listeria monocytogenes</em> Genes Contributing to Intracellular Replication by Expression Profiling and Mutant Screening.” Journal of Bacteriology
188, no. 2: 556–568. 10.1128/jb.188.2.556-568.2006.
</cite> [<a href="https://doi.org/10.1128/jb.188.2.556-568.2006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1347271/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16385046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Bacteriology&amp;title=Identification%20of%20Listeria%20monocytogenes%20Genes%20Contributing%20to%20Intracellular%20Replication%20by%20Expression%20Profiling%20and%20Mutant%20Screening&amp;author=B.%20Joseph&amp;author=K.%20Przybilla&amp;author=C.%20St%C3%BChler&amp;volume=188&amp;issue=2&amp;publication_year=2006&amp;pages=556-568&amp;pmid=16385046&amp;doi=10.1128/jb.188.2.556-568.2006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0092">
<cite>

Joshi, S. G.
, Cooper M., Yost A., et al. 2011. “Nonthermal Dielectric‐Barrier Discharge Plasma‐Induced Inactivation Involves Oxidative DNA Damage and Membrane Lipid Peroxidation in <em>Escherichia coli</em>
.” Antimicrobial Agents and Chemotherapy
55, no. 3: 1053–1062. 10.1128/aac.01002-10.
</cite> [<a href="https://doi.org/10.1128/aac.01002-10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3067084/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21199923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antimicrobial%20Agents%20and%20Chemotherapy&amp;title=Nonthermal%20Dielectric%E2%80%90Barrier%20Discharge%20Plasma%E2%80%90Induced%20Inactivation%20Involves%20Oxidative%20DNA%20Damage%20and%20Membrane%20Lipid%20Peroxidation%20in%20Escherichia%20coli&amp;author=S.%20G.%20Joshi&amp;author=M.%20Cooper&amp;author=A.%20Yost&amp;volume=55&amp;issue=3&amp;publication_year=2011&amp;pages=1053-1062&amp;pmid=21199923&amp;doi=10.1128/aac.01002-10&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0093">
<cite>

Juttukonda, L. J.
, and Skaar E. P.. 2015. “Manganese Homeostasis and Utilization in Pathogenic Bacteria.” Molecular Microbiology
97, no. 2: 216–228. 10.1111/mmi.13034.
</cite> [<a href="https://doi.org/10.1111/mmi.13034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4631260/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25898914/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Microbiology&amp;title=Manganese%20Homeostasis%20and%20Utilization%20in%20Pathogenic%20Bacteria&amp;author=L.%20J.%20Juttukonda&amp;author=E.%20P.%20Skaar&amp;volume=97&amp;issue=2&amp;publication_year=2015&amp;pages=216-228&amp;pmid=25898914&amp;doi=10.1111/mmi.13034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0094">
<cite>

Jyung, S.
, Kang J.‐W., and Kang D.‐H.. 2023. “Inactivation of <em>Listeria monocytogenes</em> Through the Synergistic Interaction Between Plasma‐Activated Water and Organic Acid.” Food Research International
167, no. May: 112687. 10.1016/j.foodres.2023.112687.
</cite> [<a href="https://doi.org/10.1016/j.foodres.2023.112687" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37087257/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Research%20International&amp;title=Inactivation%20of%20Listeria%20monocytogenes%20Through%20the%20Synergistic%20Interaction%20Between%20Plasma%E2%80%90Activated%20Water%20and%20Organic%20Acid&amp;author=S.%20Jyung&amp;author=J.%E2%80%90W.%20Kang&amp;author=D.%E2%80%90H.%20Kang&amp;volume=167&amp;issue=May&amp;publication_year=2023&amp;pages=112687&amp;pmid=37087257&amp;doi=10.1016/j.foodres.2023.112687&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0095">
<cite>

Kamp, H. D.
, and Higgins D. E.. 2009. “Transcriptional and Post‐Transcriptional Regulation of the GmaR Antirepressor Governs Temperature‐Dependent Control of Flagellar Motility in <em>Listeria monocytogenes</em>
.” Molecular Microbiology
74, no. 2: 421–435. 10.1111/j.1365-2958.2009.06874.x.
</cite> [<a href="https://doi.org/10.1111/j.1365-2958.2009.06874.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2807129/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19796338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Microbiology&amp;title=Transcriptional%20and%20Post%E2%80%90Transcriptional%20Regulation%20of%20the%20GmaR%20Antirepressor%20Governs%20Temperature%E2%80%90Dependent%20Control%20of%20Flagellar%20Motility%20in%20Listeria%20monocytogenes&amp;author=H.%20D.%20Kamp&amp;author=D.%20E.%20Higgins&amp;volume=74&amp;issue=2&amp;publication_year=2009&amp;pages=421-435&amp;pmid=19796338&amp;doi=10.1111/j.1365-2958.2009.06874.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0096">
<cite>

Kamp, H. D.
, and Higgins D. E.. 2011. “A Protein Thermometer Controls Temperature‐Dependent Transcription of Flagellar Motility Genes in <em>Listeria monocytogenes</em>
.” PLoS Pathogens
7, no. 8: e1002153. 10.1371/journal.ppat.1002153.
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1002153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3150276/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21829361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathogens&amp;title=A%20Protein%20Thermometer%20Controls%20Temperature%E2%80%90Dependent%20Transcription%20of%20Flagellar%20Motility%20Genes%20in%20Listeria%20monocytogenes&amp;author=H.%20D.%20Kamp&amp;author=D.%20E.%20Higgins&amp;volume=7&amp;issue=8&amp;publication_year=2011&amp;pages=e1002153&amp;pmid=21829361&amp;doi=10.1371/journal.ppat.1002153&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0097">
<cite>

Karmakar, R.

2021. “State of the Art of Bacterial Chemotaxis.” Journal of Basic Microbiology
61, no. 5: 366–379. 10.1002/jobm.202000661.
</cite> [<a href="https://doi.org/10.1002/jobm.202000661" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33687766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Basic%20Microbiology&amp;title=State%20of%20the%20Art%20of%20Bacterial%20Chemotaxis&amp;author=R.%20Karmakar&amp;volume=61&amp;issue=5&amp;publication_year=2021&amp;pages=366-379&amp;pmid=33687766&amp;doi=10.1002/jobm.202000661&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0098">
<cite>

Kastbjerg, V. G.
, Halberg Larsen M., Gram L., and Ingmer H.. 2010. “Influence of Sublethal Concentrations of Common Disinfectants on Expression of Virulence Genes in <em>Listeria monocytogenes</em>
.” Applied and Environmental Microbiology
76, no. 1: 303–309. 10.1128/AEM.00925-09.
</cite> [<a href="https://doi.org/10.1128/AEM.00925-09" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2798637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19897753/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=Influence%20of%20Sublethal%20Concentrations%20of%20Common%20Disinfectants%20on%20Expression%20of%20Virulence%20Genes%20in%20Listeria%20monocytogenes&amp;author=V.%20G.%20Kastbjerg&amp;author=M.%20Halberg%20Larsen&amp;author=L.%20Gram&amp;author=H.%20Ingmer&amp;volume=76&amp;issue=1&amp;publication_year=2010&amp;pages=303-309&amp;pmid=19897753&amp;doi=10.1128/AEM.00925-09&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0099">
<cite>

Kearse, M.
, Moir R., Wilson A., et al. 2012. “Geneious Basic: An Integrated and Extendable Desktop Software Platform for the Organization and Analysis of Sequence Data.” Bioinformatics
28, no. 12: 1647–1649. 10.1093/bioinformatics/bts199.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3371832/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22543367/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Geneious%20Basic:%20An%20Integrated%20and%20Extendable%20Desktop%20Software%20Platform%20for%20the%20Organization%20and%20Analysis%20of%20Sequence%20Data&amp;author=M.%20Kearse&amp;author=R.%20Moir&amp;author=A.%20Wilson&amp;volume=28&amp;issue=12&amp;publication_year=2012&amp;pages=1647-1649&amp;pmid=22543367&amp;doi=10.1093/bioinformatics/bts199&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0100">
<cite>

Kermasha, S.
, and Eskin M. N. A.. 2021. “Chapter Nine—Selected Industrial Enzymes.” In Enzymes, edited by Kermasha Selim and Eskin Michael N. A., 259–305. Academic Press. 10.1016/B978-0-12-800217-9.00009-5.</cite> [<a href="https://doi.org/10.1016/B978-0-12-800217-9.00009-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Enzymes&amp;author=S.%20Kermasha&amp;author=M.%20N.%20A.%20Eskin&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0101">
<cite>

Khadre, M. A.
, Yousef A. E., and Kim J.‐G.. 2001. “Microbiological Aspects of Ozone Applications in Food: A Review.” Journal of Food Science
66, no. 9: 1242–1252. 10.1111/j.1365-2621.2001.tb15196.x.</cite> [<a href="https://doi.org/10.1111/j.1365-2621.2001.tb15196.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Food%20Science&amp;title=Microbiological%20Aspects%20of%20Ozone%20Applications%20in%20Food:%20A%20Review&amp;author=M.%20A.%20Khadre&amp;author=A.%20E.%20Yousef&amp;author=J.%E2%80%90G.%20Kim&amp;volume=66&amp;issue=9&amp;publication_year=2001&amp;pages=1242-1252&amp;doi=10.1111/j.1365-2621.2001.tb15196.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0102">
<cite>

Kikuchi, G.
, Motokawa Y., Yoshida T., and Hiraga K.. 2008. “Glycine Cleavage System: Reaction Mechanism, Physiological Significance, and Hyperglycinemia.” Proceedings of the Japan Academy, Series B
84, no. 7: 246–263. 10.2183/pjab.84.246.</cite> [<a href="https://doi.org/10.2183/pjab.84.246" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3666648/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18941301/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20Japan%20Academy,%20Series%20B&amp;title=Glycine%20Cleavage%20System:%20Reaction%20Mechanism,%20Physiological%20Significance,%20and%20Hyperglycinemia&amp;author=G.%20Kikuchi&amp;author=Y.%20Motokawa&amp;author=T.%20Yoshida&amp;author=K.%20Hiraga&amp;volume=84&amp;issue=7&amp;publication_year=2008&amp;pages=246-263&amp;pmid=18941301&amp;doi=10.2183/pjab.84.246&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0103">
<cite>

Kim, J.
, Kim G.‐L., Norambuena J., Boyd J. M., and Parker D.. 2023. “Impact of the Pentose Phosphate Pathway on Metabolism and Pathogenesis of <em>Staphylococcus aureus</em>
.” PLOS Pathogens
19, no. 7: e1011531. 10.1371/journal.ppat.1011531.
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1011531" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10368262/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37440594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Pathogens&amp;title=Impact%20of%20the%20Pentose%20Phosphate%20Pathway%20on%20Metabolism%20and%20Pathogenesis%20of%20Staphylococcus%20aureus&amp;author=J.%20Kim&amp;author=G.%E2%80%90L.%20Kim&amp;author=J.%20Norambuena&amp;author=J.%20M.%20Boyd&amp;author=D.%20Parker&amp;volume=19&amp;issue=7&amp;publication_year=2023&amp;pages=e1011531&amp;pmid=37440594&amp;doi=10.1371/journal.ppat.1011531&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0104">
<cite>

Kohanski, M. A.
, Dwyer D. J., and Collins J. J.. 2010. “How Antibiotics Kill Bacteria: from Targets to Networks.” Nature Reviews Microbiology
8, no. 6: 423–435. 10.1038/nrmicro2333.
</cite> [<a href="https://doi.org/10.1038/nrmicro2333" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2896384/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20440275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Microbiology&amp;title=How%20Antibiotics%20Kill%20Bacteria:%20from%20Targets%20to%20Networks&amp;author=M.%20A.%20Kohanski&amp;author=D.%20J.%20Dwyer&amp;author=J.%20J.%20Collins&amp;volume=8&amp;issue=6&amp;publication_year=2010&amp;pages=423-435&amp;pmid=20440275&amp;doi=10.1038/nrmicro2333&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0252">
<cite>

Kratz, J. C.
, and Banerjee S.. 2024. “Gene Expression Tradeoffs Determine Bacterial Survival and Adaptation to Antibiotic Stress.” BioRxiv January 23, 2024.01.20.576495. 10.1101/2024.01.20.576495.</cite> [<a href="https://doi.org/10.1101/2024.01.20.576495" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11500821/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39449977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BioRxiv&amp;title=Gene%20Expression%20Tradeoffs%20Determine%20Bacterial%20Survival%20and%20Adaptation%20to%20Antibiotic%20Stress&amp;author=J.%20C.%20Kratz&amp;author=S.%20Banerjee&amp;publication_year=2024&amp;pmid=39449977&amp;doi=10.1101/2024.01.20.576495&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0105">
<cite>

Kurosu, M.
, and Begari E.. 2010. “Vitamin K2 inElectron Transport System: Are Enzymes Involved in Vitamin K2 BiosynthesisPromising Drug Targets?.” Molecules (Basel, Switzerland)
15, no. 3: 1531–1553.
</cite> [<a href="https://doi.org/10.3390/molecules15031531" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6257245/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20335999/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules%20(Basel,%20Switzerland)&amp;title=Vitamin%20K2%20inElectron%20Transport%20System:%20Are%20Enzymes%20Involved%20in%20Vitamin%20K2%20BiosynthesisPromising%20Drug%20Targets?&amp;author=M.%20Kurosu&amp;author=E.%20Begari&amp;volume=15&amp;issue=3&amp;publication_year=2010&amp;pages=1531-1553&amp;pmid=20335999&amp;doi=10.3390/molecules15031531&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0106">
<cite>

Landick, R.
, Oxender D. L., and Ames G. F.‐L.. 1985. “Bacterial Amino Acid Transport Systems.” In The Enzymes of Biological Membranes, edited by Martonosi Anthony N., 577–615. Springer US. 10.1007/978-1-4684-4601-2_17.</cite> [<a href="https://doi.org/10.1007/978-1-4684-4601-2_17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=The%20Enzymes%20of%20Biological%20Membranes&amp;author=R.%20Landick&amp;author=D.%20L.%20Oxender&amp;author=G.%20F.%E2%80%90L.%20Ames&amp;publication_year=1985&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0107">
<cite>

Lee, T. K.
, and Huang K. C.. 2013. “The Role of Hydrolases in Bacterial Cell‐Wall Growth.” Current Opinion in Microbiology
16, no. 6: 760. 10.1016/j.mib.2013.08.005.
</cite> [<a href="https://doi.org/10.1016/j.mib.2013.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3986426/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24035761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Microbiology&amp;title=The%20Role%20of%20Hydrolases%20in%20Bacterial%20Cell%E2%80%90Wall%20Growth&amp;author=T.%20K.%20Lee&amp;author=K.%20C.%20Huang&amp;volume=16&amp;issue=6&amp;publication_year=2013&amp;pages=760&amp;pmid=24035761&amp;doi=10.1016/j.mib.2013.08.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0108">
<cite>

Leimkühler, S.

2020. “The Biosynthesis of the Molybdenum Cofactors in.” Environmental Microbiology
22, no. 6: 2007–2026. 10.1111/1462-2920.15003.
</cite> [<a href="https://doi.org/10.1111/1462-2920.15003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32239579/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environmental%20Microbiology&amp;title=The%20Biosynthesis%20of%20the%20Molybdenum%20Cofactors%20in&amp;author=S.%20Leimk%C3%BChler&amp;volume=22&amp;issue=6&amp;publication_year=2020&amp;pages=2007-2026&amp;pmid=32239579&amp;doi=10.1111/1462-2920.15003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0109">
<cite>

Lemos, J. A. C.
, and Burne R. A.. 2002. “Regulation and Physiological Significance of ClpC and ClpP in <em>Streptococcus mutans</em>
.” Journal of Bacteriology
184, no. 22: 6357. 10.1128/JB.184.22.6357-6366.2002.
</cite> [<a href="https://doi.org/10.1128/JB.184.22.6357-6366.2002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC151938/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12399506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Bacteriology&amp;title=Regulation%20and%20Physiological%20Significance%20of%20ClpC%20and%20ClpP%20in%20Streptococcus%20mutans&amp;author=J.%20A.%20C.%20Lemos&amp;author=R.%20A.%20Burne&amp;volume=184&amp;issue=22&amp;publication_year=2002&amp;pages=6357&amp;pmid=12399506&amp;doi=10.1128/JB.184.22.6357-6366.2002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0110">
<cite>

Leonardi, R.
, and Jackowski S.. 2007. “Biosynthesis of Pantothenic Acid and Coenzyme A.” EcoSal Plus
2, no. 2: 1–18. 10.1128/ecosalplus.3.6.3.4.</cite> [<a href="https://doi.org/10.1128/ecosalplus.3.6.3.4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4950986/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26443589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EcoSal%20Plus&amp;title=Biosynthesis%20of%20Pantothenic%20Acid%20and%20Coenzyme%20A&amp;author=R.%20Leonardi&amp;author=S.%20Jackowski&amp;volume=2&amp;issue=2&amp;publication_year=2007&amp;pmid=26443589&amp;doi=10.1128/ecosalplus.3.6.3.4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0111">
<cite>

LeRoux, M.
, Peterson S. B., and Mougous J. D.. 2015. “Bacterial Danger Sensing.” Journal of Molecular Biology
427, no. 23: 3744. 10.1016/j.jmb.2015.09.018.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2015.09.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4658316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26434507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Molecular%20Biology&amp;title=Bacterial%20Danger%20Sensing&amp;author=M.%20LeRoux&amp;author=S.%20B.%20Peterson&amp;author=J.%20D.%20Mougous&amp;volume=427&amp;issue=23&amp;publication_year=2015&amp;pages=3744&amp;pmid=26434507&amp;doi=10.1016/j.jmb.2015.09.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0112">
<cite>

Li, H.
, Liang D., Huang J., et al. 2021. “The Bactericidal Efficacy and the Mechanism of Action of Slightly Acidic Electrolyzed Water on <em>Listeria monocytogenes</em>' Survival.” Foods
10, no. 11: 2671. 10.3390/foods10112671.
</cite> [<a href="https://doi.org/10.3390/foods10112671" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8621911/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34828952/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Foods&amp;title=The%20Bactericidal%20Efficacy%20and%20the%20Mechanism%20of%20Action%20of%20Slightly%20Acidic%20Electrolyzed%20Water%20on%20Listeria%20monocytogenes'%20Survival&amp;author=H.%20Li&amp;author=D.%20Liang&amp;author=J.%20Huang&amp;volume=10&amp;issue=11&amp;publication_year=2021&amp;pages=2671&amp;pmid=34828952&amp;doi=10.3390/foods10112671&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0113">
<cite>

Li, Y.
, and Zhang Y.. 2007. “PhoU Is a Persistence Switch Involved in Persister Formation and Tolerance to Multiple Antibiotics and Stresses in <em>Escherichia coli</em>
.” Antimicrobial Agents and Chemotherapy
51, no. 6: 2092. 10.1128/AAC.00052-07.
</cite> [<a href="https://doi.org/10.1128/AAC.00052-07" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1891003/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17420206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antimicrobial%20Agents%20and%20Chemotherapy&amp;title=PhoU%20Is%20a%20Persistence%20Switch%20Involved%20in%20Persister%20Formation%20and%20Tolerance%20to%20Multiple%20Antibiotics%20and%20Stresses%20in%20Escherichia%20coli&amp;author=Y.%20Li&amp;author=Y.%20Zhang&amp;volume=51&amp;issue=6&amp;publication_year=2007&amp;pages=2092&amp;pmid=17420206&amp;doi=10.1128/AAC.00052-07&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0114">
<cite>

Litschko, C.
, Domenico V. D., Schulze J., et al. 2024. “Transition Transferases Prime Bacterial Capsule Polymerization.” Nature Chemical Biology no. July: 1–11. 10.1038/s41589-024-01664-8.
</cite> [<a href="https://doi.org/10.1038/s41589-024-01664-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11666461/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38951648/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Chemical%20Biology&amp;title=Transition%20Transferases%20Prime%20Bacterial%20Capsule%20Polymerization&amp;author=C.%20Litschko&amp;author=V.%20D.%20Domenico&amp;author=J.%20Schulze&amp;issue=July&amp;publication_year=2024&amp;pages=1-11&amp;pmid=38951648&amp;doi=10.1038/s41589-024-01664-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0115">
<cite>

Liu, S.‐S.
, Qu H.‐M., Yang D., et al. 2018. “Chlorine Disinfection Increases Both Intracellular and Extracellular Antibiotic Resistance Genes in a Full‐Scale Wastewater Treatment Plant.” Water Research
136, no. June: 131–136. 10.1016/j.watres.2018.02.036.
</cite> [<a href="https://doi.org/10.1016/j.watres.2018.02.036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29501757/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Water%20Research&amp;title=Chlorine%20Disinfection%20Increases%20Both%20Intracellular%20and%20Extracellular%20Antibiotic%20Resistance%20Genes%20in%20a%20Full%E2%80%90Scale%20Wastewater%20Treatment%20Plant&amp;author=S.%E2%80%90S.%20Liu&amp;author=H.%E2%80%90M.%20Qu&amp;author=D.%20Yang&amp;volume=136&amp;issue=June&amp;publication_year=2018&amp;pages=131-136&amp;pmid=29501757&amp;doi=10.1016/j.watres.2018.02.036&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0116">
<cite>

Liu, Y.
, Sun W., Sun T., et al. 2020. “The Prevalence of <em>Listeria monocytogenes</em> in Meat Products in China: A Systematic Literature Review and Novel Meta‐Analysis Approach.” International Journal of Food Microbiology
312, no. January: 108358. 10.1016/j.ijfoodmicro.2019.108358.
</cite> [<a href="https://doi.org/10.1016/j.ijfoodmicro.2019.108358" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31655356/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Food%20Microbiology&amp;title=The%20Prevalence%20of%20Listeria%20monocytogenes%20in%20Meat%20Products%20in%20China:%20A%20Systematic%20Literature%20Review%20and%20Novel%20Meta%E2%80%90Analysis%20Approach&amp;author=Y.%20Liu&amp;author=W.%20Sun&amp;author=T.%20Sun&amp;volume=312&amp;issue=January&amp;publication_year=2020&amp;pages=108358&amp;pmid=31655356&amp;doi=10.1016/j.ijfoodmicro.2019.108358&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0117">
<cite>

Loh, E.
, Memarpour F., Vaitkevicius K., Kallipolitis B. H., Johansson J., and Sondén B.. 2012. “An Unstructured 5'‐Coding Region of the prfA mRNA Is Required for Efficient Translation.” Nucleic Acids Research
40, no. 4: 1818–1827. 10.1093/nar/gkr850.
</cite> [<a href="https://doi.org/10.1093/nar/gkr850" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3287183/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22053088/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=An%20Unstructured%205'%E2%80%90Coding%20Region%20of%20the%20prfA%20mRNA%20Is%20Required%20for%20Efficient%20Translation&amp;author=E.%20Loh&amp;author=F.%20Memarpour&amp;author=K.%20Vaitkevicius&amp;author=B.%20H.%20Kallipolitis&amp;author=J.%20Johansson&amp;volume=40&amp;issue=4&amp;publication_year=2012&amp;pages=1818-1827&amp;pmid=22053088&amp;doi=10.1093/nar/gkr850&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0254">
<cite>

Lou, Y.
, and Yousef A. E.. 1997. “Adaptation to Sublethal Environmental Stresses Protects Listeria Monocytogenes Against Lethal Preservation Factors.” Applied and Environmental Microbiology
63, no. 4: 1252–1255. 10.1128/aem.63.4.1252-1255.1997.
</cite> [<a href="https://doi.org/10.1128/aem.63.4.1252-1255.1997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC168417/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9097420/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=Adaptation%20to%20Sublethal%20Environmental%20Stresses%20Protects%20Listeria%20Monocytogenes%20Against%20Lethal%20Preservation%20Factors&amp;author=Y.%20Lou&amp;author=A.%20E.%20Yousef&amp;volume=63&amp;issue=4&amp;publication_year=1997&amp;pages=1252-1255&amp;pmid=9097420&amp;doi=10.1128/aem.63.4.1252-1255.1997&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0118">
<cite>

Luo, Q.
, Ding N., Liu Y., Zhang H., Fang Y., and Yin L.. 2023. “Metabolic Engineering of Microorganisms to Produce Pyruvate and Derived Compounds.” Molecules (Basel, Switzerland)
28, no. 3: 1418. 10.3390/molecules28031418.
</cite> [<a href="https://doi.org/10.3390/molecules28031418" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9919917/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36771084/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules%20(Basel,%20Switzerland)&amp;title=Metabolic%20Engineering%20of%20Microorganisms%20to%20Produce%20Pyruvate%20and%20Derived%20Compounds&amp;author=Q.%20Luo&amp;author=N.%20Ding&amp;author=Y.%20Liu&amp;author=H.%20Zhang&amp;author=Y.%20Fang&amp;volume=28&amp;issue=3&amp;publication_year=2023&amp;pages=1418&amp;pmid=36771084&amp;doi=10.3390/molecules28031418&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0119">
<cite>

Mahone, C. R.
, and Goley E. D.. 2020. “Bacterial Cell Division at a Glance.” Journal of Cell Science
133, no. 7: jcs237057. 10.1242/jcs.237057.
</cite> [<a href="https://doi.org/10.1242/jcs.237057" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7157936/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32269092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cell%20Science&amp;title=Bacterial%20Cell%20Division%20at%20a%20Glance&amp;author=C.%20R.%20Mahone&amp;author=E.%20D.%20Goley&amp;volume=133&amp;issue=7&amp;publication_year=2020&amp;pages=jcs237057&amp;pmid=32269092&amp;doi=10.1242/jcs.237057&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0120">
<cite>

Maillard, J.‐Y.
, and Pascoe M.. 2024. “Disinfectants and Antiseptics: Mechanisms of Action and Resistance.” Nature Reviews Microbiology
22, no. 1: 4–17. 10.1038/s41579-023-00958-3.
</cite> [<a href="https://doi.org/10.1038/s41579-023-00958-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37648789/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Microbiology&amp;title=Disinfectants%20and%20Antiseptics:%20Mechanisms%20of%20Action%20and%20Resistance&amp;author=J.%E2%80%90Y.%20Maillard&amp;author=M.%20Pascoe&amp;volume=22&amp;issue=1&amp;publication_year=2024&amp;pages=4-17&amp;pmid=37648789&amp;doi=10.1038/s41579-023-00958-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0121">
<cite>

Maleki, F.
, Khosravi A., Nasser A., Taghinejad H., and Azizian M.. 2016. “Bacterial Heat Shock Protein Activity.” Journal of Clinical and Diagnostic Research : JCDR
10, no. 3: BE01. 10.7860/JCDR/2016/14568.7444.</cite> [<a href="https://doi.org/10.7860/JCDR/2016/14568.7444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4843247/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27134861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20and%20Diagnostic%20Research%20:%20JCDR&amp;title=Bacterial%20Heat%20Shock%20Protein%20Activity&amp;author=F.%20Maleki&amp;author=A.%20Khosravi&amp;author=A.%20Nasser&amp;author=H.%20Taghinejad&amp;author=M.%20Azizian&amp;volume=10&amp;issue=3&amp;publication_year=2016&amp;pages=BE01&amp;pmid=27134861&amp;doi=10.7860/JCDR/2016/14568.7444&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0122">
<cite>

Mancuso, G.
, Midiri A., Gerace E., and Biondo C.. 2021. “Bacterial Antibiotic Resistance: The Most Critical Pathogens.” Pathogens
10, no. 10: 1310. 10.3390/pathogens10101310.
</cite> [<a href="https://doi.org/10.3390/pathogens10101310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8541462/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34684258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pathogens&amp;title=Bacterial%20Antibiotic%20Resistance:%20The%20Most%20Critical%20Pathogens&amp;author=G.%20Mancuso&amp;author=A.%20Midiri&amp;author=E.%20Gerace&amp;author=C.%20Biondo&amp;volume=10&amp;issue=10&amp;publication_year=2021&amp;pages=1310&amp;pmid=34684258&amp;doi=10.3390/pathogens10101310&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0123">
<cite>

Manso, B.
, Melero B., Stessl B., et al. 2020. “The Response to Oxidative Stress in <em>Listeria monocytogenes</em> Is Temperature Dependent.” Microorganisms
8, no. 4: 521. 10.3390/microorganisms8040521.
</cite> [<a href="https://doi.org/10.3390/microorganisms8040521" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7232520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32260575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microorganisms&amp;title=The%20Response%20to%20Oxidative%20Stress%20in%20Listeria%20monocytogenes%20Is%20Temperature%20Dependent&amp;author=B.%20Manso&amp;author=B.%20Melero&amp;author=B.%20Stessl&amp;volume=8&amp;issue=4&amp;publication_year=2020&amp;pages=521&amp;pmid=32260575&amp;doi=10.3390/microorganisms8040521&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0125">
<cite>

Martinez‐Bond, E. A.
, Lopez‐Ayala I., Lobanovska M., et al. 2025. “Phosphorylation State Dictates Bacterial Stressosome Assembly and Function.” Preprint, BioRxiv, March 5. 10.1101/2025.03.04.640944.</cite> [<a href="https://doi.org/10.1101/2025.03.04.640944" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="crf370260-bib-0126">
<cite>

Mata, M. T.
, Baquero F., and Pérez‐Díaz J. C.. 2000. “A Multidrug Efflux Transporter in <em>Listeria monocytogenes</em>
.” FEMS Microbiology Letters
187, no. 2: 185–188. 10.1111/j.1574-6968.2000.tb09158.x.
</cite> [<a href="https://doi.org/10.1111/j.1574-6968.2000.tb09158.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10856655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiology%20Letters&amp;title=A%20Multidrug%20Efflux%20Transporter%20in%20Listeria%20monocytogenes&amp;author=M.%20T.%20Mata&amp;author=F.%20Baquero&amp;author=J.%20C.%20P%C3%A9rez%E2%80%90D%C3%ADaz&amp;volume=187&amp;issue=2&amp;publication_year=2000&amp;pages=185-188&amp;pmid=10856655&amp;doi=10.1111/j.1574-6968.2000.tb09158.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0127">
<cite>

Matthews, R. G.

2009. “Cobalamin‐ and Corrinoid‐Dependent Enzymes.” Metal Ions in Life Sciences
6, no. January: 53. 10.1039/BK9781847559159-00053.
</cite> [<a href="https://doi.org/10.1039/BK9781847559159-00053" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3120101/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20877792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Metal%20Ions%20in%20Life%20Sciences&amp;title=Cobalamin%E2%80%90%20and%20Corrinoid%E2%80%90Dependent%20Enzymes&amp;author=R.%20G.%20Matthews&amp;volume=6&amp;issue=January&amp;publication_year=2009&amp;pages=53&amp;pmid=20877792&amp;doi=10.1039/BK9781847559159-00053&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0128">
<cite>

McDonald, A. G.
, and Tipton K. F.. 2023. “Enzyme Nomenclature and Classification: The State of the Art.” The FEBS Journal
290, no. 9: 2214–2231. 10.1111/febs.16274.
</cite> [<a href="https://doi.org/10.1111/febs.16274" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34773359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20FEBS%20Journal&amp;title=Enzyme%20Nomenclature%20and%20Classification:%20The%20State%20of%20the%20Art&amp;author=A.%20G.%20McDonald&amp;author=K.%20F.%20Tipton&amp;volume=290&amp;issue=9&amp;publication_year=2023&amp;pages=2214-2231&amp;pmid=34773359&amp;doi=10.1111/febs.16274&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0129">
<cite>

Mendel, R. R.
, and Leimkühler S.. 2015. “The Biosynthesis of the Molybdenum Cofactors.” JBIC Journal of Biological Inorganic Chemistry
20, no. 2: 337–347. 10.1007/s00775-014-1173-y.
</cite> [<a href="https://doi.org/10.1007/s00775-014-1173-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24980677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JBIC%20Journal%20of%20Biological%20Inorganic%20Chemistry&amp;title=The%20Biosynthesis%20of%20the%20Molybdenum%20Cofactors&amp;author=R.%20R.%20Mendel&amp;author=S.%20Leimk%C3%BChler&amp;volume=20&amp;issue=2&amp;publication_year=2015&amp;pages=337-347&amp;pmid=24980677&amp;doi=10.1007/s00775-014-1173-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0130">
<cite>

Meyrat, A.
, and von Ballmoos C.. 2019. “ATP Synthesis at Physiological Nucleotide Concentrations.” Scientific Reports
9, no. 1: 3070. 10.1038/s41598-019-38564-0.
</cite> [<a href="https://doi.org/10.1038/s41598-019-38564-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6395684/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30816129/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=ATP%20Synthesis%20at%20Physiological%20Nucleotide%20Concentrations&amp;author=A.%20Meyrat&amp;author=C.%20von%20Ballmoos&amp;volume=9&amp;issue=1&amp;publication_year=2019&amp;pages=3070&amp;pmid=30816129&amp;doi=10.1038/s41598-019-38564-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0131">
<cite>

Minami, M.
, Ando T., Hashikawa S.‐n., et al. 2004. “Effect of Glycine on Helicobacter Pylori in Vitro.” Antimicrobial Agents and Chemotherapy
48, no. 10: 3782–3788. 10.1128/AAC.48.10.3782-3788.2004.
</cite> [<a href="https://doi.org/10.1128/AAC.48.10.3782-3788.2004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC521915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15388434/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antimicrobial%20Agents%20and%20Chemotherapy&amp;title=Effect%20of%20Glycine%20on%20Helicobacter%20Pylori%20in%20Vitro&amp;author=M.%20Minami&amp;author=T.%20Ando&amp;author=S.%E2%80%90n.%20Hashikawa&amp;volume=48&amp;issue=10&amp;publication_year=2004&amp;pages=3782-3788&amp;pmid=15388434&amp;doi=10.1128/AAC.48.10.3782-3788.2004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0132">
<cite>

Miyamoto, T.
, and Homma H.. 2021. “D‐Amino Acid Metabolism in Bacteria.” The Journal of Biochemistry
170, no. 1: 5–13. 10.1093/jb/mvab043.
</cite> [<a href="https://doi.org/10.1093/jb/mvab043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33788945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Biochemistry&amp;title=D%E2%80%90Amino%20Acid%20Metabolism%20in%20Bacteria&amp;author=T.%20Miyamoto&amp;author=H.%20Homma&amp;volume=170&amp;issue=1&amp;publication_year=2021&amp;pages=5-13&amp;pmid=33788945&amp;doi=10.1093/jb/mvab043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0133">
<cite>

Monniot, C.
, Zébré A. C., Aké F. M. D., Deutscher J., and Milohanic E.. 2012. “Novel Listerial Glycerol Dehydrogenase‐ and Phosphoenolpyruvate‐Dependent Dihydroxyacetone Kinase System Connected to the Pentose Phosphate Pathway.” Journal of Bacteriology
194, no. 18: 4972–4982. 10.1128/JB.00801-12.
</cite> [<a href="https://doi.org/10.1128/JB.00801-12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3430356/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22773791/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Bacteriology&amp;title=Novel%20Listerial%20Glycerol%20Dehydrogenase%E2%80%90%20and%20Phosphoenolpyruvate%E2%80%90Dependent%20Dihydroxyacetone%20Kinase%20System%20Connected%20to%20the%20Pentose%20Phosphate%20Pathway&amp;author=C.%20Monniot&amp;author=A.%20C.%20Z%C3%A9br%C3%A9&amp;author=F.%20M.%20D.%20Ak%C3%A9&amp;author=J.%20Deutscher&amp;author=E.%20Milohanic&amp;volume=194&amp;issue=18&amp;publication_year=2012&amp;pages=4972-4982&amp;pmid=22773791&amp;doi=10.1128/JB.00801-12&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0134">
<cite>

Muchaamba, F.
, Eshwar A. K., Stevens M. J. A., Stephan R., and Tasara T.. 2022. “Different Shades of <em>Listeria monocytogenes</em>: Strain, Serotype, and Lineage‐Based Variability in Virulence and Stress Tolerance Profiles.” Frontiers in Microbiology
12, no. January: 792162. 10.3389/fmicb.2021.792162.
</cite> [<a href="https://doi.org/10.3389/fmicb.2021.792162" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8764371/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35058906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Different%20Shades%20of%20Listeria%20monocytogenes:%20Strain,%20Serotype,%20and%20Lineage%E2%80%90Based%20Variability%20in%20Virulence%20and%20Stress%20Tolerance%20Profiles&amp;author=F.%20Muchaamba&amp;author=A.%20K.%20Eshwar&amp;author=M.%20J.%20A.%20Stevens&amp;author=R.%20Stephan&amp;author=T.%20Tasara&amp;volume=12&amp;issue=January&amp;publication_year=2022&amp;pages=792162&amp;pmid=35058906&amp;doi=10.3389/fmicb.2021.792162&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0135">
<cite>

Muchaamba, F.
, Stephan R., and Tasara T.. 2021. “
<em>Listeria monocytogenes</em> Cold Shock Proteins: Small Proteins With a Huge Impact.” Microorganisms
9, no. 5: 1061. 10.3390/microorganisms9051061.
</cite> [<a href="https://doi.org/10.3390/microorganisms9051061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8155936/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34068949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microorganisms&amp;title=Listeria%20monocytogenes%20Cold%20Shock%20Proteins:%20Small%20Proteins%20With%20a%20Huge%20Impact&amp;author=F.%20Muchaamba&amp;author=R.%20Stephan&amp;author=T.%20Tasara&amp;volume=9&amp;issue=5&amp;publication_year=2021&amp;pages=1061&amp;pmid=34068949&amp;doi=10.3390/microorganisms9051061&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0136">
<cite>

Nardini, M.
, Ridder I. S., Rozeboom H. J., et al. 1999. “The X‐Ray Structure of Epoxide Hydrolase From Agrobacterium Radiobacter AD1: An Enzyme to Detoxify Harmful Epoxides.” Journal of Biological Chemistry
274, no. 21: 14579–14586. 10.1074/jbc.274.21.14579.
</cite> [<a href="https://doi.org/10.1074/jbc.274.21.14579" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10329649/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=The%20X%E2%80%90Ray%20Structure%20of%20Epoxide%20Hydrolase%20From%20Agrobacterium%20Radiobacter%20AD1:%20An%20Enzyme%20to%20Detoxify%20Harmful%20Epoxides&amp;author=M.%20Nardini&amp;author=I.%20S.%20Ridder&amp;author=H.%20J.%20Rozeboom&amp;volume=274&amp;issue=21&amp;publication_year=1999&amp;pages=14579-14586&amp;pmid=10329649&amp;doi=10.1074/jbc.274.21.14579&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0137">
<cite>

Nguyen, T. T. H.
, Myrold D. D., and Mueller R. S.. 2019. “Distributions of Extracellular Peptidases Across Prokaryotic Genomes Reflect Phylogeny and Habitat.” Frontiers in Microbiology
10, no. March: 413. 10.3389/fmicb.2019.00413.
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.00413" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6411800/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30891022/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Distributions%20of%20Extracellular%20Peptidases%20Across%20Prokaryotic%20Genomes%20Reflect%20Phylogeny%20and%20Habitat&amp;author=T.%20T.%20H.%20Nguyen&amp;author=D.%20D.%20Myrold&amp;author=R.%20S.%20Mueller&amp;volume=10&amp;issue=March&amp;publication_year=2019&amp;pages=413&amp;pmid=30891022&amp;doi=10.3389/fmicb.2019.00413&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0138">
<cite>

Nightingale, K. K.
, Windham K., and Wiedmann M.. 2005. “Evolution and Molecular Phylogeny of <em>Listeria monocytogenes</em> Isolated From Human and Animal Listeriosis Cases and Foods.” Journal of Bacteriology
187, no. 16: 5537–5551. 10.1128/JB.187.16.5537-5551.2005.
</cite> [<a href="https://doi.org/10.1128/JB.187.16.5537-5551.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1196091/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16077098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Bacteriology&amp;title=Evolution%20and%20Molecular%20Phylogeny%20of%20Listeria%20monocytogenes%20Isolated%20From%20Human%20and%20Animal%20Listeriosis%20Cases%20and%20Foods&amp;author=K.%20K.%20Nightingale&amp;author=K.%20Windham&amp;author=M.%20Wiedmann&amp;volume=187&amp;issue=16&amp;publication_year=2005&amp;pages=5537-5551&amp;pmid=16077098&amp;doi=10.1128/JB.187.16.5537-5551.2005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0139">
<cite>

Nord, F. F.

2009. Advances in Enzymology and Related Areas of Molecular Biology. John Wiley &amp; Sons.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20Enzymology%20and%20Related%20Areas%20of%20Molecular%20Biology&amp;author=F.%20F.%20Nord&amp;publication_year=2009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0143">
<cite>

O'Neil, H. S.
, and Marquis H.. 2006. “
<em>Listeria monocytogenes</em> Flagella Are Used for Motility, Not as Adhesins, to Increase Host Cell Invasion.” Infection and Immunity
74, no. 12: 6675–6681. 10.1128/iai.00886-06.
</cite> [<a href="https://doi.org/10.1128/iai.00886-06" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1698079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16982842/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infection%20and%20Immunity&amp;title=Listeria%20monocytogenes%20Flagella%20Are%20Used%20for%20Motility,%20Not%20as%20Adhesins,%20to%20Increase%20Host%20Cell%20Invasion&amp;author=H.%20S.%20O'Neil&amp;author=H.%20Marquis&amp;volume=74&amp;issue=12&amp;publication_year=2006&amp;pages=6675-6681&amp;pmid=16982842&amp;doi=10.1128/iai.00886-06&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0140">
<cite>

Ogunleye, S. C.
, Islam S., Monzur Kader Chowdhury Q. M., Ozdemir O., Lawrence M. L., and Abdelhamed H.. 2024. “Catabolite Control Protein C Contributes to Virulence and Hydrogen Peroxide‐Induced Oxidative Stress Responses in <em>Listeria monocytogenes</em>
.” Frontiers in Microbiology
15, no. May: 1403694. 10.3389/fmicb.2024.1403694.
</cite> [<a href="https://doi.org/10.3389/fmicb.2024.1403694" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11176438/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38881664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Catabolite%20Control%20Protein%20C%20Contributes%20to%20Virulence%20and%20Hydrogen%20Peroxide%E2%80%90Induced%20Oxidative%20Stress%20Responses%20in%20Listeria%20monocytogenes&amp;author=S.%20C.%20Ogunleye&amp;author=S.%20Islam&amp;author=Q.%20M.%20Monzur%20Kader%20Chowdhury&amp;author=O.%20Ozdemir&amp;author=M.%20L.%20Lawrence&amp;volume=15&amp;issue=May&amp;publication_year=2024&amp;pages=1403694&amp;pmid=38881664&amp;doi=10.3389/fmicb.2024.1403694&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0142">
<cite>

Ölmez, H.
, and Akbas M. Y.. 2009. “Optimization of Ozone Treatment of Fresh‐Cut Green Leaf Lettuce.” Journal of Food Engineering
90, no. 4: 487–494. 10.1016/j.jfoodeng.2008.07.026.</cite> [<a href="https://doi.org/10.1016/j.jfoodeng.2008.07.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Food%20Engineering&amp;title=Optimization%20of%20Ozone%20Treatment%20of%20Fresh%E2%80%90Cut%20Green%20Leaf%20Lettuce&amp;author=H.%20%C3%96lmez&amp;author=M.%20Y.%20Akbas&amp;volume=90&amp;issue=4&amp;publication_year=2009&amp;pages=487-494&amp;doi=10.1016/j.jfoodeng.2008.07.026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0256">
<cite>

Papadimitriou, K.
, Alegría Á. n., Bron P. A., et al. 2016. “Stress Physiology of Lactic Acid Bacteria.” Microbiology and Molecular Biology Reviews : MMBR
80, no. 3: 837–890. 10.1128/MMBR.00076-15.
</cite> [<a href="https://doi.org/10.1128/MMBR.00076-15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4981675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27466284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology%20and%20Molecular%20Biology%20Reviews%20:%20MMBR&amp;title=Stress%20Physiology%20of%20Lactic%20Acid%20Bacteria&amp;author=K.%20Papadimitriou&amp;author=%C3%81.%20n.%20Alegr%C3%ADa&amp;author=P.%20A.%20Bron&amp;volume=80&amp;issue=3&amp;publication_year=2016&amp;pages=837-890&amp;pmid=27466284&amp;doi=10.1128/MMBR.00076-15&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0144">
<cite>

Parsons, J. B.
, and Rock C. O.. 2013. “Bacterial Lipids: Metabolism and Membrane Homeostasis.” Progress in Lipid Research
52, no. 3: 249–276. 10.1016/j.plipres.2013.02.002.
</cite> [<a href="https://doi.org/10.1016/j.plipres.2013.02.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3665635/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23500459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Progress%20in%20Lipid%20Research&amp;title=Bacterial%20Lipids:%20Metabolism%20and%20Membrane%20Homeostasis&amp;author=J.%20B.%20Parsons&amp;author=C.%20O.%20Rock&amp;volume=52&amp;issue=3&amp;publication_year=2013&amp;pages=249-276&amp;pmid=23500459&amp;doi=10.1016/j.plipres.2013.02.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0241">
<cite>

Pfaller, S.
, King D., Mistry J. H., et al. 2021. “Chloramine Concentrations Within Distribution Systems and Their Effect on Heterotrophic Bacteria, Mycobacterial Species, and Disinfection Byproducts.” Water Research
205, no. October: 117689. 10.1016/j.watres.2021.117689.
</cite> [<a href="https://doi.org/10.1016/j.watres.2021.117689" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8682803/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34607086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Water%20Research&amp;title=Chloramine%20Concentrations%20Within%20Distribution%20Systems%20and%20Their%20Effect%20on%20Heterotrophic%20Bacteria,%20Mycobacterial%20Species,%20and%20Disinfection%20Byproducts&amp;author=S.%20Pfaller&amp;author=D.%20King&amp;author=J.%20H.%20Mistry&amp;volume=205&amp;issue=October&amp;publication_year=2021&amp;pages=117689&amp;pmid=34607086&amp;doi=10.1016/j.watres.2021.117689&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0145">
<cite>

Pillich, H.
, Puri M., and Chakraborty T.. 2017. “ActA of <em>Listeria monocytogenes</em> and Its Manifold Activities as an Important Listerial Virulence Factor.” Current Topics in Microbiology and Immunology
399: 113–132. 10.1007/82_2016_30.
</cite> [<a href="https://doi.org/10.1007/82_2016_30" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27726006/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Topics%20in%20Microbiology%20and%20Immunology&amp;title=ActA%20of%20Listeria%20monocytogenes%20and%20Its%20Manifold%20Activities%20as%20an%20Important%20Listerial%20Virulence%20Factor&amp;author=H.%20Pillich&amp;author=M.%20Puri&amp;author=T.%20Chakraborty&amp;volume=399&amp;publication_year=2017&amp;pages=113-132&amp;pmid=27726006&amp;doi=10.1007/82_2016_30&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0146">
<cite>

Piskovsky, V.
, and Oliveira N. M.. 2023. “Bacterial Motility Can Govern the Dynamics of Antibiotic Resistance Evolution.” Nature Communications
14, no. 1: 5584. 10.1038/s41467-023-41196-8.</cite> [<a href="https://doi.org/10.1038/s41467-023-41196-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10495427/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37696800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Bacterial%20Motility%20Can%20Govern%20the%20Dynamics%20of%20Antibiotic%20Resistance%20Evolution&amp;author=V.%20Piskovsky&amp;author=N.%20M.%20Oliveira&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=5584&amp;pmid=37696800&amp;doi=10.1038/s41467-023-41196-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0147">
<cite>

Pizarro‐Cerdá, J.
, and Cossart P.. 2019. “Microbe Profile: <em>Listeria monocytogenes</em>: A Paradigm Among Intracellular Bacterial Pathogens.” Microbiology (Reading, England)
165, no. 7: 719–721. 10.1099/mic.0.000800.
</cite> [<a href="https://doi.org/10.1099/mic.0.000800" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31124782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology%20(Reading,%20England)&amp;title=Microbe%20Profile:%20Listeria%20monocytogenes:%20A%20Paradigm%20Among%20Intracellular%20Bacterial%20Pathogens&amp;author=J.%20Pizarro%E2%80%90Cerd%C3%A1&amp;author=P.%20Cossart&amp;volume=165&amp;issue=7&amp;publication_year=2019&amp;pages=719-721&amp;pmid=31124782&amp;doi=10.1099/mic.0.000800&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0148">
<cite>

Pleitner, A. M.
, Trinetta V., Morgan M. T., Linton R. L., and Oliver H. F.. 2014. “Transcriptional and Phenotypic Responses of <em>Listeria monocytogenes</em> to Chlorine Dioxide.” Applied and Environmental Microbiology
80, no. 9: 2951–2963. 10.1128/AEM.00004-14.
</cite> [<a href="https://doi.org/10.1128/AEM.00004-14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3993309/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24610841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=Transcriptional%20and%20Phenotypic%20Responses%20of%20Listeria%20monocytogenes%20to%20Chlorine%20Dioxide&amp;author=A.%20M.%20Pleitner&amp;author=V.%20Trinetta&amp;author=M.%20T.%20Morgan&amp;author=R.%20L.%20Linton&amp;author=H.%20F.%20Oliver&amp;volume=80&amp;issue=9&amp;publication_year=2014&amp;pages=2951-2963&amp;pmid=24610841&amp;doi=10.1128/AEM.00004-14&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0149">
<cite>

Poimenidou, S. V.
, Caccia N., Paramithiotis S., Hébraud M., Nychas G.‐J., and Skandamis P. N.. 2023. “Influence of Temperature on Regulation of Key Virulence and Stress Response Genes in <em>Listeria monocytogenes</em> Biofilms.” Food Microbiology
111, no. May: 104190. 10.1016/j.fm.2022.104190.
</cite> [<a href="https://doi.org/10.1016/j.fm.2022.104190" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36681396/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Microbiology&amp;title=Influence%20of%20Temperature%20on%20Regulation%20of%20Key%20Virulence%20and%20Stress%20Response%20Genes%20in%20Listeria%20monocytogenes%20Biofilms&amp;author=S.%20V.%20Poimenidou&amp;author=N.%20Caccia&amp;author=S.%20Paramithiotis&amp;author=M.%20H%C3%A9braud&amp;author=G.%E2%80%90J.%20Nychas&amp;volume=111&amp;issue=May&amp;publication_year=2023&amp;pages=104190&amp;pmid=36681396&amp;doi=10.1016/j.fm.2022.104190&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0150">
<cite>

Popowska, M.
, Osińska M., and Rzeczkowska M.. 2012. “
<em>N</em>‐Acetylglucosamine‐6‐Phosphate Deacetylase (NagA) of <em>Listeria monocytogenes</em> EGD, an Essential Enzyme for the Metabolism and Recycling of Amino Sugars.” Archives of Microbiology
194, no. 4: 255–268. 10.1007/s00203-011-0752-3.
</cite> [<a href="https://doi.org/10.1007/s00203-011-0752-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3304070/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21947170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Archives%20of%20Microbiology&amp;title=N%E2%80%90Acetylglucosamine%E2%80%906%E2%80%90Phosphate%20Deacetylase%20(NagA)%20of%20Listeria%20monocytogenes%20EGD,%20an%20Essential%20Enzyme%20for%20the%20Metabolism%20and%20Recycling%20of%20Amino%20Sugars&amp;author=M.%20Popowska&amp;author=M.%20Osi%C5%84ska&amp;author=M.%20Rzeczkowska&amp;volume=194&amp;issue=4&amp;publication_year=2012&amp;pages=255-268&amp;pmid=21947170&amp;doi=10.1007/s00203-011-0752-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0151">
<cite>

Potempa, J.
, and Pike R. N.. 2004. “Bacterial Peptidases.” In Contributions to Microbiology, edited by Russell W. and Herwald H., 12: 132–180. Karger. 10.1159/000081693.</cite> [<a href="https://doi.org/10.1159/000081693" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15496780/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Contributions%20to%20Microbiology&amp;author=J.%20Potempa&amp;author=R.%20N.%20Pike&amp;publication_year=2004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0152">
<cite>

Preda, V. G.
, and Săndulescu O.. 2019. “Communication Is the Key: Biofilms, Quorum Sensing, Formation and Prevention.” Discoveries
7, no. 3: e100. 10.15190/d.2019.13.
</cite> [<a href="https://doi.org/10.15190/d.2019.13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7086079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32309618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Discoveries&amp;title=Communication%20Is%20the%20Key:%20Biofilms,%20Quorum%20Sensing,%20Formation%20and%20Prevention&amp;author=V.%20G.%20Preda&amp;author=O.%20S%C4%83ndulescu&amp;volume=7&amp;issue=3&amp;publication_year=2019&amp;pages=e100&amp;pmid=32309618&amp;doi=10.15190/d.2019.13&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0153">
<cite>

Primerano, D. A.
, and Burns R. O.. 1983. “Role of Acetohydroxy Acid Isomeroreductase in Biosynthesis of Pantothenic Acid in Salmonella Typhimurium.” Journal of Bacteriology
153, no. 1: 259. 10.1128/jb.153.1.259-269.1983.
</cite> [<a href="https://doi.org/10.1128/jb.153.1.259-269.1983" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC217364/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6401279/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Bacteriology&amp;title=Role%20of%20Acetohydroxy%20Acid%20Isomeroreductase%20in%20Biosynthesis%20of%20Pantothenic%20Acid%20in%20Salmonella%20Typhimurium&amp;author=D.%20A.%20Primerano&amp;author=R.%20O.%20Burns&amp;volume=153&amp;issue=1&amp;publication_year=1983&amp;pages=259&amp;pmid=6401279&amp;doi=10.1128/jb.153.1.259-269.1983&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0154">
<cite>

Pucciarelli, M. G.
, Bierne H., and Portillo F. G.‐d.. 2007. “The Cell Wall of <em>Listeria monocytogenes</em> and Its Role in Pathogenicity.” In Listeria monocytogenes: Pathogenesis and Host Response, edited by Goldfine Howard and Shen Hao, 81–110. Springer US. 10.1007/978-0-387-49376-3_5.</cite> [<a href="https://doi.org/10.1007/978-0-387-49376-3_5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Listeria%20monocytogenes:%20Pathogenesis%20and%20Host%20Response&amp;author=M.%20G.%20Pucciarelli&amp;author=H.%20Bierne&amp;author=F.%20G.%E2%80%90d.%20Portillo&amp;publication_year=2007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0155">
<cite>

Qi, H.
, Wright R. H. G., Beato M., and Price B. D.. 2022. “The ADP‐Ribose Hydrolase NUDT5 Is Important for DNA Repair.” Cell Reports
41, no. 12: 111866. 10.1016/j.celrep.2022.111866.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2022.111866" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36543120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Reports&amp;title=The%20ADP%E2%80%90Ribose%20Hydrolase%20NUDT5%20Is%20Important%20for%20DNA%20Repair&amp;author=H.%20Qi&amp;author=R.%20H.%20G.%20Wright&amp;author=M.%20Beato&amp;author=B.%20D.%20Price&amp;volume=41&amp;issue=12&amp;publication_year=2022&amp;pages=111866&amp;pmid=36543120&amp;doi=10.1016/j.celrep.2022.111866&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0156">
<cite>

Qian, J.
, Zhang M., Dai C., Huo S., and Ma H.. 2020. “Transcriptomic Analysis of <em>Listeria monocytogenes</em> Under Pulsed Magnetic Field Treatment.” Food Research International
133, no. July: 109195. 10.1016/j.foodres.2020.109195.
</cite> [<a href="https://doi.org/10.1016/j.foodres.2020.109195" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32466941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Research%20International&amp;title=Transcriptomic%20Analysis%20of%20Listeria%20monocytogenes%20Under%20Pulsed%20Magnetic%20Field%20Treatment&amp;author=J.%20Qian&amp;author=M.%20Zhang&amp;author=C.%20Dai&amp;author=S.%20Huo&amp;author=H.%20Ma&amp;volume=133&amp;issue=July&amp;publication_year=2020&amp;pages=109195&amp;pmid=32466941&amp;doi=10.1016/j.foodres.2020.109195&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0157">
<cite>

Ranjbar, S.
, Shahmansouri M., Attri P., and Bogaerts A.. 2020. “Effect of Plasma‐Induced Oxidative Stress on the Glycolysis Pathway of <em>Escherichia coli</em>
.” Computers in Biology and Medicine
127, no. December: 104064. 10.1016/j.compbiomed.2020.104064.
</cite> [<a href="https://doi.org/10.1016/j.compbiomed.2020.104064" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33171288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Computers%20in%20Biology%20and%20Medicine&amp;title=Effect%20of%20Plasma%E2%80%90Induced%20Oxidative%20Stress%20on%20the%20Glycolysis%20Pathway%20of%20Escherichia%20coli&amp;author=S.%20Ranjbar&amp;author=M.%20Shahmansouri&amp;author=P.%20Attri&amp;author=A.%20Bogaerts&amp;volume=127&amp;issue=December&amp;publication_year=2020&amp;pages=104064&amp;pmid=33171288&amp;doi=10.1016/j.compbiomed.2020.104064&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0158">
<cite>

Raux, E.
, Schubert H. L., and Warren M. J.. 2000. “Biosynthesis of Cobalamin (Vitamin B<sub>12</sub>): A Bacterial Conundrum.” Cellular and Molecular Life Sciences: CMLS
57, no. 13–14: 1880–1893. 10.1007/PL00000670.
</cite> [<a href="https://doi.org/10.1007/PL00000670" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11147154/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11215515/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cellular%20and%20Molecular%20Life%20Sciences:%20CMLS&amp;title=Biosynthesis%20of%20Cobalamin%20(Vitamin%20B12):%20A%20Bacterial%20Conundrum&amp;author=E.%20Raux&amp;author=H.%20L.%20Schubert&amp;author=M.%20J.%20Warren&amp;volume=57&amp;issue=13%E2%80%9314&amp;publication_year=2000&amp;pages=1880-1893&amp;pmid=11215515&amp;doi=10.1007/PL00000670&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0159">
<cite>

Reed, P.
, Atilano M. L., Alves R., et al. 2015. “
<em>Staphylococcus aureus</em> Survives With a Minimal Peptidoglycan Synthesis Machine but Sacrifices Virulence and Antibiotic Resistance.” PLoS Pathogens
11, no. 5: e1004891. 10.1371/journal.ppat.1004891.
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1004891" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4423922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25951442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathogens&amp;title=Staphylococcus%20aureus%20Survives%20With%20a%20Minimal%20Peptidoglycan%20Synthesis%20Machine%20but%20Sacrifices%20Virulence%20and%20Antibiotic%20Resistance&amp;author=P.%20Reed&amp;author=M.%20L.%20Atilano&amp;author=R.%20Alves&amp;volume=11&amp;issue=5&amp;publication_year=2015&amp;pages=e1004891&amp;pmid=25951442&amp;doi=10.1371/journal.ppat.1004891&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0160">
<cite>

Regulski, E. E.
, Moy R. H., Weinberg Z., et al. 2008. “A Widespread Riboswitch Candidate That Controls Bacterial Genes Involved in Molybdenum Cofactor and Tungsten Cofactor Metabolism.” Molecular Microbiology
68, no. 4: 918–932. 10.1111/j.1365-2958.2008.06208.x.
</cite> [<a href="https://doi.org/10.1111/j.1365-2958.2008.06208.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2408646/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18363797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Microbiology&amp;title=A%20Widespread%20Riboswitch%20Candidate%20That%20Controls%20Bacterial%20Genes%20Involved%20in%20Molybdenum%20Cofactor%20and%20Tungsten%20Cofactor%20Metabolism&amp;author=E.%20E.%20Regulski&amp;author=R.%20H.%20Moy&amp;author=Z.%20Weinberg&amp;volume=68&amp;issue=4&amp;publication_year=2008&amp;pages=918-932&amp;pmid=18363797&amp;doi=10.1111/j.1365-2958.2008.06208.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0161">
<cite>

Riedel, C. U.
, Monk I. R., Casey P. G., Waidmann M. S., Gahan C. G. M., and Hill C.. 2009. “AgrD‐Dependent Quorum Sensing Affects Biofilm Formation, Invasion, Virulence and Global Gene Expression Profiles in <em>Listeria monocytogenes</em>
.” Molecular Microbiology
71, no. 5: 1177–1189. 10.1111/j.1365-2958.2008.06589.x.
</cite> [<a href="https://doi.org/10.1111/j.1365-2958.2008.06589.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19154329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Microbiology&amp;title=AgrD%E2%80%90Dependent%20Quorum%20Sensing%20Affects%20Biofilm%20Formation,%20Invasion,%20Virulence%20and%20Global%20Gene%20Expression%20Profiles%20in%20Listeria%20monocytogenes&amp;author=C.%20U.%20Riedel&amp;author=I.%20R.%20Monk&amp;author=P.%20G.%20Casey&amp;author=M.%20S.%20Waidmann&amp;author=C.%20G.%20M.%20Gahan&amp;volume=71&amp;issue=5&amp;publication_year=2009&amp;pages=1177-1189&amp;pmid=19154329&amp;doi=10.1111/j.1365-2958.2008.06589.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0162">
<cite>

Rieu, A.
, Weidmann S., Garmyn D., Piveteau P., and Guzzo J.. 2007. “Agr System of <em>Listeria monocytogenes</em> EGD‐e: Role in Adherence and Differential Expression Pattern.” Applied and Environmental Microbiology
73, no. 19: 6125–6133. 10.1128/AEM.00608-07.
</cite> [<a href="https://doi.org/10.1128/AEM.00608-07" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2075002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17675424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=Agr%20System%20of%20Listeria%20monocytogenes%20EGD%E2%80%90e:%20Role%20in%20Adherence%20and%20Differential%20Expression%20Pattern&amp;author=A.%20Rieu&amp;author=S.%20Weidmann&amp;author=D.%20Garmyn&amp;author=P.%20Piveteau&amp;author=J.%20Guzzo&amp;volume=73&amp;issue=19&amp;publication_year=2007&amp;pages=6125-6133&amp;pmid=17675424&amp;doi=10.1128/AEM.00608-07&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0163">
<cite>

Rivera‐Lugo, R.
, Deng D., Anaya‐Sanchez A., et al. 2022. “
<em>Listeria monocytogenes</em> Requires Cellular Respiration for NAD+ Regeneration and Pathogenesis.” Elife
11: e75424. 10.7554/eLife.75424.
</cite> [<a href="https://doi.org/10.7554/eLife.75424" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9094743/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35380108/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Elife&amp;title=Listeria%20monocytogenes%20Requires%20Cellular%20Respiration%20for%20NAD+%20Regeneration%20and%20Pathogenesis&amp;author=R.%20Rivera%E2%80%90Lugo&amp;author=D.%20Deng&amp;author=A.%20Anaya%E2%80%90Sanchez&amp;volume=11&amp;publication_year=2022&amp;pages=e75424&amp;pmid=35380108&amp;doi=10.7554/eLife.75424&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0164">
<cite>

Robinson, B. R.
, and D'Amico D. J.. 2021. “Hydrogen Peroxide Treatments for the Control of <em>Listeria monocytogenes</em> on High‐Moisture Soft Cheese.” International Dairy Journal
114, no. March: 104931. 10.1016/j.idairyj.2020.104931.</cite> [<a href="https://doi.org/10.1016/j.idairyj.2020.104931" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Dairy%20Journal&amp;title=Hydrogen%20Peroxide%20Treatments%20for%20the%20Control%20of%20Listeria%20monocytogenes%20on%20High%E2%80%90Moisture%20Soft%20Cheese&amp;author=B.%20R.%20Robinson&amp;author=D.%20J.%20D'Amico&amp;volume=114&amp;issue=March&amp;publication_year=2021&amp;pages=104931&amp;doi=10.1016/j.idairyj.2020.104931&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0165">
<cite>

Rodgers, S. L.
, Cash J. N., Siddiq M., and Ryser E. T.. 2004. “A Comparison of Different Chemical Sanitizers for Inactivating <em>Escherichia coli</em> O157:H7 and <em>Listeria monocytogenes</em> in Solution and on Apples, Lettuce, Strawberries, and Cantaloupe.” Journal of Food Protection
67, no. 4: 721–731. 10.4315/0362-028X-67.4.721.
</cite> [<a href="https://doi.org/10.4315/0362-028X-67.4.721" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15083724/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Food%20Protection&amp;title=A%20Comparison%20of%20Different%20Chemical%20Sanitizers%20for%20Inactivating%20Escherichia%20coli%20O157:H7%20and%20Listeria%20monocytogenes%20in%20Solution%20and%20on%20Apples,%20Lettuce,%20Strawberries,%20and%20Cantaloupe&amp;author=S.%20L.%20Rodgers&amp;author=J.%20N.%20Cash&amp;author=M.%20Siddiq&amp;author=E.%20T.%20Ryser&amp;volume=67&amp;issue=4&amp;publication_year=2004&amp;pages=721-731&amp;pmid=15083724&amp;doi=10.4315/0362-028X-67.4.721&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0166">
<cite>

Rodriguez, C.
, Taminiau B., García‐Fuentes E., Daube G., and Korsak N.. 2021. “
<em>Listeria monocytogenes</em> Dissemination in Farming and Primary Production: Sources, Shedding and Control Measures.” Food Control
120, no. February: 107540. 10.1016/j.foodcont.2020.107540.</cite> [<a href="https://doi.org/10.1016/j.foodcont.2020.107540" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Control&amp;title=Listeria%20monocytogenes%20Dissemination%20in%20Farming%20and%20Primary%20Production:%20Sources,%20Shedding%20and%20Control%20Measures&amp;author=C.%20Rodriguez&amp;author=B.%20Taminiau&amp;author=E.%20Garc%C3%ADa%E2%80%90Fuentes&amp;author=G.%20Daube&amp;author=N.%20Korsak&amp;volume=120&amp;issue=February&amp;publication_year=2021&amp;pages=107540&amp;doi=10.1016/j.foodcont.2020.107540&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0255">
<cite>

Roemhild, R.
, Bollenbach T., and Andersson D. I.. 2022. “The Physiology and Genetics of Bacterial Responses to Antibiotic Combinations.” Nature Reviews Microbiology
20, no. 8: 478–490. 10.1038/s41579-022-00700-5.
</cite> [<a href="https://doi.org/10.1038/s41579-022-00700-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35241807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Microbiology&amp;title=The%20Physiology%20and%20Genetics%20of%20Bacterial%20Responses%20to%20Antibiotic%20Combinations&amp;author=R.%20Roemhild&amp;author=T.%20Bollenbach&amp;author=D.%20I.%20Andersson&amp;volume=20&amp;issue=8&amp;publication_year=2022&amp;pages=478-490&amp;pmid=35241807&amp;doi=10.1038/s41579-022-00700-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0167">
<cite>

Roller, B. R. K.
, Stoddard S. F., and Schmidt T. M.. 2016. “Exploiting rRNA Operon Copy Number to Investigate Bacterial Reproductive Strategies.” Nature Microbiology
1, no. September: 16160. 10.1038/nmicrobiol.2016.160.</cite> [<a href="https://doi.org/10.1038/nmicrobiol.2016.160" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5061577/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27617693/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Microbiology&amp;title=Exploiting%20rRNA%20Operon%20Copy%20Number%20to%20Investigate%20Bacterial%20Reproductive%20Strategies&amp;author=B.%20R.%20K.%20Roller&amp;author=S.%20F.%20Stoddard&amp;author=T.%20M.%20Schmidt&amp;volume=1&amp;issue=September&amp;publication_year=2016&amp;pages=16160&amp;pmid=27617693&amp;doi=10.1038/nmicrobiol.2016.160&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0168">
<cite>

Romano, A. H.
, and Conway T.. 1996. “Evolution of Carbohydrate Metabolic Pathways.” Research in Microbiology
147, no. 6: 448–455. 10.1016/0923-2508(96)83998-2.
</cite> [<a href="https://doi.org/10.1016/0923-2508(96)83998-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9084754/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Research%20in%20Microbiology&amp;title=Evolution%20of%20Carbohydrate%20Metabolic%20Pathways&amp;author=A.%20H.%20Romano&amp;author=T.%20Conway&amp;volume=147&amp;issue=6&amp;publication_year=1996&amp;pages=448-455&amp;pmid=9084754&amp;doi=10.1016/0923-2508(96)83998-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0169">
<cite>

Romanova, N. A.
, Wolffs P. F. G., Brovko L. Y., and Griffiths M. W.. 2006. “Role of Efflux Pumps in Adaptation and Resistance of <em>Listeria monocytogenes</em> to Benzalkonium Chloride.” Applied and Environmental Microbiology
72, no. 5: 3498–3503. 10.1128/AEM.72.5.3498-3503.2006.
</cite> [<a href="https://doi.org/10.1128/AEM.72.5.3498-3503.2006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1472371/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16672496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=Role%20of%20Efflux%20Pumps%20in%20Adaptation%20and%20Resistance%20of%20Listeria%20monocytogenes%20to%20Benzalkonium%20Chloride&amp;author=N.%20A.%20Romanova&amp;author=P.%20F.%20G.%20Wolffs&amp;author=L.%20Y.%20Brovko&amp;author=M.%20W.%20Griffiths&amp;volume=72&amp;issue=5&amp;publication_year=2006&amp;pages=3498-3503&amp;pmid=16672496&amp;doi=10.1128/AEM.72.5.3498-3503.2006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0170">
<cite>

Roncarati, D.
, and Scarlato V.. 2017. “Regulation of Heat‐Shock Genes in Bacteria: from Signal Sensing to Gene Expression Output.” FEMS Microbiology Reviews
41, no. 4: 549–574. 10.1093/femsre/fux015.
</cite> [<a href="https://doi.org/10.1093/femsre/fux015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28402413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiology%20Reviews&amp;title=Regulation%20of%20Heat%E2%80%90Shock%20Genes%20in%20Bacteria:%20from%20Signal%20Sensing%20to%20Gene%20Expression%20Output&amp;author=D.%20Roncarati&amp;author=V.%20Scarlato&amp;volume=41&amp;issue=4&amp;publication_year=2017&amp;pages=549-574&amp;pmid=28402413&amp;doi=10.1093/femsre/fux015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0171">
<cite>

Roth, J. R.
, Lawrence J. G., and Bobik T. A.. 1996. “Cobalamin (Coenzyme B<sub>12</sub>): Synthesis and Biological Significance.” Annual Review of Microbiology
50: 137–181. 10.1146/annurev.micro.50.1.137.</cite> [<a href="https://doi.org/10.1146/annurev.micro.50.1.137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8905078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Microbiology&amp;title=Cobalamin%20(Coenzyme%20B12):%20Synthesis%20and%20Biological%20Significance&amp;author=J.%20R.%20Roth&amp;author=J.%20G.%20Lawrence&amp;author=T.%20A.%20Bobik&amp;volume=50&amp;publication_year=1996&amp;pages=137-181&amp;pmid=8905078&amp;doi=10.1146/annurev.micro.50.1.137&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0172">
<cite>

Rothwell, J. G.
, Alam D., Carter D. A., et al. 2022. “The Antimicrobial Efficacy of Plasma‐Activated Water Against <em>Listeria</em> and <em>E. coli</em> Is Modulated by Reactor Design and Water Composition.” Journal of Applied Microbiology
132, no. 4: 2490–2500. 10.1111/jam.15429.
</cite> [<a href="https://doi.org/10.1111/jam.15429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34957649/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Applied%20Microbiology&amp;title=The%20Antimicrobial%20Efficacy%20of%20Plasma%E2%80%90Activated%20Water%20Against%20Listeria%20and%20E.%20coli%20Is%20Modulated%20by%20Reactor%20Design%20and%20Water%20Composition&amp;author=J.%20G.%20Rothwell&amp;author=D.%20Alam&amp;author=D.%20A.%20Carter&amp;volume=132&amp;issue=4&amp;publication_year=2022&amp;pages=2490-2500&amp;pmid=34957649&amp;doi=10.1111/jam.15429&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0249">
<cite>

Rouhi, A.
, Azghandi M., Ali Mortazavi S., Tabatabaei‐Yazdi F., and Vasiee A.. 2024. “Exploring the Anti‐Biofilm Activity and Suppression of Virulence Genes Expression by Thanatin in Listeria Monocytogenes.” LWT
199, no. May: 116084. 10.1016/j.lwt.2024.116084.</cite> [<a href="https://doi.org/10.1016/j.lwt.2024.116084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=LWT&amp;title=Exploring%20the%20Anti%E2%80%90Biofilm%20Activity%20and%20Suppression%20of%20Virulence%20Genes%20Expression%20by%20Thanatin%20in%20Listeria%20Monocytogenes&amp;author=A.%20Rouhi&amp;author=M.%20Azghandi&amp;author=S.%20Ali%20Mortazavi&amp;author=F.%20Tabatabaei%E2%80%90Yazdi&amp;author=A.%20Vasiee&amp;volume=199&amp;issue=May&amp;publication_year=2024&amp;pages=116084&amp;doi=10.1016/j.lwt.2024.116084&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0173">
<cite>

Rozman, U.
, Pušnik M., Kmetec S., Duh D., and Turk S. Š.. 2021. “Reduced Susceptibility and Increased Resistance of Bacteria Against Disinfectants: A Systematic Review.” Microorganisms
9, no. 12: 2550. 10.3390/microorganisms9122550.
</cite> [<a href="https://doi.org/10.3390/microorganisms9122550" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8706950/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34946151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microorganisms&amp;title=Reduced%20Susceptibility%20and%20Increased%20Resistance%20of%20Bacteria%20Against%20Disinfectants:%20A%20Systematic%20Review&amp;author=U.%20Rozman&amp;author=M.%20Pu%C5%A1nik&amp;author=S.%20Kmetec&amp;author=D.%20Duh&amp;author=S.%20%C5%A0.%20Turk&amp;volume=9&amp;issue=12&amp;publication_year=2021&amp;pages=2550&amp;pmid=34946151&amp;doi=10.3390/microorganisms9122550&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0174">
<cite>

Ruiz, S. J.
, Schuurman‐Wolters G. K., and Poolman B.. 2016. “Crystal Structure of the Substrate‐Binding Domain From <em>Listeria monocytogenes</em> Bile‐Resistance Determinant BilE.” Crystals
6, no. 12: 162. 10.3390/cryst6120162.</cite> [<a href="https://doi.org/10.3390/cryst6120162" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crystals&amp;title=Crystal%20Structure%20of%20the%20Substrate%E2%80%90Binding%20Domain%20From%20Listeria%20monocytogenes%20Bile%E2%80%90Resistance%20Determinant%20BilE&amp;author=S.%20J.%20Ruiz&amp;author=G.%20K.%20Schuurman%E2%80%90Wolters&amp;author=B.%20Poolman&amp;volume=6&amp;issue=12&amp;publication_year=2016&amp;pages=162&amp;doi=10.3390/cryst6120162&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0175">
<cite>

Rutherford, S. T.
, and Bassler B. L.. 2012. “Bacterial Quorum Sensing: Its Role in Virulence and Possibilities for Its Control.” Cold Spring Harbor Perspectives in Medicine
2, no. 11: a012427. 10.1101/cshperspect.a012427.
</cite> [<a href="https://doi.org/10.1101/cshperspect.a012427" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3543102/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23125205/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harbor%20Perspectives%20in%20Medicine&amp;title=Bacterial%20Quorum%20Sensing:%20Its%20Role%20in%20Virulence%20and%20Possibilities%20for%20Its%20Control&amp;author=S.%20T.%20Rutherford&amp;author=B.%20L.%20Bassler&amp;volume=2&amp;issue=11&amp;publication_year=2012&amp;pages=a012427&amp;pmid=23125205&amp;doi=10.1101/cshperspect.a012427&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0176">
<cite>

Salam, M. A.
, Al‐Amin M. Y., Salam M. T., et al. 2023. “Antimicrobial Resistance: A Growing Serious Threat for Global Public Health.” Healthcare
11, no. 13: 1946. 10.3390/healthcare11131946.
</cite> [<a href="https://doi.org/10.3390/healthcare11131946" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10340576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37444780/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Healthcare&amp;title=Antimicrobial%20Resistance:%20A%20Growing%20Serious%20Threat%20for%20Global%20Public%20Health&amp;author=M.%20A.%20Salam&amp;author=M.%20Y.%20Al%E2%80%90Amin&amp;author=M.%20T.%20Salam&amp;volume=11&amp;issue=13&amp;publication_year=2023&amp;pages=1946&amp;pmid=37444780&amp;doi=10.3390/healthcare11131946&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0177">
<cite>

Santos‐Beneit, F.

2015. “The Pho Regulon: A Huge Regulatory Network in Bacteria.” Frontiers in Microbiology
6, no. April: 402. 10.3389/fmicb.2015.00402.
</cite> [<a href="https://doi.org/10.3389/fmicb.2015.00402" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4415409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25983732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=The%20Pho%20Regulon:%20A%20Huge%20Regulatory%20Network%20in%20Bacteria&amp;author=F.%20Santos%E2%80%90Beneit&amp;volume=6&amp;issue=April&amp;publication_year=2015&amp;pages=402&amp;pmid=25983732&amp;doi=10.3389/fmicb.2015.00402&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0178">
<cite>

Sazykin, I. S.
, and Sazykina M. A.. 2023. “The Role of Oxidative Stress in Genome Destabilization and Adaptive Evolution of Bacteria.” Gene
857, no. March: 147170. 10.1016/j.gene.2023.147170.
</cite> [<a href="https://doi.org/10.1016/j.gene.2023.147170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36623672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gene&amp;title=The%20Role%20of%20Oxidative%20Stress%20in%20Genome%20Destabilization%20and%20Adaptive%20Evolution%20of%20Bacteria&amp;author=I.%20S.%20Sazykin&amp;author=M.%20A.%20Sazykina&amp;volume=857&amp;issue=March&amp;publication_year=2023&amp;pages=147170&amp;pmid=36623672&amp;doi=10.1016/j.gene.2023.147170&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0179">
<cite>

Schär, J.
, Stoll R., Schauer K., et al. 2010. “Pyruvate Carboxylase Plays a Crucial Role in Carbon Metabolism of Extra‐ and Intracellularly Replicating <em>Listeria monocytogenes</em>
.” Journal of Bacteriology
192, no. 7: 1774–1784. 10.1128/JB.01132-09.
</cite> [<a href="https://doi.org/10.1128/JB.01132-09" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2838035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20097852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Bacteriology&amp;title=Pyruvate%20Carboxylase%20Plays%20a%20Crucial%20Role%20in%20Carbon%20Metabolism%20of%20Extra%E2%80%90%20and%20Intracellularly%20Replicating%20Listeria%20monocytogenes&amp;author=J.%20Sch%C3%A4r&amp;author=R.%20Stoll&amp;author=K.%20Schauer&amp;volume=192&amp;issue=7&amp;publication_year=2010&amp;pages=1774-1784&amp;pmid=20097852&amp;doi=10.1128/JB.01132-09&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0180">
<cite>

Schauer, K.
, Geginat G., Liang C., Goebel W., Dandekar T., and Fuchs T. M.. 2010. “Deciphering the Intracellular Metabolism of <em>Listeria monocytogenes</em> by Mutant Screening and Modelling.” BMC Genomics [Electronic Resource]
11, no. 1: 573. 10.1186/1471-2164-11-573.
</cite> [<a href="https://doi.org/10.1186/1471-2164-11-573" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3091722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20955543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics%20%5BElectronic%20Resource%5D&amp;title=Deciphering%20the%20Intracellular%20Metabolism%20of%20Listeria%20monocytogenes%20by%20Mutant%20Screening%20and%20Modelling&amp;author=K.%20Schauer&amp;author=G.%20Geginat&amp;author=C.%20Liang&amp;author=W.%20Goebel&amp;author=T.%20Dandekar&amp;volume=11&amp;issue=1&amp;publication_year=2010&amp;pages=573&amp;pmid=20955543&amp;doi=10.1186/1471-2164-11-573&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0181">
<cite>

Schmid, B.
, Klumpp J., Raimann E., Mj L., Stephan R., and Tasara T.
2009. “Role of Cold Shock Proteins in Growth of <em>Listeria monocytogenes</em> Under Cold and Osmotic Stress Conditions.” Applied and Environmental Microbiology
75, no. 6: 1621–1627. 10.1128/AEM.02154-08.
</cite> [<a href="https://doi.org/10.1128/AEM.02154-08" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2655451/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19151183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=Role%20of%20Cold%20Shock%20Proteins%20in%20Growth%20of%20Listeria%20monocytogenes%20Under%20Cold%20and%20Osmotic%20Stress%20Conditions&amp;author=B.%20Schmid&amp;author=J.%20Klumpp&amp;author=E.%20Raimann&amp;author=L.%20Mj&amp;author=R.%20Stephan&amp;volume=75&amp;issue=6&amp;publication_year=2009&amp;pages=1621-1627&amp;pmid=19151183&amp;doi=10.1128/AEM.02154-08&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0182">
<cite>

Schmid, J.
, Heider D., Wendel N. J., Sperl N., and Sieber V.. 2016. “Bacterial Glycosyltransferases: Challenges and Opportunities of a Highly Diverse Enzyme Class Toward Tailoring Natural Products.” Frontiers in Microbiology
7, no. February: 182. 10.3389/fmicb.2016.00182.
</cite> [<a href="https://doi.org/10.3389/fmicb.2016.00182" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4757703/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26925049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Bacterial%20Glycosyltransferases:%20Challenges%20and%20Opportunities%20of%20a%20Highly%20Diverse%20Enzyme%20Class%20Toward%20Tailoring%20Natural%20Products&amp;author=J.%20Schmid&amp;author=D.%20Heider&amp;author=N.%20J.%20Wendel&amp;author=N.%20Sperl&amp;author=V.%20Sieber&amp;volume=7&amp;issue=February&amp;publication_year=2016&amp;pages=182&amp;pmid=26925049&amp;doi=10.3389/fmicb.2016.00182&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0183">
<cite>

Schulz, L. M.
, Rothe P., Halbedel S., Gründling A., and Rismondo J.. 2022. “Imbalance of Peptidoglycan Biosynthesis Alters the Cell Surface Charge of <em>Listeria monocytogenes</em>
.” The Cell Surface
8, no. October: 100085. 10.1016/j.tcsw.2022.100085.
</cite> [<a href="https://doi.org/10.1016/j.tcsw.2022.100085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9593813/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36304571/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Cell%20Surface&amp;title=Imbalance%20of%20Peptidoglycan%20Biosynthesis%20Alters%20the%20Cell%20Surface%20Charge%20of%20Listeria%20monocytogenes&amp;author=L.%20M.%20Schulz&amp;author=P.%20Rothe&amp;author=S.%20Halbedel&amp;author=A.%20Gr%C3%BCndling&amp;author=J.%20Rismondo&amp;volume=8&amp;issue=October&amp;publication_year=2022&amp;pages=100085&amp;pmid=36304571&amp;doi=10.1016/j.tcsw.2022.100085&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0243">
<cite>

Schulz, L. M.
, Dreier F., Miranda L. M. D. E. S., and Rismondo J.. 2023. “Adaptation Mechanisms of Listeria Monocytogenes to Quaternary Ammonium Compounds.” Microbiology Spectrum
11, no. 5: e01441–23. 10.1128/spectrum.01441-23.
</cite> [<a href="https://doi.org/10.1128/spectrum.01441-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10580936/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37695041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology%20Spectrum&amp;title=Adaptation%20Mechanisms%20of%20Listeria%20Monocytogenes%20to%20Quaternary%20Ammonium%20Compounds&amp;author=L.%20M.%20Schulz&amp;author=F.%20Dreier&amp;author=L.%20M.%20D.%20E.%20S.%20Miranda&amp;author=J.%20Rismondo&amp;volume=11&amp;issue=5&amp;publication_year=2023&amp;pages=e01441-23&amp;pmid=37695041&amp;doi=10.1128/spectrum.01441-23&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0259">
<cite>

Seixas, A.
, Filipe A., Filipa Q., et al. 2025. “The RNA Chaperone Hfq Is a Novel Regulator of Catalase Expression and Hydrogen Peroxide‐Induced Oxidative Stress Response in Listeria Monocytogenes EGD‐e.” Free Radical Biology &amp; Medicine
227, no. February: 103–116. 10.1016/j.freeradbiomed.2024.11.038.
</cite> [<a href="https://doi.org/10.1016/j.freeradbiomed.2024.11.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39608557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Free%20Radical%20Biology%20&amp;%20Medicine&amp;title=The%20RNA%20Chaperone%20Hfq%20Is%20a%20Novel%20Regulator%20of%20Catalase%20Expression%20and%20Hydrogen%20Peroxide%E2%80%90Induced%20Oxidative%20Stress%20Response%20in%20Listeria%20Monocytogenes%20EGD%E2%80%90e&amp;author=A.%20Seixas&amp;author=A.%20Filipe&amp;author=Q.%20Filipa&amp;volume=227&amp;issue=February&amp;publication_year=2025&amp;pages=103-116&amp;pmid=39608557&amp;doi=10.1016/j.freeradbiomed.2024.11.038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0184">
<cite>

Sezgin, E.
, and Tekin B.. 2023. “Molecular Evolution and Population Genetics of Glutamate Decarboxylase Acid Resistance Pathway in Lactic Acid Bacteria.” Frontiers in Genetics
14, no. January: 1027156. 10.3389/fgene.2023.1027156.
</cite> [<a href="https://doi.org/10.3389/fgene.2023.1027156" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9909107/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36777729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Molecular%20Evolution%20and%20Population%20Genetics%20of%20Glutamate%20Decarboxylase%20Acid%20Resistance%20Pathway%20in%20Lactic%20Acid%20Bacteria&amp;author=E.%20Sezgin&amp;author=B.%20Tekin&amp;volume=14&amp;issue=January&amp;publication_year=2023&amp;pages=1027156&amp;pmid=36777729&amp;doi=10.3389/fgene.2023.1027156&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0185">
<cite>

Sharma, S.
, Jaiswal S., Duffy B., and Jaiswal A. K.. 2022. “Advances in Emerging Technologies for the Decontamination of the Food Contact Surfaces.” Food Research International
151: 110865.
</cite> [<a href="https://doi.org/10.1016/j.foodres.2021.110865" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34980401/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Research%20International&amp;title=Advances%20in%20Emerging%20Technologies%20for%20the%20Decontamination%20of%20the%20Food%20Contact%20Surfaces&amp;author=S.%20Sharma&amp;author=S.%20Jaiswal&amp;author=B.%20Duffy&amp;author=A.%20K.%20Jaiswal&amp;volume=151&amp;publication_year=2022&amp;pages=110865&amp;pmid=34980401&amp;doi=10.1016/j.foodres.2021.110865&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0186">
<cite>

Shen, A.
, and Higgins D. E.. 2006. “The MogR Transcriptional Repressor Regulates Nonhierarchical Expression of Flagellar Motility Genes and Virulence in <em>Listeria monocytogenes</em>
.” PLoS Pathogens
2, no. 4: e30. 10.1371/journal.ppat.0020030.
</cite> [<a href="https://doi.org/10.1371/journal.ppat.0020030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1435793/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16617375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathogens&amp;title=The%20MogR%20Transcriptional%20Repressor%20Regulates%20Nonhierarchical%20Expression%20of%20Flagellar%20Motility%20Genes%20and%20Virulence%20in%20Listeria%20monocytogenes&amp;author=A.%20Shen&amp;author=D.%20E.%20Higgins&amp;volume=2&amp;issue=4&amp;publication_year=2006&amp;pages=e30&amp;pmid=16617375&amp;doi=10.1371/journal.ppat.0020030&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0187">
<cite>

Shimizu, T.
, Lengalova A., Martínek V., and Martínková M.. 2019. “Heme: Emergent Roles of Heme in Signal Transduction, Functional Regulation and as Catalytic Centres.” Chemical Society Reviews
48, no. 24: 5624–5657. 10.1039/C9CS00268E.
</cite> [<a href="https://doi.org/10.1039/C9CS00268E" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31748766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemical%20Society%20Reviews&amp;title=Heme:%20Emergent%20Roles%20of%20Heme%20in%20Signal%20Transduction,%20Functional%20Regulation%20and%20as%20Catalytic%20Centres&amp;author=T.%20Shimizu&amp;author=A.%20Lengalova&amp;author=V.%20Mart%C3%ADnek&amp;author=M.%20Mart%C3%ADnkov%C3%A1&amp;volume=48&amp;issue=24&amp;publication_year=2019&amp;pages=5624-5657&amp;pmid=31748766&amp;doi=10.1039/C9CS00268E&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0188">
<cite>

Shu, X.
, Singh M., Karampudi N. B. R., et al. 2021. “Responses of <em>Escherichia coli</em> and <em>Listeria monocytogenes</em> to Ozone Treatment on Non‐Host Tomato: Efficacy of Intervention and Evidence of Induced Acclimation.” Edited by Marcos Pileggi. PLoS ONE
16, no. 10: e0256324. 10.1371/journal.pone.0256324.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0256324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8553054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34710139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Responses%20of%20Escherichia%20coli%20and%20Listeria%20monocytogenes%20to%20Ozone%20Treatment%20on%20Non%E2%80%90Host%20Tomato:%20Efficacy%20of%20Intervention%20and%20Evidence%20of%20Induced%20Acclimation&amp;author=X.%20Shu&amp;author=M.%20Singh&amp;author=N.%20B.%20R.%20Karampudi&amp;volume=16&amp;issue=10&amp;publication_year=2021&amp;pages=e0256324&amp;pmid=34710139&amp;doi=10.1371/journal.pone.0256324&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0189">
<cite>

Sibanda, T.
, and Buys E. M.. 2022. “
<em>Listeria monocytogenes</em> Pathogenesis: The Role of Stress Adaptation.” Microorganisms
10, no. 8: 1522. 10.3390/microorganisms10081522.
</cite> [<a href="https://doi.org/10.3390/microorganisms10081522" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9416357/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36013940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microorganisms&amp;title=Listeria%20monocytogenes%20Pathogenesis:%20The%20Role%20of%20Stress%20Adaptation&amp;author=T.%20Sibanda&amp;author=E.%20M.%20Buys&amp;volume=10&amp;issue=8&amp;publication_year=2022&amp;pages=1522&amp;pmid=36013940&amp;doi=10.3390/microorganisms10081522&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0190">
<cite>

Siderakou, D.
, Zilelidou E., Poimenidou S., et al. 2022. “In Vitro Virulence Potential, Surface Attachment, and Transcriptional Response of Sublethally Injured <em>Listeria monocytogenes</em> Following Exposure to Peracetic Acid.” Applied and Environmental Microbiology
88, no. 2: e01582–21. 10.1128/AEM.01582-21.
</cite> [<a href="https://doi.org/10.1128/AEM.01582-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8788704/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34731051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=In%20Vitro%20Virulence%20Potential,%20Surface%20Attachment,%20and%20Transcriptional%20Response%20of%20Sublethally%20Injured%20Listeria%20monocytogenes%20Following%20Exposure%20to%20Peracetic%20Acid&amp;author=D.%20Siderakou&amp;author=E.%20Zilelidou&amp;author=S.%20Poimenidou&amp;volume=88&amp;issue=2&amp;publication_year=2022&amp;pages=e01582-21&amp;pmid=34731051&amp;doi=10.1128/AEM.01582-21&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0191">
<cite>

Silber, N.
, de Opitz C. L. M., Mayer C., and Sass P.. 2020. “Cell Division Protein FtsZ: from Structure and Mechanism to Antibiotic Target.” Future Microbiology
15, no. 9: 801–831. 10.2217/fmb-2019-0348.
</cite> [<a href="https://doi.org/10.2217/fmb-2019-0348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32692252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Future%20Microbiology&amp;title=Cell%20Division%20Protein%20FtsZ:%20from%20Structure%20and%20Mechanism%20to%20Antibiotic%20Target&amp;author=N.%20Silber&amp;author=C.%20L.%20M.%20de%20Opitz&amp;author=C.%20Mayer&amp;author=P.%20Sass&amp;volume=15&amp;issue=9&amp;publication_year=2020&amp;pages=801-831&amp;pmid=32692252&amp;doi=10.2217/fmb-2019-0348&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0192">
<cite>

Simpson, A. M.‐A.
, and Mitch W. A.. 2022. “Chlorine and Ozone Disinfection and Disinfection Byproducts in Postharvest Food Processing Facilities: A Review.” Critical Reviews in Environmental Science and Technology
52, no. 11: 1825–1867. 10.1080/10643389.2020.1862562.</cite> [<a href="https://doi.org/10.1080/10643389.2020.1862562" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Critical%20Reviews%20in%20Environmental%20Science%20and%20Technology&amp;title=Chlorine%20and%20Ozone%20Disinfection%20and%20Disinfection%20Byproducts%20in%20Postharvest%20Food%20Processing%20Facilities:%20A%20Review&amp;author=A.%20M.%E2%80%90A.%20Simpson&amp;author=W.%20A.%20Mitch&amp;volume=52&amp;issue=11&amp;publication_year=2022&amp;pages=1825-1867&amp;doi=10.1080/10643389.2020.1862562&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0193">
<cite>

Skowron, K.
, Hulisz K., Gryń G., Olszewska H., Wiktorczyk N., and Paluszak Z.. 2018. “Comparison of Selected Disinfectants Efficiency Against <em>Listeria monocytogenes</em> Biofilm Formed on Various Surfaces.” International Microbiology
21, no. 1: 23–33. 10.1007/s10123-018-0002-5.
</cite> [<a href="https://doi.org/10.1007/s10123-018-0002-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30810917/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Microbiology&amp;title=Comparison%20of%20Selected%20Disinfectants%20Efficiency%20Against%20Listeria%20monocytogenes%20Biofilm%20Formed%20on%20Various%20Surfaces&amp;author=K.%20Skowron&amp;author=K.%20Hulisz&amp;author=G.%20Gry%C5%84&amp;author=H.%20Olszewska&amp;author=N.%20Wiktorczyk&amp;volume=21&amp;issue=1&amp;publication_year=2018&amp;pages=23-33&amp;pmid=30810917&amp;doi=10.1007/s10123-018-0002-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0194">
<cite>

Smith, H. B.
, Lee K., Freeman M. J., Stevenson D. M., Amador‐Noguez D., and Sauer J.‐D.. 2023. “
<em>Listeria monocytogenes</em> Requires DHNA‐Dependent Intracellular Redox Homeostasis Facilitated by Ndh2 for Survival and Virulence.” Infection and Immunity
91, no. 10: e00022–23. 10.1128/iai.00022-23.
</cite> [<a href="https://doi.org/10.1128/iai.00022-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10580952/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37754681/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infection%20and%20Immunity&amp;title=Listeria%20monocytogenes%20Requires%20DHNA%E2%80%90Dependent%20Intracellular%20Redox%20Homeostasis%20Facilitated%20by%20Ndh2%20for%20Survival%20and%20Virulence&amp;author=H.%20B.%20Smith&amp;author=K.%20Lee&amp;author=M.%20J.%20Freeman&amp;author=D.%20M.%20Stevenson&amp;author=D.%20Amador%E2%80%90Noguez&amp;volume=91&amp;issue=10&amp;publication_year=2023&amp;pages=e00022-23&amp;pmid=37754681&amp;doi=10.1128/iai.00022-23&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0195">
<cite>

Soares da Costa, T. P.
, Nanson J. D., and Forwood J. K.. 2017. “Structural Characterisation of the Fatty Acid Biosynthesis Enzyme FabF From the Pathogen <em>Listeria monocytogenes</em>
.” Scientific Reports
7, no. 1: 39277. 10.1038/srep39277.
</cite> [<a href="https://doi.org/10.1038/srep39277" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5206705/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28045020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Structural%20Characterisation%20of%20the%20Fatty%20Acid%20Biosynthesis%20Enzyme%20FabF%20From%20the%20Pathogen%20Listeria%20monocytogenes&amp;author=T.%20P.%20Soares%20da%20Costa&amp;author=J.%20D.%20Nanson&amp;author=J.%20K.%20Forwood&amp;volume=7&amp;issue=1&amp;publication_year=2017&amp;pages=39277&amp;pmid=28045020&amp;doi=10.1038/srep39277&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0196">
<cite>

Song, W.‐J.
, Shin J.‐Y., Ryu S., and Kang D.‐H.. 2015. “Inactivation of <em>Escherichia coli</em> O157:H7, Salmonella Typhimurium and <em>Listeria monocytogenes</em> in Apple Juice at Different pH Levels by Gaseous Ozone Treatment.” Journal of Applied Microbiology
119, no. 2: 465–474. 10.1111/jam.12861.
</cite> [<a href="https://doi.org/10.1111/jam.12861" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26033289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Applied%20Microbiology&amp;title=Inactivation%20of%20Escherichia%20coli%20O157:H7,%20Salmonella%20Typhimurium%20and%20Listeria%20monocytogenes%20in%20Apple%20Juice%20at%20Different%20pH%20Levels%20by%20Gaseous%20Ozone%20Treatment&amp;author=W.%E2%80%90J.%20Song&amp;author=J.%E2%80%90Y.%20Shin&amp;author=S.%20Ryu&amp;author=D.%E2%80%90H.%20Kang&amp;volume=119&amp;issue=2&amp;publication_year=2015&amp;pages=465-474&amp;pmid=26033289&amp;doi=10.1111/jam.12861&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0197">
<cite>

Spry, C.
, Kirk K., and Saliba K. J.. 2008. “Coenzyme A Biosynthesis: An Antimicrobial Drug Target.” FEMS Microbiology Reviews
32, no. 1: 56–106. 10.1111/j.1574-6976.2007.00093.x.
</cite> [<a href="https://doi.org/10.1111/j.1574-6976.2007.00093.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18173393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiology%20Reviews&amp;title=Coenzyme%20A%20Biosynthesis:%20An%20Antimicrobial%20Drug%20Target&amp;author=C.%20Spry&amp;author=K.%20Kirk&amp;author=K.%20J.%20Saliba&amp;volume=32&amp;issue=1&amp;publication_year=2008&amp;pages=56-106&amp;pmid=18173393&amp;doi=10.1111/j.1574-6976.2007.00093.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0246">
<cite>

Strahl, H.
, and Errington J.. 2017. “Bacterial Membranes: Structure, Domains, and Function.” Annual Review of Microbiology
71, no. 1: 519–538. 10.1146/annurev-micro-102215-095630.</cite> [<a href="https://doi.org/10.1146/annurev-micro-102215-095630" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28697671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Microbiology&amp;title=Bacterial%20Membranes:%20Structure,%20Domains,%20and%20Function&amp;author=H.%20Strahl&amp;author=J.%20Errington&amp;volume=71&amp;issue=1&amp;publication_year=2017&amp;pages=519-538&amp;pmid=28697671&amp;doi=10.1146/annurev-micro-102215-095630&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0198">
<cite>

TeSlaa, T.
, Ralser M., Fan J., and Rabinowitz J. D.. 2023. “The Pentose Phosphate Pathway in Health and Disease.” Nature Metabolism
5, no. 8: 1275–1289. 10.1038/s42255-023-00863-2.</cite> [<a href="https://doi.org/10.1038/s42255-023-00863-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11251397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37612403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Metabolism&amp;title=The%20Pentose%20Phosphate%20Pathway%20in%20Health%20and%20Disease&amp;author=T.%20TeSlaa&amp;author=M.%20Ralser&amp;author=J.%20Fan&amp;author=J.%20D.%20Rabinowitz&amp;volume=5&amp;issue=8&amp;publication_year=2023&amp;pages=1275-1289&amp;pmid=37612403&amp;doi=10.1038/s42255-023-00863-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0199">
<cite>

Theis, K.
, Skorvaga M., Machius M., Nakagawa N., Van Houten B., and Kisker C.. 2000. “The Nucleotide Excision Repair Protein UvrB, a Helicase‐Like Enzyme With a Catch.” Mutation Research/DNA Repair
460, no. 3: 277–300. 10.1016/S0921-8777(00)00032-X.
</cite> [<a href="https://doi.org/10.1016/S0921-8777(00)00032-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10946234/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mutation%20Research/DNA%20Repair&amp;title=The%20Nucleotide%20Excision%20Repair%20Protein%20UvrB,%20a%20Helicase%E2%80%90Like%20Enzyme%20With%20a%20Catch&amp;author=K.%20Theis&amp;author=M.%20Skorvaga&amp;author=M.%20Machius&amp;author=N.%20Nakagawa&amp;author=B.%20Van%20Houten&amp;volume=460&amp;issue=3&amp;publication_year=2000&amp;pages=277-300&amp;pmid=10946234&amp;doi=10.1016/S0921-8777(00)00032-X&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0200">
<cite>

Tong, C.
, Hu H., Chen G., Li Z., Li A., and Zhang J.. 2021. “Chlorine Disinfectants Promote Microbial Resistance in <em>Pseudomonas</em> sp.” Environmental Research
199: 111296.
</cite> [<a href="https://doi.org/10.1016/j.envres.2021.111296" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34010624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environmental%20Research&amp;title=Chlorine%20Disinfectants%20Promote%20Microbial%20Resistance%20in%20Pseudomonas%20sp&amp;author=C.%20Tong&amp;author=H.%20Hu&amp;author=G.%20Chen&amp;author=Z.%20Li&amp;author=A.%20Li&amp;volume=199&amp;publication_year=2021&amp;pages=111296&amp;pmid=34010624&amp;doi=10.1016/j.envres.2021.111296&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0201">
<cite>

Truglio, J. J.
, Croteau D. L., Skorvaga M., et al. 2004. “Interactions Between UvrA and UvrB: The Role of UvrB's Domain 2 in Nucleotide Excision Repair.” The EMBO Journal
23, no. 13: 2498. 10.1038/sj.emboj.7600263.
</cite> [<a href="https://doi.org/10.1038/sj.emboj.7600263" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC449773/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15192705/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20EMBO%20Journal&amp;title=Interactions%20Between%20UvrA%20and%20UvrB:%20The%20Role%20of%20UvrB's%20Domain%202%20in%20Nucleotide%20Excision%20Repair&amp;author=J.%20J.%20Truglio&amp;author=D.%20L.%20Croteau&amp;author=M.%20Skorvaga&amp;volume=23&amp;issue=13&amp;publication_year=2004&amp;pages=2498&amp;pmid=15192705&amp;doi=10.1038/sj.emboj.7600263&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0202">
<cite>

Truglio, J. J.
, Theis K., Feng Y., et al. 2003. “Crystal Structure of <em>Mycobacterium tuberculosis</em> MenB, a Key Enzyme in Vitamin K2 Biosynthesis.” Journal of Biological Chemistry
278, no. 43: 42352–42360. 10.1074/jbc.M307399200.
</cite> [<a href="https://doi.org/10.1074/jbc.M307399200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12909628/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=Crystal%20Structure%20of%20Mycobacterium%20tuberculosis%20MenB,%20a%20Key%20Enzyme%20in%20Vitamin%20K2%20Biosynthesis&amp;author=J.%20J.%20Truglio&amp;author=K.%20Theis&amp;author=Y.%20Feng&amp;volume=278&amp;issue=43&amp;publication_year=2003&amp;pages=42352-42360&amp;pmid=12909628&amp;doi=10.1074/jbc.M307399200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0203">
<cite>

Tsai, M.‐F.
, McCarthy P., and Miller C.. 2013. “Substrate Selectivity in Glutamate‐Dependent Acid Resistance in Enteric Bacteria.” Proceedings of the National Academy of Sciences
110, no. 15: 5898–5902. 10.1073/pnas.1301444110.</cite> [<a href="https://doi.org/10.1073/pnas.1301444110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3625338/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23530225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Substrate%20Selectivity%20in%20Glutamate%E2%80%90Dependent%20Acid%20Resistance%20in%20Enteric%20Bacteria&amp;author=M.%E2%80%90F.%20Tsai&amp;author=P.%20McCarthy&amp;author=C.%20Miller&amp;volume=110&amp;issue=15&amp;publication_year=2013&amp;pages=5898-5902&amp;pmid=23530225&amp;doi=10.1073/pnas.1301444110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0204">
<cite>

Van Der Veen, S.
, Schalkwijk S. V., Molenaar D., Vos W. M. D., Abee T., and Wells‐Bennik M. H. J.. 2010. “The SOS Response of <em>Listeria monocytogenes</em> Is Involved in Stress Resistance and Mutagenesis.” Microbiology (Reading, England)
156, no. 2: 374–384. 10.1099/mic.0.035196-0.
</cite> [<a href="https://doi.org/10.1099/mic.0.035196-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19892760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology%20(Reading,%20England)&amp;title=The%20SOS%20Response%20of%20Listeria%20monocytogenes%20Is%20Involved%20in%20Stress%20Resistance%20and%20Mutagenesis&amp;author=S.%20Van%20Der%20Veen&amp;author=S.%20V.%20Schalkwijk&amp;author=D.%20Molenaar&amp;author=W.%20M.%20D.%20Vos&amp;author=T.%20Abee&amp;volume=156&amp;issue=2&amp;publication_year=2010&amp;pages=374-384&amp;pmid=19892760&amp;doi=10.1099/mic.0.035196-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0206">
<cite>

Van Haute, S.
, Sampers I., Holvoet K., and Uyttendaele M.. 2013. “Physicochemical Quality and Chemical Safety of Chlorine as a Reconditioning Agent and Wash Water Disinfectant for Fresh‐Cut Lettuce Washing.” Applied and Environmental Microbiology
79, no. 9: 2850–2861. 10.1128/AEM.03283-12.
</cite> [<a href="https://doi.org/10.1128/AEM.03283-12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3623159/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23396332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20and%20Environmental%20Microbiology&amp;title=Physicochemical%20Quality%20and%20Chemical%20Safety%20of%20Chlorine%20as%20a%20Reconditioning%20Agent%20and%20Wash%20Water%20Disinfectant%20for%20Fresh%E2%80%90Cut%20Lettuce%20Washing&amp;author=S.%20Van%20Haute&amp;author=I.%20Sampers&amp;author=K.%20Holvoet&amp;author=M.%20Uyttendaele&amp;volume=79&amp;issue=9&amp;publication_year=2013&amp;pages=2850-2861&amp;pmid=23396332&amp;doi=10.1128/AEM.03283-12&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0207">
<cite>

Verhoeven, E. E. A.
, Wyman C., Moolenaar G. F., and Goosen N.. 2002. “The Presence of Two UvrB Subunits in the UvrAB Complex Ensures Damage Detection in both DNA Strands.” The EMBO Journal
21, no. 15: 4196–4205. 10.1093/emboj/cdf396.
</cite> [<a href="https://doi.org/10.1093/emboj/cdf396" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC126143/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12145219/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20EMBO%20Journal&amp;title=The%20Presence%20of%20Two%20UvrB%20Subunits%20in%20the%20UvrAB%20Complex%20Ensures%20Damage%20Detection%20in%20both%20DNA%20Strands&amp;author=E.%20E.%20A.%20Verhoeven&amp;author=C.%20Wyman&amp;author=G.%20F.%20Moolenaar&amp;author=N.%20Goosen&amp;volume=21&amp;issue=15&amp;publication_year=2002&amp;pages=4196-4205&amp;pmid=12145219&amp;doi=10.1093/emboj/cdf396&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0208">
<cite>

Vollmer, W.
, and Seligman S. J.. 2010. “Architecture of Peptidoglycan: More Data and More Models.” Trends in Microbiology
18, no. 2: 59–66. 10.1016/j.tim.2009.12.004.
</cite> [<a href="https://doi.org/10.1016/j.tim.2009.12.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20060721/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Microbiology&amp;title=Architecture%20of%20Peptidoglycan:%20More%20Data%20and%20More%20Models&amp;author=W.%20Vollmer&amp;author=S.%20J.%20Seligman&amp;volume=18&amp;issue=2&amp;publication_year=2010&amp;pages=59-66&amp;pmid=20060721&amp;doi=10.1016/j.tim.2009.12.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0209">
<cite>

Vorob'eva, L. I.

2004. “Stressors, Stress Reactions, and Survival of Bacteria: A Review.” Applied Biochemistry and Microbiology
40, no. 3: 217–224. 10.1023/B:ABIM.0000025941.11643.19.</cite> [<a href="https://doi.org/10.1023/B:ABIM.0000025941.11643.19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15283326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20Biochemistry%20and%20Microbiology&amp;title=Stressors,%20Stress%20Reactions,%20and%20Survival%20of%20Bacteria:%20A%20Review&amp;author=L.%20I.%20Vorob'eva&amp;volume=40&amp;issue=3&amp;publication_year=2004&amp;pages=217-224&amp;pmid=15283326&amp;doi=10.1023/B:ABIM.0000025941.11643.19&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0210">
<cite>

Walker, G. C.

1996. “The SOS Response of Escherichia coli.” In Escherichia coli and Salmonella: Cellular and Molecular Biology, ed Neidhardt F. C. (ASM Press, Washington, DC: ). 1400–1416.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Escherichia%20coli%20and%20Salmonella:%20Cellular%20and%20Molecular%20Biology&amp;author=G.%20C.%20Walker&amp;publication_year=1996&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0211">
<cite>

Wamelink, M. M. C.
, Struys E. A., and Jakobs C.. 2008. “The Biochemistry, Metabolism and Inherited Defects of the Pentose Phosphate Pathway: A Review.” Journal of Inherited Metabolic Disease
31, no. 6: 703–717. 10.1007/s10545-008-1015-6.
</cite> [<a href="https://doi.org/10.1007/s10545-008-1015-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18987987/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Inherited%20Metabolic%20Disease&amp;title=The%20Biochemistry,%20Metabolism%20and%20Inherited%20Defects%20of%20the%20Pentose%20Phosphate%20Pathway:%20A%20Review&amp;author=M.%20M.%20C.%20Wamelink&amp;author=E.%20A.%20Struys&amp;author=C.%20Jakobs&amp;volume=31&amp;issue=6&amp;publication_year=2008&amp;pages=703-717&amp;pmid=18987987&amp;doi=10.1007/s10545-008-1015-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0212">
<cite>

Wang, C.
, Cui Y., and Qu X.. 2018. “Mechanisms and Improvement of Acid Resistance in Lactic Acid Bacteria.” Archives of Microbiology
200, no. 2: 195–201. 10.1007/s00203-017-1446-2.
</cite> [<a href="https://doi.org/10.1007/s00203-017-1446-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29075866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Archives%20of%20Microbiology&amp;title=Mechanisms%20and%20Improvement%20of%20Acid%20Resistance%20in%20Lactic%20Acid%20Bacteria&amp;author=C.%20Wang&amp;author=Y.%20Cui&amp;author=X.%20Qu&amp;volume=200&amp;issue=2&amp;publication_year=2018&amp;pages=195-201&amp;pmid=29075866&amp;doi=10.1007/s00203-017-1446-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0213">
<cite>

Wang, Q.
, Wang P., Liu P., and Ou J.. 2022. “Comparative Transcriptome Analysis Reveals Regulatory Factors Involved in <em>Vibrio parahaemolyticus</em> Biofilm Formation.” Frontiers in Cellular and Infection Microbiology
12, no. July: 917131. 10.3389/fcimb.2022.917131.
</cite> [<a href="https://doi.org/10.3389/fcimb.2022.917131" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9309355/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35899046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Cellular%20and%20Infection%20Microbiology&amp;title=Comparative%20Transcriptome%20Analysis%20Reveals%20Regulatory%20Factors%20Involved%20in%20Vibrio%20parahaemolyticus%20Biofilm%20Formation&amp;author=Q.%20Wang&amp;author=P.%20Wang&amp;author=P.%20Liu&amp;author=J.%20Ou&amp;volume=12&amp;issue=July&amp;publication_year=2022&amp;pages=917131&amp;pmid=35899046&amp;doi=10.3389/fcimb.2022.917131&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0214">
<cite>

Wang, Y.
, Liu B., Grenier D., and Yi L.. 2019. “Regulatory Mechanisms of the LuxS/AI‐2 System and Bacterial Resistance.” Antimicrobial Agents and Chemotherapy
63, no. 10: e01186‐19. 10.1128/aac.01186-19.
</cite> [<a href="https://doi.org/10.1128/aac.01186-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6761564/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31383657/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antimicrobial%20Agents%20and%20Chemotherapy&amp;title=Regulatory%20Mechanisms%20of%20the%20LuxS/AI%E2%80%902%20System%20and%20Bacterial%20Resistance&amp;author=Y.%20Wang&amp;author=B.%20Liu&amp;author=D.%20Grenier&amp;author=L.%20Yi&amp;volume=63&amp;issue=10&amp;publication_year=2019&amp;pages=e01186%E2%80%9019&amp;pmid=31383657&amp;doi=10.1128/aac.01186-19&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0215">
<cite>

Wani, S.
, Maker J. K., Thompson J. R., Barnes J., and Singleton I.. 2015. “Effect of Ozone Treatment on Inactivation of <em>Escherichia coli</em> and Listeria Sp. on Spinach.” Agriculture
5, no. 2: 155–169. 10.3390/agriculture5020155.</cite> [<a href="https://doi.org/10.3390/agriculture5020155" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agriculture&amp;title=Effect%20of%20Ozone%20Treatment%20on%20Inactivation%20of%20Escherichia%20coli%20and%20Listeria%20Sp.%20on%20Spinach&amp;author=S.%20Wani&amp;author=J.%20K.%20Maker&amp;author=J.%20R.%20Thompson&amp;author=J.%20Barnes&amp;author=I.%20Singleton&amp;volume=5&amp;issue=2&amp;publication_year=2015&amp;pages=155-169&amp;doi=10.3390/agriculture5020155&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0216">
<cite>

Waters, C. M.
, and Bassler B. L.. 2005. “Quorum Sensing: Cell‐to‐Cell Communication in Bacteria.” Annual Review of Cell and Developmental Biology
21: 319–346. 10.1146/annurev.cellbio.21.012704.131001.</cite> [<a href="https://doi.org/10.1146/annurev.cellbio.21.012704.131001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16212498/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Cell%20and%20Developmental%20Biology&amp;title=Quorum%20Sensing:%20Cell%E2%80%90to%E2%80%90Cell%20Communication%20in%20Bacteria&amp;author=C.%20M.%20Waters&amp;author=B.%20L.%20Bassler&amp;volume=21&amp;publication_year=2005&amp;pages=319-346&amp;pmid=16212498&amp;doi=10.1146/annurev.cellbio.21.012704.131001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0217">
<cite>

Whiteley, A. T.
, Garelis N. E., Peterson B. N., et al. 2017. “C‐Di‐AMP Modulates <em>Listeria monocytogenes</em> Central Metabolism to Regulate Growth, Antibiotic Resistance, and Osmoregulation.” Molecular Microbiology
104, no. 2: 212–233. 10.1111/mmi.13622.
</cite> [<a href="https://doi.org/10.1111/mmi.13622" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5391996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28097715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Microbiology&amp;title=C%E2%80%90Di%E2%80%90AMP%20Modulates%20Listeria%20monocytogenes%20Central%20Metabolism%20to%20Regulate%20Growth,%20Antibiotic%20Resistance,%20and%20Osmoregulation&amp;author=A.%20T.%20Whiteley&amp;author=N.%20E.%20Garelis&amp;author=B.%20N.%20Peterson&amp;volume=104&amp;issue=2&amp;publication_year=2017&amp;pages=212-233&amp;pmid=28097715&amp;doi=10.1111/mmi.13622&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0218">
<cite>

Wiktorczyk‐Kapischke, N.
, Skowron K., Grudlewska‐Buda K., Wałecka‐Zacharska E., Korkus J., and Gospodarek‐Komkowska E.. 2021. “Adaptive Response of <em>Listeria monocytogenes</em> to the Stress Factors in the Food Processing Environment.” Frontiers in Microbiology
12, no. August: 710085. 10.3389/fmicb.2021.710085.
</cite> [<a href="https://doi.org/10.3389/fmicb.2021.710085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8417233/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34489900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Adaptive%20Response%20of%20Listeria%20monocytogenes%20to%20the%20Stress%20Factors%20in%20the%20Food%20Processing%20Environment&amp;author=N.%20Wiktorczyk%E2%80%90Kapischke&amp;author=K.%20Skowron&amp;author=K.%20Grudlewska%E2%80%90Buda&amp;author=E.%20Wa%C5%82ecka%E2%80%90Zacharska&amp;author=J.%20Korkus&amp;volume=12&amp;issue=August&amp;publication_year=2021&amp;pages=710085&amp;pmid=34489900&amp;doi=10.3389/fmicb.2021.710085&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0219">
<cite>

Wilson, K.
, Hofmann A., Walker J. M., and Clokie S.. 2018. Wilson and Walker's Principles and Techniques of Biochemistry and Molecular Biology. Cambridge University Press.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Wilson%20and%20Walker's%20Principles%20and%20Techniques%20of%20Biochemistry%20and%20Molecular%20Biology&amp;author=K.%20Wilson&amp;author=A.%20Hofmann&amp;author=J.%20M.%20Walker&amp;author=S.%20Clokie&amp;publication_year=2018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0220">
<cite>

Winterbourn, C. C.

2020. “Biological Chemistry of Superoxide Radicals.” ChemTexts
6, no. 1: 7. 10.1007/s40828-019-0101-8.</cite> [<a href="https://doi.org/10.1007/s40828-019-0101-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ChemTexts&amp;title=Biological%20Chemistry%20of%20Superoxide%20Radicals&amp;author=C.%20C.%20Winterbourn&amp;volume=6&amp;issue=1&amp;publication_year=2020&amp;pages=7&amp;doi=10.1007/s40828-019-0101-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0221">
<cite>

Wong, K. S.
, Chew N. S. L., Low M., and Tan M. K.. 2023. “Plasma‐Activated Water: Physicochemical Properties, Generation Techniques, and Applications.” Processes
11, no. 7: 2213. 10.3390/pr11072213.</cite> [<a href="https://doi.org/10.3390/pr11072213" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Processes&amp;title=Plasma%E2%80%90Activated%20Water:%20Physicochemical%20Properties,%20Generation%20Techniques,%20and%20Applications&amp;author=K.%20S.%20Wong&amp;author=N.%20S.%20L.%20Chew&amp;author=M.%20Low&amp;author=M.%20K.%20Tan&amp;volume=11&amp;issue=7&amp;publication_year=2023&amp;pages=2213&amp;doi=10.3390/pr11072213&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0222">
<cite>

Wurihan, G.
, Brambilla E., Wang S., et al. 2018. “DnaA and LexA Proteins Regulate Transcription of the uvrB Gene in <em>Escherichia coli</em>: The Role of DnaA in the Control of the SOS Regulon.” Frontiers in Microbiology
9, no. June: 1212. 10.3389/fmicb.2018.01212.
</cite> [<a href="https://doi.org/10.3389/fmicb.2018.01212" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6015884/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29967594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=DnaA%20and%20LexA%20Proteins%20Regulate%20Transcription%20of%20the%20uvrB%20Gene%20in%20Escherichia%20coli:%20The%20Role%20of%20DnaA%20in%20the%20Control%20of%20the%20SOS%20Regulon&amp;author=G.%20Wurihan&amp;author=E.%20Brambilla&amp;author=S.%20Wang&amp;volume=9&amp;issue=June&amp;publication_year=2018&amp;pages=1212&amp;pmid=29967594&amp;doi=10.3389/fmicb.2018.01212&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0223">
<cite>

Xiao, W.
, Wang R.‐S., Handy D. E., and Loscalzo J.. 2018. “NAD(H) and NADP(H) Redox Couples and Cellular Energy Metabolism.” Antioxidants &amp; Redox Signaling
28, no. 3: 251–272. 10.1089/ars.2017.7216.
</cite> [<a href="https://doi.org/10.1089/ars.2017.7216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5737637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28648096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antioxidants%20&amp;%20Redox%20Signaling&amp;title=NAD(H)%20and%20NADP(H)%20Redox%20Couples%20and%20Cellular%20Energy%20Metabolism&amp;author=W.%20Xiao&amp;author=R.%E2%80%90S.%20Wang&amp;author=D.%20E.%20Handy&amp;author=J.%20Loscalzo&amp;volume=28&amp;issue=3&amp;publication_year=2018&amp;pages=251-272&amp;pmid=28648096&amp;doi=10.1089/ars.2017.7216&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0224">
<cite>

Yao, J.
, and Rock C. O.. 2017. “Exogenous Fatty Acid Metabolism in Bacteria.” Biochimie
141, no. October: 30–39. 10.1016/j.biochi.2017.06.015.
</cite> [<a href="https://doi.org/10.1016/j.biochi.2017.06.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5665373/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28668270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimie&amp;title=Exogenous%20Fatty%20Acid%20Metabolism%20in%20Bacteria&amp;author=J.%20Yao&amp;author=C.%20O.%20Rock&amp;volume=141&amp;issue=October&amp;publication_year=2017&amp;pages=30-39&amp;pmid=28668270&amp;doi=10.1016/j.biochi.2017.06.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0225">
<cite>

Yokoyama, K.
, and Leimkühler S.. 2015. “The Role of FeS Clusters for Molybdenum Cofactor Biosynthesis and Molybdoenzymes in Bacteria.” Biochimica et Biophysica Acta (BBA)—Molecular Cell Research
1853, no. 6: 1335–1349. 10.1016/j.bbamcr.2014.09.021.
</cite> [<a href="https://doi.org/10.1016/j.bbamcr.2014.09.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4834969/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25268953/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta%20(BBA)%E2%80%94Molecular%20Cell%20Research&amp;title=The%20Role%20of%20FeS%20Clusters%20for%20Molybdenum%20Cofactor%20Biosynthesis%20and%20Molybdoenzymes%20in%20Bacteria&amp;author=K.%20Yokoyama&amp;author=S.%20Leimk%C3%BChler&amp;volume=1853&amp;issue=6&amp;publication_year=2015&amp;pages=1335-1349&amp;pmid=25268953&amp;doi=10.1016/j.bbamcr.2014.09.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0226">
<cite>

Yusupov, M.
, Bogaerts A., Huygh S., Snoeckx R., Duin A. C. T. v., and Neyts E. C.. 2013. “Plasma‐Induced Destruction of Bacterial Cell Wall Components: A Reactive Molecular Dynamics Simulation.” The Journal of Physical Chemistry C
117, no. 11: 5993–5998. 10.1021/jp3128516.</cite> [<a href="https://doi.org/10.1021/jp3128516" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Physical%20Chemistry%20C&amp;title=Plasma%E2%80%90Induced%20Destruction%20of%20Bacterial%20Cell%20Wall%20Components:%20A%20Reactive%20Molecular%20Dynamics%20Simulation&amp;author=M.%20Yusupov&amp;author=A.%20Bogaerts&amp;author=S.%20Huygh&amp;author=R.%20Snoeckx&amp;author=A.%20C.%20T.%20v.%20Duin&amp;volume=117&amp;issue=11&amp;publication_year=2013&amp;pages=5993-5998&amp;doi=10.1021/jp3128516&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0227">
<cite>

Zakrzewski, A. J.
, Gajewska J., Chajęcka‐Wierzchowska W., Załuski D., and Zadernowska A.. 2024. “Prevalence of <em>Listeria monocytogenes</em> and Other Listeria Species in Fish, Fish Products and Fish Processing Environment: A Systematic Review and Meta‐Analysis.” Science of the Total Environment
907, no. January: 167912. 10.1016/j.scitotenv.2023.167912.
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2023.167912" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37866612/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20of%20the%20Total%20Environment&amp;title=Prevalence%20of%20Listeria%20monocytogenes%20and%20Other%20Listeria%20Species%20in%20Fish,%20Fish%20Products%20and%20Fish%20Processing%20Environment:%20A%20Systematic%20Review%20and%20Meta%E2%80%90Analysis&amp;author=A.%20J.%20Zakrzewski&amp;author=J.%20Gajewska&amp;author=W.%20Chaj%C4%99cka%E2%80%90Wierzchowska&amp;author=D.%20Za%C5%82uski&amp;author=A.%20Zadernowska&amp;volume=907&amp;issue=January&amp;publication_year=2024&amp;pages=167912&amp;pmid=37866612&amp;doi=10.1016/j.scitotenv.2023.167912&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0228">
<cite>

Zetzmann, M.
, Sánchez‐Kopper A., Waidmann M. S., Blombach B., and Riedel C. U.. 2016. “Identification of the Agr Peptide of <em>Listeria monocytogenes</em>
.” Frontiers in Microbiology
7, no. June: 989. 10.3389/fmicb.2016.00989.
</cite> [<a href="https://doi.org/10.3389/fmicb.2016.00989" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4916163/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27446029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Identification%20of%20the%20Agr%20Peptide%20of%20Listeria%20monocytogenes&amp;author=M.%20Zetzmann&amp;author=A.%20S%C3%A1nchez%E2%80%90Kopper&amp;author=M.%20S.%20Waidmann&amp;author=B.%20Blombach&amp;author=C.%20U.%20Riedel&amp;volume=7&amp;issue=June&amp;publication_year=2016&amp;pages=989&amp;pmid=27446029&amp;doi=10.3389/fmicb.2016.00989&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0229">
<cite>

Zhang, F.
, and Cheng W.. 2022. “The Mechanism of Bacterial Resistance and Potential Bacteriostatic Strategies.” Antibiotics
11, no. 9: 1215. 10.3390/antibiotics11091215.
</cite> [<a href="https://doi.org/10.3390/antibiotics11091215" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9495013/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36139994/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antibiotics&amp;title=The%20Mechanism%20of%20Bacterial%20Resistance%20and%20Potential%20Bacteriostatic%20Strategies&amp;author=F.%20Zhang&amp;author=W.%20Cheng&amp;volume=11&amp;issue=9&amp;publication_year=2022&amp;pages=1215&amp;pmid=36139994&amp;doi=10.3390/antibiotics11091215&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0230">
<cite>

Zhang, Y.‐M.
, White S. W., and Rock C. O.. 2006. “Inhibiting Bacterial Fatty Acid Synthesis *.” Journal of Biological Chemistry
281, no. 26: 17541–17544. 10.1074/jbc.R600004200.
</cite> [<a href="https://doi.org/10.1074/jbc.R600004200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16648134/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=Inhibiting%20Bacterial%20Fatty%20Acid%20Synthesis%20*&amp;author=Y.%E2%80%90M.%20Zhang&amp;author=S.%20W.%20White&amp;author=C.%20O.%20Rock&amp;volume=281&amp;issue=26&amp;publication_year=2006&amp;pages=17541-17544&amp;pmid=16648134&amp;doi=10.1074/jbc.R600004200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0231">
<cite>

Zhang, Z.
, Meng Q., Qiao J., et al. 2013. “RsbV of <em>Listeria monocytogenes</em> Contributes to Regulation of Environmental Stress and Virulence.” Archives of Microbiology
195, no. 2: 113–120. 10.1007/s00203-012-0855-5.
</cite> [<a href="https://doi.org/10.1007/s00203-012-0855-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23192174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Archives%20of%20Microbiology&amp;title=RsbV%20of%20Listeria%20monocytogenes%20Contributes%20to%20Regulation%20of%20Environmental%20Stress%20and%20Virulence&amp;author=Z.%20Zhang&amp;author=Q.%20Meng&amp;author=J.%20Qiao&amp;volume=195&amp;issue=2&amp;publication_year=2013&amp;pages=113-120&amp;pmid=23192174&amp;doi=10.1007/s00203-012-0855-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0245">
<cite>

Zhang, X.
, Xue C., Zhao F., et al. 2014. “Suitable Extracellular Oxidoreduction Potential Inhibit Rex Regulation and Effect Central Carbon and Energy Metabolism in Saccharopolyspora Spinosa.” Microbial Cell Factories
13. 10.1186/s12934-014-0098-z.</cite> [<a href="https://doi.org/10.1186/s12934-014-0098-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4172946/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25158803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbial%20Cell%20Factories&amp;title=Suitable%20Extracellular%20Oxidoreduction%20Potential%20Inhibit%20Rex%20Regulation%20and%20Effect%20Central%20Carbon%20and%20Energy%20Metabolism%20in%20Saccharopolyspora%20Spinosa&amp;author=X.%20Zhang&amp;author=C.%20Xue&amp;author=F.%20Zhao&amp;volume=13&amp;publication_year=2014&amp;pmid=25158803&amp;doi=10.1186/s12934-014-0098-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0232">
<cite>

Zhao, W.
, Hou Y., Wei L., et al. 2025. “Chlorination‐Induced Spread of Antibiotic Resistance Genes in Drinking Water Systems.” Water Research
274, no. April: 123092. 10.1016/j.watres.2025.123092.
</cite> [<a href="https://doi.org/10.1016/j.watres.2025.123092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39787839/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Water%20Research&amp;title=Chlorination%E2%80%90Induced%20Spread%20of%20Antibiotic%20Resistance%20Genes%20in%20Drinking%20Water%20Systems&amp;author=W.%20Zhao&amp;author=Y.%20Hou&amp;author=L.%20Wei&amp;volume=274&amp;issue=April&amp;publication_year=2025&amp;pages=123092&amp;pmid=39787839&amp;doi=10.1016/j.watres.2025.123092&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0233">
<cite>

Zhao, Y.‐M.
, Patange A., Sun D.‐W., and Tiwari B.. 2020. “Plasma‐Activated Water: Physicochemical Properties, Microbial Inactivation Mechanisms, Factors Influencing Antimicrobial Effectiveness, and Applications in the Food Industry.” Comprehensive Reviews in Food Science and Food Safety
19, no. 6: 3951–3979. 10.1111/1541-4337.12644.
</cite> [<a href="https://doi.org/10.1111/1541-4337.12644" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33337045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comprehensive%20Reviews%20in%20Food%20Science%20and%20Food%20Safety&amp;title=Plasma%E2%80%90Activated%20Water:%20Physicochemical%20Properties,%20Microbial%20Inactivation%20Mechanisms,%20Factors%20Influencing%20Antimicrobial%20Effectiveness,%20and%20Applications%20in%20the%20Food%20Industry&amp;author=Y.%E2%80%90M.%20Zhao&amp;author=A.%20Patange&amp;author=D.%E2%80%90W.%20Sun&amp;author=B.%20Tiwari&amp;volume=19&amp;issue=6&amp;publication_year=2020&amp;pages=3951-3979&amp;pmid=33337045&amp;doi=10.1111/1541-4337.12644&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0234">
<cite>

Zheng, M.
, Liu Y., Zhang G., Yang Z., Xu W., and Chen Q.. 2023. “The Applications and Mechanisms of Superoxide Dismutase in Medicine, Food, and Cosmetics.” Antioxidants
12, no. 9: 1675. 10.3390/antiox12091675.
</cite> [<a href="https://doi.org/10.3390/antiox12091675" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10525108/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37759978/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antioxidants&amp;title=The%20Applications%20and%20Mechanisms%20of%20Superoxide%20Dismutase%20in%20Medicine,%20Food,%20and%20Cosmetics&amp;author=M.%20Zheng&amp;author=Y.%20Liu&amp;author=G.%20Zhang&amp;author=Z.%20Yang&amp;author=W.%20Xu&amp;volume=12&amp;issue=9&amp;publication_year=2023&amp;pages=1675&amp;pmid=37759978&amp;doi=10.3390/antiox12091675&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0235">
<cite>

Zhong, Q.
, Kobe B., and Kappler U.. 2020. “Molybdenum Enzymes and How They Support Virulence in Pathogenic Bacteria.” Frontiers in Microbiology
11: 615860. 10.3389/fmicb.2020.615860.
</cite> [<a href="https://doi.org/10.3389/fmicb.2020.615860" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7759655/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33362753/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Microbiology&amp;title=Molybdenum%20Enzymes%20and%20How%20They%20Support%20Virulence%20in%20Pathogenic%20Bacteria&amp;author=Q.%20Zhong&amp;author=B.%20Kobe&amp;author=U.%20Kappler&amp;volume=11&amp;publication_year=2020&amp;pages=615860&amp;pmid=33362753&amp;doi=10.3389/fmicb.2020.615860&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="crf370260-bib-0236">
<cite>

Zoltner, M.
, Ng W. M. A. V., Money J. J., et al. 2016. “EssC: Domain Structures Inform on the Elusive Translocation Channel in the Type VII Secretion System.” Biochemical Journal
473, no. 13: 1941–1952. 10.1042/BCJ20160257.
</cite> [<a href="https://doi.org/10.1042/BCJ20160257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4925161/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27130157/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemical%20Journal&amp;title=EssC:%20Domain%20Structures%20Inform%20on%20the%20Elusive%20Translocation%20Channel%20in%20the%20Type%20VII%20Secretion%20System&amp;author=M.%20Zoltner&amp;author=W.%20M.%20A.%20V.%20Ng&amp;author=J.%20J.%20Money&amp;volume=473&amp;issue=13&amp;publication_year=2016&amp;pages=1941-1952&amp;pmid=27130157&amp;doi=10.1042/BCJ20160257&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>
<strong>Supporting Material</strong>: crf370260‐sup‐0001‐SuppMat.docx</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12371860/bin/CRF3-24-e70260-s002.docx" data-ga-action="click_feat_suppl" class="usa-link">CRF3-24-e70260-s002.docx</a><sup> (95.6KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>
<strong>Supporting Table and FIgure</strong>: crf370260‐sup‐0002‐SuppMat.docx</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12371860/bin/CRF3-24-e70260-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">CRF3-24-e70260-s001.docx</a><sup> (121.3KB, docx) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Comprehensive Reviews in Food Science and Food Safety are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1111/1541-4337.70260"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/CRF3-24-e70260.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371860/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371860/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371860%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371860/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371860/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371860/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40843964/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371860/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40843964/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371860/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371860/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="7YgczH3OyRX23qAG0ChAjdQ7eFa0uMU7VFPzopjMBBwWR7eKV5Aino9WCrArsocn">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
