
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            In-silico screening of small compounds against Lassa fever haemorrhagic virus nucleoprotein - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5339248AF31443053924001EA2C16A.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368151/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="In-silico screening of small compounds against Lassa fever haemorrhagic virus nucleoprotein">
<meta name="citation_author" content="Ekpo Eyo Antai">
<meta name="citation_author_institution" content="Department of Biological (Microbiology) Oceanography, Faculty of Oceanography, University of Calabar, Calabar, Nigeria">
<meta name="citation_author" content="Uwem Okon Edet">
<meta name="citation_author_institution" content="Department of Biological Science (Microbiology Unit), Faculty of Natural and Applied Sciences, Arthur Jarvis University, Akpabuyo, Cross River State Nigeria">
<meta name="citation_author" content="Elizabeth N Mbim">
<meta name="citation_author_institution" content="Department of Microbiology, Faculty of Sciences, Cross River State University of Sciences and Technology, Calabar, Cross River State Nigeria">
<meta name="citation_author" content="Ini Ubi Bassey">
<meta name="citation_author_institution" content="Department of Microbiology, Faculty of Biological Sciences, University of Calabar, Calabar, Cross River State Nigeria">
<meta name="citation_author" content="Sozan M Abdelkhalig">
<meta name="citation_author_institution" content="Department of Basic Medical Sciences, College of Medicine, AlMaarefa University, P.O.Box 71666, Riyadh, 11597 Saudi Arabia">
<meta name="citation_author" content="Francisca O Nwaokorie">
<meta name="citation_author_institution" content="Department of Medical Laboratory Science, College of Medicine, University of Lagos, Yaba, Lagos State Nigeria">
<meta name="citation_author" content="Ehssan Moglad">
<meta name="citation_author_institution" content="Department of Pharmaceutics, College of Pharmacy, Prince Sattam bin Abdulaziz University, P.O. Box 173, Alkharj, 11942 Saudi Arabia">
<meta name="citation_author" content="Jamda Ponmak">
<meta name="citation_author_institution" content="Department of Molecular Biology, Federal College Veterinary and Medical Laboratory Technology, Vom, Plateau State Nigeria">
<meta name="citation_author" content="Samia Al-Shouli">
<meta name="citation_author_institution" content="Immunology Unit, Department of Pathology, College of Medicine, King Saud University, Riyadh, 11461 Saudi Arabia">
<meta name="citation_author" content="Sawsan AlShouli">
<meta name="citation_author_institution" content="Pharmacy Department, Security forces Hospital, Riyadh, 11481 Saudi Arabia">
<meta name="citation_author" content="Destiny E Charlie">
<meta name="citation_author_institution" content="Department of Pure and Applied Chemistry, Faculty of Physical Sciences, University of Calabar, Calabar, Cross River State Nigeria">
<meta name="citation_author" content="Mohnad Abdalla">
<meta name="citation_author_institution" content="Pediatric Research Institute, Children’s Hospital Affiliated to Shandong University, Jinan, 250022 China">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30558">
<meta name="citation_doi" content="10.1038/s41598-025-89989-9">
<meta name="citation_pmid" content="40835743">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368151/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368151/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368151/pdf/41598_2025_Article_89989.pdf">
<meta name="description" content="Lassa fever hemorrhagic virus (LFHV) outbreaks in Nigeria continue to increase. With the possibility of a potentially deadly pandemic and the absence of a vaccine, an effective candidate drug that presents with no adverse effects and better clinical ...">
<meta name="og:title" content="In-silico screening of small compounds against Lassa fever haemorrhagic virus nucleoprotein">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Lassa fever hemorrhagic virus (LFHV) outbreaks in Nigeria continue to increase. With the possibility of a potentially deadly pandemic and the absence of a vaccine, an effective candidate drug that presents with no adverse effects and better clinical ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368151/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12368151">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-89989-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_89989.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368151%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12368151/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12368151/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368151/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 20;15:30558. doi: <a href="https://doi.org/10.1038/s41598-025-89989-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-89989-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>In-silico screening of small compounds against Lassa fever haemorrhagic virus nucleoprotein</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Antai%20EE%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Ekpo Eyo Antai</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Ekpo Eyo Antai</span></h3>
<div class="p">
<sup>1</sup>Department of Biological (Microbiology) Oceanography, Faculty of Oceanography, University of Calabar, Calabar, Nigeria </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Antai%20EE%22%5BAuthor%5D" class="usa-link"><span class="name western">Ekpo Eyo Antai</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Edet%20UO%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Uwem Okon Edet</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Uwem Okon Edet</span></h3>
<div class="p">
<sup>2</sup>Department of Biological Science (Microbiology Unit), Faculty of Natural and Applied Sciences, Arthur Jarvis University, Akpabuyo, Cross River State Nigeria </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Edet%20UO%22%5BAuthor%5D" class="usa-link"><span class="name western">Uwem Okon Edet</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mbim%20EN%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Elizabeth N Mbim</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Elizabeth N Mbim</span></h3>
<div class="p">
<sup>3</sup>Department of Microbiology, Faculty of Sciences, Cross River State University of Sciences and Technology, Calabar, Cross River State Nigeria </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mbim%20EN%22%5BAuthor%5D" class="usa-link"><span class="name western">Elizabeth N Mbim</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bassey%20IU%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Ini Ubi Bassey</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Ini Ubi Bassey</span></h3>
<div class="p">
<sup>4</sup>Department of Microbiology, Faculty of Biological Sciences, University of Calabar, Calabar, Cross River State Nigeria </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bassey%20IU%22%5BAuthor%5D" class="usa-link"><span class="name western">Ini Ubi Bassey</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abdelkhalig%20SM%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Sozan M Abdelkhalig</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Sozan M Abdelkhalig</span></h3>
<div class="p">
<sup>5</sup>Department of Basic Medical Sciences, College of Medicine, AlMaarefa University, P.O.Box 71666, Riyadh, 11597 Saudi Arabia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abdelkhalig%20SM%22%5BAuthor%5D" class="usa-link"><span class="name western">Sozan M Abdelkhalig</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nwaokorie%20FO%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Francisca O Nwaokorie</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Francisca O Nwaokorie</span></h3>
<div class="p">
<sup>6</sup>Department of Medical Laboratory Science, College of Medicine, University of Lagos, Yaba, Lagos State Nigeria </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nwaokorie%20FO%22%5BAuthor%5D" class="usa-link"><span class="name western">Francisca O Nwaokorie</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moglad%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Ehssan Moglad</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Ehssan Moglad</span></h3>
<div class="p">
<sup>7</sup>Department of Pharmaceutics, College of Pharmacy, Prince Sattam bin Abdulaziz University, P.O. Box 173, Alkharj, 11942 Saudi Arabia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moglad%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Ehssan Moglad</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ponmak%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Jamda Ponmak</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Jamda Ponmak</span></h3>
<div class="p">
<sup>8</sup>Department of Molecular Biology, Federal College Veterinary and Medical Laboratory Technology, Vom, Plateau State Nigeria </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ponmak%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jamda Ponmak</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Al-Shouli%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Samia Al-Shouli</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Samia Al-Shouli</span></h3>
<div class="p">
<sup>9</sup>Immunology Unit, Department of Pathology, College of Medicine, King Saud University, Riyadh, 11461 Saudi Arabia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Al-Shouli%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Samia Al-Shouli</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22AlShouli%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Sawsan AlShouli</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Sawsan AlShouli</span></h3>
<div class="p">
<sup>10</sup>Pharmacy Department, Security forces Hospital, Riyadh, 11481 Saudi Arabia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22AlShouli%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sawsan AlShouli</span></a>
</div>
</div>
<sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Charlie%20DE%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Destiny E Charlie</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Destiny E Charlie</span></h3>
<div class="p">
<sup>11</sup>Department of Pure and Applied Chemistry, Faculty of Physical Sciences, University of Calabar, Calabar, Cross River State Nigeria </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Charlie%20DE%22%5BAuthor%5D" class="usa-link"><span class="name western">Destiny E Charlie</span></a>
</div>
</div>
<sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abdalla%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Mohnad Abdalla</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Mohnad Abdalla</span></h3>
<div class="p">
<sup>12</sup>Pediatric Research Institute, Children’s Hospital Affiliated to Shandong University, Jinan, 250022 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abdalla%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mohnad Abdalla</span></a>
</div>
</div>
<sup>12</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Biological (Microbiology) Oceanography, Faculty of Oceanography, University of Calabar, Calabar, Nigeria </div>
<div id="Aff2">
<sup>2</sup>Department of Biological Science (Microbiology Unit), Faculty of Natural and Applied Sciences, Arthur Jarvis University, Akpabuyo, Cross River State Nigeria </div>
<div id="Aff3">
<sup>3</sup>Department of Microbiology, Faculty of Sciences, Cross River State University of Sciences and Technology, Calabar, Cross River State Nigeria </div>
<div id="Aff4">
<sup>4</sup>Department of Microbiology, Faculty of Biological Sciences, University of Calabar, Calabar, Cross River State Nigeria </div>
<div id="Aff5">
<sup>5</sup>Department of Basic Medical Sciences, College of Medicine, AlMaarefa University, P.O.Box 71666, Riyadh, 11597 Saudi Arabia </div>
<div id="Aff6">
<sup>6</sup>Department of Medical Laboratory Science, College of Medicine, University of Lagos, Yaba, Lagos State Nigeria </div>
<div id="Aff7">
<sup>7</sup>Department of Pharmaceutics, College of Pharmacy, Prince Sattam bin Abdulaziz University, P.O. Box 173, Alkharj, 11942 Saudi Arabia </div>
<div id="Aff8">
<sup>8</sup>Department of Molecular Biology, Federal College Veterinary and Medical Laboratory Technology, Vom, Plateau State Nigeria </div>
<div id="Aff9">
<sup>9</sup>Immunology Unit, Department of Pathology, College of Medicine, King Saud University, Riyadh, 11461 Saudi Arabia </div>
<div id="Aff10">
<sup>10</sup>Pharmacy Department, Security forces Hospital, Riyadh, 11481 Saudi Arabia </div>
<div id="Aff11">
<sup>11</sup>Department of Pure and Applied Chemistry, Faculty of Physical Sciences, University of Calabar, Calabar, Cross River State Nigeria </div>
<div id="Aff12">
<sup>12</sup>Pediatric Research Institute, Children’s Hospital Affiliated to Shandong University, Jinan, 250022 China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Jun 3; Accepted 2025 Feb 10; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12368151  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40835743/" class="usa-link">40835743</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Lassa fever hemorrhagic virus (LFHV) outbreaks in Nigeria continue to increase. With the possibility of a potentially deadly pandemic and the absence of a vaccine, an effective candidate drug that presents with no adverse effects and better clinical outcomes is urgently needed; hence, the aim of the study. Retrieved ligands: 5-(Methylene) evodiamine, <em>N</em>-(2-Fluorobenzoyl) evodiamine, <em>N</em>-[[2-(3-chlorophenyl)-5-(trifluoromethyl) pyrazol-3-yl]methyl]-2-(1-methyl-2,3-dihydroindol-5-yl) propanamide (NCTPMMDP), lauric acid, and sofosbuvir, and the target protein, 3MX5, were prepared and utilised for docking, molecular dynamics simulation, ADMET profiling, and DFT using standard protocols. The docking score for lauric acid was the lowest, with a value of − 6.1, followed by the standard drug and co-crystal control drug with scores of − 9.1 and − 8.9 kcal/mol, respectively. On the other hand, the scores of the other ligands ranged from − 9.5 to − 12.3 kcal/mol. Molecular simulations revealed fluctuations in the RMSD and RMSF values; however, overall, the formed complexes were stable during most of the docking simulation runs. Principal component analysis (PCA) analysis showed that all the ligands, apart from lauric acid, showed better conformational changes within the backbone of the protein. ADMET profiling showed that the ligands were better oral drug candidates than sofosbuvir, as revealed by their better absorption values and absolute compliance with the Lipinski rule of five. The DFT result showed that, apart from lauric acid, all the test ligands possess small energy gaps indicative of high reactivity. Put together, these findings indicate that the test ligands possess anti-LHFV potential that merits further in vivo and in vitro evaluations.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-89989-9.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Lassa fever virus, DFT, Docking, Simulation, Nucleoprotein, Nigeria</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Computational biology and bioinformatics, Drug discovery, Microbiology</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">The Lassa fever virus was first discovered in 1969 in Lassa town in Borno State, Nigeria, hence its name<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. The virus is a member of the <em>Arenaviridae</em> family, and it is a single-stranded RNA virus<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Since the index case, the LHFV has continued to spread across Nigeria with no end in sight<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. As of April 2023, there were 4702 and 877 suspected and confirmed cases, respectively in Nigeria reported within the first quarter of 2023<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. A weekly update from the National Centre for Disease Control (NCDC) for epidemiological week 7 reported a total of 507 confirmed cases, 66 suspected cases, and 13 deaths for several states, and these included Edo, Ondo, Kogi, Ebonyi, Bauchi, Cross River, Taraba, Enugu, Imo, Delta, Niger, and Yobe States in Nigeria<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Once limited to Nigeria, the virus has now spread to other countries in the West African region were cases have also been reported<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>.</p>
<p id="Par3">Interestingly, in all these countries, the outbreaks appear to overlap with the prevalence of wild multimammate rats (<em>Mastomys</em> species)<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>–<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. The implicated rodent species comes into contact with food, water, and various surfaces in various homes and, in the process, leave behind their droppings. At the moment, human transmission occurs via contact of the virus from rodent droppings with compromised skin or mucous membrane<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Human-to-human transmission cycle has been reported to be via infected secretions that include blood, saliva, and urine<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Once a human infection occurs, death is imminent from organ failure within 10–14 days, during which the infected persons present with its unique symptoms<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>.</p>
<p id="Par4">At the moment, there is no approved LFHV vaccine, but there are reports of a first-in-human trial<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. The candidate vaccine study reported some adverse events that were limited to mostly mild or moderate in severity and good safety profiles following its administration<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Uptake of vaccines in Africa as a whole has been a problem. The acceptance of the COVID-19 vaccine in Africa was far less than that of the rest of the world<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. These issues put together necessitate the urgent need for an approved drug for the virus that can be used to effectively treat cases<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Among the numerous targets for drug development, the viral nucleoprotein is an important option, as it is involved in various aspects of the virus life cycle<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Earlier studies relied on the use of rational drug design in the search for a novel therapeutic agent for LHFV<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Odey et al.<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup> reported docking scores that ranged from − 6.0 to − 7.7 kcal/mol for cissampeline against various LFHV proteins. In another study, rheosmin was reported to have a binding score of − 6.5 kcal/mol when docked against the 3mWp protein. Eyo et al.<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup> reported feruloyltyramine’s excellent docking activities against various LFHV proteins.</p>
<p id="Par5">We aimed to screen four (4) small compounds against the LFHV nucleoprotein 3MX5 using in silico approaches. The compounds were selected for several reasons. First, the selection was based on their structural diversity, reported bioactivities, and potential to interact with the target protein, 3MX5. Compounds with PubChem IDs 56,967,508 and 49,804,912 are well-known evodiamine derivatives whose anti-virus properties are well known<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. On the one hand, lauric acid (PubChem ID 3893), a saturated fatty acid, was selected to explore the effect of hydrophobic interactions on ligand binding. Furthermore, the compound PubChem ID 45,375,808 was selected based on its structural complexity and predicted high binding affinity, while PubChem ID 49,804,500 was selected as it provides diverse functional group diversity for exploring hydrogen bonding and van der Waals interactions. Put together, the ligands were selected to provide a detailed investigation into binding dynamics and stability as they complex with the target protein and to also provide more insight into their pharmacokinetic properties, laying the groundwork for future therapeutic developments.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Ligand and protein retrieval</h3>
<p id="Par6">The ligands utilised in this study were 5-(Methylene) evodiamine (PubChem ID 56967508), lauric acid (PubChem ID 3893), <em>N</em>-(2-Fluorobenzoyl) evodiamine (PubChem ID 49804912), and <em>N</em>-[[2-(3-chlorophenyl)-5-(trifluoromethyl) pyrazol-3-yl]methyl]-2-(1-methyl-2,3-dihydroindol-5-yl) propanamide (NCTPMMDP) (PubChem ID 49804500) and sofosbuvir (PubChem ID 45375808)<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>–<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. On the other hand, the target protein, 3MX5, was obtained from the Research Collaboratory for Structural Bioinformatics (RCSB) website<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. The published properties of the protein indicate that it was obtained via X-ray diffraction and had a resolution of 1.90 Å.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Preparation of the ligands</h3>
<p id="Par7">The ligands, including the controls (<em>n</em> = 6), were prepared as previously reported using the ChemDraw software<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. The energies of the ligands were minimised using default settings, their SMILES strings retrieved for ADMET prediction, and the resulting minimised structure was stored using the MDL SD file (^.sdf) format<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Preparation of the target protein (3MX5)</h3>
<p id="Par8">In the Biovia Discovery Studio tool<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>, the 3MX5 protein was prepared for docking where it was first loaded in PDB format. The protein’s chain revealed where all the native ligands were attached. This was followed by the removal of all water molecules and heteroatom. Next, the addition of of polar hydrogen atoms was performed. The active site of the protein was defined using the active site bubble and the native ligands as guides. The coordinates were x = 17.7295, y = 26.7623, and z = 3.0223. From the prepared protein, incomplete charges were corrected and the resulting protein saved for further use<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Molecular docking of the ligands to the target protein</h3>
<p id="Par9">The ligands, one after the other, were docked against the LFHV nucleoprotein target protein, 3MX5. Molecular docking was performed as reported<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>–<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. First, the protein’s active site was identified with the aid of the Allosite online tool (<a href="https://mdl.shsmu.edu.cn/AST/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://mdl.shsmu.edu.cn/AST/</a>). This was followed by the identification of the allosteric pocket using the molecular operating environment (MOE) site finder of the Molecular Operating Environment (MOE) version 2023.02, as previously reported<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Following the identifications, the prepared protein and ligands utilised for molecular docking analysis as previously reported using the London dG scoring function<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. The docking scores and poses were visualised in 2 and 3D using the Biovia Discovery Studio and Pymol too, respectively.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Molecular dynamics simulation (MDS) studies</h3>
<p id="Par10">A 300 nanosecond (ns) MDS run for the studied ligands and the control co-crystal was carried out using the Schrodinger-Desmond module. A solvated system was created to ensure the presence of water molecules during the docking run, and this was achieved via the TIP3P model. A periodic boundary condition of 10 Å was utilised for the orthorhombic simulation box. Neutrality was achieved in the solvated system via the addition of adequate counter ions. In addition to the solvation and neutrality states, an isosmotic state was created and maintained in the simulation box by adding 0.15 M NaCl. Before the execution of the MDS, the system was allowed to reach equilibrium, and thereafter, at 310 K and 1.013 bar, a 300 ns MDS was carried out for the various complexes<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. In addition, the force field employed for the simulation was the OPLS (Optimized Potentials for Liquid Simulations). Furthermore, PCA and dynamic cross-correlation matrix analyses were also evaluated.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Density functional theory (DFT) evaluations</h3>
<p id="Par11">In addition to the MD and MDS carried out, all the ligands in this study were subjected to DFT evaluations. DFT has emerged as an invaluable quantum mechanical method that allows for the examination of electronic structure and the elucidation of potential reaction mechanisms<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. The DFT computations were done using methods already reported using the Guassian 09 software<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. The computations were done using the basic sets of atoms, which were C, N, O, and H, and the B3LYP and exchange-correlation functional (ECF) with a 6–31 G (d, p). The geometry, configuration, and optimisation were done using the B3LYP/6–31 þ G (d, p) basis set to increase the reliability of the results<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Lipinski’s rule of five (LROF), ADMET and target protein predictions of the ligands</h3>
<p id="Par12">First, using the SWISSADME tool, the ligands were screened to evaluate their compliance with LROF and ADME properties at <a href="http://www.swissadme.ch/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.swissadme.ch/</a>. The ADME properties were then validated using the ADMET tool of the pkCSM tool at <a href="https://biosig.lab.uq.edu.au/pkcsm/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://biosig.lab.uq.edu.au/pkcsm/</a><sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Results and discussion</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Molecular docking</h3>
<p id="Par13">The docking output is presented in Table <a href="#Tab1" class="usa-link">1</a>. All the ligands, including the control, returned higher docking scores than the control (co-crystal), apart from lauric acid, which returned a docking score of − 6.1 kcal/mol. The binding affinities for the rest of the ligands ranged from − 9.1 to − 12.3 kcal/mol. Compared to the standard drug, that is, sofosbuvir, all the test ligands, apart from lauric, gave higher scores that ranged from − 9.5 to − 12.3 kcal/mol. The test ligand with the highest score was NCTPMMDP with a score of 12.3 kcal/mol. The various docking scores indicate varying degrees of ligands and protein interactions. The high scores tentatively suggest that the test ligands bind more favourably to the target as much as the control and standard drug<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. All the test ligands in our study apart from lauric acid gave better binding affinities than those utilised in previous studies that ranged from − 6.0 to − 7.7 kal/mol [15 24}. Our findings align better with an earlier study that also docked various evodiamine derivatives against LFHV glycoprotein (5FT2) and nucleoprotein (3MX5)<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Their docking results in their results ranged from − 1.7 to − 8.5 kcal/mol and − 2.0 to − 11.0 kcal/mol, respectively against the glycoprotein and the nucleoprotein of LFHV in their study. Furthermore, the docking scores for the control drug for both proteins were − 5.9 and − 7.1 kcal/mol, respectively, for the control. In another study that also utilised evodiamine as ligands against Sars-Cov2 protein, the docking scores that ranged from − 6.31 to − 9.99 kcal/mol were observed<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Compared to our study, the results indicate that our test ligands returned better interactions than those reported earlier, and this was also the case with the control drug. The observed differences could be as a result of the various amino acid residues involved in the different active sites of the proteins utilised in the various studies, and the various viruses involved in the various studies.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Docking affinities for various ligands.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Ligand</th>
<th align="left" colspan="1" rowspan="1">Binding energy (kcal/mol)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">3mx5_ Control(co-crystal)</td>
<td align="center" colspan="1" rowspan="1">− 8.9</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PubChem ID 45,375,808</td>
<td align="center" colspan="1" rowspan="1">− 9.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PubChem ID 3893</td>
<td align="center" colspan="1" rowspan="1">− 6.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PubChem ID 49,804,500</td>
<td align="center" colspan="1" rowspan="1">− 9.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PubChem ID 49,804,912</td>
<td align="center" colspan="1" rowspan="1">− 12.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">PubChem ID 56,967,508</td>
<td align="center" colspan="1" rowspan="1">− 9.5</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p17"><p><strong>Key</strong>: PubChem ID 45,375,808 = Sofosbuvir; 3893 = Lauric acid; 49,804,500 = NCTPMMDP = <em>N</em>-[[2-(3-chlorophenyl)-5-(trifluoromethyl)pyrazol-3-yl]methyl]-2-(1-methyl-2,3-dihydroindol-5-yl) propanamide; 49,804,912 = <em>N</em>-(2-Fluorobenzoyl) evodiamine; 56,967,508 = 5-(Methylene) evodiamine.</p></div></div></section><p id="Par15">Added to the binding scores or affinities, the docking poses or favourable docking positions of the ligands and the target, as well as the control drug and co-crystal, were visualised in 2D and 3D (Figs. <a href="#Fig1" class="usa-link">1</a> and <a href="#Fig2" class="usa-link">2</a>) to reveal the various amino acids of the target protein that interacted with the ligands. Figure <a href="#Fig1" class="usa-link">1</a> shows the poses in 2D. From the result, it can be seen that the ligands interacted differently with the target protein. The co-crystal gave the most amino acids (<em>n</em> = 9) that interacted with the visible bonds depicted using arrows (see legend in Fig. <a href="#Fig1" class="usa-link">1</a>). For the rest of the ligands depicted with labels B to F, the amino acids with visible bonds (in arrows) were 3, 6, 4, 3, and 2, respectively, for ligands B, C, D, E, and F. The main interaction types were greasy (Van der Waal, shown in green) and polar interactions (shown in pink).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368151_41598_2025_89989_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7bc7/12368151/eb0802a4847b/41598_2025_89989_Fig1_HTML.jpg" loading="lazy" id="d33e689" height="719" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Visualisation outputs of the various ligands co-crystal to the target protein in 2D. Key: Figures A to F shows the 2 poses for the various ligands, while the unlabeled portion shows the various bond types.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368151_41598_2025_89989_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7bc7/12368151/9e8f67f6da73/41598_2025_89989_Fig2_HTML.jpg" loading="lazy" id="d33e699" height="450" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Visualisation outputs of the various ligands co-crystal to the target protein in 3D. Key: Figures A to F shows the 3D poses for the various ligands, while G and H show the various protein chains and ligand in the active site of the target protein.</p></figcaption></figure><p id="Par18">Figure <a href="#Fig2" class="usa-link">2</a> reveals the binding poses interactions in 3D (A to F) and the structure of the target proteins (G and H) in various modes. It reveals the interacting amino acids, and these were for ligand A: Trp164, Arg323, Thr178, Lys253, Phe176, Ile241, Lys309, Arg300, and Asn249. For ligand B, the amino acids were Asn174 and Arg329, while for ligand C, the amino acids were Arg300, Asn305, Leu239, PHe176, and Arg323. For ligand D, the amino acids were Asn305, Phe176, Arg300, and Glu117, while for ligand E, the amino acids were Gln175, Asn174, Arg329, Phe176, Ile241, Asn240, Glu117, Try308, Trp164, and Arg300. For ligand F, the amino acids were Phe176, Leu172, and Try308. This implies that ligand F, 5-(methylene) evodiamine, had the highest number of interacting amino acid residues, one residue more than the co-crystal and eight (8) more than the control drug, sofosbuvir. The various amino acid residues observed, their receptors, and how they interacted with the various atoms of the ligands are summarised in Table <a href="#Tab2" class="usa-link">2</a>. The result indicates that the co-crystal returned the highest number of interactions (<em>n</em> = 33), followed by sofosbuvir (<em>n</em> = 6), lauric acid (<em>n</em> = 3), NCTPMMDP and evodiamine (<em>n</em> = 4 each), and 5-(Methylene) evodiamine (<em>n</em> = 2). The dominant interactions were H-acceptors and ionic bonds. Hydrogen acceptors and donors indicate the presence of hydrogen bonds<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>A summary of the various interacting atoms, their interactions, distances and binding energies.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Ligand</th>
<th align="left" colspan="1" rowspan="1">Receptor</th>
<th align="left" colspan="1" rowspan="1">Interaction</th>
<th align="left" colspan="1" rowspan="1">Distance</th>
<th align="left" colspan="1" rowspan="1">E (kcal/mol)</th>
</tr></thead>
<tbody>
<tr><td align="left" colspan="5" rowspan="1">3mx5 / 3mx5_ Control(co-crystal)</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O5’ 7</td>
<td align="left" colspan="1" rowspan="1">NH2 ARG 300</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">3.04</td>
<td align="left" colspan="1" rowspan="1">− 1.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O1A 22</td>
<td align="left" colspan="1" rowspan="1">NH1 ARG 300</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.69</td>
<td align="left" colspan="1" rowspan="1">− 8.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O1A 22</td>
<td align="left" colspan="1" rowspan="1">NZ LYS 309</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.73</td>
<td align="left" colspan="1" rowspan="1">− 15.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O1B 23</td>
<td align="left" colspan="1" rowspan="1">N THR 178</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.72</td>
<td align="left" colspan="1" rowspan="1">− 6.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O1B 23</td>
<td align="left" colspan="1" rowspan="1">OG1 THR 178</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.62</td>
<td align="left" colspan="1" rowspan="1">− 2.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O1G 24</td>
<td align="left" colspan="1" rowspan="1">NH1 ARG 300</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.60</td>
<td align="left" colspan="1" rowspan="1">− 6.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O2 26</td>
<td align="left" colspan="1" rowspan="1">N ASN 240</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.77</td>
<td align="left" colspan="1" rowspan="1">− 3.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O2B 28</td>
<td align="left" colspan="1" rowspan="1">NZ LYS 253</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.76</td>
<td align="left" colspan="1" rowspan="1">− 2.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O2B 28</td>
<td align="left" colspan="1" rowspan="1">NZ LYS 309</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.72</td>
<td align="left" colspan="1" rowspan="1">− 11.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O2B 28</td>
<td align="left" colspan="1" rowspan="1">NH1 ARG 323</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.76</td>
<td align="left" colspan="1" rowspan="1">− 8.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O2B 28</td>
<td align="left" colspan="1" rowspan="1">NH2 ARG 323</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.75</td>
<td align="left" colspan="1" rowspan="1">− 8.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O3G 34</td>
<td align="left" colspan="1" rowspan="1">NZ LYS 253</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.64</td>
<td align="left" colspan="1" rowspan="1">− 19.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O3G 34</td>
<td align="left" colspan="1" rowspan="1">ND2 ASN 305</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.75</td>
<td align="left" colspan="1" rowspan="1">− 7.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O3G 34</td>
<td align="left" colspan="1" rowspan="1">NZ LYS 309</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.74</td>
<td align="left" colspan="1" rowspan="1">− 2.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O4 36</td>
<td align="left" colspan="1" rowspan="1">NE1 TRP 164</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">3.04</td>
<td align="left" colspan="1" rowspan="1">− 1.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O1A 22</td>
<td align="left" colspan="1" rowspan="1">NH1 ARG 300</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">2.69</td>
<td align="left" colspan="1" rowspan="1">− 6.9</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O1A 22</td>
<td align="left" colspan="1" rowspan="1">NH2 ARG 300</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">3.77</td>
<td align="left" colspan="1" rowspan="1">− 1.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O1A 22</td>
<td align="left" colspan="1" rowspan="1">NZ LYS 309</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">2.73</td>
<td align="left" colspan="1" rowspan="1">− 6.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O1A 22</td>
<td align="left" colspan="1" rowspan="1">NH1 ARG 323</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">2.98</td>
<td align="left" colspan="1" rowspan="1">− 4.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O1B 23</td>
<td align="left" colspan="1" rowspan="1">NZ LYS 253</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">3.89</td>
<td align="left" colspan="1" rowspan="1">− 0.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O1B 23</td>
<td align="left" colspan="1" rowspan="1">NH2 ARG 323</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">3.88</td>
<td align="left" colspan="1" rowspan="1">− 0.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O1G 24</td>
<td align="left" colspan="1" rowspan="1">NE ARG 300</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">3.55</td>
<td align="left" colspan="1" rowspan="1">− 1.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O1G 24</td>
<td align="left" colspan="1" rowspan="1">NH1 ARG 300</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">2.60</td>
<td align="left" colspan="1" rowspan="1">− 7.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O1G 24</td>
<td align="left" colspan="1" rowspan="1">NZ LYS 309</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">2.74</td>
<td align="left" colspan="1" rowspan="1">− 6.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O2A 27</td>
<td align="left" colspan="1" rowspan="1">NH1 ARG 323</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">2.98</td>
<td align="left" colspan="1" rowspan="1">− 4.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O2A 27</td>
<td align="left" colspan="1" rowspan="1">NH2 ARG 323</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">3.86</td>
<td align="left" colspan="1" rowspan="1">− 0.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O2B 28</td>
<td align="left" colspan="1" rowspan="1">NZ LYS 253</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">2.76</td>
<td align="left" colspan="1" rowspan="1">− 6.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O2B 28</td>
<td align="left" colspan="1" rowspan="1">NZ LYS 309</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">2.72</td>
<td align="left" colspan="1" rowspan="1">− 6.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O2B 28</td>
<td align="left" colspan="1" rowspan="1">NH1 ARG 323</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">2.76</td>
<td align="left" colspan="1" rowspan="1">− 6.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O2B 28</td>
<td align="left" colspan="1" rowspan="1">NH2 ARG 323</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">2.75</td>
<td align="left" colspan="1" rowspan="1">− 6.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O3G 34</td>
<td align="left" colspan="1" rowspan="1">NZ LYS 253</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">2.64</td>
<td align="left" colspan="1" rowspan="1">− 7.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O3G 34</td>
<td align="left" colspan="1" rowspan="1">NZ LYS 309</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">2.74</td>
<td align="left" colspan="1" rowspan="1">− 6.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> 6-ring</td>
<td align="left" colspan="1" rowspan="1">6-ring PHE 176</td>
<td align="left" colspan="1" rowspan="1">pi-pi</td>
<td align="left" colspan="1" rowspan="1">3.48</td>
<td align="left" colspan="1" rowspan="1">− 0.0</td>
</tr>
<tr>
<td align="left" colspan="3" rowspan="1">3mx5 / 3893_model1_out</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O2 36</td>
<td align="left" colspan="1" rowspan="1">N ASN 174</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">3.09</td>
<td align="left" colspan="1" rowspan="1">− 1.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O2 36</td>
<td align="left" colspan="1" rowspan="1">NH1 ARG 329</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.81</td>
<td align="left" colspan="1" rowspan="1">− 2.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> C1 7</td>
<td align="left" colspan="1" rowspan="1">6-ring PHE 176</td>
<td align="left" colspan="1" rowspan="1">H-pi</td>
<td align="left" colspan="1" rowspan="1">3.95</td>
<td align="left" colspan="1" rowspan="1">− 0.8</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1">3mx5 / 45375808_model1_out</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> C3 43</td>
<td align="left" colspan="1" rowspan="1">O SER 238</td>
<td align="left" colspan="1" rowspan="1">H-donor</td>
<td align="left" colspan="1" rowspan="1">3.33</td>
<td align="left" colspan="1" rowspan="1">− 0.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O6 3</td>
<td align="left" colspan="1" rowspan="1">NH1 ARG 323</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">3.07</td>
<td align="left" colspan="1" rowspan="1">− 3.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O9 25</td>
<td align="left" colspan="1" rowspan="1">NZ LYS 253</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.91</td>
<td align="left" colspan="1" rowspan="1">− 5.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O9 25</td>
<td align="left" colspan="1" rowspan="1">NZ LYS 309</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.90</td>
<td align="left" colspan="1" rowspan="1">− 8.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O4 55</td>
<td align="left" colspan="1" rowspan="1">N ASN 240</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.84</td>
<td align="left" colspan="1" rowspan="1">− 3.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> 6-ring</td>
<td align="left" colspan="1" rowspan="1">CG GLU 117</td>
<td align="left" colspan="1" rowspan="1">pi-H</td>
<td align="left" colspan="1" rowspan="1">3.81</td>
<td align="left" colspan="1" rowspan="1">− 0.6</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1">3mx5 / 49804500_model1_out</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> N2 11</td>
<td align="left" colspan="1" rowspan="1">OG SER 238</td>
<td align="left" colspan="1" rowspan="1">H-donor</td>
<td align="left" colspan="1" rowspan="1">2.87</td>
<td align="left" colspan="1" rowspan="1">− 1.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> N1 32</td>
<td align="left" colspan="1" rowspan="1">NZ LYS 309</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.85</td>
<td align="left" colspan="1" rowspan="1">90.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> N3 1</td>
<td align="left" colspan="1" rowspan="1">OE2 GLU 117</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">3.34</td>
<td align="left" colspan="1" rowspan="1">− 2.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> N4 2</td>
<td align="left" colspan="1" rowspan="1">OE2 GLU 117</td>
<td align="left" colspan="1" rowspan="1">Ionic</td>
<td align="left" colspan="1" rowspan="1">2.63</td>
<td align="left" colspan="1" rowspan="1">− 7.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> 6-ring</td>
<td align="left" colspan="1" rowspan="1">6-ring PHE 176</td>
<td align="left" colspan="1" rowspan="1">pi-pi</td>
<td align="left" colspan="1" rowspan="1">3.81</td>
<td align="left" colspan="1" rowspan="1">− 0.0</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1">3mx5 / 49804912_model1_out</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O1 1</td>
<td align="left" colspan="1" rowspan="1">N ASN 240</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.84</td>
<td align="left" colspan="1" rowspan="1">− 4.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> O2 2</td>
<td align="left" colspan="1" rowspan="1">NH2 ARG 300</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">3.12</td>
<td align="left" colspan="1" rowspan="1">− 1.9</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> 5-ring</td>
<td align="left" colspan="1" rowspan="1">NH1 ARG 300</td>
<td align="left" colspan="1" rowspan="1">pi-cation</td>
<td align="left" colspan="1" rowspan="1">3.64</td>
<td align="left" colspan="1" rowspan="1">− 1.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> 6-ring</td>
<td align="left" colspan="1" rowspan="1">6-ring PHE 176</td>
<td align="left" colspan="1" rowspan="1">pi-pi</td>
<td align="left" colspan="1" rowspan="1">3.98</td>
<td align="left" colspan="1" rowspan="1">− 0.0</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1">3mx5 / 56967508_model1_out</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> C3 8</td>
<td align="left" colspan="1" rowspan="1">OE2 GLU 117</td>
<td align="left" colspan="1" rowspan="1">H-donor</td>
<td align="left" colspan="1" rowspan="1">3.23</td>
<td align="left" colspan="1" rowspan="1">− 0.9</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> N2 3</td>
<td align="left" colspan="1" rowspan="1">NH2 ARG 300</td>
<td align="left" colspan="1" rowspan="1">H-acceptor</td>
<td align="left" colspan="1" rowspan="1">2.91</td>
<td align="left" colspan="1" rowspan="1">70.9</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec12"><h3 class="pmc_sec_title">Molecular dynamics simulation (MDS) evalutions</h3>
<p id="Par20">Given the inability of docking as a technique to reveal the overall thermodynamics and stability of the complexes formed, the molecular dynamics simulation evaluations were performed as previously reported<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. For all the complexes formed, a 300 ns MDS run was performed<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. The MDS runs were performed for both the complexes and the individual ligands, and the results are presented in Figs. <a href="#Fig3" class="usa-link">3</a>, <a href="#Fig4" class="usa-link">4</a>, <a href="#Fig5" class="usa-link">5</a>, <a href="#Fig6" class="usa-link">6</a> and <a href="#Fig7" class="usa-link">7</a>. Figure <a href="#Fig3" class="usa-link">3</a> shows the result of the RMSD analysis for the various complexes formed in the study. The result indicates fluctuations throughout the simulation runs for all the complexes formed. For the control (co-crystal), the RMSD values showed fluctuations over the entire simulation period. Its RMSD values that were lower than 2Å were observed around 20, 100–150, 200–220, and at 270–300 ns. For ligand 3893, regions of stability for the ligand were observed for the first 130 ns and thereafter, values higher than 2Å were observed, indicating instability. For the complex formed between ligands 45,375,808 and 49,804,500 and the target protein, the RMSD values only showed minimal stability at the first 50 ns of the simulation, while for the rest of the simulation, they were above 2Å for both the ligand and the target protein except for ligand 49,804,500 that gave stable values at 100–150 and 260–300 ns periods. The ligand 49,804,912 gave the most stable values for the majority of the simulation period with values less than 2.5 Å, except for the first 30 ns that were above 3Å. For ligand 49,804,912, the first 0-150 and 250–300 ns gave values that were less than 2 Å indicating stability. Overall, the ligands returned more stable values than the protein.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368151_41598_2025_89989_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7bc7/12368151/eb19a2ce9ec2/41598_2025_89989_Fig3_HTML.jpg" loading="lazy" id="d33e1409" height="437" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>RMSD values of the various complexes.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368151_41598_2025_89989_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7bc7/12368151/c271a2edc841/41598_2025_89989_Fig4_HTML.jpg" loading="lazy" id="d33e1424" height="419" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>RMSF values of the formed complexes.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368151_41598_2025_89989_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7bc7/12368151/a17e4795f8e9/41598_2025_89989_Fig5_HTML.jpg" loading="lazy" id="d33e1434" height="428" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>A graphical illustration of the contacts (protein-ligand).</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368151_41598_2025_89989_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7bc7/12368151/df1e7f43bf36/41598_2025_89989_Fig6_HTML.jpg" loading="lazy" id="d33e1480" height="448" width="669" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Frequency of contacts made by the various amino acids during simulation.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368151_41598_2025_89989_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7bc7/12368151/c64b4185b02f/41598_2025_89989_Fig7_HTML.jpg" loading="lazy" id="d33e1519" height="449" width="669" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Ligand interaction properties (RMSD, rGyr, MolSA, SASA and PSA).</p></figcaption></figure><p id="Par22">In addition to the RMSD values, the RMSF was also evaluated for all the complexes formed (Fig. <a href="#Fig4" class="usa-link">4</a>). The RMSF is a structural feature that is revealed when simulation is performed. It reveals changes in the amino acid residues during the simulation runs. For the control co-crystal, regions of high RMSF values above 2.4 Å were observed for amino residues at 130, 320, and 460, while for the complex with ligand 3893, high RMSF values higher than 2.4 Å were observed at residues 200, 320, and 460. For the complex between 45,375,808 and the target protein, spikes above 2.4Å were noted around amino acid residues 120 to 149, 320, and 460, while for 49,804,500 and the target, the spikes were observed around residues 150, 320, and 380–400 and 460. Similar high spikes were observed for the complexes formed between ligands 49,804,912 and 56,967,508 and the target protein, and these were at residues 150, 200, 320, 380, 400, and 460. Compared with the co-crystal control, the various complexes formed between the test ligands and the standard drug were all within the high spikes observed and were similar. Overall, in all the complexes, there were more regions with low RMSF than those with high RMSF, indicating that those regions with low RMSF values showed little divergence from the control co-crystal and the stability of the amino acids in those regions.</p>
<p id="Par25">In addition to the RMSF evaluation, an evaluation of the amino acid residues in the various complexes was also carried out (Fig. <a href="#Fig5" class="usa-link">5</a>). The result revealed the various amino acids that made contact with the ligands and the bonding type utilised in the process. The ligand (45375808)-protein complex contact returned 44 amino acids, while that of the control returned a total of 34 amino acids. The rest of the complexes returned 28, 52, 30, and 18 amino acid residues in contact, and these were complexes formed with ligands 56,967,508, 3893, 49,804,500, and 49,804,912, respectively. The top three bonding types in all the complexes were hydrogen, water bridges, and hydrophobic bonds. These bonding types were reported in an earlier study that also evaluated ligand-contact interaction<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. These bond types were also observed during the docking evaluations. Collectively, these bonds determine the binding affinities of the ligands to the target protein<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>–<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Specifically, the presence of hydrogen bonds is known to provide specificity and stability via the aligning of the ligands within the vicinity of the protein’s binding pocket, while hydrophobic interactions are also pervasive and are known to enhance the binding strength by minimising desolvation energy of the formed complexes. On the other hand, salt bridges have been shown to promote electrostatic complementarity. Furthermore, as a type of bond, H bond play an integral role in ligand-protein interactions where they function as either an electron acceptor or donor, as seen earlier in the docking results<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Ionic/polar interactions are the interactions of two opposite charges that are several distances apart but do not have a H bond. Water bridges, on the other hand, are H-driven protein-ligand interactions driven by a water molecule<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. The hydrogen bond is also important in drug discovery, specificity, metabolism, and adsorption<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. As observed in all the complexes, apart from the complex formed between ligand 49,804,912 and the target protein, the most common amino acid residue Arg300 made contact with the ligands using either with or ionic bond or a water bridge. Additionally, Leu120 consistently made contact with the various ligands even in the control co-crystal using only a hydrophobic bond, while Glu117 consistently utilised hydrogen and water bridges or ionic and water bridges. The presence of unique amino acids and their bonds is key to optimising small molecules as drug candidates<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>.</p>
<p id="Par27">In addition to the ligand-protein contact analysis, the actual number of contacts made during the simulation runs is depicted in Fig. <a href="#Fig6" class="usa-link">6</a>. The top-level plot for each ligand-protein interaction shows the number of contacts made during the simulation run. For the various ligands and their respective complexes, the number of contacts made were 30, 9, 15, 9, 9, and 6 for the control, 3893, 45,375,808, 4,980,500, 49,804,912, and 56,967,508, respectively. Furthermore, the number of contacts made by the individual amino acids depicted by the brown colour legend to the left of each plot indicates that the control had more amino acids that made contacts with the respective ligands during the 300 ns simulation run. The result also indicates that for complexes formed with ligands 3893, 49,804,500, 4,537,808, and, to a lesser extent, 49,804,912, the co-crystal uniformly made contact throughout the 300 ns simulation run.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Ligand properties analysis</h3>
<p id="Par28">Using simulation, the properties of our ligands were further explained using five parameters: RMSD, radius of gyration (rGyr), molecular surface area (MolSA), solvent accessible surface area (SASA), and polar surface area (PSA). Figure <a href="#Fig7" class="usa-link">7</a> shows the outputs of the various ligand properties following 300 ns of simulation run. The RMSD values ranged from 0.8 to 1.5, 1–3.5, 1.5–3.5, 0.8–2.4, 0.4–1.6, and 0.25–0.75 Å, respectively, for the control and ligands 3893, 45,375,808, 49,804,500, 49,804,912 and 56,967,508, respectively. Generally, RMSD values less than 2 Å indicates stability, implying that ligands 49,804,912 and 49,804,500 were the most stable. Furthermore, the radius of gyration also reveals fluctuations. The radii range were 4.2–4.6, 3.0–5.4, 4.0–5.2, 4.5–5.4, 3.92–4.08, and 3.5–3.7 Å<sup>2</sup>, respectively, for the control and ligands 3893, 45,375,808, 49,804,500, 49,804,912 and 56,967,508. The radius of the various ligands, when compared to the control, did not vary so much, indicating an overall compactness of the complexes<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. The molSA indicates minor fluctuations for all the ligands as well as the control co-crystal. Steady values were reached at 330, 260, 450, 430, 380, and 300 Å<sup>2</sup>, for the control, and ligands 33,893, 45,375,808, 49,804,500, 49,804,912 and 56,967,508, respectively. Similarly, steady values were reached for the SASA values at 50, 50, 90, 90, 60, and 50 Å<sup>2,</sup> respectively, for the control and ligands 3893, 45,375,808, 49,804,500, 49,804,912 and 56,967,508, while for the PSA, steady values were obtained at 460, 95, 180, 50, 48, and 20 Å<sup>2</sup>, respectively, for the control and ligands 3893, 45,375,808, 49,804,500, 49,804,912 and 56,967,508. The values of molSA, SASA, and PSA are measures of the surface area available for contact with water molecules. The larger the surface area, the more complexes are available for contact with water. As a parameter, it is useful in predicting the stability and folding of a molecule<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Principal component analysis (PCA) and dynamic cross-correlation matrix analysis</h3>
<p id="Par30">We further evaluated the nucleoprotein, 3MX5, dynamics using PCA as previously reported<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>–<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> and the results presented in Figs. <a href="#Fig8" class="usa-link">8</a> and <a href="#Fig9" class="usa-link">9</a>. The evaluations were done using the generated eigenvalues and eigenvectors. Both variables are important in the description of key dynamics of a protein, such as motion, folding, and any conformational changes in its backbone<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>–<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. The motion of the formed complexes in our study varied (Fig. <a href="#Fig8" class="usa-link">8</a>). Consistently, in all the complexes formed, the motion of the complexes was accounted for by principal components (PC) 1, 2, and 3. For the control, depicted as A, the values were 53.01, 8.34, and 6.21, while for B, the values were 36.23, 20.56, and 15.32. For ligands C and D, the values were 43.55, 10.25, and 8.17, 46.95, 8.37, and 6.18, respectively. For ligand D, the values were 40.81, 17.06, and 5.38, while for ligand E, their respective PC 1, 2, and 3 values were 28.01, 16.63, and 9.58. The variations in the PC values observed align with previous reports indicating conformational changes<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>–<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. The PC1 values indicate that the ligands E, D, and C had the highest values, indicating that the complexes they formed were subjected to the highest conformation changes after the control ligand. Figure <a href="#Fig9" class="usa-link">9</a> shows the result of the dynamics cross-correlation matrix for all the test ligands. The dynamics cross-correlation analysis allows for the evaluation of the positive and negative impacts of amino acids<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>–<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. The correlation values ranged from − 1 to + 1, that is, from strong negative to positive correlations (see colour legend in Fig. <a href="#Fig9" class="usa-link">9</a>). Apart from ligand 56,967,508, all the ligands had more positive interacting amino acids, thus more cumulative positive interactions or correlations than repulsive.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368151_41598_2025_89989_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7bc7/12368151/961713122f25/41598_2025_89989_Fig8_HTML.jpg" loading="lazy" id="d33e1615" height="945" width="668" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>PCA and eigenvalue rank plots for all the ligands. Key: Top right = control while for ligands 3893, 4,537,808, 49,804,500, 49,804,912 and 56,967,508, top left, middle left, middle right, bottom left and bottom right/.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368151_41598_2025_89989_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7bc7/12368151/5e9986aa5b29/41598_2025_89989_Fig9_HTML.jpg" loading="lazy" id="d33e1625" height="945" width="732" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Dynamic cross-correlation matrix for the ligands. Key: Top right = control while for ligands 3893, 4,537,808, 49,804,500, 49,804,912 and 56,967,508, top left, middle left, middle right, bottom left and bottom right/.</p></figcaption></figure></section><section id="Sec15"><h3 class="pmc_sec_title">FMO analysis of the various ligands</h3>
<p id="Par31">The results of the Frontier Molecular Orbitals (FMO) analysis is presented in Fig. <a href="#Fig10" class="usa-link">10</a>; Table <a href="#Tab3" class="usa-link">3</a>. Analyzing the FMO yields significant insights into their chemical properties, potentially informing drug discovery efforts and therapeutic strategies against Lassa fever<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. The energy levels of the FMO, as represented by the Highest Occupied Molecular (HOMO) and Lowest Unoccupied Molecular Orbital (LUMO), alongside the energy gap (ΔE), are crucial indicators of molecular reactivity<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. A smaller ΔE suggests heightened reactivity, which could be exploited in targeting these nucleoprotein for therapeutic intervention<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Moreover, parameters such as electronegativity (χ Pauling), global electrophilicity index (ω eV), and chemical potential (µ) offer insights into the nucleoprotein’s electron donation, acceptance capabilities, and potential for work transfer, respectively<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. These factor are pivotal in understanding the interactions of the nucleoprotein with other molecules, including potential drug candidates<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Additionally, hardness (η) and global softness (σ) shed light on the stability and reactivity of the nucleoprotein, essential considerations in drug design and optimization<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. The values of the highlighted parameters are presented in Table <a href="#Tab3" class="usa-link">3</a>. Therefore, in comparing the five proposed Lassa fever virus nucleoprotein based on their FMO parameters, significant variations.</p>
<figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7bc7/12368151/84408f9d501b/41598_2025_89989_Fig10_HTML.jpg" loading="lazy" id="d33e1667" height="556" width="788" alt="Fig. 10"></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>HUMO-LUMO energy analysis of the various ligands.</p></figcaption></figure><section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>FMO parameters of the various ligands.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Ligands</th>
<th align="left" colspan="1" rowspan="1">Lumo</th>
<th align="left" colspan="1" rowspan="1">Homo</th>
<th align="left" colspan="1" rowspan="1">ΔE in eV</th>
<th align="left" colspan="1" rowspan="1">χ Pauling</th>
<th align="left" colspan="1" rowspan="1">η in eV</th>
<th align="left" colspan="1" rowspan="1">σ</th>
<th align="left" colspan="1" rowspan="1">µ in eV</th>
<th align="left" colspan="1" rowspan="1">S</th>
<th align="left" colspan="1" rowspan="1">ω ev</th>
<th align="left" colspan="1" rowspan="1">Pubchem <br>IDs</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Lauric acid</td>
<td align="center" colspan="1" rowspan="1">0.06874535</td>
<td align="center" colspan="1" rowspan="1">− 0.2607218</td>
<td align="center" colspan="1" rowspan="1">0.329467175</td>
<td align="center" colspan="1" rowspan="1">− 0.09599</td>
<td align="center" colspan="1" rowspan="1">0.164734</td>
<td align="center" colspan="1" rowspan="1">6.070407</td>
<td align="center" colspan="1" rowspan="1">0.095988</td>
<td align="center" colspan="1" rowspan="1">3.035204</td>
<td align="center" colspan="1" rowspan="1">0.027966</td>
<td align="center" colspan="1" rowspan="1">3893</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Sofosbuvir</td>
<td align="center" colspan="1" rowspan="1">0.031427633</td>
<td align="center" colspan="1" rowspan="1">− 0.215992422</td>
<td align="center" colspan="1" rowspan="1">0.247420055</td>
<td align="center" colspan="1" rowspan="1">− 0.09228</td>
<td align="center" colspan="1" rowspan="1">0.12371</td>
<td align="center" colspan="1" rowspan="1">8.083419</td>
<td align="center" colspan="1" rowspan="1">0.092282</td>
<td align="center" colspan="1" rowspan="1">4.04171</td>
<td align="center" colspan="1" rowspan="1">0.034419</td>
<td align="center" colspan="1" rowspan="1">45,375,808</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p><em>N</em>-[[2-(3-chlorophenyl)-5-</p>
<p>(trifluoromethyl)pyrazol-3-yl]methyl]-2-(1-</p>
<p>methyl-2,3-dihydroindol-5-yl)</p>
<p>propanamide</p>
</td>
<td align="center" colspan="1" rowspan="1">− 0.100695902</td>
<td align="center" colspan="1" rowspan="1">-0.294780145</td>
<td align="center" colspan="1" rowspan="1">0.194084243</td>
<td align="center" colspan="1" rowspan="1">− 0.19774</td>
<td align="center" colspan="1" rowspan="1">0.097042</td>
<td align="center" colspan="1" rowspan="1">10.3048</td>
<td align="center" colspan="1" rowspan="1">0.197738</td>
<td align="center" colspan="1" rowspan="1">5.152402</td>
<td align="center" colspan="1" rowspan="1">0.201461</td>
<td align="center" colspan="1" rowspan="1">49,804,500</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p><em>N</em>-(2-Fluorobenzoyl)</p>
<p> Evodiamine</p>
</td>
<td align="center" colspan="1" rowspan="1">0.06675338</td>
<td align="center" colspan="1" rowspan="1">− 0.199010451</td>
<td align="center" colspan="1" rowspan="1">0.265763831</td>
<td align="center" colspan="1" rowspan="1">− 0.06613</td>
<td align="center" colspan="1" rowspan="1">0.132882</td>
<td align="center" colspan="1" rowspan="1">7.525479</td>
<td align="center" colspan="1" rowspan="1">0.066129</td>
<td align="center" colspan="1" rowspan="1">3.762739</td>
<td align="center" colspan="1" rowspan="1">0.016454</td>
<td align="center" colspan="1" rowspan="1">49,804,912</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>5-(Methylene)</p>
<p> Evodiamine</p>
</td>
<td align="center" colspan="1" rowspan="1">− 0.098762173</td>
<td align="center" colspan="1" rowspan="1">− 0.336175237</td>
<td align="center" colspan="1" rowspan="1">0.237413064</td>
<td align="center" colspan="1" rowspan="1">− 0.21747</td>
<td align="center" colspan="1" rowspan="1">0.118707</td>
<td align="center" colspan="1" rowspan="1">8.424136</td>
<td align="center" colspan="1" rowspan="1">0.217469</td>
<td align="center" colspan="1" rowspan="1">4.212068</td>
<td align="center" colspan="1" rowspan="1">0.1992</td>
<td align="center" colspan="1" rowspan="1">56,967,508</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par34">emerge. Lauric acid exhibits a relatively larger energy gap (ΔE = 0.329467175 eV) compared to others, suggesting lower reactivity, while compounds like NCTPMMDP (ΔE = 0.194084243 eV) and 5-(Methylene) evodiamine (ΔE = 0.237413064 eV), and <em>N</em>-(2-Fluorobenzoyl) evodiamine (ΔE = 0.265763831) display smaller energy gaps, indicative of higher reactivity.</p>
<p id="Par35">Electronegativity values vary across the compounds, influencing their electron donation and acceptance capabilities, such as lauric acid (-0.09599), sofosbuvir (– 0.09228), and NCTPMMDP (– 0.19774), 5-(Methylene) evodiamine (– 0.21747), and <em>N</em>-(2-Fluorobenzoyl) evodiamine (-0.06613). Differences in hardness (η) and global softness (σ) reflect variations in stability and polarizability. For instance, NCTPMMDP exhibits the least hardness and highest softness (0.097042 eV and 10.3048, respectively) compared to lauric acid (0.164734 eV and 6.070407), sofosbuvir (0.12371 eV and 0.12371), <em>N</em>-(2-Fluorobenzoyl) evodiamine (0.132882 eV and 7.525479), and 5-(Methylene) evodiamine (0.118707 eV and 8.424136). The HOMO-LUMO iso-surface plot is provided in Fig. <a href="#Fig10" class="usa-link">10</a> to describe the electron-rich region (HOMO) and electron-deficient region (LUMO). <em>N</em>-[[2-(3-chlo rophenyl)-5-(trifluoromethyl)pyrazol-3-yl]methyl]-2-(1-methyl-2,3-dihydroindol-5-yl) propenamide is observed to possess higher reactivity as a result of a smaller energy gap, the smallest chemical hardness, and the highest softness, indicating increased reactivity compared to the other studied compounds. This is followed by 5-(Methylene) evodiamine and Sofosbuvir, which possess a relatively lowered energy gap, reduced chemical hardness, and high softness, indicating relatively high chemical reactivity.</p>
<p id="Par36">The gradient observed among the ligands as evidenced by the docking scores was strongly influenced by the electronic properties of the ligands, such as electronegativity, energy gaps, chemical hardness, and softness<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. In our study, the highest docking scores was attained by <em>N</em>-(2-Fluorobenzoyl) evodiamine. This can be attributed to its electronic properties properties that included moderate electronegativity, a relatively low energy gap, reduced chemical hardness, and high softness. These properties put together enabled the ligand to form strong and diverse interactions, such as hydrogen bonds, hydrophobic contacts, and van der Waals forces, within the binding pocket. The molecular dynamics simulations evaluation further supported the findings via the observed stable RMSD, RMSF, and rGyr values, as well as significant conformational changes in the protein structure observed through PCA analysis. This underscores the ligand’s capacity to establish and maintain robust protein-ligand interactions and aligns with the previous findings<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. On the other hand, 5-(Methylene) evodiamine, while not achieving the highest docking score, still exhibited commendable activity that was driven by its very low energy gap and high softness. These properties of the ligand may have enhanced its adaptability and to the binding pocket of the target protein, allowing it to engage in stable interactions with the target protein. Its RMSD and PCA results further demonstrated its ability to induce conformational changes in the protein, reinforcing its potential as a viable inhibitor. Compared to N-(2-Fluorobenzoyl) Evodiamine and 5-(Methylene) evodiamine, sofosbuvir showed low energy gap and high softness, supporting its ability to form stable hydrogen bonds and polar interactions, Collectively, these findings highlight the importance of electronic properties in predicting docking outcomes. Specifically, our study indicates that ligands with lower energy gaps and higher softness are more reactive and adaptable, enabling stronger and more stable protein-ligand interactions.</p></section><section id="Sec16"><h3 class="pmc_sec_title">ADMET prediction</h3>
<p id="Par39">The process involved in the discovery of a new drug is complex. However, it is evolving rapidly with the advent of tools to predict ADMET properties, an important set of parameters in the drug development process<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. The ADMET properties of the various ligands and the co-crystal were predicted to further access their drug-like ability. First, the molecular descriptors were evaluated to reveal compliance with the LROF. The LROF advocates that for a compound to be regarded as having the potential to be orally absorbed effectively, it has to violate none or just one of the rules<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. These rules include molecular weight (MW ≤ 500), octanol/water partition coefficient (AlogP ≤ 5), hydrogen bond donor-HBD (≤ 5), and hydrogen bond acceptor (≤ 10). Only sofosbuvir violated one of the rules (MW &gt; 500). Violation of one of the rules does not make the drug less orally active<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. This implies that all the ligands and the control drug would be active orally. Next, to further evaluate their drug-likeness, their ADMET properties were evaluated.</p>
<p id="Par40">The result of the molecular descriptors is presented in supplementary Table <a href="#MOESM1" class="usa-link">1</a>, while the result of the pharmacokinetic properties (ADMET) of various ligands was predicted using the pkcsm tool and is presented in supplementary Table <a href="#MOESM1" class="usa-link">2</a>. The adsorption parameters examined were water solubility, CaCO<sub>2</sub> permeability, intestinal absorption (human), skin permeability, P-glycoprotein substrate, P-glycoprotein I inhibitor, and P-glycoprotein II inhibitor. The water solubility values of the ligands ranged from − 5.467 to – 4.181, while that of the control standard drug was − 3.953. The values overall indicate the solubility of the compounds<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. The CaCO<sub>2</sub> permeability values of the various ligands were all higher than the ideal value of &gt; 0.90<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. The intestinal absorption values for the various ligands (&gt; 90%) indicate that all of them, including the standard drug with the least value of 64.31, were all above the poor absorption cut-off value of 30%. This implies that the ligands have the capacity to survive the various intestinal hurdles and still have a good tissue distribution<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. The values for skin permeability being greater than log Kp &gt; − 2.5 for all the ligands depict poor skin permeability<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. The VDss values for ligand 5-(Methylene) evodiamine was the highest, followed by NCTPMMDP.</p>
<p id="Par41">Their high VD values indicate that they have the potential to reach tissues and not just plasma<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. BBB permeability being greater than − 0.3 for a given drug indicates poor brain distribution, while ligands 3893 and 45,375,808 showed the ability to cross the CNS with values greater than − 2<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Ligands 3893 and 45,375,808 were not substrates or inhibitors of the cytochrome system, an important detoxifying enzyme system in the body. Other ligands showed variable results in the cytochrome system. The excretion results for the various ligands indicate variable results for renal OCT2 substrate and total clearance. Only the ligand 56,967,508 was a substrate for renal OCT2. Toxicity prediction indicates that only ligands 56,967,508 and 49,804,500 showed cancer-causing potential as they were positive to the AMES toxicity test, while the standard drug, sofosbuvir, <em>N</em>-[[2-(3-chlorophenyl)-5-(trifluoromethyl)pyrazol-3-yl]methyl]-2-(methyl-2,3-dihydroindol-5-yl) propanamide and <em>N</em>-(2-fluorobenzoyl) evodiamine showed potential to be hepatotoxic<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. The hepatotoxicity concerns for both compounds can be handled by modification of both ligands to remove potentially toxic function groups or via the use of encapsulation and controlled release<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>.</p>
<p id="Par42">Chemically, the ligands used in the study possess diverse chemical structures. 5-(Methylene) evodiamine and N-(2-fluorobenzoyl) evodiamine are derivatives of evodiamine. The former possess a methylene group and an alkaloid ring, while the latter possess a fluorobenzoyl moiety attached to the evodiamine core. On the other hand, NCTPMMDP possesses a complex structure that contains the pyrazole-indole derivative that contains multiple functional groups, which include the trifluoromethyl group and a chlorophenyl ring. Lauric acid possesses carboxylic acid as its main functional group. The alkaloid ring, which is a key structure of evodiamine, is known for its diverse biological roles<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>–<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>, even against SARS-CoV-2 main protease and spike glycoprotein<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. Our docking result for the evodiamine derivatives indicates that they can interact with key amino acid residues within the milieu of the active site of the target protein via the observed hydrogen bonds and hydrophobic interactions, stabilising the protein-ligand complexes formed. The observed strong docking exhibited by these derivatives returned strong docking scores, indicating a strong binding affinity to the 3MX5 protein aligns with an earlier report that indicated that evodiamine derivatives can bind to both glycoprotein and nucleoprotein of lassa fever virus<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. These compounds most probably inhibit the viral protein by stabilising its inactive conformation, thereby blocking essential viral replication functions. On the other hand, NCTPMMDP possession of diverse groups enhances its electron-withdrawing and lipophilic properties. Thus, it is probable from our docking studies that this compound is capable of forming stable hydrogen bonds and van der Waals interactions with key residues that are central to the activity of the target protein and, in the process, block its enzymatic activity. The simple structure of lauric acid was visible by the lowest binding energy it returned via fewer interactions. Lauric acid as a ligand in this study may exert its anti-viral activity by inserting itself into the lipid layer of the virus and in the process disrupt the membrane activity of the virus<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. Sofosbuvir, unlike the rest of the ligands, is an approved drug; it is a nucleotide analogue pro-drug that targets RNA-dependent RNA polymerase (RdRp)<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. Even though our target protein is a nucleoprotein, it is essential for encapsidating the genome of the virus and also enables the activity of the RNA-dependent RNA polymerase (RdRp) activity<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. The present study hold a lot of prospects for the development of specific therapeutics against LFHV that is still elusive at the moment. In this study we have identified some ligands (<em>n</em> = 5) including the control drug with potential anti LHFV properties and this provides the theoretical basis for advancing the course for an anti-LFHV therapeutic. Furthermore, the findings in this offer a reproducible methodology to guide the screening of future compounds.</p></section></section><section id="Sec17"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par43">The aim of the study was to evaluate the anti-LHFV properties of -(Methylene) evodiamine, <em>N</em>-(2-Fluorobenzoyl) evodiamine, <em>N</em>-[[2-(3-chlorophenyl)-5-(trifluoromethyl)pyrazol-3-yl]methyl]-2-(1-methyl-2,3-dihydroindol-5-yl) propanamide, lauric acid, and sofosbuvir against the target protein, 3MX5, using entirely an in silico study approach. Docking scores for the test ligands ranged from − 9.5 to -12.3 kcal/mol, while that of the control and the standard drug were much lower and ranged from − 8.9 to -9.1 kcal/mol. Overall, the scores indicate a favourable binding to the active site of the target protein by the test ligands. The RMSD for the test ligands as well of the control all returned stable values that were generally less than 2.5 Å<sup>2</sup>. Similar low RMSF values (less than 2.4 Å<sup>2</sup>.) were also observed for the ligands and the control except for amino acid residues 150, 200, 320, 380, 400, and 460, respectively that recorded higher values. Put together, the RMSD and RMSF values of the ligands were indicative of ligand-protein complexes that were stable at some intervals of the simulation runs. The ligand interactions studies revealed stable ligand, as revealed by the favourable rGyr and RMSD values. The molSA, SASA, and PSA values corroborated the predicted absorption and Lipinski findings. Except for the control that had a MW greater than 500 g/mol, the test ligands all obeyed the Lipinski rule. All the ligands drove strong conformational changes within the backbone of the protein except for lauric that gave the least PCA values. Except for ligand 56,967,508, other ligands returned an excess of positive interacting amino acids that cumulated into more positive interactions than repulsive. Overall, the various test ligands showed more favourable ADMET and docking properties than the control and standard drug. Future studies should include in-vivo and in-vitro experimental studies, exploration of a broader number of ligands and viral targets.</p>
<section id="Sec18"><h3 class="pmc_sec_title">Limitations of the study</h3>
<p id="Par44">The present study provided excellent insights into the potential anti-LHFV properties of the tested ligands; however, there were some limitations. First, the study was an in silico study in design. Second, our simulation time frame was kept at 300 ns. Third, our tested ligands were small in terms of number. Future studies should focus on in vivo and in vitro designs, while future in silico studies aimed at assessing the anti-LHFV properties of the tested ligands or more ligands should utilise longer simulation times. Despite the limitations, the present study offers a number of advantages. First the study leverages on silico approaches and has identified promising lead compounds with better ADMET properties. Furthermore, our findings had provided useful insight into the interactions of the test ligands and the target protein, stability dynamics, and its changes of their complexes.</p></section></section><section id="Sec19"><h2 class="pmc_sec_title">Electronic supplementary material</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12368151/bin/41598_2025_89989_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (17.5KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>Not applicable.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>AEE, UOE, EM, IUB and MA conceived the project/idea; Funding: AEE, UOE, EM, IUB and FON; Methods: UOE and MA; Analyses: MA, SAM, EM, JP, SA, SA, DEC, and UOE; Writing of draft: UOE, EM, IUB, DEC, MA and JP. Review: All the authors reviewed the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>The study was funded by the 2021 Tertiary Education Trust Fund (TetFund) National Research Fund (NRF) (NRF2021_SETI_HSW_00478).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Data is provided within the manuscript or supplementary information files.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par45">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Frame, J. D., Baldwin, J. M. Jr, Gocke, D. J. &amp; Troup, J. M. Lassa fever, a new virus disease of man from West Africa. I. Clinical description and pathological findings. <em>Am. J. Trop. Med. Hyg.</em><strong>19</strong>(4), 670–676. 10.4269/ajtmh.1970.19.670 (1970).
</cite> [<a href="https://doi.org/10.4269/ajtmh.1970.19.670" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4246571/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Frame,%20J.%20D.,%20Baldwin,%20J.%20M.%20Jr,%20Gocke,%20D.%20J.%20&amp;%20Troup,%20J.%20M.%20Lassa%20fever,%20a%20new%20virus%20disease%20of%20man%20from%20West%20Africa.%20I.%20Clinical%20description%20and%20pathological%20findings.%20Am.%20J.%20Trop.%20Med.%20Hyg.19(4),%20670%E2%80%93676.%2010.4269/ajtmh.1970.19.670%20(1970)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Yun, N. E. &amp; Walker, D. H. Pathogenesis of Lassa fever. <em>Viruses</em><strong>4</strong>(10), 2031–2048. 10.3390/v4102031 (2012).
</cite> [<a href="https://doi.org/10.3390/v4102031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3497040/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23202452/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yun,%20N.%20E.%20&amp;%20Walker,%20D.%20H.%20Pathogenesis%20of%20Lassa%20fever.%20Viruses4(10),%202031%E2%80%932048.%2010.3390/v4102031%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>WHO. Lassa fever—Nigeria. (2023, accessed 30 Apr 2024). <a href="https://www.who.int/emergencies/disease-outbreak-news/item/2023-DON463" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.who.int/emergencies/disease-outbreak-news/item/2023-DON463</a>.</cite>
</li>
<li id="CR4">
<span class="label">4.</span><cite>NCDC. Lassa fever situation report. Epidemiology. Week 7 2024. (2024, accessed 26 Apr 2024). <a href="https://ncdc.gov.ng/themes/common/files/sitreps/3702b20687cb7bea4834b4a2db1a13ac.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ncdc.gov.ng/themes/common/files/sitreps/3702b20687cb7bea4834b4a2db1a13ac.pdf</a>.</cite>
</li>
<li id="CR5">
<span class="label">5.</span><cite>Aloke, C. et al. Combating Lassa Fever in West African Sub-region: progress, challenges, and future perspectives. <em>Viruses</em><strong>15</strong>(1), 146. 10.3390/v15010146 (2023).
</cite> [<a href="https://doi.org/10.3390/v15010146" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9864412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36680186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aloke,%20C.%20et%20al.%20Combating%20Lassa%20Fever%20in%20West%20African%20Sub-region:%20progress,%20challenges,%20and%20future%20perspectives.%20Viruses15(1),%20146.%2010.3390/v15010146%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Monath, T. P., Newhouse, V. F., Kemp, G. E., Setzer, H. W. &amp; Cacciapuoti, A. Lassa virus isolation from Mastomys natalensis rodents during an epidemic in Sierra Leone. <em>Science</em><strong>185</strong>(4147), 263–265 (1974).
</cite> [<a href="https://doi.org/10.1126/science.185.4147.263" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4833828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Monath,%20T.%20P.,%20Newhouse,%20V.%20F.,%20Kemp,%20G.%20E.,%20Setzer,%20H.%20W.%20&amp;%20Cacciapuoti,%20A.%20Lassa%20virus%20isolation%20from%20Mastomys%20natalensis%20rodents%20during%20an%20epidemic%20in%20Sierra%20Leone.%20Science185(4147),%20263%E2%80%93265%20(1974)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>NCDC. NCDC Lassa fever outbreak weekly Situation Report No. 46–December 1, 2017. (2017, accessed 28 Apr 2024). <a href="https://ncdc.gov.ng/themes/common/files/sitreps/f4e54bdfa7457b4f3cdac28d93712a3c.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ncdc.gov.ng/themes/common/files/sitreps/f4e54bdfa7457b4f3cdac28d93712a3c.pdf</a>. </cite>
</li>
<li id="CR8">
<span class="label">8.</span><cite>Tewogbola, T. &amp; Aung, N. Lassa fever: history, causes, effects, and reduction strategies. <em>Int. J. One Health</em>. <strong>6</strong>, 95–98 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Tewogbola,%20T.%20&amp;%20Aung,%20N.%20Lassa%20fever:%20history,%20causes,%20effects,%20and%20reduction%20strategies.%20Int.%20J.%20One%20Health.%206,%2095%E2%80%9398%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>WHO. Lassa fever. (2017, accessed 31 Apr 2024). <a href="https://www.who.int/news-room/fact-sheets/detail/lassa-fever" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.who.int/news-room/fact-sheets/detail/lassa-fever</a>. </cite>
</li>
<li id="CR10">
<span class="label">10.</span><cite>Isaac, A. B. et al. Prospects of lassa fever candidate vaccines. <em>Afr. J. Infect. Dis.</em><strong>16</strong>(2 Suppl), 46–58. 10.21010/Ajid.v16i2S.6 (2022).
</cite> [<a href="https://doi.org/10.21010/Ajid.v16i2S.6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9480887/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36124324/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Isaac,%20A.%20B.%20et%20al.%20Prospects%20of%20lassa%20fever%20candidate%20vaccines.%20Afr.%20J.%20Infect.%20Dis.16(2%20Suppl),%2046%E2%80%9358.%2010.21010/Ajid.v16i2S.6%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Tschismarov, R. et al. Immunogenicity, safety, and tolerability of a recombinant measles-vectored Lassa fever vaccine: a randomised, placebo-controlled, first-in-human trial. <em>The Lancet</em><strong>401</strong>(10384), 1267–1276 (2023).</cite> [<a href="https://doi.org/10.1016/S0140-6736(23)00048-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36934733/" class="usa-link">PubMed</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Sallam, M., Al-Sanafi, M. &amp; Sallam, M. AA global map of COVID-19 vaccine acceptance rates per Country: an updated concise narrative review. <em>J. Multidiscipl. Healthc.</em><strong>15</strong>, 21–45. 10.2147/JMDH.S347669 (2022).</cite> [<a href="https://doi.org/10.2147/JMDH.S347669" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8760993/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35046661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sallam,%20M.,%20Al-Sanafi,%20M.%20&amp;%20Sallam,%20M.%20AA%20global%20map%20of%20COVID-19%20vaccine%20acceptance%20rates%20per%20Country:%20an%20updated%20concise%20narrative%20review.%20J.%20Multidiscipl.%20Healthc.15,%2021%E2%80%9345.%2010.2147/JMDH.S347669%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Wake, A. D. The acceptance rate toward COVID-19 vaccine in Africa: a systematic review and meta-analysis. <em>Glob. Pediatr. Health</em>. <strong>8</strong>, 2333794X211048738. 10.1177/2333794X211048738 (2021).
</cite> [<a href="https://doi.org/10.1177/2333794X211048738" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8488505/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34616860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wake,%20A.%20D.%20The%20acceptance%20rate%20toward%20COVID-19%20vaccine%20in%20Africa:%20a%20systematic%20review%20and%20meta-analysis.%20Glob.%20Pediatr.%20Health.%208,%202333794X211048738.%2010.1177/2333794X211048738%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Eyo, A. E. et al. Geometry optimization, impact of solvation on the spectral (FT-IR, UV,NMR) analysis, quantum chemical parameters, and the bioactivity of feruloyltyramine as a potential anti-Lassa virus agent via molecular docking. <em>Chemi. Phys. Impact</em><strong>7</strong>, 100338 (2023). </cite>
</li>
<li id="CR15">
<span class="label">15.</span><cite>Odey, M. O. et al. Unraveling the impact of polar solvation on the molecular geometry, spectroscopy (FT-IR, UV, NMR), reactivity (ELF, NBO, HOMO-LUMO) and antiviral inhibitory potential of Cissampeline by molecular docking approach. <em>Chem. Phys. Impact</em><strong>7</strong>, 100346 (2023). </cite>
</li>
<li id="CR16">
<span class="label">16.</span><cite>Kechi, E. L. et al. X-ray crystallography, molecular structure investigation,Quantum chemical studies, and molecular dynamic of rheosmin as inhibitors of hemorrhagic Lassa Virus. <em>J. Mol. Struct.</em><strong>2023</strong>, 136048 (2023).</cite>
</li>
<li id="CR17">
<span class="label">17.</span><cite>Akash, S. et al. Development of a new drug candidate for the inhibition of Lassa virus glycoprotein and nucleoprotein by modification of evodiamine as promising therapeutic agents. <em>Front. Microbiol.</em><strong>14</strong>, 1206872. 10.3389/fmicb.2023.1206872 (2023).
</cite> [<a href="https://doi.org/10.3389/fmicb.2023.1206872" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10366616/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37497547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Akash,%20S.%20et%20al.%20Development%20of%20a%20new%20drug%20candidate%20for%20the%20inhibition%20of%20Lassa%20virus%20glycoprotein%20and%20nucleoprotein%20by%20modification%20of%20evodiamine%20as%20promising%20therapeutic%20agents.%20Front.%20Microbiol.14,%201206872.%2010.3389/fmicb.2023.1206872%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Gavaraskar, K., Dhulap, S. &amp; Hirwani, R. R. Therapeutic and cosmetic applications of evodiamine and its derivatives—a patent review. <em>Fitoterapia</em><strong>106</strong>, 22–35. 10.1016/j.fitote.2015.07.019 (2015).
</cite> [<a href="https://doi.org/10.1016/j.fitote.2015.07.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26255828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gavaraskar,%20K.,%20Dhulap,%20S.%20&amp;%20Hirwani,%20R.%20R.%20Therapeutic%20and%20cosmetic%20applications%20of%20evodiamine%20and%20its%20derivatives%E2%80%94a%20patent%20review.%20Fitoterapia106,%2022%E2%80%9335.%2010.1016/j.fitote.2015.07.019%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>National Center for Biotechnology Information. PubChem Compound Summary for CID 45375808, Sofosbuvir. (2024, accessed 1 Mar 2024). <a href="https://pubchem.ncbi.nlm.nih.gov/compound/Sofosbuvir" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubchem.ncbi.nlm.nih.gov/compound/Sofosbuvir</a>.</cite>
</li>
<li id="CR20">
<span class="label">20.</span><cite>National Center for Biotechnology Information. PubChem Compound Summary for CID 49804500. (2024, accessed 1 Mar 2024). <a href="https://pubchem.ncbi.nlm.nih.gov/compound/49804500" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubchem.ncbi.nlm.nih.gov/compound/49804500</a>.</cite>
</li>
<li id="CR21">
<span class="label">21.</span><cite>Liu, J. et al. Evodiamine and Rutaecarpine as potential anticancer compounds: a combined computational study. <em>Int. J. Mol. Sci.</em><strong>23</strong>(19), 11513. 10.3390/ijms231911513 (2022).
</cite> [<a href="https://doi.org/10.3390/ijms231911513" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9570036/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36232809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20J.%20et%20al.%20Evodiamine%20and%20Rutaecarpine%20as%20potential%20anticancer%20compounds:%20a%20combined%20computational%20study.%20Int.%20J.%20Mol.%20Sci.23(19),%2011513.%2010.3390/ijms231911513%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Matsue, M. et al. Measuring the antimicrobial activity of Lauric Acid against various Bacteria in human gut microbiota using a New Method. <em>Cell Transplant.</em><strong>28</strong>(12), 1528–1541. 10.1177/0963689719881366 (2019).
</cite> [<a href="https://doi.org/10.1177/0963689719881366" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6923562/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31665909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Matsue,%20M.%20et%20al.%20Measuring%20the%20antimicrobial%20activity%20of%20Lauric%20Acid%20against%20various%20Bacteria%20in%20human%20gut%20microbiota%20using%20a%20New%20Method.%20Cell%20Transplant.28(12),%201528%E2%80%931541.%2010.1177/0963689719881366%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Qi, X. et al. Cap binding and immune evasion revealed by Lassa nucleoprotein structure. <em>Nature</em><strong>468</strong>(7325), 779–783. 10.1038/nature09605 (2010).
</cite> [<a href="https://doi.org/10.1038/nature09605" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3057469/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21085117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qi,%20X.%20et%20al.%20Cap%20binding%20and%20immune%20evasion%20revealed%20by%20Lassa%20nucleoprotein%20structure.%20Nature468(7325),%20779%E2%80%93783.%2010.1038/nature09605%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Zielesny, A. Chemistry Software Package ChemOffice Ultra 2005 (2005).</cite>
</li>
<li id="CR25">
<span class="label">25.</span><cite>Belal, A. et al. Toward the discovery of SARS-CoV-2 main protease inhibitors: exploring therapeutic potentials of evodiamine and its derivatives, virtual screening, molecular docking, and molecular dynamic studies. <em>Nat. Prod. Commun.</em><strong>17</strong>(12), 1934578X221143082 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Belal,%20A.%20et%20al.%20Toward%20the%20discovery%20of%20SARS-CoV-2%20main%20protease%20inhibitors:%20exploring%20therapeutic%20potentials%20of%20evodiamine%20and%20its%20derivatives,%20virtual%20screening,%20molecular%20docking,%20and%20molecular%20dynamic%20studies.%20Nat.%20Prod.%20Commun.17(12),%201934578X221143082%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Jejurikar, B. L. &amp; Rohane, S. H. Drug designing in discovery studio (2021). </cite>
</li>
<li id="CR27">
<span class="label">27.</span><cite>Kubra, B. et al. Inhibition of the predicted allosteric site of the SARS-CoV-2 main protease through flavonoids. <em>J. Biomol. Struct. Dyn.</em><strong>41</strong>(18), 9103–9120 (2023).
</cite> [<a href="https://doi.org/10.1080/07391102.2022.2140201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36404610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kubra,%20B.%20et%20al.%20Inhibition%20of%20the%20predicted%20allosteric%20site%20of%20the%20SARS-CoV-2%20main%20protease%20through%20flavonoids.%20J.%20Biomol.%20Struct.%20Dyn.41(18),%209103%E2%80%939120%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Huang, W. et al. Allosite: a method for predicting allosteric sites. <em>Bioinf. (Oxford Engl.)</em><strong>29</strong>(18), 2357–2359. 10.1093/bioinformatics/btt399 (2013).</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt399" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23842804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20W.%20et%20al.%20Allosite:%20a%20method%20for%20predicting%20allosteric%20sites.%20Bioinf.%20(Oxford%20Engl.)29(18),%202357%E2%80%932359.%2010.1093/bioinformatics/btt399%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Stefaniu, A. Introductory chapter: molecular docking and molecular dynamics techniques to achieve rational drug design. <em>IntechOpen</em>10.5772/intechopen.84200 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Stefaniu,%20A.%20Introductory%20chapter:%20molecular%20docking%20and%20molecular%20dynamics%20techniques%20to%20achieve%20rational%20drug%20design.%20IntechOpen10.5772/intechopen.84200%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Sarma, M., Borkotoky, S. &amp; Dubey, V. K. Structure-based drug designing against Leishmania donovani using docking and molecular dynamics simulation studies: exploring glutathione synthetase as a drug target. <em>J. Biomol. Struct. Dynamics</em><strong>2023</strong>, 1–9. (2023).</cite> [<a href="https://doi.org/10.1080/07391102.2023.2240429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37491862/" class="usa-link">PubMed</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Trott, O. &amp; Olson, A. J. AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading. <em>J. Comput. Chem.</em><strong>31</strong>(2), 455–461. 10.1002/jcc.21334 (2009).</cite> [<a href="https://doi.org/10.1002/jcc.21334" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3041641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19499576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trott,%20O.%20&amp;%20Olson,%20A.%20J.%20AutoDock%20Vina:%20improving%20the%20speed%20and%20accuracy%20of%20docking%20with%20a%20new%20scoring%20function,%20efficient%20optimization,%20and%20multithreading.%20J.%20Comput.%20Chem.31(2),%20455%E2%80%93461.%2010.1002/jcc.21334%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Ye, N., Yang, Z. &amp; Liu, Y. Applications of density functional theory in COVID-19 drug modeling. <em>Drug Discovery Today</em>. <strong>27</strong>(5), 1411–1419. 10.1016/j.drudis.2021.12.017 (2022).
</cite> [<a href="https://doi.org/10.1016/j.drudis.2021.12.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8695517/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34954327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ye,%20N.,%20Yang,%20Z.%20&amp;%20Liu,%20Y.%20Applications%20of%20density%20functional%20theory%20in%20COVID-19%20drug%20modeling.%20Drug%20Discovery%20Today.%2027(5),%201411%E2%80%931419.%2010.1016/j.drudis.2021.12.017%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Hiscocks, J. &amp; Frisch, M. J. <em>Gaussian 09: IOps Reference</em> (eds. Caricato, M. &amp; Frisch, M. J.) (Gaussian, 2009).</cite>
</li>
<li id="CR34">
<span class="label">34.</span><cite>Lipinski, C. A., Lombardo, F., Dominy, B. W. &amp; Feeney, P. J. Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings. <em>Adv. Drug Deliv. Rev.</em><strong>46</strong>(1–3), 3–26. 10.1016/s0169-409x(00)00129-0 (2001).
</cite> [<a href="https://doi.org/10.1016/s0169-409x(00)00129-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11259830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lipinski,%20C.%20A.,%20Lombardo,%20F.,%20Dominy,%20B.%20W.%20&amp;%20Feeney,%20P.%20J.%20Experimental%20and%20computational%20approaches%20to%20estimate%20solubility%20and%20permeability%20in%20drug%20discovery%20and%20development%20settings.%20Adv.%20Drug%20Deliv.%20Rev.46(1%E2%80%933),%203%E2%80%9326.%2010.1016/s0169-409x(00)00129-0%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Pires, D. E., Blundell, T. L. &amp; Ascher, D. B. pkCSM: predicting small-molecule pharmacokinetic and toxicity properties using graph-based signatures. <em>J. Med. Chem.</em><strong>58</strong>(9), 4066–4072 (2015).
</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.5b00104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4434528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25860834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pires,%20D.%20E.,%20Blundell,%20T.%20L.%20&amp;%20Ascher,%20D.%20B.%20pkCSM:%20predicting%20small-molecule%20pharmacokinetic%20and%20toxicity%20properties%20using%20graph-based%20signatures.%20J.%20Med.%20Chem.58(9),%204066%E2%80%934072%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Mboto, C. I. et al. In-silico evaluation of bioactive compounds from selected medicinal plants from Southern Nigeria against hepatitis C virus genotype 1 RNA-directed RNA polymerase. <em>Sci. Afr.</em><strong>22</strong>, e01919 (2023).</cite>
</li>
<li id="CR37">
<span class="label">37.</span><cite>Pantsar, T. &amp; Poso, A. Binding affinity via docking: fact and fiction. <em>Mol. (Basel Switzerl.)</em><strong>23</strong>(8), 1899. 10.3390/molecules23081899 (2018).</cite> [<a href="https://doi.org/10.3390/molecules23081899" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6222344/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30061498/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pantsar,%20T.%20&amp;%20Poso,%20A.%20Binding%20affinity%20via%20docking:%20fact%20and%20fiction.%20Mol.%20(Basel%20Switzerl.)23(8),%201899.%2010.3390/molecules23081899%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Ramírez, D. &amp; Caballero, J. IIs it reliable to take the molecular docking top scoring position as the best solution without considering available structural data? <em>Mol. (Basel Switzerl.)</em><strong>23</strong>(5), 1038. 10.3390/molecules23051038 (2018).</cite> [<a href="https://doi.org/10.3390/molecules23051038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6102569/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29710787/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ram%C3%ADrez,%20D.%20&amp;%20Caballero,%20J.%20IIs%20it%20reliable%20to%20take%20the%20molecular%20docking%20top%20scoring%20position%20as%20the%20best%20solution%20without%20considering%20available%20structural%20data?%20Mol.%20(Basel%20Switzerl.)23(5),%201038.%2010.3390/molecules23051038%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>ur Rashid, H. et al. Molecular docking and dynamic simulations of Cefixime, Etoposide and Nebrodenside a against the pathogenic proteins of SARS-CoV-2. <em>J. Mol. Struct.</em><strong>1247</strong>, 131296 (2022).
</cite> [<a href="https://doi.org/10.1016/j.molstruc.2021.131296" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8362340/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34404957/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?ur%20Rashid,%20H.%20et%20al.%20Molecular%20docking%20and%20dynamic%20simulations%20of%20Cefixime,%20Etoposide%20and%20Nebrodenside%20a%20against%20the%20pathogenic%20proteins%20of%20SARS-CoV-2.%20J.%20Mol.%20Struct.1247,%20131296%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Owoloye, A. J. et al. Molecular docking, simulation and binding free energy analysis of small molecules as pf HT1 inhibitors. <em>Plos One</em><strong>17</strong>(8), e0268269 (2022).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0268269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9417013/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36026508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Owoloye,%20A.%20J.%20et%20al.%20Molecular%20docking,%20simulation%20and%20binding%20free%20energy%20analysis%20of%20small%20molecules%20as%20pf%20HT1%20inhibitors.%20Plos%20One17(8),%20e0268269%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Sun, Q. The hydrophobic effects: our current understanding. <em>Molecules</em><strong>27</strong>(20), 7009 (2022).
</cite> [<a href="https://doi.org/10.3390/molecules27207009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9609269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36296602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20Q.%20The%20hydrophobic%20effects:%20our%20current%20understanding.%20Molecules27(20),%207009%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Basu, S. &amp; Biswas, P. Salt-bridge dynamics in intrinsically disordered proteins: a trade-off between electrostatic interactions and structural flexibility. <em>Biochim. Biophys. Acta (BBA)-Proteins Proteom.</em><strong>1866</strong>(5–6), 624–641 (2018).</cite> [<a href="https://doi.org/10.1016/j.bbapap.2018.03.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29548979/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Basu,%20S.%20&amp;%20Biswas,%20P.%20Salt-bridge%20dynamics%20in%20intrinsically%20disordered%20proteins:%20a%20trade-off%20between%20electrostatic%20interactions%20and%20structural%20flexibility.%20Biochim.%20Biophys.%20Acta%20(BBA)-Proteins%20Proteom.1866(5%E2%80%936),%20624%E2%80%93641%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Madushanka, A., Moura, R. T. Jr., Verma, N. &amp; Kraka, E. Quantum mechanical assessment of protein–ligand hydrogen bond strength patterns: insights from semiempirical tight-binding and local vibrational mode theory. <em>Int. J. Mol. Sci.</em><strong>24</strong>(7), 6311. 10.3390/ijms24076311 (2023).
</cite> [<a href="https://doi.org/10.3390/ijms24076311" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10094336/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37047283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Madushanka,%20A.,%20Moura,%20R.%20T.%20Jr.,%20Verma,%20N.%20&amp;%20Kraka,%20E.%20Quantum%20mechanical%20assessment%20of%20protein%E2%80%93ligand%20hydrogen%20bond%20strength%20patterns:%20insights%20from%20semiempirical%20tight-binding%20and%20local%20vibrational%20mode%20theory.%20Int.%20J.%20Mol.%20Sci.24(7),%206311.%2010.3390/ijms24076311%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Rampogu, S., Lee, G., Park, J. S., Lee, K. W. &amp; Kim, M. O. Molecular docking and molecular dynamics simulations discover curcumin analogue as a plausible dual inhibitor for SARS-CoV-2. <em>Int. J. Mol. Sci.</em><strong>23</strong>(3), 1771. 10.3390/ijms23031771 (2022).
</cite> [<a href="https://doi.org/10.3390/ijms23031771" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8836015/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35163692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rampogu,%20S.,%20Lee,%20G.,%20Park,%20J.%20S.,%20Lee,%20K.%20W.%20&amp;%20Kim,%20M.%20O.%20Molecular%20docking%20and%20molecular%20dynamics%20simulations%20discover%20curcumin%20analogue%20as%20a%20plausible%20dual%20inhibitor%20for%20SARS-CoV-2.%20Int.%20J.%20Mol.%20Sci.23(3),%201771.%2010.3390/ijms23031771%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Savojardo, C., Manfredi, M., Martelli, P. L. &amp; Casadio, R. Solvent accessibility of residues undergoing pathogenic variations in humans: from protein structures to protein sequences. <em>Front. Mol. Biosci.</em><strong>7</strong>, 626363 (2021).
</cite> [<a href="https://doi.org/10.3389/fmolb.2020.626363" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7817970/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33490109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Savojardo,%20C.,%20Manfredi,%20M.,%20Martelli,%20P.%20L.%20&amp;%20Casadio,%20R.%20Solvent%20accessibility%20of%20residues%20undergoing%20pathogenic%20variations%20in%20humans:%20from%20protein%20structures%20to%20protein%20sequences.%20Front.%20Mol.%20Biosci.7,%20626363%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Ugbe, F. A. et al. Cheminformatics-based discovery of new organoselenium compounds with potential for the treatment of cutaneous and visceral leishmaniasis. <em>J. Biomol. Struct. Dyn.</em><strong>2023</strong>, 1–24 (2023). </cite> [<a href="https://doi.org/10.1080/07391102.2023.2279269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37937770/" class="usa-link">PubMed</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Akash, S. et al. Novel computational and drug design strategies for the inhibition of human T-cell leukemia virus 1-associated lymphoma by Astilbin derivatives. <em>J. Biomol. Struct. Dyn. </em><strong>2023</strong>, 1–16 (2023). </cite> [<a href="https://doi.org/10.1080/07391102.2023.2294376" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38131136/" class="usa-link">PubMed</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Nwaokorie, F. et al. In-silico assessment of bioactive compounds from chewing stick(Salvadora persica) against N-acetylneuraminate lyase (5ZKA) of Fusobacterium nucleatum involved in salicyclic acid metabolism. <em>J. Mol. Struct.</em><strong>1316</strong>, 138733 (2024).</cite>
</li>
<li id="CR49">
<span class="label">49.</span><cite>Ganesan, A., Coote, M. L. &amp; Barakat, K. Molecular dynamics-driven drug discovery: leaping forward with confidence. <em>Drug Discov. Today</em><strong>22</strong>(2), 249–269 (2017).
</cite> [<a href="https://doi.org/10.1016/j.drudis.2016.11.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27890821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ganesan,%20A.,%20Coote,%20M.%20L.%20&amp;%20Barakat,%20K.%20Molecular%20dynamics-driven%20drug%20discovery:%20leaping%20forward%20with%20confidence.%20Drug%20Discov.%20Today22(2),%20249%E2%80%93269%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Babashkina, M. G., Taskin-Tok, T., Burkhanova, T. M. &amp; Safin, D. A. 1-hydroxy-6, 6-Dimethyl-3-Phenyl-1, 6-Dihydropyridine-2, 5-Dione as a promising inhibitor of the SARS-CoV-2 proteins: insight into the crystal structure, Hirshfeld surface analysis and computational study. <em>Polycycl. Aromat. Compd.</em><strong>43</strong>(5), 4729–4749 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Babashkina,%20M.%20G.,%20Taskin-Tok,%20T.,%20Burkhanova,%20T.%20M.%20&amp;%20Safin,%20D.%20A.%201-hydroxy-6,%206-Dimethyl-3-Phenyl-1,%206-Dihydropyridine-2,%205-Dione%20as%20a%20promising%20inhibitor%20of%20the%20SARS-CoV-2%20proteins:%20insight%20into%20the%20crystal%20structure,%20Hirshfeld%20surface%20analysis%20and%20computational%20study.%20Polycycl.%20Aromat.%20Compd.43(5),%204729%E2%80%934749%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Bakheit, A. H., Attwa, M. W., Kadi, A. A., Ghabbour, H. A. &amp; Alkahtani, H. M. Exploring the chemical reactivity, molecular docking, molecular dynamic simulation and ADMET properties of a tetrahydrothienopyridine derivative using computational methods. <em>Crystals</em><strong>13</strong>(7), 1020 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bakheit,%20A.%20H.,%20Attwa,%20M.%20W.,%20Kadi,%20A.%20A.,%20Ghabbour,%20H.%20A.%20&amp;%20Alkahtani,%20H.%20M.%20Exploring%20the%20chemical%20reactivity,%20molecular%20docking,%20molecular%20dynamic%20simulation%20and%20ADMET%20properties%20of%20a%20tetrahydrothienopyridine%20derivative%20using%20computational%20methods.%20Crystals13(7),%201020%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Guan, L. et al. ADMET-score—a comprehensive scoring function for evaluation of chemical drug-likeness. <em>MedChemComm</em><strong>10</strong>(1), 148–157. 10.1039/c8md00472b (2018).
</cite> [<a href="https://doi.org/10.1039/c8md00472b" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6350845/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30774861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guan,%20L.%20et%20al.%20ADMET-score%E2%80%94a%20comprehensive%20scoring%20function%20for%20evaluation%20of%20chemical%20drug-likeness.%20MedChemComm10(1),%20148%E2%80%93157.%2010.1039/c8md00472b%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Flores-Holguín, N., Frau, J. &amp; Glossman-Mitnik, D. In silico pharmacokinetics, ADMET study and conceptual DFT analysis of two plant cyclopeptides isolated from rosaceae as a computational Peptidology approach. <em>Front. Chem.</em><strong>9</strong>, 708364 (2021).
</cite> [<a href="https://doi.org/10.3389/fchem.2021.708364" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8397472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34458236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Flores-Holgu%C3%ADn,%20N.,%20Frau,%20J.%20&amp;%20Glossman-Mitnik,%20D.%20In%20silico%20pharmacokinetics,%20ADMET%20study%20and%20conceptual%20DFT%20analysis%20of%20two%20plant%20cyclopeptides%20isolated%20from%20rosaceae%20as%20a%20computational%20Peptidology%20approach.%20Front.%20Chem.9,%20708364%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Herdiana, Y., Wathoni, N., Shamsuddin, S. &amp; Muchtaridi, M. Drug release study of the chitosan-based nanoparticles. <em>Heliyon</em><strong>8</strong>, 1 (2022).</cite> [<a href="https://doi.org/10.1016/j.heliyon.2021.e08674" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8741465/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35028457/" class="usa-link">PubMed</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Lin, L. et al. Evodiamine: a extremely potential drug development candidate of alkaloids from <em>Evodia rutaecarpa</em>. <em>Int. J. Nanomed.</em><strong>19</strong>, 9843–9870. 10.2147/IJN.S459510 (2024).</cite> [<a href="https://doi.org/10.2147/IJN.S459510" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11430234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39345907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin,%20L.%20et%20al.%20Evodiamine:%20a%20extremely%20potential%20drug%20development%20candidate%20of%20alkaloids%20from%20Evodia%20rutaecarpa.%20Int.%20J.%20Nanomed.19,%209843%E2%80%939870.%2010.2147/IJN.S459510%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Hu, X. et al. Antiproliferative effects of alkaloid evodiamine and its derivatives. <em>Int. J. Mol. Sci.</em><strong>19</strong>(11), 3403. 10.3390/ijms19113403 (2018).
</cite> [<a href="https://doi.org/10.3390/ijms19113403" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6274956/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30380774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu,%20X.%20et%20al.%20Antiproliferative%20effects%20of%20alkaloid%20evodiamine%20and%20its%20derivatives.%20Int.%20J.%20Mol.%20Sci.19(11),%203403.%2010.3390/ijms19113403%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Bhambhani, S., Kondhare, K. R. &amp; Giri, A. P. Diversity in chemical structures and biological properties of plant alkaloids. <em>Mol. (Basel Switzerl.)</em><strong>26</strong>(11), 3374. 10.3390/molecules26113374 (2021).</cite> [<a href="https://doi.org/10.3390/molecules26113374" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8199754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34204857/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bhambhani,%20S.,%20Kondhare,%20K.%20R.%20&amp;%20Giri,%20A.%20P.%20Diversity%20in%20chemical%20structures%20and%20biological%20properties%20of%20plant%20alkaloids.%20Mol.%20(Basel%20Switzerl.)26(11),%203374.%2010.3390/molecules26113374%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Yamin, R. et al. Identifying plant-derived antiviral alkaloids as dual inhibitors of SARS- CoV-2 main protease and spike glycoprotein through computational screening. <em>Front. Pharmacol.</em><strong>15</strong>, 1369659 (2024). </cite> [<a href="https://doi.org/10.3389/fphar.2024.1369659" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11288853/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39086396/" class="usa-link">PubMed</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Nitbani, F. O., Tjitda, P. J. P., Nitti, F., Jumina, J. &amp; Detha, A. I. R. Antimicrobial properties of lauric acid and monolaurin in virgin coconut oil: a review. <em>ChemBioEng Rev.</em><strong>9</strong>(5), 442–461 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Nitbani,%20F.%20O.,%20Tjitda,%20P.%20J.%20P.,%20Nitti,%20F.,%20Jumina,%20J.%20&amp;%20Detha,%20A.%20I.%20R.%20Antimicrobial%20properties%20of%20lauric%20acid%20and%20monolaurin%20in%20virgin%20coconut%20oil:%20a%20review.%20ChemBioEng%20Rev.9(5),%20442%E2%80%93461%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Alrehaily, A., Elfiky, A. A., Ibrahim, I. M., Ibrahim, M. N. &amp; Sonousi, A. Novel sofosbuvir derivatives against SARS-CoV-2 RNA-dependent RNA polymerase: an in silico perspective. <em>Sci. Rep.</em><strong>13</strong>(1), 23080. 10.1038/s41598-023-49712-y (2023).
</cite> [<a href="https://doi.org/10.1038/s41598-023-49712-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10754943/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38155165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alrehaily,%20A.,%20Elfiky,%20A.%20A.,%20Ibrahim,%20I.%20M.,%20Ibrahim,%20M.%20N.%20&amp;%20Sonousi,%20A.%20Novel%20sofosbuvir%20derivatives%20against%20SARS-CoV-2%20RNA-dependent%20RNA%20polymerase:%20an%20in%20silico%20perspective.%20Sci.%20Rep.13(1),%2023080.%2010.1038/s41598-023-49712-y%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Radoshitzky, S. R., Buchmeier, M. J. &amp; de la Torre, J. C. Fields virology. In <em>Emerging Viruses, Vol. 1 </em> (eds. Peter, M. H. et al.) 784–809 (Wolters Kluwer/Lippincott Williams &amp; Wilkins, 2020).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368151/bin/41598_2025_89989_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (17.5KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Data is provided within the manuscript or supplementary information files.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-89989-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_89989.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (7.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12368151/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12368151/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368151%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368151/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12368151/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12368151/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40835743/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12368151/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40835743/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12368151/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12368151/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="glbnO8wzaaquMHedD0Xr9FM8lvvVcIkT326wxg1dD5bozMtSsx9c5GyyHhM8Yv1V">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
