
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Nanopore sequencing of intact aminoacylated tRNAs - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152541A68AF2BC430541A600629503F2.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368100/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Nanopore sequencing of intact aminoacylated tRNAs">
<meta name="citation_author" content="Laura K White">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO USA">
<meta name="citation_author" content="Aleksandar Radakovic">
<meta name="citation_author_institution" content="Department of Genetics, Harvard Medical School, Boston, MA USA">
<meta name="citation_author_institution" content="Department of Chemistry, Howard Hughes Medical Institute, The University of Chicago, Chicago, IL USA">
<meta name="citation_author" content="Marcin P Sajek">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO USA">
<meta name="citation_author_institution" content="Polish Academy of Sciences, Institute of Human Genetics, Poznan, Poland">
<meta name="citation_author" content="Kezia Dobson">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO USA">
<meta name="citation_author" content="Kent A Riemondy">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO USA">
<meta name="citation_author" content="Samantha del Pozo">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO USA">
<meta name="citation_author" content="Jack W Szostak">
<meta name="citation_author_institution" content="Department of Chemistry, Howard Hughes Medical Institute, The University of Chicago, Chicago, IL USA">
<meta name="citation_author" content="Jay R Hesselberth">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO USA">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7781">
<meta name="citation_doi" content="10.1038/s41467-025-62545-9">
<meta name="citation_pmid" content="40835813">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368100/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368100/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368100/pdf/41467_2025_Article_62545.pdf">
<meta name="description" content="The intricate landscape of tRNA modification presents persistent analytical challenges, which have impeded efforts to simultaneously resolve sequence, modification, and aminoacylation state at the level of individual tRNAs. To address these ...">
<meta name="og:title" content="Nanopore sequencing of intact aminoacylated tRNAs">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The intricate landscape of tRNA modification presents persistent analytical challenges, which have impeded efforts to simultaneously resolve sequence, modification, and aminoacylation state at the level of individual tRNAs. To address these ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368100/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12368100">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62545-9"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62545.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368100%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12368100/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12368100/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368100/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 20;16:7781. doi: <a href="https://doi.org/10.1038/s41467-025-62545-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62545-9</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Nanopore sequencing of intact aminoacylated tRNAs</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22White%20LK%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Laura K White</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Laura K White</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22White%20LK%22%5BAuthor%5D" class="usa-link"><span class="name western">Laura K White</span></a>
</div>
</div>
<sup>1,</sup><sup>✉,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Radakovic%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Aleksandar Radakovic</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Aleksandar Radakovic</span></h3>
<div class="p">
<sup>2</sup>Department of Genetics, Harvard Medical School, Boston, MA USA </div>
<div class="p">
<sup>3</sup>Department of Chemistry, Howard Hughes Medical Institute, The University of Chicago, Chicago, IL USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Radakovic%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Aleksandar Radakovic</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sajek%20MP%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Marcin P Sajek</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Marcin P Sajek</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO USA </div>
<div class="p">
<sup>4</sup>Polish Academy of Sciences, Institute of Human Genetics, Poznan, Poland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sajek%20MP%22%5BAuthor%5D" class="usa-link"><span class="name western">Marcin P Sajek</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dobson%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Kezia Dobson</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Kezia Dobson</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dobson%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kezia Dobson</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Riemondy%20KA%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Kent A Riemondy</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Kent A Riemondy</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Riemondy%20KA%22%5BAuthor%5D" class="usa-link"><span class="name western">Kent A Riemondy</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22del%20Pozo%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Samantha del Pozo</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Samantha del Pozo</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22del%20Pozo%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Samantha del Pozo</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Szostak%20JW%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Jack W Szostak</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Jack W Szostak</span></h3>
<div class="p">
<sup>3</sup>Department of Chemistry, Howard Hughes Medical Institute, The University of Chicago, Chicago, IL USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Szostak%20JW%22%5BAuthor%5D" class="usa-link"><span class="name western">Jack W Szostak</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hesselberth%20JR%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Jay R Hesselberth</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Jay R Hesselberth</span></h3>
<div class="p">
<sup>1</sup>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hesselberth%20JR%22%5BAuthor%5D" class="usa-link"><span class="name western">Jay R Hesselberth</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO USA </div>
<div id="Aff2">
<sup>2</sup>Department of Genetics, Harvard Medical School, Boston, MA USA </div>
<div id="Aff3">
<sup>3</sup>Department of Chemistry, Howard Hughes Medical Institute, The University of Chicago, Chicago, IL USA </div>
<div id="Aff4">
<sup>4</sup>Polish Academy of Sciences, Institute of Human Genetics, Poznan, Poland </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 26; Accepted 2025 Jul 24; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12368100  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40835813/" class="usa-link">40835813</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article"><div>
<strong>Previous version available:</strong> This article is based on a previously available preprint posted on bioRxiv on November 18, 2024: "<a href="/articles/PMC11601438/" class="usa-link">Nanopore sequencing of intact aminoacylated tRNAs</a>".</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The intricate landscape of tRNA modification presents persistent analytical challenges, which have impeded efforts to simultaneously resolve sequence, modification, and aminoacylation state at the level of individual tRNAs. To address these challenges, we introduce “aa-tRNA-seq”, an integrated method that uses chemical ligation to sandwich the amino acid of a charged tRNA in between the body of the tRNA and an adaptor oligonucleotide, followed by high throughput nanopore sequencing. Our approach reveals the identity of the amino acids attached to all tRNAs in a cellular sample, at the single molecule level. We describe machine learning models that enable the accurate identification of amino acid identities based on the unique signal distortions generated by the interactions between the amino acid in the RNA backbone and the nanopore motor protein and reader head. We apply aa-tRNA-seq to characterize the impact of the loss of specific tRNA modification enzymes, confirming the hypomodification-associated instability of specific tRNAs, and identifying additional candidate targets of modification. Our studies lay the groundwork for understanding the efficiency and fidelity of tRNA aminoacylation as a function of tRNA sequence, modification, and environmental conditions.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> tRNAs, RNA sequencing, Nanopores, Machine learning</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Accurate protein synthesis depends on aminoacylated tRNAs, but their identities have been hard to measure. Here, authors present aa-tRNA-seq, a nanopore-based method that reveals the amino acid, sequence, and modification status of individual tRNAs at single-molecule resolution.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Transfer RNAs (tRNAs) are the fundamental adaptors of translation, linking mRNA decoding and polypeptide synthesis<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Biogenesis of tRNAs is complex: following transcription, precursor tRNA molecules undergo multiple trimming and modification events, yielding small (~76 nt) and densely modified (~13 modifications/molecule) RNA molecules that are then competent for aminoacylation (reviewed in ref. <sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>). tRNA modifications can profoundly impact tRNA function through their impacts on stability<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>, aminoacylation efficiency<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, and expanded decoding capacity<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. tRNA charging levels vary in response to environmental conditions<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> and drive various physiological responses to stress<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>, starvation<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>, and proliferation<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. In addition, aminoacylation of tRNAs with non-cognate amino acids (“misaminoacylation”) has been observed at high rates in aminoacyl synthetases with proofreading defects and during oxidative stress<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>.</p>
<p id="Par4">High throughput analysis of tRNAs has been hampered by a lack of incisive methods to comprehensively and directly measure tRNA abundance, modification, and aminoacylation within the same experiment. Short-read DNA sequencing approaches have been developed to study tRNA charging, but begin by selective oxidation and β-elimination of non-aminoacylated tRNAs, marking this population through removal of the 3′-terminal nucleotide, followed by deacylation, adapter ligation, and reverse transcription<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>–<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. However, the large number of modifications installed on tRNAs make them poor substrates for reverse transcriptase (RT) enzymes. In addition, most tRNA modifications do not produce detectable RT signatures<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> or require chemical derivatization to do so<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. As such, current sequencing approaches for studying aminoacylated tRNAs rely on indirect information to analyze key properties of these molecules, using chemical and enzymatic treatments to infer the positions of body modifications and presence of charged amino acids. Hence, lower throughput and more cumbersome techniques capable of physically separating uncharged tRNA and aminoacylated tRNA (aa-tRNA)<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup> remain the gold standard for quantification of tRNA aminoacylation.</p>
<p id="Par5">A promising new direction is the application of nanopores to tRNA sequencing. Successful discrimination of several tRNAs by solid-state<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> and biological<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> nanopores has been followed by methods to directly sequence tRNAs on a commercially available nanopore platform<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>, enabling direct analysis of the RNA molecule<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Further refinement in adapter design and computational analysis have improved tRNA nanopore sequencing to the point that it is now possible to directly assess tRNA abundance and modification status with single molecule precision<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>.</p>
<p id="Par6">Here, we develop a nanopore sequencing approach (“aa-tRNA-seq”) that directly captures information on tRNA sequence, modifications, and aminoacylation in a single read. Our method enables selective capture of tRNAs based on their aminoacylation status, selectively embedding the amino acid of aa-tRNAs within the adapter-ligated tRNA molecule. In a separate step, we capture non-aminoacylated tRNA, facilitating comparative analyses of tRNA charging. We characterize nanopore signals produced by 20 proteinogenic amino acids using synthetic tRNA, and leverage these signals to train a recurrent neural network (RNN) to discriminate aminoacylated tRNAs from their uncharged counterparts, and extend this approach for pairwise amino acid classification. We apply the method to study changes in budding yeast tRNA populations during nutrient limitation and in conditions of tRNA hypomodification, confirming known and identifying unexpected changes in tRNA aminoacylation and abundance during these stress conditions.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">A chemical ligation approach enables selective capture of intact aminoacylated tRNAs</h3>
<p id="Par7">While investigating prebiotic roles of aminoacyl-RNAs, we developed a splinted ligation reaction that generates amino acid-bridged chimeric RNA molecules using a 5′-phosphorimidazole-activated oligoribonucleotide<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. We realized that aminoacyl-tRNAs were analogous to these substrates, so we tested and found that a synthetic tRNA-Gly-GCC aminoacylated with glycine or lysine using the Flexizyme<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup> underwent chemical ligation with moderate kinetics, while the non-aminoacylated tRNA yielded no detectable product (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>). To accelerate the reaction between the ɑ-amino group of the aminoacyl-tRNA and the activated adapter, we included 1-(2-Hydroxyethyl)imidazole (HEI) as an organocatalyst<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> (Fig. <a href="#Fig1" class="usa-link">1A</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">1c</a>), and subjected <em>Saccharomyces cerevisiae</em> tRNA to a 30 min chemical ligation at pH 5.5. Under these conditions, we achieved efficient ligation of aminoacylated budding yeast tRNA as measured by acid northern blot, with &lt;0.1% background ligation to a chemically deacylated tRNA control for tRNA-Gly-GCC (Fig. <a href="#Fig1" class="usa-link">1B</a>). To examine the efficiency of this reaction on budding yeast tRNA substrates, we stripped and reprobed the same membrane from Fig. <a href="#Fig1" class="usa-link">1B</a> using oligonucleotide probes complementary to isodecoders from each of the 20 tRNA isoacceptor families in yeast (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>). Of the 16 tRNA isodecoders that were separated sufficiently to enable densitometric quantification, the percent of aminoacylated species shifted upon chemical ligation ranged from 62 to 100%, with quantitative ligation of arginyl, asparaginyl, cysteinyl, glutaminyl, glycyl, and lysyl tRNA species (Supplementary Fig. <a href="#MOESM1" class="usa-link">1b</a>). Using a fluorescently labeled tRNA analog, we confirmed this HEI-catalyzed reaction yields nearly quantitative ligation in vitro to substrates that were Flexizyme-charged with 13 different amino acids (Supplementary Fig. <a href="#MOESM1" class="usa-link">1b, d</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. A chemical ligation strategy for capture of aminoacylated tRNAs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368100_41467_2025_62545_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/edee/12368100/5e8a7be40470/41467_2025_62545_Fig1_HTML.jpg" loading="lazy" id="d33e455" height="1000" width="765" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic of splinted chemical ligation of 5′-phosphorimidazolide activated adapter (blue) to aminoacyl-tRNA (black, with amino acid in red) in the presence of the catalyst 1-(2-Hydroxyethyl)imidazole (HEI). <strong>B</strong> Acidic charging northern of deacylated or untreated wild-type (BY4741) <em>S. cerevisiae</em> tRNA after chemical reaction in the presence (+) or absence (−) of activated 3′ adapter, compared to tRNA input only (I). Blot has been probed for budding yeast tRNA-Gly-GCC; asterisk indicates a presumed ligation intermediate. Experiment was performed independently twice with similar results. <strong>C</strong> “Chemical-charging northern blot” resolving charged and uncharged tRNA-Gly-GCC via chemical ligation and analysis on a mini (8 × 10 cm) TBE/7 M urea polyacrylamide gel followed by membrane transfer and hybridization with a Gly-GCC probe. Band sizes correspond to a single-stranded RNA ladder (in nucleotides, nt). Experiment was repeated multiple times during method development with consistent results. <strong>D</strong> Small RNAs (17–200 nt, 200 ng) from budding yeast were chemically ligated to activated 3′ RNA adapter in a splinted ligation as previous panels, followed by an optional enzymatic ligation with T4 RNA ligase 2 (RNL2) to attach a second, splinted RNA adapter to the 5′ end of the tRNA. Products were run on a 10% denaturing PAGE gel and stained with SYBR Gold. Band sizes correspond to a single-stranded RNA ladder (in nucleotides, nt). Source data for (<strong>B</strong>–<strong>D</strong>) are provided as a <a href="#MOESM4" class="usa-link">Source Data</a> file. <strong>E</strong> Schematic illustrating strategy for chemical ligation and sequencing of charged and uncharged biological tRNAs via nanopore direct RNA sequencing. <strong>F</strong> Normalized mean current in picoamps for synthetic tRNA charged with glycine vs an uncharged control. The region visualized includes the 3′ terminus of the tRNA (6 nt, positions 68–73), the CCA tail (positions 74–76), the aminoacylated position (dashed line), and the entirety of the 3′ adapter sequence. For each colored trace, the solid line is the mean signal, and the shading spans the standard deviation.</p>
<p><a href="#MOESM4" class="usa-link">Source Data</a>.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">A chemical-northern blot simplifies analysis of aminoacylated tRNA</h3>
<p id="Par8">The analysis of tRNA aminoacylation by acidic northern blotting requires careful handling to preserve the labile ester linkage, using a large format, low pH polyacrylamide gel run in cold, acidic buffer conditions to provide adequate resolution of charged and uncharged tRNA, which is achieved by &gt;12 h of electrophoresis depending on the isodecoder of interest<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>, though we found some aa-tRNAs (e.g., Glu-UUC, Asp-GUC) are poorly resolved (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>). Chemical ligation of aminoacyl-tRNA stabilizes the ester linkage<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> and significantly increases its size, enabling robust separation from non-acylated tRNA by acidic northern blot (Fig. <a href="#Fig1" class="usa-link">1B</a>). These features motivated a simpler approach to separate charged and uncharged tRNA using non-acidic denaturing polyacrylamide gel electrophoresis. In this “chemical-charging northern” (Fig. <a href="#Fig1" class="usa-link">1C</a>), budding yeast tRNA was chemically ligated as in Fig. <a href="#Fig1" class="usa-link">1B</a>, followed by a ~30 min electrophoretic separation on a standard TBE-urea mini-gel, membrane transfer, and probe hybridization for the same glycyl isodecoder. We found that 35% of Gly-GCC tRNA displays a gel shift after chemical ligation (Fig. <a href="#Fig1" class="usa-link">1C</a>, lane 4, consistent with the value from Fig. <a href="#Fig1" class="usa-link">1B</a>), with 7% background ligation to the deacylated sample (lane 2) due to incomplete deacylation (Supplementary Fig. <a href="#MOESM1" class="usa-link">1e, f</a><sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>). In the absence of the phosphorimidazole-activated 3′ oligoribonucleotide (lanes 1 &amp; 3), no gel shift for the Gly-GCC tRNA was apparent (Fig. <a href="#Fig1" class="usa-link">1C</a>).</p>
<p id="Par9">This strategy of stabilizing the aminoacyl ester and appending a bulky group to the alpha-amine to make it amenable to standard PAGE is reminiscent of earlier studies that leveraged highly-reactive biotin-NHS reagents to biotinylate the alpha-amine, which after incubation with streptavidin, could be resolved from deacylated tRNA with standard PAGE<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>–<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. However, our approach offers two key advantages: (1) very low background reactivity with deacylated tRNA (Fig. <a href="#Fig1" class="usa-link">1B</a>) due to the low concentration of the activated adapter used in the reaction and (2) direct modification of aminoacylated tRNA with an oligonucleotide adapter, which allows both the resolution of the aminoacylated from the deacylated tRNA by standard PAGE and the subsequent preparation of aminoacylated tRNA libraries for nanopore sequencing (vide infra).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Aminoacylated tRNAs can be analyzed by nanopore sequencing</h3>
<p id="Par10">We next assessed whether synthetic tRNA ligated to an activated 3′ oligoribonucleotide via a bridging amino acid was amenable to nanopore sequencing. We confirmed that adapters for direct tRNA nanopore sequencing<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> can be attached to budding yeast tRNA via sequential chemical and enzymatic ligations of the 3′ and 5′ sequences, respectively (Fig. <a href="#Fig1" class="usa-link">1D</a>), suggesting a clear strategy for the preparation of nanopore sequencing libraries containing aminoacylated tRNAs (Fig. <a href="#Fig1" class="usa-link">1E</a>). We synthesized Gly-tRNA charged with the Flexizyme<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, chemically ligated this to the same phosphorimidazolated 3′ adapter, gel-purified the ligation product, and enzymatically ligated the 5′ adapter using T4 RNA ligase 2 (RNL2). We then prepared a nanopore direct RNA sequencing library from the individual ligated Gly-tRNA using the RNA004 chemistry from Oxford Nanopore Technologies (ONT). Figure <a href="#Fig1" class="usa-link">1F</a> shows the difference in ionic current between a synthetic tRNA and the same sequence charged with glycine, with lower current for Gly-tRNA spanning multiple nucleotides as the aminoacylated molecule is pulled into the nanopore from its 3′ end during motor-catalyzed translocation.</p>
<p id="Par11">We next synthesized tRNA charged with each of the 20 standard proteinogenic amino acids using the Flexizyme and subjected them to the same library preparation. Quality control assessment revealed no major issues for 18 of 20 of these libraries; however, 41.0% of Asn-tRNA reads and 27.6% of cysteinyl-tRNA reads terminated aberrantly, consistent with these tRNAs becoming stuck in nanopores and subsequently ejected (“unblocked”) at higher rates than those charged with other, bulkier amino acids or the uncharged tRNA control (Supplementary Fig. <a href="#MOESM1" class="usa-link">3a</a>). As the Asn-tRNA and Cys-tRNA libraries also had lower sequencing yields, we sought to resolve this issue and tested multiple strategies, illustrated in Supplementary Fig. <a href="#MOESM1" class="usa-link">3b</a>. For Cys-tRNA, either (i) alkylation with chloroacetamide after chemical ligation or (ii) omitting the enzymatic ligation to the 5′ adapter resolved the pore blocking (Supplementary Fig. <a href="#MOESM1" class="usa-link">3c</a>). This second strategy also reduced pore blocking for tRNA-Asn, suggesting that the thiol of Cys and amide of Asn might interact with the 5′ RNA adapter, leading to the molecule becoming stuck in the nanopore during sequencing. Further testing revealed that changing the 5′ adapter from an all-RNA oligonucleotide to a DNA oligonucleotide with five ribonucleotides at its 3′ terminus (i.e., nearest the amino acid) ameliorated the issue for both Cys and Asn (Supplementary Fig. <a href="#MOESM1" class="usa-link">3c, d</a>); therefore, this DNA/RNA hybrid adapter was used in all subsequent experiments.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Discrimination of charged and uncharged biological tRNAs by nanopore sequencing</h3>
<p id="Par12">Modified basecalling of nanopore sequencing signals is achieved by training neural networks<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>; in the case of direct RNA sequencing, detecting the presence of a modification via comparison to an unmodified control is often a more trivial computational challenge than precise characterization of a modification’s identity<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>–<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. To extend this concept to aminoacylated tRNA sequencing, we sought to classify charged and uncharged tRNA sequencing reads, with the goal of developing a one-pot approach for nanopore tRNA sequencing to capture charged and uncharged molecules within the same sequencing library. We prepared separate “ground truth” libraries containing either (<strong><em>i</em></strong>) chemically ligated, charged tRNA, or (<strong><em>ii</em></strong>) deacylated, enzymatically ligated tRNA from budding yeast (Fig. <a href="#Fig2" class="usa-link">2A</a>). tRNAs were ligated to 3′ adapters containing unique sequences for charged and uncharged molecules, and a common 5′ adapter incorporating the design changes (DNA/RNA hybrid) from Supplementary Fig. <a href="#MOESM1" class="usa-link">3b</a>, and sequenced these libraries on PromethION <em>RNA</em> flow cells.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Model training and classification of aminoacylated tRNA reads from nanopore signal.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368100_41467_2025_62545_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/edee/12368100/48dfa407d08a/41467_2025_62545_Fig2_HTML.jpg" loading="lazy" id="d33e615" height="372" width="748" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic illustrating ground truth libraries for model training. Library <em>(i)</em> was prepared by only chemical ligation of the 3′ adapter to biological aminoacylated tRNAs, followed by gel purification and enzymatic ligation of the 5′ adapter. Library <em>(ii) was</em> prepared by enzymatic ligation of both adapters to deacylated biological tRNAs. <strong>B</strong> Strategy for dataset preparation and neural network training of a Remora model to identify charged and uncharged tRNA reads from nanopore current signals. <strong>C</strong> Schematic depicting reference-anchored signal from libraries <em>(i) and (ii)</em>, with the 6-nt training window enclosed within the yellow box, and the position of the amino acid via dashed line. Mean current signals in picoamps for charged and uncharged tRNA reads are indicated by solid lines within each colored trace, with corresponding shading spanning the standard deviation. <strong>D</strong> Density plot showing the distribution of aminoacylation prediction scores (modification likelihood or “ML” score, 0–255 scale) assigned by Remora model in validation libraries (95,859 deacylated tRNA reads, 33,494 aminoacylated tRNA reads). <strong>E</strong> Confusion matrix and key metrics illustrating the model’s performance at classifying a second set of replicates prepared as in (<strong>A</strong>), using a cutoff of ML &gt; 199 for identification of aminoacylated reads from nanopore signal.</p></figcaption></figure><p id="Par13">Informed by our synthetic tRNA sequencing results (Fig. <a href="#Fig1" class="usa-link">1F</a>), we trained a RNN on this dataset using the Remora software from ONT (Fig. <a href="#Fig2" class="usa-link">2B</a>), which predicts the presence of modified nucleotides using ionic current signals generated during nanopore sequencing. For training, we defined a six-nucleotide signal window spanning the invariant CCA at the 3′ terminus of mature tRNA (“CCA-G-GC”, where “G” represents the first base of the adapter most affected by the amino acid, and “GC” represent the next two bases of the adapter; depicted in Fig. <a href="#Fig2" class="usa-link">2C</a>), as the sequence in this region would be shared across all tRNA isodecoders. We generated a Remora dataset from 80% of these reads, with labels defined by the library of origin (<strong><em>i</em></strong> and <strong><em>ii</em></strong>, above), and then trained and evaluated a model on the reserved 20% test set. During signal-anchored inference, Remora models output predictions for “modified” positions in the ML (“modification likelihood”) tag of a BAM file; these values range from 0 to 255, where lower values are more likely to be canonical nucleotides. Inspection of these values in our libraries revealed a bimodal distribution, with 96.6% of reads from the charged library (<strong><em>i</em></strong>) bearing ML scores ≥200, compared to 1.2% of in the deacylated sample (<strong><em>ii</em></strong>) (Fig. <a href="#Fig2" class="usa-link">2D</a>).</p>
<p id="Par14">We generated a second biological replicate of the libraries in Fig. <a href="#Fig2" class="usa-link">2A</a> for model validation, performing reference-anchored inference using the trained model above and using a ML cutoff of &gt;199 to classify aminoacylated tRNAs. Figure <a href="#Fig2" class="usa-link">2E</a> illustrates the Remora model’s performance on this new dataset, with an F1 score of 0.966. This signal-based approach substantially outperformed an alignment-based one using unique 3′ adapter sequences to discriminate charged and uncharged tRNAs, which yielded an F1 score of 0.322 with low sensitivity for identifying charged tRNA reads (Fig. <a href="#Fig2" class="usa-link">2F</a>), and which we attribute to increased base-calling error caused by the embedded amino acid. Based on its clear improvement over the alignment-based approach, we focused on signal-based classification of tRNA aminoacylation state.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Measurement of tRNA aminoacylation with aa-tRNA-seq</h3>
<p id="Par15">To determine whether this approach could report on tRNA charging levels, we generated replicate aa-tRNA-seq libraries from wild-type budding yeast under conditions identical to Fig. <a href="#Fig1" class="usa-link">1B</a> (wild-type yeast in rich media), enabling an orthogonal comparison of tRNA aminoacylation measured by acidic northern (Fig. <a href="#Fig1" class="usa-link">1B</a>) and model-based classification of charged and uncharged nanopore sequencing reads. Across the 16 tRNA isodecoders quantifiable by densitometry (Supplementary Fig. <a href="#MOESM1" class="usa-link">1b</a>), we observed a Pearson’s correlation of 0.68; however, estimates of isodecoder charging by sequencing were uniformly lower than charging levels measured by northern (Fig. <a href="#Fig3" class="usa-link">3A</a>). While in principle this difference might be explained by differences in the rates of aberrant read termination between charged and uncharged molecules, the magnitude of this result was inconsistent with our observations in synthetic tRNA sequencing data after optimization of the 5′ adapter sequence (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). We therefore re-analyzed the initial dataset used for Remora model training, comparing the translocation time for tRNA reads in the first library prepared via chemical ligation to aminoacylated tRNA to the second library where tRNAs were deacylated and enzymatically ligated. This analysis revealed that the total translocation time for aminoacylated tRNA—across all isodecoders—is on average ~1.2-fold longer than their uncharged counterparts (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>), which is likely the primary driver of the underestimation of charging we observe in aa-tRNA-seq libraries (Fig. <a href="#Fig3" class="usa-link">3A</a>): in a complex mixture of biological tRNAs, uncharged tRNAs are sequenced faster than charged tRNAs, leading to a sampling bias toward uncharged tRNAs. As such, while absolute quantitation of aa-tRNAs requires further optimization, we focused on relative quantitation of tRNA abundance and charging.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Sequencing and analysis of budding yeast tRNAs via chemical-charging northern and aa-tRNA-seq.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368100_41467_2025_62545_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/edee/12368100/8de8902087fd/41467_2025_62545_Fig3_HTML.jpg" loading="lazy" id="d33e792" height="999" width="708" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Correlation between tRNA aminoacylation (% charging) as measured by acid northern (<em>y</em>-axis), and aa-tRNA-seq (mean ± SD of 3 biological replicates). Dashed line indicates Pearson correlation (two-sided; <em>r</em> and <em>p</em>-value shown); solid line indicates <em>y</em> = <em>x</em>. <strong>B</strong> Chemical-charging northern of Leu-CAA aminoacylation in a leucine auxotroph after 15 m leucine depletion. Shown are 3 biological replicates; ligated tRNA (% per lane) was normalized to 5S rRNA (lower inset). Calculated abundance values were normalized within each replicate to the sample grown in complete media (set to 1.0) to assess relative abundance under leucine-starved conditions. Source data are provided as a Source Data file. <strong>C</strong> Log<sub>2</sub> fold change in tRNA abundance and percent charged reads for all isodecoders from the same 3 biological replicates as (<strong>B</strong>). Data are shown as mean ± SD. <strong>D</strong> Volcano plot of mean aminoacylation fold change between leucine-depleted and complete media, based on replicates in (<strong>B</strong>, <strong>C</strong>). Each point represents the mean value for a tRNA isodecoder. <em>P</em>-values from two-sided <em>Z</em>-test with Benjamini–Hochberg correction; dashed line indicates the α threshold. <strong>E</strong> Chemical-charging northern Val-AAC tRNA in <em>trm8∆ trm4∆</em> (RTD-sensitive) and <em>trm8∆ trm4∆ met22∆</em> (RTD-resistant) strains after 3 h at 28 °C or 37 °C. Two biological replicates shown; signals normalized to 5S rRNA. The percent of chemically ligated Val-AAC tRNA (upper band) is indicated below each lane. Val-AAC tRNA abundance levels were normalized within each replicate to the sample grown at 28 °C (set to 1.0) and compared to a matched sample shifted to 37 °C. Source data are provided as a <a href="#MOESM4" class="usa-link">Source Data</a> file. <strong>F</strong> Log<sub>2</sub> fold change in tRNA abundance and percent charged reads in <em>trm8∆ trm4∆</em> cells after temperature shift, based on 3 biological replicates, including those in (<strong>E</strong>). Data shown as mean ± SD. <strong>G</strong> Volcano plot of mean fold change in aminoacylation between 37 °C and 28 °C for <em>trm8Δ trm4Δ</em> cells, using same replicates as (<strong>E</strong>, <strong>F</strong>). <em>P</em>-values from two-sided <em>Z</em>-test with Benjamini–Hochberg correction; dashed line indicates the α threshold. Data are presented as mean values. <strong>H</strong> Log<sub>2</sub> fold change in tRNA abundance and percent charged reads upon temperature shift in <em>trm8Δ trm4Δ met22Δ</em> cells, based on <em>n</em> = 3 biological replicates. Data are shown as mean ± SD.</p>
<p><a href="#MOESM4" class="usa-link">Source Data</a>.</p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">aa-tRNA-seq detects changes in tRNA aminoacylation during nutrient stress</h3>
<p id="Par16">We next sought to assess whether aa-tRNA-seq could detect dynamic changes in tRNA charging and focused on nutrient limitation, which causes rapid changes to the aminoacyl-tRNA pool. We subjected a <em>S. cerevisiae</em> leucine auxotroph to leucine starvation for 15 min, and found that, as previously reported<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>, this caused rapid depletion of aminoacylated leucyl-tRNA isodecoders (Fig. <a href="#Fig3" class="usa-link">3B–D</a>). Using a chemical-charging northern (Fig. <a href="#Fig3" class="usa-link">3B</a>), we observed a mean 28.4% decrease in the abundance of aminoacylated leucyl-tRNA. We performed aa-tRNA-seq on the same material, and examined global changes in tRNA abundance and aminoacylation upon leucine starvation. This analysis revealed a significant decrease in the charging of all four leucyl tRNA isodecoders present in budding yeast, while most other tRNAs remained unaffected. Surprisingly, we also found a concomitant increase in the charging of Ala-tRNA isodecoders (3.97-fold for Ala-UGC, <em>p</em> &lt; 0.001; 2.12-fold for Ala-AGC, <em>p</em> = 0.08<em>;</em> BH-adjusted <em>Z</em> test; Fig. <a href="#Fig3" class="usa-link">3C, D</a>), an effect that was not detected in earlier microarray-based experiments examining tRNA charging in response to leucine starvation<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. We validated this isodecoder-specific effect by chemical-charging northern, where we observed a mean 2.62-fold increase for Ala-TGC compared to a mean 1.10-fold increase across three other unaffected isodecoders (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>).</p></section><section id="Sec9"><h3 class="pmc_sec_title">aa-tRNA-seq detects interdependence between tRNA modifications and aminoacylation during rapid tRNA decay</h3>
<p id="Par17">Hypomodified tRNAs are susceptible to “rapid tRNA decay” (RTD) in budding yeast, fission yeast, and bacteria<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>–<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. For example, Val-AAC tRNA lacking 5-methyl cytidine and 7-methyl guanine in <em>trm8∆ trm4∆</em> budding yeast is rapidly destabilized and degraded at high temperature by 5′−3′ exonucleolytic decay<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. We cultured a <em>trm4∆ trm8∆</em> strain and a control bearing an additional disruption of <em>MET22</em> at the permissive temperature of 28 °C followed by a shift to the non-permissive temperature of 37 °C for three hours. Deletion of <em>MET22</em> causes accumulation of pAp (adenosine 3′,5′ bisphosphate, a competitive inhibitor of 5′−3′ exonucleases<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>), suppressing RTD<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. We isolated small RNA from each strain and temperature in biological triplicate, and prepared this material for analysis by chemical-charging northern and aa-tRNA-seq. Both approaches confirmed defects in stability and aminoacylation for Val-AAC<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. By chemical-charging northern, aminoacylation levels for Val-AAC drop ~twofold upon shift to the nonpermissive temperature in <em>trm8∆ trm4∆</em> cells, and this effect was suppressed by <em>met22∆</em> (Fig. <a href="#Fig3" class="usa-link">3E</a>). This effect is readily detected by aa-tRNA-seq, where Val-AAC undergoes significant changes in both tRNA abundance (~3.6-fold reduction, <em>p</em> = 2.93 × 10<sup>−8</sup>, BH-adjusted Z test) and aminoacylation (~3.3-fold decrease, <em>p</em> = 1.99 × 10<sup>−6</sup>, BH-adjusted <em>Z</em> test) upon temperature shift (Fig. <a href="#Fig3" class="usa-link">3F</a>). Consistent with previous reports<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>, in yeast strains bearing single deletions of <em>TRM4</em> or <em>TRM8</em>, Val-AAC aminoacylation remains near constant upon shift to the non-permissive temperature (Supplementary Fig. <a href="#MOESM1" class="usa-link">6d, e</a>).</p>
<p id="Par18">We also observed statistically significant increases in aminoacylation (2.54-fold for Gly-GCC, <em>p</em>-value = 0.009; 2.73-fold for Gly-CCC, <em>p</em>-value = 0.004, BH-adjusted <em>Z</em> test) for two glycyl-tRNA isodecoders upon a shift to 37 °C (Fig. <a href="#Fig3" class="usa-link">3F, G</a>), which were eliminated by <em>MET22</em> co-deletion in our sequencing data (Fig. <a href="#Fig3" class="usa-link">3H</a>). However, validation of this result by chemical-charging northern was less conclusive. While we detected a 1.22-fold increase in Gly-GCC aminoacylation in the double mutant, aminoacylated Gly-GCC tRNA in the <em>trm8∆ trm4∆ met22∆</em> triple mutant also increased by a similar magnitude (Supplementary Fig. <a href="#MOESM1" class="usa-link">6a</a>). Two additional isodecoders, Cys-GCA and Leu-CAA, also displayed more modest (~1.1-fold increases) changes in aminoacylation upon temperature shift (Supplementary Fig. <a href="#MOESM1" class="usa-link">6b, c</a>). To investigate this result further, we again turned to single deletions of <em>TRM4</em>, <em>TRM8</em>, and <em>MET22</em>. For Gly-GCC, the mean change in aminoacylation upon temperature shift was 1.54-fold for <em>trm4∆</em>, 1.20-fold for <em>trm8∆</em>, and 1.27-fold for <em>met22∆</em>, respectively (<em>p</em>-value of 0.09 by ANOVA, Supplementary Fig. <a href="#MOESM1" class="usa-link">6f, g</a>). While not statistically significant across biological replicates, the results hint at a potential distinct role for <em>TRM4</em> in modulating Gly-GCC aminoacylation during temperature stress, in contrast to <em>TRM8</em>, which is not known to act directly on Gly-GCC<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>.</p>
<p id="Par19">RTD has also been described in <em>tan1</em>∆ <em>trm44∆</em> cells, which are temperature sensitive due to lack of acetylation at C<sub>12</sub> by <em>TAN1</em> and loss of methylation at U<sub>44</sub> by <em>TRM44</em>; these enzymes share activity on multiple serine and leucine isodecoders in budding yeast<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>,<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. As in <em>trm8∆ trm4∆</em>, temperature sensitivity is accompanied by exonuclease-mediated decay of specific tRNA isodecoders (Ser-UGA, Ser-CGA) and both the molecular and growth phenotypes are alleviated by co-deletion of <em>MET22</em> <sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>,<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. We grew <em>tan1</em>∆ <em>trm44∆</em> and <em>tan1</em>∆ <em>trm44∆ met22∆</em> cells under the same RTD-inducing conditions described above, isolated small RNA, and again performed chemical ligation and analysis by chemical-charging northern and aa-tRNA-seq. Our results indicate that the targets of RTD in <em>tan1</em>∆ <em>trm44∆</em> cells are restricted to the tRNA species originally identified; however, aminoacylation of these hypomodified tRNAs is unaffected upon temperature stress. After three hours of temperature stress, we detected modest decreases in abundance of both serine tRNA isodecoders previously reported to be affected in <em>tan1</em>∆ <em>trm44∆</em> cells via aa-tRNA-seq, with levels of Ser-CGA (the more strongly affected isodecoder in previous studies) decreasing by a statistically significant 22.2% and Ser-UGA showing a 13.8% reduction in abundance upon temperature stress (Supplementary Fig. <a href="#MOESM1" class="usa-link">7a–c</a>). When validated by chemical-charging northern, <em>tan1∆ trm44∆</em> cells displayed 6–11% drops in Ser-UGA abundance upon shift to the nonpermissive temperature (Supplementary Fig. <a href="#MOESM1" class="usa-link">8a–c</a>), while levels of the unaffected isodecoder Leu-CAA remained constant or increased.</p></section><section id="Sec10"><h3 class="pmc_sec_title">A machine learning approach to discriminate embedded amino acids in aa-tRNA-seq</h3>
<p id="Par20">Because the identity of the amino acid on an aminoacylated tRNA is invisible to other tRNA sequencing approaches, a major advance for the field would be the detection of misaminoacylation events directly from high throughput tRNA sequencing libraries. Towards this goal, we performed detailed analysis of signals produced during nanopore sequencing for aminoacyl-tRNA charged with each of the 20 proteinogenic amino acids using the Flexizyme system. While nanopore basecallers and other machine learning approaches for detection of modified bases are typically trained on ionic current signatures produced at specific residues<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>,<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>, the raw signal generated during nanopore sequencing is a composite of changes in current and translocation speed (measured in “dwell time”, the time between inferred translocation states). Because they are sequence-identical (with the exception of Cys- and Asn-tRNA libraries, which were prepared using the DNA/RNA 5′ adapter sequence tested in Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a> and used in all biological sequencing experiments), our synthetic aa-tRNA-seq libraries enable the isolation of amino acid specific signals by comparing each of the 20 aa-tRNAs to an uncharged control.</p>
<p id="Par21">Aminoacylated tRNAs yield specific distortions in nanopore signals, generating unique signatures that vary between amino acids. Figure <a href="#Fig4" class="usa-link">4A</a> displays the mean dwell time in milliseconds for each of our 20 synthetic aa-tRNA libraries and an uncharged control tRNA. We observed large increases in dwell time for charged tRNAs at a position located 9 nucleotides downstream from the amino acid (position 86), with dwell times exceeding 1 s for 9 of 20 amino acids. Because nanopore direct RNA sequencing proceeds in a 3′ to 5′ direction, we speculate that this signal represents specific interactions between amino acid and the motor protein as the 3′ adapter is transiting through the nanopore reader head. Figure <a href="#Fig4" class="usa-link">4B</a> shows the relative change in normalized current for each of the aminoacylated tRNA reads over this same window, compared to the uncharged tRNA control. Charged tRNA libraries display lower mean current values than the non-aminoacylated substrate across most of this region, suggesting that amino acids occlude ionic flow as they transit through the helicase/pore assembly. Consistent with this explanation, we observed the largest reductions in mean current at the precise site of aminoacylation for the most bulky amino acids, indicating that the largest distortions in ionic current distortions occur within the narrowest aperture of the nanopore itself.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Signal analysis and classification of amino acid identity using nanopore sequencing.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368100_41467_2025_62545_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/edee/12368100/cd490fcf7cd0/41467_2025_62545_Fig4_HTML.jpg" loading="lazy" id="d33e1169" height="821" width="799" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Mean dwell time for synthetic tRNAs charged with each of the 20 amino acids and an uncharged control, sequenced after enzymatic ligation to the 3′ adapter. Sequences are ordered top to bottom on the <em>y</em>-axis by amino acid side chain molecular weight. The plotted region spans the tRNA 3′ terminus (positions 68–73), the CCA tail (74–76), the aminoacylated position (77, dashed line) and complete 3′ adapter. The schematic below indicates the translocation direction and approximate amino acid position within the helicase motor (orange) when the 3′ adapter sequence is centered in the nanopore (blue). <strong>B</strong> Relative differences in normalized mean current between charged and uncharged synthetic tRNAs across the same region as (<strong>A</strong>). Schematic shows approximate amino acid location within the nanopore during maximal current change. <strong>C</strong> Scatter plot showing the relationship between mean dwell time at position 86 and amino acid side-chain volume (in cubic angstroms); the y-axis shows relative difference in mean current. Each point represetns one amino acid. Dashed line indicates linear regression fit; <em>r</em> and <em>p</em>-values are from a two-sided Pearson correlation. <strong>D</strong> As in (<strong>C</strong>), but with molecular weight (g/mol) on the <em>x</em>-axis. <strong>E</strong> As in (<strong>C</strong>), but with hydrophobicity index<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a></sup> at pH 7 on the <em>x</em>-axis. <strong>F</strong> Scatter plot showing the relationship between mean current differences at position 77 and amino acid volume (cubic angstroms). As in (<strong>C</strong>) for all statistical and plotting conventions. <strong>G</strong> As in (<strong>F</strong>), but with molecular weight (g/mol) on the <em>x</em>-axis. <strong>H</strong> As in (<strong>F</strong>), but with hydrophobicity index<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a></sup> at pH 7 on the <em>x</em>-axis. <strong>I</strong> Schematic of pairwise Remora model training using signal distributions from synthetic tRNAs charged with proline vs lysine. Shaded regions represent standard deviation around the mean current; dashed line indicates the amino acid position. A yellow box outlines the 4-nt signal window used for training, with the amino acid location marked by a dashed line. <strong>J</strong> F1 scores for all pairwise amino acid classification models trained on synthetic tRNA reads using the strategy in (<strong>I</strong>).</p></figcaption></figure><p id="Par22">We examined the correlation between the largest current and dwell time effects and various amino acid properties. While the putative helicase interactions at position 86 are poorly correlated with amino acid volume and molecular weight (Fig. <a href="#Fig4" class="usa-link">4C, D</a>), we identified strong correlations between dwell time and hydrophobicity (Fig. <a href="#Fig4" class="usa-link">4E</a>), as well as correlations between amino acid mass and volume, and the observed current differences at position 77 (the position of the amino acid; Fig. <a href="#Fig4" class="usa-link">4F, G</a>). Together, these observations likely reflect straightforward physical occlusion of ion flow within the nanopore reader head and more complex interactions between the tRNA-embedded amino acid and the motor protein.</p>
<p id="Par23">Closer examination of our data revealed substantial variation in the magnitude of current (Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>) and dwell time (Supplementary Fig. <a href="#MOESM1" class="usa-link">10</a>) signals produced by different synthetic aminoacyl-tRNAs. We trained 380 pairwise models to discriminate one AA from another (Fig. <a href="#Fig4" class="usa-link">4I</a>), and evaluated their performance on our synthetic data (Fig. <a href="#Fig4" class="usa-link">4J</a>). The performance of these models was generally high with a median F1 score of 0.86, indicating that differences in nanopore current between several pairs of amino acids represent a strong signal for classification. Notable underperforming outliers included isoleucyl-tRNAs, which are poorly distinguished from other hydrophobic side chains, and other specific comparisons (e.g., Phe-Tyr, Pro-Ala, Cys-Arg). We are currently exploring the improvement of these models by explicit inclusion of dwell time information during model training, and plan to apply these models to understand specific cases of potential tRNA misaminoacylation (Fig. <a href="#Fig3" class="usa-link">3D, G</a>).</p></section></section><section id="Sec11"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par24">Transfer RNAs are long-lived and undergo repeated cycles of charging and deacylation throughout their lifetime. While tRNA biogenesis is recognized as an intricately coordinated process—spanning transcription, processing, modification, and aminoacylation—less is known about the molecular transformations that tRNAs undergo throughout their functional lifetime and the specific events that trigger their turnover. We developed a chemical ligation approach to selectively capture aminoacylated tRNAs and used it to study tRNA aminoacylation in a variety of contexts. Increased accuracy of aa-tRNA identification may further enhance our understanding of how tRNA modifications act as determinants or anti-determinants of synthetase recognition and specificity. Moreover, by enabling simultaneous screening of tRNA modifications, charging, and stability, this approach provides a versatile framework for engineering synthetic or therapeutic tRNAs with precisely tuned properties<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>–<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. Direct identification of amino acids and their corresponding tRNA sequences could also advance studies on aminoacyl-tRNA synthetase evolution and engineering, bypassing the indirect readouts and negative selections commonly employed by current synthetase engineering approaches<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>,<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>.</p>
<p id="Par25">We leveraged chemical ligation to simplify the analysis of aminoacylated tRNAs via northern blotting (Fig. <a href="#Fig1" class="usa-link">1C</a>). This approach offers key advantages, including simplified electrophoretic separation of chemically ligated species, and is therefore a compelling alternative to more laborious techniques (but see refs. <sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>–<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>). However, it is important to note that under the existing reaction conditions, there is some variability in ligation efficiency among biological tRNA isoacceptors and only some were ligated quantitatively (Supplementary Fig. <a href="#MOESM1" class="usa-link">1b</a>), limiting the performance of this method compared to a traditional acidic northern blot. It is also noteworthy that chemical ligation is likely useful beyond the capture of aminoacylated tRNA: a variety of nucleophiles react with phosphorimidazole-activated oligonucleotides, including RNA terminal hydroxyl groups at elevated pH<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. Juxtaposition of the phosphorimidazolide near the ɑ-amino group of the aa-tRNA was facilitated by splinted base-pairing to the 3′-CCA overhang (Fig. <a href="#Fig1" class="usa-link">1A</a>), but one could imagine other sequence-specific targeting scenarios, including the 3′-ends of mRNA poly-A tails.</p>
<p id="Par26">Next, we adapted chemical ligation to enable direct nanopore sequencing of aminoacylated tRNA ligated to a downstream oligonucleotide (“aa-tRNA-seq”). We trained a Remora classification model to distinguish charged and uncharged tRNAs from nanopore sequencing data, and showed that this signal-based classification approach outperformed an alignment-based one (Fig. <a href="#Fig2" class="usa-link">2</a>). Our application of aa-tRNA-seq confirmed known effects of hypomodification and nutrient deprivation on tRNA stability and aminoacylation (Fig. <a href="#Fig3" class="usa-link">3</a>), revealing that despite the broad specificity of <em>TRM8</em> and <em>TRM4</em>, their co-deletion uniquely impacts the stability of Val-AAC (Fig. <a href="#Fig3" class="usa-link">3F</a>). Similarly, the combined impact of <em>TAN1</em> and <em>TRM44 deletion</em> is largely restricted to Ser-CGA and Ser-UGA stability (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>). We also found significantly increased aminoacylation for two isoacceptor families in cases of hypomodification (Gly-GCC and Gly-CCC, Fig. <a href="#Fig3" class="usa-link">3G</a>) and nutrient deprivation (Ala-TGC, Fig. <a href="#Fig3" class="usa-link">3D</a>). These may simply be increases in cognate aminoacylation of these isodecoders that were not previously observed. Alternatively, they may represent new cases wherein a lack of modification or charging enables misaminoacylation of Gly and Ala isodecoders. Methionine misaminoacylation is a common response to stress<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup> wherein the methionyl tRNA synthetase non-specifically charges several tRNA isodecoders with methionine. However, the pattern of misaminoacylation we observe is restricted to a few tRNAs, suggesting an alternative mechanism.</p>
<p id="Par27">Like other tRNA sequencing methods, aa-tRNA-seq has tradeoffs and technical biases. First, we identified intrinsic differences in the total translocation time between charged and uncharged tRNA during nanopore sequencing (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>), which is a barrier to absolute quantitation of tRNA charging. Second, under these reaction conditions, not all biological tRNA isoacceptors are quantitatively ligated (Supplementary Fig. <a href="#MOESM1" class="usa-link">1b</a>), reflecting an additional source of bias that may be further optimized. Ligation inefficiency reflects a broader challenge: enzymatic adapter ligation bias has complicated other tRNA sequencing methods, often making comparisons of tRNA abundances within the same sample unreliable<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>,<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. While methods relying on periodate oxidation to distinguish charged from uncharged tRNAs have proved valuable to the field, they also face technical challenges, including tRNA damage during periodate treatment<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>, variability in the sensitivity of tRNA 3′ termini to oxidation, and inconsistency in the number of nucleotides removed during treatment<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. Additionally, as periodate-based methods require cDNA generation, they are subject to biases from tRNA modifications that inhibit RT processivity. In protocols where these modifications are instead removed, such methods can introduce additional RNA damage from harsh buffer conditions during enzymatic pretreatment<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>–<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>. To date, we have not tested cDNA conversion in aa-tRNA-seq libraries; while inclusion of a reverse transcription step in nanopore direct RNA sequencing libraries is known to increase sequencing throughput, this addition could introduce additional biases due to non-uniform RT processivity over different amino acids.</p>
<p id="Par28">Chemically ligated aa-tRNAs generate discrete signals during nanopore sequencing due to interactions between the embedded amino acid and the motor protein and nanopore architecture (Fig. <a href="#Fig4" class="usa-link">4</a>). This unique feature enabled accurate identification of bona fide aa-tRNAs while minimizing false positives: we classified 3.6% of reads from the “charged-only” validation library as uncharged. However, some of these may be true non-acylated tRNA that were enzymatically ligated to hydrolyzed phosphorimidazole-adapter, which has a 5′-phosphate competent for T4 RNL2 ligation. While our current approach demonstrates that different amino acids produce distinct signal properties that enable their discrimination in pairwise comparisons (Fig. <a href="#Fig4" class="usa-link">4J</a>), further advances in our machine learning approach may enable the direct, de novo identification of amino acids in biological samples, unlocking new questions about tRNA charging and misaminoacylation.</p>
<p id="Par29">We anticipate that incorporation of translocation time information into models for classifying aminoacylated tRNAs will generate additional improvements to aa-tRNA-seq. Dwell time provided useful information in the detection of RNA modifications using the previous direct RNA (RNA002) ONT chemistry, including pseudouridine<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup>, 2′-O-methyl<sup><a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup>, and 2′-phosphate<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> modifications, but existing approaches for training models on nanopore signal do not leverage this information directly, due to the fact that Remora and other software re-anchor ionic current information onto an aligned sequence. While dwell time information is retained in this process, it is transformed and thereby de-emphasized in model training. Notably, the increased dwell times observed for aminoacylated tRNAs—likely due to unique helicase interactions—are a robust signal in our data. However, these effects are less strongly correlated with amino acid properties than the current differences at the aminoacylated position (Fig. <a href="#Fig4" class="usa-link">4C–H</a>), where we have focused our model training.</p>
<p id="Par30">The translocation rate for aa-tRNAs also impacts the pore blocking effects we described for Cys- and Asn-tRNA (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>), as read ejection (“unblocking”) is initiated during nanopore sequencing when a constant signal (indicating a stalled molecule) exceeds a set time threshold. While we resolved pore blocking issues via optimization of the 5′ adapter sequence, we do not fully understand why chemical ligation of these substrates produced these artifacts, or why they were resolved by the substitution of deoxyribonucleotides in our 5′ RNA/DNA splint adapter. While Asn-tRNA yielded significant pore blocking, Gln-tRNA did not (Supplementary Fig. <a href="#MOESM1" class="usa-link">3a</a>), suggesting an issue beyond simply the presence of an amide side chain. We note that the Asn side is uniquely capable of cyclization rearrangements during intein catalysis, which may contribute to its unique pore blocking phenotype<sup><a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup>.</p>
<p id="Par31">Transfer RNA modifications are installed in evolutionarily conserved but incompletely understood circuits<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>,<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>. Described links between tRNA modifications and aminoacylation<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup> remain sparse and poorly characterized, due in part to the lack of incisive and accessible tools to study these relationships. Comprehensive identification and characterization of circuits linking modification and aminoacylation along with tRNA abundance will require further optimization of aa-tRNA-seq and ongoing work to map the 67+ unique RNA modifications present in the tRNA epitranscriptome<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>, which will likely require a combination of nanopore sequencing, mass spectrometry<sup><a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>,<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a></sup>, and other technologies, with the aim of characterizing the complete collection of tRNA modifications (using lower throughput approaches) and understanding the signals they produce during high throughput, direct RNA sequencing experiments<sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup>.</p></section><section id="Sec12"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec13"><h3 class="pmc_sec_title">Preparation and chemical ligation of synthetic aminoacylated tRNAs</h3>
<section id="Sec14"><h4 class="pmc_sec_title">Synthesis</h4>
<p id="Par32">Oligonucleotides listed in Supplementary Table <a href="#MOESM1" class="usa-link">2</a> were either purchased from IDT or synthesized on the K&amp;A H-6 RNA/DNA synthesizer. Phosphoramidite coupling times and the remaining synthesis method parameters were as instructed by the manufacturer (ChemGenes and Glen Research). After solid-phase synthesis, oligonucleotides were cleaved and the nucleobases deprotected as recommended by ChemGenes and Glen Research. The cleaved and deprotected solutions were evaporated using a speed-vac for 2 h followed by overnight lyophilization. The dry material was dissolved in 100 µL DMSO to which 125 µL of TEA. 3HF was added followed by incubation at 65 °C for 2.5 h. The fully deprotected oligonucleotides were precipitated with 0.1 volumes of 5 M ammonium acetate and 5 volumes of cold isopropanol. The precipitated material was dissolved in 5 mM EDTA, 99% v/v formamide and purified by denaturing PAGE. The desired gel bands were visualized by UV shadowing, cut out, crushed, and soaked in 2 mM EDTA, 5 mM sodium acetate on a rotator overnight. The rotated solutions were filtered through a 5 µm syringe filter after which the filtered solutions were concentrated using Amicon MWCO filters. The concentrated solutions were finally precipitated using 0.1 volumes of 3 M sodium acetate and 5 volumes of ethanol, washed twice with 80% v/v ethanol, and air dried.</p>
<p id="Par33">The 3,5-dinitrobenzyl esters of amino acids (DBE-aas) were synthesized as described in ref. <sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup> with the following modifications:</p>
<ol class="list" style="list-style-type:decimal">
<li><p id="Par34">Boc protecting groups were removed by dissolving the dry crude DBE-Boc-aa material in 2 mL neat TFA and incubating it at room temperature for 10 min. The TFA was removed under a stream of nitrogen and the deprotected product was washed twice with diethyl ether. The diethyl ether was removed under vacuum and the final DBE-aa product was dissolved in DMSO and used in the aminoacylation assays.</p></li>
<li><p id="Par35">Boc-Ser, Boc-Thr, and Boc-Tyr (ChemImpex) were purchased with O-<em>tert</em>-butyl protection on the side chain. The deprotection was performed in 90:10 TFA:triethylsilane for 2 h.</p></li>
<li><p id="Par36">Boc-Met (ChemImpex) was used without additional side chain protection, but during the TFA deprotection two side products were observed: oxidation to produce a DBE-Met dimer and <em>tert</em>-butylation of the sulfur. This necessitated reversed-phase purification. RediSep Gold® C18 Reversed Phase Column was used with the 5–90% gradient of solvent B (solvent A = 2 mM TEAB pH 8; solvent B = acetonitrile).</p></li>
<li><p id="Par37">Boc-Gln and Boc-Asn were purchased with Xan protection on the side chain amide (ChemImpex). DBE-Asn required reversed-phase purification. RediSep Gold® C18 Reversed Phase Column was used with the 5–90% gradient of solvent B (solvent A = 2 mM TEAB pH 8; solvent B = acetonitrile). DBE-Asn additionally requires immediate use after purification due to the presumed rapid intramolecular attack of the side chain amide onto the activated ester.</p></li>
<li><p id="Par38">Boc-Cys was purchased with Trt protection on the side chain thiol (ChemImpex). The deprotection was performed in 90:10 TFA:triethylsilane for 2 h.</p></li>
</ol></section><section id="Sec15"><h4 class="pmc_sec_title">Activation</h4>
<p id="Par39">The 5′-phosphorimidazolide adapter was generated by incubating a solution containing 200 µM of the 5′-phosphorylated adapter, 100 mM imidazole pH 7, and 100 mM EDC.HCl for 2 h at room temperature. The activated adapter was then precipitated by adding 0.1 volumes of saturated sodium perchlorate in acetone and 3 volumes of cold acetone. The pellet was washed twice with a 1:1 v/v solution of acetone:diethyl ether followed by drying under vacuum. The activated adapter was dissolved in 1 mM imidazole pH 8 and stored at −80 °C until use. The same stock of the activated adapter was used throughout the experiment, but care was taken to thaw the stock immediately prior to the experiment, to store it on ice while using it, and to return it to −80 °C as quickly as possible.</p></section><section id="Sec16"><h4 class="pmc_sec_title">Aminoacylation</h4>
<p id="Par40">The aminoacylation reactions containing 100 mM HEPES pH 8, 10 mM MgCl<sub>2</sub>, 40 µM synthetic tRNA, 36.7 µM dFx Flexizyme, 5 mM DBE-aa (20% v/v DMSO), were incubated on ice for 16 h.</p></section><section id="Sec17"><h4 class="pmc_sec_title">Chemical ligation</h4>
<p id="Par41">The ligation reactions were set up by diluting the aminoacylation reactions eightfold so that the final solution contained 5 µM of the aminoacylated tRNA, 50 µM 5′-phosphorimidazolide adapter, 50 µM splint (also called the 5′-adapter below), 5 mM EDTA, and 37.5 mM HEPES pH 8. The reactions were allowed to proceed for 24 h on ice, before being diluted with an equal volume of a solution of 5 mM EDTA, 99% v/v formamide and purified by 16% denaturing PAGE. The ligated products were cut out from the gel, crushed, and soaked in a solution of 2 mM EDTA, 5 mM sodium acetate acidified to pH 5 on a rotator for 3 h at 4 °C. The extracted aa-bridged tRNA products were then filtered using 0.22 µm spin filters, concentrated using Amicon 10 k MWCO filters, and desalted using the Oligo Clean and Concentrator kit (Zymo Research).</p></section><section id="Sec18"><h4 class="pmc_sec_title">Cys-bridged tRNA alkylation</h4>
<p id="Par42">After chemical ligation and gel purification, the Cys-bridged tRNA was reduced with DTT for 1 h at room temperature. The reaction contained 1.2 µM Cys-bridged tRNA, 50 mM HEPES pH 8, and 10 mM DTT. After the 1 h incubation, the reduction reaction was diluted 1.33-fold so that the final alkylation solution contained 0.9 µM Cys-bridged tRNA, 37.5 mM HEPES pH 8, 7.5 mM DTT, and 50 mM chloroacetamide. The alkylation reaction was allowed to proceed for 30 min in the dark, after which it was cleaned up using the RNAClean XP beads (Beckman Coulter) according to the manufacturer protocol with the following change: immediately after the addition of the bead suspension to the ligation reaction, isopropanol equal to volume of the reaction+beads was added.</p></section></section><section id="Sec19"><h3 class="pmc_sec_title">Nanopore library preparation and sequencing of synthetic aminoacylated tRNAs</h3>
<p id="Par43">The chemically ligated tRNA products from above were enzymatically ligated to the 5′-adapter/splint for 30 min at room temperature. The ligation reactions contained 16 pmol of the chemically ligated tRNA, 80 pmol of the 5′-adapter, 1× NEB T4 RNA Ligase 2 buffer supplemented with 5% PEG 8000, 2 mM ATP, 6.25 mM DTT, 6.25 mM MgCl2, and 0.5 units/μL T4 RNA ligase 2 (10,000 units/mL). The ligated material was purified using the RNAClean XP beads (Beckman Coulter) as above. This material was then prepared for nanopore direct RNA sequencing via RTA ligation, which was performed using tRNA purification specific magnetic beads (BioDynami Cat.# 40054S). The remaining library prep and nanopore sequencing was performed as described below on P2solo sequencing instruments, using MinKNOW version 23.11.7.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Synthetic tRNA mini-substrate experiments</h3>
<p id="Par44">The acceptor stem mimic oligonucleotide (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>) was aminoacylated with all 20 amino acids as described above. At the end of the 16 h incubation:</p>
<ol class="list" style="list-style-type:decimal">
<li><p id="Par45">1 µL aliquots were diluted in 9 µL of acidic quenching buffer (10 mM EDTA pH 8.0, 1× bromophenol blue, 100 mM sodium acetate pH 5.0, 150 mM HCl, 75% v/v formamide) and analyzed by 20% acidic denaturing PAGE (acidic gels contained 100 mM sodium acetate pH 5.0 instead of the usual 1× Tris-Borate-EDTA). The acidic gels were run in 100 mM sodium acetate pH 5 at 25 W for 3 h at 4 °C. Aminoacylation percentage was obtained by quantifying the per-lane normalized band intensity in the ImageQuant TL software. The amino acids that displayed sufficient aminoacylated versus non-aminoacylated gel band resolution were subjected to the next step.</p></li>
<li><p id="Par46">The remaining aminoacylation reaction was immediately diluted tenfold in the chemical ligation buffer in three separate replicates. The ligation reaction contained 1 µM of the aminoacylated RNA, 4 µM of the 5′-phosphorimidazolide activated hairpin adapter (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>), 10 µM of the acceptor stem mimic complement (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>), 200 mM HEPES pH 6.5, 5 mM MgCl<sub>2</sub>, and 100 mM of HEI pH 6.5. After 90 min at room temperature, 1 µL aliquots were diluted in 9 µL of acidic quenching buffer, and analyzed by standard 20% denaturing urea-PAGE. The efficiency of the ligation reaction was obtained by quantifying the per-lane normalized band intensity in the ImageQuant TL software. The normalized ligation efficiency was obtained by dividing the fraction ligated by the fraction aminoacylated and multiplying by 100%.</p></li>
</ol></section><section id="Sec21"><h3 class="pmc_sec_title">Yeast strains and growth conditions</h3>
<p id="Par47">Yeast strains used in this study are listed in Supplementary Table <a href="#MOESM1" class="usa-link">1</a>. For chemical ligation validation by acid northern (Fig. <a href="#Fig1" class="usa-link">1B</a>), a single colony of S288C was inoculated into YEP glucose (yeast extract, peptone, 2% glucose) and incubated at 30 °C overnight with rotation before dilution to an OD<sub>660</sub> of 0.2 in YEPD media. The culture was grown to log phase shaking at 30 °C before a pellet was collected, flash frozen in liquid nitrogen, and stored at −80 °C.</p>
<p id="Par48">For validation of chemical charging northerns (Fig. <a href="#Fig1" class="usa-link">1C</a>), a single colony of WY798 was inoculated into synthetic complete media and incubated at 30 °C overnight with rotation. This culture was diluted to 50 mL of synthetic complete media the next day and allowed to shake overnight at 30 °C before dilution to an OD<sub>660</sub> of 0.2 in 100 mL synthetic complete media. The culture was grown to log phase at 30 °C, collected by centrifugation, and resuspended in 100 mL room temperature synthetic complete media. The culture was allowed to grow in the new media for 15 min, shaking at 30 °C before it was pelleted, washed with water, flash frozen in liquid nitrogen, and stored at −80 °C.</p>
<p id="Par49">For the nutrient stress experiment (Fig. <a href="#Fig3" class="usa-link">3B–D</a>), 3 colonies of WY795 were inoculated into synthetic complete media and incubated at 30 °C overnight with rotation before dilution to an OD<sub>660</sub> of 0.2 in 50 mL the same media. The cultures were grown to log phase shaking at 30 °C before pellets were collected by centrifugation. Each pellet was washed by resuspension in a small volume of synthetic media lacking uracil, tryptophan, histidine, and leucine and split into two tubes. Cells were pelleted again and the wash media was removed. One pellet from each original culture was resuspended in 25 mL 30 °C synthetic complete media and the other in 25 mL 30 °C synthetic media lacking leucine. They were allowed to grow in the new media for 15 min, shaking at 30 °C, before they were pelleted, washed with water, flash frozen in liquid nitrogen, and stored at −80 °C.</p>
<p id="Par50">For the temperature stress experiment (Fig. <a href="#Fig3" class="usa-link">3E–H</a>), 3 colonies of JMW 009 and 3 colonies of JMW 510 were inoculated into YEPD media and incubated at 30 °C overnight with rotation before dilution to an OD<sub>660</sub> of 0.2 in 60 mL YEPD media. The cultures were grown to log phase shaking at 30 °C. At this point, 50 mL of culture was pelleted by centrifugation, washed with water, flash frozen in liquid nitrogen, and stored at −80 °C. 40 mL of 40 °C YP glucose media was added to the remaining 10 mL of culture and these cells were grown for 3 h at 37 °C. They were then pelleted, washed with water, flash frozen in liquid nitrogen, and stored at −80 °C.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Isolation and chemical ligation of aminoacylated tRNAs from budding yeast</h3>
<section id="Sec23"><h4 class="pmc_sec_title">A complete protocol for aa-tRNA-seq is available on <a href="https://benchling.com/protocols/1vXce4Gw/acylated-deacylated-trna-library-preparation-for-rna004-sequencing-final" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Benchling</a>
</h4>
<p id="Par51">Yeast pellets were thawed on ice and resuspended in 400 µL of cold AES (10 mM NaOAc pH 4.5, 1 mM ETDA pH 8, 0.5% SDS). 400 µL of cold 25:24:1 acid phenol:chloroform:isoamyl alcohol was added. Samples were vortexed for 15 s and allowed to rest on ice for 20 min, vortexing every 5 min. They were then spun at 18,000 × <em>g</em> for 10 min at 40 °C and the aqueous phase was moved to a new tube.</p>
<p id="Par52">A 0.4× volume of Ampure XP beads (Fisher Scientific A63881) were added to 100 µL of aqueous phase. They were rotated for 2 min at RT and placed on a magnet until the beads had settled. The supernatant was moved to a new tube and quantified via nanodrop. Small RNAs were isolated from 100 µg of this supernatant using a Zymo Research RNA Clean and Concentrator kit (R1018) according to the manufacturer’s instructions. Dilution of the bead supernatant for the first step of the kit was done with 10 mM NaOAc pH 4.5, not with water. Small RNA was eluted in 30 µL of 10 mM NaOAc pH 4.5 and quantified via nanodrop. It was stored at −80 °C.</p>
<p id="Par53">Two 3′ DNA-RNA hybrid splint adapters were designed with different internal sequences, one to ligate to deacylated tRNAs (“uncharged 3′ adapter”) and the other to acylated tRNAs (“charged 3′ adapter”, see Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). A universal 5′ adapter was designed to pair with either of the 3′ adapters. Syntheses of these adapters were ordered from IDT and resuspended in water to a concentration of 2 mM. Adapters were run on a 1.5 mM 6% TBU (Tris, boric acid EDTA) V16 polyacrylamide gel with 10 nmol loaded per lane (10-well comb). Staining was not performed and UV shadowing was used to excise the adapters. Gel slices were cut into small pieces and rotated end over end in crush + soak buffer (300 mM NaOAc pH 5.5, 1 mM EDTA pH 8.0, 0.1% SDS) overnight at 4 °C. Nucleic acids were precipitated with 100% ethanol and resuspended in water to a final concentration of 100 µM.</p>
<p id="Par54">Gel-purified charged 3′ adapter was incubated with a 500-fold molar excess of both imidazole and EDC (1-Ethyl-3-(3-dimethylaminopropyl)carbodiimide) for 2 h at 25 °C to imidazolate the 5′ end of the adapter. 30 µL of cold 99.5%+ acetone saturated with perchlorate and 1 mL of cold 99.5%+ acetone were added to precipitate the imidazolated adapter. The sample was incubated for 20 min on dry ice and then centrifuged at maximum speed for 10 min at 4 °C. The supernatant was removed and the pellet was washed twice with 1 mL 1:1 acetone: diethyl ether. The pellet was dried in a speed vacuum and resuspended in 10 mM imidazole pH 7.0 to a final concentration of 200 µM.</p>
<p id="Par55">Small RNA (15-50 pmol) was incubated in 100 mM MES pH 5.5, 2.5 mM MgCl<sub>2</sub>, a fivefold molar excess of both imidazolated 3′ adapter and gel-purified 5′ adapter, and 50 mM HEI pH 6.5 for 30 min at 25 °C, establishing a phosphoramidate covalent linkage between the 3′ splint adapter and aminoacylated tRNAs. Ligated products were purified by crush and soak (0.3 M NaOAc pH 5.5, 1 mM EDTA pH 8.0, 0.1% SDS) at 4 °C, overnight) from a 10% TBU polyacrylamide gel, isolating the regions between 70 and 150 nts. The eluate was precipitated by addition of ethanol and GlycoBlue coprecipitant (Invitrogen) resuspended in a small volume of 10 mM NaOAc pH 4.5 and quantified via absorbance at 260 nm (Nanodrop).</p></section></section><section id="Sec24"><h3 class="pmc_sec_title">Northern blotting</h3>
<p id="Par56">Transfer RNA charging was measured by acidic northern blot, resolving 75 ng of small RNA on a gel (6% 19:1 acrylamide, 0.1 M sodium acetate, pH 4.5, 8 M urea) 42 cm in length which was run at 450 V for 22 h in a cold room. For chemical-charging northern blots, chemically ligated tRNA (220 ng) was loaded onto 10% TBU polyacrylamide gels (7.5 cm, 6% 19:1 acrylamide, 1× TBE, 8 M urea) and electrophoresed in 1× TBE at room temperature at 250 V for 40 min. Acid-urea and TBU gels were transferred to charged nylon membranes (Hybond N+, GE) via electroblot transfer at 1 Amp for 1 h for acid gels and 3 mA/cm<sup>2</sup> based on the membrane area for 35 min for TBU gels. After transfer, membranes were UV-crosslinked at 254 nm using a 120 mJ dose and blocked in ULTRAhyb-Oligo (Thermo) before an incubation with <sup>32</sup>P-labeled oligonucleotide probes in ULTRAhyb-Oligo overnight at 42 °C (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). Membranes were washed four times at 42 °C (2× SSC, 0.1% SDS), wrapped in plastic, and exposed to a phosphor-imager screen before imaging on a Typhoon 9400 (GE Healthcare). Membranes were stripped with two 30 min washes in 2% SDS at 80 °C, prior to reblocking and incubation with labeled probe.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Nanopore library generation for budding yeast tRNAs</h3>
<p id="Par57">Gel-purified tRNAs from chemical ligation were enzymatically ligated to capture deacylated tRNAs with 3′ splint adapters and attach 5′ adapters to all tRNA. tRNA from the first ligation (20 pmol) was incubated in a 20 µL reaction consisting of 10% PEG 8000, 1 µL of RNase inhibitor (Watchmaker Genomics), 9 pmol gel-purified uncharged 3′ splint adapter, 9 pmol gel-purified 5′ adapter, 1× T4 RNA ligase 2 buffer, and 2 µL of T4 RNA ligase 2 (homemade preparation, 0.74 mg/mL). This ligation was incubated at 25 °C for 30 min.</p>
<p id="Par58">Ligation products were purified by addition of a 1.8× volume of tRNA beads (BioDynami), mixing by pipetting, and incubation on ice for 4 min, followed by magnetic separation. The supernatant was discarded. Beads were washed with 180 µL 80% EtOH and air dried. Beads were resuspended in 13 µL of water, and the elution was moved to a new tube and quantified.</p>
<p id="Par59">Splint-adapter-ligated tRNAs are next ligated to RT adapters (RTA) (provided in the RNA004 ONT kit): 12.5 µL of sample was incubated with 1.5 µL RTA, 0.5 µL RNase inhibitor (Watchmaker Genomics), 4 µL T4 DNA ligase buffer, and 1.5 µL T4 DNA ligase (Watchmaker Genomics, 600 U/µL) for 30 min at 25 °C, and cleaned up at RT using the tRNA beads as above, using a 1.35× volume of beads, and elution in 26 µL water. Each sample was quantified with the Quant-iT Qubit dsDNA HS kit and the library size distribution was confirmed by Agilent TapeStation (HS DNA 1000).</p>
<p id="Par60">Finally, ligation products were ligated to ONT’s RNA ligation adapter (RLA) on the same day that sequencing was conducted. 50–400 fmol of sample in 23 µL was incubated with 6 µL RLA (ONT RNA004), 8 µL T4 DNA ligase buffer, and 3 µL T4 DNA ligase (Watchmaker Genomics, 600 U/µL) for 30 min at 25 °C. These final ligations were cleaned up using a 1.8× volume of Ampure XP SPRI beads (Beckman Coulter), and washed with WSB wash buffer (ONT) following the protocol for ONT SQK-RNA004.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Sequencing run conditions for budding yeast tRNAs</h3>
<p id="Par61">Libraries were loaded onto “RNA” flow cells on a PromethION P2 Solo instrument connected to a A5000 GPU workstation or a PromethION P2integrated instrument, using MinKNOW software version 24.06.10. We found the throughput of aa-tRNA-seq to be comparable or superior to previous nanopore tRNA sequencing approaches<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup>, collecting a median &gt;8 million reads for biological tRNA sequencing libraries, and median ~250 thousand reads for synthetic tRNA sequencing libraries.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Base-calling and alignment</h3>
<p id="Par62">Libraries were basecalled with Dorado v0.7.2 (ONT, <a href="https://github.com/nanoporetech/dorado" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/nanoporetech/dorado</a>) using the “super high accuracy” (rna004_130bps_sup) v5.0.0 model and –emit-moves parameter. Basecalled bams were converted to fastq format using samtools v. 1.21<sup><a href="#CR87" class="usa-link" aria-describedby="CR87">87</a></sup> with -T “*” flag to retain move tables, and then aligned to using BWA-MEM version 0.7.16-r1181<sup><a href="#CR88" class="usa-link" aria-describedby="CR88">88</a></sup> with the parameters bwa mem –C –W 13 –k 6 –x ont2d, enabling transfer of move tables to aligned bams.</p>
<p id="Par63">To evaluate the performance of an alignment-based classification approach for identifying charged vs. uncharged tRNA reads, aligned bams were further filtered to contain reads mapped to full length tRNAs. Aside from this filtering, no data were excluded from the analysis. tRNA reference files were constructed by appending CCA sequences to each mature tRNA sequence, along with the unique 3′ and universal 5′ adapter sequences, and primary alignments for each read assessed. A Snakemake<sup><a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup> analysis pipeline is available at <a href="https://github.com/rnabioco/aa-tRNA-seq-pipeline" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/rnabioco/aa-tRNA-seq-pipeline</a>.</p></section><section id="Sec28"><h3 class="pmc_sec_title">Remora model training and validation</h3>
<p id="Par64">To train a machine learning model distinguishing charged and uncharged tRNAs we used the Remora software (v3.2, <a href="https://github.com/nanoporetech/remora" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/nanoporetech/remora</a>) and training procedure for modified nucleosides. First, we prepared fully charged yeast tRNA libraries (treated as modified base in the training) and deacetylated libraries (treated as modified base control). To make our model universal for all tRNAs we defined a 6-nt modification kmer spanning the universal CCA 3′ end of tRNA and the first three nucleotides of the 3′ adapter (CCA<span class="text-underline">G</span>GC), where the underlined G was defined as the modification site. We extract chunks from both libraries using remora dataset prepare with the default remora 9mer table for rescaling and following parameters: –refine-rough-rescale –reverse-signal –motif CCAGGC 3. We prepared training configuration files with Remora dataset make_config using –dataset-weights 1 1. Finally, we trained the model using the parameters: –model ConvLSTM_w_ref.py –chunk-context 200 200 –num-test-chunks 20000. After internal remora validation on 20000 chunks, this model was validated on independent libraries using reference-anchored remora inference (remora infer from_pod5_and_bam –reference-anchored). We used ML tags from inference output bams to calculate model performance, with the following assumption: ML &lt; 200 = uncharged tRNA, ML ≥ 200 = charged tRNA.</p>
<p id="Par65">Pairwise machine learning classifiers to distinguish individual amino acids (binary recognition between pairs of amino acids totaling 380 models) were trained using Remora v. 3.2 using a procedure similar to that described above, using Flexizyme-charged synthetic tRNA reads for the training. Synthetic tRNA was aligned to a reference sequence with one nucleotide (“T”) inserted between the CCA sequence at the 3′ terminus and the start of the 3′ adapter sequence. Pairwise models were trained with one amino acid (i) treated as modified base and the second one (j) as modified base control on the CCAT motif, with the inserted T identified as the modification position for Remora training; the mean F1 score for AA(i,j) and AA(j,i) are displayed in the performance data in Fig. <a href="#Fig4" class="usa-link">4J</a>. 10000 chunks were used for internal Remora validation, except for pairwise comparisons with alanine, where 9000 chunks were used due to lower library depth for the alanine-charged library. F1 scores were calculated for all models using the equation: 2TP/(2TP + FP + FN).</p></section><section id="Sec29"><h3 class="pmc_sec_title">Nanopore signal analysis</h3>
<p id="Par66">Signal metrics (dwell time and trimmed means of ionic current) at reference-anchored positions were extracted from POD5 files using the Remora API and stored in TSV files for analysis and plotting in R. Example scripts for signal extraction and plotting will be available at <a href="https://github.com/rnabioco/aa-tRNA-seq" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/rnabioco/aa-tRNA-seq</a>.</p></section><section id="Sec30"><h3 class="pmc_sec_title">Statistics &amp; reproducibility</h3>
<p id="Par67">Sample sizes for synthetic tRNA ligation and nanopore sequencing were selected based on established practices in RNA biochemistry and pilot experiments that demonstrated clear signal resolution. For biological tRNA libraries, sample sizes were guided by the practical throughput of Oxford Nanopore direct RNA sequencing, which improved over the course of the study but remained a limiting factor due to singleplex flow cell use. For nutrient limitation and temperature stress experiments, three biological replicates were used to capture biological variability and support reproducibility of observed effects. These sample sizes are consistent with prior work using northern blotting and nanopore sequencing to assess tRNA charging and abundance.</p>
<p id="Par68">Experiments were not randomized, and investigators were not blinded to sample allocation or outcome assessment.</p>
<p id="Par69">Statistical comparisons of tRNA abundance and aminoacylation levels were performed using two-sided <em>Z</em> tests on normalized read counts, with Benjamini–Hochberg correction applied for multiple hypothesis testing. Changes in gel band intensity were quantified using ImageQuant TL software, and values were normalized to total lane signal. For classification model performance (e.g., Remora models), F1 scores were computed as 2TP/(2TP + FP + FN) using held-out test sets. All statistical analyses were conducted in Python or R.</p></section><section id="Sec31"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par70">Further information on research design is available in the <a href="#MOESM2" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec32"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12368100/bin/41467_2025_62545_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (15.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12368100/bin/41467_2025_62545_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (89.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12368100/bin/41467_2025_62545_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (211.8KB, pdf) </sup>
</div></div></section></section><section id="Sec33"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12368100/bin/41467_2025_62545_MOESM4_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (163.5MB, zip) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was supported by the National Institutes of Health (R35 GM119550 to J.H.) and National Science Foundation (Award #2330283 to J.H.). J.W.S. is an Investigator of the HHMI. We thank Filip Bošković for inspiring discussions during the project planning phase, Ron Wek and Eric Phizicky for yeast strains, Marcus Stoiber for advice and assistance with Remora model training, and members of the Hesselberth and Szostak labs for constructive feedback.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>L.W., A.R., J.S., and J.H. designed and conceived the experiments. L.W., A.R., M.S., K.D., K.R., and S.P. performed experiments. L.W., A.R., J.S., and J.H. wrote the paper.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par71"><em>Nature Communications</em> thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Raw nanopore sequencing data generated in this study has been deposited as POD5 files in the European Nucleotide Archive under accession number <a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB90828" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJEB90828</a>. Specific links to access the ENA POD5 files are available at <a href="https://github.com/rnabioco/aa-tRNA-seq/blob/main/README.md" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/rnabioco/aa-tRNA-seq/blob/main/README.md</a>. Per-read current and dwell time metrics for synthetic tRNA libraries are available on Zenodo under 10.5281/zenodo.14194755 [10.5281/zenodo.14194755]. Source data supporting the findings of this study are provided with this paper. <a href="#Sec33" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>All code used for analysis and figure generation, as well as a reproducible pipeline for aa-tRNA-seq data processing, is available at the associated GitHub repository: <a href="https://github.com/rnabioco/aa-tRNA-seq" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/rnabioco/aa-tRNA-seq</a>. A permanent archive of the version used in this study is available on Zenodo under 10.5281/zenodo.15653411 [10.5281/zenodo.15653411]<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a></sup>.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par72">A.R., L.K.W., J.W.S., and J.R.H. have submitted a patent application to the USPTO (PCT/US2025/031145) pertaining to the chemistry, sequencing method, and machine learning aspects of this work. L.K.W. has received travel and accommodation expenses from Oxford Nanopore Technologies to present at scientific meetings, and has participated in beta tests of ONT’s direct RNA sequencing products. L.K.W. is a co-founder of an early-stage entity that may seek to license and commercialize aspects of the technology described in this work. The remaining authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Laura K. White, Aleksandar Radakovic.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Laura K. White, Email: laura.k.white@cuanschutz.edu</p>
<p>Jay R. Hesselberth, Email: jay.hesselberth@cuanschutz.edu</p></section><section id="sec34"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62545-9.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Crick, F. H. The origin of the genetic code. <em>J. Mol. Biol.</em><strong>38</strong>, 367–379 (1968).
</cite> [<a href="https://doi.org/10.1016/0022-2836(68)90392-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4887876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crick,%20F.%20H.%20The%20origin%20of%20the%20genetic%20code.%20J.%20Mol.%20Biol.38,%20367%E2%80%93379%20(1968)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Phizicky, E. M. &amp; Hopper, A. K. The life and times of a tRNA. <em>RNA</em><strong>29</strong>, 898–957 (2023).
</cite> [<a href="https://doi.org/10.1261/rna.079620.123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10275265/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37055150/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Phizicky,%20E.%20M.%20&amp;%20Hopper,%20A.%20K.%20The%20life%20and%20times%20of%20a%20tRNA.%20RNA29,%20898%E2%80%93957%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Yared, M.-J., Marcelot, A. &amp; Barraud, P. Beyond the anticodon: tRNA core modifications and their impact on structure, translation and stress adaptation. <em>Genes (Basel)</em><strong>15</strong>, 374 (2024).
</cite> [<a href="https://doi.org/10.3390/genes15030374" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10969862/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38540433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yared,%20M.-J.,%20Marcelot,%20A.%20&amp;%20Barraud,%20P.%20Beyond%20the%20anticodon:%20tRNA%20core%20modifications%20and%20their%20impact%20on%20structure,%20translation%20and%20stress%20adaptation.%20Genes%20(Basel)15,%20374%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Perret, V. et al. Relaxation of a transfer RNA specificity by removal of modified nucleotides. <em>Nature</em><strong>344</strong>, 787–789 (1990).
</cite> [<a href="https://doi.org/10.1038/344787a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2330033/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perret,%20V.%20et%20al.%20Relaxation%20of%20a%20transfer%20RNA%20specificity%20by%20removal%20of%20modified%20nucleotides.%20Nature344,%20787%E2%80%93789%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Giegé, R. &amp; Eriani, G. The tRNA identity landscape for aminoacylation and beyond. <em>Nucleic Acids Res.</em><strong>51</strong>, 1528–1570 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkad007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9976931/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36744444/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gieg%C3%A9,%20R.%20&amp;%20Eriani,%20G.%20The%20tRNA%20identity%20landscape%20for%20aminoacylation%20and%20beyond.%20Nucleic%20Acids%20Res.51,%201528%E2%80%931570%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Agris, P. F., Vendeix, F. A. P. &amp; Graham, W. D. tRNA’s wobble decoding of the genome: 40 years of modification. <em>J. Mol. Biol.</em><strong>366</strong>, 1–13 (2007).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2006.11.046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17187822/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Agris,%20P.%20F.,%20Vendeix,%20F.%20A.%20P.%20&amp;%20Graham,%20W.%20D.%20tRNA%E2%80%99s%20wobble%20decoding%20of%20the%20genome:%2040%20years%20of%20modification.%20J.%20Mol.%20Biol.366,%201%E2%80%9313%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Avcilar-Kucukgoze, I. et al. Discharging tRNAs: a tug of war between translation and detoxification in Escherichia coli. <em>Nucleic Acids Res.</em><strong>44</strong>, 8324–8334 (2016).
</cite> [<a href="https://doi.org/10.1093/nar/gkw697" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5041488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27507888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Avcilar-Kucukgoze,%20I.%20et%20al.%20Discharging%20tRNAs:%20a%20tug%20of%20war%20between%20translation%20and%20detoxification%20in%20Escherichia%20coli.%20Nucleic%20Acids%20Res.44,%208324%E2%80%938334%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Subramaniam, A. R., Pan, T. &amp; Cluzel, P. Environmental perturbations lift the degeneracy of the genetic code to regulate protein levels in bacteria. <em>Proc. Natl. Acad. Sci. USA</em><strong>110</strong>, 2419–2424 (2013).
</cite> [<a href="https://doi.org/10.1073/pnas.1211077110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3568297/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23277573/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Subramaniam,%20A.%20R.,%20Pan,%20T.%20&amp;%20Cluzel,%20P.%20Environmental%20perturbations%20lift%20the%20degeneracy%20of%20the%20genetic%20code%20to%20regulate%20protein%20levels%20in%20bacteria.%20Proc.%20Natl.%20Acad.%20Sci.%20USA110,%202419%E2%80%932424%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Dever, T. E. et al. Phosphorylation of initiation factor 2 alpha by protein kinase GCN2 mediates gene-specific translational control of GCN4 in yeast. <em>Cell</em><strong>68</strong>, 585–596 (1992).
</cite> [<a href="https://doi.org/10.1016/0092-8674(92)90193-g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1739968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dever,%20T.%20E.%20et%20al.%20Phosphorylation%20of%20initiation%20factor%202%20alpha%20by%20protein%20kinase%20GCN2%20mediates%20gene-specific%20translational%20control%20of%20GCN4%20in%20yeast.%20Cell68,%20585%E2%80%93596%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Sørensen, M. A. Charging levels of four tRNA species in <em>Escherichia coli</em> Rel(+) and Rel(-) strains during amino acid starvation: a simple model for the effect of ppGpp on translational accuracy. <em>J. Mol. Biol.</em><strong>307</strong>, 785–798 (2001).
</cite> [<a href="https://doi.org/10.1006/jmbi.2001.4525" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11273701/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?S%C3%B8rensen,%20M.%20A.%20Charging%20levels%20of%20four%20tRNA%20species%20in%20Escherichia%20coli%20Rel(+)%20and%20Rel(-)%20strains%20during%20amino%20acid%20starvation:%20a%20simple%20model%20for%20the%20effect%20of%20ppGpp%20on%20translational%20accuracy.%20J.%20Mol.%20Biol.307,%20785%E2%80%93798%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Zaborske, J. M. et al. Genome-wide analysis of tRNA charging and activation of the eIF2 kinase Gcn2p. <em>J. Biol. Chem.</em><strong>284</strong>, 25254–25267 (2009).
</cite> [<a href="https://doi.org/10.1074/jbc.M109.000877" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2757228/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19546227/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zaborske,%20J.%20M.%20et%20al.%20Genome-wide%20analysis%20of%20tRNA%20charging%20and%20activation%20of%20the%20eIF2%20kinase%20Gcn2p.%20J.%20Biol.%20Chem.284,%2025254%E2%80%9325267%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Passarelli, M. C. et al. Leucyl-tRNA synthetase is a tumour suppressor in breast cancer and regulates codon-dependent translation dynamics. <em>Nat. Cell Biol.</em><strong>24</strong>, 307–315 (2022).
</cite> [<a href="https://doi.org/10.1038/s41556-022-00856-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8977047/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35288656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Passarelli,%20M.%20C.%20et%20al.%20Leucyl-tRNA%20synthetase%20is%20a%20tumour%20suppressor%20in%20breast%20cancer%20and%20regulates%20codon-dependent%20translation%20dynamics.%20Nat.%20Cell%20Biol.24,%20307%E2%80%93315%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Hyeon, D. Y. et al. Evolution of the multi-tRNA synthetase complex and its role in cancer. <em>J. Biol. Chem.</em><strong>294</strong>, 5340–5351 (2019).
</cite> [<a href="https://doi.org/10.1074/jbc.REV118.002958" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6462501/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30782841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hyeon,%20D.%20Y.%20et%20al.%20Evolution%20of%20the%20multi-tRNA%20synthetase%20complex%20and%20its%20role%20in%20cancer.%20J.%20Biol.%20Chem.294,%205340%E2%80%935351%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Netzer, N. et al. Innate immune and chemically triggered oxidative stress modifies translational fidelity. <em>Nature</em><strong>462</strong>, 522–526 (2009).
</cite> [<a href="https://doi.org/10.1038/nature08576" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2785853/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19940929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Netzer,%20N.%20et%20al.%20Innate%20immune%20and%20chemically%20triggered%20oxidative%20stress%20modifies%20translational%20fidelity.%20Nature462,%20522%E2%80%93526%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Sun, L. et al. Evolutionary gain of alanine mischarging to noncognate tRNAs with a G4:U69 base pair. <em>J. Am. Chem. Soc.</em><strong>138</strong>, 12948–12955 (2016).
</cite> [<a href="https://doi.org/10.1021/jacs.6b07121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5356932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27622773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20L.%20et%20al.%20Evolutionary%20gain%20of%20alanine%20mischarging%20to%20noncognate%20tRNAs%20with%20a%20G4:U69%20base%20pair.%20J.%20Am.%20Chem.%20Soc.138,%2012948%E2%80%9312955%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Ling, J., Yadavalli, S. S. &amp; Ibba, M. Phenylalanyl-tRNA synthetase editing defects result in efficient mistranslation of phenylalanine codons as tyrosine. <em>RNA</em><strong>13</strong>, 1881–1886 (2007).
</cite> [<a href="https://doi.org/10.1261/rna.684107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2040089/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17804641/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ling,%20J.,%20Yadavalli,%20S.%20S.%20&amp;%20Ibba,%20M.%20Phenylalanyl-tRNA%20synthetase%20editing%20defects%20result%20in%20efficient%20mistranslation%20of%20phenylalanine%20codons%20as%20tyrosine.%20RNA13,%201881%E2%80%931886%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Shepherd, J. &amp; Ibba, M. Relaxed substrate specificity leads to extensive tRNA mischarging by Streptococcus pneumoniae class I and class II aminoacyl-tRNA synthetases. <em>mBio</em><strong>5</strong>, e01656–14 (2014).
</cite> [<a href="https://doi.org/10.1128/mBio.01656-14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4173786/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25205097/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shepherd,%20J.%20&amp;%20Ibba,%20M.%20Relaxed%20substrate%20specificity%20leads%20to%20extensive%20tRNA%20mischarging%20by%20Streptococcus%20pneumoniae%20class%20I%20and%20class%20II%20aminoacyl-tRNA%20synthetases.%20mBio5,%20e01656%E2%80%9314%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Kelly, P. Alanyl-tRNA synthetase quality control prevents global dysregulation of the Escherichia coli proteome. mBio <strong>10</strong>, e02921-19 (2019).
</cite> [<a href="https://doi.org/10.1128/mBio.02921-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6918089/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31848288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kelly,%20P.%20Alanyl-tRNA%20synthetase%20quality%20control%20prevents%20global%20dysregulation%20of%20the%20Escherichia%20coli%20proteome.%20mBio%2010,%20e02921-19%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Evans, M. E., Clark, W. C., Zheng, G. &amp; Pan, T. Determination of tRNA aminoacylation levels by high-throughput sequencing. <em>Nucleic Acids Res.</em><strong>45</strong>, e133 (2017).
</cite> [<a href="https://doi.org/10.1093/nar/gkx514" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5737633/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28586482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Evans,%20M.%20E.,%20Clark,%20W.%20C.,%20Zheng,%20G.%20&amp;%20Pan,%20T.%20Determination%20of%20tRNA%20aminoacylation%20levels%20by%20high-throughput%20sequencing.%20Nucleic%20Acids%20Res.45,%20e133%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Watkins, C. P., Zhang, W., Wylder, A. C., Katanski, C. D. &amp; Pan, T. A multiplex platform for small RNA sequencing elucidates multifaceted tRNA stress response and translational regulation. <em>Nat. Commun.</em><strong>13</strong>, 2491 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-30261-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9072684/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35513407/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Watkins,%20C.%20P.,%20Zhang,%20W.,%20Wylder,%20A.%20C.,%20Katanski,%20C.%20D.%20&amp;%20Pan,%20T.%20A%20multiplex%20platform%20for%20small%20RNA%20sequencing%20elucidates%20multifaceted%20tRNA%20stress%20response%20and%20translational%20regulation.%20Nat.%20Commun.13,%202491%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Davidsen, K. &amp; Sullivan, L. B. A robust method for measuring aminoacylation through tRNA-Seq. <em>Elife</em><strong>12</strong>, RP91554 (2024).</cite> [<a href="https://doi.org/10.7554/eLife.91554" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11288633/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39076160/" class="usa-link">PubMed</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Hernandez-Alias, X. et al. Single-read tRNA-seq analysis reveals coordination of tRNA modification and aminoacylation and fragmentation. <em>Nucleic Acids Res.</em><strong>51</strong>, e17 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkac1185" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9943672/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36537222/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hernandez-Alias,%20X.%20et%20al.%20Single-read%20tRNA-seq%20analysis%20reveals%20coordination%20of%20tRNA%20modification%20and%20aminoacylation%20and%20fragmentation.%20Nucleic%20Acids%20Res.51,%20e17%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Carlile, T. M. et al. Pseudouridine profiling reveals regulated mRNA pseudouridylation in yeast and human cells. <em>Nature</em><strong>515</strong>, 143–146 (2014).
</cite> [<a href="https://doi.org/10.1038/nature13802" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4224642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25192136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carlile,%20T.%20M.%20et%20al.%20Pseudouridine%20profiling%20reveals%20regulated%20mRNA%20pseudouridylation%20in%20yeast%20and%20human%20cells.%20Nature515,%20143%E2%80%93146%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Varshney, U., Lee, C. P. &amp; RajBhandary, U. L. Direct analysis of aminoacylation levels of tRNAs in vivo. Application to studying recognition of <em>Escherichia coli</em> initiator tRNA mutants by glutaminyl-tRNA synthetase. <em>J. Biol. Chem.</em><strong>266</strong>, 24712–24718 (1991).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/1761566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Varshney,%20U.,%20Lee,%20C.%20P.%20&amp;%20RajBhandary,%20U.%20L.%20Direct%20analysis%20of%20aminoacylation%20levels%20of%20tRNAs%20in%20vivo.%20Application%20to%20studying%20recognition%20of%20Escherichia%20coli%20initiator%20tRNA%20mutants%20by%20glutaminyl-tRNA%20synthetase.%20J.%20Biol.%20Chem.266,%2024712%E2%80%9324718%20(1991)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Grosjean, H., Droogmans, L., Roovers, M. &amp; Keith, G. Detection of enzymatic activity of transfer RNA modification enzymes using radiolabeled tRNA substrates. <em>Methods Enzymol.</em><strong>425</strong>, 55–101 (2007).
</cite> [<a href="https://doi.org/10.1016/S0076-6879(07)25003-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17673079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grosjean,%20H.,%20Droogmans,%20L.,%20Roovers,%20M.%20&amp;%20Keith,%20G.%20Detection%20of%20enzymatic%20activity%20of%20transfer%20RNA%20modification%20enzymes%20using%20radiolabeled%20tRNA%20substrates.%20Methods%20Enzymol.425,%2055%E2%80%93101%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Henley, R. Y. et al. Electrophoretic deformation of individual transfer RNA molecules reveals their identity. <em>Nano Lett.</em><strong>16</strong>, 138–144 (2016).
</cite> [<a href="https://doi.org/10.1021/acs.nanolett.5b03331" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4890568/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26609994/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Henley,%20R.%20Y.%20et%20al.%20Electrophoretic%20deformation%20of%20individual%20transfer%20RNA%20molecules%20reveals%20their%20identity.%20Nano%20Lett.16,%20138%E2%80%93144%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Smith, A. M., Abu-Shumays, R., Akeson, M. &amp; Bernick, D. L. Capture, unfolding, and detection of individual tRNA molecules using a nanopore device. <em>Front. Bioeng. Biotechnol.</em><strong>3</strong>, 91 (2015).
</cite> [<a href="https://doi.org/10.3389/fbioe.2015.00091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4478443/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26157798/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith,%20A.%20M.,%20Abu-Shumays,%20R.,%20Akeson,%20M.%20&amp;%20Bernick,%20D.%20L.%20Capture,%20unfolding,%20and%20detection%20of%20individual%20tRNA%20molecules%20using%20a%20nanopore%20device.%20Front.%20Bioeng.%20Biotechnol.3,%2091%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Wang, Y. et al. Structural-profiling of low molecular weight RNAs by nanopore trapping/translocation using Mycobacterium smegmatis porin A. <em>Nat. Commun.</em><strong>12</strong>, 3368 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-23764-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8185011/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34099723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Y.%20et%20al.%20Structural-profiling%20of%20low%20molecular%20weight%20RNAs%20by%20nanopore%20trapping/translocation%20using%20Mycobacterium%20smegmatis%20porin%20A.%20Nat.%20Commun.12,%203368%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Thomas, N. K. et al. Direct nanopore sequencing of individual full length tRNA strands. <em>ACS Nano</em><strong>15</strong>, 16642–16653 (2021).
</cite> [<a href="https://doi.org/10.1021/acsnano.1c06488" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10189790/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34618430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thomas,%20N.%20K.%20et%20al.%20Direct%20nanopore%20sequencing%20of%20individual%20full%20length%20tRNA%20strands.%20ACS%20Nano15,%2016642%E2%80%9316653%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Garalde, D. R. et al. Highly parallel direct RNA sequencing on an array of nanopores. <em>Nat. Methods</em><strong>15</strong>, 201–206 (2018).
</cite> [<a href="https://doi.org/10.1038/nmeth.4577" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29334379/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Garalde,%20D.%20R.%20et%20al.%20Highly%20parallel%20direct%20RNA%20sequencing%20on%20an%20array%20of%20nanopores.%20Nat.%20Methods15,%20201%E2%80%93206%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>White, L. K., Strugar, S. M., MacFadden, A. &amp; Hesselberth, J. R. Nanopore sequencing of internal 2’-PO modifications installed by RNA repair. <em>RNA</em><strong>29</strong>, 847–861 (2023).
</cite> [<a href="https://doi.org/10.1261/rna.079290.122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10187680/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36854608/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?White,%20L.%20K.,%20Strugar,%20S.%20M.,%20MacFadden,%20A.%20&amp;%20Hesselberth,%20J.%20R.%20Nanopore%20sequencing%20of%20internal%202%E2%80%99-PO%20modifications%20installed%20by%20RNA%20repair.%20RNA29,%20847%E2%80%93861%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Lucas, M. C. et al. Quantitative analysis of tRNA abundance and modifications by nanopore RNA sequencing. <em>Nat. Biotechnol.</em><strong>42</strong>, 72–86 (2024).
</cite> [<a href="https://doi.org/10.1038/s41587-023-01743-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10791586/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37024678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lucas,%20M.%20C.%20et%20al.%20Quantitative%20analysis%20of%20tRNA%20abundance%20and%20modifications%20by%20nanopore%20RNA%20sequencing.%20Nat.%20Biotechnol.42,%2072%E2%80%9386%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Radakovic, A., Wright, T. H., Lelyveld, V. S. &amp; Szostak, J. W. A potential role for aminoacylation in primordial RNA copying chemistry. <em>Biochemistry</em><strong>60</strong>, 477–488 (2021).
</cite> [<a href="https://doi.org/10.1021/acs.biochem.0c00943" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9634692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33523633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Radakovic,%20A.,%20Wright,%20T.%20H.,%20Lelyveld,%20V.%20S.%20&amp;%20Szostak,%20J.%20W.%20A%20potential%20role%20for%20aminoacylation%20in%20primordial%20RNA%20copying%20chemistry.%20Biochemistry60,%20477%E2%80%93488%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Radakovic, A., DasGupta, S., Wright, T. H., Aitken, H. R. M. &amp; Szostak, J. W. Nonenzymatic assembly of active chimeric ribozymes from aminoacylated RNA oligonucleotides. Proc. Natl. Acad. Sci. USA 119, e2116840119 (2022).
</cite> [<a href="https://doi.org/10.1073/pnas.2116840119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8851484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35140183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Radakovic,%20A.,%20DasGupta,%20S.,%20Wright,%20T.%20H.,%20Aitken,%20H.%20R.%20M.%20&amp;%20Szostak,%20J.%20W.%20Nonenzymatic%20assembly%20of%20active%20chimeric%20ribozymes%20from%20aminoacylated%20RNA%20oligonucleotides.%20Proc.%20Natl.%20Acad.%20Sci.%20USA%20119,%20e2116840119%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Murakami, H., Ohta, A., Ashigai, H. &amp; Suga, H. A highly flexible tRNA acylation method for non-natural polypeptide synthesis. <em>Nat. Methods</em><strong>3</strong>, 357–359 (2006).
</cite> [<a href="https://doi.org/10.1038/nmeth877" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16628205/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Murakami,%20H.,%20Ohta,%20A.,%20Ashigai,%20H.%20&amp;%20Suga,%20H.%20A%20highly%20flexible%20tRNA%20acylation%20method%20for%20non-natural%20polypeptide%20synthesis.%20Nat.%20Methods3,%20357%E2%80%93359%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Mansy, S. S. et al. Template-directed synthesis of a genetic polymer in a model protocell. <em>Nature</em><strong>454</strong>, 122–125 (2008).
</cite> [<a href="https://doi.org/10.1038/nature07018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2743009/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18528332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mansy,%20S.%20S.%20et%20al.%20Template-directed%20synthesis%20of%20a%20genetic%20polymer%20in%20a%20model%20protocell.%20Nature454,%20122%E2%80%93125%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Radakovic, A. et al. A potential role for RNA aminoacylation prior to its role in peptide synthesis. <em>Proc. Natl. Acad. Sci. USA</em><strong>121</strong>, e2410206121 (2024).
</cite> [<a href="https://doi.org/10.1073/pnas.2410206121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11363276/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39178230/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Radakovic,%20A.%20et%20al.%20A%20potential%20role%20for%20RNA%20aminoacylation%20prior%20to%20its%20role%20in%20peptide%20synthesis.%20Proc.%20Natl.%20Acad.%20Sci.%20USA121,%20e2410206121%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Liu, Z., Ajram, G., Rossi, J.-C. &amp; Pascal, R. The chemical likelihood of ribonucleotide-α-amino acid copolymers as players for early stages of evolution. <em>J. Mol. Evol.</em><strong>87</strong>, 83–92 (2019).
</cite> [<a href="https://doi.org/10.1007/s00239-019-9887-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6443614/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30788531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Z.,%20Ajram,%20G.,%20Rossi,%20J.-C.%20&amp;%20Pascal,%20R.%20The%20chemical%20likelihood%20of%20ribonucleotide-%CE%B1-amino%20acid%20copolymers%20as%20players%20for%20early%20stages%20of%20evolution.%20J.%20Mol.%20Evol.87,%2083%E2%80%9392%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Peacock, J. R. et al. Amino acid-dependent stability of the acyl linkage in aminoacyl-tRNA. <em>RNA</em><strong>20</strong>, 758–764 (2014).
</cite> [<a href="https://doi.org/10.1261/rna.044123.113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4024630/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24751649/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peacock,%20J.%20R.%20et%20al.%20Amino%20acid-dependent%20stability%20of%20the%20acyl%20linkage%20in%20aminoacyl-tRNA.%20RNA20,%20758%E2%80%93764%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Pütz, J. et al. Rapid selection of aminoacyl-tRNAs based on biotinylation of alpha-NH2 group of charged amino acids. <em>Nucleic Acids Res.</em><strong>25</strong>, 1862–1863 (1997).
</cite> [<a href="https://doi.org/10.1093/nar/25.9.1862" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC146665/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9162902/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?P%C3%BCtz,%20J.%20et%20al.%20Rapid%20selection%20of%20aminoacyl-tRNAs%20based%20on%20biotinylation%20of%20alpha-NH2%20group%20of%20charged%20amino%20acids.%20Nucleic%20Acids%20Res.25,%201862%E2%80%931863%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Murakami, H., Kourouklis, D. &amp; Suga, H. Using a solid-phase ribozyme aminoacylation system to reprogram the genetic code. <em>Chem. Biol.</em><strong>10</strong>, 1077–1084 (2003).
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2003.10.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14652075/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Murakami,%20H.,%20Kourouklis,%20D.%20&amp;%20Suga,%20H.%20Using%20a%20solid-phase%20ribozyme%20aminoacylation%20system%20to%20reprogram%20the%20genetic%20code.%20Chem.%20Biol.10,%201077%E2%80%931084%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Gamper, H. &amp; Hou, Y.-M. A label-free assay for aminoacylation of tRNA. Genes (Basel) <strong>11</strong>, 1173 (2020).
</cite> [<a href="https://doi.org/10.3390/genes11101173" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7601589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33036365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gamper,%20H.%20&amp;%20Hou,%20Y.-M.%20A%20label-free%20assay%20for%20aminoacylation%20of%20tRNA.%20Genes%20(Basel)%2011,%201173%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Rand, A. C. et al. Mapping DNA methylation with high-throughput nanopore sequencing. <em>Nat. Methods</em><strong>14</strong>, 411–413 (2017).
</cite> [<a href="https://doi.org/10.1038/nmeth.4189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5704956/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28218897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rand,%20A.%20C.%20et%20al.%20Mapping%20DNA%20methylation%20with%20high-throughput%20nanopore%20sequencing.%20Nat.%20Methods14,%20411%E2%80%93413%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Furlan, M. et al. Computational methods for RNA modification detection from nanopore direct RNA sequencing data. <em>RNA Biol.</em><strong>18</strong>, 31–40 (2021).
</cite> [<a href="https://doi.org/10.1080/15476286.2021.1978215" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8677041/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34559589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Furlan,%20M.%20et%20al.%20Computational%20methods%20for%20RNA%20modification%20detection%20from%20nanopore%20direct%20RNA%20sequencing%20data.%20RNA%20Biol.18,%2031%E2%80%9340%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>White, L. K. &amp; Hesselberth, J. R. Modification mapping by nanopore sequencing. <em>Front. Genet.</em><strong>13</strong>, 1037134 (2022).
</cite> [<a href="https://doi.org/10.3389/fgene.2022.1037134" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9650216/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36386798/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?White,%20L.%20K.%20&amp;%20Hesselberth,%20J.%20R.%20Modification%20mapping%20by%20nanopore%20sequencing.%20Front.%20Genet.13,%201037134%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Begik, O. et al. Quantitative profiling of pseudouridylation dynamics in native RNAs with nanopore sequencing. <em>Nat. Biotechnol.</em><strong>39</strong>, 1278–1291 (2021).
</cite> [<a href="https://doi.org/10.1038/s41587-021-00915-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33986546/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Begik,%20O.%20et%20al.%20Quantitative%20profiling%20of%20pseudouridylation%20dynamics%20in%20native%20RNAs%20with%20nanopore%20sequencing.%20Nat.%20Biotechnol.39,%201278%E2%80%931291%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Leger, A. et al. RNA modifications detection by comparative Nanopore direct RNA sequencing. <em>Nat. Commun.</em><strong>12</strong>, 7198 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-27393-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8664944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34893601/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leger,%20A.%20et%20al.%20RNA%20modifications%20detection%20by%20comparative%20Nanopore%20direct%20RNA%20sequencing.%20Nat.%20Commun.12,%207198%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Alexandrov, A. et al. Rapid tRNA decay can result from lack of nonessential modifications. <em>Mol. Cell</em><strong>21</strong>, 87–96 (2006).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2005.10.036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16387656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alexandrov,%20A.%20et%20al.%20Rapid%20tRNA%20decay%20can%20result%20from%20lack%20of%20nonessential%20modifications.%20Mol.%20Cell21,%2087%E2%80%9396%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Tasak, M. &amp; Phizicky, E. M. Initiator tRNA lacking 1-methyladenosine is targeted by the rapid tRNA decay pathway in evolutionarily distant yeast species. <em>PLoS Genet.</em><strong>18</strong>, e1010215 (2022).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1010215" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9362929/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35901126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tasak,%20M.%20&amp;%20Phizicky,%20E.%20M.%20Initiator%20tRNA%20lacking%201-methyladenosine%20is%20targeted%20by%20the%20rapid%20tRNA%20decay%20pathway%20in%20evolutionarily%20distant%20yeast%20species.%20PLoS%20Genet.18,%20e1010215%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>De Zoysa, T. &amp; Phizicky, E. M. Hypomodified tRNA in evolutionarily distant yeasts can trigger rapid tRNA decay to activate the general amino acid control response, but with different consequences. <em>PLoS Genet.</em><strong>16</strong>, e1008893 (2020).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1008893" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7473580/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32841241/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Zoysa,%20T.%20&amp;%20Phizicky,%20E.%20M.%20Hypomodified%20tRNA%20in%20evolutionarily%20distant%20yeasts%20can%20trigger%20rapid%20tRNA%20decay%20to%20activate%20the%20general%20amino%20acid%20control%20response,%20but%20with%20different%20consequences.%20PLoS%20Genet.16,%20e1008893%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Dewe, J. M., Whipple, J. M., Chernyakov, I., Jaramillo, L. N. &amp; Phizicky, E. M. The yeast rapid tRNA decay pathway competes with elongation factor 1A for substrate tRNAs and acts on tRNAs lacking one or more of several modifications. <em>RNA</em><strong>18</strong>, 1886–1896 (2012).
</cite> [<a href="https://doi.org/10.1261/rna.033654.112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3446711/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22895820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dewe,%20J.%20M.,%20Whipple,%20J.%20M.,%20Chernyakov,%20I.,%20Jaramillo,%20L.%20N.%20&amp;%20Phizicky,%20E.%20M.%20The%20yeast%20rapid%20tRNA%20decay%20pathway%20competes%20with%20elongation%20factor%201A%20for%20substrate%20tRNAs%20and%20acts%20on%20tRNAs%20lacking%20one%20or%20more%20of%20several%20modifications.%20RNA18,%201886%E2%80%931896%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Guy, M. P. et al. Identification of the determinants of tRNA function and susceptibility to rapid tRNA decay by high-throughput in vivo analysis. <em>Genes Dev.</em><strong>28</strong>, 1721–1732 (2014).
</cite> [<a href="https://doi.org/10.1101/gad.245936.114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4117946/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25085423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guy,%20M.%20P.%20et%20al.%20Identification%20of%20the%20determinants%20of%20tRNA%20function%20and%20susceptibility%20to%20rapid%20tRNA%20decay%20by%20high-throughput%20in%20vivo%20analysis.%20Genes%20Dev.28,%201721%E2%80%931732%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Payea, M. J. et al. Widespread temperature sensitivity and tRNA decay due to mutations in a yeast tRNA. <em>RNA</em><strong>24</strong>, 410–422 (2018).
</cite> [<a href="https://doi.org/10.1261/rna.064642.117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5824359/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29259051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Payea,%20M.%20J.%20et%20al.%20Widespread%20temperature%20sensitivity%20and%20tRNA%20decay%20due%20to%20mutations%20in%20a%20yeast%20tRNA.%20RNA24,%20410%E2%80%93422%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Payea, M. J., Hauke, A. C., De Zoysa, T. &amp; Phizicky, E. M. Mutations in the anticodon stem of tRNA cause accumulation and Met22-dependent decay of pre-tRNA in yeast. <em>RNA</em><strong>26</strong>, 29–43 (2020).
</cite> [<a href="https://doi.org/10.1261/rna.073155.119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6913130/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31619505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Payea,%20M.%20J.,%20Hauke,%20A.%20C.,%20De%20Zoysa,%20T.%20&amp;%20Phizicky,%20E.%20M.%20Mutations%20in%20the%20anticodon%20stem%20of%20tRNA%20cause%20accumulation%20and%20Met22-dependent%20decay%20of%20pre-tRNA%20in%20yeast.%20RNA26,%2029%E2%80%9343%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Han, L., Guy, M. P., Kon, Y. &amp; Phizicky, E. M. Lack of 2’-O-methylation in the tRNA anticodon loop of two phylogenetically distant yeast species activates the general amino acid control pathway. <em>PLoS Genet.</em><strong>14</strong>, e1007288 (2018).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1007288" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5892943/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29596413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Han,%20L.,%20Guy,%20M.%20P.,%20Kon,%20Y.%20&amp;%20Phizicky,%20E.%20M.%20Lack%20of%202%E2%80%99-O-methylation%20in%20the%20tRNA%20anticodon%20loop%20of%20two%20phylogenetically%20distant%20yeast%20species%20activates%20the%20general%20amino%20acid%20control%20pathway.%20PLoS%20Genet.14,%20e1007288%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>De Zoysa, T. et al. A connection between the ribosome and two S. pombe tRNA modification mutants subject to rapid tRNA decay. <em>PLoS Genet.</em><strong>20</strong>, e1011146 (2024).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1011146" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10861057/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38295128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Zoysa,%20T.%20et%20al.%20A%20connection%20between%20the%20ribosome%20and%20two%20S.%20pombe%20tRNA%20modification%20mutants%20subject%20to%20rapid%20tRNA%20decay.%20PLoS%20Genet.20,%20e1011146%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Chernyakov, I., Whipple, J. M., Kotelawala, L., Grayhack, E. J. &amp; Phizicky, E. M. Degradation of several hypomodified mature tRNA species in Saccharomyces cerevisiae is mediated by Met22 and the 5’−3’ exonucleases Rat1 and Xrn1. <em>Genes Dev.</em><strong>22</strong>, 1369–1380 (2008).
</cite> [<a href="https://doi.org/10.1101/gad.1654308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2377191/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18443146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chernyakov,%20I.,%20Whipple,%20J.%20M.,%20Kotelawala,%20L.,%20Grayhack,%20E.%20J.%20&amp;%20Phizicky,%20E.%20M.%20Degradation%20of%20several%20hypomodified%20mature%20tRNA%20species%20in%20Saccharomyces%20cerevisiae%20is%20mediated%20by%20Met22%20and%20the%205%E2%80%99%E2%88%923%E2%80%99%20exonucleases%20Rat1%20and%20Xrn1.%20Genes%20Dev.22,%201369%E2%80%931380%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Dichtl, B., Stevens, A. &amp; Tollervey, D. Lithium toxicity in yeast is due to the inhibition of RNA processing enzymes. <em>EMBO J.</em><strong>16</strong>, 7184–7195 (1997).
</cite> [<a href="https://doi.org/10.1093/emboj/16.23.7184" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1170319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9384595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dichtl,%20B.,%20Stevens,%20A.%20&amp;%20Tollervey,%20D.%20Lithium%20toxicity%20in%20yeast%20is%20due%20to%20the%20inhibition%20of%20RNA%20processing%20enzymes.%20EMBO%20J.16,%207184%E2%80%937195%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Cappannini, A. et al. MODOMICS: a database of RNA modifications and related information. 2023 update. <em>Nucleic Acids Res.</em><strong>52</strong>, D239–D244 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkad1083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767930/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38015436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cappannini,%20A.%20et%20al.%20MODOMICS:%20a%20database%20of%20RNA%20modifications%20and%20related%20information.%202023%20update.%20Nucleic%20Acids%20Res.52,%20D239%E2%80%93D244%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Kotelawala, L., Grayhack, E. J. &amp; Phizicky, E. M. Identification of yeast tRNA Um(44) 2’-O-methyltransferase (Trm44) and demonstration of a Trm44 role in sustaining levels of specific tRNA(Ser) species. <em>RNA</em><strong>14</strong>, 158–169 (2008).
</cite> [<a href="https://doi.org/10.1261/rna.811008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2151035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18025252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kotelawala,%20L.,%20Grayhack,%20E.%20J.%20&amp;%20Phizicky,%20E.%20M.%20Identification%20of%20yeast%20tRNA%20Um(44)%202%E2%80%99-O-methyltransferase%20(Trm44)%20and%20demonstration%20of%20a%20Trm44%20role%20in%20sustaining%20levels%20of%20specific%20tRNA(Ser)%20species.%20RNA14,%20158%E2%80%93169%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Stoiber, M. et al. De novo identification of DNA modifications enabled by genome-guided nanopore signal processing. Preprint at <em>bioRxiv</em>10.1101/094672 (2017).</cite>
</li>
<li id="CR62">
<span class="label">62.</span><cite>Diensthuber, G. et al. Enhanced detection of RNA modifications and read mapping with high-accuracy nanopore RNA basecalling models. <em>Genome Res.</em>10.1101/gr.278849.123 (2024).</cite> [<a href="https://doi.org/10.1101/gr.278849.123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11610583/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39271295/" class="usa-link">PubMed</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Lueck, J. D. et al. Engineered transfer RNAs for suppression of premature termination codons. <em>Nat. Commun.</em><strong>10</strong>, 822 (2019).
</cite> [<a href="https://doi.org/10.1038/s41467-019-08329-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6379413/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30778053/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lueck,%20J.%20D.%20et%20al.%20Engineered%20transfer%20RNAs%20for%20suppression%20of%20premature%20termination%20codons.%20Nat.%20Commun.10,%20822%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Albers, S. et al. Engineered tRNAs suppress nonsense mutations in cells and in vivo. <em>Nature</em><strong>618</strong>, 842–848 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06133-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10284701/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37258671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Albers,%20S.%20et%20al.%20Engineered%20tRNAs%20suppress%20nonsense%20mutations%20in%20cells%20and%20in%20vivo.%20Nature618,%20842%E2%80%93848%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Anastassiadis, T. &amp; Köhrer, C. Ushering in the era of tRNA medicines. <em>J. Biol. Chem.</em><strong>299</strong>, 105246 (2023).
</cite> [<a href="https://doi.org/10.1016/j.jbc.2023.105246" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10583094/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37703991/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Anastassiadis,%20T.%20&amp;%20K%C3%B6hrer,%20C.%20Ushering%20in%20the%20era%20of%20tRNA%20medicines.%20J.%20Biol.%20Chem.299,%20105246%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Uhlenbeck, O. C. &amp; Schrader, J. M. Evolutionary tuning impacts the design of bacterial tRNAs for the incorporation of unnatural amino acids by ribosomes. <em>Curr. Opin. Chem. Biol.</em><strong>46</strong>, 138–145 (2018).
</cite> [<a href="https://doi.org/10.1016/j.cbpa.2018.07.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6601615/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30059836/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Uhlenbeck,%20O.%20C.%20&amp;%20Schrader,%20J.%20M.%20Evolutionary%20tuning%20impacts%20the%20design%20of%20bacterial%20tRNAs%20for%20the%20incorporation%20of%20unnatural%20amino%20acids%20by%20ribosomes.%20Curr.%20Opin.%20Chem.%20Biol.46,%20138%E2%80%93145%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Katoh, T. &amp; Suga, H. In vitro genetic code reprogramming for the expansion of usable noncanonical amino acids. <em>Annu. Rev. Biochem.</em><strong>91</strong>, 221–243 (2022).
</cite> [<a href="https://doi.org/10.1146/annurev-biochem-040320-103817" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35729073/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Katoh,%20T.%20&amp;%20Suga,%20H.%20In%20vitro%20genetic%20code%20reprogramming%20for%20the%20expansion%20of%20usable%20noncanonical%20amino%20acids.%20Annu.%20Rev.%20Biochem.91,%20221%E2%80%93243%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Dunkelmann, D. L. et al. Adding α,α-disubstituted and β-linked monomers to the genetic code of an organism. <em>Nature</em><strong>625</strong>, 603–610 (2024).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06897-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10794150/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38200312/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dunkelmann,%20D.%20L.%20et%20al.%20Adding%20%CE%B1,%CE%B1-disubstituted%20and%20%CE%B2-linked%20monomers%20to%20the%20genetic%20code%20of%20an%20organism.%20Nature625,%20603%E2%80%93610%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Soni, C. et al. A translation-independent directed evolution strategy to engineer aminoacyl-tRNA synthetases. <em>ACS Central Sci.</em>10.1021/acscentsci.3c01557 (2024).</cite> [<a href="https://doi.org/10.1021/acscentsci.3c01557" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11212135/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38947215/" class="usa-link">PubMed</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Zhou, L., O’Flaherty, D. K. &amp; Szostak, J. W. Template-directed copying of RNA by non-enzymatic ligation. <em>Angew. Chem. Int. Ed.</em><strong>59</strong>, 15682–15687 (2020).</cite> [<a href="https://doi.org/10.1002/anie.202004934" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7496532/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32558121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20L.,%20O%E2%80%99Flaherty,%20D.%20K.%20&amp;%20Szostak,%20J.%20W.%20Template-directed%20copying%20of%20RNA%20by%20non-enzymatic%20ligation.%20Angew.%20Chem.%20Int.%20Ed.59,%2015682%E2%80%9315687%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Wiltrout, E., Goodenbour, J. M., Fréchin, M. &amp; Pan, T. Misacylation of tRNA with methionine in Saccharomyces cerevisiae. <em>Nucleic Acids Res.</em><strong>40</strong>, 10494–10506 (2012).
</cite> [<a href="https://doi.org/10.1093/nar/gks805" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3488245/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22941646/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wiltrout,%20E.,%20Goodenbour,%20J.%20M.,%20Fr%C3%A9chin,%20M.%20&amp;%20Pan,%20T.%20Misacylation%20of%20tRNA%20with%20methionine%20in%20Saccharomyces%20cerevisiae.%20Nucleic%20Acids%20Res.40,%2010494%E2%80%9310506%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Shigematsu, M. et al. YAMAT-seq: an efficient method for high-throughput sequencing of mature transfer RNAs. <em>Nucleic Acids Res.</em><strong>45</strong>, e70 (2017).
</cite> [<a href="https://doi.org/10.1093/nar/gkx005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5605243/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28108659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shigematsu,%20M.%20et%20al.%20YAMAT-seq:%20an%20efficient%20method%20for%20high-throughput%20sequencing%20of%20mature%20transfer%20RNAs.%20Nucleic%20Acids%20Res.45,%20e70%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Warren, J. M. et al. Combining tRNA sequencing methods to characterize plant tRNA expression and post-transcriptional modification. <em>RNA Biol.</em><strong>18</strong>, 64–78 (2021).
</cite> [<a href="https://doi.org/10.1080/15476286.2020.1792089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7834048/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32715941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Warren,%20J.%20M.%20et%20al.%20Combining%20tRNA%20sequencing%20methods%20to%20characterize%20plant%20tRNA%20expression%20and%20post-transcriptional%20modification.%20RNA%20Biol.18,%2064%E2%80%9378%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Ceriotti, L. F., Warren, J. M., Virginia Sanchez-Puerta, M. &amp; Sloan, D. B. The landscape of Arabidopsis tRNA aminoacylation. Preprint at <em>bioRxiv</em>10.1101/2024.09.09.612099 (2024).</cite> [<a href="https://doi.org/10.1111/tpj.17146" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11658184/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39555621/" class="usa-link">PubMed</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Cozen, A. E. et al. ARM-seq: AlkB-facilitated RNA methylation sequencing reveals a complex landscape of modified tRNA fragments. <em>Nat. Methods</em><strong>12</strong>, 879–884 (2015).
</cite> [<a href="https://doi.org/10.1038/nmeth.3508" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4553111/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26237225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cozen,%20A.%20E.%20et%20al.%20ARM-seq:%20AlkB-facilitated%20RNA%20methylation%20sequencing%20reveals%20a%20complex%20landscape%20of%20modified%20tRNA%20fragments.%20Nat.%20Methods12,%20879%E2%80%93884%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Zheng, G. et al. Efficient and quantitative high-throughput tRNA sequencing. <em>Nat. Methods</em><strong>12</strong>, 835–837 (2015).
</cite> [<a href="https://doi.org/10.1038/nmeth.3478" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4624326/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26214130/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20G.%20et%20al.%20Efficient%20and%20quantitative%20high-throughput%20tRNA%20sequencing.%20Nat.%20Methods12,%20835%E2%80%93837%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Scheepbouwer, C. et al. ALL-tRNAseq enables robust tRNA profiling in tissue samples. <em>Genes Dev.</em><strong>37</strong>, 243–257 (2023).
</cite> [<a href="https://doi.org/10.1101/gad.350233.122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10111867/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36810209/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scheepbouwer,%20C.%20et%20al.%20ALL-tRNAseq%20enables%20robust%20tRNA%20profiling%20in%20tissue%20samples.%20Genes%20Dev.37,%20243%E2%80%93257%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Fleming, A. M., Mathewson, N. J., Howpay Manage, S. A. &amp; Burrows, C. J. Nanopore dwell time analysis permits sequencing and conformational assignment of pseudouridine in SARS-CoV-2. <em>ACS Cent. Sci.</em><strong>7</strong>, 1707–1717 (2021).
</cite> [<a href="https://doi.org/10.1021/acscentsci.1c00788" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8554835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34729414/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fleming,%20A.%20M.,%20Mathewson,%20N.%20J.,%20Howpay%20Manage,%20S.%20A.%20&amp;%20Burrows,%20C.%20J.%20Nanopore%20dwell%20time%20analysis%20permits%20sequencing%20and%20conformational%20assignment%20of%20pseudouridine%20in%20SARS-CoV-2.%20ACS%20Cent.%20Sci.7,%201707%E2%80%931717%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Stephenson, W. et al. Direct detection of RNA modifications and structure using single-molecule nanopore sequencing. Cell Genom. <strong>2</strong>, 10009 (2022).</cite> [<a href="https://doi.org/10.1016/j.xgen.2022.100097" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8896822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35252946/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stephenson,%20W.%20et%20al.%20Direct%20detection%20of%20RNA%20modifications%20and%20structure%20using%20single-molecule%20nanopore%20sequencing.%20Cell%20Genom.%202,%2010009%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Shah, N. H. &amp; Muir, T. W. Inteins: nature’s gift to protein chemists. <em>Chem. Sci.</em><strong>5</strong>, 446–461 (2014).
</cite> [<a href="https://doi.org/10.1039/C3SC52951G" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3949740/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24634716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shah,%20N.%20H.%20&amp;%20Muir,%20T.%20W.%20Inteins:%20nature%E2%80%99s%20gift%20to%20protein%20chemists.%20Chem.%20Sci.5,%20446%E2%80%93461%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Schultz, S. K. et al. Modifications in the T arm of tRNA globally determine tRNA maturation, function, and cellular fitness. <em>Proc. Natl. Acad. Sci. USA</em><strong>121</strong>, e2401154121 (2024).
</cite> [<a href="https://doi.org/10.1073/pnas.2401154121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11214086/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38889150/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schultz,%20S.%20K.%20et%20al.%20Modifications%20in%20the%20T%20arm%20of%20tRNA%20globally%20determine%20tRNA%20maturation,%20function,%20and%20cellular%20fitness.%20Proc.%20Natl.%20Acad.%20Sci.%20USA121,%20e2401154121%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Han, L. &amp; Phizicky, E. M. A rationale for tRNA modification circuits in the anticodon loop. <em>RNA</em><strong>24</strong>, 1277–1284 (2018).
</cite> [<a href="https://doi.org/10.1261/rna.067736.118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6140457/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30026310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Han,%20L.%20&amp;%20Phizicky,%20E.%20M.%20A%20rationale%20for%20tRNA%20modification%20circuits%20in%20the%20anticodon%20loop.%20RNA24,%201277%E2%80%931284%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Jones, J. D., Simcox, K. M., Kennedy, R. T. &amp; Koutmou, K. S. Direct sequencing of total tRNAs by LC-MS/MS. <em>RNA</em><strong>29</strong>, 1201–1214 (2023).
</cite> [<a href="https://doi.org/10.1261/rna.079656.123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10351886/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37169396/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jones,%20J.%20D.,%20Simcox,%20K.%20M.,%20Kennedy,%20R.%20T.%20&amp;%20Koutmou,%20K.%20S.%20Direct%20sequencing%20of%20total%20tRNAs%20by%20LC-MS/MS.%20RNA29,%201201%E2%80%931214%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Shaw, E. A. et al. Combining Nanopore direct RNA sequencing with genetics and mass spectrometry for analysis of T-loop base modifications across 42 yeast tRNA isoacceptors. <em>Nucleic Acids Res.</em><strong>52</strong>, 12074–12092 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkae796" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11514469/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39340295/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shaw,%20E.%20A.%20et%20al.%20Combining%20Nanopore%20direct%20RNA%20sequencing%20with%20genetics%20and%20mass%20spectrometry%20for%20analysis%20of%20T-loop%20base%20modifications%20across%2042%20yeast%20tRNA%20isoacceptors.%20Nucleic%20Acids%20Res.52,%2012074%E2%80%9312092%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>National Academies of Sciences, Engineering &amp; Medicine. Toward sequencing and mapping of RNA modifications: proceedings of a workshop–in brief. 10.17226/27149 (2023).</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/37523464/" class="usa-link">PubMed</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>White, L. K. et al. Comparative analysis of 43 distinct RNA modifications by nanopore tRNA sequencing. Preprint at <em>bioRxiv</em>10.1101/2024.07.23.604651 (2024).</cite>
</li>
<li id="CR87">
<span class="label">87.</span><cite>Li, H. et al. The sequence alignment/map format and SAMtools. <em>Bioinformatics</em><strong>25</strong>, 2078–2079 (2009).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2723002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20H.%20et%20al.%20The%20sequence%20alignment/map%20format%20and%20SAMtools.%20Bioinformatics25,%202078%E2%80%932079%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Li, H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. arXiv:1303.3997v2 [q-bio.GN]. (2013).</cite>
</li>
<li id="CR89">
<span class="label">89.</span><cite>Mölder, F. et al. Sustainable data analysis with Snakemake. <em>F1000Res.</em><strong>10</strong>, 33 (2021).
</cite> [<a href="https://doi.org/10.12688/f1000research.29032.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8114187/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34035898/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?M%C3%B6lder,%20F.%20et%20al.%20Sustainable%20data%20analysis%20with%20Snakemake.%20F1000Res.10,%2033%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>White, L. &amp; Hesselberth, J. rnabioco/aa-tRNA-seq: Zenodo release. 10.5281/zenodo.15653411 (2025).</cite>
</li>
<li id="CR91">
<span class="label">91.</span><cite>Monera, O. D., Sereda, T. J., Zhou, N. E., Kay, C. M. &amp; Hodges, R. S. Relationship of sidechain hydrophobicity and alpha-helical propensity on the stability of the single-stranded amphipathic alpha-helix. <em>J. Pept. Sci.</em><strong>1</strong>, 319–329 (1995).
</cite> [<a href="https://doi.org/10.1002/psc.310010507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9223011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Monera,%20O.%20D.,%20Sereda,%20T.%20J.,%20Zhou,%20N.%20E.,%20Kay,%20C.%20M.%20&amp;%20Hodges,%20R.%20S.%20Relationship%20of%20sidechain%20hydrophobicity%20and%20alpha-helical%20propensity%20on%20the%20stability%20of%20the%20single-stranded%20amphipathic%20alpha-helix.%20J.%20Pept.%20Sci.1,%20319%E2%80%93329%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368100/bin/41467_2025_62545_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (15.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368100/bin/41467_2025_62545_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (89.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368100/bin/41467_2025_62545_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (211.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368100/bin/41467_2025_62545_MOESM4_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (163.5MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Raw nanopore sequencing data generated in this study has been deposited as POD5 files in the European Nucleotide Archive under accession number <a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB90828" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJEB90828</a>. Specific links to access the ENA POD5 files are available at <a href="https://github.com/rnabioco/aa-tRNA-seq/blob/main/README.md" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/rnabioco/aa-tRNA-seq/blob/main/README.md</a>. Per-read current and dwell time metrics for synthetic tRNA libraries are available on Zenodo under 10.5281/zenodo.14194755 [10.5281/zenodo.14194755]. Source data supporting the findings of this study are provided with this paper. <a href="#Sec33" class="usa-link">Source data</a> are provided with this paper.</p>
<p>All code used for analysis and figure generation, as well as a reproducible pipeline for aa-tRNA-seq data processing, is available at the associated GitHub repository: <a href="https://github.com/rnabioco/aa-tRNA-seq" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/rnabioco/aa-tRNA-seq</a>. A permanent archive of the version used in this study is available on Zenodo under 10.5281/zenodo.15653411 [10.5281/zenodo.15653411]<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a></sup>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62545-9"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62545.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12368100/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12368100/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368100%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368100/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12368100/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12368100/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40835813/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12368100/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40835813/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12368100/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12368100/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="J1nSOTYfVMxniNd7MpU0HQ83hRuZbeVsqLykeepxzfhXWTS9KHqpTOT3IcsWLuX9">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
