
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Metabolomics and nutrient intake reveal metabolite–nutrient interactions in metabolic syndrome: insights from the Korean Genome and Epidemiology Study - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4B3D98AF38B9305B3D90047C0EC92.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="nutrj">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366009/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nutrition Journal">
<meta name="citation_title" content="Metabolomics and nutrient intake reveal metabolite–nutrient interactions in metabolic syndrome: insights from the Korean Genome and Epidemiology Study">
<meta name="citation_author" content="Minyeong Kim">
<meta name="citation_author_institution" content="Department of Food and Nutrition, Inha University, Incheon, Republic of Korea">
<meta name="citation_author" content="Suyeon Lee">
<meta name="citation_author_institution" content="Department of Food and Nutrition, Inha University, Incheon, Republic of Korea">
<meta name="citation_author" content="Junguk Hur">
<meta name="citation_author_institution" content="Department of Biomedical Sciences, School of Medicine and Health Sciences, University of North Dakota, Grand Forks, ND USA">
<meta name="citation_author" content="Dayeon Shin">
<meta name="citation_author_institution" content="Department of Food and Nutrition, Inha University, Incheon, Republic of Korea">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="24">
<meta name="citation_firstpage" content="128">
<meta name="citation_doi" content="10.1186/s12937-025-01189-3">
<meta name="citation_pmid" content="40830960">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366009/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366009/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366009/pdf/12937_2025_Article_1189.pdf">
<meta name="description" content="Despite advances in metabolomics, the complex relationship between metabolites and nutrient intake in metabolic syndrome (MetS) remains poorly understood in the Korean population. This study aimed to characterize the metabolomic profiles and ...">
<meta name="og:title" content="Metabolomics and nutrient intake reveal metabolite–nutrient interactions in metabolic syndrome: insights from the Korean Genome and Epidemiology Study">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Despite advances in metabolomics, the complex relationship between metabolites and nutrient intake in metabolic syndrome (MetS) remains poorly understood in the Korean population. This study aimed to characterize the metabolomic profiles and ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366009/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366009">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12937-025-01189-3"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12937_2025_Article_1189.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366009%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366009/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366009/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366009/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-nutrj.png" alt="Nutrition Journal logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nutrition Journal" title="Link to Nutrition Journal" shape="default" href="https://nutritionj.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nutr J</button></div>. 2025 Aug 19;24:128. doi: <a href="https://doi.org/10.1186/s12937-025-01189-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12937-025-01189-3</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nutr%20J%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nutr%20J%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nutr%20J%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nutr%20J%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Metabolomics and nutrient intake reveal metabolite–nutrient interactions in metabolic syndrome: insights from the Korean Genome and Epidemiology Study</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Minyeong Kim</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Minyeong Kim</span></h3>
<div class="p">
<sup>1</sup>Department of Food and Nutrition, Inha University, Incheon, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Minyeong Kim</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Suyeon Lee</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Suyeon Lee</span></h3>
<div class="p">
<sup>1</sup>Department of Food and Nutrition, Inha University, Incheon, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Suyeon Lee</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hur%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Junguk Hur</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Junguk Hur</span></h3>
<div class="p">
<sup>2</sup>Department of Biomedical Sciences, School of Medicine and Health Sciences, University of North Dakota, Grand Forks, ND USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hur%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Junguk Hur</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shin%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Dayeon Shin</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Dayeon Shin</span></h3>
<div class="p">
<sup>1</sup>Department of Food and Nutrition, Inha University, Incheon, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shin%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dayeon Shin</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Food and Nutrition, Inha University, Incheon, Republic of Korea </div>
<div id="Aff2">
<sup>2</sup>Department of Biomedical Sciences, School of Medicine and Health Sciences, University of North Dakota, Grand Forks, ND USA </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 6; Accepted 2025 Jul 24; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366009  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830960/" class="usa-link">40830960</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Despite advances in metabolomics, the complex relationship between metabolites and nutrient intake in metabolic syndrome (MetS) remains poorly understood in the Korean population.</p></section><section id="sec2"><h3 class="pmc_sec_title">Objective</h3>
<p id="Par2">This study aimed to characterize the metabolomic profiles and nutrient intake associated with MetS and to examine their relationships in the Ansan-Ansung cohort of the Korean Genome and Epidemiology Study (KoGES).</p></section><section id="sec3"><h3 class="pmc_sec_title">Methods</h3>
<p id="Par3">Data from 2,306 middle-aged adults (1,109 men and 1,197 women) in the KoGES Ansan-Ansung cohort were analyzed. Plasma metabolites were measured using liquid chromatography-mass spectrometry, identifying 135 metabolites. Nutrient intake was assessed using a validated semi-quantitative food frequency questionnaire covering 23 nutrients. MetS-associated metabolites and nutrients were identified using the Wilcoxon rank-sum test, logistic regression, partial least squares-discriminant analysis, and group least absolute shrinkage and selection operator analysis. Pathway enrichment analysis identified key metabolic pathways, and fixed-effects models were applied to assess metabolite–nutrient relationships based on MetS status.</p></section><section id="sec4"><h3 class="pmc_sec_title">Results</h3>
<p id="Par4">Eleven metabolites, including hexose (FC = 0.95, <em>P</em> = 7.04 × 10<sup>–54</sup>), alanine, and branched-chain amino acids, and three nutrients including fat, retinol, and cholesterol, were significantly associated with MetS (FC range = 0.87–0.93; all <em>P</em> &lt; 0.05). Pathway analysis highlighted disruptions in arginine biosynthesis and arginine–proline metabolism. The MetS group exhibited six unique metabolite–nutrient pairs that were not observed in the non-MetS group, including ‘isoleucine–fat,’ ‘isoleucine–P,’ ‘proline–fat,’ ‘leucine–fat,’ ‘leucine–P,’ and ‘valerylcarnitine–niacin.’ Notably, dysregulated metabolism of branched-chain amino acids, such as isoleucine and leucine, has been implicated in oxidative stress. Importantly, the stochastic gradient descent classifier achieved the best predictive performance among the eight machine learning models (area under the curve, AUC = 0.84), highlighting the robustness of classification based on metabolite data. However, the absence of external validation limits the generalizability of these findings.</p></section><section id="sec5"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par5">This comprehensive metabolomic analysis of the KoGES Ansan-Ansung cohort revealed distinct metabolic profiles and nutrient intake patterns associated with MetS, highlighting altered metabolite–nutrient relationships and disrupted metabolic pathways. These findings provide new insights into potential associations between metabolic phenotypes and dietary intake, which may help inform individualized dietary approaches related to MetS, such as branched-chain amino acids-restricted diets (valine, isoleucine, leucine), reduced intake of hexose-rich carbohydrates, and modulation of niacin-rich protein sources according to individual metabolic profiles.</p></section><section id="sec6"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12937-025-01189-3.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Metabolic syndrome, Metabolic biomarkers, Nutrients, Machine learning algorithms, Personalized nutrition, Precision nutrition</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par34">The prevention and management of metabolic syndrome (MetS) pose increasing challenges to public health due to its rising global prevalence [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. MetS is a cluster of metabolic abnormalities—including abdominal obesity, elevated blood pressure, high fasting glucose levels, hypertriglyceridemia, and low high-density lipoprotein (HDL) cholesterol—that collectively increase the risk of chronic conditions such as cardiovascular disease, type 2 diabetes, and chronic kidney disease [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. In Korea, the prevalence of MetS reached 24.9% in 2021, with a concurrent increase in diabetic kidney disease from 12.4% in 2015 to 25.4% in 2021 [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. This trend highlights the urgent need for effective strategies to detect and manage MetS, especially in high-risk populations.</p>
<p id="Par35">Recent studies have highlighted the relevance of metabolomics in uncovering the pathophysiology of MetS. Specific metabolites, such as phosphatidylcholines (PCs) and sphingolipids, are positively associated with lipid abnormalities, while acylcarnitines and amino acids are inversely correlated with obesity markers [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Notably, lysoPC a C18:2 has emerged as a potential biomarker associated with all five MetS components and linked to impaired glucose metabolism and increased cardiovascular risk [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>–<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. These findings suggest that metabolomic profiling could enhance early identification of individuals at risk of MetS.</p>
<p id="Par36">Lifestyle modifications, particularly dietary changes, are recommended as the first-line treatment strategy for managing MetS. Despite extensive research on dietary influences, establishing causal links between nutrient intake and MetS remains challenging due to inconsistent findings from clinical trials. For example, a literature review on single nutrients and MetS found that intervention trials showed that taking vitamin D and calcium supplements together was effective in patients with glucose intolerance, whereas randomized controlled trials reported inconsistent clinical improvements with vitamin D supplementation [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. Although specific dietary interventions, such as improving food quality [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>] or altering macronutrient distribution [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>], have shown promise in managing MetS and its components, optimal nutritional strategies for MetS prevention are still unclear.</p>
<p id="Par37">Current healthcare primarily focuses on treating patients after disease onset, which is both costly and less effective for managing chronic and late-stage diseases. Thus, as traditional healthcare approaches often fall short in addressing individual variability, there is growing emphasis on personalized strategies. Metabolomics offers a powerful tool for capturing molecular signatures reflective of diet and metabolic health, providing opportunities for precision nutrition and individualized risk prediction [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>, <a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. Precision nutrition involves tailoring dietary interventions based on metabolic responses to specific nutrients or dietary patterns. As noted in previous studies, metabolites associated with strict adherence to Mediterranean and Nordic diets—such as carnitines, sphingomyelins, and phospholipids—are derived from endogenous metabolism and have been linked to cardiometabolic risk markers including blood pressure and insulin resistance [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. These findings suggest the potential of metabolomics to inform tailored dietary strategies for managing complex conditions such as MetS. Furthermore, beyond informing interventions, metabolomic profiling can enhance the accuracy of dietary assessments themselves. Particularly in nutritional epidemiology, integrating metabolomic biomarkers can improve assessments beyond self-reporting methods [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. Considering the rising prevalence of MetS and its associated health risks, there is an urgent need for effective diagnostic and prognostic tools. Therefore, developing a prediction model based on serum metabolomic profiles represents a promising avenue for enhancing early detection and personalized risk stratification of MetS, ultimately improving clinical outcomes through tailored interventions [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>].</p>
<p id="Par38">Although previous studies have examined the associations between metabolic profiles and individual components of MetS in Korean populations [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>–<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>], comprehensive analyses directly linking MetS status to metabolite–nutrient interactions remain limited. Moreover, to our knowledge, no study has applied a machine learning approach using metabolite profiles to predict MetS within a Korean cohort. In addition, although some studies have examined the associations among MetS, metabolic profiles, and dietary intake, most of these investigations have primarily focused on non-Korean populations [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>–<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>].</p>
<p id="Par39">In view of the complexity of MetS as a multifactorial condition, integrating insights from multiple biological domains through advanced omics technologies is crucial for a deeper understanding of its underlying mechanisms. We hypothesize that MetS alters metabolite-nutrient interactions, with distinct profiles in Korean adults detectable via metabolomics. To address this gap, the present study investigates the associations between serum metabolite levels, nutrient intake, and MetS in middle-aged Korean adults. Additionally, it aims to develop and validate a prediction model for MetS based on metabolomic profiles, providing a robust tool for early identification and intervention strategies tailored to the Korean population.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Study design and participants</h3>
<p id="Par40">This study utilized data from the Korean Genome and Epidemiology Study (KoGES), conducted by the Korea National Institute of Health, which collects comprehensive information on demographic variables, health status, medical history, biochemical variables, genotypes, and metabolites [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. The baseline survey was conducted between 2001 and 2002, with follow-up surveys conducted biennially over a span of 20 years. Although KoGES is a prospective cohort study with repeated follow-up surveys, this study utilized data from a single time point (2005–2006) because plasma metabolite data were only available for the specific period. Therefore, this study was designed as a cross-sectional analysis and was limited to the 7,515 individuals who participated in 2005–2006. The response rate at the second follow-up was 76.0%, and no notable differences in baseline characteristics were observed between respondents and non-respondents (i.e., those who participated in the baseline survey only) [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>].</p>
<p id="Par41">A flowchart of the study population is presented in Fig. <a href="#Fig1" class="usa-link">1</a>. From the initial cohort of 7,515 participants, exclusions were made based on the following criteria: absence of plasma metabolite data (<em>n</em> = 5,085), missing dietary information (<em>n</em> = 14), incomplete MetS components data (<em>n</em> = 101), and missing demographic details, including sex, age, and body mass index (BMI; <em>n</em> = 9). Consequently, a total of 2,306 participants, comprising 1,109 men and 1,197 women, were included in the final analysis. This study was approved by the Institutional Review Board of Inha University on March 20, 2024 (protocol number: 240307-1A). Patients and members of the public were not involved in the design, conduct, reporting, or dissemination of this study.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366009_12937_2025_1189_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/730f/12366009/438884eb11fd/12937_2025_1189_Fig1_HTML.jpg" loading="lazy" id="MO1" height="759" width="708" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Flowchart illustrating the participant selection process</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Definition of MetS</h3>
<p id="Par42">The criteria for MetS diagnosis were based on the National Cholesterol Education Program Adult Treatment Panel III (NCEP ATP III) guidelines [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>], with waist circumference cutoffs established by the Korean Society for the Study of Obesity [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. The NCEP ATP III guidelines are still widely used in epidemiological studies and have been applied to diverse populations, including Korean cohorts. Because the international standard cutoffs for waist circumference have been reported to be inappropriate for Asians, including Koreans [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>], ethnicity-specific values were applied to accurately reflect the metabolic risk associated with abdominal obesity in the Korean population. MetS was defined as the presence of at least three of the following five risk factors: (1) waist circumference ≥ 90 cm for men and ≥ 85 cm for women, (2) triglyceride levels ≥ 150 mg/dL, (3) HDL cholesterol &lt; 40 mg/dL for men and &lt; 50 mg/dL for women, (4) systolic blood pressure ≥ 130 mmHg, diastolic blood pressure ≥ 85 mmHg, history of hypertension, use of antihypertensive medication, or ongoing hypertension treatment, and (5) fasting blood glucose &gt; 100 mg/dL, a diabetes diagnosis, ongoing diabetes treatment, insulin therapy, or oral diabetes medication.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Metabolite measurement</h3>
<p id="Par43">Targeted metabolomics was performed using electrospray ionization liquid chromatography–mass spectrometry (ESI-LC/MS) and tandem mass spectrometry (MS/MS) with the AbsoluteIDQ™ p180 kit (BIOCRATES Life Sciences AG, Innsbruck, Austria). This kit enables the quantification of 40 acylcarnitines, 21 amino acids, 19 biogenic amines, 1 hexose, 90 glycerophospholipids, and 15 sphingolipids. All measurements were carried out strictly following the manufacturer’s protocol (Manual UM-P180). The assay procedures of the AbsoluteIDQ™ p180 kit, as well as the metabolite nomenclature, have been described in detail previously [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. Briefly, 10 µL of serum was aliquoted onto a 96-well plate with a filter, followed by metabolite extraction. The extracts were analyzed by flow injection analysis–tandem mass spectrometry for acylcarnitines, hexose, glycerophospholipids, and sphingolipids in both positive (acylcarnitines, glycerophospholipids, and sphingolipids) and negative (hexose) ion modes. Amino acids and biogenic amines were quantified using liquid chromatography–tandem mass spectrometry in positive ion mode. Metabolite concentrations were automatically measured using the MetVal™ software package (BIOCRATES Life Sciences AG), with internal standards serving as references for concentration calculations, and were expressed in µM. The quality of metabolite data was assessed based on the following criteria: (1) coefficient of variation for each metabolite in the reference standards &lt; 25%, (2) at least half of the analyzed metabolite concentrations in the reference standards exceeding the limit of detection, and (3) at least half of the analyzed metabolite concentrations in the experimental samples exceeding the limit of detection [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. Quality control was performed by the KoGES team prior to data release, and only metabolites meeting predefined analytical standards were provided to researchers. Following the exclusion of 51 metabolites that did not meet quality control standards, a total of 135 metabolites were included in the final analysis. These comprised 13 acylcarnitines, 21 amino acids, 10 biogenic amines, 1 hexose, 78 glycerophospholipids, and 12 sphingolipids.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Dietary assessment</h3>
<p id="Par44">Dietary information was collected at baseline using a semi-quantitative food frequency questionnaire (FFQ), which included 106 food items and reference amounts commonly consumed by Koreans to assess the annual average intake frequency and quantity [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. The intake frequency of each food item was recorded at nine levels: almost never, once per month, two to three times per month, one to two times per week, three to four times per week, five to six times per week, and once, twice, or three times per day. In addition, intake amounts were classified as large, medium, or small. Daily nutrient intake was estimated using the Korean Food Composition Table [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>], which evaluated total energy intake in calories along with 22 nutrients—protein, fat, carbohydrate, calcium, phosphorus, iron (Fe), potassium, vitamin A, sodium, vitamin B1, vitamin B2, niacin, vitamin C, zinc, vitamin B6, folate, retinol, carotene, ash, fiber, vitamin E, and cholesterol. In this study, we included all 23 nutrients systematically assessed in the KoGES dataset, including total energy and the 22 macro- and micronutrients. These were selected not based on a hypothesis-driven approach but to comprehensively capture the full nutritional information available from the validated FFQ and food composition table.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Statistical analyses</h3>
<section id="Sec8"><h4 class="pmc_sec_title">Descriptive analysis</h4>
<p id="Par45">Descriptive summaries of the demographic and clinical characteristics were generated for participants with and without MetS. Differences in continuous variables between the MetS and non-MetS groups were assessed using the Wilcoxon rank-sum test, with results expressed as means and standard deviations. Categorical variables were analyzed using the chi-square test and presented as frequencies and percentages.</p></section></section><section id="Sec9"><h3 class="pmc_sec_title">Differential metabolite and nutrient identification</h3>
<p id="Par46">The intake of each metabolite and nutrient was log-transformed and standardized to a z-score. Four complementary methods—the Wilcoxon rank-sum test, partial least squares-discriminant analysis (PLS-DA), logistic regression, and group least absolute shrinkage and selection operator (LASSO)—were used to identify metabolites associated with MetS. Each method provides a unique perspective on the data and reveals distinct patterns of association. For nutrient intake, only the Wilcoxon rank-sum test and PLS-DA were applied, whereas all four methods were used for metabolite analysis. The Wilcoxon rank-sum test was used to evaluate univariate differences between the MetS and non-MetS groups using the R package <em>stats</em>. The PLS-DA, implemented using the R package <em>mixOmics</em>, was used to identify metabolites that best distinguished the two groups based on the first latent variable, with importance assessed by the variable importance in projection (VIP) score (VIP &gt; 1). Logistic regression and group LASSO were applied exclusively to metabolites, with logistic regression models adjusted for age, sex, and BMI. Covariates were selected based on their established roles as biological determinants of MetS. Age and sex are known to influence hormonal regulation, fat distribution, and insulin sensitivity, which are key factors in MetS pathophysiology [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Additionally, indicators of adiposity such as BMI have been consistently associated with MetS, insulin resistance, and metabolic profiles [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. <em>P</em>-values ​​were corrected for multiple comparisons using the Benjamini–Hochberg method. Group LASSO, implemented using the R package <em>glmnet</em>, further explored sub-pathway differences by mapping the metabolites to the Kyoto Encyclopedia of Genes and Genomes (KEGG) pathways [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>] and the Human Metabolome Database (HMDB, <a href="http://www.hmdb.ca" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.hmdb.ca</a>) [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. Group LASSO was employed to explore metabolite sub-pathway differences by matching metabolites to KEGG and HMDB pathways, resulting in the inclusion of 41 metabolites in the analysis. The analysis was conducted using matched metabolites as predictors (X) and MetS status as the outcome variable (Y), with metabolites grouped according to KEGG and HMDB pathways. A binary logistic regression model with group penalties was applied using the R package <em>gglasso</em>, and significant metabolites associated with MetS were identified based on the model coefficients.</p>
<p id="Par47">To compare the results across the different methods (Wilcoxon, logistic regression, PLS-DA, and group LASSO), the overlapping metabolites and nutrients identified by each model were visualized using Venn diagrams in the R <em>VennDetail</em> package. Each method was chosen for its distinct analytical strength: the Wilcoxon rank-sum test identified unadjusted group differences in metabolite and nutrient levels; logistic regression assessed adjusted associations with MetS while controlling for age, sex, and BMI; PLS-DA was employed to identify variables contributing to group separation; and group LASSO was applied to select entire sub-pathways, enabling pathway-level interpretation by accounting for within-pathway correlation structures. This approach allowed for the identification of variables that were consistently important across multiple methods, as well as those unique to specific models. The use of Venn diagrams facilitated the interpretation of intersections and differences between the methods, providing insights into their respective contributions to identifying key variables.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Pathway enrichment analysis</h3>
<p id="Par48">Pathway enrichment analysis was conducted using the KEGG database and MetaboAnalyst 6.0 (<a href="http://www.metaboanalyst.ca/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.metaboanalyst.ca/</a>). Metabolites were mapped to KEGG pathways using the R package <em>KEGGREST</em> to validate annotations and confirm their biological relevance to metabolic pathways, resulting in the inclusion of 41 annotated metabolites, comprising five acylcarnitines, 21 amino acids, 10 biogenic amines, and one hexose. Enrichment analysis was then performed on the significant metabolites identified by each method—Wilcoxon, logistic regression, PLS-DA, and group LASSO—using MetaboAnalyst 6.0 to determine their pathway associations.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Correlation analysis</h3>
<p id="Par49">Spearman’s rank correlation was used to assess the associations between metabolites and nutrients, with the results visualized using the R package <em>corrplot</em>. Correlation networks were constructed to explore the relationships among metabolites, nutrients, and MetS components, providing insights into their interconnections and potential influences.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Machine learning-based classification models</h3>
<p id="Par50">To comprehensively evaluate modeling approaches suitable for metabolite-based prediction of MetS, we selected eight machine learning (ML) algorithms based on their demonstrated suitability for metabolomics data, characterized by high dimensionality, multicollinearity, and complex biochemical interactions. Logistic regression (LR) and stochastic gradient descent (SGD) were included for their interpretability and scalability. These linear models are widely used in metabolomics due to their ability to handle large numbers of predictors while maintaining transparent model structures. Regularization allows LR and SGD to identify a parsimonious set of predictive metabolites, which facilitates biological interpretation and clinical translation [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. The support vector machine (SVM) was selected for its strong performance in classifying nonlinear relationships between metabolic features and disease states. Kernel-based SVMs have been shown to accurately discriminate metabolic profiles associated with MetS and related phenotypes by projecting complex patterns into higher-dimensional space [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. Random forest (RF) and AdaBoost (ADB) are ensemble learning algorithms that are particularly effective in metabolomics, where predictor variables are correlated and biological signals are embedded in noisy, heterogeneous data. RF identifies key metabolites through internal feature ranking, aiding biomarker discovery, while ADB enhances classification performance by focusing on samples with borderline metabolic signatures [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>, <a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. K-nearest neighbors (KNN) was included as a non-parametric model that classifies individuals based on the overall similarity of their metabolic profiles. KNN has demonstrated competitive performance in metabolomics-based MetS classification tasks, especially when using curated panels of metabolites with strong biological signals [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. Naïve Bayes (NB) is a probabilistic classification algorithm based on Bayes’ Theorem and the assumption of conditional independence among features. It offers fast computation and is known to perform relatively well, particularly when the training dataset is limited. In metabolomics, NB can serve as a lightweight baseline classifier for exploratory analyses or preliminary screening, especially when computational efficiency and interpretability are prioritized [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. Deep neural network (DNN) is suited to metabolomics data because it can capture complex, nonlinear interactions among metabolites through multilayer architectures. Their ability to learn hierarchical representations allows them to model intricate biological relationships embedded in omics data, which has led to superior classification performance in recent metabolomics and multi-omics disease classification studies [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. All models were implemented in R, version 4.4.1, utilizing established packages.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Data preprocessing and feature selection</h3>
<p id="Par51">A total of 186 metabolites were initially quantified using the AbsoluteIDQ™ p180 kit. Prior to model development, metabolites were filtered based on predefined quality control criteria: (1) a coefficient of variation (CV) &lt; 25% in reference standards, (2) ≥ 50% of reference standard concentrations exceeding the limit of detection, and (3) ≥ 50% of experimental sample concentrations exceeding the limit of detection. Based on these thresholds, 51 metabolites were excluded, and the remaining 135 metabolites were retained for analysis. All predictor variables were standardized using z-score normalization prior to analysis. The outcome variable was MetS status, defined according to the NCEP ATP III and the Korean Society for the Study of Obesity criteria. It was encoded as a binary variable (1 = MetS, 0 = non-MetS).</p></section><section id="Sec14"><h3 class="pmc_sec_title">Data partitioning and internal validation</h3>
<p id="Par52">The dataset was randomly partitioned into training (70%) and test (30%) subsets using stratified sampling to preserve the class distribution, which was implemented via the <em>createDataPartition()</em> function from the <em>caret</em> package. Model development and internal validation were conducted exclusively on the training set, while model performance was evaluated on the independent test set. External validation could not be performed due to the unavailability of an independent dataset. This limitation has been acknowledged and addressed in the discussion section.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Model implementation and configuration</h3>
<p id="Par53">All eight models were trained using consistent procedures and default or minimal-tuned hyperparameters as implemented in their respective R packages. SVM was implemented using the <em>e1071</em> package with a radial basis function kernel and probability estimates enabled. RF, developed via the <em>randomForest</em> package, was trained with 500 trees while retaining default settings for all other parameters. KNN was configured using the <em>kknn</em> package with <em>k</em> = 5 and a rectangular kernel. LR was fitted using the <em>glm</em> function with a binomial logistic regression model that included all available predictors. DNN, implemented via the <em>neuralnet</em> package, utilized two hidden layers with 5 and 10 neurons respectively, a logistic activation function, a training threshold of 0.25, and a maximum of 1 × 10<sup>6</sup> iterations. NB, also implemented using <em>e1071</em>, was applied with default settings. ADB was constructed using the <em>adabag</em> package with 100 boosting iterations (mfinal = 100) and boos set to TRUE. Finally, SGD was implemented as Lasso-penalized logistic regression using the <em>glmnet</em> package through <em>caret</em>, with the hyperparameters set to α = 1 and λ = 0.01. No formal hyperparameter optimization procedures were applied beyond manual specification of core model parameters for convergence and interpretability. No automated grid search or cross-validation-based tuning was conducted. Key parameters were set based on established defaults and literature to prioritize interpretability and convergence.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Model evaluation</h3>
<p id="Par54">Model performance was assessed on the independent hold-out test set using the area under the receiver operating characteristic curve (AUC), sensitivity, specificity, and overall accuracy (non-error rate). AUC confidence intervals were calculated using DeLong’s method, and ROC curves were generated with the <em>pROC</em> and <em>ggplot2</em> packages. A fixed classification threshold of 0.5 was applied consistently across all models [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>].</p></section><section id="Sec18"><h3 class="pmc_sec_title">Reporting quality assessment</h3>
<p id="Par56">To evaluate reporting quality, we utilized the TRIPOD + AI statement, which provides a comprehensive checklist of items that should be reported in studies describing the development and/or validation of predictive AI models [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. The checklist encompasses all major sections of a manuscript, including the title, abstract, introduction, methods, results, and discussion. In addition to these conventional sections, the TRIPOD + AI statement places particular emphasis on open science practices, patient and public involvement, and equity considerations. Each item's adherence was evaluated as “Yes,” “Partial,” “No,” or “Not Applicable” and was accompanied by the corresponding page number. The completed checklist is available in Supplementary Table 2.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Fixed-effects model for effects of nutrition</h3>
<p id="Par57">A fixed-effects model was applied to examine the interactions between metabolites and nutrients according to MetS status. First, linear regression analyses identified metabolites significantly associated with MetS (<em>P</em> &lt; 0.05). Second, relationships between these significant metabolites and nutrient intake were evaluated using the lm() function from the stats package in R, with fixed effects including MetS status and nutrient intake. The fixed-effects models were designed to examine nutrient–metabolite associations stratified by MetS status, without additional covariate adjustment, to isolate interaction patterns across strata. To focus on robust associations, we selected 11 metabolites consistently associated with MetS using four statistical approaches: the Wilcoxon test, logistic regression, PLS-DA, and cluster LASSO. In addition, metabolite-nutrient pairs significantly associated with nutrient intake were included. Finally, the dataset was stratified by MetS status to further evaluate how nutrient intake influences metabolite levels within each group.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Statistical implementation and modeling considerations</h3>
<p id="Par58">All <em>P</em>-values were ​​adjusted for the false discovery rate (FDR) using the Benjamini‒Hochberg method, with values less than 0.05 considered statistically significant. All statistical analyses, except for pathway enrichment analysis, were conducted using R, version 4.4.1 (R Foundation for Statistical Computing). Visualization was additionally performed using Cytoscape, version 3.10.3. Covariate adjustment was applied only in the logistic regression models to control for age, sex, and BMI, which are well‑established confounders in nutritional metabolomics. Other analytic methods—such as the Wilcoxon rank‑sum test, PLS‑DA, fixed-effects models, and machine learning classifiers—were conducted without covariate adjustment. This approach aligns with standard best practices, where univariate and dimension-reduction methods are employed to explore inherent group differences without introducing potential overadjustment bias [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>], while logistic regression is specifically designed to estimate adjusted associations. Machine learning models were similarly built using the full set of features for classification, focusing on predictive performance rather than inference, as is common in biomarker discovery applications.</p></section></section><section id="Sec21"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec22"><h3 class="pmc_sec_title">Clinical characteristics of the study population</h3>
<p id="Par59">A total of 838 participants with MetS and 1,468 non-MetS were included in this study. Table <a href="#Tab1" class="usa-link">1</a> presents the detailed demographic, biochemical, and anthropometric characteristics of the two groups. The mean age of the MetS group was significantly higher than that of the non-MetS group (58.33 ± 8.81 years vs. 55.62 ± 8.91 years, respectively). The proportion of males was higher in the non-MetS group than in the MetS group. BMI, waist circumference, systolic and diastolic blood pressure, and triglyceride and blood glucose levels were significantly higher in the MetS group, whereas HDL cholesterol levels were significantly lower. Furthermore, the prevalence of MetS-related phenotypes, including abdominal obesity, elevated blood pressure, elevated blood glucose, elevated triglycerides, and low HDL cholesterol levels, was significantly higher in the MetS group compared to the non-MetS group.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Characteristics of study participants stratified by the presence of metabolic syndrome</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Characteristics</th>
<th align="left" colspan="1" rowspan="1">MetS group (<em>n</em> = 838)</th>
<th align="left" colspan="1" rowspan="1">95% CI</th>
<th align="left" colspan="1" rowspan="1">non-MetS group (<em>n</em> = 1,468)</th>
<th align="left" colspan="1" rowspan="1">95% CI</th>
<th align="left" colspan="1" rowspan="1">
<em>P</em>-value</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Age (years)</td>
<td align="left" colspan="1" rowspan="1">58.33 ± 8.81</td>
<td align="left" colspan="1" rowspan="1">57.7–58.9</td>
<td align="left" colspan="1" rowspan="1">55.62 ± 8.91</td>
<td align="left" colspan="1" rowspan="1">55.2–56.1</td>
<td align="left" colspan="1" rowspan="1"> &lt; 0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Male (%)</td>
<td align="left" colspan="1" rowspan="1">371 (44.27)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">738 (50.27)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">0.0063</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">BMI (kg/m<sup>2</sup>)</td>
<td align="left" colspan="1" rowspan="1">26.45 ± 2.93</td>
<td align="left" colspan="1" rowspan="1">26.2–26.6</td>
<td align="left" colspan="1" rowspan="1">23.38 ± 2.82</td>
<td align="left" colspan="1" rowspan="1">23.2–23.5</td>
<td align="left" colspan="1" rowspan="1"> &lt; 0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Waist circumference (cm)</td>
<td align="left" colspan="1" rowspan="1">91.06 ± 7.52</td>
<td align="left" colspan="1" rowspan="1">90.6–91.6</td>
<td align="left" colspan="1" rowspan="1">81.40 ± 7.66</td>
<td align="left" colspan="1" rowspan="1">81.0–81.8</td>
<td align="left" colspan="1" rowspan="1"> &lt; 0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Systolic blood pressure (mmHg)</td>
<td align="left" colspan="1" rowspan="1">125.05 ± 15.94</td>
<td align="left" colspan="1" rowspan="1">124.0–126.1</td>
<td align="left" colspan="1" rowspan="1">113.47 ± 15.74</td>
<td align="left" colspan="1" rowspan="1">112.7–114.3</td>
<td align="left" colspan="1" rowspan="1"> &lt; 0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Diastolic blood pressure (mmHg)</td>
<td align="left" colspan="1" rowspan="1">83.13 ± 9.63</td>
<td align="left" colspan="1" rowspan="1">82.5–83.8</td>
<td align="left" colspan="1" rowspan="1">76.02 ± 9.80</td>
<td align="left" colspan="1" rowspan="1">75.5–76.5</td>
<td align="left" colspan="1" rowspan="1"> &lt; 0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Triglycerides (mg/dL)</td>
<td align="left" colspan="1" rowspan="1">212.25 ± 161.04</td>
<td align="left" colspan="1" rowspan="1">201.3–223.2</td>
<td align="left" colspan="1" rowspan="1">109.22 ± 54.18</td>
<td align="left" colspan="1" rowspan="1">106.4–112.0</td>
<td align="left" colspan="1" rowspan="1"> &lt; 0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Blood glucose (mg/dL)</td>
<td align="left" colspan="1" rowspan="1">104.89 ± 24.44</td>
<td align="left" colspan="1" rowspan="1">103.2–106.5</td>
<td align="left" colspan="1" rowspan="1">89.88 ± 13.91</td>
<td align="left" colspan="1" rowspan="1">89.2–90.6</td>
<td align="left" colspan="1" rowspan="1"> &lt; 0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HDL cholesterol (mg/dL)</td>
<td align="left" colspan="1" rowspan="1">39.00 ± 7.91</td>
<td align="left" colspan="1" rowspan="1">38.5–39.5</td>
<td align="left" colspan="1" rowspan="1">46.97 ± 10.39</td>
<td align="left" colspan="1" rowspan="1">46.4–47.5</td>
<td align="left" colspan="1" rowspan="1"> &lt; 0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Abdominal obesity (%)</td>
<td align="left" colspan="1" rowspan="1">625 (74.58)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">253 (17.23)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"> &lt; 0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Elevated blood pressure (%)</td>
<td align="left" colspan="1" rowspan="1">639 (76.25)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">409 (27.86)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"> &lt; 0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Elevated fasting glucose (%)</td>
<td align="left" colspan="1" rowspan="1">465 (55.49)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">249 (16.96)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"> &lt; 0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Elevated triglycerides (%)</td>
<td align="left" colspan="1" rowspan="1">558 (66.59)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">195 (13.28)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"> &lt; 0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Low HDL cholesterol (%)</td>
<td align="left" colspan="1" rowspan="1">684 (81.62)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">615 (41.89)</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"> &lt; 0.0001</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p34">
<p>Categorical variables are presented as numbers and percentages (%), whereas continuous variables are expressed as mean ± standard deviation <em>P</em>-values were calculated using the Wilcoxon rank-sum test for continuous variables and the chi-square test for categorical variables</p>
<p><em>BMI</em> body mass index, <em>HDL</em> high-density lipoprotein, <em>MetS</em> metabolic syndrome, <em>CI</em> confidence interval</p>
</div></div></section></section><section id="Sec23"><h3 class="pmc_sec_title">Identification of metabolic biomarkers</h3>
<p id="Par60">We matched 41 out of 135 metabolites to KEGG and HMDB pathways and identified those significantly associated with MetS using four different methods (Fig. <a href="#Fig3" class="usa-link">3</a>). A volcano plot was used to visualize the differences between the two groups based on MetS status using the Wilcoxon rank-sum test (Fig. <a href="#Fig2" class="usa-link">2</a>A). Among the 41 metabolites, 27 exhibited significant differences, with hexose and alanine showing the most pronounced differences, both of which were elevated in the MetS group (FC = 0.95 and 1.46, respectively). In addition, among the 23 nutrients analyzed, three—fat, retinol, and cholesterol—showed significant differences and were decreased in the MetS group (FC = 0.93, 0.87, and 0.91, respectively).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366009_12937_2025_1189_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/730f/12366009/988ce5eb0089/12937_2025_1189_Fig3_HTML.jpg" loading="lazy" id="MO2" height="291" width="708" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Overlap of metabolites and nutrients associated with metabolic syndrome across statistical methods. <strong>A</strong> Venn diagram showing the overlap of metabolites identified by Wilcoxon (yellow), adjusted logistic regression (blue), PLS-DA (green), and group LASSO (orange). <strong>B</strong> A separate Venn diagram illustrating the overlap of nutrients identified by Wilcoxon (yellow) and PLS-DA (green). LASSO, least absolute shrinkage and selection operator; PLS-DA, partial least squares-discriminant analysis</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366009_12937_2025_1189_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/730f/12366009/10ff33d3691e/12937_2025_1189_Fig2_HTML.jpg" loading="lazy" id="MO3" height="705" width="778" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Identification of metabolic biomarkers and disrupted pathways associated with metabolic syndrome. <strong>A</strong> Volcano plot of metabolites and nutrients identified using the Wilcoxon rank-sum test. The x-axis represents the log(fold-change), where fold-change indicates the relative abundance of each metabolite or nutrient in MetS cases compared with non-MetS. The y-axis represents − log10(adjusted <em>P</em>-value), with <em>P</em>-values corrected for multiple comparisons using the Benjamini–Hochberg method. Purple dots represent metabolites or nutrients with non-significant <em>P</em>-values, whereas green dots represent those with significant <em>P</em>-values. The horizontal red line denotes the significance threshold of 0.05. <strong>B</strong> VIP score plot showing the top metabolites and nutrients that differentiate MetS cases from non-MetS, as identified by PLS-DA. All metabolite and nutrient values were standardized (z-scores) before VIP analysis. MetS, metabolic syndrome; PLS-DA, partial least squares-discriminant analysis; VIP, variable importance in projection</p></figcaption></figure><p id="Par61">Differential metabolites and nutrients between the two groups were identified using PLS-DA with VIP values ​&gt; 1. Among the 41 metabolites and 23 nutrients, 14 metabolites and 10 nutrients effectively distinguished the MetS group from the non-MetS group. The top 14 metabolites and top 10 nutrients with the highest VIP scores and greatest contributions to the separation between the MetS and non-MetS groups were visualized in the VIP score plot (Fig. <a href="#Fig2" class="usa-link">2</a>B). Among the 14 metabolites, only glycine tended to decrease in the MetS group, whereas the other metabolites showed an increase. In contrast, the intake of all 10 nutrients was lower in the MetS group, with hexose having the highest VIP score among metabolites and fat being the most prominent nutrient.</p>
<p id="Par62">Among the 41 candidate metabolites, 14 exhibited significant differences between the MetS and non-MetS groups based on the Wilcoxon rank-sum test and had VIP values &gt; 1 (Table <a href="#Tab2" class="usa-link">2</a>). Similarly, among the 23 candidate nutrients, 10 were identified with VIP values ​​ &gt; 1, of which three—fat, retinol, and cholesterol—showed significant differences according to the Wilcoxon rank-sum test (Table <a href="#Tab3" class="usa-link">3</a>).</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Differential metabolites identified using the Wilcoxon rank-sum test and PLS-DA</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Metabolites</th>
<th align="left" colspan="2" rowspan="1">MetS group</th>
<th align="left" colspan="2" rowspan="1">non-MetS group</th>
<th align="left" rowspan="2" colspan="1">Trend</th>
<th align="left" rowspan="2" colspan="1">FC</th>
<th align="left" rowspan="2" colspan="1">VIP</th>
<th align="left" rowspan="2" colspan="1">
<em>P</em>-value</th>
<th align="left" rowspan="2" colspan="1">BH <em>P</em>-value</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Mean</th>
<th align="left" colspan="1" rowspan="1">SD</th>
<th align="left" colspan="1" rowspan="1">Mean</th>
<th align="left" colspan="1" rowspan="1">SD</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Hexose (µM)</td>
<td align="center" colspan="1" rowspan="1">5623.39</td>
<td align="center" colspan="1" rowspan="1">1498.80</td>
<td align="center" colspan="1" rowspan="1">4859.05</td>
<td align="center" colspan="1" rowspan="1">953.18</td>
<td align="left" colspan="1" rowspan="1">Up</td>
<td align="center" colspan="1" rowspan="1">0.95</td>
<td align="center" colspan="1" rowspan="1">2.58</td>
<td align="left" colspan="1" rowspan="1">7.04E-54</td>
<td align="left" colspan="1" rowspan="1">2.89E-52</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Alanine (µM)</td>
<td align="center" colspan="1" rowspan="1">546.55</td>
<td align="center" colspan="1" rowspan="1">111.96</td>
<td align="center" colspan="1" rowspan="1">477.00</td>
<td align="center" colspan="1" rowspan="1">108.22</td>
<td align="left" colspan="1" rowspan="1">Up</td>
<td align="center" colspan="1" rowspan="1">1.46</td>
<td align="center" colspan="1" rowspan="1">2.45</td>
<td align="left" colspan="1" rowspan="1">2.15E-46</td>
<td align="left" colspan="1" rowspan="1">4.40E-45</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Glutamate (µM)</td>
<td align="center" colspan="1" rowspan="1">187.81</td>
<td align="center" colspan="1" rowspan="1">81.30</td>
<td align="center" colspan="1" rowspan="1">161.31</td>
<td align="center" colspan="1" rowspan="1">76.93</td>
<td align="left" colspan="1" rowspan="1">Up</td>
<td align="center" colspan="1" rowspan="1">1.48</td>
<td align="center" colspan="1" rowspan="1">1.76</td>
<td align="left" colspan="1" rowspan="1">1.83E-29</td>
<td align="left" colspan="1" rowspan="1">2.50E-28</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Valine (µM)</td>
<td align="center" colspan="1" rowspan="1">230.96</td>
<td align="center" colspan="1" rowspan="1">39.87</td>
<td align="center" colspan="1" rowspan="1">212.30</td>
<td align="center" colspan="1" rowspan="1">39.23</td>
<td align="left" colspan="1" rowspan="1">Up</td>
<td align="center" colspan="1" rowspan="1">1.48</td>
<td align="center" colspan="1" rowspan="1">1.85</td>
<td align="left" colspan="1" rowspan="1">1.46E-26</td>
<td align="left" colspan="1" rowspan="1">1.50E-25</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">lysoPhosphatidylcholine acyl C16:0 (µM)</td>
<td align="center" colspan="1" rowspan="1">281.35</td>
<td align="center" colspan="1" rowspan="1">59.17</td>
<td align="center" colspan="1" rowspan="1">256.44</td>
<td align="center" colspan="1" rowspan="1">59.58</td>
<td align="left" colspan="1" rowspan="1">Up</td>
<td align="center" colspan="1" rowspan="1">1.51</td>
<td align="center" colspan="1" rowspan="1">1.69</td>
<td align="left" colspan="1" rowspan="1">3E-23</td>
<td align="left" colspan="1" rowspan="1">2.46E-22</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Hexadecanoylcarnitine (µM)</td>
<td align="center" colspan="1" rowspan="1">0.16</td>
<td align="center" colspan="1" rowspan="1">0.05</td>
<td align="center" colspan="1" rowspan="1">0.14</td>
<td align="center" colspan="1" rowspan="1">0.07</td>
<td align="left" colspan="1" rowspan="1">Up</td>
<td align="center" colspan="1" rowspan="1">1.90</td>
<td align="center" colspan="1" rowspan="1">1.23</td>
<td align="left" colspan="1" rowspan="1">6.91E-17</td>
<td align="left" colspan="1" rowspan="1">4.72E-16</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Valerylcarnitine (µM)</td>
<td align="center" colspan="1" rowspan="1">0.16</td>
<td align="center" colspan="1" rowspan="1">0.05</td>
<td align="center" colspan="1" rowspan="1">0.14</td>
<td align="center" colspan="1" rowspan="1">0.08</td>
<td align="left" colspan="1" rowspan="1">Up</td>
<td align="center" colspan="1" rowspan="1">1.35</td>
<td align="center" colspan="1" rowspan="1">1.30</td>
<td align="left" colspan="1" rowspan="1">1.06E-16</td>
<td align="left" colspan="1" rowspan="1">6.21E-16</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Isoleucine (µM)</td>
<td align="center" colspan="1" rowspan="1">87.18</td>
<td align="center" colspan="1" rowspan="1">19.62</td>
<td align="center" colspan="1" rowspan="1">80.48</td>
<td align="center" colspan="1" rowspan="1">18.14</td>
<td align="left" colspan="1" rowspan="1">Up</td>
<td align="center" colspan="1" rowspan="1">1.56</td>
<td align="center" colspan="1" rowspan="1">1.41</td>
<td align="left" colspan="1" rowspan="1">3.08E-16</td>
<td align="left" colspan="1" rowspan="1">1.58E-15</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Proline (µM)</td>
<td align="center" colspan="1" rowspan="1">174.27</td>
<td align="center" colspan="1" rowspan="1">48.14</td>
<td align="center" colspan="1" rowspan="1">160.51</td>
<td align="center" colspan="1" rowspan="1">47.92</td>
<td align="left" colspan="1" rowspan="1">Up</td>
<td align="center" colspan="1" rowspan="1">1.20</td>
<td align="center" colspan="1" rowspan="1">1.27</td>
<td align="left" colspan="1" rowspan="1">2.13E-15</td>
<td align="left" colspan="1" rowspan="1">9.71E-15</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Tyrosine (µM)</td>
<td align="center" colspan="1" rowspan="1">74.04</td>
<td align="center" colspan="1" rowspan="1">14.13</td>
<td align="center" colspan="1" rowspan="1">69.89</td>
<td align="center" colspan="1" rowspan="1">14.43</td>
<td align="left" colspan="1" rowspan="1">Up</td>
<td align="center" colspan="1" rowspan="1">1.20</td>
<td align="center" colspan="1" rowspan="1">1.19</td>
<td align="left" colspan="1" rowspan="1">9.12E-14</td>
<td align="left" colspan="1" rowspan="1">3.74E-13</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Leucine (µM)</td>
<td align="center" colspan="1" rowspan="1">182.04</td>
<td align="center" colspan="1" rowspan="1">35.94</td>
<td align="center" colspan="1" rowspan="1">171.05</td>
<td align="center" colspan="1" rowspan="1">33.47</td>
<td align="left" colspan="1" rowspan="1">Up</td>
<td align="center" colspan="1" rowspan="1">1.59</td>
<td align="center" colspan="1" rowspan="1">1.25</td>
<td align="left" colspan="1" rowspan="1">5.55E-13</td>
<td align="left" colspan="1" rowspan="1">2.07E-12</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Phenylalanine (µM)</td>
<td align="center" colspan="1" rowspan="1">104.79</td>
<td align="center" colspan="1" rowspan="1">16.71</td>
<td align="center" colspan="1" rowspan="1">99.73</td>
<td align="center" colspan="1" rowspan="1">16.97</td>
<td align="left" colspan="1" rowspan="1">Up</td>
<td align="center" colspan="1" rowspan="1">1.24</td>
<td align="center" colspan="1" rowspan="1">1.22</td>
<td align="left" colspan="1" rowspan="1">6.45E-13</td>
<td align="left" colspan="1" rowspan="1">2.21E-12</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Glycine (µM)</td>
<td align="center" colspan="1" rowspan="1">325.49</td>
<td align="center" colspan="1" rowspan="1">82.51</td>
<td align="center" colspan="1" rowspan="1">346.74</td>
<td align="center" colspan="1" rowspan="1">84.26</td>
<td align="left" colspan="1" rowspan="1">Down</td>
<td align="center" colspan="1" rowspan="1">1.27</td>
<td align="center" colspan="1" rowspan="1">1.11</td>
<td align="left" colspan="1" rowspan="1">5.53E-12</td>
<td align="left" colspan="1" rowspan="1">1.75E-11</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Propionylcarnitine (µM)</td>
<td align="center" colspan="1" rowspan="1">0.48</td>
<td align="center" colspan="1" rowspan="1">0.15</td>
<td align="center" colspan="1" rowspan="1">0.44</td>
<td align="center" colspan="1" rowspan="1">0.15</td>
<td align="left" colspan="1" rowspan="1">Up</td>
<td align="center" colspan="1" rowspan="1">1.22</td>
<td align="center" colspan="1" rowspan="1">1.07</td>
<td align="left" colspan="1" rowspan="1">1.7E-10</td>
<td align="left" colspan="1" rowspan="1">4.99E-10</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p41">
<p>The mean and SD for the MetS and non-MetS groups are presented, along with the trend (down: lower in the MetS group), FC, VIP score, <em>P</em>-value (unadjusted from the Wilcoxon rank-sum test), and BH <em>P</em>-value (adjusted for multiple comparisons using the BH method)</p>
<p>Mean and SD values represent raw metabolite concentrations (µM)</p>
<p><em>BH</em> Benjamini–Hochberg, <em>FC</em> fold change, <em>MetS</em> metabolic syndrome, <em>PLS-DA</em> partial least squares-discriminant analysis, <em>SD</em> standard deviation, <em>VIP</em> variable importance in projection</p>
</div></div></section><section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Differential nutrients identified using the Wilcoxon rank-sum test and PLS-DA</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Nutrients</th>
<th align="left" colspan="2" rowspan="1">MetS group</th>
<th align="left" colspan="2" rowspan="1">non-MetS group</th>
<th align="left" rowspan="2" colspan="1">Trend</th>
<th align="left" rowspan="2" colspan="1">FC</th>
<th align="left" rowspan="2" colspan="1">VIP</th>
<th align="left" rowspan="2" colspan="1">
<em>P</em>-value</th>
<th align="left" rowspan="2" colspan="1">BH <em>P</em>-value</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Mean</th>
<th align="left" colspan="1" rowspan="1">SD</th>
<th align="left" colspan="1" rowspan="1">Mean</th>
<th align="left" colspan="1" rowspan="1">SD</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Fat (g)</td>
<td align="center" colspan="1" rowspan="1">26.15</td>
<td align="center" colspan="1" rowspan="1">19.07</td>
<td align="center" colspan="1" rowspan="1">28.14</td>
<td align="center" colspan="1" rowspan="1">18.61</td>
<td align="left" colspan="1" rowspan="1">Down</td>
<td align="center" colspan="1" rowspan="1">0.93</td>
<td align="center" colspan="1" rowspan="1">1.82</td>
<td align="center" colspan="1" rowspan="1">0.0008</td>
<td align="center" colspan="1" rowspan="1">0.0100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Retinol (µg)</td>
<td align="center" colspan="1" rowspan="1">52.19</td>
<td align="center" colspan="1" rowspan="1">48.54</td>
<td align="center" colspan="1" rowspan="1">59.66</td>
<td align="center" colspan="1" rowspan="1">57.14</td>
<td align="left" colspan="1" rowspan="1">Down</td>
<td align="center" colspan="1" rowspan="1">0.87</td>
<td align="center" colspan="1" rowspan="1">1.58</td>
<td align="center" colspan="1" rowspan="1">0.0011</td>
<td align="center" colspan="1" rowspan="1">0.0100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cholesterol (mg)</td>
<td align="center" colspan="1" rowspan="1">134.95</td>
<td align="center" colspan="1" rowspan="1">113.35</td>
<td align="center" colspan="1" rowspan="1">147.98</td>
<td align="center" colspan="1" rowspan="1">125.02</td>
<td align="left" colspan="1" rowspan="1">Down</td>
<td align="center" colspan="1" rowspan="1">0.91</td>
<td align="center" colspan="1" rowspan="1">1.46</td>
<td align="center" colspan="1" rowspan="1">0.0013</td>
<td align="center" colspan="1" rowspan="1">0.0100</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Vitamin E (mg)</td>
<td align="center" colspan="1" rowspan="1">7.65</td>
<td align="center" colspan="1" rowspan="1">4.04</td>
<td align="center" colspan="1" rowspan="1">8.13</td>
<td align="center" colspan="1" rowspan="1">5.08</td>
<td align="left" colspan="1" rowspan="1">Down</td>
<td align="center" colspan="1" rowspan="1">0.94</td>
<td align="center" colspan="1" rowspan="1">1.32</td>
<td align="center" colspan="1" rowspan="1">0.0328</td>
<td align="center" colspan="1" rowspan="1">0.1509</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Potassium (mg)</td>
<td align="center" colspan="1" rowspan="1">2194.67</td>
<td align="center" colspan="1" rowspan="1">1029.01</td>
<td align="center" colspan="1" rowspan="1">2284.92</td>
<td align="center" colspan="1" rowspan="1">1046.23</td>
<td align="left" colspan="1" rowspan="1">Down</td>
<td align="center" colspan="1" rowspan="1">0.96</td>
<td align="center" colspan="1" rowspan="1">1.24</td>
<td align="center" colspan="1" rowspan="1">0.0286</td>
<td align="center" colspan="1" rowspan="1">0.1509</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Folate (µg)</td>
<td align="center" colspan="1" rowspan="1">209.13</td>
<td align="center" colspan="1" rowspan="1">115.66</td>
<td align="center" colspan="1" rowspan="1">219.16</td>
<td align="center" colspan="1" rowspan="1">124.00</td>
<td align="left" colspan="1" rowspan="1">Down</td>
<td align="center" colspan="1" rowspan="1">0.95</td>
<td align="center" colspan="1" rowspan="1">1.14</td>
<td align="center" colspan="1" rowspan="1">0.0495</td>
<td align="center" colspan="1" rowspan="1">0.1821</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Phosphorus (mg)</td>
<td align="center" colspan="1" rowspan="1">868.51</td>
<td align="center" colspan="1" rowspan="1">343.51</td>
<td align="center" colspan="1" rowspan="1">896.81</td>
<td align="center" colspan="1" rowspan="1">360.18</td>
<td align="left" colspan="1" rowspan="1">Down</td>
<td align="center" colspan="1" rowspan="1">0.97</td>
<td align="center" colspan="1" rowspan="1">1.11</td>
<td align="center" colspan="1" rowspan="1">0.0645</td>
<td align="center" colspan="1" rowspan="1">0.1821</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Fe (mg)</td>
<td align="center" colspan="1" rowspan="1">9.44</td>
<td align="center" colspan="1" rowspan="1">4.50</td>
<td align="center" colspan="1" rowspan="1">9.73</td>
<td align="center" colspan="1" rowspan="1">4.70</td>
<td align="left" colspan="1" rowspan="1">Down</td>
<td align="center" colspan="1" rowspan="1">0.97</td>
<td align="center" colspan="1" rowspan="1">1.06</td>
<td align="center" colspan="1" rowspan="1">0.0650</td>
<td align="center" colspan="1" rowspan="1">0.1821</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Calcium (mg)</td>
<td align="center" colspan="1" rowspan="1">410.93</td>
<td align="center" colspan="1" rowspan="1">241.09</td>
<td align="center" colspan="1" rowspan="1">428.57</td>
<td align="center" colspan="1" rowspan="1">257.10</td>
<td align="left" colspan="1" rowspan="1">Down</td>
<td align="center" colspan="1" rowspan="1">0.96</td>
<td align="center" colspan="1" rowspan="1">1.03</td>
<td align="center" colspan="1" rowspan="1">0.1153</td>
<td align="center" colspan="1" rowspan="1">0.1974</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Vitamin B2 (mg)</td>
<td align="center" colspan="1" rowspan="1">0.86</td>
<td align="center" colspan="1" rowspan="1">0.43</td>
<td align="center" colspan="1" rowspan="1">0.89</td>
<td align="center" colspan="1" rowspan="1">0.44</td>
<td align="left" colspan="1" rowspan="1">Down</td>
<td align="center" colspan="1" rowspan="1">0.96</td>
<td align="center" colspan="1" rowspan="1">1.03</td>
<td align="center" colspan="1" rowspan="1">0.0792</td>
<td align="center" colspan="1" rowspan="1">0.1821</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p45">
<p>The mean and SD for the MetS and non-MetS groups are presented, along with the trend (down: lower in the MetS group), FC, VIP score, <em>P</em>-value (unadjusted from the Wilcoxon rank-sum test), and BH <em>P</em>-value (adjusted for multiple comparisons using the BH method)</p>
<p><em>BH</em> Benjamini–Hochberg, <em>FC</em> fold change, <em>Fe</em> iron, <em>MetS </em>metabolic syndrome, <em>PLS-DA</em> partial least squares-discriminant analysis, <em>SD</em> standard deviation, <em>VIP</em> variable importance in projection</p>
</div></div></section><p id="Par63">Figure <a href="#Fig3" class="usa-link">3</a> presents Venn diagrams illustrating the overlap of metabolites and nutrients significantly associated with MetS, as identified using Wilcoxon, logistic regression, PLS-DA, and group LASSO. Nutrients were excluded from the logistic regression and group LASSO analyses owing to the lack of significant findings after adjustment, leaving the overlap based solely on Wilcoxon and PLS-DA results. The metabolite analysis identified 11 overlapping metabolites—hexose, alanine, glutamate, valine, isoleucine, proline, leucine, valerylcarnitine, glycine, propionylcarnitine, and phenylalanine—which accounted for 26.8% of the 41 metabolites analyzed and were consistently detected across all four analytical approaches. The nutrient analysis revealed three overlapping nutrients: fat, retinol, and cholesterol. This integrative approach provides a comprehensive understanding of the relationships between metabolites, nutrients, and MetS from multiple analytical perspectives.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Metabolic pathway enrichment analysis</h3>
<p id="Par64">Significantly enriched sub-pathways, derived from 11 metabolites identified by the Wilcoxon, adjusted logistic regression, PLS-DA, and group LASSO models, are illustrated as dot plots in Fig. <a href="#Fig4" class="usa-link">4</a>. ‘Arginine biosynthesis’ and ‘arginine and proline metabolism’ were particularly significant in the group LASSO analysis, indicating a strong association between these pathways and MetS (<em>P</em> = 1.37 × 10<sup>–65</sup>; <em>P</em> = 4.55 × 10<sup>–43</sup>, respectively). Furthermore, ‘glutathione metabolism,’ ‘porphyrin metabolism,’ ‘glyoxylate and dicarboxylate metabolism,’ ‘glycine, serine, and threonine metabolism,’ ‘primary bile acid biosynthesis,’ ‘valine, leucine, and isoleucine biosynthesis,’ and ‘valine, leucine, and isoleucine degradation’ were identified as significant sub-pathways by all four analytical methods. In contrast, pathways that were not significant are depicted with faded colors and smaller sizes, indicating their relatively lower importance in the interpretation process.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366009_12937_2025_1189_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/730f/12366009/5d9c984d9030/12937_2025_1189_Fig4_HTML.jpg" loading="lazy" id="MO4" height="696" width="778" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Pathway enrichment analysis of metabolites associated with metabolic syndrome. Significantly enriched KEGG pathways are shown in a dot plot, where the horizontal axis indicates the analysis methods, and the vertical axis lists the sub-pathway names. Dot color represents statistical significance (− log10(<em>P</em>-value)), and dot size indicates the number of matched metabolites. The pathways identified as significantly enriched are labeled with false discovery rate-corrected <em>p</em>-values and pathway impact scores. CoA, coenzyme A; KEGG, Kyoto Encyclopedia of Genes and Genomes; LASSO, least absolute shrinkage and selection operator; PLS-DA, partial least squares-discriminant analysis</p></figcaption></figure></section><section id="Sec25"><h3 class="pmc_sec_title">Associations between metabolic biomarkers and nutrients</h3>
<p id="Par65">Spearman’s correlation coefficients were calculated to explore the potential relationships between the 11 differential metabolites identified by all four methods and the 23 nutrients. The correlation coefficient matrix is ​​shown in Fig. <a href="#Fig5" class="usa-link">5</a>, with significant correlation coefficients ranging from − 0.097 to 0.162. Among the 11 metabolites, valine exhibited a significant positive correlation with all nutrients. Valerylcarnitine showed significant correlations with 12 nutrients, with only one, vitamin C, being negatively correlated. Propionylcarnitine was associated with six nutrients and showed a positive correlation with energy and carbohydrates. Proline demonstrated positive correlations with eight nutrients, whereas leucine and isoleucine were positively correlated with 18 and 17 nutrients, respectively. In contrast, glycine exhibited a significant negative correlation with 11 nutrients. Glutamate showed a weak but significant positive correlation with sodium, whereas phenylalanine and alanine did not show any significant correlations with any nutrient. Among all significant correlations, the highest coefficient was observed between valine and niacin (r = 0.16, <em>P</em> &lt; 0.0001). All computed Spearman’s correlation coefficients are presented in Supplementary Table 1.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366009_12937_2025_1189_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/730f/12366009/c9eba812c209/12937_2025_1189_Fig5_HTML.jpg" loading="lazy" id="MO5" height="354" width="777" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Correlation coefficient matrix of potential metabolic biomarkers and nutrients. All statistically significant correlations between metabolites (rows) and nutrients (columns) are color-coded, whereas insignificant correlations are left blank. Positive correlations are shown in blue, and negative correlations in red. <em>P</em>-values were adjusted for multiple comparisons using the Benjamini–Hochberg method. Ca, calcium; Fe, iron; K, potassium; Na, sodium; P, phosphorus; Vit, vitamin</p></figcaption></figure><p id="Par66">We subsequently examined the correlation network to assess the associations between nutrients, metabolites, and MetS components (Supplementary Fig. <a href="#Fig1" class="usa-link">1</a>). Overall, the associations between nutrients and metabolites, as well as between nutrients and MetS components, were generally weak. In contrast, a strong association was observed between MetS components and metabolites, with blood glucose and hexose levels showing the strongest positive correlation (r = 0.67, <em>P</em> &lt; 0.0001).</p></section><section id="Sec26"><h3 class="pmc_sec_title">Associations between metabolic biomarkers, nutrients, and MetS</h3>
<p id="Par67">A fixed-effects model was constructed to investigate potential interactions between nutrients and metabolites that differ by MetS status. Significant differences in metabolite–nutrient pairs were observed between the two groups. In the MetS group, 16 pairs showed significant correlations, with 6 pairs—‘isoleucine–fat,’ ‘isoleucine–P,’ ‘proline–fat,’ ‘leucine–fat,’ ‘leucine–P,’ and ‘valerylcarnitine–niacin’—being exclusive to this group. In contrast, the non-MetS group exhibited 23 significant pairs, including 13 unique ones, such as ‘hexose–vitamin B6,’ ‘hexose–fiber,’ ‘valine–folate,’ ‘valine–ash,’ ‘valine–fiber,’ ‘isoleucine–carbohydrate,’ ‘isoleucine–Fe,’ ‘leucine–Fe,’ ‘leucine–fiber,’ ‘valerylcarnitine–protein,’ ‘valerylcarnitine–carbohydrate,’ ‘glycine–vitamin B1,’ and ‘glycine–vitamin B6’ (Fig. <a href="#Fig6" class="usa-link">6</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366009_12937_2025_1189_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/730f/12366009/3774ce08c785/12937_2025_1189_Fig6_HTML.jpg" loading="lazy" id="MO6" height="749" width="778" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The effect of nutrients on metabolites based on the presence or absence of metabolic syndrome. <strong>A</strong> MetS group. <strong>B</strong> non-MetS group. Significant nutrient–metabolite associations (<em>P</em> &lt; 0.05) are represented by lines, with red indicating positive correlations and blue indicating negative correlations. The intensity of the line color corresponds to the magnitude of the estimated effect, with stronger associations depicted in more prominent colors. Ca, calcium; CHO, carbohydrate; Fe, iron; MetS, metabolic syndrome; P, phosphorus; Vit, vitamin</p></figcaption></figure><p id="Par68">A negative association between glycine and protein levels was observed in both the MetS (estimate = −0.615, <em>P</em> = 0.002) and non-MetS group (estimate = −0.486, <em>P</em> = 0.002). In the MetS group, positive associations were identified between isoleucine and niacin (estimate = 0.450, <em>P</em> &lt; 0.001), as well as between leucine and niacin (estimate = 0.436, <em>P</em> &lt; 0.001). In particular, isoleucine (estimate = 0.199, <em>P</em> = 0.008), proline (estimate = 0.177, <em>P</em> = 0.004), and leucine (estimate = 0.166, <em>P</em> = 0.015) showed significant positive associations with fat intake only in the MetS group. Conversely, in the non-MetS group, positive associations were observed between valerylcarnitine and protein (estimate = 0.340, <em>P</em> = 0.015), as well as between valerylcarnitine and carbohydrate (estimate = 0.398, <em>P</em> = 0.019). Furthermore, in the non-MetS group, niacin showed a positive association with hexose, valine, isoleucine, proline, and leucine <em>P</em> &lt; 0.005) (Table <a href="#Tab4" class="usa-link">4</a>).</p>
<section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Nutrient–metabolite associations stratified by the presence of metabolic syndrome</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Fixed effects</th>
<th align="left" colspan="1" rowspan="1">Estimate</th>
<th align="left" colspan="1" rowspan="1">SE</th>
<th align="left" colspan="1" rowspan="1">z</th>
<th align="left" colspan="1" rowspan="1">
<em>P</em>-value</th>
</tr></thead>
<tbody>
<tr><td align="left" colspan="5" rowspan="1"><strong>MetS</strong></td></tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>Hexose</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Niacin</td>
<td align="center" colspan="1" rowspan="1">0.205</td>
<td align="center" colspan="1" rowspan="1">0.082</td>
<td align="center" colspan="1" rowspan="1">2.504</td>
<td align="center" colspan="1" rowspan="1">0.012</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Vitamin B6</td>
<td align="center" colspan="1" rowspan="1">−0.177</td>
<td align="center" colspan="1" rowspan="1">0.121</td>
<td align="center" colspan="1" rowspan="1">−1.465</td>
<td align="center" colspan="1" rowspan="1">0.143</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fiber</td>
<td align="center" colspan="1" rowspan="1">0.005</td>
<td align="center" colspan="1" rowspan="1">0.073</td>
<td align="center" colspan="1" rowspan="1">0.069</td>
<td align="center" colspan="1" rowspan="1">0.945</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>Glutamate</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Ca</td>
<td align="center" colspan="1" rowspan="1">−0.045</td>
<td align="center" colspan="1" rowspan="1">0.032</td>
<td align="center" colspan="1" rowspan="1">−1.402</td>
<td align="center" colspan="1" rowspan="1">0.161</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>Valine</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fat</td>
<td align="center" colspan="1" rowspan="1">0.030</td>
<td align="center" colspan="1" rowspan="1">0.057</td>
<td align="center" colspan="1" rowspan="1">0.519</td>
<td align="center" colspan="1" rowspan="1">0.604</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Carbohydrate</td>
<td align="center" colspan="1" rowspan="1">−0.059</td>
<td align="center" colspan="1" rowspan="1">0.045</td>
<td align="center" colspan="1" rowspan="1">−1.311</td>
<td align="center" colspan="1" rowspan="1">0.190</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Niacin</td>
<td align="center" colspan="1" rowspan="1">0.288</td>
<td align="center" colspan="1" rowspan="1">0.075</td>
<td align="center" colspan="1" rowspan="1">3.860</td>
<td align="center" colspan="1" rowspan="1"> &lt; 0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Folate</td>
<td align="center" colspan="1" rowspan="1">−0.121</td>
<td align="center" colspan="1" rowspan="1">0.089</td>
<td align="center" colspan="1" rowspan="1">−1.367</td>
<td align="center" colspan="1" rowspan="1">0.172</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Ash</td>
<td align="center" colspan="1" rowspan="1">−0.084</td>
<td align="center" colspan="1" rowspan="1">0.055</td>
<td align="center" colspan="1" rowspan="1">−1.530</td>
<td align="center" colspan="1" rowspan="1">0.126</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fiber</td>
<td align="center" colspan="1" rowspan="1">0.064</td>
<td align="center" colspan="1" rowspan="1">0.097</td>
<td align="center" colspan="1" rowspan="1">0.664</td>
<td align="center" colspan="1" rowspan="1">0.507</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>Isoleucine</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Protein</td>
<td align="center" colspan="1" rowspan="1">0.015</td>
<td align="center" colspan="1" rowspan="1">0.191</td>
<td align="center" colspan="1" rowspan="1">0.080</td>
<td align="center" colspan="1" rowspan="1">0.936</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fat</td>
<td align="center" colspan="1" rowspan="1">0.199</td>
<td align="center" colspan="1" rowspan="1">0.075</td>
<td align="center" colspan="1" rowspan="1">2.641</td>
<td align="center" colspan="1" rowspan="1">0.008</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Carbohydrate</td>
<td align="center" colspan="1" rowspan="1">0.033</td>
<td align="center" colspan="1" rowspan="1">0.050</td>
<td align="center" colspan="1" rowspan="1">0.658</td>
<td align="center" colspan="1" rowspan="1">0.511</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> P</td>
<td align="center" colspan="1" rowspan="1">−0.385</td>
<td align="center" colspan="1" rowspan="1">0.151</td>
<td align="center" colspan="1" rowspan="1">−2.544</td>
<td align="center" colspan="1" rowspan="1">0.011</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fe</td>
<td align="center" colspan="1" rowspan="1">−0.140</td>
<td align="center" colspan="1" rowspan="1">0.092</td>
<td align="center" colspan="1" rowspan="1">−1.520</td>
<td align="center" colspan="1" rowspan="1">0.129</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Niacin</td>
<td align="center" colspan="1" rowspan="1">0.450</td>
<td align="center" colspan="1" rowspan="1">0.099</td>
<td align="center" colspan="1" rowspan="1">4.567</td>
<td align="center" colspan="1" rowspan="1"> &lt; 0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Ash</td>
<td align="center" colspan="1" rowspan="1">−0.003</td>
<td align="center" colspan="1" rowspan="1">0.0493</td>
<td align="center" colspan="1" rowspan="1">−0.069</td>
<td align="center" colspan="1" rowspan="1">0.945</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>Proline</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fat</td>
<td align="center" colspan="1" rowspan="1">0.177</td>
<td align="center" colspan="1" rowspan="1">0.061</td>
<td align="center" colspan="1" rowspan="1">2.891</td>
<td align="center" colspan="1" rowspan="1">0.004</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> P</td>
<td align="center" colspan="1" rowspan="1">−0.388</td>
<td align="center" colspan="1" rowspan="1">0.085</td>
<td align="center" colspan="1" rowspan="1">−4.557</td>
<td align="center" colspan="1" rowspan="1"> &lt; 0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Niacin</td>
<td align="center" colspan="1" rowspan="1">0.300</td>
<td align="center" colspan="1" rowspan="1">0.081</td>
<td align="center" colspan="1" rowspan="1">3.720</td>
<td align="center" colspan="1" rowspan="1"> &lt; 0.001</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>Leucine</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fat</td>
<td align="center" colspan="1" rowspan="1">0.166</td>
<td align="center" colspan="1" rowspan="1">0.068</td>
<td align="center" colspan="1" rowspan="1">2.430</td>
<td align="center" colspan="1" rowspan="1">0.015</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> P</td>
<td align="center" colspan="1" rowspan="1">−0.267</td>
<td align="center" colspan="1" rowspan="1">0.108</td>
<td align="center" colspan="1" rowspan="1">−2.466</td>
<td align="center" colspan="1" rowspan="1">0.014</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fe</td>
<td align="center" colspan="1" rowspan="1">−0.105</td>
<td align="center" colspan="1" rowspan="1">0.124</td>
<td align="center" colspan="1" rowspan="1">−0.842</td>
<td align="center" colspan="1" rowspan="1">0.400</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Niacin</td>
<td align="center" colspan="1" rowspan="1">0.436</td>
<td align="center" colspan="1" rowspan="1">0.088</td>
<td align="center" colspan="1" rowspan="1">4.937</td>
<td align="center" colspan="1" rowspan="1"> &lt; 0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Folate</td>
<td align="center" colspan="1" rowspan="1">−0.152</td>
<td align="center" colspan="1" rowspan="1">0.096</td>
<td align="center" colspan="1" rowspan="1">−1.589</td>
<td align="center" colspan="1" rowspan="1">0.112</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Ash</td>
<td align="center" colspan="1" rowspan="1">−0.013</td>
<td align="center" colspan="1" rowspan="1">0.057</td>
<td align="center" colspan="1" rowspan="1">−0.226</td>
<td align="center" colspan="1" rowspan="1">0.822</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fiber</td>
<td align="center" colspan="1" rowspan="1">0.086</td>
<td align="center" colspan="1" rowspan="1">0.109</td>
<td align="center" colspan="1" rowspan="1">0.795</td>
<td align="center" colspan="1" rowspan="1">0.427</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>Valerylcarnitine</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Energy</td>
<td align="center" colspan="1" rowspan="1">0.086</td>
<td align="center" colspan="1" rowspan="1">0.256</td>
<td align="center" colspan="1" rowspan="1">0.338</td>
<td align="center" colspan="1" rowspan="1">0.736</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Protein</td>
<td align="center" colspan="1" rowspan="1">−0.112</td>
<td align="center" colspan="1" rowspan="1">0.147</td>
<td align="center" colspan="1" rowspan="1">−0.762</td>
<td align="center" colspan="1" rowspan="1">0.446</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Carbohydrate</td>
<td align="center" colspan="1" rowspan="1">−0.063</td>
<td align="center" colspan="1" rowspan="1">0.164</td>
<td align="center" colspan="1" rowspan="1">−0.386</td>
<td align="center" colspan="1" rowspan="1">0.700</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Niacin</td>
<td align="center" colspan="1" rowspan="1">0.260</td>
<td align="center" colspan="1" rowspan="1">0.096</td>
<td align="center" colspan="1" rowspan="1">2.720</td>
<td align="center" colspan="1" rowspan="1">0.007</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Vitamin C</td>
<td align="center" colspan="1" rowspan="1">−0.117</td>
<td align="center" colspan="1" rowspan="1">0.047</td>
<td align="center" colspan="1" rowspan="1">−2.517</td>
<td align="center" colspan="1" rowspan="1">0.012</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>Glycine</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Energy</td>
<td align="center" colspan="1" rowspan="1">0.512</td>
<td align="center" colspan="1" rowspan="1">0.331</td>
<td align="center" colspan="1" rowspan="1">1.547</td>
<td align="center" colspan="1" rowspan="1">0.122</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Protein</td>
<td align="center" colspan="1" rowspan="1">−0.615</td>
<td align="center" colspan="1" rowspan="1">0.198</td>
<td align="center" colspan="1" rowspan="1">−3.110</td>
<td align="center" colspan="1" rowspan="1">0.002</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Carbohydrate</td>
<td align="center" colspan="1" rowspan="1">−0.264</td>
<td align="center" colspan="1" rowspan="1">0.212</td>
<td align="center" colspan="1" rowspan="1">−1.248</td>
<td align="center" colspan="1" rowspan="1">0.212</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fe</td>
<td align="center" colspan="1" rowspan="1">0.354</td>
<td align="center" colspan="1" rowspan="1">0.116</td>
<td align="center" colspan="1" rowspan="1">3.044</td>
<td align="center" colspan="1" rowspan="1">0.002</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Vitamin B1</td>
<td align="center" colspan="1" rowspan="1">−0.180</td>
<td align="center" colspan="1" rowspan="1">0.113</td>
<td align="center" colspan="1" rowspan="1">−1.591</td>
<td align="center" colspan="1" rowspan="1">0.112</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Vitamin B6</td>
<td align="center" colspan="1" rowspan="1">0.086</td>
<td align="center" colspan="1" rowspan="1">0.125</td>
<td align="center" colspan="1" rowspan="1">0.688</td>
<td align="center" colspan="1" rowspan="1">0.492</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>Phenylalanine</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fe</td>
<td align="center" colspan="1" rowspan="1">−0.190</td>
<td align="center" colspan="1" rowspan="1">0.099</td>
<td align="center" colspan="1" rowspan="1">−1.924</td>
<td align="center" colspan="1" rowspan="1">0.055</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Vitamin B6</td>
<td align="center" colspan="1" rowspan="1">0.353</td>
<td align="center" colspan="1" rowspan="1">0.104</td>
<td align="center" colspan="1" rowspan="1">3.394</td>
<td align="center" colspan="1" rowspan="1">0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Folate</td>
<td align="center" colspan="1" rowspan="1">−0.146</td>
<td align="center" colspan="1" rowspan="1">0.076</td>
<td align="center" colspan="1" rowspan="1">−1.932</td>
<td align="center" colspan="1" rowspan="1">0.054</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>non-MetS</strong></td></tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>Hexose</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Niacin</td>
<td align="center" colspan="1" rowspan="1">0.180</td>
<td align="center" colspan="1" rowspan="1">0.051</td>
<td align="center" colspan="1" rowspan="1">3.543</td>
<td align="center" colspan="1" rowspan="1"> &lt; 0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Vitamin B6</td>
<td align="center" colspan="1" rowspan="1">−0.198</td>
<td align="center" colspan="1" rowspan="1">0.074</td>
<td align="center" colspan="1" rowspan="1">−2.680</td>
<td align="center" colspan="1" rowspan="1">0.007</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fiber</td>
<td align="center" colspan="1" rowspan="1">0.104</td>
<td align="center" colspan="1" rowspan="1">0.045</td>
<td align="center" colspan="1" rowspan="1">2.335</td>
<td align="center" colspan="1" rowspan="1">0.020</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>Glutamate</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Ca</td>
<td align="center" colspan="1" rowspan="1">−0.009</td>
<td align="center" colspan="1" rowspan="1">0.027</td>
<td align="center" colspan="1" rowspan="1">−0.346</td>
<td align="center" colspan="1" rowspan="1">0.729</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>Valine</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fat</td>
<td align="center" colspan="1" rowspan="1">0.012</td>
<td align="center" colspan="1" rowspan="1">0.050</td>
<td align="center" colspan="1" rowspan="1">0.248</td>
<td align="center" colspan="1" rowspan="1">0.804</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Carbohydrate</td>
<td align="center" colspan="1" rowspan="1">0.041</td>
<td align="center" colspan="1" rowspan="1">0.039</td>
<td align="center" colspan="1" rowspan="1">1.058</td>
<td align="center" colspan="1" rowspan="1">0.290</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Niacin</td>
<td align="center" colspan="1" rowspan="1">0.189</td>
<td align="center" colspan="1" rowspan="1">0.064</td>
<td align="center" colspan="1" rowspan="1">2.973</td>
<td align="center" colspan="1" rowspan="1">0.003</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Folate</td>
<td align="center" colspan="1" rowspan="1">−0.192</td>
<td align="center" colspan="1" rowspan="1">0.070</td>
<td align="center" colspan="1" rowspan="1">−2.735</td>
<td align="center" colspan="1" rowspan="1">0.006</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Ash</td>
<td align="center" colspan="1" rowspan="1">−0.172</td>
<td align="center" colspan="1" rowspan="1">0.052</td>
<td align="center" colspan="1" rowspan="1">−3.309</td>
<td align="center" colspan="1" rowspan="1">0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fiber</td>
<td align="center" colspan="1" rowspan="1">0.233</td>
<td align="center" colspan="1" rowspan="1">0.076</td>
<td align="center" colspan="1" rowspan="1">3.085</td>
<td align="center" colspan="1" rowspan="1">0.002</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>Isoleucine</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Protein</td>
<td align="center" colspan="1" rowspan="1">0.116</td>
<td align="center" colspan="1" rowspan="1">0.142</td>
<td align="center" colspan="1" rowspan="1">0.820</td>
<td align="center" colspan="1" rowspan="1">0.412</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fat</td>
<td align="center" colspan="1" rowspan="1">0.044</td>
<td align="center" colspan="1" rowspan="1">0.064</td>
<td align="center" colspan="1" rowspan="1">0.682</td>
<td align="center" colspan="1" rowspan="1">0.495</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Carbohydrate</td>
<td align="center" colspan="1" rowspan="1">0.151</td>
<td align="center" colspan="1" rowspan="1">0.042</td>
<td align="center" colspan="1" rowspan="1">3.634</td>
<td align="center" colspan="1" rowspan="1"> &lt; 0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> P</td>
<td align="center" colspan="1" rowspan="1">−0.236</td>
<td align="center" colspan="1" rowspan="1">0.121</td>
<td align="center" colspan="1" rowspan="1">−1.955</td>
<td align="center" colspan="1" rowspan="1">0.051</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fe</td>
<td align="center" colspan="1" rowspan="1">−0.177</td>
<td align="center" colspan="1" rowspan="1">0.071</td>
<td align="center" colspan="1" rowspan="1">−2.498</td>
<td align="center" colspan="1" rowspan="1">0.013</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Niacin</td>
<td align="center" colspan="1" rowspan="1">0.239</td>
<td align="center" colspan="1" rowspan="1">0.073</td>
<td align="center" colspan="1" rowspan="1">3.295</td>
<td align="center" colspan="1" rowspan="1">0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Ash</td>
<td align="center" colspan="1" rowspan="1">−0.006</td>
<td align="center" colspan="1" rowspan="1">0.043</td>
<td align="center" colspan="1" rowspan="1">−0.135</td>
<td align="center" colspan="1" rowspan="1">0.892</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>Proline</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fat</td>
<td align="center" colspan="1" rowspan="1">0.078</td>
<td align="center" colspan="1" rowspan="1">0.054</td>
<td align="center" colspan="1" rowspan="1">1.436</td>
<td align="center" colspan="1" rowspan="1">0.151</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> P</td>
<td align="center" colspan="1" rowspan="1">−0.240</td>
<td align="center" colspan="1" rowspan="1">0.071</td>
<td align="center" colspan="1" rowspan="1">−3.376</td>
<td align="center" colspan="1" rowspan="1">0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Niacin</td>
<td align="center" colspan="1" rowspan="1">0.215</td>
<td align="center" colspan="1" rowspan="1">0.066</td>
<td align="center" colspan="1" rowspan="1">3.273</td>
<td align="center" colspan="1" rowspan="1">0.001</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>Leucine</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fat</td>
<td align="center" colspan="1" rowspan="1">0.093</td>
<td align="center" colspan="1" rowspan="1">0.059</td>
<td align="center" colspan="1" rowspan="1">1.581</td>
<td align="center" colspan="1" rowspan="1">0.114</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> P</td>
<td align="center" colspan="1" rowspan="1">−0.082</td>
<td align="center" colspan="1" rowspan="1">0.087</td>
<td align="center" colspan="1" rowspan="1">−0.935</td>
<td align="center" colspan="1" rowspan="1">0.350</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fe</td>
<td align="center" colspan="1" rowspan="1">−0.259</td>
<td align="center" colspan="1" rowspan="1">0.097</td>
<td align="center" colspan="1" rowspan="1">−2.672</td>
<td align="center" colspan="1" rowspan="1">0.008</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Niacin</td>
<td align="center" colspan="1" rowspan="1">0.311</td>
<td align="center" colspan="1" rowspan="1">0.068</td>
<td align="center" colspan="1" rowspan="1">4.591</td>
<td align="center" colspan="1" rowspan="1"> &lt; 0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Folate</td>
<td align="center" colspan="1" rowspan="1">−0.096</td>
<td align="center" colspan="1" rowspan="1">0.072</td>
<td align="center" colspan="1" rowspan="1">−1.325</td>
<td align="center" colspan="1" rowspan="1">0.185</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Ash</td>
<td align="center" colspan="1" rowspan="1">−0.097</td>
<td align="center" colspan="1" rowspan="1">0.052</td>
<td align="center" colspan="1" rowspan="1">−1.888</td>
<td align="center" colspan="1" rowspan="1">0.059</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fiber</td>
<td align="center" colspan="1" rowspan="1">0.228</td>
<td align="center" colspan="1" rowspan="1">0.081</td>
<td align="center" colspan="1" rowspan="1">2.812</td>
<td align="center" colspan="1" rowspan="1">0.005</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>Valerylcarnitine</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Energy</td>
<td align="center" colspan="1" rowspan="1">−0.519</td>
<td align="center" colspan="1" rowspan="1">0.265</td>
<td align="center" colspan="1" rowspan="1">−1.960</td>
<td align="center" colspan="1" rowspan="1">0.050</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Protein</td>
<td align="center" colspan="1" rowspan="1">0.340</td>
<td align="center" colspan="1" rowspan="1">0.139</td>
<td align="center" colspan="1" rowspan="1">2.444</td>
<td align="center" colspan="1" rowspan="1">0.015</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Carbohydrate</td>
<td align="center" colspan="1" rowspan="1">0.398</td>
<td align="center" colspan="1" rowspan="1">0.170</td>
<td align="center" colspan="1" rowspan="1">2.342</td>
<td align="center" colspan="1" rowspan="1">0.019</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Niacin</td>
<td align="center" colspan="1" rowspan="1">0.027</td>
<td align="center" colspan="1" rowspan="1">0.079</td>
<td align="center" colspan="1" rowspan="1">0.341</td>
<td align="center" colspan="1" rowspan="1">0.733</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Vitamin C</td>
<td align="center" colspan="1" rowspan="1">−0.238</td>
<td align="center" colspan="1" rowspan="1">0.041</td>
<td align="center" colspan="1" rowspan="1">−5.856</td>
<td align="center" colspan="1" rowspan="1"> &lt; 0.001</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>Glycine</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Energy</td>
<td align="center" colspan="1" rowspan="1">0.396</td>
<td align="center" colspan="1" rowspan="1">0.274</td>
<td align="center" colspan="1" rowspan="1">1.444</td>
<td align="center" colspan="1" rowspan="1">0.149</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Protein</td>
<td align="center" colspan="1" rowspan="1">−0.486</td>
<td align="center" colspan="1" rowspan="1">0.158</td>
<td align="center" colspan="1" rowspan="1">−3.077</td>
<td align="center" colspan="1" rowspan="1">0.002</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Carbohydrate</td>
<td align="center" colspan="1" rowspan="1">−0.274</td>
<td align="center" colspan="1" rowspan="1">0.177</td>
<td align="center" colspan="1" rowspan="1">−1.547</td>
<td align="center" colspan="1" rowspan="1">0.122</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fe</td>
<td align="center" colspan="1" rowspan="1">0.206</td>
<td align="center" colspan="1" rowspan="1">0.087</td>
<td align="center" colspan="1" rowspan="1">2.367</td>
<td align="center" colspan="1" rowspan="1">0.018</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Vitamin B1</td>
<td align="center" colspan="1" rowspan="1">−0.214</td>
<td align="center" colspan="1" rowspan="1">0.084</td>
<td align="center" colspan="1" rowspan="1">−2.557</td>
<td align="center" colspan="1" rowspan="1">0.011</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Vitamin B6</td>
<td align="center" colspan="1" rowspan="1">0.304</td>
<td align="center" colspan="1" rowspan="1">0.093</td>
<td align="center" colspan="1" rowspan="1">3.260</td>
<td align="center" colspan="1" rowspan="1">0.001</td>
</tr>
<tr><td align="left" colspan="5" rowspan="1"><strong>Phenylalanine</strong></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Fe</td>
<td align="center" colspan="1" rowspan="1">−0.130</td>
<td align="center" colspan="1" rowspan="1">0.082</td>
<td align="center" colspan="1" rowspan="1">−1.586</td>
<td align="center" colspan="1" rowspan="1">0.113</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Vitamin B6</td>
<td align="center" colspan="1" rowspan="1">0.238</td>
<td align="center" colspan="1" rowspan="1">0.085</td>
<td align="center" colspan="1" rowspan="1">2.782</td>
<td align="center" colspan="1" rowspan="1">0.005</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> Folate</td>
<td align="center" colspan="1" rowspan="1">−0.107</td>
<td align="center" colspan="1" rowspan="1">0.061</td>
<td align="center" colspan="1" rowspan="1">−1.741</td>
<td align="center" colspan="1" rowspan="1">0.082</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p58">
<p>Data were analyzed using a fixed-effects model to account for individual variability, with significant associations defined as <em>P</em> &lt; 0.05</p>
<p><em>Ca</em> calcium, <em>Fe</em> iron, <em>MetS</em> metabolic syndrome, <em>P</em> phosphorus, <em>SE</em> standard error</p>
</div></div></section></section><section id="Sec27"><h3 class="pmc_sec_title">Machine learning-based diagnostic models for MetS using serum metabolomics data: a comparative analysis of eight classification algorithms</h3>
<p id="Par69">Using data from 135 metabolites, classification models were developed using eight machine learning algorithms: SVM, RF, KNN, LR, DNN, NB, ADB, and SGD. The classification performance of these models is presented in Table <a href="#Tab5" class="usa-link">5</a> and illustrated in Fig. <a href="#Fig7" class="usa-link">7</a>. Among the eight algorithms, SGD achieved the highest performance, with an AUC of 0.84 (95% confidence interval: 0.81–0.87). The AUC values across models ranged from 0.74 to 0.84, demonstrating their robustness in distinguishing MetS cases from non-MetS.</p>
<section class="tw xbox font-sm" id="Tab5"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Comparative performance metrics of eight machine learning algorithms for metabolic syndrome classification using serum metabolomics</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Models</th>
<th align="left" colspan="1" rowspan="1">Sensitivity</th>
<th align="left" colspan="1" rowspan="1">Specificity</th>
<th align="left" colspan="1" rowspan="1">Non-error Rate</th>
<th align="left" colspan="1" rowspan="1">AUC (95% CI)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Support vector machine</td>
<td align="center" colspan="1" rowspan="1">0.394</td>
<td align="center" colspan="1" rowspan="1">0.145</td>
<td align="center" colspan="1" rowspan="1">0.236</td>
<td align="center" colspan="1" rowspan="1">0.835 (0.804–0.865)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Random forest</td>
<td align="center" colspan="1" rowspan="1">0.510</td>
<td align="center" colspan="1" rowspan="1">0.877</td>
<td align="center" colspan="1" rowspan="1">0.744</td>
<td align="center" colspan="1" rowspan="1">0.812 (0.779–0.845)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">K-nearest neighbors</td>
<td align="center" colspan="1" rowspan="1">0.462</td>
<td align="center" colspan="1" rowspan="1">0.841</td>
<td align="center" colspan="1" rowspan="1">0.703</td>
<td align="center" colspan="1" rowspan="1">0.749 (0.713–0.786)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Logistic regression</td>
<td align="center" colspan="1" rowspan="1">0.614</td>
<td align="center" colspan="1" rowspan="1">0.809</td>
<td align="center" colspan="1" rowspan="1">0.738</td>
<td align="center" colspan="1" rowspan="1">0.819 (0.788–0.851)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Deep neural network</td>
<td align="center" colspan="1" rowspan="1">0.327</td>
<td align="center" colspan="1" rowspan="1">0.282</td>
<td align="center" colspan="1" rowspan="1">0.298</td>
<td align="center" colspan="1" rowspan="1">0.744 (0.705–0.783)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Naive Bayes</td>
<td align="center" colspan="1" rowspan="1">0.705</td>
<td align="center" colspan="1" rowspan="1">0.705</td>
<td align="center" colspan="1" rowspan="1">0.705</td>
<td align="center" colspan="1" rowspan="1">0.749 (0.712–0.786)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AdaBoost</td>
<td align="center" colspan="1" rowspan="1">0.606</td>
<td align="center" colspan="1" rowspan="1">0.841</td>
<td align="center" colspan="1" rowspan="1">0.755</td>
<td align="center" colspan="1" rowspan="1">0.822 (0.790–0.853)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Stochastic gradient descent</td>
<td align="center" colspan="1" rowspan="1">0.653</td>
<td align="center" colspan="1" rowspan="1">0.866</td>
<td align="center" colspan="1" rowspan="1">0.789</td>
<td align="center" colspan="1" rowspan="1">0.841 (0.811–0.870)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p62"><p><em>AUC</em> area under the curve, <em>CI</em> confidence interval</p></div></div></section><figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366009_12937_2025_1189_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/730f/12366009/63b524487fc0/12937_2025_1189_Fig7_HTML.jpg" loading="lazy" id="MO7" height="580" width="709" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>ROC curves of eight machine learning algorithms for metabolic syndrome classification using serum metabolomics profiles. ADB, AdaBoost; ROC, receiver operating characteristic; AUC, area under the curve; DNN, deep neural network; KNN, k-nearest neighbors; LR, logistic regression; NB, Naive Bayes; RF, random forest; SGD, stochastic gradient descent; SVM, support vector machine</p></figcaption></figure></section></section><section id="Sec28"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par70">In this study, we identified specific metabolic and nutritional markers associated with MetS in middle-aged Korean adults, highlighting disrupted metabolic pathways that may be relevant for informing future dietary recommendations. A comprehensive analysis using multiple statistical models identified 11 metabolites and three nutrients that were differentially abundant in MetS. Notably, the ‘arginine biosynthesis’ and ‘arginine and proline metabolism’ pathways were significantly altered. This study identified novel, MetS-specific nutrient–metabolite relationships through an integrated network analysis, shedding light on the complex interactions between diet and metabolism. Furthermore, incorporating machine learning into metabolomics enabled the characterization of key metabolic markers, offering a new perspective on predictive modeling in a large Korean cohort.</p>
<p id="Par71">Among the 11 metabolites consistently identified across all four methods, hexose emerged as the most prominent, exhibiting a positive correlation with MetS. A previous study in older cohorts from Japan and the United States also reported a similar association between serum hexose levels and MetS [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. Among hexoses, glucose functions as the primary energy source in living organisms. During glucose metabolism, a fraction enters the hexosamine biosynthesis pathway (HBP), a relatively minor branch of glycolysis. In this pathway, fructose-6-phosphate is converted to glucosamine-6-phosphate by the rate-limiting enzyme glutamine:fructose-6-phosphate amidotransferase (GFAT), which is subsequently converted to UDP-N-acetylglucosamine (UDP-GlcNAc), the major end product. UDP-GlcNAc is used for O-GlcNAcylation of intracellular proteins at serine/threonine residues, and elevated O-GlcNAcylation has been implicated in insulin resistance and vascular complications of diabetes [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. In our study, hexose showed a strong correlation with blood glucose levels, a key component of MetS. This finding aligns with a previous KoGES-based study that reported a significant association between serum hexoses and the risk of developing type 2 diabetes [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>].</p>
<p id="Par72">The levels of branched-chain and aromatic amino acids (BCAA and AAA), including leucine, isoleucine, valine, tyrosine, and phenylalanine, were significantly higher in the MetS group. A study conducted in a Mediterranean male population reported significantly higher BCAA and AAA levels in individuals with MetS [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Similarly, a Chinese study reported a 2.17-fold higher prevalence of MetS among individuals with the highest BCAA levels [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. These patterns suggest that elevated levels of BCAA and AAA reflect dysregulated metabolic pathways associated with MetS<strong>.</strong></p>
<p id="Par73">Short-chain acylcarnitines, including valerylcarnitine and propionylcarnitine, were positively associated with MetS, consistent with previous findings [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. As both are byproducts of BCAA catabolism, their elevated levels in our study may reflect the increased BCAA concentrations observed in the MetS group. Among the 11 candidate metabolites, glycine was the only one significantly lower in individuals with MetS. This finding aligns with consistent reports of reduced plasma glycine levels in MetS among older Chinese adults [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>].</p>
<p id="Par74">Regarding nutrient intake, the MetS group had lower levels of fat, retinol, and cholesterol consumption. While some studies have reported higher fat intake among individuals with MetS, these contrasting findings may reflect cultural dietary differences, particularly in the types of fats commonly consumed and the overall dietary patterns, such as whether the primary fat sources are plant-based or animal-based, across populations. This suggests that not only the quantity of fat, but also its quality, may vary across cultures and may be differentially associated with MetS risk across populations. A systematic review highlighted that replacing saturated fats with unsaturated fats reduces the risk of MetS [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>], and in Korean adults, very low fat intake—especially below 15% of total energy—has been associated with an increased prevalence of MetS [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. Population-specific dietary patterns, such as high intake of fermented vegetables and legumes, and low consumption of processed meats and dairy, contrast sharply with Western diets, which are generally richer in saturated fats and refined sugars. These findings underscore the importance of considering culturally specific dietary patterns when examining the associations between nutrient intake and MetS, suggesting that both nutrient quality and overall dietary context may be linked to MetS risk in population-specific ways. Therefore, the observed lower fat intake among Korean adults with MetS may reflect complex interactions between dietary quality, cultural habits, and disease risk, which may guide future research or inform hypothesis generation regarding population-specific recommendations. In addition to macronutrients, bioactive food components such as phytochemicals are emerging as important contributors to metabolic health. Although the KoGES dataset does not explicitly include variables for flavonoids and polyphenols—compounds known to modulate key metabolic pathways—their intake can be indirectly estimated from patterns of phytochemical-rich food consumption [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>]. Given their abundance in plant-based diets and well-documented metabolic benefits, these compounds warrant further investigation in future research to profile their associated metabolomics, which may reveal key biochemical pathways involved in the development of MetS. Specifically, elucidating the interactions between phytochemical intake and metabolite concentrations may clarify how these bioactive compounds modulate glucose and lipid metabolism, inflammation, and oxidative stress, thereby identifying novel interactions that influence MetS risk or protection and offering new avenues for precision nutrition.</p>
<p id="Par75">The ‘arginine biosynthesis’ and ‘arginine and proline metabolism’ pathways exhibited the strongest associations with MetS in the group LASSO model. These pathways are closely linked to vascular function through their role in nitric oxide (NO) production. Clinical studies have reported that insufficient L-arginine levels in obese adolescent patients with MetS may impair NO production, which plays a crucial role in vasodilation and blood pressure regulation [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. Decreased L-arginine has also been associated with vascular dysfunction and key MetS features such as hypertension, impaired glucose tolerance, and obesity [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>, <a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. Although antihypertensive medications were included in the diagnostic criteria for MetS and therefore could not be treated as confounding variables in this study, future studies aiming to elucidate the role of arginine-related pathways in MetS more precisely should consider including medications related to vascular function and blood pressure as potential confounders. These findings support a potential link between arginine-related metabolism and metabolic alterationsin MetS, with possible implications for cardiovascular health.</p>
<p id="Par76">In addition to the arginine pathway, several other metabolically enriched pathways identified in this study provide further insight into the pathophysiology of MetS. Glutathione metabolism was enriched across multiple analytical models and may indicate redox imbalance as a contributing factor to insulin resistance and vascular dysfunction [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. Likewise, the glycine, serine, and threonine metabolism pathways play important roles in antioxidant defense, one-carbon metabolism, and glucose regulation [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>, <a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. The lower levels of glycine observed in MetS participants may reflect impaired redox homeostasis and compromised metabolic flexibility. Furthermore, the accumulation of short-chain acylcarnitines such as valerylcarnitine and propionylcarnitine points to dysregulated BCAA catabolism, which has been linked to mitochondrial overload and lipid oxidation defects [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>, <a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. These metabolite-level disruptions support the hypothesis that mitochondrial dysfunction is a central mechanistic axis in the development of MetS [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>, <a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. Impaired synthesis of L-arginine and NO elevates oxidative stress, increasing the cellular demand for glutathione [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. The metabolism of glycine, serine, and threonine contributes to glutathione production by providing critical precursors like glycine and serine, thus directly linking amino acid availability to redox regulation [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. At the same time, disruptions in BCAA catabolism lead to mitochondrial overload, which further exacerbates oxidative stress. These interconnected metabolic disturbances collectively create a pro-inflammatory, insulin-resistant environment—hallmarks of MetS, along with endothelial dysfunction [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. Furthermore, the accumulation of short-chain acylcarnitines reflects impaired β-oxidation capacity within mitochondria. This leads to incomplete lipid breakdown and increased generation of reactive oxygen species (ROS) [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]. The resulting oxidative stress compromises mitochondrial integrity, disrupts the electron transport chain and reduces ATP synthesis [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]. In this context, glutathione metabolism serves as a key antioxidant defense, neutralizing ROS and maintaining mitochondrial membrane stability [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. Additionally, glycine and serine—through the one-carbon metabolism pathway—support epigenetic regulation and mitochondrial protein maintenance via methylation reactions [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. Taken together, these findings underscore the central role of mitochondrial homeostasis in metabolic health. Disruptions in these metabolic pathways can both initiate and perpetuate mitochondrial dysfunction, driving the progression of MetS through mechanisms such as insulin resistance and vascular impairment [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>, <a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>].</p>
<p id="Par77">Metabolites serve as valuable biomarkers for exploring diet–disease associations. In this study, BCAAs were positively correlated with energy and nutrient intake, consistent with their role in energy metabolism. Excessive nutrient intake or obesity may lead to elevated BCAA turnover, which has been linked to impaired fatty acid oxidation and insulin resistance [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]. Hexoses were positively correlated with all 23 nutrients in this study, highlighting the influence of overall dietary load on metabolic profiles. A cross-sectional analysis from the European Prospective Investigation into Cancer and Nutrition-Potsdam study similarly reported that high consumption of red meat and fish, combined with low intake of whole-wheat bread and tea, was associated with elevated plasma hexose levels [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]. In contrast, glycine showed a significant negative correlation with energy and calorie-rich nutrients, consistent with its protective role against insulin resistance [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>]. Valerylcarnitine and propionylcarnitine levels were negatively associated with antioxidant nutrients, consistent with the role of antioxidants in promoting fatty acid oxidation and reducing acylcarnitine accumulation resulting from incomplete oxidation [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. Although individual metabolite–nutrient correlations were generally weak in this study, such as the correlation between valine and niacin (r = 0.16), previous research suggests that combining multiple metabolites can improve the accuracy of dietary predictions [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>], underscoring the complexity of underlying metabolic pathways and the need for integrative analytical approaches. Importantly, when statistically significant, correlation coefficients as low as r = 0.15 are often considered biologically meaningful, as they may reflect subtle but functionally relevant dietary influences on metabolism, such as protein influencing the biosynthesis of BCAAs, fat modulating mitochondrial β-oxidation, or niacin and riboflavin serving as cofactors in redox-related enzymatic pathways [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>]. However, given the large number of metabolite–nutrient correlations examined, the potential for false-positive associations due to multiple testing cannot be completely excluded, even with FDR correction.</p>
<p id="Par78">We identified metabolite–nutrient associations that differed by MetS status, with 16 pairs significant in the MetS group and 23 in the non-MetS group. Notably, niacin also exhibited significant positive associations with hexose, valine, isoleucine, leucine, and valerylcarnitine in the MetS group. Niacin serves as a precursor to nicotinamide adenine dinucleotide (NAD⁺) and nicotinamide adenine dinucleotide phosphate (NADP⁺), which play key roles in mitochondrial respiration, glycolysis, and lipid β-oxidation [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. Niacin is mainly obtained from dietary sources such as meat and fish, suggesting that dietary patterns may partly explain the observed associations. The valerylcarnitine-niacin pair showed significantly positive results only in the MetS group. As a metabolite involved in fatty acid β-oxidation and BCAA metabolism, valerylcarnitine reflects metabolic responses to glucose [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>]. Its altered levels may indicate mitochondrial dysfunction and impaired energy metabolism associated with insulin resistance [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. These findings highlight the need for further investigation into valerylcarnitine pathways and their role in metabolic health in MetS.</p>
<p id="Par79">The unique metabolite–nutrient relationships observed in the non-MetS and MetS groups underscore the complex interplay of metabolic pathways in maintaining health. For instance, vitamin B6 was positively associated with glycine but negatively associated with hexose in the non-MetS group, suggesting its involvement in amino acid and carbohydrate metabolism. Pyridoxal phosphate (PLP), the active form of vitamin B6, functions as a coenzyme in glycine synthesis via serine hydroxymethyltransferase and in glycogenolysis through glycogen phosphorylase. In the non-MetS group, PLP likely contributed to lower hexose levels by promoting glycine production and supporting efficient glucose metabolism [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>]. Conversely, in the MetS group, reduced PLP utilization and insulin resistance may have impaired glycine production, leading to hexose accumulation. Insulin resistance further contributes to elevated blood hexose levels by impairing glucose transport to tissues [<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>]. These findings highlight the critical role of vitamin B6 in metabolic health, particularly in the regulation of glycine and glucose metabolism.</p>
<p id="Par80">Similarly, the relationship between Fe intake and BCAA levels underscores how nutrient metabolism differs between the MetS and non-MetS groups. In the non-MetS group, Fe intake was negatively associated with BCAA, including leucine and isoleucine, which may reflect a higher intake of non-heme Fe, primarily found in plant-based foods, compared with heme Fe from animal products. Non-heme Fe has been linked to reduced inflammation and improved metabolic function [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>]. Conversely, in the MetS group, impaired Fe metabolism, likely owing to chronic inflammation and elevated hepcidin levels, may have inhibited Fe absorption and utilization [<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]. These disruptions could explain why Fe intake did not significantly influence BCAA concentrations in the MetS group. Although this study did not differentiate between heme and non-heme Fe, future research should investigate their distinct effects on BCAA metabolism and their potential role in MetS.</p>
<p id="Par81">The SGD model showed the best performance (AUC = 0.84) in our study, likely because it is well suited for small datasets with a limited number of variables, like our metabolite-only data. Although the number of metabolites was not large, some of them were correlated. SGD handles this kind of structure efficiently by applying regularization, which helps avoid overfitting. It also produces results that are easy to interpret, making it a practical and reliable choice for predicting MetS using metabolite data. These features make SGD particularly suitable for clinical deployment in Korea, where a unified national health insurance system and high electronic medical record (EMR) adoption provide a strong foundation for digital health tools. In practical terms, such a model could be embedded within EMR platforms at primary care clinics or health screening centers to provide automated risk scores for MetS, aiding early intervention. For effective translation into practice, collaboration with the National Health Insurance Service (NHIS), development of user-friendly interfaces for clinicians, and structured training modules will be key to promoting provider engagement and sustained use [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>].</p>
<p id="Par82">This study is the first to examine the association between serum metabolites, nutrient intake, and MetS in middle-aged Korean adults, emphasizing that metabolite–nutrient relationships can vary based on MetS status. The identified metabolite–nutrient pairs associated with MetS provide a foundation for exploring these relationships in other populations, underscoring their broader applicability. Furthermore, by applying machine learning algorithms to metabolomic data, we developed diagnostic models for MetS classification. Among these, the SGD model demonstrated the highest performance (AUC = 0.84), with high specificity (0.87) and moderate sensitivity (0.65). This trade-off suggests its current utility may lie more in confirmatory diagnosis than in early screening. However, the model also shows promise as a tool for early identification, and future refinement—such as incorporating additional clinical or dietary variables—may enhance its sensitivity and broaden its application in preventive strategies. Despite these strengths, the study has some limitations. First, its cross-sectional design limits the ability to infer causal relationships, highlighting the need for future investigations to distinguish between metabolic profile correlations and causality in MetS. Second, the combined analysis of men and women may have masked sex-specific metabolic variations, which should be addressed in future research. Third, the influence of metabolite–nutrient pairs on MetS, as assessed through the fixed-effects model, has not been experimentally validated. Fourth, in this study, dietary intake was assessed using a semi-quantitative FFQ, which is known to be susceptible to recall bias and measurement error. To minimize these biases, we used a validated FFQ instrument specifically developed for the Korean population. The validity and reproducibility of the study were evaluated against 12-day dietary records, which showed that nutrient density-based comparisons had similar levels of agreement. These findings support the reasonable validity of the FFQ in distinguishing differences in nutrient intake levels among Korean adults [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. Nevertheless, as with all self-reported dietary methods, potential for non-differential misclassification remains, which may attenuate the observed associations between nutrient intake and metabolite levels. Thus, the associations observed in this study may be conservative estimates. Future research employing more objective measures such as biomarkers or multiple 24-h recalls is warranted to validate and extend our findings. Fifth, our analysis employed a minimally adjusted model controlling only for age, sex, and BMI. While these covariates are well-established confounders in nutritional metabolomics, other lifestyle-related variables such as physical activity, medication use, and dietary behaviors were not included. This modeling choice was made to avoid overadjustment bias and preserve the real-world lifestyle differences between MetS and non-MetS groups. Notably, prior studies investigating the associations between dietary exposures and metabolomic profiles have also adopted similar minimally adjusted models, demonstrating that meaningful and biologically relevant findings can be derived even when only basic demographic covariates are controlled for [<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>, <a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>]. However, we acknowledge that the absence of these covariates may result in residual confounding, which should be addressed in future studies using more comprehensive adjustment or stratified analyses. Additionally, in this study, a large proportion of participants (67%) from the initial cohort were excluded due to missing plasma metabolite data. Although this exclusion was based on data availability rather than participant characteristics, it may have introduced selection bias and reduced the generalizability of the findings. Finally, external validation of the predictive models could not be performed due to the unavailability of an independent dataset. While internal validation was conducted using a stratified split of the original dataset, this limitation may restrict the generalizability of the findings. To enhance model generalizability, future research should validate these findings in independent populations with diverse demographic and clinical characteristics.</p>
<p id="Par83">This study identified key nutrients—such as fat, retinol, and cholesterol—as well as metabolites including hexoses, alanine, and valine, that are associated with MetS in a middle-aged Korean cohort. Among the 11 metabolite sub-pathways enriched across all analytical methods, arginine biosynthesis emerged as the most strongly associated with MetS, highlighting its potential role in vascular and metabolic dysfunction. In particular, hexose was identified as a prominent marker, showing the strongest correlation with blood glucose, a core component of MetS. Fixed-effects modeling revealed distinct metabolite–nutrient relationships stratified by MetS status, with six unique pairs observed in the MetS group and 13 in the non-MetS group. These findings underscore the differential nature of nutrient–metabolite interactions in the context of MetS and support their potential utility as biomarkers for metabolic dysregulation. The distinct metabolic profiles and nutrient–metabolite associations observed in this Korean population should be interpreted within the broader global context. A recent meta-analysis of 28 million adults across six WHO regions (Africa, Americas, South-East Asia, Europe, Eastern Mediterranean, and Western Pacific) reported a global prevalence of elevated fasting glucose of 24.5% [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. In our Korean cohort, 55.5% of individuals with MetS exhibited elevated fasting glucose levels, substantially exceeding the global average. Even among individuals without MetS, the prevalence was 17.0%, which is slightly lower than the global prevalence based on the general population, but still noteworthy. These findings reinforce the global relevance of dysregulated glucose metabolism as a key hallmark of MetS and underscore its particularly high burden in the Korean population. In this context, our observation of elevated hexose and its strong correlation with blood glucose further supports the central role of glycemic dysregulation across diverse populations, while also reflecting population-specific dietary influences. Future research should incorporate inter-individual variability, including sex-specific differences and genetic polymorphisms, which may modulate these interactions and further inform personalized dietary strategies. Sex hormones have been shown to influence amino acid metabolism [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>], insulin sensitivity, and gut microbiota [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>], all of which are relevant to MetS pathophysiology. Notably, both one-carbon metabolism and BCAA catabolism may be differentially regulated based on sex or genetic background. For instance, polymorphisms in <em>MTHFR</em> can impair folate metabolism and methylation efficiency [<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>], while variants in the <em>BCKDH</em> or <em>PPM1K</em> genes affect BCAA breakdown [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>]. These inter-individual differences are critical for advancing precision nutrition and personalized medicine. For instance, individuals with genetic variants in BCAA metabolism may benefit from adjusted macronutrient intake, while those with impaired one-carbon metabolism may require tailored folate or B-vitamin supplementation [<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>, <a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>]. Integrating metabolomic and genomic data may support the development of more precise individualized nutrition beyond a one-size-fits-all approach. The SGD classifier demonstrated the highest predictive accuracy among the models tested (AUC = 0.84), but external validation was not conducted, limiting the generalizability of the results. Therefore, the generalizability of these predictive models remains limited, and future research involving independent cohorts is warranted to confirm their robustness.</p>
<p id="Par84">Overall, this study highlights the value of an integrative multi-omics approach for elucidating how dietary factors modulate metabolic phenotypes and molecular pathways underlying MetS. Based on the observed associations, personalized dietary interventions for individuals with MetS may include BCAA-restricted diets to mitigate the metabolic impact of elevated BCAA and their catabolic byproducts, such as acylcarnitines, as well as targeted carbohydrate modulation, such as reducing high-hexose dietary sources, to improve glycemic control. Additionally, regulating the intake of niacin-rich proteins may be beneficial in managing amino acid and hexose metabolism, and targeted micronutrient supplementation, such as vitamin B6 to support glycine metabolism, could further enhance the metabolic environment in MetS.</p></section><section id="Sec29"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12366009/bin/12937_2025_1189_MOESM1_ESM.xls" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (38KB, xls) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12366009/bin/12937_2025_1189_MOESM2_ESM.xls" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (43KB, xls) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12366009/bin/12937_2025_1189_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12937_2025_1189_MOESM3_ESM.docx</a><sup> (525.9KB, docx) </sup><p>Supplementary Material 3. Supplementary Figure 1. Correlation network of metabolic syndrome components, metabolites, and nutrients. Statistically significant correlations between two metabolites are represented as edges, with positive correlations shown in blue and negative correlations in red. The color gradient and thickness of the edges are proportional to the absolute value of the correlation coefficient. <em>P</em>-values were adjusted for the false discovery rate using the Benjamini‒Hochberg method. ADMA, asymmetric dimethylarginine; Ac-Orn, acetylornithine; Ala, alanine; Arg, arginine; Asn, asparagine; Asp, aspartate; C0, free carnitine; C2, propionylcarnitine; C3, butyrylcarnitine; C5, isovalerylcarnitine; C8, octanoylcarnitine; Ca, calcium; Cit, citrulline; Fe, iron; Gln, glutamine; Glu, glutamate; Gly, glycine; H1, hexose; HDL, high-density lipoprotein; His, histidine; Ile, isoleucine; K, potassium; Leu, leucine; Lys, lysine; Met, methionine; Na, sodium; Orn, ornithine; P, phosphorus; Phe, phenylalanine; Pro, proline; Ser, serine; Thr, threonine; Trp, tryptophan; Tyr, tyrosine; Val, valine; Vit, vitamin.</p>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This study was conducted using bioresources from the National Biobank of Korea and the Korea Disease Control and Prevention Agency (NBK-2024-019). The authors gratefully acknowledge Dr. Sunjin Moon for his valuable statistical advice.</p>
<section id="FPar1"><h3 class="pmc_sec_title">Clinical trial number</h3>
<p id="Par85">Not applicable.</p></section></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>AAA</dt>
<dd><p id="Par6">Aromatic amino acid</p></dd>
<dt>ADB</dt>
<dd><p id="Par7">AdaBoost</p></dd>
<dt>AUC</dt>
<dd><p id="Par8">Area under the curve</p></dd>
<dt>BCAA</dt>
<dd><p id="Par9">Branched-chain amino acid</p></dd>
<dt>BMI</dt>
<dd><p id="Par10">Body mass index</p></dd>
<dt>CI</dt>
<dd><p id="Par11">Confidence interval</p></dd>
<dt>DNN</dt>
<dd><p id="Par12">Deep neural network</p></dd>
<dt>EMR</dt>
<dd><p id="Par13">Electronic medical record</p></dd>
<dt>FDR</dt>
<dd><p id="Par14">False discovery rate</p></dd>
<dt>FFQ</dt>
<dd><p id="Par15">Food frequency questionnaire</p></dd>
<dt>HDL</dt>
<dd><p id="Par16">High-density lipoprotein</p></dd>
<dt>HMDB</dt>
<dd><p id="Par17">Human Metabolome Database</p></dd>
<dt>KEGG</dt>
<dd><p id="Par18">Kyoto Encyclopedia of Genes and Genomes</p></dd>
<dt>KNN</dt>
<dd><p id="Par19">K-nearest neighbors</p></dd>
<dt>LASSO</dt>
<dd><p id="Par20">Least absolute shrinkage and selection operator</p></dd>
<dt>LR</dt>
<dd><p id="Par21">Logistic regression</p></dd>
<dt>MetS</dt>
<dd><p id="Par22">Metabolic syndrome</p></dd>
<dt>NB</dt>
<dd><p id="Par23">Naive Bayes</p></dd>
<dt>NO</dt>
<dd><p id="Par24">Nitric oxide</p></dd>
<dt>PC</dt>
<dd><p id="Par25">Phosphatidylcholine</p></dd>
<dt>PLP</dt>
<dd><p id="Par26">Pyridoxal 5′-phosphate</p></dd>
<dt>PLS-DA</dt>
<dd><p id="Par27">Partial least squares-discriminant analysis</p></dd>
<dt>RF</dt>
<dd><p id="Par28">Random forest</p></dd>
<dt>ROS</dt>
<dd><p id="Par29">Reactive oxygen species</p></dd>
<dt>SD</dt>
<dd><p id="Par30">Standard deviation</p></dd>
<dt>SGD</dt>
<dd><p id="Par31">Stochastic gradient descent</p></dd>
<dt>SVM</dt>
<dd><p id="Par32">Support vector machine</p></dd>
<dt>VIP</dt>
<dd><p id="Par33">Variable importance in projection</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>Authors'contributions Conceptualization: Shin D; Methodology: Kim M, Lee S, Shin D, Hur J; Formal analysis: Kim M, Lee S, Shin D, Hur J; Investigation: Kim M, Lee S, Shin D, Hur J; Writing–original draft: Kim M, Lee S; Writing–review &amp; editing: Kim M, Lee S, Shin D, Hur J; Funding Acquisition: Shin D; Supervision: Shin D.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by a grant from the National Research Foundation of Korea, funded by the Korean Government (MSIT) (RS-2024–00340086).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Data from the Korean Genome and Epidemiology Study are available through the procedure described at <a href="https://biobank.nih.go.kr/eng/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://biobank.nih.go.kr/eng/</a> (accessed on August 12, 2024). In compliance with institutional policies and data privacy regulations, the underlying code and datasets cannot be made publicly available. Nonetheless, all methodological steps and analytical workflows have been transparently described to ensure reproducibility.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar2"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par86">The study protocol was reviewed and approved by the Institutional Review Board of Inha University, Korea, on March 20, 2024 (protocol number: 240307-1A).</p>
<p id="Par87">All participants provided written informed consent prior to participation in the KoGES.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par88">Not applicable.</p></section><section id="FPar4"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par89">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Minyeong Kim and Suyeon Lee contributed equally as co-first authors.</p></div>
</div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Noubiap JJ, Nansseu JR, Lontchi-Yimagou E, Nkeck JR, Nyaga UF, Ngouo AT, et al. Geographic distribution of metabolic syndrome and its components in the general adult population: a meta-analysis of global data from 28 million individuals. Diabetes Res Clin Pract. 2022;188:109924.
</cite> [<a href="https://doi.org/10.1016/j.diabres.2022.109924" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35584716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Noubiap%20JJ,%20Nansseu%20JR,%20Lontchi-Yimagou%20E,%20Nkeck%20JR,%20Nyaga%20UF,%20Ngouo%20AT,%20et%20al.%20Geographic%20distribution%20of%20metabolic%20syndrome%20and%20its%20components%20in%20the%20general%20adult%20population:%20a%20meta-analysis%20of%20global%20data%20from%2028%20million%20individuals.%20Diabetes%20Res%20Clin%20Pract.%202022;188:109924." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Neeland IJ, Lim S, Tchernof A, Gastaldelli A, Rangaswami J, Ndumele CE, et al. Metabolic syndrome. Nat Rev Dis Primers. 2024;10(1):77.
</cite> [<a href="https://doi.org/10.1038/s41572-024-00563-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39420195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Neeland%20IJ,%20Lim%20S,%20Tchernof%20A,%20Gastaldelli%20A,%20Rangaswami%20J,%20Ndumele%20CE,%20et%20al.%20Metabolic%20syndrome.%20Nat%20Rev%20Dis%20Primers.%202024;10(1):77." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Kim NH, Seo M-H, Jung JH, Do Han K, Kim MK, Kim NH. 2023 Diabetic kidney disease fact sheet in Korea. Diabetes Metab J. 2024;48(3):463–72.
</cite> [<a href="https://doi.org/10.4093/dmj.2023.0310" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11140395/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38499437/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20NH,%20Seo%20M-H,%20Jung%20JH,%20Do%20Han%20K,%20Kim%20MK,%20Kim%20NH.%202023%20Diabetic%20kidney%20disease%20fact%20sheet%20in%20Korea.%20Diabetes%20Metab%20J.%202024;48(3):463%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Kim HJ, Kang DR, Kim JY, Kim W, Jeong YW, Chun K-H, et al. Metabolic syndrome fact sheet 2024: executive report. CardioMetabolic Syndrome J. 2024;4(2):70–80.</cite> [<a href="https://scholar.google.com/scholar_lookup?Kim%20HJ,%20Kang%20DR,%20Kim%20JY,%20Kim%20W,%20Jeong%20YW,%20Chun%20K-H,%20et%20al.%20Metabolic%20syndrome%20fact%20sheet%202024:%20executive%20report.%20CardioMetabolic%20Syndrome%20J.%202024;4(2):70%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Allam-Ndoul B, Guénard F, Garneau V, Cormier H, Barbier O, Pérusse L, et al. Association between metabolite profiles, metabolic syndrome and obesity status. Nutrients. 2016;8(6):324.
</cite> [<a href="https://doi.org/10.3390/nu8060324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4924165/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27240400/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Allam-Ndoul%20B,%20Gu%C3%A9nard%20F,%20Garneau%20V,%20Cormier%20H,%20Barbier%20O,%20P%C3%A9russe%20L,%20et%20al.%20Association%20between%20metabolite%20profiles,%20metabolic%20syndrome%20and%20obesity%20status.%20Nutrients.%202016;8(6):324." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Shi M, Han S, Klier K, Fobo G, Montrone C, Yu S, et al. Identification of candidate metabolite biomarkers for metabolic syndrome and its five components in population-based human cohorts. Cardiovasc Diabetol. 2023;22(1):141.
</cite> [<a href="https://doi.org/10.1186/s12933-023-01862-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10276453/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37328862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shi%20M,%20Han%20S,%20Klier%20K,%20Fobo%20G,%20Montrone%20C,%20Yu%20S,%20et%20al.%20Identification%20of%20candidate%20metabolite%20biomarkers%20for%20metabolic%20syndrome%20and%20its%20five%20components%20in%20population-based%20human%20cohorts.%20Cardiovasc%20Diabetol.%202023;22(1):141." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Kim M, Kim M, Han JY, Lee S-H, Jee SH, Lee JH. The metabolites in peripheral blood mononuclear cells showed greater differences between patients with impaired fasting glucose or type 2 diabetes and healthy controls than those in plasma. Diabetes Vasc Dis Res. 2017;14(2):130–8.</cite> [<a href="https://doi.org/10.1177/1479164116678157" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28185532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20M,%20Kim%20M,%20Han%20JY,%20Lee%20S-H,%20Jee%20SH,%20Lee%20JH.%20The%20metabolites%20in%20peripheral%20blood%20mononuclear%20cells%20showed%20greater%20differences%20between%20patients%20with%20impaired%20fasting%20glucose%20or%20type%202%20diabetes%20and%20healthy%20controls%20than%20those%20in%20plasma.%20Diabetes%20Vasc%20Dis%20Res.%202017;14(2):130%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Zhu M, Han Y, Zhang Y, Zhang S, Wei C, Cong Z, et al. Metabolomics study of the biochemical changes in the plasma of myocardial infarction patients. Front Physiol. 2018;9:1017.
</cite> [<a href="https://doi.org/10.3389/fphys.2018.01017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6123371/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30210352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu%20M,%20Han%20Y,%20Zhang%20Y,%20Zhang%20S,%20Wei%20C,%20Cong%20Z,%20et%20al.%20Metabolomics%20study%20of%20the%20biochemical%20changes%20in%20the%20plasma%20of%20myocardial%20infarction%20patients.%20Front%20Physiol.%202018;9:1017." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Kern HJ, Mitmesser SH. Role of nutrients in metabolic syndrome: a 2017 update. Nutr Diet Suppl. 2018. 10.2147/NDS.S148987.</cite> [<a href="https://scholar.google.com/scholar_lookup?Kern%20HJ,%20Mitmesser%20SH.%20Role%20of%20nutrients%20in%20metabolic%20syndrome:%20a%202017%20update.%20Nutr%20Diet%20Suppl.%202018.%2010.2147/NDS.S148987." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Harrison S, Couture P, Lamarche B. Diet quality, saturated fat and metabolic syndrome. Nutrients. 2020;12(11):3232.
</cite> [<a href="https://doi.org/10.3390/nu12113232" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7690379/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33105691/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harrison%20S,%20Couture%20P,%20Lamarche%20B.%20Diet%20quality,%20saturated%20fat%20and%20metabolic%20syndrome.%20Nutrients.%202020;12(11):3232." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Abete I, Astrup A, Martínez JA, Thorsdottir I, Zulet MA. Obesity and the metabolic syndrome: role of different dietary macronutrient distribution patterns and specific nutritional components on weight loss and maintenance. Nutr Rev. 2010;68(4):214–31.
</cite> [<a href="https://doi.org/10.1111/j.1753-4887.2010.00280.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20416018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abete%20I,%20Astrup%20A,%20Mart%C3%ADnez%20JA,%20Thorsdottir%20I,%20Zulet%20MA.%20Obesity%20and%20the%20metabolic%20syndrome:%20role%20of%20different%20dietary%20macronutrient%20distribution%20patterns%20and%20specific%20nutritional%20components%20on%20weight%20loss%20and%20maintenance.%20Nutr%20Rev.%202010;68(4):214%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Subramanian I, Verma S, Kumar S, Jere A, Anamika K. Multi-omics data integration, interpretation, and its application. Bioinform Biol Insights. 2020. 10.1177/1177932219899051.
</cite> [<a href="https://doi.org/10.1177/1177932219899051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7003173/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32076369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Subramanian%20I,%20Verma%20S,%20Kumar%20S,%20Jere%20A,%20Anamika%20K.%20Multi-omics%20data%20integration,%20interpretation,%20and%20its%20application.%20Bioinform%20Biol%20Insights.%202020.%2010.1177/1177932219899051." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Noerman S, Landberg R. Blood metabolite profiles linking dietary patterns with health—toward precision nutrition. J Intern Med. 2023;293(4):408–32.
</cite> [<a href="https://doi.org/10.1111/joim.13596" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36484466/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Noerman%20S,%20Landberg%20R.%20Blood%20metabolite%20profiles%20linking%20dietary%20patterns%20with%20health%E2%80%94toward%20precision%20nutrition.%20J%20Intern%20Med.%202023;293(4):408%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Brennan L, Hu FB, Sun Q. Metabolomics meets nutritional epidemiology: harnessing the potential in metabolomics data. Metabolites. 2021;11(10):709.
</cite> [<a href="https://doi.org/10.3390/metabo11100709" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8537466/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34677424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brennan%20L,%20Hu%20FB,%20Sun%20Q.%20Metabolomics%20meets%20nutritional%20epidemiology:%20harnessing%20the%20potential%20in%20metabolomics%20data.%20Metabolites.%202021;11(10):709." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Wang H, Wang Y, Li X, Deng X, Kong Y, Wang W, et al. Machine learning of plasma metabolome identifies biomarker panels for metabolic syndrome: findings from the China suboptimal health cohort. Cardiovasc Diabetol. 2022;21(1):288.
</cite> [<a href="https://doi.org/10.1186/s12933-022-01716-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9789589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36564831/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20H,%20Wang%20Y,%20Li%20X,%20Deng%20X,%20Kong%20Y,%20Wang%20W,%20et%20al.%20Machine%20learning%20of%20plasma%20metabolome%20identifies%20biomarker%20panels%20for%20metabolic%20syndrome:%20findings%20from%20the%20China%20suboptimal%20health%20cohort.%20Cardiovasc%20Diabetol.%202022;21(1):288." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Yang SJ, Kwak SY, Jo G, Song TJ, Shin MJ. Serum metabolite profile associated with incident type 2 diabetes in Koreans: findings from the Korean genome and epidemiology study. Sci Rep. 2018;8(1):8207.
</cite> [<a href="https://doi.org/10.1038/s41598-018-26320-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5974077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29844477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20SJ,%20Kwak%20SY,%20Jo%20G,%20Song%20TJ,%20Shin%20MJ.%20Serum%20metabolite%20profile%20associated%20with%20incident%20type%202%20diabetes%20in%20Koreans:%20findings%20from%20the%20Korean%20genome%20and%20epidemiology%20study.%20Sci%20Rep.%202018;8(1):8207." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Lee H, Jang HB, Yoo MG, Chung KS, Lee HJ. Protective effects of dietary MUFAs mediating metabolites against hypertension risk in the Korean genome and epidemiology study. Nutrients. 2019;11(8):1928.
</cite> [<a href="https://doi.org/10.3390/nu11081928" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6722700/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31426326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20H,%20Jang%20HB,%20Yoo%20MG,%20Chung%20KS,%20Lee%20HJ.%20Protective%20effects%20of%20dietary%20MUFAs%20mediating%20metabolites%20against%20hypertension%20risk%20in%20the%20Korean%20genome%20and%20epidemiology%20study.%20Nutrients.%202019;11(8):1928." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Park S. Associations between APOE gene variants and metabolite levels in hypercholesterolemia: a metabolite GWAS study in a Korean cohort. Biomed Sci Lett. 2024;30(3):173–80.</cite> [<a href="https://scholar.google.com/scholar_lookup?Park%20S.%20Associations%20between%20APOE%20gene%20variants%20and%20metabolite%20levels%20in%20hypercholesterolemia:%20a%20metabolite%20GWAS%20study%20in%20a%20Korean%20cohort.%20Biomed%20Sci%20Lett.%202024;30(3):173%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Kim YJ, Lee HS, Kim YK, Park S, Kim JM, Yun JH, et al. Association of metabolites with obesity and type 2 diabetes based on FTO genotype. PLoS ONE. 2016;11(6):e0156612.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0156612" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4889059/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27249024/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20YJ,%20Lee%20HS,%20Kim%20YK,%20Park%20S,%20Kim%20JM,%20Yun%20JH,%20et%20al.%20Association%20of%20metabolites%20with%20obesity%20and%20type%202%20diabetes%20based%20on%20FTO%20genotype.%20PLoS%20ONE.%202016;11(6):e0156612." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Hahn SJ, Kim S, Choi YS, Lee J, Kang J. Prediction of type 2 diabetes using genome-wide polygenic risk score and metabolic profiles: a machine learning analysis of population-based 10-year prospective cohort study. EBioMedicine. 2022. 10.1016/j.ebiom.2022.104383.
</cite> [<a href="https://doi.org/10.1016/j.ebiom.2022.104383" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9713286/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36462406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hahn%20SJ,%20Kim%20S,%20Choi%20YS,%20Lee%20J,%20Kang%20J.%20Prediction%20of%20type%202%20diabetes%20using%20genome-wide%20polygenic%20risk%20score%20and%20metabolic%20profiles:%20a%20machine%20learning%20analysis%20of%20population-based%2010-year%20prospective%20cohort%20study.%20EBioMedicine.%202022.%2010.1016/j.ebiom.2022.104383." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Bondia-Pons I, Martinez JA, de la Iglesia R, Lopez-Legarrea P, Poutanen K, Hanhineva K, et al. Effects of short-and long-term Mediterranean-based dietary treatment on plasma LC-QTOF/MS metabolic profiling of subjects with metabolic syndrome features: the Metabolic Syndrome Reduction in Navarra (RESMENA) randomized controlled trial. Mol Nutr Food Res. 2015;59(4):711–28.
</cite> [<a href="https://doi.org/10.1002/mnfr.201400309" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25641909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bondia-Pons%20I,%20Martinez%20JA,%20de%20la%20Iglesia%20R,%20Lopez-Legarrea%20P,%20Poutanen%20K,%20Hanhineva%20K,%20et%20al.%20Effects%20of%20short-and%20long-term%20Mediterranean-based%20dietary%20treatment%20on%20plasma%20LC-QTOF/MS%20metabolic%20profiling%20of%20subjects%20with%20metabolic%20syndrome%20features:%20the%20Metabolic%20Syndrome%20Reduction%20in%20Navarra%20(RESMENA)%20randomized%20controlled%20trial.%20Mol%20Nutr%20Food%20Res.%202015;59(4):711%E2%80%9328." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Pallister T, Jackson MA, Martin TC, Zierer J, Jennings A, Mohney RP, et al. Hippurate as a metabolomic marker of gut microbiome diversity: modulation by diet and relationship to metabolic syndrome. Sci Rep. 2017;7(1):13670.
</cite> [<a href="https://doi.org/10.1038/s41598-017-13722-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5651863/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29057986/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pallister%20T,%20Jackson%20MA,%20Martin%20TC,%20Zierer%20J,%20Jennings%20A,%20Mohney%20RP,%20et%20al.%20Hippurate%20as%20a%20metabolomic%20marker%20of%20gut%20microbiome%20diversity:%20modulation%20by%20diet%20and%20relationship%20to%20metabolic%20syndrome.%20Sci%20Rep.%202017;7(1):13670." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Wan M, Li Q, Lei Q, Zhou D, Wang S. Polyphenols and polysaccharides from <em>morus</em><em>alba</em> L. fruit attenuate high-fat diet-induced metabolic syndrome modifying the gut microbiota and metabolite profile. Foods. 2022;11(12):1818.
</cite> [<a href="https://doi.org/10.3390/foods11121818" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9223293/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35742014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wan%20M,%20Li%20Q,%20Lei%20Q,%20Zhou%20D,%20Wang%20S.%20Polyphenols%20and%20polysaccharides%20from%20morusalba%20L.%20fruit%20attenuate%20high-fat%20diet-induced%20metabolic%20syndrome%20modifying%20the%20gut%20microbiota%20and%20metabolite%20profile.%20Foods.%202022;11(12):1818." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Kim Y, Han BG, Group K. Cohort profile: the Korean genome and epidemiology study (KoGES) consortium. Int J Epidemiol. 2017;46(2):e20-e.
</cite> [<a href="https://doi.org/10.1093/ije/dyv316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5837648/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27085081/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20Y,%20Han%20BG,%20Group%20K.%20Cohort%20profile:%20the%20Korean%20genome%20and%20epidemiology%20study%20(KoGES)%20consortium.%20Int%20J%20Epidemiol.%202017;46(2):e20-e." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Third Report of the National Cholesterol Education Program (NCEP) Expert Panel on Detection, Evaluation, and Treatment of High Blood Cholesterol in Adults (Adult Treatment Panel III) final report. Circulation. 2002;106(25):3143–421.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/12485966/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Seo MH, Lee W-Y, Kim SS, Kang J-H, Kang J-H, Kim KK, et al. 2018 Korean society for the study of obesity guideline for the management of obesity in Korea. J Obes Metab Syndrome. 2019;28(1):40.</cite> [<a href="https://doi.org/10.7570/jomes.2019.28.1.40" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6484940/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31089578/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Seo%20MH,%20Lee%20W-Y,%20Kim%20SS,%20Kang%20J-H,%20Kang%20J-H,%20Kim%20KK,%20et%20al.%202018%20Korean%20society%20for%20the%20study%20of%20obesity%20guideline%20for%20the%20management%20of%20obesity%20in%20Korea.%20J%20Obes%20Metab%20Syndrome.%202019;28(1):40." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Hong AR, Lim S. Clinical characteristics of metabolic syndrome in Korea, and its comparison with other A sian countries. J Diabetes Investig. 2015;6(5):508–15.
</cite> [<a href="https://doi.org/10.1111/jdi.12313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4578487/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26417407/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hong%20AR,%20Lim%20S.%20Clinical%20characteristics%20of%20metabolic%20syndrome%20in%20Korea,%20and%20its%20comparison%20with%20other%20A%20sian%20countries.%20J%20Diabetes%20Investig.%202015;6(5):508%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Ahn Y, Kwon E, Shim J, Park M, Joo Y, Kimm K, et al. Validation and reproducibility of food frequency questionnaire for Korean genome epidemiologic study. Eur J Clin Nutr. 2007;61(12):1435–41.
</cite> [<a href="https://doi.org/10.1038/sj.ejcn.1602657" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17299477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ahn%20Y,%20Kwon%20E,%20Shim%20J,%20Park%20M,%20Joo%20Y,%20Kimm%20K,%20et%20al.%20Validation%20and%20reproducibility%20of%20food%20frequency%20questionnaire%20for%20Korean%20genome%20epidemiologic%20study.%20Eur%20J%20Clin%20Nutr.%202007;61(12):1435%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Doo M, Kim Y. The consumption of dietary antioxidant vitamins modifies the risk of obesity among Korean men with short sleep duration. Nutrients. 2017;9(7):780.
</cite> [<a href="https://doi.org/10.3390/nu9070780" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5537894/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28726758/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Doo%20M,%20Kim%20Y.%20The%20consumption%20of%20dietary%20antioxidant%20vitamins%20modifies%20the%20risk%20of%20obesity%20among%20Korean%20men%20with%20short%20sleep%20duration.%20Nutrients.%202017;9(7):780." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Li FE, Zhang FL, Zhang P, Liu D, Liu HY, Guo ZN, et al. Sex-based differences in and risk factors for metabolic syndrome in adults aged 40 years and above in Northeast China: results from the cross-sectional China national stroke screening survey. BMJ Open. 2021;11(3):e038671.
</cite> [<a href="https://doi.org/10.1136/bmjopen-2020-038671" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7993204/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33762227/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20FE,%20Zhang%20FL,%20Zhang%20P,%20Liu%20D,%20Liu%20HY,%20Guo%20ZN,%20et%20al.%20Sex-based%20differences%20in%20and%20risk%20factors%20for%20metabolic%20syndrome%20in%20adults%20aged%2040%20years%20and%20above%20in%20Northeast%20China:%20results%20from%20the%20cross-sectional%20China%20national%20stroke%20screening%20survey.%20BMJ%20Open.%202021;11(3):e038671." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Kanehisa M, Goto S. KEGG: kyoto encyclopedia of genes and genomes. Nucleic Acids Res. 2000;28(1):27–30.
</cite> [<a href="https://doi.org/10.1093/nar/28.1.27" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC102409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10592173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kanehisa%20M,%20Goto%20S.%20KEGG:%20kyoto%20encyclopedia%20of%20genes%20and%20genomes.%20Nucleic%20Acids%20Res.%202000;28(1):27%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Wishart DS, Jewison T, Guo AC, Wilson M, Knox C, Liu Y, et al. HMDB 3.0—the human metabolome database in 2013. Nucleic Acids Res. 2012;41(D1):D801–7.
</cite> [<a href="https://doi.org/10.1093/nar/gks1065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3531200/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23161693/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wishart%20DS,%20Jewison%20T,%20Guo%20AC,%20Wilson%20M,%20Knox%20C,%20Liu%20Y,%20et%20al.%20HMDB%203.0%E2%80%94the%20human%20metabolome%20database%20in%202013.%20Nucleic%20Acids%20Res.%202012;41(D1):D801%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Evans ED, Duvallet C, Chu ND, Oberst MK, Murphy MA, Rockafellow I, et al. Predicting human health from biofluid-based metabolomics using machine learning. Sci Rep. 2020;10(1):17635.
</cite> [<a href="https://doi.org/10.1038/s41598-020-74823-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7572502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33077825/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Evans%20ED,%20Duvallet%20C,%20Chu%20ND,%20Oberst%20MK,%20Murphy%20MA,%20Rockafellow%20I,%20et%20al.%20Predicting%20human%20health%20from%20biofluid-based%20metabolomics%20using%20machine%20learning.%20Sci%20Rep.%202020;10(1):17635." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Galal A, Talal M, Moustafa A. Applications of machine learning in metabolomics: disease modeling and classification. Front Genet. 2022;13:1017340.
</cite> [<a href="https://doi.org/10.3389/fgene.2022.1017340" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9730048/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36506316/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Galal%20A,%20Talal%20M,%20Moustafa%20A.%20Applications%20of%20machine%20learning%20in%20metabolomics:%20disease%20modeling%20and%20classification.%20Front%20Genet.%202022;13:1017340." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Yagin FH, Colak C, Algarni A, Algarni A, Al-Hashem F, Ardigò LP. Explainable boosting machines identify key metabolomic biomarkers in rheumatoid arthritis. Medicina. 2025;61(5):833.
</cite> [<a href="https://doi.org/10.3390/medicina61050833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12113160/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40428791/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yagin%20FH,%20Colak%20C,%20Algarni%20A,%20Algarni%20A,%20Al-Hashem%20F,%20Ardig%C3%B2%20LP.%20Explainable%20boosting%20machines%20identify%20key%20metabolomic%20biomarkers%20in%20rheumatoid%20arthritis.%20Medicina.%202025;61(5):833." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Gong Y, Ding W, Wang P, Wu Q, Yao X, Yang Q. Evaluating machine learning methods of analyzing multiclass metabolomics. J Chem Inf Model. 2023;63(24):7628–41.
</cite> [<a href="https://doi.org/10.1021/acs.jcim.3c01525" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38079572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gong%20Y,%20Ding%20W,%20Wang%20P,%20Wu%20Q,%20Yao%20X,%20Yang%20Q.%20Evaluating%20machine%20learning%20methods%20of%20analyzing%20multiclass%20metabolomics.%20J%20Chem%20Inf%20Model.%202023;63(24):7628%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Kim H, Heo JH, Lim DH, Kim Y. Development of a metabolic syndrome classification and prediction model for Koreans using deep learning technology: the Korea National Health and Nutrition Examination Survey (KNHANES)(2013–2018). Clin Nutr Res. 2023;12(2):138.
</cite> [<a href="https://doi.org/10.7762/cnr.2023.12.2.138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10193438/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37214780/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20H,%20Heo%20JH,%20Lim%20DH,%20Kim%20Y.%20Development%20of%20a%20metabolic%20syndrome%20classification%20and%20prediction%20model%20for%20Koreans%20using%20deep%20learning%20technology:%20the%20Korea%20National%20Health%20and%20Nutrition%20Examination%20Survey%20(KNHANES)(2013%E2%80%932018).%20Clin%20Nutr%20Res.%202023;12(2):138." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Du Y, Rafferty AR, McAuliffe FM, Wei L, Mooney C. An explainable machine learning-based clinical decision support system for prediction of gestational diabetes mellitus. Sci Rep. 2022;12(1):1170.
</cite> [<a href="https://doi.org/10.1038/s41598-022-05112-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8782851/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35064173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Du%20Y,%20Rafferty%20AR,%20McAuliffe%20FM,%20Wei%20L,%20Mooney%20C.%20An%20explainable%20machine%20learning-based%20clinical%20decision%20support%20system%20for%20prediction%20of%20gestational%20diabetes%20mellitus.%20Sci%20Rep.%202022;12(1):1170." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Collins GS, Moons KG, Dhiman P, Riley RD, Beam AL, Van Calster B, et al. TRIPOD+ AI statement: updated guidance for reporting clinical prediction models that use regression or machine learning methods. BMJ. 2024;385.</cite> [<a href="https://doi.org/10.1136/bmj-2023-078378" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11019967/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38626948/" class="usa-link">PubMed</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Das SK, Ainsworth HC, Dimitrov L, Okut H, Comeau ME, Sharma N, et al. Metabolomic architecture of obesity implicates metabolonic lactone sulfate in cardiometabolic disease. Mol Metab. 2021;54:101342.
</cite> [<a href="https://doi.org/10.1016/j.molmet.2021.101342" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8640864/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34563731/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Das%20SK,%20Ainsworth%20HC,%20Dimitrov%20L,%20Okut%20H,%20Comeau%20ME,%20Sharma%20N,%20et%20al.%20Metabolomic%20architecture%20of%20obesity%20implicates%20metabolonic%20lactone%20sulfate%20in%20cardiometabolic%20disease.%20Mol%20Metab.%202021;54:101342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Mahajan UV, Varma VR, Huang C-W, An Y, Tanaka T, Ferrucci L, et al. Blood metabolite signatures of metabolic syndrome in two cross-cultural older adult cohorts. Int J Mol Sci. 2020;21(4):1324.
</cite> [<a href="https://doi.org/10.3390/ijms21041324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7072935/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32079087/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mahajan%20UV,%20Varma%20VR,%20Huang%20C-W,%20An%20Y,%20Tanaka%20T,%20Ferrucci%20L,%20et%20al.%20Blood%20metabolite%20signatures%20of%20metabolic%20syndrome%20in%20two%20cross-cultural%20older%20adult%20cohorts.%20Int%20J%20Mol%20Sci.%202020;21(4):1324." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Paneque A, Fortus H, Zheng J, Werlen G, Jacinto E. The hexosamine biosynthesis pathway: regulation and function. Genes. 2023;14(4):933.
</cite> [<a href="https://doi.org/10.3390/genes14040933" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10138107/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37107691/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Paneque%20A,%20Fortus%20H,%20Zheng%20J,%20Werlen%20G,%20Jacinto%20E.%20The%20hexosamine%20biosynthesis%20pathway:%20regulation%20and%20function.%20Genes.%202023;14(4):933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Ntzouvani A, Nomikos T, Panagiotakos D, Fragopoulou E, Pitsavos C, McCann A, et al. Amino acid profile and metabolic syndrome in a male Mediterranean population: a cross-sectional study. Nutr Metab Cardiovasc Dis. 2017;27(11):1021–30.
</cite> [<a href="https://doi.org/10.1016/j.numecd.2017.07.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28958693/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ntzouvani%20A,%20Nomikos%20T,%20Panagiotakos%20D,%20Fragopoulou%20E,%20Pitsavos%20C,%20McCann%20A,%20et%20al.%20Amino%20acid%20profile%20and%20metabolic%20syndrome%20in%20a%20male%20Mediterranean%20population:%20a%20cross-sectional%20study.%20Nutr%20Metab%20Cardiovasc%20Dis.%202017;27(11):1021%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Hu W, Sun L, Gong Y, Zhou Y, Yang P, Ye Z, et al. Relationship between branched-chain amino acids, metabolic syndrome, and cardiovascular risk profile in a chinese population: a cross-sectional study. Int J Endocrinol. 2016;2016(1):8173905.
</cite> [<a href="https://doi.org/10.1155/2016/8173905" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4977397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27528871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu%20W,%20Sun%20L,%20Gong%20Y,%20Zhou%20Y,%20Yang%20P,%20Ye%20Z,%20et%20al.%20Relationship%20between%20branched-chain%20amino%20acids,%20metabolic%20syndrome,%20and%20cardiovascular%20risk%20profile%20in%20a%20chinese%20population:%20a%20cross-sectional%20study.%20Int%20J%20Endocrinol.%202016;2016(1):8173905." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Taghizadeh H, Emamgholipour S, Hosseinkhani S, Arjmand B, Rezaei N, Dilmaghani-Marand A, et al. The association between acylcarnitine and amino acids profile and metabolic syndrome and its components in Iranian adults: data from STEPs 2016. Front Endocrinol. 2023;14: 1058952.</cite> [<a href="https://doi.org/10.3389/fendo.2023.1058952" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10008865/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36923214/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Taghizadeh%20H,%20Emamgholipour%20S,%20Hosseinkhani%20S,%20Arjmand%20B,%20Rezaei%20N,%20Dilmaghani-Marand%20A,%20et%20al.%20The%20association%20between%20acylcarnitine%20and%20amino%20acids%20profile%20and%20metabolic%20syndrome%20and%20its%20components%20in%20Iranian%20adults:%20data%20from%20STEPs%202016.%20Front%20Endocrinol.%202023;14:%201058952." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Li X, Sun L, Zhang W, Li H, Wang S, Mu H, et al. Association of serum glycine levels with metabolic syndrome in an elderly Chinese population. Nutr Metab. 2018;15:1–9.</cite> [<a href="https://doi.org/10.1186/s12986-018-0325-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6297971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30568717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20X,%20Sun%20L,%20Zhang%20W,%20Li%20H,%20Wang%20S,%20Mu%20H,%20et%20al.%20Association%20of%20serum%20glycine%20levels%20with%20metabolic%20syndrome%20in%20an%20elderly%20Chinese%20population.%20Nutr%20Metab.%202018;15:1%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Julibert A, del Mar Bibiloni M, Tur JA. Dietary fat intake and metabolic syndrome in adults: a systematic review. Nutr Metab Cardiovasc Dis. 2019;29(9):887–905.
</cite> [<a href="https://doi.org/10.1016/j.numecd.2019.05.055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31377181/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Julibert%20A,%20del%20Mar%20Bibiloni%20M,%20Tur%20JA.%20Dietary%20fat%20intake%20and%20metabolic%20syndrome%20in%20adults:%20a%20systematic%20review.%20Nutr%20Metab%20Cardiovasc%20Dis.%202019;29(9):887%E2%80%93905." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Park S, Ahn J, Lee B-K. Very-low-fat diets may be associated with increased risk of metabolic syndrome in the adult population. Clin Nutr. 2016;35(5):1159–67.
</cite> [<a href="https://doi.org/10.1016/j.clnu.2015.09.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26602244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Park%20S,%20Ahn%20J,%20Lee%20B-K.%20Very-low-fat%20diets%20may%20be%20associated%20with%20increased%20risk%20of%20metabolic%20syndrome%20in%20the%20adult%20population.%20Clin%20Nutr.%202016;35(5):1159%E2%80%9367." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Yoo H, Park K. Association between phytochemical index and osteoporosis in women: a prospective cohort study in Korea. Nutrients. 2023;15(7): 1605.
</cite> [<a href="https://doi.org/10.3390/nu15071605" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10097207/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37049447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoo%20H,%20Park%20K.%20Association%20between%20phytochemical%20index%20and%20osteoporosis%20in%20women:%20a%20prospective%20cohort%20study%20in%20Korea.%20Nutrients.%202023;15(7):%201605." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Assumpção CR, Brunini TM, Pereira NR, Godoy-Matos AF, Siqueira MA, Mann GE, et al. Insulin resistance in obesity and metabolic syndrome: is there a connection with platelet l-arginine transport? Blood Cells Mol Dis. 2010;45(4):338–42.
</cite> [<a href="https://doi.org/10.1016/j.bcmd.2010.10.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21078563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Assump%C3%A7%C3%A3o%20CR,%20Brunini%20TM,%20Pereira%20NR,%20Godoy-Matos%20AF,%20Siqueira%20MA,%20Mann%20GE,%20et%20al.%20Insulin%20resistance%20in%20obesity%20and%20metabolic%20syndrome:%20is%20there%20a%20connection%20with%20platelet%20l-arginine%20transport?%20Blood%20Cells%20Mol%20Dis.%202010;45(4):338%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Mone P, Pansini A, Jankauskas SS, Varzideh F, Kansakar U, Lombardi A, et al. L-arginine improves cognitive impairment in hypertensive frail older adults. Front Cardiovasc Med. 2022;9: 868521.
</cite> [<a href="https://doi.org/10.3389/fcvm.2022.868521" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9039514/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35498050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mone%20P,%20Pansini%20A,%20Jankauskas%20SS,%20Varzideh%20F,%20Kansakar%20U,%20Lombardi%20A,%20et%20al.%20L-arginine%20improves%20cognitive%20impairment%20in%20hypertensive%20frail%20older%20adults.%20Front%20Cardiovasc%20Med.%202022;9:%20868521." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Hu S, Han M, Rezaei A, Li D, Wu G, Ma X. L-arginine modulates glucose and lipid metabolism in obesity and diabetes. Curr Protein Pept Sci. 2017;18(6):599–608.
</cite> [<a href="https://doi.org/10.2174/1389203717666160627074017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27356939/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu%20S,%20Han%20M,%20Rezaei%20A,%20Li%20D,%20Wu%20G,%20Ma%20X.%20L-arginine%20modulates%20glucose%20and%20lipid%20metabolism%20in%20obesity%20and%20diabetes.%20Curr%20Protein%20Pept%20Sci.%202017;18(6):599%E2%80%93608." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Lutchmansingh FK, Hsu JW, Bennett FI, Badaloo AV, McFarlane-Anderson N, Gordon-Strachan GM, et al. Glutathione metabolism in type 2 diabetes and its relationship with microvascular complications and glycemia. PLoS ONE. 2018;13(6):e0198626.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0198626" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5991679/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29879181/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lutchmansingh%20FK,%20Hsu%20JW,%20Bennett%20FI,%20Badaloo%20AV,%20McFarlane-Anderson%20N,%20Gordon-Strachan%20GM,%20et%20al.%20Glutathione%20metabolism%20in%20type%202%20diabetes%20and%20its%20relationship%20with%20microvascular%20complications%20and%20glycemia.%20PLoS%20ONE.%202018;13(6):e0198626." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Díaz-Flores M, Cruz M, Duran-Reyes G, Munguia-Miranda C, Loza-Rodríguez H, Pulido-Casas E, et al. Oral supplementation with glycine reduces oxidative stress in patients with metabolic syndrome, improving their systolic blood pressure. Can J Physiol Pharmacol. 2013;91(10):855–60.
</cite> [<a href="https://doi.org/10.1139/cjpp-2012-0341" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24144057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?D%C3%ADaz-Flores%20M,%20Cruz%20M,%20Duran-Reyes%20G,%20Munguia-Miranda%20C,%20Loza-Rodr%C3%ADguez%20H,%20Pulido-Casas%20E,%20et%20al.%20Oral%20supplementation%20with%20glycine%20reduces%20oxidative%20stress%20in%20patients%20with%20metabolic%20syndrome,%20improving%20their%20systolic%20blood%20pressure.%20Can%20J%20Physiol%20Pharmacol.%202013;91(10):855%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Xie W, Wood AR, Lyssenko V, Weedon MN, Knowles JW, Alkayyali S, et al. Genetic variants associated with glycine metabolism and their role in insulin sensitivity and type 2 diabetes. Diabetes. 2013;62(6):2141–50.
</cite> [<a href="https://doi.org/10.2337/db12-0876" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3661655/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23378610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xie%20W,%20Wood%20AR,%20Lyssenko%20V,%20Weedon%20MN,%20Knowles%20JW,%20Alkayyali%20S,%20et%20al.%20Genetic%20variants%20associated%20with%20glycine%20metabolism%20and%20their%20role%20in%20insulin%20sensitivity%20and%20type%202%20diabetes.%20Diabetes.%202013;62(6):2141%E2%80%9350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Mihalik SJ, Goodpaster BH, Kelley DE, Chace DH, Vockley J, Toledo FG, et al. Increased levels of plasma acylcarnitines in obesity and type 2 diabetes and identification of a marker of glucolipotoxicity. Obesity. 2010;18(9):1695–700.
</cite> [<a href="https://doi.org/10.1038/oby.2009.510" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3984458/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20111019/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mihalik%20SJ,%20Goodpaster%20BH,%20Kelley%20DE,%20Chace%20DH,%20Vockley%20J,%20Toledo%20FG,%20et%20al.%20Increased%20levels%20of%20plasma%20acylcarnitines%20in%20obesity%20and%20type%202%20diabetes%20and%20identification%20of%20a%20marker%20of%20glucolipotoxicity.%20Obesity.%202010;18(9):1695%E2%80%93700." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Strand E, Pedersen ER, Svingen GF, Olsen T, Bjørndal B, Karlsson T, et al. Serum acylcarnitines and risk of cardiovascular death and acute myocardial infarction in patients with stable angina pectoris. J Am Heart Assoc. 2017;6(2):e003620.
</cite> [<a href="https://doi.org/10.1161/JAHA.116.003620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5523736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28159823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Strand%20E,%20Pedersen%20ER,%20Svingen%20GF,%20Olsen%20T,%20Bj%C3%B8rndal%20B,%20Karlsson%20T,%20et%20al.%20Serum%20acylcarnitines%20and%20risk%20of%20cardiovascular%20death%20and%20acute%20myocardial%20infarction%20in%20patients%20with%20stable%20angina%20pectoris.%20J%20Am%20Heart%20Assoc.%202017;6(2):e003620." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Lemos GdO, Torrinhas RS, Waitzberg DL. Nutrients, physical activity, and mitochondrial dysfunction in the setting of metabolic syndrome. Nutrients. 2023;15(5): 1217.
</cite> [<a href="https://doi.org/10.3390/nu15051217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10004804/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36904216/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lemos%20GdO,%20Torrinhas%20RS,%20Waitzberg%20DL.%20Nutrients,%20physical%20activity,%20and%20mitochondrial%20dysfunction%20in%20the%20setting%20of%20metabolic%20syndrome.%20Nutrients.%202023;15(5):%201217." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Romani M, Berger MM, D’Amelio P. From the bench to the bedside: branched amino acid and micronutrient strategies to improve mitochondrial dysfunction leading to sarcopenia. Nutrients. 2022;14(3):483.
</cite> [<a href="https://doi.org/10.3390/nu14030483" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8838610/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35276842/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Romani%20M,%20Berger%20MM,%20D%E2%80%99Amelio%20P.%20From%20the%20bench%20to%20the%20bedside:%20branched%20amino%20acid%20and%20micronutrient%20strategies%20to%20improve%20mitochondrial%20dysfunction%20leading%20to%20sarcopenia.%20Nutrients.%202022;14(3):483." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Liang M, Wang Z, Li H, Cai L, Pan J, He H, et al. L-Arginine induces antioxidant response to prevent oxidative stress via stimulation of glutathione synthesis and activation of Nrf2 pathway. Food Chem Toxicol. 2018;115:315–28.
</cite> [<a href="https://doi.org/10.1016/j.fct.2018.03.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29577948/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liang%20M,%20Wang%20Z,%20Li%20H,%20Cai%20L,%20Pan%20J,%20He%20H,%20et%20al.%20L-Arginine%20induces%20antioxidant%20response%20to%20prevent%20oxidative%20stress%20via%20stimulation%20of%20glutathione%20synthesis%20and%20activation%20of%20Nrf2%20pathway.%20Food%20Chem%20Toxicol.%202018;115:315%E2%80%9328." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Wu X, Liu C, Yang S, Shen N, Wang Y, Zhu Y, et al. Glycine-serine-threonine metabolic axis delays intervertebral disc degeneration through antioxidant effects: an imaging and metabonomics study. Oxid Med Cell Longev. 2021;2021(1):5579736.
</cite> [<a href="https://doi.org/10.1155/2021/5579736" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8416401/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34484565/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20X,%20Liu%20C,%20Yang%20S,%20Shen%20N,%20Wang%20Y,%20Zhu%20Y,%20et%20al.%20Glycine-serine-threonine%20metabolic%20axis%20delays%20intervertebral%20disc%20degeneration%20through%20antioxidant%20effects:%20an%20imaging%20and%20metabonomics%20study.%20Oxid%20Med%20Cell%20Longev.%202021;2021(1):5579736." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Liu S, Li L, Lou P, Zhao M, Wang Y, Tang M, et al. Elevated branched-chain α-keto acids exacerbate macrophage oxidative stress and chronic inflammatory damage in type 2 diabetes mellitus. Free Radic Biol Med. 2021;175:141–54.
</cite> [<a href="https://doi.org/10.1016/j.freeradbiomed.2021.08.240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34474107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20S,%20Li%20L,%20Lou%20P,%20Zhao%20M,%20Wang%20Y,%20Tang%20M,%20et%20al.%20Elevated%20branched-chain%20%CE%B1-keto%20acids%20exacerbate%20macrophage%20oxidative%20stress%20and%20chronic%20inflammatory%20damage%20in%20type%202%20diabetes%20mellitus.%20Free%20Radic%20Biol%20Med.%202021;175:141%E2%80%9354." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Rutkowsky JM, Knotts TA, Ono-Moore KD, McCoin CS, Huang S, Schneider D, et al. Acylcarnitines activate proinflammatory signaling pathways. Am J Physiol Endocrinol Metab. 2014;306(12):E1378–87.
</cite> [<a href="https://doi.org/10.1152/ajpendo.00656.2013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4059985/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24760988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rutkowsky%20JM,%20Knotts%20TA,%20Ono-Moore%20KD,%20McCoin%20CS,%20Huang%20S,%20Schneider%20D,%20et%20al.%20Acylcarnitines%20activate%20proinflammatory%20signaling%20pathways.%20Am%20J%20Physiol%20Endocrinol%20Metab.%202014;306(12):E1378%E2%80%9387." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Schieber M, Chandel NS. ROS function in redox signaling and oxidative stress. Curr Biol. 2014;24(10):R453–62.
</cite> [<a href="https://doi.org/10.1016/j.cub.2014.03.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4055301/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24845678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schieber%20M,%20Chandel%20NS.%20ROS%20function%20in%20redox%20signaling%20and%20oxidative%20stress.%20Curr%20Biol.%202014;24(10):R453%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Amorim R, Simões IC, Teixeira J, Cagide F, Potes Y, Soares P, et al. Mitochondria-targeted anti-oxidant AntiOxCIN4 improved liver steatosis in western diet-fed mice by preventing lipid accumulation due to upregulation of fatty acid oxidation, quality control mechanism and antioxidant defense systems. Redox Biol. 2022;55:102400.
</cite> [<a href="https://doi.org/10.1016/j.redox.2022.102400" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9304680/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35863265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Amorim%20R,%20Sim%C3%B5es%20IC,%20Teixeira%20J,%20Cagide%20F,%20Potes%20Y,%20Soares%20P,%20et%20al.%20Mitochondria-targeted%20anti-oxidant%20AntiOxCIN4%20improved%20liver%20steatosis%20in%20western%20diet-fed%20mice%20by%20preventing%20lipid%20accumulation%20due%20to%20upregulation%20of%20fatty%20acid%20oxidation,%20quality%20control%20mechanism%20and%20antioxidant%20defense%20systems.%20Redox%20Biol.%202022;55:102400." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Choi S-W, Friso S. Modulation of DNA methylation by one-carbon metabolism: a milestone for healthy aging. Nurs Res Pract. 2023;17(4):597–615.</cite> [<a href="https://doi.org/10.4162/nrp.2023.17.4.597" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10375321/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37529262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Choi%20S-W,%20Friso%20S.%20Modulation%20of%20DNA%20methylation%20by%20one-carbon%20metabolism:%20a%20milestone%20for%20healthy%20aging.%20Nurs%20Res%20Pract.%202023;17(4):597%E2%80%93615." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Wang XP, Yan D, Jin XP, Zhang WY, Shi T, Wang X, et al. The role of amino acid metabolism alterations in acute ischemic stroke: From mechanism to application. Pharmacol Res. 2024;207:107313.
</cite> [<a href="https://doi.org/10.1016/j.phrs.2024.107313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39025169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20XP,%20Yan%20D,%20Jin%20XP,%20Zhang%20WY,%20Shi%20T,%20Wang%20X,%20et%20al.%20The%20role%20of%20amino%20acid%20metabolism%20alterations%20in%20acute%20ischemic%20stroke:%20From%20mechanism%20to%20application.%20Pharmacol%20Res.%202024;207:107313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Gojda J, Cahova M. Gut microbiota as the link between elevated BCAA serum levels and insulin resistance. Biomolecules. 2021;11(10):1414.
</cite> [<a href="https://doi.org/10.3390/biom11101414" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8533624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34680047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gojda%20J,%20Cahova%20M.%20Gut%20microbiota%20as%20the%20link%20between%20elevated%20BCAA%20serum%20levels%20and%20insulin%20resistance.%20Biomolecules.%202021;11(10):1414." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Floegel A, Von Ruesten A, Drogan D, Schulze M, Prehn C, Adamski J, et al. Variation of serum metabolites related to habitual diet: a targeted metabolomic approach in EPIC-Potsdam. Eur J Clin Nutr. 2013;67(10):1100–8.
</cite> [<a href="https://doi.org/10.1038/ejcn.2013.147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23942179/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Floegel%20A,%20Von%20Ruesten%20A,%20Drogan%20D,%20Schulze%20M,%20Prehn%20C,%20Adamski%20J,%20et%20al.%20Variation%20of%20serum%20metabolites%20related%20to%20habitual%20diet:%20a%20targeted%20metabolomic%20approach%20in%20EPIC-Potsdam.%20Eur%20J%20Clin%20Nutr.%202013;67(10):1100%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Gar C, Rottenkolber M, Prehn C, Adamski J, Seissler J, Lechner A. Serum and plasma amino acids as markers of prediabetes, insulin resistance, and incident diabetes. Crit Rev Clin Lab Sci. 2018;55(1):21–32.
</cite> [<a href="https://doi.org/10.1080/10408363.2017.1414143" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29239245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gar%20C,%20Rottenkolber%20M,%20Prehn%20C,%20Adamski%20J,%20Seissler%20J,%20Lechner%20A.%20Serum%20and%20plasma%20amino%20acids%20as%20markers%20of%20prediabetes,%20insulin%20resistance,%20and%20incident%20diabetes.%20Crit%20Rev%20Clin%20Lab%20Sci.%202018;55(1):21%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Karlsson T, Winkvist A, Rådjursöga M, Ellegård L, Pedersen A, Lindqvist HM. Identification of single and combined serum metabolites associated with food intake. Metabolites. 2022;12(10): 908.
</cite> [<a href="https://doi.org/10.3390/metabo12100908" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9607433/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36295810/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karlsson%20T,%20Winkvist%20A,%20R%C3%A5djurs%C3%B6ga%20M,%20Elleg%C3%A5rd%20L,%20Pedersen%20A,%20Lindqvist%20HM.%20Identification%20of%20single%20and%20combined%20serum%20metabolites%20associated%20with%20food%20intake.%20Metabolites.%202022;12(10):%20908." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Low DY, Mina TH, Sadhu N, Wong KE, Jain PR, Dalan R, et al. Metabolic variation reflects dietary intake in a multi-ethnic Asian population. medRxiv. 2023:2023.12. 04.23299350.</cite>
</li>
<li id="CR73">
<span class="label">73.</span><cite>Meyer-Ficca M, Kirkland JB. Niacin. Adv Nutr. 2016;7(3):556–8.
</cite> [<a href="https://doi.org/10.3945/an.115.011239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4863271/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27184282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meyer-Ficca%20M,%20Kirkland%20JB.%20Niacin.%20Adv%20Nutr.%202016;7(3):556%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Henninger J, Eliasson B, Smith U, Rawshani A. Identification of markers that distinguish adipose tissue and glucose and insulin metabolism using a multi-modal machine learning approach. Sci Rep. 2021;11(1):17050.
</cite> [<a href="https://doi.org/10.1038/s41598-021-95688-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8382765/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34426590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Henninger%20J,%20Eliasson%20B,%20Smith%20U,%20Rawshani%20A.%20Identification%20of%20markers%20that%20distinguish%20adipose%20tissue%20and%20glucose%20and%20insulin%20metabolism%20using%20a%20multi-modal%20machine%20learning%20approach.%20Sci%20Rep.%202021;11(1):17050." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Aichler M, Borgmann D, Krumsiek J, Buck A, MacDonald PE, Fox JEM, et al. N-acyl taurines and acylcarnitines cause an imbalance in insulin synthesis and secretion provoking β cell dysfunction in type 2 diabetes. Cell Metabolism. 2017;25(6):1334-47. e4.
</cite> [<a href="https://doi.org/10.1016/j.cmet.2017.04.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28591636/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aichler%20M,%20Borgmann%20D,%20Krumsiek%20J,%20Buck%20A,%20MacDonald%20PE,%20Fox%20JEM,%20et%20al.%20N-acyl%20taurines%20and%20acylcarnitines%20cause%20an%20imbalance%20in%20insulin%20synthesis%20and%20secretion%20provoking%20%CE%B2%20cell%20dysfunction%20in%20type%202%20diabetes.%20Cell%20Metabolism.%202017;25(6):1334-47.%20e4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Caldow MK, Ham DJ, Godeassi DP, Chee A, Lynch GS, Koopman R. Glycine supplementation during calorie restriction accelerates fat loss and protects against further muscle loss in obese mice. Clin Nutr. 2016;35(5):1118–26.
</cite> [<a href="https://doi.org/10.1016/j.clnu.2015.08.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26431812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Caldow%20MK,%20Ham%20DJ,%20Godeassi%20DP,%20Chee%20A,%20Lynch%20GS,%20Koopman%20R.%20Glycine%20supplementation%20during%20calorie%20restriction%20accelerates%20fat%20loss%20and%20protects%20against%20further%20muscle%20loss%20in%20obese%20mice.%20Clin%20Nutr.%202016;35(5):1118%E2%80%9326." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Samuel VT, Shulman GI. The pathogenesis of insulin resistance: integrating signaling pathways and substrate flux. J Clin Invest. 2016;126(1):12–22.
</cite> [<a href="https://doi.org/10.1172/JCI77812" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4701542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26727229/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Samuel%20VT,%20Shulman%20GI.%20The%20pathogenesis%20of%20insulin%20resistance:%20integrating%20signaling%20pathways%20and%20substrate%20flux.%20J%20Clin%20Invest.%202016;126(1):12%E2%80%9322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Kosacka J, Woidt K, Toyka KV, Paeschke S, Klöting N, Bechmann I, et al. The role of dietary non-heme iron load and peripheral nerve inflammation in the development of peripheral neuropathy (PN) in obese non-diabetic leptin-deficient ob/ob mice. Neurol Res. 2019;41(4):341–53.
</cite> [<a href="https://doi.org/10.1080/01616412.2018.1564191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30638160/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kosacka%20J,%20Woidt%20K,%20Toyka%20KV,%20Paeschke%20S,%20Kl%C3%B6ting%20N,%20Bechmann%20I,%20et%20al.%20The%20role%20of%20dietary%20non-heme%20iron%20load%20and%20peripheral%20nerve%20inflammation%20in%20the%20development%20of%20peripheral%20neuropathy%20(PN)%20in%20obese%20non-diabetic%20leptin-deficient%20ob/ob%20mice.%20Neurol%20Res.%202019;41(4):341%E2%80%9353." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Abu AlSel BT, Mahmoud AA, Hamed EO, Hakim NA, Sindi AA, Jawad NM, et al. Iron homeostasis-related parameters and hepcidin/ferritin ratio: emerging sex-specific predictive markers for metabolic syndrome. Metabolites. 2024;14(9):473.
</cite> [<a href="https://doi.org/10.3390/metabo14090473" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11434056/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39330480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abu%20AlSel%20BT,%20Mahmoud%20AA,%20Hamed%20EO,%20Hakim%20NA,%20Sindi%20AA,%20Jawad%20NM,%20et%20al.%20Iron%20homeostasis-related%20parameters%20and%20hepcidin/ferritin%20ratio:%20emerging%20sex-specific%20predictive%20markers%20for%20metabolic%20syndrome.%20Metabolites.%202024;14(9):473." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Yang H. Implementation of artificial intelligence in primary care. Korean J Fam Pract. 2023;13(2):73–9.</cite> [<a href="https://scholar.google.com/scholar_lookup?Yang%20H.%20Implementation%20of%20artificial%20intelligence%20in%20primary%20care.%20Korean%20J%20Fam%20Pract.%202023;13(2):73%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Gibson R, Lau C-HE, Loo RL, Ebbels TM, Chekmeneva E, Dyer AR, et al. The association of fish consumption and its urinary metabolites with cardiovascular risk factors: the International Study of Macro-/Micronutrients and Blood Pressure (INTERMAP). Am J Clin Nutr. 2020;111(2):280–90.
</cite> [<a href="https://doi.org/10.1093/ajcn/nqz293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6997096/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31782492/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gibson%20R,%20Lau%20C-HE,%20Loo%20RL,%20Ebbels%20TM,%20Chekmeneva%20E,%20Dyer%20AR,%20et%20al.%20The%20association%20of%20fish%20consumption%20and%20its%20urinary%20metabolites%20with%20cardiovascular%20risk%20factors:%20the%20International%20Study%20of%20Macro-/Micronutrients%20and%20Blood%20Pressure%20(INTERMAP).%20Am%20J%20Clin%20Nutr.%202020;111(2):280%E2%80%9390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Miles FL, Orlich MJ, Mashchak A, Chandler PD, Lampe JW, Duerksen-Hughes P, et al. The biology of veganism: plasma metabolomics analysis reveals distinct profiles of vegans and non-vegetarians in the Adventist Health Study-2 cohort. Nutrients. 2022;14(3):709.
</cite> [<a href="https://doi.org/10.3390/nu14030709" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8839915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35277064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miles%20FL,%20Orlich%20MJ,%20Mashchak%20A,%20Chandler%20PD,%20Lampe%20JW,%20Duerksen-Hughes%20P,%20et%20al.%20The%20biology%20of%20veganism:%20plasma%20metabolomics%20analysis%20reveals%20distinct%20profiles%20of%20vegans%20and%20non-vegetarians%20in%20the%20Adventist%20Health%20Study-2%20cohort.%20Nutrients.%202022;14(3):709." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Palacios-González B, León-Reyes G, Rivera-Paredez B, Ibarra-González I, Vela-Amieva M, Flores YN, et al. Targeted metabolomics revealed a sex-dependent signature for metabolic syndrome in the Mexican population. Nutrients. 2022;14(18):3678.
</cite> [<a href="https://doi.org/10.3390/nu14183678" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9504093/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36145054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palacios-Gonz%C3%A1lez%20B,%20Le%C3%B3n-Reyes%20G,%20Rivera-Paredez%20B,%20Ibarra-Gonz%C3%A1lez%20I,%20Vela-Amieva%20M,%20Flores%20YN,%20et%20al.%20Targeted%20metabolomics%20revealed%20a%20sex-dependent%20signature%20for%20metabolic%20syndrome%20in%20the%20Mexican%20population.%20Nutrients.%202022;14(18):3678." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Santos-Marcos JA, Mora-Ortiz M, Tena-Sempere M, Lopez-Miranda J, Camargo A. Interaction between gut microbiota and sex hormones and their relation to sexual dimorphism in metabolic diseases. Biol Sex Differ. 2023;14(1): 4.
</cite> [<a href="https://doi.org/10.1186/s13293-023-00490-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9903633/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36750874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Santos-Marcos%20JA,%20Mora-Ortiz%20M,%20Tena-Sempere%20M,%20Lopez-Miranda%20J,%20Camargo%20A.%20Interaction%20between%20gut%20microbiota%20and%20sex%20hormones%20and%20their%20relation%20to%20sexual%20dimorphism%20in%20metabolic%20diseases.%20Biol%20Sex%20Differ.%202023;14(1):%204." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Liew S-C, Gupta ED. Methylenetetrahydrofolate reductase (MTHFR) C677T polymorphism: epidemiology, metabolism and the associated diseases. Eur J Med Genet. 2015;58(1):1–10.
</cite> [<a href="https://doi.org/10.1016/j.ejmg.2014.10.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25449138/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liew%20S-C,%20Gupta%20ED.%20Methylenetetrahydrofolate%20reductase%20(MTHFR)%20C677T%20polymorphism:%20epidemiology,%20metabolism%20and%20the%20associated%20diseases.%20Eur%20J%20Med%20Genet.%202015;58(1):1%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>White PJ, McGarrah RW, Grimsrud PA, Tso SC, Yang WH, Haldeman JM, et al. The BCKDH kinase and phosphatase integrate BCAA and lipid metabolism via regulation of ATP-citrate lyase. Cell Metabolism. 2018;27(6):1281-93. e7.
</cite> [<a href="https://doi.org/10.1016/j.cmet.2018.04.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5990471/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29779826/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?White%20PJ,%20McGarrah%20RW,%20Grimsrud%20PA,%20Tso%20SC,%20Yang%20WH,%20Haldeman%20JM,%20et%20al.%20The%20BCKDH%20kinase%20and%20phosphatase%20integrate%20BCAA%20and%20lipid%20metabolism%20via%20regulation%20of%20ATP-citrate%20lyase.%20Cell%20Metabolism.%202018;27(6):1281-93.%20e7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Goni L, Qi L, Cuervo M, Milagro FI, Saris WH, MacDonald IA, et al. Effect of the interaction between diet composition and the PPM1K genetic variant on insulin resistance and β cell function markers during weight loss: results from the nutrient gene interactions in human obesity: implications for dietary guidelines (NUGENOB) randomized trial. Am J Clin Nutr. 2017;106(3):902–8.
</cite> [<a href="https://doi.org/10.3945/ajcn.117.156281" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28768654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goni%20L,%20Qi%20L,%20Cuervo%20M,%20Milagro%20FI,%20Saris%20WH,%20MacDonald%20IA,%20et%20al.%20Effect%20of%20the%20interaction%20between%20diet%20composition%20and%20the%20PPM1K%20genetic%20variant%20on%20insulin%20resistance%20and%20%CE%B2%20cell%20function%20markers%20during%20weight%20loss:%20results%20from%20the%20nutrient%20gene%20interactions%20in%20human%20obesity:%20implications%20for%20dietary%20guidelines%20(NUGENOB)%20randomized%20trial.%20Am%20J%20Clin%20Nutr.%202017;106(3):902%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>He Q, Li J. The evolution of folate supplementation–from one size for all to personalized, precision, poly-paths. J Transl Intern Med. 2023;11(2):128–37.</cite> [<a href="https://doi.org/10.2478/jtim-2023-0087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10318921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37408570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He%20Q,%20Li%20J.%20The%20evolution%20of%20folate%20supplementation%E2%80%93from%20one%20size%20for%20all%20to%20personalized,%20precision,%20poly-paths.%20J%20Transl%20Intern%20Med.%202023;11(2):128%E2%80%9337." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366009/bin/12937_2025_1189_MOESM1_ESM.xls" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (38KB, xls) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366009/bin/12937_2025_1189_MOESM2_ESM.xls" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2.</a><sup> (43KB, xls) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366009/bin/12937_2025_1189_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">12937_2025_1189_MOESM3_ESM.docx</a><sup> (525.9KB, docx) </sup><p>Supplementary Material 3. Supplementary Figure 1. Correlation network of metabolic syndrome components, metabolites, and nutrients. Statistically significant correlations between two metabolites are represented as edges, with positive correlations shown in blue and negative correlations in red. The color gradient and thickness of the edges are proportional to the absolute value of the correlation coefficient. <em>P</em>-values were adjusted for the false discovery rate using the Benjamini‒Hochberg method. ADMA, asymmetric dimethylarginine; Ac-Orn, acetylornithine; Ala, alanine; Arg, arginine; Asn, asparagine; Asp, aspartate; C0, free carnitine; C2, propionylcarnitine; C3, butyrylcarnitine; C5, isovalerylcarnitine; C8, octanoylcarnitine; Ca, calcium; Cit, citrulline; Fe, iron; Gln, glutamine; Glu, glutamate; Gly, glycine; H1, hexose; HDL, high-density lipoprotein; His, histidine; Ile, isoleucine; K, potassium; Leu, leucine; Lys, lysine; Met, methionine; Na, sodium; Orn, ornithine; P, phosphorus; Phe, phenylalanine; Pro, proline; Ser, serine; Thr, threonine; Trp, tryptophan; Tyr, tyrosine; Val, valine; Vit, vitamin.</p>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Data from the Korean Genome and Epidemiology Study are available through the procedure described at <a href="https://biobank.nih.go.kr/eng/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://biobank.nih.go.kr/eng/</a> (accessed on August 12, 2024). In compliance with institutional policies and data privacy regulations, the underlying code and datasets cannot be made publicly available. Nonetheless, all methodological steps and analytical workflows have been transparently described to ensure reproducibility.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nutrition Journal are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12937-025-01189-3"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12937_2025_Article_1189.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366009/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366009/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366009%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366009/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366009/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366009/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830960/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366009/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830960/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366009/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366009/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="JxtXeB74qHugvgTpfzXV01wC7g2EswwQluJYLWBkEn6R50pKic547HJLCx2bxow4">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
