
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Melody: meta-analysis of microbiome association studies for discovering generalizable microbial signatures - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152553848AF44273055384000C6B2785.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="genbio">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359909/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Genome Biology">
<meta name="citation_title" content="Melody: meta-analysis of microbiome association studies for discovering generalizable microbial signatures">
<meta name="citation_author" content="Zhoujingpeng Wei">
<meta name="citation_author_institution" content="Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, WI 53715 USA">
<meta name="citation_author" content="Guanhua Chen">
<meta name="citation_author_institution" content="Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, WI 53715 USA">
<meta name="citation_author" content="Zheng-Zheng Tang">
<meta name="citation_author_institution" content="Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, WI 53715 USA">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="26">
<meta name="citation_firstpage" content="245">
<meta name="citation_doi" content="10.1186/s13059-025-03721-4">
<meta name="citation_pmid" content="40826361">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359909/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359909/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359909/pdf/13059_2025_Article_3721.pdf">
<meta name="description" content="Standard protocols for meta-analysis of association studies are inadequate for microbiome data due to their complex compositional structure, leading to inaccurate and unstable microbial signature selection. To address this issue, we introduce ...">
<meta name="og:title" content="Melody: meta-analysis of microbiome association studies for discovering generalizable microbial signatures">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Standard protocols for meta-analysis of association studies are inadequate for microbiome data due to their complex compositional structure, leading to inaccurate and unstable microbial signature selection. To address this issue, we introduce ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359909/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12359909">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s13059-025-03721-4"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/13059_2025_Article_3721.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12359909%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12359909/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12359909/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359909/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-genbio.png" alt="Genome Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Genome Biology" title="Link to Genome Biology" shape="default" href="https://genomebiology.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Genome Biol</button></div>. 2025 Aug 18;26:245. doi: <a href="https://doi.org/10.1186/s13059-025-03721-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s13059-025-03721-4</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Genome%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Genome%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Genome%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Genome%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Melody: meta-analysis of microbiome association studies for discovering generalizable microbial signatures</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wei%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Zhoujingpeng Wei</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Zhoujingpeng Wei</span></h3>
<div class="p">
<sup>1</sup>Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, WI 53715 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wei%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhoujingpeng Wei</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Guanhua Chen</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Guanhua Chen</span></h3>
<div class="p">
<sup>1</sup>Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, WI 53715 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Guanhua Chen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20ZZ%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Zheng-Zheng Tang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Zheng-Zheng Tang</span></h3>
<div class="p">
<sup>1</sup>Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, WI 53715 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20ZZ%22%5BAuthor%5D" class="usa-link"><span class="name western">Zheng-Zheng Tang</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Biostatistics and Medical Informatics, University of Wisconsin-Madison, Madison, WI 53715 USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Jul 8; Accepted 2025 Jul 31; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12359909  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40826361/" class="usa-link">40826361</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Standard protocols for meta-analysis of association studies are inadequate for microbiome data due to their complex compositional structure, leading to inaccurate and unstable microbial signature selection. To address this issue, we introduce Melody, a framework that generates, harmonizes, and combines study-specific summary association statistics to powerfully and robustly identify microbial signatures in meta-analysis. Comprehensive and realistic simulations demonstrate that Melody substantially outperforms existing approaches in prioritizing true signatures. In the meta-analyses of five studies on colorectal cancer and eight studies on the gut metabolome, we showcase the superior stability, reliability, and predictive performance of Melody-identified signatures.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s13059-025-03721-4.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Absolute abundance, Best subset selection, Compositional data, Meta-analysis, Quasi-multinomial model, Relative abundance, Signature harmonization, Summary statistics</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par2">Advanced sequencing technologies have enabled the study of microbial communities that colonize the human body in a culture-independent manner [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Microbiome association analysis is routinely performed to discover microbial signatures associated with covariates such as health outcomes, exposures, and molecular traits. The signature identified in one study is often hard to generalize to other cohorts and populations because of the small sample size in a single study. A natural way to increase sample size is to perform meta-analyses. Recent years have witnessed a surge in conducting meta-analyses of microbiome association studies [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>–<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Unfortunately, standard meta-analysis strategies commonly used in clinical and genomics studies fail to address the unique characteristics of microbiome data, undermining the ability of meta-analysis to identify generalizable microbial signatures.</p>
<p id="Par3">Microbiome data generated from marker-gene or metagenomics sequencing contain sequencing read counts allocated to various microbial features, such as taxonomic and functional units [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. The sequencing depth (the total number of reads obtained for a sample) is not correlated with the actual microbial load (the total amount of microorganisms or their DNA volumes) in the sample. Therefore, microbiome data are compositional data that represent the relative abundance (RA) of microbial features. These counts are distorted from the underlying absolute abundance (AA) per unit volume [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. Although techniques have been developed to recover AA data [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>], the cost is prohibitive and the effectiveness is debatable because they cannot correct measurement biases arising from different efficiencies (e.g., DNA extraction efficiency, primer binding, and amplification efficiency) with which various features are measured [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. The RA of all features would shift even if only one feature changes in AA. The small shift may be too subtle to affect the association analysis of a single study but would become more pronounced and impactful as the sample size and data heterogeneity accumulate in the meta-analysis.</p>
<p id="Par4">Several compositionality-aware methods have been developed for microbiome association analysis [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. These methods employ different strategies to address compositionality. Methods such as ANCOM-BC2 [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>], LinDA [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>], and radEmu [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>] explicitly estimate and correct compositional bias to recover AA association coefficients. SRI [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>] keeps the analysis on the relative scale but marginalizes over a prior for each sample’s unknown microbial load, so compositionality is treated as uncertainty rather than bias. DR [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>] ranks features using log-ratio comparisons to a reference frame. While these methods are useful, they are not designed to estimate and harmonize AA association coefficients across studies for meta-analysis.</p>
<p id="Par5">In addition to compositionality, other characteristics of microbiome data further compound the challenges of microbiome association meta-analyses. The sequencing depth strongly influences the sequencing read counts. Methods that fail to consider the variation in sequencing depth across samples can lead to a high number of false positives, particularly when the depths vary significantly between groups of comparison [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. While rarefaction is commonly used to address uneven sequencing depth, it may reduce statistical power in certain analyses due to data loss introduced by subsampling [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. Moreover, for many rare features, insufficient sampling and the absence of features result in an excessive number of zero counts. Some features may be completely missing in a subset of studies in meta-analysis. While the imputation of zero to non-zero values is a widely adopted practice in processing microbiome data, consensus on the best method is lacking even for data from a single study, and it may introduce additional bias to the analysis [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. Filtering out rare features is another common practice under the belief that rare features are uninformative. However, recent research has noted an alarming phenomenon that analysis with different filtering criteria can result in identifying very different sets of microbial signatures [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>].</p>
<p id="Par6">Pooling individual-level data and combining summary data are two types of meta-analysis strategies, each of which faces different challenges in microbiome association meta-analyses. Pooled-data meta-analysis requires careful harmonization of individual-level data that usually involves correcting batch effects in microbiome data across studies. The distribution of microbiome data can substantially differ among studies involving various age groups, genders, geographic regions, and measurement time points. Batch effects can also occur in sequencing experiments due to differences in reagents, equipment, protocols, and personnel. More importantly, batch effects are often entangled with association effects in microbiome studies [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>–<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. Correcting batch effects while preserving the signal of interest is challenging due to the aforementioned characteristics of microbiome data. Prior studies in genomics have shown that batch effect correction may distort associations in pooled-data meta-analyses [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>–<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>].</p>
<p id="Par7">Summary-data meta-analysis circumvents the need for individual-level data harmonization, as summary statistics are generated for each study separately. This type of meta-analysis is preferable to pooled-data meta-analysis when there are differences in sample ascertainment and in data measurements and availability across studies. If the individual-level data is not shared, the logistical burden and privacy concerns can be significantly reduced, especially when the data contain sensitive information such as genetics or medical history. For these critical reasons, summary-data meta-analysis is widely adopted in many research fields, such as genetic association studies [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. However, standard approaches for generating and combining summary statistics are inadequate for microbiome data due to their unique characteristics mentioned above.</p>
<p id="Par8">We introduce a summary-data meta-analysis framework named Melody (meta-analysis for selecting microbial drivers in compositionality). Melody generates accurate and robust summary association statistics between the RA of microbial features and the covariate of interest by respecting the microbiome data characteristics. The procedure does not require microbiome data normalization, rarefaction, zero imputation, or batch effect correction across studies. By harmonizing and combining the summary statistics across studies, Melody identifies microbial signatures with their AA consistently associated with the covariate in all studies. Melody allows study-specific confounder adjustments, accommodates correlated samples, and is computationally efficient.</p>
<p id="Par9">The recovery of AA associations in Melody relies on the assumption that the true signatures are “drivers,” defined as the minimal set of microbial features whose changes in AA are sufficient to explain the association signal observed at the RA level. The concept of driver signatures is illustrated in Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S1. The figure demonstrates that the underlying AA association profile compatible with the same RA association profile (referred to as “putative AA association”) is not unique, leading to ambiguity in inferred signatures. Driver signatures are those features with nonzero effects in the sparsest putative AA association profile. The figure also shows that driver signatures may or may not overlap with features exhibiting nonzero RA associations. In subsequent sections, we show that focusing on identifying drivers is crucial for harmonizing RA summary statistics across studies and for discovering stable, generalizable microbial signatures in meta-analyses.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Melody overview</h3>
<p id="Par10">Melody is described in the “<a href="#Sec9" class="usa-link">Methods</a>” section; its technical details are provided in the Additional file <a href="#MOESM1" class="usa-link">1</a>: Supplementary Note A, and its schematic overview is shown in Fig. <a href="#Fig1" class="usa-link">1</a>. For each study, we link the microbiome count data with the covariate of interest using a quasi-multinomial regression model and obtain summary statistics consisting of the estimates of the RA association coefficients and their variances. The RA association coefficients are estimated based on the scaled score statistics, which are numerically stable, statistically accurate, and computationally efficient [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>, <a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. The quasi-multinomial framework accommodates overdispersion in microbiome count data and mitigates the risk of inaccurate inference associated with the strict multinomial assumption. The model easily accommodates confounder adjustments and naturally handles correlated samples (e.g., from longitudinal studies). The RA association coefficients in the model are formulated at the log-ratio scale, with a specified feature as the reference. The RA association coefficients are expressed as the shifted AA association coefficients by a constant (“<a href="#Sec9" class="usa-link">Methods</a>” section).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359909_13059_2025_3721_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd40/12359909/c27ca662333b/13059_2025_3721_Fig1_HTML.jpg" loading="lazy" id="MO1" height="212" width="709" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Schematic overview of Melody. The Melody framework begins by generating summary statistics for each study using the quasi-multinomial model. In the model, the vector of true proportions <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e329"><msub><mi mathvariant="bold">P</mi><mi>i</mi></msub></math></span> is allowed to follow an arbitrary distribution, and only its mean is modeled to link with the covariate of interest <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e341"><msub><mi>X</mi><mi>i</mi></msub></math></span>. The <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e352"><msubsup><mi>β</mi><mi>k</mi><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup></math></span> is the relative abundance (RA) association coefficient of feature <em>k</em> given the reference feature <em>R</em>. The panel for summary statistics displays the estimates of the RA association coefficient (open dots) and the estimates plus or minus standard errors (horizontal lines). Next, the RA coefficient estimates in each study <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e376"><mi>ℓ</mi></math></span> are shifted by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e385"><msub><mi>δ</mi><mi>ℓ</mi></msub></math></span>. These <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e396"><msub><mi>δ</mi><mi>ℓ</mi></msub></math></span> values are jointly tuned with the sparsity level hyperparameter <em>s</em> for the best subset selection. In this illustration, the optimal value for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e410"><msub><mi>δ</mi><mn>1</mn></msub></math></span> is close to zero, for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e421"><msub><mi>δ</mi><mn>2</mn></msub></math></span> is positive, and for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e433"><msub><mi>δ</mi><mn>3</mn></msub></math></span> is negative. As a result, the RA coefficients in Study 2 are shifted to the right, and those in Study 3 are shifted to the left to obtain the corresponding estimates of study-specific absolute abundance (AA) association coefficients (solid dots). These study-specific coefficients are then combined to estimate meta AA association coefficients. The optimal <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><math id="d33e444"><msub><mi>δ</mi><mi>ℓ</mi></msub></math></span> values ensure that the RA coefficients are shifted appropriately to harmonize signals across studies and recover sparse meta AA associations</p></figcaption></figure><p id="Par11">Next, we combine the RA summary statistics across studies to estimate the sparse meta AA association coefficients. This meta-analysis is framed as a best subset selection problem [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>, <a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>], where we employ a cardinality constraint with the subset size hyperparameter <em>s</em> to encourage the sparsity of meta AA association coefficients. To recover meta coefficients at the AA level using summary statistics at the RA level, we introduce a hyperparameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><math id="d33e468"><msub><mi>δ</mi><mi>ℓ</mi></msub></math></span> in each study <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><math id="d33e479"><mi>ℓ</mi></math></span> to shift the RA association coefficient estimates, producing different versions of putative AA association coefficients by varying <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><math id="d33e489"><msub><mi>δ</mi><mi>ℓ</mi></msub></math></span>. These study-specific AA coefficients are then combined to estimate meta AA coefficients. The hyperparameters <em>s</em> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><math id="d33e503"><msub><mi>δ</mi><mi>ℓ</mi></msub></math></span>’s are jointly tuned using the Bayesian information criterion (BIC) to balance the trade-off between the model fit and sparsity. As illustrated in Fig. <a href="#Fig1" class="usa-link">1</a>, the optimal <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><math id="d33e517"><msub><mi>δ</mi><mi>ℓ</mi></msub></math></span> values shift the RA coefficients by an appropriate amount to identify a version of putative AA association coefficients that are sparse and consistent across studies. The Melody-identified driver signatures are microbial features with non-zero estimates of meta AA association coefficients under the optimal hyperparameters.</p>
<p id="Par12">It is important to note that while we need to select a reference feature for generating summary statistics, the signature selection is not sensitive to the choice of the reference. We can even choose different references for different studies (see the sensitivity analysis in the real data application). This is because the reference effects across studies are offset by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><math id="d33e530"><msub><mi>δ</mi><mi>ℓ</mi></msub></math></span>’s in the process of estimating the sparse meta AA association coefficients.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Simulations</h3>
<p id="Par13">In the simulations, we compared Melody with the MMUPHin summary-data meta-analysis method and several pooled-data meta-analysis approaches for selecting microbial signatures with differential abundance across two groups of interest. MMUPHin [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>] is an R package that includes a batch-effect correction algorithm and a summary-data meta-analysis method. The meta-analysis first generates summary statistics using MaAsLin2 [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] within individual studies and then combines these summary statistics using standard meta-analysis methods [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] for association testing. In the pooled-data meta-analysis, the pooled data were analyzed using representative microbiome association tests ALDEx2 [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>], ANCOM-BC2 [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>], and the blocked Wilcoxon test (BW) [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>], as well as a penalized regression method CLR-LASSO [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. In ALDEx2 and ANCOM-BC2, we adjusted for studies as confounders. In BW, we rarefied count data within each study and performed the blocked Wilcoxon rank-sum test with studies specified as blocks. In CLR-LASSO, we applied LASSO logistic regression [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>] using centered log-ratio (CLR) transformed microbiome data as predictors. Melody was applied to the original data, and the compared methods were applied to both the original data and study-effect corrected data using the MMUPHin batch-effect correction algorithm. We compared the performance using the area under the precision-recall curve (AUPRC) for signature selection. Specifically, the precision-recall curves of the test-based methods (MMUPHin, BW, ALDEx2, ANCOM-BC2) were constructed by changing the <em>p</em>-value cutoff, and those of the penalized methods (Melody and CLR-LASSO) were constructed by varying the hyperparameter that controls the signature sparsity level.</p>
<p id="Par14">We simulated microbiome data for five studies that resemble the structure of five real microbiome datasets [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>] (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S2). The detailed simulation strategy is described in the “<a href="#Sec9" class="usa-link">Methods</a>” section. In brief, we consider a wide range of settings by varying the study sample size, the number of signatures, the degree of sign imbalance among signatures, and the degree of sequencing depth unevenness between the two groups of interest. Melody produces the highest AUPRC values in all settings and is robust with varying degrees of sign imbalance and sequencing depth unevenness (Fig. <a href="#Fig2" class="usa-link">2</a>). The AUPRC of ANCOM-BC2 reduces substantially as the degrees of the two factors increase. The AUPRC of MMUPHin, CLR-LASSO, BW, and ALDEx2 reduce significantly as the degree of sign imbalance increases. When using the study-effect corrected data, the AUPRC of MMUPHin slightly increases, while the pooled-data meta-analysis methods show minimal change. We performed another set of simulations for the meta-analysis of studies with correlated samples and compared Melody with MMUPHin and ANCOM-BC2, which can handle correlated samples. The patterns of the results remain the same (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S3).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359909_13059_2025_3721_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd40/12359909/8eab5f53cf40/13059_2025_3721_Fig2_HTML.jpg" loading="lazy" id="MO2" height="432" width="708" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Precision-recall evaluation of different meta-analysis signature selection approaches in simulated data with independent samples from five studies. Melody was applied to the original data of the five studies, and the other methods were applied to both the original data and study-effect corrected data using the MMUPHin batch-effect correction algorithm. The <em>x</em>-axis represents the number of signatures. The left and right sections show the results under different sample size settings (the sample size range for the five studies is 100–180 or 20–60). Within each section, the columns of the plots represent various degrees of sign imbalance among signatures (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><math id="d33e600"><mrow><mi>p</mi><mi>o</mi><mi>s</mi><mo>%</mo></mrow></math></span> denotes the percentage of signatures with positive effects); the rows of the plots represent various degrees of sequencing depth unevenness between the two groups of interest (<em>u</em> denotes the sequencing depth relative change ratio for the group with higher sequencing depth, and 0 indicates no unevenness). Each panel displays the mean area under the precision-recall curve (AUPRC) with ± standard errors (indicated by error bars) based on 100 simulation replicates</p></figcaption></figure><p id="Par15">To strengthen our benchmarking, we ran an additional suite of simulation studies based on a different data-generation scheme (Additional file <a href="#MOESM1" class="usa-link">1</a>: Supplementary Note B). We first simulated AA data, introduced AA-level differential abundance, and then added study-specific, feature-level measurement biases to mimic technical batch effects. We further varied the batch-effect magnitude, number of studies, and association strength. Across all of these scenarios, Melody consistently outperforms competing methods (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S4).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Meta-analysis of microbiome association studies for colorectal cancer</h3>
<p id="Par16">We conducted a meta-analysis of five metagenomics studies [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>–<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>] for colorectal cancer (CRC) to identify microbial signatures associated with CRC. The data contain 849 gut bacterial species measured on 574 fecal samples across the five studies (Additional file <a href="#MOESM1" class="usa-link">1</a>: Table S1). Microbial communities display a pattern of cross-study heterogeneity (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S2). The sequencing depth between CRC cases and controls is significantly different in some studies (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S5). Our main analysis focused on 401 microbial species with at least <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><math id="d33e649"><mrow><mn>20</mn><mo>%</mo></mrow></math></span> prevalence across all samples in the five studies. We applied the methods that were compared in the simulations to analyze the data. For the test-based methods, the signatures were selected using the Benjamini-Hochberg (BH) procedure [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>] at the false discovery rate (FDR) level of 0.05. For CLR-LASSO, the signatures were selected based on the best LASSO model. The model tuning was performed via five-fold cross-validation using the default settings of the cv.glmnet function from the glmnet R package (v4.1-8), which selects the hyperparameter by minimizing the cross-validated binomial deviance.</p>
<p id="Par17">The meta-analysis results of all methods are presented in Additional file <a href="#MOESM2" class="usa-link">2</a>: Table S2. Melody identified 51 signatures for CRC (Fig. <a href="#Fig3" class="usa-link">3</a>a), with the top two signatures <em>Fusobacterium nucleatum</em> and <em>Parvimonas micra</em> being the well-known species promoting CRC [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>, <a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. Using original data, MMUPHin, CLR-LASSO, BW, ALDEx2, and ANCOM-BC2 selected 84, 94, 110, 44, and 72 signatures, respectively. Signatures identified on the study-effect corrected data are similar to those identified on the original data. The top-selected species and their rankings vary markedly across methods (Fig. <a href="#Fig3" class="usa-link">3</a>b, Additional file <a href="#MOESM1" class="usa-link">1</a>: Figs. S6 and S7).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359909_13059_2025_3721_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd40/12359909/35763a4ad923/13059_2025_3721_Fig3_HTML.jpg" loading="lazy" id="MO3" height="614" width="708" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Meta-analysis results from five metagenomics studies for identifying signatures for colorectal cancer (CRC). <strong>a</strong> Estimates of the absolute-abundance (AA) association coefficient for the 51 signatures identified by Melody. The scatter plot displays the estimates of the meta coefficients, and the heatmap displays the estimates of the study-specific coefficients. <strong>b</strong> Heatmap of the Jaccard index measuring the overlap of the top 51 selected species across different methods. The dendrogram is constructed using hierarchical clustering of the Jaccard index matrix. Results of all methods were generated on the original data. <strong>c</strong> Heatmap of the Jaccard index measuring the overlap of the Melody-identified signatures across different reference configurations. The ref1 represents the scenario where we use species <em>Coprococcus catus</em> as the reference in all studies. The ref2-ref5 represent scenarios where we replace ref1 with a different reference and use the reference in all studies. The ref.mix represents the scenario where we use different references for different studies (ref1-ref5 for CRC1-CRC5). The number in parentheses is the count of the identified signatures</p></figcaption></figure><p id="Par18">We used species <em>Coprococcus catus</em> as the reference when generating Melody summary statistics for individual studies. We also chose other references and evaluated the sensitivity of the Melody result. Additionally, we used different references for different studies and repeated the analysis. The Melody signatures are highly consistent across these reference configurations (Fig. <a href="#Fig3" class="usa-link">3</a>c). Although the RA association coefficient estimates show noticeable differences across various choices of reference, the Melody-recovered AA association coefficients are almost invariant (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S8). The AA coefficients also show better harmonization with consistent association directions across studies than the RA coefficients.</p>
<p id="Par19">We evaluated the stability of signature selection from three perspectives. (1) Is the selection stable under different subsets of the samples? We conducted a sensitivity analysis by removing one study at a time from the meta-analysis. Specifically, we excluded all samples from a single study, identified signatures based on the remaining four studies, and evaluated the overlap of the signatures with those identified using the full dataset comprising all five studies. (2) Is the selection stable after applying different prevalence filters? Our main analysis was on <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><math id="d33e723"><mrow><mi>K</mi><mo>=</mo><mn>401</mn></mrow></math></span> species after applying the 20% prevalence filter. We also identified signatures based on all <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><math id="d33e735"><mrow><mi>K</mi><mo>=</mo><mn>849</mn></mrow></math></span> species, kept the subset of the signatures within the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><math id="d33e747"><mrow><mi>K</mi><mo>=</mo><mn>401</mn></mrow></math></span> species list, and evaluated the overlap of the subset with those selected in the main analysis. (3) Is the selection stable if analyzing a subcomposition? We identified signatures based on a subcomposition with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><math id="d33e759"><mrow><mi>K</mi><mo>=</mo><mn>267</mn></mrow></math></span> species under order <em>Clostridiales</em>. We evaluated the overlap of the identified signatures with the subset of main-analysis-identified signatures within the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><math id="d33e775"><mrow><mi>K</mi><mo>=</mo><mn>267</mn></mrow></math></span> species list. We performed 100 bootstrap resamples of the combined data (i.e., bootstrap resampling by randomly drawing, with replacement, a sample of the same size from each study, and then combined the resampled datasets across studies), repeated these stability evaluations on the bootstrap samples, and reported the Jaccard index to measure set overlap (Fig. <a href="#Fig4" class="usa-link">4</a>). When one study is excluded from the meta-analysis, Melody exhibits higher stability compared to MMUPHin, CLR-Lasso, ALDEx2, and ANCOM-BC2 (Wilcoxon rank-sum test FDR-adjusted <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><math id="d33e790"><mrow><mi>P</mi><mo>&lt;</mo><mn>0.001</mn></mrow></math></span>); BW shows comparable stability to Melody when CRC1 or CRC4 is excluded, and has slightly higher stability when CRC2, CRC3, or CRC5 are excluded (Wilcoxon rank-sum test FDR-adjusted <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><math id="d33e802"><mrow><mi>P</mi><mo>&lt;</mo><mn>0.1</mn></mrow></math></span>). When the set of microbial features is larger (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><math id="d33e814"><mrow><mi>K</mi><mo>=</mo><mn>849</mn></mrow></math></span>), the stability of Melody is much higher than that of ALDEx2, ANCOM-BC2, BW, and CLR-Lasso (Wilcoxon rank-sum test FDR-adjusted <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><math id="d33e826"><mrow><mi>P</mi><mo>&lt;</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>6</mn></mrow></msup></mrow></math></span>). When the set of microbial features is smaller (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><math id="d33e843"><mrow><mi>K</mi><mo>=</mo><mn>267</mn></mrow></math></span>), the stability of Melody is much higher than that of BW, CLR-Lasso, and MMUPHin (Wilcoxon rank-sum test FDR-adjusted <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><math id="d33e855"><mrow><mi>P</mi><mo>&lt;</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>6</mn></mrow></msup></mrow></math></span>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359909_13059_2025_3721_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd40/12359909/48d5674764d1/13059_2025_3721_Fig4_HTML.jpg" loading="lazy" id="MO4" height="632" width="709" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of the stability of signature selection of different methods when performing two sets of meta-analyses based on metagenomics studies for colorectal cancer. Melody was applied to the original data, and the other methods were applied to both the original data and study-effect corrected data. The Jaccard index measures the overlap of the identified signatures from two sets of meta-analyses. Upper panel: Analysis 1 is based on data from all five studies, and Analysis 2 is based on data with one study left out. Lower left panel: Analysis 1 includes <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><math id="d33e877"><mrow><mi>K</mi><mo>=</mo><mn>401</mn></mrow></math></span> species, and Analysis 2 includes all <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><math id="d33e889"><mrow><mi>K</mi><mo>=</mo><mn>849</mn></mrow></math></span> species. Lower right panel: Analysis 1 includes <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><math id="d33e901"><mrow><mi>K</mi><mo>=</mo><mn>401</mn></mrow></math></span> species, and Analysis 2 includes <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><math id="d33e913"><mrow><mi>K</mi><mo>=</mo><mn>267</mn></mrow></math></span> species under order <em>Clostridiales</em>. Box plots were derived from 100 bootstrap samples of the complete data from the five studies. Each box indicates the median (center line), the first and third quartiles (box edges), whiskers extending to 1.5 times the interquartile range, and individual outliers (dots)</p></figcaption></figure><p id="Par20">In addition, we evaluated the out-of-sample prediction power of the top-selected species using the leave-one-study-out (LOSO) strategy. We held out one study for prediction evaluation and randomly split samples from the remaining four studies into a feature selection set and a prediction model training set at a sample size ratio of 3:1. We performed the meta-analysis on the feature selection set to pick the top <em>N</em> CRC-associated species. We then used the sample proportions of these species in the training set to train a random forest model for CRC prediction. Finally, we calculated the area under the receiver operating characteristic (AUROC) of the best random forest model on the hold-out study. We repeated the procedure 100 times with different random splits of the samples and reported the average AUROC (Fig. <a href="#Fig5" class="usa-link">5</a>). The Melody-identified top species have substantially higher AUROC than those identified by the test-based methods in CRC1, CRC2, and CRC5 (Wilcoxon rank-sum test FDR-adjusted <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><math id="d33e938"><mrow><mi>P</mi><mo>&lt;</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>5</mn></mrow></msup></mrow></math></span>), and are comparable to the highest AUROC in other studies. The top species identified by CLR-LASSO performed well in CRC1 but much worse in CRC2, CRC4, and CRC5. Overall, the group of species prioritized by Melody has the highest average AUROC in CRC prediction (Wilcoxon rank-sum test FDR-adjusted <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><math id="d33e954"><mrow><mi>P</mi><mo>&lt;</mo><msup><mn>10</mn><mrow><mo>-</mo><mn>10</mn></mrow></msup></mrow></math></span>), whereas the species prioritized by other methods yielded similar results (Combined panel in Fig. <a href="#Fig5" class="usa-link">5</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359909_13059_2025_3721_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd40/12359909/2672396ed9f4/13059_2025_3721_Fig5_HTML.jpg" loading="lazy" id="MO5" height="402" width="708" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of colorectal cancer (CRC) prediction using the top species identified by different meta-analysis methods. One study was held out each time for prediction evaluation, and samples from the remaining studies were randomly split into a feature selection set and a prediction model training set. Meta-analysis methods were applied to the feature selection set to identify the top <em>N</em> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><math id="d33e983"><mrow><mi>N</mi><mo>=</mo><mn>15</mn><mo>,</mo><mo>…</mo><mo>,</mo><mn>80</mn></mrow></math></span>) CRC-associated species. Melody was applied to the original data, and the other methods were applied to both the original data and study-effect corrected data. A random forest model for CRC prediction was trained using the sample proportions of the top species in the training set, and the best model was evaluated on the hold-out study based on the area under the receiver operating characteristic (AUROC). The procedure was repeated 100 times with different random splits of the samples. In the first five panels, the average AUROC is provided in each panel, with the hold-out study ID listed at the top. The “Combined” panel displays the average AUROC across all hold-out studies</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">Meta-analysis of microbiome-metabolome association studies</h3>
<p id="Par21">We conducted another meta-analysis of eight microbiome-metabolome association studies [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>–<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>] to identify microbial signatures associated with human gut metabolites. Most of these studies are case-control studies with one group of individuals with a specific medical condition and the other group of healthy control individuals (Additional file <a href="#MOESM1" class="usa-link">1</a>: Table S3). The raw data contain 12195 gut bacterial genera and 1151 HMDB-annotated [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>] metabolites measured on 2127 fecal samples across the eight studies. We focused on metabolites and genera with at least <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><math id="d33e1017"><mrow><mn>10</mn><mo>%</mo></mrow></math></span> prevalence within individual studies. Most metabolites are available in only a subset of studies (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S9). We considered metabolites that are present in at least two studies. The final datasets for analysis contain 450 metabolites and 101 genera, resulting in 43722 metabolite-genus pairs. There is notable heterogeneity among studies in study designs, microbiome sequencing methods, and metabolite measurements (Additional file <a href="#MOESM1" class="usa-link">1</a>: Table S3). In particular, metabolite levels cannot be directly compared between studies due to differences between metabolomics platforms. Short-chain fatty acids, for example, are primarily detected using gas chromatography-mass spectrometry (GC-MS) and are rarely detected by liquid chromatography-mass spectrometry (LC-MS) due to their poor ionization efficiency. Therefore, the pooled-data meta-analysis is unsuitable and we performed Melody and MMUPHin summary-data meta-analyses on the original data. In both methods, we analyzed each metabolite variable as a covariate of interest while adjusting the disease status as a confounder when generating summary statistics for individual studies. For MMUPHin, the signatures for each metabolite were selected using the BH procedure at the FDR level of 0.05.</p>
<p id="Par22">Melody completed the analysis within 20 min, while MMUPHin took 25 min on a laptop (Apple M1 Pro 3.2GHz processor). Melody identified 5847 metabolite-genus pairs, among which 2499 have positive and 3348 have negative associations (Additional file <a href="#MOESM2" class="usa-link">2</a>: Table S4). MMUPHin identified 5921 pairs, among which 2065 have positive and 3856 have negative associations (Additional file <a href="#MOESM2" class="usa-link">2</a>: Table S5). Figure <a href="#Fig6" class="usa-link">6</a> displays the meta association coefficient estimates of the two methods for the same set of top metabolites, which are associated with a large number of genera. Melody identifies a greater number of genera that are positively associated with the majority of top metabolites, compared to MMUPHin (Fig. <a href="#Fig6" class="usa-link">6</a>). These genera are associated with much fewer metabolites in MMUPHin results (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S10b). Many of them, such as <em>Anaerostipes</em>, <em>Blautia</em>, and <em>Roseburia</em>, are known to play essential roles in key metabolic processes, including the fermentation of dietary fibers and diverse carbohydrates [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>, <a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>], and are closely associated with the production of short-chain fatty acids, bile acids, and amino acid derivatives. These genera have also been linked with multiple human complex diseases such as metabolic syndrome, inflammatory bowel disease, and obesity [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>–<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. Besides the direct engagement of the genus in metabolite production, consumption, or degradation, correlations between metabolites and genera can also result from indirect associations, such as interactions between gut bacteria or co-abundant metabolites.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359909_13059_2025_3721_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd40/12359909/7fd073bbcc04/13059_2025_3721_Fig6_HTML.jpg" loading="lazy" id="MO6" height="666" width="708" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Heatmaps of meta association coefficient estimates for the Melody and MMUPHin identified metabolite-genus pairs. The coefficient is shown as zero if the method does not select the metabolite-genus pair. The rows of both heatmaps are for the same set of top metabolites that are associated with at least 30 genera in Melody results. Most of the metabolites are grouped into three HMDB superclasses (names shown on the leftmost), while the remaining metabolites are placed at the bottom rows. The columns of the two heatmaps show genera associated with at least 70 metabolites, one from Melody results and the other from MMUPHin results. The names of the genera that have disproportionately positive associations with the top metabolites are in bold. Bacterial orders are summarized by the color code. Rows within each superclass and columns are clustered by Euclidean distance</p></figcaption></figure><p id="Par23">We further examined metabolites for which the results of the two methods were strikingly different. Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S11 highlights one such metabolite, 4-trimethylammoniobutanoate (also known as gamma-butyrobetaine), a key gut microbial intermediate in the metabolism of L-carnitine to trimethylamine N-oxide (TMAO) with demonstrated proatherogenic effects in preclinical models [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. Melody identified 13 positive-associated genera and 10 negative-associated genera, whereas MMUPHin identified 7 positive-associated genera and 21 negative-associated genera. For each method, we removed 5 genera with the highest positive meta association identified by the method and reran the meta-analysis. The Melody signature selection is stable after the genus removal. MMUPHin identified many new positive-associated genera, and the set of negative-associated genera also changed substantially. This is a global phenomenon, not limited to a few metabolites. We performed the same evaluation for all 106 metabolites with at least 5 positive-associated genera detected by Melody and MMUPHin. The overlap of the signatures before and after genus removal is much higher in Melody than in MMUPHin for most of these metabolites (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S12). The stability of the Melody results shown in this experiment is mainly attributed to its ability to capture the fluid association signals in the compositional data. This evaluation underscores the importance of addressing compositionality to ensure the stability and reliability of results in microbiome association meta-analyses.</p></section></section><section id="Sec7"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par24">We have developed Melody to address major challenges in the meta-analysis of microbiome association studies. Melody’s key innovations are (a) generating accurate and robust summary statistics by accounting for the unique characteristics of microbiome data; (b) recovering meta association coefficients at the absolute abundance level and identifying driver signatures consistent across studies. Melody has a unique combination of features that make it desirable for practical use: (1) no need to share individual-level data or to correct study effects; (2) no need to perform data normalization, rarefaction, or zero imputation; (3) no need to assume that the same set of microbial features is measured and analyzed across studies; (4) allows for different designs across studies (e.g., some studies are cross-sectional, and some are longitudinal); (5) easily adjusts for confounders and allows different sets of confounders available across studies; (6) fast computation for large-scale association scans.</p>
<p id="Par25">Melody tackles compositionality in a novel way. It corrects study-specific compositional biases by jointly tuning the per-study scaling factors <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><math id="d33e1099"><msub><mi>δ</mi><mi>ℓ</mi></msub></math></span> and the signature size <em>s</em> within a best subset selection framework, yielding consistent AA estimates. Joint tuning is critical and simply fixing <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><math id="d33e1113"><msub><mi>δ</mi><mi>ℓ</mi></msub></math></span> to the median of the RA coefficient estimates fails to achieve comparable performance (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S13). However, Melody’s effectiveness rests on a sparsity assumption. In simulations where more than half of the features are driver signatures, the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><math id="d33e1127"><msub><mi>δ</mi><mi>ℓ</mi></msub></math></span> estimates degrade and Melody does not outperform the compared methods (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S14). Note, however, that observing RA shifts in many features does not automatically imply a dense signal; large effects from a small set of drivers can induce widespread apparent shifts because of compositional constraints.</p>
<p id="Par26">As demonstrated in the real data application, the set of Melody-identified signatures is not sensitive to the prevalence filter. However, like any method for high-dimensional feature selection, including lots of noise features could adversely impact the selection accuracy. Therefore, we still suggest applying a prevalence filter before data analysis if investigators are not interested in detecting very rare features. The filter can be applied to pooled samples of all studies (as in the meta-analysis for CRC) or samples within each study in case a feature may be more prevalent in some studies and worth keeping (as in the meta-analysis for metabolome). Melody naturally accommodates different sets of features across studies in meta-analysis.</p>
<p id="Par27">This article focuses on identifying microbial signatures that are shared across studies, as these are more likely to be generalizable to broader populations. It can also be of interest to identify microbial signatures with heterogeneous associations across populations (e.g., microbial features associated with the covariate in females but not in males) [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]. Melody can be extended to detect such heterogeneous associations by adapting random-effects meta-analysis and meta-regression techniques [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>].</p></section><section id="Sec8"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par28">Melody is a new meta-analysis framework that enables the identification of true microbial signatures with absolute-abundance level associations using microbiome relative-abundance data. Compared to existing meta-analysis approaches, Melody substantially improves cross-study signal harmonization and enhances signature detection accuracy and stability. With the ever-increasing number of microbiome datasets, we anticipate that Melody will facilitate the accumulation of sufficiently large sample sizes to accelerate the reliable discovery of generalizable microbiome signatures. As a general meta-analysis framework, Melody can be applied to the integration of association studies involving high-dimensional compositional omics data. We have provided an efficient R package (miMeta) for the broad utility of Melody.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Melody framework</h3>
<p id="Par29"><em>Generating summary statistics for individual studies.</em> The first step of Melody is to obtain summary association statistics between microbial composition and the covariate of interest in each participating study. Suppose we have <em>n</em> independent microbiome samples with <em>K</em> features in a given study. For sample <em>i</em>, let <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><math id="d33e1173"><msub><mi>X</mi><mi>i</mi></msub></math></span> denote the covariate, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><math id="d33e1184"><msub><mi>N</mi><mi>i</mi></msub></math></span> denote the sequencing depth of the microbiome data, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><math id="d33e1196"><msub><mi>Y</mi><mrow><mi mathvariant="italic">ik</mi></mrow></msub></math></span> denote the number of sequencing reads assigned to feature <em>k</em> (i.e., microbiome count data), and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><math id="d33e1212"><msub><mi>W</mi><mrow><mi mathvariant="italic">ik</mi></mrow></msub></math></span> denote the unobserved AA of feature <em>k</em>. It is natural to model the mean of AA as</p>
<table class="disp-formula p" id="Equ6"><tr><td class="formula"><math id="d33e1228" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mi>E</mi><mrow><mo stretchy="false">(</mo><msub><mi>W</mi><mrow><mi mathvariant="italic">ik</mi></mrow></msub><mo stretchy="false">)</mo></mrow><mo>=</mo><mo>exp</mo><mrow><mo stretchy="false">(</mo><msub><mi>β</mi><mrow><mn>0</mn><mi>k</mi></mrow></msub><mo>+</mo><msub><mi>β</mi><mi>k</mi></msub><msub><mi>X</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow><mo>,</mo><mspace width="4pt"></mspace><mi>k</mi><mo>=</mo><mn>1</mn><mo>,</mo><mo>…</mo><mo>,</mo><mi>K</mi><mo>,</mo></mrow></mtd></mtr></mtable></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><math id="d33e1282"><msub><mi>β</mi><mrow><mn>0</mn><mi>k</mi></mrow></msub></math></span> is the intercept and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><math id="d33e1296"><msub><mi>β</mi><mi>k</mi></msub></math></span> represents the AA association coefficient for feature <em>k</em>. The relation between the RA mean and the AA mean can be expressed as</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e1311" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mi>E</mi><mrow><mo stretchy="false">(</mo><msub><mi>Y</mi><mrow><mi mathvariant="italic">ik</mi></mrow></msub><mo stretchy="false">)</mo></mrow><mo>=</mo><msub><mi>c</mi><mi>i</mi></msub><msub><mi>b</mi><mi>k</mi></msub><mi>E</mi><mrow><mo stretchy="false">(</mo><msub><mi>W</mi><mrow><mi mathvariant="italic">ik</mi></mrow></msub><mo stretchy="false">)</mo></mrow><mo>=</mo><msub><mi>c</mi><mi>i</mi></msub><msub><mi>b</mi><mi>k</mi></msub><mo>exp</mo><mrow><mo stretchy="false">(</mo><msub><mi>β</mi><mrow><mn>0</mn><mi>k</mi></mrow></msub><mo>+</mo><msub><mi>β</mi><mi>k</mi></msub><msub><mi>X</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow><mo>,</mo><mspace width="3.33333pt"></mspace><mi>k</mi><mo>=</mo><mn>1</mn><mo>,</mo><mo>…</mo><mo>,</mo><mi>K</mi><mo>,</mo></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">1</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><math id="d33e1389"><msub><mi>c</mi><mi>i</mi></msub></math></span> is the sample-level multiplicative factor that absorbs quantities such as the sequencing depth and the microbial load in a unit volume for a sample, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><math id="d33e1400"><msub><mi>b</mi><mi>k</mi></msub></math></span> is the feature-level multiplicative factor that absorbs the measurement biases arising from different efficiencies (e.g., DNA extraction efficiency, primer binding, and amplification efficiency) with which various features are measured [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>].</p>
<p id="Par30">The AA coefficients are not identifiable in the model (<a href="#Equ1" class="usa-link">1</a>). Therefore, we choose a feature <em>R</em> as the reference and construct the proportion mean parameter as</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e1423" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><msub><mi>p</mi><mrow><mi mathvariant="italic">ik</mi></mrow></msub><mo>=</mo><mfrac><mrow><mi>E</mi><mo stretchy="false">(</mo><msub><mi>Y</mi><mrow><mi mathvariant="italic">ik</mi></mrow></msub><mo stretchy="false">)</mo></mrow><mrow><msubsup><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mi>K</mi></msubsup><mi>E</mi><mrow><mo stretchy="false">(</mo><msub><mi>Y</mi><mrow><mi mathvariant="italic">ij</mi></mrow></msub><mo stretchy="false">)</mo></mrow></mrow></mfrac><mo>=</mo><mfrac><mrow><mo>exp</mo><mo stretchy="false">(</mo><msubsup><mi>β</mi><mrow><mn>0</mn><mi>k</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>+</mo><msubsup><mi>β</mi><mi>k</mi><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup><msub><mi>X</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow><mrow><mn>1</mn><mo>+</mo><msub><mo>∑</mo><mrow><mi>j</mi><mo>∈</mo><mo stretchy="true">\</mo><mi>R</mi></mrow></msub><mo>exp</mo><mrow><mo stretchy="false">(</mo><msubsup><mi>β</mi><mrow><mn>0</mn><mi>j</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>+</mo><msubsup><mi>β</mi><mi>j</mi><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup><msub><mi>X</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow></mrow></mfrac><mo>,</mo></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">2</td>
</tr></table>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><math id="d33e1547" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><msubsup><mi>β</mi><mrow><mn>0</mn><mi>k</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>=</mo><mo>log</mo><mrow><mo stretchy="false">(</mo><msub><mi>b</mi><mi>k</mi></msub><mo stretchy="false">)</mo></mrow><mo>+</mo><msub><mi>β</mi><mrow><mn>0</mn><mi>k</mi></mrow></msub><mo>-</mo><mo>log</mo><mrow><mo stretchy="false">(</mo><msub><mi>b</mi><mi>R</mi></msub><mo stretchy="false">)</mo></mrow><mo>-</mo><msub><mi>β</mi><mrow><mn>0</mn><mi>R</mi></mrow></msub><mspace width="0.333333em"></mspace><mtext>and</mtext><mspace width="0.333333em"></mspace><msubsup><mi>β</mi><mi>k</mi><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>=</mo><msub><mi>β</mi><mi>k</mi></msub><mo>-</mo><msub><mi>β</mi><mi>R</mi></msub><mrow><mo>,</mo><mspace width="3.33333pt"></mspace><mi>k</mi><mo>∈</mo><mo stretchy="true">\</mo><mi>R</mi><mo>,</mo></mrow></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">3</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><math id="d33e1633"><mrow><mo stretchy="true">\</mo><mi>R</mi></mrow></math></span> is the index set of non-reference features, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><math id="d33e1645"><msubsup><mi>β</mi><mrow><mn>0</mn><mi>k</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup></math></span>’s and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><math id="d33e1665"><msubsup><mi>β</mi><mi>k</mi><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup></math></span>’s are the RA intercept and association coefficients, respectively. Note that the RA association coefficients of the non-reference features <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><math id="d33e1682"><msubsup><mi>β</mi><mi>k</mi><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup></math></span>’s are simply shifted AA association coefficients <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq53"><math id="d33e1699"><msub><mi>β</mi><mi>k</mi></msub></math></span>’s by a constant <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq54"><math id="d33e1710"><msub><mi>β</mi><mi>R</mi></msub></math></span>.</p>
<p id="Par31">The model can be extended to incorporate confounding variables. Let <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq55"><math id="d33e1723"><msub><mi mathvariant="bold">X</mi><mi>i</mi></msub></math></span> denote the design matrix that includes the intercept and all the explanatory variables and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq56"><math id="d33e1735"><mrow><msup><mrow><mi mathvariant="bold-italic">θ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup><mo>=</mo><mrow><mo stretchy="false">(</mo><msup><mrow><mi mathvariant="bold-italic">β</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup><mo>,</mo><msup><mrow><mi mathvariant="bold-italic">γ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup><mo stretchy="false">)</mo></mrow></mrow></math></span> be the collection of all coefficients, in which <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq57"><math id="d33e1781"><msup><mrow><mi mathvariant="bold-italic">β</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup></math></span> contain the RA association coefficients of interest and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq58"><math id="d33e1799"><msup><mrow><mi mathvariant="bold-italic">γ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup></math></span> contain the nuisance coefficients.</p>
<p id="Par32">We propose a quasi-multinomial model to estimate the coefficients. Specifically, given the true proportions of various features <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq59"><math id="d33e1819"><mrow><mo stretchy="false">(</mo><msub><mi>P</mi><mrow><mi>i</mi><mn>1</mn></mrow></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mi>P</mi><mrow><mi mathvariant="italic">iK</mi></mrow></msub><mo stretchy="false">)</mo></mrow></math></span>, the counts <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq60"><math id="d33e1845"><mrow><mo stretchy="false">(</mo><msub><mi>Y</mi><mrow><mi>i</mi><mn>1</mn></mrow></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mi>Y</mi><mrow><mi mathvariant="italic">iK</mi></mrow></msub><mo stretchy="false">)</mo></mrow></math></span> naturally arise from multinomial sampling with parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq61"><math id="d33e1871"><mrow><mo stretchy="false">(</mo><msub><mi>P</mi><mrow><mi>i</mi><mn>1</mn></mrow></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mi>P</mi><mrow><mi mathvariant="italic">iK</mi></mrow></msub><mo stretchy="false">)</mo></mrow></math></span> and size <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq62"><math id="d33e1897"><msub><mi>N</mi><mi>i</mi></msub></math></span>. The multinomial distribution models microbial features competing for the sequencing reads bounded by the sequencing depth <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq63"><math id="d33e1908"><msub><mi>N</mi><mi>i</mi></msub></math></span> (i.e., allocating more reads to one feature means that fewer reads are available to assign to other features). The multinomial layer is particularly important for properly accounting for the effect of uneven sequencing depth on the variability of the counts given the true proportions. To address the over-dispersion, zero-inflation, and complex structure of the microbiome data, an additional layer of randomness is needed for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq64"><math id="d33e1920"><mrow><mo stretchy="false">(</mo><msub><mi>P</mi><mrow><mi>i</mi><mn>1</mn></mrow></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mi>P</mi><mrow><mi mathvariant="italic">iK</mi></mrow></msub><mo stretchy="false">)</mo></mrow></math></span>. We do not specify a particular distribution for this layer but only assume the proportion mean model <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq65"><math id="d33e1946"><mrow><mi>E</mi><mrow><mo stretchy="false">(</mo><msub><mi>P</mi><mrow><mi mathvariant="italic">ik</mi></mrow></msub><mo stretchy="false">)</mo></mrow><mo>=</mo><msub><mi>p</mi><mrow><mi mathvariant="italic">ik</mi></mrow></msub></mrow></math></span> specified in (<a href="#Equ2" class="usa-link">2</a>). To estimate the coefficients, we propose the following quasi-score function</p>
<table class="disp-formula p" id="Equ7"><tr><td class="formula"><math id="d33e1975" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mrow><mi mathvariant="bold-italic">S</mi></mrow><mrow><mo stretchy="false">(</mo><msup><mrow><mi mathvariant="bold-italic">θ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup><mo stretchy="false">)</mo></mrow><mo>=</mo><munderover><mo movablelimits="false">∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><mrow><mo stretchy="false">(</mo><msub><mrow><mi mathvariant="bold-italic">Y</mi></mrow><mi>i</mi></msub><mo>-</mo><msub><mi>N</mi><mi>i</mi></msub><msub><mi mathvariant="bold">p</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow><mo>⊗</mo><msub><mi mathvariant="bold">X</mi><mi>i</mi></msub><mo>,</mo></mrow></mtd></mtr></mtable></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq66"><math id="d33e2039"><mo>⊗</mo></math></span> represents the Kronecker product, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq67"><math id="d33e2048"><mrow><msub><mrow><mi mathvariant="bold-italic">Y</mi></mrow><mi>i</mi></msub><mo>=</mo><msub><mfenced close="}" open="{"><msub><mi>Y</mi><mrow><mi mathvariant="italic">ik</mi></mrow></msub></mfenced><mrow><mi>k</mi><mo>∈</mo><mo stretchy="true">\</mo><mi>R</mi></mrow></msub></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq68"><math id="d33e2077"><mrow><msub><mi mathvariant="bold">p</mi><mi>i</mi></msub><mo>=</mo><msub><mfenced close="}" open="{"><msub><mi>p</mi><mrow><mi mathvariant="italic">ik</mi></mrow></msub></mfenced><mrow><mi>k</mi><mo>∈</mo><mo stretchy="true">\</mo><mi>R</mi></mrow></msub></mrow></math></span>. The parameters can be consistently estimated by solving the estimating equation <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq69"><math id="d33e2104"><mrow><mrow><mi mathvariant="bold-italic">S</mi></mrow><mo stretchy="false">(</mo><msup><mrow><mi mathvariant="bold-italic">θ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup><mo stretchy="false">)</mo><mo>=</mo><mrow><mn mathvariant="bold">0</mn></mrow></mrow></math></span>. This is equivalent to finding the maximum likelihood estimator from multinomial logistic regression because the quasi-score function has the same form as the score function of multinomial regression. To prevent the potential estimation bias due to the small sample size and the presence of rare features, we adopted the mean bias reduction procedure in estimating the parameters of the multinomial regression [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>].</p>
<p id="Par33">Let <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq70"><math id="d33e2139"><msub><mrow><mi mathvariant="bold-italic">S</mi></mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq71"><math id="d33e2154"><msub><mrow><mi mathvariant="bold-italic">S</mi></mrow><mrow><mi mathvariant="bold-italic">γ</mi></mrow></msub></math></span> denote the components in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq72"><math id="d33e2169"><mrow><mi mathvariant="bold-italic">S</mi></mrow></math></span> corresponding to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq73"><math id="d33e2180"><msup><mrow><mi mathvariant="bold-italic">β</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq74"><math id="d33e2198"><msup><mrow><mi mathvariant="bold-italic">γ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup></math></span>, respectively. Score statistics have been shown to be more accurate and numerically stable than Wald statistics in genetic association studies [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. In light of this, we propose estimating <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq75"><math id="d33e2220"><msup><mrow><mi mathvariant="bold-italic">β</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup></math></span> based on the scaled score statistics</p>
<table class="disp-formula p" id="Equ8"><tr><td class="formula"><math id="d33e2238" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><msup><mover accent="true"><mrow><mi mathvariant="bold-italic">β</mi></mrow><mo stretchy="false">^</mo></mover><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup><mo>=</mo><msubsup><mrow><mi mathvariant="bold-script">I</mi></mrow><mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msubsup><mrow><mo stretchy="false">(</mo><msup><mover accent="true"><mrow><mi mathvariant="bold-italic">θ</mi></mrow><mo stretchy="false">~</mo></mover><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup><mo stretchy="false">)</mo></mrow><msub><mrow><mi mathvariant="bold-italic">S</mi></mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow></msub><mrow><mo stretchy="false">(</mo><msup><mover accent="true"><mrow><mi mathvariant="bold-italic">θ</mi></mrow><mo stretchy="false">~</mo></mover><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup><mo stretchy="false">)</mo></mrow><mo>,</mo></mrow></mtd></mtr></mtable></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq76"><math id="d33e2320"><mrow><msup><mover accent="true"><mrow><mi mathvariant="bold-italic">θ</mi></mrow><mo stretchy="false">~</mo></mover><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup><mo>=</mo><mrow><mo stretchy="false">(</mo><mrow><mn mathvariant="bold">0</mn></mrow><mo>,</mo><msup><mover accent="true"><mrow><mi mathvariant="bold-italic">γ</mi></mrow><mo stretchy="false">~</mo></mover><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup><mo stretchy="false">)</mo></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq77"><math id="d33e2365"><msup><mover accent="true"><mrow><mi mathvariant="bold-italic">γ</mi></mrow><mo stretchy="false">~</mo></mover><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup></math></span> is the estimate of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq78"><math id="d33e2386"><msup><mrow><mrow><mi mathvariant="bold-italic">γ</mi></mrow></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup></math></span> under the reduced model without the covariate of interest in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq79"><math id="d33e2406"><msub><mi mathvariant="bold">X</mi><mi>i</mi></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq80"><math id="d33e2418"><mrow><msub><mrow><mi mathvariant="bold-script">I</mi></mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow></msub><mo>=</mo><msub><mrow><mi mathvariant="bold-script">I</mi></mrow><mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow></mrow></msub><mo>-</mo><msub><mrow><mi mathvariant="bold-script">I</mi></mrow><mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow><mrow><mi mathvariant="bold-italic">γ</mi></mrow></mrow></msub><msubsup><mrow><mi mathvariant="bold-script">I</mi></mrow><mrow><mrow><mi mathvariant="bold-italic">γ</mi></mrow><mrow><mi mathvariant="bold-italic">γ</mi></mrow></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msubsup><msub><mrow><mi mathvariant="bold-script">I</mi></mrow><mrow><mrow><mi mathvariant="bold-italic">γ</mi></mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow></mrow></msub></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq81"><math id="d33e2483"><mfenced close="]" open="["><mrow><mtable><mtr><mtd><msub><mrow><mi mathvariant="bold-script">I</mi></mrow><mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow></mrow></msub></mtd><mtd><msub><mrow><mi mathvariant="bold-script">I</mi></mrow><mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow><mrow><mi mathvariant="bold-italic">γ</mi></mrow></mrow></msub></mtd></mtr><mtr><mtd><mrow><mrow></mrow><msub><mrow><mi mathvariant="bold-script">I</mi></mrow><mrow><mrow><mi mathvariant="bold-italic">γ</mi></mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow></mrow></msub></mrow></mtd><mtd><msub><mrow><mi mathvariant="bold-script">I</mi></mrow><mrow><mrow><mi mathvariant="bold-italic">γ</mi></mrow><mrow><mi mathvariant="bold-italic">γ</mi></mrow></mrow></msub></mtd></mtr></mtable></mrow></mfenced></math></span> is the partition of the quasi-information matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq82"><math id="d33e2546"><mrow><mo>-</mo><mfrac><mrow><mi>∂</mi><mrow><mi mathvariant="bold-italic">S</mi></mrow><mo stretchy="false">(</mo><msup><mrow><mi mathvariant="bold-italic">θ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup><mo stretchy="false">)</mo></mrow><mrow><mi>∂</mi><msup><mrow><mi mathvariant="bold-italic">θ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup></mrow></mfrac></mrow></math></span> for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq83"><math id="d33e2588"><msup><mrow><mi mathvariant="bold-italic">β</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq84"><math id="d33e2606"><msup><mrow><mi mathvariant="bold-italic">γ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup></math></span>.</p>
<p id="Par34">The sandwich estimator of the covariance of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq85"><math id="d33e2626"><msup><mover accent="true"><mrow><mi mathvariant="bold-italic">β</mi></mrow><mo stretchy="false">^</mo></mover><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup></math></span> can be written as</p>
<table class="disp-formula p" id="Equ9"><tr><td class="formula"><math id="d33e2647" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><msup><mrow><mrow><mi mathvariant="bold-italic">V</mi></mrow></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup><mo>=</mo><msubsup><mrow><mi mathvariant="bold-script">I</mi></mrow><mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msubsup><mrow><mo stretchy="false">(</mo><msup><mover accent="true"><mrow><mi mathvariant="bold-italic">θ</mi></mrow><mo stretchy="false">~</mo></mover><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup><mo stretchy="false">)</mo></mrow><mfenced close=")" open="("><munderover><mo movablelimits="false">∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><msub><mrow><mi mathvariant="bold-italic">U</mi></mrow><mi>i</mi></msub><mrow><mo stretchy="false">(</mo><msup><mover accent="true"><mrow><mi mathvariant="bold-italic">θ</mi></mrow><mo stretchy="false">~</mo></mover><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup><mo stretchy="false">)</mo></mrow><msubsup><mrow><mi mathvariant="bold-italic">U</mi></mrow><mi>i</mi><mtext>T</mtext></msubsup><mrow><mo stretchy="false">(</mo><msup><mover accent="true"><mrow><mi mathvariant="bold-italic">θ</mi></mrow><mo stretchy="false">~</mo></mover><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup><mo stretchy="false">)</mo></mrow></mfenced><msubsup><mrow><mi mathvariant="bold-script">I</mi></mrow><mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msubsup><mrow><mo stretchy="false">(</mo><msup><mover accent="true"><mrow><mi mathvariant="bold-italic">θ</mi></mrow><mo stretchy="false">~</mo></mover><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup><mo stretchy="false">)</mo></mrow><mo>,</mo></mrow></mtd></mtr></mtable></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq86"><math id="d33e2787"><mrow><msub><mrow><mi mathvariant="bold-italic">U</mi></mrow><mi>i</mi></msub><mo>=</mo><msub><mrow><mi mathvariant="bold-italic">S</mi></mrow><mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow><mo>,</mo><mi>i</mi></mrow></msub><mo>-</mo><msub><mrow><mi mathvariant="bold-script">I</mi></mrow><mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow><mrow><mi mathvariant="bold-italic">γ</mi></mrow></mrow></msub><msubsup><mrow><mi mathvariant="bold-script">I</mi></mrow><mrow><mrow><mi mathvariant="bold-italic">γ</mi></mrow><mrow><mi mathvariant="bold-italic">γ</mi></mrow></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msubsup><msub><mrow><mi mathvariant="bold-italic">S</mi></mrow><mrow><mrow><mi mathvariant="bold-italic">γ</mi></mrow><mo>,</mo><mi>i</mi></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq87"><math id="d33e2848"><msub><mrow><mi mathvariant="bold-italic">S</mi></mrow><mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow><mo>,</mo><mi>i</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq88"><math id="d33e2866"><msub><mrow><mi mathvariant="bold-italic">S</mi></mrow><mrow><mi>γ</mi><mo>,</mo><mi>i</mi></mrow></msub></math></span> are the <em>i</em>th summand of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq89"><math id="d33e2886"><msub><mrow><mi mathvariant="bold-italic">S</mi></mrow><mrow><mi mathvariant="bold-italic">β</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq90"><math id="d33e2901"><msub><mrow><mi mathvariant="bold-italic">S</mi></mrow><mrow><mi mathvariant="bold-italic">γ</mi></mrow></msub></math></span> respectively. The correlations cannot be accurately estimated when the number of features is close to or larger than the sample size. Therefore, we use the diagonal matrix (with variances on the diagonal elements) extracted from <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq91"><math id="d33e2916"><msup><mrow><mrow><mi mathvariant="bold-italic">V</mi></mrow></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msup></math></span> in the meta-analysis. This procedure of generating summary statistics can be modified for studies with correlated samples (e.g., from longitudinal studies), and the details are provided in the Additional file <a href="#MOESM1" class="usa-link">1</a>: Supplementary Note A.</p>
<p id="Par35"><em>Harmonizing and combining summary statistics across studies to identify signatures.</em> Suppose we want to meta-analyze <em>L</em> independent microbiome association studies and identify microbial signatures shared by the studies. Let <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq92"><math id="d33e2945"><msub><mrow><mi mathvariant="bold-italic">β</mi></mrow><mrow><mo>∙</mo><mi>ℓ</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq93"><math id="d33e2960"><msubsup><mrow><mi mathvariant="bold-italic">β</mi></mrow><mrow><mo>∙</mo><mi>ℓ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup></math></span> respectively denote the AA and RA association coefficients for study <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq94"><math id="d33e2981"><mi>ℓ</mi></math></span>. The length of these coefficients could differ among studies because a feature may not be observed in all studies. Let <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq95"><math id="d33e2990"><msub><mi mathvariant="script">T</mi><mi>ℓ</mi></msub></math></span> be the index set of non-reference features observed in study <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq96"><math id="d33e3003"><mi>ℓ</mi></math></span>. For each study <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq97"><math id="d33e3012"><mi>ℓ</mi></math></span>, we obtain RA association coefficient estimates denoted by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq98"><math id="d33e3021"><mrow><msubsup><mover accent="true"><mrow><mi mathvariant="bold-italic">β</mi></mrow><mo stretchy="false">^</mo></mover><mrow><mo>∙</mo><mi>ℓ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>=</mo><msub><mrow><mo stretchy="false">{</mo><msubsup><mover accent="true"><mi>β</mi><mo stretchy="false">^</mo></mover><mrow><mi>k</mi><mi>ℓ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup><mo stretchy="false">}</mo></mrow><mrow><mi>k</mi><mo>∈</mo><msub><mi mathvariant="script">T</mi><mi>ℓ</mi></msub></mrow></msub></mrow></math></span> and the diagonal matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq99"><math id="d33e3074"><msubsup><mrow><mrow><mi mathvariant="bold-italic">V</mi></mrow></mrow><mi>ℓ</mi><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup></math></span> in which the diagonal elements are the variance of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq100"><math id="d33e3094"><msubsup><mover accent="true"><mrow><mi mathvariant="bold-italic">β</mi></mrow><mo stretchy="false">^</mo></mover><mrow><mo>∙</mo><mi>ℓ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup></math></span>. In the meta-analysis, we are interested in estimating the meta AA association coefficients that are common across all study-specific AA coefficients <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq101"><math id="d33e3118"><msub><mrow><mi mathvariant="bold-italic">β</mi></mrow><mrow><mo>∙</mo><mi>ℓ</mi></mrow></msub></math></span> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq102"><math id="d33e3134"><mrow><mi>ℓ</mi><mo>=</mo><mn>1</mn><mo>,</mo><mo>…</mo><mo>,</mo><mi>L</mi></mrow></math></span>). Let <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq103"><math id="d33e3150"><msub><mi mathvariant="script">T</mi><mn>0</mn></msub></math></span> denote the union set of all <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq104"><math id="d33e3162"><msub><mi mathvariant="script">T</mi><mi>ℓ</mi></msub></math></span>’s, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq105"><math id="d33e3174"><msub><mi>K</mi><mn>0</mn></msub></math></span> denote the size of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq106"><math id="d33e3185"><msub><mi mathvariant="script">T</mi><mn>0</mn></msub></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq107"><math id="d33e3197"><mrow><mrow><mi mathvariant="bold-italic">μ</mi></mrow><mo>=</mo><msub><mrow><mo stretchy="false">{</mo><msub><mi>μ</mi><mi>k</mi></msub><mo stretchy="false">}</mo></mrow><mrow><mi>k</mi><mo>∈</mo><msub><mi mathvariant="script">T</mi><mn>0</mn></msub></mrow></msub></mrow></math></span> be the collection of the meta AA association coefficients for all the non-reference features appear in at least one study. We propose to estimate <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq108"><math id="d33e3227"><mrow><mi mathvariant="bold-italic">μ</mi></mrow></math></span> by minimizing the objective function</p>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><math id="d33e3239" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mi mathvariant="script">L</mi><mrow><mo stretchy="false">(</mo><mrow><mi mathvariant="bold-italic">μ</mi></mrow><mo stretchy="false">)</mo></mrow><mo>=</mo><msubsup><mi>n</mi><mn>0</mn><mrow><mo>-</mo><mn>1</mn></mrow></msubsup><munderover><mo movablelimits="false">∑</mo><mrow><mi>ℓ</mi><mo>=</mo><mn>1</mn></mrow><mi>L</mi></munderover><msup><mrow><mo stretchy="false">(</mo><msubsup><mover accent="true"><mrow><mi mathvariant="bold-italic">β</mi></mrow><mo stretchy="false">^</mo></mover><mrow><mo>∙</mo><mi>ℓ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>+</mo><msub><mi>δ</mi><mi>ℓ</mi></msub><msub><mn mathvariant="bold">1</mn><mi>ℓ</mi></msub><mo>-</mo><msub><mrow><mi mathvariant="bold-italic">μ</mi></mrow><msub><mi mathvariant="script">T</mi><mi>ℓ</mi></msub></msub><mo stretchy="false">)</mo></mrow><mtext>T</mtext></msup><msup><mrow><msubsup><mrow><mi mathvariant="bold-italic">V</mi></mrow><mi>ℓ</mi><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msup><mrow><mo stretchy="false">(</mo><msubsup><mover accent="true"><mrow><mi mathvariant="bold-italic">β</mi></mrow><mo stretchy="false">^</mo></mover><mrow><mo>∙</mo><mi>ℓ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>+</mo><msub><mi>δ</mi><mi>ℓ</mi></msub><msub><mn mathvariant="bold">1</mn><mi>ℓ</mi></msub><mo>-</mo><msub><mrow><mi mathvariant="bold-italic">μ</mi></mrow><msub><mi mathvariant="script">T</mi><mi>ℓ</mi></msub></msub><mo stretchy="false">)</mo></mrow><mo>,</mo><mspace width="4pt"></mspace><mtext>subject to</mtext><mspace width="4pt"></mspace><msub><mrow><mo stretchy="false">‖</mo><mrow><mi mathvariant="bold-italic">μ</mi></mrow><mo stretchy="false">‖</mo></mrow><mn>0</mn></msub><mo>≤</mo><mi>s</mi><mo>,</mo></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">4</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq109"><math id="d33e3389"><msub><mi>n</mi><mn>0</mn></msub></math></span> is the total sample size across studies, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq110"><math id="d33e3400"><msub><mn mathvariant="bold">1</mn><mi>ℓ</mi></msub></math></span> is a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq111"><math id="d33e3412"><msub><mi>K</mi><mi>ℓ</mi></msub></math></span>-vector of ones, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq112"><math id="d33e3423"><msub><mrow><mi mathvariant="bold-italic">μ</mi></mrow><msub><mi mathvariant="script">T</mi><mi>ℓ</mi></msub></msub></math></span> is the subset of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq113"><math id="d33e3440"><mrow><mi mathvariant="bold-italic">μ</mi></mrow></math></span> for features in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq114"><math id="d33e3451"><msub><mi mathvariant="script">T</mi><mi>ℓ</mi></msub></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq115"><math id="d33e3463"><mrow><msub><mrow><mo stretchy="false">‖</mo><mrow><mi mathvariant="bold-italic">μ</mi></mrow><mo stretchy="false">‖</mo></mrow><mn>0</mn></msub><mo>=</mo><msubsup><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mn>0</mn></msubsup><mi>I</mi><mrow><mo stretchy="false">(</mo><msub><mi>μ</mi><mi>k</mi></msub><mo>≠</mo><mn>0</mn><mo stretchy="false">)</mo></mrow></mrow></math></span> is the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq116"><math id="d33e3503"><msub><mi>ℓ</mi><mn>0</mn></msub></math></span> norm of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq117"><math id="d33e3514"><mrow><mi mathvariant="bold-italic">μ</mi></mrow></math></span>. The form of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq118"><math id="d33e3525"><mrow><mi mathvariant="script">L</mi><mo stretchy="false">(</mo><mrow><mi mathvariant="bold-italic">μ</mi></mrow><mo stretchy="false">)</mo></mrow></math></span> is motivated by the inverse-variance estimator commonly used in meta-analyses [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>], and the cardinality constraint with the sparsity level <em>s</em> is formulated as in the classical best subset selection problem [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>, <a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. By noting the relationship between AA and RA association coefficients <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq119"><math id="d33e3556"><mrow><msubsup><mi>β</mi><mrow><mi>k</mi><mi>ℓ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>=</mo><msub><mi>β</mi><mrow><mi>k</mi><mi>ℓ</mi></mrow></msub><mo>-</mo><msub><mi>β</mi><mrow><mi>R</mi><mi>ℓ</mi></mrow></msub></mrow></math></span> (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq120"><math id="d33e3588"><mrow><mi>k</mi><mo>∈</mo><msub><mi mathvariant="script">T</mi><mi>ℓ</mi></msub></mrow></math></span>), we introduce a hyperparameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq121"><math id="d33e3604"><msub><mi>δ</mi><mi>ℓ</mi></msub></math></span> in each study <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq122"><math id="d33e3615"><mi>ℓ</mi></math></span> to offset <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq123"><math id="d33e3624"><msub><mi>β</mi><mrow><mi>R</mi><mi>ℓ</mi></mrow></msub></math></span> in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq124"><math id="d33e3637"><msubsup><mi>β</mi><mrow><mi>k</mi><mi>ℓ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup></math></span>. By varying the value of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq125"><math id="d33e3656"><msub><mi>δ</mi><mi>ℓ</mi></msub></math></span>, one essentially traverses possible versions of putative AA association coefficients in study <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq126"><math id="d33e3667"><mi>ℓ</mi></math></span>. Given <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq127"><math id="d33e3677"><mrow><msub><mi>δ</mi><mn>1</mn></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mi>δ</mi><mi>L</mi></msub></mrow></math></span> and <em>s</em>, we solve the best subset selection problem (<a href="#Equ4" class="usa-link">4</a>) using the polynomial algorithm [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>, <a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]. The optimal values of the hyperparameters are determined by minimizing BIC defined as</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><math id="d33e3708" display="block"><mrow><mtable><mtr><mtd columnalign="right"><mrow><mtext>BIC</mtext><mrow><mo stretchy="false">(</mo><mi>s</mi><mo>,</mo><msub><mi>δ</mi><mn>1</mn></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mi>δ</mi><mi>L</mi></msub><mo stretchy="false">)</mo></mrow><mo>=</mo><msubsup><mi>n</mi><mn>0</mn><mrow><mo>-</mo><mn>1</mn></mrow></msubsup><munderover><mo movablelimits="false">∑</mo><mrow><mi>ℓ</mi><mo>=</mo><mn>1</mn></mrow><mi>L</mi></munderover><msup><mrow><mo stretchy="false">(</mo><msubsup><mover accent="true"><mrow><mi mathvariant="bold-italic">β</mi></mrow><mo stretchy="false">^</mo></mover><mrow><mo>∙</mo><mi>ℓ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>+</mo><msub><mi>δ</mi><mi>ℓ</mi></msub><msub><mn mathvariant="bold">1</mn><mi>ℓ</mi></msub><mo>-</mo><msub><mover accent="true"><mrow><mi mathvariant="bold-italic">μ</mi></mrow><mo stretchy="false">~</mo></mover><msub><mi mathvariant="script">T</mi><mi>ℓ</mi></msub></msub><mo stretchy="false">)</mo></mrow><mtext>T</mtext></msup><msup><mrow><msubsup><mrow><mi mathvariant="bold-italic">V</mi></mrow><mi>ℓ</mi><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msup><mrow><mo stretchy="false">(</mo><msubsup><mover accent="true"><mrow><mi mathvariant="bold-italic">β</mi></mrow><mo stretchy="false">^</mo></mover><mrow><mo>∙</mo><mi>ℓ</mi></mrow><mrow><mo stretchy="false">(</mo><mi>R</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>+</mo><msub><mi>δ</mi><mi>ℓ</mi></msub><msub><mn mathvariant="bold">1</mn><mi>ℓ</mi></msub><mo>-</mo><msub><mover accent="true"><mrow><mi mathvariant="bold-italic">μ</mi></mrow><mo stretchy="false">~</mo></mover><msub><mi mathvariant="script">T</mi><mi>ℓ</mi></msub></msub><mo stretchy="false">)</mo></mrow><mo>+</mo><msub><mrow><mo stretchy="false">‖</mo><mover accent="true"><mrow><mi mathvariant="bold-italic">μ</mi></mrow><mo stretchy="false">~</mo></mover><mo stretchy="false">‖</mo></mrow><mn>0</mn></msub><mfrac><mrow><mo>log</mo><msub><mi>n</mi><mn>0</mn></msub></mrow><msub><mi>n</mi><mn>0</mn></msub></mfrac><mo>,</mo></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">5</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq128"><math id="d33e3879"><mover accent="true"><mrow><mi mathvariant="bold-italic">μ</mi></mrow><mo stretchy="false">~</mo></mover></math></span> is the estimate of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq129"><math id="d33e3893"><mrow><mi mathvariant="bold-italic">μ</mi></mrow></math></span> under a specification of hyperparameters. Details on solving the best subset selection problem and hyperparameter tuning algorithms that utilized Powell’s algorithm [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>], the successive parabolic interpolation (SPI) algorithm [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>], and the golden section search algorithm [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>], are provided in the Additional file <a href="#MOESM1" class="usa-link">1</a>: Supplementary Note A.</p>
<p id="Par36">The <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq130"><math id="d33e3918"><mover accent="true"><mrow><mi mathvariant="bold-italic">μ</mi></mrow><mo stretchy="false">~</mo></mover></math></span> under the optimal hyperparameter values is the estimate of the most parsimonious version of the putative AA associations consistent across studies. The features with non-zero values in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq131"><math id="d33e3932"><mover accent="true"><mrow><mi mathvariant="bold-italic">μ</mi></mrow><mo stretchy="false">~</mo></mover></math></span> are the driver signatures detected by Melody.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Simulation strategy</h3>
<p id="Par37">We simulated microbiome data by mimicking the species composition patterns of the five metagenomics studies for CRC [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. For each study, we first fit the generalized Dirichlet multinomial distribution [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>] to the 401 species with a prevalence of at least 20% in the real data and obtained the maximum likelihood parameter estimates. Using the five sets of estimated parameters, we randomly generated independent samples of proportions for five studies from the generalized Dirichlet distribution. These samples served as the true proportions of the microbial features. The synthetic data closely resemble the structure of the real microbiome data (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S2). We considered two settings for the sample sizes of the five studies: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq132"><math id="d33e3959"><mrow><mi>n</mi><mo>=</mo><mn>100</mn><mo>,</mo><mn>120</mn><mo>,</mo><mn>140</mn><mo>,</mo><mn>160</mn><mo>,</mo><mn>180</mn></mrow></math></span> for relatively large studies, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq133"><math id="d33e3979"><mrow><mi>n</mi><mo>=</mo><mn>20</mn><mo>,</mo><mn>30</mn><mo>,</mo><mn>40</mn><mo>,</mo><mn>50</mn><mo>,</mo><mn>60</mn></mrow></math></span> for small studies. We randomly assigned the samples into 50% cases and 50% controls. We varied the number of signatures from 4 (1% of the features) to 160 (40%). Half of the signatures were randomly selected from the set of relatively abundant features with an average proportion of at least <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq134"><math id="d33e4000"><msup><mn>10</mn><mrow><mo>-</mo><mn>3</mn></mrow></msup></math></span> in the original real data, while the other half were randomly selected from the remaining features. We let <em>pos</em>% of the signatures have positive effects and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq135"><math id="d33e4016"><mrow><mn>1</mn><mo>-</mo><mi>p</mi><mi>o</mi><mi>s</mi></mrow></math></span>% have negative effects. We considered three settings of <em>pos</em>%: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq136"><math id="d33e4033"><mrow><mn>60</mn><mo>%</mo></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq137"><math id="d33e4044"><mrow><mn>80</mn><mo>%</mo></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq138"><math id="d33e4056"><mrow><mn>100</mn><mo>%</mo></mrow></math></span>, which reflect various degrees of sign imbalance among signatures. For each positive-effect signature, we increased their proportions in cases; for each negative-effect signature, we increased their proportions in controls. The increased fold for each signature was randomly sampled from Uniform(0, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq139"><math id="d33e4067"><mi mathvariant="normal">Δ</mi></math></span>), where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq140"><math id="d33e4077"><mrow><mi mathvariant="normal">Δ</mi><mo>=</mo><mn>2</mn></mrow></math></span> under the large study setting and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq141"><math id="d33e4090"><mrow><mi mathvariant="normal">Δ</mi><mo>=</mo><mn>5</mn></mrow></math></span> under the small study setting. The larger effect size in the small study was chosen to make performance differences among the compared methods more discernible. Next, we normalized the proportions of all features in each sample to ensure that the total sum of all proportions is one. Notably, the proportions of all features are shifted after this step, but the features initially changed are considered the driver signatures with different underlying absolute abundances between the case and control groups. We then used the normalized proportions as multinomial parameters to sample count data. The sequencing depth for samples in each study was randomly drawn from the pool of sequencing depth in the corresponding CRC study. Within each study, we increased the sequencing depth of all samples in either case or control group by a factor <em>u</em> to reflect the potential uneven sequencing depth between the two groups of interest. We considered three settings of <em>u</em>: 0, 0.5, and 1. We simulated 100 datasets for each scenario and reported the average AUPRC for signature selection in the simulated data for each meta-analysis approach.</p>
<p id="Par38">We conducted another set of simulation experiments for meta-analyzing studies with correlated samples. We first simulated independent samples as described above. We randomly picked two samples <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq142"><math id="d33e4111"><msub><mi>X</mi><mrow><mi>i</mi><mn>1</mn></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq143"><math id="d33e4124"><msub><mi>X</mi><mrow><mi>i</mi><mn>2</mn></mrow></msub></math></span> to form cluster <em>i</em>. To introduce correlation between samples within a cluster, we sample baseline composition <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq144"><math id="d33e4140"><mmultiscripts><mi>X</mi><mi>i</mi><mrow><mrow></mrow><mo>∗</mo></mrow></mmultiscripts></math></span> from the generalized Dirichlet distribution and merge this baseline composition into <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq145"><math id="d33e4154"><msub><mi>X</mi><mrow><mi>i</mi><mn>1</mn></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq146"><math id="d33e4168"><msub><mi>X</mi><mrow><mi>i</mi><mn>2</mn></mrow></msub></math></span> and form the correlated samples <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq147"><math id="d33e4181"><mrow><mmultiscripts><mi>X</mi><mrow><mi>i</mi><mn>1</mn></mrow><mrow><mrow></mrow><mo>∗</mo></mrow></mmultiscripts><mo>=</mo><mrow><mo stretchy="false">(</mo><mmultiscripts><mi>X</mi><mi>i</mi><mrow><mrow></mrow><mo>∗</mo></mrow></mmultiscripts><mo>+</mo><msub><mi>X</mi><mrow><mi>i</mi><mn>1</mn></mrow></msub><mo stretchy="false">)</mo></mrow><mo stretchy="false">/</mo><mn>2</mn></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq148"><math id="d33e4219"><mrow><mmultiscripts><mi>X</mi><mrow><mi>i</mi><mn>2</mn></mrow><mrow><mrow></mrow><mo>∗</mo></mrow></mmultiscripts><mo>=</mo><mrow><mo stretchy="false">(</mo><mmultiscripts><mi>X</mi><mi>i</mi><mrow><mrow></mrow><mo>∗</mo></mrow></mmultiscripts><mo>+</mo><msub><mi>X</mi><mrow><mi>i</mi><mn>2</mn></mrow></msub><mo stretchy="false">)</mo></mrow><mo stretchy="false">/</mo><mn>2</mn></mrow></math></span> for cluster <em>i</em>. The case and control labels are randomly assigned to clusters. Differential abundance between cases and controls is introduced in the same manner as for independent samples.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Datasets of the five metagenomics studies for CRC</h3>
<p id="Par39">All raw sequencing data across studies were reprocessed using the same bioinformatics pipeline for taxonomic profiling [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. The species-level microbiome count data and the variable for CRC status were obtained from <a href="https://github.com/zellerlab/crc_meta" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/zellerlab/crc_meta</a>. The five studies involve subjects from five different countries. The sample sizes of the studies are 109, 127, 120, 114, and 104 (Additional file <a href="#MOESM1" class="usa-link">1</a>: Table S1). We removed two samples with an outlier sequencing depth of less than 2000. Our main analysis was on <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq149"><math id="d33e4274"><mrow><mi>K</mi><mo>=</mo><mn>401</mn></mrow></math></span> species after removing species with a prevalence less than <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq150"><math id="d33e4286"><mrow><mn>20</mn><mo>%</mo></mrow></math></span> (prevalence was defined as the percentage of non-zero counts in all samples across studies). Additionally, we analyzed the data with all <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq151"><math id="d33e4298"><mrow><mi>K</mi><mo>=</mo><mn>849</mn></mrow></math></span> species and with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq152"><math id="d33e4310"><mrow><mi>K</mi><mo>=</mo><mn>267</mn></mrow></math></span> species under order <em>Clostridiales</em> to assess the stability of signature selection.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Datasets of the eight microbiome-metabolome studies</h3>
<p id="Par40">In all studies, the microbiome and metabolome were profiled from the same fecal samples. The datasets undergo curation and processing as part of an established effort [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. The genus-level microbiome count data and the variables for metabolites and disease status were obtained from <a href="https://github.com/borenstein-lab/microbiome-metabolome-curated-data" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/borenstein-lab/microbiome-metabolome-curated-data</a>. We used the eight adult cohorts from this resource with sample sizes of 96, 294, 240, 220, 444, 382, 287, and 164. There is notable heterogeneity among studies in study designs, microbiome sequencing methods, and metabolite measurements (Additional file <a href="#MOESM1" class="usa-link">1</a>: Table S3). Our analysis focused on 450 metabolites with study-specific prevalence <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq153"><math id="d33e4339"><mrow><mo>≥</mo><mn>10</mn><mo>%</mo></mrow></math></span> in at least two studies and 101 genera with study-specific prevalence <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq154"><math id="d33e4351"><mrow><mo>≥</mo><mn>10</mn><mo>%</mo></mrow></math></span> in at least one study. We applied rank-based inverse normal transformation to the metabolite variable. We performed Melody and MMUPHin to discover genus signatures for each metabolite. In both methods, we analyze each metabolite variable as the covariate of interest while adjusting the disease status as a confounder when generating summary statistics within individual studies.</p></section><section id="Sec14"><h3 class="pmc_sec_title">R packages utilized</h3>
<p id="Par41"><em>MMUPHin</em> (v1.14.0), <em>ALDEx2</em> (v1.32.0), <em>ANCOMBC</em> (v2.2.2), <em>coin</em> (v1.4-2), <em>glmnet</em> (v4.1-8), <em>brglm2</em> (v 0.9.2), <em>abess</em> (v0.4.8), <em>mlr3</em> (v0.16.1).</p></section></section><section id="Sec15"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12359909/bin/13059_2025_3721_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3721_MOESM1_ESM.pdf</a><sup> (2MB, pdf) </sup><p>Additional file 1: Supplementary Notes A and B, Figs S1-S14, and Tables S1, S3.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12359909/bin/13059_2025_3721_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Additional file 2: Tables S2, S4, S5.</a><sup> (424.8KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors thank the editors and two anonymous reviewers for their insightful comments and suggestions, which helped improve the quality of the manuscript.</p></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>ZZT conceived the idea and oversaw the study. ZZT developed Melody with contributions from GC and ZW. ZW implemented the method, conducted numerical studies, and developed the miMeta R package. ZZT wrote the first version of the manuscript. GC and ZW also contributed to the writing. All authors read and approved the final manuscript.</p>
<section id="FPar1"><h3 class="pmc_sec_title">Authors’ X handles</h3>
<p id="Par42">X handles: @tangzz_lab (Zheng-Zheng Tang).</p></section></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by the NIH grants R01AI192972, R01GM140464, and NSF grant DMS-2054346.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>This study made use of 13 publicly available datasets. The five metagenomics data for colorectal cancer were obtained from <a href="https://github.com/zellerlab/crc_meta" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/zellerlab/crc_meta</a> [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>] and the eight microbiome-metabolome data were obtained from <a href="https://github.com/borenstein-lab/microbiome-metabolome-curated-data" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/borenstein-lab/microbiome-metabolome-curated-data</a> [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. All raw data and processed data used for analysis are also available at <a href="https://github.com/ZjpWei/Melody" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ZjpWei/Melody</a> [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>]. Melody is implemented in the R package miMeta publicly available at <a href="https://github.com/ZjpWei/miMeta" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ZjpWei/miMeta</a> [<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>]. The site also contains a complete set of manuals and instructions. The scripts generating reported results are deposited at Github <a href="https://github.com/ZjpWei/Melody" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ZjpWei/Melody</a> [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>] and Zenodo <a href="https://doi.org/10.5281/zenodo.15666043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5281/zenodo.15666043</a> [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>] under the GPL-3.0 license.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar2"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par43">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par44">Not applicable.</p></section><section id="FPar4"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par45">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Tringe SG, Rubin EM. Metagenomics: DNA sequencing of environmental samples. Nat Rev Genet. 2005;6(11):805–14.
</cite> [<a href="https://doi.org/10.1038/nrg1709" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16304596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tringe%20SG,%20Rubin%20EM.%20Metagenomics:%20DNA%20sequencing%20of%20environmental%20samples.%20Nat%20Rev%20Genet.%202005;6(11):805%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Duvallet C, Gibbons SM, Gurry T, Irizarry RA, Alm EJ. Meta-analysis of gut microbiome studies identifies disease-specific and shared responses. Nat Commun. 2017;8(1):1784.
</cite> [<a href="https://doi.org/10.1038/s41467-017-01973-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5716994/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29209090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duvallet%20C,%20Gibbons%20SM,%20Gurry%20T,%20Irizarry%20RA,%20Alm%20EJ.%20Meta-analysis%20of%20gut%20microbiome%20studies%20identifies%20disease-specific%20and%20shared%20responses.%20Nat%20Commun.%202017;8(1):1784." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Wirbel J, Pyl PT, Kartal E, Zych K, Kashani A, Milanese A, et al. Meta-analysis of fecal metagenomes reveals global microbial signatures that are specific for colorectal cancer. Nat Med. 2019;25:679–89.
</cite> [<a href="https://doi.org/10.1038/s41591-019-0406-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7984229/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30936547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wirbel%20J,%20Pyl%20PT,%20Kartal%20E,%20Zych%20K,%20Kashani%20A,%20Milanese%20A,%20et%20al.%20Meta-analysis%20of%20fecal%20metagenomes%20reveals%20global%20microbial%20signatures%20that%20are%20specific%20for%20colorectal%20cancer.%20Nat%20Med.%202019;25:679%E2%80%9389." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Muller E, Algavi YM, Borenstein E. A meta-analysis study of the robustness and universality of gut microbiome-metabolome associations. Microbiome. 2021;9(1):1–18.
</cite> [<a href="https://doi.org/10.1186/s40168-021-01149-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8507343/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34641974/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Muller%20E,%20Algavi%20YM,%20Borenstein%20E.%20A%20meta-analysis%20study%20of%20the%20robustness%20and%20universality%20of%20gut%20microbiome-metabolome%20associations.%20Microbiome.%202021;9(1):1%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Ma S, Shungin D, Mallick H, Schirmer M, Nguyen LH, Kolde R, et al. Population structure discovery in meta-analyzed microbial communities and inflammatory bowel disease using MMUPHin. Genome Biol. 2022;23(1):1–31.
</cite> [<a href="https://doi.org/10.1186/s13059-022-02753-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9531436/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36192803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma%20S,%20Shungin%20D,%20Mallick%20H,%20Schirmer%20M,%20Nguyen%20LH,%20Kolde%20R,%20et%20al.%20Population%20structure%20discovery%20in%20meta-analyzed%20microbial%20communities%20and%20inflammatory%20bowel%20disease%20using%20MMUPHin.%20Genome%20Biol.%202022;23(1):1%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Hamady M, Knight R. Microbial community profiling for human microbiome projects: tools, techniques, and challenges. Genome Res. 2009;19(7):1141–52.
</cite> [<a href="https://doi.org/10.1101/gr.085464.108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3776646/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19383763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hamady%20M,%20Knight%20R.%20Microbial%20community%20profiling%20for%20human%20microbiome%20projects:%20tools,%20techniques,%20and%20challenges.%20Genome%20Res.%202009;19(7):1141%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Gloor GB, Wu JR, Pawlowsky-Glahn V, Egozcue JJ. It’s all relative: analyzing microbiome data as compositions. Ann Epidemiol. 2016;26(5):322–9.
</cite> [<a href="https://doi.org/10.1016/j.annepidem.2016.03.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27143475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gloor%20GB,%20Wu%20JR,%20Pawlowsky-Glahn%20V,%20Egozcue%20JJ.%20It%E2%80%99s%20all%20relative:%20analyzing%20microbiome%20data%20as%20compositions.%20Ann%20Epidemiol.%202016;26(5):322%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Gloor GB, Macklaim JM, Pawlowsky-Glahn V, Egozcue JJ. Microbiome datasets are compositional: and this is not optional. Front Microbiol. 2017;8:2224.
</cite> [<a href="https://doi.org/10.3389/fmicb.2017.02224" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5695134/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29187837/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gloor%20GB,%20Macklaim%20JM,%20Pawlowsky-Glahn%20V,%20Egozcue%20JJ.%20Microbiome%20datasets%20are%20compositional:%20and%20this%20is%20not%20optional.%20Front%20Microbiol.%202017;8:2224." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Fredricks DN, Fiedler TL, Thomas KK, Oakley BB, Marrazzo JM. Targeted PCR for detection of vaginal bacteria associated with bacterial vaginosis. J Clin Microbiol. 2007;45(10):3270–6.
</cite> [<a href="https://doi.org/10.1128/JCM.01272-07" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2045326/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17687006/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fredricks%20DN,%20Fiedler%20TL,%20Thomas%20KK,%20Oakley%20BB,%20Marrazzo%20JM.%20Targeted%20PCR%20for%20detection%20of%20vaginal%20bacteria%20associated%20with%20bacterial%20vaginosis.%20J%20Clin%20Microbiol.%202007;45(10):3270%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Ryu H, Henson M, Elk M, Toledo-Hernandez C, Griffith J, Blackwood D, et al. Development of quantitative PCR assays targeting the 16S rRNA genes of Enterococcus spp. and their application to the identification of Enterococcus species in environmental samples. Appl Environ Microbiol. 2013;79(1):196–204.
</cite> [<a href="https://doi.org/10.1128/AEM.02802-12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3536114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23087032/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ryu%20H,%20Henson%20M,%20Elk%20M,%20Toledo-Hernandez%20C,%20Griffith%20J,%20Blackwood%20D,%20et%20al.%20Development%20of%20quantitative%20PCR%20assays%20targeting%20the%2016S%20rRNA%20genes%20of%20Enterococcus%20spp.%20and%20their%20application%20to%20the%20identification%20of%20Enterococcus%20species%20in%20environmental%20samples.%20Appl%20Environ%20Microbiol.%202013;79(1):196%E2%80%93204." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Vandeputte D, Kathagen G, D’hoe K, Vieira-Silva S, Valles-Colomer M, Sabino J, et al. Quantitative microbiome profiling links gut community variation to microbial load. Nature. 2017;551(7681):507–11.
</cite> [<a href="https://doi.org/10.1038/nature24460" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29143816/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vandeputte%20D,%20Kathagen%20G,%20D%E2%80%99hoe%20K,%20Vieira-Silva%20S,%20Valles-Colomer%20M,%20Sabino%20J,%20et%20al.%20Quantitative%20microbiome%20profiling%20links%20gut%20community%20variation%20to%20microbial%20load.%20Nature.%202017;551(7681):507%E2%80%9311." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>McLaren MR, Willis AD, Callahan BJ. Consistent and correctable bias in metagenomic sequencing experiments. Elife. 2019;8:e46923.
</cite> [<a href="https://doi.org/10.7554/eLife.46923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6739870/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31502536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McLaren%20MR,%20Willis%20AD,%20Callahan%20BJ.%20Consistent%20and%20correctable%20bias%20in%20metagenomic%20sequencing%20experiments.%20Elife.%202019;8:e46923." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Morton JT, Marotz C, Washburne A, Silverman J, Zaramela LS, Edlund A, et al. Establishing microbial composition measurement standards with reference frames. Nat Commun. 2019;10(1):2719.
</cite> [<a href="https://doi.org/10.1038/s41467-019-10656-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6586903/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31222023/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morton%20JT,%20Marotz%20C,%20Washburne%20A,%20Silverman%20J,%20Zaramela%20LS,%20Edlund%20A,%20et%20al.%20Establishing%20microbial%20composition%20measurement%20standards%20with%20reference%20frames.%20Nat%20Commun.%202019;10(1):2719." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Hu Y, Satten GA, Hu YJ. LOCOM: A logistic regression model for testing differential abundance in compositional microbiome data with false discovery rate control. Proc Natl Acad Sci. 2022;119(30):e2122788119.
</cite> [<a href="https://doi.org/10.1073/pnas.2122788119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9335309/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35867822/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu%20Y,%20Satten%20GA,%20Hu%20YJ.%20LOCOM:%20A%20logistic%20regression%20model%20for%20testing%20differential%20abundance%20in%20compositional%20microbiome%20data%20with%20false%20discovery%20rate%20control.%20Proc%20Natl%20Acad%20Sci.%202022;119(30):e2122788119." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Zhou H, He K, Chen J, Zhang X. LinDA: linear models for differential abundance analysis of microbiome compositional data. Genome Biol. 2022;23(1):1–23.
</cite> [<a href="https://doi.org/10.1186/s13059-022-02655-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9012043/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35421994/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou%20H,%20He%20K,%20Chen%20J,%20Zhang%20X.%20LinDA:%20linear%20models%20for%20differential%20abundance%20analysis%20of%20microbiome%20compositional%20data.%20Genome%20Biol.%202022;23(1):1%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Nixon MP, McGovern KC, Letourneau J, David LA, Lazar NA, Mukherjee S, Silverman JD. Scale reliant inference. 2022. arXiv preprint arXiv:2201.03616.</cite>
</li>
<li id="CR17">
<span class="label">17.</span><cite>Lin H, Peddada SD. Multigroup analysis of compositions of microbiomes with covariate adjustments and repeated measures. Nat Methods. 2024;21(1):83–91.
</cite> [<a href="https://doi.org/10.1038/s41592-023-02092-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10776411/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38158428/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin%20H,%20Peddada%20SD.%20Multigroup%20analysis%20of%20compositions%20of%20microbiomes%20with%20covariate%20adjustments%20and%20repeated%20measures.%20Nat%20Methods.%202024;21(1):83%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Clausen DS, Teichman S, Willis AD. Estimating fold changes from partially observed outcomes with applications in microbial metagenomics. 2024. arXiv preprint arXiv:2402.05231.</cite>
</li>
<li id="CR19">
<span class="label">19.</span><cite>Weiss S, Xu ZZ, Peddada S, Amir A, Bittinger K, Gonzalez A, et al. Normalization and microbial differential abundance strategies depend upon data characteristics. Microbiome. 2017;5:1–18.
</cite> [<a href="https://doi.org/10.1186/s40168-017-0237-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5335496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28253908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weiss%20S,%20Xu%20ZZ,%20Peddada%20S,%20Amir%20A,%20Bittinger%20K,%20Gonzalez%20A,%20et%20al.%20Normalization%20and%20microbial%20differential%20abundance%20strategies%20depend%20upon%20data%20characteristics.%20Microbiome.%202017;5:1%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>McMurdie PJ, Holmes S. Waste not, want not: why rarefying microbiome data is inadmissible. PLoS Comput Biol. 2014;10(4):e1003531.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1003531" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3974642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24699258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McMurdie%20PJ,%20Holmes%20S.%20Waste%20not,%20want%20not:%20why%20rarefying%20microbiome%20data%20is%20inadmissible.%20PLoS%20Comput%20Biol.%202014;10(4):e1003531." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Willis AD. Rarefaction, alpha diversity, and statistics. Front Microbiol. 2019;10:2407.
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.02407" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6819366/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31708888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Willis%20AD.%20Rarefaction,%20alpha%20diversity,%20and%20statistics.%20Front%20Microbiol.%202019;10:2407." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Costea PI, Zeller G, Sunagawa S, Bork P. A fair comparison. Nat Methods. 2014;11(4):359.
</cite> [<a href="https://doi.org/10.1038/nmeth.2897" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24681719/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Costea%20PI,%20Zeller%20G,%20Sunagawa%20S,%20Bork%20P.%20A%20fair%20comparison.%20Nat%20Methods.%202014;11(4):359." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Nearing JT, Douglas GM, Hayes MG, MacDonald J, Desai DK, Allward N, et al. Microbiome differential abundance methods produce different results across 38 datasets. Nat Commun. 2022;13(1):342.
</cite> [<a href="https://doi.org/10.1038/s41467-022-28034-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8763921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35039521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nearing%20JT,%20Douglas%20GM,%20Hayes%20MG,%20MacDonald%20J,%20Desai%20DK,%20Allward%20N,%20et%20al.%20Microbiome%20differential%20abundance%20methods%20produce%20different%20results%20across%2038%20datasets.%20Nat%20Commun.%202022;13(1):342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Schloss PD. Removal of rare amplicon sequence variants from 16S rRNA gene sequence surveys biases the interpretation of community structure data. bioRxiv. 2020. 10.1101/2020.12.11.422279.</cite>
</li>
<li id="CR25">
<span class="label">25.</span><cite>Gibbons SM, Duvallet C, Alm EJ. Correcting for batch effects in case-control microbiome studies. PLoS Comput Biol. 2018;14(4):e1006102.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1006102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5940237/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29684016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gibbons%20SM,%20Duvallet%20C,%20Alm%20EJ.%20Correcting%20for%20batch%20effects%20in%20case-control%20microbiome%20studies.%20PLoS%20Comput%20Biol.%202018;14(4):e1006102." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Weiss S, Amir A, Hyde ER, Metcalf JL, Song SJ, Knight R. Tracking down the sources of experimental contamination in microbiome studies. Genome Biol. 2014;15(12):1–3.</cite> [<a href="https://doi.org/10.1186/s13059-014-0564-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4311479/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25608874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weiss%20S,%20Amir%20A,%20Hyde%20ER,%20Metcalf%20JL,%20Song%20SJ,%20Knight%20R.%20Tracking%20down%20the%20sources%20of%20experimental%20contamination%20in%20microbiome%20studies.%20Genome%20Biol.%202014;15(12):1%E2%80%933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Nguyen NH, Smith D, Peay K, Kennedy P. Parsing ecological signal from noise in next generation amplicon sequencing. New Phytol. 2015;205(4):1389–93.
</cite> [<a href="https://doi.org/10.1111/nph.12923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24985885/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen%20NH,%20Smith%20D,%20Peay%20K,%20Kennedy%20P.%20Parsing%20ecological%20signal%20from%20noise%20in%20next%20generation%20amplicon%20sequencing.%20New%20Phytol.%202015;205(4):1389%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Leek JT, Scharpf RB, Bravo HC, Simcha D, Langmead B, Johnson WE, et al. Tackling the widespread and critical impact of batch effects in high-throughput data. Nat Rev Genet. 2010;11(10):733–9.
</cite> [<a href="https://doi.org/10.1038/nrg2825" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3880143/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20838408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leek%20JT,%20Scharpf%20RB,%20Bravo%20HC,%20Simcha%20D,%20Langmead%20B,%20Johnson%20WE,%20et%20al.%20Tackling%20the%20widespread%20and%20critical%20impact%20of%20batch%20effects%20in%20high-throughput%20data.%20Nat%20Rev%20Genet.%202010;11(10):733%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Nygaard V, Rødland EA, Hovig E. Methods that remove batch effects while retaining group differences may lead to exaggerated confidence in downstream analyses. Biostatistics. 2016;17(1):29–39.
</cite> [<a href="https://doi.org/10.1093/biostatistics/kxv027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4679072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26272994/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nygaard%20V,%20R%C3%B8dland%20EA,%20Hovig%20E.%20Methods%20that%20remove%20batch%20effects%20while%20retaining%20group%20differences%20may%20lead%20to%20exaggerated%20confidence%20in%20downstream%20analyses.%20Biostatistics.%202016;17(1):29%E2%80%9339." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Goh WWB, Wang W, Wong L. Why batch effects matter in omics data, and how to avoid them. Trends Biotechnol. 2017;35(6):498–507.
</cite> [<a href="https://doi.org/10.1016/j.tibtech.2017.02.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28351613/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goh%20WWB,%20Wang%20W,%20Wong%20L.%20Why%20batch%20effects%20matter%20in%20omics%20data,%20and%20how%20to%20avoid%20them.%20Trends%20Biotechnol.%202017;35(6):498%E2%80%93507." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Evangelou E, Ioannidis JP. Meta-analysis methods for genome-wide association studies and beyond. Nat Rev Genet. 2013;14(6):379–89.
</cite> [<a href="https://doi.org/10.1038/nrg3472" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23657481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Evangelou%20E,%20Ioannidis%20JP.%20Meta-analysis%20methods%20for%20genome-wide%20association%20studies%20and%20beyond.%20Nat%20Rev%20Genet.%202013;14(6):379%E2%80%9389." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Lin DY, Tang ZZ. A general framework for detecting disease associations with rare variants in sequencing studies. Am J Hum Genet. 2011;89(3):354–67.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2011.07.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3169821/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21885029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin%20DY,%20Tang%20ZZ.%20A%20general%20framework%20for%20detecting%20disease%20associations%20with%20rare%20variants%20in%20sequencing%20studies.%20Am%20J%20Hum%20Genet.%202011;89(3):354%E2%80%9367." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Tang ZZ, Lin DY. Meta-analysis of sequencing studies with heterogeneous genetic associations. Genet Epidemiol. 2014;38(5):389–401.
</cite> [<a href="https://doi.org/10.1002/gepi.21798" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4157393/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24799183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang%20ZZ,%20Lin%20DY.%20Meta-analysis%20of%20sequencing%20studies%20with%20heterogeneous%20genetic%20associations.%20Genet%20Epidemiol.%202014;38(5):389%E2%80%93401." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Beale EML, Kendall MG, Mann D. The discarding of variables in multivariate analysis. Biometrika. 1967;54(3–4):357–66.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/6063999/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beale%20EML,%20Kendall%20MG,%20Mann%20D.%20The%20discarding%20of%20variables%20in%20multivariate%20analysis.%20Biometrika.%201967;54(3%E2%80%934):357%E2%80%9366." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Hocking RR, Leslie R. Selection of the best subset in regression analysis. Technometrics. 1967;9(4):531–40.</cite> [<a href="https://scholar.google.com/scholar_lookup?Hocking%20RR,%20Leslie%20R.%20Selection%20of%20the%20best%20subset%20in%20regression%20analysis.%20Technometrics.%201967;9(4):531%E2%80%9340." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Mallick H, Rahnavard A, McIver LJ, Ma S, Zhang Y, Nguyen LH, et al. Multivariable association discovery in population-scale meta-omics studies. PLoS Comput Biol. 2021;17(11):e1009442.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1009442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8714082/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34784344/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mallick%20H,%20Rahnavard%20A,%20McIver%20LJ,%20Ma%20S,%20Zhang%20Y,%20Nguyen%20LH,%20et%20al.%20Multivariable%20association%20discovery%20in%20population-scale%20meta-omics%20studies.%20PLoS%20Comput%20Biol.%202021;17(11):e1009442." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Viechtbauer W. Conducting meta-analyses in R with the metafor package. J Stat Softw. 2010;36(3):1–48.</cite> [<a href="https://scholar.google.com/scholar_lookup?Viechtbauer%20W.%20Conducting%20meta-analyses%20in%20R%20with%20the%20metafor%20package.%20J%20Stat%20Softw.%202010;36(3):1%E2%80%9348." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Fernandes AD, Reid JN, Macklaim JM, McMurrough TA, Edgell DR, Gloor GB. Unifying the analysis of high-throughput sequencing datasets: characterizing RNA-seq, 16S rRNA gene sequencing and selective growth experiments by compositional data analysis. Microbiome. 2014;2:1–13.
</cite> [<a href="https://doi.org/10.1186/2049-2618-2-15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4030730/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24910773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fernandes%20AD,%20Reid%20JN,%20Macklaim%20JM,%20McMurrough%20TA,%20Edgell%20DR,%20Gloor%20GB.%20Unifying%20the%20analysis%20of%20high-throughput%20sequencing%20datasets:%20characterizing%20RNA-seq,%2016S%20rRNA%20gene%20sequencing%20and%20selective%20growth%20experiments%20by%20compositional%20data%20analysis.%20Microbiome.%202014;2:1%E2%80%9313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Susin A, Wang Y, Lê Cao KA, Calle ML. Variable selection in microbiome compositional data analysis. NAR Genomics Bioinforma. 2020;2(2):lqaa029.</cite> [<a href="https://doi.org/10.1093/nargab/lqaa029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7671404/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33575585/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Susin%20A,%20Wang%20Y,%20L%C3%AA%20Cao%20KA,%20Calle%20ML.%20Variable%20selection%20in%20microbiome%20compositional%20data%20analysis.%20NAR%20Genomics%20Bioinforma.%202020;2(2):lqaa029." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Tibshirani R. Regression shrinkage and selection via the lasso. J R Stat Soc Ser B Stat Methodol. 1996;58(1):267–88.</cite> [<a href="https://scholar.google.com/scholar_lookup?Tibshirani%20R.%20Regression%20shrinkage%20and%20selection%20via%20the%20lasso.%20J%20R%20Stat%20Soc%20Ser%20B%20Stat%20Methodol.%201996;58(1):267%E2%80%9388." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Yu J, Feng Q, Wong SH, Zhang D, Yi Liang Q, Qin Y, et al. Metagenomic analysis of faecal microbiome as a tool towards targeted non-invasive biomarkers for colorectal cancer. Gut. 2017;66(1):70–8.
</cite> [<a href="https://doi.org/10.1136/gutjnl-2015-309800" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26408641/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu%20J,%20Feng%20Q,%20Wong%20SH,%20Zhang%20D,%20Yi%20Liang%20Q,%20Qin%20Y,%20et%20al.%20Metagenomic%20analysis%20of%20faecal%20microbiome%20as%20a%20tool%20towards%20targeted%20non-invasive%20biomarkers%20for%20colorectal%20cancer.%20Gut.%202017;66(1):70%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Vogtmann E, Hua X, Zeller G, Sunagawa S, Voigt AY, Hercog R, et al. Colorectal cancer and the human gut microbiome: reproducibility with whole-genome shotgun sequencing. PloS One. 2016;11(5):e0155362.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0155362" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4865240/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27171425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vogtmann%20E,%20Hua%20X,%20Zeller%20G,%20Sunagawa%20S,%20Voigt%20AY,%20Hercog%20R,%20et%20al.%20Colorectal%20cancer%20and%20the%20human%20gut%20microbiome:%20reproducibility%20with%20whole-genome%20shotgun%20sequencing.%20PloS%20One.%202016;11(5):e0155362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Feng Q, Liang S, Jia H, Stadlmayr A, Tang L, Lan Z, et al. Gut microbiome development along the colorectal adenoma-carcinoma sequence. Nat Commun. 2015;6(1):6528.
</cite> [<a href="https://doi.org/10.1038/ncomms7528" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25758642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Feng%20Q,%20Liang%20S,%20Jia%20H,%20Stadlmayr%20A,%20Tang%20L,%20Lan%20Z,%20et%20al.%20Gut%20microbiome%20development%20along%20the%20colorectal%20adenoma-carcinoma%20sequence.%20Nat%20Commun.%202015;6(1):6528." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Zeller G, Tap J, Voigt AY, Sunagawa S, Kultima JR, Costea PI, et al. Potential of fecal microbiota for early-stage detection of colorectal cancer. Mol Syst Biol. 2014;10(11):766.
</cite> [<a href="https://doi.org/10.15252/msb.20145645" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4299606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25432777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeller%20G,%20Tap%20J,%20Voigt%20AY,%20Sunagawa%20S,%20Kultima%20JR,%20Costea%20PI,%20et%20al.%20Potential%20of%20fecal%20microbiota%20for%20early-stage%20detection%20of%20colorectal%20cancer.%20Mol%20Syst%20Biol.%202014;10(11):766." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J R Stat Soc Ser B (Methodol). 1995;57(1):289–300.</cite> [<a href="https://scholar.google.com/scholar_lookup?Benjamini%20Y,%20Hochberg%20Y.%20Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing.%20J%20R%20Stat%20Soc%20Ser%20B%20(Methodol).%201995;57(1):289%E2%80%93300." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Zhao L, Zhang X, Zhou Y, Fu K, Lau HCH, Chun TWY, et al. Parvimonas micra promotes colorectal tumorigenesis and is associated with prognosis of colorectal cancer patients. Oncogene. 2022;41(36):4200–10.
</cite> [<a href="https://doi.org/10.1038/s41388-022-02395-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9439953/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35882981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20L,%20Zhang%20X,%20Zhou%20Y,%20Fu%20K,%20Lau%20HCH,%20Chun%20TWY,%20et%20al.%20Parvimonas%20micra%20promotes%20colorectal%20tumorigenesis%20and%20is%20associated%20with%20prognosis%20of%20colorectal%20cancer%20patients.%20Oncogene.%202022;41(36):4200%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Wong SH, Yu J. Gut microbiota in colorectal cancer: mechanisms of action and clinical applications. Nat Rev Gastroenterol Hepatol. 2019;16(11):690–704.
</cite> [<a href="https://doi.org/10.1038/s41575-019-0209-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31554963/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wong%20SH,%20Yu%20J.%20Gut%20microbiota%20in%20colorectal%20cancer:%20mechanisms%20of%20action%20and%20clinical%20applications.%20Nat%20Rev%20Gastroenterol%20Hepatol.%202019;16(11):690%E2%80%93704." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Erawijantari PP, Mizutani S, Shiroma H, Shiba S, Nakajima T, Sakamoto T, et al. Influence of gastrectomy for gastric cancer treatment on faecal microbiome and metabolome profiles. Gut. 2020;69(8):1404–15.
</cite> [<a href="https://doi.org/10.1136/gutjnl-2019-319188" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7398469/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31953253/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Erawijantari%20PP,%20Mizutani%20S,%20Shiroma%20H,%20Shiba%20S,%20Nakajima%20T,%20Sakamoto%20T,%20et%20al.%20Influence%20of%20gastrectomy%20for%20gastric%20cancer%20treatment%20on%20faecal%20microbiome%20and%20metabolome%20profiles.%20Gut.%202020;69(8):1404%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Yachida S, Mizutani S, Shiroma H, Shiba S, Nakajima T, Sakamoto T, et al. Metagenomic and metabolomic analyses reveal distinct stage-specific phenotypes of the gut microbiota in colorectal cancer. Nat Med. 2019;25(6):968–76.
</cite> [<a href="https://doi.org/10.1038/s41591-019-0458-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31171880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yachida%20S,%20Mizutani%20S,%20Shiroma%20H,%20Shiba%20S,%20Nakajima%20T,%20Sakamoto%20T,%20et%20al.%20Metagenomic%20and%20metabolomic%20analyses%20reveal%20distinct%20stage-specific%20phenotypes%20of%20the%20gut%20microbiota%20in%20colorectal%20cancer.%20Nat%20Med.%202019;25(6):968%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Kim M, Vogtmann E, Ahlquist DA, Devens ME, Kisiel JB, Taylor WR, et al. Fecal metabolomic signatures in colorectal adenoma patients are associated with gut microbiota and early events of colorectal cancer pathogenesis. MBio. 2020;11(1):10–1128.</cite> [<a href="https://doi.org/10.1128/mBio.03186-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7029137/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32071266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20M,%20Vogtmann%20E,%20Ahlquist%20DA,%20Devens%20ME,%20Kisiel%20JB,%20Taylor%20WR,%20et%20al.%20Fecal%20metabolomic%20signatures%20in%20colorectal%20adenoma%20patients%20are%20associated%20with%20gut%20microbiota%20and%20early%20events%20of%20colorectal%20cancer%20pathogenesis.%20MBio.%202020;11(1):10%E2%80%931128." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Franzosa EA, Sirota-Madi A, Avila-Pacheco J, Fornelos N, Haiser HJ, Reinker S, et al. Gut microbiome structure and metabolic activity in inflammatory bowel disease. Nat Microbiol. 2019;4(2):293–305.
</cite> [<a href="https://doi.org/10.1038/s41564-018-0306-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6342642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30531976/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Franzosa%20EA,%20Sirota-Madi%20A,%20Avila-Pacheco%20J,%20Fornelos%20N,%20Haiser%20HJ,%20Reinker%20S,%20et%20al.%20Gut%20microbiome%20structure%20and%20metabolic%20activity%20in%20inflammatory%20bowel%20disease.%20Nat%20Microbiol.%202019;4(2):293%E2%80%93305." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Mars RA, Yang Y, Ward T, Houtti M, Priya S, Lekatz HR, et al. Longitudinal multi-omics reveals subset-specific mechanisms underlying irritable bowel syndrome. Cell. 2020;182(6):1460–73.
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.08.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8109273/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32916129/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mars%20RA,%20Yang%20Y,%20Ward%20T,%20Houtti%20M,%20Priya%20S,%20Lekatz%20HR,%20et%20al.%20Longitudinal%20multi-omics%20reveals%20subset-specific%20mechanisms%20underlying%20irritable%20bowel%20syndrome.%20Cell.%202020;182(6):1460%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Lloyd-Price J, Arze C, Ananthakrishnan AN, Schirmer M, Avila-Pacheco J, Poon TW, et al. Multi-omics of the gut microbial ecosystem in inflammatory bowel diseases. Nature. 2019;569(7758):655–62.
</cite> [<a href="https://doi.org/10.1038/s41586-019-1237-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6650278/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31142855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lloyd-Price%20J,%20Arze%20C,%20Ananthakrishnan%20AN,%20Schirmer%20M,%20Avila-Pacheco%20J,%20Poon%20TW,%20et%20al.%20Multi-omics%20of%20the%20gut%20microbial%20ecosystem%20in%20inflammatory%20bowel%20diseases.%20Nature.%202019;569(7758):655%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Wang X, Yang S, Li S, Zhao L, Hao Y, Qin J, et al. Aberrant gut microbiota alters host metabolome and impacts renal failure in humans and rodents. Gut. 2020;69(12):2131–42.
</cite> [<a href="https://doi.org/10.1136/gutjnl-2019-319766" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7677483/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32241904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20X,%20Yang%20S,%20Li%20S,%20Zhao%20L,%20Hao%20Y,%20Qin%20J,%20et%20al.%20Aberrant%20gut%20microbiota%20alters%20host%20metabolome%20and%20impacts%20renal%20failure%20in%20humans%20and%20rodents.%20Gut.%202020;69(12):2131%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Poyet M, Groussin M, Gibbons SM, Avila-Pacheco J, Jiang X, Kearney SM, et al. A library of human gut bacterial isolates paired with longitudinal multiomics data enables mechanistic microbiome research. Nat Med. 2019;25(9):1442–52.
</cite> [<a href="https://doi.org/10.1038/s41591-019-0559-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31477907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Poyet%20M,%20Groussin%20M,%20Gibbons%20SM,%20Avila-Pacheco%20J,%20Jiang%20X,%20Kearney%20SM,%20et%20al.%20A%20library%20of%20human%20gut%20bacterial%20isolates%20paired%20with%20longitudinal%20multiomics%20data%20enables%20mechanistic%20microbiome%20research.%20Nat%20Med.%202019;25(9):1442%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Wishart DS, Feunang YD, Marcu A, Guo AC, Liang K, Vázquez-Fresno R, et al. HMDB 4.0: the human metabolome database for 2018. Nucleic Acids Res. 2018;46(D1):D608–17.
</cite> [<a href="https://doi.org/10.1093/nar/gkx1089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5753273/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29140435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wishart%20DS,%20Feunang%20YD,%20Marcu%20A,%20Guo%20AC,%20Liang%20K,%20V%C3%A1zquez-Fresno%20R,%20et%20al.%20HMDB%204.0:%20the%20human%20metabolome%20database%20for%202018.%20Nucleic%20Acids%20Res.%202018;46(D1):D608%E2%80%9317." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Duncan SH, Louis P, Flint HJ. Lactate-utilizing bacteria, isolated from human feces, that produce butyrate as a major fermentation product. Appl Environ Microbiol. 2004;70(10):5810–7.
</cite> [<a href="https://doi.org/10.1128/AEM.70.10.5810-5817.2004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC522113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15466518/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duncan%20SH,%20Louis%20P,%20Flint%20HJ.%20Lactate-utilizing%20bacteria,%20isolated%20from%20human%20feces,%20that%20produce%20butyrate%20as%20a%20major%20fermentation%20product.%20Appl%20Environ%20Microbiol.%202004;70(10):5810%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Louis P, Hold GL, Flint HJ. The gut microbiota, bacterial metabolites and colorectal cancer. Nat Rev Microbiol. 2014;12(10):661–72.
</cite> [<a href="https://doi.org/10.1038/nrmicro3344" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25198138/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Louis%20P,%20Hold%20GL,%20Flint%20HJ.%20The%20gut%20microbiota,%20bacterial%20metabolites%20and%20colorectal%20cancer.%20Nat%20Rev%20Microbiol.%202014;12(10):661%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Machiels K, Joossens M, Sabino J, De Preter V, Arijs I, Eeckhaut V, et al. A decrease of the butyrate-producing species Roseburia hominis and Faecalibacterium prausnitzii defines dysbiosis in patients with ulcerative colitis. Gut. 2014;63(8):1275–83.
</cite> [<a href="https://doi.org/10.1136/gutjnl-2013-304833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24021287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Machiels%20K,%20Joossens%20M,%20Sabino%20J,%20De%20Preter%20V,%20Arijs%20I,%20Eeckhaut%20V,%20et%20al.%20A%20decrease%20of%20the%20butyrate-producing%20species%20Roseburia%20hominis%20and%20Faecalibacterium%20prausnitzii%20defines%20dysbiosis%20in%20patients%20with%20ulcerative%20colitis.%20Gut.%202014;63(8):1275%E2%80%9383." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Org E, Blum Y, Kasela S, Mehrabian M, Kuusisto J, Kangas AJ, et al. Relationships between gut microbiota, plasma metabolites, and metabolic syndrome traits in the METSIM cohort. Genome Biol. 2017;18:1–14.
</cite> [<a href="https://doi.org/10.1186/s13059-017-1194-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5390365/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28407784/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Org%20E,%20Blum%20Y,%20Kasela%20S,%20Mehrabian%20M,%20Kuusisto%20J,%20Kangas%20AJ,%20et%20al.%20Relationships%20between%20gut%20microbiota,%20plasma%20metabolites,%20and%20metabolic%20syndrome%20traits%20in%20the%20METSIM%20cohort.%20Genome%20Biol.%202017;18:1%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Zeng Q, Li D, He Y, Li Y, Yang Z, Zhao X, et al. Discrepant gut microbiota markers for the classification of obesity-related metabolic abnormalities. Sci Rep. 2019;9(1):13424.
</cite> [<a href="https://doi.org/10.1038/s41598-019-49462-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6748942/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31530820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng%20Q,%20Li%20D,%20He%20Y,%20Li%20Y,%20Yang%20Z,%20Zhao%20X,%20et%20al.%20Discrepant%20gut%20microbiota%20markers%20for%20the%20classification%20of%20obesity-related%20metabolic%20abnormalities.%20Sci%20Rep.%202019;9(1):13424." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Koeth RA, Levison BS, Culley MK, Buffa JA, Wang Z, Gregory JC, et al. <img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bd40/12359909/70e7f14307d9/13059_2025_3721_Article_IEq156.gif" loading="lazy" alt="Inline graphic">-Butyrobetaine is a proatherogenic intermediate in gut microbial metabolism of L-carnitine to TMAO. Cell Metab. 2014;20(5):799–812.</cite> [<a href="https://doi.org/10.1016/j.cmet.2014.10.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4255476/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25440057/" class="usa-link">PubMed</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Min Y, Ma X, Sankaran K, Ru Y, Chen L, Baiocchi M, et al. Sex-specific association between gut microbiome and fat distribution. Nat Commun. 2019;10(1):2408.
</cite> [<a href="https://doi.org/10.1038/s41467-019-10440-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6546740/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31160598/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Min%20Y,%20Ma%20X,%20Sankaran%20K,%20Ru%20Y,%20Chen%20L,%20Baiocchi%20M,%20et%20al.%20Sex-specific%20association%20between%20gut%20microbiome%20and%20fat%20distribution.%20Nat%20Commun.%202019;10(1):2408." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Borenstein M, Hedges LV, Higgins JP, Rothstein HR. Introduction to meta-analysis. Wiley; 2021.</cite>
</li>
<li id="CR65">
<span class="label">65.</span><cite>Kosmidis I, Kenne Pagui EC, Sartori N. Mean and median bias reduction in generalized linear models. Stat Comput. 2020;30(1):43–59.</cite> [<a href="https://scholar.google.com/scholar_lookup?Kosmidis%20I,%20Kenne%20Pagui%20EC,%20Sartori%20N.%20Mean%20and%20median%20bias%20reduction%20in%20generalized%20linear%20models.%20Stat%20Comput.%202020;30(1):43%E2%80%9359." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>He Q, Zhang HH, Avery CL, Lin D. Sparse meta-analysis with high-dimensional data. Biostatistics. 2016;17(2):205–20.
</cite> [<a href="https://doi.org/10.1093/biostatistics/kxv038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4834947/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26395907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He%20Q,%20Zhang%20HH,%20Avery%20CL,%20Lin%20D.%20Sparse%20meta-analysis%20with%20high-dimensional%20data.%20Biostatistics.%202016;17(2):205%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Zhu J, Wen C, Zhu J, Zhang H, Wang X. A polynomial algorithm for best-subset selection problem. Proc Natl Acad Sci. 2020;117(52):33117–23.
</cite> [<a href="https://doi.org/10.1073/pnas.2014241117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7777147/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33328272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhu%20J,%20Wen%20C,%20Zhu%20J,%20Zhang%20H,%20Wang%20X.%20A%20polynomial%20algorithm%20for%20best-subset%20selection%20problem.%20Proc%20Natl%20Acad%20Sci.%202020;117(52):33117%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Zhu J, Wang X, Hu L, Huang J, Jiang K, Zhang Y, et al. abess: a fast best-subset selection library in python and R. J Mach Learn Res. 2022;23(1):9206–12.</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhu%20J,%20Wang%20X,%20Hu%20L,%20Huang%20J,%20Jiang%20K,%20Zhang%20Y,%20et%20al.%20abess:%20a%20fast%20best-subset%20selection%20library%20in%20python%20and%20R.%20J%20Mach%20Learn%20Res.%202022;23(1):9206%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Powell MJ. An efficient method for finding the minimum of a function of several variables without calculating derivatives. Comput J. 1964;7(2):155–62.</cite> [<a href="https://scholar.google.com/scholar_lookup?Powell%20MJ.%20An%20efficient%20method%20for%20finding%20the%20minimum%20of%20a%20function%20of%20several%20variables%20without%20calculating%20derivatives.%20Comput%20J.%201964;7(2):155%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Heath MT. Scientific computing: an introductory survey, revised second edition. Philadelphia, PA: SIAM; 2018.</cite>
</li>
<li id="CR71">
<span class="label">71.</span><cite>Kiefer J. Sequential minimax search for a maximum. Proc Am Math Soc. 1953;4(3):502–6.</cite> [<a href="https://scholar.google.com/scholar_lookup?Kiefer%20J.%20Sequential%20minimax%20search%20for%20a%20maximum.%20Proc%20Am%20Math%20Soc.%201953;4(3):502%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Tang ZZ, Chen G. Zero-inflated generalized Dirichlet multinomial regression model for microbiome compositional data analysis. Biostatistics. 2019;20(4):698–713.
</cite> [<a href="https://doi.org/10.1093/biostatistics/kxy025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7410344/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29939212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang%20ZZ,%20Chen%20G.%20Zero-inflated%20generalized%20Dirichlet%20multinomial%20regression%20model%20for%20microbiome%20compositional%20data%20analysis.%20Biostatistics.%202019;20(4):698%E2%80%93713." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Muller E, Algavi YM, Borenstein E. The gut microbiome-metabolome dataset collection: a curated resource for integrative meta-analysis. npj Biofilms Microbiomes. 2022;8(1):79.
</cite> [<a href="https://doi.org/10.1038/s41522-022-00345-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9569371/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36243731/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Muller%20E,%20Algavi%20YM,%20Borenstein%20E.%20The%20gut%20microbiome-metabolome%20dataset%20collection:%20a%20curated%20resource%20for%20integrative%20meta-analysis.%20npj%20Biofilms%20Microbiomes.%202022;8(1):79." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Wirbel J. Colorectal cancer meta-analysis. Github. 2019. <a href="https://github.com/zellerlab/crc_meta" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/zellerlab/crc_meta</a>. Accessed 01 Dec 2023.</cite>
</li>
<li id="CR75">
<span class="label">75.</span><cite>Borenstein E. The curated gut microbiome metabolome data resource. Github. 2024. <a href="https://github.com/borensteinlab/microbiome-metabolome-curated-data" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/borensteinlab/microbiome-metabolome-curated-data</a>. Accessed 01 Jan 2024.</cite>
</li>
<li id="CR76">
<span class="label">76.</span><cite>Wei Z, Chen G, Tang ZZ. Melody: meta-analysis of microbiome association studies for discovering generalizable microbial signatures. Github. 2025. <a href="https://github.com/ZjpWei/Melody" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ZjpWei/Melody</a>. Accessed 01 Jun 2025.</cite>
</li>
<li id="CR77">
<span class="label">77.</span><cite>Wei Z, Chen G, Tang ZZ. miMeta R package. Github. 2025. <a href="https://github.com/ZjpWei/miMeta" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ZjpWei/miMeta</a>. Accessed 01 May 2025.</cite>
</li>
<li id="CR78">
<span class="label">78.</span><cite>Wei Z, Chen G, Tang ZZ. Melody: meta-analysis of microbiome association studies for discovering generalizable microbial signatures. Zenodo. 2025. 10.5281/zenodo.15666043.</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359909/bin/13059_2025_3721_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3721_MOESM1_ESM.pdf</a><sup> (2MB, pdf) </sup><p>Additional file 1: Supplementary Notes A and B, Figs S1-S14, and Tables S1, S3.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359909/bin/13059_2025_3721_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Additional file 2: Tables S2, S4, S5.</a><sup> (424.8KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>This study made use of 13 publicly available datasets. The five metagenomics data for colorectal cancer were obtained from <a href="https://github.com/zellerlab/crc_meta" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/zellerlab/crc_meta</a> [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>] and the eight microbiome-metabolome data were obtained from <a href="https://github.com/borenstein-lab/microbiome-metabolome-curated-data" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/borenstein-lab/microbiome-metabolome-curated-data</a> [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. All raw data and processed data used for analysis are also available at <a href="https://github.com/ZjpWei/Melody" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ZjpWei/Melody</a> [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>]. Melody is implemented in the R package miMeta publicly available at <a href="https://github.com/ZjpWei/miMeta" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ZjpWei/miMeta</a> [<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>]. The site also contains a complete set of manuals and instructions. The scripts generating reported results are deposited at Github <a href="https://github.com/ZjpWei/Melody" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ZjpWei/Melody</a> [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>] and Zenodo <a href="https://doi.org/10.5281/zenodo.15666043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5281/zenodo.15666043</a> [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>] under the GPL-3.0 license.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Genome Biology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s13059-025-03721-4"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/13059_2025_Article_3721.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12359909/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12359909/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12359909%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359909/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12359909/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12359909/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40826361/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12359909/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40826361/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12359909/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12359909/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="ltAlj6RpmwKpOmipRDw5H0FKGPHLqWRsxmJXzpwGGbCcXZQ68oPZ55B2TDesxT47">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
