
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Autoencoders with shared and specific embeddings for multi-omics data integration - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4AD888AF2F2B305AD88005BA60367.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcbioi">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362917/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Bioinformatics">
<meta name="citation_title" content="Autoencoders with shared and specific embeddings for multi-omics data integration">
<meta name="citation_author" content="Chao Wang">
<meta name="citation_author_institution" content="Ben May Department for Cancer Research, University of Chicago, 929 E. 57th St., Chicago, IL 60637 USA">
<meta name="citation_author" content="Michael J O’Connell">
<meta name="citation_author_institution" content="Department of Statistics, Miami University, 105 Tallawanda Rd., Oxford, OH 45056 USA">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="26">
<meta name="citation_firstpage" content="214">
<meta name="citation_doi" content="10.1186/s12859-025-06245-7">
<meta name="citation_pmid" content="40830833">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362917/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362917/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362917/pdf/12859_2025_Article_6245.pdf">
<meta name="description" content="In cancer research, different levels of high-dimensional data are often collected for the same subjects. Effective integration of these data by considering the shared and specific information from each data source can help us better understand ...">
<meta name="og:title" content="Autoencoders with shared and specific embeddings for multi-omics data integration">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="In cancer research, different levels of high-dimensional data are often collected for the same subjects. Effective integration of these data by considering the shared and specific information from each data source can help us better understand ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362917/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12362917">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12859-025-06245-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12859_2025_Article_6245.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12362917%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12362917/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12362917/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362917/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcbioi.png" alt="BMC Bioinformatics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Bioinformatics" title="Link to BMC Bioinformatics" shape="default" href="https://bmcbioinformatics.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Bioinformatics</button></div>. 2025 Aug 19;26:214. doi: <a href="https://doi.org/10.1186/s12859-025-06245-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12859-025-06245-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Bioinformatics%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Bioinformatics%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Bioinformatics%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Bioinformatics%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Autoencoders with shared and specific embeddings for multi-omics data integration</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Chao Wang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Chao Wang</span></h3>
<div class="p">
<sup>1</sup>Ben May Department for Cancer Research, University of Chicago, 929 E. 57th St., Chicago, IL 60637 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chao Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22O%E2%80%99Connell%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Michael J O’Connell</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Michael J O’Connell</span></h3>
<div class="p">
<sup>2</sup>Department of Statistics, Miami University, 105 Tallawanda Rd., Oxford, OH 45056 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22O%E2%80%99Connell%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael J O’Connell</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Ben May Department for Cancer Research, University of Chicago, 929 E. 57th St., Chicago, IL 60637 USA </div>
<div id="Aff2">
<sup>2</sup>Department of Statistics, Miami University, 105 Tallawanda Rd., Oxford, OH 45056 USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 23; Accepted 2025 Jul 30; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12362917  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830833/" class="usa-link">40830833</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title"><strong>Background</strong></h3>
<p id="Par1">In cancer research, different levels of high-dimensional data are often collected for the same subjects. Effective integration of these data by considering the shared and specific information from each data source can help us better understand different types of cancer.</p></section><section id="sec2"><h3 class="pmc_sec_title"><strong>Results</strong></h3>
<p id="Par2">In this study we propose a novel autoencoder (AE) structure with explicitly defined orthogonal loss between the shared and specific embeddings to integrate different data sources. We compare our model with previously proposed AE structures based on simulated data and real cancer data from The Cancer Genome Atlas. Using simulations with different proportions of differentially expressed genes, we compare the performance of AE methods for subsequent classification tasks. We also compare the model performance with a commonly used dimension reduction method, joint and individual variance explained (JIVE). In terms of reconstruction loss, our proposed AE models with orthogonal constraints have a slightly better reconstruction loss. All AE models achieve higher classification accuracy than the original features, demonstrating the usefulness of the embeddings extracted by the model.</p></section><section id="sec3"><h3 class="pmc_sec_title"><strong>Conclusions</strong></h3>
<p id="Par3">We show that the proposed models have consistently high classification accuracy on both training and testing sets. In comparison, the recently proposed MOCSS model that imposes an orthogonality penalty in the post-processing step has lower classification accuracy that is on par with JIVE.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Multi-omics, Data integration, Autoencoders, Shared and specific information</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par4">Data integration is the process of combining data from different modalities or sources that each provide a separate view of the common phenomenon. Compared with the analysis of the individual data modality, different data sources can provide complementary information thus promising a more comprehensive understanding of the system [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. This is particularly important in the biomedical field, where multi-omics data including genomics, transcriptomics, proteomics, epigenomics, metabolomics, and meta-genomics are increasingly made available for different study objectives. One of the most important examples is The Cancer Genome Atlas (TCGA), a project that has collected different types of omics data for over 20,000 tumor and normal samples, which collectively cover 33 different types of cancers with clinical information for a subset of subjects [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. In addition to an effective integration strategy to account for the distributional difference of the data source, each modality of data is often also high dimensional. For example, gene expression profiles for each subject in the TCGA database consist of 60,483 measurements (FPKM: fragments per kilobase of transcript per million mapped reads) corresponding to the expression measurement for each gene at the transcript level. DNA methylation profile for each subject was determined by Infinium HumanMethylation 450 BeadChip Arrays containing 485,578 probes that measure the methylation level for a small part of the human genome [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. The high dimensionality can pose significant analysis challenges for classic statistical methods and machine-learning techniques since the number of features in each modality is larger than the number of observations (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/150a91157ba1/d33e187.gif" loading="lazy" id="d33e187" alt="Inline graphic"></span>). Consequently, dimension reduction such as principal component analysis (PCA) or customized filtering (e.g., only considering differentially expressed genes or methylation probes in the analysis) are often needed either before or during the data integration process.</p>
<p id="Par5">The majority of the data integration methods can be grouped into four different categories: dimension reduction, kernel, network, and deep learning methods [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Among these, joint dimension reduction methods are probably the most heavily explored, with integrative extensions of commonly used dimension reduction methods like principal component analysis [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>], canonical correlation analysis [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>], and non-negative matrix factorization [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>, <a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>].</p>
<p id="Par6">However, there has recently been more interest in deep learning methods for data integration. Deep learning methods are more flexible, and they can identify non-linear patterns much more easily than most dimension reduction methods, due to the non-linear activation functions of most deep learning methods. Deep learning models can learn the hierarchical representation of data automatically without the constraints of linear relationships, making them particularly suitable and flexible for the integration of multi-omics data in biomedical applications [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>].</p>
<p id="Par7">An autoencoder (AE) is a deep-learning approach to find the latent representation of the input with a lower dimension that contains the necessary information to reconstruct the original input. An AE consists of an encoder function <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/75a72dbb8086/d33e221.gif" loading="lazy" id="d33e221" alt="Inline graphic"></span> parameterized by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/edd700580a15/d33e227.gif" loading="lazy" id="d33e227" alt="Inline graphic"></span> and a decoder function <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/f609b7072445/d33e233.gif" loading="lazy" id="d33e233" alt="Inline graphic"></span> parameterized by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/dbad5f9d7f55/d33e239.gif" loading="lazy" id="d33e239" alt="Inline graphic"></span> such that for a single input <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/d437809d78f6/d33e245.gif" loading="lazy" id="d33e245" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/5542f09c1bb6/d33e252.gif" loading="lazy" id="d33e252" alt="Inline graphic"></span> and the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/6f72f23f78cc/d33e258.gif" loading="lazy" id="d33e258" alt="Inline graphic"></span> is the embedding of the original input and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/1bf57fade40c/d33e264.gif" loading="lazy" id="d33e264" alt="Inline graphic"></span> is the reconstructed input. The goal of the AE is to capture the essential information in the data by minimizing the reconstruction error. Different metrics can be used to measure the error such as mean squared error: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/ed2ee7050dcb/d33e270.gif" loading="lazy" id="d33e270" alt="Inline graphic"></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/0a12696f95c2/d33e276.gif" loading="lazy" id="d33e276" alt="Inline graphic"></span> is the entire dataset and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/3c5886be7974/d33e282.gif" loading="lazy" id="d33e282" alt="Inline graphic"></span> is the corresponding reconstruction of the entire dataset after applying the AE model [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. If <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/5bc639cc779f/d33e292.gif" loading="lazy" id="d33e292" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/88ba5dfa5308/d33e298.gif" loading="lazy" id="d33e298" alt="Inline graphic"></span> are linear functions then <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/3c5886be7974/d33e304.gif" loading="lazy" id="d33e304" alt="Inline graphic"></span> will lie in the principle component subspace making the AE similar to PCA. If <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/5bc639cc779f/d33e310.gif" loading="lazy" id="d33e310" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/88ba5dfa5308/d33e316.gif" loading="lazy" id="d33e316" alt="Inline graphic"></span> are nonlinear, the input will then be mapped onto a lower dimensional manifold that is more informative than principle components when there are non-linear interactions in the data [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. In practice, AEs are neural network architecture that often consists of fully connected layers of encoder and decoder. The encoder takes the input data and compresses them into lower dimensional representations and the decoder takes this lower dimensional representation and reconstructs the input at the final output layer.</p>
<p id="Par8">A number of different AE architectures with variational components were previously proposed for multi-omics data integration by [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. The most straightforward method is the Autoencoder with Concatenated Inputs (CNC_AE), which simply concatenates the scaled data sources as the input for AE. For this AE architecture, the encoder consists of a number of fully connected layers including the embedding layer to transform the concatenated input of different data sources (i.e., concatenation of X1 and X2) from a higher dimension to a lower dimensional representation. The corresponding decoder then takes this lower dimensional representation as the input aiming to reconstruct the original input for different data sources separately [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. The X-shaped Autoencoder (X_AE) is a slightly modified version of the CNC_AE, in which the individual data sources are first preprocessed separately before being joined together to be further processed by the model [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Finally, instead of directly concatenating the preprocessed data source, a pair-wise mutual concatenation of the input is used in the Mixed-Modal Autoencoder (MM_AE) model to take advantage of the potentially shared information among the different data sources [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Despite these different architectures, all these AE structures fail to take advantage of the potentially joint and individual components/information across the data source in their architectural design.</p>
<p id="Par9">A recent study implemented the notion of joint and individual components in AE for multi-omics data integration with the multi-omics data clustering and cancer subtyping via shared and specific representation learning (MOCSS) model [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. Specifically, two separate AEs were created for each data source and designated as the shared and specific respectively. With 2 different data sources, all 4 AE have the same architecture. However, instead of using the concatenated input from different data sources to extract the potentially shared information, 2 separate shared embedding vectors were extracted from each data source based on the aforementioned shared AEs. To encourage these embeddings to be shared from data source to data source, these vectors were first transformed through another fully connected neural network (MLP: multiple linear perceptrons) before a contrastive learning process was used to align them by reducing the pair-wise contrastive loss between these vectors. The orthogonal penalties were applied between the shared embedding vector and the specific embedding vector for each data source individually and then added together as the final orthogonal loss. Despite taking advantage of the potentially shared and specific information from the data source, the fact that MOCSS model aligns the shared information and imposed the orthogonal penalty during the post-processing process may loss some important information.</p>
<p id="Par10">In this study, we propose a novel architecture of AE model for multi-omics data integration, where the joint component is derived from the concatenated data sources and the individual component come from the corresponding individual data source. To encourage the model to separate and extract the joint/shared information contained between different omic data and the specific information contained in each data source, an additional orthogonal penalty is applied between the joint and the individual embedding layers. For comprehensiveness, 3 different version of orthogonal penalty was defined and compared, and the model without orthogonal penalty was also compared. Unlike MOCSS model, the joint component and orthogonal penalty are natural components of our model structure. For comparison, we implemented aforementioned AE structures and compared them on a comprehensive simulation setting in terms of input reconstruction and classification accuracy. For further validation, we also applied them on a real-world cancer dataset from TCGA and compared their performance with a dimension reduction model: Joint and Individual Variation Explained (JIVE), which is based on PCA [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. We chose to compare with JIVE because it similarly imposes an orthogonality constraint between the joint and individual components. It should be noted that, for a fair comparison, all AE models implemented in this study do not contain variational components.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Construction of AE models of different architectures</h3>
<p id="Par11">To compare the effectiveness of multi-omics data integration methods, different AE models were first applied to the simulated data and further validated on the real-world cancer data. We implemented the previously proposed AE models for omics data integration, CNC_AE, X_AE, MM_AE and MOCSS, as shown in Fig. <a href="#Fig1" class="usa-link">1</a>a–d. The proposed novel AE structure, Joint and Individual Simultaneous Autoencoder (JISAE) is shown in Fig. <a href="#Fig1" class="usa-link">1</a>e, where the encoder starts with the individual omics data as well as their concatenation. These three inputs were separately processed by 4 fully connected layers including the separate final embedding layers for three inputs. To encourage the model to separate and extract the joint (shared) information between the 2 different omics data sources and the individual (specific) information unique to each data source, in addition to the reconstruction loss, an additional orthogonal penalty is applied between the embedding layer of joint and the individual inputs as shown in the orthogonal loss diagram in Fig. <a href="#Fig1" class="usa-link">1</a>e. We refer to the models with an orthogonality penalty as JISAE-O. To evaluate the importance of the orthogonality penalties for the model performance, the JISAE model without orthogonal constraints is implemented for comparison, as shown in Fig. <a href="#Fig1" class="usa-link">1</a>f.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12362917_12859_2025_6245_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/c4ce438891fe/12859_2025_6245_Fig1_HTML.jpg" loading="lazy" id="MO1" height="589" width="709" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Different AE model structures compared in this study. <strong>a</strong> AE with direct X1, X2 concatenated together as the model input, referred as CNC_AE; <strong>b</strong> AE with processed X1, X2 concatenated together as the model input, referred as X_AE; <strong>c</strong> AE with different concatenation of processed X1, X2 as the model input, referred as MM_AE; <strong>d</strong> AE structure from previous study with orthogonal constraints imposed during the post-processing, referred as MOCSS [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]; <strong>e</strong> AE with both individual and concatenated X1, X2 as the model input with additional imposed orthogonal constraints, referred as JISAE-O; <strong>f</strong> similarly to model in <strong>e</strong> without additional orthogonal constraints, referred as JISAE. For the JISAE-O models, the orthogonality constraint was imposed directly in the loss function for estimating the model, rather than through post-processing</p></figcaption></figure><p id="Par12">To estimate the parameters for each of the AE models, the MSE between the inputs and the reconstructed inputs was used as the loss function for the optimization algorithm. Let <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/2cbbf2c5d697/d33e370.gif" loading="lazy" id="d33e370" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/be0bbab608e1/d33e376.gif" loading="lazy" id="d33e376" alt="Inline graphic"></span> be the reconstructed inputs for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/143a31c38863/d33e382.gif" loading="lazy" id="d33e382" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/7ded08912416/d33e388.gif" loading="lazy" id="d33e388" alt="Inline graphic"></span> respectively from AE models other than MOCSS (All of these matrices are normalized with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/89ac728b71d6/d33e394.gif" loading="lazy" id="d33e394" alt="Inline graphic"></span> normalization of inputs over embedding dimensions) and the reconstruction loss can be expressed in (1):</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/e023e70a1f0c/d33e401.gif" loading="lazy" id="d33e401" alt="graphic file with name d33e401.gif"></td>
<td class="label">1</td>
</tr></table>
<p>For the MOCSS model, for each data source, there are two reconstructed inputs from shared and specific AE models as shown in Fig. <a href="#Fig1" class="usa-link">1</a>d. Let <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/68199072221a/d33e411.gif" loading="lazy" id="d33e411" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/68199072221a/d33e417.gif" loading="lazy" id="d33e417" alt="Inline graphic"></span> represent the reconstructed inputs from shared and specific AE models respectively for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/143a31c38863/d33e423.gif" loading="lazy" id="d33e423" alt="Inline graphic"></span> and let <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/21de8a634c1f/d33e429.gif" loading="lazy" id="d33e429" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/0b44ceffe5c8/d33e436.gif" loading="lazy" id="d33e436" alt="Inline graphic"></span> represent the reconstructed inputs from shared and specific AE models respectively for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/7ded08912416/d33e442.gif" loading="lazy" id="d33e442" alt="Inline graphic"></span> (All of these matrices are normalized). The reconstruction loss for the MOCSS model can be expressed in (2):</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/a115756b8744/d33e448.gif" loading="lazy" id="d33e448" alt="graphic file with name d33e448.gif"></td>
<td class="label">2</td>
</tr></table>
<p>In the MOCSS model, the following penalty was imposed between the shared and specific embeddings for each data source. Let <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/a0b917b6883b/d33e455.gif" loading="lazy" id="d33e455" alt="Inline graphic"></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/6e2e51cd5f10/d33e461.gif" loading="lazy" id="d33e461" alt="Inline graphic"></span> to represent the shared and specific embeddings for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/143a31c38863/d33e467.gif" loading="lazy" id="d33e467" alt="Inline graphic"></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/5a66bea3e1ed/d33e474.gif" loading="lazy" id="d33e474" alt="Inline graphic"></span>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/06d22ca04533/d33e480.gif" loading="lazy" id="d33e480" alt="Inline graphic"></span> to represent the shared and specific embeddings for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/7ded08912416/d33e486.gif" loading="lazy" id="d33e486" alt="Inline graphic"></span> as shown in Fig. <a href="#Fig1" class="usa-link">1</a>d (All of these matrices are normalized). Then the orthogonal loss is calculated as shown in (3), where the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/19a719214a21/d33e495.gif" loading="lazy" id="d33e495" alt="Inline graphic"></span> represents the element-wise product and avg represents the average.</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/842b7201c648/d33e501.gif" loading="lazy" id="d33e501" alt="graphic file with name d33e501.gif"></td>
<td class="label">3</td>
</tr></table>
<p>To compare the effectiveness of different orthogonal loss penalties in our proposed AE structure, 3 different orthogonal loss functions were implemented resulting in three different JISAE-O models (distinguished as JISAE-O1, JISAE-O2, and JISAE-O3). Specifically, let <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/babcb219508d/d33e509.gif" loading="lazy" id="d33e509" alt="Inline graphic"></span> to represent the embedding from joint input <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/143a31c38863/d33e515.gif" loading="lazy" id="d33e515" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/7ded08912416/d33e521.gif" loading="lazy" id="d33e521" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/6e2e51cd5f10/d33e527.gif" loading="lazy" id="d33e527" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/06d22ca04533/d33e533.gif" loading="lazy" id="d33e533" alt="Inline graphic"></span> to represent the embedding from individual input <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/143a31c38863/d33e539.gif" loading="lazy" id="d33e539" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/7ded08912416/d33e546.gif" loading="lazy" id="d33e546" alt="Inline graphic"></span> respectively (All of these matrices are normalized). JISAE-O1 is similar to the implementation of the MOCSS method, applying an element-wise penalty then taking the overall average. This penalty was chosen for consistency with comparing to the MOCSS method. The orthogonal loss for JISAE-O1 is defined in (4).</p>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/2f01d7023107/d33e552.gif" loading="lazy" id="d33e552" alt="graphic file with name d33e552.gif"></td>
<td class="label">4</td>
</tr></table>
<p>JISAE-O2 directly calculates the orthogonal loss between the embeddings for each sample, aiming to keep the information learned from the samples orthogonal between joint and individual embeddings [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. JISAE-O3 calculates the orthogonal loss for each component in the embeddings, meaning information from the features should not be redundant between the joint and individual embeddings. This constraint is similar to the one used by the JIVE method. With the same notation for embeddings, the orthogonal loss functions for JISAE-O2 and JISAE-O3 are defined in (5) and (6) respectively:</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/7049f47d7bc2/d33e562.gif" loading="lazy" id="d33e562" alt="graphic file with name d33e562.gif"></td>
<td class="label">5</td>
</tr></table>
<table class="disp-formula p" id="Equ6"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/3877790df71c/d33e568.gif" loading="lazy" id="d33e568" alt="graphic file with name d33e568.gif"></td>
<td class="label">6</td>
</tr></table>
<p>The rows in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/babcb219508d/d33e575.gif" loading="lazy" id="d33e575" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/6e2e51cd5f10/d33e582.gif" loading="lazy" id="d33e582" alt="Inline graphic"></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/06d22ca04533/d33e588.gif" loading="lazy" id="d33e588" alt="Inline graphic"></span> represent individual subjects in a given batch of training examples. The columns from each matrix represent elements of features from the low dimensional representation for the shared and individual input respectively. Therefore the orthogonal loss defined for JISAE-O2 (see equation 5) imposes the orthogonality constraints among shared and specific embedding for each subject (taking the squared sum of the diagonal from the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/58deb7d0195d/d33e594.gif" loading="lazy" id="d33e594" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/957f3ecab0cb/d33e600.gif" loading="lazy" id="d33e600" alt="Inline graphic"></span> respectively or squared dot product between shared and specific embeddings for each subject) whereas the orthogonal loss in JISAE-O3 (see equation 6) imposes the orthogonality constraints among shared and specific embedding across all subjects in a given batch of training examples for every feature in the low dimensional representation. Equations (4) and (5) both calculate the dot product between shared and specific embeddings for a given subject. When taking the average, equation (4) also normalize the dimension of embeddings, while equation (5) will scale with the dimension of embeddings. Therefore, the total loss for the MOCSS and JISAE-O models is defined as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/9d1bfd559b2e/d33e606.gif" loading="lazy" id="d33e606" alt="Inline graphic"></span>. For a fair comparison, we also kept the original contrastive loss for MOCSS model. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/95c7c0c413ff/d33e612.gif" loading="lazy" id="d33e612" alt="Inline graphic"></span> is a hyperparameter that controls the relative contribution of the orthogonal loss to the total loss in these models. The total loss for other AE models is simply <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/6e97862cd685/d33e619.gif" loading="lazy" id="d33e619" alt="Inline graphic"></span>. With the total loss function defined above, the gradient-based optimization algorithm was used to reduce the loss by iteratively optimizing the model weights.</p>
<p id="Par13">The R package r.JIVE [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>] was used to compare the AE methods with JIVE. All AE models in this study were created using the Pytorch framework 1.13.1 [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>] with CUDA Toolkit 11.4. The KFold from sklearn 1.0.2 [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>] was used to split the dataset randomly for cross-validation (CV). The Optuna package (https://github.com/optuna/optuna) was used for hyperparameter searching based on the average reconstruction loss on the validation set during the 5-fold cross-validation.</p>
<p id="Par14">For the cancer dataset, multi-omics TCGA preprocessed datasets were downloaded from <a href="https://acgt.cs.tau.ac.il/multi_omic_benchmark/download.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://acgt.cs.tau.ac.il/multi_omic_benchmark/download.html</a> [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>], which includes gene expression, miRNA expression, and patient clinical data from 10 different cancers (AML: Acute Myeloid Leukemia, BIC: Breast cancer, COAD: Colon adenocarcinoma, GBM: Glioblastoma multiforme, KIRC: Kidney renal clear cell carcinoma, LIHC: Liver hepatocellular carcinoma, LUSC: Lung squamous cell carcinoma, SKCM: Skin Cutaneous Melanoma, OV: Ovarian serous cystadenocarcinoma and SARC: Sarcoma). Subjects were limited to only those with both gene expression and miRNA expression data. Since the number of measured features can be different across the cancer types, we selected the cancer types that have the most consistent feature dimensions across both omics sources. We also restricted our analysis to primary tumors only. After this filtering, 6 cancer types: BIC, LUSC, SKCM, LIHC, SARC, and KIRC were left for subsequent analysis. The resulting data set contained 1866 subjects with 20,531 genes and 1046 miRNAs. The resulting sample sizes for each cancer type were 621 BIC, 341 LUSC, 97 SKCM, 367 LIHC, 257 SARC, and 183 KIRC, as shown in Fig. <a href="#Fig2" class="usa-link">2</a>a.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12362917_12859_2025_6245_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/22cf7fcc8dd7/12859_2025_6245_Fig2_HTML.jpg" loading="lazy" id="MO2" height="258" width="709" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>TCGA dataset used for model training and testing in this study. <strong>a</strong> TCGA samples downloaded from <a href="https://acgt.cs.tau.ac.il/multi_omic_benchmark/download.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://acgt.cs.tau.ac.il/multi_omic_benchmark/download.html</a> with gene and miRNA expression data available; <strong>b</strong> Primary tumor samples were used for training and testing</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Simulation</h3>
<p id="Par15">To compare the effectiveness of multi-omics data integration with different models, we benchmarked different AE performance first on a simulation dataset generated by using R package MOSim [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>] and further validated on a dataset based on TCGA. For the simulation, RNA-seq (with a total 20,531 features) and miRNA-seq (with a total 1,046 features) were jointly generated from a series of settings. Specifically, data of 2, 3, 4, and 5 groups were generated separately and for each group setting, 20%, 40%, 60%, 80% and 100% of the genes were set to be differentially expressed. For the miRNA data, 50% of the miRNAs were set to be activators and 50% were set to be repressors. Sequencing depth was set to 10 for all features, and the replicate variability (replicateParams) was set to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/4f46794cd95a/d33e767.gif" loading="lazy" id="d33e767" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/9e4c6313df1c/d33e773.gif" loading="lazy" id="d33e773" alt="Inline graphic"></span>. All other parameters used the defaults from the MOSim package. Thus a total of 20 different dataset were simulated representing different level of difficulty to test the model and the total number of samples were kept to 600 in each dataset as shown in Table <a href="#Tab1" class="usa-link">1</a>.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>The simulated datasets used for model training and testing in this study were generated from 20 different settings with increasing level of difficulties from top to bottom and from right to left. (% DEG: percentage of differentially expressed genes)</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Group / % DEG</th>
<th align="left" colspan="1" rowspan="1">20%</th>
<th align="left" colspan="1" rowspan="1">40%</th>
<th align="left" colspan="1" rowspan="1">60%</th>
<th align="left" colspan="1" rowspan="1">80%</th>
<th align="left" colspan="1" rowspan="1">100%</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">300 samples/group</td>
<td align="left" colspan="1" rowspan="1">300 samples/group</td>
<td align="left" colspan="1" rowspan="1">300 samples/group</td>
<td align="left" colspan="1" rowspan="1">300 samples/group</td>
<td align="left" colspan="1" rowspan="1">300 samples/group</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">200 samples/group</td>
<td align="left" colspan="1" rowspan="1">200 samples/group</td>
<td align="left" colspan="1" rowspan="1">200 samples/group</td>
<td align="left" colspan="1" rowspan="1">200 samples/group</td>
<td align="left" colspan="1" rowspan="1">200 samples/group</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">150 samples/group</td>
<td align="left" colspan="1" rowspan="1">150 samples/group</td>
<td align="left" colspan="1" rowspan="1">150 samples/group</td>
<td align="left" colspan="1" rowspan="1">150 samples/group</td>
<td align="left" colspan="1" rowspan="1">150 samples/group</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">120 samples/group</td>
<td align="left" colspan="1" rowspan="1">120 samples/group</td>
<td align="left" colspan="1" rowspan="1">120 samples/group</td>
<td align="left" colspan="1" rowspan="1">120 samples/group</td>
<td align="left" colspan="1" rowspan="1">120 samples/group</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec5"><h3 class="pmc_sec_title">Hyperparameter tuning for AE models</h3>
<p id="Par16">To choose the optimal hyperparameters for different AE models, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq53"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/4981a5826698/d33e804.gif" loading="lazy" id="d33e804" alt="Inline graphic"></span> of the data was allocated for training while the remaining <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq54"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/33f140f063ec/d33e810.gif" loading="lazy" id="d33e810" alt="Inline graphic"></span> was used for final model evaluation as shown in Fig. <a href="#Fig3" class="usa-link">3</a>a. 5-fold cross-validation (CV) was used on the training set to choose the optimal hyperparameters for each AE model based on the lowest average reconstruction loss on the validation set. It is computationally infeasible to cover the entire hyperparameter space, so Optuna (<a href="https://github.com/optuna/optuna" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/optuna/optuna</a>), a hyperparameter optimization framework for hyperparameter searching, was used by conducting 50 random trails for each model in the simulation setting and 100 random trials for cancer dataset, where each trial corresponds to a different combination of hyperparameters. The selected optimal hyperparameters were then used to retrain the model on the entire training set in order to extract the embeddings for subsequent analysis. For the cancer data, after excluding solid tissue normal and metastatic samples, as shown in Fig. <a href="#Fig2" class="usa-link">2</a>a, the primary tumor samples were divided into non-overlapping training and testing sets following the same partition rule as shown in Fig. <a href="#Fig2" class="usa-link">2</a>b with the same training strategy shown in Fig. <a href="#Fig3" class="usa-link">3</a>a.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/aea9ad8ee278/12859_2025_6245_Fig3_HTML.jpg" loading="lazy" id="MO3" height="624" width="784" alt="Fig. 3"></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Model training and comparison process in this study</p></figcaption></figure><p id="Par17">The number of nodes for all hidden layers across for AE model was chosen from <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq55"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/ed0f30f03ea8/d33e835.gif" loading="lazy" id="d33e835" alt="Inline graphic"></span>. Except for the MOCSS model, batch normalization followed by dropout layers was included in all other AE models, and the dropout rate was selected from <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq56"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/3dbc8e9398b8/d33e841.gif" loading="lazy" id="d33e841" alt="Inline graphic"></span>. The MOCSS model is symmetric where the number of nodes across the encoder mirrors the number of the nodes across the decoder. In terms of nonlinear activation functions, instead of ReLu, hyperbolic tangent activation was used in the MOCSS model. The learning rate was chosen from <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq57"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/78ab87ec6305/d33e847.gif" loading="lazy" id="d33e847" alt="Inline graphic"></span> to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq58"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/58891c1095b7/d33e853.gif" loading="lazy" id="d33e853" alt="Inline graphic"></span> on a logarithmic scale; l2 penalty for model weights is chosen from <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq59"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/9beb3865bde4/d33e859.gif" loading="lazy" id="d33e859" alt="Inline graphic"></span> to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq60"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/78ab87ec6305/d33e866.gif" loading="lazy" id="d33e866" alt="Inline graphic"></span> on a logarithmic scale; number of training epoch is chosen from <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq61"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/6d89d774ad1a/d33e872.gif" loading="lazy" id="d33e872" alt="Inline graphic"></span>; batch size was chosen from <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq62"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/cd4e83df7471/d33e878.gif" loading="lazy" id="d33e878" alt="Inline graphic"></span> and the Adam optimizer was used for model weights optimization. Finally, for the three AE models with orthogonal loss constraints, as mentioned above, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq63"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/95c7c0c413ff/d33e884.gif" loading="lazy" id="d33e884" alt="Inline graphic"></span> was chosen from <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq64"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/ed3dfb0366b9/d33e890.gif" loading="lazy" id="d33e890" alt="Inline graphic"></span> to control the relative contribution of the orthogonal loss to the total loss.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Model comparison metrics</h3>
<p id="Par18">To assess whether the AE models can compress the data and retain the essential information for the data, reconstruction loss was calculated for the training and testing sets for each AE model except MOCSS. MOCSS has separate reconstruction loss for joint and individual embeddings, which is difficult to compare with other AE models. The reconstruction loss was based on the individual subject (contains two data modalities) in order to assess the uncertainty associated with average reconstruction loss. To investigate whether the omics data integration via AE can be useful for subsequent analysis, three classifiers were tested using the embeddings from the models. The first was a Gaussian naive Bayes classifier that did not require any additional hyperparameters. The other two classifiers were a random forest (RF) model and a support vector machine (SVM). All three classifiers were implemented using the sklearn package in Python with default parameters [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>].</p>
<p id="Par19">The classification was conducted with 5-fold CV in order to assess the classification uncertainty. This comparison was conducted for both simulation and cancer datasets. For the cancer dataset, we additionally compared the AE model with JIVE. The concatenated joint and individual components extracted from the JIVE model were also used for classification. To assess the effectiveness of AE and JIVE models, the original given features as well as their concatenation were also used as the baseline. The gene expression and miRNA expression data in the training and testing set were normalized by using the MinMaxScaler from the sklearn package [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>] separately. MinMaxScaler scaled the data set such that all feature values are in the range of [0, 1]. For a fair comparison, the input for all AE and JIVE models was also scaled by MinMaxScaler for training and testing sets separately.</p></section></section><section id="Sec7"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec8"><h3 class="pmc_sec_title">Simulation study</h3>
<p id="Par20">Data sets with different levels of difficulty were simulated from 20 settings. 600 samples were simulated from each setting, with 2, 3, 4, or 5 groups containing 300, 200, 150 or 120 samples respectively, as shown in Table <a href="#Tab1" class="usa-link">1</a>. The total number of features for gene and miRNA expression in the simulation data was chosen to match the total number of features in original TCGA dataset (20,531 genes and 1046 miRNAs).</p>
<p id="Par21">The TCGA dataset contained 685 BIC, 221 COAD, 344 LUSC, 291 OV, 450 SKCM, 410 LIHC, 261 SARC, 170 AML, 274 GBM, and 208 KIRC subjects that had both omics measurements. In the gene expression data across these 10 cancer types, GBM contained expression measurements for a total of 12,042 genes while the rest of the cancer types all contained the expression level measurements of 20,531 genes. In the miRNA expression data, six cancer types, BIC, LUSC, SKCM, LIHC, SARC, and KIRC, had expression measurements of 1,046 miRNAs. Three cancer types, COAD, OV, and AML, had the expression measurements of 705 miRNAs, and GBM contained expression measurements of 534 miRNAs. To make sure each omics data had the same number of features across the selected cancer types, 6 cancer types including BIC, LUSC, SKCM, LIHC, SARC, and KIRC were selected for subsequent analysis. Some of these cancer types include multiple sample types (e.g., primary tumor, solid tissue normal, and metastatic samples), but we restricted our analysis to primary tumors only (i.e., a total of 1,866 subjects) with 621, 341, 97, 367, 257, and 183 for breast, lung, melanoma, liver, sarcoma and kidney tumor samples respectively, as shown in Fig. <a href="#Fig2" class="usa-link">2</a>a.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Comparison of model performance</h3>
<p id="Par22">The optimized hyperparameters based on the lowest average reconstruction loss were used to retrain all the AE models on the entire training set separately (see Figs. <a href="#MOESM1" class="usa-link">S1</a> and <a href="#MOESM1" class="usa-link">S2</a> in supplementary material). To evaluate the generalization of AE models for data reconstruction, the reconstruction loss on the training and testing set was calculated for all AE models other than MOCSS based on the retrained model weights. This calculation is based on the reconstruction error for each individual subject including both data modalities (i.e., miRNA and gene expression). The average reconstruction loss and the corresponding standard deviation on the simulation training dataset is shown in Fig. <a href="#MOESM1" class="usa-link">S3</a>a while the reconstruction results on the testing dataset is shown in Fig. <a href="#Fig4" class="usa-link">4</a>. As expected, as the difficulty of the simulation settings increased, the averaged reconstruction loss also increased. The extracted embeddings were used as the new features in a Gaussian naive Bayes classifier for classification task with 5-fold CV. As shown in Figs. <a href="#MOESM1" class="usa-link">S3</a>b and <a href="#MOESM1" class="usa-link">S4</a> for the simulation training and testing dataset respectively, almost all the data points in both datasets were classified with accuracy of 100% (average accuracy with corresponding standard deviation shown in parentheses).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12362917_12859_2025_6245_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/da4664fbf013/12859_2025_6245_Fig4_HTML.jpg" loading="lazy" id="MO4" height="709" width="709" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Average reconstruction loss (with corresponding standard deviations showed in parentheses) for AE models on each of the simulation testing dataset</p></figcaption></figure><p id="Par23">For the TCGA cancer dataset, the optimal dimensions of embeddings and corresponding reconstruction loss for each AE model except MOCSS are shown in Table <a href="#Tab2" class="usa-link">2</a>. After hyperparameter selection, CNC_AE reduced the dimension the most from 21,577 (including 2,0531 genes and 1,046 miRNA sequences) to 32 embeddings, followed by JISAE-O3 with 192 embeddings. JISAE and JISAE-O1 achieved relatively lower training reconstruction loss on average compared to the other AE models. JISAE had the lowest testing reconstruction error, with X_AE close behind.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Reconstruction loss (average with standard deviation) from the different autoencoder methods for comparison of the TCGA data, based on the retrained model weights on the entire training set</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Method</th>
<th align="left" colspan="1" rowspan="1">Embedding dimension</th>
<th align="left" colspan="1" rowspan="1">Reconstruction loss<sup>a</sup>
</th>
<th align="left" colspan="1" rowspan="1">Reconstruction loss<sup>b</sup>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">CNC_AE</td>
<td align="left" colspan="1" rowspan="1">32</td>
<td align="left" colspan="1" rowspan="1">1.1705 (0.1356)</td>
<td align="left" colspan="1" rowspan="1">1.4164 (0.1113)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">X_AE</td>
<td align="left" colspan="1" rowspan="1">256</td>
<td align="left" colspan="1" rowspan="1">1.1602 (0.1379)</td>
<td align="left" colspan="1" rowspan="1">1.3998 (0.1106)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">MM_AE</td>
<td align="left" colspan="1" rowspan="1">512</td>
<td align="left" colspan="1" rowspan="1">1.1487 (0.1209)</td>
<td align="left" colspan="1" rowspan="1">1.4538 (0.1101)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">JISAE</td>
<td align="left" colspan="1" rowspan="1">1,536</td>
<td align="left" colspan="1" rowspan="1">1.0568 (0.1187)</td>
<td align="left" colspan="1" rowspan="1">1.382 (0.1104)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">JISAE-O1</td>
<td align="left" colspan="1" rowspan="1">3,072</td>
<td align="left" colspan="1" rowspan="1">0.9618 (0.1646)</td>
<td align="left" colspan="1" rowspan="1">1.5178 (0.1682)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">JISAE-O2</td>
<td align="left" colspan="1" rowspan="1">3,072</td>
<td align="left" colspan="1" rowspan="1">1.113 (0.1564)</td>
<td align="left" colspan="1" rowspan="1">1.6055 (0.1437)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">JISAE-O3</td>
<td align="left" colspan="1" rowspan="1">192</td>
<td align="left" colspan="1" rowspan="1">1.1939 (0.1376)</td>
<td align="left" colspan="1" rowspan="1">1.4464 (0.1316)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p38"><p><sup>a</sup>Calculation based on the training set that contains n = 1494 subjects; <sup>b</sup>Calculation based on the testing set that contains n = 372 subjects</p></div></div></section><p id="Par24">For classification, the model performance was also compared with JIVE. Using the permutation test in the r.jive package [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>], we selected a rank of 6 for the joint matrix and ranks of 160 and 100 for individual matrices. The score matrices for joint and individual components were then extracted with PCA as mentioned earlier. On the testing set, by directly applying the loadings from the training set to the testing set, we obtained the corresponding joint and individual score matrices with the same ranks as the training set.</p>
<p id="Par25">To evaluate the effectiveness of the embeddings from the AE models and the joint and individual components from JIVE model on the TCGA cancer dataset in the subsequent analysis, three different classifiers were used for cancer type classification: a Gaussian naive Bayes classifier, an RF classifier, and an SVM. We performed a 5-fold CV and reported the average classification accuracy and also showed the corresponding standard deviations on both the training and testing sets. The inputs for the classifiers were either the original data (gene expression, methylation, or both), the embeddings from each of the AE models, or the JIVE components, as illustrated in Fig. <a href="#Fig3" class="usa-link">3</a>a. As shown in Fig. <a href="#Fig3" class="usa-link">3</a>b, the joint and individual components from the JIVE model were concatenated together for classification.</p>
<p id="Par26">Training classification results are shown in supplementary figures (Figs. <a href="#MOESM1" class="usa-link">S5</a>, <a href="#MOESM1" class="usa-link">S6</a>, <a href="#MOESM1" class="usa-link">S7</a>). For the naive Bayes classifier, the MM_AE model resulted in the highest average training classification accuracy (95.18%), followed by JISAE (93.51%), JISAE-O1 (92.77%) and JISAE-O2 (91.68%). The JIVE model produced the lowest average training classification accuracy (67.2%), which was even lower than the original features. MM_AE also produced the highest average training classification accuracy for the RF and SVM classifiers among the dimension reduction methods, but these were lower than the original feature sets.</p>
<p id="Par27">For each AE model, by applying the trained model weights from the entire training set to the testing set, we obtained the embeddings for the testing set. The classification accuracy using the naive Bayes classifier on the testing set is shown in Fig. <a href="#Fig5" class="usa-link">5</a>. The original features generated much lower classification accuracy (all around 58%) compared with AE and JIVE models. While the CNC_AE model achieved 87% average classification accuracy on the training set, it achieved the highest average accuracy of 95.7% on the testing set. In comparison, JISAE-O1 and JISAE-O3 achieved similar performance (around 91%) on the testing sets. The joint and individual components from the JIVE model produced a classification accuracy of 85.74%, despite having considerably lower training accuracy.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12362917_12859_2025_6245_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/df5a04f7b184/12859_2025_6245_Fig5_HTML.jpg" loading="lazy" id="MO5" height="709" width="708" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>5-fold CV accuracy (average with corresponding standard deviation based on 5-fold CV) using original features or model embeddings as inputs for the naive Bayes classifier to classify the 6 different types of cancers on the testing set</p></figcaption></figure><p id="Par28">For the RF and SVM models, the original features produced higher classification accuracies than the dimension reduction methods. This is not entirely surprising, since RF and SVM are supervised methods, while the AE models and JIVE are all unsupervised dimension reduction methods. The performance of all of the dimension reduction methods was close to the original features for the RF classifier (Fig. <a href="#Fig6" class="usa-link">6</a>). JISAE-O1 had the highest performance among the AE models (97.0%), which was still lower than the testing classification accuracy for full feature set (98.9%). For the SVM classifier, JISAE with no orthogonality penalty performed as well as the full feature set (97.3%). However, for the SVM classifier, JISAE-O2 (72.9%) and JISAE-O3 (33.6%) performed very poorly (Fig. <a href="#Fig7" class="usa-link">7</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12362917_12859_2025_6245_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/ef66a9c432d9/12859_2025_6245_Fig6_HTML.jpg" loading="lazy" id="MO6" height="709" width="708" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>5-fold CV accuracy (average with corresponding standard deviation based on 5-fold CV) using original features or model embeddings as inputs for the random forest classifier to classify the 6 different types of cancers on the testing set</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12362917_12859_2025_6245_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1c88/12362917/989f898a8c5e/12859_2025_6245_Fig7_HTML.jpg" loading="lazy" id="MO7" height="709" width="708" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>5-fold CV accuracy (average with corresponding standard deviation based on 5-fold CV) using original features or model embeddings as inputs for the support vector machine to classify the 6 different types of cancers on the testing set</p></figcaption></figure></section></section><section id="Sec10"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par29">In this study, we proposed a flexible new AE model structure for data integration in multi-omics data. Compared to previous integrative AE models, the proposed models use both shared and specific inputs and allow flexibility to incorporate orthogonality in the loss function. We compared the reconstruction loss and classification accuracy of the novel model structures to four existing integrative autoencoders: CNC_AE, X_AE, MM_AE, and MOCSS [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Although we focused on the setting with two data sources in this manuscript, the proposed methods should easily extend to three or more data sources by simply adding additional inputs into the AE.</p>
<p id="Par30">Among the simulated datasets, the most difficult setting for the models contained 5 groups with 20% differentially expressed genes (DEGs). In this setting, X_AE and MM_AE showed slightly higher reconstruction loss on both training and testing set compared with other AE models. Despite the slightly difference in reconstruction loss among different AE models, the embeddings extracted from AE models are all informative in terms of classification of different groups (see Fig. <a href="#MOESM1" class="usa-link">S4</a>).</p>
<p id="Par31">On the TCGA application, JISAE-O1 achieved the lowest training reconstruction loss among all AE models, indicating the effectiveness of this AE model in reconstructing the model input with the learned embeddings. More importantly, the JISAE model achieved the lowest reconstruction loss on the testing set. The models with an orthogonality penalty performed worse on the testing set, indicating possible overfitting. Table <a href="#Tab2" class="usa-link">2</a> shows that the test reconstruction loss is considerably higher than the training reconstruction loss for JISAE-O1. This suggests possible overfitting that is not as severe without an orthogonality penalty. The orthogonality penalty increases the model complexity, as it prevents information from overlapping between the joint and individual components. So it is more likely to overfit the data using the orthogonality penalties. Based on the optimal embedding dimensions, CNC_AE reduced the dimension of the input the most. Overall, the different AE architectures constructed in this study showed slightly different capacities to capture the essential input information and generalize that to the new data.</p>
<p id="Par32">Comparing the model performance in terms of reconstruction loss derived from the embeddings and subsequent classification tasks with these embeddings, it is clear that the low reconstruction loss is not directly translatable to subsequent higher model performance. For instance, CNC_AE is a simple structure, that has the lowest dimension of embeddings and is performing the best for testing set classification, nevertheless, the reconstruction loss for CNC_AE is not the lowest one among all AE models constructed in this study. Although the highest average accuracy of 95.7% on the testing set was achieved by the CNC_AE model, JISAE-O1 achieved consistently high model performance on both training (92.77%) and testing set (91.94%) indicating the greater robustness of this model. Although not as highly accurate as the JISAE-O1, the MOCSS model also demonstrated consistently higher model performance between the training and testing set (86.21% on the training set and 85.73% on the testing set). The fact that the same orthogonal loss function was used in JISAE-O1 and MOCSS models (see equations (3) and (4) respectively) indicates that this orthogonal loss might be the optimal one compared to the other 2 orthogonal loss (see equations (5) and (6)) defined in this study for the classification tasks.</p>
<p id="Par33">In theory, deep-learning-based multi-omics data integration methods such as AEs can have more flexible architectures with the possibility of designing specific structures for processing each data source. This flexibility and relatively higher model performance in this study demonstrated its potential superiority over the JIVE model for data integration. Although all AE models considered in this study achieved similarly high model performance on the simulation dataset in terms of reconstruction loss and classification accuracy, with the real-world TCGA dataset, the novel AE model structure coupled with orthogonal penalties proposed in this study showed relatively higher model performance compared with previous MOCSS model (see Fig. <a href="#Fig5" class="usa-link">5</a>).</p>
<p id="Par34">There are some limitations to the current study. Although 5-fold CV was used to select the optimal hyperparameters for each AE model, L2 loss and dropout layers were also implemented in the models, a relatively large discrepancy still appears between the classification accuracy on the training and testing set for some of the AE models with the TCGA data. To mitigate this, more data could be collected for training and testing in the future, and variational AE models that contain intrinsic model regularization can be used to replace the static AE structures in this study. The AE methods introduced in this analysis are unsupervised. This was clearest for the RF and SVM classifiers, where supervised classifiers for the full feature set outperformed the AE models. For classification, supervised neural networks that use the classes directly as outputs could perform even better. Alternatively, in addition to the reconstruction loss, a loss function that is directly related to the subsequent analysis can be added to the model training process (e.g., classification error), so the extracted embeddings are also directed towards better performance in that context. Finally, although the feature dimension of the simulated data was the same as the original TCGA data, the sample size for the simulation study was smaller for easier computation. On a larger sample, it is possible that the performances of each of the methods would be different.</p></section><section id="Sec12"><h2 class="pmc_sec_title">Additional file</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12362917/bin/12859_2025_6245_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file 1 (pdf 624 KB)</a><sup> (624.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12362917/bin/12859_2025_6245_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file 2 (xlsx 37 KB)</a><sup> (37.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12362917/bin/12859_2025_6245_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file 3 (xlsx 12 KB)</a><sup> (12.5KB, xlsx) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>C.W. and M.O. conceived the project. M.O. supervised the project. C.W. and M.O. developed the methodology. C.W. performed the analysis and validation. C.W. and M.O. wrote and approved the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This project was supported by the Miami University start-up fund (to MO).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All data used in this project comes from The Cancer Genome Atlas and was obtained from this site:<a href="https://acgt.cs.tau.ac.il/multi_omic_benchmark/download.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://acgt.cs.tau.ac.il/multi_omic_benchmark/download.html</a>. All code for this project, along with the simulated data that was used, can be found at the following Github link:<a href="https://github.com/wangc90/AE_Data_Integration" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/wangc90/AE_Data_Integration</a>.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par35">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par36">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par37">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher's Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="sec15"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>The online version contains supplementary material available at 10.1186/s12859-025-06245-7.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Athieniti E, Spyrou GM. A guide to multi-omics data collection and integration for translational medicine. Comput Struct Biotechnol J. 2023;21:134–49.
</cite> [<a href="https://doi.org/10.1016/j.csbj.2022.11.050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9747357/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36544480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Athieniti%20E,%20Spyrou%20GM.%20A%20guide%20to%20multi-omics%20data%20collection%20and%20integration%20for%20translational%20medicine.%20Comput%20Struct%20Biotechnol%20J.%202023;21:134%E2%80%9349." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Chen Y, Wen Y, Xie C, Chen X, He S, Bo X, Zhang Z (2023) Mocss: Multi-omics data clustering and cancer subtyping via shared and specific representation learning. Iscience, 26(8)</cite> [<a href="https://doi.org/10.1016/j.isci.2023.107378" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10407241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37559907/" class="usa-link">PubMed</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Feng Q, Jiang M, Hannig J, Marron J. Angle-based joint and individual variation explained. J Multivar Anal. 2018;166:241–65.</cite> [<a href="https://scholar.google.com/scholar_lookup?Feng%20Q,%20Jiang%20M,%20Hannig%20J,%20Marron%20J.%20Angle-based%20joint%20and%20individual%20variation%20explained.%20J%20Multivar%20Anal.%202018;166:241%E2%80%9365." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Lock EF, Hoadley KA, Marron JS, Nobel AB. Joint and individual variation explained (jive) for integrated analysis of multiple data types. Ann Appl Stat. 2013;7(1):523.
</cite> [<a href="https://doi.org/10.1214/12-AOAS597" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3671601/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23745156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lock%20EF,%20Hoadley%20KA,%20Marron%20JS,%20Nobel%20AB.%20Joint%20and%20individual%20variation%20explained%20(jive)%20for%20integrated%20analysis%20of%20multiple%20data%20types.%20Ann%20Appl%20Stat.%202013;7(1):523." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Monzó C, Martínez-Mira C, Arzalluz-Luque Á, Conesa A, Tarazona S (2018) Mosim: bulk and single-cell multi-layer regulatory network simulator. bioRxiv, 421834</cite> [<a href="https://doi.org/10.1093/bib/bbaf110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11926980/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40116657/" class="usa-link">PubMed</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>O’Connell MJ, Lock EF. R. jive for exploration of multi-source molecular data. Bioinformatics. 2016;32(18):2877–9.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btw324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6090891/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27273669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?O%E2%80%99Connell%20MJ,%20Lock%20EF.%20R.%20jive%20for%20exploration%20of%20multi-source%20molecular%20data.%20Bioinformatics.%202016;32(18):2877%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Paszke A, Gross S, Massa F, Lerer A, Bradbury J, Chanan G, Killeen T, Lin Z, Gimelshein N, Antiga L, et al. (2019) Pytorch: An imperative style, high-performance deep learning library. Adv Neural Inf Process Syst, 32</cite>
</li>
<li id="CR8">
<span class="label">8.</span><cite>Pedregosa F, Varoquaux G, Gramfort A, Michel V, Thirion B, Grisel O, Blondel M, Prettenhofer P, Weiss R, Dubourg V, et al. Scikit-learn: machine learning in python. J Mach Learn Res. 2011;12:2825–30.</cite> [<a href="https://scholar.google.com/scholar_lookup?Pedregosa%20F,%20Varoquaux%20G,%20Gramfort%20A,%20Michel%20V,%20Thirion%20B,%20Grisel%20O,%20Blondel%20M,%20Prettenhofer%20P,%20Weiss%20R,%20Dubourg%20V,%20et%20al.%20Scikit-learn:%20machine%20learning%20in%20python.%20J%20Mach%20Learn%20Res.%202011;12:2825%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Rappoport N, Shamir R. Multi-omic and multi-view clustering algorithms: review and cancer benchmark. Nucleic Acids Res. 2018;46(20):10546–62.
</cite> [<a href="https://doi.org/10.1093/nar/gky889" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6237755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30295871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rappoport%20N,%20Shamir%20R.%20Multi-omic%20and%20multi-view%20clustering%20algorithms:%20review%20and%20cancer%20benchmark.%20Nucleic%20Acids%20Res.%202018;46(20):10546%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Simidjievski N, Bodnar C, Tariq I, Scherer P, Andres Terre H, Shams Z, Jamnik M, Liò P. Variational autoencoders for cancer data integration: design principles and computational practice. Front Genet. 2019;10:1205.
</cite> [<a href="https://doi.org/10.3389/fgene.2019.01205" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6917668/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31921281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Simidjievski%20N,%20Bodnar%20C,%20Tariq%20I,%20Scherer%20P,%20Andres%20Terre%20H,%20Shams%20Z,%20Jamnik%20M,%20Li%C3%B2%20P.%20Variational%20autoencoders%20for%20cancer%20data%20integration:%20design%20principles%20and%20computational%20practice.%20Front%20Genet.%202019;10:1205." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Stahlschmidt SR, Ulfenborg B, Synnergren J. Multimodal deep learning for biomedical data fusion: a review. Brief Bioinform. 2022;23(2):569.</cite> [<a href="https://doi.org/10.1093/bib/bbab569" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8921642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35089332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stahlschmidt%20SR,%20Ulfenborg%20B,%20Synnergren%20J.%20Multimodal%20deep%20learning%20for%20biomedical%20data%20fusion:%20a%20review.%20Brief%20Bioinform.%202022;23(2):569." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>The Cancer Genome Atlas Research Network. Comprehensive genomic characterization defines human glioblastoma genes and core pathways. Nature. 2008;455(7216):1061–8.
</cite> [<a href="https://doi.org/10.1038/nature07385" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2671642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18772890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?The%20Cancer%20Genome%20Atlas%20Research%20Network.%20Comprehensive%20genomic%20characterization%20defines%20human%20glioblastoma%20genes%20and%20core%20pathways.%20Nature.%202008;455(7216):1061%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Yang Z, Michailidis G. A non-negative matrix factorization method for detecting modules in heterogeneous omics multi-modal data. Bioinformatics. 2016;32(1):1–8.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv544" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5006236/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26377073/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20Z,%20Michailidis%20G.%20A%20non-negative%20matrix%20factorization%20method%20for%20detecting%20modules%20in%20heterogeneous%20omics%20multi-modal%20data.%20Bioinformatics.%202016;32(1):1%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Zhou G, Cichocki A, Zhang Y, Mandic DP. Group component analysis for multiblock data: common and individual feature extraction. IEEE Trans Neural Netw Learn Syst. 2015;27(11):2426–39.
</cite> [<a href="https://doi.org/10.1109/TNNLS.2015.2487364" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26529787/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou%20G,%20Cichocki%20A,%20Zhang%20Y,%20Mandic%20DP.%20Group%20component%20analysis%20for%20multiblock%20data:%20common%20and%20individual%20feature%20extraction.%20IEEE%20Trans%20Neural%20Netw%20Learn%20Syst.%202015;27(11):2426%E2%80%9339." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Zhang S, Liu C-C, Li W, Shen H, Laird PW, Zhou XJ. Discovery of multi-dimensional modules by integrative analysis of cancer genomic data. Nucleic Acids Res. 2012;40(19):9379–91.
</cite> [<a href="https://doi.org/10.1093/nar/gks725" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3479191/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22879375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20S,%20Liu%20C-C,%20Li%20W,%20Shen%20H,%20Laird%20PW,%20Zhou%20XJ.%20Discovery%20of%20multi-dimensional%20modules%20by%20integrative%20analysis%20of%20cancer%20genomic%20data.%20Nucleic%20Acids%20Res.%202012;40(19):9379%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Zhang X, Zhang J, Sun K, Yang X, Dai C, Guo Y (2019) Integrated multi-omics analysis using variational autoencoders: application to pan-cancer classification. In: 2019 IEEE International Conference on Bioinformatics and Biomedicine (BIBM), 765–769 IEEE</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12362917/bin/12859_2025_6245_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file 1 (pdf 624 KB)</a><sup> (624.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12362917/bin/12859_2025_6245_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file 2 (xlsx 37 KB)</a><sup> (37.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12362917/bin/12859_2025_6245_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary file 3 (xlsx 12 KB)</a><sup> (12.5KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data used in this project comes from The Cancer Genome Atlas and was obtained from this site:<a href="https://acgt.cs.tau.ac.il/multi_omic_benchmark/download.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://acgt.cs.tau.ac.il/multi_omic_benchmark/download.html</a>. All code for this project, along with the simulated data that was used, can be found at the following Github link:<a href="https://github.com/wangc90/AE_Data_Integration" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/wangc90/AE_Data_Integration</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Bioinformatics are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12859-025-06245-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12859_2025_Article_6245.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12362917/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12362917/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12362917%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362917/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12362917/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12362917/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830833/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12362917/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830833/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12362917/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12362917/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="FmL77ckXEg6W9wx4MhP5nemoeMauIZ6SiJOByS3OAaxBITt4GRoAuCKEduwgBPkx">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
