
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            A spatial single-cell atlas of the claustro-insular region uncovers key regulators of neuronal identity and excitability - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E485B88AEF32F30585B800457BA97E.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373765/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="A spatial single-cell atlas of the claustro-insular region uncovers key regulators of neuronal identity and excitability">
<meta name="citation_author" content="Leon Fodoulian">
<meta name="citation_author_institution" content="Department of Genetics and Evolution, Faculty of Sciences, University of Geneva, Geneva, Switzerland">
<meta name="citation_author_institution" content="Department of Basic Neurosciences, Faculty of Medicine, University of Geneva, Geneva, Switzerland">
<meta name="citation_author" content="Madlaina Boillat">
<meta name="citation_author_institution" content="Department of Genetics and Evolution, Faculty of Sciences, University of Geneva, Geneva, Switzerland">
<meta name="citation_author_institution" content="Department of Basic Neurosciences, Faculty of Medicine, University of Geneva, Geneva, Switzerland">
<meta name="citation_author" content="Marie Moulinier">
<meta name="citation_author_institution" content="Department of Genetics and Evolution, Faculty of Sciences, University of Geneva, Geneva, Switzerland">
<meta name="citation_author_institution" content="Department of Basic Neurosciences, Faculty of Medicine, University of Geneva, Geneva, Switzerland">
<meta name="citation_author" content="Alan Carleton">
<meta name="citation_author_institution" content="Department of Basic Neurosciences, Faculty of Medicine, University of Geneva, Geneva, Switzerland">
<meta name="citation_author" content="Ivan Rodriguez">
<meta name="citation_author_institution" content="Department of Genetics and Evolution, Faculty of Sciences, University of Geneva, Geneva, Switzerland">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7830">
<meta name="citation_doi" content="10.1038/s41467-025-63138-2">
<meta name="citation_pmid" content="40846807">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373765/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373765/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373765/pdf/41467_2025_Article_63138.pdf">
<meta name="description" content="The claustro-insular region is an evolutionarily conserved and extensively interconnected brain area, critical for functions such as attention, cognitive flexibility, interoception, and affective processing. Despite its importance, its cellular ...">
<meta name="og:title" content="A spatial single-cell atlas of the claustro-insular region uncovers key regulators of neuronal identity and excitability">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The claustro-insular region is an evolutionarily conserved and extensively interconnected brain area, critical for functions such as attention, cognitive flexibility, interoception, and affective processing. Despite its importance, its cellular ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373765/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373765">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63138-2"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63138.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373765%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373765/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373765/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373765/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 22;16:7830. doi: <a href="https://doi.org/10.1038/s41467-025-63138-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63138-2</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>A spatial single-cell atlas of the claustro-insular region uncovers key regulators of neuronal identity and excitability</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fodoulian%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Leon Fodoulian</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Leon Fodoulian</span></h3>
<div class="p">
<sup>1</sup>Department of Genetics and Evolution, Faculty of Sciences, University of Geneva, Geneva, Switzerland </div>
<div class="p">
<sup>2</sup>Department of Basic Neurosciences, Faculty of Medicine, University of Geneva, Geneva, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fodoulian%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Leon Fodoulian</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Boillat%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Madlaina Boillat</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Madlaina Boillat</span></h3>
<div class="p">
<sup>1</sup>Department of Genetics and Evolution, Faculty of Sciences, University of Geneva, Geneva, Switzerland </div>
<div class="p">
<sup>2</sup>Department of Basic Neurosciences, Faculty of Medicine, University of Geneva, Geneva, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Boillat%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Madlaina Boillat</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moulinier%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Marie Moulinier</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Marie Moulinier</span></h3>
<div class="p">
<sup>1</sup>Department of Genetics and Evolution, Faculty of Sciences, University of Geneva, Geneva, Switzerland </div>
<div class="p">
<sup>2</sup>Department of Basic Neurosciences, Faculty of Medicine, University of Geneva, Geneva, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moulinier%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marie Moulinier</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Carleton%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Alan Carleton</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Alan Carleton</span></h3>
<div class="p">
<sup>2</sup>Department of Basic Neurosciences, Faculty of Medicine, University of Geneva, Geneva, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Carleton%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alan Carleton</span></a>
</div>
</div>
<sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodriguez%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Ivan Rodriguez</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Ivan Rodriguez</span></h3>
<div class="p">
<sup>1</sup>Department of Genetics and Evolution, Faculty of Sciences, University of Geneva, Geneva, Switzerland </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodriguez%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ivan Rodriguez</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Genetics and Evolution, Faculty of Sciences, University of Geneva, Geneva, Switzerland </div>
<div id="Aff2">
<sup>2</sup>Department of Basic Neurosciences, Faculty of Medicine, University of Geneva, Geneva, Switzerland </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Nov 4; Accepted 2025 Aug 11; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373765  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40846807/" class="usa-link">40846807</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The claustro-insular region is an evolutionarily conserved and extensively interconnected brain area, critical for functions such as attention, cognitive flexibility, interoception, and affective processing. Despite its importance, its cellular composition and organization remain poorly characterized, hindering a comprehensive understanding of the mechanisms underlying its diverse functions. By combining single-cell RNA sequencing and spatial transcriptomics, we create a high-resolution atlas of this region in mice, uncovering distinct neuronal subtypes and unexpected complexity. Leveraging this atlas, we investigate the role of NR4A2, a neuropsychiatric risk factor expressed in several claustro-insular neuronal subtypes. In an <em>Nr4a2</em> haploinsufficiency model, we find that only claustrum neurons exhibited shifts in molecular identity. This identity shift, which involves the activation of a transcription factor cascade, is associated with alterations in neuronal firing activity. Our findings provide new insights into the cellular architecture of the claustro-insular region and highlights <em>Nr4a2</em> as a key regulator of its component’s identities.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Genetics of the nervous system, Excitability, RNA sequencing, Neurodevelopmental disorders</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Here the authors present a high-resolution atlas of the mouse claustro-insular region and reveal unexpected neuronal diversity. <em>Nr4a2</em> haploinsufficiency alters claustrum neuron identity and activity, implicating it as a key regulator in this brain area’s organization.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">The claustro-insular region of the mammalian brain is an evolutionarily conserved bilateral complex composed of multiple structures. The insula, located laterally, is a neocortical area involved in diverse functions, including interoception, salience detection, pain processing, and regulation of affective states<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. The claustrum (CLA), situated medially adjacent to the external capsule, is an intricate subcortical nucleus characterized by dense, reciprocal connections with nearly all cortical areas<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>–<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. This extensive connectivity suggests that the CLA plays a role in modulating cortical activity and may also contribute directly to specific cognitive functions. Consistent with this, a growing body of research has implicated the CLA in attention modulation, cognitive flexibility, sleep regulation, and working memory, among other processes<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Despite increasing interest, progress in understanding the CLA has been hindered by anatomical challenges, particularly in rodent species, where the CLA boundaries with the adjacent insula remain poorly outlined.</p>
<p id="Par4">Visual examination of spatial gene expression patterns in the mouse claustro-insular area reveals an unusually complex landscape of cellular composition and organization<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Several studies have, sometimes indirectly, profiled the gene expression of CLA projection neurons at single-cell resolution<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Most identified a single transcriptomically defined population; however, in rare instances, two distinct populations were reported<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>, which were suggested to correspond to a proposed subdivision of the CLA into a core and a shell<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. The observation of a relatively homogenous CLA cell population at the transcriptomic level is counterintuitive, especially considering that numerous CLA subpopulations have been defined based on their electrophysiological properties<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>–<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> or on their projection targets in the brain<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>–<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>, and that CLA neurons have been shown to have both inhibitory and excitatory effects in the cortex<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. The discrepancy between transcriptomic homogeneity and functional diversity underscores the need for deeper investigation into the cellular architecture of the CLA. A thorough characterization and spatial mapping of claustro-insular cell types are thus essential for further elucidating the complex roles of this brain region.</p>
<p id="Par5">Although anatomically separated, the CLA and the dorsal endopiriform nucleus (EP) have been suggested to be part of the same complex based on ontogenic and gene expression data<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Excitatory neurons in this claustrum/dorsal endopiriform complex (CLA/EP) are characterized by their expression of a set of canonical marker genes, including early and sustained transcription of <em>Nr4a2</em> (also called <em>Nurr1</em>), which codes for the NR4A2 transcription factor<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>–<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. In the dopaminergic system, NR4A2 is critical for determining neuronal identity<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>–<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> and has been linked to various neurodevelopmental diseases<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>–<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. The role played by <em>Nr4a2</em> in the development and function of the CLA/EP is still being explored<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>,<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>.</p>
<p id="Par6">To address gaps in our understanding of the identity and organization of cells in the claustro-insular region, we generated a high-resolution map of its cell types by combining single-cell RNA sequencing (scRNAseq) with spatial transcriptomics. Furthermore, given the early and persistent expression of <em>Nr4a2</em> in the CLA and surrounding cells, and its critical and established role in defining and maintaining cellular phenotypes in other brain regions, we investigated its function in determining the cellular identities of claustro-insular neurons.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Transcriptomic characterization of cell types in the claustro-insular region</h3>
<p id="Par7">To characterize the cellular composition and organization of the claustro-insular region, we performed a dual analysis combining scRNAseq and multiplexed error-robust fluorescence in situ hybridization (MERFISH) spatial transcriptomics (Fig. <a href="#Fig1" class="usa-link">1a</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>). First, we performed scRNAseq on isolated cells from claustro-insular microdissections, covering a substantial portion of the CLA/EP along the anteroposterior axis (bregma +1.545 to −1.355). We recovered a total of 67,226 cells, encompassing various neuronal and non-neuronal cell types (Supplementary Fig. <a href="#MOESM1" class="usa-link">1b–d</a>). To uncover the neuronal cell types populating the claustro-insular region, we filtered for high-quality neurons, clustered them, and identified their marker genes. This analysis revealed the presence of CLA projection neurons and known adjacent cortical and subcortical cell types, including deep-layer L5b, L6a and L6b cortical neurons, upper-layer L2/3 cortical neurons, piriform cortex neurons (pir), a heterogeneous group of interneurons (IN), and multiple subtypes of medium spiny neurons (MSN) of the striatum (Fig. <a href="#Fig1" class="usa-link">1b</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">1e–g</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. High-resolution characterization of cell types in the claustro-insular region.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373765_41467_2025_63138_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5df/12373765/e693b47b1c22/41467_2025_63138_Fig1_HTML.jpg" loading="lazy" id="d33e406" height="899" width="792" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic of the experimental design for high-resolution characterization of the claustro-insular region. <strong>b</strong> Visualization of neuronal cell types from the scRNAseq dataset on a Uniform Manifold Approximation and Projection (UMAP) plot. CLA claustrum projection neuron, dMSN striatal direct pathway medium spiny neuron, iMSN striatal indirect pathway medium spiny neuron, IN interneuron, L2/3 layer 2/3 glutamatergic neuron, L5b layer 5b glutamatergic neuron, L6a layer 6a glutamatergic neuron, L6b layer 6b glutamatergic neuron, mMSN mixed striatal medium spiny neuron, pir piriform glutamatergic neuron, shell shell projection neuron, sMSN striosome medium spiny neuron. n = 13 mice. Exact sample details are provided in the “Methods” section and in Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>. <strong>c</strong> Visualization of expression of cell type-specific marker genes on a representative coronal section of a mouse brain after MERFISH labeling and imaging. <strong>d</strong> Visualization of neuronal cell types from the MERFISH dataset on a UMAP plot. Cell type attributes of cells were obtained after mapping the MERFISH data onto the scRNAseq dataset. n = 7 mice. Exact sample details are provided in the “Methods” section and in Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>. <strong>e</strong> Violin plots showing the distribution of expression of cell type-specific marker genes in the scRNAseq dataset (normalized data are shown on a log<sub>10</sub> scale). <strong>f</strong> Visualization of the expression of cell type-specific marker genes on a representative coronal section of the claustro-insular region from the MERFISH dataset. Cells expressing at least 5 mRNA molecules of the corresponding gene are plotted. Colors indicate the cell type to which the genes are specific. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par8">Within the excitatory neuron population, we identified two subpopulations characterized by a canonical CLA transcriptional profile. These subtypes, named CLA 1 and CLA 2, express high levels of <em>Nr4a2, Gnb4</em>, <em>Oprk1</em>, <em>Lxn</em>, <em>Slc17a6</em>, and <em>Smim32</em> (a recently identified marker of the CLA)<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>, among other known CLA marker genes (Fig. <a href="#Fig1" class="usa-link">1e</a>). The CLA 2 subtype is further distinguished by the expression of immediate early genes such as <em>Fosl2</em> and <em>Egr2</em>. Additionally, we identified four subtypes of L6a neurons, all characterized by high expression levels of <em>Nfib</em>, <em>Sdk2</em>, and <em>Rprm</em>, with variable expression levels of <em>Nnat</em> and <em>Crym</em>, which distinguish the subtypes (Fig. <a href="#Fig1" class="usa-link">1b, e</a>).</p>
<p id="Par9">Beyond these neuronal types, we identified a large cluster of cells with a unique transcriptomic identity, intermediate between a CLA-like and a cortical identity. We refer to these as shell projection neurons (or “shell” for short) (Fig. <a href="#Fig1" class="usa-link">1b, e</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">1g</a>). Three subpopulations of shell neurons were identified, all characterized by low expression of <em>Nr4a2</em>, <em>Gnb4</em>, and <em>Oprk1</em>, high expression of <em>Nfib</em>, <em>Col6a1</em>, and <em>Nnat</em>, low but relatively specific expression of <em>Syt17</em>, sparse expression of <em>Npsr1</em>, and the absence of expression of <em>Slc17a6</em>, <em>Smim32</em>, or L6a markers like <em>Rprm</em> and <em>Crym</em> (Fig. <a href="#Fig1" class="usa-link">1e</a>). Among these subpopulations, shell 1 and shell 2 are distinguished by differing levels of <em>Nr4a2</em> expression, with shell 2 further characterized by <em>Npy</em> expression. The third subtype, shell 3, exhibits expression of immediate early genes, akin to CLA 2.</p>
<p id="Par10">To localize the identified cell types and subtypes within the claustro-insular region, particularly to resolve the spatial distribution of CLA and shell neurons within the CLA/EP, we designed a custom panel of 300 genes enriched for markers specific to the populations described above and performed MERFISH spatial transcriptomics (Fig. <a href="#Fig1" class="usa-link">1a, c</a>, Supplementary Data <a href="#MOESM3" class="usa-link">2</a>). Multiple coronal sections at bregma +1.045, as well as two more posterior sections (bregma +0.145 and −1.355) were imaged, revealing robust and specific expression of these cell type markers (Fig. <a href="#Fig1" class="usa-link">1c</a>). To assign cell types in the MERFISH dataset, we performed a reference-based mapping of the data onto the scRNAseq dataset<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. This strategy enabled the detection of most previously identified cell types and subtypes (Fig. <a href="#Fig1" class="usa-link">1d</a>). Many populations exhibited high prediction and mapping scores, indicating strong confidence in cell type assignments and good concordance between the MERFISH and scRNAseq datasets (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>). Visualization of cell type marker gene expression confirmed their spatial specificity across the claustro-insular region and underscored the complex organization and intermingling of diverse cell types within this brain region (Fig. <a href="#Fig1" class="usa-link">1f</a>).</p></section><section id="Sec4"><h3 class="pmc_sec_title">Generation of a map of the claustro-insular region</h3>
<p id="Par11">The complexity of gene expression patterns observed in MERFISH sections indicates that drawing discrete, non-overlapping anatomical boundaries is not an optimal way to represent the organization of the claustro-insular region. Instead, our goal was to create a map that captures this complexity while still providing insight into the relative localization of its components. To this end, we adopted a data-driven strategy by labeling tissue sections for gene expression, identifying cell types based on their gene expression profiles, and delineating contours that represent the 2-dimensional density distributions of these cell types across the sections. We used single-molecule fluorescence in situ hybridization (smFISH) to label multiple coronal sections spanning four anteroposterior levels of the CLA/EP, targeting five key marker genes. These markers enabled us to distinguish four major components of the claustro-insular region: <em>Nr4a2</em> for CLA, shell, and L6b neurons; <em>Slc17a6</em> for CLA neurons; <em>Ccn2</em> for L6b neurons; <em>Rprm</em> for L6a neurons; and <em>Syt17</em> for shell neurons (Fig. <a href="#Fig2" class="usa-link">2a–c</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Map of the claustro-insular region.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373765_41467_2025_63138_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5df/12373765/3a8d306df39a/41467_2025_63138_Fig2_HTML.jpg" loading="lazy" id="d33e601" height="716" width="667" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic of the data processing approach. <strong>b</strong> Representative image of a coronal section (bregma +1.045) of the claustro-insular region labeled by smFISH with probes for <em>Nr4a2</em> (purple), <em>Ccn2</em> (cyan), and <em>Rprm</em> (blue). The white square highlights the region magnified in the inset. The dotted line highlights the border of the external capsule. Scale bar: 200 μm. The same labeling was done on a total of n = 15 sections. <strong>c</strong> Violin plots showing the distribution of expression of marker genes across identified cell types in the smFISH data (raw data are shown on a linear scale). Probe set A: n = 2 mice, 15 sections, 91,634 clustered cells; Probe set B: n = 2 mice, 15 sections, 80,627 clustered cells. <strong>d</strong> Spatial organization of neuronal cell populations identified in the smFISH dataset. Multiple images are aligned, and cells are projected on the same framework. Scale bar: 500 μm. n = 2 mice, 8 sections. <strong>e</strong> Proportional visualization of the claustro-insular region, binned into 20 μm bins. Bin colors are weighted by the relative proportion of neuronal cell types within that bin. n = 2 mice, 8 sections. <strong>f</strong> 2D density maps showing the spatial distribution of neuronal cell populations within the claustro-insular region. The total area comprises 90% of the 2D kernel density estimation of the distribution of all cells within the respective population. n = 2 mice, 8 sections. <strong>g</strong>–<strong>j</strong> Maps of the claustro-insular region at four different anteroposterior levels. Colored areas represent the highest density region of cell types at a 50% probability level of the two-dimensional kernel density estimate. The inset shows the representation of the CLA and endopiriform nucleus (EP) in the Allen Mouse Brain Atlas (mouse.brain-map.org), at the corresponding anteroposterior position. Red squares highlight the region imaged after smFISH labeling for map generation. Scale bar: 500 μm. n = 2 mice, 6–8 sections/bregma level. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par12">We chose smFISH for its high quantitative resolution, which enabled us to capture the substantial variability in expression levels of marker genes (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>). After staining, we clustered the cells, identified cell types, aligned images from sections at the same anteroposterior level, and projected the cells’ coordinates onto a reference image to construct 2D maps of cell type localization for each brain level (Fig. <a href="#Fig2" class="usa-link">2d</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">5a</a>). To illustrate the diversity of cell types in the claustro-insular region, we generated proportional visualizations of these maps (Fig. <a href="#Fig2" class="usa-link">2e</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">5b</a>). We binned the 2D spatial maps into 20 μm bins, and the proportion of each cell type in each bin was used to assign a color weighted by these proportions. To characterize the spatial distribution of individual cell types within the claustro-insular region, we generated 2D density maps for CLA neurons, shell neurons, <em>Ccn2</em>+ neurons (labeling mainly L6b), and <em>Rprm</em>+ neurons (labeling mainly L6a, L5b, and pir) at each brain level (Fig. <a href="#Fig2" class="usa-link">2f</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">5c</a>). The final maps represent the highest density regions of CLA, shell, <em>Ccn2</em>+, and <em>Rprm</em>+ neurons at the 50% probability level of their respective 2D kernel density estimates (Fig. <a href="#Fig2" class="usa-link">2g–j</a>). These data-driven maps proved more flexible than conventional brain atlases, reducing discrepancies in boundary annotations and accommodating overlapping borders between structures (Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>), and accurately reflecting the high complexity of the claustro-insular region, where distinct cell types are closely juxtaposed and interspersed. Notably, neurons expressing canonical CLA markers were distributed throughout the entire CLA/EP, as well as in L6a in more caudal sections. Moreover, the high-density regions for CLA and shell neurons showed substantial spatial overlap with each other and with cortical L6a and L6b neurons (Fig. <a href="#Fig2" class="usa-link">2g–j</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Spatial organization of cell types and subtypes in the claustro-insular region</h3>
<p id="Par13">We leveraged our spatial map to localize each cell type and subtype identified by scRNAseq within the brain. As expected, well-defined cell types from the MERFISH dataset, located within or near the claustro-insular region boundaries, were reliably mapped (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>). We then focused on characterizing the spatial localization of CLA, shell, and L6a subtypes (Fig. <a href="#Fig3" class="usa-link">3a</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Spatial organization of cell types and subtypes in the claustro-insular region.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373765_41467_2025_63138_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5df/12373765/904105e7f363/41467_2025_63138_Fig3_HTML.jpg" loading="lazy" id="d33e686" height="625" width="667" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> UMAP plot highlighting the cell types or subtypes whose localization was determined by smFISH and MERFISH spatial transcriptomics. <strong>b</strong>–<strong>i</strong> Spatial organization of different neuronal types and subtypes projected on the map of the claustro-insular region at bregma +1.045. <em>Left</em>: cells from the MERFISH dataset. Prediction scores highlight the confidence of cell type attribution in the MERFISH data after reference-based mapping onto the scRNAseq dataset. Dotted lines highlight the border of the ROI used for cell selection. Cells from one representative section are shown for each cell type and subtype. <em>Right</em>: cells from smFISH datasets. Violin plots below the maps show the distribution of expression of marker genes used for cell type and subtype identification in the smFISH data (raw data are shown on a linear scale). Plus and minus signs indicate whether the gene is considered a marker of the given cell population or not, respectively; max count corresponds to the maximum gene expression levels in all cells analyzed in the respective experiments. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par14">Shell subtypes were consistently identified across animals and exhibited highly stereotyped spatial localizations (Fig. <a href="#Fig3" class="usa-link">3d–f</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a>), suggesting that they are not merely a heterogeneous group of cells adjacent to the CLA, but rather represent distinct and spatially organized neuronal subpopulations. Contrary to the shell and L6a subpopulations, CLA 1 and CLA 2 subtypes were spatially intermingled (Fig. <a href="#Fig3" class="usa-link">3b, c</a>). Consistent with the patterns observed in Fig. <a href="#Fig2" class="usa-link">2</a>, neurons exhibiting a CLA transcriptomic identity (i.e., CLA 1 and CLA 2) were distributed throughout the entire CLA/EP.</p>
<p id="Par15">Nonetheless, subtle differences in the expression levels of select genes may distinguish neuronal populations between the CLA and EP, even if these differences are not necessarily captured by clustering alone. To identify region-specific genes, we performed a joint analysis of the scRNAseq and MERFISH datasets, investigating how their transcriptionally defined nearest neighbors were distributed across subregions of the CLA/EP complex (Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>). We were able to assign a regional identity to ~10% of CLA 1 and CLA 2 neurons in the scRNAseq dataset and identified a handful of differentially expressed genes between the CLA and EP, including <em>Cdh8</em>, a gene known for its CLA specificity<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>,<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">9b</a>). Furthermore, the MERFISH data revealed regional differences in cell type composition, with the EP containing a slightly higher proportion of CLA 2 neurons compared to the CLA (Supplementary Fig. <a href="#MOESM1" class="usa-link">9c</a>).</p>
<p id="Par16">To evaluate whether cell type composition varies along the anteroposterior axis, we compared sections from three different anteroposterior levels (Supplementary Fig. <a href="#MOESM1" class="usa-link">10</a>). While CLA and shell subtypes were uniformly distributed (even in more posterior sections where these populations are more dispersed within layer 6), L6a subtypes displayed notable regional variation. Specifically, L6a 3 neurons were detected exclusively in more posterior sections, where they showed partial overlap with L6a 2 neurons. In contrast, only a small number of L6a 4 neurons were identified in the MERFISH dataset (Supplementary Figs. <a href="#MOESM1" class="usa-link">7g, h</a> and <a href="#MOESM1" class="usa-link">10</a>), which may reflect sampling differences between the scRNAseq and MERFISH experiments (this subtype may be predominantly located in more anterior cortical regions not included in the MERFISH dataset).</p>
<p id="Par17">Since MERFISH may not be accessible to all researchers, we compared our results with a simplified strategy using smFISH, which relies on combinations of only 2–3 marker genes to distinguish cell subpopulations (Fig. <a href="#Fig3" class="usa-link">3b–i</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">11</a>). By assessing the presence or absence of select marker genes, we were able to match subpopulations from the smFISH data to those identified in the scRNAseq dataset. We observed strong correspondence between the spatial localization of cell types and subtypes identified by smFISH and those detected by MERFISH, particularly for cell populations with high prediction scores in the MERFISH dataset (Fig. <a href="#Fig3" class="usa-link">3b–i</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a>). Together, these results highlight the robustness of our claustro-insular maps and underscore the value of integrating spatial transcriptomics with scRNAseq for the high-resolution characterization of complex brain regions.</p></section><section id="Sec6"><h3 class="pmc_sec_title">
<em>Nr4a2</em> haploinsufficiency alters the transcriptome of CLA projection neurons</h3>
<p id="Par18">We next aimed to investigate how the transcriptomic identity of CLA neurons is established. Noting the variable levels of <em>Nr4a2</em> expression across different subpopulations of the claustro-insular region, particularly in CLA and shell subtypes (Fig. <a href="#Fig1" class="usa-link">1e</a>), and given the early developmental onset of <em>Nr4a2</em> expression in CLA neurons<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>, we hypothesized that this transcription factor may regulate gene expression within these subpopulations. To test this, we used <em>Nr4a2</em> heterozygous mice (<em>Nr4a2</em><sup><em>del/wt</em></sup>) carrying one non-functional <em>Nr4a2</em> allele, and compared them to wild-type littermates (<em>Nr4a2</em><sup><em>wt/wt</em></sup>). smFISH quantification of <em>Nr4a2</em> mRNA in the claustro-insular region showed a 2.6-fold reduction of <em>Nr4a2</em> expression in heterozygous mice (Fig. <a href="#Fig4" class="usa-link">4a–c</a>). To assess the impact of <em>Nr4a2</em> heterozygosity on gene expression in the claustro-insular region, we compared the scRNAseq-derived transcriptomes of various cell types and subtypes between <em>Nr4a2</em><sup><em>wt/wt</em></sup> and <em>Nr4a2</em><sup><em>del/wt</em></sup> mice. CLA projection neurons, particularly the CLA 1 subtype, exhibited a substantial number of differentially expressed genes, while other cell types appeared relatively unaffected (Fig. <a href="#Fig4" class="usa-link">4d</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>, Supplementary Data <a href="#MOESM3" class="usa-link">5</a>–<a href="#MOESM3" class="usa-link">20</a>). The CLA 2 subtype had fewer cells represented in the dataset, likely contributing to the low number of significantly modulated genes. Among the differentially expressed genes, we identified key marker genes (e.g., <em>Cdh13</em>, <em>Rxfp1</em>, <em>Nr2f2</em>, <em>Syt17</em>, and <em>Ntm</em>) as well as genes coding for ion channels that could influence neuronal function (e.g., <em>Scn1b</em> and <em>Ryr2</em>) (Fig. <a href="#Fig4" class="usa-link">4e, f</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>, Supplementary Data <a href="#MOESM3" class="usa-link">5</a>–<a href="#MOESM3" class="usa-link">20</a>). This strong modulation of gene expression in <em>Nr4a2</em>-expressing cells was further confirmed by smFISH quantification of selected up- and downregulated genes in <em>Nr4a2</em> heterozygous mice (Fig. <a href="#Fig4" class="usa-link">4g, h</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">14a–h</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Transcriptomic perturbations in <em>Nr4a2</em> haploinsufficient mice.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373765_41467_2025_63138_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5df/12373765/c7b7ed866e0d/41467_2025_63138_Fig4_HTML.jpg" loading="lazy" id="d33e1056" height="658" width="667" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Images of the CLA in coronal sections from <em>Nr4a2</em><sup><em>wt/wt</em></sup> (<strong>a</strong>) and <em>Nr4a2</em><sup><em>del/wt</em></sup> (<strong>b</strong>) mice, with <em>Nr4a2</em> mRNA transcripts labeled by smFISH (white). Insets show magnified regions (white squares). Nuclei are stained with DAPI (blue). Scale bar: 200 μm. <strong>c</strong> Mean number of <em>Nr4a2</em> mRNA puncta per cell in the claustro-insular area. Each dot represents one section of an <em>Nr4a2</em><sup><em>del/wt</em></sup> mouse relative to the mean of all sections of <em>Nr4a2</em><sup><em>wt/wt</em></sup> control mice processed in the same batch. Data are mean ± SD. n = 4 mice per genotype; ***p &lt; 0.001; χ<sup>2</sup>-based LRT applied to a negative binomial GLMM with quadratic parametrization. <strong>d</strong> Total number of significantly modulated genes in <em>Nr4a2</em><sup><em>del/wt</em></sup> mice in each cell type from the scRNAseq dataset (n = 5 <em>Nr4a2</em><sup><em>wt/wt</em></sup> and 8 <em>Nr4a2</em><sup><em>del/wt</em></sup> mice). Two-sided Wilcoxon rank sum test; <em>p</em>-values were adjusted for multiple comparisons using the Bonferroni method. Violin plots illustrating the distribution of expression for downregulated (<strong>e</strong>) and upregulated (<strong>f</strong>) genes across claustro-insular neuronal populations from <em>Nr4a2</em><sup><em>wt/wt</em></sup> and <em>Nr4a2</em><sup><em>del/wt</em></sup> mice in the scRNAseq dataset (normalized data are shown on a log<sub>10</sub> scale). Mean number of mRNA puncta per cell of downregulated (<strong>g</strong>) and upregulated (<strong>h</strong>) genes in <em>Nr4a2</em>+ and/or <em>Oprk1</em>+ cells from the claustro-insular region. Each dot represents one section of an <em>Nr4a2</em><sup><em>del/wt</em></sup> mouse relative to the mean of all sections of <em>Nr4a2</em><sup><em>wt/wt</em></sup> control mice processed in the same batch. Data are mean ± SD. n = 2-3 mice per genotype. *p &lt; 0.05, ***p &lt; 0.001; χ<sup>2</sup>-based LRT applied to a negative binomial GLMM with quadratic parametrization; <em>p</em>-values were adjusted for multiple comparisons using the Holm method. Cell type-specific modulation of <em>Ntm</em> (<strong>i</strong>) and <em>Ryr2</em> (<strong>j</strong>) expression. <em>Top:</em> Violin plots show the distribution of expression of marker genes used for cell type identification in the smFISH data (raw data are shown on a linear scale). <em>Bottom:</em> Box plots show the mean <em>Ntm</em> and <em>Ryr2</em> expression levels within specific cell populations from <em>Nr4a2</em><sup><em>wt/wt</em></sup> and <em>Nr4a2</em><sup><em>del/wt</em></sup> mice (n = 2 mice per genotype, 6–8 sections). Each dot represents one section. Box limits: Q1–Q3; line: median; whiskers: ±1.5*IQR. *p &lt; 0.05, ***p &lt; 0.001; χ<sup>2</sup>-based LRT applied to a negative binomial GLMM; <em>p</em>-values adjusted using the Holm method. Source data are provided as a Source Data file. Exact sample details are provided in Supplementary Data <a href="#MOESM3" class="usa-link">1</a>.</p></figcaption></figure><p id="Par19">The scRNAseq analyses also suggested differential sensitivity of CLA and shell neurons to a reduction in <em>Nr4a2</em> mRNA levels (Fig. <a href="#Fig4" class="usa-link">4e, f</a>). To explore this further, we clustered cells from the smFISH experiments to differentiate CLA from shell neurons based on the expression patterns of <em>Nr4a2</em> and <em>Syt17</em>, and evaluated the expression of two genes that showed strong upregulation in <em>Nr4a2</em><sup><em>del/wt</em></sup> mice (Fig. <a href="#Fig4" class="usa-link">4i, j</a>). <em>Ntm</em> and <em>Ryr2</em> were either not or only slightly modulated in cells lacking CLA identity, indicating that <em>Nr4a2</em> heterozygosity primarily affects transcription in CLA neurons, with no or minimal effects on neighboring shell and other cell populations.</p></section><section id="Sec7"><h3 class="pmc_sec_title">
<em>Nr4a2</em> haploinsufficiency perturbs CLA projection neuron firing</h3>
<p id="Par20">Since multiple genes encoding ion channels, neurotransmitter receptors, and related signaling proteins were dysregulated in CLA projection neurons of <em>Nr4a2</em> haploinsufficient mice (Fig. <a href="#Fig5" class="usa-link">5a</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">15a</a>), we investigated whether the transcriptomic changes led to alterations in spiking behavior. Whole-cell patch-clamp recordings of mPFC-projecting <em>Slc17a6</em><sup><em>+</em></sup> CLA neurons (Fig. <a href="#Fig5" class="usa-link">5b</a>) revealed reduced firing frequency in <em>Nr4a2</em><sup><em>del/wt</em></sup> cells (Fig. <a href="#Fig5" class="usa-link">5c–e</a>), with no change in resting membrane potential (Fig. <a href="#Fig5" class="usa-link">5f</a>). This hypoexcitability persisted even in the presence of blockers for fast GABAergic and glutamatergic synaptic transmission (Fig. <a href="#Fig5" class="usa-link">5d, e</a>), suggesting intrinsic alterations in cellular properties. Action potential (AP) analysis showed that while rise time and amplitude were unaffected, decay time and half-width were shorter in <em>Nr4a2</em><sup><em>del/wt</em></sup> neurons (Fig. <a href="#Fig5" class="usa-link">5g, h</a>), a feature typically indicative of higher excitability. Several genes encoding different types of sodium and potassium channels were dysregulated in <em>Nr4a2</em> heterozygous mice (Fig. <a href="#Fig5" class="usa-link">5a</a>). Notably, the Nav1.6 α subunit (encoded by <em>Scn8a</em>) was upregulated, while the auxiliary β1 subunit of voltage-gated sodium channels (encoded by <em>Scn1b;</em> VGSCs) was downregulated (Fig. <a href="#Fig5" class="usa-link">5a</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">15a</a>). β subunits are known to influence VGSCs’ trafficking and modulate their biophysical properties, such as the resurgent sodium current. Reduced <em>Scn1b</em> expression has been associated with reduced resurgent current and decreased firing rates<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>,<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. However, neither AP depolarization kinetics (Fig. <a href="#Fig5" class="usa-link">5h</a>) nor the pharmacological modulation of resurgent sodium current differed between genotypes (Supplementary Fig. <a href="#MOESM1" class="usa-link">15b</a>), suggesting that changes in VGSC function are unlikely to underlie the observed excitability deficits.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. <em>Nr4a2</em> haploinsufficiency suppresses CLA neuron firing via enhanced recruitment of Ca<sup>2+</sup>-activated potassium channels.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373765_41467_2025_63138_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5df/12373765/c70d3fa5fb22/41467_2025_63138_Fig5_HTML.jpg" loading="lazy" id="d33e1295" height="879" width="800" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Significantly modulated genes encoding channels, neurotransmitter receptors, and related signaling proteins in CLA cells of <em>Nr4a2</em><sup><em>del/wt</em></sup> mice when compared to wild-type littermates. <strong>b</strong> Retrogradely labeled CLA neurons projecting to the mPFC were recorded. <em>Bottom</em>: example of a patched cell filled with biocytin. The same procedure was performed on n = 585 CLA neurons. <strong>c</strong> Example of the spiking activity evoked by a 1s 250 pA depolarizing step. <strong>d</strong> Input/Output functions of CLA neurons recorded in artificial cerebrospinal fluid (ACSF) or in blockers of the fast glutamatergic (NBQX and APV) and GABAergic (Gabazine) synaptic transmission (* indicates significant using Fisher’s LSD test corrected with a 5% FDR Benjamini–Hochberg). <strong>e</strong> Area under the curve (AUC) of the input/output function for each <em>Nr4a2</em><sup><em>wt/wt</em></sup> and <em>Nr4a2</em><sup><em>del/wt</em></sup> cell in ACSF and synaptic blockers. <strong>f</strong> Resting membrane potential is recorded at the opening of cells. <strong>g</strong> Examples of recorded action potentials (AP), (bottom traces: truncated AP). HW half-width, RT rise time, DT decay time, AHP afterhyperpolarization potential. <strong>h</strong> Quantifications of the different AP parameters (each circle represents the mean parameter value computed on all spikes of all steps for a given cell). <strong>i</strong> Example of the firing frequency evoked by a 250 pA current step under various pharmacological conditions. <strong>j</strong> Input/Output functions of CLA neurons recorded in the presence of VGCC blockers (10 µM nifedipine, 100 nM ω-agatoxin IVA, 3 µM ω-conotoxin GVIA, 400 nM SNX-482, 2 µM ML218), BK channels blocker (100 nM iberiotoxin), and RyR2 channel blocker (10–20 µM ryanodine). <strong>k</strong> Comparison of the firing functions between <em>Nr4a2</em><sup><em>wt/wt</em></sup> and <em>Nr4a2</em><sup><em>del/wt</em></sup> cells, expressed as a percentage of the mean wild-type AUC calculated for each pharmacological condition (see “Methods” section). Cav2&amp;3: ω-Agatoxin IVA, ω-Conotoxin GVIA, SNX-482, and ML218. Cav2.1&amp;2.2: ω-Agatoxin IVA and ω-Conotoxin GVIA. Cav2.3: SNX-482. Cav3: ML218. *p &lt; 0.05, ***p &lt; 0.001; Mann–Whitney U test. <strong>l</strong> Comparison of the relative AHP amplitude between <em>Nr4a2</em><sup><em>wt/wt</em></sup> and <em>Nr4a2</em><sup><em>del/wt</em></sup> cells under each pharmacological condition. **p &lt; 0.01; Mann–Whitney test. <strong>m</strong> Summary diagram of the mechanism controlling changes in firing frequency through the activation of BK channels. Data are shown as mean ± SEM. In <strong>e</strong>, <strong>f</strong>, <strong>h</strong>, <strong>k</strong>, and <strong>l</strong>, each circle represents one recorded neuron. Source data are provided as a Source Data file. See Supplementary Data <a href="#MOESM3" class="usa-link">1</a> for more details.</p></figcaption></figure><p id="Par21">We next investigated whether altered expression of voltage-gated potassium channel (VGPC, Kv) genes could account for the firing phenotype. Notably, <em>Kcnc1</em> (which encodes Kv3.1), which supports fast spiking by promoting AP repolarization, was downregulated in <em>Nr4a2</em><sup><em>del/wt</em></sup> neurons. Although reduced <em>Kcnc1</em> expression would be expected to prolong AP repolarization, we observed the opposite effect (Fig. <a href="#Fig5" class="usa-link">5h</a>). Additional dysregulation of <em>Kcnv1</em> (Kv8.1) and <em>Kcnf1</em> (Kv5.1), modulators of Kv2 channel biophysics<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>, may alter Kv channel function and sensitivity to antagonists like 4-aminopyridine<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>,<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. SCN1B can also form complexes with various VGPCs, including Kv1, Kv4.2, and Kv7<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>,<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. Application of 4-aminopyridine, an antagonist of Kv1-4 channels, strongly affected firing in wild-type cells but had no effect in <em>Nr4a2</em><sup><em>del/wt</em></sup> neurons (Supplementary Fig. <a href="#MOESM1" class="usa-link">15c, d</a>), while a Kv7 antagonist altered input/output curve shape but preserved the difference between genotypes (Supplementary Fig. <a href="#MOESM1" class="usa-link">15e</a>). These results indicate that VGPC dysfunction alone cannot explain the altered excitability. GIRK1 and GIRK3 channels (encoded by <em>Kcnj3</em> and <em>Kcnj4</em>, respectively), which were upregulated in <em>Nr4a2</em><sup><em>del/wt</em></sup> neurons, are G-protein-activated inward rectifier potassium channels that reduce excitability<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. However, blocking inward rectifier channels with barium altered the input/output curve shape but maintained the genotype difference (Supplementary Fig. <a href="#MOESM1" class="usa-link">15f</a>).</p>
<p id="Par22">In contrast, we observed a 25% increase in afterhyperpolarization potential (AHP) amplitude in <em>Nr4a2</em><sup><em>del/wt</em></sup> neurons, a key determinant of firing frequency (Fig. <a href="#Fig5" class="usa-link">5g, h</a>). During APs, activation of voltage-gated calcium channels (VGCCs) leads to calcium influx, which in turn activates calcium-gated potassium channels, including BK and SK subtypes, to regulate different phases of AHP<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>–<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. While <em>Kcnma1</em> (BK channel), which mediates the fast AHP, was highly transcribed in CLA neurons, its expression was unchanged across genotypes (Supplementary Fig. <a href="#MOESM1" class="usa-link">15</a>). Similarly, <em>Kcnn1, 2, and 3</em> (encoding SK channels) were expressed at low levels in both groups (Supplementary Fig. <a href="#MOESM1" class="usa-link">15a</a>). Strikingly, pharmacological blockade of VGCCs normalized both AHP amplitude and firing frequency (Fig. <a href="#Fig5" class="usa-link">5i–l</a>). Cav2.3 and Cav3 channels contributed to this rescue effect, despite their transcript levels remaining unchanged (Fig. <a href="#Fig5" class="usa-link">5k, l</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">15g–k</a>). Furthermore, blocking BK channels with iberiotoxin fully restored excitability in <em>Nr4a2</em><sup><em>del/wt</em></sup> neurons (Fig. <a href="#Fig5" class="usa-link">5j–l</a>).</p>
<p id="Par23">How can we link transcriptional dysregulation to BK channel-mediated hypoexcitability? We next investigated the potential role of altered calcium handling in the process. <em>Ryr2</em>, which encodes a ryanodine receptor central to calcium-induced calcium release (CICR) (Fig. <a href="#Fig5" class="usa-link">5m</a>), was overexpressed in <em>Nr4a2</em><sup><em>del/wt</em></sup> neurons (Fig. <a href="#Fig4" class="usa-link">4</a> and Fig. <a href="#Fig5" class="usa-link">5a</a>). Pharmacological blockade of ryanodine receptors significantly improved excitability and reduced AHP amplitude in <em>Nr4a2</em><sup><em>del/wt</em></sup> neurons (Fig. <a href="#Fig5" class="usa-link">5l</a>), supporting a role for enhanced CICR in mediating <em>Nr4a2</em>-dependent hypoexcitability. However, this might reflect only one component of a broader, more complex regulatory network. It also remains possible that <em>Ryr2</em> dysfunction arises not solely from transcriptional changes, but from interactions with other dysregulated signaling pathways or post-translational mechanisms. We thus examined the potential contribution of disrupted G-protein-coupled receptor (GPCR) signaling. We did not attempt direct activation of specific GPCR pathways, as the potential effects on non-CLA projection neurons would have complicated the interpretation of pharmacological manipulations. Instead, we focused on downstream components of GPCR signaling to assess whether altered cAMP/PKA signaling, potentially resulting from downregulation of <em>Gnao1/Gnb4/Gnal</em>, could affect RyR2 function<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">16</a>). Since <em>Gnao1</em> encodes Gα<sub>o</sub>, an inhibitor of adenylate cyclase (AC), its downregulation could hypothetically lead to increased cAMP levels and PKA activation, which is known to modulate RyR2 activity. However, inhibition of AC (SQ22536) did not rescue the reduced firing frequency observed in <em>Nr4a2</em><sup><em>del/wt</em></sup> neurons (Supplementary Fig. <a href="#MOESM1" class="usa-link">16a, d, g, j, k</a>). Interestingly, PKA inhibition selectively altered the input/output function kinetics in wild-type neurons, producing a block at high current injections, possibly reflecting altered voltage-gated sodium channel inactivation dynamics (Supplementary Fig. <a href="#MOESM1" class="usa-link">16b, e</a>). Despite these genotype-dependent effects on intracellular signaling, PKA inhibition did not restore normal excitability in <em>Nr4a2</em><sup>del/wt</sup> neurons within the depolarization range that excludes the block (Supplementary Fig. <a href="#MOESM1" class="usa-link">16h, j, k</a>). Nonetheless, other GPCR pathways, such as Gα<sub>q</sub>-mediated IP<sub>3</sub> signaling or pathways converging on other calcium stores, may still contribute to the phenotype. Like the effects of PKA inhibition, blocking IP3 receptors (2-APB) primarily altered the input/output function kinetics in wild-type neurons, leading to a depolarization block at higher current injections (Supplementary Fig. <a href="#MOESM1" class="usa-link">16c, f</a>). However, 2-APB treatment did not restore the firing frequency in <em>Nr4a2</em><sup>del/wt</sup> neurons within the depolarization range that excluded depolarization block (Supplementary Fig. <a href="#MOESM1" class="usa-link">16i, j</a>). Collectively, these findings suggest that while the AC–cAMP–PKA pathway and IP3 receptor signaling modulate neuronal excitability, they are unlikely to represent the primary mechanisms underlying the hypoexcitability observed in <em>Nr4a2</em><sup>del/wt</sup> neurons. Instead, we propose that upregulation of <em>Ryr2</em> plays a central role by enhancing CICR, which in turn increases activation of BK channels (Fig. <a href="#Fig5" class="usa-link">5m</a>). This leads to an elevated AHP amplitude, thereby reducing firing frequency.</p></section><section id="Sec8"><h3 class="pmc_sec_title">
<em>Nr4a2</em> haploinsufficiency disrupts claustro-insular cellular architecture</h3>
<p id="Par24">Given the alterations observed in other brain systems<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>–<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> and the substantial transcriptomic changes in CLA identity genes in <em>Nr4a2</em><sup><em>del/wt</em></sup> mice (Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>), we investigated whether these modifications could affect the cellular composition of the claustro-insular area. Our smFISH analyses showed that while the overall number of cells in the claustro-insular region remained unchanged (Fig. <a href="#Fig6" class="usa-link">6a</a>), there was a 2.3-fold reduction in the number of cells exhibiting robust <em>Nr4a2</em> expression (defined as a minimum of 20 mRNA puncta/cell) (Fig. <a href="#Fig6" class="usa-link">6b</a>). The scRNAseq dataset revealed a strong decrease in the proportion of cells classified as CLA 1 and CLA 2, with no significant differences noted in other cell types and subtypes between genotypes (Fig. <a href="#Fig6" class="usa-link">6c</a>). This reduction was also evident in the MERFISH dataset (Fig. <a href="#Fig6" class="usa-link">6d, e</a>). To further characterize the changes in cellular composition, we performed a similar analysis on smFISH data, confirming cell type-specific alterations (Fig. <a href="#Fig6" class="usa-link">6f</a>). We aligned multiple images, overlaid cell identities onto a reference section, and generated a contrast map to highlight the relative proportions of CLA and shell cells within 20 µm bins (Fig. <a href="#Fig6" class="usa-link">6g</a>). The resulting map revealed a notable reduction in CLA cell density, particularly in the core of the CLA and in the EP.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Alteration of cellular composition in the claustro-insular region of <em>Nr4a2</em> haploinsufficient mice.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373765_41467_2025_63138_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5df/12373765/ba20f5bf8da8/41467_2025_63138_Fig6_HTML.jpg" loading="lazy" id="d33e1765" height="803" width="668" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Quantification of the total number of cells present in the ROI around the CLA in <em>Nr4a2</em><sup><em>wt/wt</em></sup> and <em>Nr4a2</em><sup><em>del/wt</em></sup> mice. Each dot represents the mean of one section. Data are mean ± SD. n = 4 <em>Nr4a2</em><sup><em>wt/wt</em></sup> mice, 26 sections, 55,697 cells; n = 4 <em>Nr4a2</em><sup><em>del/wt</em></sup> mice, 34 sections, 71,389 cells. <strong>b</strong> Total number of <em>Nr4a2+</em> cells ( ≥ 20 mRNA puncta/cell) within the ROI. Each dot represents one section of an <em>Nr4a2</em><sup><em>del/wt</em></sup> mouse relative to the mean of all sections of <em>Nr4a2</em><sup><em>wt/wt</em></sup> control mice processed in the same batch. Data are mean ± SD. n = 4 <em>Nr4a2</em><sup><em>wt/wt</em></sup> mice, 26 sections, 55,697 cells; n = 4 <em>Nr4a2</em><sup><em>del/wt</em></sup> mice, 34 sections, 71,389 cells. ***p &lt; 0.001; χ<sup>2</sup>-based LRT applied to a negative binomial GLMM with quadratic parametrization. <strong>c</strong> Ratio of cell type proportions from the scRNAseq dataset. Each dot represents one <em>Nr4a2</em><sup><em>del/wt</em></sup> mouse relative to the mean of all <em>Nr4a2</em><sup><em>wt/wt</em></sup> control mice. Data are mean ± SD. n = 5 <em>Nr4a2</em><sup><em>wt/wt</em></sup> mice, 6851 cells; n = 8 <em>Nr4a2</em><sup><em>del/wt</em></sup> mice, 8001 cells. *p &lt; 0.05; moderated <em>t</em>-test with robust empirical Bayes variance shrinkage; <em>p</em>-values adjusted using the Benjamini–Hochberg false discovery rate (FDR) method. Spatial organization of cell types and subtypes on a representative coronal section of the claustro-insular region of an <em>Nr4a2</em><sup><em>wt/wt</em></sup> (<strong>d</strong>) and an <em>Nr4a2</em><sup><em>del/wt</em></sup> (<strong>e</strong>) mouse from the MERFISH dataset. <em>Left:</em> all cell types and subtypes. <em>Right:</em> CLA 1 and CLA 2 cells. <strong>f</strong> Ratio of cell type proportions from the smFISH dataset. Each dot represents one section of an <em>Nr4a2</em><sup><em>del/wt</em></sup> mouse relative to the mean of all sections of <em>Nr4a2</em><sup><em>wt/wt</em></sup> control mice. Data are mean ± SD. n = 2 <em>Nr4a2</em><sup><em>wt/wt</em></sup> mice, 14 sections, 17,737 cells; n = 2 <em>Nr4a2</em><sup><em>del/wt</em></sup> mice, 15 sections, 16,174 cells. ***p &lt; 0.001; moderated <em>t</em>-test with empirical Bayes variance shrinkage; <em>p</em>-values adjusted using the Benjamini–Hochberg FDR method. <strong>g</strong>
<em>Left</em>: Spatial organization of neuronal populations identified in the smFISH dataset, with aligned images from <em>Nr4a2</em><sup><em>wt/wt</em></sup> and <em>Nr4a2</em><sup><em>del/wt</em></sup> mice projected on the same framework. <em>Right</em>: CLA/shell contrast maps of the claustro-insular region, binned into 20 μm bins. Bin colors correspond to the relative proportion of CLA or shell neurons within each bin. n = 2 <em>Nr4a2</em><sup><em>wt/wt</em></sup> mice, 14 sections; n = 2 <em>Nr4a2</em><sup><em>del/wt</em></sup> mice, 15 sections. Source data are provided as a Source Data file.</p></figcaption></figure></section><section id="Sec9"><h3 class="pmc_sec_title">
<em>Nr4a2</em> is a key regulator of CLA projection neuron identity</h3>
<p id="Par25"><em>Nr4a2</em> haploinsufficient CLA projection neurons exhibited downregulation of CLA identity genes, including <em>Cdh13</em>, <em>Rxfp1</em>, and <em>Nr2f2</em>, alongside upregulation of shell identity genes, such as <em>Syt17</em> and <em>Ntm</em> (Fig. <a href="#Fig4" class="usa-link">4e–j</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>). To further explore the role of <em>Nr4a2</em> in establishing CLA projection neuron identity, we trained linear classifiers (Fig. <a href="#Fig7" class="usa-link">7a, b</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">17</a>). First, we trained a classifier to distinguish <em>Nr4a2</em><sup><em>wt/wt</em></sup> CLA, shell and L6a neurons using the whole transcriptome and found that modulated genes had significantly higher weights than unmodulated genes, indicating that these genes are important to discriminate the three cell types (Fig. <a href="#Fig7" class="usa-link">7a</a>). Modulated genes were also significantly enriched among high-weighted genes from the classifier compared to their representation in the transcriptome (Supplementary Data <a href="#MOESM3" class="usa-link">1</a>). When inspecting the expression of modulated genes in <em>Nr4a2</em><sup><em>del/wt</em></sup> CLA neurons across CLA, shell and L6a neuron subtypes, it became apparent that among these modulated genes (1) downregulated genes typically were CLA-specific markers, (2) upregulated genes typically were expressed by both shell and L6a subtypes, and (3) <em>Nr4a2</em><sup><em>del/wt</em></sup> CLA neurons often exhibited expression of these genes at the same level as L6a neurons (Fig. <a href="#Fig4" class="usa-link">4e–f</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>). We therefore sought to explore whether these modulated genes indicate a shift towards a shell-like or L6a-like identity. To identify the cell type toward which the identity of <em>Nr4a2</em><sup><em>del/wt</em></sup> CLA projection neurons shifts, we trained another linear classifier using only modulated genes (Fig. <a href="#Fig7" class="usa-link">7b</a>). The expression pattern of these genes in wild-type cells allowed for accurate classification of CLA, shell, and L6a neurons. Similarly, shell and L6a neurons from <em>Nr4a2</em><sup><em>del/wt</em></sup> mice were classified with accuracy scores comparable to that of <em>Nr4a2</em><sup><em>wt/wt</em></sup> cells. However, CLA neurons from <em>Nr4a2</em> haploinsufficient mice showed a significant decrease in classification accuracy, with nearly 18% of the cells being misclassified as shell projection neurons. These results suggest that CLA and shell neurons are transcriptomically related and that <em>Nr4a2</em> influences their transcriptional programs based on its expression levels.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7. CLA and shell neuronal identities are driven by <em>Nr4a2.</em>
</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373765_41467_2025_63138_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c5df/12373765/381f851c92f4/41467_2025_63138_Fig7_HTML.jpg" loading="lazy" id="d33e1958" height="531" width="667" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Box plots showing the distribution of gene weights for modulated and unmodulated genes from a linear classifier trained on the whole transcriptome. Each dot represents one gene. n = 324 modulated genes; n = 17,707 unmodulated genes. Box limits: Q1–Q3; line: median; whiskers: ±1.5*IQR. ***p &lt; 0.001; Two-sided Wilcoxon rank sum test with continuity correction. <strong>b</strong> Cell type classification performance of CLA, shell, and L6a neurons of <em>Nr4a2</em><sup><em>wt/wt</em></sup> and <em>Nr4a2</em><sup><em>del/wt</em></sup> mice from the scRNAseq dataset using a linear support vector classification algorithm trained on modulated genes. Violin plots show the distribution of classification outcomes across 1000 iterations. n = 5 <em>Nr4a2</em><sup><em>wt/wt</em></sup> mice, 4363 cells; n = 8 <em>Nr4a2</em><sup><em>del/wt</em></sup> mice, 4537 cells. ***p &lt; 0.001; χ<sup>2</sup>-based LRT applied to a logistic regression model; <em>p</em>-values were adjusted for multiple comparisons using the Holm method. <strong>c</strong> Dot plot showing the contrast in regulon specificity scores between <em>Nr4a2</em><sup><em>wt/wt</em></sup> and <em>Nr4a2</em><sup><em>del/wt</em></sup> mice for CLA, shell, and L6a cell types from the scRNAseq dataset. <strong>d</strong> Violin plots showing the distribution of downregulated and upregulated transcription factors expression across claustro-insular neuronal populations from <em>Nr4a2</em><sup><em>wt/wt</em></sup> and <em>Nr4a2</em><sup><em>del/wt</em></sup> mice in the scRNAseq dataset (normalized data are shown on a log<sub>10</sub> scale). <strong>e</strong> Raster plot showing transcription factors from enriched regulons and their target-modulated genes in <em>Nr4a2</em><sup><em>del/wt</em></sup> CLA neurons. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par26">Several transcription factors were modulated as a result of <em>Nr4a2</em> haploinsufficiency. To assess whether the shift in CLA identity was directly mediated by <em>Nr4a2</em>, that is, whether modulated genes are direct targets of <em>Nr4a2</em> or whether other transcription factors play an intermediary role, we inferred gene regulatory networks (i.e., regulons) from the list of expressed genes. We identified 28 regulons involving 16 transcription factors (Fig. <a href="#Fig7" class="usa-link">7c</a>), characterized cell type and genotype-specific regulons by computing regulon specificity scores<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>, and calculated contrasts in these scores between genotypes. While the Nr4a2 regulon showed differential enrichment in CLA neurons between <em>Nr4a2</em><sup><em>wt/wt</em></sup> and <em>Nr4a2</em><sup><em>del/wt</em></sup> mice, three other regulons, involving <em>Nr2f2</em>, <em>Cux1</em>, and <em>Nfib</em>, displayed higher enrichment scores in <em>Nr4a2</em><sup><em>wt/wt</em></sup> CLA projection neurons (Fig. <a href="#Fig7" class="usa-link">7c</a>). Among these, <em>Nr2f2</em> and <em>Cux1</em> were downregulated due to <em>Nr4a2</em> haploinsufficiency, while <em>Nfib</em> was upregulated (Fig. <a href="#Fig7" class="usa-link">7d</a>). Matching the downstream targets of these transcription factors with the list of modulated genes revealed that <em>Nr2f2</em> and <em>Cux1</em> play critical roles in activating many downregulated genes and repressing several upregulated genes (Fig. <a href="#Fig7" class="usa-link">7e</a>). These transcription factors are direct targets of <em>Nr4a2</em>, which promotes their transcription (Fig. <a href="#Fig7" class="usa-link">7e</a>). Together, these results indicate that <em>Nr4a2</em> exerts a dose-dependent regulation over CLA and shell identities by modulating the expression of <em>Nr2f2</em> and <em>Cux1</em>.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par27">In this study, we first characterized the transcriptomic identity and spatial organization of cell types and subtypes in the mouse claustro-insular region at high-resolution, then created a versatile map of this region using cell type-specific marker genes, and ultimately demonstrated that <em>Nr4a2</em>, a gene encoding a nuclear transcription factor, plays a crucial role in the development of this brain area.</p>
<p id="Par28">By combining single-cell RNA sequencing with MERFISH spatial transcriptomics, we molecularly distinguished and spatially localized several cortical and subcortical neuronal cell types in the mouse claustro-insular region, including L2/3, L5b, L6b, and several L6a and striatal populations. Our analysis revealed two subtypes of CLA projection neurons, CLA 1 and CLA 2, both defined by their expression of canonical CLA marker genes such as <em>Nr4a2</em>, <em>Gnb4</em>, <em>Smim32</em>, and <em>Slc17a6</em>. While the CLA 2 population can be distinguished by the expression of immediate early genes, both CLA subtypes are genetically extremely close and spatially indistinguishable, supporting previous findings of a single transcriptomically defined CLA population<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>–<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Notably, the CLA 2 subtype may represent recently activated CLA 1 neurons whose transcriptomic profile is partially and transiently modulated — a sparse population missed in previous studies.</p>
<p id="Par29">Using MERFISH and smFISH, we explored whether the CLA 1 and CLA 2 subtypes corresponded exclusively to neurons located within the CLA. We found that neurons with a CLA transcriptional identity were not restricted to the CLA, but were also located in the EP and in L6a of some cortical areas, often neighboring the CLA. Previous transcriptomic studies primarily focused on CLA neurons, often overlooking EP neurons and the less characterized dorsal and caudal CLA-like neurons located within L6a (e.g., bregma −1.355)<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Our results suggest that these neurons represent the same cell type but are differentiated by their localization, by the expression of a select number of genes, and likely by their projection targets and functions within the brain.</p>
<p id="Par30">In addition to CLA neurons, we identified a second neuronal population with a transcriptomic identity that lies intermediate between claustral and cortical neurons. We named these neurons “shell projection neurons”, as they likely correspond to the “claustrum shell” neurons described by Erwin et al.<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. However, their transcriptomic profiles and spatial organization significantly differ from those of CLA neurons, leading us to classify them as a separate population. This population is subdivided into three subtypes, each characterized by specific expression levels of <em>Syt17</em> and low levels of <em>Nr4a2</em>, and exhibiting distinct and stereotypical spatial organizations within the mouse brain. Notably, all shell neurons express <em>Gnb4</em> (as do some L6b and L5b neurons), although at lower levels than CLA neurons. This indicates that <em>Gnb4</em> is not a unique marker of one population in the claustro-insular area, and caution should be exercised when using the <em>Gnb4</em>-Cre line for targeting. Finally, like the CLA 2 subtype, the shell 3 cluster is characterized by strong expression of immediate early genes and may represent recently activated neurons from other shell subpopulations.</p>
<p id="Par31">Several anatomical definitions of the CLA boundaries have been proposed over the years<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>–<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>, including a subdivision of the CLA into a core and a shell<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Our findings demonstrate that CLA, shell, and cortical neurons exhibit a significant overlap in their spatial localizations, indicating that sharp delineations around claustro-insular structures are inadequate. Additionally, studies by other groups using retrograde tracing have shown a partial overlap in the projection sites of CLA and shell neurons, complicating the distinction between these populations based solely on their projection targets<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Given the complexity of this brain region, we propose a new map for the claustro-insular area, grounded in its cell types. Using a data-driven approach, involving smFISH labeling of coronal sections, we have constructed a versatile map that accounts for this overlap. Whether or not shell neurons are considered part of the CLA/EP complex, in this map, these cells are appropriately annotated. We demonstrate the utility of this map by projecting cells from the MERFISH dataset onto it. Moreover, this map can also be overlaid over any labeled coronal section of the mouse claustro-insular region, using landmarks such as the tip of the piriform cortex and the lining of the external capsule.</p>
<p id="Par32">The potentially shared developmental origin of CLA and shell neurons is supported by their common expression of <em>Nr4a2</em>, although at different levels. Given the early and sustained expression of this transcription factor in developing CLA neurons<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>, we aimed to investigate <em>Nr4a2’s</em> role in the establishment of neuronal populations within the claustro-insular region. Using a <em>Nr4a2</em> haploinsufficiency model, we unexpectedly found that only neurons with high <em>Nr4a2</em> expression (i.e., CLA neurons) exhibited transcriptomic alterations and reduced abundance when lacking one <em>Nr4a2</em> allele, whereas shell neurons, characterized by lower <em>Nr4a2</em> expression levels, were minimally affected, if at all. In CLA neurons, many of the dysregulated genes were associated with cell type identity, including the downregulation of numerous canonical CLA markers. Another research group, whose data was published while preparing this manuscript, also demonstrated a downregulation of CLA identity genes (though at the population level) in <em>Nr4a2-deficient</em> mice<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. Our results expand on these observations at the single-cell level. Building on our high-resolution atlas, we also show that upregulated genes in <em>Nr4a2</em><sup><em>del/wt</em></sup> CLA neurons were indicative of shell marker genes. These findings, along with predictions of a linear classifier, suggest that both shell and CLA neuronal populations are developmentally related, with <em>Nr4a2</em> transcription levels influencing, if not determining, their cell type identity in a dose-dependent manner. Unexpectedly, our results also indicate that NR4A2 does not directly modulate the expression of these identity genes, but instead orchestrates a cascade of other transcription factors that influence a broad array of target genes. Future studies should investigate whether CLA and shell neurons arise from a common pool of precursor cells and explore the roles of NR2F2 and CUX1 in defining their identities.</p>
<p id="Par33">Among genes modulated in <em>Nr4a2</em> haploinsufficient mice, we identified those coding for channels or channel subunits, such as <em>Scn1b</em> and <em>Ryr2</em>. <em>Scn1b</em> showed higher transcript levels in wild-type CLA neurons, whereas <em>Ryr2</em> was more prevalent in shell neurons, suggesting differing firing properties between these neurons. Several studies have reported the existence of distinct groups of CLA neurons characterized by varied electrophysiological profiles<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>–<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Moreover, the projection sites and functional effects of CLA neurons in the cortex remain controversial, with studies reporting disynaptic inhibition of deep-layer cortical neurons and others excitation of upper-layer neurons<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Whether these functional subtypes of neurons correspond to CLA and shell neurons remains to be determined, but these results constitute an important first step in that direction.</p>
<p id="Par34">We observed that <em>Nr4a2</em> haploinsufficiency alters the spiking behavior of CLA neurons, identified both genetically (<em>Slc17a6</em><sup><em>+</em></sup>) and by their projections to the mPFC. Specifically, <em>Nr4a2</em><sup>del/wt</sup> CLA neurons exhibited reduced firing frequency. To understand the molecular basis of this hypoexcitability, we examined the link between this phenotype and the transcriptomic dysregulation of genes encoding ion channels, neurotransmitter receptors, and signaling proteins, including G-protein subunits such as Gnb4, a proposed CLA marker. Our electrophysiological and pharmacological analyses suggest that changes in VGSCs, VGPCs, or downstream components of GPCR signaling—such as GIRK channels, the AC–cAMP–PKA cascade, or IP3 receptor signaling—are unlikely to be the primary drivers of the observed excitability deficits. Instead, our data point to upregulation of <em>Ryr2</em> as a key factor: enhanced CICR via RyR2 appears to increase activation of BK channels, leading to a larger afterhyperpolarization (AHP) and reduced firing frequency. Importantly, we acknowledge that the dysregulation of additional genes encoding voltage-gated ion channels, receptors, and associated signaling molecules could exert cumulative or context-dependent effects on excitability. While our ex vivo slice recordings isolate intrinsic properties, further alterations may manifest in vivo under conditions of network activity, neuromodulation, or behavioral engagement. These indirect or state-dependent mechanisms may further shape CLA neuron function and merit investigation in future studies.</p>
<p id="Par35">What are the behavioral implications of <em>Nr4a2</em> haploinsufficiency-mediated alterations in the claustro-insular region? Several studies have reported that alterations in the excitation of CLA projection neurons lead to behavioral deficits in mice, particularly affecting higher-order cognitive functions<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>–<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>. We demonstrate that the overexpression of <em>Ryr2</em> in <em>Nr4a2</em> heterozygous mice is responsible for the hypoexcitability of CLA projection neurons. Given that <em>Nr4a2</em> coding variants are associated with neurodevelopmental disorders<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>–<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>, including schizophrenia<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>,<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>, and considering the electrophysiological alterations we observed, we hypothesize that these functional impairments may contribute to specific cognitive deficits, rather than broad or nonspecific impairments arising from dysfunction in upstream or downstream brain regions connected to the CLA.</p>
<p id="Par36">In summary, we developed a high-resolution map of the claustro-insular region, providing a nuanced understanding of the CLA, shell, and cortical populations. This work lays the foundation for future studies aimed at exploring the developmental origins of CLA and shell neurons and their roles in higher-order cognitive functions, particularly in the context of neurodevelopmental disorders.</p></section><section id="Sec11"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec12"><h3 class="pmc_sec_title">Animals</h3>
<p id="Par37">All experiments followed veterinary guidelines and regulations of the University and the state of Geneva (Direction de l’expérimentation animale de l’UNIGE). Male C57BL/6 J mice (5–7 weeks old) were purchased from Charles River Laboratories. Nr4a2-SA-IRES-Dre mice (<em>Nr4a2</em><sup><em>tm1(dreo)Hze</em></sup>, MGI: MGI: 6790280)<sup><a href="#CR83" class="usa-link" aria-describedby="CR83">83</a></sup> were generated by replacing the complete <em>Nr4a2</em> coding sequence with a splice acceptor–ires-Dre sequence. Heterozygous <em>Nr4a2</em> mice (<em>Nr4a2</em><sup><em>del/wt</em></sup>) were bred by crossing heterozygous and wild-type parents of the same genetic background. Wild-type littermates were designated <em>Nr4a2</em><sup><em>wt/wt</em></sup>. For electrophysiological recordings of genetically identified CLA neurons, <em>Slc17a6</em><sup><em>tm2(cre)Lowl</em></sup>/J mice (<em>Vglut2</em>-IRES-Cre; Jackson laboratory, strain #016963)<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a></sup> were crossed to <em>Nr4a2</em><sup>tm1(dreo)Hze</sup> mice. In Vglut2-IRES-Cre mice, the Cre coding sequence, placed after the stop codon, does not disrupt the <em>Slc17a6</em> gene. Upon arrival, mice were group-housed in standard type II cages with ad libitum food and water, maintained on a 12/12 h dark/light cycle, at 21–22 °C and 45–55% humidity.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Single-cell RNA sequencing</h3>
<section id="Sec14"><h4 class="pmc_sec_title">Tissue collection and dissociation</h4>
<p id="Par38"><em>Nr4a2</em><sup><em>wt/wt</em></sup> and <em>Nr4a2</em><sup><em>del/wt</em></sup> male (n = 4, 8–9 weeks and 3, 15 weeks old) and female (n = 3, 8–9 weeks and 3, 15 weeks old) littermate mice were used for scRNAseq experiments. For each age/gender combination, one or two <em>Nr4a2</em><sup><em>wt/wt</em></sup> mice were used for two <em>Nr4a2</em><sup><em>del/wt</em></sup> mice. Tissue collection and dissociation were performed following the methods described by Fodoulian et al.<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Mice were anesthetized with isoflurane and sacrificed by decapitation. Brains were immediately extracted and placed in ice-cold oxygenated artificial cerebrospinal fluid (ACSF) containing (in mM): 124 NaCl, 3 KCl, 2 CaCl<sub>2</sub>, 1.3 MgSO<sub>4</sub>, 26 NaHCO<sub>3</sub>, 1.25 NaH<sub>2</sub>PO<sub>4</sub>, 10 D-glucose with an osmolarity of 300 mOsm and pH at 7.4 when oxygenated with 95% O<sub>2</sub>, 5% CO<sub>2</sub>. Coronal sections of the brain spanning the anteroposterior localization of the CLA were cut at a thickness of 300 μm using a vibrating-blade microtome (Leica VT1000S; Leica Biosystems). Immediately after sectioning, the CLA and adjacent brain regions were microdissected under a stereomicroscope. Tissue dissociations were performed using the Papain Dissociation System (ref. <a href="https://www.ncbi.nlm.nih.gov/nuccore/LK003150" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">LK003150</a>; Worthington® Biochemical Corporation) according to the manufacturer’s protocol with slight modifications, using EBSS and medium with serum solutions adapted from Saxena et al.<sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup>. After microdissection, extracted tissues were kept in ice-cold oxygenated EBBS#1<sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup>. Tissues from each mouse were processed individually, with no pooling across different animals. Papain and DNase I vials provided with the kit were reconstituted using the oxygenated EBSS#1 solution. Tissues from each mouse were transferred to a 50 ml Falcon tube containing 5 ml of the papain/DNase I mix, and were incubated at 37 °C for 20 min in a rotating laboratory oven (ProBlot<sup>TM</sup> Hybridization Oven; Labnet International) at a speed of 8–10 rpm. Tissues were then gently triturated using a P1000 pipette and left to settle for 1 min. Cell suspensions were then passed through a 70 μm nylon cell strainer (ref. 352350, Falcon) to remove undigested tissue fragments, transferred to 15 ml Falcon tubes, and centrifuged (Centrifuge 5430R; Eppendorf) at 300 × <em>g</em> for 5 min. Supernatants were discarded, and cell pellets were resuspended with 3 ml of EBSS#2<sup><a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup> to stop digestion. In order to remove myelin debris, two discontinuous density gradients were produced by carefully adding the 3 ml of cell suspension on top of 5 ml of an ovomucoid-albumin protease inhibitor solution reconstituted following the manufacturer’s instructions. Tubes were centrifuged at 70 g for 6 min at room temperature, the supernatants discarded, and the cell pellets resuspended in 1 ml medium solution with serum. Cell suspensions were then passed through another 70 μm nylon cell strainer.</p></section><section id="Sec15"><h4 class="pmc_sec_title">Fluorescence-activated cell sorting (FACS)</h4>
<p id="Par39">To ensure the exclusive isolation of live nucleated cells, cell suspensions were incubated with 2 μg/ml of Hoechst 33342 (a UV fluorescent adenine-thymine binding dye; ref. H1399, Life Technologies<sup>TM</sup>) at 37 °C for 15 min. In order to exclude dead cells, 1 μM of DRAQ7<sup>TM</sup> (a far-red fluorescent DNA intercalating dye; ref. DR71000, BioStatus) was added to the cell suspensions before FACS sorting. Hoechst<sup>+</sup>/DRAQ7<sup>-</sup> cells were then sorted in an empty Eppendorf tube according to their forward scatter (FSC) and side scatter (SSC) properties using a Beckman Coulter MoFlo Astrios cell sorter with a 100 μm nozzle at a pressure of 25 psi. Doublets were excluded after gating on FSC-A/FSC-H, followed by SSC-H/SSC-W. Between 11,000 and 25,000 Hoechst<sup>+</sup>/DRAQ7<sup>-</sup> cells were collected from each sample.</p></section><section id="Sec16"><h4 class="pmc_sec_title">Single-cell capture, cDNA library preparation, and RNA sequencing</h4>
<p id="Par40">Single-cell capture, reverse transcription, cDNA amplification, and library preparation were performed using the 10x Genomics Chromium Controller instrument and the Single Cell 3′ v3.1 reagent kit with dual indexes, following the manufacturer’s protocol (<a href="https://www.ncbi.nlm.nih.gov/nuccore/CG000315" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CG000315</a> Rev B User Guide). Cells were loaded on the 10x Genomics Chromium Controller instrument at a concentration of 400 cells/μl, and the targeted cell recovery was adjusted for each sample based on the number of sorted cells. Eleven rounds of PCR amplification were conducted after reverse transcription, and 13–14 rounds during library preparation. After the first round of PCR, cDNA quality and quantity were assessed using the Agilent 2100 Bioanalyzer with the Agilent Bioanalyzer High Sensitivity DNA Assay Kit (ref. 5067-4626; Agilent Technologies). After library preparation, cDNA library molarity and quality were assessed using the Qubit 4.0 with the Qubit™ 1× dsDNA HS Assay Kit (ref. <a href="https://www.ncbi.nlm.nih.gov/protein/Q33231" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q33231</a>; Thermo Fisher Scientific) and the TapeStation with the Agilent D1000 ScreenTape (ref. 5067-5582; Agilent Technologies). Libraries from mice of the same age were then pooled and each loaded at 2 nM on 4 lanes for clustering on a paired-end Illumina Flow Cell using the HiSeq 4000 PE Cluster Kit (ref. PE-410-1001; Illumina) and sequenced on an Illumina HiSeq 4000 sequencer using the HiSeq 4000 SBS Kit (ref. FC-410-1001; Illumina) chemistry. Read 1 consisted of 28 bases, while read 2 was either 90 or 100 bases. An average of 5171 cells were sequenced with 42,199 mean reads, 1125 median genes, and 1999 median UMI counts per cell.</p></section></section><section id="Sec17"><h3 class="pmc_sec_title">scRNAseq data analysis</h3>
<section id="Sec18"><h4 class="pmc_sec_title">Demultiplexing</h4>
<p id="Par41">Demultiplexing of Illumina indices was performed on the raw Illumina data using bcl2fastq2 Conversion Software version 2.20.0 (Illumina), which generated a pair of fastq files per sample (i.e., reads 1 and 2).</p></section><section id="Sec19"><h4 class="pmc_sec_title">Mapping</h4>
<p id="Par42">Digital gene expression matrices (i.e., matrices with genes in rows, cells in columns, and gene counts as matrix entries) were generated using Cell Ranger version 6.0.0<sup><a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup>. Fastq files were mapped to the <em>Mus musculus</em> genome primary assembly reference 38 (GRCm38) using STAR<sup><a href="#CR87" class="usa-link" aria-describedby="CR87">87</a></sup>, implemented in Cell Ranger. A modified version of Ensembl release 102 of the <em>Mus musculus</em> GTF annotation was used, with the <em>Gm45623</em> annotation replaced by a custom annotation for <em>Smim32</em>, as described in Tuberosa et al.<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Mapping, UMI counting, and barcode filtering were performed using the count function of Cell Ranger. The expected number of recovered cells (--expect-cells argument) was customized for each sample and set to the targeted cell recovery of the sample.</p></section><section id="Sec20"><h4 class="pmc_sec_title">Data filtering</h4>
<p id="Par43">scRNAseq data analyses were performed using R version 4.3.1<sup><a href="#CR88" class="usa-link" aria-describedby="CR88">88</a></sup>. The filtered feature-barcode matrices from the Cell Ranger output served as the initial files for downstream analyses. Three rounds of clustering (following the steps detailed below) were applied to the data. The datasets at each round will be named “all cells” (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>), “neurons unfiltered” (Supplementary Fig. <a href="#MOESM1" class="usa-link">1d</a>), and “neurons filtered” (Fig. <a href="#Fig1" class="usa-link">1b</a>) in the rest of the “Methods” section. Cells were iteratively filtered to retain high-quality cells. After the initial clustering step of the “all cells” dataset, only neuronal clusters were retained for further analysis. These cells (forming the “neurons unfiltered” dataset) were re-clustered to remove clusters formed by low-quality cells or doublets that could not be detected in the full dataset. A final filtering step was applied to the cells, following the criteria detailed by Fodoulian et al.<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Briefly, we removed all cells (1) that were characterized by fewer than 1000 expressed genes, (2) for which mitochondrial counts exceeded 10% of their total counts, as high mitochondrial counts indicate suffering or dead cells, (3) for which the total number of detected genes, total number of UMIs, and percentage of mitochondrial counts were three median absolute deviations away from the median after log<sub>10</sub> transformation, and (4) that showed unusually high or low numbers of genes given their total number of UMIs, after log<sub>10</sub> transformation, by fitting a loess curve (<em>loess</em> function of the stats R package, with span = 0.5 and degree = 2), where the number of genes was taken as the response variable and the number of UMIs as the predictor. Cells for which the model residual was not within three median absolute deviations of the median were filtered out. The final number of cells in each dataset was 67,226 for “all cells”, 18,386 for “neurons unfiltered”, and 14,852 for “neurons filtered”. After cell filtering, we also selected genes expressed in at least three cells separately in each sample, with a minimum expression threshold of 1 UMI count. This filtering resulted in a total of 27,453 genes for “all cells”, 24,549 for “neurons unfiltered”, and 23,916 for “neurons filtered”.</p></section><section id="Sec21"><h4 class="pmc_sec_title">Selection of high dropout genes</h4>
<p id="Par44">In order to identify distinct cellular populations, we selected genes for downstream analyses based on their mean expression and dropout-rate relationship using the M3Drop R package version 3.10.6<sup><a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup>. We fitted a depth-adjusted negative binomial (DANB) model on the raw counts of each sample using the <em>NBumiFitModel</em> function of M3Drop. This model accounts for differences in sequencing depth and UMI tagging efficiency between cells. Genes with significantly higher dropout rates than expected by the model were selected for dimensionality reduction and clustering using the <em>NBumiFeatureSelectionCombinedDrop</em> function of M3Drop (<em>p</em>-value &lt; 0.05), adjusted for multiple comparisons using the Benjamini–Hochberg false discovery rate method<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a></sup>. This selection corresponded to a total of 3860 high dropout genes for “all cells”, 2316 for “neurons unfiltered”, and 1780 for “neurons filtered”.</p></section><section id="Sec22"><h4 class="pmc_sec_title">Data normalization and confounder regression</h4>
<p id="Par45">To account for technical variability (i.e., tissue dissociation effects and sequencing depth differences between cells) in downstream analyses, a variance stabilizing transformation was applied to the raw counts of each sample using the <em>SCTransform</em> function of the Seurat R package version 5.1.0<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>–<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a></sup>. This function implements the <em>vst</em> function of the sctransform R package (version 0.4.1 was used)<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>–<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a></sup>. All cells and genes were used for initial negative binomial regression parameter estimation, with the method set to <em>glmGamPoi_offset</em> (implemented using the glmGamPoi R package version 1.12.2)<sup><a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>,<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a></sup>. The Pearson residuals from the second regularized negative binomial model were computed for all genes and clipped to a range of [−<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e2552"><msqrt><mrow><mi>n</mi><mo>/</mo><mn>30</mn></mrow></msqrt></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e2565"><msqrt><mrow><mi>n</mi><mo>/</mo><mn>30</mn></mrow></msqrt></math></span>]. To correct for confounding variables, a final linear model was fitted to the Pearson residuals to regress out the number of UMIs per gene and the percentage of mitochondrial counts for each cell, and the residuals from this model (i.e., corrected Pearson residuals) were centered at the mean. Corrected counts and their log1p transformation were also computed. The process described here was applied either separately to each sample, with the resulting data used for dimensional reduction, clustering, or differential expression analysis - this latter after re-correcting UMI counts using the minimum of the median UMIs across samples via the <em>PrepSCTFindMarkers</em> function of Seurat - or applied to all cells together for the analysis of cell type identity shifts, gene regulatory network inference, and reference-based mapping on the “neurons filtered” dataset.</p>
<p id="Par46">For gene expression plotting, raw gene expression values were normalized by the total number of counts per cell and scaled to 10<sup>4</sup> (i.e., corresponding to norm. UMI) using the <em>NormalizeData</em> function of Seurat. Plotted gene expression values are actual normalized values, but the scales are in log<sub>10</sub> after adding a pseudocount of 1.</p></section><section id="Sec23"><h4 class="pmc_sec_title">Dimensionality reduction, dataset integration, and unsupervised graph-based clustering</h4>
<p id="Par47">Prior to cell clustering, principal component analysis (PCA) was performed on the corrected Pearson residuals of the high dropout genes (see “Selection of high dropout genes” section<span class="text-underline">)</span> using the irlba R package version 2.3.5.1<sup><a href="#CR95" class="usa-link" aria-describedby="CR95">95</a></sup>, implemented in the <em>RunPCA</em> function of Seurat. Only the first 100 PCs were computed. The elbow in the decrease of the explained standard deviation of the successive PCs was detected using the <em>KneeLocator</em> function of the kneed Python package version 0.8.5<sup><a href="#CR96" class="usa-link" aria-describedby="CR96">96</a></sup> with the following parameters: S = 1, curve = “convex”, direction = “decreasing”. The location of the elbow was then used as a threshold to retain the top relevant PCs for downstream processing of the datasets. This resulted in retaining 19 PCs for “all cells”, 20 for “neurons unfiltered”, and 19 for “neurons filtered” (explaining 82%, 78%, and 77% of the variance calculated with the top 100 PCs, respectively). These PC embeddings were subsequently used for dataset integration through the <em>RunHarmony</em> function of the harmony R package version 1.2.0<sup><a href="#CR97" class="usa-link" aria-describedby="CR97">97</a></sup>. The ridge regression penalty was automatically estimated (with lambda = NULL), the maximum number of Harmony rounds was set to 50, the maximum number of clustering rounds at each Harmony round was set to 200, and the convergence tolerance values for clustering and Harmony rounds were set to 1e-5. Harmony-corrected cell embeddings were then used for clustering and Uniform Manifold Approximation and Projection (UMAP)<sup><a href="#CR98" class="usa-link" aria-describedby="CR98">98</a>,<a href="#CR99" class="usa-link" aria-describedby="CR99">99</a></sup> dimensional reduction. To construct a shared nearest-neighbor (SNN) graph, 15 approximate nearest neighbors were identified for each cell using the RcppAnnoy R package version 0.0.22, implemented in the <em>FindNeighbors</em> function of Seurat. A forest of 50 trees was built for the nearest-neighbor search, using the Euclidean distance metric. The ranked nearest neighbors were then used to construct the SNN graph, and a Jaccard index cutoff of 1/15 was applied to prune weak connections from the graph. Cells were clustered using the leidenalg Python package version 0.10.2<sup><a href="#CR100" class="usa-link" aria-describedby="CR100">100</a></sup>, implemented in the <em>FindClusters</em> function of Seurat. The RBConfigurationVertexPartition singleton partition type was used, implementing Reichardt and Bornholdt’s Potts model with a null configuration<sup><a href="#CR101" class="usa-link" aria-describedby="CR101">101</a>,<a href="#CR102" class="usa-link" aria-describedby="CR102">102</a></sup>. The Leiden algorithm was run until no improvement in iteration was found (with n.iter = −1L). If singleton clusters were found, these were grouped into the same cluster. Clustering resolution selection varied by dataset. For the “all cells” and “neurons unfiltered” datasets, clustering resolution parameters ranged from 0 to 1 in increments of 0.1, and the lowest resolution that allowed segregation of large “noise” clusters was selected. For the “neurons filtered” dataset, the variation of information (VI)<sup><a href="#CR103" class="usa-link" aria-describedby="CR103">103</a></sup> between successive resolutions (ranging from 0 to 1.5 in increments of 0.1) was computed and normalized by the absolute difference in the number of clusters, and the resolution with the highest normalized VI was chosen. VI was computed using the <em>compare</em> function of the igraph R package version 2.0.3. These steps resulted in clustering resolutions of 0.3 for “all cells”, 0.2 for “neurons unfiltered”, and 1.2 for “neurons filtered”. UMAP dimensional reduction was computed using the uwot R package version 0.1.16, implemented in the <em>RunUMAP</em> function of Seurat. Since 15 nearest neighbors were identified for each cell for SNN graph construction, 15 neighboring points were used during the local approximation of the manifold structure, and the cosine metric was used to measure distances. Note that UMAP was solely used to visualize the cluster assignment of cells on a 2-dimensional plot.</p></section><section id="Sec24"><h4 class="pmc_sec_title">Differential expression analysis</h4>
<p id="Par48">To identify cell type and subtype marker genes, or genes modulated due to <em>Nr4a2</em> haploinsufficiency, a two-sided Wilcoxon rank sum test was performed on the corrected, log1p-transformed counts, implemented in the <em>FindMarkers</em> function of Seurat. When identifying cell type marker genes, differential expression analysis was computed between each cell type and all other cell types combined. For cell subtype marker genes, pairwise comparisons between related cell types were conducted. To identify modulated genes, comparisons within cell types or subtypes were performed, comparing <em>Nr4a2</em><sup><em>wt/wt</em></sup> cells to <em>Nr4a2</em><sup><em>del/wt</em></sup> cells. Genes expressed in at least 10% of the cells in either group, with a log fold-change of 0.25 and an adjusted <em>p</em>-value &lt; 0.05 using the Bonferroni method, were considered significant. To test for robustness in the obtained results, a leave-one-out approach was implemented, where a given sample was removed from the dataset prior to differential expression analysis. To assess false-positive results, cell type, subtype, or genotype labels were permuted before conducting the differential expression analysis (n = 1000 permutations).</p></section><section id="Sec25"><h4 class="pmc_sec_title">Analysis of cell type identity shifts</h4>
<p id="Par49">To study shifts in cell type identities due to <em>Nr4a2</em> haploinsufficiency, classification and prediction of cell type identities based on the corrected Pearson residuals of modulated genes were performed using the <em>LinearSVC</em> function of the scikit-learn Python package version 1.3.2<sup><a href="#CR104" class="usa-link" aria-describedby="CR104">104</a>,<a href="#CR105" class="usa-link" aria-describedby="CR105">105</a></sup>. Only CLA, shell, and L6a neurons from the “neurons filtered” dataset were used in this analysis. Linear classifier parameters were optimized with <em>Nr4a2</em><sup><em>wt/wt</em></sup> cells using cross-validated grid search via the <em>GridSearchCV</em> function of scikit-learn. The optimized parameters were “C” for regularization strength, “penalty” for the norm type, and “dual” for the optimization problem formulation. This search was conducted over 10-fold cross-validation iterations, and the performance of the cross-validated model was evaluated by computing the balanced accuracy, resulting in the following optimized values: C = 9e-04, penalty = “l2”, and dual = “auto”. The tolerance value for the stopping criteria was set to 1e-5, and the maximum number of iterations was set to 50,000. Linear classifier models were trained with the optimized parameters described above using 80% of the <em>Nr4a2</em><sup><em>wt/wt</em></sup> cells, and cell type labels were predicted for the remaining 20% of <em>Nr4a2</em><sup><em>wt/wt</em></sup> cells and all <em>Nr4a2</em><sup><em>del/wt</em></sup> cells. To obtain a balanced partition of the data prior to model training, the <em>partition</em> function of the groupdata2 R package version 2.0.3 was used. Cell type classification performance was tested over 1000 iterations. Significant differences between the genotypes in the percentage of classified cells across iterations for each actual-predicted cell type label pair were assessed using a generalized linear model (GLM)<sup><a href="#CR106" class="usa-link" aria-describedby="CR106">106</a>–<a href="#CR108" class="usa-link" aria-describedby="CR108">108</a></sup> with a binomial distribution. Models were fitted via the glmmTMB R package version 1.1.9<sup><a href="#CR109" class="usa-link" aria-describedby="CR109">109</a>–<a href="#CR111" class="usa-link" aria-describedby="CR111">111</a></sup>, using Template Model Builder (TMB). No random effect term was specified. Likelihood ratio tests (LRT) with χ<sup>2</sup>-based <em>p</em>-values were then computed by comparing the resulting models with null models where the fixed effects term (i.e., genotype) was dropped, using the <em>drop1</em> function of the stats R package<sup><a href="#CR112" class="usa-link" aria-describedby="CR112">112</a></sup>. To correct for multiple comparisons, <em>p</em>-values were adjusted using the Holm method<sup><a href="#CR113" class="usa-link" aria-describedby="CR113">113</a></sup>. To test for false negative results, permutation of cell identities was performed prior to model training, and accuracy differences between genotypes were compared to the observed results.</p>
<p id="Par50">To assess whether modulated genes are important for the discrimination of CLA, shell, and L6a neurons, a linear classifier was trained on <em>Nr4a2</em><sup><em>wt/wt</em></sup> cells using genes expressed in at least 3 cells. Linear classifier parameters were set to the values described above. Since the multiclass linear classifier is implemented following a one-vs-the-rest scheme, gene weights were computed as the sum of the absolute weights across all one-vs-the-rest classifiers. The difference in gene weights between modulated genes and all other genes was tested using a two-sided Wilcoxon rank sum test with continuity correction. Genes whose weights were at least three median absolute deviations away from the median of all gene weights were considered as high-weighted genes. A two-sided one-sample proportions test with continuity correction was then used to test whether modulated genes were enriched among high-weighted genes compared to their representation in the transcriptome.</p></section><section id="Sec26"><h4 class="pmc_sec_title">Gene regulatory network inference</h4>
<p id="Par51">To investigate alterations in transcription factor activity due to <em>Nr4a2</em> haploinsufficiency, gene regulatory network inference was conducted on the corrected, log1p-transformed counts of genes expressed in at least 3 cells using the pySCENIC Python package version 0.12.1<sup><a href="#CR114" class="usa-link" aria-describedby="CR114">114</a>,<a href="#CR115" class="usa-link" aria-describedby="CR115">115</a></sup>. Only CLA, shell, and L6a neurons from the “neurons filtered” dataset were used in this analysis. Motif2TF annotations and cisTarget databases generated with the 2022 SCENIC+<sup><a href="#CR116" class="usa-link" aria-describedby="CR116">116</a></sup> motif collection were utilized (mm10 annotation files). The list of transcription factors was filtered to include only those modulated in <em>Nr4a2</em><sup><em>del/wt</em></sup> mice. Genes lacking ranking values in the “genes versus motifs” ranking files were also excluded from the analysis. Gene regulatory networks (i.e., transcription factor-target gene co-expression modules) were inferred from the expression of genes using the GRNBoost2 algorithm<sup><a href="#CR117" class="usa-link" aria-describedby="CR117">117</a></sup> via the <em>grnboost2</em> function of the arboreto Python package version 0.1.6<sup><a href="#CR114" class="usa-link" aria-describedby="CR114">114</a></sup>, with an early stop window length set to 100. Regulons (i.e., gene targets of a given transcription factor) were then derived from the co-expression modules using the <em>modules_from_adjacencies</em> function of pySCENIC, with the following criteria: gene targets with importance values above the 90th and 95th percentiles were retained; the top 25 and 50 targets of each transcription factor were retained; and the targets for which the transcription factor is within its top 5, 10, and 50 regulators were retained. Activating and repressing regulons were inferred using a Pearson correlation coefficient threshold of 0.03, calculated using all cells. Only regulons containing at least 20 target genes were kept for further analysis. Regulons were pruned, and target genes with a cis-regulatory footprint for the corresponding transcription factor were retained using the “genes versus motifs” ranking files via the <em>prune2df</em> function of pySCENIC. Recovery curves were created using the top-800 ranked target genes of each transcription factor, and the area under the curve was calculated using 2.5% of the ranked genome. Regulons with a normalized enrichment score above 2.1 were retained, and regulon enrichment scores were calculated for each cell using the <em>aucell</em> function of pySCENIC. To identify regulons enriched in either <em>Nr4a2</em><sup><em>wt/wt</em></sup> or <em>Nr4a2</em><sup><em>del/wt</em></sup> cells, regulon-specific scores (RSS)<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup> were calculated for each cell type and genotype using the <em>regulon_specificity_scores</em> function of pySCENIC, and the contrast in these scores was computed between genotypes.</p></section><section id="Sec27"><h4 class="pmc_sec_title">Claustro-insular cellular composition analysis</h4>
<p id="Par52">Differences between genotypes in cell type proportions were tested using the <em>propeller</em> function of the speckle Bioconductor package<sup><a href="#CR118" class="usa-link" aria-describedby="CR118">118</a></sup> version 1.0.0. This package implements functions from the limma Bioconductor package (version 3.56.2 was used)<sup><a href="#CR119" class="usa-link" aria-describedby="CR119">119</a></sup>. The mouse from which each cell was derived served as a biological replicate. A linear model was fitted to the logit-transformed proportions, and a robust empirical Bayes shrinkage<sup><a href="#CR120" class="usa-link" aria-describedby="CR120">120</a></sup> of the variances was applied. Since there are two genotypes, a <em>t</em>-test was used to calculate <em>p</em>-values, which were adjusted for multiple comparisons using the Benjamini–Hochberg false discovery rate method<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a></sup>.</p></section></section><section id="Sec28"><h3 class="pmc_sec_title">MERFISH</h3>
<section id="Sec29"><h4 class="pmc_sec_title">MERFISH gene panel design</h4>
<p id="Par53">For the MERFISH<sup><a href="#CR121" class="usa-link" aria-describedby="CR121">121</a></sup> gene panel, marker genes that effectively segregate neuronal cell populations in the mouse claustro-insular region were selected. Criteria included: (1) markers of major claustro-insular cell types, (2) differentially expressed genes segregating related cell types, (3) previously described claustro-insular markers, and (4) variable genes in the scRNAseq data. Generic markers such as <em>Slc17a7, Slc32a1, Pvalb, Sst, Vip</em>, and <em>Gfap</em> were also added. A list of 500 genes was submitted to the Vizgen portal; genes with insufficient target regions or exceeding abundance thresholds were excluded to prevent optical crowding. A final panel of 300 genes was used.</p></section><section id="Sec30"><h4 class="pmc_sec_title">Sample preparation</h4>
<p id="Par54">Mice (~12 weeks old; n = 2 C57BL6 male, n = 3 <em>Nr4a2</em><sup><em>wt/wt</em></sup> males, and n = 3 <em>Nr4a2</em><sup><em>del/wt</em></sup> males) were anesthetized with 4% isoflurane (Attane<sup>TM</sup>, Piramal Healthcare) and euthanized by cervical dislocation. Brains were removed, embedded in OCT (ref. KMA-0100-00A, CellPath Ltd), frozen in an isopentane and liquid nitrogen bath, and stored at −80 °C until sectioning.</p></section><section id="Sec31"><h4 class="pmc_sec_title">Cryosections</h4>
<p id="Par55">Brains were sectioned on a Leica CM3050 cryostat (chamber: −21 °C, sample: −16 °C). Brains were placed in the cryostat for at least 30 min before starting sectioning. The OCT block was trimmed until reaching the beginning of the claustro-insular region. After alignment using the Allen Mouse Brain Atlas, 10 μm sections (approx. bregma +1.045, +0.145, and −1.355) were mounted onto MERSCOPE slides (ref. PN 20400001, Vizgen), air-dried in the cryostat for 10–15 min. RNA quality was checked using additional sections, stored in 350 μl RLT Plus lysis buffer (ref. 74034, Qiagen) at 4 °C until analysis.</p></section><section id="Sec32"><h4 class="pmc_sec_title">RNA extraction and quality control</h4>
<p id="Par56">RNA was extracted using the RNeasy Plus Micro Kit (Qiagen) per protocol, eluted with 14 μl RNAse-free water, and analyzed on the 2100 Bioanalyzer (RNA Nano Chips, ref. 5067-1511, Agilent). All samples showed RIN = 10 and DV200 = 98–100%, confirming RNA integrity.</p></section><section id="Sec33"><h4 class="pmc_sec_title">MERFISH sample preparation and hybridization</h4>
<p id="Par57">Slides were fixed in 4% PFA in 1× PBS for 15 min at room temperature, washed in 1× PBS, covered in 70% EtOH, sealed in parafilm, and stored at 4 °C (maximum 6 weeks). For hybridization, Vizgen protocols were followed. Verification was done on adjacent sections using the Sample Verification Kit (ref. 10400008, Vizgen). The hybridization procedure was the same for verification and experimental samples. Slides were washed and incubated for 30 min at 37 °C in Formamide Wash Buffer (ref. 20300002, Vizgen) in an incubator. Verification Gene Probes or the custom gene panel mix (50 μl) were applied to sections, covered with parafilm, and incubated at 37 °C for 18–24 h. Sections were incubated twice in Formamide Wash Buffer for 30 min at 47 °C. Slides were washed with Sample Prep Wash Buffer for 2 min. A Gel Coverslip (ref. 20400003, Vizgen) was coated with 100 μl of Gel Slick Solution. Wash buffer was removed from the sample, which was then covered with a thin layer of Gel Embedding Solution (ref. 20300004, Vizgen) containing 10% ammonium persulfate solution and N,N,N′,N′-tetramethylethylenediamine. The Gel Coverslip was placed on top of the sample, which was incubated at room temperature for approximately 90 min, until the gel was solidified. For tissue clearing, after removing the Gel Coverslip, slides were incubated in 5 ml warmed Clearing Solution (ref. 20300003, Vizgen) with Proteinase K (ref. P8107S, NEB) for 24 h at 37 °C.</p></section><section id="Sec34"><h4 class="pmc_sec_title">MERFISH imaging</h4>
<p id="Par58">Sections were washed twice for 5 min in Sample Prep Wash Buffer and incubated for 15 min on a rocker with either 3 ml of Verification Staining Reagent (ref. 20300014, Vizgen, for verification samples) or 3 ml of DAPI and PolyT Staining Reagent (ref. 20300021, Vizgen, for experimental samples). Sections were washed with 5 ml of Formamide Wash Buffer for 10 min and then placed in Sample Prep Wash Buffer. For verification samples, slides were placed in the MERSCOPE Flow Chamber and Imaging Buffer (ref. 20300016, Vizgen), with Verification Imaging Buffer Activator (ref. 20300015, Vizgen) was injected into the chamber, which was inserted into the MERSCOPE instrument (ref. 10000001, Vizgen) and imaged according to the manufacturer’s protocol. For imaging of samples labeled with the 300-gene panel, the MERSCOPE Imaging Cartridge (ref. 20300018, Vizgen) was prepared by injecting 250 μl of Imaging Buffer activator (ref. 20300022, Vizgen) mixed with 100 μl of RNase inhibitor (ref. M0314L, NEB) into the cartridge. Next, 15 ml of mineral oil (ref. M5904-500ML, Sigma) was pipetted on top of the Imaging Activation Mix. The cartridge was inserted into the MERSCOPE instruments, and the fluidics system was primed according to instructions. The sample was placed in the Flow Chamber, inserted into the MERSCOPE instrument, and connected to the fluidics system. After acquiring a low-resolution image of DAPI labeling, the region of interest was drawn, and imaging was initiated. Sections were imaged with a 60× lens and 7 planes on the z-axis.</p></section></section><section id="Sec35"><h3 class="pmc_sec_title">MERFISH data analysis</h3>
<section id="Sec36"><h4 class="pmc_sec_title">MERFISH data preprocessing</h4>
<p id="Par59">Raw images of entire coronal sections were automatically processed by the MERSCOPE Instrument software, generating cell-by-gene and cell-metadata output matrices for each sample. Regions of interest, including cortical deep layers, endopiriform nucleus, piriform cortex, and striatum, were selected around the claustro-insular regions of each section for downstream analyses using R version 4.3.1. Three brain hemisections of wild-type mice were removed from the analysis because one detached from the slide during the imaging procedure (bregma +1.045), and two were out of focus (bregma +0.145 and −1.355). The number of expressed genes and mRNA molecules was computed for each cell. Cells with a volume of 500–3500 µm<sup>3</sup> (inclusive) and containing at least 10 but no more than 3000 mRNA molecules were retained for further analysis. To filter out non-neuronal cells, clustering was performed on the cells following the same pipeline used for scRNAseq data analysis, with the following differences: (1) high dropout genes were not computed, (2) Pearson residuals were clipped to a range of [−10, 10] and the effect of cell size on the number of mRNA molecules was corrected by regressing out cell volume, (3) PCA was performed on the corrected Pearson residuals of all 300 genes, (4) 21 PCs were used for dataset integration, and (5) a clustering resolution of 0.1 was used for cluster identification. Clusters corresponding to non-neuronal cells, typically not expressing <em>Slc17a7</em> and <em>Slc32a1</em>, were removed from downstream analysis. No genes were filtered for downstream analysis.</p></section><section id="Sec37"><h4 class="pmc_sec_title">Projection of MERFISH data on the scRNAseq data</h4>
<p id="Par60">To identify cell types in the MERFISH dataset, a reference-based mapping approach was employed, where cell types were predicted using the scRNAseq dataset as a reference. To build this reference, supervised PCA (SPCA)<sup><a href="#CR122" class="usa-link" aria-describedby="CR122">122</a></sup> was performed on the corrected Pearson residuals of the MERFISH panel genes expressed in the “neurons filtered” dataset (296 genes were used) using the <em>RunSPCA</em> function of Seurat. The linear matrix factorization during SPCA calculation was supervised using the SNN graph from the “neurons filtered” dataset. Only the first 100 SPCs were computed, and the top relevant SPCs were selected using the <em>KneeLocator</em> function of kneed (see “scRNAseq data analysis” and “Dimensionality reduction, dataset integration, and unsupervised graph-based clustering” sections). An Azimuth reference was created using the variance-stabilized data, the top SPCs, and the UMAP model from the “neurons filtered” dataset via the <em>AzimuthReference</em> function of the Azimuth R package version 0.5.0<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. The cell type and subtype identities of cells were used for prediction in the MERFISH dataset. Prior to dataset projection, a variance stabilizing transformation was applied to the raw counts of all filtered MERFISH cells using the <em>SCTransform</em> function of Seurat<sup><a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>,<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a></sup>, as described in the “Data normalization and confounder regression” section, with the following differences: Pearson residuals were clipped to a range of [−10, 10], and the effect of cell size on the number of mRNA molecules was corrected by regressing out cell volume. This variance-stabilized data was then projected onto the scRNAseq reference dataset, and cell type and subtype label prediction was performed via the <em>RunAzimuth</em> function of Azimuth.</p>
<p id="Par61">For visualizing the 2D brain maps of predicted cell types and subtypes, the rotation angle for each section was visually estimated, and the rotated coordinates were computed using a rotation matrix.</p></section><section id="Sec38"><h4 class="pmc_sec_title">Characterization of differences in CLA and EP cellular composition and gene expression</h4>
<p id="Par62">To characterize differences between CLA projection neurons located in the CLA, EP, or L6, a joint analysis of the scRNAseq and MERFISH datasets was performed. First, a region identity (namely, CLA, EP, or CLA-like L6) was attributed to CLA cells from the MERFISH dataset with a minimum prediction score of 0.7, based on their anatomical localization. The CLA-like L6 region was assigned to dorsally located CLA neurons from the bregma −1.355 hemisection, differentiating them from more ventrally located CLA and EP cells, which in this section could not be easily segregated. Differences between CLA and EP in cell type proportions were tested using the <em>propeller</em> function of speckle, as described in the “Claustro-insular cellular composition analysis” section, with the following difference: the region of interest from which each cell was derived served as a biological replicate, and a non-robust empirical Bayes shrinkage of the variances was applied.</p>
<p id="Par63">To identify differentially expressed genes between neurons from these regions, the MERFISH dataset could not be used because: (1) it contains expression values for only 300 genes, and (2) some genes exhibit spurious expression with this technique (mainly due to probe design). Instead, the scRNAseq dataset was used, as it includes expression measurements for thousands of genes per cell. Because scRNAseq lacks spatial information, the initial localization—representing the likely region of origin—of CLA cells in this dataset was inferred by examining how their nearest neighbors from the MERFISH dataset were distributed across subregions of the CLA/EP complex. The list of nearest neighbors from the reference-based mapping step—where 20 nearest neighbors from the scRNAseq dataset were identified for each cell in the MERFISH dataset—was used for this analysis. A one-sample proportions test with continuity correction was then applied for each CLA cell from the scRNAseq dataset. Cells displaying inequalities in their proportion distribution of nearest neighbors between subregions of the CLA/EP complex, compared to expected proportions calculated from the MERFISH dataset, were assigned a region identity. To increase the statistical power of this analysis, MERFISH samples from both <em>Nr4a2</em><sup><em>wt/wt</em></sup> and <em>Nr4a2</em><sup><em>del/wt</em></sup> were used. From the 1192 <em>Nr4a2</em><sup><em>wt/wt</em></sup> CLA cells in the scRNAseq dataset, 31 were predicted to have originated from the CLA, 67 from the EP, and 2 caudally from the L6. Due to their low number, these latter were excluded from further analysis. Differentially expressed genes between CLA projection neurons with a predicted CLA or EP region attribute were finally computed using the <em>FindMarkers</em> function of Seurat, as described in the “Differential expression analysis” section.</p></section></section><section id="Sec39"><h3 class="pmc_sec_title">RNAscope smFISH</h3>
<section id="Sec40"><h4 class="pmc_sec_title">RNAscope sample preparation and hybridization</h4>
<p id="Par64">Mice (details in Supplementary Data <a href="#MOESM3" class="usa-link">3</a>) were anesthetized with isoflurane and euthanized by cervical dislocation. Brains were removed, embedded in OCT, frozen in an isopentane/liquid nitrogen bath, and stored at −80 °C until sectioning. Coronal sections (10 μm) were cut on a cryostat microtome, mounted on Superfrost™ Plus slides (ref. J1800AMNZ, Erpredia), dried for 1–2 h at −20 °C, and stored at −80 °C. Sections were labeled using the RNAscope™ Multiplex Fluorescent V2 Assay (ref. 323136, Advanced Cell Diagnostics) following the manufacturer’s guidelines for fresh-frozen tissues. Before hybridization, sections were post-fixed in 10% formalin (15 min, 4 °C), dehydrated, treated with hydrogen peroxide (10 min, room temperature), and incubated with Protease IV (15 min, room temperature). Sections were hybridized with sets of 2–3 probes (see Supplementary Data <a href="#MOESM3" class="usa-link">3</a>) for 2 h at 40 °C in a humidified chamber. Signals were developed using TSA Vivid Fluorophores 520 (1:750, ref. 323271), 570 (1:1000–1500, ref. 323272), and 650 (1:1500–2000, ref. 323273), diluted in TSA buffer. Sections were counterstained with DAPI and mounted with ProLong<sup>TM</sup> Gold antifade (ref. <a href="https://www.ncbi.nlm.nih.gov/protein/P36935" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P36935</a>, Invitrogen<sup>TM</sup>). Imaging was performed with a Nikon Ti/CSU-W1 Spinning Disc Confocal microscope (405, 488, 561, 640 nm lasers, 60 × 1.49 NA objective). Image tiles of the claustro-insular region were acquired in 6 z-planes, and merging and orthogonal projections were applied using the SlideBook software (Intelligent Imaging Innovations).</p></section><section id="Sec41"><h4 class="pmc_sec_title">RNAscope/Immunofluorescence assay</h4>
<p id="Par65">To characterize <em>Nr2f2</em> expression in the claustro-insular region, we performed dual RNAscope/immunofluorescence labeling to visualize both <em>Nr2f2</em> mRNA and NR2F2 protein. The procedure followed the manufacturer’s protocol for dual ISH-IHC assays (Advanced Cell Diagnostics). Briefly, after labeling sections for <em>Nr4a2</em> and <em>Nr2f2</em> mRNA using the RNAscope Multiplex Fluorescent V2 Assay, sections were incubated with a primary antibody against NR2F2 (1:250, ref. ab211775, Abcam), followed by an HRP-conjugated secondary antibody (1:500, ref. W4011, Promega). Signal was then developed using TSA Vivid Fluorophore 650 (1:750, ref. 323273).</p></section><section id="Sec42"><h4 class="pmc_sec_title">RNAscope image analysis</h4>
<p id="Par66">Images were analyzed using QuPath version 0.4.3. Cells were selected by drawing a region of interest (ROI) (see Supplementary Data <a href="#MOESM3" class="usa-link">3</a> for details). Cell segmentation based on DAPI staining was performed using the <em>Cell detection</em> function. The cell expansion parameter was adjusted depending on the analysis: for co-expression analyses (Figs. <a href="#Fig2" class="usa-link">2</a> and <a href="#Fig3" class="usa-link">3</a>), a 2 μm expansion was used to minimize false-positive co-expression by restricting the analysis to areas close to the nucleus. For the quantification of the expression of individual genes (Figs. <a href="#Fig4" class="usa-link">4</a> and <a href="#Fig6" class="usa-link">6</a>), a 4 μm expansion was applied to maximize transcript detection. In the particular case of <em>Nr2f2</em>, we observed widespread nuclear localization of transcripts across most cells in the cortex and striatum (as also seen in the Allen Brain Atlas ISH dataset; mouse.brain-map.org, experiment 308055507), except in <em>Nr4a2</em> + CLA/EP neurons, where nuclear and cytoplasmic <em>Nr2f2</em> transcripts were observed (Supplementary Fig. <a href="#MOESM1" class="usa-link">14i–o</a>). This reflects the CLA/EP-specific expression pattern of NR2F2 observed by IHC<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>. Therefore, only cytoplasmic <em>Nr2f2</em> puncta were quantified, as these likely represent the translated mRNA pool specific to CLA/EP cells. Next, puncta corresponding to individual mRNA molecules were automatically detected and quantified using the <em>Subcellular detection</em> function. For each fluorophore, a detection threshold was set, and all puncta above this threshold were counted. When multiple puncta were clustered together, an estimation of the number of individual puncta within the cluster was performed automatically by the <em>Subcellular detection</em> function. The same parameters were applied to all images from the same experiment. When referring to one section, this corresponds to an image of one claustro-insular region.</p></section></section><section id="Sec43"><h3 class="pmc_sec_title">RNAscope data analysis</h3>
<section id="Sec44"><h4 class="pmc_sec_title">Cluster identification</h4>
<p id="Par67">A custom R script was used for the analysis of RNAscope data on R version 4.3.1. Cells not expressing any of the three genes tested were removed. Two transformation steps were applied to the count data: (1) to account for outlier cells, the number of puncta per cell was log-transformed, (2) to eliminate gene-specific biases in the clustering of cells, gene-wise standardization of the data was applied by scaling and centering the numbers of puncta of a given gene across cells, using the <em>scale</em> function of the base R package. Euclidean distances were then computed between each pair of cells using the <em>dist</em> function of the stats R package. The data was finally clustered using Ward’s minimum variance agglomeration method implemented in the <em>hclust</em> function of the fastcluster R package version 1.2.6 (with method = “ward.D2”)<sup><a href="#CR123" class="usa-link" aria-describedby="CR123">123</a>–<a href="#CR125" class="usa-link" aria-describedby="CR125">125</a></sup>. This package provides a fast implementation of hierarchical agglomerative clustering. The resulting hierarchical clustering tree was iteratively cut into 2–12 groups using the <em>cutree</em> function of the stats R package. The optimal number of groups was determined through visual inspection of gene expression across groups (typically using violin plots<sup><a href="#CR126" class="usa-link" aria-describedby="CR126">126</a></sup>) and was guided by the scRNAseq data (details in Supplementary Data <a href="#MOESM3" class="usa-link">4</a>). Clustering was performed separately for each probe set. The number of mice, images, and cells varied for each experiment and are provided in Supplementary Data <a href="#MOESM3" class="usa-link">3</a>. When the experiment included mice with different genotypes, the data was jointly clustered, and no correction for the genotype effect was performed. When the probe set was designed to identify cell subtypes, a second round of clustering was necessary. This involved identifying the group (or groups) containing the desired cells (identified based on their gene expression profiles), and a second round of clustering only on these cells was applied following the steps described above. Right (medial-lateral) and left (lateral-medial) claustro-insular images were used in all experiments and were clustered together per probe set. However, these images did not always respectively correspond to sections from the right and left sides of the brain.</p></section><section id="Sec45"><h4 class="pmc_sec_title">Rigid image registration</h4>
<p id="Par68">Claustro-insular maps (Fig. <a href="#Fig2" class="usa-link">2</a>, Fig. <a href="#Fig6" class="usa-link">6</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>) were created using multiple images. For each anteroposterior level, a representative reference image was selected, and all other images from the same position were registered to it. The left images were horizontally flipped to align all images to the right side. A reference point located between the CLA and dorsal EP near the external capsule was used for image centering, along with two landmark points for landmark-based image rotation. The relative rotation angle of each image with the reference image was computed through slope-based angle calculation. Finally, 2D coordinate rotation using a rotation matrix was applied. No scaling of images was performed.</p></section><section id="Sec46"><h4 class="pmc_sec_title">Claustro-insular cell type map generation</h4>
<p id="Par69">2D coordinates of analyzed cells were used to generate a map of major claustro-insular cell types. As the claustrum extends over several millimeters along the anteroposterior axis of the brain, four representative coronal sections (bregma +1.42, +1.045, +0.145, and −1.355) were chosen. Probe sets A and B (see Supplementary Data <a href="#MOESM3" class="usa-link">3</a>) were clustered separately (“Cluster identification” section) and projected onto the same map via rigid image registration (“Rigid image registration” section). The number of mice, images, and cells analyzed is described in Supplementary Data <a href="#MOESM3" class="usa-link">3</a>. Reference images for each anteroposterior level corresponded to probe set A images, with landmark points differing by brain level: dorsal and ventral EP tips for bregma +1.42 to +0.145 and dorsal and ventral tips of the claustrum-like L6a region for bregma −1.355. Two bregma −1.355 images that were used for clustering were excluded during map generation due to distortions (one was anterior to the other images, and one was oblique). The cluster characterized by high expression levels of <em>Nr4a2</em> but no expression of <em>Slc17a6</em> from probe set B was also excluded from the map: since both CLA and L6b neurons exhibit high expression levels of <em>Nr4a2</em> and as this probe set did not include a probe for <em>Ccn2</em> (also known as <em>Ctgf</em>), a marker gene for L6b neurons, this cluster constituted a mixed set of neuronal populations that distorted the final map. However, CLA projection neurons expressing <em>Nr4a2</em> and <em>Slc17a6</em> were identified using probe set B.</p>
<p id="Par70">To highlight the diversity of cell types constituting the mouse claustro-insular region, proportional visualizations of the maps (Fig. <a href="#Fig2" class="usa-link">2e</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">4b</a>) were created by binning the 2D spatial maps into 20 μm bins, calculating cluster proportions per bin, and assigning weighted colors to each bin. To visualize the density distribution of CLA, shell, <em>Ccn2</em><sup><em>+</em></sup>, and <em>Rprm</em><sup><em>+</em></sup> cells at each brain level (Fig. <a href="#Fig2" class="usa-link">2f–j</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">5c</a>), two-dimensional kernel density estimations were computed using the <em>kde2d</em> function of the MASS R package version 7.3.60.0.1<sup><a href="#CR88" class="usa-link" aria-describedby="CR88">88</a></sup>, implemented in the <em>stat_density_2d_filled</em> function of the ggplot2 R package version 3.5.0. Density estimates were scaled to a maximum of 1 with contour breaks ranging from 0.1 to 1 in increments of 0.1. The number of grid points in the dorso-ventral and medio-lateral directions was set to 100, and bandwidths were estimated using the normal reference distribution via the <em>bandwidth.nrd</em> function of the MASS package. The maps were finally generated by plotting the highest density region of CLA, shell, <em>Ccn2</em><sup><em>+</em></sup>, and <em>Rprm</em><sup><em>+</em></sup> cells at a 50% probability level of the two-dimensional kernel density estimate using the <em>stat_hdr</em> function of the ggdensity R package version 1.0.0. All the maps were scaled to the same coordinate space.</p></section><section id="Sec47"><h4 class="pmc_sec_title">Spatial alterations in Nr4a2 haploinsufficient mice</h4>
<p id="Par71">To visualize spatial alterations in the claustro-insular region of <em>Nr4a2</em> haploinsufficient mice, a map of their CLA and shell neurons was produced, using two probe sets (see Supplementary Data <a href="#MOESM3" class="usa-link">3</a>) and following the same method as described above. Only coronal sections from the bregma level +1.045 were used. The number of mice, images, and cells analyzed per genotype is described in Supplementary Data <a href="#MOESM3" class="usa-link">1</a>. Cells were spatially projected in 2D (Fig. <a href="#Fig6" class="usa-link">6g</a>) and binned into 20 μm bins following the steps outlined above. The normalized CLA/shell contrast for each bin, representing the relative difference between CLA and shell clusters, was calculated as:</p>
<table class="disp-formula p" id="Equ1"><tr><td class="formula"><math id="d33e3231" display="block"><mfrac><mrow><msub><mrow><mi>N</mi></mrow><mrow><mi>C</mi><mi>L</mi><mi>A</mi></mrow></msub><mo>−</mo><msub><mrow><mi>N</mi></mrow><mrow><mi>s</mi><mi>h</mi><mi>e</mi><mi>l</mi><mi>l</mi></mrow></msub></mrow><mrow><msub><mrow><mi>N</mi></mrow><mrow><mi>C</mi><mi>L</mi><mi>A</mi></mrow></msub><mo>+</mo><msub><mrow><mi>N</mi></mrow><mrow><mi>s</mi><mi>h</mi><mi>e</mi><mi>l</mi><mi>l</mi></mrow></msub></mrow></mfrac></math></td></tr></table>
<p>where N is the number of cells from the corresponding cluster.</p></section><section id="Sec48"><h4 class="pmc_sec_title">RNAscope statistical analyses</h4>
<p id="Par72">All statistical analyses of RNAscope data were performed using R. Differences between genotypes in the total number of cells (Fig. <a href="#Fig6" class="usa-link">6a</a>), number of <em>Nr4a2</em><sup><em>+</em></sup> cells (Fig. <a href="#Fig6" class="usa-link">6b</a>), number of <em>Nr4a2</em> puncta (Fig.<a href="#Fig4" class="usa-link">4c</a>), and the expression levels of modulated genes (Fig. <a href="#Fig4" class="usa-link">4g–j</a>) were tested using generalized linear mixed models (GLMM) with negative binomial distributions, quadratic parameterization<sup><a href="#CR127" class="usa-link" aria-describedby="CR127">127</a></sup>, and a log link function (with family = nbinom2(link = “log”)). Models were fitted via the glmmTMB<sup><a href="#CR109" class="usa-link" aria-describedby="CR109">109</a>–<a href="#CR111" class="usa-link" aria-describedby="CR111">111</a></sup> R package version 1.1.9, using Template Model Builder (TMB). Probe sets and/or images were included as random effects depending on the test, as detailed in Supplementary Data <a href="#MOESM3" class="usa-link">1</a>. Likelihood ratio tests (LRT) with χ<sup>2</sup>-based <em>p</em>-values were then computed by comparing the resulting models with null models where the fixed effects term (i.e., genotype) was dropped, using the <em>drop1</em> R function<sup><a href="#CR112" class="usa-link" aria-describedby="CR112">112</a></sup>. When testing for differences in the expression levels of modulated genes between genotypes, <em>p</em>-values were adjusted for multiple comparisons using the Holm method<sup><a href="#CR113" class="usa-link" aria-describedby="CR113">113</a></sup>. Differences between genotypes in cell type proportions were tested using the <em>propeller</em> function of speckle, as described in the “Claustro-insular cellular composition analysis” section, with the following difference: the image from which each cell was derived served as a biological replicate.</p></section></section><section id="Sec49"><h3 class="pmc_sec_title">Electrophysiological recordings</h3>
<section id="Sec50"><h4 class="pmc_sec_title">Slice recordings</h4>
<p id="Par73">Slice recordings were conducted on 16–23-week-old Vglut2-IRES-Cre;<em>Nr4a2</em><sup><em>wt/wt</em></sup> and Vglut2-IRES-Cre;<em>Nr4a2</em><sup><em>del/wt</em></sup> mice, infected with retrogradely transported adeno-associated viral particles AAV(rg)-hSyn-DIO-EGFP (Addgene #50457) in the mPFC 3–4 weeks prior. In brief, mice were anesthetized with isoflurane (3–5% induction, 1–2% maintenance), and the skin over the skull was removed under local anesthesia using carbostesin (AstraZeneca). Animals were head-fixed in a stereotaxic apparatus (Stoelting) using ear bars and a nose clamp, with their eyes protected by artificial tears. Body temperature was maintained at ~37 °C via a heating pad (FHC). Bilateral craniotomies above the mPFC were performed using an air-pressurized drill. A glass pipette filled with AAV solution was inserted one hemisphere at a time at the following coordinates relative to bregma: AP: 1.7 mm, ML: ±0.3 mm, and DV: ‒1.5 mm from the surface of the brain. A volume of 200 nl was injected per site. After the procedure, mice were allowed to recover for 3–4 weeks. For slice recordings, animals were anesthetized with isoflurane, and the brain was quickly extracted. 300-μm-thick coronal slices were cut using a vibratome (Leica VT 928 S1000, Germany). Slices were transferred to continuously oxygenated artificial cerebrospinal fluid (ACSF) at 37 °C for 60 min, then maintained at room temperature for the remainder of the experiment. The ACSF contained (in mM): 124 NaCl, 3 KCl, 2 CaCl<sub>2</sub>, 1.3 MgSO<sub>4</sub>, 26 NaHCO<sub>3</sub>, 1.25 NaH<sub>2</sub>PO<sub>4</sub>, 10 D-glucose, with an osmolarity of 300 mOsm and a pH of 7.4 when bubbled with 95% O<sub>2</sub>−5% CO<sub>2</sub>. Slices were transferred to the recording chamber and perfused with oxygenated ACSF. All recordings were performed at ~37 °C (using a recording chamber and temperature controller from Luigs &amp; Neumann, Germany). Claustral neurons projecting to the mPFC and expressing the viral construct were identified using a 473 nm LED (Thorlabs, Germany). Neurons were visualized using an IR-DIC microscope (Olympus BX51, Germany), and whole-cell recordings of identified neurons were performed using borosilicate glass pipettes with a resistance of 4–7 MΩ, filled with an intracellular solution containing (in mM): 120 K-gluconate, 10 KCl, 10 HEPES, 4 ATP, 0.3 GTP, 10 phosphocreatine, 1 EGTA, and 0.4% biocytin. The solution had a pH of 7.2–7.3 and an osmolarity of 280–290 mOsm. The signal was amplified using Multiclamp 700B amplifiers (Molecular Devices, USA) with a sampling rate of 50 kHz, high-pass filtered at 4 kHz, and digitized using the PulseQ electrophysiology package running on Igor Pro (Wavemetrics, USA). Neurons were recorded either in ACSF or in ACSF supplemented with various pharmacological agents. Fast GABAergic and glutamatergic transmission was blocked using SR 95531 hydrobromide (GBZ, 10 μM; Tocris), NBQX-disodium salts (10 μM; Tocris), and DL-AP5 (100 μM; Tocris). ATX-II (10 nM, Alomone Labs) is an enhancer of the resurgent current of voltage-gated sodium channels. Kv1-4 Channels were blocked with 4-aminopyridine (1 mM, Tocris), Kv7 channels with XE991 dihydrochloride (20 µM, Alomone Labs), and IRK channels with barium chloride (1 mM, Sigma-Aldrich). Adenylate cyclase was blocked with SQ22536 (100 µM, Tocris), PKA blocked with H89 (10 µM, Tocris), and IP<sub>3</sub> receptors blocked with 2-APB (100 µM, Tocris). Finally, the mechanism underlying calcium-induced calcium release modulation of BK channels was dissected using the following pharmacological channel blockers: Cav1: Nifedipine (10 μM), Cav2.1: ω-Agatoxin IVA (100 nM, Alomone Labs), Cav2.2: ω-Conotoxin GVIA (3 μM, Alomone Labs), Cav2.3: SNX-482 (400 nM, Alomone Labs), Cav3: ML218 (2 μM, Alomone Labs), BKCa channels: Iberiotoxin (100 nM, Alomone Labs), RyR2: Ryanodine (10–20 μM, Alomone Labs).</p></section><section id="Sec51"><h4 class="pmc_sec_title">Patch-clamp data analysis</h4>
<p id="Par74">To analyze patch-clamp data, waveforms acquired in IgorPro6 were converted to MATLAB files. Action potential (AP) peaks were detected with MATLAB’s built-in ‘findpeaks’ function. To identify AP onsets, we computed the rolling average and rolling standard deviation over a window of 30 time points. An onset was defined when the absolute difference between the membrane potential and the rolling average exceeded four times the rolling standard deviation, and the potential was above the rolling average. The half-width was measured at half the AP’s amplitude. Offsets were identified after the peak as the point closest to the onset potential. Rise time was calculated from onset to peak, decay time from peak to offset, and the afterhyperpolarization potential (AHP) was measured from the offset to the lowest potential following the offset.</p>
<p id="Par75">The area under the curve (AUC) for each cell was calculated as the total surface area under the input/output function curve, using GraphPad Prism 8. To better compare the change in firing frequency between genotypes under different pharmacological conditions, the AUC and AHP amplitude were normalized to the wild-type values. For each pharmacological condition, we calculated the difference between a cell AUC and the mean <em>Nr4a2</em><sup><em>wt/wt</em></sup> AUC, then divided this difference by the mean AUC of the control <em>Nr4a2</em><sup><em>wt/wt</em></sup> cells for that condition (data presented as a percentage change). For each pharmacological condition, AHP amplitude was normalized by dividing each cell's AHP amplitude by the mean <em>Nr4a2</em><sup><em>wt/wt</em></sup> AHP amplitude for that condition. Statistical analyses were performed using GraphPad Prism 8. For input/output functions, statistical differences were assessed with a two-way ANOVA test. Single comparisons were analyzed using Fisher’s LSD test, and <em>p</em>-values were corrected for multiple comparisons with a 5% false discovery rate (Benjamini–Hochberg method). Group differences between <em>Nr4a2</em><sup><em>wt/wt</em></sup> and <em>Nr4a2</em><sup><em>del/wt</em></sup> for AP properties were determined with a two-way ANOVA followed by a Fisher’s LSD test, and <em>p</em>-values were adjusted for multiple comparisons using the two-stage linear step-up procedure of Benjamini, Krieger, and Yekutieli. Finally, changes in AUC and normalized AHP amplitude were assessed using a Mann–Whitney test. All statistical details can be found in Supplementary Data <a href="#MOESM3" class="usa-link">1</a>.</p></section></section><section id="Sec52"><h3 class="pmc_sec_title">Statistics and reproducibility</h3>
<p id="Par76">No statistical method was used to predetermine sample size; sample sizes were based on established protocols and aligned with veterinary and ethical guidelines. For scRNAseq, MERFISH, and RNAscope smFISH experiments, the high resolution and cell numbers per sample provided strong statistical power despite small animal numbers. For scRNAseq, 13 mice spanning age, sex, and genotype combinations yielded 14,852 quality-filtered neurons. For spatial transcriptomics, n = 2–3 animals per condition were used, with n = 1 for certain localization-only experiments. Patch-clamp sample sizes were based on prior literature and ensured genotype balance. No data were excluded from scRNAseq analyses; limited exclusions were made in MERFISH and smFISH due to technical issues (e.g., tissue damage, imaging artifacts), and patch-clamp recordings were excluded based on predefined quality criteria. Experiments were reproducible across independent biological replicates, with consistent findings across platforms and batches. Randomization was not necessary, as there was no treatment allocation and the only grouping variable was genotype. The investigators were blinded to genotype during patch-clamp recordings, but not during other experiments, where group allocation was not applicable and analyses were automated.</p></section><section id="Sec53"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par77">Further information on research design is available in the <a href="#MOESM4" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec54"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373765/bin/41467_2025_63138_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (19.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12373765/bin/41467_2025_63138_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63138_MOESM2_ESM.pdf</a><sup> (171.4KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12373765/bin/41467_2025_63138_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1-20</a><sup> (108.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12373765/bin/41467_2025_63138_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (107.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12373765/bin/41467_2025_63138_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (2.9MB, pdf) </sup>
</div></div></section></section><section id="Sec55"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12373765/bin/41467_2025_63138_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (74.1MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank all members of I.R and A.C laboratories for helpful discussions, Francisco Resende for expert mouse care, Loris Mannino and Leonardo Marconi for technical assistance, Dr. Jose Manuel Nunes for statistical advice, the Photonic Bioimaging Center of the University of Geneva for assistance with imaging, the NeuroNA Human Cellular Neuroscience Platform of Campus Biotech Geneva for assistance with MERFISH spatial transcriptomics, the Flow Cytometry core facility and the iGE3 Genomics Platform at the Faculty of Medicine of the University of Geneva for expert technical assistance for the scRNAseq experiment. We warmly thank Dr. Hongkui Zeng (the Allen Institute) for generously providing the <em>Nr4a2</em><sup>tm1(dreo)Hze</sup> mice. This research was supported by the University of Geneva and the European Research Council (contract ERC-SyG-856439-CLAUSTROFUNCT to A.C. and I.R.), the Swiss National Science Foundation (grant 501100001711-215572 and 501100001711-219531 to A.C. and I.R., respectively), the Fondation Privée des HUG (grant RC03-11 to A.C. and I.R.), and the Novartis Foundation for Medical Research (A.C. and I.R.).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>A.C. and I.R. acquired funding for the research. L.F., M.B., M.M., A.C., and I.R. conceived and designed the experiments. L.F., M.B., and M.M. acquired and analyzed data. L.F., M.B., M.M., A.C., and I.R. interpreted the data, wrote, and edited the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par78"><em>Nature Communications</em> thanks the anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The scRNAseq data generated in this study have been deposited in the NCBI GEO database under accession code <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291781" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE291781</a>. The MERFISH and smFISH data generated in this study have been deposited in the Figshare database [10.6084/m9.figshare.c.7953965.v1]. <a href="#Sec55" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>All codes used for data analysis and the modified GTF annotation file are provided at the GitHub repository [<a href="https://github.com/leonfodoulian/CLA_Nr4a2_hemizygous" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/leonfodoulian/CLA_Nr4a2_hemizygous</a>].</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par79">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Leon Fodoulian, Madlaina Boillat, Marie Moulinier.</p></div>
<div class="fn p" id="fn3"><p>These authors jointly supervised this work: Alan Carleton, Ivan Rodriguez.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Alan Carleton, Email: alan.carleton@unige.ch.</p>
<p>Ivan Rodriguez, Email: ivan.rodriguez@unige.ch.</p></section><section id="sec56"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63138-2.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Uddin, L. Q. Salience processing and insular cortical function and dysfunction. <em>Nat. Rev. Neurosci.</em><strong>16</strong>, 55–61 (2015).
</cite> [<a href="https://doi.org/10.1038/nrn3857" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25406711/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Uddin,%20L.%20Q.%20Salience%20processing%20and%20insular%20cortical%20function%20and%20dysfunction.%20Nat.%20Rev.%20Neurosci.16,%2055%E2%80%9361%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Gogolla, N. The insular cortex. <em>Curr. Biol.</em><strong>27</strong>, R580–R586 (2017).
</cite> [<a href="https://doi.org/10.1016/j.cub.2017.05.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28633023/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gogolla,%20N.%20The%20insular%20cortex.%20Curr.%20Biol.27,%20R580%E2%80%93R586%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Livneh, Y. &amp; Andermann, M. L. Cellular activity in insular cortex across seconds to hours: Sensations and predictions of bodily states. <em>Neuron</em><strong>109</strong>, 3576–3593 (2021).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2021.08.036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8602715/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34582784/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Livneh,%20Y.%20&amp;%20Andermann,%20M.%20L.%20Cellular%20activity%20in%20insular%20cortex%20across%20seconds%20to%20hours:%20Sensations%20and%20predictions%20of%20bodily%20states.%20Neuron109,%203576%E2%80%933593%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Baizer, J. S., Sherwood, C. C., Noonan, M. &amp; Hof, P. R. Comparative organization of the claustrum: what does structure tell us about function? <em>Front. Syst. Neurosci.</em><strong>8</strong>, 117 (2014).
</cite> [<a href="https://doi.org/10.3389/fnsys.2014.00117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4079070/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25071474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baizer,%20J.%20S.,%20Sherwood,%20C.%20C.,%20Noonan,%20M.%20&amp;%20Hof,%20P.%20R.%20Comparative%20organization%20of%20the%20claustrum:%20what%20does%20structure%20tell%20us%20about%20function?%20Front.%20Syst.%20Neurosci.8,%20117%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Atlan, G., Terem, A., Peretz-Rivlin, N., Groysman, M. &amp; Citri, A. Mapping synaptic cortico-claustral connectivity in the mouse. <em>J. Comp. Neurol.</em><strong>525</strong>, 1381–1402 (2017).
</cite> [<a href="https://doi.org/10.1002/cne.23997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26973027/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Atlan,%20G.,%20Terem,%20A.,%20Peretz-Rivlin,%20N.,%20Groysman,%20M.%20&amp;%20Citri,%20A.%20Mapping%20synaptic%20cortico-claustral%20connectivity%20in%20the%20mouse.%20J.%20Comp.%20Neurol.525,%201381%E2%80%931402%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Zingg, B., Dong, H.-W., Tao, H. W. &amp; Zhang, L. I. Input-output organization of the mouse claustrum. <em>J. Comp. Neurol.</em><strong>526</strong>, 2428–2443 (2018).
</cite> [<a href="https://doi.org/10.1002/cne.24502" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6196111/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30252130/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zingg,%20B.,%20Dong,%20H.-W.,%20Tao,%20H.%20W.%20&amp;%20Zhang,%20L.%20I.%20Input-output%20organization%20of%20the%20mouse%20claustrum.%20J.%20Comp.%20Neurol.526,%202428%E2%80%932443%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Crick, F. C. &amp; Koch, C. What is the function of the claustrum? <em>Philos. Trans. R. Soc. Lond. B Biol. Sci.</em><strong>360</strong>, 1271–1279 (2005).
</cite> [<a href="https://doi.org/10.1098/rstb.2005.1661" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1569501/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16147522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crick,%20F.%20C.%20&amp;%20Koch,%20C.%20What%20is%20the%20function%20of%20the%20claustrum?%20Philos.%20Trans.%20R.%20Soc.%20Lond.%20B%20Biol.%20Sci.360,%201271%E2%80%931279%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Brown, S. P. et al. New breakthroughs in understanding the role of functional interactions between the neocortex and the claustrum. <em>J. Neurosci.</em><strong>37</strong>, 10877–10881 (2017).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.1837-17.2017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5678020/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29118217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brown,%20S.%20P.%20et%20al.%20New%20breakthroughs%20in%20understanding%20the%20role%20of%20functional%20interactions%20between%20the%20neocortex%20and%20the%20claustrum.%20J.%20Neurosci.37,%2010877%E2%80%9310881%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Smith, J. B., Lee, A. K. &amp; Jackson, J. The claustrum. <em>Curr. Biol.</em><strong>30</strong>, R1401–R1406 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cub.2020.09.069" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33290700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith,%20J.%20B.,%20Lee,%20A.%20K.%20&amp;%20Jackson,%20J.%20The%20claustrum.%20Curr.%20Biol.30,%20R1401%E2%80%93R1406%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Madden, M. B. et al. A role for the claustrum in cognitive control. <em>Trends Cogn. Sci.</em><strong>26</strong>, 1133–1152 (2022).
</cite> [<a href="https://doi.org/10.1016/j.tics.2022.09.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9669149/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36192309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Madden,%20M.%20B.%20et%20al.%20A%20role%20for%20the%20claustrum%20in%20cognitive%20control.%20Trends%20Cogn.%20Sci.26,%201133%E2%80%931152%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Bhattacharjee, A. S. et al. The claustrum is critical for maintaining working memory information. Preprint at <em>bioRxiv</em>10.1101/2024.10.28.620649 (2024).</cite>
</li>
<li id="CR12">
<span class="label">12.</span><cite>Lein, E. S. et al. Genome-wide atlas of gene expression in the adult mouse brain. <em>Nature</em><strong>445</strong>, 168–176 (2007).
</cite> [<a href="https://doi.org/10.1038/nature05453" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17151600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lein,%20E.%20S.%20et%20al.%20Genome-wide%20atlas%20of%20gene%20expression%20in%20the%20adult%20mouse%20brain.%20Nature445,%20168%E2%80%93176%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Zeisel, A. et al. Cell types in the mouse cortex and hippocampus revealed by single-cell RNA-seq. <em>Science</em><strong>347</strong>, 1138–1142 (2015).
</cite> [<a href="https://doi.org/10.1126/science.aaa1934" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25700174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeisel,%20A.%20et%20al.%20Cell%20types%20in%20the%20mouse%20cortex%20and%20hippocampus%20revealed%20by%20single-cell%20RNA-seq.%20Science347,%201138%E2%80%931142%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Saunders, A. et al. Molecular diversity and specializations among the cells of the adult mouse brain. <em>Cell</em><strong>174</strong>, 1015–1030.e16 (2018).
</cite> [<a href="https://doi.org/10.1016/j.cell.2018.07.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6447408/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30096299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saunders,%20A.%20et%20al.%20Molecular%20diversity%20and%20specializations%20among%20the%20cells%20of%20the%20adult%20mouse%20brain.%20Cell174,%201015%E2%80%931030.e16%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Rosenberg, A. B. et al. Single-cell profiling of the developing mouse brain and spinal cord with split-pool barcoding. <em>Science</em><strong>360</strong>, 176–182 (2018).
</cite> [<a href="https://doi.org/10.1126/science.aam8999" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7643870/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29545511/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosenberg,%20A.%20B.%20et%20al.%20Single-cell%20profiling%20of%20the%20developing%20mouse%20brain%20and%20spinal%20cord%20with%20split-pool%20barcoding.%20Science360,%20176%E2%80%93182%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Chen, S., Lake, B. B. &amp; Zhang, K. High-throughput sequencing of the transcriptome and chromatin accessibility in the same cell. <em>Nat. Biotechnol.</em><strong>37</strong>, 1452–1457 (2019).
</cite> [<a href="https://doi.org/10.1038/s41587-019-0290-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6893138/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31611697/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20S.,%20Lake,%20B.%20B.%20&amp;%20Zhang,%20K.%20High-throughput%20sequencing%20of%20the%20transcriptome%20and%20chromatin%20accessibility%20in%20the%20same%20cell.%20Nat.%20Biotechnol.37,%201452%E2%80%931457%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Fodoulian, L. et al. The claustrum-medial prefrontal cortex network controls attentional set-shifting. Preprint at <em>bioRxiv</em>10.1101/2020.10.14.339259 (2020).</cite>
</li>
<li id="CR18">
<span class="label">18.</span><cite>Peng, H. et al. Morphological diversity of single neurons in molecularly defined cell types. <em>Nature</em><strong>598</strong>, 174–181 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03941-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8494643/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34616072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peng,%20H.%20et%20al.%20Morphological%20diversity%20of%20single%20neurons%20in%20molecularly%20defined%20cell%20types.%20Nature598,%20174%E2%80%93181%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Erwin, S. R. et al. Spatially patterned excitatory neuron subtypes and projections of the claustrum. <em>eLife</em><strong>10</strong>, e68967 (2021).
</cite> [<a href="https://doi.org/10.7554/eLife.68967" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8367382/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34397382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Erwin,%20S.%20R.%20et%20al.%20Spatially%20patterned%20excitatory%20neuron%20subtypes%20and%20projections%20of%20the%20claustrum.%20eLife10,%20e68967%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Yao, Z. et al. A taxonomy of transcriptomic cell types across the isocortex and hippocampal formation. <em>Cell</em><strong>184</strong>, 3222–3241.e26 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.04.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8195859/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34004146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yao,%20Z.%20et%20al.%20A%20taxonomy%20of%20transcriptomic%20cell%20types%20across%20the%20isocortex%20and%20hippocampal%20formation.%20Cell184,%203222%E2%80%933241.e26%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Yao, Z. et al. A high-resolution transcriptomic and spatial atlas of cell types in the whole mouse brain. <em>Nature</em><strong>624</strong>, 317–332 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06812-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10719114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38092916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yao,%20Z.%20et%20al.%20A%20high-resolution%20transcriptomic%20and%20spatial%20atlas%20of%20cell%20types%20in%20the%20whole%20mouse%20brain.%20Nature624,%20317%E2%80%93332%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Shi, H. et al. Spatial atlas of the mouse central nervous system at molecular resolution. <em>Nature</em><strong>622</strong>, 552–561 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06569-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10709140/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37758947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shi,%20H.%20et%20al.%20Spatial%20atlas%20of%20the%20mouse%20central%20nervous%20system%20at%20molecular%20resolution.%20Nature622,%20552%E2%80%93561%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Zhang, M. et al. Molecularly defined and spatially resolved cell atlas of the whole mouse brain. <em>Nature</em><strong>624</strong>, 343–354 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06808-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10719103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38092912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20M.%20et%20al.%20Molecularly%20defined%20and%20spatially%20resolved%20cell%20atlas%20of%20the%20whole%20mouse%20brain.%20Nature624,%20343%E2%80%93354%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Langlieb, J. et al. The molecular cytoarchitecture of the adult mouse brain. <em>Nature</em><strong>624</strong>, 333–342 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06818-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10719111/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38092915/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langlieb,%20J.%20et%20al.%20The%20molecular%20cytoarchitecture%20of%20the%20adult%20mouse%20brain.%20Nature624,%20333%E2%80%93342%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Chia, Z., Silberberg, G. &amp; Augustine, G. J. Functional properties, topological organization and sexual dimorphism of claustrum neurons projecting to anterior cingulate cortex. <em>Claustrum</em><strong>2</strong>, 1357412 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Chia,%20Z.,%20Silberberg,%20G.%20&amp;%20Augustine,%20G.%20J.%20Functional%20properties,%20topological%20organization%20and%20sexual%20dimorphism%20of%20claustrum%20neurons%20projecting%20to%20anterior%20cingulate%20cortex.%20Claustrum2,%201357412%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Graf, M., Nair, A., Wong, K. L. L., Tang, Y. &amp; Augustine, G. J. Identification of mouse claustral neuron types based on their intrinsic electrical properties. <em>eNeuro</em><strong>7</strong>, ENEURO.0216-20.2020 (2020).
</cite> [<a href="https://doi.org/10.1523/ENEURO.0216-20.2020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7405070/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32527746/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Graf,%20M.,%20Nair,%20A.,%20Wong,%20K.%20L.%20L.,%20Tang,%20Y.%20&amp;%20Augustine,%20G.%20J.%20Identification%20of%20mouse%20claustral%20neuron%20types%20based%20on%20their%20intrinsic%20electrical%20properties.%20eNeuro7,%20ENEURO.0216-20.2020%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Shelton, A. M. et al. Single neurons and networks in the mouse claustrum integrate input from widespread cortical sources. <em>eLife</em><strong>13</strong>, RP98002 (2025).</cite> [<a href="https://doi.org/10.7554/eLife.98002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12266724/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40667970/" class="usa-link">PubMed</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Qadir, H. et al. The mouse claustrum synaptically connects cortical network motifs. <em>Cell Rep.</em><strong>41</strong>, 111860 (2022).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2022.111860" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9838879/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36543121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qadir,%20H.%20et%20al.%20The%20mouse%20claustrum%20synaptically%20connects%20cortical%20network%20motifs.%20Cell%20Rep.41,%20111860%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Marriott, B. A., Do, A. D., Zahacy, R. &amp; Jackson, J. Topographic gradients define the projection patterns of the claustrum core and shell in mice. <em>J. Comp. Neurol</em>. 10.1002/cne.25043 (2020).</cite> [<a href="https://doi.org/10.1002/cne.25043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8048916/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32975316/" class="usa-link">PubMed</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Chia, Z., Augustine, G. J. &amp; Silberberg, G. Synaptic connectivity between the cortex and claustrum is organized into functional modules. <em>Curr. Biol.</em><strong>30</strong>, 2777–2790.e4 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cub.2020.05.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32531275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chia,%20Z.,%20Augustine,%20G.%20J.%20&amp;%20Silberberg,%20G.%20Synaptic%20connectivity%20between%20the%20cortex%20and%20claustrum%20is%20organized%20into%20functional%20modules.%20Curr.%20Biol.30,%202777%E2%80%932790.e4%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Ham, G. X. &amp; Augustine, G. J. Topologically organized networks in the claustrum reflect functional modularization. <em>Front. Neuroanat.</em><strong>16</strong>, 901807 (2022).
</cite> [<a href="https://doi.org/10.3389/fnana.2022.901807" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9259979/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35815332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ham,%20G.%20X.%20&amp;%20Augustine,%20G.%20J.%20Topologically%20organized%20networks%20in%20the%20claustrum%20reflect%20functional%20modularization.%20Front.%20Neuroanat.16,%20901807%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Wang, Q. et al. Regional and cell-type-specific afferent and efferent projections of the mouse claustrum. <em>Cell Rep.</em><strong>42</strong>, 112118 (2023).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2023.112118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10415534/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36774552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Q.%20et%20al.%20Regional%20and%20cell-type-specific%20afferent%20and%20efferent%20projections%20of%20the%20mouse%20claustrum.%20Cell%20Rep.42,%20112118%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Jackson, J., Karnani, M. M., Zemelman, B. V., Burdakov, D. &amp; Lee, A. K. Inhibitory control of prefrontal cortex by the claustrum. <em>Neuron</em><strong>99</strong>, 1029–1039.e4 (2018).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2018.07.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6168643/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30122374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jackson,%20J.,%20Karnani,%20M.%20M.,%20Zemelman,%20B.%20V.,%20Burdakov,%20D.%20&amp;%20Lee,%20A.%20K.%20Inhibitory%20control%20of%20prefrontal%20cortex%20by%20the%20claustrum.%20Neuron99,%201029%E2%80%931039.e4%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Smith, J. B. et al. The relationship between the claustrum and endopiriform nucleus: A perspective towards consensus on cross-species homology. <em>J. Comp. Neurol.</em><strong>527</strong>, 476–499 (2019).
</cite> [<a href="https://doi.org/10.1002/cne.24537" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6421118/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30225888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith,%20J.%20B.%20et%20al.%20The%20relationship%20between%20the%20claustrum%20and%20endopiriform%20nucleus:%20A%20perspective%20towards%20consensus%20on%20cross-species%20homology.%20J.%20Comp.%20Neurol.527,%20476%E2%80%93499%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Grimstvedt, J. S. et al. A multifaceted architectural framework of the mouse claustrum complex. <em>J. Comp. Neurol.</em><strong>531</strong>, 1772–1795 (2023).
</cite> [<a href="https://doi.org/10.1002/cne.25539" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10953385/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37782702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grimstvedt,%20J.%20S.%20et%20al.%20A%20multifaceted%20architectural%20framework%20of%20the%20mouse%20claustrum%20complex.%20J.%20Comp.%20Neurol.531,%201772%E2%80%931795%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Zetterström, R. H., Williams, R., Perlmann, T. &amp; Olson, L. Cellular expression of the immediate early transcription factors Nurr1 and NGFI-B suggests a gene regulatory role in several brain regions including the nigrostriatal dopamine system. <em>Mol. Brain Res.</em><strong>41</strong>, 111–120 (1996).
</cite> [<a href="https://doi.org/10.1016/0169-328x(96)00074-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8883941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zetterstr%C3%B6m,%20R.%20H.,%20Williams,%20R.,%20Perlmann,%20T.%20&amp;%20Olson,%20L.%20Cellular%20expression%20of%20the%20immediate%20early%20transcription%20factors%20Nurr1%20and%20NGFI-B%20suggests%20a%20gene%20regulatory%20role%20in%20several%20brain%20regions%20including%20the%20nigrostriatal%20dopamine%20system.%20Mol.%20Brain%20Res.41,%20111%E2%80%93120%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Arimatsu, Y., Ishida, M., Kaneko, T., Ichinose, S. &amp; Omori, A. Organization and development of corticocortical associative neurons expressing the orphan nuclear receptor Nurr1. <em>J. Comp. Neurol.</em><strong>466</strong>, 180–196 (2003).
</cite> [<a href="https://doi.org/10.1002/cne.10875" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14528447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arimatsu,%20Y.,%20Ishida,%20M.,%20Kaneko,%20T.,%20Ichinose,%20S.%20&amp;%20Omori,%20A.%20Organization%20and%20development%20of%20corticocortical%20associative%20neurons%20expressing%20the%20orphan%20nuclear%20receptor%20Nurr1.%20J.%20Comp.%20Neurol.466,%20180%E2%80%93196%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Watson, C. &amp; Puelles, L. Developmental gene expression in the mouse clarifies the organization of the claustrum and related endopiriform nuclei. <em>J. Comp. Neurol.</em><strong>525</strong>, 1499–1508 (2017).
</cite> [<a href="https://doi.org/10.1002/cne.24034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27159785/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Watson,%20C.%20&amp;%20Puelles,%20L.%20Developmental%20gene%20expression%20in%20the%20mouse%20clarifies%20the%20organization%20of%20the%20claustrum%20and%20related%20endopiriform%20nuclei.%20J.%20Comp.%20Neurol.525,%201499%E2%80%931508%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Fang, C., Wang, H. &amp; Naumann, R. K. Developmental patterning and neurogenetic gradients of Nurr1 positive neurons in the rat claustrum and lateral cortex. <em>Front. Neuroanat.</em><strong>15</strong>, 786329 (2021).
</cite> [<a href="https://doi.org/10.3389/fnana.2021.786329" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8675902/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34924965/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fang,%20C.,%20Wang,%20H.%20&amp;%20Naumann,%20R.%20K.%20Developmental%20patterning%20and%20neurogenetic%20gradients%20of%20Nurr1%20positive%20neurons%20in%20the%20rat%20claustrum%20and%20lateral%20cortex.%20Front.%20Neuroanat.15,%20786329%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Hoerder-Suabedissen, A. et al. Temporal origin of mouse claustrum and development of its cortical projections. <em>Cereb. Cortex</em><strong>33</strong>, 3944–3959 (2023).
</cite> [<a href="https://doi.org/10.1093/cercor/bhac318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10068282/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36104852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoerder-Suabedissen,%20A.%20et%20al.%20Temporal%20origin%20of%20mouse%20claustrum%20and%20development%20of%20its%20cortical%20projections.%20Cereb.%20Cortex33,%203944%E2%80%933959%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Li, H., Duque, A. &amp; Rakic, P. Origin and development of the claustrum in rhesus macaque. <em>Proc. Natl. Acad. Sci. USA</em><strong>120</strong>, e2220918120 (2023).
</cite> [<a href="https://doi.org/10.1073/pnas.2220918120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10334778/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37406098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20H.,%20Duque,%20A.%20&amp;%20Rakic,%20P.%20Origin%20and%20development%20of%20the%20claustrum%20in%20rhesus%20macaque.%20Proc.%20Natl.%20Acad.%20Sci.%20USA120,%20e2220918120%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Zetterström, R. H. et al. Dopamine neuron agenesis in Nurr1-deficient mice. <em>Science</em><strong>276</strong>, 248–250 (1997).
</cite> [<a href="https://doi.org/10.1126/science.276.5310.248" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9092472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zetterstr%C3%B6m,%20R.%20H.%20et%20al.%20Dopamine%20neuron%20agenesis%20in%20Nurr1-deficient%20mice.%20Science276,%20248%E2%80%93250%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Castillo, S. O. et al. Dopamine biosynthesis is selectively abolished in substantia nigra/ventral tegmental area but not in hypothalamic neurons in mice with targeted disruption of the Nurr1 gene. <em>Mol. Cell. Neurosci.</em><strong>11</strong>, 36–46 (1998).
</cite> [<a href="https://doi.org/10.1006/mcne.1998.0673" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9608532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Castillo,%20S.%20O.%20et%20al.%20Dopamine%20biosynthesis%20is%20selectively%20abolished%20in%20substantia%20nigra/ventral%20tegmental%20area%20but%20not%20in%20hypothalamic%20neurons%20in%20mice%20with%20targeted%20disruption%20of%20the%20Nurr1%20gene.%20Mol.%20Cell.%20Neurosci.11,%2036%E2%80%9346%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Saucedo-Cardenas, O. et al. Nurr1 is essential for the induction of the dopaminergic phenotype and the survival of ventral mesencephalic late dopaminergic precursor neurons. <em>Proc. Natl. Acad. Sci. USA</em><strong>95</strong>, 4013–4018 (1998).
</cite> [<a href="https://doi.org/10.1073/pnas.95.7.4013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC19954/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9520484/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saucedo-Cardenas,%20O.%20et%20al.%20Nurr1%20is%20essential%20for%20the%20induction%20of%20the%20dopaminergic%20phenotype%20and%20the%20survival%20of%20ventral%20mesencephalic%20late%20dopaminergic%20precursor%20neurons.%20Proc.%20Natl.%20Acad.%20Sci.%20USA95,%204013%E2%80%934018%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Le, W. et al. Selective agenesis of mesencephalic dopaminergic neurons in Nurr1-deficient mice. <em>Exp. Neurol.</em><strong>159</strong>, 451–458 (1999).
</cite> [<a href="https://doi.org/10.1006/exnr.1999.7191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10506516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Le,%20W.%20et%20al.%20Selective%20agenesis%20of%20mesencephalic%20dopaminergic%20neurons%20in%20Nurr1-deficient%20mice.%20Exp.%20Neurol.159,%20451%E2%80%93458%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Buervenich, S. et al. NURR1 mutations in cases of schizophrenia and manic-depressive disorder. <em>Am. J. Med. Genet.</em><strong>96</strong>, 808–813 (2000).
</cite> [<a href="https://doi.org/10.1002/1096-8628(20001204)96:6&lt;808::aid-ajmg23&gt;3.0.co;2-e" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11121187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buervenich,%20S.%20et%20al.%20NURR1%20mutations%20in%20cases%20of%20schizophrenia%20and%20manic-depressive%20disorder.%20Am.%20J.%20Med.%20Genet.96,%20808%E2%80%93813%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Chen, Y., Tsai, M., Shaw, C. &amp; Chen, C. Mutation analysis of the human NR4A2 gene, an essential gene for midbrain dopaminergic neurogenesis, in schizophrenic patients. <em>Am. J. Med. Genet.</em><strong>105</strong>, 753–757 (2001).
</cite> [<a href="https://doi.org/10.1002/ajmg.10036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11803525/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20Y.,%20Tsai,%20M.,%20Shaw,%20C.%20&amp;%20Chen,%20C.%20Mutation%20analysis%20of%20the%20human%20NR4A2%20gene,%20an%20essential%20gene%20for%20midbrain%20dopaminergic%20neurogenesis,%20in%20schizophrenic%20patients.%20Am.%20J.%20Med.%20Genet.105,%20753%E2%80%93757%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Reuter, M. S. et al. Haploinsufficiency of <em>NR4A2</em> is associated with a neurodevelopmental phenotype with prominent language impairment. <em>Am. J. Med. Genet. A</em><strong>173</strong>, 2231–2234 (2017).
</cite> [<a href="https://doi.org/10.1002/ajmg.a.38288" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28544326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reuter,%20M.%20S.%20et%20al.%20Haploinsufficiency%20of%20NR4A2%20is%20associated%20with%20a%20neurodevelopmental%20phenotype%20with%20prominent%20language%20impairment.%20Am.%20J.%20Med.%20Genet.%20A173,%202231%E2%80%932234%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Lévy, J. et al. <em>NR4A2</em> haploinsufficiency is associated with intellectual disability and autism spectrum disorder. <em>Clin. Genet.</em><strong>94</strong>, 264–268 (2018).
</cite> [<a href="https://doi.org/10.1111/cge.13383" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29770430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?L%C3%A9vy,%20J.%20et%20al.%20NR4A2%20haploinsufficiency%20is%20associated%20with%20intellectual%20disability%20and%20autism%20spectrum%20disorder.%20Clin.%20Genet.94,%20264%E2%80%93268%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Guo, H. et al. Genome sequencing identifies multiple deleterious variants in autism patients with more severe phenotypes. <em>Genet. Med.</em><strong>21</strong>, 1611–1620 (2019).
</cite> [<a href="https://doi.org/10.1038/s41436-018-0380-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6546556/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30504930/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo,%20H.%20et%20al.%20Genome%20sequencing%20identifies%20multiple%20deleterious%20variants%20in%20autism%20patients%20with%20more%20severe%20phenotypes.%20Genet.%20Med.21,%201611%E2%80%931620%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Singh, S. et al. De novo variants of NR4A2 are associated with neurodevelopmental disorder and epilepsy. <em>Genet. Med.</em><strong>22</strong>, 1413–1417 (2020).
</cite> [<a href="https://doi.org/10.1038/s41436-020-0815-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7394879/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32366965/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Singh,%20S.%20et%20al.%20De%20novo%20variants%20of%20NR4A2%20are%20associated%20with%20neurodevelopmental%20disorder%20and%20epilepsy.%20Genet.%20Med.22,%201413%E2%80%931417%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Jesús, S. et al. <em>NR4A2</em> mutations can cause intellectual disability and language impairment with persistent dystonia-Parkinsonism. <em>Neurol. Genet.</em><strong>7</strong>, e543 (2021).
</cite> [<a href="https://doi.org/10.1212/NXG.0000000000000543" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7879338/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33585677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jes%C3%BAs,%20S.%20et%20al.%20NR4A2%20mutations%20can%20cause%20intellectual%20disability%20and%20language%20impairment%20with%20persistent%20dystonia-Parkinsonism.%20Neurol.%20Genet.7,%20e543%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Singh, T. et al. Rare coding variants in ten genes confer substantial risk for schizophrenia. <em>Nature</em><strong>604</strong>, 509–516 (2022).
</cite> [<a href="https://doi.org/10.1038/s41586-022-04556-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9805802/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35396579/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Singh,%20T.%20et%20al.%20Rare%20coding%20variants%20in%20ten%20genes%20confer%20substantial%20risk%20for%20schizophrenia.%20Nature604,%20509%E2%80%93516%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Oshima, K. et al. A unique “reversed” migration of neurons in the developing claustrum. <em>J. Neurosci.</em><strong>43</strong>, 693–708 (2023).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.0704-22.2022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9899091/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36631266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oshima,%20K.%20et%20al.%20A%20unique%20%E2%80%9Creversed%E2%80%9D%20migration%20of%20neurons%20in%20the%20developing%20claustrum.%20J.%20Neurosci.43,%20693%E2%80%93708%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Mantas, I. et al. Claustrum and dorsal endopiriform cortex complex cell-identity is determined by Nurr1 and regulates hallucinogenic-like states in mice. <em>Nat. Commun.</em><strong>15</strong>, 8176 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-52429-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11408527/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39289358/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mantas,%20I.%20et%20al.%20Claustrum%20and%20dorsal%20endopiriform%20cortex%20complex%20cell-identity%20is%20determined%20by%20Nurr1%20and%20regulates%20hallucinogenic-like%20states%20in%20mice.%20Nat.%20Commun.15,%208176%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Tuberosa, J. et al. A toolbox for genetic targeting of the claustrum. Preprint at 10.7554/eLife.99168.1 (2024).</cite>
</li>
<li id="CR57">
<span class="label">57.</span><cite>Hao, Y. et al. Integrated analysis of multimodal single-cell data. <em>Cell</em><strong>184</strong>, 3573–3587.e29 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.04.048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8238499/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34062119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hao,%20Y.%20et%20al.%20Integrated%20analysis%20of%20multimodal%20single-cell%20data.%20Cell184,%203573%E2%80%933587.e29%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Dávila, J. C. et al. Embryonic and postnatal development of GABA, calbindin, calretinin, and parvalbumin in the mouse claustral complex. <em>J. Comp. Neurol.</em><strong>481</strong>, 42–57 (2005).
</cite> [<a href="https://doi.org/10.1002/cne.20347" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15558732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?D%C3%A1vila,%20J.%20C.%20et%20al.%20Embryonic%20and%20postnatal%20development%20of%20GABA,%20calbindin,%20calretinin,%20and%20parvalbumin%20in%20the%20mouse%20claustral%20complex.%20J.%20Comp.%20Neurol.481,%2042%E2%80%9357%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Legaz, I., García-López, M. &amp; Medina, L. Subpallial origin of part of the calbindin-positive neurons of the claustral complex and piriform cortex. <em>Brain Res. Bull.</em><strong>66</strong>, 470–474 (2005).
</cite> [<a href="https://doi.org/10.1016/j.brainresbull.2005.05.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16144634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Legaz,%20I.,%20Garc%C3%ADa-L%C3%B3pez,%20M.%20&amp;%20Medina,%20L.%20Subpallial%20origin%20of%20part%20of%20the%20calbindin-positive%20neurons%20of%20the%20claustral%20complex%20and%20piriform%20cortex.%20Brain%20Res.%20Bull.66,%20470%E2%80%93474%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Ransdell, J. L. et al. Loss of Navβ4-mediated regulation of sodium currents in adult Purkinje neurons disrupts firing and impairs motor coordination and balance. <em>Cell Rep.</em><strong>19</strong>, 532–544 (2017).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2017.03.068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5473293/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28423317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ransdell,%20J.%20L.%20et%20al.%20Loss%20of%20Nav%CE%B24-mediated%20regulation%20of%20sodium%20currents%20in%20adult%20Purkinje%20neurons%20disrupts%20firing%20and%20impairs%20motor%20coordination%20and%20balance.%20Cell%20Rep.19,%20532%E2%80%93544%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Hull, J. M. et al. Excitatory and inhibitory neuron defects in a mouse model of <em>Scn1b</em>-linked EIEE52. <em>Ann. Clin. Transl. Neurol.</em><strong>7</strong>, 2137–2149 (2020).
</cite> [<a href="https://doi.org/10.1002/acn3.51205" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7664274/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32979291/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hull,%20J.%20M.%20et%20al.%20Excitatory%20and%20inhibitory%20neuron%20defects%20in%20a%20mouse%20model%20of%20Scn1b-linked%20EIEE52.%20Ann.%20Clin.%20Transl.%20Neurol.7,%202137%E2%80%932149%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Bocksteins, E. Kv5, Kv6, Kv8, and Kv9 subunits: no simple silent bystanders. <em>J. Gen. Physiol.</em><strong>147</strong>, 105–125 (2016).
</cite> [<a href="https://doi.org/10.1085/jgp.201511507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4727947/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26755771/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bocksteins,%20E.%20Kv5,%20Kv6,%20Kv8,%20and%20Kv9%20subunits:%20no%20simple%20silent%20bystanders.%20J.%20Gen.%20Physiol.147,%20105%E2%80%93125%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Thorneloe, K. S. &amp; Nelson, M. T. Properties and molecular basis of the mouse urinary bladder voltage-gated K <sup>+</sup> current. <em>J. Physiol.</em><strong>549</strong>, 65–74 (2003).
</cite> [<a href="https://doi.org/10.1113/jphysiol.2003.039859" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2342925/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12679374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thorneloe,%20K.%20S.%20&amp;%20Nelson,%20M.%20T.%20Properties%20and%20molecular%20basis%20of%20the%20mouse%20urinary%20bladder%20voltage-gated%20K%20+%20current.%20J.%20Physiol.549,%2065%E2%80%9374%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Stas, J. I., Bocksteins, E., Labro, A. J. &amp; Snyders, D. J. Modulation of closed−state inactivation in Kv2.1/Kv6.4 heterotetramers as mechanism for 4−AP induced potentiation. <em>PLoS ONE</em><strong>10</strong>, e0141349 (2015).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0141349" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4623978/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26505474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stas,%20J.%20I.,%20Bocksteins,%20E.,%20Labro,%20A.%20J.%20&amp;%20Snyders,%20D.%20J.%20Modulation%20of%20closed%E2%88%92state%20inactivation%20in%20Kv2.1/Kv6.4%20heterotetramers%20as%20mechanism%20for%204%E2%88%92AP%20induced%20potentiation.%20PLoS%20ONE10,%20e0141349%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Marionneau, C. et al. The sodium channel accessory subunit Navβ1 regulates neuronal excitability through modulation of repolarizing voltage-gated K <sup>+</sup> channels. <em>J. Neurosci.</em><strong>32</strong>, 5716–5727 (2012).
</cite> [<a href="https://doi.org/10.1523/JNEUROSCI.6450-11.2012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3347704/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22539834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marionneau,%20C.%20et%20al.%20The%20sodium%20channel%20accessory%20subunit%20Nav%CE%B21%20regulates%20neuronal%20excitability%20through%20modulation%20of%20repolarizing%20voltage-gated%20K%20+%20channels.%20J.%20Neurosci.32,%205716%E2%80%935727%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Nguyen, H. M. et al. Modulation of voltage-gated K <sup>+</sup> channels by the sodium channel β1 subunit. <em>Proc. Natl. Acad. Sci. USA</em><strong>109</strong>, 18577–18582 (2012).
</cite> [<a href="https://doi.org/10.1073/pnas.1209142109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3494885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23090990/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nguyen,%20H.%20M.%20et%20al.%20Modulation%20of%20voltage-gated%20K%20+%20channels%20by%20the%20sodium%20channel%20%CE%B21%20subunit.%20Proc.%20Natl.%20Acad.%20Sci.%20USA109,%2018577%E2%80%9318582%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Lüscher, C. &amp; Slesinger, P. A. Emerging roles for G protein-gated inwardly rectifying potassium (GIRK) channels in health and disease. <em>Nat. Rev. Neurosci.</em><strong>11</strong>, 301–315 (2010).
</cite> [<a href="https://doi.org/10.1038/nrn2834" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3052907/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20389305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?L%C3%BCscher,%20C.%20&amp;%20Slesinger,%20P.%20A.%20Emerging%20roles%20for%20G%20protein-gated%20inwardly%20rectifying%20potassium%20(GIRK)%20channels%20in%20health%20and%20disease.%20Nat.%20Rev.%20Neurosci.11,%20301%E2%80%93315%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Sausbier, M. et al. Cerebellar ataxia and Purkinje cell dysfunction caused by Ca <sup>2+</sup> -activated K <sup>+</sup> channel deficiency. <em>Proc. Natl. Acad. Sci. USA</em><strong>101</strong>, 9474–9478 (2004).
</cite> [<a href="https://doi.org/10.1073/pnas.0401702101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC439001/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15194823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sausbier,%20M.%20et%20al.%20Cerebellar%20ataxia%20and%20Purkinje%20cell%20dysfunction%20caused%20by%20Ca%202+%20-activated%20K%20+%20channel%20deficiency.%20Proc.%20Natl.%20Acad.%20Sci.%20USA101,%209474%E2%80%939478%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Berkefeld, H. et al. BK <sub>Ca</sub>-Cav channel complexes mediate rapid and localized Ca <sup>2+</sup>-activated K <sup>+</sup> signaling. <em>Science</em><strong>314</strong>, 615–620 (2006).
</cite> [<a href="https://doi.org/10.1126/science.1132915" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17068255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Berkefeld,%20H.%20et%20al.%20BK%20Ca-Cav%20channel%20complexes%20mediate%20rapid%20and%20localized%20Ca%202+-activated%20K%20+%20signaling.%20Science314,%20615%E2%80%93620%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Berkefeld, H., Fakler, B. &amp; Schulte, U. Ca <sup>2+</sup> -Activated K <sup>+</sup> Channels: from protein complexes to function. <em>Physiol. Rev.</em><strong>90</strong>, 1437–1459 (2010).
</cite> [<a href="https://doi.org/10.1152/physrev.00049.2009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20959620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Berkefeld,%20H.,%20Fakler,%20B.%20&amp;%20Schulte,%20U.%20Ca%202+%20-Activated%20K%20+%20Channels:%20from%20protein%20complexes%20to%20function.%20Physiol.%20Rev.90,%201437%E2%80%931459%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Wehrens, X. H. T. et al. Ryanodine receptor/calcium release channel PKA phosphorylation: a critical mediator of heart failure progression. <em>Proc. Natl. Acad. Sci. USA</em><strong>103</strong>, 511–518 (2006).
</cite> [<a href="https://doi.org/10.1073/pnas.0510113103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1334677/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16407108/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wehrens,%20X.%20H.%20T.%20et%20al.%20Ryanodine%20receptor/calcium%20release%20channel%20PKA%20phosphorylation:%20a%20critical%20mediator%20of%20heart%20failure%20progression.%20Proc.%20Natl.%20Acad.%20Sci.%20USA103,%20511%E2%80%93518%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Suo, S. et al. Revealing the critical regulators of cell identity in the Mouse Cell Atlas. <em>Cell Rep.</em><strong>25</strong>, 1436–1445.e3 (2018).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2018.10.045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6281296/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30404000/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Suo,%20S.%20et%20al.%20Revealing%20the%20critical%20regulators%20of%20cell%20identity%20in%20the%20Mouse%20Cell%20Atlas.%20Cell%20Rep.25,%201436%E2%80%931445.e3%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Real, M. Á, Dávila, J. C. &amp; Guirado, S. Immunohistochemical localization of the vesicular glutamate transporter VGLUT2 in the developing and adult mouse claustrum. <em>J. Chem. Neuroanat.</em><strong>31</strong>, 169–177 (2006).
</cite> [<a href="https://doi.org/10.1016/j.jchemneu.2005.12.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16434168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Real,%20M.%20%C3%81,%20D%C3%A1vila,%20J.%20C.%20&amp;%20Guirado,%20S.%20Immunohistochemical%20localization%20of%20the%20vesicular%20glutamate%20transporter%20VGLUT2%20in%20the%20developing%20and%20adult%20mouse%20claustrum.%20J.%20Chem.%20Neuroanat.31,%20169%E2%80%93177%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Mathur, B. N. The claustrum in review. <em>Front. Syst. Neurosci.</em><strong>8</strong>, 48 (2014).
</cite> [<a href="https://doi.org/10.3389/fnsys.2014.00048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3983483/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24772070/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mathur,%20B.%20N.%20The%20claustrum%20in%20review.%20Front.%20Syst.%20Neurosci.8,%2048%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Johnson, J.-I., Fenske, B. A., Jaswa, A. S. &amp; Morris, J. A. Exploitation of puddles for breakthroughs in claustrum research. <em>Front. Syst. Neurosci.</em><strong>8</strong>, 78 (2014).
</cite> [<a href="https://doi.org/10.3389/fnsys.2014.00078" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4030192/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24860441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Johnson,%20J.-I.,%20Fenske,%20B.%20A.,%20Jaswa,%20A.%20S.%20&amp;%20Morris,%20J.%20A.%20Exploitation%20of%20puddles%20for%20breakthroughs%20in%20claustrum%20research.%20Front.%20Syst.%20Neurosci.8,%2078%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Binks, D., Watson, C. &amp; Puelles, L. A Re-evaluation of the anatomy of the claustrum in rodents and primates—analyzing the effect of pallial expansion. <em>Front. Neuroanat.</em><strong>13</strong>, 34 (2019).
</cite> [<a href="https://doi.org/10.3389/fnana.2019.00034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6443842/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30971901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Binks,%20D.,%20Watson,%20C.%20&amp;%20Puelles,%20L.%20A%20Re-evaluation%20of%20the%20anatomy%20of%20the%20claustrum%20in%20rodents%20and%20primates%E2%80%94analyzing%20the%20effect%20of%20pallial%20expansion.%20Front.%20Neuroanat.13,%2034%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Shaker, T. et al. A simple and reliable method for claustrum localization across age in mice. <em>Mol. Brain</em><strong>17</strong>, 10 (2024).
</cite> [<a href="https://doi.org/10.1186/s13041-024-01082-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10874566/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38368400/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shaker,%20T.%20et%20al.%20A%20simple%20and%20reliable%20method%20for%20claustrum%20localization%20across%20age%20in%20mice.%20Mol.%20Brain17,%2010%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Atlan, G. et al. The claustrum supports resilience to distraction. <em>Curr. Biol.</em><strong>28</strong>, 2752–2762.e7 (2018).
</cite> [<a href="https://doi.org/10.1016/j.cub.2018.06.068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6485402/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30122531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Atlan,%20G.%20et%20al.%20The%20claustrum%20supports%20resilience%20to%20distraction.%20Curr.%20Biol.28,%202752%E2%80%932762.e7%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>White, M. G. et al. Anterior cingulate cortex input to the claustrum is required for top-down action control. <em>Cell Rep.</em><strong>22</strong>, 84–95 (2018).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2017.12.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5779631/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29298436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?White,%20M.%20G.%20et%20al.%20Anterior%20cingulate%20cortex%20input%20to%20the%20claustrum%20is%20required%20for%20top-down%20action%20control.%20Cell%20Rep.22,%2084%E2%80%9395%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>White, M. G. et al. The mouse claustrum is required for optimal behavioral performance under high cognitive demand. <em>Biol. Psychiatry</em><strong>88</strong>, 719–726 (2020).
</cite> [<a href="https://doi.org/10.1016/j.biopsych.2020.03.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7554117/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32456782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?White,%20M.%20G.%20et%20al.%20The%20mouse%20claustrum%20is%20required%20for%20optimal%20behavioral%20performance%20under%20high%20cognitive%20demand.%20Biol.%20Psychiatry88,%20719%E2%80%93726%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Dixsaut, L. &amp; Gräff, J. Brain-wide screen of prelimbic cortex inputs reveals a functional shift during early fear memory consolidation. <em>eLife</em><strong>11</strong>, e78542 (2022).
</cite> [<a href="https://doi.org/10.7554/eLife.78542" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9286739/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35838139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dixsaut,%20L.%20&amp;%20Gr%C3%A4ff,%20J.%20Brain-wide%20screen%20of%20prelimbic%20cortex%20inputs%20reveals%20a%20functional%20shift%20during%20early%20fear%20memory%20consolidation.%20eLife11,%20e78542%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Atlan, G. et al. Claustrum neurons projecting to the anterior cingulate restrict engagement during sleep and behavior. <em>Nat. Commun.</em><strong>15</strong>, 5415 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-48829-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11208603/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38926345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Atlan,%20G.%20et%20al.%20Claustrum%20neurons%20projecting%20to%20the%20anterior%20cingulate%20restrict%20engagement%20during%20sleep%20and%20behavior.%20Nat.%20Commun.15,%205415%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Madisen, L. et al. Transgenic mice for intersectional targeting of neural sensors and effectors with high specificity and performance. <em>Neuron</em><strong>85</strong>, 942–958 (2015).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2015.02.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4365051/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25741722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Madisen,%20L.%20et%20al.%20Transgenic%20mice%20for%20intersectional%20targeting%20of%20neural%20sensors%20and%20effectors%20with%20high%20specificity%20and%20performance.%20Neuron85,%20942%E2%80%93958%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Vong, L. et al. Leptin action on GABAergic neurons prevents obesity and reduces inhibitory tone to POMC neurons. <em>Neuron</em><strong>71</strong>, 142–154 (2011).
</cite> [<a href="https://doi.org/10.1016/j.neuron.2011.05.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3134797/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21745644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vong,%20L.%20et%20al.%20Leptin%20action%20on%20GABAergic%20neurons%20prevents%20obesity%20and%20reduces%20inhibitory%20tone%20to%20POMC%20neurons.%20Neuron71,%20142%E2%80%93154%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Saxena, A. et al. Trehalose-enhanced isolation of neuronal sub-types from adult mouse brain. <em>BioTechniques</em><strong>52</strong>, 381–385 (2012).
</cite> [<a href="https://doi.org/10.2144/0000113878" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3696583/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22668417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saxena,%20A.%20et%20al.%20Trehalose-enhanced%20isolation%20of%20neuronal%20sub-types%20from%20adult%20mouse%20brain.%20BioTechniques52,%20381%E2%80%93385%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Zheng, G. X. Y. et al. Massively parallel digital transcriptional profiling of single cells. <em>Nat. Commun.</em><strong>8</strong>, 14049 (2017).
</cite> [<a href="https://doi.org/10.1038/ncomms14049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5241818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28091601/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20G.%20X.%20Y.%20et%20al.%20Massively%20parallel%20digital%20transcriptional%20profiling%20of%20single%20cells.%20Nat.%20Commun.8,%2014049%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. <em>Bioinformatics</em><strong>29</strong>, 15–21 (2013).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3530905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dobin,%20A.%20et%20al.%20STAR:%20ultrafast%20universal%20RNA-seq%20aligner.%20Bioinformatics29,%2015%E2%80%9321%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Venables, W. N. &amp; Ripley, B. D. <em>Modern Applied Statistics with S</em> (Springer New York, New York, NY, 2002).</cite>
</li>
<li id="CR89">
<span class="label">89.</span><cite>Andrews, T. S. &amp; Hemberg, M. M3Drop: dropout-based feature selection for scRNASeq. <em>Bioinformatics</em><strong>35</strong>, 2865–2867 (2019).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty1044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6691329/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30590489/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andrews,%20T.%20S.%20&amp;%20Hemberg,%20M.%20M3Drop:%20dropout-based%20feature%20selection%20for%20scRNASeq.%20Bioinformatics35,%202865%E2%80%932867%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Benjamini, Y. &amp; Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. <em>J. R. Stat. Soc. Ser. B Stat. Methodol.</em><strong>57</strong>, 289–300 (1995).</cite> [<a href="https://scholar.google.com/scholar_lookup?Benjamini,%20Y.%20&amp;%20Hochberg,%20Y.%20Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing.%20J.%20R.%20Stat.%20Soc.%20Ser.%20B%20Stat.%20Methodol.57,%20289%E2%80%93300%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Satija, R., Farrell, J. A., Gennert, D., Schier, A. F. &amp; Regev, A. Spatial reconstruction of single-cell gene expression data. <em>Nat. Biotechnol.</em><strong>33</strong>, 495–502 (2015).
</cite> [<a href="https://doi.org/10.1038/nbt.3192" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4430369/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25867923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Satija,%20R.,%20Farrell,%20J.%20A.,%20Gennert,%20D.,%20Schier,%20A.%20F.%20&amp;%20Regev,%20A.%20Spatial%20reconstruction%20of%20single-cell%20gene%20expression%20data.%20Nat.%20Biotechnol.33,%20495%E2%80%93502%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Hafemeister, C. &amp; Satija, R. Normalization and variance stabilization of single-cell RNA-seq data using regularized negative binomial regression. <em>Genome Biol.</em><strong>20</strong>, 296 (2019).
</cite> [<a href="https://doi.org/10.1186/s13059-019-1874-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6927181/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31870423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hafemeister,%20C.%20&amp;%20Satija,%20R.%20Normalization%20and%20variance%20stabilization%20of%20single-cell%20RNA-seq%20data%20using%20regularized%20negative%20binomial%20regression.%20Genome%20Biol.20,%20296%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Choudhary, S. &amp; Satija, R. Comparison and evaluation of statistical error models for scRNA-seq. <em>Genome Biol.</em><strong>23</strong>, 27 (2022).
</cite> [<a href="https://doi.org/10.1186/s13059-021-02584-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8764781/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35042561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Choudhary,%20S.%20&amp;%20Satija,%20R.%20Comparison%20and%20evaluation%20of%20statistical%20error%20models%20for%20scRNA-seq.%20Genome%20Biol.23,%2027%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Ahlmann-Eltze, C. &amp; Huber, W. glmGamPoi: fitting Gamma-Poisson generalized linear models on single cell count data. <em>Bioinformatics</em><strong>36</strong>, 5701–5702 (2021).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaa1009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8023675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33295604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ahlmann-Eltze,%20C.%20&amp;%20Huber,%20W.%20glmGamPoi:%20fitting%20Gamma-Poisson%20generalized%20linear%20models%20on%20single%20cell%20count%20data.%20Bioinformatics36,%205701%E2%80%935702%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Baglama, J. &amp; Reichel, L. Augmented implicitly restarted lanczos bidiagonalization methods. <em>SIAM J. Sci. Comput.</em><strong>27</strong>, 19–42 (2005).</cite> [<a href="https://scholar.google.com/scholar_lookup?Baglama,%20J.%20&amp;%20Reichel,%20L.%20Augmented%20implicitly%20restarted%20lanczos%20bidiagonalization%20methods.%20SIAM%20J.%20Sci.%20Comput.27,%2019%E2%80%9342%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Satopaa, V., Albrecht, J., Irwin, D. &amp; Raghavan, B. Finding a ‘kneedle’ in a haystack: detecting knee points in system behavior. In <em>2011 31st International Conference on Distributed Computing Systems Workshops</em> 166–171 (IEEE, Minneapolis, MN, USA, 2011).</cite>
</li>
<li id="CR97">
<span class="label">97.</span><cite>Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. <em>Nat. Methods</em><strong>16</strong>, 1289–1296 (2019).
</cite> [<a href="https://doi.org/10.1038/s41592-019-0619-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6884693/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31740819/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Korsunsky,%20I.%20et%20al.%20Fast,%20sensitive%20and%20accurate%20integration%20of%20single-cell%20data%20with%20Harmony.%20Nat.%20Methods16,%201289%E2%80%931296%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>McInnes, L., Healy, J. &amp; Melville, J. UMAP: uniform manifold approximation and projection for dimension reduction. Preprint at 10.48550/ARXIV.1802.03426 (2018).</cite>
</li>
<li id="CR99">
<span class="label">99.</span><cite>Becht, E. et al. Dimensionality reduction for visualizing single-cell data using UMAP. <em>Nat. Biotechnol.</em><strong>37</strong>, 38–44 (2019).</cite> [<a href="https://doi.org/10.1038/nbt.4314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30531897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Becht,%20E.%20et%20al.%20Dimensionality%20reduction%20for%20visualizing%20single-cell%20data%20using%20UMAP.%20Nat.%20Biotechnol.37,%2038%E2%80%9344%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>Traag, V. A., Waltman, L. &amp; Van Eck, N. J. From Louvain to Leiden: guaranteeing well-connected communities. <em>Sci. Rep.</em><strong>9</strong>, 5233 (2019).
</cite> [<a href="https://doi.org/10.1038/s41598-019-41695-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6435756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30914743/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Traag,%20V.%20A.,%20Waltman,%20L.%20&amp;%20Van%20Eck,%20N.%20J.%20From%20Louvain%20to%20Leiden:%20guaranteeing%20well-connected%20communities.%20Sci.%20Rep.9,%205233%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite>Reichardt, J. &amp; Bornholdt, S. Statistical mechanics of community detection. <em>Phys. Rev. E</em><strong>74</strong>, 016110 (2006).</cite> [<a href="https://doi.org/10.1103/PhysRevE.74.016110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16907154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reichardt,%20J.%20&amp;%20Bornholdt,%20S.%20Statistical%20mechanics%20of%20community%20detection.%20Phys.%20Rev.%20E74,%20016110%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Leicht, E. A. &amp; Newman, M. E. J. Community structure in directed networks. <em>Phys. Rev. Lett.</em><strong>100</strong>, 118703 (2008).
</cite> [<a href="https://doi.org/10.1103/PhysRevLett.100.118703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18517839/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leicht,%20E.%20A.%20&amp;%20Newman,%20M.%20E.%20J.%20Community%20structure%20in%20directed%20networks.%20Phys.%20Rev.%20Lett.100,%20118703%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR103">
<span class="label">103.</span><cite>Meilă, M. Comparing clusterings by the variation of information. In <em>Learning Theory and Kernel Machines</em> (eds. Schölkopf, B. &amp; Warmuth, M. K.) vol. 2777 173–187 (Springer Berlin Heidelberg, Berlin, Heidelberg, 2003).</cite>
</li>
<li id="CR104">
<span class="label">104.</span><cite>Pedregosa, F. et al. Scikit-learn: Machine Learning in Python. <em>JMLR</em><strong>12</strong>, 2825-2830 (2011).</cite>
</li>
<li id="CR105">
<span class="label">105.</span><cite>Buitinck, L. et al. API design for machine learning software: experiences from the scikit-learn project. Preprint at 10.48550/ARXIV.1309.0238 (2013).</cite>
</li>
<li id="CR106">
<span class="label">106.</span><cite>McCullagh, P. &amp; Nelder, J. A. <em>Generalized Linear Models</em> (Springer US, Boston, MA, 1989).</cite>
</li>
<li id="CR107">
<span class="label">107.</span><cite>Dobson, A. J. <em>An Introduction to Generalized Linear Models</em>. (Springer US, Boston, MA, 1990).</cite>
</li>
<li id="CR108">
<span class="label">108.</span><cite>Hastie, T. J. &amp; Pregibon, D. Generalized linear models. In <em>Statistical Models in S</em> (Chapman &amp; Hall, London, 1992).</cite>
</li>
<li id="CR109">
<span class="label">109.</span><cite>Wood, S. N. Stable and efficient multiple smoothing parameter estimation for generalized additive models. <em>J. Am. Stat. Assoc.</em><strong>99</strong>, 673–686 (2004).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wood,%20S.%20N.%20Stable%20and%20efficient%20multiple%20smoothing%20parameter%20estimation%20for%20generalized%20additive%20models.%20J.%20Am.%20Stat.%20Assoc.99,%20673%E2%80%93686%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR110">
<span class="label">110.</span><cite>Kristensen, K., Nielsen, A., Berg, C. W., Skaug, H. &amp; Bell, B. M. TMB: Automatic differentiation and Laplace approximation. J. Stat. Softw. <strong>70</strong>, 1–21 (2016).</cite> [<a href="https://scholar.google.com/scholar_lookup?Kristensen,%20K.,%20Nielsen,%20A.,%20Berg,%20C.%20W.,%20Skaug,%20H.%20&amp;%20Bell,%20B.%20M.%20TMB:%20Automatic%20differentiation%20and%20Laplace%20approximation.%20J.%20Stat.%20Softw.%2070,%201%E2%80%9321%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR111">
<span class="label">111.</span><cite>Brooks, M. E. et al. glmmTMB Balances Speed and Flexibility Among Packages for Zero-inflated Generalized Linear Mixed Modeling. <em>R. J.</em><strong>9</strong>, 378 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Brooks,%20M.%20E.%20et%20al.%20glmmTMB%20Balances%20Speed%20and%20Flexibility%20Among%20Packages%20for%20Zero-inflated%20Generalized%20Linear%20Mixed%20Modeling.%20R.%20J.9,%20378%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR112">
<span class="label">112.</span><cite><em>Statistical Models in S</em> (Chapman &amp; Hall, London, 1997).</cite>
</li>
<li id="CR113">
<span class="label">113.</span><cite>Holm, S. A simple sequentially rejective multiple test procedure. <em>Scand. J. Stat.</em><strong>6</strong>, 65–70 (1979).</cite> [<a href="https://scholar.google.com/scholar_lookup?Holm,%20S.%20A%20simple%20sequentially%20rejective%20multiple%20test%20procedure.%20Scand.%20J.%20Stat.6,%2065%E2%80%9370%20(1979)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR114">
<span class="label">114.</span><cite>Aibar, S. et al. SCENIC: single-cell regulatory network inference and clustering. <em>Nat. Methods</em><strong>14</strong>, 1083–1086 (2017).
</cite> [<a href="https://doi.org/10.1038/nmeth.4463" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5937676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28991892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aibar,%20S.%20et%20al.%20SCENIC:%20single-cell%20regulatory%20network%20inference%20and%20clustering.%20Nat.%20Methods14,%201083%E2%80%931086%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR115">
<span class="label">115.</span><cite>Van De Sande, B. et al. A scalable SCENIC workflow for single-cell gene regulatory network analysis. <em>Nat. Protoc.</em><strong>15</strong>, 2247–2276 (2020).
</cite> [<a href="https://doi.org/10.1038/s41596-020-0336-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32561888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20De%20Sande,%20B.%20et%20al.%20A%20scalable%20SCENIC%20workflow%20for%20single-cell%20gene%20regulatory%20network%20analysis.%20Nat.%20Protoc.15,%202247%E2%80%932276%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR116">
<span class="label">116.</span><cite>Bravo González-Blas, C. et al. SCENIC+: single-cell multiomic inference of enhancers and gene regulatory networks. <em>Nat. Methods</em><strong>20</strong>, 1355–1367 (2023).
</cite> [<a href="https://doi.org/10.1038/s41592-023-01938-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10482700/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37443338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bravo%20Gonz%C3%A1lez-Blas,%20C.%20et%20al.%20SCENIC+:%20single-cell%20multiomic%20inference%20of%20enhancers%20and%20gene%20regulatory%20networks.%20Nat.%20Methods20,%201355%E2%80%931367%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR117">
<span class="label">117.</span><cite>Moerman, T. et al. GRNBoost2 and Arboreto: efficient and scalable inference of gene regulatory networks. <em>Bioinformatics</em><strong>35</strong>, 2159–2161 (2019).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty916" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30445495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moerman,%20T.%20et%20al.%20GRNBoost2%20and%20Arboreto:%20efficient%20and%20scalable%20inference%20of%20gene%20regulatory%20networks.%20Bioinformatics35,%202159%E2%80%932161%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR118">
<span class="label">118.</span><cite>Phipson, B. et al. <em>propeller:</em> testing for differences in cell type proportions in single cell data. <em>Bioinformatics</em><strong>38</strong>, 4720–4726 (2022).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btac582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9563678/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36005887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Phipson,%20B.%20et%20al.%20propeller:%20testing%20for%20differences%20in%20cell%20type%20proportions%20in%20single%20cell%20data.%20Bioinformatics38,%204720%E2%80%934726%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR119">
<span class="label">119.</span><cite>Ritchie, M. E. et al. limma powers differential expression analyses for RNA-sequencing and microarray studies. <em>Nucleic Acids Res.</em><strong>43</strong>, e47–e47 (2015).
</cite> [<a href="https://doi.org/10.1093/nar/gkv007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4402510/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25605792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ritchie,%20M.%20E.%20et%20al.%20limma%20powers%20differential%20expression%20analyses%20for%20RNA-sequencing%20and%20microarray%20studies.%20Nucleic%20Acids%20Res.43,%20e47%E2%80%93e47%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR120">
<span class="label">120.</span><cite>Smyth, G. K. Linear models and empirical Bayes methods for assessing differential expression in microarray experiments. <em>Stat. Appl. Genet. Mol. Biol.</em><strong>3</strong>, 1–25 (2004).</cite> [<a href="https://doi.org/10.2202/1544-6115.1027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16646809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smyth,%20G.%20K.%20Linear%20models%20and%20empirical%20Bayes%20methods%20for%20assessing%20differential%20expression%20in%20microarray%20experiments.%20Stat.%20Appl.%20Genet.%20Mol.%20Biol.3,%201%E2%80%9325%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR121">
<span class="label">121.</span><cite>Chen, K. H., Boettiger, A. N., Moffitt, J. R., Wang, S. &amp; Zhuang, X. Spatially resolved, highly multiplexed RNA profiling in single cells. <em>Science</em><strong>348</strong>, aaa6090 (2015).
</cite> [<a href="https://doi.org/10.1126/science.aaa6090" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4662681/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25858977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20K.%20H.,%20Boettiger,%20A.%20N.,%20Moffitt,%20J.%20R.,%20Wang,%20S.%20&amp;%20Zhuang,%20X.%20Spatially%20resolved,%20highly%20multiplexed%20RNA%20profiling%20in%20single%20cells.%20Science348,%20aaa6090%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR122">
<span class="label">122.</span><cite>Barshan, E., Ghodsi, A., Azimifar, Z. &amp; Zolghadri Jahromi, M. Supervised principal component analysis: visualization, classification and regression on subspaces and submanifolds. <em>Pattern Recognit.</em><strong>44</strong>, 1357–1371 (2011).</cite> [<a href="https://scholar.google.com/scholar_lookup?Barshan,%20E.,%20Ghodsi,%20A.,%20Azimifar,%20Z.%20&amp;%20Zolghadri%20Jahromi,%20M.%20Supervised%20principal%20component%20analysis:%20visualization,%20classification%20and%20regression%20on%20subspaces%20and%20submanifolds.%20Pattern%20Recognit.44,%201357%E2%80%931371%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR123">
<span class="label">123.</span><cite>Ward, J. H. Hierarchical grouping to optimize an objective function. <em>J. Am. Stat. Assoc.</em><strong>58</strong>, 236–244 (1963).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ward,%20J.%20H.%20Hierarchical%20grouping%20to%20optimize%20an%20objective%20function.%20J.%20Am.%20Stat.%20Assoc.58,%20236%E2%80%93244%20(1963)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR124">
<span class="label">124.</span><cite>Murtagh, F. &amp; Legendre, P. Ward’s hierarchical agglomerative clustering method: which algorithms implement Ward’s criterion? <em>J. Classif.</em><strong>31</strong>, 274–295 (2014).</cite> [<a href="https://scholar.google.com/scholar_lookup?Murtagh,%20F.%20&amp;%20Legendre,%20P.%20Ward%E2%80%99s%20hierarchical%20agglomerative%20clustering%20method:%20which%20algorithms%20implement%20Ward%E2%80%99s%20criterion?%20J.%20Classif.31,%20274%E2%80%93295%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR125">
<span class="label">125.</span><cite>Müllner, D. fastcluster: fast hierarchical, agglomerative clustering routines for R and Python. J. Stat. Softw. <strong>53</strong>, 1–18 (2013).</cite> [<a href="https://scholar.google.com/scholar_lookup?M%C3%BCllner,%20D.%20fastcluster:%20fast%20hierarchical,%20agglomerative%20clustering%20routines%20for%20R%20and%20Python.%20J.%20Stat.%20Softw.%2053,%201%E2%80%9318%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR126">
<span class="label">126.</span><cite>Hintze, J. L. &amp; Nelson, R. D. Violin plots: a box plot-density trace synergism. <em>Am. Stat.</em><strong>52</strong>, 181–184 (1998).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hintze,%20J.%20L.%20&amp;%20Nelson,%20R.%20D.%20Violin%20plots:%20a%20box%20plot-density%20trace%20synergism.%20Am.%20Stat.52,%20181%E2%80%93184%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR127">
<span class="label">127.</span><cite>Hardin, J. W. &amp; Hilbe, J. M. <em>Generalized Linear Models and Extensions</em> (Stata Press, College Station, Texas, 2018).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373765/bin/41467_2025_63138_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (19.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373765/bin/41467_2025_63138_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63138_MOESM2_ESM.pdf</a><sup> (171.4KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373765/bin/41467_2025_63138_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1-20</a><sup> (108.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373765/bin/41467_2025_63138_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (107.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373765/bin/41467_2025_63138_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (2.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373765/bin/41467_2025_63138_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (74.1MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The scRNAseq data generated in this study have been deposited in the NCBI GEO database under accession code <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE291781" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE291781</a>. The MERFISH and smFISH data generated in this study have been deposited in the Figshare database [10.6084/m9.figshare.c.7953965.v1]. <a href="#Sec55" class="usa-link">Source data</a> are provided with this paper.</p>
<p>All codes used for data analysis and the modified GTF annotation file are provided at the GitHub repository [<a href="https://github.com/leonfodoulian/CLA_Nr4a2_hemizygous" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/leonfodoulian/CLA_Nr4a2_hemizygous</a>].</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63138-2"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63138.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (12.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373765/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373765/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373765%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373765/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373765/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373765/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40846807/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373765/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40846807/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373765/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373765/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="O2qd9MJDnlbh1oLAK7kJTxKWIIFeseMXtJoxWSrkcocHq4odSDYnalFyrMRU8pj6">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
