
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Unsupervised fake news detection on social media using hybrid Gaussian Mixture Model - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152553848AF4427305538400198FEC02.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="plosone">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360590/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS One">
<meta name="citation_title" content="Unsupervised fake news detection on social media using hybrid Gaussian Mixture Model">
<meta name="citation_author" content="Sajida Perveen">
<meta name="citation_author_institution" content="Department of computer Science, National Textile University, Faisalabad, Pakistan">
<meta name="citation_author" content="Muhammad Shahbaz">
<meta name="citation_author_institution" content="Department of Computer Engineering, University of Engineering &amp; Technology, Lahore, Pakistan">
<meta name="citation_author" content="Sami S Albouq">
<meta name="citation_author_institution" content="Faculty of Computer and Information Systems, Islamic University of Madinah, Madinah, Saudi Arabia">
<meta name="citation_author" content="Khlood Shinan">
<meta name="citation_author_institution" content="Department of Computers, College of Engineering and Computers in Al-Lith, Umm Al-Qura University, Makkah, Saudi Arabia">
<meta name="citation_author" content="Hanan E Alhazmi">
<meta name="citation_author_institution" content="Department of Cybersecurity, College of Computing, Umm Al-Qura University, Makkah, Saudi Arabia">
<meta name="citation_author" content="Fatmah Alanazi">
<meta name="citation_author_institution" content="Computer Science Department, College of Computer and Information Sciences, Imam Muhammad Bin Saud University, Riyadh, Saudi Arabia">
<meta name="citation_author" content="M Usman Ashraf">
<meta name="citation_author_institution" content="Department of Computer Science, GC Women University Sialkot, Pakistan">
<meta name="citation_author" content="Rehan Ashraf">
<meta name="citation_author_institution" content="Department of computer Science, National Textile University, Faisalabad, Pakistan">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="20">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e0330421">
<meta name="citation_doi" content="10.1371/journal.pone.0330421">
<meta name="citation_pmid" content="40824941">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360590/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360590/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360590/pdf/pone.0330421.pdf">
<meta name="description" content="The rise of social media has revolutionized information dissemination, creating new opportunities but also significant challenges. One such challenge is the proliferation of fake news, which undermines the credibility of journalism and contributes ...">
<meta name="og:title" content="Unsupervised fake news detection on social media using hybrid Gaussian Mixture Model">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The rise of social media has revolutionized information dissemination, creating new opportunities but also significant challenges. One such challenge is the proliferation of fake news, which undermines the credibility of journalism and contributes ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360590/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12360590">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pone.0330421"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pone.0330421.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12360590%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12360590/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12360590/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360590/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-plosone.png" alt="PLOS One logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS One" title="Link to PLOS One" shape="default" href="https://doi.org/10.1371/journal.pone.0330421" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS One</button></div>. 2025 Aug 18;20(8):e0330421. doi: <a href="https://doi.org/10.1371/journal.pone.0330421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330421</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20One%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20One%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Unsupervised fake news detection on social media using hybrid Gaussian Mixture Model</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Perveen%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Sajida Perveen</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Sajida Perveen</span></h3>
<div class="p">
<sup>1</sup>Department of computer Science, National Textile University, Faisalabad, Pakistan</div>
<div>Conceptualization, Methodology, Writing – original draft</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Perveen%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sajida Perveen</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shahbaz%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Muhammad Shahbaz</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Muhammad Shahbaz</span></h3>
<div class="p">
<sup>2</sup>Department of Computer Engineering, University of Engineering &amp; Technology, Lahore, Pakistan</div>
<div>Project administration, Supervision, Validation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shahbaz%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Muhammad Shahbaz</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Albouq%20SS%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Sami S Albouq</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Sami S Albouq</span></h3>
<div class="p">
<sup>3</sup>Faculty of Computer and Information Systems, Islamic University of Madinah, Madinah, Saudi Arabia</div>
<div>Data curation, Funding acquisition, Resources, Software</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Albouq%20SS%22%5BAuthor%5D" class="usa-link"><span class="name western">Sami S Albouq</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shinan%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Khlood Shinan</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Khlood Shinan</span></h3>
<div class="p">
<sup>4</sup>Department of Computers, College of Engineering and Computers in Al-Lith, Umm Al-Qura University, Makkah, Saudi Arabia</div>
<div>Project administration, Validation, Visualization, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shinan%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Khlood Shinan</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alhazmi%20HE%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Hanan E Alhazmi</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Hanan E Alhazmi</span></h3>
<div class="p">
<sup>5</sup>Department of Cybersecurity, College of Computing, Umm Al-Qura University, Makkah, Saudi Arabia</div>
<div>Investigation, Project administration, Visualization, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alhazmi%20HE%22%5BAuthor%5D" class="usa-link"><span class="name western">Hanan E Alhazmi</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alanazi%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Fatmah Alanazi</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Fatmah Alanazi</span></h3>
<div class="p">
<sup>6</sup>Computer Science Department, College of Computer and Information Sciences, Imam Muhammad Bin Saud University, Riyadh, Saudi Arabia</div>
<div>Data curation, Investigation, Resources</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alanazi%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fatmah Alanazi</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ashraf%20MU%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">M Usman Ashraf</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">M Usman Ashraf</span></h3>
<div class="p">
<sup>7</sup>Department of Computer Science, GC Women University Sialkot, Pakistan</div>
<div>Formal analysis, Funding acquisition</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ashraf%20MU%22%5BAuthor%5D" class="usa-link"><span class="name western">M Usman Ashraf</span></a>
</div>
</div>
<sup>7,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ashraf%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Rehan Ashraf</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Rehan Ashraf</span></h3>
<div class="p">
<sup>1</sup>Department of computer Science, National Textile University, Faisalabad, Pakistan</div>
<div>Conceptualization, Data curation, Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ashraf%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rehan Ashraf</span></a>
</div>
</div>
<sup>1</sup>
</div>
<div class="cg p">Editor: <span class="name western">Hirenkumar Kantilal Mewada</span><sup>8</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>Department of computer Science, National Textile University, Faisalabad, Pakistan</div>
<div id="aff002">
<sup>2</sup>Department of Computer Engineering, University of Engineering &amp; Technology, Lahore, Pakistan</div>
<div id="aff003">
<sup>3</sup>Faculty of Computer and Information Systems, Islamic University of Madinah, Madinah, Saudi Arabia</div>
<div id="aff004">
<sup>4</sup>Department of Computers, College of Engineering and Computers in Al-Lith, Umm Al-Qura University, Makkah, Saudi Arabia</div>
<div id="aff005">
<sup>5</sup>Department of Cybersecurity, College of Computing, Umm Al-Qura University, Makkah, Saudi Arabia</div>
<div id="aff006">
<sup>6</sup>Computer Science Department, College of Computer and Information Sciences, Imam Muhammad Bin Saud University, Riyadh, Saudi Arabia</div>
<div id="aff007">
<sup>7</sup>Department of Computer Science, GC Women University Sialkot, Pakistan</div>
<div id="edit1">
<sup>8</sup>Prince Mohammad Bin Fahd University, SAUDI ARABIA</div>
<div class="author-notes p">
<div class="fn" id="coi001"><p><strong>Competing Interests: </strong>The authors have declared that no competing interests exist.</p></div>
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>usman.ashraf@gcwus.edu.pk</span></p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Sajida Perveen</span></strong>: <span class="role">Conceptualization, Methodology, Writing – original draft</span>
</div>
<div>
<strong class="contrib"><span class="name western">Muhammad Shahbaz</span></strong>: <span class="role">Project administration, Supervision, Validation, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Sami S Albouq</span></strong>: <span class="role">Data curation, Funding acquisition, Resources, Software</span>
</div>
<div>
<strong class="contrib"><span class="name western">Khlood Shinan</span></strong>: <span class="role">Project administration, Validation, Visualization, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Hanan E Alhazmi</span></strong>: <span class="role">Investigation, Project administration, Visualization, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Fatmah Alanazi</span></strong>: <span class="role">Data curation, Investigation, Resources</span>
</div>
<div>
<strong class="contrib"><span class="name western">M Usman Ashraf</span></strong>: <span class="role">Formal analysis, Funding acquisition</span>
</div>
<div>
<strong class="contrib"><span class="name western">Rehan Ashraf</span></strong>: <span class="role">Conceptualization, Data curation, Investigation</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Hirenkumar Kantilal Mewada</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Nov 3; Accepted 2025 Jul 23; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Perveen et al</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12360590  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40824941/" class="usa-link">40824941</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The rise of social media has revolutionized information dissemination, creating new opportunities but also significant challenges. One such challenge is the proliferation of fake news, which undermines the credibility of journalism and contributes to societal unrest. Manually identifying fake news is impractical due to the vast volume of content, prompting the development of automated systems for fake news detection. This challenge has motivated numerous research efforts aimed at developing automated systems for fake news detection. However, most of these approaches rely on supervised learning, which requires significant time and effort to construct labeled datasets. While there have been a few attempts to develop unsupervised methods for fake news detection, their reported accuracy results thereof remain unsatisfactory. This research proposes an unsupervised approach using clustering algorithms, including Gaussian Mixture Model (GMM), K-means, and K-medoids, to eliminate the need for manual labeling in detecting fake news. In particular, it also proposes a novel hybrid method that leverages the Gaussian Mixture Model (GMM) in conjunction with the Group Counseling Optimizer (GCO), a metaheuristic optimization algorithm, to identify the optimal number of clusters for the detection of fake news. The comparative analysis of the evaluation results on real-world data demonstrated that the proposed hybrid GMM outperforms the state-of-the-art techniques, with a silhouette score of 0.77, ARI of 0.83, and a purity score of 0.88, indicating a significantly improved quality of clustering results.</p></section><section id="sec001"><h2 class="pmc_sec_title">1 Introduction</h2>
<p>The emergence of the Internet and the increasing popularity of social media platforms, such as Facebook and Twitter, have revolutionized the way information is disseminated in today’s world. [<a href="#pone.0330421.ref001" class="usa-link" aria-describedby="pone.0330421.ref001">1</a>,<a href="#pone.0330421.ref002" class="usa-link" aria-describedby="pone.0330421.ref002">2</a>]. This revolution has led news providers to transition from traditional media formats, such as newspapers and magazines, towards digital formats including online news platforms, blogs, and social media feeds, enabling them to deliver up-to-date news in near real-time to their subscribers [<a href="#pone.0330421.ref003" class="usa-link" aria-describedby="pone.0330421.ref003">3</a>,<a href="#pone.0330421.ref004" class="usa-link" aria-describedby="pone.0330421.ref004">4</a>]. As the internet usage has expanded, an increasing number of consumers turned away from traditional media channels, preferring to access information through online platforms [<a href="#pone.0330421.ref005" class="usa-link" aria-describedby="pone.0330421.ref005">5</a>,<a href="#pone.0330421.ref006" class="usa-link" aria-describedby="pone.0330421.ref006">6</a>]. This shift has made it incredibly convenient for consumers to access the latest news instantly [<a href="#pone.0330421.ref007" class="usa-link" aria-describedby="pone.0330421.ref007">7</a>,<a href="#pone.0330421.ref008" class="usa-link" aria-describedby="pone.0330421.ref008">8</a>]. Consequently, web portals have become a popular tool for delivering and receiving news, especially for users who are always on the move [<a href="#pone.0330421.ref009" class="usa-link" aria-describedby="pone.0330421.ref009">9</a>]. Notably, Facebook alone drives 70% of the traffic on news websites [<a href="#pone.0330421.ref010" class="usa-link" aria-describedby="pone.0330421.ref010">10</a>].</p>
<p>Social media platforms have become powerful tools for discussion, idea-sharing, and debate on topics like democracy, education, and health [<a href="#pone.0330421.ref011" class="usa-link" aria-describedby="pone.0330421.ref011">11</a>–<a href="#pone.0330421.ref014" class="usa-link" aria-describedby="pone.0330421.ref014">14</a>], enabling decentralized content creation and consumption [<a href="#pone.0330421.ref015" class="usa-link" aria-describedby="pone.0330421.ref015">15</a>]. However, alongside these benefits come significant challenges [<a href="#pone.0330421.ref002" class="usa-link" aria-describedby="pone.0330421.ref002">2</a>,<a href="#pone.0330421.ref011" class="usa-link" aria-describedby="pone.0330421.ref011">11</a>,<a href="#pone.0330421.ref016" class="usa-link" aria-describedby="pone.0330421.ref016">16</a>–<a href="#pone.0330421.ref019" class="usa-link" aria-describedby="pone.0330421.ref019">19</a>]. The overwhelming volume of information can make it difficult to distinguish credible sources [<a href="#pone.0330421.ref019" class="usa-link" aria-describedby="pone.0330421.ref019">19</a>,<a href="#pone.0330421.ref020" class="usa-link" aria-describedby="pone.0330421.ref020">20</a>]. Moreover, these platforms are often misused for financial gain, spreading biased views, manipulating opinions, and disseminating misinformation or fake news [<a href="#pone.0330421.ref021" class="usa-link" aria-describedby="pone.0330421.ref021">21</a>–<a href="#pone.0330421.ref024" class="usa-link" aria-describedby="pone.0330421.ref024">24</a>].</p>
<p>The phenomenon of fake news is not a recent development. It existed even long before the advent of the Internet [<a href="#pone.0330421.ref025" class="usa-link" aria-describedby="pone.0330421.ref025">25</a>]. Allcott and Gentzkow [<a href="#pone.0330421.ref026" class="usa-link" aria-describedby="pone.0330421.ref026">26</a>] initially defined the concept of fake news as news that is intentionally fabricated and verifiably false and designed to mislead readers [<a href="#pone.0330421.ref027" class="usa-link" aria-describedby="pone.0330421.ref027">27</a>]. Another widely used definition of fake news is that fake news itself is not actual but is made real for a specific purpose [<a href="#pone.0330421.ref028" class="usa-link" aria-describedby="pone.0330421.ref028">28</a>]. Additionally, the dissemination of fake news on social media platforms is exacerbated by its user-friendly and rapid sharing capabilities, which provide remarkable convenience in the creation and distribution of misinformation, further imploring the audience to disseminate the content at an exponential pace on the Internet. Unlike social media, mainstream media follow editorial standards and guidelines, and are operated by professional journalists [<a href="#pone.0330421.ref029" class="usa-link" aria-describedby="pone.0330421.ref029">29</a>]. On the other hand, social media primarily relies on user-generated content with limited editorial controls or gatekeeping procedures [<a href="#pone.0330421.ref030" class="usa-link" aria-describedby="pone.0330421.ref030">30</a>]. Instead of encountering reliable and unbiased information, users are inundated with a multitude of fake news on social media platforms [<a href="#pone.0330421.ref031" class="usa-link" aria-describedby="pone.0330421.ref031">31</a>,<a href="#pone.0330421.ref032" class="usa-link" aria-describedby="pone.0330421.ref032">32</a>]. Therefore, social media has also become a breeding ground for misinformation [<a href="#pone.0330421.ref033" class="usa-link" aria-describedby="pone.0330421.ref033">33</a>].</p>
<p>There have been numerous examples of fake news throughout history. A recent example is the proliferation of information about COVID-19. In late 2019, massive amounts of informative data related to COVID-19 pandemic were generated and disseminated on social media networks. Although, these networks make it possible to disseminate information to a large audience [<a href="#pone.0330421.ref034" class="usa-link" aria-describedby="pone.0330421.ref034">34</a>]. Unfortunately, not all disseminated information about the outbreak is accurate and trustworthy. During this era, some of the information disseminated around these networks was identified as fake news or even exacerbated the crisis by promoting unproven treatments and fueling vaccine hesitancy leading to detrimental health, social, and cultural consequences [<a href="#pone.0330421.ref035" class="usa-link" aria-describedby="pone.0330421.ref035">35</a>]. Thus, WHO called it an “information epidemic”. Fake news can also incite violence, panic, or harm to public safety in various contexts [<a href="#pone.0330421.ref036" class="usa-link" aria-describedby="pone.0330421.ref036">36</a>,<a href="#pone.0330421.ref037" class="usa-link" aria-describedby="pone.0330421.ref037">37</a>]. Unfortunately, the epidemic spread of fake news is a byproduct of the popularity of the social media platforms. According to a report, only 26% of the participants expressed a high level of confidence in discriminating between real and fake [<a href="#pone.0330421.ref028" class="usa-link" aria-describedby="pone.0330421.ref028">28</a>,<a href="#pone.0330421.ref038" class="usa-link" aria-describedby="pone.0330421.ref038">38</a>]. This ratio is comparatively low and indicates insufficient capacity to discriminate between genuine and fake news.</p>
<p>Consequently, it has become increasingly challenging for authoritative institutions to effectively combat the rapid dissemination of fake news. Although significant efforts have been made to address this issue, researchers are increasingly turning to artificial intelligence (AI) solutions [<a href="#pone.0330421.ref039" class="usa-link" aria-describedby="pone.0330421.ref039">39</a>]. Machine learning, particularly natural language processing techniques, have shown promising contribution in fake news detection [<a href="#pone.0330421.ref040" class="usa-link" aria-describedby="pone.0330421.ref040">40</a>–<a href="#pone.0330421.ref043" class="usa-link" aria-describedby="pone.0330421.ref043">43</a>]. Other researchers [<a href="#pone.0330421.ref044" class="usa-link" aria-describedby="pone.0330421.ref044">44</a>–<a href="#pone.0330421.ref046" class="usa-link" aria-describedby="pone.0330421.ref046">46</a>] explored to utilize a combination of different deep learning models. Some have even demonstrated that deep learning techniques outperform traditional machine-learning techniques [<a href="#pone.0330421.ref046" class="usa-link" aria-describedby="pone.0330421.ref046">46</a>]. However, the inherent “black-box” nature of advanced deep learning models presents significant challenges, particularly in fields requiring transparency and interpretability [<a href="#pone.0330421.ref047" class="usa-link" aria-describedby="pone.0330421.ref047">47</a>].</p>
<p>According to the existing literature, considerable attention has been given to machine learning-based supervised methods for detecting fake news [<a href="#pone.0330421.ref041" class="usa-link" aria-describedby="pone.0330421.ref041">41</a>–<a href="#pone.0330421.ref043" class="usa-link" aria-describedby="pone.0330421.ref043">43</a>]. These studies developed classification models using different sets of features including user profiles, news content [<a href="#pone.0330421.ref029" class="usa-link" aria-describedby="pone.0330421.ref029">29</a>], message propagation [<a href="#pone.0330421.ref048" class="usa-link" aria-describedby="pone.0330421.ref048">48</a>] and social contexts [<a href="#pone.0330421.ref023" class="usa-link" aria-describedby="pone.0330421.ref023">23</a>,<a href="#pone.0330421.ref049" class="usa-link" aria-describedby="pone.0330421.ref049">49</a>–<a href="#pone.0330421.ref051" class="usa-link" aria-describedby="pone.0330421.ref051">51</a>]. While these models have shown promising results to some extent, these supervised methods require labeled data to build a classification model [<a href="#pone.0330421.ref052" class="usa-link" aria-describedby="pone.0330421.ref052">52</a>,<a href="#pone.0330421.ref053" class="usa-link" aria-describedby="pone.0330421.ref053">53</a>].</p>
<p>However, obtaining a large number of annotations is time consuming and labor intensive, as the process requires careful examination of news content, as well as other additional evidence such as authoritative reports [<a href="#pone.0330421.ref054" class="usa-link" aria-describedby="pone.0330421.ref054">54</a>,<a href="#pone.0330421.ref055" class="usa-link" aria-describedby="pone.0330421.ref055">55</a>]. Leveraging a crowdsourcing approach to obtain annotations could alleviate the burden of expert checking; however, the quality of annotations may be compromised [<a href="#pone.0330421.ref056" class="usa-link" aria-describedby="pone.0330421.ref056">56</a>]. As fake news is intentionally written to mislead readers [<a href="#pone.0330421.ref057" class="usa-link" aria-describedby="pone.0330421.ref057">57</a>], individual human workers alone may not possess the domain expertise to differentiate between real and fake news [<a href="#pone.0330421.ref058" class="usa-link" aria-describedby="pone.0330421.ref058">58</a>]. Besides this, fake news detection often involves identifying and tracking patterns in data that change dynamically over time, such as shifts in the spread of misinformation, the emergence of new narratives, or changes in the behavior of online communities. These evolving patterns require models that can adapt to and account for changes in the data distribution, similar to how dynamic systems evolve based on state variables and external inputs.</p>
<p>In the context of Gaussian Mixture Models (GMMs) [<a href="#pone.0330421.ref059" class="usa-link" aria-describedby="pone.0330421.ref059">59</a>], these models have been widely applied for modeling dynamic systems to capture the underlying probabilistic structure of time-varying processes [<a href="#pone.0330421.ref060" class="usa-link" aria-describedby="pone.0330421.ref060">60</a>–<a href="#pone.0330421.ref067" class="usa-link" aria-describedby="pone.0330421.ref067">67</a>]. Similarly, in fake news detection, GMMs can be used to model the distribution of news articles or social media posts that change over time, grouping them into clusters that reflect the underlying patterns of authenticity versus misinformation [<a href="#pone.0330421.ref058" class="usa-link" aria-describedby="pone.0330421.ref058">58</a>]. By leveraging GMMs, we can identify clusters of fake and real news articles that evolve as new information is disseminated, without requiring labeled data for training [<a href="#pone.0330421.ref068" class="usa-link" aria-describedby="pone.0330421.ref068">68</a>].</p>
<p>Although, GMM is a potentially powerful technique for fake news detection, it has several limitations when applied to textual data [<a href="#pone.0330421.ref068" class="usa-link" aria-describedby="pone.0330421.ref068">68</a>]. As, it relies on the assumption that data is generated from a mixture of Gaussian distributions, which may not hold for textual data, as these features often exhibit complex, non-Gaussian patterns [<a href="#pone.0330421.ref069" class="usa-link" aria-describedby="pone.0330421.ref069">69</a>]. Additionally, textual data is typically high-dimensional, and GMM can struggle with the curse of dimensionality [<a href="#pone.0330421.ref064" class="usa-link" aria-describedby="pone.0330421.ref064">64</a>,<a href="#pone.0330421.ref070" class="usa-link" aria-describedby="pone.0330421.ref070">70</a>–<a href="#pone.0330421.ref073" class="usa-link" aria-describedby="pone.0330421.ref073">73</a>]. The Expectation-Maximization (EM) algorithm used in GMM is also sensitive to choice of initial parameters selection, potentially leading to convergence on local optima rather than the global optimum, leading to erroneous results [<a href="#pone.0330421.ref074" class="usa-link" aria-describedby="pone.0330421.ref074">74</a>–<a href="#pone.0330421.ref076" class="usa-link" aria-describedby="pone.0330421.ref076">76</a>]. Overall, these limitations highlight the need for additional techniques or enhancements to improve the model’s effectiveness in fake news detection. Unlike EM algorithm, GCO, a meta-heuristic optimization technique, explores the parameter space more effectively through iterative improvements based on group counseling behavior, helping find the global optimum and avoiding local optima [<a href="#pone.0330421.ref077" class="usa-link" aria-describedby="pone.0330421.ref077">77</a>,<a href="#pone.0330421.ref078" class="usa-link" aria-describedby="pone.0330421.ref078">78</a>]. GCO handles large datasets well, making it suitable for real-world news classification [<a href="#pone.0330421.ref079" class="usa-link" aria-describedby="pone.0330421.ref079">79</a>]. To address these challenges, this paper presents a novel hybrid Gaussian Mixture model, GCOGMM, a framework to identify fake news in a fully unsupervised manner. The primary objective of this study is to optimize parameters selection and avoid local optima, thereby enhancing the model’s ability to effectively handle complex, high-dimensional textual data to effectively identify fake information.</p>
<p>The following are the key aspects of this paper:</p>
<ol class="list" style="list-style-type:decimal">
<li><p>The proposed research integrates Group Counseling Optimization (GCO) with the Gaussian Mixture Model (GMM) to optimize parameters selection and avoid local optima, thereby enhancing the model’s ability to effectively handle complex, high-dimensional textual data.</p></li>
<li><p>To explore the automatic fake news detection based on the analysis of surface-level linguistic patterns.</p></li>
<li><p>A comprehensive evaluation analysis of the proposed approach is proformed on benchmark dataset. Particularly, for evaluation, the proposed research developed three baseline unsupervised methods for detecting fake news, namely k-means, K-mediods and Gaussian Mixture Models (GMM).</p></li>
<li><p>Additionally, to the best of available knowledge, the utility of the proposed approach when combined with standard GMM has not been investigated to address the issue of fake news identification on social media, which has become a daunting societal and individual issue. Furthermore, evaluation results depicted that the proposed method outperforms over all the baseline models. It also demonstrated that this hybrid approach can improve a text-only Gaussian model.</p></li>
</ol></section><section id="sec002"><h2 class="pmc_sec_title">2 Proposed methodology</h2>
<section id="sec003"><h3 class="pmc_sec_title">2.1 Data representation</h3>
<p>In this study, we aimed at to enhance the current state of the art in fake news detection, with the ultimate goal of developing reliable and robust tools to combat misinformation in the digital era. To achieve this, we utilized the publicly available LIAR dataset for fake news detection, collected from POLITIFACT.COM. This dataset comprises 12.8K short statements related to various contexts collected over a decade. After data acquisition, the next crucial step is data processing. These preprocessing steps are vital for cleaning and standardizing such textual data, making it more suitable for natural language processing (NLP) tasks.’</p>
<p>The dataset provides a detailed classification of news articles into six subcategories: True, Mostly True, Half True, Barely True, False, and Pants on Fire. These subcategories offer a nuanced perspective on the varying degrees of truthfulness within the dataset. These subcategories are represented by varying frequencies within the dataset: Half True (2,114 instances), False (1,994 instances), Mostly True (1,962 instances), True (1,676 instances), Barely True (1,654 instances), and Pants on Fire (839 instances). Each data point in the dataset have a unique ID. This distribution reveals an imbalance in the dataset, with certain categories like Half True and False being more prevalent, while others such as Pants on Fire are significantly less represented. This imbalance highlights the complexities of truthfulness identification and may pose significant challenges during the analysis process. To address this problem and ensure an equitable representation across all the categories, we balanced the dataset by randomly selecting 839 instances from each category without replacement. This approach helps to create a more uniform dataset. However, to ensure that the clusters formed reflect the actual structure of the data, rather than being influenced by human-imposed labels that may not fully capture the complexity or nuances of the dataset, all pre-existing labels are removed before conducting the clustering analysis. This approach aligns with the principles of unsupervised learning and is essential for achieving accurate, data-driven clustering outcomes in the context of this study.</p></section><section id="sec004"><h3 class="pmc_sec_title">2.2 Preprocessing</h3>
<section id="sec005"><h4 class="pmc_sec_title">2.2.1 Converting all data to lower case.</h4>
<p>As a written document is composed of sentences, and sentences can comprise a combination of uppercase and lowercase letters, the initial preprocessing step involves converting all text data to a single text case. We have chosen lowercase for this purpose. This is crucial to ensure consistency in the text analysis procedure [<a href="#pone.0330421.ref080" class="usa-link" aria-describedby="pone.0330421.ref080">80</a>,<a href="#pone.0330421.ref081" class="usa-link" aria-describedby="pone.0330421.ref081">81</a>]. By converting everything to lowercase, we eliminate potential inconsistencies that may arise from various cases of the same word.</p></section><section id="sec006"><h4 class="pmc_sec_title">2.2.2 Tokenization.</h4>
<p>It is a crucial step in NLP where a stream of text is divided into smaller units, which typically referred to as tokens [<a href="#pone.0330421.ref082" class="usa-link" aria-describedby="pone.0330421.ref082">82</a>]. These tokens can be words, phrases, symbols, or individual characters, depending on the level of granularity needed for the analysis. This process is essential for many NLP tasks as it facilitates the understanding and manipulation of text by converting it into a format that computational models can process. Thus tokenization is often the first step in a pipeline that includes other tasks, which collectively enhance the performance of NLP applications [<a href="#pone.0330421.ref082" class="usa-link" aria-describedby="pone.0330421.ref082">82</a>]. Therefore, tokenization is performed on each sentence in the dataset.</p></section><section id="sec007"><h4 class="pmc_sec_title">2.2.3 Removing non-alphanumeric characters.</h4>
<p>Following tokenization, it is quite common to remove non-alphanumeric characters from the text. This particular step includes eliminating symbols, punctuation marks, and special characters that do not hold substantial semantic meaning in the text [<a href="#pone.0330421.ref083" class="usa-link" aria-describedby="pone.0330421.ref083">83</a>]. Examples of non-alphanumeric characters include commas, periods, exclamation marks, and other symbols that are not part of words or numbers.</p></section><section id="sec008"><h4 class="pmc_sec_title">2.2.4 Removing stop words and punctuation.</h4>
<p>Stop words are indeed terms that are commonly found in documents and are considered to have little semantic meaning in the context of text analysis. Their removal is a standard preprocessing step in NLP and text mining applications to improve the performance and efficiency of these systems [<a href="#pone.0330421.ref084" class="usa-link" aria-describedby="pone.0330421.ref084">84</a>]. Therefore removed from the text during the analysis process. Examples of stop words include “the,” “is,” “and,” “of,” and so on. The removal of stop words helps to reduce noise in the text data and concentrate on more significant terms. Moreover, the elimination of punctuation marks further refines the text and prepares it for analysis. NLTK library in Python contains a list of stop words for the English language.</p></section><section id="sec009"><h4 class="pmc_sec_title">2.2.5 Stemming.</h4>
<p>Stemming is a process widely used to reduce words to their derived words known as stem, base, or root form. This process typically entails the elimination of extraneous characters to normalize words, which involves removing prefixes and suffixes [<a href="#pone.0330421.ref085" class="usa-link" aria-describedby="pone.0330421.ref085">85</a>,<a href="#pone.0330421.ref086" class="usa-link" aria-describedby="pone.0330421.ref086">86</a>]. For instance, the words “write”, “wrote,”, “written” and “writing” would all be stemmed to the base form “write” By stemming words, the vocabulary size is reduced, and similar words are consolidated, which can enhance the effectiveness of text analysis algorithms [<a href="#pone.0330421.ref087" class="usa-link" aria-describedby="pone.0330421.ref087">87</a>]. Porter Stemmer is one of the stemming model which is used in this study to convert the words into their root form.</p></section><section id="sec010"><h4 class="pmc_sec_title">2.2.6 Creating word cloud.</h4>
<p>After applying the aforementioned preprocessing steps, a high-dimensional feature space of 184,412 unique tokens were obtained and are reunited to form a processed text dataset that is prepared for further analysis. This consolidated text preserves the pertinent semantic information while eradicating extraneous details and noise, making it appropriate for tasks such as text classification, sentiment analysis, or information retrieval. <a href="#pone.0330421.g001" class="usa-link">Fig 1</a> showcases a more detailed representation of the dataset in the form of word cloud. The conspicuous presence of political news and the substantial inclusion of global events indicate particular thematic priorities within the media. This observation invites a deeper analysis of the content, aiming to uncover potential underlying influences that may be shaping the portrayal of political and global events in the dataset.</p>
<figure class="fig xbox font-sm" id="pone.0330421.g001"><h5 class="obj_head">Fig 1. Abstract representation of the dataset in the form of word cloud.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360590_pone.0330421.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a3e1/12360590/9db44cf7adcd/pone.0330421.g001.jpg" loading="lazy" height="338" width="675" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330421.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section></section><section id="sec011"><h3 class="pmc_sec_title">2.3 Feature extraction or vectorization</h3>
<section id="sec012"><h4 class="pmc_sec_title">2.3.1 Text vectorization using term frequencey-inverse document freuency.</h4>
<p>The choice of metrics for encoding textual data is pivotal in determining the performance and suitability of a particular model. While advanced encoding methods like contextual embeddings or topic modeling techniques (e.g., BERT, LDA) offer deeper semantic understanding, their high computational cost and preprocessing requirements [<a href="#pone.0330421.ref088" class="usa-link" aria-describedby="pone.0330421.ref088">88</a>] may not align with the study’s objectives. As the primary research objective is to enhance the efficacy of detecting fake news using unsupervised techniques. Therefore, to achieve this, TF-IDF (Term Frequency-Inverse Document Frequency) [<a href="#pone.0330421.ref089" class="usa-link" aria-describedby="pone.0330421.ref089">89</a>] was chosen based on its balance between simplicity, interpretability, and its effectiveness in identifying informative features within textual data. It excels in distinguishing relevant terms by penalizing high-frequency words that are common across documents, making it suitable for clustering tasks where capturing term-specific relevance is crucial.</p>
<p>Vectorizer as a critical element in the preprocessing of textual news data. The TF-IDF Vectorizer is a widely adopted technique in natural language processing, taking into account both the frequency of terms in individual documents and their significance across the entire corpus [<a href="#pone.0330421.ref088" class="usa-link" aria-describedby="pone.0330421.ref088">88</a>,<a href="#pone.0330421.ref090" class="usa-link" aria-describedby="pone.0330421.ref090">90</a>]. The vectorization process essentially entails assigning numerical values to words based on their occurrence frequency in a document (Term Frequency) and inversely scaling these values by their frequency in the entire corpus (Inverse Document Frequency), thereby emphasizing terms that are both frequent in a document and unique across the corpus. Mathematically, the TF-IDF can be expressed as:</p>
<table class="disp-formula p" id="pone.0330421.e001"><tr>
<td class="formula"><math id="M1" display="block" overflow="linebreak"><mrow><mi>t</mi><mi>f</mi><mo>−</mo><msub><mrow><mi>i</mi><mi>d</mi><mi>f</mi></mrow><mrow><mi>t</mi><mo>,</mo><mi>D</mi></mrow></msub><mo>=</mo><msub><mrow><mi>T</mi><mi>F</mi></mrow><mrow><mi>t</mi><mo>,</mo><mi>D</mi></mrow></msub><mo>*</mo><msub><mrow><mi>I</mi><mi>D</mi><mi>F</mi></mrow><mrow><mi>t</mi></mrow></msub></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<table class="disp-formula p" id="pone.0330421.e002"><tr>
<td class="formula"><math id="M2" display="block" overflow="linebreak"><mrow><msub><mrow><mi>T</mi><mi>F</mi></mrow><mrow><mi>t</mi><mo>,</mo><mi>D</mi></mrow></msub><mo>=</mo><mfrac><mrow><mrow><mi mathvariant="normal">N</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">b</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">O</mi><mi mathvariant="normal">f</mi><mi mathvariant="normal">R</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">p</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">f</mi><mi mathvariant="normal">T</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">m</mi></mrow><mi>t</mi><mrow><mi mathvariant="normal">i</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">D</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">D</mi></mrow></mrow><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">o</mi><mo>.</mo><mi mathvariant="normal">o</mi><mi mathvariant="normal">f</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">D</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">t</mi></mrow></mfrac></mrow></math></td>
<td class="label">(2)</td>
</tr></table>
<p>And</p>
<table class="disp-formula p" id="pone.0330421.e003"><tr>
<td class="formula"><math id="M3" display="block" overflow="linebreak"><mrow><msub><mrow><mi>I</mi><mi>D</mi><mi>F</mi></mrow><mrow><mi>t</mi></mrow></msub><mo>=</mo><mi>log</mi><mfrac><mrow><mi mathvariant="normal">N</mi><mi mathvariant="normal">o</mi><mo>.</mo><mi mathvariant="normal">o</mi><mi mathvariant="normal">f</mi><mi mathvariant="normal">D</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">s</mi></mrow><mrow><mrow><mi mathvariant="normal">N</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">b</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">O</mi><mi mathvariant="normal">f</mi><mi mathvariant="normal">D</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">C</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">g</mi><mi mathvariant="normal">T</mi><mi mathvariant="normal">h</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">m</mi></mrow><mi>t</mi></mrow></mfrac></mrow></math></td>
<td class="label">(3)</td>
</tr></table>
<p>Thus, the TF-IDF is instrumental in transforming unstructured textual information into a format that is compatible with machine learning techniques [<a href="#pone.0330421.ref091" class="usa-link" aria-describedby="pone.0330421.ref091">91</a>]. It represents text as numerical vectors enables algorithms to operate on the data efficiently to extract meaningful patterns, and make informed decisions.</p></section><section id="sec013"><h4 class="pmc_sec_title">2.3.2 Principal Component Analysis (PCA).</h4>
<p>After carrying out above mentioned preprocessing tasks, it is necessary to decrease the number of features to lower computational complexity and improve performance [<a href="#pone.0330421.ref092" class="usa-link" aria-describedby="pone.0330421.ref092">92</a>–<a href="#pone.0330421.ref094" class="usa-link" aria-describedby="pone.0330421.ref094">94</a>]. The optimal selection of feature extraction algorithm is crucial to the quality of the results [<a href="#pone.0330421.ref095" class="usa-link" aria-describedby="pone.0330421.ref095">95</a>]. According to existing literature to deal with the high dimensional data Information Gain (IG) [<a href="#pone.0330421.ref096" class="usa-link" aria-describedby="pone.0330421.ref096">96</a>], Mutual Information [<a href="#pone.0330421.ref097" class="usa-link" aria-describedby="pone.0330421.ref097">97</a>,<a href="#pone.0330421.ref098" class="usa-link" aria-describedby="pone.0330421.ref098">98</a>], Gini Coefficient (GI), Term Frequency-Inverse Document Frequency (TF-IDF), Principal Component Analysis (PCA) [<a href="#pone.0330421.ref027" class="usa-link" aria-describedby="pone.0330421.ref027">27</a>,<a href="#pone.0330421.ref099" class="usa-link" aria-describedby="pone.0330421.ref099">99</a>,<a href="#pone.0330421.ref100" class="usa-link" aria-describedby="pone.0330421.ref100">100</a>] and Chi-Square Statistics (CHI) [<a href="#pone.0330421.ref101" class="usa-link" aria-describedby="pone.0330421.ref101">101</a>] are frequently used for feature extraction. However, to improve the scalability of the obtained results, PCA is widely used as the dimensionality reduction approach in fake news detection applications [<a href="#pone.0330421.ref102" class="usa-link" aria-describedby="pone.0330421.ref102">102</a>–<a href="#pone.0330421.ref106" class="usa-link" aria-describedby="pone.0330421.ref106">106</a>].</p>
<p>PCA applies a linear transformation to reduce the dimensionality of a feature set, simplifying the dataset while preserving the essential characteristics of the original data [<a href="#pone.0330421.ref102" class="usa-link" aria-describedby="pone.0330421.ref102">102</a>]. The transformed dataset may have the same or fewer features compared to the original. Principal components are derived by calculating the covariance matrix [<a href="#pone.0330421.ref107" class="usa-link" aria-describedby="pone.0330421.ref107">107</a>]. These components are arranged in decreasing order of importance.</p>
<p>Let X denote the Dataset with n observations and p variables and Σ represent the covariance matrix of mean centered dataset X ´. Σ matrix captures the pairwise covariance between variables as depicted below:</p>
<table class="disp-formula p" id="pone.0330421.e004"><tr>
<td class="formula"><math id="M4" display="block" overflow="linebreak"><mrow><mtable columnspacing="1" displaystyle="true" rowspacing="3pt"><mtr><mtd columnalign="right"><mi>Σ</mi><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mi>n</mi><mo>−</mo><mn>1</mn></mrow></mfrac><mo>∑</mo><msubsup><mi>\nolimits</mi><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup><mrow><msup><mrow><mo stretchy="false">(</mo><msub><mrow><mover><mrow><mi>X</mi></mrow><mo>´</mo></mover></mrow><mrow><mi>i</mi></mrow></msub><mo>−</mo><mover><mrow><mi>X</mi></mrow><mo accent="true">―</mo></mover><mo stretchy="false">)</mo></mrow><mrow><mi>T</mi></mrow></msup><mo stretchy="false">(</mo><msub><mrow><mover><mrow><mi>X</mi></mrow><mo>´</mo></mover></mrow><mrow><mi>i</mi></mrow></msub><mo>−</mo><mover><mrow><mi>X</mi></mrow><mo accent="true">―</mo></mover><mo stretchy="false">)</mo></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">(4)</td>
</tr></table>
<p>To find principal components eigenvalue decomposition of Σ matrix is performed. Subsequently the eigenvectors corresponding to the largest eigenvalues were selected as principal components as depicted below:</p>
<table class="disp-formula p" id="pone.0330421.e005"><tr>
<td class="formula"><math id="M5" display="block" overflow="linebreak"><mrow><mi>Σ</mi><mo>=</mo><mi>Q</mi><mo>⋀</mo><msup><mi>Q</mi><mrow><mi>T</mi></mrow></msup></mrow></math></td>
<td class="label">(5)</td>
</tr></table>
<p>Q is the matrix of eigenvectors of Σ derived from the text vectors, where each column represents an eigenvector. Where Λ is a diagonal matrix containing the corresponding eigenvalues.</p>
<p>This reduction in dimensionality facilitated faster processing and aided in uncovering underlying patterns and structures within the textual data. The transformed data, now represented in a lower-dimensional feature space, served as the input for our clustering algorithm as depicted in <a href="#pone.0330421.g002" class="usa-link">Fig 2</a>.</p>
<figure class="fig xbox font-sm" id="pone.0330421.g002"><h5 class="obj_head">Fig 2. A visual representation of input data points transformed into a lower-dimensional feature space using PCA.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360590_pone.0330421.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a3e1/12360590/8e9d1f4ddcc4/pone.0330421.g002.jpg" loading="lazy" height="581" width="682" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330421.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec014"><h4 class="pmc_sec_title">2.3.3 Gaussian mixture model.</h4>
<p>A Gaussian Mixture Model (GMM) is an unsupervised probabilistic model that represents a dataset as a mixture of several Gaussian distributions [<a href="#pone.0330421.ref108" class="usa-link" aria-describedby="pone.0330421.ref108">108</a>]. GMM is parameterized by two type of values, the component weight and the component mean and variance. For a GMM with k Gaussian component, each K in the mixture model represents a cluster or subgroup within the data. GMM serves as an effective means for addressing intricate data distributions that cannot be easily segregated using conventional clustering techniques. In a multivariate GMM, each K is characterized by its mean <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e006"><math id="M6" display="inline" overflow="linebreak"><mrow><msub><mi>μ</mi><mrow><mi>K</mi></mrow></msub></mrow></math></span> Covariance matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e007"><math id="M7" display="inline" overflow="linebreak"><mrow><msub><mi>Σ</mi><mrow><mi>K</mi></mrow></msub></mrow></math></span> and weight <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e008"><math id="M8" display="inline" overflow="linebreak"><mrow><msub><mi>π</mi><mrow><mi>K</mi></mrow></msub></mrow></math></span> the probability of belonging to component k. The key components of a multivariate GMM include the Probability Density Function (PDF) of a multivariate Gaussian distribution, the mixture model PDF, and the Expectation-Maximization (EM) algorithm for parameter estimation. The probability density function of a multivariate Gaussian distribution with mean vector and covariance matrix is calculated as: below:</p>
<table class="disp-formula p" id="pone.0330421.e010"><tr>
<td class="formula"><math id="M10" display="block" overflow="linebreak"><mrow><mrow><mi mathvariant="script">N</mi><mo stretchy="false">(</mo></mrow><mi>x</mi><mo stretchy="false">|</mo><mi>μ</mi><mo>,</mo><mi>Σ</mi><mo stretchy="false">)</mo><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mo stretchy="false">(</mo><mn>2</mn><mi>π</mi><msup><mo stretchy="false">)</mo><mrow><mfrac><mrow><mi>D</mi></mrow><mrow><mn>2</mn></mrow></mfrac></mrow></msup><msup><mrow><mo stretchy="false">|</mo><mi>Σ</mi><mo stretchy="false">|</mo></mrow><mrow><mn>1</mn><mo>/</mo><mn>2</mn></mrow></msup></mrow></mfrac><mi>exp</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><mo>−</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mn>2</mn></mrow></mfrac><msup><mrow><mo stretchy="false">(</mo><mi>x</mi><mo>−</mo><mi>μ</mi><mo stretchy="false">)</mo></mrow><mrow><mi>T</mi></mrow></msup><msup><mi>Σ</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup><mo stretchy="false">(</mo><mi>x</mi><mo>−</mo><mi>μ</mi><mo stretchy="false">)</mo><mo fence="true" form="postfix" stretchy="true">)</mo></mrow></mrow></math></td>
<td class="label">(6)</td>
</tr></table>
<p>Where x is a D-dimensional data point μ is the D-dimensional mean vector, Σ is a D × D variance matrix, |Σ| determinant of the variance matrix and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e011"><math id="M11" display="inline" overflow="linebreak"><mrow><msup><mi>Σ</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow></math></span>is the inverse of the variance matrix, allowing the model to capture complex, multidimensional relationships within the data..</p>
<p>The PDF of a multivariate Gaussian Mixture Model (GMM) with K components is given by the weighted sum of the PDFs of its components as depicted below:</p>
<table class="disp-formula p" id="pone.0330421.e012"><tr>
<td class="formula"><math id="M12" display="block" overflow="linebreak"><mrow><mtable columnspacing="1" displaystyle="true" rowspacing="3pt"><mtr><mtd columnalign="right"><mi>P</mi><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo><mo>=</mo><mo>∑</mo><msubsup><mi>\nolimits</mi><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>K</mi></mrow></msubsup><mrow><msub><mi>π</mi><mrow><mi>k</mi></mrow></msub><mrow><mtext mathvariant="script"> </mtext><mi mathvariant="script">N</mi></mrow><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">|</mo><msub><mi>μ</mi><mrow><mi>k</mi></mrow></msub><msub><mi>Σ</mi><mrow><mi>k</mi></mrow></msub><mo stretchy="false">)</mo></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">(7)</td>
</tr></table>
<p>Here P represents the probability of component k such that <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e013"><math id="M13" display="inline" overflow="linebreak"><mrow><msubsup><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>K</mi></mrow></msubsup><msub><mi>π</mi><mrow><mi>k</mi></mrow></msub><mo>=</mo><mn>1</mn></mrow></math></span>. Whereas <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e014"><math id="M14" display="inline" overflow="linebreak"><mrow><msub><mi>μ</mi><mrow><mi>k</mi></mrow></msub></mrow></math></span> is the D-dimensional mean vector of component k, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e015"><math id="M15" display="inline" overflow="linebreak"><mrow><msub><mi>Σ</mi><mrow><mi>k</mi></mrow></msub></mrow></math></span> is the variance matrix of component k and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e016"><math id="M16" display="inline" overflow="linebreak"><mrow><mi>𝒩</mi></mrow></math></span> (x|<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e017"><math id="M17" display="inline" overflow="linebreak"><mrow><msub><mi>μ</mi><mrow><mi>k</mi></mrow></msub></mrow></math></span>,<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e018"><math id="M18" display="inline" overflow="linebreak"><mrow><msub><mi>Σ</mi><mrow><mi>k</mi></mrow></msub></mrow></math></span>) is the multivariate Gaussian PDF of component k. Subsequently, Expectation-Maximization (EM) Algorithm for Multivariate GMM consists of two steps. The first step, known as the expectation step to calculate the probability for each data point <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e019"><math id="M19" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mrow><mi>i</mi></mrow></msub></mrow></math></span> to each <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e020"><math id="M20" display="inline" overflow="linebreak"><mrow><msub><mi>C</mi><mrow><mi>k</mi></mrow></msub></mrow></math></span> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e021"><math id="M21" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mrow><mi>i</mi></mrow></msub><mo>∈</mo><mi>X</mi></mrow></math></span> given the model parameters |<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e022"><math id="M22" display="inline" overflow="linebreak"><mrow><msub><mi>μ</mi><mrow><mi>k</mi></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e023"><math id="M23" display="inline" overflow="linebreak"><mrow><msub><mi>Σ</mi><mrow><mi>k</mi></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e024"><math id="M24" display="inline" overflow="linebreak"><mrow><msub><mi>π</mi><mrow><mi>k</mi></mrow></msub></mrow></math></span>.</p>
<table class="disp-formula p" id="pone.0330421.e025"><tr>
<td class="formula"><math id="M25" display="block" overflow="linebreak"><mrow><mi>γ</mi><mo stretchy="false">(</mo><msub><mi>𝒵</mi><mrow><mi>n</mi><mi>k</mi></mrow></msub><mo stretchy="false">)</mo><mo>=</mo><mfrac><mrow><msub><mi>π</mi><mrow><mi>k</mi></mrow></msub><mrow><mo mathvariant="script">.</mo><mtext mathvariant="script"> </mtext><mi mathvariant="script">N</mi><mo stretchy="false">(</mo></mrow><msub><mi>x</mi><mrow><mi>k</mi></mrow></msub><mo stretchy="false">|</mo><msub><mi>μ</mi><mrow><mi>k</mi></mrow></msub><mo>,</mo><msub><mi>Σ</mi><mrow><mi>k</mi></mrow></msub><mo stretchy="false">)</mo></mrow><mrow><msubsup><mo>∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>K</mi></mrow></msubsup><msub><mi>π</mi><mrow><mi>j</mi></mrow></msub><mrow><mo mathvariant="script">.</mo><mtext mathvariant="script"> </mtext><mi mathvariant="script">N</mi><mo stretchy="false">(</mo></mrow><msub><mi>x</mi><mrow><mi>j</mi></mrow></msub><mo stretchy="false">|</mo><msub><mi>μ</mi><mrow><mi>j</mi></mrow></msub><mo>,</mo><msub><mi>Σ</mi><mrow><mi>j</mi></mrow></msub><mo stretchy="false">)</mo></mrow></mfrac></mrow></math></td>
<td class="label">(8)</td>
</tr></table>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e026"><math id="M26" display="inline" overflow="linebreak"><mrow><mi>γ</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>𝒵</mi><mrow><mi>n</mi><mi>k</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">)</mo></mrow></mrow></math></span> is the probability that data point <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e027"><math id="M27" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mrow><mi>k</mi></mrow></msub></mrow></math></span> belongs to component k. whereas the second step known as Maximization in this step, the parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e028"><math id="M28" display="inline" overflow="linebreak"><mrow><msub><mi>π</mi><mrow><mi>k</mi></mrow></msub><mo>,</mo><msub><mi>μ</mi><mrow><mi>k</mi></mrow></msub><mi>a</mi><mi>n</mi><mi>d</mi><msub><mi>Σ</mi><mrow><mi>k</mi></mrow></msub></mrow></math></span> are updated based on the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e029"><math id="M29" display="inline" overflow="linebreak"><mrow><mi>γ</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>𝒵</mi><mrow><mi>n</mi><mi>k</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">)</mo></mrow></mrow></math></span> computed in the expectation step. Thus, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e030"><math id="M30" display="inline" overflow="linebreak"><mrow><mi>γ</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>𝒵</mi><mrow><mi>n</mi><mi>k</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>=</mo><mi>p</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>C</mi><mrow><mi>k</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">|</mo></mrow><msub><mi>x</mi><mrow><mi>i</mi></mrow></msub><mo>,</mo><mi>μ</mi><mo>,</mo><mi>Σ</mi><mo>,</mo><mi>π</mi></mrow></math></span>where.</p>
<table class="disp-formula p" id="pone.0330421.e031"><tr>
<td class="formula"><math id="M31" display="block" overflow="linebreak"><mrow><mtable columnspacing="1" displaystyle="true" rowspacing="3pt"><mtr><mtd columnalign="right"><mi>p</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>C</mi><mrow><mi>k</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">|</mo></mrow><msub><mi>x</mi><mrow><mi>i</mi></mrow></msub><mo>,</mo><mi>μ</mi><mo>,</mo><mi>Σ</mi><mo>,</mo><mi>π</mi><mo stretchy="false">)</mo><mo>=</mo><mo>∑</mo><msubsup><mi>\nolimits</mi><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup><mrow><mi>log</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><mo>∑</mo><msubsup><mi>\nolimits</mi><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>K</mi></mrow></msubsup><mrow><msub><mi>π</mi><mrow><mi>k</mi></mrow></msub><mrow><mo mathvariant="script">.</mo><mtext mathvariant="script"> </mtext><mi mathvariant="script">N</mi><mo stretchy="false">(</mo></mrow><msub><mi>x</mi><mrow><mi>i</mi></mrow></msub><mo stretchy="false">|</mo><msub><mi>μ</mi><mrow><mi>k</mi></mrow></msub><mo>,</mo><msub><mi>Σ</mi><mrow><mi>k</mi></mrow></msub><mo stretchy="false">)</mo></mrow><mo fence="true" form="postfix" stretchy="true">)</mo></mrow></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">(9)</td>
</tr></table>
<p>Here <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e032"><math id="M32" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mrow><mi>i</mi></mrow></msub></mrow></math></span>is the data point <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e033"><math id="M33" display="inline" overflow="linebreak"><mrow><mo>,</mo><mi>μ</mi><mo>,</mo><mi>Σ</mi><mo>,</mo><mi>π</mi></mrow></math></span> are the parameters of the GMM and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e034"><math id="M34" display="inline" overflow="linebreak"><mrow><mrow><mi mathvariant="script">N</mi><mo stretchy="false">(</mo></mrow><msub><mi>x</mi><mrow><mi>i</mi></mrow></msub><mo stretchy="false">|</mo><msub><mi>μ</mi><mrow><mi>k</mi></mrow></msub><mo>,</mo><msub><mi>Σ</mi><mrow><mi>k</mi></mrow></msub><mo stretchy="false">)</mo></mrow></math></span> is the probability density function of component k evaluated at a data point <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e035"><math id="M35" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mrow><mi>i</mi></mrow></msub></mrow></math></span>.This process continues iteratively until convergence criteria are met, such as changes in log-likelihood or parameter stability.</p></section></section><section id="sec015"><h3 class="pmc_sec_title">2.4 Hybrid GMM using Group Counseling Optimizer (GMM_GCO)</h3>
<p>Incorporating Group Counseling Optimization (GCO) with a Gaussian Mixture Model (GMM) automates and improves the parameter tuning process addressing the limitation of traditional GMM. As discussed above the performance of traditional GMM is sensitive to key hyper-parameters, such as the number of components and the convergence threshold of the Expectation-Maximization (EM) algorithm. Furthermore, manual tuning methods often fail to ensure optimal performance, especially with complex datasets. Group Counseling Optimization (GCO) is a a population-based metaheuristic optimization algorithm inspired by the idea of collaborative learning and behavioral improvement in group counseling dynamics. It avoids local optima primarily through population diversity, iterative feedback mechanisms, and dynamic exploration-exploitation balancing.</p>
<p>Unlike traditional optimization algorithms that follow a single trajectory (e.g., gradient descent), GCO maintains a diverse population of candidate solutions. It maintains a diverse population of candidate solutions, allowing it to explore multiple regions of the solution space simultaneously and avoid being trapped in local optima. Individuals learn from better-performing peers through guided interactions, helping weaker candidates move toward more promising areas. Furthermore, during each iteration, individuals undergo small perturbations or adjustments that introduce randomness in their behavior (solution state). This allows GCO to explore new areas in the solution space rather than converging prematurely to a suboptimal point. The randomness ensures that even if the population converges temporarily, there’s always a chance of discovering better global optima.</p>
<p>It also balances exploitation of strong solutions with exploration of new possibilities, ensuring a robust search process. Over successive iterations, the population improves collectively, enhancing convergence toward globally optimal solutions. It efficiently explores the parameter space, preventing local optima and ensuring globally optimized solutions. This integration not only addresses the limitations of traditional GMM but also provides a scalable and purely unsupervised framework suitable for real-world applications where labeled data is limited or unavailable.The pseudocode of the proposed Hybrid GMM is depicted in <a href="#pone.0330421.g003" class="usa-link">Fig 3</a>.</p>
<figure class="fig xbox font-sm" id="pone.0330421.g003"><h4 class="obj_head">Fig 3. Pseudocode for Hybrid GMM and Pseudocode of proposed Hybrid GMM.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360590_pone.0330421.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a3e1/12360590/44ef82382fa6/pone.0330421.g003.jpg" loading="lazy" height="834" width="682" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330421.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Thus, to optimize a Gaussian Mixture Model (GMM) for fake news detection using Group Counseling Optimization (GCO), we start by initializing the population of agents. An agent represent an individual candidate solution in the search space. Each agent in the population is represented by a parameter vector <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e059"><math id="M59" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mrow><mi>i</mi></mrow></msub><mo>=</mo><mo stretchy="false">[</mo><msub><mi>x</mi><mrow><mi>i</mi><mn>1</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e060"><math id="M60" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mrow><mi>i</mi><mn>2</mn></mrow></msub></mrow></math></span>,<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e061"><math id="M61" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mrow><mi>i</mi><mn>3</mn></mrow></msub></mrow></math></span>,……, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e062"><math id="M62" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mrow><mi>i</mi><mi>d</mi></mrow></msub></mrow></math></span>], where each parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e063"><math id="M63" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></math></span>, is randomly generated within its respective bounds [<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e064"><math id="M64" display="inline" overflow="linebreak"><mrow><msub><mi>l</mi><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mi>u</mi><mrow><mi>i</mi></mrow></msub></mrow></math></span>}. Here, bounds variable particularly defines the ranges of the parameters that the Group Counseling Optimization (GCO) algorithm will optimize. Each element of the bounds list is a tuple that specifies the lower and upper limits for a particular parameter. Specifically, the bounds cover key GMM hyperparameters, including the number of components, Regularization parameter for covariance matrices, Maximum number of iterations for the Expectation-Maximization (EM) algorithm, and tolerance (Convergence threshold. Mathematically, each parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e065"><math id="M65" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></math></span>, for agent <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e066"><math id="M66" display="inline" overflow="linebreak"><mrow><mi>i</mi></mrow></math></span> is drawn from a uniform distribution within its bounds <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e067"><math id="M67" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mi>~</mi><mi>u</mi><mo stretchy="false">(</mo><msub><mi>l</mi><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mi>u</mi><mrow><mi>i</mi></mrow></msub><mo stretchy="false">)</mo></mrow></math></span>.</p>
<p>In the proposed GCO-based optimization, the fitness of each agent is evaluated using an internal clustering validation metric Silhouette Score rather than external ground truth labels. This choice ensures that the entire optimization process remains fully unsupervised and does not rely on labeled data during training process.The fitness of each agent is then evaluated using an objective function can be defined as:</p>
<table class="disp-formula p" id="pone.0330421.e068"><tr>
<td class="formula"><math id="M68" display="block" overflow="linebreak"><mrow><mi>f</mi><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo><mo>=</mo><mo>−</mo><mi>S</mi><mi>i</mi><mi>l</mi><mi>h</mi><mi>o</mi><mi>u</mi><mi>e</mi><mi>t</mi><mi>t</mi><mi>e</mi><mi>S</mi><mi>c</mi><mi>o</mi><mi>r</mi><mi>e</mi><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo></mrow></math></td>
<td class="label">(10)</td>
</tr></table>
<p>This transformation is used because optimization algorithms, including GCO, are typically designed to minimize an objective function.</p>
<p>The optimization process involves iterating for a predefined number of iterations. In each iteration, the mean vector of the population is calculated, and a random vector with values in the range [− <a href="#pone.0330421.ref001" class="usa-link" aria-describedby="pone.0330421.ref001">1</a>] is generated. Each agent’s parameters are updated based on the difference between the agent’s parameters and the mean vector, scaled by the random vector <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e069"><math id="M69" display="inline" overflow="linebreak"><mrow><mi>r</mi></mrow></math></span>. The updated parameters are adjusted to ensure they range within the predefined bounds. The fitness of the updated agent <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e070"><math id="M70" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>f</mi><mo stretchy="false">(</mo><mi>x</mi></mrow><mrow><mi>n</mi><mi>e</mi><mi>w</mi></mrow></msub><mo stretchy="false">)</mo></mrow></math></span> is then evaluated. If the new fitness is better (lower) than the current fitness, the agent’s parameters are updated to the new values. This process is repeated for all agents in the population and continues until convergence or maximum iterations are reached.</p>
<p>After completing the iterations, the agent with the best (lowest) fitness in the population is identified as the best solution<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e071"><math id="M71" display="inline" overflow="linebreak"><mrow><munder><mrow><msup><mi>x</mi><mrow><mo>*</mo></mrow></msup><mo>=</mo><mi>a</mi><mi>r</mi><mi>g</mi><mi>m</mi><mi>i</mi><mi>n</mi></mrow><mrow><mi>x</mi><mi>_</mi><mi>i</mi></mrow></munder><mrow><mi>f</mi><mo stretchy="false">(</mo><msub><mi>x</mi><mrow><mi>i</mi></mrow></msub><mo stretchy="false">)</mo></mrow></mrow></math></span>. This optimization process using GCO ensures that the GMM parameters are tuned to achieve the highest accuracy possible to effectively distinguish between real and fake news.</p></section><section id="sec016"><h3 class="pmc_sec_title">2.5 Classical clustering techniques</h3>
<section id="sec017"><h4 class="pmc_sec_title">2.5.1 K means clustering.</h4>
<p>K-means clustering is a fundamental clustering algorithm within the realms of unsupervised techniques. It discover concealed patterns within the data while effectively grouping data points that share similarities together [<a href="#pone.0330421.ref109" class="usa-link" aria-describedby="pone.0330421.ref109">109</a>]. Its primary objective is to iteratively partition the data points into k non overlapping clusters. During this partitioning process, each cluster finds its representation through a centroid point based on a distance metric [<a href="#pone.0330421.ref110" class="usa-link" aria-describedby="pone.0330421.ref110">110</a>]. This process aims to minimize intra-cluster variance and maximize inter-cluster variance, leading to well-defined clusters [<a href="#pone.0330421.ref111" class="usa-link" aria-describedby="pone.0330421.ref111">111</a>].</p>
<p>Let D is a D-dimensional dataset of n observations represented as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e072"><math id="M72" display="inline" overflow="linebreak"><mrow><mi>X</mi><mo>=</mo><mrow><mo fence="true" form="prefix" stretchy="true">{</mo><msub><mi>x</mi><mrow><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>x</mi><mrow><mn>2</mn></mrow></msub><mi>…</mi><mi>…</mi><msub><mi>x</mi><mrow><mi>n</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">}</mo></mrow></mrow></math></span> where each data point belongs to a cluster C = {C<sub>1</sub>, C<sub>2</sub>,….., C<sub>k</sub>} based on some proximity measure. In this research to find out the similarity between data points we incorporated Euclidean distance as a proximity measure or partitioning method Dist(i, j) that assures how similar or distant observation i from j are; consequently build k distinct partitioning from a D dataset of n observations. Where each k corresponds to a distinct cluster consist of at least one object and K ≤ n. Euclidean proximity measure can be determined as follows.</p>
<table class="disp-formula p" id="pone.0330421.e073"><tr>
<td class="formula"><math id="M73" display="block" overflow="linebreak"><mrow><mtable columnspacing="1" displaystyle="true" rowspacing="3pt"><mtr><mtd columnalign="right"><mi>a</mi><mi>r</mi><mi>g</mi><mi>m</mi><mi>i</mi><mi>n</mi><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mi>i</mi></mrow><mrow><mi>k</mi></mrow></msubsup><msub><mo>∑</mo><mrow><mi>x</mi><mo>∈</mo><mi>C</mi><mi>i</mi></mrow></msub><mrow><mo stretchy="false">|</mo><mo stretchy="false">|</mo></mrow><msup><mrow><mi>x</mi><mo>−</mo><msub><mi>μ</mi><mrow><mi>i</mi></mrow></msub><mo stretchy="false">|</mo><mo stretchy="false">|</mo></mrow><mrow><mn>2</mn></mrow></msup><mo>=</mo><mrow><mi mathvariant="normal">a</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">g</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">n</mi><msubsup><mo>∑</mo><mrow><mi mathvariant="normal">i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">k</mi></mrow></msubsup><mrow><mo stretchy="false">|</mo><msub><mi mathvariant="normal">C</mi><mrow><mi mathvariant="normal">i</mi></mrow></msub><mo stretchy="false">|</mo></mrow></mrow><mrow><mi>V</mi><mi>a</mi><mi>r</mi></mrow><msub><mi>C</mi><mrow><mi>i</mi></mrow></msub></mtd></mtr></mtable></mrow></math></td>
<td class="label">(11)</td>
</tr></table>
<p>Where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e074"><math id="M74" display="inline" overflow="linebreak"><mrow><msub><mi>μ</mi><mrow><mi>i</mi></mrow></msub></mrow></math></span> represents the centroid of data points in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e075"><math id="M75" display="inline" overflow="linebreak"><mrow><msub><mi>C</mi><mrow><mi>i</mi></mrow></msub></mrow></math></span> can be calculated as:</p>
<table class="disp-formula p" id="pone.0330421.e076"><tr>
<td class="formula"><math id="M76" display="block" overflow="linebreak"><mrow><mtable columnspacing="1" displaystyle="true" rowspacing="3pt"><mtr><mtd columnalign="right"><msub><mi>μ</mi><mrow><mi>i</mi></mrow></msub><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mo stretchy="false">|</mo><msub><mi>C</mi><mrow><mi>i</mi></mrow></msub><mo stretchy="false">|</mo></mrow></mfrac><mo>∑</mo><msub><mi>\nolimits</mi><mrow><mi>x</mi><mo>∈</mo><mi>C</mi><mi>i</mi></mrow></msub><mi>x</mi></mtd></mtr></mtable></mrow></math></td>
<td class="label">(12)</td>
</tr></table>
<p>In this research, we also incorporated K-means clustering to group TF-IDF vectors of news data with similar characteristics, aiding in the identification of patterns and clusters within the dataset. However, it requires specifying the number of clusters (K) in advance, which is a challenging task. In this research we incorporated elbow and Bayesian Information Criterion (BIC) techniques to address this challenge.</p></section><section id="sec018"><h4 class="pmc_sec_title">2.5.2 K-medoids clustering.</h4>
<p>K-medoids clustering [<a href="#pone.0330421.ref112" class="usa-link" aria-describedby="pone.0330421.ref112">112</a>] is similar to K-Means, but instead of using the mean of the data points to determine the cluster centre point, it uses actual data points as the cluster centers (medoids). For each data point <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e077"><math id="M77" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mrow><mi>i</mi></mrow></msub></mrow></math></span> find the medoid <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e078"><math id="M78" display="inline" overflow="linebreak"><mrow><msub><mi>m</mi><mrow><mi>j</mi></mrow></msub></mrow></math></span> that best represents the center of each cluster by minimizing the sum of distances within each cluster, enhancing the robustness and interpretability of the clustering results.</p>
<p>Let <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e079"><math id="M79" display="inline" overflow="linebreak"><mrow><mi>M</mi><mo>=</mo><mrow><mo fence="true" form="prefix" stretchy="true">{</mo><msub><mi>m</mi><mrow><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>m</mi><mrow><mn>2</mn></mrow></msub><mi>…</mi><mi>…</mi><msub><mi>m</mi><mrow><mi>k</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">}</mo></mrow></mrow></math></span>be the set of randomly selected initial medoids from the dataset<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e080"><math id="M80" display="inline" overflow="linebreak"><mrow><mi>X</mi></mrow></math></span></p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e081"><math id="M81" display="inline" overflow="linebreak"><mrow><mi>w</mi><mi>h</mi><mi>e</mi><mi>r</mi><mi>e</mi><mi>X</mi><mo>=</mo><mrow><mo fence="true" form="prefix" stretchy="true">{</mo><msub><mi>x</mi><mrow><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>x</mi><mrow><mn>2</mn></mrow></msub><mi>…</mi><mi>…</mi><msub><mi>x</mi><mrow><mi>n</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">}</mo></mrow></mrow></math></span>. For each data point <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e082"><math id="M82" display="inline" overflow="linebreak"><mrow><msub><mi>x</mi><mrow><mi>i</mi></mrow></msub></mrow></math></span>, find the medoid <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e083"><math id="M83" display="inline" overflow="linebreak"><mrow><msub><mi>m</mi><mrow><mi>j</mi></mrow></msub></mrow></math></span>, that minimizes the distance:</p>
<table class="disp-formula p" id="pone.0330421.e084"><tr>
<td class="formula"><math id="M84" display="block" overflow="linebreak"><mrow><mi>A</mi><mi>s</mi><mi>s</mi><mi>i</mi><mi>g</mi><mi>n</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>x</mi><mrow><mi>i</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>=</mo><mrow><mi>a</mi><mi>r</mi><mi>g</mi><msub><mo>min</mo><mrow><mi>x</mi><mo>∈</mo><msub><mi>c</mi><mrow><mi>j</mi></mrow></msub></mrow></msub></mrow><mrow><mi mathvariant="normal">d</mi><mrow><mo stretchy="false">(</mo><msub><mi mathvariant="normal">x</mi><mrow><mi mathvariant="normal">i</mi></mrow></msub><mo>,</mo><msub><mi mathvariant="normal">m</mi><mrow><mi mathvariant="normal">j</mi></mrow></msub><mo stretchy="false">)</mo></mrow></mrow></mrow></math></td>
<td class="label">(13)</td>
</tr></table>
<p>Mathematically, for each cluster <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e085"><math id="M85" display="inline" overflow="linebreak"><mrow><msub><mi>c</mi><mrow><mi>j</mi></mrow></msub></mrow></math></span> with data points <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e086"><math id="M86" display="inline" overflow="linebreak"><mrow><mrow><mo fence="true" form="prefix" stretchy="true">{</mo><msub><mi>x</mi><mrow><mi>j</mi><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>x</mi><mrow><mi>j</mi><mn>2</mn></mrow></msub><mi>…</mi><mi>…</mi><msub><mi>x</mi><mrow><mi>j</mi><mi>n</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">}</mo></mrow></mrow></math></span>, the new medoid <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e087"><math id="M87" display="inline" overflow="linebreak"><mrow><msub><mi>m</mi><mrow><mi>j</mi></mrow></msub></mrow></math></span> is selected as:</p>
<table class="disp-formula p" id="pone.0330421.e088"><tr>
<td class="formula"><math id="M88" display="block" overflow="linebreak"><mrow><mi>A</mi><mi>s</mi><mi>s</mi><mi>i</mi><mi>g</mi><mi>n</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>x</mi><mrow><mi>i</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>=</mo><mi>a</mi><mi>r</mi><mi>g</mi><msub><mo>min</mo><mrow><mi>x</mi><mo>∈</mo><msub><mi>c</mi><mrow><mi>j</mi></mrow></msub></mrow></msub><mrow><msub><mo>∑</mo><mrow><msub><mi>x</mi><mrow><mi>i</mi></mrow></msub><mo>∈</mo><msub><mi>c</mi><mrow><mi>j</mi></mrow></msub></mrow></msub><mrow><mrow><mi mathvariant="normal">d</mi></mrow><mrow><mo stretchy="false">(</mo><mi>x</mi><mo>,</mo><msub><mi>x</mi><mrow><mi>i</mi></mrow></msub><mo stretchy="false">)</mo></mrow></mrow></mrow></mrow></math></td>
<td class="label">(14)</td>
</tr></table></section></section><section id="sec019"><h3 class="pmc_sec_title">2.6 Determine the optimal number of components for clustering algorithms</h3>
<section id="sec020"><h4 class="pmc_sec_title">2.6.1 Elbow method for K-means and K-medoid Clustering.</h4>
<p>The Elbow method is a commonly used technique for identifying the optimal number of clusters in a dataset when applying clustering algorithms like K-means [<a href="#pone.0330421.ref113" class="usa-link" aria-describedby="pone.0330421.ref113">113</a>] and K-medoids [<a href="#pone.0330421.ref114" class="usa-link" aria-describedby="pone.0330421.ref114">114</a>]. This method relies on the principle that the within-cluster sum of squares (WCSS) typically decreases as the number of clusters increases. Mathematically, WCSS can be defined as:</p>
<table class="disp-formula p" id="pone.0330421.e089"><tr>
<td class="formula"><math id="M89" display="block" overflow="linebreak"><mrow><mtable columnspacing="1" displaystyle="true" rowspacing="3pt"><mtr><mtd columnalign="right"><mi>W</mi><mi>C</mi><mi>S</mi><mi>S</mi><mo stretchy="false">(</mo><mi>k</mi><mo stretchy="false">)</mo><mo>=</mo><mo>∑</mo><msubsup><mi>\nolimits</mi><mrow><mi>i</mi><mo>=</mo><mi>i</mi></mrow><mrow><mi>k</mi></mrow></msubsup><mo>∑</mo><msub><mi>\nolimits</mi><mrow><mi>x</mi><mo>∈</mo><mi>C</mi><mi>i</mi></mrow></msub><mrow><mo stretchy="false">|</mo><mo stretchy="false">|</mo></mrow><msup><mrow><mi>x</mi><mo>−</mo><msub><mi>μ</mi><mrow><mi>i</mi></mrow></msub><mo stretchy="false">|</mo><mo stretchy="false">|</mo></mrow><mrow><mn>2</mn></mrow></msup></mtd></mtr></mtable></mrow></math></td>
<td class="label">(15)</td>
</tr></table>
<p>Where k is the number of clusters, x is the data point in the ith cluster with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e090"><math id="M90" display="inline" overflow="linebreak"><mrow><msub><mi>μ</mi><mrow><mi>i</mi></mrow></msub></mrow></math></span> centre point.</p>
<p>The Elbow method involves running the algorithm for varying numbers of clusters and plotting the corresponding WCSS values. The resulting plot often resembles an “elbow” shape, where the WCSS decreases rapidly initially and then starts to level off. The optimal number of clusters is identified at the “elbow point,” which is where the rate of decrease in WCSS (∆WCSS (k)) starts diminishing considerably [<a href="#pone.0330421.ref056" class="usa-link" aria-describedby="pone.0330421.ref056">56</a>]. This point indicates a balance between minimizing WCSS (which reflects cluster compactness) and avoiding over-fitting by using too many clusters [<a href="#pone.0330421.ref115" class="usa-link" aria-describedby="pone.0330421.ref115">115</a>]. This involves calculating the difference in WCSS for consecutive values of k as depicted below:</p>
<table class="disp-formula p" id="pone.0330421.e091"><tr>
<td class="formula"><math id="M91" display="block" overflow="linebreak"><mrow><mrow><mi>Δ</mi></mrow><mi>W</mi><mi>C</mi><mi>S</mi><mi>S</mi><mo stretchy="false">(</mo><mi>k</mi><mo stretchy="false">)</mo><mo>=</mo><mi>W</mi><mi>C</mi><mi>S</mi><mi>S</mi><mo stretchy="false">(</mo><mi>k</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo>−</mo><mi>W</mi><mi>C</mi><mi>S</mi><mi>S</mi><mo stretchy="false">(</mo><mi>k</mi><mo stretchy="false">)</mo></mrow></math></td>
<td class="label">(16)</td>
</tr></table></section><section id="sec021"><h4 class="pmc_sec_title">2.6.2 Bayesian Information Criterion (BIC) for GMM.</h4>
<p>In this study, we employed the Bayesian Information Criterion (BIC) [<a href="#pone.0330421.ref116" class="usa-link" aria-describedby="pone.0330421.ref116">116</a>] to determine the optimal number of components for a Gaussian Mixture Model (GMM) applied to our dataset. The BIC is a statistical criterion used for model selection, balancing model complexity and goodness of fit [<a href="#pone.0330421.ref117" class="usa-link" aria-describedby="pone.0330421.ref117">117</a>].</p>
<p>We considered a range of candidate numbers of components and explored different covariance types. Then for each combination of parameters, we trained GMM <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e092"><math id="M92" display="inline" overflow="linebreak"><mrow><msub><mrow><mo stretchy="false">(</mo><mi mathvariant="script">M</mi></mrow><mrow><mi>k</mi></mrow></msub><mo stretchy="false">)</mo></mrow></math></span>. We computed the BIC for each trained model <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e093"><math id="M93" display="inline" overflow="linebreak"><mrow><msub><mi>ℳ</mi><mrow><mi>k</mi></mrow></msub></mrow></math></span> as depicted below:</p>
<table class="disp-formula p" id="pone.0330421.e094"><tr>
<td class="formula"><math id="M94" display="block" overflow="linebreak"><mrow><mi>B</mi><mi>I</mi><mi>C</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>ℳ</mi><mrow><mi>k</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>=</mo><mo>−</mo><mn>2</mn><mi>ln</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><mover><mrow><mi>L</mi></mrow><mo>´</mo></mover><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>ℳ</mi><mrow><mi>k</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>+</mo><msub><mi>p</mi><mrow><mi>K</mi></mrow></msub><mi>ln</mi><mo stretchy="false">(</mo><mi>N</mi><mo stretchy="false">)</mo></mrow></math></td>
<td class="label">(17)</td>
</tr></table>
<p>Where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e095"><math id="M95" display="inline" overflow="linebreak"><mrow><mover><mrow><mi>L</mi></mrow><mo>´</mo></mover><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>ℳ</mi><mrow><mi>k</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">)</mo></mrow></mrow></math></span> is the likelihood of the data under the model <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e096"><math id="M96" display="inline" overflow="linebreak"><mrow><msub><mi>ℳ</mi><mrow><mi>k</mi></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e097"><math id="M97" display="inline" overflow="linebreak"><mrow><msub><mi>p</mi><mrow><mi>K</mi></mrow></msub></mrow></math></span> is the number of parameters in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e098"><math id="M98" display="inline" overflow="linebreak"><mrow><msub><mi>ℳ</mi><mrow><mi>k</mi></mrow></msub></mrow></math></span>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e099"><math id="M99" display="inline" overflow="linebreak"><mrow><msup><mi>ℳ</mi><mrow><mo>*</mo></mrow></msup></mrow></math></span> is the model with lowest BIC as depicted below:</p>
<table class="disp-formula p" id="pone.0330421.e100"><tr>
<td class="formula"><math id="M100" display="block" overflow="linebreak"><mrow><msup><mi>ℳ</mi><mrow><mo>*</mo></mrow></msup><mo>=</mo><msub><mrow><mi>a</mi><mi>r</mi><mi>g</mi><mi>m</mi><mi>i</mi><mi>n</mi></mrow><mrow><msub><mi>ℳ</mi><mrow><mi>k</mi></mrow></msub></mrow></msub><mrow><mo stretchy="false">(</mo><msub><mi>ℳ</mi><mrow><mi>k</mi></mrow></msub><mo stretchy="false">)</mo></mrow></mrow></math></td>
<td class="label">(18)</td>
</tr></table>
<p>The model with the lowest BIC was considered the most suitable in terms of both model complexity and ability to capture the underlying structure of the data [<a href="#pone.0330421.ref088" class="usa-link" aria-describedby="pone.0330421.ref088">88</a>].</p>
<p>The Elbow method and BIC are incorporated as baseline techniques for cluster estimation for classsical K-means, K-medoids, and GMM techniques to facilitate comparative evaluation.While commonly used, the Elbow Method, especially when relying on first-order derivatives (Δ) often lacks robustness and precision. This highlights the need for a more adaptive approach. The proposed GCO-based optimization addresses this by effectively exploring the parameter space without relying on heuristic or visual methods.</p></section></section><section id="sec022"><h3 class="pmc_sec_title">2.7 Implementation configrarion</h3>
<p>In this study, the proposed method is implemented in Python. While the K-means [<a href="#pone.0330421.ref112" class="usa-link" aria-describedby="pone.0330421.ref112">112</a>], K-medoids [<a href="#pone.0330421.ref113" class="usa-link" aria-describedby="pone.0330421.ref113">113</a>] and GMM [<a href="#pone.0330421.ref059" class="usa-link" aria-describedby="pone.0330421.ref059">59</a>] based clustering algorithms were implemented using python scikit-learn package, a widely adopted library for machine learning in Python. The computational evaluation were conducted on a Laptop equipped with an Intel Core i7 10th Generation processor, 16 GB of RAM, Windows 10 operating system and a solid-state drive (SSD) connected via the Serial Advanced Technology Attachment (SATA) interface.</p>
<section id="sec023"><h4 class="pmc_sec_title">2.7.1 Evaluation measure.</h4>
<p>To evaluate the performance of the clustering results, three evaluation measures are commonly used in existing literature including silhouette score, adjusted rand index and purity score. These evaluation metrics help in assessing the effectiveness of the unsupervised techniques and provide quantitative measures of its performance. Hence, we also incorporated these measures to evaluate the performance of our method.</p>
<p>
<strong>2.7.1.1 Silhouette Score</strong>
</p>
<p>The silhouette score offers an intuitive way to evaluate the quality of a clustering result by determining the mean silhouette coefficient across all samples [<a href="#pone.0330421.ref118" class="usa-link" aria-describedby="pone.0330421.ref118">118</a>,<a href="#pone.0330421.ref119" class="usa-link" aria-describedby="pone.0330421.ref119">119</a>]. This quantifies how well each data point fits within its assigned cluster compared to other clusters. [<a href="#pone.0330421.ref120" class="usa-link" aria-describedby="pone.0330421.ref120">120</a>]. Its value ranges from −1–1, where a higher silhouette score suggests better-defined, well-separated clustering results [<a href="#pone.0330421.ref121" class="usa-link" aria-describedby="pone.0330421.ref121">121</a>]. The silhouette score for a data point <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e101"><math id="M101" display="inline" overflow="linebreak"><mrow><mi>i</mi></mrow></math></span> is defined as: a metric that quantifies how well each data point fits within its assigned cluster compared to other clusters.</p>
<table class="disp-formula p" id="pone.0330421.e102"><tr>
<td class="formula"><math id="M102" display="block" overflow="linebreak"><mrow><mi>S</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><mo>=</mo><mfrac><mrow><mi>b</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><mo>−</mo><mi>a</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow><mrow><mo>max</mo><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><mi>a</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><mo>,</mo><mi>b</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><mo fence="true" form="postfix" stretchy="true">)</mo></mrow></mrow></mfrac></mrow></math></td>
<td class="label">(19)</td>
</tr></table>
<p>Where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e103"><math id="M103" display="inline" overflow="linebreak"><mrow><mi>a</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></math></span> represents the average distance between the point <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e104"><math id="M104" display="inline" overflow="linebreak"><mrow><mi>i</mi></mrow></math></span> and all other points within the same cluster while <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e105"><math id="M105" display="inline" overflow="linebreak"><mrow><mi>b</mi><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo></mrow></math></span> is the minimum average distance between the point <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e106"><math id="M106" display="inline" overflow="linebreak"><mrow><mi>i</mi></mrow></math></span> and all points in the next nearest cluster [<a href="#pone.0330421.ref122" class="usa-link" aria-describedby="pone.0330421.ref122">122</a>]. However, the overall silhouette score for the entire dataset is calculated by the average of the silhouette scores for all the individual points.</p>
<p>
<strong>2.7.1.2 Adjusted Rand Index (ARI)</strong>
</p>
<p>The Adjusted Rand Index (ARI) is a widely used measure for evaluating the agreement between two data partitions in cluster analysis. It is often used to assess the quality of clustering algorithms [<a href="#pone.0330421.ref123" class="usa-link" aria-describedby="pone.0330421.ref123">123</a>–<a href="#pone.0330421.ref126" class="usa-link" aria-describedby="pone.0330421.ref126">126</a>], particularly between the obtained clustering results and ground-truth partition, ranging from 0 (complete disagreement) to 1 (complete agreement). Unlike the classical Rand Index, ARI adjusts the chance grouping of elements, making it a more robust and reliable measure of evaluating clustering results [<a href="#pone.0330421.ref127" class="usa-link" aria-describedby="pone.0330421.ref127">127</a>]. Mathematically it can be defined as:</p>
<table class="disp-formula p" id="pone.0330421.e107"><tr>
<td class="formula"><math id="M107" display="block" overflow="linebreak"><mrow><mi>A</mi><mi>R</mi><mi>I</mi><mo>=</mo><mfrac><mrow><mi>R</mi><mi>I</mi><mo>−</mo><mi>E</mi><mo stretchy="false">[</mo><mi>R</mi><mi>I</mi><mo stretchy="false">]</mo></mrow><mrow><mo>max</mo><mo stretchy="false">(</mo><mi>R</mi><mi>I</mi><mo stretchy="false">)</mo><mo>−</mo><mi>E</mi><mo stretchy="false">[</mo><mi>R</mi><mi>I</mi><mo stretchy="false">]</mo></mrow></mfrac></mrow></math></td>
<td class="label">(20)</td>
</tr></table>
<p>Where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e108"><math id="M108" display="inline" overflow="linebreak"><mrow><mi>R</mi><mi>I</mi></mrow></math></span> denotes the Rand Index that measures the agreement between two clustering results by counting pairs of data points that are either grouped together or assigned to different clusters in both the predicted and true clustering. Where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e109"><math id="M109" display="inline" overflow="linebreak"><mrow><mi>E</mi><mo stretchy="false">[</mo><mi>R</mi><mi>I</mi><mo stretchy="false">]</mo></mrow></math></span>represents the expected Rand Index, accounting for random chance.</p>
<p>The ARI has been generalized to compare a set of clustering solutions with a consensus matrix (ARImp) and to evaluate the consistency between two similarity matrices (ARImm), preserving desirable properties of the original ARI and introducing new ones, such as the ability to detect negative correlation and to compute in a distributed environment.</p>
<p>
<strong>2.7.1.3 Purity Score</strong>
</p>
<p>It is also a widely used external validation measure in clustering analysis, it measures the degree to which each cluster contains data points from a single ground-truth class. It represents how “pure” or homogeneous the clusters are in terms of the true labels, provide a simple technique to evaluate clustering quality. The Purity Score is defined as:</p>
<table class="disp-formula p" id="pone.0330421.e110"><tr>
<td class="formula"><math id="M110" display="block" overflow="linebreak"><mrow><mtable columnspacing="1" displaystyle="true" rowspacing="3pt"><mtr><mtd columnalign="right"><mi>P</mi><mi>u</mi><mi>r</mi><mi>i</mi><mi>t</mi><mi>y</mi><mi>S</mi><mi>c</mi><mi>o</mi><mi>r</mi><mi>e</mi><mo>=</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mi>n</mi></mrow></mfrac><mo>∑</mo><msubsup><mi>\nolimits</mi><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>k</mi></mrow></msubsup><mrow><mo>max</mo><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><mo stretchy="false">|</mo><msub><mi>C</mi><mrow><mi>i</mi></mrow></msub><mo>∩</mo><msub><mi>L</mi><mrow><mi>j</mi></mrow></msub><mo stretchy="false">|</mo><mo fence="true" form="postfix" stretchy="true">)</mo></mrow></mrow></mtd></mtr></mtable></mrow></math></td>
<td class="label">(21)</td>
</tr></table>
<p>Where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e111"><math id="M111" display="inline" overflow="linebreak"><mrow><msub><mi>C</mi><mrow><mi>i</mi></mrow></msub></mrow></math></span> represents the set of points in ith cluster, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e112"><math id="M112" display="inline" overflow="linebreak"><mrow><msub><mi>L</mi><mrow><mi>j</mi></mrow></msub></mrow></math></span> denotes the set of points in the ground-truth class <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e113"><math id="M113" display="inline" overflow="linebreak"><mrow><mi>j</mi></mrow></math></span>.Whereas the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e114"><math id="M114" display="inline" overflow="linebreak"><mrow><mo>max</mo><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><mo stretchy="false">|</mo><msub><mi>C</mi><mrow><mi>i</mi></mrow></msub><mo>∩</mo><msub><mi>L</mi><mrow><mi>j</mi></mrow></msub><mo stretchy="false">|</mo><mo fence="true" form="postfix" stretchy="true">)</mo></mrow></mrow></math></span> represents the largest number of points in cluster <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e115"><math id="M115" display="inline" overflow="linebreak"><mrow><mi>i</mi></mrow></math></span> belonging to a single ground-truth class <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e116"><math id="M116" display="inline" overflow="linebreak"><mrow><msub><mi>L</mi><mrow><mi>j</mi></mrow></msub></mrow></math></span>.</p>
<p>
<strong>2.7.1.4 F-measure</strong>
</p>
<p>We also included the F-measure in our evaluation to provide a more reliable assessment of the proposed techniquue’s performance. This can be mathematically defined as follows:</p>
<table class="disp-formula p" id="pone.0330421.e117"><tr>
<td class="formula"><math id="M117" display="block" overflow="linebreak"><mrow><mi>F</mi><mo>−</mo><mi>M</mi><mi>e</mi><mi>a</mi><mi>s</mi><mi>u</mi><mi>r</mi><mi>e</mi><mo>=</mo><mfrac><mrow><mn>2</mn></mrow><mrow><mfrac><mrow><mn>1</mn></mrow><mrow><mi>P</mi><mi>r</mi><mi>e</mi><mi>c</mi><mi>i</mi><mi>s</mi><mi>i</mi><mi>o</mi><mi>n</mi></mrow></mfrac><mo>+</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mi>R</mi><mi>e</mi><mi>c</mi><mi>a</mi><mi>l</mi><mi>l</mi></mrow></mfrac></mrow></mfrac></mrow></math></td>
<td class="label">(22)</td>
</tr></table>
<p>The computational time complexity of hybrid GM to identifying the clustering results is <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e118"><math id="M118" display="inline" overflow="linebreak"><mrow><mi>O</mi><mo stretchy="false">(</mo><msub><mrow><mi>t</mi><mo>*</mo><mi>K</mi></mrow><mrow><mi>i</mi></mrow></msub><msub><mi>n</mi><mrow><mi>i</mi></mrow></msub><msup><mi>D</mi><mrow><mn>2</mn></mrow></msup><mo stretchy="false">)</mo></mrow></math></span>. Where t is the number of iterations, n is the number of data points, K is the number of components, and D is the dimensionality of the data. It can be observed that the dimensionality dominates the computation particularly for high-dimensional data sets. In E-step, computing the posterior probability <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e119"><math id="M119" display="inline" overflow="linebreak"><mrow><mi>γ</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>𝒵</mi><mrow><mi>n</mi><mi>k</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">)</mo></mrow></mrow></math></span> requires <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e120"><math id="M120" display="inline" overflow="linebreak"><mrow><mi>O</mi><mo stretchy="false">(</mo><msub><mi>K</mi><mrow><mi>i</mi></mrow></msub><msub><mi>n</mi><mrow><mi>i</mi></mrow></msub><mi>D</mi><mo stretchy="false">)</mo></mrow></math></span>operations. In M-step, computing the parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e121"><math id="M121" display="inline" overflow="linebreak"><mrow><msub><mi>π</mi><mrow><mi>k</mi></mrow></msub><mo>,</mo><msub><mi>μ</mi><mrow><mi>k</mi></mrow></msub><mi>a</mi><mi>n</mi><mi>d</mi><msub><mi>Σ</mi><mrow><mi>k</mi></mrow></msub></mrow></math></span> involve <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e122"><math id="M122" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>O</mi><mo stretchy="false">(</mo><mi>K</mi></mrow><mrow><mi>i</mi></mrow></msub><msub><mi>n</mi><mrow><mi>i</mi></mrow></msub><mo stretchy="false">)</mo></mrow></math></span> operations. The estimation of D-dimensional mean vector of component k and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0330421.e123"><math id="M123" display="inline" overflow="linebreak"><mrow><msub><mi>Σ</mi><mrow><mi>k</mi></mrow></msub></mrow></math></span> variance matrix of component k required additional operations. Therefore, the overall time complexity is a combination of the complexities of both the Expectation-Maximization (EM) algorithm for Gaussian Mixture Models (GMM) and the Group Counseling Optimization (GCO) algorithm.</p></section></section></section><section id="sec024"><h2 class="pmc_sec_title">3. Results and discussion</h2>
<p>In this research, our primary objective is to identify fake news on social media platforms based on unsupervised method, specifically utilizing a Gaussian Mixture Model (GMM). While it is well-known that the Expectation-Maximization (EM) algorithm, commonly used for fitting GMMs, is highly sensitive to initial values and requires the number of components to be specified a priori [<a href="#pone.0330421.ref128" class="usa-link" aria-describedby="pone.0330421.ref128">128</a>]. To address these challenges, we introduce a novel hybrid Gaussian Mixture model for detecting fake news in a fully unsupervised manner. Unlike previous works that separately address the issues of initialization [<a href="#pone.0330421.ref129" class="usa-link" aria-describedby="pone.0330421.ref129">129</a>] and the estimation of the number of components [<a href="#pone.0330421.ref130" class="usa-link" aria-describedby="pone.0330421.ref130">130</a>,<a href="#pone.0330421.ref131" class="usa-link" aria-describedby="pone.0330421.ref131">131</a>] we tackle these problems simultaneously.</p>
<p>The proposed approach also introduced a novel objective function for the GMM based on mixture distributions, and then leveraged a meta-heuristic optimization algorithm, Group Counseling Optimization (GCO) to find the best set of parameters to fit the Gaussian Mixture Model on news data. It iteratively adjusts the GMM parameters, ensuring that the model is both robust to initial values and capable of automatically determining the optimal number of clusters. This method significantly enhances the performance and reliability of the EM algorithm, thus, addressing the limitations of traditional GMM particularly sensitivity toward initial parameters values and number of cluster estimations, leading to more stable and optimal clustering. Finally, we also demonstrated the effectiveness of our innovative approach by applying it on a real-world news dataset.</p>
<p>In the initial step of the proposed Hybrid GMM, we initialized all the parameters of bounds vector as follows: the number of components (n_components) was adjusted within a range of 2–10, allowing the algorithm to identify the optimal number of clusters. This range was selected because a minimum of two components is necessary for meaningful clustering, and ten components represent a practical upper limit for this dataset. As there is a trade-off in the selection of an appropriate number of clusters. A higher number of clusters can offer a more detailed and fine-grained representation of the data, potentially enhancing clustering accuracy. However, a large number of clusters also leads to greater computational complexity and adds to the workload of post-processing [<a href="#pone.0330421.ref132" class="usa-link" aria-describedby="pone.0330421.ref132">132</a>]. The regularization on covariance (reg_covar) was adjusted between 10<sup>−6</sup> and 10<sup>−2</sup> ensuring that the covariance matrices remained stable and invertible while preventing numerical issues. This range provided a balance between too little and too much regularization, both of which could negatively impact the GMM’s performance.</p>
<p>Furthermore, the maximum number of iterations (max_iter) was set between 100 and 500, providing the Expectation-Maximization (EM) algorithm with ample opportunity to converge while minimizing unnecessary computational overhead. The tolerance parameter (tol), ranging from 10<sup>−4</sup> and 10<sup>−2</sup>, defined the convergence threshold for the EM algorithm, balancing the need for precision with the desire for computational efficiency. These carefully selected bounds allowed the GCO algorithm to effectively search for the optimal GMM parameters, leading to enhanced clustering performance and demonstrating the robustness of the approach in finding an optimal solution.</p>
<p>Following the optimization process, the GCO algorithm identified the optimal parameters for hybrid GMM, resulting in a model comprising of 3 components, with a regularization on covariance of 0.00617, a maximum of 224 iterations, and a tolerance of 0.00632. These optimized parameters were subsequently utilized to fit the hybrid GMM, superseding the initial values set within the bounds vector. Subsequently, the proposed hybrid GMM, configured with the identified 3 mixture components and the aforementioned optimized parameters is employed to cluster the two-dimensional unlabeled data, where each observation corresponds to a news statement. Upon fitting the hybrid GMM, the observations belonging to the same mixture component were considered as in the same cluster, facilitating the identification of underlying patterns in the news data. <a href="#pone.0330421.g004" class="usa-link">Fig 4</a> illustrated the clustering results produced by hybrid GMM over scaled data obtained from the preprocessing steps. It demonstrated that the proposed hybrid GMM partitioned the news data into specified number of clusters based on the mixture components as identified by the GCO optimizer and discussed above. A Superficial comparative analysis of <a href="#pone.0330421.g002" class="usa-link">Figs 2</a> and <a href="#pone.0330421.g004" class="usa-link">4</a> might demonstrated that hybrid GMM primarily provides cluster delineation to the PCA representation, however, the integration of GMM with PCA offers substantial advantages that extend beyond mere visualization. It enables the modeling of data as a mixture of multiple Gaussian distributions, facilitating the identification of underlying data structures and probabilistic cluster assignments.</p>
<figure class="fig xbox font-sm" id="pone.0330421.g004"><h3 class="obj_head">Fig 4. Clustering results obtained by hybrid GMM over scaled data, showing distinct groupings of real and fake news articles based on textual similarity.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360590_pone.0330421.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a3e1/12360590/e241e7f37569/pone.0330421.g004.jpg" loading="lazy" height="489" width="682" alt="Fig 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330421.g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Nevertheless, the clustering results yielded by the hybrid Gaussian Mixture Model (GMM) are unlabeled meaning that while the data points are grouped into distinct clusters based on their features, the specific nature or identity of these clusters is not immediately discernible [<a href="#pone.0330421.ref064" class="usa-link" aria-describedby="pone.0330421.ref064">64</a>], as depicted in <a href="#pone.0330421.g004" class="usa-link">Fig 4</a>. Thus, it does not provide any explicit indication of which cluster corresponds to which type of news. Furthermore, without labels, we are unable to ascertain the characteristics or thematic content of the clusters, nor can we directly identify the nature of the news articles within each cluster. Labeling these clusters would be a crucial step toward understanding the underlying structure of the data [<a href="#pone.0330421.ref133" class="usa-link" aria-describedby="pone.0330421.ref133">133</a>], as it would allow us to interpret the clusters in a meaningful way, such as identifying whether a cluster predominantly contains real or fake news. Thus, the process of labeling or annotating these clusters is essential for translating the unsupervised clustering results into actionable insights and for making informed decisions based on the classification of the news articles.</p>
<p>To address this challenge and to enhance the interpretability of the clusters generated by the hybrid GMM, a method leveraging the majority heuristic to label the clusters based on the ground truth was employed. This approach is pivotal in aligning the initially unlabeled clusters with the predefined categories within the dataset, thereby ensuring that the clustering outcomes are not only meaningful but also actionable.</p>
<p>Thus, the analysis of the ground truth labels associated with each data point, uniquely identified by an individual identifier, is conducted within each cluster. This process involves systematically evaluating the distribution of ground truth labels among the data points in each cluster. By determining the most prevalent or dominant ground truth label within a cluster, that label is designated as the representative identity for the entire cluster. For example, if the majority of data points within a cluster were labeled as “True,” this label is assigned to the entire cluster, thus defining its identity.</p>
<p>The supplementary file presents a subset of data points (news) randomly sampled from each cluster, providing a representative overview of the clustering results. According to supplementary file, cluster 1, as demonstrated in the given Table, predominantly includes news statements labeled as “Mostly True”. In contrast, the cluster 2 is characterized by a majority of news items labeled as either “False” or “Barely True.” Furthermore, the cluster 3 is predominantly composed of news items labeled as “Pants on Fire,” a designation reserved for statements that are not only false but are also egregiously incorrect or exaggerated. The consistent presence of these labels within a particular cluster highlights the effectiveness of the hybrid GMM algorithm in grouping news items that share a similar information.</p>
<p>A more thorough examination of the clustering results revealed that the proposed hybrid Gaussian Mixture Model (GMM) effectively consolidated the six subcategories of the news dataset into three primary categories. As previously discussed, Cluster 1 (green colored) primarily comprises news items labeled as “Mostly True,” this cluster can be labeled as “Mostly Accurate” cluster. On the other hand, Cluster 2 (purple colored) is characterized by a majority of news items labeled as either “False” or “Barely True.” Consequently, this cluster can be categorized under the “Partially Accurate” label, reflecting the presence of news items that, while containing some elements of truth, are either misleading or significantly distorted. Additionally, Cluster 3 (yellow colored), is predominantly composed of news items labeled as “Pants on Fire”. As a result, this cluster can be labeled as “Inaccurate”. Furthermore, it also simplifies the data, facilitating clearer distinctions between authentic and misleading information, which is crucial for the detection and analysis of fake information.</p>
<p>These categorizations illustrate the capability of the clustering algorithm to effectively distinguish among varying levels of truthfulness or misinformation in the news items and accurately reflecting the underlying structure of the dataset. Furthermore, by aligning clusters with these major categories, the analysis reinforces the robustness and reliability of the clustering approach, providing a clear understanding of the distribution of authentic and misleading information across the dataset. This structured categorization also enhances the potential for practical applications, such as misinformation detection and content validation, by enabling targeted analysis and intervention based on the identified cluster identities.</p>
<p>We also conducted a comprehensive set of evaluation to compare the performance of our proposed approach against traditional clustering algorithms, including Gaussian Mixture Model (GMM), K-means and K-medoids using the same news dataset. This comparative analysis was essential to evaluate the effectiveness and robustness of our method in contrast to classical clustering techniques.</p>
<p>During the evaluation each algorithm was meticulously applied under standardized conditions. The Bayesian Information Criterion (BIC) was utilized to ascertain the optimal number of components for traditional GMM [<a href="#pone.0330421.ref134" class="usa-link" aria-describedby="pone.0330421.ref134">134</a>]. For K-means and K-medoids clustering techniques, the Elbow method was employed to determine the optimal number of clusters, providing a visual assessment of the elbow point where adding more clusters fails to significantly improve the model [135].</p>
<p>The elbow point serves as a visual indicator for selecting the optimal number of clusters by analyzing the Within-Cluster Sum of Squares (WCSS) [<a href="#pone.0330421.ref133" class="usa-link" aria-describedby="pone.0330421.ref133">133</a>,<a href="#pone.0330421.ref134" class="usa-link" aria-describedby="pone.0330421.ref134">134</a>]. When applied to the news dataset, the Elbow method identified three clusters for K-means and five clusters for K-medoids, as shown in <a href="#pone.0330421.g005" class="usa-link">Fig 5</a>. For K-means, it can be observed that increasing the number of clusters up to three as depicted in <a href="#pone.0330421.g005" class="usa-link">Fig 5(A)</a> resulted in a significant reduction in WCSS (Elbow point), beyond which the improvement becomes marginal. At this point, the trade-off between reducing WCSS and avoiding the unnecessary complexity of adding more clusters is balanced, ensuring that the selected model is efficient and well-fitted to the data and avoiding over-fitting. Similarly, for K-medoids, adding up to five clusters led to a substantial decrease in WCSS, after which further increases did not lead to noticeable improvement as depicted in <a href="#pone.0330421.g005" class="usa-link">Fig 5(B)</a>. Whereas BIC also suggested five clusters (components) for the classical GMM as can be seen in <a href="#pone.0330421.g006" class="usa-link">Fig 6</a>. It also demonstrated that lower BIC scores indicating better model fit. The asterisk (*) highlights the configuration with the lowest BIC score, representing the optimal number of clusters and covariance structure for the GMM on the given news dataset.</p>
<figure class="fig xbox font-sm" id="pone.0330421.g005"><h3 class="obj_head">Fig 5. Optimal Number of Cluster selection using Elbow method whereas (A) represent the Elbow graph for K-means to assess the Elbow point and (B) represent the Elbow graph for K-medoids clustering techniques respectively.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360590_pone.0330421.g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a3e1/12360590/1983ca4421b9/pone.0330421.g005.jpg" loading="lazy" height="357" width="775" alt="Fig 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330421.g005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><figure class="fig xbox font-sm" id="pone.0330421.g006"><h3 class="obj_head">Fig 6. Optimal number of cluster selection using BIC method for classical GMM clustering technique.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360590_pone.0330421.g006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a3e1/12360590/4ff4126e709c/pone.0330421.g006.jpg" loading="lazy" height="503" width="675" alt="Fig 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330421.g006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>This divergence in the optimal number of clusters across the algorithms highlights the sensitivity of each clustering technique to the underlying data. The justification related to the selection of these parameter selection techniques is detailed in the Methodology section.</p>
<p>Furthermore in the context of fake news detection, <a href="#pone.0330421.g007" class="usa-link">Figs 7</a> and <a href="#pone.0330421.g008" class="usa-link">8</a> illustrates the clustering results obtained through traditional GMM and K-medoid clustering respectively, on the same dataset. These visual representations suggest a notable contrast in the distribution of data points between fake and true labels. Both traditional GMM and K-medoid identified five clusters, closely aligning with the five subcategories present in the news dataset. On the other hand, K-means discovered only three clusters as depicted in <a href="#pone.0330421.g009" class="usa-link">Fig 9</a>. The evaluation results demonstrated that K-means algorithm tends to generalize the data, potentially merging subcategories that share similar characteristics, unlike GMM, which assigns data points based on probability distributions.</p>
<figure class="fig xbox font-sm" id="pone.0330421.g007"><h3 class="obj_head">Fig 7. Clustering results obtained using Classical GMM on scaled TF-IDF data, representing the distribution of news articles among different clusters based on textual similarities.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360590_pone.0330421.g007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a3e1/12360590/2cf0a34ec467/pone.0330421.g007.jpg" loading="lazy" height="476" width="660" alt="Fig 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330421.g007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><figure class="fig xbox font-sm" id="pone.0330421.g008"><h3 class="obj_head">Fig 8. Clustering results obtained using K-medoid.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360590_pone.0330421.g008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a3e1/12360590/47b13c711397/pone.0330421.g008.jpg" loading="lazy" height="476" width="660" alt="Fig 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330421.g008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><figure class="fig xbox font-sm" id="pone.0330421.g009"><h3 class="obj_head">Fig 9. Clustering results obtained using K-means.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12360590_pone.0330421.g009.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a3e1/12360590/5100155660aa/pone.0330421.g009.jpg" loading="lazy" height="476" width="660" alt="Fig 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0330421.g009/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Furthermore, K-means algorithm minimize intra-cluster variance and effectively consolidates subcategories with shared characteristics into unified clusters. This integration is particularly beneficial as it reduces the complexity of the dataset and highlights broader patterns across related subcategories that exhibit surface-level similarities. It also provide a more comprehensive analysis of general trends and themes within the data. Such clustering can provide valuable insights into the structural relationships between different types of news content, enhancing the overall understanding of the dataset without compromising interpretability.</p>
<p>Therefore, to provide insight into the internal structure of clusters and their alignment with ground truth, we employed three performance measures including the Silhouette Score, Adjusted Rand Index (ARI), and Purity Score. Furthermore, the performance of these varied techniques in selecting the most suitable cluster count can be assessed through their respective Silhouette Score, Adjusted Rand Index (ARI), and Purity Score. <a href="#pone.0330421.t001" class="usa-link">Table 1</a> demonstrated a comparative analysis of the performance for each clustering algorithm.</p>
<section class="tw xbox font-sm" id="pone.0330421.t001"><h3 class="obj_head">Table 1. Comparative Analysis of evaluation metrics for Hybrid GMM, K-medoid, Classical GMM and K-means.</h3>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Algorithm</th>
<th align="left" rowspan="1" colspan="1">Silhouette Score</th>
<th align="left" rowspan="1" colspan="1">Adjusted Rand Index (ARI)</th>
<th align="left" rowspan="1" colspan="1">Purity Score</th>
<th align="left" rowspan="1" colspan="1">F-measure</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Hybrid GMM</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>0.77</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>0.83</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>0.88</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<strong>0.85</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>K-medoid</strong>
</td>
<td align="left" rowspan="1" colspan="1">0.52</td>
<td align="left" rowspan="1" colspan="1">0.62</td>
<td align="left" rowspan="1" colspan="1">0.73</td>
<td align="left" rowspan="1" colspan="1">0.69</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Classical GMM</strong>
</td>
<td align="left" rowspan="1" colspan="1">0.61</td>
<td align="left" rowspan="1" colspan="1">0.68</td>
<td align="left" rowspan="1" colspan="1">0.75</td>
<td align="left" rowspan="1" colspan="1">0.72</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>K-means</strong>
</td>
<td align="left" rowspan="1" colspan="1">0.65</td>
<td align="left" rowspan="1" colspan="1">0.72</td>
<td align="left" rowspan="1" colspan="1">0.79</td>
<td align="left" rowspan="1" colspan="1">0.76</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0330421.t001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>The evaluation results demonstrated the superiority of our proposed Hybrid GMM, achieving a silhouette score of 0.77, which demonstrated more compact and well-separated clustering results, leading to better-defined cluster boundaries. The K-means algorithm also performed relatively well, with a silhouette score of 0.65, indicating that while its clusters are reasonably distinct, they are less cohesive than those produced by the Hybrid GMM.</p>
<p>The Classical GMM yielded a silhouette score of 0.61, showing moderate cluste r separation and cohesion. Although it performed better than K-medoids, its lower score compared to the Hybrid GMM reflects its inability to define well-separated clusters.</p>
<p>On the other hand, the K-medoid algorithm demonstrated the lowest performance, with a silhouette score of 0.52. This low score suggests that the clusters produced by K-medoid are less distinct and more prone to overlap, potentially leading to ambiguous clustering assignments. It can also be inferred that K-medoid may not be as effective in capturing the complex structure of the data. Furthermore, comparative analysis of evaluation results also depicted that hyprid GMM depicted an adjusted rand score of 0.83 and a purity score of 0.88. The purity score of hybrid GMM demonstrated a strong ability to form clusters with or homogeneous data points. On the other hand, K-means clustering algorithm yields 0.72 and 0.79 an adjusted rand score and purity score respectively. Classical GMM and K- medoid clustering slightly performed poorly with an adjusted rand score and purity score of 0.68, of 0.75, 0.62 and 0.73 respectively.</p>
<p>In the context of clustering Fake News, the emphasis is often on achieving high Purity to ensure that most articles within a cluster are from the same category. Hybrid GMM, with the superior silhouette, adjusted rand score and purity appears to be the more effective model for this specific task. The present findings highlight substantial differences in clustering performance as measured by ARI on news data among the clustering techniques incorporated in this research. Therefore, based on the evaluation measures and the specific focus on clustering fake news, the proposed hybrid GMM emerges as significantly promising model for this analysis. Its promising performance across all the performance measures particularly purity score highlighted the Hybrid GMM’s ability to effectively handle complex data, providing more reliable and well-defined clustering outcomes compared to classical GMM, k-means and k-medoid cluster methods incorporated in this research.</p>
<p>Furthermore, the proposed method is also aligned with real-world practical applications, especially in the realm of social media monitoring and content regulation. While the results are encouraging, the proposed technique has several limitation. such as it does not incorporate community detection algorithms, which could aid in addressing the spread of misinformation and enable network-wide countermeasures. Moreover, the inclusion of hybrid models utilizing deep learning techniques and transformer-based architectures could improve detection precision by capturing intricate contextual subtleties within the text.</p></section><section id="sec025"><h2 class="pmc_sec_title">Conclusion</h2>
<p>This research proposed a hybrid Gaussian Mixture Model (GMM) for fake news detection. By leveraging Group Counseling Optimization (GCO), a meta-heuristic optimization algorithm, the proposed approach ensured robust parameter optimization for hybrid GMM and automatically determining the optimal number of clusters, significantly enhancing the reliability and performance of the Expectation-Maximization (EM) algorithm. The evaluation results also demonstrated that the proposed hybrid GMM outperformed traditional clustering methods, including classical GMM, K-means, and K-medoids. It can be concluded that the Hybrid GMM is a robust and promising solution for fake news detection, with practical potential for combating misinformation on social media platforms.</p></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>
<a href="https://github.com/Sajida-Perveen/Project-code" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Sajida-Perveen/Project-code</a>.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>The author(s) received no specific funding for this work.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pone.0330421.ref001">
<span class="label">1.</span><cite>Livas C, Delli K. Sharing and assessing the impact of orthodontic research on social media: The case of altimetric. In Seminars in Orthodontics. 2023. 
Jul
22.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Sharing%20and%20assessing%20the%20impact%20of%20orthodontic%20research%20on%20social%20media:%20The%20case%20of%20altimetric&amp;author=C%20Livas&amp;author=K%20Delli&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref002">
<span class="label">2.</span><cite>Elejalde E, Ferres L, Schifanella R. Understanding news outlets’ audience-targeting patterns. EPJ Data Science. 2019;8(1):1–20.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=EPJ%20Data%20Science&amp;title=Understanding%20news%20outlets%E2%80%99%20audience-targeting%20patterns&amp;author=E%20Elejalde&amp;author=L%20Ferres&amp;author=R%20Schifanella&amp;volume=8&amp;issue=1&amp;publication_year=2019&amp;pages=1-20&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref003">
<span class="label">3.</span><cite>Alzubi A. The digital age: transforming Jordanian traditional newspapers through online media in information dissemination. Journal of Information Science. 2023;3:1–10.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Information%20Science&amp;title=The%20digital%20age:%20transforming%20Jordanian%20traditional%20newspapers%20through%20online%20media%20in%20information%20dissemination&amp;author=A%20Alzubi&amp;volume=3&amp;publication_year=2023&amp;pages=1-10&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref004">
<span class="label">4.</span><cite>Salawu A, Tsutsa F, Mpofu P. Adoption of digital media by African language print newspapers: The case of Isolezwe. e-BANGI Journal. 2022;19(5).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=e-BANGI%20Journal&amp;title=Adoption%20of%20digital%20media%20by%20African%20language%20print%20newspapers:%20The%20case%20of%20Isolezwe&amp;author=A%20Salawu&amp;author=F%20Tsutsa&amp;author=P%20Mpofu&amp;volume=19&amp;issue=5&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref005">
<span class="label">5.</span><cite>Dash M, Belgaonkar P. Comparative effectiveness of radio, print and web advertising. Print and Web Advertising (August 15, 2012). 2012. 
Aug
15.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Comparative%20effectiveness%20of%20radio,%20print%20and%20web%20advertising&amp;author=M%20Dash&amp;author=P%20Belgaonkar&amp;publication_year=2012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref006">
<span class="label">6.</span><cite>Duan LY, Wang J, Gao W, Lu H, Jin JS, Xu C. Linking traditional TV advertising to internet advertising. Online multimedia advertising: Techniques and technologies. IGI Global. 2011. p. 166–93.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Online%20multimedia%20advertising:%20Techniques%20and%20technologies&amp;author=LY%20Duan&amp;author=J%20Wang&amp;author=W%20Gao&amp;author=H%20Lu&amp;author=JS%20Jin&amp;publication_year=2011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref007">
<span class="label">7.</span><cite>Kulshrestha J, Zafar M, Noboa L, Gummadi K, Ghosh S. Characterizing information diets of social media users. In Proceedings of the international AAAI conference on web and social media. 2015. 218–27.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=In%20Proceedings%20of%20the%20international%20AAAI%20conference%20on%20web%20and%20social%20media&amp;title=Characterizing%20information%20diets%20of%20social%20media%20users&amp;author=J%20Kulshrestha&amp;author=M%20Zafar&amp;author=L%20Noboa&amp;author=K%20Gummadi&amp;author=S%20Ghosh&amp;publication_year=2015&amp;pages=218-27&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref008">
<span class="label">8.</span><cite>Webb HC, Emam M. Social media and fake news impact on social movements: examples from Tunisia and Egypt. Global perspectives on the impact of mass media on electoral processes. IGI Global. 2021. p. 40–56.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Global%20perspectives%20on%20the%20impact%20of%20mass%20media%20on%20electoral%20processes&amp;author=HC%20Webb&amp;author=M%20Emam&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref009">
<span class="label">9.</span><cite>Abdul Wahab S, Sakrani SN, Hashim MA, Ismail R. Understanding behavior of users in delivering and receiving entertainment news via accessing web portals. In: Proceedings of the Colloquium on Administrative Science and Technology: CoAST 2013, 2014. 263–74.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Understanding%20behavior%20of%20users%20in%20delivering%20and%20receiving%20entertainment%20news%20via%20accessing%20web%20portals.%20In:%20Proceedings%20of%20the%20Colloquium%20on%20Administrative%20Science%20and%20Technology:%20CoAST%202013&amp;author=S%20Abdul%20Wahab&amp;author=SN%20Sakrani&amp;author=MA%20Hashim&amp;author=R%20Ismail&amp;publication_year=2014&amp;pages=263-74&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref010">
<span class="label">10.</span><cite>Chen Z, Chen H, Freire J, Nagler J, Tucker JA. Understanding how people consume low quality and extreme news using web traffic data. arXiv preprint. 2022. doi: 10.48550/arXiv.2201.04226</cite> [<a href="https://doi.org/10.48550/arXiv.2201.04226" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=arXiv%20preprint&amp;title=Understanding%20how%20people%20consume%20low%20quality%20and%20extreme%20news%20using%20web%20traffic%20data&amp;author=Z%20Chen&amp;author=H%20Chen&amp;author=J%20Freire&amp;author=J%20Nagler&amp;author=JA%20Tucker&amp;publication_year=2022&amp;doi=10.48550/arXiv.2201.04226&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref011">
<span class="label">11.</span><cite>Arun C. Making choices: Social media platforms and freedom of expression norms. Regardless of frontiers: Global freedom of expression in a troubled world. Columbia University Press. 2021. p. 275–87.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Regardless%20of%20frontiers:%20Global%20freedom%20of%20expression%20in%20a%20troubled%20world&amp;author=C%20Arun&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref012">
<span class="label">12.</span><cite>Zhou T, Li S. Understanding user switch of information seeking: from search engines to generative AI. Journal of Librarianship and Information Science. 2024. doi: 09610006241244800</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Librarianship%20and%20Information%20Science&amp;title=Understanding%20user%20switch%20of%20information%20seeking:%20from%20search%20engines%20to%20generative%20AI&amp;author=T%20Zhou&amp;author=S%20Li&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref013">
<span class="label">13.</span><cite>Jain S, Jaiswal P, Jain A, Sinha A, Gupta A. Analyzing Influential Impact of Online Trends on Social Media Channels. In: 2022 8th International Conference on Signal Processing and Communication (ICSC). 2022. 49–53.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Analyzing%20Influential%20Impact%20of%20Online%20Trends%20on%20Social%20Media%20Channels.%20In:%202022%208th%20International%20Conference%20on%20Signal%20Processing%20and%20Communication%20(ICSC)&amp;author=S%20Jain&amp;author=P%20Jaiswal&amp;author=A%20Jain&amp;author=A%20Sinha&amp;author=A%20Gupta&amp;publication_year=2022&amp;pages=49-53&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref014">
<span class="label">14.</span><cite>Steinberg M. The Impact of Social Media in Web-Based Information Management and Digital Media. Integrated Information and Computing Systems for Natural, Spatial, and Social Sciences. IGI Global. 2013. p. 432–53.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Integrated%20Information%20and%20Computing%20Systems%20for%20Natural,%20Spatial,%20and%20Social%20Sciences&amp;author=M%20Steinberg&amp;publication_year=2013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref015">
<span class="label">15.</span><cite>Poleac G, Ghergut-Babii AN. How social media algorithms influence the way users decide - perspectives of social media users and practitioners. Technium Soc Sci J. 2024;57:69.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Technium%20Soc%20Sci%20J&amp;title=How%20social%20media%20algorithms%20influence%20the%20way%20users%20decide%20-%20perspectives%20of%20social%20media%20users%20and%20practitioners&amp;author=G%20Poleac&amp;author=AN%20Ghergut-Babii&amp;volume=57&amp;publication_year=2024&amp;pages=69&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref016">
<span class="label">16.</span><cite>Pinto R, Filgueiras E. Media appropriations of young consumers: production and consumption of digital content on social networks. Ergonomics In Design. 2023;77(77).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Ergonomics%20In%20Design&amp;title=Media%20appropriations%20of%20young%20consumers:%20production%20and%20consumption%20of%20digital%20content%20on%20social%20networks&amp;author=R%20Pinto&amp;author=E%20Filgueiras&amp;volume=77&amp;issue=77&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref017">
<span class="label">17.</span><cite>Plenković M, Mustić D. Odnos starosti korisnika i vizualne putanje kod korištenja digitalnih medija. Informatologia. 2019;52(3–4):164–72.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Informatologia&amp;title=Odnos%20starosti%20korisnika%20i%20vizualne%20putanje%20kod%20kori%C5%A1tenja%20digitalnih%20medija&amp;author=M%20Plenkovi%C4%87&amp;author=D%20Musti%C4%87&amp;volume=52&amp;publication_year=2019&amp;pages=164-72&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref018">
<span class="label">18.</span><cite>Beckerle H, Finston R, Sussman B. Social media debate position 1: against the use of social media as a credible source of information. Internet Reference Services Quarterly. 2021;25(1–2):25–35.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Internet%20Reference%20Services%20Quarterly&amp;title=Social%20media%20debate%20position%201:%20against%20the%20use%20of%20social%20media%20as%20a%20credible%20source%20of%20information&amp;author=H%20Beckerle&amp;author=R%20Finston&amp;author=B%20Sussman&amp;volume=25&amp;publication_year=2021&amp;pages=25-35&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref019">
<span class="label">19.</span><cite>Pang H. How compulsive WeChat use and information overload affect social media fatigue and well-being during the COVID-19 pandemic? A stressor-strain-outcome perspective. Telematics and Informatics. 2021;64:101690.
</cite> [<a href="https://doi.org/10.1016/j.tele.2021.101690" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9759653/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36567817/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Telematics%20and%20Informatics&amp;title=How%20compulsive%20WeChat%20use%20and%20information%20overload%20affect%20social%20media%20fatigue%20and%20well-being%20during%20the%20COVID-19%20pandemic?%20A%20stressor-strain-outcome%20perspective&amp;author=H%20Pang&amp;volume=64&amp;publication_year=2021&amp;pages=101690&amp;pmid=36567817&amp;doi=10.1016/j.tele.2021.101690&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref020">
<span class="label">20.</span><cite>Yi S, Guo Y, Lin Z, Cao C. Is information evaluated subjectively? Social media has changed the way users search for medical information. Digit Health. 2024;10:20552076241259039. doi: 10.1177/20552076241259039

</cite> [<a href="https://doi.org/10.1177/20552076241259039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11135116/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38812844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Digit%20Health&amp;title=Is%20information%20evaluated%20subjectively?%20Social%20media%20has%20changed%20the%20way%20users%20search%20for%20medical%20information&amp;author=S%20Yi&amp;author=Y%20Guo&amp;author=Z%20Lin&amp;author=C%20Cao&amp;volume=10&amp;publication_year=2024&amp;pages=20552076241259039&amp;pmid=38812844&amp;doi=10.1177/20552076241259039&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref021">
<span class="label">21.</span><cite>Mohammed A, Kumar S, Mu’Azu HG, Kumar R, Shah P, Memoria M, et al. Data security and protection: A mechanism for managing data theft and cybercrime in online platforms of educational institutions. In: 2022 International Conference on Machine Learning, Big Data, Cloud and Parallel Computing (COM-IT-CON), 2022. 758–61.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=2022%20International%20Conference%20on%20Machine%20Learning,%20Big%20Data,%20Cloud%20and%20Parallel%20Computing%20(COM-IT-CON)&amp;title=Data%20security%20and%20protection:%20A%20mechanism%20for%20managing%20data%20theft%20and%20cybercrime%20in%20online%20platforms%20of%20educational%20institutions&amp;author=A%20Mohammed&amp;author=S%20Kumar&amp;author=HG%20Mu%E2%80%99Azu&amp;author=R%20Kumar&amp;author=P%20Shah&amp;publication_year=2022&amp;pages=758-61&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref022">
<span class="label">22.</span><cite>Iftikhar I, Yasmeen B, Ahmad N. Honeycomb framework: examining the social and personal drivers of fake news sharing on social media platforms. Journal of Development and Social Sciences. 2022;3(3):01–16.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Development%20and%20Social%20Sciences&amp;title=Honeycomb%20framework:%20examining%20the%20social%20and%20personal%20drivers%20of%20fake%20news%20sharing%20on%20social%20media%20platforms&amp;author=I%20Iftikhar&amp;author=B%20Yasmeen&amp;author=N%20Ahmad&amp;volume=3&amp;issue=3&amp;publication_year=2022&amp;pages=01-16&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref023">
<span class="label">23.</span><cite>Gerbina TV. Science disinformation: on the problem of fake news. Scientific and Technical Information Processing. 2021;48(4):290–8.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20and%20Technical%20Information%20Processing&amp;title=Science%20disinformation:%20on%20the%20problem%20of%20fake%20news&amp;author=TV%20Gerbina&amp;volume=48&amp;issue=4&amp;publication_year=2021&amp;pages=290-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref024">
<span class="label">24.</span><cite>Baiyere A, Avital M, Dennis A, Gibbs J, Te’eni D. Fake News: What Is It and Why Does It Matter. In The Academy of Management Annual Meeting 2020: Broadening Our Sight. 2020.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=In%20The%20Academy%20of%20Management%20Annual%20Meeting%202020:%20Broadening%20Our%20Sight&amp;author=A%20Baiyere&amp;author=M%20Avital&amp;author=A%20Dennis&amp;author=J%20Gibbs&amp;author=D%20Te%E2%80%99eni&amp;publication_year=2020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref025">
<span class="label">25.</span><cite>Myers ME. Propaganda, fake news, and deepfaking. Understanding media psychology. Routledge. 2021. p. 161–81.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Understanding%20media%20psychology&amp;author=ME%20Myers&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref026">
<span class="label">26.</span><cite>Allcott H, Gentzkow M. Social media and fake news in the 2016 election. Journal of Economic Perspectives. 2017;31(2):211–36.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Economic%20Perspectives&amp;title=Social%20media%20and%20fake%20news%20in%20the%202016%20election&amp;author=H%20Allcott&amp;author=M%20Gentzkow&amp;volume=31&amp;issue=2&amp;publication_year=2017&amp;pages=211-36&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref027">
<span class="label">27.</span><cite>Vereshchaka A, Cosimini S, Dong W. Analyzing and distinguishing fake and real news to mitigate the problem of disinformation. Computational and Mathematical Organization Theory. 2020;26:350–64.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Computational%20and%20Mathematical%20Organization%20Theory&amp;title=Analyzing%20and%20distinguishing%20fake%20and%20real%20news%20to%20mitigate%20the%20problem%20of%20disinformation&amp;author=A%20Vereshchaka&amp;author=S%20Cosimini&amp;author=W%20Dong&amp;volume=26&amp;publication_year=2020&amp;pages=350-64&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref028">
<span class="label">28.</span><cite>Tandoc EC, Seet SK. War of the words: How individuals respond to “fake news,” “misinformation,” “disinformation,” and “online falsehoods”. Journalism Practice. 2024;18(6):1503–19.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journalism%20Practice&amp;title=War%20of%20the%20words:%20How%20individuals%20respond%20to%20%E2%80%9Cfake%20news,%E2%80%9D%20%E2%80%9Cmisinformation,%E2%80%9D%20%E2%80%9Cdisinformation,%E2%80%9D%20and%20%E2%80%9Conline%20falsehoods%E2%80%9D&amp;author=EC%20Tandoc&amp;author=SK%20Seet&amp;volume=18&amp;issue=6&amp;publication_year=2024&amp;pages=1503-19&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref029">
<span class="label">29.</span><cite>Liu Y, Wu YF. Early detection of fake news on social media through propagation path classification with recurrent and convolutional networks. In: Proceedings of the AAAI conference on artificial intelligence. 2018.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20AAAI%20conference%20on%20artificial%20intelligence&amp;title=Early%20detection%20of%20fake%20news%20on%20social%20media%20through%20propagation%20path%20classification%20with%20recurrent%20and%20convolutional%20networks&amp;author=Y%20Liu&amp;author=YF%20Wu&amp;publication_year=2018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref030">
<span class="label">30.</span><cite>Afify E, Sharaf Eldin A, Khedr E, Kamal Alsheref F. User-generated content (UGC) credibility on social media using sentiment classification. النشرة المعلوماتية في الحاسبات والمعلومات. 2019;1(1):1–9.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=%D8%A7%D9%84%D9%86%D8%B4%D8%B1%D8%A9%20%D8%A7%D9%84%D9%85%D8%B9%D9%84%D9%88%D9%85%D8%A7%D8%AA%D9%8A%D8%A9%20%D9%81%D9%8A%20%D8%A7%D9%84%D8%AD%D8%A7%D8%B3%D8%A8%D8%A7%D8%AA%20%D9%88%D8%A7%D9%84%D9%85%D8%B9%D9%84%D9%88%D9%85%D8%A7%D8%AA&amp;title=User-generated%20content%20(UGC)%20credibility%20on%20social%20media%20using%20sentiment%20classification&amp;author=E%20Afify&amp;author=A%20Sharaf%20Eldin&amp;author=E%20Khedr&amp;author=F%20Kamal%20Alsheref&amp;volume=1&amp;issue=1&amp;publication_year=2019&amp;pages=1-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref031">
<span class="label">31.</span><cite>Kumar S, Shah N. False information on web and social media: a survey. arXiv preprint. 2018. doi: 10.48550/arXiv:1804.08559</cite> [<a href="https://doi.org/10.48550/arXiv:1804.08559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=arXiv%20preprint&amp;title=False%20information%20on%20web%20and%20social%20media:%20a%20survey&amp;author=S%20Kumar&amp;author=N%20Shah&amp;publication_year=2018&amp;doi=10.48550/arXiv:1804.08559&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref032">
<span class="label">32.</span><cite>Riaz Sherani M. Investigation of information sharing via multiple social media platforms: a comparison of Facebook and WeChat adoption. Quality &amp; Quantity. 2021;55(5):1751–73.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Quality%20&amp;%20Quantity&amp;title=Investigation%20of%20information%20sharing%20via%20multiple%20social%20media%20platforms:%20a%20comparison%20of%20Facebook%20and%20WeChat%20adoption&amp;author=M%20Riaz%20Sherani&amp;volume=55&amp;issue=5&amp;publication_year=2021&amp;pages=1751-73&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref033">
<span class="label">33.</span><cite>Arora N. Misinformation, fake news and rumor detection. Principles of social networking: The new horizon and emerging challenges. Singapore: Springer Singapore. 2021. p. 307–24.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Principles%20of%20social%20networking:%20The%20new%20horizon%20and%20emerging%20challenges&amp;author=N%20Arora&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref034">
<span class="label">34.</span><cite>Ajao O, Garg A, Da Costa-Abreu M. Exploring content-based and meta-data analysis for detecting fake news infodemic: A case study on COVID-19. In: 2022 12th International Conference on Pattern Recognition Systems (ICPRS). 2022. 1–8.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=2022%2012th%20International%20Conference%20on%20Pattern%20Recognition%20Systems%20(ICPRS)&amp;title=Exploring%20content-based%20and%20meta-data%20analysis%20for%20detecting%20fake%20news%20infodemic:%20A%20case%20study%20on%20COVID-19.&amp;author=O%20Ajao&amp;author=A%20Garg&amp;author=M%20Da%20Costa-Abreu&amp;publication_year=2022&amp;pages=1-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref035">
<span class="label">35.</span><cite>Kulkarni P, Prabhu S, Kumar S, Ramraj B. COVID-19-infodemic overtaking pandemic? Time to disseminate facts over fear. Indian journal of community health. 2020;32(2):264–8.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Indian%20journal%20of%20community%20health&amp;title=COVID-19-infodemic%20overtaking%20pandemic?%20Time%20to%20disseminate%20facts%20over%20fear&amp;author=P%20Kulkarni&amp;author=S%20Prabhu&amp;author=S%20Kumar&amp;author=B%20Ramraj&amp;volume=32&amp;issue=2&amp;publication_year=2020&amp;pages=264-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref036">
<span class="label">36.</span><cite>Ali K, Zain-ul-abdin K. Post-truth propaganda: heuristic processing of political fake news on Facebook during the 2016 US presidential election. Journal of Applied Communication Research. 2021;49(1):109–28.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Applied%20Communication%20Research&amp;title=Post-truth%20propaganda:%20heuristic%20processing%20of%20political%20fake%20news%20on%20Facebook%20during%20the%202016%20US%20presidential%20election&amp;author=K%20Ali&amp;author=K%20Zain-ul-abdin&amp;volume=49&amp;issue=1&amp;publication_year=2021&amp;pages=109-28&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref037">
<span class="label">37.</span><cite>Mayakul T, Srisawat W, Kiattisin S. Fake news and online disinformation. A perspectives of Thai government officials. In: 2019 4th Technology Innovation Management and Engineering Science International Conference (TIMES-iCON). 2019. 1–4.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=2019%204th%20Technology%20Innovation%20Management%20and%20Engineering%20Science%20International%20Conference%20(TIMES-iCON)&amp;title=Fake%20news%20and%20online%20disinformation.%20A%20perspectives%20of%20Thai%20government%20officials.&amp;author=T%20Mayakul&amp;author=W%20Srisawat&amp;author=S%20Kiattisin&amp;publication_year=2019&amp;pages=1-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref038">
<span class="label">38.</span><cite>Aïmeur E, Amri S, Brassard G. Fake news, disinformation and misinformation in social media: a review. Social Network Analysis and Mining. 2023. 
Feb
9;13(1):30.
</cite> [<a href="https://doi.org/10.1007/s13278-023-01028-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9910783/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36789378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Social%20Network%20Analysis%20and%20Mining&amp;title=Fake%20news,%20disinformation%20and%20misinformation%20in%20social%20media:%20a%20review&amp;author=E%20A%C3%AFmeur&amp;author=S%20Amri&amp;author=G%20Brassard&amp;volume=13&amp;issue=1&amp;publication_year=2023&amp;pages=30&amp;pmid=36789378&amp;doi=10.1007/s13278-023-01028-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref039">
<span class="label">39.</span><cite>Shushkevich E, Cardiff J. Detecting fake news about Covid-19 on small datasets with machine learning algorithms. In: 2021 30th Conference of Open Innovations Association FRUCT.2021. 253–8.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=2021%2030th%20Conference%20of%20Open%20Innovations%20Association%20FRUCT&amp;title=Detecting%20fake%20news%20about%20Covid-19%20on%20small%20datasets%20with%20machine%20learning%20algorithms.&amp;author=E%20Shushkevich&amp;author=J%20Cardiff&amp;publication_year=2021&amp;pages=253-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref040">
<span class="label">40.</span><cite>Singh A, Patidar S. Fake news detection using supervised machine learning classification algorithms. Inventive computation and information technologies: Proceedings of ICICIT 2022. Singapore: Springer Nature Singapore. 2023. p. 919–33.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Inventive%20computation%20and%20information%20technologies:%20Proceedings%20of%20ICICIT%202022&amp;author=A%20Singh&amp;author=S%20Patidar&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref041">
<span class="label">41.</span><cite>Adeyiga JA, Toriola PG, Abioye TE, Oluwatosin AE. Fake News Detection Using a Logistic Regression Model and Natural Language Processing Techniques.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Fake%20News%20Detection%20Using%20a%20Logistic%20Regression%20Model%20and%20Natural%20Language%20Processing%20Techniques&amp;author=JA%20Adeyiga&amp;author=PG%20Toriola&amp;author=TE%20Abioye&amp;author=AE%20Oluwatosin&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref042">
<span class="label">42.</span><cite>Bussa S, Bodhankar A, Patil VH, Pal H, Bunkar SK, Qureshi AK. An Implementation of Machine Learning Algorithm for Fake News Detection. International Journal on Recent and Innovation Trends in Computing and Communication. 2023.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20on%20Recent%20and%20Innovation%20Trends%20in%20Computing%20and%20Communication&amp;title=An%20Implementation%20of%20Machine%20Learning%20Algorithm%20for%20Fake%20News%20Detection&amp;author=S%20Bussa&amp;author=A%20Bodhankar&amp;author=VH%20Patil&amp;author=H%20Pal&amp;author=SK%20Bunkar&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref043">
<span class="label">43.</span><cite>Palahan S. Comparative analysis of deep learning models for predicting online review helpfulness. In: Proceedings of the 2023 Asia Conference on Computer Vision, Image Processing and Pattern Recognition, 2023. 1–5.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%202023%20Asia%20Conference%20on%20Computer%20Vision,%20Image%20Processing%20and%20Pattern%20Recognition&amp;title=Comparative%20analysis%20of%20deep%20learning%20models%20for%20predicting%20online%20review%20helpfulness.&amp;author=S%20Palahan&amp;publication_year=2023&amp;pages=1-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref044">
<span class="label">44.</span><cite>Paredes DG. Strategies based on artificial intelligence for the detection of fake news. AWARI. 2023;4:1–6.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=AWARI&amp;title=Strategies%20based%20on%20artificial%20intelligence%20for%20the%20detection%20of%20fake%20news&amp;author=DG%20Paredes&amp;volume=4&amp;publication_year=2023&amp;pages=1-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref045">
<span class="label">45.</span><cite>Mridha MF, Keya AJ, Hamid MA, Monowar MM, Rahman MS. A comprehensive review on fake news detection with deep learning. IEEE Access. 2021;9:156151–70.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Access&amp;title=A%20comprehensive%20review%20on%20fake%20news%20detection%20with%20deep%20learning&amp;author=MF%20Mridha&amp;author=AJ%20Keya&amp;author=MA%20Hamid&amp;author=MM%20Monowar&amp;author=MS%20Rahman&amp;volume=9&amp;publication_year=2021&amp;pages=156151-70&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref046">
<span class="label">46.</span><cite>Degadwala S, Vyas D. Systematic analysis of deep learning models vs. machine learning.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Systematic%20analysis%20of%20deep%20learning%20models%20vs.%20machine%20learning&amp;author=S%20Degadwala&amp;author=D%20Vyas&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref047">
<span class="label">47.</span><cite>Shi C, Wei B, Wei S, Wang W, Liu H, Liu J. A quantitative discriminant method of elbow point for the optimal number of clusters in clustering algorithm. EURASIP Journal on Wireless Communications and Networking. 2021;2021:1–6.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=EURASIP%20Journal%20on%20Wireless%20Communications%20and%20Networking&amp;title=A%20quantitative%20discriminant%20method%20of%20elbow%20point%20for%20the%20optimal%20number%20of%20clusters%20in%20clustering%20algorithm&amp;author=C%20Shi&amp;author=B%20Wei&amp;author=S%20Wei&amp;author=W%20Wang&amp;author=H%20Liu&amp;volume=2021&amp;publication_year=2021&amp;pages=1-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref048">
<span class="label">48.</span><cite>Nadeem M, Aslam MU, Shahzad S. Unravelling the Web of Fake News: Understanding. 2023.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Unravelling%20the%20Web%20of%20Fake%20News:%20Understanding&amp;author=M%20Nadeem&amp;author=MU%20Aslam&amp;author=S%20Shahzad&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref049">
<span class="label">49.</span><cite>Jiang Y, Porter MD. Quantifying the influence of user behaviors on the dissemination of fake news on Twitter with multivariate Hawkes processes. arXiv preprint. 2023. doi: arXiv:2308.13927</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=arXiv%20preprint&amp;title=Quantifying%20the%20influence%20of%20user%20behaviors%20on%20the%20dissemination%20of%20fake%20news%20on%20Twitter%20with%20multivariate%20Hawkes%20processes&amp;author=Y%20Jiang&amp;author=MD%20Porter&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref050">
<span class="label">50.</span><cite>Ojha RP, Srivastava PK, Awasthi S, Srivastava V, Pandey PS, Dwivedi RS, et al. Controlling of fake information dissemination in online social networks: an epidemiological approach. IEEE Access. 2023;11:32229–40.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Access&amp;title=Controlling%20of%20fake%20information%20dissemination%20in%20online%20social%20networks:%20an%20epidemiological%20approach&amp;author=RP%20Ojha&amp;author=PK%20Srivastava&amp;author=S%20Awasthi&amp;author=V%20Srivastava&amp;author=PS%20Pandey&amp;volume=11&amp;publication_year=2023&amp;pages=32229-40&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref051">
<span class="label">51.</span><cite>Shah D, Singh A, Prasad SS. Sentimental Analysis Using Supervised Learning Algorithms. In2022 3rd International Conference on Computation, Automation and Knowledge Management (ICCAKM). IEEE. 2022. 1–6.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=In2022%203rd%20International%20Conference%20on%20Computation,%20Automation%20and%20Knowledge%20Management%20(ICCAKM)&amp;author=D%20Shah&amp;author=A%20Singh&amp;author=SS%20Prasad&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref052">
<span class="label">52.</span><cite>Choubey PK, Lee A, Huang R, Wang L. Discourse as a function of event: Profiling discourse structure in news articles around the main event. In: Proceedings of the 58th Annual Meeting of the Association for Computational Linguistics. 2020.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%2058th%20Annual%20Meeting%20of%20the%20Association%20for%20Computational%20Linguistics&amp;title=Discourse%20as%20a%20function%20of%20event:%20Profiling%20discourse%20structure%20in%20news%20articles%20around%20the%20main%20event.&amp;author=PK%20Choubey&amp;author=A%20Lee&amp;author=R%20Huang&amp;author=L%20Wang&amp;publication_year=2020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref053">
<span class="label">53.</span><cite>Endres I, Farhadi A, Hoiem D, Forsyth DA. The benefits and challenges of collecting richer object annotations. In: 2010 IEEE Computer Society Conference on Computer Vision and Pattern Recognition-Workshops. 2010. 1–8.26413416
</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=2010%20IEEE%20Computer%20Society%20Conference%20on%20Computer%20Vision%20and%20Pattern%20Recognition-Workshops&amp;title=The%20benefits%20and%20challenges%20of%20collecting%20richer%20object%20annotations.&amp;author=I%20Endres&amp;author=A%20Farhadi&amp;author=D%20Hoiem&amp;author=DA%20Forsyth&amp;publication_year=2010&amp;pages=1-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref054">
<span class="label">54.</span><cite>Imran AS, Cheikh FA, Kowalski SJ. Media annotations in hyperlinked pedagogical platforms. In2014 International Conference on Web and Open Access to Learning (ICWOAL). 2014. 1–6.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=In2014%20International%20Conference%20on%20Web%20and%20Open%20Access%20to%20Learning%20(ICWOAL)&amp;author=AS%20Imran&amp;author=FA%20Cheikh&amp;author=SJ%20Kowalski&amp;publication_year=2014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref055">
<span class="label">55.</span><cite>Lee S, Kim J, Kim D, Kim KJ, Park E. Computational approaches to developing the implicit media bias dataset: assessing political orientations of nonpolitical news articles. Applied Mathematics and Computation. 2023;458:128219.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20Mathematics%20and%20Computation&amp;title=Computational%20approaches%20to%20developing%20the%20implicit%20media%20bias%20dataset:%20assessing%20political%20orientations%20of%20nonpolitical%20news%20articles&amp;author=S%20Lee&amp;author=J%20Kim&amp;author=D%20Kim&amp;author=KJ%20Kim&amp;author=E%20Park&amp;volume=458&amp;publication_year=2023&amp;pages=128219&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref056">
<span class="label">56.</span><cite>Bojjireddy S, Chun SA, Geller J. Machine learning approach to detect fake news, misinformation in COVID-19 pandemic. In DG. O2021: The 22nd Annual International Conference on Digital Government Research. 2021. 575–8.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=In%20DG.%20O2021:%20The%2022nd%20Annual%20International%20Conference%20on%20Digital%20Government%20Research&amp;author=S%20Bojjireddy&amp;author=SA%20Chun&amp;author=J%20Geller&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref057">
<span class="label">57.</span><cite>Batailler C, Brannon SM, Teas PE, Gawronski B. A Signal Detection Approach to Understanding the Identification of Fake News. Perspect Psychol Sci. 2022;17(1):78–98. doi: 10.1177/1745691620986135

</cite> [<a href="https://doi.org/10.1177/1745691620986135" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34264150/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Perspect%20Psychol%20Sci&amp;title=A%20Signal%20Detection%20Approach%20to%20Understanding%20the%20Identification%20of%20Fake%20News&amp;author=C%20Batailler&amp;author=SM%20Brannon&amp;author=PE%20Teas&amp;author=B%20Gawronski&amp;volume=17&amp;issue=1&amp;publication_year=2022&amp;pages=78-98&amp;pmid=34264150&amp;doi=10.1177/1745691620986135&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref058">
<span class="label">58.</span><cite>Zhu X, Wang C. Unsupervised posture modeling and recognition based on Gaussian mixture model and EM estimation. J Softw. 2011;6(8):1445–51.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Softw&amp;title=Unsupervised%20posture%20modeling%20and%20recognition%20based%20on%20Gaussian%20mixture%20model%20and%20EM%20estimation&amp;author=X%20Zhu&amp;author=C%20Wang&amp;volume=6&amp;issue=8&amp;publication_year=2011&amp;pages=1445-51&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref059">
<span class="label">59.</span><cite>Januzaj Y, Beqiri E, Luma A. Determining the Optimal Number of Clusters using Silhouette Score as a Data Mining Technique. International Journal of Online &amp; Biomedical Engineering. 2023;19(4).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Online%20&amp;%20Biomedical%20Engineering&amp;title=Determining%20the%20Optimal%20Number%20of%20Clusters%20using%20Silhouette%20Score%20as%20a%20Data%20Mining%20Technique&amp;author=Y%20Januzaj&amp;author=E%20Beqiri&amp;author=A%20Luma&amp;volume=19&amp;issue=4&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref060">
<span class="label">60.</span><cite>Li G, Li B, Chen C, Tan S, Qiu G. Learning General Gaussian Mixture Model with Integral Cosine Similarity. In: IJCAI 2022. 3201–7.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Learning%20General%20Gaussian%20Mixture%20Model%20with%20Integral%20Cosine%20Similarity.%20In:%20IJCAI%202022&amp;author=G%20Li&amp;author=B%20Li&amp;author=C%20Chen&amp;author=S%20Tan&amp;author=G%20Qiu&amp;pages=3201-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref061">
<span class="label">61.</span><cite>An P, Wang Z, Zhang C. Ensemble unsupervised autoencoders and Gaussian mixture model for cyberattack detection. Information Processing &amp; Management. 2022;59(2):102844.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Information%20Processing%20&amp;%20Management&amp;title=Ensemble%20unsupervised%20autoencoders%20and%20Gaussian%20mixture%20model%20for%20cyberattack%20detection&amp;author=P%20An&amp;author=Z%20Wang&amp;author=C%20Zhang&amp;volume=59&amp;issue=2&amp;publication_year=2022&amp;pages=102844&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref062">
<span class="label">62.</span><cite>Wang LW, Feng JF. Learning Gaussian mixture models by structural risk minimization. In: 2005 International Conference on Machine Learning and Cybernetics. 2005. 4858–63.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=2005%20International%20Conference%20on%20Machine%20Learning%20and%20Cybernetics&amp;title=Learning%20Gaussian%20mixture%20models%20by%20structural%20risk%20minimization.&amp;author=LW%20Wang&amp;author=JF%20Feng&amp;publication_year=2005&amp;pages=4858-63&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref063">
<span class="label">63.</span><cite>Ming-ming G, Tai-hua C, Xiang-xiang G. Application of Gaussian Mixture Model Genetic Algorithm in data stream clustering analysis. In 2010 IEEE International Conference on Intelligent Computing and Intelligent Systems. 2010.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=In%202010%20IEEE%20International%20Conference%20on%20Intelligent%20Computing%20and%20Intelligent%20Systems&amp;author=G%20Ming-ming&amp;author=C%20Tai-hua&amp;author=G%20Xiang-xiang&amp;publication_year=2010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref064">
<span class="label">64.</span><cite>Yang X, Huang K, Goulermas JY, Zhang R. Joint learning of unsupervised dimensionality reduction and gaussian mixture model. Neural Processing Letters. 2017;45:791–806.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Neural%20Processing%20Letters&amp;title=Joint%20learning%20of%20unsupervised%20dimensionality%20reduction%20and%20gaussian%20mixture%20model&amp;author=X%20Yang&amp;author=K%20Huang&amp;author=JY%20Goulermas&amp;author=R%20Zhang&amp;volume=45&amp;publication_year=2017&amp;pages=791-806&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref065">
<span class="label">65.</span><cite>Liu X, Zhu S, Yang F, Liang S. Research on unsupervised anomaly data detection method based on improve Zhu automatic encoder and Gaussian mixture model. Journal of Cloud Computing. 2022;11(1):58.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cloud%20Computing&amp;title=Research%20on%20unsupervised%20anomaly%20data%20detection%20method%20based%20on%20improve%20Zhu%20automatic%20encoder%20and%20Gaussian%20mixture%20model&amp;author=X%20Liu&amp;author=S%20Zhu&amp;author=F%20Yang&amp;author=S%20Liang&amp;volume=11&amp;issue=1&amp;publication_year=2022&amp;pages=58&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref066">
<span class="label">66.</span><cite>Ding N, Ma H, Gao H, Ma Y, Tan G. Real-time anomaly detection based on long short-term memory and Gaussian mixture model. Computers &amp; Electrical Engineering. 2019;79:106458.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Computers%20&amp;%20Electrical%20Engineering&amp;title=Real-time%20anomaly%20detection%20based%20on%20long%20short-term%20memory%20and%20Gaussian%20mixture%20model&amp;author=N%20Ding&amp;author=H%20Ma&amp;author=H%20Gao&amp;author=Y%20Ma&amp;author=G%20Tan&amp;volume=79&amp;publication_year=2019&amp;pages=106458&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref067">
<span class="label">67.</span><cite>Si L, Jin R. Adjusting mixture weights of gaussian mixture model via regularized probabilistic latent semantic analysis. In: Pacific-Asia Conference on Knowledge Discovery and Data Mining. 2005. 622–31.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Pacific-Asia%20Conference%20on%20Knowledge%20Discovery%20and%20Data%20Mining&amp;title=Adjusting%20mixture%20weights%20of%20gaussian%20mixture%20model%20via%20regularized%20probabilistic%20latent%20semantic%20analysis.&amp;author=L%20Si&amp;author=R%20Jin&amp;publication_year=2005&amp;pages=622-31&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref068">
<span class="label">68.</span><cite>Gillani N. Uncovering latent features in massive open online courses (Doctoral dissertation, Oxford University, UK).</cite>
</li>
<li id="pone.0330421.ref069">
<span class="label">69.</span><cite>Wang B, Lin C, Fan X, Jiang N, Farina D. Hierarchical Bayes based adaptive sparsity in Gaussian mixture model. Pattern Recognition Letters. 2014;49:238–47.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Pattern%20Recognition%20Letters&amp;title=Hierarchical%20Bayes%20based%20adaptive%20sparsity%20in%20Gaussian%20mixture%20model&amp;author=B%20Wang&amp;author=C%20Lin&amp;author=X%20Fan&amp;author=N%20Jiang&amp;author=D%20Farina&amp;volume=49&amp;publication_year=2014&amp;pages=238-47&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref070">
<span class="label">70.</span><cite>Yu G, Sun J, Li C. Machine performance assessment using Gaussian mixture model (GMM). In2008 2nd International Symposium on Systems and Control in Aerospace and Astronautics
IEEE. 2008. 1–6.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Machine%20performance%20assessment%20using%20Gaussian%20mixture%20model%20(GMM).%20In2008%202nd%20International%20Symposium%20on%20Systems%20and%20Control%20in%20Aerospace%20and%20Astronautics&amp;author=G%20Yu&amp;author=J%20Sun&amp;author=C%20Li&amp;publication_year=2008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref071">
<span class="label">71.</span><cite>Yang H, Suyong EU. Feature analysis on English word difficulty by Gaussian mixture model. In: 2018 International Conference on Information and Communication Technology Convergence (ICTC), 2018. 191–4.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=2018%20International%20Conference%20on%20Information%20and%20Communication%20Technology%20Convergence%20(ICTC)&amp;title=Feature%20analysis%20on%20English%20word%20difficulty%20by%20Gaussian%20mixture%20model.&amp;author=H%20Yang&amp;author=EU%20Suyong&amp;publication_year=2018&amp;pages=191-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref072">
<span class="label">72.</span><cite>Zhang J, Zhu Z, Zou J. Supervised Gaussian process latent variable model based on Gaussian mixture model. In: 2017 International Conference on Security, Pattern Analysis, and Cybernetics (SPAC). IEEE. 2017. 124–9.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=2017%20International%20Conference%20on%20Security,%20Pattern%20Analysis,%20and%20Cybernetics%20(SPAC)&amp;author=J%20Zhang&amp;author=Z%20Zhu&amp;author=J%20Zou&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref073">
<span class="label">73.</span><cite>Arı Ç. Maximum likelihood estimation of robust constrained gaussian mixture models. Turkey: Bilkent Universitesi. 2023.</cite> [<a href="https://scholar.google.com/scholar_lookup?Ar%C4%B1%20%C3%87.%20Maximum%20likelihood%20estimation%20of%20robust%20constrained%20gaussian%20mixture%20models.%20Turkey:%20Bilkent%20Universitesi.%202023." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref074">
<span class="label">74.</span><cite>Melnykov V, Melnikova I. Initializing the EM algorithm in Gaussian mixture models with an unknown number of components. Computational Statistics &amp; Data Analysis. 2012;56(6):1381–95.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Computational%20Statistics%20&amp;%20Data%20Analysis&amp;title=Initializing%20the%20EM%20algorithm%20in%20Gaussian%20mixture%20models%20with%20an%20unknown%20number%20of%20components&amp;author=V%20Melnykov&amp;author=I%20Melnikova&amp;volume=56&amp;issue=6&amp;publication_year=2012&amp;pages=1381-95&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref075">
<span class="label">75.</span><cite>Abdolazimi R, Jin S, Varshney PK, Zafarani R. Harnessing the power of noise: A survey of techniques and applications. arXiv preprint. 2024. doi: 10.48550/arXiv.2410.06348</cite> [<a href="https://doi.org/10.48550/arXiv.2410.06348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=arXiv%20preprint&amp;title=Harnessing%20the%20power%20of%20noise:%20A%20survey%20of%20techniques%20and%20applications&amp;author=R%20Abdolazimi&amp;author=S%20Jin&amp;author=PK%20Varshney&amp;author=R%20Zafarani&amp;publication_year=2024&amp;doi=10.48550/arXiv.2410.06348&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref076">
<span class="label">76.</span><cite>Eita MA, Fahmy MM. Group counseling optimization: a novel approach. Research and Development in Intelligent Systems XXVI: Incorporating Applications and Innovations in Intelligent Systems XVII. London: Springer London. 2009. p. 195–208.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Research%20and%20Development%20in%20Intelligent%20Systems%20XXVI:%20Incorporating%20Applications%20and%20Innovations%20in%20Intelligent%20Systems%20XVII&amp;author=MA%20Eita&amp;author=MM%20Fahmy&amp;publication_year=2009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref077">
<span class="label">77.</span><cite>Biswas S, Eita MA, Das S, Vasilakos AV. Evaluating the performance of group counseling optimizer on CEC 2014 problems for computational expensive optimization. In2014 IEEE Congress on Evolutionary Computation (CEC). 2014. 1076–83.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=In2014%20IEEE%20Congress%20on%20Evolutionary%20Computation%20(CEC)&amp;author=S%20Biswas&amp;author=MA%20Eita&amp;author=S%20Das&amp;author=AV%20Vasilakos&amp;publication_year=2014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref078">
<span class="label">78.</span><cite>Gupta A, Sharma M, Srivastava A. Group counselling optimization-derived software bug prediction framework by optimal feature selection-based dimensionality reduction. In: 2023 13th International Conference on Cloud Computing, Data Science &amp; Engineering (Confluence). IEEE; 2023. 636–42.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=2023%2013th%20International%20Conference%20on%20Cloud%20Computing,%20Data%20Science%20&amp;%20Engineering%20(Confluence).%20IEEE&amp;title=Group%20counselling%20optimization-derived%20software%20bug%20prediction%20framework%20by%20optimal%20feature%20selection-based%20dimensionality%20reduction.&amp;author=A%20Gupta&amp;author=M%20Sharma&amp;author=A%20Srivastava&amp;publication_year=2023&amp;pages=636-42&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref079">
<span class="label">79.</span><cite>Kaur R, Kaur A. Text Generator using Natural Language Processing Methods. In2023 International Conference on Computational Intelligence, Communication Technology and Networking (CICTN). IEEE; 2023. 421–5.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=In2023%20International%20Conference%20on%20Computational%20Intelligence,%20Communication%20Technology%20and%20Networking%20(CICTN)&amp;author=R%20Kaur&amp;author=A%20Kaur&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref080">
<span class="label">80.</span><cite>Vishnuprabha V, Murali L, Viswanathan DM. Natural Language Processing. Cognitive Computing Systems. Apple Academic Press. 2021. p. 349–400.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Cognitive%20Computing%20Systems&amp;author=V%20Vishnuprabha&amp;author=L%20Murali&amp;author=DM%20Viswanathan&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref081">
<span class="label">81.</span><cite>Yogish D, Manjunath TN, Hegadi RS. Review on natural language processing trends and techniques using NLTK. Recent Trends in Image Processing and Pattern Recognition: Second International Conference, RTIP2R 2018, Solapur, India, December 21–22, 2018, Revised Selected Papers, Part III. Springer Singapore. 2019. p. 589–606.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Recent%20Trends%20in%20Image%20Processing%20and%20Pattern%20Recognition:%20Second%20International%20Conference,%20RTIP2R%202018,%20Solapur,%20India,%20December%2021%E2%80%9322,%202018,%20Revised%20Selected%20Papers,%20Part%20III&amp;author=D%20Yogish&amp;author=TN%20Manjunath&amp;author=RS%20Hegadi&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref082">
<span class="label">82.</span><cite>Chai CP. Comparison of text preprocessing methods. Natural Language Engineering. 2023;29(3):509–53.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Natural%20Language%20Engineering&amp;title=Comparison%20of%20text%20preprocessing%20methods&amp;author=CP%20Chai&amp;volume=29&amp;issue=3&amp;publication_year=2023&amp;pages=509-53&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref083">
<span class="label">83.</span><cite>Kochhar TS, Goyal G. Design and implementation of stop words removal method for Punjabi language using finite automata. In: Advances in Data Computing, Communication and Security: Proceedings of I3CS2021. 2022. 89–98.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Advances%20in%20Data%20Computing,%20Communication%20and%20Security:%20Proceedings%20of%20I3CS2021&amp;title=Design%20and%20implementation%20of%20stop%20words%20removal%20method%20for%20Punjabi%20language%20using%20finite%20automata.&amp;author=TS%20Kochhar&amp;author=G%20Goyal&amp;publication_year=2022&amp;pages=89-98&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref084">
<span class="label">84.</span><cite>Bijal D, Sanket S. Overview of stemming algorithms for Indian and non-Indian languages. arXiv preprint. 2014. doi: arXiv:1404.2878</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=arXiv%20preprint&amp;title=Overview%20of%20stemming%20algorithms%20for%20Indian%20and%20non-Indian%20languages&amp;author=D%20Bijal&amp;author=S%20Sanket&amp;publication_year=2014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref085">
<span class="label">85.</span><cite>Elov B, Hamroyeva S, Abdullayeva O, Husainova Z, Xudayberganov N. Agglutinativ tillar uchun pos teglash va stemming masalasi (turk, uyg ‘ur, o ‘zbek tillari misolida). Language and Culture. 2023;2(2).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Language%20and%20Culture&amp;title=Agglutinativ%20tillar%20uchun%20pos%20teglash%20va%20stemming%20masalasi%20(turk,%20uyg%20%E2%80%98ur,%20o%20%E2%80%98zbek%20tillari%20misolida)&amp;author=B%20Elov&amp;author=S%20Hamroyeva&amp;author=O%20Abdullayeva&amp;author=Z%20Husainova&amp;author=N%20Xudayberganov&amp;volume=2&amp;issue=2&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref086">
<span class="label">86.</span><cite>Jain L, Agrawal P. Text independent root word identification in Hindi language using natural language processing. International Journal of Advanced Intelligence Paradigms. 2015;7(3–4):240–9.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Advanced%20Intelligence%20Paradigms&amp;title=Text%20independent%20root%20word%20identification%20in%20Hindi%20language%20using%20natural%20language%20processing&amp;author=L%20Jain&amp;author=P%20Agrawal&amp;volume=7&amp;publication_year=2015&amp;pages=240-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref087">
<span class="label">87.</span><cite>Jabbar A, Iqbal S, Ilahi M. High performance stemming algorithm to handle multi-level inflections in Urdu Language.</cite>
</li>
<li id="pone.0330421.ref088">
<span class="label">88.</span><cite>Zhang C, Peng B, Sun X, Niu Q, Liu J, Chen K, Li M, Feng P, Bi Z, Liu M, Zhang Y. From Word Vectors to Multimodal Embeddings: Techniques, Applications, and Future Directions For Large Language Models. arXiv preprint arXiv:2411.05036. 2024. 
Nov
6. doi: arXiv:2411.05036</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=arXiv%20preprint%20arXiv:2411.05036&amp;title=From%20Word%20Vectors%20to%20Multimodal%20Embeddings:%20Techniques,%20Applications,%20and%20Future%20Directions%20For%20Large%20Language%20Models&amp;author=C%20Zhang&amp;author=B%20Peng&amp;author=X%20Sun&amp;author=Q%20Niu&amp;author=J%20Liu&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref089">
<span class="label">89.</span><cite>Deo TK, Deshmukh RK, Sharma G. Comparative study among term frequency-inverse document frequency and count vectorizer towards K nearest neighbor and decision tree classifiers for text dataset. Nepal Journal of Multidisciplinary Research. 2024;7(2):1–1.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Nepal%20Journal%20of%20Multidisciplinary%20Research&amp;title=Comparative%20study%20among%20term%20frequency-inverse%20document%20frequency%20and%20count%20vectorizer%20towards%20K%20nearest%20neighbor%20and%20decision%20tree%20classifiers%20for%20text%20dataset&amp;author=TK%20Deo&amp;author=RK%20Deshmukh&amp;author=G%20Sharma&amp;volume=7&amp;issue=2&amp;publication_year=2024&amp;pages=1-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref090">
<span class="label">90.</span><cite>Dai W. Classification and analysis of literary works based on distribution weighted term frequency-inverse document frequency. Journal of Physics: Conference Series. 2021. 012018.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Physics:%20Conference%20Series&amp;title=Classification%20and%20analysis%20of%20literary%20works%20based%20on%20distribution%20weighted%20term%20frequency-inverse%20document%20frequency&amp;author=W%20Dai&amp;publication_year=2021&amp;pages=012018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref091">
<span class="label">91.</span><cite>Katya E. Exploring feature engineering strategies for improving predictive models in data science. Research Journal of Computer Systems and Engineering. 2023;4(2):201–15.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Research%20Journal%20of%20Computer%20Systems%20and%20Engineering&amp;title=Exploring%20feature%20engineering%20strategies%20for%20improving%20predictive%20models%20in%20data%20science&amp;author=E%20Katya&amp;volume=4&amp;issue=2&amp;publication_year=2023&amp;pages=201-15&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref092">
<span class="label">92.</span><cite>Khalid S, Khalil T, Nasreen S. A survey of feature selection and feature extraction techniques in machine learning. In: 2014 Science and Information Conference. IEEE; 2014. 372–8.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=2014%20Science%20and%20Information%20Conference.%20IEEE&amp;title=A%20survey%20of%20feature%20selection%20and%20feature%20extraction%20techniques%20in%20machine%20learning.&amp;author=S%20Khalid&amp;author=T%20Khalil&amp;author=S%20Nasreen&amp;publication_year=2014&amp;pages=372-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref093">
<span class="label">93.</span><cite>Yan J, Zhang B, Liu N, Yan S, Cheng Q, Fan W, et al. Effective and efficient dimensionality reduction for large-scale and streaming data preprocessing. IEEE Transactions on Knowledge and Data Engineering. 2006;18(3):320–33.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Transactions%20on%20Knowledge%20and%20Data%20Engineering&amp;title=Effective%20and%20efficient%20dimensionality%20reduction%20for%20large-scale%20and%20streaming%20data%20preprocessing&amp;author=J%20Yan&amp;author=B%20Zhang&amp;author=N%20Liu&amp;author=S%20Yan&amp;author=Q%20Cheng&amp;volume=18&amp;issue=3&amp;publication_year=2006&amp;pages=320-33&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref094">
<span class="label">94.</span><cite>Piles M, Bergsma R, Gianola D, Gilbert H, Tusell L. Feature selection stability and accuracy of prediction models for genomic prediction of residual feed intake in pigs using machine learning. Frontiers in Genetics. 2021;12:611506.
</cite> [<a href="https://doi.org/10.3389/fgene.2021.611506" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7938892/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33692825/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Feature%20selection%20stability%20and%20accuracy%20of%20prediction%20models%20for%20genomic%20prediction%20of%20residual%20feed%20intake%20in%20pigs%20using%20machine%20learning&amp;author=M%20Piles&amp;author=R%20Bergsma&amp;author=D%20Gianola&amp;author=H%20Gilbert&amp;author=L%20Tusell&amp;volume=12&amp;publication_year=2021&amp;pages=611506&amp;pmid=33692825&amp;doi=10.3389/fgene.2021.611506&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref095">
<span class="label">95.</span><cite>Qu K, Xu J, Hou Q, Qu K, Sun Y. Feature selection using information gain and decision information in neighborhood decision system. Applied Soft Computing. 2023;136:110100.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20Soft%20Computing&amp;title=Feature%20selection%20using%20information%20gain%20and%20decision%20information%20in%20neighborhood%20decision%20system&amp;author=K%20Qu&amp;author=J%20Xu&amp;author=Q%20Hou&amp;author=K%20Qu&amp;author=Y%20Sun&amp;volume=136&amp;publication_year=2023&amp;pages=110100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref096">
<span class="label">96.</span><cite>Qian J, Mao Y, Tang J, Wang L. Feature selection method based on maximum conditional and joint mutual information. In: 2019 IEEE 4th International Conference on Image, Vision and Computing (ICIVC). IEEE. 2019. 607–11.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=2019%20IEEE%204th%20International%20Conference%20on%20Image,%20Vision%20and%20Computing%20(ICIVC).%20IEEE&amp;title=Feature%20selection%20method%20based%20on%20maximum%20conditional%20and%20joint%20mutual%20information.&amp;author=J%20Qian&amp;author=Y%20Mao&amp;author=J%20Tang&amp;author=L%20Wang&amp;publication_year=2019&amp;pages=607-11&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref097">
<span class="label">97.</span><cite>Yuanfang W, Lianshou L. Conditional entropy and mutual information in random cascading processes. Physical Review D. 1991;43(9):3077.</cite> [<a href="https://doi.org/10.1103/physrevd.43.3077" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10013706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Physical%20Review%20D&amp;title=Conditional%20entropy%20and%20mutual%20information%20in%20random%20cascading%20processes&amp;author=W%20Yuanfang&amp;author=L%20Lianshou&amp;volume=43&amp;issue=9&amp;publication_year=1991&amp;pages=3077&amp;pmid=10013706&amp;doi=10.1103/physrevd.43.3077&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref098">
<span class="label">98.</span><cite>Ezinne Ogbuanya C. Improved Dimensionality Reduction of various Datasets using Novel Multiplicative Factoring Principal Component Analysis (MPCA). arXiv e-prints. 2020 Sep:arXiv. 2009.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=arXiv%20e-prints.%202020%20Sep:arXiv&amp;author=C%20Ezinne%20Ogbuanya&amp;publication_year=2009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref099">
<span class="label">99.</span><cite>Jolliffe IT. Principal component analysis and factor analysis. Principal component analysis. 2002. p. 150–66.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Principal%20component%20analysis&amp;author=IT%20Jolliffe&amp;publication_year=2002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref100">
<span class="label">100.</span><cite>Mejia H, Chipantiza C, Llumiquinga J, Amaro IR, Fonseca-Delgado R. Covid-19 fake news detection using joint doc2vec and text features with pca. In International Conference on Advanced Research in Technologies, Information, Innovation and Sustainability. Cham: Springer Nature Switzerland. 2022. 316–30.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=In%20International%20Conference%20on%20Advanced%20Research%20in%20Technologies,%20Information,%20Innovation%20and%20Sustainability&amp;author=H%20Mejia&amp;author=C%20Chipantiza&amp;author=J%20Llumiquinga&amp;author=IR%20Amaro&amp;author=R%20Fonseca-Delgado&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref101">
<span class="label">101.</span><cite>Ali AA, Latif S, Ghauri SA, Song OY, Abbasi AA, Malik AJ. Linguistic Features and Bi-LSTM for Identification of Fake News. Electronics. 2023;12(13):2942.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Electronics&amp;title=Linguistic%20Features%20and%20Bi-LSTM%20for%20Identification%20of%20Fake%20News&amp;author=AA%20Ali&amp;author=S%20Latif&amp;author=SA%20Ghauri&amp;author=OY%20Song&amp;author=AA%20Abbasi&amp;volume=12&amp;issue=13&amp;publication_year=2023&amp;pages=2942&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref102">
<span class="label">102.</span><cite>Chovanec D, Paralic J. Features to distinguish between trustworthy and fake news. Acta Electrotechnica et Informatica. 2021;21(2):3–6.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Electrotechnica%20et%20Informatica&amp;title=Features%20to%20distinguish%20between%20trustworthy%20and%20fake%20news&amp;author=D%20Chovanec&amp;author=J%20Paralic&amp;volume=21&amp;issue=2&amp;publication_year=2021&amp;pages=3-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref103">
<span class="label">103.</span><cite>Aljrees T, Cheng X, Ahmed MM, Umer M, Majeed R, Alnowaiser K, et al. Fake news stance detection using selective features and FakeNET. PLoS One. 2023;18(7):e0287298. doi: 10.1371/journal.pone.0287298

</cite> [<a href="https://doi.org/10.1371/journal.pone.0287298" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10389754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37523404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Fake%20news%20stance%20detection%20using%20selective%20features%20and%20FakeNET&amp;author=T%20Aljrees&amp;author=X%20Cheng&amp;author=MM%20Ahmed&amp;author=M%20Umer&amp;author=R%20Majeed&amp;volume=18&amp;issue=7&amp;publication_year=2023&amp;pmid=37523404&amp;doi=10.1371/journal.pone.0287298&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref104">
<span class="label">104.</span><cite>Mejia H, Chipantiza C, Llumiquinga J, Amaro IR, Fonseca-Delgado R. Covid-19 fake news detection using joint doc2vec and text features with pca. In International Conference on Advanced Research in Technologies, Information, Innovation and Sustainability. Cham: Springer Nature Switzerland. 2022. 316–30.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Covid-19%20fake%20news%20detection%20using%20joint%20doc2vec%20and%20text%20features%20with%20pca.%20In%20International%20Conference%20on%20Advanced%20Research%20in%20Technologies,%20Information,%20Innovation%20and%20Sustainability&amp;author=H%20Mejia&amp;author=C%20Chipantiza&amp;author=J%20Llumiquinga&amp;author=IR%20Amaro&amp;author=R%20Fonseca-Delgado&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref105">
<span class="label">105.</span><cite>Wu J, Ye X. FakeSwarm: Improving Fake News Detection with Swarming Characteristics. arXiv preprint. 2023. doi: 10.48550/arXiv.2305.19194</cite> [<a href="https://doi.org/10.48550/arXiv.2305.19194" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=arXiv%20preprint&amp;title=FakeSwarm:%20Improving%20Fake%20News%20Detection%20with%20Swarming%20Characteristics&amp;author=J%20Wu&amp;author=X%20Ye&amp;publication_year=2023&amp;doi=10.48550/arXiv.2305.19194&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref106">
<span class="label">106.</span><cite>Gordon MH, Cerezo M, Cincio L, Coles PJ. Covariance matrix preparation for quantum principal component analysis. PRX Quantum. 2022;3(3):030334.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=PRX%20Quantum&amp;title=Covariance%20matrix%20preparation%20for%20quantum%20principal%20component%20analysis&amp;author=MH%20Gordon&amp;author=M%20Cerezo&amp;author=L%20Cincio&amp;author=PJ%20Coles&amp;volume=3&amp;issue=3&amp;publication_year=2022&amp;pages=030334&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref107">
<span class="label">107.</span><cite>Xiong H, Wu J, Chen J. K-means clustering versus validation measures: a data distribution perspective. In Proceedings of the 12th ACM SIGKDD international conference on Knowledge discovery and data mining. 2006. 779–84.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=In%20Proceedings%20of%20the%2012th%20ACM%20SIGKDD%20international%20conference%20on%20Knowledge%20discovery%20and%20data%20mining&amp;author=H%20Xiong&amp;author=J%20Wu&amp;author=J%20Chen&amp;publication_year=2006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref108">
<span class="label">108.</span><cite>Ren X, Wang M, Dai G, Peng L, Chen X, Song Z. Balancing convergence and diversity: Gaussian mixture models in adaptive weight vector strategies for multi-objective algorithms. Information Sciences. 2025;700:121858.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Information%20Sciences&amp;title=Balancing%20convergence%20and%20diversity:%20Gaussian%20mixture%20models%20in%20adaptive%20weight%20vector%20strategies%20for%20multi-objective%20algorithms&amp;author=X%20Ren&amp;author=M%20Wang&amp;author=G%20Dai&amp;author=L%20Peng&amp;author=X%20Chen&amp;volume=700&amp;publication_year=2025&amp;pages=121858&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref109">
<span class="label">109.</span><cite>Ikotun AM, Ezugwu AE, Abualigah L, Abuhaija B, Heming J. K-means clustering algorithms: A comprehensive review, variants analysis, and advances in the era of big data. Information Sciences. 2023;622:178–210.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Information%20Sciences&amp;title=K-means%20clustering%20algorithms:%20A%20comprehensive%20review,%20variants%20analysis,%20and%20advances%20in%20the%20era%20of%20big%20data&amp;author=AM%20Ikotun&amp;author=AE%20Ezugwu&amp;author=L%20Abualigah&amp;author=B%20Abuhaija&amp;author=J%20Heming&amp;volume=622&amp;publication_year=2023&amp;pages=178-210&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref110">
<span class="label">110.</span><cite>Ahmed M, Seraj R, Islam SM. The k-means algorithm: A comprehensive survey and performance evaluation. Electronics. 2020;9(8):1295.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Electronics&amp;title=The%20k-means%20algorithm:%20A%20comprehensive%20survey%20and%20performance%20evaluation&amp;author=M%20Ahmed&amp;author=R%20Seraj&amp;author=SM%20Islam&amp;volume=9&amp;issue=8&amp;publication_year=2020&amp;pages=1295&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref111">
<span class="label">111.</span><cite>Shamsuddin NR, Mahat NI. Comparison between k-means and k-medoids for mixed variables clustering. In: Proceedings of the Third International Conference on Computing, Mathematics and Statistics (iCMS2017) Transcending Boundaries, Embracing Multidisciplinary Diversities. Springer Singapore. 2019. 303–8.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20Third%20International%20Conference%20on%20Computing,%20Mathematics%20and%20Statistics%20(iCMS2017)%20Transcending%20Boundaries,%20Embracing%20Multidisciplinary%20Diversities.%20Springer%20Singapore&amp;title=Comparison%20between%20k-means%20and%20k-medoids%20for%20mixed%20variables%20clustering.&amp;author=NR%20Shamsuddin&amp;author=NI%20Mahat&amp;publication_year=2019&amp;pages=303-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref112">
<span class="label">112.</span><cite>Qona’ah N, Devi AR, Dana IM. Laboratory clustering using k-means, k-medoids, and model-based clustering. Indonesian Journal of Applied Statistics. 2020;3(1):64–77.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Indonesian%20Journal%20of%20Applied%20Statistics&amp;title=Laboratory%20clustering%20using%20k-means,%20k-medoids,%20and%20model-based%20clustering&amp;author=N%20Qona%E2%80%99ah&amp;author=AR%20Devi&amp;author=IM%20Dana&amp;volume=3&amp;issue=1&amp;publication_year=2020&amp;pages=64-77&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref113">
<span class="label">113.</span><cite>Hidaka A, Kurita T. Nonlinear discriminant analysis based on probability estimation by Gaussian mixture model. In: Structural, syntactic, and statistical pattern recognition: Joint IAPR International Workshop, S SSPR 2014, Joensuu, Finland, August 20-22, 2014. Proceedings 2014. Springer Berlin Heidelberg. 2014. 133–42.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Structural,%20syntactic,%20and%20statistical%20pattern%20recognition:%20Joint%20IAPR%20International%20Workshop,%20S%20SSPR%202014,%20Joensuu,%20Finland,%20August%2020-22,%202014.%20Proceedings%202014&amp;author=A%20Hidaka&amp;author=T%20Kurita&amp;publication_year=2014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref114">
<span class="label">114.</span><cite>Baftiu D, Dodds T. Adapting to Twitter: The entanglement of journalistic values and online personas. Journalism Studies. 2023;24(10):1295–315.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journalism%20Studies&amp;title=Adapting%20to%20Twitter:%20The%20entanglement%20of%20journalistic%20values%20and%20online%20personas&amp;author=D%20Baftiu&amp;author=T%20Dodds&amp;volume=24&amp;issue=10&amp;publication_year=2023&amp;pages=1295-315&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref115">
<span class="label">115.</span><cite>Schwarz G. Estimating the dimension of a model. The Annals of Statistics. 1978;:461–4.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Annals%20of%20Statistics&amp;title=Estimating%20the%20dimension%20of%20a%20model&amp;author=G%20Schwarz&amp;publication_year=1978&amp;pages=461-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref116">
<span class="label">116.</span><cite>Shaik T, Tao X, Li L, Higgins N, Gururajan R, Zhou X, et al. Clustered fedstack: Intermediate global models with Bayesian information criterion. Pattern Recognition Letters. 2024;177:121–7.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Pattern%20Recognition%20Letters&amp;title=Clustered%20fedstack:%20Intermediate%20global%20models%20with%20Bayesian%20information%20criterion&amp;author=T%20Shaik&amp;author=X%20Tao&amp;author=L%20Li&amp;author=N%20Higgins&amp;author=R%20Gururajan&amp;volume=177&amp;publication_year=2024&amp;pages=121-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref117">
<span class="label">117.</span><cite>Choiruddin A, Coeurjolly JF, Waagepetersen R. Information criteria for inhomogeneous spatial point processes. Australian &amp; New Zealand Journal of Statistics. 2021;63(1):119–43.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Australian%20&amp;%20New%20Zealand%20Journal%20of%20Statistics&amp;title=Information%20criteria%20for%20inhomogeneous%20spatial%20point%20processes&amp;author=A%20Choiruddin&amp;author=JF%20Coeurjolly&amp;author=R%20Waagepetersen&amp;volume=63&amp;issue=1&amp;publication_year=2021&amp;pages=119-43&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref118">
<span class="label">118.</span><cite>Punhani A, Faujdar N, Mishra KK, Subramanian M. Binning-based silhouette approach to find the optimal cluster using K-means. IEEE Access. 2022;10:115025–32.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Access&amp;title=Binning-based%20silhouette%20approach%20to%20find%20the%20optimal%20cluster%20using%20K-means&amp;author=A%20Punhani&amp;author=N%20Faujdar&amp;author=KK%20Mishra&amp;author=M%20Subramanian&amp;volume=10&amp;publication_year=2022&amp;pages=115025-32&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref119">
<span class="label">119.</span><cite>Subbalakshmi C, Krishna GR, Rao SK, Rao PV. A method to find optimum number of clusters based on fuzzy silhouette on dynamic data set. Procedia Computer Science. 2015;46:346–53.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Procedia%20Computer%20Science&amp;title=A%20method%20to%20find%20optimum%20number%20of%20clusters%20based%20on%20fuzzy%20silhouette%20on%20dynamic%20data%20set&amp;author=C%20Subbalakshmi&amp;author=GR%20Krishna&amp;author=SK%20Rao&amp;author=PV%20Rao&amp;volume=46&amp;publication_year=2015&amp;pages=346-53&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref120">
<span class="label">120.</span><cite>Shahapure KR, Nicholas C. Cluster quality analysis using silhouette score. In: 2020 IEEE 7th International Conference on Data Science and Advanced Analytics (DSAA). IEEE. 2020. 747–8.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Cluster%20quality%20analysis%20using%20silhouette%20score.%20In:%202020%20IEEE%207th%20International%20Conference%20on%20Data%20Science%20and%20Advanced%20Analytics%20(DSAA)&amp;author=KR%20Shahapure&amp;author=C%20Nicholas&amp;publication_year=2020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref121">
<span class="label">121.</span><cite>Chacón JE, Rastrojo AI. Minimum adjusted Rand index for two clusterings of a given size. Advances in Data Analysis and Classification. 2023;17(1):125–33.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Advances%20in%20Data%20Analysis%20and%20Classification&amp;title=Minimum%20adjusted%20Rand%20index%20for%20two%20clusterings%20of%20a%20given%20size&amp;author=JE%20Chac%C3%B3n&amp;author=AI%20Rastrojo&amp;volume=17&amp;issue=1&amp;publication_year=2023&amp;pages=125-33&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref122">
<span class="label">122.</span><cite>D’Ambrosio A, Amodio S, Iorio C, Pandolfo G, Siciliano R. Adjusted concordance index: an extension of the adjusted rand index to fuzzy partitions. Journal of Classification. 2021;38:112–28.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Classification&amp;title=Adjusted%20concordance%20index:%20an%20extension%20of%20the%20adjusted%20rand%20index%20to%20fuzzy%20partitions&amp;author=A%20D%E2%80%99Ambrosio&amp;author=S%20Amodio&amp;author=C%20Iorio&amp;author=G%20Pandolfo&amp;author=R%20Siciliano&amp;volume=38&amp;publication_year=2021&amp;pages=112-28&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref123">
<span class="label">123.</span><cite>Amodio S, D’Ambrosio A, Iorio C, Siciliano R. Adjusted concordance index, an extension of the adjusted rand index to fuzzy partitions. arXiv preprint. 2015. doi: 10.48550/arXiv.1509.00803</cite> [<a href="https://doi.org/10.48550/arXiv.1509.00803" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=arXiv%20preprint&amp;title=Adjusted%20concordance%20index,%20an%20extension%20of%20the%20adjusted%20rand%20index%20to%20fuzzy%20partitions&amp;author=S%20Amodio&amp;author=A%20D%E2%80%99Ambrosio&amp;author=C%20Iorio&amp;author=R%20Siciliano&amp;publication_year=2015&amp;doi=10.48550/arXiv.1509.00803&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref124">
<span class="label">124.</span><cite>de de Vargas RR, Bedregal BR. A way to obtain the quality of a partition by adjusted Rand index. In2013 2nd Workshop-School on Theoretical Computer Science. IEEE. 2013. 67–71.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=A%20way%20to%20obtain%20the%20quality%20of%20a%20partition%20by%20adjusted%20Rand%20index.%20In2013%202nd%20Workshop-School%20on%20Theoretical%20Computer%20Science&amp;author=RR%20de%20de%20Vargas&amp;author=BR%20Bedregal&amp;publication_year=2013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref125">
<span class="label">125.</span><cite>Steinley D. Properties of the Hubert-Arabie adjusted Rand index. Psychol Methods. 2004;9(3):386–96. doi: 10.1037/1082-989X.9.3.386

</cite> [<a href="https://doi.org/10.1037/1082-989X.9.3.386" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15355155/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Psychol%20Methods&amp;title=Properties%20of%20the%20Hubert-Arabie%20adjusted%20Rand%20index&amp;author=D%20Steinley&amp;volume=9&amp;issue=3&amp;publication_year=2004&amp;pages=386-96&amp;pmid=15355155&amp;doi=10.1037/1082-989X.9.3.386&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref126">
<span class="label">126.</span><cite>Martínez AM, Vitria J. Learning mixture models using a genetic version of the EM algorithm. Pattern Recognition Letters. 2000;21(8):759–69.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Pattern%20Recognition%20Letters&amp;title=Learning%20mixture%20models%20using%20a%20genetic%20version%20of%20the%20EM%20algorithm&amp;author=AM%20Mart%C3%ADnez&amp;author=J%20Vitria&amp;volume=21&amp;issue=8&amp;publication_year=2000&amp;pages=759-69&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref127">
<span class="label">127.</span><cite>Biernacki C, Celeux G, Govaert G. Choosing starting values for the EM algorithm for getting the highest likelihood in multivariate Gaussian mixture models. Computational Statistics &amp; Data Analysis. 2003;41(3–4):561–75.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Computational%20Statistics%20&amp;%20Data%20Analysis&amp;title=Choosing%20starting%20values%20for%20the%20EM%20algorithm%20for%20getting%20the%20highest%20likelihood%20in%20multivariate%20Gaussian%20mixture%20models&amp;author=C%20Biernacki&amp;author=G%20Celeux&amp;author=G%20Govaert&amp;volume=41&amp;publication_year=2003&amp;pages=561-75&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref128">
<span class="label">128.</span><cite>Reddy CK, Chiang H-D, Rajaratnam B. TRUST-TECH-based expectation maximization for learning finite mixture models. IEEE Trans Pattern Anal Mach Intell. 2008;30(7):1146–57. doi: 10.1109/TPAMI.2007.70775

</cite> [<a href="https://doi.org/10.1109/TPAMI.2007.70775" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18550899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Trans%20Pattern%20Anal%20Mach%20Intell&amp;title=TRUST-TECH-based%20expectation%20maximization%20for%20learning%20finite%20mixture%20models&amp;author=CK%20Reddy&amp;author=H-D%20Chiang&amp;author=B%20Rajaratnam&amp;volume=30&amp;issue=7&amp;publication_year=2008&amp;pages=1146-57&amp;pmid=18550899&amp;doi=10.1109/TPAMI.2007.70775&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref129">
<span class="label">129.</span><cite>Richardson S, Green PJ. On Bayesian analysis of mixtures with an unknown number of components (with discussion). Journal of the Royal Statistical Society Series B: Statistical Methodology. 1997;59(4):731–92.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Royal%20Statistical%20Society%20Series%20B:%20Statistical%20Methodology&amp;title=On%20Bayesian%20analysis%20of%20mixtures%20with%20an%20unknown%20number%20of%20components%20(with%20discussion)&amp;author=S%20Richardson&amp;author=PJ%20Green&amp;volume=59&amp;issue=4&amp;publication_year=1997&amp;pages=731-92&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref130">
<span class="label">130.</span><cite>Wang Y, Xiang Y, Zhang J, Zhou W, Xie B. Internet traffic clustering with side information. Journal of Computer and System Sciences. 2014;80(5):1021–36.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Computer%20and%20System%20Sciences&amp;title=Internet%20traffic%20clustering%20with%20side%20information&amp;author=Y%20Wang&amp;author=Y%20Xiang&amp;author=J%20Zhang&amp;author=W%20Zhou&amp;author=B%20Xie&amp;volume=80&amp;issue=5&amp;publication_year=2014&amp;pages=1021-36&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref131">
<span class="label">131.</span><cite>Fan H, Bennetts VH, Schaffernicht E, Lilienthal AJ. A cluster analysis approach based on exploiting density peaks for gas discrimination with electronic noses in open environments. Sensors and Actuators B: Chemical. 2018;259:183–203.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Sensors%20and%20Actuators%20B:%20Chemical&amp;title=A%20cluster%20analysis%20approach%20based%20on%20exploiting%20density%20peaks%20for%20gas%20discrimination%20with%20electronic%20noses%20in%20open%20environments&amp;author=H%20Fan&amp;author=VH%20Bennetts&amp;author=E%20Schaffernicht&amp;author=AJ%20Lilienthal&amp;volume=259&amp;publication_year=2018&amp;pages=183-203&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref132">
<span class="label">132.</span><cite>Zhao J. Efficient model selection for mixtures of probabilistic PCA via hierarchical BIC. IEEE Trans Cybern. 2014;44(10):1871–83. doi: 10.1109/TCYB.2014.2298401

</cite> [<a href="https://doi.org/10.1109/TCYB.2014.2298401" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25222728/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Trans%20Cybern&amp;title=Efficient%20model%20selection%20for%20mixtures%20of%20probabilistic%20PCA%20via%20hierarchical%20BIC&amp;author=J%20Zhao&amp;volume=44&amp;issue=10&amp;publication_year=2014&amp;pages=1871-83&amp;pmid=25222728&amp;doi=10.1109/TCYB.2014.2298401&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref133">
<span class="label">133.</span><cite>PM, Sinaga B, Manurung J, Al Hashim SA. Best cluster optimization with combination of K-means algorithm and elbow method towards rice production status determination. International Journal of Artificial Intelligence Research. 2021;5(1):102–10.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Artificial%20Intelligence%20Research&amp;title=Best%20cluster%20optimization%20with%20combination%20of%20K-means%20algorithm%20and%20elbow%20method%20towards%20rice%20production%20status%20determination&amp;author=B%20Sinaga&amp;author=J%20Manurung&amp;author=SA%20Al%20Hashim&amp;volume=5&amp;issue=1&amp;publication_year=2021&amp;pages=102-10&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0330421.ref134">
<span class="label">134.</span><cite>Onumanyi AJ, Molokomme DN, Isaac SJ, Abu-Mahfouz AM. AutoElbow: An automatic elbow detection method for estimating the number of clusters in a dataset. Applied Sciences. 2022;12(15):7515.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Applied%20Sciences&amp;title=AutoElbow:%20An%20automatic%20elbow%20detection%20method%20for%20estimating%20the%20number%20of%20clusters%20in%20a%20dataset&amp;author=AJ%20Onumanyi&amp;author=DN%20Molokomme&amp;author=SJ%20Isaac&amp;author=AM%20Abu-Mahfouz&amp;volume=12&amp;issue=15&amp;publication_year=2022&amp;pages=7515&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section></section><article class="sub-article" id="pone.0330421.r001"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0330421.r001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330421.r001</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 0</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ishengoma%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Fredrick Ishengoma</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Fredrick Ishengoma</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ishengoma%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fredrick Ishengoma</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.c" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.c" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.c" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Fredrick Ishengoma</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.c" class="d-panel p" style="display: none">
<div>© 2025 Fredrick Ishengoma</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>22 Nov 2024</em>
</p>
<p>Dear Dr. Ashraf,</p>
<p>Thank you for submitting your manuscript to PLOS ONE. After careful consideration, we feel that it has merit but does not fully meet PLOS ONE’s publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the points raised during the review process.</p>
<p>Please submit your revised manuscript by Jan 06 2025 11:59PM. If you will need more time than this to complete your revisions, please reply to this message or contact the journal office at <span>plosone@plos.org</span> . When you're ready to submit your revision, log on to <a href="https://www.editorialmanager.com/pone/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.editorialmanager.com/pone/</a> and select the 'Submissions Needing Revision' folder to locate your manuscript file.</p>
<ul class="list" style="list-style-type:disc">
<li><p>A rebuttal letter that responds to each point raised by the academic editor and reviewer(s). You should upload this letter as a separate file labeled 'Response to Reviewers'.</p></li>
<li><p>A marked-up copy of your manuscript that highlights changes made to the original version. You should upload this as a separate file labeled 'Revised Manuscript with Track Changes'.</p></li>
<li><p>An unmarked version of your revised paper without tracked changes. You should upload this as a separate file labeled 'Manuscript'.</p></li>
</ul>
<p>If you would like to make changes to your financial disclosure, please include your updated statement in your cover letter. Guidelines for resubmitting your figure files are available below the reviewer comments at the end of this letter.</p>
<p>If applicable, we recommend that you deposit your laboratory protocols in protocols.io to enhance the reproducibility of your results. Protocols.io assigns your protocol its own identifier (DOI) so that it can be cited independently in the future. For instructions see: <a href="https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols</a> . Additionally, PLOS ONE offers an option for publishing peer-reviewed Lab Protocol articles, which describe protocols hosted on protocols.io. Read more information on sharing protocols at <a href="https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols</a> .</p>
<p>We look forward to receiving your revised manuscript.</p>
<p>Kind regards,</p>
<p>Fredrick Romanus Ishengoma</p>
<p>Academic Editor</p>
<p>PLOS ONE</p>
<p>Journal Requirements:</p>
<p>When submitting your revision, we need you to address these additional requirements.</p>
<p>1. Please ensure that your manuscript meets PLOS ONE's style requirements, including those for file naming. The PLOS ONE style templates can be found at </p>
<p><a href="https://journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/file?id=wjVg/PLOSOne_formatting_sample_main_body.pdf</a> and </p>
<p>
<a href="https://journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/file?id=ba62/PLOSOne_formatting_sample_title_authors_affiliations.pdf</a>
</p>
<p>2. Please note that PLOS ONE has specific guidelines on code sharing for submissions in which author-generated code underpins the findings in the manuscript. In these cases, we expect all author-generated code to be made available without restrictions upon publication of the work. Please review our guidelines at <a href="https://journals.plos.org/plosone/s/materials-and-software-sharing#loc-sharing-code" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/materials-and-software-sharing#loc-sharing-code</a> and ensure that your code is shared in a way that follows best practice and facilitates reproducibility and reuse.</p>
<p>3. We note that your Data Availability Statement is currently as follows: All relevant data are within the manuscript and its Supporting Information files.</p>
<p>Please confirm at this time whether or not your submission contains all raw data required to replicate the results of your study. Authors must share the “minimal data set” for their submission. PLOS defines the minimal data set to consist of the data required to replicate all study findings reported in the article, as well as related metadata and methods (<a href="https://journals.plos.org/plosone/s/data-availability#loc-minimal-data-set-definition" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/data-availability#loc-minimal-data-set-definition</a>).</p>
<p>For example, authors should submit the following data:</p>
<p>- The values behind the means, standard deviations and other measures reported;</p>
<p>- The values used to build graphs;</p>
<p>- The points extracted from images for analysis.</p>
<p>Authors do not need to submit their entire data set if only a portion of the data was used in the reported study.</p>
<p>If your submission does not contain these data, please either upload them as Supporting Information files or deposit them to a stable, public repository and provide us with the relevant URLs, DOIs, or accession numbers. For a list of recommended repositories, please see <a href="https://journals.plos.org/plosone/s/recommended-repositories" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/recommended-repositories</a>.</p>
<p>If there are ethical or legal restrictions on sharing a de-identified data set, please explain them in detail (e.g., data contain potentially sensitive information, data are owned by a third-party organization, etc.) and who has imposed them (e.g., an ethics committee). Please also provide contact information for a data access committee, ethics committee, or other institutional body to which data requests may be sent. If data are owned by a third party, please indicate how others may request data access.</p>
<p>[Note: HTML markup is below. Please do not edit.]</p>
<p>Reviewers' comments:</p>
<p>Reviewer's Responses to Questions</p>
<p>
<strong>Comments to the Author</strong>
</p>
<p>1. Is the manuscript technically sound, and do the data support the conclusions?</p>
<p>Reviewer #1: Partly</p>
<p>Reviewer #2: Partly</p>
<p>**********</p>
<p>2. Has the statistical analysis been performed appropriately and rigorously? --&gt;?&gt;</p>
<p>Reviewer #1: No</p>
<p>Reviewer #2: No</p>
<p>**********</p>
<p>3. Have the authors made all data underlying the findings in their manuscript fully available??&gt;</p>
<p>The <a href="http://www.plosone.org/static/policies.action#sharing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PLOS Data policy</a></p>
<p>Reviewer #1: No</p>
<p>Reviewer #2: No</p>
<p>**********</p>
<p>4. Is the manuscript presented in an intelligible fashion and written in standard English??&gt;</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>Reviewer #1: This article presents work on unsupervised fake news detection.</p>
<p>I find that the work presented in this manuscript is not complete, the authors barely scratched the surface, and this work falls short of being ready for publication.</p>
<p>The methods used are not novel at all.</p>
<p>1. I find that the authors have ignored a lot of related literature on the subject that proposes new ways of feature extraction.</p>
<p>Please read, present, and discuss at least related work regarding the following aspects (please respond individually):</p>
<p>- the use effect of word embeddings to encode fake news [1]</p>
<p>- use of transformer embeddings to encode fake news [2]</p>
<p>- how document embeddings are used to encode fake news [3]</p>
<p>- real-time architectures for fake news and harmful content detection and mitigation [4] (e.g., ContCommRTD <a href="https://doi.org/10.1109/TKDE.2024.3417232" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1109/TKDE.2024.3417232</a>) or [5] (e.g. StopHC <a href="https://arxiv.org/abs/2411.06138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://arxiv.org/abs/2411.06138(</a></p>
<p>- using social network features for fake news detection [6] (e.g., <a href="https://doi.org/10.1016/j.knosys.2024.111715" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.knosys.2024.111715</a>)</p>
<p>- the effect of multilingual transformers on fake news detection [7] (e.g., <a href="http://ceur-ws.org/Vol-3180/paper-61.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://ceur-ws.org/Vol-3180/paper-61.pdf</a>)</p>
<p>2. What happens after detection? There are many solutions used for this proposed in the literature that also propose models for identifying fake news and mitigation.</p>
<p>Some related work that should be discussed (please respond individually):</p>
<p>- use of community detection for fake news network immunization [8] (e.g., <a href="https://doi.org/10.1016/j.jestch.2024.101728" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.jestch.2024.101728</a>)</p>
<p>- use of weighted directed spanning trees for fake news detection and mitigation in real time [9] (e.g., <a href="https://doi.org/10.1109/ACCESS.2023.3331220" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1109/ACCESS.2023.3331220</a>)</p>
<p>- use of budget-based immunization algorithms to stop fake news from spreading [10] (e.g., <a href="https://doi.org/10.1145/3459637.3482481" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1145/3459637.3482481</a>)</p>
<p>3. There are many methods for document similarity detection, not just clustering. The manuscript must be improved and it must discuss the following (please respond individually):</p>
<p>a. Document similarity using clustering for dimensionality reduction [11] (e.g., <a href="https://doi.org/10.1109/AQTR49680.2020.9129967" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1109/AQTR49680.2020.9129967</a>)</p>
<p>b. Novel novel density-based clustering algorithms that can be used for topic modeling that should at least be mentioned, e.g., DenLAC [12], some even based on DBScan algorithms [13], for example, QuickDBScan and KDTreeDBSCAN [14].</p>
<p>c. Community Detection-based methods for document similarity [15].</p>
<p>d. Similarity based on context-aware document clustering [16].</p>
<p>e. How are the metrics for encoding the textual data selected? Please compare with other methods, you can start from [17] (e.g., <a href="http://doi.org/10.1109/SYNASC.2016.055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://doi.org/10.1109/SYNASC.2016.055</a>)</p>
<p>4. An in-depth exploratory data analysis is required.</p>
<p>5. The experiments should be improved (please respond individually to each point in the answer to reviewers)</p>
<p>a. There is no hyperparameter tuning for the proposed model.</p>
<p>b. There is no time performance evaluation.</p>
<p>c. The experiments are done on relatively small datasets. How does the proposed model perform on a larger dataset?</p>
<p>d. There is no comparison with state-of-the-art models.</p>
<p>6. The conclusions are very shallow.</p>
<p>7. What are the limitations of this study?</p>
<p>8. For reproducibility purposes, the authors should make the code publicly available. Without publicly available source code, models, and datasets, a shadow of doubt can fall on the results of this work. How can anyone check if these results are real or made up without the code, models, and datasets?</p>
<p>9. Please proofread and spell-check the article before resubmitting.</p>
<p>[1] <a href="https://scholar.google.com/scholar?q=misinformation+detection+word+embeddings" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=misinformation+detection+word+embeddings</a></p>
<p>[2] <a href="https://scholar.google.com/scholar?q=transformers+misinformation" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=transformers+misinformation</a></p>
<p>[3] <a href="https://scholar.google.com/scholar?q=fake+news+document+embeddings" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=fake+news+document+embeddings</a></p>
<p>[4] <a href="https://scholar.google.com/scholar?q=content-based+misinformation+real-time" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=content-based+misinformation+real-time</a></p>
<p>[5] <a href="https://scholar.google.com/scholar?q=content+detection+mitigation+architecture+social+media" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=content+detection+mitigation+architecture+social+media</a></p>
<p>[6] <a href="https://scholar.google.com/scholar?q=deep+neural+network+ensemble+social+context+fake+news+detection" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=deep+neural+network+ensemble+social+context+fake+news+detection</a></p>
<p>[7] <a href="https://scholar.google.com/scholar?q=fake+news+detection+sentence+transformer" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=fake+news+detection+sentence+transformer</a></p>
<p>[8] <a href="https://scholar.google.com/scholar?q=community+algorithm+network+immunization+fake+news" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=community+algorithm+network+immunization+fake+news</a></p>
<p>[9] <a href="https://scholar.google.com/scholar?q=tree+algorithm+real-time+fake+news+mitigation+social+media" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=tree+algorithm+real-time+fake+news+mitigation+social+media</a></p>
<p>[10] <a href="https://scholar.google.com/scholar?q=social+network+immunization+harmful+speech" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=social+network+immunization+harmful+speech</a></p>
<p>[11] <a href="https://scholar.google.com/scholar?q=document+vector+model+clustering+dimensionality+reduction" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=document+vector+model+clustering+dimensionality+reduction</a></p>
<p>[12] <a href="https://scholar.google.com/scholar?q=DenLAC" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=DenLAC</a></p>
<p>[13] <a href="https://scholar.google.com/scholar?q=density+text+clustering+document+embeddings" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=density+text+clustering+document+embeddings</a></p>
<p>[14] <a href="https://scholar.google.com/scholar?q=evaluation+DBSCAN+similarity+join" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=evaluation+DBSCAN+similarity+join</a></p>
<p>[15] <a href="https://scholar.google.com/scholar?q=community+detection+document+clustering" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=community+detection+document+clustering</a></p>
<p>[16] <a href="https://scholar.google.com/scholar?q=topic+modeling+contextual+cues" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=topic+modeling+contextual+cues</a></p>
<p>[17] <a href="https://scholar.google.com/scholar?q=comparing+term+weighting+topic+modeling" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=comparing+term+weighting+topic+modeling</a></p>
<p>Reviewer #2: The authors propose to detect fake news using an unsupervised detection approach.</p>
<p>In its current state, the manuscript is not ready for publication and it requires improvements.</p>
<p>The proposed method combines existing methods.</p>
<p>1. Why choose TF-IDF for encoding the documents?</p>
<p>Why not word embeddings [1], transformers [2], or document embeddings [3]?</p>
<p>2. Does the method work in real-time? Please compare with existing methods and architectures that work in real-time [4] or [5].</p>
<p>3. What happens when encoding more than the textual content? For example, if using social network features for fake news detection [6]</p>
<p>4. How do you deal with multilingual textual data? Compared with existing methods [7].</p>
<p>5. What's next after detection? In the current literature, many works propose models for fake news detection and mitigation, e.g., community-based [8], trees-based [9], or pre-emptive [10] algorithms. The authors must discuss some direction at least in future work.</p>
<p>6. Why use only classic clustering methods for your approach? There are many clustering methods that employ dimensionality reduction [11]. Please improve the manuscript and compare your methods with these methods.</p>
<p>7. On the topic of clustering, why not use density-based algorithms [12] (e.g., DenLAC) or DBScan algorithms [13], for example, QuickDBScan and KDTreeDBSCAN [14]. Please improve the manuscript and compare your methods with these methods.</p>
<p>8. Community Detection-based methods for document similarity [15] are also used for clustering documents. Please improve the manuscript and compare your methods with these methods.</p>
<p>9. Why not use topic modeling for clustering documents? There are many context-aware document topic modeling algorithms [16] used in the literature for document clustering. Please improve the manuscript and compare your methods with these methods.</p>
<p>10. On what basis was TF-IDF selected for encoding the textual data? There are many works that discuss different weighting schemes for document clustering [17]. Please improve the manuscript and compare your methods with these methods.</p>
<p>11. The datasets are not properly presented. Please improve the manuscript with an exploratory data analysis.</p>
<p>12. The experiments are missing a hyperparameter tuning analysis.</p>
<p>13. The experiments are missing a time performance evaluation.</p>
<p>14. The size of the datasets is limited, how does the proposed model perform on a large dataset?</p>
<p>15. Improve the manuscript by comparing your method with existing state-of-the-art models.</p>
<p>16. The conclusions must be improved.</p>
<p>17. A limitation section is missing. What are the limitations of your methods?</p>
<p>18. The authors should make the code publicly available for reproducibility purposes.</p>
<p>19. Before resubmitting, I recommend proofreading and spell-checking the article.</p>
<p>[1] <a href="https://scholar.google.com/scholar?q=misinformation+detection+word+embeddings" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=misinformation+detection+word+embeddings</a></p>
<p>[2] <a href="https://scholar.google.com/scholar?q=transformers+misinformation" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=transformers+misinformation</a></p>
<p>[3] <a href="https://scholar.google.com/scholar?q=fake+news+document+embeddings" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=fake+news+document+embeddings</a></p>
<p>[4] <a href="https://scholar.google.com/scholar?q=content-based+misinformation+real-time" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=content-based+misinformation+real-time</a></p>
<p>[5] <a href="https://scholar.google.com/scholar?q=content+detection+mitigation+architecture+social+media" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=content+detection+mitigation+architecture+social+media</a></p>
<p>[6] <a href="https://scholar.google.com/scholar?q=deep+neural+network+ensemble+social+context+fake+news+detection" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=deep+neural+network+ensemble+social+context+fake+news+detection</a></p>
<p>[7] <a href="https://scholar.google.com/scholar?q=fake+news+detection+sentence+transformer" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=fake+news+detection+sentence+transformer</a></p>
<p>[8] <a href="https://scholar.google.com/scholar?q=community+algorithm+network+immunization+fake+news" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=community+algorithm+network+immunization+fake+news</a></p>
<p>[9] <a href="https://scholar.google.com/scholar?q=tree+algorithm+real-time+fake+news+mitigation+social+media" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=tree+algorithm+real-time+fake+news+mitigation+social+media</a></p>
<p>[10] <a href="https://scholar.google.com/scholar?q=social+network+immunization+harmful+speech" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=social+network+immunization+harmful+speech</a></p>
<p>[11] <a href="https://scholar.google.com/scholar?q=document+vector+model+clustering+dimensionality+reduction" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=document+vector+model+clustering+dimensionality+reduction</a></p>
<p>[12] <a href="https://scholar.google.com/scholar?q=DenLAC" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=DenLAC</a></p>
<p>[13] <a href="https://scholar.google.com/scholar?q=density+text+clustering+document+embeddings" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=density+text+clustering+document+embeddings</a></p>
<p>[14] <a href="https://scholar.google.com/scholar?q=evaluation+DBSCAN+similarity+join" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=evaluation+DBSCAN+similarity+join</a></p>
<p>[15] <a href="https://scholar.google.com/scholar?q=community+detection+document+clustering" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=community+detection+document+clustering</a></p>
<p>[16] <a href="https://scholar.google.com/scholar?q=topic+modeling+contextual+cues" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=topic+modeling+contextual+cues</a></p>
<p>[17] <a href="https://scholar.google.com/scholar?q=comparing+term+weighting+topic+modeling" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scholar.google.com/scholar?q=comparing+term+weighting+topic+modeling</a></p>
<p>**********</p>
<p><a href="https://journals.plos.org/plosone/s/editorial-and-peer-review-process#loc-peer-review-history" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">what does this mean?</a> ). If published, this will include your full peer review and any attached files.</p>
<p>If you choose “no”, your identity will remain anonymous but your review may still be made public.</p>
<p><strong>Do you want your identity to be public for this peer review?</strong> For information about this choice, including consent withdrawal, please see our <a href="https://www.plos.org/privacy-policy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Privacy Policy</a></p>
<p>Reviewer #1: No</p>
<p>Reviewer #2: No</p>
<p>**********</p>
<p>[NOTE: If reviewer comments were submitted as an attachment file, they will be attached to this email and accessible via the submission site. Please log into your account, locate the manuscript record, and check for the action link "View Attachments". If this link does not appear, there are no attachment files.]</p>
<p>While revising your submission, please upload your figure files to the Preflight Analysis and Conversion Engine (PACE) digital diagnostic tool, <a href="https://pacev2.apexcovantage.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pacev2.apexcovantage.com/</a> . PACE helps ensure that figures meet PLOS requirements. To use PACE, you must first register as a user. Registration is free. Then, login and navigate to the UPLOAD tab, where you will find detailed instructions on how to use the tool. If you encounter any issues or have any questions when using PACE, please email PLOS at <span>figures@plos.org</span> . Please note that Supporting Information files do not need this step.</p></section></article><article class="sub-article" id="pone.0330421.r002"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. 2025 Aug 18;20(8):e0330421. doi: <a href="https://doi.org/10.1371/journal.pone.0330421.r002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330421.r002</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Author response to Decision Letter 1</h1></hgroup><ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="anp_a.d" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.d" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="anp_a.d" class="d-panel p" style="display: none"><div class="notes p"><section id="historyfront-stub2" class="history"><p>Collection date 2025.</p></section></div></div>
<div id="clp_a.d" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>5 Dec 2024</em>
</p>
<p>Original Manuscript ID: PONE-D-24-50144</p>
<p>Original Article Title: “Unsupervised fake news detection on social media using hybrid gaussian mixture model”</p>
<p>To: PLOS ONE Editor</p>
<p>Re: Response to reviewers</p>
<p>Dear Editor,</p>
<p>Thank you for allowing a resubmission of our manuscript, with an opportunity to address the reviewers’ comments.</p>
<p>We are uploading (a) our point-by-point response to the comments (below) (response to reviewers, under “Response-to-Reviewers Files”), (b) an updated manuscript with yellow highlighting indicating changes (as “Revised Article with Changes Highlighted”), and (c) a clean updated manuscript without highlights (“Manuscript”).</p>
<p>Best regards,</p>
<p>Reviewer 1, Concern # 1. (Please read, present, and discuss at least related work regarding the following aspects (please respond individually):- the use effect of word embeddings to encode fake news [1]- use of transformer embeddings to encode fake news [2]- how document embeddings are used to encode fake news [3]- real-time architectures for fake news and harmful content detection and mitigation [4] (e.g., ContCommRTD <a href="https://doi.org/10.1109/TKDE.2024.3417232" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1109/TKDE.2024.3417232</a>) or [5] (e.g. StopHC <a href="https://arxiv.org/abs/2411.06138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://arxiv.org/abs/2411.06138(-</a> using social network features for fake news detection [6] (e.g., <a href="https://doi.org/10.1016/j.knosys.2024.111715" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.knosys.2024.111715)-</a> the effect of multilingual transformers on fake news detection [7] (e.g., <a href="http://ceur-ws.org/Vol-3180/paper-61.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://ceur-ws.org/Vol-3180/paper-61.pdf)):</a></p>
<p>Author response: We appreciate the insightful feedback from the reviewer specifically focusing on different type of embedding techniques and their role in our proposed methodology. The point-by-point response to the comments is given below:</p>
<p>The use effect of word embeddings to encode fake news</p>
<p>The use of word embeddings like Word2Vec and GloVe for fake news detection has been extensively studied due to their ability to capture semantic relationships between words. However, there are specific reasons why these embeddings were not included in the proposed research:</p>
<p>Word embeddings generate static word vectors, which fail to account for contextual nuances, such as changes in meaning based on surrounding text [1, 2]. While effective in supervised learning tasks, these are less suited for unsupervised methods like clustering, where the focus is on structural and statistical characteristics of the data. Approaches like TF-IDF provide term-specific weights that are more interpretable and better aligned with the objectives of the proposed research based on clustering [3]. Furthermore, word embeddings typically require additional preprocessing and fine-tuning to adapt them for domain-specific fake news detection. This process adds computational complexity and might not yield significant improvements for unsupervised learning methods [4]. The proposed research incorporated TF-IDF, being simpler and domain-agnostic, offers a scalable solution without the overhead of embedding model training or fine-tuning [5].</p>
<p>Use of transformer embeddings to encode fake news</p>
<p>Transformer embeddings are typically leveraged in supervised learning settings or fine-tuned for specific tasks. The proposed research incorporated unsupervised clustering techniques, where traditional representations like TF-IDF are often more effective and interpretable for capturing textual patterns [6]. Another key challenge associated with transformer-based models is their lack of transparency and interpretability. These models generate high-dimensional embeddings that can be difficult to analyze and interpret in the context of clustering, where understanding the relationship between data points is critical [7].</p>
<p>How document embeddings are used to encode fake news</p>
<p>Document embeddings are a powerful tool for encoding fake news by transforming textual data into numerical representations that capture the semantic and contextual essence of entire documents. Unlike traditional word-level embeddings, document embeddings aggregate information across sentences or paragraphs, making them particularly effective for tasks requiring an understanding of broader context.</p>
<p>However, the primary goal of this research is to analyze surface-level linguistic patterns for fake news detection. Therefore, traditional methods like TF-IDF were sufficient to capture the necessary textual features without the additional complexity of embedding-based approaches.</p>
<p>Real-time architectures for fake news and harmful content detection and mitigation</p>
<p>The proposed research primarily investigates unsupervised learning techniques, specifically GMM-GCO, a hybrid model to effectively identify fake news. Real-time architectures often involve supervised or semi-supervised methods, leveraging labeled data streams for real-time predictions. Integrating these techniques could deviate from the core aim of developing robust unsupervised methods for textual clustering. Furthermore, real-time feedback systems, such as those used in platforms like ContCommRTD or StopHC could demand infrastructure beyond the scope of the proposed research to implement and test these architectures</p>
<p>Using social network features for fake news detection</p>
<p>Real-time systems incorporate interdisciplinary components, such as user behavior analysis, network traffic monitoring, and large-scale content moderation, which is beyond the scope of our research. As the primary goal of this research is to analyze surface-level linguistic patterns for fake news detection.</p>
<p>The effect of multilingual transformers on fake news detection</p>
<p>Our study primarily focused on English-language datasets and align with several existing benchmark corpora in fake news detection studies. Since multilingual transformers like mBERT or XLM-R are designed to handle diverse languages, their inclusion is not necessary given the considered dataset's linguistic homogeneity. Multilingual models like mBERT and XLM-R have shown success in cross-lingual natural language understanding tasks, but their effectiveness in unsupervised or clustering techniques remains less explored [8, 9].</p>
<p>Author action:</p>
<p>We trust that these justifications align with the reviewer’s expectations and justify the exclusion of word or transformer embeddings in favor of simpler, domain-independent text representation techniques better aligned with the objectives of our proposed research.</p>
<p>References:</p>
<p>1. Mikolov T, Sutskever I, Chen K, Corrado GS, Dean J. Distributed representations of words and phrases and their compositionality. Advances in neural information processing systems. 2013;26.</p>
<p>2. Sun C, Yang Z, Luo L, Wang L, Zhang Y, Lin H, Wang J. A deep learning approach with deep contextualized word representations for chemical–protein interaction extraction from biomedical literature. IEEE Access. 2019 Oct 21;7:151034-46.</p>
<p>3. Sparck Jones K. A statistical interpretation of term specificity and its application in retrieval. Journal of documentation. 1972 Jan 1;28(1):11-21.</p>
<p>4. Farhangian F, Cruz RM, Cavalcanti GD. Fake news detection: Taxonomy and comparative study. Information Fusion. 2024 Mar 1;103:102140.</p>
<p>5. Pennington J, Socher R, Manning CD. Glove: Global vectors for word representation. InProceedings of the 2014 conference on empirical methods in natural language processing (EMNLP) 2014 Oct (pp. 1532-1543).</p>
<p>6. Park J, Park C, Kim J, Cho M, Park S. ADC: Advanced document clustering using contextualized representations. Expert Systems with Applications. 2019 Dec 15;137:157-66.</p>
<p>7. Li Y, Wang J, Dai X, Wang L, Yeh CC, Zheng Y, Zhang W, Ma KL. How does attention work in vision transformers? A visual analytics attempt. IEEE transactions on visualization and computer graphics. 2023 Mar 27;29(6):2888-900.</p>
<p>8. Conneau A, Baevski A, Collobert R, Mohamed A, Auli M. Unsupervised cross-lingual representation learning for speech recognition. arXiv preprint arXiv:2006.13979. 2020 Jun 24.</p>
<p>9. Pires T. How multilingual is multilingual BERT. arXiv preprint arXiv:1906.01502. 2019.</p>
<p>Reviewer#1, Concern # 2. (What happens after detection? There are many solutions used for this proposed in the literature that also propose models for identifying fake news and mitigation.</p>
<p>Some related work that should be discussed (please respond individually):</p>
<p>- use of community detection for fake news network immunization [8] (e.g., <a href="https://doi.org/10.1016/j.jestch.2024.101728" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1016/j.jestch.2024.101728</a>)</p>
<p>- use of weighted directed spanning trees for fake news detection and mitigation in real time [9] (e.g., <a href="https://doi.org/10.1109/ACCESS.2023.3331220" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1109/ACCESS.2023.3331220</a>)</p>
<p>- use of budget-based immunization algorithms to stop fake news from spreading [10] (e.g., <a href="https://doi.org/10.1145/3459637.3482481" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1145/3459637.3482481).):</a></p>
<p>Author response: We sincerely appreciate the valuable feedback provided by the reviewer, a point by point response for each comment is given below:</p>
<p>Use of community detection for fake news network immunization</p>
<p>The proposed research primarily focuses on unsupervised clustering techniques and hybrid GMM methods for detecting fake news based on linguistic patterns in textual data. Community detection in fake news network immunization is more relevant to network-based methods, where relationships between social media accounts or information dissemination pathways are analyzed. This network-based focus falls outside the linguistic and clustering scope of your study. Community detection approaches rely on graph-based representations of social networks, where nodes represent users or sources and edges represent interactions or information flows. In contrast, our study employs textual data and uses TF-IDF vectorization combined with Gaussian Mixture Models (GMM) and Group Counseling Optimization (GCO). The lack of graph-structured data in the proposed research makes the integration of community detection techniques unsuitable.</p>
<p>Community detection frameworks often propose mitigation strategies like node immunization to curb the spread of fake news by identifying and neutralizing influential nodes. While valuable, such approaches require a detailed understanding of network dynamics, user influence, and propagation pathways, which is out of the scope of our research.</p>
<p>Use of weighted directed spanning trees for fake news detection and mitigation in real time</p>
<p>Our research primarily emphasizes the detection of fake news using unsupervised clustering techniques such as GMM, K-means, K-medoid and GCO-GMM, focusing on linguistic patterns in textual data. The proposed method focused on transforming text into numerical features (e.g., TF-IDF) and applying clustering algorithms to identify patterns. In contrast, approaches using weighted directed spanning tree (WDST) often operate on graph-based representations of information spread, focusing on the propagation of fake news across networks, which is beyond the scope of the proposed research.</p>
<p>Furthermore, our dataset does not include interaction or network related information (such as user engagement data, timestamps, or propagation paths) required for weighted directed spanning trees methods. The absence of such data makes it impractical to integrate WDST based methodologies into in the proposed research.</p>
<p>Use of budget-based immunization algorithms to stop fake news from spreading</p>
<p>Our proposed research is fundamentally designed to address fake news detection through unsupervised clustering techniques, particularly focusing on linguistic and textual patterns in news data. Budget-based immunization algorithms, which are primarily used in network-based propagation control, fall outside the scope of the proposed research. These algorithms focus on identifying key nodes (users or sources) in a social network to block or mitigate the spread of misinformation, requiring a network representation and interaction data.</p>
<p>Furthermore, our dataset does not include interaction or network related information (such as user engagement data, timestamps, or propagation paths) required for weighted directed spanning trees methods. The absence of such data makes it impractical to integrate WDST based methodologies in the proposed research.</p>
<p>Author action: We sincerely hope that these clarifications will meet the reviewers' expectations and effectively address their concerns.</p>
<p>Reviewer#2, Concern # 3. (There are many methods for document similarity detection, not just clustering. The manuscript must be improved and it must discuss the following (please respond individually):</p>
<p>a. Document similarity using clustering for dimensionality reduction [11] (e.g., <a href="https://doi.org/10.1109/AQTR49680.2020.9129967" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1109/AQTR49680.2020.9129967</a>)</p>
<p>b. Novel novel density-based clustering algorithms that can be used for topic modeling that should at least be mentioned, e.g., DenLAC [12], some even based on DBScan algorithms [13], for example, QuickDBScan and KDTreeDBSCAN [14].</p>
<p>c. Community Detection-based methods for document similarity [15].</p>
<p>d. Similarity based on context-aware document clustering [16].</p>
<p>e. How are the metrics for encoding the textual data selected? Please compare with other methods, you can start from [17] (e.g., <a href="http://doi.org/10.1109/SYNASC.2016.055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://doi.org/10.1109/SYNASC.2016.055</a>)).</p>
<p>a. Document similarity using clustering for dimensionality reduction</p>
<p>The primary objective of the proposed research is to optimize GMM parameters using GCO for improved clustering results and efficiently detection of fake news. Clustering for dimensionality reduction, while valuable, does not directly contribute to the parameter optimization goals.</p>
<p>Furthermore the proposed research already employs TF-IDF and PCA for feature extraction and dimensionality reduction (Please see the highlighted content under 2. 1. 8 heading on page 10 and 11). These techniques have strong theoretical and empirical implications in fake news detection, and introducing clustering for dimensionality reduction might not add substantial value over the current methods.</p>
<p>The focus on fake news detection through clustering assumes a direct clustering algorithm application. Clustering for dimensionality reduction serves a more generalized purpose and could dilute the research scope.</p>
<p>b. Novel density-based clustering algorithms that can be used for topic modeling that should at least be mentioned, e.g., DenLAC [12], some even based on DBScan algorithms [13], for example, QuickDBScan and KDTreeDBSCAN [14].</p>
<p>Our prior experience with DBSCAN yielded unsatisfactory results in terms of clustering accuracy for fake news detection. Although these newer density-based algorithms (QuickDBScan, KDTreeDBSCAN and DenLAC) may offer improvements, the reason to exclude them was likely based on the failure of DBSCAN to meet the requirements for our specific dataset and the added complexity these algorithms could introduce without guaranteed improvements.</p>
<p>The primary focus of the proposed research is on optimizing the parameters of the Gaussian Mixture Model (GMM) using the Group Counseling Optimization (GCO) method. As GMM offers a more probabilistic approach to clustering, that has the potential to capture the uncertainty of fake news data more effectively, it was deemed more suitable for our goals than attempting to integrate another clustering method that may require additional optimization efforts.</p>
<p>c. Community Detection-based methods for document similarity</p>
<p>The proposed research prioritizes optimizing GMM for clustering due to its probabilistic nature. While community detection offers graph-based insights, it differs significantly in its underlying assumptions and might not align directly with our optimization framework.</p>
<p>Furthermore, our dataset does not include interaction or network related information (such as user engagement data, timestamps, or propagation paths) required for graph-based methods. The absence of such data makes it impractical to integrate graph based methodologies in the proposed research.</p>
<p>d. Similarity based on context-aware document clustering</p>
<p>The use of word embeddings like Word2Vec and GloVe for fake news detection has been extensively studied due to their ability to capture semantic relationships between words. However, there are specific reasons why these embeddings were not included in the proposed research:</p>
<p>Word embeddings generate static word vectors, which fail to account for contextual nuances, such as changes in meaning based on surrounding text [1, 2]. While effective in supervised learning tasks, word embeddings are less suite</p>
<section class="sm xbox font-sm" id="pone.0330421.s002"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response-to-Reviewers.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360590/bin/pone.0330421.s002.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330421.s002.docx</a><sup> (49.7KB, docx) </sup>
</div></div></section></section></article><article class="sub-article" id="pone.0330421.r003"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0330421.r003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330421.r003</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 1</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ishengoma%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Fredrick Ishengoma</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Fredrick Ishengoma</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ishengoma%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fredrick Ishengoma</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.e" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.e" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.e" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Fredrick Ishengoma</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.e" class="d-panel p" style="display: none">
<div>© 2025 Fredrick Ishengoma</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>12 Mar 2025</em>
</p>
<p>Dear Dr. Ashraf,</p>
<p>Thank you for submitting your manuscript to PLOS ONE. After careful consideration, we feel that it has merit but does not fully meet PLOS ONE’s publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the points raised during the review process.</p>
<p>Please submit your revised manuscript by Apr 26 2025 11:59PM. If you will need more time than this to complete your revisions, please reply to this message or contact the journal office at <span>plosone@plos.org</span> . When you're ready to submit your revision, log on to <a href="https://www.editorialmanager.com/pone/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.editorialmanager.com/pone/</a> and select the 'Submissions Needing Revision' folder to locate your manuscript file.</p>
<ul class="list" style="list-style-type:disc">
<li><p>A rebuttal letter that responds to each point raised by the academic editor and reviewer(s). You should upload this letter as a separate file labeled 'Response to Reviewers'.</p></li>
<li><p>A marked-up copy of your manuscript that highlights changes made to the original version. You should upload this as a separate file labeled 'Revised Manuscript with Track Changes'.</p></li>
<li><p>An unmarked version of your revised paper without tracked changes. You should upload this as a separate file labeled 'Manuscript'.</p></li>
</ul>
<p>If you would like to make changes to your financial disclosure, please include your updated statement in your cover letter. Guidelines for resubmitting your figure files are available below the reviewer comments at the end of this letter.</p>
<p>If applicable, we recommend that you deposit your laboratory protocols in protocols.io to enhance the reproducibility of your results. Protocols.io assigns your protocol its own identifier (DOI) so that it can be cited independently in the future. For instructions see: <a href="https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols</a> . Additionally, PLOS ONE offers an option for publishing peer-reviewed Lab Protocol articles, which describe protocols hosted on protocols.io. Read more information on sharing protocols at <a href="https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols</a> .</p>
<p>We look forward to receiving your revised manuscript.</p>
<p>Kind regards,</p>
<p>Fredrick Romanus Ishengoma</p>
<p>Academic Editor</p>
<p>PLOS ONE</p>
<p>[Note: HTML markup is below. Please do not edit.]</p>
<p>Reviewers' comments:</p>
<p>Reviewer's Responses to Questions</p>
<p>
<strong>Comments to the Author</strong>
</p>
<p>Reviewer #3: All comments have been addressed</p>
<p>Reviewer #4: (No Response)</p>
<p>Reviewer #5: (No Response)</p>
<p>Reviewer #6: All comments have been addressed</p>
<p>Reviewer #7: (No Response)</p>
<p>**********</p>
<p>2. Is the manuscript technically sound, and do the data support the conclusions??&gt;</p>
<p>Reviewer #3: Yes</p>
<p>Reviewer #4: Partly</p>
<p>Reviewer #5: Partly</p>
<p>Reviewer #6: Yes</p>
<p>Reviewer #7: Yes</p>
<p>**********</p>
<p>3. Has the statistical analysis been performed appropriately and rigorously? --&gt;?&gt;</p>
<p>Reviewer #3: Yes</p>
<p>Reviewer #4: N/A</p>
<p>Reviewer #5: N/A</p>
<p>Reviewer #6: No</p>
<p>Reviewer #7: N/A</p>
<p>**********</p>
<p>4. Have the authors made all data underlying the findings in their manuscript fully available??&gt;</p>
<p>The <a href="http://www.plosone.org/static/policies.action#sharing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PLOS Data policy</a></p>
<p>Reviewer #3: Yes</p>
<p>Reviewer #4: No</p>
<p>Reviewer #5: Yes</p>
<p>Reviewer #6: Yes</p>
<p>Reviewer #7: Yes</p>
<p>**********</p>
<p>5. Is the manuscript presented in an intelligible fashion and written in standard English??&gt;</p>
<p>Reviewer #3: Yes</p>
<p>Reviewer #4: No</p>
<p>Reviewer #5: Yes</p>
<p>Reviewer #6: Yes</p>
<p>Reviewer #7: Yes</p>
<p>**********</p>
<p>Reviewer #3: The authors deal with an interesting topic, which could be of benefit to specific journal collection, the topic is well work out at the required level in term of content and of formal aspect:</p>
<p>- the aims are clear formulated;</p>
<p>- related work is worked out at the good level, the used of the resources are sufficient (138 resources);</p>
<p>- the paper is well written, it meets the formal requirements;</p>
<p>- the extent of the paper is adequate, the structure is clear and the conclusions are sufficient.</p>
<p>Reviewer #4: The paper is poorly prepared. Few fundamental writing issues are mentioned below.</p>
<p>1.</p>
<p>Abstract is filled by useless background text starting from the "advent of World Wide Web" and its revolutionary rise making me doubt which century this paper was written. There are also typos (reserch) indicating that nobody has put serious efforts for the writing. This is amazing since there are 8 authors in the paper. It takes just couple of sentences to define the research question (fake news detection), and the proposed approach (unsupervised clustering) and the novelty (measure to select the number of clustrers).</p>
<p>2.</p>
<p>Introduction focus on superficial literature review, and near the end, suddenly jumping from the backgroud to explaining that Gaussian mixture models has been widely used modeling dynamic systems without telling what is the dynamic system and how it relates to the previous text.</p>
<p>3.</p>
<p>Section 2 is non-innovatively titles as "Proposed methodology" covering everything from pre-processing (TF-IDF is hardly pre-processing anymore), to Gaussian mixture models and selecting the number of clusters including four levels of sections showing the complete failure to organize the writing properly. The results then follow in section which is not even numbered.</p>
<p>4.</p>
<p>GCO is claimed to be used to solve the hyperparameters like the number of components and converging threshold but later Elbow method and BIC are introduced anyway. Why? Elbow method seems to refer calculating first derivative (delta), which is not sufficient to alone for the task.</p>
<p>5.</p>
<p>The more essential are missing from the paper. Why (and how) we can expect to detect fake news from the clustering result, or Gaussian mixture models? None of the pictures show the true/fake labels for the points, only there cluster id.</p>
<p>6.</p>
<p>Figures are missing from the paper but included separately making it hard to follow. The document is also mixed by review reports, responses, two versions of the paper, in seemingly random order. Not sure if this is automated or author organized order of the material, but it just does not work well.</p>
<p>Overall, I cannot sure has any real research done. The text might be just written by AI, added some fake graphs and curves. At least I did not find any evidence of any intellectual human involvment (despite of 8 names). On the contrary, seems there is lack of human involvmenent in the process here. I am amazed this kind of paper is even allowed to be revised. Reject straightforward.</p>
<p>Reviewer #5: Referee report</p>
<p>Re: PONE-D-24-50144R1</p>
<p>In this work, clustering algorithms are applied and compared to provide an unsupervised method of detecting fake news in social media. Specifically, a combined method that was shown to be efficient in this field was proposed. The authors have submitted a revision of their original manuscript addressing the comments of the first referee round. However, though they have replied in detail (including respective references) directly to the comments in the rebuttal letter, few of these comments were addresses in the revised manuscript. At the same time, it does not seem to be any page 19 or 20 in the manuscript on which new content was added - see e.g. reply to Reviewer 1, Concern #5). The authors should consider addressing the comments in their manuscript and add there the new references. Especially, the comments on applicability, conclusions, reproducibility, and limitations of this study should be addressed in detail.</p>
<p>In that respect, the manuscript should be considered for publication in PLOS One after all comments of the previous and current referee round are carefully addressed (also in the manuscript).</p>
<p>Comments</p>
<p>1. Balancing the data: were class balancing algorithms used for this purpose or data were chosen based on a random number generator or similar? Please, clarify. If that is the case, why weren't class balancing algorithms used? Also, what parts of the data are used: are these vectors of words and/or TF-IDF values?</p>
<p>2. PCA analysis: please clarify the input data for this. What is the dimension of the post-processed data? Are only the words in form of a vector used for the PCA?</p>
<p>3. Proposed hybrid GMM model: is this taking as input only the 2D data from the PCA analysis? Please, clarify, what data are used exactly. On pg. 8 it is stated that the fitness is based on a comparison of predicted to true labels. However, the authors claim that their method is unsupervised. Please, clarify in detail the process. Also, the term 'agent' should be defined clearly.</p>
<p>4. Why were both the elbow method and the Silhouette score used? Both provide the number of clusters. One of these should be sufficient.</p>
<p>5. On pg.12 reference on the posterior probability, E-step and M-step is made without providing any details of these terms. These should be clarified in the context of this work.</p>
<p>6. Some benchmark numbers on the computational performance of the hybrid GMM model should be provided, i.e. resources used and computing time for the analysis.</p>
<p>7. Comparison of Fig.2 and 4 reveals that the only additional information obtained from the hybrid GMM is the division of the representation in clusters. Is the whole computational effort worth it or could the PCA analysis be used in another more efficient way to reach the detection goal? The authors should clarify this to strengthen their proposed model.</p>
<p>8. The clusters in Fig.4 were assigned a label based on the labels in the original data set. How does then the statement of unsupervision hold? Is the whole model applied only to go back to assign labels from the original set? That supports the fact that the labels are needed.</p>
<p>Also parts of the data of cluster 1 and 3 are quite spread and include many outliers. How can these be handled in terms of fake news detection?</p>
<p>9. Fig.5: the elbow method seems not to be efficient for the k-Medoid method, as there are two abrupt changes in the curve.</p>
<p>10. The authors state on pg.15 that "This divergence in the optimal number of clusters across the algorithms highlights the sensitivity of each clustering technique to the underlying data." This would be a very important factor that would hinder the use of the proposed method for the detection of fake news, as the detection would be prone to large errors based on the number of clusters detected. How can the authors overcome this view?</p>
<p>11. Comparing Fig. 4 and 9: it can be clearly seen that k-means performs the same as the hybrid GMM and should also be computationally faster. How does then the hybrid GMM method perform better, as implied in the manuscript on pg.16and in the conclusions?</p>
<p>12. Finally, it is not clear how the hybrid GMM model could realistically contribute in fake news detection. First, it does not provide additional valuable information beyond the k-means algorithm. At the same time, the detected clusters need to be labeled to distinguish between 'fake' and 'true' news. In that case, it seems to be practically of higher interest to apply an efficient supervised scheme that can directly classify the news, even if that is computationally more demanding. In that respect, the authors should either strengthen the arguments used and analysis performed or direct their proposed methods to other applications.</p>
<p>Minor Comments</p>
<p>(a) Some typesetting errors throughout should be corrected, e.g. 'vactorization' on pg.6.</p>
<p>(b) Provide references - were missing - for the algorithms used, e.g. GMM (pg.7) or Bayesian Information Criterion (BIC) or k-Medoids (pg.10), when these are first introduced. The same holds for other used concepts/codes/libraries, like scikit (pg.11).</p>
<p>(c) Define abbreviations when first used, e.g. PDF on pg.7.</p>
<p>(d) The title of subsection 2.7 should be changed, as there are no experiments involved.</p>
<p>(e) Pg. 15: "We also conducted a comprehensive set of experiments" and "During the experiments", please rephrase as no experiments were performed. The same on all pages referring to 'experiments'.</p>
<p>(f) The legends in Fig.6 are not clear, please define.</p>
<p>Reviewer #6: 1. The paper introduces a novel hybrid approach combining GMM with GCO for unsupervised fake news detection. This is addresses the limitations of traditional GMM, such as sensitivity to initial parameters and the need for manual cluster selection. However, the paper could benefit from a more detailed discussion on how this hybrid approach compares to other state-of-the-art unsupervised methods in fake news detection.</p>
<p>2. The methodology section should provide a more detailed explanation of the GCO algorithm and its integration with the GMM. Specifically, the paper should elaborate on how GCO avoids local optima through its population-based optimization approach and how it enhances GMM's performance in fake news detection by automating parameter tuning and improving convergence to globally optimal solutions.</p>
<p>3. The paper uses Silhouette Score, Adjusted Rand Index, and Purity Score for evaluation. However, the paper should also consider including other metrics like F1-score or precision/recall to provide a more comprehensive evaluation of the model's performance.</p>
<p>4. The paper should provide more details on the data preprocessing steps, such as how the dataset was balanced and why certain preprocessing techniques were chosen.</p>
<p>5. Ensure all figures, tables, and equations are referenced in the text. For example, Fig. 5 and Fig. 6 are not discussed in detail in the results section.</p>
<p>6. In Figure 4 and Figure 7 captions lack sufficient detail. For example, Fig. 4 should specify what the clusters represent.</p>
<p>7. In table 1 should include additional metrics like F1-score or precision/recall for a more comprehensive comparison.</p>
<p>8. Equation 6, Equation 7 are difficult to follow due to dense notation. Break them down into simpler steps or provide more explanatory text. Ensure all equations are referenced in the text. For example, Equation 10 (fitness function for GCO) is introduced without sufficient context. Explain how this function is used in the optimization process.</p>
<p>9. Many references are outdated (from 2017-2019). Include more recent studies (2022-2024) to reflect the latest advancements in fake news detection and unsupervised learning. Some references are incomplete or lack key details. For example, Reference does not include the conference name or page numbers. Ensure all references follow the required citation style. Ensure consistent formatting across references. For example, some references include full first names, while others use initials. Follow the journal's style guide.</p>
<p>Reviewer #7: In the manuscript, Dr. Perveen et al. proposed to use a hybrid method based on Gaussian Mixture Model (GMM) and the Group Counseling Optimizer (GCO) to detect fake news on social media. Their methodology are statistically sound and the results are promising. I have read their revised paper and their response to previous reviewer comments. They have addressed most of their concerns. Yet, there is one important comment that I found their answer unsatisfactory.</p>
<p>Both Reviewer #1 (Comment #3) and Reviewer #2 (Comment #7) suggested the authors to compare their method to some other available methods like the DBScan algorithms. But, the author only provided an empirical discussion about DBScan without showing any solid result ("Our prior experience with DBSCAN yielded unsatisfactory results in terms of clustering accuracy for fake news detection. Although these newer density-based algorithms (QuickDBScan,</p>
<p>KDTreeDBSCAN and DenLAC) may offer improvements, the reason to exclude them was likely based on the</p>
<p>failure of DBSCAN to meet the requirements for our specific dataset and the added complexity these</p>
<p>algorithms could introduce without guaranteed improvements.") I think the authors should further address this comment with some numerical results or real data analysis to show that why DBScan's results are unsatisfactory and the other available methods like QuickDBScan only provide limited improvement. This comment is important to help the readers to understand the advantages of their newly proposed method.</p>
<p>Apart from this comment, I also have a few Minor concerns:</p>
<p>Minor concerns:</p>
<p>1. How can users know which cluster is the fake news after applying your method? In your data analysis, you have the ground truth label. So, it's quite easy to find the cluster standing for fake news. But, in reality, people do not have access to such labels. Please discuss how to mitigate this problem.</p>
<p>2. The authors should double check the mathematical formulas in their manuscript. There are still many typos in the revised version:</p>
<p>(1) In line 228 and 230 on page 6, descriptive words in formula (2) and (3) should not be italic. But variables like t and n should be italic.</p>
<p>(2) In equation (5) on page 7, it should be \Lambda rather than \wedge.</p>
<p>(3) In line 275 on page 7, "D X D" should be $D \times D$.</p>
<p>(4) In equation 7 on page 8, \mathcal{N} is missing before (x|\mu_k,\Sigma_k). The same problem in line 281.</p>
<p>(5) In the sentence beginning from line 280, "Whereas \mu_k is the D-dimensional mean vector" should be ", where as \mu_k is the D-dimensional mean vector". Also, there is an unnecessary comma before \Sigma_k.</p>
<p>(6) In the denominator equation 8, there is an unnecessary word "that".</p>
<p>(7) In the end of line 312, a right parenthesis is missing.</p>
<p>**********</p>
<p><a href="https://journals.plos.org/plosone/s/editorial-and-peer-review-process#loc-peer-review-history" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">what does this mean?</a> ). If published, this will include your full peer review and any attached files.</p>
<p>If you choose “no”, your identity will remain anonymous but your review may still be made public.</p>
<p><strong>Do you want your identity to be public for this peer review?</strong> For information about this choice, including consent withdrawal, please see our <a href="https://www.plos.org/privacy-policy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Privacy Policy</a></p>
<p>Reviewer #3: No</p>
<p>Reviewer #4: No</p>
<p>Reviewer #5: No</p>
<p>Reviewer #6: No</p>
<p>Reviewer #7: No</p>
<p>**********</p>
<p>[NOTE: If reviewer comments were submitted as an attachment file, they will be attached to this email and accessible via the submission site. Please log into your account, locate the manuscript record, and check for the action link "View Attachments". If this link does not appear, there are no attachment files.]</p>
<p>While revising your submission, please upload your figure files to the Preflight Analysis and Conversion Engine (PACE) digital diagnostic tool, <a href="https://pacev2.apexcovantage.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pacev2.apexcovantage.com/</a> . PACE helps ensure that figures meet PLOS requirements. To use PACE, you must first register as a user. Registration is free. Then, login and navigate to the UPLOAD tab, where you will find detailed instructions on how to use the tool. If you encounter any issues or have any questions when using PACE, please email PLOS at <span>figures@plos.org</span> . Please note that Supporting Information files do not need this step.</p></section></article><article class="sub-article" id="pone.0330421.r004"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. 2025 Aug 18;20(8):e0330421. doi: <a href="https://doi.org/10.1371/journal.pone.0330421.r004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330421.r004</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Author response to Decision Letter 2</h1></hgroup><ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="anp_a.f" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.f" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="anp_a.f" class="d-panel p" style="display: none"><div class="notes p"><section id="historyfront-stub4" class="history"><p>Collection date 2025.</p></section></div></div>
<div id="clp_a.f" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>24 Apr 2025</em>
</p>
<p>Respected Reviewers,</p>
<p>On behalf of my all co-authors, I am really grateful for your precious time to review our article and your inslightful comments that indeed improve the quality of our paper. In second round of revision, we have addressed your all constructive comments very carefully. The response against each comment is given in response to reviewer file.</p>
<p>The revised manuscript with highlighted text presents all the changes as per your kind directions.</p>
<p>Many Thanks.</p>
<p>Prfound Regard</p>
<p>Dr. M. Usman</p>
<section class="sm xbox font-sm" id="pone.0330421.s003"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response-to-Reviewers 23-4-25.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360590/bin/pone.0330421.s003.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330421.s003.docx</a><sup> (50.9KB, docx) </sup>
</div></div></section></section></article><article class="sub-article" id="pone.0330421.r005"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0330421.r005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330421.r005</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 2</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ishengoma%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Fredrick Ishengoma</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Fredrick Ishengoma</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ishengoma%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fredrick Ishengoma</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.g" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.g" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.g" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Fredrick Ishengoma</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.g" class="d-panel p" style="display: none">
<div>© 2025 Fredrick Ishengoma</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>12 May 2025</em>
</p>
<p>Dear Dr. Ashraf,</p>
<p>Thank you for submitting your manuscript to PLOS ONE. After careful consideration, we feel that it has merit but does not fully meet PLOS ONE’s publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the points raised during the review process.</p>
<p>Please submit your revised manuscript by Jun 26 2025 11:59PM. If you will need more time than this to complete your revisions, please reply to this message or contact the journal office at <span>plosone@plos.org</span> . When you're ready to submit your revision, log on to <a href="https://www.editorialmanager.com/pone/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.editorialmanager.com/pone/</a> and select the 'Submissions Needing Revision' folder to locate your manuscript file.</p>
<ul class="list" style="list-style-type:disc">
<li><p>A rebuttal letter that responds to each point raised by the academic editor and reviewer(s). You should upload this letter as a separate file labeled 'Response to Reviewers'.</p></li>
<li><p>A marked-up copy of your manuscript that highlights changes made to the original version. You should upload this as a separate file labeled 'Revised Manuscript with Track Changes'.</p></li>
<li><p>An unmarked version of your revised paper without tracked changes. You should upload this as a separate file labeled 'Manuscript'.</p></li>
</ul>
<p>If you would like to make changes to your financial disclosure, please include your updated statement in your cover letter. Guidelines for resubmitting your figure files are available below the reviewer comments at the end of this letter.</p>
<p>If applicable, we recommend that you deposit your laboratory protocols in protocols.io to enhance the reproducibility of your results. Protocols.io assigns your protocol its own identifier (DOI) so that it can be cited independently in the future. For instructions see: <a href="https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/s/submission-guidelines#loc-laboratory-protocols</a> . Additionally, PLOS ONE offers an option for publishing peer-reviewed Lab Protocol articles, which describe protocols hosted on protocols.io. Read more information on sharing protocols at <a href="https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols</a> .</p>
<p>We look forward to receiving your revised manuscript.</p>
<p>Kind regards,</p>
<p>Fredrick Romanus Ishengoma</p>
<p>Academic Editor</p>
<p>PLOS ONE</p>
<p>[Note: HTML markup is below. Please do not edit.]</p>
<p>Reviewers' comments:</p>
<p>Reviewer's Responses to Questions</p>
<p>
<strong>Comments to the Author</strong>
</p>
<p>Reviewer #6: All comments have been addressed</p>
<p>Reviewer #7: All comments have been addressed</p>
<p>**********</p>
<p>2. Is the manuscript technically sound, and do the data support the conclusions??&gt;</p>
<p>Reviewer #6: Yes</p>
<p>Reviewer #7: (No Response)</p>
<p>**********</p>
<p>3. Has the statistical analysis been performed appropriately and rigorously? --&gt;?&gt;</p>
<p>Reviewer #6: Yes</p>
<p>Reviewer #7: (No Response)</p>
<p>**********</p>
<p>4. Have the authors made all data underlying the findings in their manuscript fully available??&gt;</p>
<p>The <a href="http://www.plosone.org/static/policies.action#sharing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PLOS Data policy</a></p>
<p>Reviewer #6: Yes</p>
<p>Reviewer #7: (No Response)</p>
<p>**********</p>
<p>5. Is the manuscript presented in an intelligible fashion and written in standard English??&gt;</p>
<p>Reviewer #6: Yes</p>
<p>Reviewer #7: (No Response)</p>
<p>**********</p>
<p>Reviewer #6: 1. The citations in the paper are not presented in sequential order, which disrupts the flow and can confuse readers.</p>
<p>2. The references lack consistency in formatting. For example, reference [12] is incomplete and does not follow standard citation structure. Several entries are missing key details such as publication year or venue. A thorough revision is needed to ensure accuracy and adherence to a uniform citation style.</p>
<p>3. The paper presents the core GCO algorithm as a figure, which helps with visualization, but a textual pseudocode version would enhance clarity and reproducibility. Additionally, the algorithm's complexity and convergence behavior should be discussed to provide a deeper understanding of its scalability and efficiency.</p>
<p>4. There is a discrepancy between the table reference in the text ("Table 1") and its actual label ("Table 5"). This should be corrected to maintain proper cross-referencing.</p>
<p>5. Some sections, like the social media history in the introduction, are too long and could be shortened.</p>
<p>6. Equations need consistent formatting, and terms like "Silhouette Score" should be defined before use.</p>
<p>Reviewer #7: The authors have addressed all my concerns. Yet, I think they should add the response to my comment on "discussions about how users can decide which cluster is the fake news after applying their method" to their discussion part. It will be helpful for users to have some guidance on this point.</p>
<p>**********</p>
<p><a href="https://journals.plos.org/plosone/s/editorial-and-peer-review-process#loc-peer-review-history" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">what does this mean?</a> ). If published, this will include your full peer review and any attached files.</p>
<p>If you choose “no”, your identity will remain anonymous but your review may still be made public.</p>
<p><strong>Do you want your identity to be public for this peer review?</strong> For information about this choice, including consent withdrawal, please see our <a href="https://www.plos.org/privacy-policy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Privacy Policy</a></p>
<p>Reviewer #6: No</p>
<p>Reviewer #7: No</p>
<p>**********</p>
<p>[NOTE: If reviewer comments were submitted as an attachment file, they will be attached to this email and accessible via the submission site. Please log into your account, locate the manuscript record, and check for the action link "View Attachments". If this link does not appear, there are no attachment files.]</p>
<p>While revising your submission, please upload your figure files to the Preflight Analysis and Conversion Engine (PACE) digital diagnostic tool, <a href="https://pacev2.apexcovantage.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pacev2.apexcovantage.com/</a> . PACE helps ensure that figures meet PLOS requirements. To use PACE, you must first register as a user. Registration is free. Then, login and navigate to the UPLOAD tab, where you will find detailed instructions on how to use the tool. If you encounter any issues or have any questions when using PACE, please email PLOS at <span>figures@plos.org</span> . Please note that Supporting Information files do not need this step.</p></section></article><article class="sub-article" id="pone.0330421.r006"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. 2025 Aug 18;20(8):e0330421. doi: <a href="https://doi.org/10.1371/journal.pone.0330421.r006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330421.r006</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Author response to Decision Letter 3</h1></hgroup><ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="anp_a.h" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.h" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="anp_a.h" class="d-panel p" style="display: none"><div class="notes p"><section id="historyfront-stub6" class="history"><p>Collection date 2025.</p></section></div></div>
<div id="clp_a.h" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>20 May 2025</em>
</p>
<p>Respected Reviewers,</p>
<p>On behalf of all co-authors, I am really grateful to all the reviewers for their precious time to review and giving constructive comments.</p>
<p>We are uploading (a) our point-by-point response to the comments (below) (response to reviewers, under “Response-to-Reviewers Files”), (b) an updated manuscript with yellow highlighting indicating changes (as “Revised Article with Changes Highlighted”), and (c) a clean updated manuscript without highlights (“Manuscript”).</p>
<p>thanks.</p>
<section class="sm xbox font-sm" id="pone.0330421.s004"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response-to-Reviewers 19-5.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360590/bin/pone.0330421.s004.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330421.s004.docx</a><sup> (27.5KB, docx) </sup>
</div></div></section></section></article><article class="sub-article" id="pone.0330421.r007"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0330421.r007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330421.r007</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 3</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mewada%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Hirenkumar Mewada</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Hirenkumar Mewada</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mewada%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hirenkumar Mewada</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.i" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.i" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.i" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Hirenkumar Mewada</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.i" class="d-panel p" style="display: none">
<div>© 2025 Hirenkumar Mewada</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>1 Aug 2025</em>
</p>
<p>Unsupervised fake news detection on social media using hybrid gaussian mixture model</p>
<p>PONE-D-24-50144R3</p>
<p>Dear Dr. Ashraf,</p>
<p>We’re pleased to inform you that your manuscript has been judged scientifically suitable for publication and will be formally accepted for publication once it meets all outstanding technical requirements.</p>
<p>Within one week, you’ll receive an e-mail detailing the required amendments. When these have been addressed, you’ll receive a formal acceptance letter and your manuscript will be scheduled for publication.</p>
<p>An invoice will be generated when your article is formally accepted. Please note, if your institution has a publishing partnership with PLOS and your article meets the relevant criteria, all or part of your publication costs will be covered. Please make sure your user information is up-to-date by logging into Editorial Manager at <a href="https://www.editorialmanager.com/pone/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Editorial Manager®</a>  and clicking the ‘Update My Information' link at the top of the page. For questions related to billing, please contact <a href="https://plos.my.site.com/s/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">billing support</a> .</p>
<p>If your institution or institutions have a press office, please notify them about your upcoming paper to help maximize its impact. If they’ll be preparing press materials, please inform our press team as soon as possible -- no later than 48 hours after receiving the formal acceptance. Your manuscript will remain under strict press embargo until 2 pm Eastern Time on the date of publication. For more information, please contact onepress@plos.org.</p>
<p>Kind regards,</p>
<p>Hirenkumar Kantilal Mewada</p>
<p>Academic Editor</p>
<p>PLOS ONE</p>
<p>Additional Editor Comments (optional):</p>
<p>Reviewers' comments:</p>
<p>Reviewer's Responses to Questions</p>
<p>
<strong>Comments to the Author</strong>
</p>
<p>Reviewer #6: All comments have been addressed</p>
<p>Reviewer #7: All comments have been addressed</p>
<p>**********</p>
<p>2. Is the manuscript technically sound, and do the data support the conclusions??&gt;</p>
<p>Reviewer #6: Yes</p>
<p>Reviewer #7: Yes</p>
<p>**********</p>
<p>3. Has the statistical analysis been performed appropriately and rigorously? --&gt;?&gt;</p>
<p>Reviewer #6: Yes</p>
<p>Reviewer #7: Yes</p>
<p>**********</p>
<p>4. Have the authors made all data underlying the findings in their manuscript fully available??&gt;</p>
<p>The <a href="http://www.plosone.org/static/policies.action#sharing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PLOS Data policy</a></p>
<p>Reviewer #6: Yes</p>
<p>Reviewer #7: Yes</p>
<p>**********</p>
<p>5. Is the manuscript presented in an intelligible fashion and written in standard English??&gt;</p>
<p>Reviewer #6: Yes</p>
<p>Reviewer #7: Yes</p>
<p>**********</p>
<p>Reviewer #6: I have reviewed the revised manuscript and confirm that all previous comments have been addressed:</p>
<p>Citations are now sequential; The introduction is more concise now and clearly explains dynamic systems and GMMs; The algorithm includes textual pseudocode with complexity and convergence discussion; Table references are corrected; Equations are consistently formatted, and terms like “Silhouette Score” are defined.</p>
<p>Overall, the manuscript has improved in clarity and presentation.</p>
<p>Reviewer #7: (No Response)</p>
<p>**********</p>
<p><a href="https://journals.plos.org/plosone/s/editorial-and-peer-review-process#loc-peer-review-history" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">what does this mean?</a> ). If published, this will include your full peer review and any attached files.</p>
<p>If you choose “no”, your identity will remain anonymous but your review may still be made public.</p>
<p><strong>Do you want your identity to be public for this peer review?</strong> For information about this choice, including consent withdrawal, please see our <a href="https://www.plos.org/privacy-policy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Privacy Policy</a></p>
<p>Reviewer #6: No</p>
<p>Reviewer #7: No</p>
<p>**********</p></section></article><article class="sub-article" id="pone.0330421.r008"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS One. doi: <a href="https://doi.org/10.1371/journal.pone.0330421.r008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0330421.r008</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Acceptance letter</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mewada%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Hirenkumar Mewada</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Hirenkumar Mewada</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mewada%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hirenkumar Mewada</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.j" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.j" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.j" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Hirenkumar Mewada</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.j" class="d-panel p" style="display: none">
<div>© 2025 Hirenkumar Mewada</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>PONE-D-24-50144R3</p>
<p>PLOS ONE</p>
<p>Dear Dr. Ashraf,</p>
<p>I'm pleased to inform you that your manuscript has been deemed suitable for publication in PLOS ONE. Congratulations! Your manuscript is now being handed over to our production team.</p>
<p>At this stage, our production department will prepare your paper for publication. This includes ensuring the following:</p>
<p>* All references, tables, and figures are properly cited</p>
<p>* All relevant supporting information is included in the manuscript submission,</p>
<p>* There are no issues that prevent the paper from being properly typeset</p>
<p>You will receive further instructions from the production team, including instructions on how to review your proof when it is ready. Please keep in mind that we are working through a large volume of accepted articles, so please give us a few days to review your paper and let you know the next and final steps.</p>
<p>Lastly, if your institution or institutions have a press office, please let them know about your upcoming paper now to help maximize its impact. If they'll be preparing press materials, please inform our press team within the next 48 hours. Your manuscript will remain under strict press embargo until 2 pm Eastern Time on the date of publication. For more information, please contact onepress@plos.org.</p>
<p>You will receive an invoice from PLOS for your publication fee after your manuscript has reached the completed accept phase. If you receive an email requesting payment before acceptance or for any other service, this may be a phishing scheme. Learn how to identify phishing emails and protect your accounts at <a href="https://explore.plos.org/phishing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://explore.plos.org/phishing</a>.</p>
<p>If we can help with anything else, please email us at customercare@plos.org.</p>
<p>Thank you for submitting your work to PLOS ONE and supporting open access.</p>
<p>Kind regards,</p>
<p>PLOS ONE Editorial Office Staff</p>
<p>on behalf of</p>
<p>Dr. Hirenkumar Kantilal Mewada</p>
<p>Academic Editor</p>
<p>PLOS ONE</p></section></article><article class="sub-article" id="_ad93_"><section class="pmc-layout__citation font-secondary font-xs"><div></div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Associated Data</h1></hgroup><ul class="d-buttons inline-list"></ul>
<div class="d-panels font-secondary-light"></div>
<div></div>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
</div></section><section class="body sub-article-body"><section id="_adsm93_" lang="en" class="supplementary-materials"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response-to-Reviewers.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360590/bin/pone.0330421.s002.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330421.s002.docx</a><sup> (49.7KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response-to-Reviewers 23-4-25.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360590/bin/pone.0330421.s003.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330421.s003.docx</a><sup> (50.9KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response-to-Reviewers 19-5.docx</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12360590/bin/pone.0330421.s004.docx" data-ga-action="click_feat_suppl" class="usa-link">pone.0330421.s004.docx</a><sup> (27.5KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>
<a href="https://github.com/Sajida-Perveen/Project-code" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Sajida-Perveen/Project-code</a>.</p></section></section></article><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS One are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pone.0330421"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pone.0330421.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12360590/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12360590/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12360590%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12360590/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12360590/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12360590/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40824941/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12360590/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40824941/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12360590/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12360590/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="eiPnc7I9xW7REg2Y5U371e3e9z3g5985mbk26fgW2LW7vQho2hNyhmAFm2dCEPQh">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
