
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Deciphering the binding mechanism of tafamidis to calf thymus DNA: multimodal spectroscopic, thermodynamic, and computational perspectives - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4BBD08AF434D305BBD00028968B55.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="rscadv">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376933/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="RSC Advances">
<meta name="citation_title" content="Deciphering the binding mechanism of tafamidis to calf thymus DNA: multimodal spectroscopic, thermodynamic, and computational perspectives">
<meta name="citation_author" content="Manal A Alossaimi">
<meta name="citation_author_institution" content="Pharmaceutical Chemistry Department, College of Pharmacy, Prince Sattam bin Abdulaziz University, Al-Kharj 11942, Saudi Arabia, Email: m.alossaimi@psau.edu.sa">
<meta name="citation_author" content="Heba Elmansi">
<meta name="citation_author_institution" content="Pharmaceutical Analytical Chemistry Department, Faculty of Pharmacy, Mansoura University, Mansoura, 35516, Egypt">
<meta name="citation_author" content="Fathalla Belal">
<meta name="citation_author_institution" content="Pharmaceutical Analytical Chemistry Department, Faculty of Pharmacy, Mansoura University, Mansoura, 35516, Egypt">
<meta name="citation_author" content="Galal Magdy">
<meta name="citation_author_institution" content="Pharmaceutical Analytical Chemistry Department, Faculty of Pharmacy, Kafrelsheikh University, Kafrelsheikh, 33511, Egypt, Email: galal_magdy@pharm.kfs.edu.eg">
<meta name="citation_author_institution" content="Department of Pharmaceutical Analytical Chemistry, Faculty of Pharmacy, Mansoura National University, Gamasa, 7731168, Egypt">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="15">
<meta name="citation_issue" content="35">
<meta name="citation_firstpage" content="28937">
<meta name="citation_doi" content="10.1039/d5ra04723d">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376933/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376933/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376933/pdf/RA-015-D5RA04723D.pdf">
<meta name="description" content="The interaction between small molecules and biological macromolecules is a crucial area of research with significant implications across various scientific disciplines. Tafamidis is a clinically approved drug for transthyretin-mediated amyloidosis, ...">
<meta name="og:title" content="Deciphering the binding mechanism of tafamidis to calf thymus DNA: multimodal spectroscopic, thermodynamic, and computational perspectives">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The interaction between small molecules and biological macromolecules is a crucial area of research with significant implications across various scientific disciplines. Tafamidis is a clinically approved drug for transthyretin-mediated amyloidosis, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376933/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12376933">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1039/d5ra04723d"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/RA-015-D5RA04723D.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12376933%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12376933/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12376933/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376933/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-rscadv.png" alt="RSC Advances logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to RSC Advances" title="Link to RSC Advances" shape="default" href="https://www.rsc.org/journals-books-databases/about-journals/rsc-advances/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">RSC Adv</button></div>. 2025 Aug 18;15(35):28937–28948. doi: <a href="https://doi.org/10.1039/d5ra04723d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1039/d5ra04723d</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22RSC%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22RSC%20Adv%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22RSC%20Adv%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22RSC%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Deciphering the binding mechanism of tafamidis to calf thymus DNA: multimodal spectroscopic, thermodynamic, and computational perspectives</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alossaimi%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Manal A Alossaimi</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Manal A Alossaimi</span></h3>
<div class="p">
<sup>a</sup>Pharmaceutical Chemistry Department, College of Pharmacy, Prince Sattam bin Abdulaziz University, Al-Kharj 11942, Saudi Arabia, Email: m.alossaimi@psau.edu.sa</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alossaimi%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Manal A Alossaimi</span></a>
</div>
</div>
<sup>a,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Elmansi%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Heba Elmansi</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Heba Elmansi</span></h3>
<div class="p">
<sup>b</sup>Pharmaceutical Analytical Chemistry Department, Faculty of Pharmacy, Mansoura University, Mansoura, 35516, Egypt</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Elmansi%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Heba Elmansi</span></a>
</div>
</div>
<sup>b</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Belal%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Fathalla Belal</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Fathalla Belal</span></h3>
<div class="p">
<sup>b</sup>Pharmaceutical Analytical Chemistry Department, Faculty of Pharmacy, Mansoura University, Mansoura, 35516, Egypt</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Belal%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fathalla Belal</span></a>
</div>
</div>
<sup>b</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Magdy%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Galal Magdy</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Galal Magdy</span></h3>
<div class="p">
<sup>c</sup>Pharmaceutical Analytical Chemistry Department, Faculty of Pharmacy, Kafrelsheikh University, Kafrelsheikh, 33511, Egypt, Email: galal_magdy@pharm.kfs.edu.eg</div>
<div class="p">
<sup>d</sup>Department of Pharmaceutical Analytical Chemistry, Faculty of Pharmacy, Mansoura National University, Gamasa, 7731168, Egypt</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Magdy%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Galal Magdy</span></a>
</div>
</div>
<sup>c,</sup><sup>d,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="affa">
<sup>a</sup>Pharmaceutical Chemistry Department, College of Pharmacy, Prince Sattam bin Abdulaziz University, Al-Kharj 11942, Saudi Arabia, Email: m.alossaimi@psau.edu.sa</div>
<div id="affb">
<sup>b</sup>Pharmaceutical Analytical Chemistry Department, Faculty of Pharmacy, Mansoura University, Mansoura, 35516, Egypt</div>
<div id="affc">
<sup>c</sup>Pharmaceutical Analytical Chemistry Department, Faculty of Pharmacy, Kafrelsheikh University, Kafrelsheikh, 33511, Egypt, Email: galal_magdy@pharm.kfs.edu.eg</div>
<div id="affd">
<sup>d</sup>Department of Pharmaceutical Analytical Chemistry, Faculty of Pharmacy, Mansoura National University, Gamasa, 7731168, Egypt</div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jul 3; Accepted 2025 Aug 11; Collection date 2025 Aug 11.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>This journal is © The Royal Society of Chemistry</div>
<p>This article is licensed under a <a href="https://creativecommons.org/licenses/by/3.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution 3.0 Unported Licence</a>. You can use material from this article in other publications without requesting further permissions from the RSC, provided that the correct acknowledgement is given.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12376933</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The interaction between small molecules and biological macromolecules is a crucial area of research with significant implications across various scientific disciplines. Tafamidis is a clinically approved drug for transthyretin-mediated amyloidosis, yet its molecular interactions with biological macromolecules such as DNA remain unexplored. Investigating such interactions is crucial for understanding its broader pharmacodynamic profile and potential off-target effects. In this study, the binding interaction between tafamidis and calf thymus DNA (ct-DNA) was investigated, for the first time, using a combination of spectroscopic techniques, viscosity measurements, ionic strength studies, thermodynamic analysis, and computational modeling. The key binding parameters, including the binding constant, number of binding sites, binding forces, and binding mode were determined. UV-vis spectroscopic analysis revealed a binding constant on the order of 10<sup>5</sup> M<sup>−1</sup>, indicating a moderate-to-strong binding affinity between tafamidis and ct-DNA. Thermodynamic parameters (Δ<em>H</em>° &gt; 0 and Δ<em>S</em>° &gt; 0) suggested that hydrophobic interactions primarily drive the binding process. Fluorescence spectroscopy, viscosity measurements, and molecular modeling indicated that tafamidis preferentially binds to AT-rich regions of ct-DNA and acts as a minor groove binder. Ethidium bromide displacement assays showed no significant effect of tafamidis on the ethidium bromide–DNA complex, further supporting the groove binding mechanism. Molecular dynamics simulations corroborated these findings. Moreover, molecular dynamics simulations validated the dynamic behavior and stability of the tafamidis–ct-DNA complex, demonstrating its excellent structural stability. Overall, these results enhance the understanding of tafamidis by providing valuable insights into its pharmacological mechanisms and molecular interactions.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p>This work introduces the first study on the binding interaction between tafamidis and calf thymus DNA using multi-spectroscopic, viscosity, ionic strength, thermodynamic, molecular docking, and molecular dynamic simulation approaches.<a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376933_d5ra04723d-ga.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/14891efd2f8a/d5ra04723d-ga.jpg" loading="lazy" id="ga" height="473" width="678" alt="graphic file with name d5ra04723d-ga.jpg"></a></p></section><section id="sec1"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Tafamidis, (TAF, <a href="#fig1" class="usa-link">Fig. 1</a>) is a medication used to treat transthyretin-mediated amyloidosis, which is a diverse collection of illnesses that result in end-organ damage due to the extracellular accumulation of insoluble misfolded proteins.<sup><a href="#cit1" class="usa-link" aria-describedby="cit1">1</a></sup> It is chemically named as 2-(3,5-dichlorophenyl)-benzoxazole-6-carboxylic acid. It is a non-NSAID small molecule that is orally bioavailable and administered once daily.<sup><a href="#cit2" class="usa-link" aria-describedby="cit2">2</a></sup> This benzoxazole-based small molecule belongs to transthyretin stabilizers. It acts by the prevention of the formation of the transthyretin protein which deposits in the heart. The liver creates a protein called transthyretin (TTR). This protein normally exists as a stable group of four units. However, changes in its shape can cause it to become unstable and break down into individual units. This can happen naturally as we age (called ATTRwt) or due to inherited genetic problems (called ATTRv). It is sold under the trade names: Vyndamax® and Vyndaqel®.<sup><a href="#cit3" class="usa-link" aria-describedby="cit3">3</a></sup></p>
<figure class="fig xbox font-sm" id="fig1"><h3 class="obj_head">Fig. 1. Chemical structure of tafamidis.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/cce5d85c6c88/d5ra04723d-f1.jpg" loading="lazy" height="331" width="708" alt="Fig. 1"></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Deoxyribonucleic acid is essential to life because it contains information about ancestry. One of the main areas of research in the last few decades has been the process by which certain tiny molecules bind to DNA.<sup><a href="#cit4" class="usa-link" aria-describedby="cit4">4</a></sup> There are three patterns of attachments of small molecules to DNA double helical structures: (i) electrostatic binding results from the interaction of small molecules' positively charged ends with the negatively charged DNA phosphate backbone; (ii) intercalative binding occurs when tiny molecules intercalate within stacked base pairs, changing the DNA shape,<sup><a href="#cit5" class="usa-link" aria-describedby="cit5">5</a></sup> (iii) groove binding: happens in the deep major groove or the shallow minor groove as a result of van der Waals interaction or hydrogen bonding with nucleic acid bases and small molecules. The DNA backbone is either not distorted at all or only slightly by groove binders.<sup><a href="#cit6" class="usa-link" aria-describedby="cit6">6</a></sup> Nonetheless, DNA can directly interact with a wide variety of tiny molecules, and the processes influencing these interactions are quite intricate. Usually, no one approach provides a thorough comprehension of drug–DNA interactions. Consequently, it is crucial to create quick, high-throughput, economical, and ongoing methods to assess these interactions for different medications. These developments can greatly facilitate the processes of drug approval and discovery. According to this viewpoint, DNA is acknowledged as a vital biomacromolecule.</p>
<p>The effectiveness of many drugs is closely linked to their binding mode and affinity to DNA. As a result, studying the intermolecular interactions between small-molecule drugs and DNA has turn out to be a significant focus in the different fields of life sciences, chemistry, and medicine. By reviewing the literature varied drugs were investigated for their DNA binding utilizing fluorescence quenching and molecular docking techniques.<sup><a href="#cit7" class="usa-link" aria-describedby="cit7">7–11</a></sup> Although TAF is primarily recognized for its high-affinity binding to transthyretin protein, its interactions with other essential biomacromolecules, such as DNA, remain largely unexplored. Given that drug–DNA interactions can influence gene expression, mutagenicity, or potential off-target effects, it is valuable to assess whether TAF can interact with DNA and to characterize the nature of such binding. Studying its DNA-binding behavior can reveal off-target effects, provide insight into its safety profile, and contribute to understanding its broader pharmacological interactions. Therefore, the aim of this study is to systematically investigate the binding interaction between TAF and calf thymus DNA, for the first time, using a multimodal approach, including UV-visible and fluorescence spectroscopy, viscosity measurements, ionic strength effects, thermodynamic analysis, and computational modeling. The results of this study may enhance the understanding of TAF's interaction landscape and contribute to future drug safety and mechanism-of-action studies.</p></section><section id="sec2"><h2 class="pmc_sec_title">2. Experimental</h2>
<section id="sec3"><h3 class="pmc_sec_title">2.1. Materials, reagents and methods</h3>
<p>Analytical grade chemicals were purchased and utilized directly without extra purification. Calf thymus DNA (ct-DNA) was bought from Sigma Aldrich (St. Louis, MO, USA). A 0.05 M Tris–HCl buffer solution was made by dissolving 0.78 g in 100.0 mL of distilled water, with 1 M HCl employed to adjust the pH to 7.4. A stock solution of ct-DNA was prepared in Tris–HCl buffer by dissolving 0.01 g of ct-DNA and diluting to a final volume of 50.0 mL with the buffer. The solution was subsequently stored at 4 °C in dark for duration of up to 7 days. The ct-DNA solution was sonicated prior to the tests to guarantee solution homogeneity. The purity of DNA was assessed by measuring the UV absorbance ratio of <em>A</em><sub>260</sub>/<em>A</em><sub>280</sub>, which exceeded 1.9, indicating that the DNA was suitable for experimental purposes.<sup><a href="#cit12" class="usa-link" aria-describedby="cit12">12,13</a></sup> Distilled water was used whenever needed in the study.</p>
<p>Tafamidis was purchased from Pfizer, New York, NY 10017, USA. Ethanol was used to prepare the stock solution of TAF, which had a final concentration of 1.0 × 10<sup>−3</sup> M. We purchased ethidium bromide (EB), rhodamine B (RB), Tris–HCl from Sigma Aldrich (St. Louis, MO, USA). The solutions of RB (2.0 × 10<sup>−3</sup> M) and EB (1.2 × 10<sup>−3</sup> M) were prepared by dissolving in ethanol and stored at 4 °C.</p></section><section id="sec4"><h3 class="pmc_sec_title">2.2. Instruments</h3>
<p>• T80+ UV/vis PC spectrophotometer (PG Instruments Ltd, Woodway Lane, Wibtoft, England) with a 1.0 cm quartz cell.</p>
<p>• Spectrofluorimetric measurements were conducted using a Cary Eclipse spectrofluorimeter with a xenon flash lamp (Agilent Technologies, Santa Clara, CA, USA).</p>
<p>• An Oswald viscometer, maintained at a controlled temperature of 298 K, was utilized in the viscosity measurements.</p></section><section id="sec5"><h3 class="pmc_sec_title">2.3. Techniques for studying the binding interaction</h3>
<section id="sec6"><h4 class="pmc_sec_title">2.3.1. Spectrophotometric measurements</h4>
<p>Standard procedures for drug–DNA interaction studies <em>via</em> UV-vis spectroscopy were followed as described in earlier literature.<sup><a href="#cit14" class="usa-link" aria-describedby="cit14">14,15</a></sup> Using T80+ UV/vis PC spectrophotometer; the UV spectra of 3.0 × 10<sup>−5</sup> M ct-DNA solutions were recorded within the 200–400 nm range while gradually increasing TAF concentration (from 0 to 1.5 × 10<sup>−5</sup> M). Baseline correction was performed using appropriate blanks (buffer or solvent only), and all measurements were conducted at least in triplicate. Measurements were conducted accurately at three temperatures (298, 303, and 308 K) to assess binding constants and assess the influence of temperature on the drug–DNA interaction. Furthermore, the absorbance of a solution containing TAF (0.9 × 10<sup>−5</sup> M) and ct-DNA (3.0 × 10<sup>−5</sup> M) was analyzed at 298 K under varying NaCl concentrations (0 to 0.07 M) to assess the influence of strength of ions on interaction. NaCl solutions of various concentrations were freshly prepared and added to the DNA–TAF complex to assess ionic strength effects. Absorbance changes were monitored after equilibration at 298 K.<sup><a href="#cit16" class="usa-link" aria-describedby="cit16">16</a></sup> All absorbance values were measured at 260 nm.</p></section><section id="sec7"><h4 class="pmc_sec_title">2.3.2. Viscosity measurements</h4>
<p>An Oswald viscometer (a capillary inner diameter of 0.57 mm), was used for viscosity measurements, thermostated at 298 K by means of a fixed-temperature bath. The ct-DNA concentration was maintained constant at 30.0 μM in a Tris–HCl buffer solution (pH 7.4), with varying TAF concentrations (0–15.0 μM). A digital stopwatch was employed to accurately compute the flow time of the solutions. Each viscosity measurement was repeated three times to ensure reproducibility, and the average flow time was recorded. The relative specific viscosity was calculated using established methods.<sup><a href="#cit17" class="usa-link" aria-describedby="cit17">17,18</a></sup> Hence, the relative specific viscosity (<em>μ</em>/<em>μ</em><sub>0</sub>)<sup>1/3</sup> was calculated based on the average values obtained from the three independent replicate measurements. Here, <em>μ</em><sub>0</sub> represents the specific viscosity of the ct-DNA solution, while <em>μ</em> denotes the specific viscosity of the ct-DNA–TAF complex. The viscosity measurements were executed to assess the structural changes in DNA on interaction with TAF, providing insights into the binding mode. The findings were analyzed by plotting (<em>η</em>/<em>η</em><sub>0</sub>)<sup>1/3</sup> as a function of the binding ratio <em>r</em> (<em>r</em> = [TAF]/[ct-DNA]), allowing for a quantitative evaluation of the impact of TAF on the DNA's hydrodynamic properties.</p></section><section id="sec8"><h4 class="pmc_sec_title">2.3.3. Spectrofluorimetric measurements</h4>
<p>Standard procedures for drug–DNA interaction studies <em>via</em> fluorescence spectroscopy were followed as described in earlier literature.<sup><a href="#cit14" class="usa-link" aria-describedby="cit14">14,15</a></sup> Mixtures containing ct-DNA (30.0 μM) were measured in the existence of either of the fluorescent probes. The fluorescence measurements were conducted to examine the binding interactions between ct-DNA and the probes, offering insights into structural and environmental changes upon interaction. These probes are RB (2.0 × 10<sup>−3</sup> M, <em>λ</em><sub>ex</sub>/<em>λ</em><sub>em</sub> = 465/577 nm) and EB (1.2 × 10<sup>−3</sup> M, <em>λ</em><sub>ex</sub>/<em>λ</em><sub>em</sub> = 525/602 nm), which correspond to different groove and intercalation binding probes, respectively, in the presence and absence of varying TAF concentrations (0–15.0 μM). Three measurements of each emission spectrum were made, and the average reading was calculated.</p></section><section id="sec9"><h4 class="pmc_sec_title">2.3.4. Molecular docking</h4>
<p>Molecular docking studies with TAF and two calf thymus B-DNA sequences were executed by using AutoDock 4.2 software. The 3D coordinates of the calf thymus B-DNA sequences (PDB ID: 1D29 and 3EY0) were acquired from the Protein Data Bank and TAF structure from the PubChem database. The B-DNA was prepared for docking by adding the Kollman charges and saving in .pdbqt format. The ligand is energy reduced utilizing MMFF94 force-field in Chem3D ultra 16.0 software.<sup><a href="#cit19" class="usa-link" aria-describedby="cit19">19,20</a></sup> The autogrid function was set up with the grid box dimensions at a spacing point of 0.375 as mentioned in <a href="#tab1" class="usa-link">Table 1</a>. The Lamarckian genetic algorithm was investigated for binding energy calculation with a population size of 100 and iteration of 2.5 million times. The visualization of 3D and 2D interacting diagram for the presentation for TAF–calf thymus DNA complexes were obtained by Chimera 1.14 version and Biovia Discovery Studio software, respectively.<sup><a href="#cit21" class="usa-link" aria-describedby="cit21">21,22</a></sup></p>
<section class="tw xbox font-sm" id="tab1"><h5 class="obj_head">Table 1. PDB entries and grid point data for the two DNA sequences.</h5>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
</colgroup>
<thead><tr>
<th rowspan="1" colspan="1">DNA (PDB ID)</th>
<th rowspan="1" colspan="1">Sequence</th>
<th rowspan="1" colspan="1">Grid points (coordinates) <em>x</em>, <em>y</em>, <em>z</em>
</th>
<th rowspan="1" colspan="1">Grid points (sizes) <em>x</em>, <em>y</em>, <em>z</em>
</th>
<th rowspan="1" colspan="1">Population size</th>
<th rowspan="1" colspan="1">Iteration (million)</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="1" colspan="1">
3EY0
</td>
<td rowspan="1" colspan="1">5′-(ATA TAT ATAT)-3′</td>
<td rowspan="1" colspan="1">16.418, 10.417, 90.217</td>
<td rowspan="1" colspan="1">100, 62, 94</td>
<td rowspan="1" colspan="1">150</td>
<td align="center" rowspan="1" colspan="1">2.5</td>
</tr>
<tr>
<td rowspan="1" colspan="1">
1D29
</td>
<td rowspan="1" colspan="1">5′-(CGT GAA TTC ACG)-3′</td>
<td rowspan="1" colspan="1">14.92, 20.905, 8.82</td>
<td rowspan="1" colspan="1">72, 72, 126</td>
<td rowspan="1" colspan="1">150</td>
<td align="center" rowspan="1" colspan="1">2.5</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec10"><h4 class="pmc_sec_title">2.3.5. Molecular dynamic simulation</h4>
<p>Molecular dynamics (MD) simulation was performed using GROMACS 2022.2. The simulations were executed in a three-stage process.</p>
<section id="sec11"><h5 class="pmc_sec_title">2.3.5.1. Enzyme preparation</h5>
<p>The three-dimensional structures (3D) of the ligand–DNA complexes were saved in .pdb format by PyMOL software for further analysis. Molecular dynamics simulations (MD) were performed utilizing the GROMACS software package (version 2022.2) to analyze the dynamic behavior of the complexes. The simulations afforded atomic-level insights into the binding mechanism and structural stability of the TAF–DNA complex.<sup><a href="#cit23" class="usa-link" aria-describedby="cit23">23–25</a></sup> The DNA topology was created with the AMBER99SB force field,<sup><a href="#cit26" class="usa-link" aria-describedby="cit26">26</a></sup> while the ligand topology was prepared utilizing the SwissParam server.<sup><a href="#cit27" class="usa-link" aria-describedby="cit27">27</a></sup></p></section><section id="sec12"><h5 class="pmc_sec_title">2.3.5.2. Developing a system for simulation</h5>
<p>Following the application of the force field, the DNA–drug complexes were incorporated into the system and solvated using the TIP3P water model within a cubic box, maintaining a minimum distance of 1 nm between the DNA and the box edges.<sup><a href="#cit28" class="usa-link" aria-describedby="cit28">28</a></sup> Periodic boundary conditions were imposed, beside neutralization of the system with Na<sup>+</sup> ions. Energy minimization was performed for 50 000 steps using the steepest descent algorithm, subsequently a 100 ps <em>NVT</em> simulation at 300 K and a 100 ps <em>NPT</em> simulation for system equilibration. The leapfrog algorithm was applied within the constant-temperature and pressure system (<em>NPT</em>) ensemble to independently couple the DNA, ligand, water molecules, and ions.<sup><a href="#cit29" class="usa-link" aria-describedby="cit29">29</a></sup> To ensure a stable environment at 300 K and 1 bar pressure, the Berendsen temperature and pressure coupling constants were set to 0.1 and 2, respectively.<sup><a href="#cit30" class="usa-link" aria-describedby="cit30">30</a></sup> Subsequently, a 100 ns molecular dynamics (MD) simulation was investigated under isothermal–isobaric conditions at 300 K. A pressure coupling time constant of 1 ps was applied to keep a stable pressure of 1 bar, while the LINCS algorithm was employed to constrain bond lengths.<sup><a href="#cit31" class="usa-link" aria-describedby="cit31">31</a></sup> van der Waals and Coulomb interactions were shortened at 1.2 nm, and the Particle Mesh Ewald (PME) method<sup><a href="#cit32" class="usa-link" aria-describedby="cit32">32</a></sup> implemented in GROMACS was utilized to reduce errors associated with interaction truncations.<sup><a href="#cit33" class="usa-link" aria-describedby="cit33">33–35</a></sup></p></section><section id="sec13"><h5 class="pmc_sec_title">2.3.5.3. Analysis and visualization of simulation</h5>
<p>The trajectory files were examined using VMD (Visual Molecular Dynamics) 1.9.2 (ref. <a href="#cit36" class="usa-link" aria-describedby="cit36">36</a>) and analyzed with the custom-developed tool HeroMDAnalysis<sup><a href="#cit37" class="usa-link" aria-describedby="cit37">37,38</a></sup> and Xmgrace 5.1.25.<sup><a href="#cit39" class="usa-link" aria-describedby="cit39">39</a></sup></p></section></section></section></section><section id="sec14"><h2 class="pmc_sec_title">3. Results and discussion</h2>
<p>Understanding the binding interactions among drug molecules and DNA has provided insights into the drug's binding mode. Consequently, DNA–ligand binding studies play a vital role in the improvement of molecular probes and therapeutic agents, guiding the design and synthesis of new drugs. Owing to the importance of TAF in treatment of different diseases, this work investigates the interaction between TAF and ct-DNA using multi-spectroscopic techniques. By analyzing the binding mode, affinity, and interaction forces, the findings offer insights into TAF's molecular behavior, contributing to its understanding in drug development, industry and therapeutic applications.</p>
<section id="sec15"><h3 class="pmc_sec_title">3.1. UV-vis absorption spectra and binding constant determination</h3>
<p>The interaction between TAF and ct-DNA was initially assessed using UV-vis absorption spectroscopy by monitoring spectral changes of DNA at 260 nm upon incremental addition of TAF. As shown in <a href="#fig2" class="usa-link">Fig. 2</a>, a distinct hyperchromic effect was observed with no significant shift in the maximum absorbance wavelength. This observation suggests that TAF binds to DNA through non-intercalative mechanisms, as intercalative binding is usually associated with bathochromic or hypsochromic shifts due to π–π stacking interactions that alter the electronic environment of the DNA base pairs. The increase in absorbance (hyperchromicity) indicates partial unwinding or loosening of the double helix, which supports a groove-binding mode, particularly within the minor groove. Groove-binding agents typically cause slight spectral changes due to local conformational alterations without major disruption of base stacking, in contrast to classical intercalators such as ethidium bromide.<sup><a href="#cit14" class="usa-link" aria-describedby="cit14">14,15,17</a></sup></p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">Fig. 2. UV absorption spectra of ct-DNA (3.0 × 10<sup>−5</sup> M) with different concentrations of TAF. The concentrations of TAF from 0 to 5 were 0, 0.3 × 10<sup>−5</sup>, 0.6 × 10<sup>−5</sup>, 0.9 × 10<sup>−5</sup>, 1.2 × 10<sup>−5</sup>, and 1.5 × 10<sup>−5</sup> M, respectively.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376933_d5ra04723d-f2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/56cc337e3bf2/d5ra04723d-f2.jpg" loading="lazy" height="632" width="779" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The efficacy of a drug is closely linked to its binding affinity, making the study of its interaction with biomacromolecules essential. Binding affinity can be assessed using either the binding constant (<em>K</em><sub>b</sub>) or the dissociation constant (<em>K</em><sub>d</sub>). To quantitatively evaluate the binding strength between TAF and ct-DNA, the binding constant (<em>K</em><sub>b</sub>) was calculated using the Benesi–Hildebrand <a href="#eqn1" class="usa-link">eqn (1)</a>, which is commonly applied to 1 : 1 ligand–DNA interactions:<sup><a href="#cit15" class="usa-link" aria-describedby="cit15">15,17</a></sup></p>
<table class="disp-formula p" id="eqn1"><tr>
<td class="formula"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376933_d5ra04723d-t1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/051146e06491/d5ra04723d-t1.jpg" loading="lazy" id="ugt1" height="90" width="660" alt="3.1."></a></td>
<td class="label">1</td>
</tr></table>
<p>As: <em>A</em> and <em>A</em><sub>0</sub> denote the absorbances of ct-DNA with and without the drug, respectively. <em>ε</em><sub>DNA</sub> and <em>ε</em><sub>TAF–DNA</sub> represent the molar extinction coefficients of ct-DNA and the drug–ct-DNA complex, respectively. <em>C</em><sub>TAF</sub> corresponds to the concentration of TAF.</p>
<p>The absorbance values were fitted to the Benesi–Hildebrand equation, and plots of <em>A</em><sub>0</sub>/(<em>A</em> − <em>A</em><sub>0</sub>) <em>versus</em> 1/[TAF] yielded linear relationships at different temperatures (298, 303, and 308 K) as presented in <a href="#fig3" class="usa-link">Fig. 3</a>. These linear plots confirm a 1 : 1 binding stoichiometry between TAF and ct-DNA. The calculated binding constants (<em>K</em><sub>b</sub>) were on the order of 10<sup>5</sup> M<sup>−1</sup> (<a href="#tab2" class="usa-link">Table 2</a>), which indicates a moderate-to-strong binding affinity, comparable to other known minor groove binders.<sup><a href="#cit7" class="usa-link" aria-describedby="cit7">7</a></sup> These values are lower than those typically reported for intercalators such as ethidium bromide (<em>K</em><sub>b</sub> ≈ 10<sup>6</sup> M<sup>−1</sup>),<sup><a href="#cit40" class="usa-link" aria-describedby="cit40">40,41</a></sup> further supporting the conclusion that TAF interacts with ct-DNA <em>via</em> minor groove binding.</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">Fig. 3. Plot of <em>A</em><sub>0</sub>/(<em>A</em> − <em>A</em><sub>0</sub>) against 1/<em>C</em><sub>TAF</sub> at different temperature settings (<em>C</em><sub>DNA</sub> = 3.0 × 10<sup>−5</sup> M), while <em>r</em> is the correlation coefficient.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376933_d5ra04723d-f3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/49c7944748f2/d5ra04723d-f3.jpg" loading="lazy" height="382" width="779" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><section class="tw xbox font-sm" id="tab2"><h4 class="obj_head">Table 2. Estimation of the binding constants (<em>K</em><sub>b</sub>) at three varied temperatures and thermodynamic parameters of ct-DNA–TAF complex.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
</colgroup>
<thead><tr>
<th rowspan="1" colspan="1">
<em>T</em> (K)</th>
<th rowspan="1" colspan="1">
<em>K</em>
<sub>b</sub> (M<sup>−1</sup>)</th>
<th rowspan="1" colspan="1">Δ<em>H</em>° (kJ mol<sup>−1</sup>)</th>
<th rowspan="1" colspan="1">Δ<em>S</em>° (J mol<sup>−1</sup> K<sup>−1</sup>)</th>
<th rowspan="1" colspan="1">Δ<em>G</em>°<a href="#tab2fna" class="usa-link"><sup>a</sup></a> (kJ mol<sup>−1</sup>)</th>
<th rowspan="1" colspan="1">Δ<em>G</em>°<a href="#tab2fnb" class="usa-link"><sup>b</sup></a> (kJ mol<sup>−1</sup>)</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="1" colspan="1">298</td>
<td align="center" rowspan="1" colspan="1">2.2 × 10<sup>5</sup>
</td>
<td rowspan="3" align="center" colspan="1">6.23</td>
<td rowspan="3" align="center" colspan="1">123.48</td>
<td align="center" rowspan="1" colspan="1">−30.57</td>
<td align="center" rowspan="1" colspan="1">−30.47</td>
</tr>
<tr>
<td rowspan="1" colspan="1">303</td>
<td align="center" rowspan="1" colspan="1">2.4 × 10<sup>5</sup>
</td>
<td align="center" rowspan="1" colspan="1">−31.17</td>
<td align="center" rowspan="1" colspan="1">−31.18</td>
</tr>
<tr>
<td rowspan="1" colspan="1">308</td>
<td align="center" rowspan="1" colspan="1">2.5 × 10<sup>5</sup>
</td>
<td align="center" rowspan="1" colspan="1">−31.80</td>
<td align="center" rowspan="1" colspan="1">−31.88</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tab2fna">
<sup>a</sup><p class="display-inline">Δ<em>G</em>° = <em>RT</em> ln <em>K</em><sub>b</sub>.</p>
</div>
<div class="fn" id="tab2fnb">
<sup>b</sup><p class="display-inline">Δ<em>G</em>° = Δ<em>H</em>° − <em>T</em>Δ<em>S</em>°.</p>
</div>
</div></section><p>These spectroscopic observations, along with the calculated binding constants and absence of major spectral shifts, suggest that the interaction between TAF and ct-DNA is characterized by a specific and stable minor groove binding mode, rather than intercalation or electrostatic attraction. This conclusion is consistent with the findings from viscosity studies, fluorescence displacement assays, and molecular docking simulations discussed in the following sections.</p></section><section id="sec16"><h3 class="pmc_sec_title">3.2. Viscosity measurements</h3>
<p>While UV-vis spectroscopy provides initial insight into the mode of interaction, it alone cannot definitively distinguish between groove binding and intercalation. Therefore, to further validate the proposed non-intercalative (groove) binding of TAF to ct-DNA, its effect on the viscosity of DNA solutions was investigated.<sup><a href="#cit18" class="usa-link" aria-describedby="cit18">18</a></sup> Viscosity measurements are a classical method for probing DNA structural changes: intercalative binding increases viscosity by unwinding the helix to accommodate small molecules, while non-classical intercalation reduces viscosity by twisting and shortening the helix, whereas groove binding or electrostatic interactions typically result in negligible changes.<sup><a href="#cit15" class="usa-link" aria-describedby="cit15">15,17,18,42</a></sup> Hence, viscosity studies serve as a complementary tool to clarify the binding mode suggested by spectroscopic analysis.</p>
<p>The findings indicated that the relative specific viscosity (<em>η</em>/<em>η</em><sub>0</sub>)<sup>1/3</sup> values of ct-DNA largely remained stable with rising TAF concentrations (<a href="#fig4" class="usa-link">Fig. 4</a>), implying that the interaction between TAF and ct-DNA occurs through a groove binding mechanism. This aligns with the results of spectrophotometric analyses.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">Fig. 4. Influence of variable concentrations of TAF (0–15.0 μM) on the viscosity of ct-DNA (30.0 μM) in Tris–HCl buffer.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376933_d5ra04723d-f4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/3a79004137a1/d5ra04723d-f4.jpg" loading="lazy" height="461" width="779" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec17"><h3 class="pmc_sec_title">3.3. Competitive spectrofluorimetric study for determination of binding sites</h3>
<p>This method utilizes well-characterized DNA-binding probes, such as rhodamine B (RB) or ethidium bromide (EB), which exhibit a notable fluorescence increase upon interaction with DNA.<sup><a href="#cit43" class="usa-link" aria-describedby="cit43">43,44</a></sup> Studies have shown that EB binds by intercalating between DNA base pairs, whereas RB primarily associates with the minor groove of DNA, favoring AT-rich sequences.<sup><a href="#cit16" class="usa-link" aria-describedby="cit16">16,45</a></sup></p>
<p>This study examined how TAF interacts with DNA when two different fluorescent probes (EB and RB) are already bound to the DNA. When TAF was added to the DNA–EB complex, there was little change in the fluorescence of the probe, indicating that the drug and EB do not compete for binding sites on the DNA. However, when TAF was added to the DNA–RB complex, the fluorescence of RB gradually decreased. This suggests that the studied drug competes with RB for binding to the smaller groove of the DNA molecule.</p>
<p>These findings support the idea that TAF binds to DNA by fitting into the smaller groove, rather than inserting itself between the DNA base pairs. This assumption is consistent too with the findings obtained from the spectrophotometric and viscosity studies (<a href="#fig5" class="usa-link">Fig. 5</a>).</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">Fig. 5. (A) Fluorescence emission spectra of the RB–ct-DNA complex in presence and absence of TAF at 298 K. <em>C</em>(ct-DNA): 30.0 μM; <em>C</em>(RB): 2.0 × 10<sup>−3</sup> M; <em>C</em>(TAF): (0 → 5): 0, 3.0, 6.0, 9.0, 12.0, 15.0 μM (<em>λ</em><sub>ex</sub>/<em>λ</em><sub>em</sub> = 465/577 nm). (B) Fluorescence emission spectra of the EB–ct-DNA complex in presence and absence of TAF at 298 K. <em>C</em>(ct-DNA): 30.0 μM; <em>C</em>(EB): 1.2 × 10<sup>−3</sup> M; <em>C</em>(TAF) (0 → 5): 0, 3.0, 6.0, 9.0, 12.0, 15.0 μM (<em>λ</em><sub>ex</sub>/<em>λ</em><sub>em</sub> = 525/602 nm).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376933_d5ra04723d-f5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/3332d5f7d247/d5ra04723d-f5.jpg" loading="lazy" height="799" width="779" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>These results not only reinforce the hypothesis of TAF binding in the minor groove but also indicate a preferential interaction with AT-rich sequences, consistent with the known binding preferences of groove-targeting ligands.</p></section><section id="sec18"><h3 class="pmc_sec_title">3.4. Influence of ionic strength</h3>
<p>The strength of electrical forces between molecules is significantly weakened in environments with high salt concentrations, like those found in living organisms. To determine if these forces played a part in the interaction between a ligand (TAF) and DNA (ct-DNA), NaCl concentration was increased from 0 to 0.07 M, as illustrated in <a href="#fig6" class="usa-link">Fig. 6</a>. It was found that the ct-DNA–TAF complex's absorbance value was somewhat steady signifying that electrical forces are not the primary factor driving this interaction. Instead, the data indicates that TAF is likely binding to a specific groove within the DNA structure.<sup><a href="#cit46" class="usa-link" aria-describedby="cit46">46</a></sup></p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">Fig. 6. Effect of NaCl ionic strength on the absorbance of ct-DNA–TAF complex. Concentrations of TAF and ct-DNA were 0.9 × 10<sup>−5</sup> M and 3.0 × 10<sup>−5</sup> M, respectively. NaCl concentration was raising from 0 to 0.07 M.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376933_d5ra04723d-f6.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/16b11aff0b2c/d5ra04723d-f6.jpg" loading="lazy" height="434" width="779" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec19"><h3 class="pmc_sec_title">3.5. Evaluation of thermodynamic parameters and major interaction forces</h3>
<p>Hydrogen bonding, electrostatic interactions, hydrophobic forces, and van der Waals forces are the four primary types of non-covalent interactions known to mediate binding between biomacromolecules and small molecules. Hence, these forces play a critical role in deciding the specificity, stability, and strength of molecular interactions.<sup><a href="#cit47" class="usa-link" aria-describedby="cit47">47</a></sup> Additionally, one can deduce the type of binding forces by examining both the sign and value of entropy (Δ<em>S</em>°) and enthalpy (Δ<em>H</em>°) values. It is commonly considered that van der Waals force and/or hydrogen bonding are the fundamental contact forces when both Δ<em>S</em>° and Δ<em>H</em>° are negative. The main attraction is hydrophobic when both Δ<em>S</em>° and Δ<em>H</em>° are positive, or electrostatic when Δ<em>S</em>° is positive and Δ<em>H</em>° is close to zero.<sup><a href="#cit48" class="usa-link" aria-describedby="cit48">48,49</a></sup> The thermodynamic parameters, such as Gibbs free energy change (Δ<em>G</em>°), Δ<em>S</em>°, and Δ<em>H</em>° in the interaction of TAF with ct-DNA were computed using Van't Hoff <a href="#eqn2" class="usa-link">eqn (2)</a> and <a href="#eqn3" class="usa-link">(3)</a>:<sup><a href="#cit49" class="usa-link" aria-describedby="cit49">49</a></sup></p>
<table class="disp-formula p" id="eqn2"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/718a54a75d2b/d5ra04723d-t2.jpg" loading="lazy" id="ugt2" height="208" width="658" alt="3.5."></td>
<td class="label">2</td>
</tr></table>
<table class="disp-formula p" id="eqn3"><tr>
<td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML">Δ<em>G</em>° = Δ<em>H</em>° − <em>T</em>Δ<em>S</em>°</span></td>
<td class="label">3</td>
</tr></table>
<p>where, <em>R</em> represents a gas constant.</p>
<p>Using the slope and intercept of the Van't Hoff plot of ln <em>K</em><sub>b</sub><em>vs.</em> 1/<em>T</em> (<a href="#fig7" class="usa-link">Fig. 7</a>), the values of Δ<em>H</em>° and Δ<em>S</em>° were computed and are shown in <a href="#tab2" class="usa-link">Table 2</a>. It is found that there is a spontaneous binding interaction between ct-DNA and TAF since Δ<em>G</em>° is less than zero. From the obtained results, the positive Δ<em>H</em>° and Δ<em>S</em>° values suggest that hydrophobic interactions dominate the binding process. Such interactions are typical of minor groove binders, which often rely on shape complementarity and van der Waals forces, rather than classical ionic or stacking interactions. This thermodynamic profile supports the fluorescence and docking results and contributes to the overall understanding of TAF's biophysical behavior.</p>
<figure class="fig xbox font-sm" id="fig7"><h4 class="obj_head">Fig. 7. Van't Hoff plot for the ct-DNA–TAF complex.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376933_d5ra04723d-f7.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/eb278fc55033/d5ra04723d-f7.jpg" loading="lazy" height="427" width="779" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>While the spectroscopic and thermodynamic data collectively suggest that TAF interacts with ct-DNA primarily through a minor groove binding mode driven by hydrophobic forces, these techniques cannot directly reveal the precise atomic-level interactions or spatial conformation of the ligand within the DNA structure. Therefore, to gain deeper molecular insight into the binding mechanism, computational approaches were employed, including molecular docking and molecular dynamics (MD) simulations. These theoretical tools complement the experimental findings by visualizing the TAF–DNA complex, identifying specific interaction sites, and assessing the dynamic behavior and stability of the complex over time.</p></section><section id="sec20"><h3 class="pmc_sec_title">3.6. Molecular modeling</h3>
<p>Docking studies of TAF with two calf thymus B-DNA sequences, identified by PDB IDs 3EY0 and 1D29, provide key insights into its binding behavior and interaction<sup><a href="#cit7" class="usa-link" aria-describedby="cit7">7,9,10</a></sup> (<a href="#tab3" class="usa-link">Table 3</a>). The 3D and 2D visualization of the TAF–calf thymus DNA complex is provided in <a href="#fig8" class="usa-link">Fig. 8</a> and <a href="#fig9" class="usa-link">9</a>. The binding affinities were determined to be −7.75 kcal mol<sup>−1</sup> for 1D29 and −7.61 kcal mol<sup>−1</sup> for 3EY0, indicating a slightly stronger interaction with the B-DNA represented by 1D29. Tafamidis demonstrates conformational flexibility due to three active torsional bonds namely C9–C11, C19–C20, and C20–O21. These torsional movements enable the molecule to adapt its conformation to fit the curvature of the DNA minor groove, ensuring stable and specific binding. This alignment is essential for establishing effective hydrogen bonds and hydrophobic interactions with DNA bases, contributing to the stability of the TAF–DNA complexes. For 1D29, hydrogen bonds are formed with adenine residues DA6 and DA5, while hydrophobic interactions include pi–alkyl bonds with DT19 and DT8 and a C–H bond with DT20. Similarly, in 3EY0, hydrogen bonds are observed with DA5 and DA7, while hydrophobic interactions include a C–H bond with DT4 and pi–alkyl bonds with DA3 and DA9.</p>
<section class="tw xbox font-sm" id="tab3"><h4 class="obj_head">Table 3. Binding affinity and interaction of TAF with calf thymus DNA sequence.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
</colgroup>
<thead>
<tr>
<th rowspan="2" colspan="1">PDB ID</th>
<th rowspan="2" colspan="1">Binding affinity (kcal mol<sup>−1</sup>)</th>
<th colspan="2" rowspan="1">Interaction sites</th>
</tr>
<tr>
<th rowspan="1" colspan="1">Hydrogen bonds</th>
<th rowspan="1" colspan="1">Hydrophobic bonds</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="1" colspan="1">
1D29
</td>
<td align="center" rowspan="1" colspan="1">−7.75</td>
<td rowspan="1" colspan="1">DA6, DA5</td>
<td rowspan="1" colspan="1">DT19, DT8 (pi–alkyl), DT20 (C–H bond)</td>
</tr>
<tr>
<td rowspan="1" colspan="1">
3EY0
</td>
<td align="center" rowspan="1" colspan="1">−7.61</td>
<td rowspan="1" colspan="1">DA5, DA7</td>
<td rowspan="1" colspan="1">DT4 (C–H bond), DA3, DA9 (pi–alkyl)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="fig8"><h4 class="obj_head">Fig. 8. 3D visualization of tafamidis–calf thymus DNA complex.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376933_d5ra04723d-f8.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/c11e2f9aecf8/d5ra04723d-f8.jpg" loading="lazy" height="644" width="779" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><figure class="fig xbox font-sm" id="fig9"><h4 class="obj_head">Fig. 9. 2D visualization of tafamidis–calf thymus DNA complex.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376933_d5ra04723d-f9.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/6c8bfbd1d18e/d5ra04723d-f9.jpg" loading="lazy" height="353" width="708" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The estimated binding free energies (Δ<em>G</em>) are consistent with the docking results. The intermolecular energy (<em>E</em><sub>1</sub>), representing contributions from van der Waals forces, hydrogen bonds, desolvation, and electrostatic interactions, is −8.64 kcal mol<sup>−1</sup> for 1D29 and −8.50 kcal mol<sup>−1</sup> for 3EY0, indicating favorable interactions. Internal strain energy (<em>E</em><sub>2</sub>) values are minimal (−0.28 kcal mol<sup>−1</sup> for 1D29 and −0.29 kcal mol<sup>−1</sup> for 3EY0), suggesting minor conformational changes upon binding. The torsional free energy (<em>E</em><sub>3</sub>) remains constant at 0.89 kcal mol<sup>−1</sup>, while the unbound system energy (<em>E</em><sub>4</sub>) values are identical for both complexes (−0.28 kcal mol<sup>−1</sup> and −0.29 kcal mol<sup>−1</sup>). The binding free energies exhibited by the TAF–DNA complex are provided in <a href="#tab4" class="usa-link">Table 4</a>. These findings highlight the ligand's ability to establish stable and energetically favorable interactions with both B-DNA sequences.</p>
<section class="tw xbox font-sm" id="tab4"><h4 class="obj_head">Table 4. Theoretical binding energies of the TAF with calf thymus DNA sequence complex involved in docking<a href="#tab4fna" class="usa-link"><sup>a</sup></a>.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
</colgroup>
<thead><tr>
<th rowspan="1" colspan="1">PDB ID</th>
<th rowspan="1" colspan="1">Δ<em>G</em> (kcal mol<sup>−1</sup>)</th>
<th rowspan="1" colspan="1">
<em>E</em>
<sub>1</sub> (kcal mol<sup>−1</sup>)</th>
<th rowspan="1" colspan="1">
<em>E</em>
<sub>2</sub> (kcal mol<sup>−1</sup>)</th>
<th rowspan="1" colspan="1">
<em>E</em>
<sub>3</sub> (kcal mol<sup>−1</sup>)</th>
<th rowspan="1" colspan="1">
<em>E</em>
<sub>4</sub> (kcal mol<sup>−1</sup>)</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="1" colspan="1">
1D29
</td>
<td align="center" rowspan="1" colspan="1">−7.75</td>
<td align="center" rowspan="1" colspan="1">−8.64</td>
<td align="center" rowspan="1" colspan="1">−0.28</td>
<td align="center" rowspan="1" colspan="1">0.89</td>
<td align="center" rowspan="1" colspan="1">−0.28</td>
</tr>
<tr>
<td rowspan="1" colspan="1">
3EY0
</td>
<td align="center" rowspan="1" colspan="1">−7.61</td>
<td align="center" rowspan="1" colspan="1">−8.50</td>
<td align="center" rowspan="1" colspan="1">−0.29</td>
<td align="center" rowspan="1" colspan="1">0.89</td>
<td align="center" rowspan="1" colspan="1">−0.29</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tab4fna">
<sup>a</sup><p class="display-inline">Δ<em>G</em> – total binding free energy estimated (<em>E</em><sub>1</sub> + <em>E</em><sub>2</sub> + <em>E</em><sub>3</sub> − <em>E</em><sub>4</sub>); <em>E</em><sub>1</sub> – total intermolecular energy (sum of van der Waals, hydrogen bond, desolvation, electrostatic energies); <em>E</em><sub>2</sub> – total internal energy; <em>E</em><sub>3</sub> – torsional free energy; <em>E</em><sub>4</sub> – unbound system energy.</p>
</div></div></section><p>Although this study did not directly assess the biological consequences of TAF binding to ct-DNA, the observed minor groove binding to AT-rich regions may have implications for gene regulation and DNA-associated processes. Minor groove binders are known to interfere with transcription factor binding, RNA polymerase progression, or replication machinery, depending on their binding location and affinity. Given TAF's preferential interaction with DNA without major structural distortion, it is plausible that subtle regulatory interference could occur at specific genomic loci. Further studies using cell-based models, gene expression profiling, or DNA polymerase assays would be necessary to determine whether TAF impacts gene transcription or replication under physiological conditions.<sup><a href="#cit50" class="usa-link" aria-describedby="cit50">50</a></sup></p></section><section id="sec21"><h3 class="pmc_sec_title">3.7. Molecular dynamic simulations</h3>
<p>To evaluate the binding interactions of TAF with B-DNA (PDB ID: 1D29), molecular dynamics (MD) simulations were established for 100 nanoseconds.<sup><a href="#cit8" class="usa-link" aria-describedby="cit8">8,9</a></sup> The simulation data was analyzed utilizing different statistical values, including Root-Mean-Square Deviation (RMSD), Root-Mean-Square Fluctuation (RMSF), hydrogen bond interactions, the percentage occupancies of these interactions, radius of gyration (<em>R</em><sub>g</sub>), and Solvent Accessible Surface Area (SASA) over the simulation period.</p>
<section id="sec22"><h4 class="pmc_sec_title">3.7.1. RMSD analysis</h4>
<p>The RMSD analysis offered crucial insights into the structural stability of the DNA–ligand complex. As illustrated in <a href="#fig10" class="usa-link">Fig. 10</a>, the RMSD values of the DNA remained consistent throughout the simulation, ranging between 0.2 and 0.4 nm, suggesting minimal structural deviations and a stable configuration. The ligand's RMSD was also analyzed to determine its binding stability to the DNA. The ligand exhibited stable RMSD values between 0.025 and 0.1 nm, reflecting a strong and reliable interaction with the B-DNA of calf thymus. A minor fluctuation was observed at 72 ns, where the RMSD increased to 0.75 nm, indicating slight conformational adjustments to accommodate the minor groove of the DNA.</p>
<figure class="fig xbox font-sm" id="fig10"><h5 class="obj_head">Fig. 10. Graphical representation of the plots displaying DNA RMSD (nm) against time (100 ns) for tafamidis–calf thymus DNA (1D29) complex.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376933_d5ra04723d-f10.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/f8a43089cc4b/d5ra04723d-f10.jpg" loading="lazy" height="555" width="779" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec23"><h4 class="pmc_sec_title">3.7.2. RMSF analysis</h4>
<p>The RMSF analysis, presented in <a href="#fig11" class="usa-link">Fig. 11</a>, provided details on localized fluctuations within the DNA structure. The plot showed that most residues fluctuated within the range of 0.1 to 0.3 nm. However, residues around positions 11 to 14 displayed slightly higher fluctuations, peaking at 0.35 nm. These findings highlight the overall structural stability of the B-DNA during the simulation, with localized flexibility in specific regions.</p>
<figure class="fig xbox font-sm" id="fig11"><h5 class="obj_head">Fig. 11. Graphical representation of the plots displaying the DNA RMSF (nm) <em>versus</em> residue index number of tafamidis–calf thymus DNA (3EY0) complex.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376933_d5ra04723d-f11.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/6ea415799c56/d5ra04723d-f11.jpg" loading="lazy" height="598" width="779" alt="Fig. 11"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec24"><h4 class="pmc_sec_title">3.7.3. H-bond interaction</h4>
<p>Hydrogen bond interactions, a vital factor in the stability of the DNA–ligand complex, were also evaluated. As shown in <a href="#fig12" class="usa-link">Fig. 12</a>, TAF consistently formed an average of two hydrogen bonds with the DNA throughout the simulation. The percentage occupancies of these hydrogen bonds, detailed in <a href="#fig13" class="usa-link">Fig. 13</a>, revealed that TAF exhibited the strongest interaction with residue DG4, with an occupancy of 24.20%. The significant interactions were also observed with adenine residues DA5 and DA6, showing occupancies of 12.04% and 9.69%, respectively, and thymine residue DT7, with an occupancy of 9.25%. These findings confirm the formation of stable hydrogen bonds these residues during the simulation, emphasizing TAF's strong and stable binding to the AT-region of DNA.</p>
<figure class="fig xbox font-sm" id="fig12"><h5 class="obj_head">Fig. 12. Pictorial representation of the number of H-bond contacts formed by tafamidis–calf thymus DNA (1D29) complex.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376933_d5ra04723d-f12.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/7ddaea44040a/d5ra04723d-f12.jpg" loading="lazy" height="682" width="779" alt="Fig. 12"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig12/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><figure class="fig xbox font-sm" id="fig13"><h5 class="obj_head">Fig. 13. Histogram representation of % occupancies of the H-bond DNA–ligand contacts of tafamidis–calf thymus DNA (1D29) complex.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376933_d5ra04723d-f13.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/61d86fbb191a/d5ra04723d-f13.jpg" loading="lazy" height="335" width="661" alt="Fig. 13"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig13/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec25"><h4 class="pmc_sec_title">3.7.4. MMGBSA analysis</h4>
<p>The binding free energy Δ<em>G</em> values for the TAF–calf thymus DNA complex (PDB ID: 1D29) were analyzed at different time points during the molecular dynamics simulation (0 ns, 50 ns, and 100 ns) to evaluate the stability and strength of the interaction (<a href="#tab5" class="usa-link">Table 5</a> and <a href="#fig14" class="usa-link">Fig. 14</a>). At the beginning of the simulation (0 ns), the complex exhibited a binding free energy of −36.65 kcal mol<sup>−1</sup>, signifying a stable interaction. By 50 ns, the binding free energy significantly decreased to −69.691 kcal mol<sup>−1</sup>, reflecting a stronger and more favorable interaction, likely due to conformational adjustments optimizing the ligand's fit within the DNA's minor groove. At 100 ns, the binding free energy increased slightly to −51.907 kcal mol<sup>−1</sup>, suggesting some conformational relaxation while maintaining a strong binding interaction. The average binding free energy over the simulation period was −52.75 kcal mol<sup>−1</sup>, confirming a consistently favourable and stable interaction between TAF and the DNA. These results demonstrate the ligand's ability to adapt its conformation over time to maintain robust binding with the DNA target.</p>
<section class="tw xbox font-sm" id="tab5"><h5 class="obj_head">Table 5. MMGBSA Δ<em>G</em> binding energy calculations for TAF–calf thymus DNA (3EY0) complex.</h5>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
</colgroup>
<thead><tr>
<th rowspan="1" colspan="1">Ligand–DNA complex</th>
<th rowspan="1" colspan="1">0 ns</th>
<th rowspan="1" colspan="1">50 ns</th>
<th rowspan="1" colspan="1">100 ns</th>
<th rowspan="1" colspan="1">Average</th>
</tr></thead>
<tbody><tr>
<td rowspan="1" colspan="1">TAF–calf thymus DNA complex (1D29)</td>
<td align="center" rowspan="1" colspan="1">−36.65</td>
<td align="center" rowspan="1" colspan="1">−69.691</td>
<td align="center" rowspan="1" colspan="1">−51.907</td>
<td align="center" rowspan="1" colspan="1">−52.75</td>
</tr></tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="fig14"><h5 class="obj_head">Fig. 14. MMGBSA Δ<em>G</em> binding energy calculations for tafamidis–calf thymus DNA (3EY0) complex.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376933_d5ra04723d-f14.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/1edb1f1bb020/d5ra04723d-f14.jpg" loading="lazy" height="465" width="779" alt="Fig. 14"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig14/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec26"><h4 class="pmc_sec_title">3.7.5. Radius of gyration (<em>R</em><sub>g</sub>)</h4>
<p>The plot of the radius of gyration (<em>R</em><sub>g</sub>) <em>versus</em> time provides insights into the structural compactness and stability of the TAF–calf thymus DNA complex during the 100 ns molecular dynamics simulation (<a href="#fig15" class="usa-link">Fig. 15</a>). The <em>R</em><sub>g</sub> values range between approximately 1.32 nm and 1.48 nm, indicating consistent structural integrity. Initially, the <em>R</em><sub>g</sub> values remain stable with minor fluctuations, suggesting that the complex maintains its compactness. However, towards the latter part of the simulation (after 80 ns), there is a slight increase in <em>R</em><sub>g</sub>, indicating a minor expansion of the complex. This could reflect small conformational changes within the DNA–ligand complex as it adapts to stabilize the interaction. Overall, the relatively stable <em>R</em><sub>g</sub> values throughout the simulation highlight the structural stability and compact nature of the complex.</p>
<figure class="fig xbox font-sm" id="fig15"><h5 class="obj_head">Fig. 15. Radius of gyration for tafamidis–calf thymus DNA (1D29) complex.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376933_d5ra04723d-f15.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/3541efbe142d/d5ra04723d-f15.jpg" loading="lazy" height="590" width="779" alt="Fig. 15"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig15/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec27"><h4 class="pmc_sec_title">3.7.6. Solvent accessible surface area (SASA)</h4>
<p>The plot of Solvent Accessible Surface Area (SASA) <em>versus</em> time provides insights into the exposure of the TAF–calf thymus DNA complex to the solvent during the 100 ns molecular dynamics simulation (<a href="#fig16" class="usa-link">Fig. 16</a>). The SASA values range between approximately 46 nm<sup>2</sup> and 51 nm<sup>2</sup>, indicating moderate fluctuations in solvent exposure. Initially, the SASA values fluctuate around 48–50 nm<sup>2</sup>, suggesting that the complex maintains a consistent level of solvent exposure. After 60 ns, there are brief increases and decreases in SASA, reflecting minor conformational adjustments in the complex that temporarily alter the surface area exposed to the solvent. These variations could be attributed to the ligand's movement or slight structural changes in the DNA as it stabilizes the interaction. Overall, the SASA values remain within a narrow range, demonstrating that the TAF–DNA complex maintains its overall structural integrity and interaction stability throughout the simulation, with only minor fluctuations in solvent exposure.</p>
<figure class="fig xbox font-sm" id="fig16"><h5 class="obj_head">Fig. 16. Solvent accessible surface area for tafamidis–calf thymus DNA (1D29) complex.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12376933_d5ra04723d-f16.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/e15b/12376933/e0a795cdba61/d5ra04723d-f16.jpg" loading="lazy" height="581" width="779" alt="Fig. 16"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig16/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Overall, the MD simulation results confirm the stability and persistence of TAF within the minor groove, highlighting its ability to form stable hydrogen bonds and hydrophobic interactions with AT-rich regions of DNA. This dynamic behavior further validates the experimental conclusions and reveals a consistent binding pattern across methodologies.</p>
<p>Regarding the mechanism of DNA conformational change after TAF binding to ct-DNA, the binding of TAF to ct-DNA does not cause major structural distortion, as evidenced by the viscosity measurements showing negligible changes in the DNA's hydrodynamic length. This supports a minor groove binding mode, which typically induces minimal helical unwinding or extension. Molecular dynamics simulations further clarify the conformational impact: RMSD and RMSF values indicate that the DNA retains overall structural stability with only localized fluctuations, particularly around the binding region (residues 11–14). The radius of gyration (<em>R</em><sub>g</sub>) shows a slight increase toward the end of the simulation, which may reflect minor relaxation or groove widening to accommodate the ligand. Additionally, SASA analysis suggests stable solvent exposure throughout the binding, with no drastic changes in DNA compactness. Together, these findings suggest that TAF binding induces subtle local conformational changes (<em>e.g.</em>, minor groove widening or base pair tilting) without disrupting the overall B-form of DNA.</p></section></section></section><section id="sec28"><h2 class="pmc_sec_title">4. Study advantages, limitations, and future perspectives</h2>
<p>This study provides the first comprehensive investigation of the interaction between TAF and ct-DNA, combining multi-spectroscopic, thermodynamic, and computational modeling techniques. A one major strength of this work is its integrated approach, which enables detailed characterization of the binding mode, interaction forces, and dynamic stability of the TAF–DNA complex. The use of molecular docking and dynamics simulations, supported by experimental data, offers valuable structural insights at the atomic level. From a clinical perspective, our findings may help inform the off-target interaction profile of TAF, particularly in the context of chronic exposure. The observed moderate, minor groove binding to DNA without major structural disruption suggests low genotoxic potential, but also raises important considerations for drug safety and pharmacodynamics. Additionally, understanding such interactions can support regulatory evaluation, especially when considering analogues or modifications of TAF. We believe that this study provides clinically relevant mechanistic insight into TAF behavior at the molecular level, which may be useful in toxicological risk assessment, long-term safety monitoring, and the rational design of next-generation transthyretin stabilizers.</p>
<p>However, the study has some limitations. It should be noted that the present study is limited to equilibrium binding analysis. The kinetics of TAF–DNA interaction (<em>i.e.</em>, association and dissociation rate constants) were not assessed. Future investigations using time-resolved spectroscopic or biosensor-based techniques (<em>e.g.</em>, surface plasmon resonance) could provide valuable information on the dynamics of this interaction and further elucidate its biological relevance. Moreover, although MD simulations offered detailed insight into the structural dynamics of the TAF–DNA complex, these predictions were not experimentally validated using direct conformational analysis tools such as circular dichroism (CD) spectroscopy or X-ray crystallography. Future work incorporating these techniques could provide experimental confirmation of the localized groove widening and hydrogen-bonding interactions inferred from simulation data, and further clarify the biological significance of TAF–DNA binding. In addition, although TAF is not known to exhibit genotoxic effects clinically, its observed binding to DNA, even through minor groove interaction, suggests that chronic exposure scenarios could merit additional study. Future investigations including genotoxicity assays, DNA repair evaluations, or <em>in vivo</em> long-term studies may help clarify whether TAF poses any risk to DNA integrity under repeat dosing conditions.</p></section><section id="sec29"><h2 class="pmc_sec_title">5. Conclusion</h2>
<p>The present study offers the first detailed analysis of the interaction between TAF, a clinically relevant drug for transthyretin amyloidosis, and ct-DNA using a comprehensive multimodal approach. UV-vis spectroscopy revealed a hyperchromic effect without a shift in absorption maximum, suggesting a non-intercalative binding mode. This was further supported by viscosity measurements, which showed negligible changes in DNA length, and fluorescence displacement studies that indicated competition with RB but not EB, confirming minor groove binding behavior. The calculated binding constants (∼10<sup>5</sup> M<sup>−1</sup>) suggest a moderate-to-strong interaction with ct-DNA. Thermodynamic analysis revealed positive Δ<em>H</em>° and Δ<em>S</em>° values, indicating that hydrophobic interactions are the primary driving force, supported by the lack of salt-dependence in the binding. Molecular docking confirmed TAF's insertion into the DNA minor groove, establishing hydrogen bonds and hydrophobic contacts with AT-rich regions. Molecular dynamics simulations demonstrated the structural stability of the TAF–DNA complex, while MMGBSA calculations revealed consistent binding free energies throughout the simulation time. Collectively, the experimental and computational findings provide strong evidence that TAF binds to ct-DNA <em>via</em> a minor groove binding mechanism, primarily stabilized by hydrophobic and hydrogen interactions. These results offer deeper insights into the molecular behavior of TAF beyond its known protein targets and may inform future studies on off-target DNA interactions, drug safety profiling, or the design of TAF analogues with enhanced biocompatibility or specificity.</p></section><section id="sec30"><h2 class="pmc_sec_title">Conflicts of interest</h2>
<p>The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.</p></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors extend their appreciation to Prince Sattam bin Abdulaziz University for funding this research work through the project number (PSAU/2023/01/9020).</p></section><section id="sec31"><h2 class="pmc_sec_title">Data availability</h2>
<p>The datasets generated and/or analyzed during the current study are available from the corresponding author on reasonable request.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="cit1">
<cite>Bulawa C. E. Connelly S. DeVit M. Wang L. Weigel C. Fleming J. A. Packman J. Powers E. T. Wiseman R. L. Foss T. R. Tafamidis, a potent and selective transthyretin kinetic stabilizer that inhibits the amyloid cascade. Proc. Natl. Acad. Sci. U. S. A. 2012;109:9629–9634. doi: 10.1073/pnas.1121005109.</cite> [<a href="https://doi.org/10.1073/pnas.1121005109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3386102/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22645360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&amp;title=Tafamidis,%20a%20potent%20and%20selective%20transthyretin%20kinetic%20stabilizer%20that%20inhibits%20the%20amyloid%20cascade&amp;author=C.%20E.%20Bulawa&amp;author=S.%20Connelly&amp;author=M.%20DeVit&amp;author=L.%20Wang&amp;author=C.%20Weigel&amp;volume=109&amp;publication_year=2012&amp;pages=9629-9634&amp;pmid=22645360&amp;doi=10.1073/pnas.1121005109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit2">
<cite>Nawarskas J. J. Shephard E. A. Tafamidis: a novel treatment for transthyretin amyloid cardiomyopathy. Cardiol. Rev. 2020;28:156–160. doi: 10.1097/CRD.0000000000000306.</cite> [<a href="https://doi.org/10.1097/CRD.0000000000000306" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32101907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cardiol.%20Rev.&amp;title=Tafamidis:%20a%20novel%20treatment%20for%20transthyretin%20amyloid%20cardiomyopathy&amp;author=J.%20J.%20Nawarskas&amp;author=E.%20A.%20Shephard&amp;volume=28&amp;publication_year=2020&amp;pages=156-160&amp;pmid=32101907&amp;doi=10.1097/CRD.0000000000000306&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit3"><cite>
National Center for Biotechnology Information, PubChem Compound Summary for CID 24970412, Tafamidis Meglumine, <a href="https://pubchem.ncbi.nlm.nih.gov/compound/Tafamidis-Meglumine" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubchem.ncbi.nlm.nih.gov/compound/Tafamidis-Meglumine</a>, accessed 9 February, 2025</cite></li>
<li id="cit4">
<cite>Chen Q.-y. Li D.-h. Yang H.-h. Zhu Q.-z. Xu J.-g. Zhao Y. Interaction of a novel red-region fluorescent probe, Nile Blue, with DNA and its application to nucleic acids assay. Analyst. 1999;124:901–906. doi: 10.1039/a901174i.</cite> [<a href="https://doi.org/10.1039/a901174i" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10736873/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analyst&amp;title=Interaction%20of%20a%20novel%20red-region%20fluorescent%20probe,%20Nile%20Blue,%20with%20DNA%20and%20its%20application%20to%20nucleic%20acids%20assay&amp;author=Q.-y.%20Chen&amp;author=D.-h.%20Li&amp;author=H.-h.%20Yang&amp;author=Q.-z.%20Zhu&amp;author=J.-g.%20Xu&amp;volume=124&amp;publication_year=1999&amp;pages=901-906&amp;pmid=10736873&amp;doi=10.1039/a901174i&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit5">
<cite>Erkkila K. E. Odom D. T. Barton J. K. Recognition and reaction of metallointercalators with DNA. Chem. Rev. 1999;99:2777–2796. doi: 10.1021/cr9804341.</cite> [<a href="https://doi.org/10.1021/cr9804341" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11749500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Rev.&amp;title=Recognition%20and%20reaction%20of%20metallointercalators%20with%20DNA&amp;author=K.%20E.%20Erkkila&amp;author=D.%20T.%20Odom&amp;author=J.%20K.%20Barton&amp;volume=99&amp;publication_year=1999&amp;pages=2777-2796&amp;pmid=11749500&amp;doi=10.1021/cr9804341&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit6">
<cite>Barone G. Terenzi A. Lauria A. Almerico A. M. Leal J. M. Busto N. Garcia B. DNA-binding of nickel(II), copper(II) and zinc(II) complexes: Structure–affinity relationships. Coord. Chem. Rev. 2013;257:2848–2862.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Coord.%20Chem.%20Rev.&amp;title=DNA-binding%20of%20nickel(II),%20copper(II)%20and%20zinc(II)%20complexes:%20Structure%E2%80%93affinity%20relationships&amp;author=G.%20Barone&amp;author=A.%20Terenzi&amp;author=A.%20Lauria&amp;author=A.%20M.%20Almerico&amp;author=J.%20M.%20Leal&amp;volume=257&amp;publication_year=2013&amp;pages=2848-2862&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit7">
<cite>Magdy G. Belal F. Hakiem A. F. A. Abdel-Megied A. M. Salmon sperm DNA binding study to cabozantinib, a tyrosine kinase inhibitor: Multi-spectroscopic and molecular docking approaches. Int. J. Biol. Macromol. 2021;182:1852–1862. doi: 10.1016/j.ijbiomac.2021.05.164.</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2021.05.164" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34062156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Biol.%20Macromol.&amp;title=Salmon%20sperm%20DNA%20binding%20study%20to%20cabozantinib,%20a%20tyrosine%20kinase%20inhibitor:%20Multi-spectroscopic%20and%20molecular%20docking%20approaches&amp;author=G.%20Magdy&amp;author=F.%20Belal&amp;author=A.%20F.%20A.%20Hakiem&amp;author=A.%20M.%20Abdel-Megied&amp;volume=182&amp;publication_year=2021&amp;pages=1852-1862&amp;pmid=34062156&amp;doi=10.1016/j.ijbiomac.2021.05.164&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit8">
<cite>Magdy G. Shaldam M. A. Belal F. Elmansi H. Multi-spectroscopic, thermodynamic, and molecular docking/dynamic approaches for characterization of the binding interaction between calf thymus DNA and palbociclib. Sci. Rep. 2022;12:14723. doi: 10.1038/s41598-022-19015-9.</cite> [<a href="https://doi.org/10.1038/s41598-022-19015-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9427788/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36042232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Multi-spectroscopic,%20thermodynamic,%20and%20molecular%20docking/dynamic%20approaches%20for%20characterization%20of%20the%20binding%20interaction%20between%20calf%20thymus%20DNA%20and%20palbociclib&amp;author=G.%20Magdy&amp;author=M.%20A.%20Shaldam&amp;author=F.%20Belal&amp;author=H.%20Elmansi&amp;volume=12&amp;publication_year=2022&amp;pages=14723&amp;pmid=36042232&amp;doi=10.1038/s41598-022-19015-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit9">
<cite>Radwan A. S. Salim M. M. Elkhoudary M. M. Hadad G. M. Shaldam M. A. Belal F. Magdy G. Study of the binding interaction of salmon sperm DNA with nintedanib, a tyrosine kinase inhibitor using multi-spectroscopic, thermodynamic, and in silico approaches. J. Biomol. Struct. Dyn. 2024;42:1170–1180. doi: 10.1080/07391102.2023.2202776.</cite> [<a href="https://doi.org/10.1080/07391102.2023.2202776" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37079322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20Struct.%20Dyn.&amp;title=Study%20of%20the%20binding%20interaction%20of%20salmon%20sperm%20DNA%20with%20nintedanib,%20a%20tyrosine%20kinase%20inhibitor%20using%20multi-spectroscopic,%20thermodynamic,%20and%20in%20silico%20approaches&amp;author=A.%20S.%20Radwan&amp;author=M.%20M.%20Salim&amp;author=M.%20M.%20Elkhoudary&amp;author=G.%20M.%20Hadad&amp;author=M.%20A.%20Shaldam&amp;volume=42&amp;publication_year=2024&amp;pages=1170-1180&amp;pmid=37079322&amp;doi=10.1080/07391102.2023.2202776&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit10">
<cite>Shaldam M. Tawfik H. Elmansi H. Belal F. Yamaguchi K. Sugiura M. Magdy G. Synthesis, crystallographic, DNA binding, and molecular docking/dynamic studies of a privileged chalcone-sulfonamide hybrid scaffold as a promising anticancer agent. J. Biomol. Struct. Dyn. 2023;41:8876–8890. doi: 10.1080/07391102.2022.2138551.</cite> [<a href="https://doi.org/10.1080/07391102.2022.2138551" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36310097/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20Struct.%20Dyn.&amp;title=Synthesis,%20crystallographic,%20DNA%20binding,%20and%20molecular%20docking/dynamic%20studies%20of%20a%20privileged%20chalcone-sulfonamide%20hybrid%20scaffold%20as%20a%20promising%20anticancer%20agent&amp;author=M.%20Shaldam&amp;author=H.%20Tawfik&amp;author=H.%20Elmansi&amp;author=F.%20Belal&amp;author=K.%20Yamaguchi&amp;volume=41&amp;publication_year=2023&amp;pages=8876-8890&amp;pmid=36310097&amp;doi=10.1080/07391102.2022.2138551&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit11">
<cite>Al-Karmalawy A. A. Elmaaty A. A. Magdy G. Radwan A. S. Alnajjar R. Shaldam M. A. Al Khatib A. O. Almujri S. S. Alzahrani A. Y. A. Tawfik H. O. Targeted synthesis of a trimethoxyphenyltetrahydropyrimidine analogue designed as a DNA intercalator: in silico, multi-spectroscopic, thermodynamic, and in vitro approaches. RSC Adv. 2025;15:14946–14965. doi: 10.1039/d5ra02179k.</cite> [<a href="https://doi.org/10.1039/d5ra02179k" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12061050/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40343313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Adv.&amp;title=Targeted%20synthesis%20of%20a%20trimethoxyphenyltetrahydropyrimidine%20analogue%20designed%20as%20a%20DNA%20intercalator:%20in%20silico,%20multi-spectroscopic,%20thermodynamic,%20and%20in%20vitro%20approaches&amp;author=A.%20A.%20Al-Karmalawy&amp;author=A.%20A.%20Elmaaty&amp;author=G.%20Magdy&amp;author=A.%20S.%20Radwan&amp;author=R.%20Alnajjar&amp;volume=15&amp;publication_year=2025&amp;pages=14946-14965&amp;pmid=40343313&amp;doi=10.1039/d5ra02179k&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit12">
<cite>Shi J. H. Chen J. Wang J. Zhu Y. Y. Binding interaction between sorafenib and calf thymus DNA: spectroscopic methodology, viscosity measurement and molecular docking. Spectrochim. Acta, Part A. 2015;136:443–450. doi: 10.1016/j.saa.2014.09.056.</cite> [<a href="https://doi.org/10.1016/j.saa.2014.09.056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25311519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Spectrochim.%20Acta,%20Part%20A&amp;title=Binding%20interaction%20between%20sorafenib%20and%20calf%20thymus%20DNA:%20spectroscopic%20methodology,%20viscosity%20measurement%20and%20molecular%20docking&amp;author=J.%20H.%20Shi&amp;author=J.%20Chen&amp;author=J.%20Wang&amp;author=Y.%20Y.%20Zhu&amp;volume=136&amp;publication_year=2015&amp;pages=443-450&amp;pmid=25311519&amp;doi=10.1016/j.saa.2014.09.056&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit13">
<cite>Shakibapour N. Dehghani Sani F. Beigoli S. Sadeghian H. Chamani J. Multi-spectroscopic and molecular modeling studies to reveal the interaction between propyl acridone and calf thymus DNA in the presence of histone H1: binary and ternary approaches. J. Biomol. Struct. Dyn. 2019;37:359–371. doi: 10.1080/07391102.2018.1427629.</cite> [<a href="https://doi.org/10.1080/07391102.2018.1427629" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29338579/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20Struct.%20Dyn.&amp;title=Multi-spectroscopic%20and%20molecular%20modeling%20studies%20to%20reveal%20the%20interaction%20between%20propyl%20acridone%20and%20calf%20thymus%20DNA%20in%20the%20presence%20of%20histone%20H1:%20binary%20and%20ternary%20approaches&amp;author=N.%20Shakibapour&amp;author=F.%20Dehghani%20Sani&amp;author=S.%20Beigoli&amp;author=H.%20Sadeghian&amp;author=J.%20Chamani&amp;volume=37&amp;publication_year=2019&amp;pages=359-371&amp;pmid=29338579&amp;doi=10.1080/07391102.2018.1427629&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit14">
<cite>Sirajuddin M. Ali S. Badshah A. Drug–DNA interactions and their study by UV–Visible, fluorescence spectroscopies and cyclic voltammetry. J. Photochem. Photobiol., B. 2013;124:1–19. doi: 10.1016/j.jphotobiol.2013.03.013.</cite> [<a href="https://doi.org/10.1016/j.jphotobiol.2013.03.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23648795/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Photochem.%20Photobiol.,%20B&amp;title=Drug%E2%80%93DNA%20interactions%20and%20their%20study%20by%20UV%E2%80%93Visible,%20fluorescence%20spectroscopies%20and%20cyclic%20voltammetry&amp;author=M.%20Sirajuddin&amp;author=S.%20Ali&amp;author=A.%20Badshah&amp;volume=124&amp;publication_year=2013&amp;pages=1-19&amp;pmid=23648795&amp;doi=10.1016/j.jphotobiol.2013.03.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit15">
<cite>Shi J.-H. Liu T.-T. Jiang M. Chen J. Wang Q. Characterization of interaction of calf thymus DNA with gefitinib: Spectroscopic methods and molecular docking. J. Photochem. Photobiol., B. 2015;147:47–55. doi: 10.1016/j.jphotobiol.2015.03.005.</cite> [<a href="https://doi.org/10.1016/j.jphotobiol.2015.03.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25839749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Photochem.%20Photobiol.,%20B&amp;title=Characterization%20of%20interaction%20of%20calf%20thymus%20DNA%20with%20gefitinib:%20Spectroscopic%20methods%20and%20molecular%20docking&amp;author=J.-H.%20Shi&amp;author=T.-T.%20Liu&amp;author=M.%20Jiang&amp;author=J.%20Chen&amp;author=Q.%20Wang&amp;volume=147&amp;publication_year=2015&amp;pages=47-55&amp;pmid=25839749&amp;doi=10.1016/j.jphotobiol.2015.03.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit16">
<cite>Islam M. M. Chakraborty M. Pandya P. Al Masum A. Gupta N. Mukhopadhyay S. Binding of DNA with Rhodamine B: Spectroscopic and molecular modeling studies. Dyes Pigm. 2013;99:412–422.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Dyes%20Pigm.&amp;title=Binding%20of%20DNA%20with%20Rhodamine%20B:%20Spectroscopic%20and%20molecular%20modeling%20studies&amp;author=M.%20M.%20Islam&amp;author=M.%20Chakraborty&amp;author=P.%20Pandya&amp;author=A.%20Al%20Masum&amp;author=N.%20Gupta&amp;volume=99&amp;publication_year=2013&amp;pages=412-422&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit17">
<cite>Chen K.-Y. Zhou K.-L. Lou Y.-Y. Shi J.-H. Exploring the binding interaction of calf thymus DNA with lapatinib, a tyrosine kinase inhibitor: Multi-spectroscopic techniques combined with molecular docking. J. Biomol. Struct. Dyn. 2019;37:576–583. doi: 10.1080/07391102.2018.1433067.</cite> [<a href="https://doi.org/10.1080/07391102.2018.1433067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29375016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20Struct.%20Dyn.&amp;title=Exploring%20the%20binding%20interaction%20of%20calf%20thymus%20DNA%20with%20lapatinib,%20a%20tyrosine%20kinase%20inhibitor:%20Multi-spectroscopic%20techniques%20combined%20with%20molecular%20docking&amp;author=K.-Y.%20Chen&amp;author=K.-L.%20Zhou&amp;author=Y.-Y.%20Lou&amp;author=J.-H.%20Shi&amp;volume=37&amp;publication_year=2019&amp;pages=576-583&amp;pmid=29375016&amp;doi=10.1080/07391102.2018.1433067&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit18">
<cite>Palchaudhuri R. Hergenrother P. J. DNA as a target for anticancer compounds: methods to determine the mode of binding and the mechanism of action. Curr. Opin. Biotechnol. 2007;18:497–503. doi: 10.1016/j.copbio.2007.09.006.</cite> [<a href="https://doi.org/10.1016/j.copbio.2007.09.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17988854/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Biotechnol.&amp;title=DNA%20as%20a%20target%20for%20anticancer%20compounds:%20methods%20to%20determine%20the%20mode%20of%20binding%20and%20the%20mechanism%20of%20action&amp;author=R.%20Palchaudhuri&amp;author=P.%20J.%20Hergenrother&amp;volume=18&amp;publication_year=2007&amp;pages=497-503&amp;pmid=17988854&amp;doi=10.1016/j.copbio.2007.09.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit19">
<cite>Chagaleti B. K. Saravanan V. Vellapandian C. Kathiravan M. K. Exploring cyclin-dependent kinase inhibitors: a comprehensive study in search of CDK-6 inhibitors using a pharmacophore modelling and dynamics approach. RSC Adv. 2023;13:33770–33785. doi: 10.1039/d3ra05672d.</cite> [<a href="https://doi.org/10.1039/d3ra05672d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10655667/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38019988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Adv.&amp;title=Exploring%20cyclin-dependent%20kinase%20inhibitors:%20a%20comprehensive%20study%20in%20search%20of%20CDK-6%20inhibitors%20using%20a%20pharmacophore%20modelling%20and%20dynamics%20approach&amp;author=B.%20K.%20Chagaleti&amp;author=V.%20Saravanan&amp;author=C.%20Vellapandian&amp;author=M.%20K.%20Kathiravan&amp;volume=13&amp;publication_year=2023&amp;pages=33770-33785&amp;pmid=38019988&amp;doi=10.1039/d3ra05672d&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit20">
<cite>Chagaleti B. K. Kumar B. S. Rajagopal R. Alfarhan A. Arockiaraj J. Kumaradoss K. M. Namasivayam S. K. R. Targeting cyclin-dependent kinase 2 CDK2: Insights from Molecular Docking and Dynamics Simulation-A systematic computational approach to discover novel cancer therapeutics. Comput. Biol. Chem. 2024;112:108134. doi: 10.1016/j.compbiolchem.2024.108134.</cite> [<a href="https://doi.org/10.1016/j.compbiolchem.2024.108134" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38964206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput.%20Biol.%20Chem.&amp;title=Targeting%20cyclin-dependent%20kinase%202%20CDK2:%20Insights%20from%20Molecular%20Docking%20and%20Dynamics%20Simulation-A%20systematic%20computational%20approach%20to%20discover%20novel%20cancer%20therapeutics&amp;author=B.%20K.%20Chagaleti&amp;author=B.%20S.%20Kumar&amp;author=R.%20Rajagopal&amp;author=A.%20Alfarhan&amp;author=J.%20Arockiaraj&amp;volume=112&amp;publication_year=2024&amp;pages=108134&amp;pmid=38964206&amp;doi=10.1016/j.compbiolchem.2024.108134&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit21">
<cite>Chagaleti B. K. Saravanan V. Kathiravan M. Discovery of novel CDK2 inhibitors for cancer treatment: integrating ligand-based pharmacophore modelling, molecular docking, DFT, ADMET, and molecular dynamics simulation studies. Beni-Suef Univ. J. Basic Appl. Sci. 2024;13:118.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Beni-Suef%20Univ.%20J.%20Basic%20Appl.%20Sci.&amp;title=Discovery%20of%20novel%20CDK2%20inhibitors%20for%20cancer%20treatment:%20integrating%20ligand-based%20pharmacophore%20modelling,%20molecular%20docking,%20DFT,%20ADMET,%20and%20molecular%20dynamics%20simulation%20studies&amp;author=B.%20K.%20Chagaleti&amp;author=V.%20Saravanan&amp;author=M.%20Kathiravan&amp;volume=13&amp;publication_year=2024&amp;pages=118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit22">
<cite>Babu N. R. Sahoo B. Radhika T. Malothu N. Synthesis, Antitubercular Evaluation, Molecular Docking Study, and Teratogenicity Studies of Novel Triazolo Quinazoline Hybrids. Russ. J. Gen. Chem. 2023;93:2684–2693.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Russ.%20J.%20Gen.%20Chem.&amp;title=Synthesis,%20Antitubercular%20Evaluation,%20Molecular%20Docking%20Study,%20and%20Teratogenicity%20Studies%20of%20Novel%20Triazolo%20Quinazoline%20Hybrids&amp;author=N.%20R.%20Babu&amp;author=B.%20Sahoo&amp;author=T.%20Radhika&amp;author=N.%20Malothu&amp;volume=93&amp;publication_year=2023&amp;pages=2684-2693&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit23">
<cite>
Bekker H., Berendsen H., Dijkstra E., Achterop S., Vondrumen R. v., Vanderspoel D., Sijbers A., Keegstra H. and Renardus M., Gromacs – A parallel computer for molecular-dynamics simulations, 4th International Conference on Computational Physics (PC 92), World Scientific Publishing, 1993, pp. 252–256</cite> [<a href="https://scholar.google.com/scholar_lookup?title=4th%20International%20Conference%20on%20Computational%20Physics%20(PC%2092)&amp;author=H.%20Bekker&amp;author=H.%20Berendsen&amp;author=E.%20Dijkstra&amp;author=S.%20Achterop&amp;author=R.%20v.%20Vondrumen&amp;publication_year=1993&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit24">
<cite>Ganesan A. Coote M. L. Barakat K. Molecular dynamics-driven drug discovery: leaping forward with confidence. Drug Discovery Today. 2017;22:249–269. doi: 10.1016/j.drudis.2016.11.001.</cite> [<a href="https://doi.org/10.1016/j.drudis.2016.11.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27890821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Discovery%20Today&amp;title=Molecular%20dynamics-driven%20drug%20discovery:%20leaping%20forward%20with%20confidence&amp;author=A.%20Ganesan&amp;author=M.%20L.%20Coote&amp;author=K.%20Barakat&amp;volume=22&amp;publication_year=2017&amp;pages=249-269&amp;pmid=27890821&amp;doi=10.1016/j.drudis.2016.11.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit25">
<cite>Nandi N. K. Das S. Choudhary D. Saini S. Bhatia R. Chawla P. Kaur R. Kalra S. Rawat R. Eyupoglu V. Exploration of oxadiazole clubbed benzhydrylpiperazine pharmacophoric features as structural feature for antidepressant activity: In vitro, in vivo and in silico analysis. Bioorg. Chem. 2024;144:107148. doi: 10.1016/j.bioorg.2024.107148.</cite> [<a href="https://doi.org/10.1016/j.bioorg.2024.107148" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38306828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Chem.&amp;title=Exploration%20of%20oxadiazole%20clubbed%20benzhydrylpiperazine%20pharmacophoric%20features%20as%20structural%20feature%20for%20antidepressant%20activity:%20In%20vitro,%20in%20vivo%20and%20in%20silico%20analysis&amp;author=N.%20K.%20Nandi&amp;author=S.%20Das&amp;author=D.%20Choudhary&amp;author=S.%20Saini&amp;author=R.%20Bhatia&amp;volume=144&amp;publication_year=2024&amp;pages=107148&amp;pmid=38306828&amp;doi=10.1016/j.bioorg.2024.107148&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit26">
<cite>Nerenberg P. S. Head-Gordon T. Optimizing protein–solvent force fields to reproduce intrinsic conformational preferences of model peptides. J. Chem. Theory Comput. 2011;7:1220–1230. doi: 10.1021/ct2000183.</cite> [<a href="https://doi.org/10.1021/ct2000183" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26606367/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Theory%20Comput.&amp;title=Optimizing%20protein%E2%80%93solvent%20force%20fields%20to%20reproduce%20intrinsic%20conformational%20preferences%20of%20model%20peptides&amp;author=P.%20S.%20Nerenberg&amp;author=T.%20Head-Gordon&amp;volume=7&amp;publication_year=2011&amp;pages=1220-1230&amp;pmid=26606367&amp;doi=10.1021/ct2000183&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit27">
<cite>Van Aalten D. M. Bywater R. Findlay J. B. Hendlich M. Hooft R. W. Vriend G. PRODRG, a program for generating molecular topologies and unique molecular descriptors from coordinates of small molecules. J. Comput.-Aided Mol. Des. 1996;10:255–262. doi: 10.1007/BF00355047.</cite> [<a href="https://doi.org/10.1007/BF00355047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8808741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.-Aided%20Mol.%20Des.&amp;title=PRODRG,%20a%20program%20for%20generating%20molecular%20topologies%20and%20unique%20molecular%20descriptors%20from%20coordinates%20of%20small%20molecules&amp;author=D.%20M.%20Van%20Aalten&amp;author=R.%20Bywater&amp;author=J.%20B.%20Findlay&amp;author=M.%20Hendlich&amp;author=R.%20W.%20Hooft&amp;volume=10&amp;publication_year=1996&amp;pages=255-262&amp;pmid=8808741&amp;doi=10.1007/BF00355047&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit28">
<cite>Mark P. Nilsson L. Structure and dynamics of the TIP3P, SPC, and SPC/E water models at 298 K. J. Phys. Chem. A. 2001;105:9954–9960.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Phys.%20Chem.%20A&amp;title=Structure%20and%20dynamics%20of%20the%20TIP3P,%20SPC,%20and%20SPC/E%20water%20models%20at%20298%20K&amp;author=P.%20Mark&amp;author=L.%20Nilsson&amp;volume=105&amp;publication_year=2001&amp;pages=9954-9960&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit29">
<cite>Van Gunsteren W. F. Berendsen H. J. A leap-frog algorithm for stochastic dynamics. Mol. Simul. 1988;1:173–185.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Simul.&amp;title=A%20leap-frog%20algorithm%20for%20stochastic%20dynamics&amp;author=W.%20F.%20Van%20Gunsteren&amp;author=H.%20J.%20Berendsen&amp;volume=1&amp;publication_year=1988&amp;pages=173-185&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit30">
<cite>Berendsen H. J. van der Spoel D. van Drunen R. GROMACS: A message-passing parallel molecular dynamics implementation. Comput. Phys. Commun. 1995;91:43–56.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Comput.%20Phys.%20Commun.&amp;title=GROMACS:%20A%20message-passing%20parallel%20molecular%20dynamics%20implementation&amp;author=H.%20J.%20Berendsen&amp;author=D.%20van%20der%20Spoel&amp;author=R.%20van%20Drunen&amp;volume=91&amp;publication_year=1995&amp;pages=43-56&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit31">
<cite>Hess B. Bekker H. Berendsen H. J. Fraaije J. G. LINCS: a linear constraint solver for molecular simulations. J. Comput. Chem. 1997;18:1463–1472.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&amp;title=LINCS:%20a%20linear%20constraint%20solver%20for%20molecular%20simulations&amp;author=B.%20Hess&amp;author=H.%20Bekker&amp;author=H.%20J.%20Berendsen&amp;author=J.%20G.%20Fraaije&amp;volume=18&amp;publication_year=1997&amp;pages=1463-1472&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit32">
<cite>Di Pierro M. Elber R. Leimkuhler B. A stochastic algorithm for the isobaric–isothermal ensemble with Ewald summations for all long range forces. J. Chem. Theory Comput. 2015;11:5624–5637. doi: 10.1021/acs.jctc.5b00648.</cite> [<a href="https://doi.org/10.1021/acs.jctc.5b00648" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4890727/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26616351/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Theory%20Comput.&amp;title=A%20stochastic%20algorithm%20for%20the%20isobaric%E2%80%93isothermal%20ensemble%20with%20Ewald%20summations%20for%20all%20long%20range%20forces&amp;author=M.%20Di%20Pierro&amp;author=R.%20Elber&amp;author=B.%20Leimkuhler&amp;volume=11&amp;publication_year=2015&amp;pages=5624-5637&amp;pmid=26616351&amp;doi=10.1021/acs.jctc.5b00648&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit33">
<cite>Moulishankar A. Sundarrajan T. Pharmacophore, QSAR, molecular docking, molecular dynamics and ADMET study of trisubstituted benzimidazole derivatives as potent anti-tubercular agents. Chem. Phys. Impact. 2024;8:100512.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Phys.%20Impact&amp;title=Pharmacophore,%20QSAR,%20molecular%20docking,%20molecular%20dynamics%20and%20ADMET%20study%20of%20trisubstituted%20benzimidazole%20derivatives%20as%20potent%20anti-tubercular%20agents&amp;author=A.%20Moulishankar&amp;author=T.%20Sundarrajan&amp;volume=8&amp;publication_year=2024&amp;pages=100512&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit34">
<cite>Moulishankar A. Sankaranarayanan M. Thirugnanasambandam S. Dhamotharan J. Mohanradja D. Sivakumar P. M. Identification of novel DNA gyrase inhibitor by combined pharmacophore modeling, QSAR analysis, molecular docking, molecular dynamics, ADMET and DFT approaches. Acta Trop. 2024;260:107460. doi: 10.1016/j.actatropica.2024.107460.</cite> [<a href="https://doi.org/10.1016/j.actatropica.2024.107460" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39527993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Trop.&amp;title=Identification%20of%20novel%20DNA%20gyrase%20inhibitor%20by%20combined%20pharmacophore%20modeling,%20QSAR%20analysis,%20molecular%20docking,%20molecular%20dynamics,%20ADMET%20and%20DFT%20approaches&amp;author=A.%20Moulishankar&amp;author=M.%20Sankaranarayanan&amp;author=S.%20Thirugnanasambandam&amp;author=J.%20Dhamotharan&amp;author=D.%20Mohanradja&amp;volume=260&amp;publication_year=2024&amp;pages=107460&amp;pmid=39527993&amp;doi=10.1016/j.actatropica.2024.107460&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit35">
<cite>Moulishankar A. Sundarrajan T. QSAR modeling, molecular docking, dynamic simulation and ADMET study of novel tetrahydronaphthalene derivatives as potent antitubercular agents. Beni-Suef Univ. J. Basic Appl. Sci. 2023;12:111.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Beni-Suef%20Univ.%20J.%20Basic%20Appl.%20Sci.&amp;title=QSAR%20modeling,%20molecular%20docking,%20dynamic%20simulation%20and%20ADMET%20study%20of%20novel%20tetrahydronaphthalene%20derivatives%20as%20potent%20antitubercular%20agents&amp;author=A.%20Moulishankar&amp;author=T.%20Sundarrajan&amp;volume=12&amp;publication_year=2023&amp;pages=111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit36">
<cite>Humphrey W. Dalke A. Schulten K. VMD: visual molecular dynamics. J. Mol. Graphics. 1996;14:33–38. doi: 10.1016/0263-7855(96)00018-5.</cite> [<a href="https://doi.org/10.1016/0263-7855(96)00018-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8744570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Graphics&amp;title=VMD:%20visual%20molecular%20dynamics&amp;author=W.%20Humphrey&amp;author=A.%20Dalke&amp;author=K.%20Schulten&amp;volume=14&amp;publication_year=1996&amp;pages=33-38&amp;pmid=8744570&amp;doi=10.1016/0263-7855(96)00018-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit37">
<cite>Rawat R. Kant K. Kumar A. Bhati K. Verma S. M. HeroMDAnalysis: an automagical tool for GROMACS-based molecular dynamics simulation analysis. Future Med. Chem. 2021;13:447–456. doi: 10.4155/fmc-2020-0191.</cite> [<a href="https://doi.org/10.4155/fmc-2020-0191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33496197/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Future%20Med.%20Chem.&amp;title=HeroMDAnalysis:%20an%20automagical%20tool%20for%20GROMACS-based%20molecular%20dynamics%20simulation%20analysis&amp;author=R.%20Rawat&amp;author=K.%20Kant&amp;author=A.%20Kumar&amp;author=K.%20Bhati&amp;author=S.%20M.%20Verma&amp;volume=13&amp;publication_year=2021&amp;pages=447-456&amp;pmid=33496197&amp;doi=10.4155/fmc-2020-0191&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit38">
<cite>Yırtıcı Ü. Ergene A. Adem Ş. Atalar M. N. Eyüpoğlu V. Rawat R. Arat E. Hamzaoğlu E. Centaurea mersinensis phytochemical composition and multi-dimensional bioactivity properties supported by molecular modeling. J. Biomol. Struct. Dyn. 2024;42:2341–2357. doi: 10.1080/07391102.2023.2204496.</cite> [<a href="https://doi.org/10.1080/07391102.2023.2204496" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37098809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20Struct.%20Dyn.&amp;title=Centaurea%20mersinensis%20phytochemical%20composition%20and%20multi-dimensional%20bioactivity%20properties%20supported%20by%20molecular%20modeling&amp;author=%C3%9C.%20Y%C4%B1rt%C4%B1c%C4%B1&amp;author=A.%20Ergene&amp;author=%C5%9E.%20Adem&amp;author=M.%20N.%20Atalar&amp;author=V.%20Ey%C3%BCpo%C4%9Flu&amp;volume=42&amp;publication_year=2024&amp;pages=2341-2357&amp;pmid=37098809&amp;doi=10.1080/07391102.2023.2204496&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit39">
<cite>Vaught A. Graphing with Gnuplot and Xmgr: two graphing packages available under linux. Linux J. 1996;1996</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Linux%20J.&amp;title=Graphing%20with%20Gnuplot%20and%20Xmgr:%20two%20graphing%20packages%20available%20under%20linux&amp;author=A.%20Vaught&amp;volume=1996&amp;publication_year=1996&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]; <cite>, 7-es</cite>
</li>
<li id="cit40">
<cite>Waring M. Complex formation between ethidium bromide and nucleic acids. J. Mol. Biol. 1965;13:269–282. doi: 10.1016/s0022-2836(65)80096-1.</cite> [<a href="https://doi.org/10.1016/s0022-2836(65)80096-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/5859041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=Complex%20formation%20between%20ethidium%20bromide%20and%20nucleic%20acids&amp;author=M.%20Waring&amp;volume=13&amp;publication_year=1965&amp;pages=269-282&amp;pmid=5859041&amp;doi=10.1016/s0022-2836(65)80096-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit41">
<cite>Kashanian S. Javanmardi S. Chitsazan A. Omidfar K. Paknejad M. DNA-binding studies of fluoxetine antidepressant. DNA Cell Biol. 2012;31:1349–1355. doi: 10.1089/dna.2012.1657.</cite> [<a href="https://doi.org/10.1089/dna.2012.1657" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22510099/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=DNA%20Cell%20Biol.&amp;title=DNA-binding%20studies%20of%20fluoxetine%20antidepressant&amp;author=S.%20Kashanian&amp;author=S.%20Javanmardi&amp;author=A.%20Chitsazan&amp;author=K.%20Omidfar&amp;author=M.%20Paknejad&amp;volume=31&amp;publication_year=2012&amp;pages=1349-1355&amp;pmid=22510099&amp;doi=10.1089/dna.2012.1657&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit42">
<cite>Grueso E. López-Pérez G. Castellano M. Prado-Gotor R. Thermodynamic and structural study of phenanthroline derivative ruthenium complex/DNA interactions: probing partial intercalation and binding properties. J. Inorg. Biochem. 2012;106:1–9. doi: 10.1016/j.jinorgbio.2011.09.028.</cite> [<a href="https://doi.org/10.1016/j.jinorgbio.2011.09.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22105011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Inorg.%20Biochem.&amp;title=Thermodynamic%20and%20structural%20study%20of%20phenanthroline%20derivative%20ruthenium%20complex/DNA%20interactions:%20probing%20partial%20intercalation%20and%20binding%20properties&amp;author=E.%20Grueso&amp;author=G.%20L%C3%B3pez-P%C3%A9rez&amp;author=M.%20Castellano&amp;author=R.%20Prado-Gotor&amp;volume=106&amp;publication_year=2012&amp;pages=1-9&amp;pmid=22105011&amp;doi=10.1016/j.jinorgbio.2011.09.028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit43">
<cite>Li J. Dong C. Study on the interaction of morphine chloride with deoxyribonucleic acid by fluorescence method. Spectrochim. Acta, Part A. 2009;71:1938–1943. doi: 10.1016/j.saa.2008.07.033.</cite> [<a href="https://doi.org/10.1016/j.saa.2008.07.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18818119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Spectrochim.%20Acta,%20Part%20A&amp;title=Study%20on%20the%20interaction%20of%20morphine%20chloride%20with%20deoxyribonucleic%20acid%20by%20fluorescence%20method&amp;author=J.%20Li&amp;author=C.%20Dong&amp;volume=71&amp;publication_year=2009&amp;pages=1938-1943&amp;pmid=18818119&amp;doi=10.1016/j.saa.2008.07.033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit44">
<cite>Shi J.-H. Pan D.-Q. Zhou K.-L. Lou Y.-Y. Exploring the binding interaction between herring sperm DNA and sunitinib: insights from spectroscopic and molecular docking approaches. J. Biomol. Struct. Dyn. 2019;37:837–845. doi: 10.1080/07391102.2018.1445033.</cite> [<a href="https://doi.org/10.1080/07391102.2018.1445033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29471738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20Struct.%20Dyn.&amp;title=Exploring%20the%20binding%20interaction%20between%20herring%20sperm%20DNA%20and%20sunitinib:%20insights%20from%20spectroscopic%20and%20molecular%20docking%20approaches&amp;author=J.-H.%20Shi&amp;author=D.-Q.%20Pan&amp;author=K.-L.%20Zhou&amp;author=Y.-Y.%20Lou&amp;volume=37&amp;publication_year=2019&amp;pages=837-845&amp;pmid=29471738&amp;doi=10.1080/07391102.2018.1445033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit45">
<cite>Hegde A. H. Prashanth S. Seetharamappa J. Interaction of antioxidant flavonoids with calf thymus DNA analyzed by spectroscopic and electrochemical methods. J. Pharm. Biomed. Anal. 2012;63:40–46. doi: 10.1016/j.jpba.2012.01.034.</cite> [<a href="https://doi.org/10.1016/j.jpba.2012.01.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22349882/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Pharm.%20Biomed.%20Anal.&amp;title=Interaction%20of%20antioxidant%20flavonoids%20with%20calf%20thymus%20DNA%20analyzed%20by%20spectroscopic%20and%20electrochemical%20methods&amp;author=A.%20H.%20Hegde&amp;author=S.%20Prashanth&amp;author=J.%20Seetharamappa&amp;volume=63&amp;publication_year=2012&amp;pages=40-46&amp;pmid=22349882&amp;doi=10.1016/j.jpba.2012.01.034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit46">
<cite>Sahoo D. Bhattacharya P. Chakravorti S. Quest for mode of binding of 2-(4-(dimethylamino)styryl)-1-methylpyridinium iodide with calf thymus DNA. J. Phys. Chem. B. 2010;114:2044–2050. doi: 10.1021/jp910766q.</cite> [<a href="https://doi.org/10.1021/jp910766q" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20088531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Phys.%20Chem.%20B&amp;title=Quest%20for%20mode%20of%20binding%20of%202-(4-(dimethylamino)styryl)-1-methylpyridinium%20iodide%20with%20calf%20thymus%20DNA&amp;author=D.%20Sahoo&amp;author=P.%20Bhattacharya&amp;author=S.%20Chakravorti&amp;volume=114&amp;publication_year=2010&amp;pages=2044-2050&amp;pmid=20088531&amp;doi=10.1021/jp910766q&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit47">
<cite>Klotz I. M. Physicochemical aspects of drug-protein interactions: a general perspective. Ann. N. Y. Acad. Sci. 1973;226:18–35. doi: 10.1111/j.1749-6632.1973.tb20465.x.</cite> [<a href="https://doi.org/10.1111/j.1749-6632.1973.tb20465.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4588441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann.%20N.%20Y.%20Acad.%20Sci.&amp;title=Physicochemical%20aspects%20of%20drug-protein%20interactions:%20a%20general%20perspective&amp;author=I.%20M.%20Klotz&amp;volume=226&amp;publication_year=1973&amp;pages=18-35&amp;pmid=4588441&amp;doi=10.1111/j.1749-6632.1973.tb20465.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit48">
<cite>Ross P. D. Subramanian S. Thermodynamics of protein association reactions: forces contributing to stability. Biochemistry. 1981;20:3096–3102. doi: 10.1021/bi00514a017.</cite> [<a href="https://doi.org/10.1021/bi00514a017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7248271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Thermodynamics%20of%20protein%20association%20reactions:%20forces%20contributing%20to%20stability&amp;author=P.%20D.%20Ross&amp;author=S.%20Subramanian&amp;volume=20&amp;publication_year=1981&amp;pages=3096-3102&amp;pmid=7248271&amp;doi=10.1021/bi00514a017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit49">
<cite>Shi J.-H. Chen J. Wang J. Zhu Y.-Y. Binding interaction between sorafenib and calf thymus DNA: spectroscopic methodology, viscosity measurement and molecular docking. Spectrochim. Acta, Part A. 2015;136:443–450. doi: 10.1016/j.saa.2014.09.056.</cite> [<a href="https://doi.org/10.1016/j.saa.2014.09.056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25311519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Spectrochim.%20Acta,%20Part%20A&amp;title=Binding%20interaction%20between%20sorafenib%20and%20calf%20thymus%20DNA:%20spectroscopic%20methodology,%20viscosity%20measurement%20and%20molecular%20docking&amp;author=J.-H.%20Shi&amp;author=J.%20Chen&amp;author=J.%20Wang&amp;author=Y.-Y.%20Zhu&amp;volume=136&amp;publication_year=2015&amp;pages=443-450&amp;pmid=25311519&amp;doi=10.1016/j.saa.2014.09.056&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit50">
<cite>Rohs R. West S. M. Sosinsky A. Liu P. Mann R. S. Honig B. The role of DNA shape in protein–DNA recognition. Nature. 2009;461:1248–1253. doi: 10.1038/nature08473.</cite> [<a href="https://doi.org/10.1038/nature08473" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2793086/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19865164/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20role%20of%20DNA%20shape%20in%20protein%E2%80%93DNA%20recognition&amp;author=R.%20Rohs&amp;author=S.%20M.%20West&amp;author=A.%20Sosinsky&amp;author=P.%20Liu&amp;author=R.%20S.%20Mann&amp;volume=461&amp;publication_year=2009&amp;pages=1248-1253&amp;pmid=19865164&amp;doi=10.1038/nature08473&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets generated and/or analyzed during the current study are available from the corresponding author on reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from RSC Advances are provided here courtesy of <strong>Royal Society of Chemistry</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1039/d5ra04723d"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/RA-015-D5RA04723D.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12376933/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12376933/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12376933%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12376933/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12376933/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12376933/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12376933/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12376933/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12376933/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="rRBY9Hp5pLiRtE4uy87u218uzdgawFz2j4dYkuG3rhZ6N2G6cbxoGVYxVevTCS7D">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
