
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            VEGF Signal Complexity Confers Resistance to Atezolizumab, Bevacizumab, Carboplatin, and Paclitaxel in EGFR‐Tyrosine Kinase Inhibitor‐Resistant Non‐Small Cell Lung Cancer - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4B73E8AF3D42305B73E000CC97E8E.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="mcomm">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365383/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="MedComm">
<meta name="citation_title" content="VEGF Signal Complexity Confers Resistance to Atezolizumab, Bevacizumab, Carboplatin, and Paclitaxel in EGFR‐Tyrosine Kinase Inhibitor‐Resistant Non‐Small Cell Lung Cancer">
<meta name="citation_author" content="Sehwa Hong">
<meta name="citation_author_institution" content="Research Institute, National Cancer Center, Gyeonggi, Republic of Korea">
<meta name="citation_author" content="Namhee Yu">
<meta name="citation_author_institution" content="Research Institute, National Cancer Center, Gyeonggi, Republic of Korea">
<meta name="citation_author" content="Ju Young Cho">
<meta name="citation_author_institution" content="Research Institute, National Cancer Center, Gyeonggi, Republic of Korea">
<meta name="citation_author" content="Geon Kook Lee">
<meta name="citation_author_institution" content="Center for Lung Cancer, National Cancer Center Hospital, Gyeonggi‐do, Republic of Korea">
<meta name="citation_author" content="Beung‐Chul Ahn">
<meta name="citation_author_institution" content="Research Institute, National Cancer Center, Gyeonggi, Republic of Korea">
<meta name="citation_author_institution" content="Center for Lung Cancer, National Cancer Center Hospital, Gyeonggi‐do, Republic of Korea">
<meta name="citation_author" content="Youngjoo Lee">
<meta name="citation_author_institution" content="Research Institute, National Cancer Center, Gyeonggi, Republic of Korea">
<meta name="citation_author_institution" content="Center for Lung Cancer, National Cancer Center Hospital, Gyeonggi‐do, Republic of Korea">
<meta name="citation_author" content="Hanna Sim">
<meta name="citation_author_institution" content="National Cancer Center Graduate School of Cancer Science and Policy, Gyeonggi, Republic of Korea">
<meta name="citation_author" content="Bo Ram Song">
<meta name="citation_author_institution" content="Research Institute, National Cancer Center, Gyeonggi, Republic of Korea">
<meta name="citation_author" content="Mihwa Hwang">
<meta name="citation_author_institution" content="Research Institute, National Cancer Center, Gyeonggi, Republic of Korea">
<meta name="citation_author" content="Sunshin Kim">
<meta name="citation_author_institution" content="Research Institute, National Cancer Center, Gyeonggi, Republic of Korea">
<meta name="citation_author" content="Jung‐Hyun Kim">
<meta name="citation_author_institution" content="Research Institute, National Cancer Center, Gyeonggi, Republic of Korea">
<meta name="citation_author" content="Charny Park">
<meta name="citation_author_institution" content="Research Institute, National Cancer Center, Gyeonggi, Republic of Korea">
<meta name="citation_author" content="Ji‐Youn Han">
<meta name="citation_author_institution" content="Research Institute, National Cancer Center, Gyeonggi, Republic of Korea">
<meta name="citation_author_institution" content="Center for Lung Cancer, National Cancer Center Hospital, Gyeonggi‐do, Republic of Korea">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="6">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="e70335">
<meta name="citation_doi" content="10.1002/mco2.70335">
<meta name="citation_pmid" content="40843133">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365383/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365383/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365383/pdf/MCO2-6-e70335.pdf">
<meta name="description" content="Atezolizumab, bevacizumab, carboplatin, and paclitaxel (ABCP) therapy is beneficial for epidermal growth factor receptor‐tyrosine kinase inhibitor (EGFR‐TKI)‐resistant non‐small cell lung cancer (NSCLC); however, the resistance mechanisms are not ...">
<meta name="og:title" content="VEGF Signal Complexity Confers Resistance to Atezolizumab, Bevacizumab, Carboplatin, and Paclitaxel in EGFR‐Tyrosine Kinase Inhibitor‐Resistant Non‐Small Cell Lung Cancer">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Atezolizumab, bevacizumab, carboplatin, and paclitaxel (ABCP) therapy is beneficial for epidermal growth factor receptor‐tyrosine kinase inhibitor (EGFR‐TKI)‐resistant non‐small cell lung cancer (NSCLC); however, the resistance mechanisms are not ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365383/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365383">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/mco2.70335"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/MCO2-6-e70335.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365383%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365383/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365383/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365383/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-mcomm.jpg" alt="MedComm logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to MedComm" title="Link to MedComm" shape="default" href="https://onlinelibrary.wiley.com/journal/26882663" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">MedComm (2020)</button></div>. 2025 Aug 19;6(9):e70335. doi: <a href="https://doi.org/10.1002/mco2.70335" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/mco2.70335</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22MedComm%20(2020)%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22MedComm%20(2020)%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22MedComm%20(2020)%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22MedComm%20(2020)%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>VEGF Signal Complexity Confers Resistance to Atezolizumab, Bevacizumab, Carboplatin, and Paclitaxel in EGFR‐Tyrosine Kinase Inhibitor‐Resistant Non‐Small Cell Lung Cancer</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hong%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Sehwa Hong</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Sehwa Hong</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Research Institute, National Cancer Center, Gyeonggi, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hong%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sehwa Hong</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Namhee Yu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Namhee Yu</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Research Institute, National Cancer Center, Gyeonggi, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Namhee Yu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cho%20JY%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Ju Young Cho</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Ju Young Cho</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Research Institute, National Cancer Center, Gyeonggi, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cho%20JY%22%5BAuthor%5D" class="usa-link"><span class="name western">Ju Young Cho</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20GK%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Geon Kook Lee</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Geon Kook Lee</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Center for Lung Cancer, National Cancer Center Hospital, Gyeonggi‐do, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20GK%22%5BAuthor%5D" class="usa-link"><span class="name western">Geon Kook Lee</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahn%20BC%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Beung‐Chul Ahn</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Beung‐Chul Ahn</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Research Institute, National Cancer Center, Gyeonggi, Republic of Korea</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Center for Lung Cancer, National Cancer Center Hospital, Gyeonggi‐do, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ahn%20BC%22%5BAuthor%5D" class="usa-link"><span class="name western">Beung‐Chul Ahn</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Youngjoo Lee</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Youngjoo Lee</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Research Institute, National Cancer Center, Gyeonggi, Republic of Korea</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Center for Lung Cancer, National Cancer Center Hospital, Gyeonggi‐do, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Youngjoo Lee</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sim%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Hanna Sim</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Hanna Sim</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>National Cancer Center Graduate School of Cancer Science and Policy, Gyeonggi, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sim%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hanna Sim</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20BR%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Bo Ram Song</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Bo Ram Song</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Research Institute, National Cancer Center, Gyeonggi, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20BR%22%5BAuthor%5D" class="usa-link"><span class="name western">Bo Ram Song</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hwang%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Mihwa Hwang</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Mihwa Hwang</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Research Institute, National Cancer Center, Gyeonggi, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hwang%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mihwa Hwang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Sunshin Kim</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Sunshin Kim</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Research Institute, National Cancer Center, Gyeonggi, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sunshin Kim</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20JH%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Jung‐Hyun Kim</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Jung‐Hyun Kim</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Research Institute, National Cancer Center, Gyeonggi, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20JH%22%5BAuthor%5D" class="usa-link"><span class="name western">Jung‐Hyun Kim</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Charny Park</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Charny Park</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Research Institute, National Cancer Center, Gyeonggi, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Charny Park</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Han%20JY%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Ji‐Youn Han</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Ji‐Youn Han</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Research Institute, National Cancer Center, Gyeonggi, Republic of Korea</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Center for Lung Cancer, National Cancer Center Hospital, Gyeonggi‐do, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Han%20JY%22%5BAuthor%5D" class="usa-link"><span class="name western">Ji‐Youn Han</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="mco270335-aff-0001">
<sup>
<sup>1</sup>
</sup>Research Institute, National Cancer Center, Gyeonggi, Republic of Korea</div>
<div id="mco270335-aff-0002">
<sup>
<sup>2</sup>
</sup>Center for Lung Cancer, National Cancer Center Hospital, Gyeonggi‐do, Republic of Korea</div>
<div id="mco270335-aff-0003">
<sup>
<sup>3</sup>
</sup>National Cancer Center Graduate School of Cancer Science and Policy, Gyeonggi, Republic of Korea</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>: Jung‐Hyun Kim (<span>jungkim@ncc.re.kr</span>) | Charny Park (<span>charn78@ncc.re.kr</span>) | Ji‐Youn Han (<span>jymama@ncc.re.kr</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jun 16; Received 2024 Dec 30; Accepted 2025 Jul 15; Collection date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>MedComm</em> published by Sichuan International Medical Exchange &amp; Promotion Association (SCIMEA) and John Wiley &amp; Sons Australia, Ltd.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365383  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40843133/" class="usa-link">40843133</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>Atezolizumab, bevacizumab, carboplatin, and paclitaxel (ABCP) therapy is beneficial for epidermal growth factor receptor‐tyrosine kinase inhibitor (EGFR‐TKI)‐resistant non‐small cell lung cancer (NSCLC); however, the resistance mechanisms are not fully understood. In this study, we conducted a single‐cell RNA‐sequencing analysis of EGFR‐TKI‐resistant NSCLC patients grouped into ABCP responders and non‐responders. <em>VEGFA</em> was overexpressed in ABCP responders, whereas <em>VEGFC</em> was upregulated in non‐responders. VEGFA and VEGFC had exclusive distributions and interactions, suggesting their distinct roles. VEGFA facilitated the proliferation of responder tumor subcluster cells, whereas VEGFC secreted from non‐responder tumor cells interacted with tumor microenvironment cells. VEGFC predominantly cooperated with drug resistance pathways such as fibroblast growth factor signaling and YAP‐TAZ regulation, whereas VEGFA coordinated several oncogenic signaling pathways. <em>VEGFC</em> expression was the most significant prognostic marker (hazard ratio, 1.8 [95% confidence interval, 1.1–3.0], <em>p</em> = 0.015). Both VEGFA and VEGFC inhibition effectively suppressed tumor growth, suggesting that VEGF signaling complexity hampers the response to ABCP. In conclusion, combinatorial targeting of both ligands (VEGFA and VEGFC) or their receptors (VEGFR2 and KDR) may enhance the clinical benefit of ABCP in EGFR‐TKI‐resistant NSCLC patients.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> ABCP therapy response, combination immunotherapy, non‐small cell lung cancer, single‐cell transcriptome analysis, VEGF signaling</p></section></section><section class="abstract" id="mco270335-abs-0001"><hr class="headless">
<p>Single‐cell RNA sequencing of EGFR‐TKI‐resistant NSCLC reveals distinct roles of VEGFA and VEGFC in ABCP therapy response. VEGFA drives tumor proliferation in responders, while VEGFC promotes resistance through TME interaction, FGF signaling, and YAP‐TAZ regulation. Dual inhibition of VEGFA/VEGFC suppresses tumor growth, suggesting that combined VEGF targeting may enhance efficacy of ABCP in resistant NSCLC patients.
</p>
<section class="bt xbox font-sm" id="boxed-text1"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9ce7/12365383/88500cbfda12/MCO2-6-e70335-g003.jpg" loading="lazy" id="jats-graphic-1" height="156" width="180" alt="graphic file with name MCO2-6-e70335-g003.jpg"></p></section></section><section id="mco270335-sec-0010"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Despite the significant improvement in survival with epidermal growth factor receptor (EGFR)‐tyrosine kinase inhibitors (TKIs), most patients with EGFR‐mutant non‐small cell lung cancer (NSCLC) develop EGFR‐TKI resistance. Therefore, additional therapeutic options following first‐line TKI treatment are urgently needed [<a href="#mco270335-bib-0001" class="usa-link" aria-describedby="mco270335-bib-0001">1</a>]. NSCLC tumor cells and the tumor microenvironment (TME) have distinct cellular characteristics. The TME of TKI‐resistant tumors plays a critical role in tumor cell proliferation and anti‐apoptosis. As tumors progress, immunosuppressive T cells, including regulatory and dysfunctional T cells, become predominant [<a href="#mco270335-bib-0002" class="usa-link" aria-describedby="mco270335-bib-0002">2</a>]. Cancer‐associated fibroblasts (CAFs) interact with tumor cells and secrete cytokines to suppress immune function, shaping a drug‐resistant microenvironment [<a href="#mco270335-bib-0003" class="usa-link" aria-describedby="mco270335-bib-0003">3</a>, <a href="#mco270335-bib-0004" class="usa-link" aria-describedby="mco270335-bib-0004">4</a>]. Despite efforts to overcome EGFR‐TKI resistance, establishing a successful treatment for EGFR‐TKI‐resistant NSCLC remains challenging.</p>
<p>The Impower150 study aimed at addressing TKI resistance demonstrated a significant survival benefit of a combination of atezolizumab, bevacizumab, carboplatin, and paclitaxel (ABCP) over other immune checkpoint inhibitors combined with chemotherapy [<a href="#mco270335-bib-0001" class="usa-link" aria-describedby="mco270335-bib-0001">1</a>, <a href="#mco270335-bib-0005" class="usa-link" aria-describedby="mco270335-bib-0005">5</a>]. Bevacizumab, a monoclonal antibody against vascular endothelial growth factor (VEGF)A, improved the response to ABCP compared to immunotherapy or chemotherapy and conferred a therapeutic benefit in EGFR‐TKI‐resistant NSCLC [<a href="#mco270335-bib-0006" class="usa-link" aria-describedby="mco270335-bib-0006">6</a>]. The upregulation of EGFR signaling may drive VEGF signaling in EGFR‐mutant NSCLC cells, contributing to TKI resistance [<a href="#mco270335-bib-0006" class="usa-link" aria-describedby="mco270335-bib-0006">6</a>]. VEGFs secreted from tumor and stromal cells affect regulatory T‐cell functions and desmoplastic stroma development [<a href="#mco270335-bib-0007" class="usa-link" aria-describedby="mco270335-bib-0007">7</a>]. VEGF family proteins contributing to tumor cell survival exhibit distinct biological activities depending on the cell type [<a href="#mco270335-bib-0008" class="usa-link" aria-describedby="mco270335-bib-0008">8</a>, <a href="#mco270335-bib-0009" class="usa-link" aria-describedby="mco270335-bib-0009">9</a>, <a href="#mco270335-bib-0010" class="usa-link" aria-describedby="mco270335-bib-0010">10</a>, <a href="#mco270335-bib-0011" class="usa-link" aria-describedby="mco270335-bib-0011">11</a>]. Therefore, the superior outcomes of ABCP in EGFR‐mutant NSCLC may be owing to increased sensitivity to bevacizumab, which effectively blocks VEGF‐mediated oncogenic signaling activation or reverses immune suppression [<a href="#mco270335-bib-0006" class="usa-link" aria-describedby="mco270335-bib-0006">6</a>]. A randomized phase III trial demonstrated a significant progression‐free survival benefit of ABCP compared to pemetrexed plus platinum combination chemotherapy [<a href="#mco270335-bib-0012" class="usa-link" aria-describedby="mco270335-bib-0012">12</a>]. However, there were no significant differences in overall survival and response duration (median 7.1 months), and 50% of patients developed disease progression within 9 months [<a href="#mco270335-bib-0013" class="usa-link" aria-describedby="mco270335-bib-0013">13</a>]. A comprehensive understanding of the resistance mechanisms to ABCP would allow the development of a more efficient subsequent treatment.</p>
<p>Single‐cell RNA sequencing (scRNA‐seq) has provided valuable insights into the intricate interactions within the TME of EGFR‐TKI‐resistant NSCLC, particularly regarding the resistance mechanisms to combination therapies like atezolizumab and bevacizumab. These therapies target distinct aspects of the immune and vasculature systems, yet the complexity of the TME complicates their effectiveness. Single‐cell analyses of the TME have revealed significant cell heterogeneity, particularly the dominance of immunosuppressive T cells, fibroblast, and vascular development, in EGFR‐TKI‐resistant tumors. These immune cells contribute to immune evasion, undermining the efficacy of atezolizumab and bevacizumab [<a href="#mco270335-bib-0014" class="usa-link" aria-describedby="mco270335-bib-0014">14</a>, <a href="#mco270335-bib-0015" class="usa-link" aria-describedby="mco270335-bib-0015">15</a>]. Moreover, CAFs contribute to immune exclusion and resistance to immunotherapy, and targeting these CAFs may enhance the efficacy of immune checkpoint blockade therapies [<a href="#mco270335-bib-0016" class="usa-link" aria-describedby="mco270335-bib-0016">16</a>]. The presence of hypoxic conditions also exacerbates the immune suppression [<a href="#mco270335-bib-0017" class="usa-link" aria-describedby="mco270335-bib-0017">17</a>]. Understanding these mechanisms at the single‐cell level provides crucial insights into why some patients respond better to ABCP therapy while others develop resistance, paving the way for more tailored therapeutic strategies.</p>
<p>We aimed to elucidate the resistance mechanisms to ABCP further by conducting a scRNA‐seq analysis of EGFR‐TKI‐resistant NSCLC patients grouped into ABCP responders and nonresponders (Figure <a href="#mco270335-fig-0001" class="usa-link">1A</a>). To identify ABCP response‐predictive markers, samples were acquired prior to ABCP therapy. To characterize superior responses to ABCP, we extracted biomarkers and identified the regulatory mechanisms. Our findings were validated using multiple platforms, including spatial scRNA‐seq, additional TKI‐resistant scRNA‐seq profiling, bulk transcriptome profiling, and immunohistochemistry (IHC) as well as both in vitro and in vivo experiments.</p>
<figure class="fig xbox font-sm" id="mco270335-fig-0001"><h3 class="obj_head">FIGURE 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9ce7/12365383/a9ebeae32b03/MCO2-6-e70335-g008.jpg" loading="lazy" id="jats-graphic-3" height="617" width="558" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/mco270335-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Combination immunotherapy (atezolizumab plus bevacizumab plus cisplatin and paclitaxel, ABCP) benefit and single‐cell transcriptome analysis. ABCP indicates the combination therapy, and PemC indicates pemetrexed. IO indicates immune oncology drugs. ABCP responder (R), red; non‐responder (NR), blue. (A) CONSORT flow diagram for NSCLC. (B) Overall survival and progression‐free survival plots according to the three therapy types. (C) Bar plot of therapy response percentage (R: responder, red, NR: non‐responder, blue) and OR with its 95% confidence interval (CI). (D) UMAP plots of the single‐cell transcriptome profile of seven patients, colored by ABCP responses or cell lineages. (E) Bar plots of cell proportions for each patient and cell lineage. (F) Heatmap of canonical markers to identify cell types. The circle size indicates the percentage of expressed cells, and the color indicates the average gene expression. (G) Activation status violin plots of ABCP target pathways for each cell lineage according to ABCP responses.</p></figcaption></figure></section><section id="mco270335-sec-0020"><h2 class="pmc_sec_title">2. Results</h2>
<section id="mco270335-sec-0030"><h3 class="pmc_sec_title">2.1. Clinical Characteristics and Cell Distributions Prior to ABCP in TKI‐Progressed Patients</h3>
<p>A total of 127 patients with EGFR‐mutant NSCLC progressed on prior EGFR‐TKI and received salvage treatment—ABCP arm: <em>n</em> = 35; immune oncology drugs (IO) arm: <em>n</em> = 53; pemetrexed and carboplatin (PemC) arm: <em>n</em> = 39—were analyzed for clinical efficacy (Table <a href="#mco270335-supinfo-0001" class="usa-link">1S</a>, Figure <a href="#mco270335-fig-0001" class="usa-link">1A</a>). EGFR mutations (Table <a href="#mco270335-supinfo-0001" class="usa-link">S1</a>, 85.7%–89.7%) were successfully inhibited by TKI (2.6%) to acquire the resistance. EGFR‐independent resistant genetic factors were observed (MET 5.7%–23.1% and other oncogenes KRAS, BRAF, and PIK3CA 2.6%–5.7%) [<a href="#mco270335-bib-0018" class="usa-link" aria-describedby="mco270335-bib-0018">18</a>]. These factors showed no association with ABCP response. ABCP treatment improved progression‐free survival compared to single immunotherapy (<em>p</em> = 0.009), and overall survival exhibited no dramatic outcome (<em>p</em> = 0.172; Figure <a href="#mco270335-fig-0001" class="usa-link">1B</a>). More than 60% of patients responded to ABCP (<em>p</em> &lt; 0.001, odds ratio [OR] = 5.32; Figure <a href="#mco270335-fig-0001" class="usa-link">1C</a>), whereas approximately 40% did not. To identify the mechanisms and key predictive biomarkers of ABCP responsiveness, we conducted single‐cell transcriptome profiling of seven samples established before ABCP treatment using patient‐derived cells (PDCs) collected from malignant pleural effusions (responders, <em>n</em> = 3 and non‐responders, <em>n</em> = 4; Table <a href="#mco270335-supinfo-0001" class="usa-link">S2</a>, Figure <a href="#mco270335-fig-0001" class="usa-link">1A,D</a>). In a previous study, it was already evaluated that the transcriptomic characteristics of these PDCs resemble those of tumor biopsy samples [<a href="#mco270335-bib-0019" class="usa-link" aria-describedby="mco270335-bib-0019">19</a>]. In total, 43,647 cells were isolated from live cell preparations with total counts ranging from 230,000 to 1,000,000 and viabilities from 55.3% to 90.5% for scRNA‐seq analysis. Next, these cells were clustered into four major cell lineages: tumor, stroma, lymphoid, and myeloid (Figure <a href="#mco270335-fig-0001" class="usa-link">1D</a>). Stromal cells were enriched in non‐responders (OR = 29.09), whereas tumor (OR = 5.79) and myeloid (OR = 34.71) cells were more prevalent in responders (Figure <a href="#mco270335-fig-0001" class="usa-link">1E</a>). Lymphoid cells constituted a small fraction (4.44%) of total cells. Cell clusters were annotated using canonical markers (Figure <a href="#mco270335-fig-0001" class="usa-link">1F</a>, Table <a href="#mco270335-supinfo-0001" class="usa-link">S3</a>).</p>
<p>Overexpression of <em>EPCAM</em>, <em>CDH1</em>, <em>SFTPA1</em>, <em>MET</em>, and <em>ELF3</em> indicated MET‐dependent malignancies originating from epithelial cells (Figure <a href="#mco270335-fig-0001" class="usa-link">1F</a>). The stromal cluster exhibited expression of fibrosis markers such as <em>COL1A2</em>, <em>ACTA2</em>, <em>TAGLN</em>, and <em>THY1</em>, whereas lymphoid cells expressed <em>PTPRC</em>, <em>CD3E</em>, and <em>TRAC</em>. Global ABCP target pathway assessment revealed higher PD‐1 signaling activation in responders than in non‐responders in both tumor (fold change [FC] = 1.98) and lymphoid (FC = 1.46) cells (Figure <a href="#mco270335-fig-0001" class="usa-link">1G</a>). VEGF signaling did not differ between the two response groups. Although MPE samples are less heterogeneous than solid tumor biopsies, our analysis showed distinct differences between tumor and stromal cells that correlated with ABCP responsiveness [<a href="#mco270335-bib-0020" class="usa-link" aria-describedby="mco270335-bib-0020">20</a>].</p></section><section id="mco270335-sec-0040"><h3 class="pmc_sec_title">2.2. Regulatory Programs and Therapeutic Responses Based on Tumor Cell Clustering</h3>
<p>Tumor cell subclusters were associated with distinct oncogenic mechanisms. We identified five tumor subclusters with unique marker expression (T1–T5; Figure <a href="#mco270335-fig-0002" class="usa-link">2A,B</a>, Figure <a href="#mco270335-supinfo-0001" class="usa-link">S1A</a>). Prior to characterizing the tumor cells, we assessed their malignancy. Overall, tumor cells had significantly higher copy number variation (CNV) scores than reference normal cells (Figure <a href="#mco270335-fig-0002" class="usa-link">2C</a>). Amplifications in 5p15.33 (<em>TERT</em>) and 7q31.2 (<em>MET</em>) have been previously found to characterize lung adenocarcinoma (LUAD) profiles (Figure <a href="#mco270335-fig-0002" class="usa-link">2D</a>) [<a href="#mco270335-bib-0021" class="usa-link" aria-describedby="mco270335-bib-0021">21</a>]. CNV analysis uncovered TKI‐induced resistance mechanisms. When categorized by therapeutic groups, amplifications of <em>TERT</em>, <em>EGFR</em>, <em>MET</em>, and <em>ERBB2</em> were observed in ABCP responders (Figure <a href="#mco270335-fig-0002" class="usa-link">2D</a>). <em>TERT</em> and <em>MET</em> amplifications persisted in non‐responders. <em>MET</em> and <em>ERBB2</em> amplifications are established mechanisms of TKI resistance [<a href="#mco270335-bib-0018" class="usa-link" aria-describedby="mco270335-bib-0018">18</a>]. These CNVs indicated that these genes are potential therapeutic candidates.</p>
<figure class="fig xbox font-sm" id="mco270335-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9ce7/12365383/25f1219ba23d/MCO2-6-e70335-g006.jpg" loading="lazy" id="jats-graphic-5" height="511" width="561" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/mco270335-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Tumor cell subclusters and their molecular mechanisms. R, ABCP responder; NR, non‐responder. (A) UMAP of tumor cells divided into five subclusters. (B) Cell proportion bar plot of five subclusters according to ABCP responses and ORs for each cluster. (C) CNV plots (<em>Y</em>‐axis) for all chromosomes (<em>X</em>‐axis) inferred from single‐cell transcriptomes and classified by ABCP response. Amplification gene names are shown, along with their locus. (D) Box plot of CNV scores for five tumor subclusters and the reference cell type (ref; myeloid). (E) Two dot plots of trajectory trees projected by two components of pseudo‐time. Dots are colored by cell types (left) or trajectory states (right). (F) Bar plot of <em>p</em>‐values for top‐ranked pathways extracted by GSEA using DEG sets of three states. (G) Heatmap of DEGs according to pseudo‐time (<em>X</em>‐axis) for three trajectory states (<em>Y</em>‐axis).</p></figcaption></figure><p>To further demonstrate their malignancy, we compared the malignant cells identified in this study with all epithelial lineages in single‐cell transcriptome profiles of surgically resected LUAD samples (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE131907" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE131907</a>) classified into tS1 (mixed), tS2 (tumor), and tS3 (normal) states based on their epithelial state [<a href="#mco270335-bib-0022" class="usa-link" aria-describedby="mco270335-bib-0022">22</a>]. We assessed the similarity between our data and the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE131907" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE131907</a> data after eliminating batch effects (Figure <a href="#mco270335-supinfo-0001" class="usa-link">S1B</a>). The five clusters in the present study were clearly dissimilar from normal epithelial cells (tS3). Subclusters T2–T4 exhibited the highest similarity with tumor cells (tS2). Clusters T1 and T5 had the highest CNV scores, indicating malignancy. Therefore, the tumor cluster in this study comprised malignant cells.</p>
<p>The five tumor cell clusters evolved into three distinct branches (States 1–3; Figure <a href="#mco270335-fig-0002" class="usa-link">2E</a>). T1, predominantly associated with non‐responders (OR = 5.34; Figure <a href="#mco270335-fig-0002" class="usa-link">2B</a>), progressed to State 1. T2–T5, primarily found in responders (OR &lt; 0.64), branched into State 2 (T2–T4) or State 3 (T5). State 1 was characterized by activation of EMT, MET signaling, and angiogenesis pathways involving <em>FGF2</em>, <em>PTK2</em>, and <em>VEGFC</em> (Figure <a href="#mco270335-fig-0002" class="usa-link">2F–G</a>). State 2 was associated with the upregulation of antigen‐processing and interferon signaling pathways and maintained proliferation via VEGFA. The State 3 branch, T5, demonstrated regulation of Wnt signaling and cell matrix adhesion involving <em>BCAS3</em>, <em>SMURF2</em>, and <em>ROR1</em>.</p></section><section id="mco270335-sec-0050"><h3 class="pmc_sec_title">2.3. TME Involvement in the Response to ABCP Therapy</h3>
<p>Stromal cells were classified into five subclusters, four of which corresponded to CAF types previously established based on canonical markers: myofibroblastic (mCAFs), cycling (cCAFs), inflammatory (iCAFs), and antigen‐presenting (apCAFs) (Figure <a href="#mco270335-fig-0003" class="usa-link">3A,B</a>) [<a href="#mco270335-bib-0023" class="usa-link" aria-describedby="mco270335-bib-0023">23</a>]. We identified a novel subcluster, mesothelial CAFs (meCAFs) (expressing <em>UPK3B</em>, <em>CALB2</em>, and <em>WT1</em>, Figure <a href="#mco270335-fig-0003" class="usa-link">3C</a>). This subtype exhibited EMT reprogramming and resembled CAF subtypes identified in scRNA‐seq profiles of MPE samples from triple‐negative breast cancer [<a href="#mco270335-bib-0020" class="usa-link" aria-describedby="mco270335-bib-0020">20</a>]. Common CAF subtype markers included extracellular matrix‐remodeling genes (<em>COL1A2</em>, <em>ACTA2</em>, and <em>TAGLN</em>), although each subtype exhibited distinct fibrotic programs. meCAFs showed FGF and Hippo signaling activation (Figure <a href="#mco270335-fig-0003" class="usa-link">3D</a>). In contrast, mCAFs (<em>VEGFA</em>, <em>VCAN</em>, and <em>THBS2</em>) exhibited PDGF, VEGF, and angiogenic signaling activation.</p>
<figure class="fig xbox font-sm" id="mco270335-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9ce7/12365383/7ce1eaeb6f32/MCO2-6-e70335-g007.jpg" loading="lazy" id="jats-graphic-7" height="593" width="561" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/mco270335-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Stromal cell subclusters and their molecular mechanisms of cell–cell interactions. (A) Stromal cell subtype identification. UMAP of stromal cells, proportion bar plot of these cell types, and heatmap of canonical markers. CAFs were classified into five types: meCAFs (mesenchymal), mCAFs (myelofibrotic), cCAFs (cycling), iCAFs (inflammatory), and apCAFs (antigen‐presenting). These subtypes are indicated in five distinct colors. (B) Bar plot of pathway <em>p</em>‐values assessed by GSEA using differentially expressed genes for five CAF types. The circle size indicates the log‐scale <em>p</em>‐value, and the color indicates the number of detected genes. (C) Two cell–cell interaction networks for ABCP response groups. Nodes indicate cell types, and the edge width denotes the scale of cell–cell interaction counts. (D) Heatmap of cell–cell interactions for ligand–receptor pairing tumor subclusters with CAF subclusters. Interactions detected in the responder and non‐responder groups are denoted in red and blue, respectively. Signaling directions are colored differently on the left panel (green: tumor sender, yellow: tumor receiver). (G) <em>VEGFA</em>–KDR and <em>VEGFC</em>–<em>KDR</em> interaction between the tumor and CAF subcluster.</p></figcaption></figure><p>Although the other three CAF types comprised smaller fractions (&lt; 30%), subcluster identification revealed their main functional characteristics: cell cycle and Rho GTPases in cCAFs; IL6‐JAK‐Stat3, transforming growth factor‐beta (TGF‐β) receptor, and interferon‐gamma signaling in iCAFs; and antigen‐processing and T‐cell receptor signaling in apCAFs (Figure <a href="#mco270335-fig-0003" class="usa-link">3D</a>). meCAFs were enriched in ABCP non‐responders (OR = 2.81), and mCAFs in ABCP responders (OR = 0.51; Figure <a href="#mco270335-fig-0003" class="usa-link">3B</a>). Our findings align with previous classifications of three main CAF subtypes involved in activating tumor protection programs via FGF, PDGF, and TGF‐β signaling [<a href="#mco270335-bib-0024" class="usa-link" aria-describedby="mco270335-bib-0024">24</a>]. <em>VEGFA</em>‐<em>NRP2</em> interactions between mCAFs and T1 cell types were more frequently observed in ABCP responders (Figure <a href="#mco270335-fig-0003" class="usa-link">3E,F</a>). Meanwhile, tumor cell type T1 sent signals through <em>VEGFC</em>‐<em>KDR</em> interactions to meCAFs and iCAFs. VEGF signaling consists of complex interactions to maintain tumor proliferation or to facilitate TMEs. ABCP combination therapy seems to effectively target VEGFA‐mediated signaling received by tumor cells from CAF subtypes.</p>
<p>Lymphoid and myeloid cells represented a relatively small fraction (21.71%) (Figure <a href="#mco270335-fig-0001" class="usa-link">1D,E</a>), which is nearly half the fraction reported in a previous MPE single‐cell transcriptome profiling study [<a href="#mco270335-bib-0011" class="usa-link" aria-describedby="mco270335-bib-0011">11</a>]. Myeloid cells, primarily found in responders, were divided into five subclusters: alveolar macrophages, macrophages, inflammatory monocytes, proliferating monocytes, and dendritic cells (Figure <a href="#mco270335-supinfo-0001" class="usa-link">S2A</a>). Dendritic cells were more prevalent in non‐responders than in responders, although their population was small. Lymphoid cells were categorized into CD8<sup>+</sup> T cells, CD4<sup>+</sup> T cells, proliferating T cells, and B cells, which were evenly distributed between responders and nonresponders (Figure <a href="#mco270335-supinfo-0001" class="usa-link">S2B</a>). PD‐1 signaling was activated across CD4<sup>+</sup> T cells and B cells. Our lymphoid profile was insufficient to resolve the specific cell subtypes or to infer their association with ABCP response. Subclustering analysis of tumor‐associated macrophages (TAM) identified three distinct subtypes: <em>SPP1</em>
<sup>+</sup> TAM, M1 macrophages, and M2 macrophages (Figure <a href="#mco270335-supinfo-0001" class="usa-link">S2C</a>). M2 macrophages were predominantly derived from non‐responders (OR = 9.43), whereas M1 macrophages were enriched in responders (OR = 2.31). Our results provide evidence that M2 TAM contributes to PD‐1 blockade resistance by T‐cell exclusion from TME [<a href="#mco270335-bib-0025" class="usa-link" aria-describedby="mco270335-bib-0025">25</a>].</p></section><section id="mco270335-sec-0060"><h3 class="pmc_sec_title">2.4. Cell–Cell Communication Emerges as a Therapeutic Vulnerability</h3>
<p>To assess global cell–cell communication, we extrapolated interactions among all subclusters assorted according to the ABCP response and cell type. Interestingly, the cell interaction network divided into ABCP responses revealed distinct communication patterns (Figure <a href="#mco270335-fig-0003" class="usa-link">3E–G</a>). In responders, mCAFs exhibited extensive interactions with meCAFs, cCAFs, and iCAFs and T2 and T3 tumor cells, whereas in non‐responders, all five CAF subtypes communicated intensively with each other and with T1 tumor cells (Figure <a href="#mco270335-fig-0003" class="usa-link">3E</a>).</p>
<p>When analyzing ligand–receptor interactions, we identified subcluster‐specific and response‐specific interactions (Figure <a href="#mco270335-fig-0003" class="usa-link">3F</a>). Tumor–stroma interactions were the most prevalent in both responders and non‐responders. Notably, in non‐responders, T1 tumor cells were engaged in an interplay with meCAFs via VEGFC–KDR signaling (kinase insert domain receptor; VEGF receptor 2) and exclusively received FGF2–FGFR1 signals from meCAFs (Figure <a href="#mco270335-fig-0003" class="usa-link">3F</a>). Additionally, CXCL11/CXCL12–ACKR3 interactions in apCAFs promoted tumor progression in non‐responders [<a href="#mco270335-bib-0026" class="usa-link" aria-describedby="mco270335-bib-0026">26</a>].</p>
<p>Responder data revealed an interaction between epidermal growth factor (EGF) on T5 cells and EGFR on iCAFs, whereas TNFSF9 on T4 cells interacted with HLA‐DPA1 on apCAFs, which is favorable for immunotherapy [<a href="#mco270335-bib-0027" class="usa-link" aria-describedby="mco270335-bib-0027">27</a>]. CXCL9/CXCL10/CXCL11–DPP4 interactions, known to attenuate anticancer immunity, protect the T3 subtype [<a href="#mco270335-bib-0028" class="usa-link" aria-describedby="mco270335-bib-0028">28</a>]. Among angiogenesis regulators, the VEGFA–NRP2 signal, transmitted from mCAFs to T1 subtype cells, was commonly detected in both responders and non‐responders (Figure <a href="#mco270335-fig-0003" class="usa-link">3F</a>). However, interactions between VEGFA or VEGFC and KDR displayed distinct patterns according to the cell type and ABCP response (Figure <a href="#mco270335-fig-0003" class="usa-link">3G</a>). While VEGFA, expressed by all tumor cell types, facilitated tumor cell proliferation, the VEGFC–KDR signal was restricted to interactions between specific T1 tumor cells and other CAF subtypes, promoting the development of meCAFs. Our analysis implies that VEGF ligands could exert differential effects across cell types: VEGFA may be linked to tumor cell proliferation, while VEGFC appears to support CAF‐mediated tumor protection.</p></section><section id="mco270335-sec-0070"><h3 class="pmc_sec_title">2.5. Spatial scRNA‐Seq Reveals Tumor Protection Resulting From ABCP Treatment</h3>
<p>To validate our observations, we generated spatially resolved single‐cell transcriptomic profiles using tumor samples obtained from TKI‐resistant patients (<em>n</em> = 2) who had progressed following second‐line osimertinib treatment. We detected 4518–4886 cell spots and assigned cell types to the spatial transcriptome profiles based on cell scoring referring to the above cell subclusters and known global LUAD cell types (Figure <a href="#mco270335-fig-0004" class="usa-link">4A,B</a>, Figure <a href="#mco270335-supinfo-0001" class="usa-link">S3A,D</a>) [<a href="#mco270335-bib-0022" class="usa-link" aria-describedby="mco270335-bib-0022">22</a>]. The abundance of stromal cell populations (22.86%–24.30%) resembled that of other TKI‐resistant cells (fibroblasts and endothelial cells: &gt; 20%; Figure <a href="#mco270335-fig-0004" class="usa-link">4A,B</a>, Figure <a href="#mco270335-supinfo-0001" class="usa-link">S3C</a>) [<a href="#mco270335-bib-0002" class="usa-link" aria-describedby="mco270335-bib-0002">2</a>].</p>
<figure class="fig xbox font-sm" id="mco270335-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9ce7/12365383/d1abf5ce10f5/MCO2-6-e70335-g001.jpg" loading="lazy" id="jats-graphic-9" height="483" width="561" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/mco270335-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Spatial transcriptome profile of EGFR‐TKI‐resistant patient tissue. Capture area was 6.5 × 6.5 mm. (A) Two bar plots of cell proportions according to cell types and cell lineages identified from spatial transcriptome profiles. (B) Spatial cell type distribution. (C) Heatmap of gene expression and pathway scores for each cell type. (D) Spatial expression of VEGFA and VEGFC on a spatial tissue biopsy. Circular plots for VEGFA–KDR and VEGFC–KDR interactions between two cell types. (E) Two bar plot ORs between cell lineages and gene expression (<em>VEGFA</em> and <em>VEGFC</em>). (F) Bar plots for correlation coefficients of VEGFA or VEGFC expression with stromal markers assessed from IHC of patient tumor biopsies (<em>n</em> = 29). Color indicates the <em>R</em> value. The <em>X</em>‐axis indicates the <em>p</em>‐value. The <em>Y</em>‐axis indicates IHC marker sets (FGFs: FGF1 and FGF2, EMT: CDH2 and VIM, angiogenesis: POSTIN).</p></figcaption></figure><p>
<em>VEGFA</em> and <em>VEGFC</em> were expressed in different locations. <em>VEGFA</em> was notably overexpressed in tumor subcluster T2 (Figure <a href="#mco270335-fig-0004" class="usa-link">4C–E</a>, Figure <a href="#mco270335-supinfo-0001" class="usa-link">S3E–G</a>), whereas <em>VEGFC</em> expression was observed in mCAFs and iCAFs, alongside the activation of EMT, PD‐1 signaling, YAP/TAZ signaling, and other fibrosis pathways, except Hippo signaling. Globally, <em>VEGFA</em> expression was over‐expressed in epithelial lineages rather than stromal cells, whereas <em>VEGFC</em> was expressed in stromal and lymphoid cells (Figure <a href="#mco270335-fig-0004" class="usa-link">4E</a>). To further demonstrate co‐localization, we performed IHC using tissues acquired from EGFR‐TKI‐resistant patients (<em>n</em> = 30). Co‐localization between VEGF signal and TME‐associated markers was assessed in tissues (FGF signal: FGF1/2, angiogenesis: POSTN, and EMT: CDH2 and VIM; Figure <a href="#mco270335-fig-0004" class="usa-link">4F</a>). <em>VEGFC</em> and receptors <em>VEGFR3</em>/<em>NRP2</em> exhibited a strong positive correlation with angiogenesis (<em>R</em> = 0.57) and FGFs (<em>R</em> = 0.82). VEGFA expression globally exhibited no correlation with EMT (CDH2<sup>+</sup> VIM<sup>+</sup>; <em>R</em> = 0.26) and FGF signal (<em>R</em> = 0.25). In summary, <em>VEGFA</em> and <em>VEGFC</em> were expressed in mutually exclusive cell regions in epithelial and CAF cells of our TKI‐resistant patients, respectively. The correlation between FGF and VEGFA showed a moderate negative correlation, whereas VEGFC strongly interacted with FGF‐activated CAF subtypes, with angiogenesis.</p>
<p>VEGFA–KDR interaction was observed between tumor cell subclusters and immune cell types (Figure <a href="#mco270335-fig-0004" class="usa-link">4D</a>), whereas the VEGFC–KDR interaction signal was exclusive to CAF cells. Consistent results were obtained for another TKI‐resistant sample showing <em>VEGFA</em> overexpression in tumor cell types and mutual exclusiveness of VEGFA and VEGFC (Figure <a href="#mco270335-supinfo-0001" class="usa-link">S3E</a>). Taken together, these findings indicated that VEGFA in tumor cells appears to maintain tumor cell proliferation, whereas VEGFC in TME cells protects tumor cells via mCAF development with FGF signaling activation or YAP‐TAZ regulation.</p></section><section id="mco270335-sec-0080"><h3 class="pmc_sec_title">2.6. Angiogenic and Fibroblastic Signals According to Therapeutic Status</h3>
<p>To uncover the dynamic changes in angiogenic and fibroblastic signals according to therapeutic status, we used an scRNA‐seq dataset of dissected tumor biopsies (<em>n</em> = 49) from three TKI therapeutic groups: TKI‐naïve (TN), residual disease (RD) during treatment and stable status, and progressive disease (PD) [<a href="#mco270335-bib-0029" class="usa-link" aria-describedby="mco270335-bib-0029">29</a>]. As these samples were collected during disease progression following TKI treatment of the primary tumor, the target mechanisms of ABCP could be dissected in the context of stepwise resistance acquisition. Cell types were defined referring to the original report (Figure <a href="#mco270335-fig-0005" class="usa-link">5A</a>). In RD, the endothelial cell and fibroblast populations were twice as abundant as others (Figure <a href="#mco270335-fig-0005" class="usa-link">5A</a>), implying that angiogenic and fibroblastic development is amplified in RD. When examining the above markers identified in ABCP non‐responders, we found that <em>VEGFA</em> expression was upregulated in RD, <em>VEGFC</em> expression in PD, and VEGF receptor gene expression in RD (Figure <a href="#mco270335-fig-0005" class="usa-link">5B</a>). <em>VEGFA</em> was upregulated by 1.103‐ to 1.110‐fold in RD and PD epithelial cells compared to TN (Figure <a href="#mco270335-supinfo-0001" class="usa-link">S4</a>). <em>VEGFC</em> expression showed a 6.04‐fold increase in PD epithelial cells compared to TN and RD, although its absolute expression level was lower than that of <em>VEGFA</em>. VEGF receptor gene expression showed a twofold increase in RD endothelial cells, whereas co‐receptor genes had the highest expression levels in PD epithelial cells. FGF signaling was activated, along with an increase in fibroblasts, in RD. In summary, VEGFA secreted from tumor cells exhibits predominant expression starting in the TN stage. Alternatively, VEGFC originating from fibroblasts induces tumor cell progression from TN to PD. Endothelial cell development via VEGF receptors is directly relevant to ongoing TKI treatment. Meanwhile, the co‐receptors NRP1 and NRP2 play cooperative roles in epithelial cells and fibroblasts during treatment and are co‐expressed with VEGF receptors in endothelial cells. This implies that the complexity of VEGF signal interaction increases after TKI treatment and progresses to dominant VEGFA regulation, supporting the need for diversified interventions to target interactions with TME cells.</p>
<figure class="fig xbox font-sm" id="mco270335-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9ce7/12365383/16bd24c5fbbe/MCO2-6-e70335-g004.jpg" loading="lazy" id="jats-graphic-11" height="508" width="561" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/mco270335-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Demonstration of VEGF signal genes and cooperative pathways using therapy‐induced transcriptome dynamics. TKI treatment groups were classified into three types: TN: treatment‐naïve, blue; RD: residual disease, yellow‐green; PD: progressive disease, orange). (A) Bar plots of cell proportions for three therapy groups and four cell lineages. (B) Heatmap of average gene expression of VEGF and FGF signaling for treatment groups. (C) Correlation coefficient bar plots of stromal‐associated pathways with <em>VEGFA</em> and <em>VEGFC</em>. Bar color indicates correlation coefficients. The X‐axis indicates the log‐scale <em>p</em>‐value. (D) Scatter plots of <em>VEGFC</em> expression (X‐axis) and pathway scores (Y‐axis). Each spot is an expression value calculated by pseudo‐bulking for each patient, each cell type, and all cell types. Dots are colored by therapy groups.</p></figcaption></figure><p>To investigate cooperative functions with global <em>VEGFA</em> and <em>VEGFC</em> expression, we assessed correlations with the stromal pathways extracted from the ABCP cohort (Figure <a href="#mco270335-fig-0005" class="usa-link">5C</a>). <em>VEGFC</em> expression showed predominantly positive correlations with five stromal pathways, particularly, YAP‐TAZ and EMT (<em>p</em> &lt; 0.001), whereas TGF‐β was associated with <em>VEGFA</em> expression. We next decomposed VEGFC‐associated signals to cell types for YAP‐TAZ and FGF signaling (Figure <a href="#mco270335-fig-0005" class="usa-link">5D</a>). In epithelial and endothelial cells, <em>VEGFC</em> expression stimulated the upregulation of YAP‐TAZ and in endothelial cells, it promoted FGF signaling activation (<em>p</em> &lt; 0.05). Thus, the increase in <em>VEGFC</em> expression following therapeutic progression facilitates endothelial cell development via YAP‐TAZ regulation and FGF signaling activation.</p></section><section id="mco270335-sec-0090"><h3 class="pmc_sec_title">2.7. Global Regulation Programs and Prognosis Governed by VEGFA and VEGFC</h3>
<p>To dissect the differential global regulatory programs governed by VEGFA and VEGFC, we conducted gene–gene interaction network analysis using bulk RNA‐seq transcriptome profiles (META1460, <em>n</em> = 1460) obtained from several NSCLC cohorts (Figure <a href="#mco270335-fig-0006" class="usa-link">6A</a>, Table <a href="#mco270335-supinfo-0001" class="usa-link">S4</a>). We established co‐expression interaction networks regulated by VEGFA and VEGFC. Submodules of these networks were identified, and their regulatory pathways were investigated through gene set enrichment analysis. For further investigation of clinical profiles, we employed our National Cancer Center (NCC) PDC cohort (<em>n</em> = 95; Table <a href="#mco270335-supinfo-0001" class="usa-link">S5</a>) data acquired from MPE samples [<a href="#mco270335-bib-0019" class="usa-link" aria-describedby="mco270335-bib-0019">19</a>].</p>
<figure class="fig xbox font-sm" id="mco270335-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9ce7/12365383/7bfdc7167d57/MCO2-6-e70335-g005.jpg" loading="lazy" id="jats-graphic-13" height="557" width="560" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/mco270335-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Meta‐transcriptome analysis of NSCLC reveals global regulatory programs of <em>VEGFA</em> and <em>VEGFC</em>. Samples were classified into two types by VEGF gene expression. The high group is the upper median, and the low group is the lower median grouped by <em>VEGFA</em> and <em>VEGFC</em> gene expression. EGFR‐TKI treatment profiles were categorized into three groups: base, baseline (green); responder, R (red); non‐responder, NR (blue). (A) Gene–gene interaction network regulated by VEGFC. Nodes are colored by network modules, and pathways are highlighted based on gene set enrichment analysis. (B) Violin plots for YAP‐TAZ score according to <em>VEGFA</em> and <em>VEGFC</em> gene expression for META1460 and NCC PDC cohort patients. (C) Survival plots according to <em>VEGFA</em> and <em>VEGFC</em> expression for overall survival of the NCC PDC cohort. (D) Forest plot for HR by overall survival according to VEGF gene and pathway scores of the META1460 profile. (E) Two heatmaps of scores and correlation coefficients. Left: heatmap for scores for cell types and pathways according to three groups. *<em>p</em> &lt; 0.05 versus responders. Right: heatmap for correlation coefficients with <em>VEGFA</em> and <em>VEGFC</em> expression. (F) Violin plots for <em>VEGFA</em> and <em>VEGFC</em> gene expression according to EGFR‐TKI treatment groups.</p></figcaption></figure><p>Network analysis identified common pathway modules between VEGFA and VEGFC involved in extracellular matrix organization, angiogenesis, VEGF signaling, and PDGF‐β signaling (Figure <a href="#mco270335-supinfo-0001" class="usa-link">S5</a>). We also found distinct modules, suggesting potential differences in cascading or cooperative signaling mechanisms employed by VEGFA and VEGFC. VEGFA was associated with oncogenic signals such as ERBB2, RAS, mTOR, and the p53 pathway [<a href="#mco270335-bib-0030" class="usa-link" aria-describedby="mco270335-bib-0030">30</a>], whereas VEGFC cooperated with TME development through pathways including TGF‐β, T‐cell activation, interleukin signaling, Rho GTPase activation, and Toll‐like receptor signaling (Figure <a href="#mco270335-fig-0006" class="usa-link">6A</a>, Figure <a href="#mco270335-supinfo-0001" class="usa-link">S5</a>) [<a href="#mco270335-bib-0031" class="usa-link" aria-describedby="mco270335-bib-0031">31</a>]. <em>FGF1</em> and <em>FGF7</em> were detected in the VEGFC co‐expression network, in the TGF‐β and Toll‐like receptor modules, respectively. YAP‐TAZ upregulation by VEGFC was found in both the META1460 and NCC cohorts (VEGFC, <em>p</em> &lt; 0.001; VEGFA, <em>p</em> = 0.014; Figure <a href="#mco270335-fig-0006" class="usa-link">6B</a>). Overall, these co‐expression networks elucidated the distinct roles of the two VEGF ligands: VEGFA contributes to tumor proliferation, whereas VEGFC potentiates desmoplasia and anticancer immunity.</p>
<p>To assess the clinical outcomes associated with the expression of these genes, we performed survival analysis. In the refractory lung cancer NCC PDC cohort, <em>VEGFC</em> expression upregulation was significantly correlated with poor overall survival (<em>p</em> = 0.015, hazard ratio [HR] = 1.8, Figure <a href="#mco270335-fig-0006" class="usa-link">6C</a>). In the NSCLC‐integrated cohort META1460, <em>VEGFC</em> expression was a stronger predictor of overall survival (<em>p</em> &lt; 0.001; HR = 1.51) than other pathway markers (Figure <a href="#mco270335-fig-0006" class="usa-link">6D</a>).</p>
<p>We additionally analyzed NCC PDC cohort EGFR‐TKI response profiles (EGFR‐TKI groups: baseline <em>n</em> = 15; responder <em>n</em> = 38; non‐responder <em>n</em> = 42) to characterize the therapy‐induced status of VEGFs and additional signatures (Figure <a href="#mco270335-fig-0006" class="usa-link">6E</a>, Table <a href="#mco270335-supinfo-0001" class="usa-link">S5</a>). EGFR‐TKI non‐responders showed increased <em>VEGFC</em> expression (<em>p</em> = 0.023), whereas <em>VEGFA</em> remained consistent across all therapeutic statuses (Figure <a href="#mco270335-fig-0006" class="usa-link">6F</a>). FGF, YAP‐TAZ signaling, and meCAF scores were elevated in non‐responders, as was <em>VEGFC</em> expression (<em>p</em> = 0.054). These cell and pathway scores were mostly correlated with <em>VEGFC</em> expression rather than <em>VEGFA</em> expression (Figure <a href="#mco270335-fig-0006" class="usa-link">6E</a>). Particularly, EMT and YAP‐TAZ exhibited the strongest correlations (<em>R</em> = 0.691, <em>p</em> &lt; 0.001). These findings suggested that, while a VEGFA‐targeted drug such as bevacizumab may be effective from baseline, TKI‐non‐responders have an increased risk of bevacizumab failure due to VEGFC in cooperation with FGF signaling and YAP‐TAZ regulation. Therefore, targeting VEGFC or blockade of its interactor KDR should be considered for TKI‐non‐responders.</p></section><section id="mco270335-sec-0100"><h3 class="pmc_sec_title">2.8. Combinational Knockdown of <em>VEGFC</em> and Its Receptors Blocks Drug Resistance and Angiogenesis by Inhibiting FGF/YAP/TAZ</h3>
<p>To elucidate the role of VEGFs/VEGF receptors in drug resistance, we conducted in vitro assays using lung cancer cell lines classified as sensitive or resistant based on VEGF/VEGF receptor (<em>VEGFC</em>, <em>KDR</em>, and <em>NRP2</em>) expression and IC<sub>50</sub> values of anti‐VEGF drugs (cabozantinib, pazopanib, and sitravatinib). H1573, H2444, and H2279 cells were categorized as resistant, and H1581 and H1437 as sensitive (Figure <a href="#mco270335-supinfo-0001" class="usa-link">S6A,B</a>).</p>
<p>To assess whether VEGFC, KDR, and NRP2 are required for the anti‐VEGF drug response, we investigated the effects of cabozantinib on the proliferation of resistant cells with or without shRNA‐mediated depletion of <em>VEGFA</em>, <em>VEGFC</em>, <em>KDR</em>, and <em>NRP2</em>. Silencing of both <em>VEGFC</em> and <em>KDR</em> or <em>NRP2</em> significantly decreased cell proliferation compared to a scramble control or single‐gene silencing in the three resistant cell lines (Figure <a href="#mco270335-fig-0007" class="usa-link">7A</a> and Figure <a href="#mco270335-supinfo-0001" class="usa-link">S6C,D</a>). However, the effect of <em>VEGFA</em> knockdown was not significantly stronger than that of <em>VEGFC</em> knockdown when co‐silenced with <em>KDR</em> or <em>NRP2</em> (Figure <a href="#mco270335-supinfo-0001" class="usa-link">S6E</a>). Two well‐characterized anti‐VEGF drugs, bevacizumab and ramucirumab, did not demonstrate combinational effects with <em>VEGFC/KDR/NRP2</em> silencing, as they did not affect cell proliferation in vitro, which is consistent with previous findings [<a href="#mco270335-bib-0032" class="usa-link" aria-describedby="mco270335-bib-0032">32</a>, <a href="#mco270335-bib-0033" class="usa-link" aria-describedby="mco270335-bib-0033">33</a>].</p>
<figure class="fig xbox font-sm" id="mco270335-fig-0007"><h4 class="obj_head">FIGURE 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9ce7/12365383/de1062449dea/MCO2-6-e70335-g002.jpg" loading="lazy" id="jats-graphic-15" height="738" width="561" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/mco270335-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Inhibition of the VEGF pathway significantly enhances the anticancer effects of VEGF‐targeted drugs in both in vitro and in vivo conditions. (A) Bar graphs of the relative cell proliferation ratio following cabozantinib treatment with or without specific target gene knockdown in H2444 (left) and H2279 (right) cell lines. The black bar indicates DMSO treatment, and the blue bars indicate cabozantinib treatment (<em>n</em> = 3). (B) Bar graphs of qPCR results (<em>n</em> = 3) visualizing the expression of <em>FGF2</em> (left) and <em>AXL</em> (right) following target gene silencing with or without cabozantinib treatment (marked by color bars). (C) Experimental scheme outlining in vivo mouse experiments with various treatment options: human IgG (Control), bevacizumab (VEGFA‐targeted antibody), ramucirumab (KDR/VEGFR2 targeted antibody), and bevacizumab + ramucirumab. (D) Tumor growth curves of each treatment group. H2444 cells were subcutaneously implanted into flank of mice and treated accordingly. The left plot shows tumor growth across all four groups, while the right plot highlights only the bevacizumab‐treated group and bevacizumab + ramucirumab‐treated group for clear comparison between two groups. (E) Representative image of the tumor masses (left) and corresponding tumor weights (right) from H2444‐engrafted nude mice, scale bar = 1 cm. (F) Representative CD31‐stained IHC images from each tumor sample. Scale bar = 200 µm. (G) RT‐qPCR results (<em>n</em> = 6) for the expression of <em>FGF1, YAP, TAZ, AXL</em> in xenografted mouse tumor samples with various treatments. *<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001, ****<em>p</em> &lt; 0.0001, one‐way ANOVA followed by Tukey's multiple comparison tests.</p></figcaption></figure><p>We found a positive correlation between <em>VEGFC/KDR/NRP2</em> expression and the FGF pathway (associated with angiogenesis) and YAP/TAZ/AXL pathway (related to drug resistance) in non‐responders (Figure <a href="#mco270335-fig-0003" class="usa-link">3</a>). As these pathways exhibit compensatory activation upon cabozantinib treatment in resistant cells [<a href="#mco270335-bib-0034" class="usa-link" aria-describedby="mco270335-bib-0034">34</a>], we investigated whether <em>VEGFC/KDR/NRP2</em> knockdown elicited a decrease in <em>FGF/YAP/TAZ/AXL</em> expression in resistant lung cancer cells. Combined knockdown of <em>VEGFC</em> and <em>KDR</em> or <em>NRP2</em> efficiently diminished cabozantinib‐mediated gene expression compared to scramble or single knockdown (Figure <a href="#mco270335-fig-0007" class="usa-link">7B</a> and Figure <a href="#mco270335-supinfo-0001" class="usa-link">S6F</a>).</p>
<p>Next, we performed in vitro angiogenesis assays using HUVECs cultured in conditioned medium derived from lung cancer cells with or without shRNA. HUVECs cultured in conditioned medium from cells silenced for both <em>VEGFC</em> and <em>KDR</em> or <em>NRP2</em> exhibited decreased tube formation and tube length compared to HUVECs cultured in conditioned medium derived from control or single‐silenced cells (Figure <a href="#mco270335-supinfo-0001" class="usa-link">S6G</a>). These findings demonstrated that VEGFC–KDR and VEGFC–NRP2 play critical roles in drug resistance, angiogenesis, and related gene expression in drug‐resistant lung cancer cell lines.</p></section><section id="mco270335-sec-0110"><h3 class="pmc_sec_title">2.9. KDR (VEGFR2) Blockade by Ramucirumab Enhances Sensitivity to Bevacizumab</h3>
<p>To assess the antitumor efficacy of VEGFC–KDR blockade, we evaluated the efficacy of ramucirumab in H2444 cell‐engrafted nude mice. After subcutaneous injection, xenografted mice were randomly divided into four groups (<em>n</em> = 6 per group) and injected intraperitoneally with bevacizumab (2 mg/kg), ramucirumab (5 mg/kg), bevacizumab plus ramucirumab, or human IgG control (7 mg/kg) once a week (Figure <a href="#mco270335-fig-0007" class="usa-link">7C</a>). Tumor growth and size were monitored. As expected, mice treated with human IgG exhibited the highest tumor growth rate, with the largest tumor size observed 49 days after H2444 cell injection. Single treatment with bevacizumab or ramucirumab reduced tumor growth and size. Recipients of H2444 cells with bevacizumab and ramucirumab showed the lowest tumor growth rate (Figure <a href="#mco270335-fig-0007" class="usa-link">7D,E</a>) and size, without any loss in body weight (data not shown). Consistent with in vitro experimental results, cotreatment with bevacizumab and ramucirumab drastically diminished tumor angiogenesis, as evidenced by decreased CD31‐positive staining (Figure <a href="#mco270335-fig-0007" class="usa-link">7F</a>), and suppressed the expression of angiogenesis and drug resistance‐related genes, <em>FGF</em> and <em>YAP/TAZ/AXL</em> (Figure <a href="#mco270335-fig-0007" class="usa-link">7G</a>).</p></section></section><section id="mco270335-sec-0120"><h2 class="pmc_sec_title">3. Discussion</h2>
<p>Although ABCP treatment is a promising option for EGFR‐TKI resistant patients to improve PFS, the therapeutic mechanism of non‐responsiveness is still unclear. Single cell transcriptome analyses have revealed the characteristics of the response of NSCLC neoadjuvant immunotherapy combined with chemotherapy [<a href="#mco270335-bib-0035" class="usa-link" aria-describedby="mco270335-bib-0035">35</a>, <a href="#mco270335-bib-0036" class="usa-link" aria-describedby="mco270335-bib-0036">36</a>, <a href="#mco270335-bib-0037" class="usa-link" aria-describedby="mco270335-bib-0037">37</a>]. These studies demonstrated that TME populations, such as TAM, NK cells, or <em>CCR8</em>
<sup>+</sup> regulatory T cells, are associated with therapeutic response. While these studies provide a comprehensive view of the TME landscape and its role in therapy, the mechanisms underlying responses related to VEGF signaling are not yet fully understood. Therefore, we assessed various transcriptomes from TKI‐resistant NSCLC and uncovered molecular features that disrupt the response to ABCP. Our results indicate that the VEGFC signal plays an important role in the development of fibroblasts regulated by the FGF signal and YAP/TAZ/AXL axis, as well as angiogenesis. Notably, VEGFC exhibited a stronger prognostic potential compared to VEGFA.</p>
<p>VEGF signaling network is a complex ligand–receptor system to play a role in angiogenesis, lymphangiogenesis, and vascular permeability [<a href="#mco270335-bib-0007" class="usa-link" aria-describedby="mco270335-bib-0007">7</a>]. VEGF ligands exhibit selective binding affinities: VEGFA primarily binds to VEGFR1/R2, whereas VEGFC/D predominantly bind to FLT4 (VEGFR3). Additionally, co‐receptor NRP1 enhances VEGFA‐VEGFR2 interaction, and NRP2 facilitates VEGFC/D interactions with their receptors. It implies that VEGFA inhibition alone could not perfectly block VEGF‐mediated intercellular communication due to alternative ligand and co‐receptor interactions. Here, we identified <em>VEGFA</em>‐expressing tumor cells in ABCP responders, which is consistent with previous LUAD studies [<a href="#mco270335-bib-0022" class="usa-link" aria-describedby="mco270335-bib-0022">22</a>, <a href="#mco270335-bib-0038" class="usa-link" aria-describedby="mco270335-bib-0038">38</a>]. In ABCP non‐responders, <em>VEGFC</em> expression was substantially lower than <em>VEGFA</em> expression. Computational scores for VEGF signaling or angiogenesis pathways predominantly followed the expression trends of <em>VEGFA</em> and its receptors, showing increased abundance. To precisely assess the cell interaction complexity of VEGF signaling, we comprehensively investigated all ligand–receptor pairs using bulk transcriptome data. Interestingly, VEGFC showed no cell‐type‐specific expression across tumor cells and CAFs. However, the final phenotypic condition by VEGFC exhibited concurrent pathway activation and stromal development. <em>VEGFC</em> expression was significantly correlated with fibroblastic pathways, including FGF, TGF‐β, and PDGF‐β signaling. Increased FGF signaling is associated with resistance to anti‐VEGF agents [<a href="#mco270335-bib-0039" class="usa-link" aria-describedby="mco270335-bib-0039">39</a>]. VEGFC activates the canonical Hippo pathway in lymphatic endothelial cells, affecting YAP‐TAZ signaling [<a href="#mco270335-bib-0040" class="usa-link" aria-describedby="mco270335-bib-0040">40</a>]. In the tumor ecosystem, the biological functions of VEGFA and VEGFC are primarily linked to tumor angiogenesis and lymphangiogenesis. Our findings further emphasize the role of VEGFC in cooperating with fibroblastic regulation in tumor development and resistance, highlighting the specific mechanisms by which CAFs interact with VEGF signaling.</p>
<p>High tumoral VEGFC expression poses a higher risk in terms of survival than high VEGFA expression in NSCLC [<a href="#mco270335-bib-0041" class="usa-link" aria-describedby="mco270335-bib-0041">41</a>]. Moreover, VEGFC is a better biomarker for predicting poor outcomes than other representative fibroblastic regulators. Our NCC PDC refractory lung cancer cohort includes multiple TKI‐progressed patients, making it a valuable resource for developing ABCP strategies that consider TKI treatment status. Targeting VEGFA may be effective as an initial therapy, whereas targeting VEGFC would be more beneficial after the acquisition of TKI resistance.</p>
<p>While advancements in anti‐VEGF drugs such as bevacizumab have allowed targeted inhibition of VEGFA, these drugs have various side effects and trigger the upregulation of other factors promoting angiogenesis and drug resistance [<a href="#mco270335-bib-0042" class="usa-link" aria-describedby="mco270335-bib-0042">42</a>, <a href="#mco270335-bib-0043" class="usa-link" aria-describedby="mco270335-bib-0043">43</a>]. VEGFC mediates glioblastoma survival, tumorigenicity, and bevacizumab resistance via KDR activation [<a href="#mco270335-bib-0044" class="usa-link" aria-describedby="mco270335-bib-0044">44</a>]. These findings suggest a potential role for VEGFC/KDR signaling in VEGF‐targeted drug resistance. Our scRNA‐seq analysis identified VEGFC/KDR as key molecules for ABCP resistance, and in vitro and in vivo models successfully revealed the resistance mechanisms of VEGFC/KDR to anti‐VEGF agents in NSCLC. Treatment with ramucirumab, a KDR‐targeting monoclonal antibody, in combination with bevacizumab efficiently reduced tumor growth and angiogenesis in an in vivo xenograft mouse model, suggesting a novel therapeutic strategy for drug‐resistant patients with NSCLC. Moreover, combined knockdown of <em>VEGFC</em> and its receptor (<em>KDR</em> or <em>NRP2</em>), targeting the VEGFC/KDR pathway, significantly enhanced the antitumor effects of cabozantinib in vitro and drastically suppressed the expression of cabozantinib‐induced drug resistance genes, <em>YAP/TAZ</em> and <em>FGF</em>. The YAP‐TAZ targets AXL and FGFs were upregulated in TKI‐resistant cells, and clinical trials and new drug developments for the inhibition of AXL and FGFs are ongoing [<a href="#mco270335-bib-0045" class="usa-link" aria-describedby="mco270335-bib-0045">45</a>, <a href="#mco270335-bib-0046" class="usa-link" aria-describedby="mco270335-bib-0046">46</a>]. Therefore, developing combination therapies that inhibit FGF and YAP‐TAZ may further improve therapeutic success.</p>
<p>This study has several limitations. First, MPE samples exhibit less cellular heterogeneity than tumor biopsies [<a href="#mco270335-bib-0035" class="usa-link" aria-describedby="mco270335-bib-0035">35</a>, <a href="#mco270335-bib-0036" class="usa-link" aria-describedby="mco270335-bib-0036">36</a>, <a href="#mco270335-bib-0037" class="usa-link" aria-describedby="mco270335-bib-0037">37</a>]. The ABCP patient sample size was insufficient to clearly determine molecular features according to ABCP response. However, MPE can be used for tumor progression diagnosis without a biopsy burden, and we successfully uncovered the ABCP resistance mechanism from the limited samples. To overcome this limitation, we validated our findings using various sequencing platforms and a meta‐cohort of over 1000 samples. Second, the transcriptomic characteristics of our samples resembled those of tumor biopsies, but the reliability of interactions between tumor and TME cells may be limited. Therefore, we demonstrated concordance between our results and tumor tissues by performing cross validations using spatial scRNA‐seq and external tumor scRNA‐seq datasets. Furthermore, our samples exhibited an inadequate landscape of immune cells to infer the association with therapeutic response. Other immunotherapy studies could explore comprehensive mechanisms of anti‐PD1 therapy to cooperate with immune cells, including macrophages and T‐cell subtypes. In our analysis of macrophage subtypes, we observed that M2‐TAM showed less responsive to ABCP. However, the dynamics of macrophage polarization during immunotherapy and the complex crosstalk among the tumor and various immune cell types remain to be fully elucidated.</p>
<p>In summary, we profiled single‐cell transcriptomes of EGFR‐TKI‐resistant NSCLC patients to classify the molecular features determining the response to ABCP. Cross‐evaluations in multiple cohorts highlighted that VEGFC cooperates with fibroblastic mechanisms and promotes worse outcomes. Our results provide for potential combination therapy to overcome EGFR‐TKI resistance, and we anticipate an accelerated development of various anti‐VEGF agents.</p></section><section id="mco270335-sec-0130"><h2 class="pmc_sec_title">4. Methods</h2>
<section id="mco270335-sec-0140"><h3 class="pmc_sec_title">4.1. Patient Population</h3>
<p>We established lung cancer cohort on August 22, 2016 and have prospectively collected clinical information and tumor samples (IRB approval number: NCC2016‐0208). We also established PDCs from malignant pleural effusion samples. Among these, we retrospectively analyzed outcomes from patients who had progressed following prior EGFR‐TKI treatment (Table <a href="#mco270335-supinfo-0001" class="usa-link">S1</a>). Transcriptomic profiles from both scRNA‐seq and spatial scRNA‐seq were obtained from patients progressed to stage IV after EGFR‐TKI treatment (Table <a href="#mco270335-supinfo-0001" class="usa-link">S2</a>).</p></section><section id="mco270335-sec-0150"><h3 class="pmc_sec_title">4.2. Sample Preparation for Single Cell RNA‐Sequencing</h3>
<p>All details were included in the <a href="#mco270335-supinfo-0001" class="usa-link">Supporting Information</a>.</p></section><section id="mco270335-sec-0160"><h3 class="pmc_sec_title">4.3. Target‐Seq and scRNA‐seq Analyses of ABCP‐Treated Patients</h3>
<p>Mutation profiles were acquired from a previous study in an equal NCC PDC cohort [<a href="#mco270335-bib-0019" class="usa-link" aria-describedby="mco270335-bib-0019">19</a>]. Sample preparation and target‐seq analysis were conducted using the same protocol used in the previous study, following The Cancer Genomic Atlas Genomic Data Commons pipeline. To acquire single‐cell transcriptome profiles, samples were labeled using a BD Single‐Cell Multiplexing Kit (BD Biosciences, Franklin Lakes, NJ, USA), and cDNA libraries were constructed using the BD Rhapsody system, a BD Rhapsody Whole Transcriptome Analysis (WTA) Amplification Kit, and a BD Rhapsody cDNA Kit (BD Biosciences). Paired‐end (2 × 150 bp) sequencing was performed using an Illumina HiSeq 2500 Sequencing System (Illumina, San Diego, CA, USA). Single‐cell‐level gene expression profiles were extracted using the BD Rhapsody Analysis pipeline. All subsequent single‐cell analyses were performed using Seurat v4.3.0 [<a href="#mco270335-bib-0047" class="usa-link" aria-describedby="mco270335-bib-0047">47</a>]. In BD Rhapsody quality analysis, cells were filtered out based on the presence of double tags, mitochondrial percentage &gt; 30%, and feature count &lt; 200 or &gt; 6000. Finally, gene expression profiles were acquired. To eliminate the batch effect, data were normalized for each multiplexed group using the Seurat SCTransform function, and Harmony was used [<a href="#mco270335-bib-0048" class="usa-link" aria-describedby="mco270335-bib-0048">48</a>]. The top 3000 variable features were extracted using the SelectIntegrationFeatures function. Cell clustering was performed using FindNeighbors and FindClusters, and uniform manifold approximation and projection (UMAP) was used for visualization. Differentially expressed genes (DEGs) were identified using the FindMarkers function. The top 200 DEGs in each cluster were selected based on a log2FC &gt; 0.25 and adjusted <em>p</em> &lt; 0.05 (Table <a href="#mco270335-supinfo-0001" class="usa-link">S3</a>). Canonical markers to identify cell types in each cluster were referred to from previous studies (Table <a href="#mco270335-supinfo-0001" class="usa-link">S4</a>). Cell type enrichment according to two groups was analyzed using the chi‐square test.</p>
<p>For all single‐cell analyses, trajectory trees were created using Monocle v2.26.0, and DEGs were extracted from evolving states [<a href="#mco270335-bib-0049" class="usa-link" aria-describedby="mco270335-bib-0049">49</a>]. Gene set enrichment analysis (GSEA) of the DEGs was performed using clusterProfiler v4.6.2 [<a href="#mco270335-bib-0050" class="usa-link" aria-describedby="mco270335-bib-0050">50</a>]. Reference gene sets were collected from the Hallmark, Reactome, Kyoto Encyclopedia of Genes and Genomes, WikiPathway, and Gene Ontology in MSigDB v7.5.1 [<a href="#mco270335-bib-0051" class="usa-link" aria-describedby="mco270335-bib-0051">51</a>, <a href="#mco270335-bib-0052" class="usa-link" aria-describedby="mco270335-bib-0052">52</a>, <a href="#mco270335-bib-0053" class="usa-link" aria-describedby="mco270335-bib-0053">53</a>, <a href="#mco270335-bib-0054" class="usa-link" aria-describedby="mco270335-bib-0054">54</a>]. The pathway score for each cell was calculated using the AddModuleScore function. Pseudo‐bulking of the single‐cell data was performed using the average expression for each patient or cell type.</p>
<p>To evaluate tumor cell clusters, we performed CNV inference using inferCNV v1.14.2, using normal myeloid cells as a control [<a href="#mco270335-bib-0055" class="usa-link" aria-describedby="mco270335-bib-0055">55</a>]. CNV scores were acquired for each cell. Additionally, we analyzed whether the tumor cell clusters were close to malignant or a mixture of tumor and normal epithelial cells. For comparison with a gold standard, we collected LUAD single‐cell transcriptome (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE131907" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE131907</a>) data acquired from tumor lung biopsies and adjacent normal tissues [<a href="#mco270335-bib-0022" class="usa-link" aria-describedby="mco270335-bib-0022">22</a>]. Epithelial‐type normal and malignant cells were extracted by cell annotation of the data. The transcriptome profiles obtained in this study were compared with <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE131907" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE131907</a> dataset after z‐transformation rescaling. To determine the similarity of the cells identified with malignant cells, all pairwise Euclidean distances were calculated between the cells identified and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE131907" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE131907</a> epithelial cells.</p></section><section id="mco270335-sec-0170"><h3 class="pmc_sec_title">4.4. Intercellular Interaction Identification</h3>
<p>CellPhoneDB v4.0.0 was used to extract cell–cell interactions, and ligand receptors in each cell type were selected based on logFC &gt; 0.25 and adjusted <em>p</em> &lt; 0.05 [<a href="#mco270335-bib-0056" class="usa-link" aria-describedby="mco270335-bib-0056">56</a>]. Ligand–receptor pairs with <em>p</em> &lt; 0.05 and rank &lt; 0.3 were considered significant. Interaction counts between two cell types were determined from a cell–cell interaction adjacency matrix and sorted according to the ABCP response. The count network according to ABCP response was visualized using Cytoscape v3.10.0 [<a href="#mco270335-bib-0057" class="usa-link" aria-describedby="mco270335-bib-0057">57</a>]. Autocrine and paracrine ligand–receptor signaling communication among cell types was inferred and visualized using CellChat v2.1.1 [<a href="#mco270335-bib-0058" class="usa-link" aria-describedby="mco270335-bib-0058">58</a>].</p></section><section id="mco270335-sec-0180"><h3 class="pmc_sec_title">4.5. Tissue Preparation and Spatial Transcriptome Profiling</h3>
<p>Lung cancer formalin‐fixed paraffin‐embedded (FFPE) tissue sections were acquired from patients with EGFR‐mutant advanced NSCLC who had progressed after osimertinib treatment. We followed the Visium CytAssist Spatial Gene Expression for FFPE workflow for sectioning and RNA quality assessment (DV200 ≥ 30%). Tissue sections were deparaffinized, stained with hematoxylin and eosin, imaged, and decross‐linked, immediately followed by probe hybridization using a Visium Spatial Gene Expression for FFPE reagent kit (10X Genomics). The probe pairs hybridize with their target genes and are then ligated to one another. The single‐stranded ligation products are released from the tissue to a capture area on the Visium slide in the CytAssist instrument. Once the ligation products were captured, the slides were removed from the instrument, and probes were extended by the addition of a unique molecular identifier, Spatial Barcode, and Partial read 1.</p>
<p>To extract gene expression profiles, we used the Space Ranger v2.1.1 (10X Genomics) pipeline for FASTQ file generation from raw sequencing data. Visium spatial expression libraries were analyzed using the “count” command. Image processing proceeded in two stages. First, we used the fiducial alignment grid of the tissue image to determine the orientation and position of the input image. Next, we determined the region covered by the tissue on the slide. Sequencing reads were aligned to the GRCh38 reference genome using STAR (v2.5.1b) [<a href="#mco270335-bib-0059" class="usa-link" aria-describedby="mco270335-bib-0059">59</a>]. Normalization, dimension reduction, clustering, and DEG testing were performed using Seurat. After identifying variable features and anchors according to the “moransi” method using the SpatiallyVariableFeatures and FindTransferAnchors functions, respectively, cell type prediction was performed using the TransferData function in Seurat by referencing the previously identified cell types. Signature and pathway scores were calculated using the AddModuleScore function. Cell types were annotated to each cluster based on the scores using previously assessed ABCP patient cell type DEGs. Certain cell types unidentified in MPE samples were annotated using canonical markers (Table <a href="#mco270335-supinfo-0001" class="usa-link">S4</a>). A cell–cell communication network was established and investigated using CellChat.</p></section><section id="mco270335-sec-0190"><h3 class="pmc_sec_title">4.6. Multiplex IHC Staining of FFPE Tissues</h3>
<p>All details were included in the <a href="#mco270335-supinfo-0001" class="usa-link">Supporting Information</a>.</p></section><section id="mco270335-sec-0200"><h3 class="pmc_sec_title">4.7. Multispectral Imaging and Analysis</h3>
<p>All details were included in the <a href="#mco270335-supinfo-0001" class="usa-link">Supporting Information</a>.</p></section><section id="mco270335-sec-0210"><h3 class="pmc_sec_title">4.8. Cellular Mechanism Evaluation According to Tumor Progression After TKI Treatment</h3>
<p>All details were included in the <a href="#mco270335-supinfo-0001" class="usa-link">Supporting Information</a>.</p></section><section id="mco270335-sec-0220"><h3 class="pmc_sec_title">4.9. Clinical Outcome Investigation Using Meta‐Transcriptome Analysis of Multiple Cohorts</h3>
<p>All details were included in the <a href="#mco270335-supinfo-0001" class="usa-link">Supporting Information</a>.</p></section><section id="mco270335-sec-0230"><h3 class="pmc_sec_title">4.10. VEGFA/C Global Interaction Network Analysis</h3>
<p>All details were included in the <a href="#mco270335-supinfo-0001" class="usa-link">Supporting Information</a>.</p></section><section id="mco270335-sec-0240"><h3 class="pmc_sec_title">4.11. In Vitro and In Vivo VEGF Signaling Analyses</h3>
<p>All details were included in the <a href="#mco270335-supinfo-0001" class="usa-link">Supporting Information</a>.</p></section></section><section id="mco270335-sec-0250"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>J.H. and C.P. developed the study concept and experimental design. J.Y.H., G.K.L., B.C.A., and Y.L. contributed to the management of ABCP therapy and NCC cohorts and the collection of patient clinical information. S.H., N.Y., and C.P. performed the computational analyses. J.Y.C. and J.H.K. performed the <em>i</em>n vitro and in vivo experiments. H.S., B.R.S., M.H., and S.K. collected the patient samples. J.H.K., C.P., and J.Y.H. wrote and revised the manuscript. All authors approved the final manuscript.</p></section><section id="mco270335-sec-0270"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="mco270335-sec-0300"><h2 class="pmc_sec_title">Ethics Statement</h2>
<p>This study was approved by the Institutional Review Board (IRB), and all patients provided informed consent. (approval No. NCC2016‐0208).</p></section><section id="sec28"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="mco270335-supinfo-0001"></section><section class="sm xbox font-sm" id="mco270335-supitem-0001"><div class="caption p"><p>Table S1. Clinical profile of patients to receive ABCP treatment.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365383/bin/MCO2-6-e70335-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">MCO2-6-e70335-s002.xlsx</a><sup> (323KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="mco270335-supitem-0002"><div class="caption p">
<p>
<strong>Figure S1. Tumor cell cluster analysis</strong>. A) Euclidean distance heatmap of cell type expression profile between our tumor subcluster and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE131907" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE131907</a> epithelial subcluster. B) Heatmap of top‐ranked differentially expressed genes for tumor subclusters.</p>
<p>
<strong>Figure S2. Immune cell cluster analysis</strong>. A) Analysis of myeloid cell type for subcluster identification, its cell proportions, and its canonical markers. Pathways for subclusters were further assessed by gene set enrichment analysis (GSEA). B) Analysis of lymphoid cell type for subcluster identification, its cell proportions, and its canonical markers. Pathways were investigated by GSEA. C) Macrophage subclusters’ cell proportions according to ABCP response, and its canonical markers.</p>
<p>
<strong>Figure S3. Additional spatial transcriptome analysis</strong>. Capture area was 6.5 x 6.5 mm. A‐B are results of Slide1. C‐G are associated with Slide2. A) Cell score distribution on Slide1 according to tumor, stromal, lymphoid, and myeloid scores. B) Heatmap of marker expression to identify the cell type. C) Cell type proportion bar plot and its distribution in Slide2. D) Cell lineage scores for identified cell types. F) Heatmap of expression and scores of VEGF genes and pathways for cell types of Slide2. E) VEGFA and VEGF expression on Slide2 and circular plots of its cell–cell interactions. G) Bar plots of odds ratios (ORs) for <em>VEGFA</em> and <em>VEGFC</em> expression according to cell types.</p>
<p>
<strong>Figure S4. Heatmap of gene expression profile according to cell types and therapeutic groups</strong>. TKI treatment groups were classified into three types: TN, treatment‐naïve (blue); RD, residual disease (yellow‐green), PD, progressive disease (orange).</p>
<p>
<strong>Figure S5. Heatmap of gene–gene interaction network module analysis from <em>VEGFA‐</em> and <em>VEGFC</em>‐regulated networks</strong>. Pathways for each module were extracted by gene set enrichment analysis. Pathways categorized as ‘Common’ were detected in both <em>VEGFA</em> and <em>VEGFC</em> networks. The dominant mechanisms of <em>VEGFA</em> modules (VEGFA_0‐VEGFA_7) and <em>VEGFC</em> (VEGFC_0‐VEGFC_7) are highlighted for each case.</p>
<p>
<strong>Figure 6. Combinational knockdown of <em>VEGFC</em> and its receptor (<em>KDR</em> and <em>NRP2</em>) efficiently enhance the anti‐proliferation and anti‐angiogenesis effects of cabozantinib <em>in vitro</em>
</strong>. A) qPCR results demonstrate the relative mRNA expression of <em>VEGFC</em>, <em>KDR</em>, <em>NRP2</em>, <em>VEGFA</em>, and <em>FLT4</em> in diverse lung cancer cell lines (<em>n</em> = 3). B) IC<sub>50</sub> table of lung cancer cell lines exposed to three VEGF inhibitors. Three cell lines highlighted in red exhibited higher IC<sub>50</sub> values associated with increased expression of <em>VEGFC</em>, <em>KDR</em>, and <em>NRP2</em>. C) Bar graphs depict qPCR results (<em>n</em> = 3) confirming the efficiency of target gene silencing. D‐E) MTT assay results show the relative cell proliferation ratio following cabozantinib treatment with or without specific target gene knockdown in H1573 (D) and H2279 (E) cell lines. The orange bars below each graph indicate cabozantinib treatment (<em>n</em> = 3). F) qPCR results demonstrate the relative mRNA expression of <em>YAP1</em> (left) and <em>TAZ</em> (right) following target gene silencing with or without cabozantinib treatment (marked by an orange bar). G) In vitro angiogenesis assay demonstrates the relative change in capillary‐like tubular structures of HUVECs on Matrigel by conditioned media (CM) treatment. CMs were harvested from H2279 cells following cabozantinib treatment with or without target gene silencing. (Left) Phase contrast microscopic images of capillary‐like tubular structures on Matrigel. Magnification, x100. (Right) Graph bars show the tube length of HUVECs. *<em>P</em>&lt;0.05, **<em>P</em>&lt;0.01, ***<em>P</em>&lt;0.001, ****<em>P</em>&lt;0.0001, one‐way ANOVA, followed by Tukey's multiple‐comparison test.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365383/bin/MCO2-6-e70335-s003.docx" data-ga-action="click_feat_suppl" class="usa-link">MCO2-6-e70335-s003.docx</a><sup> (2.6MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="mco270335-supitem-0003"><div class="caption p"><p>Supporting information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365383/bin/MCO2-6-e70335-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">MCO2-6-e70335-s001.docx</a><sup> (71.6KB, docx) </sup>
</div></div></section></section><section id="mco270335-sec-0260" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We would like to acknowledge the patients and the clinical help obtained for this project. Our study of Figure <a href="#mco270335-fig-0001" class="usa-link">1</a> is a retrospective observational study not to be random assignment. Patients received treatments such as ABCP, PemC, or IO after developing resistance to EGFR‐TKI therapy. The timing of each treatment is summarized in Table <a href="#mco270335-supinfo-0001" class="usa-link">S1</a>. All figures were generated by own experiments with appropriate programs and required to avoid copyright issues.</p></section><section id="notes1"><section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="mco270335-note-0001"><p>Sehwa Hong and Namhee Yu contributed equally to this work as first authors.</p></div>
<div class="fn p" id="mco270335-note-0002"><p>
<strong>Funding</strong>: This work was supported by the National Cancer Center (NCC‐2510840 to J.Y.H.) and National Research Foundation of Korea (RS‐2023‐NR076994 to J.Y.H. and RS‐2022‐NR069261 to C.P.).</p></div>
</div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Jung‐Hyun Kim, Email: jungkim@ncc.re.kr.</p>
<p>Charny Park, Email: charn78@ncc.re.kr.</p>
<p>Ji‐Youn Han, Email: jymama@ncc.re.kr.</p></section><section id="mco270335-sec-0290"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>All gene expression profiling data generated in this study were deposited in the Gene Expression Omnibus (ABCP: scRNA‐seq GSE233203, spatial scRNA‐seq GSE267960; NCC PDC: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE165611" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE165611</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE229535" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE229535</a>, and PRJNA694788).</p></section><section id="mco270335-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="mco270335-bibl-0001_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="mco270335-bib-0001">
<span class="label">1.</span><cite>
Socinski M. A., Nishio M., Jotte R. M., et al., “IMpower150 Final Overall Survival Analyses for Atezolizumab Plus Bevacizumab and Chemotherapy in First‐Line Metastatic Nonsquamous NSCLC,” Journal of Thoracic Oncology
16 (2021): 1909–1924.
</cite> [<a href="https://doi.org/10.1016/j.jtho.2021.07.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34311108/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Thoracic%20Oncology&amp;title=IMpower150%20Final%20Overall%20Survival%20Analyses%20for%20Atezolizumab%20Plus%20Bevacizumab%20and%20Chemotherapy%20in%20First%E2%80%90Line%20Metastatic%20Nonsquamous%20NSCLC&amp;author=M.%20A.%20Socinski&amp;author=M.%20Nishio&amp;author=R.%20M.%20Jotte&amp;volume=16&amp;publication_year=2021&amp;pages=1909-1924&amp;pmid=34311108&amp;doi=10.1016/j.jtho.2021.07.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0002">
<span class="label">2.</span><cite>
Wu F., Fan J., He Y., et al., “Single‐Cell Profiling of Tumor Heterogeneity and the Microenvironment in Advanced Non‐Small Cell Lung Cancer,” Nature Communications
12, no. 1 (2021): 2540.</cite> [<a href="https://doi.org/10.1038/s41467-021-22801-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8100173/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33953163/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Single%E2%80%90Cell%20Profiling%20of%20Tumor%20Heterogeneity%20and%20the%20Microenvironment%20in%20Advanced%20Non%E2%80%90Small%20Cell%20Lung%20Cancer&amp;author=F.%20Wu&amp;author=J.%20Fan&amp;author=Y.%20He&amp;volume=12&amp;issue=1&amp;publication_year=2021&amp;pages=2540&amp;pmid=33953163&amp;doi=10.1038/s41467-021-22801-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0003">
<span class="label">3.</span><cite>
Yang Y., Li S., Wang Y., Zhao Y., and Li Q., “Protein Tyrosine Kinase Inhibitor Resistance in Malignant Tumors: Molecular Mechanisms and Future Perspective,” Signal Transduction and Targeted Therapy
7 (2022): 329.
</cite> [<a href="https://doi.org/10.1038/s41392-022-01168-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9482625/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36115852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Signal%20Transduction%20and%20Targeted%20Therapy&amp;title=Protein%20Tyrosine%20Kinase%20Inhibitor%20Resistance%20in%20Malignant%20Tumors:%20Molecular%20Mechanisms%20and%20Future%20Perspective&amp;author=Y.%20Yang&amp;author=S.%20Li&amp;author=Y.%20Wang&amp;author=Y.%20Zhao&amp;author=Q.%20Li&amp;volume=7&amp;publication_year=2022&amp;pages=329&amp;pmid=36115852&amp;doi=10.1038/s41392-022-01168-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0004">
<span class="label">4.</span><cite>
Joosten S. C., Hamming L., Soetekouw P. M., et al., “Resistance to Sunitinib in Renal Cell Carcinoma: From Molecular Mechanisms to Predictive Markers and Future Perspectives,” Biochimica et Biophysica Acta‐Reviews on Cancer
1855, no. 1 (2015): 1–16.</cite> [<a href="https://doi.org/10.1016/j.bbcan.2014.11.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25446042/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta%E2%80%90Reviews%20on%20Cancer&amp;title=Resistance%20to%20Sunitinib%20in%20Renal%20Cell%20Carcinoma:%20From%20Molecular%20Mechanisms%20to%20Predictive%20Markers%20and%20Future%20Perspectives&amp;author=S.%20C.%20Joosten&amp;author=L.%20Hamming&amp;author=P.%20M.%20Soetekouw&amp;volume=1855&amp;issue=1&amp;publication_year=2015&amp;pages=1-16&amp;pmid=25446042&amp;doi=10.1016/j.bbcan.2014.11.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0005">
<span class="label">5.</span><cite>
Moya‐Horno I., Viteri S., Karachaliou N., and Rosell R., “Combination of Immunotherapy With Targeted Therapies in Advanced Non‐Small Cell Lung Cancer (NSCLC),” Therapeutic Advances in Medical Oncology
10, no. 1 (2018): 1758834017745012.
</cite> [<a href="https://doi.org/10.1177/1758834017745012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5784559/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29383034/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Therapeutic%20Advances%20in%20Medical%20Oncology&amp;title=Combination%20of%20Immunotherapy%20With%20Targeted%20Therapies%20in%20Advanced%20Non%E2%80%90Small%20Cell%20Lung%20Cancer%20(NSCLC)&amp;author=I.%20Moya%E2%80%90Horno&amp;author=S.%20Viteri&amp;author=N.%20Karachaliou&amp;author=R.%20Rosell&amp;volume=10&amp;issue=1&amp;publication_year=2018&amp;pages=1758834017745012&amp;pmid=29383034&amp;doi=10.1177/1758834017745012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0006">
<span class="label">6.</span><cite>
Le X., Nilsson M., Goldman J., et al., “Dual EGFR‐VEGF Pathway Inhibition: A Promising Strategy for Patients With EGFR‐Mutant NSCLC,” Journal of Thoracic Oncology
16, no. 2 (2021): 205–215.
</cite> [<a href="https://doi.org/10.1016/j.jtho.2020.10.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33096270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Thoracic%20Oncology&amp;title=Dual%20EGFR%E2%80%90VEGF%20Pathway%20Inhibition:%20A%20Promising%20Strategy%20for%20Patients%20With%20EGFR%E2%80%90Mutant%20NSCLC&amp;author=X.%20Le&amp;author=M.%20Nilsson&amp;author=J.%20Goldman&amp;volume=16&amp;issue=2&amp;publication_year=2021&amp;pages=205-215&amp;pmid=33096270&amp;doi=10.1016/j.jtho.2020.10.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0007">
<span class="label">7.</span><cite>
Goel H. L., and Mercurio A. M., “VEGF Targets the Tumour Cell,” Nature Reviews Cancer
13, no. 11 (2013): 871–882.
</cite> [<a href="https://doi.org/10.1038/nrc3627" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4011842/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24263190/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Cancer&amp;title=VEGF%20Targets%20the%20Tumour%20Cell&amp;author=H.%20L.%20Goel&amp;author=A.%20M.%20Mercurio&amp;volume=13&amp;issue=11&amp;publication_year=2013&amp;pages=871-882&amp;pmid=24263190&amp;doi=10.1038/nrc3627&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0008">
<span class="label">8.</span><cite>
Shaik F., Cuthbert G. A., Homer‐Vanniasinkam S., Muench S. P., Ponnambalam S., and Harrison M. A., “Structural Basis for Vascular Endothelial Growth Factor Receptor Activation and Implications for Disease Therapy,” Biomolecules
10, no. 12 (2020): 1673.
</cite> [<a href="https://doi.org/10.3390/biom10121673" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7765180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33333800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomolecules&amp;title=Structural%20Basis%20for%20Vascular%20Endothelial%20Growth%20Factor%20Receptor%20Activation%20and%20Implications%20for%20Disease%20Therapy&amp;author=F.%20Shaik&amp;author=G.%20A.%20Cuthbert&amp;author=S.%20Homer%E2%80%90Vanniasinkam&amp;author=S.%20P.%20Muench&amp;author=S.%20Ponnambalam&amp;volume=10&amp;issue=12&amp;publication_year=2020&amp;pages=1673&amp;pmid=33333800&amp;doi=10.3390/biom10121673&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0009">
<span class="label">9.</span><cite>
Guo S., Colbert L. S., Fuller M., Zhang Y., and Gonzalez‐Perez R. R., “Vascular Endothelial Growth Factor Receptor‐2 in Breast Cancer,” Biochimica et Biophysica Acta‐Reviews on Cancer
1806, no. 2 (2010): 108–121.</cite> [<a href="https://doi.org/10.1016/j.bbcan.2010.04.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2885515/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20462514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta%E2%80%90Reviews%20on%20Cancer&amp;title=Vascular%20Endothelial%20Growth%20Factor%20Receptor%E2%80%902%20in%20Breast%20Cancer&amp;author=S.%20Guo&amp;author=L.%20S.%20Colbert&amp;author=M.%20Fuller&amp;author=Y.%20Zhang&amp;author=R.%20R.%20Gonzalez%E2%80%90Perez&amp;volume=1806&amp;issue=2&amp;publication_year=2010&amp;pages=108-121&amp;pmid=20462514&amp;doi=10.1016/j.bbcan.2010.04.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0010">
<span class="label">10.</span><cite>
Shibuya M. and Claesson‐Welsh L., “Signal Transduction by VEGF Receptors in Regulation of Angiogenesis and Lymphangiogenesis,” Experimental Cell Research
312, no. 4 (2006): 549–560.
</cite> [<a href="https://doi.org/10.1016/j.yexcr.2005.11.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16336962/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Experimental%20Cell%20Research&amp;title=Signal%20Transduction%20by%20VEGF%20Receptors%20in%20Regulation%20of%20Angiogenesis%20and%20Lymphangiogenesis&amp;author=M.%20Shibuya&amp;author=L.%20Claesson%E2%80%90Welsh&amp;volume=312&amp;issue=4&amp;publication_year=2006&amp;pages=549-560&amp;pmid=16336962&amp;doi=10.1016/j.yexcr.2005.11.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0011">
<span class="label">11.</span><cite>
Masłowska K., Halik P. K., Tymecka D., Misicka A., and Gniazdowska E., “The Role of VEGF Receptors as Molecular Targets in Nuclear Medicine for Cancer Diagnosis and Combination Therapy,” Cancers (Basel)
13, no. 5 (2021): 1072.
</cite> [<a href="https://doi.org/10.3390/cancers13051072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7959315/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33802353/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&amp;title=The%20Role%20of%20VEGF%20Receptors%20as%20Molecular%20Targets%20in%20Nuclear%20Medicine%20for%20Cancer%20Diagnosis%20and%20Combination%20Therapy&amp;author=K.%20Mas%C5%82owska&amp;author=P.%20K.%20Halik&amp;author=D.%20Tymecka&amp;author=A.%20Misicka&amp;author=E.%20Gniazdowska&amp;volume=13&amp;issue=5&amp;publication_year=2021&amp;pages=1072&amp;pmid=33802353&amp;doi=10.3390/cancers13051072&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0012">
<span class="label">12.</span><cite>
Park S., Kim T. M., Han J.‐Y., et al., “Phase III, Randomized Study of Atezolizumab Plus Bevacizumab and Chemotherapy in Patients With EGFR—or ALK ‐Mutated Non–Small‐Cell Lung Cancer (ATTLAS, KCSG‐LU19‐04),” Journal of Clinical Oncology
42, no. 5 (2023): 1241–1251.
</cite> [<a href="https://doi.org/10.1200/JCO.23.01891" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11095857/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37861993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Oncology&amp;title=Phase%20III,%20Randomized%20Study%20of%20Atezolizumab%20Plus%20Bevacizumab%20and%20Chemotherapy%20in%20Patients%20With%20EGFR%E2%80%94or%20ALK%20%E2%80%90Mutated%20Non%E2%80%93Small%E2%80%90Cell%20Lung%20Cancer%20(ATTLAS,%20KCSG%E2%80%90LU19%E2%80%9004)&amp;author=S.%20Park&amp;author=T.%20M.%20Kim&amp;author=J.%E2%80%90Y.%20Han&amp;volume=42&amp;issue=5&amp;publication_year=2023&amp;pages=1241-1251&amp;pmid=37861993&amp;doi=10.1200/JCO.23.01891&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0013">
<span class="label">13.</span><cite>
Socinski M. A., Jotte R. M., Cappuzzo F., et al., “Atezolizumab for First‐Line Treatment of Metastatic Nonsquamous NSCLC,” New England Journal of Medicine
378, no. 22 (2018): 2288–2301.
</cite> [<a href="https://doi.org/10.1056/NEJMoa1716948" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29863955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20England%20Journal%20of%20Medicine&amp;title=Atezolizumab%20for%20First%E2%80%90Line%20Treatment%20of%20Metastatic%20Nonsquamous%20NSCLC&amp;author=M.%20A.%20Socinski&amp;author=R.%20M.%20Jotte&amp;author=F.%20Cappuzzo&amp;volume=378&amp;issue=22&amp;publication_year=2018&amp;pages=2288-2301&amp;pmid=29863955&amp;doi=10.1056/NEJMoa1716948&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0014">
<span class="label">14.</span><cite>
Feng J., Chen S., Li S., et al., “The Association Between Monocytic Myeloid‐Derived Suppressor Cells Levels and the Anti‐Tumor Efficacy of Anti‐PD‐1 Therapy in NSCLC Patients,” Translational Oncology
13, no. 1 (2020): 100865.
</cite> [<a href="https://doi.org/10.1016/j.tranon.2020.100865" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7492992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32920330/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Translational%20Oncology&amp;title=The%20Association%20Between%20Monocytic%20Myeloid%E2%80%90Derived%20Suppressor%20Cells%20Levels%20and%20the%20Anti%E2%80%90Tumor%20Efficacy%20of%20Anti%E2%80%90PD%E2%80%901%20Therapy%20in%20NSCLC%20Patients&amp;author=J.%20Feng&amp;author=S.%20Chen&amp;author=S.%20Li&amp;volume=13&amp;issue=1&amp;publication_year=2020&amp;pages=100865&amp;pmid=32920330&amp;doi=10.1016/j.tranon.2020.100865&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0015">
<span class="label">15.</span><cite>
Martino E., Misso G., Pastina P., et al., “Immune‐Modulating Effects of Bevacizumab in Metastatic Non‐Small‐Cell Lung Cancer Patients,” Cell Death Discovery
2, no. 1 (2016): 2.</cite> [<a href="https://doi.org/10.1038/cddiscovery.2016.25" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5045963/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27752361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Death%20Discovery&amp;title=Immune%E2%80%90Modulating%20Effects%20of%20Bevacizumab%20in%20Metastatic%20Non%E2%80%90Small%E2%80%90Cell%20Lung%20Cancer%20Patients&amp;author=E.%20Martino&amp;author=G.%20Misso&amp;author=P.%20Pastina&amp;volume=2&amp;issue=1&amp;publication_year=2016&amp;pages=2&amp;pmid=27752361&amp;doi=10.1038/cddiscovery.2016.25&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0016">
<span class="label">16.</span><cite>
Jenkins L., Jungwirth U., Avgustinova A., et al., “Cancer‐Associated Fibroblasts Suppress CD8+ T‐cell Infiltration and Confer Resistance to Immune‐Checkpoint Blockade,” Cancer Research
82, no. 7 (2022): 2904–2917.
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-21-4141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9379365/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35749591/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Research&amp;title=Cancer%E2%80%90Associated%20Fibroblasts%20Suppress%20CD8+%20T%E2%80%90cell%20Infiltration%20and%20Confer%20Resistance%20to%20Immune%E2%80%90Checkpoint%20Blockade&amp;author=L.%20Jenkins&amp;author=U.%20Jungwirth&amp;author=A.%20Avgustinova&amp;volume=82&amp;issue=7&amp;publication_year=2022&amp;pages=2904-2917&amp;pmid=35749591&amp;doi=10.1158/0008-5472.CAN-21-4141&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0017">
<span class="label">17.</span><cite>
Fu Z., Mowday A. M., Smaill J. B., Hermans I. F., and Patterson A. V., “Tumour Hypoxia‐Mediated Immunosuppression: Mechanisms and Therapeutic Approaches to Improve Cancer Immunotherapy,” Cells
10, no. 11 (2021): 10.</cite> [<a href="https://doi.org/10.3390/cells10051006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8146304/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33923305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=Tumour%20Hypoxia%E2%80%90Mediated%20Immunosuppression:%20Mechanisms%20and%20Therapeutic%20Approaches%20to%20Improve%20Cancer%20Immunotherapy&amp;author=Z.%20Fu&amp;author=A.%20M.%20Mowday&amp;author=J.%20B.%20Smaill&amp;author=I.%20F.%20Hermans&amp;author=A.%20V.%20Patterson&amp;volume=10&amp;issue=11&amp;publication_year=2021&amp;pages=10&amp;pmid=33923305&amp;doi=10.3390/cells10051006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0018">
<span class="label">18.</span><cite>
Leonetti A., Sharma S., Minari R., Perego P., Giovannetti E., and Tiseo M., “Resistance Mechanisms to Osimertinib in EGFR‐Mutated Non‐Small Cell Lung Cancer,” British Journal of Cancer
121, no. 5 (2019): 725–737.
</cite> [<a href="https://doi.org/10.1038/s41416-019-0573-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6889286/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31564718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=British%20Journal%20of%20Cancer&amp;title=Resistance%20Mechanisms%20to%20Osimertinib%20in%20EGFR%E2%80%90Mutated%20Non%E2%80%90Small%20Cell%20Lung%20Cancer&amp;author=A.%20Leonetti&amp;author=S.%20Sharma&amp;author=R.%20Minari&amp;author=P.%20Perego&amp;author=E.%20Giovannetti&amp;volume=121&amp;issue=5&amp;publication_year=2019&amp;pages=725-737&amp;pmid=31564718&amp;doi=10.1038/s41416-019-0573-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0019">
<span class="label">19.</span><cite>
Yu N., Hwang M., Lee Y., et al., “Patient‐Derived Cell‐Based Pharmacogenomic Assessment to Unveil Underlying Resistance Mechanisms and Novel Therapeutics for Advanced Lung Cancer,” Journal of Experimental &amp; Clinical Cancer Research
42, no. 1 (2023): 37.
</cite> [<a href="https://doi.org/10.1186/s13046-023-02606-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9885631/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36717865/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20&amp;%20Clinical%20Cancer%20Research&amp;title=Patient%E2%80%90Derived%20Cell%E2%80%90Based%20Pharmacogenomic%20Assessment%20to%20Unveil%20Underlying%20Resistance%20Mechanisms%20and%20Novel%20Therapeutics%20for%20Advanced%20Lung%20Cancer&amp;author=N.%20Yu&amp;author=M.%20Hwang&amp;author=Y.%20Lee&amp;volume=42&amp;issue=1&amp;publication_year=2023&amp;pages=37&amp;pmid=36717865&amp;doi=10.1186/s13046-023-02606-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0020">
<span class="label">20.</span><cite>
Whitfield H. J., Berthelet J., Mangiola S., et al., “Single‐Cell RNA Sequencing Captures Patient‐Level Heterogeneity and Associated Molecular Phenotypes in Breast Cancer Pleural Effusions,” Clinical and Translational Medicine
13, no. 1 (2023): e1356.
</cite> [<a href="https://doi.org/10.1002/ctm2.1356" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10493486/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37691350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clinical%20and%20Translational%20Medicine&amp;title=Single%E2%80%90Cell%20RNA%20Sequencing%20Captures%20Patient%E2%80%90Level%20Heterogeneity%20and%20Associated%20Molecular%20Phenotypes%20in%20Breast%20Cancer%20Pleural%20Effusions&amp;author=H.%20J.%20Whitfield&amp;author=J.%20Berthelet&amp;author=S.%20Mangiola&amp;volume=13&amp;issue=1&amp;publication_year=2023&amp;pages=e1356&amp;pmid=37691350&amp;doi=10.1002/ctm2.1356&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0021">
<span class="label">21.</span><cite>
Collisson E. A., Campbell J. D., Brooks A. N., et al., “Comprehensive Molecular Profiling of Lung Adenocarcinoma: The Cancer Genome Atlas Research Network,” Nature
511, no. 7507 (2014): 543–550.
</cite> [<a href="https://doi.org/10.1038/nature13385" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4231481/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25079552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Comprehensive%20Molecular%20Profiling%20of%20Lung%20Adenocarcinoma:%20The%20Cancer%20Genome%20Atlas%20Research%20Network&amp;author=E.%20A.%20Collisson&amp;author=J.%20D.%20Campbell&amp;author=A.%20N.%20Brooks&amp;volume=511&amp;issue=7507&amp;publication_year=2014&amp;pages=543-550&amp;pmid=25079552&amp;doi=10.1038/nature13385&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0022">
<span class="label">22.</span><cite>
Kim N., Kim H. K., Lee K., et al., “Single‐Cell RNA Sequencing Demonstrates the Molecular and Cellular Reprogramming of Metastatic Lung Adenocarcinoma,” Nature Communications
11, no. 1 (2020): 2285.</cite> [<a href="https://doi.org/10.1038/s41467-020-16164-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7210975/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32385277/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Single%E2%80%90Cell%20RNA%20Sequencing%20Demonstrates%20the%20Molecular%20and%20Cellular%20Reprogramming%20of%20Metastatic%20Lung%20Adenocarcinoma&amp;author=N.%20Kim&amp;author=H.%20K.%20Kim&amp;author=K.%20Lee&amp;volume=11&amp;issue=1&amp;publication_year=2020&amp;pages=2285&amp;pmid=32385277&amp;doi=10.1038/s41467-020-16164-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0023">
<span class="label">23.</span><cite>
Kieffer Y., Hocine H. R., Gentric G., et al., “Single‐Cell Analysis Reveals Fibroblast Clusters Linked to Immunotherapy Resistance in Cancer,” Cancer Discovery
10, no. 10 (2020): 1330–1351.
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-19-1384" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32434947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Discovery&amp;title=Single%E2%80%90Cell%20Analysis%20Reveals%20Fibroblast%20Clusters%20Linked%20to%20Immunotherapy%20Resistance%20in%20Cancer&amp;author=Y.%20Kieffer&amp;author=H.%20R.%20Hocine&amp;author=G.%20Gentric&amp;volume=10&amp;issue=10&amp;publication_year=2020&amp;pages=1330-1351&amp;pmid=32434947&amp;doi=10.1158/2159-8290.CD-19-1384&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0024">
<span class="label">24.</span><cite>
Hu H., Piotrowska Z., Hare P. J., et al., “Three Subtypes of Lung Cancer Fibroblasts Define Distinct Therapeutic Paradigms,” Cancer Cell
39, no. 10 (2021): 1531–1547.e10.
</cite> [<a href="https://doi.org/10.1016/j.ccell.2021.09.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8578451/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34624218/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Three%20Subtypes%20of%20Lung%20Cancer%20Fibroblasts%20Define%20Distinct%20Therapeutic%20Paradigms&amp;author=H.%20Hu&amp;author=Z.%20Piotrowska&amp;author=P.%20J.%20Hare&amp;volume=39&amp;issue=10&amp;publication_year=2021&amp;pages=1531-1547.e10&amp;pmid=34624218&amp;doi=10.1016/j.ccell.2021.09.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0025">
<span class="label">25.</span><cite>
Li W., Wu F., Zhao S., Shi P., Wang S., Cui D., “Correlation Between PD‐1/PD‐L1 Expression and Polarization in Tumor‐Associated Macrophages: A Key Player in Tumor Immunotherapy,” Cytokine &amp; Growth Factor Reviews
67, no. 2 (2022): 49–57.
</cite> [<a href="https://doi.org/10.1016/j.cytogfr.2022.07.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35871139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cytokine%20&amp;%20Growth%20Factor%20Reviews&amp;title=Correlation%20Between%20PD%E2%80%901/PD%E2%80%90L1%20Expression%20and%20Polarization%20in%20Tumor%E2%80%90Associated%20Macrophages:%20A%20Key%20Player%20in%20Tumor%20Immunotherapy&amp;author=W.%20Li&amp;author=F.%20Wu&amp;author=S.%20Zhao&amp;author=P.%20Shi&amp;author=S.%20Wang&amp;volume=67&amp;issue=2&amp;publication_year=2022&amp;pages=49-57&amp;pmid=35871139&amp;doi=10.1016/j.cytogfr.2022.07.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0026">
<span class="label">26.</span><cite>
Puchert M., Obst J., Koch C., Zieger K., and Engele J., “CXCL11 Promotes Tumor Progression by the Biased Use of the Chemokine Receptors CXCR3 and CXCR7,” Cytokine
125, no. 1 (2020): 154809.
</cite> [<a href="https://doi.org/10.1016/j.cyto.2019.154809" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31437604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cytokine&amp;title=CXCL11%20Promotes%20Tumor%20Progression%20by%20the%20Biased%20Use%20of%20the%20Chemokine%20Receptors%20CXCR3%20and%20CXCR7&amp;author=M.%20Puchert&amp;author=J.%20Obst&amp;author=C.%20Koch&amp;author=K.%20Zieger&amp;author=J.%20Engele&amp;volume=125&amp;issue=1&amp;publication_year=2020&amp;pages=154809&amp;pmid=31437604&amp;doi=10.1016/j.cyto.2019.154809&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0027">
<span class="label">27.</span><cite>
Makkouk A., Chester C., and Kohrt H. E., “Rationale for Anti‐CD137 Cancer Immunotherapy,” European Journal of Cancer
54, no. 1 (2016): 112–119.
</cite> [<a href="https://doi.org/10.1016/j.ejca.2015.09.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26751393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Cancer&amp;title=Rationale%20for%20Anti%E2%80%90CD137%20Cancer%20Immunotherapy&amp;author=A.%20Makkouk&amp;author=C.%20Chester&amp;author=H.%20E.%20Kohrt&amp;volume=54&amp;issue=1&amp;publication_year=2016&amp;pages=112-119&amp;pmid=26751393&amp;doi=10.1016/j.ejca.2015.09.026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0028">
<span class="label">28.</span><cite>
Nishina S. and Hino K., “CD26/DPP4 as a Therapeutic Target in Nonalcoholic Steatohepatitis Associated Hepatocellular Carcinoma,” Cancers (Basel)
14, no. 3 (2022): 454.
</cite> [<a href="https://doi.org/10.3390/cancers14020454" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8774170/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35053615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&amp;title=CD26/DPP4%20as%20a%20Therapeutic%20Target%20in%20Nonalcoholic%20Steatohepatitis%20Associated%20Hepatocellular%20Carcinoma&amp;author=S.%20Nishina&amp;author=K.%20Hino&amp;volume=14&amp;issue=3&amp;publication_year=2022&amp;pages=454&amp;pmid=35053615&amp;doi=10.3390/cancers14020454&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0029">
<span class="label">29.</span><cite>
Maynard A., McCoach C. E., Rotow J. K., et al., “Therapy‐Induced Evolution of Human Lung Cancer Revealed by Single‐Cell RNA Sequencing,” Cell
182, no. 6 (2020): 1232–1251.e22.
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.07.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7484178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32822576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Therapy%E2%80%90Induced%20Evolution%20of%20Human%20Lung%20Cancer%20Revealed%20by%20Single%E2%80%90Cell%20RNA%20Sequencing&amp;author=A.%20Maynard&amp;author=C.%20E.%20McCoach&amp;author=J.%20K.%20Rotow&amp;volume=182&amp;issue=6&amp;publication_year=2020&amp;pages=1232-1251.e22&amp;pmid=32822576&amp;doi=10.1016/j.cell.2020.07.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0030">
<span class="label">30.</span><cite>
Sanchez‐Vega F., Mina M., Armenia J., et al., “Oncogenic Signaling Pathways in the Cancer Genome Atlas,” Cell
173, no. 2 (2018): 321–337.e10.
</cite> [<a href="https://doi.org/10.1016/j.cell.2018.03.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6070353/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29625050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Oncogenic%20Signaling%20Pathways%20in%20the%20Cancer%20Genome%20Atlas&amp;author=F.%20Sanchez%E2%80%90Vega&amp;author=M.%20Mina&amp;author=J.%20Armenia&amp;volume=173&amp;issue=2&amp;publication_year=2018&amp;pages=321-337.e10&amp;pmid=29625050&amp;doi=10.1016/j.cell.2018.03.035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0031">
<span class="label">31.</span><cite>
Mao X., Xu J., Wang W., et al., “Crosstalk Between Cancer‐Associated Fibroblasts and Immune Cells in the Tumor Microenvironment: New Findings and Future Perspectives,” Molecular Cancer
20, no. 1 (2021): 131.
</cite> [<a href="https://doi.org/10.1186/s12943-021-01428-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8504100/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34635121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cancer&amp;title=Crosstalk%20Between%20Cancer%E2%80%90Associated%20Fibroblasts%20and%20Immune%20Cells%20in%20the%20Tumor%20Microenvironment:%20New%20Findings%20and%20Future%20Perspectives&amp;author=X.%20Mao&amp;author=J.%20Xu&amp;author=W.%20Wang&amp;volume=20&amp;issue=1&amp;publication_year=2021&amp;pages=131&amp;pmid=34635121&amp;doi=10.1186/s12943-021-01428-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0032">
<span class="label">32.</span><cite>
Naumov G. N., Nilsson M. B., Cascone T., et al., “Combined Vascular Endothelial Growth Factor Receptor and Epidermal Growth Factor Receptor (EGFR) Blockade Inhibits Tumor Growth in Xenograft Models of EGFR Inhibitor Resistance,” Clinical Cancer Research
15, no. 7 (2009): 3484–3494.
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-08-2904" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2893040/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19447865/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clinical%20Cancer%20Research&amp;title=Combined%20Vascular%20Endothelial%20Growth%20Factor%20Receptor%20and%20Epidermal%20Growth%20Factor%20Receptor%20(EGFR)%20Blockade%20Inhibits%20Tumor%20Growth%20in%20Xenograft%20Models%20of%20EGFR%20Inhibitor%20Resistance&amp;author=G.%20N.%20Naumov&amp;author=M.%20B.%20Nilsson&amp;author=T.%20Cascone&amp;volume=15&amp;issue=7&amp;publication_year=2009&amp;pages=3484-3494&amp;pmid=19447865&amp;doi=10.1158/1078-0432.CCR-08-2904&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0033">
<span class="label">33.</span><cite>
Li H., Takayama K., Wang S., et al., “Addition of Bevacizumab Enhances Antitumor Activity of Erlotinib Against Non‐Small Cell Lung Cancer Xenografts Depending on VEGF Expression,” Cancer Chemotheraphy and Pharmacology
74, no. 6 (2014): 1297–305.</cite> [<a href="https://doi.org/10.1007/s00280-014-2610-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4236614/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25344762/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Chemotheraphy%20and%20Pharmacology&amp;title=Addition%20of%20Bevacizumab%20Enhances%20Antitumor%20Activity%20of%20Erlotinib%20Against%20Non%E2%80%90Small%20Cell%20Lung%20Cancer%20Xenografts%20Depending%20on%20VEGF%20Expression&amp;author=H.%20Li&amp;author=K.%20Takayama&amp;author=S.%20Wang&amp;volume=74&amp;issue=6&amp;publication_year=2014&amp;pages=1297-305&amp;pmid=25344762&amp;doi=10.1007/s00280-014-2610-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0034">
<span class="label">34.</span><cite>
Koinis F., Corn P., Parikh N., et al., “Resistance to MET/VEGFR2 Inhibition by Cabozantinib Is Mediated by YAP/TBX5‐Dependent Induction of FGFR1 in Castration‐Resistant Prostate Cancer,” Cancers (Basel)
12, no. 2 (2020): 244.
</cite> [<a href="https://doi.org/10.3390/cancers12010244" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7016532/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31963871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancers%20(Basel)&amp;title=Resistance%20to%20MET/VEGFR2%20Inhibition%20by%20Cabozantinib%20Is%20Mediated%20by%20YAP/TBX5%E2%80%90Dependent%20Induction%20of%20FGFR1%20in%20Castration%E2%80%90Resistant%20Prostate%20Cancer&amp;author=F.%20Koinis&amp;author=P.%20Corn&amp;author=N.%20Parikh&amp;volume=12&amp;issue=2&amp;publication_year=2020&amp;pages=244&amp;pmid=31963871&amp;doi=10.3390/cancers12010244&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0035">
<span class="label">35.</span><cite>
De Zuani M., Xue H., Park J. S., et al., “Single‐Cell and Spatial Transcriptomics Analysis of Non‐Small Cell Lung Cancer,” Nature Communications
15, no. 1 (2024): 4388.</cite> [<a href="https://doi.org/10.1038/s41467-024-48700-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11116453/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38782901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Single%E2%80%90Cell%20and%20Spatial%20Transcriptomics%20Analysis%20of%20Non%E2%80%90Small%20Cell%20Lung%20Cancer&amp;author=M.%20De%20Zuani&amp;author=H.%20Xue&amp;author=J.%20S.%20Park&amp;volume=15&amp;issue=1&amp;publication_year=2024&amp;pages=4388&amp;pmid=38782901&amp;doi=10.1038/s41467-024-48700-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0036">
<span class="label">36.</span><cite>
Hu J., Zhang L., Xia H., et al., “Tumor Microenvironment Remodeling After Neoadjuvant Immunotherapy in Non‐Small Cell Lung Cancer Revealed by Single‐Cell RNA Sequencing,” Genome Medicine
15, no. 1 (2023): 14.
</cite> [<a href="https://doi.org/10.1186/s13073-023-01164-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9985263/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36869384/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Medicine&amp;title=Tumor%20Microenvironment%20Remodeling%20After%20Neoadjuvant%20Immunotherapy%20in%20Non%E2%80%90Small%20Cell%20Lung%20Cancer%20Revealed%20by%20Single%E2%80%90Cell%20RNA%20Sequencing&amp;author=J.%20Hu&amp;author=L.%20Zhang&amp;author=H.%20Xia&amp;volume=15&amp;issue=1&amp;publication_year=2023&amp;pages=14&amp;pmid=36869384&amp;doi=10.1186/s13073-023-01164-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0037">
<span class="label">37.</span><cite>
Liu Z., Yang Z., Wu J., et al., “A Single‐Cell Atlas Reveals Immune Heterogeneity in Anti‐PD‐1‐Treated Non‐Small Cell Lung Cancer,” Cell
11, no. 7 (2025): 3081–3096. e19.</cite> [<a href="https://doi.org/10.1016/j.cell.2025.03.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40147443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20Single%E2%80%90Cell%20Atlas%20Reveals%20Immune%20Heterogeneity%20in%20Anti%E2%80%90PD%E2%80%901%E2%80%90Treated%20Non%E2%80%90Small%20Cell%20Lung%20Cancer&amp;author=Z.%20Liu&amp;author=Z.%20Yang&amp;author=J.%20Wu&amp;volume=11&amp;issue=7&amp;publication_year=2025&amp;pages=3081-3096&amp;pmid=40147443&amp;doi=10.1016/j.cell.2025.03.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0038">
<span class="label">38.</span><cite>
Yang L., He Y. T., Dong S., et al., “Single‐Cell Transcriptome Analysis Revealed a Suppressive Tumor Immune Microenvironment in EGFR Mutant Lung Adenocarcinoma,” Journal for ImmunoTherapy of Cancer
10, no. 1 (2022): e003534.
</cite> [<a href="https://doi.org/10.1136/jitc-2021-003534" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8830346/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35140113/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20for%20ImmunoTherapy%20of%20Cancer&amp;title=Single%E2%80%90Cell%20Transcriptome%20Analysis%20Revealed%20a%20Suppressive%20Tumor%20Immune%20Microenvironment%20in%20EGFR%20Mutant%20Lung%20Adenocarcinoma&amp;author=L.%20Yang&amp;author=Y.%20T.%20He&amp;author=S.%20Dong&amp;volume=10&amp;issue=1&amp;publication_year=2022&amp;pages=e003534&amp;pmid=35140113&amp;doi=10.1136/jitc-2021-003534&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0039">
<span class="label">39.</span><cite>
Murakami M., Nguyen L. T., Hatanaka K., et al., “FGF‐Dependent Regulation of VEGF Receptor 2 Expression in Mice,” Journal of Clinical Investigation
121, no. 6 (2011): 2668–2678.
</cite> [<a href="https://doi.org/10.1172/JCI44762" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3223828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21633168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Clinical%20Investigation&amp;title=FGF%E2%80%90Dependent%20Regulation%20of%20VEGF%20Receptor%202%20Expression%20in%20Mice&amp;author=M.%20Murakami&amp;author=L.%20T.%20Nguyen&amp;author=K.%20Hatanaka&amp;volume=121&amp;issue=6&amp;publication_year=2011&amp;pages=2668-2678&amp;pmid=21633168&amp;doi=10.1172/JCI44762&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0040">
<span class="label">40.</span><cite>
Cha B., Ho Y. C., Geng X., et al., “YAP and TAZ Maintain PROX1 Expression in the Developing Lymphatic and Lymphovenous Valves in Response to VEGF‐C Signaling,” Development (Cambridge, England)
147, no. 13 (2020): dev195453.
</cite> [<a href="https://doi.org/10.1242/dev.195453" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7758626/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33060128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development%20(Cambridge,%20England)&amp;title=YAP%20and%20TAZ%20Maintain%20PROX1%20Expression%20in%20the%20Developing%20Lymphatic%20and%20Lymphovenous%20Valves%20in%20Response%20to%20VEGF%E2%80%90C%20Signaling&amp;author=B.%20Cha&amp;author=Y.%20C.%20Ho&amp;author=X.%20Geng&amp;volume=147&amp;issue=13&amp;publication_year=2020&amp;pages=dev195453&amp;pmid=33060128&amp;doi=10.1242/dev.195453&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0041">
<span class="label">41.</span><cite>
Ogawa E., Takenaka K., Yanagihara K., et al., “Clinical Significance of VEGF‐C Status in Tumour Cells and Stromal Macrophages in Non‐Small Cell Lung Cancer Patients,” British Journal of Cancer
91, no. 4 (2004): 498–503.
</cite> [<a href="https://doi.org/10.1038/sj.bjc.6601992" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2409842/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15226767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=British%20Journal%20of%20Cancer&amp;title=Clinical%20Significance%20of%20VEGF%E2%80%90C%20Status%20in%20Tumour%20Cells%20and%20Stromal%20Macrophages%20in%20Non%E2%80%90Small%20Cell%20Lung%20Cancer%20Patients&amp;author=E.%20Ogawa&amp;author=K.%20Takenaka&amp;author=K.%20Yanagihara&amp;volume=91&amp;issue=4&amp;publication_year=2004&amp;pages=498-503&amp;pmid=15226767&amp;doi=10.1038/sj.bjc.6601992&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0042">
<span class="label">42.</span><cite>
Ebos J. M. L., Lee C. R., Cruz‐Munoz W., Bjarnason G. A., Christensen J. G., and Kerbel R. S., “Accelerated Metastasis After Short‐Term Treatment With a Potent Inhibitor of Tumor Angiogenesis,” Cancer Cell
15, no. 3 (2009): 232–239.
</cite> [<a href="https://doi.org/10.1016/j.ccr.2009.01.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4540346/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19249681/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Accelerated%20Metastasis%20After%20Short%E2%80%90Term%20Treatment%20With%20a%20Potent%20Inhibitor%20of%20Tumor%20Angiogenesis&amp;author=J.%20M.%20L.%20Ebos&amp;author=C.%20R.%20Lee&amp;author=W.%20Cruz%E2%80%90Munoz&amp;author=G.%20A.%20Bjarnason&amp;author=J.%20G.%20Christensen&amp;volume=15&amp;issue=3&amp;publication_year=2009&amp;pages=232-239&amp;pmid=19249681&amp;doi=10.1016/j.ccr.2009.01.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0043">
<span class="label">43.</span><cite>
Loges S., Schmidt T., and Carmeliet P., “Mechanisms of Resistance to Anti‐Angiogenic Therapy and Development of Third‐Generation Anti‐Angiogenic Drug Candidates,” Genes &amp; Cancer
1, no. 1 (2010): 12–25.
</cite> [<a href="https://doi.org/10.1177/1947601909356574" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3092176/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21779425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20&amp;%20Cancer&amp;title=Mechanisms%20of%20Resistance%20to%20Anti%E2%80%90Angiogenic%20Therapy%20and%20Development%20of%20Third%E2%80%90Generation%20Anti%E2%80%90Angiogenic%20Drug%20Candidates&amp;author=S.%20Loges&amp;author=T.%20Schmidt&amp;author=P.%20Carmeliet&amp;volume=1&amp;issue=1&amp;publication_year=2010&amp;pages=12-25&amp;pmid=21779425&amp;doi=10.1177/1947601909356574&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0044">
<span class="label">44.</span><cite>
Pedrosa R., Mustafa D. A., Soffietti R., and Kros J. M., “Breast Cancer Brain Metastasis: Molecular Mechanisms and Directions for Treatment,” Neuro‐oncology
20, no. 10 (2018): 1439–1449.
</cite> [<a href="https://doi.org/10.1093/neuonc/noy044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6176797/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29566179/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuro%E2%80%90oncology&amp;title=Breast%20Cancer%20Brain%20Metastasis:%20Molecular%20Mechanisms%20and%20Directions%20for%20Treatment&amp;author=R.%20Pedrosa&amp;author=D.%20A.%20Mustafa&amp;author=R.%20Soffietti&amp;author=J.%20M.%20Kros&amp;volume=20&amp;issue=10&amp;publication_year=2018&amp;pages=1439-1449&amp;pmid=29566179&amp;doi=10.1093/neuonc/noy044&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0045">
<span class="label">45.</span><cite>
Zhu C., Wei Y., and Wei X., “AXL Receptor Tyrosine Kinase as a Promising Anti‐Cancer Approach: Functions, Molecular Mechanisms and Clinical Applications,” Molecular Cancer
18, no. 1 (2019): 153.
</cite> [<a href="https://doi.org/10.1186/s12943-019-1090-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6827209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31684958/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cancer&amp;title=AXL%20Receptor%20Tyrosine%20Kinase%20as%20a%20Promising%20Anti%E2%80%90Cancer%20Approach:%20Functions,%20Molecular%20Mechanisms%20and%20Clinical%20Applications&amp;author=C.%20Zhu&amp;author=Y.%20Wei&amp;author=X.%20Wei&amp;volume=18&amp;issue=1&amp;publication_year=2019&amp;pages=153&amp;pmid=31684958&amp;doi=10.1186/s12943-019-1090-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0046">
<span class="label">46.</span><cite>
Beenken A. and Mohammadi M., “The FGF Family: Biology, Pathophysiology and Therapy,” Nat Rev Drug Discovery
8, no. 2 (2009): 235–253.
</cite> [<a href="https://doi.org/10.1038/nrd2792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3684054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19247306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Drug%20Discovery&amp;title=The%20FGF%20Family:%20Biology,%20Pathophysiology%20and%20Therapy&amp;author=A.%20Beenken&amp;author=M.%20Mohammadi&amp;volume=8&amp;issue=2&amp;publication_year=2009&amp;pages=235-253&amp;pmid=19247306&amp;doi=10.1038/nrd2792&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0047">
<span class="label">47.</span><cite>
Stuart T., Butler A., Hoffman P., et al., “Comprehensive Integration of Single‐Cell Data,” Cell
177, no. 6 (2019): 1888–1902. e21.
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.05.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6687398/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Comprehensive%20Integration%20of%20Single%E2%80%90Cell%20Data&amp;author=T.%20Stuart&amp;author=A.%20Butler&amp;author=P.%20Hoffman&amp;volume=177&amp;issue=6&amp;publication_year=2019&amp;pages=1888-1902&amp;pmid=31178118&amp;doi=10.1016/j.cell.2019.05.031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0048">
<span class="label">48.</span><cite>
Korsunsky I., Millard N., Fan J., et al., “Fast, Sensitive and Accurate Integration of Single‐Cell Data With Harmony,” Nature Methods
16, no. 12 (2019): 1289–1296.
</cite> [<a href="https://doi.org/10.1038/s41592-019-0619-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6884693/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31740819/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Fast,%20Sensitive%20and%20Accurate%20Integration%20of%20Single%E2%80%90Cell%20Data%20With%20Harmony&amp;author=I.%20Korsunsky&amp;author=N.%20Millard&amp;author=J.%20Fan&amp;volume=16&amp;issue=12&amp;publication_year=2019&amp;pages=1289-1296&amp;pmid=31740819&amp;doi=10.1038/s41592-019-0619-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0049">
<span class="label">49.</span><cite>
Qiu X., Hill A., Packer J., Lin D., Ma Y. A., and Trapnell C., “Single‐Cell mRNA Quantification and Differential Analysis With Census,” Nature Methods
14, no. 5 (2017): 309–315.
</cite> [<a href="https://doi.org/10.1038/nmeth.4150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5330805/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28114287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Single%E2%80%90Cell%20mRNA%20Quantification%20and%20Differential%20Analysis%20With%20Census&amp;author=X.%20Qiu&amp;author=A.%20Hill&amp;author=J.%20Packer&amp;author=D.%20Lin&amp;author=Y.%20A.%20Ma&amp;volume=14&amp;issue=5&amp;publication_year=2017&amp;pages=309-315&amp;pmid=28114287&amp;doi=10.1038/nmeth.4150&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0050">
<span class="label">50.</span><cite>
Wu T., Hu E., Xu S., et al., “clusterProfiler 4.0: A Universal Enrichment Tool for Interpreting Omics Data,” Innovation
2, no. 1 (2021): 100141.
</cite> [<a href="https://doi.org/10.1016/j.xinn.2021.100141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8454663/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34557778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Innovation&amp;title=clusterProfiler%204.0:%20A%20Universal%20Enrichment%20Tool%20for%20Interpreting%20Omics%20Data&amp;author=T.%20Wu&amp;author=E.%20Hu&amp;author=S.%20Xu&amp;volume=2&amp;issue=1&amp;publication_year=2021&amp;pages=100141&amp;pmid=34557778&amp;doi=10.1016/j.xinn.2021.100141&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0051">
<span class="label">51.</span><cite>
Liberzon A., Birger C., Thorvaldsdóttir H., Ghandi M., Mesirov J. P., and Tamayo P., “The Molecular Signatures Database Hallmark Gene Set Collection,” Cell Systems
1, no. 6 (2015): 417–425.
</cite> [<a href="https://doi.org/10.1016/j.cels.2015.12.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4707969/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26771021/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Systems&amp;title=The%20Molecular%20Signatures%20Database%20Hallmark%20Gene%20Set%20Collection&amp;author=A.%20Liberzon&amp;author=C.%20Birger&amp;author=H.%20Thorvaldsd%C3%B3ttir&amp;author=M.%20Ghandi&amp;author=J.%20P.%20Mesirov&amp;volume=1&amp;issue=6&amp;publication_year=2015&amp;pages=417-425&amp;pmid=26771021&amp;doi=10.1016/j.cels.2015.12.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0052">
<span class="label">52.</span><cite>
Jassal B., Matthews L., Viteri G., et al., “The Reactome Pathway Knowledgebase,” Nucleic Acids Research
48, no. D1 (2020): D498–D503.
</cite> [<a href="https://doi.org/10.1093/nar/gkz1031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7145712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31691815/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=The%20Reactome%20Pathway%20Knowledgebase&amp;author=B.%20Jassal&amp;author=L.%20Matthews&amp;author=G.%20Viteri&amp;volume=48&amp;issue=D1&amp;publication_year=2020&amp;pages=D498-D503&amp;pmid=31691815&amp;doi=10.1093/nar/gkz1031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0053">
<span class="label">53.</span><cite>
Kanehisa M., Sato Y., Kawashima M., Furumichi M., and Tanabe M., “KEGG as a Reference Resource for Gene and Protein Annotation,” Nucleic Acids Research
44, no. D1 (2016): D457–D462.
</cite> [<a href="https://doi.org/10.1093/nar/gkv1070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4702792/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26476454/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=KEGG%20as%20a%20Reference%20Resource%20for%20Gene%20and%20Protein%20Annotation&amp;author=M.%20Kanehisa&amp;author=Y.%20Sato&amp;author=M.%20Kawashima&amp;author=M.%20Furumichi&amp;author=M.%20Tanabe&amp;volume=44&amp;issue=D1&amp;publication_year=2016&amp;pages=D457-D462&amp;pmid=26476454&amp;doi=10.1093/nar/gkv1070&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0054">
<span class="label">54.</span><cite>
Martens M., Ammar A., Riutta A., et al., “WikiPathways: Connecting Communities,” Nucleic Acids Research
49, no. D1 (2021): D613–D621.
</cite> [<a href="https://doi.org/10.1093/nar/gkaa1024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7779061/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33211851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=WikiPathways:%20Connecting%20Communities&amp;author=M.%20Martens&amp;author=A.%20Ammar&amp;author=A.%20Riutta&amp;volume=49&amp;issue=D1&amp;publication_year=2021&amp;pages=D613-D621&amp;pmid=33211851&amp;doi=10.1093/nar/gkaa1024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0055">
<span class="label">55.</span><cite>
Patel A. P., Tirosh I., Trombetta J. J., et al., “Single‐cell RNA‐seq Highlights Intratumoral Heterogeneity in Primary Glioblastoma,” Science
344, no. 6186 (2014): 1396–1401.
</cite> [<a href="https://doi.org/10.1126/science.1254257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4123637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24925914/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Single%E2%80%90cell%20RNA%E2%80%90seq%20Highlights%20Intratumoral%20Heterogeneity%20in%20Primary%20Glioblastoma&amp;author=A.%20P.%20Patel&amp;author=I.%20Tirosh&amp;author=J.%20J.%20Trombetta&amp;volume=344&amp;issue=6186&amp;publication_year=2014&amp;pages=1396-1401&amp;pmid=24925914&amp;doi=10.1126/science.1254257&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0056">
<span class="label">56.</span><cite>
Garcia‐Alonso L., Lorenzi V., Mazzeo C. I., Alves‐Lopes J. P., Roberts K., and Sancho‐Serra C., “Single‐Cell Roadmap of human Gonadal Development,” Nature
607, no. 7917 (2022): 540–547.
</cite> [<a href="https://doi.org/10.1038/s41586-022-04918-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9300467/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35794482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Single%E2%80%90Cell%20Roadmap%20of%20human%20Gonadal%20Development&amp;author=L.%20Garcia%E2%80%90Alonso&amp;author=V.%20Lorenzi&amp;author=C.%20I.%20Mazzeo&amp;author=J.%20P.%20Alves%E2%80%90Lopes&amp;author=K.%20Roberts&amp;volume=607&amp;issue=7917&amp;publication_year=2022&amp;pages=540-547&amp;pmid=35794482&amp;doi=10.1038/s41586-022-04918-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0057">
<span class="label">57.</span><cite>
Shannon P., Markiel A., Ozier O., et al., “Cytoscape: A Software Environment for Integrated Models of Biomolecular Interaction Networks,” Genome Research
13, no. 11 (2003): 2498–2504.
</cite> [<a href="https://doi.org/10.1101/gr.1239303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC403769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14597658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=Cytoscape:%20A%20Software%20Environment%20for%20Integrated%20Models%20of%20Biomolecular%20Interaction%20Networks&amp;author=P.%20Shannon&amp;author=A.%20Markiel&amp;author=O.%20Ozier&amp;volume=13&amp;issue=11&amp;publication_year=2003&amp;pages=2498-2504&amp;pmid=14597658&amp;doi=10.1101/gr.1239303&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0058">
<span class="label">58.</span><cite>
Jin S., Guerrero‐Juarez C. F., Zhang L., et al., “Inference and Analysis of Cell‐Cell Communication Using CellChat,” Nature Communications
12, no. 1 (2021): 1088.</cite> [<a href="https://doi.org/10.1038/s41467-021-21246-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7889871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33597522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Inference%20and%20Analysis%20of%20Cell%E2%80%90Cell%20Communication%20Using%20CellChat&amp;author=S.%20Jin&amp;author=C.%20F.%20Guerrero%E2%80%90Juarez&amp;author=L.%20Zhang&amp;volume=12&amp;issue=1&amp;publication_year=2021&amp;pages=1088&amp;pmid=33597522&amp;doi=10.1038/s41467-021-21246-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="mco270335-bib-0059">
<span class="label">59.</span><cite>
Dobin A., Davis C. A., Schlesinger F., et al., “STAR: Ultrafast Universal RNA‐seq Aligner,” Bioinformatics
29, no. 1 (2013): 15–21.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3530905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=STAR:%20Ultrafast%20Universal%20RNA%E2%80%90seq%20Aligner&amp;author=A.%20Dobin&amp;author=C.%20A.%20Davis&amp;author=F.%20Schlesinger&amp;volume=29&amp;issue=1&amp;publication_year=2013&amp;pages=15-21&amp;pmid=23104886&amp;doi=10.1093/bioinformatics/bts635&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Table S1. Clinical profile of patients to receive ABCP treatment.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365383/bin/MCO2-6-e70335-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">MCO2-6-e70335-s002.xlsx</a><sup> (323KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<p>
<strong>Figure S1. Tumor cell cluster analysis</strong>. A) Euclidean distance heatmap of cell type expression profile between our tumor subcluster and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE131907" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE131907</a> epithelial subcluster. B) Heatmap of top‐ranked differentially expressed genes for tumor subclusters.</p>
<p>
<strong>Figure S2. Immune cell cluster analysis</strong>. A) Analysis of myeloid cell type for subcluster identification, its cell proportions, and its canonical markers. Pathways for subclusters were further assessed by gene set enrichment analysis (GSEA). B) Analysis of lymphoid cell type for subcluster identification, its cell proportions, and its canonical markers. Pathways were investigated by GSEA. C) Macrophage subclusters’ cell proportions according to ABCP response, and its canonical markers.</p>
<p>
<strong>Figure S3. Additional spatial transcriptome analysis</strong>. Capture area was 6.5 x 6.5 mm. A‐B are results of Slide1. C‐G are associated with Slide2. A) Cell score distribution on Slide1 according to tumor, stromal, lymphoid, and myeloid scores. B) Heatmap of marker expression to identify the cell type. C) Cell type proportion bar plot and its distribution in Slide2. D) Cell lineage scores for identified cell types. F) Heatmap of expression and scores of VEGF genes and pathways for cell types of Slide2. E) VEGFA and VEGF expression on Slide2 and circular plots of its cell–cell interactions. G) Bar plots of odds ratios (ORs) for <em>VEGFA</em> and <em>VEGFC</em> expression according to cell types.</p>
<p>
<strong>Figure S4. Heatmap of gene expression profile according to cell types and therapeutic groups</strong>. TKI treatment groups were classified into three types: TN, treatment‐naïve (blue); RD, residual disease (yellow‐green), PD, progressive disease (orange).</p>
<p>
<strong>Figure S5. Heatmap of gene–gene interaction network module analysis from <em>VEGFA‐</em> and <em>VEGFC</em>‐regulated networks</strong>. Pathways for each module were extracted by gene set enrichment analysis. Pathways categorized as ‘Common’ were detected in both <em>VEGFA</em> and <em>VEGFC</em> networks. The dominant mechanisms of <em>VEGFA</em> modules (VEGFA_0‐VEGFA_7) and <em>VEGFC</em> (VEGFC_0‐VEGFC_7) are highlighted for each case.</p>
<p>
<strong>Figure 6. Combinational knockdown of <em>VEGFC</em> and its receptor (<em>KDR</em> and <em>NRP2</em>) efficiently enhance the anti‐proliferation and anti‐angiogenesis effects of cabozantinib <em>in vitro</em>
</strong>. A) qPCR results demonstrate the relative mRNA expression of <em>VEGFC</em>, <em>KDR</em>, <em>NRP2</em>, <em>VEGFA</em>, and <em>FLT4</em> in diverse lung cancer cell lines (<em>n</em> = 3). B) IC<sub>50</sub> table of lung cancer cell lines exposed to three VEGF inhibitors. Three cell lines highlighted in red exhibited higher IC<sub>50</sub> values associated with increased expression of <em>VEGFC</em>, <em>KDR</em>, and <em>NRP2</em>. C) Bar graphs depict qPCR results (<em>n</em> = 3) confirming the efficiency of target gene silencing. D‐E) MTT assay results show the relative cell proliferation ratio following cabozantinib treatment with or without specific target gene knockdown in H1573 (D) and H2279 (E) cell lines. The orange bars below each graph indicate cabozantinib treatment (<em>n</em> = 3). F) qPCR results demonstrate the relative mRNA expression of <em>YAP1</em> (left) and <em>TAZ</em> (right) following target gene silencing with or without cabozantinib treatment (marked by an orange bar). G) In vitro angiogenesis assay demonstrates the relative change in capillary‐like tubular structures of HUVECs on Matrigel by conditioned media (CM) treatment. CMs were harvested from H2279 cells following cabozantinib treatment with or without target gene silencing. (Left) Phase contrast microscopic images of capillary‐like tubular structures on Matrigel. Magnification, x100. (Right) Graph bars show the tube length of HUVECs. *<em>P</em>&lt;0.05, **<em>P</em>&lt;0.01, ***<em>P</em>&lt;0.001, ****<em>P</em>&lt;0.0001, one‐way ANOVA, followed by Tukey's multiple‐comparison test.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365383/bin/MCO2-6-e70335-s003.docx" data-ga-action="click_feat_suppl" class="usa-link">MCO2-6-e70335-s003.docx</a><sup> (2.6MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p"><p>Supporting information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12365383/bin/MCO2-6-e70335-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">MCO2-6-e70335-s001.docx</a><sup> (71.6KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All gene expression profiling data generated in this study were deposited in the Gene Expression Omnibus (ABCP: scRNA‐seq GSE233203, spatial scRNA‐seq GSE267960; NCC PDC: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE165611" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE165611</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE229535" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE229535</a>, and PRJNA694788).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from MedComm are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/mco2.70335"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/MCO2-6-e70335.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (7.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365383/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365383/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365383%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365383/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365383/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365383/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40843133/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365383/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40843133/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365383/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365383/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="MHdKeDg0xpDfAea1Auctjhc7gbDZxCU4HQ9eurKO25doYfWlxlbGOboELwp7m9hN">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
