
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Uncertainty Quantification in Epigenetic Clocks via Conformalized Quantile Regression - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="61E605067FFF70430705060029232918.m_1" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11948180/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Genetic Epidemiology">
<meta name="citation_title" content="Uncertainty Quantification in Epigenetic Clocks via Conformalized Quantile Regression">
<meta name="citation_author" content="Yanping Li">
<meta name="citation_author_institution" content="School of Statistics and Data Science, Nankai University, Tianjin, China">
<meta name="citation_author" content="Jaclyn M Goodrich">
<meta name="citation_author_institution" content="Department of Environmental Health Sciences, University of Michigan, Ann Arbor, Michigan, USA">
<meta name="citation_author" content="Karen E Peterson">
<meta name="citation_author_institution" content="Department of Nutritional Sciences, University of Michigan, Ann Arbor, Michigan, USA">
<meta name="citation_author" content="Peter X‐K Song">
<meta name="citation_author_institution" content="Department of Biostatistics, University of Michigan, Ann Arbor, Michigan, USA">
<meta name="citation_author" content="Lan Luo">
<meta name="citation_author_institution" content="Department of Biostatistics and Epidemiology, Rutgers University, Piscataway, New Jersey, USA">
<meta name="citation_publication_date" content="2025 Mar 27">
<meta name="citation_volume" content="49">
<meta name="citation_issue" content="4">
<meta name="citation_firstpage" content="e70008">
<meta name="citation_doi" content="10.1002/gepi.70008">
<meta name="citation_pmid" content="40145332">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11948180/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11948180/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11948180/pdf/GEPI-49-0.pdf">
<meta name="description" content="DNA methylation (DNAm) is a chemical modification of DNA that can be influenced by various factors, including age, the environment, and lifestyle. An epigenetic clock is a predictive tool that measures biological age based on DNAm levels. It can ...">
<meta name="og:title" content="Uncertainty Quantification in Epigenetic Clocks via Conformalized Quantile Regression">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="DNA methylation (DNAm) is a chemical modification of DNA that can be influenced by various factors, including age, the environment, and lifestyle. An epigenetic clock is a predictive tool that measures biological age based on DNAm levels. It can ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11948180/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11948180">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/gepi.70008"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/GEPI-49-0.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11948180%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11948180/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11948180/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11948180/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/gepi.70008" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Genet Epidemiol</button></div>. 2025 Mar 27;49(4):e70008. doi: <a href="https://doi.org/10.1002/gepi.70008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/gepi.70008</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Genet%20Epidemiol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Genet%20Epidemiol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Genet%20Epidemiol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Genet%20Epidemiol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Uncertainty Quantification in Epigenetic Clocks via Conformalized Quantile Regression</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Yanping Li</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yanping Li</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>School of Statistics and Data Science, Nankai University, Tianjin, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yanping Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Goodrich%20JM%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Jaclyn M Goodrich</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jaclyn M Goodrich</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Environmental Health Sciences, University of Michigan, Ann Arbor, Michigan, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Goodrich%20JM%22%5BAuthor%5D" class="usa-link"><span class="name western">Jaclyn M Goodrich</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peterson%20KE%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Karen E Peterson</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Karen E Peterson</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Nutritional Sciences, University of Michigan, Ann Arbor, Michigan, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peterson%20KE%22%5BAuthor%5D" class="usa-link"><span class="name western">Karen E Peterson</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20PXK%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Peter X‐K Song</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Peter X‐K Song</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Department of Biostatistics, University of Michigan, Ann Arbor, Michigan, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20PXK%22%5BAuthor%5D" class="usa-link"><span class="name western">Peter X‐K Song</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luo%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Lan Luo</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Lan Luo</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Department of Biostatistics and Epidemiology, Rutgers University, Piscataway, New Jersey, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luo%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lan Luo</span></a>
</div>
</div>
<sup>5,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="gepi70008-aff-0001">
<sup>
<sup>1</sup>
</sup>School of Statistics and Data Science, Nankai University, Tianjin, China</div>
<div id="gepi70008-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Environmental Health Sciences, University of Michigan, Ann Arbor, Michigan, USA</div>
<div id="gepi70008-aff-0003">
<sup>
<sup>3</sup>
</sup>Department of Nutritional Sciences, University of Michigan, Ann Arbor, Michigan, USA</div>
<div id="gepi70008-aff-0004">
<sup>
<sup>4</sup>
</sup>Department of Biostatistics, University of Michigan, Ann Arbor, Michigan, USA</div>
<div id="gepi70008-aff-0005">
<sup>
<sup>5</sup>
</sup>Department of Biostatistics and Epidemiology, Rutgers University, Piscataway, New Jersey, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong> Lan Luo (<span>ll1118@sph.rutgers.edu</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jan 20; Received 2024 Sep 22; Accepted 2025 Feb 21; Issue date 2025 Jun.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Genetic Epidemiology</em> published by Wiley Periodicals LLC.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11948180  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40145332/" class="usa-link">40145332</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article"><div>
<strong>Previous version available:</strong> This article is based on a previously available preprint posted on medRxiv on February 11, 2025: "<a href="/articles/PMC11398601/" class="usa-link">Uncertainty quantification in epigenetic clocks via conformalized quantile regression</a>".</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>DNA methylation (DNAm) is a chemical modification of DNA that can be influenced by various factors, including age, the environment, and lifestyle. An epigenetic clock is a predictive tool that measures biological age based on DNAm levels. It can provide insights into an individual's biological age, which may differ from their chronological age. This difference, known as the epigenetic age acceleration, may reflect health status and the risk for age‐related diseases. Moreover, epigenetic clocks are used in studies of aging to assess the effectiveness of antiaging interventions and to understand the underlying mechanisms of aging and disease. Various epigenetic clocks have been developed using samples from different populations, tissues, and cell types, typically by training high‐dimensional linear regression models with an elastic net penalty. While these models can predict mean biological age based on DNAm with high precision, there is a lack of uncertainty quantification which is important for interpreting the precision of age estimations and for clinical decision‐making. To understand the distribution of a biological age clock beyond its mean, we propose a general pipeline for training epigenetic clocks, based on an integration of high‐dimensional quantile regression and conformal prediction, to effectively reveal population heterogeneity and construct prediction intervals. Our approach produces adaptive prediction intervals not only achieving nominal coverage but also accounting for the inherent variability across individuals. By using the data collected from 728 blood samples in 11 DNAm data sets from children, we find that our quantile regression‐based prediction intervals are narrower than those derived from conventional mean regression‐based epigenetic clocks. This observation demonstrates an improved statistical efficiency over the existing pipeline for training epigenetic clocks. In addition, the resulting intervals have a synchronized varying pattern to age acceleration, effectively revealing cellular evolutionary heterogeneity in age patterns in different developmental stages during individual childhoods and adolescent cohort. Our findings suggest that conformalized high‐dimensional quantile regression can produce valid prediction intervals and uncover underlying population heterogeneity. Although our methodology focuses on the distribution of measures of biological aging in children, it is applicable to a broader range of age groups to improve understanding of epigenetic age beyond the mean. This inference‐based toolbox could provide valuable insights for future applications of epigenetic interventions for age‐related diseases.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> biological age, conformal prediction, DNA methylation, epigenetic clock, heterogeneity, pediatrics</p></section></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list">
<dt>CpG</dt>
<dd><p>cytosine‐phosphate‐guanine</p></dd>
<dt>DNAm</dt>
<dd><p>DNA methylation</p></dd>
<dt>GEO</dt>
<dd><p>Gene Expression Omnibus</p></dd>
</dl>
<section id="gepi70008-sec-0010"><h2 class="pmc_sec_title">1. Background</h2>
<p>While chronological age is arguably a strong risk factor for aging‐related death and diseases, individuals of the same chronological age may exhibit great heterogeneity in physiologic functions and rate of biological aging. Identifying aging biomarkers is a crucial step in the evaluation of interventions aimed at promoting healthier aging. Epigenetic age is a biomarker of aging that has been reported to be associated with age‐related disease and all‐cause mortality (Horvath <a href="#gepi70008-bib-0015" class="usa-link" aria-describedby="gepi70008-bib-0015">2013</a>; Hannum et al. <a href="#gepi70008-bib-0013" class="usa-link" aria-describedby="gepi70008-bib-0013">2013</a>; Levine et al. <a href="#gepi70008-bib-0021" class="usa-link" aria-describedby="gepi70008-bib-0021">2018</a>; Lu et al. <a href="#gepi70008-bib-0023" class="usa-link" aria-describedby="gepi70008-bib-0023">2019</a>). It has been found that composite measures of DNA methylation (DNAm) levels across specific sets of cytosine‐phosphate‐guanine (CpG) sites, often called epigenetic clocks, are strongly associated with chronological age or age‐related diseases. One of the first and most widely used epigenetic age predictors is Horvath's epigenetic clock (Horvath <a href="#gepi70008-bib-0015" class="usa-link" aria-describedby="gepi70008-bib-0015">2013</a>), a statistical prediction model that uses DNAm at 353 CpG sites to predict chronological age.</p>
<p>Standard approaches for training epigenetic clocks involve several key steps: (i) collecting biological samples from individuals with diverse backgrounds; (ii) extracting DNA and performing DNA methylation analysis; (iii) conducting data preprocessing procedures such as missing data imputation, outliers removal, and data normalization; (iv) adopting a feature screening method for identifying relevant CpG sites that are predictive of age or linked with aging processes; (v) fitting a high‐dimensional regression model with elastic net penalty; and (vi) evaluating model performance on an independent test data set to verify its accuracy and robustness.</p>
<p>Despite the well‐established pipelines for constructing epigenetic clocks, most of them only provide point mean predictions (Horvath <a href="#gepi70008-bib-0015" class="usa-link" aria-describedby="gepi70008-bib-0015">2013</a>; Hannum et al. <a href="#gepi70008-bib-0013" class="usa-link" aria-describedby="gepi70008-bib-0013">2013</a>; Wu et al. <a href="#gepi70008-bib-0039" class="usa-link" aria-describedby="gepi70008-bib-0039">2019</a>). Recently, there has been work focusing on the construction of confidence intervals for DNAm age using a novel U‐learning approach Fei and Li (<a href="#gepi70008-bib-0012" class="usa-link" aria-describedby="gepi70008-bib-0012">2024</a>). While their method is valuable for quantifying uncertainty in parameter estimation, it differs from our goal of constructing prediction intervals that include both variability of sample points and parameter uncertainty. In biological age prediction, it is important not only to predict accurately but also to quantify the uncertainty of the predictions. This is especially true in biological systems that are inherently complex and exhibit variability both within and between individuals. Uncertainty quantification helps to assess the reliability of the models used to construct these clocks, accounting for the variability and stochastic nature of biological aging processes. The uncertainty in a prediction can be quantified using a prediction interval, giving lower and upper bounds between which the response variable lies with high probability. More specifically, epigenetic clocks aim to measure biological age, which can differ significantly from chronological age due to various factors such as lifestyle, genetics, and the environment. Prediction intervals help to interpret these predictions by indicating how much variation there is around the estimated biological age, thus helping to better understand the potential impacts of these factors. In addition, when clinicians or researchers use epigenetic clocks to study aging or assess the risk of age‐related diseases, prediction intervals can help them assess the reliability of the clock in different populations or under different conditions. A narrow interval suggests high reliability and vice versa, influencing the confidence in using these tools for medical or research decisions. However, it is worth noting that existing epigenetic clocks are primarily built from ultra high‐dimensional DNAm data where most inferential methods rely on nontrivial assumptions such as the linear model being true, the error distribution, the homoscedasticity of errors, etc. (Belloni et al. <a href="#gepi70008-bib-0006" class="usa-link" aria-describedby="gepi70008-bib-0006">2012</a>; Berk et al. <a href="#gepi70008-bib-0007" class="usa-link" aria-describedby="gepi70008-bib-0007">2013</a>; Zhang and Zhang <a href="#gepi70008-bib-0041" class="usa-link" aria-describedby="gepi70008-bib-0041">2014</a>; Tibshirani et al. <a href="#gepi70008-bib-0033" class="usa-link" aria-describedby="gepi70008-bib-0033">2016</a>).</p>
<p>In high‐dimensional settings, where the number of predictors (CpGs) exceeds the number of subjects, traditional statistical methods often struggle to provide reliable uncertainty estimates due to overfitting and high variance. Conformal prediction methods can mitigate these issues by constructing prediction intervals that are valid under a minimal set of assumptions about the data distribution (Vovk et al. <a href="#gepi70008-bib-0036" class="usa-link" aria-describedby="gepi70008-bib-0036">2005</a>). They can be applied on top of any predictive model, and this flexibility is particularly useful in constructing prediction intervals for epigenetic clocks. A key component of conformal prediction is the nonconformity score, which quantifies how well a new data point conforms to previously observed data. In the study of epigenetic aging, one of the natural choices of the nonconformity score can be the residual from regressing an epigenetic clock on chronological age, referred to as epigenetic age acceleration which occurs when an individual's DNAm age exceeds their chronological age. Prior studies suggest that older epigenetic age may be associated with lower levels of physical functioning, and declines in global cognitive functioning among long‐lived individuals (Levine et al. <a href="#gepi70008-bib-0021" class="usa-link" aria-describedby="gepi70008-bib-0021">2018</a>; Lu et al. <a href="#gepi70008-bib-0023" class="usa-link" aria-describedby="gepi70008-bib-0023">2019</a>; Marioni et al. <a href="#gepi70008-bib-0025" class="usa-link" aria-describedby="gepi70008-bib-0025">2015</a>; Levine et al. <a href="#gepi70008-bib-0020" class="usa-link" aria-describedby="gepi70008-bib-0020">2015</a>; Sibbett et al. <a href="#gepi70008-bib-0029" class="usa-link" aria-describedby="gepi70008-bib-0029">2020</a>). Since epigenetic age acceleration has also been found to be related to adverse outcomes such as cardiovascular diseases and cancer, it is also scientifically meaningful to incorporate this important biological measure in constructing age prediction intervals. However, commonly used high‐dimensional linear regression models for building existing epigenetic clocks implicitly assume that the association between the DNAm profile and chronological age remains the same across different subpopulations, which may not hold due to heterogeneity across different developmental stages, health conditions, and genders (Wu et al. <a href="#gepi70008-bib-0039" class="usa-link" aria-describedby="gepi70008-bib-0039">2019</a>). Moreover, constructing prediction intervals from conditional mean regression results in intervals of uniform width, failing to capture individual variability in aging rates or age acceleration, which are influenced by many factors including sociodemographic factors, diet, physical activity, genetics, environmental chemical exposures, and more. Apparently, prediction intervals of the same width, even if statistically valid with nominal coverage, are not able to capture cross‐individual heterogeneity in the age acceleration rate across different risk levels.</p>
<p>To bridge this gap, we are interested in looking into subgroup differences across various factors that may affect age acceleration and incorporating such variability into age prediction intervals. First, we propose to build a quantile regression framework for constructing epigenetic clocks. This would allow researchers to examine relationships between DNAm levels and different quantiles of the chronological age. Apparently, different age groups may show a different pattern in their methylation levels across CpG sites, calling for age‐specific epigenetic clocks. For example, we expect that the same CpG site to have varying effects across different age groups. Characterizing such information can provide insight on the impact of DNAm on different age groups or health outcomes. Furthermore, we plan to construct age prediction intervals within the quantile regression framework. For example, to obtain prediction intervals with 90% nominal coverage, we simply fit the conditional quantile function at the 5% and 95% levels to form the corresponding intervals. This method is robust to data with high heteroscedasticity and adaptive to local variability. Besides providing valid coverage in finite samples, the intervals are as short as possible with their length adaptive to individual variability. This enables the identification of individual characteristics that are associated with different levels of uncertainty in their outcome predictions, facilitating tailored interventions.</p>
<p>Throughout this article, we will use children's DNAm data from the Gene Expression Omnibus (GEO) database to illustrate our proposed pipeline for constructing prediction intervals for biological age. Similar to the first‐generation epigenetic clocks, we will use chronological age as the outcome of interest. Previous works have adopted a high‐dimensional linear regression model with elastic net penalty (Zou and Hastie <a href="#gepi70008-bib-0044" class="usa-link" aria-describedby="gepi70008-bib-0044">2005</a>), see for example, the Horvath clock (Horvath <a href="#gepi70008-bib-0015" class="usa-link" aria-describedby="gepi70008-bib-0015">2013</a>), Hannum clock (Hannum et al. <a href="#gepi70008-bib-0013" class="usa-link" aria-describedby="gepi70008-bib-0013">2013</a>), and Levine clock (Levine et al. <a href="#gepi70008-bib-0021" class="usa-link" aria-describedby="gepi70008-bib-0021">2018</a>). However, due to the underlying heterogeneity across different age groups, the associations between DNAm levels and age may vary. As a comparison to the conventionally used high‐dimensional linear regression model, we will first fit a high‐dimensional quantile regression model with the elastic net penalty. Then we will construct prediction intervals and demonstrate their validity as well as statistical efficiency. Our results show that the median regression model outperforms the mean regression model, exhibiting a higher correlation between chronological and predicted ages and a lower median error. Furthermore, our newly proposed approach for constructing prediction intervals not only accommodates subpopulation heterogeneity but is also statistically more efficient than the conventional mean regression‐based method.</p>
<p>We describe the generic pipeline of prediction intervals construction for epigenetic clocks in Section <a href="#gepi70008-sec-0020" class="usa-link">2</a>. Application of the pipeline to establish epigenetic clocks with the children's data set is given in Section <a href="#gepi70008-sec-0060" class="usa-link">3</a>. Finally, in Section <a href="#gepi70008-sec-0120" class="usa-link">4</a>, we present a discussion, outline the limitations, and suggest areas for future research.</p></section><section id="gepi70008-sec-0020"><h2 class="pmc_sec_title">2. Methods</h2>
<p>The overarching goal of this study is to construct prediction intervals for epigenetic clocks with adaptive widths that account for cross‐subjects variability. Our proposed generic pipeline involves the following two major components: (i) training high‐dimensional predictive models for predicting epigenetic ages, and (ii) constructing valid prediction intervals with the conformal inference methods. A diagram of the proposed generic pipeline is shown in Figure <a href="#gepi70008-fig-0001" class="usa-link">1</a>.</p>
<figure class="fig xbox font-sm" id="gepi70008-fig-0001"><h3 class="obj_head">Figure 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0549/11948180/a43f76bf7f32/GEPI-49-0-g002.jpg" loading="lazy" id="jats-graphic-1" height="416" width="702" alt="Figure 1"></p>
<div class="p text-right font-secondary"><a href="figure/gepi70008-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Flowchart of the proposed generic pipeline that integrates two key components: (i) training epigenetic clock, (ii) constructing prediction intervals, and (iii) evaluating performance as highlighted in the blue boxes. Details of each step (shown in yellow boxes) will be presented in Section <a href="#gepi70008-sec-0020" class="usa-link">2</a>.</p></figcaption></figure><section id="gepi70008-sec-0030"><h3 class="pmc_sec_title">2.1. Quantile Regression Based Epigenetic Clock</h3>
<p>DNA methylation data is typically characterized by ultrahigh dimensions since hundreds of thousands of CpG sites are profiled. The entire procedure to construct quantile regression based epigenetic clock can be separated into three steps. In our regression problem, we observe independent and identically distributed samples <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-1" display="inline"><mrow><mrow><mrow><mrow><mo>(</mo><msub><mi>X</mi><mi>i</mi></msub><mo>,</mo><msub><mi>Y</mi><mi>i</mi></msub><mo>)</mo></mrow><mo>∈</mo><msup><mi mathvariant="normal">ℝ</mi><mi>q</mi></msup><mo>×</mo><mi mathvariant="normal">ℝ</mi><mo>~</mo><mi>P</mi></mrow></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-2" display="inline"><mrow><mrow><mrow><msub><mi>X</mi><mi>i</mi></msub><mo>∈</mo><msup><mi mathvariant="double-struck">R</mi><mi>q</mi></msup></mrow></mrow></mrow></math></span> is the vector of methylation levels of profiled CpG sites, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-3" display="inline"><mrow><mrow><mrow><msub><mi>Y</mi><mi>i</mi></msub><mo>∈</mo><mi mathvariant="double-struck">R</mi></mrow></mrow></mrow></math></span> is the chronological age of subject <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline"><mrow><mrow><mrow><mi>i</mi></mrow></mrow></mrow></math></span> for <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-5" display="inline"><mrow><mrow><mrow><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mi mathvariant="normal">…</mi><mo>,</mo><mi>n</mi></mrow></mrow></mrow></math></span>.</p>
<p>Step 1 (Data preprocessing): As mentioned by Horvath (<a href="#gepi70008-bib-0015" class="usa-link" aria-describedby="gepi70008-bib-0015">2013</a>) and Wu et al. (<a href="#gepi70008-bib-0039" class="usa-link" aria-describedby="gepi70008-bib-0039">2019</a>), we adopt similar data preprocessing methods. In particular, we first integrate data by concentrating on common CpG sites, with several public Illumina DNA data sets relevant to target population. Then we discard any CpG site with more than 10 missing DNA methylation values. For sites with fewer than 10 missing values, we impute with the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline"><mrow><mrow><mrow><mi>k</mi></mrow></mrow></mrow></math></span>‐nearest‐neighbors approach with the <kbd>impute</kbd> package in <kbd>R</kbd>. We study overlapping CpGs that are present on all data sets and carry out the normalization step to ensure that these data are comparable by adapting the <kbd>BMIQ R</kbd> function from Teschendorff et al. (<a href="#gepi70008-bib-0031" class="usa-link" aria-describedby="gepi70008-bib-0031">2013</a>) so that it would rescale all probes of each array to match their distribution with the determined gold standard. We also perform a principal component analysis to identify and remove outliers by converting each sample into a Z‐score statistic, transforming it to the false‐discovery rate, and removing samples falling below a false‐discovery rate of 0.2. To avoid negative values in the predicted age, we apply a transformation function <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline"><mrow><mrow><mrow><mi>G</mi><mrow><mo>(</mo><mi>Y</mi><mo>)</mo></mrow></mrow></mrow></mrow></math></span> to the original chronological age before model fitting, with the transformed age denoted by <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-8" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>Y</mi><mo stretchy="true">˜</mo></mover></mrow></mrow></mrow></math></span>. The transformation function <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-9" display="inline"><mrow><mrow><mrow><mi>G</mi><mrow><mo>(</mo><mi>Y</mi><mo>)</mo></mrow></mrow></mrow></mrow></math></span> should be strictly increasing to preserve the order of ages, invertible to allow mapping back to the original scale, and continuous to ensure smoothness. Additionally, the inverse‐transformed values <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-10" display="inline"><mrow><mrow><mrow><msup><mi>G</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup><mrow><mo>(</mo><mi>Y</mi><mo>)</mo></mrow></mrow></mrow></mrow></math></span> must be strictly positive to ensure valid age predictions. For example, a logarithmic transformation satisfies all these requirements. After constructing prediction intervals, we will convert them back to their original scale using the exponential function to ensure positivity. Lastly, the entire data set is randomly split into training set, calibration set and test set with a proper proportion (Romano et al. <a href="#gepi70008-bib-0028" class="usa-link" aria-describedby="gepi70008-bib-0028">2019</a>). Let <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-11" display="inline"><mrow><mrow><mrow><msub><mi class="MJX-tex-caligraphic" mathvariant="script">D</mi><mi mathvariant="monospace">train</mi></msub><mo>,</mo><msub><mi class="MJX-tex-caligraphic" mathvariant="script">D</mi><mi mathvariant="monospace">cal</mi></msub></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-12" display="inline"><mrow><mrow><mrow><msub><mi class="MJX-tex-caligraphic" mathvariant="script">D</mi><mi mathvariant="monospace">test</mi></msub></mrow></mrow></mrow></math></span> denote the set of sample indices in the training set, calibration set and test set, respectively.</p>
<p>Step 2 (Feature screening): Similar to the training procedures for most existing epigenetic clocks, we first adopt the sure independence screening (SIS) method to reduce the number of CpG sites. Since we plan to fit high‐dimensional quantile regression models in our subsequent analysis, we choose to work with a model‐free generic SIS procedure with fewer and less restrictive assumptions (Barut et al. <a href="#gepi70008-bib-0005" class="usa-link" aria-describedby="gepi70008-bib-0005">2016</a>). For example, correlation‐based SIS can be summarized as several sketched stages. Firstly, for each feature <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-13" display="inline"><mrow><mrow><mrow><msub><mi>X</mi><mi>j</mi></msub></mrow></mrow></mrow></math></span>, compute its correlation with the response variable <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-14" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>Y</mi><mo stretchy="true">˜</mo></mover></mrow></mrow></mrow></math></span>, that is,
</p>
<table class="disp-formula p" id="gepi70008-disp-0001"><tr><td class="formula"><math id="jats-math-15" display="block"><mrow><mrow><mrow><mstyle><mspace width="0.1em"></mspace><mtext>Corr</mtext><mspace width="0.1em"></mspace></mstyle><mrow><mo>(</mo><mrow><msub><mi>X</mi><mi>j</mi></msub><mo>,</mo><mover accent="true"><mi>Y</mi><mo stretchy="true">˜</mo></mover></mrow><mo>)</mo></mrow><mo>=</mo><mfrac><mrow><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></msubsup><mrow><mo>(</mo><mrow><msub><mi>X</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>−</mo><mover accent="true"><msub><mi>X</mi><mi>j</mi></msub><mo>¯</mo></mover></mrow><mo>)</mo></mrow><mrow><mo>(</mo><mrow><msub><mover accent="true"><mi>Y</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>−</mo><mover accent="true"><mi>Y</mi><mo>¯</mo></mover></mrow><mo>)</mo></mrow></mrow><msqrt><mrow><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></msubsup><msup><mrow><mo>(</mo><mrow><msub><mi>X</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>−</mo><mover accent="true"><msub><mi>X</mi><mi>j</mi></msub><mo>¯</mo></mover></mrow><mo>)</mo></mrow><mn>2</mn></msup><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></msubsup><msup><mrow><mo>(</mo><mrow><msub><mover accent="true"><mi>Y</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>−</mo><mover accent="true"><mi>Y</mi><mo>¯</mo></mover></mrow><mo>)</mo></mrow><mn>2</mn></msup></mrow></msqrt></mfrac><mo>,</mo></mrow></mrow></mrow></math></td></tr></table>
<p> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-16" display="inline"><mrow><mrow><mrow><mover accent="true"><msub><mi>X</mi><mi>j</mi></msub><mo>¯</mo></mover></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-17" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>Y</mi><mo>¯</mo></mover></mrow></mrow></mrow></math></span> are the mean values of the feature <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-18" display="inline"><mrow><mrow><mrow><msub><mi>X</mi><mi>j</mi></msub></mrow></mrow></mrow></math></span> and the response variable <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-19" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>Y</mi><mo stretchy="true">˜</mo></mover></mrow></mrow></mrow></math></span>, respectively. Then rank the features based on the absolute values of their correlations, i.e., <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-20" display="inline"><mrow><mrow><mrow><mo>∣</mo><mstyle><mspace width="0.1em"></mspace><mtext>Corr</mtext><mspace width="0.1em"></mspace></mstyle><mrow><mo>(</mo><mrow><msub><mi>X</mi><mi>j</mi></msub><mo>,</mo><mover accent="true"><mi>Y</mi><mo stretchy="true">˜</mo></mover></mrow><mo>)</mo></mrow><mo>∣</mo></mrow></mrow></mrow></math></span>. Finally, select the top <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-21" display="inline"><mrow><mrow><mrow><mi>p</mi></mrow></mrow></mrow></math></span> features with the highest absolute correlations, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-22" display="inline"><mrow><mrow><mrow><mi>p</mi></mrow></mrow></mrow></math></span> is a predetermined number of features to retain. Let <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-23" display="inline"><mrow><mrow><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">S</mi><mo>⊂</mo><mrow><mo class="MathClass-open">{</mo><mrow><mn>1</mn><mo>,</mo><mn>2</mn><mo>,</mo><mi mathvariant="normal">…</mi><mo>,</mo><mi>q</mi></mrow><mo class="MathClass-close">}</mo></mrow></mrow></mrow></mrow></math></span> be the set of the index of selected features, and the size of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-24" display="inline"><mrow><mrow><mrow><mi class="MJX-tex-caligraphic" mathvariant="script">S</mi></mrow></mrow></mrow></math></span> is obviously <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-25" display="inline"><mrow><mrow><mrow><mi>p</mi></mrow></mrow></mrow></math></span>. To determine the most appropriate feature screening procedure for the analysis of our data set, we try several popular screening methods that have been implemented in the <kbd>MFSIS</kbd> package in <kbd>R</kbd>, such as SIRS (Zhu et al. <a href="#gepi70008-bib-0043" class="usa-link" aria-describedby="gepi70008-bib-0043">2011</a>), DC‐SIS (Li et al. <a href="#gepi70008-bib-0022" class="usa-link" aria-describedby="gepi70008-bib-0022">2012</a>), Kfilter (Mai and Zou <a href="#gepi70008-bib-0024" class="usa-link" aria-describedby="gepi70008-bib-0024">2015</a>), CSIS (Barut et al. <a href="#gepi70008-bib-0005" class="usa-link" aria-describedby="gepi70008-bib-0005">2016</a>), Bcor‐SIS (Pan et al. <a href="#gepi70008-bib-0027" class="usa-link" aria-describedby="gepi70008-bib-0027">2019</a>), and WLS (Zhong et al. <a href="#gepi70008-bib-0042" class="usa-link" aria-describedby="gepi70008-bib-0042">2023</a>) to the training set. For each method, we use a 10‐fold cross‐validation approach to fit regression models predicting chronological age from the selected CpG sites. Within each fold, we optimize the models by selecting the best tuning parameters and computing the correlation coefficient between predicted values and true values. We evaluate the performances of all different types of feature screening methods across all models in our subsequent analyses, and choose the one that fits best to each of the regression models.</p>
<p>Step 3 (High‐dimensional quantile regression): After applying the feature screening method, we then fit a high‐dimensional quantile regression model with elastic net penalty with selected CpGs. It is implemented with the <kbd>conquer</kbd> package that adopts a convolution‐type smoothed quantile regression (Tan et al. <a href="#gepi70008-bib-0030" class="usa-link" aria-describedby="gepi70008-bib-0030">2022</a>). Let <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-26" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mo>=</mo><mrow><mo>(</mo><mrow><msub><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mn>1</mn></msub><mo>,</mo><mi mathvariant="normal">…</mi><mo>,</mo><msub><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mi>n</mi></msub></mrow><mo>)</mo></mrow><mo>∈</mo><msup><mi mathvariant="double-struck">R</mi><mrow><mi>p</mi><mo>×</mo><mi>n</mi></mrow></msup></mrow></mrow></mrow></math></span> be the matrix that contains only selected features in Step 2. The estimated coefficient vector <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-27" display="inline"><mrow><mrow><mrow><msub><mover accent="true"><mi>β</mi><mo stretchy="true">^</mo></mover><mi>τ</mi></msub></mrow></mrow></mrow></math></span> is given by
</p>
<table class="disp-formula p" id="gepi70008-disp-0002"><tr><td class="formula"><math id="jats-math-28" display="block"><mrow><mrow><mrow><msub><mover accent="true"><mi>β</mi><mo stretchy="true">^</mo></mover><mi>τ</mi></msub><mo>=</mo><munder><mrow><mtext>arg</mtext><mi>min</mi></mrow><mrow><mi>β</mi><mo>∈</mo><msup><mi mathvariant="double-struck">R</mi><mi>p</mi></msup></mrow></munder><mspace width="0.25em"></mspace><mfenced close="}" open="{"><mrow><mfrac><mn>1</mn><mi>n</mi></mfrac><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><msub><mi>ρ</mi><mi>τ</mi></msub><mfenced close=")" open="("><mrow><msub><mover accent="true"><mi>Y</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>−</mo><msubsup><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mi>i</mi><mo>⊤</mo></msubsup><mi>β</mi></mrow></mfenced><mo>+</mo><mi>α</mi><mfenced close=")" open="("><mrow><mi>λ</mi><mo>∥</mo><mi>β</mi><msub><mo>∥</mo><mn>1</mn></msub><mo>+</mo><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>λ</mi></mrow><mo>)</mo></mrow><mo>∥</mo><mi>β</mi><msubsup><mo>∥</mo><mn>2</mn><mn>2</mn></msubsup></mrow></mfenced></mrow></mfenced><mo>,</mo></mrow></mrow></mrow></math></td></tr></table>
<p> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-29" display="inline"><mrow><mrow><mrow><msub><mi>ρ</mi><mi>τ</mi></msub><mrow><mo>(</mo><mi>u</mi><mo>)</mo></mrow><mo>=</mo><mi>u</mi><mrow><mo>(</mo><mrow><mi>τ</mi><mo>−</mo><msub><mn mathvariant="bold">1</mn><mrow><mo class="MathClass-open">{</mo><mrow><mi>u</mi><mo>&lt;</mo><mn>0</mn></mrow><mo class="MathClass-close">}</mo></mrow></msub></mrow><mo>)</mo></mrow></mrow></mrow></mrow></math></span> is the quantile loss function, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-30" display="inline"><mrow><mrow><mrow><mo>∥</mo><mi>β</mi><msub><mo>∥</mo><mn>1</mn></msub></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-31" display="inline"><mrow><mrow><mrow><mo>∥</mo><mi>β</mi><msub><mo>∥</mo><mn>2</mn></msub></mrow></mrow></mrow></math></span> refer to the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-32" display="inline"><mrow><mrow><mrow><msub><mi>ℓ</mi><mn>1</mn></msub></mrow></mrow></mrow></math></span>‐norm and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-33" display="inline"><mrow><mrow><mrow><msub><mi>ℓ</mi><mn>2</mn></msub></mrow></mrow></mrow></math></span>‐norm separately of the coefficient vector, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-34" display="inline"><mrow><mrow><mrow><mi>α</mi><mo>&gt;</mo><mn>0</mn></mrow></mrow></mrow></math></span> is the overall regularization strength, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-35" display="inline"><mrow><mrow><mrow><mn>0</mn><mo>≤</mo><mi>λ</mi><mo>≤</mo><mn>1</mn></mrow></mrow></mrow></math></span> is the regularization parameter that controls the balance between <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-36" display="inline"><mrow><mrow><mrow><msub><mi>ℓ</mi><mn>1</mn></msub></mrow></mrow></mrow></math></span> (Lasso) and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-37" display="inline"><mrow><mrow><mrow><msub><mi>ℓ</mi><mn>2</mn></msub></mrow></mrow></mrow></math></span> (Ridge) regularization. As a result, the fitted <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-38" display="inline"><mrow><mrow><mrow><mi>τ</mi></mrow></mrow></mrow></math></span>th conditional quantile of epigenetic age given the methylation levels is formed by <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-39" display="inline"><mrow><mrow><mrow><msub><mover accent="true"><mi>f</mi><mo stretchy="true">^</mo></mover><mi>τ</mi></msub><mrow><mo>(</mo><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mo>)</mo></mrow><mo>=</mo><msup><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mo>⊤</mo></msup><msub><mover accent="true"><mi>β</mi><mo stretchy="true">^</mo></mover><mi>τ</mi></msub></mrow></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-40" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover></mrow></mrow></mrow></math></span> is the vector of DNA methylation level in a new subject.</p>
<p>As a baseline comparison, we also fit a high‐dimensional conditional mean regression model with elastic net penalty (Zou and Hastie <a href="#gepi70008-bib-0044" class="usa-link" aria-describedby="gepi70008-bib-0044">2005</a>) using the <kbd>R</kbd> package <kbd>glmnet</kbd>. The corresponding estimated coefficient vector is the solution to the following optimization problem:
</p>
<table class="disp-formula p" id="gepi70008-disp-0003"><tr><td class="formula"><math id="jats-math-41" display="block"><mrow><mrow><mrow><msub><mover accent="true"><mi>β</mi><mo stretchy="true">^</mo></mover><mi mathvariant="monospace">mean</mi></msub><mo>=</mo><munder><mrow><mtext>arg</mtext><mi>min</mi></mrow><mrow><mi>β</mi><mo>∈</mo><msup><mi mathvariant="double-struck">R</mi><mi>p</mi></msup></mrow></munder><mfenced close="}" open="{"><mrow><mfrac><mn>1</mn><mi>n</mi></mfrac><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>n</mi></munderover><msup><mfenced close=")" open="("><mrow><msub><mover accent="true"><mi>Y</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>−</mo><msubsup><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mi>i</mi><mo>⊤</mo></msubsup><mi>β</mi></mrow></mfenced><mn>2</mn></msup><mo>+</mo><mi>α</mi><mfenced close=")" open="("><mrow><mi>λ</mi><mo>∥</mo><mi>β</mi><msub><mo>∥</mo><mn>1</mn></msub><mo>+</mo><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>λ</mi></mrow><mo>)</mo></mrow><mo>∥</mo><mi>β</mi><msubsup><mo>∥</mo><mn>2</mn><mn>2</mn></msubsup></mrow></mfenced></mrow></mfenced><mo>.</mo></mrow></mrow></mrow></math></td></tr></table>
<p>The resulting mean regression‐based epigenetic clock is given by <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-42" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>μ</mi><mo stretchy="true">^</mo></mover><mrow><mo>(</mo><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mo>)</mo></mrow><mo>=</mo><msup><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mo>⊤</mo></msup><msub><mover accent="true"><mi>β</mi><mo stretchy="true">^</mo></mover><mi mathvariant="monospace">mean</mi></msub></mrow></mrow></mrow></math></span>.</p></section><section id="gepi70008-sec-0040"><h3 class="pmc_sec_title">2.2. Uncertainty Quantification and Interval Construction in Age Prediction</h3>
<p>The aforementioned pipeline for constructing epigenetic clocks in Section <a href="#gepi70008-sec-0030" class="usa-link">2.1</a> provides point predictions only. However, relying solely on point predictions, without considering the uncertainty or variability, makes it difficult to evaluate the trustworthiness of the predicted biological age or to relate these measures to age‐related diseases. Consequently, it is desirable to construct prediction intervals as this quantifies the uncertainty in the prediction and offers a clearer picture of how precise or reliable the age prediction is. While there are various types of conformal prediction methods, we choose to work with the split conformal prediction due to its computational efficiency (Lei et al. <a href="#gepi70008-bib-0019" class="usa-link" aria-describedby="gepi70008-bib-0019">2018</a>). Our goal in this section is to construct a level <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-43" display="inline"><mrow><mrow><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>γ</mi></mrow><mo>)</mo></mrow></mrow></mrow></math></span> prediction interval for biological age prediction.</p>
<p>Ideally, we would like to construct prediction intervals that adapt to the heteroscedasticity in the data. This means the resulting intervals can have wider width in subpopulations with high uncertainty and narrow in those with low uncertainty, providing a more accurate depiction of the confidence in the age prediction model. Conformalized quantile regression (Romano et al. <a href="#gepi70008-bib-0028" class="usa-link" aria-describedby="gepi70008-bib-0028">2019</a>) merges quantile regression and conformal prediction to produce prediction intervals that adapt to the underlying distribution of the data while maintaining rigorous coverage guarantees. This motivates us to integrate high‐dimensional quantile regression models into the conformal prediction framework for epigenetic age predictions.</p>
<p>The framework of split conformal prediction will be integrated with epigenetic clocks based on mean regression and quantile regression, respectively. As a continuation of Section <a href="#gepi70008-sec-0030" class="usa-link">2.1</a>, we use <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-44" display="inline"><mrow><mrow><mrow><msub><mover accent="true"><mi>f</mi><mo stretchy="true">^</mo></mover><mrow><mi>γ</mi><mo>∕</mo><mn>2</mn></mrow></msub><mrow><mo>(</mo><mi>X</mi><mo>)</mo></mrow></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-45" display="inline"><mrow><mrow><mrow><msub><mover accent="true"><mi>f</mi><mo stretchy="true">^</mo></mover><mrow><mn>1</mn><mo>−</mo><mi>γ</mi><mo>∕</mo><mn>2</mn></mrow></msub><mrow><mo>(</mo><mi>X</mi><mo>)</mo></mrow></mrow></mrow></mrow></math></span> to denote the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-46" display="inline"><mrow><mrow><mrow><mi>γ</mi><mo>∕</mo><mn>2</mn></mrow></mrow></mrow></math></span>th and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-47" display="inline"><mrow><mrow><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>γ</mi><mo>∕</mo><mn>2</mn></mrow><mo>)</mo></mrow></mrow></mrow></math></span>th condition quantile functions, respectively. We further use <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-48" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>μ</mi><mo stretchy="true">^</mo></mover><mrow><mo>(</mo><mi>X</mi><mo>)</mo></mrow></mrow></mrow></mrow></math></span> to denote the mean regression‐based epigenetic clock. The split conformal prediction separates the fitting and ranking steps using sample splitting. More specifically, to avoid overfitting, the fitting or training steps in Section <a href="#gepi70008-sec-0030" class="usa-link">2.1</a> is done with the training set <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-49" display="inline"><mrow><mrow><mrow><msub><mi class="MJX-tex-caligraphic" mathvariant="script">D</mi><mi mathvariant="monospace">train</mi></msub></mrow></mrow></mrow></math></span> while the calibration is carried out with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-50" display="inline"><mrow><mrow><mrow><msub><mi class="MJX-tex-caligraphic" mathvariant="script">D</mi><mi mathvariant="monospace">cal</mi></msub></mrow></mrow></mrow></math></span>.</p>
<p>Step 4 (Residuals and conformity scores): For the mean regression model, we apply <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-51" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>μ</mi><mo stretchy="true">^</mo></mover><mrow><mo>(</mo><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mo>)</mo></mrow></mrow></mrow></mrow></math></span> to the samples in the calibration set and calculate the absolute values of residuals <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-52" display="inline"><mrow><mrow><mrow><msub><mi>R</mi><mi>i</mi></msub><mo>=</mo><mo>∣</mo><msub><mover accent="true"><mi>Y</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>−</mo><mover accent="true"><mi>μ</mi><mo stretchy="true">^</mo></mover><mrow><mo>(</mo><msub><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>)</mo></mrow><mo>∣</mo><mo>,</mo><mi>i</mi><mo>∈</mo><msub><mi class="MJX-tex-caligraphic" mathvariant="script">D</mi><mi mathvariant="monospace">cal</mi></msub></mrow></mrow></mrow></math></span>. This deviation, in the field of epigenetic aging, is interpreted as the magnitude of age acceleration or deceleration, depending on whether <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-53" display="inline"><mrow><mrow><mrow><msub><mover accent="true"><mi>Y</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub></mrow></mrow></mrow></math></span> is greater than <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-54" display="inline"><mrow><mrow><mrow><mover accent="true"><mi>μ</mi><mo stretchy="true">^</mo></mover><mrow><mo>(</mo><msub><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>)</mo></mrow></mrow></mrow></mrow></math></span>. Among the calculated residuals, we find the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-55" display="inline"><mrow><mrow><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>γ</mi></mrow><mo>)</mo></mrow></mrow></mrow></math></span>th quantile of the empirical distribution of the absolute values of residuals, denoted by <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-56" display="inline"><mrow><mrow><mrow><msub><mi>d</mi><mi mathvariant="monospace">mean</mi></msub><mo>≔</mo><mi>d</mi><mrow><mo>(</mo><msub><mi class="MJX-tex-caligraphic" mathvariant="normal">ℛ</mi><mi mathvariant="monospace">cal</mi></msub><mo>)</mo></mrow><mo>=</mo></mrow></mrow></mrow></math></span> the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-57" display="inline"><mrow><mrow><mrow><mi>k</mi></mrow></mrow></mrow></math></span>th smallest value in <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-58" display="inline"><mrow><mrow><mrow><msub><mi class="MJX-tex-caligraphic" mathvariant="normal">ℛ</mi><mi mathvariant="monospace">cal</mi></msub><mo>=</mo><mo>{</mo><mrow><msub><mi>R</mi><mi>i</mi></msub><mo>:</mo><mi>i</mi><mo>∈</mo><msub><mi class="MJX-tex-caligraphic" mathvariant="script">D</mi><mi mathvariant="monospace">cal</mi></msub></mrow><mo>}</mo></mrow></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-59" display="inline"><mrow><mrow><mrow><mi>k</mi><mo>=</mo><mrow><mo stretchy="false">⌈</mo><mrow><mo>(</mo><mrow><mi>n</mi><mo>∕</mo><mn>2</mn><mo>+</mo><mn>1</mn></mrow><mo>)</mo><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>γ</mi></mrow><mo>)</mo></mrow><mo stretchy="false">⌉</mo></mrow></mrow></mrow></mrow></math></span>.</p>
<p>Different from the mean regression models, quantile regression offers a natural framework for constructing prediction intervals possibly formed by <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-60" display="inline"><mrow><mrow><mrow><msub><mover accent="true"><mi>f</mi><mo stretchy="true">^</mo></mover><mrow><mi>γ</mi><mo>∕</mo><mn>2</mn></mrow></msub><mrow><mo>(</mo><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mo>)</mo></mrow></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-61" display="inline"><mrow><mrow><mrow><msub><mover accent="true"><mi>f</mi><mo stretchy="true">^</mo></mover><mrow><mn>1</mn><mo>−</mo><mi>γ</mi><mo>∕</mo><mn>2</mn></mrow></msub><mrow><mo>(</mo><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mo>)</mo></mrow></mrow></mrow></mrow></math></span>. To avoid overfitting, we apply these two fitted quantile functions to the calibration set and calculate the conformity scores defined as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-62" display="inline"><mrow><mrow><mrow><msub><mi>S</mi><mi>i</mi></msub><mo>=</mo><mi>max</mi><mrow><mo class="MathClass-open">{</mo><mrow><msub><mover accent="true"><mi>f</mi><mo stretchy="true">^</mo></mover><mrow><mi>γ</mi><mo>∕</mo><mn>2</mn></mrow></msub><mrow><mo>(</mo><msub><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>)</mo></mrow><mo>−</mo><msub><mover accent="true"><mi>Y</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>,</mo><msub><mover accent="true"><mi>Y</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>−</mo><msub><mover accent="true"><mi>f</mi><mo stretchy="true">^</mo></mover><mrow><mn>1</mn><mo>−</mo><mi>γ</mi><mo>∕</mo><mn>2</mn></mrow></msub><mrow><mo>(</mo><msub><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>)</mo></mrow></mrow><mo class="MathClass-close">}</mo></mrow><mo>,</mo><mi>i</mi><mo>∈</mo><msub><mi class="MJX-tex-caligraphic" mathvariant="script">D</mi><mi mathvariant="monospace">cal</mi></msub></mrow></mrow></mrow></math></span>. Similarly, among the calculated conformity scores, we find the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-63" display="inline"><mrow><mrow><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>γ</mi></mrow><mo>)</mo></mrow></mrow></mrow></math></span>th quantile of the empirical conformity score distribution, denoted by <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-64" display="inline"><mrow><mrow><mrow><msub><mi>d</mi><mi mathvariant="monospace">quantile</mi></msub><mo>≔</mo><mi>d</mi><mrow><mo>(</mo><msub><mi class="MJX-tex-caligraphic" mathvariant="script">S</mi><mi mathvariant="monospace">cal</mi></msub><mo>)</mo></mrow></mrow></mrow></mrow></math></span> the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-65" display="inline"><mrow><mrow><mrow><mi>k</mi></mrow></mrow></mrow></math></span>th smallest value in <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-66" display="inline"><mrow><mrow><mrow><msub><mi class="MJX-tex-caligraphic" mathvariant="script">S</mi><mi mathvariant="monospace">cal</mi></msub><mo>=</mo><mo>{</mo><mrow><msub><mi>S</mi><mi>i</mi></msub><mo>:</mo><mi>i</mi><mo>∈</mo><msub><mi class="MJX-tex-caligraphic" mathvariant="script">D</mi><mi mathvariant="monospace">cal</mi></msub></mrow><mo>}</mo></mrow></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-67" display="inline"><mrow><mrow><mrow><mi>k</mi><mo>=</mo><mrow><mo stretchy="false">⌈</mo><mrow><mo>(</mo><mrow><mi>n</mi><mo>∕</mo><mn>2</mn><mo>+</mo><mn>1</mn></mrow><mo>)</mo><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>γ</mi></mrow><mo>)</mo></mrow><mo stretchy="false">⌉</mo></mrow></mrow></mrow></mrow></math></span>.</p>
<p>Step 5 (Constructing prediction intervals): The prediction interval for the mean regression model is constructed with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-68" display="inline"><mrow><mrow><mrow><msub><mi>C</mi><mi mathvariant="monospace">mean</mi></msub><mrow><mo>(</mo><msub><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>)</mo></mrow><mo>=</mo><mrow><mo class="MathClass-open">[</mo><mrow><msup><mi>G</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup><mrow><mo>(</mo><mrow><mover accent="true"><mi>μ</mi><mo stretchy="true">^</mo></mover><mrow><mo>(</mo><msub><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>)</mo></mrow><mo>−</mo><msub><mi>d</mi><mi mathvariant="monospace">mean</mi></msub></mrow><mo>)</mo></mrow><mo>,</mo><msup><mi>G</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup><mrow><mo>(</mo><mrow><mover accent="true"><mi>μ</mi><mo stretchy="true">^</mo></mover><mrow><mo>(</mo><msub><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>)</mo></mrow><mo>+</mo><msub><mi>d</mi><mi mathvariant="monospace">mean</mi></msub></mrow><mo>)</mo></mrow></mrow><mo class="MathClass-close">]</mo></mrow></mrow></mrow></mrow></math></span>, for <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-69" display="inline"><mrow><mrow><mrow><mi>i</mi><mo>∈</mo><msub><mi class="MJX-tex-caligraphic" mathvariant="script">D</mi><mi mathvariant="monospace">test</mi></msub></mrow></mrow></mrow></math></span>. It is clear to see that the conformal prediction technique, if being integrated into the conditional mean regression‐based epigenetic clock, can be less informative because the resulting prediction intervals are of a relatively consistent width of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-70" display="inline"><mrow><mrow><mrow><mn>2</mn><msub><mi>d</mi><mi mathvariant="monospace">mean</mi></msub></mrow></mrow></mrow></math></span> across all subjects. More importantly, intervals of approximately constant length do not truly reflect the variation in epigenetic age acceleration across different stages of human lifespan, or more generally, across subpopulations with different health status, gender, ethnicity, and environmental exposures (Zavala et al. <a href="#gepi70008-bib-0040" class="usa-link" aria-describedby="gepi70008-bib-0040">2024</a>).</p>
<p>The prediction interval for quantile regression models is constructed with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-71" display="inline"><mrow><mrow><mrow><msub><mi>C</mi><mi mathvariant="monospace">quantile</mi></msub><mrow><mo>(</mo><msub><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>)</mo></mrow><mo>=</mo><mo class="MathClass-open">[</mo><mrow><msup><mi>G</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup><mrow><mo>(</mo><mrow><msub><mover accent="true"><mi>f</mi><mo stretchy="true">^</mo></mover><mrow><mi>γ</mi><mo>∕</mo><mn>2</mn></mrow></msub><mrow><mo>(</mo><msub><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>)</mo></mrow><mo>−</mo><msub><mi>d</mi><mi mathvariant="monospace">quantile</mi></msub></mrow><mo>)</mo><mo>,</mo><msup><mi>G</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup><mrow><mo>(</mo><mrow><msub><mover accent="true"><mi>f</mi><mo stretchy="true">^</mo></mover><mrow><mn>1</mn><mo>−</mo><mi>γ</mi><mo>∕</mo><mn>2</mn></mrow></msub><mrow><mo>(</mo><msub><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>)</mo></mrow><mo>+</mo><msub><mi>d</mi><mi mathvariant="monospace">quantile</mi></msub></mrow><mo>)</mo></mrow></mrow><mo class="MathClass-close">]</mo></mrow></mrow></mrow></mrow></math></span>, for <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-72" display="inline"><mrow><mrow><mrow><mi>i</mi><mo>∈</mo><msub><mi class="MJX-tex-caligraphic" mathvariant="script">D</mi><mi mathvariant="monospace">test</mi></msub></mrow></mrow></mrow></math></span>. Different from the prediction intervals constructed from mean regression‐based conformal prediction, this method generates intervals with width adaptive to individual DNAm profiles and ensures strictly positive lower and upper bounds. In an ideal case where the fitted conditional quantile functions fit perfectly to the calibration set, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-73" display="inline"><mrow><mrow><mrow><msub><mi>S</mi><mi>i</mi></msub></mrow></mrow></mrow></math></span>'s are zero, the prediction interval is simply formed by <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-74" display="inline"><mrow><mrow><mrow><mo class="MathClass-open">[</mo><mrow><msup><mi>G</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup><mrow><mo>(</mo><mrow><msub><mover accent="true"><mi>f</mi><mo stretchy="true">^</mo></mover><mrow><mi>γ</mi><mo>∕</mo><mn>2</mn></mrow></msub><mrow><mo>(</mo><msub><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>)</mo></mrow></mrow><mo>)</mo></mrow><mo>,</mo><msup><mi>G</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup><mrow><mo>(</mo><mrow><msub><mover accent="true"><mi>f</mi><mo stretchy="true">^</mo></mover><mrow><mn>1</mn><mo>−</mo><mi>γ</mi><mo>∕</mo><mn>2</mn></mrow></msub><mrow><mo>(</mo><msub><mover accent="true"><mi>X</mi><mo stretchy="true">˜</mo></mover><mi>i</mi></msub><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mrow><mo class="MathClass-close">]</mo></mrow></mrow></mrow></math></span> and the prediction intervals will be formed exactly with the two fitted quantile curves. However, if most <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-75" display="inline"><mrow><mrow><mrow><msub><mi>S</mi><mi>i</mi></msub></mrow></mrow></mrow></math></span>'s are positive and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-76" display="inline"><mrow><mrow><mrow><msub><mi>q</mi><mi mathvariant="monospace">quantile</mi></msub></mrow></mrow></mrow></math></span> is also positive, the resulting interval will be calibrated to a wider interval, and vice versa. As a result, this way of prediction interval construction automatically adjusts for both undercoverage and overcoverage.</p></section><section id="gepi70008-sec-0050"><h3 class="pmc_sec_title">2.3. Child‐Specific Methylation‐Based Datasets</h3>
<p>We acquire several publicly available DNAm datasets from the GEO database, profiled with Illumina 27K and Illumina 450K array platforms (Table <a href="#gepi70008-tbl-0001" class="usa-link">1</a>). We exclude DNAm samples if their chronological ages are missing. These datasets consist of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-77" display="inline"><mrow><mrow><mrow><mi>n</mi><mo>=</mo><mn>728</mn></mrow></mrow></mrow></math></span> DNAm samples from healthy children with the age ranging from 1 month to 216 months. Such a large cohort of children covers the whole age period from 0 to 18 years helps to study the heterogeneous aging pattern precisely throughout childhood. In addition, utilizing these well‐established datasets also ensures a rigorous assessment of its performance, thereby reinforcing the credibility and reliability of our proposed pipeline. These datasets will be instrumental in demonstrating the pipeline designed for developing prediction intervals for epigenetic age, as discussed in Section <a href="#gepi70008-sec-0060" class="usa-link">3</a>.</p>
<section class="tw xbox font-sm" id="gepi70008-tbl-0001"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Basic information of the children's DNA methylation datasets from the GEO database. After preprocessing, the pooled data set consists of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-78" display="inline"><mrow><mrow><mrow><mi>n</mi><mo>=</mo><mn>728</mn></mrow></mrow></mrow></math></span> DNAm samples and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-79" display="inline"><mrow><mrow><mrow><mi>q</mi><mo>=</mo><mn>22</mn><mo>,</mo><mn>233</mn></mrow></mrow></mrow></math></span> CpG sites.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="left" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<thead valign="bottom"><tr valign="bottom" style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">ID</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Availability</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Methylation array</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<table class="disp-formula p" id="jats-math-80_df"><tr><td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-80" display="inline"><mrow><mrow><mrow><mi>n</mi></mrow></mrow></mrow></math></span></td></tr></table>
</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Age range (months)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Mean (SD) age</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Citation</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">1</td>
<td align="center" valign="top" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE27097" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE27097</a></td>
<td align="center" valign="top" rowspan="1" colspan="1">Illumina 27K</td>
<td align="center" valign="top" rowspan="1" colspan="1">316</td>
<td align="center" valign="top" rowspan="1" colspan="1">43.00–214.00</td>
<td align="center" valign="top" rowspan="1" colspan="1">118.70 (43.79)</td>
<td align="center" valign="top" rowspan="1" colspan="1">Alisch et al. (<a href="#gepi70008-bib-0001" class="usa-link" aria-describedby="gepi70008-bib-0001">2012</a>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2</td>
<td align="center" valign="top" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE32148" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE32148</a></td>
<td align="center" valign="top" rowspan="1" colspan="1">Illumina 450K</td>
<td align="center" valign="top" rowspan="1" colspan="1">11</td>
<td align="center" valign="top" rowspan="1" colspan="1">42.00–210.00</td>
<td align="center" valign="top" rowspan="1" colspan="1">136.36 (58.55)</td>
<td align="center" valign="top" rowspan="1" colspan="1">Harris et al. (<a href="#gepi70008-bib-0014" class="usa-link" aria-describedby="gepi70008-bib-0014">2012</a>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">3</td>
<td align="center" valign="top" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE57484" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE57484</a></td>
<td align="center" valign="top" rowspan="1" colspan="1">Illumina 27K</td>
<td align="center" valign="top" rowspan="1" colspan="1">8</td>
<td align="center" valign="top" rowspan="1" colspan="1">120.24–127.56</td>
<td align="center" valign="top" rowspan="1" colspan="1">123.11 (2.82)</td>
<td align="center" valign="top" rowspan="1" colspan="1">Voisin et al. (<a href="#gepi70008-bib-0034" class="usa-link" aria-describedby="gepi70008-bib-0034">2015</a>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">4</td>
<td align="center" valign="top" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE64495" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE64495</a></td>
<td align="center" valign="top" rowspan="1" colspan="1">Illumina 450K</td>
<td align="center" valign="top" rowspan="1" colspan="1">15</td>
<td align="center" valign="top" rowspan="1" colspan="1">27.60–129.60</td>
<td align="center" valign="top" rowspan="1" colspan="1">71.60 (31.95)</td>
<td align="center" valign="top" rowspan="1" colspan="1">Walker et al. (<a href="#gepi70008-bib-0038" class="usa-link" aria-describedby="gepi70008-bib-0038">2015</a>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">5</td>
<td align="center" valign="top" rowspan="1" colspan="1">E‐MTAB‐4187</td>
<td align="center" valign="top" rowspan="1" colspan="1">Illumina 450K</td>
<td align="center" valign="top" rowspan="1" colspan="1">81</td>
<td align="center" valign="top" rowspan="1" colspan="1">67.04–196.70</td>
<td align="center" valign="top" rowspan="1" colspan="1">143.17 (40.30)</td>
<td align="center" valign="top" rowspan="1" colspan="1">Almstrup et al. (<a href="#gepi70008-bib-0002" class="usa-link" aria-describedby="gepi70008-bib-0002">2016</a>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">6</td>
<td align="center" valign="top" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE34257" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE34257</a></td>
<td align="center" valign="top" rowspan="1" colspan="1">Illumina 27K</td>
<td align="center" valign="top" rowspan="1" colspan="1">63</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.00–12.00</td>
<td align="center" valign="top" rowspan="1" colspan="1">6.52 (2.87)</td>
<td align="center" valign="top" rowspan="1" colspan="1">Khulan et al. (<a href="#gepi70008-bib-0018" class="usa-link" aria-describedby="gepi70008-bib-0018">2012</a>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">7</td>
<td align="center" valign="top" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE36054" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE36054</a></td>
<td align="center" valign="top" rowspan="1" colspan="1">Illumina 450K</td>
<td align="center" valign="top" rowspan="1" colspan="1">121</td>
<td align="center" valign="top" rowspan="1" colspan="1">12.00–203.00</td>
<td align="center" valign="top" rowspan="1" colspan="1">59.26 (50.24)</td>
<td align="center" valign="top" rowspan="1" colspan="1">Alisch et al. (<a href="#gepi70008-bib-0001" class="usa-link" aria-describedby="gepi70008-bib-0001">2012</a>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">8</td>
<td align="center" valign="top" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE23638" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE23638</a></td>
<td align="center" valign="top" rowspan="1" colspan="1">Illumina 27K</td>
<td align="center" valign="top" rowspan="1" colspan="1">13</td>
<td align="center" valign="top" rowspan="1" colspan="1">36.00–192.00</td>
<td align="center" valign="top" rowspan="1" colspan="1">111.69 (51.70)</td>
<td align="center" valign="top" rowspan="1" colspan="1">Chen et al. (<a href="#gepi70008-bib-0009" class="usa-link" aria-describedby="gepi70008-bib-0009">2011</a>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">9</td>
<td align="center" valign="top" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE41037" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE41037</a></td>
<td align="center" valign="top" rowspan="1" colspan="1">Illumina 27K</td>
<td align="center" valign="top" rowspan="1" colspan="1">9</td>
<td align="center" valign="top" rowspan="1" colspan="1">192.00–216.00</td>
<td align="center" valign="top" rowspan="1" colspan="1">209.33 (8.72)</td>
<td align="center" valign="top" rowspan="1" colspan="1">Horvath et al. (<a href="#gepi70008-bib-0016" class="usa-link" aria-describedby="gepi70008-bib-0016">2012</a>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">10</td>
<td align="center" valign="top" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE52588" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE52588</a></td>
<td align="center" valign="top" rowspan="1" colspan="1">Illumina 450K</td>
<td align="center" valign="top" rowspan="1" colspan="1">2</td>
<td align="center" valign="top" rowspan="1" colspan="1">168.00–192.00</td>
<td align="center" valign="top" rowspan="1" colspan="1">180.00 (16.97)</td>
<td align="center" valign="top" rowspan="1" colspan="1">Bacalini et al. (<a href="#gepi70008-bib-0003" class="usa-link" aria-describedby="gepi70008-bib-0003">2015</a>)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">11</td>
<td align="center" valign="top" rowspan="1" colspan="1"><a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73103" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE73103</a></td>
<td align="center" valign="top" rowspan="1" colspan="1">Illumina 450K</td>
<td align="center" valign="top" rowspan="1" colspan="1">89</td>
<td align="center" valign="top" rowspan="1" colspan="1">168.00–216.00</td>
<td align="center" valign="top" rowspan="1" colspan="1">185.39 (9.23)</td>
<td align="center" valign="top" rowspan="1" colspan="1">Voisin et al. (<a href="#gepi70008-bib-0035" class="usa-link" aria-describedby="gepi70008-bib-0035">2015</a>)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/gepi70008-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section></section><section id="gepi70008-sec-0060"><h2 class="pmc_sec_title">3. Results</h2>
<p>In this section, we illustrate the proposed pipeline for constructing prediction intervals for epigenetic age with the children's datasets. We first introduce the basic characteristics of the DNAm datasets being used. Then we compare the prediction performances between both mean and quantile regression‐based epigenetic clocks. Lastly, we assess the validity and statistical efficiency of the prediction intervals constructed from these two types of epigenetic clocks, respectively.</p>
<section id="gepi70008-sec-0070"><h3 class="pmc_sec_title">3.1. Establishment of a Child‐Specific Epigenetic Clock Based on Quantile Regression</h3>
<p>Despite the popularity of conditional mean regression, it is sensitive to outliers and fails to capture heterogeneous relationships between epigenetic age and DNAm profile across different subpopulations. Moreover, in the applications of epigenetic age and aging‐related disease predictions, the underlying heterogeneity across subpopulations may not be fully addressed by inferring the conditional mean. As an alternative modeling approach to the conventional linear regression model, quantile regression allows the relationship to vary across quantiles of the chronological age. This helps to uncover the underlying association patterns which may be quite different between children and adults. Basically, quantile regression provides greater flexibility to identify differing relationships at different parts of the distribution of the outcome variable. It has been reported that the variability in epigenetic age differs across human lifespan (Kanherkar et al. <a href="#gepi70008-bib-0017" class="usa-link" aria-describedby="gepi70008-bib-0017">2014</a>). In children and adolescence cohort, for example, the variability in epigenetic age is found to be more drastic in mid‐childhood than in toddlers (Wu et al. <a href="#gepi70008-bib-0039" class="usa-link" aria-describedby="gepi70008-bib-0039">2019</a>). For this reason, as an analog of existing epigenetic clocks, we propose to fit a high‐dimensional quantile regression model with elastic net penalty.</p></section><section id="gepi70008-sec-0080"><h3 class="pmc_sec_title">3.2. Characteristics of the DNA Methylation Datasets (Step 1)</h3>
<p>With the children's datasets, we follow the preprocessing approach illustrated in Step 1 in Section <a href="#gepi70008-sec-0030" class="usa-link">2.1</a>. Specifically, the gold standard is defined as the mean DNAm values (beta values) from the largest single data set (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE27097" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE27097</a>). This gold standard is used to rescale the probes of all other datasets to ensure comparability, as detailed in Step 1. The age transform function <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-81" display="inline"><mrow><mrow><mrow><mi>G</mi><mrow><mo>(</mo><mi>Y</mi><mo>)</mo></mrow></mrow></mrow></mrow></math></span> is the same as Function <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-82" display="inline"><mrow><mrow><mrow><mi>F</mi></mrow></mrow></mrow></math></span> adopted by Wu et al. (<a href="#gepi70008-bib-0039" class="usa-link" aria-describedby="gepi70008-bib-0039">2019</a>), which is defined as:
</p>
<table class="disp-formula p" id="gepi70008-disp-0004"><tr><td class="formula"><math id="jats-math-83" display="block"><mrow><mrow><mrow><mi>G</mi><mrow><mo>(</mo><mi>Y</mi><mo>)</mo></mrow><mo>=</mo><mfenced close="" open="{"><mtable columnalign="left" displaystyle="true"><mtr><mtd columnalign="left"><mi>log</mi><mrow><mo>(</mo><mrow><mi>Y</mi><mo>+</mo><mn>1</mn></mrow><mo>)</mo></mrow><mo>−</mo><mi>log</mi><mrow><mo>(</mo><mrow><msub><mi>Y</mi><mi mathvariant="monospace">toddler</mi></msub><mo>+</mo><mn>1</mn></mrow><mo>)</mo></mrow><mo>,</mo></mtd><mtd columnalign="left"><mstyle><mspace width="0.1em"></mspace><mtext>if</mtext><mspace width="0.1em"></mspace></mstyle><mspace width="0.25em"></mspace><mi>Y</mi><mo>≤</mo><msub><mi>Y</mi><mi mathvariant="monospace">toddler</mi></msub><mo>,</mo></mtd></mtr><mtr><mtd columnalign="left"><mrow><mo>(</mo><mrow><mi>Y</mi><mo>−</mo><msub><mi>Y</mi><mi mathvariant="monospace">toddler</mi></msub></mrow><mo>)</mo></mrow><mo>∕</mo><mrow><mo>(</mo><mrow><msub><mi>Y</mi><mi mathvariant="monospace">toddler</mi></msub><mo>+</mo><mn>1</mn></mrow><mo>)</mo></mrow><mo>,</mo></mtd><mtd columnalign="left"><mstyle><mspace width="0.1em"></mspace><mtext>if</mtext><mspace width="0.1em"></mspace></mstyle><mspace width="0.25em"></mspace><mi>Y</mi><mo>&gt;</mo><msub><mi>Y</mi><mi mathvariant="monospace">toddler</mi></msub><mo>,</mo></mtd></mtr></mtable></mfenced></mrow></mrow></mrow></math></td></tr></table>
<p> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-84" display="inline"><mrow><mrow><mrow><msub><mi>Y</mi><mi mathvariant="monospace">toddler</mi></msub></mrow></mrow></mrow></math></span> is set to 4 years. Then the inverse‐transformed function is
</p>
<table class="disp-formula p" id="gepi70008-disp-0005"><tr><td class="formula"><math id="jats-math-85" display="block"><mrow><mrow><mrow><msup><mi>G</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup><mrow><mo>(</mo><mi>Y</mi><mo>)</mo></mrow><mo>=</mo><mfenced close="" open="{"><mtable columnalign="left" displaystyle="true"><mtr><mtd columnalign="left"><mrow><mo>(</mo><mrow><msub><mi>Y</mi><mi mathvariant="monospace">toddler</mi></msub><mo>+</mo><mn>1</mn></mrow><mo>)</mo></mrow><msup><mi>e</mi><mi>Y</mi></msup><mo>−</mo><mn>1</mn><mo>,</mo></mtd><mtd columnalign="left"><mstyle><mspace width="0.1em"></mspace><mtext>if</mtext><mspace width="0.1em"></mspace></mstyle><mspace width="0.25em"></mspace><mi>Y</mi><mo>≤</mo><mn>0</mn><mo>,</mo></mtd></mtr><mtr><mtd columnalign="left"><mrow><mo>(</mo><mrow><msub><mi>Y</mi><mi mathvariant="monospace">toddler</mi></msub><mo>+</mo><mn>1</mn></mrow><mo>)</mo></mrow><mi>Y</mi><mo>+</mo><msub><mi>Y</mi><mi mathvariant="monospace">toddler</mi></msub><mo>,</mo></mtd><mtd columnalign="left"><mstyle><mspace width="0.1em"></mspace><mtext>if</mtext><mspace width="0.1em"></mspace></mstyle><mspace width="0.25em"></mspace><mi>Y</mi><mo>&gt;</mo><mn>0</mn><mo>.</mo></mtd></mtr></mtable></mfenced></mrow></mrow></mrow></math></td></tr></table>
<p>Thus, both <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-86" display="inline"><mrow><mrow><mrow><mi>G</mi><mrow><mo>(</mo><mi>Y</mi><mo>)</mo></mrow></mrow></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-87" display="inline"><mrow><mrow><mrow><msup><mi>G</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup><mrow><mo>(</mo><mi>Y</mi><mo>)</mo></mrow></mrow></mrow></mrow></math></span> satisfy the conditions mentioned in Step 1 in Section <a href="#gepi70008-sec-0030" class="usa-link">2.1</a>: The transformation function <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-88" display="inline"><mrow><mrow><mrow><mi>G</mi><mrow><mo>(</mo><mi>Y</mi><mo>)</mo></mrow></mrow></mrow></mrow></math></span> is strictly increasing, invertible and continuous, and the inverse‐transformed values <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-89" display="inline"><mrow><mrow><mrow><msup><mi>G</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup><mrow><mo>(</mo><mi>Y</mi><mo>)</mo></mrow></mrow></mrow></mrow></math></span> is strictly positive. Additionally, the log transformation in the early stages of life, that is, toddlerhood, reflects the fact that DNAm changes more rapidly during these periods. As individuals grow older, the linear transformation in mid‐childhood and adolescence better captures the slower and more gradual changes in DNAm. This approach allows the transformation to adapt to different stages of life in a biologically meaningful way. We apply <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-90" display="inline"><mrow><mrow><mrow><mi>G</mi><mrow><mo>(</mo><mi>Y</mi><mo>)</mo></mrow></mrow></mrow></mrow></math></span> before building the regression models and use <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-91" display="inline"><mrow><mrow><mrow><msup><mi>G</mi><mrow><mo>−</mo><mn>1</mn></mrow></msup><mrow><mo>(</mo><mi>Y</mi><mo>)</mo></mrow></mrow></mrow></mrow></math></span> to transform the predicted values back into the DNAm age. After these steps, our pooled data set consists of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-92" display="inline"><mrow><mrow><mrow><mi>n</mi><mo>=</mo><mn>728</mn></mrow></mrow></mrow></math></span> subjects and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-93" display="inline"><mrow><mrow><mrow><mi>q</mi><mo>=</mo><mn>22</mn><mo>,</mo><mn>233</mn></mrow></mrow></mrow></math></span> CpG sites. Their ages range from 1 month to 216 months (0–18 years). Detailed information is summarized in Table <a href="#gepi70008-tbl-0001" class="usa-link">1</a>.</p>
<p>The entire data set is randomly split into three sub‐datasets: 50% for training set, 30% for calibration set and 20% for test set (Romano et al. <a href="#gepi70008-bib-0028" class="usa-link" aria-describedby="gepi70008-bib-0028">2019</a>). The training set will be used for training a new epigenetic clock while the test set will be used for evaluating the prediction performance of the newly trained clock, see Section <a href="#gepi70008-sec-0090" class="usa-link">3.3</a>. The holdout calibration set will be used in Section <a href="#gepi70008-sec-0100" class="usa-link">3.4</a> for uncertainty quantification and prediction interval construction.</p></section><section id="gepi70008-sec-0090"><h3 class="pmc_sec_title">3.3. Training a Quantile Regression Based Epigenetic Clock (Steps 2 to 3)</h3>
<p>We first implement a 10‐fold cross‐validation procedure to determine the number of recruited predictors <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-94" display="inline"><mrow><mrow><mrow><mi>p</mi></mrow></mrow></mrow></math></span> from the candidate set <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-95" display="inline"><mrow><mrow><mrow><mo class="MathClass-open">{</mo><mrow><mrow><mo class="MathClass-open">[</mo><mrow><mi>n</mi><mo>∕</mo><mi>log</mi><mrow><mo>(</mo><mi>n</mi><mo>)</mo></mrow></mrow><mo class="MathClass-close">]</mo></mrow><mo>,</mo><mrow><mo class="MathClass-open">[</mo><mrow><mn>3</mn><mi>n</mi><mo>∕</mo><mi>log</mi><mrow><mo>(</mo><mi>n</mi><mo>)</mo></mrow></mrow><mo class="MathClass-close">]</mo></mrow><mo>,</mo><mrow><mo class="MathClass-open">[</mo><mrow><mn>5</mn><mi>n</mi><mo>∕</mo><mi>log</mi><mrow><mo>(</mo><mi>n</mi><mo>)</mo></mrow></mrow><mo class="MathClass-close">]</mo></mrow><mo>,</mo><mi>n</mi><mo>−</mo><mn>1</mn></mrow><mo class="MathClass-close">}</mo></mrow></mrow></mrow></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-96" display="inline"><mrow><mrow><mrow><mo class="MathClass-open">[</mo><mi>x</mi><mo class="MathClass-close">]</mo></mrow></mrow></mrow></math></span> denotes the integer part of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-97" display="inline"><mrow><mrow><mrow><mi>x</mi></mrow></mrow></mrow></math></span>. This set is chosen to align with the sure screening property of SIS, as described in Theorem 1 of Fan and Lv (<a href="#gepi70008-bib-0010" class="usa-link" aria-describedby="gepi70008-bib-0010">2008</a>). The optimal number of features is selected as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-98" display="inline"><mrow><mrow><mrow><mi>p</mi><mo>=</mo><mrow><mo class="MathClass-open">[</mo><mrow><mn>3</mn><mi>n</mi><mo>∕</mo><mi>log</mi><mrow><mo>(</mo><mi>n</mi><mo>)</mo></mrow></mrow><mo class="MathClass-close">]</mo></mrow><mo>=</mo><mn>331</mn></mrow></mrow></mrow></math></span>. Following Step 2 in Section <a href="#gepi70008-sec-0030" class="usa-link">2.1</a> and according to the results summarized in Table <a href="#gepi70008-tbl-0002" class="usa-link">2</a>, we choose the SIRS method for the mean regression model and the Bcor‐SIS method for the quantile regression models for dimension reduction because they have the highest overall correlation coefficient within each of the regression models. With these selected CpGs and following Step 3 in Section <a href="#gepi70008-sec-0030" class="usa-link">2.1</a>, we then fit a high‐dimensional median regression model with elastic net penalty. As a baseline comparison, we also fit a high‐dimensional mean regression model with elastic net penalty. The mixing parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-99" display="inline"><mrow><mrow><mrow><mi>α</mi></mrow></mrow></mrow></math></span> are set at 0.15 and 0.45, and the penalty tuning parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-100" display="inline"><mrow><mrow><mrow><mi>λ</mi></mrow></mrow></mrow></math></span> are set at 0.061 and 0.057 in median (<kbd>R</kbd> function <kbd>conquer.cv.reg</kbd>) and mean (<kbd>R</kbd> function <kbd>cv.glmnet</kbd>) regression models respectively, based on 10‐fold cross‐validation of the training data.</p>
<section class="tw xbox font-sm" id="gepi70008-tbl-0002"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>After applying different screening methods for dimension reduction in the training sets, we fit various regression models and evaluate their performances by calculating the Pearson correlation coefficients between predicted values and true values. We choose the SIRS method for the mean regression and the Bcor‐SIS method for both the 0.05th and 0.95th quantile regression because they have the highest correlation coefficient in each of the regression models.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="left" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<col style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" align="center" span="1">
<thead valign="bottom"><tr valign="bottom" style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Screening methods</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Mean regression</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">0.05th quantile regression</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">0.95th quantile regression</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Median regression</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Average correlation</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">SIRS</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.92</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.88</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.90</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.90</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.89</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">DC‐SIS</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.91</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.89</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.89</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.90</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.90</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Kfilter</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.86</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.82</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.84</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.87</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.79</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">CSIS</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.91</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.88</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.89</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.90</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.90</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Bcor‐SIS</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.91</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.89</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.90</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.91</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.90</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">WLS</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.86</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.83</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.86</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.88</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.76</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/gepi70008-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>To evaluate the prediction accuracy, we apply two different training models to predict ages in the test sets and then compare their Pearson correlation coefficients between DNAm age (predicted age) and chronological age, as well as the median absolute difference between DNAm age and chronological age (median error). As shown in Figure <a href="#gepi70008-fig-0002" class="usa-link">2</a>, the predictive model based on median regression performs slightly better than the mean regression. We find that the median regression model has a correlation coefficient of 0.886 and a median error of 17 months (Figure <a href="#gepi70008-fig-0002" class="usa-link">2b</a>), while the correlation coefficient and median error of the mean regression model are 0.878 and 22 months, respectively (Figure <a href="#gepi70008-fig-0002" class="usa-link">2a</a>). This improvement is largely due to the skewness of the age distribution.</p>
<figure class="fig xbox font-sm" id="gepi70008-fig-0002"><h4 class="obj_head">Figure 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0549/11948180/c313acaf7636/GEPI-49-0-g003.jpg" loading="lazy" id="jats-graphic-3" height="340" width="702" alt="Figure 2"></p>
<div class="p text-right font-secondary"><a href="figure/gepi70008-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Scatterplots for individuals in the test sets (age unit in month). The straight diagonal lines in both plots are the diagonal references for visualizing the consistency between DNAm age and chronological age. The median regression model (black circles) closely aligns along the diagonal line, indicating a strong correlation with chronological age. Meanwhile, the 0.05th quantile regression model (blue triangles) and the 0.95th quantile regression model (red diamonds) display deviations above and below this line, respectively.</p></figcaption></figure><p>In addition to the median regression model, we also fit the 0.05th and 0.95th high‐dimensional quantile regression models to examine the lower and upper tails of the chronological age distributions. In our data example, they correspond to toddlerhood and adolescence, respectively. The mixing parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-101" display="inline"><mrow><mrow><mrow><mi>α</mi></mrow></mrow></mrow></math></span> are set at 0.31 and 0.23, and the penalty tuning parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-102" display="inline"><mrow><mrow><mrow><mi>λ</mi></mrow></mrow></mrow></math></span> are set at 0.022 and 0.049 in the 0.05th and 0.95th quantile regression models, respectively, based on a 10‐fold cross‐validation of the training data. The correspondences between the estimated DNAm and the chronological ages are also plotted in Figure <a href="#gepi70008-fig-0002" class="usa-link">2b</a>. We observe that the blue labels show an upward trend relative to the diagonal reference line. This is because the DNAm ages represented by these blue triangles are the predicted epigenetic ages with a 0.05th quantile regression model, and they represent the estimated threshold for the 5th percentile of the chronological age, conditional on their DNAm levels. Therefore, estimated ages based on DNAm are generally smaller than their true chronological age. Moreover, the deviation from the diagonal line is much smaller for those in the lower tail of the chronological age distribution, while it gradually becomes larger as we move to the upper tail. A similar deviation pattern can be observed from the red diamonds, which show the estimated threshold for the 95th percentile of the chronological age. This phenomenon reveals different variability in epigenetic age across different developmental stages: It varies much less in toddlerhood than in mid‐childhood or adolescence. These results demonstrate the benefit and flexibility of fitting high‐dimensional quantile regression models as they provide distributional‐level knowledge about the epigenetic age.</p></section><section id="gepi70008-sec-0100"><h3 class="pmc_sec_title">3.4. Constructing Prediction Intervals and Evaluating Coverage Rate (Steps 4 to 5)</h3>
<p>Following Section <a href="#gepi70008-sec-0040" class="usa-link">2.2</a>, we construct level 90% prediction intervals based on the mean regression and quantile regression models, respectively, by setting <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-103" display="inline"><mrow><mrow><mrow><mi>γ</mi><mo>=</mo><mn>0.1</mn></mrow></mrow></mrow></math></span>. Figure <a href="#gepi70008-fig-0003" class="usa-link">3a</a> shows the prediction intervals derived from the conditional mean regression model. Although they are statistically valid with a coverage rate of 91.1%, which is close to the nominal level, the width of the intervals remains almost the same for all subjects. However, it has been found that there is considerable heterogeneity in physiologic functions and the rate of biological aging among individuals of the same chronological age (Wagner et al. <a href="#gepi70008-bib-0037" class="usa-link" aria-describedby="gepi70008-bib-0037">2016</a>). For example, as has been reported in Wu et al. (<a href="#gepi70008-bib-0039" class="usa-link" aria-describedby="gepi70008-bib-0039">2019</a>), age acceleration is the greatest in mid‐childhood (5–11 years), and we expect to see wider prediction intervals than those in toddlerhood (0–4 years). Consequently, this way of constructing prediction interval overlooks various heterogeneity across different subpopulations or different developmental stages.</p>
<figure class="fig xbox font-sm" id="gepi70008-fig-0003"><h4 class="obj_head">Figure 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0549/11948180/5a580646816b/GEPI-49-0-g001.jpg" loading="lazy" id="jats-graphic-5" height="264" width="702" alt="Figure 3"></p>
<div class="p text-right font-secondary"><a href="figure/gepi70008-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Prediction intervals for individuals in the test sets (age unit in year). The <em>x</em>‐axis denotes subjects indexes, and the <em>y</em>‐axis represents the true chronological ages. Subject indexes are ordered by chronological ages, as shown in the black dots with an increasing pattern. The blue and red lines denote the upper and lower bounds of prediction intervals, respectively.</p></figcaption></figure><p>In contrast, prediction intervals derived from the quantile regression model are presented in Figure <a href="#gepi70008-fig-0003" class="usa-link">3b</a>. With a coverage rate of 89.7% that is also close to the nominal level, the width of the intervals is adapted to individuals with different chronological age. Specifically, we observe that the prediction intervals for toddler age group under 4 years old are much narrower than those of the mid‐childhood or adolescence according to Figure <a href="#gepi70008-fig-0004" class="usa-link">4</a>. Such an observation coincides with that has been reported in Wu et al. (<a href="#gepi70008-bib-0039" class="usa-link" aria-describedby="gepi70008-bib-0039">2019</a>). We further conduct <em>t</em>‐test and find that the width of intervals from conformalized quantile method are significantly narrower than those from the mean regression at the significance level <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-104" display="inline"><mrow><mrow><mrow><mi>α</mi><mo>=</mo><mn>0.05</mn></mrow></mrow></mrow></math></span>, with a <em>p</em>‐value of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-105" display="inline"><mrow><mrow><mrow><mn>4.357</mn><mo>×</mo><mn>1</mn><msup><mn>0</mn><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></mrow></mrow></math></span>. If we stratify by age groups, for subjects in their toddlerhood (<em>p</em>‐value = 0.014), mid‐childhood (<em>p</em>‐value = <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-106" display="inline"><mrow><mrow><mrow><mn>1.134</mn><mo>×</mo><mn>1</mn><msup><mn>0</mn><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></mrow></mrow></math></span>) and adolescence (<em>p</em>‐value = 0.003), the prediction intervals constructed by the conformalized mean regression are all significantly wider than those constructed by the conformalized quantile regression. Besides the width of prediction intervals, comparing the variability of the widths is also of great interest because they reveal the level of heterogeneity across different age groups. We also conduct theLevene's test to assess whether the variances of the prediction interval widths differ overall and across three age groups. Overall, results show that the variances of the prediction interval widths differ significantly between the two conformal prediction methods at the significance level <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-107" display="inline"><mrow><mrow><mrow><mi>α</mi><mo>=</mo><mn>0.05</mn></mrow></mrow></mrow></math></span>, with a <em>p</em>‐value of 0.002. Specifically, in toddlerhood (<em>p</em>‐value = 0.451), the variance difference is not statistically significant. However, in mid‐childhood (<em>p</em>‐value = <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-108" display="inline"><mrow><mrow><mrow><mn>7.036</mn><mo>×</mo><mn>1</mn><msup><mn>0</mn><mrow><mo>−</mo><mn>16</mn></mrow></msup></mrow></mrow></mrow></math></span>) and adolescence (<em>p</em>‐value = <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-109" display="inline"><mrow><mrow><mrow><mn>7.645</mn><mo>×</mo><mn>1</mn><msup><mn>0</mn><mrow><mo>−</mo><mn>21</mn></mrow></msup></mrow></mrow></mrow></math></span>), the variance differences are highly significant. These results suggest that the conformalized quantile regression method captures greater variability in the width of prediction intervals, particularly in the mid‐childhood and adolescence stages. Clearly, this approach for constructing prediction intervals provides a more accurate reflection of the heterogeneity in age acceleration across different developmental stages in children.</p>
<figure class="fig xbox font-sm" id="gepi70008-fig-0004"><h4 class="obj_head">Figure 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0549/11948180/5d44d0d303cf/GEPI-49-0-g006.jpg" loading="lazy" id="jats-graphic-7" height="418" width="609" alt="Figure 4"></p>
<div class="p text-right font-secondary"><a href="figure/gepi70008-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Boxplots of prediction intervals' width (unit in year) distribution stratified by age groups. The red, blue and green boxplots show the width distributions in toddlerhood, mid‐childhood, and adolescence, respectively. The toddler age group has much narrower prediction intervals than the other two age groups.</p></figcaption></figure><p>With both methods attaining the nominal coverage rate, we further compare their statistical efficiency in terms of prediction interval width. Apparently, efficient statistical methods can result in narrower prediction intervals, reflecting greater certainty and confidence in predictions. As shown in Figure <a href="#gepi70008-fig-0005" class="usa-link">5</a>, prediction intervals derived from the conformalized quantile regression method are narrower than those approximately constant width intervals produced by conformalized mean regression method: the top of the box on the right, corresponding to the third quartile, is slightly lower than the horizontal line on the left. Consequently, our newly proposed way of constructing epigenetic clocks and their corresponding prediction intervals is not only adaptable to heterogeneity among subpopulations but also statistically more efficient than conventional mean‐based methods.</p>
<figure class="fig xbox font-sm" id="gepi70008-fig-0005"><h4 class="obj_head">Figure 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0549/11948180/4032e5708cac/GEPI-49-0-g005.jpg" loading="lazy" id="jats-graphic-9" height="418" width="609" alt="Figure 5"></p>
<div class="p text-right font-secondary"><a href="figure/gepi70008-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Boxplots of width (unit in year) distribution of prediction intervals derived from conformalized mean regression and conformalized quantile regression methods. The red line refers to the relatively consistent width of prediction intervals derived from conformalized mean regression. The blue boxplot denotes the width distribution of prediction intervals derived from conformalized quantile regression. Clearly, conformalized quantile regression method presents narrower prediction intervals than conformalized mean regression, demonstrating its improved statistical efficiency.</p></figcaption></figure></section><section id="gepi70008-sec-0110"><h3 class="pmc_sec_title">3.5. Prediction Intervals With Adaptive Width Better Reveals Aging Pattern</h3>
<p>To further investigate the association of the width of prediction intervals with age acceleration, we integrate the average age acceleration (stratified by chronological age) and width of prediction intervals into one plot, as shown in Figure <a href="#gepi70008-fig-0006" class="usa-link">6</a>. First, we observe that the average acceleration in conformalized mean regression (Figure <a href="#gepi70008-fig-0006" class="usa-link">6a</a>) is greater than that in conformalized quantile regression (Figure <a href="#gepi70008-fig-0006" class="usa-link">6b</a>). This implies that the median regression‐based epigenetic clock fits better into the data, and the corresponding age acceleration better captures the true biological differences rather than measurement error. In addition, both mean and median based methods reveal similar aging patterns: epigenetic age acceleration is the greatest in mid‐childhood, and there is a phenomenon of age deceleration in adolescence. However, in contrast to the approximately fixed width intervals constructed via the conformalized mean regression, the width of prediction intervals produced by the quantile regression has a varying pattern that mimics the variation in age acceleration. Such prediction intervals with widths synchronized to biological age acceleration provide important scientific insights: the possible range of epigenetic age varies greatly for children under different developmental stages. Specifically, for children in their mid‐childhood and adolescence, the range of their epigenetic ages may be larger than those in early childhood. In particular, the center of the prediction interval in mid‐childhood may be slightly higher than the corresponding chronological age while it is the opposite for those in adolescence.</p>
<figure class="fig xbox font-sm" id="gepi70008-fig-0006"><h4 class="obj_head">Figure 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0549/11948180/967748ba1680/GEPI-49-0-g004.jpg" loading="lazy" id="jats-graphic-11" height="266" width="702" alt="Figure 6"></p>
<div class="p text-right font-secondary"><a href="figure/gepi70008-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Histogram of the mean value distribution of the age acceleration for individuals in the test sets (age unit in year). The <em>x</em>‐axis shows chronological ages, while the <em>y</em>‐axis represents the average value in age acceleration. The red column denotes negative value that corresponds to age deceleration, while the blue bar denotes positive average value that corresponds to age acceleration. The orange square indicates the average age acceleration, that is, predicted values minus the true chronological age, while the purple error bars represent the range of the average age acceleration, that is, upper and lower bounds of the prediction intervals minus the true chronological age.</p></figcaption></figure></section></section><section id="gepi70008-sec-0120"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>Although various epigenetic clocks have been developed to estimate the biological age of an individual based on cellular DNA methylation, their models focus on point prediction only (Horvath <a href="#gepi70008-bib-0015" class="usa-link" aria-describedby="gepi70008-bib-0015">2013</a>; Hannum et al. <a href="#gepi70008-bib-0013" class="usa-link" aria-describedby="gepi70008-bib-0013">2013</a>). However, a single point prediction of biological age is not sufficient for clinical decision‐making because it does not convey a level of confidence for predicting the age. In this paper, we extend the conformal inference framework to quantify the uncertainty in age prediction, and integrate it with our newly proposed epigenetic clocks developed with high‐dimensional quantile regression model with elastic net penalty. We employ data profiled on Illumina 27K and Illumina 450K array platforms to construct a child‐specific epigenetic clock that covers the entire period of childhood (0–18 years old). Our prediction and inference framework has the following advantages over existing age prediction models: (a) it comprehensively reflects the heterogeneity in aging patterns across different subpopulations; (b) it provides prediction intervals that will cover the ground truth with certain probability, and thus can enhance the interpretability in practice; (c) the prediction intervals are adaptive to individual DNAm profile and their varying widths uncover the underlying heterogeneity in age acceleration; and (d) the widths derived from the quantile regression models are generally narrower than those from the mean regression models, and thus are statistically more efficient.</p>
<p>Lifespan dynamics in epigenetic biomarkers are rapidly evolving, with significant implications for aging research, personalized medicine, and public health. Birth to late adolescence, for example, is known to be a tremendously dynamic period of development and growth (McEwen et al. <a href="#gepi70008-bib-0026" class="usa-link" aria-describedby="gepi70008-bib-0026">2020</a>). It is critical to build biological age prediction models that uncover the underlying population heterogeneity. In our study, we adopt the SIS method to select relevant predictors first for age prediction in our paper. As an alternative, we also explore variance filtering, which selects features based solely on their variance. However, when we apply the same number of features selected by variance filtering in the mean regression model, its prediction performance is worse than the SIS method, with a correlation of 0.842 and MAE of 24 months. While mean regression has been widely used due to its simplicity and ease of interpretation, it focuses on predictions about the mean outcome. In contrast, quantile regression provides a more detailed picture by estimating the effects of DNAm across the entire distribution of the chronological age, making it especially useful when understanding the impacts on different developmental stages across lifespan is important. Childhood and adolescence are unique periods of rapid change that is unlikely to mimic adulthood in methylome dynamics (Barker et al. <a href="#gepi70008-bib-0004" class="usa-link" aria-describedby="gepi70008-bib-0004">2018</a>; Cecil et al. <a href="#gepi70008-bib-0008" class="usa-link" aria-describedby="gepi70008-bib-0008">2023</a>). Given the differences in the pace of developmental and age‐related changes across the life course, developing a unifying prediction model that quantifies the effects of DNAm across different chronological ages will provide a more comprehensive understanding of this dynamic relationship. Our proposed method, when applied to children‐specific DNAm datasets, uncovers a clear heterogeneous pattern in different stages of childhood: age acceleration in mid‐childhood is much greater than that in toddlerhood, and there is a trend of decceleration in adolescence.</p>
<p>The width of the interval reflects the level of uncertainty associated with the age prediction. Wider intervals indicate higher uncertainty while narrower intervals suggest more confidence in the prediction. This uncertainty can be influenced by various factors such as the inherent noise in the data, the model predictive performance, and the distribution of the data points. According to Figure <a href="#gepi70008-fig-0002" class="usa-link">2</a>, we observe larger deviations in data points from mid‐childhood and adolescence in comparison to those from toddlerhood. This may explain why the resulting intervals in Figure <a href="#gepi70008-fig-0004" class="usa-link">4</a> are wider in those two groups to accommodate the increased uncertainty. Such a high variability uncovers the dynamics of DNA methylation levels during mid‐childhood, as this period involves significant biological and developmental changes that can be influenced by environmental exposures, genetic factors and lifestyle (Levine et al. <a href="#gepi70008-bib-0021" class="usa-link" aria-describedby="gepi70008-bib-0021">2018</a>; Teschendorff and Relton <a href="#gepi70008-bib-0032" class="usa-link" aria-describedby="gepi70008-bib-0032">2018</a>). While a prediction interval with a width of 10 years seems to be less informative in terms of age prediction, it offers insights on a possible range of DNAm ages in a specific developmental stage. In general, epigenetic age acceleration has been used as a measure of biological aging rate and has been linked to various clinical traits such as physical capability and cognitive functioning (Faul et al. <a href="#gepi70008-bib-0011" class="usa-link" aria-describedby="gepi70008-bib-0011">2023</a>). An age acceleration of 8 years, for example, should be interpreted differently for a person in mid‐childhood versus in toddlerhood. Because for the former, such a large deviation is still within its 90% confidence range. From this perspective, our newly proposed pipeline for constructing prediction intervals provides a normal range in which the observed difference between DNAm age and chronological age may arise from dynamics in that specific developmental stage, rather than a systematic indicator of healthy status.</p>
<p>Although our proposed quantile regression‐based inference framework can be applied to other types of continuous outcomes such as risk of disease or time to death, we currently do not have data on these outcomes to explore whether there will be new insights when the outcome of interest is not chronological age. For example, using quantile regression, we can estimate how DNAm affect different quantiles of the cardiovascular disease risk distribution. The prediction intervals may have varying lengths for those in lower, median and higher risk groups. In conclusion, constructing prediction intervals based on quantile regression provides a nuanced understanding of how DNAm levels and other demographic features influence outcomes of interest across the entire distribution. By leveraging our newly proposed framework, researchers can gain deeper insights into the dynamics of disease risk and develop interventions that are tailored to the specific needs of different risk groups.</p></section><section id="gepi70008-sec-0130"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="gepi70008-sec-0150"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The data that support the findings of this study are openly available in NCBI GEO (Gene Expression Omnibus) database at <a href="https://www.ncbi.nlm.nih.gov/geo/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/</a>.</p></section><section id="gepi70008-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="gepi70008-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="gepi70008-bib-0001">
<cite>

Alisch, R. S.
, Barwick B. G., Chopra P., et al. 2012. “Age‐Associated DNA Methylation in Pediatric Populations.” Genome Research
22, no. 4: 623–632. 10.1101/gr.125187.111.
</cite> [<a href="https://doi.org/10.1101/gr.125187.111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3317145/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22300631/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=Age%E2%80%90Associated%20DNA%20Methylation%20in%20Pediatric%20Populations&amp;author=R.%20S.%20Alisch&amp;author=B.%20G.%20Barwick&amp;author=P.%20Chopra&amp;volume=22&amp;issue=4&amp;publication_year=2012&amp;pages=623-632&amp;pmid=22300631&amp;doi=10.1101/gr.125187.111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0002">
<cite>

Almstrup, K.
, Lindhardt Johansen M., Busch A. S., et al. 2016. “Pubertal Development in Healthy Children Is Mirrored by DNA Methylation Patterns in Peripheral Blood.” Scientific Reports
6, no. 1: 28657. 10.1038/srep28657.
</cite> [<a href="https://doi.org/10.1038/srep28657" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4923870/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27349168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Pubertal%20Development%20in%20Healthy%20Children%20Is%20Mirrored%20by%20DNA%20Methylation%20Patterns%20in%20Peripheral%20Blood&amp;author=K.%20Almstrup&amp;author=M.%20Lindhardt%20Johansen&amp;author=A.%20S.%20Busch&amp;volume=6&amp;issue=1&amp;publication_year=2016&amp;pages=28657&amp;pmid=27349168&amp;doi=10.1038/srep28657&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0003">
<cite>

Bacalini, M. G.
, Gentilini D., Boattini A., et al. 2015. “Identification of a DNA Methylation Signature in Blood Cells From Persons With Down Syndrome.” Aging (Albany NY)
7, no. 2: 82. 10.18632/aging.v7i2.
</cite> [<a href="https://doi.org/10.18632/aging.v7i2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4359691/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25701644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aging%20(Albany%20NY)&amp;title=Identification%20of%20a%20DNA%20Methylation%20Signature%20in%20Blood%20Cells%20From%20Persons%20With%20Down%20Syndrome&amp;author=M.%20G.%20Bacalini&amp;author=D.%20Gentilini&amp;author=A.%20Boattini&amp;volume=7&amp;issue=2&amp;publication_year=2015&amp;pages=82&amp;pmid=25701644&amp;doi=10.18632/aging.v7i2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0004">
<cite>

Barker, E. D.
, Walton E., and Cecil C. A.. 2018. “Annual Research Review: DNA Methylation as a Mediator in the Association Between Risk Exposure and Child and Adolescent Psychopathology.” Journal of Child Psychology and Psychiatry
59, no. 4: 303–322. 10.1111/jcpp.2018.59.issue-4.
</cite> [<a href="https://doi.org/10.1111/jcpp.2018.59.issue-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28736860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Child%20Psychology%20and%20Psychiatry&amp;title=Annual%20Research%20Review:%20DNA%20Methylation%20as%20a%20Mediator%20in%20the%20Association%20Between%20Risk%20Exposure%20and%20Child%20and%20Adolescent%20Psychopathology&amp;author=E.%20D.%20Barker&amp;author=E.%20Walton&amp;author=C.%20A.%20Cecil&amp;volume=59&amp;issue=4&amp;publication_year=2018&amp;pages=303-322&amp;pmid=28736860&amp;doi=10.1111/jcpp.2018.59.issue-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0005">
<cite>

Barut, E.
, Fan J., and Verhasselt A.. 2016. “Conditional Sure Independence Screening.” Journal of the American Statistical Association
111, no. 515: 1266–1277. 10.1080/01621459.2015.1092974.
</cite> [<a href="https://doi.org/10.1080/01621459.2015.1092974" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5367860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28360436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Statistical%20Association&amp;title=Conditional%20Sure%20Independence%20Screening&amp;author=E.%20Barut&amp;author=J.%20Fan&amp;author=A.%20Verhasselt&amp;volume=111&amp;issue=515&amp;publication_year=2016&amp;pages=1266-1277&amp;pmid=28360436&amp;doi=10.1080/01621459.2015.1092974&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0006">
<cite>

Belloni, A.
, Chen D., Chernozhukov V., and Hansen C.. 2012. “Sparse Models and Methods for Optimal Instruments With an Application to Eminent Domain.” Econometrica
80, no. 6: 2369–2429.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Econometrica&amp;title=Sparse%20Models%20and%20Methods%20for%20Optimal%20Instruments%20With%20an%20Application%20to%20Eminent%20Domain&amp;author=A.%20Belloni&amp;author=D.%20Chen&amp;author=V.%20Chernozhukov&amp;author=C.%20Hansen&amp;volume=80&amp;issue=6&amp;publication_year=2012&amp;pages=2369-2429&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0007">
<cite>

Berk, R.
, Brown L., Buja A., Zhang K., and Zhao L.. 2013. “Valid Post‐Selection Inference.” Annals of Statistics
41, no. 2: 802–837. 10.1214/12-AOS1077.</cite> [<a href="https://doi.org/10.1214/12-AOS1077" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Statistics&amp;title=Valid%20Post%E2%80%90Selection%20Inference&amp;author=R.%20Berk&amp;author=L.%20Brown&amp;author=A.%20Buja&amp;author=K.%20Zhang&amp;author=L.%20Zhao&amp;volume=41&amp;issue=2&amp;publication_year=2013&amp;pages=802-837&amp;doi=10.1214/12-AOS1077&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0008">
<cite>

Cecil, C. A.
, Neumann A., and Walton E.. 2023. “Epigenetics Applied to Child and Adolescent Mental Health: Progress, Challenges and Opportunities.” JCPP Advances
3, no. 1: e12133.
</cite> [<a href="https://doi.org/10.1002/jcv2.12133" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7614304/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36910008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JCPP%20Advances&amp;title=Epigenetics%20Applied%20to%20Child%20and%20Adolescent%20Mental%20Health:%20Progress,%20Challenges%20and%20Opportunities&amp;author=C.%20A.%20Cecil&amp;author=A.%20Neumann&amp;author=E.%20Walton&amp;volume=3&amp;issue=1&amp;publication_year=2023&amp;pages=e12133&amp;pmid=36910008&amp;doi=10.1002/jcv2.12133&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0009">
<cite>

Chen, Y.‐A.
, Choufani S., Ferreira J. C., Grafodatskaya D., Butcher D. T., and Weksberg R.. 2011. “Sequence Overlap Between Autosomal and Sex‐Linked Probes on the Illumina HumanMethylation27 Microarray.” Genomics
97, no. 4: 214–222.
</cite> [<a href="https://doi.org/10.1016/j.ygeno.2010.12.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21211562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genomics&amp;title=Sequence%20Overlap%20Between%20Autosomal%20and%20Sex%E2%80%90Linked%20Probes%20on%20the%20Illumina%20HumanMethylation27%20Microarray&amp;author=Y.%E2%80%90A.%20Chen&amp;author=S.%20Choufani&amp;author=J.%20C.%20Ferreira&amp;author=D.%20Grafodatskaya&amp;author=D.%20T.%20Butcher&amp;volume=97&amp;issue=4&amp;publication_year=2011&amp;pages=214-222&amp;pmid=21211562&amp;doi=10.1016/j.ygeno.2010.12.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0010">
<cite>

Fan, J.
, and Lv J.. 2008. “Sure Independence Screening for Ultrahigh Dimensional Feature Space.” Journal of the Royal Statistical Society Series B: Statistical Methodology
70, no. 5: 849–911.</cite> [<a href="https://doi.org/10.1111/j.1467-9868.2008.00674.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2709408/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19603084/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Royal%20Statistical%20Society%20Series%20B:%20Statistical%20Methodology&amp;title=Sure%20Independence%20Screening%20for%20Ultrahigh%20Dimensional%20Feature%20Space&amp;author=J.%20Fan&amp;author=J.%20Lv&amp;volume=70&amp;issue=5&amp;publication_year=2008&amp;pages=849-911&amp;pmid=19603084&amp;doi=10.1111/j.1467-9868.2008.00674.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0011">
<cite>

Faul, J. D.
, Kim J. K., Levine M. E., Thyagarajan B., Weir D. R., and Crimmins E. M.. 2023. “Epigenetic‐Based Age Acceleration in a Representative Sample of Older Americans: Associations With Aging‐Related Morbidity and Mortality.” Proceedings of the National Academy of Sciences
120, no. 9: e2215840120.</cite> [<a href="https://doi.org/10.1073/pnas.2215840120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9992763/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36802439/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Epigenetic%E2%80%90Based%20Age%20Acceleration%20in%20a%20Representative%20Sample%20of%20Older%20Americans:%20Associations%20With%20Aging%E2%80%90Related%20Morbidity%20and%20Mortality&amp;author=J.%20D.%20Faul&amp;author=J.%20K.%20Kim&amp;author=M.%20E.%20Levine&amp;author=B.%20Thyagarajan&amp;author=D.%20R.%20Weir&amp;volume=120&amp;issue=9&amp;publication_year=2023&amp;pages=e2215840120&amp;pmid=36802439&amp;doi=10.1073/pnas.2215840120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0012"><cite>

Fei, Z.
, and Li Y.. 2024. “ U‐Learning for Prediction Inference via Combinatory Multi‐Subsampling: With Applications to LASSO and Neural Networks.” arXiv preprint arXiv:2407.15301.</cite></li>
<li id="gepi70008-bib-0013">
<cite>

Hannum, G.
, Guinney J., Zhao L., et al. 2013. “Genome‐Wide Methylation Profiles Reveal Quantitative Views of Human Aging Rates.” Molecular Cell
49, no. 2: 359–367. 10.1016/j.molcel.2012.10.016.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2012.10.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3780611/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23177740/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cell&amp;title=Genome%E2%80%90Wide%20Methylation%20Profiles%20Reveal%20Quantitative%20Views%20of%20Human%20Aging%20Rates&amp;author=G.%20Hannum&amp;author=J.%20Guinney&amp;author=L.%20Zhao&amp;volume=49&amp;issue=2&amp;publication_year=2013&amp;pages=359-367&amp;pmid=23177740&amp;doi=10.1016/j.molcel.2012.10.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0014">
<cite>

Harris, A. R.
, Nagy‐Szakal D., Pedersen N., et al. 2012. “Genome‐Wide Peripheral Blood Leukocyte DNA Methylation Microarrays Identified a Single Association With Inflammatory Bowel Diseases.” Inflammatory Bowel Diseases
18, no. 12: 2334–2341. 10.1002/ibd.22956.
</cite> [<a href="https://doi.org/10.1002/ibd.22956" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3812910/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22467598/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Inflammatory%20Bowel%20Diseases&amp;title=Genome%E2%80%90Wide%20Peripheral%20Blood%20Leukocyte%20DNA%20Methylation%20Microarrays%20Identified%20a%20Single%20Association%20With%20Inflammatory%20Bowel%20Diseases&amp;author=A.%20R.%20Harris&amp;author=D.%20Nagy%E2%80%90Szakal&amp;author=N.%20Pedersen&amp;volume=18&amp;issue=12&amp;publication_year=2012&amp;pages=2334-2341&amp;pmid=22467598&amp;doi=10.1002/ibd.22956&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0015">
<cite>

Horvath, S.

2013. “DNA Methylation Age of Human Tissues and Cell Types.” Genome Biology
14, no. 10: 1–20. 10.1186/gb-2013-14-10-r115.</cite> [<a href="https://doi.org/10.1186/gb-2013-14-10-r115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4015143/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24138928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=DNA%20Methylation%20Age%20of%20Human%20Tissues%20and%20Cell%20Types&amp;author=S.%20Horvath&amp;volume=14&amp;issue=10&amp;publication_year=2013&amp;pages=1-20&amp;pmid=24138928&amp;doi=10.1186/gb-2013-14-10-r115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0016">
<cite>

Horvath, S.
, Zhang Y., Langfelder P., et al. 2012. “Aging Effects on DNA Methylation Modules in Human Brain and Blood Tissue.” Genome Biology
13, no. 10: 1–18. 10.1186/gb-2012-13-10-r97.</cite> [<a href="https://doi.org/10.1186/gb-2012-13-10-r97" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4053733/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23034122/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Aging%20Effects%20on%20DNA%20Methylation%20Modules%20in%20Human%20Brain%20and%20Blood%20Tissue&amp;author=S.%20Horvath&amp;author=Y.%20Zhang&amp;author=P.%20Langfelder&amp;volume=13&amp;issue=10&amp;publication_year=2012&amp;pages=1-18&amp;pmid=23034122&amp;doi=10.1186/gb-2012-13-10-r97&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0017">
<cite>

Kanherkar, R. R.
, Bhatia‐Dey N., and Csoka A. B.. 2014. “Epigenetics Across the Human Lifespan.” Frontiers in Cell and Developmental Biology
2, no. 37: 49. 10.3389/fcell.2014.00049.
</cite> [<a href="https://doi.org/10.3389/fcell.2014.00049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4207041/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25364756/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Cell%20and%20Developmental%20Biology&amp;title=Epigenetics%20Across%20the%20Human%20Lifespan&amp;author=R.%20R.%20Kanherkar&amp;author=N.%20Bhatia%E2%80%90Dey&amp;author=A.%20B.%20Csoka&amp;volume=2&amp;issue=37&amp;publication_year=2014&amp;pages=49&amp;pmid=25364756&amp;doi=10.3389/fcell.2014.00049&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0018">
<cite>

Khulan, B.
, Cooper W. N., Skinner B. M., et al. 2012. “Periconceptional Maternal Micronutrient Supplementation Is Associated With Widespread Gender Related Changes in the Epigenome: A Study of a Unique Resource in the Gambia.” Human Molecular Genetics
21, no. 9: 2086–2101. 10.1093/hmg/dds026.
</cite> [<a href="https://doi.org/10.1093/hmg/dds026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22307237/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Human%20Molecular%20Genetics&amp;title=Periconceptional%20Maternal%20Micronutrient%20Supplementation%20Is%20Associated%20With%20Widespread%20Gender%20Related%20Changes%20in%20the%20Epigenome:%20A%20Study%20of%20a%20Unique%20Resource%20in%20the%20Gambia&amp;author=B.%20Khulan&amp;author=W.%20N.%20Cooper&amp;author=B.%20M.%20Skinner&amp;volume=21&amp;issue=9&amp;publication_year=2012&amp;pages=2086-2101&amp;pmid=22307237&amp;doi=10.1093/hmg/dds026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0019">
<cite>

Lei, J.
, G'Sell M., Rinaldo A., Tibshirani R. J., and Wasserman L.. 2018. “Distribution‐Free Predictive Inference for Regression.” Journal of the American Statistical Association
113, no. 523: 1094–1111. 10.1080/01621459.2017.1307116.</cite> [<a href="https://doi.org/10.1080/01621459.2017.1307116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Statistical%20Association&amp;title=Distribution%E2%80%90Free%20Predictive%20Inference%20for%20Regression&amp;author=J.%20Lei&amp;author=M.%20G'Sell&amp;author=A.%20Rinaldo&amp;author=R.%20J.%20Tibshirani&amp;author=L.%20Wasserman&amp;volume=113&amp;issue=523&amp;publication_year=2018&amp;pages=1094-1111&amp;doi=10.1080/01621459.2017.1307116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0020">
<cite>

Levine, M. E.
, Lu A. T., Bennett D. A., and Horvath S.. 2015. “Epigenetic Age of the Pre‐Frontal Cortex Is Associated With Neuritic Plaques, Amyloid Load, and Alzheimer's Disease Related Cognitive Functioning.” Aging (Albany NY)
7, no. 12: 1198. 10.18632/aging.v7i12.
</cite> [<a href="https://doi.org/10.18632/aging.v7i12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4712342/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26684672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aging%20(Albany%20NY)&amp;title=Epigenetic%20Age%20of%20the%20Pre%E2%80%90Frontal%20Cortex%20Is%20Associated%20With%20Neuritic%20Plaques,%20Amyloid%20Load,%20and%20Alzheimer's%20Disease%20Related%20Cognitive%20Functioning&amp;author=M.%20E.%20Levine&amp;author=A.%20T.%20Lu&amp;author=D.%20A.%20Bennett&amp;author=S.%20Horvath&amp;volume=7&amp;issue=12&amp;publication_year=2015&amp;pages=1198&amp;pmid=26684672&amp;doi=10.18632/aging.v7i12&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0021">
<cite>

Levine, M. E.
, Lu A. T., Quach A., et al. 2018. “An Epigenetic Biomarker of Aging for Lifespan and Healthspan.” Aging (Albany NY)
10, no. 4: 573. 10.18632/aging.v10i4.
</cite> [<a href="https://doi.org/10.18632/aging.v10i4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5940111/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29676998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aging%20(Albany%20NY)&amp;title=An%20Epigenetic%20Biomarker%20of%20Aging%20for%20Lifespan%20and%20Healthspan&amp;author=M.%20E.%20Levine&amp;author=A.%20T.%20Lu&amp;author=A.%20Quach&amp;volume=10&amp;issue=4&amp;publication_year=2018&amp;pages=573&amp;pmid=29676998&amp;doi=10.18632/aging.v10i4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0022">
<cite>

Li, R.
, Zhong W., and Zhu L.. 2012. “Feature Screening via Distance Correlation Learning.” Journal of the American Statistical Association
107, no. 499: 1129–1139. 10.1080/01621459.2012.695654.
</cite> [<a href="https://doi.org/10.1080/01621459.2012.695654" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4170057/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25249709/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Statistical%20Association&amp;title=Feature%20Screening%20via%20Distance%20Correlation%20Learning&amp;author=R.%20Li&amp;author=W.%20Zhong&amp;author=L.%20Zhu&amp;volume=107&amp;issue=499&amp;publication_year=2012&amp;pages=1129-1139&amp;pmid=25249709&amp;doi=10.1080/01621459.2012.695654&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0023">
<cite>

Lu, A. T.
, Quach A., Wilson J. G., et al. 2019. “DNA Methylation Grimage Strongly Predicts Lifespan and Healthspan.” Aging (Albany NY)
11, no. 2: 303. 10.18632/aging.v11i2.
</cite> [<a href="https://doi.org/10.18632/aging.v11i2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6366976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30669119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aging%20(Albany%20NY)&amp;title=DNA%20Methylation%20Grimage%20Strongly%20Predicts%20Lifespan%20and%20Healthspan&amp;author=A.%20T.%20Lu&amp;author=A.%20Quach&amp;author=J.%20G.%20Wilson&amp;volume=11&amp;issue=2&amp;publication_year=2019&amp;pages=303&amp;pmid=30669119&amp;doi=10.18632/aging.v11i2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0024">
<cite>

Mai, Q.
, and Zou H.. 2015. “The Fused Kolmogorov Filter: A Nonparametric Model‐Free Screening Method.” Annals of Statistics
43, no. 4: 1471–1497. 10.1214/14-AOS1303.</cite> [<a href="https://doi.org/10.1214/14-AOS1303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Statistics&amp;title=The%20Fused%20Kolmogorov%20Filter:%20A%20Nonparametric%20Model%E2%80%90Free%20Screening%20Method&amp;author=Q.%20Mai&amp;author=H.%20Zou&amp;volume=43&amp;issue=4&amp;publication_year=2015&amp;pages=1471-1497&amp;doi=10.1214/14-AOS1303&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0025">
<cite>

Marioni, R. E.
, Shah S., McRae A. F., et al. 2015. “The Epigenetic Clock Is Correlated With Physical and Cognitive Fitness in the Lothian Birth Cohort 1936.” International Journal of Epidemiology
44, no. 4: 1388–1396.
</cite> [<a href="https://doi.org/10.1093/ije/dyu277" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4588858/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25617346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Epidemiology&amp;title=The%20Epigenetic%20Clock%20Is%20Correlated%20With%20Physical%20and%20Cognitive%20Fitness%20in%20the%20Lothian%20Birth%20Cohort%201936&amp;author=R.%20E.%20Marioni&amp;author=S.%20Shah&amp;author=A.%20F.%20McRae&amp;volume=44&amp;issue=4&amp;publication_year=2015&amp;pages=1388-1396&amp;pmid=25617346&amp;doi=10.1093/ije/dyu277&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0026">
<cite>

McEwen, L. M.
, O'Donnell K. J., McGill M. G., et al. 2020. “The PedBE Clock Accurately Estimates DNA Methylation Age in Pediatric Buccal Cells.” Proceedings of the National Academy of Sciences
117, no. 38: 23329–23335.</cite> [<a href="https://doi.org/10.1073/pnas.1820843116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7519312/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31611402/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=The%20PedBE%20Clock%20Accurately%20Estimates%20DNA%20Methylation%20Age%20in%20Pediatric%20Buccal%20Cells&amp;author=L.%20M.%20McEwen&amp;author=K.%20J.%20O'Donnell&amp;author=M.%20G.%20McGill&amp;volume=117&amp;issue=38&amp;publication_year=2020&amp;pages=23329-23335&amp;pmid=31611402&amp;doi=10.1073/pnas.1820843116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0027">
<cite>

Pan, W.
, Wang X., Xiao W., and Zhu H.. 2019. “A Generic Sure Independence Screening Procedure.” Journal of the American Statistical Association
114, no. 526: 928–937. 10.1080/01621459.2018.1462709.
</cite> [<a href="https://doi.org/10.1080/01621459.2018.1462709" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6831100/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31692981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Statistical%20Association&amp;title=A%20Generic%20Sure%20Independence%20Screening%20Procedure&amp;author=W.%20Pan&amp;author=X.%20Wang&amp;author=W.%20Xiao&amp;author=H.%20Zhu&amp;volume=114&amp;issue=526&amp;publication_year=2019&amp;pages=928-937&amp;pmid=31692981&amp;doi=10.1080/01621459.2018.1462709&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0028">
<cite>

Romano, Y.
, Patterson E., and Candes E.. 2019. “Conformalized Quantile Regression.” Advances in Neural Information Processing Systems
32. <a href="https://proceedings.neurips.cc/paper_files/paper/2019/file/5103c3584b063c431bd1268e9b5e76fb-Paper.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://proceedings.neurips.cc/paper_files/paper/2019/file/5103c3584b063c431bd1268e9b5e76fb-Paper.pdf</a>.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Advances%20in%20Neural%20Information%20Processing%20Systems&amp;title=Conformalized%20Quantile%20Regression&amp;author=Y.%20Romano&amp;author=E.%20Patterson&amp;author=E.%20Candes&amp;volume=32&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0029">
<cite>

Sibbett, R. A.
, Altschul D. M., Marioni R. E., Deary I. J., Starr J. M., and Russ T. C.. 2020. “DNA Methylation‐Based Measures of Accelerated Biological Ageing and the Risk of Dementia in the Oldest‐Old: A Study of the Lothian Birth Cohort 1921.” BMC Psychiatry
20, no. 1: 1–15. 10.1186/s12888-020-2469-9.
</cite> [<a href="https://doi.org/10.1186/s12888-020-2469-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7048023/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32111184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Psychiatry&amp;title=DNA%20Methylation%E2%80%90Based%20Measures%20of%20Accelerated%20Biological%20Ageing%20and%20the%20Risk%20of%20Dementia%20in%20the%20Oldest%E2%80%90Old:%20A%20Study%20of%20the%20Lothian%20Birth%20Cohort%201921&amp;author=R.%20A.%20Sibbett&amp;author=D.%20M.%20Altschul&amp;author=R.%20E.%20Marioni&amp;author=I.%20J.%20Deary&amp;author=J.%20M.%20Starr&amp;volume=20&amp;issue=1&amp;publication_year=2020&amp;pages=1-15&amp;pmid=32111184&amp;doi=10.1186/s12888-020-2469-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0030">
<cite>

Tan, K. M.
, Wang L., and Zhou W. X.. 2022. “High‐Dimensional Quantile Regression: Convolution Smoothing and Concave Regularization.” Journal of the Royal Statistical Society Series B: Statistical Methodology
84, no. 1: 205–233.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Royal%20Statistical%20Society%20Series%20B:%20Statistical%20Methodology&amp;title=High%E2%80%90Dimensional%20Quantile%20Regression:%20Convolution%20Smoothing%20and%20Concave%20Regularization&amp;author=K.%20M.%20Tan&amp;author=L.%20Wang&amp;author=W.%20X.%20Zhou&amp;volume=84&amp;issue=1&amp;publication_year=2022&amp;pages=205-233&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0031">
<cite>

Teschendorff, A. E.
, Marabita F., Lechner M., et al. 2013. “A Beta‐Mixture Quantile Normalization Method for Correcting Probe Design Bias in Illumina Infinium 450 K DNA Methylation Data.” Bioinformatics
29, no. 2: 189–196. 10.1093/bioinformatics/bts680.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts680" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3546795/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23175756/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20Beta%E2%80%90Mixture%20Quantile%20Normalization%20Method%20for%20Correcting%20Probe%20Design%20Bias%20in%20Illumina%20Infinium%20450%20K%20DNA%20Methylation%20Data&amp;author=A.%20E.%20Teschendorff&amp;author=F.%20Marabita&amp;author=M.%20Lechner&amp;volume=29&amp;issue=2&amp;publication_year=2013&amp;pages=189-196&amp;pmid=23175756&amp;doi=10.1093/bioinformatics/bts680&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0032">
<cite>

Teschendorff, A. E.
, and Relton C. L.. 2018. “Statistical and Integrative System‐Level Analysis of DNA Methylation Data.” Nature Reviews Genetics
19, no. 3: 129–147. 10.1038/nrg.2017.86.</cite> [<a href="https://doi.org/10.1038/nrg.2017.86" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29129922/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Genetics&amp;title=Statistical%20and%20Integrative%20System%E2%80%90Level%20Analysis%20of%20DNA%20Methylation%20Data&amp;author=A.%20E.%20Teschendorff&amp;author=C.%20L.%20Relton&amp;volume=19&amp;issue=3&amp;publication_year=2018&amp;pages=129-147&amp;pmid=29129922&amp;doi=10.1038/nrg.2017.86&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0033">
<cite>

Tibshirani, R. J.
, Taylor J., Lockhart R., and Tibshirani R.. 2016. “Exact Post‐Selection Inference for Sequential Regression Procedures.” Journal of the American Statistical Association
111, no. 514: 600–620. 10.1080/01621459.2015.1108848.</cite> [<a href="https://doi.org/10.1080/01621459.2015.1108848" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Statistical%20Association&amp;title=Exact%20Post%E2%80%90Selection%20Inference%20for%20Sequential%20Regression%20Procedures&amp;author=R.%20J.%20Tibshirani&amp;author=J.%20Taylor&amp;author=R.%20Lockhart&amp;author=R.%20Tibshirani&amp;volume=111&amp;issue=514&amp;publication_year=2016&amp;pages=600-620&amp;doi=10.1080/01621459.2015.1108848&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0034">
<cite>

Voisin, S.
, Almén M. S., Moschonis G., Chrousos G. P., Manios Y., and Schiöth H. B.. 2015. “Dietary Fat Quality Impacts Genome‐Wide DNA Methylation Patterns in a Cross‐Sectional Study of Greek Preadolescents.” European Journal of Human Genetics
23, no. 5: 654–662. 10.1038/ejhg.2014.139.
</cite> [<a href="https://doi.org/10.1038/ejhg.2014.139" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4402618/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25074463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Human%20Genetics&amp;title=Dietary%20Fat%20Quality%20Impacts%20Genome%E2%80%90Wide%20DNA%20Methylation%20Patterns%20in%20a%20Cross%E2%80%90Sectional%20Study%20of%20Greek%20Preadolescents&amp;author=S.%20Voisin&amp;author=M.%20S.%20Alm%C3%A9n&amp;author=G.%20Moschonis&amp;author=G.%20P.%20Chrousos&amp;author=Y.%20Manios&amp;volume=23&amp;issue=5&amp;publication_year=2015&amp;pages=654-662&amp;pmid=25074463&amp;doi=10.1038/ejhg.2014.139&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0035">
<cite>

Voisin, S.
, Almén M. S., Zheleznyakova G. Y., et al. 2015. “Many Obesity‐Associated SNPs Strongly Associate With DNA Methylation Changes at Proximal Promoters and Enhancers.” Genome medicine
7, no. 1: 1–16. 10.1186/s13073-015-0225-4.
</cite> [<a href="https://doi.org/10.1186/s13073-015-0225-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4599317/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26449484/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20medicine&amp;title=Many%20Obesity%E2%80%90Associated%20SNPs%20Strongly%20Associate%20With%20DNA%20Methylation%20Changes%20at%20Proximal%20Promoters%20and%20Enhancers&amp;author=S.%20Voisin&amp;author=M.%20S.%20Alm%C3%A9n&amp;author=G.%20Y.%20Zheleznyakova&amp;volume=7&amp;issue=1&amp;publication_year=2015&amp;pages=1-16&amp;pmid=26449484&amp;doi=10.1186/s13073-015-0225-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0036">
<cite>

Vovk, V.
, Gammerman A., and Shafer G.. 2005. Algorithmic Learning in a Random World. Vol. 29. Springer.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Algorithmic%20Learning%20in%20a%20Random%20World&amp;author=V.%20Vovk&amp;author=A.%20Gammerman&amp;author=G.%20Shafer&amp;publication_year=2005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0037">
<cite>

Wagner, K. H.
, Cameron‐Smith D., Wessner B., and Franzke B.. 2016. “Biomarkers of Aging: From Function to Molecular Biology.” Nutrients
8, no. 6: 338. 10.3390/nu8060338.
</cite> [<a href="https://doi.org/10.3390/nu8060338" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4924179/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27271660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nutrients&amp;title=Biomarkers%20of%20Aging:%20From%20Function%20to%20Molecular%20Biology&amp;author=K.%20H.%20Wagner&amp;author=D.%20Cameron%E2%80%90Smith&amp;author=B.%20Wessner&amp;author=B.%20Franzke&amp;volume=8&amp;issue=6&amp;publication_year=2016&amp;pages=338&amp;pmid=27271660&amp;doi=10.3390/nu8060338&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0038">
<cite>

Walker, R. F.
, Liu J. S., Peters B. A., et al. 2015. “Epigenetic Age Analysis of Children Who Seem to Evade Aging.” Aging (Albany NY)
7, no. 5: 334. 10.18632/aging.v7i5.
</cite> [<a href="https://doi.org/10.18632/aging.v7i5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4468314/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25991677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aging%20(Albany%20NY)&amp;title=Epigenetic%20Age%20Analysis%20of%20Children%20Who%20Seem%20to%20Evade%20Aging&amp;author=R.%20F.%20Walker&amp;author=J.%20S.%20Liu&amp;author=B.%20A.%20Peters&amp;volume=7&amp;issue=5&amp;publication_year=2015&amp;pages=334&amp;pmid=25991677&amp;doi=10.18632/aging.v7i5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0039">
<cite>

Wu, X.
, Chen W., Lin F., et al. 2019. “DNA Methylation Profile Is a Quantitative Measure of Biological Aging in Children.” Aging (Albany NY)
11, no. 22: 10031. 10.18632/aging.v11i22.
</cite> [<a href="https://doi.org/10.18632/aging.v11i22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6914436/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31756171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aging%20(Albany%20NY)&amp;title=DNA%20Methylation%20Profile%20Is%20a%20Quantitative%20Measure%20of%20Biological%20Aging%20in%20Children&amp;author=X.%20Wu&amp;author=W.%20Chen&amp;author=F.%20Lin&amp;volume=11&amp;issue=22&amp;publication_year=2019&amp;pages=10031&amp;pmid=31756171&amp;doi=10.18632/aging.v11i22&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0040">
<cite>

Zavala, D. V.
, Dzikowski N., Gopalan S., et al. 2024. “Epigenetic Age Acceleration and Chronological Age: Associations With Cognitive Performance in Daily Life.” Journals of Gerontology: Series A
79, no. 1: glad242.</cite> [<a href="https://doi.org/10.1093/gerona/glad242" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10733172/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37899644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journals%20of%20Gerontology:%20Series%20A&amp;title=Epigenetic%20Age%20Acceleration%20and%20Chronological%20Age:%20Associations%20With%20Cognitive%20Performance%20in%20Daily%20Life&amp;author=D.%20V.%20Zavala&amp;author=N.%20Dzikowski&amp;author=S.%20Gopalan&amp;volume=79&amp;issue=1&amp;publication_year=2024&amp;pages=glad242&amp;pmid=37899644&amp;doi=10.1093/gerona/glad242&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0041">
<cite>

Zhang, C. H.
, and Zhang S. S.. 2014. “Confidence Intervals for Low Dimensional Parameters in High Dimensional Linear Models.” Journal of the Royal Statistical Society Series B: Statistical Methodology
76, no. 1: 217–242.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Royal%20Statistical%20Society%20Series%20B:%20Statistical%20Methodology&amp;title=Confidence%20Intervals%20for%20Low%20Dimensional%20Parameters%20in%20High%20Dimensional%20Linear%20Models&amp;author=C.%20H.%20Zhang&amp;author=S.%20S.%20Zhang&amp;volume=76&amp;issue=1&amp;publication_year=2014&amp;pages=217-242&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0042">
<cite>

Zhong, W.
, Liu Y., and Zeng P.. 2023. “A Model‐Free Variable Screening Method Based on Leverage Score.” Journal of the American Statistical Association
118, no. 541: 135–146. 10.1080/01621459.2021.1918554.
</cite> [<a href="https://doi.org/10.1080/01621459.2021.1918554" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10281711/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37346228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Statistical%20Association&amp;title=A%20Model%E2%80%90Free%20Variable%20Screening%20Method%20Based%20on%20Leverage%20Score&amp;author=W.%20Zhong&amp;author=Y.%20Liu&amp;author=P.%20Zeng&amp;volume=118&amp;issue=541&amp;publication_year=2023&amp;pages=135-146&amp;pmid=37346228&amp;doi=10.1080/01621459.2021.1918554&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0043">
<cite>

Zhu, L. P.
, Li L., Li R., and Zhu L. X.. 2011. “Model‐Free Feature Screening for Ultrahigh‐Dimensional Data.” Journal of the American Statistical Association
106, no. 496: 1464–1475. 10.1198/jasa.2011.tm10563.
</cite> [<a href="https://doi.org/10.1198/jasa.2011.tm10563" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3384506/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22754050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Statistical%20Association&amp;title=Model%E2%80%90Free%20Feature%20Screening%20for%20Ultrahigh%E2%80%90Dimensional%20Data&amp;author=L.%20P.%20Zhu&amp;author=L.%20Li&amp;author=R.%20Li&amp;author=L.%20X.%20Zhu&amp;volume=106&amp;issue=496&amp;publication_year=2011&amp;pages=1464-1475&amp;pmid=22754050&amp;doi=10.1198/jasa.2011.tm10563&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gepi70008-bib-0044">
<cite>

Zou, H.
, and Hastie T.. 2005. “Regularization and Variable Selection via the Elastic Net.” Journal of the Royal Statistical Society Series B: Statistical Methodology
67, no. 2: 301–320.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Royal%20Statistical%20Society%20Series%20B:%20Statistical%20Methodology&amp;title=Regularization%20and%20Variable%20Selection%20via%20the%20Elastic%20Net&amp;author=H.%20Zou&amp;author=T.%20Hastie&amp;volume=67&amp;issue=2&amp;publication_year=2005&amp;pages=301-320&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are openly available in NCBI GEO (Gene Expression Omnibus) database at <a href="https://www.ncbi.nlm.nih.gov/geo/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Genetic Epidemiology are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/gepi.70008"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/GEPI-49-0.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11948180/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11948180/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11948180%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11948180/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11948180/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11948180/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40145332/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11948180/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40145332/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11948180/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11948180/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="GCUx73oePBszX2UT0ZhRRdpTuZJFcM0LmsKwMFKMgLZBpvhyDPvax0HEFnREqE2C">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
