
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Genetic mapping of loci associated with yield and their components in black common bean (Phaseolus vulgaris L.) - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="37B2C3BF7FFFA04306C3BF00076261CF.m_1" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11972933/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Genetic mapping of loci associated with yield and their components in black common bean (Phaseolus vulgaris L.)">
<meta name="citation_author" content="Deivid Lincoln Reche">
<meta name="citation_author_institution" content="Department of Agronomy, Universidade Estadual de Maringá‐UEM, Maringá, Brazil">
<meta name="citation_author" content="Maria Celeste Gonçalves‐Vidigal">
<meta name="citation_author_institution" content="Department of Agronomy, Universidade Estadual de Maringá‐UEM, Maringá, Brazil">
<meta name="citation_author" content="Pedro Soares Vidigal Filho">
<meta name="citation_author_institution" content="Department of Agronomy, Universidade Estadual de Maringá‐UEM, Maringá, Brazil">
<meta name="citation_author" content="Mariana Vaz Bisneta">
<meta name="citation_author_institution" content="Department of Agronomy, Universidade Estadual de Maringá‐UEM, Maringá, Brazil">
<meta name="citation_author" content="Giselly Figueiredo Lacanallo">
<meta name="citation_author_institution" content="Department of Agronomy, Universidade Estadual de Maringá‐UEM, Maringá, Brazil">
<meta name="citation_author" content="Alessandro A Brito dos Santos">
<meta name="citation_author_institution" content="Department of Agronomy, Universidade Estadual de Maringá‐UEM, Maringá, Brazil">
<meta name="citation_author" content="Abner Pais dos Santos">
<meta name="citation_author_institution" content="Department of Agronomy, Universidade Estadual de Maringá‐UEM, Maringá, Brazil">
<meta name="citation_publication_date" content="2025 Apr 6">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="2">
<meta name="citation_firstpage" content="e70024">
<meta name="citation_doi" content="10.1002/tpg2.70024">
<meta name="citation_pmid" content="40189482">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11972933/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11972933/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11972933/pdf/TPG2-18-e70024.pdf">
<meta name="description" content="The increase in world population linked to climate change leads to the need to develop more productive and more adapted cultivars of food species. Quantitative trait loci (QTLs) mapping is a useful tool although, interaction between genotype and the ...">
<meta name="og:title" content="Genetic mapping of loci associated with yield and their components in black common bean (Phaseolus vulgaris L.)">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The increase in world population linked to climate change leads to the need to develop more productive and more adapted cultivars of food species. Quantitative trait loci (QTLs) mapping is a useful tool although, interaction between genotype and the ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11972933/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11972933">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70024"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70024.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11972933%2F%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11972933/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11972933/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11972933/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70024" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Apr 6;18(2):e70024. doi: <a href="https://doi.org/10.1002/tpg2.70024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70024</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Genetic mapping of loci associated with yield and their components in black common bean (<em>Phaseolus vulgaris</em> L.)</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reche%20DL%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Deivid Lincoln Reche</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Deivid Lincoln Reche</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Agronomy, Universidade Estadual de Maringá‐UEM, Maringá, Brazil</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reche%20DL%22%5BAuthor%5D" class="usa-link"><span class="name western">Deivid Lincoln Reche</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gon%C3%A7alves%E2%80%90Vidigal%20MC%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Maria Celeste Gonçalves‐Vidigal</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Maria Celeste Gonçalves‐Vidigal</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Agronomy, Universidade Estadual de Maringá‐UEM, Maringá, Brazil</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gon%C3%A7alves%E2%80%90Vidigal%20MC%22%5BAuthor%5D" class="usa-link"><span class="name western">Maria Celeste Gonçalves‐Vidigal</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vidigal%20Filho%20PS%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Pedro Soares Vidigal Filho</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Pedro Soares Vidigal Filho</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Agronomy, Universidade Estadual de Maringá‐UEM, Maringá, Brazil</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vidigal%20Filho%20PS%22%5BAuthor%5D" class="usa-link"><span class="name western">Pedro Soares Vidigal Filho</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vaz%20Bisneta%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Mariana Vaz Bisneta</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Mariana Vaz Bisneta</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Agronomy, Universidade Estadual de Maringá‐UEM, Maringá, Brazil</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vaz%20Bisneta%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mariana Vaz Bisneta</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lacanallo%20GF%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Giselly Figueiredo Lacanallo</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Giselly Figueiredo Lacanallo</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Agronomy, Universidade Estadual de Maringá‐UEM, Maringá, Brazil</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lacanallo%20GF%22%5BAuthor%5D" class="usa-link"><span class="name western">Giselly Figueiredo Lacanallo</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22dos%20Santos%20AAB%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Alessandro A Brito dos Santos</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Alessandro A Brito dos Santos</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Agronomy, Universidade Estadual de Maringá‐UEM, Maringá, Brazil</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22dos%20Santos%20AAB%22%5BAuthor%5D" class="usa-link"><span class="name western">Alessandro A Brito dos Santos</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22dos%20Santos%20AP%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Abner Pais dos Santos</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Abner Pais dos Santos</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Agronomy, Universidade Estadual de Maringá‐UEM, Maringá, Brazil</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22dos%20Santos%20AP%22%5BAuthor%5D" class="usa-link"><span class="name western">Abner Pais dos Santos</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270024-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Agronomy, Universidade Estadual de Maringá‐UEM, Maringá, Brazil</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Maria Celeste Gonçalves‐Vidigal, Department of Agronomy, Universidade Estadual de Maringá‐UEM. Av. Colombo 5790, CEP 87020‐900, Maringá, PR, Brazil. Email: <span>mcgvidigal@uem.br</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jan 16; Received 2024 Jun 17; Accepted 2025 Feb 24; Issue date 2025 Jun.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11972933  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40189482/" class="usa-link">40189482</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The increase in world population linked to climate change leads to the need to develop more productive and more adapted cultivars of food species. Quantitative trait loci (QTLs) mapping is a useful tool although, interaction between genotype and the environment is still a challenge. In this study, we sought to identify QTL related to grain yield and the production components in common beans (<em>Phaseolus vulgaris</em> L.) supported by QTL <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-1" display="inline"><mrow><mo>×</mo></mrow></math></span> environment interaction. Two hundred eight recombinant inbred lines obtained from the Awauna UEM <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-2" display="inline"><mrow><mo>×</mo></mrow></math></span> IPR88 Uirapuru common bean cross were evaluated in 2017, 2018, and 2019 in field conditions under a 15 × 15 triple lattice experimental design. QTL mapping was estimated using genotypic means and a genetic linkage map with 288 single nucleotide polymorphism markers. Five QTLs associated with plant height (PH), number of pods per plant (NPP), first pod height (FPH), 100‐seed weight (SW), and grain yield per plant (GYP) were identified on chromosomes Pv01, Pv04, Pv08, and Pv10. Interestingly, three of these QTLs were co‐localized for more than one trait, where the QTL for PH, NPP, and GYP co‐locate on Pv01, the QTL for PH and FPH co‐locate on Pv04, and the QTL for NPP and SW co‐locate on Pv08. In turn, on Pv10, two distinct QTLs were found for SW. The identification of these QTLs stands out in Brazil since relatively little research is directed at this economically important commercial group. It is noteworthy that the molecular markers found linked to the QTLs must later be validated to be used in a multi‐trait marker‐assisted selection.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270024-list-10001" class="list" style="list-style-type:disc">
<li><p>Genetic mapping of the loci using recombinant inbreeding lines.</p></li>
<li><p>Quantitative trait loci (QTLs) mapping and a genetic linkage map with 288 single nucleotide polymorphism markers.</p></li>
<li><p>Five QTLs associated yield and their components on Pv01, Pv04, Pv08, and Pv10 chromosomes.</p></li>
<li><p>QTL related to grain yield and the production components in common bean supported by QTL environment interaction.</p></li>
<li><p>Genetic mapping of Awauna UEM × IPR88 Uirapuru population of black common bean.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270024-dl-0001">
<dt>FPH</dt>
<dd><p>first pod height</p></dd>
<dt>GYP</dt>
<dd><p>grain yield per plant</p></dd>
<dt>NPP</dt>
<dd><p>number of pods per plant</p></dd>
<dt>NSP</dt>
<dd><p>number of seeds per pod</p></dd>
<dt>PVE</dt>
<dd><p>explained phenotypic variation</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait locus</p></dd>
<dt>RILs</dt>
<dd><p>recombinant inbred lines</p></dd>
<dt>SNP</dt>
<dd><p>single nucleotide polymorphism</p></dd>
<dt>SW</dt>
<dd><p>100‐seed weight</p></dd>
</dl>
<section id="tpg270024-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>According to projections, the global population is expected to reach 10.9 billion by the year 2100. As a result, various sectors of society are increasingly focused on achieving food security. A key implication of this population increase is a greater demand for agricultural expansion, which requires more intensive land use. This shift calls for the adaptation of existing crop varieties or the development of new, more productive cultivars to meet the growing food needs (Garrett et al., <a href="#tpg270024-bib-0055" class="usa-link" aria-describedby="tpg270024-bib-0055">2018</a>).</p>
<p>Climate change requires the development of cultivars that can adapt to diverse and changing environmental conditions. The common bean (<em>Phaseolus vulgaris</em> L.) is an important crop with significant nutritional benefits. It provides essential amino acids, complex carbohydrates, dietary fiber, unsaturated fats, vitamins, and minerals. Moreover, it contains bioactive compounds with potential health benefits, including hypocholesterolemic, antiatherogenic, anticarcinogenic, and hypoglycemic properties (Celmeli et al., <a href="#tpg270024-bib-0007" class="usa-link" aria-describedby="tpg270024-bib-0007">2018</a>; Kotue et al., <a href="#tpg270024-bib-0025" class="usa-link" aria-describedby="tpg270024-bib-0025">2018</a>).</p>
<p>One of the major challenges faced by plant breeding programs is the variation in genotype performance across different environments, commonly referred to as genotype by environment interaction (G × E interaction; Singamsetti et al., <a href="#tpg270024-bib-0047" class="usa-link" aria-describedby="tpg270024-bib-0047">2021</a>). Identifying the underlying causes of G × E interaction and integrating this knowledge into breeding strategies requires extensive multi‐environment experiments, which demand large sample populations and considerable financial investment (A. A. Elias et al., <a href="#tpg270024-bib-0016" class="usa-link" aria-describedby="tpg270024-bib-0016">2016</a>).</p>
<p>Mapping studies identify genomic regions using molecular markers (Kromdijk et al., <a href="#tpg270024-bib-0026" class="usa-link" aria-describedby="tpg270024-bib-0026">2014</a>). Quantitative trait loci (QTLs) mapping in plants helps uncover the inheritance patterns and genetic architecture of quantitative traits, aiding the identification of functional markers that can be used for selecting complex traits (Bernardo, <a href="#tpg270024-bib-0003" class="usa-link" aria-describedby="tpg270024-bib-0003">2008</a>). However, most identified QTLs account for less than half of the total genetic variation; this can be attributed to the presence of numerous undetected minor QTL and epistatic interactions within the mapping process (Boopathi, <a href="#tpg270024-bib-0005" class="usa-link" aria-describedby="tpg270024-bib-0005">2020</a>). As a result, the interaction between QTL and environmental factors (QTL × E interaction) involves additional complexity, requiring more advanced statistical approaches to accurately assess and quantify these effects (Kromdijk et al., <a href="#tpg270024-bib-0026" class="usa-link" aria-describedby="tpg270024-bib-0026">2014</a>).</p>
<p>Several researchers have made significant contributions to mapping QTL in common beans. González et al. (<a href="#tpg270024-bib-0022" class="usa-link" aria-describedby="tpg270024-bib-0022">2016</a>) identified important QTL for plant height (PH) and the number of pods per plant (NPP) on chromosome Pv01. Similarly, Delfini et al. (<a href="#tpg270024-bib-0013" class="usa-link" aria-describedby="tpg270024-bib-0013">2021</a>) found QTLs associated with PH, the NPP, and 100‐seed weight (SW) on Pv01, as well as QTL for PH and the height of the first pod on Pv04. Langat et al. (<a href="#tpg270024-bib-0027" class="usa-link" aria-describedby="tpg270024-bib-0027">2019</a>) focused on mapping QTL related to drought tolerance in an F<sub>2</sub> population of common beans. They found QTL for the NPP on Pv01, QTL for SW on Pv08 and Pv10, and QTL for grain yield on Pv01. Mir et al. (<a href="#tpg270024-bib-0035" class="usa-link" aria-describedby="tpg270024-bib-0035">2021</a>) identified QTL for the NPP on Pv01 and Pv08, QTL for SW on Pv08 and Pv10, and QTL for grain yield per plant (GYP) on Pv01. These studies collectively highlight chromosome Pv01 as a significant locus for traits related to PH, the number of pods, and grain yield, with other QTL identified on chromosomes Pv04, Pv08, and Pv10. Cordoba‐Novoa et al. (<a href="#tpg270024-bib-0011" class="usa-link" aria-describedby="tpg270024-bib-0011">2022</a>) conducted an investigation for the genetic variability and structural characteristics of traits related to transpiration, stomatal regulation, and water utilization within Mesoamerican common bean germplasm. They successfully identified QTLs on chromosomes Pv01 and Pv07, as well as candidate genes and regulatory elements associated with tolerance to abiotic stress, including various transcription factors. Arriagada et al. (<a href="#tpg270024-bib-0001" class="usa-link" aria-describedby="tpg270024-bib-0001">2023</a>) performed a meta‐QTL (MQTL) analysis encompassing 667 QTLs derived from 21 distinct studies with the aim of discerning reliable genomic regions linked to yield‐related traits in common bean. Their comprehensive analysis unveiled 42 MQTLs, characterized by confidence intervals that were 3.41 times more precise than those of the original QTL, thereby enhancing the accuracy of mapping. Notably, among these MQTLs, 28 were correlated with both yield and phenological traits, highlighting their significant potential applicability in breeding programs. In a similar vein, Njau et al. (<a href="#tpg270024-bib-0040" class="usa-link" aria-describedby="tpg270024-bib-0040">2024</a>) identified 44 QTLs pertinent to pod quality and yield traits within a recombinant inbred line (RIL) population originating from a Vanilla × MCM5001 snap × dry bean cross. The QTLs associated with pod quality were localized on chromosomes Pv01, Pv02, Pv03, Pv04, Pv06, and Pv07, whereas those associated with pod yield were discerned on chromosome Pv08.</p>
<p>Traits controlled by multiple loci are often heavily influenced by environmental factors, making it crucial to assess QTL responses across various environments (A. A. Elias et al., <a href="#tpg270024-bib-0016" class="usa-link" aria-describedby="tpg270024-bib-0016">2016</a>). QTL analysis allows researchers to determine the number and location of key genetic regions, as well as to explore the interactions among loci and identify essential genes and their functions (Li et al., <a href="#tpg270024-bib-0029" class="usa-link" aria-describedby="tpg270024-bib-0029">2015</a>). Yuste‐Lisbona et al. (<a href="#tpg270024-bib-0054" class="usa-link" aria-describedby="tpg270024-bib-0054">2014</a>) discovered five QTLs with only individual additive effects, six with only epistatic effects, and 12 that exhibited both types of effects for pod traits. Hoyos‐Villegas et al. (<a href="#tpg270024-bib-0056" class="usa-link" aria-describedby="tpg270024-bib-0056">2016</a>) conducted a study over two agricultural years and identified QTL associated with seed yield, seed size, days to flowering, days to maturity, number of seeds, and PH.</p>
<p>Sandhu et al. (<a href="#tpg270024-bib-0045" class="usa-link" aria-describedby="tpg270024-bib-0045">2018</a>) evaluated an RIL population at two locations in southern Manitoba over the years 2014–2016 and found QTL associated with seed hydration and hardness, traits that impact cooking quality. Geravandi et al. (<a href="#tpg270024-bib-0021" class="usa-link" aria-describedby="tpg270024-bib-0021">2020</a>) identified QTL linked to seed size and yield‐related traits. González et al. (<a href="#tpg270024-bib-0022" class="usa-link" aria-describedby="tpg270024-bib-0022">2016</a>) evaluated an RIL population across various environments and identified QTL that control the number of days to flowering as a function of increasing degree‐days. J. C. F. Elias et al. (<a href="#tpg270024-bib-0017" class="usa-link" aria-describedby="tpg270024-bib-0017">2021</a>) examined RILs grown under both drought and non‐drought conditions for 2 years and identified 16 QTLs across chromosomes Pv01, Pv02, Pv03, Pv05, Pv07, Pv08, Pv09, Pv10, and Pv11. These studies highlight the complex genetic basis of common bean traits, demonstrating the presence of both single‐locus and epistatic QTL and the importance of environmental interaction. They also underscore the need for multi‐environmental studies to capture the full range of QTL effects, which is critical for plant breeding and the development of improved cultivars.</p>
<p>The present study aimed to map QTL related to production components, as well as interaction between QTL and the environment within a RIL population of black common bean derived from a cross between Awauna UEM × IPR88 Uirapuru.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270024-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270024-list-0001" class="list" style="list-style-type:disc">
<li><p>Genetic mapping of the loci using recombinant inbreeding lines.</p></li>
<li><p>Quantitative trait loci (QTLs) mapping and a genetic linkage map with 288 single nucleotide polymorphism markers.</p></li>
<li><p>Five QTLs associated yield and their components on Pv01, Pv04, Pv08, and Pv10 chromosomes.</p></li>
<li><p>QTL related to grain yield and the production components in common bean supported by QTL environment interaction.</p></li>
<li><p>Genetic mapping of Awauna UEM × IPR88 Uirapuru population of black common bean.</p></li>
</ul></section></section></section><section id="tpg270024-sec-0030"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270024-sec-0040"><h3 class="pmc_sec_title">2.1. Plant material</h3>
<p>In this study, common bean RILs were generated via artificial intragenic (Mesoamerican) hybridizations between the cultivars Awauna UEM (female) and IPR88 Uirapuru (male) within a controlled greenhouse environment at the Núcleo de Pesquisa Aplicada à Agricultura (Nupagri‐UEM). A total of 208 progenies were produced using the Single Seed Descent method, progressing through generations until reaching the F<sub>2:5</sub> stage. Subsequently, the RIL population (F<sub>2:6</sub>, F<sub>2:7</sub>, and F<sub>2:8</sub>) underwent phenotypic evaluations in field conditions at Centro de Treinamento em Irrigação (CTI‐UEM). Both facilities are part of the Universidade Estadual de Maringá‐UEM, situated in Maringá, in the Northwestern region of the state of Paraná, Brazil.</p>
<p>Awauna UEM is a black common bean cultivar product of the Nupagri‐UEM Common Bean Breeding Program. It derives from the crossing of the cultivar FT Nobre and the SEL 1308 line. Noteworthy characteristics of this cultivar include an average maturation period of 71 days from emergence to harvest, resistance to lodging, and a yield potential of 3400 kg ha<sup>−1</sup>, particularly suited for the southern region of the country. Moreover, it exhibits resistance to angular leaf spot (<em>Pseudocercospora griseola</em>), common bean mosaic virus (BCMV), and most of the Brazilian endemic races of <em>Colletotrichum lindemuthianum</em>, owing to the presence of the <em>Co‐4</em>
<sup>2</sup> anthracnose resistance allele.</p>
<p>IPR88 Uirapuru, a member of the black commercial group of common bean cultivars, is the result of breeding efforts of the Common Bean Improvement Program of the Agronomic Institute of Paraná. Its genetic lineage traces back to a combination of traditional cultivars, including BAC 29, PR1711/3, NEP2/2, Puebla 173, and ICA Pijao. Phenotypically, IPR88 Uirapuru exhibits wide adaptability across the primary production regions of Paraná, with optimal size for mechanical harvesting, an average growth cycle of 86 days, and a yield potential of 2407 kg ha<sup>−1</sup>. Additionally, it demonstrates tolerance to water deficit and high temperatures. Furthermore, it is resistant to BCMV, rust (<em>Uromyces phaseoli</em>), and powdery mildew (<em>Erysiphe polygoni</em>) (Moda‐Cirino et al., <a href="#tpg270024-bib-0036" class="usa-link" aria-describedby="tpg270024-bib-0036">2001</a>).</p></section><section id="tpg270024-sec-0050"><h3 class="pmc_sec_title">2.2. Field trial design, evaluation of productivity, and morpho‐agronomic traits</h3>
<p>A total of 208 RILs, the parent varieties Awauna UEM and IPR88 Uirapuru, and control cultivars, including BRS Esteio, FT Noble, FT Soberano, FT 120, Crioulo, and CHP996524, were evaluated. These assessments spanned three agricultural years, from August to November of 2017, 2018, and 2019, within the experimental fields of CTI‐UEM. Each experimental unit comprised a 2.0 m row, spaced 0.5 m apart, accommodating 24 plants. The experimental design was a triple lattice arrangement of 15×15.</p>
<p>The following agronomic traits were evaluated: PH: the distance from the soil surface to the tip of the leaves measured in centimeters; NPP: calculated as the total number of pods per plot divided by the number of plants within the plot; number of seeds per pod (NSP): determined by counting the seeds from 10 randomly selected pods within a plot; first pod height (FPH): determined by averaging the heights (in centimeters) of five plants at maturity, measured from the cotyledonary scar to the base of the first pod; SW: the weight of a sample containing 100 seeds expressed in grams; GYP: calculated by dividing the total mass of plant seeds by the number of plants within the effective plot area, considering a moisture content of 13%.</p></section><section id="tpg270024-sec-0060"><h3 class="pmc_sec_title">2.3. Phenotypic data analysis</h3>
<p>The phenotypic data from each trial underwent analysis using SELEGEN software (Resende, <a href="#tpg270024-bib-0043" class="usa-link" aria-describedby="tpg270024-bib-0043">2016</a>) and the R software (R Core Team, <a href="#tpg270024-bib-0042" class="usa-link" aria-describedby="tpg270024-bib-0042">2013</a>), utilizing the ggplot2 package (Wickham, <a href="#tpg270024-bib-0053" class="usa-link" aria-describedby="tpg270024-bib-0053">2016</a>). Within the R environment, we generated frequency density curves to illustrate the quantitative characteristics of the evaluated traits. Additionally, genetic and phenotypic correlation analyses were conducted, and scatter plots. These visualizations aimed to elucidate the potential influence of co‐localized QTLs across different traits (Figure <a href="#tpg270024-supinfo-0001" class="usa-link">S1</a>).</p>
<p>The prediction of genotypic values was performed via best linear unbiased predictor (BLUP), while estimates of variance components for random factors were derived through the restricted maximum likelihood method. Phenotypic data from each year (2017, 2018, and 2019) were independently analyzed using mixed models, specifically employing mathematical Model 1. In evaluating the genotype by environment (G × E) interaction, the Harmonic Mean of the Relative Performance of Genotypic Values (Model 2) was employed. The G × E interaction, both between pairs of agricultural years and across all 3 years, was treated as the environmental component for QTL mapping purposes, denoted as “Combination of agricultural years.”</p>
<p>The significance of model effects was assessed using the likelihood ratio test (LRT), facilitating a deviance analysis (ANADEV). Goodness of fit was determined by comparing models with and without genotype effects and genotype‐environment interactions. This comparison involved subtracting the deviance of the model without the effect from the deviance of the complete model, subsequently evaluating it against the chi‐square value (<em>χ</em>
<sup>2</sup>) with 1 degree of freedom at 1% and 5% probability levels (Resende &amp; Duarte, <a href="#tpg270024-bib-0044" class="usa-link" aria-describedby="tpg270024-bib-0044">2007</a>). Mathematically, this can be expressed as follows:
</p>
<table class="disp-formula p" id="tpg270024-disp-0001"><tr><td class="formula"><math id="jats-math-4" display="block"><mrow><mrow><mi>LRT</mi><mo linebreak="badbreak">=</mo><mo>−</mo><mn>2</mn><mi>ln</mi><mfenced separators="" open="(" close=")"><mfrac><mrow><mi>ML</mi><mspace width="0.28em"></mspace><mi>of</mi><mspace width="0.28em"></mspace><mi>reduced</mi><mspace width="0.28em"></mspace><mi>model</mi></mrow><mrow><mi>ML</mi><mspace width="0.28em"></mspace><mi>of</mi><mspace width="0.28em"></mspace><mi>complete</mi><mspace width="0.28em"></mspace><mi>model</mi></mrow></mfrac></mfenced></mrow></mrow></math></td></tr></table>
<p>where ln represents the Naperian logarithm and ML is the maximum likelihood estimation.</p>
<table class="disp-formula p" id="tpg270024-disp-0002"><tr><td class="formula"><math id="jats-math-5" display="block"><mrow><mrow><mrow><mi>Model</mi><mspace width="0.28em"></mspace></mrow><mn>1</mn><mo>:</mo><mi>y</mi><mo linebreak="goodbreak">=</mo><msub><mi>x</mi><mi>r</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>z</mi><mi>g</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>w</mi><mi>b</mi></msub><mo linebreak="goodbreak">+</mo><mi>e</mi></mrow></mrow></math></td></tr></table>
<table class="disp-formula p" id="tpg270024-disp-0003"><tr><td class="formula"><math id="jats-math-6" display="block"><mrow><mrow><mrow><mi>Model</mi><mspace width="0.28em"></mspace></mrow><mn>2</mn><mo>:</mo><mi>y</mi><mo linebreak="goodbreak">=</mo><msub><mi>x</mi><mi>r</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>z</mi><mi>g</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>w</mi><mi>b</mi></msub><mo linebreak="goodbreak">+</mo><msub><mi>T</mi><mi>i</mi></msub><mo linebreak="goodbreak">+</mo><mi>e</mi></mrow></mrow></math></td></tr></table>
<p>where <em>y</em> is the data vector; <em>r</em> is the replication effect vector (assumed as fixed) added to the overall mean; <em>g</em> is the genotypic effect vector (assumed as random); <em>b</em> is the block effect vector (assumed as random); <em>i</em> is the genotype environment interaction effects (random); <em>e</em> is the error or residuals value (random). The uppercase letters represent the incidence matrices for the aforementioned effects.</p></section><section id="tpg270024-sec-0070"><h3 class="pmc_sec_title">2.4. Genotyping of the RILs using BARCBean6K_3 BeadChip</h3>
<p>Genomic DNA extraction was performed on the 208 RILs (F<sub>2:8</sub> generation) and their parental lines (Awauna UEM and IPR88 Uirapuru) using the DNeasy plant mini kit (Qiagen) in accordance with the manufacturer's guidelines. The extracted DNA was then quantified using a 1.5% agarose gel (Agarose SFR, Amresco) in TBE buffer (tris‐borate‐ethylenediamine tetraacetic acid) and visualized by staining with 1 g mL<sup>−1</sup> Sybr Green (Sigma‐Aldrich).</p>
<p>The DNA samples were screened with 5398 SNP (single nucleotide polymorphism) DNA markers using the BARCBean6K_3 Illumina BeadChip, employing the Infinium HD assay ultra‐protocol (Illumina, Inc.). Fluorescence intensity measurements were obtained by imaging the BeadChip with the Illumina beadarray reader. Automatic allele calling for each locus was carried out using the GenomeStudio Genotyping Module v1.8.4 software (Illumina), followed by visual inspection. Errors in allele calling due to improper cluster identification were corrected, resulting in a final set of 800 SNPs.</p>
<p>Following quality assessment of SNPs using the GenomeStudio software, we employed a Microsoft Excel electronic spreadsheet to execute the filtering process. This involved excluding monomorphic SNPs present in both parental lines, Awauna UEM and IPR88 Uirapuru, as well as those with missing data exceeding 5% and a minimum allele frequency below 30%. Furthermore, SNP markers were mapped to their respective chromosomes and assigned nomenclature deposited at the National Center for Biotechnology Information database.</p></section><section id="tpg270024-sec-0080"><h3 class="pmc_sec_title">2.5. Genetic mapping and QTL analysis</h3>
<p>The QTL analysis, conducted on the new adjusted genotypic means of the phenotypic traits evaluated across the years 2017, 2018, and 2019, as well as the combination of agricultural years, was carried out using the QTL software IciMapping Version 4.2 (Meng et al., <a href="#tpg270024-bib-0033" class="usa-link" aria-describedby="tpg270024-bib-0033">2015</a>). This software employs the inclusive composite interval mapping (ICIM) method, integrating both the BIP and MET analysis modules in the appropriate sequence. These modules enable the mapping of QTL with additive effects in individual environments and QTL with environmental interactions across multiple environments (Li et al., <a href="#tpg270024-bib-0029" class="usa-link" aria-describedby="tpg270024-bib-0029">2015</a>).</p>
<p>Significance of QTLs was determined using the 1000‐permutation test for each trait in QTL IciMapping, establishing the LOD threshold at <em>p</em> = 0.05. The peak LOD score was considered as the location of the QTL. For QTLs with additive effect genes, a 1 cM scan step was utilized, with a probability of 0.001 for inserting variables in stepwise regression. Detection and declaration of putative QTLs relied on a minimum LOD score of three in individual settings and a minimum LOD score of 4.5 in multi‐environmental analyses (<em>α</em>: 0.05; Li et al., <a href="#tpg270024-bib-0029" class="usa-link" aria-describedby="tpg270024-bib-0029">2015</a>).</p>
<p>In QTL classification, the following criteria were applied: a main QTL is defined by a minimum LOD score of five and a proportion of the total explained phenotypic variation (total PVE) exceeding 10%; a stable QTL exhibits a LOD score greater than three in at least two environments; and a specific QTL is characterized by a minimum LOD score of five and significant QTL × E interaction. QTLs were named according to the guidelines proposed by Miklas and Porch (<a href="#tpg270024-bib-0034" class="usa-link" aria-describedby="tpg270024-bib-0034">2010</a>) for QTL naming in common bean. Finally, using MapChart 2.3 (Voorrips, <a href="#tpg270024-bib-0052" class="usa-link" aria-describedby="tpg270024-bib-0052">2002</a>), the QTLs were visually depicted on genetic linkage maps for <em>P. vulgaris</em> developed using the RIL population Awauna UEM × IPR88 Uirapuru.</p></section></section><section id="tpg270024-sec-0090"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270024-sec-0100"><h3 class="pmc_sec_title">3.1. Phenotypic performance of RILs and assessment of experimental quality</h3>
<p>Evaluation of the RIL population at the field conditions for three crop years provided an opportunity to account for their phenotypic performance under a variety of environmental conditions (Table <a href="#tpg270024-tbl-0001" class="usa-link">1</a> and Table <a href="#tpg270024-supinfo-0001" class="usa-link">S1</a>). Despite the phenotypic similarity in terms of averages between parents and RILs, these showed a wide range of extreme values (max and min), denoting a substantial transgressive segregation and indicating that both parents carry genes that contribute to the variation of traits.</p>
<section class="tw xbox font-sm" id="tpg270024-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Descriptive statistics of the traits plant height, number of pods per plant, number of seeds per pod, first pod height, 100‐seed weight and grain yield per plant of the 208 common bean recombinant inbred lines (RILs) from Awauna UEM <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline"><mrow><mo>×</mo></mrow></math></span> IPR88 Uirapuru cross in the agricultural years 2017, 2018, and 2019.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Traits</th>
<th align="left" rowspan="1" colspan="1">Mean</th>
<th align="left" rowspan="1" colspan="1">SD</th>
<th align="left" rowspan="1" colspan="1">CV%</th>
<th align="left" rowspan="1" colspan="1">Max</th>
<th align="left" rowspan="1" colspan="1">Min</th>
<th align="left" rowspan="1" colspan="1">Skew</th>
<th align="left" rowspan="1" colspan="1">Inf.S</th>
<th align="left" rowspan="1" colspan="1">Kurt</th>
<th align="left" rowspan="1" colspan="1">Inf.K</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>Plant Height</strong>
</td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2017</td>
<td align="center" rowspan="1" colspan="1">58.11</td>
<td align="center" rowspan="1" colspan="1">8.09</td>
<td align="center" rowspan="1" colspan="1">13.93</td>
<td align="center" rowspan="1" colspan="1">64.86</td>
<td align="center" rowspan="1" colspan="1">53.23</td>
<td align="left" rowspan="1" colspan="1">0.50</td>
<td align="left" rowspan="1" colspan="1">(+)</td>
<td align="center" rowspan="1" colspan="1">−0.10</td>
<td align="left" rowspan="1" colspan="1">Meso</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2018</td>
<td align="center" rowspan="1" colspan="1">58.35</td>
<td align="center" rowspan="1" colspan="1">7.80</td>
<td align="center" rowspan="1" colspan="1">13.37</td>
<td align="center" rowspan="1" colspan="1">64.37</td>
<td align="center" rowspan="1" colspan="1">53.66</td>
<td align="left" rowspan="1" colspan="1">0.48</td>
<td align="left" rowspan="1" colspan="1">(+)</td>
<td align="center" rowspan="1" colspan="1">0.10</td>
<td align="left" rowspan="1" colspan="1">Meso</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2019</td>
<td align="center" rowspan="1" colspan="1">61.71</td>
<td align="center" rowspan="1" colspan="1">8.00</td>
<td align="center" rowspan="1" colspan="1">12.97</td>
<td align="center" rowspan="1" colspan="1">67.75</td>
<td align="center" rowspan="1" colspan="1">57.64</td>
<td align="left" rowspan="1" colspan="1">0.31</td>
<td align="left" rowspan="1" colspan="1">(+)</td>
<td align="center" rowspan="1" colspan="1">0.05</td>
<td align="left" rowspan="1" colspan="1">Meso</td>
</tr>
</tbody>
</table></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">No. of pods per plant</th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">2017</td>
<td align="center" rowspan="1" colspan="1">12.84</td>
<td align="center" rowspan="1" colspan="1">2.04</td>
<td align="center" rowspan="1" colspan="1">15.92</td>
<td align="center" rowspan="1" colspan="1">19.58</td>
<td align="center" rowspan="1" colspan="1">8.15</td>
<td align="center" rowspan="1" colspan="1">0.34</td>
<td align="left" rowspan="1" colspan="1">(+)</td>
<td align="center" rowspan="1" colspan="1">−0.13</td>
<td align="left" rowspan="1" colspan="1">Meso</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2018</td>
<td align="center" rowspan="1" colspan="1">13.06</td>
<td align="center" rowspan="1" colspan="1">2.34</td>
<td align="center" rowspan="1" colspan="1">17.96</td>
<td align="center" rowspan="1" colspan="1">19.92</td>
<td align="center" rowspan="1" colspan="1">2.36</td>
<td align="center" rowspan="1" colspan="1">−0.43</td>
<td align="left" rowspan="1" colspan="1">(−)</td>
<td align="center" rowspan="1" colspan="1">0.90</td>
<td align="left" rowspan="1" colspan="1">Lepto</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2019</td>
<td align="center" rowspan="1" colspan="1">12.73</td>
<td align="center" rowspan="1" colspan="1">1.85</td>
<td align="center" rowspan="1" colspan="1">14.52</td>
<td align="center" rowspan="1" colspan="1">18.74</td>
<td align="center" rowspan="1" colspan="1">8.77</td>
<td align="center" rowspan="1" colspan="1">0.30</td>
<td align="left" rowspan="1" colspan="1">(+)</td>
<td align="center" rowspan="1" colspan="1">0.01</td>
<td align="left" rowspan="1" colspan="1">Meso</td>
</tr>
</tbody>
</table></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">No. of seeds per pod</th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">2017</td>
<td align="center" rowspan="1" colspan="1">5.56</td>
<td align="center" rowspan="1" colspan="1">0.74</td>
<td align="center" rowspan="1" colspan="1">13.24</td>
<td align="center" rowspan="1" colspan="1">7.33</td>
<td align="center" rowspan="1" colspan="1">4.00</td>
<td align="center" rowspan="1" colspan="1">−0.16</td>
<td align="left" rowspan="1" colspan="1">Sym</td>
<td align="left" rowspan="1" colspan="1">−0.23</td>
<td align="center" rowspan="1" colspan="1">Meso</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2018</td>
<td align="center" rowspan="1" colspan="1">5.82</td>
<td align="center" rowspan="1" colspan="1">0.62</td>
<td align="center" rowspan="1" colspan="1">10.57</td>
<td align="center" rowspan="1" colspan="1">7.33</td>
<td align="center" rowspan="1" colspan="1">4.00</td>
<td align="center" rowspan="1" colspan="1">−0.61</td>
<td align="left" rowspan="1" colspan="1">(−)</td>
<td align="left" rowspan="1" colspan="1">0.45</td>
<td align="center" rowspan="1" colspan="1">Lepto</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2019</td>
<td align="center" rowspan="1" colspan="1">5.78</td>
<td align="center" rowspan="1" colspan="1">0.66</td>
<td align="center" rowspan="1" colspan="1">11.41</td>
<td align="center" rowspan="1" colspan="1">7.00</td>
<td align="center" rowspan="1" colspan="1">4.00</td>
<td align="center" rowspan="1" colspan="1">−0.21</td>
<td align="left" rowspan="1" colspan="1">(−)</td>
<td align="left" rowspan="1" colspan="1">−0.12</td>
<td align="center" rowspan="1" colspan="1">Meso</td>
</tr>
</tbody>
</table></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">First pod height</th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1">f.S</th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">2017</td>
<td align="center" rowspan="1" colspan="1">18.11</td>
<td align="center" rowspan="1" colspan="1">2.29</td>
<td align="center" rowspan="1" colspan="1">12.62</td>
<td align="center" rowspan="1" colspan="1">24.86</td>
<td align="center" rowspan="1" colspan="1">13.23</td>
<td align="center" rowspan="1" colspan="1">0.50</td>
<td align="left" rowspan="1" colspan="1">(+)</td>
<td align="center" rowspan="1" colspan="1">−0.10</td>
<td align="left" rowspan="1" colspan="1">Meso</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2018</td>
<td align="center" rowspan="1" colspan="1">18.35</td>
<td align="center" rowspan="1" colspan="1">1.97</td>
<td align="center" rowspan="1" colspan="1">10.73</td>
<td align="center" rowspan="1" colspan="1">24.37</td>
<td align="center" rowspan="1" colspan="1">13.66</td>
<td align="center" rowspan="1" colspan="1">0.48</td>
<td align="left" rowspan="1" colspan="1">(+)</td>
<td align="center" rowspan="1" colspan="1">0.10</td>
<td align="left" rowspan="1" colspan="1">Meso</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2019</td>
<td align="center" rowspan="1" colspan="1">17.16</td>
<td align="center" rowspan="1" colspan="1">1.63</td>
<td align="center" rowspan="1" colspan="1">9.51</td>
<td align="center" rowspan="1" colspan="1">22.44</td>
<td align="center" rowspan="1" colspan="1">13.16</td>
<td align="center" rowspan="1" colspan="1">0.52</td>
<td align="left" rowspan="1" colspan="1">(+)</td>
<td align="center" rowspan="1" colspan="1">0.10</td>
<td align="left" rowspan="1" colspan="1">Meso</td>
</tr>
</tbody>
</table></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<thead><tr>
<th align="left" rowspan="1" colspan="1">100‐Seed weight</th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">2017</td>
<td align="center" rowspan="1" colspan="1">24.60</td>
<td align="center" rowspan="1" colspan="1">2.21</td>
<td align="center" rowspan="1" colspan="1">9.00</td>
<td align="center" rowspan="1" colspan="1">32.18</td>
<td align="center" rowspan="1" colspan="1">15.84</td>
<td align="center" rowspan="1" colspan="1">−0.01</td>
<td align="left" rowspan="1" colspan="1">Sym</td>
<td align="center" rowspan="1" colspan="1">0.62</td>
<td align="left" rowspan="1" colspan="1">Lepto</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2018</td>
<td align="center" rowspan="1" colspan="1">25.74</td>
<td align="center" rowspan="1" colspan="1">2.34</td>
<td align="center" rowspan="1" colspan="1">9.11</td>
<td align="center" rowspan="1" colspan="1">32.60</td>
<td align="center" rowspan="1" colspan="1">15.04</td>
<td align="center" rowspan="1" colspan="1">−0.43</td>
<td align="left" rowspan="1" colspan="1">(−)</td>
<td align="center" rowspan="1" colspan="1">0.90</td>
<td align="left" rowspan="1" colspan="1">Lepto</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2019</td>
<td align="center" rowspan="1" colspan="1">24.45</td>
<td align="center" rowspan="1" colspan="1">2.49</td>
<td align="center" rowspan="1" colspan="1">10.18</td>
<td align="center" rowspan="1" colspan="1">32.11</td>
<td align="center" rowspan="1" colspan="1">15.22</td>
<td align="center" rowspan="1" colspan="1">−0.53</td>
<td align="left" rowspan="1" colspan="1">(−)</td>
<td align="center" rowspan="1" colspan="1">0.80</td>
<td align="left" rowspan="1" colspan="1">Lepto</td>
</tr>
</tbody>
</table></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<thead><tr>
<th align="left" rowspan="1" colspan="1">Grain yield per plant</th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">2017</td>
<td align="center" rowspan="1" colspan="1">15.24</td>
<td align="center" rowspan="1" colspan="1">2.04</td>
<td align="center" rowspan="1" colspan="1">13.42</td>
<td align="center" rowspan="1" colspan="1">21.98</td>
<td align="center" rowspan="1" colspan="1">10.55</td>
<td align="center" rowspan="1" colspan="1">0.34</td>
<td align="left" rowspan="1" colspan="1">(+)</td>
<td align="center" rowspan="1" colspan="1">−0.13</td>
<td align="left" rowspan="1" colspan="1">Meso</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2018</td>
<td align="center" rowspan="1" colspan="1">15.74</td>
<td align="center" rowspan="1" colspan="1">2.00</td>
<td align="center" rowspan="1" colspan="1">12.70</td>
<td align="center" rowspan="1" colspan="1">22.99</td>
<td align="center" rowspan="1" colspan="1">11.44</td>
<td align="center" rowspan="1" colspan="1">0.45</td>
<td align="left" rowspan="1" colspan="1">(+)</td>
<td align="center" rowspan="1" colspan="1">0.40</td>
<td align="left" rowspan="1" colspan="1">Lepto</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2019</td>
<td align="center" rowspan="1" colspan="1">15.13</td>
<td align="center" rowspan="1" colspan="1">1.85</td>
<td align="center" rowspan="1" colspan="1">12.22</td>
<td align="center" rowspan="1" colspan="1">21.14</td>
<td align="center" rowspan="1" colspan="1">11.17</td>
<td align="center" rowspan="1" colspan="1">0.30</td>
<td align="left" rowspan="1" colspan="1">(+)</td>
<td align="center" rowspan="1" colspan="1">0.01</td>
<td align="left" rowspan="1" colspan="1">Meso</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270024-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270024-tbl1-note-0001"><p>Abbreviations: CV%, coefficient of variation of the phenotypic data; Inf.K, inference about the kurtosis; Inf.S, inference about the skewness; Kurt, kurtosis of the phenotypic data; Lepto, leptokurtic; Max, maximum of the phenotypic data from the RILs; Meso, mesokurtic; Min, minimum of the phenotypic data from the RILs; Skew, skewness of the phenotypic data. (+), (−), and Sym means positive asymmetric, negative asymmetric, and symmetric, respectively.</p></div></div></section><p>Observing the pattern of data distribution (Figure <a href="#tpg270024-supinfo-0001" class="usa-link">S1</a>), the inference about the asymmetry reveals that there is a certain disturbance in the data distribution due to the complementary gene interaction in the traits PH, NPP, FPH, and GYP, as well as duplicated gene interaction in the traits NSP and SW (Table <a href="#tpg270024-tbl-0001" class="usa-link">1</a>). Obviously, such interactions are not mutually exclusive events, as they are polygenic traits.</p>
<p>Another important information provided by the frequency distribution in phenotypic classes concerns kurtosis. All evaluated traits reveal some form of gene interaction, especially the NPP, NSP, SW, and GYP for both showing, at some point, leptokurtic behavior of the data distribution (Table <a href="#tpg270024-tbl-0001" class="usa-link">1</a>).</p>
<p>The selective accuracy values obtained with the experimental data (Table <a href="#tpg270024-supinfo-0001" class="usa-link">S2</a>) were, on average, &gt;70%, being considered, therefore, high and consistent with the need for data required for the purpose of QTL mapping.</p></section><section id="tpg270024-sec-0110"><h3 class="pmc_sec_title">3.2. Genetic correlation between the evaluated traits</h3>
<p>In the partition of the effects involved in the differential expression of the correlated traits, it was observed that, under the experimental conditions, the genetic effects played a major role in the correlations, since most of the traits, at some point, were genetically associated with each other in greater or lesser degree (Table <a href="#tpg270024-tbl-0002" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="tpg270024-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Genetic correlation coefficient of the traits plant height (PH), number of pods per plant (NPP), number of seeds per pod (NSP), first pod height (FPH), 100‐seed weight (SW), and grain yield per plant (GYP) of the 208 common bean recombinant inbred lines (RILs) to the agricultural years 2017, 2018, and 2019.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Traits</th>
<th align="left" rowspan="1" colspan="1">Agricultural year</th>
<th align="left" rowspan="1" colspan="1">PH</th>
<th align="left" rowspan="1" colspan="1">NPP</th>
<th align="left" rowspan="1" colspan="1">NSP</th>
<th align="left" rowspan="1" colspan="1">FSH</th>
<th align="left" rowspan="1" colspan="1">SW</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="3" align="left" colspan="1">NPP</td>
<td align="left" rowspan="1" colspan="1">2017</td>
<td align="center" rowspan="1" colspan="1">0.066</td>
<td align="center" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2018</td>
<td align="center" rowspan="1" colspan="1">0.162<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2019</td>
<td align="center" rowspan="1" colspan="1">0.999<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">NSP</td>
<td align="left" rowspan="1" colspan="1">2017</td>
<td align="center" rowspan="1" colspan="1">−0.107</td>
<td align="center" rowspan="1" colspan="1">0.034</td>
<td align="center" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2018</td>
<td align="center" rowspan="1" colspan="1">−0.031</td>
<td align="center" rowspan="1" colspan="1">−0.224<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2019</td>
<td align="center" rowspan="1" colspan="1">0.128</td>
<td align="center" rowspan="1" colspan="1">0.130</td>
<td align="center" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">FPH</td>
<td align="left" rowspan="1" colspan="1">2017</td>
<td align="center" rowspan="1" colspan="1">0.999<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">0.066</td>
<td align="center" rowspan="1" colspan="1">−0.107</td>
<td align="center" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2018</td>
<td align="center" rowspan="1" colspan="1">0.999<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">0.162<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">−0.031</td>
<td align="center" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2019</td>
<td align="center" rowspan="1" colspan="1">0.399<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">0.396<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">0.118</td>
<td align="center" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">SW</td>
<td align="left" rowspan="1" colspan="1">2017</td>
<td align="center" rowspan="1" colspan="1">0.158<sup>*</sup>
</td>
<td align="center" rowspan="1" colspan="1">0.185<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">0.107</td>
<td align="center" rowspan="1" colspan="1">0.158<sup>*</sup>
</td>
<td align="center" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2018</td>
<td align="center" rowspan="1" colspan="1">0.162<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">0.999<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">−0.224<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">0.162<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2019</td>
<td align="center" rowspan="1" colspan="1">0.314<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">0.316<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">−0.014</td>
<td align="center" rowspan="1" colspan="1">0.427<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">–</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">GYP</td>
<td align="left" rowspan="1" colspan="1">2017</td>
<td align="center" rowspan="1" colspan="1">0.066</td>
<td align="center" rowspan="1" colspan="1">0.999<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">0.034</td>
<td align="center" rowspan="1" colspan="1">0.066</td>
<td align="center" rowspan="1" colspan="1">0.185<sup>**</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2018</td>
<td align="center" rowspan="1" colspan="1">0.130*</td>
<td align="center" rowspan="1" colspan="1">0.230<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">0.052</td>
<td align="center" rowspan="1" colspan="1">0.130</td>
<td align="center" rowspan="1" colspan="1">0.230<sup>**</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2019</td>
<td align="center" rowspan="1" colspan="1">0.999<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">0.999<sup>**</sup>
</td>
<td align="center" rowspan="1" colspan="1">0.130</td>
<td align="center" rowspan="1" colspan="1">0.196</td>
<td align="center" rowspan="1" colspan="1">0.316<sup>**</sup>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270024-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270024-tbl2-note-0001"><p>** and * effects at 1% and 5% probability by <em>t</em> test, respectively.</p></div></div></section><p>The genotypic correlation coefficients of PH were significant and positive with all traits, except for the NSP, which was not significant. The NPP showed a significant and positive correlation with all traits; however, its correlation with the NSP was negative. The SW had a significant and negative correlation with the NSP, but it was positively correlated with the height of the first pod and GYP. (Table <a href="#tpg270024-tbl-0002" class="usa-link">2</a>).</p>
<p>Although the high estimates of genetic correlation observed indicate the presence of pleiotropy, the contrast is not true, as different pleiotropic loci can result in positive or negative covariances between traits, which can cancel each other out. In other words, pleiotropic effects of different genes that affect pairs of traits may not lie in a single direction for each trait and, therefore, will not result in significant genetic correlations between them (Mackay et al., <a href="#tpg270024-bib-0031" class="usa-link" aria-describedby="tpg270024-bib-0031">2009</a>).</p>
<p>The phenotypic correlation and its respective graphical representation, the scatter plot (Figure <a href="#tpg270024-supinfo-0001" class="usa-link">S1</a>), evidenced the important role of the G × E interaction in the associated pairs, since the genotypic correlation only partially reflected the phenotypic correlation. Of the 11 pairs of significant genotypic correlations (Table <a href="#tpg270024-tbl-0002" class="usa-link">2</a>), only six remained at a phenotypic level, either the pairs: FPH and NPP; MS and NPP; GYP and NPP; NSP and MS; FPH and MS; and GYP and MS (Figure <a href="#tpg270024-supinfo-0001" class="usa-link">S1</a>). In part, the explanation of the observed is supported by the premise that the interaction between pleiotropic loci and environments can alter the genetic effects of traits according to the environment, leading to differences in covariances between traits and consequent changes in correlations due to imposed environmental differences (Pavličev &amp; Cheverud, <a href="#tpg270024-bib-0041" class="usa-link" aria-describedby="tpg270024-bib-0041">2015</a>).</p></section><section id="tpg270024-sec-0120"><h3 class="pmc_sec_title">3.3. Deviation analysis and detection of G × E interaction</h3>
<p>Deviation analysis showed a significant difference between the RILs and a significant G × E interaction (<em>α</em>: 1% and 5%) for all evaluated traits, years, and combination of years (Table <a href="#tpg270024-supinfo-0001" class="usa-link">S3</a>). Therefore, it was necessary to adjust the phenotypic means using BLUPs, mitigating environmental effects that reduce the range of population variation that would restrict QTL detection. Furthermore, the new averages avoid confusion between environmental effects and QTL, resulting in consistent values of PVE.</p></section><section id="tpg270024-sec-0130"><h3 class="pmc_sec_title">3.4. Genetic mapping</h3>
<p>This study involved genotyping 208 RILs derived from the Awauna UEM × IPR88 Uirapuru cross using 5398 SNPs (Table <a href="#tpg270024-supinfo-0001" class="usa-link">S4</a>). Following the filtering process, a total of 288 SNP markers were selected for mapping procedures (Table <a href="#tpg270024-tbl-0003" class="usa-link">3</a>). The mapping of the Awauna UEM × IPR88 Uirapuru population spanned 590.3 cM, with an average marker distance of 2.05 cM. The number of markers ranged from 12 to 39 per chromosome.</p>
<section class="tw xbox font-sm" id="tpg270024-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>Distribution of SNPs markers on the linkage map constructed from the recombinant inbred line (RIL) population Awauna UEM <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-8" display="inline"><mrow><mo>×</mo></mrow></math></span> IPR88 Uirapuru.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Chromosome</th>
<th align="left" rowspan="1" colspan="1">No. of markers</th>
<th align="left" rowspan="1" colspan="1">Starting marker</th>
<th align="left" rowspan="1" colspan="1">Ending marker</th>
<th align="left" rowspan="1" colspan="1">Map length (cM)</th>
<th align="left" rowspan="1" colspan="1">Density<a href="#tpg270024-tbl3-note-0001" class="usa-link">
<sup>a</sup>
</a>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">30</td>
<td align="left" rowspan="1" colspan="1">ss715645919</td>
<td align="left" rowspan="1" colspan="1">ss715648161</td>
<td align="center" rowspan="1" colspan="1">93.98</td>
<td align="center" rowspan="1" colspan="1">3.13</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">39</td>
<td align="left" rowspan="1" colspan="1">ss715648504</td>
<td align="left" rowspan="1" colspan="1">ss715646370</td>
<td align="center" rowspan="1" colspan="1">15.83</td>
<td align="center" rowspan="1" colspan="1">0.41</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">ss715646392</td>
<td align="left" rowspan="1" colspan="1">ss715647338</td>
<td align="center" rowspan="1" colspan="1">83.51</td>
<td align="center" rowspan="1" colspan="1">4.91</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">22</td>
<td align="left" rowspan="1" colspan="1">ss715646899</td>
<td align="left" rowspan="1" colspan="1">ss715640244</td>
<td align="center" rowspan="1" colspan="1">86.09</td>
<td align="center" rowspan="1" colspan="1">3.91</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">35</td>
<td align="left" rowspan="1" colspan="1">ss715651063</td>
<td align="left" rowspan="1" colspan="1">ss715645454</td>
<td align="center" rowspan="1" colspan="1">59.93</td>
<td align="center" rowspan="1" colspan="1">1.71</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">24</td>
<td align="left" rowspan="1" colspan="1">ss715646671</td>
<td align="left" rowspan="1" colspan="1">ss715650275</td>
<td align="center" rowspan="1" colspan="1">53.16</td>
<td align="center" rowspan="1" colspan="1">2.22</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">ss715649072</td>
<td align="left" rowspan="1" colspan="1">ss715646021</td>
<td align="center" rowspan="1" colspan="1">7.22</td>
<td align="center" rowspan="1" colspan="1">0.60</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">29</td>
<td align="left" rowspan="1" colspan="1">ss715646515</td>
<td align="left" rowspan="1" colspan="1">ss715648623</td>
<td align="center" rowspan="1" colspan="1">84.30</td>
<td align="center" rowspan="1" colspan="1">2.91</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">28</td>
<td align="left" rowspan="1" colspan="1">ss715646185</td>
<td align="left" rowspan="1" colspan="1">ss715649931</td>
<td align="center" rowspan="1" colspan="1">23.86</td>
<td align="center" rowspan="1" colspan="1">0.85</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">28</td>
<td align="left" rowspan="1" colspan="1">ss715641336</td>
<td align="left" rowspan="1" colspan="1">ss715639844</td>
<td align="center" rowspan="1" colspan="1">61.27</td>
<td align="center" rowspan="1" colspan="1">2.19</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">24</td>
<td align="left" rowspan="1" colspan="1">ss715640322</td>
<td align="left" rowspan="1" colspan="1">ss715649886</td>
<td align="center" rowspan="1" colspan="1">21.14</td>
<td align="center" rowspan="1" colspan="1">0.88</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Entire genome</td>
<td align="left" rowspan="1" colspan="1">288</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="left" rowspan="1" colspan="1">–</td>
<td align="center" rowspan="1" colspan="1">590.293</td>
<td align="center" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-9" display="inline"><mrow><mover accent="true"><mi mathvariant="normal">x</mi><mo>¯</mo></mover></mrow></math></span>: 2.05</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270024-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270024-tbl3-note-0001">
<sup>
<sup>a</sup>
</sup><p class="display-inline">Density refers to marker density in cM/marker.</p>
</div></div></section></section><section id="tpg270024-sec-0140"><h3 class="pmc_sec_title">3.5. QTL mapping</h3>
<p>Utilizing Inclusive Composite Interval QTL mapping within individual environments, we identified a total of five QTLs associated with PH, NPP, FPH, SW, and GYP on chromosomes Pv01, Pv04, Pv08, and Pv10 (Figure <a href="#tpg270024-fig-0001" class="usa-link">1</a>; Table <a href="#tpg270024-tbl-0004" class="usa-link">4</a>).</p>
<figure class="fig xbox font-sm" id="tpg270024-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ce84/11972933/777efd47e0a1/TPG2-18-e70024-g001.jpg" loading="lazy" id="jats-graphic-1" height="495" width="800" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270024-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genetic linkage map showing the quantitative trait loci (QTL) mapped on the chromosomes of the Awauna UEM <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-10" display="inline"><mrow><mo>×</mo></mrow></math></span> IPR88 Uirapuru population. Distances between markers are indicated in cM.</p></figcaption></figure><section class="tw xbox font-sm" id="tpg270024-tbl-0004"><h4 class="obj_head">TABLE 4.</h4>
<div class="caption p"><p>Quantitative trait loci (QTL) for agronomic traits identified in the common bean recombinant inbred line (RIL) population Awauna UEM <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-11" display="inline"><mrow><mo>×</mo></mrow></math></span> IPR88 Uirapuru cultivated under field conditions in Maringá, PR, in the agricultural years of 2017, 2018, and 2019 and combined.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">QTL</th>
<th align="left" rowspan="1" colspan="1">Year</th>
<th align="left" rowspan="1" colspan="1">Chr.</th>
<th align="left" rowspan="1" colspan="1">Peak</th>
<th align="left" rowspan="1" colspan="1">SNP left</th>
<th align="left" rowspan="1" colspan="1">SNP right</th>
<th align="left" rowspan="1" colspan="1">QTL interval</th>
<th align="left" rowspan="1" colspan="1">Nearest SNP</th>
<th align="left" rowspan="1" colspan="1">LOD</th>
<th align="left" rowspan="1" colspan="1">PVE</th>
<th align="left" rowspan="1" colspan="1">Add</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="3" align="left" colspan="1">Plant height</td>
<td align="left" rowspan="1" colspan="1">PH4<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">2017</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">ss715649973</td>
<td align="left" rowspan="1" colspan="1">ss715647823</td>
<td align="left" rowspan="1" colspan="1">4.35–12.03</td>
<td align="left" rowspan="1" colspan="1">ss715649973</td>
<td align="center" rowspan="1" colspan="1">3.33</td>
<td align="center" rowspan="1" colspan="1">2.03</td>
<td align="center" rowspan="1" colspan="1">−1.00</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PH4<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">2018</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">ss715649973</td>
<td align="left" rowspan="1" colspan="1">ss715647823</td>
<td align="left" rowspan="1" colspan="1">4.35–12.03</td>
<td align="left" rowspan="1" colspan="1">ss715649973</td>
<td align="center" rowspan="1" colspan="1">3.20</td>
<td align="center" rowspan="1" colspan="1">2.43</td>
<td align="center" rowspan="1" colspan="1">−0.89</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PH1<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">2019</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">29</td>
<td align="left" rowspan="1" colspan="1">ss715646884</td>
<td align="left" rowspan="1" colspan="1">ss715645259</td>
<td align="left" rowspan="1" colspan="1">24.09–42.43</td>
<td align="left" rowspan="1" colspan="1">ss715646884</td>
<td align="center" rowspan="1" colspan="1">3.47</td>
<td align="center" rowspan="1" colspan="1">8.88</td>
<td align="center" rowspan="1" colspan="1">0.26</td>
</tr>
<tr>
<td rowspan="4" align="left" colspan="1">No. of pods per plant</td>
<td align="left" rowspan="1" colspan="1">NPP8<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">2018</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">14</td>
<td align="left" rowspan="1" colspan="1">ss715647389</td>
<td align="left" rowspan="1" colspan="1">ss715646109</td>
<td align="left" rowspan="1" colspan="1">8.85–17.40</td>
<td align="left" rowspan="1" colspan="1">ss715646109</td>
<td align="center" rowspan="1" colspan="1">10.95</td>
<td align="center" rowspan="1" colspan="1">11.23</td>
<td align="center" rowspan="1" colspan="1">0.41</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">NPP8<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">2018</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">19</td>
<td align="left" rowspan="1" colspan="1">ss715646109</td>
<td align="left" rowspan="1" colspan="1">ss715646101</td>
<td align="left" rowspan="1" colspan="1">17.40–23.04</td>
<td align="left" rowspan="1" colspan="1">ss715646109</td>
<td align="center" rowspan="1" colspan="1">10.67</td>
<td align="center" rowspan="1" colspan="1">11.30</td>
<td align="center" rowspan="1" colspan="1">0.42</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">NPP1<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">2019</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">31</td>
<td align="left" rowspan="1" colspan="1">ss715646884</td>
<td align="left" rowspan="1" colspan="1">ss715645259</td>
<td align="left" rowspan="1" colspan="1">24.09–42.43</td>
<td align="left" rowspan="1" colspan="1">ss715646884</td>
<td align="center" rowspan="1" colspan="1">3.70</td>
<td align="center" rowspan="1" colspan="1">9.60</td>
<td align="center" rowspan="1" colspan="1">0.26</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">NPP1<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">Comb</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">37</td>
<td align="left" rowspan="1" colspan="1">ss715646884</td>
<td align="left" rowspan="1" colspan="1">ss715645259</td>
<td align="left" rowspan="1" colspan="1">24.09–42.43</td>
<td align="left" rowspan="1" colspan="1">ss715645259</td>
<td align="center" rowspan="1" colspan="1">4.72</td>
<td align="center" rowspan="1" colspan="1">10.13</td>
<td align="center" rowspan="1" colspan="1">0.53</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">First pod height</td>
<td align="left" rowspan="1" colspan="1">FPH4<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">2017</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">ss715649973</td>
<td align="left" rowspan="1" colspan="1">ss715647823</td>
<td align="left" rowspan="1" colspan="1">4.35–12.03</td>
<td align="left" rowspan="1" colspan="1">ss715649973</td>
<td align="center" rowspan="1" colspan="1">3.35</td>
<td align="center" rowspan="1" colspan="1">2.20</td>
<td align="center" rowspan="1" colspan="1">−1.03</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FPH4<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">2018</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">ss715649973</td>
<td align="left" rowspan="1" colspan="1">ss715647823</td>
<td align="left" rowspan="1" colspan="1">4.35–12.03</td>
<td align="left" rowspan="1" colspan="1">ss715649973</td>
<td align="center" rowspan="1" colspan="1">3.32</td>
<td align="center" rowspan="1" colspan="1">2.62</td>
<td align="center" rowspan="1" colspan="1">−0.93</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FPH4<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">2019</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">ss715649973</td>
<td align="left" rowspan="1" colspan="1">ss715647823</td>
<td align="left" rowspan="1" colspan="1">4.35–12.03</td>
<td align="left" rowspan="1" colspan="1">ss715649973</td>
<td align="center" rowspan="1" colspan="1">3.79</td>
<td align="center" rowspan="1" colspan="1">2.00</td>
<td align="center" rowspan="1" colspan="1">−0.79</td>
</tr>
<tr>
<td rowspan="7" align="left" colspan="1">100‐Seed weight</td>
<td align="left" rowspan="1" colspan="1">SW8<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">2017</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">21</td>
<td align="left" rowspan="1" colspan="1">ss715646109</td>
<td align="left" rowspan="1" colspan="1">ss715646101</td>
<td align="left" rowspan="1" colspan="1">17.40–23.04</td>
<td align="left" rowspan="1" colspan="1">ss715646101</td>
<td align="center" rowspan="1" colspan="1">9.61</td>
<td align="center" rowspan="1" colspan="1">14.25</td>
<td align="center" rowspan="1" colspan="1">0.48</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SW8<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">2018</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">19</td>
<td align="left" rowspan="1" colspan="1">ss715646109</td>
<td align="left" rowspan="1" colspan="1">ss715646101</td>
<td align="left" rowspan="1" colspan="1">17.40–23.04</td>
<td align="left" rowspan="1" colspan="1">ss715646109</td>
<td align="center" rowspan="1" colspan="1">12.58</td>
<td align="center" rowspan="1" colspan="1">12.67</td>
<td align="center" rowspan="1" colspan="1">0.51</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SW10<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">2018</td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">25</td>
<td align="left" rowspan="1" colspan="1">ss715645516</td>
<td align="left" rowspan="1" colspan="1">ss715650584</td>
<td align="left" rowspan="1" colspan="1">21.1–26.87</td>
<td align="left" rowspan="1" colspan="1">ss715650584</td>
<td align="center" rowspan="1" colspan="1">3.02</td>
<td align="center" rowspan="1" colspan="1">2.76</td>
<td align="center" rowspan="1" colspan="1">0.31</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SW8<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">2019</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">21</td>
<td align="left" rowspan="1" colspan="1">ss715646109</td>
<td align="left" rowspan="1" colspan="1">ss715646101</td>
<td align="left" rowspan="1" colspan="1">17.40–23.04</td>
<td align="left" rowspan="1" colspan="1">ss715646101</td>
<td align="center" rowspan="1" colspan="1">7.75</td>
<td align="center" rowspan="1" colspan="1">13.95</td>
<td align="center" rowspan="1" colspan="1">0.45</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SW8<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">Comb</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">20</td>
<td align="left" rowspan="1" colspan="1">ss715646109</td>
<td align="left" rowspan="1" colspan="1">ss715646101</td>
<td align="left" rowspan="1" colspan="1">17.40–23.04</td>
<td align="left" rowspan="1" colspan="1">ss715646109</td>
<td align="center" rowspan="1" colspan="1">8.34</td>
<td align="center" rowspan="1" colspan="1">8.27</td>
<td align="center" rowspan="1" colspan="1">0.72</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SW10<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">Comb</td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">25</td>
<td align="left" rowspan="1" colspan="1">ss715645516</td>
<td align="left" rowspan="1" colspan="1">ss715650584</td>
<td align="left" rowspan="1" colspan="1">21.1–26.87</td>
<td align="left" rowspan="1" colspan="1">ss715650584</td>
<td align="center" rowspan="1" colspan="1">2.96</td>
<td align="center" rowspan="1" colspan="1">2.95</td>
<td align="center" rowspan="1" colspan="1">0.56</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SW10.2<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">Comb</td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">35</td>
<td align="left" rowspan="1" colspan="1">ss715641827</td>
<td align="left" rowspan="1" colspan="1">ss715649351</td>
<td align="left" rowspan="1" colspan="1">29.06–55.08</td>
<td align="left" rowspan="1" colspan="1">ss715641827</td>
<td align="center" rowspan="1" colspan="1">3.41</td>
<td align="center" rowspan="1" colspan="1">7.60</td>
<td align="center" rowspan="1" colspan="1">0.88</td>
</tr>
<tr>
<td rowspan="2" align="left" colspan="1">Grain yield per plant</td>
<td align="left" rowspan="1" colspan="1">GY1<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">2019</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">32</td>
<td align="left" rowspan="1" colspan="1">ss715646884</td>
<td align="left" rowspan="1" colspan="1">ss715645259</td>
<td align="left" rowspan="1" colspan="1">24.09–42.43</td>
<td align="left" rowspan="1" colspan="1">ss715646884</td>
<td align="center" rowspan="1" colspan="1">2.98</td>
<td align="center" rowspan="1" colspan="1">7.87</td>
<td align="center" rowspan="1" colspan="1">0.23</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GY1<sup>AU</sup>
</td>
<td align="left" rowspan="1" colspan="1">Comb</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">36</td>
<td align="left" rowspan="1" colspan="1">ss715646884</td>
<td align="left" rowspan="1" colspan="1">ss715645259</td>
<td align="left" rowspan="1" colspan="1">24.09–42.43</td>
<td align="left" rowspan="1" colspan="1">ss715645259</td>
<td align="center" rowspan="1" colspan="1">2.97</td>
<td align="center" rowspan="1" colspan="1">6.78</td>
<td align="center" rowspan="1" colspan="1">0.38</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270024-tbl-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270024-tbl4-note-0001"><p>Abbreviations: Add, estimated additive effect of QTL at the current scanning position; Chr., chromosome; Comb, combination of agricultural years; LOD, logarithm of the odds score calculated from single marker analysis; Peak, peak of QTL position in cM; PVE (%), phenotypic variation explained by QTL at the current scanning position; SNP, single nucleotide polymorphism; Year, agricultural year.</p></div></div></section><p>Our study revealed the ss715646884 marker, situated at 46,961,454 bp on chromosome Pv01 associated with PH, NPP, and GYP. Similarly, marker ss715649973 (at 1,575,721 bp) on chromosome Pv04 was found in close proximity to the peak of the QTL for both PH and FPH. Noteworthy findings emerged on Pv08, where markers ss715646109 (at 58,231,277 bp) and ss715646101 (at 58,006,525 bp) were linked to the QTLs for NPP and SW. Moreover, Pv10 unveiled two distinct QTLs for SW, associated with markers ss715650584 (at 40,255,828 bp) and ss715641827 (at 16,330,477 bp). The co‐occurrence of QTLs for different traits suggests potential pleiotropy or closely linked genes (Haggard et al., <a href="#tpg270024-bib-0023" class="usa-link" aria-describedby="tpg270024-bib-0023">2015</a>). Furthermore, QTL mapping for PH, FPH, NPP, and SW revealed consistent performance across the experimental years, underscoring stability in trait expression.</p>
<section id="tpg270024-sec-0150"><h4 class="pmc_sec_title">3.5.1. QTL for PH</h4>
<p>QTLs associated with PH were detected on chromosomes Pv01 and Pv04. The peak of the QTL PH1<sup>AU</sup> on Pv01 was located at 4.91 cM from the marker ss715646884 (46,961,454 bp), based on data from the 2019 assay, explaining 8.9% of the phenotypic variation. QTL mapping on Pv04 (PH4<sup>AU</sup>) revealed consistent identification across both the 2017 and 2018 datasets, positioned at 2.65 cM from marker ss715649973 (1,575,721 bp). This QTL accounted for over 2.0% of the phenotypic variation in each year.</p></section><section id="tpg270024-sec-0160"><h4 class="pmc_sec_title">3.5.2. QTL for NPP</h4>
<p>QTLs associated with NPP were identified on chromosomes Pv01 and Pv08. The QTL on Pv01 (NPP1<sup>AU</sup>) was detected in 2019 and across the combination of agricultural years, within the interval delimited by markers ss715646884 (46,961,454 bp) and ss715645259 (50,130,201 bp). This QTL explained 9.6% and 10.1% of the phenotypic variation, respectively, in each year. Intriguingly, this QTL overlaps with the region where QTLs for PH and GYP were also found. On Pv08, the NPP8<sup>AU</sup> QTL was identified in 2018, spanning the region between markers ss715647389 (59,483,302 bp) and ss715646109 (at 58,231,277 bp). This QTL explained 11.2% and 11.3% of the phenotypic variation in each respective year.</p></section><section id="tpg270024-sec-0170"><h4 class="pmc_sec_title">3.5.3. QTL for FPH</h4>
<p>On chromosome Pv04, the QTL FPH4<sup>AU</sup> was consistently identified across all years (2017, 2018, and 2019), positioned at 2.65 cM from marker ss715649973 (1,575,721 bp). This QTL explained &gt;2.0% of the phenotypic variation in each year. Intriguingly, this QTL coincides with the QTL identified for PH.</p></section><section id="tpg270024-sec-0180"><h4 class="pmc_sec_title">3.5.4. QTL for seed weight (SW)</h4>
<p>QTLs associated with SW were detected on chromosomes Pv08 and Pv10. On Pv08, the QTL SW8<sup>AU</sup> was consistently identified across all years (2017, 2018, and 2019) and in the combination of agricultural years. It spanned the interval between markers ss715646109 (at 58,231,277 bp) and ss715646101 (at 58,006,525 bp), explaining 14.2%, 12.7%, 13.9%, and 8.3% of the phenotypic variation in each respective analysis. Notably, this QTL coincides with the QTL found for NPP. On Pv10, two distinct QTLs were identified. The QTL SW10<sup>AU</sup> was detected in 2018, and across the combination of agricultural years, it was positioned at 1.87 cM from marker ss715650584 (at 40,255,828 bp), explaining 2.7% and 2.9% of the phenotypic variation, respectively. Additionally, the QTL SW10.2<sup>AU</sup> was identified in the combination of agricultural years, spanning the interval between markers ss715641827 (at 16,330,477 bp) and ss715649351 (at 61,114,365 bp), explaining 7.6% of the phenotypic variation.</p></section><section id="tpg270024-sec-0190"><h4 class="pmc_sec_title">3.5.5. QTL for GYP</h4>
<p>The QTL GY1<sup>AU</sup>, located on chromosome Pv01 between the markers ss715646884 and ss715645259 at positions 46,961,454 bp and 50,130,201 bp, was identified both in 2019 and across the combination of agricultural years. It explains from 6.8% to 7.9% of the phenotypic variation and coincides with the region where QTLs for PH and NPP were also found.</p></section></section></section><section id="tpg270024-sec-0200"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<section id="tpg270024-sec-0210"><h3 class="pmc_sec_title">4.1. Correlation</h3>
<p>The observed genetic correlations between NPP and GYP across all years, as well as between PH and NPP, and PH and GYP in 2018 and 2019 (Table <a href="#tpg270024-tbl-0002" class="usa-link">2</a>), suggest the presence of pleiotropic loci or loci located in tightly linked regions of the genome, as noted by Haggard et al. (<a href="#tpg270024-bib-0023" class="usa-link" aria-describedby="tpg270024-bib-0023">2015</a>). This implies that a greater number of shared QTL could lead to larger values of genetic correlation (Gardner &amp; Latta, <a href="#tpg270024-bib-0020" class="usa-link" aria-describedby="tpg270024-bib-0020">2007</a>). Indeed, QTL co‐localization for correlated variables has been documented in various studies (Briñez et al., <a href="#tpg270024-bib-0006" class="usa-link" aria-describedby="tpg270024-bib-0006">2017</a>; García‐Fernández et al., <a href="#tpg270024-bib-0019" class="usa-link" aria-describedby="tpg270024-bib-0019">2021</a>; Mukeshimana et al., <a href="#tpg270024-bib-0037" class="usa-link" aria-describedby="tpg270024-bib-0037">2014</a>; Nabateregga et al., <a href="#tpg270024-bib-0039" class="usa-link" aria-describedby="tpg270024-bib-0039">2019</a>; Sedlar et al., <a href="#tpg270024-bib-0046" class="usa-link" aria-describedby="tpg270024-bib-0046">2020</a>; Trapp et al., <a href="#tpg270024-bib-0049" class="usa-link" aria-describedby="tpg270024-bib-0049">2015</a>).</p></section><section id="tpg270024-sec-0220"><h3 class="pmc_sec_title">4.2. QTL mapping</h3>
<section id="tpg270024-sec-0230"><h4 class="pmc_sec_title">4.2.1. PH trait</h4>
<p>In the study, putative QTLs were identified for the PH trait on both chromosome Pv01 (PH1<sup>AU</sup>) and chromosome Pv04 (PH1<sup>AU</sup>). Notably, on chromosome Pv01, a co‐localized QTL was discovered for PH, NPP, and GYP in close proximity to the marker ss715646884 (46,961,454 bp). This finding was supported by a significant and positive genetic correlation coefficient among these traits. Interestingly, González et al. (<a href="#tpg270024-bib-0022" class="usa-link" aria-describedby="tpg270024-bib-0022">2016</a>) also reported similar co‐localization for these three traits.</p>
<p>Stable QTLs for both PH and FPH were consistently mapped together at the beginning of chromosome Pv04, near the marker ss715649973 (1,575,721 bp). Notably, this QTL stands out for its stability across multiple years, exhibiting non‐significant QTL × E interaction (Table <a href="#tpg270024-supinfo-0001" class="usa-link">S5</a>). Furthermore, these traits demonstrated a consistent genetic correlation across all assays. It is worth mentioning that the primary genetic influence driving the increase in PH originates from the parental line Awauna UEM.</p>
<p>PH plays a crucial role in the common bean crop, influencing the formation of yield components (Fang et al., <a href="#tpg270024-bib-0018" class="usa-link" aria-describedby="tpg270024-bib-0018">2020</a>). An upright growth habit and lower susceptibility to lodging are vital factors for facilitating both manual and mechanized harvesting processes. Moreover, optimal PH facilitates cultivation practices, minimizing harvest losses and the incidence of diseases. This, in turn, allows for increased harvest frequency per year by reducing the crop cycle (Mendes et al., <a href="#tpg270024-bib-0032" class="usa-link" aria-describedby="tpg270024-bib-0032">2009</a>).</p>
<p>Several studies have consistently mapped QTLs for the PH trait on chromosomes Pv01 and Pv04 (Delfini et al., <a href="#tpg270024-bib-0013" class="usa-link" aria-describedby="tpg270024-bib-0013">2021</a>; J. C. F. Elias et al., <a href="#tpg270024-bib-0017" class="usa-link" aria-describedby="tpg270024-bib-0017">2021</a>; González et al., <a href="#tpg270024-bib-0022" class="usa-link" aria-describedby="tpg270024-bib-0022">2016</a>; Silva et al., <a href="#tpg270024-bib-0012" class="usa-link" aria-describedby="tpg270024-bib-0012">2018</a>). Additionally, Tar'an et al. (<a href="#tpg270024-bib-0048" class="usa-link" aria-describedby="tpg270024-bib-0048">2002</a>), Blair et al. (<a href="#tpg270024-bib-0004" class="usa-link" aria-describedby="tpg270024-bib-0004">2006</a>), Checa and Blair (<a href="#tpg270024-bib-0009" class="usa-link" aria-describedby="tpg270024-bib-0009">2008</a>), and Chavarro and Blair (<a href="#tpg270024-bib-0008" class="usa-link" aria-describedby="tpg270024-bib-0008">2010</a>) have reported QTLs for the PH trait on both Pv01 and Pv04 in their respective studies.</p>
<p>Indeed, comparing results across different studies can be challenging due to variations in populations and marker technologies utilized. Despite methodological differences, the consistent identification of QTLs on Pv01 across multiple studies underscores the robustness of this genetic region's association with the PH trait in common bean crops.</p></section><section id="tpg270024-sec-0240"><h4 class="pmc_sec_title">4.2.2. NPP trait</h4>
<p>In the NPP trait, putative QTLs were identified on both chromosome Pv01 (NPP1<sup>AU</sup>) and Pv08 (NPP8<sup>AU</sup>). Once more, our study revealed a QTL on Pv01, which was co‐localized with QTLs for PH, NPP, and GYP in close proximity to the marker ss715646884 (46,961,454 bp).</p>
<p>This stable QTL, NPP1<sup>AU</sup>, exceeds the contribution to the increase in NPP derived from the parental line IPR88 Uirapuru, as indicated by the additivity values. Despite exhibiting QTL × environment interaction (Table <a href="#tpg270024-supinfo-0001" class="usa-link">S5</a>), the predominant portion of the phenotypic variation is attributed to additive effects (<em>A</em>), with minimal environmental influence. This observation confirms the authenticity of the QTL, suggesting its reliability in genetic prediction across diverse environments. Consequently, markers such as ss715646884 emerge as compelling candidates for multi‐trait marker‐assisted selection strategies.</p>
<p>On chromosome Pv08, we detected a QTL co‐localized for both the NPP and SW in close proximity to marker ss715646109 (58,231,277 bp). Notably, NPP and SW exhibited a consistent genetic correlation across all assays. Additive effects predominantly influenced both traits, with the increase in NPP attributed to the parental line IPR88 Uirapuru and the increase in SW attributed to Awauna UEM. This QTL demonstrated no significant QTL × environment interaction, underscoring its importance for marker‐assisted selection strategies (Table <a href="#tpg270024-supinfo-0001" class="usa-link">S5</a>).</p>
<p>Several studies have consistently mapped QTLs for NPP trait on chromosomes Pv01 and Pv08. Mukeshimana et al. (<a href="#tpg270024-bib-0037" class="usa-link" aria-describedby="tpg270024-bib-0037">2014</a>) and Langat et al. (<a href="#tpg270024-bib-0027" class="usa-link" aria-describedby="tpg270024-bib-0027">2019</a>) mapped QTLs associated with drought tolerance in common bean and identified QTLs for NPP on Pv01. Similarly, González et al. (<a href="#tpg270024-bib-0022" class="usa-link" aria-describedby="tpg270024-bib-0022">2016</a>) and Delfini et al. (<a href="#tpg270024-bib-0013" class="usa-link" aria-describedby="tpg270024-bib-0013">2021</a>) also identified significant QTLs for NPP on Pv01. Additionally, Vargas et al. (<a href="#tpg270024-bib-0051" class="usa-link" aria-describedby="tpg270024-bib-0051">2021</a>) and Mir et al. (<a href="#tpg270024-bib-0035" class="usa-link" aria-describedby="tpg270024-bib-0035">2021</a>) also mapped QTLs for NPP on chromosomes Pv01 and Pv08. Cordoba‐Novoa et al. (<a href="#tpg270024-bib-0011" class="usa-link" aria-describedby="tpg270024-bib-0011">2022</a>) conducted a comprehensive analysis of the genetic variability and underlying architecture of traits pertinent to transpiration, stomatal regulation, and the overall water utilization of the whole plant within Mesoamerican common bean germplasm. They successfully identified specific genomic regions located on Pv01 and Pv07 that are significantly associated with critical traits, such as the mean normalized transpiration rate and the essential fraction of transpirable soil water.</p></section><section id="tpg270024-sec-0250"><h4 class="pmc_sec_title">4.2.3. FPH trait</h4>
<p>In the FPH trait, a putative stable QTL, FPH4<sup>AU</sup>, was uniquely mapped to chromosome Pv04. The primary genetic contribution to the increase in FPH was attributed to the parental line Awauna UEM. This trait demonstrated genetic correlation with other evaluated traits, across all assays, except with the NSP and GYP. Moreover, besides being stable over the years, the QTL FPH4<sup>AU</sup> identified on this chromosome for FPH, co‐localized with PH, showed no significant QTL × environment interaction. Therefore, these QTLs are highly desirable in breeding programs as they offer greater reliability for marker‐assisted selection (Table <a href="#tpg270024-tbl-0001" class="usa-link">1</a> and Table <a href="#tpg270024-supinfo-0001" class="usa-link">S5</a>).</p>
<p>The FPH trait has received considerable attention in common bean breeding programs, particularly with the modernization of crop management and the increase in large‐scale production. There is a growing need for plants with higher and more uniform pod insertion to facilitate mechanized harvesting. Despite this, relatively few studies have mapped QTLs for FPH on chromosome Pv04. Beattie et al. (<a href="#tpg270024-bib-0002" class="usa-link" aria-describedby="tpg270024-bib-0002">2003</a>), in their investigations into mapping QTLs for a common bean ideotype, identified QTLs for FPH on Pv04. Similarly, more recently, Delfini et al. (<a href="#tpg270024-bib-0013" class="usa-link" aria-describedby="tpg270024-bib-0013">2021</a>) also reported mapping QTLs for FPH on this chromosome. These findings underscore the importance of further research in this area to facilitate the development of common bean varieties with improved FPH characteristics.</p></section><section id="tpg270024-sec-0260"><h4 class="pmc_sec_title">4.2.4. Seed weight trait (SW)</h4>
<p>In SW trait, putative QTLs were identified on chromosomes Pv08 and Pv10. Specifically, on Pv08, we consistently identified a QTL for SW across all evaluated crop years, as well as in the combined analysis, co‐localized with NPP near the marker ss715646109 (58,231,277 bp). It is noteworthy that SW and NPP exhibited a genetic correlation in all assays. Both traits were primarily influenced by additive effects, with the increase in SW attributed to Awauna UEM and the increase in NPP attributed to IPR88 Uirapuru. While this QTL did show significant QTL × environment interaction, the partition of total (PVE; 12.10) indicated that additive effects (11.49) outweighed the effects of the genotype‐by‐environment interaction (0.61), confirming its importance for assisted selection (Table <a href="#tpg270024-supinfo-0001" class="usa-link">S5</a>).</p>
<p>On Pv10, we identified two QTL for SW. The SW10<sup>AU</sup>, linked to the marker ss715650584 (40,255,828 bp), exhibited QTL <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-12" display="inline"><mrow><mo>×</mo></mrow></math></span> E interaction (Table <a href="#tpg270024-supinfo-0001" class="usa-link">S5</a>); however, the majority of the phenotypic variation is attributed to additive effects (<em>A</em>) with small environment influence, and therefore confirmed that it is a true QTL. In turn, the QTL SW10.2<sup>AU</sup>, linked to the marker ss715641827 (16,330,477 bp), did not show significant QTL <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-13" display="inline"><mrow><mo>×</mo></mrow></math></span> E interaction and therefore is desirable in breeding programs, as it is more reliable for assisted selection (Table <a href="#tpg270024-tbl-0001" class="usa-link">1</a> and Table <a href="#tpg270024-supinfo-0001" class="usa-link">S5</a>). The genetic contribution to the increase in the SW derives from the parental UEM <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-14" display="inline"><mrow><mo>×</mo></mrow></math></span> IPR88 Uirapuru.</p>
<p>It is indeed quite common for various studies to map QTLs related to SW on chromosomes Pv08 and Pv10 in common bean. Silva et al. (<a href="#tpg270024-bib-0012" class="usa-link" aria-describedby="tpg270024-bib-0012">2018</a>), Kamfwa et al. (<a href="#tpg270024-bib-0024" class="usa-link" aria-describedby="tpg270024-bib-0024">2018</a>), and Nabateregga et al. (<a href="#tpg270024-bib-0039" class="usa-link" aria-describedby="tpg270024-bib-0039">2019</a>) also reported QTLs on Pv08 associated with this trait. Lei et al. (<a href="#tpg270024-bib-0028" class="usa-link" aria-describedby="tpg270024-bib-0028">2020</a>) and J. C. F. Elias et al. (<a href="#tpg270024-bib-0017" class="usa-link" aria-describedby="tpg270024-bib-0017">2021</a>) mapped QTLs for SW on Pv10. Similarly, Trapp et al. (<a href="#tpg270024-bib-0049" class="usa-link" aria-describedby="tpg270024-bib-0049">2015</a>), Sandhu et al. (<a href="#tpg270024-bib-0045" class="usa-link" aria-describedby="tpg270024-bib-0045">2018</a>), Langat et al. (<a href="#tpg270024-bib-0027" class="usa-link" aria-describedby="tpg270024-bib-0027">2019</a>), Murube et al. (<a href="#tpg270024-bib-0038" class="usa-link" aria-describedby="tpg270024-bib-0038">2020</a>), Valdisser et al. (<a href="#tpg270024-bib-0050" class="usa-link" aria-describedby="tpg270024-bib-0050">2020</a>), Vargas et al. (<a href="#tpg270024-bib-0051" class="usa-link" aria-describedby="tpg270024-bib-0051">2021</a>), and Mir et al. (<a href="#tpg270024-bib-0035" class="usa-link" aria-describedby="tpg270024-bib-0035">2021</a>) identified QTLs for SW on Pv08 and Pv10.</p>
<p>The stability observed in the majority of the identified QTLs across multiple years, coupled with their relatively modest phenotypic contribution, suggests the involvement of numerous loci with small effects in governing the expression of these traits. This finding aligns with the insights provided by Luo et al. (<a href="#tpg270024-bib-0030" class="usa-link" aria-describedby="tpg270024-bib-0030">2020</a>), indicating that the genetic architecture underlying these traits may be characterized by the cumulative impact of multiple minor effect loci.</p></section><section id="tpg270024-sec-0270"><h4 class="pmc_sec_title">4.2.5. GYP trait</h4>
<p>In the GYP trait, we identified a single putative QTL situated on chromosome Pv01. Remarkably, this QTL, designated as GY1<sup>AU</sup>, was co‐localized for GYP, PH, and NPP in close proximity to the marker ss715646884 (46,961,454 bp). The genetic correlation observed among PH, NPP, and GYP (Table <a href="#tpg270024-tbl-0002" class="usa-link">2</a>) underscores the significance of uncovering co‐located QTL governing these traits. Notably, this stable QTL, GY1<sup>AU</sup>, demonstrates a greater contribution to the enhancement of GYP, surpassing the additive effects derived from the UEM × IPR88 Uirapuru parental lines, as evidenced by the additivity values (Table <a href="#tpg270024-tbl-0004" class="usa-link">4</a>). Cordoba‐Novoa and Hoyos‐Villegas (<a href="#tpg270024-bib-0010" class="usa-link" aria-describedby="tpg270024-bib-0010">2024</a>) discerned loci correlated with agronomic characteristics such as flowering time, maturation rate, and yield in Middle American common beans, with loci located on Pv02 and Pv04 demonstrating interaction effects on both flowering time and yield. Furthermore, they underlined the significance of epistatic interactions among genomic regions to facilitate the optimal stacking of advantageous alleles aimed at enhancing yield outcomes. Importantly, the utilization of Pv03 and Pv08 as reference loci could lead to yield enhancement, while the allocation of markers on Pv01 and Pv04 as reference and alternative loci, respectively, would further augment yield efficiency.</p>
<p>Several studies have mapped QTL for grain yield on chromosome Pv01, underscoring the significance of our findings (Briñez et al., <a href="#tpg270024-bib-0006" class="usa-link" aria-describedby="tpg270024-bib-0006">2017</a>; L. M. Diaz et al., <a href="#tpg270024-bib-0014" class="usa-link" aria-describedby="tpg270024-bib-0014">2018</a>; Langat et al., <a href="#tpg270024-bib-0027" class="usa-link" aria-describedby="tpg270024-bib-0027">2019</a>; S. Diaz et al., <a href="#tpg270024-bib-0015" class="usa-link" aria-describedby="tpg270024-bib-0015">2020</a>). Additionally, research by Sedlar et al. (<a href="#tpg270024-bib-0046" class="usa-link" aria-describedby="tpg270024-bib-0046">2020</a>), Valdisser et al. (<a href="#tpg270024-bib-0050" class="usa-link" aria-describedby="tpg270024-bib-0050">2020</a>), Delfini et al. (<a href="#tpg270024-bib-0013" class="usa-link" aria-describedby="tpg270024-bib-0013">2021</a>), and Mir et al. (<a href="#tpg270024-bib-0035" class="usa-link" aria-describedby="tpg270024-bib-0035">2021</a>) has consistently mapped QTL related to grain yield on chromosome Pv01. Arriagada et al. (<a href="#tpg270024-bib-0001" class="usa-link" aria-describedby="tpg270024-bib-0001">2023</a>) executed a meta‐QTL (MQTL) analysis aimed at discerning robust genomic regions correlated with yield‐related characteristics in common bean. They elucidated that 28 MQTL were linked to both yield and phenological characteristics, underscoring their applicability for breeding programs.</p>
<p>The clustering of QTL for GYP, PH, and NPP in close proximity on Pv01 highlights the potential significance of this chromosomal region for commercial bean breeders (Geravandi et al., <a href="#tpg270024-bib-0021" class="usa-link" aria-describedby="tpg270024-bib-0021">2020</a>). Such co‐localized QTL clusters offer promising targets for marker‐assisted selection, enabling the simultaneous improvement of multiple agronomic traits and thereby enhancing overall crop productivity and performance.</p>
<p>In conclusion, this study marks a substantial progression in the genetic characterization of the RIL population derived from the Awauna UEM × IPR88 Uirapuru cross. Utilizing 288 polymorphic SNP markers and comprehensive phenotypic data, we identified consistent QTL associated with key agronomic traits. A total of five QTL linked to PH, NPP, FPH, SW, and GYP were detected across chromosomes Pv01, Pv04, Pv08, and Pv10. Notably, three QTL exhibited co‐localization for multiple traits, indicating potential pleiotropic effects or close genetic linkage. On chromosome Pv01, a QTL was identified that simultaneously influenced PH, NPP, and GYP. Similarly, chromosome Pv04 harbored QTL affecting both PH and FPH, while Pv08 contained QTL linked to NPP and SW. Finally, two distinct QTLs associated with SW were identified on Pv10. QTL mapping in individual environments successfully identified co‐located, main, and relatively stable QTL, while multi‐environment QTL mapping confirmed their relative importance. This comprehensive mapping QTL in the Awauna UEM × IPR88 Uirapuru population holds significant implications for common bean breeding programs, particularly for the Black commercial group in Brazil, which has historically received limited research attention despite its economic importance.</p></section></section></section><section id="tpg270024-sec-0280"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Deivid Lincoln Reche</strong>: Conceptualization; data curation; formal analysis; investigation; methodology; software; writing—original draft. <strong>Maria Celeste Gonçalves‐Vidigal</strong>: Conceptualization; funding acquisition; investigation; project administration; software; supervision; visualization; writing—review and editing. <strong>Pedro Soares Vidigal Filho</strong>: Conceptualization; data curation; investigation; project administration; resources; supervision; validation; visualization. <strong>Mariana Vaz Bisneta</strong>: Conceptualization; data curation; methodology; supervision; validation; visualization; writing—original draft; writing—review and editing. <strong>Giselly Figueiredo Lacanallo</strong>: Data curation; methodology; visualization. <strong>Alessandro A. Brito dos Santos</strong>: Data curation; methodology. <strong>Abner Pais dos Santos</strong>: Data curation; investigation.</p></section><section id="tpg270024-sec-0290"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec30"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270024-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270024-supitem-0001"><div class="caption p">
<p>Table S1: Phenotypic means for the evaluated traits.</p>
<p>Table S2: Variance components of the evaluated traits to the agricultural years 2017, 2018, 2019 and combination of agricultural years.</p>
<p>Table S3: Deviance analysis for the evaluated traits.</p>
<p>Table S4: Genotypic data of 208 RILs derived from the Awauna UEM × IPR88 Uirapuru cross using 5,398 SNPs.</p>
<p>Table S5: Identification of QTL by environment interactions for multi‐environment trial to the evaluated traits.</p>
<p>Figure S1: Scatter plot, frequency density curve and Pearson correlation coefficient (Corr) of the evaluated traits.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11972933/bin/TPG2-18-e70024-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70024-s001.docx</a><sup> (368.7KB, docx) </sup>
</div></div></section></section><section id="notes1"><p>


Reche, D. L.
, 
Gonçalves‐Vidigal, M. C.
, 
Vidigal Filho, P. S.
, 
Vaz Bisneta, M.
, 
Lacanallo, G. F.
, 
Santos, A. A. B.
, &amp; 
Santos, A. P.
 (2025). Genetic mapping of loci associated with yield and their components in black common bean (<em>Phaseolus vulgaris</em> L.). The Plant Genome, 18, e70024. 10.1002/tpg2.70024

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270024-note-0001"><p>Assigned to Associate Editor Roberto Tuberosa.</p></div></div></section></section><section id="tpg270024-sec-0310"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>All data generated or analyzed during this study are included in this published article.</p></section><section id="tpg270024-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270024-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270024-bib-0001">
<cite>

Arriagada, O.
, 
Arévalo, B.
, 
Cabeza, R. A.
, 
Carrasco, B.
, &amp; 
Schwember, A. R.
 (2023). Meta‐QTL analysis for yield components in common bean (<em>Phaseoulus vulgaris</em> L.). Plants, 12, 117. 10.3390/plants12010117
</cite> [<a href="https://doi.org/10.3390/plants12010117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9824219/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36616246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Meta%E2%80%90QTL%20analysis%20for%20yield%20components%20in%20common%20bean%20(Phaseoulus%20vulgaris%20L.)&amp;author=O.%20Arriagada&amp;author=B.%20Ar%C3%A9valo&amp;author=R.%20A.%20Cabeza&amp;author=B.%20Carrasco&amp;author=A.%20R.%20Schwember&amp;volume=12&amp;publication_year=2023&amp;pages=117&amp;pmid=36616246&amp;doi=10.3390/plants12010117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0002">
<cite>

Beattie, A. D.
, 
Larsen, J.
, 
Michaels, T. E.
, &amp; 
Pauls, K. P.
 (2003). Mapping quantitative trait loci for a common bean (<em>Phaseolus vulgaris</em> L.) ideotype. Genome, 46, 411–422. 10.1139/g03-015

</cite> [<a href="https://doi.org/10.1139/g03-015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12834057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome&amp;title=Mapping%20quantitative%20trait%20loci%20for%20a%20common%20bean%20(Phaseolus%20vulgaris%20L.)%20ideotype&amp;author=A.%20D.%20Beattie&amp;author=J.%20Larsen&amp;author=T.%20E.%20Michaels&amp;author=K.%20P.%20Pauls&amp;volume=46&amp;publication_year=2003&amp;pages=411-422&amp;pmid=12834057&amp;doi=10.1139/g03-015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0003">
<cite>

Bernardo, R.
 (2008). Molecular markers and selection for complex traits in plants: Learning from the last 20 years. Crop Science, 48, 1649–1664. 10.2135/cropsci2008.03.0131
</cite> [<a href="https://doi.org/10.2135/cropsci2008.03.0131" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Molecular%20markers%20and%20selection%20for%20complex%20traits%20in%20plants:%20Learning%20from%20the%20last%2020%20years&amp;author=R.%20Bernardo&amp;volume=48&amp;publication_year=2008&amp;pages=1649-1664&amp;doi=10.2135/cropsci2008.03.0131&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0004">
<cite>

Blair, M. W.
, 
Iriarte, G.
, &amp; 
Beebe, S.
 (2006). QTL analysis of yield traits in an advanced backcross population derived from a cultivated Andean × wild common bean (<em>Phaseolus vulgaris</em> L.) cross. Theoretical and Applied Genetics, 112, 1149–63. 10.1007/s00122-006-0217-2

</cite> [<a href="https://doi.org/10.1007/s00122-006-0217-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16432734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=QTL%20analysis%20of%20yield%20traits%20in%20an%20advanced%20backcross%20population%20derived%20from%20a%20cultivated%20Andean%20%C3%97%20wild%20common%20bean%20(Phaseolus%20vulgaris%20L.)%20cross&amp;author=M.%20W.%20Blair&amp;author=G.%20Iriarte&amp;author=S.%20Beebe&amp;volume=112&amp;publication_year=2006&amp;pages=1149-63&amp;pmid=16432734&amp;doi=10.1007/s00122-006-0217-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0005">
<cite>

Boopathi, N. M.
 (2020). QTL analysis. In Genetic mapping and marker assisted selection: Basics, practice and benefits (2nd ed., pp. 516). Springer. 10.1007/978-981-15-2949-8
</cite> [<a href="https://doi.org/10.1007/978-981-15-2949-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Genetic%20mapping%20and%20marker%20assisted%20selection:%20Basics,%20practice%20and%20benefits&amp;author=N.%20M.%20Boopathi&amp;publication_year=2020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0006">
<cite>

Briñez, B.
, 
Perseguini, J. M. K. C.
, 
Rosa, J. S.
, 
Bassi, D.
, 
Gonçalves, J. G. R.
, 
Almeida, C.
, 
Paulino, J. F. D. C.
, 
Blair, M. W.
, 
Chioratto, A. F.
, 
Carbonell, S. A. M.
, 
Valdisser, P. A. M. R.
, 
Vianello, R. P.
, &amp; 
Benchimol‐Reis, L. L.
 (2017). Mapping QTL for drought tolerance in a SEA 5×AND 277 common bean cross with SSRs and SNP markers. Genetics and Molecular Biology, 40, 813–823. 10.1590/1678-4685-GMB-2016-0222

</cite> [<a href="https://doi.org/10.1590/1678-4685-GMB-2016-0222" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5738610/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29064511/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20and%20Molecular%20Biology&amp;title=Mapping%20QTL%20for%20drought%20tolerance%20in%20a%20SEA%205%C3%97AND%20277%20common%20bean%20cross%20with%20SSRs%20and%20SNP%20markers&amp;author=B.%20Bri%C3%B1ez&amp;author=J.%20M.%20K.%20C.%20Perseguini&amp;author=J.%20S.%20Rosa&amp;author=D.%20Bassi&amp;author=J.%20G.%20R.%20Gon%C3%A7alves&amp;volume=40&amp;publication_year=2017&amp;pages=813-823&amp;pmid=29064511&amp;doi=10.1590/1678-4685-GMB-2016-0222&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0007">
<cite>

Celmeli, T.
, 
Sari, H.
, 
Canci, H.
, 
Sari, D.
, 
Adak, A.
, 
Eker, T.
, &amp; 
Toker, C.
 (2018). The nutritional content of common bean (<em>Phaseolus vulgaris</em> L.) landraces in comparison to modern varieties. Agronomy, 8, 166. 10.3390/agronomy8090166
</cite> [<a href="https://doi.org/10.3390/agronomy8090166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy&amp;title=The%20nutritional%20content%20of%20common%20bean%20(Phaseolus%20vulgaris%20L.)%20landraces%20in%20comparison%20to%20modern%20varieties&amp;author=T.%20Celmeli&amp;author=H.%20Sari&amp;author=H.%20Canci&amp;author=D.%20Sari&amp;author=A.%20Adak&amp;volume=8&amp;publication_year=2018&amp;pages=166&amp;doi=10.3390/agronomy8090166&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0008">
<cite>

Chavarro, M. C.
, &amp; 
Blair, M. W.
 (2010). QTL analysis and effect of the fin locus on tropical adaptation in an inter‐gene pool common bean population. Tropical Plant Biology, 3, 204–218. 10.1007/s12042-010-9058-x
</cite> [<a href="https://doi.org/10.1007/s12042-010-9058-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Tropical%20Plant%20Biology&amp;title=QTL%20analysis%20and%20effect%20of%20the%20fin%20locus%20on%20tropical%20adaptation%20in%20an%20inter%E2%80%90gene%20pool%20common%20bean%20population&amp;author=M.%20C.%20Chavarro&amp;author=M.%20W.%20Blair&amp;volume=3&amp;publication_year=2010&amp;pages=204-218&amp;doi=10.1007/s12042-010-9058-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0009">
<cite>

Checa, O. E.
, &amp; 
Blair, M. W.
 (2008). Mapping QTL for climbing ability and component traits in common bean (<em>Phaseolus vulgaris</em> L.). Molecular Breeding, 22, 201–215. 10.1007/s11032-008-9167-5
</cite> [<a href="https://doi.org/10.1007/s11032-008-9167-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=Mapping%20QTL%20for%20climbing%20ability%20and%20component%20traits%20in%20common%20bean%20(Phaseolus%20vulgaris%20L.)&amp;author=O.%20E.%20Checa&amp;author=M.%20W.%20Blair&amp;volume=22&amp;publication_year=2008&amp;pages=201-215&amp;doi=10.1007/s11032-008-9167-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0010">
<cite>
Cordoba‐Novoa &amp; Hoyos‐Villegas
. (2024). Genetic architecture of ideotype‐related traits in middle American beans through single trait, multi‐trait and epistatic genome‐wide analyses. agriRxiv, Article 20240338132. 10.31220/agriRxiv.2024.00265
</cite> [<a href="https://doi.org/10.31220/agriRxiv.2024.00265" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=agriRxiv&amp;title=Genetic%20architecture%20of%20ideotype%E2%80%90related%20traits%20in%20middle%20American%20beans%20through%20single%20trait,%20multi%E2%80%90trait%20and%20epistatic%20genome%E2%80%90wide%20analyses&amp;publication_year=2024&amp;pages=20240338132&amp;doi=10.31220/agriRxiv.2024.00265&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0011">
<cite>

Cordoba‐Novoa, H.
, 
Zhang, B.
, 
Guo, Y.
, 
Aslam, M. M.
, 
Fristchi, F. B.
, &amp; 
Hoyos‐Villegas, V.
 (2022). Whole plant transpiration responses of common bean (<em>Phaseolus vulgaris</em>L.) to drying soil: Water channels and transcription factors. BioRxiv, Article 509270. 10.1101/2022.09.23.509270
</cite> [<a href="https://doi.org/10.1101/2022.09.23.509270" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40068460/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BioRxiv&amp;title=Whole%20plant%20transpiration%20responses%20of%20common%20bean%20(Phaseolus%20vulgarisL.)%20to%20drying%20soil:%20Water%20channels%20and%20transcription%20factors&amp;author=H.%20Cordoba%E2%80%90Novoa&amp;author=B.%20Zhang&amp;author=Y.%20Guo&amp;author=M.%20M.%20Aslam&amp;author=F.%20B.%20Fristchi&amp;publication_year=2022&amp;pages=509270&amp;pmid=40068460&amp;doi=10.1101/2022.09.23.509270&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0012">
<cite>

Da Silva, L. C.
, 
De Souza, T. L. P. O.
, 
Cruz, C. D.
, 
Carneiro, P. C. S.
, 
E Silva, F. F.
, 
De Barros, E. G.
, 
Vianello, R. P.
, 
Da Fonseca, C. E. L.
, 
Song, Q.
, 
Cregan, P. B.
, &amp; 
De Souza Carneiro, J. E.
 (2018). Linkage fine‐mapping and QTL affecting morpho‐agronomic traits of a Mesoamerican × Andean RIL common bean population. Euphytica, 214, 1–15. 10.1007/s10681-018-2299-8
</cite> [<a href="https://doi.org/10.1007/s10681-018-2299-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Linkage%20fine%E2%80%90mapping%20and%20QTL%20affecting%20morpho%E2%80%90agronomic%20traits%20of%20a%20Mesoamerican%E2%80%89%C3%97%E2%80%89Andean%20RIL%20common%20bean%20population&amp;author=L.%20C.%20Da%20Silva&amp;author=T.%20L.%20P.%20O.%20De%20Souza&amp;author=C.%20D.%20Cruz&amp;author=P.%20C.%20S.%20Carneiro&amp;author=F.%20F.%20E%20Silva&amp;volume=214&amp;publication_year=2018&amp;pages=1-15&amp;doi=10.1007/s10681-018-2299-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0013">
<cite>

Delfini, J.
, 
Moda‐Cirino, V.
, 
Dos Santos Neto, J.
, 
Zeffa, D. M.
, 
Nogueira, A. F.
, 
Ribeiro, L. A. B.
, 
Ruas, P. M.
, 
Gepts, P.
, &amp; 
Gonçalves, L. S. A.
 (2021). Genome‐wide association study identifies genomic regions for important morpho‐agronomic traits in Mesoamerican common bean. Frontiers in Plant Science, 12, 748829. 10.3389/fpls.2021.748829

</cite> [<a href="https://doi.org/10.3389/fpls.2021.748829" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8528967/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34691125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genome%E2%80%90wide%20association%20study%20identifies%20genomic%20regions%20for%20important%20morpho%E2%80%90agronomic%20traits%20in%20Mesoamerican%20common%20bean&amp;author=J.%20Delfini&amp;author=V.%20Moda%E2%80%90Cirino&amp;author=J.%20Dos%20Santos%20Neto&amp;author=D.%20M.%20Zeffa&amp;author=A.%20F.%20Nogueira&amp;volume=12&amp;publication_year=2021&amp;pages=748829&amp;pmid=34691125&amp;doi=10.3389/fpls.2021.748829&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0014">
<cite>

Diaz, L. M.
, 
Ricaurte, J.
, 
Tovar, E.
, 
Cajiao, C.
, 
Terán, H.
, 
Grajales, M.
, 
Polanía, J.
, 
Rao, I.
, 
Beebe, S.
, &amp; 
Raatz, B.
 (2018). QTL analyses for tolerance to abiotic stresses in a common bean (<em>Phaseolus vulgaris</em> L.) population. PLoS One, 13, e0202342. 10.1371/journal.pone.0202342

</cite> [<a href="https://doi.org/10.1371/journal.pone.0202342" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6114847/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30157265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=QTL%20analyses%20for%20tolerance%20to%20abiotic%20stresses%20in%20a%20common%20bean%20(Phaseolus%20vulgaris%20L.)%20population&amp;author=L.%20M.%20Diaz&amp;author=J.%20Ricaurte&amp;author=E.%20Tovar&amp;author=C.%20Cajiao&amp;author=H.%20Ter%C3%A1n&amp;volume=13&amp;publication_year=2018&amp;pages=e0202342&amp;pmid=30157265&amp;doi=10.1371/journal.pone.0202342&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0015">
<cite>

Diaz, S.
, 
Ariza‐Suarez, D.
, 
Izquierdo, P.
, 
Lobaton, J. D.
, 
De La Hoz, J. F.
, 
Acevedo, F.
, 
Duitama, J.
, 
Guerrero, A. F.
, 
Cajiao, C.
, 
Mayor, V.
, 
Beebe, S. E.
, &amp; 
Raatz, B.
 (2020). Genetic mapping for agronomic traits in a MAGIC population of common bean (<em>Phaseolus vulgaris</em> L.) under drought conditions. BMC Genomics, 21, Article 799. 10.1186/s12864-020-07213-6

</cite> [<a href="https://doi.org/10.1186/s12864-020-07213-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7670608/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33198642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Genetic%20mapping%20for%20agronomic%20traits%20in%20a%20MAGIC%20population%20of%20common%20bean%20(Phaseolus%20vulgaris%20L.)%20under%20drought%20conditions&amp;author=S.%20Diaz&amp;author=D.%20Ariza%E2%80%90Suarez&amp;author=P.%20Izquierdo&amp;author=J.%20D.%20Lobaton&amp;author=J.%20F.%20De%20La%20Hoz&amp;volume=21&amp;publication_year=2020&amp;pages=799&amp;pmid=33198642&amp;doi=10.1186/s12864-020-07213-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0016">
<cite>

Elias, A. A.
, 
Robbins, K. R.
, 
Doerge, R. W.
, &amp; 
Tuinstra, M. R.
 (2016). Half a century of studying genotype by environment interactions in plant breeding experiments. Crop Science, 56, 2090–2105. 10.2135/cropsci2015.01.0061
</cite> [<a href="https://doi.org/10.2135/cropsci2015.01.0061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Half%20a%20century%20of%20studying%20genotype%20by%20environment%20interactions%20in%20plant%20breeding%20experiments&amp;author=A.%20A.%20Elias&amp;author=K.%20R.%20Robbins&amp;author=R.%20W.%20Doerge&amp;author=M.%20R.%20Tuinstra&amp;volume=56&amp;publication_year=2016&amp;pages=2090-2105&amp;doi=10.2135/cropsci2015.01.0061&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0017">
<cite>

Elias, J. C. F.
, 
Gonçalves‐Vidigal, M. C.
, 
Vaz Bisneta, M.
, 
Valentini, G.
, 
Vidigal Filho, P. S.
, 
Gilio, T. A. S.
, 
Moda‐Cirino, V.
, &amp; 
Song, Q.
 (2021). Genetic mapping for agronomic traits in IAPAR 81/LP97‐28 population of common bean (<em>Phaseolus vulgaris</em> L.) under drought conditions. Plants, 10, 1568. 10.3390/plants10081568

</cite> [<a href="https://doi.org/10.3390/plants10081568" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8400692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34451614/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Genetic%20mapping%20for%20agronomic%20traits%20in%20IAPAR%2081/LP97%E2%80%9028%20population%20of%20common%20bean%20(Phaseolus%20vulgaris%20L.)%20under%20drought%20conditions&amp;author=J.%20C.%20F.%20Elias&amp;author=M.%20C.%20Gon%C3%A7alves%E2%80%90Vidigal&amp;author=M.%20Vaz%20Bisneta&amp;author=G.%20Valentini&amp;author=P.%20S.%20Vidigal%20Filho&amp;volume=10&amp;publication_year=2021&amp;pages=1568&amp;pmid=34451614&amp;doi=10.3390/plants10081568&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0018">
<cite>

Fang, Y.
, 
Liu, S.
, 
Dong, Q.
, 
Zhang, K.
, 
Tian, Z.
, 
Li, X.
, 
Li, W.
, 
Qi, Z.
, 
Wang, Y.
, 
Tian, X.
, 
Song, J.
, 
Wang, J.
, 
Yang, C.
, 
Jiang, S.
, 
Li, W.‐X.
, &amp; 
Ning, H.
 (2020). Linkage analysis and multi‐locus genome‐wide association studies identify QTNs controlling soybean plant height. Frontiers in Plant Science, 11, Article 9. 10.3389/fpls.2020.00009

</cite> [<a href="https://doi.org/10.3389/fpls.2020.00009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7033546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32117360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Linkage%20analysis%20and%20multi%E2%80%90locus%20genome%E2%80%90wide%20association%20studies%20identify%20QTNs%20controlling%20soybean%20plant%20height&amp;author=Y.%20Fang&amp;author=S.%20Liu&amp;author=Q.%20Dong&amp;author=K.%20Zhang&amp;author=Z.%20Tian&amp;volume=11&amp;publication_year=2020&amp;pages=9&amp;pmid=32117360&amp;doi=10.3389/fpls.2020.00009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0019">
<cite>

García‐Fernández, C.
, 
Campa, A.
, 
Garzón, A. S.
, 
Miklas, P.
, &amp; 
Ferreira, J. J.
 (2021). GWAS of pod morphological and color characters in common bean. BMC Plant Biology, 21, Article 184. 10.1186/s12870-021-02967-x

</cite> [<a href="https://doi.org/10.1186/s12870-021-02967-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8053278/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33865309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=GWAS%20of%20pod%20morphological%20and%20color%20characters%20in%20common%20bean&amp;author=C.%20Garc%C3%ADa%E2%80%90Fern%C3%A1ndez&amp;author=A.%20Campa&amp;author=A.%20S.%20Garz%C3%B3n&amp;author=P.%20Miklas&amp;author=J.%20J.%20Ferreira&amp;volume=21&amp;publication_year=2021&amp;pages=184&amp;pmid=33865309&amp;doi=10.1186/s12870-021-02967-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0020">
<cite>

Gardner, K. M.
, &amp; 
Latta, R. G.
 (2007). Shared quantitative trait loci underlying the genetic correlation between continuous traits. Molecular Ecology, 16, 4195–209. 10.1111/j.1365-294X.2007.03499.x

</cite> [<a href="https://doi.org/10.1111/j.1365-294X.2007.03499.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17850272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Shared%20quantitative%20trait%20loci%20underlying%20the%20genetic%20correlation%20between%20continuous%20traits&amp;author=K.%20M.%20Gardner&amp;author=R.%20G.%20Latta&amp;volume=16&amp;publication_year=2007&amp;pages=4195-209&amp;pmid=17850272&amp;doi=10.1111/j.1365-294X.2007.03499.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0055">
<cite>

Garrett, R. D.
, 
Koh, I.
, 
Lambin, E. F.
, 
le Polain de Waroux, Y.
, 
Kastens, J. H.
, &amp; 
Brown, J. C.
 (2018). Intensification in agriculture‐forest frontiers: Land use responses to development and conservation policies in Brazil. Global Environmental Change, 53, 233–243. 10.1016/j.gloenvcha.2018.09.011
</cite> [<a href="https://doi.org/10.1016/j.gloenvcha.2018.09.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Global%20Environmental%20Change&amp;title=Intensification%20in%20agriculture%E2%80%90forest%20frontiers:%20Land%20use%20responses%20to%20development%20and%20conservation%20policies%20in%20Brazil&amp;author=R.%20D.%20Garrett&amp;author=I.%20Koh&amp;author=E.%20F.%20Lambin&amp;author=Y.%20le%20Polain%20de%20Waroux&amp;author=J.%20H.%20Kastens&amp;volume=53&amp;publication_year=2018&amp;pages=233-243&amp;doi=10.1016/j.gloenvcha.2018.09.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0021">
<cite>

Geravandi, M.
, 
Cheghamirza, K.
, 
Farshadfar, E.
, &amp; 
Gepts, P.
 (2020). QTL analysis of seed size and yield‐related traits in an inter‐genepool population of common bean (<em>Phaseolus vulgaris</em> L.). Scientia Horticulturae, 274, 109678. 10.1016/j.scienta.2020.109678
</cite> [<a href="https://doi.org/10.1016/j.scienta.2020.109678" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientia%20Horticulturae&amp;title=QTL%20analysis%20of%20seed%20size%20and%20yield%E2%80%90related%20traits%20in%20an%20inter%E2%80%90genepool%20population%20of%20common%20bean%20(Phaseolus%20vulgaris%20L.)&amp;author=M.%20Geravandi&amp;author=K.%20Cheghamirza&amp;author=E.%20Farshadfar&amp;author=P.%20Gepts&amp;volume=274&amp;publication_year=2020&amp;pages=109678&amp;doi=10.1016/j.scienta.2020.109678&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0022">
<cite>

González, A. M.
, 
Yuste‐Lisbona, F. J.
, 
Saburido, S.
, 
Bretones, S.
, 
De Ron, A. M.
, 
Lozano, R.
, &amp; 
Santalla, M.
 (2016). Major contribution of flowering time and vegetative growth to plant production in common bean as deduced from a comparative genetic mapping. Frontiers in Plant Science, 7, 1940. 10.1016/j.scienta.2020.109678

</cite> [<a href="https://doi.org/10.1016/j.scienta.2020.109678" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5183638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28082996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Major%20contribution%20of%20flowering%20time%20and%20vegetative%20growth%20to%20plant%20production%20in%20common%20bean%20as%20deduced%20from%20a%20comparative%20genetic%20mapping&amp;author=A.%20M.%20Gonz%C3%A1lez&amp;author=F.%20J.%20Yuste%E2%80%90Lisbona&amp;author=S.%20Saburido&amp;author=S.%20Bretones&amp;author=A.%20M.%20De%20Ron&amp;volume=7&amp;publication_year=2016&amp;pages=1940&amp;pmid=28082996&amp;doi=10.1016/j.scienta.2020.109678&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0023">
<cite>

Haggard, J. E.
, 
Johnson, E. B.
, &amp; 
St Clair, D. A.
 (2015). Multiple QTL for horticultural traits and quantitative resistance to <em>Phytophthora infestans</em> linked on <em>Solanum habrochaites</em> chromosome 11. G3 Genes|Genomes|Genetics, 5, 2219–233. 10.1534/g3.114.014654

</cite> [<a href="https://doi.org/10.1534/g3.114.014654" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4321030/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25504736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20Genes%7CGenomes%7CGenetics&amp;title=Multiple%20QTL%20for%20horticultural%20traits%20and%20quantitative%20resistance%20to%20Phytophthora%20infestans%20linked%20on%20Solanum%20habrochaites%20chromosome%2011&amp;author=J.%20E.%20Haggard&amp;author=E.%20B.%20Johnson&amp;author=D.%20A.%20St%20Clair&amp;volume=5&amp;publication_year=2015&amp;pages=2219-233&amp;pmid=25504736&amp;doi=10.1534/g3.114.014654&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0056">
<cite>

Hoyos‐Villegas, V.
, 
Song, Q.
, 
Wright, E. M.
, 
Beebe, S. E.
, &amp; 
Kelly, J. D.
 (2016). Joint linkage QTL mapping for yield and agronomic traits in a composite map of three common bean RIL populations. Crop Science, 56(5), 2546–2563. 10.2135/cropsci2016.01.0063
</cite> [<a href="https://doi.org/10.2135/cropsci2016.01.0063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Joint%20linkage%20QTL%20mapping%20for%20yield%20and%20agronomic%20traits%20in%20a%20composite%20map%20of%20three%20common%20bean%20RIL%20populations&amp;author=V.%20Hoyos%E2%80%90Villegas&amp;author=Q.%20Song&amp;author=E.%20M.%20Wright&amp;author=S.%20E.%20Beebe&amp;author=J.%20D.%20Kelly&amp;volume=56&amp;issue=5&amp;publication_year=2016&amp;pages=2546-2563&amp;doi=10.2135/cropsci2016.01.0063&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0024">
<cite>

Kamfwa, K.
, 
Beaver, J. S.
, 
Cichy, K. A.
, &amp; 
Kelly, J. D.
 (2018). QTL mapping of resistance to bean weevil in common bean. Crop Science, 58, 2370–2378. 10.2135/cropsci2018.02.0106
</cite> [<a href="https://doi.org/10.2135/cropsci2018.02.0106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=QTL%20mapping%20of%20resistance%20to%20bean%20weevil%20in%20common%20bean&amp;author=K.%20Kamfwa&amp;author=J.%20S.%20Beaver&amp;author=K.%20A.%20Cichy&amp;author=J.%20D.%20Kelly&amp;volume=58&amp;publication_year=2018&amp;pages=2370-2378&amp;doi=10.2135/cropsci2018.02.0106&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0025">
<cite>

Kotue, T. C.
, 
Marlyne, J. M.
, &amp; 
Wirba, L. Y.
 (2018). Nutritional properties and nutrients chemical analysis of common beans seed. MOJ Biology and Medicine, 3, 41–47. 10.15406/mojbm.2018.03.00074
</cite> [<a href="https://doi.org/10.15406/mojbm.2018.03.00074" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=MOJ%20Biology%20and%20Medicine&amp;title=Nutritional%20properties%20and%20nutrients%20chemical%20analysis%20of%20common%20beans%20seed&amp;author=T.%20C.%20Kotue&amp;author=J.%20M.%20Marlyne&amp;author=L.%20Y.%20Wirba&amp;volume=3&amp;publication_year=2018&amp;pages=41-47&amp;doi=10.15406/mojbm.2018.03.00074&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0026">
<cite>

Kromdijk, J.
, 
Bertin, N.
, 
Heuvelink, E.
, 
Molenaar, J.
, 
De Visser, P. H. B.
, 
Marcelis, L. F. M.
, &amp; 
Struik, P. C.
 (2014). Crop management impacts the efficiency of quantitative trait loci (QTL) detection and use: Case study of fruit load x QTL interactions. Journal of Experimental Botany, 65, 11–22. 10.1093/jxb/ert365

</cite> [<a href="https://doi.org/10.1093/jxb/ert365" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24227339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Crop%20management%20impacts%20the%20efficiency%20of%20quantitative%20trait%20loci%20(QTL)%20detection%20and%20use:%20Case%20study%20of%20fruit%20load%20x%20QTL%20interactions&amp;author=J.%20Kromdijk&amp;author=N.%20Bertin&amp;author=E.%20Heuvelink&amp;author=J.%20Molenaar&amp;author=P.%20H.%20B.%20De%20Visser&amp;volume=65&amp;publication_year=2014&amp;pages=11-22&amp;pmid=24227339&amp;doi=10.1093/jxb/ert365&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0027">
<cite>

Langat, C. K.
, 
Ombori, O.
, 
Cheruiyot, R.
, 
Gathaara, M.
, 
Karanja, D.
, &amp; 
Leley, P.
 (2019). Mapping of quantitative trait loci (QTL) related to drought tolerance in common bean (<em>Phaseolus vulgaris</em> L.) using F<sub>2</sub> population from (KATB1×GLP2). International Annals of Science, 8, 75–86. 10.21467/ias.8.1.75-86
</cite> [<a href="https://doi.org/10.21467/ias.8.1.75-86" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Annals%20of%20Science&amp;title=Mapping%20of%20quantitative%20trait%20loci%20(QTL)%20related%20to%20drought%20tolerance%20in%20common%20bean%20(Phaseolus%20vulgaris%20L.)%20using%20F2%20population%20from%20(KATB1%C3%97GLP2)&amp;author=C.%20K.%20Langat&amp;author=O.%20Ombori&amp;author=R.%20Cheruiyot&amp;author=M.%20Gathaara&amp;author=D.%20Karanja&amp;volume=8&amp;publication_year=2019&amp;pages=75-86&amp;doi=10.21467/ias.8.1.75-86&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0028">
<cite>

Lei, L.
, 
Wang, L.
, 
Wang, S.
, &amp; 
Wu, J.
 (2020). Marker‐trait association analysis of seed traits in accessions of common bean (<em>Phaseolus vulgaris</em> L.) in China. Frontiers in Genetics, 11, Article 698. 10.3389/fgene.2020.00698

</cite> [<a href="https://doi.org/10.3389/fgene.2020.00698" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7344293/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32714377/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Marker%E2%80%90trait%20association%20analysis%20of%20seed%20traits%20in%20accessions%20of%20common%20bean%20(Phaseolus%20vulgaris%20L.)%20in%20China&amp;author=L.%20Lei&amp;author=L.%20Wang&amp;author=S.%20Wang&amp;author=J.%20Wu&amp;volume=11&amp;publication_year=2020&amp;pages=698&amp;pmid=32714377&amp;doi=10.3389/fgene.2020.00698&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0029">
<cite>

Li, S.
, 
Wang, J.
, &amp; 
Zhang, L.
 (2015). Inclusive composite interval mapping of QTL by environment interactions in biparental populations. PLoS One, 10, e0132414. 10.1371/journal.pone.0132414

</cite> [<a href="https://doi.org/10.1371/journal.pone.0132414" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4498613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26161656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Inclusive%20composite%20interval%20mapping%20of%20QTL%20by%20environment%20interactions%20in%20biparental%20populations&amp;author=S.%20Li&amp;author=J.%20Wang&amp;author=L.%20Zhang&amp;volume=10&amp;publication_year=2015&amp;pages=e0132414&amp;pmid=26161656&amp;doi=10.1371/journal.pone.0132414&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0030">
<cite>

Luo, Z.
, 
Cui, R.
, 
Chavarro, C.
, 
Tseng, Y.
, 
Zhou, H.
, 
Peng, Z.
, 
Chu, Y.
, 
Yang, X.
, 
Lopez, Y.
, 
Tillman, B.
, 
Dufault, N.
, 
Brenneman, T.
, 
Isleib, T. G.
, 
Holbrook, C.
, 
Ozias‐Akins, P.
, &amp; 
Wang, J.
 (2020). Mapping quantitative trait loci (QTL) and estimating the epistasis controlling stem rot resistance in cultivated peanut (<em>Arachis hypogaea</em>). Theoretical and Applied Genetics, 133, 1201–1212. 10.1007/s00122-020-03542-y

</cite> [<a href="https://doi.org/10.1007/s00122-020-03542-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31974667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Mapping%20quantitative%20trait%20loci%20(QTL)%20and%20estimating%20the%20epistasis%20controlling%20stem%20rot%20resistance%20in%20cultivated%20peanut%20(Arachis%20hypogaea)&amp;author=Z.%20Luo&amp;author=R.%20Cui&amp;author=C.%20Chavarro&amp;author=Y.%20Tseng&amp;author=H.%20Zhou&amp;volume=133&amp;publication_year=2020&amp;pages=1201-1212&amp;pmid=31974667&amp;doi=10.1007/s00122-020-03542-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0031">
<cite>

Mackay, T. F.
, 
Stone, E. A.
, &amp; 
Ayroles, J. F.
 (2009). The genetics of quantitative traits: Challenges and prospects. Nature Reviews Genetics, 10, 565–77. 10.1038/nrg2612
</cite> [<a href="https://doi.org/10.1038/nrg2612" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19584810/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Genetics&amp;title=The%20genetics%20of%20quantitative%20traits:%20Challenges%20and%20prospects&amp;author=T.%20F.%20Mackay&amp;author=E.%20A.%20Stone&amp;author=J.%20F.%20Ayroles&amp;volume=10&amp;publication_year=2009&amp;pages=565-77&amp;pmid=19584810&amp;doi=10.1038/nrg2612&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0032">
<cite>

Mendes, F. F.
, 
Ramalho, M. A. P.
, &amp; 
Abreu, F. B. A.
 (2009). Índice de seleção para escolha de populações segregantes de feijoeiro comum. Pesquisa Agropecuária Brasileira, 44, 1312–1318. 10.1590/S0100-204x2009001000015
</cite> [<a href="https://doi.org/10.1590/S0100-204x2009001000015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pesquisa%20Agropecu%C3%A1ria%20Brasileira&amp;title=%C3%8Dndice%20de%20sele%C3%A7%C3%A3o%20para%20escolha%20de%20popula%C3%A7%C3%B5es%20segregantes%20de%20feijoeiro%20comum&amp;author=F.%20F.%20Mendes&amp;author=M.%20A.%20P.%20Ramalho&amp;author=F.%20B.%20A.%20Abreu&amp;volume=44&amp;publication_year=2009&amp;pages=1312-1318&amp;doi=10.1590/S0100-204x2009001000015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0033">
<cite>

Meng, L.
, 
Li, H.
, 
Zhang, L.
, &amp; 
Wang, J.
 (2015). QTL IciMapping: Integrated software for genetic linkage map construction and quantitative trait locus mapping in biparental populations. The Crop Journal, 3, 269–283. 10.1016/j.cj.2015.01.001
</cite> [<a href="https://doi.org/10.1016/j.cj.2015.01.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Crop%20Journal&amp;title=QTL%20IciMapping:%20Integrated%20software%20for%20genetic%20linkage%20map%20construction%20and%20quantitative%20trait%20locus%20mapping%20in%20biparental%20populations&amp;author=L.%20Meng&amp;author=H.%20Li&amp;author=L.%20Zhang&amp;author=J.%20Wang&amp;volume=3&amp;publication_year=2015&amp;pages=269-283&amp;doi=10.1016/j.cj.2015.01.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0034">
<cite>

Miklas, P. N.
, &amp; 
Porch, T.
 (2010). Guidelines for common bean QTL nomenclature. Annual Report of the Bean Improvement Cooperative, 53, 202–204.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Report%20of%20the%20Bean%20Improvement%20Cooperative&amp;title=Guidelines%20for%20common%20bean%20QTL%20nomenclature&amp;author=P.%20N.%20Miklas&amp;author=T.%20Porch&amp;volume=53&amp;publication_year=2010&amp;pages=202-204&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0035">
<cite>

Mir, R. R.
, 
Choudhary, N.
, 
Bawa, V.
, 
Jan, S.
, 
Singh, B.
, 
Bhat, M. A.
, 
Paliwal, R.
, 
Kumar, A.
, 
Chitikineni, A.
, 
Thudi, M.
, &amp; 
Varshney, R. K.
 (2021). Allelic diversity, structural analysis, and Genome‐wide association study (GWAS) for yield and related traits using unexplored common bean (<em>Phaseolus vulgaris</em> L.) germplasm from Western Himalayas. Frontiers in Genetics, 11, Article 609603. 10.3389/fgene.2020.609603

</cite> [<a href="https://doi.org/10.3389/fgene.2020.609603" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7876396/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33584807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Allelic%20diversity,%20structural%20analysis,%20and%20Genome%E2%80%90wide%20association%20study%20(GWAS)%20for%20yield%20and%20related%20traits%20using%20unexplored%20common%20bean%20(Phaseolus%20vulgaris%20L.)%20germplasm%20from%20Western%20Himalayas&amp;author=R.%20R.%20Mir&amp;author=N.%20Choudhary&amp;author=V.%20Bawa&amp;author=S.%20Jan&amp;author=B.%20Singh&amp;volume=11&amp;publication_year=2021&amp;pages=609603&amp;pmid=33584807&amp;doi=10.3389/fgene.2020.609603&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0036">
<cite>

Moda‐Cirino, V.
, 
Oliari, L.
, 
Lollato, M. A.
, &amp; 
Fonseca Júnior, N. S.
 (2001). IPR 88 Uirapuru: Common bean. Crop Breeding and Applied Biotechnology, 1, 205–206.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Breeding%20and%20Applied%20Biotechnology&amp;title=IPR%2088%20Uirapuru:%20Common%20bean&amp;author=V.%20Moda%E2%80%90Cirino&amp;author=L.%20Oliari&amp;author=M.%20A.%20Lollato&amp;author=N.%20S.%20Fonseca%20J%C3%BAnior&amp;volume=1&amp;publication_year=2001&amp;pages=205-206&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0037">
<cite>

Mukeshimana, G.
, 
Butare, L.
, 
Cregan, P. B.
, 
Blair, M. W.
, &amp; 
Kelly, J. D.
 (2014). Quantitative trait loci associated with drought tolerance in common bean. Crop Science, 54, 923–938. 10.2135/cropsci2013.06.0427
</cite> [<a href="https://doi.org/10.2135/cropsci2013.06.0427" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Quantitative%20trait%20loci%20associated%20with%20drought%20tolerance%20in%20common%20bean&amp;author=G.%20Mukeshimana&amp;author=L.%20Butare&amp;author=P.%20B.%20Cregan&amp;author=M.%20W.%20Blair&amp;author=J.%20D.%20Kelly&amp;volume=54&amp;publication_year=2014&amp;pages=923-938&amp;doi=10.2135/cropsci2013.06.0427&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0038">
<cite>

Murube, E.
, 
Campa, A.
, 
Song, Q.
, 
McClean, P.
, &amp; 
Ferreira, J. J.
 (2020). Toward validation of QTL associated with pod and seed size in common bean using two nested recombinant inbred line populations. Molecular Breeding, 40, Article 70. 10.1007/s11032-020-01155-3
</cite> [<a href="https://doi.org/10.1007/s11032-020-01155-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=Toward%20validation%20of%20QTL%20associated%20with%20pod%20and%20seed%20size%20in%20common%20bean%20using%20two%20nested%20recombinant%20inbred%20line%20populations&amp;author=E.%20Murube&amp;author=A.%20Campa&amp;author=Q.%20Song&amp;author=P.%20McClean&amp;author=J.%20J.%20Ferreira&amp;volume=40&amp;publication_year=2020&amp;pages=70&amp;doi=10.1007/s11032-020-01155-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0039">
<cite>

Nabateregga, M.
, 
Mukankusi, C.
, 
Raatz, B.
, 
Edema, R.
, 
Nkalubo, S.
, &amp; 
Alladassi, B. M. E.
 (2019). Quantitative trait loci (QTL) mapping for intermittent drought tolerance in BRB 191 × SEQ 1027 Andean intragene cross recombinant inbred line population of common bean (<em>Phaseolus vulgaris</em> L.). African Journal of Biotechnology, 18, 452–461. 10.5897/AJB2019.16768
</cite> [<a href="https://doi.org/10.5897/AJB2019.16768" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7691753/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33281891/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=African%20Journal%20of%20Biotechnology&amp;title=Quantitative%20trait%20loci%20(QTL)%20mapping%20for%20intermittent%20drought%20tolerance%20in%20BRB%20191%20%C3%97%20SEQ%201027%20Andean%20intragene%20cross%20recombinant%20inbred%20line%20population%20of%20common%20bean%20(Phaseolus%20vulgaris%20L.)&amp;author=M.%20Nabateregga&amp;author=C.%20Mukankusi&amp;author=B.%20Raatz&amp;author=R.%20Edema&amp;author=S.%20Nkalubo&amp;volume=18&amp;publication_year=2019&amp;pages=452-461&amp;pmid=33281891&amp;doi=10.5897/AJB2019.16768&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0040">
<cite>

Njau, S. N.
, 
Parker, T. A.
, 
Duitama, J.
, 
Gepts, P.
, &amp; 
Arunga, E. E.
 (2024). QTL mapping for pod quality and yield traits in snap bean (<em>Phaseolus vulgaris</em> L.). Frontiers in Plant Science, 15, 1422957. 10.3389/fpls.2024.1422957

</cite> [<a href="https://doi.org/10.3389/fpls.2024.1422957" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11345156/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39188542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=QTL%20mapping%20for%20pod%20quality%20and%20yield%20traits%20in%20snap%20bean%20(Phaseolus%20vulgaris%20L.)&amp;author=S.%20N.%20Njau&amp;author=T.%20A.%20Parker&amp;author=J.%20Duitama&amp;author=P.%20Gepts&amp;author=E.%20E.%20Arunga&amp;volume=15&amp;publication_year=2024&amp;pages=1422957&amp;pmid=39188542&amp;doi=10.3389/fpls.2024.1422957&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0041">
<cite>

Pavličev, M.
, &amp; 
Cheverud, J. M.
 (2015). Constraints evolve: Context dependency of gene effects allows evolution of pleiotropy. Annual Review of Ecology, Evolution and Systematics, 46, 413–434. 10.1146/annurev-ecolsys-120213-091721
</cite> [<a href="https://doi.org/10.1146/annurev-ecolsys-120213-091721" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Ecology,%20Evolution%20and%20Systematics&amp;title=Constraints%20evolve:%20Context%20dependency%20of%20gene%20effects%20allows%20evolution%20of%20pleiotropy&amp;author=M.%20Pavli%C4%8Dev&amp;author=J.%20M.%20Cheverud&amp;volume=46&amp;publication_year=2015&amp;pages=413-434&amp;doi=10.1146/annurev-ecolsys-120213-091721&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0042">
<cite>
R Core Team
. (2013). R: A language and environment for statistical computing. R Foundation for Statistical Computing. <a href="http://www.R-project.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.R‐project.org</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=R:%20A%20language%20and%20environment%20for%20statistical%20computing&amp;publication_year=2013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0043">
<cite>

Resende, M. D. V.
 (2016). Software selegen‐REML/BLUP: A useful tool for plant breeding. Crop Breeding and Applied Biotechnology, 16, 330–339. 10.1590/1984-70332016v16n4a49
</cite> [<a href="https://doi.org/10.1590/1984-70332016v16n4a49" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Breeding%20and%20Applied%20Biotechnology&amp;title=Software%20selegen%E2%80%90REML/BLUP:%20A%20useful%20tool%20for%20plant%20breeding&amp;author=M.%20D.%20V.%20Resende&amp;volume=16&amp;publication_year=2016&amp;pages=330-339&amp;doi=10.1590/1984-70332016v16n4a49&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0044">
<cite>

Resende, M. D. V.
, &amp; 
Duarte, J. B.
 (2007). Precisão e controle de qualidade em experimentos de avaliação de cultivares. Pesquisa Agropecuária Tropical, 37, 182–194.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Pesquisa%20Agropecu%C3%A1ria%20Tropical&amp;title=Precis%C3%A3o%20e%20controle%20de%20qualidade%20em%20experimentos%20de%20avalia%C3%A7%C3%A3o%20de%20cultivares&amp;author=M.%20D.%20V.%20Resende&amp;author=J.%20B.%20Duarte&amp;volume=37&amp;publication_year=2007&amp;pages=182-194&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0045">
<cite>

Sandhu, K. S.
, 
You, F. M.
, 
Conner, R. L.
, 
Balasubramanian, P. M.
, &amp; 
Hou, A.
 (2018). Genetic analysis and QTL mapping of the seed hardness trait in a black common bean (<em>Phaseolus vulgaris</em>) recombinant inbred line (RIL) population. Molecular Breeding, 38, 1–13. 10.1007/s11032-018-0789-y
</cite> [<a href="https://doi.org/10.1007/s11032-018-0789-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5842266/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29568229/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=Genetic%20analysis%20and%20QTL%20mapping%20of%20the%20seed%20hardness%20trait%20in%20a%20black%20common%20bean%20(Phaseolus%20vulgaris)%20recombinant%20inbred%20line%20(RIL)%20population&amp;author=K.%20S.%20Sandhu&amp;author=F.%20M.%20You&amp;author=R.%20L.%20Conner&amp;author=P.%20M.%20Balasubramanian&amp;author=A.%20Hou&amp;volume=38&amp;publication_year=2018&amp;pages=1-13&amp;pmid=29568229&amp;doi=10.1007/s11032-018-0789-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0046">
<cite>

Sedlar, A.
, 
Zupin, M.
, 
Maras, M.
, 
Razinger, J.
, 
Šuštar‐Vozlič, J.
, 
Pipan, B.
, &amp; 
Meglič, V.
 (2020). QTL Mapping for drought‐responsive agronomic traits associated with physiology, phenology, and yield in an andean intra‐gene pool common bean population. Agronomy, 10, 225. 10.3390/agronomy10020225
</cite> [<a href="https://doi.org/10.3390/agronomy10020225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy&amp;title=QTL%20Mapping%20for%20drought%E2%80%90responsive%20agronomic%20traits%20associated%20with%20physiology,%20phenology,%20and%20yield%20in%20an%20andean%20intra%E2%80%90gene%20pool%20common%20bean%20population&amp;author=A.%20Sedlar&amp;author=M.%20Zupin&amp;author=M.%20Maras&amp;author=J.%20Razinger&amp;author=J.%20%C5%A0u%C5%A1tar%E2%80%90Vozli%C4%8D&amp;volume=10&amp;publication_year=2020&amp;pages=225&amp;doi=10.3390/agronomy10020225&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0047">
<cite>

Singamsetti, A.
, 
Shahi, J. P.
, 
Zaidi, P. H.
, 
Seetharam, K.
, 
Vinayan, M. T.
, 
Kumar, M.
, 
Singla, S.
, 
Shikha, K.
, &amp; 
Madankar, K.
 (2021). Genotype × environment interaction and selection of maize (<em>Zea mays</em> L.) hybrids across moisture regimes. Field Crops Research, 270, 108224. 10.1016/j.fcr.2021.108224
</cite> [<a href="https://doi.org/10.1016/j.fcr.2021.108224" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Field%20Crops%20Research&amp;title=Genotype%20%C3%97%20environment%20interaction%20and%20selection%20of%20maize%20(Zea%20mays%20L.)%20hybrids%20across%20moisture%20regimes&amp;author=A.%20Singamsetti&amp;author=J.%20P.%20Shahi&amp;author=P.%20H.%20Zaidi&amp;author=K.%20Seetharam&amp;author=M.%20T.%20Vinayan&amp;volume=270&amp;publication_year=2021&amp;pages=108224&amp;doi=10.1016/j.fcr.2021.108224&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0048">
<cite>

Tar'an, B.
, 
Michaels, T. E.
, &amp; 
Pauls, K. P.
 (2002). Genetic mapping of agronomic traits in common bean. Crop Science, 42, 544–556. 10.2135/cropsci2002.5440
</cite> [<a href="https://doi.org/10.2135/cropsci2002.5440" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genetic%20mapping%20of%20agronomic%20traits%20in%20common%20bean&amp;author=B.%20Tar'an&amp;author=T.%20E.%20Michaels&amp;author=K.%20P.%20Pauls&amp;volume=42&amp;publication_year=2002&amp;pages=544-556&amp;doi=10.2135/cropsci2002.5440&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0049">
<cite>

Trapp, J. J.
, 
Urrea, C. A.
, 
Cregan, P. B.
, &amp; 
Miklas, P. N.
 (2015). Quantitative trait loci for yield under multiple stress and drought conditions in a dry bean population. Crop Science, 55, 1596–1607. 10.2135/cropsci2014.11.0792
</cite> [<a href="https://doi.org/10.2135/cropsci2014.11.0792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Quantitative%20trait%20loci%20for%20yield%20under%20multiple%20stress%20and%20drought%20conditions%20in%20a%20dry%20bean%20population&amp;author=J.%20J.%20Trapp&amp;author=C.%20A.%20Urrea&amp;author=P.%20B.%20Cregan&amp;author=P.%20N.%20Miklas&amp;volume=55&amp;publication_year=2015&amp;pages=1596-1607&amp;doi=10.2135/cropsci2014.11.0792&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0050">
<cite>

Valdisser, P. A. M. R.
, 
Müller, B. S. F.
, 
De Almeida Filho, J. E.
, 
Morais Júnior, O. P.
, 
Guimarães, C. M.
, 
Borba, T. C. O.
, 
De Souza, I. P.
, 
Zucchi, M. I.
, 
Neves, L. G.
, 
Coelho, A. S. G.
, 
Brondani, C.
, &amp; 
Vianello, R. P.
 (2020). Genome‐Wide association studies detect multiple QTL for productivity in mesoamerican diversity panel of common bean under drought stress. Frontiers in Plant Science, 11, 574674. 10.3389/fpls.2020.574674

</cite> [<a href="https://doi.org/10.3389/fpls.2020.574674" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7738703/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33343591/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genome%E2%80%90Wide%20association%20studies%20detect%20multiple%20QTL%20for%20productivity%20in%20mesoamerican%20diversity%20panel%20of%20common%20bean%20under%20drought%20stress&amp;author=P.%20A.%20M.%20R.%20Valdisser&amp;author=B.%20S.%20F.%20M%C3%BCller&amp;author=J.%20E.%20De%20Almeida%20Filho&amp;author=O.%20P.%20Morais%20J%C3%BAnior&amp;author=C.%20M.%20Guimar%C3%A3es&amp;volume=11&amp;publication_year=2020&amp;pages=574674&amp;pmid=33343591&amp;doi=10.3389/fpls.2020.574674&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0051">
<cite>

Vargas, Y.
, 
Mayor‐Duran, V. M.
, 
Buendia, H. F.
, 
Ruiz‐Guzman, H.
, &amp; 
Raatz, B.
 (2021). Physiological and genetic characterization of heat stress effects in a common bean RIL population. PLoS One, 16, e0249859. 10.1371/journal.pone.0249859

</cite> [<a href="https://doi.org/10.1371/journal.pone.0249859" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8084131/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33914759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Physiological%20and%20genetic%20characterization%20of%20heat%20stress%20effects%20in%20a%20common%20bean%20RIL%20population&amp;author=Y.%20Vargas&amp;author=V.%20M.%20Mayor%E2%80%90Duran&amp;author=H.%20F.%20Buendia&amp;author=H.%20Ruiz%E2%80%90Guzman&amp;author=B.%20Raatz&amp;volume=16&amp;publication_year=2021&amp;pages=e0249859&amp;pmid=33914759&amp;doi=10.1371/journal.pone.0249859&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0052">
<cite>

Voorrips, R. E.
 (2002). Mapchart: Software for the graphical presentation of linkage maps and QTL. Journal of Heredity, 93, 77–78. 10.1093/jhered/93.1.77

</cite> [<a href="https://doi.org/10.1093/jhered/93.1.77" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12011185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Heredity&amp;title=Mapchart:%20Software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTL&amp;author=R.%20E.%20Voorrips&amp;volume=93&amp;publication_year=2002&amp;pages=77-78&amp;pmid=12011185&amp;doi=10.1093/jhered/93.1.77&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0053">
<cite>

Wickham, H.
 (2016). Package “ggplot2”: Elegant graphics for data analysis. Springer‐Verlag.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Package%20%E2%80%9Cggplot2%E2%80%9D:%20Elegant%20graphics%20for%20data%20analysis&amp;author=H.%20Wickham&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270024-bib-0054">
<cite>

Yuste‐Lisbona, F. J.
, 
González, A. M.
, 
Capel, C.
, 
García‐Alcázar, M.
, 
Capel, J.
, 
De Ron, A. M.
, 
Lozano, R.
, &amp; 
Santalla, M.
 (2014). Genetic analysis of single‐locus and epistatic QTL for seed traits in an adapted nuña RIL population of common bean (<em>Phaseolus vulgaris</em> L.). Theoretical and Applied Genetics, 127, 897–912. 10.1007/s00122-014-2265-3

</cite> [<a href="https://doi.org/10.1007/s00122-014-2265-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24441949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Genetic%20analysis%20of%20single%E2%80%90locus%20and%20epistatic%20QTL%20for%20seed%20traits%20in%20an%20adapted%20nu%C3%B1a%20RIL%20population%20of%20common%20bean%20(Phaseolus%20vulgaris%20L.)&amp;author=F.%20J.%20Yuste%E2%80%90Lisbona&amp;author=A.%20M.%20Gonz%C3%A1lez&amp;author=C.%20Capel&amp;author=M.%20Garc%C3%ADa%E2%80%90Alc%C3%A1zar&amp;author=J.%20Capel&amp;volume=127&amp;publication_year=2014&amp;pages=897-912&amp;pmid=24441949&amp;doi=10.1007/s00122-014-2265-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<p>Table S1: Phenotypic means for the evaluated traits.</p>
<p>Table S2: Variance components of the evaluated traits to the agricultural years 2017, 2018, 2019 and combination of agricultural years.</p>
<p>Table S3: Deviance analysis for the evaluated traits.</p>
<p>Table S4: Genotypic data of 208 RILs derived from the Awauna UEM × IPR88 Uirapuru cross using 5,398 SNPs.</p>
<p>Table S5: Identification of QTL by environment interactions for multi‐environment trial to the evaluated traits.</p>
<p>Figure S1: Scatter plot, frequency density curve and Pearson correlation coefficient (Corr) of the evaluated traits.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11972933/bin/TPG2-18-e70024-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70024-s001.docx</a><sup> (368.7KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data generated or analyzed during this study are included in this published article.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70024"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70024.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (689.9 KB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11972933/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11972933/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11972933%2F%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11972933/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11972933/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11972933/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40189482/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11972933/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40189482/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11972933/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11972933/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="wyTpjVg54VkGGgZ3Ge3Q2oltpBIvtJmqFd2JkUmskIiIVh8bFYpVVGVViEPumGf8">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
