
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            The Structural Biology of T‐Cell Antigen Detection at Close Contacts - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="68F7DF1C809521A300DF1C003AA5F3B4.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11969063/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Immunological Reviews">
<meta name="citation_title" content="The Structural Biology of T‐Cell Antigen Detection at Close Contacts">
<meta name="citation_author" content="Yuan Lui">
<meta name="citation_author_institution" content="Weatherall Institute of Molecular Medicine, Radcliffe Department of Medicine, John Radcliffe Hospital, University of Oxford, Oxford, UK">
<meta name="citation_author_institution" content="Medical Research Council Translational Immune Discovery Unit, John Radcliffe Hospital, University of Oxford, Oxford, UK">
<meta name="citation_author" content="João Ferreira Fernandes">
<meta name="citation_author_institution" content="Weatherall Institute of Molecular Medicine, Radcliffe Department of Medicine, John Radcliffe Hospital, University of Oxford, Oxford, UK">
<meta name="citation_author_institution" content="Medical Research Council Translational Immune Discovery Unit, John Radcliffe Hospital, University of Oxford, Oxford, UK">
<meta name="citation_author" content="Mai T Vuong">
<meta name="citation_author_institution" content="Weatherall Institute of Molecular Medicine, Radcliffe Department of Medicine, John Radcliffe Hospital, University of Oxford, Oxford, UK">
<meta name="citation_author_institution" content="Medical Research Council Translational Immune Discovery Unit, John Radcliffe Hospital, University of Oxford, Oxford, UK">
<meta name="citation_author" content="Sumana Sharma">
<meta name="citation_author_institution" content="Weatherall Institute of Molecular Medicine, Radcliffe Department of Medicine, John Radcliffe Hospital, University of Oxford, Oxford, UK">
<meta name="citation_author_institution" content="Medical Research Council Translational Immune Discovery Unit, John Radcliffe Hospital, University of Oxford, Oxford, UK">
<meta name="citation_author" content="Ana Mafalda Santos">
<meta name="citation_author_institution" content="Weatherall Institute of Molecular Medicine, Radcliffe Department of Medicine, John Radcliffe Hospital, University of Oxford, Oxford, UK">
<meta name="citation_author_institution" content="Medical Research Council Translational Immune Discovery Unit, John Radcliffe Hospital, University of Oxford, Oxford, UK">
<meta name="citation_author" content="Simon J Davis">
<meta name="citation_author_institution" content="Weatherall Institute of Molecular Medicine, Radcliffe Department of Medicine, John Radcliffe Hospital, University of Oxford, Oxford, UK">
<meta name="citation_author_institution" content="Medical Research Council Translational Immune Discovery Unit, John Radcliffe Hospital, University of Oxford, Oxford, UK">
<meta name="citation_publication_date" content="2025 Apr 3">
<meta name="citation_volume" content="331">
<meta name="citation_issue" content="1">
<meta name="citation_firstpage" content="e70014">
<meta name="citation_doi" content="10.1111/imr.70014">
<meta name="citation_pmid" content="40181535">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11969063/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11969063/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11969063/pdf/IMR-331-0.pdf">
<meta name="description" content="T cells physically interrogate their targets using tiny membrane protrusions called microvilli, forming junctions ~400 nm in diameter and ~ 15 nm deep, referred to as “close contacts”. These contacts, which are stabilized by the binding of the small ...">
<meta name="og:title" content="The Structural Biology of T‐Cell Antigen Detection at Close Contacts">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="T cells physically interrogate their targets using tiny membrane protrusions called microvilli, forming junctions ~400 nm in diameter and ~ 15 nm deep, referred to as “close contacts”. These contacts, which are stabilized by the binding of the small ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11969063/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11969063">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1111/imr.70014"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/IMR-331-0.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11969063%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11969063/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11969063/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11969063/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1111/imr.70014" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Immunol Rev</button></div>. 2025 Apr 3;331(1):e70014. doi: <a href="https://doi.org/10.1111/imr.70014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1111/imr.70014</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Immunol%20Rev%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Immunol%20Rev%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Immunol%20Rev%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Immunol%20Rev%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>The Structural Biology of T‐Cell Antigen Detection at Close Contacts</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lui%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Yuan Lui</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yuan Lui</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Weatherall Institute of Molecular Medicine, Radcliffe Department of Medicine, John Radcliffe Hospital, University of Oxford, Oxford, UK</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Medical Research Council Translational Immune Discovery Unit, John Radcliffe Hospital, University of Oxford, Oxford, UK</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lui%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuan Lui</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferreira%20Fernandes%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">João Ferreira Fernandes</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">João Ferreira Fernandes</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Weatherall Institute of Molecular Medicine, Radcliffe Department of Medicine, John Radcliffe Hospital, University of Oxford, Oxford, UK</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Medical Research Council Translational Immune Discovery Unit, John Radcliffe Hospital, University of Oxford, Oxford, UK</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferreira%20Fernandes%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">João Ferreira Fernandes</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vuong%20MT%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Mai T Vuong</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Mai T Vuong</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Weatherall Institute of Molecular Medicine, Radcliffe Department of Medicine, John Radcliffe Hospital, University of Oxford, Oxford, UK</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Medical Research Council Translational Immune Discovery Unit, John Radcliffe Hospital, University of Oxford, Oxford, UK</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vuong%20MT%22%5BAuthor%5D" class="usa-link"><span class="name western">Mai T Vuong</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sharma%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Sumana Sharma</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Sumana Sharma</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Weatherall Institute of Molecular Medicine, Radcliffe Department of Medicine, John Radcliffe Hospital, University of Oxford, Oxford, UK</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Medical Research Council Translational Immune Discovery Unit, John Radcliffe Hospital, University of Oxford, Oxford, UK</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sharma%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sumana Sharma</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Santos%20AM%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Ana Mafalda Santos</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Ana Mafalda Santos</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Weatherall Institute of Molecular Medicine, Radcliffe Department of Medicine, John Radcliffe Hospital, University of Oxford, Oxford, UK</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Medical Research Council Translational Immune Discovery Unit, John Radcliffe Hospital, University of Oxford, Oxford, UK</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Santos%20AM%22%5BAuthor%5D" class="usa-link"><span class="name western">Ana Mafalda Santos</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Davis%20SJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Simon J Davis</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Simon J Davis</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Weatherall Institute of Molecular Medicine, Radcliffe Department of Medicine, John Radcliffe Hospital, University of Oxford, Oxford, UK</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Medical Research Council Translational Immune Discovery Unit, John Radcliffe Hospital, University of Oxford, Oxford, UK</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Davis%20SJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Simon J Davis</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="imr70014-aff-0001">
<sup>
<sup>1</sup>
</sup>Weatherall Institute of Molecular Medicine, Radcliffe Department of Medicine, John Radcliffe Hospital, University of Oxford, Oxford, UK</div>
<div id="imr70014-aff-0002">
<sup>
<sup>2</sup>
</sup>Medical Research Council Translational Immune Discovery Unit, John Radcliffe Hospital, University of Oxford, Oxford, UK</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong>
 
Simon J. Davis (<span>simon.davis@imm.ox.ac.uk</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 6; Accepted 2025 Feb 16; Issue date 2025 May.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Immunological Reviews</em> published by John Wiley &amp; Sons Ltd.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11969063  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40181535/" class="usa-link">40181535</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>T cells physically interrogate their targets using tiny membrane protrusions called microvilli, forming junctions ~400 nm in diameter and ~ 15 nm deep, referred to as “close contacts”. These contacts, which are stabilized by the binding of the small adhesion protein CD2 to its ligand, CD58 and locally exclude large proteins such as the phosphatase CD45, are the sites of antigen recognition by the T‐cell receptor (TCR) and very early signaling by T cells. With our collaborators, we have characterized the molecular structures of several of the key proteins mediating these early events: i.e., CD2 and its ligands, CD45, the αβ‐ and γδ‐TCRs, and the accessory proteins CD28, CTLA‐4, and PD‐1. Here, we review our structural work and the insights it offers into the early events underpinning T‐cell responsiveness that take place in the confined space of the close contact. We reflect on the crucial roles that the structural organization and dimensions of these proteins are likely to have in determining the sequence of events leading to antigen recognition at close contacts and consider the general implications of the structural work for explanations of how immune receptor signaling is initiated.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> antigen recognition, close contacts, receptor triggering, structural biology, T‐cell signaling</p></section></section><section id="imr70014-sec-0001"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>To perform their immune functions, T cells need to be highly tactile [<a href="#imr70014-bib-0001" class="usa-link" aria-describedby="imr70014-bib-0001">1</a>, <a href="#imr70014-bib-0002" class="usa-link" aria-describedby="imr70014-bib-0002">2</a>]. They protect us from infections and tumors by being able to sense the physical presence, on prospective target cells, of antigenic peptides bound to major histocompatibility complex molecules (pMHC) using specialized structures, that is, T‐cell receptors (TCRs) [<a href="#imr70014-bib-0003" class="usa-link" aria-describedby="imr70014-bib-0003">3</a>, <a href="#imr70014-bib-0004" class="usa-link" aria-describedby="imr70014-bib-0004">4</a>]. Encounters with pMHC result in TCR phosphorylation, the first step in T‐cell activation. Exactly how the receptor becomes phosphorylated is uncertain [<a href="#imr70014-bib-0005" class="usa-link" aria-describedby="imr70014-bib-0005">5</a>, <a href="#imr70014-bib-0006" class="usa-link" aria-describedby="imr70014-bib-0006">6</a>]. Our proposal, known as the “kinetic‐segregation” (KS) model [<a href="#imr70014-bib-0007" class="usa-link" aria-describedby="imr70014-bib-0007">7</a>, <a href="#imr70014-bib-0008" class="usa-link" aria-describedby="imr70014-bib-0008">8</a>], supposes that the net level of TCR phosphorylation by kinases, such as Lck, increases when large receptor‐type protein tyrosine phosphatases (RPTPs), such as CD45, are sterically excluded from sites of TCR/pMHC binding, preventing them from reversing the constitutive phosphorylation of the receptor. Incipient phosphorylation of the receptor is likely to be amplified by the kinase‐bearing co‐receptors, CD4 and CD8 [<a href="#imr70014-bib-0009" class="usa-link" aria-describedby="imr70014-bib-0009">9</a>, <a href="#imr70014-bib-0010" class="usa-link" aria-describedby="imr70014-bib-0010">10</a>, <a href="#imr70014-bib-0011" class="usa-link" aria-describedby="imr70014-bib-0011">11</a>], allowing recruitment and activation of the cytosolic kinase, ZAP70 (reviewed in [<a href="#imr70014-bib-0012" class="usa-link" aria-describedby="imr70014-bib-0012">12</a>]). ZAP70 phosphorylates the linker for activation of T cells, which in turn forms two‐dimensional condensates controlling downstream signaling [<a href="#imr70014-bib-0013" class="usa-link" aria-describedby="imr70014-bib-0013">13</a>, <a href="#imr70014-bib-0014" class="usa-link" aria-describedby="imr70014-bib-0014">14</a>]. TCR‐derived signals are integrated with those from “accessory” receptors, for example, the costimulatory protein CD28, which amplifies TCR signaling, and the immune checkpoints, CTLA‐4 and PD‐1, which suppress it (reviewed in [<a href="#imr70014-bib-0015" class="usa-link" aria-describedby="imr70014-bib-0015">15</a>, <a href="#imr70014-bib-0016" class="usa-link" aria-describedby="imr70014-bib-0016">16</a>]).</p>
<p>The modern view of the structure of the mammalian cell surface is that it comprises two molecular layers [<a href="#imr70014-bib-0017" class="usa-link" aria-describedby="imr70014-bib-0017">17</a>, <a href="#imr70014-bib-0018" class="usa-link" aria-describedby="imr70014-bib-0018">18</a>]. Adjacent to the cell membrane, the first layer consists of “communication devices” that is, receptors extending up to 20–30 nm from the cell surface that bind cognate ligands secreted by or expressed on the surfaces of other cells. The communication devices of T cells include, for example, the TCR, CD28, and PD‐1. The second layer, known as the “glycocalyx” is formed of dense arrays of highly‐expressed, negatively charged, and heavily‐glycosylated proteins and polysaccharides that extend beyond the first layer, creating a physical barrier to cell contact [<a href="#imr70014-bib-0017" class="usa-link" aria-describedby="imr70014-bib-0017">17</a>]. The glycocalyx of nucleated cells of hematopoietic origin is comprised of CD45 and the mucin‐like glycoprotein, CD43 [<a href="#imr70014-bib-0019" class="usa-link" aria-describedby="imr70014-bib-0019">19</a>, <a href="#imr70014-bib-0020" class="usa-link" aria-describedby="imr70014-bib-0020">20</a>].</p>
<p>We have studied how T cells overcome glycocalyx barriers, using supported lipid bilayer‐based models of the antigen‐presenting cell surface presenting the extracellular domains (ECDs) of the glycocalyx elements CD43 and CD45, the ligands of small (CD2) and large (LFA‐1) adhesive proteins, that is, CD58 and ICAM‐1, respectively, and both null and model agonistic TCR ligands (Figure <a href="#imr70014-fig-0001" class="usa-link">1a</a>) [<a href="#imr70014-bib-0023" class="usa-link" aria-describedby="imr70014-bib-0023">23</a>]. We found that T cells use very narrow surface protrusions called microvilli to ‘punch’ through glycocalyx barriers, forming numerous tiny (400 nm‐wide) “close contacts” visualized via the accumulation of CD58 (Figure <a href="#imr70014-fig-0001" class="usa-link">1b</a>) [<a href="#imr70014-bib-0023" class="usa-link" aria-describedby="imr70014-bib-0023">23</a>]. Because they are stabilized by the binding of CD2 to CD58 [<a href="#imr70014-bib-0023" class="usa-link" aria-describedby="imr70014-bib-0023">23</a>], close contacts are likely ~15 nm deep. Close contacts are sites of antigen detection and signaling, as is indicated by their accumulation of agonistic pMHC and their recruitment of ZAP70 [<a href="#imr70014-bib-0023" class="usa-link" aria-describedby="imr70014-bib-0023">23</a>]. The first sign of productive signaling, occurring within seconds, is that the cells spread on the bilayer, forming more close contacts; in the absence of the TCR, just a few close contacts form, and the cells do not spread. ICAM‐1 is excluded from the close contacts but is required during the spreading stage [<a href="#imr70014-bib-0023" class="usa-link" aria-describedby="imr70014-bib-0023">23</a>], presumably when LFA‐1 adopts a high‐affinity conformation [<a href="#imr70014-bib-0024" class="usa-link" aria-describedby="imr70014-bib-0024">24</a>]. If the contacts are made to be artificially large (i.e., &gt; 1 μm), TCR discrimination is lost [<a href="#imr70014-bib-0023" class="usa-link" aria-describedby="imr70014-bib-0023">23</a>], as predicted by computational models of T‐cell signaling based on the KS model [<a href="#imr70014-bib-0025" class="usa-link" aria-describedby="imr70014-bib-0025">25</a>, <a href="#imr70014-bib-0026" class="usa-link" aria-describedby="imr70014-bib-0026">26</a>]. Close contacts are likely, therefore, to be sites of early decision‐making by T cells.</p>
<figure class="fig xbox font-sm" id="imr70014-fig-0001"><h3 class="obj_head">FIGURE 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/05a4/11969063/b455737ae6e1/IMR-331-0-g006.jpg" loading="lazy" id="jats-graphic-1" height="448" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/imr70014-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Close contacts. (a) 'Second‐generation' glass‐supported lipid bilayers present histidine‐tagged, soluble forms of the ECDs of agonistic and 'null' pMHC, the fluorescently‐tagged ligand of CD2, that is, CD58, ICAM‐1, and the two major glycocalyx elements of the antigen‐presenting cell surface, CD45 and CD43, via nickellated lipids. (b) T cells, labeled with fluorescent anti‐CD45 antibody (blue), punch through the glycocalyx barrier on the bilayer and form CD2‐mediated close contacts marked by the local accumulation of CD58 (magenta) in 400 nm‐wide puncta visible using total internal reflection fluorescence (TIRF) microscopy (two such puncta are circled, <em>left</em>). The intensity line profile (<em>right</em>) showing the local exclusion of CD45 at regions of CD58 accumulation, is taken along the line indicated by the white arrow. (c) Depictions of the structures and interactions of proteins discussed in this review, prepared using the program Illustrate [<a href="#imr70014-bib-0021" class="usa-link" aria-describedby="imr70014-bib-0021">21</a>]; the TCR complexes were positioned using membrane coordinates from the Orientations of Proteins in Membranes database [<a href="#imr70014-bib-0022" class="usa-link" aria-describedby="imr70014-bib-0022">22</a>] (using PDB: 7phr).</p></figcaption></figure><p>Working closely with structural biologists over many years, we have characterized the molecular structures of key actors in these early signaling events: that is, the small adhesion protein CD2 [<a href="#imr70014-bib-0027" class="usa-link" aria-describedby="imr70014-bib-0027">27</a>, <a href="#imr70014-bib-0028" class="usa-link" aria-describedby="imr70014-bib-0028">28</a>] and its ligands CD58 [<a href="#imr70014-bib-0029" class="usa-link" aria-describedby="imr70014-bib-0029">29</a>] and CD48 [<a href="#imr70014-bib-0030" class="usa-link" aria-describedby="imr70014-bib-0030">30</a>], the large phosphatase CD45 [<a href="#imr70014-bib-0031" class="usa-link" aria-describedby="imr70014-bib-0031">31</a>], αβ‐ [<a href="#imr70014-bib-0032" class="usa-link" aria-describedby="imr70014-bib-0032">32</a>] and γδ‐ [<a href="#imr70014-bib-0033" class="usa-link" aria-describedby="imr70014-bib-0033">33</a>] TCRs, and the accessory proteins, CD28 [<a href="#imr70014-bib-0034" class="usa-link" aria-describedby="imr70014-bib-0034">34</a>] and CTLA‐4 [<a href="#imr70014-bib-0035" class="usa-link" aria-describedby="imr70014-bib-0035">35</a>, <a href="#imr70014-bib-0036" class="usa-link" aria-describedby="imr70014-bib-0036">36</a>] and their ligand CD80 [<a href="#imr70014-bib-0037" class="usa-link" aria-describedby="imr70014-bib-0037">37</a>], and PD‐1 [<a href="#imr70014-bib-0038" class="usa-link" aria-describedby="imr70014-bib-0038">38</a>]. This was made possible by first solving the ‘glycosylation problem’, which is that whereas the folding of glycoproteins is often glycosylation‐dependent (reviewed in [<a href="#imr70014-bib-0039" class="usa-link" aria-describedby="imr70014-bib-0039">39</a>]), the chemical and conformational heterogeneity of their N‐glycans frequently inhibits their crystallization (see, e.g., [<a href="#imr70014-bib-0040" class="usa-link" aria-describedby="imr70014-bib-0040">40</a>, <a href="#imr70014-bib-0041" class="usa-link" aria-describedby="imr70014-bib-0041">41</a>]). We established several methods for blocking N‐glycan processing during the production of glycoproteins in mammalian cells, allowing removal of the glycans with endoglycosidases after the glycoproteins are purified [<a href="#imr70014-bib-0042" class="usa-link" aria-describedby="imr70014-bib-0042">42</a>, <a href="#imr70014-bib-0043" class="usa-link" aria-describedby="imr70014-bib-0043">43</a>, <a href="#imr70014-bib-0044" class="usa-link" aria-describedby="imr70014-bib-0044">44</a>]. Here, we review our structural work and consider how the detailed structures of these proteins contribute to antigen recognition and signaling by T cells in the confined space of the close contact. The proteins considered here are presented in the likely temporal order of their contribution to antigen recognition. An overview of the structures is given in Figure <a href="#imr70014-fig-0001" class="usa-link">1c</a>. Early stages of this work have been reviewed previously [<a href="#imr70014-bib-0045" class="usa-link" aria-describedby="imr70014-bib-0045">45</a>, <a href="#imr70014-bib-0046" class="usa-link" aria-describedby="imr70014-bib-0046">46</a>].</p></section><section id="imr70014-sec-0002"><h2 class="pmc_sec_title">2. 
CD2 and Its Ligands</h2>
<p>CD2 belongs to a family of cell surface proteins forming a subgroup of the immunoglobulin (Ig) superfamily [<a href="#imr70014-bib-0047" class="usa-link" aria-describedby="imr70014-bib-0047">47</a>]. Human CD2 and its ligand, CD58, were the first heterophilic cell adhesion molecules to be identified [<a href="#imr70014-bib-0048" class="usa-link" aria-describedby="imr70014-bib-0048">48</a>, <a href="#imr70014-bib-0049" class="usa-link" aria-describedby="imr70014-bib-0049">49</a>], and analyses of rat CD2/ligand interactions using surface plasmon resonance‐based technology were the first to suggest that protein interactions at cell surfaces would typically have very fast off‐rates [<a href="#imr70014-bib-0050" class="usa-link" aria-describedby="imr70014-bib-0050">50</a>, <a href="#imr70014-bib-0051" class="usa-link" aria-describedby="imr70014-bib-0051">51</a>]. Human CD2 is expressed by T‐ and NK cells and, by binding to the broadly‐expressed CD58, contributes to antiviral responses, autoimmunity, transplant rejection, and tumor immune evasion [<a href="#imr70014-bib-0052" class="usa-link" aria-describedby="imr70014-bib-0052">52</a>], likely by setting cellular activation thresholds [<a href="#imr70014-bib-0053" class="usa-link" aria-describedby="imr70014-bib-0053">53</a>, <a href="#imr70014-bib-0054" class="usa-link" aria-describedby="imr70014-bib-0054">54</a>]. In mice and rats, CD2 binds a different but related protein, CD48 [<a href="#imr70014-bib-0055" class="usa-link" aria-describedby="imr70014-bib-0055">55</a>]. CD48 also binds to a second related protein, CD244 [<a href="#imr70014-bib-0056" class="usa-link" aria-describedby="imr70014-bib-0056">56</a>], that triggers the killing of CD48‐expressing targets by NK cells [<a href="#imr70014-bib-0057" class="usa-link" aria-describedby="imr70014-bib-0057">57</a>].</p>
<section id="imr70014-sec-0003"><h3 class="pmc_sec_title">2.1. CD2</h3>
<p>Rat soluble (s) CD2 was the test case demonstrating the utility of N‐glycan engineering for enhancing glycoprotein crystallization [<a href="#imr70014-bib-0042" class="usa-link" aria-describedby="imr70014-bib-0042">42</a>], yielding the first structure of the ECD of a cell adhesion molecule [<a href="#imr70014-bib-0027" class="usa-link" aria-describedby="imr70014-bib-0027">27</a>]. The ECD comprises two domains with Ig superfamily folds, as had been expected [<a href="#imr70014-bib-0058" class="usa-link" aria-describedby="imr70014-bib-0058">58</a>], connected by a highly conserved “linker” [<a href="#imr70014-bib-0059" class="usa-link" aria-describedby="imr70014-bib-0059">59</a>] (Figure <a href="#imr70014-fig-0002" class="usa-link">2a</a>). In the linker, the polypeptide continues in an extended conformation for four residues beyond β‐strand G of domain (d) 1 through to β‐strand A of d2. In this way, it adds ~15 Å to the length of the protein and loosens its overall structure versus an otherwise similar arrangement of N‐terminal domains in CD4 (Figure <a href="#imr70014-fig-0002" class="usa-link">2b</a>). The linker positions the d1 GFCC'C″ β‐sheet, identified as the ligand‐binding site by mutagenesis of human CD2 [<a href="#imr70014-bib-0062" class="usa-link" aria-describedby="imr70014-bib-0062">62</a>, <a href="#imr70014-bib-0063" class="usa-link" aria-describedby="imr70014-bib-0063">63</a>, <a href="#imr70014-bib-0064" class="usa-link" aria-describedby="imr70014-bib-0064">64</a>], ~75 Å above the cell surface, close‐to‐parallel with the membrane, likely favoring head‐to‐head <em>in trans</em> interactions as cell contact is initiated (Figure <a href="#imr70014-fig-0002" class="usa-link">2a</a>). A conserved glycosylation site at the base of d2 of CD2 (and likely CD58 [<a href="#imr70014-bib-0060" class="usa-link" aria-describedby="imr70014-bib-0060">60</a>]) may hold the protein ‘upright’ at the cell surface. Substantial ‘hinge‐bending’ resulting from small changes in main‐chain torsion angles in the linker could also favor binding [<a href="#imr70014-bib-0027" class="usa-link" aria-describedby="imr70014-bib-0027">27</a>].</p>
<figure class="fig xbox font-sm" id="imr70014-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/05a4/11969063/69f480db2b87/IMR-331-0-g001.jpg" loading="lazy" id="jats-graphic-3" height="509" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/imr70014-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>CD2. (a) Ribbon and surface‐rendered views of the rat CD2 ECD (rsCD2; PDB: 1hng) showing the Ig superfamily domain organization of the protein (<em>left</em>), and the location of residues mediating the copy 1 head‐to‐head lattice contact expected to mimic CD2/ligand interactions (red; <em>right</em>). (b) Domain 2 superposition of rsCD2 and human CD4d1d2 (PDB: 1cdh) showing the lengthening effect of the conserved linker region of CD2. (c) The two versions of the head‐to‐head lattice contact observed for the two crystallographically independent copies of the CD2 ECD in the rat sCD2 crystals. (d) The electrostatic potential calculated at neutral pH projected onto the ligand‐binding surfaces of human (h) CD2 (PDB: 1hnf; <em>upper</em>) and CD58 (PDB: 1ccz; <em>lower</em>). Blue corresponds to positive potential, white neutral, and red negative potential, contoured at ±10 kT/e. The directions of the C strands of each domain are represented by the blue (CD2) and red (CD58) arrows (relative to CD2, the CD58 ligand‐binding domain is rotated 90° clockwise in the plane of the page). (e) The structure of the CD2/CD58 d1 complex, predicted by superimposing the ligand‐binding domains of human CD2 and CD58 onto the head‐to‐head lattice contact observed in crystals of human sCD2, is shown alongside the actual structure of the complex (PDB: 1qa9), determined subsequently [<a href="#imr70014-bib-0060" class="usa-link" aria-describedby="imr70014-bib-0060">60</a>]. (f) Structural differences between the CD58‐complexed and apo forms of hCD2 d1 (<em>top</em>). Thickness of the putty cartoon representation reflects the distance between equivalent Cα atoms following superposition of the structures. The apo form of hCD2 d1 is shown; the model helix (<em>lower</em>) scales the differences. (g) Glycosylation of CD2. Fifty N‐linked glycan conformers, depicted as yellow sticks and corresponding to NeuNAc‐Gal‐(GlcNAc)<sub>2</sub>Man<sub>3</sub>(Fuc)(GlcNAc)<sub>2</sub>, were modeled at each of the glycosylation sites occupied with sugar in the hsCD2 crystals, using GlycoSHIELD [<a href="#imr70014-bib-0061" class="usa-link" aria-describedby="imr70014-bib-0061">61</a>]. The view shown is equivalent to that in the ribbon representation of rsCD2 in Figure <a href="#imr70014-fig-0001" class="usa-link">1a</a>, <em>left</em>. The CD58 binding site of human CD2 is indicated (red).</p></figcaption></figure><p>A “head‐to‐head” contact in the crystals notable for being especially large (burying 650–690 Å<sup>2</sup>/molecule), suggested a plausible model of CD2/ligand interactions, comprising a complex of ~15 nm (Figure <a href="#imr70014-fig-0002" class="usa-link">2a,c</a>). The ligand‐binding d1 GFCC'C″ β‐sheets of each of two copies of the rat CD2 ECD in the crystal lattice formed the contact, which was seen again in crystals of the human CD2 ECD despite their distinct space groups and the substantial compositional differences of their GFCC'C″ β‐sheets [<a href="#imr70014-bib-0028" class="usa-link" aria-describedby="imr70014-bib-0028">28</a>]. The GFCC'C″ β‐sheets of the two homologues are unusual as sites of protein–protein recognition in being both flat and highly charged (Figure <a href="#imr70014-fig-0002" class="usa-link">2d</a>, upper). Alanine substitutions and the effects of ionic strength on binding indicated that two aromatics and one aliphatic residue in the GFCC'C″ β‐sheet provide all the energy needed for CD2 to bind weakly to its ligands (<em>K</em>
<sub>D</sub> = 60–90 μM [<a href="#imr70014-bib-0050" class="usa-link" aria-describedby="imr70014-bib-0050">50</a>]) and that eight charged or polar residues confer specificity without compromising the low affinity [<a href="#imr70014-bib-0065" class="usa-link" aria-describedby="imr70014-bib-0065">65</a>]. This is because the favorable effects of charged‐residue interactions at protein interfaces are offset by the energetic costs of de‐solvating the charged residues [<a href="#imr70014-bib-0066" class="usa-link" aria-describedby="imr70014-bib-0066">66</a>]. For more typical protein interactions, both specificity and affinity rely on the surface‐shape complementarity of relatively hydrophobic surfaces [<a href="#imr70014-bib-0067" class="usa-link" aria-describedby="imr70014-bib-0067">67</a>].</p></section><section id="imr70014-sec-0004"><h3 class="pmc_sec_title">2.2. The Ligands, CD58 and CD48
</h3>
<p>Structural analysis of the human and rat CD2 ligands, CD58 and CD48, respectively, strengthened these arguments. The ligand‐binding domain of CD58, crystallized as a chimera with d2 of rat CD2, was broadly similar to that of human CD2 [<a href="#imr70014-bib-0029" class="usa-link" aria-describedby="imr70014-bib-0029">29</a>]. It had standard Ig superfamily V‐set AGFCC'C″:DEB domain topology, but the head‐to‐head lattice contact characteristic of the CD2 ECD crystals [<a href="#imr70014-bib-0027" class="usa-link" aria-describedby="imr70014-bib-0027">27</a>, <a href="#imr70014-bib-0028" class="usa-link" aria-describedby="imr70014-bib-0028">28</a>] was absent. The CD2‐binding site of CD58, comprised also of its d1 GFCC'C″ β‐sheet [<a href="#imr70014-bib-0068" class="usa-link" aria-describedby="imr70014-bib-0068">68</a>, <a href="#imr70014-bib-0069" class="usa-link" aria-describedby="imr70014-bib-0069">69</a>], had surface depressions rather than being flat, resulting in it having no shape complementarity with the binding site of CD2. The AGFCC'C″ β‐sheet was notably acidic, however (Figure <a href="#imr70014-fig-0002" class="usa-link">2d</a>, lower [<a href="#imr70014-bib-0029" class="usa-link" aria-describedby="imr70014-bib-0029">29</a>]), complementing the basic CD58‐binding site of CD2. Docking CD58 and CD2 in the head‐to‐head arrangement observed in CD2 ECD crystals produced a high degree of electrostatic complementarity at the interface [<a href="#imr70014-bib-0029" class="usa-link" aria-describedby="imr70014-bib-0029">29</a>], leading us to predict that this is how they interact, as was subsequently confirmed when the actual structure of the CD2/CD58 complex was determined (Figure <a href="#imr70014-fig-0002" class="usa-link">2e</a>) [<a href="#imr70014-bib-0060" class="usa-link" aria-describedby="imr70014-bib-0060">60</a>]. The structure of CD48 d1 [<a href="#imr70014-bib-0030" class="usa-link" aria-describedby="imr70014-bib-0030">30</a>] is similar to that of d1 of CD58. It differs in that, although it is also highly charged, the ligand‐binding GFCC'C″ β‐sheet of CD48 is much flatter. Modeling the interactions of rat CD2 and CD48 required a ∼7° rotation of CD48 and a ∼3 Å translation versus the arrangement observed in the CD2/CD58 complex [<a href="#imr70014-bib-0060" class="usa-link" aria-describedby="imr70014-bib-0060">60</a>] to maximize electrostatic complementarity [<a href="#imr70014-bib-0030" class="usa-link" aria-describedby="imr70014-bib-0030">30</a>]. Moreover, whereas human CD2/CD58 binding must surmount a significant entropic barrier, the rat CD2/CD48 interaction relies on equivalent, weak enthalpic and entropic effects, revealing a degree of plasticity in the interactions of CD2 with its ligands in the course of their evolution [<a href="#imr70014-bib-0030" class="usa-link" aria-describedby="imr70014-bib-0030">30</a>]. The interaction of CD2 with CD58 is rigid‐body in character (Figure <a href="#imr70014-fig-0002" class="usa-link">2f</a>). A model of the intact, glycosylated form of the human CD2 ECD is shown in Figure <a href="#imr70014-fig-0002" class="usa-link">2g</a>.<a href="#imr70014-note-0001" class="usa-link">
<sup>1</sup>
</a>
</p></section></section><section id="imr70014-sec-0005"><h2 class="pmc_sec_title">3. CD45</h2>
<p>The RPTP CD45 is among the most highly expressed proteins on the surfaces of T cells (i.e., &gt; 100,000 molecules per cell and &gt; 3 molecules per TCR) [<a href="#imr70014-bib-0020" class="usa-link" aria-describedby="imr70014-bib-0020">20</a>]. Since its catalytic efficiency is also very high (10‐ to 1000‐fold greater than that of relevant kinases [<a href="#imr70014-bib-0070" class="usa-link" aria-describedby="imr70014-bib-0070">70</a>]), CD45 is very likely to constrain tyrosine phosphorylation in resting cells. But CD45 is also needed to initiate signaling, in part owing to its activating effects on Src kinases [<a href="#imr70014-bib-0071" class="usa-link" aria-describedby="imr70014-bib-0071">71</a>, <a href="#imr70014-bib-0072" class="usa-link" aria-describedby="imr70014-bib-0072">72</a>]. Expression of the multiple isoforms of CD45, whose ECDs consist of shared folded regions and variable mucin‐like segments, changes with cell type, developmental stage, and cell‐activation state [<a href="#imr70014-bib-0073" class="usa-link" aria-describedby="imr70014-bib-0073">73</a>]. The distinct isoforms result from alternative splicing of exon 4‐, 5‐, and 6‐encoded regions of <em>PTPRC</em> transcripts, with each region encoding, in turn, extra mucin‐like protein sequence. The ECD of CD45R0 has none of this extra sequence and consists of the shared folded region and a mucin‐like segment of 41 residues only, whereas the longest isoform, CD45RABC, has all the extra sequence and a mucin‐like segment of 202 residues (Figure <a href="#imr70014-fig-0003" class="usa-link">3a</a>).</p>
<figure class="fig xbox font-sm" id="imr70014-fig-0003"><h3 class="obj_head">FIGURE 3.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/05a4/11969063/4e04c372c3aa/IMR-331-0-g004.jpg" loading="lazy" id="jats-graphic-5" height="606" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/imr70014-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>CD45. (a) Organization of the CD45 isoforms, CD45RABC and CD45R0. The extra mucin‐like protein sequence generated by alternative splicing of exons 4, 5, and 6 of the <em>PTPRC</em> gene is indicated as a red insert in the ECD of CD45RABC. The folded and mucin‐like regions are drawn approximately to scale. (b) Crystal structure of d1‐d4 of the CD45 ECD (PDB: 5fmv) with each domain color‐coded as in panel a. (c) Ribbon diagrams of ECD domains d1‐d4, with β‐strands numbered sequentially; β‐strands 3 and 4 are missing in d1 and β‐strand 3 is truncated in d2. (d) Level of conservation of CD45 surface residues; residues present or conserved in 1–6 of 11 sequences are colored white, 7–8 pink, and 9–11 red. (e) Locations of disulfide bonds (represented as cystines) in the folded part of the CD45 ECD. (f) Locations of glycosylation sequons for seven mammalian species mapped onto the CD45 d1‐d4 structure. Sites occupied in one species are colored green, 2–3 yellow, 4–5 orange, 6 red, and in 7 species, purple. The oval marks the region mostly lacking glycosylation sequons. (g) Dimensions of the CD45R0 ECD measured using negative‐staining EM, versus those of the CD4/TCR/pMHC class II complex (PDB: 3t0e) and the modeled hCD2/CD58 complex. O‐linked glycans have been modeled onto CD45R0. (h) A model of the glycosylated CD45 ECD. N‐glycosylation (yellow), at sites observed to be occupied in the crystal structure of the d1‐d4 region of the CD45 ECD, was modeled using GlycoSHIELD (as in Figure <a href="#imr70014-fig-0002" class="usa-link">2g</a>). Core 2 O‐linked glycans were positioned at serine and threonine residues in a model of the mucin‐like region of CD45R0 based on the negative‐staining EM analysis, also using GlycoSHIELD.</p></figcaption></figure><p>The crystal structure of the folded region of human CD45, solved to 2.9 Å resolution [<a href="#imr70014-bib-0031" class="usa-link" aria-describedby="imr70014-bib-0031">31</a>], showed that it comprises a ‘beads on a string’ array of four concatenated modular domains (Figure <a href="#imr70014-fig-0003" class="usa-link">3b</a>). Despite being fibronectin domain 3 (FN3)‐related, d1 and d2 exhibit striking levels of degeneracy insofar as they lack or have unique arrangements of certain β‐strands (Figure <a href="#imr70014-fig-0003" class="usa-link">3c</a>). According to sequence comparisons, the d1d2 region of CD45 has exhibited considerable structural plasticity in the course of vertebrate evolution [<a href="#imr70014-bib-0031" class="usa-link" aria-describedby="imr70014-bib-0031">31</a>]. Overall, the level of sequence identity shared by CD45 ECDs from mammals is extremely low (&lt; 15%), contrasting with, e.g., the ECDs of type II RPTPs (&gt; 90% identity) [<a href="#imr70014-bib-0074" class="usa-link" aria-describedby="imr70014-bib-0074">74</a>]. What conservation there is maps mostly to the domain interfaces (Figure <a href="#imr70014-fig-0003" class="usa-link">3d</a>). The domains are connected by unusually short linkers versus other large RPTPs [<a href="#imr70014-bib-0074" class="usa-link" aria-describedby="imr70014-bib-0074">74</a>, <a href="#imr70014-bib-0075" class="usa-link" aria-describedby="imr70014-bib-0075">75</a>], ensuring that there is extensive inter‐domain contact and that, overall, the assembly comprises a single rigid unit as confirmed by the largely identical copies of the folded d1‐d4 region in the asymmetric unit [<a href="#imr70014-bib-0031" class="usa-link" aria-describedby="imr70014-bib-0031">31</a>]. Similarly, whereas FN3 domains typically lack cysteines, the d1‐d4 region is 'cross‐linked' by eight disulfide bonds (Figure <a href="#imr70014-fig-0003" class="usa-link">3e</a>). Overall, most glycosylation sites of d1‐d4 are poorly conserved across mammals, and the d1d2 region is likely to be more heavily glycosylated than d3d4. Intriguingly, one side of the d3d4 region is largely free of putative glycosylation sites (Figure <a href="#imr70014-fig-0003" class="usa-link">3f</a>). This may help CD45 to preferentially access target proteins with relatively small ECDs, such as the TCR. On the other hand, if their cytosolic regions are unstructured and extended, close access of CD45 to its substrates might not be important. Consistent with the free diffusion of CD45 monomers at the cell surface [<a href="#imr70014-bib-0076" class="usa-link" aria-describedby="imr70014-bib-0076">76</a>], the CD45 ECD did not form dimers in the crystal lattice. We were also unable to identify a conserved candidate ligand‐binding surface [<a href="#imr70014-bib-0031" class="usa-link" aria-describedby="imr70014-bib-0031">31</a>].</p>
<p>By fusing the full‐length ECD of CD45R0 to the globular ∼70 kD ECD of a semaphorin [<a href="#imr70014-bib-0077" class="usa-link" aria-describedby="imr70014-bib-0077">77</a>], the length and flexibility of the mucin‐like segment of CD45R0 could be determined using negative staining electron microscopy [<a href="#imr70014-bib-0031" class="usa-link" aria-describedby="imr70014-bib-0031">31</a>]. This region of CD45R0 proved to be 6.6 nm in length (Figure <a href="#imr70014-fig-0003" class="usa-link">3g</a>) and, since a full reconstruction was possible [<a href="#imr70014-bib-0031" class="usa-link" aria-describedby="imr70014-bib-0031">31</a>], it must also be rigid, as expected for mucins of this length [<a href="#imr70014-bib-0078" class="usa-link" aria-describedby="imr70014-bib-0078">78</a>]. Variable‐angle TIRF‐based imaging experiments implied that the ECDs of CD45R0 and CD45RABC are ‘upright’ at the cell surface [<a href="#imr70014-bib-0031" class="usa-link" aria-describedby="imr70014-bib-0031">31</a>]. A model of the intact, glycosylated form of the CD45 ECD is shown in Figure <a href="#imr70014-fig-0003" class="usa-link">3h</a>.</p></section><section id="imr70014-sec-0006"><h2 class="pmc_sec_title">4. The pMHC‐Bound αβ TCR
</h2>
<p>The ɑβ TCR consists of four non‐covalently associated, disulfide‐linked dimers, TCR‐ɑβ, CD3‐εδ, CD3‐εγ, and CD3‐ζ<sub>2</sub>. TCR‐ɑβ subunits are responsible for ligand reactivity via their variable regions and do not directly drive signaling. Instead, phosphorylation of the cytosolic regions of the three covalent CD3 dimers initiates intracellular signaling [<a href="#imr70014-bib-0079" class="usa-link" aria-describedby="imr70014-bib-0079">79</a>]. Prior to the structures of fully‐assembled TCR complexes becoming attainable, nuclear magnetic resonance (NMR)‐based and crystal structures of the subunits, mostly expressed in soluble forms, were determined [<a href="#imr70014-bib-0080" class="usa-link" aria-describedby="imr70014-bib-0080">80</a>, <a href="#imr70014-bib-0081" class="usa-link" aria-describedby="imr70014-bib-0081">81</a>, <a href="#imr70014-bib-0082" class="usa-link" aria-describedby="imr70014-bib-0082">82</a>, <a href="#imr70014-bib-0083" class="usa-link" aria-describedby="imr70014-bib-0083">83</a>, <a href="#imr70014-bib-0084" class="usa-link" aria-describedby="imr70014-bib-0084">84</a>, <a href="#imr70014-bib-0085" class="usa-link" aria-describedby="imr70014-bib-0085">85</a>]. These pioneering studies established that TCR/pMHC recognition occurs in a conserved manner, but with no common binding mode explaining MHC restriction [<a href="#imr70014-bib-0086" class="usa-link" aria-describedby="imr70014-bib-0086">86</a>]. In addition, for the most part, the very small differences in the structures of ligated and unligated TCR‐ɑβ ECDs seemed to indicate that these parts of the receptor do not undergo shared conformational rearrangements that might otherwise have explained how TCR ligands differing in quality produce distinct signaling outcomes [<a href="#imr70014-bib-0086" class="usa-link" aria-describedby="imr70014-bib-0086">86</a>].</p>
<p>Discerning whether or how the intact TCR responds to ligand required structures of fully assembled ligated and unligated TCRs to be determined. Using cryo‐electron microscopy (cryo‐EM)‐based analysis of detergent‐solubilized receptors, Dong et al. reported the first structure of an intact ɑβ TCR, solved to 3.7 Å global resolution, in 2019 [<a href="#imr70014-bib-0087" class="usa-link" aria-describedby="imr70014-bib-0087">87</a>]. This confirmed that the receptor has TCRαβ/CD3δγε<sub>2</sub>ζ<sub>2</sub> stoichiometry and offered two crucial insights into receptor assembly unobtainable from the earlier structural studies. First, assembly depends on interactions between the transmembrane (TM) regions of the TCR‐ɑβ heterodimer and CD3 dimers, forming a relatively tight eight‐helix bundle [<a href="#imr70014-bib-0087" class="usa-link" aria-describedby="imr70014-bib-0087">87</a>]. The TCR‐ɑ and TCR‐β TM regions associate via hydrophobic contacts and interact with each of the CD3 dimers via electrostatic contacts. Second, the new work revealed the nature of the interactions between the ECDs of the CD3‐εδ, CD3‐εγ, and TCR‐ɑβ heterodimers, which help to stabilize the extracellular parts of the receptor in an apparently rigid arrangement alongside interactions involving the membrane‐proximal connecting peptides (CPs) of each ECD pair, producing a “trimer‐like” arrangement of the receptor subunit ECDs [<a href="#imr70014-bib-0087" class="usa-link" aria-describedby="imr70014-bib-0087">87</a>]. No density was observed for the cytosolic regions of any receptor chain, implying that they are largely unstructured [<a href="#imr70014-bib-0087" class="usa-link" aria-describedby="imr70014-bib-0087">87</a>]. Subsequent refinement of the model to 3.2 Å resolution by the same team resulted in the identification of cholesterol moieties at two sites in the TM assembly [<a href="#imr70014-bib-0088" class="usa-link" aria-describedby="imr70014-bib-0088">88</a>]. Mutation of the cholesterol‐binding residues in the TM region led to TCR hyper‐reactivity, along with a 7.7 Å shift in the position of one of the CD3‐ζ chains [<a href="#imr70014-bib-0088" class="usa-link" aria-describedby="imr70014-bib-0088">88</a>]. Possible roles of cholesterol in αβ‐ and γδ‐TCR signaling are considered below.</p>
<p>These seminal studies offered the first insights into the structure of the resting state of the fully assembled TCR. To isolate a ligand‐bound receptor, we used a clonotypic, affinity‐matured TCR. Our expectation was that the best‐folded receptors would be those that reach the cell surface and that, if they had sufficient affinity, these TCRs could be ‘tagged’ prior to cell lysis with soluble pMHC monomers carrying a purification tag. The αβ TCR we assembled, called GPa3b17, binds with an affinity (<em>K</em>
<sub>D</sub>) of 13 pM to a tumor antigen, that is, a glycoprotein 100‐derived peptide, gp100, complexed with HLA‐A2 [<a href="#imr70014-bib-0089" class="usa-link" aria-describedby="imr70014-bib-0089">89</a>]. The gp100/HLA‐A2/TCR complex, solubilized in the detergent glyco‐diosgenin and comprising 11 individual proteins and peptides (Figure <a href="#imr70014-fig-0004" class="usa-link">4a</a>), was solved to a global resolution of 3.08 Å. This first view of pMHC recognition outside a crystal lattice confirmed that it occurs in the complementarity determining region (CDR)‐dependent, ‘diagonal’ binding mode observed in the crystals [<a href="#imr70014-bib-0086" class="usa-link" aria-describedby="imr70014-bib-0086">86</a>]. The improved binding of the affinity‐matured TCR to gp100/HLA‐A2 was explained by direct and indirect effects of CDR substitutions on both HLA‐A2 and gp100 recognition. The TCR‐αβ subunits, bound to gp100/HLA‐A2, are cradled by the ECDs of the CD3‐εδ and CD3‐εγ heterodimers, supported by the eight‐helix TM bundle (Figure <a href="#imr70014-fig-0004" class="usa-link">4a</a>). Interactions with the CD3 subunits position the pMHC binding site of the TCR toward the ‘side’ of the complex and lock the TCR‐αβ subunit/pMHC complex at an angle of 59° relative to the plane of the membrane (Figure <a href="#imr70014-fig-0004" class="usa-link">4a</a>). These interactions comprise multivalent contacts across two structural layers involving the CD3‐εδ and CD3‐εγ ECDs and the constant (C) regions and CPs of TCR‐αβ, an additional layer of contacts involving the TCR‐αβ TM bundle that crucially allows charged TM‐residue neutralization (Figure <a href="#imr70014-fig-0004" class="usa-link">4b</a>), and interactions with cholesterol moieties incorporated into the TM bundle (Figure <a href="#imr70014-fig-0004" class="usa-link">4b</a>) [<a href="#imr70014-bib-0032" class="usa-link" aria-describedby="imr70014-bib-0032">32</a>].</p>
<figure class="fig xbox font-sm" id="imr70014-fig-0004"><h3 class="obj_head">FIGURE 4.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/05a4/11969063/ea225b6d2a99/IMR-331-0-g003.jpg" loading="lazy" id="jats-graphic-7" height="488" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/imr70014-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The αβ TCR. (a) Model of the fully‐assembled gp100/HLA‐A2/GPa3b17 TCR complex (PDB: 7phr) viewed along the plane of the membrane. Protein subunits are shown in ribbon format and the gp100 peptide antigen as a space‐filling model; the approximate boundaries of the membrane are shown as gray lines. The 59° tilt in the position of the complexed pMHC/TCR ECDs versus the plane of the membrane, and the three structural layers underpinning the assembly of the TCR complex are indicated. (b) Arrangement of the Layer 3 TM regions viewed toward the cell along an axis orthogonal to the membrane. Residues forming salt bridges are indicated. (c) Likely assembly intermediates in complex formation. The Layer 2 linker regions of the TCR‐αβ and CD3‐δε and ‐γε heterodimers are each stabilized by disulfide bonds (yellow). The 'tilting back' of the ECDs of the TCR‐αβ and CD3‐δε and ‐γε subunits might, especially, facilitate the close apposition of their TM regions, allowing charged TM‐residue interactions during step 1 of assembly. Cholesterol and CD3‐ζζ homodimer recruitment would complete assembly in step 2. (d, e) Structural differences between the gp100/HLA‐A2/GPa3b17 ligated TCR complex and the apo form of the TCR solved by Dong et al. [<a href="#imr70014-bib-0087" class="usa-link" aria-describedby="imr70014-bib-0087">87</a>] (PDB: 6jxr) (d), and the ligated (PDB: 8es8) and apo (PDB: 8es7) forms of the PN45545 TCR solved by Saotome et al. [<a href="#imr70014-bib-0090" class="usa-link" aria-describedby="imr70014-bib-0090">90</a>] (e), are shown (the representation is the same as in Figure <a href="#imr70014-fig-0002" class="usa-link">2f</a>, except that the ligated structures are used for the putty diagrams). (f) Modeling of glycosylation (yellow) at sites observed to be occupied in the cryo EM structure of the GPa3b17 TCR, using GlycoSHIELD (as in Figure <a href="#imr70014-fig-0002" class="usa-link">2g</a>).</p></figcaption></figure><p>We were also interested in how such an assembly might form, having observed that ligand‐binding complexes lacking CD3‐ζ can be expressed at high levels in Chinese hamster ovary (CHO) cells [<a href="#imr70014-bib-0032" class="usa-link" aria-describedby="imr70014-bib-0032">32</a>]. We noticed that the CPs of the TCR‐αβ and CD3 heterodimers create distinctive, rigid links to their TM regions (Figure <a href="#imr70014-fig-0004" class="usa-link">4c</a>). A short helix from TCR‐α fills the space between TCR‐Cβ and the N‐terminal ends of the TCR‐αβ TM helices, an arrangement reinforced by a disulfide bond to the TCR‐β CP. The linkers of each CD3 heterodimer, on the other hand, are stabilized by short β‐strands and paired disulfide bonds formed by vicinal cysteines in conserved CXXC motifs next to their TM regions, adjacent to and above the TCR‐αβ TM bundle. The hydrophobic interaction of cystines in this manner seemed to be unprecedented. The CPs, which also impose tilts in the CD3 heterodimer ECDs relative to their long axes (Figure <a href="#imr70014-fig-0004" class="usa-link">4c</a>), may in this way facilitate the rigid‐body, close packing of the TM regions of the three heterodimers, in an arrangement stabilized initially by TM‐region salt‐bridges and then by interactions of their ECDs, explaining the formation of the CD3‐ζζ homodimer‐free complexes we detected in CHO cells. In a straightforward way, docking sites for cholesterol and CD3‐ζζ homodimers would be created, allowing completion of assembly.</p>
<p>Most importantly, our work allowed a first analysis of the effects of pMHC binding on the fully assembled TCR. Comparisons of the backbone conformations of our pMHC‐bound GPa3b17 TCR and the unligated structure solved by Dong et al. [<a href="#imr70014-bib-0087" class="usa-link" aria-describedby="imr70014-bib-0087">87</a>] indicated that the TCR resists ligand‐induced changes: the root‐mean‐square‐deviation between the two structures for all Cα atoms was 1.2 Å (Figure <a href="#imr70014-fig-0004" class="usa-link">4d</a>) [<a href="#imr70014-bib-0032" class="usa-link" aria-describedby="imr70014-bib-0032">32</a>]. Previously, the binding of pMHC by the TCR was proposed to induce structural changes in four regions: (1) the TCR‐Cβ FG loop, (2) the TCR‐Cα AB loop, (3) the TCR‐Cβ αA helix, and (4) the TM regions, but these regions were essentially unchanged [<a href="#imr70014-bib-0032" class="usa-link" aria-describedby="imr70014-bib-0032">32</a>]. Only very minor displacements were observed at the C termini of the TM regions of CD3‐ε and CD3‐ε' (of 2.5–3.5 Å), perhaps only reflecting differences in construct design [<a href="#imr70014-bib-0032" class="usa-link" aria-describedby="imr70014-bib-0032">32</a>]. A caveat of these observations was that the strong binding of the GPa3b17 TCR could prevent the complex from accessing dynamic states used by more physiological, lower‐affinity TCRs during signaling. However, molecular dynamic simulations suggested that lower‐affinity TCRs would behave similarly, as was quickly confirmed by Saotome et al. [<a href="#imr70014-bib-0090" class="usa-link" aria-describedby="imr70014-bib-0090">90</a>], who determined unbound and ligand‐bound structures of a native, low affinity (<em>K</em>
<sub>D</sub> = 6 μM) TCR (PN45545; Figure <a href="#imr70014-fig-0004" class="usa-link">4e</a>). A model of the intact, glycosylated form of the GPa3b17 TCR in the absence of ligand (Figure <a href="#imr70014-fig-0004" class="usa-link">4f</a>) suggests that surprisingly large areas of the surface of the receptor will be occluded by N‐linked glycans, likely preventing receptor oligomerization.</p></section><section id="imr70014-sec-0007"><h2 class="pmc_sec_title">5. The γδ TCR
</h2>
<p>γδ T‐cells comprise a T‐cell lineage that is distinct from αβ T‐cells and contributes to tumor and mucosal immunity [<a href="#imr70014-bib-0091" class="usa-link" aria-describedby="imr70014-bib-0091">91</a>]. In contrast to the αβ TCR, the γδ TCR reacts with a variety of structurally diverse ligands. These include the stress‐induced MHC I‐like molecules, CD1 and MR1, and non‐MHC‐like ligands, such as butyrophilin and butyrophilin‐related proteins [<a href="#imr70014-bib-0092" class="usa-link" aria-describedby="imr70014-bib-0092">92</a>]. From the outset, structural work on the γδ TCR suggested that γδ and αβ TCRs would be substantially different. In the TCR‐γδ ECD structure solved by Allison et al. [<a href="#imr70014-bib-0093" class="usa-link" aria-describedby="imr70014-bib-0093">93</a>], the C domains “swung out” from underneath the V domains versus the arrangement seen in TCR‐αβ ECDs. In addition, no conserved surfaces in the Cα/Cβ and Cγ/Cδ domains seemed likely to mediate shared CD3 subunit interactions. The TCR‐γ chain also lacked the extended FG loop of TCR‐β that has been linked to mechanotransduction [<a href="#imr70014-bib-0094" class="usa-link" aria-describedby="imr70014-bib-0094">94</a>], and Cδ had conventional Ig superfamily domain topology with C, F, and G β‐strands, in contrast to Cα whose topology is unique. Finally, Allison et al. [<a href="#imr70014-bib-0093" class="usa-link" aria-describedby="imr70014-bib-0093">93</a>] noted, like others previously [<a href="#imr70014-bib-0095" class="usa-link" aria-describedby="imr70014-bib-0095">95</a>, <a href="#imr70014-bib-0096" class="usa-link" aria-describedby="imr70014-bib-0096">96</a>], that the TCR‐γ and ‐δ subunits have substantially longer CPs.</p>
<p>Two structures of detergent‐solubilized, fully assembled γδ TCRs solved using cryo‐EM were reported by Xin et al. [<a href="#imr70014-bib-0097" class="usa-link" aria-describedby="imr70014-bib-0097">97</a>], followed soon after by a third structure determined by us (Figure <a href="#imr70014-fig-0005" class="usa-link">5a</a>) [<a href="#imr70014-bib-0033" class="usa-link" aria-describedby="imr70014-bib-0033">33</a>]. Additional work by Saotome and colleagues [<a href="#imr70014-bib-0098" class="usa-link" aria-describedby="imr70014-bib-0098">98</a>] on the γδ TCR clonotypes studied by Xin et al. extended and confirmed all of the initial findings. The TCRs studied by Xin et al. were of the Vγ9Vδ2 and Vγ5Vδ1 subtypes, whereas we solved the structure of a Vγ8Vδ3 TCR whose interactions with MR1 are well‐characterized [<a href="#imr70014-bib-0099" class="usa-link" aria-describedby="imr70014-bib-0099">99</a>, <a href="#imr70014-bib-0100" class="usa-link" aria-describedby="imr70014-bib-0100">100</a>]. We reasoned that although MR1‐reactive T cells comprise only a minor subset of circulating γδ T‐cells [<a href="#imr70014-bib-0099" class="usa-link" aria-describedby="imr70014-bib-0099">99</a>, <a href="#imr70014-bib-0100" class="usa-link" aria-describedby="imr70014-bib-0100">100</a>], because the C regions of γδ TCRs seemed likely to determine their overall organization, whatever was learned about the Vγ8Vδ3 TCR would probably apply to receptors comprising other combinations of Vγ and Vδ domains. The study of an MR1‐binding γδ TCR would also allow direct functional comparisons with MR1‐binding αβ TCRs. Whereas γδ TCR stoichiometry had been controversial [<a href="#imr70014-bib-0101" class="usa-link" aria-describedby="imr70014-bib-0101">101</a>, <a href="#imr70014-bib-0102" class="usa-link" aria-describedby="imr70014-bib-0102">102</a>], the solved structures all consisted of 1:1:1:1 associations of TCR‐γδ heterodimers with CD3‐δε, ‐γε, and ‐ζζ dimers (Figure <a href="#imr70014-fig-0005" class="usa-link">5a</a>), although a γδ TCR lacking CD3‐γ could also be purified (Figure <a href="#imr70014-fig-0005" class="usa-link">5b</a>). Once again, very little, if any, density was observed for the cytosolic regions of the TCR‐γδ and CD3 subunits, confirming their flexibility.</p>
<figure class="fig xbox font-sm" id="imr70014-fig-0005"><h3 class="obj_head">FIGURE 5.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/05a4/11969063/438da1e4e2e0/IMR-331-0-g002.jpg" loading="lazy" id="jats-graphic-9" height="516" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/imr70014-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The γδ TCR. (a) Model of the fully‐assembled MR1‐specific G83.C4 γδ TCR (PDB: 9cia) viewed along the plane of the membrane. Protein subunits are shown in ribbon format; the approximate boundaries of the membrane are shown as gray lines. (b) Elution of the fully‐assembled G83.C4 γδ TCR, and a form of the complex expressed in the absence of the CD3‐γ chain, from an analytical Superose 6 Increase 10/300GL size‐exclusion chromatography column. The horizontal bar indicates the elution position of material used for structural analysis of the fully‐constituted complex. (c) Displacement of the CD3 ECDs within the γδ‐ (colored) and αβ‐ (PDB: 7phr; white) TCR complexes, observed after superimposing the CD3‐δε heterodimer ECDs. The centers of mass of the CD3 subunit ECDs differ by up to ~8 Å between the complexes [<a href="#imr70014-bib-0033" class="usa-link" aria-describedby="imr70014-bib-0033">33</a>]. (d) Conservation of organization of the TM regions in the γδ TCR (colored) versus the apo αβ TCR (PDB: 7phr; white). (e) Differences in TCR CP sequence lengths for γδ and αβ TCRs from different mammals. (f) Effects of αβ and γδ TCR flexibility on signaling outcomes. A schematic of the G83.C4 and AF‐7 wild‐type and chimeric (CH) TCR constructs wherein, for the chimeras, the variable and constant domains were interchanged between the two receptors, is shown on the <em>left</em>. Mean fluorescence intensities (MFI) for CD69 expression are also shown, for Jurkat T‐cells expressing G83.C4 and G83/AF7<sub>CH</sub> TCRs (<em>middle</em>), and AF‐7 and AF7/G83<sub>CH</sub> TCRs (<em>right</em>), following stimulation with C1R cells treated with different amounts of 5‐OP‐RU. Reducing the flexibility of the γδ TCR increased signaling whereas enhancing it for the αβ TCR reduced signaling. <em>p</em> values are indicated: **&lt; 0.01; ****&lt; 0.0001. Error bars represent standard deviations. Statistical comparisons were performed using one way analysis of variance with Tukey's multiple comparison test; two independent experiments with <em>n</em> = 3 cocultures each were analyzed. (g) Modeling of glycosylation (yellow) at sites observed to be occupied in subunits of the γδ TCR (PDB: 9cia), using GlycoSHIELD (as in Figure <a href="#imr70014-fig-0002" class="usa-link">2g</a>).</p></figcaption></figure><p>The first key observation was that the overall organization of the CD3 ECDs in both TCR subclasses is largely conserved [<a href="#imr70014-bib-0033" class="usa-link" aria-describedby="imr70014-bib-0033">33</a>, <a href="#imr70014-bib-0097" class="usa-link" aria-describedby="imr70014-bib-0097">97</a>], despite the limited number of pairwise interactions stabilizing the positions of the CD3εδ and CD3εγ heterodimers (Figure <a href="#imr70014-fig-0005" class="usa-link">5c</a>) [<a href="#imr70014-bib-0033" class="usa-link" aria-describedby="imr70014-bib-0033">33</a>]. Residues contributing to a three‐way interaction that position CD3‐εγ next to TCR‐β and CD3‐εδ alongside TCR‐α were poorly resolved in the γδ TCR [<a href="#imr70014-bib-0033" class="usa-link" aria-describedby="imr70014-bib-0033">33</a>], perhaps because stabilizing contacts with the TCR‐γδ C domains are absent. Instead, the positioning of the CD3‐εδ and CD3‐εγ heterodimer ECDs relies on the relatively rigid linker regions packing the CD3 ECDs against the top of the TM helix bundle. For the γδ and αβ TCRs, the centers of mass of the CD3‐δε and ‐γε ECDs differ by as much as 8 Å [<a href="#imr70014-bib-0033" class="usa-link" aria-describedby="imr70014-bib-0033">33</a>], suggesting that precise positioning of the signaling subunit ECDs of TCRs is unimportant.</p>
<p>The second important finding was that the organization of the TM regions of γδ and αβ TCRs was very similar, despite the limited similarity of the TCR‐γ/TCR‐δ and TCR‐α/TCR‐β TM region sequences (Figure <a href="#imr70014-fig-0005" class="usa-link">5d</a>) [<a href="#imr70014-bib-0033" class="usa-link" aria-describedby="imr70014-bib-0033">33</a>, <a href="#imr70014-bib-0097" class="usa-link" aria-describedby="imr70014-bib-0097">97</a>]. Whereas, for the αβ TCR, TM region packing depends partly on interactions in the membrane‐proximal regions of the subunit ECDs [<a href="#imr70014-bib-0032" class="usa-link" aria-describedby="imr70014-bib-0032">32</a>], the γδ TCR relies almost exclusively on the electrostatic interactions of conserved, charged residues in its TM region. Cholesterol is consistently incorporated into the TM regions of γδ TCRs as in the case of ɑβ TCRs, but the distribution appears distinct (Table <a href="#imr70014-tbl-0001" class="usa-link">1</a>). Whereas robust density is seen in the outer leaflet‐embedded region of the ɑβ TCR, similar‐quality density is only seen in the inner leaflet‐associated portion of the γδ TCR [<a href="#imr70014-bib-0033" class="usa-link" aria-describedby="imr70014-bib-0033">33</a>, <a href="#imr70014-bib-0097" class="usa-link" aria-describedby="imr70014-bib-0097">97</a>]. The outer leaflet site in the γδ TCR is blocked by the reorientation of a CD3‐ζ residue (Lys30) toward a TCR‐γ TM region residue (Met254) [<a href="#imr70014-bib-0033" class="usa-link" aria-describedby="imr70014-bib-0033">33</a>]. Mutations that reduced cholesterol binding lowered signaling by the γδ TCR [<a href="#imr70014-bib-0097" class="usa-link" aria-describedby="imr70014-bib-0097">97</a>], but the changes in cholesterol levels and their effects on signaling were each relatively modest. Cholesterol de‐sequestration did, however, initiate spontaneous T‐cell activation [<a href="#imr70014-bib-0097" class="usa-link" aria-describedby="imr70014-bib-0097">97</a>]. But if cholesterol stabilizes the assembly, its removal could induce receptor aggregation and signaling. Considering the native structures only, given (1) that the overall organization of the TM domains is largely unaffected by the presence of cholesterol in the inner versus the outer leaflet TM regions, and (2) that the sterol is present in both ligated and unligated TCRs, it seems more likely that cholesterol affects receptor assembly rather than signaling.</p>
<section class="tw xbox font-sm" id="imr70014-tbl-0001"><h3 class="obj_head">TABLE 1.</h3>
<div class="caption p"><p>Summary of locations of cholesterol in solved αβ and γδ TCR structures.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">PDB ID</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">7fjd</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">8es7</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">8es8</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">8es9</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">7phr</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">8jc0</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">9cia</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">TCR</td>
<td align="center" valign="top" rowspan="1" colspan="1">αβ</td>
<td align="center" valign="top" rowspan="1" colspan="1">αβ</td>
<td align="center" valign="top" rowspan="1" colspan="1">αβ</td>
<td align="center" valign="top" rowspan="1" colspan="1">αβ</td>
<td align="center" valign="top" rowspan="1" colspan="1">αβ</td>
<td align="center" valign="top" rowspan="1" colspan="1">γδ</td>
<td align="center" valign="top" rowspan="1" colspan="1">γδ</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Apo</td>
<td align="center" valign="top" rowspan="1" colspan="1">×</td>
<td align="center" valign="top" rowspan="1" colspan="1">×</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1">×</td>
<td align="center" valign="top" rowspan="1" colspan="1">×</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Ligated</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1">×</td>
<td align="center" valign="top" rowspan="1" colspan="1">×</td>
<td align="center" valign="top" rowspan="1" colspan="1">×</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Cholesterol (outer leaflet)</td>
<td align="center" valign="top" rowspan="1" colspan="1">×</td>
<td align="center" valign="top" rowspan="1" colspan="1">×</td>
<td align="center" valign="top" rowspan="1" colspan="1">×</td>
<td align="center" valign="top" rowspan="1" colspan="1">×</td>
<td align="center" valign="top" rowspan="1" colspan="1">×</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Cholesterol (inner leaflet)</td>
<td align="center" valign="top" rowspan="1" colspan="1">×</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1">×</td>
<td align="center" valign="top" rowspan="1" colspan="1">×</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/imr70014-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>Easily the most striking and telling difference between the two subclasses of receptors, however, was that, in contrast to the αβ TCRs [<a href="#imr70014-bib-0032" class="usa-link" aria-describedby="imr70014-bib-0032">32</a>, <a href="#imr70014-bib-0087" class="usa-link" aria-describedby="imr70014-bib-0087">87</a>, <a href="#imr70014-bib-0090" class="usa-link" aria-describedby="imr70014-bib-0090">90</a>], for all three γδ receptors studied, the TCR‐γδ ECDs were either absent from the density or the associated density was discontinuous with that of the CD3 assembly [<a href="#imr70014-bib-0033" class="usa-link" aria-describedby="imr70014-bib-0033">33</a>, <a href="#imr70014-bib-0097" class="usa-link" aria-describedby="imr70014-bib-0097">97</a>]. The mobility of the γδ TCR is explained by several differences between the two receptor subclasses. First, αβ and γδ C domain sequences are poorly conserved (~12% and 25% identity for C‐δ/C‐α and C‐γ/C‐β comparisons, respectively), and the absence, in the FG loop of TCR‐γ and the DE loop of TCR‐δ, of residues forming important electrostatic contacts prevents the TCR‐γδ heterodimer from associating with the CD3 ECDs in the same way [<a href="#imr70014-bib-0033" class="usa-link" aria-describedby="imr70014-bib-0033">33</a>]. Automated structure comparisons ranked numerous antibody C domains ahead of C‐β and C‐α domains as the structures most like C‐γ and C‐δ, respectively. Second, as noted [<a href="#imr70014-bib-0093" class="usa-link" aria-describedby="imr70014-bib-0093">93</a>, <a href="#imr70014-bib-0095" class="usa-link" aria-describedby="imr70014-bib-0095">95</a>, <a href="#imr70014-bib-0096" class="usa-link" aria-describedby="imr70014-bib-0096">96</a>], the lengths of the membrane‐proximal C‐γ and C‐δ CPs are significantly longer (by 14 and 13 residues, respectively, depending on exon usage) than the equivalent CPs of the αβ TCR, and the CPs are unstructured or unresolved in the cryo‐EM maps. TCR‐γ CPs are typically encoded by multiple exons, vary significantly in length among mammals, and can be as long as sixty residues (Figure <a href="#imr70014-fig-0005" class="usa-link">5e</a>). The length of the single TCR‐δ CP is, however, highly conserved between species and constrained to 22–25 residues only, limiting the maximum 'reach' of the receptor [<a href="#imr70014-bib-0033" class="usa-link" aria-describedby="imr70014-bib-0033">33</a>].</p>
<p>We directly tested the impact of TCR flexibility on its function [<a href="#imr70014-bib-0033" class="usa-link" aria-describedby="imr70014-bib-0033">33</a>]. The αβ TCR, AF‐7, and the γδ TCR, G83.C4, bind with similar affinity and in a similar head‐to‐head orientation to a shared ligand, MR1, presenting the bacterial metabolite 5‐(2‐oxopropylideneamino)‐6‐D‐ribitylaminouracil (5‐OP‐RU) [<a href="#imr70014-bib-0100" class="usa-link" aria-describedby="imr70014-bib-0100">100</a>]. T cells transduced with the AF‐7 ɑβ TCR signaled more potently in response to MR1/5‐OP‐RU than cells bearing the G83.C4 γδ TCR [<a href="#imr70014-bib-0033" class="usa-link" aria-describedby="imr70014-bib-0033">33</a>]. Importantly, when the variable regions of the G83.C4 γδ TCR were grafted onto the presumably rigid AF‐7 TCR scaffold, T cells expressing the chimeric receptor produced much stronger signaling responses, measured as CD3ζ phosphorylation, which were comparable to those produced by AF‐7, suggesting that the flexibility of the γδ TCR constrains its responsiveness [<a href="#imr70014-bib-0033" class="usa-link" aria-describedby="imr70014-bib-0033">33</a>]. We also find that reciprocal swaps of all of the V and C domains between the two receptors increase signaling following G83.C4‐based MR1/5‐OP‐RU recognition (i.e., after ‘rigidifying’ the G83.C4 TCR), and reduce it after AF‐7–dependent ligand binding (i.e., when the AF‐7 TCR becomes more flexible; Figure <a href="#imr70014-fig-0005" class="usa-link">5f</a>). Xin et al. found that T cells expressing a γδ TCR comprised of the Cγ region encoded by <em>TRGC1</em> produced stronger responses to the MHC‐like ligand, CD1d, than cells expressing receptors with the 16‐residue longer CPs encoded by <em>TRGC2</em> [<a href="#imr70014-bib-0097" class="usa-link" aria-describedby="imr70014-bib-0097">97</a>], also suggesting that the flexibility of the γδ TCR limits its reactivity. We proposed that the reduced reactivity of the more flexible γδ TCR reflects a compromise between it having to bind structurally diverse ligands, and having to engage them efficiently. A model of the intact, glycosylated form of the G83.C4 γδ TCR is shown in Figure <a href="#imr70014-fig-0005" class="usa-link">5g</a>.</p>
<p>Xin et al. were also able to characterize the structure of the flexible ECD of a Vγ5Vδ1 TCR [<a href="#imr70014-bib-0097" class="usa-link" aria-describedby="imr70014-bib-0097">97</a>] because, unexpectedly, this TCR, but not the Vγ9Vδ2 or Vγ8Vδ3 TCRs [<a href="#imr70014-bib-0033" class="usa-link" aria-describedby="imr70014-bib-0033">33</a>, <a href="#imr70014-bib-0097" class="usa-link" aria-describedby="imr70014-bib-0097">97</a>], formed receptor dimers. Dimer formation relied on homotypic interactions involving residues in the invariant HV4 segment of Vγ5 that differ in Vγ9 or Vγ8 domains, with other contributions from CDR2; these residues are also present in Vγ2 domains, which also dimerized in solution [<a href="#imr70014-bib-0097" class="usa-link" aria-describedby="imr70014-bib-0097">97</a>]. Mutating these residues prevented dimerization of purified Vγ5Vδ1 receptors and impaired ligand binding and T‐cell signaling, suggesting that dimerization potentiates signaling by increasing ligand binding avidity. Similar observations were made by Hoque et al. in their studies of the same receptor [<a href="#imr70014-bib-0098" class="usa-link" aria-describedby="imr70014-bib-0098">98</a>].</p></section><section id="imr70014-sec-0008"><h2 class="pmc_sec_title">6. Accessory Receptors: CD28, CTLA‐4, and PD‐1</h2>
<p>The covalent homodimers, CD28 and CTLA‐4, are strong modulators of T‐cell responses. Their similar architectures comprise ECDs with single V‐set Ig superfamily domains and stalks linked to TM and cytosolic regions. The receptors bind via “MYPPPY” motifs [<a href="#imr70014-bib-0103" class="usa-link" aria-describedby="imr70014-bib-0103">103</a>] to two shared ligands: a weak dimer, CD80, and the monomer, CD86 [<a href="#imr70014-bib-0103" class="usa-link" aria-describedby="imr70014-bib-0103">103</a>, <a href="#imr70014-bib-0104" class="usa-link" aria-describedby="imr70014-bib-0104">104</a>, <a href="#imr70014-bib-0105" class="usa-link" aria-describedby="imr70014-bib-0105">105</a>]. CD28 and CTLA‐4 have orthogonal functions: CD28 provides signals essential for full T‐cell activation, whereas CTLA‐4 suppresses T‐cell responses [<a href="#imr70014-bib-0103" class="usa-link" aria-describedby="imr70014-bib-0103">103</a>, <a href="#imr70014-bib-0106" class="usa-link" aria-describedby="imr70014-bib-0106">106</a>]. The binding affinities and valencies of CD28 and CTLA‐4 also differ substantially [<a href="#imr70014-bib-0107" class="usa-link" aria-describedby="imr70014-bib-0107">107</a>]. PD‐1 (programmed cell death protein 1) is among the most potent inhibitory receptors in the immune system, and the pre‐eminent target of transformative immunotherapies [<a href="#imr70014-bib-0108" class="usa-link" aria-describedby="imr70014-bib-0108">108</a>]. PD‐1 consists of a single extracellular V‐set Ig superfamily domain also attached via a stalk to TM and cytosolic regions. The TM region is proposed to dimerize PD‐1 at the cell surface [<a href="#imr70014-bib-0109" class="usa-link" aria-describedby="imr70014-bib-0109">109</a>]. Its two ligands, PD‐L1 and PD‐L2, differ 3–4 fold in affinity for the receptor [<a href="#imr70014-bib-0038" class="usa-link" aria-describedby="imr70014-bib-0038">38</a>]. We helped determine the structure of the CTLA‐4/CD80 ECD complex [<a href="#imr70014-bib-0035" class="usa-link" aria-describedby="imr70014-bib-0035">35</a>] and published the structure of the apo CTLA‐4 ECD [<a href="#imr70014-bib-0036" class="usa-link" aria-describedby="imr70014-bib-0036">36</a>] in 2001 and 2011, respectively. Our structure of a human CD28 ECD monomer, complexed with the fragment antigen‐binding (Fab) of a mitogenic antibody, 5.11A1, was published in 2005 [<a href="#imr70014-bib-0034" class="usa-link" aria-describedby="imr70014-bib-0034">34</a>]. Finally, our collaborators' structures of the apo PD‐1 ECD [<a href="#imr70014-bib-0038" class="usa-link" aria-describedby="imr70014-bib-0038">38</a>] and of the PD‐1 ECD complexed with our antibody agonist, Clone 19 [<a href="#imr70014-bib-0110" class="usa-link" aria-describedby="imr70014-bib-0110">110</a>], were reported in 2013 and 2024.</p>
<section id="imr70014-sec-0009"><h3 class="pmc_sec_title">6.1. CD28</h3>
<p>CD28 enhances T‐cell activation via the PI3‐kinase pathway; in the absence of CD28 engagement, TCR signaling drives T cells into an anergic state [<a href="#imr70014-bib-0111" class="usa-link" aria-describedby="imr70014-bib-0111">111</a>]. Following unsuccessful attempts to crystallize intact, deglycosylated CD28 ECD homodimers, the structure of reduced, glycosylated monomers bound to the 5.11A1 Fab [<a href="#imr70014-bib-0112" class="usa-link" aria-describedby="imr70014-bib-0112">112</a>] was determined instead [<a href="#imr70014-bib-0034" class="usa-link" aria-describedby="imr70014-bib-0034">34</a>]. A major lattice contact in the crystals produced a plausible model of the CD28 ECD homodimer (Figure <a href="#imr70014-fig-0006" class="usa-link">6a</a>, upper) similar to the known CTLA‐4 dimer (Figure <a href="#imr70014-fig-0006" class="usa-link">6a</a>, lower). However, whereas CTLA‐4 homodimerization relies on residues next to and including the stalk‐like region, the CD28 lattice contact was formed by a small, three‐stranded A′G′F β‐sheet present in V‐set Ig superfamily domains [<a href="#imr70014-bib-0034" class="usa-link" aria-describedby="imr70014-bib-0034">34</a>]. Single‐particle cryo‐EM analysis of an intact CD28‐Fc fusion protein bound to the 5.11A1 antibody showed that the lattice‐based CD28 dimer was a better fit for the electron density than a CTLA‐4–based model. Moreover, mutations of residues mediating the lattice contact prevented CD28 expression at the cell surface, whereas mutations of residues equivalent to those involved in CTLA‐4 dimerization had smaller effects [<a href="#imr70014-bib-0034" class="usa-link" aria-describedby="imr70014-bib-0034">34</a>]. Our lattice‐based homodimer is therefore very likely to mimic the native CD28 ECD.</p>
<figure class="fig xbox font-sm" id="imr70014-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/05a4/11969063/8b250a44bafc/IMR-331-0-g007.jpg" loading="lazy" id="jats-graphic-11" height="492" width="709" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/imr70014-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>CD28 and CTLA‐4. (a) Ribbon and surface‐rendered views of the ECDs of apo CD28 (PDB: 1yjd) and CTLA‐4 (PDB: 3osk). The ligand (i.e., CD80) binding surface of CTLA‐4 incorporating the MYPPPY sequence, and the equivalent region of CD28, are indicated. (b, c) Superimposition of the apo CD28 and CTLA‐4 (chain A) Ig superfamily domains (b), and of the FG loop MYPPPY sequences of CD28 and the apo and complexed forms of CTLA‐4 (with CD80, PDB: 1i8l; with CD86, PDB: 1i85) (c). (d) The d2 clash resulting from superimposing CD80 monomers with the CD28 homodimer in the manner of CD80 binding by CTLA‐4 (PDB: 1i8l); for the CTLA‐4 homodimer, which forms a complex in which the CD80 monomers are parallel, there is no such clash. (e) Orthogonal views of the crystallographic CD28 ECD homodimer/5.11A1 Fab complex. The view in the upper panel is along the plane of the membrane. The position of a residue equivalent to one in rat CD28, which disrupts the binding of a non‐mitogenic anti‐rat CD28 antibody when mutated [<a href="#imr70014-bib-0112" class="usa-link" aria-describedby="imr70014-bib-0112">112</a>], is marked with a blue circle. (f) The homodimer formed in crystals of the CD80 ECD (PDB: 1dr9); the location of the CTLA‐4 binding site, determined subsequently [<a href="#imr70014-bib-0035" class="usa-link" aria-describedby="imr70014-bib-0035">35</a>], is indicated. (g) Structural differences between the CD80‐ligated (PDB: 1i8l) and apo CTLA‐4 homodimer (PDB: 3osk) (see Figure <a href="#imr70014-fig-0002" class="usa-link">2f</a> for details of the representation). (h) The repeating array of CTLA‐4 and CD80 ECD homodimers observed in crystals of the complex (PDB: 1i8l). (i) Modeling of glycosylation (yellow) at sites found to be occupied in crystals of CD28 (PDB: 1yjd; predicted CD80 binding site, green), CTLA‐4 (PDB: 3osk; CD80 binding site, green), and CD80 (PDB: 1dr9; CTLA‐4 binding site, blue), using GlycoSHIELD (as in Figure <a href="#imr70014-fig-0002" class="usa-link">2g</a>).</p></figcaption></figure><p>The V‐set Ig superfamily domain of CD28 is structurally most similar to that of CTLA‐4 [<a href="#imr70014-bib-0034" class="usa-link" aria-describedby="imr70014-bib-0034">34</a>]. Variations are found in the G β‐strand of CD28, as well as a reduction in the length of the CD28 CC' loop versus that of CTLA‐4, which would have prevented CD28 dimerization in the way observed in the crystal (Figure <a href="#imr70014-fig-0006" class="usa-link">6b</a>). The structure of the shared ligand‐binding MYPPPY motif of CD28, with the <em>cis‐trans‐cis</em> conformations of the three proline residues, is virtually identical to that of CTLA‐4 (Figure <a href="#imr70014-fig-0006" class="usa-link">6c</a>). The shared surface used by CTLA‐4 to bind to both CD80 and CD86 is preserved in CD28, but the surfaces mediating CTLA‐4/ligand complex formation outside this region differ in CD28 and account for the reduced affinity of CD28 for its ligands [<a href="#imr70014-bib-0034" class="usa-link" aria-describedby="imr70014-bib-0034">34</a>, <a href="#imr70014-bib-0107" class="usa-link" aria-describedby="imr70014-bib-0107">107</a>]. The lattice‐based CD28 dimer slightly reorients each subunit, explaining the differences in the valency of CTLA‐4 and CD28 [<a href="#imr70014-bib-0034" class="usa-link" aria-describedby="imr70014-bib-0034">34</a>]. Docking experiments suggested that bivalent ligand‐binding by CD28 is prevented by a steric ‘clash’ in the membrane‐proximal domains of bivalently‐bound CD80 (Figure <a href="#imr70014-fig-0006" class="usa-link">6d</a>), and that CD86 binding will be greatly constrained in the same manner. The mitogenic 5.11A1 antibody bound CD28 bivalently via a membrane‐proximal epitope, along an axis parallel to the membrane (Figure <a href="#imr70014-fig-0006" class="usa-link">6e</a>), in contrast to non‐mitogenic antibodies that bind epitopes further away from the membrane [<a href="#imr70014-bib-0034" class="usa-link" aria-describedby="imr70014-bib-0034">34</a>, <a href="#imr70014-bib-0112" class="usa-link" aria-describedby="imr70014-bib-0112">112</a>]. This suggested to us that the mitogenic antibody could induce signaling by locally excluding RPTPs such as CD45, when the antibody also forms complexes with Fc receptor‐expressing cells [<a href="#imr70014-bib-0034" class="usa-link" aria-describedby="imr70014-bib-0034">34</a>], as we subsequently demonstrated [<a href="#imr70014-bib-0110" class="usa-link" aria-describedby="imr70014-bib-0110">110</a>]. Our structure of the ECD of the CD28 and CTLA‐4 ligand, CD80 (Figure <a href="#imr70014-fig-0006" class="usa-link">6f</a>), published in 2000 [<a href="#imr70014-bib-0037" class="usa-link" aria-describedby="imr70014-bib-0037">37</a>], revealed that it forms a 2‐fold rotationally symmetric homodimer and consists of a novel combination of Ig superfamily domains, that is, a V‐set domain found in adhesion proteins and a C1‐set domain typical of antigen receptors.</p></section><section id="imr70014-sec-0010"><h3 class="pmc_sec_title">6.2. CTLA‐4</h3>
<p>CTLA‐4 suppresses signaling in T cells by recruiting cytosolic tyrosine phosphatases, for example, Src homology 2 domain‐containing tyrosine phosphatases (SHP) 1/2, which reverse tyrosine phosphorylation, and/or by impairing CD28 signaling directly by competing for their shared ligands [<a href="#imr70014-bib-0103" class="usa-link" aria-describedby="imr70014-bib-0103">103</a>], or indirectly via the capture and degradation of the ligands by trans‐endocytosis [<a href="#imr70014-bib-0113" class="usa-link" aria-describedby="imr70014-bib-0113">113</a>]. As expected, the CTLA‐4 V‐set Ig superfamily domain<a href="#imr70014-note-0002" class="usa-link">
<sup>2</sup>
</a> is most similar to that of CD28, and to a lesser extent, PD‐1, rather than those of antigen receptors and adhesion molecules [<a href="#imr70014-bib-0035" class="usa-link" aria-describedby="imr70014-bib-0035">35</a>, <a href="#imr70014-bib-0036" class="usa-link" aria-describedby="imr70014-bib-0036">36</a>, <a href="#imr70014-bib-0114" class="usa-link" aria-describedby="imr70014-bib-0114">114</a>]. Dimerization relies on (1) intermolecular disulfide bond formation, (2) hydrophobic interactions of the Ig superfamily domains involving the A′ and G strands of each monomer, and (3) three networks of water molecules that help rigidify the stalk, perhaps favoring ligand engagement [<a href="#imr70014-bib-0035" class="usa-link" aria-describedby="imr70014-bib-0035">35</a>, <a href="#imr70014-bib-0036" class="usa-link" aria-describedby="imr70014-bib-0036">36</a>, <a href="#imr70014-bib-0114" class="usa-link" aria-describedby="imr70014-bib-0114">114</a>].</p>
<p>The MYPPPY segment of CTLA‐4 dominates interactions with its ligands [<a href="#imr70014-bib-0035" class="usa-link" aria-describedby="imr70014-bib-0035">35</a>, <a href="#imr70014-bib-0114" class="usa-link" aria-describedby="imr70014-bib-0114">114</a>]. CTLA‐4 forms hydrophobic contacts with the mostly nonpolar surface of CD80 [<a href="#imr70014-bib-0035" class="usa-link" aria-describedby="imr70014-bib-0035">35</a>]. In further contrast to CD2 [<a href="#imr70014-bib-0029" class="usa-link" aria-describedby="imr70014-bib-0029">29</a>], the binding surfaces of CTLA‐4 and CD80 exhibit a high degree of shape complementarity [<a href="#imr70014-bib-0035" class="usa-link" aria-describedby="imr70014-bib-0035">35</a>]. The <em>Sc</em> metric quantifying this parameter [<a href="#imr70014-bib-0115" class="usa-link" aria-describedby="imr70014-bib-0115">115</a>] is 0.74–0.77 for the CTLA‐4/CD80 binding interface, which is considerably higher than that for CD2 and CD58 (0.58) [<a href="#imr70014-bib-0060" class="usa-link" aria-describedby="imr70014-bib-0060">60</a>], and even higher than that of antibody/protein antigen interfaces (0.64–0.68) [<a href="#imr70014-bib-0115" class="usa-link" aria-describedby="imr70014-bib-0115">115</a>]. However, the CD80 binding site of CTLA‐4 is relatively small, with only 460 Å<sup>2</sup> of surface being buried per monomer [<a href="#imr70014-bib-0035" class="usa-link" aria-describedby="imr70014-bib-0035">35</a>] (for CD2 and CD58, 570–590 Å<sup>2</sup> of surface is buried [<a href="#imr70014-bib-0060" class="usa-link" aria-describedby="imr70014-bib-0060">60</a>]). This helps to explain the fast kinetics of CTLA‐4/ligand interactions [<a href="#imr70014-bib-0107" class="usa-link" aria-describedby="imr70014-bib-0107">107</a>, <a href="#imr70014-bib-0116" class="usa-link" aria-describedby="imr70014-bib-0116">116</a>].</p>
<p>The binding of CD80 to CTLA‐4 is accompanied by minimal changes in the CTLA‐4 ECD (Figure <a href="#imr70014-fig-0006" class="usa-link">6g</a>) [<a href="#imr70014-bib-0036" class="usa-link" aria-describedby="imr70014-bib-0036">36</a>]. Notably, the Ig superfamily domains of the apo ECD homodimer, which are very similar, both exhibit more similarity with equivalent domains in the ligated structures than they do to each other, emphasizing the rigid‐body nature of complex formation. There is a small rotation in the sidechain of M99 (part of the MYPPPY motif) when binding CD86, as well as some changes in the position of the C‐terminus of the G strand, neither of which occurs when CTLA‐4 binds CD80 [<a href="#imr70014-bib-0036" class="usa-link" aria-describedby="imr70014-bib-0036">36</a>]. There are also small shifts in the positions of the C″ strand and the BC and CC′ loops. Because these changes are not produced by both CD80 and CD86, they are unlikely to function allosterically. Despite binding the shared MYPPPY motif, CD80 and CD86 interact with CTLA‐4 in different ways [<a href="#imr70014-bib-0035" class="usa-link" aria-describedby="imr70014-bib-0035">35</a>, <a href="#imr70014-bib-0036" class="usa-link" aria-describedby="imr70014-bib-0036">36</a>, <a href="#imr70014-bib-0114" class="usa-link" aria-describedby="imr70014-bib-0114">114</a>]. Whereas minimal reorganization of CD80 is needed to accommodate CTLA‐4, CD86 undergoes induced‐fit including, e.g., the rotation of the FG loop allowing contacts to form with the MYPPPY motif. Otherwise, ligand binding is rigid‐body in character [<a href="#imr70014-bib-0036" class="usa-link" aria-describedby="imr70014-bib-0036">36</a>]. Strikingly, in crystals, the CTLA‐4 and CD80 ECD dimers form a zipper‐like arrangement (Figure <a href="#imr70014-fig-0006" class="usa-link">6h</a>), revealing a structural basis for the formation of very stable inhibitory complexes at the T‐cell surface [<a href="#imr70014-bib-0035" class="usa-link" aria-describedby="imr70014-bib-0035">35</a>]. Models of the intact, glycosylated forms of the CD28, CTLA‐4, and CD80 homodimer ECDs are shown in Figure <a href="#imr70014-fig-0006" class="usa-link">6i</a>.</p></section><section id="imr70014-sec-0011"><h3 class="pmc_sec_title">6.3. PD‐1</h3>
<p>Inhibitory signaling by PD‐1 is at least partially explained by its recruitment of the tyrosine phosphatases, SHP1/2, after it's phosphorylated on cytosolic signaling motifs following ligand binding [<a href="#imr70014-bib-0117" class="usa-link" aria-describedby="imr70014-bib-0117">117</a>]. NMR analysis of the human PD‐1 ECD<a href="#imr70014-note-0003" class="usa-link">
<sup>3</sup>
</a> showed that it comprises a two‐layer β sandwich with the topology of Ig superfamily V‐set domains, i.e., GFCC′ and ABED sheets stabilized by a disulfide bond (Figure <a href="#imr70014-fig-0007" class="usa-link">7a</a>) [<a href="#imr70014-bib-0038" class="usa-link" aria-describedby="imr70014-bib-0038">38</a>]. Structure comparisons showed that it is most similar to the V‐set domains of antigen receptors and CTLA‐4, as anticipated by sequence comparisons [<a href="#imr70014-bib-0036" class="usa-link" aria-describedby="imr70014-bib-0036">36</a>]. The first PD‐1/PD‐L1 ECD complex solved (PDB: 3bik) revealed an <em>in trans</em> binding mode reminiscent of the <em>in cis</em> interactions of antigen receptor variable domains [<a href="#imr70014-bib-0119" class="usa-link" aria-describedby="imr70014-bib-0119">119</a>], suggesting an evolutionary path from <em>in trans</em> interacting monomeric receptors to <em>in cis</em> heterodimers, or <em>vice versa</em>. Compared with typical V‐set domains, however, the edge of the GFCC′ sheet of PD‐1 is unusually flexible, and it completely lacks a C″ strand. The crystal structure of the human PD‐1 ECD (PDB: 3rrq) confirmed that the C'D loop is unstructured and that the CC′ and FG loops are highly flexible (Figure <a href="#imr70014-fig-0007" class="usa-link">7b</a>). The ligand‐binding sites of PD‐1 were mapped via perturbations of PD‐1 backbone NMR signals in the presence of soluble PD‐L1 and PD‐L2 [<a href="#imr70014-bib-0038" class="usa-link" aria-describedby="imr70014-bib-0038">38</a>] (Figure <a href="#imr70014-fig-0007" class="usa-link">7c</a>). For PD‐L1, the changes were localized to a patch of surface residues on the GFCC′ β‐sheet: i.e., β‐strand C and the CC′ loop, β‐strand C′, the C'D loop, and β‐strands E and F, along with the FG loop, β‐strand G, and the BC and FG loops, defining the PD‐L1 binding site (as confirmed subsequently [<a href="#imr70014-bib-0118" class="usa-link" aria-describedby="imr70014-bib-0118">118</a>]). Although PD‐L1 and PD‐L2 bound the same region of PD‐1, for PD‐L2 there were fewer contributions from its BC and FG loops. Importantly, there was little or no indication that ligand‐induced changes occurred beyond the region of direct contact [<a href="#imr70014-bib-0038" class="usa-link" aria-describedby="imr70014-bib-0038">38</a>].</p>
<figure class="fig xbox font-sm" id="imr70014-fig-0007"><h4 class="obj_head">FIGURE 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/05a4/11969063/c0c8ce575e9d/IMR-331-0-g005.jpg" loading="lazy" id="jats-graphic-13" height="456" width="709" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/imr70014-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>PD‐1. (a) Ribbon representation of the PD‐1 ECD (PDB: 2m2d). (b) Comparison of the NMR and the crystal (PDB: 3rrq) structures of the PD‐1 ECD. (c) Surface renderings of PD‐1 (PDB: 2m2d) showing how the structure is affected by ligand binding, based on how the backbone (<sup>15</sup>N, <sup>13</sup>C′, and <sup>1</sup>H<sup>N</sup>) NMR signals of surface residues are perturbed by PD‐L1. Residues with minimal shift values lower than 1 standard deviation (SD) versus the average for all residues are colored white, those &gt; 1 SD yellow, &gt; 1.5 SD orange, and residues with minimal shifts &gt; 2 SD are colored red. Both the ligand‐binding face and the back of the molecule are shown. Very similar data were obtained for PD‐L2 binding [<a href="#imr70014-bib-0038" class="usa-link" aria-describedby="imr70014-bib-0038">38</a>]. (d) Crystal structures of the complexes of Clone 19 (PDB: 9hk1), nivolumab (PDB: 5wt9), and pembrolizumab (PDB: 5b8c) antibody Fv or Fab fragments, with the PD‐1 ECD. For reference, the PD‐L1 binding site on PD‐1 is also indicated [<a href="#imr70014-bib-0118" class="usa-link" aria-describedby="imr70014-bib-0118">118</a>]. (e) Structural differences between apo PD‐1 ECD (PDB: 2m2d), and the PD‐1 ECD in the Clone 19 Fab/PD‐1 ECD complex (PDB: 9hk1; see Figure <a href="#imr70014-fig-0002" class="usa-link">2f</a> for details of the representation). (f) Modeling of glycosylation (yellow) at sites found to be occupied in structures of the PD‐1 ECD, using GlycoSHIELD (as in Figure <a href="#imr70014-fig-0002" class="usa-link">2g</a>).</p></figcaption></figure><p>The possibility that the anti‐CD28 antibody we crystallized with CD28 was agonistic because it bound a membrane‐proximal epitope [<a href="#imr70014-bib-0034" class="usa-link" aria-describedby="imr70014-bib-0034">34</a>] prompted the generation of an anti‐PD‐1 antibody agonist, Clone 19, for use as a potential suppressor of inflammatory processes [<a href="#imr70014-bib-0110" class="usa-link" aria-describedby="imr70014-bib-0110">110</a>]. The structure of the ECD of PD‐1 complexed with the Fab fragment of Clone 19 (PDB: 9hk1) confirmed that the antibody binds to the base of β‐strands A, F, and G, on the opposite side of the protein from the PD‐L1/PD‐L2 binding site (Figure <a href="#imr70014-fig-0007" class="usa-link">7d</a>) [<a href="#imr70014-bib-0110" class="usa-link" aria-describedby="imr70014-bib-0110">110</a>]. In contrast, ligand‐blocking anti‐PD‐1 antibodies used in the clinic, i.e., nivolumab and pembrolizumab, bind closer to the top of PD‐1 (PDB: 5b8c, PD‐1/pembrolizumab Fab complex; PDB: 5wt9, PD‐1/nivolumab Fab complex; Figure <a href="#imr70014-fig-0007" class="usa-link">7d</a>) [<a href="#imr70014-bib-0110" class="usa-link" aria-describedby="imr70014-bib-0110">110</a>]. We found a striking inverse correlation between epitope position relative to the membrane and the level of agonistic signaling elicited, insofar as the three antibodies were agonistic in the following manner: Clone 19 &gt; pembrolizumab &gt; nivolumab [<a href="#imr70014-bib-0110" class="usa-link" aria-describedby="imr70014-bib-0110">110</a>]. This supported the general idea that the closer an antibody binds to the membrane, the better able it is to create small gaps between the receptor‐expressing target cell and Fc receptor‐expressing cells that co‐engage the antibody, leading to the more efficient exclusion of large RPTPs such as CD45 and stronger signaling [<a href="#imr70014-bib-0110" class="usa-link" aria-describedby="imr70014-bib-0110">110</a>]. Comparison of the apo PD‐1 ECD and the PD‐1 ECD/Clone 19 Fab complex indicated that the structural core of PD‐1 is largely unaffected by antibody binding (Figure <a href="#imr70014-fig-0007" class="usa-link">7e</a>) [<a href="#imr70014-bib-0110" class="usa-link" aria-describedby="imr70014-bib-0110">110</a>], indicating that, like the native PD‐1 ligands, signaling by Clone 19 is not dependent on large‐scale structural rearrangements. A model of the intact glycosylated form of the PD‐1 ECD is shown in Figure <a href="#imr70014-fig-0007" class="usa-link">7f</a>.</p></section></section><section id="imr70014-sec-0012"><h2 class="pmc_sec_title">7. Conclusions</h2>
<p>Our structural work on CD2 and its ligands, CD45, αβ‐ and γδ‐TCRs, and the accessory proteins, CD28 and CTLA‐4 and their ligand CD80, and PD‐1, offers a number of insights into the early events underpinning antigen detection and signaling at close contacts.</p>
<p>Its linker region, which lengthens CD2 and positions its ligand‐binding site close to the top of the protein, and its flexibility [<a href="#imr70014-bib-0027" class="usa-link" aria-describedby="imr70014-bib-0027">27</a>, <a href="#imr70014-bib-0028" class="usa-link" aria-describedby="imr70014-bib-0028">28</a>], are likely to favor head‐to‐head binding of CD2 to its ubiquitously expressed and relatively abundant ligands and facilitate close‐contact formation during the early stages of cell–cell contact. The head‐to‐head lattice contact seen in all our CD2 ECD crystals provided a model of the ligand complexes that CD2 forms, measuring ~15 nm along an axis orthogonal to the membrane [<a href="#imr70014-bib-0027" class="usa-link" aria-describedby="imr70014-bib-0027">27</a>, <a href="#imr70014-bib-0028" class="usa-link" aria-describedby="imr70014-bib-0028">28</a>]. These complexes are likely to set the ‘height’ of the close contact, and it is now clear that a large number of receptors form complexes matching these dimensions [<a href="#imr70014-bib-0120" class="usa-link" aria-describedby="imr70014-bib-0120">120</a>], allowing them to function at close contacts depending on when and where their ligands are expressed. The structural plasticity inherent in the complexes CD2 forms with its related ligands suggests that the evolutionary diversification of these proteins was rapid and constrained only by binding needing to be weak and specific, and in the case of CD2, its topology and dimensions being maintained, which will have required only that its linker region is conserved. Overall, CD2 and CTLA‐4 provide what are likely to be extreme examples of low‐affinity, highly‐specific protein binding modes at the cell surface: one reliant on poor surface‐shape complementarity and electrostatic contact (i.e., CD2) [<a href="#imr70014-bib-0029" class="usa-link" aria-describedby="imr70014-bib-0029">29</a>], and the other on small, hydrophobic surfaces with considerable shape complementarity (i.e., CTLA‐4) [<a href="#imr70014-bib-0035" class="usa-link" aria-describedby="imr70014-bib-0035">35</a>, <a href="#imr70014-bib-0114" class="usa-link" aria-describedby="imr70014-bib-0114">114</a>].</p>
<p>The structural work confirmed that the ECDs of all forms of CD45 extend beyond the 15 nm inter‐membrane gap created by CD2 and its ligands at close contacts [<a href="#imr70014-bib-0031" class="usa-link" aria-describedby="imr70014-bib-0031">31</a>]. The smallest isoform of CD45, i.e., CD45R0, is 21.6 nm, and the RABC isoform is larger, as expected [<a href="#imr70014-bib-0031" class="usa-link" aria-describedby="imr70014-bib-0031">31</a>]. Size differences as small as 5 nm are enough to drive unbound proteins from membrane interfaces created by smaller complexes of binding proteins [<a href="#imr70014-bib-0121" class="usa-link" aria-describedby="imr70014-bib-0121">121</a>]. This indicates that all forms of CD45 would be physically excluded from close contacts created by CD2/ligand interactions, as we observed in the case of Jurkat T‐cells (Figure <a href="#imr70014-fig-0001" class="usa-link">1b</a> [<a href="#imr70014-bib-0023" class="usa-link" aria-describedby="imr70014-bib-0023">23</a>]), which mostly express the smaller forms of CD45 [<a href="#imr70014-bib-0122" class="usa-link" aria-describedby="imr70014-bib-0122">122</a>]. Altogether, the structural data indicated that the fine‐structure of the CD45 ECD is less important than its mechanical properties given, on the one hand, the degenerate structures of its constituent domains and poor sequence conservation beyond the domain interfaces, and the tight domain packing and disulfide bond‐based ‘cross‐linking’ of the ECD, on the other [<a href="#imr70014-bib-0031" class="usa-link" aria-describedby="imr70014-bib-0031">31</a>]. This can be explained by the ECD of CD45 functioning primarily as a physical lever, driving the phosphatase out of close contacts [<a href="#imr70014-bib-0031" class="usa-link" aria-describedby="imr70014-bib-0031">31</a>].</p>
<p>Whereas the ligand‐binding site of CD2 is at the top [<a href="#imr70014-bib-0027" class="usa-link" aria-describedby="imr70014-bib-0027">27</a>, <a href="#imr70014-bib-0028" class="usa-link" aria-describedby="imr70014-bib-0028">28</a>], the binding surfaces of the αβ TCR [<a href="#imr70014-bib-0032" class="usa-link" aria-describedby="imr70014-bib-0032">32</a>], CD28 [<a href="#imr70014-bib-0034" class="usa-link" aria-describedby="imr70014-bib-0034">34</a>], CTLA‐4 [<a href="#imr70014-bib-0035" class="usa-link" aria-describedby="imr70014-bib-0035">35</a>, <a href="#imr70014-bib-0114" class="usa-link" aria-describedby="imr70014-bib-0114">114</a>], and CD80 [<a href="#imr70014-bib-0037" class="usa-link" aria-describedby="imr70014-bib-0037">37</a>] are all toward or on the side of each structure. For the αβ TCR, we proposed that positioning the ligand‐binding site in this way, rather than at the top, serves two important purposes [<a href="#imr70014-bib-0032" class="usa-link" aria-describedby="imr70014-bib-0032">32</a>]. First, the TCR does not have to form head‐to‐head complexes in the short period of time during which close contacts are created by CD2 and its ligands, or after contact is made and only ‘sideways’ movement is possible. Once close contacts form, the tilted arrangement of the TCR‐αβ subunits and the location of the pMHC ligand‐binding site on the side of the TCR would result in the TCR and pMHC‐binding sites overlapping, facilitating productive scanning by the TCR for rare pMHC ligands via sideways interactions. Second, TCR/pMHC binding in this manner positions the pMHC for favorable interactions with co‐receptors, explaining why their recruitment follows TCR/pMHC engagement and initial signaling [<a href="#imr70014-bib-0010" class="usa-link" aria-describedby="imr70014-bib-0010">10</a>]. We proposed that TCRs engaging pMHC with reverse topology [<a href="#imr70014-bib-0123" class="usa-link" aria-describedby="imr70014-bib-0123">123</a>] may fail to initiate strong signaling because the co‐receptor binding site is inaccessible once stable close contacts form [<a href="#imr70014-bib-0032" class="usa-link" aria-describedby="imr70014-bib-0032">32</a>]. Similar arguments can be applied to CD28 and CTLA‐4. These observations suggest that receptor topology could have a large impact on the sequence of signaling events at close contacts. In marked contrast to these proteins, the ligand‐binding regions of γδ TCRs are topologically unconstrained [<a href="#imr70014-bib-0033" class="usa-link" aria-describedby="imr70014-bib-0033">33</a>, <a href="#imr70014-bib-0097" class="usa-link" aria-describedby="imr70014-bib-0097">97</a>] so that these receptors can engage a variety of structurally‐distinct ligands [<a href="#imr70014-bib-0092" class="usa-link" aria-describedby="imr70014-bib-0092">92</a>]. Strikingly, the maximum dimensions of the γδ TCR are nevertheless constrained by the fixed‐length TCR‐δ CP regions. This could ensure that highly flexible γδ TCRs are trapped in the shallow, robustly CD45‐excluding close contacts created by CD2 and its ligands rather than deeper regions of contact stabilized by larger e.g., ICAM‐1/LFA‐1 complexes, allowing more‐productive signaling.</p>
<p>But what of receptor triggering per se? The signaling receptors we chose to study belong to the set of ~100 “immune receptors” characterized by (1) having mostly small ECDs (&lt; 20 nm), (2) their binding to ligands anchored to the surfaces of other cells, (3) having unstructured cytosolic regions with multiple tyrosine residues and no intrinsic catalytic activity, and (4) being phosphorylated and dephosphorylated by membrane‐associated, extrinsic tyrosine kinases, e.g., Lck, and phosphatases, such as CD45, respectively [<a href="#imr70014-bib-0120" class="usa-link" aria-describedby="imr70014-bib-0120">120</a>]. To us and to others [<a href="#imr70014-bib-0008" class="usa-link" aria-describedby="imr70014-bib-0008">8</a>, <a href="#imr70014-bib-0120" class="usa-link" aria-describedby="imr70014-bib-0120">120</a>], these similarities imply that immune receptors are triggered via a single, shared mechanism, that is, they are 'all‐in or none‐in'. A corollary of this proposition is that insights obtained for any one receptor will apply to the others, which we used to justify our initial focus on receptors that could be studied easily. The work on CTLA‐4 offered the first and, perhaps, clearest evidence that immune receptor triggering does not invariably depend on spontaneous conformational changes [<a href="#imr70014-bib-0036" class="usa-link" aria-describedby="imr70014-bib-0036">36</a>]. This seems also to be true of αβ TCR triggering [<a href="#imr70014-bib-0032" class="usa-link" aria-describedby="imr70014-bib-0032">32</a>] and antibody‐induced signaling [<a href="#imr70014-bib-0110" class="usa-link" aria-describedby="imr70014-bib-0110">110</a>], but as we have sought to emphasize throughout, rigid‐body binding is a shared feature of all the interactions we have studied. Whilst the role of forces in immune receptor signaling has not been explored very broadly, we note that the γδ TCR is not a mechanosensor [<a href="#imr70014-bib-0124" class="usa-link" aria-describedby="imr70014-bib-0124">124</a>], ruling this out as a general requirement of immune receptor triggering. The structural basis for CD45 exclusion from close contacts formed by the interaction of CD2 with its ligands now seems to be settled, however. It follows that the phosphorylation of any immune receptor would be potentiated by being trapped in close contacts by its ligands, just as slowing the diffusion of the αβ TCR at close contacts suffices to initiate strong signaling [<a href="#imr70014-bib-0125" class="usa-link" aria-describedby="imr70014-bib-0125">125</a>], satisfying the all‐in or none‐in requirement.</p></section><section id="imr70014-sec-0013"><h2 class="pmc_sec_title">8. Afterword</h2>
<p>Recently, a preprint published by Notti et al. [<a href="#imr70014-bib-0126" class="usa-link" aria-describedby="imr70014-bib-0126">126</a>] addressed the question of how the αβ TCR might be configured in a membrane‐like environment, rather than in detergent micelles. In their experiments, Notti et al. reconstituted detergent‐solubilized TCRs into lipid nanodiscs containing a mixture of polar lipids and cholesteryl hemisuccinate, and determined their structures using cryo‐EM. Rather than the largely identical conformations of unligated and ligated receptors observed by ourselves and others for detergent‐solubilized receptors [<a href="#imr70014-bib-0032" class="usa-link" aria-describedby="imr70014-bib-0032">32</a>, <a href="#imr70014-bib-0087" class="usa-link" aria-describedby="imr70014-bib-0087">87</a>, <a href="#imr70014-bib-0090" class="usa-link" aria-describedby="imr70014-bib-0090">90</a>], “closed and compacted” conformations of the unligated receptor were observed that were 3.5 nm shorter than the ligand‐bound complex. These striking findings confound the notions (1) that CD2/ligand binding and close‐contact formation favors pMHC engagement by the TCR by creating an overlap in the positions of their binding sites [<a href="#imr70014-bib-0032" class="usa-link" aria-describedby="imr70014-bib-0032">32</a>], and (2) that the TCR engages ligands in a rigid‐body manner [<a href="#imr70014-bib-0032" class="usa-link" aria-describedby="imr70014-bib-0032">32</a>, <a href="#imr70014-bib-0090" class="usa-link" aria-describedby="imr70014-bib-0090">90</a>]. It will be important to carefully determine the extent to which the TCR occupies these various states at the T‐cell surface.</p></section><section id="imr70014-sec-0015"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="imr70014-sec-0014" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors have been fortunate enough to have worked with a large number of world‐class structural biologists over several decades. They have been E. Yvonne Jones, Karl Harlos, Robert Esnouf, Robert J.C. Gilbert, A. Radu Aricescu, and David I. Stuart (University of Oxford; CD2, CD28, CD48, CD45, CD58, and CD80); Shinji Ikemizu (Kumamoto University; CD48, CD58, CD80, and apo and complexed forms of CTLA‐4), Mark Carr (University of Leicester; PD‐1), Lukas Sušac, Christoph Thomas, and Robert Tampé (Goethe University Frankfurt; αβ TCR); and Benjamin S. Gully, Hariprasad Venugopal, and Jamie Rossjohn (Monash University; γδ TCR). We have also benefitted very significantly from long‐term collaborations with David Klenerman and Steven Lee (University of Cambridge), and with Omer Dushek and P. Anton van der Merwe (University of Oxford). Edward Jenkins (University of Oxford) and Markus Körbel (University of Cambridge) pioneered the fluorescence imaging of close‐contact formation on second‐generation bilayers (Figure <a href="#imr70014-fig-0001" class="usa-link">1</a>).</p></section><section id="notes1"><section id="imr70014-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="imr70014-note-0001"><p>Yuan Lui and João Ferreira Fernandes contributed equally to this article.</p></div>
<div class="fn p" id="imr70014-note-0101"><p>
<strong>Funding:</strong> This work was supported by the Wellcome Trust, 207547/Z/17/Z and 215883/Z/19/Z, and Cancer Research UK, DRCCIPA\100010.</p></div>
</div></section></section><section id="imr70014-ntgp-0002" class="fn-group"><h2 class="pmc_sec_title">Endnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="imr70014-note-0002">
<sup>
<sup>1</sup>
</sup><p class="display-inline">To emphasize the impact of O‐ and/or N‐glycosylation on their structures and interactions, we have generated structural models of the proteins we have studied, each with 50 representative glycan conformers at each of the glycosylation sites found to be occupied with sugar during the structure determinations. The glycan conformers were generated from 3 μs molecular dynamics simulations by GlycoSHIELD [<a href="#imr70014-bib-0061" class="usa-link" aria-describedby="imr70014-bib-0061">61</a>]. No attempt has been made to model the constraining effects of the cell surface at membrane‐proximal sites.</p>
</div>
<div class="fn p" id="imr70014-note-0003">
<sup>
<sup>2</sup>
</sup><p class="display-inline">The structure of the complex of soluble forms of CD80 and CTLA‐4 was solved using the coordinates of the CD80 ECD structure [<a href="#imr70014-bib-0037" class="usa-link" aria-describedby="imr70014-bib-0037">37</a>], by Stamper et al. of Wyeth Research [<a href="#imr70014-bib-0035" class="usa-link" aria-describedby="imr70014-bib-0035">35</a>].</p>
</div>
<div class="fn p" id="imr70014-note-0004">
<sup>
<sup>3</sup>
</sup><p class="display-inline">Expression and NMR analysis of soluble forms of the human PD‐1 ECD in E. coli were undertaken by Meryn Griffiths et al. at UCB Pharma and by M. Carr and colleagues at the University of Leicester, respectively [<a href="#imr70014-bib-0038" class="usa-link" aria-describedby="imr70014-bib-0038">38</a>]; crystallographic studies of the PD‐1 ECD expressed in mammalian cells were done by C. Paluch et al. at Gilead Sciences [<a href="#imr70014-bib-0110" class="usa-link" aria-describedby="imr70014-bib-0110">110</a>].</p>
</div>
</div></section><section id="imr70014-sec-0017"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The data that support the findings of this study are available from the corresponding author upon reasonable request.</p></section><section id="imr70014-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="imr70014-bibl-0001_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="imr70014-bib-0001">
<span class="label">1.</span><cite>
Germain R. N., Robey E. A., and Cahalan M. D., “A Decade of Imaging Cellular Motility and Interaction Dynamics in the Immune System,” Science
336, no. 6089 (2012): 1676–1681.
</cite> [<a href="https://doi.org/10.1126/science.1221063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3405774/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22745423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A%20Decade%20of%20Imaging%20Cellular%20Motility%20and%20Interaction%20Dynamics%20in%20the%20Immune%20System&amp;author=R.%20N.%20Germain&amp;author=E.%20A.%20Robey&amp;author=M.%20D.%20Cahalan&amp;volume=336&amp;issue=6089&amp;publication_year=2012&amp;pages=1676-1681&amp;pmid=22745423&amp;doi=10.1126/science.1221063&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0002">
<span class="label">2.</span><cite>
Cai E., Marchuk K., Beemiller P., et al., “Visualizing Dynamic Microvillar Search and Stabilization During Ligand Detection by T Cells,” Science
356, no. 6338 (2017): 3118, 10.1126/science.aal3118.</cite> [<a href="https://doi.org/10.1126/science.aal3118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6364556/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28495700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Visualizing%20Dynamic%20Microvillar%20Search%20and%20Stabilization%20During%20Ligand%20Detection%20by%20T%20Cells&amp;author=E.%20Cai&amp;author=K.%20Marchuk&amp;author=P.%20Beemiller&amp;volume=356&amp;issue=6338&amp;publication_year=2017&amp;pages=3118&amp;pmid=28495700&amp;doi=10.1126/science.aal3118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0003">
<span class="label">3.</span><cite>
Davis M. M. and Bjorkman P. J., “T‐Cell Antigen Receptor Genes and T‐Cell Recognition,” Nature
334, no. 6181 (1988): 395–402.
</cite> [<a href="https://doi.org/10.1038/334395a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3043226/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=T%E2%80%90Cell%20Antigen%20Receptor%20Genes%20and%20T%E2%80%90Cell%20Recognition&amp;author=M.%20M.%20Davis&amp;author=P.%20J.%20Bjorkman&amp;volume=334&amp;issue=6181&amp;publication_year=1988&amp;pages=395-402&amp;pmid=3043226&amp;doi=10.1038/334395a0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0004">
<span class="label">4.</span><cite>
Oliveira G. and Wu C. J., “Dynamics and Specificities of T Cells in Cancer Immunotherapy,” Nature Reviews. Cancer
23, no. 5 (2023): 295–316.
</cite> [<a href="https://doi.org/10.1038/s41568-023-00560-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10773171/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37046001/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews.%20Cancer&amp;title=Dynamics%20and%20Specificities%20of%20T%20Cells%20in%20Cancer%20Immunotherapy&amp;author=G.%20Oliveira&amp;author=C.%20J.%20Wu&amp;volume=23&amp;issue=5&amp;publication_year=2023&amp;pages=295-316&amp;pmid=37046001&amp;doi=10.1038/s41568-023-00560-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0005">
<span class="label">5.</span><cite>
Mallis R. J., Brazin K. N., Duke‐Cohan J. S., et al., “Biophysical and Structural Features of αβT‐Cell Receptor Mechanosensing: A Paradigmatic Shift in Understanding T‐Cell Activation,” Immunological Reviews
329, no. 1 (2025): e13432, 10.1111/imr.13432.
</cite> [<a href="https://doi.org/10.1111/imr.13432" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11744257/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39745432/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunological%20Reviews&amp;title=Biophysical%20and%20Structural%20Features%20of%20%CE%B1%CE%B2T%E2%80%90Cell%20Receptor%20Mechanosensing:%20A%20Paradigmatic%20Shift%20in%20Understanding%20T%E2%80%90Cell%20Activation&amp;author=R.%20J.%20Mallis&amp;author=K.%20N.%20Brazin&amp;author=J.%20S.%20Duke%E2%80%90Cohan&amp;volume=329&amp;issue=1&amp;publication_year=2025&amp;pages=e13432&amp;pmid=39745432&amp;doi=10.1111/imr.13432&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0006">
<span class="label">6.</span><cite>
Dustin M. L., “Recent Advances in Understanding TCR Signaling: A Synaptic Perspective,” Faculty Reviews
12 (2023): 25.
</cite> [<a href="https://doi.org/10.12703/r/12-25" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10608137/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37900153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Faculty%20Reviews&amp;title=Recent%20Advances%20in%20Understanding%20TCR%20Signaling:%20A%20Synaptic%20Perspective&amp;author=M.%20L.%20Dustin&amp;volume=12&amp;publication_year=2023&amp;pages=25&amp;pmid=37900153&amp;doi=10.12703/r/12-25&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0007">
<span class="label">7.</span><cite>
Davis S. J. and van der Merwe P. A., “The Structure and Ligand Interactions of CD2: Implications for T‐Cell Function,” Immunology Today
17, no. 4 (1996): 177–187.
</cite> [<a href="https://doi.org/10.1016/0167-5699(96)80617-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8871350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunology%20Today&amp;title=The%20Structure%20and%20Ligand%20Interactions%20of%20CD2:%20Implications%20for%20T%E2%80%90Cell%20Function&amp;author=S.%20J.%20Davis&amp;author=P.%20A.%20van%20der%20Merwe&amp;volume=17&amp;issue=4&amp;publication_year=1996&amp;pages=177-187&amp;pmid=8871350&amp;doi=10.1016/0167-5699(96)80617-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0008">
<span class="label">8.</span><cite>
Davis S. J. and van der Merwe P. A., “The Kinetic‐Segregation Model: TCR Triggering and Beyond,” Nature Immunology
7, no. 8 (2006): 803–809.
</cite> [<a href="https://doi.org/10.1038/ni1369" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16855606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Immunology&amp;title=The%20Kinetic%E2%80%90Segregation%20Model:%20TCR%20Triggering%20and%20Beyond&amp;author=S.%20J.%20Davis&amp;author=P.%20A.%20van%20der%20Merwe&amp;volume=7&amp;issue=8&amp;publication_year=2006&amp;pages=803-809&amp;pmid=16855606&amp;doi=10.1038/ni1369&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0009">
<span class="label">9.</span><cite>
Xu H. and Littman D. R., “A Kinase‐Independent Function of Lck in Potentiating Antigen‐Specific T Cell Activation,” Cell
74, no. 4 (1993): 633–643.
</cite> [<a href="https://doi.org/10.1016/0092-8674(93)90511-n" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8358792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20Kinase%E2%80%90Independent%20Function%20of%20Lck%20in%20Potentiating%20Antigen%E2%80%90Specific%20T%20Cell%20Activation&amp;author=H.%20Xu&amp;author=D.%20R.%20Littman&amp;volume=74&amp;issue=4&amp;publication_year=1993&amp;pages=633-643&amp;pmid=8358792&amp;doi=10.1016/0092-8674(93)90511-n&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0010">
<span class="label">10.</span><cite>
Jiang N., Huang J., Edwards L. J., et al., “Two‐Stage Cooperative T Cell Receptor‐Peptide Major Histocompatibility Complex‐CD8 Trimolecular Interactions Amplify Antigen Discrimination,” Immunity
34, no. 1 (2011): 13–23, 10.1016/j.immuni.2010.12.017.
</cite> [<a href="https://doi.org/10.1016/j.immuni.2010.12.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3381515/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21256056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Two%E2%80%90Stage%20Cooperative%20T%20Cell%20Receptor%E2%80%90Peptide%20Major%20Histocompatibility%20Complex%E2%80%90CD8%20Trimolecular%20Interactions%20Amplify%20Antigen%20Discrimination&amp;author=N.%20Jiang&amp;author=J.%20Huang&amp;author=L.%20J.%20Edwards&amp;volume=34&amp;issue=1&amp;publication_year=2011&amp;pages=13-23&amp;pmid=21256056&amp;doi=10.1016/j.immuni.2010.12.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0011">
<span class="label">11.</span><cite>
Casas J., Brzostek J., Zarnitsyna V. I., et al., “Ligand‐Engaged TCR Is Triggered by Lck Not Associated With CD8 Coreceptor,” Nature Communications
5, no. 1 (2014): 5624.</cite> [<a href="https://doi.org/10.1038/ncomms6624" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4248239/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25427562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Ligand%E2%80%90Engaged%20TCR%20Is%20Triggered%20by%20Lck%20Not%20Associated%20With%20CD8%20Coreceptor&amp;author=J.%20Casas&amp;author=J.%20Brzostek&amp;author=V.%20I.%20Zarnitsyna&amp;volume=5&amp;issue=1&amp;publication_year=2014&amp;pages=5624&amp;pmid=25427562&amp;doi=10.1038/ncomms6624&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0012">
<span class="label">12.</span><cite>
Courtney A. H., Lo W. L., and Weiss A., “TCR Signaling: Mechanisms of Initiation and Propagation,” Trends in Biochemical Sciences
43, no. 2 (2018): 108–123.
</cite> [<a href="https://doi.org/10.1016/j.tibs.2017.11.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5801066/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29269020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Biochemical%20Sciences&amp;title=TCR%20Signaling:%20Mechanisms%20of%20Initiation%20and%20Propagation&amp;author=A.%20H.%20Courtney&amp;author=W.%20L.%20Lo&amp;author=A.%20Weiss&amp;volume=43&amp;issue=2&amp;publication_year=2018&amp;pages=108-123&amp;pmid=29269020&amp;doi=10.1016/j.tibs.2017.11.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0013">
<span class="label">13.</span><cite>
Su X., Ditlev J. A., Hui E., et al., “Phase Separation of Signaling Molecules Promotes T Cell Receptor Signal Transduction,” Science
352, no. 6285 (2016): 595–599, 10.1126/science.aad9964.
</cite> [<a href="https://doi.org/10.1126/science.aad9964" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4892427/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27056844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Phase%20Separation%20of%20Signaling%20Molecules%20Promotes%20T%20Cell%20Receptor%20Signal%20Transduction&amp;author=X.%20Su&amp;author=J.%20A.%20Ditlev&amp;author=E.%20Hui&amp;volume=352&amp;issue=6285&amp;publication_year=2016&amp;pages=595-599&amp;pmid=27056844&amp;doi=10.1126/science.aad9964&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0014">
<span class="label">14.</span><cite>
McAffee D. B., O'Dair M. K., Lin J. J., et al., “Discrete LAT Condensates Encode Antigen Information From Single pMHC:TCR Binding Events,” Nature Communications
13, no. 1 (2022): 7446, 10.1038/s41467-022-35093-9.</cite> [<a href="https://doi.org/10.1038/s41467-022-35093-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9718779/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36460640/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Discrete%20LAT%20Condensates%20Encode%20Antigen%20Information%20From%20Single%20pMHC:TCR%20Binding%20Events&amp;author=D.%20B.%20McAffee&amp;author=M.%20K.%20O'Dair&amp;author=J.%20J.%20Lin&amp;volume=13&amp;issue=1&amp;publication_year=2022&amp;pages=7446&amp;pmid=36460640&amp;doi=10.1038/s41467-022-35093-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0015">
<span class="label">15.</span><cite>
Edner N. M., Carlesso G., Rush J. S., and Walker L. S. K., “Targeting Co‐Stimulatory Molecules in Autoimmune Disease,” Nature Reviews. Drug Discovery
19, no. 12 (2020): 860–883.
</cite> [<a href="https://doi.org/10.1038/s41573-020-0081-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32939077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews.%20Drug%20Discovery&amp;title=Targeting%20Co%E2%80%90Stimulatory%20Molecules%20in%20Autoimmune%20Disease&amp;author=N.%20M.%20Edner&amp;author=G.%20Carlesso&amp;author=J.%20S.%20Rush&amp;author=L.%20S.%20K.%20Walker&amp;volume=19&amp;issue=12&amp;publication_year=2020&amp;pages=860-883&amp;pmid=32939077&amp;doi=10.1038/s41573-020-0081-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0016">
<span class="label">16.</span><cite>
Burke K. P., Chaudhri A., Freeman G. J., and Sharpe A. H., “The B7:CD28 Family and Friends: Unraveling Coinhibitory Interactions,” Immunity
57, no. 2 (2024): 223–244.
</cite> [<a href="https://doi.org/10.1016/j.immuni.2024.01.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10889489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38354702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=The%20B7:CD28%20Family%20and%20Friends:%20Unraveling%20Coinhibitory%20Interactions&amp;author=K.%20P.%20Burke&amp;author=A.%20Chaudhri&amp;author=G.%20J.%20Freeman&amp;author=A.%20H.%20Sharpe&amp;volume=57&amp;issue=2&amp;publication_year=2024&amp;pages=223-244&amp;pmid=38354702&amp;doi=10.1016/j.immuni.2024.01.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0017">
<span class="label">17.</span><cite>
Chin‐Hun Kuo J., Gandhi J. G., Zia R. N., and Paszek M. J., “Physical Biology of the Cancer Cell Glycocalyx,” Nature Physics
14, no. 7 (2018): 658–669.
</cite> [<a href="https://doi.org/10.1038/s41567-018-0186-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8046174/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33859716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Physics&amp;title=Physical%20Biology%20of%20the%20Cancer%20Cell%20Glycocalyx&amp;author=J.%20Chin%E2%80%90Hun%20Kuo&amp;author=J.%20G.%20Gandhi&amp;author=R.%20N.%20Zia&amp;author=M.%20J.%20Paszek&amp;volume=14&amp;issue=7&amp;publication_year=2018&amp;pages=658-669&amp;pmid=33859716&amp;doi=10.1038/s41567-018-0186-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0018">
<span class="label">18.</span><cite>
Belardi B., Son S., Felce J. H., Dustin M. L., and Fletcher D. A., “Cell‐Cell Interfaces as Specialized Compartments Directing Cell Function,” Nature Reviews. Molecular Cell Biology
21, no. 12 (2020): 750–764.
</cite> [<a href="https://doi.org/10.1038/s41580-020-00298-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33093672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews.%20Molecular%20Cell%20Biology&amp;title=Cell%E2%80%90Cell%20Interfaces%20as%20Specialized%20Compartments%20Directing%20Cell%20Function&amp;author=B.%20Belardi&amp;author=S.%20Son&amp;author=J.%20H.%20Felce&amp;author=M.%20L.%20Dustin&amp;author=D.%20A.%20Fletcher&amp;volume=21&amp;issue=12&amp;publication_year=2020&amp;pages=750-764&amp;pmid=33093672&amp;doi=10.1038/s41580-020-00298-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0019">
<span class="label">19.</span><cite>
Brown W. R., Barclay A. N., Sunderland C. A., and Williams A. F., “Identification of a Glycophorin‐Like Molecule at the Cell Surface of Rat Thymocytes,” Nature
289, no. 5797 (1981): 456–460.
</cite> [<a href="https://doi.org/10.1038/289456a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6970337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Identification%20of%20a%20Glycophorin%E2%80%90Like%20Molecule%20at%20the%20Cell%20Surface%20of%20Rat%20Thymocytes&amp;author=W.%20R.%20Brown&amp;author=A.%20N.%20Barclay&amp;author=C.%20A.%20Sunderland&amp;author=A.%20F.%20Williams&amp;volume=289&amp;issue=5797&amp;publication_year=1981&amp;pages=456-460&amp;pmid=6970337&amp;doi=10.1038/289456a0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0020">
<span class="label">20.</span><cite>
Williams A. F. and Barclay A. N., “Glycoprotein Antigens of the Lymphocyte Surface and Their Purification by Antibody Affinity Chromatography,” in Handbook of Experimental Immunology (Blackwell Scientific Publications, 1986), 22.21–22.24.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Handbook%20of%20Experimental%20Immunology&amp;author=A.%20F.%20Williams&amp;author=A.%20N.%20Barclay&amp;publication_year=1986&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0021">
<span class="label">21.</span><cite>
Goodsell D. S., Autin L., and Olson A. J., “Illustrate: Software for Biomolecular Illustration,” Structure
27, no. 11 (2019): 1716–1720, 10.1016/j.str.2019.08.011.
</cite> [<a href="https://doi.org/10.1016/j.str.2019.08.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6834899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31519398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Illustrate:%20Software%20for%20Biomolecular%20Illustration&amp;author=D.%20S.%20Goodsell&amp;author=L.%20Autin&amp;author=A.%20J.%20Olson&amp;volume=27&amp;issue=11&amp;publication_year=2019&amp;pages=1716-1720&amp;pmid=31519398&amp;doi=10.1016/j.str.2019.08.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0022">
<span class="label">22.</span><cite>
Lomize M. A., Pogozheva I. D., Joo H., Mosberg H. I., and Lomize A. L., “OPM Database and PPM Web Server: Resources for Positioning of Proteins in Membranes,” Nucleic Acids Research
40 (2012): D370–D376, 10.1093/nar/gkr703.
</cite> [<a href="https://doi.org/10.1093/nar/gkr703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3245162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21890895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=OPM%20Database%20and%20PPM%20Web%20Server:%20Resources%20for%20Positioning%20of%20Proteins%20in%20Membranes&amp;author=M.%20A.%20Lomize&amp;author=I.%20D.%20Pogozheva&amp;author=H.%20Joo&amp;author=H.%20I.%20Mosberg&amp;author=A.%20L.%20Lomize&amp;volume=40&amp;publication_year=2012&amp;pages=D370-D376&amp;pmid=21890895&amp;doi=10.1093/nar/gkr703&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0023">
<span class="label">23.</span><cite>
Jenkins E., Körbel M., O'Brien‐Ball C., et al., “Antigen Discrimination by T Cells Relies on Size‐Constrained Microvillar Contact,” Nature Communications
14, no. 1 (2023): 1611, 10.1038/s41467-023-36855-9.</cite> [<a href="https://doi.org/10.1038/s41467-023-36855-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10036606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36959206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Antigen%20Discrimination%20by%20T%20Cells%20Relies%20on%20Size%E2%80%90Constrained%20Microvillar%20Contact&amp;author=E.%20Jenkins&amp;author=M.%20K%C3%B6rbel&amp;author=C.%20O'Brien%E2%80%90Ball&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=1611&amp;pmid=36959206&amp;doi=10.1038/s41467-023-36855-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0024">
<span class="label">24.</span><cite>
Gérard A., Cope A. P., Kemper C., Alon R., and Köchl R., “LFA‐1 in T Cell Priming, Differentiation, and Effector Functions,” Trends in Immunology
42, no. 8 (2021): 706–722.
</cite> [<a href="https://doi.org/10.1016/j.it.2021.06.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10734378/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34266767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Immunology&amp;title=LFA%E2%80%901%20in%20T%20Cell%20Priming,%20Differentiation,%20and%20Effector%20Functions&amp;author=A.%20G%C3%A9rard&amp;author=A.%20P.%20Cope&amp;author=C.%20Kemper&amp;author=R.%20Alon&amp;author=R.%20K%C3%B6chl&amp;volume=42&amp;issue=8&amp;publication_year=2021&amp;pages=706-722&amp;pmid=34266767&amp;doi=10.1016/j.it.2021.06.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0025">
<span class="label">25.</span><cite>
Fernandes R. A., Ganzinger K. A., Tzou J. C., et al., “A Cell Topography‐Based Mechanism for Ligand Discrimination by the T Cell Receptor,” Proceedings of the National Academy of Sciences of the United States of America
116, no. 28 (2019): 14002–14010.
</cite> [<a href="https://doi.org/10.1073/pnas.1817255116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6628812/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31221762/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=A%20Cell%20Topography%E2%80%90Based%20Mechanism%20for%20Ligand%20Discrimination%20by%20the%20T%20Cell%20Receptor&amp;author=R.%20A.%20Fernandes&amp;author=K.%20A.%20Ganzinger&amp;author=J.%20C.%20Tzou&amp;volume=116&amp;issue=28&amp;publication_year=2019&amp;pages=14002-14010&amp;pmid=31221762&amp;doi=10.1073/pnas.1817255116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0026">
<span class="label">26.</span><cite>
Burroughs N. J., Lazic Z., and van der Merwe P. A., “Ligand Detection and Discrimination by Spatial Relocalization: A Kinase‐Phosphatase Segregation Model of TCR Activation,” Biophysical Journal
91, no. 5 (2006): 1619–1629.
</cite> [<a href="https://doi.org/10.1529/biophysj.105.080044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1544308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16751250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophysical%20Journal&amp;title=Ligand%20Detection%20and%20Discrimination%20by%20Spatial%20Relocalization:%20A%20Kinase%E2%80%90Phosphatase%20Segregation%20Model%20of%20TCR%20Activation&amp;author=N.%20J.%20Burroughs&amp;author=Z.%20Lazic&amp;author=P.%20A.%20van%20der%20Merwe&amp;volume=91&amp;issue=5&amp;publication_year=2006&amp;pages=1619-1629&amp;pmid=16751250&amp;doi=10.1529/biophysj.105.080044&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0027">
<span class="label">27.</span><cite>
Jones E. Y., Davis S. J., Williams A. F., Harlos K., and Stuart D. I., “Crystal Structure at 2.8 Å Resolution of a Soluble Form of the Cell Adhesion Molecule CD2,” Nature
360, no. 6401 (1992): 232–239.
</cite> [<a href="https://doi.org/10.1038/360232a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1279440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Crystal%20Structure%20at%202.8%20%C3%85%20Resolution%20of%20a%20Soluble%20Form%20of%20the%20Cell%20Adhesion%20Molecule%20CD2&amp;author=E.%20Y.%20Jones&amp;author=S.%20J.%20Davis&amp;author=A.%20F.%20Williams&amp;author=K.%20Harlos&amp;author=D.%20I.%20Stuart&amp;volume=360&amp;issue=6401&amp;publication_year=1992&amp;pages=232-239&amp;pmid=1279440&amp;doi=10.1038/360232a0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0028">
<span class="label">28.</span><cite>
Bodian D. L., Jones E. Y., Harlos K., Stuart D. I., and Davis S. J., “Crystal Structure of the Extracellular Region of the Human Cell Adhesion Molecule CD2 at 2.5Å Resolution,” Structure
2, no. 8 (1994): 755–766.
</cite> [<a href="https://doi.org/10.1016/s0969-2126(94)00076-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7994575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Crystal%20Structure%20of%20the%20Extracellular%20Region%20of%20the%20Human%20Cell%20Adhesion%20Molecule%20CD2%20at%202.5%C3%85%20Resolution&amp;author=D.%20L.%20Bodian&amp;author=E.%20Y.%20Jones&amp;author=K.%20Harlos&amp;author=D.%20I.%20Stuart&amp;author=S.%20J.%20Davis&amp;volume=2&amp;issue=8&amp;publication_year=1994&amp;pages=755-766&amp;pmid=7994575&amp;doi=10.1016/s0969-2126(94)00076-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0029">
<span class="label">29.</span><cite>
Ikemizu S., Sparks L. M., van der Merwe P. A., et al., “Crystal Structure of the CD2‐Binding Domain of CD58 (Lymphocyte Function‐Associated Antigen 3) at 1.8‐Å Resolution,” Proceedings of the National Academy of Sciences of the United States of America
96, no. 8 (1999): 4289–4294.
</cite> [<a href="https://doi.org/10.1073/pnas.96.8.4289" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC16325/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10200255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Crystal%20Structure%20of%20the%20CD2%E2%80%90Binding%20Domain%20of%20CD58%20(Lymphocyte%20Function%E2%80%90Associated%20Antigen%203)%20at%201.8%E2%80%90%C3%85%20Resolution&amp;author=S.%20Ikemizu&amp;author=L.%20M.%20Sparks&amp;author=P.%20A.%20van%20der%20Merwe&amp;volume=96&amp;issue=8&amp;publication_year=1999&amp;pages=4289-4294&amp;pmid=10200255&amp;doi=10.1073/pnas.96.8.4289&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0030">
<span class="label">30.</span><cite>
Evans E. J., Castro M. A. A., O'Brien R., et al., “Crystal Structure and Binding Properties of the CD2 and CD244 (2B4)‐Binding Protein, CD48,” Journal of Biological Chemistry
281, no. 39 (2006): 29309–29320.
</cite> [<a href="https://doi.org/10.1074/jbc.M601314200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16803907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=Crystal%20Structure%20and%20Binding%20Properties%20of%20the%20CD2%20and%20CD244%20(2B4)%E2%80%90Binding%20Protein,%20CD48&amp;author=E.%20J.%20Evans&amp;author=M.%20A.%20A.%20Castro&amp;author=R.%20O'Brien&amp;volume=281&amp;issue=39&amp;publication_year=2006&amp;pages=29309-29320&amp;pmid=16803907&amp;doi=10.1074/jbc.M601314200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0031">
<span class="label">31.</span><cite>
Chang V. T., Fernandes R. A., Ganzinger K. A., et al., “Initiation of T Cell Signaling by CD45 Segregation at “Close Contacts”,” Nature Immunology
17, no. 5 (2016): 574–582.
</cite> [<a href="https://doi.org/10.1038/ni.3392" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4839504/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26998761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Immunology&amp;title=Initiation%20of%20T%20Cell%20Signaling%20by%20CD45%20Segregation%20at%20%E2%80%9CClose%20Contacts%E2%80%9D&amp;author=V.%20T.%20Chang&amp;author=R.%20A.%20Fernandes&amp;author=K.%20A.%20Ganzinger&amp;volume=17&amp;issue=5&amp;publication_year=2016&amp;pages=574-582&amp;pmid=26998761&amp;doi=10.1038/ni.3392&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0032">
<span class="label">32.</span><cite>
Sušac L., Vuong M. T., Thomas C., et al., “Structure of a Fully Assembled Tumor‐Specific T Cell Receptor Ligated by pMHC,” Cell
185, no. 17 (2022): 3201–3213.
</cite> [<a href="https://doi.org/10.1016/j.cell.2022.07.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9630439/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35985289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Structure%20of%20a%20Fully%20Assembled%20Tumor%E2%80%90Specific%20T%20Cell%20Receptor%20Ligated%20by%20pMHC&amp;author=L.%20Su%C5%A1ac&amp;author=M.%20T.%20Vuong&amp;author=C.%20Thomas&amp;volume=185&amp;issue=17&amp;publication_year=2022&amp;pages=3201-3213&amp;pmid=35985289&amp;doi=10.1016/j.cell.2022.07.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0033">
<span class="label">33.</span><cite>
Gully B. S., Ferreira Fernandes J., Gunasinghe S. D., et al., “Structure of a Fully Assembled γδ T Cell Antigen Receptor,” Nature
634, no. 8034 (2024): 729–736.
</cite> [<a href="https://doi.org/10.1038/s41586-024-07920-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11485255/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39146975/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structure%20of%20a%20Fully%20Assembled%20%CE%B3%CE%B4%20T%20Cell%20Antigen%20Receptor&amp;author=B.%20S.%20Gully&amp;author=J.%20Ferreira%20Fernandes&amp;author=S.%20D.%20Gunasinghe&amp;volume=634&amp;issue=8034&amp;publication_year=2024&amp;pages=729-736&amp;pmid=39146975&amp;doi=10.1038/s41586-024-07920-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0034">
<span class="label">34.</span><cite>
Evans E. J., Esnouf R. M., Manso‐Sancho R., et al., “Crystal Structure of a Soluble CD28‐Fab Complex,” Nature Immunology
6, no. 3 (2005): 271–279, 10.1038/ni1170.
</cite> [<a href="https://doi.org/10.1038/ni1170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15696168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Immunology&amp;title=Crystal%20Structure%20of%20a%20Soluble%20CD28%E2%80%90Fab%20Complex&amp;author=E.%20J.%20Evans&amp;author=R.%20M.%20Esnouf&amp;author=R.%20Manso%E2%80%90Sancho&amp;volume=6&amp;issue=3&amp;publication_year=2005&amp;pages=271-279&amp;pmid=15696168&amp;doi=10.1038/ni1170&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0035">
<span class="label">35.</span><cite>
Stamper C. C., Zhang Y., Tobin J. F., et al., “Crystal Structure of the B7‐1/CTLA‐4 Complex That Inhibits Human Immune Responses,” Nature
410, no. 6828 (2001): 608–611.
</cite> [<a href="https://doi.org/10.1038/35069118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11279502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Crystal%20Structure%20of%20the%20B7%E2%80%901/CTLA%E2%80%904%20Complex%20That%20Inhibits%20Human%20Immune%20Responses&amp;author=C.%20C.%20Stamper&amp;author=Y.%20Zhang&amp;author=J.%20F.%20Tobin&amp;volume=410&amp;issue=6828&amp;publication_year=2001&amp;pages=608-611&amp;pmid=11279502&amp;doi=10.1038/35069118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0036">
<span class="label">36.</span><cite>
Yu C., Sonnen A. F. P., George R., et al., “Rigid‐Body Ligand Recognition Drives Cytotoxic T‐Lymphocyte Antigen 4 (CTLA‐4) Receptor Triggering,” Journal of Biological Chemistry
286, no. 8 (2011): 6685–6696.
</cite> [<a href="https://doi.org/10.1074/jbc.M110.182394" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3057841/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21156796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=Rigid%E2%80%90Body%20Ligand%20Recognition%20Drives%20Cytotoxic%20T%E2%80%90Lymphocyte%20Antigen%204%20(CTLA%E2%80%904)%20Receptor%20Triggering&amp;author=C.%20Yu&amp;author=A.%20F.%20P.%20Sonnen&amp;author=R.%20George&amp;volume=286&amp;issue=8&amp;publication_year=2011&amp;pages=6685-6696&amp;pmid=21156796&amp;doi=10.1074/jbc.M110.182394&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0037">
<span class="label">37.</span><cite>
Ikemizu S., Gilbert R. J., Fennelly J. A., et al., “Structure and Dimerization of a Soluble Form of B7‐1,” Immunity
12, no. 1 (2000): 51–60, 10.1016/S1074-7613(00)80158-2.
</cite> [<a href="https://doi.org/10.1016/S1074-7613(00)80158-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10661405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Structure%20and%20Dimerization%20of%20a%20Soluble%20Form%20of%20B7%E2%80%901&amp;author=S.%20Ikemizu&amp;author=R.%20J.%20Gilbert&amp;author=J.%20A.%20Fennelly&amp;volume=12&amp;issue=1&amp;publication_year=2000&amp;pages=51-60&amp;pmid=10661405&amp;doi=10.1016/S1074-7613(00)80158-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0038">
<span class="label">38.</span><cite>
Cheng X., Veverka V., Radhakrishnan A., et al., “Structure and Interactions of the Human Programmed Cell Death 1 Receptor,” Journal of Biological Chemistry
288, no. 17 (2013): 11771–11785.
</cite> [<a href="https://doi.org/10.1074/jbc.M112.448126" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3636866/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23417675/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=Structure%20and%20Interactions%20of%20the%20Human%20Programmed%20Cell%20Death%201%20Receptor&amp;author=X.%20Cheng&amp;author=V.%20Veverka&amp;author=A.%20Radhakrishnan&amp;volume=288&amp;issue=17&amp;publication_year=2013&amp;pages=11771-11785&amp;pmid=23417675&amp;doi=10.1074/jbc.M112.448126&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0039">
<span class="label">39.</span><cite>
Parodi A. J., “Protein Glucosylation and Its Role in Protein Folding,” Annual Review of Biochemistry
69, no. 1 (2000): 69–93.</cite> [<a href="https://doi.org/10.1146/annurev.biochem.69.1.69" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10966453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Biochemistry&amp;title=Protein%20Glucosylation%20and%20Its%20Role%20in%20Protein%20Folding&amp;author=A.%20J.%20Parodi&amp;volume=69&amp;issue=1&amp;publication_year=2000&amp;pages=69-93&amp;pmid=10966453&amp;doi=10.1146/annurev.biochem.69.1.69&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0040">
<span class="label">40.</span><cite>
Kwong P. D., Wyatt R., Desjardins E., et al., “Probability Analysis of Variational Crystallization and Its Application to gp120, the Exterior Envelope Glycoprotein of Type 1 Human Immunodeficiency Virus (HIV‐1),” Journal of Biological Chemistry
274, no. 7 (1999): 4115–4123.
</cite> [<a href="https://doi.org/10.1074/jbc.274.7.4115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9933605/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=Probability%20Analysis%20of%20Variational%20Crystallization%20and%20Its%20Application%20to%20gp120,%20the%20Exterior%20Envelope%20Glycoprotein%20of%20Type%201%20Human%20Immunodeficiency%20Virus%20(HIV%E2%80%901)&amp;author=P.%20D.%20Kwong&amp;author=R.%20Wyatt&amp;author=E.%20Desjardins&amp;volume=274&amp;issue=7&amp;publication_year=1999&amp;pages=4115-4123&amp;pmid=9933605&amp;doi=10.1074/jbc.274.7.4115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0041">
<span class="label">41.</span><cite>
Prestegard J. H., “A Perspective on the PDB's Impact on the Field of Glycobiology,” Journal of Biological Chemistry
296 (2021): 100556.
</cite> [<a href="https://doi.org/10.1016/j.jbc.2021.100556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8058564/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33744289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=A%20Perspective%20on%20the%20PDB's%20Impact%20on%20the%20Field%20of%20Glycobiology&amp;author=J.%20H.%20Prestegard&amp;volume=296&amp;publication_year=2021&amp;pages=100556&amp;pmid=33744289&amp;doi=10.1016/j.jbc.2021.100556&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0042">
<span class="label">42.</span><cite>
Davis S. J., Puklavec M. J., Ashford D. A., et al., “Expression of Soluble Recombinant Glycoproteins With Predefined Glycosylation: Application to the Crystallization of the T‐Cell Glycoprotein CD2,” Protein Engineering, Design &amp; Selection
6, no. 2 (1993): 229–232.</cite> [<a href="https://doi.org/10.1093/protein/6.2.229" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8097313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Engineering,%20Design%20&amp;%20Selection&amp;title=Expression%20of%20Soluble%20Recombinant%20Glycoproteins%20With%20Predefined%20Glycosylation:%20Application%20to%20the%20Crystallization%20of%20the%20T%E2%80%90Cell%20Glycoprotein%20CD2&amp;author=S.%20J.%20Davis&amp;author=M.%20J.%20Puklavec&amp;author=D.%20A.%20Ashford&amp;volume=6&amp;issue=2&amp;publication_year=1993&amp;pages=229-232&amp;pmid=8097313&amp;doi=10.1093/protein/6.2.229&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0043">
<span class="label">43.</span><cite>
Davis S. J., Davies E. A., Barclay A. N., et al., “Ligand Binding by the Immunoglobulin Superfamily Recognition Molecule CD2 Is Glycosylation‐Independent,” Journal of Biological Chemistry
270, no. 1 (1995): 369–375.
</cite> [<a href="https://doi.org/10.1074/jbc.270.1.369" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7529232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=Ligand%20Binding%20by%20the%20Immunoglobulin%20Superfamily%20Recognition%20Molecule%20CD2%20Is%20Glycosylation%E2%80%90Independent&amp;author=S.%20J.%20Davis&amp;author=E.%20A.%20Davies&amp;author=A.%20N.%20Barclay&amp;volume=270&amp;issue=1&amp;publication_year=1995&amp;pages=369-375&amp;pmid=7529232&amp;doi=10.1074/jbc.270.1.369&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0044">
<span class="label">44.</span><cite>
Chang V. T., Crispin M., Aricescu A. R., et al., “Glycoprotein Structural Genomics: Solving the Glycosylation Problem,” Structure
15, no. 3 (2007): 267–273.
</cite> [<a href="https://doi.org/10.1016/j.str.2007.01.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1885966/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17355862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Glycoprotein%20Structural%20Genomics:%20Solving%20the%20Glycosylation%20Problem&amp;author=V.%20T.%20Chang&amp;author=M.%20Crispin&amp;author=A.%20R.%20Aricescu&amp;volume=15&amp;issue=3&amp;publication_year=2007&amp;pages=267-273&amp;pmid=17355862&amp;doi=10.1016/j.str.2007.01.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0045">
<span class="label">45.</span><cite>
van der Merwe P. A. and Davis S. J., “Molecular Interactions Mediating T Cell Antigen Recognition,” Annual Review of Immunology
21, no. 1 (2003): 659–684.</cite> [<a href="https://doi.org/10.1146/annurev.immunol.21.120601.141036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12615890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Immunology&amp;title=Molecular%20Interactions%20Mediating%20T%20Cell%20Antigen%20Recognition&amp;author=P.%20A.%20van%20der%20Merwe&amp;author=S.%20J.%20Davis&amp;volume=21&amp;issue=1&amp;publication_year=2003&amp;pages=659-684&amp;pmid=12615890&amp;doi=10.1146/annurev.immunol.21.120601.141036&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0046">
<span class="label">46.</span><cite>
Davis S. J., Ikemizu S., Evans E. J., Fugger L., Bakker T. R., and van der Merwe P. A., “The Nature of Molecular Recognition by T Cells,” Nature Immunology
4, no. 3 (2003): 217–224.
</cite> [<a href="https://doi.org/10.1038/ni0303-217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12605231/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Immunology&amp;title=The%20Nature%20of%20Molecular%20Recognition%20by%20T%20Cells&amp;author=S.%20J.%20Davis&amp;author=S.%20Ikemizu&amp;author=E.%20J.%20Evans&amp;author=L.%20Fugger&amp;author=T.%20R.%20Bakker&amp;volume=4&amp;issue=3&amp;publication_year=2003&amp;pages=217-224&amp;pmid=12605231&amp;doi=10.1038/ni0303-217&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0047">
<span class="label">47.</span><cite>
McNerney M. E. and Kumar V., “The CD2 Family of Natural Killer Cell Receptors,” Current Topics in Microbiology and Immunology
298 (2006): 91–120.
</cite> [<a href="https://doi.org/10.1007/3-540-27743-9_5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16323413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Topics%20in%20Microbiology%20and%20Immunology&amp;title=The%20CD2%20Family%20of%20Natural%20Killer%20Cell%20Receptors&amp;author=M.%20E.%20McNerney&amp;author=V.%20Kumar&amp;volume=298&amp;publication_year=2006&amp;pages=91-120&amp;pmid=16323413&amp;doi=10.1007/3-540-27743-9_5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0048">
<span class="label">48.</span><cite>
Hünig T., “The Cell Surface Molecule Recognized by the Erythrocyte Receptor of T Lymphocytes. Identification and Partial Characterization Using a Monoclonal Antibody,” Journal of Experimental Medicine
162, no. 3 (1985): 890–901.
</cite> [<a href="https://doi.org/10.1084/jem.162.3.890" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2187812/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2411842/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Medicine&amp;title=The%20Cell%20Surface%20Molecule%20Recognized%20by%20the%20Erythrocyte%20Receptor%20of%20T%20Lymphocytes.%20Identification%20and%20Partial%20Characterization%20Using%20a%20Monoclonal%20Antibody&amp;author=T.%20H%C3%BCnig&amp;volume=162&amp;issue=3&amp;publication_year=1985&amp;pages=890-901&amp;pmid=2411842&amp;doi=10.1084/jem.162.3.890&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0049">
<span class="label">49.</span><cite>
Selvaraj P., Plunkett M. L., Dustin M., Sanders M. E., Shaw S., and Springer T. A., “The T Lymphocyte Glycoprotein CD2 Binds the Cell Surface Ligand LFA‐3,” Nature
326, no. 6111 (1987): 400–403.
</cite> [<a href="https://doi.org/10.1038/326400a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2951597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20T%20Lymphocyte%20Glycoprotein%20CD2%20Binds%20the%20Cell%20Surface%20Ligand%20LFA%E2%80%903&amp;author=P.%20Selvaraj&amp;author=M.%20L.%20Plunkett&amp;author=M.%20Dustin&amp;author=M.%20E.%20Sanders&amp;author=S.%20Shaw&amp;volume=326&amp;issue=6111&amp;publication_year=1987&amp;pages=400-403&amp;pmid=2951597&amp;doi=10.1038/326400a0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0050">
<span class="label">50.</span><cite>
van der Merwe P. A., Brown M. H., Davis S. J., and Barclay A. N., “Affinity and Kinetic Analysis of the Interaction of the Cell Adhesion Molecules Rat CD2 and CD48,” EMBO Journal
12, no. 13 (1993): 4945–4954.
</cite> [<a href="https://doi.org/10.1002/j.1460-2075.1993.tb06188.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC413755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7903240/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Journal&amp;title=Affinity%20and%20Kinetic%20Analysis%20of%20the%20Interaction%20of%20the%20Cell%20Adhesion%20Molecules%20Rat%20CD2%20and%20CD48&amp;author=P.%20A.%20van%20der%20Merwe&amp;author=M.%20H.%20Brown&amp;author=S.%20J.%20Davis&amp;author=A.%20N.%20Barclay&amp;volume=12&amp;issue=13&amp;publication_year=1993&amp;pages=4945-4954&amp;pmid=7903240&amp;doi=10.1002/j.1460-2075.1993.tb06188.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0051">
<span class="label">51.</span><cite>
van der Merwe P. A. and Barclay A. N., “Transient Intercellular Adhesion: The Importance of Weak Protein‐Protein Interactions,” Trends in Biochemical Sciences
19, no. 9 (1994): 354–358.
</cite> [<a href="https://doi.org/10.1016/0968-0004(94)90109-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7985226/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Biochemical%20Sciences&amp;title=Transient%20Intercellular%20Adhesion:%20The%20Importance%20of%20Weak%20Protein%E2%80%90Protein%20Interactions&amp;author=P.%20A.%20van%20der%20Merwe&amp;author=A.%20N.%20Barclay&amp;volume=19&amp;issue=9&amp;publication_year=1994&amp;pages=354-358&amp;pmid=7985226&amp;doi=10.1016/0968-0004(94)90109-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0052">
<span class="label">52.</span><cite>
Zhang Y., Liu Q., Yang S., and Liao Q., “CD58 Immunobiology at a Glance,” Frontiers in Immunology
12 (2021): 705260.
</cite> [<a href="https://doi.org/10.3389/fimmu.2021.705260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8218816/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34168659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Immunology&amp;title=CD58%20Immunobiology%20at%20a%20Glance&amp;author=Y.%20Zhang&amp;author=Q.%20Liu&amp;author=S.%20Yang&amp;author=Q.%20Liao&amp;volume=12&amp;publication_year=2021&amp;pages=705260&amp;pmid=34168659&amp;doi=10.3389/fimmu.2021.705260&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0053">
<span class="label">53.</span><cite>
Killeen N., Stuart S. G., and Littman D. R., “Development and Function of T Cells in Mice With a Disrupted CD2 Gene,” EMBO Journal
11, no. 12 (1992): 4329–4336.
</cite> [<a href="https://doi.org/10.1002/j.1460-2075.1992.tb05532.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC557006/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1358605/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Journal&amp;title=Development%20and%20Function%20of%20T%20Cells%20in%20Mice%20With%20a%20Disrupted%20CD2%20Gene&amp;author=N.%20Killeen&amp;author=S.%20G.%20Stuart&amp;author=D.%20R.%20Littman&amp;volume=11&amp;issue=12&amp;publication_year=1992&amp;pages=4329-4336&amp;pmid=1358605&amp;doi=10.1002/j.1460-2075.1992.tb05532.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0054">
<span class="label">54.</span><cite>
Bachmann M. F., Barner M., and Kopf M., “CD2 Sets Quantitative Thresholds in T Cell Activation,” Journal of Experimental Medicine
190, no. 10 (1999): 1383–1392.
</cite> [<a href="https://doi.org/10.1084/jem.190.10.1383" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2195700/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10562314/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Medicine&amp;title=CD2%20Sets%20Quantitative%20Thresholds%20in%20T%20Cell%20Activation&amp;author=M.%20F.%20Bachmann&amp;author=M.%20Barner&amp;author=M.%20Kopf&amp;volume=190&amp;issue=10&amp;publication_year=1999&amp;pages=1383-1392&amp;pmid=10562314&amp;doi=10.1084/jem.190.10.1383&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0055">
<span class="label">55.</span><cite>
Kato K., Koyanagi M., Okada H., et al., “CD48 Is a Counter‐Receptor for Mouse CD2 and Is Involved in T Cell Activation,” Journal of Experimental Medicine
176, no. 5 (1992): 1241–1249.
</cite> [<a href="https://doi.org/10.1084/jem.176.5.1241" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2119417/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1383383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Medicine&amp;title=CD48%20Is%20a%20Counter%E2%80%90Receptor%20for%20Mouse%20CD2%20and%20Is%20Involved%20in%20T%20Cell%20Activation&amp;author=K.%20Kato&amp;author=M.%20Koyanagi&amp;author=H.%20Okada&amp;volume=176&amp;issue=5&amp;publication_year=1992&amp;pages=1241-1249&amp;pmid=1383383&amp;doi=10.1084/jem.176.5.1241&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0056">
<span class="label">56.</span><cite>
Brown M. H., Boles K., van der Merwe P. A., Kumar V., Mathew P. A., and Barclay A. N., “2B4, the Natural Killer and T Cell Immunoglobulin Superfamily Surface Protein, Is a Ligand for CD48,” Journal of Experimental Medicine
188, no. 11 (1998): 2083–2090.
</cite> [<a href="https://doi.org/10.1084/jem.188.11.2083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2212392/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9841922/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Medicine&amp;title=2B4,%20the%20Natural%20Killer%20and%20T%20Cell%20Immunoglobulin%20Superfamily%20Surface%20Protein,%20Is%20a%20Ligand%20for%20CD48&amp;author=M.%20H.%20Brown&amp;author=K.%20Boles&amp;author=P.%20A.%20van%20der%20Merwe&amp;author=V.%20Kumar&amp;author=P.%20A.%20Mathew&amp;volume=188&amp;issue=11&amp;publication_year=1998&amp;pages=2083-2090&amp;pmid=9841922&amp;doi=10.1084/jem.188.11.2083&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0057">
<span class="label">57.</span><cite>
Garni‐Wagner B. A., Purohit A., Mathew P. A., Bennett M., and Kumar V., “A Novel Function‐Associated Molecule Related to Non‐MHC‐Restricted Cytotoxicity Mediated by Activated Natural Killer Cells and T Cells,” Journal of Immunology
151, no. 1 (1993): 60–70.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/8326140/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Immunology&amp;title=A%20Novel%20Function%E2%80%90Associated%20Molecule%20Related%20to%20Non%E2%80%90MHC%E2%80%90Restricted%20Cytotoxicity%20Mediated%20by%20Activated%20Natural%20Killer%20Cells%20and%20T%20Cells&amp;author=B.%20A.%20Garni%E2%80%90Wagner&amp;author=A.%20Purohit&amp;author=P.%20A.%20Mathew&amp;author=M.%20Bennett&amp;author=V.%20Kumar&amp;volume=151&amp;issue=1&amp;publication_year=1993&amp;pages=60-70&amp;pmid=8326140&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0058">
<span class="label">58.</span><cite>
Williams A. F., Barclay A. N., Clark S. J., Paterson D. J., and Willis A. C., “Similarities in Sequences and Cellular Expression Between Rat CD2 and CD4 Antigens,” Journal of Experimental Medicine
165, no. 2 (1987): 368–380.
</cite> [<a href="https://doi.org/10.1084/jem.165.2.368" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2188524/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3102667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Medicine&amp;title=Similarities%20in%20Sequences%20and%20Cellular%20Expression%20Between%20Rat%20CD2%20and%20CD4%20Antigens&amp;author=A.%20F.%20Williams&amp;author=A.%20N.%20Barclay&amp;author=S.%20J.%20Clark&amp;author=D.%20J.%20Paterson&amp;author=A.%20C.%20Willis&amp;volume=165&amp;issue=2&amp;publication_year=1987&amp;pages=368-380&amp;pmid=3102667&amp;doi=10.1084/jem.165.2.368&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0059">
<span class="label">59.</span><cite>
Tavernor A. S., Kydd J. H., Bodian D. L., et al., “Expression Cloning of an Equine T‐Lymphocyte Glycoprotein CD2 cDNA. Structure‐Based Analysis of Conserved Sequence Elements: Structure‐Based Analysis of Conserved Sequence Elements,” European Journal of Biochemistry
219, no. 3 (1994): 969–976.
</cite> [<a href="https://doi.org/10.1111/j.1432-1033.1994.tb18579.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7906650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Biochemistry&amp;title=Expression%20Cloning%20of%20an%20Equine%20T%E2%80%90Lymphocyte%20Glycoprotein%20CD2%20cDNA.%20Structure%E2%80%90Based%20Analysis%20of%20Conserved%20Sequence%20Elements:%20Structure%E2%80%90Based%20Analysis%20of%20Conserved%20Sequence%20Elements&amp;author=A.%20S.%20Tavernor&amp;author=J.%20H.%20Kydd&amp;author=D.%20L.%20Bodian&amp;volume=219&amp;issue=3&amp;publication_year=1994&amp;pages=969-976&amp;pmid=7906650&amp;doi=10.1111/j.1432-1033.1994.tb18579.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0060">
<span class="label">60.</span><cite>
Wang J. H., Smolyar A., Tan K., et al., “Structure of a Heterophilic Adhesion Complex Between the Human CD2 and CD58 (LFA‐3) Counterreceptors,” Cell
97, no. 6 (1999): 791–803.
</cite> [<a href="https://doi.org/10.1016/s0092-8674(00)80790-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10380930/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Structure%20of%20a%20Heterophilic%20Adhesion%20Complex%20Between%20the%20Human%20CD2%20and%20CD58%20(LFA%E2%80%903)%20Counterreceptors&amp;author=J.%20H.%20Wang&amp;author=A.%20Smolyar&amp;author=K.%20Tan&amp;volume=97&amp;issue=6&amp;publication_year=1999&amp;pages=791-803&amp;pmid=10380930&amp;doi=10.1016/s0092-8674(00)80790-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0061">
<span class="label">61.</span><cite>
Tsai Y. X., Chang N. E., Reuter K., et al., “Rapid Simulation of Glycoprotein Structures by Grafting and Steric Exclusion of Glycan Conformer Libraries,” Cell
187, no. 5 (2024): 1296–1311.e26.
</cite> [<a href="https://doi.org/10.1016/j.cell.2024.01.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38428397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Rapid%20Simulation%20of%20Glycoprotein%20Structures%20by%20Grafting%20and%20Steric%20Exclusion%20of%20Glycan%20Conformer%20Libraries&amp;author=Y.%20X.%20Tsai&amp;author=N.%20E.%20Chang&amp;author=K.%20Reuter&amp;volume=187&amp;issue=5&amp;publication_year=2024&amp;pages=1296-1311.e26&amp;pmid=38428397&amp;doi=10.1016/j.cell.2024.01.034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0062">
<span class="label">62.</span><cite>
Peterson A. and Seed B., “Monoclonal Antibody and Ligand Binding Sites of the T Cell Erythrocyte Receptor (CD2),” Nature
329, no. 6142 (1987): 842–846.
</cite> [<a href="https://doi.org/10.1038/329842a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2444890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Monoclonal%20Antibody%20and%20Ligand%20Binding%20Sites%20of%20the%20T%20Cell%20Erythrocyte%20Receptor%20(CD2)&amp;author=A.%20Peterson&amp;author=B.%20Seed&amp;volume=329&amp;issue=6142&amp;publication_year=1987&amp;pages=842-846&amp;pmid=2444890&amp;doi=10.1038/329842a0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0063">
<span class="label">63.</span><cite>
Somoza C., Driscoll P. C., Cyster J. G., and Williams A. F., “Mutational Analysis of the CD2/CD58 Interaction: The Binding Site for CD58 Lies on One Face of the First Domain of Human CD2,” Journal of Experimental Medicine
178, no. 2 (1993): 549–558.
</cite> [<a href="https://doi.org/10.1084/jem.178.2.549" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2191138/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7688025/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Medicine&amp;title=Mutational%20Analysis%20of%20the%20CD2/CD58%20Interaction:%20The%20Binding%20Site%20for%20CD58%20Lies%20on%20One%20Face%20of%20the%20First%20Domain%20of%20Human%20CD2&amp;author=C.%20Somoza&amp;author=P.%20C.%20Driscoll&amp;author=J.%20G.%20Cyster&amp;author=A.%20F.%20Williams&amp;volume=178&amp;issue=2&amp;publication_year=1993&amp;pages=549-558&amp;pmid=7688025&amp;doi=10.1084/jem.178.2.549&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0064">
<span class="label">64.</span><cite>
Arulanandam A. R., Withka J. M., Wyss D. F., et al., “The CD58 (LFA‐3) Binding Site Is a Localized and Highly Charged Surface Area on the AGFCC'C Face of the Human CD2 Adhesion Domain,” Proceedings of the National Academy of Sciences of the United States of America
90, no. 24 (1993): 11613–11617.
</cite> [<a href="https://doi.org/10.1073/pnas.90.24.11613" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC48034/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7505442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=The%20CD58%20(LFA%E2%80%903)%20Binding%20Site%20Is%20a%20Localized%20and%20Highly%20Charged%20Surface%20Area%20on%20the%20AGFCC'C%20Face%20of%20the%20Human%20CD2%20Adhesion%20Domain&amp;author=A.%20R.%20Arulanandam&amp;author=J.%20M.%20Withka&amp;author=D.%20F.%20Wyss&amp;volume=90&amp;issue=24&amp;publication_year=1993&amp;pages=11613-11617&amp;pmid=7505442&amp;doi=10.1073/pnas.90.24.11613&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0065">
<span class="label">65.</span><cite>
Davis S. J., Davies E. A., Tucknott M. G., Jones E. Y., and van der Merwe P. A., “The Role of Charged Residues Mediating Low Affinity Protein‐Protein Recognition at the Cell Surface by CD2,” Proceedings of the National Academy of Sciences of the United States of America
95, no. 10 (1998): 5490–5494.
</cite> [<a href="https://doi.org/10.1073/pnas.95.10.5490" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC20404/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9576909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=The%20Role%20of%20Charged%20Residues%20Mediating%20Low%20Affinity%20Protein%E2%80%90Protein%20Recognition%20at%20the%20Cell%20Surface%20by%20CD2&amp;author=S.%20J.%20Davis&amp;author=E.%20A.%20Davies&amp;author=M.%20G.%20Tucknott&amp;author=E.%20Y.%20Jones&amp;author=P.%20A.%20van%20der%20Merwe&amp;volume=95&amp;issue=10&amp;publication_year=1998&amp;pages=5490-5494&amp;pmid=9576909&amp;doi=10.1073/pnas.95.10.5490&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0066">
<span class="label">66.</span><cite>
Honig B. and Nicholls A., “Classical Electrostatics in Biology and Chemistry,” Science
268, no. 5214 (1995): 1144–1149.
</cite> [<a href="https://doi.org/10.1126/science.7761829" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7761829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Classical%20Electrostatics%20in%20Biology%20and%20Chemistry&amp;author=B.%20Honig&amp;author=A.%20Nicholls&amp;volume=268&amp;issue=5214&amp;publication_year=1995&amp;pages=1144-1149&amp;pmid=7761829&amp;doi=10.1126/science.7761829&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0067">
<span class="label">67.</span><cite>
Janin J. and Chothia C., “The Structure of Protein‐Protein Recognition Sites,” Journal of Biological Chemistry
265, no. 27 (1990): 16027–16030.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/2204619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=The%20Structure%20of%20Protein%E2%80%90Protein%20Recognition%20Sites&amp;author=J.%20Janin&amp;author=C.%20Chothia&amp;volume=265&amp;issue=27&amp;publication_year=1990&amp;pages=16027-16030&amp;pmid=2204619&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0068">
<span class="label">68.</span><cite>
Arulanandam A. R., Kister A., McGregor M. J., Wyss D. F., Wagner G., and Reinherz E. L., “Interaction Between Human CD2 and CD58 Involves the Major Beta Sheet Surface of Each of Their Respective Adhesion Domains,” Journal of Experimental Medicine
180, no. 5 (1994): 1861–1871.
</cite> [<a href="https://doi.org/10.1084/jem.180.5.1861" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2191747/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7525842/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Medicine&amp;title=Interaction%20Between%20Human%20CD2%20and%20CD58%20Involves%20the%20Major%20Beta%20Sheet%20Surface%20of%20Each%20of%20Their%20Respective%20Adhesion%20Domains&amp;author=A.%20R.%20Arulanandam&amp;author=A.%20Kister&amp;author=M.%20J.%20McGregor&amp;author=D.%20F.%20Wyss&amp;author=G.%20Wagner&amp;volume=180&amp;issue=5&amp;publication_year=1994&amp;pages=1861-1871&amp;pmid=7525842&amp;doi=10.1084/jem.180.5.1861&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0069">
<span class="label">69.</span><cite>
Osborn L., Day E. S., Miller G. T., et al., “Amino Acid Residues Required for Binding of Lymphocyte Function‐Associated Antigen 3 (CD58) to Its Counter‐Receptor CD2,” Journal of Experimental Medicine
181, no. 1 (1995): 429–434.
</cite> [<a href="https://doi.org/10.1084/jem.181.1.429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2191834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7528777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Medicine&amp;title=Amino%20Acid%20Residues%20Required%20for%20Binding%20of%20Lymphocyte%20Function%E2%80%90Associated%20Antigen%203%20(CD58)%20to%20Its%20Counter%E2%80%90Receptor%20CD2&amp;author=L.%20Osborn&amp;author=E.%20S.%20Day&amp;author=G.%20T.%20Miller&amp;volume=181&amp;issue=1&amp;publication_year=1995&amp;pages=429-434&amp;pmid=7528777&amp;doi=10.1084/jem.181.1.429&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0070">
<span class="label">70.</span><cite>
Fischer E. H., Charbonneau H., Cool D. E., and Tonks N. K., “Tyrosine phosphatases and their possible interplay with tyrosine kinases,” Ciba Found Symp
164 (1992): 132–140, 10.1002/9780470514207.
</cite> [<a href="https://doi.org/10.1002/9780470514207" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1395930/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ciba%20Found%20Symp&amp;title=Tyrosine%20phosphatases%20and%20their%20possible%20interplay%20with%20tyrosine%20kinases&amp;author=E.%20H.%20Fischer&amp;author=H.%20Charbonneau&amp;author=D.%20E.%20Cool&amp;author=N.%20K.%20Tonks&amp;volume=164&amp;publication_year=1992&amp;pages=132-140&amp;pmid=1395930&amp;doi=10.1002/9780470514207&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0071">
<span class="label">71.</span><cite>
Pingel J. T. and Thomas M. L., “Evidence That the Leukocyte‐Common Antigen Is Required for Antigen‐Induced T Lymphocyte Proliferation,” Cell
58, no. 6 (1989): 1055–1065.
</cite> [<a href="https://doi.org/10.1016/0092-8674(89)90504-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7127598/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2550143/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Evidence%20That%20the%20Leukocyte%E2%80%90Common%20Antigen%20Is%20Required%20for%20Antigen%E2%80%90Induced%20T%20Lymphocyte%20Proliferation&amp;author=J.%20T.%20Pingel&amp;author=M.%20L.%20Thomas&amp;volume=58&amp;issue=6&amp;publication_year=1989&amp;pages=1055-1065&amp;pmid=2550143&amp;doi=10.1016/0092-8674(89)90504-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0072">
<span class="label">72.</span><cite>
Desai D. M., Sap J., Silvennoinen O., Schlessinger J., and Weiss A., “The Catalytic Activity of the CD45 Membrane‐Proximal Phosphatase Domain Is Required for TCR Signaling and Regulation,” EMBO Journal
13, no. 17 (1994): 4002–4010.
</cite> [<a href="https://doi.org/10.1002/j.1460-2075.1994.tb06716.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC395320/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8076596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Journal&amp;title=The%20Catalytic%20Activity%20of%20the%20CD45%20Membrane%E2%80%90Proximal%20Phosphatase%20Domain%20Is%20Required%20for%20TCR%20Signaling%20and%20Regulation&amp;author=D.%20M.%20Desai&amp;author=J.%20Sap&amp;author=O.%20Silvennoinen&amp;author=J.%20Schlessinger&amp;author=A.%20Weiss&amp;volume=13&amp;issue=17&amp;publication_year=1994&amp;pages=4002-4010&amp;pmid=8076596&amp;doi=10.1002/j.1460-2075.1994.tb06716.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0073">
<span class="label">73.</span><cite>
Hermiston M. L., Xu Z., and Weiss A., “CD45: A Critical Regulator of Signaling Thresholds in Immune Cells,” Annual Review of Immunology
21, no. 1 (2003): 107–137.</cite> [<a href="https://doi.org/10.1146/annurev.immunol.21.120601.140946" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12414720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Immunology&amp;title=CD45:%20A%20Critical%20Regulator%20of%20Signaling%20Thresholds%20in%20Immune%20Cells&amp;author=M.%20L.%20Hermiston&amp;author=Z.%20Xu&amp;author=A.%20Weiss&amp;volume=21&amp;issue=1&amp;publication_year=2003&amp;pages=107-137&amp;pmid=12414720&amp;doi=10.1146/annurev.immunol.21.120601.140946&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0074">
<span class="label">74.</span><cite>
Aricescu A. R., Siebold C., Choudhuri K., et al., “Structure of a Tyrosine Phosphatase Adhesive Interaction Reveals a Spacer‐Clamp Mechanism,” Science
317, no. 5842 (2007): 1217–1220.
</cite> [<a href="https://doi.org/10.1126/science.1144646" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17761881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Structure%20of%20a%20Tyrosine%20Phosphatase%20Adhesive%20Interaction%20Reveals%20a%20Spacer%E2%80%90Clamp%20Mechanism&amp;author=A.%20R.%20Aricescu&amp;author=C.%20Siebold&amp;author=K.%20Choudhuri&amp;volume=317&amp;issue=5842&amp;publication_year=2007&amp;pages=1217-1220&amp;pmid=17761881&amp;doi=10.1126/science.1144646&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0075">
<span class="label">75.</span><cite>
Coles C. H., Mitakidis N., Zhang P., et al., “Structural Basis for Extracellular cis and Trans RPTPσ Signal Competition in Synaptogenesis,” Nature Communications
5, no. 1 (2014): 5209.</cite> [<a href="https://doi.org/10.1038/ncomms6209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4239663/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25385546/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Structural%20Basis%20for%20Extracellular%20cis%20and%20Trans%20RPTP%CF%83%20Signal%20Competition%20in%20Synaptogenesis&amp;author=C.%20H.%20Coles&amp;author=N.%20Mitakidis&amp;author=P.%20Zhang&amp;volume=5&amp;issue=1&amp;publication_year=2014&amp;pages=5209&amp;pmid=25385546&amp;doi=10.1038/ncomms6209&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0076">
<span class="label">76.</span><cite>
James J. R., McColl J., Oliveira M. I., et al., “The T Cell Receptor Triggering Apparatus Is Composed of Monovalent or Monomeric Proteins,” Journal of Biological Chemistry
286, no. 37 (2011): 31993–32001.
</cite> [<a href="https://doi.org/10.1074/jbc.M111.219212" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3173209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21757710/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=The%20T%20Cell%20Receptor%20Triggering%20Apparatus%20Is%20Composed%20of%20Monovalent%20or%20Monomeric%20Proteins&amp;author=J.%20R.%20James&amp;author=J.%20McColl&amp;author=M.%20I.%20Oliveira&amp;volume=286&amp;issue=37&amp;publication_year=2011&amp;pages=31993-32001&amp;pmid=21757710&amp;doi=10.1074/jbc.M111.219212&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0077">
<span class="label">77.</span><cite>
Nogi T., Yasui N., Mihara E., et al., “Structural Basis for Semaphorin Signalling Through the Plexin Receptor,” Nature
467, no. 7319 (2010): 1123–1127.
</cite> [<a href="https://doi.org/10.1038/nature09473" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20881961/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structural%20Basis%20for%20Semaphorin%20Signalling%20Through%20the%20Plexin%20Receptor&amp;author=T.%20Nogi&amp;author=N.%20Yasui&amp;author=E.%20Mihara&amp;volume=467&amp;issue=7319&amp;publication_year=2010&amp;pages=1123-1127&amp;pmid=20881961&amp;doi=10.1038/nature09473&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0078">
<span class="label">78.</span><cite>
Jentoft N., “Why Are Proteins O‐Glycosylated?,” Trends in Biochemical Sciences
15, no. 8 (1990): 291–294.
</cite> [<a href="https://doi.org/10.1016/0968-0004(90)90014-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2204153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Biochemical%20Sciences&amp;title=Why%20Are%20Proteins%20O%E2%80%90Glycosylated?&amp;author=N.%20Jentoft&amp;volume=15&amp;issue=8&amp;publication_year=1990&amp;pages=291-294&amp;pmid=2204153&amp;doi=10.1016/0968-0004(90)90014-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0079">
<span class="label">79.</span><cite>
Morath A. and Schamel W. W., “αβ and γδ T Cell Receptors: Similar but Different,” Journal of Leukocyte Biology
107, no. 6 (2020): 1045–1055.
</cite> [<a href="https://doi.org/10.1002/JLB.2MR1219-233R" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31994778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Leukocyte%20Biology&amp;title=%CE%B1%CE%B2%20and%20%CE%B3%CE%B4%20T%20Cell%20Receptors:%20Similar%20but%20Different&amp;author=A.%20Morath&amp;author=W.%20W.%20Schamel&amp;volume=107&amp;issue=6&amp;publication_year=2020&amp;pages=1045-1055&amp;pmid=31994778&amp;doi=10.1002/JLB.2MR1219-233R&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0080">
<span class="label">80.</span><cite>
Garcia K. C., Degano M., Stanfield R. L., et al., “An Alphabeta T Cell Receptor Structure at 2.5 A and Its Orientation in the TCR‐MHC Complex,” Science
274, no. 5285 (1996): 209–219, 10.1126/science.274.5285.209.
</cite> [<a href="https://doi.org/10.1126/science.274.5285.209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8824178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=An%20Alphabeta%20T%20Cell%20Receptor%20Structure%20at%202.5%20A%20and%20Its%20Orientation%20in%20the%20TCR%E2%80%90MHC%20Complex&amp;author=K.%20C.%20Garcia&amp;author=M.%20Degano&amp;author=R.%20L.%20Stanfield&amp;volume=274&amp;issue=5285&amp;publication_year=1996&amp;pages=209-219&amp;pmid=8824178&amp;doi=10.1126/science.274.5285.209&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0081">
<span class="label">81.</span><cite>
Garboczi D. N., Ghosh P., Utz U., Fan Q. R., Biddison W. E., and Wiley D. C., “Structure of the Complex Between Human T‐Cell Receptor, Viral Peptide and HLA‐A2,” Nature
384, no. 6605 (1996): 134–141.
</cite> [<a href="https://doi.org/10.1038/384134a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8906788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structure%20of%20the%20Complex%20Between%20Human%20T%E2%80%90Cell%20Receptor,%20Viral%20Peptide%20and%20HLA%E2%80%90A2&amp;author=D.%20N.%20Garboczi&amp;author=P.%20Ghosh&amp;author=U.%20Utz&amp;author=Q.%20R.%20Fan&amp;author=W.%20E.%20Biddison&amp;volume=384&amp;issue=6605&amp;publication_year=1996&amp;pages=134-141&amp;pmid=8906788&amp;doi=10.1038/384134a0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0082">
<span class="label">82.</span><cite>
Sun Z. J., Kim K. S., Wagner G., and Reinherz E. L., “Mechanisms Contributing to T Cell Receptor Signaling and Assembly Revealed by the Solution Structure of an Ectodomain Fragment of the CD3 Epsilon Gamma Heterodimer,” Cell
105, no. 7 (2001): 913–923.
</cite> [<a href="https://doi.org/10.1016/s0092-8674(01)00395-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11439187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Mechanisms%20Contributing%20to%20T%20Cell%20Receptor%20Signaling%20and%20Assembly%20Revealed%20by%20the%20Solution%20Structure%20of%20an%20Ectodomain%20Fragment%20of%20the%20CD3%20Epsilon%20Gamma%20Heterodimer&amp;author=Z.%20J.%20Sun&amp;author=K.%20S.%20Kim&amp;author=G.%20Wagner&amp;author=E.%20L.%20Reinherz&amp;volume=105&amp;issue=7&amp;publication_year=2001&amp;pages=913-923&amp;pmid=11439187&amp;doi=10.1016/s0092-8674(01)00395-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0083">
<span class="label">83.</span><cite>
Arnett K. L., Harrison S. C., and Wiley D. C., “Crystal Structure of a Human CD3‐Epsilon/delta Dimer in Complex With a UCHT1 Single‐Chain Antibody Fragment,” Proceedings of the National Academy of Sciences of the United States of America
101, no. 46 (2004): 16268–16273.
</cite> [<a href="https://doi.org/10.1073/pnas.0407359101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC528977/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15534202/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Crystal%20Structure%20of%20a%20Human%20CD3%E2%80%90Epsilon/delta%20Dimer%20in%20Complex%20With%20a%20UCHT1%20Single%E2%80%90Chain%20Antibody%20Fragment&amp;author=K.%20L.%20Arnett&amp;author=S.%20C.%20Harrison&amp;author=D.%20C.%20Wiley&amp;volume=101&amp;issue=46&amp;publication_year=2004&amp;pages=16268-16273&amp;pmid=15534202&amp;doi=10.1073/pnas.0407359101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0084">
<span class="label">84.</span><cite>
Kjer‐Nielsen L., Dunstone M. A., Kostenko L., et al., “Crystal Structure of the Human T Cell Receptor CD3 Epsilon Gamma Heterodimer Complexed to the Therapeutic mAb OKT3,” Proceedings of the National Academy of Sciences of the United States of America
101, no. 20 (2004): 7675–7680.
</cite> [<a href="https://doi.org/10.1073/pnas.0402295101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC419665/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15136729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Crystal%20Structure%20of%20the%20Human%20T%20Cell%20Receptor%20CD3%20Epsilon%20Gamma%20Heterodimer%20Complexed%20to%20the%20Therapeutic%20mAb%20OKT3&amp;author=L.%20Kjer%E2%80%90Nielsen&amp;author=M.%20A.%20Dunstone&amp;author=L.%20Kostenko&amp;volume=101&amp;issue=20&amp;publication_year=2004&amp;pages=7675-7680&amp;pmid=15136729&amp;doi=10.1073/pnas.0402295101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0085">
<span class="label">85.</span><cite>
Call M. E., Schnell J. R., Xu C., Lutz R. A., Chou J. J., and Wucherpfennig K. W., “The Structure of the Zetazeta Transmembrane Dimer Reveals Features Essential for Its Assembly With the T Cell Receptor,” Cell
127, no. 2 (2006): 355–368.
</cite> [<a href="https://doi.org/10.1016/j.cell.2006.08.044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3466601/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17055436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The%20Structure%20of%20the%20Zetazeta%20Transmembrane%20Dimer%20Reveals%20Features%20Essential%20for%20Its%20Assembly%20With%20the%20T%20Cell%20Receptor&amp;author=M.%20E.%20Call&amp;author=J.%20R.%20Schnell&amp;author=C.%20Xu&amp;author=R.%20A.%20Lutz&amp;author=J.%20J.%20Chou&amp;volume=127&amp;issue=2&amp;publication_year=2006&amp;pages=355-368&amp;pmid=17055436&amp;doi=10.1016/j.cell.2006.08.044&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0086">
<span class="label">86.</span><cite>
Rudolph M. G., Stanfield R. L., and Wilson I. A., “How TCRs Bind MHCs, Peptides, and Coreceptors,” Annual Review of Immunology
24, no. 1 (2006): 419–466.</cite> [<a href="https://doi.org/10.1146/annurev.immunol.23.021704.115658" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16551255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Immunology&amp;title=How%20TCRs%20Bind%20MHCs,%20Peptides,%20and%20Coreceptors&amp;author=M.%20G.%20Rudolph&amp;author=R.%20L.%20Stanfield&amp;author=I.%20A.%20Wilson&amp;volume=24&amp;issue=1&amp;publication_year=2006&amp;pages=419-466&amp;pmid=16551255&amp;doi=10.1146/annurev.immunol.23.021704.115658&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0087">
<span class="label">87.</span><cite>
Dong D., Zheng L., Lin J., et al., “Structural Basis of Assembly of the Human T Cell Receptor‐CD3 Complex,” Nature
573, no. 7775 (2019): 546–552.
</cite> [<a href="https://doi.org/10.1038/s41586-019-1537-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31461748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structural%20Basis%20of%20Assembly%20of%20the%20Human%20T%20Cell%20Receptor%E2%80%90CD3%20Complex&amp;author=D.%20Dong&amp;author=L.%20Zheng&amp;author=J.%20Lin&amp;volume=573&amp;issue=7775&amp;publication_year=2019&amp;pages=546-552&amp;pmid=31461748&amp;doi=10.1038/s41586-019-1537-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0088">
<span class="label">88.</span><cite>
Chen Y., Zhu Y., Li X., et al., “Cholesterol Inhibits TCR Signaling by Directly Restricting TCR‐CD3 Core Tunnel Motility,” Molecular Cell
82, no. 7 (2022): 1278–1287.e5.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2022.02.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35271814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cell&amp;title=Cholesterol%20Inhibits%20TCR%20Signaling%20by%20Directly%20Restricting%20TCR%E2%80%90CD3%20Core%20Tunnel%20Motility&amp;author=Y.%20Chen&amp;author=Y.%20Zhu&amp;author=X.%20Li&amp;volume=82&amp;issue=7&amp;publication_year=2022&amp;pages=1278-1287.e5&amp;pmid=35271814&amp;doi=10.1016/j.molcel.2022.02.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0089">
<span class="label">89.</span><cite>
Liddy N., “Molecular Engineering of High Affinity T‐Cell Receptors for Bispecific Therapeutics. Phd,”
2013.  Cardiff University, <a href="https://orca.cardiff.ac.uk/id/eprint/47271/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://orca.cardiff.ac.uk/id/eprint/47271/</a>.</cite>
</li>
<li id="imr70014-bib-0090">
<span class="label">90.</span><cite>
Saotome K., Dudgeon D., Colotti K., et al., “Structural Analysis of Cancer‐Relevant TCR‐CD3 and Peptide‐MHC Complexes by cryoEM,” Nature Communications
14, no. 1 (2023): 2401.</cite> [<a href="https://doi.org/10.1038/s41467-023-37532-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10132440/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37100770/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Structural%20Analysis%20of%20Cancer%E2%80%90Relevant%20TCR%E2%80%90CD3%20and%20Peptide%E2%80%90MHC%20Complexes%20by%20cryoEM&amp;author=K.%20Saotome&amp;author=D.%20Dudgeon&amp;author=K.%20Colotti&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=2401&amp;pmid=37100770&amp;doi=10.1038/s41467-023-37532-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0091">
<span class="label">91.</span><cite>
Hu Y., Hu Q., Li Y., et al., “γδ T Cells: Origin and Fate, Subsets, Diseases and Immunotherapy,” Signal Transduction and Targeted Therapy
8, no. 1 (2023): 434.
</cite> [<a href="https://doi.org/10.1038/s41392-023-01653-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10663641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37989744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Signal%20Transduction%20and%20Targeted%20Therapy&amp;title=%CE%B3%CE%B4%20T%20Cells:%20Origin%20and%20Fate,%20Subsets,%20Diseases%20and%20Immunotherapy&amp;author=Y.%20Hu&amp;author=Q.%20Hu&amp;author=Y.%20Li&amp;volume=8&amp;issue=1&amp;publication_year=2023&amp;pages=434&amp;pmid=37989744&amp;doi=10.1038/s41392-023-01653-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0092">
<span class="label">92.</span><cite>
Rossjohn J., Gras S., Miles J. J., Turner S. J., Godfrey D. I., and McCluskey J., “T Cell Antigen Receptor Recognition of Antigen‐Presenting Molecules,” Annual Review of Immunology
33 (2015): 169–200.</cite> [<a href="https://doi.org/10.1146/annurev-immunol-032414-112334" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25493333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Immunology&amp;title=T%20Cell%20Antigen%20Receptor%20Recognition%20of%20Antigen%E2%80%90Presenting%20Molecules&amp;author=J.%20Rossjohn&amp;author=S.%20Gras&amp;author=J.%20J.%20Miles&amp;author=S.%20J.%20Turner&amp;author=D.%20I.%20Godfrey&amp;volume=33&amp;publication_year=2015&amp;pages=169-200&amp;pmid=25493333&amp;doi=10.1146/annurev-immunol-032414-112334&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0093">
<span class="label">93.</span><cite>
Allison T. J., Winter C. C., Fournié J. J., Bonneville M., and Garboczi D. N., “Structure of a Human Gammadelta T‐Cell Antigen Receptor,” Nature
411, no. 6839 (2001): 820–824.
</cite> [<a href="https://doi.org/10.1038/35081115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11459064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structure%20of%20a%20Human%20Gammadelta%20T%E2%80%90Cell%20Antigen%20Receptor&amp;author=T.%20J.%20Allison&amp;author=C.%20C.%20Winter&amp;author=J.%20J.%20Fourni%C3%A9&amp;author=M.%20Bonneville&amp;author=D.%20N.%20Garboczi&amp;volume=411&amp;issue=6839&amp;publication_year=2001&amp;pages=820-824&amp;pmid=11459064&amp;doi=10.1038/35081115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0094">
<span class="label">94.</span><cite>
Hwang W., Mallis R. J., Lang M. J., and Reinherz E. L., “The αβTCR Mechanosensor Exploits Dynamic Ectodomain Allostery to Optimize Its Ligand Recognition Site,” Proceedings of the National Academy of Sciences of the United States of America
117, no. 35 (2020): 21336–21345.
</cite> [<a href="https://doi.org/10.1073/pnas.2005899117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7474670/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32796106/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=The%20%CE%B1%CE%B2TCR%20Mechanosensor%20Exploits%20Dynamic%20Ectodomain%20Allostery%20to%20Optimize%20Its%20Ligand%20Recognition%20Site&amp;author=W.%20Hwang&amp;author=R.%20J.%20Mallis&amp;author=M.%20J.%20Lang&amp;author=E.%20L.%20Reinherz&amp;volume=117&amp;issue=35&amp;publication_year=2020&amp;pages=21336-21345&amp;pmid=32796106&amp;doi=10.1073/pnas.2005899117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0095">
<span class="label">95.</span><cite>
Pelicci P. G., Subar M., Weiss A., Dalla‐Favera R., and Littman D. R., “Molecular Diversity of the Human T‐Gamma Constant Region Genes,” Science
237, no. 4818 (1987): 1051–1055.
</cite> [<a href="https://doi.org/10.1126/science.3112943" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3112943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Molecular%20Diversity%20of%20the%20Human%20T%E2%80%90Gamma%20Constant%20Region%20Genes&amp;author=P.%20G.%20Pelicci&amp;author=M.%20Subar&amp;author=A.%20Weiss&amp;author=R.%20Dalla%E2%80%90Favera&amp;author=D.%20R.%20Littman&amp;volume=237&amp;issue=4818&amp;publication_year=1987&amp;pages=1051-1055&amp;pmid=3112943&amp;doi=10.1126/science.3112943&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0096">
<span class="label">96.</span><cite>
Lefranc M. P., Forster A., and Rabbitts T. H., “Genetic Polymorphism and Exon Changes of the Constant Regions of the Human T‐Cell Rearranging Gene Gamma,” Proceedings of the National Academy of Sciences of the United States of America
83, no. 24 (1986): 9596–9600.
</cite> [<a href="https://doi.org/10.1073/pnas.83.24.9596" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC387187/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2879283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Genetic%20Polymorphism%20and%20Exon%20Changes%20of%20the%20Constant%20Regions%20of%20the%20Human%20T%E2%80%90Cell%20Rearranging%20Gene%20Gamma&amp;author=M.%20P.%20Lefranc&amp;author=A.%20Forster&amp;author=T.%20H.%20Rabbitts&amp;volume=83&amp;issue=24&amp;publication_year=1986&amp;pages=9596-9600&amp;pmid=2879283&amp;doi=10.1073/pnas.83.24.9596&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0097">
<span class="label">97.</span><cite>
Xin W., Huang B., Chi X., et al., “Structures of Human γδ T Cell Receptor‐CD3 Complex,” Nature
630, no. 8015 (2024): 222–229.
</cite> [<a href="https://doi.org/10.1038/s41586-024-07439-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11153141/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38657677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structures%20of%20Human%20%CE%B3%CE%B4%20T%20Cell%20Receptor%E2%80%90CD3%20Complex&amp;author=W.%20Xin&amp;author=B.%20Huang&amp;author=X.%20Chi&amp;volume=630&amp;issue=8015&amp;publication_year=2024&amp;pages=222-229&amp;pmid=38657677&amp;doi=10.1038/s41586-024-07439-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0098">
<span class="label">98.</span><cite>
Hoque M., Grigg J. B., Ramlall T., et al., “Structural Characterization of Two γδ TCR/CD3 Complexes,” Nature Communications
16, no. 1 (2025): 318, 10.1038/s41467-024-55467-5.</cite> [<a href="https://doi.org/10.1038/s41467-024-55467-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11697310/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39747888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Structural%20Characterization%20of%20Two%20%CE%B3%CE%B4%20TCR/CD3%20Complexes&amp;author=M.%20Hoque&amp;author=J.%20B.%20Grigg&amp;author=T.%20Ramlall&amp;volume=16&amp;issue=1&amp;publication_year=2025&amp;pages=318&amp;pmid=39747888&amp;doi=10.1038/s41467-024-55467-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0099">
<span class="label">99.</span><cite>
Le Nours J., Gherardin N. A., Ramarathinam S. H., et al., “A Class of γδ T Cell Receptors Recognize the Underside of the Antigen‐Presenting Molecule MR1,” Science
366, no. 6472 (2019): 1522–1527.
</cite> [<a href="https://doi.org/10.1126/science.aav3900" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31857486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A%20Class%20of%20%CE%B3%CE%B4%20T%20Cell%20Receptors%20Recognize%20the%20Underside%20of%20the%20Antigen%E2%80%90Presenting%20Molecule%20MR1&amp;author=J.%20Le%20Nours&amp;author=N.%20A.%20Gherardin&amp;author=S.%20H.%20Ramarathinam&amp;volume=366&amp;issue=6472&amp;publication_year=2019&amp;pages=1522-1527&amp;pmid=31857486&amp;doi=10.1126/science.aav3900&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0100">
<span class="label">100.</span><cite>
Rice M. T., von Borstel A., Chevour P., et al., “Recognition of the Antigen‐Presenting Molecule MR1 by a Vδ3+ γδ T Cell Receptor,” Proceedings of the National Academy of Sciences of the United States of America
118, no. 49 (2021): e2110288118.
</cite> [<a href="https://doi.org/10.1073/pnas.2110288118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8694053/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34845016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Recognition%20of%20the%20Antigen%E2%80%90Presenting%20Molecule%20MR1%20by%20a%20V%CE%B43+%20%CE%B3%CE%B4%20T%20Cell%20Receptor&amp;author=M.%20T.%20Rice&amp;author=A.%20von%20Borstel&amp;author=P.%20Chevour&amp;volume=118&amp;issue=49&amp;publication_year=2021&amp;pages=e2110288118&amp;pmid=34845016&amp;doi=10.1073/pnas.2110288118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0101">
<span class="label">101.</span><cite>
Hayes S. M. and Love P. E., “Distinct Structure and Signaling Potential of the Gamma Delta TCR Complex,” Immunity
16, no. 6 (2002): 827–838.
</cite> [<a href="https://doi.org/10.1016/s1074-7613(02)00320-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12121664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Distinct%20Structure%20and%20Signaling%20Potential%20of%20the%20Gamma%20Delta%20TCR%20Complex&amp;author=S.%20M.%20Hayes&amp;author=P.%20E.%20Love&amp;volume=16&amp;issue=6&amp;publication_year=2002&amp;pages=827-838&amp;pmid=12121664&amp;doi=10.1016/s1074-7613(02)00320-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0102">
<span class="label">102.</span><cite>
Siegers G. M., Swamy M., Fernández‐Malavé E., et al., “Different Composition of the Human and the Mouse Gammadelta T Cell Receptor Explains Different Phenotypes of CD3gamma and CD3delta Immunodeficiencies,” Journal of Experimental Medicine
204, no. 11 (2007): 2537–2544.
</cite> [<a href="https://doi.org/10.1084/jem.20070782" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2118495/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17923503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Medicine&amp;title=Different%20Composition%20of%20the%20Human%20and%20the%20Mouse%20Gammadelta%20T%20Cell%20Receptor%20Explains%20Different%20Phenotypes%20of%20CD3gamma%20and%20CD3delta%20Immunodeficiencies&amp;author=G.%20M.%20Siegers&amp;author=M.%20Swamy&amp;author=E.%20Fern%C3%A1ndez%E2%80%90Malav%C3%A9&amp;volume=204&amp;issue=11&amp;publication_year=2007&amp;pages=2537-2544&amp;pmid=17923503&amp;doi=10.1084/jem.20070782&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0103">
<span class="label">103.</span><cite>
Rudd C. E., Taylor A., and Schneider H., “CD28 and CTLA‐4 Coreceptor Expression and Signal Transduction,” Immunological Reviews
229, no. 1 (2009): 12–26.
</cite> [<a href="https://doi.org/10.1111/j.1600-065X.2009.00770.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4186963/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19426212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunological%20Reviews&amp;title=CD28%20and%20CTLA%E2%80%904%20Coreceptor%20Expression%20and%20Signal%20Transduction&amp;author=C.%20E.%20Rudd&amp;author=A.%20Taylor&amp;author=H.%20Schneider&amp;volume=229&amp;issue=1&amp;publication_year=2009&amp;pages=12-26&amp;pmid=19426212&amp;doi=10.1111/j.1600-065X.2009.00770.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0104">
<span class="label">104.</span><cite>
James J. R., Oliveira M. I., Carmo A. M., Iaboni A., and Davis S. J., “A Rigorous Experimental Framework for Detecting Protein Oligomerization Using Bioluminescence Resonance Energy Transfer,” Nature Methods
3, no. 12 (2006): 1001–1006.
</cite> [<a href="https://doi.org/10.1038/nmeth978" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17086179/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=A%20Rigorous%20Experimental%20Framework%20for%20Detecting%20Protein%20Oligomerization%20Using%20Bioluminescence%20Resonance%20Energy%20Transfer&amp;author=J.%20R.%20James&amp;author=M.%20I.%20Oliveira&amp;author=A.%20M.%20Carmo&amp;author=A.%20Iaboni&amp;author=S.%20J.%20Davis&amp;volume=3&amp;issue=12&amp;publication_year=2006&amp;pages=1001-1006&amp;pmid=17086179&amp;doi=10.1038/nmeth978&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0105">
<span class="label">105.</span><cite>
Bhatia S., Edidin M., Almo S. C., and Nathenson S. G., “Different cell surface oligomeric states of B7‐1 and B7‐2: implications for signaling,” Proc Natl Acad Sci U S A
102, no. 43 (2005): 15569–15574, 10.1073/pnas.0507257102.
</cite> [<a href="https://doi.org/10.1073/pnas.0507257102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1266120/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16221763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Different%20cell%20surface%20oligomeric%20states%20of%20B7%E2%80%901%20and%20B7%E2%80%902:%20implications%20for%20signaling&amp;author=S.%20Bhatia&amp;author=M.%20Edidin&amp;author=S.%20C.%20Almo&amp;author=S.%20G.%20Nathenson&amp;volume=102&amp;issue=43&amp;publication_year=2005&amp;pages=15569-15574&amp;pmid=16221763&amp;doi=10.1073/pnas.0507257102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0106">
<span class="label">106.</span><cite>
Sansom D. M., “CD28, CTLA‐4 and Their Ligands: Who Does What and to Whom?,” Immunology
101, no. 2 (2000): 169–177.
</cite> [<a href="https://doi.org/10.1046/j.1365-2567.2000.00121.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2327073/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11012769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunology&amp;title=CD28,%20CTLA%E2%80%904%20and%20Their%20Ligands:%20Who%20Does%20What%20and%20to%20Whom?&amp;author=D.%20M.%20Sansom&amp;volume=101&amp;issue=2&amp;publication_year=2000&amp;pages=169-177&amp;pmid=11012769&amp;doi=10.1046/j.1365-2567.2000.00121.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0107">
<span class="label">107.</span><cite>
Collins A. V., Brodie D. W., Gilbert R. J. C., et al., “The Interaction Properties of Costimulatory Molecules Revisited,” Immunity
17, no. 2 (2002): 201–210.
</cite> [<a href="https://doi.org/10.1016/s1074-7613(02)00362-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12196291/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=The%20Interaction%20Properties%20of%20Costimulatory%20Molecules%20Revisited&amp;author=A.%20V.%20Collins&amp;author=D.%20W.%20Brodie&amp;author=R.%20J.%20C.%20Gilbert&amp;volume=17&amp;issue=2&amp;publication_year=2002&amp;pages=201-210&amp;pmid=12196291&amp;doi=10.1016/s1074-7613(02)00362-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0108">
<span class="label">108.</span><cite>
Sharma P., Goswami S., Raychaudhuri D., et al., “Immune Checkpoint Therapy‐Current Perspectives and Future Directions,” Cell
186, no. 8 (2023): 1652–1669.
</cite> [<a href="https://doi.org/10.1016/j.cell.2023.03.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37059068/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Immune%20Checkpoint%20Therapy%E2%80%90Current%20Perspectives%20and%20Future%20Directions&amp;author=P.%20Sharma&amp;author=S.%20Goswami&amp;author=D.%20Raychaudhuri&amp;volume=186&amp;issue=8&amp;publication_year=2023&amp;pages=1652-1669&amp;pmid=37059068&amp;doi=10.1016/j.cell.2023.03.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0109">
<span class="label">109.</span><cite>
Philips E. A., Liu J., Kvalvaag A., et al., “Transmembrane Domain‐Driven PD‐1 Dimers Mediate T Cell Inhibition,” Science Immunology
9, no. 93 (2024): eade6256, 10.1126/sciimmunol.ade6256.
</cite> [<a href="https://doi.org/10.1126/sciimmunol.ade6256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11166110/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38457513/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20Immunology&amp;title=Transmembrane%20Domain%E2%80%90Driven%20PD%E2%80%901%20Dimers%20Mediate%20T%20Cell%20Inhibition&amp;author=E.%20A.%20Philips&amp;author=J.%20Liu&amp;author=A.%20Kvalvaag&amp;volume=9&amp;issue=93&amp;publication_year=2024&amp;pages=eade6256&amp;pmid=38457513&amp;doi=10.1126/sciimmunol.ade6256&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0110">
<span class="label">110.</span><cite>
Lippert A. H., Paluch C., Gaglioni M., et al., “Antibody Agonists Trigger Immune Receptor Signaling Through Local Exclusion of Receptor‐Type Protein Tyrosine Phosphatases,” Immunity
57, no. 2 (2024): 256–270.
</cite> [<a href="https://doi.org/10.1016/j.immuni.2024.01.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38354703/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Antibody%20Agonists%20Trigger%20Immune%20Receptor%20Signaling%20Through%20Local%20Exclusion%20of%20Receptor%E2%80%90Type%20Protein%20Tyrosine%20Phosphatases&amp;author=A.%20H.%20Lippert&amp;author=C.%20Paluch&amp;author=M.%20Gaglioni&amp;volume=57&amp;issue=2&amp;publication_year=2024&amp;pages=256-270&amp;pmid=38354703&amp;doi=10.1016/j.immuni.2024.01.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0111">
<span class="label">111.</span><cite>
Sansom D. M. and Walker L. S. K., “The Role of CD28 and Cytotoxic T‐Lymphocyte Antigen‐4 (CTLA‐4) in Regulatory T‐Cell Biology,” Immunological Reviews
212, no. 1 (2006): 131–148.
</cite> [<a href="https://doi.org/10.1111/j.0105-2896.2006.00419.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16903911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunological%20Reviews&amp;title=The%20Role%20of%20CD28%20and%20Cytotoxic%20T%E2%80%90Lymphocyte%20Antigen%E2%80%904%20(CTLA%E2%80%904)%20in%20Regulatory%20T%E2%80%90Cell%20Biology&amp;author=D.%20M.%20Sansom&amp;author=L.%20S.%20K.%20Walker&amp;volume=212&amp;issue=1&amp;publication_year=2006&amp;pages=131-148&amp;pmid=16903911&amp;doi=10.1111/j.0105-2896.2006.00419.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0112">
<span class="label">112.</span><cite>
Lühder F., Huang Y., Dennehy K. M., et al., “Topological Requirements and Signaling Properties of T Cell‐Activating, Anti‐CD28 Antibody Superagonists,” Journal of Experimental Medicine
197, no. 8 (2003): 955–966, 10.1084/jem.20021024.
</cite> [<a href="https://doi.org/10.1084/jem.20021024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2193880/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12707299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Medicine&amp;title=Topological%20Requirements%20and%20Signaling%20Properties%20of%20T%20Cell%E2%80%90Activating,%20Anti%E2%80%90CD28%20Antibody%20Superagonists&amp;author=F.%20L%C3%BChder&amp;author=Y.%20Huang&amp;author=K.%20M.%20Dennehy&amp;volume=197&amp;issue=8&amp;publication_year=2003&amp;pages=955-966&amp;pmid=12707299&amp;doi=10.1084/jem.20021024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0113">
<span class="label">113.</span><cite>
Qureshi O. S., Zheng Y., Nakamura K., et al., “Trans‐Endocytosis of CD80 and CD86: A Molecular Basis for the Cell‐Extrinsic Function of CTLA‐4,” Science
332, no. 6029 (2011): 600–603, 10.1126/science.1202947.
</cite> [<a href="https://doi.org/10.1126/science.1202947" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3198051/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21474713/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Trans%E2%80%90Endocytosis%20of%20CD80%20and%20CD86:%20A%20Molecular%20Basis%20for%20the%20Cell%E2%80%90Extrinsic%20Function%20of%20CTLA%E2%80%904&amp;author=O.%20S.%20Qureshi&amp;author=Y.%20Zheng&amp;author=K.%20Nakamura&amp;volume=332&amp;issue=6029&amp;publication_year=2011&amp;pages=600-603&amp;pmid=21474713&amp;doi=10.1126/science.1202947&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0114">
<span class="label">114.</span><cite>
Schwartz J. C., Zhang X., Fedorov A. A., Nathenson S. G., and Almo S. C., “Structural Basis for Co‐Stimulation by the Human CTLA‐4/B7‐2 Complex,” Nature
410, no. 6828 (2001): 604–608.
</cite> [<a href="https://doi.org/10.1038/35069112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11279501/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structural%20Basis%20for%20Co%E2%80%90Stimulation%20by%20the%20Human%20CTLA%E2%80%904/B7%E2%80%902%20Complex&amp;author=J.%20C.%20Schwartz&amp;author=X.%20Zhang&amp;author=A.%20A.%20Fedorov&amp;author=S.%20G.%20Nathenson&amp;author=S.%20C.%20Almo&amp;volume=410&amp;issue=6828&amp;publication_year=2001&amp;pages=604-608&amp;pmid=11279501&amp;doi=10.1038/35069112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0115">
<span class="label">115.</span><cite>
Lawrence M. C. and Colman P. M., “Shape Complementarity at Protein/Protein Interfaces,” Journal of Molecular Biology
234, no. 4 (1993): 946–950.
</cite> [<a href="https://doi.org/10.1006/jmbi.1993.1648" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8263940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Molecular%20Biology&amp;title=Shape%20Complementarity%20at%20Protein/Protein%20Interfaces&amp;author=M.%20C.%20Lawrence&amp;author=P.%20M.%20Colman&amp;volume=234&amp;issue=4&amp;publication_year=1993&amp;pages=946-950&amp;pmid=8263940&amp;doi=10.1006/jmbi.1993.1648&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0116">
<span class="label">116.</span><cite>
van der Merwe P. A., Bodian D. L., Daenke S., Linsley P., and Davis S. J., “CD80 (B7‐1) Binds Both CD28 and CTLA‐4 With a Low Affinity and Very Fast Kinetics,” Journal of Experimental Medicine
185, no. 3 (1997): 393–403.
</cite> [<a href="https://doi.org/10.1084/jem.185.3.393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2196039/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9053440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Medicine&amp;title=CD80%20(B7%E2%80%901)%20Binds%20Both%20CD28%20and%20CTLA%E2%80%904%20With%20a%20Low%20Affinity%20and%20Very%20Fast%20Kinetics&amp;author=P.%20A.%20van%20der%20Merwe&amp;author=D.%20L.%20Bodian&amp;author=S.%20Daenke&amp;author=P.%20Linsley&amp;author=S.%20J.%20Davis&amp;volume=185&amp;issue=3&amp;publication_year=1997&amp;pages=393-403&amp;pmid=9053440&amp;doi=10.1084/jem.185.3.393&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0117">
<span class="label">117.</span><cite>
Xu X., Hou B., Fulzele A., et al., “PD‐1 and BTLA Regulate T Cell Signaling Differentially and Only Partially Through SHP1 and SHP2,” Journal of Cell Biology
219, no. 6 (2020): 5085, 10.1083/jcb.201905085.</cite> [<a href="https://doi.org/10.1083/jcb.201905085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7265324/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32437509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cell%20Biology&amp;title=PD%E2%80%901%20and%20BTLA%20Regulate%20T%20Cell%20Signaling%20Differentially%20and%20Only%20Partially%20Through%20SHP1%20and%20SHP2&amp;author=X.%20Xu&amp;author=B.%20Hou&amp;author=A.%20Fulzele&amp;volume=219&amp;issue=6&amp;publication_year=2020&amp;pages=5085&amp;pmid=32437509&amp;doi=10.1083/jcb.201905085&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0118">
<span class="label">118.</span><cite>
Zak K. M., Kitel R., Przetocka S., et al., “Structure of the Complex of Human Programmed Death 1, PD‐1, and Its Ligand PD‐L1,” Structure
23, no. 12 (2015): 2341–2348, 10.1016/j.str.2015.09.010.
</cite> [<a href="https://doi.org/10.1016/j.str.2015.09.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4752817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26602187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Structure%20of%20the%20Complex%20of%20Human%20Programmed%20Death%201,%20PD%E2%80%901,%20and%20Its%20Ligand%20PD%E2%80%90L1&amp;author=K.%20M.%20Zak&amp;author=R.%20Kitel&amp;author=S.%20Przetocka&amp;volume=23&amp;issue=12&amp;publication_year=2015&amp;pages=2341-2348&amp;pmid=26602187&amp;doi=10.1016/j.str.2015.09.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0119">
<span class="label">119.</span><cite>
Lin D. Y. W., Tanaka Y., Iwasaki M., et al., “The PD‐1/PD‐L1 Complex Resembles the Antigen‐Binding Fv Domains of Antibodies and T Cell Receptors,” Proceedings of the National Academy of Sciences of the United States of America
105, no. 8 (2008): 3011–3016.
</cite> [<a href="https://doi.org/10.1073/pnas.0712278105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2268576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18287011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=The%20PD%E2%80%901/PD%E2%80%90L1%20Complex%20Resembles%20the%20Antigen%E2%80%90Binding%20Fv%20Domains%20of%20Antibodies%20and%20T%20Cell%20Receptors&amp;author=D.%20Y.%20W.%20Lin&amp;author=Y.%20Tanaka&amp;author=M.%20Iwasaki&amp;volume=105&amp;issue=8&amp;publication_year=2008&amp;pages=3011-3016&amp;pmid=18287011&amp;doi=10.1073/pnas.0712278105&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0120">
<span class="label">120.</span><cite>
Dushek O., Goyette J., and van der Merwe P. A., “Non‐Catalytic Tyrosine‐Phosphorylated Receptors,” Immunological Reviews
250, no. 1 (2012): 258–276.
</cite> [<a href="https://doi.org/10.1111/imr.12008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23046135/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunological%20Reviews&amp;title=Non%E2%80%90Catalytic%20Tyrosine%E2%80%90Phosphorylated%20Receptors&amp;author=O.%20Dushek&amp;author=J.%20Goyette&amp;author=P.%20A.%20van%20der%20Merwe&amp;volume=250&amp;issue=1&amp;publication_year=2012&amp;pages=258-276&amp;pmid=23046135&amp;doi=10.1111/imr.12008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0121">
<span class="label">121.</span><cite>
Schmid E. M., Bakalar M. H., Choudhuri K., et al., “Size‐Dependent Protein Segregation at Membrane Interfaces,” Nature Physics
12, no. 7 (2016): 704–711.
</cite> [<a href="https://doi.org/10.1038/nphys3678" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5152624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27980602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Physics&amp;title=Size%E2%80%90Dependent%20Protein%20Segregation%20at%20Membrane%20Interfaces&amp;author=E.%20M.%20Schmid&amp;author=M.%20H.%20Bakalar&amp;author=K.%20Choudhuri&amp;volume=12&amp;issue=7&amp;publication_year=2016&amp;pages=704-711&amp;pmid=27980602&amp;doi=10.1038/nphys3678&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0122">
<span class="label">122.</span><cite>
Rothstein D. M., Saito H., Streuli M., Schlossman S. F., and Morimoto C., “The Alternative Splicing of the CD45 Tyrosine Phosphatase Is Controlled by Negative Regulatory Trans‐Acting Splicing Factors,” Journal of Biological Chemistry
267, no. 10 (1992): 7139–7147.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/1532394/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=The%20Alternative%20Splicing%20of%20the%20CD45%20Tyrosine%20Phosphatase%20Is%20Controlled%20by%20Negative%20Regulatory%20Trans%E2%80%90Acting%20Splicing%20Factors&amp;author=D.%20M.%20Rothstein&amp;author=H.%20Saito&amp;author=M.%20Streuli&amp;author=S.%20F.%20Schlossman&amp;author=C.%20Morimoto&amp;volume=267&amp;issue=10&amp;publication_year=1992&amp;pages=7139-7147&amp;pmid=1532394&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0123">
<span class="label">123.</span><cite>
Zareie P., Szeto C., Farenc C., et al., “Canonical T Cell Receptor Docking on Peptide‐MHC Is Essential for T Cell Signaling,” Science
372, no. 6546 (2021): eabe9124, 10.1126/science.abe9124.
</cite> [<a href="https://doi.org/10.1126/science.abe9124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34083463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Canonical%20T%20Cell%20Receptor%20Docking%20on%20Peptide%E2%80%90MHC%20Is%20Essential%20for%20T%20Cell%20Signaling&amp;author=P.%20Zareie&amp;author=C.%20Szeto&amp;author=C.%20Farenc&amp;volume=372&amp;issue=6546&amp;publication_year=2021&amp;pages=eabe9124&amp;pmid=34083463&amp;doi=10.1126/science.abe9124&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0124">
<span class="label">124.</span><cite>
Mallis R. J., Duke‐Cohan J. S., Das D. K., et al., “Molecular Design of the γδT Cell Receptor Ectodomain Encodes Biologically Fit Ligand Recognition in the Absence of Mechanosensing,” Proceedings of the National Academy of Sciences of the United States of America
118, no. 26 (2021): e2023050118.
</cite> [<a href="https://doi.org/10.1073/pnas.2023050118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8256041/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34172580/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Molecular%20Design%20of%20the%20%CE%B3%CE%B4T%20Cell%20Receptor%20Ectodomain%20Encodes%20Biologically%20Fit%20Ligand%20Recognition%20in%20the%20Absence%20of%20Mechanosensing&amp;author=R.%20J.%20Mallis&amp;author=J.%20S.%20Duke%E2%80%90Cohan&amp;author=D.%20K.%20Das&amp;volume=118&amp;issue=26&amp;publication_year=2021&amp;pages=e2023050118&amp;pmid=34172580&amp;doi=10.1073/pnas.2023050118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0125">
<span class="label">125.</span><cite>
Chen K. Y., Jenkins E., Körbel M., et al., “Trapping or Slowing the Diffusion of T Cell Receptors at Close Contacts Initiates T Cell Signaling,” Proceedings of the National Academy of Sciences of the United States of America
118, no. 39 (2021): e2024250118.
</cite> [<a href="https://doi.org/10.1073/pnas.2024250118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8488633/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34526387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Trapping%20or%20Slowing%20the%20Diffusion%20of%20T%20Cell%20Receptors%20at%20Close%20Contacts%20Initiates%20T%20Cell%20Signaling&amp;author=K.%20Y.%20Chen&amp;author=E.%20Jenkins&amp;author=M.%20K%C3%B6rbel&amp;volume=118&amp;issue=39&amp;publication_year=2021&amp;pages=e2024250118&amp;pmid=34526387&amp;doi=10.1073/pnas.2024250118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="imr70014-bib-0126">
<span class="label">126.</span><cite>
Notti R. Q., Yi F., Heissel S., et al., “The Resting and Ligand‐Bound States of the Membrane‐Embedded Human T‐Cell Receptor‐CD3 Complex,” bioRxivorg
2024 (2024): 54360, 10.1101/2023.08.22.554360.</cite> [<a href="https://doi.org/10.1101/2023.08.22.554360" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxivorg&amp;title=The%20Resting%20and%20Ligand%E2%80%90Bound%20States%20of%20the%20Membrane%E2%80%90Embedded%20Human%20T%E2%80%90Cell%20Receptor%E2%80%90CD3%20Complex&amp;author=R.%20Q.%20Notti&amp;author=F.%20Yi&amp;author=S.%20Heissel&amp;volume=2024&amp;publication_year=2024&amp;pages=54360&amp;doi=10.1101/2023.08.22.554360&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are available from the corresponding author upon reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Immunological Reviews are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1111/imr.70014"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/IMR-331-0.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11969063/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11969063/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11969063%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11969063/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11969063/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11969063/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40181535/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11969063/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40181535/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11969063/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11969063/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="H0IFDELIgTbOB50qOfWpitMRiXN3X4F8bVjKOLhBzfNPKvCWWG7WJkGGreKsaIV3">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
