
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Overcoming Analytical Challenges in Proximity Labeling Proteomics - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="3D96E0BB80955AD300E0BB000A85DC1C.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11976124/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Journal of Mass Spectrometry">
<meta name="citation_title" content="Overcoming Analytical Challenges in Proximity Labeling Proteomics">
<meta name="citation_author" content="Haorong Li">
<meta name="citation_author_institution" content="Department of Chemistry, The George Washington University, Washington, District of Columbia, USA">
<meta name="citation_author" content="Wan Nur Atiqah Binti Mazli">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of Maryland, College Park, Maryland, USA">
<meta name="citation_author" content="Ling Hao">
<meta name="citation_author_institution" content="Department of Chemistry, The George Washington University, Washington, District of Columbia, USA">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of Maryland, College Park, Maryland, USA">
<meta name="citation_publication_date" content="2025 Apr 7">
<meta name="citation_volume" content="60">
<meta name="citation_issue" content="5">
<meta name="citation_firstpage" content="e5134">
<meta name="citation_doi" content="10.1002/jms.5134">
<meta name="citation_pmid" content="40195276">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11976124/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11976124/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11976124/pdf/JMS-60-e5134.pdf">
<meta name="description" content="Proximity labeling (PL) proteomics has emerged as a powerful tool to capture both stable and transient protein interactions and subcellular networks. Despite the wide biological applications, PL still faces technical challenges in robustness, ...">
<meta name="og:title" content="Overcoming Analytical Challenges in Proximity Labeling Proteomics">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Proximity labeling (PL) proteomics has emerged as a powerful tool to capture both stable and transient protein interactions and subcellular networks. Despite the wide biological applications, PL still faces technical challenges in robustness, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11976124/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11976124">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/jms.5134"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/JMS-60-e5134.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11976124%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11976124/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11976124/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11976124/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/jms.5134" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">J Mass Spectrom</button></div>. 2025 Apr 7;60(5):e5134. doi: <a href="https://doi.org/10.1002/jms.5134" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/jms.5134</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22J%20Mass%20Spectrom%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22J%20Mass%20Spectrom%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22J%20Mass%20Spectrom%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22J%20Mass%20Spectrom%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Overcoming Analytical Challenges in Proximity Labeling Proteomics</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Haorong Li</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Haorong Li</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, The George Washington University, Washington, District of Columbia, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Haorong Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mazli%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Wan Nur Atiqah Binti Mazli</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Wan Nur Atiqah Binti Mazli</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Chemistry and Biochemistry, University of Maryland, College Park, Maryland, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mazli%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wan Nur Atiqah Binti Mazli</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hao%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Ling Hao</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Ling Hao</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, The George Washington University, Washington, District of Columbia, USA</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Chemistry and Biochemistry, University of Maryland, College Park, Maryland, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hao%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Ling Hao</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="jms5134-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Chemistry, The George Washington University, Washington, District of Columbia, USA</div>
<div id="jms5134-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Chemistry and Biochemistry, University of Maryland, College Park, Maryland, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong>
 
Ling Hao (<span>linghao1@umd.edu</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Mar 21; Received 2024 Dec 16; Accepted 2025 Mar 26; Issue date 2025 May.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Journal of Mass Spectrometry</em> published by John Wiley &amp; Sons Ltd.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11976124  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40195276/" class="usa-link">40195276</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>Proximity labeling (PL) proteomics has emerged as a powerful tool to capture both stable and transient protein interactions and subcellular networks. Despite the wide biological applications, PL still faces technical challenges in robustness, reproducibility, specificity, and sensitivity. Here, we discuss major analytical challenges in PL proteomics and highlight how the field is advancing to address these challenges by refining study design, tackling interferences, overcoming variation, developing novel tools, and establishing more robust platforms. We also provide our perspectives on best practices and the need for more robust, scalable, and quantitative PL technologies.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> biotinylation, mass spectrometry, proximity labeling</p></section></section><section id="jms5134-sec-0001"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Proximity labeling (PL) has emerged as a powerful tool to elucidate protein environments and interactions in their native cellular contexts. The major advantage of PL compared to traditional immunoprecipitation is the ability to capture transient and weak protein interactions with high spatial specificity. PL enzyme is genetically fused to a “bait” protein of interest and catalyzes covalent labeling of vicinity “prey” proteins within 10–20 nm of radius [<a href="#jms5134-bib-0001" class="usa-link" aria-describedby="jms5134-bib-0001">1</a>]. Commonly used PL enzymes include biotin ligases (e.g., BioID and TurboID) and peroxidases (e.g., HRP and APEX) [<a href="#jms5134-bib-0002" class="usa-link" aria-describedby="jms5134-bib-0002">2</a>, <a href="#jms5134-bib-0003" class="usa-link" aria-describedby="jms5134-bib-0003">3</a>, <a href="#jms5134-bib-0004" class="usa-link" aria-describedby="jms5134-bib-0004">4</a>, <a href="#jms5134-bib-0005" class="usa-link" aria-describedby="jms5134-bib-0005">5</a>]. Biotin ligase‐based PL enzymes work under mild conditions and require no exogenous cofactors, allowing broad applications in living cells and whole organisms [<a href="#jms5134-bib-0006" class="usa-link" aria-describedby="jms5134-bib-0006">6</a>]. However, their slower labeling kinetics may limit the capture of rapidly changing interactions. In contrast, peroxidase‐based enzymes provide better temporal resolution with rapid labeling (~1 min) but require peroxide treatment that may cause oxidative stress or toxicity for the applications in vivo. Other recently developed PL methods, such as photocatalytic PL [<a href="#jms5134-bib-0007" class="usa-link" aria-describedby="jms5134-bib-0007">7</a>, <a href="#jms5134-bib-0008" class="usa-link" aria-describedby="jms5134-bib-0008">8</a>, <a href="#jms5134-bib-0009" class="usa-link" aria-describedby="jms5134-bib-0009">9</a>] and split‐enzyme PL [<a href="#jms5134-bib-0004" class="usa-link" aria-describedby="jms5134-bib-0004">4</a>], offer improved spatiotemporal precision, enabling precise control over the labeling process.</p>
<p>PL has been coupled with mass spectrometry (MS)‐based proteomics to provide comprehensive identification and quantification of proteins labeled in the PL reaction [<a href="#jms5134-bib-0002" class="usa-link" aria-describedby="jms5134-bib-0002">2</a>]. PL‐proteomics has transformed our understanding of protein networks and subcellular microenvironment with various biological applications [<a href="#jms5134-bib-0001" class="usa-link" aria-describedby="jms5134-bib-0001">1</a>, <a href="#jms5134-bib-0010" class="usa-link" aria-describedby="jms5134-bib-0010">10</a>, <a href="#jms5134-bib-0011" class="usa-link" aria-describedby="jms5134-bib-0011">11</a>]. However, technical challenges still exist, such as interferences from nonspecific labeling and contaminations, experimental variation, and quantitative accuracy. Addressing these issues remains essential for improving the robustness, data quality, and for unlocking the full potential of PL in elucidating complex biological systems (Figure <a href="#jms5134-fig-0001" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="jms5134-fig-0001"><h3 class="obj_head">FIGURE 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d326/11976124/a46dcca803a9/JMS-60-e5134-g001.jpg" loading="lazy" id="jats-graphic-1" height="313" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/jms5134-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Overview of analytical challenges in proximity labeling proteomics. Abbreviations: B: biotin; iTRAQ: isobaric tags for relative and absolute quantitation; LFQ: label‐free quantification; SA: streptavidin; SILAC: stable isotope labeling by amino acids in cell culture; TMT: tandem mass tag.</p></figcaption></figure></section><section id="jms5134-sec-0002"><h2 class="pmc_sec_title">2. Study Design: The Foundation for Success</h2>
<p>A meticulous study design is key to produce reliable data and meaningful biological insights in PL experiments. Several critical factors include cell line establishment, the selection and validation of the expression of bait‐PL construct, the use of effective controls, and the selection of enrichment strategies. PL probe requires the genetic fusion of a PL enzyme to the bait protein which can be achieved by establishing a stable cell line or transient transfection. Therefore, users need to balance specific experimental needs for consistent long‐term study (stable cell line) versus the flexibility for rapid assessment (transient transfection) [<a href="#jms5134-bib-0012" class="usa-link" aria-describedby="jms5134-bib-0012">12</a>]. The expression level of the bait protein‐PL construct also influences the sensitivity and selectivity of resulted interactors. Endogenously knocking in the PL enzyme to the bait protein preserves the physiological relevance with great labeling specificity but compromises sensitivity and requires larger amount of starting material compared to the overexpression probe [<a href="#jms5134-bib-0013" class="usa-link" aria-describedby="jms5134-bib-0013">13</a>, <a href="#jms5134-bib-0014" class="usa-link" aria-describedby="jms5134-bib-0014">14</a>]. Different PL enzymes catalyze reactions that label different protein residues. Biotin ligase‐based PL reaction labels the primary amine group (lysine residues), whereas peroxidase‐catalyzed reaction labels the electron‐rich structure (tyrosine residues) [<a href="#jms5134-bib-0014" class="usa-link" aria-describedby="jms5134-bib-0014">14</a>, <a href="#jms5134-bib-0015" class="usa-link" aria-describedby="jms5134-bib-0015">15</a>]. Most proteins are rich in lysine and tyrosine and therefore should not be significantly influenced. However, proteins that do not have surface‐exposed lysine or tyrosine residues may not be labeled during PL reaction despite their proximity to the bait protein.</p>
<p>The proper use of controls is perhaps the most crucial aspect of study design. A negative control without the PL enzyme expression can provide insights into contaminations and nonspecific bindings associated with the enrichment strategies. A spatial control expressing PL probe at a specific subcellular location can reduce possible stochastic labeling of high abundant bystanders during the PL reaction [<a href="#jms5134-bib-0016" class="usa-link" aria-describedby="jms5134-bib-0016">16</a>]. The control group is often used to compare with the experimental group to generate relative fold changes/ratios in the proteomics results. It is beneficial to tune the expression level of PL probes to be at similar levels for ratiometric comparison. If ratio is highly skewed, normalization strategies can also be used during sample preparation or data analysis. Validating the labeling reaction and subcellular location of the PL probe also involves the use of control groups in fluorescence microscopy and western blot experiments. Whenever possible, including both negative control and spatial control will improve the confidence of the results.</p>
<p>PL labeling involves the use of biotin or biotin derivatives as the substrate. Biotinylated proteins are then enriched at the protein or peptide level. Protein‐level enrichment purifies intact biotinylated proteins using affinity matrices such as streptavidin or neutravidin‐conjugated beads. In contrast, peptide‐level enrichment specifically captures biotinylated peptides after enzymatic digestion by using biotin antibody, neutravidin, or reversible biotin‐binding entities [<a href="#jms5134-bib-0017" class="usa-link" aria-describedby="jms5134-bib-0017">17</a>, <a href="#jms5134-bib-0018" class="usa-link" aria-describedby="jms5134-bib-0018">18</a>, <a href="#jms5134-bib-0019" class="usa-link" aria-describedby="jms5134-bib-0019">19</a>, <a href="#jms5134-bib-0020" class="usa-link" aria-describedby="jms5134-bib-0020">20</a>]. The selection of protein‐level or peptide‐level enrichment strategy should align with the specific objectives of the PL study, as each approach offers distinct advantages and limitations [<a href="#jms5134-bib-0021" class="usa-link" aria-describedby="jms5134-bib-0021">21</a>]. Protein‐level enrichment identifies more proteins and unique peptides per protein for more confident protein identification and quantification compared to the peptide‐level enrichment. Protein‐level enrichment also allows flexible buffer exchange for various downstream experiments, such as MS‐based proteomics and western blotting [<a href="#jms5134-bib-0002" class="usa-link" aria-describedby="jms5134-bib-0002">2</a>, <a href="#jms5134-bib-0003" class="usa-link" aria-describedby="jms5134-bib-0003">3</a>]. But protein‐level enrichment often copurifies nonspecific labeling or secondary protein interactors, leading to increased false positives. On the other hand, peptide‐level enrichment that captures biotinylation residue provides direct evidence that the candidate protein was once near the bait protein during its life span. Since biotinylation only occurs at surface‐exposed amino acid residues, biotinylation sites also provide valuable information for characterizing protein structure and membrane protein topology [<a href="#jms5134-bib-0019" class="usa-link" aria-describedby="jms5134-bib-0019">19</a>]. However, peptide‐level enrichment may be subject to false negatives due to limited peptide coverage of each protein [<a href="#jms5134-bib-0022" class="usa-link" aria-describedby="jms5134-bib-0022">22</a>]. A large amount of starting material is often needed for protease digestion before peptide‐level enrichment, which significantly increases experimental cost.</p></section><section id="jms5134-sec-0003"><h2 class="pmc_sec_title">3. Tackling Interferences Throughout the Workflow</h2>
<p>PL experiments are prone to various interferences that can bury true interactors and impede biological interpretations. It is essential to understand the causes of interferences and implementing optimizations to reduce interferences, such as nonspecific biotin labeling during PL reaction, nonspecific binding during enrichment, and contaminations throughout the proteomics pipeline. During the PL reaction, the diffusion of reactive biotin radicals beyond the target region will compromise the spatial and temporal resolution of the PL proteomics results. Therefore, it is critical to precisely control labeling time and promptly quench the reaction to limit nonspecific labeling. Biotin‐dependent carboxylases within the mitochondria, such as PC, PCC, ACACA, and MCC, are highly enriched in PL‐proteomics dataset [<a href="#jms5134-bib-0023" class="usa-link" aria-describedby="jms5134-bib-0023">23</a>, <a href="#jms5134-bib-0024" class="usa-link" aria-describedby="jms5134-bib-0024">24</a>]. Together with their interactors inside mitochondria, PL‐proteomics data may therefore present a false positive GO term enrichment at the mitochondrial location. We previously found that using a cleavable biotin substrate can significantly reduce the levels of these endogenously biotinylated proteins [<a href="#jms5134-bib-0015" class="usa-link" aria-describedby="jms5134-bib-0015">15</a>]. Although difficult to remove, endogenously biotinylated proteins can be used to normalize different batches of PL labeling experiments to reduce variations in the proteomics data [<a href="#jms5134-bib-0014" class="usa-link" aria-describedby="jms5134-bib-0014">14</a>]. Bioinformatic tools such as SAINT [<a href="#jms5134-bib-0025" class="usa-link" aria-describedby="jms5134-bib-0025">25</a>] can be used to assign confidence scores to proteins based on quantitative data and the negative control to filter out interference proteins from the proteomics results.</p>
<p>Besides false‐positive interactors during the labeling reaction, nonspecific bindings during the enrichment step represent another layer of interferences in PL experiments. Insufficient removal of nonbiotinylated proteins during enrichment can originate from the limited specificity of affinity beads, high viscosity of cell lysate, and insufficient washing step. Several methods can be used to decrease the viscosity of cell lysate, such as removing nucleic acids from the samples by sonication or RNase treatment and diluting the cell lysate before enrichment. Stringent beads washing with high‐salt and detergent‐containing buffers could effectively disrupt nonspecific bindings to the beads. At the data analysis level, a common interference protein list can be helpful to mark and remove interference proteins, such as the CRAPome repository for affinity purification MS data [<a href="#jms5134-bib-0026" class="usa-link" aria-describedby="jms5134-bib-0026">26</a>].</p>
<p>Streptavidin‐coated bead is the most commonly used method to enrich biotinylated proteins because of the strong affinity between streptavidin and biotin (Kd ~ 10<sup>−14</sup>). Streptavidin is a 52‐kDa protein that will be inevitably leaked and/or digested into the samples during biotinylation enrichment, on‐bead digestion, or elution steps [<a href="#jms5134-bib-0022" class="usa-link" aria-describedby="jms5134-bib-0022">22</a>, <a href="#jms5134-bib-0027" class="usa-link" aria-describedby="jms5134-bib-0027">27</a>]. Several methods have been developed in recent years to mitigate streptavidin contamination. Beads titration assay needs to be used for each PL probe to obtain an optimal ratio between streptavidin beads and input protein ratio to avoid excessive use of streptavidin [<a href="#jms5134-bib-0004" class="usa-link" aria-describedby="jms5134-bib-0004">4</a>, <a href="#jms5134-bib-0024" class="usa-link" aria-describedby="jms5134-bib-0024">24</a>]. Protease‐resistant beads can be used to reduce streptavidin peptides during on‐bead digestion [<a href="#jms5134-bib-0028" class="usa-link" aria-describedby="jms5134-bib-0028">28</a>]. Cleavable biotin probes were developed to allow convenient release of biotinylated proteins without the need for on‐bead digestion [<a href="#jms5134-bib-0015" class="usa-link" aria-describedby="jms5134-bib-0015">15</a>, <a href="#jms5134-bib-0029" class="usa-link" aria-describedby="jms5134-bib-0029">29</a>]. Other type of beads such as neutravidin, avidin, and biotin antibody can be used as described in the previous section. Streptavidin, together with other commonly observed contaminant proteins from the proteomics workflow, should be marked by introducing a contaminant FASTA library and removed after data analysis [<a href="#jms5134-bib-0030" class="usa-link" aria-describedby="jms5134-bib-0030">30</a>]. Because of the use of detergent during cell lysis and beads washing, PL proteomics is also prone to detergent contamination. Detection and removal of residue detergent are critical to ensure data quality and protect LC–MS instruments. Recently developed ContamSPOT assay [<a href="#jms5134-bib-0031" class="usa-link" aria-describedby="jms5134-bib-0031">31</a>] can be used to detect and quantify trace levels of detergents from 1 μL of sample input via colorimetric and fluorometric assays on a TLC plate, which can be coupled with ethyl‐acetate liquid–liquid extraction to remove residue detergent prior to MS injection.</p></section><section id="jms5134-sec-0004"><h2 class="pmc_sec_title">4. Overcoming Variation and Improving Scalability</h2>
<p>PL proteomics is influenced by multiple sources of variations, from <em>in situ</em> PL labeling reaction, complex sample preparation procedures, and LC–MS analysis [<a href="#jms5134-bib-0032" class="usa-link" aria-describedby="jms5134-bib-0032">32</a>]. First, the efficiency of the labeling reaction can fluctuate. Biotin substrate may be difficult to dissolve, which influences the local biotin concentration for PL reaction. Prewarming, sonication, and predissolving in warm cell culture medium can help dissolve the reagent for homogenous labeling among cell populations. Likewise, PL labeling reaction duration should be precisely controlled and promptly quenched to reduce variations. Additionally, the lot‐to‐lot production variations of enrichment beads, as well as beads from different vendors have been found to influence binding capacity, coating density, and beads concentration [<a href="#jms5134-bib-0033" class="usa-link" aria-describedby="jms5134-bib-0033">33</a>]. This highlights the importance of rigorous quality control and vendor consistency in bead manufacturing, as well as the necessity of standardizing the test for binding capacity by dot blot or colorimetric assays [<a href="#jms5134-bib-0024" class="usa-link" aria-describedby="jms5134-bib-0024">24</a>, <a href="#jms5134-bib-0027" class="usa-link" aria-describedby="jms5134-bib-0027">27</a>]. Stable isotope labeling technique with multiplexing capabilities can be used to reduce experimental and instrumental variations, such as TMT [<a href="#jms5134-bib-0034" class="usa-link" aria-describedby="jms5134-bib-0034">34</a>], iTRAQ [<a href="#jms5134-bib-0035" class="usa-link" aria-describedby="jms5134-bib-0035">35</a>], and SILAC labeling [<a href="#jms5134-bib-0036" class="usa-link" aria-describedby="jms5134-bib-0036">36</a>]. Trysin/Lys‐C mix could be used instead of trypsin for protein digestion to improve digestion efficiency and reproducibility [<a href="#jms5134-bib-0037" class="usa-link" aria-describedby="jms5134-bib-0037">37</a>]. Additionally, data normalization and batch corrections are important for large‐scale studies to enhance comparability across runs [<a href="#jms5134-bib-0038" class="usa-link" aria-describedby="jms5134-bib-0038">38</a>]. Emerging machine learning‐based bioinformatic tools are transforming LC–MS data normalization by enhancing peak classification and retention time alignment [<a href="#jms5134-bib-0039" class="usa-link" aria-describedby="jms5134-bib-0039">39</a>, <a href="#jms5134-bib-0040" class="usa-link" aria-describedby="jms5134-bib-0040">40</a>]. With recent advancements in robotics for proteomics sample preparation, such as using liquid handlers [<a href="#jms5134-bib-0041" class="usa-link" aria-describedby="jms5134-bib-0041">41</a>, <a href="#jms5134-bib-0042" class="usa-link" aria-describedby="jms5134-bib-0042">42</a>] or magnetic beads processors [<a href="#jms5134-bib-0043" class="usa-link" aria-describedby="jms5134-bib-0043">43</a>], automating PL sample preparation is an attractive route to mitigate human error, reduce handling time, ensure consistent sample handling, and increase throughput (with 96‐well plate) [<a href="#jms5134-bib-0021" class="usa-link" aria-describedby="jms5134-bib-0021">21</a>, <a href="#jms5134-bib-0044" class="usa-link" aria-describedby="jms5134-bib-0044">44</a>, <a href="#jms5134-bib-0045" class="usa-link" aria-describedby="jms5134-bib-0045">45</a>].</p></section><section id="jms5134-sec-0005"><h2 class="pmc_sec_title">5. Conclusions</h2>
<p>Addressing the analytical challenges in PL proteomics is essential to ensure reliable and reproducible results and unlock the full potential of PL. In this perspective, we discussed critical technical issues in PL proteomics experiments, including study design, the origins of interferences, and sources of variability, and highlighted recent strategies to overcome these challenges. Developing next‐generation PL probes will further improve its spatial and temporal resolution. To achieve these advances, it is important to foster interdisciplinary collaborations among biologists, chemists, engineers, instrumentation experts, and data scientists to transform the PL technique into a more quantitative, robust, scalable, and versatile tool. These advancements will ultimately provide deeper insights into cellular microenvironments, dynamic molecular interactions, and signaling networks of biological systems.</p></section><section id="jms5134-sec-0006" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This study is supported by the NSF CAREER grant (CHE 2239214, Hao) and the Cottrell Scholar Award from the Research Corporation for Science Advancement (Hao). H.L. acknowledges the Mary Hopkins Shepard Doctoral Fellowship.</p></section><section id="notes1"><section id="jms5134-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="jms5134-note-0001"><p>
<strong>Funding:</strong> This work was supported by the National Science Foundation, CHEM (2239214) and the Research Corporation for Science Advancement, Cottrell Scholar.</p></div></div></section></section><section id="jms5134-sec-0008"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>Data sharing is not applicable to this article as no datasets were generated or analyzed during the current study.</p></section><section id="jms5134-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="jms5134-bibl-0001_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="jms5134-bib-0001">
<span class="label">1.</span><cite>
Qin W., Cho K. F., Cavanagh P. E., and Ting A. Y., “Deciphering Molecular Interactions by Proximity Labeling,” Nature Methods
18 (2021): 133–143.
</cite> [<a href="https://doi.org/10.1038/s41592-020-01010-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10548357/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33432242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Deciphering%20Molecular%20Interactions%20by%20Proximity%20Labeling&amp;author=W.%20Qin&amp;author=K.%20F.%20Cho&amp;author=P.%20E.%20Cavanagh&amp;author=A.%20Y.%20Ting&amp;volume=18&amp;publication_year=2021&amp;pages=133-143&amp;pmid=33432242&amp;doi=10.1038/s41592-020-01010-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0002">
<span class="label">2.</span><cite>
Rhee H. W., Zou P., Udeshi N. D., et al., “Proteomic Mapping of Mitochondria in Living Cells via Spatially Restricted Enzymatic Tagging,” Science
1979, no. 339 (2013): 1328–1331.</cite> [<a href="https://doi.org/10.1126/science.1230593" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3916822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23371551/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Proteomic%20Mapping%20of%20Mitochondria%20in%20Living%20Cells%20via%20Spatially%20Restricted%20Enzymatic%20Tagging&amp;author=H.%20W.%20Rhee&amp;author=P.%20Zou&amp;author=N.%20D.%20Udeshi&amp;volume=1979&amp;issue=339&amp;publication_year=2013&amp;pages=1328-1331&amp;pmid=23371551&amp;doi=10.1126/science.1230593&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0003">
<span class="label">3.</span><cite>
Roux K. J., Kim D. I., Raida M., and Burke B., “A Promiscuous Biotin Ligase Fusion Protein Identifies Proximal and Interacting Proteins in Mammalian Cells,” Journal of Cell Biology
196 (2012): 801–810.
</cite> [<a href="https://doi.org/10.1083/jcb.201112098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3308701/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22412018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cell%20Biology&amp;title=A%20Promiscuous%20Biotin%20Ligase%20Fusion%20Protein%20Identifies%20Proximal%20and%20Interacting%20Proteins%20in%20Mammalian%20Cells&amp;author=K.%20J.%20Roux&amp;author=D.%20I.%20Kim&amp;author=M.%20Raida&amp;author=B.%20Burke&amp;volume=196&amp;publication_year=2012&amp;pages=801-810&amp;pmid=22412018&amp;doi=10.1083/jcb.201112098&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0004">
<span class="label">4.</span><cite>
Cho K. F., Branon T. C., Udeshi N. D., Myers S. A., Carr S. A., and Ting A. Y., “Proximity Labeling in Mammalian Cells With TurboID and Split‐TurboID,” Nature Protocols
15 (2020): 3971–3999.
</cite> [<a href="https://doi.org/10.1038/s41596-020-0399-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33139955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Protocols&amp;title=Proximity%20Labeling%20in%20Mammalian%20Cells%20With%20TurboID%20and%20Split%E2%80%90TurboID&amp;author=K.%20F.%20Cho&amp;author=T.%20C.%20Branon&amp;author=N.%20D.%20Udeshi&amp;author=S.%20A.%20Myers&amp;author=S.%20A.%20Carr&amp;volume=15&amp;publication_year=2020&amp;pages=3971-3999&amp;pmid=33139955&amp;doi=10.1038/s41596-020-0399-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0005">
<span class="label">5.</span><cite>
Samavarchi‐Tehrani P., Samson R., and Gingras A. C., “Proximity Dependent Biotinylation: Key Enzymes and Adaptation to Proteomics Approaches,” Molecular and Cellular Proteomics
19 (2020): 757–773.
</cite> [<a href="https://doi.org/10.1074/mcp.R120.001941" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7196579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32127388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20and%20Cellular%20Proteomics&amp;title=Proximity%20Dependent%20Biotinylation:%20Key%20Enzymes%20and%20Adaptation%20to%20Proteomics%20Approaches&amp;author=P.%20Samavarchi%E2%80%90Tehrani&amp;author=R.%20Samson&amp;author=A.%20C.%20Gingras&amp;volume=19&amp;publication_year=2020&amp;pages=757-773&amp;pmid=32127388&amp;doi=10.1074/mcp.R120.001941&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0006">
<span class="label">6.</span><cite>
Kim K. E., Park I., Kim J., et al., “Dynamic Tracking and Identification of Tissue‐Specific Secretory Proteins in the Circulation of Live Mice,” Nature Communications
12 (2021): 5204.</cite> [<a href="https://doi.org/10.1038/s41467-021-25546-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8410947/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34471136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Dynamic%20Tracking%20and%20Identification%20of%20Tissue%E2%80%90Specific%20Secretory%20Proteins%20in%20the%20Circulation%20of%20Live%20Mice&amp;author=K.%20E.%20Kim&amp;author=I.%20Park&amp;author=J.%20Kim&amp;volume=12&amp;publication_year=2021&amp;pages=5204&amp;pmid=34471136&amp;doi=10.1038/s41467-021-25546-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0007">
<span class="label">7.</span><cite>
Tay N. E. S., Ryu K. A., Weber J. L., et al., “Targeted Activation in Localized Protein Environments via Deep red Photoredox Catalysis,” Nature Chemistry
15 (2023): 101–109.</cite> [<a href="https://doi.org/10.1038/s41557-022-01057-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9840673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36216892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Chemistry&amp;title=Targeted%20Activation%20in%20Localized%20Protein%20Environments%20via%20Deep%20red%20Photoredox%20Catalysis&amp;author=N.%20E.%20S.%20Tay&amp;author=K.%20A.%20Ryu&amp;author=J.%20L.%20Weber&amp;volume=15&amp;publication_year=2023&amp;pages=101-109&amp;pmid=36216892&amp;doi=10.1038/s41557-022-01057-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0008">
<span class="label">8.</span><cite>
Zhai Y., Huang X., Zhang K., et al., “Spatiotemporal‐Resolved Protein Networks Profiling With Photoactivation Dependent Proximity Labeling,” Nature Communications
13 (2022): 4906.</cite> [<a href="https://doi.org/10.1038/s41467-022-32689-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9392063/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35987950/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Spatiotemporal%E2%80%90Resolved%20Protein%20Networks%20Profiling%20With%20Photoactivation%20Dependent%20Proximity%20Labeling&amp;author=Y.%20Zhai&amp;author=X.%20Huang&amp;author=K.%20Zhang&amp;volume=13&amp;publication_year=2022&amp;pages=4906&amp;pmid=35987950&amp;doi=10.1038/s41467-022-32689-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0009">
<span class="label">9.</span><cite>
Lee S. Y., Cheah J. S., Zhao B., et al., “Engineered Allostery in Light‐Regulated LOV‐Turbo Enables Precise Spatiotemporal Control of Proximity Labeling in Living Cells,” Nature Methods
20 (2023): 908–917.
</cite> [<a href="https://doi.org/10.1038/s41592-023-01880-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10539039/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37188954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Engineered%20Allostery%20in%20Light%E2%80%90Regulated%20LOV%E2%80%90Turbo%20Enables%20Precise%20Spatiotemporal%20Control%20of%20Proximity%20Labeling%20in%20Living%20Cells&amp;author=S.%20Y.%20Lee&amp;author=J.%20S.%20Cheah&amp;author=B.%20Zhao&amp;volume=20&amp;publication_year=2023&amp;pages=908-917&amp;pmid=37188954&amp;doi=10.1038/s41592-023-01880-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0010">
<span class="label">10.</span><cite>
Han S., Li J., and Ting A. Y., “Proximity Labeling: Spatially Resolved Proteomic Mapping for Neurobiology,” Current Opinion in Neurobiology
50 (2018): 17–23.
</cite> [<a href="https://doi.org/10.1016/j.conb.2017.10.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6726430/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29125959/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Neurobiology&amp;title=Proximity%20Labeling:%20Spatially%20Resolved%20Proteomic%20Mapping%20for%20Neurobiology&amp;author=S.%20Han&amp;author=J.%20Li&amp;author=A.%20Y.%20Ting&amp;volume=50&amp;publication_year=2018&amp;pages=17-23&amp;pmid=29125959&amp;doi=10.1016/j.conb.2017.10.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0011">
<span class="label">11.</span><cite>
Go C. D., Knight J. D. R., Rajasekharan A., et al., “A Proximity‐Dependent Biotinylation Map of a Human Cell,” Nature
595 (2021): 120–124.
</cite> [<a href="https://doi.org/10.1038/s41586-021-03592-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34079125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A%20Proximity%E2%80%90Dependent%20Biotinylation%20Map%20of%20a%20Human%20Cell&amp;author=C.%20D.%20Go&amp;author=J.%20D.%20R.%20Knight&amp;author=A.%20Rajasekharan&amp;volume=595&amp;publication_year=2021&amp;pages=120-124&amp;pmid=34079125&amp;doi=10.1038/s41586-021-03592-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0012">
<span class="label">12.</span><cite>
Kim T. K. and Eberwine J. H., “Mammalian Cell Transfection: The Present and the Future,” Analytical and Bioanalytical Chemistry
397 (2010): 3173–3178.
</cite> [<a href="https://doi.org/10.1007/s00216-010-3821-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2911531/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20549496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20and%20Bioanalytical%20Chemistry&amp;title=Mammalian%20Cell%20Transfection:%20The%20Present%20and%20the%20Future&amp;author=T.%20K.%20Kim&amp;author=J.%20H.%20Eberwine&amp;volume=397&amp;publication_year=2010&amp;pages=3173-3178&amp;pmid=20549496&amp;doi=10.1007/s00216-010-3821-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0013">
<span class="label">13.</span><cite>
Stockhammer A., Spalt C., Klemt A., et al., “When Less Is More – A Fast TurboID Knock‐In Approach for High‐Sensitivity Endogenous Interactome Mapping,” Journal of Cell Science
137 (2024): 16.</cite> [<a href="https://doi.org/10.1242/jcs.261952" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11385326/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39056144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cell%20Science&amp;title=When%20Less%20Is%20More%20%E2%80%93%20A%20Fast%20TurboID%20Knock%E2%80%90In%20Approach%20for%20High%E2%80%90Sensitivity%20Endogenous%20Interactome%20Mapping&amp;author=A.%20Stockhammer&amp;author=C.%20Spalt&amp;author=A.%20Klemt&amp;volume=137&amp;publication_year=2024&amp;pages=16&amp;pmid=39056144&amp;doi=10.1242/jcs.261952&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0014">
<span class="label">14.</span><cite>
Frankenfield A. M., Fernandopulle M. S., Hasan S., Ward M. E., and Hao L., “Development and Comparative Evaluation of Endolysosomal Proximity Labeling‐Based Proteomic Methods in Human iPSC‐Derived Neurons,” Analytical Chemistry
92 (2020): 15437–15444.
</cite> [<a href="https://doi.org/10.1021/acs.analchem.0c03107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8895409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33201688/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Development%20and%20Comparative%20Evaluation%20of%20Endolysosomal%20Proximity%20Labeling%E2%80%90Based%20Proteomic%20Methods%20in%20Human%20iPSC%E2%80%90Derived%20Neurons&amp;author=A.%20M.%20Frankenfield&amp;author=M.%20S.%20Fernandopulle&amp;author=S.%20Hasan&amp;author=M.%20E.%20Ward&amp;author=L.%20Hao&amp;volume=92&amp;publication_year=2020&amp;pages=15437-15444&amp;pmid=33201688&amp;doi=10.1021/acs.analchem.0c03107&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0015">
<span class="label">15.</span><cite>
Li H., Frankenfield A. M., Houston R., Sekine S., and Hao L., “Thiol‐Cleavable Biotin for Chemical and Enzymatic Biotinylation and Its Application to Mitochondrial TurboID Proteomics,” Journal of the American Society for Mass Spectrometry
32 (2021): 2358–2365.
</cite> [<a href="https://doi.org/10.1021/jasms.1c00079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8898397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33909971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Mass%20Spectrometry&amp;title=Thiol%E2%80%90Cleavable%20Biotin%20for%20Chemical%20and%20Enzymatic%20Biotinylation%20and%20Its%20Application%20to%20Mitochondrial%20TurboID%20Proteomics&amp;author=H.%20Li&amp;author=A.%20M.%20Frankenfield&amp;author=R.%20Houston&amp;author=S.%20Sekine&amp;author=L.%20Hao&amp;volume=32&amp;publication_year=2021&amp;pages=2358-2365&amp;pmid=33909971&amp;doi=10.1021/jasms.1c00079&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0016">
<span class="label">16.</span><cite>
Lobingier B. T., Hüttenhain R., Eichel K., et al., “An Approach to Spatiotemporally Resolve Protein Interaction Networks in Living Cells,” Cell
169 (2017): 350–360.e12.
</cite> [<a href="https://doi.org/10.1016/j.cell.2017.03.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5616215/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28388416/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=An%20Approach%20to%20Spatiotemporally%20Resolve%20Protein%20Interaction%20Networks%20in%20Living%20Cells&amp;author=B.%20T.%20Lobingier&amp;author=R.%20H%C3%BCttenhain&amp;author=K.%20Eichel&amp;volume=169&amp;publication_year=2017&amp;pages=350-360.e12&amp;pmid=28388416&amp;doi=10.1016/j.cell.2017.03.022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0017">
<span class="label">17.</span><cite>
Motani K. and Kosako H., “BioID Screening of Biotinylation Sites Using the Avidin‐Like Protein Tamavidin 2‐REV Identifies Global Interactors of Stimulator of Interferon Genes (STING),” Journal of Biological Chemistry
295 (2020): 11174–11183.
</cite> [<a href="https://doi.org/10.1074/jbc.RA120.014323" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7415991/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32554809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=BioID%20Screening%20of%20Biotinylation%20Sites%20Using%20the%20Avidin%E2%80%90Like%20Protein%20Tamavidin%202%E2%80%90REV%20Identifies%20Global%20Interactors%20of%20Stimulator%20of%20Interferon%20Genes%20(STING)&amp;author=K.%20Motani&amp;author=H.%20Kosako&amp;volume=295&amp;publication_year=2020&amp;pages=11174-11183&amp;pmid=32554809&amp;doi=10.1074/jbc.RA120.014323&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0018">
<span class="label">18.</span><cite>
Kim D. I., Cutler J. A., Na C. H., et al., “BioSITe: A Method for Direct Detection and Quantitation of Site‐Specific Biotinylation,” Journal of Proteome Research
17 (2018): 759–769.
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.7b00775" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6092923/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29249144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteome%20Research&amp;title=BioSITe:%20A%20Method%20for%20Direct%20Detection%20and%20Quantitation%20of%20Site%E2%80%90Specific%20Biotinylation&amp;author=D.%20I.%20Kim&amp;author=J.%20A.%20Cutler&amp;author=C.%20H.%20Na&amp;volume=17&amp;publication_year=2018&amp;pages=759-769&amp;pmid=29249144&amp;doi=10.1021/acs.jproteome.7b00775&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0019">
<span class="label">19.</span><cite>
Udeshi N. D., Pedram K., Svinkina T., et al., “Antibodies to Biotin Enable Large‐Scale Detection of Biotinylation Sites on Proteins,” Nature Methods
14 (2017): 1167–1170.
</cite> [<a href="https://doi.org/10.1038/nmeth.4465" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8954634/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29039416/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Antibodies%20to%20Biotin%20Enable%20Large%E2%80%90Scale%20Detection%20of%20Biotinylation%20Sites%20on%20Proteins&amp;author=N.%20D.%20Udeshi&amp;author=K.%20Pedram&amp;author=T.%20Svinkina&amp;volume=14&amp;publication_year=2017&amp;pages=1167-1170&amp;pmid=29039416&amp;doi=10.1038/nmeth.4465&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0020">
<span class="label">20.</span><cite>
Schiapparelli L. M., McClatchy D. B., Liu H. H., Sharma P., 
J. R.
Yates, III
, and Cline H. T., “Direct Detection of Biotinylated Proteins by Mass Spectrometry,” Journal of Proteome Research
13 (2014): 3966–3978.
</cite> [<a href="https://doi.org/10.1021/pr5002862" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4156236/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25117199/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteome%20Research&amp;title=Direct%20Detection%20of%20Biotinylated%20Proteins%20by%20Mass%20Spectrometry&amp;author=L.%20M.%20Schiapparelli&amp;author=D.%20B.%20McClatchy&amp;author=H.%20H.%20Liu&amp;author=P.%20Sharma&amp;author=J.%20R.%20Yates&amp;volume=13&amp;publication_year=2014&amp;pages=3966-3978&amp;pmid=25117199&amp;doi=10.1021/pr5002862&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0021">
<span class="label">21.</span><cite>
Li H., Smeriglio N., Ni J., Wang Y., Sekine S., and Hao L., “Benchmarking and Automating the Biotinylation Proteomics Workflow,” Research Square (2024): preprint, rs‐3, 10.21203/rs.3.rs-4590410/v1.</cite> [<a href="https://doi.org/10.21203/rs.3.rs-4590410/v1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Research%20Square&amp;title=Benchmarking%20and%20Automating%20the%20Biotinylation%20Proteomics%20Workflow&amp;author=H.%20Li&amp;author=N.%20Smeriglio&amp;author=J.%20Ni&amp;author=Y.%20Wang&amp;author=S.%20Sekine&amp;publication_year=2024&amp;doi=10.21203/rs.3.rs-4590410/v1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0022">
<span class="label">22.</span><cite>
Niinae T., Ishihama Y., and Imami K., “Biotinylation‐Based Proximity Labelling Proteomics: Basics, Applications and Technical Considerations,” Journal of Biochemistry
170 (2021): 569–576.
</cite> [<a href="https://doi.org/10.1093/jb/mvab123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34752609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biochemistry&amp;title=Biotinylation%E2%80%90Based%20Proximity%20Labelling%20Proteomics:%20Basics,%20Applications%20and%20Technical%20Considerations&amp;author=T.%20Niinae&amp;author=Y.%20Ishihama&amp;author=K.%20Imami&amp;volume=170&amp;publication_year=2021&amp;pages=569-576&amp;pmid=34752609&amp;doi=10.1093/jb/mvab123&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0023">
<span class="label">23.</span><cite>
Tytgat H. L., Schoofs G., Driesen M., et al., “Endogenous Biotin‐Binding Proteins: An Overlooked Factor Causing False Positives in Streptavidin‐Based Protein Detection,” Microbial Biotechnology
8 (2015): 164–168.
</cite> [<a href="https://doi.org/10.1111/1751-7915.12150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4321382/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25211245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbial%20Biotechnology&amp;title=Endogenous%20Biotin%E2%80%90Binding%20Proteins:%20An%20Overlooked%20Factor%20Causing%20False%20Positives%20in%20Streptavidin%E2%80%90Based%20Protein%20Detection&amp;author=H.%20L.%20Tytgat&amp;author=G.%20Schoofs&amp;author=M.%20Driesen&amp;volume=8&amp;publication_year=2015&amp;pages=164-168&amp;pmid=25211245&amp;doi=10.1111/1751-7915.12150&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0024">
<span class="label">24.</span><cite>
Frankenfield A., Ni J., and Hao L., “Characterization of Neuronal Lysosome Interactome With Proximity Labeling Proteomics,” Journal of Visualized Experiments
184 (2022): e64132.</cite> [<a href="https://doi.org/10.3791/64132" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9997703/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35815987/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Visualized%20Experiments&amp;title=Characterization%20of%20Neuronal%20Lysosome%20Interactome%20With%20Proximity%20Labeling%20Proteomics&amp;author=A.%20Frankenfield&amp;author=J.%20Ni&amp;author=L.%20Hao&amp;volume=184&amp;publication_year=2022&amp;pages=e64132&amp;pmid=35815987&amp;doi=10.3791/64132&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0025">
<span class="label">25.</span><cite>
Choi H., Larsen B., Lin Z. Y., et al., “SAINT: Probabilistic Scoring of Affinity Purification‐Mass Spectrometry Data,” Nature Methods
8 (2011): 70–73.
</cite> [<a href="https://doi.org/10.1038/nmeth.1541" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3064265/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21131968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=SAINT:%20Probabilistic%20Scoring%20of%20Affinity%20Purification%E2%80%90Mass%20Spectrometry%20Data&amp;author=H.%20Choi&amp;author=B.%20Larsen&amp;author=Z.%20Y.%20Lin&amp;volume=8&amp;publication_year=2011&amp;pages=70-73&amp;pmid=21131968&amp;doi=10.1038/nmeth.1541&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0026">
<span class="label">26.</span><cite>
Mellacheruvu D., Wright Z., Couzens A. L., et al., “The CRAPome: A Contaminant Repository for Affinity Purification‐Mass Spectrometry Data,” Nature Methods
10 (2013): 730–736.
</cite> [<a href="https://doi.org/10.1038/nmeth.2557" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3773500/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23921808/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=The%20CRAPome:%20A%20Contaminant%20Repository%20for%20Affinity%20Purification%E2%80%90Mass%20Spectrometry%20Data&amp;author=D.%20Mellacheruvu&amp;author=Z.%20Wright&amp;author=A.%20L.%20Couzens&amp;volume=10&amp;publication_year=2013&amp;pages=730-736&amp;pmid=23921808&amp;doi=10.1038/nmeth.2557&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0027">
<span class="label">27.</span><cite>
Berg Luecke L. and Gundry R. L., “Assessment of Streptavidin Bead Binding Capacity to Improve Quality of Streptavidin‐Based Enrichment Studies,” Journal of Proteome Research
20 (2021): 1153–1164.
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.0c00772" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8116117/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33270449/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteome%20Research&amp;title=Assessment%20of%20Streptavidin%20Bead%20Binding%20Capacity%20to%20Improve%20Quality%20of%20Streptavidin%E2%80%90Based%20Enrichment%20Studies&amp;author=L.%20Berg%20Luecke&amp;author=R.%20L.%20Gundry&amp;volume=20&amp;publication_year=2021&amp;pages=1153-1164&amp;pmid=33270449&amp;doi=10.1021/acs.jproteome.0c00772&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0028">
<span class="label">28.</span><cite>
Rafiee M. R., Sigismondo G., Kalxdorf M., et al., “Protease‐Resistant Streptavidin for Interaction Proteomics,” Molecular Systems Biology
16 (2020): 1–12.</cite> [<a href="https://doi.org/10.15252/msb.20199370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7218406/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32400114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Systems%20Biology&amp;title=Protease%E2%80%90Resistant%20Streptavidin%20for%20Interaction%20Proteomics&amp;author=M.%20R.%20Rafiee&amp;author=G.%20Sigismondo&amp;author=M.%20Kalxdorf&amp;volume=16&amp;publication_year=2020&amp;pages=1-12&amp;pmid=32400114&amp;doi=10.15252/msb.20199370&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0029">
<span class="label">29.</span><cite>
Li Y., Tian C., Liu K., Zhou Y., Yang J., and Zou P., “A Clickable APEX Probe for Proximity‐Dependent Proteomic Profiling in Yeast,” Cell Chemical Biology
27 (2020): 858–865.e8.
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2020.05.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32470320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Chemical%20Biology&amp;title=A%20Clickable%20APEX%20Probe%20for%20Proximity%E2%80%90Dependent%20Proteomic%20Profiling%20in%20Yeast&amp;author=Y.%20Li&amp;author=C.%20Tian&amp;author=K.%20Liu&amp;author=Y.%20Zhou&amp;author=J.%20Yang&amp;volume=27&amp;publication_year=2020&amp;pages=858-865.e8&amp;pmid=32470320&amp;doi=10.1016/j.chembiol.2020.05.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0030">
<span class="label">30.</span><cite>
Frankenfield A. M., Ni J., Ahmed M., and Hao L., “Protein Contaminants Matter: Building Universal Protein Contaminant Libraries for DDA and DIA Proteomics,” Journal of Proteome Research
21 (2022): 2104–2113.
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.2c00145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10040255/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35793413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteome%20Research&amp;title=Protein%20Contaminants%20Matter:%20Building%20Universal%20Protein%20Contaminant%20Libraries%20for%20DDA%20and%20DIA%20Proteomics&amp;author=A.%20M.%20Frankenfield&amp;author=J.%20Ni&amp;author=M.%20Ahmed&amp;author=L.%20Hao&amp;volume=21&amp;publication_year=2022&amp;pages=2104-2113&amp;pmid=35793413&amp;doi=10.1021/acs.jproteome.2c00145&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0031">
<span class="label">31.</span><cite>
Smeriglio N., Li H., Mazli W. N. A. B., Bendel K., and Hao L., “Contaminant Spot Check and Removal Assay (ContamSPOT) for Mass Spectrometry Analysis,” Analytical Chemistry
96 (2024): 2574–2581.
</cite> [<a href="https://doi.org/10.1021/acs.analchem.3c05020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38291764/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Contaminant%20Spot%20Check%20and%20Removal%20Assay%20(ContamSPOT)%20for%20Mass%20Spectrometry%20Analysis&amp;author=N.%20Smeriglio&amp;author=H.%20Li&amp;author=W.%20N.%20A.%20B.%20Mazli&amp;author=K.%20Bendel&amp;author=L.%20Hao&amp;volume=96&amp;publication_year=2024&amp;pages=2574-2581&amp;pmid=38291764&amp;doi=10.1021/acs.analchem.3c05020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0032">
<span class="label">32.</span><cite>
Hung V., Udeshi N. D., Lam S. S., et al., “Spatially Resolved Proteomic Mapping in Living Cells With the Engineered Peroxidase APEX2,” Nature Protocols
11 (2016): 456–475.
</cite> [<a href="https://doi.org/10.1038/nprot.2016.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4863649/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26866790/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Protocols&amp;title=Spatially%20Resolved%20Proteomic%20Mapping%20in%20Living%20Cells%20With%20the%20Engineered%20Peroxidase%20APEX2&amp;author=V.%20Hung&amp;author=N.%20D.%20Udeshi&amp;author=S.%20S.%20Lam&amp;volume=11&amp;publication_year=2016&amp;pages=456-475&amp;pmid=26866790&amp;doi=10.1038/nprot.2016.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0033">
<span class="label">33.</span><cite>
St‐Germain J. R., Samavarchi Tehrani P., Wong C., Larsen B., Gingras A. C., and Raught B., “Variability in Streptavidin‐Sepharose Matrix Quality Can Significantly Affect Proximity‐Dependent Biotinylation (BioID) Data,” Journal of Proteome Research
19 (2020): 3554–3561.
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.0c00117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32628020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteome%20Research&amp;title=Variability%20in%20Streptavidin%E2%80%90Sepharose%20Matrix%20Quality%20Can%20Significantly%20Affect%20Proximity%E2%80%90Dependent%20Biotinylation%20(BioID)%20Data&amp;author=J.%20R.%20St%E2%80%90Germain&amp;author=P.%20Samavarchi%20Tehrani&amp;author=C.%20Wong&amp;author=B.%20Larsen&amp;author=A.%20C.%20Gingras&amp;volume=19&amp;publication_year=2020&amp;pages=3554-3561&amp;pmid=32628020&amp;doi=10.1021/acs.jproteome.0c00117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0034">
<span class="label">34.</span><cite>
Paek J., Kalocsay M., Staus D. P., et al., “Multidimensional Tracking of GPCR Signaling via Peroxidase‐Catalyzed Proximity Labeling,” Cell
169 (2017): 338–349.e11.
</cite> [<a href="https://doi.org/10.1016/j.cell.2017.03.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5514552/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28388415/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Multidimensional%20Tracking%20of%20GPCR%20Signaling%20via%20Peroxidase%E2%80%90Catalyzed%20Proximity%20Labeling&amp;author=J.%20Paek&amp;author=M.%20Kalocsay&amp;author=D.%20P.%20Staus&amp;volume=169&amp;publication_year=2017&amp;pages=338-349.e11&amp;pmid=28388415&amp;doi=10.1016/j.cell.2017.03.028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0035">
<span class="label">35.</span><cite>
Chung C. Y., Khurana V., Yi S., et al., “In Situ Peroxidase Labeling and Mass‐Spectrometry Connects Alpha‐Synuclein Directly to Endocytic Trafficking and mRNA Metabolism in Neurons,” Cell Systems
4 (2017): 242–250.e4.
</cite> [<a href="https://doi.org/10.1016/j.cels.2017.01.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5578869/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28131823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Systems&amp;title=In%20Situ%20Peroxidase%20Labeling%20and%20Mass%E2%80%90Spectrometry%20Connects%20Alpha%E2%80%90Synuclein%20Directly%20to%20Endocytic%20Trafficking%20and%20mRNA%20Metabolism%20in%20Neurons&amp;author=C.%20Y.%20Chung&amp;author=V.%20Khurana&amp;author=S.%20Yi&amp;volume=4&amp;publication_year=2017&amp;pages=242-250.e4&amp;pmid=28131823&amp;doi=10.1016/j.cels.2017.01.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0036">
<span class="label">36.</span><cite>
Mann M., “Functional and Quantitative Proteomics Using SILAC,” Nature Reviews. Molecular Cell Biology
7 (2006): 952–958.
</cite> [<a href="https://doi.org/10.1038/nrm2067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17139335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews.%20Molecular%20Cell%20Biology&amp;title=Functional%20and%20Quantitative%20Proteomics%20Using%20SILAC&amp;author=M.%20Mann&amp;volume=7&amp;publication_year=2006&amp;pages=952-958&amp;pmid=17139335&amp;doi=10.1038/nrm2067&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0037">
<span class="label">37.</span><cite>
Saveliev S., Bratz M., Zubarev R., Szapacs M., Budamgunta H., and Urh M., “Trypsin/Lys‐C Protease mix for Enhanced Protein Mass Spectrometry Analysis,” Nature Methods
10 (2013): i–ii.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Trypsin/Lys%E2%80%90C%20Protease%20mix%20for%20Enhanced%20Protein%20Mass%20Spectrometry%20Analysis&amp;author=S.%20Saveliev&amp;author=M.%20Bratz&amp;author=R.%20Zubarev&amp;author=M.%20Szapacs&amp;author=H.%20Budamgunta&amp;volume=10&amp;publication_year=2013&amp;pages=i-ii&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0038">
<span class="label">38.</span><cite>
Callister S. J., Barry R. C., Adkins J. N., et al., “Normalization Approaches for Removing Systematic Biases Associated With Mass Spectrometry and Label‐Free Proteomics,” Journal of Proteome Research
5 (2006): 277–286.
</cite> [<a href="https://doi.org/10.1021/pr050300l" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1992440/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16457593/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteome%20Research&amp;title=Normalization%20Approaches%20for%20Removing%20Systematic%20Biases%20Associated%20With%20Mass%20Spectrometry%20and%20Label%E2%80%90Free%20Proteomics&amp;author=S.%20J.%20Callister&amp;author=R.%20C.%20Barry&amp;author=J.%20N.%20Adkins&amp;volume=5&amp;publication_year=2006&amp;pages=277-286&amp;pmid=16457593&amp;doi=10.1021/pr050300l&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0039">
<span class="label">39.</span><cite>
Kantz E. D., Tiwari S., Watrous J. D., Cheng S., and Jain M., “Deep Neural Networks for Classification of LC‐MS Spectral Peaks,” Analytical Chemistry
91 (2019): 12407–12413.
</cite> [<a href="https://doi.org/10.1021/acs.analchem.9b02983" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7089603/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31483992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Deep%20Neural%20Networks%20for%20Classification%20of%20LC%E2%80%90MS%20Spectral%20Peaks&amp;author=E.%20D.%20Kantz&amp;author=S.%20Tiwari&amp;author=J.%20D.%20Watrous&amp;author=S.%20Cheng&amp;author=M.%20Jain&amp;volume=91&amp;publication_year=2019&amp;pages=12407-12413&amp;pmid=31483992&amp;doi=10.1021/acs.analchem.9b02983&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0040">
<span class="label">40.</span><cite>
Liu Y., Yang Y., Chen W., et al., “DeepRTAlign: Toward Accurate Retention Time Alignment for Large Cohort Mass Spectrometry Data Analysis,” Nature Communications
14 (2023): 8188.</cite> [<a href="https://doi.org/10.1038/s41467-023-43909-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10713976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38081814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=DeepRTAlign:%20Toward%20Accurate%20Retention%20Time%20Alignment%20for%20Large%20Cohort%20Mass%20Spectrometry%20Data%20Analysis&amp;author=Y.%20Liu&amp;author=Y.%20Yang&amp;author=W.%20Chen&amp;volume=14&amp;publication_year=2023&amp;pages=8188&amp;pmid=38081814&amp;doi=10.1038/s41467-023-43909-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0041">
<span class="label">41.</span><cite>
Mun D. G., Joshi N. S., Budhraja R., et al., “Automated Sample Preparation Workflow for Tandem Mass Tag‐Based Proteomics,” Journal of the American Society for Mass Spectrometry
34 (2023): 2087–2092.
</cite> [<a href="https://doi.org/10.1021/jasms.3c00095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10557128/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37657774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20American%20Society%20for%20Mass%20Spectrometry&amp;title=Automated%20Sample%20Preparation%20Workflow%20for%20Tandem%20Mass%20Tag%E2%80%90Based%20Proteomics&amp;author=D.%20G.%20Mun&amp;author=N.%20S.%20Joshi&amp;author=R.%20Budhraja&amp;volume=34&amp;publication_year=2023&amp;pages=2087-2092&amp;pmid=37657774&amp;doi=10.1021/jasms.3c00095&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0042">
<span class="label">42.</span><cite>
Müller T., Kalxdorf M., Longuespée R., Kazdal D. N., Stenzinger A., and Krijgsveld J., “Automated Sample Preparation With SP3 for Low‐Input Clinical Proteomics,” Molecular Systems Biology
16 (2020): e9111.
</cite> [<a href="https://doi.org/10.15252/msb.20199111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6966100/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32129943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Systems%20Biology&amp;title=Automated%20Sample%20Preparation%20With%20SP3%20for%20Low%E2%80%90Input%20Clinical%20Proteomics&amp;author=T.%20M%C3%BCller&amp;author=M.%20Kalxdorf&amp;author=R.%20Longuesp%C3%A9e&amp;author=D.%20N.%20Kazdal&amp;author=A.%20Stenzinger&amp;volume=16&amp;publication_year=2020&amp;pages=e9111&amp;pmid=32129943&amp;doi=10.15252/msb.20199111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0043">
<span class="label">43.</span><cite>
Muraoka S., Hirano M., Isoyama J., Ishida M., Tomonaga T., and Adachi J., “Automated Proteomics Sample Preparation of Phosphatidylserine‐Positive Extracellular Vesicles From Human Body Fluids,” ACS Omega
7 (2022): 41472–41479.
</cite> [<a href="https://doi.org/10.1021/acsomega.2c05244" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9670285/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36406491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Omega&amp;title=Automated%20Proteomics%20Sample%20Preparation%20of%20Phosphatidylserine%E2%80%90Positive%20Extracellular%20Vesicles%20From%20Human%20Body%20Fluids&amp;author=S.%20Muraoka&amp;author=M.%20Hirano&amp;author=J.%20Isoyama&amp;author=M.%20Ishida&amp;author=T.%20Tomonaga&amp;volume=7&amp;publication_year=2022&amp;pages=41472-41479&amp;pmid=36406491&amp;doi=10.1021/acsomega.2c05244&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0044">
<span class="label">44.</span><cite>
Zhong X., Li Q., Polacco B. J., et al., “A Proximity Proteomics Pipeline With Improved Reproducibility and Throughput,” Molecular Systems Biology
20 (2024): 952–971.
</cite> [<a href="https://doi.org/10.1038/s44320-024-00049-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11297269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38951684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Systems%20Biology&amp;title=A%20Proximity%20Proteomics%20Pipeline%20With%20Improved%20Reproducibility%20and%20Throughput&amp;author=X.%20Zhong&amp;author=Q.%20Li&amp;author=B.%20J.%20Polacco&amp;volume=20&amp;publication_year=2024&amp;pages=952-971&amp;pmid=38951684&amp;doi=10.1038/s44320-024-00049-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5134-bib-0045">
<span class="label">45.</span><cite>
Cirri E., Knaudt H., Di Fraia D., et al., “Optimized Automated Workflow for BioID Improves Reproducibility and Identification of Protein‐Protein Interactions,” Journal of Proteome Research
23 (2024): 4359–4368, 10.1021/acs.jproteome.4c00308.
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.4c00308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11460324/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39231529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteome%20Research&amp;title=Optimized%20Automated%20Workflow%20for%20BioID%20Improves%20Reproducibility%20and%20Identification%20of%20Protein%E2%80%90Protein%20Interactions&amp;author=E.%20Cirri&amp;author=H.%20Knaudt&amp;author=D.%20Di%20Fraia&amp;volume=23&amp;publication_year=2024&amp;pages=4359-4368&amp;pmid=39231529&amp;doi=10.1021/acs.jproteome.4c00308&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Data sharing is not applicable to this article as no datasets were generated or analyzed during the current study.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Journal of Mass Spectrometry are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/jms.5134"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/JMS-60-e5134.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (529.1 KB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11976124/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11976124/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11976124%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11976124/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11976124/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11976124/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40195276/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11976124/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40195276/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11976124/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11976124/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="6SdTq3cyF2kU5xdydFpfoDaPvoX3T5oR9OmTs86cvMuMITg9cgkxdDnkAPezugLh">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
