
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Varying phylogenetic signal in susceptibility to four bacterial pathogens across species of Drosophilidae - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="3D96E12980956EF300E12900063E2C67.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="procb">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12001086/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Proceedings of the Royal Society B: Biological Sciences">
<meta name="citation_title" content="Varying phylogenetic signal in susceptibility to four bacterial pathogens across species of Drosophilidae">
<meta name="citation_author" content="Hongbo Sun">
<meta name="citation_author_institution" content="Centre for Ecology and Conservation, Faculty of Environment, Science and Economy, University of Exeter, Cornwall, UK">
<meta name="citation_author" content="Mark A Hanson">
<meta name="citation_author_institution" content="Centre for Ecology and Conservation, Faculty of Environment, Science and Economy, University of Exeter, Cornwall, UK">
<meta name="citation_author" content="Sarah K Walsh">
<meta name="citation_author_institution" content="Centre for Ecology and Conservation, Faculty of Environment, Science and Economy, University of Exeter, Cornwall, UK">
<meta name="citation_author_institution" content="Environment and Sustainability Institute, University of Exeter, Cornwall, UK">
<meta name="citation_author" content="Ryan M Imrie">
<meta name="citation_author_institution" content="Centre for Ecology and Conservation, Faculty of Environment, Science and Economy, University of Exeter, Cornwall, UK">
<meta name="citation_author" content="Ben Raymond">
<meta name="citation_author_institution" content="Centre for Ecology and Conservation, Faculty of Environment, Science and Economy, University of Exeter, Cornwall, UK">
<meta name="citation_author" content="Ben Longdon">
<meta name="citation_author_institution" content="Centre for Ecology and Conservation, Faculty of Environment, Science and Economy, University of Exeter, Cornwall, UK">
<meta name="citation_publication_date" content="2025 Apr 16">
<meta name="citation_volume" content="292">
<meta name="citation_issue" content="2045">
<meta name="citation_firstpage" content="20242239">
<meta name="citation_doi" content="10.1098/rspb.2024.2239">
<meta name="citation_pmid" content="40237085">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12001086/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12001086/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12001086/pdf/rspb.2024.2239.pdf">
<meta name="description" content="Bacterial infections are a major threat to public health. Pathogen host shifts—where a pathogen jumps from one host species to another—are important sources of emerging infectious diseases. However, compared with viruses, we know relatively little ...">
<meta name="og:title" content="Varying phylogenetic signal in susceptibility to four bacterial pathogens across species of Drosophilidae">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Bacterial infections are a major threat to public health. Pathogen host shifts—where a pathogen jumps from one host species to another—are important sources of emerging infectious diseases. However, compared with viruses, we know relatively little ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12001086/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12001086">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1098/rspb.2024.2239"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/rspb.2024.2239.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12001086%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12001086/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12001086/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12001086/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-procb.gif" alt="Proceedings of the Royal Society B: Biological Sciences logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Proceedings of the Royal Society B: Biological Sciences" title="Link to Proceedings of the Royal Society B: Biological Sciences" shape="default" href="http://rspb.royalsocietypublishing.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Proc Biol Sci</button></div>. 2025 Apr 16;292(2045):20242239. doi: <a href="https://doi.org/10.1098/rspb.2024.2239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1098/rspb.2024.2239</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Proc%20Biol%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Proc%20Biol%20Sci%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Proc%20Biol%20Sci%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Proc%20Biol%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Varying phylogenetic signal in susceptibility to four bacterial pathogens across species of Drosophilidae</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Hongbo Sun</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Hongbo Sun</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Centre for Ecology and Conservation, Faculty of Environment, Science and Economy, University of Exeter, Cornwall, UK</div>
<div>Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Project administration, Resources, Software, Validation, Visualization, Writing – original draft, Writing – review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hongbo Sun</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hanson%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Mark A Hanson</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Mark A Hanson</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Centre for Ecology and Conservation, Faculty of Environment, Science and Economy, University of Exeter, Cornwall, UK</div>
<div>Data curation, Investigation, Methodology, Writing – original draft, Writing – review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hanson%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Mark A Hanson</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Walsh%20SK%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Sarah K Walsh</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Sarah K Walsh</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Centre for Ecology and Conservation, Faculty of Environment, Science and Economy, University of Exeter, Cornwall, UK</div>
<div class="p">
<sup><sup>2</sup></sup>Environment and Sustainability Institute, University of Exeter, Cornwall, UK</div>
<div>Formal analysis, Investigation, Visualization</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Walsh%20SK%22%5BAuthor%5D" class="usa-link"><span class="name western">Sarah K Walsh</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Imrie%20RM%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Ryan M Imrie</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Ryan M Imrie</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Centre for Ecology and Conservation, Faculty of Environment, Science and Economy, University of Exeter, Cornwall, UK</div>
<div>Formal analysis, Investigation, Visualization</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Imrie%20RM%22%5BAuthor%5D" class="usa-link"><span class="name western">Ryan M Imrie</span></a>
</div>
</div>
<sup>1,</sup><sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Raymond%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Ben Raymond</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Ben Raymond</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Centre for Ecology and Conservation, Faculty of Environment, Science and Economy, University of Exeter, Cornwall, UK</div>
<div>Conceptualization, Formal analysis, Funding acquisition, Investigation, Methodology, Resources, Supervision, Writing – original draft, Writing – review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Raymond%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Ben Raymond</span></a>
</div>
</div>
<sup>1,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Longdon%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Ben Longdon</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Ben Longdon</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Centre for Ecology and Conservation, Faculty of Environment, Science and Economy, University of Exeter, Cornwall, UK</div>
<div>Conceptualization, Formal analysis, Funding acquisition, Investigation, Methodology, Resources, Supervision, Visualization, Writing – original draft, Writing – review and editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Longdon%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Ben Longdon</span></a>
</div>
</div>
<sup>1,</sup><sup>†</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup><sup>1</sup></sup>Centre for Ecology and Conservation, Faculty of Environment, Science and Economy, University of Exeter, Cornwall, UK</div>
<div id="aff2">
<sup><sup>2</sup></sup>Environment and Sustainability Institute, University of Exeter, Cornwall, UK</div>
<div class="author-notes p">
<div class="fn" id="fn1"><p>Electronic supplementary material is available online at <a href="https://doi.org/10.6084/m9.figshare.c.7728436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.6084/m9.figshare.c.7728436</a>.</p></div>
<div class="fn" id="equal-contrib1">
<sup>
<sup>†</sup>
</sup><p class="display-inline">These authors contributed equally to the study.</p>
</div>
<div class="fn" id="pa1"><p><strong><sup>‡</sup>Present address: </strong>MRC-University of Glasgow Centre for Virus Research, Glasgow, UK</p></div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Hongbo Sun</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Project administration, Resources, Software, Validation, Visualization, Writing – original draft, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Mark A Hanson</span></strong>: <span class="role">Data curation, Investigation, Methodology, Writing – original draft, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Sarah K Walsh</span></strong>: <span class="role">Formal analysis, Investigation, Visualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Ryan M Imrie</span></strong>: <span class="role">Formal analysis, Investigation, Visualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Ben Raymond</span></strong>: <span class="role">Conceptualization, Formal analysis, Funding acquisition, Investigation, Methodology, Resources, Supervision, Writing – original draft, Writing – review and editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Ben Longdon</span></strong>: <span class="role">Conceptualization, Formal analysis, Funding acquisition, Investigation, Methodology, Resources, Supervision, Visualization, Writing – original draft, Writing – review and editing</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Sep 19; Revised 2025 Feb 11; Accepted 2025 Mar 14; Collection date 2025 Apr.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s).</div>
<p>Published by the Royal Society under the terms of the Creative Commons Attribution License <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>, which permits unrestricted use, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12001086  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40237085/" class="usa-link">40237085</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Bacterial infections are a major threat to public health. Pathogen host shifts—where a pathogen jumps from one host species to another—are important sources of emerging infectious diseases. However, compared with viruses, we know relatively little about the factors that determine whether bacteria can infect a novel host, such as how host phylogenetics constrains variation in pathogen host range and the link between host phylogeny and the infectivity and virulence of a pathogen. Here, we experimentally examined susceptibility to bacterial infections using a panel of 36 Drosophilidae species and four pathogens (<em>Providencia rettgeri</em>, <em>Pseudomonas entomophila</em>, <em>Enterococcus faecalis</em>, <em>Staphylococcus aureus</em>). The outcomes of infection differed greatly among pathogens and across host species. The host phylogeny explains a considerable amount of variation in susceptibility, with the greatest phylogenetic signal for <em>P. rettgeri</em> infection, explaining 94% of the variation in mortality. Positive correlations were observed between mortality and bacterial load for three out of the four pathogens. Correlations in susceptibility between the four pathogens were positive but largely non-significant, suggesting susceptibility is mostly pathogen-specific. These results suggest that susceptibility to bacterial pathogens may be predicted by the host phylogeny, but the effect may vary in magnitude between different bacteria.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> bacterial infection, susceptibility, phylogeny, host shift, host–microbe interaction</p></section></section><section id="s1"><h2 class="pmc_sec_title">1.  Introduction</h2>
<p>Emerging and re-emerging infectious diseases are a major threat to public health [<a href="#B1" class="usa-link" aria-describedby="B1">1</a>–<a href="#B3" class="usa-link" aria-describedby="B3">3</a>]. Multi drug-resistant bacterial infections are a major cause of mortality, responsible for millions of deaths globally [<a href="#B4" class="usa-link" aria-describedby="B4">4</a>]. Recent studies have highlighted that host shifts—where a pathogen jumps from one host species to another—are important sources of emerging diseases, not only by viruses, but also by bacterial pathogens [<a href="#B5" class="usa-link" aria-describedby="B5">5</a>,<a href="#B6" class="usa-link" aria-describedby="B6">6</a>]. There is an increasing awareness that bacterial pathogens also commonly undergo host shifts [<a href="#B6" class="usa-link" aria-describedby="B6">6</a>–<a href="#B8" class="usa-link" aria-describedby="B8">8</a>], which has been linked to their recent emergence as causative agents of disease in several host systems, including humans, livestock, wildlife and plants [<a href="#B7" class="usa-link" aria-describedby="B7">7</a>–<a href="#B9" class="usa-link" aria-describedby="B9">9</a>]. In <em>Staphylococcus aureus</em>, a human pathogen associated with several severe hospital and community-acquired infections [<a href="#B10" class="usa-link" aria-describedby="B10">10</a>], humans are the primary transmission hub for host shifts into a range of livestock species, and cows have been considered the main animal reservoirs for human epidemics [<a href="#B11" class="usa-link" aria-describedby="B11">11</a>–<a href="#B13" class="usa-link" aria-describedby="B13">13</a>]. Another extensively studied example is the jump of <em>Mycoplasma gallisepticum</em> from poultry to wild house finches [<a href="#B14" class="usa-link" aria-describedby="B14">14</a>]. Here, the <em>Mycoplasma</em> host shift has driven the evolution of resistance in the house finch hosts [<a href="#B15" class="usa-link" aria-describedby="B15">15</a>], led to changes in bacterial gene content [<a href="#B7" class="usa-link" aria-describedby="B7">7</a>] and the evolution of virulence [<a href="#B16" class="usa-link" aria-describedby="B16">16</a>]. In plants, the bacterial pathogen <em>Xylella fastidiosa</em> is found in over 600 host species [<a href="#B17" class="usa-link" aria-describedby="B17">17</a>]. A single gene loss event is associated with the host shift of <em>X. fastidiosa</em> from coffee plants to olive trees [<a href="#B18" class="usa-link" aria-describedby="B18">18</a>]. Most studies on bacterial host shifts are based on genomic data following natural host shifts, and there are limited experimental studies of bacterial host shifts that move beyond two-host systems.</p>
<p>Perhaps the best experimental studies relating to bacterial host shifts come from insects and their bacterial endosymbionts. In a study of coccinellid beetles and a male-killing isolate of <em>Spiroplasma</em>, the bacterium’s ability to distort offspring sex ratios in novel hosts was negatively correlated with the genetic distance from the bacterium’s natural host [<a href="#B19" class="usa-link" aria-describedby="B19">19</a>]. Similar patterns have been found for <em>Wolbachia</em>, a bacterial endosymbiont that infects 40–60% of all arthropods [<a href="#B20" class="usa-link" aria-describedby="B20">20</a>–<a href="#B22" class="usa-link" aria-describedby="B22">22</a>]. Artificial transfers of <em>Wolbachia</em> are most successful between closely related hosts [<a href="#B23" class="usa-link" aria-describedby="B23">23</a>,<a href="#B24" class="usa-link" aria-describedby="B24">24</a>]. This tallies with field data on <em>Wolbachia</em> distributions that found strong effects of the host phylogeny, with high rates of <em>Wolbachia</em> sharing between closely related species, and a decline in co-occurrence in different hosts with increasing phylogenetic distance between these hosts [<a href="#B9" class="usa-link" aria-describedby="B9">9</a>]. Viruses show similar trends, with phylogenetic distance and clade effects explaining large amounts of variation in susceptibility to several viruses [<a href="#B25" class="usa-link" aria-describedby="B25">25</a>,<a href="#B26" class="usa-link" aria-describedby="B26">26</a>]. When comparing different viruses from the same family, positive inter-specific correlations in susceptibility have been observed, suggesting similar mechanisms underlie differences in susceptibility observed among species [<a href="#B26" class="usa-link" aria-describedby="B26">26</a>–<a href="#B28" class="usa-link" aria-describedby="B28">28</a>]. Whether such patterns hold true for free-living bacteria, which are less tightly linked to their hosts, remains to be tested.</p>
<p>Here, we have used a large cross-infection experiment to understand how susceptibility to bacterial pathogens varies across host species. We infected a large panel of 36 species of Drosophilidae with four bacterial pathogens: two Gram-positives, <em>Staphylococcus aureus</em> and <em>Enterococcus faecalis</em>, and two Gram-negatives, <em>Providencia rettgeri</em> and <em>Pseudomonas entomophila</em>. In <em>Drosophila melanogaster</em>, the major immune response to bacterial pathogens involves the Toll and IMD pathways. The Toll pathway primarily controls resistance to fungal and Gram-positive bacterial infections, whereas the IMD pathway mediates response to Gram-negative bacterial infections [<a href="#B29" class="usa-link" aria-describedby="B29">29</a>,<a href="#B30" class="usa-link" aria-describedby="B30">30</a>]. Both of these NF-κB pathways result in the upregulation of antimicrobial peptides (AMPs) and other defence response genes [<a href="#B31" class="usa-link" aria-describedby="B31">31</a>,<a href="#B32" class="usa-link" aria-describedby="B32">32</a>]. Genes in these pathways evolve rapidly. However, it remains unclear how differences in immune genes among species contributes to differences in infection outcome [<a href="#B33" class="usa-link" aria-describedby="B33">33</a>,<a href="#B34" class="usa-link" aria-describedby="B34">34</a>]. Recent studies have demonstrated that AMPs show specific importance for certain bacterial infections and that different alleles of AMPs are linked to differences in susceptibility both within and across species [<a href="#B35" class="usa-link" aria-describedby="B35">35</a>,<a href="#B36" class="usa-link" aria-describedby="B36">36</a>].</p>
<p>The bacterial pathogens used here have been well characterized in <em>D. melanogaster</em> [<a href="#B37" class="usa-link" aria-describedby="B37">37</a>,<a href="#B38" class="usa-link" aria-describedby="B38">38</a>]. <em>E. faecalis</em> was initially derived from a human clinical isolate [<a href="#B39" class="usa-link" aria-describedby="B39">39</a>,<a href="#B40" class="usa-link" aria-describedby="B40">40</a>]. The virulence of <em>E. faecalis</em> varies with different strains [<a href="#B41" class="usa-link" aria-describedby="B41">41</a>]. The <em>E. faecalis</em> FA2-2(pAM714) used in this study produces toxins (e.g. cytolysin) that act as virulence factors to improve bacterial colonization and decrease host survival in <em>Drosophila</em> [<a href="#B40" class="usa-link" aria-describedby="B40">40</a>]. The Toll pathway, particularly the host defence peptides Bomanin and Baramicin, have been shown to play a role in response to <em>E. faecalis</em> [<a href="#B42" class="usa-link" aria-describedby="B42">42</a>–<a href="#B44" class="usa-link" aria-describedby="B44">44</a>]. <em>S. aureus</em> was initially isolated from humans and has been used in mice and <em>Drosophila</em> infection models [<a href="#B37" class="usa-link" aria-describedby="B37">37</a>,<a href="#B45" class="usa-link" aria-describedby="B45">45</a>]. <em>S. aureus</em> is a multispecies pathogen that can cause disease in humans and livestock. The dynamic gain and loss of host-specific adaptive genes, usually located on mobile genetic elements (MGEs), enable <em>S. aureus</em> to jump between different species [<a href="#B13" class="usa-link" aria-describedby="B13">13</a>,<a href="#B46" class="usa-link" aria-describedby="B46">46</a>]. In <em>D. melanogaster</em>, resistance to <em>S. aureus</em> specifically relies on the melanization response [<a href="#B47" class="usa-link" aria-describedby="B47">47</a>,<a href="#B48" class="usa-link" aria-describedby="B48">48</a>]. <em>P. entomophila</em> was isolated from a wild female <em>D. melanogaster</em> and is a versatile soil bacterium that can infect <em>D. melanogaster</em> as well as insects from different orders. Upon infection, <em>D. melanogaster</em> show differential expression of genes including AMPs, oxidative stress response genes and detoxification genes [<a href="#B49" class="usa-link" aria-describedby="B49">49</a>–<a href="#B52" class="usa-link" aria-describedby="B52">52</a>]. Genes involved in host damage prevention, signalling, renewal and regulation also affect the susceptibility of <em>D. melanogaster</em> when <em>D. melanogaster</em> is orally challenged with <em>P. entomophila</em> [<a href="#B53" class="usa-link" aria-describedby="B53">53</a>]. <em>P. entomophila</em> causes digestive obstruction and eventually leads to the host’s death by damage and perforation of the gut through the production of insecticidal protease-based toxins [<a href="#B50" class="usa-link" aria-describedby="B50">50</a>,<a href="#B52" class="usa-link" aria-describedby="B52">52</a>]. <em>P. rettgeri</em> was initially isolated from the haemolymph of wild <em>D. melanogaster</em> [<a href="#B54" class="usa-link" aria-describedby="B54">54</a>]. The IMD pathway, and particularly the AMP Diptericin has been shown to be an important part of the immune response to <em>P. rettgeri</em> [<a href="#B35" class="usa-link" aria-describedby="B35">35</a>,<a href="#B36" class="usa-link" aria-describedby="B36">36</a>,<a href="#B55" class="usa-link" aria-describedby="B55">55</a>]. A single nucleotide polymorphism at residue 69 in <em>Diptericin A</em>, ‘S69R’ of <em>DptA</em>, is associated with increased susceptibility to <em>P. rettgeri</em> bacterial infection [<a href="#B35" class="usa-link" aria-describedby="B35">35</a>,<a href="#B36" class="usa-link" aria-describedby="B36">36</a>].</p>
<p>Here, we examine the role of the host phylogeny in determining the outcomes of bacterial infections. We investigated how variation in bacterial virulence varies across the host phylogeny as well as the relationship between virulence and bacterial load for each of the four bacterial pathogens.</p></section><section id="s2"><h2 class="pmc_sec_title">2.  Methodology</h2>
<section id="s2-1"><h3 class="pmc_sec_title">(a). Bacterial strains and culture</h3>
<p>We used four bacterial pathogens in this study. Two Gram-positive bacteria, <em>Staphylococcus aureus</em> PIG1 (<em>S. aureus</em>) [<a href="#B45" class="usa-link" aria-describedby="B45">45</a>] and <em>Enterococcus faecalis</em> FA2-2/pAM714 (<em>E. faeclis</em>) [<a href="#B39" class="usa-link" aria-describedby="B39">39</a>,<a href="#B40" class="usa-link" aria-describedby="B40">40</a>], and two Gram-negative bacteria, <em>Providencia rettgeri</em> Dmel (<em>P. rettgeri</em>) [<a href="#B54" class="usa-link" aria-describedby="B54">54</a>] and <em>Pseudomonas entomophila</em> (<em>P. entomophila</em>) [<a href="#B50" class="usa-link" aria-describedby="B50">50</a>]. Bacterial cultures were initiated from frozen glycerol stocks onto LB (Lysogeny Broth) agar, with the exception of <em>P. entomophila,</em> which were plated on egg yolk agar (10 g of agar, 10 g of LB, 1 l of water and 20 ml of egg yolk (Sigma)). Colonies of <em>P. entomophila</em> with halos on egg yolk agar (indicating phospholipase activity) were used to prepare inocula. Insect inocula were cultured in LB broth, with the exception of <em>E. faecalis</em> which used BHI (brain heart infusion) broth. All bacteria were grown at a 30℃ incubator, shaking 180 rpm for liquid culture. Inocula were prepared in a three-step process in order to standardize physiology in the stationary phase. Initially, colonies were used to prepare cultures of 5 ml in 30 ml sterile universals. These were diluted in liquid media to OD<sub>600</sub> 0.2, and then 50 µl of this was inoculated into new vials. Following overnight growth, cultures were diluted with culture media to the desired OD value (OD 2.0 for <em>E. faecalis</em> and OD 1.0 for the other bacteria based on pilot mortality data).</p></section><section id="s2-2"><h3 class="pmc_sec_title">(b). Fly stocks and rearing</h3>
<p>All flies were maintained in multi-generation stock bottles (Fisher brand) at 22°C, at approximately 50% relative humidity in a 12 h light–dark cycle. Each stock bottle contained 50 ml of one of four varieties of <em>Drosophila</em> media which were chosen to optimize rearing conditions (full details in electronic supplementary material, table S1). Male flies were used to avoid any effect of sex or mating status, which has been shown to influence the susceptibility of female flies to various pathogens [<a href="#B56" class="usa-link" aria-describedby="B56">56</a>–<a href="#B59" class="usa-link" aria-describedby="B59">59</a>]. Before inoculation, 0- to 1-day-old male flies were transferred to vials containing cornmeal media. These flies were then transferred to fresh media every 3 or 4 days for a week (age 7−8 days, flip once before infection), at which point they were inoculated. Flies were kept at 22°C, 70% relative humidity in a 12 h light–dark cycle throughout the experiments. Flies were inoculated under CO<sub>2</sub> anaesthesia via septic pin prick with 10 μm diameter stainless steel needles (26002-10, Fine Science Tools, CA, USA). These needles were bent approximately 250 μm from the end to provide a depth stop and dipped in the inoculum before being pricked into the anepisternal cleft in the thorax of anaesthetized flies. Inoculation by this method bypasses the gut immune barrier but avoids differences in inoculation dose due to variations in feeding rate in oral inoculation. This method has been demonstrated to be highly repeatable (see §3) and gives a consistent initial dose (our estimate of initial dose (CFUs) introduced into individual fly for each pathogen: OD = 1 <em>P</em>. <em>rettgeri</em>: 10^(1.94 ± 0.08); OD = 1 <em>P. entomophila</em>: 10^(1.50 ± 0.14); OD = 2 <em>E. faecalis</em>: 10^(2.67 ± 0.12); OD = 1 <em>S</em>. <em>aureus</em>: 10^(2.75 ± 0.12)). All inoculations were done by a single experimenter (H.S.).</p>
<p>For the survival assay, we inoculated male flies of 36 fly species. We inoculated three replicate vials for 165 species–bacteria combinations, two replicate vials for 13 combinations and one replicate vial for two combinations. For each combination, one of 36 species was infected with either a control Ringer’s solution [<a href="#B60" class="usa-link" aria-describedby="B60">60</a>] or one of the four bacterial pathogens listed above. Vials contained a mean of 14 flies (range = 5–18 flies). Following pin prick infection, the number of dead flies in each vial was recorded daily for 14 days. Flies were tipped into new vials of food every 2 days.</p>
<p>We measured bacterial load in 18 species of fly; these species were selected based on their survival results and covered a broad range of survival values for each pathogen. We measured colony-forming units (CFUs) for individual male flies of each species (mean = 15 replicate flies, range 1−16) infected with each species–bacteria combination (there is only one replicate for <em>D. teisseri</em> in <em>P. entomophila</em> infection as most flies died due to infection before collection; there are at least 7 flies per species with a mean of 15 flies for all other treatments). CFUs were measured at 16 h post-infection. Flies were prepared and infected as above. Before homogenization, anaesthetized flies were moved to 96-well tubes containing a 4 mm steel ball and 200 µl 70% ethanol. Flies were then gently cleaned using 70% ethanol and homogenized in 200 µl sterile LB or sterile BHI (for flies infected with <em>E. faecalis</em>) in a Tissue Lyser II at 30 Hz for 3 min. Homogenates were diluted 10<sup>−1</sup> to 10<sup>−5</sup> and plated on LB agar or BHI agar in two technical replicates before being cultured overnight at 30℃. Bacterial loads were measured as CFUs per fly, following the formula: CFU per fly = (colony counts × dilution factor × homogenate volume)/plating volume.</p></section><section id="s2-3"><h3 class="pmc_sec_title">(c). Host phylogeny</h3>
<p>The method used to infer the host phylogeny has been described elsewhere [<a href="#B61" class="usa-link" aria-describedby="B61">61</a>] and details are provided in the electronic supplementary material, methods.</p></section><section id="s2-4"><h3 class="pmc_sec_title">(d). Sequencing of <em>Diptericin A</em>
</h3>
<p>To determine the <em>Diptericin A</em> allele of each species (which has been linked to susceptibility to bacterial pathogens [<a href="#B36" class="usa-link" aria-describedby="B36">36</a>]), <em>Diptericin</em> genes were sequenced across species to determine their S69R allele (see electronic supplementary material, methods).</p></section><section id="s2-5"><h3 class="pmc_sec_title">(e). Statistical analysis</h3>
<p>We used phylogenetic mixed models to look at the effects of host species relatedness on bacterial susceptibility across the 36 host species. We fitted all models using a Bayesian approach in the R package MCMCglmm v2.29 [<a href="#B62" class="usa-link" aria-describedby="B62">62</a>] in RStudio (R v4.2.2) [<a href="#B63" class="usa-link" aria-describedby="B63">63</a>]. We used a multivariate model with mortality following inoculation with each of the four bacteria or control treatments as the response variables. Mortality was calculated as each vial’s mean proportion of dead flies over 14 days. The models took the following form:</p>
<table class="disp-formula p" id="uFD1"><tr><td class="formula"><math id="um1" display="block" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mtable columnalign="right left right left right left right left right left right left" rowspacing="3pt" columnspacing="0em 2em 0em 2em 0em 2em 0em 2em 0em 2em 0em" displaystyle="true"><mlabeledtr><mtd id="mjx-eqn-2.1_3"><mtext>(2.1)</mtext></mtd><mtd><msub><mi>y</mi><mrow><mi>h</mi><mi>i</mi><mi>t</mi></mrow></msub></mtd><mtd><mo>=</mo><mtext> </mtext><msub><mi>β</mi><mrow><mn>1</mn><mo>:</mo><mi>t</mi></mrow></msub><mo>+</mo><mtext> </mtext><msub><mi>u</mi><mrow><mi>p</mi><mo>:</mo><mi>h</mi><mi>t</mi></mrow></msub><mo>+</mo><mrow><mtext> </mtext></mrow><msub><mi>u</mi><mrow><mi>s</mi><mo>:</mo><mi>h</mi><mi>t</mi></mrow></msub><mtext> </mtext><mo>+</mo><msub><mi>e</mi><mrow><mi>h</mi><mi>i</mi><mi>t</mi></mrow></msub><mo>,</mo></mtd></mlabeledtr><mlabeledtr><mtd id="mjx-eqn-2.2_3"><mtext>(2.2)</mtext></mtd><mtd><msub><mi>y</mi><mrow><mi>h</mi><mi>i</mi><mi>t</mi></mrow></msub></mtd><mtd><mo>=</mo><mtext> </mtext><msub><mi>β</mi><mrow><mn>1</mn><mo>:</mo><mi>t</mi></mrow></msub><mo>+</mo><mtext> </mtext><msub><mi>u</mi><mrow><mi>p</mi><mo>:</mo><mi>h</mi><mi>t</mi></mrow></msub><mtext> </mtext><mo>+</mo><msub><mi>e</mi><mrow><mi>h</mi><mi>i</mi><mi>t</mi></mrow></msub><mo>,</mo></mtd></mlabeledtr></mtable></mrow></mstyle></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui1"><math id="u1" display="inline" overflow="linebreak"><msub><mrow><mi>y</mi></mrow><mrow><mi></mi></mrow></msub><mi></mi></math></span> is the mortality of the <em>i</em>th biological replicate of host species <em>h</em>, for treatment <em>t</em> (each of the four bacteria and the non-infected control). <em>β</em> are the fixed effects, with <em>β</em><sub>1</sub> being the intercepts for each trait.</p>
<p>We also ran bivariate models for each of the four pathogens across the 18 host species tested to investigate how mortality was related to bacterial load. These models had the following form:</p>
<table class="disp-formula p" id="uFD2_3"><tr>
<td class="formula"><math id="um2" display="block" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><msub><mi>y</mi><mrow><mi>h</mi><mi>i</mi><mi>t</mi></mrow></msub><mo>=</mo><mtext> </mtext><msub><mi>β</mi><mrow><mn>1</mn><mo>:</mo><mi>t</mi></mrow></msub><mo>+</mo><mtext> </mtext><msub><mi>u</mi><mrow><mi>p</mi><mo>:</mo><mi>h</mi><mi>t</mi></mrow></msub><mo>+</mo><msub><mi>e</mi><mrow><mi>h</mi><mi>i</mi><mi>t</mi></mrow></msub><mo>,</mo></mrow></mstyle></math></td>
<td class="label">(2.3)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui2"><math id="u2" display="inline" overflow="linebreak"><msub><mrow><mi>y</mi></mrow><mrow><mi></mi></mrow></msub><mi></mi></math></span> is the response of the <em>i</em>th biological replicate of host species <em>h</em>, for treatment <em>t</em> (either mortality or the bacterial load (CFUs) of an individual fly infected with that pathogen). <em>U</em> are random effects for the phylogenetic species effects (<em>p</em>), non-phylogenetic species effects (<em>s</em>) and residuals (<em>e</em>). <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui3"><math id="u3" display="inline" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><msub><mi>u</mi><mrow><mi>s</mi><mo>:</mo><mi>h</mi><mi>t</mi></mrow></msub></mrow></mstyle></math></span> was removed from models (2) and (3) as the models struggled to separate the phylogenetic and non-phylogenetic effects. The random effects and the residuals are assumed to be multivariate normal with a zero mean and a covariance structure <strong><em>V<sub>p</sub></em></strong>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui4"><math id="u4" display="inline" overflow="linebreak"><mo>⊗</mo></math></span>
<strong>A</strong> for the phylogenetic effects and <strong><em>V<sub>e</sub></em></strong>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui5"><math id="u5" display="inline" overflow="linebreak"><mo>⊗</mo></math></span>
<strong>I</strong> for the residuals (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="ui6"><math id="u6" display="inline" overflow="linebreak"><mo>⊗</mo></math></span> here is the Kronecker product). <strong>A</strong> is the phylogenetic relatedness matrix, <strong>I</strong> is an identity matrix and the <strong>V</strong> are 5 × 5 (co)variance matrices describing the (co)variances between mortality when infected with each of the four bacteria or the control. In model (3), the <strong>V</strong> are 2 × 2 (co)variance matrices describing the (co)variances between mortality and bacterial load when infected with each of the four bacteria. Specifically, the matrices <strong><em>V<sub>p</sub></em></strong> and <strong><em>V<sub>s</sub></em></strong> describe the phylogenetic and nonphylogenetic between-species variances in mortality for each treatment and the covariances between them, whereas the residual covariance matrix <strong><em>V<sub>e</sub></em></strong> describes within-species variance that includes both actual within-species effects and measurement error. The off-diagonal elements in <strong><em>V<sub>e</sub></em></strong> (the covariances) are unable to be estimated because no vial has multiple measurements—so they were set to zero. Model (1) was also run with two additional fixed effects of body size and <em>Diptericin A</em> S69R site status (see electronic supplementary material, methods).</p>
<p>The MCMC chain was run for 13 million iterations with a burn-in of 3 million iterations and a thinning interval of 5000 iterations. We confirmed our experimental design and sample sizes had sufficient power to detect effects by down sampling a similar dataset [<a href="#B27" class="usa-link" aria-describedby="B27">27</a>]. The results presented were obtained using parameter expanded priors for the <strong><em>V</em></strong><sub><strong><em>p</em></strong></sub> and <strong><em>V</em></strong><sub><strong><em>s</em></strong></sub> matrices [<a href="#B64" class="usa-link" aria-describedby="B64">64</a>]. Results were tested for sensitivity to the use of different priors by being run with different prior structures. The alternative priors gave qualitatively similar results, except for the correlations where we found mean estimates were higher with the majority being significant. To determine what may be driving this we simulated response data based on mean estimates from the parameter expanded prior model using the ‘simulate.MCMCglmm’ function. We then refitted models on these data with either parameter expanded or alternative priors (inverse wishart or flat). The alternative priors had higher estimates of the correlations in the refitted models suggesting they may be inadvertently influencing the estimates. The proportion of the between-species variance that the phylogeny can explain was calculated from model (1) using the equation <strong><em>V<sub>p</sub></em></strong> /(<strong><em>V<sub>p</sub></em></strong>
<em>+</em>
<strong><em>V<sub>s</sub></em></strong>), where <strong><em>V<sub>p</sub></em></strong> and <strong><em>V<sub>s</sub></em></strong> represent the phylogenetic and species-specific components of between-species variance [<a href="#B65" class="usa-link" aria-describedby="B65">65</a>], respectively, and are equivalent to phylogenetic heritability or Pagel’s lambda [<a href="#B66" class="usa-link" aria-describedby="B66">66</a>,<a href="#B67" class="usa-link" aria-describedby="B67">67</a>]. The repeatability of measurements was calculated from model (2) as <strong><em>V<sub>p</sub></em></strong>/(<strong><em>V<sub>p</sub></em></strong>
<em>+</em>
<strong><em>V<sub>e</sub></em></strong>), where <strong><em>V<sub>e</sub></em></strong> is the residual variance of the model [<a href="#B68" class="usa-link" aria-describedby="B68">68</a>]. Interspecific correlations were calculated from models (2) and (3) <strong><em>V<sub>p</sub></em></strong> matrix as <strong><em>cov<sub>x,y</sub></em></strong>/<em>√</em>(<strong><em>var<sub>x</sub></em></strong>
<em>×</em>
<strong><em>var<sub>y</sub></em></strong>). Parameter estimates reported are means of the posterior density, and 95% credible intervals (CIs) were taken to be the 95% highest posterior density intervals.</p>
<p>We ran additional models using Abbott’s correction on data [<a href="#B69" class="usa-link" aria-describedby="B69">69</a>,<a href="#B70" class="usa-link" aria-describedby="B70">70</a>], and a binomial model on day 14 mortality (see electronic supplementary material, methods). We confirmed our results were robust to these different analyses (electronic supplementary material, tables S5, S6 and figure S3) and so results presented are the main analysis described above.</p></section></section><section id="s3"><h2 class="pmc_sec_title">3.  Results</h2>
<section id="s3-1"><h3 class="pmc_sec_title">(a). Patterns of susceptibility to bacterial pathogens across host species</h3>
<p>To investigate how susceptibility to bacterial pathogens varies across host species, we measured survival in 7402 flies from 36 species of Drosophilidae inoculated with one of four bacterial pathogens or a control. We found large differences in the mean virulence of the bacteria; the mean mortality was 70% for <em>P. entomophila</em> infection, 46% for <em>P. rettgeri</em>, 39% for <em>S. aureus</em> and 36% for <em>E. faecalis</em> (compared with only 4% in control inoculated flies). There were dramatic differences in the survival of the different species with the majority of individuals of certain species dying, while others showed little mortality when infected with the same pathogen (<a href="#F1" class="usa-link">figure 1</a>). For example, just over half of the species showed rapid mortality following <em>P. entomophila</em> infection, with over 50% of flies dying within 1 day post-infection in 19 out of 36 fly species (<a href="#F1" class="usa-link">figure 1</a>). Conversely, survival remained high for most species infected with <em>E. faecalis</em>, with the exceptions of <em>D. yakuba</em> and <em>D. sucinea</em> where &gt;85% of flies had died by the third day post-infection (<a href="#F1" class="usa-link">figure 1</a>).</p>
<figure class="fig xbox font-sm" id="F1" title="Mortality and bacterial load to four bacterial pathogens across 18 fly species."><h4 class="obj_head">Figure 1. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12001086_rspb.2024.2239.f001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d71f/12001086/8e0b2d274211/rspb.2024.2239.f001.jpg" loading="lazy" height="725" width="747" alt="Mortality and bacterial load to four bacterial pathogens across 18 fly species."></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Survival of 36 species of Drosophilidae after infection with four bacterial pathogens. Each point represents the mean survival of three replicates; colours represent the inoculation treatment.</p></figcaption></figure></section><section id="s3-2"><h3 class="pmc_sec_title">(b). The host phylogeny explains varying amounts of variation in mortality</h3>
<p>Different clades of hosts show different mortality levels with each bacterium, with hosts with high and low mortality clustering together on the host phylogeny (<a href="#F2" class="usa-link">figure 2</a>). For example, a clade within the subgenus <em>Drosophila</em> (delineated by <em>D. micromelanica</em> down to <em>D. flavomontana</em> in <a href="#F2" class="usa-link">figure 2</a>) shows low mortality when infected with <em>P. rettgeri</em> compared with the other major host clades. We used a phylogenetic mixed model to determine the proportion of variation in mortality explained by the host phylogeny for each pathogen (<a href="#T1" class="usa-link">table 1</a>). The estimates vary among the four pathogens, with the host phylogeny explaining large amounts of variation for three out of four bacterial infections. We find that almost all of the variation in mortality is explained by the phylogeny following <em>P. rettgeri</em> infections (94%).</p>
<figure class="fig xbox font-sm" id="F2" title="Survival of 36 species of Drosophilidae after infection with four bacterial pathogens."><h4 class="obj_head">Figure 2. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12001086_rspb.2024.2239.f002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d71f/12001086/c2e5cacef18a/rspb.2024.2239.f002.jpg" loading="lazy" height="702" width="766" alt="Survival of 36 species of Drosophilidae after infection with four bacterial pathogens."></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Susceptibility to four bacterial pathogens varies across 36 Drosophilidae species. Bar length and colour represent mean mortality change across pathogen and fly species with error bars representing standard errors. The phylogeny of Drosophilidae host species is presented on the left, with the scale bar representing the number of nucleotide substitutions per site. Names of host species are given on the right; the column label shows different treatments (bacterial or control inoculation).</p></figcaption></figure><section class="tw xbox font-sm" id="T1"><h4 class="obj_head">Table 1. </h4>
<div class="caption p"><p>Estimates of the mean proportion of variation explained by the host phylogeny (taken from model (1)), and the repeatability of estimates (taken from model (2)).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="left" valign="top" rowspan="1" colspan="1">
<p>pathogen</p>
</th>
<th align="left" valign="top" rowspan="1" colspan="1">
<p>variance explained by phylogeny</p>
</th>
<th align="left" valign="top" rowspan="1" colspan="1">
<p>repeatability</p>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<em>P. rettgeri</em>
</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>0.94 (95% CI: 0.81, 1.00)</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>0.84 (95% CI: 0.75,0.93)</p>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<em>P. entomophila</em>
</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>0.36 (95% CI: 0.00, 0.74)</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>0.70 (95% CI: 0.48,0.89)</p>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<em>E. faecalis</em>
</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>0.14 (95% CI: 0.00, 0.50)</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>0.90 (95% CI: 0.80,0.97)</p>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<em>S. aureus</em>
</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>0.64 (95% CI: 0.24, 0.94)</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>0.80 (95% CI: 0.63,0.92)</p>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>The phylogeny explains a substantial proportion of the variation in mortality for <em>S. aureus</em> (64%) but less for <em>P. entomophila</em> (36%) and <em>E. faecalis</em> (14%). However, these have broad 95% CIs around estimates of the posterior mean, which are pushed up against zero for <em>P. entomophila</em> and <em>E. faecalis</em>. If we compare the phylogenetic signal in mortality for each of the pathogens we find that the phylogeny explains a greater amount of variation in mortality to <em>P. rettgeri</em> infection than for the other pathogens (although the effect is marginal for <em>P. rettgeri–S. aureus</em>), and <em>S. aureus</em> has greater phylogenetic signal than <em>E. faecalis</em> (electronic supplementary material, table S2). Measurements were highly repeatable (<a href="#T1" class="usa-link">table 1</a>) with a large proportion of the variance being explained by the inter-specific phylogenetic component, with little within-species variation or measurement error (repeatability = <em>V<sub>p</sub></em>/(<em>V<sub>p </sub>+ V<sub>e</sub></em>)).</p></section><section id="s3-3"><h3 class="pmc_sec_title">(c). Patterns of susceptibility are largely distinct for each bacterium</h3>
<p>In <em>D. melanogaster</em>, the host immune response differs markedly between infections by Gram-positive and Gram-negative bacteria [<a href="#B71" class="usa-link" aria-describedby="B71">71</a>,<a href="#B72" class="usa-link" aria-describedby="B72">72</a>]. If mechanisms of host defence depend on the induced immune response, we might expect positive correlations in host susceptibility among species for bacteria of the same Gram type. We found the correlations between bacteria all had positive estimates (<a href="#T2" class="usa-link">table 2</a>), although the 95% CIs overlapped zero for all except for the correlations between <em>S. aureus–E. faecalis</em> and <em>E. faecalis–P. rettgeri</em>. The correlation coefficient was less than 0.4 for all correlations bar the estimate between the two Gram-positive bacteria <em>S. aureus</em> and <em>E. faecalis</em> where <em>r</em> = 0.66. Overall, this suggests that although hosts may show similar trends in susceptibility to all the bacteria tested (correlations are largely positive), host susceptibility generally differs by pathogen, with shifts in the rank order of species susceptibilities to the different pathogens (i.e. as correlations are all less than one, so species do not show overall high or low susceptibility to bacteria <em>per se</em>, but rather pathogen-specific susceptibilities).</p>
<section class="tw xbox font-sm" id="T2"><h4 class="obj_head">Table 2. </h4>
<div class="caption p"><p>Correlations in mortality between different pathogen and control treatments. Numbers represent the mean estimates of correlation coefficients from model (2) with 95% credible intervals shown in parentheses. Significant correlation coefficients (with 95% CI that do not span zero) are highlighted in bold.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="left" valign="top" rowspan="1" colspan="1">
<p>bacteria</p>
</th>
<th align="left" valign="top" rowspan="1" colspan="1">
<p>inter-specific correlation <em>r</em></p>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<em>S. aureus–E. faecalis</em>
</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p><strong>0.66 (95% CI: 0.37, 0.90</strong>)</p>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<em>S. aureus–P. rettgeri</em>
</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>0.35 (95% CI: −0.04, 0.70)</p>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<em>S. aureus–P. entomophila</em>
</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>0.34 (95% CI: −0.13, 0.73)</p>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<em>E. faecalis–P. rettgeri</em>
</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p><strong>0.39 (95% CI: 0.02, 0.72</strong>)</p>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<em>E. faecalis–P. entomophila</em>
</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>0.31 (95% CI: −0.19, 0.74)</p>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<em>P. rettgeri–P. entomophila</em>
</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>0.37 (95% CI: −0.05, 0.75)</p>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p><em>P. rettgeri–</em>control</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>−0.06 (95% CI: −0.65, 0.61)</p>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p><em>P. entomophila–</em>control</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>−0.04 (95% CI: −0.65, 0.66)</p>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p><em>E. faecalis–</em>control</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>−0.15 (95% CI: −0.85, 0.66)</p>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p><em>S. aureus–</em>control</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>−0.12 (95% CI: −0.80, 0.64)</p>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="s3-4"><h3 class="pmc_sec_title">(d). Mortality correlates with bacterial load in most cases</h3>
<p>To test whether increased mortality is due to higher bacterial loads, we selected 18 fly species that varied in their mortality and measured their bacterial load to each pathogen at 16 h post-infection. We found that 24% of the flies infected by <em>P. entomophila</em> had died by 16 h post-infection (&lt;2% of flies infected with each of the other bacteria died by this time point). We excluded dead flies from the analysis below to avoid confounding bacterial growth rates, as these flies may have died prematurely from injury, and bacterial growth rates may have developed differently in carcasses or flies failing to mount a full immune response. However, we note that the results were robust if we included the bacterial load measured in corpses. We quantified bacterial load in 1328 individual flies (<a href="#F3" class="usa-link">figure 3</a>) and ran models combining this with the mortality data above for these 18 fly species.</p>
<figure class="fig xbox font-sm" id="F3" title="Susceptibility to four bacterial pathogens varies across 36 Drosophilidae species"><h4 class="obj_head">Figure 3. </h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12001086_rspb.2024.2239.f003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d71f/12001086/219e839d635b/rspb.2024.2239.f003.jpg" loading="lazy" height="656" width="750" alt="Susceptibility to four bacterial pathogens varies across 36 Drosophilidae species"></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Mortality and bacterial load to four bacterial pathogens across 18 fly species. Bar length and colour represent mean mortality (left) or bacterial load (right) across 18 fly species bearing either <em>P. rettgeri</em> (A), <em>P. entomophila</em> (B), <em>E. faecalis</em> (C) or <em>S. aureus</em> (D) infection. Bacterial loads are data point at 16 hours post infection, with the error bars representing standard errors of a mean of 15 replicate flies for each species. The phylogeny of Drosophilidae host species is presented on the left, with the scale bar representing the number of nucleotide substitutions per site. Names of host species are presented on the right.</p></figcaption></figure><p>Mortality estimates from this model using a subset of the data from the survival experiment (<a href="#T3" class="usa-link">table 3</a>) are consistent with observations in survival experiments (<a href="#F1" class="usa-link">figure 1</a> and <a href="#F2" class="usa-link">figure 2</a> ). Mean bacterial loads are relatively consistent across the four pathogens (<a href="#T3" class="usa-link">table 3</a>). We found strong positive correlations between mortality and bacterial load for <em>P. rettgeri</em> and <em>P. entomophila</em> and a slightly lower positive correlation for <em>S. aureus</em> with 95% CIs that just overlap zero (<a href="#T4" class="usa-link">table 4</a>; electronic supplementary material, figure S1). However, we found no evidence of a significant positive correlation between mortality and bacterial load for <em>E. faecalis</em> (<a href="#T4" class="usa-link">table 4</a>; electronic supplementary material, figure S1). However, the 95% CIs around estimates were broad so there was no clear indication that bacteria were behaving differently.</p>
<section class="tw xbox font-sm" id="T3"><h4 class="obj_head">Table 3. </h4>
<div class="caption p"><p>Estimations of mortality and bacterial load for each pathogen. Numbers represent the mean estimates of mortality (mean proportion dead) and bacterial load (CFU per fly on a log<sub>10</sub> scale) for each pathogen from model (3) with 95% credible intervals shown in parentheses.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="left" valign="top" rowspan="1" colspan="1">
<p>pathogen</p>
</th>
<th align="left" valign="top" rowspan="1" colspan="1">
<p>bacterial load</p>
</th>
<th align="left" valign="top" rowspan="1" colspan="1">
<p>mortality</p>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<em>P. rettgeri</em>
</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>4.63 (95% CI: 3.67, 5.50)</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>0.46 (95% CI: 0.17, 0.76)</p>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<em>P. entomophila</em>
</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>3.78 (95% CI: 2.97, 4.46)</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>0.62 (95% CI: 0.33, 0.90)</p>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<em>E. faecalis</em>
</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>3.36 (95% CI: 2.80, 3.92)</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>0.37 (95% CI: −0.02, 0.74)</p>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<em>S. aureus</em>
</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>3.12 (95% CI: 2.32, 3.85)</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>0.36 (95% CI: 0.06, 0.68)</p>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="T4"><h4 class="obj_head">Table 4. </h4>
<div class="caption p"><p>Correlations between mortality and bacterial load for each pathogen. Numbers represent the mean estimates of correlation coefficients from model (3) with 95% credible intervals shown in parentheses. Significant correlation coefficients (with 95% CIs that do not span zero) are highlighted in bold.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="left" valign="top" rowspan="1" colspan="1">
<p>pathogen</p>
</th>
<th align="left" valign="top" rowspan="1" colspan="1">
<p>inter-specific correlation <em>r</em></p>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<em>P. rettgeri</em>
</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<strong>0.59 (95% CI: 0.16, 0.94)</strong>
</p>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<em>P. entomophila</em>
</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<strong>0.54 (95% CI: 0.10, 0.90)</strong>
</p>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<em>E. faecalis</em>
</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>0.22 (95% CI: −0.43, 0.74)</p>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>
<em>S. aureus</em>
</p>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<p>0.48 (95% CI: −0.03, 0.93)</p>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>We found some evidence of microbiome proliferation in pathogen-infected insects when measuring CFU in four species (<em>D. americana</em>, <em>S. lebanonensis</em>, <em>D. putrida</em> and <em>S. pattersoni</em>). We note there was an elevated count of likely microbiome-specific colonies in these species based on colony morphology and growth rate compared with colonies of pathogenic bacteria on LB/BHI agar. We included these colonies in our final bacterial load counts to avoid subjective calls on microbiome or pathogen colony identification, but will note that excluding these species from our analysis did not alter the results qualitatively.</p></section><section id="s3-5"><h3 class="pmc_sec_title">(e). Testing for effect of <em>Diptericin A</em> allelic variation and body size</h3>
<p>We observed an exceptionally high proportion of variance explained by host phylogeny for <em>P. rettgeri</em> (<a href="#T1" class="usa-link">table 1</a>). This suggests some phylogenetically inherited factor, or factors, strongly determines susceptibility to this pathogen. Previous studies linked the gene <em>Diptericin A</em> (<em>DptA</em>) to defence against <em>P. rettgeri</em>, including resistant or susceptible alleles at a polymorphic residue both within and across species [<a href="#B36" class="usa-link" aria-describedby="B36">36</a>,<a href="#B55" class="usa-link" aria-describedby="B55">55</a>].</p>
<p>We successfully amplified and sequenced <em>DptA</em> from 29 species and annotated the allele status at a previously identified polymorphic residue (residue 69 of <em>D. melanogaster DptA</em>). Four residues were found in our panel at this site (electronic supplementary material, figure S2). These were dominated by the resistant allele encoding serine (S) in 75% of species, and three alternate alleles each found in only 2−3 species, including a known susceptible allele encoding arginine (R), and alleles encoding either glutamine (Q) or asparagine (N), preliminarily associated with resistance and susceptibility respectively [<a href="#B36" class="usa-link" aria-describedby="B36">36</a>]. Given the limited replication of each individual allele, we fitted a fixed effect of <em>DptA</em> residue 69 by pathogen with species being either resistant (S and Q) or susceptible (R and N). We found no significant effect of the susceptible <em>DptA</em> alleles for any of the four pathogens (electronic supplementary material, table S4) but note that there was limited power to detect an effect with only five species having susceptible alleles. There was little change in the variance explained by the phylogeny when the <em>DptA</em> allele status was included as a fixed effect (electronic supplementary material, table S4) indicating this allele does not readily explain the phylogenetic patterns observed.</p>
<p>Wing size in <em>Drosophila</em> is a proxy for body size [<a href="#B73" class="usa-link" aria-describedby="B73">73</a>], and is known to be phylogenetically dependent [<a href="#B74" class="usa-link" aria-describedby="B74">74</a>]. We included wing size as a fixed effect in models to see if wing size explained variation in host susceptibility. We found no effect of wing size on susceptibility for any pathogen (mean and 95% CIs for each pathogen: <em>P. rettgeri</em>: −0.34 (95% CI:−0.75, 0.02), <em>P. entomophila</em>: −0.02 (95% CI:−0.54, 0.45), <em>E. faecalis</em>: 0.01 (95% CI:−0.37, 0.37), <em>S. aureus</em>: 0.03 (-0.37, 0.42)).</p></section></section><section id="s4"><h2 class="pmc_sec_title">4.  Discussion</h2>
<p>We examined how susceptibility to bacterial pathogens varies across the host phylogeny, using a panel of 36 Drosophilidae species and four bacterial pathogens. Using phylogenetic mixed models we found the host phylogeny can explain a large proportion of variation in susceptibility, but differs in the extent of variation explained between pathogens. Distinct infection patterns were seen between the different pathogens, with all correlations in susceptibility between the pathogens being less than one, suggesting susceptibility generally varies between bacterial pathogens. However, despite being largely insignificant, all correlations had positive estimates, and the strongest correlation was seen between the two Gram-positive bacteria (<a href="#T2" class="usa-link">table 2</a>) suggesting that differences in susceptibility may have shared underlying mechanisms. Positive correlations between mortality and bacterial load for <em>P. rettgeri</em>, <em>P. entomophila</em> and <em>S. aureus</em> suggest virulence is a direct result of bacterial accumulation in the host. The lack of correlation between virulence and bacterial load for <em>E. faecalis</em> may have a biological underpinning, such as the production of toxins, or inhibited replication capabilities of the human <em>E. faecalis</em> strain we used, which may not be well adapted to invertebrate hosts [<a href="#B41" class="usa-link" aria-describedby="B41">41</a>]; alternatively this may be due to limited statistical power. These results (electronic supplementary material, figure S1) also demonstrate that different species can have differences in mortality with similar bacterial loads. As such, examination of whether potential tolerance mechanisms underly differences in susceptibility [<a href="#B75" class="usa-link" aria-describedby="B75">75</a>] could be an interesting follow-up to this work.</p>
<p>Our survival results find that a substantial amount of variation in susceptibility can be explained by the host phylogeny, with the largest effect for <em>P. rettgeri,</em> where almost all of the variation in susceptibility is explained by the host phylogeny. One potential explanation is that susceptibility is linked to variation in immune genes with certain host clades having lost or gained different aspects of the immune system. Previous studies have shown that a specific AMP, <em>Diptericin</em>, plays an essential role in defence against <em>P. rettgeri</em> infection. Flies encoding arginine (R) at the <em>Diptericin</em> S69R polymorphic site are more susceptible than flies encoding serine (S) [<a href="#B36" class="usa-link" aria-describedby="B36">36</a>,<a href="#B55" class="usa-link" aria-describedby="B55">55</a>]. Due to the limited statistical replication of the different <em>Diptericin</em> alleles (75% of the 29 species with allele data had the ‘S’ resistance allele, with each of the other three alleles only replicated in two or three species) we had little power to detect effects here. However, there is also substantial variation in susceptibility just among species with the ‘S’ allele, suggesting this <em>Diptericin</em> allele is not the sole driver of the patterns observed here. While we did not find an effect of the <em>Diptericin</em> S69R allele, we cannot rule out a role for <em>Diptericin</em> in explaining variation in host susceptibility, although other mechanisms may well also play a role in explaining among-species differences in susceptibility. Further study could consider a broader range of species encoding different variants at the S69R site, variation elsewhere in the <em>Diptericin</em> gene, or differences in induction of <em>Diptericin</em> and other AMPs across species given that rapid induction is important for host survival [<a href="#B36" class="usa-link" aria-describedby="B36">36</a>,<a href="#B37" class="usa-link" aria-describedby="B37">37</a>,<a href="#B55" class="usa-link" aria-describedby="B55">55</a>].</p>
<p>We found that among species correlations in susceptibility between bacteria all had positive estimates. The strongest significant positive correlation was observed between the two Gram-positive bacteria: <em>S. aureus</em> and <em>E. faecalis</em>. One explanation for this could be differences in immune defence across the host phylogeny. In <em>D. melanogaster</em> Gram-positive bacteria are largely targeted by the Toll pathway [<a href="#B31" class="usa-link" aria-describedby="B31">31</a>,<a href="#B32" class="usa-link" aria-describedby="B32">32</a>,<a href="#B76" class="usa-link" aria-describedby="B76">76</a>,<a href="#B77" class="usa-link" aria-describedby="B77">77</a>]. Toll pathway members are highly conserved across arthropods and many Toll pathway genes have large copy number variations across species, which may be linked to their functional diversity [<a href="#B33" class="usa-link" aria-describedby="B33">33</a>,<a href="#B78" class="usa-link" aria-describedby="B78">78</a>]. We also found a significant positive correlation across Gram types between <em>E. faecalis</em> and <em>P. rettgeri</em>. Several studies have demonstrated there is cross-talk between the Toll and IMD pathways, which may play a part in explaining the shared susceptibility [<a href="#B79" class="usa-link" aria-describedby="B79">79</a>–<a href="#B81" class="usa-link" aria-describedby="B81">81</a>]. However, it is also possible that the positive correlations are driven by other factors, for example, the suitability of the host physiological environment for bacterial proliferation.</p>
<p>We found the lowest proportion of variance explained by the phylogeny for <em>E. faecalis</em>, for which we also found no correlation between bacterial load and host mortality. Susceptibility to <em>E. faecalis</em> has been shown to involve the action of Bomanin and Baramicin host defence peptides, whose mechanism remains unclear, but could contribute to promoting tolerance to infection rather than pathogen suppression [<a href="#B42" class="usa-link" aria-describedby="B42">42</a>,<a href="#B43" class="usa-link" aria-describedby="B43">43</a>,<a href="#B82" class="usa-link" aria-describedby="B82">82</a>]. A previous genome-wide association study in <em>D. melanogaster</em> found variation in susceptibility to <em>E. faecalis</em> was due to genes not classically associated with immunity [<a href="#B44" class="usa-link" aria-describedby="B44">44</a>], suggesting factors unrelated to bacterial suppression may be important in surviving infection by this pathogen. Further, a study of immune priming showed that flies primed with a low dose of <em>E. faecalis</em> were better able to survive subsequent infection by <em>E. faecalis</em>, but this was not related to bacterial clearance [<a href="#B83" class="usa-link" aria-describedby="B83">83</a>]. <em>E. faecalis</em> virulence is known to involve the action of toxins, including cytolysin and enterocin V, and induced immune responses including AMPs and iron sequestration have minimal impact on <em>E. faecalis</em> virulence [<a href="#B55" class="usa-link" aria-describedby="B55">55</a>,<a href="#B84" class="usa-link" aria-describedby="B84">84</a>,<a href="#B85" class="usa-link" aria-describedby="B85">85</a>]. The disconnect between mortality and replication could certainly be explained by toxin-based virulence if low doses of toxins are sufficient to cause mortality as in other Gram-positive insect pathogens [<a href="#B86" class="usa-link" aria-describedby="B86">86</a>]; for example, cytolysin, is haemolytic in animal cells in the nanomolar range. Differences in the host response to bacterial toxins such as presence of Toll pathway effector Baramicin A may explain the phylogenetic patterns of susceptibility to <em>E. faecalis</em> [<a href="#B43" class="usa-link" aria-describedby="B43">43</a>].</p>
<p>We found evidence of microbiome growth in four species (<em>D. americana</em>, <em>S. lebanonensis</em>, <em>D. putrida</em> and <em>S. pattersoni</em>) when measuring bacterial loads in homogenized pathogen-infected insects. Anecdotally, these species exhibited a relatively high survival to most of the bacteria (<a href="#F3" class="usa-link">figure 3</a>). The microbiome can play an important role in priming the immune response and defending against bacterial infections [<a href="#B87" class="usa-link" aria-describedby="B87">87</a>,<a href="#B88" class="usa-link" aria-describedby="B88">88</a>]. Gut probiotics such as <em>Lactobacillus</em> have been demonstrated to reduce fly mortality caused by pathogenic bacteria [<a href="#B89" class="usa-link" aria-describedby="B89">89</a>]. It has been reported that there is limited variation in the microbiome across species of Drosophilidae in laboratory culture [<a href="#B90" class="usa-link" aria-describedby="B90">90</a>,<a href="#B91" class="usa-link" aria-describedby="B91">91</a>]. As such, a more comprehensive study of how the microbiome varies across these host species, to what extent the microbiome is related to host phylogeny, and whether this relates to differences in susceptibility is warranted.</p>
<p>Although we have studied a relatively large number of host species, only four bacterial taxa were used, limiting our ability to draw broad conclusions on differences between bacterial taxa (for example differences between Gram type). Practical constraints resulted in fewer species being used to estimate the bacterial load data, limiting our ability to assess phylogenetic signal for this trait, or explore the role of tolerance in determining the patterns we observe among host species. In addition, further work is needed to understand the mechanisms underlying the differences in susceptibility among species, and whether the same or different factors affect susceptibility in different host clades.</p>
<p>In conclusion, we have found that the host phylogeny can explain varying amounts of variation in susceptibility depending on the pathogen. These varying phylogenetic effects (or lack thereof) have important consequences for understanding susceptibility across host species and its impact on pathogen host shifts [<a href="#B26" class="usa-link" aria-describedby="B26">26</a>,<a href="#B61" class="usa-link" aria-describedby="B61">61</a>], which is helpful for our understanding of overcoming increasing bacterial infection issues in public health. For some pathogens, certain clades of related hosts will be particularly susceptible, meaning pathogens will be more likely to jump into those species, and they could also act as disease reservoirs [<a href="#B92" class="usa-link" aria-describedby="B92">92</a>–<a href="#B95" class="usa-link" aria-describedby="B95">95</a>]. However, patterns may be less clear for other pathogens, with relatedness not being a strong predictor of susceptibility across species. As such, caution should be used when estimating the host range of novel pathogens based on susceptibility in closely related host species, without suitable information on patterns of susceptibility across host species for the bacterial taxa in question.</p></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>Bacterial strains were kindly provided by Brian Lazzaro and Ashley Frank. Thanks to Megan Wallace and Michael Jardine for their assistance in the code writing and experimental process for this study. Thanks to editor and two anonymous reviewers for the thoughtful and constructive comments. For the purpose of Open Access, the author has applied a CC BY public copyright licence to any Author Accepted Manuscript version arising from this submission.</p></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Hongbo Sun, Email: hs766@exeter.ac.uk; ssunhongbo9526@126.com.</p>
<p>Mark A. Hanson, Email: m.hanson@exeter.ac.uk.</p>
<p>Sarah K. Walsh, Email: Sarah.Walsh.2@glasgow.ac.uk.</p>
<p>Ryan M. Imrie, Email: Ryan.Imrie@glasgow.ac.uk.</p>
<p>Ben Raymond, Email: B.Raymond@exeter.ac.uk.</p>
<p>Ben Longdon, Email: b.longdon2@exeter.ac.uk.</p></section><section id="s5"><h2 class="pmc_sec_title">Ethics</h2>
<p>This work was assessed by the University of Exeter ethics committee (application no. 523751).</p></section><section id="s6"><h2 class="pmc_sec_title">Data accessibility</h2>
<p>All data files and R scripts for analysis and visualization used in this study are available in an online repository [<a href="#B96" class="usa-link" aria-describedby="B96">96</a>].</p>
<p>Supplementary material is available online [<a href="#B97" class="usa-link" aria-describedby="B97">97</a>].</p></section><section id="s7"><h2 class="pmc_sec_title">Declaration of AI use</h2>
<p>We have not used AI-assisted technologies in creating this article.</p></section><section id="s8"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>H.S.: conceptualization, data curation, formal analysis, investigation, methodology, project administration, resources, software, validation, visualization, writing—original draft, writing—review and editing; M.A.H.: data curation, investigation, methodology, writing—original draft, writing—review and editing; S.K.W.: formal analysis, investigation, visualization; R.M.I.: formal analysis, investigation, visualization; B.R.: conceptualization, formal analysis, funding acquisition, investigation, methodology, resources, supervision, writing—original draft, writing—review and editing; B.L.: conceptualization, formal analysis, funding acquisition, investigation, methodology, resources, supervision, visualization, writing—original draft, writing—review and editing.</p>
<p>All authors gave final approval for publication and agreed to be held accountable for the work performed therein.</p></section><section id="s9"><h2 class="pmc_sec_title">Conflict of interest declaration</h2>
<p>We declare we have no competing interests.</p></section><section id="s10"><h2 class="pmc_sec_title">Funding</h2>
<p>This work has been supported by the following funding resources: H.S. is supported by China Scholarship Council and University of Exeter PhD Scholarships (https://www.exeter.ac.uk/study/pg-research/csc-scholarships/). B.L. and R.M.I. are supported by a Sir Henry Dale Fellowship jointly funded by the Wellcome Trust and the Royal Society (grant no. 109356/Z/15/Z; https://wellcome.ac.uk/funding/sir henry-dale-fellowships). B.R. is supported by NERC NE/V012053/1 and BBSRC BB/S002928/1. S.K.W. was supported by a studentship funded by the Biotechnology and Biological Sciences Research Council (BBSRC) South West Biosciences Doctoral Training Partnership (BB/M009122/1). M.A.H. is supported by Swiss National Science Foundation (SNSF) grant P500PB_211082 and a Wellcome Trust ECA Grant (no. 227559/Z/23/Z).</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1">
<span class="label">1.</span><cite>
Bonneaud C, Longdon B. 2020. 
Emerging pathogen evolution. EMBO Rep.
21, e51374. ( 10.15252/embr.202051374)
</cite> [<a href="https://doi.org/10.15252/embr.202051374" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7507376/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32864788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Rep.&amp;title=Emerging%20pathogen%20evolution&amp;volume=21&amp;publication_year=2020&amp;pages=e51374&amp;pmid=32864788&amp;doi=10.15252/embr.202051374&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B2">
<span class="label">2.</span><cite>
Petrosillo N. 2019. 
Emerging infections and future threats. Erciyes Med. J. ( 10.14744/etd.2019.57805)</cite> [<a href="https://doi.org/10.14744/etd.2019.57805" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Erciyes%20Med.%20J.&amp;title=Emerging%20infections%20and%20future%20threats&amp;publication_year=2019&amp;doi=10.14744/etd.2019.57805&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B3">
<span class="label">3.</span><cite>
Watkins K. 2018. 
Emerging infectious diseases: a review. Curr. Emerg. Hosp. Med. Rep.
6, 86–93. ( 10.1007/s40138-018-0162-9)
</cite> [<a href="https://doi.org/10.1007/s40138-018-0162-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7100414/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32226656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Emerg.%20Hosp.%20Med.%20Rep.&amp;title=Emerging%20infectious%20diseases:%20a%20review&amp;volume=6&amp;publication_year=2018&amp;pages=86-93&amp;pmid=32226656&amp;doi=10.1007/s40138-018-0162-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B4">
<span class="label">4.</span><cite>
Murray CJL, et al. 2022. 
Global burden of bacterial antimicrobial resistance in 2019: a systematic analysis. Lancet
399, 629–655. ( 10.1016/s0140-6736(21)02724-0)
</cite> [<a href="https://doi.org/10.1016/s0140-6736(21)02724-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8841637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35065702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Global%20burden%20of%20bacterial%20antimicrobial%20resistance%20in%202019:%20a%20systematic%20analysis&amp;volume=399&amp;publication_year=2022&amp;pages=629-655&amp;pmid=35065702&amp;doi=10.1016/s0140-6736(21)02724-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B5">
<span class="label">5.</span><cite>
Richardson EJ, et al. 2018. 
Gene exchange drives the ecological success of a multi-host bacterial pathogen. Nat. Ecol. Evol.
2, 1468–1478. ( 10.1038/s41559-018-0617-0)
</cite> [<a href="https://doi.org/10.1038/s41559-018-0617-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7610605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30038246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Ecol.%20Evol.&amp;title=Gene%20exchange%20drives%20the%20ecological%20success%20of%20a%20multi-host%20bacterial%20pathogen&amp;volume=2&amp;publication_year=2018&amp;pages=1468-1478&amp;pmid=30038246&amp;doi=10.1038/s41559-018-0617-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B6">
<span class="label">6.</span><cite>
Mohapatra S, Menon NG. 2022. 
Factors responsible for the emergence of novel viruses: an emphasis on SARS-CoV-2. Curr. Opin. Environ. Sci. Health
27, 100358. ( 10.1016/j.coesh.2022.100358)
</cite> [<a href="https://doi.org/10.1016/j.coesh.2022.100358" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8958772/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35369608/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Environ.%20Sci.%20Health&amp;title=Factors%20responsible%20for%20the%20emergence%20of%20novel%20viruses:%20an%20emphasis%20on%20SARS-CoV-2&amp;volume=27&amp;publication_year=2022&amp;pages=100358&amp;pmid=35369608&amp;doi=10.1016/j.coesh.2022.100358&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B7">
<span class="label">7.</span><cite>
Delaney NF, Balenger S, Bonneaud C, Marx CJ, Hill GE, Ferguson-Noel N, Tsai P, Rodrigo A, Edwards SV. 2012. 
Ultrafast evolution and loss of CRISPRs following a host shift in a novel wildlife pathogen, Mycoplasma gallisepticum. PLoS Genet.
8, e1002511. ( 10.1371/journal.pgen.1002511)
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1002511" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3276549/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22346765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&amp;title=Ultrafast%20evolution%20and%20loss%20of%20CRISPRs%20following%20a%20host%20shift%20in%20a%20novel%20wildlife%20pathogen,%20Mycoplasma%20gallisepticum&amp;volume=8&amp;publication_year=2012&amp;pages=e1002511&amp;pmid=22346765&amp;doi=10.1371/journal.pgen.1002511&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B8">
<span class="label">8.</span><cite>
Murray S, et al. 2017. 
Recombination-mediated host adaptation by avian Staphylococcus aureus. Genome Biol. Evol.
9, 830–842. ( 10.1093/gbe/evx037)
</cite> [<a href="https://doi.org/10.1093/gbe/evx037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5469444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28338786/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.%20Evol.&amp;title=Recombination-mediated%20host%20adaptation%20by%20avian%20Staphylococcus%20aureus&amp;volume=9&amp;publication_year=2017&amp;pages=830-842&amp;pmid=28338786&amp;doi=10.1093/gbe/evx037&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B9">
<span class="label">9.</span><cite>
Sanaei E, Albery GF, Yeoh YK, Lin Y, Cook LG, Engelstädter J. 2023. 
Host phylogeny and ecological associations best explain Wolbachia host shifts in scale insects. Mol. Ecol.
32, 2351–2363. ( 10.1111/mec.16883)
</cite> [<a href="https://doi.org/10.1111/mec.16883" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36785954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Ecol.&amp;title=Host%20phylogeny%20and%20ecological%20associations%20best%20explain%20Wolbachia%20host%20shifts%20in%20scale%20insects&amp;volume=32&amp;publication_year=2023&amp;pages=2351-2363&amp;pmid=36785954&amp;doi=10.1111/mec.16883&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B10">
<span class="label">10.</span><cite>
Lowy FD. 1998. 
Staphylococcus aureus infections. N. Engl. J. Med.
339, 520–532. ( 10.1056/nejm199808203390806)
</cite> [<a href="https://doi.org/10.1056/nejm199808203390806" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9709046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&amp;title=Staphylococcus%20aureus%20infections&amp;volume=339&amp;publication_year=1998&amp;pages=520-532&amp;pmid=9709046&amp;doi=10.1056/nejm199808203390806&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B11">
<span class="label">11.</span><cite>
Cuny C, et al. 2010. 
Emergence of methicillin-resistant Staphylococcus aureus (MRSA) in different animal species. Int. J. Med. Microbiol.
300, 109–117. ( 10.1016/j.ijmm.2009.11.002)
</cite> [<a href="https://doi.org/10.1016/j.ijmm.2009.11.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20005777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Med.%20Microbiol.&amp;title=Emergence%20of%20methicillin-resistant%20Staphylococcus%20aureus%20(MRSA)%20in%20different%20animal%20species&amp;volume=300&amp;publication_year=2010&amp;pages=109-117&amp;pmid=20005777&amp;doi=10.1016/j.ijmm.2009.11.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B12">
<span class="label">12.</span><cite>
Price LB, et al. 2012. 
Staphylococcus aureus CC398: host adaptation and emergence of methicillin resistance in livestock. mBio
3, e00305-11. ( 10.1128/mBio.00305-11)
</cite> [<a href="https://doi.org/10.1128/mBio.00305-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3280451/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22354957/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Staphylococcus%20aureus%20CC398:%20host%20adaptation%20and%20emergence%20of%20methicillin%20resistance%20in%20livestock&amp;volume=3&amp;publication_year=2012&amp;pages=e00305-11&amp;pmid=22354957&amp;doi=10.1128/mBio.00305-11&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B13">
<span class="label">13.</span><cite>
Howden BP, Giulieri SG, Wong Fok Lung T, Baines SL, Sharkey LK, Lee JYH, Hachani A, Monk IR, Stinear TP. 2023. 
Staphylococcus aureus host interactions and adaptation. Nat. Rev. Microbiol.
21, 380–395. ( 10.1038/s41579-023-00852-y)
</cite> [<a href="https://doi.org/10.1038/s41579-023-00852-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9882747/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36707725/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Microbiol.&amp;title=Staphylococcus%20aureus%20host%20interactions%20and%20adaptation&amp;volume=21&amp;publication_year=2023&amp;pages=380-395&amp;pmid=36707725&amp;doi=10.1038/s41579-023-00852-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B14">
<span class="label">14.</span><cite>
Fischer JR, Stallknecht DE, Luttrell P, Dhondt AA, Converse KA. 1997. 
Mycoplasmal conjunctivitis in wild songbirds: the spread of a new contagious disease in a mobile host population. Emerging Infect. Dis.
3, 69–72. ( 10.3201/eid0301.970110)</cite> [<a href="https://doi.org/10.3201/eid0301.970110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2627586/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9126448/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Emerging%20Infect.%20Dis.&amp;title=Mycoplasmal%20conjunctivitis%20in%20wild%20songbirds:%20the%20spread%20of%20a%20new%20contagious%20disease%20in%20a%20mobile%20host%20population&amp;volume=3&amp;publication_year=1997&amp;pages=69-72&amp;pmid=9126448&amp;doi=10.3201/eid0301.970110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B15">
<span class="label">15.</span><cite>
Bonneaud C, Balenger SL, Zhang J, Edwards SV, Hill GE. 2012. 
Innate immunity and the evolution of resistance to an emerging infectious disease in a wild bird. Mol. Ecol.
21, 2628–2639. ( 10.1111/j.1365-294x.2012.05551.x)
</cite> [<a href="https://doi.org/10.1111/j.1365-294x.2012.05551.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22512302/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Ecol.&amp;title=Innate%20immunity%20and%20the%20evolution%20of%20resistance%20to%20an%20emerging%20infectious%20disease%20in%20a%20wild%20bird&amp;volume=21&amp;publication_year=2012&amp;pages=2628-2639&amp;pmid=22512302&amp;doi=10.1111/j.1365-294x.2012.05551.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B16">
<span class="label">16.</span><cite>
Fleming-Davies AE, Williams PD, Dhondt AA, Dobson AP, Hochachka WM, Leon AE, Ley DH, Osnas EE, Hawley DM. 2018. 
Incomplete host immunity favors the evolution of virulence in an emergent pathogen. Science
359, 1030–1033. ( 10.1126/science.aao2140)
</cite> [<a href="https://doi.org/10.1126/science.aao2140" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6317705/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29496878/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Incomplete%20host%20immunity%20favors%20the%20evolution%20of%20virulence%20in%20an%20emergent%20pathogen&amp;volume=359&amp;publication_year=2018&amp;pages=1030-1033&amp;pmid=29496878&amp;doi=10.1126/science.aao2140&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B17">
<span class="label">17.</span><cite>
Delbianco A, Gibin D, Pasinato L, Boscia D, Morelli M, European Food Safety . 2022. 
Update of the Xylella spp. host plant database—systematic literature search up to 31 December 2021. EFSA J.
20, e07356. ( 10.2903/j.efsa.2022.7356)
</cite> [<a href="https://doi.org/10.2903/j.efsa.2022.7356" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9198695/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35734284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EFSA%20J.&amp;title=Update%20of%20the%20Xylella%20spp.%20host%20plant%20database%E2%80%94systematic%20literature%20search%20up%20to%2031%20December%202021&amp;volume=20&amp;publication_year=2022&amp;pages=e07356&amp;pmid=35734284&amp;doi=10.2903/j.efsa.2022.7356&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B18">
<span class="label">18.</span><cite>
Donegan MA, Coletta‐Filho HD, Almeida RPP. 2023. 
Parallel host shifts in a bacterial plant pathogen suggest independent genetic solutions. Mol. Plant Pathol.
24, 527–535. ( 10.1111/mpp.13316)
</cite> [<a href="https://doi.org/10.1111/mpp.13316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10189759/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36992605/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Plant%20Pathol.&amp;title=Parallel%20host%20shifts%20in%20a%20bacterial%20plant%20pathogen%20suggest%20independent%20genetic%20solutions&amp;volume=24&amp;publication_year=2023&amp;pages=527-535&amp;pmid=36992605&amp;doi=10.1111/mpp.13316&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B19">
<span class="label">19.</span><cite>
Tinsley MC, Majerus ME. 2007. 
Small steps or giant leaps for male-killers? Phylogenetic constraints to male-killer host shifts. BMC Evol. Biol.
7, 238. ( 10.1186/1471-2148-7-238)
</cite> [<a href="https://doi.org/10.1186/1471-2148-7-238" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2222251/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18047670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Evol.%20Biol.&amp;title=Small%20steps%20or%20giant%20leaps%20for%20male-killers?%20Phylogenetic%20constraints%20to%20male-killer%20host%20shifts&amp;volume=7&amp;publication_year=2007&amp;pages=238&amp;pmid=18047670&amp;doi=10.1186/1471-2148-7-238&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B20">
<span class="label">20.</span><cite>
Weinert LA, Araujo-Jnr EV, Ahmed MZ, Welch JJ. 2015. 
The incidence of bacterial endosymbionts in terrestrial arthropods. Proc. R. Soc. B
282, 20150249. ( 10.1098/rspb.2015.0249)</cite> [<a href="https://doi.org/10.1098/rspb.2015.0249" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4424649/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25904667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20R.%20Soc.%20B&amp;title=The%20incidence%20of%20bacterial%20endosymbionts%20in%20terrestrial%20arthropods&amp;volume=282&amp;publication_year=2015&amp;pages=20150249&amp;pmid=25904667&amp;doi=10.1098/rspb.2015.0249&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B21">
<span class="label">21.</span><cite>
Hilgenboecker K, Hammerstein P, Schlattmann P, Telschow A, Werren JH. 2008. 
How many species are infected with Wolbachia? A statistical analysis of current data. FEMS Microbiol. Lett.
281, 215–220. ( 10.1111/j.1574-6968.2008.01110.x)
</cite> [<a href="https://doi.org/10.1111/j.1574-6968.2008.01110.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2327208/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18312577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Lett.&amp;title=How%20many%20species%20are%20infected%20with%20Wolbachia?%20A%20statistical%20analysis%20of%20current%20data&amp;volume=281&amp;publication_year=2008&amp;pages=215-220&amp;pmid=18312577&amp;doi=10.1111/j.1574-6968.2008.01110.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B22">
<span class="label">22.</span><cite>
Zug R, Hammerstein P. 2012. 
Still a host of hosts for Wolbachia: analysis of recent data suggests that 40% of terrestrial arthropod species are infected. PLoS ONE
7, e38544. ( 10.1371/journal.pone.0038544)
</cite> [<a href="https://doi.org/10.1371/journal.pone.0038544" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3369835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22685581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Still%20a%20host%20of%20hosts%20for%20Wolbachia:%20analysis%20of%20recent%20data%20suggests%20that%2040%%20of%20terrestrial%20arthropod%20species%20are%20infected&amp;volume=7&amp;publication_year=2012&amp;pages=e38544&amp;pmid=22685581&amp;doi=10.1371/journal.pone.0038544&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B23">
<span class="label">23.</span><cite>
Russell JA, Goldman-Huertas B, Moreau CS, Baldo L, Stahlhut JK, Werren JH, Pierce NE. 2009. 
Specialization and geographic isolation among Wolbachia symbionts from ants and lycaenid butterflies. Evolution
63, 624–640. ( 10.1111/j.1558-5646.2008.00579.x)
</cite> [<a href="https://doi.org/10.1111/j.1558-5646.2008.00579.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19054050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evolution&amp;title=Specialization%20and%20geographic%20isolation%20among%20Wolbachia%20symbionts%20from%20ants%20and%20lycaenid%20butterflies&amp;volume=63&amp;publication_year=2009&amp;pages=624-640&amp;pmid=19054050&amp;doi=10.1111/j.1558-5646.2008.00579.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B24">
<span class="label">24.</span><cite>
Sanaei E, Charlat S, Engelstädter J. 2021. 
Wolbachia host shifts: routes, mechanisms, constraints and evolutionary consequences. Biol. Rev.
96, 433–453. ( 10.1111/brv.12663)
</cite> [<a href="https://doi.org/10.1111/brv.12663" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33128345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biol.%20Rev.&amp;title=Wolbachia%20host%20shifts:%20routes,%20mechanisms,%20constraints%20and%20evolutionary%20consequences&amp;volume=96&amp;publication_year=2021&amp;pages=433-453&amp;pmid=33128345&amp;doi=10.1111/brv.12663&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B25">
<span class="label">25.</span><cite>
Longdon B, Brockhurst MA, Russell CA, Welch JJ, Jiggins FM. 2014. 
The evolution and genetics of virus host shifts. PLoS Pathog.
10, e1004395. ( 10.1371/journal.ppat.1004395)
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1004395" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4223060/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25375777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&amp;title=The%20evolution%20and%20genetics%20of%20virus%20host%20shifts&amp;volume=10&amp;publication_year=2014&amp;pages=e1004395&amp;pmid=25375777&amp;doi=10.1371/journal.ppat.1004395&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B26">
<span class="label">26.</span><cite>
Longdon B, Hadfield JD, Webster CL, Obbard DJ, Jiggins FM. 2011. 
Host phylogeny determines viral persistence and replication in novel hosts. PLoS Pathog.
7, e1002260. ( 10.1371/journal.ppat.1002260)
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1002260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3178573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21966271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&amp;title=Host%20phylogeny%20determines%20viral%20persistence%20and%20replication%20in%20novel%20hosts&amp;volume=7&amp;publication_year=2011&amp;pages=e1002260&amp;pmid=21966271&amp;doi=10.1371/journal.ppat.1002260&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B27">
<span class="label">27.</span><cite>
Imrie RM, Roberts KE, Longdon B. 2021. 
Between virus correlations in the outcome of infection across host species: evidence of virus by host species interactions. Evol. Lett.
5, 472–483. ( 10.1002/evl3.247)
</cite> [<a href="https://doi.org/10.1002/evl3.247" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8484721/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34621534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evol.%20Lett.&amp;title=Between%20virus%20correlations%20in%20the%20outcome%20of%20infection%20across%20host%20species:%20evidence%20of%20virus%20by%20host%20species%20interactions&amp;volume=5&amp;publication_year=2021&amp;pages=472-483&amp;pmid=34621534&amp;doi=10.1002/evl3.247&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B28">
<span class="label">28.</span><cite>
Imrie RM, Wallace MA, Longdon B. 2025. 
Positive correlations in susceptibility to a diverse panel of viruses across Drosophilidae host species. Evol. Lett.
qraf002. ( 10.1093/evlett/qraf002)
</cite> [<a href="https://doi.org/10.1093/evlett/qraf002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7617412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40007858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evol.%20Lett.&amp;title=Positive%20correlations%20in%20susceptibility%20to%20a%20diverse%20panel%20of%20viruses%20across%20Drosophilidae%20host%20species&amp;publication_year=2025&amp;pages=qraf002&amp;pmid=40007858&amp;doi=10.1093/evlett/qraf002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B29">
<span class="label">29.</span><cite>
Huan Y, Kong Q, Mou H, Yi H. 2020. 
Antimicrobial peptides: classification, design, application and research progress in multiple fields. Front. Microbiol.
11, 582779. ( 10.3389/fmicb.2020.582779)
</cite> [<a href="https://doi.org/10.3389/fmicb.2020.582779" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7596191/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33178164/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=Antimicrobial%20peptides:%20classification,%20design,%20application%20and%20research%20progress%20in%20multiple%20fields&amp;volume=11&amp;publication_year=2020&amp;pages=582779&amp;pmid=33178164&amp;doi=10.3389/fmicb.2020.582779&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B30">
<span class="label">30.</span><cite>
Hoffmann JA, Reichhart JM. 2002. 
Drosophila innate immunity: an evolutionary perspective. Nat. Immunol.
3, 121–126. ( 10.1038/ni0202-121)
</cite> [<a href="https://doi.org/10.1038/ni0202-121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11812988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Immunol.&amp;title=Drosophila%20innate%20immunity:%20an%20evolutionary%20perspective&amp;volume=3&amp;publication_year=2002&amp;pages=121-126&amp;pmid=11812988&amp;doi=10.1038/ni0202-121&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B31">
<span class="label">31.</span><cite>
Lemaitre B, Hoffmann J. 2007. 
The host defense of Drosophila melanogaster. Annu. Rev. Immunol.
25, 697–743. ( 10.1146/annurev.immunol.25.022106.141615)
</cite> [<a href="https://doi.org/10.1146/annurev.immunol.25.022106.141615" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17201680/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Immunol.&amp;title=The%20host%20defense%20of%20Drosophila%20melanogaster&amp;volume=25&amp;publication_year=2007&amp;pages=697-743&amp;pmid=17201680&amp;doi=10.1146/annurev.immunol.25.022106.141615&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B32">
<span class="label">32.</span><cite>
Buchon N, Silverman N, Cherry S. 2014. 
Immunity in Drosophila melanogaster—from microbial recognition to whole-organism physiology. Nat. Rev. Immunol.
14, 796–810. ( 10.1038/nri3763)
</cite> [<a href="https://doi.org/10.1038/nri3763" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6190593/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25421701/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Immunol.&amp;title=Immunity%20in%20Drosophila%20melanogaster%E2%80%94from%20microbial%20recognition%20to%20whole-organism%20physiology&amp;volume=14&amp;publication_year=2014&amp;pages=796-810&amp;pmid=25421701&amp;doi=10.1038/nri3763&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B33">
<span class="label">33.</span><cite>
Sackton TB, Lazzaro BP, Schlenke TA, Evans JD, Hultmark D, Clark AG. 2007. 
Dynamic evolution of the innate immune system in Drosophila. Nat. Genet.
39, 1461–1468. ( 10.1038/ng.2007.60)
</cite> [<a href="https://doi.org/10.1038/ng.2007.60" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17987029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&amp;title=Dynamic%20evolution%20of%20the%20innate%20immune%20system%20in%20Drosophila&amp;volume=39&amp;publication_year=2007&amp;pages=1461-1468&amp;pmid=17987029&amp;doi=10.1038/ng.2007.60&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B34">
<span class="label">34.</span><cite>
Obbard DJ, Welch JJ, Kim KW, Jiggins FM. 2009. 
Quantifying adaptive evolution in the Drosophila immune system. PLoS Genet.
5, e1000698. ( 10.1371/journal.pgen.1000698)
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1000698" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2759075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19851448/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&amp;title=Quantifying%20adaptive%20evolution%20in%20the%20Drosophila%20immune%20system&amp;volume=5&amp;publication_year=2009&amp;pages=e1000698&amp;pmid=19851448&amp;doi=10.1371/journal.pgen.1000698&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B35">
<span class="label">35.</span><cite>
Unckless RL, Howick VM, Lazzaro BP. 2016. 
Convergent balancing selection on an antimicrobial peptide in Drosophila. Curr. Biol.
26, 257–262. ( 10.1016/j.cub.2015.11.063)
</cite> [<a href="https://doi.org/10.1016/j.cub.2015.11.063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4729654/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26776733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=Convergent%20balancing%20selection%20on%20an%20antimicrobial%20peptide%20in%20Drosophila&amp;volume=26&amp;publication_year=2016&amp;pages=257-262&amp;pmid=26776733&amp;doi=10.1016/j.cub.2015.11.063&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B36">
<span class="label">36.</span><cite>
Hanson MA, Grollmus L, Lemaitre B. 2023. 
Ecology-relevant bacteria drive the evolution of host antimicrobial peptides in Drosophila. Science
381, eadg5725. ( 10.1126/science.adg5725)
</cite> [<a href="https://doi.org/10.1126/science.adg5725" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37471548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Ecology-relevant%20bacteria%20drive%20the%20evolution%20of%20host%20antimicrobial%20peptides%20in%20Drosophila&amp;volume=381&amp;publication_year=2023&amp;pages=eadg5725&amp;pmid=37471548&amp;doi=10.1126/science.adg5725&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B37">
<span class="label">37.</span><cite>
Duneau D, Ferdy JB, Revah J, Kondolf H, Ortiz GA, Lazzaro BP, Buchon N. 2017. 
Stochastic variation in the initial phase of bacterial infection predicts the probability of survival in D. melanogaster. eLife
6, 28298. ( 10.7554/elife.28298)</cite> [<a href="https://doi.org/10.7554/elife.28298" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5703640/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29022878/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Stochastic%20variation%20in%20the%20initial%20phase%20of%20bacterial%20infection%20predicts%20the%20probability%20of%20survival%20in%20D.%20melanogaster&amp;volume=6&amp;publication_year=2017&amp;pages=28298&amp;pmid=29022878&amp;doi=10.7554/elife.28298&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B38">
<span class="label">38.</span><cite>
Needham AJ, Kibart M, Crossley H, Ingham PW, Foster SJ. 2004. 
Drosophila melanogaster as a model host for Staphylococcus aureus infection. Microbiology
150, 2347–2355. ( 10.1099/mic.0.27116-0)
</cite> [<a href="https://doi.org/10.1099/mic.0.27116-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15256576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Drosophila%20melanogaster%20as%20a%20model%20host%20for%20Staphylococcus%20aureus%20infection&amp;volume=150&amp;publication_year=2004&amp;pages=2347-2355&amp;pmid=15256576&amp;doi=10.1099/mic.0.27116-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B39">
<span class="label">39.</span><cite>
Clewell DB, Tomich PK, Gawron-Burke MC, Franke AE, Yagi Y, An FY. 1982. 
Mapping of Streptococcus faecalis plasmids pAD1 and pAD2 and studies relating to transposition of Tn917. J. Bacteriol.
152, 1220–1230. ( 10.1128/jb.152.3.1220-1230.1982)
</cite> [<a href="https://doi.org/10.1128/jb.152.3.1220-1230.1982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC221629/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6292164/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Bacteriol.&amp;title=Mapping%20of%20Streptococcus%20faecalis%20plasmids%20pAD1%20and%20pAD2%20and%20studies%20relating%20to%20transposition%20of%20Tn917&amp;volume=152&amp;publication_year=1982&amp;pages=1220-1230&amp;pmid=6292164&amp;doi=10.1128/jb.152.3.1220-1230.1982&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B40">
<span class="label">40.</span><cite>
Cox CR, Gilmore MS. 2007. 
Native microbial colonization of Drosophila melanogaster and its use as a model of Enterococcus faecalis pathogenesis. Infect. Immun.
75, 1565–1576. ( 10.1128/iai.01496-06)
</cite> [<a href="https://doi.org/10.1128/iai.01496-06" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1865669/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17220307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&amp;title=Native%20microbial%20colonization%20of%20Drosophila%20melanogaster%20and%20its%20use%20as%20a%20model%20of%20Enterococcus%20faecalis%20pathogenesis&amp;volume=75&amp;publication_year=2007&amp;pages=1565-1576&amp;pmid=17220307&amp;doi=10.1128/iai.01496-06&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B41">
<span class="label">41.</span><cite>
Gentry-Weeks CR, Karkhoff-Schweizer R, Pikis A, Estay M, Keith JM. 1999. 
Survival of Enterococcus faecalis in mouse peritoneal macrophages. Infect. Immun.
67, 2160–2165. ( 10.1128/iai.67.5.2160-2165.1999)
</cite> [<a href="https://doi.org/10.1128/iai.67.5.2160-2165.1999" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC115952/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10225869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&amp;title=Survival%20of%20Enterococcus%20faecalis%20in%20mouse%20peritoneal%20macrophages&amp;volume=67&amp;publication_year=1999&amp;pages=2160-2165&amp;pmid=10225869&amp;doi=10.1128/iai.67.5.2160-2165.1999&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B42">
<span class="label">42.</span><cite>
Hanson MA, Cohen LB, Marra A, Iatsenko I, Wasserman SA, Lemaitre B. 2021. 
The Drosophila Baramicin polypeptide gene protects against fungal infection. PLoS Pathog.
17, e1009846. ( 10.1371/journal.ppat.1009846)
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1009846" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8423362/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34432851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&amp;title=The%20Drosophila%20Baramicin%20polypeptide%20gene%20protects%20against%20fungal%20infection&amp;volume=17&amp;publication_year=2021&amp;pages=e1009846&amp;pmid=34432851&amp;doi=10.1371/journal.ppat.1009846&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B43">
<span class="label">43.</span><cite>
Huang J, et al. 2023. 
A Toll pathway effector protects Drosophila specifically from distinct toxins secreted by a fungus or a bacterium. Proc. Natl Acad. Sci. USA
120, e2205140120. ( 10.1073/pnas.2205140120)
</cite> [<a href="https://doi.org/10.1073/pnas.2205140120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10041126/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36917667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;title=A%20Toll%20pathway%20effector%20protects%20Drosophila%20specifically%20from%20distinct%20toxins%20secreted%20by%20a%20fungus%20or%20a%20bacterium&amp;volume=120&amp;publication_year=2023&amp;pages=e2205140120&amp;pmid=36917667&amp;doi=10.1073/pnas.2205140120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B44">
<span class="label">44.</span><cite>
Chapman JR, Dowell MA, Chan R, Unckless RL. 2020. 
The genetic basis of natural variation in Drosophila melanogaster immune defense against Enterococcus faecalis. Genes
11, 234. ( 10.3390/genes11020234)
</cite> [<a href="https://doi.org/10.3390/genes11020234" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7074548/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32098395/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes&amp;title=The%20genetic%20basis%20of%20natural%20variation%20in%20Drosophila%20melanogaster%20immune%20defense%20against%20Enterococcus%20faecalis&amp;volume=11&amp;publication_year=2020&amp;pages=234&amp;pmid=32098395&amp;doi=10.3390/genes11020234&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B45">
<span class="label">45.</span><cite>
Liu GY, Essex A, Buchanan JT, Datta V, Hoffman HM, Bastian JF, Fierer J, Nizet V. 2005. 
Staphylococcus aureus golden pigment impairs neutrophil killing and promotes virulence through its antioxidant activity. J. Exp. Med.
202, 209–215. ( 10.1084/jem.20050846)
</cite> [<a href="https://doi.org/10.1084/jem.20050846" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2213009/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16009720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Exp.%20Med.&amp;title=Staphylococcus%20aureus%20golden%20pigment%20impairs%20neutrophil%20killing%20and%20promotes%20virulence%20through%20its%20antioxidant%20activity&amp;volume=202&amp;publication_year=2005&amp;pages=209-215&amp;pmid=16009720&amp;doi=10.1084/jem.20050846&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B46">
<span class="label">46.</span><cite>
Matuszewska M, Murray GGR, Harrison EM, Holmes MA, Weinert LA. 2020. 
The evolutionary genomics of host specificity in Staphylococcus aureus. Trends Microbiol.
28, 465–477. ( 10.1016/j.tim.2019.12.007)
</cite> [<a href="https://doi.org/10.1016/j.tim.2019.12.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31948727/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Microbiol.&amp;title=The%20evolutionary%20genomics%20of%20host%20specificity%20in%20Staphylococcus%20aureus&amp;volume=28&amp;publication_year=2020&amp;pages=465-477&amp;pmid=31948727&amp;doi=10.1016/j.tim.2019.12.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B47">
<span class="label">47.</span><cite>
Binggeli O, Neyen C, Poidevin M, Lemaitre B. 2014. 
Prophenoloxidase activation is required for survival to microbial infections in Drosophila. PLoS Pathog.
10, e1004067. ( 10.1371/journal.ppat.1004067)
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1004067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4006879/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24788090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&amp;title=Prophenoloxidase%20activation%20is%20required%20for%20survival%20to%20microbial%20infections%20in%20Drosophila&amp;volume=10&amp;publication_year=2014&amp;pages=e1004067&amp;pmid=24788090&amp;doi=10.1371/journal.ppat.1004067&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B48">
<span class="label">48.</span><cite>
Dudzic JP, Hanson MA, Iatsenko I, Kondo S, Lemaitre B. 2019. 
More than black or white: melanization and Toll share regulatory serine proteases in Drosophila. Cell Rep.
27, 1050–1061.( 10.1016/j.celrep.2019.03.101)
</cite> [<a href="https://doi.org/10.1016/j.celrep.2019.03.101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31018123/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&amp;title=More%20than%20black%20or%20white:%20melanization%20and%20Toll%20share%20regulatory%20serine%20proteases%20in%20Drosophila&amp;volume=27&amp;publication_year=2019&amp;pages=1050-1061&amp;pmid=31018123&amp;doi=10.1016/j.celrep.2019.03.101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B49">
<span class="label">49.</span><cite>
Mulet M, Gomila M, Lemaitre B, Lalucat J, García-Valdés E. 2012. 
Taxonomic characterisation of Pseudomonas strain L48 and formal proposal of Pseudomonas entomophila sp. nov. Syst. Appl. Microbiol.
35, 145–149. ( 10.1016/j.syapm.2011.12.003)
</cite> [<a href="https://doi.org/10.1016/j.syapm.2011.12.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22326814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Syst.%20Appl.%20Microbiol.&amp;title=Taxonomic%20characterisation%20of%20Pseudomonas%20strain%20L48%20and%20formal%20proposal%20of%20Pseudomonas%20entomophila%20sp.%20nov&amp;volume=35&amp;publication_year=2012&amp;pages=145-149&amp;pmid=22326814&amp;doi=10.1016/j.syapm.2011.12.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B50">
<span class="label">50.</span><cite>
Vodovar N, Vinals M, Liehl P, Basset A, Degrouard J, Spellman P, Boccard F, Lemaitre B. 2005. 
Drosophila host defense after oral infection by an entomopathogenic Pseudomonas species. Proc. Natl Acad. Sci. USA
102, 11414–11419. ( 10.1073/pnas.0502240102)
</cite> [<a href="https://doi.org/10.1073/pnas.0502240102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1183552/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16061818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;title=Drosophila%20host%20defense%20after%20oral%20infection%20by%20an%20entomopathogenic%20Pseudomonas%20species&amp;volume=102&amp;publication_year=2005&amp;pages=11414-11419&amp;pmid=16061818&amp;doi=10.1073/pnas.0502240102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B51">
<span class="label">51.</span><cite>
Vodovar N, et al. 2006. 
Complete genome sequence of the entomopathogenic and metabolically versatile soil bacterium Pseudomonas entomophila. Nat. Biotechnol.
24, 673–679. ( 10.1038/nbt1212)
</cite> [<a href="https://doi.org/10.1038/nbt1212" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16699499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&amp;title=Complete%20genome%20sequence%20of%20the%20entomopathogenic%20and%20metabolically%20versatile%20soil%20bacterium%20Pseudomonas%20entomophila&amp;volume=24&amp;publication_year=2006&amp;pages=673-679&amp;pmid=16699499&amp;doi=10.1038/nbt1212&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B52">
<span class="label">52.</span><cite>
Dieppois G, Opota O, Lalucat J, Lemaitre B. 2015. 
<em>Pseudomonas entomophila</em>: a versatile bacterium with entomopathogenic properties. In Pseudomonas: new aspects of Pseudomonas biology (eds Ramos JL, Goldberg JB, Filloux A), pp. 25–49. Dordrecht, The Netherlands: Springer. ( 10.1007/978-94-017-9555-5_2)</cite> [<a href="https://doi.org/10.1007/978-94-017-9555-5_2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Pseudomonas:%20new%20aspects%20of%20Pseudomonas%20biology&amp;publication_year=2015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B53">
<span class="label">53.</span><cite>
Prakash A, Monteith KM, Vale PF. 2022. 
Mechanisms of damage prevention, signalling and repair impact disease tolerance. Proc. R. Soc. B
289, 20220837. ( 10.1098/rspb.2022.0837)</cite> [<a href="https://doi.org/10.1098/rspb.2022.0837" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9382215/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35975433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20R.%20Soc.%20B&amp;title=Mechanisms%20of%20damage%20prevention,%20signalling%20and%20repair%20impact%20disease%20tolerance&amp;volume=289&amp;publication_year=2022&amp;pages=20220837&amp;pmid=35975433&amp;doi=10.1098/rspb.2022.0837&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B54">
<span class="label">54.</span><cite>
Juneja P, Lazzaro BP. 2009. 
Providencia sneebia sp. nov. and Providencia burhodogranariea sp. nov., isolated from wild Drosophila melanogaster. Int. J. Syst. Evol. Microbiol.
59, 1108–1111. ( 10.1099/ijs.0.000117-0)
</cite> [<a href="https://doi.org/10.1099/ijs.0.000117-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19406801/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&amp;title=Providencia%20sneebia%20sp.%20nov.%20and%20Providencia%20burhodogranariea%20sp.%20nov.,%20isolated%20from%20wild%20Drosophila%20melanogaster&amp;volume=59&amp;publication_year=2009&amp;pages=1108-1111&amp;pmid=19406801&amp;doi=10.1099/ijs.0.000117-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B55">
<span class="label">55.</span><cite>
Hanson MA, Dostálová A, Ceroni C, Poidevin M, Kondo S, Lemaitre B. 2019. 
Synergy and remarkable specificity of antimicrobial peptides in vivo using a systematic knockout approach. eLife
8, e44341. ( 10.7554/elife.44341)
</cite> [<a href="https://doi.org/10.7554/elife.44341" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6398976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30803481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Synergy%20and%20remarkable%20specificity%20of%20antimicrobial%20peptides%20in%20vivo%20using%20a%20systematic%20knockout%20approach&amp;volume=8&amp;publication_year=2019&amp;pages=e44341&amp;pmid=30803481&amp;doi=10.7554/elife.44341&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B56">
<span class="label">56.</span><cite>
Short SM, Lazzaro BP. 2010. 
Female and male genetic contributions to post-mating immune defence in female Drosophila melanogaster. Proc. R. Soc. B
277, 3649–3657. ( 10.1098/rspb.2010.0937)</cite> [<a href="https://doi.org/10.1098/rspb.2010.0937" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2982248/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20573620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20R.%20Soc.%20B&amp;title=Female%20and%20male%20genetic%20contributions%20to%20post-mating%20immune%20defence%20in%20female%20Drosophila%20melanogaster&amp;volume=277&amp;publication_year=2010&amp;pages=3649-3657&amp;pmid=20573620&amp;doi=10.1098/rspb.2010.0937&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B57">
<span class="label">57.</span><cite>
Schwenke RA, Lazzaro BP. 2017. 
Juvenile hormone suppresses resistance to infection in mated female Drosophila melanogaster. Curr. Biol.
27, 596–601. ( 10.1016/j.cub.2017.01.004)
</cite> [<a href="https://doi.org/10.1016/j.cub.2017.01.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5319889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28190728/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Biol.&amp;title=Juvenile%20hormone%20suppresses%20resistance%20to%20infection%20in%20mated%20female%20Drosophila%20melanogaster&amp;volume=27&amp;publication_year=2017&amp;pages=596-601&amp;pmid=28190728&amp;doi=10.1016/j.cub.2017.01.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B58">
<span class="label">58.</span><cite>
Duneau DF, et al. 2017. 
The Toll pathway underlies host sexual dimorphism in resistance to both Gram-negative and Gram-positive bacteria in mated Drosophila. BMC Biol.
15, 124. ( 10.1186/s12915-017-0466-3)
</cite> [<a href="https://doi.org/10.1186/s12915-017-0466-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5740927/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29268741/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Biol.&amp;title=The%20Toll%20pathway%20underlies%20host%20sexual%20dimorphism%20in%20resistance%20to%20both%20Gram-negative%20and%20Gram-positive%20bacteria%20in%20mated%20Drosophila&amp;volume=15&amp;publication_year=2017&amp;pages=124&amp;pmid=29268741&amp;doi=10.1186/s12915-017-0466-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B59">
<span class="label">59.</span><cite>
Gupta V, Frank AM, Matolka N, Lazzaro BP. 2022. 
Inherent constraints on a polyfunctional tissue lead to a reproduction-immunity tradeoff. BMC Biol.
20, 127. ( 10.1186/s12915-022-01328-w)
</cite> [<a href="https://doi.org/10.1186/s12915-022-01328-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9161490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35655304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Biol.&amp;title=Inherent%20constraints%20on%20a%20polyfunctional%20tissue%20lead%20to%20a%20reproduction-immunity%20tradeoff&amp;volume=20&amp;publication_year=2022&amp;pages=127&amp;pmid=35655304&amp;doi=10.1186/s12915-022-01328-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B60">
<span class="label">60.</span><cite>
Larson EL, Strom MS, Evans CA. 1980. 
Analysis of three variables in sampling solutions used to assay bacteria of hands: type of solution, use of antiseptic neutralizers, and solution temperature. J. Clin. Microbiol.
12, 355–360. ( 10.1128/jcm.12.3.355-360.1980)
</cite> [<a href="https://doi.org/10.1128/jcm.12.3.355-360.1980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC273590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7012171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Microbiol.&amp;title=Analysis%20of%20three%20variables%20in%20sampling%20solutions%20used%20to%20assay%20bacteria%20of%20hands:%20type%20of%20solution,%20use%20of%20antiseptic%20neutralizers,%20and%20solution%20temperature&amp;volume=12&amp;publication_year=1980&amp;pages=355-360&amp;pmid=7012171&amp;doi=10.1128/jcm.12.3.355-360.1980&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B61">
<span class="label">61.</span><cite>
Longdon B, Hadfield JD, Day JP, Smith SCL, McGonigle JE, Cogni R, Cao C, Jiggins FM. 2015. 
The causes and consequences of changes in virulence following pathogen host shifts. PLoS Pathog.
11, e1004728. ( 10.1371/journal.ppat.1004728)
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1004728" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4361674/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25774803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&amp;title=The%20causes%20and%20consequences%20of%20changes%20in%20virulence%20following%20pathogen%20host%20shifts&amp;volume=11&amp;publication_year=2015&amp;pages=e1004728&amp;pmid=25774803&amp;doi=10.1371/journal.ppat.1004728&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B62">
<span class="label">62.</span><cite>
Jarrod H. 2010. 
MCMC methods for multi-response generalized linear mixed models: the MCMCglmm R package. J. Stat. Softw.
33, 1–22. ( 10.18637/jss.v033.i02)20808728
</cite> [<a href="https://doi.org/10.18637/jss.v033.i02" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Stat.%20Softw.&amp;title=MCMC%20methods%20for%20multi-response%20generalized%20linear%20mixed%20models:%20the%20MCMCglmm%20R%20package&amp;volume=33&amp;publication_year=2010&amp;pages=1-22&amp;doi=10.18637/jss.v033.i02&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B63">
<span class="label">63.</span><cite>
R Core Team . 2023. 
R: a language and environment for statistical computing. Vienna, Austria: R Foundation for Statistical Computing.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=R:%20a%20language%20and%20environment%20for%20statistical%20computing&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B64">
<span class="label">64.</span><cite>
Tokuda T, Goodrich B, Mechelen IV, Gelman A, Tuerlinckx F. 2011. 
Visualizing distributions of covariance matrices. Technical report. New York, NY: Columbia University.</cite> [<a href="https://scholar.google.com/scholar_lookup?Tokuda%20T,%20Goodrich%20B,%20Mechelen%20IV,%20Gelman%20A,%20Tuerlinckx%20F.%202011.%20Visualizing%20distributions%20of%20covariance%20matrices.%20Technical%20report.%20New%20York,%20NY:%20Columbia%20University." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B65">
<span class="label">65.</span><cite>
Freckleton RP, Harvey PH, Pagel M. 2002. 
Phylogenetic analysis and comparative data: a test and review of evidence. Am. Nat.
160, 712–726. ( 10.1086/343873)
</cite> [<a href="https://doi.org/10.1086/343873" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18707460/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am.%20Nat.&amp;title=Phylogenetic%20analysis%20and%20comparative%20data:%20a%20test%20and%20review%20of%20evidence&amp;volume=160&amp;publication_year=2002&amp;pages=712-726&amp;pmid=18707460&amp;doi=10.1086/343873&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B66">
<span class="label">66.</span><cite>
Housworth EA, Martins EP, Lynch M. 2004. 
The phylogenetic mixed model. Am. Nat.
163, 84–96. ( 10.1086/380570)
</cite> [<a href="https://doi.org/10.1086/380570" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14767838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am.%20Nat.&amp;title=The%20phylogenetic%20mixed%20model&amp;volume=163&amp;publication_year=2004&amp;pages=84-96&amp;pmid=14767838&amp;doi=10.1086/380570&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B67">
<span class="label">67.</span><cite>
Pagel M. 1999. 
Inferring the historical patterns of biological evolution. Nature
401, 877–884. ( 10.1038/44766)
</cite> [<a href="https://doi.org/10.1038/44766" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10553904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Inferring%20the%20historical%20patterns%20of%20biological%20evolution&amp;volume=401&amp;publication_year=1999&amp;pages=877-884&amp;pmid=10553904&amp;doi=10.1038/44766&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B68">
<span class="label">68.</span><cite>
Falconer D, Mackay T. 1996. 
Introduction to quantitative genetics, 4th edn. Harlow, UK: Addison Wesley Longman.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Introduction%20to%20quantitative%20genetics&amp;publication_year=1996&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B69">
<span class="label">69.</span><cite>
Abbott WS. 1925. 
A method of computing the effectiveness of an insecticide. J. Econ. Entomol.
18, 265–267. ( 10.1093/jee/18.2.265a)</cite> [<a href="https://doi.org/10.1093/jee/18.2.265a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Econ.%20Entomol.&amp;title=A%20method%20of%20computing%20the%20effectiveness%20of%20an%20insecticide&amp;volume=18&amp;publication_year=1925&amp;pages=265-267&amp;doi=10.1093/jee/18.2.265a&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B70">
<span class="label">70.</span><cite>
Rosenheim JA, Hoy MA. 1989. 
Confidence intervals for the Abbott’s formula correction of bioassay data for control response. J. Econ. Entomol.
82, 331–335. ( 10.1093/jee/82.2.331)</cite> [<a href="https://doi.org/10.1093/jee/82.2.331" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Econ.%20Entomol.&amp;title=Confidence%20intervals%20for%20the%20Abbott%E2%80%99s%20formula%20correction%20of%20bioassay%20data%20for%20control%20response&amp;volume=82&amp;publication_year=1989&amp;pages=331-335&amp;doi=10.1093/jee/82.2.331&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B71">
<span class="label">71.</span><cite>
De Gregorio E, Spellman PT, Tzou P, Rubin GM, Lemaitre B. 2002. 
The Toll and Imd pathways are the major regulators of the immune response in Drosophila. EMBO J.
21, 2568–2579. ( 10.1093/emboj/21.11.2568)
</cite> [<a href="https://doi.org/10.1093/emboj/21.11.2568" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC126042/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12032070/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20J.&amp;title=The%20Toll%20and%20Imd%20pathways%20are%20the%20major%20regulators%20of%20the%20immune%20response%20in%20Drosophila&amp;volume=21&amp;publication_year=2002&amp;pages=2568-2579&amp;pmid=12032070&amp;doi=10.1093/emboj/21.11.2568&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B72">
<span class="label">72.</span><cite>
Troha K, Im JH, Revah J, Lazzaro BP, Buchon N. 2018. 
Comparative transcriptomics reveals CrebA as a novel regulator of infection tolerance in D. melanogaster. PLoS Pathog.
14, e1006847. ( 10.1371/journal.ppat.1006847)
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1006847" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5812652/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29394281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&amp;title=Comparative%20transcriptomics%20reveals%20CrebA%20as%20a%20novel%20regulator%20of%20infection%20tolerance%20in%20D.%20melanogaster&amp;volume=14&amp;publication_year=2018&amp;pages=e1006847&amp;pmid=29394281&amp;doi=10.1371/journal.ppat.1006847&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B73">
<span class="label">73.</span><cite>
Gilchrist GW, Huey RB, Serra L. 2001. 
Rapid evolution of wing size clines in Drosophila subobscura. Genetica
112–113, 273–286. ( 10.1007/978-94-010-0585-2_17)</cite> [<a href="https://doi.org/10.1007/978-94-010-0585-2_17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11838770/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetica&amp;title=Rapid%20evolution%20of%20wing%20size%20clines%20in%20Drosophila%20subobscura&amp;volume=112%E2%80%93113&amp;publication_year=2001&amp;pages=273-286&amp;pmid=11838770&amp;doi=10.1007/978-94-010-0585-2_17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B74">
<span class="label">74.</span><cite>
Rainford JL, Hofreiter M, Mayhew PJ. 2016. 
Phylogenetic analyses suggest that diversification and body size evolution are independent in insects. BMC Evol. Biol.
16, 8. ( 10.1186/s12862-015-0570-3)
</cite> [<a href="https://doi.org/10.1186/s12862-015-0570-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4706648/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26746988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Evol.%20Biol.&amp;title=Phylogenetic%20analyses%20suggest%20that%20diversification%20and%20body%20size%20evolution%20are%20independent%20in%20insects&amp;volume=16&amp;publication_year=2016&amp;pages=8&amp;pmid=26746988&amp;doi=10.1186/s12862-015-0570-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B75">
<span class="label">75.</span><cite>
McCarville JL, Ayres JS. 2018. 
Disease tolerance: concept and mechanisms. Curr. Opin. Immunol.
50, 88–93. ( 10.1016/j.coi.2017.12.003)
</cite> [<a href="https://doi.org/10.1016/j.coi.2017.12.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5884632/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29253642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Immunol.&amp;title=Disease%20tolerance:%20concept%20and%20mechanisms&amp;volume=50&amp;publication_year=2018&amp;pages=88-93&amp;pmid=29253642&amp;doi=10.1016/j.coi.2017.12.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B76">
<span class="label">76.</span><cite>
Rutschmann S, Kilinc A, Ferrandon D. 2002. 
Cutting edge: the toll pathway is required for resistance to Gram-positive bacterial infections in Drosophila. J. Immunol.
168, 1542–1546. ( 10.4049/jimmunol.168.4.1542)
</cite> [<a href="https://doi.org/10.4049/jimmunol.168.4.1542" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11823479/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.&amp;title=Cutting%20edge:%20the%20toll%20pathway%20is%20required%20for%20resistance%20to%20Gram-positive%20bacterial%20infections%20in%20Drosophila&amp;volume=168&amp;publication_year=2002&amp;pages=1542-1546&amp;pmid=11823479&amp;doi=10.4049/jimmunol.168.4.1542&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B77">
<span class="label">77.</span><cite>
Ambrosi P, Chahda JS, Koslen HR, Chiel HJ, Mizutani CM. 2014. 
Modeling of the dorsal gradient across species reveals interaction between embryo morphology and toll signaling pathway during evolution. PLoS Comput. Biol.
10, e1003807. ( 10.1371/journal.pcbi.1003807)
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1003807" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4148200/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25165818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput.%20Biol.&amp;title=Modeling%20of%20the%20dorsal%20gradient%20across%20species%20reveals%20interaction%20between%20embryo%20morphology%20and%20toll%20signaling%20pathway%20during%20evolution&amp;volume=10&amp;publication_year=2014&amp;pages=e1003807&amp;pmid=25165818&amp;doi=10.1371/journal.pcbi.1003807&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B78">
<span class="label">78.</span><cite>
Palmer WJ, Jiggins FM. 2015. 
Comparative genomics reveals the origins and diversity of arthropod immune systems. Mol. Biol. Evol.
32, 2111–2129. ( 10.1093/molbev/msv093)
</cite> [<a href="https://doi.org/10.1093/molbev/msv093" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4833078/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25908671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Biol.%20Evol.&amp;title=Comparative%20genomics%20reveals%20the%20origins%20and%20diversity%20of%20arthropod%20immune%20systems&amp;volume=32&amp;publication_year=2015&amp;pages=2111-2129&amp;pmid=25908671&amp;doi=10.1093/molbev/msv093&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B79">
<span class="label">79.</span><cite>
Nishide Y, Kageyama D, Yokoi K, Jouraku A, Tanaka H, Futahashi R, Fukatsu T. 2019. 
Functional crosstalk across IMD and Toll pathways: insight into the evolution of incomplete immune cascades. Proc. R. Soc. B
286, 20182207. ( 10.1098/rspb.2018.2207)</cite> [<a href="https://doi.org/10.1098/rspb.2018.2207" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6408883/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30963836/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20R.%20Soc.%20B&amp;title=Functional%20crosstalk%20across%20IMD%20and%20Toll%20pathways:%20insight%20into%20the%20evolution%20of%20incomplete%20immune%20cascades&amp;volume=286&amp;publication_year=2019&amp;pages=20182207&amp;pmid=30963836&amp;doi=10.1098/rspb.2018.2207&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B80">
<span class="label">80.</span><cite>
Kounatidis I, Ligoxygakis P. 2012. 
Drosophila as a model system to unravel the layers of innate immunity to infection. Open Biol.
2, 120075. ( 10.1098/rsob.120075)
</cite> [<a href="https://doi.org/10.1098/rsob.120075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3376734/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22724070/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Open%20Biol.&amp;title=Drosophila%20as%20a%20model%20system%20to%20unravel%20the%20layers%20of%20innate%20immunity%20to%20infection&amp;volume=2&amp;publication_year=2012&amp;pages=120075&amp;pmid=22724070&amp;doi=10.1098/rsob.120075&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B81">
<span class="label">81.</span><cite>
Valanne S, et al. 2010. 
Genome-wide RNA interference in Drosophila cells identifies G protein-coupled receptor kinase 2 as a conserved regulator of NF-κB signaling. J. Immunol.
184, 6188–6198. ( 10.4049/jimmunol.1000261)
</cite> [<a href="https://doi.org/10.4049/jimmunol.1000261" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20421637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.&amp;title=Genome-wide%20RNA%20interference%20in%20Drosophila%20cells%20identifies%20G%20protein-coupled%20receptor%20kinase%202%20as%20a%20conserved%20regulator%20of%20NF-%CE%BAB%20signaling&amp;volume=184&amp;publication_year=2010&amp;pages=6188-6198&amp;pmid=20421637&amp;doi=10.4049/jimmunol.1000261&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B82">
<span class="label">82.</span><cite>
Xu R, et al. 2023. 
The Toll pathway mediates Drosophila resilience to Aspergillus mycotoxins through specific bomanins. EMBO Rep.
24, e56036. ( 10.15252/embr.202256036)
</cite> [<a href="https://doi.org/10.15252/embr.202256036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9827548/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36322050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Rep.&amp;title=The%20Toll%20pathway%20mediates%20Drosophila%20resilience%20to%20Aspergillus%20mycotoxins%20through%20specific%20bomanins&amp;volume=24&amp;publication_year=2023&amp;pages=e56036&amp;pmid=36322050&amp;doi=10.15252/embr.202256036&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B83">
<span class="label">83.</span><cite>
Cabrera K, Hoard DS, Gibson O, Martinez DI, Wunderlich Z. 2023. 
Drosophila immune priming to Enterococcus faecalis relies on immune tolerance rather than resistance. PLoS Pathog.
19, e1011567. ( 10.1371/journal.ppat.1011567)
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1011567" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10446173/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37566589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&amp;title=Drosophila%20immune%20priming%20to%20Enterococcus%20faecalis%20relies%20on%20immune%20tolerance%20rather%20than%20resistance&amp;volume=19&amp;publication_year=2023&amp;pages=e1011567&amp;pmid=37566589&amp;doi=10.1371/journal.ppat.1011567&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B84">
<span class="label">84.</span><cite>
Iatsenko I, Marra A, Boquete JP, Peña J, Lemaitre B. 2020. 
Iron sequestration by transferrin 1 mediates nutritional immunity in Drosophila melanogaster. Proc. Natl Acad. Sci. USA
117, 7317–7325. ( 10.1073/pnas.1914830117)
</cite> [<a href="https://doi.org/10.1073/pnas.1914830117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7132258/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32188787/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;title=Iron%20sequestration%20by%20transferrin%201%20mediates%20nutritional%20immunity%20in%20Drosophila%20melanogaster&amp;volume=117&amp;publication_year=2020&amp;pages=7317-7325&amp;pmid=32188787&amp;doi=10.1073/pnas.1914830117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B85">
<span class="label">85.</span><cite>
Wadhawan A, Simoes da Silva CJ, Nunes CD, Edwards AM, Dionne MS. 2022. 
E. faecalis acquires resistance to antimicrobials and insect immunity via common mechanisms. bioRxiv. ( 10.1101/2022.08.17.504265)</cite> [<a href="https://doi.org/10.1101/2022.08.17.504265" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="B86">
<span class="label">86.</span><cite>
Cornforth DM, Matthews A, Brown SP, Raymond B. 2015. 
Bacterial cooperation causes systematic errors in pathogen risk assessment due to the failure of the independent action hypothesis. PLoS Pathog.
11, e1004775. ( 10.1371/journal.ppat.1004775)
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1004775" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4409216/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25909384/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Pathog.&amp;title=Bacterial%20cooperation%20causes%20systematic%20errors%20in%20pathogen%20risk%20assessment%20due%20to%20the%20failure%20of%20the%20independent%20action%20hypothesis&amp;volume=11&amp;publication_year=2015&amp;pages=e1004775&amp;pmid=25909384&amp;doi=10.1371/journal.ppat.1004775&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B87">
<span class="label">87.</span><cite>
Panwar RB, Sequeira RP, Clarke TB. 2021. 
Microbiota-mediated protection against antibiotic-resistant pathogens. Genes Immun.
22, 255–267. ( 10.1038/s41435-021-00129-5)
</cite> [<a href="https://doi.org/10.1038/s41435-021-00129-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8497270/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33947987/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Immun.&amp;title=Microbiota-mediated%20protection%20against%20antibiotic-resistant%20pathogens&amp;volume=22&amp;publication_year=2021&amp;pages=255-267&amp;pmid=33947987&amp;doi=10.1038/s41435-021-00129-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B88">
<span class="label">88.</span><cite>
Barron AJ, Agrawal S, Lesperance DNA, Doucette J, Calle S, Broderick NA. 2024. 
Microbiome-derived acidity protects against microbial invasion in Drosophila. Cell Rep.
43, 114087. ( 10.1016/j.celrep.2024.114087)
</cite> [<a href="https://doi.org/10.1016/j.celrep.2024.114087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11163447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38583152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&amp;title=Microbiome-derived%20acidity%20protects%20against%20microbial%20invasion%20in%20Drosophila&amp;volume=43&amp;publication_year=2024&amp;pages=114087&amp;pmid=38583152&amp;doi=10.1016/j.celrep.2024.114087&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B89">
<span class="label">89.</span><cite>
Blum JE, Fischer CN, Miles J, Handelsman J. 2013. 
Frequent replenishment sustains the beneficial microbiome of Drosophila melanogaster. mBio
4, e00860-13. ( 10.1128/mBio.00860-13)
</cite> [<a href="https://doi.org/10.1128/mBio.00860-13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3892787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24194543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Frequent%20replenishment%20sustains%20the%20beneficial%20microbiome%20of%20Drosophila%20melanogaster&amp;volume=4&amp;publication_year=2013&amp;pages=e00860-13&amp;pmid=24194543&amp;doi=10.1128/mBio.00860-13&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B90">
<span class="label">90.</span><cite>
Chandler JA, Morgan Lang J, Bhatnagar S, Eisen JA, Kopp A. 2011. 
Bacterial communities of diverse Drosophila species: ecological context of a host–microbe model system. PLoS Genet.
7, e1002272. ( 10.1371/journal.pgen.1002272)
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1002272" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3178584/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21966276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genet.&amp;title=Bacterial%20communities%20of%20diverse%20Drosophila%20species:%20ecological%20context%20of%20a%20host%E2%80%93microbe%20model%20system&amp;volume=7&amp;publication_year=2011&amp;pages=e1002272&amp;pmid=21966276&amp;doi=10.1371/journal.pgen.1002272&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B91">
<span class="label">91.</span><cite>
Brooks AW, Kohl KD, Brucker RM, van Opstal EJ, Bordenstein SR. 2016. 
Phylosymbiosis: relationships and functional effects of microbial communities across host evolutionary history. PLoS Biol.
14, e2000225. ( 10.1371/journal.pbio.2000225)
</cite> [<a href="https://doi.org/10.1371/journal.pbio.2000225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5115861/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27861590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Biol.&amp;title=Phylosymbiosis:%20relationships%20and%20functional%20effects%20of%20microbial%20communities%20across%20host%20evolutionary%20history&amp;volume=14&amp;publication_year=2016&amp;pages=e2000225&amp;pmid=27861590&amp;doi=10.1371/journal.pbio.2000225&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B92">
<span class="label">92.</span><cite>
Thines M. 2019. 
An evolutionary framework for host shifts—jumping ships for survival. New Phytol.
224, 605–617. ( 10.1111/nph.16092)
</cite> [<a href="https://doi.org/10.1111/nph.16092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31381166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=An%20evolutionary%20framework%20for%20host%20shifts%E2%80%94jumping%20ships%20for%20survival&amp;volume=224&amp;publication_year=2019&amp;pages=605-617&amp;pmid=31381166&amp;doi=10.1111/nph.16092&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B93">
<span class="label">93.</span><cite>
Bonneaud C, Weinert LA, Kuijper B. 2019. 
Understanding the emergence of bacterial pathogens in novel hosts. Phil. Trans. R. Soc. B
374, 20180328. ( 10.1098/rstb.2018.0328)
</cite> [<a href="https://doi.org/10.1098/rstb.2018.0328" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6711297/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31401968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phil.%20Trans.%20R.%20Soc.%20B&amp;title=Understanding%20the%20emergence%20of%20bacterial%20pathogens%20in%20novel%20hosts&amp;volume=374&amp;publication_year=2019&amp;pages=20180328&amp;pmid=31401968&amp;doi=10.1098/rstb.2018.0328&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B94">
<span class="label">94.</span><cite>
Bacigalupe R, Tormo-Mas MÁ, Penadés JR, Fitzgerald JR. 2019. 
A multihost bacterial pathogen overcomes continuous population bottlenecks to adapt to new host species. Sci. Adv.
5, eaax0063. ( 10.1126/sciadv.aax0063)
</cite> [<a href="https://doi.org/10.1126/sciadv.aax0063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6881152/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31807698/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Adv.&amp;title=A%20multihost%20bacterial%20pathogen%20overcomes%20continuous%20population%20bottlenecks%20to%20adapt%20to%20new%20host%20species&amp;volume=5&amp;publication_year=2019&amp;pages=eaax0063&amp;pmid=31807698&amp;doi=10.1126/sciadv.aax0063&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B95">
<span class="label">95.</span><cite>
van Baarlen P, van Belkum A, Summerbell RC, Crous PW, Thomma B. 2007. 
Molecular mechanisms of pathogenicity: how do pathogenic microorganisms develop cross-kingdom host jumps?
FEMS Microbiol. Rev.
31, 239–277. ( 10.1111/j.1574-6976.2007.00065.x)
</cite> [<a href="https://doi.org/10.1111/j.1574-6976.2007.00065.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17326816/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Rev.&amp;title=Molecular%20mechanisms%20of%20pathogenicity:%20how%20do%20pathogenic%20microorganisms%20develop%20cross-kingdom%20host%20jumps?&amp;volume=31&amp;publication_year=2007&amp;pages=239-277&amp;pmid=17326816&amp;doi=10.1111/j.1574-6976.2007.00065.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B96">
<span class="label">96.</span><cite>
Sun H, Hanson MA, Walsh SK, Imrie RM, Raymond B, Longdon B. 2024. 
Varying phylogenetic signal in susceptibility to four bacterial pathogens across species of Drosophilidae. Figshare. ( 10.6084/m9.figshare.25648563)</cite> [<a href="https://doi.org/10.6084/m9.figshare.25648563" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12001086/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40237085/" class="usa-link">PubMed</a>]</li>
<li id="B97">
<span class="label">97.</span><cite>
Sun H, Hanson MA, Walsh SK, Imrie RM, Raymond B, Longdon B. 2025. 
Supplementary material from: Varying phylogenetic signal in susceptibility to four bacterial pathogens across species of Drosophilidae. Figshare ( 10.6084/m9.figshare.c.7728436)</cite> [<a href="https://doi.org/10.6084/m9.figshare.c.7728436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12001086/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40237085/" class="usa-link">PubMed</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data files and R scripts for analysis and visualization used in this study are available in an online repository [<a href="#B96" class="usa-link" aria-describedby="B96">96</a>].</p>
<p>Supplementary material is available online [<a href="#B97" class="usa-link" aria-describedby="B97">97</a>].</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Proceedings of the Royal Society B: Biological Sciences are provided here courtesy of <strong>The Royal Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1098/rspb.2024.2239"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/rspb.2024.2239.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (7.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12001086/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12001086/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12001086%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12001086/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12001086/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12001086/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40237085/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12001086/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40237085/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12001086/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12001086/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="CHfdt0cF2dRDHatjT6oqEqjBN9TD4vgr2bd1rqcBlVdyunL11JuhXVcF7MwRAMc2">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
