
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Cryo‐EM Structures of Native Chromatin Units From Human Cells - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="68F7DE7A80950C2300DE7A0040412F56.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11995444/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Genes to Cells">
<meta name="citation_title" content="Cryo‐EM Structures of Native Chromatin Units From Human Cells">
<meta name="citation_author" content="Suguru Hatazawa">
<meta name="citation_author_institution" content="Laboratory of Chromatin Structure and Function, Institute for Quantitative Biosciences, The University of Tokyo, Tokyo, Japan">
<meta name="citation_author" content="Yoshiyuki Fukuda">
<meta name="citation_author_institution" content="Division of Molecular CytoMorphology, Institute of Advanced Medical Sciences, Tokushima University, Tokushima, Tokushima, Japan">
<meta name="citation_author" content="Yuki Kobayashi">
<meta name="citation_author_institution" content="Laboratory of Chromatin Structure and Function, Institute for Quantitative Biosciences, The University of Tokyo, Tokyo, Japan">
<meta name="citation_author" content="Lumi Negishi">
<meta name="citation_author_institution" content="Laboratory of Chromatin Structure and Function, Institute for Quantitative Biosciences, The University of Tokyo, Tokyo, Japan">
<meta name="citation_author" content="Masahide Kikkawa">
<meta name="citation_author_institution" content="Department of Cell Biology and Anatomy, Graduate School of Medicine, The University of Tokyo, Tokyo, Japan">
<meta name="citation_author" content="Yoshimasa Takizawa">
<meta name="citation_author_institution" content="Laboratory of Chromatin Structure and Function, Institute for Quantitative Biosciences, The University of Tokyo, Tokyo, Japan">
<meta name="citation_author_institution" content="Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Tokyo, Japan">
<meta name="citation_author" content="Hitoshi Kurumizaka">
<meta name="citation_author_institution" content="Laboratory of Chromatin Structure and Function, Institute for Quantitative Biosciences, The University of Tokyo, Tokyo, Japan">
<meta name="citation_author_institution" content="Department of Biological Sciences, Graduate School of Science, The University of Tokyo, Tokyo, Japan">
<meta name="citation_author_institution" content="RIKEN Center for Biosystems Dynamics Research, Yokohama, Japan">
<meta name="citation_publication_date" content="2025 Apr 14">
<meta name="citation_volume" content="30">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70019">
<meta name="citation_doi" content="10.1111/gtc.70019">
<meta name="citation_pmid" content="40223781">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11995444/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11995444/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11995444/pdf/GTC-30-0.pdf">
<meta name="description" content="In eukaryotic cells, genomic DNA is compacted by nucleosomes, as basic repeating units, into chromatin. The nucleosome arrangement in chromatin fibers could be an important determinant for chromatin folding, by which genomic DNA is regulated in the ...">
<meta name="og:title" content="Cryo‐EM Structures of Native Chromatin Units From Human Cells">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="In eukaryotic cells, genomic DNA is compacted by nucleosomes, as basic repeating units, into chromatin. The nucleosome arrangement in chromatin fibers could be an important determinant for chromatin folding, by which genomic DNA is regulated in the ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11995444/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11995444">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1111/gtc.70019"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/GTC-30-0.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11995444%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11995444/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11995444/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11995444/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1111/gtc.70019" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Genes Cells</button></div>. 2025 Apr 14;30(3):e70019. doi: <a href="https://doi.org/10.1111/gtc.70019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1111/gtc.70019</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Genes%20Cells%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Genes%20Cells%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Genes%20Cells%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Genes%20Cells%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Cryo‐EM Structures of Native Chromatin Units From Human Cells</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hatazawa%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Suguru Hatazawa</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Suguru Hatazawa</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Laboratory of Chromatin Structure and Function, Institute for Quantitative Biosciences, The University of Tokyo, Tokyo, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hatazawa%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Suguru Hatazawa</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fukuda%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yoshiyuki Fukuda</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yoshiyuki Fukuda</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Division of Molecular CytoMorphology, Institute of Advanced Medical Sciences, Tokushima University, Tokushima, Tokushima, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fukuda%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yoshiyuki Fukuda</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kobayashi%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yuki Kobayashi</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yuki Kobayashi</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Laboratory of Chromatin Structure and Function, Institute for Quantitative Biosciences, The University of Tokyo, Tokyo, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kobayashi%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuki Kobayashi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Negishi%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Lumi Negishi</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Lumi Negishi</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Laboratory of Chromatin Structure and Function, Institute for Quantitative Biosciences, The University of Tokyo, Tokyo, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Negishi%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lumi Negishi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kikkawa%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Masahide Kikkawa</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Masahide Kikkawa</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Cell Biology and Anatomy, Graduate School of Medicine, The University of Tokyo, Tokyo, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kikkawa%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Masahide Kikkawa</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Takizawa%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Yoshimasa Takizawa</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Yoshimasa Takizawa</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Laboratory of Chromatin Structure and Function, Institute for Quantitative Biosciences, The University of Tokyo, Tokyo, Japan</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Tokyo, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Takizawa%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yoshimasa Takizawa</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kurumizaka%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Hitoshi Kurumizaka</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Hitoshi Kurumizaka</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Laboratory of Chromatin Structure and Function, Institute for Quantitative Biosciences, The University of Tokyo, Tokyo, Japan</div>
<div class="p">
<sup>
<sup>5</sup>
</sup>Department of Biological Sciences, Graduate School of Science, The University of Tokyo, Tokyo, Japan</div>
<div class="p">
<sup>
<sup>6</sup>
</sup>RIKEN Center for Biosystems Dynamics Research, Yokohama, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kurumizaka%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hitoshi Kurumizaka</span></a>
</div>
</div>
<sup>1,</sup><sup>5,</sup><sup>6,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="gtc70019-aff-0001">
<sup>
<sup>1</sup>
</sup>Laboratory of Chromatin Structure and Function, Institute for Quantitative Biosciences, The University of Tokyo, Tokyo, Japan</div>
<div id="gtc70019-aff-0002">
<sup>
<sup>2</sup>
</sup>Division of Molecular CytoMorphology, Institute of Advanced Medical Sciences, Tokushima University, Tokushima, Tokushima, Japan</div>
<div id="gtc70019-aff-0003">
<sup>
<sup>3</sup>
</sup>Department of Cell Biology and Anatomy, Graduate School of Medicine, The University of Tokyo, Tokyo, Japan</div>
<div id="gtc70019-aff-0004">
<sup>
<sup>4</sup>
</sup>Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Tokyo, Japan</div>
<div id="gtc70019-aff-0005">
<sup>
<sup>5</sup>
</sup>Department of Biological Sciences, Graduate School of Science, The University of Tokyo, Tokyo, Japan</div>
<div id="gtc70019-aff-0006">
<sup>
<sup>6</sup>
</sup>RIKEN Center for Biosystems Dynamics Research, Yokohama, Japan</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong>
 
Yoshimasa Takizawa (<span>ytakizawa@iqb.u-tokyo.ac.jp</span>), 
Hitoshi Kurumizaka (<span>kurumizaka@iqb.u-tokyo.ac.jp</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Apr 2; Received 2025 Mar 18; Accepted 2025 Apr 2; Issue date 2025 May.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Genes to Cells</em> published by Molecular Biology Society of Japan and John Wiley &amp; Sons Australia, Ltd.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11995444  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40223781/" class="usa-link">40223781</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>In eukaryotic cells, genomic DNA is compacted by nucleosomes, as basic repeating units, into chromatin. The nucleosome arrangement in chromatin fibers could be an important determinant for chromatin folding, by which genomic DNA is regulated in the nucleus. To study the structures of chromatin units in cells, we have established a method for the structural analysis of native mono‐ and poly‐nucleosomes prepared from HeLa cells. In this method, the chromatin in isolated nuclei was crosslinked to preserve the proximity information between nucleosomes, followed by chromatin fragmentation by micrococcal nuclease treatment. The mono‐ and poly‐nucleosomes were then fractionated by sucrose gradient ultracentrifugation, and their structures were analyzed by cryo‐electron microscopy. Cryo‐electron microscopy single particle analysis and cryo‐electron tomography visualized a native nucleosome structure and secondary nucleosome arrangements in cellular chromatin. This method provides a complementary strategy to fill the gap between in vitro and in situ analyses of chromatin structure.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> chromatin, cryo‐electron tomography, cryo‐EM, nucleosome, single particle analysis</p></section></section><section class="abstract" id="abstract2"><hr class="headless">
<p>We have established a method for the cryo‐EM structural analysis of native mono‐ and poly‐nucleosomes extracted from human cells. Single particle cryo‐EM and cryo‐electron tomography analyses have visualized a native nucleosome structure and secondary nucleosome arrangements in cellular chromatin.</p>
<section class="bt xbox font-sm" id="gtc70019-blkfxd-0001"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a37/11995444/a3585ed0b31f/GTC-30-0-g003.jpg" loading="lazy" id="jats-graphic-1" height="208" width="390" alt="graphic file with name GTC-30-0-g003.jpg"></p></section></section><section id="gtc70019-sec-0001"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>In eukaryotes, the genome is compacted as chromatin, in which nucleosomes are the basic architectural units (Olins and Olins <a href="#gtc70019-bib-0026" class="usa-link" aria-describedby="gtc70019-bib-0026">1974</a>; Kornberg <a href="#gtc70019-bib-0020" class="usa-link" aria-describedby="gtc70019-bib-0020">1974</a>). In the nucleosome, the four histones H2A, H2B, H3, and H4 form heterodimers of H2A–H2B and H3–H4, and two H2A–H2B and H3–H4 dimers associate to form the histone octamer (Richmond et al. <a href="#gtc70019-bib-0029" class="usa-link" aria-describedby="gtc70019-bib-0029">1984</a>; Arents et al. <a href="#gtc70019-bib-0002" class="usa-link" aria-describedby="gtc70019-bib-0002">1991</a>; Luger et al. <a href="#gtc70019-bib-0023" class="usa-link" aria-describedby="gtc70019-bib-0023">1997</a>). This octamer is left‐handedly wrapped with 145–147 base pairs of DNA in the nucleosome (Richmond et al. <a href="#gtc70019-bib-0029" class="usa-link" aria-describedby="gtc70019-bib-0029">1984</a>; Luger et al. <a href="#gtc70019-bib-0023" class="usa-link" aria-describedby="gtc70019-bib-0023">1997</a>).</p>
<p>Chromatin was historically thought to be folded into regular, hierarchical higher‐order structures. The 30 nm fiber was proposed as a model for higher‐order chromatin architecture, as supported by in vitro reconstitution studies under high‐salt conditions (Finch and Klug <a href="#gtc70019-bib-0014" class="usa-link" aria-describedby="gtc70019-bib-0014">1976</a>; Woodcock et al. <a href="#gtc70019-bib-0040" class="usa-link" aria-describedby="gtc70019-bib-0040">1984</a>). Recent technological advances have challenged this classical view. Pioneering cryo‐electron microscopy (cryo‐EM) studies by Eltsov et al. (<a href="#gtc70019-bib-0011" class="usa-link" aria-describedby="gtc70019-bib-0011">2008</a>) reported an absence of regular 30 nm fibers in mitotic chromatin (Eltsov et al. <a href="#gtc70019-bib-0011" class="usa-link" aria-describedby="gtc70019-bib-0011">2008</a>). Consistently, the 30 nm fibers were not dominantly observed in human cells when the DNA paths of chromosomes were traced by electron microscopy tomography with a DNA staining fluorescent dye (ChromEMT) (Ou et al. <a href="#gtc70019-bib-0027" class="usa-link" aria-describedby="gtc70019-bib-0027">2017</a>). In situ structural analyses of nucleosomes and poly‐nucleosomes by cryo‐electron tomography (cryo‐ET) have also revealed that chromatin fibers adopt various configurations in the nucleus, and the native and reconstituted nucleosome structures are basically similar (Cai, Böck, et al. <a href="#gtc70019-bib-0006" class="usa-link" aria-describedby="gtc70019-bib-0006">2018</a>). A structural study of isolated mitotic chromosomes by cryo‐ET has also been reported (Beel et al. <a href="#gtc70019-bib-0004" class="usa-link" aria-describedby="gtc70019-bib-0004">2021</a>). However, in these structural analyses with native chromatin fibers, the higher‐order chromatin structure, reflecting the 30 nm fiber, may not be the dominant structure.</p>
<p>These findings are consistent with the previous X‐ray scattering data for native cellular chromatin, again suggesting that the 30 nm fiber may not be the major higher‐order structure of chromatin in mammalian cells (Nishino et al. <a href="#gtc70019-bib-0025" class="usa-link" aria-describedby="gtc70019-bib-0025">2012</a>). Quantitative super‐resolution nanoscopy coupled with computer simulations has visualized “nucleosome clutches,” in which several nucleosomes are heterogeneously associated in the nucleus (Ricci et al. <a href="#gtc70019-bib-0028" class="usa-link" aria-describedby="gtc70019-bib-0028">2015</a>). These results reinforce the emerging view that chromatin exists in a more dynamic and irregular state under physiological conditions.</p>
<p>Using cryo‐focused ion beam (FIB) milling followed by cryo‐ET, chromatin has been visualized in situ in human interphase nuclei (Hou et al. <a href="#gtc70019-bib-0019" class="usa-link" aria-describedby="gtc70019-bib-0019">2023</a>). This approach enables the visualization of chromatin in its native context and has consistently shown that regular 30 nm fibers are not the dominant form of chromatin organization in cells. FIB milling‐based cryo‐ET is powerful but technically demanding and difficult to reproduce, limiting its widespread application.</p>
<p>To overcome these constraints, we established an alternative method for the structural analysis of native chromatin. We prepared native mono‐ and poly‐nucleosomes from human cells under conditions with crosslinking, which preserve the information on spatial association between proximal nucleosomes. We then determined the structures of native human nucleosomes and chromatin units by cryo‐EM single particle analysis and tomography, respectively. This approach provides structural insights comparable to in situ analyses, while being technically more accessible.</p></section><section id="gtc70019-sec-0002"><h2 class="pmc_sec_title">2. Results</h2>
<section id="gtc70019-sec-0003"><h3 class="pmc_sec_title">2.1. Preparation of Native Chromatin Fragments From HeLa Cells</h3>
<p>To prepare chromatin fragments, nuclei were isolated from cultured HeLa S3 cells (Figure <a href="#gtc70019-fig-0001" class="usa-link">1A</a>). The isolated nuclei were treated with formaldehyde, which forms protein–protein and protein–DNA crosslinks in situ, and the native chromatin was crosslinked to preserve the information of spatial association between proximal nucleosomes. The chromatin was solubilized by sonication and then fragmented by micrococcal nuclease (MNase), which digests the linker regions between nucleosomes. The native chromatin fragments were fractionated by sucrose gradient ultracentrifugation. To assess the DNA lengths in the fractionated chromatin, samples were deproteinized, and the resulting DNA fragments were analyzed using agarose gel electrophoresis. As shown in Figure <a href="#gtc70019-fig-0001" class="usa-link">1B</a>, the chromatin fragments containing the DNA lengths corresponding to mono‐, di‐, tri‐, tetra‐, penta‐, and larger nucleosome(s) were separated into fractions. To confirm the nucleosome‐nucleosome crosslinking in situ, we analyzed these chromatin fractions without deproteinization (Figure <a href="#gtc70019-fig-0001" class="usa-link">1C</a>). As anticipated, all chromatin fractions, and even the mono‐nucleosome fraction, contained not only the four core histones but also additional proteins, forming large complexes (Figures <a href="#gtc70019-fig-0001" class="usa-link">1C</a> and <a href="#gtc70019-supitem-0001" class="usa-link">S1A–C</a>). In contrast, the mono‐nucleosome fraction without crosslinking did not form large complexes (Figure <a href="#gtc70019-supitem-0001" class="usa-link">S1D</a>). These results suggest that the in situ crosslinking preserves the nucleosome‐nucleosome interactions in nuclei. Therefore, we used chromatin fragments prepared from HeLa cell nuclei with crosslinking for further analysis of the mono‐ and poly‐nucleosome structures.</p>
<figure class="fig xbox font-sm" id="gtc70019-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a37/11995444/dbb39ee37d52/GTC-30-0-g002.jpg" loading="lazy" id="jats-graphic-3" height="600" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/gtc70019-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Isolation of chromatin fragments from HeLa cell nuclear extract. (A) Schematic illustration of chromatin fragment isolation. (B and C) Agarose gel electrophoretic analyses of chromatin fragments from HeLa cells. Chromatin samples fractionated by sucrose gradient ultracentrifugation were obtained from top to bottom (lanes 1–7) and analyzed by agarose gel electrophoresis with (B) or without (C) deproteinization. DNA was stained with ethidium bromide.</p></figcaption></figure></section><section id="gtc70019-sec-0004"><h3 class="pmc_sec_title">2.2. Cryo‐EM Structure of the Native Nucleosome</h3>
<p>We first performed the cryo‐EM single particle analysis with the HeLa cell nucleosome. We collected and concentrated the MNase‐digested mono‐nucleosome fraction (Figure <a href="#gtc70019-supitem-0001" class="usa-link">S1A</a>) and subjected it to cryo‐EM data collection (Figure <a href="#gtc70019-supitem-0001" class="usa-link">S2</a>). In the raw cryo‐EM images, most of the nucleosomes are closely associated, suggesting that the in situ crosslinking preserved the nucleosome‐nucleosome interactions, even in the mono‐nucleosome fraction (Figure <a href="#gtc70019-fig-0002" class="usa-link">2A</a>).</p>
<figure class="fig xbox font-sm" id="gtc70019-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a37/11995444/184a6fdc5801/GTC-30-0-g004.jpg" loading="lazy" id="jats-graphic-5" height="742" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/gtc70019-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cryo‐EM structure of the native nucleosome prepared from HeLa cell nuclear extract. (A) Representative cryo‐EM micrograph of native HeLa nucleosomes. The scale bar indicates 100 nm. (B) The cryo‐EM map of the native HeLa nucleosome. Front and side views are presented. (C) An atomic model of the reconstituted nucleosome (PDB ID: 7XD1) was superimposed on the cryo‐EM map of the native HeLa nucleosome. Front and side views are presented. (D) Comparison of the cryo‐EM structure of the native HeLa nucleosome (left) with the cryo‐EM structure of the reconstituted nucleosome with recombinant histones and a Widom 601 DNA fragment (EMD‐33132, right). (E) Representative cryo‐EM micrograph of native HeLa poly‐nucleosomes (Figure <a href="#gtc70019-fig-0001" class="usa-link">1B</a>, fraction 6). The scale bar indicates 100 nm. (F) Cryo‐EM maps of the native HeLa nucleosome structures obtained from poly‐nucleosomes. Front and side views are presented. The three classes of cryo‐EM maps show different linker DNA orientations.</p></figcaption></figure><p>We then determined the cryo‐EM structure of the HeLa nucleosome by single particle analysis with 1,050,958 particles, at 3.1 Å resolution (Figure <a href="#gtc70019-fig-0002" class="usa-link">2B</a> and Table <a href="#gtc70019-supitem-0001" class="usa-link">S1</a>). In the HeLa nucleosome, the structures of the histones and DNA fit very well with the reconstituted nucleosome structure with recombinant histones and an artificial DNA sequence (PDB ID: 7XD1) (Ai et al. <a href="#gtc70019-bib-0001" class="usa-link" aria-describedby="gtc70019-bib-0001">2022</a>) (Figure <a href="#gtc70019-fig-0002" class="usa-link">2C</a>). However, the entry‐exit DNA regions of the HeLa nucleosome were less obvious as compared to those of the reconstituted nucleosome (EMD‐33132) (Ai et al. <a href="#gtc70019-bib-0001" class="usa-link" aria-describedby="gtc70019-bib-0001">2022</a>) (Figure <a href="#gtc70019-fig-0002" class="usa-link">2D</a>), suggesting that the entry‐exit DNA regions are more flexible and/or structurally diverse in cells. Indeed, we observed different classes of nucleosome structures in which the entry‐exit regions of the nucleosomal DNA were apparently distinct (Figure <a href="#gtc70019-supitem-0001" class="usa-link">S2B</a>). The structural diversity of the nucleosomal DNA around the entry‐exit regions was also detected in the mono‐nucleosome structures obtained from the single particle analysis with the poly‐nucleosomes (Figures <a href="#gtc70019-fig-0002" class="usa-link">2E,F</a> and <a href="#gtc70019-supitem-0001" class="usa-link">S3</a>). These results are consistent with previous cryo‐EM analyses in human and yeast cells (Cai, Böck, et al. <a href="#gtc70019-bib-0006" class="usa-link" aria-describedby="gtc70019-bib-0006">2018</a>; Cai, Song, et al. <a href="#gtc70019-bib-0007" class="usa-link" aria-describedby="gtc70019-bib-0007">2018</a>).</p></section><section id="gtc70019-sec-0005"><h3 class="pmc_sec_title">2.3. Structures of Native Poly‐Nucleosomes</h3>
<p>We next performed cryo‐ET with poly‐nucleosomes from HeLa cells. To do so, we used penta‐nucleosome and larger HeLa chromatin fragments (Figure <a href="#gtc70019-fig-0001" class="usa-link">1B</a>, fraction 6). We then obtained cryo‐EM images of native chromatin fragments from HeLa cells and detected chromatin fibers containing various distributions of nucleosome arrangements (Figure <a href="#gtc70019-fig-0003" class="usa-link">3A</a>). In these chromatin fragments, most of the nucleosomes probably contact the nearest nucleosome by in situ crosslinking, suggesting that these nucleosome arrangements may represent those in the cellular nucleus.</p>
<figure class="fig xbox font-sm" id="gtc70019-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a37/11995444/eb9adb79a206/GTC-30-0-g001.jpg" loading="lazy" id="jats-graphic-7" height="704" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/gtc70019-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cryo‐EM structures of native chromatin fragments and subtomogram average of HeLa nucleosome. (A) Representative z‐slice of the tomogram. The scale bar indicates 100 nm. (B) Subtomogram average of the HeLa nucleosome from 1334 particles in five tomograms. (C) Superimposed views of the atomic model of the reconstituted nucleosome (PDB ID: 7VZ4) onto the subtomogram average of the HeLa nucleosome.</p></figcaption></figure><p>To obtain an EM map of nucleosome from the tomograms, subtomogram averaging of nucleosome particles detected in tomograms was performed (Figures <a href="#gtc70019-fig-0003" class="usa-link">3B</a> and <a href="#gtc70019-supitem-0001" class="usa-link">S4</a>). Similar to the HeLa nucleosome structures observed from the single particle analysis, the subtomogram average of HeLa nucleosomes fitted well with the atomic model of reconstructed nucleosomes (Figure <a href="#gtc70019-fig-0003" class="usa-link">3C</a>). To further elucidate the spatial association of nucleosomes in poly‐nucleosomes, the subtomogram averaged structure was pasted back into tomograms at the coordinates detected by a template matching method (Figures <a href="#gtc70019-fig-0004" class="usa-link">4A,B</a> and <a href="#gtc70019-supitem-0001" class="usa-link">S4</a>). The nucleosome particles localized within a 15 nm distance were classed, and each class containing three or more particles was displayed to visualize nucleosome clusters (Figures <a href="#gtc70019-fig-0004" class="usa-link">4C</a> and <a href="#gtc70019-supitem-0001" class="usa-link">S5</a>).</p>
<figure class="fig xbox font-sm" id="gtc70019-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a37/11995444/64af727fed14/GTC-30-0-g005.jpg" loading="lazy" id="jats-graphic-9" height="787" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/gtc70019-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>3D organization of nucleosomes in poly‐nucleosomes. (A) Images overlaying z‐slices of the tomogram with mapping back the HeLa nucleosomes in the original tomogram. (B) Mapping back all HeLa nucleosomes in the original tomogram. (C) Mapping back the clustered HeLa nucleosomes in the original tomogram. Each class of nucleosomes is colored according to the number of particles in the cluster.</p></figcaption></figure><p>In previous studies, it was suggested that every other nucleosome is stacked on the disk‐like histone surface in the 30 nm fiber‐like poly‐nucleosomes, and the nucleosomes are connected by linker DNAs with a zig‐zag configuration (Schalch et al. <a href="#gtc70019-bib-0031" class="usa-link" aria-describedby="gtc70019-bib-0031">2005</a>; Song et al. <a href="#gtc70019-bib-0034" class="usa-link" aria-describedby="gtc70019-bib-0034">2014</a>; Takizawa et al. <a href="#gtc70019-bib-0037" class="usa-link" aria-describedby="gtc70019-bib-0037">2020</a>). Therefore, in this study, we defined poly‐nucleosomes as structures containing a stacked nucleosome unit with the zig‐zag configuration. In the cryo‐EM analysis of native chromatin, a structure that could fit with the zig‐zag configuration by nucleosome stacking was detected (Figure <a href="#gtc70019-fig-0005" class="usa-link">5A</a>). We also visualized heterogeneous nucleosome associations, possibly reflecting the nucleosome clutches (Figure <a href="#gtc70019-fig-0005" class="usa-link">5B</a>). In contrast, chromatin segments formed by nucleosome association were rarely observed in the cryo‐EM specimen without crosslinking (Figure <a href="#gtc70019-supitem-0001" class="usa-link">S6</a>), suggesting that the crosslinking preserves both the fiber‐like and clutch‐like structures of poly‐nucleosomes in the fractionated chromatin samples. The fiber‐like and clutch‐like chromatin architectures were also visualized in a previous in situ cryo‐ET analysis with human interphase nuclei (Hou et al. <a href="#gtc70019-bib-0019" class="usa-link" aria-describedby="gtc70019-bib-0019">2023</a>). Therefore, these structures may provide the proximity information between nucleosomes preserved by crosslinking and could represent the fundamental secondary units of chromatin architecture in eukaryotic cells.</p>
<figure class="fig xbox font-sm" id="gtc70019-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a37/11995444/8157151223dd/GTC-30-0-g006.jpg" loading="lazy" id="jats-graphic-11" height="362" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/gtc70019-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Enlarged views of the mapping back from the tomograms. (A) Fiber‐like and (B) nucleosome clutch‐like structures are surrounded by green and blue dotted lines, respectively.</p></figcaption></figure><p>Notably, the poly‐nucleosome fraction analyzed by cryo‐EM contained linker histone H1 (Figure <a href="#gtc70019-supitem-0001" class="usa-link">S1C</a>). However, the H1 density could not be clearly observed in the actual tomograms of chromatin, probably due to its flexibility and/or fast exchange (Misteli et al. <a href="#gtc70019-bib-0024" class="usa-link" aria-describedby="gtc70019-bib-0024">2000</a>). Alternatively, the number of particles analyzed may have been insufficient to visualize H1 density. Moreover, the poly‐nucleosomes in this study were solubilized by MNase, which preferentially digests chromatin regions in a relatively open conformation. The absence of detectable H1 density may therefore reflect an open chromatin state, where H1 levels are low.</p></section></section><section id="gtc70019-sec-0006"><h2 class="pmc_sec_title">3. Discussion</h2>
<p>Nucleosome structures have been determined by X‐ray crystallography and cryo‐EM technology, but most contained artificial nucleosome positioning DNA sequences (Luger et al. <a href="#gtc70019-bib-0023" class="usa-link" aria-describedby="gtc70019-bib-0023">1997</a>; Tachiwana et al. <a href="#gtc70019-bib-0035" class="usa-link" aria-describedby="gtc70019-bib-0035">2010</a>, <a href="#gtc70019-bib-0036" class="usa-link" aria-describedby="gtc70019-bib-0036">2011</a>; Vasudevan et al. <a href="#gtc70019-bib-0039" class="usa-link" aria-describedby="gtc70019-bib-0039">2010</a>; Chua et al. <a href="#gtc70019-bib-0009" class="usa-link" aria-describedby="gtc70019-bib-0009">2012</a>, <a href="#gtc70019-bib-0010" class="usa-link" aria-describedby="gtc70019-bib-0010">2016</a>; Ai et al. <a href="#gtc70019-bib-0001" class="usa-link" aria-describedby="gtc70019-bib-0001">2022</a>). These artificial DNA sequences are advantageous for obtaining homogeneous samples, but they reinforce the DNA‐histone associations, especially at the entry‐exit regions of the nucleosome. In the present study, we determined the cryo‐EM structure of the native HeLa cell nucleosome at 3.1 Å resolution by single particle analysis (Figure <a href="#gtc70019-fig-0002" class="usa-link">2</a>). This HeLa nucleosome structure must represent an average structure of the various nucleosomes in cells (Figure <a href="#gtc70019-supitem-0001" class="usa-link">S2</a>). Despite this heterogeneity, the HeLa nucleosome structure is quite similar to that of the homogeneously reconstituted nucleosome, except for the entry‐exit DNA regions. This fact indicates that the previously reported structure of the nucleosome core is also highly conserved in native nucleosomes in human cells.</p>
<p>In contrast, the entry‐exit regions of the nucleosomal DNA are ambiguous in the native human nucleosome, as compared to those in the reconstituted nucleosome (Figure <a href="#gtc70019-fig-0002" class="usa-link">2D</a>). The ambiguity of these nucleosomal DNA regions may represent their structural versatility among native nucleosomes (Figures <a href="#gtc70019-fig-0002" class="usa-link">2F</a> and <a href="#gtc70019-supitem-0001" class="usa-link">S2</a>). Consistently, the structural variations around the entry‐exit DNA regions of native nucleosomes from 
<em>Xenopus laevis</em>
 egg extracts and in human and yeast cells were previously reported (Cai, Böck, et al. <a href="#gtc70019-bib-0006" class="usa-link" aria-describedby="gtc70019-bib-0006">2018</a>; Cai, Song, et al. <a href="#gtc70019-bib-0007" class="usa-link" aria-describedby="gtc70019-bib-0007">2018</a>; Arimura et al. <a href="#gtc70019-bib-0003" class="usa-link" aria-describedby="gtc70019-bib-0003">2021</a>; Hou et al. <a href="#gtc70019-bib-0019" class="usa-link" aria-describedby="gtc70019-bib-0019">2023</a>). The incorporation of histone variants reportedly enhances the flexibility of the nucleosomal DNA ends (Kurumizaka et al. <a href="#gtc70019-bib-0022" class="usa-link" aria-describedby="gtc70019-bib-0022">2021</a>). The entry‐exit regions of the nucleosomal DNA directly dictate the spatial locations of neighboring nucleosomes and may affect the higher‐order association of nucleosomes in chromatin (Takizawa et al. <a href="#gtc70019-bib-0038" class="usa-link" aria-describedby="gtc70019-bib-0038">2018</a>). The nucleosomal DNA end flexibility may also depend on DNA sequences. Further structural studies are required to clarify the contributions of DNA sequences and histone species to the chromatin architecture in cells.</p>
<p>Using a cryo‐ET technique, we determined the structures of chromatin fragments obtained from HeLa cells (Figures <a href="#gtc70019-fig-0003" class="usa-link">3</a>, <a href="#gtc70019-fig-0004" class="usa-link">4</a>, <a href="#gtc70019-fig-0005" class="usa-link">5</a>). In our structures of HeLa chromatin fragments, most of the nucleosomes contacted other nucleosomes, and accordingly, these chromatin architectures may preserve the contact information between nucleosomes in cells. The structures of the HeLa chromatin fragments are not homogeneous, suggesting the structural adaptability of the chromatin fibers in cells. We found fiber‐like and nucleosome clutch‐like architectures in the chromatin fragments (Figure <a href="#gtc70019-fig-0005" class="usa-link">5A,B</a>), which may be representative secondary structural elements for chromatin folding in the nucleus. It will be intriguing to study the structural basis for the formation of these secondary chromatin structures, which may depend on the DNA sequence, histone variants, histone modifications, the cell cycle, and/or stress responses.</p>
<p>The native poly‐nucleosome structures observed in this in vitro analysis are mostly consistent with those found in the nuclear lamella prepared by FIB milling and analyzed by scanning electron microscopy (Hou et al. <a href="#gtc70019-bib-0019" class="usa-link" aria-describedby="gtc70019-bib-0019">2023</a>). This fact indicates that our method will become a useful complementary method to analyze native chromatin architectures. Further studies of native chromatin structures are awaited.</p></section><section id="gtc70019-sec-0007"><h2 class="pmc_sec_title">4. Experimental Procedures</h2>
<section id="gtc70019-sec-0008"><h3 class="pmc_sec_title">4.1. Preparation of Mono‐ and Poly‐Nucleosomes From Human HeLa Cells</h3>
<p>HeLa S3 cells (ATCC, CCL‐2.2), grown in DMEM supplemented with 10% FBS at 37°C under a 5% CO<sub>2</sub> atmosphere, were collected from cell culture dishes using a cell scraper. HeLa nuclei were isolated using NE‐PER Nuclear and Cytoplasmic Extraction Reagents (Thermo Fisher Scientific) and crosslinked by 1% formaldehyde at 25°C for 5 min. The crosslinking reaction was quenched by adding 2 M glycine to a final concentration of 150 mM, and the sample was incubated at 25°C for 10 min. Fixed HeLa nuclei were collected by centrifugation at 16,000 × <em>g</em> at 4°C for 5 min and resuspended in 500 μL of buffer (10 mM Tris–HCl [pH 8.0], 200 mM KCl, 1 mM CaCl<sub>2</sub>, and 0.5% NP‐40). For the preparation of mono‐nucleosomes, HeLa chromatin from 1.4 × 10<sup>7</sup> cells was fragmented by sonication using a VP‐050 N ultrasonic homogenizer (TAITEC) at 20% PWM at room temperature for 18 s (3 s × 6 cycles) and then digested with 2510 units of MNase (New England Biolabs). For the preparation of poly‐nucleosomes, HeLa chromatin from 2.0 × 10<sup>7</sup> cells was fragmented by sonication as described above, but at 25% PWM at room temperature for 9 s (3 s × 3 cycles) and then digested with 50.2 units of MNase (New England Biolabs). MNase digestion was performed at 37°C for 40 min, and quenched by adding 0.5 M EDTA to a final concentration of 10 mM, and then the samples were centrifuged at 15,000 × <em>g</em> at 4°C for 5 min. The supernatants were applied onto the top of a 10%–50% sucrose density gradient (11.3 mL in an Ultra‐Clear Tube [Beckman Coulter, 14 × 89 mm], prepared in 10 mM HEPES‐NaOH [pH 7.5], 30 mM NaCl, and 1 mM dithiothreitol) and centrifuged at 35,000 rpm at 4°C for 21 h in an SW 41 Ti rotor (Beckman Coulter). After the centrifugation, 640–660 μL aliquots were collected from the top of the gradient and analyzed by 1.5% agarose gel electrophoresis in 1 × TAE buffer, with or without deproteinization, followed by ethidium bromide staining. For the deproteinization of HeLa chromatin, the fractions (20 μL each) were treated with 9.45 μg of Proteinase K (TAKARA) and 0.08% SDS and incubated at 37°C for 15 min. The fractions containing HeLa nucleosomes with similar DNA lengths were combined, and the buffer was exchanged using a PD‐10 column (Cytiva) to the final sample buffer (10 mM Tris–HCl [pH 7.5], 30 mM NaCl, and 1 mM dithiothreitol). The samples were concentrated with an Amicon Ultra‐4 centrifugal filter unit (Merck, 30,000 MWCO) and used for grid preparation. The protein components of the HeLa poly‐nucleosomes were separated by 16% SDS‐polyacrylamide gel electrophoresis, and the resulting bands were analyzed by mass spectrometry.</p></section><section id="gtc70019-sec-0009"><h3 class="pmc_sec_title">4.2. Preparation of Uncrosslinked Nucleosome Samples From Human HeLa Cells</h3>
<p>The nuclei of HeLa S3 cells (ATCC, CCL‐2.2) were isolated using NE‐PER Nuclear and Cytoplasmic Extraction Reagents (Thermo Fisher Scientific), and collected by centrifugation at 16,000 × <em>g</em>, at 4°C for 5 min. HeLa chromatin from 2 × 10<sup>7</sup> cells was resuspended in 500 μL of 10 mM Tris–HCl (pH 8.0) buffer, containing 200 mM KCl, 1 mM CaCl<sub>2</sub>, and 0.5% NP‐40, and digested with 50.2 units of MNase (New England Biolabs) at 37°C for 40 min. The MNase reaction was quenched by adding 0.5 M EDTA to a final concentration of 10 mM, and then the samples were centrifuged at 15,000 × <em>g</em>, at 4°C for 5 min. The subsequent processes (sucrose density gradient ultracentrifugation, fractionation, buffer exchange, and concentration of HeLa chromatin) were performed using the same methods described above.</p></section><section id="gtc70019-sec-0010"><h3 class="pmc_sec_title">4.3. Grid Preparation and Data Collection for Cryo‐EM Single Particle Analysis</h3>
<p>The HeLa nucleosome sample (2.5 μL) was applied to a glow‐discharged grid (Quantifoil R1.2/1.3 200‐mesh Cu). The grid was blotted for 8 s at 4°C under 100% humidity, and then plunged into liquid ethane using a Vitrobot Mark IV (Thermo Fisher Scientific). Cryo‐EM data of the HeLa nucleosome sample were collected using the EPU software (Thermo Fisher Scientific) on a Krios G4 microscope (Thermo Fisher Scientific) equipped with an energy‐filtered K3 detector, operating at 300 kV at a magnification of 81,000× (pixel size of 1.06 Å). For the collection of the HeLa mono‐nucleosome data, all images containing 50 movie frames were recorded with defocus ranging from −1.0 to −2.5 μm and a 10‐s exposure time in the electron counting mode, on an energy‐filtered K3 direct electron detector (Gatan), at a total dose of 58.508 electrons per Å<sup>2</sup>. For the collection of the HeLa nucleosomes in poly‐nucleosomes data, all images containing 40 movie frames were recorded with defocus ranging from −1.0 to −2.5 μm and a 10‐s exposure time in the electron counting mode, using an energy‐filtered K3 direct electron detector (Gatan), at a total dose of 58.094 electrons per Å<sup>2</sup>.</p></section><section id="gtc70019-sec-0011"><h3 class="pmc_sec_title">4.4. Structure Determination of HeLa Mono‐Nucleosome by Cryo‐EM Single Particle Analysis</h3>
<p>In total, 8586 movies were aligned by MOTIONCOR2 (Zheng et al. <a href="#gtc70019-bib-0041" class="usa-link" aria-describedby="gtc70019-bib-0041">2017</a>) with dose weighting, and the contrast transfer function (CTF) was estimated using CTFFIND4 (Rohou and Grigorieff <a href="#gtc70019-bib-0030" class="usa-link" aria-describedby="gtc70019-bib-0030">2015</a>) from digital micrographs. Based on good CTF fit correlation, 6910 micrographs were selected, and all subsequent processing was performed with Relion 3.1 (Zivanov et al. <a href="#gtc70019-bib-0042" class="usa-link" aria-describedby="gtc70019-bib-0042">2018</a>). After Laplacian of Gaussian (LoG)‐based auto‐picking, picked particles were subjected to 2D classification. Three 2D class averages with nucleosome structures were used as references for the following particle picking. After reference‐based auto‐picking, picked particles were subjected to two rounds of 2D classification to discard junk particles. The selected particles were then subjected to five rounds of 3D classification, using the crystal structure of a canonical nucleosome (PDB: 3LZ0) (Vasudevan et al. <a href="#gtc70019-bib-0039" class="usa-link" aria-describedby="gtc70019-bib-0039">2010</a>) as the reference model, with low‐pass filtering of 60 Å. The best 3D class was subjected to 3D auto‐refinement and used for training a Topaz (Bepler et al. <a href="#gtc70019-bib-0005" class="usa-link" aria-describedby="gtc70019-bib-0005">2019</a>) particle‐picking model. Using this model, 3,062,600 particles were picked and subjected to three rounds of 2D classification. After removing junk particles, 1,587,277 particles were subjected to four rounds of 3D classification, using the crystal structure of a human canonical nucleosome (PDB: 3AFA) (Tachiwana et al. <a href="#gtc70019-bib-0035" class="usa-link" aria-describedby="gtc70019-bib-0035">2010</a>) as the reference model, with low‐pass filtering of 60 Å, and 1,050,958 particles were selected and processed by Bayesian polishing and CTF refinement. The resulting particles were subjected to 3D auto‐refinement and post‐processing, and the final cryo‐EM map of the HeLa nucleosome with a resolution of 3.1 Å was obtained. The resolution of the HeLa nucleosome was estimated by the “gold standard” Fourier shell correlation (FSC) at an FSC = 0.143 (Scheres <a href="#gtc70019-bib-0032" class="usa-link" aria-describedby="gtc70019-bib-0032">2016</a>). The figures of the HeLa nucleosome were created by UCSF ChimeraX (Goddard et al. <a href="#gtc70019-bib-0016" class="usa-link" aria-describedby="gtc70019-bib-0016">2018</a>).</p></section><section id="gtc70019-sec-0012"><h3 class="pmc_sec_title">4.5. Structure Determination of HeLa Mono‐Nucleosome in Poly‐Nucleosomes by Cryo‐EM Single Particle Analysis</h3>
<p>In total, 8408 movies were aligned by MOTIONCOR2 (Zheng et al. <a href="#gtc70019-bib-0041" class="usa-link" aria-describedby="gtc70019-bib-0041">2017</a>) with dose weighting, and the CTF was estimated using CTFFIND4 (Rohou and Grigorieff <a href="#gtc70019-bib-0030" class="usa-link" aria-describedby="gtc70019-bib-0030">2015</a>) from digital micrographs. Based on good CTF fit correlation, 7504 micrographs were selected, and all subsequent processing was performed with Relion 3.1 (Zivanov et al. <a href="#gtc70019-bib-0042" class="usa-link" aria-describedby="gtc70019-bib-0042">2018</a>). After LoG‐based auto‐picking in 30 selected micrographs, picked particles were subjected to 2D classification. Three 2D class averages of nucleosome structures were used as references for reference‐based auto‐picking, and 3,778,210 particles were picked from 7504 micrographs and subjected to three rounds of 2D classification to discard junk particles. The 1,111,442 selected particles were subjected to four rounds of 3D classification, using the crystal structure of a canonical nucleosome (PDB: 3AFA) (Tachiwana et al. <a href="#gtc70019-bib-0035" class="usa-link" aria-describedby="gtc70019-bib-0035">2010</a>) as the reference model, with low‐pass filtering of 60 Å. Each of the three selected classes was subjected to 3D auto‐refinement and post‐processing, and the final cryo‐EM maps of the HeLa nucleosome classes 1, 2, and 3 were obtained. The resolutions of the HeLa nucleosome classes 1, 2, and 3 were estimated at 4.09, 3.89, and 3.64 Å, respectively, by the “gold standard” FSC at an FSC = 0.143 (Scheres <a href="#gtc70019-bib-0032" class="usa-link" aria-describedby="gtc70019-bib-0032">2016</a>). The figures of the HeLa nucleosomes were created by UCSF ChimeraX (Goddard et al. <a href="#gtc70019-bib-0016" class="usa-link" aria-describedby="gtc70019-bib-0016">2018</a>). In Figure <a href="#gtc70019-fig-0002" class="usa-link">2F</a>, HeLa nucleosome maps ad‐hoc low‐pass filtered at 3.6 Å are presented.</p></section><section id="gtc70019-sec-0013"><h3 class="pmc_sec_title">4.6. Cryo‐ET of Poly‐Nucleosomes</h3>
<p>As preparation for plunge‐freezing, HeLa poly‐nucleosomes were mixed with 20‐fold concentrated BSA Gold Tracer (6 nm, AURION) (in final sample buffer: 10 mM Tris–HCl [pH 7.5], 30 mM NaCl, and 1 mM dithiothreitol) at a 5:1 volume ratio. A 2.5 μL portion of the specimen was applied to a glow‐discharged grid (Quantifoil R1.2/1.3 200‐mesh Cu), which was plunge‐frozen in liquid ethane using a Vitrobot Mark IV (Thermo Fisher Scientific). The Vitrobot blotting chamber conditions were set to 4°C and 100% humidity, and the blot time was 6 s.</p>
<p>Cryo‐ET was performed on a Krios G4 microscope (Thermo Fisher Scientific), equipped with a Thermofisher Volta phase plate and a Gatan postcolumn energy filter, and operated at an accelerated voltage of 300 kV. Data were recorded using a direct detector camera (K3, Gatan) operated in the counting mode. Tilt‐series images were collected by a dose‐symmetric tilt scheme (Hagen et al. <a href="#gtc70019-bib-0017" class="usa-link" aria-describedby="gtc70019-bib-0017">2017</a>) using the Tomography 5 software (Thermo Fisher Scientific). In total, 41 tilt‐series images were collected with the following parameters: magnification 53,000×, resulting in a pixel size of 1.7 Å at the specimen level; tilt range of ±60°; tilt increment 3°; start tilt angle 0°; energy filter slit width 20 eV; and total dose ~126 e<sup>−</sup>/Å<sup>2</sup>. For data acquisition, images were collected with a Volta phase plate with −0.5 μm defocus.</p>
<p>Movie frames were aligned and summed with motion correction by the Inspect3D software (Thermo Fisher Scientific). The motion‐corrected tilt‐series images were aligned based on fiducial gold markers and localized using the IMOD package (Kremer et al. <a href="#gtc70019-bib-0021" class="usa-link" aria-describedby="gtc70019-bib-0021">1996</a>). The 8‐times binned tomograms (512 × 720 × 125 pixels volume) were then reconstructed by weighted back projection (WBP).</p></section><section id="gtc70019-sec-0014"><h3 class="pmc_sec_title">4.7. Template Matching</h3>
<p>A human nucleosome structure (PDB: 7VZ4) was downloaded from the Protein Data Bank (PDB), and the Chimera software (35) was used to prepare a density map with a voxel size of 1.7 Å. A template and a mask for template matching of nucleosomes were prepared by pytom‐match‐pick (Chaillet et al. <a href="#gtc70019-bib-0008" class="usa-link" aria-describedby="gtc70019-bib-0008">2023</a>). The template for the nucleosome was prepared from the generated density map of the human nucleosome structure by pytom_create_template.py with the parameters: output voxel size 13.6; center; CTF correction; defocus 0.5; amplitude contrast 0.07; voltage 300; Cs 2.7; flip phase invert; box size 64. The spherical mask for template matching was prepared by pytom_create_mask.py with the parameters: box size 64; voxel size 13.6; radius 6; sigma 1. The template and mask were used for template matching with eight times binned tomograms, using the parameter of angular search 7 in pytom‐match‐pick (Chaillet et al. <a href="#gtc70019-bib-0008" class="usa-link" aria-describedby="gtc70019-bib-0008">2023</a>). After the template matching, 12,693 particles were extracted from five tomograms. The coordinates of the extracted particles for eight times binned tomograms were calibrated for the unbinned tomograms.</p></section><section id="gtc70019-sec-0015"><h3 class="pmc_sec_title">4.8. Data Processing and Subtomogram Averaging of Poly‐Nucleosomes</h3>
<p>Initial defocus and phase shift estimation per tilt image for subtomogram averaging were performed by CTFFIND4 (Rohou and Grigorieff <a href="#gtc70019-bib-0030" class="usa-link" aria-describedby="gtc70019-bib-0030">2015</a>). The particle coordinates of the nucleosome picked by PyTom template matching were imported to RELION‐4.0.1 (Zivanov et al. <a href="#gtc70019-bib-0043" class="usa-link" aria-describedby="gtc70019-bib-0043">2022</a>). The alignment of particles was carried out by RELION‐4.0.1 (Zivanov et al. <a href="#gtc70019-bib-0043" class="usa-link" aria-describedby="gtc70019-bib-0043">2022</a>). The detailed workflow of subtomogram averaging is shown in Figure <a href="#gtc70019-supitem-0001" class="usa-link">S4</a>. Following the initial 3D classification, 8545 particles were selected and subjected to refinement for mapping back (Figures <a href="#gtc70019-fig-0004" class="usa-link">4</a> and <a href="#gtc70019-supitem-0001" class="usa-link">S4</a>, right column). For fitting the nucleosome atomic model, 1334 particles were selected to obtain a 3D volume through several rounds of 3D classification (Figures <a href="#gtc70019-fig-0003" class="usa-link">3B,C</a> and <a href="#gtc70019-supitem-0001" class="usa-link">S4</a>, left column). The pixel size of the nucleosome volumes was adjusted to 1.63 Å/pix to match the scale of the nucleosome atomic model.</p></section><section id="gtc70019-sec-0016"><h3 class="pmc_sec_title">4.9. Mapping Back of Nucleosomes in the Tomograms</h3>
<p>Mapping back of nucleosomes was performed using ArtiaX in ChimeraX (Ermel et al. <a href="#gtc70019-bib-0012" class="usa-link" aria-describedby="gtc70019-bib-0012">2022</a>). The clustering of nucleosome coordinates in the tomograms was conducted based on the DBSCAN algorithm (Ester et al. <a href="#gtc70019-bib-0013" class="usa-link" aria-describedby="gtc70019-bib-0013">1996</a>). Particles located within a radius of 15 nm were grouped into the same class, and classes containing three or more particles were visualized in Figures <a href="#gtc70019-fig-0004" class="usa-link">4</a> and <a href="#gtc70019-supitem-0001" class="usa-link">S5</a>.</p></section><section id="gtc70019-sec-0017"><h3 class="pmc_sec_title">4.10. Protein Identification by LC–MS/MS Analysis</h3>
<p>HeLa chromatin fragments were separated by SDS‐PAGE and visualized with Oriole fluorescent gel stain (Bio‐Rad). Target proteins in the excised gel slices were in‐gel digested with trypsin and identified by nano LC–MS/MS analysis, following a modified procedure from previous studies (Shevchenko et al. <a href="#gtc70019-bib-0033" class="usa-link" aria-describedby="gtc70019-bib-0033">1996</a>; Funato et al. <a href="#gtc70019-bib-0015" class="usa-link" aria-describedby="gtc70019-bib-0015">2018</a>; Hatazawa et al. <a href="#gtc70019-bib-0018" class="usa-link" aria-describedby="gtc70019-bib-0018">2022</a>). The LC–MS/MS analysis was conducted using an LTQ‐Orbitrap Fusion mass spectrometer equipped with an Ultimate3000 nano LC containing an autosampler (Thermo Fisher Scientific). The nano LC gradient was delivered at 300 nL/min and consisted of a linear gradient from 2% to 40% acetonitrile over 60 min. The 10 most intense precursor ions were selected during MS1 scans and subsequently fragmented by higher‐energy collisional dissociation in MS2 scans. These data were searched against the 
<em>Homo sapiens</em>
 protein sequence database from UniProt through the search program application Proteome Discoverer 2.1 (Thermo Fisher Scientific) for protein identification. Spectra were searched with a precursor mass tolerance of 10 ppm and a fragment mass tolerance of 0.6 Da, allowing up to two missed cleavage sites. Carbamidomethylation of cysteine was set as a fixed modification, while oxidation of methionine and phosphorylation of serine, threonine, and tyrosine were set as variable modifications.</p></section><section id="gtc70019-sec-0018"><h3 class="pmc_sec_title">4.11. The Use of Large Language Models</h3>
<p>ChatGPT was used for grammatical corrections in the text and for providing the Python code to perform DBSCAN.</p></section></section><section id="gtc70019-sec-0023"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>
<strong>Suguru Hatazawa:</strong> methodology, formal analysis, investigation, resources, writing – review and editing, visualization, funding acquisition. <strong>Yoshiyuki Fukuda:</strong> methodology, formal analysis, writing – review and editing, visualization. <strong>Yuki Kobayashi:</strong> formal analysis, investigation, resources. <strong>Lumi Negishi:</strong> methodology, formal analysis, investigation. <strong>Masahide Kikkawa:</strong> writing – review and editing, funding acquisition, project administration. <strong>Yoshimasa Takizawa:</strong> formal analysis, writing – review and editing, data curation, funding acquisition. <strong>Hitoshi Kurumizaka:</strong> conceptualization, visualization, supervision, funding acquisition, writing – original draft, writing – review and editing, project administration.</p></section><section id="gtc70019-sec-0022"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec21"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="gtc70019-supitem-0001"><div class="caption p"><p>
<strong>Data S1.</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11995444/bin/GTC-30-0-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">GTC-30-0-s001.pdf</a><sup> (5.7MB, pdf) </sup>
</div></div></section></section><section id="gtc70019-sec-0021" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank Dr. Hiroaki Tachiwana (Cancer Institute of JFCR) for technical assistance with preparing HeLa nucleosomes and chromatin, Mr. Mitsuo Ogasawara (The University of Tokyo) for technical assistance with cryo‐EM data collection, and Ms. Yukari Iikura (The University of Tokyo), Ms. Yasuko Takeda (The University of Tokyo), and Ms. Yuki Fukaya (The University of Tokyo) for their assistance.</p></section><section id="notes1"><section id="gtc70019-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="gtc70019-note-0001"><p>
<strong>Funding:</strong> This work was supported by the Japan Agency for Medical Research and Development (JP24ama121002, JP24ama121009), the Japan Science and Technology Agency (JPMJCR24T3, JPMJER1901), and the Japan Society for the Promotion of Science (JP22K06098, JP23H05475, JP23K14134, JP24H02319, JP24H02328).</p></div>
<div class="fn p" id="gtc70019-note-0002"><p>
<strong>Transmitting Editor:</strong> Takehiko Kobayashi</p></div>
</div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Yoshimasa Takizawa, Email: ytakizawa@iqb.u-tokyo.ac.jp.</p>
<p>Hitoshi Kurumizaka, Email: kurumizaka@iqb.u-tokyo.ac.jp.</p></section><section id="gtc70019-sec-0020"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The cryo‐EM reconstructions of HeLa nucleosomes have been deposited in the Electron Microscopy Data Bank (EMDB) under the following accession codes: EMD‐63078, EMD‐63079, EMD‐63080, and EMD‐63081. The subtomogram‐averaged maps of HeLa nucleosomes in poly‐nucleosomes have been deposited in the EMDB under the following accession codes: EMD‐63144 and EMD‐63145.</p></section><section id="gtc70019-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="gtc70019-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="gtc70019-bib-0001">
<cite>

Ai, H.
, Sun M., Liu A., et al. 2022. “H2B Lys34 Ubiquitination Induces Nucleosome Distortion to Stimulate Dot1L Activity.” Nature Chemical Biology
18: 972–980. 10.1038/s41589-022-01067-7.
</cite> [<a href="https://doi.org/10.1038/s41589-022-01067-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35739357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Chemical%20Biology&amp;title=H2B%20Lys34%20Ubiquitination%20Induces%20Nucleosome%20Distortion%20to%20Stimulate%20Dot1L%20Activity&amp;author=H.%20Ai&amp;author=M.%20Sun&amp;author=A.%20Liu&amp;volume=18&amp;publication_year=2022&amp;pages=972-980&amp;pmid=35739357&amp;doi=10.1038/s41589-022-01067-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0002">
<cite>

Arents, G.
, Burlingame R. W., Wang B. C., Love W. E., and Moudrianakis E. N.. 1991. “The Nucleosomal Core Histone Octamer at 3.1 A Resolution: A Tripartite Protein Assembly and a Left‐Handed Superhelix.” Proceedings of the National Academy of Sciences of the United States of America
88, no. 22: 10148–10152. 10.1073/pnas.88.22.10148.
</cite> [<a href="https://doi.org/10.1073/pnas.88.22.10148" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC52885/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1946434/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=The%20Nucleosomal%20Core%20Histone%20Octamer%20at%203.1%20A%20Resolution:%20A%20Tripartite%20Protein%20Assembly%20and%20a%20Left%E2%80%90Handed%20Superhelix&amp;author=G.%20Arents&amp;author=R.%20W.%20Burlingame&amp;author=B.%20C.%20Wang&amp;author=W.%20E.%20Love&amp;author=E.%20N.%20Moudrianakis&amp;volume=88&amp;issue=22&amp;publication_year=1991&amp;pages=10148-10152&amp;pmid=1946434&amp;doi=10.1073/pnas.88.22.10148&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0003">
<cite>

Arimura, Y.
, Shih R. M., Froom R., and Funabiki H.. 2021. “Structural Features of Nucleosomes in Interphase and Metaphase Chromosomes.” Molecular Cell
81, no. 21: 4377–4397.e12. 10.1016/j.molcel.2021.08.010.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2021.08.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8571072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34478647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cell&amp;title=Structural%20Features%20of%20Nucleosomes%20in%20Interphase%20and%20Metaphase%20Chromosomes&amp;author=Y.%20Arimura&amp;author=R.%20M.%20Shih&amp;author=R.%20Froom&amp;author=H.%20Funabiki&amp;volume=81&amp;issue=21&amp;publication_year=2021&amp;pages=4377-4397.e12&amp;pmid=34478647&amp;doi=10.1016/j.molcel.2021.08.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0004">
<cite>

Beel, A. J.
, Azubel M., Matteï P. J., and Kornberg R. D.. 2021. “Structure of Mitotic Chromosomes.” Molecular Cell
81, no. 21: 4369–4376.e3. 10.1016/j.molcel.2021.08.020.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2021.08.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8571045/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34520722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cell&amp;title=Structure%20of%20Mitotic%20Chromosomes&amp;author=A.%20J.%20Beel&amp;author=M.%20Azubel&amp;author=P.%20J.%20Matte%C3%AF&amp;author=R.%20D.%20Kornberg&amp;volume=81&amp;issue=21&amp;publication_year=2021&amp;pages=4369-4376.e3&amp;pmid=34520722&amp;doi=10.1016/j.molcel.2021.08.020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0005">
<cite>

Bepler, T.
, Morin A., Rapp M., et al. 2019. “Positive‐Unlabeled Convolutional Neural Networks for Particle Picking in Cryo‐Electron Micrographs.” Nature Methods
16, no. 11: 1153–1160. 10.1038/s41592-019-0575-8.
</cite> [<a href="https://doi.org/10.1038/s41592-019-0575-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6858545/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31591578/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Positive%E2%80%90Unlabeled%20Convolutional%20Neural%20Networks%20for%20Particle%20Picking%20in%20Cryo%E2%80%90Electron%20Micrographs&amp;author=T.%20Bepler&amp;author=A.%20Morin&amp;author=M.%20Rapp&amp;volume=16&amp;issue=11&amp;publication_year=2019&amp;pages=1153-1160&amp;pmid=31591578&amp;doi=10.1038/s41592-019-0575-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0006">
<cite>

Cai, S.
, Böck D., Pilhofer M., and Gan L.. 2018. “The In Situ Structures of Mono‐, Di‐, and Trinucleosomes in Human Heterochromatin.” Molecular Biology of the Cell
29, no. 20: 2450–2457. 10.1091/mbc.E18-05-0331.
</cite> [<a href="https://doi.org/10.1091/mbc.E18-05-0331" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6233054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30091658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20of%20the%20Cell&amp;title=The%20In%C2%A0Situ%20Structures%20of%20Mono%E2%80%90,%20Di%E2%80%90,%20and%20Trinucleosomes%20in%20Human%20Heterochromatin&amp;author=S.%20Cai&amp;author=D.%20B%C3%B6ck&amp;author=M.%20Pilhofer&amp;author=L.%20Gan&amp;volume=29&amp;issue=20&amp;publication_year=2018&amp;pages=2450-2457&amp;pmid=30091658&amp;doi=10.1091/mbc.E18-05-0331&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0007">
<cite>

Cai, S.
, Song Y., Chen C., Shi J., and Gan L.. 2018. “Natural Chromatin Is Heterogeneous and Self‐Associates In Vitro.” Molecular Biology of the Cell
29, no. 13: 1652–1663. 10.1091/mbc.E17-07-0449.
</cite> [<a href="https://doi.org/10.1091/mbc.E17-07-0449" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6080658/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29742050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20of%20the%20Cell&amp;title=Natural%20Chromatin%20Is%20Heterogeneous%20and%20Self%E2%80%90Associates%20In%C2%A0Vitro&amp;author=S.%20Cai&amp;author=Y.%20Song&amp;author=C.%20Chen&amp;author=J.%20Shi&amp;author=L.%20Gan&amp;volume=29&amp;issue=13&amp;publication_year=2018&amp;pages=1652-1663&amp;pmid=29742050&amp;doi=10.1091/mbc.E17-07-0449&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0008">
<cite>

Chaillet, M. L.
, van der Schot G., Gubins I., Roet S., Veltkamp R. C., and Förster F.. 2023. “Extensive Angular Sampling Enables the Sensitive Localization of Macromolecules in Electron Tomograms.” International Journal of Molecular Sciences
24, no. 17: 13375. 10.3390/ijms241713375.
</cite> [<a href="https://doi.org/10.3390/ijms241713375" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10487639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37686180/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Extensive%20Angular%20Sampling%20Enables%20the%20Sensitive%20Localization%20of%20Macromolecules%20in%20Electron%20Tomograms&amp;author=M.%20L.%20Chaillet&amp;author=G.%20van%20der%20Schot&amp;author=I.%20Gubins&amp;author=S.%20Roet&amp;author=R.%20C.%20Veltkamp&amp;volume=24&amp;issue=17&amp;publication_year=2023&amp;pages=13375&amp;pmid=37686180&amp;doi=10.3390/ijms241713375&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0009">
<cite>

Chua, E. Y. D.
, Vasudevan D., Davey G. E., Wu B., and Davey C. A.. 2012. “The Mechanics Behind DNA Sequence‐Dependent Properties of the Nucleosome.” Nucleic Acids Research
40, no. 13: 6338–6352. 10.1093/nar/gks261.
</cite> [<a href="https://doi.org/10.1093/nar/gks261" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3401446/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22453276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=The%20Mechanics%20Behind%20DNA%20Sequence%E2%80%90Dependent%20Properties%20of%20the%20Nucleosome&amp;author=E.%20Y.%20D.%20Chua&amp;author=D.%20Vasudevan&amp;author=G.%20E.%20Davey&amp;author=B.%20Wu&amp;author=C.%20A.%20Davey&amp;volume=40&amp;issue=13&amp;publication_year=2012&amp;pages=6338-6352&amp;pmid=22453276&amp;doi=10.1093/nar/gks261&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0010">
<cite>

Chua, E. Y. D.
, Vogirala V. K., Inian O., et al. 2016. “3.9 Å Structure of the Nucleosome Core Particle Determined by Phase‐Plate Cryo‐EM.” Nucleic Acids Research
44, no. 17: 8013–8019. 10.1093/nar/gkw708.
</cite> [<a href="https://doi.org/10.1093/nar/gkw708" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5041491/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27563056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=3.9%20%C3%85%20Structure%20of%20the%20Nucleosome%20Core%20Particle%20Determined%20by%20Phase%E2%80%90Plate%20Cryo%E2%80%90EM&amp;author=E.%20Y.%20D.%20Chua&amp;author=V.%20K.%20Vogirala&amp;author=O.%20Inian&amp;volume=44&amp;issue=17&amp;publication_year=2016&amp;pages=8013-8019&amp;pmid=27563056&amp;doi=10.1093/nar/gkw708&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0011">
<cite>

Eltsov, M.
, MacLellan K. M., Maeshima K., Frangakis A. S., and Dubochet J.. 2008. “Analysis of Cryo‐Electron Microscopy Images Does Not Support the Existence of 30‐Nm Chromatin Fibers in Mitotic Chromosomes In Situ.” Proceedings of the National Academy of Sciences of the United States of America
105, no. 50: 19732–19737. 10.1073/pnas.0810057105.
</cite> [<a href="https://doi.org/10.1073/pnas.0810057105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2604964/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19064912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Analysis%20of%20Cryo%E2%80%90Electron%20Microscopy%20Images%20Does%20Not%20Support%20the%20Existence%20of%2030%E2%80%90Nm%20Chromatin%20Fibers%20in%20Mitotic%20Chromosomes%20In%C2%A0Situ&amp;author=M.%20Eltsov&amp;author=K.%20M.%20MacLellan&amp;author=K.%20Maeshima&amp;author=A.%20S.%20Frangakis&amp;author=J.%20Dubochet&amp;volume=105&amp;issue=50&amp;publication_year=2008&amp;pages=19732-19737&amp;pmid=19064912&amp;doi=10.1073/pnas.0810057105&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0012">
<cite>

Ermel, U. H.
, Arghittu S. M., and Frangakis A. S.. 2022. “ArtiaX: An Electron Tomography Toolbox for the Interactive Handling of Sub‐Tomograms in UCSF ChimeraX.” Protein Science
31, no. 12: 1–10. 10.1002/pro.4472.</cite> [<a href="https://doi.org/10.1002/pro.4472" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9667824/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36251681/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Science&amp;title=ArtiaX:%20An%20Electron%20Tomography%20Toolbox%20for%20the%20Interactive%20Handling%20of%20Sub%E2%80%90Tomograms%20in%20UCSF%20ChimeraX&amp;author=U.%20H.%20Ermel&amp;author=S.%20M.%20Arghittu&amp;author=A.%20S.%20Frangakis&amp;volume=31&amp;issue=12&amp;publication_year=2022&amp;pages=1-10&amp;pmid=36251681&amp;doi=10.1002/pro.4472&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0013">
<cite>

Ester, M.
, Kriegel H.‐P., Sander J., and Xu X.. 1996. “A Density‐Based Algorithm for Discovering Clusters in Large Spatial Databases With Noise.” Knowledge Discovery and Data Mining
96: 226–231.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Knowledge%20Discovery%20and%20Data%20Mining&amp;title=A%20Density%E2%80%90Based%20Algorithm%20for%20Discovering%20Clusters%20in%20Large%20Spatial%20Databases%20With%20Noise&amp;author=M.%20Ester&amp;author=H.%E2%80%90P.%20Kriegel&amp;author=J.%20Sander&amp;author=X.%20Xu&amp;volume=96&amp;publication_year=1996&amp;pages=226-231&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0014">
<cite>

Finch, J. T.
, and Klug A.. 1976. “Solenoidal Model for Superstructure in Chromatin.” Proceedings of the National Academy of Sciences of the United States of America
73, no. 6: 1897–1901. 10.1073/pnas.73.6.1897.
</cite> [<a href="https://doi.org/10.1073/pnas.73.6.1897" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC430414/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1064861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Solenoidal%20Model%20for%20Superstructure%20in%20Chromatin&amp;author=J.%20T.%20Finch&amp;author=A.%20Klug&amp;volume=73&amp;issue=6&amp;publication_year=1976&amp;pages=1897-1901&amp;pmid=1064861&amp;doi=10.1073/pnas.73.6.1897&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0015">
<cite>

Funato, K.
, Hayashi T., Echizen K., et al. 2018. “SIRT2‐Mediated Inactivation of p73 Is Required for Glioblastoma Tumorigenicity.” EMBO Reports
19, no. 11: e45587. 10.15252/embr.201745587.
</cite> [<a href="https://doi.org/10.15252/embr.201745587" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6216266/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30213795/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Reports&amp;title=SIRT2%E2%80%90Mediated%20Inactivation%20of%20p73%20Is%20Required%20for%20Glioblastoma%20Tumorigenicity&amp;author=K.%20Funato&amp;author=T.%20Hayashi&amp;author=K.%20Echizen&amp;volume=19&amp;issue=11&amp;publication_year=2018&amp;pages=e45587&amp;pmid=30213795&amp;doi=10.15252/embr.201745587&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0016">
<cite>

Goddard, T. D.
, Huang C. C., Meng E. C., et al. 2018. “UCSF ChimeraX: Meeting Modern Challenges in Visualization and Analysis.” Protein Science
27, no. 1: 14–25. 10.1002/pro.3235.
</cite> [<a href="https://doi.org/10.1002/pro.3235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5734306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28710774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Science&amp;title=UCSF%20ChimeraX:%20Meeting%20Modern%20Challenges%20in%20Visualization%20and%20Analysis&amp;author=T.%20D.%20Goddard&amp;author=C.%20C.%20Huang&amp;author=E.%20C.%20Meng&amp;volume=27&amp;issue=1&amp;publication_year=2018&amp;pages=14-25&amp;pmid=28710774&amp;doi=10.1002/pro.3235&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0017">
<cite>

Hagen, W. J. H.
, Wan W., and Briggs J. A. G.. 2017. “Implementation of a Cryo‐Electron Tomography Tilt‐Scheme Optimized for High Resolution Subtomogram Averaging.” Journal of Structural Biology
197, no. 2: 191–198. 10.1016/j.jsb.2016.06.007.
</cite> [<a href="https://doi.org/10.1016/j.jsb.2016.06.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5287356/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27313000/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Structural%20Biology&amp;title=Implementation%20of%20a%20Cryo%E2%80%90Electron%20Tomography%20Tilt%E2%80%90Scheme%20Optimized%20for%20High%20Resolution%20Subtomogram%20Averaging&amp;author=W.%20J.%20H.%20Hagen&amp;author=W.%20Wan&amp;author=J.%20A.%20G.%20Briggs&amp;volume=197&amp;issue=2&amp;publication_year=2017&amp;pages=191-198&amp;pmid=27313000&amp;doi=10.1016/j.jsb.2016.06.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0018">
<cite>

Hatazawa, S.
, Liu J., Takizawa Y., et al. 2022. “Structural Basis for Binding Diversity of Acetyltransferase p300 to the Nucleosome.” iScience
25, no. 7: 104563. 10.1016/j.isci.2022.104563.
</cite> [<a href="https://doi.org/10.1016/j.isci.2022.104563" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9218434/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35754730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=iScience&amp;title=Structural%20Basis%20for%20Binding%20Diversity%20of%20Acetyltransferase%20p300%20to%20the%20Nucleosome&amp;author=S.%20Hatazawa&amp;author=J.%20Liu&amp;author=Y.%20Takizawa&amp;volume=25&amp;issue=7&amp;publication_year=2022&amp;pages=104563&amp;pmid=35754730&amp;doi=10.1016/j.isci.2022.104563&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0019">
<cite>

Hou, Z.
, Nightingale F., Zhu Y., MacGregor‐Chatwin C., and Zhang P.. 2023. “Structure of Native Chromatin Fibres Revealed by Cryo‐ET In Situ.” Nature Communications
14, no. 1: 1–7. 10.1038/s41467-023-42072-1.</cite> [<a href="https://doi.org/10.1038/s41467-023-42072-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10564948/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37816746/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Structure%20of%20Native%20Chromatin%20Fibres%20Revealed%20by%20Cryo%E2%80%90ET%20In%C2%A0Situ&amp;author=Z.%20Hou&amp;author=F.%20Nightingale&amp;author=Y.%20Zhu&amp;author=C.%20MacGregor%E2%80%90Chatwin&amp;author=P.%20Zhang&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=1-7&amp;pmid=37816746&amp;doi=10.1038/s41467-023-42072-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0020">
<cite>

Kornberg, R. D.

1974. “Chromatin Structure: A Repeating Unit of Histones and DNA.” Science
184, no. 4139: 868–871. 10.1126/science.184.4139.868.
</cite> [<a href="https://doi.org/10.1126/science.184.4139.868" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4825889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Chromatin%20Structure:%20A%20Repeating%20Unit%20of%20Histones%20and%20DNA&amp;author=R.%20D.%20Kornberg&amp;volume=184&amp;issue=4139&amp;publication_year=1974&amp;pages=868-871&amp;pmid=4825889&amp;doi=10.1126/science.184.4139.868&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0021">
<cite>

Kremer, J. R.
, Mastronarde D. N., and McIntosh J. R.. 1996. “Computer Visualization of Three‐Dimensional Image Data Using IMOD.” Journal of Structural Biology
116, no. 1: 71–76. 10.1006/jsbi.1996.0013.
</cite> [<a href="https://doi.org/10.1006/jsbi.1996.0013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8742726/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Structural%20Biology&amp;title=Computer%20Visualization%20of%20Three%E2%80%90Dimensional%20Image%20Data%20Using%20IMOD&amp;author=J.%20R.%20Kremer&amp;author=D.%20N.%20Mastronarde&amp;author=J.%20R.%20McIntosh&amp;volume=116&amp;issue=1&amp;publication_year=1996&amp;pages=71-76&amp;pmid=8742726&amp;doi=10.1006/jsbi.1996.0013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0022">
<cite>

Kurumizaka, H.
, Kujirai T., and Takizawa Y.. 2021. “Contributions of Histone Variants in Nucleosome Structure and Function.” Journal of Molecular Biology
433, no. 6: 166678. 10.1016/j.jmb.2020.10.012.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2020.10.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33065110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Molecular%20Biology&amp;title=Contributions%20of%20Histone%20Variants%20in%20Nucleosome%20Structure%20and%20Function&amp;author=H.%20Kurumizaka&amp;author=T.%20Kujirai&amp;author=Y.%20Takizawa&amp;volume=433&amp;issue=6&amp;publication_year=2021&amp;pages=166678&amp;pmid=33065110&amp;doi=10.1016/j.jmb.2020.10.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0023">
<cite>

Luger, K.
, Mäder A. W., Richmond R. K., Sargent D. F., and Richmond T. J.. 1997. “Crystal Structure of the Nucleosome Core Particle at 2.8 Å Resolution.” Nature
389, no. 6648: 251–260. 10.1038/38444.
</cite> [<a href="https://doi.org/10.1038/38444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9305837/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Crystal%20Structure%20of%20the%20Nucleosome%20Core%20Particle%20at%202.8%20%C3%85%20Resolution&amp;author=K.%20Luger&amp;author=A.%20W.%20M%C3%A4der&amp;author=R.%20K.%20Richmond&amp;author=D.%20F.%20Sargent&amp;author=T.%20J.%20Richmond&amp;volume=389&amp;issue=6648&amp;publication_year=1997&amp;pages=251-260&amp;pmid=9305837&amp;doi=10.1038/38444&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0024">
<cite>

Misteli, T.
, Gunjan A., Hock R., Bustin M., and Brown D. T.. 2000. “Dynamic Binding of Histone H1 to Chromatin in Living Cells.” Nature
408, no. 6814: 877–881. 10.1038/35048610.
</cite> [<a href="https://doi.org/10.1038/35048610" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11130729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Dynamic%20Binding%20of%20Histone%20H1%20to%20Chromatin%20in%20Living%20Cells&amp;author=T.%20Misteli&amp;author=A.%20Gunjan&amp;author=R.%20Hock&amp;author=M.%20Bustin&amp;author=D.%20T.%20Brown&amp;volume=408&amp;issue=6814&amp;publication_year=2000&amp;pages=877-881&amp;pmid=11130729&amp;doi=10.1038/35048610&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0025">
<cite>

Nishino, Y.
, Eltsov M., Joti Y., et al. 2012. “Human Mitotic Chromosomes Consist Predominantly of Irregularly Folded Nucleosome Fibres Without a 30‐Nm Chromatin Structure.” EMBO Journal
31, no. 7: 1644–1653. 10.1038/emboj.2012.35.
</cite> [<a href="https://doi.org/10.1038/emboj.2012.35" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3321210/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22343941/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Journal&amp;title=Human%20Mitotic%20Chromosomes%20Consist%20Predominantly%20of%20Irregularly%20Folded%20Nucleosome%20Fibres%20Without%20a%2030%E2%80%90Nm%20Chromatin%20Structure&amp;author=Y.%20Nishino&amp;author=M.%20Eltsov&amp;author=Y.%20Joti&amp;volume=31&amp;issue=7&amp;publication_year=2012&amp;pages=1644-1653&amp;pmid=22343941&amp;doi=10.1038/emboj.2012.35&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0026">
<cite>

Olins, A. L.
, and Olins D. E.. 1974. “Spheroid Chromatin Units (v Bodies).” Science
183, no. 4122: 330–332. 10.1126/science.183.4122.330.
</cite> [<a href="https://doi.org/10.1126/science.183.4122.330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4128918/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Spheroid%20Chromatin%20Units%20(v%20Bodies)&amp;author=A.%20L.%20Olins&amp;author=D.%20E.%20Olins&amp;volume=183&amp;issue=4122&amp;publication_year=1974&amp;pages=330-332&amp;pmid=4128918&amp;doi=10.1126/science.183.4122.330&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0027">
<cite>

Ou, H. D.
, Phan S., Deerinck T. J., Thor A., Ellisman M. H., and O'Shea C. C.. 2017. “ChromEMT: Visualizing 3D Chromatin Structure and Compaction in Interphase and Mitotic Cells.” Science
357, no. 6349: eaag0025. 10.1126/science.aag0025.
</cite> [<a href="https://doi.org/10.1126/science.aag0025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5646685/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28751582/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=ChromEMT:%20Visualizing%203D%20Chromatin%20Structure%20and%20Compaction%20in%20Interphase%20and%20Mitotic%20Cells&amp;author=H.%20D.%20Ou&amp;author=S.%20Phan&amp;author=T.%20J.%20Deerinck&amp;author=A.%20Thor&amp;author=M.%20H.%20Ellisman&amp;volume=357&amp;issue=6349&amp;publication_year=2017&amp;pages=eaag0025&amp;pmid=28751582&amp;doi=10.1126/science.aag0025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0028">
<cite>

Ricci, M. A.
, Manzo C., García‐Parajo M. F., Lakadamyali M., and Cosma M. P.. 2015. “Chromatin Fibers Are Formed by Heterogeneous Groups of Nucleosomes In Vivo.” Cell
160, no. 6: 1145–1158. 10.1016/j.cell.2015.01.054.
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.01.054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25768910/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Chromatin%20Fibers%20Are%20Formed%20by%20Heterogeneous%20Groups%20of%20Nucleosomes%20In%C2%A0Vivo&amp;author=M.%20A.%20Ricci&amp;author=C.%20Manzo&amp;author=M.%20F.%20Garc%C3%ADa%E2%80%90Parajo&amp;author=M.%20Lakadamyali&amp;author=M.%20P.%20Cosma&amp;volume=160&amp;issue=6&amp;publication_year=2015&amp;pages=1145-1158&amp;pmid=25768910&amp;doi=10.1016/j.cell.2015.01.054&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0029">
<cite>

Richmond, T. J.
, Finch J. T., Rushton B., Rhodes D., and Klug A.. 1984. “Structure of the Nucleosome Core Particle at 7 Å Resolution.” Nature
311, no. 5986: 532–537. 10.1038/311532a0.
</cite> [<a href="https://doi.org/10.1038/311532a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6482966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Structure%20of%20the%20Nucleosome%20Core%20Particle%20at%207%20%C3%85%20Resolution&amp;author=T.%20J.%20Richmond&amp;author=J.%20T.%20Finch&amp;author=B.%20Rushton&amp;author=D.%20Rhodes&amp;author=A.%20Klug&amp;volume=311&amp;issue=5986&amp;publication_year=1984&amp;pages=532-537&amp;pmid=6482966&amp;doi=10.1038/311532a0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0030">
<cite>

Rohou, A.
, and Grigorieff N.. 2015. “CTFFIND4: Fast and Accurate Defocus Estimation From Electron Micrographs.” Journal of Structural Biology
192, no. 2: 216–221. 10.1016/j.jsb.2015.08.008.
</cite> [<a href="https://doi.org/10.1016/j.jsb.2015.08.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6760662/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26278980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Structural%20Biology&amp;title=CTFFIND4:%20Fast%20and%20Accurate%20Defocus%20Estimation%20From%20Electron%20Micrographs&amp;author=A.%20Rohou&amp;author=N.%20Grigorieff&amp;volume=192&amp;issue=2&amp;publication_year=2015&amp;pages=216-221&amp;pmid=26278980&amp;doi=10.1016/j.jsb.2015.08.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0031">
<cite>

Schalch, T.
, Duda S., Sargent D. F., and Richmond T. J.. 2005. “X‐Ray Structure of a Tetranucleosome and Its Implications for the Chromatin Fibre.” Nature
436, no. 7047: 138–141. 10.1038/nature03686.
</cite> [<a href="https://doi.org/10.1038/nature03686" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16001076/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=X%E2%80%90Ray%20Structure%20of%20a%20Tetranucleosome%20and%20Its%20Implications%20for%20the%20Chromatin%20Fibre&amp;author=T.%20Schalch&amp;author=S.%20Duda&amp;author=D.%20F.%20Sargent&amp;author=T.%20J.%20Richmond&amp;volume=436&amp;issue=7047&amp;publication_year=2005&amp;pages=138-141&amp;pmid=16001076&amp;doi=10.1038/nature03686&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0032">
<cite>

Scheres, S. H. W.

2016. “Processing of Structurally Heterogeneous Cryo‐EM Data in RELION.” In Methods in Enzymology, edited by Scheres S. H. W., vol. 579, 1st ed., 125–157. Elsevier Inc. 10.1016/bs.mie.2016.04.012.
</cite> [<a href="https://doi.org/10.1016/bs.mie.2016.04.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27572726/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20Enzymology&amp;author=S.%20H.%20W.%20Scheres&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0033">
<cite>

Shevchenko, A.
, Wilm M., Vorm O., and Mann M.. 1996. “Mass Spectrometric Sequencing of Proteins From Silver‐Stained Polyacrylamide Gels.” Analytical Chemistry
68, no. 5: 850–858. 10.1021/ac950914h.
</cite> [<a href="https://doi.org/10.1021/ac950914h" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8779443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Mass%20Spectrometric%20Sequencing%20of%20Proteins%20From%20Silver%E2%80%90Stained%20Polyacrylamide%20Gels&amp;author=A.%20Shevchenko&amp;author=M.%20Wilm&amp;author=O.%20Vorm&amp;author=M.%20Mann&amp;volume=68&amp;issue=5&amp;publication_year=1996&amp;pages=850-858&amp;pmid=8779443&amp;doi=10.1021/ac950914h&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0034">
<cite>

Song, F.
, Chen P., Sun D., et al. 2014. “Cryo‐EM Study of the Chromatin Fiber Reveals a Double Helix Twisted by Tetranucleosomal Units.” Science
344, no. 6182: 376–380. 10.1126/science.1251413.
</cite> [<a href="https://doi.org/10.1126/science.1251413" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24763583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Cryo%E2%80%90EM%20Study%20of%20the%20Chromatin%20Fiber%20Reveals%20a%20Double%20Helix%20Twisted%20by%20Tetranucleosomal%20Units&amp;author=F.%20Song&amp;author=P.%20Chen&amp;author=D.%20Sun&amp;volume=344&amp;issue=6182&amp;publication_year=2014&amp;pages=376-380&amp;pmid=24763583&amp;doi=10.1126/science.1251413&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0035">
<cite>

Tachiwana, H.
, Kagawa W., Osakabe A., et al. 2010. “Structural Basis of Instability of the Nucleosome Containing a Testis‐Specific Histone Variant, Human H3T.” Proceedings of the National Academy of Sciences of the United States of America
107, no. 23: 10454–10459. 10.1073/pnas.1003064107.
</cite> [<a href="https://doi.org/10.1073/pnas.1003064107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2890842/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20498094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Structural%20Basis%20of%20Instability%20of%20the%20Nucleosome%20Containing%20a%20Testis%E2%80%90Specific%20Histone%20Variant,%20Human%20H3T&amp;author=H.%20Tachiwana&amp;author=W.%20Kagawa&amp;author=A.%20Osakabe&amp;volume=107&amp;issue=23&amp;publication_year=2010&amp;pages=10454-10459&amp;pmid=20498094&amp;doi=10.1073/pnas.1003064107&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0036">
<cite>

Tachiwana, H.
, Kagawa W., Shiga T., et al. 2011. “Crystal Structure of the Human Centromeric Nucleosome Containing CENP‐A.” Nature
476, no. 7359: 232–235. 10.1038/nature10258.
</cite> [<a href="https://doi.org/10.1038/nature10258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21743476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Crystal%20Structure%20of%20the%20Human%20Centromeric%20Nucleosome%20Containing%20CENP%E2%80%90A&amp;author=H.%20Tachiwana&amp;author=W.%20Kagawa&amp;author=T.%20Shiga&amp;volume=476&amp;issue=7359&amp;publication_year=2011&amp;pages=232-235&amp;pmid=21743476&amp;doi=10.1038/nature10258&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0037">
<cite>

Takizawa, Y.
, Ho C. H., Tachiwana H., et al. 2020. “Cryo‐EM Structures of Centromeric Tri‐Nucleosomes Containing a Central CENP‐A Nucleosome.” Structure
28, no. 1: 44–53.e4. 10.1016/j.str.2019.10.016.
</cite> [<a href="https://doi.org/10.1016/j.str.2019.10.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31711756/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Cryo%E2%80%90EM%20Structures%20of%20Centromeric%20Tri%E2%80%90Nucleosomes%20Containing%20a%20Central%20CENP%E2%80%90A%20Nucleosome&amp;author=Y.%20Takizawa&amp;author=C.%20H.%20Ho&amp;author=H.%20Tachiwana&amp;volume=28&amp;issue=1&amp;publication_year=2020&amp;pages=44-53.e4&amp;pmid=31711756&amp;doi=10.1016/j.str.2019.10.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0038">
<cite>

Takizawa, Y.
, Tanaka H., Machida S., et al. 2018. “Cryo‐EM Structure of the Nucleosome Containing the ALB1 Enhancer DNA Sequence.” Open Biology
8, no. 3: 170255. 10.1098/rsob.170255.
</cite> [<a href="https://doi.org/10.1098/rsob.170255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5881032/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29563192/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Open%20Biology&amp;title=Cryo%E2%80%90EM%20Structure%20of%20the%20Nucleosome%20Containing%20the%20ALB1%20Enhancer%20DNA%20Sequence&amp;author=Y.%20Takizawa&amp;author=H.%20Tanaka&amp;author=S.%20Machida&amp;volume=8&amp;issue=3&amp;publication_year=2018&amp;pages=170255&amp;pmid=29563192&amp;doi=10.1098/rsob.170255&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0039">
<cite>

Vasudevan, D.
, Chua E. Y. D., and Davey C. A.. 2010. “Crystal Structures of Nucleosome Core Particles Containing the “601” Strong Positioning Sequence.” Journal of Molecular Biology
403, no. 1: 1–10. 10.1016/j.jmb.2010.08.039.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2010.08.039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20800598/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Molecular%20Biology&amp;title=Crystal%20Structures%20of%20Nucleosome%20Core%20Particles%20Containing%20the%20%E2%80%9C601%E2%80%9D%20Strong%20Positioning%20Sequence&amp;author=D.%20Vasudevan&amp;author=E.%20Y.%20D.%20Chua&amp;author=C.%20A.%20Davey&amp;volume=403&amp;issue=1&amp;publication_year=2010&amp;pages=1-10&amp;pmid=20800598&amp;doi=10.1016/j.jmb.2010.08.039&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0040">
<cite>

Woodcock, C. L.
, Frado L. L., and Rattner J. B.. 1984. “The Higher‐Order Structure of Chromatin: Evidence for a Helical Ribbon Arrangement.” Journal of Cell Biology
99, no. 1: 42–52. 10.1083/jcb.99.1.42.
</cite> [<a href="https://doi.org/10.1083/jcb.99.1.42" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2275637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6736132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cell%20Biology&amp;title=The%20Higher%E2%80%90Order%20Structure%20of%20Chromatin:%20Evidence%20for%20a%20Helical%20Ribbon%20Arrangement&amp;author=C.%20L.%20Woodcock&amp;author=L.%20L.%20Frado&amp;author=J.%20B.%20Rattner&amp;volume=99&amp;issue=1&amp;publication_year=1984&amp;pages=42-52&amp;pmid=6736132&amp;doi=10.1083/jcb.99.1.42&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0041">
<cite>

Zheng, S. Q.
, Palovcak E., Armache J. P., Verba K. A., Cheng Y., and Agard D. A.. 2017. “MotionCor2: Anisotropic Correction of Beam‐Induced Motion for Improved Cryo‐Electron Microscopy.” Nature Methods
14, no. 4: 331–332. 10.1038/nmeth.4193.
</cite> [<a href="https://doi.org/10.1038/nmeth.4193" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5494038/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28250466/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=MotionCor2:%20Anisotropic%20Correction%20of%20Beam%E2%80%90Induced%20Motion%20for%20Improved%20Cryo%E2%80%90Electron%20Microscopy&amp;author=S.%20Q.%20Zheng&amp;author=E.%20Palovcak&amp;author=J.%20P.%20Armache&amp;author=K.%20A.%20Verba&amp;author=Y.%20Cheng&amp;volume=14&amp;issue=4&amp;publication_year=2017&amp;pages=331-332&amp;pmid=28250466&amp;doi=10.1038/nmeth.4193&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0042">
<cite>

Zivanov, J.
, Nakane T., Forsberg B., et al. 2018. “New Tools for Automated High‐Resolution Cryo‐EM Structure Determination in RELION‐3.” eLife
7: e42166. 10.7554/eLife.42166.
</cite> [<a href="https://doi.org/10.7554/eLife.42166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6250425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30412051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=New%20Tools%20for%20Automated%20High%E2%80%90Resolution%20Cryo%E2%80%90EM%20Structure%20Determination%20in%20RELION%E2%80%903&amp;author=J.%20Zivanov&amp;author=T.%20Nakane&amp;author=B.%20Forsberg&amp;volume=7&amp;publication_year=2018&amp;pages=e42166&amp;pmid=30412051&amp;doi=10.7554/eLife.42166&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="gtc70019-bib-0043">
<cite>

Zivanov, J.
, Otón J., Ke Z., et al. 2022. “A Bayesian Approach to Single‐Particle Electron Cryo‐Tomography in RELION‐4.0.” eLife
11: 1–22. 10.7554/ELIFE.83724.</cite> [<a href="https://doi.org/10.7554/ELIFE.83724" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9815803/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36468689/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=A%20Bayesian%20Approach%20to%20Single%E2%80%90Particle%20Electron%20Cryo%E2%80%90Tomography%20in%20RELION%E2%80%904.0&amp;author=J.%20Zivanov&amp;author=J.%20Ot%C3%B3n&amp;author=Z.%20Ke&amp;volume=11&amp;publication_year=2022&amp;pages=1-22&amp;pmid=36468689&amp;doi=10.7554/ELIFE.83724&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
<strong>Data S1.</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11995444/bin/GTC-30-0-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">GTC-30-0-s001.pdf</a><sup> (5.7MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The cryo‐EM reconstructions of HeLa nucleosomes have been deposited in the Electron Microscopy Data Bank (EMDB) under the following accession codes: EMD‐63078, EMD‐63079, EMD‐63080, and EMD‐63081. The subtomogram‐averaged maps of HeLa nucleosomes in poly‐nucleosomes have been deposited in the EMDB under the following accession codes: EMD‐63144 and EMD‐63145.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Genes to Cells are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1111/gtc.70019"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/GTC-30-0.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11995444/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11995444/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11995444%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11995444/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11995444/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11995444/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40223781/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11995444/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40223781/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11995444/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11995444/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="ykZZGVOSOJwlrtWJ37MPg77RoW6rlZ3h0fkToKERpBCFpCqTU3zA3f4EUNd7LRJT">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
