
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Structure of human MUTYH and functional profiling of cancer-associated variants reveal an allosteric network between its [4Fe-4S] cluster cofactor and active site required for DNA repair - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="616CF66F8095648300F66F000EB93152.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12000561/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Structure of human MUTYH and functional profiling of cancer-associated variants reveal an allosteric network between its [4Fe-4S] cluster cofactor and active site required for DNA repair">
<meta name="citation_author" content="Carlos H Trasviña-Arenas">
<meta name="citation_author_institution" content="Department of Chemistry, University of California, Davis, CA USA">
<meta name="citation_author_institution" content="Present Address: Research Center on Aging, Center for Research and Advanced Studies (CINVESTAV), Mexico City, Mexico">
<meta name="citation_author" content="Upeksha C Dissanayake">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of Texas at Dallas, Richardson, TX USA">
<meta name="citation_author" content="Nikole Tamayo">
<meta name="citation_author_institution" content="Department of Chemistry, University of California, Davis, CA USA">
<meta name="citation_author_institution" content="Chemistry and Chemical Biology Graduate Program, University of California, Davis, CA USA">
<meta name="citation_author" content="Mohammad Hashemian">
<meta name="citation_author_institution" content="Department of Chemistry, University of California, Davis, CA USA">
<meta name="citation_author_institution" content="Chemistry and Chemical Biology Graduate Program, University of California, Davis, CA USA">
<meta name="citation_author" content="Wen-Jen Lin">
<meta name="citation_author_institution" content="Department of Chemistry, University of California, Davis, CA USA">
<meta name="citation_author_institution" content="Chemistry and Chemical Biology Graduate Program, University of California, Davis, CA USA">
<meta name="citation_author" content="Merve Demir">
<meta name="citation_author_institution" content="Department of Chemistry, University of California, Davis, CA USA">
<meta name="citation_author_institution" content="Chemistry and Chemical Biology Graduate Program, University of California, Davis, CA USA">
<meta name="citation_author" content="Nallely Hoyos-Gonzalez">
<meta name="citation_author_institution" content="Department of Chemistry, University of California, Davis, CA USA">
<meta name="citation_author" content="Andrew J Fisher">
<meta name="citation_author_institution" content="Department of Chemistry, University of California, Davis, CA USA">
<meta name="citation_author_institution" content="Chemistry and Chemical Biology Graduate Program, University of California, Davis, CA USA">
<meta name="citation_author_institution" content="Department of Molecular and Cellular Biology, University of California, Davis, CA USA">
<meta name="citation_author" content="G Andrés Cisneros">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of Texas at Dallas, Richardson, TX USA">
<meta name="citation_author_institution" content="Department of Physics, University of Texas at Dallas, Richardson, TX USA">
<meta name="citation_author" content="Martin P Horvath">
<meta name="citation_author_institution" content="School of Biological Sciences, University of Utah, Salt Lake City, UT USA">
<meta name="citation_author" content="Sheila S David">
<meta name="citation_author_institution" content="Department of Chemistry, University of California, Davis, CA USA">
<meta name="citation_author_institution" content="Chemistry and Chemical Biology Graduate Program, University of California, Davis, CA USA">
<meta name="citation_publication_date" content="2025 Apr 16">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="3596">
<meta name="citation_doi" content="10.1038/s41467-025-58361-w">
<meta name="citation_pmid" content="40234396">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12000561/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12000561/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12000561/pdf/41467_2025_Article_58361.pdf">
<meta name="description" content="MUTYH is a clinically important DNA glycosylase that thwarts mutations by initiating base-excision repair at 8-oxoguanine (OG):A lesions. The roles for its [4Fe-4S] cofactor in DNA repair remain enigmatic. Functional profiling of cancer-associated ...">
<meta name="og:title" content="Structure of human MUTYH and functional profiling of cancer-associated variants reveal an allosteric network between its [4Fe-4S] cluster cofactor and active site required for DNA repair">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="MUTYH is a clinically important DNA glycosylase that thwarts mutations by initiating base-excision repair at 8-oxoguanine (OG):A lesions. The roles for its [4Fe-4S] cofactor in DNA repair remain enigmatic. Functional profiling of cancer-associated ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12000561/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12000561">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-58361-w"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_58361.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12000561%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12000561/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12000561/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12000561/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Apr 16;16:3596. doi: <a href="https://doi.org/10.1038/s41467-025-58361-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-58361-w</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Structure of human MUTYH and functional profiling of cancer-associated variants reveal an allosteric network between its [4Fe-4S] cluster cofactor and active site required for DNA repair</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Trasvi%C3%B1a-Arenas%20CH%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Carlos H Trasviña-Arenas</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Carlos H Trasviña-Arenas</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry, University of California, Davis, CA USA </div>
<div class="p">
<sup>7</sup>Present Address: Research Center on Aging, Center for Research and Advanced Studies (CINVESTAV), Mexico City, Mexico </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Trasvi%C3%B1a-Arenas%20CH%22%5BAuthor%5D" class="usa-link"><span class="name western">Carlos H Trasviña-Arenas</span></a>
</div>
</div>
<sup>1,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dissanayake%20UC%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Upeksha C Dissanayake</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Upeksha C Dissanayake</span></h3>
<div class="p">
<sup>2</sup>Department of Chemistry and Biochemistry, University of Texas at Dallas, Richardson, TX USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dissanayake%20UC%22%5BAuthor%5D" class="usa-link"><span class="name western">Upeksha C Dissanayake</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tamayo%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Nikole Tamayo</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Nikole Tamayo</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry, University of California, Davis, CA USA </div>
<div class="p">
<sup>3</sup>Chemistry and Chemical Biology Graduate Program, University of California, Davis, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tamayo%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nikole Tamayo</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hashemian%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Mohammad Hashemian</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Mohammad Hashemian</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry, University of California, Davis, CA USA </div>
<div class="p">
<sup>3</sup>Chemistry and Chemical Biology Graduate Program, University of California, Davis, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hashemian%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mohammad Hashemian</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20WJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Wen-Jen Lin</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Wen-Jen Lin</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry, University of California, Davis, CA USA </div>
<div class="p">
<sup>3</sup>Chemistry and Chemical Biology Graduate Program, University of California, Davis, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20WJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Wen-Jen Lin</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Demir%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Merve Demir</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Merve Demir</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry, University of California, Davis, CA USA </div>
<div class="p">
<sup>3</sup>Chemistry and Chemical Biology Graduate Program, University of California, Davis, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Demir%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Merve Demir</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hoyos-Gonzalez%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Nallely Hoyos-Gonzalez</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Nallely Hoyos-Gonzalez</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry, University of California, Davis, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hoyos-Gonzalez%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nallely Hoyos-Gonzalez</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fisher%20AJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Andrew J Fisher</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Andrew J Fisher</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry, University of California, Davis, CA USA </div>
<div class="p">
<sup>3</sup>Chemistry and Chemical Biology Graduate Program, University of California, Davis, CA USA </div>
<div class="p">
<sup>4</sup>Department of Molecular and Cellular Biology, University of California, Davis, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fisher%20AJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrew J Fisher</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cisneros%20GA%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">G Andrés Cisneros</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">G Andrés Cisneros</span></h3>
<div class="p">
<sup>2</sup>Department of Chemistry and Biochemistry, University of Texas at Dallas, Richardson, TX USA </div>
<div class="p">
<sup>5</sup>Department of Physics, University of Texas at Dallas, Richardson, TX USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cisneros%20GA%22%5BAuthor%5D" class="usa-link"><span class="name western">G Andrés Cisneros</span></a>
</div>
</div>
<sup>2,</sup><sup>5,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Horvath%20MP%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Martin P Horvath</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Martin P Horvath</span></h3>
<div class="p">
<sup>6</sup>School of Biological Sciences, University of Utah, Salt Lake City, UT USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Horvath%20MP%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin P Horvath</span></a>
</div>
</div>
<sup>6,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22David%20SS%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Sheila S David</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Sheila S David</span></h3>
<div class="p">
<sup>1</sup>Department of Chemistry, University of California, Davis, CA USA </div>
<div class="p">
<sup>3</sup>Chemistry and Chemical Biology Graduate Program, University of California, Davis, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22David%20SS%22%5BAuthor%5D" class="usa-link"><span class="name western">Sheila S David</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Chemistry, University of California, Davis, CA USA </div>
<div id="Aff2">
<sup>2</sup>Department of Chemistry and Biochemistry, University of Texas at Dallas, Richardson, TX USA </div>
<div id="Aff3">
<sup>3</sup>Chemistry and Chemical Biology Graduate Program, University of California, Davis, CA USA </div>
<div id="Aff4">
<sup>4</sup>Department of Molecular and Cellular Biology, University of California, Davis, CA USA </div>
<div id="Aff5">
<sup>5</sup>Department of Physics, University of Texas at Dallas, Richardson, TX USA </div>
<div id="Aff6">
<sup>6</sup>School of Biological Sciences, University of Utah, Salt Lake City, UT USA </div>
<div id="Aff7">
<sup>7</sup>Present Address: Research Center on Aging, Center for Research and Advanced Studies (CINVESTAV), Mexico City, Mexico </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Oct 4; Accepted 2025 Mar 20; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12000561  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40234396/" class="usa-link">40234396</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">MUTYH is a clinically important DNA glycosylase that thwarts mutations by initiating base-excision repair at 8-oxoguanine (OG):A lesions. The roles for its [4Fe-4S] cofactor in DNA repair remain enigmatic. Functional profiling of cancer-associated variants near the [4Fe-4S] cofactor reveals that most variations abrogate both retention of the cofactor and enzyme activity. Surprisingly, R241Q and N238S retained the metal cluster and bound substrate DNA tightly, but were completely inactive. We determine the crystal structure of human MUTYH bound to a transition state mimic and this shows that Arg241 and Asn238 build an H-bond network connecting the [4Fe-4S] cluster to the catalytic Asp236 that mediates base excision. The structure of the bacterial MutY variant R149Q, along with molecular dynamics simulations of the human enzyme, support a model in which the cofactor functions to position and activate the catalytic Asp. These results suggest that allosteric cross-talk between the DNA binding [4Fe-4S] cofactor and the base excision site of MUTYH regulate its DNA repair function.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Enzyme mechanisms, DNA, Enzyme mechanisms, Cancer genetics</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">MUTYH removes A mis-incorporated opposite oxidized guanine to thwart mutations. Here, the authors present crystal structures and assess functional impacts and molecular dynamics of cancer-linked variants to illuminate how the [4Fe-4S] cofactor and active site are allosterically connected with significance for fidelity and clinical impact.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Oxidative DNA damage and its repair are intimately linked to disease<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Arguably, the most studied oxidative DNA lesion is 8-oxo-7,8-dihydro-guanine (OG) which has a high miscoding potential due to its mimicry of thymine during DNA replication, leading to GC → TA transversion mutations. Improperly placed adenines within pro-mutagenic OG:A base pairs are removed by the adenine glycosylase MUTYH, as the first step in Base Excision Repair (BER). Subsequent action of downstream BER enzymes and the OG glycosylase, OGG1, complete the repair process to restore the G:C base pair. The impact of defective repair of OG:A lesions is underscored by the link between inherited <em>MUTYH</em> variants and colorectal cancer, a cancer susceptibility syndrome referred to as MUTYH-associated polyposis (MAP)<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. MAP is defined as an autosomal recessive disorder where inherited biallelic <em>MUTYH</em> mutations lead to multiple colorectal polyps with an increased likelihood of developing colorectal cancer<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Moreover, mutations in <em>MUTYH</em> are increasingly associated with other types of cancer including extraintestinal cancers such as breast, ovary, bladder, thyroid and skin cancers<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>.</p>
<p id="Par4">Many of the &gt;2800 germline and 600 somatic mutations reported in <em>MUTYH</em> map in proximity to its metal cofactors: a [4Fe-4S] cluster and a Zn linchpin (Fig. <a href="#Fig1" class="usa-link">1</a>)<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. The [4Fe-4S] cluster is coordinated by four cysteine ligands within the N-terminal catalytic domain, while the Zn(II) ion is coordinated by three cysteine ligands in the interdomain connector region (IDC) and one histidine within the catalytic domain (Fig. <a href="#Fig1" class="usa-link">1</a>)<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. In previous work, we showed that the two metal cofactors are required for mediating DNA lesion engagement necessary for MUTYH base excision activity; however, the cofactors do not directly participate in base excision catalysis via redox chemistry or as Lewis acids<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1. Human MUTYH-TSAC structure and cancer-associated variants within the [4Fe-4S] cluster motif.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12000561_41467_2025_58361_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ffc/12000561/a5b506e98e9c/41467_2025_58361_Fig1_HTML.jpg" loading="lazy" id="d33e437" height="598" width="799" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Global structure (shown in ribbons) highlighting functional domains, catalytic residues and the [4Fe-4S] cluster. The simulated annealing composite omit (gray) and anomalous difference (orange) maps were contoured at 1.1- and 6-sigma, respectively. Functional domains and motifs are indicated in different colors. A logo sequence shows the conservation of the cysteine ligands across archaea, bacteria and eukaryote MutY/MUTYH. <strong>B</strong> Frequency and distribution of pathogenic germline and somatic mutations mapped in MUTYH amino acid sequence. Domains and motifs are indicated with labels below amino acid numbering. Pathogenic somatic mutations (yellow) were obtained from the COSMIC database<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Pathogenic germline mutations were acquired from ExAC<sup><a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup> (blue), gnomAD<sup><a href="#CR87" class="usa-link" aria-describedby="CR87">87</a></sup> (red) and TOPMED<sup><a href="#CR88" class="usa-link" aria-describedby="CR88">88</a></sup> (green) databases. Reported frequencies were log-normalized (log base 10). Nonsense mutations are indicated with asterisk. Protein-protein interaction sites are shown with black labels. <strong>C</strong> Structural mapping of cancer-associated variants within the [4Fe-4S] cluster motif.</p></figcaption></figure><p id="Par5">The role of the [4Fe-4S] cluster cofactor in MUTYH and related glycosylases has been a topic of great interest. A loop made by two Cys residues that coordinate the [4Fe-4S] cluster, referred to as the iron-sulfur cluster loop, mediates electrostatic interactions with the DNA backbone that are critical for recognition of the lesion containing substrate<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. In addition, the [4Fe-4S] cluster in MUTYH and related [4Fe-4S] cluster-containing BER glycosylases has been proposed to facilitate DNA lesion location through redox communication with other [4Fe-4S] cluster-containing enzymes<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. This process takes advantage of DNA dependent redox cycling of the cofactor ([4Fe-4S]<sup>2+/3+</sup>) to modulate the affinity of the repair enzyme for DNA, and thereby, influence the efficiency of pinpointing the lesion’s location<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>.</p>
<p id="Par6">To provide insight into the impact of cancer associated variants (CAVs) and roles of the [4Fe-4S] cofactor in MUTYH function, we obtained the first crystal structure of human MUTYH in complex with DNA and defined structure-function relationships for 12 [4Fe-4S] associated CAVs. The majority of the CAVs within the [4Fe-4S] cluster motif cause complete loss of both cofactors, and lead to failure for both adenine glycosylase activity and lesion DNA recognition. Notably, R241Q and N238S CAVs disrupt a hydrogen bond network that bridges the [4Fe-4S] cluster to the active site comprised of residues Cys290 ([4Fe-4S] cluster ligand), Arg241, Asn238, and catalytic residue Asp236. We show using functional and computational approaches that this H-bond network mediates a structural communication between the [4Fe-4S] cluster and the active site that influences the proper positioning and protonation status of the catalytic Asp236, thereby altering catalytic efficiency. Our findings provide evidence of an allosteric network between the [4Fe-4S] cluster and the active site that supports DNA repair activity of MUTYH. The [4Fe-4S] cluster is not simply a bystander; CAVs break the allosteric network as evidenced by loss of function for R241Q and N238S, which retain structure, hold onto the metal cofactor, and bind DNA but nevertheless fail as adenine glycosylases.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Crystal structure of human MUTYH</h3>
<p id="Par7">Mapping CAVs onto the structure of MUTYH provides the ability to predict and rationalize structural and functional impacts. The available structures of mammalian MUTYH are a truncated human MUTYH structure which lacks the C-terminal domain<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>, two mouse MUTYH (mMutyh) structures with DNA and a mMutyh C-terminal fragment with PCNA<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. The human and mouse homologs share 74% identity (Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>), and therefore appropriate analysis of CAVs necessitates a human MUTYH structure. We designed and validated an appropriate human MUTYH construct, and along with optimization of overexpression and purification, obtained high yields of pure recombinant human MUTYH protein (see “Methods” and Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>). The crystallization conditions were optimized from those reported for mMutyh<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup> with a key difference being that we paired OG with the azaribose (1N) transition state analog (Fig. <a href="#Fig1" class="usa-link">1A</a>), whereas the mMutyh structure contains tetrahydrofuran (THF, a product analog) across from OG. Crystals diffracted to high resolution (1.9 Å) with a synchrotron source and refinement yielded final R/Rfree values of 0.180/0.209.</p>
<p id="Par8">In the MUTYH-TSAC (Fig. <a href="#Fig1" class="usa-link">1</a>), MUTYH embraces the entire DNA helix with contacts mediated by the two functional domains in a manner similar to other MutY homologs<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. Likewise, important residues within the catalytic pocket such as Asp236, Glu134, Tyr218, 1N and the OG recognition sphere (OG and Ser447) have similar positionings and interactions as found for murine and bacterial homologs<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup> (Fig. <a href="#Fig1" class="usa-link">1A</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>). The IDC region that connects the C-terminal OG recognition and N-terminal catalytic domains in mammalian MUTYH is distinct from other homologs by its longer length and harboring a Zn<sup>2+</sup> ion. We previously coined this region as the “Zn linchpin motif” to reflect its role in coordinating the activity of both domains to support MUTYH activity<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. ICP-MS analysis of the recombinant human MUTYH used for crystallography showed that prior to crystallization the protein was fully loaded with Fe and Zn (Table <a href="#Tab1" class="usa-link">1</a>). However, within the IDC region we were unable to observe electron density for residues 324-347 in chain A and 319-346 in chain D (there are two copies of the MUTYH-TSAC in the asymmetric unit) nor did we observe electron density for the Zn and its coordination sphere. X-ray absorption spectra of the MUTYH-TSAC crystals lacked the ~9650-eV peak characteristic for Zn (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>). These observations suggest Zn was lost during the crystallization process. CAVs are distributed throughout MUTYH and impinge on the many functional domains and motifs (Fig. <a href="#Fig1" class="usa-link">1B</a>). Importantly, the residues corresponding to CAVs nearby the [4Fe-4S] cofactor are clearly defined by electron density. These CAV positions are highly but not absolutely conserved between mMutyh and MUTYH; for example, the residue corresponding to Arg309 in the human enzyme is a Tyr in mMutyh. Moreover, each CAV position is within 4.5 Å of a species-specific variation, further underscoring the need for a structure of the human enzyme.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Metal analysis by Inductively Coupled Plasma-Mass Spectrometry (ICP-MS)</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">MUTYH</th>
<th colspan="1" rowspan="1">[4Fe-4S] load</th>
<th colspan="1" rowspan="1">Zn load</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1">WT (MBP free)</td>
<td colspan="1" rowspan="1">5.1 ± 0.1</td>
<td colspan="1" rowspan="1">1.2 ± 0.1</td>
</tr>
<tr>
<td colspan="1" rowspan="1">WT</td>
<td colspan="1" rowspan="1">4.4 ± 0.1</td>
<td colspan="1" rowspan="1">0.2 ± 0.1</td>
</tr>
<tr>
<td colspan="1" rowspan="1">W103C</td>
<td colspan="1" rowspan="1">0.0</td>
<td colspan="1" rowspan="1">0.0</td>
</tr>
<tr>
<td colspan="1" rowspan="1">W103R</td>
<td colspan="1" rowspan="1">0.0</td>
<td colspan="1" rowspan="1">0.0</td>
</tr>
<tr>
<td colspan="1" rowspan="1">N238S</td>
<td colspan="1" rowspan="1">4.1 ± 0.2</td>
<td colspan="1" rowspan="1">0.3 ± 0.1</td>
</tr>
<tr>
<td colspan="1" rowspan="1">R241Q</td>
<td colspan="1" rowspan="1">4.4 ± 0.1</td>
<td colspan="1" rowspan="1">0.1 ± 0.02</td>
</tr>
<tr>
<td colspan="1" rowspan="1">R241W</td>
<td colspan="1" rowspan="1">0.0</td>
<td colspan="1" rowspan="1">0.0</td>
</tr>
<tr>
<td colspan="1" rowspan="1">R245H</td>
<td colspan="1" rowspan="1">0.0</td>
<td colspan="1" rowspan="1">0.0</td>
</tr>
<tr>
<td colspan="1" rowspan="1">R245C</td>
<td colspan="1" rowspan="1">0.0</td>
<td colspan="1" rowspan="1">0.0</td>
</tr>
<tr>
<td colspan="1" rowspan="1">V246F</td>
<td colspan="1" rowspan="1">3.9 ± 0.5</td>
<td colspan="1" rowspan="1">0.2 ± 0.1</td>
</tr>
<tr>
<td colspan="1" rowspan="1">C290W</td>
<td colspan="1" rowspan="1">0.0</td>
<td colspan="1" rowspan="1">0.0</td>
</tr>
<tr>
<td colspan="1" rowspan="1">P295L</td>
<td colspan="1" rowspan="1">0.0</td>
<td colspan="1" rowspan="1">0.0</td>
</tr>
<tr>
<td colspan="1" rowspan="1">C306W</td>
<td colspan="1" rowspan="1">0.0</td>
<td colspan="1" rowspan="1">0.0</td>
</tr>
<tr>
<td colspan="1" rowspan="1">R309C</td>
<td colspan="1" rowspan="1">4.9 ± 0.3</td>
<td colspan="1" rowspan="1">0.2 ± 0.01</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p11"><p>Values equal to 0 mean that the protein sample had Zn and Fe content equal to the blank. Source data are provided as a <a href="#MOESM6" class="usa-link">Source Data</a> file.</p></div></div></section></section><section id="Sec4"><h3 class="pmc_sec_title">Structural mapping of cancer-associated variants on the MUTYH-TSAC structure reveal a structural interaction between the [4Fe-4S] cluster and the active site</h3>
<p id="Par9">CAVs surround the [4Fe-4S] cofactor in the human MUTYH-TSAC (Fig. <a href="#Fig1" class="usa-link">1C</a>) underscoring the importance of this region in enzyme function. At the posterior face of the cofactor (as defined in Fig. <a href="#Fig1" class="usa-link">1C</a>) are the CAVs C306W (cluster ligand), and V246F. At the frontal face are P295L, R245H/C and at the external face R309C. Finally, at the inner face which forms an intersection between the [4Fe-4S] cluster and the catalytic pocket are W103C/R along with another cluster cysteinyl ligand mutant, C290W, and R241Q/W.</p>
<p id="Par10">Intriguingly, inspection of the MUTYH-TSAC crystal structure revealed an intricate hydrogen bonding network that spans the 20 Å distance between the [4Fe-4S] cluster and the adenine excision pocket (Fig. <a href="#Fig2" class="usa-link">2A</a>). The connection involves four residues with strong evolutionary signals as shown by residue coupling correlation analysis (Fig. <a href="#Fig2" class="usa-link">2B</a>), starting with the [4Fe-4S] cluster ligand Cys290 that H-bonds with Arg241, which in turn interacts with Asn238, and lastly this Asn residue H-bonds with the key catalytic residue Asp236 (Fig. <a href="#Fig2" class="usa-link">2C</a>). In addition, the amide hydrogen of the side chain of the Asn238 also interacts with the 5’ phosphate of the 1N nucleotide. These connections make use of the multivalent Arg241 and Asn238 residues: Arg241 adopts a C-like shape that enables the positioning of internal NH<sub>1</sub><sup>ε</sup> moiety of the guanidine group to donate a H-bond to the main chain carbonyl amide of Cys290. Likewise, the NH<sub>2</sub><sup>η2</sup> group of Arg241 maintains the H-bond network, interacting with the carbonyl of Asn238 side chain and the phosphodiester backbone nucleotide two bases upstream of the 1N moiety. Remarkably, all of the residues in the implicated structural bridge between the [4Fe-4S] cluster and the active site are annotated as CAVs (including N238S and D236N mutations). Moreover, a similar H-bond network is evolutionarily conserved in EndoIII and MIG (Fig. <a href="#Fig2" class="usa-link">2E</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>), suggesting that its functional significance is shared in [4Fe-4S] cluster containing HhH BER glycosylases. In addition to the CAVs studied herein, several others have been reported near the [4Fe-4S] cofactor in LOVD database<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup> (R245S and R309H) as well as a predicted pathogenic somatic mutation in the COSMIC database<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup> (R241L), further implicating this region as a “hotspot” for functional disruption.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Structural interplay between the [4Fe-4S] cluster and active site.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12000561_41467_2025_58361_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ffc/12000561/5e8fe687b4a3/41467_2025_58361_Fig2_HTML.jpg" loading="lazy" id="d33e730" height="638" width="782" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Structural composition of the catalytic pocket and proximity to the [4Fe-4S] cluster. The catalytic pocket comprises 4 α-helixes (H1-H4) and a Helix-Helix (H-H) connector. The H-H connector links the α-helix H3 with an α-helix, which is part of both the catalytic pocket, where Asn238 resides, and the [4Fe-4S] cluster. Catalytic Glu134 and Asp236 reside within the α-helix H2 and the H-H connector, respectively. <strong>B</strong> Mutual evolutionary relationships show a strong coevolutionary signal among the different structural components of the catalytic pocket and the [4Fe-4S] cluster motif of MUTYH/MutY proteins. Mutual Information (MI) detects positions within a multiple sequence alignment that are co-evolving to reveal evolutionary constraints imposed by structure or function. The protein sequence is presented as a circular plot, residue by residue. The different components taken into account by the coevolutionary analysis such as conservation, cumulative mutual information (MI), proximity MI, and MI pairs are depicted in the figure. <strong>C</strong> Residues implied in the structural connectivity between the [4Fe-4S] cluster and the active site. The H-bond network establishing this connectivity is shown in cyan dotted lines. The simulated annealing composite omit (gray) and anomalous difference (orange) maps were contoured at 1.1- and 6-sigma, respectively. <strong>D</strong> Disruption of the structural connectivity between the [4Fe-4S] cluster and the active site in the R149Q GsMutY-THF:OG crystal structure. This mutation shows alternative conformations for the [4Fe-4S] cluster with a displacement of 1.4 Å along with two different conformers of Cys 198. R149Q is the analogous R241Q mutation in MUTYH. The simulated annealing composite omit map (gray) was calculated to the 1.51 Å resolution limit and contoured at 1.0-sigma. <strong>E</strong> Conservation of the residues involved in the [4Fe-4S] cluster-active site structural connectivity among Helix-hairpin-Helix (HhH) DNA glycosylases that also contain the iron-sulfur cofactor (MUTYH/MutY, MIG and EndoIII).</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Functional assays revealed distinct sets of MUTYH variants localized near the [4Fe-4S] cofactor</h3>
<p id="Par11">We purified 11 of the MUTYH CAVs near the [4Fe-4S] cofactor along with N238S, to discern the functional impact of the amino acid variations (Fig. <a href="#Fig3" class="usa-link">3</a>). Due to instability of the variant enzymes to MBP removal, all analyses were carried out with the MBP-MUTYH fusion protein (Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>) as previously reported<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Of note, ICP-MS analysis of MBP-free and MBP-fusion MUTYH showed above 4 nmol of Fe per nmol of MUTYH suggesting that forms of the recombinant proteins were fully loaded with the [4Fe-4S] cluster. In the case of Zn loading, the MBP free MUTYH contained a full complement of the cofactor (1.2 nmol of Zn per/nmol of MUTYH; Table <a href="#Tab1" class="usa-link">1</a>), while only 20% of the MBP-fusion MUTYH population retained Zn. The MBP is located at the N-terminal end of the MUTYH protein near the [4Fe-4S] cluster and Zn binding sites. Thus, the close proximity of the MBP to the metal binding sites might cause a propensity to lose the Zn coordination during the purification; a situation that is circumvented by the immediate MBP removal after Nickel affinity chromatography for the MBP-free MUTYH purification. Indeed, the Zn site in MUTYH is more labile than the [4Fe-4S] cluster, as it was also lost during the crystallization process.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Functional impact of [4Fe-4S] cluster cancer-associated variants of MUTYH.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12000561_41467_2025_58361_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ffc/12000561/1eb52636dcdb/41467_2025_58361_Fig3_HTML.jpg" loading="lazy" id="d33e784" height="745" width="764" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Total product formation as a function of [MUTYH] from adenine glycosylase assays of cancer-associated [4Fe-4S] cluster variants with OG:A-containing DNA substrates. The OG:A 30-bp duplex (20 nM) was incubated with increasing MBP-MUTYH concentrations (0–250 nM) for 1 h at 37 °C; see “Methods” for assay details <strong>B</strong> Percent DNA-bound as a function of [MUTYH] determined from electrophoretic mobility shift assays (EMSA) with cancer-associated variants. The THF:OG-containing 30-bp DNA duplex was titrated with increasing concentrations of MBP-MUTYH (0-400 nM absolute concentrations) and incubated for 20 min at 25 °C. Glycosylase and binding experiments were performed in triplicate with error bars representing standard deviation from the average. <strong>C</strong> Representative storage phosphor autoradiograms of qualitative glycosylase and electrophoretic mobility shift assays (EMSA) with WT, R241Q and N238S MBP-MUTYH mutants. These experiments were carried out as described in Methods. <strong>D</strong> Mutation frequency measured using rifampicin resistance assay with GT100 muty<sup>-</sup> mutm<sup>-</sup>
<em>E. coli</em> cells transformed with pMAL-MBP-MUTYH plasmid harboring cancer-associated variations. For each variant, 8 colonies were evaluated in triplicate on rifampicin plates (24 total measurements/variants). The 95% confidence limits based on the mean value are shown in the relevant plots. <strong>E</strong> CD spectra of purified cancer-associated MBP-MUTYH. Source data are provided as a <a href="#MOESM6" class="usa-link">Source Data</a> file.</p></figcaption></figure><p id="Par12">A qualitative activity and binding screen showed that most of the MUTYH variants exhibited no adenine excision activity on a 30-bp OG:A-containing duplex (Duplex II, methods) nor any ability to bind the product analog-containing (OG:THF) DNA duplex (Fig. <a href="#Fig3" class="usa-link">3A, B</a>, and Supplementary Figs. <a href="#MOESM1" class="usa-link">7</a> and <a href="#MOESM1" class="usa-link">8</a>). Only V246F and R309C exhibited activity and duplex affinity at levels near that of WT MUTYH. Of particular note, R241Q and N238S retained the metal cofactor, bound the OG:THF-DNA duplex yet were inactive enzymes suggesting these two CAVs impact a previously unrecognized functionally required element (Fig. <a href="#Fig3" class="usa-link">3C</a>).</p>
<p id="Par13">The ability of the MUTYH CAVs to suppress DNA mutations in <em>E. coli</em> provided a means to assess activity in a cellular context. Specifically, rifampicin resistance assays measure the ability of MUTYH and variants, expressed as an MBP-MUTYH fusion protein, to suppress mutations in a <em>mutY</em> and <em>mutM</em> deficient GT100 <em>E. coli</em> strain (Fig. <a href="#Fig3" class="usa-link">3D</a>, Supplementary Table <a href="#MOESM1" class="usa-link">1</a>)<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. When cells were transformed with the <em>MUTYH</em>-containing plasmid, the mutation frequency (<em>f</em>) was significantly reduced compared to the vector alone (50-fold) demonstrating the activity of the MBP-MUTYH fusion in <em>E. coli</em>. In contrast, transformation with plasmids encoding W103C/W, R241W, R245C, C290W, P295L and C306W CAVs, yielded cultures with high <em>f</em> values (Fig. <a href="#Fig3" class="usa-link">3D</a>, Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). These results are consistent with analysis of the purified proteins that indicated an absence of enzyme activity. In <em>E. coli</em> expressing variants R241Q, R245H, and N238S <em>f</em> values 18-, 23- and 12-fold higher than observed with WT MBP-MUTYH expression; however, these <em>f</em> values are less than the empty vector control, suggesting a potential ability to suppress mutations to some extent. The corresponding mutation suppression assays performed with MUTYH variants V246F and R309C revealed slightly higher <em>f</em> values (5- and 10-fold) than those from cells harboring WT MBP-MUTYH, also suggesting a mildly reduced mutation suppression activity for these two variants despite appearing quite similar to WT in their in vitro activity. Overall, the complementation assays recapitulate the in vitro activity results but provide additional gradations within the variant functional groupings by distinguishing V246F and R309C from WT, and R245H/C from the set of variants that were completely inactive and unable to bind to DNA in vitro.</p>
<p id="Par14">The impact of MUTYH variations on retention of the metal cofactors and proper folding of MUTYH was assessed via ICP-MS metal analysis and Circular Dichroism (CD) spectroscopy. The MUTYH CAVs that exhibited significant levels of binding to the OG:THF duplex, i.e., N238S, R241Q, V246F and R309C, were found to retain similar levels of Zn and Fe as the WT MBP-MUTYH (Table <a href="#Tab1" class="usa-link">1</a>). CD spectra of N238S, R241Q, V246F and R309C featured a strong signal between 200 nm and 240 nm, similar to that of the WT enzyme, consistent with the high alpha helical content of MUTYH and MBP (Fig. <a href="#Fig3" class="usa-link">3E</a>)<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. However, all the other inactive CAVs that that lacked significant DNA binding capacity, exhibited only background levels of both metal ion cofactors (Table <a href="#Tab1" class="usa-link">1</a>). In the case of mutations at Trp103 (W103C/R) that is part of a hydrophobic interface of the [4Fe-4S] cluster domain and the catalytic pocket, the CD spectrum indicates a dramatic loss of secondary structure consistent with protein unfolding. Notably, the structural changes caused by Trp103 replacement also impact the secondary structure of MBP based on reduction of CD signal. Destabilizing mutations in passenger proteins that affect MBP structure and vice versa have been reported previously<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. In all other variants that had failed to retain the metal cofactors, there was no significant unfolding of the protein as indicated by CD spectroscopy. Surprisingly, MUTYH variants R241W, R245H/C, C290W, P295L, and C306W showed a larger signal at the alpha helical region in the CD spectra suggesting an increase in alpha helicity upon loss of the cluster cofactor. Clearly, the CAVs lead to variable impacts on structure that show that the [4Fe-4S] cofactor and surrounding residues are linked to structure and function. In the case of Trp103 substitution, the global structure appears to be destabilized, while in several other examples, such as N238S and R241Q, the structural impact appears to be more local yet nevertheless detrimental for function.</p>
<p id="Par15">To discern subtleties in functional defects of V246F and R309C MUTYH, we performed detailed adenine glycosylase and EMSA analyses to measure relevant kinetic and binding parameters. Specifically, we measured the base excision step (<em>k</em><sub><em>2</em></sub>) and product release (<em>k</em><sub><em>3</em></sub>) rate constants (Scheme 1) with full-time course adenine glycosylase assays of variant enzyme reacting with a 30-bp OG:A duplex under single turnover (STO, [E] &gt; [S]) and multiple turnover (MTO, [E] &lt; [S]) conditions (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>)<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Relative dissociation constants (<em>K</em><sub><em>D</em></sub>) were measured with the corresponding duplexes replacing the “A” with either a non-cleavable substrate analog (2’-fluoro-2’-deoxyadenosine [fA]) or product (tetrahydrofuran [THF]) via EMSA(Supplementary Table <a href="#MOESM1" class="usa-link">2</a>)<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. The adenine glycosylase assays (Fig. <a href="#Fig4" class="usa-link">4A</a>) revealed that V246F and R309C CAVs had similar base excision <em>k</em><sub><em>2</em></sub> and product release <em>k</em><sub>3</sub> rate constants as WT MBP-MUTYH. However, the MTO experiments showed a reduced burst amplitude, indicating a reduced active fraction, for R309C relative to WT and V246F. Both R309C and V246F variants exhibited WT-like high affinity for THF:OG-containing product analog duplex (<em>K</em><sub><em>D</em></sub> &lt; 10 pM); notably, these values represent upper limit estimates due to experimental detection limitations (Fig. <a href="#Fig4" class="usa-link">4B</a>). The high affinity of MUTYH to the product analog and its observed slow rate of product release is consistent with previous work with MutY enzymes<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. The R309C variant recognized substrate DNA poorly as assessed by an 8-fold increase in <em>K</em><sub><em>D</em></sub> with the fA:OG duplex relative to WT MUTYH. The V246F variant had apparently no detectable impact on substrate recognition as its <em>K</em><sub><em>D</em></sub> was comparable to that of WT MUTYH. The low active fraction and increased <em>K</em><sub><em>D</em></sub> for R309C suggest a compromised ability to engage the lesion substrate, potentially due to local structural changes at the external face of the [4Fe-4S] cluster which are transmitted to the DNA-protein interface. These results further illustrate the range of activities that result from CAVs, and the robust activity for V246F and R309C suggest that these variations may erode mutation suppression by other means. For instance, the activity may be reduced due to cellular conditions of high oxidative stress, or by compromised interactions with down-stream repair machinery. Additional studies are warranted to reveal functional defects associated with these CAVs.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Kinetics of adenine glycosylase activity and dissociation constants, Kd.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12000561_41467_2025_58361_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ffc/12000561/d7d96556c257/41467_2025_58361_Fig4_HTML.jpg" loading="lazy" id="d33e1002" height="770" width="674" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Full-time course adenine glycosylase assays for determination of kinetic parameters for WT, V246F, and R309C MUTYH. Left panel; time course reaction of MBP-MUTYH adenine excision under single turnover condition ([E] = 100 nM active concentration, [A:OG] = 20 nM). Right panel; kinetics of adenine excision under multiple turnover conditions ([E] = 2.5–5 nM active concentration, [A:OG] = 20 nM). Data fitting was used to obtained the glycosylase (<em>k</em><sub><em>2</em></sub>) and turnover (<em>k</em><sub><em>3</em></sub>) rate constants (See “Methods” for more information). <strong>B</strong> DNA binding isotherms of electrophoretic mobility shift assays (EMSA) for WT, R241Q, V246F and R309C. Left panel; DNA binding isotherms obtained with substrate adenine analog (fA) across OG (10 pM) titrated with increasing concentrations of MBP-MUTYH (0–190 nM). Right panel; DNA binding isotherms obtained with abasic site analog (THF) across OG (10 pM) titrated with increasing concentrations of MBP-MUTYH (0-3000 nM). <strong>C</strong> DNA binding isotherms obtained for <em>K</em><sub><em>D</em></sub> determination with fA:OG- and THF:OG-containing DNA duplex binding with WT, R241Q and N238S MBP-MUTYH. Data for R241Q are shown in both panel B and C for comparison. All experiments were repeated in triplicate and the error bars the standard deviation from the mean. Source data are provided as a <a href="#MOESM6" class="usa-link">Source Data</a> file.</p></figcaption></figure><p id="Par16">Differential impacts of the R241Q and N238S variants on product and substrate DNA affinity were revealed in the measured <em>K</em><sub><em>D</em></sub> values with THF:OG and fA:OG-containing DNA duplexes. We observed ~4-fold reduction in the affinity of N238S for the product and substrate analog duplex. In contrast, a larger reduction in affinity for R241Q of 45- and 20-fold for the product or substrate analog, respectively, was observed (Fig. <a href="#Fig4" class="usa-link">4C</a>). These reductions in affinity are likely a consequence of removing the electrostatic and H-bonding interactions with the DNA phosphodiester backbone. However, of note, binding defects with R241Q and N238S are not compensated for at the higher concentrations used in the STO glycosylase assays ([E] &gt; <em>K</em><sub><em>D</em></sub>), as may be expected for non-specific DNA interactions. This suggests that the erosion of binding affinity is substrate specific and reflects an altered binding mode that does not support catalysis. Moreover, unlike CAVs like R241W, which lacks both cofactors and exhibits no glycosylase activity or DNA binding affinity, R241Q and N238S retain levels of Fe and Zn (Table <a href="#Tab1" class="usa-link">1</a>) and overall folding (Fig. <a href="#Fig3" class="usa-link">3E</a>) like the WT, further supporting distinct alterations caused by these variations that is communicated to the active site to thwart catalysis.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Impact of R241Q variant on the structural interplay between the [4Fe-4S] cluster and the active site</h3>
<p id="Par17">In order to delineate the structural basis for the unexpected absence of activity for the R241Q MUTYH, we turned to the corresponding variant in <em>Geobacillus stearothermophilus</em> MutY (GsMutY), R149Q. Of note, the Arg residue, and its participation in the H-bond network with the [4Fe-4S] cluster is highly conserved in MutY enzymes, and therefore likely plays a similar role (Fig. <a href="#Fig2" class="usa-link">2E</a>). However, unlike the mutation in the human homolog, R149Q GsMutY retained measurable adenine glycosylase activity, but with a significant reduction on the base excision rate constant (<em>k</em><sub>2</sub>) and turnover rate constant (<em>k</em><sub>3</sub>) of ~50- and 5-fold relative to WT GsMutY, respectively (Supplementary Table <a href="#MOESM1" class="usa-link">3</a>).</p>
<p id="Par18">Crystals of R149Q GsMutY complexed with a THF:OG-containing duplex diffracted synchrotron radiation to the 1.51-Å resolution limit and the structure was refined through phase extension with simulated annealing, restrained minimization and model rebuilding to yield R/Rfree values of 0.207/0.230 (PDB ID 9BS2). As shown in Fig. <a href="#Fig2" class="usa-link">2D</a>, alterations in the GsMutY structure in the immediate neighborhood of position 149 propagate to the [4Fe-4S] cluster and its Cys290 ligand to induce alternate conformations for the metal cofactor in the R149Q variant. Elongated features in the map calculated from anomalous differences clearly define two different conformations of the [4Fe-4S] cluster with a 1.4-Å displacement separating conformation A and conformation B, which refined with approximately equal group occupancies: <em>q</em> = 0.53 (A) and 0.47 (B) (Fig. <a href="#Fig2" class="usa-link">2D</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>). Another difference apparent for the R149Q GsMutY structure, is a calcium ion chelated by O<sup>δ1</sup> of Asn146. In the WT GsMutY structure, O<sup>δ1</sup> of Asn146 accepts an H-bond from Arg149. Apparently, loss of this Arg149-Asn146 H-bond creates an opportunity for invasion by divalent metal ions, such as Ca<sup>2+</sup>, which is abundantly present in the crystallization condition. Alternate conformations for the [4Fe-4S] cofactor and invasion by Ca<sup>2+</sup> has been seen previously for a structure of N146S GsMutY CAV (corresponds to N238S in MUTYH) captured with substrate DNA (PDB ID 8DVP) that similarly disrupts the Arg-Asn connection at the Asn<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Two structural states for the [4Fe-4S] cofactor, as revealed by these two variants, N146S and R149Q, in proximity to the active site and [4Fe-4S] cluster respectively, suggests that this allosteric network communicates events between the cofactor and active site for critical functional outcomes.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Molecular dynamics simulations reveal an allosteric network between the [4Fe-4S] cluster and the active site</h3>
<p id="Par19">MD simulations were performed to investigate the structural and dynamic relationships between the [4Fe-4S] cluster and the active site in the WT, R241Q and N238S mutant in both mouse and human homologs, using reported structures (see “Methods”). For simulations based on human MUTYH TSAC, the 1N nucleotide was replaced with an AP site for comparison to the mouse structure. Overall human and mouse structures exhibited similar behavior (Fig. <a href="#Fig5" class="usa-link">5</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">10</a>–<a href="#MOESM1" class="usa-link">13</a>). The discussion below focuses on the results for the systems for the MUTYH structures, while detailed results for the mMutyh structures are provided in the supporting information (Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a> and <a href="#MOESM1" class="usa-link">13</a>). Analysis on the resulting trajectories were performed including root mean square deviation (RMSD), normal mode analysis (NMA), and energy decomposition analysis (EDA) to understand the dynamic features and differences between the systems (Fig. <a href="#Fig5" class="usa-link">5</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">10</a>–<a href="#MOESM1" class="usa-link">13</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Molecular dynamics simulation.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12000561_41467_2025_58361_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ffc/12000561/78499450f23e/41467_2025_58361_Fig5_HTML.jpg" loading="lazy" id="d33e1112" height="770" width="696" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Energy decomposition (left), normal mode (middle), and network (right) analyses based on the MD trajectories are shown. The left and right panels show a close-up view of the residues that participate in the structural interplay between the [4Fe-4S] cluster and the active site along with values for energy decomposition analysis (EDA, left) and network analysis results (right). The EDA reports the intermolecular non-bonded interactions (Coulomb and van der Waals interactions; kcal/mol) involving Asn238 or Arg241 (black and red values, respectively). Below, the total energy contribution for the system is shown for Asn238 (black) and Arg241 (red) and corresponding cancer-associated mutants. For the network analysis the betweenness of the nodes involved in the multi-motif bridge is shown in brown (right-upper panel) and the optimal path between the AP site and [4Fe-4S] cluster is displayed in magenta (right-lower panel). The first normal mode is shown in the middle where the MUTYH and DNA are shown in blue and gray, respectively.</p></figcaption></figure><p id="Par20">NMA indicates that both the WT and R241Q systems show similar behavior in terms of percentage contribution of motion, with the first mode contributing ∼90% and the second mode contributing ∼10% to the overall motion (Fig. <a href="#Fig5" class="usa-link">5</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">10</a>), and movie found at Zenodo repository (10.5281/zenodo.10161357)<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. However, the first two modes for WT correspond to a breathing-like motion whereas the R241Q system exhibits a rocking-like motion. Conversely, the first mode of the N238S system constitutes about ∼69%, while the second mode and third mode contribute∼29% and ∼2% respectively to the overall motion. These modes indicate different types of rocking motion. NMA analysis suggests that the mutations change the dynamics of the system to a more rigid rocking motion from breathing motion and specifically, the first normal mode for N238S is drastically reduced with respect to percentage contribution compared to WT and R241Q.</p>
<p id="Par21">We performed EDA to calculate the non-bonded intermolecular interaction energies (Coulomb and Van der Waals) as a function of specific reference fragments (e.g., residue, nucleotide, etc.). This approach allows us to qualitatively investigate the interactions of individual residues in the allosteric network linking the [4Fe-4S] cluster, the catalytic Asp236 and the AP site. Both Arg241 and Asn238 show significant contributions to the overall stability of the protein with total interaction energy of −423.1 and −66.4 kcal/mol, with respect to residues 241 and 238 respectively (Fig. <a href="#Fig5" class="usa-link">5</a>). However, mutation of these residues reduces the overall stability of the systems (−61.9 and −56.8 kcal/mol for R241Q and N238S, respectively). Interestingly, the R241Q mutation severely reduces the energy contributions for the catalytic Asp236 and the AP site, diminishing these from −42.7 and −62.7 kcal/mol for each residue in the WT MUTYH to −4.1 and −2.9 kcal/mol. Furthermore, the difference in non-bonded interaction energy (ΔE) was calculated between the mutant systems and the WT with respect to the [4Fe-4S] cluster. Our results suggest that the mutation of Asn238 to serine results in decreased interactions between several nucleotides/residues and the [4Fe-4S] cluster. Interestingly, for the R241Q variant, the DNA strand containing the AP site shows improved interactions with the [4Fe-4S] cluster (Supplementary Fig. <a href="#MOESM1" class="usa-link">11</a>), while the other DNA strand is destabilized. In addition, Arg241 is destabilized in WT compared to the R241Q mutant (−49.5 kcal/mol), as both the [4Fe-4S] cluster and Arg are positively charged. The sum of ΔE between N238S and WT is +23.1 kcal/mol and the sum of ΔE between R241Q and WT is +4.9 kcal/mol, which suggests that both mutations result in overall decreased interactions between the [4Fe-4S] cluster, and the rest of the protein compared to WT.</p>
<p id="Par22">To further investigate the allosteric network between the [4Fe-4S] cluster and the catalytic Asp236 we performed dynamic network analysis. The nodes connecting the AP site and Arg241 in the WT structure show a significant betweenness. This is disrupted by both mutations, N238S and R241Q (Fig. <a href="#Fig5" class="usa-link">5</a>). Additionally, the betweenness of the AP site and Asn238 shows a connection associating these two nodes in the WT system, which is broken in both mutant systems. The optimal path connecting the [4Fe-4S] cluster to the AP site was determined for all systems. The optimal path between the AP site and the [4Fe-4S] cluster in WT is through Arg241 (Fig. <a href="#Fig5" class="usa-link">5</a>). However, this optimal path is altered for N238S, in which Cys290 is involved in a series of the H-bonds that connect to the [4Fe-4S] cluster. Furthermore, this optimal path is completely changed for the R241Q system and an alternative path through the protein is found without involving the residues in the H-bond bridge. Altogether, analysis of the MD simulations predicts the existence of a network connecting the [4Fe-4S] metal site to the catalytically critical Asp236. We suggest that this network serves as an allosteric regulator to ensure adenine excision from rare OG:A lesions but not highly abundant T:A bps, and as a means to control enzyme function in response to cellular conditions.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par23">Herein, we reported the first human MUTYH transition state analog complex (MUTYH TSAC) structure and delineated structural and functional consequences of 12 CAVs near to the [4Fe-4S] cluster of MUTYH. These results lead to a significant conceptual advance in our understanding of this critical base excision DNA repair enzyme. Whereas we and others have speculated as to the nature of a functional connection between the [4Fe-4S] metal cluster and the active site<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, herein, we define how that connection is mechanistically established. The MUTYH TSAC structure provided a means to correlate position and nature of substitution of the CAVs with the impact on MUTYH activity; an annotated structure-activity summary of the 12 variants studied herein is shown in Fig. <a href="#Fig6" class="usa-link">6</a>. Our results with MBP-MUTYH, consistent with previous work with <em>E. coli</em> MutY, show that the loss of the [4Fe-4S] cofactor, either by denaturation, chelation or by mutation of coordinating ligands or surrounding residues, leads to loss of glycosylase activity and DNA binding capabilities without drastic changes in secondary structure<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Notable exceptions are MUTYH CAVs W103C and W103R which disrupt the hydrophobic packing between the cluster domain and the active site, highlighting the close association of these domains. In previous work, McDonnell et al. identified and characterized the biochemical and redox properties of C306W as an MBP-MUTYH recombinant protein<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. These authors found similar results to those observed herein with C306W: compromised glycosylase activity and no detectable DNA binding. However, McDonnell et al. reported that C306W MBP-MUTYH retained a low level of [4Fe-4S] ((9%) based on [Fe]), that was sensitive to oxidative degradation. In addition, Komine et al. reported similar results for mutation suppression activity of W103R, R241W, R245C/H, V246F, C290W, and P296L in a MutY-deficient <em>E. coli</em> strain<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Importantly, our results and these studies underscore the functional importance of the [4Fe-4S] cofactor in MUTYH.</p>
<figure class="fig xbox font-sm" id="Fig6"><h3 class="obj_head">Fig. 6. Summary of functional impacts of MUTYH [4Fe-4S] CAVs.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12000561_41467_2025_58361_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ffc/12000561/aca7d87d8a7e/41467_2025_58361_Fig6_HTML.jpg" loading="lazy" id="d33e1197" height="287" width="696" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The different functional impacts of MUTYH cancer-associated variants in proximity to the [4Fe-4S] cluster allowed for grouping into distinct classes.  The impact statements correlate with the color of the residue where the variants are localized.</p></figcaption></figure><p id="Par24">Our inspection of the human MUTYH structure revealed an intricate H-bonding network which spans from the [4Fe-4S] cluster, Cys290 ligand, Arg291, Asn238 up to the catalytic Asp236 and the transition state analog 1N. Comparison with mouse MUTYH and bacterial GsMutY structures shows this H-bonding network to be conserved in structure across evolution (Fig. <a href="#Fig2" class="usa-link">2E</a>). Curiously, the network includes two CAVs, R241Q and N238S, that each displayed WT levels of Fe and Zn ion and were capable of binding to substrate and product DNA but were completely inactive as glycosylases. The structures of MUTYH, along with those of GsMutY Arg→Gln and Asn→Ser variant<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, and the MD simulations illuminate how the allosteric network between the [4Fe-4S] cofactor DNA binding site and the active site regulates positioning and protonation state of the catalytic Asp required for base excision catalysis.</p>
<p id="Par25">In MD/QM studies previously reported for WT and the Asn→Ser variant of GsMutY<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>, the Asn-Asp interaction was found to be quite dynamic during catalysis. Indeed, in the bacterial enzyme MD simulations, the H-bond that connects Asn to Asp breaks halfway through catalysis, with a concomitant change in Asp protonation. This is consistent with the increase in pKa of the Asp we observed previously with N146S GsMutY relative to the WT enzyme<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Notably, we also observed an altered position of the Purine base in the active site in the structure of N146S GsMutY with an OG:Purine substrate that likely hinders protonation of N7 by Glu43. These studies combined with those herein illustrate the exquisite control exerted over the base excision chemistry of MutY enzymes to ensure DNA repair fidelity.</p>
<p id="Par26">Sequence and structural analyses of other [4Fe-4S] cluster-containing Helix-hairpin-Helix (HhH) DNA glycosylases reveal a similarly conserved allosteric network connecting the [4Fe-4S] cluster and the active site (Fig. <a href="#Fig2" class="usa-link">2E</a>). The thymine-DNA glycosylase MIG has an identical, Cys-Arg-Asn-Asp network<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. The EndoIII/NTHL1 glycosylase conserves a similar H-bond connectivity with His instead of Asn (Cys-Arg-His-Asp; Fig. <a href="#Fig2" class="usa-link">2E</a>)<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> The mechanistic details of other HhH [4Fe-4S] cluster containing BER glycosylases have not been studied as extensively as MutY, though likely share an S<sub>N</sub>1-like mechanism<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup> (Fig. <a href="#Fig7" class="usa-link">7</a>). Despite the differences in substrate processed by these other HhH DNA glycosylases, it is quite striking that there is a high degree of conservation of the residues to maintain the H-bond network between the [4Fe-4S] cluster and the catalytic pocket. This suggests that such a multi residue-bridging motif has been a functionally important structural element throughout the evolution of [4Fe-4S] cluster-containing HhH BER glycosylases. Similarly, in single domain plant-type ferredoxins, allostery between a loop and a [2Fe-2S] cluster 20 Å apart has been shown to involve minimal structural perturbations propagated by short-range interactions and display concordant patterns of evolution<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig7"><h3 class="obj_head">Fig. 7. Proposed catalytic mechanism for MUTYH and disrupted mechanism in R241Q and N238S cancer-associated variants.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12000561_41467_2025_58361_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3ffc/12000561/0040a76dadab/41467_2025_58361_Fig7_HTML.jpg" loading="lazy" id="d33e1260" height="358" width="702" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The adenine excision catalyzed by MUTYH is initiated with the protonation of the adenine by Glu134 which is followed by the glycosidic bond cleavage and formation of an oxacarbenium ion intermediate. Asp236 is proposed to attack the oxacarbenium ion at C1’ to stabilize it by a covalent intermediate. Finally, Glu134 activates a water molecule for nucleophilic attack at C1 to form the AP site product. The structural interplay between the [4Fe-4S] cluster is important for step 2 of catalysis, where Asp236 attacks the oxacarbenium ion. Asp236 has to be oriented near the N-glycosidic bond and deprotonated for the nucleophilic attack. Such orientation and deprotonation status of Asp236 is modulated by the H-bond with Gln238 and allosterically through the [4Fe-4S] cluster via the Cys290-Arg241-Asn238-Asp236 structural bridge. Upon R241Q and N238S mutations the H-bond between Asp236 and Asn238 becomes unstable (cyan dashed arrow) or is broken (black curved lines). With the structural bridge thus compromised the position, protonation state, and charge of catalytic Asp236 position are also impacted (double-headed dashed arrows) with the net consequence of impeding nucleophilic attack of the oxacarbenium ion. This pathological scenario is accompanied by structural instability of the [4Fe-4S] cluster and loss of DNA-MUTYH interactions.</p></figcaption></figure><p id="Par27">The impact on catalysis of mutations to the Arg-Asn residues linking the [4Fe-4S] cluster to the active site suggests a means by which changes in the redox state of the [4Fe-4S]<sup>2+/3+</sup> cluster alters base excision catalysis. In previous collaborative studies with the Barton laboratory, the [4Fe-4S]<sup>2+</sup> cluster in <em>E. coli</em> MutY was shown to become redox active upon DNA binding, facilitating oxidation from [4Fe-4S]<sup>2+</sup> to [4Fe-4S]<sup>3+</sup><sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>–<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. In addition, a variety of DNA repair and replication enzymes harboring a [4Fe-4S] were shown to have DNA-mediated dependent redox activity<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Notably, in the case of EndoIII, oxidation was shown to increase DNA affinity<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Based on these studies, Barton and co-workers have proposed that [4Fe-4S] cluster nucleic acid processing enzymes utilize the metal cofactor to sense the DNA integrity and locate DNA lesions by DNA association versus dissociation controlled by the cluster redox state and communicated via DNA-mediated charge-transport<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. In the work herein, many of the [4Fe-4S] CAVs were found to be destabilizing to the [4Fe-4S] cluster; this may be a reflection of an altered redox potential. In contrast, the V246F and R309C variants exhibited WT-like enzyme activity, however, the cluster in these enzyme variants may exhibit reduced activity due to cluster instability when challenged by conditions of oxidative stress. Two computational studies using QM/MD calculations and hole hopping analyses, have defined a charge-transport pathway between the [4Fe-4S] cluster and the active site (OG or A) using structures of GsMutY<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Indeed, these studies have suggested that Arg241<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup> and Trp103<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup> are part of the charge transport pathway and that their mutation would hamper this redox communication. Although many details of the cofactor redox roles in MUTYH remain to be elucidated, it is clear that the [4Fe-4S] cluster plays multiple essential roles in the enzyme’s function.</p>
<p id="Par28">The current study provides functional and structural information on MUTYH CAVs and reveals fundamental features that aid in understanding cancer etiology at an atomic level. Through studying MUTYH CAVs, we provide structural and biochemical evidence of an allosteric role for the [4Fe-4S] cluster in HhH DNA glycosylases, where an H-bond network coordinates DNA binding near the [4Fe-4S] cluster to adenine excision via the key catalytic Asp. CAVs that disrupt the allosteric network, such R241Q and N238S, compromise MUTYH function, providing for increased mutagenesis to trigger carcinogenesis. The influence of changes near the [4Fe-4S] cluster and the active site, also suggests a potential regulatory role in DNA repair where DNA interactions at the [4Fe-4S] cluster binding domain are transmitted to the active site to reduce or enhance base excision (Fig. <a href="#Fig5" class="usa-link">5</a>). Alterations at the [4Fe-4S] may be influenced by the DNA context and or redox status of the cell. Under conditions of high oxidative stress, the loss of the [4Fe-4S] cluster may be a means to down-regulate MUTYH activity to prevent accumulation of unrepaired AP sites that may lead to genotoxic strand breaks as evidenced in telomere instability<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. In contrast, under conditions of slightly elevated oxidative stress, cluster oxidation may be a way to augment MUTYH binding and enhance its activity. Of note, structurally, the [4Fe-4S] cluster is not an inert element. The oxidized state of the cofactor has an increased net positive charge (+3 versus +2) and smaller Van der Waals volume and surface area than the reduced cofactor (∼550 Å<sup>3</sup> and ∼504 Å<sup>2</sup> versus 570 Å<sup>3</sup> and 530 Å<sup>2</sup>, respectively)<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. These changes in cofactor size and charge may be transmitted to the active site via the newly identified H-bond network to modulate MUTYH activity. Thus, we propose that the characteristic 2+ redox state of the cofactor of the DNA-free MUTYH represents the less active allosteric mode where the reduced [4Fe-4S]<sup>2+</sup> cofactor positions the catalytic Asp in an orientation and protonation state that are suboptimal to perform catalysis (Fig. <a href="#Fig7" class="usa-link">7</a>). However, upon DNA binding and oxidation to the [4Fe-4S]<sup>3+</sup> state cluster, adenine excision may be enhanced via control of the catalytic Asp, representing a more active allosteric state. Interestingly, under oxidative stress MUTYH along with OGG1 activities are reported to exert telomeric instability by means of replicative stress<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>,<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Therefore, this type of allostery may be a mechanism to control aberrant MUTYH activity under oxidative stress.</p>
<p id="Par29">The sensitivity of MUTYH activity to alterations and loss of the cluster also suggest this as a site for targeting with small molecules. Indeed, TEMPOL, a stable nitroxide, was recently shown to cause oxidative degradation of the [4Fe-4S]<sup>2+</sup> cluster in the RNA-dependent RNA polymerase of SARS-CoV-2 to block viral replication in animals<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Remarkably, optimal function of MUTYH and likely other HhH glycosylases depends on its [4Fe-4S]<sup>2+</sup> cluster, however the reliance on a sensitive redox cofactor provides avenues for its degradation under oxidative stress, which would explain how chronic inflammation and associated oxidative stress contributes to cancer etiology. Ironically, the “Achilles heel” of MUTYH may be its [4Fe-4S] cluster that could be targeted in cancer cells that have become reliant on MUTYH to survive.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Human <em>MUTYH</em> cloning and mutagenesis</h3>
<p id="Par30">A codon optimized human <em>MUTYH</em> gene (beta 3 isoform) for <em>E. coli</em> overexpression was designed and purchased from IDT as a gBlock (See Supplementary information for ORF’s sequence). The <em>MUTYH</em> ORF is devoid of the initial fourteen codons to alleviate protein toxicity as previously suggested<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. The <em>MUTYH</em> gBlock was subcloned into pJET2.1 vector and ultimately cloned into a modified version of pET28a vector using NdeI and NcoI restriction sites. The modified version of pET28 allows the overexpression of MBP-MUTYH protein with 2 histidine tags both at N- and C-terminus regions. Two internal TEV protease cleavage sites were introduced to remove the His-tags and MBP segments from the MUTYH protein for crystallography. A scheme of the final pET28-MBP-MUTYH design is included in supplementary data (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>). For crystallography the human <em>MUTYH</em> gene was trimmed, removing the initial 34 and last 28 codons, analogous to N- and C-terminal truncations found in the murine structure<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>.</p>
<p id="Par31">Mutagenesis of the pET28-MBP-MUTYH construct was carried out by PCR-driven overlap extension<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Due to toxicity of pET28-MBP-MUTYH plasmid in GT100 <em>muty</em><sup>-</sup>
<em>mutm</em><sup>-</sup>
<em>E. coli</em> strain, for complementation experiments we used the construct pMAL-MBP-MUTYH which contains a non-codon-optimized <em>MUTYH</em> gene as previously reported<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. The mutagenesis of pMAL-MBP-MUTYH construct was done using the Q5 site-directed mutagenesis kit from New England BioLabs (Catalog no. E0554S).</p></section><section id="Sec11"><h3 class="pmc_sec_title">MUTYH overexpression and purification</h3>
<p id="Par32">The overexpression and purification of MUTYH was carried out as previously reported<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Briefly, a BL21(DE3) strain containing pRKISC and pKJE7 vectors was used as the expression host. The pRKISC plasmid co-expresses the [4Fe-4S] cluster assembly machinery<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, and pKJE7 coexpresses dnaK, dnaJ and grgE chaperones<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. The BL21(+pKRISC+pKJE7) strain was transformed with the pET28-MBP-MUTYH construct and plated onto Luria Broth plates supplemented with 50 µg/mL of kanamycin (Kan), 15 µg/ mL of tetracycline (Tet) and 34 µg/mL of chloramphenicol (Cam). Colonies obtained from the transformation were used to inoculate 2 L of Terrific broth media supplemented with Kan+Tet+Cam in a 4 L flask with a flat bottom and grown for 6 h at 37 °C with shaking at 180 rpm until an OD<sub>600nm</sub> of at least 1.5. The culture was cooled without shaking 1 h at 4 °C, and then protein production was induced with IPTG (0.25 mM) and addition of ferrous sulfate (0.1 g) and ferric citrate (0.1 g). The overexpression was carried out at 15 °C for 16–24 h with continuous shaking. Bacteria pellets were obtained by centrifugation (6000 × <em>g</em>/10 min/4°) and stored at −80 °C until needed.</p>
<p id="Par33">For protein purification the pellets were thawed and resuspended in Lysis buffer (30 mM Tris [pH 7.5], 1 M NaCl, 30 mM 2-mercaptoethanol and 10% glycerol) supplemented with 1 mM of phenylmethylsulfonyl fluoride. The cellular lysis was carried out by sonication on ice in 20 s cycles using a Branson Sonifier 250 followed by centrifugation at 15 000 × <em>g</em> for 50 min at 4 °C. The clarified supernatant was incubated with 1.5 mL of Ni<sup>2+</sup> NTA resin (Qiagen; catalog no. 30210) for 1 h at 4 °C with rotation. The slurry was poured over a PD10 column (Cytiva; catalog no. 17-0435-01) and allowed to flow through via gravity. The protein-loaded resin was washed with at least 25 mL of Lysis buffer followed 10 mL of elution buffer (30 mM Tris [pH 7.5], 200 mM NaCl, 30 mM 2-mercaptoethanol, 10% glycerol and 500 mM imidazole). Removal of imidazole immediately following elution proved to be critical for protein stability. To accomplish buffer exchange, the protein was subjected to heparin-affinity chromatography where the elution was diluted with an equal volume of buffer A (30 mM Tris [pH 7.5], 10 % glycerol, 1 mM DTT and 1 mM EDTA) to reduce the NaCl concertation to 100 mM. The diluted nickel elution was loaded onto a 1 mL Heparin column (Cytiva; catalog no. 17040601) previously equilibrated with 10% buffer B (30 mM Tris [pH 7.5], 1 M NaCl, 10 % glycerol, 1 mM DTT and 1 mM EDTA). The heparin column containing the bound protein was washed with 25 mL of 10% buffer B and the protein was eluted using linear gradient of buffer A and B (10-100%) over 45 min with a flowrate of 1.5 mL/min using an AKTA FPLC instrument (GE Healthcare). The fractions containing pure MBP-MUTYH protein were analyzed by SDS-PAGE (Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>) and concentrated using Amicon ultracentrifugation filters (30,000 MWCO; catalog no. UFC5030). The protein concentration was estimated by measuring the 280-nm UV absorbance with an extinction coefficient of 152,470 M<sup>−1</sup> cm<sup>−1</sup>. The purified protein was then aliquoted and stored at −80 °C.</p>
<p id="Par34">The protein for crystallography was treated with TEV protease after nickel-affinity chromatography with a ratio 30:1 (w/w) of MUTYH:TEV at 4 °C for 16 h in 10% buffer B to remove the His-tag and MBP tags. A second Heparin affinity chromatography step using a 5 mL Heparin column was performed to remove the released tags, followed by size-exclusion chromatography using Superdex 200 column with 20% buffer B. Purification of R149Q GsMutY protein was carried out as previously reported for the N146S <em>Gs</em> MutY enzyme<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Preparation of oligonucleotide substrates</h3>
<p id="Par35">The 1N transition state analog (3R,4R)-3-(hydroxymethyl) pyrrolidine-1-ium phosphoramidite was synthesized with modifications of literature procedures<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>, as described in the Supplementary Information. The 1N, OG, FA, and A containing 2’deoxyribonucleotides used for crystallography, binding, and kinetic experiments were synthesized at the University of Utah DNA and peptide synthesis core facility. The OG-containing DNA strands were cleaved from the column and deprotected using ammonium hydroxide with 0.25 M 2-mercaptoethanol for 17 h at 55 °C. All the oligonucleotides were HPLC-purified, desalted with Sep-Pak C18 desalting cartridge (Waters) and correct mass confirmed by matrix-assisted laser-desorption/ionization (MALDI) mass spectrometry at the UC Davis Campus Mass Spectrometry Facility (cmsf.ucdavis.edu). The oligonucleotides sequences used in this study are listed in supplementary table <a href="#MOESM1" class="usa-link">7</a>.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Crystallography</h3>
<p id="Par36">For human MUTYH crystallization, conditions were optimized from those previously reported with the murine enzyme<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. The 1N- and OG-containing oligonucleotides used for crystallography (Duplex 1; Supplementary Table <a href="#MOESM1" class="usa-link">7</a>) were annealed to 1:1 ratio (263 µM) in 30 mM Tris [pH 7.5] by heating at 90 °C for 5 min followed by slow cool annealing to 4 °C. The 1N:OG duplex I (263 µM) was added to MUTYH protein (236 µM) in buffer containing 30 mM Tris [pH 7.5], 100 mM NaCl and 0.5 mM DTT. The resulting MUTYH-DNA complex (118 µM) was incubated for 20 min at room temperature and mixed with crystallization solutions in 1:1 ratio in 3 µL final volume onto a coverslip (Hampton Research). The best quality crystals in terms of size and X-ray diffraction resolution limit were obtained in 0.1 M Bis-Tris [pH 5.5], 0.2 M ammonium sulfate and 20% PEG 3350. The crystals were grown at room temperature by the hanging-drop vapor-diffusion method. Golden rod or needle-like crystals grew over a 12–24 h window. Crystals were harvested directly from the drop and flash cooled in liquid nitrogen. X-ray diffraction data were collected with 0.2° oscillation on beamline 24-ID-E at the Advanced Photon Source (Argonne National Laboratory). Crystallization and X-ray diffraction experiment of R149Q GsMutY and the product analog (THF):OG duplex were carried out using methods similar to those reported previously with N146S and WT GsMutY with DNA<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Briefly, clusters of crystals containing R149Q GsMutY in complex with THF:OG DNA duplex I were grown with 350 µM of DNA-protein complex in 14% PEG, 400 mM Ca(OAc)<sub>2</sub> and 2% ethylene glycol (pH 8.5). The final individual crystals feasible for X-ray diffraction experiments were grown by microseeding with 10 X dilution of crushed crystals clusters initially obtained.</p>
<p id="Par37">Diffraction data for the MUTYH-DNA complex structure were processed with XDS and scaled with XSCALE<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. The mouse Mutyh-DNA complex crystal structure (PDB ID: 7EF9) was used as the search model to obtain initial phases by molecular replacement with Phaser as implemented by PHENIX<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> and refined with iterative cycles of torsion angle simulated annealing, restrained minimization, and manual model building to yield final R/Rfree values of 0.180/0.209. The structure was refined using PHENIX including 1N, OG, and [4Fe-4S] cluster coordination restraints<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. The statistics in data processing and model refinement are shown in Supplementary Table <a href="#MOESM1" class="usa-link">4</a>. The asymmetric unit includes two copies of the hMUTYH-DNA complex. X-ray diffraction data processing, and molecular replacement and refinements for R149Q GsMutY were carried out as previously reported for the N126S GsMutY variant<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> using GsMutY-TSAC structure (PDB ID: 6U7T)<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> as model to obtain the initial phases (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>). Calcium ions were placed as indicated by coordination geometry analysis provided by the validation tool “highly coordinated waters…” within Coot<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. Although data were measured far from the optimal wavelength, calcium ions were in clear 4-sigma anomalous difference peaks. All figures depicting structure were generated using PyMOL (The PyMOL Molecular Graphics System, Version 2.0 Schrödinger, LLC), UCSF Chimera<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>, and Coot<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. Coordinates for the human MUTYH-transition state analog complex and the GsMutY R149Q bound with DNA containing THF have been deposited in the Protein Data Bank with PDB IDs <a href="http://doi.org/10.2210/pdb8FAY/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8FAY</a> and <a href="http://doi.org/10.2210/pdb9BS2/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9BS2</a>.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Coevolutionary analysis</h3>
<p id="Par38">A multiple sequence alignment including 687 amino acid sequence of Archaea, Bacteria, and Eukaryote MutY homologs was generated using the MUSCLE algorithm<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> as implemented with the Geneious software package (Version 4.8, Biomatters). The MSA is included in the SI files. The MUSCLE-generated MSA was manually curated and uploaded into the MISTIC web server<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup> along with coordinates for chain A of the human MUTYH structure. The coevolutionary analysis was run with the default parameters and results were visualized using the tools incorporated within the MISTIC web server.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Inductively Coupled Plasma-Mass Spectrometry (ICP-MS)</h3>
<p id="Par39">Metal analysis by ICP-MS used purified WT and mutant MBP-fusion and MBP free MUTYH proteins that was buffer exchanged to 20 mM Tris-HCl [pH 7.5], 250 NaCl and 10 % glycerol. Samples and blanks were prepared as previously described<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup> in a range of 500–250 μL and submitted in triplicate to the UC Davis Interdisciplinary Center for Inductively Coupled Plasma-Mass Spectrometry (<a href="https://icpms.ucdavis.edu" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://icpms.ucdavis.edu</a>).</p></section><section id="Sec16"><h3 class="pmc_sec_title">Glycosylase assay and binding experiments</h3>
<p id="Par40">The glycosylase activity of MUTYH was measured under single turnover and multiple turnover conditions (STO and MTO) to determine the base excision rate (<em>k</em><sub><em>2</em></sub>) and turnover (<em>k</em><sub><em>3</em></sub>), respectively as previously reported<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup> following a minimal kinetic scheme as described below.</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e1603" display="block"><mi>M</mi><mi>U</mi><mi>T</mi><mi>Y</mi><mi>H</mi><mo>+</mo><mfenced close=")" open="("><mrow><mi>D</mi><mi>N</mi><mi>A</mi></mrow></mfenced><mi>s</mi><mtable><mtr><mtd columnalign="center"><msub><mrow><mi>k</mi></mrow><mrow><mn>1</mn></mrow></msub></mtd></mtr><mtr><mtd columnalign="center"><mo>↔</mo></mtd></mtr><mtr><mtd columnalign="center"><msub><mrow><mi>k</mi></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msub></mtd></mtr></mtable><mi>M</mi><mi>U</mi><mi>T</mi><mi>Y</mi><mi>H</mi><mo>⋅</mo><mfenced close=")" open="("><mrow><mi>D</mi><mi>N</mi><mi>A</mi></mrow></mfenced><mi>s</mi><msup><mrow><mo>→</mo></mrow><mrow><msub><mrow><mi>k</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></msup><mi>M</mi><mi>U</mi><mi>T</mi><mi>Y</mi><mi>H</mi><mo>⋅</mo><mfenced close=")" open="("><mrow><mi>D</mi><mi>N</mi><mi>A</mi></mrow></mfenced><mi>p</mi><msup><mrow><mo>→</mo></mrow><mrow><msub><mrow><mi>k</mi></mrow><mrow><mn>3</mn></mrow></msub></mrow></msup><mi>M</mi><mi>U</mi><mi>T</mi><mi>Y</mi><mi>H</mi><mo>+</mo><mfenced close=")" open="("><mrow><mi>D</mi><mi>N</mi><mi>A</mi></mrow></mfenced><mi>p</mi></math></td>
<td class="label">1</td>
</tr></table>
<p id="Par41">To measure binding affinity of the MAP variants we utilized Electrophoretic Mobility Shift Assay (EMSA) with uncleavable fluorinated-adenine (fA):OG-containing DNA duplex to determine <em>K</em><sub><em>D</em></sub> for substrate and tetrahydrofuran (THF):OG-containing DNA duplex for product affinity, as previously reported<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Briefly, the 5′-end of radiolabeled 2′-FA or THF-containing strand was annealed to its complementary strand with OG and a DNA master mix was prepared in buffer containing 40 mM Tris pH 7.6, 2 mM EDTA, 200 M sodium chloride, 20% (w/v) glycerol, 0.2 mg/ml BSA, 2 mM DTT and 20 pM of radiolabeled Duplex 2 (Supplementary table <a href="#MOESM1" class="usa-link">7</a>). Equal volumes of DNA master mix were combined with enzyme in decreasing concentrations (prepared at 4 °C in dilution buffer containing 20 mM Tris pH 7.6, 10 mM EDTA and 20% glycerol) and incubated for 30 min at 25 °C. The enzyme bound DNA was separated from the unbound DNA using 6% nondenaturing polyacrylamide gel ran with 0.5X TBE buffer at 120 V for 2 h at 4 °C. The gels were dried and quantified, and the data was fit to single binding isotherm model to derive the dissociation constant, <em>K</em><sub><em>D</em></sub> adjusting the data to nonlinear regression fit of one site-specific binding as described in Eq. <a href="#Equ2" class="usa-link">2</a>.</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e1727" display="block"><mi>Y</mi><mo>=</mo><mfrac><mrow><mi>B</mi><mi>max</mi><mi>X</mi></mrow><mrow><msub><mrow><mi>K</mi></mrow><mrow><mi>D</mi></mrow></msub><mspace width="0.25em"></mspace><mi>X</mi></mrow></mfrac></math></td>
<td class="label">2</td>
</tr></table>
<p>Where X is the concentration of MUTYH, Y is the specific binding (%), Bmax is the maximum binding. Kinetics and EMSA experiments were carried out in triplicate.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Circular dichroism spectroscopy</h3>
<p id="Par42">Circular dichroism spectroscopy was performed with 0.1 mg/mL of protein in 30 mM Tris [pH 7.5] and 50 mM sodium sulfate, using a 1 mm CD quartz cuvette at room temperature and the Jasco J720 CD spectrophotometer, scanning at a range of 190–240 nm. The data was acquired by averaging triplicate scans and normalized to millidegree to delta epsilon (Δε, M<sup>−1</sup> cm<sup>−1</sup>).</p></section><section id="Sec18"><h3 class="pmc_sec_title">Rifampicin resistance assay</h3>
<p id="Par43">MBP-MUTYH<em>-</em>pMAL construct was used to analyze mutation suppression activity of CAVs in <em>E. coli</em> as previously reported<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Briefly, we measured the mutation frequency (as determined by RifR colonies) of GT100 <em>muty</em><sup>-</sup>
<em>mutm</em><sup>-</sup>
<em>E. coli</em> strain transformed with the WT MBP-MUTYH<em>-</em>pMAL construct or corresponding CAVs relative to the parent strain. For each variant, 8 colonies were evaluated in triplicate on rifampicin plates (24 total measurements/variants), and analyzed as described previously<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. The estimation of the mutation frequency (<em>f</em>) was carried out as follows.</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><math id="d33e1795" display="block"><mi>f</mi><mo>=</mo><mfrac><mrow><mi>m</mi><mi>e</mi><mi>d</mi><mi>i</mi><mi>a</mi><mi>n</mi><mspace width="0.16em"></mspace><mi>n</mi><mi>u</mi><mi>m</mi><mi>b</mi><mi>e</mi><mi>r</mi><mspace width="0.16em"></mspace><mi>o</mi><mi>f</mi><mspace width="0.16em"></mspace><mi>r</mi><mi>e</mi><mi>s</mi><mi>i</mi><mi>s</mi><mi>t</mi><mi>a</mi><mi>n</mi><mi>t</mi><mspace width="0.16em"></mspace><mi>c</mi><mi>o</mi><mi>l</mi><mi>o</mi><mi>n</mi><mi>i</mi><mi>e</mi><mi>s</mi></mrow><mrow><mi>a</mi><mi>v</mi><mi>e</mi><mi>r</mi><mi>a</mi><mi>g</mi><mi>e</mi><mspace width="0.16em"></mspace><mi>n</mi><mi>u</mi><mi>m</mi><mi>b</mi><mi>e</mi><mi>r</mi><mspace width="0.16em"></mspace><mi>o</mi><mi>f</mi><mspace width="0.16em"></mspace><mi>v</mi><mi>a</mi><mi>i</mi><mi>b</mi><mi>l</mi><mi>e</mi><mspace width="0.16em"></mspace><mi>c</mi><mi>o</mi><mi>l</mi><mi>o</mi><mi>n</mi><mi>i</mi><mi>e</mi><mi>s</mi></mrow></mfrac></math></td>
<td class="label">3</td>
</tr></table></section><section id="Sec19"><h3 class="pmc_sec_title">Molecular dynamics simulations</h3>
<p id="Par44">The crystal structure for the human MUTYH-DNA complex reported herein was modified to include missing regions as modeled with SWISS-MODEL server<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. The crystal structure of mouse MUTYH-DNA complex with the PDB ID <a href="http://doi.org/10.2210/pdb7EF8/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7EF8</a><sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup> was used as the initial model. A comparative protein structure modeling was performed, and the missing regions were incorporated with MODELLER 10.4<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. The mutations of both human and mouse models were introduced using UCSF Chimera<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Three systems were considered for each MUTYH and mMutyh models, human: wild type (WT), N238S, R241Q and mouse: WT, N209S, and R212Q. 8-oxo-7,8-dihydro-guanine (OG), AP site, [4Fe-4S] cluster and Zn<sup>2+</sup> Linchpin motif needed to be parameterized prior to the MD simulation. The azaribose (1N) transition state in the MUTYH-TSAC structure was converted to an AP site using UCSF Chimera. The AMBER force field parameters for OG and AP site were obtained from the AMBER parameter database<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup> and the missing parameters were calculated by ANTECHAMBER<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>,<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. The parameters of the [4Fe-4S] cluster were obtained from the publication by Squier and co-workers<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup> and the parameters of the Zn<sup>2+</sup> Linchpin motif were obtained from the Zinc AMBER force field<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. Side chain clashes and protonation states were assessed using ProPKA<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup> and MolProbity<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. All systems were prepared with the Leap module<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup> of AMBER21<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup> by solvation in a TIP3P<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup> cubic box extending a minimum of 10 Å distance from the edge of the protein, and neutralized while setting the ionic strength to 50 mM KCl (Supplementary Table <a href="#MOESM1" class="usa-link">6</a>).</p>
<p id="Par45">All molecular dynamics (MD) simulations were carried out with the AMBER ff19SB<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>, gaff<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>, and OL15<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup> force fields with the AMBER21 pmemd.cuda program<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup>. Initially, protein and DNA were minimized with a restraint of 300 kcal mol<sup>−1</sup> Å<sup>2</sup> for 500 cycles using conjugate gradient, continuing for 6000 cycles of steepest descent. Subsequently, each system was heated to 300 K gradually by 50 K in 10,000 MD step intervals at constant volume using Langevin dynamics<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>, with the protein restrained by a force constant of 500 kcal mol<sup>−1</sup> Å<sup>2</sup>. Next, the systems were equilibrated via gradually reducing the restraints on the protein and DNA using the NVT ensemble with a 1 fs time step (Supplementary Table <a href="#MOESM1" class="usa-link">5</a>). The production simulations were carried in triplicate for 500 ns each using the NPT ensemble, with a 2 fs time step, without restraints. Constant temperature (300 K) and pressure (1.0 bar) were maintained using the Langevin thermostat and Berendsen barostat<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>,<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>. All bonds involving hydrogen atoms were constrained using SHAKE<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>. Long-range electrostatic interactions were addressed with the Smooth Particle-Mesh Ewald method, while Van der Waals interactions were controlled by employing the Isotropic Periodic Sum method with a real-space distance of 10 Å<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>,<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup>.</p>
<p id="Par46">The AMBER21 CPPTRAJ<sup><a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup> module was used to calculate the RMSD and root-mean-square fluctuation in the production trajectories. NMA was carried out using ProDy<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a></sup>. EDA was performed using Fortran90 program to calculate intermolecular non-bonded interactions (Coulomb and Vander Waals interactions) between a residue of interest and the rest of the system<sup><a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>,<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a></sup>. Dynamic network analysis was conducted using the Dynamic Network Analysis Python package<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>,<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a></sup>. Network analysis was performed to investigate the node-node interactions in all systems. In this analysis, each residue is represented by nodes, where each amino acid is depicted by a single node on the alpha-carbons and each nucleotide by two nodes on the backbone phosphorous and nitrogen atom in the nitrogenous base. The shortest distance between two nodes is calculated to identify which nodes are in contact. If a pair of nodes maintain contact in more than 75% of the simulation frames with a distance ≤4.5 Å, they are considered to be in contact and those nodes are connected by edges. The weight of an edge between nodes corresponds to the probability of information transfer along that connection as calculated based on the correlation values of two residues. The count of shortest paths that pass through an edge in the network are described as the betweenness of an edge. This serves as a measure for assessing the significance of the edge for communication within the network. Furthermore, the optimal path which refers to the most efficient or shortest path was calculated between AP site node and Fe-S cluster node using correlations as weights to determine the shortest distances between these two nodes.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par47">Further information on research design is available in the <a href="#MOESM3" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec21"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12000561/bin/41467_2025_58361_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (5.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12000561/bin/41467_2025_58361_MOESM2_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Suppplementary Data 1</a><sup> (940.5KB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12000561/bin/41467_2025_58361_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (7.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12000561/bin/41467_2025_58361_MOESM4_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Description of Supplementary Data files</a><sup> (13.6KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12000561/bin/41467_2025_58361_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (95.3KB, pdf) </sup>
</div></div></section></section><section id="Sec22"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12000561/bin/41467_2025_58361_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (43KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Razan Kaddoura for technical help with mutagenesis and Savannah Conlon for purification of OG oligonucleotide for duplex 2. We also thank Madhu Budamagunta and John Voss for access and experimental assistance to obtain the CD spectra. C.H.T.A. was supported in part by a postdoctoral fellowship from UC-MEXUS/CONACYT and N.T. was supported by an AGEP-Graduate Research Supplement (CHE-2039752). This work was supported by research grants from the National Cancer Institute (CA069785 to S.S.D.) and the National Institute of General Medical Sciences (GM108583 and GM151951 to G.A.C.), and the National Science Foundation (CHE:CLP- 1905249, 2204229 to M.P.H.). In addition, computing time from University of North Texas CASCaM supported by the National Science Foundation (OAC-2117247 to G.A.C.) and from the University of Texas at Dallas Cyberinfrastructure are gratefully acknowledged. Part of this work is based upon research conducted at the Northeastern Collaborative Access Team beamlines, which are funded by the National Institute of General Medical Sciences from the National Institutes of Health (P30 GM124165). The Eiger 16M detector on the 24-ID-E beam line is funded by a NIH-ORIP HEI grant (S10OD021527). This research used resources of the Advanced Photon Source, a U.S. Department of Energy (DOE) Office of Science User Facility operated for the DOE Office of Science by Argonne National Laboratory under Contract No. DE-AC02-06CH11357. We thank ALS Beamline 5.0.1 staff Marc Allaire and Core Ralston for assistance with data collection at the Advanced Light Source. The Berkeley Center for Structural Biology is supported by the Howard Hughes Medical Institute, Participating Research Team members, and the National Institutes of Health, National Institute of General Medical Sciences, ALS-ENABLE grant P30 GM124169. The Advanced Light Source is a Department of Energy Office of Science User Facility under Contract No. DE-AC02-05CH11231. The Pilatus detector on beamline 5.0.1 was funded under NIH grant S10OD026941.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>C.H.T.A and S.S.D conceived the project; C.H.T.A., U.C.D., S.S.D. and G.A.C designed the experiments; C.H.T.A., N.T., M.H., M.D., N.H.G., performed biochemical and structural experiments; U.C.D., C.H.T.A., performed computational experiments and analyses with oversight from G.A.C.; W.J.L. synthesized transition state mimic-containing oligonucleotides; C.H.T.A., S.S.D., A.J.F., G.A.C., and M.P.H. analyzed data and wrote the paper. All authors provided comments on the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par48"><em>Nature Communications</em> thanks Walter Chazin and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The atomic coordinates and structure factors generated in this study have been deposited in the Protein Data Bank (<a href="http://www.rcsb.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.rcsb.org</a>) under accession codes <a href="https://doi.org/10.2210/pdb8FAY/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8FAY</a> (human MUTYH complexed with DNA containing the transition state analog 1N) and <a href="https://doi.org/10.2210/pdb9BS2/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9BS2</a> (Gs MutY R149Q complexed with DNA containing the product analog THF). The initial coordinates and parameter files for each WT and mutant systems in human and mouse MUTYH generated as part of the molecular dynamics studies, as well as parameters describing the [4Fe-4S] cluster and Zn<sup>2+</sup> Linchpin motif have been deposited to the Zenodo repository<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> found at 10.5281/zenodo.10161357. <a href="#Sec22" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par49">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>G. Andrés Cisneros, Email: andres@utdallas.edu.</p>
<p>Martin P. Horvath, Email: martin.horvath@utah.edu</p>
<p>Sheila S. David, Email: ssdavid@ucdavis.edu</p></section><section id="sec23"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-58361-w.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Al-Tassan, N. et al. Inherited variants of MYH associated with somatic G: C→ T: a mutations in colorectal tumors. <em>Nat. Genet.</em><strong>30</strong>, 227–232 (2002).
</cite> [<a href="https://doi.org/10.1038/ng828" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11818965/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Al-Tassan,%20N.%20et%20al.%20Inherited%20variants%20of%20MYH%20associated%20with%20somatic%20G:%20C%E2%86%92%20T:%20a%20mutations%20in%20colorectal%20tumors.%20Nat.%20Genet.30,%20227%E2%80%93232%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Raetz, A. G. &amp; David, S. S. When you’re strange: Unusual features of the MUTYH glycosylase and implications in cancer. <em>DNA Repair</em><strong>80</strong>, 16–25 (2019).</cite> [<a href="https://doi.org/10.1016/j.dnarep.2019.05.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6812671/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31203172/" class="usa-link">PubMed</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Engstrom, L. M. et al. A zinc linchpin motif in the MUTYH glycosylase interdomain connector is required for efficient repair of DNA damage. <em>J. Am. Chem. Soc.</em><strong>136</strong>, 7829–7832 (2014).
</cite> [<a href="https://doi.org/10.1021/ja502942d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4063174/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24841533/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Engstrom,%20L.%20M.%20et%20al.%20A%20zinc%20linchpin%20motif%20in%20the%20MUTYH%20glycosylase%20interdomain%20connector%20is%20required%20for%20efficient%20repair%20of%20DNA%20damage.%20J.%20Am.%20Chem.%20Soc.136,%207829%E2%80%937832%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Messick, T. E. et al. Noncysteinyl coordination to the [4Fe-4S] 2+ cluster of the DNA repair adenine glycosylase MutY introduced via site-directed mutagenesis. Structural characterization of an unusual histidinyl-coordinated cluster. <em>Biochemistry</em><strong>41</strong>, 3931–3942 (2002).
</cite> [<a href="https://doi.org/10.1021/bi012035x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11900536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Messick,%20T.%20E.%20et%20al.%20Noncysteinyl%20coordination%20to%20the%20%5B4Fe-4S%5D%202+%20cluster%20of%20the%20DNA%20repair%20adenine%20glycosylase%20MutY%20introduced%20via%20site-directed%20mutagenesis.%20Structural%20characterization%20of%20an%20unusual%20histidinyl-coordinated%20cluster.%20Biochemistry41,%203931%E2%80%933942%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Porello, S. L., Leyes, A. E. &amp; David, S. S. Single-turnover and pre-steady-state kinetics of the reaction of the adenine glycosylase MutY with mismatch-containing DNA substrates. <em>Biochemistry</em><strong>37</strong>, 14756–14764 (1998).
</cite> [<a href="https://doi.org/10.1021/bi981594+" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9778350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Porello,%20S.%20L.,%20Leyes,%20A.%20E.%20&amp;%20David,%20S.%20S.%20Single-turnover%20and%20pre-steady-state%20kinetics%20of%20the%20reaction%20of%20the%20adenine%20glycosylase%20MutY%20with%20mismatch-containing%20DNA%20substrates.%20Biochemistry37,%2014756%E2%80%9314764%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Russelburg, L. P. et al. Structural basis for finding OG lesions and avoiding undamaged G by the DNA glycosylase MutY. <em>ACS Chem. Biol.</em><strong>15</strong>, 93–102 (2020).
</cite> [<a href="https://doi.org/10.1021/acschembio.9b00639" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7069122/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31829624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Russelburg,%20L.%20P.%20et%20al.%20Structural%20basis%20for%20finding%20OG%20lesions%20and%20avoiding%20undamaged%20G%20by%20the%20DNA%20glycosylase%20MutY.%20ACS%20Chem.%20Biol.15,%2093%E2%80%93102%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Trasvina-Arenas, C., Demir, M., Lin, W.-J. &amp; David, S. S. Structure, function and evolution of the Helix-hairpin-Helix DNA glycosylase superfamily: Piecing together the evolutionary puzzle of DNA base damage repair mechanisms. <em>DNA Repair</em><strong>108</strong>, 103231 (2021).
</cite> [<a href="https://doi.org/10.1016/j.dnarep.2021.103231" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34649144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trasvina-Arenas,%20C.,%20Demir,%20M.,%20Lin,%20W.-J.%20&amp;%20David,%20S.%20S.%20Structure,%20function%20and%20evolution%20of%20the%20Helix-hairpin-Helix%20DNA%20glycosylase%20superfamily:%20Piecing%20together%20the%20evolutionary%20puzzle%20of%20DNA%20base%20damage%20repair%20mechanisms.%20DNA%20Repair108,%20103231%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Woods, R. D. et al. Structure and stereochemistry of the base excision repair glycosylase MutY reveal a mechanism similar to retaining glycosidases. <em>Nucleic Acids Res.</em><strong>44</strong>, 801–810 (2016).
</cite> [<a href="https://doi.org/10.1093/nar/gkv1469" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4737165/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26673696/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Woods,%20R.%20D.%20et%20al.%20Structure%20and%20stereochemistry%20of%20the%20base%20excision%20repair%20glycosylase%20MutY%20reveal%20a%20mechanism%20similar%20to%20retaining%20glycosidases.%20Nucleic%20Acids%20Res.44,%20801%E2%80%93810%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Manlove, A. H. et al. Structure–activity relationships reveal key features of 8-oxoguanine: a mismatch detection by the MutY glycosylase. <em>ACS Chem. Biol.</em><strong>12</strong>, 2335–2344 (2017).
</cite> [<a href="https://doi.org/10.1021/acschembio.7b00389" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5603899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28723094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manlove,%20A.%20H.%20et%20al.%20Structure%E2%80%93activity%20relationships%20reveal%20key%20features%20of%208-oxoguanine:%20a%20mismatch%20detection%20by%20the%20MutY%20glycosylase.%20ACS%20Chem.%20Biol.12,%202335%E2%80%932344%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Banda, D. M., Nunez, N. N., Burnside, M. A., Bradshaw, K. M. &amp; David, S. S. Repair of 8-oxoG:A mismatches by the MUTYH glycosylase: mechanism, metals and medicine. <em>Free Radic. Biol. Med.</em><strong>107</strong>, 202–215 (2017).
</cite> [<a href="https://doi.org/10.1016/j.freeradbiomed.2017.01.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5457711/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28087410/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Banda,%20D.%20M.,%20Nunez,%20N.%20N.,%20Burnside,%20M.%20A.,%20Bradshaw,%20K.%20M.%20&amp;%20David,%20S.%20S.%20Repair%20of%208-oxoG:A%20mismatches%20by%20the%20MUTYH%20glycosylase:%20mechanism,%20metals%20and%20medicine.%20Free%20Radic.%20Biol.%20Med.107,%20202%E2%80%93215%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Magrin, L. et al. MUTYH-associated tumor syndrome: the other face of MAP. <em>Oncogene</em><strong>41</strong>, 2531–2539 (2022).
</cite> [<a href="https://doi.org/10.1038/s41388-022-02304-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35422474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Magrin,%20L.%20et%20al.%20MUTYH-associated%20tumor%20syndrome:%20the%20other%20face%20of%20MAP.%20Oncogene41,%202531%E2%80%932539%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Nuñez, N. N. et al. The zinc linchpin motif in the DNA repair glycosylase MUTYH: identifying the Zn2+ ligands and roles in damage recognition and repair. <em>J. Am. Chem. Soc.</em><strong>140</strong>, 13260–13271 (2018).
</cite> [<a href="https://doi.org/10.1021/jacs.8b06923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6443246/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30208271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nu%C3%B1ez,%20N.%20N.%20et%20al.%20The%20zinc%20linchpin%20motif%20in%20the%20DNA%20repair%20glycosylase%20MUTYH:%20identifying%20the%20Zn2+%20ligands%20and%20roles%20in%20damage%20recognition%20and%20repair.%20J.%20Am.%20Chem.%20Soc.140,%2013260%E2%80%9313271%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Nakamura, T. et al. Structure of the mammalian adenine DNA glycosylase MUTYH: insights into the base excision repair pathway and cancer. <em>Nucleic Acids Res.</em><strong>49</strong>, 7154–7163 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkab492" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8266592/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34142156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nakamura,%20T.%20et%20al.%20Structure%20of%20the%20mammalian%20adenine%20DNA%20glycosylase%20MUTYH:%20insights%20into%20the%20base%20excision%20repair%20pathway%20and%20cancer.%20Nucleic%20Acids%20Res.49,%207154%E2%80%937163%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Porello, S. L., Cannon, M. J. &amp; David, S. S. A substrate recognition role for the [4Fe-4S] 2+ cluster of the DNA repair glycosylase MutY. <em>Biochemistry</em><strong>37</strong>, 6464–6475 (1998).</cite> [<a href="https://doi.org/10.1021/bi972433t" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9572864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Porello,%20S.%20L.,%20Cannon,%20M.%20J.%20&amp;%20David,%20S.%20S.%20A%20substrate%20recognition%20role%20for%20the%20%5B4Fe-4S%5D%202+%20cluster%20of%20the%20DNA%20repair%20glycosylase%20MutY.%20Biochemistry37,%206464%E2%80%936475%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Chepanoske, C. L., Golinelli, M.-P., Williams, S. D. &amp; David, S. S. Positively Charged Residues within the Iron–Sulfur Cluster Loop of E. coli MutY Participate in Damage Recognition and Removal. <em>Arch. Biochem. Biophys.</em><strong>380</strong>, 11–19 (2000).
</cite> [<a href="https://doi.org/10.1006/abbi.2000.1890" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10900127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chepanoske,%20C.%20L.,%20Golinelli,%20M.-P.,%20Williams,%20S.%20D.%20&amp;%20David,%20S.%20S.%20Positively%20Charged%20Residues%20within%20the%20Iron%E2%80%93Sulfur%20Cluster%20Loop%20of%20E.%20coli%20MutY%20Participate%20in%20Damage%20Recognition%20and%20Removal.%20Arch.%20Biochem.%20Biophys.380,%2011%E2%80%9319%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Guan, Y. et al. MutY catalytic core, mutant and bound adenine structures define specificity for DNA repair enzyme superfamily. <em>Nat. Struct. Mol. Biol.</em><strong>5</strong>, 1058 (1998).</cite> [<a href="https://doi.org/10.1038/4168" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9846876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guan,%20Y.%20et%20al.%20MutY%20catalytic%20core,%20mutant%20and%20bound%20adenine%20structures%20define%20specificity%20for%20DNA%20repair%20enzyme%20superfamily.%20Nat.%20Struct.%20Mol.%20Biol.5,%201058%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Boal, A. K. et al. DNA-bound redox activity of DNA repair glycosylases containing [4Fe-4S] clusters. <em>Biochemistry</em><strong>44</strong>, 8397–8407 (2005).
</cite> [<a href="https://doi.org/10.1021/bi047494n" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15938629/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boal,%20A.%20K.%20et%20al.%20DNA-bound%20redox%20activity%20of%20DNA%20repair%20glycosylases%20containing%20%5B4Fe-4S%5D%20clusters.%20Biochemistry44,%208397%E2%80%938407%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>McDonnell, K. J. et al. A human MUTYH variant linking colonic polyposis to redox degradation of the [4Fe4S](2+) cluster. <em>Nat. Chem.</em><strong>10</strong>, 873–880 (2018).
</cite> [<a href="https://doi.org/10.1038/s41557-018-0068-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6060025/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29915346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McDonnell,%20K.%20J.%20et%20al.%20A%20human%20MUTYH%20variant%20linking%20colonic%20polyposis%20to%20redox%20degradation%20of%20the%20%5B4Fe4S%5D(2+)%20cluster.%20Nat.%20Chem.10,%20873%E2%80%93880%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Luncsford, P. J. et al. A structural hinge in eukaryotic MutY homologues mediates catalytic activity and Rad9–Rad1–Hus1 checkpoint complex interactions. <em>J. Mol. Biol.</em><strong>403</strong>, 351–370 (2010).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2010.08.045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2953589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20816984/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luncsford,%20P.%20J.%20et%20al.%20A%20structural%20hinge%20in%20eukaryotic%20MutY%20homologues%20mediates%20catalytic%20activity%20and%20Rad9%E2%80%93Rad1%E2%80%93Hus1%20checkpoint%20complex%20interactions.%20J.%20Mol.%20Biol.403,%20351%E2%80%93370%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Fromme, J. C., Banerjee, A., Huang, S. J. &amp; Verdine, G. L. Structural basis for removal of adenine mispaired with 8-oxoguanine by MutY adenine DNA glycosylase. <em>Nature</em><strong>427</strong>, 652–656 (2004).
</cite> [<a href="https://doi.org/10.1038/nature02306" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14961129/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fromme,%20J.%20C.,%20Banerjee,%20A.,%20Huang,%20S.%20J.%20&amp;%20Verdine,%20G.%20L.%20Structural%20basis%20for%20removal%20of%20adenine%20mispaired%20with%208-oxoguanine%20by%20MutY%20adenine%20DNA%20glycosylase.%20Nature427,%20652%E2%80%93656%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Lee, S. &amp; Verdine, G. L. Atomic substitution reveals the structural basis for substrate adenine recognition and removal by adenine DNA glycosylase. <em>Proc. Natl Acad. Sci.</em><strong>106</strong>, 18497–18502 (2009).
</cite> [<a href="https://doi.org/10.1073/pnas.0902908106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2773975/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19841264/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20S.%20&amp;%20Verdine,%20G.%20L.%20Atomic%20substitution%20reveals%20the%20structural%20basis%20for%20substrate%20adenine%20recognition%20and%20removal%20by%20adenine%20DNA%20glycosylase.%20Proc.%20Natl%20Acad.%20Sci.106,%2018497%E2%80%9318502%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Fokkema, I. F. et al. The LOVD3 platform: efficient genome-wide sharing of genetic variants. <em>Eur. J. Hum. Genet.</em><strong>29</strong>, 1796–1803 (2021).
</cite> [<a href="https://doi.org/10.1038/s41431-021-00959-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8632977/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34521998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fokkema,%20I.%20F.%20et%20al.%20The%20LOVD3%20platform:%20efficient%20genome-wide%20sharing%20of%20genetic%20variants.%20Eur.%20J.%20Hum.%20Genet.29,%201796%E2%80%931803%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Forbes, S. A. et al. COSMIC: somatic cancer genetics at high-resolution. <em>Nucleic Acids Res.</em><strong>45</strong>, D777–D783 (2017).
</cite> [<a href="https://doi.org/10.1093/nar/gkw1121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5210583/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27899578/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Forbes,%20S.%20A.%20et%20al.%20COSMIC:%20somatic%20cancer%20genetics%20at%20high-resolution.%20Nucleic%20Acids%20Res.45,%20D777%E2%80%93D783%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Komine, K. et al. Functional complementation assay for 47 MUTYH variants in a MutY‐disrupted Escherichia coli strain. <em>Hum. Mutat.</em><strong>36</strong>, 704–711 (2015).
</cite> [<a href="https://doi.org/10.1002/humu.22794" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4682456/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25820570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Komine,%20K.%20et%20al.%20Functional%20complementation%20assay%20for%2047%20MUTYH%20variants%20in%20a%20MutY%E2%80%90disrupted%20Escherichia%20coli%20strain.%20Hum.%20Mutat.36,%20704%E2%80%93711%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Kundu, S., Brinkmeyer, M. K., Livingston, A. L. &amp; David, S. S. Adenine removal activity and bacterial complementation with the human MutY homologue (MUTYH) and Y165C, G382D, P391L and Q324R variants associated with colorectal cancer. <em>DNA Repair</em><strong>8</strong>, 1400–1410 (2009).
</cite> [<a href="https://doi.org/10.1016/j.dnarep.2009.09.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2789978/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19836313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kundu,%20S.,%20Brinkmeyer,%20M.%20K.,%20Livingston,%20A.%20L.%20&amp;%20David,%20S.%20S.%20Adenine%20removal%20activity%20and%20bacterial%20complementation%20with%20the%20human%20MutY%20homologue%20(MUTYH)%20and%20Y165C,%20G382D,%20P391L%20and%20Q324R%20variants%20associated%20with%20colorectal%20cancer.%20DNA%20Repair8,%201400%E2%80%931410%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Chakraborty, D., Rodgers, K. K., Conley, S. M. &amp; Naash, M. I. Structural characterization of the second intra‐discal loop of the photoreceptor tetraspanin RDS. <em>FEBS J.</em><strong>280</strong>, 127–138 (2013).
</cite> [<a href="https://doi.org/10.1111/febs.12055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3652632/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23121719/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chakraborty,%20D.,%20Rodgers,%20K.%20K.,%20Conley,%20S.%20M.%20&amp;%20Naash,%20M.%20I.%20Structural%20characterization%20of%20the%20second%20intra%E2%80%90discal%20loop%20of%20the%20photoreceptor%20tetraspanin%20RDS.%20FEBS%20J.280,%20127%E2%80%93138%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Nallamsetty, S. &amp; Waugh, D. S. Mutations that alter the equilibrium between open and closed conformations of Escherichia coli maltose-binding protein impede its ability to enhance the solubility of passenger proteins. <em>Biochem. Biophys. Res. Commun.</em><strong>364</strong>, 639–644 (2007).
</cite> [<a href="https://doi.org/10.1016/j.bbrc.2007.10.060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2129132/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17964542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nallamsetty,%20S.%20&amp;%20Waugh,%20D.%20S.%20Mutations%20that%20alter%20the%20equilibrium%20between%20open%20and%20closed%20conformations%20of%20Escherichia%20coli%20maltose-binding%20protein%20impede%20its%20ability%20to%20enhance%20the%20solubility%20of%20passenger%20proteins.%20Biochem.%20Biophys.%20Res.%20Commun.364,%20639%E2%80%93644%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Demir, M. et al. Structural snapshots of base excision by the cancer-associated variant MutY N146S reveal a retaining mechanism. <em>Nucleic Acids Res.</em><strong>51</strong>, 1034–1049 (2023).</cite> [<a href="https://doi.org/10.1093/nar/gkac1246" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9943663/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36631987/" class="usa-link">PubMed</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Trasviña-Arenas, C. H. et al. Structure of human MUTYH and functional profiling of cancer-associated variants reveal an allosteric network between its [4Fe-4S] cluster cofactor and active site required for DNA repair. 10.5281/zenodo.10161357 (2024).</cite> [<a href="https://doi.org/10.1038/s41467-025-58361-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12000561/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40234396/" class="usa-link">PubMed</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Golinelli, M.-P., Chmiel, N. H. &amp; David, S. S. Site-directed mutagenesis of the cysteine ligands to the [4Fe− 4S] cluster of Escherichia coli MutY. <em>Biochemistry</em><strong>38</strong>, 6997–7007 (1999).
</cite> [<a href="https://doi.org/10.1021/bi982300n" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10353811/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Golinelli,%20M.-P.,%20Chmiel,%20N.%20H.%20&amp;%20David,%20S.%20S.%20Site-directed%20mutagenesis%20of%20the%20cysteine%20ligands%20to%20the%20%5B4Fe%E2%88%92%204S%5D%20cluster%20of%20Escherichia%20coli%20MutY.%20Biochemistry38,%206997%E2%80%937007%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Kellie, J. L., Wilson, K. A. &amp; Wetmore, S. D. Standard role for a conserved aspartate or more direct involvement in deglycosylation? An ONIOM and MD investigation of adenine–DNA glycosylase. <em>Biochemistry</em><strong>52</strong>, 8753–8765 (2013).
</cite> [<a href="https://doi.org/10.1021/bi401310w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24168684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kellie,%20J.%20L.,%20Wilson,%20K.%20A.%20&amp;%20Wetmore,%20S.%20D.%20Standard%20role%20for%20a%20conserved%20aspartate%20or%20more%20direct%20involvement%20in%20deglycosylation?%20An%20ONIOM%20and%20MD%20investigation%20of%20adenine%E2%80%93DNA%20glycosylase.%20Biochemistry52,%208753%E2%80%938765%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Nikkel, D. J. &amp; Wetmore, S. D. Distinctive formation of a DNA–protein Cross-Link during the Repair of DNA Oxidative Damage: Insights into Human Disease from MD Simulations and QM/MM Calculations. <em>J. Am. Chem. Soc.</em><strong>145</strong>, 13114–13125 (2023).</cite> [<a href="https://doi.org/10.1021/jacs.3c01773" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37285289/" class="usa-link">PubMed</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Mol, C. D., Arvai, A. S., Begley, T. J., Cunningham, R. P. &amp; Tainer, J. A. Structure and activity of a thermostable thymine-DNA glycosylase: evidence for base twisting to remove mismatched normal DNA bases. <em>J. Mol. Biol.</em><strong>315</strong>, 373–384 (2002).
</cite> [<a href="https://doi.org/10.1006/jmbi.2001.5264" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11786018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mol,%20C.%20D.,%20Arvai,%20A.%20S.,%20Begley,%20T.%20J.,%20Cunningham,%20R.%20P.%20&amp;%20Tainer,%20J.%20A.%20Structure%20and%20activity%20of%20a%20thermostable%20thymine-DNA%20glycosylase:%20evidence%20for%20base%20twisting%20to%20remove%20mismatched%20normal%20DNA%20bases.%20J.%20Mol.%20Biol.315,%20373%E2%80%93384%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Fromme, J. C. &amp; Verdine, G. Structure of a trapped endonuclease III–DNA covalent intermediate. <em>EMBO J.</em><strong>22</strong>, 3461–3471 (2003).</cite> [<a href="https://doi.org/10.1093/emboj/cdg311" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC165637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12840008/" class="usa-link">PubMed</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Zuo, K. et al. The two redox states of the human NEET proteins’[2Fe–2S] clusters. <em>JBIC J. Biol. Inorg. Chem.</em><strong>26</strong>, 763–774 (2021).
</cite> [<a href="https://doi.org/10.1007/s00775-021-01890-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8463382/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34453614/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zuo,%20K.%20et%20al.%20The%20two%20redox%20states%20of%20the%20human%20NEET%20proteins%E2%80%99%5B2Fe%E2%80%932S%5D%20clusters.%20JBIC%20J.%20Biol.%20Inorg.%20Chem.26,%20763%E2%80%93774%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Nechushtai, R. et al. Allostery in the ferredoxin protein motif does not involve a conformational switch. <em>Proc. Natl. Acad. Sci. USA</em><strong>108</strong>, 2240–2245 (2011).
</cite> [<a href="https://doi.org/10.1073/pnas.1019502108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3038707/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21266547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nechushtai,%20R.%20et%20al.%20Allostery%20in%20the%20ferredoxin%20protein%20motif%20does%20not%20involve%20a%20conformational%20switch.%20Proc.%20Natl.%20Acad.%20Sci.%20USA108,%202240%E2%80%932245%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Boon, E. M., Livingston, A. L., Chmiel, N. H., David, S. S. &amp; Barton, J. K. DNA-mediated charge transport for DNA repair. <em>Proc. Natl. Acad. Sci.</em><strong>100</strong>, 12543–12547 (2003).
</cite> [<a href="https://doi.org/10.1073/pnas.2035257100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC240652/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14559969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boon,%20E.%20M.,%20Livingston,%20A.%20L.,%20Chmiel,%20N.%20H.,%20David,%20S.%20S.%20&amp;%20Barton,%20J.%20K.%20DNA-mediated%20charge%20transport%20for%20DNA%20repair.%20Proc.%20Natl.%20Acad.%20Sci.100,%2012543%E2%80%9312547%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Ha, Y. et al. Sulfur K-edge XAS studies of the effect of DNA binding on the [Fe4S4] site in EndoIII and MutY. <em>J. Am. Chem. Soc.</em><strong>139</strong>, 11434–11442 (2017).
</cite> [<a href="https://doi.org/10.1021/jacs.7b03966" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5568943/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28715891/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ha,%20Y.%20et%20al.%20Sulfur%20K-edge%20XAS%20studies%20of%20the%20effect%20of%20DNA%20binding%20on%20the%20%5BFe4S4%5D%20site%20in%20EndoIII%20and%20MutY.%20J.%20Am.%20Chem.%20Soc.139,%2011434%E2%80%9311442%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Bartels, P. L. et al. Electrochemistry of the [4Fe4S] cluster in base excision repair proteins: tuning the redox potential with DNA. <em>Langmuir</em><strong>33</strong>, 2523–2530 (2017).
</cite> [<a href="https://doi.org/10.1021/acs.langmuir.6b04581" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5423460/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28219007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bartels,%20P.%20L.%20et%20al.%20Electrochemistry%20of%20the%20%5B4Fe4S%5D%20cluster%20in%20base%20excision%20repair%20proteins:%20tuning%20the%20redox%20potential%20with%20DNA.%20Langmuir33,%202523%E2%80%932530%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Barton, J. K., Silva, R. M. &amp; O’Brien, E. Redox chemistry in the genome: emergence of the [4Fe4S] cofactor in repair and replication. <em>Annu. Rev. Biochem.</em><strong>88</strong>, 163–190 (2019).
</cite> [<a href="https://doi.org/10.1146/annurev-biochem-013118-110644" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6590699/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31220976/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barton,%20J.%20K.,%20Silva,%20R.%20M.%20&amp;%20O%E2%80%99Brien,%20E.%20Redox%20chemistry%20in%20the%20genome:%20emergence%20of%20the%20%5B4Fe4S%5D%20cofactor%20in%20repair%20and%20replication.%20Annu.%20Rev.%20Biochem.88,%20163%E2%80%93190%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Tse, E. C., Zwang, T. J. &amp; Barton, J. K. The oxidation state of [4Fe4S] clusters modulates the DNA-binding affinity of DNA repair proteins. <em>J. Am. Chem. Soc.</em><strong>139</strong>, 12784–12792 (2017).
</cite> [<a href="https://doi.org/10.1021/jacs.7b07230" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5929122/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28817778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tse,%20E.%20C.,%20Zwang,%20T.%20J.%20&amp;%20Barton,%20J.%20K.%20The%20oxidation%20state%20of%20%5B4Fe4S%5D%20clusters%20modulates%20the%20DNA-binding%20affinity%20of%20DNA%20repair%20proteins.%20J.%20Am.%20Chem.%20Soc.139,%2012784%E2%80%9312792%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Pinto, M. N., Ter Beek, J., Ekanger, L. A., Johansson, E. &amp; Barton, J. K. The [4Fe4S] cluster of yeast DNA polymerase ε is redox active and can undergo DNA-mediated signaling. <em>J. Am. Chem. Soc.</em><strong>143</strong>, 16147–16153 (2021).
</cite> [<a href="https://doi.org/10.1021/jacs.1c07150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8499023/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34559527/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pinto,%20M.%20N.,%20Ter%20Beek,%20J.,%20Ekanger,%20L.%20A.,%20Johansson,%20E.%20&amp;%20Barton,%20J.%20K.%20The%20%5B4Fe4S%5D%20cluster%20of%20yeast%20DNA%20polymerase%20%CE%B5%20is%20redox%20active%20and%20can%20undergo%20DNA-mediated%20signaling.%20J.%20Am.%20Chem.%20Soc.143,%2016147%E2%80%9316153%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Lin, J.-C., Singh, R. R. &amp; Cox, D. L. Theoretical study of DNA damage recognition via electron transfer from the [4Fe-4S] complex of MutY. <em>Biophys. J.</em><strong>95</strong>, 3259–3268 (2008).
</cite> [<a href="https://doi.org/10.1529/biophysj.108.132183" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2547449/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18599627/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin,%20J.-C.,%20Singh,%20R.%20R.%20&amp;%20Cox,%20D.%20L.%20Theoretical%20study%20of%20DNA%20damage%20recognition%20via%20electron%20transfer%20from%20the%20%5B4Fe-4S%5D%20complex%20of%20MutY.%20Biophys.%20J.95,%203259%E2%80%933268%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Teo, R. D., Du, X., Vera, H. L. T., Migliore, A. &amp; Beratan, D. N. Correlation between charge transport and base excision repair in the MutY–DNA glycosylase. <em>J. Phys. Chem. B</em><strong>125</strong>, 17–23 (2020).
</cite> [<a href="https://doi.org/10.1021/acs.jpcb.0c08598" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8444423/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33371674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Teo,%20R.%20D.,%20Du,%20X.,%20Vera,%20H.%20L.%20T.,%20Migliore,%20A.%20&amp;%20Beratan,%20D.%20N.%20Correlation%20between%20charge%20transport%20and%20base%20excision%20repair%20in%20the%20MutY%E2%80%93DNA%20glycosylase.%20J.%20Phys.%20Chem.%20B125,%2017%E2%80%9323%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>De Rosa, M., Barnes, R. P., Nyalapatla, P. R., Wipf, P. &amp; Opresko, P. L. OGG1 and MUTYH repair activities promote telomeric 8-oxoguanine induced cellular senescence. <em>Nat. Commun.</em><strong>16</strong>, 893 (2023).</cite> [<a href="https://doi.org/10.1038/s41467-024-55638-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11751180/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39837827/" class="usa-link">PubMed</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Mitra, D. et al. Characterization of [4Fe-4S] cluster vibrations and structure in nitrogenase Fe protein at three oxidation levels via combined NRVS, EXAFS, and DFT analyses.<em>J. Am. Chem. Soc.</em><strong>135</strong>, 2530–2543 (2013).
</cite> [<a href="https://doi.org/10.1021/ja307027n" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3644515/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23282058/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mitra,%20D.%20et%20al.%20Characterization%20of%20%5B4Fe-4S%5D%20cluster%20vibrations%20and%20structure%20in%20nitrogenase%20Fe%20protein%20at%20three%20oxidation%20levels%20via%20combined%20NRVS,%20EXAFS,%20and%20DFT%20analyses.J.%20Am.%20Chem.%20Soc.135,%202530%E2%80%932543%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Barnes, R. P. et al. Telomeric 8-oxo-guanine drives rapid premature senescence in the absence of telomere shortening. <em>Nat. Struct. Mol. Biol.</em><strong>29</strong>, 639–652 (2022).
</cite> [<a href="https://doi.org/10.1038/s41594-022-00790-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9287163/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35773409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barnes,%20R.%20P.%20et%20al.%20Telomeric%208-oxo-guanine%20drives%20rapid%20premature%20senescence%20in%20the%20absence%20of%20telomere%20shortening.%20Nat.%20Struct.%20Mol.%20Biol.29,%20639%E2%80%93652%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Maio, N. et al. An iron–sulfur cluster in the zinc-binding domain of the SARS-CoV-2 helicase modulates its RNA-binding and-unwinding activities. <em>Proc. Natl. Acad. Sci. USA</em><strong>120</strong>, e2303860120 (2023).
</cite> [<a href="https://doi.org/10.1073/pnas.2303860120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10438387/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37552760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maio,%20N.%20et%20al.%20An%20iron%E2%80%93sulfur%20cluster%20in%20the%20zinc-binding%20domain%20of%20the%20SARS-CoV-2%20helicase%20modulates%20its%20RNA-binding%20and-unwinding%20activities.%20Proc.%20Natl.%20Acad.%20Sci.%20USA120,%20e2303860120%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Heckman, K. L. &amp; Pease, L. R. Gene splicing and mutagenesis by PCR-driven overlap extension. <em>Nat. Protoc.</em><strong>2</strong>, 924–932 (2007).
</cite> [<a href="https://doi.org/10.1038/nprot.2007.132" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17446874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heckman,%20K.%20L.%20&amp;%20Pease,%20L.%20R.%20Gene%20splicing%20and%20mutagenesis%20by%20PCR-driven%20overlap%20extension.%20Nat.%20Protoc.2,%20924%E2%80%93932%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Conlon, S. G. et al. Cellular Repair of Synthetic Analogs of Oxidative DNA Damage Reveals a Key Structure–Activity Relationship of the Cancer-Associated MUTYH DNA Repair Glycosylase. <em>ACS Cent. Sci.</em><strong>10</strong>, 291–301 (2024).</cite> [<a href="https://doi.org/10.1021/acscentsci.3c00784" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10906249/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38435525/" class="usa-link">PubMed</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Nishihara, K., Kanemori, M., Kitagawa, M., Yanagi, H. &amp; Yura, T. Chaperone coexpression plasmids: differential and synergistic roles of DnaK-DnaJ-GrpE and GroEL-GroES in assisting folding of an allergen of Japanese cedar pollen, Cryj2, in Escherichia coli. <em>Appl. Environ. Microbiol.</em><strong>64</strong>, 1694–1699 (1998).
</cite> [<a href="https://doi.org/10.1128/aem.64.5.1694-1699.1998" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC106217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9572938/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nishihara,%20K.,%20Kanemori,%20M.,%20Kitagawa,%20M.,%20Yanagi,%20H.%20&amp;%20Yura,%20T.%20Chaperone%20coexpression%20plasmids:%20differential%20and%20synergistic%20roles%20of%20DnaK-DnaJ-GrpE%20and%20GroEL-GroES%20in%20assisting%20folding%20of%20an%20allergen%20of%20Japanese%20cedar%20pollen,%20Cryj2,%20in%20Escherichia%20coli.%20Appl.%20Environ.%20Microbiol.64,%201694%E2%80%931699%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Chu, A. M., Fettinger, J. C. &amp; David, S. S. Profiling base excision repair glycosylases with synthesized transition state analogs. <em>Bioorg. Med. Chem. Lett.</em><strong>21</strong>, 4969–4972 (2011).
</cite> [<a href="https://doi.org/10.1016/j.bmcl.2011.05.085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3156346/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21689934/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chu,%20A.%20M.,%20Fettinger,%20J.%20C.%20&amp;%20David,%20S.%20S.%20Profiling%20base%20excision%20repair%20glycosylases%20with%20synthesized%20transition%20state%20analogs.%20Bioorg.%20Med.%20Chem.%20Lett.21,%204969%E2%80%934972%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Kabsch, W. <em>XDS</em>. <em>Acta Crystallogr. Sect. D Biol Crystallogr.</em> <strong>66</strong>, 125–132 (2010).
</cite> [<a href="https://doi.org/10.1107/S0907444909047337" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2815665/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20124692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kabsch,%20W.%20XDS.%20Acta%20Crystallogr.%20Sect.%20D%20Biol%20Crystallogr.%C2%A066,%20125%E2%80%93132%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Afonine, P. V. et al. Towards automated crystallographic structure refinement with phenix. refine. <em>Acta Crystallogr. Sect. D Biol. Crystallogr.</em><strong>68</strong>, 352–367 (2012).
</cite> [<a href="https://doi.org/10.1107/S0907444912001308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322595/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22505256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Afonine,%20P.%20V.%20et%20al.%20Towards%20automated%20crystallographic%20structure%20refinement%20with%20phenix.%20refine.%20Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.68,%20352%E2%80%93367%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Emsley, P. &amp; Cowtan, K. Coot: model-building tools for molecular graphics. <em>Acta Crystallogr. Sect. D Biol. Crystallogr.</em><strong>60</strong>, 2126–2132 (2004).
</cite> [<a href="https://doi.org/10.1107/S0907444904019158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15572765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Emsley,%20P.%20&amp;%20Cowtan,%20K.%20Coot:%20model-building%20tools%20for%20molecular%20graphics.%20Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.60,%202126%E2%80%932132%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Pettersen, E. F. et al. UCSF Chimera—a visualization system for exploratory research and analysis. <em>J. Comput. Chem.</em><strong>25</strong>, 1605–1612 (2004).
</cite> [<a href="https://doi.org/10.1002/jcc.20084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15264254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pettersen,%20E.%20F.%20et%20al.%20UCSF%20Chimera%E2%80%94a%20visualization%20system%20for%20exploratory%20research%20and%20analysis.%20J.%20Comput.%20Chem.25,%201605%E2%80%931612%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Edgar, R. C. MUSCLE: multiple sequence alignment with high accuracy and high throughput. <em>Nucleic Acids Res.</em><strong>32</strong>, 1792–1797 (2004).
</cite> [<a href="https://doi.org/10.1093/nar/gkh340" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC390337/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15034147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Edgar,%20R.%20C.%20MUSCLE:%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput.%20Nucleic%20Acids%20Res.32,%201792%E2%80%931797%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Simonetti, F. L., Teppa, E., Chernomoretz, A., Nielsen, M. &amp; Marino Buslje, C. MISTIC: mutual information server to infer coevolution. <em>Nucleic Acids Res.</em><strong>41</strong>, W8–W14 (2013).
</cite> [<a href="https://doi.org/10.1093/nar/gkt427" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3692073/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23716641/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Simonetti,%20F.%20L.,%20Teppa,%20E.,%20Chernomoretz,%20A.,%20Nielsen,%20M.%20&amp;%20Marino%20Buslje,%20C.%20MISTIC:%20mutual%20information%20server%20to%20infer%20coevolution.%20Nucleic%20Acids%20Res.41,%20W8%E2%80%93W14%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Majumdar, C., Nuñez, N. N., Raetz, A. G., Khuu, C. &amp; David, S. S. Cellular assays for studying the Fe–S cluster containing base excision repair glycosylase MUTYH and homologs. in <em>Methods in Enzymology</em> Vol. 599, 69–99 (Elsevier, 2018).</cite> [<a href="https://doi.org/10.1016/bs.mie.2017.12.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6201757/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29746250/" class="usa-link">PubMed</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Waterhouse, A. et al. SWISS-MODEL: homology modelling of protein structures and complexes. <em>Nucleic Acids Res.</em><strong>46</strong>, W296–W303 (2018).
</cite> [<a href="https://doi.org/10.1093/nar/gky427" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6030848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29788355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Waterhouse,%20A.%20et%20al.%20SWISS-MODEL:%20homology%20modelling%20of%20protein%20structures%20and%20complexes.%20Nucleic%20Acids%20Res.46,%20W296%E2%80%93W303%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Webb, B. &amp; Sali, A. Comparative protein structure modeling using MODELLER. <em>Curr. Protoc. Bioinform.</em><strong>54</strong>, 5.6. 1–5.6. 37 (2016).</cite> [<a href="https://doi.org/10.1002/cpbi.3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5031415/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27322406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Webb,%20B.%20&amp;%20Sali,%20A.%20Comparative%20protein%20structure%20modeling%20using%20MODELLER.%20Curr.%20Protoc.%20Bioinform.54,%205.6.%201%E2%80%935.6.%2037%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Cheng, X. et al. Dynamic behavior of DNA base pairs containing 8-oxoguanine. <em>J. Am. Chem. Soc.</em><strong>127</strong>, 13906–13918 (2005).
</cite> [<a href="https://doi.org/10.1021/ja052542s" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8295720/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16201812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng,%20X.%20et%20al.%20Dynamic%20behavior%20of%20DNA%20base%20pairs%20containing%208-oxoguanine.%20J.%20Am.%20Chem.%20Soc.127,%2013906%E2%80%9313918%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Wang, J., Wolf, R. M., Caldwell, J. W., Kollman, P. A. &amp; Case, D. A. Development and testing of a general amber force field. <em>J. Comput. Chem.</em><strong>25</strong>, 1157–1174 (2004).
</cite> [<a href="https://doi.org/10.1002/jcc.20035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15116359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20J.,%20Wolf,%20R.%20M.,%20Caldwell,%20J.%20W.,%20Kollman,%20P.%20A.%20&amp;%20Case,%20D.%20A.%20Development%20and%20testing%20of%20a%20general%20amber%20force%20field.%20J.%20Comput.%20Chem.25,%201157%E2%80%931174%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Wang, J., Wang, W., Kollman, P. A. &amp; Case, D. A. Automatic atom type and bond type perception in molecular mechanical calculations. <em>J. Mol. Graph. Model.</em><strong>25</strong>, 247–260 (2006).
</cite> [<a href="https://doi.org/10.1016/j.jmgm.2005.12.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16458552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20J.,%20Wang,%20W.,%20Kollman,%20P.%20A.%20&amp;%20Case,%20D.%20A.%20Automatic%20atom%20type%20and%20bond%20type%20perception%20in%20molecular%20mechanical%20calculations.%20J.%20Mol.%20Graph.%20Model.25,%20247%E2%80%93260%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Smith, D. M., Xiong, Y., Straatsma, T., Rosso, K. M. &amp; Squier, T. C. Force-field development and molecular dynamics of [NiFe] hydrogenase. <em>J. Chem. Theory Comput.</em><strong>8</strong>, 2103–2114 (2012).
</cite> [<a href="https://doi.org/10.1021/ct300185u" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26593842/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith,%20D.%20M.,%20Xiong,%20Y.,%20Straatsma,%20T.,%20Rosso,%20K.%20M.%20&amp;%20Squier,%20T.%20C.%20Force-field%20development%20and%20molecular%20dynamics%20of%20%5BNiFe%5D%20hydrogenase.%20J.%20Chem.%20Theory%20Comput.8,%202103%E2%80%932114%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Peters, M. B. et al. Structural survey of zinc-containing proteins and development of the zinc AMBER force field (ZAFF). <em>J. Chem. Theory Comput.</em><strong>6</strong>, 2935–2947 (2010).
</cite> [<a href="https://doi.org/10.1021/ct1002626" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2941202/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20856692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peters,%20M.%20B.%20et%20al.%20Structural%20survey%20of%20zinc-containing%20proteins%20and%20development%20of%20the%20zinc%20AMBER%20force%20field%20(ZAFF).%20J.%20Chem.%20Theory%20Comput.6,%202935%E2%80%932947%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Olsson, M. H., Søndergaard, C. R., Rostkowski, M. &amp; Jensen, J. H. PROPKA3: consistent treatment of internal and surface residues in empirical p K a predictions. <em>J. Chem. Theory Comput.</em><strong>7</strong>, 525–537 (2011).
</cite> [<a href="https://doi.org/10.1021/ct100578z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26596171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Olsson,%20M.%20H.,%20S%C3%B8ndergaard,%20C.%20R.,%20Rostkowski,%20M.%20&amp;%20Jensen,%20J.%20H.%20PROPKA3:%20consistent%20treatment%20of%20internal%20and%20surface%20residues%20in%20empirical%20p%20K%20a%20predictions.%20J.%20Chem.%20Theory%20Comput.7,%20525%E2%80%93537%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Chen, V. B. et al. MolProbity: all-atom structure validation for macromolecular crystallography. <em>Acta Crystallogr. Sect. D Biol. Crystallogr.</em><strong>66</strong>, 12–21 (2010).
</cite> [<a href="https://doi.org/10.1107/S0907444909042073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2803126/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20057044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20V.%20B.%20et%20al.%20MolProbity:%20all-atom%20structure%20validation%20for%20macromolecular%20crystallography.%20Acta%20Crystallogr.%20Sect.%20D%20Biol.%20Crystallogr.66,%2012%E2%80%9321%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Schafmeister, C., Ross, W. &amp; Romanovski, V. LEAP; University of California: San Francisco, 1995. <em>Google Scholar There is no corresponding record for this reference</em>.</cite>
</li>
<li id="CR70">
<span class="label">70.</span><cite>Case, D. A. et al. <em>Amber 2021</em> (University of California, 2021).</cite>
</li>
<li id="CR71">
<span class="label">71.</span><cite>Jorgensen, W. L., Chandrasekhar, J., Madura, J. D., Impey, R. W. &amp; Klein, M. L. Comparison of simple potential functions for simulating liquid water. <em>J. Chem. Phys.</em><strong>79</strong>, 926–935 (1983).</cite> [<a href="https://scholar.google.com/scholar_lookup?Jorgensen,%20W.%20L.,%20Chandrasekhar,%20J.,%20Madura,%20J.%20D.,%20Impey,%20R.%20W.%20&amp;%20Klein,%20M.%20L.%20Comparison%20of%20simple%20potential%20functions%20for%20simulating%20liquid%20water.%20J.%20Chem.%20Phys.79,%20926%E2%80%93935%20(1983)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Tian, C. et al. ff19SB: Amino-acid-specific protein backbone parameters trained against quantum mechanics energy surfaces in solution. <em>J. Chem. Theory Comput.</em><strong>16</strong>, 528–552 (2019).
</cite> [<a href="https://doi.org/10.1021/acs.jctc.9b00591" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31714766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tian,%20C.%20et%20al.%20ff19SB:%20Amino-acid-specific%20protein%20backbone%20parameters%20trained%20against%20quantum%20mechanics%20energy%20surfaces%20in%20solution.%20J.%20Chem.%20Theory%20Comput.16,%20528%E2%80%93552%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Galindo-Murillo, R. et al. Assessing the current state of amber force field modifications for DNA. <em>J. Chem. Theory Comput.</em><strong>12</strong>, 4114–4127 (2016).
</cite> [<a href="https://doi.org/10.1021/acs.jctc.6b00186" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4980684/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27300587/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Galindo-Murillo,%20R.%20et%20al.%20Assessing%20the%20current%20state%20of%20amber%20force%20field%20modifications%20for%20DNA.%20J.%20Chem.%20Theory%20Comput.12,%204114%E2%80%934127%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Gillespie, D. T. The chemical Langevin equation. <em>J. Chem. Phys.</em><strong>113</strong>, 297–306 (2000).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gillespie,%20D.%20T.%20The%20chemical%20Langevin%20equation.%20J.%20Chem.%20Phys.113,%20297%E2%80%93306%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Berendsen, H. J., Postma, J. V., Van Gunsteren, W. F., DiNola, A. &amp; Haak, J. R. Molecular dynamics with coupling to an external bath. <em>J. Chem. Phys.</em><strong>81</strong>, 3684–3690 (1984).</cite> [<a href="https://scholar.google.com/scholar_lookup?Berendsen,%20H.%20J.,%20Postma,%20J.%20V.,%20Van%20Gunsteren,%20W.%20F.,%20DiNola,%20A.%20&amp;%20Haak,%20J.%20R.%20Molecular%20dynamics%20with%20coupling%20to%20an%20external%20bath.%20J.%20Chem.%20Phys.81,%203684%E2%80%933690%20(1984)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Davidchack, R. L., Handel, R. &amp; Tretyakov, M. Langevin thermostat for rigid body dynamics. <em>J. Chem. Phys.</em><strong>130</strong>, 234101 (2009).</cite> [<a href="https://doi.org/10.1063/1.3149788" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19548705/" class="usa-link">PubMed</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Ryckaert, J.-P., Ciccotti, G. &amp; Berendsen, H. J. Numerical integration of the cartesian equations of motion of a system with constraints: molecular dynamics of n-alkanes. <em>J. Comput. Phys.</em><strong>23</strong>, 327–341 (1977).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ryckaert,%20J.-P.,%20Ciccotti,%20G.%20&amp;%20Berendsen,%20H.%20J.%20Numerical%20integration%20of%20the%20cartesian%20equations%20of%20motion%20of%20a%20system%20with%20constraints:%20molecular%20dynamics%20of%20n-alkanes.%20J.%20Comput.%20Phys.23,%20327%E2%80%93341%20(1977)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Wu, X. &amp; Brooks, B. R. Isotropic periodic sum: a method for the calculation of long-range interactions. <em>J. Chem. Phys.</em><strong>122</strong>, 44107 (2005).</cite> [<a href="https://doi.org/10.1063/1.1836733" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6488042/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15740235/" class="usa-link">PubMed</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Essmann, U. et al. A smooth particle mesh Ewald method. <em>J. Chem. Phys.</em><strong>103</strong>, 8577–8593 (1995).</cite> [<a href="https://scholar.google.com/scholar_lookup?Essmann,%20U.%20et%20al.%20A%20smooth%20particle%20mesh%20Ewald%20method.%20J.%20Chem.%20Phys.103,%208577%E2%80%938593%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Roe, D. R. &amp; Cheatham III, T. E. PTRAJ and CPPTRAJ: software for processing and analysis of molecular dynamics trajectory data. <em>J. Chem. Theory Comput.</em><strong>9</strong>, 3084–3095 (2013).</cite> [<a href="https://doi.org/10.1021/ct400341p" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26583988/" class="usa-link">PubMed</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Bakan, A., Meireles, L. M. &amp; Bahar, I. ProDy: protein dynamics inferred from theory and experiments. <em>Bioinformatics</em><strong>27</strong>, 1575–1577 (2011).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr168" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3102222/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21471012/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bakan,%20A.,%20Meireles,%20L.%20M.%20&amp;%20Bahar,%20I.%20ProDy:%20protein%20dynamics%20inferred%20from%20theory%20and%20experiments.%20Bioinformatics27,%201575%E2%80%931577%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Graham, S. E., Syeda, F. &amp; Cisneros, G. A. s. Computational prediction of residues involved in fidelity checking for DNA synthesis in DNA polymerase I. <em>Biochemistry</em><strong>51</strong>, 2569–2578 (2012).
</cite> [<a href="https://doi.org/10.1021/bi201856m" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22397306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Graham,%20S.%20E.,%20Syeda,%20F.%20&amp;%20Cisneros,%20G.%20A.%20s.%20Computational%20prediction%20of%20residues%20involved%20in%20fidelity%20checking%20for%20DNA%20synthesis%20in%20DNA%20polymerase%20I.%20Biochemistry51,%202569%E2%80%932578%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Leddin, E., Group, C. &amp; Cisneros, G. CisnerosResearch/AMBER-EDA: First Release. (DOI, 2020).</cite>
</li>
<li id="CR84">
<span class="label">84.</span><cite>Melo, M. C., Bernardi, R. C., De La Fuente-Nunez, C. &amp; Luthey-Schulten, Z. Generalized correlation-based dynamical network analysis: a new high-performance approach for identifying allosteric communications in molecular dynamics trajectories. <em>J. Chem. Phys.</em><strong>153</strong>, 134104 (2020).</cite> [<a href="https://doi.org/10.1063/5.0018980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33032427/" class="usa-link">PubMed</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Sethi, A., Eargle, J., Black, A. A. &amp; Luthey-Schulten, Z. Dynamical networks in tRNA: protein complexes. <em>Proc. Natl. Acad. Sci. USA</em><strong>106</strong>, 6620–6625 (2009).
</cite> [<a href="https://doi.org/10.1073/pnas.0810961106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2672494/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19351898/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sethi,%20A.,%20Eargle,%20J.,%20Black,%20A.%20A.%20&amp;%20Luthey-Schulten,%20Z.%20Dynamical%20networks%20in%20tRNA:%20protein%20complexes.%20Proc.%20Natl.%20Acad.%20Sci.%20USA106,%206620%E2%80%936625%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Karczewski, K. J. et al. The ExAC browser: displaying reference data information from over 60 000 exomes. <em>Nucleic Acids Res.</em><strong>45</strong>, D840–D845 (2017).
</cite> [<a href="https://doi.org/10.1093/nar/gkw971" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5210650/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27899611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karczewski,%20K.%20J.%20et%20al.%20The%20ExAC%20browser:%20displaying%20reference%20data%20information%20from%20over%2060%20000%20exomes.%20Nucleic%20Acids%20Res.45,%20D840%E2%80%93D845%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. <em>Nature</em><strong>581</strong>, 434–443 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-2308-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7334197/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32461654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karczewski,%20K.%20J.%20et%20al.%20The%20mutational%20constraint%20spectrum%20quantified%20from%20variation%20in%20141,456%20humans.%20Nature581,%20434%E2%80%93443%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Taliun, D. et al. Sequencing of 53,831 diverse genomes from the NHLBI TOPMed Program. <em>Nature</em><strong>590</strong>, 290–299 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03205-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7875770/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33568819/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Taliun,%20D.%20et%20al.%20Sequencing%20of%2053,831%20diverse%20genomes%20from%20the%20NHLBI%20TOPMed%20Program.%20Nature590,%20290%E2%80%93299%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12000561/bin/41467_2025_58361_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (5.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12000561/bin/41467_2025_58361_MOESM2_ESM.txt" data-ga-action="click_feat_suppl" class="usa-link">Suppplementary Data 1</a><sup> (940.5KB, txt) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12000561/bin/41467_2025_58361_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (7.8MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12000561/bin/41467_2025_58361_MOESM4_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Description of Supplementary Data files</a><sup> (13.6KB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12000561/bin/41467_2025_58361_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (95.3KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12000561/bin/41467_2025_58361_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (43KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The atomic coordinates and structure factors generated in this study have been deposited in the Protein Data Bank (<a href="http://www.rcsb.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.rcsb.org</a>) under accession codes <a href="https://doi.org/10.2210/pdb8FAY/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8FAY</a> (human MUTYH complexed with DNA containing the transition state analog 1N) and <a href="https://doi.org/10.2210/pdb9BS2/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9BS2</a> (Gs MutY R149Q complexed with DNA containing the product analog THF). The initial coordinates and parameter files for each WT and mutant systems in human and mouse MUTYH generated as part of the molecular dynamics studies, as well as parameters describing the [4Fe-4S] cluster and Zn<sup>2+</sup> Linchpin motif have been deposited to the Zenodo repository<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> found at 10.5281/zenodo.10161357. <a href="#Sec22" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-58361-w"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_58361.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12000561/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12000561/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12000561%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12000561/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12000561/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12000561/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40234396/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12000561/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40234396/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12000561/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12000561/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="y2MYU31tRfeVKFrIwqAx0By935wFCeAcNAZxBNGe2Qv4jlhOcbZ8RZNpeYbYpU7E">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
