
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Multi‐Omic Evaluation of PLK1 Inhibitor—Onvansertib—In Colorectal Cancer Spheroids - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="616CF1338094D44300F1330056FF7709.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11976698/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Journal of Mass Spectrometry">
<meta name="citation_title" content="Multi‐Omic Evaluation of PLK1 Inhibitor—Onvansertib—In Colorectal Cancer Spheroids">
<meta name="citation_author" content="Brian D Fries">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, The Ohio State University, Columbus, Ohio, USA">
<meta name="citation_author" content="Emily R Sekera">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, The Ohio State University, Columbus, Ohio, USA">
<meta name="citation_author" content="Joseph H Holbrook">
<meta name="citation_author_institution" content="Ohio State Biochemistry Program, The Ohio State University, Columbus, Ohio, USA">
<meta name="citation_author" content="Amanda B Hummon">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, The Ohio State University, Columbus, Ohio, USA">
<meta name="citation_author_institution" content="Comprehensive Cancer Center, The Ohio State University, Columbus, Ohio, USA">
<meta name="citation_publication_date" content="2025 Apr 8">
<meta name="citation_volume" content="60">
<meta name="citation_issue" content="5">
<meta name="citation_firstpage" content="e5137">
<meta name="citation_doi" content="10.1002/jms.5137">
<meta name="citation_pmid" content="40197665">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11976698/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11976698/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11976698/pdf/JMS-60-e5137.pdf">
<meta name="description" content="Polo‐like kinase 1 (Plk1) is a serine/threonine kinase involved in regulating the cell cycle. It is activated by aurora kinase B along with the cofactors Borealin, INCE, and survivin. Plk1 is involved in the development of resistances to ...">
<meta name="og:title" content="Multi‐Omic Evaluation of PLK1 Inhibitor—Onvansertib—In Colorectal Cancer Spheroids">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Polo‐like kinase 1 (Plk1) is a serine/threonine kinase involved in regulating the cell cycle. It is activated by aurora kinase B along with the cofactors Borealin, INCE, and survivin. Plk1 is involved in the development of resistances to ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC11976698/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="11976698">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/jms.5137"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/JMS-60-e5137.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11976698%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/11976698/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/11976698/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11976698/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/jms.5137" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">J Mass Spectrom</button></div>. 2025 Apr 8;60(5):e5137. doi: <a href="https://doi.org/10.1002/jms.5137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/jms.5137</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22J%20Mass%20Spectrom%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22J%20Mass%20Spectrom%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22J%20Mass%20Spectrom%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22J%20Mass%20Spectrom%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Multi‐Omic Evaluation of PLK1 Inhibitor—Onvansertib—In Colorectal Cancer Spheroids</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fries%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Brian D Fries</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Brian D Fries</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry and Biochemistry, The Ohio State University, Columbus, Ohio, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fries%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Brian D Fries</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sekera%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Emily R Sekera</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Emily R Sekera</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry and Biochemistry, The Ohio State University, Columbus, Ohio, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sekera%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Emily R Sekera</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Holbrook%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Joseph H Holbrook</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Joseph H Holbrook</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Ohio State Biochemistry Program, The Ohio State University, Columbus, Ohio, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Holbrook%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Joseph H Holbrook</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hummon%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Amanda B Hummon</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Amanda B Hummon</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemistry and Biochemistry, The Ohio State University, Columbus, Ohio, USA</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Comprehensive Cancer Center, The Ohio State University, Columbus, Ohio, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hummon%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Amanda B Hummon</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="jms5137-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Chemistry and Biochemistry, The Ohio State University, Columbus, Ohio, USA</div>
<div id="jms5137-aff-0002">
<sup>
<sup>2</sup>
</sup>Ohio State Biochemistry Program, The Ohio State University, Columbus, Ohio, USA</div>
<div id="jms5137-aff-0003">
<sup>
<sup>3</sup>
</sup>Comprehensive Cancer Center, The Ohio State University, Columbus, Ohio, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong>
 
Amanda B. Hummon (<span>hummon.1@osu.edu</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Mar 25; Received 2025 Jan 15; Accepted 2025 Mar 28; Issue date 2025 May.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Journal of Mass Spectrometry</em> published by John Wiley &amp; Sons Ltd.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC11976698  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40197665/" class="usa-link">40197665</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>Polo‐like kinase 1 (Plk1) is a serine/threonine kinase involved in regulating the cell cycle. It is activated by aurora kinase B along with the cofactors Borealin, INCE, and survivin. Plk1 is involved in the development of resistances to chemotherapeutics such as doxorubicin, Taxol, and gemcitabine. It has been shown that patients with higher levels of Plk1 have lower survival rates. Onvansertib is a competitive ATP inhibitor for Plk1 in clinical trials for the treatment of tumors and has recently entered a trial for the treatment of <em>KRAS</em> mutant colorectal cancers (CRCs). In this study, we conducted an untargeted liquid chromatography–mass spectrometry (LC–MS) proteomics study as well as an untargeted lipidomics analysis of HCT 116 spheroids treated with onvansertib over a 72‐h treatment time‐course experiment. Mass spectrometry imaging (MSI) showed that onvansertib begins to accumulate most prominently after 12 h of treatment and continues to accumulate through 72 h. Proteomic results displayed alterations to cell cycle control proteins and an increasing abundance of aurora kinase B and Borealin. The proteomics data also showed alterations to many lipid metabolism enzymes. The MSI lipidomics data indicated alterations to phosphatidylcholine lipids, with many lipids increasing in abundance over time or increasing until 12 h of onvansertib treatment and decreasing after that time point. In summary, these results suggest that onvansertib is causing cells within the spheroid to halt at a certain phase of the cell cycle in accordance with previous literature. Our findings suggest the S phase is likely interrupted, with observed alterations in cell cycle control proteins and PC lipid abundance.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> cell cycle, colorectal cancer, lipidomics, mass spectrometry imaging, Plk‐1 inhibition, proteomics, spatial lipidomics, spheroids</p></section></section><section id="jms5137-sec-0001"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>The Polo gene encodes a serine/threonine kinase first discovered in 
<em>Drosophila melanogaster</em>
 flies in 1988 by Claudio Sunkel and David Glover [<a href="#jms5137-bib-0001" class="usa-link" aria-describedby="jms5137-bib-0001">1</a>]. A homologous kinase was mapped to humans in 1994, giving rise to the class of Polo‐like kinases (Plks) [<a href="#jms5137-bib-0002" class="usa-link" aria-describedby="jms5137-bib-0002">2</a>]. There are five Plks known to exist in humans, with Plk1 being the most studied. Plk1 is activated by phosphorylation from aurora kinase A along with the cofactor Borealin [<a href="#jms5137-bib-0003" class="usa-link" aria-describedby="jms5137-bib-0003">3</a>]. Plk1 has also been shown to be activated by aurora kinase B with the cofactors Borealin, INCE, and survivin [<a href="#jms5137-bib-0003" class="usa-link" aria-describedby="jms5137-bib-0003">3</a>, <a href="#jms5137-bib-0004" class="usa-link" aria-describedby="jms5137-bib-0004">4</a>]. Plk1 plays a very important role in regulating the cell cycle by phosphorylating CDC25 to activate the cyclin B/cdc2 complex, thus initiating cell proliferation [<a href="#jms5137-bib-0005" class="usa-link" aria-describedby="jms5137-bib-0005">5</a>]. Plk1 expression increases during the S phase of the cell cycle, followed by a peak at the G<sub>2</sub>‐M transition, and lastly decreases after mitosis [<a href="#jms5137-bib-0002" class="usa-link" aria-describedby="jms5137-bib-0002">2</a>]. It has been observed that patients with cancer with high levels of Plk1 expression have a lower 3‐year survival rate than those with lower expression [<a href="#jms5137-bib-0006" class="usa-link" aria-describedby="jms5137-bib-0006">6</a>]. Plk1 is involved in developed resistance of many chemotherapeutics, such as doxorubicin [<a href="#jms5137-bib-0007" class="usa-link" aria-describedby="jms5137-bib-0007">7</a>, <a href="#jms5137-bib-0008" class="usa-link" aria-describedby="jms5137-bib-0008">8</a>, <a href="#jms5137-bib-0009" class="usa-link" aria-describedby="jms5137-bib-0009">9</a>], taxol [<a href="#jms5137-bib-0010" class="usa-link" aria-describedby="jms5137-bib-0010">10</a>, <a href="#jms5137-bib-0011" class="usa-link" aria-describedby="jms5137-bib-0011">11</a>], and gemcitabine [<a href="#jms5137-bib-0012" class="usa-link" aria-describedby="jms5137-bib-0012">12</a>, <a href="#jms5137-bib-0013" class="usa-link" aria-describedby="jms5137-bib-0013">13</a>], to name a few. Understanding ways to inhibit Plk1 will help increase the efficacy of many therapeutics and has the potential to improve survival rates in patients with cancer.</p>
<p>Onvansertib is a competitive ATP inhibitor that is specific to Plk‐1 [<a href="#jms5137-bib-0014" class="usa-link" aria-describedby="jms5137-bib-0014">14</a>, <a href="#jms5137-bib-0015" class="usa-link" aria-describedby="jms5137-bib-0015">15</a>]. Onvansertib has been shown to cause a dose‐dependent response in multiple different cancer cell lines [<a href="#jms5137-bib-0015" class="usa-link" aria-describedby="jms5137-bib-0015">15</a>] as well as impair cell regrowth in HT‐29 colorectal cancer (CRC) and head and neck squamous cell carcinoma (HNSCC) cells [<a href="#jms5137-bib-0016" class="usa-link" aria-describedby="jms5137-bib-0016">16</a>]. A recent Phase Ib study in patients with relapsed or refractory acute myeloid leukemia (AML) showed that onvansertib, in combination with cytarabine or decitabine, resulted in decreased levels of mutant circulating tumor DNA [<a href="#jms5137-bib-0017" class="usa-link" aria-describedby="jms5137-bib-0017">17</a>]. A Phase IB/II clinical trial is currently underway to understand onvansertib treatment in combination with FOLFIRI and bevacizumab for patients with metastatic CRC containing a KRAS oncogenic mutation [<a href="#jms5137-bib-0018" class="usa-link" aria-describedby="jms5137-bib-0018">18</a>]. Many of the initial studies with onvansertib have been completed in two‐dimensional cell cultures, which lack the complexity of an in vivo human tumor. A three‐dimensional cell culture model—such as a spheroids—provides a much better conduit to understand the pharmacokinetics of a compound.</p>
<p>Spheroids are a three‐dimensional cell culture technique originally developed by Robert Sutherland in the 1970s [<a href="#jms5137-bib-0019" class="usa-link" aria-describedby="jms5137-bib-0019">19</a>]. Spheroids grow in a radially symmetric manner, developing distinct cell populations as well as various chemical and nutritional gradients. The core of a spheroid consists of largely necrotic cells with an acidic pH and very low levels of oxygen [<a href="#jms5137-bib-0019" class="usa-link" aria-describedby="jms5137-bib-0019">19</a>, <a href="#jms5137-bib-0020" class="usa-link" aria-describedby="jms5137-bib-0020">20</a>, <a href="#jms5137-bib-0021" class="usa-link" aria-describedby="jms5137-bib-0021">21</a>, <a href="#jms5137-bib-0022" class="usa-link" aria-describedby="jms5137-bib-0022">22</a>]. The middle layer consists of quiescent cells and the outer layer consists of proliferating cells [<a href="#jms5137-bib-0019" class="usa-link" aria-describedby="jms5137-bib-0019">19</a>]. Spheroids and other more complex cell culture systems are gaining popularity for assessing therapeutics due to the passage of the FDA Modernization Act 2.0 in 2022 [<a href="#jms5137-bib-0023" class="usa-link" aria-describedby="jms5137-bib-0023">23</a>]. Much work has been completed recently to characterize the distinct molecular differences between 2D cell cultures and 3D cell cultures [<a href="#jms5137-bib-0024" class="usa-link" aria-describedby="jms5137-bib-0024">24</a>, <a href="#jms5137-bib-0025" class="usa-link" aria-describedby="jms5137-bib-0025">25</a>, <a href="#jms5137-bib-0026" class="usa-link" aria-describedby="jms5137-bib-0026">26</a>, <a href="#jms5137-bib-0027" class="usa-link" aria-describedby="jms5137-bib-0027">27</a>, <a href="#jms5137-bib-0028" class="usa-link" aria-describedby="jms5137-bib-0028">28</a>, <a href="#jms5137-bib-0029" class="usa-link" aria-describedby="jms5137-bib-0029">29</a>]. Prior studies have shown that there are molecular differences when monolayer cultures are treated with a therapeutic compared with their spheroid counterpart [<a href="#jms5137-bib-0025" class="usa-link" aria-describedby="jms5137-bib-0025">25</a>, <a href="#jms5137-bib-0026" class="usa-link" aria-describedby="jms5137-bib-0026">26</a>, <a href="#jms5137-bib-0029" class="usa-link" aria-describedby="jms5137-bib-0029">29</a>]. One of the benefits of using spheroids for therapeutic studies is the ability to determine how well a pharmaceutical compound will penetrate into a spheroid [<a href="#jms5137-bib-0030" class="usa-link" aria-describedby="jms5137-bib-0030">30</a>], as a predictor of similar behavior in a tumor. Using mass spectrometry imaging (MSI) techniques, one can simultaneously track how deep a therapeutic has penetrated into a spheroid, assess drug metabolism, and map changes to endogenous small molecules throughout the spheroid [<a href="#jms5137-bib-0031" class="usa-link" aria-describedby="jms5137-bib-0031">31</a>].</p>
<p>In this study, we map the spatial distribution of onvansertib in HCT 116 spheroids over 72 h using MSI. Data independent acquisition (DIA) proteomics was also conducted at each time point and the proteome and lipidome were analyzed to further understand onvansertib actions in HCT 116 CRC spheroids.</p></section><section id="jms5137-sec-0002"><h2 class="pmc_sec_title">2. Methods</h2>
<section id="jms5137-sec-0003"><h3 class="pmc_sec_title">2.1. Chemicals</h3>
<p>Liquid chromatography–mass spectrometry (LC–MS) grade water, acetonitrile (ACN), 0.1% formic acid in water, and 0.1% formic acid in acetonitrile were obtained from Burdick &amp; Jackson. Dimethyl sulfoxide (DMSO) was obtained from MP Biomedicals. Sodium dodecyl sulfate (SDS), sodium orthovanadate, sodium fluoride, sodium pyrophosphate, β‐glycerophosphate, trifluoroacetic acid (TFA), 2,5‐dihydroxybenzoic acid (DHB), α‐cyano‐4‐hydroxycinnamic acid (CHCA), and triethylammonium bicarbonate (TEAB, 1 M, pH = 8.5) were obtained from Sigma‐Aldrich. Methanol was obtained from Fisher Scientific. Onvansertib (NMS‐P937) was obtained from Selleckchem. The Cell Titer Glo® 3D cell viability assay was obtained from Promega.</p></section><section id="jms5137-sec-0004"><h3 class="pmc_sec_title">2.2. Spheroid Culture</h3>
<p>HCT 116 cells were acquired from American Type Culture Collection (ATCC) and cultured in McCoy's 5A medium supplemented with 1% <span class="font-variant-small-caps">l</span>‐Glutamine, 1% Penicillin/Streptomycin, and 10% fetal bovine serum (FBS). HCT 116 cells were grown in monolayers at 37°C in 5% CO<sub>2</sub> until they reach approximately 80% confluency. Spheroids were cultured using the liquid overlay method as previously described [<a href="#jms5137-bib-0032" class="usa-link" aria-describedby="jms5137-bib-0032">32</a>, <a href="#jms5137-bib-0033" class="usa-link" aria-describedby="jms5137-bib-0033">33</a>]. In brief, outer moats of a 96‐well plate were filled with 1X PBS. A 1.5% agarose solution made of McCoy's 5A was added to the bottom of each well and allowed to cool. HCT 116 cells in suspension were added to each well at 7000 cells per well. Seeded cells were spun at 200 g for 10 min, prior to placement in the incubator to ensure proper spheroid formation. HCT 116 spheroids were grown for four days untouched. After four days, half of the initial media volume was changed every 48 h until Day 12. Spheroids were treated with onvansertib on Day 12 and harvested at each respective time point. Harvested spheroids were washed with 1X PBS and either stored at −80°C for proteomics analysis or embedded in 20% (w/v) gelatin for MALDI‐MSI.</p></section><section id="jms5137-sec-0005"><h3 class="pmc_sec_title">2.3. IC<sub>50</sub> Determination</h3>
<p>The 50% inhibitory concentration (IC<sub>50</sub>) of onvansertib was determined by using the Cell Titer Glo® 3D. Spheroids were incubated with onvansertib for 72 h in concentrations ranging from 250 to 0.120 μM. After 72 h, 100 μL of media were removed from the spheroids and 100 μL of reagent were added. The spheroids were vortexed for 5 min followed by incubation with the reagent for 25 min. The total luminescence intensity was recorded and visualized using Graph Pad Prism to calculate the IC<sub>50</sub>.</p>
<section id="jms5137-sec-0006"><h4 class="pmc_sec_title">2.3.1. MALDI‐MSI</h4>
<p>Spheroids were sectioned at 10 μm thickness using a cryostat with a chamber and chuck head temperature of −30°C. Samples were thaw mounted onto indium tin oxide‐coated glass slides (Delta Technologies). Matrix solutions of CHCA (10 mg/mL in 50% ACN with 0.1% TFA) or DHB (10 mg/mL in 50% ACN with 0.1% TFA) were used to coat samples using an HTX Imaging M5 TM‐Sprayer. Sprayer parameters are summarized in Table <a href="#jms5137-supitem-0001" class="usa-link">S1</a>. Samples were analyzed immediately after application of MALDI matrix.</p>
<p>MALDI‐MSI spectra were acquired using a Bruker timsTOF fleX with MALDI‐2. Mass spectra were acquired in positive ion mode with a mass range set to acquire between 200 and 1800 Da. The laser spot size was set to beamscan with a resulting field size of 20 × 20 μm at a frequency of 1000 Hz at 70% laser power with MALDI‐2 enabled. Spatial raster distance was set to 20 μm along the x and y axes accumulating 500 shots per pixel. External calibration was completed using Agilent ESI‐TOF calibration standard. Samples were analyzed in biological triplicate.</p></section></section><section id="jms5137-sec-0007"><h3 class="pmc_sec_title">2.4. Lipid Tandem MS</h3>
<p>HCT 116 spheroids were lysed in water using a probe sonicator. A 10‐mg/mL matrix solution of CHCA was prepared with 50% ACN and 0.1% TFA. Equal parts matrix solution and cell lysate were mixed and spotted onto a MALDI plate. Putative lipid ions discovered within the MALDI‐MSI analysis were analyzed within the timsTOF fleX mass spectrometer. The mass spectrometer was calibrated with red phosphorus. Ions were isolated in a 1.50‐<em>m/z</em> wide isolation width and fragmented with CID at varying collisional energies (<span class="font-variant-small-caps">ce</span>). Lipid fragments were manually annotated for lipid identification.</p></section><section id="jms5137-sec-0008"><h3 class="pmc_sec_title">2.5. Proteomics Sample Preparation</h3>
<p>HCT 116 spheroids were lysed in mammalian cell lysis buffer using a probe sonicator consisting of 6% SDS, 1 mM each of sodium fluoride, sodium orthovanadate, β‐glycerophosphate, 10 mM of sodium pyrophosphate, 100 mM TEAB, and one complete EDTA‐free protease inhibitor cocktail tablet (Sigma). Approximately 400 μg of protein was aliquoted for proteomics. Samples were first reduced with 20 mM DTT for 15 min at 65°C followed by alkylation with 40 mM IAA for 30 min in the dark. Next, samples were acidified with phosphoric acid to 1.2% and diluted with 90% Methanol/100 mM TEAB in a 1:6 ratio. Proteins were spun onto the S‐Trap column at 4000 ×<em>g</em> for 1 min and washed four times with 90% methanol/100 mM TEAB. Trapped proteins were spun once at 4000 ×<em>g</em> for 1 min with no wash to remove excess methanol. S‐Trap columns were moved to a new column where Trypsin Gold/Lys‐C was added in a 1:50 trypsin:protein ratio in 50 mM TEAB and incubated in a water bath overnight at 37°C. Peptides were eluted sequentially with equal volumes of 50 mM TEAB, 0.2% formic acid in water, and 0.2% formic acid in acetonitrile. Peptides were further desalted using a Waters HLB Oasis μ elution 96‐well plate (2 mg sorbent, 30 μm particle size, 80 Å pore size) and dried down prior to storage at −80°C.</p></section><section id="jms5137-sec-0009"><h3 class="pmc_sec_title">2.6. LC–MS</h3>
<p>Peptides were resuspended in 0.1% formic acid (Mobile Phase A) and separated using a Waters M‐Class Acuity UHPLC using a Waters nanoEase M/Z Peptide BEH C18 column (75 μm id × 200 mm length) at 50°C with a flow rate of 0.400 μL/min. Approximately 500 ng of peptides per sample were separated using a two‐step linear gradient starting at 2% mobile phase B (0.1% formic acid in acetonitrile) to 20% B in 100 min. The percent composition of B was increased to 32% in 20 min followed by a 1‐min ramp to 95% B and held at 95% for 4 min. Next, the percent B composition was ramped down to 2% in 1 min and held at 2% for 30 min to allow column equilibration. Peptides were analyzed using DIA on a Thermo QE‐HF mass spectrometer. DIA windows were constructed within EncylopeDIA [<a href="#jms5137-bib-0034" class="usa-link" aria-describedby="jms5137-bib-0034">34</a>]. Staggered DIA wide windows were collected for each proteomics sample from 400 to 1000 <em>m/z</em> in 16.0 <em>m/z</em> wide windows with full scan resolution of 60 000 and an MS2 resolution of 30 000 along with an AGC target of 1 × 10<sup>6</sup>. The normalized collision energy (NCE) value was set to 27. A pooled sample was also prepared and staggered narrow DIA windows were collected in 100 <em>m/z</em> intervals from 400 to 1000 <em>m/z</em> with 4 <em>m/z</em> wide DIA windows to generate a gas phase fractionated (GPF) library [<a href="#jms5137-bib-0035" class="usa-link" aria-describedby="jms5137-bib-0035">35</a>]. All .raw LC–MS files were converted to .mzML using MSConvert with peak picking and demultiplexing [<a href="#jms5137-bib-0036" class="usa-link" aria-describedby="jms5137-bib-0036">36</a>].</p></section><section id="jms5137-sec-0010"><h3 class="pmc_sec_title">2.7. Data Analysis</h3>
<p>MALDI‐MSI data was analyzed using SCiLS Lab 2023c and exported to Metaspace2020 for putative identifications [<a href="#jms5137-bib-0037" class="usa-link" aria-describedby="jms5137-bib-0037">37</a>]. MALDI data was normalized using the total ion count (TIC). Segmentation of spheroids from background signal was completed using weak denoising, bisecting k‐means, and the correlation distance metric using a list of the top 30 peaks within the spheroids (Figure <a href="#jms5137-supitem-0001" class="usa-link">S15</a>). Graphpad Prism was used to generate IC<sub>50</sub> plots and determine statistical significance from exported peak areas of segmented spheroid regions. The proteomics data was analyzed by first generating a predicted library from the Human FASTA Prosit and then searching the GPF's against this library using EncyclopeDIA with the Human FASTA as the background [<a href="#jms5137-bib-0034" class="usa-link" aria-describedby="jms5137-bib-0034">34</a>, <a href="#jms5137-bib-0035" class="usa-link" aria-describedby="jms5137-bib-0035">35</a>, <a href="#jms5137-bib-0038" class="usa-link" aria-describedby="jms5137-bib-0038">38</a>]. The searched GPF were summarized together to generate an empirically corrected chromatogram library followed by filtering to 1% FDR with Percolator [<a href="#jms5137-bib-0039" class="usa-link" aria-describedby="jms5137-bib-0039">39</a>]. The DIA proteomics samples were searched against the empirically corrected chromatogram library with the Human FASTA as the background and filtered to 1% FDR. The resulting .elib proteomics files were summarized together with match between runs (MBR) and uploaded into Skyline [<a href="#jms5137-bib-0040" class="usa-link" aria-describedby="jms5137-bib-0040">40</a>]. Within Skyline, the proteomics results were annotated and filtered to contain a minimum of two unique peptides per protein. An MSstats output was generated, and further data analysis was conducted with MSstats and Perseus as well as base R programming [<a href="#jms5137-bib-0041" class="usa-link" aria-describedby="jms5137-bib-0041">41</a>]. Functional analysis was done using Ingenuity Pathway Analysis (IPA) [<a href="#jms5137-bib-0042" class="usa-link" aria-describedby="jms5137-bib-0042">42</a>]. Gene Set Enrichment Analysis (GSEA) was conducted using the clusterProfileR package [<a href="#jms5137-bib-0043" class="usa-link" aria-describedby="jms5137-bib-0043">43</a>].</p></section></section><section id="jms5137-sec-0011"><h2 class="pmc_sec_title">3. Results</h2>
<section id="jms5137-sec-0012"><h3 class="pmc_sec_title">3.1. IC<sub>50</sub> Determination and MALDI‐MSI of Onvansertib</h3>
<p>To study onvansertib in HCT 116 spheroids, the half maximum inhibitory concentration had to be determined. This assay was completed by incubating HCT 116 spheroids on Day 12 of growth with onvansertib concentrations ranging from 250 to 0.120 μM for 72 h. The dosed spheroids were then incubated with the Cell Titer Glo 3D reagent and the total luminescent intensity was measured. The IC<sub>50</sub> was determined to be 30.87 ± 12.67 μM (IC<sub>50</sub> ± 95% Confidence Interval) (Figure <a href="#jms5137-supitem-0001" class="usa-link">S1</a>). Due to the three‐dimensional nature of the spheroid, it is important to understand how well a therapeutic can penetrate the cell mass. HCT 116 spheroids were incubated with onvansertib at the calculated IC<sub>50</sub> concentration and harvested at eight different time points (0 (control), 1, 4, 6, 12, 24, 48, and 72 h). The harvested spheroids were embedded in gelatin prior to being stored at minus 80°C. Spheroids were also harvested at each time point for complementary proteomics analysis.</p>
<p>Embedded spheroids were sectioned and placed onto the conductive side of an indium tin oxide slide. CHCA matrix in 50% ACN was sprayed onto the slides using an HTX Sprayer. Spheroids were analyzed with a Bruker timsTOF fleX with and without MALDI‐2 postionization. Each condition was analyzed in biological triplicate. The collected MSI spectra were analyzed with SCiLS lab and the spatial distribution of a putative lipid (to indicate the shape of the spheroid, Figure <a href="#jms5137-fig-0001" class="usa-link">1A</a>) and onvansertib ([M + H<sup>+</sup>] = 533.221 Da) were visualized. As shown in Figure <a href="#jms5137-fig-0001" class="usa-link">1B,C</a>, onvansertib begins to accumulate in the outer regions of the spheroid after only 1 h. Onvansertib begins to more strongly penetrate the spheroid at 12 h after incubation, where it then continues to accumulate throughout the entire spheroid. While a slight decrease in signal is observed, after 12 h, onvansertib accumulation continues through the 72‐h timepoint.</p>
<figure class="fig xbox font-sm" id="jms5137-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1b1f/11976698/5f72eccfce3a/JMS-60-e5137-g003.jpg" loading="lazy" id="jats-graphic-1" height="786" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/jms5137-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Putative lipid at 755.538 Da to show the HCT 116 spheroid in the imaging data. (B) MALDI‐MSI of onvansertib (<em>m/z</em> = 533.221 Da) over the experimental time series of 0 to 72 h. (C) Box and whisker cloud plot for onvansertib.</p></figcaption></figure></section><section id="jms5137-sec-0013"><h3 class="pmc_sec_title">3.2. Proteomics</h3>
<p>Proteins harvested from spheroids at each time point were digested into peptides using S‐Trap columns. Peptides were analyzed on a QE‐HF mass spectrometer using staggered DIA with 16 <em>m/z</em> wide windows. A pooled sample was also prepared, and narrow window DIA spectra were collected over 100 <em>m/z</em> ranges from 400 to 1000 <em>m/z</em> with 4 <em>m/z</em> wide windows. The narrow window DIA was searched against an in silico Prosit‐made spectral library and compiled together to have an empirically corrected gas phase fractionated library (EMP‐GPF) [<a href="#jms5137-bib-0035" class="usa-link" aria-describedby="jms5137-bib-0035">35</a>]. The DIA proteomics samples were searched with the EMP‐GPF library. A total of 72 249 peptides were identified and filtered to a 1% FDR [<a href="#jms5137-bib-0035" class="usa-link" aria-describedby="jms5137-bib-0035">35</a>]. After further filtering in Skyline and MSstats to require each protein contain two or more peptides and those peptides to be unique, 5751 proteins were quantified. Figure <a href="#jms5137-supitem-0001" class="usa-link">S2</a> shows consistent normalized and log<sub>2</sub> transformed MS1 TIC levels. Hierarchical clustering shows that the 72‐h time points cluster entirely together (Figure <a href="#jms5137-supitem-0001" class="usa-link">S3</a>). The other time points did not cluster distinctly from each other.</p>
<p>Next, each time point was compared against the 0‐h control time point to determine which proteins were differentially expressed (log<sub>2</sub>FC ≥ 1 or ≤ − 1 and <em>p</em>‐value &lt; 0.05). The Benjamini–Hochberg hypothesis testing correction approach was applied to the <em>p</em> values to account for false positives [<a href="#jms5137-bib-0044" class="usa-link" aria-describedby="jms5137-bib-0044">44</a>]. It was observed that all the early time points (1, 4, and 6 h) had no differentially expressed proteins (Figure <a href="#jms5137-supitem-0001" class="usa-link">S4A–C</a>). The late time points (12, 24, 48, and 72 h) were observed to have a changing number of differentially expressed proteins over time as the onvansertib was consumed by the cells (Figure <a href="#jms5137-fig-0002" class="usa-link">2A–D</a>). Comparison of the differentially expressed proteins among all four time points showed that only eight proteins were similar among all three conditions (Figure <a href="#jms5137-supitem-0001" class="usa-link">S5</a>). Aurora kinase B (AURKB) was found to be upregulated in all four time points (Figure <a href="#jms5137-fig-0003" class="usa-link">3A</a>) as well as other proteins involved in the cell cycle process, such as CCNB1, KIFC1, and PRC1 (Supplemental Table <a href="#jms5137-supitem-0002" class="usa-link">2</a>). No proteins were shared among the 12‐, 24‐, and 72‐h time points (Figure <a href="#jms5137-supitem-0001" class="usa-link">S5</a>). One of the cofactors of aurora kinase B, Borealin (BOREA), was found to be up‐regulated in the 24‐, 48‐, and 72‐h time points with an increasing intensity over time (Figure <a href="#jms5137-fig-0003" class="usa-link">3B</a>). The INCE cofactor was detected in all samples, and only differentially expressed in the 12‐, 48‐, and 72‐h time points (Figure <a href="#jms5137-fig-0003" class="usa-link">3C</a>). In the 24‐h time point, INCE had a log<sub>2</sub>FC = 1.500, but was not statistically significant. The last cofactor, survivin, was not identified. Aurora kinase A was also not identified within the proteomics data set. Plk1 (Figure <a href="#jms5137-fig-0003" class="usa-link">3D</a>) was detected in this data and was elevated in the 12‐, 24‐, and 72‐h time points (log<sub>2</sub>FC &gt; 1); however, Plk1 was not elevated in the 48‐h time point (log<sub>2</sub>FC = 0.957). Plk1 levels were not statistically significant (<em>p</em> &gt; 0.05) in all time points compared to the 0‐h time point.</p>
<figure class="fig xbox font-sm" id="jms5137-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1b1f/11976698/61d339597a9c/JMS-60-e5137-g006.jpg" loading="lazy" id="jats-graphic-3" height="786" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/jms5137-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Differentially expressed proteins found within the (A) 12‐h onvansertib exposure, (B) 24‐h onvansertib exposure, (C) 48‐h onvansertib exposure, and (D) 72‐h onvansertib exposure.</p></figcaption></figure><figure class="fig xbox font-sm" id="jms5137-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1b1f/11976698/89742d11b05f/JMS-60-e5137-g001.jpg" loading="lazy" id="jats-graphic-5" height="364" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/jms5137-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Time course changes of select proteins represented as their log<sub>2</sub> intensities against the respective time point.</p></figcaption></figure><p>The 12‐h time point was observed to have 21 up‐ and 46 down‐regulated proteins, respectively, with 41 of these proteins found to be unique to this time point (Figures <a href="#jms5137-fig-0002" class="usa-link">2A</a> and <a href="#jms5137-supitem-0001" class="usa-link">S5</a>). A pathway analysis conducted in IPA shows that the 12‐h time point only had predicted activation of one pathway, the Intra Golgi and retrograde Golgi‐ER traffic (Figure <a href="#jms5137-supitem-0001" class="usa-link">S10</a>). GSEA analysis shows activation and suppression of many different gene sets (Figure <a href="#jms5137-supitem-0001" class="usa-link">S6</a>). Cellular response to nitrogen, organonitrogen, and oxygen containing compounds were suppressed (Figure <a href="#jms5137-supitem-0001" class="usa-link">S6</a>). This suppression is exemplified by the down regulation of PRPS1 (Figure <a href="#jms5137-fig-0004" class="usa-link">4A</a>), an enzyme that catalyzes the synthesis of phosphoribosyl pyrophosphate, which is required for the synthesis of nucleotides. GSEA analysis also showed activation of ribosome and structural constituent of ribosome gene sets (Figure <a href="#jms5137-supitem-0001" class="usa-link">S6</a>). This enrichment can be observed in the proteomics data, where the 12‐h time point had unique upregulation of three different ribosomal subunit proteins, RL35, RS6, and RS7 (Supplementary Table <a href="#jms5137-supitem-0001" class="usa-link">S1</a>).</p>
<figure class="fig xbox font-sm" id="jms5137-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1b1f/11976698/397dd623c26d/JMS-60-e5137-g002.jpg" loading="lazy" id="jats-graphic-7" height="1621" width="669" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/jms5137-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Time course changes of select proteins represented as their log<sub>2</sub> intensities against the respective time point.</p></figcaption></figure><p>The 24‐h time point interestingly contained only upregulated proteins when compared against the 0 h control (Figure <a href="#jms5137-fig-0002" class="usa-link">2B</a>). Only one protein was unique to this time point, PLPL2, an enzyme involved in hydrolyzing triglycerides from lipid droplets (Figure <a href="#jms5137-fig-0002" class="usa-link">2</a>). IPA analysis predicted no activated pathways among the differentially expressed proteins, and GSEA analysis only showed nine altered gene sets (Figure <a href="#jms5137-supitem-0001" class="usa-link">S7</a>). Regulation of cell cycle process and regulation of cell cycle gene sets were activated, and catalytic activity gene sets were suppressed (Figure <a href="#jms5137-supitem-0001" class="usa-link">S7</a>). The most upregulated protein in the 24‐h time point was CCNB1 (Figure <a href="#jms5137-fig-0004" class="usa-link">4B</a>), which is involved in control of the cell cycle during the G<sub>2</sub>/M transition.</p>
<p>The 48‐h time point contained many more altered proteins with 43 up‐ and 144 down‐regulated proteins (Figure <a href="#jms5137-fig-0002" class="usa-link">2C</a>). GSEA analysis shows activation of many cell cycle processes, such as positive regulation of cytokinesis, mitotic spindle elongation, and regulation of mitotic cytokinesis gene sets (Figure <a href="#jms5137-supitem-0001" class="usa-link">S8</a>). However, there was suppression of gene sets related to nucleobase‐containing small molecule metabolic process, organophosphate biosynthetic process, nucleotide metabolic process, and nucleotide biosynthetic process (Figure <a href="#jms5137-supitem-0001" class="usa-link">S8</a>). IPA analysis shows predicted activation of mitotic metaphase and anaphase pathways as well as mitotic prometaphase (Figure <a href="#jms5137-supitem-0001" class="usa-link">S10</a>). There was predicted deactivation of rRNA processing in the nucleolus and cytosol (Figure <a href="#jms5137-supitem-0001" class="usa-link">S10</a>). Specific proteins involved in the cell cycle were upregulated and unique to this timepoint, such as CDCA5 (Figure <a href="#jms5137-fig-0004" class="usa-link">4C</a>).</p>
<p>Many differentially expressed proteins were found in the 72‐h time point with 85 up‐ and 98 down‐regulated proteins (Figure <a href="#jms5137-fig-0002" class="usa-link">2</a>). Many GO terms for cell division processes were enriched after GSEA analysis, such as organelle fission, cell division, chromosome segregation, and nuclear division gene sets (Figure <a href="#jms5137-supitem-0001" class="usa-link">S9</a>). This enrichment may be the result of up regulation of KIF11, KIF23, KIF2C, and KIFC1 (Figure <a href="#jms5137-fig-0004" class="usa-link">4D</a>), all proteins required for proper cell division. The three largest containing gene sets from this time point were all suppressed terms: nucleic acid binding, nuclear lumen, and cellular nitrogen compound metabolic process (Figure <a href="#jms5137-supitem-0001" class="usa-link">S9</a>). This response is apparent in the specific downregulation of proteins involved in DNA and RNA synthesis, such as ATRX (Figure <a href="#jms5137-fig-0004" class="usa-link">4E</a>), and PRI2. It was observed that 72‐h time points had a lower level of predicted activation of mitotic metaphase and anaphase compared to the 48‐h time points (Figure <a href="#jms5137-supitem-0001" class="usa-link">S10</a>).</p>
<p>Changes to many enzymes involved in lipid metabolism were also observed in each of the four late time points. In the 12‐h time point, the enzyme HACD3—which catalyzes the third reaction of the very long chain fatty acid elongation cycle—was observed to be downregulated (Figure <a href="#jms5137-fig-0005" class="usa-link">5A</a>), suggesting that there is alteration to lipid metabolism with onvansertib exposure (Supplementary Table <a href="#jms5137-supitem-0001" class="usa-link">S1</a>). Other lipid metabolism enzymes were observed in the 48‐ and 72‐h time points, such as upregulation of GAB1, which cleaves glucose from glucosylceramides to releases free ceramides into the cell and downregulation of AGAL, an enzyme that hydrolyzes glycosphingolipids. The 48‐h time point was observed to have upregulation of lysophospholipase D (GDPD1) (Figure <a href="#jms5137-fig-0005" class="usa-link">5B</a>). The 72‐h time point had up regulation of phosphatidylserine synthase 1 (PTSS1) and ERG24 (Figure <a href="#jms5137-fig-0005" class="usa-link">5C</a>), an enzyme involved in cholesterol synthesis. The alteration to many lipid synthesis enzymes prompted us to inspect the MSI data closer to determine if various lipid species could be visualized and alterations to their expression observed over time.</p>
<figure class="fig xbox font-sm" id="jms5137-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1b1f/11976698/6c9a090db739/JMS-60-e5137-g005.jpg" loading="lazy" id="jats-graphic-9" height="975" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/jms5137-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Time course changes of select proteins represented as their log<sub>2</sub> intensities against the respective time point.</p></figcaption></figure></section><section id="jms5137-sec-0014"><h3 class="pmc_sec_title">3.3. Spatial Lipidomics</h3>
<p>Spheroids from each time point were embedded in gelatin and sectioned onto conductive ITO slides. The sectioned spheroids were imaged using a Bruker timsTOF fleX with and without MALDI‐2 in positive ion mode using either CHCA or DHB matrix. For each time point, three different spheroids were imaged for a total of 24 total spheroids imaged across the entire experiment. The MSI data were processed using SCiLS to determine various peaks of interest. Further annotation of peaks of interest were validated by conducting tandem MS on HCT 116 spheroid lysate and each MS/MS spectrum was manually annotated to discern lipid head group identity, total number of carbons, and degree of unsaturation.</p>
<p>Many lipid peaks were observed with MALDI‐1 in the presented data. Some lipids, such as <em>m/z</em> 796.464, were observed to not be present or have very low intensity in the 0‐h time point and then gradually increase over time with onvansertib treatment (Figure <a href="#jms5137-fig-0006" class="usa-link">6A</a>). The average peak area for the 48‐ and 72‐h timepoints were found to be statistically significant at 0.0001 and &lt; 0.0001, respectively (Figure <a href="#jms5137-fig-0006" class="usa-link">6B</a>). Based on characteristic fragments, accurate mass, and prior observations, the <em>m/z</em> 796.464 in Figure <a href="#jms5137-fig-0006" class="usa-link">6A</a> is identified as the [M + K]<sup>+</sup> of PC(34:2) (Figure <a href="#jms5137-supitem-0001" class="usa-link">S13A</a>) [<a href="#jms5137-bib-0045" class="usa-link" aria-describedby="jms5137-bib-0045">45</a>]. Other lipid species were observed to have relatively high intensity from 0‐ to 12‐h followed by rapid decrease in intensity at 24‐ through 72‐h (Figure <a href="#jms5137-supitem-0001" class="usa-link">S11A</a>). The <em>m/z</em> 798.542 shown in Figure <a href="#jms5137-supitem-0001" class="usa-link">S11A</a> was annotated to be the [M + K]<sup>+</sup> of PC(34:1) based on tandem MS as well as accurate mass and prior observations (Figure <a href="#jms5137-supitem-0001" class="usa-link">S13B</a>) [<a href="#jms5137-bib-0046" class="usa-link" aria-describedby="jms5137-bib-0046">46</a>]. Lastly, there were certain lipid species that were low in intensity throughout the spheroid from 0‐ to 6‐h followed by a peak in intensity at 12‐h and then a sharp decrease in intensity from 24‐h on (Figure <a href="#jms5137-supitem-0001" class="usa-link">S11B</a>). Tandem MS confirmed the <em>m/z</em> 770.513 observed in Figure <a href="#jms5137-supitem-0001" class="usa-link">S11C</a> to be the [M + K]<sup>+</sup> of PC(32:1) along with prior confirmation of this lipid by MALDI (Figure <a href="#jms5137-supitem-0001" class="usa-link">S13C</a>) [<a href="#jms5137-bib-0047" class="usa-link" aria-describedby="jms5137-bib-0047">47</a>]. In both of these lipids, statistical significance was only observed between the control and 12‐h timepoint. While a putative increase was observed at 72 h, it was not found to be statistically significant.</p>
<figure class="fig xbox font-sm" id="jms5137-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1b1f/11976698/aeb3b7c9db09/JMS-60-e5137-g004.jpg" loading="lazy" id="jats-graphic-11" height="412" width="709" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/jms5137-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Changing abundance of lipids after onvansertib exposure. (A) Increasing intensity of <em>m/z</em> 796.464 Da identified as the [M + K]<sup>+</sup> of PC(34:2) using MALDI‐1 with average peak areas plotted and statistical analysis shown for comparison against controls (B). (C) Decreasing intensity of <em>m/z</em> 808.585 Da identified as the [M + K]<sup>+</sup> of PC(36:10) LIPID using MALDI‐2 with average peak areas plotted and statistical analysis completed against controls (D).</p></figcaption></figure><p>MALDI‐2 is a relatively new ionization technique that can be utilized to observe ions that do not ionize readily with ablation from a single MALDI source. MALDI‐2 generates ions with a second UV laser pulse (~266 nm) passing through the ion plume generated from the initial MALDI ablation at a 90° angle [<a href="#jms5137-bib-0048" class="usa-link" aria-describedby="jms5137-bib-0048">48</a>]. This second laser pulse allows ionization of certain analytes that are not typically ionizable by the initial MALDI ionization process. MALDI‐2 has been shown to ionize various sterols with great efficiency, such as cholesterol [<a href="#jms5137-bib-0049" class="usa-link" aria-describedby="jms5137-bib-0049">49</a>]. Cholesterol was a lipid of interest for this data set with the differential expression of various enzymes involved in cholesterol synthesis and trafficking (Supplemental Table <a href="#jms5137-supitem-0002" class="usa-link">S2</a>). Analysis of <em>m/z</em> 369.351 Da, identified as the [M + H<sup>+</sup>‐H<sub>2</sub>O]<sup>+</sup> of cholesterol based on prior lipidomic acquisitions, showed a gradual increase in intensity up until 4 h after onvansertib exposure, followed by a decrease in intensity over time after 4 h (Figure <a href="#jms5137-supitem-0001" class="usa-link">S12</a>) [<a href="#jms5137-bib-0049" class="usa-link" aria-describedby="jms5137-bib-0049">49</a>, <a href="#jms5137-bib-0050" class="usa-link" aria-describedby="jms5137-bib-0050">50</a>, <a href="#jms5137-bib-0051" class="usa-link" aria-describedby="jms5137-bib-0051">51</a>]. Statistical analysis of the peak areas found the 4‐h timepoint to be the only timepoint significantly altered in comparison to the control dataset. In comparison, the later time points of 12‐, 48‐, and 72‐h were all found to be lower than the 4‐h time point with statistical significance. Examination of <em>m/z</em> 808.585—identified as the [M + K<sup>+</sup>]<sup>+</sup> of PC(36:10) using MALDI‐2 ionization (Figures <a href="#jms5137-fig-0006" class="usa-link">6C</a> and <a href="#jms5137-supitem-0001" class="usa-link">S14</a>)—started at a very high intensity, but as the time exposed to onvansertib increased, the intensity of the lipid decreased over time. Statistical significance was observed between the control dataset and all other time points (Figure <a href="#jms5137-fig-0006" class="usa-link">6D</a>).</p></section></section><section id="jms5137-sec-0015"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>We have conducted a time course MALDI‐MSI and untargeted proteomics study of HCT 116 spheroids exposed to the Plk1 inhibitor onvansertib. Plk1 is a protein kinase that is involved in controlling the cell cycle. It has been reported that patients with cancer with elevated levels of Plk1 have poor survival rates [<a href="#jms5137-bib-0006" class="usa-link" aria-describedby="jms5137-bib-0006">6</a>]. Plk1 expression has also been linked to a cancer's developed resistance to many therapeutics. Onvansertib is a recently developed competitive inhibitor of ATP that is specific for Plk1 [<a href="#jms5137-bib-0014" class="usa-link" aria-describedby="jms5137-bib-0014">14</a>, <a href="#jms5137-bib-0015" class="usa-link" aria-describedby="jms5137-bib-0015">15</a>]. Though onvansertib has shown success in clinical trials, it has yet to be determined what distinct molecular differences are caused by Plk1 inhibition by onvansertib to complete the understanding of this therapeutics’ role in cancer treatment. Using DIA proteomics, more than 5000 proteins were identified and quantified. Changes to many cell cycle control proteins were observed as well as changes to proteins involved in lipid metabolism. Using MALDI‐MSI, various lipids were then analyzed, and their changing intensity over time was observed.</p>
<p>Aurora kinases are serine/threonine kinases with high levels of sequence similarity among the three members and play distinct roles within the cell [<a href="#jms5137-bib-0052" class="usa-link" aria-describedby="jms5137-bib-0052">52</a>]. The main role of AURKB is to be a component of the chromosomal passenger complex (CPC), the master controller of cell division. The CPC regulates chromosome structure, spindle assembly checkpoints, as well as kinetochore‐microtubule attachments and is formed by AURKB, BOREA, INCE, and survivin coming together [<a href="#jms5137-bib-0053" class="usa-link" aria-describedby="jms5137-bib-0053">53</a>]. AURKB has been shown to activate Plk1 along with the other CPC proteins, with AURKB phosphorylation of Plk1 maintaining Plk1 activity at the kinetochore to ensure accurate chromosome segregation [<a href="#jms5137-bib-0003" class="usa-link" aria-describedby="jms5137-bib-0003">3</a>, <a href="#jms5137-bib-0004" class="usa-link" aria-describedby="jms5137-bib-0004">4</a>]. It was observed in the following data that AURKB, BOREA, and INCE had increasing log<sub>2</sub>FCs and increased statistical significance over time with onvansertib exposure (Figure <a href="#jms5137-fig-0002" class="usa-link">2</a> and Supplemental Table <a href="#jms5137-supitem-0002" class="usa-link">S2</a>). As Plk1 is inhibited by onvansertib, it is possible that the cell is attempting to synthesize more members of the CPC over time to alleviate the inhibition and continue the process of cell division. This may also be shown by activated gene sets for ribosomes at 12 h, followed by activation of protein kinase binding after 24 h, and lastly enrichment of the CPC gene sets after 48 h. As onvansertib begins clearing out of the spheroid after 72 h, it is possible that normal cell division may be able to continue with activation of the cell division and mitotic cell process gene sets (Figure <a href="#jms5137-supitem-0001" class="usa-link">S9</a>).</p>
<p>Many gene sets related to nitrogen compound synthesis were shared among the 12‐, 24‐, 48‐, and 72‐h time points. The first began at 12 h of onvansertib exposure with suppression of the cellular response to organonitrogen compound and cellular response to nitrogen compound gene sets (Figure <a href="#jms5137-supitem-0001" class="usa-link">S6</a>). Then, after 24 h, the catalytic activity gene set was suppressed (Figure <a href="#jms5137-supitem-0001" class="usa-link">S7</a>). The most profound change occurred after 48 h, with suppression of nucleobase‐containing small molecule metabolic process, organophosphate biosynthetic process, nucleotide metabolic process, nucleotide biosynthetic process, and lastly nucleoside phosphate biosynthetic process (Figure <a href="#jms5137-supitem-0001" class="usa-link">S8</a>). Lastly, 72 h after onvansertib exposure, suppression of cellular nitrogen compound metabolic process was observed. However, other terms related to DNA/RNA were suppressed, such as ncRNA metabolic process, RNA binding, nucleic acid binding, and rRNA processing (Figure <a href="#jms5137-supitem-0001" class="usa-link">S9</a> and Figure <a href="#jms5137-supitem-0001" class="usa-link">S10</a>). It has been shown that Plk1 plays a role in DNA synthesis during the S phase of the cell cycle by phosphorylating Orc2 at Serine 188 [<a href="#jms5137-bib-0012" class="usa-link" aria-describedby="jms5137-bib-0012">12</a>, <a href="#jms5137-bib-0054" class="usa-link" aria-describedby="jms5137-bib-0054">54</a>]. With Plk1 being inhibited by onvansertib, DNA synthesis is stalled. It has also been shown that cellular metabolism plays a very important role in regulating the cell cycle [<a href="#jms5137-bib-0055" class="usa-link" aria-describedby="jms5137-bib-0055">55</a>]. The proteomics data along with the GSEA analysis show that nucleotide metabolism is also impaired, suggesting that genesis of various RNAs and nucleotide triphosphates is also hindered. These results show that Plk1 inhibition has over encompassing effects on cell metabolism that can alter the genesis of other cellular components.</p>
<p>Lipids play a vital role in cancer biology and are constantly changing as cancerous cells move through their biological processes [<a href="#jms5137-bib-0056" class="usa-link" aria-describedby="jms5137-bib-0056">56</a>]. It has been proposed that lipids may help shape the cellular cycle [<a href="#jms5137-bib-0057" class="usa-link" aria-describedby="jms5137-bib-0057">57</a>, <a href="#jms5137-bib-0058" class="usa-link" aria-describedby="jms5137-bib-0058">58</a>, <a href="#jms5137-bib-0059" class="usa-link" aria-describedby="jms5137-bib-0059">59</a>]. For example, it has been shown that PC lipids will typically increase in the S phase of the cell cycle and PI lipids will decrease in this phase while the cell enlarges [<a href="#jms5137-bib-0058" class="usa-link" aria-describedby="jms5137-bib-0058">58</a>]. It has also been shown that PE, PS, and PI lipids will attain their highest abundance during the G<sub>2</sub>/M transition in the cellular cycle [<a href="#jms5137-bib-0058" class="usa-link" aria-describedby="jms5137-bib-0058">58</a>]. Using MALDI‐MSI, shotgun lipidomics data was collected for each spheroid at each experimental time point and a handful of various lipid ions were discovered and annotated with tandem MS. A majority of the lipids identified in this experiment were PC lipids, more than likely due to PC lipids preference for ionization in positive mode MS. [<a href="#jms5137-bib-0060" class="usa-link" aria-describedby="jms5137-bib-0060">60</a>] With the trends observed for the PC lipids (Figure <a href="#jms5137-fig-0004" class="usa-link">4</a> and Figure <a href="#jms5137-supitem-0001" class="usa-link">S11A,B</a>), onvansertib may be halting the cells' ability to exit the S phase of the cell cycle, causing a buildup of cells that are enlarging and trying to enter the G<sub>2</sub> phase. The stark differences observed for PC(34:2) and PC(34:1) (Figure <a href="#jms5137-fig-0006" class="usa-link">6</a> and Figure <a href="#jms5137-supitem-0001" class="usa-link">S11A</a>, respectively) over time are intriguing. The addition of a double bond to this lipid implies further remodeling of lipid metabolism and possibly membrane structure. More experiments will need to be conducted to understand onvansertib's role in lipid double bond changes.</p></section><section id="jms5137-sec-0016"><h2 class="pmc_sec_title">5. Conclusion</h2>
<p>We have conducted both an untargeted proteomics and lipidomics experiment with the Plk1 inhibitor onvansertib. A multitude of alterations were observed at both the proteomic and lipidomic levels over time. The majority of proteins altered were involved in controlling various points of the cell cycle as well lipid metabolism enzymes, with the noteworthy increase over time of the AURKB and BOREA, two cofactors required for Plk1 activation. Alterations to PC lipids were observed, with the identified PC lipids increasing over time. As many cell cycle control proteins were elevated and PC lipids, known to be involved in S phase of the cell cycle, were elevated, it is suggested that onvansertib is halting the cells in the S phase of cell growth and causing cells to continually grow and expand but no longer enter the G<sub>2</sub> phase of growth.</p></section><section id="jms5137-sec-0019"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>B.D.F. and E.R.S. designed the experiments and collected the proteomics and MSI data. B.D.F. analyzed the proteomics data. B.D.F., J.H.H., and E.R.S. analyzed the MSI data. B.D.F., E.R.S., and A.B.H. wrote the manuscript.</p></section><section id="jms5137-sec-0020"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec19"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="jms5137-supitem-0001"><div class="caption p">
<p>
<strong>Figure S1</strong> IC<sub>50</sub> curve of HCT 116 spheroids incubated with onvansertib for 72 h.</p>
<p>
<strong>Table S1</strong> Spray parameters for MALDI‐MSI experiments with either DHB or CHCA.</p>
<p>
<strong>Figure S2</strong> Normalized MS1 TIC for log<sub>2</sub> transformed intensity values for LC–MS proteomics.</p>
<p>
<strong>Figure S3</strong> Hierarchical clustering of onvansertib proteomics data showing that 72 h with onvansertib exposure cluster entirely together.</p>
<p>
<strong>Figure S4</strong> Volcano plots of (A) 1 h, (B) 4 h, and (C) 6 h onvansertib exposure, showing only one differentially expressed protein among these entire time sets.</p>
<p>
<strong>Figure S5</strong> Venn Diagram of similar proteins between the 12‐, 24‐, 48‐, and 72‐h samples.</p>
<p>
<strong>Figure S6</strong> Gene set enrichment analysis of the differentially expressed proteins after 12 h of onvansertib exposure.</p>
<p>
<strong>Figure S7</strong> Gene set enrichment analysis of the differentially expressed proteins after 24 h of onvansertib exposure.</p>
<p>
<strong>Figure S8</strong> Gene set enrichment analysis of the differentially expressed proteins after 48 h of onvansertib exposure.</p>
<p>
<strong>Figure S9</strong> Gene set enrichment analysis of the differentially expressed proteins after 72 h of onvansertib exposure.</p>
<p>
<strong>Figure S10</strong> Ingenuity pathway analysis of statistically significant proteins among the 12‐, 24‐, 48, and 72‐h time points.</p>
<p>
<strong>Figure S11</strong> Two different lipids that are not highly present in the early time points but gradually increase over time with onvansertib exposure. (A) <em>m/z</em> 798.542 identified to be the [M + K<sup>+</sup>]<sup>+</sup> of PC(34:1). (B) <em>m/z</em> 770.513 identified to be the [M + K<sup>+</sup>]<sup>+</sup> of PC(32:1).</p>
<p>
<strong>Figure S12</strong>
<em>m/z</em> 369.351, identified as the [M + H<sup>+</sup>‐H<sub>2</sub>O]<sup>+</sup> of cholesterol, as observed with MALDI‐2 ionization.</p>
<p>
<strong>Figure S13</strong> MALDI‐MSMS of lipids found in the mass spectrometry images for validation of lipid identity. The above tandem MS spectra originate from the following precursors: (A) <em>m/z</em> 796.464, (B) <em>m/z</em> 798.542, and (C) <em>m/z</em> 770.513.</p>
<p>
<strong>Figure S14</strong> MALDI‐2 MSMS of lipids found in the mass spectrometry images for validation of lipid identity. The above tandem MS spectra originate from the precursor <em>m/z</em> 808.585.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11976698/bin/JMS-60-e5137-s004.docx" data-ga-action="click_feat_suppl" class="usa-link">JMS-60-e5137-s004.docx</a><sup> (5MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="jms5137-supitem-0002"><div class="caption p"><p>
<strong>Data S2</strong> Supplemental Table 2</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11976698/bin/JMS-60-e5137-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">JMS-60-e5137-s001.xlsx</a><sup> (1.4MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="jms5137-supitem-0003"><div class="caption p"><p>
<strong>Data S3</strong> Supplemental Table 2</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11976698/bin/JMS-60-e5137-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">JMS-60-e5137-s002.xlsx</a><sup> (12.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="jms5137-supitem-0004"><div class="caption p"><p>
<strong>Data S4</strong> Supplemental Table 2</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11976698/bin/JMS-60-e5137-s003.xlsx" data-ga-action="click_feat_suppl" class="usa-link">JMS-60-e5137-s003.xlsx</a><sup> (41.1KB, xlsx) </sup>
</div></div></section></section><section id="notes1"><section id="jms5137-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="jms5137-note-0001"><p>
<strong>Funding:</strong> This work was supported by National Cancer Institute, P30 CA016058, R01CA247863; National Institute of General Medical Studies, R01GM110406.</p></div></div></section></section><section id="jms5137-sec-0018"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>Raw LC–MS proteomics files have been uploaded to the Pride repository under PXD054453.</p></section><section id="jms5137-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="jms5137-bibl-0001_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="jms5137-bib-0001">
<span class="label">1.</span><cite>
Sunkel C. E. and Glover D. M., “Polo, a Mitotic Mutant of Drosophila Displaying Abnormal Spindle Poles,” Journal of Cell Science
89 (1988): 25–38, 10.1242/jcs.89.1.25.
</cite> [<a href="https://doi.org/10.1242/jcs.89.1.25" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3417791/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cell%20Science&amp;title=Polo,%20a%20Mitotic%20Mutant%20of%20Drosophila%20Displaying%20Abnormal%20Spindle%20Poles&amp;author=C.%20E.%20Sunkel&amp;author=D.%20M.%20Glover&amp;volume=89&amp;publication_year=1988&amp;pages=25-38&amp;pmid=3417791&amp;doi=10.1242/jcs.89.1.25&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0002">
<span class="label">2.</span><cite>
Golsteyn R. M., Schultz S. J., Bartek J., Ziemiecki A., Ried T., and Nigg E. A., “Cell Cycle Analysis and Chromosomal Localization of Human Plk1, a Putative Homologue of the Mitotic Kinases Drosophila Polo and Saccharomyces Cerevisiae Cdc5,” Journal of Cell Science
107 (1994): 1509–1517, 10.1242/jcs.107.6.1509.
</cite> [<a href="https://doi.org/10.1242/jcs.107.6.1509" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7962193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cell%20Science&amp;title=Cell%20Cycle%20Analysis%20and%20Chromosomal%20Localization%20of%20Human%20Plk1,%20a%20Putative%20Homologue%20of%20the%20Mitotic%20Kinases%20Drosophila%20Polo%20and%20Saccharomyces%20Cerevisiae%20Cdc5&amp;author=R.%20M.%20Golsteyn&amp;author=S.%20J.%20Schultz&amp;author=J.%20Bartek&amp;author=A.%20Ziemiecki&amp;author=T.%20Ried&amp;volume=107&amp;publication_year=1994&amp;pages=1509-1517&amp;pmid=7962193&amp;doi=10.1242/jcs.107.6.1509&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0003">
<span class="label">3.</span><cite>
Chan E. H. Y., Santamaria A., Sillje H. H. W., and Nigg E. A., “Plk1 Regulates Mitotic Aurora a Function Through βTrCP‐Dependent Degradation of hBora,” Chromosoma
117 (2008): 457–469, 10.1007/s00412-008-0165-5.
</cite> [<a href="https://doi.org/10.1007/s00412-008-0165-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2921497/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18521620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chromosoma&amp;title=Plk1%20Regulates%20Mitotic%20Aurora%20a%20Function%20Through%20%CE%B2TrCP%E2%80%90Dependent%20Degradation%20of%20hBora&amp;author=E.%20H.%20Y.%20Chan&amp;author=A.%20Santamaria&amp;author=H.%20H.%20W.%20Sillje&amp;author=E.%20A.%20Nigg&amp;volume=117&amp;publication_year=2008&amp;pages=457-469&amp;pmid=18521620&amp;doi=10.1007/s00412-008-0165-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0004">
<span class="label">4.</span><cite>
Shao H., Huang Y., Zhang L., et al., “Spatiotemporal Dynamics of Aurora B‐PLK1‐MCAK Signaling Axis Orchestrates Kinetochore bi‐Orientation and Faithful Chromosome Segregation,” Scientific Reports
5 (2015): 1–16, 10.1038/srep12204.</cite> [<a href="https://doi.org/10.1038/srep12204" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4513279/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26206521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Spatiotemporal%20Dynamics%20of%20Aurora%20B%E2%80%90PLK1%E2%80%90MCAK%20Signaling%20Axis%20Orchestrates%20Kinetochore%20bi%E2%80%90Orientation%20and%20Faithful%20Chromosome%20Segregation&amp;author=H.%20Shao&amp;author=Y.%20Huang&amp;author=L.%20Zhang&amp;volume=5&amp;publication_year=2015&amp;pages=1-16&amp;pmid=26206521&amp;doi=10.1038/srep12204&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0005">
<span class="label">5.</span><cite>
Gutteridge R. E. A., Ndiaye M. A., Liu X., and Ahmad N., “Plk1 Inhibitors in Cancer Therapy: From Laboratory to Clinics,” Molecular Cancer Therapeutics
15 (2016): 1427–1435, 10.1158/1535-7163.MCT-15-0897.
</cite> [<a href="https://doi.org/10.1158/1535-7163.MCT-15-0897" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4936921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27330107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cancer%20Therapeutics&amp;title=Plk1%20Inhibitors%20in%20Cancer%20Therapy:%20From%20Laboratory%20to%20Clinics&amp;author=R.%20E.%20A.%20Gutteridge&amp;author=M.%20A.%20Ndiaye&amp;author=X.%20Liu&amp;author=N.%20Ahmad&amp;volume=15&amp;publication_year=2016&amp;pages=1427-1435&amp;pmid=27330107&amp;doi=10.1158/1535-7163.MCT-15-0897&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0006">
<span class="label">6.</span><cite>
Tokumitsu Y., Mori M., Tanaka S., Akazawa K., Nakano S., and Niho Y., “Prognostic Significance of Polo‐Like Kinase Expression in Esophageal Carcinoma,” International Journal of Oncology
15 (1999): 687–692, 10.3892/ijo.15.4.687.
</cite> [<a href="https://doi.org/10.3892/ijo.15.4.687" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10493949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Oncology&amp;title=Prognostic%20Significance%20of%20Polo%E2%80%90Like%20Kinase%20Expression%20in%20Esophageal%20Carcinoma&amp;author=Y.%20Tokumitsu&amp;author=M.%20Mori&amp;author=S.%20Tanaka&amp;author=K.%20Akazawa&amp;author=S.%20Nakano&amp;volume=15&amp;publication_year=1999&amp;pages=687-692&amp;pmid=10493949&amp;doi=10.3892/ijo.15.4.687&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0007">
<span class="label">7.</span><cite>
Forrest R. A., Swift L. P., Rephaeli A., et al., “Activation of DNA Damage Response Pathways as a Consequence of Anthracycline‐DNA Adduct Formation,” Biochemical Pharmacology
83 (2012): 1602–1612, 10.1016/j.bcp.2012.02.026.
</cite> [<a href="https://doi.org/10.1016/j.bcp.2012.02.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22414726/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemical%20Pharmacology&amp;title=Activation%20of%20DNA%20Damage%20Response%20Pathways%20as%20a%20Consequence%20of%20Anthracycline%E2%80%90DNA%20Adduct%20Formation&amp;author=R.%20A.%20Forrest&amp;author=L.%20P.%20Swift&amp;author=A.%20Rephaeli&amp;volume=83&amp;publication_year=2012&amp;pages=1602-1612&amp;pmid=22414726&amp;doi=10.1016/j.bcp.2012.02.026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0008">
<span class="label">8.</span><cite>
Liu X. S., Li H., Song B., and Liu X., “Polo‐Like Kinase 1 Phosphorylation of G2 and S‐Phase‐Expressed 1 Protein Is Essential for p53 Inactivation During G2 Checkpoint Recovery,” EMBO Reports
11 (2010): 626–632, 10.1038/embor.2010.90.
</cite> [<a href="https://doi.org/10.1038/embor.2010.90" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2920445/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20577264/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Reports&amp;title=Polo%E2%80%90Like%20Kinase%201%20Phosphorylation%20of%20G2%20and%20S%E2%80%90Phase%E2%80%90Expressed%201%20Protein%20Is%20Essential%20for%20p53%20Inactivation%20During%20G2%20Checkpoint%20Recovery&amp;author=X.%20S.%20Liu&amp;author=H.%20Li&amp;author=B.%20Song&amp;author=X.%20Liu&amp;volume=11&amp;publication_year=2010&amp;pages=626-632&amp;pmid=20577264&amp;doi=10.1038/embor.2010.90&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0009">
<span class="label">9.</span><cite>
Yang X., Li H., Zhou Z., et al., “Plk1‐Mediated Phosphorylation of Topors Regulates p53 Stability *,” Journal of Biological Chemistry
284 (2009): 18588–18592, 10.1074/jbc.C109.001560.
</cite> [<a href="https://doi.org/10.1074/jbc.C109.001560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2707202/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19473992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=Plk1%E2%80%90Mediated%20Phosphorylation%20of%20Topors%20Regulates%20p53%20Stability%20*&amp;author=X.%20Yang&amp;author=H.%20Li&amp;author=Z.%20Zhou&amp;volume=284&amp;publication_year=2009&amp;pages=18588-18592&amp;pmid=19473992&amp;doi=10.1074/jbc.C109.001560&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0010">
<span class="label">10.</span><cite>
Li H., Liu X. S., Yang X., et al., “Phosphorylation of CLIP‐170 by Plk1 and CK2 Promotes Timely Formation of Kinetochore–Microtubule Attachments,” EMBO Journal
29 (2010): 2953–2965, 10.1038/emboj.2010.174.
</cite> [<a href="https://doi.org/10.1038/emboj.2010.174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2944045/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20664522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Journal&amp;title=Phosphorylation%20of%20CLIP%E2%80%90170%20by%20Plk1%20and%20CK2%20Promotes%20Timely%20Formation%20of%20Kinetochore%E2%80%93Microtubule%20Attachments&amp;author=H.%20Li&amp;author=X.%20S.%20Liu&amp;author=X.%20Yang&amp;volume=29&amp;publication_year=2010&amp;pages=2953-2965&amp;pmid=20664522&amp;doi=10.1038/emboj.2010.174&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0011">
<span class="label">11.</span><cite>
Hou X., Li Z., Huang W., et al., “Plk1‐Dependent Microtubule Dynamics Promotes Androgen Receptor Signaling in Prostate Cancer,” Prostate
73 (2013): 1352–1363, 10.1002/pros.22683.
</cite> [<a href="https://doi.org/10.1002/pros.22683" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3720713/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23661607/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Prostate&amp;title=Plk1%E2%80%90Dependent%20Microtubule%20Dynamics%20Promotes%20Androgen%20Receptor%20Signaling%20in%20Prostate%20Cancer&amp;author=X.%20Hou&amp;author=Z.%20Li&amp;author=W.%20Huang&amp;volume=73&amp;publication_year=2013&amp;pages=1352-1363&amp;pmid=23661607&amp;doi=10.1002/pros.22683&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0012">
<span class="label">12.</span><cite>
Song B., Liu X. S., Davis K., and Liu X., “Plk1 Phosphorylation of Orc2 Promotes DNA Replication Under Conditions of Stress,” Molecular and Cellular Biology
31 (2011): 4844–4856, 10.1128/MCB.06110-11.
</cite> [<a href="https://doi.org/10.1128/MCB.06110-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3232917/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21947279/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20and%20Cellular%20Biology&amp;title=Plk1%20Phosphorylation%20of%20Orc2%20Promotes%20DNA%20Replication%20Under%20Conditions%20of%20Stress&amp;author=B.%20Song&amp;author=X.%20S.%20Liu&amp;author=K.%20Davis&amp;author=X.%20Liu&amp;volume=31&amp;publication_year=2011&amp;pages=4844-4856&amp;pmid=21947279&amp;doi=10.1128/MCB.06110-11&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0013">
<span class="label">13.</span><cite>
Song B., Liu X. S., Rice S. J., et al., “Plk1 Phosphorylation of Orc2 and Hbo1 Contributes to Gemcitabine Resistance in Pancreatic Cancer,” Molecular Cancer Therapeutics
12 (2013): 58–68, 10.1158/1535-7163.MCT-12-0632.
</cite> [<a href="https://doi.org/10.1158/1535-7163.MCT-12-0632" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3732037/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23188630/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cancer%20Therapeutics&amp;title=Plk1%20Phosphorylation%20of%20Orc2%20and%20Hbo1%20Contributes%20to%20Gemcitabine%20Resistance%20in%20Pancreatic%20Cancer&amp;author=B.%20Song&amp;author=X.%20S.%20Liu&amp;author=S.%20J.%20Rice&amp;volume=12&amp;publication_year=2013&amp;pages=58-68&amp;pmid=23188630&amp;doi=10.1158/1535-7163.MCT-12-0632&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0014">
<span class="label">14.</span><cite>
Beria I., Bossi R. T., Brasca M. G., et al., “NMS‐P937, a 4,5‐Dihydro‐1H‐Pyrazolo[4,3‐h]quinazoline Derivative as Potent and Selective Polo‐Like Kinase 1 Inhibitor,” Bioorganic &amp; Medicinal Chemistry Letters
21 (2011): 2969–2974, 10.1016/j.bmcl.2011.03.054.
</cite> [<a href="https://doi.org/10.1016/j.bmcl.2011.03.054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21470862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorganic%20&amp;%20Medicinal%20Chemistry%20Letters&amp;title=NMS%E2%80%90P937,%20a%204,5%E2%80%90Dihydro%E2%80%901H%E2%80%90Pyrazolo%5B4,3%E2%80%90h%5Dquinazoline%20Derivative%20as%20Potent%20and%20Selective%20Polo%E2%80%90Like%20Kinase%201%20Inhibitor&amp;author=I.%20Beria&amp;author=R.%20T.%20Bossi&amp;author=M.%20G.%20Brasca&amp;volume=21&amp;publication_year=2011&amp;pages=2969-2974&amp;pmid=21470862&amp;doi=10.1016/j.bmcl.2011.03.054&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0015">
<span class="label">15.</span><cite>
Valsasina B., Beria I., Alli C., et al., “NMS‐P937, an Orally Available, Specific Small‐Molecule Polo‐Like Kinase 1 Inhibitor With Antitumor Activity in Solid and Hematologic Malignancies,” Molecular Cancer Therapeutics
11 (2012): 1006–1016, 10.1158/1535-7163.MCT-11-0765.
</cite> [<a href="https://doi.org/10.1158/1535-7163.MCT-11-0765" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22319201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cancer%20Therapeutics&amp;title=NMS%E2%80%90P937,%20an%20Orally%20Available,%20Specific%20Small%E2%80%90Molecule%20Polo%E2%80%90Like%20Kinase%201%20Inhibitor%20With%20Antitumor%20Activity%20in%20Solid%20and%20Hematologic%20Malignancies&amp;author=B.%20Valsasina&amp;author=I.%20Beria&amp;author=C.%20Alli&amp;volume=11&amp;publication_year=2012&amp;pages=1006-1016&amp;pmid=22319201&amp;doi=10.1158/1535-7163.MCT-11-0765&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0016">
<span class="label">16.</span><cite>
Hagege A., Ambrosetti D., Boyer J., et al., “The Polo‐Like Kinase 1 Inhibitor Onvansertib Represents a Relevant Treatment for Head and Neck Squamous Cell Carcinoma Resistant to Cisplatin and Radiotherapy,” Theranostics
11 (2021): 9571–9586, 10.7150/thno.61711.
</cite> [<a href="https://doi.org/10.7150/thno.61711" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8490521/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34646387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theranostics&amp;title=The%20Polo%E2%80%90Like%20Kinase%201%20Inhibitor%20Onvansertib%20Represents%20a%20Relevant%20Treatment%20for%20Head%20and%20Neck%20Squamous%20Cell%20Carcinoma%20Resistant%20to%20Cisplatin%20and%20Radiotherapy&amp;author=A.%20Hagege&amp;author=D.%20Ambrosetti&amp;author=J.%20Boyer&amp;volume=11&amp;publication_year=2021&amp;pages=9571-9586&amp;pmid=34646387&amp;doi=10.7150/thno.61711&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0017">
<span class="label">17.</span><cite>
Zeidan A. M., Ridinger M., Lin T. L., et al., “A Phase Ib Study of Onvansertib, a Novel Oral PLK1 Inhibitor, in Combination Therapy for Patients With Relapsed or Refractory Acute Myeloid Leukemia,” Clinical Cancer Research
26 (2020): 6132–6140, 10.1158/1078-0432.CCR-20-2586.
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-20-2586" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32998961/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clinical%20Cancer%20Research&amp;title=A%20Phase%20Ib%20Study%20of%20Onvansertib,%20a%20Novel%20Oral%20PLK1%20Inhibitor,%20in%20Combination%20Therapy%20for%20Patients%20With%20Relapsed%20or%20Refractory%20Acute%20Myeloid%20Leukemia&amp;author=A.%20M.%20Zeidan&amp;author=M.%20Ridinger&amp;author=T.%20L.%20Lin&amp;volume=26&amp;publication_year=2020&amp;pages=6132-6140&amp;pmid=32998961&amp;doi=10.1158/1078-0432.CCR-20-2586&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0018">
<span class="label">18.</span><cite>
Lenz H. J., Ahn D., Erlander M., and Barzi A., “667TiP—A Phase Ib/II Study of Onvansertib (PCM‐075) in Combination With FOLFIRI and Bevacizumab for Second‐Line Treatment of Metastatic Colorectal Cancer in Patients With a KRAS Mutation. Abstract Book of the 44th ESMO Congress <em>(ESMO 2019) 27 September – 1 October 2019</em>, Barcelona, Spain,” Annals of Oncology
30 (2019): v250, 10.1093/annonc/mdz246.144.</cite> [<a href="https://doi.org/10.1093/annonc/mdz246.144" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Oncology&amp;title=667TiP%E2%80%94A%20Phase%20Ib/II%20Study%20of%20Onvansertib%20(PCM%E2%80%90075)%20in%20Combination%20With%20FOLFIRI%20and%20Bevacizumab%20for%20Second%E2%80%90Line%20Treatment%20of%20Metastatic%20Colorectal%20Cancer%20in%20Patients%20With%20a%20KRAS%20Mutation&amp;author=H.%20J.%20Lenz&amp;author=D.%20Ahn&amp;author=M.%20Erlander&amp;author=A.%20Barzi&amp;volume=30&amp;publication_year=2019&amp;pages=v250&amp;doi=10.1093/annonc/mdz246.144&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0019">
<span class="label">19.</span><cite>
Sutherland R. M., McCredie J. A., and Inch W. R., “Growth of Multicell Spheroids in Tissue Culture as a Model of Nodular Carcinomas2,” JNCI Journal of the National Cancer Institute
46, no. 1 (1971): 113–120, 10.1093/jnci/46.1.113.
</cite> [<a href="https://doi.org/10.1093/jnci/46.1.113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/5101993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JNCI%20Journal%20of%20the%20National%20Cancer%20Institute&amp;title=Growth%20of%20Multicell%20Spheroids%20in%20Tissue%20Culture%20as%20a%20Model%20of%20Nodular%20Carcinomas2&amp;author=R.%20M.%20Sutherland&amp;author=J.%20A.%20McCredie&amp;author=W.%20R.%20Inch&amp;volume=46&amp;issue=1&amp;publication_year=1971&amp;pages=113-120&amp;pmid=5101993&amp;doi=10.1093/jnci/46.1.113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0020">
<span class="label">20.</span><cite>
Carlsson J., Stålnacke C. G., Acker H., Haji‐Karim M., Nilsson S., and Larsson B., “The Influence of Oxygen on Viability and Proliferation in Cellular Spheroids,” International Journal of Radiation Oncology, Biology, Physics
5 (1979): 2011–2020, 10.1016/0360-3016(79)90953-2.
</cite> [<a href="https://doi.org/10.1016/0360-3016(79)90953-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/544580/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Radiation%20Oncology,%20Biology,%20Physics&amp;title=The%20Influence%20of%20Oxygen%20on%20Viability%20and%20Proliferation%20in%20Cellular%20Spheroids&amp;author=J.%20Carlsson&amp;author=C.%20G.%20St%C3%A5lnacke&amp;author=H.%20Acker&amp;author=M.%20Haji%E2%80%90Karim&amp;author=S.%20Nilsson&amp;volume=5&amp;publication_year=1979&amp;pages=2011-2020&amp;pmid=544580&amp;doi=10.1016/0360-3016(79)90953-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0021">
<span class="label">21.</span><cite>
Mueller‐Klieser W. F. and Sutherland R. M., “Oxygen Tensions in Multicell Spheroids of Two Cell Lines,” British Journal of Cancer
45 (1982): 256–264, 10.1038/bjc.1982.41.
</cite> [<a href="https://doi.org/10.1038/bjc.1982.41" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2010909/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7059474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=British%20Journal%20of%20Cancer&amp;title=Oxygen%20Tensions%20in%20Multicell%20Spheroids%20of%20Two%20Cell%20Lines&amp;author=W.%20F.%20Mueller%E2%80%90Klieser&amp;author=R.%20M.%20Sutherland&amp;volume=45&amp;publication_year=1982&amp;pages=256-264&amp;pmid=7059474&amp;doi=10.1038/bjc.1982.41&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0022">
<span class="label">22.</span><cite>
Mueller‐Klieser W. F. and Sutherland R. M., “Influence of Convection in the Growth Medium on Oxygen Tensions in Multicellular Tumor Spheroids1,” Cancer Research
42 (1982): 237–242.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/7053851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Research&amp;title=Influence%20of%20Convection%20in%20the%20Growth%20Medium%20on%20Oxygen%20Tensions%20in%20Multicellular%20Tumor%20Spheroids1&amp;author=W.%20F.%20Mueller%E2%80%90Klieser&amp;author=R.%20M.%20Sutherland&amp;volume=42&amp;publication_year=1982&amp;pages=237-242&amp;pmid=7053851&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0023">
<span class="label">23.</span><cite>
Han J. J., “FDA Modernization Act 2.0 Allows for Alternatives to Animal Testing,” Artificial Organs
47 (2023): 449–450, 10.1111/aor.14503.
</cite> [<a href="https://doi.org/10.1111/aor.14503" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36762462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Artificial%20Organs&amp;title=FDA%20Modernization%20Act%202.0%20Allows%20for%20Alternatives%20to%20Animal%20Testing&amp;author=J.%20J.%20Han&amp;volume=47&amp;publication_year=2023&amp;pages=449-450&amp;pmid=36762462&amp;doi=10.1111/aor.14503&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0024">
<span class="label">24.</span><cite>
Gaedtke L., Thoenes L., Culmsee C., Mayer B., and Wagner E., “Proteomic Analysis Reveals Differences in Protein Expression in Spheroid Versus Monolayer Cultures of Low‐Passage Colon Carcinoma Cells,” Journal of Proteome Research
6 (2007): 4111–4118, 10.1021/pr0700596.
</cite> [<a href="https://doi.org/10.1021/pr0700596" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17918984/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteome%20Research&amp;title=Proteomic%20Analysis%20Reveals%20Differences%20in%20Protein%20Expression%20in%20Spheroid%20Versus%20Monolayer%20Cultures%20of%20Low%E2%80%90Passage%20Colon%20Carcinoma%20Cells&amp;author=L.%20Gaedtke&amp;author=L.%20Thoenes&amp;author=C.%20Culmsee&amp;author=B.%20Mayer&amp;author=E.%20Wagner&amp;volume=6&amp;publication_year=2007&amp;pages=4111-4118&amp;pmid=17918984&amp;doi=10.1021/pr0700596&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0025">
<span class="label">25.</span><cite>
Yue X., Lukowski J. K., Weaver E. M., Skube S. B., and Hummon A. B., “Quantitative Proteomic and Phosphoproteomic Comparison of 2D and 3D Colon Cancer Cell Culture Models,” Journal of Proteome Research
15 (2016): 4265–4276, 10.1021/acs.jproteome.6b00342.
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.6b00342" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5334570/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27696853/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteome%20Research&amp;title=Quantitative%20Proteomic%20and%20Phosphoproteomic%20Comparison%20of%202D%20and%203D%20Colon%20Cancer%20Cell%20Culture%20Models&amp;author=X.%20Yue&amp;author=J.%20K.%20Lukowski&amp;author=E.%20M.%20Weaver&amp;author=S.%20B.%20Skube&amp;author=A.%20B.%20Hummon&amp;volume=15&amp;publication_year=2016&amp;pages=4265-4276&amp;pmid=27696853&amp;doi=10.1021/acs.jproteome.6b00342&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0026">
<span class="label">26.</span><cite>
Tobias F. and Hummon A. B., “Lipidomic Comparison of 2D and 3D Colon Cancer Cell Culture Models,” Journal of Mass Spectrometry
57 (2022): e4880, 10.1002/jms.4880.
</cite> [<a href="https://doi.org/10.1002/jms.4880" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9526240/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36028991/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Mass%20Spectrometry&amp;title=Lipidomic%20Comparison%20of%202D%20and%203D%20Colon%20Cancer%20Cell%20Culture%20Models&amp;author=F.%20Tobias&amp;author=A.%20B.%20Hummon&amp;volume=57&amp;publication_year=2022&amp;pages=e4880&amp;pmid=36028991&amp;doi=10.1002/jms.4880&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0027">
<span class="label">27.</span><cite>
Wen S., Tu X., Zang Q., et al., “Liquid Chromatography–Mass Spectrometry‐Based Metabolomics and Fluxomics Reveals the Metabolic Alterations in Glioma U87MG Multicellular Tumor Spheroids Versus Two‐Dimensional Cell Cultures,” Rapid Communications in Mass Spectrometry
38 (2024): e9670, 10.1002/rcm.9670.
</cite> [<a href="https://doi.org/10.1002/rcm.9670" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38124173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rapid%20Communications%20in%20Mass%20Spectrometry&amp;title=Liquid%20Chromatography%E2%80%93Mass%20Spectrometry%E2%80%90Based%20Metabolomics%20and%20Fluxomics%20Reveals%20the%20Metabolic%20Alterations%20in%20Glioma%20U87MG%20Multicellular%20Tumor%20Spheroids%20Versus%20Two%E2%80%90Dimensional%20Cell%20Cultures&amp;author=S.%20Wen&amp;author=X.%20Tu&amp;author=Q.%20Zang&amp;volume=38&amp;publication_year=2024&amp;pages=e9670&amp;pmid=38124173&amp;doi=10.1002/rcm.9670&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0028">
<span class="label">28.</span><cite>
de Oliveira Silva N., de Lima L. V. A., de Oliveira L. M., et al., “Cellular and Molecular Antiproliferative Effects in 2D Monolayer and 3D‐Cultivated HT‐29 Cells Treated With Zerumbone,” Naunyn‐Schmiedeberg's Archives of Pharmacology
397 (2023): 1561–1573, 10.1007/s00210-023-02701-4.
</cite> [<a href="https://doi.org/10.1007/s00210-023-02701-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37672080/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Naunyn%E2%80%90Schmiedeberg's%20Archives%20of%20Pharmacology&amp;title=Cellular%20and%20Molecular%20Antiproliferative%20Effects%20in%202D%20Monolayer%20and%203D%E2%80%90Cultivated%20HT%E2%80%9029%20Cells%20Treated%20With%20Zerumbone&amp;author=N.%20de%20Oliveira%20Silva&amp;author=L.%20V.%20A.%20de%20Lima&amp;author=L.%20M.%20de%20Oliveira&amp;volume=397&amp;publication_year=2023&amp;pages=1561-1573&amp;pmid=37672080&amp;doi=10.1007/s00210-023-02701-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0029">
<span class="label">29.</span><cite>
Świerczewska M., Sterzyńska K., Ruciński M., Andrzejewska M., Nowicki M., and Januchowski R., “The Response and Resistance to Drugs in Ovarian Cancer Cell Lines in 2D Monolayers and 3D Spheroids,” Biomedicine &amp; Pharmacotherapy
165 (2023): 115152–115192, 10.1016/j.biopha.2023.115152.
</cite> [<a href="https://doi.org/10.1016/j.biopha.2023.115152" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37442067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomedicine%20&amp;%20Pharmacotherapy&amp;title=The%20Response%20and%20Resistance%20to%20Drugs%20in%20Ovarian%20Cancer%20Cell%20Lines%20in%202D%20Monolayers%20and%203D%20Spheroids&amp;author=M.%20%C5%9Awierczewska&amp;author=K.%20Sterzy%C5%84ska&amp;author=M.%20Ruci%C5%84ski&amp;author=M.%20Andrzejewska&amp;author=M.%20Nowicki&amp;volume=165&amp;publication_year=2023&amp;pages=115152-115192&amp;pmid=37442067&amp;doi=10.1016/j.biopha.2023.115152&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0030">
<span class="label">30.</span><cite>
Liu X., Weaver E. M., and Hummon A. B., “Evaluation of Therapeutics in Three‐Dimensional Cell Culture Systems by MALDI Imaging Mass Spectrometry,” Analytical Chemistry
85 (2013): 6295–6302, 10.1021/ac400519c.
</cite> [<a href="https://doi.org/10.1021/ac400519c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4118837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23724927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Evaluation%20of%20Therapeutics%20in%20Three%E2%80%90Dimensional%20Cell%20Culture%20Systems%20by%20MALDI%20Imaging%20Mass%20Spectrometry&amp;author=X.%20Liu&amp;author=E.%20M.%20Weaver&amp;author=A.%20B.%20Hummon&amp;volume=85&amp;publication_year=2013&amp;pages=6295-6302&amp;pmid=23724927&amp;doi=10.1021/ac400519c&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0031">
<span class="label">31.</span><cite>
Wang Y. and Hummon A. B., “MS Imaging of Multicellular Tumor Spheroids and Organoids as an Emerging Tool for Personalized Medicine and Drug Discovery,” Journal of Biological Chemistry
297 (2021): 101139, 10.1016/j.jbc.2021.101139.
</cite> [<a href="https://doi.org/10.1016/j.jbc.2021.101139" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8463860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34461098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biological%20Chemistry&amp;title=MS%20Imaging%20of%20Multicellular%20Tumor%20Spheroids%20and%20Organoids%20as%20an%20Emerging%20Tool%20for%20Personalized%20Medicine%20and%20Drug%20Discovery&amp;author=Y.%20Wang&amp;author=A.%20B.%20Hummon&amp;volume=297&amp;publication_year=2021&amp;pages=101139&amp;pmid=34461098&amp;doi=10.1016/j.jbc.2021.101139&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0032">
<span class="label">32.</span><cite>
Li H. and Hummon A. B., “Imaging Mass Spectrometry of Three‐Dimensional Cell Culture Systems,” Analytical Chemistry
83 (2011): 8794–8801, 10.1021/ac202356g.
</cite> [<a href="https://doi.org/10.1021/ac202356g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21992577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=Imaging%20Mass%20Spectrometry%20of%20Three%E2%80%90Dimensional%20Cell%20Culture%20Systems&amp;author=H.%20Li&amp;author=A.%20B.%20Hummon&amp;volume=83&amp;publication_year=2011&amp;pages=8794-8801&amp;pmid=21992577&amp;doi=10.1021/ac202356g&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0033">
<span class="label">33.</span><cite>
Ahlf Wheatcraft D. R., Liu X., and Hummon A. B., “Sample Preparation Strategies for Mass Spectrometry Imaging of 3D Cell Culture Models,” JoVE
94 (2014): 1–7, 10.3791/52313.</cite> [<a href="https://doi.org/10.3791/52313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4396945/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25549242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JoVE&amp;title=Sample%20Preparation%20Strategies%20for%20Mass%20Spectrometry%20Imaging%20of%203D%20Cell%20Culture%20Models&amp;author=D.%20R.%20Ahlf%20Wheatcraft&amp;author=X.%20Liu&amp;author=A.%20B.%20Hummon&amp;volume=94&amp;publication_year=2014&amp;pages=1-7&amp;pmid=25549242&amp;doi=10.3791/52313&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0034">
<span class="label">34.</span><cite>
Searle B. C., Pino L. K., Egertson J. D., et al., “Chromatogram Libraries Improve Peptide Detection and Quantification by Data Independent Acquisition Mass Spectrometry,” Nature Communications
9 (2018): 1–12, 10.1038/s41467-018-07454-w.</cite> [<a href="https://doi.org/10.1038/s41467-018-07454-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6277451/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30510204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Chromatogram%20Libraries%20Improve%20Peptide%20Detection%20and%20Quantification%20by%20Data%20Independent%20Acquisition%20Mass%20Spectrometry&amp;author=B.%20C.%20Searle&amp;author=L.%20K.%20Pino&amp;author=J.%20D.%20Egertson&amp;volume=9&amp;publication_year=2018&amp;pages=1-12&amp;pmid=30510204&amp;doi=10.1038/s41467-018-07454-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0035">
<span class="label">35.</span><cite>
Searle B. C., Swearingen K. E., Barnes C. A., et al., “Generating High Quality Libraries for DIA MS With Empirically Corrected Peptide Predictions,” Nature Communications
11 (2020): 1–10, 10.1038/s41467-020-15346-1.</cite> [<a href="https://doi.org/10.1038/s41467-020-15346-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7096433/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32214105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Generating%20High%20Quality%20Libraries%20for%20DIA%20MS%20With%20Empirically%20Corrected%20Peptide%20Predictions&amp;author=B.%20C.%20Searle&amp;author=K.%20E.%20Swearingen&amp;author=C.%20A.%20Barnes&amp;volume=11&amp;publication_year=2020&amp;pages=1-10&amp;pmid=32214105&amp;doi=10.1038/s41467-020-15346-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0036">
<span class="label">36.</span><cite>
Chambers M. C., Maclean B., Burke R., et al., “A Cross‐Platform Toolkit for Mass Spectrometry and Proteomics,” Nature Biotechnology
30 (2012): 918–920, 10.1038/nbt.2377.</cite> [<a href="https://doi.org/10.1038/nbt.2377" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3471674/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23051804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=A%20Cross%E2%80%90Platform%20Toolkit%20for%20Mass%20Spectrometry%20and%20Proteomics&amp;author=M.%20C.%20Chambers&amp;author=B.%20Maclean&amp;author=R.%20Burke&amp;volume=30&amp;publication_year=2012&amp;pages=918-920&amp;pmid=23051804&amp;doi=10.1038/nbt.2377&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0037">
<span class="label">37.</span><cite>
Palmer A., Phapale P., Chernyavsky I., et al., “FDR‐Controlled Metabolite Annotation for High‐Resolution Imaging Mass Spectrometry,” Nature Methods
14 (2017): 57–60, 10.1038/nmeth.4072.
</cite> [<a href="https://doi.org/10.1038/nmeth.4072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27842059/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=FDR%E2%80%90Controlled%20Metabolite%20Annotation%20for%20High%E2%80%90Resolution%20Imaging%20Mass%20Spectrometry&amp;author=A.%20Palmer&amp;author=P.%20Phapale&amp;author=I.%20Chernyavsky&amp;volume=14&amp;publication_year=2017&amp;pages=57-60&amp;pmid=27842059&amp;doi=10.1038/nmeth.4072&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0038">
<span class="label">38.</span><cite>
Gessulat S., Schmidt T., Zolg D. P., et al., “Prosit: Proteome‐Wide Prediction of Peptide Tandem Mass Spectra by Deep Learning,” Nature Methods
16 (2019): 509–518, 10.1038/s41592-019-0426-7.
</cite> [<a href="https://doi.org/10.1038/s41592-019-0426-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31133760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Prosit:%20Proteome%E2%80%90Wide%20Prediction%20of%20Peptide%20Tandem%20Mass%20Spectra%20by%20Deep%20Learning&amp;author=S.%20Gessulat&amp;author=T.%20Schmidt&amp;author=D.%20P.%20Zolg&amp;volume=16&amp;publication_year=2019&amp;pages=509-518&amp;pmid=31133760&amp;doi=10.1038/s41592-019-0426-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0039">
<span class="label">39.</span><cite>
Käll L., Canterbury J. D., Weston J., Noble W. S., and MacCoss M., “Semi‐Supervised Learning for Peptide Identification From Shotgun Proteomics Datasets,” Nature Methods
4 (2007): 923–925, 10.1038/nmeth1113.
</cite> [<a href="https://doi.org/10.1038/nmeth1113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17952086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Semi%E2%80%90Supervised%20Learning%20for%20Peptide%20Identification%20From%20Shotgun%20Proteomics%20Datasets&amp;author=L.%20K%C3%A4ll&amp;author=J.%20D.%20Canterbury&amp;author=J.%20Weston&amp;author=W.%20S.%20Noble&amp;author=M.%20MacCoss&amp;volume=4&amp;publication_year=2007&amp;pages=923-925&amp;pmid=17952086&amp;doi=10.1038/nmeth1113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0040">
<span class="label">40.</span><cite>
Pino L. K., Searle B. C., Bollinger J. G., Nunn B., MacLean B., and MacCoss M., “The Skyline Ecosystem: Informatics for Quantitative Mass Spectrometry Proteomics,” Mass Spectrometry Reviews
39 (2020): 229–244, 10.1002/mas.21540.
</cite> [<a href="https://doi.org/10.1002/mas.21540" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5799042/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28691345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mass%20Spectrometry%20Reviews&amp;title=The%20Skyline%20Ecosystem:%20Informatics%20for%20Quantitative%20Mass%20Spectrometry%20Proteomics&amp;author=L.%20K.%20Pino&amp;author=B.%20C.%20Searle&amp;author=J.%20G.%20Bollinger&amp;author=B.%20Nunn&amp;author=B.%20MacLean&amp;volume=39&amp;publication_year=2020&amp;pages=229-244&amp;pmid=28691345&amp;doi=10.1002/mas.21540&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0041">
<span class="label">41.</span><cite>
Choi M., Chang C. Y., Clough T., et al., “MSstats: An R Package for Statistical Analysis of Quantitative Mass Spectrometry‐Based Proteomic Experiments,” Bioinformatics
30 (2014): 2524–2526, 10.1093/bioinformatics/btu305.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu305" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24794931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=MSstats:%20An%20R%20Package%20for%20Statistical%20Analysis%20of%20Quantitative%20Mass%20Spectrometry%E2%80%90Based%20Proteomic%20Experiments&amp;author=M.%20Choi&amp;author=C.%20Y.%20Chang&amp;author=T.%20Clough&amp;volume=30&amp;publication_year=2014&amp;pages=2524-2526&amp;pmid=24794931&amp;doi=10.1093/bioinformatics/btu305&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0042">
<span class="label">42.</span><cite>
Krämer A., Green J., 
J.
Pollard, Jr.
, and Tugendreich S., “Causal Analysis Approaches in Ingenuity Pathway Analysis,” Bioinformatics
30 (2014): 523–530, 10.1093/bioinformatics/btt703.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3928520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24336805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Causal%20Analysis%20Approaches%20in%20Ingenuity%20Pathway%20Analysis&amp;author=A.%20Kr%C3%A4mer&amp;author=J.%20Green&amp;author=J.%20Pollard&amp;author=S.%20Tugendreich&amp;volume=30&amp;publication_year=2014&amp;pages=523-530&amp;pmid=24336805&amp;doi=10.1093/bioinformatics/btt703&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0043">
<span class="label">43.</span><cite>
Yu G., Wang L. G., Han Y., and He Q. Y., “clusterProfiler: An R Package for Comparing Biological Themes Among Gene Clusters,” OMICS: A Journal of Integrative Biology
16 (2012): 284–287, 10.1089/omi.2011.0118.
</cite> [<a href="https://doi.org/10.1089/omi.2011.0118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3339379/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22455463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=OMICS:%20A%20Journal%20of%20Integrative%20Biology&amp;title=clusterProfiler:%20An%20R%20Package%20for%20Comparing%20Biological%20Themes%20Among%20Gene%20Clusters&amp;author=G.%20Yu&amp;author=L.%20G.%20Wang&amp;author=Y.%20Han&amp;author=Q.%20Y.%20He&amp;volume=16&amp;publication_year=2012&amp;pages=284-287&amp;pmid=22455463&amp;doi=10.1089/omi.2011.0118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0044">
<span class="label">44.</span><cite>
Benjamini Y. and Hochberg Y., “Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing,” Journal of the Royal Statistical Society: Series B: Methodological
57 (1995): 289–300.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20the%20Royal%20Statistical%20Society:%20Series%20B:%20Methodological&amp;title=Controlling%20the%20False%20Discovery%20Rate:%20A%20Practical%20and%20Powerful%20Approach%20to%20Multiple%20Testing&amp;author=Y.%20Benjamini&amp;author=Y.%20Hochberg&amp;volume=57&amp;publication_year=1995&amp;pages=289-300&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0045">
<span class="label">45.</span><cite>
Kobayashi Y., Hayasaka T., Setou M., Itoh H., and Kanayama N., “Comparison of Phospholipid Molecular Species Between Terminal and Stem Villi of Human Term Placenta by Imaging Mass Spectrometry,” Placenta
31 (2010): 245–248, 10.1016/j.placenta.2009.12.026.
</cite> [<a href="https://doi.org/10.1016/j.placenta.2009.12.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20116095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Placenta&amp;title=Comparison%20of%20Phospholipid%20Molecular%20Species%20Between%20Terminal%20and%20Stem%20Villi%20of%20Human%20Term%20Placenta%20by%20Imaging%20Mass%20Spectrometry&amp;author=Y.%20Kobayashi&amp;author=T.%20Hayasaka&amp;author=M.%20Setou&amp;author=H.%20Itoh&amp;author=N.%20Kanayama&amp;volume=31&amp;publication_year=2010&amp;pages=245-248&amp;pmid=20116095&amp;doi=10.1016/j.placenta.2009.12.026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0046">
<span class="label">46.</span><cite>
Meriaux C., Franck J., Wisztorski M., Salzet M., and Fournier I., “Liquid Ionic Matrixes for MALDI Mass Spectrometry Imaging of Lipids. <em>Clinical Proteomics (3rd EuPA Meeting)</em>
,” Journal of Proteomics
73, no. 6 (2010): 1204–1218, 10.1016/j.jprot.2010.02.010.
</cite> [<a href="https://doi.org/10.1016/j.jprot.2010.02.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20188221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Proteomics&amp;title=Liquid%20Ionic%20Matrixes%20for%20MALDI%20Mass%20Spectrometry%20Imaging%20of%20Lipids.%20Clinical%20Proteomics%20(3rd%20EuPA%20Meeting)&amp;author=C.%20Meriaux&amp;author=J.%20Franck&amp;author=M.%20Wisztorski&amp;author=M.%20Salzet&amp;author=I.%20Fournier&amp;volume=73&amp;issue=6&amp;publication_year=2010&amp;pages=1204-1218&amp;pmid=20188221&amp;doi=10.1016/j.jprot.2010.02.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0047">
<span class="label">47.</span><cite>
Sugiura Y., Konishi Y., Zaima N., et al., “Visualization of the Cell‐Selective Distribution of PUFA‐Containing Phosphatidylcholines in Mouse Brain by Imaging Mass Spectrometry,” Journal of Lipid Research
50 (2009): 1776–1788, 10.1194/jlr.M900047-JLR200.
</cite> [<a href="https://doi.org/10.1194/jlr.M900047-JLR200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2724791/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19417221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Lipid%20Research&amp;title=Visualization%20of%20the%20Cell%E2%80%90Selective%20Distribution%20of%20PUFA%E2%80%90Containing%20Phosphatidylcholines%20in%20Mouse%20Brain%20by%20Imaging%20Mass%20Spectrometry&amp;author=Y.%20Sugiura&amp;author=Y.%20Konishi&amp;author=N.%20Zaima&amp;volume=50&amp;publication_year=2009&amp;pages=1776-1788&amp;pmid=19417221&amp;doi=10.1194/jlr.M900047-JLR200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0048">
<span class="label">48.</span><cite>
Niehaus M., Soltwisch J., Belov M. E., and Dreisewerd K., “Transmission‐Mode MALDI‐2 Mass Spectrometry Imaging of Cells and Tissues at Subcellular Resolution,” Nature Methods
16 (2019): 925–931, 10.1038/s41592-019-0536-2.
</cite> [<a href="https://doi.org/10.1038/s41592-019-0536-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31451764/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Transmission%E2%80%90Mode%20MALDI%E2%80%902%20Mass%20Spectrometry%20Imaging%20of%20Cells%20and%20Tissues%20at%20Subcellular%20Resolution&amp;author=M.%20Niehaus&amp;author=J.%20Soltwisch&amp;author=M.%20E.%20Belov&amp;author=K.%20Dreisewerd&amp;volume=16&amp;publication_year=2019&amp;pages=925-931&amp;pmid=31451764&amp;doi=10.1038/s41592-019-0536-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0049">
<span class="label">49.</span><cite>
McMillen J. C., Fincher J. A., Klein D. R., Spraggins J. M., and Caprioli R. M., “Effect of MALDI Matrices on Lipid Analyses of Biological Tissues Using MALDI‐2 Postionization Mass Spectrometry,” Journal of Mass Spectrometry
55 (2020): e4663, 10.1002/jms.4663.
</cite> [<a href="https://doi.org/10.1002/jms.4663" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8099046/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33241625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Mass%20Spectrometry&amp;title=Effect%20of%20MALDI%20Matrices%20on%20Lipid%20Analyses%20of%20Biological%20Tissues%20Using%20MALDI%E2%80%902%20Postionization%20Mass%20Spectrometry&amp;author=J.%20C.%20McMillen&amp;author=J.%20A.%20Fincher&amp;author=D.%20R.%20Klein&amp;author=J.%20M.%20Spraggins&amp;author=R.%20M.%20Caprioli&amp;volume=55&amp;publication_year=2020&amp;pages=e4663&amp;pmid=33241625&amp;doi=10.1002/jms.4663&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0050">
<span class="label">50.</span><cite>
Soltwisch J., Heijs B., Koch A., Vens‐Cappell S., Höhndorf J., and Dreisewerd K., “MALDI‐2 on a Trapped Ion Mobility Quadrupole Time‐of‐Flight Instrument for Rapid Mass Spectrometry Imaging and Ion Mobility Separation of Complex Lipid Profiles,” Analytical Chemistry
92 (2020): 8697–8703, 10.1021/acs.analchem.0c01747.
</cite> [<a href="https://doi.org/10.1021/acs.analchem.0c01747" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32449347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=MALDI%E2%80%902%20on%20a%20Trapped%20Ion%20Mobility%20Quadrupole%20Time%E2%80%90of%E2%80%90Flight%20Instrument%20for%20Rapid%20Mass%20Spectrometry%20Imaging%20and%20Ion%20Mobility%20Separation%20of%20Complex%20Lipid%20Profiles&amp;author=J.%20Soltwisch&amp;author=B.%20Heijs&amp;author=A.%20Koch&amp;author=S.%20Vens%E2%80%90Cappell&amp;author=J.%20H%C3%B6hndorf&amp;volume=92&amp;publication_year=2020&amp;pages=8697-8703&amp;pmid=32449347&amp;doi=10.1021/acs.analchem.0c01747&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0051">
<span class="label">51.</span><cite>
Bowman A. P., Bogie J. F. J., Hendriks J. J. A., et al., “Evaluation of Lipid Coverage and High Spatial Resolution MALDI‐Imaging Capabilities of Oversampling Combined With Laser Post‐Ionisation,” Analytical and Bioanalytical Chemistry
412 (2020): 2277–2289, 10.1007/s00216-019-02290-3.
</cite> [<a href="https://doi.org/10.1007/s00216-019-02290-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7118047/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31879798/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20and%20Bioanalytical%20Chemistry&amp;title=Evaluation%20of%20Lipid%20Coverage%20and%20High%20Spatial%20Resolution%20MALDI%E2%80%90Imaging%20Capabilities%20of%20Oversampling%20Combined%20With%20Laser%20Post%E2%80%90Ionisation&amp;author=A.%20P.%20Bowman&amp;author=J.%20F.%20J.%20Bogie&amp;author=J.%20J.%20A.%20Hendriks&amp;volume=412&amp;publication_year=2020&amp;pages=2277-2289&amp;pmid=31879798&amp;doi=10.1007/s00216-019-02290-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0052">
<span class="label">52.</span><cite>
Sasai K., Treekitkarnmongkol W., Kai K., Katayama H., and Sen S., “Functional Significance of Aurora Kinases–p53 Protein Family Interactions in Cancer,” Frontiers in Oncology
6 (2016): 1–12, 10.3389/fonc.2016.00247.
</cite> [<a href="https://doi.org/10.3389/fonc.2016.00247" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5122578/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27933271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Oncology&amp;title=Functional%20Significance%20of%20Aurora%20Kinases%E2%80%93p53%20Protein%20Family%20Interactions%20in%20Cancer&amp;author=K.%20Sasai&amp;author=W.%20Treekitkarnmongkol&amp;author=K.%20Kai&amp;author=H.%20Katayama&amp;author=S.%20Sen&amp;volume=6&amp;publication_year=2016&amp;pages=1-12&amp;pmid=27933271&amp;doi=10.3389/fonc.2016.00247&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0053">
<span class="label">53.</span><cite>
Carmena M., Wheelock M., Funabiki H., and Earnshaw W. C., “The Chromosomal Passenger Complex (CPC): From Easy Rider to the Godfather of Mitosis,” Nature Reviews. Molecular Cell Biology
13 (2012): 789–803, 10.1038/nrm3474.
</cite> [<a href="https://doi.org/10.1038/nrm3474" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3729939/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23175282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews.%20Molecular%20Cell%20Biology&amp;title=The%20Chromosomal%20Passenger%20Complex%20(CPC):%20From%20Easy%20Rider%20to%20the%20Godfather%20of%20Mitosis&amp;author=M.%20Carmena&amp;author=M.%20Wheelock&amp;author=H.%20Funabiki&amp;author=W.%20C.%20Earnshaw&amp;volume=13&amp;publication_year=2012&amp;pages=789-803&amp;pmid=23175282&amp;doi=10.1038/nrm3474&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0054">
<span class="label">54.</span><cite>
Mandal R. and Strebhardt K., “Plk1: Unexpected Roles in DNA Replication,” Cell Research
23 (2013): 1251–1253, 10.1038/cr.2013.130.
</cite> [<a href="https://doi.org/10.1038/cr.2013.130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3817549/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24042259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Research&amp;title=Plk1:%20Unexpected%20Roles%20in%20DNA%20Replication&amp;author=R.%20Mandal&amp;author=K.%20Strebhardt&amp;volume=23&amp;publication_year=2013&amp;pages=1251-1253&amp;pmid=24042259&amp;doi=10.1038/cr.2013.130&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0055">
<span class="label">55.</span><cite>
Papagiannakis A., Niebel B., Wit E. C., and Heinemann M., “Autonomous Metabolic Oscillations Robustly Gate the Early and Late Cell Cycle,” Molecular Cell
65 (2017): 285–295, 10.1016/j.molcel.2016.11.018.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2016.11.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27989441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cell&amp;title=Autonomous%20Metabolic%20Oscillations%20Robustly%20Gate%20the%20Early%20and%20Late%20Cell%20Cycle&amp;author=A.%20Papagiannakis&amp;author=B.%20Niebel&amp;author=E.%20C.%20Wit&amp;author=M.%20Heinemann&amp;volume=65&amp;publication_year=2017&amp;pages=285-295&amp;pmid=27989441&amp;doi=10.1016/j.molcel.2016.11.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0056">
<span class="label">56.</span><cite>
Martin‐Perez M., Urdiroz‐Urricelqui U., Bigas C., and Benitah S. A., “The Role of Lipids in Cancer Progression and Metastasis,” Cell Metabolism
34 (2022): 1675–1699, 10.1016/j.cmet.2022.09.023.
</cite> [<a href="https://doi.org/10.1016/j.cmet.2022.09.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36261043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Metabolism&amp;title=The%20Role%20of%20Lipids%20in%20Cancer%20Progression%20and%20Metastasis&amp;author=M.%20Martin%E2%80%90Perez&amp;author=U.%20Urdiroz%E2%80%90Urricelqui&amp;author=C.%20Bigas&amp;author=S.%20A.%20Benitah&amp;volume=34&amp;publication_year=2022&amp;pages=1675-1699&amp;pmid=36261043&amp;doi=10.1016/j.cmet.2022.09.023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0057">
<span class="label">57.</span><cite>
Furse S. and Shearman G. C., “Do Lipids Shape the Eukaryotic Cell Cycle?,” Biochimica et Biophysica Acta (BBA) ‐ Molecular and Cell Biology of Lipids
1863 (2018): 9–19, 10.1016/j.bbalip.2017.09.010.
</cite> [<a href="https://doi.org/10.1016/j.bbalip.2017.09.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28964796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20et%20Biophysica%20Acta%20(BBA)%20%E2%80%90%20Molecular%20and%20Cell%20Biology%20of%20Lipids&amp;title=Do%20Lipids%20Shape%20the%20Eukaryotic%20Cell%20Cycle?&amp;author=S.%20Furse&amp;author=G.%20C.%20Shearman&amp;volume=1863&amp;publication_year=2018&amp;pages=9-19&amp;pmid=28964796&amp;doi=10.1016/j.bbalip.2017.09.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0058">
<span class="label">58.</span><cite>
Hague C. V., Postle A. D., Attard G. S., and Dymond M. K., “Cell Cycle Dependent Changes in Membrane Stored Curvature Elastic Energy: Evidence From Lipidomic Studies,” Faraday Discussions
161 (2013): 481–497, 10.1039/C2FD20078C.
</cite> [<a href="https://doi.org/10.1039/C2FD20078C" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23805754/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Faraday%20Discussions&amp;title=Cell%20Cycle%20Dependent%20Changes%20in%20Membrane%20Stored%20Curvature%20Elastic%20Energy:%20Evidence%20From%20Lipidomic%20Studies&amp;author=C.%20V.%20Hague&amp;author=A.%20D.%20Postle&amp;author=G.%20S.%20Attard&amp;author=M.%20K.%20Dymond&amp;volume=161&amp;publication_year=2013&amp;pages=481-497&amp;pmid=23805754&amp;doi=10.1039/C2FD20078C&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0059">
<span class="label">59.</span><cite>
Atilla‐Gokcumen G. E., Muro E., Relat‐Goberna J., et al., “Dividing Cells Regulate Their Lipid Composition and Localization,” Cell
156 (2014): 428–439, 10.1016/j.cell.2013.12.015.
</cite> [<a href="https://doi.org/10.1016/j.cell.2013.12.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3909459/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24462247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Dividing%20Cells%20Regulate%20Their%20Lipid%20Composition%20and%20Localization&amp;author=G.%20E.%20Atilla%E2%80%90Gokcumen&amp;author=E.%20Muro&amp;author=J.%20Relat%E2%80%90Goberna&amp;volume=156&amp;publication_year=2014&amp;pages=428-439&amp;pmid=24462247&amp;doi=10.1016/j.cell.2013.12.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jms5137-bib-0060">
<span class="label">60.</span><cite>
Perry W. J., Patterson N. H., Prentice B. M., Neumann E. K., Caprioli R. M., and Spraggins J. M., “Uncovering Matrix Effects on Lipid Analyses in MALDI Imaging Mass Spectrometry Experiments,” Journal of Mass Spectrometry
55 (2020): e4491, 10.1002/jms.4491.
</cite> [<a href="https://doi.org/10.1002/jms.4491" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7383219/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31860760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Mass%20Spectrometry&amp;title=Uncovering%20Matrix%20Effects%20on%20Lipid%20Analyses%20in%20MALDI%20Imaging%20Mass%20Spectrometry%20Experiments&amp;author=W.%20J.%20Perry&amp;author=N.%20H.%20Patterson&amp;author=B.%20M.%20Prentice&amp;author=E.%20K.%20Neumann&amp;author=R.%20M.%20Caprioli&amp;volume=55&amp;publication_year=2020&amp;pages=e4491&amp;pmid=31860760&amp;doi=10.1002/jms.4491&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<p>
<strong>Figure S1</strong> IC<sub>50</sub> curve of HCT 116 spheroids incubated with onvansertib for 72 h.</p>
<p>
<strong>Table S1</strong> Spray parameters for MALDI‐MSI experiments with either DHB or CHCA.</p>
<p>
<strong>Figure S2</strong> Normalized MS1 TIC for log<sub>2</sub> transformed intensity values for LC–MS proteomics.</p>
<p>
<strong>Figure S3</strong> Hierarchical clustering of onvansertib proteomics data showing that 72 h with onvansertib exposure cluster entirely together.</p>
<p>
<strong>Figure S4</strong> Volcano plots of (A) 1 h, (B) 4 h, and (C) 6 h onvansertib exposure, showing only one differentially expressed protein among these entire time sets.</p>
<p>
<strong>Figure S5</strong> Venn Diagram of similar proteins between the 12‐, 24‐, 48‐, and 72‐h samples.</p>
<p>
<strong>Figure S6</strong> Gene set enrichment analysis of the differentially expressed proteins after 12 h of onvansertib exposure.</p>
<p>
<strong>Figure S7</strong> Gene set enrichment analysis of the differentially expressed proteins after 24 h of onvansertib exposure.</p>
<p>
<strong>Figure S8</strong> Gene set enrichment analysis of the differentially expressed proteins after 48 h of onvansertib exposure.</p>
<p>
<strong>Figure S9</strong> Gene set enrichment analysis of the differentially expressed proteins after 72 h of onvansertib exposure.</p>
<p>
<strong>Figure S10</strong> Ingenuity pathway analysis of statistically significant proteins among the 12‐, 24‐, 48, and 72‐h time points.</p>
<p>
<strong>Figure S11</strong> Two different lipids that are not highly present in the early time points but gradually increase over time with onvansertib exposure. (A) <em>m/z</em> 798.542 identified to be the [M + K<sup>+</sup>]<sup>+</sup> of PC(34:1). (B) <em>m/z</em> 770.513 identified to be the [M + K<sup>+</sup>]<sup>+</sup> of PC(32:1).</p>
<p>
<strong>Figure S12</strong>
<em>m/z</em> 369.351, identified as the [M + H<sup>+</sup>‐H<sub>2</sub>O]<sup>+</sup> of cholesterol, as observed with MALDI‐2 ionization.</p>
<p>
<strong>Figure S13</strong> MALDI‐MSMS of lipids found in the mass spectrometry images for validation of lipid identity. The above tandem MS spectra originate from the following precursors: (A) <em>m/z</em> 796.464, (B) <em>m/z</em> 798.542, and (C) <em>m/z</em> 770.513.</p>
<p>
<strong>Figure S14</strong> MALDI‐2 MSMS of lipids found in the mass spectrometry images for validation of lipid identity. The above tandem MS spectra originate from the precursor <em>m/z</em> 808.585.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11976698/bin/JMS-60-e5137-s004.docx" data-ga-action="click_feat_suppl" class="usa-link">JMS-60-e5137-s004.docx</a><sup> (5MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>
<strong>Data S2</strong> Supplemental Table 2</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11976698/bin/JMS-60-e5137-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">JMS-60-e5137-s001.xlsx</a><sup> (1.4MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>
<strong>Data S3</strong> Supplemental Table 2</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11976698/bin/JMS-60-e5137-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">JMS-60-e5137-s002.xlsx</a><sup> (12.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p"><p>
<strong>Data S4</strong> Supplemental Table 2</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/11976698/bin/JMS-60-e5137-s003.xlsx" data-ga-action="click_feat_suppl" class="usa-link">JMS-60-e5137-s003.xlsx</a><sup> (41.1KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Raw LC–MS proteomics files have been uploaded to the Pride repository under PXD054453.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Journal of Mass Spectrometry are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/jms.5137"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/JMS-60-e5137.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/11976698/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/11976698/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC11976698%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC11976698/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC11976698/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC11976698/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40197665/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC11976698/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40197665/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC11976698/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/11976698/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="4FDPDakP4G6HPgtMliKhxVd6oc7esAV58Cd53u7oMvKKjt0MiGWrCQyhSScvWRvz">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
