
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            EDNTOM: An Ensemble Learning and Weight Mechanism-Based Nanopore Methylation Detection Tool - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="FF2081DE89B735D31A81DE001E4B4B08.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsomega">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332607/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Omega">
<meta name="citation_title" content="EDNTOM: An Ensemble Learning and Weight Mechanism-Based Nanopore Methylation Detection Tool">
<meta name="citation_author" content="Ge Tian">
<meta name="citation_author_institution" content="School of Software, Shandong University, Jinan 250100, China">
<meta name="citation_author" content="Chenglin Yin">
<meta name="citation_author_institution" content="School of Software, Shandong University, Jinan 250100, China">
<meta name="citation_author" content="Jianbo Qiao">
<meta name="citation_author_institution" content="School of Software, Shandong University, Jinan 250100, China">
<meta name="citation_author" content="Ruheng Wang">
<meta name="citation_author_institution" content="School of Software, Shandong University, Jinan 250100, China">
<meta name="citation_author" content="Hua Shi">
<meta name="citation_author_institution" content="School of Optoelectronic and Communication Engineering, Xiamen University of Technology, Xiamen 361024, China">
<meta name="citation_author" content="Feifei Cui">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Hainan University, Haikou 570228, China">
<meta name="citation_author" content="Zilong Zhang">
<meta name="citation_author_institution" content="School of Computer Science and Technology, Hainan University, Haikou 570228, China">
<meta name="citation_author" content="Xinbo Jiang">
<meta name="citation_author_institution" content="School of Qilu Transportation, Shandong University, Jinan 250061, China">
<meta name="citation_author" content="Leyi Wei">
<meta name="citation_author_institution" content="Faculty of Applied Sciences, Macao Polytechnic University, Macao 999078, China">
<meta name="citation_author_institution" content="Joint SDU-NTU Centre for Artificial Intelligence Research (C-FAIR), Shandong University, Jinan 250101, China">
<meta name="citation_publication_date" content="2025 Jul 23">
<meta name="citation_volume" content="10">
<meta name="citation_issue" content="30">
<meta name="citation_firstpage" content="33031">
<meta name="citation_doi" content="10.1021/acsomega.5c01924">
<meta name="citation_pmid" content="40787313">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332607/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332607/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332607/pdf/ao5c01924.pdf">
<meta name="description" content="DNA methylation is an epigenetic modification that plays a crucial role in genome stability and cellular specialization, essential for maintaining normal cellular function and development, also a manifestation indicator of some diseases. Various ...">
<meta name="og:title" content="EDNTOM: An Ensemble Learning and Weight Mechanism-Based Nanopore Methylation Detection Tool">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="DNA methylation is an epigenetic modification that plays a crucial role in genome stability and cellular specialization, essential for maintaining normal cellular function and development, also a manifestation indicator of some diseases. Various ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332607/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12332607">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsomega.5c01924"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/ao5c01924.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332607%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12332607/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12332607/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332607/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsomega.png" alt="ACS Omega logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Omega" title="Link to ACS Omega" shape="default" href="https://pubs.acs.org/journal/acsodf" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Omega</button></div>. 2025 Jul 23;10(30):33031–33044. doi: <a href="https://doi.org/10.1021/acsomega.5c01924" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsomega.5c01924</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Omega%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Omega%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>EDNTOM: An Ensemble
Learning and Weight Mechanism-Based
Nanopore Methylation Detection Tool</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tian%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Ge Tian</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Ge Tian</span></h3>
<div class="p">
<sup>†</sup>
School
of Software, Shandong University, Jinan
250100, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tian%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Ge Tian</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yin%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Chenglin Yin</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Chenglin Yin</span></h3>
<div class="p">
<sup>†</sup>
School
of Software, Shandong University, Jinan
250100, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yin%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chenglin Yin</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qiao%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jianbo Qiao</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jianbo Qiao</span></h3>
<div class="p">
<sup>†</sup>
School
of Software, Shandong University, Jinan
250100, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Qiao%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jianbo Qiao</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Ruheng Wang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Ruheng Wang</span></h3>
<div class="p">
<sup>†</sup>
School
of Software, Shandong University, Jinan
250100, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ruheng Wang</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Hua Shi</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Hua Shi</span></h3>
<div class="p">
<sup>‡</sup>
School
of Optoelectronic and Communication Engineering, Xiamen University of Technology, Xiamen
361024, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hua Shi</span></a>
</div>
</div>
<sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cui%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Feifei Cui</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Feifei Cui</span></h3>
<div class="p">
<sup>§</sup>
School
of Computer Science and Technology, Hainan
University, Haikou
570228, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cui%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Feifei Cui</span></a>
</div>
</div>
<sup>§</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Zilong Zhang</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Zilong Zhang</span></h3>
<div class="p">
<sup>§</sup>
School
of Computer Science and Technology, Hainan
University, Haikou
570228, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zilong Zhang</span></a>
</div>
</div>
<sup>§</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Xinbo Jiang</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Xinbo Jiang</span></h3>
<div class="p">
<sup>∥</sup>
School
of Qilu Transportation, Shandong University, Jinan
250061, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xinbo Jiang</span></a>
</div>
</div>
<sup>∥,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wei%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Leyi Wei</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Leyi Wei</span></h3>
<div class="p">
<sup>⊥</sup>
Faculty
of Applied Sciences, Macao Polytechnic University, Macao
999078, China
</div>
<div class="p">
<sup>#</sup>
Joint SDU-NTU
Centre for Artificial Intelligence Research (C-FAIR), Shandong University, Jinan
250101, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wei%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Leyi Wei</span></a>
</div>
</div>
<sup>⊥,</sup><sup>#,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>†</sup>
School
of Software, Shandong University, Jinan
250100, China
</div>
<div id="aff2">
<sup>‡</sup>
School
of Optoelectronic and Communication Engineering, Xiamen University of Technology, Xiamen
361024, China
</div>
<div id="aff3">
<sup>§</sup>
School
of Computer Science and Technology, Hainan
University, Haikou
570228, China
</div>
<div id="aff4">
<sup>∥</sup>
School
of Qilu Transportation, Shandong University, Jinan
250061, China
</div>
<div id="aff5">
<sup>⊥</sup>
Faculty
of Applied Sciences, Macao Polytechnic University, Macao
999078, China
</div>
<div id="aff6">
<sup>#</sup>
Joint SDU-NTU
Centre for Artificial Intelligence Research (C-FAIR), Shandong University, Jinan
250101, China
</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
Email: <span>xinboj@gmail.com</span>.</p>
</div>
<div class="fn" id="cor2">
<sup>*</sup><p class="display-inline">
Email: <span>weileyi@sdu.edu.cn</span>.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 11; Accepted 2025 Jul 10; Revised 2025 Jun 19; Collection date 2025 Aug 5.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY-NC-ND 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12332607  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40787313/" class="usa-link">40787313</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>DNA methylation is
an epigenetic modification that plays a crucial
role in genome stability and cellular specialization, essential for
maintaining normal cellular function and development, also a manifestation
indicator of some diseases. Various tools have been proposed for methylation
detection, typically leveraging a third-generation sequencing technology
called nanopore sequencing, which provides more accurate DNA sequencing
data. However, existing tools have their own limitations and advantages
in terms of computational resources and information processing, without
achieving a good balance. In this situation, we developed EDNTOM (Ensemble
Deep Network Tool Of Methylation), a DNA methylation detection tool
based on deep learning technology. We employed ensemble learning techniques,
integrating predictions from multiple pretrained single models, and
introduced an attention weight mechanism to provide accurate and reliable
detection, reducing the consumption of computational resources. Results
demonstrate that EDNTOM outperforms individual models. Additionally,
in cross-species transfer experiments, EDNTOM exhibits strong transfer
learning capabilities. We hope this work can provide a more powerful
and reliable solution for methylation detection, contributing to the
fields of biological science and medicine. The project code is available
at <a href="https://github.com/ViceMusic/EDNTOM" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ViceMusic/EDNTOM</a>.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4484/12332607/9415858b6da1/ao5c01924_0016.jpg" loading="lazy" id="abs1" height="372" width="544" alt="graphic file with name ao5c01924_0016.jpg"></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4484/12332607/9415858b6da1/ao5c01924_0014.jpg" loading="lazy" id="tgr1" height="372" width="544" alt="graphic file with name ao5c01924_0014.jpg"></p></section><section id="sec1"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>DNA methylation is an
epigenetic modification that primarily occurs
on cytosine molecules in DNA molecules. This modification is achieved
by adding a methyl group to the fifth carbon atom of cytosine. DNA
methylation plays a crucial role in gene expression regulation, maintaining
genome stability, and cell differentiation
and development. Abnormal DNA methylation
is closely associated with the occurrence and development of various
diseases.<em>
,
</em> Luan et al. revealed
the relationship between Netrin-1 (NTN1) gene methylation and the
prognosis of Skin Cutaneous Melanoma (SKCM). By studying changes in
DNA methylation, we can gain insight into the mechanisms underlying
diseases.<em>
−

</em> Therefore, detecting DNA methylation behavior is
of great significance in the medical field for understanding disease
mechanisms, early diagnosis, treatment selection, and the formulation
of personalized medical plans.<em>
−

</em>
</p>
<p>As an epigenetic modification
on DNA, DNA methylation can be detected
using DNA sequencing data. The second-generation
sequencing methods include Illumina sequencing, which uses PCR technology to amplify DNA samples and relies
on fluorescent or chemically labeled bases for sequencing. These operations
limit the ability to analyze repetitive sequences and genomic structures.
Especially in low copy number samples, PCR amplification may lead
to distortion. And the second-generation
sequencing technology cannot achieve single-molecule sequencing. Fortunately,
we now have many methods and tools that can obtain labeling information
without damaging the molecules, such as the work of An et al. using Surface-enhanced Raman spectroscopy (SERS)
to detect molecular markers. It will not affect the molecule itself.
From this perspective, we have chosen the third-generation sequencing
technologies. In third-generation DNA
sequencing methods, we choose to use nanopore sequencing data, which
is a direct sequencing technique. Nanopore
sequencing directly detects nucleotide sequences by analyzing current
changes as DNA traverses the pore, bypassing chemical modification
and PCR amplification to preserve native DNA integrity while generating
high-quality data.</p>
<p>With the rise of deep learning, researchers
have begun to use information
methods for computational biology, such as the work of Li et al., which used deep learning techniques for N7-m7G
site prediction. In the medical field, we use methylation markers
to determine lymph node metastasis in gastric cancer.
</p>
<p>In recent years, more and more methylation detection
tools have
combined deep learning with nanopore sequencing, providing researchers
with precise sequence recognition and analysis functions.<em>
−


</em> The predominant tools for methylation detection leveraging deep
learning technology principally fall into three categories: those
utilizing convolutional neural networks (CNNs), those employing recurrent neural networks (RNNs), and those based on other deep learning architectures.</p>
<p>CNN-based deep learning tools can capture dependencies within a
limited range, extracting sequence features from the data while parallel
computing, reducing computational cost. Major works and tools based on CNN include DeepMP, DInoPORE. DeepMP is a DNA
methylation prediction tool that utilizes CNN to determine whether
the sequence is methylated. DInoPORE is a CNN-based model, distinguishing
RNA editing sites by nanopore sequencing. RNN-based deep learning
tools capture long-range dependencies through the learning and progression
of hidden states. Major works based on RNN and its variants include
DeepSignal and DeepMod. DeepSignal is a deep learning tool utilizing bidirectional
long short-term memory (LSTM) models
to capture sequence information. DeepMod, similar to DeepSignal, employs
LSTM for methylation site detection. Additionally, there are other
model architectures-based deep learning tools. MethBERT utilizes BERT for
DNA methylation state prediction. DeepMethyl is another methylation prediction software based on stacked denoising
autoencoders, used to predict the methylation status of DNA. In addition,
NanoCon is also a methylation detection
tool that utilizes transformer architecture and contrastive learning
techniques.</p>
<p>However, CNN and RNN may encounter issues with insufficient
long-term
memory and may lose dependencies over long distances. Tools based
on other models, such as BERT, show improvements in methylation prediction. Nevertheless, the computational cost of BERT
models will increase with the input length, leading to more expensive
training. In summary, each popular tool retains advantages and limitations
due to underlying architectures. Given this situation, we aim to find
a framework that integrates the strengths of mainstream tools.</p>
<p>We have observed the application of ensemble learning in this field.
Ensemble learning<em>
−

</em> is a machine learning method that combines multiple
learners to improve overall performance. In the field of bioinformatics,
the METEORE integrates the predictions
of multiple models for methylation prediction, demonstrating the universality
of ensemble learning in bioinformatics. In the work by Lasantha et
al., ensemble deep learning method is
observed to outperform single models too. In the field of biology,
the ensemble learning based model tool LightGBM has achieved the function of detecting genome assisted
breeding. In emotion recognition, Wang et al.’s work also utilized ensemble learning
techniques in emotion
recognition tasks. In the medical field, Kazemi and Mirroshandel proposed a kidney stone detection model using
ensemble learning.</p>
<p>Inspired by the aforementioned work, we decided
to adopt an ensemble
learning mechanism, integrating models. This choice aims to leverage
the strengths of different models and mitigate the impact of their
weaknesses.</p>
<p>By combining ensemble learning with attention mechanisms,
we have
developed an artificial intelligence tool called EDNTOM (Ensemble
Deep Network Tool Of Methylation), further improving the performance
of DNA methylation prediction. We have achieved promising prediction
and transfer effects on both human data sets and data sets of two
plant species ( and ). Compared with mainstream methylation
detection tools such as DeepSignal and DeepMod, EDNTOM’s prediction
results not only come from one model, but also can extract features
in different dominant dimensions based on CNN, BERT and RNN, avoiding
the preference of a single model. Moreover, we have designed a new
attention mechanism that replaces the voting operation in traditional
ensemble learning with linear projection, resulting in excellent learning
ability. We fully open source the code and encourage other developers
to integrate models according to their own needs in integrated learning.</p>
<p>EDNTOM typically achieves lower costs and more persistent convergence
effects, providing great assistance for studying bioinformatics with
complex and uneven data.</p>
<p>In the future, we expect this research
to assist exploration in
this field and gain more disease-related research or practical applications
related to medical devices. For example, using EDNTOM to promote research
and analysis of multimodal data related to methylation in the field
of disease associations. We will continue to update this tool, and
promote its implementation, such as with Qian et al.’s work
on biocompatible materials and sensor integration.
</p>
<p>In addition, in the integrated learning module, we
have open-sourced
our data processing methods. We look forward to interested researchers
exploring the boundaries of algorithm tools based on their computing
resources.</p></section><section id="sec2"><h2 class="pmc_sec_title">2. Methods</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">2.1. Data Set</h3>
<p>Nanopore
sequencing data
of <em>Arabidopsis</em> (, or , GCF_000001735.4_TAIR10.1) and <em>rice</em> (, or , IRGSP-1.0) compiled by Ni et al. were
used in this study. Because the study of the 5 mC locus is often associated
with human diseases with certain specific traits in mammals, the human
data set was added to these data sets as a representative 5 mC methylation
data set for mammalian DNA. NA12878 is a widely used reference sample
of the human genome and is part of the International Human Genome
Project (HGP). NA12878 has become one of the most widely used reference
samples in genomic research due to the high quality of its genomic
data and the wide range of available data sets and has become an important
reference point in genomic research and genomics applications. 20,000
reads from each species were randomly selected during the construction
of the data set, and each Read is the electrical signal sequence information
on a segment of nucleic acid sequence obtained from a DNA sample by
nanopore sequencing technology.</p>
<p>In the data sets of the three
species we used, the human data set NA12878 is a slightly imbalanced
data set with a data ratio of 2:1. On the data sets of the other two
species, the ratio of methylated to nonmethylated data is 1:1. In
addition, NA12878 contains a total of approximately 3.5 million pieces
of data. The rice and <em>Arabidopsis</em> data sets contain
approximately 300,000 pieces of data.</p>
<p>Through the data processing
method provided by Ni. The methylation
data we obtained were processed into the
same structure, including DNA sequence information, nanopore sequencing
current mean, electrical signal standard deviation, and signal length.
Each data represents a <em>k</em>-mer base sequence. In this
article, we set <em>k</em> to 13, which is an experience-based
choice. In some mainstream methylation frameworks, such as DeepSignal,
a <em>k</em>-mer with a length of 13 is used, so we have decided
to adopt this design.</p>
<p>For subsequent training, we divided the
data set into a 9:1 ratio,
with 90% being the training set and 10% being the validation set.
We first define the data set as</p>
<table class="disp-formula p" id="uneq1"><tr><td class="formula"><math id="d1e473" display="block" indentalign="left" indentshift="1.0em" indentshiftfirst="0em" linebreak="auto" overflow="linebreak"><mi mathvariant="script">D</mi><mo>=</mo><msubsup><mrow><mo stretchy="false">{</mo><mo stretchy="false">(</mo><msup><mi>x</mi><mi>i</mi></msup><mo>,</mo><msup><mi>y</mi><mi>i</mi></msup><mo stretchy="false">)</mo><mo stretchy="false">}</mo></mrow><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mi>N</mi></msubsup><mo linebreak="goodbreak" lspace="0em" rspace="2em">⁣</mo><msup><mi>y</mi><mi>i</mi></msup><mo>∈</mo><mrow><mo stretchy="false">{</mo><mn>0</mn><mo>,</mo><mn>1</mn><mo stretchy="false">}</mo></mrow></math></td></tr></table>
<p>extract
subsets based on methylation labels
(0 or 1), calculate segmentation points <span xmlns:mml="http://www.w3.org/1998/Math/MathML">
<math id="d1e517" display="inline" overflow="linebreak"><msub><mi>τ</mi><mi mathvariant="script">c</mi></msub></math>
</span>, divide them according to the index position
of the segmentation point</p>
<table class="disp-formula p" id="uneq2"><tr><td class="formula"><math id="d1e526" display="block" indentalign="left" indentshift="1.0em" indentshiftfirst="0em" linebreak="auto" overflow="linebreak"><mrow><mo stretchy="true">{</mo><mstyle lineleading=".25ex" rowspacing="2ex"><mtable columnalign="left" displaystyle="true"><mtr><mtd><msub><mi mathvariant="script">D</mi><mi mathvariant="script">c</mi></msub><mo>=</mo><mrow><mo stretchy="false">{</mo><msup><mi>x</mi><mi>i</mi></msup><mo stretchy="false">|</mo><msup><mi>y</mi><mi>i</mi></msup><mo>=</mo><mi mathvariant="script">c</mi><mo stretchy="false">}</mo></mrow><mo linebreak="goodbreak" lspace="0em" rspace="1em">⁣</mo><mi mathvariant="script">c</mi><mo>∈</mo><mrow><mo stretchy="false">{</mo><mn>0</mn><mo>,</mo><mn>1</mn><mo stretchy="false">}</mo></mrow></mtd></mtr><mtr><mtd><msub><mi>τ</mi><mi mathvariant="script">c</mi></msub><mo>=</mo><mi>min</mi><mrow><mo stretchy="false">(</mo><mo stretchy="false">⌊</mo><mrow><mn>0.9</mn><mrow><mo stretchy="false">|</mo><msub><mi mathvariant="script">D</mi><mi mathvariant="script">c</mi></msub><mo stretchy="false">|</mo></mrow></mrow><mo stretchy="false">⌋</mo><mo>,</mo><mrow><mo stretchy="false">|</mo><msub><mi mathvariant="script">D</mi><mi mathvariant="script">c</mi></msub><mo stretchy="false">|</mo></mrow><mo stretchy="false">−</mo><mn>1</mn><mo stretchy="false">)</mo></mrow></mtd></mtr><mtr><mtd><msub><mi mathvariant="script">D</mi><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">n</mi></mrow></msub><mo>←</mo><msub><mi mathvariant="script">D</mi><mi mathvariant="script">c</mi></msub><mrow><mo stretchy="false">[</mo><mo>:</mo><msub><mi>τ</mi><mi mathvariant="script">c</mi></msub><mo stretchy="false">]</mo></mrow><mo>,</mo><msub><mrow><mo linebreak="goodbreak" lspace="0em" rspace="1em">⁣</mo><mi mathvariant="script">D</mi></mrow><mrow><mi mathvariant="normal">v</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">l</mi></mrow></msub><mo>←</mo><msub><mi mathvariant="script">D</mi><mi mathvariant="script">c</mi></msub><mrow><mo stretchy="false">[</mo><msub><mi>τ</mi><mi mathvariant="script">c</mi></msub><mo>:</mo><mo stretchy="false">]</mo></mrow></mtd></mtr></mtable></mstyle></mrow></math></td></tr></table>
<p>and finally randomly shuffle
to ensure the
rationality of the data. This division method is relatively simple,
but convenient and rigorous.</p></section><section id="sec2.2"><h3 class="pmc_sec_title">2.2. Structure of Model</h3>
<p>As illustrated
in <a href="#fig1" class="usa-link">Figure 
</a>, EDNTOM
is a deep learning neural network model that performs nanopore methylation
detection through ensemble learning and weighting mechanisms. According
to the sequence of data flow, we divide the entire model into four
main parts: the data preprocessing module, the symmetric encoding
module, the attention weight calculation module, and the classifier
module.</p>
<figure class="fig xbox font-sm" id="fig1"><h4 class="obj_head">1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332607_ao5c01924_0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4484/12332607/196be0481a58/ao5c01924_0001.jpg" loading="lazy" id="gr1" height="652" width="694" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Architecture of EDNTOM. After processing the input data through
the model, the output of the single model is then fed into the attention
mechanism. Finally, the attention mechanism produces the ultimate
prediction results.</p></figcaption></figure><section id="sec2.2.1"><h4 class="pmc_sec_title">2.2.1. Data
Preprocessing Module</h4>
<p>In the
data preprocessing module, we transform the acquired data from the
data set into tensor forms suitable for input into the model. The
data in our data set follows a uniform format, consisting of DNA base
sequences (referred to as “seq”), nanopore data (including
means, stds, and lengths), and methylation labels (“label”).
As for the nanopore data, we extract the “means”, “stds”
and “lens” corresponding to each base, and arrange the
nanodata into a matrix of size <em>K</em>*3. The specific
image operations are illustrated in <a href="#fig2" class="usa-link">Figure 
</a>.</p>
<figure class="fig xbox font-sm" id="fig2"><h5 class="obj_head">2.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332607_ao5c01924_0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4484/12332607/6dd139e7b8ed/ao5c01924_0002.jpg" loading="lazy" id="gr2" height="140" width="667" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>We extract and parse the input data, converting
it into the tensor
format required by our model tool.</p></figcaption></figure></section><section id="sec2.2.2"><h4 class="pmc_sec_title">2.2.2. Symmetric Encoding Module</h4>
<p>We call
this module symmetric encoding because of its symmetric structure
(as shown in <a href="#fig3" class="usa-link">Figure 
</a>). In this module, the data are input into three single models (CNN,
RNN and BERT) to obtain their respective
prediction results <em>ο</em>
<sub>
<em>i</em>
</sub>.</p>
<table class="disp-formula p" id="uneq3"><tr><td class="formula"><math id="d1e726" display="block" indentalign="left" indentshift="1.0em" indentshiftfirst="0em" linebreak="auto" overflow="linebreak"><msub><mi>ο</mi><mi>i</mi></msub><mo>=</mo><msub><mrow><mi mathvariant="normal">M</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">d</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">l</mi></mrow><mi>i</mi></msub><mrow><mo stretchy="false">(</mo><mi>x</mi><mo stretchy="false">)</mo></mrow><mo linebreak="goodbreak" lspace="0em" rspace="2em">⁣</mo><mi>i</mi><mo>∈</mo><mrow><mo stretchy="false">{</mo><mrow><mi mathvariant="normal">C</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">N</mi></mrow><mo>,</mo><mrow><mi mathvariant="normal">B</mi><mi mathvariant="normal">E</mi><mi mathvariant="normal">R</mi><mi mathvariant="normal">T</mi></mrow><mo>,</mo><mrow><mi mathvariant="normal">R</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">N</mi></mrow><mo stretchy="false">}</mo></mrow></math></td></tr></table>
<figure class="fig xbox font-sm" id="fig3"><h5 class="obj_head">3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332607_ao5c01924_0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4484/12332607/0a2e899b9c8a/ao5c01924_0003.jpg" loading="lazy" id="gr3" height="179" width="667" alt="3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Workflow of symmetric encoding module. Based
on the prediction
results obtained from each model, they are further processed through
a classifier to obtain data of the same scale.</p></figcaption></figure><p>After obtaining the original output of a single
model, the data
needs to be shaped to adapt to the downstream attention weight layer.
Each model comes with a structure and a processing method that is
completely consistent in calculation. By linear projection, all outputs
of the model are projected to the same size.</p>
<table class="disp-formula p" id="uneq4"><tr><td class="formula"><math id="d1e794" display="block" overflow="linebreak"><msub><mi>f</mi><mi>i</mi></msub><mo>=</mo><msub><mrow><mi mathvariant="normal">C</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">f</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">r</mi></mrow><mi>i</mi></msub><mrow><mo stretchy="false">(</mo><msub><mi>x</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow><mo>=</mo><msub><mi>W</mi><mi>i</mi></msub><mrow><mi mathvariant="normal">R</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">L</mi><mi mathvariant="normal">U</mi></mrow><mrow><mo stretchy="false">(</mo><msub><mi>ο</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow><mo>+</mo><msub><mi>b</mi><mi>i</mi></msub></math></td></tr></table>
<p>Due to the identical
prediction and classifier processing methods
in each model, the data flow presents a symmetrical shape. Therefore,
we refer to this module as the symmetric encoding module. This structure
ensures consistency and data independence in downstream tasks after
the outputs of all models are projected.</p></section><section id="sec2.2.3"><h4 class="pmc_sec_title">2.2.3. Attention
Weight Calculation Module</h4>
<p>In the attention weight module
(as shown in <a href="#fig4" class="usa-link">Figure 
</a>), we used a neural network-based attention
mechanism to extract from the output results <em>f</em> of
the models. In all experiments in this article, we trained these three
models using a human data set, which is similar to pretraining. (We
also provide the weights of these basic models trained and saved on
different data sets in open-source code to meet users’ pursuit
of higher accuracy).</p>
<figure class="fig xbox font-sm" id="fig4"><h5 class="obj_head">4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332607_ao5c01924_0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4484/12332607/8dc5493e9938/ao5c01924_0004.jpg" loading="lazy" id="gr4" height="142" width="667" alt="4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Workflow of attention weight calculation module, we concatenate
the results of model outputs and train a linear layer based on the
concatenated result. This linear layer learns individual weights for
each model output. Then, we multiply the weight results by the model
outputs to obtain the weighted features. Finally, classification is
achieved through the classifier layer to obtain the prediction result.</p></figcaption></figure><p>We will concatenate <em>f</em>
<sub>
<em>i</em>
</sub>(<em>i</em> ∈ {RNN,BERT,CNN})­in one
dimension, and
then use linear layers and activation functions for linear projection.
In this process, we train two weight matrices, projection matrix <em>W</em>
<sub>2</sub>, and concatenation matrix <em>W</em>
<sub>1</sub>, as well as their corresponding biases. Finally, through
calculation, we obtain the integrated weight α. The mathematical
expression of its operation is as follows</p>
<table class="disp-formula p" id="uneq5"><tr><td class="formula"><math id="d1e896" display="block" overflow="linebreak"><mi>α</mi><mo>=</mo><mrow><mi mathvariant="normal">s</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">f</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">x</mi></mrow><mrow><mo stretchy="false">(</mo><msub><mi>W</mi><mn>2</mn></msub><mo>·</mo><mi>tanh</mi><mrow><mo stretchy="false">(</mo><msub><mi>W</mi><mn>1</mn></msub><mrow><mo stretchy="false">[</mo><msub><mi>f</mi><mn>1</mn></msub><mrow><mo stretchy="false">|</mo><msub><mi>f</mi><mn>2</mn></msub><mo stretchy="false">|</mo></mrow><msub><mi>f</mi><mn>3</mn></msub><mo stretchy="false">]</mo></mrow><mo>+</mo><msub><mi>b</mi><mn>1</mn></msub><mo stretchy="false">)</mo></mrow><mo>+</mo><msub><mi>b</mi><mn>2</mn></msub><mo stretchy="false">)</mo></mrow></math></td></tr></table>
<p>In addition,
we combine the original input <em>f</em>
<sub>
<em>i</em>
</sub> with integrated weight and
fuse the calculated results. The mathematical operation expression
is as follows</p>
<table class="disp-formula p" id="uneq6"><tr><td class="formula"><math id="d1e969" display="block" overflow="linebreak"><mi>y</mi><mo>=</mo><mrow><mo stretchy="false">[</mo><msub><mi>a</mi><mn>1</mn></msub><msub><mi>f</mi><mn>1</mn></msub><mo stretchy="false">∥</mo><msub><mi>a</mi><mn>2</mn></msub><msub><mi>f</mi><mn>2</mn></msub><mo stretchy="false">∥</mo><msub><mi>a</mi><mn>3</mn></msub><msub><mi>f</mi><mn>3</mn></msub><mo stretchy="false">]</mo></mrow></math></td></tr></table>
<p>Based
on this attention weighting mechanism, we can dynamically
assign weights to each individual input data based on the prediction
results of each model. Added the problem of insufficient flexibility
of trained single weights for diverse data in traditional voting.</p></section><section id="sec2.2.4"><h4 class="pmc_sec_title">2.2.4. Classifier Module</h4>
<p>The final classifier
module is responsible for classifying input data and mapping it to
a smaller dimension to obtain the final prediction output. The architecture
of this layer is shown in <a href="#fig2" class="usa-link">Figure 
</a>. Through processing in this layer, the model integrates
and refines information from all preceding layers to generate the
final prediction output. The design of this fully connected layer
aims to enable the model to capture key features in the input data
optimally and produce highly accurate final predictions.</p></section></section><section id="sec2.3"><h3 class="pmc_sec_title">2.3. Metrics</h3>
<p>We selected the following
six model evaluation metrics: accuracy (ACC), precision, recall, F1
score, area under the precision–recall curve (AUPRC), and area
under the receiver operating characteristic curve (AUROC).<em>
−





</em>
</p>
<p>These evaluation metrics are commonly used performance metrics
in deep learning to assess the accuracy and efficacy of classification
models.
</p></section><section id="sec2.4"><h3 class="pmc_sec_title">2.4. Setup</h3>
<p>On the server, we configured
four NVIDIA GeForce RTX 3090 GPUs and utilized the PyTorch Lightning
framework. During the training process, we employed the Adam optimizer
to optimize model parameters and combined it with a learning rate
decay strategy for better control over the learning process. To ensure
fair evaluation, we adopted a strict data set partitioning scheme,
splitting each data set into training, and validation sets in a 9:1
ratio. This partitioning scheme aims to achieve more stable and reliable
computational results.</p>
<p>During the training process of the model,
we set the batch to 100, the initial learning rate of the Adam optimizer
to 0.001, and used binary cross loss to calculate the loss value.
Unless otherwise specified, the default training epochs are 25.</p>
<p>In the ensemble module, our individual models (CNN, RNN, and BERT)
have already been pretrained, and each has demonstrated promising
performance. During the training of our overall model, the parameters
of individual models are frozen and do not participate in training
iterations. This separation training approach effectively reduces
training pressure and difficulty, yielding better results with fewer
computational rounds. Additionally, it mitigates the risk of overfitting
that could arise from excessive parameters and layer depths.</p></section></section><section id="sec3"><h2 class="pmc_sec_title">3. Experiments</h2>
<section id="sec3.1"><h3 class="pmc_sec_title">3.1. Comparing with Single Models</h3>
<p>We present
the actual performance results of EDNTOM and compare them with other
models, including CNN, RNN, and BERT models. In this experiment, we
compared the performance differences between EDNTOM and the other
three models. We conducted this work on data sets of three species:
humans, , and . For any of the data sets, we follow
the settings in the setup section and divide the data set into two
parts: training and validation. Each model was trained and tested
on the same data set.</p>
<p>We compared the performance of models
trained and validated on the same data set. Through the experiment,
we compared EDNTOM with three other model tools under the same training
and validation conditions on the same data set. It can be observed
that compared to RNN, BERT, and CNN, EDNTOM shows improvement in accuracy,
especially compared to RNN. BERT performs similarly to EDNTOM, and
during the training and testing process on the <em>Arabidopsis</em> data set, BERT even outperforms EDNTOM in terms of recall. However,
we attribute this to the selection of AUROC as the metric for comparison
and saving when selecting the saved model. In conclusion, the test
proves that EDNTOM integrated by a single model is not inferior to
any single model in six indicators including ACC.</p></section><section id="sec3.2"><h3 class="pmc_sec_title">3.2. Comparative Experiment with Popular Tools</h3>
<p>In order
to verify the performance of our model in multiple aspects,
we compared EDNTOM with two popular tools in the same field, DeepSignal
and DeepMod. In this experiment, we compared multiple indicators including
loss, Auroc, etc. And the same hyperparameters were tested on the <em>Arabidopsis</em> data set, and the hyperparameters and data set
partitioning methods have been described in detail in the <a href="#sec2.1" class="usa-link">Data Set</a> and <a href="#sec2.4" class="usa-link">Setup</a> sections.</p>
<p>As shown in <a href="#fig6" class="usa-link">Figure 
</a>, we conducted over 20 rounds of training
on the <em>Arabidopsis</em> data set, and EDNTOM achieved
stable convergence during the training period (final loss = 0.11497),
which was 30.5% and 51.6% lower than Deepsignal (loss = 0.16528) and
DeepMod (loss = 0.23709), respectively. By comparing the standard
deviation of the training curve (σ = 0.0098 vs 0.0235/0.0257),
the gradient fluctuation amplitude of EDNTOM is only 41.7%–38.1%
of that of the comparison model. The AUROC peak of EDNTOM reached
0.99232, which is 0.9–1.8 percentage points higher than the
comparison model. Moreover, the accuracy fluctuation of the validation
set in the later stage of training (epoch &gt;15) is ≤0.3%,
which
is significantly better than the 1.2–2.8% of the comparison
model. In summary, we believe that EDNTOM not only has superior learning
abilities, but can also achieve better accuracy in a shorter period
of time. Furthermore, in over 20 training sessions, EDNTOM maintained
a steady improvement trend, while DeepSignal failed to fully converge
and exhibited oscillations in the later stages DeepMod experienced
a gradient explosion in the midterm, which seriously affected its
performance. From this, it can be seen that EDNTOM can adapt to more
complex data and classification tasks, reducing the time consumption
of users when adjusting hyperparameters. At the same time, this experiment
shows that EDNTOM may still not reach its performance limit, and has
considerable practical value.</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332607_ao5c01924_0006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4484/12332607/e9a1e165ca58/ao5c01924_0006.jpg" loading="lazy" id="gr6" height="595" width="694" alt="6"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>We recorded
the performance changes of the models on the test set
with increasing training epochs on six metrics: loss, AUROC, Acc,
precision, recall, and F1. Among them, EDNTOM performs better than
the other two models in all aspects, demonstrating the potential of
ensemble learning technology.</p></figcaption></figure><p>Meanwhile, as shown in <a href="#tbl2" class="usa-link">Table 
</a>, in terms of time
consumption, by implementing
a strategy of freezing parameters, EDNTOM effectively reduces the
pressure on training parameters while preventing gradient explosion
caused by excessively deep neural networks EDNTOM’s total parameters
are only 35.4% of DeepSignal’s, but its performance metrics
still lead by 1.2%. In terms of inference speed, although DeepMod
has achieved the fastest speed, its unit parameter inference time
reaches 0.951, reflecting its complex structure and high computational
density. The computational density of EDNTOM and DeepSignal is relatively
low. At the same time, in terms of training efficiency, EDNTOM’s
training time and unit parameter training speed far exceed DeepMod.
Although slightly inferior to DeepSignal, EDNTOM only reduces the
number of parameters by 64.6% by increasing training time by 87% while
ensuring performance. Its overall performance can be fully comparable
to DeepSignal.</p>
<section class="tw xbox font-sm" id="tbl2"><h4 class="obj_head">2. Under the Same Hardware
and Hyperparameter
Conditions, We Conducted a Statistical Analysis of the Model’s
Parameter Count, Inference Time, etc.<sup></sup>
</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">model</th>
<th align="center" colspan="1" rowspan="1">number of
model parameters (M)</th>
<th align="center" colspan="1" rowspan="1">single data inference
time (s)</th>
<th align="center" colspan="1" rowspan="1">unit parameter time (min/M)</th>
<th align="center" colspan="1" rowspan="1">average time
per round (epoch/h)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>EDNTOM</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>1.77</strong>
<strong>(1.1 + 0.67)</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.41</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>2.43</strong>
</td>
<td align="center" colspan="1" rowspan="1">
<strong>4.3</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DeepSignal</td>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="center" colspan="1" rowspan="1">0.2</td>
<td align="center" colspan="1" rowspan="1">0.46</td>
<td align="center" colspan="1" rowspan="1">2.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DeepMod</td>
<td align="left" colspan="1" rowspan="1">0.17</td>
<td align="center" colspan="1" rowspan="1">0.16</td>
<td align="center" colspan="1" rowspan="1">28.57</td>
<td align="center" colspan="1" rowspan="1">4.8</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t2fn1">
<sup>a</sup><p class="display-inline">The unit parameter time is set as
the time required for updating every million parameters.</p>
</div></div></section><p>In summary, by comparing its performance
and accuracy with two
other widely used tools in the field of methylation detection, EDNTOM
not only leads in accuracy, but also demonstrates extremely high stability
and good parameter efficiency during training. This is beneficial
for us to extend EDNTOM and ensemble learning to more complex and
time-consuming tasks. Although the current inference training time
is slightly slower, our future work will focus on simplifying tensor
calculations in the prediction process and improving the computational
speed of EDNTOM.</p></section><section id="sec3.3"><h3 class="pmc_sec_title">3.3. Ablation and Exploration
Experiment</h3>
<p>In the experiment, we mainly focused on two aspects:
1. Conduct ablation
experiments on a single model to check the impact of a single model
on the whole two. 2. Our code and subsequent experiments have confirmed
the structure described in the previous section, but there must be
some replaceable methods locally. We conduct experiments and demonstrate
them here, such as not freezing parameters, convolution weights, etc.
We used a human data set in this experiment, and the hyperparameters
remained unchanged.</p>
<p>As shown in the following <a href="#fig7" class="usa-link">Figure 
</a>, we compare them in six different
ways: the original version of <strong>EDNTOM</strong>, unfreeze the parameters
in the integration module (refer as <strong>unfrozen</strong>), cover
different single models (refer as <strong>mask_RNN</strong>, <strong>mask_BERT</strong>, and <strong>mask_CNN</strong>), and replace the original attention
weight module (refer as <strong>Conv</strong>) with a convolution module,
and another gate attention mechanism (refer as <strong>Gate</strong>).</p>
<figure class="fig xbox font-sm" id="fig7"><h4 class="obj_head">7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332607_ao5c01924_0007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4484/12332607/a592c600360e/ao5c01924_0007.jpg" loading="lazy" id="gr7" height="178" width="694" alt="7"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Results
of ablation implementation were compared under different
evaluation indicators.</p></figcaption></figure><p>As mentioned earlier,
in this work, the CNN, RNN, and BERT models
used for ensemble learning have been trained, and their parameters
were frozen during ensemble learning. Here, we conducted an ablation
experiment by unfreezing the parameters of the three single models
and allowing them to participate in consistent training methods, enabling
their internal parameters to continue learning data features. The
results, as shown in <a href="#fig7" class="usa-link">Figure 
</a> and <a href="#tbl3" class="usa-link">Table 
</a>, demonstrate that allowing single models to continue training leads
to a decrease in performance. We believe that the reason for this
phenomenon is the increase in the number of parameters that need to
be trained, which leads to an increased training burden. By the way,
in <a href="#fig6" class="usa-link">Figure 
</a>, it can
be seen that mask CNN performs well on indicators ACC, F1 score, and
recall. This indicates that the model has improved by removing CNN
from these evaluation indicators, as CNN’s prediction results
may be slightly worse than other models in the entire EDNTOM.</p>
<section class="tw xbox font-sm" id="tbl3"><h4 class="obj_head">3. Comparison between the Original Model
and the Ablation Experimental Conditions, and We use AUROC as the
Main Evaluation Indicator.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1"> </th>
<th align="center" colspan="1" rowspan="1">EDNTOM</th>
<th align="center" colspan="1" rowspan="1">unfrozen</th>
<th align="center" colspan="1" rowspan="1">mask_RNN</th>
<th align="center" colspan="1" rowspan="1">mask_BERT</th>
<th align="center" colspan="1" rowspan="1">mask_CNN</th>
<th align="center" colspan="1" rowspan="1">conv</th>
<th align="center" colspan="1" rowspan="1">gate</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">ACC</td>
<td align="center" colspan="1" rowspan="1">0.89755</td>
<td align="center" colspan="1" rowspan="1">0.89258</td>
<td align="center" colspan="1" rowspan="1">0.89603</td>
<td align="center" colspan="1" rowspan="1">0.88116</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.8978</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.89658</td>
<td align="center" colspan="1" rowspan="1">0.89724</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AUPRC</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.97467</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.97233</td>
<td align="center" colspan="1" rowspan="1">0.97425</td>
<td align="center" colspan="1" rowspan="1">0.9671</td>
<td align="center" colspan="1" rowspan="1">0.97446</td>
<td align="center" colspan="1" rowspan="1">0.97446</td>
<td align="center" colspan="1" rowspan="1">0.97444</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AUROC</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.95504</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.95104</td>
<td align="center" colspan="1" rowspan="1">0.95444</td>
<td align="center" colspan="1" rowspan="1">0.94301</td>
<td align="center" colspan="1" rowspan="1">0.95492</td>
<td align="center" colspan="1" rowspan="1">0.95458</td>
<td align="center" colspan="1" rowspan="1">0.95468</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">F1 score</td>
<td align="center" colspan="1" rowspan="1">0.92056</td>
<td align="center" colspan="1" rowspan="1">0.91675</td>
<td align="center" colspan="1" rowspan="1">0.91911</td>
<td align="center" colspan="1" rowspan="1">0.90826</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.92081</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.91981</td>
<td align="center" colspan="1" rowspan="1">0.92032</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">precision</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.92664</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.92271</td>
<td align="center" colspan="1" rowspan="1">0.92595</td>
<td align="center" colspan="1" rowspan="1">0.9106</td>
<td align="center" colspan="1" rowspan="1">0.92449</td>
<td align="center" colspan="1" rowspan="1">0.92555</td>
<td align="center" colspan="1" rowspan="1">0.92686</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">recall</td>
<td align="center" colspan="1" rowspan="1">0.91465</td>
<td align="center" colspan="1" rowspan="1">0.91097</td>
<td align="center" colspan="1" rowspan="1">0.9125</td>
<td align="center" colspan="1" rowspan="1">0.90606</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.91725</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.91425</td>
<td align="center" colspan="1" rowspan="1">0.91399</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>In addition, we also
designed another experiment: in order to explore
the importance of different models in the overall structure, we covered
the single model in the symmetric encoding module of EDNTOM respectively,
and called them “<strong>mask_RNN</strong>”, “<strong>mask_BERT</strong>”, “<strong>mask_CNN</strong>” according
to the different models covered. And conducted the same training method,
the results obtained are shown in <a href="#fig7" class="usa-link">Figure 
</a> and <a href="#tbl3" class="usa-link">Table 
</a>. In this result, it can be seen that there was no
improvement in the results after ablation of our main evaluation metric
AUROC. On the three evaluation indicators of ACC, F1 score, and recall,
the results of mask CNN reached the level of EDNTOM. It is worth noting
that there are serious shortcomings in the evaluation results of mask-BERT,
which proves that RNN plays a more critical role in EDNTOM. This result
is also consistent with our findings in comparative experiments (as
shown in <a href="#fig5" class="usa-link">Figure 
</a> and <a href="#tbl1" class="usa-link">Table 
</a>): BERT performs better
than RNN and CNN. This demonstrates the logic and accuracy of our
experiment.</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332607_ao5c01924_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4484/12332607/58223f7e7df2/ao5c01924_0005.jpg" loading="lazy" id="gr5" height="312" width="694" alt="5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Showcased the performance of EDNTOM, RNN, BERT,
and CNN trained
on the NA12878 human data set on three species data sets in (a–c).
(a) As shown in figure (a), the performance of EDNTOM and other models
trained on human data sets. (b) As shown in figure (b), the performance
of EDNTOM trained on the data
set. (c) As shown in figure (c), the performance of EDNTOM and other
models trained on the Thalina data set.</p></figcaption></figure><section class="tw xbox font-sm" id="tbl1"><h4 class="obj_head">1. Performance of EDNTOM, RNN, BERT,
and CNN Trained on Different Datasets.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">data set</th>
<th align="center" colspan="1" rowspan="1">metric</th>
<th align="center" colspan="1" rowspan="1">EDNTOM</th>
<th align="center" colspan="1" rowspan="1">RNN</th>
<th align="center" colspan="1" rowspan="1">BERT</th>
<th align="center" colspan="1" rowspan="1">CNN</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">human</td>
<td align="left" colspan="1" rowspan="1">ACC</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.89755</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.86791</td>
<td align="center" colspan="1" rowspan="1">0.88783</td>
<td align="center" colspan="1" rowspan="1">0.88582</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">AUPRC</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.97467</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.95966</td>
<td align="center" colspan="1" rowspan="1">0.96826</td>
<td align="center" colspan="1" rowspan="1">0.96886</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">AUROC</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.95504</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.93144</td>
<td align="center" colspan="1" rowspan="1">0.94672</td>
<td align="center" colspan="1" rowspan="1">0.94569</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">F1 score</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.92056</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.89864</td>
<td align="center" colspan="1" rowspan="1">0.91225</td>
<td align="center" colspan="1" rowspan="1">0.91155</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">precision</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.92664</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.89477</td>
<td align="center" colspan="1" rowspan="1">0.92553</td>
<td align="center" colspan="1" rowspan="1">0.91569</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">recall</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.91465</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.90262</td>
<td align="center" colspan="1" rowspan="1">0.8994</td>
<td align="center" colspan="1" rowspan="1">0.90751</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">O. sativa</td>
<td align="left" colspan="1" rowspan="1">ACC</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.95897</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.92298</td>
<td align="center" colspan="1" rowspan="1">0.94567</td>
<td align="center" colspan="1" rowspan="1">0.94751</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">AUPRC</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.99267</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.97655</td>
<td align="center" colspan="1" rowspan="1">0.98672</td>
<td align="center" colspan="1" rowspan="1">0.98775</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">AUROC</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.99253</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.97724</td>
<td align="center" colspan="1" rowspan="1">0.98643</td>
<td align="center" colspan="1" rowspan="1">0.98754</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">F1 score</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.95876</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.92418</td>
<td align="center" colspan="1" rowspan="1">0.94541</td>
<td align="center" colspan="1" rowspan="1">0.94717</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">precision</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.96458</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.91133</td>
<td align="center" colspan="1" rowspan="1">0.95039</td>
<td align="center" colspan="1" rowspan="1">0.9518</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">recall</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.95308</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.93755</td>
<td align="center" colspan="1" rowspan="1">0.94059</td>
<td align="center" colspan="1" rowspan="1">0.94268</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">A. thaliana</td>
<td align="left" colspan="1" rowspan="1">ACC</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.95147</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.93179</td>
<td align="center" colspan="1" rowspan="1">0.94172</td>
<td align="center" colspan="1" rowspan="1">0.93753</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">AUPRC</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.98878</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.96566</td>
<td align="center" colspan="1" rowspan="1">0.9781</td>
<td align="center" colspan="1" rowspan="1">0.97667</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">AUROC</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.98953</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.97287</td>
<td align="center" colspan="1" rowspan="1">0.98159</td>
<td align="center" colspan="1" rowspan="1">0.98003</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">F1 score</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.95202</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.93361</td>
<td align="center" colspan="1" rowspan="1">0.94331</td>
<td align="center" colspan="1" rowspan="1">0.93905</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">precision</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.9413</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.91417</td>
<td align="center" colspan="1" rowspan="1">0.92108</td>
<td align="center" colspan="1" rowspan="1">0.92573</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">recall</td>
<td align="center" colspan="1" rowspan="1">0.96309</td>
<td align="center" colspan="1" rowspan="1">0.95404</td>
<td align="center" colspan="1" rowspan="1">
<strong>0.96673</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.95284</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>We concatenate the models along the channel dimension
to obtain
a tensor, where dim is the length of the output features of each model,
and b is the batch size. By training a convolutional kernel and performing
convolutional operations, we obtain features, thereby integrating
the output results of the three models.</p>
<p>By training under the
same conditions as the main method, we obtained
models with performance similar to EDNTOM, as shown in <a href="#fig8" class="usa-link">Figure 
</a>. The results were comparable
to our original attention weight module, possibly because both methods
produced three weight values, albeit obtained differently. The attention
weight module trains a neural network block to calculate weights based
on inputs, allowing for flexibility with different input features.
In contrast, the convolutional module learns convolutional kernels
similar to weights based on inputs. We believe this extension is an
effective data aggregation strategy given the current data scale.
We anticipate that on larger data sets, the performance of convolutional
kernels may slightly degrade because they cannot flexibly adjust weight
values based on input features.</p>
<figure class="fig xbox font-sm" id="fig8"><h4 class="obj_head">8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332607_ao5c01924_0008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4484/12332607/a3b9db3bdb91/ao5c01924_0008.jpg" loading="lazy" id="gr8" height="150" width="667" alt="8"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Workflow of new module that can replace
the attention weight module,
using convolutional neural networks to train convolutional kernels,
achieving the effect of integrating predictions from multiple models.</p></figcaption></figure><p>In addition, we also used the gate attention model
shown in <a href="#fig9" class="usa-link">Figure 
</a> to
replace our original
attention mechanism, which helped us verify the feasibility of other
attention mechanisms in this experiment. After experimental verification,
it has been proven that using this gate attention model does not bring
breakthrough results.</p>
<figure class="fig xbox font-sm" id="fig9"><h4 class="obj_head">9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332607_ao5c01924_0009.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4484/12332607/f3c19d656cd3/ao5c01924_0009.jpg" loading="lazy" id="gr9" height="146" width="667" alt="9"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Workflow of new module that can replace the attention
weight module,
we use the product of two gate values and calculate the weight, which
provides two attention values.</p></figcaption></figure><p>In summary, in this part of the experiment, we
not only conducted
ablation experiments, but also experimented with some replaceable
content. Under the current experimental setup, no significant differences
were observed. We encourage researchers using EDNTOM to adjust these
modules according to their different needs.</p></section><section id="sec3.4"><h3 class="pmc_sec_title">3.4. Comparison
with Soft Voting</h3>
<p>In the
ablation experiment above, in order to demonstrate the effectiveness
of the ensemble learning part, we chose to remove the voting step
and use simple concatenation for ablation comparison experiments.
In this experiment, in order to demonstrate the effectiveness of the
attention mechanism designed in the ensemble learning part, we used
the commonly used soft voting in ensemble learning as the experimental
object.</p>
<p>Soft voting is an algorithm widely used in ensemble
learning mechanisms, which adds up the classification probabilities
obtained by each learner and resets the threshold to integrate the
results of all learners and make judgments. In our model, the comprehensive
scoring formula for soft voting is</p>
<table class="disp-formula p" id="uneq7"><tr><td class="formula"><math id="d1e1834" display="block" indentalign="left" indentshift="1.0em" indentshiftfirst="0em" linebreak="auto" overflow="linebreak"><mrow><mi mathvariant="normal">S</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi></mrow><mo>=</mo><msub><mrow><mi>P</mi><mrow><mo stretchy="false">(</mo><mi>y</mi><mrow><mrow><mo>=</mo><mn>1</mn></mrow><mo stretchy="false">|</mo></mrow><mi>x</mi><mo stretchy="false">)</mo></mrow></mrow><mrow><mi mathvariant="normal">R</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">N</mi></mrow></msub><mo>+</mo><msub><mrow><mi>P</mi><mrow><mo stretchy="false">(</mo><mi>y</mi><mrow><mrow><mo>=</mo><mn>1</mn></mrow><mo stretchy="false">|</mo></mrow><mi>x</mi><mo stretchy="false">)</mo></mrow></mrow><mrow><mi mathvariant="normal">B</mi><mi mathvariant="normal">E</mi><mi mathvariant="normal">R</mi><mi mathvariant="normal">T</mi></mrow></msub><mo>+</mo><msub><mrow><mi>P</mi><mrow><mo stretchy="false">(</mo><mi>y</mi><mrow><mrow><mo>=</mo><mn>1</mn></mrow><mo stretchy="false">|</mo></mrow><mi>x</mi><mo stretchy="false">)</mo></mrow></mrow><mrow><mi mathvariant="normal">C</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">N</mi></mrow></msub></math></td></tr></table>
<p>And, since we used three models, we
set the weights to 1.5
and
used this to construct the decision function</p>
<table class="disp-formula p" id="uneq8"><tr><td class="formula"><math id="d1e1925" display="block" overflow="linebreak"><mrow><mi mathvariant="normal">m</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">k</mi></mrow><mo>=</mo><mfrac><mn>1</mn><mrow><mn>1</mn><mo>+</mo><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mrow><mo stretchy="false">(</mo><mi>S</mi><mo>−</mo><mn>1.5</mn><mo stretchy="false">)</mo></mrow><mo>/</mo><mn>0.1</mn></mrow></msup></mrow></mfrac></math></td></tr></table>
<p>The final label is determined by the
decision function</p>
<table class="disp-formula p" id="uneq9"><tr><td class="formula"><math id="d1e1961" display="block" overflow="linebreak"><mrow><mi mathvariant="normal">f</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">l</mi></mrow><mtext>_</mtext><mrow><mi mathvariant="normal">l</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">b</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">s</mi></mrow><mo>=</mo><mrow><mo stretchy="false">[</mo><mn>1</mn><mo>−</mo><mrow><mi mathvariant="normal">m</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">k</mi></mrow><mo>,</mo><mrow><mi mathvariant="normal">m</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">k</mi></mrow><mo stretchy="false">]</mo></mrow><mo>∈</mo><msup><mi mathvariant="double-struck">R</mi><mn>2</mn></msup></math></td></tr></table>
<p>Through this method, the experimental
results we obtained on the <em>Arabidopsis</em> data set are
shown in <a href="#fig10" class="usa-link">Figure 
</a>.</p>
<figure class="fig xbox font-sm" id="fig10"><h4 class="obj_head">10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332607_ao5c01924_0010.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4484/12332607/21a6d912d7a3/ao5c01924_0010.jpg" loading="lazy" id="gr10" height="261" width="694" alt="10"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>We compared the differences between our attention mechanism and
soft voting on six indicators, including loss and recorded the changes
in training time consumption.</p></figcaption></figure><p>Through experimental comparison, it can be concluded
that the attention
weight mechanism designed in EDNTOM has better convergence speed than
traditional ensemble learning methods such as soft voting. The speed
of loss reduction and accuracy improvement in attention weight mechanism
is significantly higher than that of soft voting, and in the later
stage, while maintaining performance growth, the training time of
attention weight mechanism is significantly lower than that of soft
voting.</p>
<p>By analyzing the trend of curve changes, we can find
that the attention
weighting mechanism can analyze features more deeply and provide reasonable
weight assignments. While soft voting has already converged to its
own limit, EDNTOM still maintains a good performance improvement trend.
We encourage open source workers to explore the limits of this mechanism
by increasing training volume, adjusting hyperparameters, and other
methods within their own computing power.</p></section><section id="sec3.5"><h3 class="pmc_sec_title">3.5. Transfer
Experiment</h3>
<p>To evaluate the
cross-data set transferability of EDNTOM, we conducted experiments
on three species: human, ,
and . The model was trained
on one species and tested on the other two, resulting in six cross-species
transfer settings. These experiments highlight EDNTOM’s ability
to generalize across diverse biological data sets, demonstrating its
potential in transfer learning.
</p>
<p>In the transfer learning experiments, we prepared two types of data
sets: the pretraining data set and the transfer data set. After training
the model on one data set, we fine-tuned it on another, following
the same training procedure and hyperparameter settings as described
in the <a href="#sec2.4" class="usa-link">Setup</a> section. Each of the three
species data sets served both as a pretraining and a transfer data
set, resulting in a total of nine pretraining–transfer data
set pairs through combinatorial permutation. During training, we monitored
multiple evaluation metrics and recorded the best-performing weights.
The final results were visualized using a heatmap to illustrate the
model’s transferability across species.</p>
<p>Verify the generalization
ability of the EDNTOM model through three
sets of cross species migration experiments. As shown in <a href="#fig11" class="usa-link">Figure 
</a> and <a href="#tbl4" class="usa-link">Table 
</a>, when the model was trained
on human data and transferred to the rice () and <em>Arabidopsis</em> () test sets, its performance showed a moderate level (AUROC: 0.917–0.937).
It is worth noting that reverse migration (from plants to humans)
showed similar performance degradation, such as a decrease in ACC
of 0.81892 for the rice training model on human data, indicating significant
differences in methylation patterns between animals and plants.</p>
<figure class="fig xbox font-sm" id="fig11"><h4 class="obj_head">11.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332607_ao5c01924_0011.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4484/12332607/d0d27968f6c0/ao5c01924_0011.jpg" loading="lazy" id="gr11" height="472" width="694" alt="11"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance
of EDNTOM transfers on the different data sets. We
trained EDNTOM on a single data set of one of the three (human, and ), and then conducted transfer validation on the other two data sets.
“Train” represents the training of EDNTOM on a certain
data set, and “transfer” represents the validation of
EDNTOM on which data set after completing the training. We conducted
three sets of experiments like this.</p></figcaption></figure><section class="tw xbox font-sm" id="tbl4"><h4 class="obj_head">4. Performance of Different Migration
Models on Different Datasets.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">train</th>
<th align="center" colspan="1" rowspan="1">transfer</th>
<th align="center" colspan="1" rowspan="1">ACC</th>
<th align="center" colspan="1" rowspan="1">AUPRC</th>
<th align="center" colspan="1" rowspan="1">AUROC</th>
<th align="center" colspan="1" rowspan="1">F1 score</th>
<th align="center" colspan="1" rowspan="1">precision</th>
<th align="center" colspan="1" rowspan="1">recall</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">human</td>
<td align="left" colspan="1" rowspan="1">O. sativa</td>
<td align="center" colspan="1" rowspan="1">0.84228</td>
<td align="center" colspan="1" rowspan="1">0.92584</td>
<td align="center" colspan="1" rowspan="1">0.91755</td>
<td align="center" colspan="1" rowspan="1">0.83502</td>
<td align="center" colspan="1" rowspan="1">0.86837</td>
<td align="center" colspan="1" rowspan="1">0.80673</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">A. thaliana</td>
<td align="center" colspan="1" rowspan="1">0.86163</td>
<td align="center" colspan="1" rowspan="1">0.9434</td>
<td align="center" colspan="1" rowspan="1">0.93737</td>
<td align="center" colspan="1" rowspan="1">0.86218</td>
<td align="center" colspan="1" rowspan="1">0.8551</td>
<td align="center" colspan="1" rowspan="1">0.87183</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">O. sativa</td>
<td align="left" colspan="1" rowspan="1">human</td>
<td align="center" colspan="1" rowspan="1">0.81892</td>
<td align="center" colspan="1" rowspan="1">0.93619</td>
<td align="center" colspan="1" rowspan="1">0.88711</td>
<td align="center" colspan="1" rowspan="1">0.86487</td>
<td align="center" colspan="1" rowspan="1">0.83379</td>
<td align="center" colspan="1" rowspan="1">0.90024</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">A. thaliana</td>
<td align="center" colspan="1" rowspan="1">0.94003</td>
<td align="center" colspan="1" rowspan="1">0.98679</td>
<td align="center" colspan="1" rowspan="1">0.98674</td>
<td align="center" colspan="1" rowspan="1">0.93838</td>
<td align="center" colspan="1" rowspan="1">0.95861</td>
<td align="center" colspan="1" rowspan="1">0.92014</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">A. thaliana</td>
<td align="left" colspan="1" rowspan="1">human</td>
<td align="center" colspan="1" rowspan="1">0.74079</td>
<td align="center" colspan="1" rowspan="1">0.91689</td>
<td align="center" colspan="1" rowspan="1">0.86496</td>
<td align="center" colspan="1" rowspan="1">0.82869</td>
<td align="center" colspan="1" rowspan="1">0.72295</td>
<td align="center" colspan="1" rowspan="1">0.97320</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">O. sativa</td>
<td align="center" colspan="1" rowspan="1">0.90025</td>
<td align="center" colspan="1" rowspan="1">0.96743</td>
<td align="center" colspan="1" rowspan="1">0.96808</td>
<td align="center" colspan="1" rowspan="1">0.90288</td>
<td align="center" colspan="1" rowspan="1">0.87224</td>
<td align="center" colspan="1" rowspan="1">0.93758</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>In the migration within the
plant domain, the model exhibits excellent
adaptability. When trained on rice and transferred to <em>Arabidopsis</em>, AUPRC reached 0.98679 (±0.002) and F1 score increased to 0.93838,
significantly better than the results of cross domain transfer. This
pattern also holds true in the <em>Arabidopsis</em> rice migration
(AUROC: 0.96808), confirming that the methylation characteristics
between species with similar evolutionary distances are highly conserved.</p>
<p>This experiment demonstrates for the first time that EDNTOM exhibits
excellent migration mobility between species with similar evolutionary
distances, providing a new computational tool for cross species epigenetic
research. Subsequent work will integrate more species data to establish
a quantitative relationship between evolutionary distance and model
performance.</p></section><section id="sec3.6"><h3 class="pmc_sec_title">3.6. Interpretability Experiment</h3>
<p>To make
the experimental results more reasonable and intuitive, we designed
the following experiment: we aggregated the features of three models
into one feature sequence and then reduced its dimensionality to two-dimensional
space using UMAP. Subsequently, we differentiated
between methylated and unmethylated samples based on the corresponding
labels and plotted them on a two-dimensional space. Throughout the
interpretative operation, we selected 1000 samples from the human
data set. To enhance the comparison and demonstrate the superiority
of our model, we performed the same feature extraction and analysis
on other individual models as well.</p>
<p>As shown in the <a href="#fig12" class="usa-link">Figure 
</a>, the data points
form two distinct clusters on the graph, indicating that our model
has successfully differentiated between methylated and unmethylated
data features in the output. This geometric representation provides
a more intuitive illustration of the effectiveness of our model. Through
such dimensionality reduction plots, we achieve a more intuitive display
of results than the raw data. In terms of feature classification performance,
EDNTOM demonstrates better classification results compared to other
models by integrating predictions from a single model. We attribute
this improvement to the reasonable configuration of our attention
weights, which incorporate the advantages of different models.</p>
<figure class="fig xbox font-sm" id="fig12"><h4 class="obj_head">12.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332607_ao5c01924_0012.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4484/12332607/cdbf4b05c4dd/ao5c01924_0012.jpg" loading="lazy" id="gr12" height="580" width="700" alt="12"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig12/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>As shown
in the figure, panels (a)–(d) display the UMAP-reduced
outputs of RNN, BERT, CNN, and our model (EDNTOM), respectively, with
methylated and unmethylated data labeled in green and orange. Among
them, EDNTOM exhibits the clearest boundary between the two classes,
highlighting its superior ability to distinguish methylation status
compared to the other models.</p></figcaption></figure></section><section id="sec3.7"><h3 class="pmc_sec_title">3.7. Performance Testing on Imbalanced Data Sets</h3>
<p>In the above experiments, we have fully demonstrated that EDNTOM
not only has performance comparable to mainstream tools. It also has
good resistance to overfitting. However, our experiments mainly focused
on balanced data sets. Therefore, in this experiment, we extracted
a new data set from the existing <em>Arabidopsis</em> data
set with a data size of 30,000 and set the methylation ratio to 5:1.
And conduct experiments on indicators such as loss. And by sliding
the variance, setting each window length to 5 (to get the change of
variance in a short-range), calculate the trend of variance change
to display its variation.</p>
<table class="disp-formula p" id="uneq10"><tr><td class="formula"><math id="d1e2300" display="block" overflow="linebreak"><msub><mrow><mi mathvariant="normal">V</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">r</mi></mrow><mi>t</mi></msub><mo>=</mo><mfrac><mn>1</mn><mn>5</mn></mfrac><munderover><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mi>t</mi></mrow><mrow><mi>t</mi><mo>+</mo><mn>4</mn></mrow></munderover><msup><mrow><mo stretchy="false">(</mo><msub><mi>x</mi><mi>t</mi></msub><mo>−</mo><msub><mi>x̅</mi><mi>i</mi></msub><mo stretchy="false">)</mo></mrow><mn>2</mn></msup></math></td></tr></table>
<p>In addition,
we set a dynamic threshold to
determine the degree of variance change.</p>
<table class="disp-formula p" id="uneq11"><tr><td class="formula"><math id="d1e2342" display="block" overflow="linebreak"><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">h</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">h</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">d</mi></mrow><mo>=</mo><mn>2</mn><mo>×</mo><mrow><mi mathvariant="normal">m</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">d</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi></mrow><mrow><mo stretchy="false">(</mo><msub><mrow><mi mathvariant="normal">V</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">r</mi></mrow><mn>1</mn></msub><mo>,</mo><mo>...</mo><mo>,</mo><msub><mrow><mi mathvariant="normal">V</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">r</mi></mrow><mrow><mi>n</mi><mo>−</mo><mn>4</mn></mrow></msub><mo stretchy="false">)</mo></mrow></math></td></tr></table>
<p>When the variance exceeds the dynamic
threshold</p>
<table class="disp-formula p" id="uneq12"><tr><td class="formula"><math id="d1e2412" display="block" overflow="linebreak"><msub><mrow><mi mathvariant="normal">V</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">r</mi></mrow><mi>t</mi></msub><mo>&gt;</mo><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">h</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">h</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">d</mi></mrow></math></td></tr></table>
<p>Then the position will be
marked in gray, indicating a significant
change in the data.</p>
<p>The experimental results show that the EDNTOM
model exhibits good
robustness under imbalanced data conditions: in the early stages of
training. (As shown in the <a href="#fig13" class="usa-link">Figure 
</a>), it quickly adapts to the data distribution through
high gradient updates (loss curve slope &gt;60°), although accompanied
by high variance fluctuations (dense gray blocks), it quickly enters
a stable optimization stage within 5 epochs, and the fluctuation amplitude
of key indicators is controlled within ±0.005. Although isolated
variance exceeded the standard in the middle and later stages, the
original indicators did not show significant deviation, indicating
that the model has self-stabilization ability. This characteristic
makes EDNTOM particularly suitable for epigenetic research scenarios
where there are differences in data distribution between species.</p>
<figure class="fig xbox font-sm" id="fig13"><h4 class="obj_head">13.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332607_ao5c01924_0013.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/4484/12332607/140efd4c93b5/ao5c01924_0013.jpg" loading="lazy" id="gr13" height="189" width="694" alt="13"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig13/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>We have
analyzed the changes in four indicators, including loss.
The first row of images shows the original data changes of the indicators,
while the second row represents the changes in sliding variance.</p></figcaption></figure></section></section><section id="sec4"><h2 class="pmc_sec_title">4. Results and Discussion</h2>
<p>In this study,
we proposed EDNTOM, a nanopore-based methylation
prediction model combining ensemble learning and attention mechanisms.
Compared with models like DeepSignal and DeepMod, EDNTOM demonstrated
faster convergence and better generalization, while using a moderate
number of parameters. Additionally, compared to traditional soft-voting
methods, it showed improved robustness against overfitting. Although
we did not explore its performance limits due to computational constraints,
the code is fully open-sourced to encourage further optimization and
customization for diverse tasks.</p></section><section id="sec5"><h2 class="pmc_sec_title">5. Conclusion</h2>
<p>In this study, we presented
EDNTOM, a deep learning tool based
on ensemble learning and attention mechanisms, designed to accurately
predict methylation patterns using nanopore sequencing data. What
sets this tool apart is its integration of multiple models and its
clever model fusion strategy, which achieves performance compared
to individual models, enabling efficient methylation prediction. This
ensemble learning approach not only enhances overall performance but
also, through a series of ablation experiments, we further demonstrated
that the current model state has reached a considerable level of suitability.
The method proposed in this study has made certain progress in this
field, and we hope that these explorations can provide new research
ideas for the development of genomics and bioinformatics..</p></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We extend our gratitude to the Natural
Science Foundation
of China (Nos. 62322112, 62372392, and 62272399) and Science and Technology
Development Fund of Macao (Nos. 0133/2024/RIB2) for their generous
support.</p></section><section id="app1" class="app"><h2 class="pmc_sec_title">Appendix A</h2>
<section id="sec6"><h3 class="pmc_sec_title">Formulas</h3>
<p>These
metrics can be calculated using the
following formulas.</p>
<table class="disp-formula p" id="uneq13"><tr><td class="formula"><math id="d1e2473" display="block" overflow="linebreak"><mrow><mo stretchy="true">{</mo><mstyle lineleading=".25ex" rowspacing="2ex"><mtable columnalign="left" displaystyle="true"><mtr><mtd><mrow><mi mathvariant="normal">A</mi><mi mathvariant="normal">C</mi><mi mathvariant="normal">C</mi></mrow><mo>=</mo><mfrac><mrow><mo stretchy="false">(</mo><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">P</mi></mrow><mo>+</mo><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">N</mi></mrow><mo stretchy="false">)</mo></mrow><mrow><mo stretchy="false">(</mo><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">P</mi></mrow><mo>+</mo><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">N</mi></mrow><mo>+</mo><mrow><mi mathvariant="normal">F</mi><mi mathvariant="normal">P</mi></mrow><mo>+</mo><mrow><mi mathvariant="normal">F</mi><mi mathvariant="normal">N</mi></mrow><mo stretchy="false">)</mo></mrow></mfrac></mtd></mtr><mtr><mtd><mi mathvariant="normal">F</mi><mn>1</mn><mspace width="0.25em"></mspace><mrow><mi mathvariant="normal">s</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi></mrow><mo>=</mo><mfrac><mrow><mn>2</mn><mo>*</mo><mrow><mi mathvariant="normal">P</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">n</mi></mrow><mo>*</mo><mrow><mi mathvariant="normal">R</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">l</mi></mrow></mrow><mrow><mrow><mi mathvariant="normal">P</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">n</mi></mrow><mo>+</mo><mrow><mi mathvariant="normal">R</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">l</mi></mrow></mrow></mfrac></mtd></mtr><mtr><mtd><mrow><mi mathvariant="normal">P</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">n</mi></mrow><mo>=</mo><mfrac><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">P</mi></mrow><mrow><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">P</mi></mrow><mo>+</mo><mrow><mi mathvariant="normal">F</mi><mi mathvariant="normal">P</mi></mrow></mrow></mfrac></mtd></mtr><mtr><mtd><mrow><mi mathvariant="normal">R</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">l</mi></mrow><mo>=</mo><mfrac><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">P</mi></mrow><mrow><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">P</mi></mrow><mo>+</mo><mrow><mi mathvariant="normal">F</mi><mi mathvariant="normal">N</mi></mrow></mrow></mfrac></mtd></mtr><mtr><mtd><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">P</mi><mi mathvariant="normal">R</mi></mrow><mo>=</mo><mfrac><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">P</mi></mrow><mrow><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">P</mi></mrow><mo>+</mo><mrow><mi mathvariant="normal">F</mi><mi mathvariant="normal">N</mi></mrow></mrow></mfrac></mtd></mtr><mtr><mtd><mrow><mi mathvariant="normal">F</mi><mi mathvariant="normal">P</mi><mi mathvariant="normal">R</mi></mrow><mo>=</mo><mfrac><mrow><mi mathvariant="normal">F</mi><mi mathvariant="normal">P</mi></mrow><mrow><mrow><mi mathvariant="normal">F</mi><mi mathvariant="normal">P</mi></mrow><mo>+</mo><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">N</mi></mrow></mrow></mfrac></mtd></mtr></mtable></mstyle></mrow></math></td></tr></table>
<p>where TP is true positive, the number
of positive
samples correctly predicted; TN is true negative, the number of negative
samples correctly predicted; FP is false positive, the number of negative
samples incorrectly predicted as positive samples; and FN denotes
false negative, the number of positive samples incorrectly classified
as negative samples.</p></section></section><section id="notes2"><p>The reference
genome of was obtained
from NCBI under the version number GCF_000001735.4_TAIR10.1. The reference genome and gene annotation of were obtained from EnsemblPlants under
the version number IRGSP-1.0. DNA nanopore
read data for NA12878 were accessed and downloaded from the European
Nucleotide Archive (ENA) via access number PRJEB23027, and gene annotation
information was obtained from the ENCODE project via access number
ENCFF835NTC. All machine learning code and trained weight files can
be obtained from the GitHub link <a href="https://github.com/ViceMusic/EDNTOM" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ViceMusic/EDNTOM</a>.</p></section><section id="notes1"><p>The authors
declare no competing financial interest.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>Brabson J. P., Leesang T., Mohammad S., Cimmino L.. Epigenetic regulation
of genomic stability by vitamin C. Front. Genet. 2021;12:675780. doi: 10.3389/fgene.2021.675780.</cite> [<a href="https://doi.org/10.3389/fgene.2021.675780" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8129186/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34017357/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Genet.&amp;title=Epigenetic%20regulation%20of%20genomic%20stability%20by%20vitamin%20C&amp;author=J.%20P.%20Brabson&amp;author=T.%20Leesang&amp;author=S.%20Mohammad&amp;author=L.%20Cimmino&amp;volume=12&amp;publication_year=2021&amp;pages=675780&amp;pmid=34017357&amp;doi=10.3389/fgene.2021.675780&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>Cedar H., Bergman Y.. Linking DNA methylation and histone
modification: patterns
and paradigms. Nat. Rev. Genet. 2009;10(5):295–304. doi: 10.1038/nrg2540.</cite> [<a href="https://doi.org/10.1038/nrg2540" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19308066/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&amp;title=Linking%20DNA%20methylation%20and%20histone%20modification:%20patterns%20and%20paradigms&amp;author=H.%20Cedar&amp;author=Y.%20Bergman&amp;volume=10&amp;issue=5&amp;publication_year=2009&amp;pages=295-304&amp;pmid=19308066&amp;doi=10.1038/nrg2540&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Krolevets M., Cate V. t., Prochaska J. H., Schulz A., Rapp S., Tenzer S., Andrade-Navarro M. A., Horvath S., Niehrs C., Wild P. S.. DNA methylation
and cardiovascular disease in humans:
a systematic review and database of known CpG methylation sites. Clin. Epigenet. 2023;15(1):56. doi: 10.1186/s13148-023-01468-y.</cite> [<a href="https://doi.org/10.1186/s13148-023-01468-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10061871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36991458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Epigenet.&amp;title=DNA%20methylation%20and%20cardiovascular%20disease%20in%20humans:%20a%20systematic%20review%20and%20database%20of%20known%20CpG%20methylation%20sites&amp;author=M.%20Krolevets&amp;author=V.%20t.%20Cate&amp;author=J.%20H.%20Prochaska&amp;author=A.%20Schulz&amp;author=S.%20Rapp&amp;volume=15&amp;issue=1&amp;publication_year=2023&amp;pages=56&amp;pmid=36991458&amp;doi=10.1186/s13148-023-01468-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>Tang W., Wan S., Yang Z., Teschendorff A. E., Zou Q.. Tumor origin detection
with tissue-specific miRNA and DNA methylation markers. Bioinformatics. 2018;34(3):398–406. doi: 10.1093/bioinformatics/btx622.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx622" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29028927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Tumor%20origin%20detection%20with%20tissue-specific%20miRNA%20and%20DNA%20methylation%20markers&amp;author=W.%20Tang&amp;author=S.%20Wan&amp;author=Z.%20Yang&amp;author=A.%20E.%20Teschendorff&amp;author=Q.%20Zou&amp;volume=34&amp;issue=3&amp;publication_year=2018&amp;pages=398-406&amp;pmid=29028927&amp;doi=10.1093/bioinformatics/btx622&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Luan F., Cui Y., Huang R., Yang Z., Qiao S.. Comprehensive pan-cancer
analysis reveals NTN1 as an immune infiltrate risk factor and its
potential prognostic value in SKCM. Sci. Rep. 2025;15(1):3223. doi: 10.1038/s41598-025-85444-x.</cite> [<a href="https://doi.org/10.1038/s41598-025-85444-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11762998/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39863609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Comprehensive%20pan-cancer%20analysis%20reveals%20NTN1%20as%20an%20immune%20infiltrate%20risk%20factor%20and%20its%20potential%20prognostic%20value%20in%20SKCM&amp;author=F.%20Luan&amp;author=Y.%20Cui&amp;author=R.%20Huang&amp;author=Z.%20Yang&amp;author=S.%20Qiao&amp;volume=15&amp;issue=1&amp;publication_year=2025&amp;pages=3223&amp;pmid=39863609&amp;doi=10.1038/s41598-025-85444-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Sharma S., Kelly T. K., Jones P. A.. Epigenetics
in cancer. Carcinogenesis. 2010;31(1):27–36. doi: 10.1093/carcin/bgp220.</cite> [<a href="https://doi.org/10.1093/carcin/bgp220" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2802667/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19752007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Epigenetics%20in%20cancer&amp;author=S.%20Sharma&amp;author=T.%20K.%20Kelly&amp;author=P.%20A.%20Jones&amp;volume=31&amp;issue=1&amp;publication_year=2010&amp;pages=27-36&amp;pmid=19752007&amp;doi=10.1093/carcin/bgp220&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Jin J., Yu Y., Wang R., Zeng X., Pang C., Jiang Y., Li Z., Dai Y., Su R., Zou Q.. et al. iDNA-ABF:
multi-scale deep biological language learning model for the interpretable
prediction of DNA methylations. Genome Biol. 2022;23(1):219. doi: 10.1186/s13059-022-02780-1.</cite> [<a href="https://doi.org/10.1186/s13059-022-02780-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9575223/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36253864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&amp;title=iDNA-ABF:%20multi-scale%20deep%20biological%20language%20learning%20model%20for%20the%20interpretable%20prediction%20of%20DNA%20methylations&amp;author=J.%20Jin&amp;author=Y.%20Yu&amp;author=R.%20Wang&amp;author=X.%20Zeng&amp;author=C.%20Pang&amp;volume=23&amp;issue=1&amp;publication_year=2022&amp;pages=219&amp;pmid=36253864&amp;doi=10.1186/s13059-022-02780-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Qiao J., Jin J., Yu H., Wei L.. Towards retraining-free RNA modification
prediction with incremental learning. Inf. Sci. 2024;660:120105. doi: 10.1016/j.ins.2024.120105.</cite> [<a href="https://doi.org/10.1016/j.ins.2024.120105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Inf.%20Sci.&amp;title=Towards%20retraining-free%20RNA%20modification%20prediction%20with%20incremental%20learning&amp;author=J.%20Qiao&amp;author=J.%20Jin&amp;author=H.%20Yu&amp;author=L.%20Wei&amp;volume=660&amp;publication_year=2024&amp;pages=120105&amp;doi=10.1016/j.ins.2024.120105&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Kulis M., Esteller M.. DNA methylation and cancer. Adv.
Genet. 2010;70:27–56. doi: 10.1016/B978-0-12-380866-0.60002-2.</cite> [<a href="https://doi.org/10.1016/B978-0-12-380866-0.60002-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20920744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Genet.&amp;title=DNA%20methylation%20and%20cancer&amp;author=M.%20Kulis&amp;author=M.%20Esteller&amp;volume=70&amp;publication_year=2010&amp;pages=27-56&amp;pmid=20920744&amp;doi=10.1016/B978-0-12-380866-0.60002-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Luo X., Wang Y., Zou Q., Xu L.. Recall DNA methylation
levels at low coverage sites using a CNN model in WGBS. PLoS Comput. Biol. 2023;19(6):e1011205.  doi: 10.1371/journal.pcbi.1011205.</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1011205" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10266633/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37315069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput.%20Biol.&amp;title=Recall%20DNA%20methylation%20levels%20at%20low%20coverage%20sites%20using%20a%20CNN%20model%20in%20WGBS&amp;author=X.%20Luo&amp;author=Y.%20Wang&amp;author=Q.%20Zou&amp;author=L.%20Xu&amp;volume=19&amp;issue=6&amp;publication_year=2023&amp;pages=e1011205&amp;pmid=37315069&amp;doi=10.1371/journal.pcbi.1011205&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Liang S.. et al. Rm-LR: a long-range-based deep learning model for predicting
multiple
types of RNA modifications. Comput. Biol. Med. 2023;164:107238. doi: 10.1016/j.compbiomed.2023.107238.</cite> [<a href="https://doi.org/10.1016/j.compbiomed.2023.107238" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37515874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput.%20Biol.%20Med.&amp;title=Rm-LR:%20a%20long-range-based%20deep%20learning%20model%20for%20predicting%20multiple%20types%20of%20RNA%20modifications&amp;author=S.%20Liang&amp;volume=164&amp;publication_year=2023&amp;pages=107238&amp;pmid=37515874&amp;doi=10.1016/j.compbiomed.2023.107238&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Hu T., Chitnis N., Monos D., Dinh A.. Next-generation sequencing
technologies: An overview. Hum. Immunol. 2021;82(11):801–811. doi: 10.1016/j.humimm.2021.02.012.</cite> [<a href="https://doi.org/10.1016/j.humimm.2021.02.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33745759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hum.%20Immunol.&amp;title=Next-generation%20sequencing%20technologies:%20An%20overview&amp;author=T.%20Hu&amp;author=N.%20Chitnis&amp;author=D.%20Monos&amp;author=A.%20Dinh&amp;volume=82&amp;issue=11&amp;publication_year=2021&amp;pages=801-811&amp;pmid=33745759&amp;doi=10.1016/j.humimm.2021.02.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Margulies M.. et al. Genome sequencing in
microfabricated high-density picolitre reactors. Nature. 2005;437(7057):376–380. doi: 10.1038/nature03959.</cite> [<a href="https://doi.org/10.1038/nature03959" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1464427/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16056220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genome%20sequencing%20in%20microfabricated%20high-density%20picolitre%20reactors&amp;author=M.%20Margulies&amp;volume=437&amp;issue=7057&amp;publication_year=2005&amp;pages=376-380&amp;pmid=16056220&amp;doi=10.1038/nature03959&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Sack L. M., Davoli T., Xu Q., Li M. Z., Elledge S. J.. Sources
of error in mammalian genetic screens. Genes,
Genomes Genomics. 2016;6(9):2781–2790. doi: 10.1534/g3.116.030973.</cite> [<a href="https://doi.org/10.1534/g3.116.030973" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5015935/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27402361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes,%20Genomes%20Genomics&amp;title=Sources%20of%20error%20in%20mammalian%20genetic%20screens&amp;author=L.%20M.%20Sack&amp;author=T.%20Davoli&amp;author=Q.%20Xu&amp;author=M.%20Z.%20Li&amp;author=S.%20J.%20Elledge&amp;volume=6&amp;issue=9&amp;publication_year=2016&amp;pages=2781-2790&amp;pmid=27402361&amp;doi=10.1534/g3.116.030973&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>An N., Chen T., Zhang J., Wang G., Yan M., Yang S.. Rational Electrochemical Design of Cuprous Oxide Hierarchical Microarchitectures
and Their Derivatives for SERS Sensing Applications. Small Methods. 2024;8(5):2300910. doi: 10.1002/smtd.202300910.</cite> [<a href="https://doi.org/10.1002/smtd.202300910" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38415973/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Small%20Methods&amp;title=Rational%20Electrochemical%20Design%20of%20Cuprous%20Oxide%20Hierarchical%20Microarchitectures%20and%20Their%20Derivatives%20for%20SERS%20Sensing%20Applications&amp;author=N.%20An&amp;author=T.%20Chen&amp;author=J.%20Zhang&amp;author=G.%20Wang&amp;author=M.%20Yan&amp;volume=8&amp;issue=5&amp;publication_year=2024&amp;pages=2300910&amp;pmid=38415973&amp;doi=10.1002/smtd.202300910&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Lee H.. et al. Third-generation sequencing
and the future of genomics. bioRxiv. 2016:048603. doi: 10.1101/048603.</cite> [<a href="https://doi.org/10.1101/048603" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Third-generation%20sequencing%20and%20the%20future%20of%20genomics&amp;author=H.%20Lee&amp;publication_year=2016&amp;pages=048603&amp;doi=10.1101/048603&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Deamer D. W., Akeson M.. Nanopores and nucleic
acids: prospects for ultrarapid
sequencing. Trends Biotechnol. 2000;18(4):147–151. doi: 10.1016/S0167-7799(00)01426-8.</cite> [<a href="https://doi.org/10.1016/S0167-7799(00)01426-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10740260/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Biotechnol.&amp;title=Nanopores%20and%20nucleic%20acids:%20prospects%20for%20ultrarapid%20sequencing&amp;author=D.%20W.%20Deamer&amp;author=M.%20Akeson&amp;volume=18&amp;issue=4&amp;publication_year=2000&amp;pages=147-151&amp;pmid=10740260&amp;doi=10.1016/S0167-7799(00)01426-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>Li C., Wang H., Wen Y., Yin R., Zeng X., Li K.. GenoM7GNet: An Efficient N 7-Methylguanosine
Site Prediction Approach
Based on a Nucleotide Language Model. IEEE ACM
Trans. Comput. Biol. Bioinf. 2024;21(6):2258–2268. doi: 10.1109/tcbb.2024.3459870.</cite> [<a href="https://doi.org/10.1109/tcbb.2024.3459870" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39302806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20ACM%20Trans.%20Comput.%20Biol.%20Bioinf.&amp;title=GenoM7GNet:%20An%20Efficient%20N%207-Methylguanosine%20Site%20Prediction%20Approach%20Based%20on%20a%20Nucleotide%20Language%20Model&amp;author=C.%20Li&amp;author=H.%20Wang&amp;author=Y.%20Wen&amp;author=R.%20Yin&amp;author=X.%20Zeng&amp;volume=21&amp;issue=6&amp;publication_year=2024&amp;pages=2258-2268&amp;pmid=39302806&amp;doi=10.1109/tcbb.2024.3459870&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Chen S., Long S., Liu Y., Wang S., Hu Q., Fu L., Luo D.. Evaluation of a three-gene methylation
model for correlating
lymph node metastasis in postoperative early gastric cancer adjacent
samples. Front. Oncol. 2024;14:1432869. doi: 10.3389/fonc.2024.1432869.</cite> [<a href="https://doi.org/10.3389/fonc.2024.1432869" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11524798/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39484038/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Oncol.&amp;title=Evaluation%20of%20a%20three-gene%20methylation%20model%20for%20correlating%20lymph%20node%20metastasis%20in%20postoperative%20early%20gastric%20cancer%20adjacent%20samples&amp;author=S.%20Chen&amp;author=S.%20Long&amp;author=Y.%20Liu&amp;author=S.%20Wang&amp;author=Q.%20Hu&amp;volume=14&amp;publication_year=2024&amp;pages=1432869&amp;pmid=39484038&amp;doi=10.3389/fonc.2024.1432869&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>Liu Y., Shen X., Gong Y., Liu Y., Song B., Zeng X.. Sequence Alignment/Map
format: a comprehensive review of approaches
and applications. Briefings Bioinf. 2023;24(5):bbad320. doi: 10.1093/bib/bbad320.</cite> [<a href="https://doi.org/10.1093/bib/bbad320" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37668049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Briefings%20Bioinf.&amp;title=Sequence%20Alignment/Map%20format:%20a%20comprehensive%20review%20of%20approaches%20and%20applications&amp;author=Y.%20Liu&amp;author=X.%20Shen&amp;author=Y.%20Gong&amp;author=Y.%20Liu&amp;author=B.%20Song&amp;volume=24&amp;issue=5&amp;publication_year=2023&amp;pages=bbad320&amp;pmid=37668049&amp;doi=10.1093/bib/bbad320&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>Ao C., Zou Q., Yu L.. NmRF: identification of multispecies
RNA 2′-O-methylation
modification sites from RNA sequences. Briefings
Bioinf. 2022;23(1):bbab480. doi: 10.1093/bib/bbab480.</cite> [<a href="https://doi.org/10.1093/bib/bbab480" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34850821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Briefings%20Bioinf.&amp;title=NmRF:%20identification%20of%20multispecies%20RNA%202%E2%80%B2-O-methylation%20modification%20sites%20from%20RNA%20sequences&amp;author=C.%20Ao&amp;author=Q.%20Zou&amp;author=L.%20Yu&amp;volume=23&amp;issue=1&amp;publication_year=2022&amp;pages=bbab480&amp;pmid=34850821&amp;doi=10.1093/bib/bbab480&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Li H., Liu B.. BioSeq-Diabolo: Biological
sequence similarity analysis using Diabolo. PLoS Comput. Biol. 2023;19(6):e1011214.  doi: 10.1371/journal.pcbi.1011214.</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1011214" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10313010/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37339155/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput.%20Biol.&amp;title=BioSeq-Diabolo:%20Biological%20sequence%20similarity%20analysis%20using%20Diabolo&amp;author=H.%20Li&amp;author=B.%20Liu&amp;volume=19&amp;issue=6&amp;publication_year=2023&amp;pages=e1011214&amp;pmid=37339155&amp;doi=10.1371/journal.pcbi.1011214&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Li H.-L., Pang Y.-H., Liu B.. BioSeq-BLM: a platform
for analyzing
DNA, RNA and protein sequences based on biological language models. Nucleic Acids Res. 2021;49(22):e129.  doi: 10.1093/nar/gkab829.</cite> [<a href="https://doi.org/10.1093/nar/gkab829" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8682797/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34581805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=BioSeq-BLM:%20a%20platform%20for%20analyzing%20DNA,%20RNA%20and%20protein%20sequences%20based%20on%20biological%20language%20models&amp;author=H.-L.%20Li&amp;author=Y.-H.%20Pang&amp;author=B.%20Liu&amp;volume=49&amp;issue=22&amp;publication_year=2021&amp;pages=e129&amp;pmid=34581805&amp;doi=10.1093/nar/gkab829&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>LeCun Y., Bottou L., Bengio Y., Haffner P.. Gradient-based learning
applied to document recognition. Proc. IEEE. 1998;86(11):2278–2324. doi: 10.1109/5.726791.</cite> [<a href="https://doi.org/10.1109/5.726791" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20IEEE&amp;title=Gradient-based%20learning%20applied%20to%20document%20recognition&amp;author=Y.%20LeCun&amp;author=L.%20Bottou&amp;author=Y.%20Bengio&amp;author=P.%20Haffner&amp;volume=86&amp;issue=11&amp;publication_year=1998&amp;pages=2278-2324&amp;doi=10.1109/5.726791&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Hochreiter S., Schmidhuber J.. Long short-term
memory. Neural
Comput. 1997;9(8):1735–1780. doi: 10.1162/neco.1997.9.8.1735.</cite> [<a href="https://doi.org/10.1162/neco.1997.9.8.1735" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9377276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neural%20Comput.&amp;title=Long%20short-term%20memory&amp;author=S.%20Hochreiter&amp;author=J.%20Schmidhuber&amp;volume=9&amp;issue=8&amp;publication_year=1997&amp;pages=1735-1780&amp;pmid=9377276&amp;doi=10.1162/neco.1997.9.8.1735&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Liu M., Li C., Chen R., Cao D., Zeng X.. Geometric deep learning
for drug discovery. Expert Syst. Appl. 2024;240:122498. doi: 10.1016/j.eswa.2023.122498.</cite> [<a href="https://doi.org/10.1016/j.eswa.2023.122498" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Expert%20Syst.%20Appl.&amp;title=Geometric%20deep%20learning%20for%20drug%20discovery&amp;author=M.%20Liu&amp;author=C.%20Li&amp;author=R.%20Chen&amp;author=D.%20Cao&amp;author=X.%20Zeng&amp;volume=240&amp;publication_year=2024&amp;pages=122498&amp;doi=10.1016/j.eswa.2023.122498&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Bonet J.. et al. DeepMP: a deep learning tool to detect DNA base modifications
on
Nanopore sequencing data. Bioinformatics. 2022;38(5):1235–1243. doi: 10.1093/bioinformatics/btab745.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btab745" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8826383/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34718417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=DeepMP:%20a%20deep%20learning%20tool%20to%20detect%20DNA%20base%20modifications%20on%20Nanopore%20sequencing%20data&amp;author=J.%20Bonet&amp;volume=38&amp;issue=5&amp;publication_year=2022&amp;pages=1235-1243&amp;pmid=34718417&amp;doi=10.1093/bioinformatics/btab745&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Nguyen T. A.. et al. Direct identification
of A-to-I editing sites with nanopore native
RNA sequencing. Nat. Methods. 2022;19(7):833–844. doi: 10.1038/s41592-022-01513-3.</cite> [<a href="https://doi.org/10.1038/s41592-022-01513-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35697834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Direct%20identification%20of%20A-to-I%20editing%20sites%20with%20nanopore%20native%20RNA%20sequencing&amp;author=T.%20A.%20Nguyen&amp;volume=19&amp;issue=7&amp;publication_year=2022&amp;pages=833-844&amp;pmid=35697834&amp;doi=10.1038/s41592-022-01513-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>Ni P.. et al. DeepSignal: detecting
DNA methylation state from Nanopore sequencing
reads using deep-learning. Bioinformatics. 2019;35(22):4586–4595. doi: 10.1093/bioinformatics/btz276.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz276" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30994904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=DeepSignal:%20detecting%20DNA%20methylation%20state%20from%20Nanopore%20sequencing%20reads%20using%20deep-learning&amp;author=P.%20Ni&amp;volume=35&amp;issue=22&amp;publication_year=2019&amp;pages=4586-4595&amp;pmid=30994904&amp;doi=10.1093/bioinformatics/btz276&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Liu Q., Fang L., Yu G., Wang D., Xiao C.-L., Wang K.. Detection of DNA base
modifications by deep recurrent neural network
on Oxford Nanopore sequencing data. Nat. Commun. 2019;10(1):2449. doi: 10.1038/s41467-019-10168-2.</cite> [<a href="https://doi.org/10.1038/s41467-019-10168-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6547721/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31164644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Detection%20of%20DNA%20base%20modifications%20by%20deep%20recurrent%20neural%20network%20on%20Oxford%20Nanopore%20sequencing%20data&amp;author=Q.%20Liu&amp;author=L.%20Fang&amp;author=G.%20Yu&amp;author=D.%20Wang&amp;author=C.-L.%20Xiao&amp;volume=10&amp;issue=1&amp;publication_year=2019&amp;pages=2449&amp;pmid=31164644&amp;doi=10.1038/s41467-019-10168-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>Chen J., Zou Q., Li J.. DeepM6ASeq-EL:
prediction of human N6-methyladenosine
(m 6 a) sites with LSTM and ensemble learning. Front. Comput. Sci. 2022;16:162302. doi: 10.1007/s11704-020-0180-0.</cite> [<a href="https://doi.org/10.1007/s11704-020-0180-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Comput.%20Sci.&amp;title=DeepM6ASeq-EL:%20prediction%20of%20human%20N6-methyladenosine%20(m%206%20a)%20sites%20with%20LSTM%20and%20ensemble%20learning&amp;author=J.%20Chen&amp;author=Q.%20Zou&amp;author=J.%20Li&amp;volume=16&amp;publication_year=2022&amp;pages=162302&amp;doi=10.1007/s11704-020-0180-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>

Zhang, Y.-z.
; et al. 
On the application of BERT models
for nanopore methylation detection. In 2021
IEEE International Conference on Bioinformatics
and Biomedicine (BIBM); IEEE, 2021, pp 320–327.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=2021%20IEEE%20International%20Conference%20on%20Bioinformatics%20and%20Biomedicine%20(BIBM)&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>

Devlin, J.
; 
Chang, M.-W.
; 
Lee, K.
; 
Toutanova, K.
. 
Bert:
Pre-training of deep bidirectional
transformers for language understanding. In Proceedings of the 2019 Conference of the North American Chapter
of the Association for Computational Linguistics: Human Language Technologies,
Volume 1 (Long and Short Papers), 2019; pp 4171–4186.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%202019%20Conference%20of%20the%20North%20American%20Chapter%20of%20the%20Association%20for%20Computational%20Linguistics:%20Human%20Language%20Technologies,%20Volume%201%20(Long%20and%20Short%20Papers)&amp;title=Bert:%20Pre-training%20of%20deep%20bidirectional%20transformers%20for%20language%20understanding&amp;publication_year=2019&amp;pages=4171-4186&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34">
<cite>Wang Y., Liu T., Xu D., Shi H., Zhang C., Mo Y. Y., Wang Z.. Predicting DNA methylation state of CpG dinucleotide using genome
topological features and deep networks. Sci.
Rep. 2016;6(1):19598. doi: 10.1038/srep19598.</cite> [<a href="https://doi.org/10.1038/srep19598" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4726425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26797014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Predicting%20DNA%20methylation%20state%20of%20CpG%20dinucleotide%20using%20genome%20topological%20features%20and%20deep%20networks&amp;author=Y.%20Wang&amp;author=T.%20Liu&amp;author=D.%20Xu&amp;author=H.%20Shi&amp;author=C.%20Zhang&amp;volume=6&amp;issue=1&amp;publication_year=2016&amp;pages=19598&amp;pmid=26797014&amp;doi=10.1038/srep19598&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref35">
<cite>Yin C., Wang R., Qiao J., Shi H., Duan H., Jiang X., Teng S., Wei L.. NanoCon: contrastive
learning-based deep hybrid network for nanopore methylation detection. Bioinformatics. 2024;40(2):btae046. doi: 10.1093/bioinformatics/btae046.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btae046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10873575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38305428/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=NanoCon:%20contrastive%20learning-based%20deep%20hybrid%20network%20for%20nanopore%20methylation%20detection&amp;author=C.%20Yin&amp;author=R.%20Wang&amp;author=J.%20Qiao&amp;author=H.%20Shi&amp;author=H.%20Duan&amp;volume=40&amp;issue=2&amp;publication_year=2024&amp;pages=btae046&amp;pmid=38305428&amp;doi=10.1093/bioinformatics/btae046&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>Dong X., Yu Z., Cao W., Shi Y., Ma Q.. A survey on ensemble
learning. Front. Comput. Sci. 2020;14:241–258. doi: 10.1007/s11704-019-8208-z.</cite> [<a href="https://doi.org/10.1007/s11704-019-8208-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Comput.%20Sci.&amp;title=A%20survey%20on%20ensemble%20learning&amp;author=X.%20Dong&amp;author=Z.%20Yu&amp;author=W.%20Cao&amp;author=Y.%20Shi&amp;author=Q.%20Ma&amp;volume=14&amp;publication_year=2020&amp;pages=241-258&amp;doi=10.1007/s11704-019-8208-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>Yang X.. et al. Modality-DTA: multimodality fusion strategy for drug–target
affinity prediction. IEEE ACM Trans. Comput.
Biol. Bioinf. 2023;20(2):1200–1210. doi: 10.1109/TCBB.2022.3205282.</cite> [<a href="https://doi.org/10.1109/TCBB.2022.3205282" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36083952/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20ACM%20Trans.%20Comput.%20Biol.%20Bioinf.&amp;title=Modality-DTA:%20multimodality%20fusion%20strategy%20for%20drug%E2%80%93target%20affinity%20prediction&amp;author=X.%20Yang&amp;volume=20&amp;issue=2&amp;publication_year=2023&amp;pages=1200-1210&amp;pmid=36083952&amp;doi=10.1109/TCBB.2022.3205282&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Wang Y., Zhai Y., Ding Y., Zou Q.. SBSM-Pro: support bio-sequence
machine for proteins. Sci. China Inf. Sci. 2024;67(11):212106. doi: 10.1007/s11432-024-4171-9.</cite> [<a href="https://doi.org/10.1007/s11432-024-4171-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20China%20Inf.%20Sci.&amp;title=SBSM-Pro:%20support%20bio-sequence%20machine%20for%20proteins&amp;author=Y.%20Wang&amp;author=Y.%20Zhai&amp;author=Y.%20Ding&amp;author=Q.%20Zou&amp;volume=67&amp;issue=11&amp;publication_year=2024&amp;pages=212106&amp;doi=10.1007/s11432-024-4171-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Yuen Z. W.-S., Srivastava A., Daniel R., McNevin D., Jack C., Eyras E.. Systematic
benchmarking of tools for CpG methylation detection from
nanopore sequencing. Nat. Commun. 2021;12(1):3438. doi: 10.1038/s41467-021-23778-6.</cite> [<a href="https://doi.org/10.1038/s41467-021-23778-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8187371/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34103501/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Systematic%20benchmarking%20of%20tools%20for%20CpG%20methylation%20detection%20from%20nanopore%20sequencing&amp;author=Z.%20W.-S.%20Yuen&amp;author=A.%20Srivastava&amp;author=R.%20Daniel&amp;author=D.%20McNevin&amp;author=C.%20Jack&amp;volume=12&amp;issue=1&amp;publication_year=2021&amp;pages=3438&amp;pmid=34103501&amp;doi=10.1038/s41467-021-23778-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Lasantha D., Vidanagamachchi S., Nallaperuma S.. Deep learning
and ensemble deep learning
for circRNA-RBP interaction prediction in the last decade: A review. Eng. Appl. Artif. Intell. 2023;123:106352. doi: 10.1016/j.engappai.2023.106352.</cite> [<a href="https://doi.org/10.1016/j.engappai.2023.106352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eng.%20Appl.%20Artif.%20Intell.&amp;title=Deep%20learning%20and%20ensemble%20deep%20learning%20for%20circRNA-RBP%20interaction%20prediction%20in%20the%20last%20decade:%20A%20review&amp;author=D.%20Lasantha&amp;author=S.%20Vidanagamachchi&amp;author=S.%20Nallaperuma&amp;volume=123&amp;publication_year=2023&amp;pages=106352&amp;doi=10.1016/j.engappai.2023.106352&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Yan J., Xu Y., Cheng Q., Jiang S., Wang Q., Xiao Y., Ma C., Yan J., Wang X.. LightGBM: accelerated genomically
designed crop breeding through ensemble learning. Genome Biol. 2021;22:271. doi: 10.1186/s13059-021-02492-y.</cite> [<a href="https://doi.org/10.1186/s13059-021-02492-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8451137/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34544450/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol.&amp;title=LightGBM:%20accelerated%20genomically%20designed%20crop%20breeding%20through%20ensemble%20learning&amp;author=J.%20Yan&amp;author=Y.%20Xu&amp;author=Q.%20Cheng&amp;author=S.%20Jiang&amp;author=Q.%20Wang&amp;volume=22&amp;publication_year=2021&amp;pages=271&amp;pmid=34544450&amp;doi=10.1186/s13059-021-02492-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Wang G., Sun J., Ma J., Xu K., Gu J.. Sentiment classification:
The contribution of ensemble learning. Decis.
Support Syst. 2014;57:77–93. doi: 10.1016/j.dss.2013.08.002.</cite> [<a href="https://doi.org/10.1016/j.dss.2013.08.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Decis.%20Support%20Syst.&amp;title=Sentiment%20classification:%20The%20contribution%20of%20ensemble%20learning&amp;author=G.%20Wang&amp;author=J.%20Sun&amp;author=J.%20Ma&amp;author=K.%20Xu&amp;author=J.%20Gu&amp;volume=57&amp;publication_year=2014&amp;pages=77-93&amp;doi=10.1016/j.dss.2013.08.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Kazemi Y., Mirroshandel S. A.. A novel
method for predicting kidney stone type using
ensemble learning. Artif. Intell. Med. 2018;84:117–126. doi: 10.1016/j.artmed.2017.12.001.</cite> [<a href="https://doi.org/10.1016/j.artmed.2017.12.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29241659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Artif.%20Intell.%20Med.&amp;title=A%20novel%20method%20for%20predicting%20kidney%20stone%20type%20using%20ensemble%20learning&amp;author=Y.%20Kazemi&amp;author=S.%20A.%20Mirroshandel&amp;volume=84&amp;publication_year=2018&amp;pages=117-126&amp;pmid=29241659&amp;doi=10.1016/j.artmed.2017.12.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref44">
<cite>Qian F., Jia R., Cheng M., Chaudhary A., Melhi S., Mekkey S. D., Zhu N., Wang C., Razak F., Xu X.. et al. An overview
of polylactic acid (PLA) nanocomposites for sensors. Adv. Compos. Hybrid Mater. 2024;7(3):75. doi: 10.1007/s42114-024-00887-6.</cite> [<a href="https://doi.org/10.1007/s42114-024-00887-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Compos.%20Hybrid%20Mater.&amp;title=An%20overview%20of%20polylactic%20acid%20(PLA)%20nanocomposites%20for%20sensors&amp;author=F.%20Qian&amp;author=R.%20Jia&amp;author=M.%20Cheng&amp;author=A.%20Chaudhary&amp;author=S.%20Melhi&amp;volume=7&amp;issue=3&amp;publication_year=2024&amp;pages=75&amp;doi=10.1007/s42114-024-00887-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref45">
<cite>Wheeler D. L.. et al. Database resources of the national center for biotechnology information. Nucleic Acids Res. 2007;36(suppl_1):D13–D21. doi: 10.1093/nar/gkm1000.</cite> [<a href="https://doi.org/10.1093/nar/gkm1000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2238880/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18045790/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Database%20resources%20of%20the%20national%20center%20for%20biotechnology%20information&amp;author=D.%20L.%20Wheeler&amp;volume=36&amp;issue=suppl_1&amp;publication_year=2007&amp;pages=D13-D21&amp;pmid=18045790&amp;doi=10.1093/nar/gkm1000&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref46">
<cite>Howe K. L.. et al. Ensembl Genomes 2020enabling non-vertebrate genomic research. Nucleic Acids Res. 2020;48(D1):D689–D695. doi: 10.1093/nar/gkz890.</cite> [<a href="https://doi.org/10.1093/nar/gkz890" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6943047/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31598706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Ensembl%20Genomes%202020%EE%97%B8enabling%20non-vertebrate%20genomic%20research&amp;author=K.%20L.%20Howe&amp;volume=48&amp;issue=D1&amp;publication_year=2020&amp;pages=D689-D695&amp;pmid=31598706&amp;doi=10.1093/nar/gkz890&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref47">
<cite>Zhang Z.-Y., Zhang Z., Ye X., Sakurai T., Lin H.. A BERT-based
model for the prediction of lncRNA subcellular localization in Homo
sapiens. Int. J. Biol. Macromol. 2024;265:130659. doi: 10.1016/j.ijbiomac.2024.130659.</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2024.130659" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38462114/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Biol.%20Macromol.&amp;title=A%20BERT-based%20model%20for%20the%20prediction%20of%20lncRNA%20subcellular%20localization%20in%20Homo%20sapiens&amp;author=Z.-Y.%20Zhang&amp;author=Z.%20Zhang&amp;author=X.%20Ye&amp;author=T.%20Sakurai&amp;author=H.%20Lin&amp;volume=265&amp;publication_year=2024&amp;pages=130659&amp;pmid=38462114&amp;doi=10.1016/j.ijbiomac.2024.130659&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref48">
<cite>Zou X., Ren L., Cai P., Zhang Y., Ding H., Deng K., Yu X., Lin H., Huang C.. Accurately identifying hemagglutinin
using sequence information and machine learning methods. Front. Med. 2023;10:1281880. doi: 10.3389/fmed.2023.1281880.</cite> [<a href="https://doi.org/10.3389/fmed.2023.1281880" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10644030/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38020152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Med.&amp;title=Accurately%20identifying%20hemagglutinin%20using%20sequence%20information%20and%20machine%20learning%20methods&amp;author=X.%20Zou&amp;author=L.%20Ren&amp;author=P.%20Cai&amp;author=Y.%20Zhang&amp;author=H.%20Ding&amp;volume=10&amp;publication_year=2023&amp;pages=1281880&amp;pmid=38020152&amp;doi=10.3389/fmed.2023.1281880&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref49">
<cite>Zhu W., Yuan S.-S., Li J., Huang C.-B., Lin H., Liao B.. A first computational
frame for recognizing heparin-binding protein. Diagnostics. 2023;13(14):2465. doi: 10.3390/diagnostics13142465.</cite> [<a href="https://doi.org/10.3390/diagnostics13142465" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10377868/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37510209/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diagnostics&amp;title=A%20first%20computational%20frame%20for%20recognizing%20heparin-binding%20protein&amp;author=W.%20Zhu&amp;author=S.-S.%20Yuan&amp;author=J.%20Li&amp;author=C.-B.%20Huang&amp;author=H.%20Lin&amp;volume=13&amp;issue=14&amp;publication_year=2023&amp;pages=2465&amp;pmid=37510209&amp;doi=10.3390/diagnostics13142465&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref50">
<cite>Zhu H., Hao H., Yu L.. Identifying
disease-related microbes based on multi-scale
variational graph autoencoder embedding Wasserstein distance. BMC Biol. 2023;21(1):294. doi: 10.1186/s12915-023-01796-8.</cite> [<a href="https://doi.org/10.1186/s12915-023-01796-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10731776/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38115088/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Biol.&amp;title=Identifying%20disease-related%20microbes%20based%20on%20multi-scale%20variational%20graph%20autoencoder%20embedding%20Wasserstein%20distance&amp;author=H.%20Zhu&amp;author=H.%20Hao&amp;author=L.%20Yu&amp;volume=21&amp;issue=1&amp;publication_year=2023&amp;pages=294&amp;pmid=38115088&amp;doi=10.1186/s12915-023-01796-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref51">
<cite>Wang L.. et al. A deep multiple kernel
learning-based higher-order fuzzy inference
system for identifying DNA N4-methylcytosine sites. Inf. Sci. 2023;630:40–52. doi: 10.1016/j.ins.2023.01.149.</cite> [<a href="https://doi.org/10.1016/j.ins.2023.01.149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Inf.%20Sci.&amp;title=A%20deep%20multiple%20kernel%20learning-based%20higher-order%20fuzzy%20inference%20system%20for%20identifying%20DNA%20N4-methylcytosine%20sites&amp;author=L.%20Wang&amp;volume=630&amp;publication_year=2023&amp;pages=40-52&amp;doi=10.1016/j.ins.2023.01.149&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref52">
<cite>Ai C., Yang H., Ding Y., Tang J., Guo F.. Low rank matrix
factorization algorithm based on multi-graph regularization for detecting
drug-disease association. IEEE ACM Trans. Comput.
Biol. Bioinf. 2023;20(5):3033–3043. doi: 10.1109/TCBB.2023.3274587.</cite> [<a href="https://doi.org/10.1109/TCBB.2023.3274587" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37159322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20ACM%20Trans.%20Comput.%20Biol.%20Bioinf.&amp;title=Low%20rank%20matrix%20factorization%20algorithm%20based%20on%20multi-graph%20regularization%20for%20detecting%20drug-disease%20association&amp;author=C.%20Ai&amp;author=H.%20Yang&amp;author=Y.%20Ding&amp;author=J.%20Tang&amp;author=F.%20Guo&amp;volume=20&amp;issue=5&amp;publication_year=2023&amp;pages=3033-3043&amp;pmid=37159322&amp;doi=10.1109/TCBB.2023.3274587&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref53">
<cite>Liu B., Gao X., Zhang H.. BioSeq-Analysis2.0: an updated platform for analyzing
DNA, RNA and protein sequences at sequence level and residue level
based on machine learning approaches. Nucleic
Acids Res. 2019;47(20):e127.  doi: 10.1093/nar/gkz740.</cite> [<a href="https://doi.org/10.1093/nar/gkz740" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6847461/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31504851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=BioSeq-Analysis2.0:%20an%20updated%20platform%20for%20analyzing%20DNA,%20RNA%20and%20protein%20sequences%20at%20sequence%20level%20and%20residue%20level%20based%20on%20machine%20learning%20approaches&amp;author=B.%20Liu&amp;author=X.%20Gao&amp;author=H.%20Zhang&amp;volume=47&amp;issue=20&amp;publication_year=2019&amp;pages=e127&amp;pmid=31504851&amp;doi=10.1093/nar/gkz740&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref54">
<cite>Tang Y.-J., Pang Y.-H., Liu B.. IDP-Seq2Seq:
identification of intrinsically
disordered regions based on sequence to sequence learning. Bioinformatics. 2021;36(21):5177–5186. doi: 10.1093/bioinformatics/btaa667.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaa667" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32702119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=IDP-Seq2Seq:%20identification%20of%20intrinsically%20disordered%20regions%20based%20on%20sequence%20to%20sequence%20learning&amp;author=Y.-J.%20Tang&amp;author=Y.-H.%20Pang&amp;author=B.%20Liu&amp;volume=36&amp;issue=21&amp;publication_year=2021&amp;pages=5177-5186&amp;pmid=32702119&amp;doi=10.1093/bioinformatics/btaa667&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref55">
<cite>Zulfiqar H., Guo Z., Ahmad R. M., Ahmed Z., Cai P., Chen X., Zhang Y., Lin H., Shi Z.. Deep-STP: a deep learning-based
approach to predict snake toxin proteins by using word embeddings. Front. Med. 2024;10:1291352. doi: 10.3389/fmed.2023.1291352.</cite> [<a href="https://doi.org/10.3389/fmed.2023.1291352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10829051/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38298505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Med.&amp;title=Deep-STP:%20a%20deep%20learning-based%20approach%20to%20predict%20snake%20toxin%20proteins%20by%20using%20word%20embeddings&amp;author=H.%20Zulfiqar&amp;author=Z.%20Guo&amp;author=R.%20M.%20Ahmad&amp;author=Z.%20Ahmed&amp;author=P.%20Cai&amp;volume=10&amp;publication_year=2024&amp;pages=1291352&amp;pmid=38298505&amp;doi=10.3389/fmed.2023.1291352&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref56">
<cite>Guo X., Huang Z., Ju F., Zhao C., Yu L.. Highly Accurate
Estimation of Cell Type Abundance in Bulk Tissues Based on Single-Cell
Reference and Domain Adaptive Matching. Adv.
Sci. 2024;11(7):2306329. doi: 10.1002/advs.202306329.</cite> [<a href="https://doi.org/10.1002/advs.202306329" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10870031/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38072669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Sci.&amp;title=Highly%20Accurate%20Estimation%20of%20Cell%20Type%20Abundance%20in%20Bulk%20Tissues%20Based%20on%20Single-Cell%20Reference%20and%20Domain%20Adaptive%20Matching&amp;author=X.%20Guo&amp;author=Z.%20Huang&amp;author=F.%20Ju&amp;author=C.%20Zhao&amp;author=L.%20Yu&amp;volume=11&amp;issue=7&amp;publication_year=2024&amp;pages=2306329&amp;pmid=38072669&amp;doi=10.1002/advs.202306329&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref57">
<cite>McInnes L., Healy J., Melville J.. Umap: Uniform
manifold approximation
and projection for dimension reduction. arXiv. 2018:arXiv:1802.03426. doi: 10.48550/arXiv.1802.03426.</cite> [<a href="https://doi.org/10.48550/arXiv.1802.03426" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=arXiv&amp;title=Umap:%20Uniform%20manifold%20approximation%20and%20projection%20for%20dimension%20reduction&amp;author=L.%20McInnes&amp;author=J.%20Healy&amp;author=J.%20Melville&amp;publication_year=2018&amp;pages=arXiv:1802.03426&amp;doi=10.48550/arXiv.1802.03426&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The reference
genome of was obtained
from NCBI under the version number GCF_000001735.4_TAIR10.1. The reference genome and gene annotation of were obtained from EnsemblPlants under
the version number IRGSP-1.0. DNA nanopore
read data for NA12878 were accessed and downloaded from the European
Nucleotide Archive (ENA) via access number PRJEB23027, and gene annotation
information was obtained from the ENCODE project via access number
ENCFF835NTC. All machine learning code and trained weight files can
be obtained from the GitHub link <a href="https://github.com/ViceMusic/EDNTOM" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ViceMusic/EDNTOM</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Omega are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsomega.5c01924"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/ao5c01924.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (7.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12332607/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12332607/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332607%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332607/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12332607/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12332607/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40787313/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12332607/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40787313/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12332607/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12332607/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="XNTR9tzo5wLgpNmsds1DItsZlCZ5k9k2M7CXFqxL3w09I8bCI9lWo9wp0ip7EKU7">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
