
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            SpyRing-Mediated Cyclization of TEV Protease, Guided by AlphaFold, Improves Thermostability - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="4DDE493489B755331B493400074FDB5D.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsomega">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332596/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Omega">
<meta name="citation_title" content="SpyRing-Mediated Cyclization of TEV Protease, Guided by AlphaFold, Improves Thermostability">
<meta name="citation_author" content="Tadashi Nakai">
<meta name="citation_author_institution" content="Graduate School of Science and Technology, Hiroshima Institute of Technology, Hiroshima 731-5193, Japan">
<meta name="citation_author_institution" content="Faculty of Life Sciences, Hiroshima Institute of Technology, Hiroshima 731-5193, Japan">
<meta name="citation_author_institution" content="Faculty of Environmental Studies, Hiroshima Institute of Technology, Hiroshima 731-5193, Japan">
<meta name="citation_author" content="Yota Nakai">
<meta name="citation_author_institution" content="Hiroshima University Senior High School, Hiroshima 734-0005, Japan">
<meta name="citation_author" content="Naoki Takami">
<meta name="citation_author_institution" content="Faculty of Life Sciences, Hiroshima Institute of Technology, Hiroshima 731-5193, Japan">
<meta name="citation_author" content="Emi Nakai">
<meta name="citation_author_institution" content="Faculty of Life Sciences, Hiroshima Institute of Technology, Hiroshima 731-5193, Japan">
<meta name="citation_author_institution" content="Faculty of Environmental Studies, Hiroshima Institute of Technology, Hiroshima 731-5193, Japan">
<meta name="citation_author" content="Toshihide Okajima">
<meta name="citation_author_institution" content="Institute of Scientific and Industrial Research (SANKEN), The University of Osaka, Ibaraki, Osaka 567-0047, Japan">
<meta name="citation_publication_date" content="2025 Jul 25">
<meta name="citation_volume" content="10">
<meta name="citation_issue" content="30">
<meta name="citation_firstpage" content="33862">
<meta name="citation_doi" content="10.1021/acsomega.5c05386">
<meta name="citation_pmid" content="40787351">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332596/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332596/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332596/pdf/ao5c05386.pdf">
<meta name="description" content="Cyclization is a promising strategy to enhance protein stability, but its applicability is often limited by structural constraints such as the distance between the N- and C-terminal regions. Here, we report the rational design and characterization ...">
<meta name="og:title" content="SpyRing-Mediated Cyclization of TEV Protease, Guided by AlphaFold, Improves Thermostability">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Cyclization is a promising strategy to enhance protein stability, but its applicability is often limited by structural constraints such as the distance between the N- and C-terminal regions. Here, we report the rational design and characterization ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332596/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12332596">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsomega.5c05386"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/ao5c05386.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332596%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12332596/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12332596/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332596/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsomega.png" alt="ACS Omega logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Omega" title="Link to ACS Omega" shape="default" href="https://pubs.acs.org/journal/acsodf" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Omega</button></div>. 2025 Jul 25;10(30):33862–33871. doi: <a href="https://doi.org/10.1021/acsomega.5c05386" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsomega.5c05386</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Omega%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Omega%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>SpyRing-Mediated
Cyclization of TEV Protease, Guided
by AlphaFold, Improves Thermostability</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nakai%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Tadashi Nakai</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Tadashi Nakai</span></h3>
<div class="p">
<sup>†</sup>
Graduate
School of Science and Technology, Hiroshima
Institute of Technology, Hiroshima
731-5193, Japan
</div>
<div class="p">
<sup>‡</sup>
Faculty
of Life Sciences, Hiroshima Institute of
Technology, Hiroshima
731-5193, Japan
</div>
<div class="p">
<sup>§</sup>
Faculty
of Environmental Studies, Hiroshima Institute
of Technology, Hiroshima
731-5193, Japan
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nakai%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tadashi Nakai</span></a>
</div>
</div>
<sup>†,</sup><sup>‡,</sup><sup>§,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nakai%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yota Nakai</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yota Nakai</span></h3>
<div class="p">
<sup>∥</sup>
Hiroshima
University Senior High School, Hiroshima
734-0005, Japan
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nakai%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yota Nakai</span></a>
</div>
</div>
<sup>∥</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Takami%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Naoki Takami</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Naoki Takami</span></h3>
<div class="p">
<sup>‡</sup>
Faculty
of Life Sciences, Hiroshima Institute of
Technology, Hiroshima
731-5193, Japan
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Takami%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Naoki Takami</span></a>
</div>
</div>
<sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nakai%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Emi Nakai</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Emi Nakai</span></h3>
<div class="p">
<sup>‡</sup>
Faculty
of Life Sciences, Hiroshima Institute of
Technology, Hiroshima
731-5193, Japan
</div>
<div class="p">
<sup>§</sup>
Faculty
of Environmental Studies, Hiroshima Institute
of Technology, Hiroshima
731-5193, Japan
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nakai%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Emi Nakai</span></a>
</div>
</div>
<sup>‡,</sup><sup>§</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Okajima%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Toshihide Okajima</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Toshihide Okajima</span></h3>
<div class="p">
<sup>⊥</sup>
Institute
of Scientific and Industrial Research (SANKEN), The University of Osaka, Ibaraki, Osaka
567-0047, Japan
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Okajima%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Toshihide Okajima</span></a>
</div>
</div>
<sup>⊥</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>†</sup>
Graduate
School of Science and Technology, Hiroshima
Institute of Technology, Hiroshima
731-5193, Japan
</div>
<div id="aff2">
<sup>‡</sup>
Faculty
of Life Sciences, Hiroshima Institute of
Technology, Hiroshima
731-5193, Japan
</div>
<div id="aff3">
<sup>§</sup>
Faculty
of Environmental Studies, Hiroshima Institute
of Technology, Hiroshima
731-5193, Japan
</div>
<div id="aff4">
<sup>∥</sup>
Hiroshima
University Senior High School, Hiroshima
734-0005, Japan
</div>
<div id="aff5">
<sup>⊥</sup>
Institute
of Scientific and Industrial Research (SANKEN), The University of Osaka, Ibaraki, Osaka
567-0047, Japan
</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
E-mail: <span>t.nakai.wj@cc.it-hiroshima.ac.jp</span>.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jun 7; Accepted 2025 Jul 15; Revised 2025 Jul 2; Collection date 2025 Aug 5.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY-NC-ND 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12332596  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40787351/" class="usa-link">40787351</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Cyclization is a promising strategy to enhance protein
stability,
but its applicability is often limited by structural constraints such
as the distance between the N- and C-terminal regions. Here, we report
the rational design and characterization of a cyclized Tobacco Etch
Virus protease (cTEVp) using the SpyRing system, which enables covalent
cyclization through SpyTag/SpyCatcher-mediated isopeptide bond formation.
We applied this approach to a widely used engineered TEVp variant
(L56V, S135G, S219V, Δ238–242) and employed AlphaFold
structure prediction to optimize linker length and domain positioning.
Despite the ∼40 Å separation between the N- and C-termini
of native TEVp, AlphaFold modeling suggested that the fused SpyTag
and SpyCatcher domains can adopt a favorable configuration for intramolecular
cyclization. The resulting cTEVp exhibited proteolytic activity comparable
to the noncyclized TEVp, indicating that structural constraint via
SpyRing-mediated cyclization did not impair enzymatic function. Importantly,
cTEVp displayed significantly improved thermostability relative to
its noncyclized counterpart, as demonstrated by higher retention of
soluble enzyme and residual activity following heat treatment at 50
°C. Our findings validate the effectiveness of SpyRing-mediated
cyclization in improving TEVp stability and highlight the utility
of computationally guided cyclization as a generalizable strategy
for engineering thermally resilient proteins. This study establishes
a framework that integrates structure prediction and rational protein
engineering, contributing to the development of robust biocatalysts
for synthetic biology and industrial biotechnology applications.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332596_ao5c05386_0007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/925b/12332596/fc83f4b9aa89/ao5c05386_0007.jpg" loading="lazy" id="abs1" height="435" width="820" alt="graphic file with name ao5c05386_0007.jpg"></a></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332596_ao5c05386_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/925b/12332596/fc83f4b9aa89/ao5c05386_0005.jpg" loading="lazy" id="tgr1" height="435" width="820" alt="graphic file with name ao5c05386_0005.jpg"></a></p></section><section id="sec1"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Recent advances in biotechnology
and bioengineering have dramatically
improved the efficiency of recombinant protein production. In particular,
the rise of synthetic biology has enabled modular and rational design
of biomolecular systems, allowing for efficient production and engineering
of recombinant proteins. In parallel, the addition of peptide tags
or fusion partners to target proteins has become a standard approach
for construct design, enabling efficient purification by affinity
chromatography. To obtain recombinant
proteins in a near-native form, it is necessary to remove these tags
using specific proteases. Among them, the Tobacco Etch Virus protease
(TEVp), which exhibits high substrate specificity, is commonly employed
for tag removal.<em>
,
</em> However, wild-type TEVp suffers
from poor solubility, autolysis, and low yield, which have long hindered
its production and utility.</p>
<p>To address these issues, numerous
engineering strategies have been
explored for TEVp. For example, substitution at the S219 residue to
suppress autolysis has been investigated, and the S219V mutant has
been reported to exhibit approximately 100-fold higher stability compared
to the wild-type enzyme. Additionally,
directed evolution experiments have shown that introducing three additional
mutations (T17S, N68D, and I77V) into the parental S219N mutant enhances
protein yield by approximately 5-fold. Furthermore, a C-terminal deletion mutant lacking residues 238–242
(Δ238–242) has been demonstrated to improve solubility
while retaining catalytic activity comparable to the wild-type. Other studies have reported that the L56V and
S135G mutations further enhance solubility,<em>
,
</em> and
that combining these mutations (L56V, S135G, S219V, and Δ238–242)
enables large-scale production of TEVp. Moreover, incorporating additional mutations (T17S, N68D, and I77V)
into this background has been shown to improve solubility, stability,
and catalytic activity, leading to a substantial increase in yield. More recently, AI-assisted rational design has
been employed to generate next-generation TEVp variants with enhanced
expression, stability, and catalytic activity. Although this extensive engineering, which involves substitutions
at more than 15% of the total amino acid residues, has significantly
improved catalytic activity and thermostability, these variants have
only been used in limited applications and require further evaluation
for enzymatic properties including stringent substrate specificity.</p>
<p>In this study, we aimed to further enhance the stability of TEVp
by introducing a cyclization strategy using the SpyRing method into
the TEVp variant containing the L56V, S135G, S219V mutations and the
C-terminal deletion Δ238–242, which is commonly used due to its improved solubility and activity.
Here, we refer to this variant as the standard TEVp. The SpyRing technique
employs the self-assembling SpyTag (ST) and SpyCatcher (SC) system,
consisting of a peptide and a small protein domain, widely used as orthogonal biological parts in synthetic
biology. By genetically fusing these elements to the N- and C-termini
of a target protein, we further enable the spontaneous and irreversible
formation of an intramolecular covalent bond, resulting in protein
cyclization. While previous reports have
suggested that SpyRing-mediated cyclization may not be suitable for
proteins with N- and C-terminal distances exceeding 15 Å, we selected TEVp as a challenging model system,
given that its termini are separated by approximately 40 Å in
the native structure. Structural models
of cyclized TEVp (cTEVp) generated using the SpyRing system are shown
in <a href="#fig1" class="usa-link">Figure 
</a>. The major
advantage of this approach is that covalent cyclization helps preserve
the overall topology of the protein, thereby promoting efficient refolding
and recovery of activity after thermal stress.<em>
,
</em> Moreover, this system is highly convenient compared to conventional
methods for the cyclization, as it does not require additional reagents
or enzymes and functions both in vivo and in vitro. For example, significant
improvements in thermostability have been reported when the SpyRing
system was applied to various enzymes, including β-lactamase, phytase, xylanase, and luciferase. Notably, several of these enzymes, such as phytase and xylanase,
are widely used in industrial applications, including animal feed
supplementation and biomass degradation processes. Beyond industrial
applications, the ST/SC system has also been explored in diverse biomedical
applications, including vaccine design,
drug delivery systems, and cell surface display technologies, underscoring
its versatility across both industrial and therapeutic fields.</p>
<figure class="fig xbox font-sm" id="fig1"><h3 class="obj_head">1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/925b/12332596/2b56f500b665/ao5c05386_0001.jpg" loading="lazy" id="gr1" height="693" width="666" alt="1"></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Structural
models of four cTEVp designs generated using AlphaFold.
Representative models from multiple linker variants are shown. (A)
Version 1the initial model obtained by directly attaching
the SpyTag (ST), SpyCatcher (SC), and the linker sequences reported
in ref  to the
N- and C-termini of TEVp. (B) Version 2generated by deleting
30 amino acid residues from the version 1 linker; this model was selected
as the final design used in this study. (C) Version 3generated
by deleting a further 17 residues from the version 2 linker. (D) Version
4generated by deleting one additional residue from the version
3 linker. In this model, the interaction between ST and SC was predicted
to be structurally disrupted. Therefore, cyclization of TEVp is unlikely
to occur. In every panel, TEVp is shown in cyan, SC in green, ST in
yellow, and the linker in magenta. The TEVp substrate (GENLYFQG) is
shown in pale cyan, and the isopeptide bond (when present) is shown
in red. N-terminal His-tags and C-terminal Strep-tags are omitted
for clarity.</p></figcaption></figure><p>In this paper, we constructed cTEVp by applying
the SpyRing method
to the standard TEVp (L56V, S135G, S219V, Δ238–242),
a widely used conventional mutant with improved solubility and activity, and evaluated its biochemical properties. Leveraging
AlphaFold structure prediction, we rationally
designed a cTEVp by optimizing the spatial arrangement of ST and SC
fused to the termini, despite the ∼40-Å separation between
the N- and C-termini of the native form. We demonstrate that cTEVp
retains proteolytic activity comparable to that of the standard TEVp
while exhibiting improved thermostability. These results validate
the applicability of the SpyRing method to TEVp and showcase a design-driven
approach that integrates computational modeling with protein engineering,
providing a versatile strategy for enhancing protein stability in
synthetic biology. Although SpyRing-mediated cyclization itself is
a well-established strategy for enhancing protein stability, this
study presents the first successful application to TEVp, a widely
used but thermolabile protease with an unusually large (∼40
Å) separation between its N- and C-termini. Our AlphaFold-guided
approach broadens the scope of SpyRing applicability and provides
a practical framework for engineering thermostable enzymes using genetically
encodable elements.</p></section><section id="sec2"><h2 class="pmc_sec_title">2. Results</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">2.1. Structural Design and Optimization of cTEVp</h3>
<p>Recent advances in artificial intelligence (AI) technology have
enabled highly accurate prediction of protein three-dimensional structures
based on amino acid sequences. In this study, we utilized AlphaFold
3 to optimize the design of cTEVp using
the SpyRing system. Initially, we constructed a model (cTEVp-ver.1)
by attaching the ST, SC, and linker sequences reported in ref . directly to the N- and
C-termini of TEVp (<a href="#fig1" class="usa-link">Figures 
</a>A and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c05386/suppl_file/ao5c05386_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S1A</a>). In this model, the
30-residue linker between TEVp and SC appeared excessively flexible
in the AlphaFold-predicted structure, with the corresponding region
exhibiting very low pLDDT scores (pLDDT &lt; 50), indicating reduced
confidence in its structural order (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c05386/suppl_file/ao5c05386_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S2A</a>). These features suggest that such a linker length may not be ideal
for effective stabilization via cyclization.</p>
<p>Therefore, we iteratively
revised the design by shortening and adjusting the linker region based
on the AlphaFold-predicted structures. In total, we evaluated over
ten linker variants with varying lengths and amino acid compositions,
using AlphaFold to assess the feasibility of cyclization based on
the spatial proximity and orientation of the fused ST and SC domains,
although only four representative designs (ver.1–ver.4) shown
in <a href="#fig1" class="usa-link">Figure 
</a> are selected
to explain the linker optimization process. All models were evaluated
by considering structural features such as linker flexibility (pLDDT
scores), domain alignment, and steric feasibility as follows. ver.2
was generated by removing 30 residues from the linker of ver.1 (<a href="#fig1" class="usa-link">Figures 
</a>B and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c05386/suppl_file/ao5c05386_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S1B</a>). In this model, ST and SC were brought into
proximity and formed a complex, while maintaining sufficient flexibility
and spacing from the TEVp termini to permit cyclization. The linker
region in ver.2 showed pLDDT scores below 50, consistent with a flexible,
disordered segment that nonetheless allowed favorable positioning
of ST and SC for covalent bond formation. We also examined variants
with slightly longer linkers than that of ver.2 (data not shown).
In these models, ST and SC remained properly aligned for interaction,
but no substantial improvement in predicted geometry was observed.
To achieve further compaction, ver.3 was designed by removing an additional
17 residues from ver.2 (<a href="#fig1" class="usa-link">Figures 
</a>C and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c05386/suppl_file/ao5c05386_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S1C</a>). While intramolecular
cyclization was still predicted by AlphaFold, the overall configuration
appeared more constrained, with reduced spacing between TEVp and the
fused domains. Although Ramachandran analysis did not reveal significant
backbone strain, the shorter linker may have reduced the flexibility
at the termini, potentially affecting proper folding or positioning
for cyclization. The linker pLDDT scores in ver.3 were generally in
the range of 50–70, indicating a modest increase in confidence
and partial ordering, possibly reflecting reduced flexibility. Experimental
evaluation of ver.3 supported this interpretation: the purified protein
exhibited reduced cyclization efficiency, increased oligomer formation,
and decreased thermostability compared to ver.2 (data not shown).
Finally, in ver.4 (<a href="#fig1" class="usa-link">Figures 
</a>D and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c05386/suppl_file/ao5c05386_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">S1D</a>), where one more residue
was removed from ver.3, the linker became too short, and cyclization
was structurally impeded. Consistent with this, the linker region
in ver.4 showed low pLDDT scores (&lt;50), indicating insufficient
spacing and flexibility to permit productive cyclization. These results
suggest that the linker in ver.2 is close to the minimal length required
to support effective intramolecular cyclization while avoiding steric
strain.</p>
<p>Thus, ver.2 was selected as the most suitable design
among the
predicted constructs in terms of both structural feasibility and predicted
stability, and all subsequent experiments were performed using this
construct. Notably, AlphaFold modeling indicated that although the
N- and C-termini of TEVp are separated by approximately 40 Å
in the native structure, the fused ST and SC domains were positioned
favorably, enabling successful intramolecular cyclization (<a href="#fig1" class="usa-link">Figure 
</a>B).</p></section><section id="sec2.2"><h3 class="pmc_sec_title">2.2. Protein Expression and Purification</h3>
<p>In this study, we constructed a new high-copy-number expression vector,
pHIT184 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c05386/suppl_file/ao5c05386_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1, Figure S3</a>), by combining
the expression cassette of a T7 promoter-based system with a pUC-derived
replication origin. This vector, which retains the IPTG-inducible
T7 promoter and <em>lacI</em>/<em>lacO</em> regulatory
elements, is advantageous for genetic manipulation and enables high-level
protein expression of the inserted gene in . We subsequently evaluated its utility by expressing
several TEVp variants and substrate proteins, obtaining sufficient
amounts of recombinant proteins from small-scale (100 mL) cultures.
Our system yielded 94 mg/L of purified protein, which is higher than
the previously reported yield of TEVp (50 mg/L). Since the amino acid sequence of the enzymatic region is
identical in both proteins (except for the addition of 11 residues,
including a C-terminal Strep-tag, in our construct), this result confirms
the high efficiency of our expression and purification system. Notably,
both the TEVp and substrate genes used in this study were synthesized
as codon-optimized sequences for , thereby allowing high expression without the need for additional
plasmids supplementing rare-codon tRNAs (e.g., pRIL). Furthermore,
TEVp produced by this system has been routinely used in our laboratory
for the removal of affinity tags from various recombinant proteins
without any issues, demonstrating its practical applicability.
</p>
<p>In addition to the standard TEVp described
above, two other TEVp variantscTEVp based on the ver.2 construct
and noncyclized TEVp (ncTEVp) having an Asp-to-Ala mutation to inhibit
the cyclization (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c05386/suppl_file/ao5c05386_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Tables S2 and S3</a>)were
similarly expressed in and
purified to high purity using affinity chromatography. Initially,
cTEVp was expressed by fusing only ST and SC to its termini; however,
the expression level was extremely low, making it difficult to obtain
sufficient amounts of purified protein (data not shown). Therefore,
similar to TEVp, maltose-binding protein (MBP) was fused to the N-terminus
of cTEVp, which significantly improved its expression level. Subsequently,
using this MBP-fused cTEVp as a template, we introduced the Asp-to-Ala
mutation to generate the MBP-fused ncTEVp. Both enzymes underwent
autocleavage by TEVp activity in , resulting in the removal of the MBP fusion, and were ultimately
isolated in a cleaved form. These results confirmed that both cTEVp
and ncTEVp, like the standard TEVp, efficiently cleave large fusion
partners, such as MBP, in vivo. Finally, ST/SC-mediated cyclization
is expected to occur only after MBP has been removed from cTEVp. Using
the same pHIT184 expression system, we also prepared three substrate
proteins: a monomeric variant of the T7 phage Ocr protein (Mocr), T4 lysozyme (T4L), and chitin-binding domain (ChBD). T4L
is a well-established fusion partner widely used in structural biology,
whereas Mocr has been reported to enhance the solubility of fusion
proteins and is compatible with structural studies. ChBD, derived
from a thermostable archaeal chitinase, offers high thermal resistance
suitable for heat stress assays. Details of all purification results
are summarized in <a href="#tbl1" class="usa-link">Table 
</a>.</p>
<section class="tw xbox font-sm" id="tbl1"><h4 class="obj_head">1. Recombinant Proteins Expressed in
This Study.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">protein name</th>
<th align="center" colspan="1" rowspan="1">expression
temperature (°C)</th>
<th align="center" colspan="1" rowspan="1">induction
time (h)</th>
<th align="center" colspan="1" rowspan="1">yield (mg)<sup></sup>
</th>
<th align="center" colspan="1" rowspan="1">theoretical
molecular weight<sup></sup>
</th>
<th align="center" colspan="1" rowspan="1">reference</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">TEVp</td>
<td align="left" colspan="1" rowspan="1">25</td>
<td align="left" colspan="1" rowspan="1">28</td>
<td align="center" colspan="1" rowspan="1">9.4<sup></sup>
</td>
<td align="center" colspan="1" rowspan="1">29061.89</td>
<td align="left" colspan="1" rowspan="1">this study<sup></sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">cTEVp</td>
<td align="left" colspan="1" rowspan="1">25</td>
<td align="left" colspan="1" rowspan="1">24</td>
<td align="center" colspan="1" rowspan="1">3.8<sup></sup>
</td>
<td align="center" colspan="1" rowspan="1">39857.95</td>
<td align="left" colspan="1" rowspan="1">this study</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ncTEVp</td>
<td align="left" colspan="1" rowspan="1">25</td>
<td align="left" colspan="1" rowspan="1">24</td>
<td align="center" colspan="1" rowspan="1">0.8<sup></sup>
</td>
<td align="center" colspan="1" rowspan="1">39813.94</td>
<td align="left" colspan="1" rowspan="1">this study</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Mocr</td>
<td align="left" colspan="1" rowspan="1">37</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="center" colspan="1" rowspan="1">9.2</td>
<td align="center" colspan="1" rowspan="1">18141.49</td>
<td align="left" colspan="1" rowspan="1">

</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">T4L</td>
<td align="left" colspan="1" rowspan="1">25</td>
<td align="left" colspan="1" rowspan="1">18</td>
<td align="center" colspan="1" rowspan="1">5.7</td>
<td align="center" colspan="1" rowspan="1">22495.44</td>
<td align="left" colspan="1" rowspan="1">

</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ChBD</td>
<td align="left" colspan="1" rowspan="1">37</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="center" colspan="1" rowspan="1">16.6</td>
<td align="center" colspan="1" rowspan="1">16459.96</td>
<td align="left" colspan="1" rowspan="1">

</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="t1fn1">
<sup>a</sup><p class="display-inline">Yield per 100 mL of culture medium.</p>
</div>
<div class="fn" id="t1fn2">
<sup>b</sup><p class="display-inline">TEVp, cTEVp, and ncTEVp were
expressed
as MBP fusion proteins and cleaved intracellularly by their own TEVp
activity; the values shown here are calculated after MBP removal.</p>
</div>
<div class="fn" id="t1fn3">
<sup>c</sup><p class="display-inline">Theoretical molecular weight
was
calculated using the Compute pI/Mw tool available at Expasy (<a href="https://web.expasy.org/compute_pi/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://web.expasy.org/compute_pi/</a>).</p>
</div>
<div class="fn" id="t1fn4">
<sup>d</sup><p class="display-inline">Differences between
this study and
ref  are shown in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c05386/suppl_file/ao5c05386_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S3</a>.</p>
</div>
</div></section><p>SDS–PAGE analysis (<a href="#fig2" class="usa-link">Figure 
</a>A) showed that all three TEVp variants were
purified
(lanes 3, 7, and 11) and that the major bands of TEVp and ncTEVp corresponded
well with their theoretical molecular weights (<a href="#tbl1" class="usa-link">Table 
</a>). In contrast, the cTEVp band shifted to
a lower molecular weight compared to ncTEVp, although the only difference
between them is a single residue substitution (<a href="#fig2" class="usa-link">Figure 
</a>A, lane 7). This shift is likely due to altered
electrophoretic mobility caused by the cyclized structure, as has
been reported for other SpyRing enzymes.<em>
,,
</em> Moreover, several ladder bands were detected
at higher-molecular-weight positions for cTEVp, likely due to intermolecular
ST-SC cross-linking, a phenomenon commonly seen with other SpyRing
enzymes.<em>
,,
</em>
</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/925b/12332596/0947f605a51f/ao5c05386_0002.jpg" loading="lazy" id="gr2" height="717" width="563" alt="2"></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>SDS–PAGE
analysis of purified enzymes and substrate proteins,
and quantitative analysis of substrate cleavage activity. (A) SDS–PAGE
analysis using Mocr as the substrate. Lane 1: Molecular weight markers.
Lanes 2 and 15: substrate only control (Mocr, 100 nmol); Lanes 3,
7, and 11: enzyme only (100 × 2<sup>–3</sup> nmol), without
substrate; Lanes 4, 8, and 12 enzyme (100 × 2<sup>–5</sup> nmol) + substrate (100 nmol); Lanes 5, 9, and 13: enzyme (100 ×
2<sup>–7</sup> nmol) + substrate (100 nmol); Lanes 6, 10, and
14: enzyme (100 × 2<sup>–9</sup> nmol) + substrate (100
nmol). Enzyme and substrate were mixed and incubated at 37 °C
for 60 min prior to electrophoresis. (B) Quantitative analysis of
substrate cleavage rates. Cleavage rates (%) were calculated based
on the integrated intensity of the uncleaved Mocr bands. The initial
band intensity (uncleaved substrate) was set to 100%, and cleavage
rate was determined by subtracting the intensity of remaining substrate
after the reaction. Data represent mean ± SD from three independent
experiments.</p></figcaption></figure></section><section id="sec2.3"><h3 class="pmc_sec_title">2.3. Evaluation of Substrate Cleavage Activity
of cTEVp</h3>
<p>To evaluate the enzymatic activity of cTEVp, we
employed three model substratesMocr, T4L, and ChBDeach
containing a TEVp cleavage site (TEVcs) between an N-terminal H<sub>6</sub>-tag and the substrate protein. These substrates, described
in the previous section, enabled comparative analysis of proteolytic
activity under standard and heat stress conditions.</p>
<p>First, we
compared the substrate cleavage activity of cTEVp with that of ncTEVp
and the standard TEVp using Mocr as the substrate (<a href="#fig2" class="usa-link">Figure 
</a>). SDS–PAGE analysis
(<a href="#fig2" class="usa-link">Figure 
</a>A) was performed
to assess the cleavage reaction after incubating enzyme and substrate
mixtures at various enzyme-to-substrate molar ratios (1:32 to 1:512)
at 37 °C for 1 h. Under a 1:128 enzyme-to-substrate molar ratio,
which approximates the commonly used standard reaction condition for
TEVp (1:100), all three enzymes cleaved more than 50% of the substrate
(corresponding to lanes 5, 9, and 13).</p>
<p>Quantitative analysis
of the substrate cleavage rates (<a href="#fig2" class="usa-link">Figure 
</a>B) showed that cTEVp,
TEVp, and ncTEVp all exhibited high cleavage activities. At a 1:128
molar ratio, cTEVp, TEVp, and ncTEVp achieved cleavage rates of 77
± 24, 62 ± 18, and 60 ± 8%, respectively. No statistically
significant differences were observed among the three enzymes (<em>p</em> = 0.44 for TEVp and <em>p</em> = 0.30 for ncTEVp,
both compared to cTEVp).</p>
<p>These results demonstrate that cTEVp,
cyclized via the SpyRing
system, exhibits comparable substrate cleavage activity to both TEVp
and ncTEVp. Thus, despite the structural constraint imposed by cyclization,
cTEVp retained practical cleavage activity equivalent to that of the
standard TEVp.</p></section><section id="sec2.4"><h3 class="pmc_sec_title">2.4. Effect of SpyRing-Mediated Cyclization on
the Thermostability of TEVp</h3>
<p>To assess thermostability, we
used T4 lysozyme (T4L) as a model substrate, which allowed simultaneous
visualization of both enzyme and substrate on SDS–PAGE due
to its relatively slow cleavage kinetics. This enabled quantitative
evaluation of both the soluble enzyme fraction and residual protease
activity after heat treatment.</p>
<p>To evaluate the thermostability
of TEVp, ncTEVp, and cTEVp, we analyzed the amount of enzyme remaining
in the soluble fraction and the residual protease activity after heat
treatment at 50 °C for 0–32 min (<a href="#fig3" class="usa-link">Figure 
</a>). Upon heat treatment, TEVp and ncTEVp rapidly
lost their soluble fraction, with almost no detectable soluble protein
remaining after 8 min (<a href="#fig3" class="usa-link">Figure 
</a>B). In contrast, cTEVp retained a higher proportion of its
soluble fraction, with 31.6% remaining after 8 min, compared to 4.9
and 3.6% for TEVp and ncTEVp, respectively. These differences were
statistically significant (<em>p</em> = 0.0057 and <em>p</em> = 0.0048, respectively, relative to cTEVp).</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/925b/12332596/ea0e7417e15a/ao5c05386_0003.jpg" loading="lazy" id="gr3" height="3439" width="3500" alt="3"></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Thermostability
of TEVp variants: time-course analysis of soluble
fraction and residual activity after heat treatment. (A) SDS–PAGE
analysis of soluble enzyme fractions and substrate cleavage products.
Enzymes (ncTEVp, cTEVp, and TEVp) were incubated at 50 °C for
the indicated durations (0, 1, 2, 4, 8, 16, or 32 min), followed by
centrifugation to separate soluble fractions. These soluble fractions
were incubated with equimolar amounts of T4L substrate at 37 °C
for 3 h. The reaction mixtures were analyzed by SDS–PAGE to
assess the amount of soluble enzyme and residual enzymatic activity.
Bands corresponding to enzyme variants (ncTEVp, cTEVp, and TEVp),
uncleaved and cleaved T4L, are indicated. (B) Quantification of soluble
enzyme fraction after heat treatment. Bar graphs and line plots show
the percentage of soluble enzyme remaining in the soluble fraction
after incubation at 50 °C for the indicated durations. Band intensities
were quantified from SDS–PAGE images in (A), with the 0 min
time point set to 100%. (C) Quantification of residual enzymatic activity.
Residual activity was assessed by measuring the cleavage of the T4L
substrate by the heat-treated soluble enzyme fractions. Product band
intensities were quantified and normalized to the 0 min control (set
as 100%). Data represent means ± SD from multiple independent
experiments (<em>n</em> = 4 for cTEVp, <em>n</em> =
3 for ncTEVp and TEVp). Asterisks indicate statistically significant
differences relative to cTEVp at the same time point (*<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001, ****<em>p</em> &lt; 0.0001; see <a href="#sec4" class="usa-link">Section 
</a> for details).</p></figcaption></figure><p>We next evaluated the proteolytic activity of the
heat-treated
soluble enzyme fractions using a substrate cleavage assay at 37 °C
(<a href="#fig3" class="usa-link">Figure 
</a>C). The time-dependent
decline in residual activity, indicated by cleaved bands of T4L substrate,
correlated well with the remaining soluble fraction, and cTEVp consistently
retained substantially higher activity throughout the time course.
For heat treatment at multiple time points, particularly at 8 min
and beyond, the residual activity of cTEVp was significantly higher
than that of TEVp and ncTEVp. For example, after 8 min at 50 °C,
cTEVp retained 24.8% of its activity, whereas TEVp and ncTEVp retained
only 6.4 and 7.9%, respectively (<em>p</em> = 0.00003 and <em>p</em> = 0.00001).</p>
<p>These results demonstrate that while
TEVp and ncTEVp exhibit similar
thermostability profiles, the SpyRing-mediated cyclization of TEVp
consistently enhanced resistance to heat-induced loss of soluble protein
and enzymatic activity across multiple time points. The pronounced
differences observed particularly at later time points underscore
the effectiveness of covalent cyclization in maintaining enzymatic
function under thermal stress.</p></section><section id="sec2.5"><h3 class="pmc_sec_title">2.5. Activity Profiling of cTEVp at Moderate and
Elevated Temperatures</h3>
<p>To evaluate the impact of SpyRing-mediated
cyclization on the thermal robustness of TEVp activity, we first compared
the catalytic efficiencies of TEVp, cTEVp, and ncTEVp using T4 lysozyme
(T4L) as a standard substrate at 30 and 37 °C. While previous
reports have suggested that TEVp exhibits optimal activity at 30 °C, our results showed that all three variants maintained
comparable activity at both temperatures (<a href="#fig4" class="usa-link">Figure 
</a>A, left panel). Based on these results, enzyme
amounts were titrated at 30 and 37 °C for subsequent comparisons
under heat stress conditions.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332596_ao5c05386_0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/925b/12332596/553632cbefb7/ao5c05386_0004.jpg" loading="lazy" id="gr4" height="873" width="776" alt="4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparative analysis of TEVp variants for proteolytic
activity
at moderate and elevated temperatures. (A) SDS–PAGE analysis
of substrate cleavage by TEVp (std), cTEVp (c), and ncTEVp (nc) using
T4 lysozyme (T4L) and chitin-binding domain (ChBD) as substrates.
For the T4L assays (left panel), enzymes and substrate were incubated
at 30 or 37 °C for 3 h. All three TEVp variants exhibited comparable
cleavage activity at both temperatures, supporting their use in normalized
comparisons. For the ChBD assays (right panel), cleavage reactions
were conducted at 30 or 50 °C using enzyme amounts that were
adjusted to yield the same cleavage activity based on the T4L results.
At 50 °C, cTEVp showed visibly stronger cleavage product bands
than either TEVp (std) or ncTEVp, indicating superior retention of
catalytic activity under heat stress. Substrate-only controls are
shown at the far left (T4L) and far right (ChBD). Molecular weight
markers (M) are included in the center. (B) Quantification of substrate
cleavage efficiency from (A). For T4L (left graph), the intensity
of the cleaved band for the condition with TEVp at 30 °C was
set to 100%, and all other values were normalized accordingly. For
ChBD (right graph), the cleaved band intensity for the condition with
TEVp at 30 °C was likewise set as 100%. Data represent mean ±
SD from three independent experiments. At 50 °C, proteolytic
activities of TEVp and ncTEVp were significantly lower than that of
cTEVp (<em>p</em> &lt; 0.01 and <em>p</em> &lt; 0.001,
respectively; unpaired two-tailed Student’s <em>t</em> test vs cTEVp).</p></figcaption></figure><p>Next, to assess enzymatic activity under heat stress,
the same
titrated enzyme amounts (adjusted based on T4L cleavage activity)
were applied to cleavage reactions using the thermostable ChBD substrate
at 50 °C for 30 min. Because T4L proved unstable at 50 °C,
the thermostable ChBD substrate was used instead for comparative analysis
under heat stress conditions.</p>
<p>At 50 °C, cTEVp retained
significantly higher proteolytic
activity (96.4 ± 10.7%) than both TEVp (40.4 ± 13.5%, <em>p</em> = 0.0049) and ncTEVp (34.1 ± 1.5%, <em>p</em> = 0.00056; <em>n</em> = 3), as shown in <a href="#fig4" class="usa-link">Figure 
</a>A (right panel) and quantitatively
summarized in <a href="#fig4" class="usa-link">Figure 
</a>B. These results demonstrate that SpyRing-mediated cyclization enhances
the catalytic function of TEVp under elevated temperatures that substantially
impair the activity of noncyclized variants.</p></section></section><section id="sec3"><h2 class="pmc_sec_title">3. Discussion</h2>
<p>In this study, we successfully
generated a cTEVp using the SpyRing
system and characterized its properties. The cTEVp retained substrate
cleavage activity comparable to both ncTEVp and standard TEVp (<a href="#fig2" class="usa-link">Figure 
</a>), demonstrating
that structural stabilization via ST-SC-mediated cyclization did not
adversely affect the enzymatic activity.</p>
<p>In contrast, comparative
analysis of thermostability revealed that
cTEVp consistently retained a higher proportion of soluble enzyme
and residual activity than both TEVp and ncTEVp throughout the heat
treatment (<a href="#fig3" class="usa-link">Figure 
</a>). These findings indicate that cyclization via the SpyRing system
stabilizes the molecular structure of TEVp and confers resistance
to heat-induced inactivation. The observed loss of soluble enzyme
and activity in TEVp and ncTEVp upon heat treatment is likely due
to heat-induced aggregation, although the contribution of partial
denaturation cannot be entirely excluded.</p>
<p>Although ncTEVp retains
the ST and SC domains at its N- and C-termini,
it was designed to prevent isopeptide bond formation by introducing
a point mutation in ST (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c05386/suppl_file/ao5c05386_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S2</a>), thereby
eliminating covalent cyclization (<a href="#fig1" class="usa-link">Figure 
</a>). Structural prediction using AlphaFold
suggested that ST and SC remain associated in ncTEVp despite the absence
of isopeptide bond formation, indicating that cTEVp and ncTEVp are
structurally highly similar. Importantly, ncTEVp served as a critical
negative control in our experimental design. By retaining the same
fused domains but lacking the covalent bond, it allowed us to specifically
isolate the contribution of intramolecular cyclization to thermal
stabilization, eliminating potential confounding effects arising from
domain fusion or molecular crowding. Furthermore, TEVp and ncTEVp
exhibited comparable behaviors in terms of their remaining soluble
fractions and proteolytic activities, both serving as indicators of
thermostability (<a href="#fig2" class="usa-link">Figures 
</a> and ). These observations suggest
that the cyclized structure itselfspecifically, the isopeptide
bond between ST and SCis the primary factor contributing to
the enhanced stability of TEVp. Thus, the inclusion of ncTEVp reinforces
the internal logic of our study and validates that the observed enhancement
in thermostability is attributed to the formation of the intramolecular
isopeptide bond rather than to the mere presence of terminal fusion
domains. This finding aligns with previous reports showing that SpyRing-mediated
cyclization enhances the thermal resilience of other model enzymes,
including β-lactamase and phytase.
</p>
<p>Notably, the TEVp variant used in this
study already incorporates
the L56V and S135G mutations found in the TEVp<sup>2M</sup> construct,
which has been reported to improve thermal tolerance above 40 °C. Despite the presence of these stabilizing mutations,
our noncyclized variants (TEVp and ncTEVp) still showed substantial
aggregation and loss of activity upon incubation at 50 °C, indicating
that these mutations alone are insufficient to fully protect the enzyme
under thermal stress (<a href="#fig3" class="usa-link">Figure 
</a>). This underscores the additive effect of SpyRing-mediated
cyclization in further enhancing thermal resilience, beyond what is
achievable through conventional mutagenesis. The improved performance
of cTEVp is likely attributable to suppression of heat-induced aggregation,
rather than promotion of reversible unfolding. This interpretation
is supported by the observation that both TEVp and ncTEVp rapidly
lost solubility and activity upon heat treatment, consistent with
irreversible aggregation. In contrast, cTEVp retained a significantly
higher proportion of soluble protein and residual activity, suggesting
that SpyRing-mediated cyclization prevents the onset of misfolding
or aggregation, rather than facilitating refolding after denaturation.
TEVp is known to lose activity irreversibly once aggregation occurs, in contrast to some other enzymes such as β-lactamase,
in which cyclization confers the ability to refold after thermal denaturation. These differences highlight that the primary
benefit of SpyRing-mediated cyclization in the case of TEVp lies in
its ability to prevent irreversible inactivation via aggregation.</p>
<p>According to the crystal structure of TEVp (PDB entry: 1LVB), the distance between the N- and C-termini is
approximately 40 Å, which is relatively long. Interestingly,
despite this separation, AlphaFold modeling predicted that the fused
ST and SC domains would be positioned favorably for intramolecular
cyclization. This suggests that the flexibility and orientation of
these domains can compensate for the relatively long native terminal
distance, facilitating successful cyclization. Although it has been
suggested that proteins with terminal distances exceeding 15 Å
may not be suitable for SpyRing-mediated cyclization, the present findings challenge this assumption. Indeed,
while phytase (with a terminal distance of 29 Å) has been reported
to exhibit significant thermostabilization via the SpyRing system, the stabilization effect on TEVp appeared to
be more modest. A recent study has proposed
that interfacial interactions between ST/SC and the target enzyme
may contribute to thermal stabilization, indicating that further optimization
of these interfaces could enhance the stabilizing effect. To clarify
these possibilities, we need to conduct high-resolution structural
analyses, such as X-ray crystallography, of proteins cyclized with
ST/SC in future studies.</p>
<p>Notably, all AlphaFold models constructed
in this study included
not only the enzyme domain but also a short substrate peptide modeled
as a separate chain within the substrate-binding site (<a href="#fig1" class="usa-link">Figure 
</a>). While minor inconsistenciessuch
as reversed orientation of the substrate peptidewere observed
in some models, the peptide was consistently located within the substrate-binding
pocket. These limitations did not affect the core prediction of successful
intramolecular complex formation. Beyond enzyme–substrate recognition,
AlphaFold 3 also accurately modeled the intramolecular association
of ST and SC, which are fused at the termini of a single polypeptide
chain (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c05386/suppl_file/ao5c05386_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S3</a>) yet must adopt a favorable
conformation to enable intramolecular complex formation (<a href="#fig1" class="usa-link">Figure 
</a>). Our results demonstrate
that AlphaFold successfully captured this association despite the
relatively long terminal distance of TEVp, suggesting that such predictive
capability can be generalized to the design of structurally constrained
proteins. This feature represents a significant advancement in computational
design, facilitating the rational engineering of proteins with enhanced
stability or novel topologies. In addition, AlphaFold modeling provided
sufficient structural insight to discriminate between suboptimal and
favorable constructs. For example, ver.1 showed a highly flexible
and disordered linker (low pLDDT scores &lt;50), suggesting suboptimal
spatial constraint and reduced potential for stabilization. In contrast,
ver.2 showed a more compact yet permissive geometry with ST and SC
domains brought into close proximity, supporting successful isopeptide
bond formation. These observations underscore the utility of AlphaFold
in predicting global topology, linker orientation, and steric feasibility
of SpyRing-mediated cyclization, even in the absence of atomic-level
resolution structural data. This approach readily enables rational
optimization of cyclization constructs and is particularly valuable
for proteins in which the N- and C-termini are separated by large
distances.</p>
<p>To evaluate the potential utility of the enzymes
under high-temperature
conditionssuch as those encountered in thermophilic organisms
or industrial bioprocesseswe assessed the proteolytic activity
of cTEVp at 50 °C using the thermostable substrate ChBD. Under
these conditions, cTEVp retained markedly higher catalytic activity
(96.4 ± 10.7%) compared to TEVp (40.4 ± 13.5%) and ncTEVp
(34.1 ± 1.5%) (<em>n</em> = 3), with statistically significant
differences (<em>p</em> = 0.0049 and 0.00056, respectively; <a href="#fig4" class="usa-link">Figure 
</a>A,B). These results
suggest that SpyRing-mediated cyclization not only confers structural
stability, but also directly improves enzymatic function under sustained
heat exposure. Although previous studies have shown that SpyRing-mediated
cyclization can facilitate refolding and recovery of enzyme activity
after thermal denaturation,<em>
,,
</em> our findings imply that it can enhance catalytic performance even
in proteases like TEVp that do not refold once aggregated. Given that
our TEVp construct includes the stabilizing L56V and S135G mutations
from TEVp<sup>2M</sup>,<em>
,
</em> the superior thermal performance
of cTEVp further highlights the additive benefit of cyclization. Further
biophysical analyses, including comparison with reversible enzymes
such as β-lactamase, may help elucidate
the mechanistic basis for this improved thermal functionality.</p>
<p>Collectively, our results support the utility of SpyRing-mediated
cyclization using discrete modular domains as a generalizable platform
for enhancing protein stability. By integrating structure-based design
via AlphaFold with genetically encodable cyclization domains, this
strategy enables the rational engineering of robust and thermostable
proteins with broad applicability in biocatalysis, therapeutic design,
and synthetic biological devices. The successful cyclization of TEVp,
despite its relatively long terminal separation, suggests that computational
modeling can effectively guide the design of cyclized proteins through
optimizing linker length or amino acid sequence, even in cases previously
considered unsuitable for SpyRing. This work not only advances protein
engineering but also exemplifies a synthetic biology framework for
designing structurally constrained, thermostable biocatalysts for
diverse industrial and biomedical applications.</p></section><section id="sec4"><h2 class="pmc_sec_title">4. Methods</h2>
<section id="sec4.1"><h3 class="pmc_sec_title">4.1. Design and Construction of Expression Plasmids</h3>
<p> strain DH5α was used
for plasmid preparation. cells
were grown aerobically at 37 °C in Luria–Bertani (LB)
medium (1% (w/v) polypeptone, 0.5% (w/v) yeast extract, and 0.5% (w/v)
NaCl). For the construction of expression plasmids, restriction enzyme-digested
fragments from existing plasmids, PCR products, or synthetic DNA were
assembled using T4 DNA Ligase (Takara) or NEBuilder (New England Biolabs).
Chemically synthesized oligonucleotides and double-stranded DNA fragments
were purchased from Eurofins Genomics or GeneArt Strings (Thermo Fisher
Scientific).</p>
<p>pHIT184 was constructed and used as an expression vector. This plasmid contains
the expression cassette from pET-11a (Merck Millipore) and the replication
origin from the pUC18 vector (GenBank: <a href="https://www.ncbi.nlm.nih.gov/nuccore/L08752.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">L08752.1</a>) (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c05386/suppl_file/ao5c05386_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1, Figure S3</a>). This plasmid retains the IPTG-inducible
T7 promoter, the <em>lacO</em> operator and the <em>lacI</em> gene, which suppresses background expression, similar to the pET-11a
vector. All plasmids used for protein production were subcloned into
this vector backbone. Various target genes were subcloned into <em>Nde</em>I/<em>Bam</em>HI or <em>Xba</em>I/<em>Bam</em>HI site downstream of the T7 promoter to generate expression
plasmids for the proteins listed in <a href="#tbl1" class="usa-link">Table 
</a>. The sequences of the constructed plasmids
were confirmed by Sanger sequencing to ensure correct assembly.</p>
<p>The TEVp variant used throughout this study (L56V, S135G, S219V,
Δ238–242), referred to as the standard TEVp in the main
text, has been widely employed due to
its improved solubility and activity. Although its expression and
purification have typically relied on the pDZ2087 plasmid, which requires coexpression of rare tRNA genes
(e.g., from pRIL) due to biased codon usage, we circumvented this
limitation by using a synthetic, codon-optimized TEVp gene for efficient
expression in .</p>
<p>First,
we constructed the plasmid pHIT184-MBP-TEVp for the expression
of TEVp (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c05386/suppl_file/ao5c05386_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S2</a>). The amino acid sequence
of the TEVp part in the recombinant fusion protein produced from this
plasmid is identical to that of TEVp produced using pDZ2087, except for an additional 11 residues at the C-terminus,
comprising a linker (ASA) and a Strep-tag (WSHPQFEK). The recombinant
protein is produced in as a
fusion protein with MBP derived from (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c05386/suppl_file/ao5c05386_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S2</a>), which promotes soluble expression. After expression, the TEVp undergoes in vivo
self-cleavage at TEVcs located between MBP and TEVp, yielding active TEVp bearing an N-terminal polyhistidine
tag (H<sub>7</sub>-tag) and a C-terminal Strep-tag.</p>
<p>For cyclized
protein expression using the SpyRing system, the N-terminus
and C-terminus of the target protein were fused with ST and SC, respectively.
However, these fusions alone did not enhance soluble expression. Therefore,
similar to the TEVp construct, cTEVp was expressed as a fusion protein
with MBP, and the expression plasmid pHIT184-MBP-cTEVp was constructed
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c05386/suppl_file/ao5c05386_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S2</a>). In this study, to facilitate
purification, all recombinant proteins were expressed with an N-terminal
polyhistidine tag (H<sub>6</sub>- or H<sub>7</sub>-tag) and a C-terminal
Strep tag. Additionally, we constructed an expression plasmid of noncyclized
TEVp (ncTEVp) (pHIT184-MBP-ncTEVp) incorporating the Asp-to-Ala (DA)
mutant, in which the aspartic acid residue
in ST essential for isopeptide bond formation is substituted with
alanine, to prepare a negative control that retains both ST and SC
but cannot form the covalent bond (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c05386/suppl_file/ao5c05386_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S2</a>).</p>
<p>As substrates for TEVp, we used Mocr and T4 lysozyme (T4L), and
the chitin-binding
domain (ChBD) from KOD1. All three constructs contained
a TEVcs between an N-terminal H<sub>6</sub>-tag and the substrate
protein (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c05386/suppl_file/ao5c05386_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Tables S2 and S3</a>).</p></section><section id="sec4.2"><h3 class="pmc_sec_title">4.2. Protein Expression and Purification</h3>
<p> C41­(DE3) cells harboring each
expression plasmid (see <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c05386/suppl_file/ao5c05386_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S2</a>) were
cultured in a modified Studier autoinduction medium containing 2% polypeptone, 0.5% yeast extract, 0.5% NaCl,
0.6% Na<sub>2</sub>HPO<sub>4</sub>, 0.3% KH<sub>2</sub>PO<sub>4</sub>, 0.5% glycerol, 0.05% glucose, 0.2% lactose, supplemented with 100
μg/mL ampicillin. Cells were grown at 37 °C with shaking
at 180 rpm until an optical density of ∼0.1 was reached. At
this point, the temperature was reduced according to the conditions
optimized for each protein (see <a href="#tbl1" class="usa-link">Table 
</a>). The cultures were further incubated, allowing protein
expression to be automatically induced upon glucose depletion and
lactose metabolism. Cells were harvested by centrifugation at 6000
g for 10 min and stored at −20 °C until use.</p>
<p>Frozen
cell pellets (∼3 g wet weight per 100 mL culture) were thawed,
resuspended in phosphate-buffered saline (PBS; 10 mM sodium phosphate,
150 mM NaCl, pH 7.4), and supplemented with hen egg white lysozyme
(final concentration: 1 mg/mL) and Triton X-100 (final concentration:
1% (v/v)). The suspension was incubated at 37 °C for 10 min to
facilitate cell lysis. After incubation, a 2% (w/v) solution of streptomycin
sulfate in PBS was added to a final concentration of 1% (w/v), and
the mixture was gently inverted to ensure thorough mixing. The suspension
was then placed on ice for 5 min to allow precipitation of nucleic
acids. The lysate was subsequently centrifuged at 20,000 g for 30
min at 4 °C.</p>
<p>The resulting supernatant was loaded onto
a Ni-affinity column
packed with cOmplete His-Tag Purification Resin (Roche), pre-equilibrated
with buffer A (25 mM Tris–HCl (pH 8.0), 500 mM NaCl, and 10%
(w/v) glycerol). After washing the column with five bed volumes of
buffer A, the bound protein was eluted using buffer A containing 500
mM imidazole. The eluted fractions were collected and loaded onto
a Strep-Tactin Superflow High-Capacity Column (1 mL; IBA), pre-equilibrated
with buffer A. The column was washed with 10 mL of buffer A, and the
bound protein was eluted with buffer A containing 1 mM <em>d</em>-desthiobiotin. Fractions containing the target protein were pooled,
aliquoted as needed, and stored at −80 °C. The purified
protein was analyzed by SDS–PAGE using an Any kD denaturing
polyacrylamide gel (Bio-Rad). The protein concentration was determined
using the Bradford method with Protein Assay CBB Solution (5 ×
concentrate) (Nacalai Tesque), using bovine serum albumin as the standard.</p></section><section id="sec4.3"><h3 class="pmc_sec_title">4.3. Protease Activity Assay</h3>
<p>Protease
activity was evaluated by assessing the cleavage of three substrates
(Mocr, T4L, and ChBD), each containing a TEVp cleavage site downstream
of the His tag. The activity of cTEVp was compared to that of the
standard TEVp (without ST and SC) and the ncTEVp, which carries a
mutation in the ST sequence preventing cyclization.</p>
<p>The cleavage
reactions were performed using Mocr as the substrate. Enzyme and substrate
were mixed in a reaction buffer containing 50 mM Tris–HCl (pH
8.0), 0.5 mM EDTA, and 1 mM dithiothreitol (DTT), and incubated at
37 °C for 60 min. The substrate concentration was fixed at 100
μM, and enzyme-to-substrate molar ratios of 1:32, 1:128, and
1:512 were used. After incubation, 20 μL of the reaction mixture
was collected, mixed with SDS–PAGE loading buffer containing
2% (v/v) β-mercaptoethanol, and heated at 100 °C for 10
min to terminate the reaction. Cleavage activity was evaluated by
SDS–PAGE using an Any kD SDS–PAGE gel (Bio-Rad), and
the band intensities of the substrate and cleavage products were quantified
using ImageJ software. For accurate quantification,
images were scanned at 48-bit resolution, and the red channel was
extracted and converted to 32-bit grayscale. Pixel values were log-transformed
and scaled to approximate optical density, following a standard procedure
involving logarithmic conversion (log<sub>10</sub>) after normalization.
Integrated densities of each band were then measured after background
subtraction.</p>
<p>For the assessment of residual soluble fraction
and activity after
heat treatment, His-tagged T4L, a substrate that is relatively resistant
to cleavage, was used. This substrate was selected because, even when
mixed with an equimolar amount of enzyme, 3 h at 37 °C are required
to achieve 50% cleavage, making it suitable for simultaneous quantification
of both the enzyme and substrate on SDS–PAGE gels. The assay
was performed by incubating 20 μL of a 50 μM enzyme solution
at 50 °C for 0, 1, 2, 4, 8, 16, or 32 min. After incubation at
50 °C, the enzyme solution was centrifuged at 15,000 g for 10
min at 4 °C. The resulting supernatant (10 μL) was then
mixed with 10 μL of 50 μM T4L substrate, followed by incubation
at 37 °C for 3 h before reaction termination. Protease activity
was subsequently quantified using the method described above.</p>
<p>For additional evaluation of protease activity under moderate and
elevated temperatures, comparative cleavage assays were conducted
using T4 lysozyme (T4L) and the thermostable chitin-binding domain
(ChBD) as substrates (<a href="#fig4" class="usa-link">Figure 
</a>). For reactions with T4L, enzyme (25 μM, 10 μL)
and substrate (100 μM, 10 μL) were mixed and incubated
at either 30 or 37 °C for 3 h under static conditions. After
incubation, the reactions were terminated and analyzed by SDS–PAGE
as described above. To achieve comparable cleavage levels to the T4L
assay, a reduced amount of enzyme (25 μM, 2 μL) was used
for reactions with ChBD (100 μM, 18 μL), and the reaction
mixture was incubated for 30 min in a PCR thermal cycler block preset
to 50 °C. Reactions were terminated by heating the samples to
98 °C for 2 min using the same thermal cycler, followed by standard
sample processing as described above.</p></section><section id="sec4.4"><h3 class="pmc_sec_title">4.4. AlphaFold Prediction</h3>
<p>Structural models
of the various cTEVp constructs were generated using AlphaFold 3 by inputting the amino acid sequences of cTEVp
and its model substrate peptide (GENLYFQG) as two separate chains
into the web-based prediction tool. Predictions were performed using
the default settings with multimer mode enabled to allow modeling
of intermolecular interactions. Although the interface allows specification
of post-translational modifications (PTMs), the isopeptide bond formation
between ST and SC is not currently supported; therefore, no PTMs were
specified. Five ranked models were generated for each input, and the
top-ranked model based on pLDDT was selected for further analysis.
Structural models were manually inspected and adjusted using Coot, and molecular graphics were rendered with PyMOL
(Schrödinger, LLC).
</p></section><section id="sec4.5"><h3 class="pmc_sec_title">4.5. Statistical Analysis</h3>
<p>All experimental
data are expressed as means ± standard deviations (SD) from at
least three independent experiments. Statistical significance was
determined using an unpaired two-tailed Student’s <em>t</em> test, with <em>p</em> &lt; 0.05 considered statistically
significant. For figures with asterisks, significance levels are denoted
as follows: *<em>p</em> &lt; 0.05, **<em>p</em> &lt;
0.01, ***<em>p</em> &lt; 0.001, ****<em>p</em> &lt;
0.0001. All statistical analyses were performed using Microsoft Excel.</p></section></section><section id="sec15"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="sifile1"><div class="media p"><div class="caption">
<a href="/articles/instance/12332596/bin/ao5c05386_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">ao5c05386_si_001.pdf</a><sup> (415.8KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank
Takashi Nagami, Ayumi Yamasaki, and Daisuke
Kimura for their contributions to plasmid construction in the early
stages of this work. The authors acknowledge the use of OpenAI’s
ChatGPT for English language editing and text polishing during manuscript
preparation. The authors reviewed and edited the content as needed
and take full responsibility for the final version of the manuscript.
This work was supported by the Japan Society for the Promotion of
Science (JSPS) KAKENHI Grant Numbers JP16K07691 and JP22K05419 awarded
to T.N., and JP15K07391 and JP22K19150 awarded to T.O. This work was
also supported by the Cooperative Research Program of the Network
Joint Research Center for Materials and Devices.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Glossary</h2>
<h3>Abbreviations</h3>
<dl class="def-list" id="dl1">
<dt>ChBD</dt>
<dd><p>chitin-binding
domain</p></dd>
<dt>cTEVp</dt>
<dd><p>cyclized
TEVp</p></dd>
<dt>ncTEVp</dt>
<dd><p>noncyclized
TEVp</p></dd>
<dt>SC</dt>
<dd><p>SpyCatcher</p></dd>
<dt>ST</dt>
<dd><p>SpyTag</p></dd>
<dt>T4L</dt>
<dd><p>T4 lysozyme</p></dd>
<dt>TEV</dt>
<dd><p>tobacco etch virus</p></dd>
<dt>TEVcs</dt>
<dd><p>TEVp cleavage
site</p></dd>
<dt>TEVp</dt>
<dd><p>TEV protease.</p></dd>
</dl></section><section id="notes1"><p>The data that
support the findings of this study are available from the corresponding
author upon reasonable request.</p></section><section id="notes4"><p>The Supporting Information is
available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsomega.5c05386?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acsomega.5c05386</a>.</p>
<ul id="silist" class="list" style="list-style-type:none"><li><p>Additional nucleotide sequences of expression plasmids,
amino acid sequences of TEVp variants, AlphaFold-predicted structures
of cTEVp variants, including variant comparisons and confidence scores,
and detailed plasmid map of the vector pHIT184 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c05386/suppl_file/ao5c05386_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p></li></ul></section><section id="notes2"><p>T.N. contributed
to conceptualization, methodology, investigation, formal analysis,
validation, writingoriginal draft preparation, writingreview
and editing, project administration, supervision, and funding acquisition.
Y.N. contributed to methodology and investigation. N.T. and E.N. contributed
to investigation. T.O. contributed to supervision, funding acquisition,
and writingreview and editing. All authors read and approved
the final version of the manuscript.</p></section><section id="notes3"><p>The authors
declare no competing financial interest.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>Pina A. S., Lowe C. R., Roque A. C. A.. Challenges and opportunities in the
purification of recombinant tagged proteins. Biotechnology Advances. 2014;32(2):366–381. doi: 10.1016/j.biotechadv.2013.12.001.</cite> [<a href="https://doi.org/10.1016/j.biotechadv.2013.12.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7125906/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24334194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnology%20Advances&amp;title=Challenges%20and%20opportunities%20in%20the%20purification%20of%20recombinant%20tagged%20proteins&amp;author=A.%20S.%20Pina&amp;author=C.%20R.%20Lowe&amp;author=A.%20C.%20A.%20Roque&amp;volume=32&amp;issue=2&amp;publication_year=2014&amp;pages=366-381&amp;pmid=24334194&amp;doi=10.1016/j.biotechadv.2013.12.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>Waugh D. S.. An overview
of enzymatic reagents for the removal of affinity tags. Protein Expr. Purif. 2011;80(2):283–293. doi: 10.1016/j.pep.2011.08.005.</cite> [<a href="https://doi.org/10.1016/j.pep.2011.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3195948/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21871965/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Expr.%20Purif.&amp;title=An%20overview%20of%20enzymatic%20reagents%20for%20the%20removal%20of%20affinity%20tags&amp;author=D.%20S.%20Waugh&amp;volume=80&amp;issue=2&amp;publication_year=2011&amp;pages=283-293&amp;pmid=21871965&amp;doi=10.1016/j.pep.2011.08.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Cesaratto F., Burrone O. R., Petris G.. Tobacco Etch
Virus protease: A shortcut
across biotechnologies. J. Biotechnol. 2016;231:239–249. doi: 10.1016/j.jbiotec.2016.06.012.</cite> [<a href="https://doi.org/10.1016/j.jbiotec.2016.06.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27312702/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biotechnol.&amp;title=Tobacco%20Etch%20Virus%20protease:%20A%20shortcut%20across%20biotechnologies&amp;author=F.%20Cesaratto&amp;author=O.%20R.%20Burrone&amp;author=G.%20Petris&amp;volume=231&amp;publication_year=2016&amp;pages=239-249&amp;pmid=27312702&amp;doi=10.1016/j.jbiotec.2016.06.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>Kapust R. B., Tözsér J., Fox J. D., Anderson D. E., Cherry S., Copeland T. D., Waugh D. S.. Tobacco etch virus protease: mechanism
of autolysis and rational design of stable mutants with wild-type
catalytic proficiency. Protein Eng. 2001;14(12):993–1000. doi: 10.1093/protein/14.12.993.</cite> [<a href="https://doi.org/10.1093/protein/14.12.993" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11809930/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Eng.&amp;title=Tobacco%20etch%20virus%20protease:%20mechanism%20of%20autolysis%20and%20rational%20design%20of%20stable%20mutants%20with%20wild-type%20catalytic%20proficiency&amp;author=R.%20B.%20Kapust&amp;author=J.%20T%C3%B6zs%C3%A9r&amp;author=J.%20D.%20Fox&amp;author=D.%20E.%20Anderson&amp;author=S.%20Cherry&amp;volume=14&amp;issue=12&amp;publication_year=2001&amp;pages=993-1000&amp;pmid=11809930&amp;doi=10.1093/protein/14.12.993&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>van
den Berg S., Löfdahl P.-A., Härd T., Berglund H.. Improved solubility of TEV protease by directed evolution. J. Biotechnol. 2006;121(3):291–298. doi: 10.1016/j.jbiotec.2005.08.006.</cite> [<a href="https://doi.org/10.1016/j.jbiotec.2005.08.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16150509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biotechnol.&amp;title=Improved%20solubility%20of%20TEV%20protease%20by%20directed%20evolution&amp;author=S.%20van%20den%20Berg&amp;author=P.-A.%20L%C3%B6fdahl&amp;author=T.%20H%C3%A4rd&amp;author=H.%20Berglund&amp;volume=121&amp;issue=3&amp;publication_year=2006&amp;pages=291-298&amp;pmid=16150509&amp;doi=10.1016/j.jbiotec.2005.08.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Blommel P. G., Fox B. G.. A combined approach
to improving large-scale production
of tobacco etch virus protease. Protein Expr.
Purif. 2007;55(1):53–68. doi: 10.1016/j.pep.2007.04.013.</cite> [<a href="https://doi.org/10.1016/j.pep.2007.04.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2047602/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17543538/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Expr.%20Purif.&amp;title=A%20combined%20approach%20to%20improving%20large-scale%20production%20of%20tobacco%20etch%20virus%20protease&amp;author=P.%20G.%20Blommel&amp;author=B.%20G.%20Fox&amp;volume=55&amp;issue=1&amp;publication_year=2007&amp;pages=53-68&amp;pmid=17543538&amp;doi=10.1016/j.pep.2007.04.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Cabrita L. D., Gilis D., Robertson A. L., Dehouck Y., Rooman M., Bottomley S. P.. Enhancing
the stability and solubility of TEV protease
using in silico design. Protein Sci. 2007;16(11):2360–2367. doi: 10.1110/ps.072822507.</cite> [<a href="https://doi.org/10.1110/ps.072822507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2211701/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17905838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci.&amp;title=Enhancing%20the%20stability%20and%20solubility%20of%20TEV%20protease%20using%20in%20silico%20design&amp;author=L.%20D.%20Cabrita&amp;author=D.%20Gilis&amp;author=A.%20L.%20Robertson&amp;author=Y.%20Dehouck&amp;author=M.%20Rooman&amp;volume=16&amp;issue=11&amp;publication_year=2007&amp;pages=2360-2367&amp;pmid=17905838&amp;doi=10.1110/ps.072822507&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Fang J., Chen L., Cheng B., Fan J.. Engineering soluble
tobacco etch virus protease accompanies the loss of stability. Protein Expr. Purif. 2013;92(1):29–35. doi: 10.1016/j.pep.2013.08.015.</cite> [<a href="https://doi.org/10.1016/j.pep.2013.08.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24012464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Expr.%20Purif.&amp;title=Engineering%20soluble%20tobacco%20etch%20virus%20protease%20accompanies%20the%20loss%20of%20stability&amp;author=J.%20Fang&amp;author=L.%20Chen&amp;author=B.%20Cheng&amp;author=J.%20Fan&amp;volume=92&amp;issue=1&amp;publication_year=2013&amp;pages=29-35&amp;pmid=24012464&amp;doi=10.1016/j.pep.2013.08.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Raran-Kurussi S., Cherry S., Zhang D., Waugh D. S.. Removal of Affinity
Tags with TEV Protease, Methods Mol
. Biol. 2017;1586:221–230. doi: 10.1007/978-1-4939-6887-9_14.</cite> [<a href="https://doi.org/10.1007/978-1-4939-6887-9_14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7974378/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28470608/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biol.&amp;title=Removal%20of%20Affinity%20Tags%20with%20TEV%20Protease,%20Methods%20Mol&amp;author=S.%20Raran-Kurussi&amp;author=S.%20Cherry&amp;author=D.%20Zhang&amp;author=D.%20S.%20Waugh&amp;volume=1586&amp;publication_year=2017&amp;pages=221-230&amp;pmid=28470608&amp;doi=10.1007/978-1-4939-6887-9_14&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Enríquez-Flores S., de la Mora-De la Mora J. I., Flores-López L. A., Cabrera N., Fernández-Lainez C., Hernández-Alcántara G., Guerrero-Beltrán C. E., López-Velázquez G., García-Torres I.. Improved yield,
stability, and cleavage reaction of
a novel tobacco etch virus protease mutant. Appl. Microbiol. Biotechnol. 2022;106(4):1475–1492. doi: 10.1007/s00253-022-11786-5.</cite> [<a href="https://doi.org/10.1007/s00253-022-11786-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35092453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&amp;title=Improved%20yield,%20stability,%20and%20cleavage%20reaction%20of%20a%20novel%20tobacco%20etch%20virus%20protease%20mutant&amp;author=S.%20Enr%C3%ADquez-Flores&amp;author=J.%20I.%20de%20la%20Mora-De%20la%20Mora&amp;author=L.%20A.%20Flores-L%C3%B3pez&amp;author=N.%20Cabrera&amp;author=C.%20Fern%C3%A1ndez-Lainez&amp;volume=106&amp;issue=4&amp;publication_year=2022&amp;pages=1475-1492&amp;pmid=35092453&amp;doi=10.1007/s00253-022-11786-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Sumida K. H., Núñez-Franco R., Kalvet I., Pellock S. J., Wicky B. I. M., Milles L. F., Dauparas J., Wang J., Kipnis Y., Jameson N., Kang A., de La Cruz J., Sankaran B., Bera A. K., Jiménez-Osés G., Baker D.. Improving Protein Expression,
Stability, and Function with ProteinMPNN. J.
Am. Chem. Soc. 2024;146(3):2054–2061. doi: 10.1021/jacs.3c10941.</cite> [<a href="https://doi.org/10.1021/jacs.3c10941" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10811672/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38194293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=Improving%20Protein%20Expression,%20Stability,%20and%20Function%20with%20ProteinMPNN&amp;author=K.%20H.%20Sumida&amp;author=R.%20N%C3%BA%C3%B1ez-Franco&amp;author=I.%20Kalvet&amp;author=S.%20J.%20Pellock&amp;author=B.%20I.%20M.%20Wicky&amp;volume=146&amp;issue=3&amp;publication_year=2024&amp;pages=2054-2061&amp;pmid=38194293&amp;doi=10.1021/jacs.3c10941&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Schoene C., Bennett S. P., Howarth M.. SpyRings Declassified:
A Blueprint
for Using Isopeptide-Mediated Cyclization to Enhance Enzyme Thermal
Resilience. Methods Enzymol. 2016;580:149–167. doi: 10.1016/bs.mie.2016.05.004.</cite> [<a href="https://doi.org/10.1016/bs.mie.2016.05.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27586332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Enzymol.&amp;title=SpyRings%20Declassified:%20A%20Blueprint%20for%20Using%20Isopeptide-Mediated%20Cyclization%20to%20Enhance%20Enzyme%20Thermal%20Resilience&amp;author=C.%20Schoene&amp;author=S.%20P.%20Bennett&amp;author=M.%20Howarth&amp;volume=580&amp;publication_year=2016&amp;pages=149-167&amp;pmid=27586332&amp;doi=10.1016/bs.mie.2016.05.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Schoene C., Fierer J. O., Bennett S. P., Howarth M.. SpyTag/SpyCatcher cyclization
confers resilience to boiling on a mesophilic enzyme. Angew. Chem. - Int. Ed. 2014;53(24):6101–6104. doi: 10.1002/anie.201402519.</cite> [<a href="https://doi.org/10.1002/anie.201402519" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4286826/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24817566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.%20-%20Int.%20Ed.&amp;title=SpyTag/SpyCatcher%20cyclization%20confers%20resilience%20to%20boiling%20on%20a%20mesophilic%20enzyme&amp;author=C.%20Schoene&amp;author=J.%20O.%20Fierer&amp;author=S.%20P.%20Bennett&amp;author=M.%20Howarth&amp;volume=53&amp;issue=24&amp;publication_year=2014&amp;pages=6101-6104&amp;pmid=24817566&amp;doi=10.1002/anie.201402519&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Phan J., Zdanov A., Evdokimov A. G., Tropea J. E., Peters H. K., Kapust R. B., Li M., Wlodawer A., Waugh D. S.. Structural
basis for the substrate specificity of tobacco etch virus protease. J. Biol. Chem. 2002;277(52):50564–50572. doi: 10.1074/jbc.M207224200.</cite> [<a href="https://doi.org/10.1074/jbc.M207224200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12377789/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Structural%20basis%20for%20the%20substrate%20specificity%20of%20tobacco%20etch%20virus%20protease&amp;author=J.%20Phan&amp;author=A.%20Zdanov&amp;author=A.%20G.%20Evdokimov&amp;author=J.%20E.%20Tropea&amp;author=H.%20K.%20Peters&amp;volume=277&amp;issue=52&amp;publication_year=2002&amp;pages=50564-50572&amp;pmid=12377789&amp;doi=10.1074/jbc.M207224200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>Schoene C., Bennett S. P., Howarth M.. SpyRing interrogation:
analyzing
how enzyme resilience can be achieved with phytase and distinct cyclization
chemistries. Sci. Rep. 2016;6:21151. doi: 10.1038/srep21151.</cite> [<a href="https://doi.org/10.1038/srep21151" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4748275/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26861173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=SpyRing%20interrogation:%20analyzing%20how%20enzyme%20resilience%20can%20be%20achieved%20with%20phytase%20and%20distinct%20cyclization%20chemistries&amp;author=C.%20Schoene&amp;author=S.%20P.%20Bennett&amp;author=M.%20Howarth&amp;volume=6&amp;publication_year=2016&amp;pages=21151&amp;pmid=26861173&amp;doi=10.1038/srep21151&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Gilbert C., Howarth M., Harwood C. R., Ellis T.. Extracellular Self-Assembly
of Functional and Tunable Protein Conjugates from Bacillus
subtilis
. ACS Synth. Biol. 2017;6(6):957–967. doi: 10.1021/acssynbio.6b00292.</cite> [<a href="https://doi.org/10.1021/acssynbio.6b00292" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28230977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Synth.%20Biol.&amp;title=Extracellular%20Self-Assembly%20of%20Functional%20and%20Tunable%20Protein%20Conjugates%20from%20Bacillus%20subtilis&amp;author=C.%20Gilbert&amp;author=M.%20Howarth&amp;author=C.%20R.%20Harwood&amp;author=T.%20Ellis&amp;volume=6&amp;issue=6&amp;publication_year=2017&amp;pages=957-967&amp;pmid=28230977&amp;doi=10.1021/acssynbio.6b00292&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Si M., Xu Q., Jiang L., Huang H.. SpyTag/SpyCatcher Cyclization Enhances
the Thermostability of Firefly Luciferase. PloS
One. 2016;11(9):e0162318.  doi: 10.1371/journal.pone.0162318.</cite> [<a href="https://doi.org/10.1371/journal.pone.0162318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5033358/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27658030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PloS%20One&amp;title=SpyTag/SpyCatcher%20Cyclization%20Enhances%20the%20Thermostability%20of%20Firefly%20Luciferase&amp;author=M.%20Si&amp;author=Q.%20Xu&amp;author=L.%20Jiang&amp;author=H.%20Huang&amp;volume=11&amp;issue=9&amp;publication_year=2016&amp;pages=e0162318&amp;pmid=27658030&amp;doi=10.1371/journal.pone.0162318&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>Hills R. A., Howarth M.. Virus-like particles against infectious
disease and
cancer: guidance for the nano-architect. Curr.
Opin. Biotechnol. 2022;73:346–354. doi: 10.1016/j.copbio.2021.09.012.</cite> [<a href="https://doi.org/10.1016/j.copbio.2021.09.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8555979/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34735984/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Biotechnol.&amp;title=Virus-like%20particles%20against%20infectious%20disease%20and%20cancer:%20guidance%20for%20the%20nano-architect&amp;author=R.%20A.%20Hills&amp;author=M.%20Howarth&amp;volume=73&amp;publication_year=2022&amp;pages=346-354&amp;pmid=34735984&amp;doi=10.1016/j.copbio.2021.09.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Abramson J., Adler J., Dunger J., Evans R., Green T., Pritzel A., Ronneberger O., Willmore L., Ballard A. J., Bambrick J., Bodenstein S. W., Evans D. A., Hung C. C., O’Neill M., Reiman D., Tunyasuvunakool K., Wu Z., Z̧emgulytė A., Arvaniti E., Beattie C., Bertolli O., Bridgland A., Cherepanov A., Congreve M., Cowen-Rivers A. I., Cowie A., Figurnov M., Fuchs F. B., Gladman H., Jain R., Khan Y. A., Low C. M. R., Perlin K., Potapenko A., Savy P., Singh S., Stecula A., Thillaisundaram A., Tong C., Yakneen S., Zhong E. D., Zielinski M., Z̧ídek A., Bapst V., Kohli P., Jaderberg M., Hassabis D., Jumper J. M.. Accurate structure
prediction of
biomolecular interactions with AlphaFold 3. Nature. 2024;630(8016):493–500. doi: 10.1038/s41586-024-07487-w.</cite> [<a href="https://doi.org/10.1038/s41586-024-07487-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11168924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38718835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Accurate%20structure%20prediction%20of%20biomolecular%20interactions%20with%20AlphaFold%203&amp;author=J.%20Abramson&amp;author=J.%20Adler&amp;author=J.%20Dunger&amp;author=R.%20Evans&amp;author=T.%20Green&amp;volume=630&amp;issue=8016&amp;publication_year=2024&amp;pages=493-500&amp;pmid=38718835&amp;doi=10.1038/s41586-024-07487-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>Nakai T., Miyagi N., Hisamura K., Matsuba S., Nishimoto K., Nakai E., Tanizawa K., Okajima T.. Metabolic role of a
genetically conserved aldehyde dehydrogenase in bacterial assimilation
of various primary amines. Arch. Biochem. Biophys. 2025;768:110387.  doi: 10.1016/j.abb.2025.110387.</cite> [<a href="https://doi.org/10.1016/j.abb.2025.110387" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40086564/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Arch.%20Biochem.%20Biophys.&amp;title=Metabolic%20role%20of%20a%20genetically%20conserved%20aldehyde%20dehydrogenase%20in%20bacterial%20assimilation%20of%20various%20primary%20amines&amp;author=T.%20Nakai&amp;author=N.%20Miyagi&amp;author=K.%20Hisamura&amp;author=S.%20Matsuba&amp;author=K.%20Nishimoto&amp;volume=768&amp;publication_year=2025&amp;pages=110387&amp;pmid=40086564&amp;doi=10.1016/j.abb.2025.110387&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>DelProposto J., Majmudar C. Y., Smith J. L., Brown W. C.. Mocr: A novel fusion
tag for enhancing solubility that is compatible with structural biology
applications. Protein Expr. Purif. 2009;63(1):40–49. doi: 10.1016/j.pep.2008.08.011.</cite> [<a href="https://doi.org/10.1016/j.pep.2008.08.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2936452/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18824232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Expr.%20Purif.&amp;title=Mocr:%20A%20novel%20fusion%20tag%20for%20enhancing%20solubility%20that%20is%20compatible%20with%20structural%20biology%20applications&amp;author=J.%20DelProposto&amp;author=C.%20Y.%20Majmudar&amp;author=J.%20L.%20Smith&amp;author=W.%20C.%20Brown&amp;volume=63&amp;issue=1&amp;publication_year=2009&amp;pages=40-49&amp;pmid=18824232&amp;doi=10.1016/j.pep.2008.08.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Zou Y., Weis W. I., Kobilka B. K.. N-terminal
T4 lysozyme fusion facilitates
crystallization of a G protein coupled receptor. PloS One. 2012;7(10):e46039.  doi: 10.1371/journal.pone.0046039.</cite> [<a href="https://doi.org/10.1371/journal.pone.0046039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3464249/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23056231/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PloS%20One&amp;title=N-terminal%20T4%20lysozyme%20fusion%20facilitates%20crystallization%20of%20a%20G%20protein%20coupled%20receptor&amp;author=Y.%20Zou&amp;author=W.%20I.%20Weis&amp;author=B.%20K.%20Kobilka&amp;volume=7&amp;issue=10&amp;publication_year=2012&amp;pages=e46039&amp;pmid=23056231&amp;doi=10.1371/journal.pone.0046039&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Hanazono Y., Takeda K., Niwa S., Hibi M., Takahashi N., Kanai T., Atomi H., Miki K.. Crystal structures
of chitin binding domains of chitinase from Thermococcus kodakarensis KOD 1. FEBS Lett. 2016;590(2):298–304. doi: 10.1002/1873-3468.12055.</cite> [<a href="https://doi.org/10.1002/1873-3468.12055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26823175/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett.&amp;title=Crystal%20structures%20of%20chitin%20binding%20domains%20of%20chitinase%20from%20Thermococcus%20kodakarensis%20KOD%201&amp;author=Y.%20Hanazono&amp;author=K.%20Takeda&amp;author=S.%20Niwa&amp;author=M.%20Hibi&amp;author=N.%20Takahashi&amp;volume=590&amp;issue=2&amp;publication_year=2016&amp;pages=298-304&amp;pmid=26823175&amp;doi=10.1002/1873-3468.12055&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Nallamsetty S., Kapust R. B., Tözsér J., Cherry S., Tropea J. E., Copeland T. D., Waugh D. S.. Efficient
site-specific
processing of fusion proteins by tobacco vein mottling virus protease
in vivo and in vitro. Protein Expr. Purif. 2004;38(1):108–115. doi: 10.1016/j.pep.2004.08.016.</cite> [<a href="https://doi.org/10.1016/j.pep.2004.08.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15477088/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Expr.%20Purif.&amp;title=Efficient%20site-specific%20processing%20of%20fusion%20proteins%20by%20tobacco%20vein%20mottling%20virus%20protease%20in%20vivo%20and%20in%20vitro&amp;author=S.%20Nallamsetty&amp;author=R.%20B.%20Kapust&amp;author=J.%20T%C3%B6zs%C3%A9r&amp;author=S.%20Cherry&amp;author=J.%20E.%20Tropea&amp;volume=38&amp;issue=1&amp;publication_year=2004&amp;pages=108-115&amp;pmid=15477088&amp;doi=10.1016/j.pep.2004.08.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Kapust R. B., Waugh D. S.. 
Escherichia coli maltose-binding protein
is uncommonly effective at promoting the solubility of polypeptides
to which it is fused. Protein Sci. 1999;8(8):1668–1674. doi: 10.1110/ps.8.8.1668.</cite> [<a href="https://doi.org/10.1110/ps.8.8.1668" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2144417/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10452611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci.&amp;title=Escherichia%20coli%20maltose-binding%20protein%20is%20uncommonly%20effective%20at%20promoting%20the%20solubility%20of%20polypeptides%20to%20which%20it%20is%20fused&amp;author=R.%20B.%20Kapust&amp;author=D.%20S.%20Waugh&amp;volume=8&amp;issue=8&amp;publication_year=1999&amp;pages=1668-1674&amp;pmid=10452611&amp;doi=10.1110/ps.8.8.1668&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Gao Q., Ming D.. Protein-protein interactions
enhance the thermal resilience of SpyRing-cyclized
enzymes: A molecular dynamic simulation study. PloS One. 2022;17(2):e0263792.  doi: 10.1371/journal.pone.0263792.</cite> [<a href="https://doi.org/10.1371/journal.pone.0263792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8853484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35176056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PloS%20One&amp;title=Protein-protein%20interactions%20enhance%20the%20thermal%20resilience%20of%20SpyRing-cyclized%20enzymes:%20A%20molecular%20dynamic%20simulation%20study&amp;author=Q.%20Gao&amp;author=D.%20Ming&amp;volume=17&amp;issue=2&amp;publication_year=2022&amp;pages=e0263792&amp;pmid=35176056&amp;doi=10.1371/journal.pone.0263792&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Song H., Wang Y., Dong W., Chen Q., Sun H., Peng H., Li R., Chang Y., Luo H.. Effect of
SpyTag/SpyCatcher cyclization on stability and refolding of green
fluorescent protein. Biotechnol. Lett. 2022;44(4):613–621. doi: 10.1007/s10529-022-03246-x.</cite> [<a href="https://doi.org/10.1007/s10529-022-03246-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35359178/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnol.%20Lett.&amp;title=Effect%20of%20SpyTag/SpyCatcher%20cyclization%20on%20stability%20and%20refolding%20of%20green%20fluorescent%20protein&amp;author=H.%20Song&amp;author=Y.%20Wang&amp;author=W.%20Dong&amp;author=Q.%20Chen&amp;author=H.%20Sun&amp;volume=44&amp;issue=4&amp;publication_year=2022&amp;pages=613-621&amp;pmid=35359178&amp;doi=10.1007/s10529-022-03246-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Zakeri B., Fierer J. O., Celik E., Chittock E. C., Schwarz-Linek U., Moy V. T., Howarth M.. Peptide tag forming a rapid covalent
bond to a protein, through engineering a bacterial adhesin. Proc. Natl. Acad. Sci. U. S. A. 2012;109(12):E690–E697. doi: 10.1073/pnas.1115485109.</cite> [<a href="https://doi.org/10.1073/pnas.1115485109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3311370/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22366317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&amp;title=Peptide%20tag%20forming%20a%20rapid%20covalent%20bond%20to%20a%20protein,%20through%20engineering%20a%20bacterial%20adhesin&amp;author=B.%20Zakeri&amp;author=J.%20O.%20Fierer&amp;author=E.%20Celik&amp;author=E.%20C.%20Chittock&amp;author=U.%20Schwarz-Linek&amp;volume=109&amp;issue=12&amp;publication_year=2012&amp;pages=E690-E697&amp;pmid=22366317&amp;doi=10.1073/pnas.1115485109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>Studier F. W.. Protein
production by auto-induction in high density shaking cultures. Protein Expr. Purif. 2005;41(1):207–234. doi: 10.1016/j.pep.2005.01.016.</cite> [<a href="https://doi.org/10.1016/j.pep.2005.01.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15915565/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Expr.%20Purif.&amp;title=Protein%20production%20by%20auto-induction%20in%20high%20density%20shaking%20cultures&amp;author=F.%20W.%20Studier&amp;volume=41&amp;issue=1&amp;publication_year=2005&amp;pages=207-234&amp;pmid=15915565&amp;doi=10.1016/j.pep.2005.01.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Schneider C. A., Rasband W. S., Eliceiri K. W.. NIH Image
to ImageJ: 25 years of
image analysis. Nat. Methods. 2012;9(7):671–675. doi: 10.1038/nmeth.2089.</cite> [<a href="https://doi.org/10.1038/nmeth.2089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5554542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22930834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=NIH%20Image%20to%20ImageJ:%2025%20years%20of%20image%20analysis&amp;author=C.%20A.%20Schneider&amp;author=W.%20S.%20Rasband&amp;author=K.%20W.%20Eliceiri&amp;volume=9&amp;issue=7&amp;publication_year=2012&amp;pages=671-675&amp;pmid=22930834&amp;doi=10.1038/nmeth.2089&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>Emsley P., Lohkamp B., Scott W. G., Cowtan K.. Features and development
of Coot. Acta Crystallogr., Sect. D: Biol. Crystallogr. 2010;66(Pt 4):486–501. doi: 10.1107/S0907444910007493.</cite> [<a href="https://doi.org/10.1107/S0907444910007493" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2852313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20383002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Crystallogr.,%20Sect.%20D:%20Biol.%20Crystallogr.&amp;title=Features%20and%20development%20of%20Coot&amp;author=P.%20Emsley&amp;author=B.%20Lohkamp&amp;author=W.%20G.%20Scott&amp;author=K.%20Cowtan&amp;volume=66&amp;issue=Pt%204&amp;publication_year=2010&amp;pages=486-501&amp;pmid=20383002&amp;doi=10.1107/S0907444910007493&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>
Schrödinger,
LLC 
The PyMOL Molecular Graphics
System,
Version 3.0. New York, NY, 2015.</cite> [<a href="https://scholar.google.com/scholar_lookup?Schr%C3%B6dinger,%20LLC%20The%20PyMOL%20Molecular%20Graphics%20System,%20Version%203.0.%20New%20York,%20NY,%202015." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12332596/bin/ao5c05386_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">ao5c05386_si_001.pdf</a><sup> (415.8KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that
support the findings of this study are available from the corresponding
author upon reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Omega are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsomega.5c05386"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/ao5c05386.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12332596/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12332596/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332596%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332596/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12332596/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12332596/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40787351/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12332596/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40787351/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12332596/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12332596/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="y5cyXRYorPHVkXqLkRhttrV0AZz3ZZCoZuPHKRmSb9fChuTOcjDUPcIszBoiwlVb">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
