
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Genomic prediction for heat and herbicide tolerance in faba bean - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="FF207E2889B6D6831A7E28004CE24F8A.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12271658/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Genomic prediction for heat and herbicide tolerance in faba bean">
<meta name="citation_author" content="Lynn Abou Khater">
<meta name="citation_author_institution" content="Biodiversity and Crop Improvement Program (BCIP), International Center for Agricultural Research in the Dry Areas (ICARDA), Terbol, Lebanon">
<meta name="citation_author" content="Reem Joukhadar">
<meta name="citation_author_institution" content="AgriSapiens Statistical Solutions, Melbourne, Australia">
<meta name="citation_author" content="Fouad Maalouf">
<meta name="citation_author_institution" content="Biodiversity and Crop Improvement Program (BCIP), International Center for Agricultural Research in the Dry Areas (ICARDA), Terbol, Lebanon">
<meta name="citation_author" content="Alsamman M Alsamman">
<meta name="citation_author_institution" content="Agricultural Research Center (ARC), Agricultural Genetic Engineering Research Institute (AGERI), Giza, Egypt">
<meta name="citation_author" content="Zayed Babiker">
<meta name="citation_author_institution" content="Agricultural Research Corporation (ARC), Wad Madani, Sudan">
<meta name="citation_author" content="Rind Balech">
<meta name="citation_author_institution" content="Biodiversity and Crop Improvement Program (BCIP), International Center for Agricultural Research in the Dry Areas (ICARDA), Terbol, Lebanon">
<meta name="citation_author" content="Jinguo Hu">
<meta name="citation_author_institution" content="USDA‐ARS, Plant Germplasm Introduction &amp; Testing Research Unit, Pullman, Washington, USA">
<meta name="citation_author" content="Yu Ma">
<meta name="citation_author_institution" content="Department of Horticulture, Washington State University, Pullman, Washington, USA">
<meta name="citation_author_institution" content="Department of Horticulture and Crop Science, Ohio State University, Columbus, Ohio, USA">
<meta name="citation_author" content="Andrew Dunham">
<meta name="citation_author_institution" content="LGC, Biosearch Technologies, Hoddesdon, England, UK">
<meta name="citation_author" content="Miguel Sanchez">
<meta name="citation_author_institution" content="Biodiversity and Crop Improvement Program (BCIP), International Center for Agricultural Research in the Dry Areas (ICARDA), Rabat, Morocco">
<meta name="citation_author" content="Abdulqader Jighly">
<meta name="citation_author_institution" content="AgriSapiens Statistical Solutions, Melbourne, Australia">
<meta name="citation_author" content="Shiv Kumar">
<meta name="citation_author_institution" content="Biodiversity and Crop Improvement Program (BCIP), International Center for Agricultural Research in the Dry Areas (ICARDA), New Delhi, India">
<meta name="citation_publication_date" content="2025 Jul 17">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70076">
<meta name="citation_doi" content="10.1002/tpg2.70076">
<meta name="citation_pmid" content="40676773">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12271658/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12271658/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12271658/pdf/TPG2-18-e70076.pdf">
<meta name="description" content="Genomic selection (GS) has potential to accelerate the genetic gain in crop plants. This study was undertaken to assess the accuracy and potential of GS in faba bean [Vicia faba (L.)] and to enhance its application in breeding programs. A set of 118 ...">
<meta name="og:title" content="Genomic prediction for heat and herbicide tolerance in faba bean">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Genomic selection (GS) has potential to accelerate the genetic gain in crop plants. This study was undertaken to assess the accuracy and potential of GS in faba bean [Vicia faba (L.)] and to enhance its application in breeding programs. A set of 118 ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12271658/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12271658">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70076"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70076.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12271658%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12271658/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12271658/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12271658/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70076" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Jul 17;18(3):e70076. doi: <a href="https://doi.org/10.1002/tpg2.70076" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70076</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Genomic prediction for heat and herbicide tolerance in faba bean</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abou%20Khater%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Lynn Abou Khater</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Lynn Abou Khater</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Biodiversity and Crop Improvement Program (BCIP), International Center for Agricultural Research in the Dry Areas (ICARDA), Terbol, Lebanon</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abou%20Khater%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lynn Abou Khater</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Joukhadar%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Reem Joukhadar</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Reem Joukhadar</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>AgriSapiens Statistical Solutions, Melbourne, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Joukhadar%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Reem Joukhadar</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Maalouf%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Fouad Maalouf</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Fouad Maalouf</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Biodiversity and Crop Improvement Program (BCIP), International Center for Agricultural Research in the Dry Areas (ICARDA), Terbol, Lebanon</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Maalouf%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fouad Maalouf</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alsamman%20AM%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Alsamman M Alsamman</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Alsamman M Alsamman</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Agricultural Research Center (ARC), Agricultural Genetic Engineering Research Institute (AGERI), Giza, Egypt</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alsamman%20AM%22%5BAuthor%5D" class="usa-link"><span class="name western">Alsamman M Alsamman</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Babiker%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Zayed Babiker</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Zayed Babiker</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Agricultural Research Corporation (ARC), Wad Madani, Sudan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Babiker%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zayed Babiker</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Balech%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Rind Balech</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Rind Balech</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Biodiversity and Crop Improvement Program (BCIP), International Center for Agricultural Research in the Dry Areas (ICARDA), Terbol, Lebanon</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Balech%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rind Balech</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Jinguo Hu</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Jinguo Hu</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>USDA‐ARS, Plant Germplasm Introduction &amp; Testing Research Unit, Pullman, Washington, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jinguo Hu</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Yu Ma</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Yu Ma</span></h3>
<div class="p">
<sup>
<sup>6</sup>
</sup>Department of Horticulture, Washington State University, Pullman, Washington, USA</div>
<div class="p">
<sup>
<sup>7</sup>
</sup>Department of Horticulture and Crop Science, Ohio State University, Columbus, Ohio, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yu Ma</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dunham%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Andrew Dunham</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Andrew Dunham</span></h3>
<div class="p">
<sup>
<sup>8</sup>
</sup>LGC, Biosearch Technologies, Hoddesdon, England, UK</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dunham%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrew Dunham</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sanchez%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Miguel Sanchez</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Miguel Sanchez</span></h3>
<div class="p">
<sup>
<sup>9</sup>
</sup>Biodiversity and Crop Improvement Program (BCIP), International Center for Agricultural Research in the Dry Areas (ICARDA), Rabat, Morocco</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sanchez%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Miguel Sanchez</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jighly%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Abdulqader Jighly</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Abdulqader Jighly</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>AgriSapiens Statistical Solutions, Melbourne, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jighly%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Abdulqader Jighly</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Shiv Kumar</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Shiv Kumar</span></h3>
<div class="p">
<sup>
<sup>10</sup>
</sup>Biodiversity and Crop Improvement Program (BCIP), International Center for Agricultural Research in the Dry Areas (ICARDA), New Delhi, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shiv Kumar</span></a>
</div>
</div>
<sup>10</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270076-aff-0001">
<sup>
<sup>1</sup>
</sup>Biodiversity and Crop Improvement Program (BCIP), International Center for Agricultural Research in the Dry Areas (ICARDA), Terbol, Lebanon</div>
<div id="tpg270076-aff-0002">
<sup>
<sup>2</sup>
</sup>AgriSapiens Statistical Solutions, Melbourne, Australia</div>
<div id="tpg270076-aff-0003">
<sup>
<sup>3</sup>
</sup>Agricultural Research Center (ARC), Agricultural Genetic Engineering Research Institute (AGERI), Giza, Egypt</div>
<div id="tpg270076-aff-0004">
<sup>
<sup>4</sup>
</sup>Agricultural Research Corporation (ARC), Wad Madani, Sudan</div>
<div id="tpg270076-aff-0005">
<sup>
<sup>5</sup>
</sup>USDA‐ARS, Plant Germplasm Introduction &amp; Testing Research Unit, Pullman, Washington, USA</div>
<div id="tpg270076-aff-0006">
<sup>
<sup>6</sup>
</sup>Department of Horticulture, Washington State University, Pullman, Washington, USA</div>
<div id="tpg270076-aff-0007">
<sup>
<sup>7</sup>
</sup>Department of Horticulture and Crop Science, Ohio State University, Columbus, Ohio, USA</div>
<div id="tpg270076-aff-0008">
<sup>
<sup>8</sup>
</sup>LGC, Biosearch Technologies, Hoddesdon, England, UK</div>
<div id="tpg270076-aff-0009">
<sup>
<sup>9</sup>
</sup>Biodiversity and Crop Improvement Program (BCIP), International Center for Agricultural Research in the Dry Areas (ICARDA), Rabat, Morocco</div>
<div id="tpg270076-aff-0010">
<sup>
<sup>10</sup>
</sup>Biodiversity and Crop Improvement Program (BCIP), International Center for Agricultural Research in the Dry Areas (ICARDA), New Delhi, India</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Fouad Maalouf, Biodiversity and Crop Improvement Program (BCIP), International Center for Agricultural Research in the Dry Areas (ICARDA), Terbol, Lebanon. Email: <span>f.maalouf@cgiar.org</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 May 15; Received 2024 Oct 16; Accepted 2025 Jun 10; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12271658  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40676773/" class="usa-link">40676773</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Genomic selection (GS) has potential to accelerate the genetic gain in crop plants. This study was undertaken to assess the accuracy and potential of GS in faba bean [<em>Vicia faba</em> (L.)] and to enhance its application in breeding programs. A set of 118 diverse faba bean accessions were phenotyped for key agronomic traits under herbicide and heat stress across 16 environments in Morocco, Lebanon, Sudan, and the United States. These accessions were genotyped, revealing 170 single nucleotide polymorphisms (SNPs) strongly associated with target traits. kompetitive allele‐specific PCR (KASP) markers were subsequently designed and validated on 4512 diverse breeding lines. Prediction accuracy (PA) was assessed using the reproducing kernel Hilbert space model, with and without genotype‐by‐environment interactions and taking into consideration two cross‐validation (CV) strategies: CV1 (predicting new lines) and CV2 (predicting complete records from unbalanced data). Additionally, 75 KASP markers with known association with heat tolerance traits were prioritized to estimate the PA of the models. The results showed a comparable PA between the two models, with CV1 outperforming CV2. This highlighted the difficulty in predicting the performance of untested lines in tested environments compared to lines evaluated in some environments but not others. Moreover, SNP subset size and composition significantly impacted PA, especially under heat stress. The highest accuracies were observed for days to flowering and plant height under heat stress and for plant height and grain yield under herbicide environments, indicating that these traits are ideal for training population selection. Optimizing the size and composition of the training population will enhance the effectiveness of GS in faba bean breeding.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270076-list-10001" class="list" style="list-style-type:disc">
<li><p>Genomic selection is applicable in faba bean.</p></li>
<li><p>Accurate prediction accuracy and genomic selection in faba bean require a high genome coverage and large training population.</p></li>
<li><p>Days to flowering and plant height are ideal traits for training population selection for heat tolerance.</p></li>
<li><p>Plant height and grain yield are ideal traits for training population selection for herbicide tolerance.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270076-dl-0001">
<dt>CI</dt>
<dd><p>chlorophyll content</p></dd>
<dt>CT</dt>
<dd><p>canopy temperature</p></dd>
<dt>CV</dt>
<dd><p>cross‐validation</p></dd>
<dt>DFLR</dt>
<dd><p>days to 50% flowering</p></dd>
<dt>DMAT</dt>
<dd><p>days to 90% maturity</p></dd>
<dt>G × E</dt>
<dd><p>genotype by environment interaction</p></dd>
<dt>GCC</dt>
<dd><p>green canopy cover</p></dd>
<dt>GS</dt>
<dd><p>genomic selection</p></dd>
<dt>GWAS</dt>
<dd><p>genome‐wide association study</p></dd>
<dt>GYPLT</dt>
<dd><p>grain yield per plant</p></dd>
<dt>HDS1</dt>
<dd><p>herbicide damage score at flowering stage</p></dd>
<dt>HDS2</dt>
<dd><p>herbicide damage score at pod development stage</p></dd>
<dt>HSW</dt>
<dd><p>100‐seed weight</p></dd>
<dt>KASP</dt>
<dd><p>kompetitive allele‐specific PCR</p></dd>
<dt>NBrPLT</dt>
<dd><p>number of branches per plant</p></dd>
<dt>NPPLT</dt>
<dd><p>number of pods per plant</p></dd>
<dt>NSPLT</dt>
<dd><p>number of seeds per plant</p></dd>
<dt>PA</dt>
<dd><p>prediction accuracy</p></dd>
<dt>PCR</dt>
<dd><p>polymerase chain reaction</p></dd>
<dt>PG</dt>
<dd><p>pollen germination</p></dd>
<dt>PLHT</dt>
<dd><p>plant height</p></dd>
<dt>QTL</dt>
<dd><p>quantitative trait loci</p></dd>
<dt>SNP</dt>
<dd><p>single nucleotide polymorphism</p></dd>
<dt>SW</dt>
<dd><p>seed weight</p></dd>
</dl>
<section id="tpg270076-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Faba bean [<em>Vicia faba</em> (L.)] holds significant importance among food legumes due to its nutritional density and agronomic versatility.</p>
<p>Among the many constraints hindering faba bean production, heat stress and weeds tend to have the most detrimental effects especially in the Mediterranean and Nile Valley regions where faba bean is grown as a vital legume crop. High temperatures surpassing the threshold of 30°C during the flowering stage impair pollen viability leading to reduced pod set and significant yield reduction (Devasirvatham et al., <a href="#tpg270076-bib-0023" class="usa-link" aria-describedby="tpg270076-bib-0023">2012</a>; Lavania et al., <a href="#tpg270076-bib-0044" class="usa-link" aria-describedby="tpg270076-bib-0044">2015</a>; Bishop et al., <a href="#tpg270076-bib-0011" class="usa-link" aria-describedby="tpg270076-bib-0011">2016</a>; Badri, <a href="#tpg270076-bib-0009" class="usa-link" aria-describedby="tpg270076-bib-0009">2023</a>). Therefore, in light of the anticipated effects of climate change (Wardhana &amp; Prawira, <a href="#tpg270076-bib-0070" class="usa-link" aria-describedby="tpg270076-bib-0070">2024</a>) and the observed rising temperature trend, breeding for heat tolerance is becoming very crucial. On the other hand, given the low competitive ability of faba bean, weeds can lead to considerable reduction in yield that can reach up to 60% if not managed properly (Frenda et al., <a href="#tpg270076-bib-0027" class="usa-link" aria-describedby="tpg270076-bib-0027">2013</a>). In conventional agriculture, preference is given to herbicide application as the most effective, timesaving, and economical method of weed management compared to alternative strategies (De Arevalo et al., <a href="#tpg270076-bib-0020" class="usa-link" aria-describedby="tpg270076-bib-0020">1992</a>; Gressel, <a href="#tpg270076-bib-0032" class="usa-link" aria-describedby="tpg270076-bib-0032">2000</a>), and therefore, herbicide tolerance is becoming a highly desirable trait in faba bean.</p>
<p>Although screening under field conditions has led to the identification of sources for tolerance to heat and herbicide in faba bean (Abou Khater et al., <a href="#tpg270076-bib-0002" class="usa-link" aria-describedby="tpg270076-bib-0002">2021</a>; Maalouf et al., <a href="#tpg270076-bib-0049" class="usa-link" aria-describedby="tpg270076-bib-0049">2022</a>, <a href="#tpg270076-bib-0050" class="usa-link" aria-describedby="tpg270076-bib-0050">2019</a>), achieving a significant genetic advancement using these conventional methods remains very challenging as they are input intensive, and their replicability depends on the environmental conditions. This is evidenced by the genetic gain obtained over the years, which, despite some increase, is still insufficient to meet the demand of the growing population (Tolessa et al., <a href="#tpg270076-bib-0064" class="usa-link" aria-describedby="tpg270076-bib-0064">2015</a>). To address the outlined challenges, prior studies used genome‐wide association study (GWAS) or linkage mapping to identify quantitative trait loci (QTLs) associated with major traits of interest in faba bean like seed size; resistance to Ascochyta blight, chocolate spot, and broomrapes; and tolerance to herbicides, frost, and heat, in addition to vicine–convicine content (Ali et al., <a href="#tpg270076-bib-0007" class="usa-link" aria-describedby="tpg270076-bib-0007">2016</a>; Kaur et al., <a href="#tpg270076-bib-0042" class="usa-link" aria-describedby="tpg270076-bib-0042">2014</a>; Ocaña‐Moral et al., <a href="#tpg270076-bib-0053" class="usa-link" aria-describedby="tpg270076-bib-0053">2017</a>; Khazaei et al., <a href="#tpg270076-bib-0043" class="usa-link" aria-describedby="tpg270076-bib-0043">2017</a>; Abou‐Khater et al., <a href="#tpg270076-bib-0001" class="usa-link" aria-describedby="tpg270076-bib-0001">2022</a>; Maalouf et al., <a href="#tpg270076-bib-0049" class="usa-link" aria-describedby="tpg270076-bib-0049">2022</a>; Gela et al., <a href="#tpg270076-bib-0029" class="usa-link" aria-describedby="tpg270076-bib-0029">2022</a>; Skovbjerg et al., <a href="#tpg270076-bib-0057" class="usa-link" aria-describedby="tpg270076-bib-0057">2023</a>). Although marker‐assisted selection is known to increase the selection accuracy compared to conventional methods (Mohan et al., <a href="#tpg270076-bib-0051" class="usa-link" aria-describedby="tpg270076-bib-0051">1997</a>), it also has its own limitations as its effectiveness is contingent upon the trait under consideration being controlled by genes with relatively major effects. Nevertheless, majority traits of breeders’ interest exhibit significant genetic complexity as they are influenced by environmental effects and by multiple minor genes with limited effects that cannot be captured thereby posing challenges to marker‐assisted selection (Xu et al., <a href="#tpg270076-bib-0065" class="usa-link" aria-describedby="tpg270076-bib-0065">2020</a>; Adhikari et al., <a href="#tpg270076-bib-0003" class="usa-link" aria-describedby="tpg270076-bib-0003">2021</a>). Luckily, genomic selection, often considered as an innovative form of marker‐assisted selection, deals with complex traits by utilizing genome‐wide marker data to capture the cumulative effects of both minor and major effect of markers contributing to the overall genetic variation underlying the quantitative traits (Xu et al., <a href="#tpg270076-bib-0065" class="usa-link" aria-describedby="tpg270076-bib-0065">2020</a>). The cumulative effect is used for calculation of the genomic‐estimated breeding values (GEBV) of each individual (Budhlakoti et al., <a href="#tpg270076-bib-0013" class="usa-link" aria-describedby="tpg270076-bib-0013">2022</a>) from early generation without the need of extensive phenotyping and multiple breeding cycles. However, to ensure the success of the genomic selection, factors that may impact the accuracy of genomic prediction should be taken into consideration like the genetic diversity and size of the breeding population, the heritability of the studied trait, the genotype by environment interaction (GE), the markers’ density, and the genetic relationship between training population and breeding population or selection candidates (Clark et al., <a href="#tpg270076-bib-0017" class="usa-link" aria-describedby="tpg270076-bib-0017">2011</a>; Daetwyler et al., <a href="#tpg270076-bib-0019" class="usa-link" aria-describedby="tpg270076-bib-0019">2010</a>; De Roos et al., <a href="#tpg270076-bib-0022" class="usa-link" aria-describedby="tpg270076-bib-0022">2009</a>; Howard et al., <a href="#tpg270076-bib-0037" class="usa-link" aria-describedby="tpg270076-bib-0037">2014</a>; Lorenzana &amp; Bernardo, <a href="#tpg270076-bib-0047" class="usa-link" aria-describedby="tpg270076-bib-0047">2009</a>; Luan et al., <a href="#tpg270076-bib-0048" class="usa-link" aria-describedby="tpg270076-bib-0048">2009</a>).</p>
<p>Genomic selection was implemented successfully in the breeding program of different crops. Recently, Budhlakoti et al. (<a href="#tpg270076-bib-0013" class="usa-link" aria-describedby="tpg270076-bib-0013">2022</a>) reviewed the genomic selection studies conducted on major cereals, pulses, oilseeds, and horticultural crops for grain yield (GY), biotic and abiotic stresses tolerance, and seed quality. In the pulses breeding, genomic selection was applied to improve the maturity time of lentil (Haile et al., <a href="#tpg270076-bib-0035" class="usa-link" aria-describedby="tpg270076-bib-0035">2020</a>) and a range of other desired traits like GY, Ascochyta blight resistance, botrytis gray mould resistance, salinity and boron stress tolerance, 100‐grain weight, seed size index and protein content (Gebremedhin et al., <a href="#tpg270076-bib-0028" class="usa-link" aria-describedby="tpg270076-bib-0028">2024</a>), the cooking time and root rot resistance of common bean (Diaz et al., <a href="#tpg270076-bib-0024" class="usa-link" aria-describedby="tpg270076-bib-0024">2021</a>), and the drought tolerance of chickpea (Li et al., <a href="#tpg270076-bib-0045" class="usa-link" aria-describedby="tpg270076-bib-0045">2018</a>). Given the huge size of faba bean genome (13 Gbp) compared to other crops and the limited funding allocated for faba bean research, it was not until recently that this strategy became applicable in faba bean as a high single nucleotide polymorphism (SNPs) density have been ascertained by the release of assembled reference genome (Jayakodi et al., <a href="#tpg270076-bib-0039" class="usa-link" aria-describedby="tpg270076-bib-0039">2023</a>). A recent, yet unpublished work on genomic selection for yield‐related traits in faba bean has shown promising predictive ability and prediction accuracy (Lippolis et al., <a href="#tpg270076-bib-0068" class="usa-link" aria-describedby="tpg270076-bib-0068">2025</a>). To the best of our knowledge, this study, in addition to ours, represents the first applications of genomic selection for improving faba bean traits, filling a critical gap in the literature and providing a foundation for future research in this field.</p>
<p>The present study was conducted to evaluate the accuracy and potential of genomic selection for herbicide and heat tolerance in faba bean using a diversified faba bean reference population, and to identify areas for future improvement and better implementation of genomic prediction in practical breeding programs.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270076-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270076-list-0001" class="list" style="list-style-type:disc">
<li><p>Genomic selection is applicable in faba bean.</p></li>
<li><p>Accurate prediction accuracy and genomic selection in faba bean require a high genome coverage and large training population.</p></li>
<li><p>Days to flowering and plant height are ideal traits for training population selection for heat tolerance.</p></li>
<li><p>Plant height and grain yield are ideal traits for training population selection for herbicide tolerance.</p></li>
</ul></section></section></section><section id="tpg270076-sec-0030"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270076-sec-0040"><h3 class="pmc_sec_title">2.1. Phenotyping</h3>
<p>A set of 118 faba bean landraces originating from 42 countries in addition to 16 breeding lines from International Center for Agricultural Research in the Dry Areas (ICARDA) were included in this study (Table <a href="#tpg270076-supinfo-0001" class="usa-link">S1</a>). The set is genetically diverse as assessed earlier using simple sequence repeat markers, and was previously classified into 42 major, 17 minor, 66 equina, and nine paucijuga class types (Maalouf et al., <a href="#tpg270076-bib-0050" class="usa-link" aria-describedby="tpg270076-bib-0050">2019</a>). The major class (seed weight [SW] ≥ 100 mg) includes accessions with large seeds, typically exhibiting a plate‐like shape. The equina class (50 &lt; SW &lt; 100 mg) consists of accessions with medium‐sized seeds, also plate‐shaped. The minor class (30 &lt; SW &lt; 50 mg) is characterized by smaller, cylindrical to rounded seeds. Finally, the paucijuga class (20 &lt; SW &lt; 30 mg) includes accessions with very small, rounded to elliptical seeds (Cubero, <a href="#tpg270076-bib-0069" class="usa-link" aria-describedby="tpg270076-bib-0069">1974</a>). Note that the seeds used in these experiments are obtained from the reserve stock, which is multiplied annually under insect‐proof cages to maintain the purity of the evaluated accessions.</p>
<section id="tpg270076-sec-0050"><h4 class="pmc_sec_title">2.1.1. Phenotyping for herbicide tolerance</h4>
<p>A total of three field experiments were conducted between 2014 and 2018 in two ICARDA stations in Terbol, Lebanon, and Marchouch, Morocco, to evaluate their tolerance to three different herbicide treatments (T1—metribuzin @250 g ai/ha, T2—imazethapyr @ 75 g ai/ha, and T3—no herbicide application). Noting that the herbicides were applied at the pre‐flowering stage. In Marchouch, the experiment was conducted using augmented design while in Terbol Alpha lattice design was used.</p>
<p>The following observations were recorded: days to 50% flowering (DFLR) and days to 90% maturity (DMAT) were recorded on plot basis. Plant height (PLHT), GY, grain yield per plant (GYPLT), number of pods per plant (NPPLT), number of seeds per plant (NSPLT), number of branches per plant (NBrPLT), and green canopy cover (GCC) were recorded on three plants selected randomly. GCC is the average percentage of green coverage of three plants quantified using the Canopeo application developed by Oklahoma State University using Matlab. In addition, herbicide damage score at flowering stage (HDS1), herbicide damage score at pod development stage (HDS2) were recorded at a 1–5 scale. A score of 1 corresponds to <em>plots with no visible damage and normal growth</em>; 2 indicates <em>very light damage with excellent phytosanitary status</em>; 3 reflects <em>moderate damage with stunted growth and significant yellowing</em>; 4 represents <em>high damage, with over 50% plant death, weak vegetative growth, and stunted plants</em>; and 5 corresponds to <em>severe damage, with more than 50% of plants affected, significant biomass reduction, and extensive deformation</em>.</p></section><section id="tpg270076-sec-0060"><h4 class="pmc_sec_title">2.1.2. Phenotyping for heat tolerance</h4>
<p>A total of six field experiments were conducted in an incomplete block design with two replicates between 2015 and 2017 to evaluate their tolerance to heat. The experiments were conducted in three research stations: during the summer season at Terbol‐Lebanon where temperatures exceeds 35°C during the flowering and pod set time; at the Central Ferry Research Farm, Pullman,  characterized by warm weather during the reproductive and terminal crop cycle,  and in the winter season at ARC Hudeiba Research Station in Sudan where the weather is hot with daytime temperatures exceeding 32°C.</p>
<p>The following observations were recorded as described previously: DFLR, DMAT, PLHT, NPPLT, NSPLT, GYPLT, and 100‐seed weight (HSW). Chlorophyll content (CI) was measured on three random leaves per plot using  a chlorophyll meter (SPAD‐502 Minolta; Spectrum Technologies Inc.) during morning hours at the end of the flowering period. Canopy temperature (CT) was measured using a portable infrared thermometer (Mikron M90 Series, Mikron Infrared Instrument Co., Inc.) during vegetative and grain filling stages. And finally, the pollen germination (PG) was measured using the method described by Brewbaker and Kwack (<a href="#tpg270076-bib-0012" class="usa-link" aria-describedby="tpg270076-bib-0012">1963</a>). In brief, fresh pollen was collected from 15 random open flowers per plot when daytime temperatures exceeded 32°C. The pollen was stored in microfuge tubes in iceboxes for a few hours before the germination test. The test was conducted in two replicates per plot, using a germination medium containing boric acid, calcium nitrate, sucrose, and agar, prepared and autoclaved before being poured into Petri dishes. The medium's pH was adjusted to 6.3. Pollen grains were spread on the surface using a needle and incubated at 37°C for 16 h. Germination was assessed by counting the grains with pollen tubes at least as long as the grain diameter, and the germination rate was determined by counting the germinated grains per hundred observed, using a stereozoom microscope with a digital camera.</p></section></section><section id="tpg270076-sec-0070"><h3 class="pmc_sec_title">2.2. Phenotypic data analysis</h3>
<p>Starting with phenotyping for herbicide tolerance, variance analysis for all traits was conducted using the Automatic Spatial Analysis of Row‐Column modules using GenStat V23,2021. The model used in this analysis was:
</p>
<table class="disp-formula p" id="tpg270076-disp-0001"><tr>
<td class="formula"><math id="jats-math-1" display="block"><mrow><mrow><mi>Y</mi><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><mi>g</mi><mo linebreak="goodbreak">+</mo><mi mathvariant="normal">τ</mi><mo linebreak="goodbreak">+</mo><mi>Row</mi><mo linebreak="goodbreak">+</mo><mi>Col</mi><mo linebreak="goodbreak">+</mo><mfrac><mi>Rep</mi><mi>Block</mi></mfrac><mo linebreak="goodbreak">+</mo><mi>Error</mi></mrow></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>where <em>Y</em> is the observed trait value; <em>μ</em> is the overall mean, <em>g</em> is the genetic effect; <em>τ</em> is the treatment effect; Row and Col represent row and column effects, Rep/Block represents replication/block effects, and Error is the residual error. Genotypes and treatments were considered fixed factors, while all others were considered random factors. Significance of variation among genotypes and treatments was assessed at a <em>p</em>‐value &lt; 0.001. Multi environment trials analyses (META) was conducted using the method of residual maximum likelihood (REML) to evaluate variation among genotypes, treatments, and the genotype × treatment interaction across trials for traits measured in more than one trial. In the META analysis, the environmental effect (<em>E</em>) is incorporated into the model equation to account for the influence of environmental factors on the phenotypic data.</p>
<p>As for the phenotyping for heat tolerance, variance analysis was conducted using the automatic incomplete block design modules of GenStat V23,2021. The model used in this analysis was:
</p>
<table class="disp-formula p" id="tpg270076-disp-0002"><tr>
<td class="formula"><math id="jats-math-2" display="block"><mrow><mrow><mi>Y</mi><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><mi>g</mi><mo linebreak="goodbreak">+</mo><mi>Rep</mi><mo linebreak="goodbreak">+</mo><mi>Rep</mi><mo>/</mo><mi>Block</mi><mo linebreak="goodbreak">+</mo><mi>Error</mi></mrow></mrow></math></td>
<td class="label">(2)</td>
</tr></table>
<p>where <em>Y</em> is the observed trait value, <em>μ</em> is the overall mean, <em>g</em> is the genetic effect, Rep represents the replication effect, Rep/Block represents block within replicate effect, and Error is the residual error. Accessions were fitted as fixed factors, while all others were considered random. Differences among accessions were considered highly significant at a <em>p‐</em>value &lt; 0.001 and significant at a <em>p‐</em>value &lt; 0.05. META analysis where environmental effect (<em>E</em>) is incorporated into the model equation to account for the influence of environmental factors on the phenotypic data was conducted to examine the differences among accessions and environments.</p>
<p>Combined phenotypic heritability (<em>h<sup>2</sup>
</em>) was estimated for the traits based on the combined analysis using REML model and best unbiased estimated values of Genstat V23,2021. Correlation analysis was conducted to examine the relationship between genomic prediction accuracy and phenotypic estimated heritability and to assess whether genomic selection aligns with phenotypic selection.</p></section><section id="tpg270076-sec-0080"><h3 class="pmc_sec_title">2.3. Genotyping</h3>
<section id="tpg270076-sec-0090"><h4 class="pmc_sec_title">2.3.1. Sampling method</h4>
<p>A total of 192 kompetitive allele‐specific PCR (KASP) markers were developed (Table <a href="#tpg270076-supinfo-0001" class="usa-link">S2</a>) based on the GWAS results obtained by Abou Khater et al. (<a href="#tpg270076-bib-0001" class="usa-link" aria-describedby="tpg270076-bib-0001">2022</a>), Maalouf et al. (<a href="#tpg270076-bib-0049" class="usa-link" aria-describedby="tpg270076-bib-0049">2022</a>), and Guiterrez et al. (<a href="#tpg270076-bib-0033" class="usa-link" aria-describedby="tpg270076-bib-0033">2023</a>). These markers were selected based on their significant associations with multiple agronomic traits in faba bean, particularly those related to heat, drought and herbicide tolerance (e.g., PG and herbicide damage score), and yield performance (e.g., GYPLT, HSW, NPPLT, and NSPLT). KASP markers were used as the prioritized set of SNPs in the prediction analysis. To provide additional information on the genomic locations of these markers, we performed a BLAST analysis using the <em>V</em>. <em>faba</em> genome assembly (GCA∖_048183575.1), available on NCBI and published by the Institute of Crop Sciences, Chinese Academy of Agricultural Sciences. Marker positions were determined based on alignment results and are included in Table <a href="#tpg270076-supinfo-0001" class="usa-link">S2</a>.</p>
<p>The developed markers were used to test 4512 accessions that include 2441 multi‐parent advanced generation inter‐crosses (MAGIC) lines, 500 elite lines,182 parent reference set, 118 generation challenge program (GCP) reference set previously studied by Abou Khater et al. (<a href="#tpg270076-bib-0001" class="usa-link" aria-describedby="tpg270076-bib-0001">2022</a>) and Maalouf et al. (<a href="#tpg270076-bib-0049" class="usa-link" aria-describedby="tpg270076-bib-0049">2022</a>), and the remaining are F3 segregating lines.</p>
<p>These accessions were sampled during the vegetative stage of the cropping season 2021–2022 at Terbol, Lebanon. Young leaves of these accessions were collected and placed in 96‐well collection plate (BioArk, LGC Biosearch Technologies) where two wells A1 and H6 were left empty as checks. Afterward, the desiccant was applied and sealed in a plastic bag and conserved in the fridge at −12°C to avoid any leaf oxidative degradation.</p>
<p>These samples were sent to LCG laboratory in Hoddesdon, United Kingdom, along with the list of KASP markers for genotyping (Table <a href="#tpg270076-supinfo-0001" class="usa-link">S2</a>).</p></section><section id="tpg270076-sec-0100"><h4 class="pmc_sec_title">2.3.2. DNA extraction and genotyping</h4>
<p>The DNA was extracted in LCG laboratory using sbeadex chemistry developed by LGC.</p>
<p>Genotyping was performed using KASP genotyping technology developed at LGC. The KASP technology is based on allele‐specific polymerase chain reaction (PCR), which enables the bi‐allelic scoring of SNPs, insertions and deletions. The SNP‐specific KASP assay and the universal KASP‐TF assay mix were added to the DNA in 1536 well plates and a series of 35cycles of PCR performed. On completion of the 35 cycles, the plates were read using a fluorescence plate reader and visually inspected to assess the progression of the PCR reaction, further cycling was performed in cases where PCR reaction could not reach its endpoint. Once the reaction reached its endpoint, the data were examined, and genotypes were assigned.</p></section><section id="tpg270076-sec-0110"><h4 class="pmc_sec_title">2.3.3. Genomic selection</h4>
<p>The following two models were used for genomic selection:</p>
<section id="tpg270076-sec-0120"><h5 class="pmc_sec_title">Model 1: Additive genetics and environmental effect (g + E model)</h5>
<table class="disp-formula p" id="tpg270076-disp-0003"><tr>
<td class="formula"><math id="jats-math-3" display="block"><mrow><mrow><mi>y</mi><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><mi>E</mi><mo linebreak="goodbreak">+</mo><mi>g</mi><mo linebreak="goodbreak">+</mo><mi mathvariant="normal">ε</mi></mrow></mrow></math></td>
<td class="label">(3)</td>
</tr></table>
<p>where, <em>y</em> is the phenotypes, <em>μ</em> is the intercept, <em>E</em> stands for the random environmental effect, 𝑬 ∼ <em>N</em> (0, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline"><mrow><mrow><msub><mi>V</mi><mi>E</mi></msub><msubsup><mi mathvariant="normal">σ</mi><mi>E</mi><mn>2</mn></msubsup></mrow></mrow></math></span>). <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-5" display="inline"><mrow><msub><mi>V</mi><mi>E</mi></msub></mrow></math></span> is defined by multiplying the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline"><mrow><msub><mi>Z</mi><mi>E</mi></msub></mrow></math></span> incidence matrix with its transpose; <em>g</em> is the genotypic effects, following a normal distribution of <em>g</em> ∼ <em>N</em> (0, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline"><mrow><mrow><msub><mi>V</mi><mi>g</mi></msub><msubsup><mi mathvariant="normal">σ</mi><mi>g</mi><mn>2</mn></msubsup></mrow></mrow></math></span>). <img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/95d8/12271658/a1c3a4bd9522/TPG2-18-e70076-e007.jpg" loading="lazy" alt="Inline graphic">, is determined by the incidence <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-9" display="inline"><mrow><msub><mi>Z</mi><mi>g</mi></msub></mrow></math></span> and the genomic relatedness matrix (GRM); and finally <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-10" display="inline"><mrow><mi mathvariant="normal">ε</mi></mrow></math></span> ∼ <em>N</em>(0, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-11" display="inline"><mrow><msubsup><mi mathvariant="normal">σ</mi><mi mathvariant="normal">ε</mi><mn>2</mn></msubsup></mrow></math></span>) is the independent and identically distributed residuals.</p></section><section id="tpg270076-sec-0130"><h5 class="pmc_sec_title">Model 2: Additive genetics, environmental and genotype‐by‐environment interaction effects (g + E + G × E model)</h5>
<table class="disp-formula p" id="tpg270076-disp-0004"><tr>
<td class="formula"><math id="jats-math-12" display="block"><mrow><mrow><mi>y</mi><mo linebreak="badbreak">=</mo><mi mathvariant="normal">μ</mi><mo linebreak="goodbreak">+</mo><mi mathvariant="normal">E</mi><mo linebreak="goodbreak">+</mo><mi mathvariant="normal">g</mi><mo linebreak="goodbreak">+</mo><mi mathvariant="normal">G</mi><mo linebreak="goodbreak">×</mo><mrow></mrow><mi mathvariant="normal">E</mi><mo linebreak="goodbreak">+</mo><mi mathvariant="normal">ε</mi></mrow></mrow></math></td>
<td class="label">(4)</td>
</tr></table>
<p>where, <em>y</em>, <em>μ</em>, <em>E</em>, <em>G</em>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-13" display="inline"><mrow><mi mathvariant="normal">ε</mi></mrow></math></span> were defined in Model 1; G × E is the genotype by environment interaction G × E ∼ <em>N</em>(0, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-14" display="inline"><mrow><mrow><msub><mi>V</mi><mrow><mi>g</mi><mi>E</mi></mrow></msub><msubsup><mi mathvariant="normal">σ</mi><mrow><mi>g</mi><mi>E</mi></mrow><mn>2</mn></msubsup></mrow></mrow></math></span>), where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-15" display="inline"><mrow><mrow><msub><mi>V</mi><mrow><mi>g</mi><mi>E</mi></mrow></msub><mo>=</mo><msub><mi>V</mi><mi>g</mi></msub><mo>⊙</mo><msub><mi>V</mi><mi>E</mi></msub></mrow></mrow></math></span>; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-16" display="inline"><mrow><mo>⊙</mo></mrow></math></span> is the cell‐to‐cell or the Hadamard multiplication.</p>
<p>
<em>Model implementation and fitting</em>. Both models were fitted using a Bayesian approach via reproducing kernel Hilbert spaces (RKHS) regression (de los Campos et al., <a href="#tpg270076-bib-0021" class="usa-link" aria-describedby="tpg270076-bib-0021">2010</a>; Gianola &amp; Van Kaam, <a href="#tpg270076-bib-0030" class="usa-link" aria-describedby="tpg270076-bib-0030">2008</a>), as implemented in the BGLR R package (Pérez &amp; De Los Campos, <a href="#tpg270076-bib-0054" class="usa-link" aria-describedby="tpg270076-bib-0054">2014</a>). It is noteworthy that when a linear kernel, such as the GRM (G), is used to define the covariance structure for genotypic effects (<em>g</em>) within the RKHS framework, the model effectively becomes a genomic best linear unbiased prediction (GBLUP) model (de los Campos et al., <a href="#tpg270076-bib-0021" class="usa-link" aria-describedby="tpg270076-bib-0021">2010</a>; Morota &amp; Gianola, <a href="#tpg270076-bib-0052" class="usa-link" aria-describedby="tpg270076-bib-0052">2014</a>). For each analysis, the Markov chain Monte Carlo was run for a total of 20,000 iterations of which the first 10,000 were considered as burn‐in iterations and the other 10,000 iterations were used for parameter estimation. The thinning hyper‐parameter was set to 5. The number of burn‐in iterations was used to be long enough to ensure convergence through initial testing of different values in which we found that robust convergence can be achieved with burn‐in values from 1000 to 3000 iterations.</p></section></section><section id="tpg270076-sec-0140"><h4 class="pmc_sec_title">2.3.4. Evaluating the prediction accuracy using 75 prioritized and 75 random KASP variants</h4>
<p>The 192 KASP markers were filtered for minor allele frequency &lt;0.01 resulting in a total of 75 markers. To evaluate the prediction ability of the prioritized 75 KASP markers for selection compared to the whole set of markers, we rerun all the previous genomic selection analyses for all traits in two different scenarios. First, using the prioritized 75 SNPs only (Table <a href="#tpg270076-supinfo-0001" class="usa-link">S3</a>), and second, with a random subset of 75 SNPs that were randomly selected from the genotyping by sequencing SNPs and has no previously detected association with any of our studied traits. The aim of the second scenario was to ensure that the prediction accuracy using the prioritized 75 SNPs is significantly higher than the randomly selected 75 SNPs, therefore, they can be used for practical selection from the unphenotyped 4741 lines.</p></section><section id="tpg270076-sec-0150"><h4 class="pmc_sec_title">2.3.5. Cross‐validation</h4>
<p>We used two different cross‐validation (CV) methods (CV1 and CV2) as described in Burgueño et al. (<a href="#tpg270076-bib-0014" class="usa-link" aria-describedby="tpg270076-bib-0014">2012</a>) by masking a subset of the phenotypic data (one‐third of the population) as missing to predict their performance as a validation set. CV1 masks random phenotypic records from the whole dataset across environments, therefore, a single individual can have actual records in the reference set and some masked records. We run 100 replicates for each evaluated scenario in both CV strategies. Average Pearson correlation coefficient across replicates between the phenotypic values and the predicted values of the validation set in both methods was used to infer the prediction accuracy. Prediction accuracy values across all replicates were used to declare the significant differences among models and between the full genotyping by sequencing SNP set and prioritized SNPs using the Student <em>t</em>‐test.</p>
<p>A diagram visualizing all the steps described from the phenotyping to genotyping and genomic selection is presented in Figure <a href="#tpg270076-supinfo-0001" class="usa-link">S1</a>.</p></section></section></section><section id="tpg270076-sec-0160"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270076-sec-0170"><h3 class="pmc_sec_title">3.1. Phenotypic results</h3>
<p>Starting with the phenotyping for herbicide tolerance experiments, significant differences were observed among genotypes and among treatments for all evaluated quantitative traits at a <em>p‐</em>value &lt; 0.001 indicating a wide range of variation in herbicide tolerance in faba bean across both genotypic and treatment factors. The genotype × environment interactions were also significant for all traits except for NBrPLT and GCC as they were only recorded in one location.</p>
<p>Our findings indicated also that the first (HDS1) and second (HDS2) herbicide damage scores for each genotype ranged from 1 to 5 across all trials. When comparing the results of both herbicide damage scores (HDS1 and HDS2), we observed that 1 month after herbicide application, 5% and 42% of the genotypes had recovered from the damage caused by metribuzin and imazethapyr treatments, while the damage worsened in 56% and 10% of the genotypes, respectively.</p>
<p>As for the phenotyping for heat tolerance experiments, significant differences were observed among genotypes and among environments for all evaluated traits at a <em>p‐</em>value &lt; 0.001 indicating a wide range of genotypic variation in faba bean for heat tolerance. The G × E interaction was also significant for all traits except for GY (<em>p</em> &lt; 0.398), indicating that yield performance of the tested accessions remained the same across environments. This could be attributed to the similar environmental conditions across the different locations as all trials were conducted under heat stress conditions, during the summer in Lebanon and the United States and during the winter in Sudan.</p>
<p>Spatial analysis of each season–location combination revealed considerable variability in genotypes for DFLR, DMAT, GY, NSPLT, NPPLT, HSW, PG, and Cl for all locations/seasons. PLHT showed significant variation at all locations, except Hudeiba in 2017 while no significant difference was observed for CT for all locations/seasons.</p></section><section id="tpg270076-sec-0180"><h3 class="pmc_sec_title">3.2. Genomic prediction accuracy using all‐SNPs</h3>
<p>The estimated prediction accuracies obtained in CV1 and CV2 for the studied traits under herbicide and heat stress environments considering all SNPs are presented in Table <a href="#tpg270076-tbl-0001" class="usa-link">1</a>.</p>
<section class="tw xbox font-sm" id="tpg270076-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Prediction accuracy values obtained in CV1 and CV2 (where CV is cross‐validation) for the studied traits under heat (HT) and herbicide (H) stress environments considering all single nucleotide polymorphisms (SNPs) using two different prediction models (g + E and g + E + G × E).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Environment</th>
<th align="left" rowspan="1" colspan="1">Trait</th>
<th align="left" rowspan="1" colspan="1">CV1_g + E</th>
<th align="left" rowspan="1" colspan="1">CV1_g +E + G × E</th>
<th align="left" rowspan="1" colspan="1">CV2_g + E</th>
<th align="left" rowspan="1" colspan="1">CV2_g + E + G × E</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="10" align="left" colspan="1">Heat stress</td>
<td align="center" rowspan="1" colspan="1">Ht_CI</td>
<td align="center" rowspan="1" colspan="1">0.1 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.14 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.16 (± 0.14)</td>
<td align="center" rowspan="1" colspan="1">0.17 (± 0.13)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ht_CT</td>
<td align="center" rowspan="1" colspan="1">0.02 (± 0.04)</td>
<td align="center" rowspan="1" colspan="1">0.04 (± 0.06)</td>
<td align="center" rowspan="1" colspan="1">0.05 (± 0.07)</td>
<td align="center" rowspan="1" colspan="1">0.05 (± 0.07)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ht_DFLR</td>
<td align="center" rowspan="1" colspan="1">0.23 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.24 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.53 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.54 (± 0.1)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ht_DMAT</td>
<td align="center" rowspan="1" colspan="1">0.02 (± 0.04)</td>
<td align="center" rowspan="1" colspan="1">0.02 (± 0.04)</td>
<td align="center" rowspan="1" colspan="1">0.06 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.05 (± 0.08)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ht_GYPLT</td>
<td align="center" rowspan="1" colspan="1">0.08 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.12 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.17 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.18 (± 0.1)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ht_HSW</td>
<td align="center" rowspan="1" colspan="1">0.23 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.18 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.11 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.14 (± 0.11)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ht_NPPLT</td>
<td align="center" rowspan="1" colspan="1">0.11 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.16 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.29 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.28 (± 0.13)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ht_NSPLT</td>
<td align="center" rowspan="1" colspan="1">0.11 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.13 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.21 (± 0.14)</td>
<td align="center" rowspan="1" colspan="1">0.19 (± 0.14)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ht_PG</td>
<td align="center" rowspan="1" colspan="1">0.25 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.22 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.16 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.19 (± 0.1)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ht_PLHT</td>
<td align="center" rowspan="1" colspan="1">0.39 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.38 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.54 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.55 (± 0.1)</td>
</tr>
<tr>
<td rowspan="11" align="left" colspan="1">Herbicide stress</td>
<td align="center" rowspan="1" colspan="1">H_DFLR</td>
<td align="center" rowspan="1" colspan="1">0.41 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.4 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.69 (± 0.08)</td>
<td align="center" rowspan="1" colspan="1">0.69 (± 0.08)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_DMAT</td>
<td align="center" rowspan="1" colspan="1">0.26 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.21 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.33 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.32 (± 0.12)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_GCC</td>
<td align="center" rowspan="1" colspan="1">0.3 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.3 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.43 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.42 (± 0.09)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_GY</td>
<td align="center" rowspan="1" colspan="1">0.32 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.3 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.52 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.51 (± 0.1)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_GYPLT</td>
<td align="center" rowspan="1" colspan="1">0.32 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.31 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.48 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.47 (± 0.1)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_NBrPLT</td>
<td align="center" rowspan="1" colspan="1">0.12 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.12 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.3 (± 0.14)</td>
<td align="center" rowspan="1" colspan="1">0.28 (± 0.13)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_NPPLT</td>
<td align="center" rowspan="1" colspan="1">0.07 (± 0.08)</td>
<td align="center" rowspan="1" colspan="1">0.1 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.28 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.29 (± 0.11)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_NSPLT</td>
<td align="center" rowspan="1" colspan="1">0.06 (± 0.07)</td>
<td align="center" rowspan="1" colspan="1">0.06 (± 0.07)</td>
<td align="center" rowspan="1" colspan="1">0.43 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.42 (± 0.12)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_PLHT</td>
<td align="center" rowspan="1" colspan="1">0.23 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.23 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.46 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.46 (± 0.11)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_Score1</td>
<td align="center" rowspan="1" colspan="1">0.14 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.13 (± 0.08)</td>
<td align="center" rowspan="1" colspan="1">0.22 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.2 (± 0.1)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_Score2</td>
<td align="center" rowspan="1" colspan="1">0.12 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.14 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.31 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.31 (± 0.12)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270076-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270076-tbl1-note-0001"><p>Abbreviations: CI, chlorophyll content; CT, canopy temperature; DFLR, days to 50% flowering; DMAT, days to maturity; GCC, green canopy cover; GY, grain yield; GYPLT, grain yield per plant; HSW, 100‐seed weight; NBrPLT, number of branches per plant; NPPLT, number of pods per plant; NSPLT, number of seeds per plant; PG, pollen germination; PLHT, plant height.</p></div></div></section><p>Under heat stress environment, the highest prediction accuracies for CV1 and CV2 using two different prediction models were obtained in the case of DFLR followed by PLHT. On the other hand, the lowest values were obtained in the case of CT and DMAT. Under herbicide stress environment, the highest prediction accuracies were obtained for PLHT followed by GY and DFLR. However, the lowest values were obtained for the herbicide damage scores in CV2 and for NPPLT and NSPLT in CV1.</p>
<p>Overall, the results showed that predicting the performance of untested lines in tested environments was less challenging than predicting the performance of lines that were evaluated in some environments but not in others. The prediction accuracy values were higher in CV2 than in CV1 across all traits, except Ht_HSW and H_PG. In CV2, prediction accuracy varied between 0.05 and 0.54 under heat stress and between 0.15 and 0.69 in herbicide stress environments. On the other hand, in CV1, the prediction accuracy varied between 0.02 and 0.39 and between 0.06 and 0.41 under heat and herbicide stress environments respectively.</p>
<p>As for the prediction accuracies of the two different prediction models, considering G × E interaction in the g + E + G × E model does not seem to have a significant effect on the prediction accuracy of all traits, which means that despite the presence of significant G × E interactions, both models provided comparable level of prediction accuracy even when accounting for these interactions. Average gains in accuracy of 3.52% in CV2 and 5.29% in CV1 were attained for G + E + G × E over G + E in the case of heat stress environment. On the contrary, in the case of herbicide stress environment, considering the g × E interaction did not benefit the prediction accuracies and has worsened them as an average gain of −5.39% in CV2 and −3.92% in CV1 was attained for g + E over g + E + G × E. However, these small differences were not significant for all previous comparisons. Moreover, Figure <a href="#tpg270076-fig-0001" class="usa-link">1</a> confirms that the effect of the prediction model used is almost negligible as an overlap or a slight difference was noticed between their trends (Figures <a href="#tpg270076-fig-0001" class="usa-link">1</a> and <a href="#tpg270076-fig-0002" class="usa-link">2</a>) across the majority of traits.</p>
<figure class="fig xbox font-sm" id="tpg270076-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/95d8/12271658/31c7a0b072a1/TPG2-18-e70076-g002.jpg" loading="lazy" id="jats-graphic-1" height="861" width="603" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270076-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Prediction accuracy of traits recorded under heat (HT) environments obtained by two prediction models and using different single nucleotide polymorphisms (SNPs) subsets. CT, canopy temperature; DFLR, days to 50% flowering; DMAT, days to maturity; GYPLT, grain yield per plant; HSW, 100‐seed weight; NPPLT, number of pods per plant; NSPLT, number of seeds per plant; PG, pollen germination; PLHT, plant height.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270076-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/95d8/12271658/c0ee200a74e2/TPG2-18-e70076-g001.jpg" loading="lazy" id="jats-graphic-3" height="865" width="602" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270076-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Prediction accuracy of traits recorded under herbicide (H) environments obtained by two prediction models and using different single nucleotide polymorphisms (SNPs) subsets. DFLR, days to 50% flowering; DMAT, days to maturity; GCC, green canopy cover; GYPLT, grain yield per plant; NBrPLT, number of branches per plant; NPPLT, number of pods per plant; NSPLT, number of seeds per plant; PLHT, plant height.</p></figcaption></figure></section><section id="tpg270076-sec-0190"><h3 class="pmc_sec_title">3.3. Genomic prediction accuracy using 75 prioritized and 75 random SNPs</h3>
<p>The estimated prediction accuracies obtained in CV1 and CV2 for the traits studied under herbicide and heat stress environments considering the 75 prioritized SNPs and 75 random SNPs are presented in Table <a href="#tpg270076-tbl-0002" class="usa-link">2</a>. Similarly to the results obtained in the case of all SNPs, the prediction accuracy differed also between traits in both CV1 and CV2. Using the two different prediction models, prediction accuracies obtained in CV1 and CV2 were the highest in the case of DFLR followed by PLHT under heat stress environment and in the case of DFLR followed by GYPLT under herbicide stress environment.</p>
<section class="tw xbox font-sm" id="tpg270076-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Prediction accuracy values obtained in CV1 and CV2 for the studied traits under heat (HT) and herbicide (H) stress environments considering 75 prioritized and 75 random single nucleotide polymorphisms (SNPs) using two different prediction models (g + E and g + E + G × E).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<col align="char" char="plusmn" span="1">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Environment</th>
<th align="left" rowspan="2" colspan="1">Trait</th>
<th align="left" colspan="4" style="border-bottom:solid 1px #000000" rowspan="1">75 prioritized SNPs</th>
<th align="left" colspan="4" style="border-bottom:solid 1px #000000" rowspan="1">75 random SNPs</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">CV1_g + E</th>
<th align="left" rowspan="1" colspan="1">CV1_ g + E + G × E</th>
<th align="left" rowspan="1" colspan="1">CV2_g + E</th>
<th align="left" rowspan="1" colspan="1">CV2_ g + E + G × E</th>
<th align="left" rowspan="1" colspan="1">CV1_g + E</th>
<th align="left" rowspan="1" colspan="1">CV1_ g + E + G × E</th>
<th align="left" rowspan="1" colspan="1">CV2_g + E</th>
<th align="left" rowspan="1" colspan="1">CV2 _g + E + G × E</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="10" align="left" colspan="1">
<strong>Heat stress</strong>
</td>
<td align="center" rowspan="1" colspan="1">Ht_CI</td>
<td align="center" rowspan="1" colspan="1">0.11 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.09 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.13 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.1 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.07 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.07 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.1 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.09 (± 0.1)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ht_CT</td>
<td align="center" rowspan="1" colspan="1">0.04 (± 0.07)</td>
<td align="center" rowspan="1" colspan="1">0.06 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.05 (± 0.07)</td>
<td align="center" rowspan="1" colspan="1">0.07 (± 0.08)</td>
<td align="center" rowspan="1" colspan="1">0.06 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.06 (± 0.08)</td>
<td align="center" rowspan="1" colspan="1">0.07 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.06 (± 0.08)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ht_DFLR</td>
<td align="center" rowspan="1" colspan="1">0.23 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.23 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.5 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.47 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.08 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.08 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.36 (± 0.14)</td>
<td align="center" rowspan="1" colspan="1">0.32 (± 0.14)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ht_DMAT</td>
<td align="center" rowspan="1" colspan="1">0.1 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.09 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.1 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.11 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.05 (± 0.08)</td>
<td align="center" rowspan="1" colspan="1">0.06 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.07 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.07 (± 0.09)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ht_GYPLT</td>
<td align="center" rowspan="1" colspan="1">0.2 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.22 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.26 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.27 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.07 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.07 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.12 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.11 (± 0.11)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ht_HSW</td>
<td align="center" rowspan="1" colspan="1">0.13 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.16 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.12 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.17 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.12 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.11 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.11 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.1 (± 0.11)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ht_NPPLT</td>
<td align="center" rowspan="1" colspan="1">0.19 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.18 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.26 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.24 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.08 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.08 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.19 (± 0.14)</td>
<td align="center" rowspan="1" colspan="1">0.17 (± 0.13)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ht_NSPLT</td>
<td align="center" rowspan="1" colspan="1">0.17 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.16 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.23 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.22 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.09 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.09 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.14 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.13 (± 0.11)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ht_PG</td>
<td align="center" rowspan="1" colspan="1">0.15 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.14 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.13 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.14 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.11 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.1 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.13 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.12 (± 0.11)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">Ht_PLHT</td>
<td align="center" rowspan="1" colspan="1">0.16 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.15 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.4 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.35 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.11 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.11 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.36 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.31 (± 0.14)</td>
</tr>
<tr>
<td rowspan="11" align="left" colspan="1">
<strong>Herbicide stress</strong>
</td>
<td align="center" rowspan="1" colspan="1">H_DFLR</td>
<td align="center" rowspan="1" colspan="1">0.16 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.15 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.54 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.51 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.13 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.12 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.5 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.47 (± 0.13)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_DMAT</td>
<td align="center" rowspan="1" colspan="1">0.11 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.1 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.28 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.19 (± 0.08)</td>
<td align="center" rowspan="1" colspan="1">0.09 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.08 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.25 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.16 (± 0.12)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_GCC</td>
<td align="center" rowspan="1" colspan="1">0.07 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.08 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.22 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.15 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.14 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.14 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.3 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.23 (± 0.11)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_GY</td>
<td align="center" rowspan="1" colspan="1">0.09 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.09 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.34 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.27 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.13 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.13 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.35 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.29 (± 0.14)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_GYPLT</td>
<td align="center" rowspan="1" colspan="1">0.12 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.12 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.35 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.33 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.13 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.13 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.35 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.32 (± 0.13)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_NBrPLT</td>
<td align="center" rowspan="1" colspan="1">0.06 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.05 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.19 (± 0.08)</td>
<td align="center" rowspan="1" colspan="1">0.13 (± 0.07)</td>
<td align="center" rowspan="1" colspan="1">0.06 (± 0.08)</td>
<td align="center" rowspan="1" colspan="1">0.06 (± 0.08)</td>
<td align="center" rowspan="1" colspan="1">0.16 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.11 (± 0.12)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_NPPLT</td>
<td align="center" rowspan="1" colspan="1">0.05 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.04 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.16 (± 0.07)</td>
<td align="center" rowspan="1" colspan="1">0.12 (± 0.06)</td>
<td align="center" rowspan="1" colspan="1">0.06 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.07 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.18 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.15 (± 0.12)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_NSPLT</td>
<td align="center" rowspan="1" colspan="1">0.06 (± 0.14)</td>
<td align="center" rowspan="1" colspan="1">0.06 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.27 (± 0.07)</td>
<td align="center" rowspan="1" colspan="1">0.21 (± 0.07)</td>
<td align="center" rowspan="1" colspan="1">0.05 (± 0.08)</td>
<td align="center" rowspan="1" colspan="1">0.05 (± 0.08)</td>
<td align="center" rowspan="1" colspan="1">0.27 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.21 (± 0.13)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_PLHT</td>
<td align="center" rowspan="1" colspan="1">0.08 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.08 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.3 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.25 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.09 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.09 (± 0.11)</td>
<td align="center" rowspan="1" colspan="1">0.31 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.26 (± 0.13)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_Score1</td>
<td align="center" rowspan="1" colspan="1">0.05 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.04 (± 0.08)</td>
<td align="center" rowspan="1" colspan="1">0.12 (± 0.08)</td>
<td align="center" rowspan="1" colspan="1">0.08 (± 0.07)</td>
<td align="center" rowspan="1" colspan="1">0.08 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.08 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.14 (± 0.13)</td>
<td align="center" rowspan="1" colspan="1">0.11 (± 0.11)</td>
</tr>
<tr>
<td align="center" rowspan="1" colspan="1">H_Score2</td>
<td align="center" rowspan="1" colspan="1">0.05 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.05 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.17 (± 0.07)</td>
<td align="center" rowspan="1" colspan="1">0.11 (± 0.07)</td>
<td align="center" rowspan="1" colspan="1">0.08 (± 0.09)</td>
<td align="center" rowspan="1" colspan="1">0.08 (± 0.1)</td>
<td align="center" rowspan="1" colspan="1">0.19 (± 0.12)</td>
<td align="center" rowspan="1" colspan="1">0.15 (± 0.11)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270076-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270076-tbl2-note-0001"><p>Abbreviations: CI, chlorophyll content; CT, canopy temperature; DFLR, days to 50% flowering; DMAT, days to maturity; GY, grain yield; GYPLT, grain yield per plant; HSW, 100‐seed weight; NBrPLT, number of branches per plant; NPPLT, number of pods per plant; NSPLT, number of seeds per plant; PG, pollen germination; PLHT, plant height.</p></div></div></section><p>On the other hand, the lowest values were obtained in CV1 and CV2 for CT and DMAT under heat stress environment in both prioritized and random SNPs. Under herbicide stress environment, the lowest values were obtained for the herbicide damage scores in the case of 75 prioritized SNPs and for NBrPLT and NSPPLT in the case of the 75 random SNPs. Moreover, similarly to the results obtained in the case of all SNPs, prediction accuracies are greater in CV2 than in CV1 values across all traits except Ht_HSW in both the 75 prioritized and random SNPs and H_PG in the case of 75 prioritized SNPs.</p>
<p>On average, prediction accuracies of the 75 prioritized SNPs were slightly lower than the ones obtained in the case of all SNPs in CV1 and CV2 in the case of heat stress environment but not in herbicide stress environment (Figures <a href="#tpg270076-fig-0001" class="usa-link">1</a> and <a href="#tpg270076-fig-0002" class="usa-link">2</a>). However, this was not true in the case of the 75 random SNPs as the prediction accuracies obtained using this SNPs subset were significantly lower than the ones obtained in all SNPs cases under both heat and herbicide stress environments.</p>
<p>As for the prediction accuracies of the two different prediction models, the gain resulting from taking into consideration G × E interaction in the g + E + G × E model was negative in CV1 and CV2 in the case of the two different 75 SNPs subsets. Under heat stress environment, the average gains in CV2 were equal to −2.35% and −10% and in CV1 −29.41% and −28% in the 75 prioritized and 75 random SNPs, respectively. As for the herbicide stress environment, the average gains in CV2 were equal to −29.41% and −28.43% and in CV1 −0.98% and −0.98 in the 75 prioritized and 75 random SNPs, respectively.</p>
<p>When comparing the effect of the number and composition of SNPs subsets on the prediction accuracy, the results visualized in Figures <a href="#tpg270076-fig-0001" class="usa-link">1</a> and <a href="#tpg270076-fig-0002" class="usa-link">2</a> show that regardless of the model used, the full set of SNPs gave higher prediction compared to the two other subsets as the accuracy tends to rapidly increase in almost all of the studied traits when the number of SNPs involved increase. This increase is especially noticed in herbicide stress environments (Figure <a href="#tpg270076-fig-0002" class="usa-link">2</a>), which means that using a smaller SNPs subset might be effective under heat stress environments more than under herbicide stress environments.</p>
<p>In addition, the results also showed that the size of SNPs subset had the maximum impact on prediction accuracies followed by its composition whether it consists of prioritized or random SNPs increases. Predictions obtained using prioritized SNPs were better than using random SNPs in heat stress environments, contrary to herbicide stress environments where on average, predictions were comparable in both subsets.</p></section><section id="tpg270076-sec-0200"><h3 class="pmc_sec_title">3.4. Relationship between phenotype‐based heritability and prediction accuracy</h3>
<p>Phenotypic heritability <em>h<sup>2</sup>
</em> was estimated for all traits in the heat and herbicide stress environments and correlation was estimated between <em>h<sup>2</sup>
</em> and the prediction accuracies obtained using the two different models and all SNPs set.</p>
<p>Under heat stress environments, DFLR and PLHT, the two traits with the highest genomic prediction accuracy also exhibited the highest <em>h<sup>2</sup>
</em>. This pattern was similarly observed in herbicide stress environments, where GYPLT, along with DFLR and PLHT, had the highest <em>h<sup>2</sup>
</em> under the three different herbicide treatments.</p>
<p>Highly significant and strong correlations (0.79 and 0.8) were observed between <em>h<sup>2</sup>
</em> and the prediction accuracy obtained for CV1 with g + E and g + E + G × E models, respectively. Also, highly significant and moderate correlations (0.59 and 0.58) were obtained between <em>h<sup>2</sup>
</em> and the prediction accuracy obtained for CV2 with g + E and g + E+ G × E models, respectively.</p></section></section><section id="tpg270076-sec-0210"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>Before applying any genomic selection, it is essential to evaluate the accuracy of genomic prediction models and to compare different sets of statistical models. In this study, the accuracy of genomic prediction was evaluated in faba bean for heat and herbicide tolerance related traits considering two CV schemes (CV1 and CV2) with two scenarios, the g + E + G × E and g +  E models. Several factors have been taken into consideration including the size and the composition of the SNPs set. To the best of our knowledge, this is the first study that deals with genomic selection in faba bean. The limited global resources allocated to this crop have constrained genomic research progress, given the huge size of the faba bean genome.</p>
<p>Our results show that predicting the performance of untested lines under tested environments is less accurate than predicting the performance of lines tested in some but not all environments, as higher prediction accuracies were recorded in CV2 than in CV1. However, the difference is small and not statistically significant for many heat stress traits (CI, CT, DMAT, and GYPLT). For other traits (herbicide tolerance traits, as well as heat tolerance DFLR, NPPL, NSPL, and PLHT) showed significantly higher prediction accuracy in CV2, while the remaining traits (e.g., HSW and PG for heat stress) showed significantly higher prediction accuracy in CV1. This finding stems from the size of the reference population used in the present study as the prediction assessment in the CV scheme CV1 is based on information borrowed from other lines within the same environment or across environments (Burgueño et al., <a href="#tpg270076-bib-0014" class="usa-link" aria-describedby="tpg270076-bib-0014">2012</a>). This means that the larger the reference population, the better the prediction, while such contradictory results could happen due to our small reference population. Our results align with Jighly (<a href="#tpg270076-bib-0040" class="usa-link" aria-describedby="tpg270076-bib-0040">2024</a>) results, who also used such a small population size, of having contradictory results of superiority of CV1 and CV2 across different traits. However, most of previous genomic selection studies (e.g., Roorkiwal et al., <a href="#tpg270076-bib-0056" class="usa-link" aria-describedby="tpg270076-bib-0056">2018</a>) reported that the performance of prediction of untested genotypes (CV1) resulted in lower prediction accuracies compared to other validation schemes, which is the case for most of our traits.</p>
<p>It has been reported previously that the accuracy of genomic predictions tends to correlate positively with the increase in the reference population size (Hayes et al., <a href="#tpg270076-bib-0036" class="usa-link" aria-describedby="tpg270076-bib-0036">2009</a>; Tan et al., <a href="#tpg270076-bib-0061" class="usa-link" aria-describedby="tpg270076-bib-0061">2017</a>). Nevertheless, expanding the size of the reference population is costly due to the high expenses associated with phenotyping and it is challenging as the quality of collected data may be affected by the large‐scale phenotyping trials (Alemu et al., <a href="#tpg270076-bib-0006" class="usa-link" aria-describedby="tpg270076-bib-0006">2024</a>). In addition, it has also been reported that in some cases prediction accuracy may stabilize once an optimal size of the reference population is reached (Grattapaglia et al., <a href="#tpg270076-bib-0031" class="usa-link" aria-describedby="tpg270076-bib-0031">2011</a>; Arruda et al., <a href="#tpg270076-bib-0008" class="usa-link" aria-describedby="tpg270076-bib-0008">2015</a>; Tan et al., <a href="#tpg270076-bib-0061" class="usa-link" aria-describedby="tpg270076-bib-0061">2017</a>; Sverrisdóttir et al., <a href="#tpg270076-bib-0060" class="usa-link" aria-describedby="tpg270076-bib-0060">2018</a>; Fernández‐González et al., <a href="#tpg270076-bib-0025" class="usa-link" aria-describedby="tpg270076-bib-0025">2023</a>). Therefore, optimizing the size of the training population in a way that increases accuracy with minimal cost is needed. To do so, many factors should be considered like heritability of the traits and genetic relatedness of the reference set. For instance, previous studies suggested that including genetically unrelated individuals in the training population affects negatively the accuracy of the genomic prediction (Clark et al., <a href="#tpg270076-bib-0016" class="usa-link" aria-describedby="tpg270076-bib-0016">2012</a>; Habier et al., <a href="#tpg270076-bib-0034" class="usa-link" aria-describedby="tpg270076-bib-0034">2010</a>; Lorenz &amp; Smith, <a href="#tpg270076-bib-0046" class="usa-link" aria-describedby="tpg270076-bib-0046">2015</a>). In the present study, the reference population consists of 120 genetically diverse lines originating from different countries, which may have affected negatively the accuracy of the models. Therefore, the composition of the reference population is as important as its size, and developing a reference population where the selection candidates are well‐represented is of great importance in genomic prediction to ensure that their genetic diversity is well captured and to minimize the probability of losing rare favorable alleles (Akdemir &amp; Isidro‐Sa´nchez, <a href="#tpg270076-bib-0004" class="usa-link" aria-describedby="tpg270076-bib-0004">2019</a>; Akdemir et al., <a href="#tpg270076-bib-0005" class="usa-link" aria-describedby="tpg270076-bib-0005">2015</a>; Lorenz &amp; Smith, <a href="#tpg270076-bib-0046" class="usa-link" aria-describedby="tpg270076-bib-0046">2015</a>). Since genomic prediction accuracy relies on the relationship between genetic markers and phenotypic traits, adding these lines to the reference population will enrich the genotypic data and phenotypic information and will therefore enhance its ability to predict the phenotypic performance. Tan et al. (<a href="#tpg270076-bib-0061" class="usa-link" aria-describedby="tpg270076-bib-0061">2017</a>) found that adding parental lines to the reference population increases the prediction accuracy.</p>
<p>Adding to that the low to moderate heritability of some traits, especially in the case of heat tolerance, explains the lower accuracy associated. Several research studies have demonstrated that prediction accuracy increases with the increase in trait heritability (Lorenzana &amp; Bernardo, <a href="#tpg270076-bib-0047" class="usa-link" aria-describedby="tpg270076-bib-0047">2009</a>; Zhong et al., <a href="#tpg270076-bib-0067" class="usa-link" aria-describedby="tpg270076-bib-0067">2009</a>; Jannink et al., <a href="#tpg270076-bib-0038" class="usa-link" aria-describedby="tpg270076-bib-0038">2010</a>; Combs &amp; Bernardo, <a href="#tpg270076-bib-0018" class="usa-link" aria-describedby="tpg270076-bib-0018">2013</a>; Kaler et al., <a href="#tpg270076-bib-0041" class="usa-link" aria-describedby="tpg270076-bib-0041">2022</a>), and this was the case in our study where highly significant and positive correlation was observed between the trait's heritability and the prediction accuracy. Bernarndo (<a href="#tpg270076-bib-0010" class="usa-link" aria-describedby="tpg270076-bib-0010">2016</a>) suggested to increase the prediction accuracy, low heritable traits should be compensated by increasing the size of the reference population. The results reported by Tan et al. (<a href="#tpg270076-bib-0061" class="usa-link" aria-describedby="tpg270076-bib-0061">2017</a>) confirmed that an increase in reference population size led to an increase in the accuracy of traits with low heritability.</p>
<p>To increase cost‐effectiveness, genomic selection analyses were rerun for all traits using a reduced number of SNPs that were previously shown to be associated with heat tolerance. Our findings show that a slight reduction in the prediction accuracy was observed as the number of KASP markers declined in the case of heat tolerance. Earlier studies have also found that a lower number of SNPs can provide equivalent prediction accuracies compared to using all SNPs (Zhang et al., <a href="#tpg270076-bib-0066" class="usa-link" aria-describedby="tpg270076-bib-0066">2011</a>; Su et al., <a href="#tpg270076-bib-0059" class="usa-link" aria-describedby="tpg270076-bib-0059">2012</a>; Tan et al., <a href="#tpg270076-bib-0061" class="usa-link" aria-describedby="tpg270076-bib-0061">2017</a>). Also, a number of studies found that increasing the number of markers in small reference populations provides little or no improvement in predictive accuracy. These findings apply to our study since we are using a small reference population. Interestingly, Kaler et al. (<a href="#tpg270076-bib-0041" class="usa-link" aria-describedby="tpg270076-bib-0041">2022</a>) concluded that additional markers might be associated with an increase in error or noise. Nevertheless, future studies should investigate the effect of increasing SNP density on prediction accuracy.</p>
<p>In the case of herbicide tolerance, our results show a more pronounced reduction in prediction accuracy. This might be due to the fact that the 75 KASP were selected from the 11,000 SNP based on their association with heat tolerance. Fewer SNPs were sufficient to capture the genetic variability and predict traits associated with heat tolerance accurately, but additional genome exploration is needed to provide sufficient coverage for the herbicide tolerance traits prediction.</p>
<p>As for the subset of SNPs used, our results show a significant reduction in accuracy when a random 75 SNPs subset was used under both heat and herbicide stress environments. For herbicide tolerance, the results of the randomly selected 75 SNPs were comparable to the 75 KASPs indicating the inefficiency of the KASP markers to predict herbicide tolerance. One the other hand, the prediction accuracy of the randomly selected 75 SNPs was significantly lower than that obtained from the KASPs for heat tolerance, indicating the superiority of the later. This also means that using the 75 random SNPs could lead to biased observations as it does not favor variants contributing to heat and herbicide tolerance related traits. Our results are in line with previous studies that found utilizing QTL‐linked markers increased accuracy more than using a random set of markers (Thavamanikumar et al., <a href="#tpg270076-bib-0062" class="usa-link" aria-describedby="tpg270076-bib-0062">2015</a>; Kaler et al., <a href="#tpg270076-bib-0041" class="usa-link" aria-describedby="tpg270076-bib-0041">2022</a>). Contrarily, Poland et al. (<a href="#tpg270076-bib-0055" class="usa-link" aria-describedby="tpg270076-bib-0055">2012</a>) and Spindel et al. (<a href="#tpg270076-bib-0058" class="usa-link" aria-describedby="tpg270076-bib-0058">2015</a>) observed no change in prediction accuracy after using random subsets of markers.</p>
<p>To summarize, this study is a proof‐of‐concept that genomic selection is applicable in faba bean, but higher genome coverage and larger training population are needed to achieve a better prediction accuracy and as a result a more accurate selection. In this regard, ICARDA is collaborating with international partners to develop high coverage SNPs markers (ICARDA, unpublished data, 2024). Also, a diverse training population that includes parental lines in addition to the lines having the highest GEBV for DFLR and PLHT in the case of heat tolerance and the highest GEBV for DFLR, GYPLT, and PLHT in the case of herbicide tolerance will be developed and used for further studies.</p></section><section id="tpg270076-sec-0220"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Lynn Abou Khater</strong>: Investigation; visualization; writing—original draft. <strong>Reem Joukhadar</strong>: Formal analysis; software; writing—original draft. <strong>Fouad Maalouf</strong>: Conceptualization; funding acquisition; investigation; methodology; resources; supervision; validation; writing—review and editing. <strong>Alsamman M. Alsamman</strong>: Software. <strong>Zayed Babiker</strong>: Investigation. <strong>Rind Balech</strong>: Investigation; writing—review and editing. <strong>Jinguo Hu</strong>: Investigation. <strong>Yu Ma</strong>: Investigation; writing—review and editing. <strong>Andrew Dunham</strong>: Investigation. <strong>Miguel Sanchez</strong>: Resources; writing—review and editing. <strong>Abdulqader Jighly</strong>: Formal analysis; methodology; software; writing—review and editing. <strong>Shiv Kumar</strong>: Funding acquisition; writing—review and editing</p></section><section id="tpg270076-sec-0240"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec24"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270076-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270076-supitem-0001"><div class="caption p"><p>Table S1 presents the list of evaluated genotypes along with their details.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12271658/bin/TPG2-18-e70076-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70076-s001.xlsx</a><sup> (9MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270076-supitem-0002"><div class="caption p"><p>Table S2 presents the list of 192 KASP markers along with the corresponding sequences developed based on the results obtained by Abou Khater et al., <a href="#tpg270076-bib-0001" class="usa-link" aria-describedby="tpg270076-bib-0001">2022</a>, Maalouf et al., 2022 and Guiterrez et al., <a href="#tpg270076-bib-0033" class="usa-link" aria-describedby="tpg270076-bib-0033">2023</a> for heat, herbicide and drought tolerance.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12271658/bin/TPG2-18-e70076-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70076-s002.xlsx</a><sup> (24.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270076-supitem-0003"><div class="caption p"><p>Table S3 presents the list of 75 KASP markers for heat tolerance traits prioritized to estimate the PA.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12271658/bin/TPG2-18-e70076-s004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70076-s004.xlsx</a><sup> (10.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270076-supitem-0004"><div class="caption p"><p>Figure S1 presents a diagram visualizing the stepwise workflow from phenotyping to genotyping and genomic selection.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12271658/bin/TPG2-18-e70076-s003.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70076-s003.pdf</a><sup> (71.1KB, pdf) </sup>
</div></div></section></section><section id="tpg270076-sec-0230" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>This research was funded by Arab Fund for Economic and Social Development (AFESD) Modernization of Crop Breeding Programs at ICARDA project, agreement number 200205, Accelerated Breeding Initiatives (ABI), agreement number 200308, and by Breeding Food Legumes for Increased Sustainability (BELIS), agreement number 200359.</p></section><section id="notes1"><p>


Abou Khater, L.
, 
Joukhadar, R.
, 
Maalouf, F.
, 
Alsamman, A. M.
, 
Babiker, Z.
, 
Balech, R.
, 
Hu, J.
, 
Ma, Y.
, 
Dunham, A.
, 
Sanchez, M.
, 
Jighly, A.
, &amp; 
Kumar, S.
 (2025). Genomic prediction for heat and herbicide tolerance in faba bean. The Plant Genome, 18, e70076. 10.1002/tpg2.70076

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="tpg270076-note-0001"><p>Assigned to Associate Editor Diego Jarquin.</p></div>
<div class="fn p" id="tpg270076-note-0002"><p>Lynn Abou Khater and Reem Joukhadar contributed equally to this study.</p></div>
</div></section></section><section id="tpg270076-sec-0260"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>All phenotypic and genotypic data including sequence data used in this study will be available at the open‐source research data depository MELSpace (<a href="https://repo.mel.cgiar.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://repo.mel.cgiar.org</a>).</p></section><section id="tpg270076-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270076-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270076-bib-0001">
<cite>

Abou‐Khater, L.
, 
Maalouf, F.
, 
Jighly, A.
, 
Alsamman, A. M.
, 
Rubiales, D.
, 
Rispail, N.
, 
Hu, J.
, 
Ma, Y.
, 
Balech, R.
, 
Hamwieh, A.
, 
Baum, M.
, &amp; 
Kumar, S.
 (2022). Genomic regions associated with herbicide tolerance in a worldwide faba bean (<em>Vicia faba</em> L.) collection. Scientific Reports, 12(1), 158. 10.1038/s41598-021-03861-0

</cite> [<a href="https://doi.org/10.1038/s41598-021-03861-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8741826/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34996977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Genomic%20regions%20associated%20with%20herbicide%20tolerance%20in%20a%20worldwide%20faba%20bean%20(Vicia%20faba%20L.)%20collection&amp;author=L.%20Abou%E2%80%90Khater&amp;author=F.%20Maalouf&amp;author=A.%20Jighly&amp;author=A.%20M.%20Alsamman&amp;author=D.%20Rubiales&amp;volume=12&amp;issue=1&amp;publication_year=2022&amp;pages=158&amp;pmid=34996977&amp;doi=10.1038/s41598-021-03861-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0002">
<cite>

Abou‐Khater, L.
, 
Maalouf, F.
, 
Patil, S. B.
, 
Balech, R.
, 
Nacouzi, D.
, 
Rubiales, D.
, &amp; 
Kumar, S.
 (2021). Identification of tolerance to metribuzin and imazethapyr herbicides in faba bean. Crop Science, 61(4), 2593–2611. 10.1002/csc2.20474
</cite> [<a href="https://doi.org/10.1002/csc2.20474" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Identification%20of%20tolerance%20to%20metribuzin%20and%20imazethapyr%20herbicides%20in%20faba%20bean&amp;author=L.%20Abou%E2%80%90Khater&amp;author=F.%20Maalouf&amp;author=S.%20B.%20Patil&amp;author=R.%20Balech&amp;author=D.%20Nacouzi&amp;volume=61&amp;issue=4&amp;publication_year=2021&amp;pages=2593-2611&amp;doi=10.1002/csc2.20474&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0003">
<cite>

Adhikari, K. N.
, 
Khazaei, H.
, 
Ghaouti, L.
, 
Maalouf, F.
, 
Vandenberg, A.
, 
Link, W.
, &amp; 
O'Sullivan, D. M.
 (2021). Conventional and molecular breeding tools for accelerating genetic gain in faba bean (<em>Vicia faba</em> L.). Frontiers in Plant Science, 12, 744259. 10.3389/fpls.2021.744259

</cite> [<a href="https://doi.org/10.3389/fpls.2021.744259" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8548637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34721470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Conventional%20and%20molecular%20breeding%20tools%20for%20accelerating%20genetic%20gain%20in%20faba%20bean%20(Vicia%20faba%20L.)&amp;author=K.%20N.%20Adhikari&amp;author=H.%20Khazaei&amp;author=L.%20Ghaouti&amp;author=F.%20Maalouf&amp;author=A.%20Vandenberg&amp;volume=12&amp;publication_year=2021&amp;pages=744259&amp;pmid=34721470&amp;doi=10.3389/fpls.2021.744259&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0004">
<cite>

Akdemir, D.
, &amp; 
Isidro‐Sánchez, J.
 (2019). Design of training populations for selective phenotyping in genomic prediction. Scientific Reports, 9(1), 1446. 10.1038/s41598-018-38081-6

</cite> [<a href="https://doi.org/10.1038/s41598-018-38081-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6363789/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30723226/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Design%20of%20training%20populations%20for%20selective%20phenotyping%20in%20genomic%20prediction&amp;author=D.%20Akdemir&amp;author=J.%20Isidro%E2%80%90S%C3%A1nchez&amp;volume=9&amp;issue=1&amp;publication_year=2019&amp;pages=1446&amp;pmid=30723226&amp;doi=10.1038/s41598-018-38081-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0005">
<cite>

Akdemir, D.
, 
Sanchez, J. I.
, &amp; 
Jannink, J. L.
 (2015). Optimization of genomic selection training populations with a genetic algorithm. Genetics Selection Evolution, 47, 1–10. 10.1186/s12711-015-0116-6
</cite> [<a href="https://doi.org/10.1186/s12711-015-0116-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4422310/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25943105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20Selection%20Evolution&amp;title=Optimization%20of%20genomic%20selection%20training%20populations%20with%20a%20genetic%20algorithm&amp;author=D.%20Akdemir&amp;author=J.%20I.%20Sanchez&amp;author=J.%20L.%20Jannink&amp;volume=47&amp;publication_year=2015&amp;pages=1-10&amp;pmid=25943105&amp;doi=10.1186/s12711-015-0116-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0006">
<cite>

Alemu, A.
, 
Åstrand, J.
, 
Montesinos‐López, O. A.
, 
Isidro Y Sánchez, J.
, 
Fernández‐Gónzalez, J.
, 
Tadesse, W.
, 
Vetukuri, R. R.
, 
Carlsson, A. S.
, 
Ceplitis, A.
, 
Crossa, J.
, 
Ortiz, R.
, &amp; 
Chawade, A.
 (2024). Genomic selection in plant breeding: Key factors shaping two decades of progress. Molecular Plant, 17(4), 552–578. 10.1016/j.molp.2024.03.007

</cite> [<a href="https://doi.org/10.1016/j.molp.2024.03.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38475993/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=Genomic%20selection%20in%20plant%20breeding:%20Key%20factors%20shaping%20two%20decades%20of%20progress&amp;author=A.%20Alemu&amp;author=J.%20%C3%85strand&amp;author=O.%20A.%20Montesinos%E2%80%90L%C3%B3pez&amp;author=J.%20Isidro%20Y%20S%C3%A1nchez&amp;author=J.%20Fern%C3%A1ndez%E2%80%90G%C3%B3nzalez&amp;volume=17&amp;issue=4&amp;publication_year=2024&amp;pages=552-578&amp;pmid=38475993&amp;doi=10.1016/j.molp.2024.03.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0007">
<cite>

Ali, M. B.
, 
Welna, G. C.
, 
Sallam, A.
, 
Martsch, R.
, 
Balko, C.
, 
Gebser, B.
, 
Sass, O.
, &amp; 
Link, W.
 (2016). Association analyses to genetically improve drought and freezing tolerance of faba bean (<em>Vicia faba</em> L.). Crop Science, 56(3), 1036–1048. 10.2135/cropsci2015.08.0503
</cite> [<a href="https://doi.org/10.2135/cropsci2015.08.0503" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Association%20analyses%20to%20genetically%20improve%20drought%20and%20freezing%20tolerance%20of%20faba%20bean%20(Vicia%20faba%20L.)&amp;author=M.%20B.%20Ali&amp;author=G.%20C.%20Welna&amp;author=A.%20Sallam&amp;author=R.%20Martsch&amp;author=C.%20Balko&amp;volume=56&amp;issue=3&amp;publication_year=2016&amp;pages=1036-1048&amp;doi=10.2135/cropsci2015.08.0503&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0008">
<cite>

Arruda, M. P.
, 
Brown, P. J.
, 
Lipka, A. E.
, 
Krill, A. M.
, 
Thurber, C.
, &amp; 
Kolb, F. L.
 (2015). Genomic selection for predicting <em>Fusarium</em> head blight resistance in a wheat breeding program. The Plant Genome, 8(3), plantgenome2015.01.0003. 10.3835/plantgenome2015.01.0003
</cite> [<a href="https://doi.org/10.3835/plantgenome2015.01.0003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33228272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genomic%20selection%20for%20predicting%20Fusarium%20head%20blight%20resistance%20in%20a%20wheat%20breeding%20program&amp;author=M.%20P.%20Arruda&amp;author=P.%20J.%20Brown&amp;author=A.%20E.%20Lipka&amp;author=A.%20M.%20Krill&amp;author=C.%20Thurber&amp;volume=8&amp;issue=3&amp;publication_year=2015&amp;pmid=33228272&amp;doi=10.3835/plantgenome2015.01.0003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0009">
<cite>

Badri, S.
 (2023). The effect of high temperature on reproductive biology of faba bean (Vicia faba L.) [Doctoral dissertation, The University of Sydney; ]. <a href="https://ses.library.usyd.edu.au/handle/2123/32004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ses.library.usyd.edu.au/handle/2123/32004</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=The%20effect%20of%20high%20temperature%20on%20reproductive%20biology%20of%20faba%20bean%20(Vicia%20faba%20L.)&amp;author=S.%20Badri&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0010">
<cite>

Bernardo, R.
 (2016). Genome wide predictions for backcrossing a quantitative trait from an exotic to an adapted line. Crop Science, 56(3), 1067–1075. 10.2135/cropsci2015.09.0586
</cite> [<a href="https://doi.org/10.2135/cropsci2015.09.0586" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genome%20wide%20predictions%20for%20backcrossing%20a%20quantitative%20trait%20from%20an%20exotic%20to%20an%20adapted%20line&amp;author=R.%20Bernardo&amp;volume=56&amp;issue=3&amp;publication_year=2016&amp;pages=1067-1075&amp;doi=10.2135/cropsci2015.09.0586&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0011">
<cite>

Bishop, J.
, 
Potts, S. G.
, &amp; 
Jones, H. E.
 (2016). Susceptibility of faba bean (<em>Vicia faba</em> L.) to heat stress during floral development and anthesis. Journal of Agronomy and Crop Science, 202(6), 508–517. 10.1111/jac.12172

</cite> [<a href="https://doi.org/10.1111/jac.12172" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5763371/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29353966/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Agronomy%20and%20Crop%20Science&amp;title=Susceptibility%20of%20faba%20bean%20(Vicia%20faba%20L.)%20to%20heat%20stress%20during%20floral%20development%20and%20anthesis&amp;author=J.%20Bishop&amp;author=S.%20G.%20Potts&amp;author=H.%20E.%20Jones&amp;volume=202&amp;issue=6&amp;publication_year=2016&amp;pages=508-517&amp;pmid=29353966&amp;doi=10.1111/jac.12172&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0012">
<cite>

Brewbaker, J. L.
, &amp; 
Kwack, B. H.
 (1963). The essential role of calcium ion in pollen germination and pollen tube growth. American Journal of Botany, 50(9), 859–865. 10.1002/j.1537-2197.1963.tb06564.x
</cite> [<a href="https://doi.org/10.1002/j.1537-2197.1963.tb06564.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Botany&amp;title=The%20essential%20role%20of%20calcium%20ion%20in%20pollen%20germination%20and%20pollen%20tube%20growth&amp;author=J.%20L.%20Brewbaker&amp;author=B.%20H.%20Kwack&amp;volume=50&amp;issue=9&amp;publication_year=1963&amp;pages=859-865&amp;doi=10.1002/j.1537-2197.1963.tb06564.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0013">
<cite>

Budhlakoti, N.
, 
Kushwaha, A. K.
, 
Rai, A.
, 
Chaturvedi, K. K.
, 
Kumar, A.
, 
Pradhan, A. K.
, 
Kumar, U.
, 
Kumar, R. R.
, 
Juliana, P.
, 
Mishra, D. C.
, &amp; 
Kumar, S.
 (2022). Genomic selection: A tool for accelerating the efficiency of molecular breeding for development of climate‐resilient crops. Frontiers in Genetics, 13, 832153. 10.3389/fgene.2022.832153

</cite> [<a href="https://doi.org/10.3389/fgene.2022.832153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8864149/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35222548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Genomic%20selection:%20A%20tool%20for%20accelerating%20the%20efficiency%20of%20molecular%20breeding%20for%20development%20of%20climate%E2%80%90resilient%20crops&amp;author=N.%20Budhlakoti&amp;author=A.%20K.%20Kushwaha&amp;author=A.%20Rai&amp;author=K.%20K.%20Chaturvedi&amp;author=A.%20Kumar&amp;volume=13&amp;publication_year=2022&amp;pages=832153&amp;pmid=35222548&amp;doi=10.3389/fgene.2022.832153&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0014">
<cite>

Burgueño, J.
, 
de los Campos, G.
, 
Weigel, K.
, &amp; 
Crossa, J.
 (2012). Genomic prediction of breeding values when modeling genotype × environment interaction using pedigree and dense molecular markers. Crop Science, 52(2), 707–719. 10.2135/cropsci2011.06.0299
</cite> [<a href="https://doi.org/10.2135/cropsci2011.06.0299" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Genomic%20prediction%20of%20breeding%20values%20when%20modeling%20genotype%20%C3%97%20environment%20interaction%20using%20pedigree%20and%20dense%20molecular%20markers&amp;author=J.%20Burgue%C3%B1o&amp;author=G.%20de%20los%20Campos&amp;author=K.%20Weigel&amp;author=J.%20Crossa&amp;volume=52&amp;issue=2&amp;publication_year=2012&amp;pages=707-719&amp;doi=10.2135/cropsci2011.06.0299&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0016">
<cite>

Clark, S. A.
, 
Hickey, J. M.
, 
Daetwyler, H. D.
, &amp; 
van der Werf, J. H.
 (2012). The importance of information on relatives for the prediction of genomic breeding values and the implications for the makeup of reference data sets in livestock breeding schemes. Genetics Selection Evolution, 44, 1–9. 10.1186/1297-9686-44-4
</cite> [<a href="https://doi.org/10.1186/1297-9686-44-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3299588/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22321529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20Selection%20Evolution&amp;title=The%20importance%20of%20information%20on%20relatives%20for%20the%20prediction%20of%20genomic%20breeding%20values%20and%20the%20implications%20for%20the%20makeup%20of%20reference%20data%20sets%20in%20livestock%20breeding%20schemes&amp;author=S.%20A.%20Clark&amp;author=J.%20M.%20Hickey&amp;author=H.%20D.%20Daetwyler&amp;author=J.%20H.%20van%20der%20Werf&amp;volume=44&amp;publication_year=2012&amp;pages=1-9&amp;pmid=22321529&amp;doi=10.1186/1297-9686-44-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0017">
<cite>

Clark, S. A.
, 
Hickey, J. M.
, &amp; 
Van der Werf, J. H.
 (2011). Different models of genetic variation and their effect on genomic evaluation. Genetics Selection Evolution, 43, 1–9. 10.1186/1297-9686-43-18
</cite> [<a href="https://doi.org/10.1186/1297-9686-43-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3114710/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21575265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20Selection%20Evolution&amp;title=Different%20models%20of%20genetic%20variation%20and%20their%20effect%20on%20genomic%20evaluation&amp;author=S.%20A.%20Clark&amp;author=J.%20M.%20Hickey&amp;author=J.%20H.%20Van%20der%20Werf&amp;volume=43&amp;publication_year=2011&amp;pages=1-9&amp;pmid=21575265&amp;doi=10.1186/1297-9686-43-18&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0018">
<cite>

Combs, E.
, &amp; 
Bernardo, R.
 (2013). Accuracy of genome wide selection for different traits with constant population size, heritability, and number of markers. The Plant Genome, 6(1), plantgenome2012.11.0030. 10.3835/plantgenome2012.11.0030
</cite> [<a href="https://doi.org/10.3835/plantgenome2012.11.0030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Accuracy%20of%20genome%20wide%20selection%20for%20different%20traits%20with%20constant%20population%20size,%20heritability,%20and%20number%20of%20markers&amp;author=E.%20Combs&amp;author=R.%20Bernardo&amp;volume=6&amp;issue=1&amp;publication_year=2013&amp;doi=10.3835/plantgenome2012.11.0030&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0069">
<cite>

Cubero, J. I.
 (1974). On the evolution of <em>Vicia faba</em> L. Theoretical and Applied Genetics, 45(2), 47–51.
</cite> [<a href="https://doi.org/10.1007/BF00283475" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24419274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=On%20the%20evolution%20of%20Vicia%20faba%20L&amp;author=J.%20I.%20Cubero&amp;volume=45&amp;issue=2&amp;publication_year=1974&amp;pages=47-51&amp;pmid=24419274&amp;doi=10.1007/BF00283475&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0019">
<cite>

Daetwyler, H. D.
, 
Pong‐Wong, R.
, 
Villanueva, B.
, &amp; 
Woolliams, J. A.
 (2010). The impact of genetic architecture on genome‐wide evaluation methods. Genetics, 185(3), 1021–1031. 10.1534/genetics.110.116855

</cite> [<a href="https://doi.org/10.1534/genetics.110.116855" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2907189/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20407128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=The%20impact%20of%20genetic%20architecture%20on%20genome%E2%80%90wide%20evaluation%20methods&amp;author=H.%20D.%20Daetwyler&amp;author=R.%20Pong%E2%80%90Wong&amp;author=B.%20Villanueva&amp;author=J.%20A.%20Woolliams&amp;volume=185&amp;issue=3&amp;publication_year=2010&amp;pages=1021-1031&amp;pmid=20407128&amp;doi=10.1534/genetics.110.116855&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0020">
<cite>

De Arevalo, R. C. G.
, 
Lusarreta, C. A.
, 
Neyra, C. B.
, 
Sanchez, M. A.
, &amp; 
Algarra, P. J.
 (1992). Chemical control of annual weeds in field beans (<em>Vicia faba</em>) in Central Spain. Weed Science, 40(1), 96–100. 10.1017/S0043174500057027
</cite> [<a href="https://doi.org/10.1017/S0043174500057027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Weed%20Science&amp;title=Chemical%20control%20of%20annual%20weeds%20in%20field%20beans%20(Vicia%20faba)%20in%20Central%20Spain&amp;author=R.%20C.%20G.%20De%20Arevalo&amp;author=C.%20A.%20Lusarreta&amp;author=C.%20B.%20Neyra&amp;author=M.%20A.%20Sanchez&amp;author=P.%20J.%20Algarra&amp;volume=40&amp;issue=1&amp;publication_year=1992&amp;pages=96-100&amp;doi=10.1017/S0043174500057027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0021">
<cite>

De Los Campos, G.
, 
Gianola, D.
, 
Rosa, G. J.
, 
Weigel, K. A.
, &amp; 
Crossa, J.
 (2010). Semi‐parametric genomic‐enabled prediction of genetic values using reproducing kernel Hilbert spaces methods. Genetics Research, 92(4), 295–308. 10.1017/S0016672310000285

</cite> [<a href="https://doi.org/10.1017/S0016672310000285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20943010/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20Research&amp;title=Semi%E2%80%90parametric%20genomic%E2%80%90enabled%20prediction%20of%20genetic%20values%20using%20reproducing%20kernel%20Hilbert%20spaces%20methods&amp;author=G.%20De%20Los%20Campos&amp;author=D.%20Gianola&amp;author=G.%20J.%20Rosa&amp;author=K.%20A.%20Weigel&amp;author=J.%20Crossa&amp;volume=92&amp;issue=4&amp;publication_year=2010&amp;pages=295-308&amp;pmid=20943010&amp;doi=10.1017/S0016672310000285&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0022">
<cite>

De Roos, A. P. W.
, 
Hayes, B. J.
, &amp; 
Goddard, M.
 (2009). Reliability of genomic predictions across multiple populations. Genetics, 183(4), 1545–1553. 10.1534/genetics.109.104935

</cite> [<a href="https://doi.org/10.1534/genetics.109.104935" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2787438/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19822733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Reliability%20of%20genomic%20predictions%20across%20multiple%20populations&amp;author=A.%20P.%20W.%20De%20Roos&amp;author=B.%20J.%20Hayes&amp;author=M.%20Goddard&amp;volume=183&amp;issue=4&amp;publication_year=2009&amp;pages=1545-1553&amp;pmid=19822733&amp;doi=10.1534/genetics.109.104935&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0023">
<cite>

Devasirvatham, V.
, 
Gaur, P. M.
, 
Mallikarjuna, N.
, 
Tokachichu, R. N.
, 
Trethowan, R. M.
, &amp; 
Tan, D. K.
 (2012). Effect of high temperature on the reproductive development of chickpea genotypes under controlled environments. Functional Plant Biology, 39(12), 1009–1018. 10.1071/FP12033

</cite> [<a href="https://doi.org/10.1071/FP12033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32480850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Functional%20Plant%20Biology&amp;title=Effect%20of%20high%20temperature%20on%20the%20reproductive%20development%20of%20chickpea%20genotypes%20under%20controlled%20environments&amp;author=V.%20Devasirvatham&amp;author=P.%20M.%20Gaur&amp;author=N.%20Mallikarjuna&amp;author=R.%20N.%20Tokachichu&amp;author=R.%20M.%20Trethowan&amp;volume=39&amp;issue=12&amp;publication_year=2012&amp;pages=1009-1018&amp;pmid=32480850&amp;doi=10.1071/FP12033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0024">
<cite>

Diaz, S.
, 
Ariza‐Suarez, D.
, 
Ramdeen, R.
, 
Aparicio, J.
, 
Arunachalam, N.
, 
Hernandez, C.
, 
Diaz, H.
, 
Ruiz, H.
, 
Piepho, H. P.
, &amp; 
Raatz, B.
 (2021). Genetic architecture and genomic prediction of cooking time in common bean (<em>Phaseolus vulgaris</em> L.). Frontiers in Plant Science, 11, 622213. 10.3389/fpls.2020.622213

</cite> [<a href="https://doi.org/10.3389/fpls.2020.622213" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7905357/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33643335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genetic%20architecture%20and%20genomic%20prediction%20of%20cooking%20time%20in%20common%20bean%20(Phaseolus%20vulgaris%20L.)&amp;author=S.%20Diaz&amp;author=D.%20Ariza%E2%80%90Suarez&amp;author=R.%20Ramdeen&amp;author=J.%20Aparicio&amp;author=N.%20Arunachalam&amp;volume=11&amp;publication_year=2021&amp;pages=622213&amp;pmid=33643335&amp;doi=10.3389/fpls.2020.622213&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0025">
<cite>

Fernández‐González, J.
, 
Akdemir, D.
, &amp; 
Isidro y Sánchez, J.
 (2023). A comparison of methods for training population optimization in genomic selection. Theoretical and Applied Genetics, 136(3), 30. 10.1007/s00122-023-04265-6

</cite> [<a href="https://doi.org/10.1007/s00122-023-04265-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9998580/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36892603/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=A%20comparison%20of%20methods%20for%20training%20population%20optimization%20in%20genomic%20selection&amp;author=J.%20Fern%C3%A1ndez%E2%80%90Gonz%C3%A1lez&amp;author=D.%20Akdemir&amp;author=J.%20Isidro%20y%20S%C3%A1nchez&amp;volume=136&amp;issue=3&amp;publication_year=2023&amp;pages=30&amp;pmid=36892603&amp;doi=10.1007/s00122-023-04265-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0027">
<cite>

Frenda, A. S.
, 
Ruisi, P.
, 
Saia, S.
, 
Frangipane, B.
, 
Di Miceli, G.
, 
Amato, G.
, &amp; 
Giambalvo, D.
 (2013). The critical period of weed control in faba bean and chickpea in Mediterranean areas. Weed Science, 61(3), 452–459. 10.1614/WS-D-12-00137.1
</cite> [<a href="https://doi.org/10.1614/WS-D-12-00137.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Weed%20Science&amp;title=The%20critical%20period%20of%20weed%20control%20in%20faba%20bean%20and%20chickpea%20in%20Mediterranean%20areas&amp;author=A.%20S.%20Frenda&amp;author=P.%20Ruisi&amp;author=S.%20Saia&amp;author=B.%20Frangipane&amp;author=G.%20Di%20Miceli&amp;volume=61&amp;issue=3&amp;publication_year=2013&amp;pages=452-459&amp;doi=10.1614/WS-D-12-00137.1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0028">
<cite>

Gebremedhin, A.
, 
Li, Y.
, 
Shunmugam, A. S.
, 
Sudheesh, S.
, 
Valipour‐Kahrood, H.
, 
Hayden, M. J.
, 
Rosewarne, G. M.
, &amp; 
Kaur, S.
 (2024). Genomic selection for target traits in the Australian lentil breeding program. Frontiers in Plant Science, 14, 1284781. 10.3389/fpls.2023.1284781

</cite> [<a href="https://doi.org/10.3389/fpls.2023.1284781" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10791954/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38235201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genomic%20selection%20for%20target%20traits%20in%20the%20Australian%20lentil%20breeding%20program&amp;author=A.%20Gebremedhin&amp;author=Y.%20Li&amp;author=A.%20S.%20Shunmugam&amp;author=S.%20Sudheesh&amp;author=H.%20Valipour%E2%80%90Kahrood&amp;volume=14&amp;publication_year=2024&amp;pages=1284781&amp;pmid=38235201&amp;doi=10.3389/fpls.2023.1284781&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0029">
<cite>

Gela, T. S.
, 
Bruce, M.
, 
Chang, W.
, 
Stoddard, F. L.
, 
Schulman, A. H.
, 
Vandenberg, A.
, &amp; 
Khazaei, H.
 (2022). Genomic regions associated with chocolate spot (<em>Botrytis fabae</em> Sard.) resistance in faba bean (<em>Vicia faba</em> L.). Molecular Breeding, 42(6), 35. 10.1007/s11032-022-01307-7

</cite> [<a href="https://doi.org/10.1007/s11032-022-01307-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10248645/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37312967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=Genomic%20regions%20associated%20with%20chocolate%20spot%20(Botrytis%20fabae%20Sard.)%20resistance%20in%20faba%20bean%20(Vicia%20faba%20L.)&amp;author=T.%20S.%20Gela&amp;author=M.%20Bruce&amp;author=W.%20Chang&amp;author=F.%20L.%20Stoddard&amp;author=A.%20H.%20Schulman&amp;volume=42&amp;issue=6&amp;publication_year=2022&amp;pages=35&amp;pmid=37312967&amp;doi=10.1007/s11032-022-01307-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0030">
<cite>

Gianola, D.
, &amp; 
Van Kaam, J. B.
 (2008). Reproducing kernel Hilbert spaces regression methods for genomic assisted prediction of quantitative traits. Genetics, 178(4), 2289–2303. 10.1534/genetics.107.084285

</cite> [<a href="https://doi.org/10.1534/genetics.107.084285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2323816/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18430950/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Reproducing%20kernel%20Hilbert%20spaces%20regression%20methods%20for%20genomic%20assisted%20prediction%20of%20quantitative%20traits&amp;author=D.%20Gianola&amp;author=J.%20B.%20Van%20Kaam&amp;volume=178&amp;issue=4&amp;publication_year=2008&amp;pages=2289-2303&amp;pmid=18430950&amp;doi=10.1534/genetics.107.084285&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0031">
<cite>

Grattapaglia, D.
, 
Vilela Resende, M. D.
, 
Resende, M. R.
, 
Sansaloni, C. P.
, 
Petroli, C. D.
, 
Missiaggia, A. A.
, 
Takahashi, E. K.
, 
Zamprogno, K. C.
, &amp; 
Kilian, A.
 (2011). Genomic selection for growth traits in <em>Eucalyptus</em>: Accuracy within and across breeding populations. BMC Proceedings, 5, Article O16 (2011). 10.1186/1753-6561-5-S7-A1
</cite> [<a href="https://doi.org/10.1186/1753-6561-5-S7-A1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Proceedings&amp;title=Genomic%20selection%20for%20growth%20traits%20in%20Eucalyptus:%20Accuracy%20within%20and%20across%20breeding%20populations&amp;author=D.%20Grattapaglia&amp;author=M.%20D.%20Vilela%20Resende&amp;author=M.%20R.%20Resende&amp;author=C.%20P.%20Sansaloni&amp;author=C.%20D.%20Petroli&amp;volume=5&amp;publication_year=2011&amp;doi=10.1186/1753-6561-5-S7-A1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0032">
<cite>

Gressel, J.
 (2000). Molecular biology of weed control. Transgenic Research, 9, 355–382. 10.1023/A:1008946628406

</cite> [<a href="https://doi.org/10.1023/A:1008946628406" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11131013/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Transgenic%20Research&amp;title=Molecular%20biology%20of%20weed%20control&amp;author=J.%20Gressel&amp;volume=9&amp;publication_year=2000&amp;pages=355-382&amp;pmid=11131013&amp;doi=10.1023/A:1008946628406&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0033">
<cite>

Gutiérrez, N.
, 
Pégard, M.
, 
Balko, C.
, &amp; 
Torres, A. M.
 (2023). Genome‐wide association analysis for drought tolerance and associated traits in faba bean (<em>Vicia faba</em> L.). Frontiers in Plant Science, 14, 1091875. 10.3389/fpls.2023.1091875

</cite> [<a href="https://doi.org/10.3389/fpls.2023.1091875" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9928957/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36818887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Genome%E2%80%90wide%20association%20analysis%20for%20drought%20tolerance%20and%20associated%20traits%20in%20faba%20bean%20(Vicia%20faba%20L.)&amp;author=N.%20Guti%C3%A9rrez&amp;author=M.%20P%C3%A9gard&amp;author=C.%20Balko&amp;author=A.%20M.%20Torres&amp;volume=14&amp;publication_year=2023&amp;pages=1091875&amp;pmid=36818887&amp;doi=10.3389/fpls.2023.1091875&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0034">
<cite>

Habier, D.
, 
Tetens, J.
, 
Seefried, F. R.
, 
Lichtner, P.
, &amp; 
Thaller, G.
 (2010). The impact of genetic relationship information on genomic breeding values in German Holstein cattle. Genetics Selection Evolution, 42, 1–12. 10.1186/1297-9686-42-5
</cite> [<a href="https://doi.org/10.1186/1297-9686-42-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2838754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20170500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20Selection%20Evolution&amp;title=The%20impact%20of%20genetic%20relationship%20information%20on%20genomic%20breeding%20values%20in%20German%20Holstein%20cattle&amp;author=D.%20Habier&amp;author=J.%20Tetens&amp;author=F.%20R.%20Seefried&amp;author=P.%20Lichtner&amp;author=G.%20Thaller&amp;volume=42&amp;publication_year=2010&amp;pages=1-12&amp;pmid=20170500&amp;doi=10.1186/1297-9686-42-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0035">
<cite>

Haile, T. A.
, 
Heidecker, T.
, 
Wright, D.
, 
Neupane, S.
, 
Ramsay, L.
, 
Vandenberg, A.
, &amp; 
Bett, K. E.
 (2020). Genomic selection for lentil breeding: Empirical evidence. The Plant Genome, 13(1), e20002. 10.1002/tpg2.20002

</cite> [<a href="https://doi.org/10.1002/tpg2.20002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33016638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genomic%20selection%20for%20lentil%20breeding:%20Empirical%20evidence&amp;author=T.%20A.%20Haile&amp;author=T.%20Heidecker&amp;author=D.%20Wright&amp;author=S.%20Neupane&amp;author=L.%20Ramsay&amp;volume=13&amp;issue=1&amp;publication_year=2020&amp;pages=e20002&amp;pmid=33016638&amp;doi=10.1002/tpg2.20002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0036">
<cite>

Hayes, B. J.
, 
Daetwyler, H. D.
, 
Bowman, P.
, 
Moser, G.
, 
Tier, B.
, 
Crump, R.
, 
Khatkar, M.
, 
Raadsma, H. W.
, &amp; 
Goddard, M. E.
 (2009). Accuracy of genomic selection: Comparing theory and results. Proceedings of the Association for the Advancement of Animal Breeding and Genetics, 17, 352–355.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20Association%20for%20the%20Advancement%20of%20Animal%20Breeding%20and%20Genetics&amp;title=Accuracy%20of%20genomic%20selection:%20Comparing%20theory%20and%20results&amp;author=B.%20J.%20Hayes&amp;author=H.%20D.%20Daetwyler&amp;author=P.%20Bowman&amp;author=G.%20Moser&amp;author=B.%20Tier&amp;volume=17&amp;publication_year=2009&amp;pages=352-355&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0037">
<cite>

Howard, R.
, 
Carriquiry, A. L.
, &amp; 
Beavis, W. D.
 (2014). Parametric and nonparametric statistical methods for genomic selection of traits with additive and epistatic genetic architectures. G3: Genes, Genomes, Genetics, 4(6), 1027–1046. 10.1534/g3.114.010298

</cite> [<a href="https://doi.org/10.1534/g3.114.010298" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4065247/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24727289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3:%20Genes,%20Genomes,%20Genetics&amp;title=Parametric%20and%20nonparametric%20statistical%20methods%20for%20genomic%20selection%20of%20traits%20with%20additive%20and%20epistatic%20genetic%20architectures&amp;author=R.%20Howard&amp;author=A.%20L.%20Carriquiry&amp;author=W.%20D.%20Beavis&amp;volume=4&amp;issue=6&amp;publication_year=2014&amp;pages=1027-1046&amp;pmid=24727289&amp;doi=10.1534/g3.114.010298&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0038">
<cite>

Jannink, J. L.
, 
Lorenz, A. J.
, &amp; 
Iwata, H.
 (2010). Genomic selection in plant breeding: From theory to practice. Briefings in Functional Genomics, 9(2), 166–177. 10.1093/bfgp/elq001

</cite> [<a href="https://doi.org/10.1093/bfgp/elq001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20156985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Briefings%20in%20Functional%20Genomics&amp;title=Genomic%20selection%20in%20plant%20breeding:%20From%20theory%20to%20practice&amp;author=J.%20L.%20Jannink&amp;author=A.%20J.%20Lorenz&amp;author=H.%20Iwata&amp;volume=9&amp;issue=2&amp;publication_year=2010&amp;pages=166-177&amp;pmid=20156985&amp;doi=10.1093/bfgp/elq001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0039">
<cite>

Jayakodi, M.
, 
Golicz, A. A.
, 
Kreplak, J.
, 
Fechete, L. I.
, 
Angra, D.
, 
Bednář, P.
, 
Bornhofen, E.
, 
Zhang, H.
, 
Boussageon, R.
, 
Kaur, S.
, 
Cheung, K.
, 
Čížková, J.
, 
Gundlach, H.
, 
Hallab, A.
, 
Imbert, B.
, 
Keeble‐Gagnère, G.
, 
Koblížková, A.
, 
Kobrlová, L.
, 
Krejčí, P.
, … 
Andersen, S. U.
 (2023). The giant diploid faba genome unlocks variation in a global protein crop. Nature, 615(7953), 652–659. 10.1038/s41586-023-05791-5

</cite> [<a href="https://doi.org/10.1038/s41586-023-05791-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10033403/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36890232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20giant%20diploid%20faba%20genome%20unlocks%20variation%20in%20a%20global%20protein%20crop&amp;author=M.%20Jayakodi&amp;author=A.%20A.%20Golicz&amp;author=J.%20Kreplak&amp;author=L.%20I.%20Fechete&amp;author=D.%20Angra&amp;volume=615&amp;issue=7953&amp;publication_year=2023&amp;pages=652-659&amp;pmid=36890232&amp;doi=10.1038/s41586-023-05791-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0040">
<cite>

Jighly, A.
 (2024). Boosting genome‐wide association power and genomic prediction accuracy for date palm fruit traits with advanced statistics. Plant Science, 344, 112110. 10.1016/j.plantsci.2024.112110

</cite> [<a href="https://doi.org/10.1016/j.plantsci.2024.112110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38704095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Science&amp;title=Boosting%20genome%E2%80%90wide%20association%20power%20and%20genomic%20prediction%20accuracy%20for%20date%20palm%20fruit%20traits%20with%20advanced%20statistics&amp;author=A.%20Jighly&amp;volume=344&amp;publication_year=2024&amp;pages=112110&amp;pmid=38704095&amp;doi=10.1016/j.plantsci.2024.112110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0041">
<cite>

Kaler, A. S.
, 
Purcell, L. C.
, 
Beissinger, T.
, &amp; 
Gillman, J. D.
 (2022). Genomic prediction models for traits differing in heritability for soybean, rice, and maize. BMC Plant Biology, 22(1), 87. 10.1186/s12870-022-03479-y

</cite> [<a href="https://doi.org/10.1186/s12870-022-03479-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8881851/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35219296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Genomic%20prediction%20models%20for%20traits%20differing%20in%20heritability%20for%20soybean,%20rice,%20and%20maize&amp;author=A.%20S.%20Kaler&amp;author=L.%20C.%20Purcell&amp;author=T.%20Beissinger&amp;author=J.%20D.%20Gillman&amp;volume=22&amp;issue=1&amp;publication_year=2022&amp;pages=87&amp;pmid=35219296&amp;doi=10.1186/s12870-022-03479-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0042">
<cite>

Kaur, S.
, 
Kimber, R. B.
, 
Cogan, N. O.
, 
Materne, M.
, 
Forster, J. W.
, &amp; 
Paull, J. G.
 (2014). SNP discovery and high‐density genetic mapping in faba bean (<em>Vicia faba</em> L.) permits identification of QTLs for ascochyta blight resistance. Plant Science, 217, 47–55. 10.1016/j.plantsci.2013.11.014

</cite> [<a href="https://doi.org/10.1016/j.plantsci.2013.11.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24467895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Science&amp;title=SNP%20discovery%20and%20high%E2%80%90density%20genetic%20mapping%20in%20faba%20bean%20(Vicia%20faba%20L.)%20permits%20identification%20of%20QTLs%20for%20ascochyta%20blight%20resistance&amp;author=S.%20Kaur&amp;author=R.%20B.%20Kimber&amp;author=N.%20O.%20Cogan&amp;author=M.%20Materne&amp;author=J.%20W.%20Forster&amp;volume=217&amp;publication_year=2014&amp;pages=47-55&amp;pmid=24467895&amp;doi=10.1016/j.plantsci.2013.11.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0043">
<cite>

Khazaei, H.
, 
Purves, R. W.
, 
Song, M.
, 
Stonehouse, R.
, 
Bett, K. E.
, 
Stoddard, F. L.
, &amp; 
Vandenberg, A.
 (2017). Development and validation of a robust, breeder‐friendly molecular marker for the vc‐locus in faba bean. Molecular Breeding, 37, 1–6. 10.1007/s11032-017-0742-5
28127252
</cite> [<a href="https://doi.org/10.1007/s11032-017-0742-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=Development%20and%20validation%20of%20a%20robust,%20breeder%E2%80%90friendly%20molecular%20marker%20for%20the%20vc%E2%80%90locus%20in%20faba%20bean&amp;author=H.%20Khazaei&amp;author=R.%20W.%20Purves&amp;author=M.%20Song&amp;author=R.%20Stonehouse&amp;author=K.%20E.%20Bett&amp;volume=37&amp;publication_year=2017&amp;pages=1-6&amp;doi=10.1007/s11032-017-0742-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0044">
<cite>

Lavania, D.
, 
Siddiqui, M. H.
, 
Al‐Whaibi, M. H.
, 
Singh, A. K.
, 
Kumar, R.
, &amp; 
Grover, A.
 (2015). Genetic approaches for breeding heat stress tolerance in faba bean (<em>Vicia faba</em> L.). Acta Physiologiae Plantarum, 37, 1–9. 10.1007/s11738-014-1737-z
</cite> [<a href="https://doi.org/10.1007/s11738-014-1737-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Physiologiae%20Plantarum&amp;title=Genetic%20approaches%20for%20breeding%20heat%20stress%20tolerance%20in%20faba%20bean%20(Vicia%20faba%20L.)&amp;author=D.%20Lavania&amp;author=M.%20H.%20Siddiqui&amp;author=M.%20H.%20Al%E2%80%90Whaibi&amp;author=A.%20K.%20Singh&amp;author=R.%20Kumar&amp;volume=37&amp;publication_year=2015&amp;pages=1-9&amp;doi=10.1007/s11738-014-1737-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0045">
<cite>

Li, Y.
, 
Ruperao, P.
, 
Batley, J.
, 
Edwards, D.
, 
Khan, T.
, 
Colmer, T. D.
, 
Pang, J.
, 
Siddique, K. H.
, &amp; 
Sutton, T.
 (2018). Investigating drought tolerance in chickpea using genome‐wide association mapping and genomic selection based on whole‐genome resequencing data. Frontiers in Plant Science, 9, 190. 10.3389/fpls.2018.00190

</cite> [<a href="https://doi.org/10.3389/fpls.2018.00190" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5825913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29515606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Investigating%20drought%20tolerance%20in%20chickpea%20using%20genome%E2%80%90wide%20association%20mapping%20and%20genomic%20selection%20based%20on%20whole%E2%80%90genome%20resequencing%20data&amp;author=Y.%20Li&amp;author=P.%20Ruperao&amp;author=J.%20Batley&amp;author=D.%20Edwards&amp;author=T.%20Khan&amp;volume=9&amp;publication_year=2018&amp;pages=190&amp;pmid=29515606&amp;doi=10.3389/fpls.2018.00190&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0068">
<cite>

Lippolis, A.
, 
Gezan, S. A.
, 
Zuidgeest, J.
, 
Cafaro, V.
, 
van Dinter, B. J.
, 
Elzes, G.
, 
Paulo, M. J.
, &amp; 
Trindade, L. M.
 (2025). Targeted genotyping (90K‐SPET) facilitates genome‐wide association studies and the prediction of yield‐related traits in faba bean (<em>Vicia faba</em> L.). BMC Plant Biology, 25(1), 558.
</cite> [<a href="https://doi.org/10.1186/s12870-025-06546-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12042580/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40301715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Targeted%20genotyping%20(90K%E2%80%90SPET)%20facilitates%20genome%E2%80%90wide%20association%20studies%20and%20the%20prediction%20of%20yield%E2%80%90related%20traits%20in%20faba%20bean%20(Vicia%20faba%20L.)&amp;author=A.%20Lippolis&amp;author=S.%20A.%20Gezan&amp;author=J.%20Zuidgeest&amp;author=V.%20Cafaro&amp;author=B.%20J.%20van%20Dinter&amp;volume=25&amp;issue=1&amp;publication_year=2025&amp;pages=558&amp;pmid=40301715&amp;doi=10.1186/s12870-025-06546-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0046">
<cite>

Lorenz, A. J.
, &amp; 
Smith, K. P.
 (2015). Adding genetically distant individuals to training populations reduces genomic prediction accuracy in barley. Crop Science, 55(6), 2657–2667. 10.2135/cropsci2014.12.0827
</cite> [<a href="https://doi.org/10.2135/cropsci2014.12.0827" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Adding%20genetically%20distant%20individuals%20to%20training%20populations%20reduces%20genomic%20prediction%20accuracy%20in%20barley&amp;author=A.%20J.%20Lorenz&amp;author=K.%20P.%20Smith&amp;volume=55&amp;issue=6&amp;publication_year=2015&amp;pages=2657-2667&amp;doi=10.2135/cropsci2014.12.0827&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0047">
<cite>

Lorenzana, R. E.
, &amp; 
Bernardo, R.
 (2009). Accuracy of genotypic value predictions for marker‐based selection in biparental plant populations. Theoretical and Applied Genetics, 120, 151–161. 10.1007/s00122-009-1166-3

</cite> [<a href="https://doi.org/10.1007/s00122-009-1166-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19841887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Accuracy%20of%20genotypic%20value%20predictions%20for%20marker%E2%80%90based%20selection%20in%20biparental%20plant%20populations&amp;author=R.%20E.%20Lorenzana&amp;author=R.%20Bernardo&amp;volume=120&amp;publication_year=2009&amp;pages=151-161&amp;pmid=19841887&amp;doi=10.1007/s00122-009-1166-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0048">
<cite>

Luan, T.
, 
Woolliams, J. A.
, 
Lien, S.
, 
Kent, M.
, 
Svendsen, M.
, &amp; 
Meuwissen, T. H.
 (2009). The accuracy of genomic selection in Norwegian red cattle assessed by cross‐validation. Genetics, 183(3), 1119–1126. 10.1534/genetics.109.107391

</cite> [<a href="https://doi.org/10.1534/genetics.109.107391" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2778964/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19704013/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=The%20accuracy%20of%20genomic%20selection%20in%20Norwegian%20red%20cattle%20assessed%20by%20cross%E2%80%90validation&amp;author=T.%20Luan&amp;author=J.%20A.%20Woolliams&amp;author=S.%20Lien&amp;author=M.%20Kent&amp;author=M.%20Svendsen&amp;volume=183&amp;issue=3&amp;publication_year=2009&amp;pages=1119-1126&amp;pmid=19704013&amp;doi=10.1534/genetics.109.107391&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0049">
<cite>

Maalouf, F.
, 
Abou‐Khater, L.
, 
Babiker, Z.
, 
Jighly, A.
, 
Alsamman, A. M.
, 
Hu, J.
, 
Ma, Y.
, 
Rispail, N.
, 
Balech, R.
, 
Hamweih, A.
, 
Baum, M.
, &amp; 
Kumar, S.
 (2022). Genetic dissection of heat stress tolerance in faba bean (<em>Vicia faba</em> L.) using GWAS. Plants, 11(9), 1108. 10.3390/plants11091108

</cite> [<a href="https://doi.org/10.3390/plants11091108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9103424/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35567109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Genetic%20dissection%20of%20heat%20stress%20tolerance%20in%20faba%20bean%20(Vicia%20faba%20L.)%20using%20GWAS&amp;author=F.%20Maalouf&amp;author=L.%20Abou%E2%80%90Khater&amp;author=Z.%20Babiker&amp;author=A.%20Jighly&amp;author=A.%20M.%20Alsamman&amp;volume=11&amp;issue=9&amp;publication_year=2022&amp;pages=1108&amp;pmid=35567109&amp;doi=10.3390/plants11091108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0050">
<cite>

Maalouf, F.
, 
Hu, J.
, 
O'Sullivan, D. M.
, 
Zong, X.
, 
Hamwieh, A.
, 
Kumar, S.
, &amp; 
Baum, M.
 (2019). Breeding and genomics status in faba bean (<em>Vicia faba</em>). Plant Breeding, 138(4), 465–473. 10.1111/pbr.12644
</cite> [<a href="https://doi.org/10.1111/pbr.12644" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Breeding&amp;title=Breeding%20and%20genomics%20status%20in%20faba%20bean%20(Vicia%20faba)&amp;author=F.%20Maalouf&amp;author=J.%20Hu&amp;author=D.%20M.%20O'Sullivan&amp;author=X.%20Zong&amp;author=A.%20Hamwieh&amp;volume=138&amp;issue=4&amp;publication_year=2019&amp;pages=465-473&amp;doi=10.1111/pbr.12644&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0051">
<cite>

Mohan, M.
, 
Nair, S.
, 
Bhagwat, A.
, 
Krishna, T. G.
, 
Yano, M.
, 
Bhatia, C. R.
, &amp; 
Sasaki, T.
 (1997). Genome mapping, molecular markers and marker‐assisted selection in crop plants. Molecular Breeding, 3, 87–103. 10.1023/A:1009651919792
</cite> [<a href="https://doi.org/10.1023/A:1009651919792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Breeding&amp;title=Genome%20mapping,%20molecular%20markers%20and%20marker%E2%80%90assisted%20selection%20in%20crop%20plants&amp;author=M.%20Mohan&amp;author=S.%20Nair&amp;author=A.%20Bhagwat&amp;author=T.%20G.%20Krishna&amp;author=M.%20Yano&amp;volume=3&amp;publication_year=1997&amp;pages=87-103&amp;doi=10.1023/A:1009651919792&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0052">
<cite>

Morota, G.
, &amp; 
Gianola, D.
 (2014). Kernel‐based whole‐genome prediction of complex traits: A review. Frontiers in Genetics, 5, 363. 10.3389/fgene.2014.00363

</cite> [<a href="https://doi.org/10.3389/fgene.2014.00363" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4199321/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25360145/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Kernel%E2%80%90based%20whole%E2%80%90genome%20prediction%20of%20complex%20traits:%20A%20review&amp;author=G.%20Morota&amp;author=D.%20Gianola&amp;volume=5&amp;publication_year=2014&amp;pages=363&amp;pmid=25360145&amp;doi=10.3389/fgene.2014.00363&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0053">
<cite>

Ocaña‐Moral, S.
, 
Gutiérrez, N.
, 
Torres, A. M.
, &amp; 
Madrid, E.
 (2017). Saturation mapping of regions determining resistance to <em>Ascochyta</em> blight and broomrape in faba bean using transcriptome‐based SNP genotyping. Theoretical and Applied Genetics, 130, 2271–2282. 10.1007/s00122-017-2958-5

</cite> [<a href="https://doi.org/10.1007/s00122-017-2958-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28791437/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Saturation%20mapping%20of%20regions%20determining%20resistance%20to%20Ascochyta%20blight%20and%20broomrape%20in%20faba%20bean%20using%20transcriptome%E2%80%90based%20SNP%20genotyping&amp;author=S.%20Oca%C3%B1a%E2%80%90Moral&amp;author=N.%20Guti%C3%A9rrez&amp;author=A.%20M.%20Torres&amp;author=E.%20Madrid&amp;volume=130&amp;publication_year=2017&amp;pages=2271-2282&amp;pmid=28791437&amp;doi=10.1007/s00122-017-2958-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0054">
<cite>

Pérez, P.
, &amp; 
de Los Campos, G.
 (2014). Genome‐wide regression and prediction with the BGLR statistical package. Genetics, 198(2), 483–495. 10.1534/genetics.114.164442

</cite> [<a href="https://doi.org/10.1534/genetics.114.164442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4196607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25009151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genome%E2%80%90wide%20regression%20and%20prediction%20with%20the%20BGLR%20statistical%20package&amp;author=P.%20P%C3%A9rez&amp;author=G.%20de%20Los%20Campos&amp;volume=198&amp;issue=2&amp;publication_year=2014&amp;pages=483-495&amp;pmid=25009151&amp;doi=10.1534/genetics.114.164442&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0055">
<cite>

Poland, J.
, 
Endelman, J.
, 
Dawson, J.
, 
Rutkoski, J.
, 
Wu, S.
, 
Manes, Y.
, 
Dreisigacker, S.
, 
Crossa, J.
, 
Sánchez‐Villeda, H.
, 
Sorrells, M.
, &amp; 
Jannink, J. L.
 (2012). Genomic selection in wheat breeding using genotyping‐by‐sequencing. The Plant Genome, 5(3). 10.3835/plantgenome2012.06.0006
</cite> [<a href="https://doi.org/10.3835/plantgenome2012.06.0006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genomic%20selection%20in%20wheat%20breeding%20using%20genotyping%E2%80%90by%E2%80%90sequencing&amp;author=J.%20Poland&amp;author=J.%20Endelman&amp;author=J.%20Dawson&amp;author=J.%20Rutkoski&amp;author=S.%20Wu&amp;volume=5&amp;issue=3&amp;publication_year=2012&amp;doi=10.3835/plantgenome2012.06.0006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0056">
<cite>

Roorkiwal, M.
, 
Jarquin, D.
, 
Singh, M. K.
, 
Gaur, P. M.
, 
Bharadwaj, C.
, 
Rathore, A.
, 
Howard, R.
, 
Srinivasan, S.
, 
Jain, A.
, 
Garg, V.
, 
Kale, S.
, 
Chitikineni, A.
, 
Tripathi, S.
, 
Jones, E.
, 
Robbins, R. K.
, 
Crossa, J.
, &amp; 
Varshney, R. K.
 (2018). Genomic‐enabled prediction models using multi‐environment trials to estimate the effect of genotype × environment interaction on prediction accuracy in chickpea. Scientific Reports, 8(1), 11701. 10.1038/s41598-018-30027-2

</cite> [<a href="https://doi.org/10.1038/s41598-018-30027-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6076323/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30076340/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Genomic%E2%80%90enabled%20prediction%20models%20using%20multi%E2%80%90environment%20trials%20to%20estimate%20the%20effect%20of%20genotype%20%C3%97%20environment%20interaction%20on%20prediction%20accuracy%20in%20chickpea&amp;author=M.%20Roorkiwal&amp;author=D.%20Jarquin&amp;author=M.%20K.%20Singh&amp;author=P.%20M.%20Gaur&amp;author=C.%20Bharadwaj&amp;volume=8&amp;issue=1&amp;publication_year=2018&amp;pages=11701&amp;pmid=30076340&amp;doi=10.1038/s41598-018-30027-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0057">
<cite>

Skovbjerg, C. K.
, 
Angra, D.
, 
Robertson‐Shersby‐Harvie, T.
, 
Kreplak, J.
, 
Keeble‐Gagnère, G.
, 
Kaur, S.
, 
Ecke, W.
, 
Windhorst, A.
, 
Nielsen, L. K.
, 
Schiemann, A.
, 
Knudsen, J.
, 
Gutierrez, N.
, 
Tagkouli, V.
, 
Fechete, L. I.
, 
Janss, L.
, 
Stougaard, J.
, 
Warsame, A.
, 
Alves, S.
, 
Khazaei, H.
, … 
Andersen, S. U.
 (2023). Genetic analysis of global faba bean diversity, agronomic traits and selection signatures. Theoretical and Applied Genetics, 136(5), 114. 10.1007/s00122-023-04360-8

</cite> [<a href="https://doi.org/10.1007/s00122-023-04360-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10115707/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37074596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Genetic%20analysis%20of%20global%20faba%20bean%20diversity,%20agronomic%20traits%20and%20selection%20signatures&amp;author=C.%20K.%20Skovbjerg&amp;author=D.%20Angra&amp;author=T.%20Robertson%E2%80%90Shersby%E2%80%90Harvie&amp;author=J.%20Kreplak&amp;author=G.%20Keeble%E2%80%90Gagn%C3%A8re&amp;volume=136&amp;issue=5&amp;publication_year=2023&amp;pages=114&amp;pmid=37074596&amp;doi=10.1007/s00122-023-04360-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0058">
<cite>

Spindel, J.
, 
Begum, H.
, 
Akdemir, D.
, 
Virk, P.
, 
Collard, B.
, 
Redoña, E.
, 
Atlin, G.
, 
Jannink, J.‐L.
, &amp; 
Mccouch, S. R.
 (2015). Genomic selection and association mapping in rice (<em>Oryza sativa</em>): Effect of trait genetic architecture, training population composition, marker number and statistical model on accuracy of rice genomic selection in elite, tropical rice breeding lines. PLOS Genetics, 11(2), e1004982. 10.1371/journal.pgen.1004982

</cite> [<a href="https://doi.org/10.1371/journal.pgen.1004982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4334555/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25689273/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genetics&amp;title=Genomic%20selection%20and%20association%20mapping%20in%20rice%20(Oryza%20sativa):%20Effect%20of%20trait%20genetic%20architecture,%20training%20population%20composition,%20marker%20number%20and%20statistical%20model%20on%20accuracy%20of%20rice%20genomic%20selection%20in%20elite,%20tropical%20rice%20breeding%20lines&amp;author=J.%20Spindel&amp;author=H.%20Begum&amp;author=D.%20Akdemir&amp;author=P.%20Virk&amp;author=B.%20Collard&amp;volume=11&amp;issue=2&amp;publication_year=2015&amp;pages=e1004982&amp;pmid=25689273&amp;doi=10.1371/journal.pgen.1004982&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0059">
<cite>

Su, G.
, 
Brøndum, R. F.
, 
Ma, P.
, 
Guldbrandtsen, B.
, 
Aamand, G. P.
, &amp; 
Lund, M. S.
 (2012). Comparison of genomic predictions using medium‐density (∼54,000) and high‐density (∼777,000) single nucleotide polymorphism marker panels in Nordic Holstein and Red Dairy Cattle populations. Journal of Dairy Science, 95(8), 4657–4665. 10.3168/jds.2012-5379

</cite> [<a href="https://doi.org/10.3168/jds.2012-5379" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22818480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Dairy%20Science&amp;title=Comparison%20of%20genomic%20predictions%20using%20medium%E2%80%90density%20(%E2%88%BC54,000)%20and%20high%E2%80%90density%20(%E2%88%BC777,000)%20single%20nucleotide%20polymorphism%20marker%20panels%20in%20Nordic%20Holstein%20and%20Red%20Dairy%20Cattle%20populations&amp;author=G.%20Su&amp;author=R.%20F.%20Br%C3%B8ndum&amp;author=P.%20Ma&amp;author=B.%20Guldbrandtsen&amp;author=G.%20P.%20Aamand&amp;volume=95&amp;issue=8&amp;publication_year=2012&amp;pages=4657-4665&amp;pmid=22818480&amp;doi=10.3168/jds.2012-5379&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0060">
<cite>

Sverrisdóttir, E.
, 
Sundmark, E. H. R.
, 
Johnsen, H. Ø.
, 
Kirk, H. G.
, 
Asp, T.
, 
Janss, L.
, 
Bryan, G.
, &amp; 
Nielsen, K. L.
 (2018). The value of expanding the training population to improve genomic selection models in tetraploid potato. Frontiers in Plant Science, 9, 1118. 10.3389/fpls.2018.01118

</cite> [<a href="https://doi.org/10.3389/fpls.2018.01118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6090097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30131817/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=The%20value%20of%20expanding%20the%20training%20population%20to%20improve%20genomic%20selection%20models%20in%20tetraploid%20potato&amp;author=E.%20Sverrisd%C3%B3ttir&amp;author=E.%20H.%20R.%20Sundmark&amp;author=H.%20%C3%98.%20Johnsen&amp;author=H.%20G.%20Kirk&amp;author=T.%20Asp&amp;volume=9&amp;publication_year=2018&amp;pages=1118&amp;pmid=30131817&amp;doi=10.3389/fpls.2018.01118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0061">
<cite>

Tan, B.
, 
Grattapaglia, D.
, 
Martins, G. S.
, 
Ferreira, K. Z.
, 
Sundberg, B.
, &amp; 
Ingvarsson, P. K.
 (2017). Evaluating the accuracy of genomic prediction of growth and wood traits in two <em>Eucalyptus</em> species and their F<sub>1</sub> hybrids. BMC Plant Biology, 17, 1–15. 10.1186/s12870-017-1059-6

</cite> [<a href="https://doi.org/10.1186/s12870-017-1059-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5492818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28662679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Evaluating%20the%20accuracy%20of%20genomic%20prediction%20of%20growth%20and%20wood%20traits%20in%20two%20Eucalyptus%20species%20and%20their%20F1%20hybrids&amp;author=B.%20Tan&amp;author=D.%20Grattapaglia&amp;author=G.%20S.%20Martins&amp;author=K.%20Z.%20Ferreira&amp;author=B.%20Sundberg&amp;volume=17&amp;publication_year=2017&amp;pages=1-15&amp;pmid=28662679&amp;doi=10.1186/s12870-017-1059-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0062">
<cite>

Thavamanikumar, S.
, 
Dolferus, R.
, &amp; 
Thumma, B. R.
 (2015). Comparison of genomic selection models to predict flowering time and spike grain number in two hexaploid wheat doubled haploid populations. G3: Genes, Genomes, Genetics, 5(10), 1991–1998. 10.1534/g3.115.019745

</cite> [<a href="https://doi.org/10.1534/g3.115.019745" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4592981/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26206349/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3:%20Genes,%20Genomes,%20Genetics&amp;title=Comparison%20of%20genomic%20selection%20models%20to%20predict%20flowering%20time%20and%20spike%20grain%20number%20in%20two%20hexaploid%20wheat%20doubled%20haploid%20populations&amp;author=S.%20Thavamanikumar&amp;author=R.%20Dolferus&amp;author=B.%20R.%20Thumma&amp;volume=5&amp;issue=10&amp;publication_year=2015&amp;pages=1991-1998&amp;pmid=26206349&amp;doi=10.1534/g3.115.019745&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0064">
<cite>

Tolessa, T. T.
, 
Keneni, G.
, &amp; 
Mohammad, H.
 (2015). Genetic progresses from over three decades of faba bean (<em>Vicia faba</em> L.) breeding in Ethiopia. Australian Journal of Crop Science, 9(1), 41–48. <a href="https://doi/10.3316/informit.915168199486697" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi/10.3316/informit.915168199486697</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Australian%20Journal%20of%20Crop%20Science&amp;title=Genetic%20progresses%20from%20over%20three%20decades%20of%20faba%20bean%20(Vicia%20faba%20L.)%20breeding%20in%20Ethiopia&amp;author=T.%20T.%20Tolessa&amp;author=G.%20Keneni&amp;author=H.%20Mohammad&amp;volume=9&amp;issue=1&amp;publication_year=2015&amp;pages=41-48&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0065">
<cite>

Xu, Z.
, 
Hua, J.
, 
Wang, F.
, 
Cheng, Z.
, 
Meng, Q.
, 
Chen, Y.
, 
Han, X.
, 
Tie, S.
, 
Liu, C.
, 
Li, X.
, 
Wang, Z.
, &amp; 
Weng, J.
 (2020). Marker‐assisted selection of qMrdd8 to improve maize resistance to rough dwarf disease. Breeding Science, 70(2), 183–192. 10.1270/jsbbs.19110

</cite> [<a href="https://doi.org/10.1270/jsbbs.19110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7272251/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32523400/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Breeding%20Science&amp;title=Marker%E2%80%90assisted%20selection%20of%20qMrdd8%20to%20improve%20maize%20resistance%20to%20rough%20dwarf%20disease&amp;author=Z.%20Xu&amp;author=J.%20Hua&amp;author=F.%20Wang&amp;author=Z.%20Cheng&amp;author=Q.%20Meng&amp;volume=70&amp;issue=2&amp;publication_year=2020&amp;pages=183-192&amp;pmid=32523400&amp;doi=10.1270/jsbbs.19110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0070">
<cite>

Wardhana, D. H. A.
, &amp; 
Prawira, M. R.
 (2024). The analysis of Indonesia's climate change policies in response to the 2021 Intergovernmental Panel on Climate Change (IPCC) Assessment Report/AR6 Group 1 (2021–2023). Proirofonic, 1(1), 42–53.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Proirofonic&amp;title=The%20analysis%20of%20Indonesia's%20climate%20change%20policies%20in%20response%20to%20the%202021%20Intergovernmental%20Panel%20on%20Climate%20Change%20(IPCC)%20Assessment%20Report/AR6%20Group%201%20(2021%E2%80%932023)&amp;author=D.%20H.%20A.%20Wardhana&amp;author=M.%20R.%20Prawira&amp;volume=1&amp;issue=1&amp;publication_year=2024&amp;pages=42-53&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0066">
<cite>

Zhang, Z.
, 
Ding, X.
, 
Liu, J.
, 
Zhang, Q.
, &amp; 
de Koning, D. J.
 (2011). Accuracy of genomic prediction using low‐density marker panels. Journal of Dairy Science, 94(7), 3642–3650. 10.3168/jds.2010-3917

</cite> [<a href="https://doi.org/10.3168/jds.2010-3917" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21700054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Dairy%20Science&amp;title=Accuracy%20of%20genomic%20prediction%20using%20low%E2%80%90density%20marker%20panels&amp;author=Z.%20Zhang&amp;author=X.%20Ding&amp;author=J.%20Liu&amp;author=Q.%20Zhang&amp;author=D.%20J.%20de%20Koning&amp;volume=94&amp;issue=7&amp;publication_year=2011&amp;pages=3642-3650&amp;pmid=21700054&amp;doi=10.3168/jds.2010-3917&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270076-bib-0067">
<cite>

Zhong, S.
, 
Dekkers, J. C.
, 
Fernando, R. L.
, &amp; 
Jannink, J. L.
 (2009). Factors affecting accuracy from genomic selection in populations derived from multiple inbred lines: A barley case study. Genetics, 182(1), 355–364. 10.1534/genetics.108.098277

</cite> [<a href="https://doi.org/10.1534/genetics.108.098277" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2674832/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19299342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Factors%20affecting%20accuracy%20from%20genomic%20selection%20in%20populations%20derived%20from%20multiple%20inbred%20lines:%20A%20barley%20case%20study&amp;author=S.%20Zhong&amp;author=J.%20C.%20Dekkers&amp;author=R.%20L.%20Fernando&amp;author=J.%20L.%20Jannink&amp;volume=182&amp;issue=1&amp;publication_year=2009&amp;pages=355-364&amp;pmid=19299342&amp;doi=10.1534/genetics.108.098277&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Table S1 presents the list of evaluated genotypes along with their details.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12271658/bin/TPG2-18-e70076-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70076-s001.xlsx</a><sup> (9MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>Table S2 presents the list of 192 KASP markers along with the corresponding sequences developed based on the results obtained by Abou Khater et al., <a href="#tpg270076-bib-0001" class="usa-link" aria-describedby="tpg270076-bib-0001">2022</a>, Maalouf et al., 2022 and Guiterrez et al., <a href="#tpg270076-bib-0033" class="usa-link" aria-describedby="tpg270076-bib-0033">2023</a> for heat, herbicide and drought tolerance.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12271658/bin/TPG2-18-e70076-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70076-s002.xlsx</a><sup> (24.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p"><p>Table S3 presents the list of 75 KASP markers for heat tolerance traits prioritized to estimate the PA.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12271658/bin/TPG2-18-e70076-s004.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70076-s004.xlsx</a><sup> (10.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p"><p>Figure S1 presents a diagram visualizing the stepwise workflow from phenotyping to genotyping and genomic selection.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12271658/bin/TPG2-18-e70076-s003.pdf" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70076-s003.pdf</a><sup> (71.1KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All phenotypic and genotypic data including sequence data used in this study will be available at the open‐source research data depository MELSpace (<a href="https://repo.mel.cgiar.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://repo.mel.cgiar.org</a>).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70076"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70076.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (673.6 KB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12271658/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12271658/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12271658%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12271658/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12271658/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12271658/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40676773/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12271658/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40676773/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12271658/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12271658/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="1dFkGeUfUbBGlszR2QF01KxNoP0n4tJPie4MHnNQKEmHwnD2Oe3itnvsQ1eL3Ac4">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
