
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            In Silico  Investigation of BTK as a Potent Immunomodulatory Target: Exploring Novel Therapeutic Approaches for Multiple Sclerosis - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="FF2082FE89B753831A82FE00037E8D1C.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsomega">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332581/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Omega">
<meta name="citation_title" content="In Silico  Investigation of BTK as a Potent Immunomodulatory Target: Exploring Novel Therapeutic Approaches for Multiple Sclerosis">
<meta name="citation_author" content="Afsheen Saba">
<meta name="citation_author_institution" content="Key Laboratory of Medical Molecule Science and Pharmaceutical Engineering, Ministry of Industry and Information Technology, School of Chemistry and Chemical Engineering, Beijing Institute of Technology, Beijing 100081, China">
<meta name="citation_author" content="Rui-Chen Liu">
<meta name="citation_author_institution" content="Key Laboratory of Medical Molecule Science and Pharmaceutical Engineering, Ministry of Industry and Information Technology, School of Chemistry and Chemical Engineering, Beijing Institute of Technology, Beijing 100081, China">
<meta name="citation_author" content="Si-Xi Wang">
<meta name="citation_author_institution" content="Key Laboratory of Medical Molecule Science and Pharmaceutical Engineering, Ministry of Industry and Information Technology, School of Chemistry and Chemical Engineering, Beijing Institute of Technology, Beijing 100081, China">
<meta name="citation_author" content="Kiran Shehzadi">
<meta name="citation_author_institution" content="Key Laboratory of Medical Molecule Science and Pharmaceutical Engineering, Ministry of Industry and Information Technology, School of Chemistry and Chemical Engineering, Beijing Institute of Technology, Beijing 100081, China">
<meta name="citation_author" content="Wei-Hai Xu">
<meta name="citation_author_institution" content="Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing 100730, China">
<meta name="citation_author" content="Yan Xu">
<meta name="citation_author_institution" content="Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College, Beijing 100730, China">
<meta name="citation_author" content="Jian-Hua Liang">
<meta name="citation_author_institution" content="Key Laboratory of Medical Molecule Science and Pharmaceutical Engineering, Ministry of Industry and Information Technology, School of Chemistry and Chemical Engineering, Beijing Institute of Technology, Beijing 100081, China">
<meta name="citation_author" content="Ming-Jia Yu">
<meta name="citation_author_institution" content="Key Laboratory of Medical Molecule Science and Pharmaceutical Engineering, Ministry of Industry and Information Technology, School of Chemistry and Chemical Engineering, Beijing Institute of Technology, Beijing 100081, China">
<meta name="citation_publication_date" content="2025 Jul 22">
<meta name="citation_volume" content="10">
<meta name="citation_issue" content="30">
<meta name="citation_firstpage" content="33481">
<meta name="citation_doi" content="10.1021/acsomega.5c03682">
<meta name="citation_pmid" content="40787374">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332581/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332581/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332581/pdf/ao5c03682.pdf">
<meta name="description" content="Multiple sclerosis (MS) is a chronic autoimmune demyelinating disease characterized by progressive neurological impairment. Bruton’s tyrosine kinase (BTK) has emerged as a crucial therapeutic target due to its role in B-cell activation and innate ...">
<meta name="og:title" content="In Silico  Investigation of BTK as a Potent Immunomodulatory Target: Exploring Novel Therapeutic Approaches for Multiple Sclerosis">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Multiple sclerosis (MS) is a chronic autoimmune demyelinating disease characterized by progressive neurological impairment. Bruton’s tyrosine kinase (BTK) has emerged as a crucial therapeutic target due to its role in B-cell activation and innate ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332581/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12332581">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsomega.5c03682"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/ao5c03682.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332581%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12332581/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12332581/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332581/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsomega.png" alt="ACS Omega logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Omega" title="Link to ACS Omega" shape="default" href="https://pubs.acs.org/journal/acsodf" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Omega</button></div>. 2025 Jul 22;10(30):33481–33496. doi: <a href="https://doi.org/10.1021/acsomega.5c03682" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsomega.5c03682</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Omega%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Omega%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>
<em>
<strong>In Silico</strong>
</em> Investigation
of BTK as a Potent Immunomodulatory Target: Exploring Novel Therapeutic
Approaches for Multiple Sclerosis</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Saba%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Afsheen Saba</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Afsheen Saba</span></h3>
<div class="p">
<sup>†</sup>
Key
Laboratory of Medical Molecule Science and Pharmaceutical Engineering,
Ministry of Industry and Information Technology, School of Chemistry
and Chemical Engineering, Beijing Institute
of Technology, Beijing
100081, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Saba%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Afsheen Saba</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20RC%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Rui-Chen Liu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Rui-Chen Liu</span></h3>
<div class="p">
<sup>†</sup>
Key
Laboratory of Medical Molecule Science and Pharmaceutical Engineering,
Ministry of Industry and Information Technology, School of Chemistry
and Chemical Engineering, Beijing Institute
of Technology, Beijing
100081, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20RC%22%5BAuthor%5D" class="usa-link"><span class="name western">Rui-Chen Liu</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20SX%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Si-Xi Wang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Si-Xi Wang</span></h3>
<div class="p">
<sup>†</sup>
Key
Laboratory of Medical Molecule Science and Pharmaceutical Engineering,
Ministry of Industry and Information Technology, School of Chemistry
and Chemical Engineering, Beijing Institute
of Technology, Beijing
100081, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20SX%22%5BAuthor%5D" class="usa-link"><span class="name western">Si-Xi Wang</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shehzadi%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Kiran Shehzadi</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Kiran Shehzadi</span></h3>
<div class="p">
<sup>†</sup>
Key
Laboratory of Medical Molecule Science and Pharmaceutical Engineering,
Ministry of Industry and Information Technology, School of Chemistry
and Chemical Engineering, Beijing Institute
of Technology, Beijing
100081, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shehzadi%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kiran Shehzadi</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20WH%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Wei-Hai Xu</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Wei-Hai Xu</span></h3>
<div class="p">
<sup>‡</sup>
Peking
Union Medical College Hospital, Chinese
Academy of Medical Sciences and Peking Union Medical College, Beijing
100730, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20WH%22%5BAuthor%5D" class="usa-link"><span class="name western">Wei-Hai Xu</span></a>
</div>
</div>
<sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Yan Xu</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Yan Xu</span></h3>
<div class="p">
<sup>‡</sup>
Peking
Union Medical College Hospital, Chinese
Academy of Medical Sciences and Peking Union Medical College, Beijing
100730, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yan Xu</span></a>
</div>
</div>
<sup>‡,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liang%20JH%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Jian-Hua Liang</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Jian-Hua Liang</span></h3>
<div class="p">
<sup>†</sup>
Key
Laboratory of Medical Molecule Science and Pharmaceutical Engineering,
Ministry of Industry and Information Technology, School of Chemistry
and Chemical Engineering, Beijing Institute
of Technology, Beijing
100081, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liang%20JH%22%5BAuthor%5D" class="usa-link"><span class="name western">Jian-Hua Liang</span></a>
</div>
</div>
<sup>†,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20MJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Ming-Jia Yu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Ming-Jia Yu</span></h3>
<div class="p">
<sup>†</sup>
Key
Laboratory of Medical Molecule Science and Pharmaceutical Engineering,
Ministry of Industry and Information Technology, School of Chemistry
and Chemical Engineering, Beijing Institute
of Technology, Beijing
100081, China
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20MJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Ming-Jia Yu</span></a>
</div>
</div>
<sup>†,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>†</sup>
Key
Laboratory of Medical Molecule Science and Pharmaceutical Engineering,
Ministry of Industry and Information Technology, School of Chemistry
and Chemical Engineering, Beijing Institute
of Technology, Beijing
100081, China
</div>
<div id="aff2">
<sup>‡</sup>
Peking
Union Medical College Hospital, Chinese
Academy of Medical Sciences and Peking Union Medical College, Beijing
100730, China
</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
Email: <span>6120210204@bit.edu.cn</span>.</p>
</div>
<div class="fn" id="cor2">
<sup>*</sup><p class="display-inline">
Email: <span>ljhbit@bit.edu.cn</span>.</p>
</div>
<div class="fn" id="cor3">
<sup>*</sup><p class="display-inline">
Email: <span>xuyanpumch@hotmail.com</span>.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 26; Accepted 2025 Jul 17; Revised 2025 Jul 12; Collection date 2025 Aug 5.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY-NC-ND 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12332581  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40787374/" class="usa-link">40787374</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Multiple sclerosis
(MS) is a chronic autoimmune demyelinating disease
characterized by progressive neurological impairment. Bruton’s
tyrosine kinase (BTK) has emerged as a crucial therapeutic target
due to its role in B-cell activation and innate immune signaling.
While BTK inhibitors (BTKi) have shown promise for treating MS and
inflammatory disorders, their high toxicity and off-target kinase
inhibition pose challenges, particularly given that many patients
experience mild symptoms. This study presents <strong>L2-a</strong>, a
structurally optimized, highly selective BTKi designed to minimize
drug-induced liver injury (DILI) and improve blood–brain barrier
(BBB) permeability. Using scaffold hopping, the nitro-substituted
aromatic scaffold was replaced with a trifluoromethyl-substituted
heterocycle, significantly reducing hepatotoxicity while enhancing
binding interactions with key BTK residues (CYS481, GLY480, and ASP539). <strong>L2-a</strong> also demonstrates lower CYP inhibition, reduced AMES
toxicity, and improved membrane absorption, ensuring better druggability.
Computational analyses, including site-directed mutation studies,
molecular dynamics (MD) simulations, and quantum chemical analysis,
confirmed <strong>L2-a</strong>’s high binding specificity, reduced
off-target effects, and strong conformational stability across BTK
variants. <strong>L2-a</strong> mitigates off-target effects through structural
optimization of the BTK binding pocket, particularly at CYS481, enhancing
binding specificity and reducing nonspecific interactions across BTK
mutants. These findings establish <strong>L2-a</strong> as a next-generation
BTKi with enhanced safety and therapeutic potential for autoimmune
and inflammatory diseases, particularly MS.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d268/12332581/55a6aba8afe0/ao5c03682_0011.jpg" loading="lazy" id="abs1" height="555" width="549" alt="graphic file with name ao5c03682_0011.jpg"></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d268/12332581/55a6aba8afe0/ao5c03682_0009.jpg" loading="lazy" id="tgr1" height="555" width="549" alt="graphic file with name ao5c03682_0009.jpg"></p></section><section id="sec1"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Multiple sclerosis (MS)
is a chronic, progressive autoimmune disorder
of the central nervous system (CNS), characterized by neuroaxonal
degeneration, neuroinflammation, and demyelination.<em>
,
</em> While
traditionally considered T cell-driven diseases, emerging evidence
underscores the significant contribution of B cells and innate immune
cells to the initiation and progression of MS. Peripheral immune cells
infiltrate the CNS, crossing the blood-brain-barrier (BBB) and contribute
to relapse episodes and the formation of demyelinating lesions.<em>
−

</em> The infiltration of these cells, including B cells, microglia, astrocytes,
and macrophages, drives local inflammation within the CNS, exacerbating
neurodegeneration by releasing cytokines, chemokines, and reactive
species.<em>
,
</em>
</p>
<p>MS manifests predominantly as relapsing-remitting
MS (RRMS) in
the early stages, with approximately 80% of patients experiencing
recurrent inflammatory attacks. Over time, most progress to secondary
progressive MS (SPMS), marked by neurodegeneration and irreversible
disability 8. A smaller proportion of patients, however, present with
primary progressive MS (PPMS), where continuous disease progression
occurs from the outset, without relapses. Both RRMS and progressive
forms of the disease involve complex immune interactions within the
CNS, with B cells playing a pivotal role in driving both systemic
and localized inflammation. B cell-mediated
immune responses, including the secretion of pro-inflammatory cytokines
such as IL-6, IL-10, and lymphotoxin-α, amplify myeloid cell
activation, contributing to the formation of ectopic lymphoid follicles
in the CNS and enhancing neuroinflammation and tissue damage.<em>
,
</em> Despite advancements in disease-modifying therapies (DMTs) that
can reduce relapse rates and slow progression, there remains a significant
unmet need for therapies that target the underlying neurodegenerative
processes, particularly in progressive MS. The development of B cell-targeting
therapeutics has revolutionized MS treatment, highlighting the potential
for therapies that specifically modulate the inflammatory cascade
driven by B cells and myeloid cells.<em>
−


</em> Bruton’s tyrosine kinase
(BTK) is recognized as a therapeutic target for MS disease due to
its pivotal role in B-lymphocyte function and maturation. Moreover, BTK is regulated in BCR-mediated antigen
presentation, affecting immune responses. The investigation revealed
that antigens such as CD19, CD22, CD40, and CD45, along with BTK signaling
kinase as a coreceptor, are substantial for B-cell function and may
influence the BCR pathway, impacting the immunological tolerance of
B cells.
</p>
<p>Consequently, BTK is pivotal
in the inflammation functions of immune
cells, specifically in the myeloid cells and B-cells, including microglia
and macrophages. BTK inhibitors (BTKi), such as Zanubrutinib, Ibrutinib,
and Acalabrutinib, are approved for treating B-cell malignancies,
demonstrating efficacy in B-cell suppression. This kinase BTK is integral
not only for B-cell receptor signaling but also for the subsequent
downstream signaling of chemokine receptors, which are crucial in triggering the release of proinflammatory cytokines and activating integrins. BTK belongs to the TEC family, which comprises at least
one member expressed in many immune cells. BTK protein is composed
of 659 amino acids. It primarily consists of five domains, with the
kinase domain featuring two nodes, the amino-terminal (N-terminal)
and the carboxyl-terminal (C-terminal) nodes. The active site of BTK
occurs among these nodes. The ATP-binding cleft is located between
the two distinct nodes, with the active cysteine (CYS481) situated
in the hinge area at the outer edge of the ATP-binding pocket, which exhibits complementary conformations and
rotations, revealing the ATP-binding cleft in the state of activation.
The catalytic site residue TYR551 is positioned within the activation
loop of the C-terminal loop.</p>
<p>BTK comprises five domains: a C-terminal
tyrosine kinase domain
(TK), also known as SRC homology 1 (SH1), an N-terminal pleckstrin
(PH) domain, a proline-rich TEC homology (TH) domain, and two SRC
homology domains (SH2 and SH3). The TH
domain contains a proline-rich region and a conserved zinc-finger
motif. The PH domain is linked to phosphatidylinositol 3,4,5-triphosphate
(PIP3), which promotes targeting of BTK to the cell membrane. Autophosphorylation
at TYR223 in the SH3 domain alleviates the active conformation and
stimulates the activity of BTK. Transphosphorylation begins at TYR551
within the catalytic BTK kinase domain, mediated by spleen tyrosine
kinase (SYK) or LYN kinase domain. The activated BTK protein will
bind to the adaptor protein BLNK through its SH2 domain, triggering
downstream signaling events (<a href="#fig1" class="usa-link">Figures 
</a>A). The BCR is consistently
associated with the Igα/Igβ heterodimer integrated within
the BCR complex. Antigen binding to the BCR induces phosphorylation
of the immunoreceptor tyrosine-based activation motifs (ITAMs) in
the cytoplasmic tails of Igα/Igβ through the SRC family
kinase LYN, which subsequently activates SYK. This activation is initiated
by the phosphorylation of SYK by LYN, which subsequently facilitates
the recruitment of BTK from the cytosol to the plasma membrane. BTK activation is characterized by phosphorylation
at TYR551, which boosts its catalytic activity and triggers autophosphorylation
at TYR223 within the SH3 domain. The activated BTK phosphorylates
phospholipase C gamma 2 (PLCγ2), leading to the generation of
the two second messengers, including diacylglycerol (DAG) and inositol
1,4,5-triphosphate (IP3). IP3 mediates the activation of calcium channels,
leading to the translocation of nuclear factor NFAT into the nucleus.
DAG activates the nuclear factor-kappa-light-chain-enhancer (NF-κB)
in activated B-cells pathway and the mitogen-activated protein kinase
(MAPK) pathway.
</p>
<figure class="fig xbox font-sm" id="fig1"><h3 class="obj_head">1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332581_ao5c03682_0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d268/12332581/ef88733e0456/ao5c03682_0001.jpg" loading="lazy" id="gr1" height="585" width="629" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>BTK structure and BCR
pathway. (A) CYS481 on the catalytic domain
of BTK is the main target of BTKi, while boosted by phosphorylation
of TYR223 and TYR551. (B) Signal transduction in the BCR signaling
pathway, which can be inhibited by BTKi through cross-linking multiple
pathways (NFAT, MAPK, NF-kB).</p></figcaption></figure><p>Recent research demonstrates that the binding of BTKi to the CYS481
residue of the BTK therapeutic target decreases the accessibility
of the activation loop, highlighting its significance as a promising
target for inhibiting kinase activity in MS disease. However, the expression of numerous genes critical for
B-cell proliferation, survival, and the production of chemokines and
cytokines such as IL-6 and IL-10 is regulated by NF-κB, which
also plays an anti-inflammatory role. Consequently, targeting BTK
inhibition offers a promising approach to treating MS disease by reducing
B-cell function and activation and suppressing microglia activity
(<a href="#fig1" class="usa-link">Figures 
</a>B). Moreover,
research exhibited that drug-induced liver injury (DILI) remains a
critical safety concern associated with BTKi, as several FDA-approved
BTKi have shown hepatotoxic effects. Many patients affected by DILI
have experienced progressive complications, including acute liver
failure, potentially due to the formation of reactive metabolites
contributing to hepatotoxicity. Although BTKi exhibits significant
therapeutic potential, its clinical application for MS disease remains
limited. Many BTKi are currently in clinical use or under investigation,
were originally developed for B-cell malignancies, and often display
inadequate penetration of the blood-brain barrier (BBB), thereby limiting
their efficacy in targeting neuroinflammation associated with MS.
Additionally, prolonged administration may result in immunosuppression
or insufficient control of disease progression. These limitations
highlight the urgent need for next-generation BTKi with enhanced CNS
accessibility and improved safety profiles tailored for MS treatment.
To mitigate this risk and enhance the pharmacological profile, we
employed scaffold optimization to improve BTKis’. In this study,
we reveal the interactions between potent inhibitors and BTK by hybrid
theoretical approaches. By fragment evolution, molecular modeling,
and the density functional theory (DFT), we investigate the action
mechanism of the optimized BTKi in the catalytic kinase domain of
BTK through dynamic behavior.</p></section><section id="sec2"><h2 class="pmc_sec_title">2. Results and Discussion</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">2.1. Interaction Analysis of Top <em>In Silico</em> Hits</h3>
<p>After the virtual screening, the ligand L2 (2-(5-methyl-3-nitro-1H-pyrazol-1-yl)-N-(quinoline-3-yl)
acetamide) emerged as the most promising compound docking on the predicted
druggable cavity site of the BTK kinase domain, which demonstrates
a strong binding affinity (−8.40 kcal/mol) and inhibitory effect,
as supported by its high docking score (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>). The docking analysis of the <strong>BTK-L2</strong> complex
revealed three conventional hydrogen bond formations involving BTK
residues, such as CYS481, ASN484, and TYR485, with bond distances
of 3.6 Å, 5.0 Å, and 4.4 Å, respectively. Additionally,
the <strong>BTK-L2</strong> complex exhibited pi-sigma interactions with
GLY480 at bond distances of 3.8 Å and 3.4 Å. Binding affinity
was significantly enhanced by the development of hydrogen bonds, which
enabled effective interactions with key residues in the receptor binding
pocket (<a href="#fig2" class="usa-link">Figure 
</a>A,B).
Although the interaction with key residue CYS481 is essential for
BTK binding activity, this interaction
was notably present in <strong>BTK</strong>-<strong>L2</strong> complex but
also exhibited a significantly high risk of drug-induced liver injury
(DILI) (DILI score: 0.99), similar to the FDA-approved BTKi Pirtobrutinib
(DILI score: 0.99), causing considerable hepatotoxic challenges for
therapeutic development (<a href="#tbl1" class="usa-link">Table 
</a>).</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d268/12332581/020a58e60ef8/ao5c03682_0002.jpg" loading="lazy" id="gr2" height="706" width="648" alt="2"></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Schematic representation of interaction analysis of <strong>L2</strong> with BTK (PDB ID: <a href="https://www.rcsb.org/structure/6OMU" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6OMU</a>). (A) 3-D cartoon model of the interaction of <strong>L2</strong> binding
mode with BTK. A close-up view shows interacting residues involved
in hydrogen bonds and hydrophobic interactions. (B) 2D interactions
of <strong>L2</strong> into the binding cavity of BTK. Green dotted lines
represent carbon–hydrogen bonds, while pink dotted lines exhibit
hydrophobic interactions.</p></figcaption></figure><section class="tw xbox font-sm" id="tbl1"><h4 class="obj_head">1. Comparative Analysis of Binding Energy,
Hepatotoxicity Risk (DILI Score) and Molecular Interaction Profiles
of <strong>L2</strong>, <strong>L2-a</strong> and Pirtobrutinib.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">Ligands</th>
<th align="center" colspan="1" rowspan="1">Binding Energy</th>
<th align="center" colspan="1" rowspan="1">DILI Score</th>
<th align="center" colspan="1" rowspan="1">Risk Level</th>
<th align="center" colspan="1" rowspan="1">Toxicity Interpretation</th>
<th align="center" colspan="1" rowspan="1">Hydrogen Bond Interactions</th>
<th align="center" colspan="1" rowspan="1">Hydrophobic Interactions</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>L2</strong>
</td>
<td align="center" colspan="1" rowspan="1">–8.40</td>
<td align="center" colspan="1" rowspan="1">0.998</td>
<td align="left" colspan="1" rowspan="1">high</td>
<td align="left" colspan="1" rowspan="1">Toxic</td>
<td align="left" colspan="1" rowspan="1">CYS481, TYR485, ASN484,
MET477</td>
<td align="left" colspan="1" rowspan="1">LEU408, VAL416, LEU528, ALA428</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>L2-a</strong>
</td>
<td align="center" colspan="1" rowspan="1">–9.62</td>
<td align="center" colspan="1" rowspan="1">0.186</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">Safe</td>
<td align="left" colspan="1" rowspan="1">CYS481,
GLY480, ASP539</td>
<td align="left" colspan="1" rowspan="1">LEU528, ILE472, LEU460</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Pirtobrutinib</strong>
</td>
<td align="center" colspan="1" rowspan="1">–9.75</td>
<td align="center" colspan="1" rowspan="1">0.998</td>
<td align="left" colspan="1" rowspan="1">high</td>
<td align="left" colspan="1" rowspan="1">Toxic</td>
<td align="left" colspan="1" rowspan="1">LYS430, CYS481, ARG525, ASN526, SER538, ASP539</td>
<td align="left" colspan="1" rowspan="1">VAL416,
ILE472, LEU528, LEU542</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>
<em>In-silico</em> ADMET analysis predicts
DILI values
range from 0 to 1, where higher values (closer to 1) indicate an increased
risk of liver toxicity, while lower values (closer to 0) correspond
to a reduced toxicity potential. To overcome this challenge, a scaffold
hopping strategy was employed, leading to the rational design of a
fragment library that effectively reduced DILI risk while preserving
strong binding affinity to the BTK target. During the lead optimization
phase, structure–activity relationship (SAR) analysis and scaffold
modifications were employed to refine the potential hits identified
through high-throughput screening (HTS). The objective was to develop
a potent BTKi to reduce DILI severity while effectively integrating
with the key residue of the BTK receptor. Subsequently, the lead optimization
strategy facilitated the transition from L2, the best-docked ligand,
to the top-hit lead candidate <strong>L2-a</strong> ((z)-N-(1-fluoro-2-(quinoline-3-yl)­vinyl)-5-(trifluoromethyl)-1,3,4-oxadiazol-2-amine),
which emerged as the most promising scaffold modification by significantly
reducing DILI risk (DILI score: 0.186), enhancing its pharmacological
profile, and demonstrating the highest docking score of −9.62
kcal/mol (<a href="#tbl1" class="usa-link">Table 
</a>).
The nitro group on the pyrazole ring in L2 was substituted with a
trifluoromethyl-substituted group, enhancing its interactions by enabling
hydrogen bond formation with the CYS481 residue (<a href="#fig3" class="usa-link">Figure 
</a>). The SAR analysis revealed
that <strong>L2-a</strong> retains the quinoline structure, which is speculated
to contribute to potential π-π interactions with the receptor
site. In contrast, the adjacent diene structure provides rigidity
to the molecule, enhancing its stability. The fluorine (F) group,
as a substantial hydrogen bond donor, improves pharmacokinetic properties
and strengthens receptor interactions. As a hydrogen bond acceptor,
fluorine further enhances drug-receptor interaction stability and
increases molecular compatibility. Its high electronegativity modifies
the overall dipole moment of the molecule, enhancing molecular stability
and facilitating electrostatic interactions within receptor sites.
Additionally, fluorine incorporation enhances pharmacokinetic properties,
including biodistribution, receptor-binding efficiency, and compound
potency.<em>
,
</em> The nitrogen atom in the chain provides
flexibility, enabling hydrogen bond formation with nearby residues
and enhancing binding affinity. The 1,3,4-oxadiazole moiety supports
hydrogen bonding and offers π-π stacking potential, further
stabilizing ligand–receptor interactions. The −CF<sub>3</sub> group is an effective hydrogen-binding site contributing
to the <strong>BTK-L2-a</strong> interaction, which improves membrane
permeability and increases molecular bioavailability.</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332581_ao5c03682_0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d268/12332581/4360e5527962/ao5c03682_0003.jpg" loading="lazy" id="gr3" height="218" width="767" alt="3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Structure-based drug
discovery. (A) <strong>L2</strong> represents
the initial hit with fragment replacement by Scaffold hopping. (B) <strong>L2-a</strong>, the optimized lead fragment, significantly improves
binding affinity and reduces toxicity. The yellow highlighted region
in <strong>L2</strong> and <strong>L2-a</strong> represents the conserved
core structure retained during optimization.</p></figcaption></figure><p>To provide a more comprehensive elucidation of the enhanced pharmacological
profile of <strong>L2-a</strong>, particular attention was given to the
steric and electronic contributions of its key substituents, with
a specific focus on the trifluoromethyl (−CF<sub>3</sub>) group.
The integration of the −CF<sub>3</sub> moiety significantly
increases molecular lipophilicity, thereby facilitating passive transcellular
diffusion and improving membrane permeability and oral bioavailability.
As a highly electronegative and metabolically stable substituent,
the −CF<sub>3</sub> group also contributes to reduced susceptibility
to enzymatic degradation and attenuates off-target interactions by
limiting nonspecific binding affinity. Moreover, the introduction
of fluorine atoms enhances the overall molecular dipole moment, which
promotes favorable electrostatic complementarity within the BTK active
site and contributes to the stabilization of the ligand–receptor
complex. Consequently, these combined electronic and spatial modifications
enhance the compound’s selectivity toward BTK and reduce the
likelihood of undesired cross-reactivity.</p>
<p>The stability of these
interactions is primarily determined by
the overall energy change associated with the formation and dissociation
of the hydrogen bonds, as reflected in the binding energies of compound <strong>L2</strong> (−8.40 kcal/mol) and <strong>L2-a</strong> (−9.62
kcal/mol). Compound <strong>L2-a</strong> forms a conventional hydrogen
bond between the fluorine atom and the essential key residue CYS481
(3.8 Å and 4.1 Å) of the receptor, which is crucial in reducing
the anti-BTK activity of the protein (<a href="#fig4" class="usa-link">Figure 
</a>). Although another conventional hydrogen
bond is formed in <strong>L2-a</strong> with the amino acid residue ASP539
(3.5 Å). Secondary forces also facilitate additional molecular
interactions, such as the sulfur-pi-alkyl interaction MET449 (5.5
Å) between the amino acid residue of the BTK receptor, leading
to significant electrostatic contacts. Interaction details of the
docked complexes <strong>L2</strong>, <strong>L2-a,</strong> and Pirtobrutinib
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S1</a>) with the active site BTK receptor
are depicted in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Tables S2–S4</a> . The
diverse molecular interactions and conventional hydrogen bond formation
collectively stabilize the ligand <strong>L2-a</strong> within the catalytic
binding pocket of the BTK protein, which is essential for ligand inhibitory
function.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d268/12332581/a22fa82c0ea7/ao5c03682_0004.jpg" loading="lazy" id="gr4" height="697" width="661" alt="4"></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Schematic representation of interaction analysis of <strong>L2-a</strong> with BTK (PDB ID: <a href="https://www.rcsb.org/structure/6OMU" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6OMU</a>). (A) 3-D cartoon model of the interaction of <strong>L2-a</strong> binding
mode with BTK. A close-up pattern shows the interacting residues involved
in the hydrogen bond and hydrophobic interactions. (B) The schematic
representation of 2D interactions of <strong>L2-a</strong> into the catalytic
cavity of BTK. Green dotted lines represent carbon–hydrogen
bonds, while pink dotted lines exhibit hydrophobic interactions.</p></figcaption></figure></section><section id="sec2.2"><h3 class="pmc_sec_title">2.2. Druggability Optimization
of Lead Compounds</h3>
<p>The scaffold hopping strategy applied in
the transition from <strong>L2</strong> to <strong>L2-a</strong> was designed
to optimize the functional
group profile, enhancing the overall druggability of the lead compounds
by improving their pharmacokinetic and pharmacodynamic properties
while minimizing the risk of liver toxicity and metabolic risks. The
optimization involved replacing the nitro-substituted pyrrole ring
in <strong>L2</strong> with a fluorinated triazole ester moiety in <strong>L2-a</strong>, resulting in structural modifications from a nitrogen
and oxygen-rich fragment to one bearing a fluorine atom and a trifluoromethyl
group. These key groups are well-known to improve metabolic stability
by reducing CYP enzyme inhibition, contributing to overall drug-likeness,
and lowering the predicted risk of drug-induced liver injury (DILI).
However, <em>In-silico</em> predictions suggest <strong>L2-a</strong> exhibits minimal interference with key cytochrome P450 (CYP) isoforms,
including CYP3A4, CYP2D6, CYP2C9, CYP2C19, and CYP1A2, indicating
a lower likelihood of metabolic complications. Subsequently, the lower
CYP inhibition exhibited by <strong>L2-a</strong> has important pharmacological
implications, particularly in reducing the risk of drug–drug
interactions (DDIs). Higher inhibition of cytochrome P450 enzymes
can interfere with the metabolism of concurrently administered drugs,
leading to increased systemic exposure, toxicity, and adverse clinical
outcomes. In contrast, the lower inhibitory potential of <strong>L2-a</strong> minimizes this risk, making it more favorable for use in multidrug
treatment regimens. Consequently, this improved metabolic profile
contributes to greater safety, reduced off-target liabilities, and
enhanced clinical compatibility compared to the higher-CYP-inhibiting
parent compound, <strong>L2.</strong> However, <strong>L2</strong> and <strong>L2-a</strong> demonstrate inhibitory activity against CYP2C19, which
may require further assessment, whereas their impact on other CYP
isoforms is minimal. Incorporating fluorine atoms in <strong>L2-a</strong> was pivotal in enhancing several ADMET properties. Notably, the
lipophilicity of <strong>L2-a</strong> (log <em>p</em> = 2.23)
was improved compared to <strong>L2</strong> (log <em>p</em> =
1.69), contributing to better membrane permeability and stronger receptor
binding, thus enhancing its potential as an oral drug candidate. However,
introducing fluorine atoms in <strong>L2-a</strong> has contributed to
improved water solubility (logS = −4.584) compared to <strong>L2</strong> (log S = −3.43) by enhancing the polarity and dispersibility
of the compound, making it a better candidate for drug absorption
and bioavailability. Moreover, integrating fluorine groups significantly
improved blood-brain barrier (BBB) permeability, positioning <strong>L2-a</strong> as a promising candidate for central nervous system (CNS)
applications without compromising bioavailability. Additionally, Fluorine
also plays a crucial role in enhancing cell penetration, facilitating
more efficient drug delivery, and interactions with target sites.
However, the reduction of nitrogen-rich groups from <strong>L2</strong> to <strong>L2-a</strong> may contribute to a lower risk of nonspecific
interactions, as predicted by the ADMET model, potentially improving
overall pharmacokinetic behavior and reducing undesirable side effects.
Furthermore, <strong>L2-a</strong> demonstrated superior gastrointestinal
absorption (GI) and a higher volume of distribution (VD = 0.23 L/kg)
compared to <strong>L2</strong> (VD = −0.19 L/kg), indicating improved
tissue penetration and systemic distribution in blood plasma (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S5</a>).</p>
<p>ADMET predictions revealed that <strong>L2-a</strong> has a significantly lower risk of DILI compared to <strong>L2</strong>, which exhibited Hepatotoxicity due to its oxygen-rich
structure. Furthermore, <strong>L2-a</strong> exhibited reduced Hematotoxicity,
reflecting a lower impact on oxygen transport and blood cell production.
Toxicity assessments also indicated that <strong>L2-a</strong> is AMES-negative,
with a lower potential for carcinogenicity and mutagenicity than <strong>L2</strong>, highlighting its improved safety profile. However, the
FDA-approved BTKi Pirtobrutinib, like <strong>L2</strong>, exhibited a
high DILI risk, reinforcing the hepatotoxic challenges commonly associated
with BTKi. The pharmacokinetic study further suggested that <strong>L2-a</strong> maintains sufficient oral bioavailability, with a balanced
profile of lipophilicity and hydrophilicity that facilitates optimal
membrane permeability and effective interactions with biological targets.
The synthetic accessibility (SA) score of <strong>L2-a</strong> (3.12)
compared to <strong>L2</strong> (2.82) suggests that <strong>L2-a</strong> is easier to synthesize, further supporting its feasibility for
drug development. Additionally, <strong>L2-a</strong> complies with key
drug-likeness criteria, including Lipinski’s Rule of Five and
Pfizer’s guidelines for nontoxicity, reinforcing its potential
as a viable oral BTK inhibitor. Consequently, <strong>L2-a</strong> demonstrated
a significantly lower DILI risk, highlighting the effectiveness of
scaffold modification in optimizing both safety and druggability.</p></section><section id="sec2.3"><h3 class="pmc_sec_title">2.3. Mitigation of Off-Target Effects of L2 and
L2-a Across BTK (C481) Mutant</h3>
<p>An integrated computational
approach incorporating IFD and metadynamics simulations accurately
predicts the impact of CYS481 mutations on the binding affinity of
the <strong>L2</strong> and <strong>L2-a</strong> inhibitors, leading to enhanced
specificity and reduced off-target effects. Off-target effects occur
when BTK inhibitors bind to unintended proteins, reducing specificity.
Our computational analysis of the CYS481 mutations in BTK suggests
that these changes improve the binding specificity of <strong>L2</strong> and <strong>L2-a</strong> and mitigate off-target effects by reducing
unintended interactions, lowering BTK activity, and enhancing selectivity.
However, structural and conformational analysis of wild-type and mutant
BTK in complex with the inhibitor <strong>L2</strong> and <strong>L2-a</strong> revealed that CYS481 substitutions significantly alter the protein
binding pocket and structural dynamics. Mutations to tyrosine (TYR481),
arginine (ARG481), serine (SER481), and phenylalanine (PHE481) retained
the residue position but introduced substantial changes in binding
dynamics, impacting ligand interactions and binding affinity. These
findings highlight the critical impact of CYS481 mutations on <strong>L2-a</strong> binding and provide key insights into drug resistance.
Therefore, the substitutions at position CYS481 altered the binding
interactions of <strong>L2-a</strong>, leading to new interaction patterns
with the mutant residues, even though the inhibitor was initially
designed to target native cysteine in the BTK active site (<a href="#fig5" class="usa-link">Figure 
</a>A,B). Binding energy
analysis revealed a notable enhancement in the affinity of <strong>L2-a</strong> (<a href="#fig5" class="usa-link">Figure 
</a>Ba-e for
specific BTK mutants, compared to <strong>L2</strong> (<a href="#fig5" class="usa-link">Figure 
</a>A­(a-e). In the wild-type BTK, <strong>L2</strong> exhibited a binding energy of −8.40 kcal/mol, whereas <strong>L2-a</strong> showed a slightly improved affinity of −9.62 kcal/mol.
However, in the mutant BTK variants, <strong>L2-a</strong> consistently
demonstrated superior binding affinity, except in the PHE481 mutant
(−8.42 kcal/mol), where <strong>L2</strong> exhibited stronger
binding (−9.53 kcal/mol). For other mutants, <strong>L2-a</strong> showed significantly excellent binding energies such as TYR481 (−10.82
kcal/mol), ARG481­(−9.21 kcal/mol), and SER481 (−8.54
kcal/mol) compared to <strong>L2</strong> involving TYR481 (−10.30
kcal/mol), ARG481 (−8.80 kcal/mol), and SER481 (−6.85
kcal/mol), suggesting an optimized interaction within the mutated
binding pocket.</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332581_ao5c03682_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d268/12332581/877466f0fd44/ao5c03682_0005.jpg" loading="lazy" id="gr5" height="955" width="768" alt="5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Representative configurations of <strong>L2</strong> and <strong>L2-a</strong> binding to wild-type and mutant forms of BTK, as determined
by IFD
binding energy analysis. (A) shows <strong>L2</strong> binding to BTK,
including (a) wild-type BTK, and the BTK mutants, (b) TYR481, (c)
ARG481, (d) SER481, and (e) PHE481. (B) illustrates <strong>L2-a</strong> binding to BTK, including (a) wild-type BTK, and the BTK mutants
(b) TYR481, (c) ARG481, (d) SER481, and (e) PHE481. The binding conformations
and associated energies demonstrate the effects of amino acid substitutions
at position 481 on the orientation and affinity of <strong>L2</strong> and <strong>L2-a</strong>.</p></figcaption></figure><p>Interestingly, the interaction
analysis revealed that CYS481 was
consistently retained in the wild-type BTK complex for both <strong>L2</strong> and <strong>L2-a</strong>, reinforcing its key role as a pivotal
residue in stabilizing BTK-ligand interactions. However, in the mutated
BTK complexes, the substitution of CYS481 with TYR, ARG, SER, and
PHE led to positional shifts in ligand orientation and significant
alterations in the hydrogen binding network. This investigation implies
that the mutations enhance the specificity of <strong>L2-a</strong> toward
BTK by optimizing its interactions within the modified binding pocket.
Furthermore, the interaction analysis indicates that CYS481 mutations
(TYR481, ARG481, SER481, and PHE481) disrupt the critical hydrogen
bonding network between BTK and <strong>L2-a</strong>, leading to a reduction
in hydrogen bonds and alteration in binding patterns, as illustrated
in <a href="#fig5" class="usa-link">Figure 
</a>B­(a–e).
Despite this disruption, <strong>L2-a</strong> sustained stronger interactions
with the mutated residues compared to <strong>L2</strong>, except for
the PHE481 mutant. Consequently, these structural modifications are
crucial in mitigating drug resistance by enhancing <strong>L2-a</strong> binding specificity as a BTKi, reducing unintended interactions,
and minimizing off-target effects while maintaining strong affinity
across BTK mutants.</p>
<p>To further validate ligand binding characteristics
across resistance-associated
BTK mutations, metadynamics-refined binding pose analysis using Schrodinger
Maestro for Pirtobrutinib in complex with wild-type BTK and four clinically
observed mutations at the CYS481 site: SER481, ARG481, TYR481, and
PHE481. As shown in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S2a–e</a>, Pirtobrutinib consistently retained high binding affinity across
all variants, with calculated binding affinities ranging from −7.23
to −9.53 kcal/mol. These results demonstrate residue-specific
alterations in the hydrogen bonding network while retaining ligand-binding
site compatibility. Moreover, for comparative analysis, <strong>L2</strong> and <strong>L2-a</strong> were similarly docked under the same meta-dynamic-based
simulation protocol. The corresponding binding poses and interaction
networks are provided in (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S3a,b</a>). <strong>L2</strong> exhibited a binding affinity of −7.85 kcal/mol,
stabilized by hydrogen bonds with CYS481, MET477, and ASN484. In contrast, <strong>L2-a</strong> achieved a more favorable binding affinity of −8.59
kcal/mol, forming stable interactions with CYS481, LEU408, and GLU488.
These findings suggest that <strong>L2-a</strong> demonstrates the potential
to provide enhanced structural compatibility with the BTK active site
compared to precursor <strong>L2</strong>.</p></section><section id="sec2.4"><h3 class="pmc_sec_title">2.4. Conformational
Stability and Structural Features
of the Leading Compound</h3>
<p>Molecular dynamics (MD) simulations
of the <strong>BTK-L2-a</strong> complex were performed to validate docking
accuracy and investigate the dynamic behavior of the protein–ligand
system under physiological conditions. Throughout the 100 ns simulation,
BTK exhibited local and global conformational changes upon <strong>L2-a</strong> binding, particularly in regions surrounding the binding pocket.
RMSD analysis was performed to monitor deviation in the backbone atoms
of the BTK protein from its initial to final conformation over a 100
ns MD simulation. The RMSD values for Cα and backbone atoms
remained relatively stable, with minor fluctuations around ∼1.5
to 2.0 Å, while heavy atoms displayed slightly higher fluctuations,
reaching up to ∼ 2.5 Å. The maximum RMSD recorded across
all peaks ranged from ∼2.5 to ∼3.0 Å (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S4A</a>). The catalytic pocket of the BTK
remained stable between 10 and 40 ns during its binding with the leading
compound <strong>L2-a</strong>, followed by a slight structural deviation
observed from 40 to 50 ns, after which the stable conformation was
regained. These findings suggest that BTK exhibits mild and transient
structural changes upon binding with the leading compound <strong>L2-a</strong>, allowing the active site to realign for interactions without compromising
the overall structural stability of the protein. The RMSF analysis
of the Cα atoms of each residue in the <strong>BTK-L2-a</strong> complex (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S4B</a>) revealed an overall
low fluctuation trend, with most residues exhibiting RMSF values between
∼1–2 Å, suggesting a stable backbone conformation
during the simulation. The resulting plot revealed an average RMSF
of ∼1.5 Å for the <strong>BTK-L2-a</strong> complex. Notably,
residues 520–540 and 610–620 exhibited higher fluctuations,
with RMSF values slightly above ∼2 Å, which may reflect
localized dynamic regions potentially involved in subtle conformational
changes upon <strong>L2-a</strong> binding. However, residues 450–480
showed minimal fluctuations, confined to approximately ∼ 1
Å, indicating a highly stable BTK structural core during the
binding process.</p>
<p>Moreover, the Rg analysis was used to evaluate
the compactness and folding behavior of BTK upon binding with the
lead compound <strong>L2-a</strong> structural core during the binding
process. Moreover, the Rg analysis was used to evaluate the compactness
and folding behavior of BTK upon binding with the lead compound <strong>L2-a</strong>. The average Rg value of approximately 19.32 Å,
with only minor fluctuations, indicates that BTK retained a compact
and stable conformation throughout the simulation. Initially, Rg values
fluctuated between ∼19.3 Å and ∼19.5 Å within
the first 20 ns, followed by a decrease to ∼19.0 Å, suggesting
that the BT became more compact, likely reflecting stabilization upon <strong>L2-a</strong> binding. This compact packing of BTK was consistently
observed throughout the 100 ns MD simulation (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S4C</a>), highlighting the structural integrity of the <strong>BTK-L2-a</strong> complex. Furthermore, the SASA analysis provides
crucial insights into the folding behavior and hydrophobic characteristics
of the <strong>BTK-L2-a</strong> complex, reflecting its structural dynamics
and solvent exposure. The average SASA value of 13,323.822 Å<sup>2</sup>, with a peak of 13,800 Å<sup>2</sup> at 80 ns, indicates
a transient increase in BTK surface exposure, possibly due to local
rearrangements at the binding interface. This was followed by a gradual
reduction to 12,800 Å<sup>2</sup> by 100 ns, implying that BTK
transitioned into a more compact and less solvent-exposed conformation,
reflecting enhanced hydrophobic packing upon ligand binding (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S4D</a>).</p>
<p>The fluctuation observed between
35 and 40 ns indicates localized
conformational flexibility at the BTK active site, contributing to
dynamic and stable interactions with the lead compound <strong>L2-a</strong>. Additionally, hydrogen bonds are crucial for molecular recognition,
complex stability, and selectivity by ensuring interaction specificity
and directionality. The total number of hydrogen bonds varies over
the simulation, ranging from 460 to 650, reflecting interactions between
the protein and ligand during the MD run. “Occupancy”
denotes the frequency of particular H-bonds sustained during the MD
simulation. The most stable conformation of the <strong>BTK-L2-a</strong> complex demonstrates around 520–540 high occupancy hydrogen
bonds, which are crucial for the complex stability and binding affinity,
as they remain intact for a significant duration of the simulation.</p>
<p>To investigate the correlated motions between the <strong>L2-a</strong> and BTK complex, a dynamic cross-correlation map (DCCM) analysis
was performed on the Cα atoms obtained from the MD simulation
trajectory. The DCCM map reflects inter-residual motions: anticorrelated
motions (blue) indicate residual movement in opposite directions,
while positively correlated motions (red) reflect coordinated motion
in the same direction, either parallel or antiparallel. In the DCCM
map, blue postulates negative correlations, and red signifies positive
correlations (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S5</a>). The analysis
revealed extensive positively correlated motion between <strong>L2-a</strong> and key active-site residues of BTK, particularly at the binding
interface, suggesting highly coordinated motion and stable interactions
that enhance complex stability. For instance, these positively correlated
regions represent coordinated fluctuations among residues, promoting
a structurally reinforced ligand-binding interface. In contrast, the
fewer anticorrelated regions may indicate areas of structural flexibility,
demonstrating local conformational changes that facilitate <strong>L2-a</strong> binding within the pocket. However, the interaction of <strong>L2-a</strong> induces favorable correlated motions that improve the structural
stability of the BTK complex, while selective anticorrelated dynamics
retain essential conformational flexibility. Consequently, the results
also showed an increase in positively correlated motions at the <strong>BTK-L2-a</strong> binding interface, along with a higher percentage
of pairwise-connected residues. This pattern suggests that the ligand
binding plays a significant role in stabilizing the BTK structure
by enhancing inter-residue coordination. Collectively, these findings
highlight the dual role of <strong>L2-a</strong> in contributing to both
the structural integrity and conformational flexibility of the <strong>BTK-L2-a</strong> complex through modulation of inter-residue motion.</p>
<p>The structural superimposition of the <strong>BTK-L2-a</strong> complexes
before and after MD simulation revealed no significant changes in
the overall protein conformation. Visual inspection indicates that
the complex obtained after MD simulation stabilized and slightly enlarged
the protein–ligand binding cavity; however, it retained the
spatial configuration of the protein and showed minimal deviation
from its final position. Additionally, after the MD simulation, a
slight shift in the orientation of the ligand <strong>L2-a</strong> occupancy
within the binding cavity of BTK was observed, potentially caused
by modifications in the secondary structure at the binding site. Quantitatively,
RMSD analysis evaluated minor variations between the docking and MD
structures (<a href="#fig6" class="usa-link">Figure 
</a>), confirmed that the structural changes were minor, and the ligand
remained securely located within the binding pocket. This investigation
aimed to determine whether the ligand endured conformational changes
to elucidate the observed deviations. An RMSD value ranging between
∼1.5 and ∼2.0 Å was observed during the MD analysis.
Since the corresponding RMSD value was below or equal to the similarity
threshold of 2 Å standard value, it can be inferred that the ligand <strong>L2-a</strong> exhibited no
significant conformational changes.</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d268/12332581/16317d8dd79c/ao5c03682_0006.jpg" loading="lazy" id="gr6" height="313" width="552" alt="6"></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Superimposition structural alignment of
two complexes, consisting
of the docking-derived complex and the last pose from the MD simulation.
Superimposition of BTK (PDB ID: <a href="https://www.rcsb.org/structure/6OMU" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6OMU</a>) and <strong>L2-a</strong> complex after
docking (gray) and after MD (pink). The RMSD value of the complexes
aligned in PyMOL is 1.17 Å.</p></figcaption></figure><p>Furthermore, the optimized structure of <strong>L2-a</strong> exhibited
accurate geometrical parameters (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S6</a>), with bond lengths (C–C, C–H, C–N, and C–O)
ranging from 1.01 Å to 1.49 Å and bond angles spanning 106.6°
to 121.96° indicating structural stability and accuracy (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S6</a>). The FMO analysis of <strong>L2-a</strong> revealed a HOMO–LUMO energy gap (ΔEH-L) of 4.46 eV,
indicating an optimal balance between kinetic stability and chemical
reactivity. The HOMO density is predominantly localized over the triazole
ring and adjacent electronegative heteroatoms (N and O), identifying
regions with high electron-donating potential. These electron-rich
regions are favorably orientated to participate in noncovalent interactions
with electrophilic amino acid residues in the BTK binding site, including
positively charged sides such as those of histidine and lysine. Conversely,
the LUMO orbital is delocalized across the fluorinated aromatic framework,
highlighting electron-deficient zones that can act as electron acceptors,
facilitating charge transfer interactions with nucleophilic residues
such as thiol or hydroxyl-containing amino acids (<a href="#fig7" class="usa-link">Figure 
</a>A). Additionally, the DOS (density
of state) analysis showed HOMO at −6.15 eV, LUMO at −1.70
eV, and an energy gap reflecting charge transfer and orbital contributions
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S7</a>). According to Koopman’s
theorem, the ionization potential (IP) corresponds to the HOMO, while
the electron affinity (EA) is associated with the LUMO, influencing
molecular reactivity and chemical behavior. Moreover, global reactivity descriptors (GRD) of <strong>L2-a,</strong> including ionization energy of −0.23 eV, reflect thermal
stability, while a low electron affinity (0.06 eV) suggests a relatively
high LUMO energy, enhancing electron acceptance. The ligand exhibits
kinetic stability with a chemical hardness (η) of 0.08 eV and
significant softness (S) of 12.20 e<sup>–1</sup> V<sup>–1</sup>, indicating higher polarizability. The chemical potential (μ)
of −0.14 eV further suggests moderate reactivity (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S7</a>). However, the MESP surface map of <strong>L2-a</strong> is visualized using a color gradient from red to blue
regions. Regions of negative potential (−7.225e<sup>–2</sup> a.u) are prominently localized around the triazole nitrogen atoms,
either oxygen or fluorine substituents corresponding to the region
of high electron density, strong electrostatic potential, nucleophilic
sites, and suggesting favorable hydrogen bond accepting capability.
However, the positive electrostatic potential region (+7.225e<sup>–2</sup> a.u), notably around the quinoline ring system, corresponds
to electrophilic hot spots, rendering them suitable for interactions
with nucleophilic residues such as cysteine, serine, or tyrosine in
the BTK binding pocket (<a href="#fig7" class="usa-link">Figure 
</a>B).</p>
<figure class="fig xbox font-sm" id="fig7"><h4 class="obj_head">7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332581_ao5c03682_0007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d268/12332581/e02a33c2e0f6/ao5c03682_0007.jpg" loading="lazy" id="gr7" height="1026" width="785" alt="7"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) FMOs, (B) MEP, and Non-Covalent Interaction analysis (NCI),
(C) 2D and (D) 3D plots of RDG: (E) ELF, (F) LOL, (G) IGM, and (H)
Iso scatter plot of δg vs sign (λ2) ρ of the lead
optimized compound L2-a calculated at B3LYP/6–311G*­(d, p).</p></figcaption></figure><p>To further insight into the electronic features
of the lead compound <strong>L2-a</strong> that may influence its binding
interaction with BTK,
we performed a comprehensive noncovalent interaction (NCI) analysis
including reduced density gradient (RDG), electron localization function
(ELF), localized orbital locator (LOL), and the independent gradient
model (IGM). Among these, the RDG method is a powerful technique for
evaluating NCI by visualizing molecular interactions within a molecular
environment, such as steric effects, hydrogen bonds (H–B),
and van der Waals (VDW) forces.</p>
<p>The RDG plots reveal distinct
interaction types, where red zones
(0.020 au &lt; sign­(λ<sub>2</sub>) ρ &lt; 0.035 au) indicate
steric repulsion, blue zones (−0.035 au &lt; sign­(λ<sub>2</sub>) ρ &lt; −0.020 au) signifies strong, attractive
forces such as hydrogen bonding, and green areas (−0.015 au
&lt; sign­(λ<sub>2</sub>) ρ &lt; −0.010 au) represent
VDW interactions, which are consistent with the reported reference. Although the DFT analysis was performed on <strong>L2-a</strong> independently, the blue regions observed in the RDG Isosurface
indicate strong attractive interactions, corresponding to electron-rich
areas with a high potential to participate in hydrogen bonding. These
regions spatially correlate with hydrogen bond contacts predicted
by molecular docking at key BTK residues such as CYS480 and GLY480,
suggesting that the intrinsic electronic properties of <strong>L2-a</strong> contribute to its favorable binding orientation and interaction
profile within the BTK active site. The 2D and 3D RDG plots confirm
the presence of these interactions, highlighting their spatial distribution
around key molecular regions, particularly around the aromatic ring
(<a href="#fig7" class="usa-link">Figure 
</a>C,D).</p>
<p>Topological analysis of the ELF, LOL, and IGM was performed to
investigate NCI in the <strong>L2-a</strong> compound. The ELF and LOL
analyses provided further insight into the electron distribution and
bonding characteristics within <strong>L2-a</strong>. ELF maps revealed
strong electron localization around hydrogen atoms, particularly on
the donor groups involved in hydrogen bonding with ASP539 and ILE472,
with localization values ranging from 0.5 to 1.0 for these regions.
In contrast, delocalized zones associated with carbon and nitrogen
atoms showed lower ELF values between 0.0 and 0.5, as reflected by
red and blue color distribution in the ELF map, respectively (<a href="#fig7" class="usa-link">Figure 
</a>E). However, the
LOL function, which reflects kinetic energy density, revealed variations
in orbital localization across the <strong>L2-a</strong> compound. It
exhibited well-defined orbital confinement along the core structure,
with white regions indicating localized bonding, red zones confirming
strong covalent bonds, and blue areas corresponding to weak interactions
such as N–H···O and C–H···O
hydrogen bonds (<a href="#fig7" class="usa-link">Figure 
</a>F). Notably, weak interaction zones were observed around LEU460 and
LEU528, consistent with hydrophobic contacts identified in the docking
analysis. A Promolecular density approach was employed to quantify
van der Waals interactions and hydrogen bonding by analyzing electron
density gradient attenuation from interatomic interactions in the
chemical system. The IGM analysis further visualizes and quantifies
these intra- and intermolecular interactions, with δg values
ranging from 0 (blue) to 0.2 (red), representing regions of electron
density accumulation and depletion (<a href="#fig7" class="usa-link">Figure 
</a>G). High δg values around hydrogen
atoms indicated strong hydrogen bonding, while lower δg values
and depletion zones near aromatic carbons suggested the presence of
π–π stacking and dispersion interactions. These
interaction patterns correspond to the observed binding pose of <strong>L2-a</strong> within BTK, where the quinoline ring forms stabilizing
van der Waals contacts with MET499 and LEU460, supporting the role
of these noncovalent forces in ligand-protein binding. For instance,
the scatter plot of δg versus sign­(λ<sub>2</sub>)­ρ
effectively distinguishes interaction types, identifying weak hydrogen
bonds and van der Waals interactions within the −0.05 to 0.05
range (<a href="#fig7" class="usa-link">Figure 
</a>H).</p>
<p>To explore charge transfer and orbital-level interactions, natural
bond orbital (NBO) analysis and transition density matrix (TDM) calculations
for <strong>L2-a</strong> were performed. NBO results (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S8</a>) identified key donor–acceptor interactions
with significant stabilization energies of 60.17, 32.12, and 21.13
kcal/mol for *LP<sub>1</sub> (N14) → π­(N16–C17),
32.12, and 21.13 kcal/mol for *LP<sub>1</sub> (N14) → π­(N16–C17),
*LP<sub>2</sub> (O18) → π­(CN15–C19), and *LP<sub>3</sub> (F24) → π­(C11–C12) **, respectively.
This delocalization suggests strong internal charge stabilization,
contributing to ligand rigidity and improved binding performance (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S8</a>). Additionally, weak interactions such
as LP<sub>3</sub> (F22) → σ­(C20–F22) * and LP<sub>3</sub> (F21) → σ­(C20–F23) * contributed to C–H···F
hydrogen bonds were observed at the CYS481 site, which further enhances
ligand-protein binding affinity. The inter- and intramolecular interactions
calculated from the NBO analysis confirm the compound’s bioavailability,
as these orbital interactions contribute directly to its stability
within the BTK binding pocket. Furthermore, the TDM plots effectively
visualized electron–hole coherence and intramolecular charge
transfer (ICT), predominantly localized in the molecular core, highlighting
the delocalization of electronic transitions within the <strong>L2-a</strong> compound (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S9</a>). The spatial distribution
of TDM-based ICT in <strong>L2-a,</strong> predominantly localized around
its central core, indicates regions of electronic activity that may
contribute to its potential interactions with BTK. Although TDM does
not directly model the protein–ligand complex, this electronic
contribution may hypothetically correspond to the binding interface
involving key residues ILE472 and ASP539, thereby providing supportive
insight into the observed binding energy of −9.62 kcal/mol
and the proposed electronic complementarity between <strong>L2-a</strong> and the BTK active site.</p></section><section id="sec2.5"><h3 class="pmc_sec_title">2.5. Proposed Synthetic Route
for Optimized Lead
Compound L2-a</h3>
<p>We designed a proposed synthetic route for
the top-ranked lead compound <strong>L2-a</strong>. The synthetic route
initiates with the preparation of compound 3. In the first step, a
25 mL round-bottomed flask equipped with a magnetic stirrer was combined
with 0.2 mmol of tri-n-butylamine and 25 mmol of compound 2 (carbonyl
fluoride compound). A water-cooled condenser was attached, and the
mixture was stirred at 100 °C on a steam bath for the necessary
time to obtain compound 3. In the second step, compound 3 (10 mmol)
was reacted with compound 4 (10 mmol) in CH<sub>2</sub>Cl<sub>2</sub>, carried out by the addition of sodium triacetoxyborohydride (14
mmol) and CH<sub>3</sub>COOH (10 mmol). The reaction mixture was stirred
at room temperature under a nitrogen atmosphere for 2 days until TLC
analysis confirmed the complete consumption of the reactants, yielding
the final product, <strong>L2-a</strong> (<a href="#fig8" class="usa-link">Figure 
</a>).</p>
<figure class="fig xbox font-sm" id="fig8"><h4 class="obj_head">8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332581_ao5c03682_0008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d268/12332581/4d8fd6aaca86/ao5c03682_0008.jpg" loading="lazy" id="gr8" height="189" width="629" alt="8"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Proposed synthetic route for <strong>L2-a</strong>. Compound <strong>1</strong> was coupled with <strong>2</strong> to yield
intermediate <strong>3</strong>, which was further reacted with <strong>4</strong> in the presence
of NaBH­(OAc)<sub>3</sub> and AcOH to produce the final compound, <strong>L2-a</strong>.</p></figcaption></figure></section></section><section id="sec3"><h2 class="pmc_sec_title">3. Conclusion</h2>
<p>Multiple sclerosis is a chronic autoimmune inflammatory demyelinating
disorder that causes gradual disability and neurological morbidity.
BTK has become a pivotal therapeutic target owing to its essential
role in controlling B-cell activation and innate immune signaling
pathways. Reported analysis revealed that BTKi are novel therapeutic
agents that have a promising feature for treating MS and inflammatory
disorders. However, a higher toxicity profile and drug resistance
associated with off-target kinase inhibition may restrict the use
of BTKi as therapeutic target agents since most patients with inflammation
exhibit mild symptoms. Therefore, this study focused on developing
highly selective BTKi to meet the criteria for clinical development
with fewer side effects, particularly minimal DILI toxicity. The newly
developed BTKi drug (<strong>L2-a</strong>) is highly safe, with a reduced
toxicity profile, a low molecular weight therapeutic agent with an
appropriate capability to infiltrate the BBB, and has significantly
less off-target drug resistance kinase activity.</p>
<p>The structural
optimization from <strong>L2 to L2-a</strong> was achieved
through scaffold hopping, strategically replacing the nitro-substituted
aromatic scaffold with a trifluoromethyl-substituted heterocycle.
This key modification not only mitigated the previously observed high
DILI risk but also introduced favorable interactions, particularly
enabling strong hydrogen bonding with the key interacting residues
such as CYS481, GLY480, and ASP539, indicating enhanced binding coverage
across the BTK binding pocket. The optimized structure of <strong>L2-a</strong> enhanced its druggability by reducing CYP inhibition, improving
BBB permeability, membrane absorption, and also showing reduced AMES
toxicity. Importantly, <strong>L2-a</strong> reduced off-target effects
through the tuning of its hydrogen bonding network and interaction
profile within the BTK binding site. Site-directed mutation analysis
performed through IFD and metadynamics simulations demonstrated that <strong>L2-a</strong> retained strong binding affinity across key CYS481 variants
(TYR481, ARG481, SER481, and PHE481) while enhancing binding specificity
and reducing off-target effect. MD simulation validated the conformational
stability of the <strong>BTK-L2-a</strong> complex, with consistent binding
interactions throughout the trajectory. Additionally, quantum chemical
analysis revealed favorable electronic properties of <strong>L2-a</strong>, indicating its potential for selective BTK inhibition.</p>
<p>In
addition to its favorable pharmacological and structural characteristics, <strong>L2-a</strong> demonstrates strong potential for clinical application
in CNS disorders, particularly MS. Its enhanced blood–brain
barrier permeability, coupled with significantly reduced risk of DILI
and minimal cytochrome P450 enzyme inhibition, supports its profile
as a more selective and pharmacologically favorable BTKi candidate
compared to existing agents. Together, these integrated results establish <strong>L2-a</strong> as a structurally optimized BTKi with reduced toxicity
and strong therapeutic potential for autoimmune and inflammatory diseases.
These pharmacokinetic and toxicological attributes are particularly
advantageous for MS patients requiring prolonged, multidrug therapy.
Furthermore, <strong>L2-a</strong> predicted oral bioavailability and
metabolic stability support its suitability for chronic dosing. Collectively,
these features highlight <strong>L2-a</strong> as a promising candidate
for further preclinical validation and advancement as a next-generation
BTKi for inflammatory CNS pathologies.</p></section><section id="sec4"><h2 class="pmc_sec_title">4. Experimental
Section</h2>
<section id="sec4.1"><h3 class="pmc_sec_title">4.1. Receptor and Ligand Preparation</h3>
<p>The
three-dimensional (3D) crystal structure of BTK was retrieved from
the Protein Data Bank (PDB ID: <a href="https://www.rcsb.org/structure/6OMU" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6OMU</a>) and meticulously prepared for subsequent
docking studies. The protein charge state was initially optimized
for a physiological pH of 7.0 and further processed using the protein
preparation wizard in Maestro v9.0 (Schrodinger, LLC, New York, USA). During the preparation phase, all heteroatoms
and water molecules were removed, and hydrogen atoms were labeled
in the structures. The protein structure was optimized, where bond
orders and formal charges were adjusted based on the number of hydrogen
bonds and Epic-derived p<em>K</em>
<sub>a</sub> values to ascertain
the optimal protonation state of the protein. The final step consisted
of energy minimization using the OPSL3 (optimized potentials for liquid
simulations) force field to alleviate
steric clashes or spatial constraints within the protein structure,
enhance stability, and provide a suitable conformation for docking
studies. The immunological and ROR-gamma-targeted library of small-molecule
drugs designed to combat autoimmune diseases was sourced in.sdf format
from the ChemDiv database (accessed on February 22, 2025), ChEMBL
version 32 (accessed on February 22, 2025), and the ZINC-22 database
(accessed on February 22, 2023). Each compound was prepared and optimized
through the Ligprep module in Maestro v9.1­(Schrodinger, LLC, New York,
USA). The ligands were converted to. Mae format during the preparation
process, with hydrogen atoms added and appropriate charges assigned.
Protonation states and tautomeric forms were generated at a pH of
7.2 ± 2.0. Using the Epik tool, the OPLS3 force field is applied
for energy calculation and accurate charges. To facilitate subsequent
docking studies, up to 32 conformers with varying stereochemical properties
were generated for each ligand, allowing for an extensive exploration
of their conformational diversity.</p></section><section id="sec4.2"><h3 class="pmc_sec_title">4.2. Prediction
of Receptor Binding Site</h3>
<p>The identification of a highly druggable
binding pocket and catalytic
site of the BTK protein was carried out using the cavity plus web
server <a href="http://www.pkumdl.cn:8000/cavityplus/computation.php" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.pkumdl.cn:8000/cavityplus/computation.php</a>, an online tool specifically designed for predicting protein cavities
with druggable potential and conducting various functional assessments (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S9</a>). The analysis
performed by Cavity Plus revealed a significant druggable cavity of
the BTK protein, including crucial key amino acid residues (CYS481,
GLU475, MET474, THR477) within the catalytic kinase domain (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S10</a>), which matches the required binding
active sites previously reported in the literature.<em>
−

</em>
</p></section><section id="sec4.3"><h3 class="pmc_sec_title">4.3. Virtual Screening</h3>
<p>The docking parameters
were meticulously chosen, with the van der Waals scaling factors for
both the receptor and ligand set to 0.50, and a maximum of 20 poses
was generated for each ligand. Additionally, refinement was applied
to residues within 5.0 Å, and the side chain was optimized to
improve docking accuracy. The most favorable docking pose was selected
based on prime energy values. Throughout the docking simulations,
both the ligands and receptors were treated as flexible entities to
account for dynamic interactions. The docking grid was generated using
the receptor grid generation feature of the GLIDE module (Receptor
Grid Generation, Maestro v9.0, Schrödinger version 203, LLC,
New York, NY, USA), a high-precision
molecular modeling package widely recognized for its empirical scoring
functions and grid-based ligand docking. The grid box was generated
around reported key amino acid residues, specifically CYS481, MET477,
THR474, and GLU475, which are crucial to BTK catalytic activity. The
grid box was defined with the subsequent dimensions: center_x = 16.35
Å, center_y = 7.728 Å, center_z = 1.107 Å, size_x =
19.96 Å, size_y = 21.62 Å, and size_z = 20.06 Å. The
HTVS glide docking mode was initially employed for virtual screen
libraries, including the immunological, ZINC database, and RORγ-targeted
libraries. The top 20% docked poses from the preliminary screening
were refined and rescored using the standard precision (SP) glide
docking mode to confirm precision and accuracy. The selected compounds
endured redocking using AutoDock Vina (version 1.2.3), a widely used
open-source docking tool that applies iterated local search global
optimization integrated into YASARA structure software (version 21.12.19,
YASARA Biosciences GmbH, Vienna, Austria, accessed in June 2023)<em>
,
</em> to further validate the top-scoring docking results. A semiflexible
docking protocol was applied, wherein the protein was kept rigid and
the ligands were allowed torsional flexibility. The exhaustiveness
parameter was set to 8, which ensures sufficient sampling of the conformational
space with a reasonable computational time frame.</p>
<p>The receptor
grid box, identical in size to that used in the initial docking, was
generated around the predicted druggable cavity. Energy minimization
of all complex structures was carried out using the NOVA force field
(YASARA proprietary, tailored for geometry optimization) to optimize
covalent geometry and efficiently resolve steric clashes between the
ligands. Furthermore, docking studies were conducted in YASARA using
the built-in “dock_run.mcr” macro module, with the AMBER03
force field applied to refine local interactions and backbone flexibility.
For each docking scenario, 25 poses were generated, resulting in 9
distinct structural clusters.</p></section><section id="sec4.4"><h3 class="pmc_sec_title">4.4. Lead
Optimization Using Scaffold Hopping Protocol</h3>
<p>The workflow
of the <strong>BTK-L2</strong> complex starts with optimizing
the protein–ligand complex through energy minimization to refine
its structure. According to the reported analysis, BTKi targeting
MS has raised significant concerns about drug-induced liver injury
(DILI) hepatotoxicity, <strong>a severe</strong> adverse effect associated
with FDA-approved BTKi. To address these
concerns, our strategy focused on optimizing potency and drug properties
to reduce adverse effects and enhance the efficacy of drug candidates.
The “Replace Fragment” protocol within the “lead
optimization” module of Discovery Studio v4.5 (accessed on 25 August 2023) was employed to address these
challenges, generating an extensive compound library (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S11</a>). This protocol replaces ligand fragments
of the input ligand to generate analogues by identifying bioisosteres
from built-in or custom fragment libraries and placing them into appropriate
scaffold positions based on receptor fit and interaction potential.</p>
<p>The software provides built-in fragment libraries containing various
linear and cyclic structures to replace specific ligand fragments,
determining improved activity and selectivity. The Replace fragment
protocol in Discovery Studio v4.5 facilitates ligand optimization
by systematically substituting specific molecular fragments with isosteric
fragments derived from its comprehensive built-in fragment libraries.
These libraries contain 447,535 linear fragments and 1,047943 cyclic
fragments, enabling the rational design and generation of novel ligand
structures with optimized pharmacological properties. This protocol
utilizes a scaffold hopping strategy, where structurally and functionally
similar fragments replace specific regions of the scaffold while maintaining
the critical ligand–receptor binding interactions. The process
begins with a systematic search of the fragment libraries to identify
appropriate isosteric fragments, which are then integrated into lead
compounds. A fragment is classified as isosteric if it exhibits a
comparable electronic, steric, and physicochemical profile to the
original fragment, as determined through molecular fingerprints and
predefined structural properties. By default, fragment similarity
is evaluated using Euclidean distance-based calculations, incorporating
key molecular properties such as aromatic ring count, overall ring
topology, and molecular surface area. The procedure involved sequential
steps, including fragment replacement similarity evaluation and in
situ ligand minimization, followed by a Pareto-based ranking that
considered binding site compatibility, minimized steric clashes, Lipinski’s
Rule of Five (RO5), and structural novelty.</p>
<p>This software provides
a tool to evaluate the new ligands based
on DILI score, binding affinity, ligand efficiency, and pharmacokinetic
properties. The newly generated lead compounds are ranked according
to their binding scores and compliance with RO5, certifying their
drug-like properties. These features provide an optimal balance between
chemical diversity and functional similarity to the original lead
compound. We generated a library of 771 novel analogues designed to
improve potency while minimizing DILI toxicity, confirm safety, and
increase efficacy for potential drug candidates. The replacement fragments
closely mimic the original structure, resulting in ligands that exhibit
an improved fit within the binding environment and enhanced functional
efficiency. The newly generated ligands are ranked based on the fragment
“novelty” and “RO5”. All candidate ligands
were screened according to RO5, which defines drug-likeness based
on the following criteria: molecular weight less than 500 Da, log
P below 5, no more than 5 hydrogen bond donors, and no more than 10
hydrogen bond acceptors. Ligands that fulfilled these parameters,
including <strong>L2</strong> and <strong>L2-a</strong>, were selected for
subsequent ADMET profiling and molecular docking analysis. Ligand
candidates were prioritized based on fragment similarity scores (ranging
from 0 to 1), ligand novelty indices, and predicted pharmacokinetic
properties. Compounds that obeyed RO5 and exhibited favorable binding
affinities were subsequently evaluated through comprehensive ADMET
profiling. Among the screened compounds, <strong>L2-a</strong> emerged
as the top-ranked ligand, exhibiting a favorable binding orientation
with key BTK active-site residues (notably CYS481), a markedly reduced
risk of drug-induced liver injury (DILI), and a superior ADMET profile,
thus representing a highly promising candidate for further investigation.
However, these fragments are further refined within the protein active
site, improving their interactions with the catalytic domain of BTK.
Thus, <strong>L2-a</strong>, the top-ranked lead ligand, emerged as a
promising candidate due to its lower risk of DILI toxicity and strong
binding affinity to the key residue of the BTK. Interestingly, despite
these structural optimizations, key binding residues such as CYS481
were consistently retained in all generated ligands, further validating
the efficiency of our scaffold hopping approach. Consequently, the
criteria, including fragment prioritization, receptor interaction
scoring, RO5 filtering, and ADMET profiling, were systematically integrated
into the scaffold hopping workflow to enable a rational and data-driven
selection of <strong>L2-a</strong>.</p></section><section id="sec4.5"><h3 class="pmc_sec_title">4.5. Visualization
and Interaction Analysis</h3>
<p>After the molecular docking study,
the molecular interactions between
the studied ligands and the targeted protein were analyzed using the
ligand interactions tool from the receptor–ligand interactions
module in Discovery Studio v4.5. This approach allowed for precisely
visualizing and characterizing the protein’s ligand-binding
regions by identifying dihedral angles, bond lengths, and types of
nonbonding interactions, specifically, amino-acid residues. Various
software tools, such as PyMOL and Discovery
Studio v4.5, were employed to visualize and analyze the structural
data to evaluate the ligand-protein molecular interactions.</p></section><section id="sec4.6"><h3 class="pmc_sec_title">4.6. Induced Fit Docking</h3>
<p>The docking of
ligand <strong>L2</strong> and <strong>L2-a</strong> into the BTK receptor
was conducted using the induced-fit docking (IFD) protocol in Maestro
v9.0 (Schrodinger, LLC, New York, USA), a computational approach specifically designed to model receptor
conformational changes induced upon ligand binding, thus enhancing
docking accuracy. The IFD workflow comprises four key steps: (1) initial
docking of the ligand into a rigid receptor using a softened-potential
mode to generate multiple docking poses, (2) protein flexibility sampling
to allow receptor rearrangements and adopt induced conformations,
(3) redocking of the ligand into low-energy receptor conformations,
and (4) final scoring of poses based on glide scoring for docking
energy combined with receptor strain and solvation terms derived from
prime energy. The IFD approach entails docking a flexible ligand into
a rigid receptor through multiple iterative steps and then refining
the protein active site to adopt the optimal conformation of the ligand.
The docking rigid box of 20 Å × 20 Å × 20 Å
was defined around the specific binding site of BTK, with the receptor
binding sites manually set to residue number 481 (PDB ID: <a href="https://www.rcsb.org/structure/6OMU" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6OMU</a>). The conformation
of the docked ligand was evaluated within an energy window of 2.5
kcal/mol to identify energetically favorable poses. During the Glide
ligand docking, van der Waals (vdW) scaling factors for both the receptor
and ligand were set to 0.50, allowing the generation of a maximum
of 20 conformations per ligand. The prime module was utilized to refine
residues near the ligand and optimize the side chains, explicitly
focusing on residues within 5.0 Å of the ligand binding pose.
Moreover, the Glide XP (extra precision) scoring function was applied
to retarget the ligand across all protein structures within a 30 kcal/mol
energy window relative to the lowest-energy structure. Consequently,
20 receptor–ligand complexes were rearranged based on the lowest
energy structure as a reference. The
optimal poses were selected after the molecular docking based on their
binding interactions and overall stability.</p></section><section id="sec4.7"><h3 class="pmc_sec_title">4.7. Metadynamics
Simulation</h3>
<p>Metadynamics
simulations were performed using the “Desmond module”
in the academic version of Maestro v9.0 (Schrödinger, LLC,
New York, USA). The highest-ranked docked poses were solvated with
SPC/E water molecules, neutralized with 0.15 M NaCl, and energy-minimized
using the OPLS_2005 (optimized liquid potential for simulations) force
field. A total of 10 independent metadynamics simulations, each running
for 10 ns, were carried out using a collective variable (CV) defined
as the RMSD of the ligand-heavy atoms relative to their initial configurations.
The RMSD calculation was initiated by aligning the protein using residues
within 3 Å of the ligand. The Cα atoms of these binding
site residues were aligned to their position in the initial frame
of the metadynamics trajectory. The RMD of the ligand’s heavy
atoms was subsequently computed relative to their original conformation.
The default parameters, including a bias height of 0.5 kcal/mol and
a temperature of 300 K, were applied, and the system was equilibrated
before starting the simulation. The final snapshot from a 0.5 ns unbiased
MD simulation was employed as the reference structure for the metadynamics
simulation phase.
</p></section><section id="sec4.8"><h3 class="pmc_sec_title">4.8. Molecular
Dynamics Simulation</h3>
<p>The
MD simulation model was conducted with the YASARA dynamics software
package (version 21.12.19, YASARA Biosciences GmbH, Vienna, Austria,
accessed on 20 June 2023) utilizing the
AMBER14 force field. Before initiating
the dynamic simulations, the hydrogen bond optimization was performed
on each protein–ligand complex to improve the accuracy of the
subsequent analysis. During the simulation, electrostatic interactions,
precisely long-range, were calculated using the particle mesh Ewald
(PME) method. Initially, energy minimization
was achieved through the simulated annealing method, employing the
steepest descent algorithms. Periodic boundary conditions were incorporated
to accurately compute van der Waals interactions and long-range Coulomb
forces with an 8 Å cutoff. The system was solvated using the
TIP3P explicit water model within a periodic cubic simulation cell.
To simulate a dynamic simulation environment, Na<sup>+</sup> and Cl<sup>–</sup> ions were added to neutralize the system and achieve
a 0.9% NaCl concentration, maintaining a pH of 7.4 and a water density
of 0.997 g/mL. During MD, no restraints were applied, particularly
during the equilibration phase in the NVT ensemble, where pressure
was left uncontrolled. Coordinates and energies were recorded every
100 ps, with the temperature constant at 298 K. The production phase
was performed in the NPA ensemble for 100 ns, maintaining a temperature
of 298 K and a pressure of 1 bar. The NPA algorithm is predictable
for its precision and sensitivity, maintaining a constant temperature
throughout the simulation. The entire MD simulation spanned 100 ns,
during which RMSD (root-mean-square deviation) values and the RMSF
(root-mean-square fluctuations), SASA (solvent accessible surface
area), and Rg (the radius of gyration) analysis were examined to evaluate
the structural stability and compactness of the protein–ligand
complexes. The RMSD profile of all MD structures indicated that variations
in RMSD values reached stability after long-range MD simulation, indicating
that the equilibrium structure has been established. Moreover, the
frame with the lowest energy from the MD simulation was selected for
further analysis using the YASARA-built macros in md_analysis.mcr
and md_analyzers.mcr. Moreover, the MD trajectory was leveraged for
a dynamic cross-correlation map (DCCM) analysis, providing insights
into the protein-integrated dynamic pattern.</p></section><section id="sec4.9"><h3 class="pmc_sec_title">4.9. Pharmacokinetic
Optimization of Lead Compounds</h3>
<p>The ADMETlab2.0 platform (<a href="https://admetmesh.scbdd.com/service/evaluation/index" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://admetmesh.scbdd.com/service/evaluation/index</a>) was utilized to investigate the pharmacokinetic analysis and druggability
of the lead compounds through the ADMET profile. ADMETlab 2.0 integrates
graph neural networks (GNNs) and consensus-based machine learning
algorithms trained on both low- and high-fidelity data sets, enabling
accurate predictions of key pharmacokinetic and toxicological properties.
This platform effectively computes a wide spectrum of ADMET-related
parameters, including absorption, distribution, metabolism, excretion,
and toxicity, while simultaneously evaluating drug-likeness, chemical
friendliness, and physicochemical features using multiple high-quality
predictive models. For ADMET evaluation, the canonical SMILIES (Simplified
Molecular Input Line Entry System) notation of lead compounds <strong>L2</strong> and <strong>L2-a</strong> was generated using the ChemBioDraw
Ultra software (version 15.0, accessed on 26 October 2023). Additionally,
the molecular structures of <strong>L2</strong> and <strong>L2-a</strong> were
analyzed to assess the features, including medicinal chemistry, ADMET
behavior, and physicochemical properties.</p>
<p>Moreover, to enhance
the robustness of the predictions obtained from the ADMETlab2.0, additional cross-validation was performed using
reputable online database web servers: pkCSM (<a href="http://structure.bioc.cam.ac.uk/pkcsm" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://structure.bioc.cam.ac.uk/pkcsm</a>) and SWISS ADME (<a href="http://www.swissadme.ch" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.swissadme.ch</a>). These
platforms provided complementary computational insights into the pharmacokinetic
characteristics of the compounds, such as lipophilicity, BBB, toxicity,
DILI (drug-induced liver injury), CYP450 metabolism, hERG inhibition,
water solubility, and drug-likeness. In addition, we rigorously applied
the widely accepted “Lipinski’s rule of five”
(RO5) to evaluate the oral bioavailability of the selected ligands. This rule indicates a compound’s drug-likeness
based on parameters like lipophilicity, molecular weight, hydrogen
bond donors and acceptors, and other vital factors critical for an
adequate drug development process. The triangulated evaluation across
multiple prediction systems improved the reliability and interpretability
of the pharmacokinetic analysis and ensured that only the most favorable
lead compounds advanced to subsequent optimization and docking validation
stages.</p></section><section id="sec4.10"><h3 class="pmc_sec_title">4.10. Density Functional Theory (DFT)</h3>
<p>In the current research work, all quantum chemical calculations were
carried out using DFT with the Gaussian 09 software package, while Gauss View 6.0.16 was employed as the graphical interface for visualization. The molecular
geometry was optimized using DFT at the B3LYP/6-311G* (d, p) basis
set. Noncovalent interactions (NCI) were
meticulously analyzed using advanced approaches, including reduced
density gradient (RDG), localized orbital locator (LOL), electron
localization function (EFL), and the independent gradient model (IGM),
all analyses conducted by the Multiwfn 3.8 program and VMD package. The molecular
electrostatic potential (MESP) surface was plotted to detect nucleophilic
and electrophilic regions contributing to hydrogen bond formation
in the lead compound. Additionally, time-dependent density functional
theory (TD-DFT) was applied to calculate the frontier molecular orbitals
(FMOs), including the lowest unoccupied molecular orbital (LUMO) and
the highest occupied molecular orbital (HOMO), to assess the molecular
chemical reactivity. Global chemical reactivity descriptors (GCRD),
such as the HOMO–LUMO gap, electron affinity, ionization potential,
and electronegativity, were also computed at the same theoretical
level. Furthermore, natural bond orbital (NBO) analysis was conducted
to examine electronic charge transfer (donor–acceptor interactions)
and determine bond orders in the molecule at the same functional level.
The density of state (DOS) calculations provided insights into the
distribution of the energy states within the molecules, while transition
density matrix (TDM) analysis shed light on electronic transitions
and charge redistribution during excitations. TDM and DOS calculations
were carried out using PyMOlyze 1.1 and
Multiwfn 3.8 software suites. Of space
before and after the caption as this allows for the rules, space above
and below the rules, and space above and below the figure to be inserted
upon editing.</p></section></section><section id="sec20"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="sifile1"><div class="media p"><div class="caption">
<a href="/articles/instance/12332581/bin/ao5c03682_si_001.zip" data-ga-action="click_feat_suppl" class="usa-link">ao5c03682_si_001.zip</a><sup> (1.4MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="sifile2"><div class="media p"><div class="caption">
<a href="/articles/instance/12332581/bin/ao5c03682_si_002.pdf" data-ga-action="click_feat_suppl" class="usa-link">ao5c03682_si_002.pdf</a><sup> (2.7MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>This research was supported by grants from the National
Key R&amp;D Program of China (2024YFF1207200), Beijing Natural Science
Foundation (L232017), National Natural Science Foundation of China
(82471380), Peking Union Medical College Deposition Research Fund
(grant 24 number: ZC201904188) and the Sichuan Province Science and
Technology Support Program (2024YFHZ0046). The authors thank Prof.
Xu-dong Qu for the support of Maestroversion9.0 in induced fit docking,
Discovery Studio v4.5 for protein–ligand interaction analysis,
and YASARA for MD simulation experiments. The authors also appreciate
the Analysis &amp; Testing Center, Beijing Institute of Technology.
The image for the graphical abstract was created using tools obtained
from Scientific Image and Illustration Software Bio Render (accessed
on January 10, 2024).</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Glossary</h2>
<h3>Abbreviations Used</h3>
<dl class="def-list" id="dl1">
<dt>MS</dt>
<dd><p>Multiple Sclerosis</p></dd>
<dt>BTK</dt>
<dd><p>Bruton Tyrosine Kinase</p></dd>
<dt>NF-κB</dt>
<dd><p>Nuclear factor
kappa-light-chain-enhancer of activated B cells</p></dd>
<dt>IL-6</dt>
<dd><p>Interleukin-6</p></dd>
<dt>IL-10</dt>
<dd><p>Interleukin-10</p></dd>
<dt>DILI</dt>
<dd><p>Drug-Induced-Liver-Injury</p></dd>
<dt>BBB</dt>
<dd><p>Blood-Brain-Barrier</p></dd>
<dt>IFD</dt>
<dd><p>Induced-fit-docking</p></dd>
<dt>MD</dt>
<dd><p>Molecular Dynamics</p></dd>
<dt>DFT</dt>
<dd><p>Density Functional
Theory</p></dd>
</dl></section><section id="notes4"><p>The data supporting
this work are available in this paper, and details are present in
the Supporting Information.</p></section><section id="notes1"><p>The Supporting Information is
available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsomega.5c03682?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acsomega.5c03682</a>.</p>
<ul id="silist1" class="list" style="list-style-type:none">
<li><p>Dock_Result_and_Metadynamic_Result
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_001.zip" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">ZIP</a>)</p></li>
<li><p>Figures
S1–S11, Tables S1–S9, ADMET result,
molecular dynamic result, and binding energy (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03682/suppl_file/ao5c03682_si_002.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p></li>
</ul></section><section id="notes2"><h2>#.</h2>
<p>A.S. and R.-C.L.
contributed equally to this work. J.-H.L. and M.-J.Y.
substantially controlled the research article. M.-J.Y. conducted the
conceptualization, supervision, and analysis, and made logistic support
for the research article and critical revision of the manuscript.
R.-C.L. is in charge of data generation, including docking, MD simulations.
S.-X.W. is in charge of synthetic route design. S.K. visualization
of data curation. A.S. was responsible for editing, drafting this
manuscript, and revising and executing this research article’s
main idea. All authors have read and agreed to the published version
of the manuscript.</p></section><section id="notes3"><p>The authors
declare no competing financial interest.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>Reich D., Lucchinetti C. F., Calabresi P. A.. Multiple Sclerosis. New Engl.
J. Med. 2018;378:169–180. doi: 10.1056/NEJMra1401483.</cite> [<a href="https://doi.org/10.1056/NEJMra1401483" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6942519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29320652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Engl.%20J.%20Med.&amp;title=Multiple%20Sclerosis&amp;author=D.%20Reich&amp;author=C.%20F.%20Lucchinetti&amp;author=P.%20A.%20Calabresi&amp;volume=378&amp;publication_year=2018&amp;pages=169-180&amp;pmid=29320652&amp;doi=10.1056/NEJMra1401483&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>Faissner S., Plemel J. R., Gold R., Yong V. W.. Progressive multiple
sclerosis: from pathophysiology to therapeutic strategies. Nat. Rev. Drug Discovery. 2019;18(12):905–922. doi: 10.1038/s41573-019-0035-2.</cite> [<a href="https://doi.org/10.1038/s41573-019-0035-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31399729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Drug%20Discovery&amp;title=Progressive%20multiple%20sclerosis:%20from%20pathophysiology%20to%20therapeutic%20strategies&amp;author=S.%20Faissner&amp;author=J.%20R.%20Plemel&amp;author=R.%20Gold&amp;author=V.%20W.%20Yong&amp;volume=18&amp;issue=12&amp;publication_year=2019&amp;pages=905-922&amp;pmid=31399729&amp;doi=10.1038/s41573-019-0035-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Dendrou C. A., Fugger L., Friese M. A.. Immunopathology
of multiple sclerosis. Nat. Rev. Immunol. 2015;15(9):545–558. doi: 10.1038/nri3871.</cite> [<a href="https://doi.org/10.1038/nri3871" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26250739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Immunol.&amp;title=Immunopathology%20of%20multiple%20sclerosis&amp;author=C.%20A.%20Dendrou&amp;author=L.%20Fugger&amp;author=M.%20A.%20Friese&amp;volume=15&amp;issue=9&amp;publication_year=2015&amp;pages=545-558&amp;pmid=26250739&amp;doi=10.1038/nri3871&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>Baecher-Allan C., Kaskow B. J., Weiner H. L.. Multiple sclerosis: mechanisms and
immunotherapy. Neuron. 2018;97(4):742–768. doi: 10.1016/j.neuron.2018.01.021.</cite> [<a href="https://doi.org/10.1016/j.neuron.2018.01.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29470968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Multiple%20sclerosis:%20mechanisms%20and%20immunotherapy&amp;author=C.%20Baecher-Allan&amp;author=B.%20J.%20Kaskow&amp;author=H.%20L.%20Weiner&amp;volume=97&amp;issue=4&amp;publication_year=2018&amp;pages=742-768&amp;pmid=29470968&amp;doi=10.1016/j.neuron.2018.01.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Ruiz F., Vigne S., Pot C.. Resolution
of inflammation during
multiple sclerosis. Semin. Immunopathol. 2019;41(6):711–726. doi: 10.1007/s00281-019-00765-0.</cite> [<a href="https://doi.org/10.1007/s00281-019-00765-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6881249/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31732775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Semin.%20Immunopathol.&amp;title=Resolution%20of%20inflammation%20during%20multiple%20sclerosis&amp;author=F.%20Ruiz&amp;author=S.%20Vigne&amp;author=C.%20Pot&amp;volume=41&amp;issue=6&amp;publication_year=2019&amp;pages=711-726&amp;pmid=31732775&amp;doi=10.1007/s00281-019-00765-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>DiSabato D. J., Quan N., Godbout J. P.. Neuroinflammation: The devil is in
the details. J. Neurochem. 2016;139:136–153. doi: 10.1111/jnc.13607.</cite> [<a href="https://doi.org/10.1111/jnc.13607" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5025335/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26990767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Neurochem.&amp;title=Neuroinflammation:%20The%20devil%20is%20in%20the%20details&amp;author=D.%20J.%20DiSabato&amp;author=N.%20Quan&amp;author=J.%20P.%20Godbout&amp;volume=139&amp;publication_year=2016&amp;pages=136-153&amp;pmid=26990767&amp;doi=10.1111/jnc.13607&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Minagar A., Maghzi A. H., McGee J. C., Alexander J. S.. Emerging
roles of endothelial cells in multiple sclerosis pathophysiology and
therapy. Neurol Res. 2012;34(8):738–745. doi: 10.1179/1743132812Y.0000000072.</cite> [<a href="https://doi.org/10.1179/1743132812Y.0000000072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22828184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neurol%20Res.&amp;title=Emerging%20roles%20of%20endothelial%20cells%20in%20multiple%20sclerosis%20pathophysiology%20and%20therapy&amp;author=A.%20Minagar&amp;author=A.%20H.%20Maghzi&amp;author=J.%20C.%20McGee&amp;author=J.%20S.%20Alexander&amp;volume=34&amp;issue=8&amp;publication_year=2012&amp;pages=738-745&amp;pmid=22828184&amp;doi=10.1179/1743132812Y.0000000072&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Lin J., Zhou J., Xu Y.. Potential
drug targets for multiple
sclerosis identified through Mendelian randomization analysis. Brain. 2023;146(8):3364–3372. doi: 10.1093/brain/awad070.</cite> [<a href="https://doi.org/10.1093/brain/awad070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10393411/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36864689/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=Potential%20drug%20targets%20for%20multiple%20sclerosis%20identified%20through%20Mendelian%20randomization%20analysis&amp;author=J.%20Lin&amp;author=J.%20Zhou&amp;author=Y.%20Xu&amp;volume=146&amp;issue=8&amp;publication_year=2023&amp;pages=3364-3372&amp;pmid=36864689&amp;doi=10.1093/brain/awad070&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Duddy M. E., Alter A., Bar-Or A.. Distinct profiles
of human B cell
effector cytokines: A role in immune regulation? J. Immunol. 2004;172(6):3422–3427. doi: 10.4049/jimmunol.172.6.3422.</cite> [<a href="https://doi.org/10.4049/jimmunol.172.6.3422" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15004141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.&amp;title=Distinct%20profiles%20of%20human%20B%20cell%20effector%20cytokines:%20A%20role%20in%20immune%20regulation?&amp;author=M.%20E.%20Duddy&amp;author=A.%20Alter&amp;author=A.%20Bar-Or&amp;volume=172&amp;issue=6&amp;publication_year=2004&amp;pages=3422-3427&amp;pmid=15004141&amp;doi=10.4049/jimmunol.172.6.3422&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Bar-Or A., Fawaz L., Fan B., Darlington P. J., Rieger A., Ghorayeb C., Calabresi P. A., Waubant E., Hauser S. L., Zhang J.. et al. Abnormal
B-cell cytokine responses a trigger of T-cell–mediated disease
in MS? Ann. Neurol. 2010;67(4):452–461. doi: 10.1002/ana.21939.</cite> [<a href="https://doi.org/10.1002/ana.21939" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20437580/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann.%20Neurol.&amp;title=Abnormal%20B-cell%20cytokine%20responses%20a%20trigger%20of%20T-cell%E2%80%93mediated%20disease%20in%20MS?&amp;author=A.%20Bar-Or&amp;author=L.%20Fawaz&amp;author=B.%20Fan&amp;author=P.%20J.%20Darlington&amp;author=A.%20Rieger&amp;volume=67&amp;issue=4&amp;publication_year=2010&amp;pages=452-461&amp;pmid=20437580&amp;doi=10.1002/ana.21939&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Van
Langelaar J., Rijvers L., Smolders J., Van Luijn M. M.. B and T
cells driving multiple sclerosis: identity, mechanisms and potential
triggers. Front. Immunol. 2020;11:760. doi: 10.3389/fimmu.2020.00760.</cite> [<a href="https://doi.org/10.3389/fimmu.2020.00760" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7225320/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32457742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Immunol.&amp;title=B%20and%20T%20cells%20driving%20multiple%20sclerosis:%20identity,%20mechanisms%20and%20potential%20triggers&amp;author=J.%20Van%20Langelaar&amp;author=L.%20Rijvers&amp;author=J.%20Smolders&amp;author=M.%20M.%20Van%20Luijn&amp;volume=11&amp;publication_year=2020&amp;pages=760&amp;pmid=32457742&amp;doi=10.3389/fimmu.2020.00760&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Häusser-Kinzel S., Weber M. S.. The role of B cells and antibodies in multiple sclerosis,
neuromyelitis optica, and related disorders. Front. Immunol. 2019;10:201. doi: 10.3389/fimmu.2019.00201.</cite> [<a href="https://doi.org/10.3389/fimmu.2019.00201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6375838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30800132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Immunol.&amp;title=The%20role%20of%20B%20cells%20and%20antibodies%20in%20multiple%20sclerosis,%20neuromyelitis%20optica,%20and%20related%20disorders&amp;author=S.%20H%C3%A4usser-Kinzel&amp;author=M.%20S.%20Weber&amp;volume=10&amp;publication_year=2019&amp;pages=201&amp;pmid=30800132&amp;doi=10.3389/fimmu.2019.00201&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Magliozzi R., Howell O., Vora A., Serafini B., Nicholas R., Puopolo M., Reynolds R., Aloisi F.. Meningeal B-cell follicles
in secondary progressive multiple sclerosis associate with early onset
of disease and severe cortical pathology. Brain. 2006;130(4):1089–1104. doi: 10.1093/brain/awm038.</cite> [<a href="https://doi.org/10.1093/brain/awm038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17438020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Brain&amp;title=Meningeal%20B-cell%20follicles%20in%20secondary%20progressive%20multiple%20sclerosis%20associate%20with%20early%20onset%20of%20disease%20and%20severe%20cortical%20pathology&amp;author=R.%20Magliozzi&amp;author=O.%20Howell&amp;author=A.%20Vora&amp;author=B.%20Serafini&amp;author=R.%20Nicholas&amp;volume=130&amp;issue=4&amp;publication_year=2006&amp;pages=1089-1104&amp;pmid=17438020&amp;doi=10.1093/brain/awm038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Kanatas P., Stouras I., Stefanis L., Stathopoulos P.. B-Cell-directed
therapies: A new era in multiple sclerosis treatment. Can. J. Neurol. Sci. 2023;50(3):355–364. doi: 10.1017/cjn.2022.60.</cite> [<a href="https://doi.org/10.1017/cjn.2022.60" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35570581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Can.%20J.%20Neurol.%20Sci.&amp;title=B-Cell-directed%20therapies:%20A%20new%20era%20in%20multiple%20sclerosis%20treatment&amp;author=P.%20Kanatas&amp;author=I.%20Stouras&amp;author=L.%20Stefanis&amp;author=P.%20Stathopoulos&amp;volume=50&amp;issue=3&amp;publication_year=2023&amp;pages=355-364&amp;pmid=35570581&amp;doi=10.1017/cjn.2022.60&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>Torke S., Pretzsch R., Häusler D., Haselmayer P., Grenningloh R., Boschert U., Brück W., Weber M. S.. Inhibition of Bruton’s tyrosine kinase interferes
with pathogenic B-cell development in inflammatory CNS demyelinating
disease. Acta Neuropathol. 2020;140:535–548. doi: 10.1007/s00401-020-02204-z.</cite> [<a href="https://doi.org/10.1007/s00401-020-02204-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7498502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32761407/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Neuropathol.&amp;title=Inhibition%20of%20Bruton%E2%80%99s%20tyrosine%20kinase%20interferes%20with%20pathogenic%20B-cell%20development%20in%20inflammatory%20CNS%20demyelinating%20disease&amp;author=S.%20Torke&amp;author=R.%20Pretzsch&amp;author=D.%20H%C3%A4usler&amp;author=P.%20Haselmayer&amp;author=R.%20Grenningloh&amp;volume=140&amp;publication_year=2020&amp;pages=535-548&amp;pmid=32761407&amp;doi=10.1007/s00401-020-02204-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Sharma S., Orlowski G., Song W.. Btk regulates
B cell receptor-mediated
antigen processing and presentation by controlling actin cytoskeleton
dynamics in B cells. J. Immunol. 2009;182(1):329–339. doi: 10.4049/jimmunol.182.1.329.</cite> [<a href="https://doi.org/10.4049/jimmunol.182.1.329" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2855895/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19109164/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.&amp;title=Btk%20regulates%20B%20cell%20receptor-mediated%20antigen%20processing%20and%20presentation%20by%20controlling%20actin%20cytoskeleton%20dynamics%20in%20B%20cells&amp;author=S.%20Sharma&amp;author=G.%20Orlowski&amp;author=W.%20Song&amp;volume=182&amp;issue=1&amp;publication_year=2009&amp;pages=329-339&amp;pmid=19109164&amp;doi=10.4049/jimmunol.182.1.329&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Lee K. G., Xu S., Wong E. T., Tergaonkar V., Lam K. P.. Bruton’s
tyrosine kinase separately regulates NFκB p65RelA activation
and cytokine interleukin (IL)-10/IL-12 production in TLR9-stimulated
B cells. J. Biol. Chem. 2008;283(17):11189–11198. doi: 10.1074/jbc.M708516200.</cite> [<a href="https://doi.org/10.1074/jbc.M708516200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18276597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Bruton%E2%80%99s%20tyrosine%20kinase%20separately%20regulates%20NF%CE%BAB%20p65RelA%20activation%20and%20cytokine%20interleukin%20(IL)-10/IL-12%20production%20in%20TLR9-stimulated%20B%20cells&amp;author=K.%20G.%20Lee&amp;author=S.%20Xu&amp;author=E.%20T.%20Wong&amp;author=V.%20Tergaonkar&amp;author=K.%20P.%20Lam&amp;volume=283&amp;issue=17&amp;publication_year=2008&amp;pages=11189-11198&amp;pmid=18276597&amp;doi=10.1074/jbc.M708516200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>Spaargaren M., Beuling E. A., Rurup M. L., Meijer H. P., Klok M. D., Middendorp S., Hendriks R. W., Pals S. T.. The B cell antigen
receptor controls integrin activity through Btk and PLCγ2. J. Biol. Chem. 2003;198(10):1539–1550. doi: 10.1084/jem.20011866.</cite> [<a href="https://doi.org/10.1084/jem.20011866" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2194118/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14610042/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=The%20B%20cell%20antigen%20receptor%20controls%20integrin%20activity%20through%20Btk%20and%20PLC%CE%B32&amp;author=M.%20Spaargaren&amp;author=E.%20A.%20Beuling&amp;author=M.%20L.%20Rurup&amp;author=H.%20P.%20Meijer&amp;author=M.%20D.%20Klok&amp;volume=198&amp;issue=10&amp;publication_year=2003&amp;pages=1539-1550&amp;pmid=14610042&amp;doi=10.1084/jem.20011866&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Lin D. Y., Andreotti A. H.. Structure
of BTK kinase domain with the second-generation
inhibitors acalabrutinib and tirabrutinib. PLoS
One. 2023;18(8):e0290872.  doi: 10.1371/journal.pone.0290872.</cite> [<a href="https://doi.org/10.1371/journal.pone.0290872" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10470882/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37651403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Structure%20of%20BTK%20kinase%20domain%20with%20the%20second-generation%20inhibitors%20acalabrutinib%20and%20tirabrutinib&amp;author=D.%20Y.%20Lin&amp;author=A.%20H.%20Andreotti&amp;volume=18&amp;issue=8&amp;publication_year=2023&amp;pages=e0290872&amp;pmid=37651403&amp;doi=10.1371/journal.pone.0290872&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>Akinleye A., Chen Y., Mukhi N., Song Y., Liu D.. Ibrutinib
and novel BTK inhibitors in clinical development. J. Hematol. Oncol. 2013;6:59. doi: 10.1186/1756-8722-6-59.</cite> [<a href="https://doi.org/10.1186/1756-8722-6-59" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3751776/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23958373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Hematol.%20Oncol.&amp;title=Ibrutinib%20and%20novel%20BTK%20inhibitors%20in%20clinical%20development&amp;author=A.%20Akinleye&amp;author=Y.%20Chen&amp;author=N.%20Mukhi&amp;author=Y.%20Song&amp;author=D.%20Liu&amp;volume=6&amp;publication_year=2013&amp;pages=59&amp;pmid=23958373&amp;doi=10.1186/1756-8722-6-59&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>Mohamed A. J., Nore B. F., Christensson B., Smith C. I.. Signalling of Bruton’s
tyrosine kinase, Btk. Scand. J. Immunol. 1999;49(2):113–118. doi: 10.1046/j.1365-3083.1999.00504.x.</cite> [<a href="https://doi.org/10.1046/j.1365-3083.1999.00504.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10075013/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scand.%20J.%20Immunol.&amp;title=Signalling%20of%20Bruton%E2%80%99s%20tyrosine%20kinase,%20Btk&amp;author=A.%20J.%20Mohamed&amp;author=B.%20F.%20Nore&amp;author=B.%20Christensson&amp;author=C.%20I.%20Smith&amp;volume=49&amp;issue=2&amp;publication_year=1999&amp;pages=113-118&amp;pmid=10075013&amp;doi=10.1046/j.1365-3083.1999.00504.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Park H., Wahl M. I., Afar D. E., Turck C. W., Rawlings D. J., Tam C., Scharenberg A. M., Kinet J.-P., Witte O. N.. Regulation
of Btk function by a major autophosphorylation site within the SH3
domain. Immunity. 1996;4(5):515–525. doi: 10.1016/S1074-7613(00)80417-3.</cite> [<a href="https://doi.org/10.1016/S1074-7613(00)80417-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8630736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Regulation%20of%20Btk%20function%20by%20a%20major%20autophosphorylation%20site%20within%20the%20SH3%20domain&amp;author=H.%20Park&amp;author=M.%20I.%20Wahl&amp;author=D.%20E.%20Afar&amp;author=C.%20W.%20Turck&amp;author=D.%20J.%20Rawlings&amp;volume=4&amp;issue=5&amp;publication_year=1996&amp;pages=515-525&amp;pmid=8630736&amp;doi=10.1016/S1074-7613(00)80417-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Pal
Singh S., Dammeijer F., Hendriks R. W.. Role of Bruton’s
tyrosine kinase in B cells and malignancies. Mol. Cancer. 2018;17(1):57. doi: 10.1186/s12943-018-0779-z.</cite> [<a href="https://doi.org/10.1186/s12943-018-0779-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5817726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29455639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cancer&amp;title=Role%20of%20Bruton%E2%80%99s%20tyrosine%20kinase%20in%20B%20cells%20and%20malignancies&amp;author=S.%20Pal%20Singh&amp;author=F.%20Dammeijer&amp;author=R.%20W.%20Hendriks&amp;volume=17&amp;issue=1&amp;publication_year=2018&amp;pages=57&amp;pmid=29455639&amp;doi=10.1186/s12943-018-0779-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Hamasy A., Wang Q., Blomberg K., Mohammad D., Yu L., Vihinen M., Berglöf A., Smith C.. Substitution scanning
identifies a novel, catalytically active ibrutinib-resistant BTK cysteine
481 to threonine (C481T) variant. Leukemia. 2017;31(1):177–185. doi: 10.1038/leu.2016.153.</cite> [<a href="https://doi.org/10.1038/leu.2016.153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5220130/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27282255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Leukemia&amp;title=Substitution%20scanning%20identifies%20a%20novel,%20catalytically%20active%20ibrutinib-resistant%20BTK%20cysteine%20481%20to%20threonine%20(C481T)%20variant&amp;author=A.%20Hamasy&amp;author=Q.%20Wang&amp;author=K.%20Blomberg&amp;author=D.%20Mohammad&amp;author=L.%20Yu&amp;volume=31&amp;issue=1&amp;publication_year=2017&amp;pages=177-185&amp;pmid=27282255&amp;doi=10.1038/leu.2016.153&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Bond D. A., Woyach J. A.. Targeting BTK in CLL: beyond ibrutinib. Curr. Hematol. Malig. Rep. 2019;14:197–205. doi: 10.1007/s11899-019-00512-0.</cite> [<a href="https://doi.org/10.1007/s11899-019-00512-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31028669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Hematol.%20Malig.%20Rep.&amp;title=Targeting%20BTK%20in%20CLL:%20beyond%20ibrutinib&amp;author=D.%20A.%20Bond&amp;author=J.%20A.%20Woyach&amp;volume=14&amp;publication_year=2019&amp;pages=197-205&amp;pmid=31028669&amp;doi=10.1007/s11899-019-00512-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Alle T., Joyasawal S., Oukoloff K., Long K., Owyang Z., Francisco K. R., Cahard D., Huryn D. M., Ballatore C.. Structure-property
relationships of fluorinated carboxylic acid bioisosteres. Bioorg. Med. Chem. Lett. 2023;91:129363. doi: 10.1016/j.bmcl.2023.129363.</cite> [<a href="https://doi.org/10.1016/j.bmcl.2023.129363" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11003450/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37295616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Med.%20Chem.%20Lett.&amp;title=Structure-property%20relationships%20of%20fluorinated%20carboxylic%20acid%20bioisosteres&amp;author=T.%20Alle&amp;author=S.%20Joyasawal&amp;author=K.%20Oukoloff&amp;author=K.%20Long&amp;author=Z.%20Owyang&amp;volume=91&amp;publication_year=2023&amp;pages=129363&amp;pmid=37295616&amp;doi=10.1016/j.bmcl.2023.129363&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Henary E., Casa S., Dost T. L., Sloop J. C., Henary M.. The Role of
Small Molecules Containing Fluorine Atoms in Medicine and Imaging
Applications. Pharmaceuticals. 2024;17(3):281. doi: 10.3390/ph17030281.</cite> [<a href="https://doi.org/10.3390/ph17030281" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10975950/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38543068/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmaceuticals&amp;title=The%20Role%20of%20Small%20Molecules%20Containing%20Fluorine%20Atoms%20in%20Medicine%20and%20Imaging%20Applications&amp;author=E.%20Henary&amp;author=S.%20Casa&amp;author=T.%20L.%20Dost&amp;author=J.%20C.%20Sloop&amp;author=M.%20Henary&amp;volume=17&amp;issue=3&amp;publication_year=2024&amp;pages=281&amp;pmid=38543068&amp;doi=10.3390/ph17030281&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Alogheli H., Olanders G., Schaal W., Brandt P., Karlén A.. Docking of
macrocycles: Comparing rigid and flexible docking in glide. J. Chem. Inf. Model. 2017;57(2):190–202. doi: 10.1021/acs.jcim.6b00443.</cite> [<a href="https://doi.org/10.1021/acs.jcim.6b00443" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28079375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Inf.%20Model&amp;title=Docking%20of%20macrocycles:%20Comparing%20rigid%20and%20flexible%20docking%20in%20glide&amp;author=H.%20Alogheli&amp;author=G.%20Olanders&amp;author=W.%20Schaal&amp;author=P.%20Brandt&amp;author=A.%20Karl%C3%A9n&amp;volume=57&amp;issue=2&amp;publication_year=2017&amp;pages=190-202&amp;pmid=28079375&amp;doi=10.1021/acs.jcim.6b00443&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>Mahmoudi S., Dehkordi M. M., Asgarshamsi M. H.. Density
functional theory studies
of the antioxidantsa review. J. Mol.
Model. 2021;27(9):271. doi: 10.1007/s00894-021-04891-1.</cite> [<a href="https://doi.org/10.1007/s00894-021-04891-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34463834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Model.&amp;title=Density%20functional%20theory%20studies%20of%20the%20antioxidants%EE%97%B8a%20review&amp;author=S.%20Mahmoudi&amp;author=M.%20M.%20Dehkordi&amp;author=M.%20H.%20Asgarshamsi&amp;volume=27&amp;issue=9&amp;publication_year=2021&amp;pages=271&amp;pmid=34463834&amp;doi=10.1007/s00894-021-04891-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Alotaibi M. T.. Noncovalent
interaction stabilizes the 2, 4-Dinitrophenylhydrazone Derivatives
over g-C3N4 surface to enhance optical properties: synthesis, characterization,
and DFT investigation. J. Mol. Struct. 2020;1214:128192. doi: 10.1016/j.molstruc.2020.128192.</cite> [<a href="https://doi.org/10.1016/j.molstruc.2020.128192" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Struct.&amp;title=Noncovalent%20interaction%20stabilizes%20the%202,%204-Dinitrophenylhydrazone%20Derivatives%20over%20g-C3N4%20surface%20to%20enhance%20optical%20properties:%20synthesis,%20characterization,%20and%20DFT%20investigation&amp;author=M.%20T.%20Alotaibi&amp;volume=1214&amp;publication_year=2020&amp;pages=128192&amp;doi=10.1016/j.molstruc.2020.128192&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>Madhavi
Sastry G., Adzhigirey M., Day T., Annabhimoju R., Sherman W.. Protein and ligand preparation: parameters, protocols,
and influence on virtual screening enrichments. J. comput.-Aided Mol. Des. 2013;27:221–234. doi: 10.1007/s10822-013-9644-8.</cite> [<a href="https://doi.org/10.1007/s10822-013-9644-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23579614/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20comput.-Aided%20Mol.%20Des.&amp;title=Protein%20and%20ligand%20preparation:%20parameters,%20protocols,%20and%20influence%20on%20virtual%20screening%20enrichments&amp;author=G.%20Madhavi%20Sastry&amp;author=M.%20Adzhigirey&amp;author=T.%20Day&amp;author=R.%20Annabhimoju&amp;author=W.%20Sherman&amp;volume=27&amp;publication_year=2013&amp;pages=221-234&amp;pmid=23579614&amp;doi=10.1007/s10822-013-9644-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>Harder E., Damm W., Maple J., Wu C., Reboul M., Xiang J. Y., Wang L., Lupyan D., Dahlgren M. K., Knight J. L.. et al. OPLS3: A force field
providing broad coverage
of drug-like small molecules and proteins. J.
Chem. Theory Comput. 2016;12(1):281–296. doi: 10.1021/acs.jctc.5b00864.</cite> [<a href="https://doi.org/10.1021/acs.jctc.5b00864" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26584231/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Theory%20Comput.&amp;title=OPLS3:%20A%20force%20field%20providing%20broad%20coverage%20of%20drug-like%20small%20molecules%20and%20proteins&amp;author=E.%20Harder&amp;author=W.%20Damm&amp;author=J.%20Maple&amp;author=C.%20Wu&amp;author=M.%20Reboul&amp;volume=12&amp;issue=1&amp;publication_year=2016&amp;pages=281-296&amp;pmid=26584231&amp;doi=10.1021/acs.jctc.5b00864&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>Xu Y., Wang S., Hu Q., Gao S., Ma X., Zhang W., Shen Y., Chen F., Lai L., Pei J.. CavityPlus: a web server for protein cavity detection with pharmacophore
modelling, allosteric site identification and covalent ligand binding
ability prediction. Nucleic Acids Res. 2018;46(W1):W374–W379. doi: 10.1093/nar/gky380.</cite> [<a href="https://doi.org/10.1093/nar/gky380" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6031032/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29750256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=CavityPlus:%20a%20web%20server%20for%20protein%20cavity%20detection%20with%20pharmacophore%20modelling,%20allosteric%20site%20identification%20and%20covalent%20ligand%20binding%20ability%20prediction&amp;author=Y.%20Xu&amp;author=S.%20Wang&amp;author=Q.%20Hu&amp;author=S.%20Gao&amp;author=X.%20Ma&amp;volume=46&amp;issue=W1&amp;publication_year=2018&amp;pages=W374-W379&amp;pmid=29750256&amp;doi=10.1093/nar/gky380&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34">
<cite>Caldwell R.
D., Qiu H., Askew B. C., Bender A. T., Brugger N., Camps M., Dhanabal M., Dutt V., Eichhorn T., Gardberg A. S.. et al. Discovery of evobrutinib: an oral, potent, and highly selective,
covalent Bruton’s tyrosine kinase (BTK) inhibitor for the treatment
of immunological diseases. J. Med. Chem. 2019;62(17):7643–7655. doi: 10.1021/acs.jmedchem.9b00794.</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.9b00794" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31368705/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.&amp;title=Discovery%20of%20evobrutinib:%20an%20oral,%20potent,%20and%20highly%20selective,%20covalent%20Bruton%E2%80%99s%20tyrosine%20kinase%20(BTK)%20inhibitor%20for%20the%20treatment%20of%20immunological%20diseases&amp;author=R.%20D.%20Caldwell&amp;author=H.%20Qiu&amp;author=B.%20C.%20Askew&amp;author=A.%20T.%20Bender&amp;author=N.%20Brugger&amp;volume=62&amp;issue=17&amp;publication_year=2019&amp;pages=7643-7655&amp;pmid=31368705&amp;doi=10.1021/acs.jmedchem.9b00794&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref35">
<cite>Lou Y., Owens T. D., Kuglstatter A., Kondru R. K., Goldstein D. M.. Bruton’s
tyrosine kinase inhibitors: approaches to potent and selective inhibition,
preclinical and clinical evaluation for inflammatory diseases and
B cell malignancies. J. Med. Chem. 2012;55(10):4539–4550. doi: 10.1021/jm300035p.</cite> [<a href="https://doi.org/10.1021/jm300035p" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22394077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.&amp;title=Bruton%E2%80%99s%20tyrosine%20kinase%20inhibitors:%20approaches%20to%20potent%20and%20selective%20inhibition,%20preclinical%20and%20clinical%20evaluation%20for%20inflammatory%20diseases%20and%20B%20cell%20malignancies&amp;author=Y.%20Lou&amp;author=T.%20D.%20Owens&amp;author=A.%20Kuglstatter&amp;author=R.%20K.%20Kondru&amp;author=D.%20M.%20Goldstein&amp;volume=55&amp;issue=10&amp;publication_year=2012&amp;pages=4539-4550&amp;pmid=22394077&amp;doi=10.1021/jm300035p&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>De
Lucca G. V., Shi Q., Liu Q., Batt D. G., Beaudoin Bertrand M., Rampulla R., Mathur A., Discenza L., D’Arienzo C., Dai J.. Small Molecule Reversible Inhibitors
of Bruton’s Tyrosine Kinase (BTK): Structure–Activity
Relationships Leading to the Identification of 7-(2-Hydroxypropan-2-yl)-4-[2-methyl-3-(4-oxo-3,
4-dihydroquinazolin-3-yl) phenyl]-9 H-carbazole-1-carboxamide (BMS-935177) J. Med. Chem. 2016;59(17):7915–7935. doi: 10.1021/acs.jmedchem.6b00722.</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.6b00722" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27531604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.&amp;title=Small%20Molecule%20Reversible%20Inhibitors%20of%20Bruton%E2%80%99s%20Tyrosine%20Kinase%20(BTK):%20Structure%E2%80%93Activity%20Relationships%20Leading%20to%20the%20Identification%20of%207-(2-Hydroxypropan-2-yl)-4-%5B2-methyl-3-(4-oxo-3,%204-dihydroquinazolin-3-yl)%20phenyl%5D-9%20H-carbazole-1-carboxamide%20(BMS-935177)&amp;author=G.%20V.%20De%20Lucca&amp;author=Q.%20Shi&amp;author=Q.%20Liu&amp;author=D.%20G.%20Batt&amp;author=M.%20Beaudoin%20Bertrand&amp;volume=59&amp;issue=17&amp;publication_year=2016&amp;pages=7915-7935&amp;pmid=27531604&amp;doi=10.1021/acs.jmedchem.6b00722&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>Friesner R. A., Banks J. L., Murphy R. B., Halgren T. A., Klicic J. J., Mainz D. T., Repasky M. P., Knoll E. H., Shelley M., Perry J. K.. et al. Glide: A new approach
for rapid, accurate docking
and scoring. 1. Method and assessment of docking accuracy. J. Med. Chem. 2004;47(7):1739–1749. doi: 10.1021/jm0306430.</cite> [<a href="https://doi.org/10.1021/jm0306430" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15027865/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.&amp;title=Glide:%20A%20new%20approach%20for%20rapid,%20accurate%20docking%20and%20scoring.%201.%20Method%20and%20assessment%20of%20docking%20accuracy&amp;author=R.%20A.%20Friesner&amp;author=J.%20L.%20Banks&amp;author=R.%20B.%20Murphy&amp;author=T.%20A.%20Halgren&amp;author=J.%20J.%20Klicic&amp;volume=47&amp;issue=7&amp;publication_year=2004&amp;pages=1739-1749&amp;pmid=15027865&amp;doi=10.1021/jm0306430&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Morris G. M., Goodsell D. S., Halliday R. S., Huey R., Hart W. E., Belew R. K., Olson A. J.. Automated docking using a Lamarckian
genetic algorithm and an empirical binding free energy function. J. Comput. Chem. 1998;19(14):1639–1662. doi: 10.1002/(SICI)1096-987X(19981115)19:14&amp;#x0003c;1639::AID-JCC10&amp;#x0003e;3.0.CO;2-B.</cite> [<a href="https://doi.org/10.1002/(SICI)1096-987X(19981115)19:14&amp;#x0003c;1639::AID-JCC10&amp;#x0003e;3.0.CO;2-B" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&amp;title=Automated%20docking%20using%20a%20Lamarckian%20genetic%20algorithm%20and%20an%20empirical%20binding%20free%20energy%20function&amp;author=G.%20M.%20Morris&amp;author=D.%20S.%20Goodsell&amp;author=R.%20S.%20Halliday&amp;author=R.%20Huey&amp;author=W.%20E.%20Hart&amp;volume=19&amp;issue=14&amp;publication_year=1998&amp;pages=1639-1662&amp;doi=10.1002/(SICI)1096-987X(19981115)19:14&amp;#x0003c;1639::AID-JCC10&amp;#x0003e;3.0.CO;2-B&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Land H., Humble M. S.. YASARA: a tool to
obtain structural guidance in biocatalytic
investigations. Methods Mol. Biol. 2018;1685:43–67. doi: 10.1007/978-1-4939-7366-8_4.</cite> [<a href="https://doi.org/10.1007/978-1-4939-7366-8_4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29086303/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol.&amp;title=YASARA:%20a%20tool%20to%20obtain%20structural%20guidance%20in%20biocatalytic%20investigations&amp;author=H.%20Land&amp;author=M.%20S.%20Humble&amp;volume=1685&amp;publication_year=2018&amp;pages=43-67&amp;pmid=29086303&amp;doi=10.1007/978-1-4939-7366-8_4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Andrade R. J., Chalasani N., Björnsson E. S., Suzuki A., Kullak-Ublick G. A., Watkins P. B., Devarbhavi H., Merz M., Lucena M. I., Kaplowitz N.. et al. Drug-induced liver injury. Nat. Rev. Dis. Primers. 2019;5(1):58. doi: 10.1038/s41572-019-0105-0.</cite> [<a href="https://doi.org/10.1038/s41572-019-0105-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31439850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Dis.%20Primers&amp;title=Drug-induced%20liver%20injury&amp;author=R.%20J.%20Andrade&amp;author=N.%20Chalasani&amp;author=E.%20S.%20Bj%C3%B6rnsson&amp;author=A.%20Suzuki&amp;author=G.%20A.%20Kullak-Ublick&amp;volume=5&amp;issue=1&amp;publication_year=2019&amp;pages=58&amp;pmid=31439850&amp;doi=10.1038/s41572-019-0105-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Studio. Discovery studio. Accelrys
[2.1] 2008. </cite> [<a href="https://scholar.google.com/scholar_lookup?title=Discovery%20studio.%20Accelrys%20%5B2.1%5D&amp;publication_year=2008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Seeliger D., de Groot B. L.. Ligand docking and binding site analysis with PyMOL
and Autodock/Vina. J. Comput.-Aided Mol. Des. 2010;24(5):417–422. doi: 10.1007/s10822-010-9352-6.</cite> [<a href="https://doi.org/10.1007/s10822-010-9352-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2881210/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20401516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.-Aided%20Mol.%20Des.&amp;title=Ligand%20docking%20and%20binding%20site%20analysis%20with%20PyMOL%20and%20Autodock/Vina&amp;author=D.%20Seeliger&amp;author=B.%20L.%20de%20Groot&amp;volume=24&amp;issue=5&amp;publication_year=2010&amp;pages=417-422&amp;pmid=20401516&amp;doi=10.1007/s10822-010-9352-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Vanichtanankul J., Taweechai S., Uttamapinant C., Chitnumsub P., Vilaivan T., Yuthavong Y., Kamchonwongpaisan S.. Combined spatial
limitation around residues 16 and 108 of Plasmodium falciparum dihydrofolate
reductase explains resistance to cycloguanil. Antimicrob. Agents Chemother. 2012;56(7):3928–3935. doi: 10.1128/AAC.00301-12.</cite> [<a href="https://doi.org/10.1128/AAC.00301-12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3393428/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22526319/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&amp;title=Combined%20spatial%20limitation%20around%20residues%2016%20and%20108%20of%20Plasmodium%20falciparum%20dihydrofolate%20reductase%20explains%20resistance%20to%20cycloguanil&amp;author=J.%20Vanichtanankul&amp;author=S.%20Taweechai&amp;author=C.%20Uttamapinant&amp;author=P.%20Chitnumsub&amp;author=T.%20Vilaivan&amp;volume=56&amp;issue=7&amp;publication_year=2012&amp;pages=3928-3935&amp;pmid=22526319&amp;doi=10.1128/AAC.00301-12&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref44">
<cite>Neumann A., Engel V., Mahardhika A. B., Schoeder C. T., Namasivayam V., Kieć-Kononowicz K., Müller C. E.. Computational
investigations on the binding mode of ligands for the cannabinoid-activated
G protein-coupled receptor GPR18. Biomolecules. 2020;10(5):686. doi: 10.3390/biom10050686.</cite> [<a href="https://doi.org/10.3390/biom10050686" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7277601/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32365486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomolecules&amp;title=Computational%20investigations%20on%20the%20binding%20mode%20of%20ligands%20for%20the%20cannabinoid-activated%20G%20protein-coupled%20receptor%20GPR18&amp;author=A.%20Neumann&amp;author=V.%20Engel&amp;author=A.%20B.%20Mahardhika&amp;author=C.%20T.%20Schoeder&amp;author=V.%20Namasivayam&amp;volume=10&amp;issue=5&amp;publication_year=2020&amp;pages=686&amp;pmid=32365486&amp;doi=10.3390/biom10050686&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref45">
<cite>Fusani L., Palmer D. S., Somers D. O., Wall I. D.. Exploring ligand
stability in protein crystal structures using binding pose metadynamics. J. Chem. Inf. Model. 2020;60(3):1528–1539. doi: 10.1021/acs.jcim.9b00843.</cite> [<a href="https://doi.org/10.1021/acs.jcim.9b00843" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7145342/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31910338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Inf.%20Model.&amp;title=Exploring%20ligand%20stability%20in%20protein%20crystal%20structures%20using%20binding%20pose%20metadynamics&amp;author=L.%20Fusani&amp;author=D.%20S.%20Palmer&amp;author=D.%20O.%20Somers&amp;author=I.%20D.%20Wall&amp;volume=60&amp;issue=3&amp;publication_year=2020&amp;pages=1528-1539&amp;pmid=31910338&amp;doi=10.1021/acs.jcim.9b00843&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref46">
<cite>Wang J., Wolf R. M., Caldwell J. W., Kollman P. A., Case D. A.. Development
and testing of a general amber force field. J. Comput. Chem. 2004;25(9):1157–1174. doi: 10.1002/jcc.20035.</cite> [<a href="https://doi.org/10.1002/jcc.20035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15116359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&amp;title=Development%20and%20testing%20of%20a%20general%20amber%20force%20field&amp;author=J.%20Wang&amp;author=R.%20M.%20Wolf&amp;author=J.%20W.%20Caldwell&amp;author=P.%20A.%20Kollman&amp;author=D.%20A.%20Case&amp;volume=25&amp;issue=9&amp;publication_year=2004&amp;pages=1157-1174&amp;pmid=15116359&amp;doi=10.1002/jcc.20035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref47">
<cite>Essmann U., Perera L., Berkowitz M. L., Darden T., Lee H., Pedersen L. G.. A smooth particle
mesh Ewald method. J. Chem. Phys. 1995;103(19):8577–8593. doi: 10.1063/1.470117.</cite> [<a href="https://doi.org/10.1063/1.470117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Phys.&amp;title=A%20smooth%20particle%20mesh%20Ewald%20method&amp;author=U.%20Essmann&amp;author=L.%20Perera&amp;author=M.%20L.%20Berkowitz&amp;author=T.%20Darden&amp;author=H.%20Lee&amp;volume=103&amp;issue=19&amp;publication_year=1995&amp;pages=8577-8593&amp;doi=10.1063/1.470117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref48">
<cite>Xiong G., Wu Z., Yi J., Fu L., Yang Z., Hsieh C., Yin M., Zeng X., Wu C., Lu A.. et al. ADMETlab
2.0: An integrated online platform for accurate and comprehensive
predictions of ADMET properties. Nucleic Acids
Res. 2021;49(W1):W5–W14. doi: 10.1093/nar/gkab255.</cite> [<a href="https://doi.org/10.1093/nar/gkab255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8262709/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33893803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=ADMETlab%202.0:%20An%20integrated%20online%20platform%20for%20accurate%20and%20comprehensive%20predictions%20of%20ADMET%20properties&amp;author=G.%20Xiong&amp;author=Z.%20Wu&amp;author=J.%20Yi&amp;author=L.%20Fu&amp;author=Z.%20Yang&amp;volume=49&amp;issue=W1&amp;publication_year=2021&amp;pages=W5-W14&amp;pmid=33893803&amp;doi=10.1093/nar/gkab255&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref49">
<cite>Pires D. E., Blundell T. L., Ascher D.. B. pkCSM:
predicting small-molecule
pharmacokinetic and toxicity properties using graph-based signatures. J. Med. Chem. 2015;58(9):4066–4072. doi: 10.1021/acs.jmedchem.5b00104.</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.5b00104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4434528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25860834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.&amp;title=B.%20pkCSM:%20predicting%20small-molecule%20pharmacokinetic%20and%20toxicity%20properties%20using%20graph-based%20signatures&amp;author=D.%20E.%20Pires&amp;author=T.%20L.%20Blundell&amp;author=D.%20Ascher&amp;volume=58&amp;issue=9&amp;publication_year=2015&amp;pages=4066-4072&amp;pmid=25860834&amp;doi=10.1021/acs.jmedchem.5b00104&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref50">
<cite>Daina A., Michielin O., Zoete V.. SwissADME: a free web tool to evaluate
pharmacokinetics, drug-likeness and medicinal chemistry friendliness
of small molecules. Sci. Rep. 2017;7(1):42717. doi: 10.1038/srep42717.</cite> [<a href="https://doi.org/10.1038/srep42717" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5335600/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28256516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=SwissADME:%20a%20free%20web%20tool%20to%20evaluate%20pharmacokinetics,%20drug-likeness%20and%20medicinal%20chemistry%20friendliness%20of%20small%20molecules&amp;author=A.%20Daina&amp;author=O.%20Michielin&amp;author=V.%20Zoete&amp;volume=7&amp;issue=1&amp;publication_year=2017&amp;pages=42717&amp;pmid=28256516&amp;doi=10.1038/srep42717&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref51">
<cite>Lipinski C. A., Lombardo F., Dominy B. W., Feeney P. J.. Experimental and
computational approaches to estimate solubility and permeability in
drug discovery and development settings. Adv.
Drug Delivery Rev. 2012;64:4–17. doi: 10.1016/j.addr.2012.09.019.</cite> [<a href="https://doi.org/10.1016/j.addr.2012.09.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11259830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Drug%20Delivery%20Rev.&amp;title=Experimental%20and%20computational%20approaches%20to%20estimate%20solubility%20and%20permeability%20in%20drug%20discovery%20and%20development%20settings&amp;author=C.%20A.%20Lipinski&amp;author=F.%20Lombardo&amp;author=B.%20W.%20Dominy&amp;author=P.%20J.%20Feeney&amp;volume=64&amp;publication_year=2012&amp;pages=4-17&amp;pmid=11259830&amp;doi=10.1016/j.addr.2012.09.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref52">
<cite>

Frisch, M.
, et al. , Gaussian 16.
2016, Gaussian,
Inc: Wallingford, CT.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Gaussian%2016.&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref53">
<cite>

Dennington, R.
; 
Keith, T.
; 
Millam, J.
. 
GaussView 6.0. 16, Semichem Inc: Shawnee
Mission, 2016. 
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=GaussView%206.0.%2016&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref54">
<cite>Zaharim W. N., Sulaiman S., Mohd Tajudin S. S., Abu Bakar S. N., Ismail N. E., Rozak H., Watanabe I.. Basis set effects in
density functional theory calculation of muoniated cytosine nucleobase. Key Eng. Mater. 2020;860:282–287. doi: 10.4028/www.scientific.net/KEM.860.282.</cite> [<a href="https://doi.org/10.4028/www.scientific.net/KEM.860.282" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Key%20Eng.%20Mater.&amp;title=Basis%20set%20effects%20in%20density%20functional%20theory%20calculation%20of%20muoniated%20cytosine%20nucleobase&amp;author=W.%20N.%20Zaharim&amp;author=S.%20Sulaiman&amp;author=S.%20S.%20Mohd%20Tajudin&amp;author=S.%20N.%20Abu%20Bakar&amp;author=N.%20E.%20Ismail&amp;volume=860&amp;publication_year=2020&amp;pages=282-287&amp;doi=10.4028/www.scientific.net/KEM.860.282&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref55">
<cite>Lu T., Chen F.. Multiwfn: A multifunctional
wavefunction analyzer. J. Comput. Chem. 2012;33(5):580–592. doi: 10.1002/jcc.22885.</cite> [<a href="https://doi.org/10.1002/jcc.22885" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22162017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&amp;title=Multiwfn:%20A%20multifunctional%20wavefunction%20analyzer&amp;author=T.%20Lu&amp;author=F.%20Chen&amp;volume=33&amp;issue=5&amp;publication_year=2012&amp;pages=580-592&amp;pmid=22162017&amp;doi=10.1002/jcc.22885&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref56">
<cite>Humphrey W., Dalke A., Schulten K.. VMD: Visual
molecular dynamics. J. Mol. Graphics. 1996;14(1):33–38. doi: 10.1016/0263-7855(96)00018-5.</cite> [<a href="https://doi.org/10.1016/0263-7855(96)00018-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8744570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Graphics&amp;title=VMD:%20Visual%20molecular%20dynamics&amp;author=W.%20Humphrey&amp;author=A.%20Dalke&amp;author=K.%20Schulten&amp;volume=14&amp;issue=1&amp;publication_year=1996&amp;pages=33-38&amp;pmid=8744570&amp;doi=10.1016/0263-7855(96)00018-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref57">
<cite>O’boyle N. M., Tenderholt A. L., Langner K. M.. Cclib: a library for package-independent
computational chemistry algorithms. J. Comput.
Chem. 2008;29(5):839–845. doi: 10.1002/jcc.20823.</cite> [<a href="https://doi.org/10.1002/jcc.20823" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17849392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&amp;title=Cclib:%20a%20library%20for%20package-independent%20computational%20chemistry%20algorithms&amp;author=N.%20M.%20O%E2%80%99boyle&amp;author=A.%20L.%20Tenderholt&amp;author=K.%20M.%20Langner&amp;volume=29&amp;issue=5&amp;publication_year=2008&amp;pages=839-845&amp;pmid=17849392&amp;doi=10.1002/jcc.20823&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12332581/bin/ao5c03682_si_001.zip" data-ga-action="click_feat_suppl" class="usa-link">ao5c03682_si_001.zip</a><sup> (1.4MB, zip) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12332581/bin/ao5c03682_si_002.pdf" data-ga-action="click_feat_suppl" class="usa-link">ao5c03682_si_002.pdf</a><sup> (2.7MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data supporting
this work are available in this paper, and details are present in
the Supporting Information.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Omega are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsomega.5c03682"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/ao5c03682.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (9.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12332581/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12332581/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332581%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332581/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12332581/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12332581/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40787374/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12332581/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40787374/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12332581/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12332581/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="mXJKiMoVd6rBCE0KGdZvpnK7LEc58HaMfTxktRSxbts1YJWKhKP583gUVhNPW6Dz">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
