
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Residue Variations in Human N‑Acetyltransferase 2 Enzyme Alleles: From Rapid to Slow Acetylation - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="67D7138889B732B31C1388002B6B9445.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsomega">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332655/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Omega">
<meta name="citation_title" content="Residue Variations in Human N‑Acetyltransferase 2 Enzyme Alleles: From Rapid to Slow Acetylation">
<meta name="citation_author" content="Wayde Veldman">
<meta name="citation_author_institution" content="Research Unit in Bioinformatics (RUBi), Department of Biochemistry, Microbiology and Bioinformatics, Rhodes University, Makhanda 6139, South Africa">
<meta name="citation_author" content="Özlem Tastan Bishop">
<meta name="citation_author_institution" content="Research Unit in Bioinformatics (RUBi), Department of Biochemistry, Microbiology and Bioinformatics, Rhodes University, Makhanda 6139, South Africa">
<meta name="citation_publication_date" content="2025 Jul 27">
<meta name="citation_volume" content="10">
<meta name="citation_issue" content="30">
<meta name="citation_firstpage" content="33045">
<meta name="citation_doi" content="10.1021/acsomega.5c02316">
<meta name="citation_pmid" content="40787317">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332655/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332655/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332655/pdf/ao5c02316.pdf">
<meta name="description" content="Understanding the effects of single-nucleotide variations on the three-dimensional structures of drug-metabolizing enzymes at the atomic level is not a well-studied field and deserves attention as it could provide insights into tackling both drug ...">
<meta name="og:title" content="Residue Variations in Human N‑Acetyltransferase 2 Enzyme Alleles: From Rapid to Slow Acetylation">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Understanding the effects of single-nucleotide variations on the three-dimensional structures of drug-metabolizing enzymes at the atomic level is not a well-studied field and deserves attention as it could provide insights into tackling both drug ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332655/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12332655">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsomega.5c02316"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/ao5c02316.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332655%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12332655/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12332655/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332655/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsomega.png" alt="ACS Omega logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Omega" title="Link to ACS Omega" shape="default" href="https://pubs.acs.org/journal/acsodf" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Omega</button></div>. 2025 Jul 27;10(30):33045–33062. doi: <a href="https://doi.org/10.1021/acsomega.5c02316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsomega.5c02316</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Omega%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Omega%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Residue Variations
in Human N‑Acetyltransferase
2 Enzyme Alleles: From Rapid to Slow Acetylation</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Veldman%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Wayde Veldman</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Wayde Veldman</span></h3>
<div class="p">
<sup>1</sup>
Research Unit in Bioinformatics
(RUBi), Department of Biochemistry, Microbiology and Bioinformatics, Rhodes University, Makhanda
6139, South Africa
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Veldman%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wayde Veldman</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tastan%20Bishop%20%C3%96%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Özlem Tastan Bishop</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Özlem Tastan Bishop</span></h3>
<div class="p">
<sup>1</sup>
Research Unit in Bioinformatics
(RUBi), Department of Biochemistry, Microbiology and Bioinformatics, Rhodes University, Makhanda
6139, South Africa
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tastan%20Bishop%20%C3%96%22%5BAuthor%5D" class="usa-link"><span class="name western">Özlem Tastan Bishop</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>
Research Unit in Bioinformatics
(RUBi), Department of Biochemistry, Microbiology and Bioinformatics, Rhodes University, Makhanda
6139, South Africa
</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
Email: <span>o.tastanbishop@ru.ac.za</span>.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 12; Accepted 2025 Jun 26; Revised 2025 Jun 11; Collection date 2025 Aug 5.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY-NC-ND 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12332655  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40787317/" class="usa-link">40787317</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Understanding the effects of single-nucleotide variations
on the
three-dimensional structures of drug-metabolizing enzymes at the atomic
level is not a well-studied field and deserves attention as it could
provide insights into tackling both drug resistance and drug efficacy/toxicity
issues. Here, we examined molecular dynamics simulations of the isoniazid-metabolizing
enzyme arylamine N-acetyltransferase 2 (NAT2) and established a computational
approach to decipher how NAT2 transitions from being a rapid acetylator
to a slow acetylator in the presence of residue variations. After
revealing slight differences between two rapid acetylators (the NAT2
reference allele and the legacy reference allele), we compared them
to five variants with slow clinical phenotypes. In all slow acetylators,
we observed a significant reduction in hydrogen bonding and dynamic
residue network metric values in functional residues. For example,
in the R64Q+K268R variant, the R64Q substitution causes a substantial
reduction of hydrogen bonds, resulting in increased root-mean-square
fluctuation and changes in residue networks. This is evidenced by
the significantly decreased <em>betweenness centrality</em> values of catalytic D122 and putative isoniazid-binding residues
S125 and F217, as well as the significantly decreased <em>eigencentrality</em> values of N72, D122, and the active site loop residue G124. Also,
in this variant, the average distance between the putative isoniazid-binding
residue F217 and catalytic residues H107 and D122 is significantly
reduced. This structural shift may affect ligand binding affinity
and catalysis. Overall, our findings offer new insights into how allosteric
variations alter the NAT2 mechanism, leading to changes in drug exposure.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/38a9/12332655/933bc1102009/ao5c02316_0013.jpg" loading="lazy" id="abs1" height="374" width="655" alt="graphic file with name ao5c02316_0013.jpg"></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/38a9/12332655/933bc1102009/ao5c02316_0011.jpg" loading="lazy" id="tgr1" height="374" width="655" alt="graphic file with name ao5c02316_0011.jpg"></p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>There are three major challenges facing
marketed drugs: drug resistance,
lack of efficacy due to insufficient drug exposure, and adverse reactions
due to increased exposure. All three are mainly linked to nucleotide
variations, the first due to evolutionary changes in the pathogen
genome under drug pressure and the second and third being due to population-specific
single-nucleotide variations/polymorphisms (SNVs/SNPs) in drug-metabolizing
enzymes (pharmacogenes) in humans. There is also a growing body of
knowledge on the connection between drug resistance and human pharmacogenomics. This is particularly important in populations
with a high incidence of infectious diseases and distinct, population-specific
variation profiles. Therefore, understanding the effects of SNVs on
drug-metabolizing enzymes might provide insight into addressing both
drug resistance and drug efficacy/toxicity issues.</p>
<p>Here, we
focused on the isoniazid-metabolizing enzyme arylamine
N-acetyltransferase 2 (NAT2). The rate of metabolic activity of NAT2
is affected by population-specific single-nucleotide variations/polymorphisms
(SNVs/SNPs), which in turn impact the rate of metabolism of isoniazid.<em>
−




</em> NAT2 is characterized as a slow or rapid acetylator, depending on
residue variation.<em>
,,−

</em> In the NAT2 nomenclature, rapid acetylators represent
increased-functioning proteins, which are associated with a higher
risk of treatment failure<em>
,
</em> due to insufficient
exposure, whereas the slow acetylator phenotype represents decreased-functioning
proteins, which are linked to increased isoniazid plasma exposure<em>
,
</em> and a higher risk of isoniazid hepatotoxicity.<em>
−

</em> Therefore,
studying the structure, function, and dynamics of human NAT2 enzyme
alleles with a slow acetylation phenotype could reveal how these residue
substitutions induce allosteric effects and cause reduced enzyme activity.</p>
<p>Until recently, the reference
allele in the NAT scientific community
was NAT2*4,<em>
,
</em> a rapid acetylator,<em>
,,−
,
</em> with lysine (K) at position 268. The current PharmVar
reference allele, NAT2*1 (legacy allele NAT2*12A), is also a rapid
acetylator but has arginine (R) at position 268. Here, we chose to
use the legacy reference enzyme (NAT2*4) as our reference because
(1) all previous functional studies have used it as a reference, and
this is still the functional reference, and (2) all global populations
have a higher frequency of the legacy reference allele, with the global
average being approximately 57%. Hereafter,
the terms “legacy reference protein” and “reference
protein” will be used interchangeably to refer to the K268
protein system (NAT2*4).</p>
<p>In this study, our aim is to understand
how residue variations
may shift the enzyme function from the “rapid” to “slow”
clinical phenotype. As a first step, we structurally and dynamically
compared the two rapid NAT2 proteins (K268 and R268) to determine
whether the residue difference at position 268 has any effect. After
identifying the protein changes caused by this variation at residue
268, we further investigated how additional residue variations might
alter the enzyme function to the “slow” clinical phenotype.
Overall, we analyzed five NAT2 enzyme alleles with the following variations
in addition to K268R: (1) R64Q (NAT2*46); (2) I114T (NAT2*5); (3)
D122N (NAT2*48); (4) R197Q (NAT2*34); and (5) R64Q and I114T together
(NAT2*29).</p>
<p>We found that the slow acetylation phenotype in these
five variants
is primarily driven by localized structural alterations, rather than
global conformational changes, resulting from increased residue fluctuations,
disrupted hydrogen bonding, and altered residue communication networks:
(1) in variant R64Q+K268R, the loss of E38-R64 and N41-R64 hydrogen
bonds caused extensive structural changes in domain I, the interdomain,
and the C-terminal tail, leading to increased RMSF. Furthermore, the <em>betweenness centrality</em> (<em>BC</em>) values of catalytic
residue D122 and isoniazid-binding residues S125 and F217, and the <em>eigencentrality</em> (<em>EC</em>) values of functional
residues N72, D122, and G124, were reduced compared to rapid acetylator
systems. Additionally, the average minimum distance between the putative
isoniazid-binding residue F217 and the catalytic residues H107 and
D122 was significantly decreased. (2) For the I114T+K268R variant,
changes in hydrogen bonding of the active site loop disrupted catalytic
D122 and key binding residues. There were significant increases in
the average distance between active site loop binding residue S125
and catalytic residues acetyl-C68, H107, and D122. (3) In the D122N+K268R
variant, the loss of ionic interactions between catalytic residues
N122 and H107, along with a significant reduction in N122-mediated
hydrogen bonds, destabilized the catalytic triad. This was accompanied
by substantial decreases in <em>EC</em> and <em>closeness
centrality</em> (<em>CC</em>) values. Additionally, many
significant distance decreases between active site residues were observed.
(4) Structural changes in the R197Q+K268R variant included reorientation
of binding residue F217 as well as reduced <em>BC</em> value
of catalytic D122. (5) The combined effects of both R64Q and I114T
substitutions in the R64Q+I114T+K268R variant disrupted E38-R64 and
N41-R64 hydrogen bonds and affected the active site loop. These changes
led to decreased <em>BC</em> value of residue N72 and reduced <em>EC</em> of residues G124, S128, and I290, likely affecting substrate
and cofactor binding.</p></section><section id="sec2"><h2 class="pmc_sec_title">Methodology</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">Protein Selection and Preparation</h3>
<p>The human NAT2 structure
PDB ID <a href="https://www.rcsb.org/structure/2PFR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2PFR</a> (chain
A) was chosen because it is the only
experimentally solved human NAT2 structure; additionally, it has good
quality metrics according to the PDB and no missing residues. Notably,
residue 268 is a lysine in structure 2PFR, corresponding to the current
legacy reference allele. Coenzyme A was removed to achieve the apo
enzyme state. Regarding the residue rotamers, only the first rotamer
was retained. Using the protein mutagenesis wizard in PyMOL version
2.5, missing atoms were added to residues
E4, E8, R16, N17, R61, K173, E174, K183, D248, E261, and E265. The
first residue in the structure, serine, is a cloning artifact that
was mutated to the first residue of the UniProt sequence, methionine,
by using PyMOL.</p>
<p>The structure was then protonated using the
H++ Web server at physiological pH 7.4.
In accordance with the catalytic mechanism for a substrate-binding
NAT2 system at physiological pH,<em>
,
</em> the hydrogen
on the sulfur atom of catalytic C68 was replaced with an acetyl group
using PyMOL, and catalytic H107 was protonated at the epsilon position
(HIE: neutral, ε-nitrogen protonated).</p></section><section id="sec2.2"><h3 class="pmc_sec_title">Selection of Variants and Preparation of Allele Proteins</h3>
<p>Along with the legacy reference structure (legacy and current NAT2*4),
we prepared six other protein systems, including the K268R variant
protein and all of the alleles with known clinical data that exhibit
a slow acetylation phenotype as a result of additional variation(s)
to the K268R variation (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>). Although
each of these alleles is associated with a different genotype, some
were included in one system due to their common missense mutations
(residue substitutions/variations), while their synonymous mutations
were ignored (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>). Each allele protein
was generated by mutating the relevant residues in PyMOL and then
protonated individually, as explained above.</p></section><section id="sec2.3"><h3 class="pmc_sec_title">Molecular Dynamics (MD) Simulations</h3>
<p>For the MD simulations,
the AMBER 14SB force field was used with
GROMACS 2020.1. The antechamber, prepgen,
and parmchk2 tools of the AmberTools18 package were utilized to create parameters for the acetyl-cysteine
residue via the AM1-BCC charge method. The tleap program, also from AmberTools18, was used to generate
the topology. During the previous step with the TIP3P water model,
water molecules were added as a solvent to a cubic periodic box with
a minimum distance of 10 Å. The systems were neutralized using
0.15 M NaCl, and the van der Waals and bonding parameters from general
amber force field 2 (GAFF2) were implemented. The topology was then
converted to GROMACS using the ACPYPE (or AnteChamber PYthon Parser
interfacE) program. For the production runs, all molecular systems
were energy-minimized using a conjugate gradient being energy relaxed
with a force tolerance of 1000 kJ/(mol nm) and capped at a maximum
of 50,000 steps. The temperature of the systems was then equilibrated
at 310.15 K over 100 ps utilizing the velocity rescale thermostat
(modified Berendsen thermostat). The
pressure was equilibrated using the Parrinello–Rahman barostat to maintain 1 bar of system pressure. Production
runs used one Nvidia Tesla v100 GPU in conjunction with 10 CPU cores
at the Centre for High Performance Computing (Cape Town, South Africa).
All simulations were run at a pH of 7.4 and a temperature of 310.15
K (37 °C). Each of the MD simulations was run for 700 ns with
2 fs time steps, and coordinates were written every 10 ps. The Linear
Constraint Solver (LINCS) algorithm was
used for the duration of the simulations to correct for rotational
bond lengthening. Coulombic and van der Waals short-range cutoffs
were set at 1.4 nm. Long-distance electrostatic interactions were
calculated using the Particle–Mesh Ewald (PME) algorithm, with
a Fourier grid spacing of 0.16 nm.</p></section><section id="sec2.4"><h3 class="pmc_sec_title">Molecular Dynamics Trajectory Analysis</h3>
<p>Following the
completion of the MD simulations, periodic boundary conditions of
the trajectories were corrected, jumps across boundaries were removed,
and the protein was centered inside the simulation box. The trajectories
were visually inspected using VMD. The
first 100 ns were omitted due to equilibration, and the final 600
ns of the simulations (100–700 ns) were utilized for analysis.
The GROMACS tools <em>gmx rms</em>, <em>gmx rmsf</em>, and <em>gmx gyrate</em> were used to determine the root-mean-square
deviation (RMSD), root-mean-square fluctuation (RMSF), and radius
of gyration (<em>R</em>
<sub>g</sub>), respectively. Protein
backbone atoms were used to calculate the protein RMSD after least-squares
fitting to the backbone atoms. The protein backbone RMSF was used
to analyze the motion of individual residues during the simulation.
To determine protein compactness, the <em>R</em>
<sub>g</sub> was computed.</p>
<p>Triplicate independent simulations for the legacy
reference and K268R systems were run to check consistency and confidence
in the system and force field chosen. The RMSD and <em>R</em>
<sub>g</sub> values of these triplicate runs were consistent and
are summarized in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S1 and S3</a> as
line plots and in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S2 and S4</a> as
violin plots, respectively.</p>
<section id="sec2.4.1"><h4 class="pmc_sec_title">Hydrogen Bonding</h4>
<p>Inter-residue hydrogen bonding information
was calculated using the “hbond” command in AMBER’s
cpptraj program with default parameters.
For each MD system, hydrogen bond frequencies between atoms were summed
by (1) residue pair and (2) individual residue. A single residue pair
can exhibit a frequency above 100% due to two or more simultaneous
atom-pair hydrogen bonds within the residue pair.</p></section><section id="sec2.4.2"><h4 class="pmc_sec_title">Dynamic Residue Network Analysis</h4>
<p>DRN calculations were
performed using MD-TASK,<em>
,
</em> where DRN metrics were
computed every 10th MD frame, and residue centrality values were averaged.
The DRN metrics included <em>averaged betweenness centrality</em> (<em>BC</em>), <em>averaged closeness centrality</em> (<em>CC</em>), and <em>averaged eigenvector centrality</em> (<em>EC</em>). From this point onward, the <em>averaged
BC</em>, <em>CC</em>, and <em>EC</em> will be
referred to as <em>BC</em>, <em>CC</em>, and <em>EC</em>, respectively.</p></section><section id="sec2.4.3"><h4 class="pmc_sec_title">Clustering Analysis</h4>
<p>Clustering analysis was performed
with GROMACS using the GROMOS method at an RMSD of 1 Å, backbone
atoms, and a step-size of 10. Cluster 1 of each system was used as
a representation of the major structural conformations of the enzymes
for enzyme mapping. The MD frame frequency of the clusters is shown
in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S2</a>.</p></section><section id="sec2.4.4"><h4 class="pmc_sec_title">In-House Python Scripts to Calculate per-Residue Metric Differences
(Delta)</h4>
<p>During comparisons, delta calculations help limit
noise and clearly highlight major differences.<em>
−


</em> The steps are as follows:</p>
<ul class="list" style="list-style-type:none">
<li><p>Step 1: Selection of the Baseline System: First, a baseline
(reference) system is selected for comparison with other systems.
We made two comparisons in the study: (1) the legacy reference average
(averaged per-residue values over triplicate simulations) was used
as the baseline system to compare with the rapid K268R system. (2)
The rapid system average (averaged per-residue values over the six
rapid system simulations) was used as the baseline system to compare
with the slow systems.</p></li>
<li><div class="p">Step 2: Per-Residue
Delta Calculation: For each system
compared, per-residue delta values for each metric (RMSF, hydrogen
bonding, or DRN) were calculated by subtracting the residue value
of the baseline system from the residue value of the system being
compared (system minus baseline system).<table class="disp-formula p" id="ueq1"><tr><td class="formula"><math id="d1e422" display="block"><msub><mi>Delta</mi><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><msub><mi>Metric</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow></msub><mo>−</mo><msub><mi>Metric</mi><mrow><mi>i</mi><mo>,</mo><mi>baseline</mi></mrow></msub></math></td></tr></table> where <em>i</em> is the residue number, <em>j</em> is the allele/system ID, and “baseline”
refers to the selected baseline system.</div></li>
<li><p>Step 3: Identification of Residues with the Largest
Delta Values: For each system compared, residues with the largest
delta values (top and bottom 5%) were identified, corresponding to
the 14 highest and 14 lowest per-residue delta values (28 residues
total per system). In addition to the per-residue hydrogen bond frequency
values, the hydrogen bond frequencies between residue pairs were identified
(the highest and lowest 14 residue pairs per system).</p></li>
<li><p>Step 4: Visualization: The metric values of the highest
and lowest 5% delta residues per system were plotted in heatmaps using
Seaborn and mapped to the enzyme structure,
shown as sticks with PyMOL. The corresponding residues of systems
not in the highest and lowest 5% delta residues were included in the
heatmaps for comparison purposes. The blue color indicates a decrease
from the reference, while the red color indicates an increase. The
color intensities of the residues in the heatmaps and structures match
for easy analysis. Hydrogen bond residue pairs were connected by lines
between their alpha-carbons, with the line thickness corresponding
to their hydrogen bond frequency. In the text and figures, residue
pairs are always ordered as “acceptor-donor”, e.g.,
E38-R64, unless more than two atoms per-residue pair are involved.</p></li>
<li><p>Step 5: Statistical Significance of Delta
Values: The
significance of the highest and lowest 5% delta values was determined
using raw values to calculate the standard deviation across baseline
systems. Residues with delta values greater than three standard deviations
from the baseline mean were considered significant. Within the highest
and lowest 5% delta values, significant changes from the baseline
system average are marked with an “X” in the heatmaps,
whereas nonsignificant changes are marked with a “-”.
Only residues with significant changes are labeled on the enzyme structure
figures.</p></li>
</ul></section></section></section><section id="sec3"><h2 class="pmc_sec_title">Results &amp; Discussion</h2>
<p>The human NAT2 enzyme contains
domain I, domain II, the mammalian
insertion which is located within domain II, the interdomain region,
and domain III<em>
,
</em> (<a href="#fig1" class="usa-link">Figure 
</a>). The C-terminus extends across the enzyme
surface, enters the active site, and forms part of the binding site<em>
,
</em> with help from the mammalian insertion, which acts as a lid to bury
the C-terminus via van der Waals interactions and hydrogen bonds,
contributing to overall protein stability.<em>
,
</em> Domains I and II maintain the orientation and geometry between the
catalytic triad residues C68, H107, and D122, and are much more conserved
between NAT enzymes than domain III.<em>
,,
</em> Potential human NAT2-isoniazid-binding residues were
identified by a consensus of interactions shared between the bacterial
1W6F-isoniazid crystal structure, the
only published human NAT2-isoniazid docking study, our NAT2-isoniazid docking result (unpublished), and the
NAT2-CoA crystal structure (PDB ID <a href="https://www.rcsb.org/structure/2PFR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2PFR</a>). Three
functional motifs are conserved between NAT enzymes<em>
,−

</em> (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S5</a>: FENL (residues
37–40), RGGWC (residues 64–68), and the “P-loop”
motif YIVDAGSG (residues 119–126). The RGGWC motif contains
the catalytic residue C68, as well as R64, which forms an important
salt bridge with FENL motif residue E38.<em>
,
</em> This salt bridge maintains the structure and stability of domain
I. The active site loop (residues 119–134), consisting of the
P-loop, the η3 helix, and the following loop, is located near
the center of the enzyme and contains catalytic residue D122 as well
as several binding residues (S125, S127, and S129) that have an important
role in determining the size of the binding pocket of different NAT
enzymes.<em>
,
</em> Additionally, the active site loop is thought
to be important for the structural integrity of the protein.<em>
−

</em>
</p>
<figure class="fig xbox font-sm" id="fig1"><h3 class="obj_head">1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332655_ao5c02316_0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/38a9/12332655/c172502208c1/ao5c02316_0001.jpg" loading="lazy" id="gr1" height="580" width="750" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Human
NAT2 enzyme is shown with domain I in red (residues 1–83),
domain II in blue (residues 84–192), the mammalian insert in
yellow (located within domain II; residues 167–183), the interdomain
region in orange (residues 193–229), and domain III in green
(residues 230–290). (a) Crystallographic structure (PDB ID <a href="https://www.rcsb.org/structure/2PFR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2PFR</a> chain A) with catalytic
residues shown as cyan sticks, putative isoniazid-binding residues
shown as magenta sticks, and residue substitutions of alleles shown
as spheres. (b) 2PFR sequence is shown with labeled α-helices
(coils), β-sheets (arrows), 3<sub>10</sub> helices (η;
coils), and strict β-turns (TT). The catalytic residues are
shown as cyan stars, the putative isoniazid-binding residues are shown
as magenta stars, and the residue substitutions of alleles with known
phenotypes are shown as black stars. The first residue in the structure,
serine, is a cloning artifact, whereas the first residue of the UniProt
sequence is methionine. The secondary structure labels were aligned
to sequences using ESPript.</p></figcaption></figure><p>In this study, analysis was conducted in two parts.
In the first
part, we analyzed the two NAT2 rapid acetylators using RMSF, hydrogen
bond, and DRN calculations. In the second part, the analysis was extended
to the slow acetylators, each of which contained additional variations
relative to the rapid acetylator proteins.</p></section><section id="sec4"><h2 class="pmc_sec_title">Part 1: Comparative Analysis of Two Rapid Acetylator NAT2 Proteins
with “K” or “R” at Position 268</h2>
<section id="sec4.1"><h3 class="pmc_sec_title">Root-Mean-Square Fluctuation Analysis Revealed Minimal Differences
between Rapid Proteins</h3>
<p>The average RMSF values from independent
triplicate MD simulations of the reference protein (K268) are mapped
onto the structure using varying intensities of red to represent fluctuation
levels (<a href="#fig2" class="usa-link">Figure 
</a>a),
where darker shades indicate higher RMSF values. The residues with
the highest RMSF values (top 10%) are shown as sticks (<a href="#fig2" class="usa-link">Figure 
</a>a) and are presented separately
in a bar plot (<a href="#fig2" class="usa-link">Figure 
</a>b).</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332655_ao5c02316_0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/38a9/12332655/eb2acc798dbc/ao5c02316_0002.jpg" loading="lazy" id="gr2" height="194" width="700" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Reference (K268) average RMSF and K268R average delta-RMSF. (a)
Reference average RMSF values mapped to structure in different intensities
of red color. The darker the red color, the larger the RMSF values.
The residues with the largest RMSF values (top 10%: 29 residues) are
shown in stick representation. (b) Bar plot showing the reference
average residues with the largest RMSF values (top 10%). (c) K268R
average delta-RMSF values (K268R average minus reference average)
mapped to structure in different intensities of red (increase) and
blue (decrease) color. The darker the colors, the larger the delta-RMSF
values. The residues with the largest delta-RMSF values (top and bottom
5%: 28 residues) are shown in stick representation. Only significant
top and bottom 5% delta residues are labeled. (d) Bar plot showing
the K268R average residues with the largest delta-RMSF values (top
and bottom 5%). Significant top and bottom 5% delta residues are indicated
with dots.</p></figcaption></figure><p>Notably, the three regions that form the active
site entrance are
all within the top 10% of RMSF residues: P96 to N99 (3<sub>10</sub> helix η2), T211 to T214 (just after helix α5), and D285
to S287 (C-terminal tail). These regions exhibit high flexibility,
which may facilitate the entry and exit of the substrate and cofactor
into the active site pocket. This is further supported by the trajectory
clustering analysis, where the top three conformational clusters from
each reference MD simulation are superimposed, revealing slight up-and-down
movement of helix α5 and a shift of the 3<sub>10</sub> helix
η2 away from the active site pocket (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S6</a>).</p>
<p>We then examined the RMSF of the K268R variant relative
to the
reference enzyme to determine whether this residue variation affects
the dynamics of the protein (<a href="#fig2" class="usa-link">Figure 
</a>c,d). The K268R system is also a rapid acetylator,
as discussed in the <a href="#sec1" class="usa-link">Introduction</a>. Functional
studies of the K268R variant recombinantly expressed in bacteria,<em>
,
</em> yeast,<em>
,
</em> and mammalian cells revealed no changes in protein expression, protein stability, or
catalytic activity. Walraven et al. suggested
that the conserved activity is due to the surface location and lack
of interactions with surrounding residues, rather than the conservative
nature of the K268R substitution. They also proposed that replacing
K268 with arginine is unlikely to affect the α-helical structure
of helix α7, where this residue is located.</p>
<p>Our comparative
analysis using delta-RMSF calculations revealed
minimal differences between these two protein systems (<a href="#fig2" class="usa-link">Figure 
</a>c). Examining the top and bottom
5% delta values to identify the residues with the greatest deviation
relative to the reference enzyme indicated decreased flexibility in
the N- and C-terminal regions. However, the delta-RMSF heatmap for
the rapid system (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S7</a>) shows that
the RMSF changes in the terminal regions do not exceed the statistical
threshold of 3 SD from the reference replicates and are therefore
not significant. In contrast, domain I residues Q46 and M48, and domain
II residues K184 and K185, show statistically significant RMSF increases,
though these remain minor, all below 0.16 Å (<a href="#fig2" class="usa-link">Figure 
</a>d). Since the K268R system
is a rapid acetylator, like the reference, these changes are unlikely
to have a substantial effect on enzyme activity or metabolism.</p></section><section id="sec4.2"><h3 class="pmc_sec_title">Delta Hydrogen Bond Analysis Showed Hydrogen Bond Gain around
the Variation</h3>
<p>
<a href="#fig3" class="usa-link">Figure 
</a> summarizes the hydrogen bond results for the rapid proteins.
In <a href="#fig3" class="usa-link">Figure 
</a>a, the reference
hydrogen bond frequencies are mapped onto the enzyme structure, while <a href="#fig3" class="usa-link">Figure 
</a>b highlights the
top 10% of residues with the highest hydrogen bond frequencies.</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332655_ao5c02316_0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/38a9/12332655/a6b97fc57b1d/ao5c02316_0003.jpg" loading="lazy" id="gr3" height="891" width="666" alt="3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Average hydrogen
bond frequencies in the reference and K268R systems.
(a) Combined hydrogen bond frequency of each residue in the reference
system over a 600 ns trajectory (100–700 ns) mapped onto the
enzyme structure. Residues are color-coded in shades of red, with
darker colors indicating higher frequencies. The top 5% of residues
(14 total) are represented as sticks, and their hydrogen bonding pairs
are connected by green lines, where line thickness reflects bonding
frequency. Additionally, three noteworthy high-frequency residue pairs
not formed by the top 5% residues are shown as blue lines. (b) Bar
plot displaying the average hydrogen bonding frequencies for the top
10% of residues (29 total) in the reference system. (c) K268R average
delta-hydrogen bond frequency (K268R average minus reference average),
represented on the enzyme structure with increases shown in red and
decreases in blue. Darker colors indicate greater changes in the bonding
frequency. The top and bottom 5% of residues (28 total) are shown
as sticks with delta-hydrogen bonding pairs connected by colored lines
corresponding to changes in frequency. Only significant top and bottom
5% delta residues and pairs are labeled. (d) Bar plots displaying
the average hydrogen bonding frequencies for the K268R system’s
top and bottom 5% residues (28 total) and top and bottom 14 residue
pairs, with significant residues and pairs indicated by dots.</p></figcaption></figure><p>Our calculations show that the catalytic residue
D122 exhibits
the highest hydrogen bond frequency among all residues (470.8%; <a href="#fig3" class="usa-link">Figure 
</a>b), forming hydrogen
bonds with N72, Q130, G124, H107, Y190, and S125 (in order of frequency).
D122 plays a significant role in maintaining the conformation and
stability of the active site loop, as G124, S125, and Q130 are all
located on this loop. Catalytic H107 is known to be stabilized by
D122 via ionic interaction to maintain the catalytic triad conformation;<em>
,
</em> our results showed that hydrogen bonding contributes to this stabilization.</p>
<p>R64 has the second-highest hydrogen bond frequency and shares the
highest <em>residue pair</em> hydrogen bond frequency with
E38 (206.9%). As mentioned earlier, R64 is known to form a salt bridge
with E38, which helps maintain the structure and stability of domain
I.<em>
,
</em> R64 forms hydrogen bonds with E38, N41,
R9, and H58, while E38 hydrogen bonds with R64, G65, L40, and G66.
These interactions contribute to the conformation and dynamics of
the domain I tertiary structure. R64,
E38, and N41 are highly conserved in mammalian NAT proteins, suggesting
that they play important roles in enzyme function and/or stability.
Significant structural changes in this region could alter the position
of catalytic acetyl-C68, potentially impacting enzyme activity.</p>
<p>I290 and R242 have the third- and fourth-highest hydrogen bond
frequencies, respectively (<a href="#fig3" class="usa-link">Figure 
</a>b), and together they form a high hydrogen bond frequency
residue pair (104.6%). I290 forms bonds with R165, R242, S128, L180,
T289, and S127, whereas R242 interacts with V231, I290, Q168, and
D251. The many hydrogen bonds of I290, R242, D251, R165, and Q163
produce a network connecting the C-terminal tail, active site loop,
mammalian insert, and domain III, which likely contributes to the
structure and stability of the enzyme and the active site pocket,
especially the C-terminal tail.</p>
<p>N72, the fifth residue with
the highest hydrogen bond frequency,
is located directly behind the catalytic site (<a href="#fig3" class="usa-link">Figure 
</a>a) and likely plays a role in maintaining
the structure and geometry of the catalytic triad, forming hydrogen
bonds with catalytic D122, L108 (adjacent to catalytic H107), A123
(adjacent to catalytic D122), Y76, and catalytic acetyl-C68. <a href="#fig3" class="usa-link">Figure 
</a>a highlights three
notable residue pairs with high hydrogen bond frequencies, shown by
blue lines, which were not formed by any of the top 5% hydrogen bond
residues. The C-terminal residue pair D285-S287 helps change the direction
90 deg of the C-terminal tail from downward across the enzyme face
to probing the active site. The F37-S224 residue pair serves as the
primary connection between domain I and the interdomain. F37, a key
residue for binding isoniazid and the cofactor,<em>
,,,
</em> is positioned
next to E38, which hydrogen bonds with the critical R64 residue. Finally,
G91 and D122 form hydrogen bonds with catalytic H107 from opposite
sides, contributing to its position and ensuring catalytic triad geometry
and enzyme function.</p>
<p>With the exception of D285 and S287, which
primarily form hydrogen
bonds with each other, the residues in the three active site entrance
regions mentioned earlier (P96→N99, T211→T214, and D285→S287)
exhibit low hydrogen bonding frequencies, placing them among the bottom
10%. This indicates an inverse correlation between RMSF and hydrogen
bonding in these residues, suggesting that they are likely unrestrained
and flexible to accommodate drug entry and binding.</p>
<p>Comparing
the K268 system to K268R via delta-hydrogen bonding calculations
(<a href="#fig3" class="usa-link">Figure 
</a>c,d) revealed
a significant hydrogen bond frequency gain around the variation site
in residue pairs E265-R268 (89%), E264-R268 (34%), and E265-N269 (18%).
The change of K268 to the larger and slightly longer arginine induces
hydrogen bonding between the R268 hydrogens and the oxygens of both
E264 and E265, pulling E265 closer to N269, thereby inducing hydrogen
bonding between these residues too. These changes in helix α7
propagate through the loop to beta-sheet β15 and the mammalian
insert, as demonstrated by the significant increase in hydrogen bonding
between the residue pair D251-R166 and the decrease between the residue
pair Q163-R165 (<a href="#fig3" class="usa-link">Figure 
</a>c,d). The effects of the residue variation may also propagate from
beta-sheet β15 across the antiparallel beta-sheets β14,
β13, and β12, leading to many hydrogen bond changes in
residue Q226 (at the end of β12), particularly the increase
in hydrogen bonding between the residue pair I32-Q226. This may result
in significant hydrogen bond changes in residues R33, I32, E29, and
D26. Given that the K268R system is a rapid acetylator, these hydrogen
bond changes, compared to those of the reference protein, are unlikely
to affect enzyme activity or metabolism rate.</p></section><section id="sec4.3"><h3 class="pmc_sec_title">Dynamic Residue Network Analysis Identified Functionally Important
Residues</h3>
<p>Here, we further analyzed the protein MD simulation
trajectories using the DRN approach and its specific metrics.<em>
,
</em> In previous studies, we showed that integrating information from
multiple DRN metrics provides complementary insights.<em>
−

</em>
</p>
<p>The DRN analysis of the reference enzyme revealed that residues
with high <em>BC</em> values are located within and around
the active site (<a href="#fig4" class="usa-link">Figure 
</a>a). Mapping the top 5% <em>BC</em> residues onto the structure
exhibited an adjacent-residue pathway formed by N72, D122, G124, Q130,
S129, S127, I290, S224, F37, P36, and V35, indicating information
flow between the catalytic site, active site loop, and domain I (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S8</a>). Residue S127, which has the highest <em>BC</em> value, formed hydrogen bonds with nine residues: T289,
Q226, Q130, S128, S224, S125, G126, I290, and S129, more than any
other residue. Located within the active site loop, S127 likely contributes
to the structure and stability of this loop by hydrogen bonding to
S125, G126, S128, S129, and Q130. It also connects the loop to the
interdomain region above and the C-terminal tail in front by forming
hydrogen bonds with S224, Q226 (interdomain), T289 and I290 (C-terminal
tail).</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332655_ao5c02316_0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/38a9/12332655/f0dce1bfbc4d/ao5c02316_0004.jpg" loading="lazy" id="gr4" height="1014" width="666" alt="4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>DRN analysis of reference and K268R enzymes. (a–c) Reference
system DRN metrics mapped onto the structure with the top 5% of residues
for each metric (14 residues) displayed in stick representation. (a)
BC values, (b) CC values, and (c) EC values are shown, with catalytic
residues in cyan and binding residues in purple. (d–f) Delta
metrics comparing the K268R system to the reference system, with red
indicating increases and blue indicating decreases. For each metric,
the top and bottom 5% of residues with the largest changes (28 residues)
are shown in stick representation, with significant delta residues
labeled on structure and marked with dots in the corresponding bar
plots. (d) Delta-BC, (e) delta-CC, and (f) delta-EC values. Note:
reference CC values were normalized to start at zero for visualization,
given the minimal difference between the highest and lowest unnormalized
values.</p></figcaption></figure><p>Corresponding to the definition of <em>CC,</em>
<em>
,,
</em> the reference residues
with the
highest <em>CC</em> values are located near the geometric
center of the protein (<a href="#fig4" class="usa-link">Figure 
</a>b). On the other hand, residues with high <em>EC</em> values cluster within and behind the catalytic site (<a href="#fig4" class="usa-link">Figure 
</a>c; <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S8</a>). <em>EC</em> measures the extent to which a node
is connected to other well-connected nodes in the network<em>
,
</em> and can identify distinct functional groups or communities of nodes.
Residue N72 has the highest <em>EC</em> value and, as mentioned
earlier, plays a crucial role in maintaining the structure and geometry
of the catalytic triad as it forms hydrogen bonds with many residues
around the catalytic site.</p>
<p>Six residues were commonly shared
among the top 10% of <em>BC</em>, <em>CC,</em> and <em>EC</em> residues:
F37, V71, N72, L109, catalytic D122, and G124. These residues are
primarily located in domain I and in the active site loop (V121-Q130)
and are likely important for enzyme residue communication and/or structure
and function. N72, as previously discussed,
plays a significant role in maintaining catalytic triad stability.
Residue V71, positioned next to N72, forms hydrogen bonds with W67,
which likely helps maintain the position of the catalytic residue
acetyl-C68. F37, a substrate/cofactor binding residue, forms hydrogen
bonds with interdomain residue S224 (top 5% <em>BC</em>),
which in turn forms hydrogen bonds with active site loop residue S127
(also top 5% <em>BC</em>). This creates a linkage between
domain I and the active site loop via the interdomain region. This
is likely the most important communication pathway of the enzyme as
all three of these residues are among the top 5% <em>BC</em> residues and are connected through hydrogen bonds at the intersection
of three different domains. L109 forms hydrogen bonds exclusively
with L89, connecting the antiparallel beta-sheets on which they are
situated. The catalytic D122 forms more hydrogen bonds than any other
residue (<a href="#fig3" class="usa-link">Figure 
</a>b),
while G124 exclusively hydrogen bonds with D122, supporting its catalytic
role.</p>
<p>Comparing the reference system to K268R using DRN calculations
revealed limited differences, particularly around the variation site
(<a href="#fig4" class="usa-link">Figure 
</a>d–f).
However, delta calculations highlighted significant deviations in
the <em>BC</em> values of residues C148 and R165 (<a href="#fig4" class="usa-link">Figure 
</a>a), possibly due
to hydrogen bonding changes in the region (<a href="#fig3" class="usa-link">Figure 
</a>). In domain I, <em>BC</em> values
decreased for residue E38 and increased for N39, while within the
active site loop, residues D122 and G124 showed a decrease and increase,
respectively. The only significant change in delta <em>CC</em> was observed for C148 (increase), which also showed a significant
increase in <em>BC</em> (<a href="#fig4" class="usa-link">Figure 
</a>e). With regard to delta <em>EC</em>, increased <em>EC</em> was observed in a group of residues
within and around interdomain α-helix 5, whereas decreased <em>EC</em> was seen in another group within and around the active
site loop. However, only residues L69 (increase) and L110 (decrease)
displayed significant changes compared to the reference replicates.
As the K268R system retains the rapid phenotype, these alterations
are unlikely to significantly affect the metabolic rate.</p></section></section><section id="sec5"><h2 class="pmc_sec_title">Part 2: Toward Understanding the Slow Acetylation Effect of
Additional Variations on the Reference Protein</h2>
<p>
<a href="#fig5" class="usa-link">Figure 
</a> compares
the rapid average values (baseline) with those of the slow variant
systems using various metrics. We interpret that residues showing
significant differences (equal to or greater than 3 SD) in the slow
acetylation systems may contribute to the reduced acetylation rates
observed in these variants. <a href="#fig6" class="usa-link">Figures 
</a>– illustrate these
differences within the 3D protein structures.</p>
<figure class="fig xbox font-sm" id="fig5"><h3 class="obj_head">5.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332655_ao5c02316_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/38a9/12332655/56aa2d59a6a8/ao5c02316_0005.jpg" loading="lazy" id="gr5" height="888" width="700" alt="5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Delta values (system
minus rapid average) of the systems are shown
in residue heatmaps. Color intensity shows the magnitude of change
from the rapid average: red is an increase, and blue is a decrease.
Within the highest and lowest 5% delta values, significant changes
from the rapid average are marked with an “X”, whereas
nonsignificant changes are marked with a “-”. Enzyme
domains are indicated with color-coded lines: Domain I (red), domain
II (blue), mammalian insert (yellow), interdomain (orange), and domain
III (green). (a) Delta-RMSF, (b) delta-hydrogen bonding of residue
pairs, (c) delta-hydrogen bonding of residues, (d) delta-BC, (e) delta-CC,
and (f) delta-EC. In the supplementary data, the same set of heatmaps
using the (1) reference average and (2) K268R average as the baseline
system can be found (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S9 and S10</a>).</p></figcaption></figure><figure class="fig xbox font-sm" id="fig6"><h3 class="obj_head">6.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332655_ao5c02316_0006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/38a9/12332655/4b168cb0d6b0/ao5c02316_0006.jpg" loading="lazy" id="gr6" height="483" width="766" alt="6"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Slow variant R64Q+K268R cluster 1 structure mapped with
values
of delta-RMSF, delta-hydrogen bonding, delta-BC, delta-CC, and delta-EC
(variant minus rapid average). The color intensities correspond to
the respective heatmaps (<a href="#fig5" class="usa-link">Figure 
</a>). Residues with the largest differences from the rapid
average (top and bottom 5%: top and bottom 14 residues) are shown
with stick representation. The residues of the top and bottom 14 hydrogen
bond residue pairs are connected with lines. Red color represents
increased values compared to the rapid average, while blue color represents
decreased values. Only residues with significant differences from
the rapid systems are labeled, except for the residue variations which
are always labeled (green text) but underlined if they show a significant
difference. Also shown is the variant structure (magenta) superimposed
with cluster 1 of each legacy reference (shades of green) and K268R
(shades of blue) MD simulation.</p></figcaption></figure><figure class="fig xbox font-sm" id="fig10"><h3 class="obj_head">10.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332655_ao5c02316_0010.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/38a9/12332655/4e69dba32562/ao5c02316_0010.jpg" loading="lazy" id="gr10" height="491" width="766" alt="10"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Slow variant R64Q+I114T+K268R cluster
1 structure mapped with values
of delta-RMSF, delta-hydrogen bonding, delta-BC, delta-CC, and delta-EC
(variant minus rapid average). The color intensities correspond to
the respective heatmaps (<a href="#fig5" class="usa-link">Figure 
</a>). Residues with the largest differences from the rapid
average (top and bottom 5%: top and bottom 14 residues) are shown
with stick representation. The residues of the top and bottom 14 hydrogen
bond pair are connected with lines. Red represents increased values
compared to the rapid average, while blue represents decreased values.
Only residues with significant differences from the rapid systems
are labeled, except for the residue variations which are always labeled
(green text) but underlined if they show a significant difference.
Also shown is the variant structure (magenta) superimposed with cluster
1 of each legacy reference (green) and K268R (blue) MD simulation.</p></figcaption></figure><section id="sec5.1"><h3 class="pmc_sec_title">R64Q+K268R</h3>
<p>Significant changes from the rapid systems
observed in this variant were primarily located in domain I, the interdomain
region, and the C-terminal tail residue R277 (<a href="#fig6" class="usa-link">Figure 
</a>). It is known that the critical hydrogen
bond interactions of residue pairs E38-R64 and N41-R64 are weakened
when R64 is substituted with Q64, resulting in reduced protein thermostability,
which may alter the conformation and dynamics of the domain I tertiary
structure.<em>
,
</em> Our hydrogen bond calculations
confirmed this decrease (<a href="#fig5" class="usa-link">Figure 
</a>b,c).</p>
<p>Notably, the RMSF increase in domain I corresponds to multiple hydrogen
bond changes within the same domain I. Specifically, significant hydrogen
bond decreases of domain I residues E38, N41, A47, Q64, and G65 explain
the RMSF increases observed in residues N41, M42, A47, and M48 (<a href="#fig6" class="usa-link">Figure 
</a>). Additionally,
the changes in hydrogen bonding involving the C-terminal tail (residue
pairs E264-R277, E264-L279, and S274-Q64) correspond to increased
RMSF at the start of the C-terminal tail (L275 to V280). This may
disrupt hydrophobic interactions among residues L40, I273, L275, and
L279, which are in close proximity to one another. Consequently, all
of this could influence the hydrogen bonding patterns of interdomain
residue pairs L209-S215, S215-T219, and S216-T220 (<a href="#fig6" class="usa-link">Figures 
</a> and b).</p>
<p>Although clustering analysis showed no significant interdomain
conformational change, the average minimum distance between the putative
isoniazid-binding residue F217 and the catalytic residues H107 and
D122 significantly decreased (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S11</a>). Conversely, the distance between binding residues F217 and L288
significantly increased, which could affect ligand binding and catalysis.</p>
<p>Regarding the DRN, the substituted Q64 residue showed significantly
decreased <em>BC</em> and <em>CC</em> values (<a href="#fig6" class="usa-link">Figure 
</a>), likely due to
the substantial loss of hydrogen bonds. Furthermore, the catalytic
residue D122, which ranked in the top 5% in the reference rapid system
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S8</a>), and the putative isoniazid-binding
residues S125 and F217 showed significantly decreased <em>BC</em> values, indicating that these key residues were less frequently
involved in the shortest paths connecting other residues in this variant.
Additionally, D122 and S125 showed significantly reduced <em>EC</em> and <em>CC</em> values, respectively (<a href="#fig6" class="usa-link">Figure 
</a>), suggesting an altered inter-residue communication
network that could impact catalysis and ligand binding. Notably, the
residues with significantly decreased <em>EC</em> values formed
a cluster within the catalytic site (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S12</a>), including the important residue N72. In contrast, residues with
significantly increased EC values clustered within the interdomain
region and around residue E38. These findings suggest a functional
reorganization of the network, marked by reduced communication within
the catalytic site and enhanced connectivity and influence within
the interdomain region and at residue E38potentially as a
compensatory mechanism for the decreased connectivity elsewhere.</p>
<p>Overall, the slow acetylation phenotype of this variant is likely
driven by the loss of the critical E38-R64 and N41-R64 hydrogen bond
interactions. This disruption triggers widespread changes in hydrogen
bonding across domain I, the interdomain region, and the C-terminal
tail, leading to changes in residue communication networks involving
active site residues. These effects are accompanied by significantly
decreased distances between the putative isoniazid-binding residue
F217 and the catalytic residues H107 and D122. This is further supported
by the decreased <em>BC</em> values of catalytic D122 and
putative isoniazid-binding residues S125 and F217, as well as decreased <em>EC</em> values of N72, D122 and the active site loop residue
G124. Collectively, these changes may impair ligand binding and catalysis.</p></section><section id="sec5.2"><h3 class="pmc_sec_title">I114T+K268R</h3>
<p>Our delta calculations showed the least
amount of change for this variant compared to the other slow acetylators
(<a href="#fig5" class="usa-link">Figure 
</a>). Significant
differences from the rapid systems were primarily observed in domains
II and III. Delta-RMSF calculations revealed slightly increased fluctuations
at residues G104 and M105, while residue D115 became slightly more
rigid. Hydrogen bonding decreased significantly for residue pairs
V146-R166 and D251-R166, while it increased for Q145-R166, indicating
that R166 traded hydrogen bonds with V146 for those with Q145 (<a href="#fig5" class="usa-link">Figure 
</a>c). Additionally,
increased hydrogen bond formation was observed in residue pairs S125-S127
and S127-Q130.</p>
<p>In the 2PFR structure, the side chain of residue
I114 forms hydrophobic interactions with helix α2 residues L21
and L24 in domain I. According to Walraven
et al. substituting the hydrophobic isoleucine
with a polar, hydrophilic threonine could weaken these interactions.
Consistent with this, we observed decreased <em>EC</em> values
on helix α2 from residue T25 to Q31, likely due to the disruption
of hydrophobic interactions with L21 and L24 caused by the I114T substitution.
This disruption may also explain why helix α2 residue L28 shows
a significant decrease in <em>BC</em> (<a href="#fig7" class="usa-link">Figure 
</a>). Conversely, the substituted residue T114
exhibits a significant increase in <em>EC</em> value, along
with V112 (<a href="#fig7" class="usa-link">Figure 
</a>). T114 has also increased <em>CC</em> relative to rapid
reference systems, as do residues D115, R117, I120, S125, and G126.
These DRN changes in and around the active site loop highlight the
alterations in residue communication networks, which may be linked
to increased hydrogen bonding of active site loop residue pairs S127-Q130
and S125-S127 (<a href="#fig5" class="usa-link">Figure 
</a>b). Consequently, this leads to a cluster of residues with significantly
increased <em>EC</em> values, including active site loop residue
S128, C-terminal tail residue I290, and antiparallel beta-sheet residues
V231, C233, and R242 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S12</a>). Furthermore,
residues I189, L161, and Y190 exhibited significantly decreased values
in <em>BC</em>, <em>CC</em>, <em>and EC</em>, respectively, possibly linked to the significant hydrogen bond
decreases of the residue pair Y190-L161 and residue Q187 (<a href="#fig7" class="usa-link">Figures 
</a> and b,c).</p>
<figure class="fig xbox font-sm" id="fig7"><h4 class="obj_head">7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332655_ao5c02316_0007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/38a9/12332655/d1ee27ac82f0/ao5c02316_0007.jpg" loading="lazy" id="gr7" height="453" width="766" alt="7"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Slow variant I114T+K268R cluster 1 structure mapped with
values
of delta-RMSF, delta-hydrogen bonding, delta-BC, delta-CC, and delta-EC
(variant minus rapid average). The color intensities correspond to
the respective heatmaps (<a href="#fig5" class="usa-link">Figure 
</a>). Residues with the largest differences from the rapid
average (top and bottom 5%: top and bottom 14 residues) are shown
with stick representation. The residues of the top and bottom 14 hydrogen
bond residue pairs are connected with lines. Red color represents
increased values compared to the rapid average, while blue color represents
decreased values. Only residues with significant differences from
the rapid systems are labeled, except for the residue variations which
are always labeled (green text) but underlined if they show a significant
difference. Also shown is the variant structure (magenta) superimposed
with cluster 1 of each legacy reference (shades of green) and K268R
(shades of blue) MD simulation.</p></figcaption></figure><p>The proposed cause of the decreased activity is
the altered hydrogen
bonding pattern observed in comparison to the rapid systems, especially
within the active site loop. Additionally, we identified significant
changes in DRN metrics for residues surrounding the active site loop,
indicating notable shifts in the residue communication network. Collectively,
these alterations may impact the catalytic residue D122 and binding
residues in the active site loop. This hypothesis is further supported
by the significant increases in average distance between the active
site loop binding residue S125 and catalytic residues acetyl-C68,
H107, and D122 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S11</a>).</p></section><section id="sec5.3"><h3 class="pmc_sec_title">D122N+K268R</h3>
<p>Our delta calculations demonstrated that
introducing the D122N variation into the fast acetylator NAT2 protein
resulted in several significant changes (<a href="#fig5" class="usa-link">Figure 
</a>). The active site loop residue D122 is a
catalytic residue. A functional study of the D122N variant demonstrated
reduced protein levels and enzymatic activity. As expected, the largest changes from the rapid systems
were observed around the D122N residue variation (<a href="#fig8" class="usa-link">Figure 
</a>). N122 experienced a 274.5%
reduction in hydrogen bond frequency compared to the rapid average
D122 residue value (<a href="#fig5" class="usa-link">Figure 
</a>c). Near the D122N variation, residue pairs L108-N72, N122-N72,
N122-H107, N122-G124, and N122-Y190 showed the largest decreases in
hydrogen bond frequency among all variants, while residue pairs A123-Q31,
G126-S129, and S129–S125 exhibited the largest increases (<a href="#fig5" class="usa-link">Figure 
</a>b). Interestingly,
residue A123 had the largest hydrogen bonding increase of any variant
(<a href="#fig5" class="usa-link">Figure 
</a>c), forming
a pair with Q31 that had a 75% hydrogen bond frequency compared to
0% in the rapid systems. This may explain the structural shift of
residue A123 observed in the clustering analysis (<a href="#fig8" class="usa-link">Figure 
</a>). These many significant hydrogen
bond changes in the active site loop likely result in C-terminal residue
I290 losing hydrogen bonds with S127 but gaining them with S128 (<a href="#fig5" class="usa-link">Figure 
</a>b). Alternatively,
the interdomain has significant hydrogen bond changes of residue pairs
L209-S215, S215-T219, and F217-H43, possibly linked to a major interdomain
structural difference seen in structural clustering (<a href="#fig8" class="usa-link">Figure 
</a>), which may affect substrate/cofactor
active site entry/exit and binding. Furthermore, interdomain residues
Q210 and T211 show significantly increased RMSF (<a href="#fig8" class="usa-link">Figure 
</a>).</p>
<figure class="fig xbox font-sm" id="fig8"><h4 class="obj_head">8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332655_ao5c02316_0008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/38a9/12332655/0c005536d997/ao5c02316_0008.jpg" loading="lazy" id="gr8" height="472" width="766" alt="8"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Slow variant D122N+K268R
cluster 1 structure mapped with values
of delta-RMSF, delta-hydrogen bonding, delta-BC, delta-CC, and delta-EC
(variant minus rapid average). The color intensities correspond to
the respective heatmaps (<a href="#fig5" class="usa-link">Figure 
</a>). Residues with the largest differences from the rapid
average (top and bottom 5%: top and bottom 14 residues) are shown
with stick representation. The residues of the top and bottom 14 hydrogen
bond residue pairs are connected with lines. Red color represents
increased values compared to the rapid average, while blue color represents
decreased values. Only residues with significant differences from
the rapid systems are labeled, except for the residue variations which
are always labeled (green text) but underlined if they show a significant
difference. Also shown is the variant structure (magenta) superimposed
with cluster 1 of each legacy reference (shades of green) and K268R
(shades of blue) MD simulation.</p></figcaption></figure><p>The largest changes in <em>BC</em> occurred
in the active
site loop (<a href="#fig5" class="usa-link">Figure 
</a>d), while several residues around the catalytic/active site showed
substantial decreases in <em>EC</em>, including W67, acetyl-C68,
L69, V71, N72, H107, L108, L109, N122, A123, and G124 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figures S12</a> and f). These
significant DRN changes highlight the potential of DRN in predicting
deleterious residue variations. The <em>BC</em> decreases
of residues N72, G124, S127, and I290, all found in the reference
top 5% <em>BC</em> residue pathway (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S8</a>), very likely affect the communication between the catalytic
site, active site loop, and domain I. EC assesses the connectivity
of residues to other well-connected residues. Notably, catalytic D122
showed the largest rapid system hydrogen bonding frequency, interacting
with six different residues. The decreased <em>EC</em> observed
in the catalytic site of this variant is likely due to a reduction
in the hydrogen bonding involving residue N122 (<a href="#fig5" class="usa-link">Figure 
</a>c), particularly with the other highly connected
hydrogen bonding residue N72 (<a href="#fig5" class="usa-link">Figure 
</a>b,c). Additionally, N72 displayed significantly reduced
values in both <em>BC</em> and <em>CC</em> metrics,
while the catalytic acetyl-C68 showed pronounced decreased <em>CC</em> and <em>EC</em> values.</p>
<p>The slow acetylation
phenotype of this variant is likely due to
the loss of ionic interactions between catalytic residues N122 and
H107,<em>
,
</em> as well as the substantial reduction in
hydrogen bonding involving N122, especially with catalytic H107 and
residue N72. These interactions typically play a crucial role in stabilizing
the catalytic triad. In addition, the inter-residue communication
within the catalytic site is significantly weakened, as indicated
by the marked decreases in <em>EC</em> and <em>CC</em>.</p></section><section id="sec5.4"><h3 class="pmc_sec_title">R197Q+K268R</h3>
<p>As it is observed in the I114T+K268R variant,
our delta calculations revealed minimal changes compared to the rapid
acetylator NAT2 proteins. The R197Q variation causes only a slight
decrease in thermostability<em>
−
,
</em> due to what is thought to be a loss of relatively weak electrostatic
interactions of residue Q197 with M105 and E195. Also, Walraven et al. suggested
that hydrogen bonding between R197 and E195 might be possible but
would likely not contribute to tertiary structure stability like the
M105 interaction. We found that the rapid average E195-R197 residue
pair had a hydrogen bond frequency of 18%, which decreased by only
4.5% in the R197Q variant. However, the substituted residue Q197 lost
a considerable 47.5% hydrogen bond frequency (<a href="#fig5" class="usa-link">Figure 
</a>c), with residue pair D201-Q197 being the
main contributor to the loss (<a href="#fig5" class="usa-link">Figure 
</a>b). On the other hand, residue pair T198-D201 had a
significant increase in hydrogen bond frequency. These changes may
affect helix α5.</p>
<p>Interestingly, the most change from the
rapid average of variant R197Q+K268R occurred on the opposite side
of the enzyme relative to the R197Q residue substitution: on antiparallel
beta-sheets β12→β15, residue T219 (interdomain),
and residue I273 (start of the C-terminal tail) (<a href="#fig9" class="usa-link">Figure 
</a>). The M105 electrostatic interaction
loss and Q197 hydrogen bonding changes caused by the R197Q variation
may travel through helix α5 and the rest of the interdomain
to beta-sheet β12 to induce the abovementioned changes, possibly
linked to the significant RMSF increase of helix α5 residues
and subsequent interdomain residues I218 and T219 (<a href="#fig9" class="usa-link">Figure 
</a>). In addition, the clustering
analysis shows an interdomain conformation change (<a href="#fig9" class="usa-link">Figure 
</a>). The altered hydrogen bonding
between helix α2 and beta-sheet β12 (Q226-R33) as well
as across the antiparallel beta-sheets β12→β15
(L225-Y232, Y241-E254, and E254-T240) (<a href="#fig9" class="usa-link">Figures 
</a> and b) corresponds
to many DRN changes within these antiparallel beta-sheets. Specifically,
there are significant <em>CC</em> changes in residues <em>F</em>222, Q226, T227, P228, V231, T240, and L252; significantly
increased <em>BC</em> in residues <em>F</em>222, V231,
Y232, Y241, R242, and F255; and significantly increased <em>EC</em> in residues <em>F</em>222, C233, L234, G236, and T240 (<a href="#fig9" class="usa-link">Figure 
</a>).</p>
<figure class="fig xbox font-sm" id="fig9"><h4 class="obj_head">9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332655_ao5c02316_0009.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/38a9/12332655/901ec9b9711c/ao5c02316_0009.jpg" loading="lazy" id="gr9" height="512" width="766" alt="9"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Slow variant R197Q+K268R
cluster 1 structure mapped with values
of delta-RMSF, delta-hydrogen bonding, delta-BC, delta-CC, and delta-EC
(variant minus rapid average). The color intensities correspond to
the respective heatmaps (<a href="#fig5" class="usa-link">Figure 
</a>). Residues with the largest differences from the rapid
average (top and bottom 5%: top and bottom 14 residues) are shown
with stick representation. The residues of the top and bottom 14 hydrogen
bond residue pairs are connected with lines. Red color represents
increased values compared to the rapid average, while blue color represents
decreased values. Only residues with significant differences from
the rapid systems are labeled, except for the residue variations which
are always labeled (green text) but underlined if they show a significant
difference. Also shown is the variant structure (magenta) superimposed
with cluster 1 of each legacy reference (shades of green) and K268R
(shades of blue) MD simulation.</p></figcaption></figure><p>Overall, the reduced acetylation phenotype of this
variant is likely
due to the structural change in the interdomain region, as indicated
by the clustering analysis (<a href="#fig9" class="usa-link">Figure 
</a>). Notably, the side chain of the putative isoniazid-binding
residue F217 is oriented in the opposite direction compared to the
rapid replicates. Additionally, catalytic residue D122 shows significantly
decreased <em>BC</em>, suggesting diminished inter-residue
communication at this important site. This aligns with the significantly
reduced average distance between binding residue F217 and catalytic
residues H107 and D122 (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S11</a>). Furthermore,
the distance between binding residues F217 and L288 is significantly
increased. Together, these changes may impact ligand binding and catalysis.</p></section><section id="sec5.5"><h3 class="pmc_sec_title">R64Q+I114T+K268R</h3>
<p>Both R64Q+K268R and R64Q+I114T+K268R
variants share the R64Q residue variation, which is known to disrupt
critical E38-R64 and N41-R64 hydrogen bond interactions, resulting
in reduced protein thermostability, which may alter the conformation
and dynamics of the domain I tertiary structure.<em>
,
</em> Compared to the R64Q+K268R variant, the R64Q+I114T+K268R variant
induced a greater number of hydrogen bond changes within domain I
but fewer interdomain hydrogen bond changes (<a href="#fig5" class="usa-link">Figure 
</a>b,c). Interestingly, both variants exhibit
a significantly lower summed total enzyme delta-hydrogen bond frequency
compared to the rapid average (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S13</a>), likely due to the loss of E38-R64 and N41-R64 hydrogen bonds (<a href="#fig5" class="usa-link">Figure 
</a>b,c). Furthermore,
the more pronounced negative total enzyme hydrogen bond frequency
observed in the R64Q+I114T+K268R variant, compared to R64Q+K268R,
is attributed to decreased hydrogen bonding of residue pairs Q168-R242
and I290-R242 (<a href="#fig5" class="usa-link">Figure 
</a>b) which is not seen in the R64Q+K268R variant.</p>
<p>However, here,
the largest amount of significant change from the rapid systems was
found in the active site loop (<a href="#fig10" class="usa-link">Figure 
</a>), particularly
residues G126 and S127, which both showed significantly decreased <em>BC</em> and <em>CC</em> values. Residue S127 is found
in the reference top 5% <em>BC</em> residue pathway (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S8</a>); therefore, the large decrease in <em>BC</em> of S127 in this variant may affect communication between
the catalytic site, active site loop, and domain I. Also, in the active
site loop, residues G124→G126 had significantly increased RMSF,
residue G124 showed significantly decreased <em>EC</em>, and
residue S127 exhibited significantly increased hydrogen bonding with
C-terminal residue I290. The significant changes of hydrogen bond
frequency involving C-terminal residue I290 (residue pairs I290-S127
and I290-R242) correspond to a conformational change of the C-terminal
tail, with residue S287 blocking more of the active site entrance
than the rapid systems and putative isoniazid-binding residue L288
in a different position compared to the rapid systems (<a href="#fig10" class="usa-link">Figure 
</a>). This may affect substrate/cofactor
active site entry/exit and binding.</p>
<p>The slow acetylation phenotype of this variant
is likely due to
the combined effects of the R64Q variation, which disrupts critical
E38-R64 and N41-R64 hydrogen bond interactions, and the I114T variation,
which appears to influence the active site loop. These changes result
in significantly decreased DRN values for active site loop residues
G124, G126, S127, and S128, potentially impacting substrate and/or
cofactor binding. Additionally, the significantly decreased <em>BC</em> value of important residue N72 may impair the catalytic
site functionality. Furthermore, the conformational change of the
C-terminal tail could hinder substrate/cofactor entry, exit, or binding
at the active site.</p></section></section><section id="sec6"><h2 class="pmc_sec_title">Conclusions</h2>
<p>In general, understanding the atomic-level
effect of variations
on the 3D structures of proteins responsible for drug metabolism (pharmacogenes)
is of utmost importance to develop population-specific pharmacogenomic
approaches. In this context, this study focused on deciphering the
structural consequences of known variations of NAT2 that shift the
enzyme function from rapid to slow acetylation. This is particularly
important because NAT2 is a key metabolizing enzyme of the TB drug,
isoniazid.</p>
<p>Here, we presented a computational framework that
can be applied
to other pharmacogenes to investigate the atomic-level effects of
variations, as we have recently demonstrated. Our approach is based on analyzing the metric differences (delta
calculations) between a baseline system and variant systems, dynamically
over MD simulations. In this study, we showed that while two rapid
acetylator NAT2 proteins with K or R at position 268 have only slight
differences overall in RMSF, hydrogen bonding, and DRN metric values,
the NAT2 variants with a known clinical “slow acetylator”
phenotype present drastic changes compared to the rapid systems. These
changes include disrupted hydrogen bonds, highly localized structural
and dynamic changes, and altered inter-residue communication networks.</p>
<p>In the R64Q+K268R variant, the loss of critical E38-R64 and N41-R64
hydrogen bond interactions likely triggers widespread hydrogen bond
changes in domain I, the interdomain, and the C-terminal tail. This
leads to decreased <em>BC</em> of catalytic D122 and putative
isoniazid-binding residues S125 and F217, as well as decreased <em>EC</em> of N72, D122, and active site loop residue G124, ultimately
affecting ligand binding and catalysis.</p>
<p>In the I114T+K268R variant,
extensive hydrogen bond changes coupled
with residue network alterations, especially in the active site loop,
likely disrupt catalytic D122 and essential binding residues.</p>
<p>For the D122N+K268R variant, the loss of ionic interactions between
catalytic N122 and H107, along with a significant reduction in N122
hydrogen bonds, particularly with catalytic H107 and residue N72,
destabilizes the catalytic triad and reduces inter-residue communication,
as evidenced by substantial <em>EC</em> and <em>CC</em> decreases.</p>
<p>R197Q+K268R exhibits reduced acetylation due to
structural changes
in the interdomain, including a reorientation of the F217 binding
residue side chain, coupled with significantly decreased <em>BC</em> of catalytic D122, further impairing inter-residue communication.</p>
<p>Finally, in the R64Q+I114T+K268R variant, the combined effects
of R64Q disrupting E38-R64 and N41-R64 hydrogen bonds and I114T affecting
the active site loop led to the significantly decreased DRN values
of active site loop residues G124, G126, S127, and S128, potentially
compromising substrate and cofactor binding.</p>
<p>Overall, we demonstrated
how introducing an additional residue
substitution into rapid acetylation enzyme systems may result in a
shift toward slow acetylation. Notably, all but one of these substitutions
are located far from the active site, suggesting that their effects
are allosteric and mediated through local, rather than direct, structural
changes. This concept of dynamic allostery, as opposed to traditional
conformational allostery, was first introduced by Cooper et al. and has since been further elaborated by others.<em>
−

</em> In our recent study on CYP3A4, we observed
a similar pattern of allosteric behavior, supporting the idea that
the functional impact of variants in pharmacogenes tends to be subtle
and localized rather than global. In both studies, these local changes
are evident through alterations in the RMSF, hydrogen bonding patterns,
and residue network properties.</p>
<p>Our future work aims to elucidate
mechanisms underlying the remaining
slow acetylator variants, which may help
us assess the accuracy of the metrics used in this study. Expanding
the sample size to include these and other known variants will strengthen
the robustness and generalizability of our findings.</p></section><section id="sec23"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="sifile1"><div class="media p"><div class="caption">
<a href="/articles/instance/12332655/bin/ao5c02316_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">ao5c02316_si_001.pdf</a><sup> (5.1MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors appreciate Rhodes University,
South
Africa, from which this work was conducted. The authors thank Dr.
Sotiria Boukouvala, Prof. Kevin Lobb and Dr. Birgit Schoeberl for
their suggestions. Authors also thank the Centre for High Performance
Computing (CHPC), Cape Town, South Africa, for providing computing
resources.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Glossary</h2>
<h3>Abbreviations</h3>
<dl class="def-list" id="dl1">
<dt>BC</dt>
<dd><p>averaged
betweenness centrality</p></dd>
<dt>CC</dt>
<dd><p>averaged closeness
centrality</p></dd>
<dt>DRN</dt>
<dd><p>dynamic
residue network</p></dd>
<dt>EC</dt>
<dd><p>averaged eigenvector centrality</p></dd>
<dt>hbond</dt>
<dd><p>hydrogen bond</p></dd>
<dt>MD</dt>
<dd><p>molecular dynamics</p></dd>
<dt>NAT2</dt>
<dd><p>arylamine N-acetyltransferase 2</p></dd>
<dt>
<em>R</em>
<sub>g</sub>
</dt>
<dd><p>radius of gyration</p></dd>
<dt>RMSD</dt>
<dd><p>root-mean-square deviation</p></dd>
<dt>RMSF</dt>
<dd><p>root-mean-square fluctuation</p></dd>
</dl></section><section id="notes1"><p>The data underlying
this study are openly available in Zenodo at <a href="https://doi.org/10.5281/zenodo.15145541" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.5281/zenodo.15145541</a>. Raw data files, delta calculations, and PyMOL sessions are available.</p></section><section id="notes2"><p>The Supporting
Information is
available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsomega.5c02316?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acsomega.5c02316</a>.</p>
<ul id="silist" class="list" style="list-style-type:none"><li><p>Table S1: list of enzyme systems selected for the study;
Table S2: number of frames in molecular dynamics trajectory of top
three conformational clusters per system; Figure S1: molecular dynamics
simulation RMSD line plot; Figure S2: molecular dynamics simulation
RMSD violin plot; Figure S3: molecular dynamics simulation radius
of gyration (<em>R</em>
<sub>g</sub>) line plot; Figure S4:
molecular dynamics simulation radius of gyration (<em>R</em>
<sub>g</sub>) violin plot; Figure S5: multiple sequence alignment
of the UniProt sequences of all known experimentally solved NAT structures;
Figure S6: top three clusters of reference MD simulations superimposed;
Figure S7: reference VS K268R delta heatmaps; Figure S8: reference
top 5% BC, CC, and EC residues shown as spheres; Figure S9: delta
heatmaps using reference average as baseline system to compare with
all variants; Figure S10: delta heatmaps using K268R average as baseline
system to compare with all variants; Figure S11: rapid vs slow delta
heatmap of average minimum distance between active site residues;
Figure S12: significant delta EC residues (system minus rapid average)
shown as spheres on slow variant structures; Figure S13: bar plot
of total-enzyme delta hbond frequency (system minus rapid average)
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c02316/suppl_file/ao5c02316_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p></li></ul></section><section id="notes3"><p>Conceptualization,
W.V. and Ö.T.B.; formal analysis, W.V and Ö.T.B.; methodology,
W.V. and Ö.T.B.; project administration, Ö.T.B.; resources,
Ö.T.B.; validation, Ö.T.B.; visualization, W.V.; writing–review
and editing, W.V. and Ö.T.B. Both authors have read and agreed
to the published version of the manuscript.</p></section><section id="notes4"><p>This research
was supported by the Grants, Innovation and Product Development Unit
of the South African Medical Research Council, with funds received
from Novartis and GSK R&amp;D (Grant #GSKNVS1/202101/002) under the
Project Africa Genomic Research Approach for Diversity and Optimizing
Therapeutics (GRADIENT) program. W.V. is funded as a postdoctoral
fellow by the National Research Foundation of South Africa (Grant
Number: 138566) and GRADIENT. Ö.T.B. thanks National Institute
for Theoretical and Computational Sciences (NITheCS) for Women's
writing
retreat award. The contents of this publication are solely the responsibility
of the authors and do not necessarily represent the official views
of the funders.</p></section><section id="notes6"><p>The human NAT2 PDB protein
structure (PDB ID <a href="https://www.rcsb.org/structure/2PFR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2PFR</a>) was downloaded from the RCSB PDB Web site (<a href="https://www.rcsb.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.rcsb.org/</a>). The ESPript
Web server (<a href="https://espript.ibcp.fr/ESPript/ESPript/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://espript.ibcp.fr/ESPript/ESPript/</a>) was used to map secondary structure labels to enzyme sequence.
PyMOL 2.5 (free to download at <a href="https://www.pymol.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.pymol.org</a>) was used to add the acetyl group to residue C68, to add missing
atoms to residues, for enzyme residue mutation, and for enzyme structure
mapping. The H++ Web server (<a href="http://biophysics.cs.vt.edu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://biophysics.cs.vt.edu/</a>) was used to protonate the protein
at a pH of 7.4. The programs antechamber, parmchk2, prepgen, tleap,
and acpype of the AmberTools18 package were used to setup the molecular
dynamics systems. AmberTools is a free, useful standalone package
(<a href="https://ambermd.org/Amber18-get.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ambermd.org/Amber18-get.html</a>). Production runs were executed using GROMACS 2020.1 (<a href="https://manual.gromacs.org/documentation/2022.1/download.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://manual.gromacs.org/documentation/2022.1/download.html</a>), which is free to download. VMD version 1.9.3 (<a href="https://www.ks.uiuc.edu/Research/vmd/vmd-1.9.3/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ks.uiuc.edu/Research/vmd/vmd-1.9.3/</a>) was used for visualization of MD simulations. GROMACS 2020.1 was
also used to RMSD, RSMF, and <em>R</em>
<sub>g</sub>. The “hbond”
program of the AmberTools18 package was used to calculate hydrogen
bonding frequencies. Python 3.0 (free at <a href="https://www.python.org/downloads/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.python.org/downloads/</a>) was used to calculate per-residue delta values (variant minus baseline
system). MD-TASK (freely available at <a href="https://md-task.readthedocs.io/en/latest/home.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://md-task.readthedocs.io/en/latest/home.html</a>) was used for DRN calculations. Seaborn (freely installed: <a href="https://seaborn.pydata.org/installing.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://seaborn.pydata.org/installing.html</a>) was used to generate heatmaps. Conformational enzyme structure
clustering was performed with GROMACS 2020.1. MD simulations will
be made available upon request.</p></section><section id="notes5"><p>The authors
declare no competing financial interest.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>Motta I., Calcagno A., Bonora S.. Pharmacokinetics and pharmacogenetics
of anti-tubercular drugs: a tool for treatment optimization? Expert Opin. Drug Metab. Toxicol. 2018;14:59–82. doi: 10.1080/17425255.2018.1416093.</cite> [<a href="https://doi.org/10.1080/17425255.2018.1416093" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29226732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Expert%20Opin.%20Drug%20Metab.%20Toxicol.&amp;title=Pharmacokinetics%20and%20pharmacogenetics%20of%20anti-tubercular%20drugs:%20a%20tool%20for%20treatment%20optimization?&amp;author=I.%20Motta&amp;author=A.%20Calcagno&amp;author=S.%20Bonora&amp;volume=14&amp;publication_year=2018&amp;pages=59-82&amp;pmid=29226732&amp;doi=10.1080/17425255.2018.1416093&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>Perwitasari D. A., Atthobari J., Wilffert B.. Pharmacogenetics of isoniazid-induced
hepatotoxicity. Drug Metab. Rev. 2015;47:222–228. doi: 10.3109/03602532.2014.984070.</cite> [<a href="https://doi.org/10.3109/03602532.2014.984070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26095714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Metab.%20Rev.&amp;title=Pharmacogenetics%20of%20isoniazid-induced%20hepatotoxicity&amp;author=D.%20A.%20Perwitasari&amp;author=J.%20Atthobari&amp;author=B.%20Wilffert&amp;volume=47&amp;publication_year=2015&amp;pages=222-228&amp;pmid=26095714&amp;doi=10.3109/03602532.2014.984070&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Matsumoto T., Ohno M., Azuma J.. Future of
pharmacogenetics-based
therapy for tuberculosis. Pharmacogenomics. 2014;15:601–607. doi: 10.2217/pgs.14.38.</cite> [<a href="https://doi.org/10.2217/pgs.14.38" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24798717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenomics&amp;title=Future%20of%20pharmacogenetics-based%20therapy%20for%20tuberculosis&amp;author=T.%20Matsumoto&amp;author=M.%20Ohno&amp;author=J.%20Azuma&amp;volume=15&amp;publication_year=2014&amp;pages=601-607&amp;pmid=24798717&amp;doi=10.2217/pgs.14.38&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>McDonagh E. M.. et al. PharmGKB summary: very
important pharmacogene information for N-acetyltransferase
2. Pharmacogenet. Genomics. 2014;24:409–425. doi: 10.1097/FPC.0000000000000062.</cite> [<a href="https://doi.org/10.1097/FPC.0000000000000062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4109976/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24892773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&amp;title=PharmGKB%20summary:%20very%20important%20pharmacogene%20information%20for%20N-acetyltransferase%202&amp;author=E.%20M.%20McDonagh&amp;volume=24&amp;publication_year=2014&amp;pages=409-425&amp;pmid=24892773&amp;doi=10.1097/FPC.0000000000000062&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Donald P. R.. et al. The influence of dose
and N-acetyltransferase-2 (NAT2) genotype and
phenotype on the pharmacokinetics and pharmacodynamics of isoniazid. Eur. J. Clin. Pharmacol. 2007;63:633–639. doi: 10.1007/s00228-007-0305-5.</cite> [<a href="https://doi.org/10.1007/s00228-007-0305-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17505821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&amp;title=The%20influence%20of%20dose%20and%20N-acetyltransferase-2%20(NAT2)%20genotype%20and%20phenotype%20on%20the%20pharmacokinetics%20and%20pharmacodynamics%20of%20isoniazid&amp;author=P.%20R.%20Donald&amp;volume=63&amp;publication_year=2007&amp;pages=633-639&amp;pmid=17505821&amp;doi=10.1007/s00228-007-0305-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Pasipanodya J. G., Srivastava S., Gumbo T.. Meta-analysis of clinical studies
supports the pharmacokinetic variability hypothesis for acquired drug
resistance and failure of antituberculosis therapy. Clin. Infect. Dis. Off. Publ. Infect. Dis. Soc. Am. 2012;55:169–177. doi: 10.1093/cid/cis353.</cite> [<a href="https://doi.org/10.1093/cid/cis353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3491771/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22467670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Infect.%20Dis.%20Off.%20Publ.%20Infect.%20Dis.%20Soc.%20Am.&amp;title=Meta-analysis%20of%20clinical%20studies%20supports%20the%20pharmacokinetic%20variability%20hypothesis%20for%20acquired%20drug%20resistance%20and%20failure%20of%20antituberculosis%20therapy&amp;author=J.%20G.%20Pasipanodya&amp;author=S.%20Srivastava&amp;author=T.%20Gumbo&amp;volume=55&amp;publication_year=2012&amp;pages=169-177&amp;pmid=22467670&amp;doi=10.1093/cid/cis353&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Chamboko C. R., Veldman W., Tata R. B., Schoeberl B., Tastan Bishop Ö.. Human Cytochrome P450 1, 2, 3 Families
as Pharmacogenes
with Emphases on Their Antimalarial and Antituberculosis Drugs and
Prevalent African Alleles. Int. J. Mol. Sci. 2023;24:3383. doi: 10.3390/ijms24043383.</cite> [<a href="https://doi.org/10.3390/ijms24043383" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9961538/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36834793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Human%20Cytochrome%20P450%201,%202,%203%20Families%20as%20Pharmacogenes%20with%20Emphases%20on%20Their%20Antimalarial%20and%20Antituberculosis%20Drugs%20and%20Prevalent%20African%20Alleles&amp;author=C.%20R.%20Chamboko&amp;author=W.%20Veldman&amp;author=R.%20B.%20Tata&amp;author=B.%20Schoeberl&amp;author=%C3%96.%20Tastan%20Bishop&amp;volume=24&amp;publication_year=2023&amp;pages=3383&amp;pmid=36834793&amp;doi=10.3390/ijms24043383&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Gutiérrez-Virgen J. E.. et al. NAT2
global landscape: Genetic diversity and acetylation statuses
from a systematic review. PLoS One. 2023;18:e0283726.  doi: 10.1371/journal.pone.0283726.</cite> [<a href="https://doi.org/10.1371/journal.pone.0283726" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10079069/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37023111/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=NAT2%20global%20landscape:%20Genetic%20diversity%20and%20acetylation%20statuses%20from%20a%20systematic%20review&amp;author=J.%20E.%20Guti%C3%A9rrez-Virgen&amp;volume=18&amp;publication_year=2023&amp;pages=e0283726&amp;pmid=37023111&amp;doi=10.1371/journal.pone.0283726&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Hein D. W.. et al. Molecular genetics and
epidemiology of the NAT1 and NAT2 acetylation
polymorphisms. Cancer Epidemiol., Biomarkers
Prev. 2000;9:29–42.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/10667461/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol.,%20Biomarkers%20Prev.&amp;title=Molecular%20genetics%20and%20epidemiology%20of%20the%20NAT1%20and%20NAT2%20acetylation%20polymorphisms&amp;author=D.%20W.%20Hein&amp;volume=9&amp;publication_year=2000&amp;pages=29-42&amp;pmid=10667461&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Hein D. W.. N-acetyltransferase
SNPs: emerging concepts serve as a paradigm for understanding complexities
of personalized medicine. Expert Opin. Drug
Metab. Toxicol. 2009;5:353–366. doi: 10.1517/17425250902877698.</cite> [<a href="https://doi.org/10.1517/17425250902877698" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2762189/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19379125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Expert%20Opin.%20Drug%20Metab.%20Toxicol.&amp;title=N-acetyltransferase%20SNPs:%20emerging%20concepts%20serve%20as%20a%20paradigm%20for%20understanding%20complexities%20of%20personalized%20medicine&amp;author=D.%20W.%20Hein&amp;volume=5&amp;publication_year=2009&amp;pages=353-366&amp;pmid=19379125&amp;doi=10.1517/17425250902877698&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Huang Y.-S.. et al. Polymorphism of the N-acetyltransferase 2 gene
as a susceptibility
risk factor for antituberculosis drug-induced hepatitis. Hepatol. Baltim. Md. 2002;35:883–889. doi: 10.1053/jhep.2002.32102.</cite> [<a href="https://doi.org/10.1053/jhep.2002.32102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11915035/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hepatol.%20Baltim.%20Md&amp;title=Polymorphism%20of%20the%20N-acetyltransferase%202%20gene%20as%20a%20susceptibility%20risk%20factor%20for%20antituberculosis%20drug-induced%20hepatitis&amp;author=Y.-S.%20Huang&amp;volume=35&amp;publication_year=2002&amp;pages=883-889&amp;pmid=11915035&amp;doi=10.1053/jhep.2002.32102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Huang Y.-S.. et al. Cytochrome P450 2E1 genotype and the susceptibility to antituberculosis
drug-induced hepatitis. Hepatol. Baltim. Md. 2003;37:924–930. doi: 10.1053/jhep.2003.50144.</cite> [<a href="https://doi.org/10.1053/jhep.2003.50144" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12668988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hepatol.%20Baltim.%20Md&amp;title=Cytochrome%20P450%202E1%20genotype%20and%20the%20susceptibility%20to%20antituberculosis%20drug-induced%20hepatitis&amp;author=Y.-S.%20Huang&amp;volume=37&amp;publication_year=2003&amp;pages=924-930&amp;pmid=12668988&amp;doi=10.1053/jhep.2003.50144&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Abel L.. et al. Genetics of human susceptibility
to active and latent tuberculosis:
present knowledge and future perspectives. Lancet
Infect. Dis. 2018;18:e64–e75. doi: 10.1016/S1473-3099(17)30623-0.</cite> [<a href="https://doi.org/10.1016/S1473-3099(17)30623-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8903186/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29111156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet%20Infect.%20Dis.&amp;title=Genetics%20of%20human%20susceptibility%20to%20active%20and%20latent%20tuberculosis:%20present%20knowledge%20and%20future%20perspectives&amp;author=L.%20Abel&amp;volume=18&amp;publication_year=2018&amp;pages=e64-e75&amp;pmid=29111156&amp;doi=10.1016/S1473-3099(17)30623-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Richardson M.. et al. NAT2 variants and toxicity related to anti-tuberculosis agents: a
systematic review and meta-analysis. Int. J.
Tuberc. Lung Dis. Off. J. Int. Union Tuberc. Lung Dis. 2019;23:293–305. doi: 10.5588/ijtld.18.0324.</cite> [<a href="https://doi.org/10.5588/ijtld.18.0324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6421944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30871660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Tuberc.%20Lung%20Dis.%20Off.%20J.%20Int.%20Union%20Tuberc.%20Lung%20Dis.&amp;title=NAT2%20variants%20and%20toxicity%20related%20to%20anti-tuberculosis%20agents:%20a%20systematic%20review%20and%20meta-analysis&amp;author=M.%20Richardson&amp;volume=23&amp;publication_year=2019&amp;pages=293-305&amp;pmid=30871660&amp;doi=10.5588/ijtld.18.0324&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>Khan S.. et al. Pharmacogenetic association
between NAT2 gene polymorphisms and isoniazid
induced hepatotoxicity: trial sequence meta-analysis as evidence. Biosci. Rep. 2019;39:BSR20180845.  doi: 10.1042/BSR20180845.</cite> [<a href="https://doi.org/10.1042/BSR20180845" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6331676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30509962/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biosci.%20Rep.&amp;title=Pharmacogenetic%20association%20between%20NAT2%20gene%20polymorphisms%20and%20isoniazid%20induced%20hepatotoxicity:%20trial%20sequence%20meta-analysis%20as%20evidence&amp;author=S.%20Khan&amp;volume=39&amp;publication_year=2019&amp;pages=BSR20180845&amp;pmid=30509962&amp;doi=10.1042/BSR20180845&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Tastan
Bishop Ö., Musyoka T. M., Barozi V.. Allostery and Missense
Mutations as Intermittently Linked Promising Aspects of Modern Computational
Drug Discovery. J. Mol. Biol. 2022;434:167610.  doi: 10.1016/j.jmb.2022.167610.</cite> [<a href="https://doi.org/10.1016/j.jmb.2022.167610" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35490897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=Allostery%20and%20Missense%20Mutations%20as%20Intermittently%20Linked%20Promising%20Aspects%20of%20Modern%20Computational%20Drug%20Discovery&amp;author=%C3%96.%20Tastan%20Bishop&amp;author=T.%20M.%20Musyoka&amp;author=V.%20Barozi&amp;volume=434&amp;publication_year=2022&amp;pages=167610&amp;pmid=35490897&amp;doi=10.1016/j.jmb.2022.167610&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17"><cite>
Previous human NAT2 database. <a href="https://nat.mbg.duth.gr/Human_NAT2_alleles.htm" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://nat.mbg.duth.gr/Human_NAT2_alleles.htm</a>.</cite></li>
<li id="ref18"><cite>
PharmVar: New human NAT2 database. <a href="https://www.pharmvar.org/gene/NAT2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.pharmvar.org/gene/NAT2</a>.</cite></li>
<li id="ref19"><cite>
rs1208 RefSNP Report - dbSNP - NCBI. <a href="https://www.ncbi.nlm.nih.gov/snp/rs1208#frequency_tab" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/snp/rs1208#frequency_tab</a>.</cite></li>
<li id="ref20">
<cite>Wu H.. et al. Structural basis of
substrate-binding specificity of human arylamine
N-acetyltransferases. J. Biol. Chem. 2007;282:30189–30197. doi: 10.1074/jbc.M704138200.</cite> [<a href="https://doi.org/10.1074/jbc.M704138200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17656365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Structural%20basis%20of%20substrate-binding%20specificity%20of%20human%20arylamine%20N-acetyltransferases&amp;author=H.%20Wu&amp;volume=282&amp;publication_year=2007&amp;pages=30189-30197&amp;pmid=17656365&amp;doi=10.1074/jbc.M704138200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21"><cite>

Schrödinger, L.
; 
DeLano, W.
. 
PyMOL, 2020.</cite></li>
<li id="ref22">
<cite>Anandakrishnan R., Aguilar B., Onufriev A. V.. H++ 3.0: automating pK prediction
and the preparation of biomolecular structures for atomistic molecular
modeling and simulations. Nucleic Acids Res. 2012;40:W537–541. doi: 10.1093/nar/gks375.</cite> [<a href="https://doi.org/10.1093/nar/gks375" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3394296/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22570416/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=H++%203.0:%20automating%20pK%20prediction%20and%20the%20preparation%20of%20biomolecular%20structures%20for%20atomistic%20molecular%20modeling%20and%20simulations&amp;author=R.%20Anandakrishnan&amp;author=B.%20Aguilar&amp;author=A.%20V.%20Onufriev&amp;volume=40&amp;publication_year=2012&amp;pages=W537-541&amp;pmid=22570416&amp;doi=10.1093/nar/gks375&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Wang H., Vath G. M., Gleason K. J., Hanna P. E., Wagner C. R.. Probing
the mechanism of hamster arylamine N-acetyltransferase 2 acetylation
by active site modification, site-directed mutagenesis, and pre-steady
state and steady state kinetic studies. Biochemistry. 2004;43:8234–8246. doi: 10.1021/bi0497244.</cite> [<a href="https://doi.org/10.1021/bi0497244" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15209520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Probing%20the%20mechanism%20of%20hamster%20arylamine%20N-acetyltransferase%202%20acetylation%20by%20active%20site%20modification,%20site-directed%20mutagenesis,%20and%20pre-steady%20state%20and%20steady%20state%20kinetic%20studies&amp;author=H.%20Wang&amp;author=G.%20M.%20Vath&amp;author=K.%20J.%20Gleason&amp;author=P.%20E.%20Hanna&amp;author=C.%20R.%20Wagner&amp;volume=43&amp;publication_year=2004&amp;pages=8234-8246&amp;pmid=15209520&amp;doi=10.1021/bi0497244&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Wang H., Liu L., Hanna P. E., Wagner C. R.. Catalytic mechanism of hamster arylamine
N-acetyltransferase 2. Biochemistry. 2005;44:11295–11306. doi: 10.1021/bi047564q.</cite> [<a href="https://doi.org/10.1021/bi047564q" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16101314/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Catalytic%20mechanism%20of%20hamster%20arylamine%20N-acetyltransferase%202&amp;author=H.%20Wang&amp;author=L.%20Liu&amp;author=P.%20E.%20Hanna&amp;author=C.%20R.%20Wagner&amp;volume=44&amp;publication_year=2005&amp;pages=11295-11306&amp;pmid=16101314&amp;doi=10.1021/bi047564q&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Maier J. A.. et al. ff14SB: Improving the
Accuracy of Protein Side Chain and Backbone
Parameters from ff99SB. J. Chem. Theory Comput. 2015;11:3696–3713. doi: 10.1021/acs.jctc.5b00255.</cite> [<a href="https://doi.org/10.1021/acs.jctc.5b00255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4821407/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26574453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Theory%20Comput.&amp;title=ff14SB:%20Improving%20the%20Accuracy%20of%20Protein%20Side%20Chain%20and%20Backbone%20Parameters%20from%20ff99SB&amp;author=J.%20A.%20Maier&amp;volume=11&amp;publication_year=2015&amp;pages=3696-3713&amp;pmid=26574453&amp;doi=10.1021/acs.jctc.5b00255&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Abraham M. J.. et al. GROMACS: High performance
molecular simulations through multi-level
parallelism from laptops to supercomputers. SoftwareX. 2015;1–2:19–25. doi: 10.1016/j.softx.2015.06.001.</cite> [<a href="https://doi.org/10.1016/j.softx.2015.06.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=SoftwareX&amp;title=GROMACS:%20High%20performance%20molecular%20simulations%20through%20multi-level%20parallelism%20from%20laptops%20to%20supercomputers&amp;author=M.%20J.%20Abraham&amp;volume=1%E2%80%932&amp;publication_year=2015&amp;pages=19-25&amp;doi=10.1016/j.softx.2015.06.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Case D. A.. et al. AmberTools. J. Chem. Inf. Model. 2023;63:6183–6191. doi: 10.1021/acs.jcim.3c01153.</cite> [<a href="https://doi.org/10.1021/acs.jcim.3c01153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10598796/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37805934/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Inf.%20Model.&amp;title=AmberTools&amp;author=D.%20A.%20Case&amp;volume=63&amp;publication_year=2023&amp;pages=6183-6191&amp;pmid=37805934&amp;doi=10.1021/acs.jcim.3c01153&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Jakalian A., Jack D. B., Bayly C. I.. Fast, efficient
generation of high-quality
atomic charges. AM1-BCC model: II. Parameterization and validation. J. Comput. Chem. 2002;23:1623–1641. doi: 10.1002/jcc.10128.</cite> [<a href="https://doi.org/10.1002/jcc.10128" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12395429/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&amp;title=Fast,%20efficient%20generation%20of%20high-quality%20atomic%20charges.%20AM1-BCC%20model:%20II.%20Parameterization%20and%20validation&amp;author=A.%20Jakalian&amp;author=D.%20B.%20Jack&amp;author=C.%20I.%20Bayly&amp;volume=23&amp;publication_year=2002&amp;pages=1623-1641&amp;pmid=12395429&amp;doi=10.1002/jcc.10128&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>Berendsen H. J. C., Postma J. P. M., van Gunsteren W. F., DiNola A., Haak J. R.. Molecular
dynamics with coupling to an external bath. J. Chem. Phys. 1984;81:3684–3690. doi: 10.1063/1.448118.</cite> [<a href="https://doi.org/10.1063/1.448118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Phys.&amp;title=Molecular%20dynamics%20with%20coupling%20to%20an%20external%20bath&amp;author=H.%20J.%20C.%20Berendsen&amp;author=J.%20P.%20M.%20Postma&amp;author=W.%20F.%20van%20Gunsteren&amp;author=A.%20DiNola&amp;author=J.%20R.%20Haak&amp;volume=81&amp;publication_year=1984&amp;pages=3684-3690&amp;doi=10.1063/1.448118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Parrinello M., Rahman A.. Polymorphic transitions in single crystals: A new molecular
dynamics method. J. Appl. Phys. 1981;52:7182–7190. doi: 10.1063/1.328693.</cite> [<a href="https://doi.org/10.1063/1.328693" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Appl.%20Phys.&amp;title=Polymorphic%20transitions%20in%20single%20crystals:%20A%20new%20molecular%20dynamics%20method&amp;author=M.%20Parrinello&amp;author=A.%20Rahman&amp;volume=52&amp;publication_year=1981&amp;pages=7182-7190&amp;doi=10.1063/1.328693&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>Hess B., Bekker H., Berendsen H. J. C., Fraaije J. G. E. M.. LINCS: A linear
constraint solver for molecular simulations. J. Comput. Chem. 1997;18:1463–1472. doi: 10.1002/(SICI)1096-987X(199709)18:12&amp;#x0003c;1463::AID-JCC4&amp;#x0003e;3.0.CO;2-H.</cite> [<a href="https://doi.org/10.1002/(SICI)1096-987X(199709)18:12&amp;#x0003c;1463::AID-JCC4&amp;#x0003e;3.0.CO;2-H" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&amp;title=LINCS:%20A%20linear%20constraint%20solver%20for%20molecular%20simulations&amp;author=B.%20Hess&amp;author=H.%20Bekker&amp;author=H.%20J.%20C.%20Berendsen&amp;author=J.%20G.%20E.%20M.%20Fraaije&amp;volume=18&amp;publication_year=1997&amp;pages=1463-1472&amp;doi=10.1002/(SICI)1096-987X(199709)18:12&amp;#x0003c;1463::AID-JCC4&amp;#x0003e;3.0.CO;2-H&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>Humphrey W., Dalke A., Schulten K.. VMD: Visual
molecular dynamics. J. Mol. Graph. 1996;14:33–38. doi: 10.1016/0263-7855(96)00018-5.</cite> [<a href="https://doi.org/10.1016/0263-7855(96)00018-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8744570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Graph.&amp;title=VMD:%20Visual%20molecular%20dynamics&amp;author=W.%20Humphrey&amp;author=A.%20Dalke&amp;author=K.%20Schulten&amp;volume=14&amp;publication_year=1996&amp;pages=33-38&amp;pmid=8744570&amp;doi=10.1016/0263-7855(96)00018-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>Roe D. R., Cheatham T. E. I.. PTRAJ and CPPTRAJ: Software for Processing
and Analysis
of Molecular Dynamics Trajectory Data. J. Chem.
Theory Comput. 2013;9:3084–3095. doi: 10.1021/ct400341p.</cite> [<a href="https://doi.org/10.1021/ct400341p" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26583988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Theory%20Comput.&amp;title=PTRAJ%20and%20CPPTRAJ:%20Software%20for%20Processing%20and%20Analysis%20of%20Molecular%20Dynamics%20Trajectory%20Data&amp;author=D.%20R.%20Roe&amp;author=T.%20E.%20I.%20Cheatham&amp;volume=9&amp;publication_year=2013&amp;pages=3084-3095&amp;pmid=26583988&amp;doi=10.1021/ct400341p&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34">
<cite>Brown D. K.. et al. MD-TASK: a software
suite for analyzing molecular dynamics trajectories. Bioinformatics. 2017;33:2768–2771. doi: 10.1093/bioinformatics/btx349.</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx349" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5860072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28575169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=MD-TASK:%20a%20software%20suite%20for%20analyzing%20molecular%20dynamics%20trajectories&amp;author=D.%20K.%20Brown&amp;volume=33&amp;publication_year=2017&amp;pages=2768-2771&amp;pmid=28575169&amp;doi=10.1093/bioinformatics/btx349&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref35">
<cite>Sheik
Amamuddy O., Glenister M., Tshabalala T., Tastan Bishop Ö.. MDM-TASK-web: MD-TASK and MODE-TASK web server for
analyzing protein dynamics. Comput. Struct.
Biotechnol. J. 2021;19:5059–5071. doi: 10.1016/j.csbj.2021.08.043.</cite> [<a href="https://doi.org/10.1016/j.csbj.2021.08.043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8455658/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34589183/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput.%20Struct.%20Biotechnol.%20J.&amp;title=MDM-TASK-web:%20MD-TASK%20and%20MODE-TASK%20web%20server%20for%20analyzing%20protein%20dynamics&amp;author=O.%20Sheik%20Amamuddy&amp;author=M.%20Glenister&amp;author=T.%20Tshabalala&amp;author=%C3%96.%20Tastan%20Bishop&amp;volume=19&amp;publication_year=2021&amp;pages=5059-5071&amp;pmid=34589183&amp;doi=10.1016/j.csbj.2021.08.043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>Negre C. F., Morzan U. N., Hendrickson H. P., Pal R., Lisi G. P., Loria J. P., Rivalta I., Ho J., Batista V. S.. Eigenvector
centrality for characterization of protein allosteric pathways. Proc. Natl. Acad. Sci. U. S. A. 2018;115:E12201–E12208. doi: 10.1073/pnas.1810452115.</cite> [<a href="https://doi.org/10.1073/pnas.1810452115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6310864/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30530700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&amp;title=Eigenvector%20centrality%20for%20characterization%20of%20protein%20allosteric%20pathways&amp;author=C.%20F.%20Negre&amp;author=U.%20N.%20Morzan&amp;author=H.%20P.%20Hendrickson&amp;author=R.%20Pal&amp;author=G.%20P.%20Lisi&amp;volume=115&amp;publication_year=2018&amp;pages=E12201-E12208&amp;pmid=30530700&amp;doi=10.1073/pnas.1810452115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>Botello-Smith W. M., Luo Y.. Robust Determination of Protein Allosteric Signaling Pathways. J. Chem. Theory Comput. 2019;15:2116–2126. doi: 10.1021/acs.jctc.8b01197.</cite> [<a href="https://doi.org/10.1021/acs.jctc.8b01197" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30836000/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Theory%20Comput.&amp;title=Robust%20Determination%20of%20Protein%20Allosteric%20Signaling%20Pathways&amp;author=W.%20M.%20Botello-Smith&amp;author=Y.%20Luo&amp;volume=15&amp;publication_year=2019&amp;pages=2116-2126&amp;pmid=30836000&amp;doi=10.1021/acs.jctc.8b01197&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Nerín-Fonz F.. et al. AlloViz: A tool for the calculation and
visualisation of protein
allosteric communication networks. Comput. Struct.
Biotechnol. J. 2024;23:1938–1944. doi: 10.1016/j.csbj.2024.04.047.</cite> [<a href="https://doi.org/10.1016/j.csbj.2024.04.047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11087696/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38736696/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput.%20Struct.%20Biotechnol.%20J.&amp;title=AlloViz:%20A%20tool%20for%20the%20calculation%20and%20visualisation%20of%20protein%20allosteric%20communication%20networks&amp;author=F.%20Ner%C3%ADn-Fonz&amp;volume=23&amp;publication_year=2024&amp;pages=1938-1944&amp;pmid=38736696&amp;doi=10.1016/j.csbj.2024.04.047&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Chepsiror C., Veldman W., Olotu F., Tastan Bishop Ö.. Allosteric
modulation of Plasmodium falciparum Isoleucyl tRNA synthetase by South
African natural compounds. PLoS One. 2025;20:e0321444.  doi: 10.1371/journal.pone.0321444.</cite> [<a href="https://doi.org/10.1371/journal.pone.0321444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12077802/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40367238/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Allosteric%20modulation%20of%20Plasmodium%20falciparum%20Isoleucyl%20tRNA%20synthetase%20by%20South%20African%20natural%20compounds&amp;author=C.%20Chepsiror&amp;author=W.%20Veldman&amp;author=F.%20Olotu&amp;author=%C3%96.%20Tastan%20Bishop&amp;volume=20&amp;publication_year=2025&amp;pages=e0321444&amp;pmid=40367238&amp;doi=10.1371/journal.pone.0321444&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Waskom M. L.. seaborn:
statistical data visualization. J. Open Source
Softw. 2021;6:3021. doi: 10.21105/joss.03021.</cite> [<a href="https://doi.org/10.21105/joss.03021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Open%20Source%20Softw.&amp;title=seaborn:%20statistical%20data%20visualization&amp;author=M.%20L.%20Waskom&amp;volume=6&amp;publication_year=2021&amp;pages=3021&amp;doi=10.21105/joss.03021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Robert X., Gouet P.. Deciphering key features
in protein structures with the new ENDscript
server. Nucleic Acids Res. 2014;42:W320–W324. doi: 10.1093/nar/gku316.</cite> [<a href="https://doi.org/10.1093/nar/gku316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4086106/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24753421/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Deciphering%20key%20features%20in%20protein%20structures%20with%20the%20new%20ENDscript%20server&amp;author=X.%20Robert&amp;author=P.%20Gouet&amp;volume=42&amp;publication_year=2014&amp;pages=W320-W324&amp;pmid=24753421&amp;doi=10.1093/nar/gku316&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Fullam E.. et al. Divergence of cofactor
recognition across evolution: coenzyme A binding
in a prokaryotic arylamine N-acetyltransferase. J. Mol. Biol. 2008;375:178–191. doi: 10.1016/j.jmb.2007.10.019.</cite> [<a href="https://doi.org/10.1016/j.jmb.2007.10.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18005984/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=Divergence%20of%20cofactor%20recognition%20across%20evolution:%20coenzyme%20A%20binding%20in%20a%20prokaryotic%20arylamine%20N-acetyltransferase&amp;author=E.%20Fullam&amp;volume=375&amp;publication_year=2008&amp;pages=178-191&amp;pmid=18005984&amp;doi=10.1016/j.jmb.2007.10.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Pluvinage B.. et al. The Bacillus anthracis arylamine N-acetyltransferase
((BACAN)­NAT1)
that inactivates sulfamethoxazole, reveals unusual structural features
compared with the other NAT isoenzymes. FEBS
Lett. 2011;585:3947–3952. doi: 10.1016/j.febslet.2011.10.041.</cite> [<a href="https://doi.org/10.1016/j.febslet.2011.10.041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22062153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett.&amp;title=The%20Bacillus%20anthracis%20arylamine%20N-acetyltransferase%20((BACAN)%C2%ADNAT1)%20that%20inactivates%20sulfamethoxazole,%20reveals%20unusual%20structural%20features%20compared%20with%20the%20other%20NAT%20isoenzymes&amp;author=B.%20Pluvinage&amp;volume=585&amp;publication_year=2011&amp;pages=3947-3952&amp;pmid=22062153&amp;doi=10.1016/j.febslet.2011.10.041&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref44">
<cite>Walraven J. M., Zang Y., Trent J. O., Hein D. W.. Structure/Function
Evaluations of Single Nucleotide Polymorphisms in Human N-Acetyltransferase
2. Curr. Drug Metab. 2008;9:471–486. doi: 10.2174/138920008784892065.</cite> [<a href="https://doi.org/10.2174/138920008784892065" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2507886/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18680467/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Drug%20Metab.&amp;title=Structure/Function%20Evaluations%20of%20Single%20Nucleotide%20Polymorphisms%20in%20Human%20N-Acetyltransferase%202&amp;author=J.%20M.%20Walraven&amp;author=Y.%20Zang&amp;author=J.%20O.%20Trent&amp;author=D.%20W.%20Hein&amp;volume=9&amp;publication_year=2008&amp;pages=471-486&amp;pmid=18680467&amp;doi=10.2174/138920008784892065&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref45">
<cite>Kubiak X., Dairou J., Dupret J.-M., Rodrigues-Lima F.. Crystal structure
of arylamine N-acetyltransferases: insights into the mechanisms of
action and substrate selectivity. Expert Opin.
Drug Metab. Toxicol. 2013;9:349–362. doi: 10.1517/17425255.2013.742505.</cite> [<a href="https://doi.org/10.1517/17425255.2013.742505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23289949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Expert%20Opin.%20Drug%20Metab.%20Toxicol.&amp;title=Crystal%20structure%20of%20arylamine%20N-acetyltransferases:%20insights%20into%20the%20mechanisms%20of%20action%20and%20substrate%20selectivity&amp;author=X.%20Kubiak&amp;author=J.%20Dairou&amp;author=J.-M.%20Dupret&amp;author=F.%20Rodrigues-Lima&amp;volume=9&amp;publication_year=2013&amp;pages=349-362&amp;pmid=23289949&amp;doi=10.1517/17425255.2013.742505&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref46">
<cite>Sandy J., Holton S., Fullam E., Sim E., Noble M.. Binding of
the anti-tubercular drug isoniazid to the arylamine N-acetyltransferase
protein from Mycobacterium smegmatis. Protein
Sci. 2005;14:775–782. doi: 10.1110/ps.041163505.</cite> [<a href="https://doi.org/10.1110/ps.041163505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2279269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15722451/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci.&amp;title=Binding%20of%20the%20anti-tubercular%20drug%20isoniazid%20to%20the%20arylamine%20N-acetyltransferase%20protein%20from%20Mycobacterium%20smegmatis&amp;author=J.%20Sandy&amp;author=S.%20Holton&amp;author=E.%20Fullam&amp;author=E.%20Sim&amp;author=M.%20Noble&amp;volume=14&amp;publication_year=2005&amp;pages=775-782&amp;pmid=15722451&amp;doi=10.1110/ps.041163505&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref47">
<cite>Unissa A. N., Sukumar S., Hanna L. E.. The Role of N-Acetyl Transferases
on Isoniazid Resistance from Mycobacterium tuberculosis and Human:
An In Silico Approach. Tuberc. Respir. Dis. 2017;80:255–264. doi: 10.4046/trd.2017.80.3.255.</cite> [<a href="https://doi.org/10.4046/trd.2017.80.3.255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5526952/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28747958/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Tuberc.%20Respir.%20Dis.&amp;title=The%20Role%20of%20N-Acetyl%20Transferases%20on%20Isoniazid%20Resistance%20from%20Mycobacterium%20tuberculosis%20and%20Human:%20An%20In%20Silico%20Approach&amp;author=A.%20N.%20Unissa&amp;author=S.%20Sukumar&amp;author=L.%20E.%20Hanna&amp;volume=80&amp;publication_year=2017&amp;pages=255-264&amp;pmid=28747958&amp;doi=10.4046/trd.2017.80.3.255&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref48">
<cite>Payton M.. et al. Evidence towards the
role of arylamine N-acetyltransferase in Mycobacterium
smegmatis and development of a specific antiserum against the homologous
enzyme of Mycobacterium tuberculosis. Microbiol.
Read. Engl. 2001;147:3295–3302. doi: 10.1099/00221287-147-12-3295.</cite> [<a href="https://doi.org/10.1099/00221287-147-12-3295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11739761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Read.%20Engl.&amp;title=Evidence%20towards%20the%20role%20of%20arylamine%20N-acetyltransferase%20in%20Mycobacterium%20smegmatis%20and%20development%20of%20a%20specific%20antiserum%20against%20the%20homologous%20enzyme%20of%20Mycobacterium%20tuberculosis&amp;author=M.%20Payton&amp;volume=147&amp;publication_year=2001&amp;pages=3295-3302&amp;pmid=11739761&amp;doi=10.1099/00221287-147-12-3295&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref49">
<cite>Atmane N.. et al. The conserved glycine/alanine residue of the active-site loop containing
the putative acetylCoA-binding motif is essential for the overall
structural integrity of Mesorhizobium loti arylamine N-acetyltransferase
1. Biochem. Biophys. Res. Commun. 2007;361:256–262. doi: 10.1016/j.bbrc.2007.07.034.</cite> [<a href="https://doi.org/10.1016/j.bbrc.2007.07.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17658468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;title=The%20conserved%20glycine/alanine%20residue%20of%20the%20active-site%20loop%20containing%20the%20putative%20acetylCoA-binding%20motif%20is%20essential%20for%20the%20overall%20structural%20integrity%20of%20Mesorhizobium%20loti%20arylamine%20N-acetyltransferase%201&amp;author=N.%20Atmane&amp;volume=361&amp;publication_year=2007&amp;pages=256-262&amp;pmid=17658468&amp;doi=10.1016/j.bbrc.2007.07.034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref50">
<cite>Goodfellow G. H., Dupret J. M., Grant D. M.. Identification
of amino acids imparting
acceptor substrate selectivity to human arylamine acetyltransferases
NAT1 and NAT2. Biochem. J. 2000;348:159–166. doi: 10.1042/bj3480159.</cite> [<a href="https://doi.org/10.1042/bj3480159" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1221049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10794727/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&amp;title=Identification%20of%20amino%20acids%20imparting%20acceptor%20substrate%20selectivity%20to%20human%20arylamine%20acetyltransferases%20NAT1%20and%20NAT2&amp;author=G.%20H.%20Goodfellow&amp;author=J.%20M.%20Dupret&amp;author=D.%20M.%20Grant&amp;volume=348&amp;publication_year=2000&amp;pages=159-166&amp;pmid=10794727&amp;doi=10.1042/bj3480159&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref51">
<cite>Hein D. W., Ferguson R. J., Doll M. A., Rustan T. D., Gray K.. Molecular
genetics of human polymorphic N-acetyltransferase: enzymatic analysis
of 15 recombinant wild-type, mutant, and chimeric NAT2 allozymes. Hum. Mol. Genet. 1994;3:729–734. doi: 10.1093/hmg/3.5.729.</cite> [<a href="https://doi.org/10.1093/hmg/3.5.729" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8081359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hum.%20Mol.%20Genet.&amp;title=Molecular%20genetics%20of%20human%20polymorphic%20N-acetyltransferase:%20enzymatic%20analysis%20of%2015%20recombinant%20wild-type,%20mutant,%20and%20chimeric%20NAT2%20allozymes&amp;author=D.%20W.%20Hein&amp;author=R.%20J.%20Ferguson&amp;author=M.%20A.%20Doll&amp;author=T.%20D.%20Rustan&amp;author=K.%20Gray&amp;volume=3&amp;publication_year=1994&amp;pages=729-734&amp;pmid=8081359&amp;doi=10.1093/hmg/3.5.729&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref52">
<cite>Hein D. W., Doll M. A., Rustan T. D., Ferguson R. J.. Metabolic activation
of N-hydroxyarylamines and N-hydroxyarylamides by 16 recombinant human
NAT2 allozymes: effects of 7 specific NAT2 nucleic acid substitutions. Cancer Res. 1995;55:3531–3536.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/7627960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&amp;title=Metabolic%20activation%20of%20N-hydroxyarylamines%20and%20N-hydroxyarylamides%20by%2016%20recombinant%20human%20NAT2%20allozymes:%20effects%20of%207%20specific%20NAT2%20nucleic%20acid%20substitutions&amp;author=D.%20W.%20Hein&amp;author=M.%20A.%20Doll&amp;author=T.%20D.%20Rustan&amp;author=R.%20J.%20Ferguson&amp;volume=55&amp;publication_year=1995&amp;pages=3531-3536&amp;pmid=7627960&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref53">
<cite>Fretland A. J., Leff M. A., Doll M. A., Hein D. W.. Functional characterization
of human N-acetyltransferase 2 (NAT2) single nucleotide polymorphisms. Pharmacogenetics. 2001;11:207–215. doi: 10.1097/00008571-200104000-00004.</cite> [<a href="https://doi.org/10.1097/00008571-200104000-00004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11337936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenetics&amp;title=Functional%20characterization%20of%20human%20N-acetyltransferase%202%20(NAT2)%20single%20nucleotide%20polymorphisms&amp;author=A.%20J.%20Fretland&amp;author=M.%20A.%20Leff&amp;author=M.%20A.%20Doll&amp;author=D.%20W.%20Hein&amp;volume=11&amp;publication_year=2001&amp;pages=207-215&amp;pmid=11337936&amp;doi=10.1097/00008571-200104000-00004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref54">
<cite>Hein D. W., Fretland A. J., Doll M. A.. Effects
of single nucleotide polymorphisms
in human N-acetyltransferase 2 on metabolic activation (O-acetylation)
of heterocyclic amine carcinogens. Int. J. Cancer
J. Int. Cancer. 2006;119:1208–1211. doi: 10.1002/ijc.21957.</cite> [<a href="https://doi.org/10.1002/ijc.21957" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2582010/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16570281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Cancer%20J.%20Int.%20Cancer&amp;title=Effects%20of%20single%20nucleotide%20polymorphisms%20in%20human%20N-acetyltransferase%202%20on%20metabolic%20activation%20(O-acetylation)%20of%20heterocyclic%20amine%20carcinogens&amp;author=D.%20W.%20Hein&amp;author=A.%20J.%20Fretland&amp;author=M.%20A.%20Doll&amp;volume=119&amp;publication_year=2006&amp;pages=1208-1211&amp;pmid=16570281&amp;doi=10.1002/ijc.21957&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref55">
<cite>Zang Y., Doll M. A., Zhao S., States J. C., Hein D. W.. Functional
characterization of single-nucleotide polymorphisms and haplotypes
of human N-acetyltransferase 2. Carcinogenesis. 2007;28:1665–1671. doi: 10.1093/carcin/bgm085.</cite> [<a href="https://doi.org/10.1093/carcin/bgm085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2085371/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17434923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Carcinogenesis&amp;title=Functional%20characterization%20of%20single-nucleotide%20polymorphisms%20and%20haplotypes%20of%20human%20N-acetyltransferase%202&amp;author=Y.%20Zang&amp;author=M.%20A.%20Doll&amp;author=S.%20Zhao&amp;author=J.%20C.%20States&amp;author=D.%20W.%20Hein&amp;volume=28&amp;publication_year=2007&amp;pages=1665-1671&amp;pmid=17434923&amp;doi=10.1093/carcin/bgm085&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref56">
<cite>Zhou X., Ma Z., Dong D., Wu B.. Arylamine N-acetyltransferases: a
structural perspective. Br. J. Pharmacol. 2013;169:748–760. doi: 10.1111/bph.12182.</cite> [<a href="https://doi.org/10.1111/bph.12182" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3687656/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23517104/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Pharmacol.&amp;title=Arylamine%20N-acetyltransferases:%20a%20structural%20perspective&amp;author=X.%20Zhou&amp;author=Z.%20Ma&amp;author=D.%20Dong&amp;author=B.%20Wu&amp;volume=169&amp;publication_year=2013&amp;pages=748-760&amp;pmid=23517104&amp;doi=10.1111/bph.12182&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref57">
<cite>Oda A., Kobayashi K., Takahashi O.. Computational study of the three-dimensional
structure of N-acetyltransferase 2-acetyl coenzyme a complex. Biol. Pharm. Bull. 2010;33:1639–1643. doi: 10.1248/bpb.33.1639.</cite> [<a href="https://doi.org/10.1248/bpb.33.1639" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20930369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biol.%20Pharm.%20Bull.&amp;title=Computational%20study%20of%20the%20three-dimensional%20structure%20of%20N-acetyltransferase%202-acetyl%20coenzyme%20a%20complex&amp;author=A.%20Oda&amp;author=K.%20Kobayashi&amp;author=O.%20Takahashi&amp;volume=33&amp;publication_year=2010&amp;pages=1639-1643&amp;pmid=20930369&amp;doi=10.1248/bpb.33.1639&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref58">
<cite>Sheik
Amamuddy O., Afriyie Boateng R., Barozi V., Wavinya
Nyamai D., Tastan Bishop Ö.. Novel dynamic residue network analysis
approaches to study allosteric modulation: SARS-CoV-2 Mpro and its
evolutionary mutations as a case study. Comput.
Struct. Biotechnol. J. 2021;19:6431–6455. doi: 10.1016/j.csbj.2021.11.016.</cite> [<a href="https://doi.org/10.1016/j.csbj.2021.11.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8613987/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34849191/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput.%20Struct.%20Biotechnol.%20J.&amp;title=Novel%20dynamic%20residue%20network%20analysis%20approaches%20to%20study%20allosteric%20modulation:%20SARS-CoV-2%20Mpro%20and%20its%20evolutionary%20mutations%20as%20a%20case%20study&amp;author=O.%20Sheik%20Amamuddy&amp;author=R.%20Afriyie%20Boateng&amp;author=V.%20Barozi&amp;author=D.%20Wavinya%20Nyamai&amp;author=%C3%96.%20Tastan%20Bishop&amp;volume=19&amp;publication_year=2021&amp;pages=6431-6455&amp;pmid=34849191&amp;doi=10.1016/j.csbj.2021.11.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref59">
<cite>Okeke C. J., Musyoka T. M., Sheik Amamuddy O., Barozi V., Tastan Bishop Ö.. Allosteric
pockets and dynamic residue network hubs of falcipain 2 in mutations
including those linked to artemisinin resistance. Comput. Struct. Biotechnol. J. 2021;19:5647–5666. doi: 10.1016/j.csbj.2021.10.011.</cite> [<a href="https://doi.org/10.1016/j.csbj.2021.10.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8545671/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34745456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput.%20Struct.%20Biotechnol.%20J.&amp;title=Allosteric%20pockets%20and%20dynamic%20residue%20network%20hubs%20of%20falcipain%202%20in%20mutations%20including%20those%20linked%20to%20artemisinin%20resistance&amp;author=C.%20J.%20Okeke&amp;author=T.%20M.%20Musyoka&amp;author=O.%20Sheik%20Amamuddy&amp;author=V.%20Barozi&amp;author=%C3%96.%20Tastan%20Bishop&amp;volume=19&amp;publication_year=2021&amp;pages=5647-5666&amp;pmid=34745456&amp;doi=10.1016/j.csbj.2021.10.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref60">
<cite>Barozi V., Edkins A. L., Tastan Bishop Ö.. Evolutionary
progression of collective
mutations in Omicron sub-lineages towards efficient RBD-hACE2: Allosteric
communications between and within viral and human proteins. Comput. Struct. Biotechnol. J. 2022;20:4562–4578. doi: 10.1016/j.csbj.2022.08.015.</cite> [<a href="https://doi.org/10.1016/j.csbj.2022.08.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9384468/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35989699/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput.%20Struct.%20Biotechnol.%20J.&amp;title=Evolutionary%20progression%20of%20collective%20mutations%20in%20Omicron%20sub-lineages%20towards%20efficient%20RBD-hACE2:%20Allosteric%20communications%20between%20and%20within%20viral%20and%20human%20proteins&amp;author=V.%20Barozi&amp;author=A.%20L.%20Edkins&amp;author=%C3%96.%20Tastan%20Bishop&amp;volume=20&amp;publication_year=2022&amp;pages=4562-4578&amp;pmid=35989699&amp;doi=10.1016/j.csbj.2022.08.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref61">
<cite>Foutch D., Pham B., Shen T.. Protein conformational
switch discerned
via network centrality properties. Comput. Struct.
Biotechnol. J. 2021;19:3599–3608. doi: 10.1016/j.csbj.2021.06.004.</cite> [<a href="https://doi.org/10.1016/j.csbj.2021.06.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8246261/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34257839/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput.%20Struct.%20Biotechnol.%20J.&amp;title=Protein%20conformational%20switch%20discerned%20via%20network%20centrality%20properties&amp;author=D.%20Foutch&amp;author=B.%20Pham&amp;author=T.%20Shen&amp;volume=19&amp;publication_year=2021&amp;pages=3599-3608&amp;pmid=34257839&amp;doi=10.1016/j.csbj.2021.06.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref62">
<cite>Ruhnau B.. Eigenvector-centrality
 a node-centrality? Soc. Netw. 2000;22:357–365. doi: 10.1016/S0378-8733(00)00031-9.</cite> [<a href="https://doi.org/10.1016/S0378-8733(00)00031-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Soc.%20Netw.&amp;title=Eigenvector-centrality%20%EE%97%B8%20a%20node-centrality?&amp;author=B.%20Ruhnau&amp;volume=22&amp;publication_year=2000&amp;pages=357-365&amp;doi=10.1016/S0378-8733(00)00031-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref63">
<cite>Zang Y., Zhao S., Doll M. A., States J. C., Hein D. W.. Functional
characterization of the A411T (L137F) and G364A (D122N) genetic polymorphisms
in human N-acetyltransferase 2. Pharmacogenet.
Genomics. 2007;17:37–45. doi: 10.1097/01.fpc.0000236325.73186.2c.</cite> [<a href="https://doi.org/10.1097/01.fpc.0000236325.73186.2c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1989107/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17264801/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacogenet.%20Genomics&amp;title=Functional%20characterization%20of%20the%20A411T%20(L137F)%20and%20G364A%20(D122N)%20genetic%20polymorphisms%20in%20human%20N-acetyltransferase%202&amp;author=Y.%20Zang&amp;author=S.%20Zhao&amp;author=M.%20A.%20Doll&amp;author=J.%20C.%20States&amp;author=D.%20W.%20Hein&amp;volume=17&amp;publication_year=2007&amp;pages=37-45&amp;pmid=17264801&amp;doi=10.1097/01.fpc.0000236325.73186.2c&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref64">
<cite>Mwaniki R. M., Veldman W., Sanyanga A., Chamboko C. R., Bishop Ö. T.. Decoding Allosteric Effects of Missense
Variations in Drug Metabolism:
Afrocentric CYP3A4 Alleles Explored. J. Mol.
Biol. 2025:169160.  doi: 10.1016/j.jmb.2025.169160.</cite> [<a href="https://doi.org/10.1016/j.jmb.2025.169160" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40252954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=Decoding%20Allosteric%20Effects%20of%20Missense%20Variations%20in%20Drug%20Metabolism:%20Afrocentric%20CYP3A4%20Alleles%20Explored&amp;author=R.%20M.%20Mwaniki&amp;author=W.%20Veldman&amp;author=A.%20Sanyanga&amp;author=C.%20R.%20Chamboko&amp;author=%C3%96.%20T.%20Bishop&amp;publication_year=2025&amp;pages=169160&amp;pmid=40252954&amp;doi=10.1016/j.jmb.2025.169160&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref65">
<cite>Cooper A., Dryden D. T. F.. Allostery without conformational
change. Eur. Biophys. J. 1984;11:103–109. doi: 10.1007/BF00276625.</cite> [<a href="https://doi.org/10.1007/BF00276625" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6544679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20Biophys.%20J.&amp;title=Allostery%20without%20conformational%20change&amp;author=A.%20Cooper&amp;author=D.%20T.%20F.%20Dryden&amp;volume=11&amp;publication_year=1984&amp;pages=103-109&amp;pmid=6544679&amp;doi=10.1007/BF00276625&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref66">
<cite>Motlagh H. N., Wrabl J. O., Li J., Hilser V. J.. The ensemble nature
of allostery. Nature. 2014;508:331–339. doi: 10.1038/nature13001.</cite> [<a href="https://doi.org/10.1038/nature13001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4224315/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24740064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20ensemble%20nature%20of%20allostery&amp;author=H.%20N.%20Motlagh&amp;author=J.%20O.%20Wrabl&amp;author=J.%20Li&amp;author=V.%20J.%20Hilser&amp;volume=508&amp;publication_year=2014&amp;pages=331-339&amp;pmid=24740064&amp;doi=10.1038/nature13001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref67">
<cite>Pang Y.-P.. Three-dimensional
model of a substrate-bound SARS chymotrypsin-like cysteine proteinase
predicted by multiple molecular dynamics simulations: Catalytic efficiency
regulated by substrate binding. Proteins Struct.
Funct. Bioinforma. 2004;57:747–757. doi: 10.1002/prot.20249.</cite> [<a href="https://doi.org/10.1002/prot.20249" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7167676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15690493/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteins%20Struct.%20Funct.%20Bioinforma.&amp;title=Three-dimensional%20model%20of%20a%20substrate-bound%20SARS%20chymotrypsin-like%20cysteine%20proteinase%20predicted%20by%20multiple%20molecular%20dynamics%20simulations:%20Catalytic%20efficiency%20regulated%20by%20substrate%20binding&amp;author=Y.-P.%20Pang&amp;volume=57&amp;publication_year=2004&amp;pages=747-757&amp;pmid=15690493&amp;doi=10.1002/prot.20249&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref68">
<cite>Nussinov R., Tsai C.-J.. Allostery in Disease and in Drug Discovery. Cell. 2013;153:293–305. doi: 10.1016/j.cell.2013.03.034.</cite> [<a href="https://doi.org/10.1016/j.cell.2013.03.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23582321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Allostery%20in%20Disease%20and%20in%20Drug%20Discovery&amp;author=R.%20Nussinov&amp;author=C.-J.%20Tsai&amp;volume=153&amp;publication_year=2013&amp;pages=293-305&amp;pmid=23582321&amp;doi=10.1016/j.cell.2013.03.034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12332655/bin/ao5c02316_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">ao5c02316_si_001.pdf</a><sup> (5.1MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data underlying
this study are openly available in Zenodo at <a href="https://doi.org/10.5281/zenodo.15145541" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.5281/zenodo.15145541</a>. Raw data files, delta calculations, and PyMOL sessions are available.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Omega are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsomega.5c02316"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/ao5c02316.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (19.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12332655/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12332655/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332655%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332655/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12332655/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12332655/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40787317/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12332655/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40787317/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12332655/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12332655/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="y5lLJ0NusGRS4AuZEPsLVXc1GHeWOunv8TjUc5evZ8lwdpwctmU65eTD67cWk1bw">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
