
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Synthesis and In vitro Biological Evaluation of Quinolone-Based Hydrazones as Potential Antidiabetic Agents Targeting Key Metabolic Enzymes - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="4DDE454A89B6D8F31B454A005F1F638A.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsomega">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332697/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Omega">
<meta name="citation_title" content="Synthesis and In vitro Biological Evaluation of Quinolone-Based Hydrazones as Potential Antidiabetic Agents Targeting Key Metabolic Enzymes">
<meta name="citation_author" content="Naik Jui Pravin">
<meta name="citation_author_institution" content="Department of Pharmaceutical Chemistry, KLE College of Pharmacy, Belagavi, KLE Academy of Higher education and Research, Belagavi, Karnataka 590 010, India">
<meta name="citation_author" content="Rohini S Kavalapure">
<meta name="citation_author_institution" content="Department of Pharmaceutical Chemistry, KLE College of Pharmacy, Belagavi, KLE Academy of Higher education and Research, Belagavi, Karnataka 590 010, India">
<meta name="citation_author" content="Shankar Gharge">
<meta name="citation_author_institution" content="Department of Pharmaceutical Chemistry, KLE College of Pharmacy, Belagavi, KLE Academy of Higher education and Research, Belagavi, Karnataka 590 010, India">
<meta name="citation_author" content="Shankar G Alegaon">
<meta name="citation_author_institution" content="Department of Pharmaceutical Chemistry, KLE College of Pharmacy, Belagavi, KLE Academy of Higher education and Research, Belagavi, Karnataka 590 010, India">
<meta name="citation_author" content="Shriram D Ranade">
<meta name="citation_author_institution" content="Department of Pharmaceutical Chemistry, KLE College of Pharmacy, Belagavi, KLE Academy of Higher education and Research, Belagavi, Karnataka 590 010, India">
<meta name="citation_author" content="Ling Shing Wong">
<meta name="citation_author_institution" content="Faculty of Health and Life Sciences, INTI International University, Nilai 71800, Malaysia">
<meta name="citation_author" content="Ramith Ramu">
<meta name="citation_author_institution" content="Department of Biotechnology &amp; Bioinformatics, JSS Academy of Higher Education &amp; Research, Mysuru, Karnataka 570015, India">
<meta name="citation_author" content="Ramya C M">
<meta name="citation_author_institution" content="Department of Physiology, JSS Academy of Higher Education &amp; Research, Mysuru, Karnataka 570015, India">
<meta name="citation_publication_date" content="2025 Jul 22">
<meta name="citation_volume" content="10">
<meta name="citation_issue" content="30">
<meta name="citation_firstpage" content="33712">
<meta name="citation_doi" content="10.1021/acsomega.5c04663">
<meta name="citation_pmid" content="40787348">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332697/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332697/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332697/pdf/ao5c04663.pdf">
<meta name="description" content="A series of novel structures featuring quinolone-based hydrazones (5a–5q) were synthesized, characterized, and screened for their potential inhibition of key enzymes involved in carbohydrate metabolism, namely human pancreatic α-amylase (HPA) and ...">
<meta name="og:title" content="Synthesis and In vitro Biological Evaluation of Quinolone-Based Hydrazones as Potential Antidiabetic Agents Targeting Key Metabolic Enzymes">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="A series of novel structures featuring quinolone-based hydrazones (5a–5q) were synthesized, characterized, and screened for their potential inhibition of key enzymes involved in carbohydrate metabolism, namely human pancreatic α-amylase (HPA) and ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332697/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12332697">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsomega.5c04663"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/ao5c04663.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332697%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12332697/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12332697/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332697/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsomega.png" alt="ACS Omega logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Omega" title="Link to ACS Omega" shape="default" href="https://pubs.acs.org/journal/acsodf" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Omega</button></div>. 2025 Jul 22;10(30):33712–33730. doi: <a href="https://doi.org/10.1021/acsomega.5c04663" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsomega.5c04663</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Omega%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Omega%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Synthesis and <em>In vitro</em> Biological
Evaluation of Quinolone-Based Hydrazones as Potential Antidiabetic
Agents Targeting Key Metabolic Enzymes</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pravin%20NJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Naik Jui Pravin</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Naik Jui Pravin</span></h3>
<div class="p">
<sup>a</sup>
Department
of Pharmaceutical Chemistry, KLE College of Pharmacy, Belagavi, KLE Academy of Higher education and Research, Belagavi, Karnataka
590 010, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pravin%20NJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Naik Jui Pravin</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kavalapure%20RS%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Rohini S Kavalapure</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Rohini S Kavalapure</span></h3>
<div class="p">
<sup>a</sup>
Department
of Pharmaceutical Chemistry, KLE College of Pharmacy, Belagavi, KLE Academy of Higher education and Research, Belagavi, Karnataka
590 010, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kavalapure%20RS%22%5BAuthor%5D" class="usa-link"><span class="name western">Rohini S Kavalapure</span></a>
</div>
</div>
<sup>a,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gharge%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Shankar Gharge</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Shankar Gharge</span></h3>
<div class="p">
<sup>a</sup>
Department
of Pharmaceutical Chemistry, KLE College of Pharmacy, Belagavi, KLE Academy of Higher education and Research, Belagavi, Karnataka
590 010, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gharge%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shankar Gharge</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alegaon%20SG%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Shankar G Alegaon</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Shankar G Alegaon</span></h3>
<div class="p">
<sup>a</sup>
Department
of Pharmaceutical Chemistry, KLE College of Pharmacy, Belagavi, KLE Academy of Higher education and Research, Belagavi, Karnataka
590 010, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alegaon%20SG%22%5BAuthor%5D" class="usa-link"><span class="name western">Shankar G Alegaon</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ranade%20SD%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Shriram D Ranade</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Shriram D Ranade</span></h3>
<div class="p">
<sup>a</sup>
Department
of Pharmaceutical Chemistry, KLE College of Pharmacy, Belagavi, KLE Academy of Higher education and Research, Belagavi, Karnataka
590 010, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ranade%20SD%22%5BAuthor%5D" class="usa-link"><span class="name western">Shriram D Ranade</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wong%20LS%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Ling Shing Wong</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Ling Shing Wong</span></h3>
<div class="p">
<sup>b</sup>
Faculty
of
Health and Life Sciences, INTI International
University, Nilai
71800, Malaysia
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wong%20LS%22%5BAuthor%5D" class="usa-link"><span class="name western">Ling Shing Wong</span></a>
</div>
</div>
<sup>b</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ramu%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Ramith Ramu</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Ramith Ramu</span></h3>
<div class="p">
<sup>c</sup>
Department
of Biotechnology &amp; Bioinformatics, JSS
Academy of Higher Education &amp; Research, Mysuru, Karnataka
570015, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ramu%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ramith Ramu</span></a>
</div>
</div>
<sup>c</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22M%20RC%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Ramya C M</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Ramya C M</span></h3>
<div class="p">
<sup>d</sup>
Department
of Physiology, JSS Academy of Higher Education
&amp; Research, Mysuru, Karnataka
570015, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22M%20RC%22%5BAuthor%5D" class="usa-link"><span class="name western">Ramya C M</span></a>
</div>
</div>
<sup>d,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>a</sup>
Department
of Pharmaceutical Chemistry, KLE College of Pharmacy, Belagavi, KLE Academy of Higher education and Research, Belagavi, Karnataka
590 010, India
</div>
<div id="aff2">
<sup>b</sup>
Faculty
of
Health and Life Sciences, INTI International
University, Nilai
71800, Malaysia
</div>
<div id="aff3">
<sup>c</sup>
Department
of Biotechnology &amp; Bioinformatics, JSS
Academy of Higher Education &amp; Research, Mysuru, Karnataka
570015, India
</div>
<div id="aff4">
<sup>d</sup>
Department
of Physiology, JSS Academy of Higher Education
&amp; Research, Mysuru, Karnataka
570015, India
</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
Email: <span>rohinikavalapure@klepharm.edu</span>; <span>rohinikavalapure@yahoo.com</span>.</p>
</div>
<div class="fn" id="cor2">
<sup>*</sup><p class="display-inline">
Email: <span>ramyacm@jssuni.edu.in</span>.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 19; Accepted 2025 Jul 11; Revised 2025 Jul 4; Collection date 2025 Aug 5.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY-NC-ND 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12332697  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40787348/" class="usa-link">40787348</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>A series of novel
structures featuring quinolone-based hydrazones
(<strong>5a</strong>–<strong>5q</strong>) were synthesized, characterized,
and screened for their potential inhibition of key enzymes involved
in carbohydrate metabolism, namely human pancreatic α-amylase
(HPA) and human lysosomal acid α-glucosidase, as well as aldose
reductase, an enzyme associated with diabetes-related complications.
The synthesized compounds exhibited a broad range of inhibitory activities
against both α-glucosidase (IC<sub>50</sub>: 7.44 ± 0.07
to 14.75 ± 0.15 μg/mL) and α-amylase (IC<sub>50</sub>: 21.05 ± 0.17 to 31.43 ± 0.11 μg/mL). Notably, compound <strong>5o</strong> (5-nitrofuran) demonstrated the most potent inhibition
against both enzymes (α-glucosidase IC<sub>50</sub> = 7.44 ±
0.07 μg/mL; α-amylase IC<sub>50</sub> = 21.05 ± 0.17
μg/mL), surpassing the standard drug acarbose. Furthermore,
these hydrazones also showed promising aldose reductase inhibitory
activities (IC<sub>50</sub>: 4.12 ± 0.09 to 11.00 ± 0.05
μg/mL), with compound <strong>5o</strong> again exhibiting the highest
potency (IC<sub>50</sub> = 4.12 ± 0.09 μg/mL), even outperforming
quercetin. Kinetic studies on <strong>5o</strong> revealed a reversible,
noncompetitive inhibition mechanism against aldose reductase with
an inhibition constant (<em>K</em>
<sub>i</sub>) of 4.65 μM.
Molecular docking studies against α-amylase, α-glucosidase,
and aldose reductase demonstrated favorable binding interactions for
several compounds, with <strong>5o</strong> showing particularly strong
interactions with the active site of aldose reductase (docking score:
−10.051). Molecular dynamics simulations of the <strong>5o</strong>-aldose reductase complex over 100 ns confirmed stable binding within
the active site. Density functional theory (DFT) analysis of <strong>5o</strong> revealed a small HOMO–LUMO energy gap (0.112566
eV) and a soft nature, suggesting good chemical reactivity. These
findings showcase the potential of quinolone-based hydrazones, particularly
compound <strong>5o</strong>, as promising candidates aiming at the development
of multitarget therapies for antidiabetic agents.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9e07/12332697/2f06825b5188/ao5c04663_0016.jpg" loading="lazy" id="abs1" height="347" width="666" alt="graphic file with name ao5c04663_0016.jpg"></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9e07/12332697/2f06825b5188/ao5c04663_0014.jpg" loading="lazy" id="tgr1" height="347" width="666" alt="graphic file with name ao5c04663_0014.jpg"></p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Diabetes mellitus (DM) is a multifactorial
and chronic metabolic
disorder characterized by persistent hyperglycemia, arising due to
impaired insulin secretion, insulin resistance, or a combination of
both factors. This dysregulation in glucose
homeostasis contributes to a range of systemic complications, impacting
vital organs such as the kidneys, heart, eyes, and peripheral nerves.<em>
,
</em> Long-term hyperglycemia often leads to microvascular complications
such as diabetic nephropathy, retinopathy, and neuropathy, as well
as macrovascular disorders that collectively present significant clinical
challenges in diabetes management. According
to the International Diabetes Federation (IDF), approximately 537
million people were living with diabetes in 2021, accounting for about
10.5% of the global population, and this figure is projected to rise
to 783 million by 2045. Alarmingly, nearly half of these cases remain
undiagnosed, with low- and middle-income countries (LMICs) in Africa,
the Western Pacific, and Southeast Asia being disproportionately affected
due to limited healthcare infrastructure and lack of access to early
diagnostic services.
</p>
<p>Diabetes mellitus
is broadly classified into Type 1 diabetes (T1D),
accounting for 5–10% of cases and primarily caused by autoimmune
destruction of pancreatic β-cells, and type 2 diabetes (T2D),
comprising nearly 90% of cases, characterized by insulin resistance
and β-cell dysfunction.<em>
,
</em> One of the principal
therapeutic strategies in managing T2D involves the inhibition of
key metabolic enzymes responsible for carbohydrate digestion and glucose
flux. Enzymes such as α-amylase and α-glucosidase are
pivotal in carbohydrate metabolism; α-amylase hydrolyzes complex
starches into oligosaccharides, while α-glucosidase catalyzes
the final conversion of these oligosaccharides into glucose within
the intestinal brush border. Inhibiting
these enzymes can delay carbohydrate digestion and minimize postprandial
glucose spikes. Additionally, aldose reductase, an enzyme activated
under hyperglycemic conditions via the polyol pathway, reduces glucose
to sorbitol. Sorbitol accumulation and NADPH depletion lead to osmotic
stress and oxidative damage, thereby contributing to diabetes-related
complications such as nephropathy, neuropathy, and cataracts. Currently approved enzyme inhibitors like acarbose,
miglitol, and voglibose function by competitively binding to carbohydrate-processing
enzymes but are often limited by gastrointestinal side effects and
suboptimal efficacy. These drawbacks
underscore the need for safer and more effective enzyme inhibitors
targeting these metabolic pathways. Simultaneous inhibition of α-glucosidase,
α-amylase, and aldose reductase represents a strategic therapeutic
approach in diabetes management. While α-glucosidase and α-amylase
are key enzymes involved in carbohydrate digestion and postprandial
blood glucose regulation, aldose reductase plays a critical role in
the development of diabetic complications by mediating the polyol
pathway. Targeting all three enzymes concurrently can help achieve
better glycemic control, while also preventing or delaying diabetes-associated
tissue damage.</p>
<p>In this context, quinoline-based hydrazone derivatives
have emerged
as promising scaffolds in the search for novel antidiabetic agents
due to their diverse pharmacological profiles. The quinoline nucleus is a privileged structural motif
widely recognized in medicinal chemistry for its therapeutic versatility
and drug-like properties. Its incorporation
into drug candidates has been associated with a broad spectrum of
biological activities, including antidiabetic, anticancer, antimicrobial, antioxidant, and
anti-inflammatory effects. Recent studies
have specifically highlighted the efficacy of quinoline derivatives
in inhibiting key metabolic enzymes, such as α-amylase, α-glucosidase,
and aldose reductase, thereby offering a rational framework for designing
new therapeutic agents for diabetes management. Therefore, the present
study is aimed at designing and synthesizing novel quinoline–hydrazone
hybrids as potential inhibitors of α-amylase, α-glucosidase,
and aldose reductase. Recent research has emphasized the potential
of quinoline-based analogs as effective inhibitors of key metabolic
enzymes involved in diabetes, particularly α-amylase, α-glucosidase,
and aldose reductase. Several reported compounds have demonstrated
remarkable inhibitory activity. For instance, compound A (IC<sub>50</sub> = 0.12 ± 0.001 μM) and compound B (IC<sub>50</sub> =
0.19 ± 0.03 μM) showed strong inhibition against aldose
reductase (AKR1B1). Compound C exhibited
superior α-amylase inhibition (IC<sub>50</sub> = 0.02 ±
0.62 μM), outperforming the standard drug acarbose. Similarly, compound D (IC<sub>50</sub> = 6.20
± 0.30 μM) and compound E (IC<sub>50</sub> = 7.40 ±
0.20 μM) demonstrated notable α-glucosidase inhibition
compared to acarbose. Additionally, compounds
F and G both displayed promising α-glucosidase inhibitory activities,
each with an IC<sub>50</sub> of 2.60 ± 0.01 μM. The presence of the imine group (CN)
is a common feature in these highly active molecules, suggesting its
crucial role in enzyme binding and overall potency. This structural
motif, characterized by its ability to participate in hydrogen bonding
and π–π stacking interactions, is a central element
that we incorporated into our synthesized compounds (<strong>5a</strong>–<strong>5q</strong>). A critical analysis of these literature
findings reveals that most of the potent inhibitors share common structural
features, particularly the presence of a quinoline core, an imine
(CN) linkage, and variation in the substitution patterns of
the aldehyde moiety. These insights guided the rational design of
a new series of quinoline-based benzo hydrazide derivatives in our
study, aimed at enhancing the inhibitory potential against α-glucosidase,
α-amylase, and aldose reductase (<a href="#fig1" class="usa-link">Figure 
</a>). The objective is to develop multifunctional
agents that can effectively target multiple metabolic pathways involved
in hyperglycemia and its complications. This work explores the structural
and functional advantages of combining the quinoline moiety with a
hydrazone linkage to enhance the inhibitory potential against these
metabolic enzymes. The novelty of this research lies in the strategic
hybridization of pharmacologically active scaffolds to yield compounds
with improved efficacy and safety profiles, offering a potential alternative
to currently available therapies.</p>
<figure class="fig xbox font-sm" id="fig1"><h3 class="obj_head">1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332697_ao5c04663_0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9e07/12332697/a019cdbeeeed/ao5c04663_0001.jpg" loading="lazy" id="gr1" height="837" width="686" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Visual timeline illustrating the progressive
design of substituted
quinoline hybrids bearing imine linkages, with a particular emphasis
on the strategic hybridization of the quinoline core with thiocarbohydrazide
moieties, highlighting their enhanced antidiabetic efficacy through
potent inhibition of key metabolic enzymes.</p></figcaption></figure></section><section id="sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">Chemistry</h3>
<p>The synthesis of <em>N</em>′-((<em>E</em>)-4-((7-chloroquinolin-4-yl) oxy)-3-methoxybenzylidene)-2-((<em>E</em>)-substituted benzylidene) hydrazine-1-carbothiohydrazide
Schiff bases (<strong>5a</strong>–<strong>5q</strong>) was carried
out as shown in <a href="#sch1" class="usa-link">Scheme 
</a>. The reaction intermediate (<strong>3</strong>) was synthesized by refluxing
4-((7-chloroquinolin-4-yl) oxy)-3-methoxybenzaldehyde (<strong>1</strong>) with thiocarbohydrazide (<strong>2</strong>) and glacial acetic acid
in catalytic proportions (0.001 mol) in methanol (10 mL) and was refluxed
for 24 h to obtain the product with 83% yield. The compound <strong>3</strong> was then reacted with different aryl and hetero aldehydes
using the catalytic proportion of glacial acetic acid to obtain the
ultimate series of compounds. The reaction was monitored by TLC for
its completion. The final crude compounds (<strong>5a</strong>–<strong>5q</strong>) were recrystallized using suitable solvents and dried.
The melting points were measured using a conventional method in a
melting point apparatus and are reported without correction. The entire
series of synthesized compounds was further characterized and confirmed
with the help of nuclear magnetic resonance (NMR) spectra, including <sup>1</sup>H (400 MHz) and <sup>13</sup>C (100 MHz), and deuterated dimethyl
sulfoxide (DMSO-<em>d</em>
<sub>6</sub>) was used as the solvent
for all NMR analysis. The characteristic peaks such as −OCH<sub>3,</sub> −NH, −CH were found in the range of 3–4,
10–12, 8–8.5 δppm in <sup>1</sup>H NMR, whereas
in <sup>13</sup>C NMR, the CS peak was found in the range
of 170–180 δppm and C–O of the −OCH<sub>3</sub> peak at 56–58 δppm. Chemical shifts (δ)
are expressed in parts per million (ppm), and Fourier transform infrared
spectroscopy (FTIR) spectra were obtained using an IR Affinity-1 spectrophotometer
(Shimadzu) and the characteristic peaks such as −NH<sub>2</sub> (3500–3650 cm<sup>–1</sup>), −NH (3200–3650
cm<sup>–1</sup>), −OH (3200–3450 cm<sup>–1</sup>), Ar–H (3000–3150 cm<sup>–1</sup>), CHN
(1500–1575 cm<sup>–1</sup>), while LCMS analysis was
performed to obtained the exact mass of the compounds. Computational
analysis was executed using the Schrödinger software suite,
employing the Glide, Prime, and Desmond modules. Interaction visualizations
were generated using Maestro from the Schrodinger suite.</p>
<figure class="fig xbox font-sm" id="sch1"><h4 class="obj_head">1. Synthesis
of <em>N</em>′-((<em>E</em>)-4-((7-Chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)-2-((<em>E</em>)-Substituted Benzylidene) Hydrazine-1-carbothiohydrazide
Schiff Bases (<strong>5a</strong>–<strong>5q</strong>)­
.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332697_ao5c04663_0012.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9e07/12332697/0a1d2c997839/ao5c04663_0012.jpg" loading="lazy" id="gr2" height="432" width="802" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/sch1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p id="sch1-fn1"><sup>a</sup> Reagents and conditions:
(i)
glacial acetic acid, MeOH, reflux at 65 °C, 24 h; (ii) N-substituted
aldehydes, glacial acetic acid, MeOH, reflux at 65 °C, 24 h in
the range of 65–96% yields.</p></figcaption></figure></section><section id="sec2.2"><h3 class="pmc_sec_title">Biological
Evaluation</h3>
<section id="sec2.2.1"><h4 class="pmc_sec_title">HPA and HLAG Inhibition Assay</h4>
<p>The synthesized quinolone-based
hydrazones (<strong>5a</strong>–<strong>5q</strong>) were evaluated
for their inhibitory activities against two crucial enzymes involved
in carbohydrate metabolism: human pancreatic α-amylase (HPA)
and human lysosomal acid α-glucosidase (HLAG). The inhibitory
effect of all the synthesized derivatives is detailed in <a href="#tbl1" class="usa-link">Table 
</a>. The biological evaluation
of the synthesized quinolone-based hydrazones (<strong>5a</strong>–<strong>5q</strong>) revealed a broad spectrum of inhibitory potencies against
human pancreatic α-amylase (HPA) and human lysosomal acid α-glucosidase
(HLAG), crucial enzymes in postprandial glucose regulation. A deeper
analysis of the structure–activity relationship (SAR) trends
offers valuable insights into the contribution of various substituents
to the observed inhibitory profiles and enzyme selectivity. Generally,
compounds bearing electron-withdrawing groups (EWGs) on the aryl ring
of the hydrazone moiety demonstrated superior inhibitory effects.
This trend is strikingly exemplified by compound <strong>5o</strong> (5-nitrofuran),
which exhibited the most potent inhibition for both α-glucosidase
(IC<sub>50</sub> = 7.44 ± 0.07 μg/mL) and α-amylase
(IC<sub>50</sub> = 21.05 ± 0.17 μg/mL). The strong electron-withdrawing
nature of the nitro group and the furan ring likely enhances the binding
affinity through favorable electronic interactions within the enzyme
active sites, possibly by increasing the electrophilicity of the imine
carbon or influencing the protonation states of key residues. This
superior performance of <strong>5o</strong> over the standard drug acarbose
(IC<sub>50</sub> for α-glucosidase = 10.67 ± 0.25 μg/mL;
α-amylase = 30.76 ± 0.25 μg/mL) underscores the significant
impact of this specific substitution. Halogen substituents, another
class of EWGs, also proved to be effective in modulating the inhibitory
activity. For instance, compound <strong>5e</strong> (4-bromo) displayed
strong inhibitory effects, particularly against α-amylase (IC<sub>50</sub> = 28.75 ± 0.33 μg/mL), while compound <strong>5c</strong> (4-chloro) showed significant inhibition against α-glucosidase
(IC<sub>50</sub> = 10.79 ± 0.74 μg/mL). The electron-withdrawing
inductive effect of halogens can stabilize transition states during
enzymatic reactions or enhance interactions with polar residues in
the active site. The notable activity of <strong>5m</strong> (4-bromothiophene)
against α-amylase (IC<sub>50</sub> = 22.48 ± 0.36 μg/mL)
further supports the positive influence of halogenated heteroaryl
moieties. Conversely, the effect of electron-donating groups (EDGs)
appears to be more varied. While compounds with electron-donating
groups such as <strong>5a</strong> (4-methyl) still demonstrated significant
inhibitory effects against both enzymes (e.g., IC<sub>50</sub> for
α-glucosidase = 14.75 ± 0.15 μg/mL, α-amylase
= 24.05 ± 0.30 μg/mL), their potency generally lagged behind
those with strong EWGs. This suggests that the electronic environment
influenced by EWGs on the aryl ring is more conducive to the optimal
binding and inhibition for these specific enzymes. Regarding enzyme
selectivity, certain trends emerge. While <strong>5o</strong> is a potent
dual inhibitor, compounds like <strong>5e</strong> and <strong>5m</strong> showed a more pronounced inhibitory effect against α-amylase
compared to α-glucosidase. This might indicate that the nature
and positioning of substituents can fine-tune interactions with the
distinct active sites of these two enzymes. The bulkier bromo group
in part <strong>5e</strong> and the thiophene ring in part <strong>5m</strong> could favor specific hydrophobic pockets or steric interactions
within the α-amylase active site. In contrast, compounds like <strong>5j</strong> (4-nitro) exhibited strong α-glucosidase inhibition
(IC<sub>50</sub> = 8.58 ± 0.32 μg/mL), suggesting a preferential
interaction profile for this enzyme, likely due to the strong electron-withdrawing
character of the nitro group. These differential inhibitory profiles
highlight the versatility of the quinolone-based hydrazone scaffold
in modulating key enzymes involved in carbohydrate digestion and glucose
regulation, offering avenues for the development of targeted antidiabetic
therapies. The results collectively underscore the potential of these
compounds as valuable leads in the ongoing search for effective antidiabetic
agents.</p>
<section class="tw xbox font-sm" id="tbl1"><h5 class="obj_head">1. 
<em>In Vitro</em> HPA, HLAG,
and AR Inhibitory Activity of the Synthesized Quinolone-Based Hydrazones
(<strong>5a</strong>–<strong>5q</strong>)­<sup></sup>
.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332697_ao5c04663_0013.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9e07/12332697/32146384ed40/ao5c04663_0013.jpg" loading="lazy" id="gr3" height="1585" width="825" alt="graphic file with name ao5c04663_0013.jpg"></a></p>
<div class="p text-right font-secondary"><a href="table/tbl1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t1fn1">
<sup>a</sup><p class="display-inline">x, y:
Values are expressed as mean
± SE. Means in the same column with distinct superscripts are
significantly different (<em>p</em> ≤ 0.05) as separated
by Duncan’s multiple range test. *: Positive control for aldose
reductase (quercetin). #: Positive control for α-glucosidase
and α-amylase (acarbose).</p>
</div></div></section></section><section id="sec2.2.2"><h4 class="pmc_sec_title">Aldose Reductase Inhibition Assay</h4>
<p>The synthesized quinolone-based
hydrazones (<strong>5a</strong>–<strong>5q</strong>) were also evaluated
for their inhibition potential against aldose reductase (AR), an NADPH-dependent
oxidoreductase responsible for converting glucose to sorbitol via
the polyol pathway. Under hyperglycemic conditions, increased aldose
reductase expression leads to elevated sorbitol accumulation, contributing
to various complications in diabetes. As a result, aldose reductase
inhibitors play a crucial role in managing diabetic complications.
While commercially available aldose reductase inhibitors like Epalrestat,
Sorbinil, and Zopolrestat have been used for this purpose, their side
effects and limited effectiveness have led to a search for more potent
and safer alternatives.</p>
<p>The synthesized quinolone-based hydrazones
(<strong>5a</strong>–<strong>5q</strong>) exhibited notable inhibitory
activity against aldose reductase (AR), a key enzyme involved in the
polyol pathway and a critical target for managing diabetic complications.
The IC<sub>50</sub> values across the series ranged from 4.12 ±
0.09 to 11.00 ± 0.05 μg/mL, demonstrating a wide range
of inhibitory potencies. Among these, compound <strong>5o</strong>, featuring
a 5-nitrofuran substituent, emerged as the most potent inhibitor with
an IC<sub>50</sub> value of 4.12 ± 0.09 μg/mL, surpassing
that of the reference compound quercetin (IC<sub>50</sub> = 8.74 ±
0.15 μg/mL). This enhanced activity is likely due to the strong
electron-withdrawing nature of the nitrofuran group, which may improve
electrostatic interactions and hydrogen bonding within the enzyme’s
active site. Similarly, other electron-withdrawing groups, such as
halogens, also contributed to significant inhibition; for example,
compounds <strong>5e</strong> (4-bromo phenyl) and <strong>5c</strong> (4-chloro
phenyl) showed strong activities with IC<sub>50</sub> values of 5.02
± 0.33 and 7.79 ± 0.76 μg/mL, respectively, suggesting
that halogen atoms facilitate favorable interactions such as halogen
bonding or enhanced hydrophobic contacts. In contrast, electron-donating
substituents like methyl (<strong>5a</strong>) and methoxy (<strong>5b</strong>) groups yielded moderate inhibition, indicating a less pronounced
effect on potency. Notably, compound <strong>5h</strong>, containing a
pyridine ring, also displayed potent inhibition (IC<sub>50</sub> =
4.87 ± 0.05 μg/mL), which can be attributed to the nitrogen
heterocycle’s ability to participate in hydrogen bonding and
π-stacking interactions within the active site. Conversely,
bulkier or less electron-withdrawing substituents, in compound <strong>5i</strong>, resulted in weaker inhibition (IC<sub>50</sub> = 10.87
± 0.22 μg/mL), underscoring the impact of both electronic
and steric factors on binding affinity. Overall, the structure–activity
relationship clearly indicates that electron-withdrawing groups, particularly
nitro and halogen substituents and nitrogen-containing heterocycles,
enhance aldose reductase inhibition, likely by promoting stronger
and more specific interactions within the enzyme’s active site.
These findings highlight the potential of quinolone-based hydrazones
as promising candidates for therapeutic intervention in diabetic complications
mediated by the polyol pathway.</p></section><section id="sec2.2.3"><h4 class="pmc_sec_title">Kinetic Analysis of Aldose
Reductase Inhibition</h4>
<p>Among
the synthesized quinolone-based hydrazones, compound <strong>5o</strong> (5-nitrofuran) was selected for detailed kinetic inhibition analysis
against aldose reductase, owing to its notable inhibition potential.
In order to elucidate the mode of inhibition, the enzyme was exposed
to varying concentrations of the substrate <span class="font-variant-small-caps">dl</span>-glyceraldehyde
in the absence (control) and presence of <strong>5o</strong> at IC<sub>2</sub>0, IC<sub>4</sub>0, and IC<sub>6</sub>0 concentrations. Kinetic
parameters, including the Michaelis–Menten constant (<em>K</em>
<sub>m</sub>) and maximum reaction velocity (<em>V</em>
<sub>max</sub>), were evaluated using Lineweaver–Burk plots
(<a href="#tbl2" class="usa-link">Table 
</a>).</p>
<section class="tw xbox font-sm" id="tbl2"><h5 class="obj_head">2. Enzyme Kinetics of Aldose Reductase
Enzyme by <strong>5o</strong>
<sup></sup>
.</h5>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="±" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">
<strong>compound</strong>
</th>
<th align="center" colspan="1" rowspan="1">
<strong>treatment</strong>
</th>
<th align="center" colspan="1" rowspan="1">
<strong>mode of inhibition</strong>
<sup>x</sup>
</th>
<th align="center" colspan="1" rowspan="1">
<strong>
<em>K</em>
<sub>m</sub> (mM)</strong>
</th>
<th align="center" colspan="1" rowspan="1">
<strong>
<em>V</em>
<sub>max</sub> (10</strong>
<sup>
<strong>3</strong>
</sup>
<strong>μM/min)</strong>
<sup>
<strong>–1</strong>
</sup>
</th>
<th align="center" colspan="1" rowspan="1">
<strong>
<em>K</em>
</strong>
<sub>
<strong>1</strong>
</sub>
<strong>(μM)</strong>
<sup>y<strong>,</strong>z</sup>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>5o</strong>
</td>
<td align="left" colspan="1" rowspan="1">control</td>
<td align="left" colspan="1" rowspan="1">noncompetitive</td>
<td align="center" colspan="1" rowspan="1">3.84</td>
<td align="center" colspan="1" rowspan="1">8.75</td>
<td align="center" colspan="1" rowspan="1">4.65 ± 0.07</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">IC<sub>20</sub> = 1.65 μg</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">4.02</td>
<td align="center" colspan="1" rowspan="1">7.05</td>
<td align="center" colspan="1" rowspan="1"> </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">IC<sub>40</sub> = 3.30 μg</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">3.95</td>
<td align="center" colspan="1" rowspan="1">5.10</td>
<td align="center" colspan="1" rowspan="1"> </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">IC<sub>60</sub> = 4.90 μg</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">4.08</td>
<td align="center" colspan="1" rowspan="1">4.44</td>
<td align="center" colspan="1" rowspan="1"> </td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t2fn1">
<sup>a</sup><p class="display-inline">x: Inhibition mode was determined
from the Lineweaver–Burk plot. y: <em>K</em>
<sub>i</sub> = dissociation constant. z: Values are expressed as mean ±
SE.</p>
</div></div></section><p>As shown in <a href="#fig2" class="usa-link">Figure 
</a>, the Lineweaver–Burk
plots for <strong>5o</strong> at different
concentrations intersected on the same <em>x</em>-axis (1/<em>K</em>
<sub>m</sub>) as the uninhibited reaction, indicating that
the <em>K</em>
<sub>m</sub> value remained unchanged. However,
both the slope and <em>y</em>-intercept increased progressively
with higher concentrations of <strong>5o</strong>, reflecting a concentration-dependent
decrease in <em>V</em>
<sub>max</sub>. These results suggest
that <strong>5o</strong> inhibits aldose reductase through a classical
noncompetitive mechanism, whereby enzyme activity is reduced without
affecting substrate binding affinity. Furthermore, the reversible
nature of inhibition was supported by the kinetic data, and the Dixon
plot analysis yielded an inhibition constant (<em>K</em>
<sub>i</sub>) of 4.65 μM (<a href="#fig3" class="usa-link">Figure 
</a>). Collectively, these findings confirm that <strong>5o</strong> acts as a reversible, noncompetitive inhibitor of aldose
reductase, underscoring its potential as a promising antidiabetic
agent targeting key metabolic enzymes.</p>
<figure class="fig xbox font-sm" id="fig2"><h5 class="obj_head">2.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332697_ao5c04663_0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9e07/12332697/1785abddc1e6/ao5c04663_0002.jpg" loading="lazy" id="gr4" height="547" width="825" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Lineweaver–Burk
plot of substrate dependent enzyme kinetics
of aldose reductase inhibition activity by <strong>5o</strong>. Aldose
reductase was exposed to varying concentrations with diverse concentrations
of <span class="font-variant-small-caps">d</span>-glyceraldehyde (2.5–40 mM) in the absence or
presence of <strong>5o</strong> at IC<sub>20</sub>, IC<sub>40</sub>, and
IC<sub>60</sub> inhibiting concentrations.</p></figcaption></figure><figure class="fig xbox font-sm" id="fig3"><h5 class="obj_head">3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332697_ao5c04663_0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9e07/12332697/96e5515912b1/ao5c04663_0003.jpg" loading="lazy" id="gr5" height="490" width="800" alt="3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Determination
of <em>K</em>
<sub>i</sub> from the secondary
plots of the slope versus inhibitor concentration [I] from the Lineweaver–Burk
plot; values are mean ± SE of three independent experiments;
each performed in triplicate.</p></figcaption></figure></section><section id="sec2.2.4"><h4 class="pmc_sec_title">Structure–Activity Relationship (SAR) of Quinolone-Based
Hydrazones (<strong>5a</strong>–<strong>5q</strong>)</h4>
<p>The structure–activity
relationship (SAR) of the synthesized quinolone-based hydrazones (<strong>5a</strong>–<strong>5q</strong>) reveals valuable insights into
their potential as antidiabetic agents targeting key metabolic enzymes.
The quinolone scaffold, a core structure in all derivatives, plays
a pivotal role in modulating the inhibitory potential against the
target enzymes human pancreatic α-amylase (HPA), human lysosomal
acid α-glucosidase (HLAG), and aldose reductase (AR). The introduction
of different substituents on the aryl ring of the hydrazone moiety
significantly influences the enzyme inhibitory effects, highlighting
the importance of specific functional group modifications. The compounds
bearing electron-withdrawing groups (EWGs) such as <strong>5o</strong> (5-nitrofuran) exhibited the most potent inhibitory effect, particularly
targeting both α-amylase and α-glucosidase, with IC<sub>50</sub> values of 7.44 ± 0.07 μg/mL and 21.05 ±
0.17 μg/mL, respectively. This suggests that nitrofuran substitution
enhances the binding affinity and enzyme inhibition. Furthermore, <strong>5o</strong> demonstrated remarkable aldose reductase inhibition with
an IC<sub>50</sub> value of 4.12 ± 0.09 μg/mL, positioning
it as a lead candidate for further development. Similarly, compounds
bearing halogen substitutions, such as <strong>5e</strong> (4-bromo) and <strong>5m</strong> (4-bromothiophene), also exhibited strong inhibitory effects,
particularly against α-amylase, highlighting the importance
of halogen groups in improving activity. The 4-bromo derivative (<strong>5e</strong>) showed significant activity across all enzymes, suggesting
that this substitution provides a versatile interaction with the enzyme
active sites. Substituents like 4-chloro (<strong>5c</strong>) and 4-nitro
(<strong>5j</strong>) also contributed to effective inhibition, particularly
against α-glucosidase, with a notable increase in activity observed
for <strong>5j</strong>, which demonstrated favorable binding interactions.
The presence of halogens or electron-withdrawing groups likely enhances
the ability of these compounds to form stable interactions with the
enzyme binding sites, improving their overall inhibitory potency.
Moreover, the SAR study revealed that the position and functional
nature of the substituent present on the quinolone scaffold significantly
impact the binding interactions, as seen with compounds <strong>5b</strong>, <strong>5d</strong>, and <strong>5h</strong>, which displayed favorable
MM/GBSA binding energies, despite having slightly lower docking scores.
These compounds exhibit strong hydrogen bonding and aromatic stacking
interactions, further emphasizing the role of steric and electronic
effects in modulating inhibitory potential. In summary, the SAR analysis
of quinolone-based hydrazones (<strong>5a</strong>–<strong>5q</strong>) demonstrates that specific substitutions, particularly those with
electron-withdrawing properties or halogen groups, enhance enzyme
binding affinity and inhibitory activity. These findings emphasize
the potential of quinolone-based hydrazones as favorable antidiabetic
agents, with the ability to target key enzymes involved in carbohydrate
metabolism and diabetic complications. Compound <strong>5o</strong>, with
its exceptional potency across all assays, serves as a prime candidate
for further optimization and development into a therapeutic agent
for managing diabetes mellitus (<a href="#fig4" class="usa-link">Figure 
</a>).</p>
<figure class="fig xbox font-sm" id="fig4"><h5 class="obj_head">4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9e07/12332697/eee0fbc8b23a/ao5c04663_0004.jpg" loading="lazy" id="gr6" height="282" width="666" alt="4"></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Structure–activity relationship for quinoline-based
hydrazones
(<strong>5a</strong>–<strong>5q</strong>).</p></figcaption></figure></section></section><section id="sec2.3"><h3 class="pmc_sec_title">Computational Study</h3>
<section id="sec2.3.1"><h4 class="pmc_sec_title">Molecular Docking</h4>
<p>To elucidate the
molecular basis
of enzyme inhibition, molecular docking studies were conducted against
three key metabolic enzymes implicated in diabetes: α-glucosidase
(PDB ID: <a href="https://www.rcsb.org/structure/5NN8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">5NN8</a>), α-amylase (PDB ID: <a href="https://www.rcsb.org/structure/4W93" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">4W93</a>), and aldose reductase (PDB ID: <a href="https://www.rcsb.org/structure/4JIR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">4JIR</a>). The quinolone-based
hydrazone derivatives demonstrated favorable binding interactions
across all three enzyme targets, with several compounds displaying
comparable or superior docking profiles relative to standard inhibitors
such as acarbose and Epalrestat (<a href="#fig8" class="usa-link">Figure 
</a>).</p>
<figure class="fig xbox font-sm" id="fig8"><h5 class="obj_head">8.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332697_ao5c04663_0008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9e07/12332697/32259c8e69c5/ao5c04663_0008.jpg" loading="lazy" id="gr10" height="256" width="700" alt="8"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) acarbose in the binding pocket of α-amylase
(PDB ID: <a href="https://www.rcsb.org/structure/4W93" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">4W93</a>), (b) acarbose in
the binding pocket of α-glucosidase (PDB ID: <a href="https://www.rcsb.org/structure/5NN8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">5NN8</a>), and (c) Epalrestat
in the binding pocket of aldose reductase (PDB ID: <a href="https://www.rcsb.org/structure/4JIR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">4JIR</a>).</p></figcaption></figure><p>In the case of α-amylase,
compound <strong>5i</strong> exhibited a superior binder with a docking
score of −6.99, exhibiting key hydrogen bonds with ILE235,
GLU233, ASP300, and π–π stacking interactions with
TRP58 and HIE101. Compound <strong>5b</strong> also showed a high docking
score (−6.967) and a significant glide energy (−58.496),
stabilized by π–π stacking interactions with TRP58
and TRP59 and a hydrogen bond with GLU233, yielding a robust MM/GBSA
binding energy of −73.14 kcal/mol. Compound <strong>5f</strong> (docking score −6.937) formed similar stabilizing interactions,
while compounds <strong>5d</strong> and <strong>5h</strong>, despite slightly
lower docking scores, exhibited stronger MM/GBSA binding energies
(−76.48 and −76.59 kcal/mol, respectively), underscoring
their stable binding via hydrogen bonding and aromatic stacking (<a href="#fig5" class="usa-link">Figure 
</a>). Notably, compound <strong>5j</strong>, with a moderate docking score of −6.678, showed
the most favorable MM/GBSA binding energy (−78.91 kcal/mol),
attributed to interactions with ASP300 and GLU233. The standard drug
acarbose demonstrated a docking score of −5.074 and MM/GBSA
of −81.12 kcal/mol, highlighting its extensive network of hydrogen
bonds and interaction points (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c04663/suppl_file/ao5c04663_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S1 and Table S1</a>).</p>
<figure class="fig xbox font-sm" id="fig5"><h5 class="obj_head">5.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332697_ao5c04663_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9e07/12332697/c37dc5d0d540/ao5c04663_0005.jpg" loading="lazy" id="gr7" height="325" width="700" alt="5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>2D poses of binding orientation of compound <strong>5i</strong> in
the binding pocket of α-amylase (PDB ID: <a href="https://www.rcsb.org/structure/4W93" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">4W93</a>).</p></figcaption></figure><p>Against α-glucosidase, compound <strong>5k</strong> presented
the best docking score (−4.883), engaging in hydrogen bonding
(SER676), halogen bonding (HIE674), π–π stacking
(TRP481), and salt bridges (ASP282, ASP616, and ASP518), resulting
in a strong MM/GBSA energy of −74.21 kcal/mol. Similarly, compound <strong>5q</strong> showed a comparable profile (docking score −4.793,
MM/GBSA −74.02 kcal/mol) with multimodal interactions. Compounds <strong>5n</strong>, <strong>5d</strong>, <strong>5e</strong>, and <strong>5h</strong> also
demonstrated favorable binding through hydrogen bonding, π–cation,
and π–π stacking interactions with key residues.
Interestingly, compound <strong>5o</strong> recorded a strong MM/GBSA
value of −77.94 kcal/mol due to stable interactions including
salt bridges and hydrogen bonds (<a href="#fig6" class="usa-link">Figure 
</a>). Although acarbose displayed the most potent
inhibition (docking score −12.535; MM/GBSA −79.22 kcal/mol),
the synthetic derivatives, especially <strong>5k</strong>, <strong>5q</strong>, and <strong>5h</strong>, exhibited notable binding characteristics,
suggesting their promise as α-glucosidase inhibitors (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c04663/suppl_file/ao5c04663_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S2, Table S2</a>). Among the series, the docking
results with aldose reductase were particularly compelling. Compound <strong>5o</strong> exhibited a docking score of −10.051, closely approaching
that of the reference drug Epalrestat (−11.24) and demonstrated
extensive stabilizing interactions, including π–π
stacking with TRP20 and TRP111, hydrogen bonds with GLN183, ASN160,
SER210, and SER214, and a salt bridge with LYS21. These interactions
resulted in a strong MM/GBSA binding energy of −79.19 kcal/mol
(<a href="#fig7" class="usa-link">Figure 
</a>). Compounds <strong>5d</strong> and <strong>5b</strong> also showed high docking scores (−9.603
and −9.967, respectively), with significant interactions contributing
to MM/GBSA energies of −79.58 and −79.35 kcal/mol. Compounds <strong>5m</strong> and <strong>5n</strong> further reinforced the binding potential
of the series, stabilized by aromatic and electrostatic interactions.
Although compounds such as <strong>5p</strong> and <strong>5q</strong> displayed
weaker affinities, the overall docking data highlight compound <strong>5o</strong> as a lead candidate, exhibiting a binding profile comparable
to Epalrestat (MM/GBSA: −79.92 kcal/mol) (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c04663/suppl_file/ao5c04663_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S3, Table S3</a>). Given its strong interaction with aldose
reductase and favorable energy profile, compound <strong>5o</strong> was
selected for further molecular dynamics (MD) simulations to determine
the stability of its binding over time. Additionally, density functional
theory (DFT) studies were conducted to evaluate its electronic properties
and reactivity, supporting its potential for further development as
an antidiabetic agent.</p>
<figure class="fig xbox font-sm" id="fig6"><h5 class="obj_head">6.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332697_ao5c04663_0006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9e07/12332697/8c3a26fa7996/ao5c04663_0006.jpg" loading="lazy" id="gr8" height="317" width="700" alt="6"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>2D poses of binding orientation
of compound <strong>5k</strong> in
the binding pocket of α-glucosidase (PDB ID: <a href="https://www.rcsb.org/structure/5NN8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">5NN8</a>).</p></figcaption></figure><figure class="fig xbox font-sm" id="fig7"><h5 class="obj_head">7.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332697_ao5c04663_0007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9e07/12332697/abad41f2e5eb/ao5c04663_0007.jpg" loading="lazy" id="gr9" height="354" width="700" alt="7"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>2D poses of binding orientation of compound <strong>5o</strong> in
the binding pocket of aldose reductase (PDB ID: <a href="https://www.rcsb.org/structure/4JIR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">4JIR</a>).</p></figcaption></figure></section></section><section id="sec2.4"><h3 class="pmc_sec_title">Molecular Dynamics Simulations</h3>
<section id="sec2.4.1"><h4 class="pmc_sec_title">Intramolecular Contacts
in Molecular Dynamics Simulation</h4>
<p>The RMSD analysis of the
aldose reductase compound <strong>5o</strong> complex (PDB ID: <a href="https://www.rcsb.org/structure/4JIR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">4JIR</a>) over a 100 ns molecular
dynamics simulation revealed a stable conformational
profile of the protein, along with moderate flexibility of the ligand
throughout the simulation. The RMSD of the protein backbone (Cα
atoms) remained consistently within the range of ∼0.9 to 1.4
Å, reflecting structural stability of aldose reductase throughout
the simulation period. In contrast, the ligand exhibited initial fluctuations
during the first 20 ns, followed by stabilization, with RMSD values
typically ranging between 1.5 and 2.5 Å. Minor deviations observed
between 30 and 100 ns, peaking around 3.0 Å, indicate that the
ligand remains accommodated within the active site, dynamically adapting
to the protein’s conformational changes while preserving binding
interactions. Further insights from root-mean-square fluctuation (RMSF)
analysis revealed that most protein residues showed low flexibility,
with RMSF values between 0.4 and 1.0 Å, suggesting a structurally
rigid and stable backbone. However, elevated fluctuations were observed
at specific residue regions, notably around indices 40, 130–150,
and 200–220, where RMSF values exceeded 1.5 Å and approached
2.4 Å. The most pronounced flexibility was detected at the C-terminal
region, where the RMSF peaked near 3.0 Å, indicating an enhanced
mobility of this terminal segment. Overall, the results confirm the
conformational integrity of the protein and the stable binding behavior
of the ligand occupying the active pocket over the period of the simulation
run (<a href="#fig9" class="usa-link">Figure 
</a>).</p>
<figure class="fig xbox font-sm" id="fig9"><h5 class="obj_head">9.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332697_ao5c04663_0009.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9e07/12332697/61c7f86442ea/ao5c04663_0009.jpg" loading="lazy" id="gr11" height="456" width="700" alt="9"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) Protein
ligand RMSD plot of compound <strong>5o</strong>, (b) protein
RMSF plot, (c) protein ligand contact timeline, and (d) ligand RMSF
plot over a period of 100 ns within aldose reductase protein.</p></figcaption></figure></section><section id="sec2.4.2"><h4 class="pmc_sec_title">Intermolecular Contacts in Molecular Dynamics
Simulation</h4>
<p>The protein–ligand contacts are illustrated
in a stacked bar
chart that categorizes the nature of interactions between a ligand
and specific amino acid residues within the binding pocket of the
aldose reductase protein (PDB ID: <a href="https://www.rcsb.org/structure/4JIR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">4JIR</a>) during a 100 ns molecular dynamics (MD)
simulation. The interaction types are grouped into five categories:
hydrogen bonds (green), water bridges (blue), ionic interactions (pink),
hydrophobic interactions (gray), and halogen bonds (orange). Among
these, hydrophobic interactions were the most prevalent, especially
with residues such as TRP20, TRP79, PHE115, PHE121, ILE260, and LEU300,
suggesting that nonpolar interactions significantly contribute to
ligand stabilization within the binding pocket. Hydrogen bonding,
a key determinant of binding specificity and directionality, was particularly
prominent with GLY169, ASN160, and SER210, indicating strong polar
interactions that may enhance the binding affinity. Notably, GLY169
displayed the highest hydrogen bond frequency, indicating its crucial
role in anchoring the ligand. Water bridges, which involve the mediation
of interactions through water molecules, were also consistently observed,
especially with residues such as GLN183, SER210, and LEU212, reflecting
a dynamic solvation shell around the ligand that may aid in flexibility
and conformational adaptability. Minor but significant ionic interactions
were present with LYS21 and ASP216, contributing to halogen bonding
was observed, such as with VAL47, indicating the possible role of
halogenated moieties in ligand design. The interaction profile underscores
a complex and cooperative binding mechanism, where hydrophobic interactions
provide anchoring and stability, hydrogen bonds specificity, and water
bridges and electrostatic interactions support binding retention of
the ligand within the binding pocket of the aldose reductase enzyme
(<a href="#fig10" class="usa-link">Figure 
</a>).</p>
<figure class="fig xbox font-sm" id="fig10"><h5 class="obj_head">10.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332697_ao5c04663_0010.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9e07/12332697/f6fd03734283/ao5c04663_0010.jpg" loading="lazy" id="gr12" height="249" width="700" alt="10"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) Protein
ligand contacts of <strong>5o</strong> with the respective
amino acids of the protein; (b) ligand protein contacts of compound <strong>5o</strong> with the respective amino acids of the aldose reductase
protein (<a href="https://www.rcsb.org/structure/4JIR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">4JIR</a>).</p></figcaption></figure></section><section id="sec2.4.3"><h4 class="pmc_sec_title">Density Functional Theory</h4>
<p>In quantum energy analysis,
molecular orbital analysis is essential for gaining insights into
a compound’s chemical characteristics. The most important orbitals
for this purpose are the highest occupied molecular orbital (HOMO)
and the lowest-unoccupied molecular orbital (LUMO). The HOMO reflects
the compound’s electron-donating ability, while the LUMO energy
indicates its tendency to accept electrons. These parameters directly
impact the compound’s stability, reactivity, and kinetics.
For compound <strong>5o</strong>, we conducted DFT (B3LYP/6–31G
(d, p)) calculations to evaluate its electronic properties and reactivity
descriptors. The HOMO and LUMO energies were observed to be −0.221459
and −0.108893 eV, respectively, leading to an energy difference
of 0.112566 eV (<a href="#fig11" class="usa-link">Figure 
</a>). This small gap suggests good chemical reactivity and potential
for electronic transitions. From this energy gap, the hardness (η)
was calculated to be 0.056283 eV, while the corresponding softness
(S) was 17.77 eV<sup>–1</sup>, indicating that the compound
is a soft molecule with high reactivity in biological environments
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c04663/suppl_file/ao5c04663_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S5</a>). The dipole moment of 2.474503
D suggests moderate molecular polarity, which is indicative of the
potential for intermolecular interactions. Furthermore, using an assumed
TD-DFT optical excitation energy of 0.08 eV, the exciton binding energy
was estimated to be 0.032566 eV, supporting the compound’s
capacity for low-energy electronic transitions. Finally, the optimized
molecular energy of the compound in the gas phase was computed as
2449.512757 kcal/mol, confirming its thermodynamic stability (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c04663/suppl_file/ao5c04663_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S6</a>).</p>
<figure class="fig xbox font-sm" id="fig11"><h5 class="obj_head">11.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332697_ao5c04663_0011.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9e07/12332697/bfa879c9406b/ao5c04663_0011.jpg" loading="lazy" id="gr13" height="403" width="785" alt="11"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) Molecular orbital contour plots of
HOMO and LUMO and (b) ESP
of <strong>5o</strong>.</p></figcaption></figure></section><section id="sec2.4.4"><h4 class="pmc_sec_title">ADMET Properties</h4>
<p>The QikProp parameters provide valuable
insights into the drug-likeness characteristics along with the absorption,
distribution, metabolism, and excretion (ADME) profiles of the synthesized
quinoline-based hydrazones (<strong>5a</strong>–<strong>5q</strong>) (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c04663/suppl_file/ao5c04663_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S4</a>). The solubility parameter
(QPlogS) values range from −8.917 to −6.727, with lower
values indicating lower aqueous solubility. The lipophilicity (QPlogPo/w)
ranges from 4.049 to 6.489, suggesting a generally high lipophilic
nature, especially for compounds such as <strong>5c</strong>, <strong>5l</strong>, and <strong>5q</strong>, which could support membrane permeability
but may also pose a risk for poor solubility. The polar surface area
(PSA) of the compounds lies between 77.787 and 137.430 Å<sup>2</sup>, with compounds like <strong>5f</strong>, <strong>5i</strong>, and <strong>5o</strong> showing relatively higher PSA, which might limit blood-brain
barrier penetration. This is confirmed by QPlogBB values, where most
compounds have values less than −0.5, indicating poor brain
permeability, except a few like <strong>5n</strong> and <strong>5c</strong>, which have better potential for central nervous system (CNS) exposure.
The CNS activity predictions also align with these values, where most
compounds are assigned scores of −1 or −2, signifying
little to no CNS activity. The Caco-2 and MDCK cell permeability values
(QPP Caco and QPP MDCK) indicate good intestinal absorption and blood-brain
barrier permeability for most of the compounds, particularly <strong>5c</strong>, <strong>5d</strong>, <strong>5e</strong>, <strong>5l</strong>, <strong>5m</strong>, <strong>5n</strong>, and <strong>5q</strong>, which have values
reaching up to or beyond the upper detection limits (10,000 nm/s for
MDCK). This suggests a promising oral bioavailability. The QPlog Khsa
values, representing the binding affinity to human serum albumin,
fall in the ideal range (∼0.1 to 0.9), indicating balanced
protein binding that supports drug transport in plasma. HERG channel
inhibition, which can signal potential cardiotoxicity, is represented
by QPlogHERG. Most compounds exhibit values below −7.0, indicating
minimal risk, except for <strong>5f</strong>, which shows a more concerning
value of −9.101. Drug-likeness is also assessed via the number
of violations observed with respect to Lipinski’s rule of five
(RoF) and rule of three (RoT), with most compounds showing zero to
two violations, indicating acceptable oral drug-likeness. Hydrogen
bond donors (HBD) and acceptors (HBA) vary slightly across the series,
with <strong>5i</strong> having the highest HBD (3.0) and HBA values peaking
around 9.95, consistent with good pharmacophoric features. Overall,
compounds <strong>5c</strong>, <strong>5l</strong>, <strong>5m</strong>, and <strong>5q</strong> stand out for their favorable pharmacokinetic profiles,
with strong absorption, moderate lipophilicity, and acceptable safety
margins, highlighting their potential as promising candidates for
further investigations (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c04663/suppl_file/ao5c04663_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S4</a>).</p></section></section></section><section id="sec3"><h2 class="pmc_sec_title">Discussion</h2>
<p>This study presents the synthesis and comprehensive
in vitro biological
evaluation of a novel series of quinoline-based hydrazones (<strong>5a</strong>–<strong>5q</strong>) as potential antidiabetic agents.
The synthetic strategy employed a two-step reflux procedure, beginning
with the condensation of 4-((7-chloroquinolin-4-yl) oxy)-3-methoxybenzaldehyde
(<strong>1</strong>) with thiocarbohydrazide (<strong>2</strong>) to yield
an intermediate compound (3). This was subsequently reacted with a
variety of aryl and heteroaryl aldehydes to afford the final Schiff
bases (<strong>5a</strong>–<strong>5q</strong>). The structural identity
of the synthesized compounds was confirmed by using multiple spectroscopic
techniques. <sup>1</sup>H and <sup>13</sup>C NMR spectra exhibited
characteristic signals corresponding to −OCH<sub>3</sub>, −NH,
−CH, CS, and C–O functionalities, while FTIR
spectroscopy revealed distinct absorption bands attributable to −NH<sub>2</sub>, −NH, −OH, aromatic C–H, and imine (CHN)
stretching vibrations. Additionally, LCMS analysis confirmed the molecular
weights, supporting the successful synthesis of the target molecules.
A critical evaluation of the literature highlights that many effective
metabolic enzyme inhibitors share common pharmacophoric elements,
particularly the quinoline scaffold, an imine (CN) or hydrazone
linkage, and varied substitution patterns on the aldehyde moiety.
Although imine groups are often associated with limited metabolic
stability due to their susceptibility to hydrolysis under physiological
conditions, hydrazone derivatives, being a more stable subclass of
imines, have garnered attention in medicinal chemistry. This is attributed
to their greater resonance stabilization, hydrogen-bonding capability,
and potential for metal ion coordination. Several hydrazone-containing
compounds have shown significant in vivo efficacy, suggesting that
with suitable structural modifications, metabolic stability can be
enhanced. In this work, the hydrazone linkage was deliberately retained
to enable key interactions within the enzyme active sites. Moreover,
the strategic incorporation of electron-donating and sterically hindered
aromatic or heterocyclic substituents was employed to protect the
imine bond from hydrolytic degradation and improve pharmacokinetic
properties. Thus, the design and development of this quinoline-based
benzo hydrazide series reflect a rational approach that balances the
need for essential pharmacophoric features with chemical stability.
These compounds were specifically designed as multifunctional enzyme
inhibitors, aiming to target α-glucosidase, α-amylase,
and aldose reductase, which are key metabolic enzymes implicated in
the pathogenesis of type 2 diabetes mellitus.</p>
<p>A crucial aspect
of this study was the biological evaluation of
these quinolone-based hydrazones for their inhibitory activities against
key enzymes implicated in carbohydrate metabolism and diabetic complications:
human pancreatic α-amylase (HPA), human lysosomal acid α-glucosidase
(HLAG), and aldose reductase (AR). The results demonstrate a promising
broad-spectrum inhibitory profile across the synthesized series. Notably,
compound <strong>5o</strong> (5-nitrofuran) emerged as the most potent
inhibitor for all three target enzymes, exhibiting IC<sub>50</sub> values of 7.44 ± 0.07 μg/mL for α-glucosidase,
21.05 ± 0.17 μg/mL for α-amylase, and 4.12 ±
0.09 μg/mL for aldose reductase. These activities are particularly
significant as compound <strong>5o</strong> surpassed the efficacy of
the standard drugs acarbose (for α-glucosidase and α-amylase)
and quercetin (for aldose reductase). The superior inhibitory potential
of <strong>5o</strong> highlights its strong candidacy for further development
as an antidiabetic agent, suggesting that the nitrofuran moiety plays
a critical role in enhancing binding affinity and enzyme inhibition.
This aligns with recent findings on the antidiabetic potential of
novel compounds targeting α-glucosidase and α-amylase.<em>
,
</em> Other notable compounds like <strong>5e</strong> (4-bromo) and <strong>5m</strong> (4-bromothiophene) also displayed strong inhibitory effects,
especially against α-amylase and aldose reductase, indicating
the importance of halogen substituents in modulating activity, which
has been observed in other recent antidiabetic drug discovery efforts.<em>
,
</em> The overall findings from this study corroborate the growing evidence
for the therapeutic utility of heterocyclic scaffolds and their derivatives
in managing diabetes, particularly through enzyme inhibition.<em>
,,
</em>
</p>
<p>Kinetic analysis of aldose
reductase inhibition with compound <strong>5o</strong> further elucidated
its mechanism of action. The Lineweaver–Burk
plots demonstrated a classical noncompetitive inhibition, characterized
by an unchanged <em>K</em>
<sub>m</sub> value and a concentration-dependent
reduction in <em>V</em>
<sub>max</sub>. This indicates that <strong>5o</strong> reduces enzyme activity without affecting substrate binding
affinity, a desirable feature for therapeutic interventions. The determined
inhibition constant (<em>K</em>
<sub>i</sub>) of 4.65 μM
from Dixon plot analysis further confirmed its potent and reversible
noncompetitive inhibition. This mode of inhibition suggests that <strong>5o</strong> binds to a site distinct from the substrate binding pocket,
leading to a conformational change that impairs the catalytic efficiency
of aldose reductase. Such noncompetitive inhibitors are often advantageous
as they can overcome high substrate concentrations, providing a robust
therapeutic strategy.</p>
<p>The structure–activity relationship
(SAR) analysis underscored
the pivotal role of specific substituents on the aryl ring of the
hydrazone moiety in influencing enzyme inhibitory effects. The presence
of electron-withdrawing groups (EWGs), exemplified by the 5-nitrofuran
substituent in <strong>5o</strong>, significantly enhanced binding affinity
and inhibitory activity across all tested enzymes. Halogen substitutions,
as seen in <strong>5e</strong> (4-bromo) and <strong>5m</strong> (4-bromothiophene),
also contributed to strong inhibitory effects, especially against
α-amylase, suggesting the importance of these groups in establishing
favorable interactions with the enzyme active sites. This is consistent
with trends observed in the design of various enzyme inhibitors, where
electronic and steric effects of the substituents are critical. Computational
studies, including molecular docking, provided crucial insights into
the molecular basis of the observed inhibitory activities. Docking
studies revealed favorable binding interactions of the quinolone-based
hydrazone derivatives with α-glucosidase, α-amylase, and
aldose reductase. Compound <strong>5o</strong> demonstrated a remarkable
docking score of −10.051 against aldose reductase, comparable
to the reference drug Epalrestat, and displayed extensive stabilizing
interactions, including π–π stacking, hydrogen
bonds, and a salt bridge. This strong binding affinity was further
supported by a favorable MM/GBSA binding energy of −79.19 kcal/mol.
The detailed interaction analysis for <strong>5o</strong> with aldose
reductase, highlighting interactions with key residues such as TRP20,
TRP111, GLN183, ASN160, SER210, SER214, and LYS21, provides a strong
basis for its potent inhibitory activity. Similarly, <strong>5i</strong> showed superior binding to α-amylase and <strong>5k</strong> to
α-glucosidase, through a variety of intermolecular forces. These
computational findings are consistent with the experimental data and
provide a strong rationale for the observed biological activities.
Molecular dynamics (MD) simulations for the aldose reductase-<strong>5o</strong> complex further validated the stability of the binding
interaction. The consistent RMSD values for the protein backbone and
moderate flexibility of the ligand throughout the 100 ns simulation
confirmed the stable accommodation of <strong>5o</strong> within the active
site. RMSF analysis highlighted the rigidity of most protein residues,
with specific flexible regions, but overall confirmed the conformational
integrity of the protein and stable binding behavior of the ligand.
Intermolecular contacts revealed that hydrophobic interactions with
residues such as TRP20, TRP79, PHE115, PHE121, ILE260, and LEU300
were most prevalent, contributing significantly to ligand stabilization.
Hydrogen bonding, particularly with GLY169, ASN160, and SER210, played
a crucial role in anchoring the ligand, while water bridges and ionic
interactions further supported binding retention. This detailed understanding
of the dynamic interactions provides a robust foundation for the further
optimization of these compounds. Density functional theory (DFT) calculations
for compound <strong>5o</strong> provided insights into its electronic
properties and reactivity. The small HOMO–LUMO energy gap (0.112566
eV) indicated good chemical reactivity and potential for electronic
transitions, consistent with its biological activity. The calculated
hardness (0.056283 eV) and softness (17.77 eV<sup>–1</sup>)
values classify <strong>5o</strong> as a soft molecule, suggesting high
reactivity in biological environments. The moderate dipole moment
(2.474503 D) points to its potential for intermolecular interactions,
further supporting its binding capabilities within enzymatic pockets.
Finally, the ADMET properties analysis, although presented in the <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c04663/suppl_file/ao5c04663_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Information</a>, provides essential insights
into the drug-likeness and pharmacokinetic profiles of the synthesized
compounds. While some compounds exhibited moderate to low aqueous
solubility and poor brain permeability, several derivatives, particularly <strong>5c</strong>, <strong>5l</strong>, <strong>5m</strong>, and <strong>5q</strong>,
showed promising oral bioavailability with good intestinal absorption
and acceptable safety margins. The generally low HERG channel inhibition
risk is also a favorable indicator of potential drug development.
These ADMET predictions, in conjunction with the strong in vitro inhibitory
activities, underscore the potential of this series of quinolone-based
hydrazones as promising lead candidates for the development of novel
antidiabetic therapies, warranting further in vivo investigations
and lead optimization. This comprehensive approach, integrating synthesis,
biological evaluation, and computational studies, provides a strong
foundation for the continued exploration of quinolone-based hydrazones
in the management of diabetes mellitus, aligning with the ongoing
search for effective and safe antidiabetic agents.<em>
−

</em>
</p></section><section id="sec4"><h2 class="pmc_sec_title">Conclusions</h2>
<p>The successful synthesis of a series of
quinolone-based hydrazones
(<strong>5a</strong>–<strong>5q</strong>) and their biological evaluation
have revealed their promising potential as antidiabetic agents by
modulating key enzymes involved in carbohydrate metabolism and diabetic
complications. Among these, compound <strong>5o</strong> (5-nitrofuran)
stood out, demonstrating the most potent inhibitory activity against
both α-amylase and α-glucosidase, surpassing the efficacy
of the reference standard drug acarbose. Additionally, <strong>5o</strong> showed significant inhibitory action against aldose reductase, outperforming
quercetin. Kinetic studies indicated that <strong>5o</strong> inhibits
aldose reductase via a reversible, noncompetitive mechanism. Molecular
docking studies supported these findings by revealing favorable binding
interactions of <strong>5o</strong> within the active sites of all three
enzymes. The stability of the <strong>5o</strong>–aldose reductase
complex was further confirmed by molecular dynamics simulations, while
DFT analysis provided insights into its electronic properties and
reactivity, highlighting features conducive to biological activity.
Together, these <em>in vitro</em> and computational results
underscore the potential of quinolone-based hydrazones, particularly
compound <strong>5o</strong>, as promising multitarget antidiabetic agents.
The next step will involve rigorous animal model studies to comprehensively
evaluate <strong>5o</strong>’spharmacological profile, safety,
and therapeutic efficacy, paving the way for its potential clinical
translation.</p></section><section id="sec5"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="sec5.1"><h3 class="pmc_sec_title">Chemistry</h3>
<p>Melting
points were determined by using open
capillary tubes with a standard melting point apparatus. NMR spectra
(<sup>1</sup>H at 400 MHz and <sup>13</sup>C at 100 MHz) were recorded
at room temperature on a Bruker spectrometer, with chemical shifts
(δ) reported in ppm and coupling constants (J) in Hz. Liquid
chromatography mass spectra (LCMS) were acquired using a Shimadzu
LCMS-Impact HD instrument, IR spectra were recorded on a Shimadzu
IR Affinity-1 spectrophotometer, and all commercial solvents and reagents
were used as received without further purification. 7-Chloroquinoline
was sourced from Acros Organics (New Jersey, USA). Computational studies,
including molecular docking, molecular dynamics simulations, and DFT
calculations, were conducted using Schrödinger’s Glide,
Prime, Desmond, and Jaguar modules, with Maestro for visualization.</p>
<p>General procedure for the synthesis of quinolone-based hydrazones
(<strong>5a</strong>–<strong>5q</strong>): A mixture of 4-((7-chloroquinolin-4-yl)
oxy)-3-methoxybenzaldehyde (<strong>1</strong>) was treated with thiocarbohydrazide
(<strong>2</strong>) and catalytic proportion of glacial acetic acid (0.001
mol) in methanol (10 mL) was refluxed for 24h, with progress monitored
by thin-layer chromatography (TLC). Then the reaction mixture was
cooled to room temperature. The formed precipitate was filtered and
washed with ethanol to obtain the reaction intermediate compound (<strong>3</strong>) by the previously reported method by Salve and Alegaon. In the subsequent step, <em>N</em>′-(4-((7-chloroquinolin-4-yl)
oxy)-3-methoxybenzylidene) hydrazinecarbothiohydrazide (<strong>3</strong>) was reacted with appropriate aryl or heteroaryl aldehydes (<strong>4a</strong>–<strong>4q</strong>) and glacial acetic acid (0.001
mol) in a catalytic ratio in methanol (10 mL). The reaction mixture
was refluxed for 24 h and was monitored using TLC. Upon completion
of the reaction, the precipitate was filtered, washed with alcohol,
and dried to yield the final series of synthesized quinoline-based
hydrazones (<strong>5a</strong>–<strong>5q</strong>) in 65–96%
yields.</p>
<section id="sec5.1.1"><h4 class="pmc_sec_title">Spectral Data of Synthesized Hydrazinecarbothiohydrazide (<strong>3</strong>)</h4>
<section id="sec5.1.1.1"><h5 class="pmc_sec_title">
<em>N</em>′-(4-((7-Chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)
Hydrazinecarbothiohydrazide (<strong>3</strong>)</h5>
<p>Pale-yellow
solid. Yield: 83%; mp: 190–192 °C; FTIR (KBr, cm<sup>–1</sup>): 3564 (−NH<sub>2</sub>), 3440 (N–H), 3166 (Ar–CH),
1570 (CC), 1520 (CN), 1206 (C–O), 1158 (CS). <sup>1</sup>H NMR (400 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 3.79 (s, 3H, −OCH<sub>3</sub>), 4.94 (s, 2H, −NH<sub>2</sub>), 6.50–6.49 (d, 1H, Ar–H), 7.40–7.31
(m, 2H, Ar–H), 7.67–7.65 (m, 1H, Ar–H), 7.85
(s, 1H, −CH), 8.07–8.06 (d, 2H, Ar–H), 8.34–8.32
(d, 1H, Ar–H), 8.68–8.66 (d, 1H, Ar–H), 9.99
(s, 1H, −NH), 11.53 (s, 1H, −NH). D<sub>2</sub>O EXCHANGE:
6.45–6.44 (d, 1H, Ar–H), 7.36–7.23 (m, 2H, Ar–H),
7.69–7.61 (m, 1H, Ar–H), 7.99 (s, 1H, −CH), 8.07–8.06
(d, 2H, Ar–H), 8.34–8.27 (d, 1H, Ar–H), 8.59–8.58
(d, 1H, Ar–H). <sup>13</sup>C NMR (100 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 56.63, 104.15, 111.26, 119.26, 122.28, 123.23,
124.24, 127.28, 127.86, 134.09, 135.29, 141.71, 142.88, 149.99, 151.92,
153.34, 161.13, 176.62. Molecular mass; calculated: C<sub>18</sub>H<sub>16</sub>ClN<sub>5</sub>O<sub>2</sub>S; 401.87; found LCMS; <em>m</em>/<em>z</em> 402 (M), 403 (M + 1), 404 (M + 2).</p></section></section><section id="sec5.1.2"><h4 class="pmc_sec_title">Spectral Data of Synthesized Quinolone-Based Hydrazones (<strong>5a</strong>–<strong>5q</strong>)</h4>
<section id="sec5.1.2.1"><h5 class="pmc_sec_title">
<em>N</em>′-((<em>E</em>)-Substituted
Benzylidene)-2-((<em>E</em>)-4-((7-chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)
Hydrazine-1-carbothiohydrazide (<strong>5a</strong>)</h5>
<p>Buff white
powder. Yield: 95%; mp: 174–176 °C; FTIR (KBr, cm<sup>–1</sup>): 3539 (N–H), 3132 (Ar–CH), 2988 (−CH<sub>3</sub>), 1549 (CC), 1520 (CN), 1253 (C–O),
1165 (CS). <sup>1</sup>H NMR (400 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 2.33 (s, 3H, −CH<sub>3</sub>), 3.80
(s, 3H, −OCH<sub>3</sub>), 6.53–6.52 (d, 1H, Ar–H),
7.27–7.25 (d, 2H, Ar–H), 7.51–7.39 (m, 2H, Ar–H),
7.84–7.66 (m, 4H, Ar–H), 8.36–8.08 (m, 3H,–Ar-H),
8.70–8.69 (d, 2H,–Ar-H), 11.62 (s, 1H, −NH),
11.99–11.92 (d, 1H, −NH). <sup>13</sup>C NMR (100 MHz,
δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 21.51, 56.45,
104.19, 111.24, 113.19, 119.27, 121.94, 123.52, 124.25, 127.31, 127.88,
129.81, 131.64, 134.56, 135.329, 140.39, 143.10, 144.42, 147.84, 148.26,
149.76, 150.02, 151.86, 153.35, 161.09, 175.17. Molecular mass; calculated:
C<sub>26</sub>H<sub>22</sub>ClN<sub>5</sub>O<sub>2</sub>S; 504.01;
found LCMS; <em>m</em>/<em>z</em> 504 (M), 505 (M
+ 1), 506 (M + 2).</p></section><section id="sec5.1.2.2"><h5 class="pmc_sec_title">
<em>N</em>′-((<em>E</em>)-Benzylidene)-2-((<em>E</em>)-4-((7-chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)
Hydrazine-1-carbothiohydrazide (<strong>5b</strong>)</h5>
<p>White powder.
Yield: 95%; mp: 170–172 °C; FTIR (KBr, cm<sup>–1</sup>): 3534 (N–H), 3122 (Ar–CH), 1549 (CN), 1506
(CC), 1210 (C–O), 1166 (CS). <sup>1</sup>H
NMR (400 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>):
3.77 (s, 3H, −OCH<sub>3</sub>), 6.50–6.49 (d, 1H, Ar–H),
7.48–7.37 (m, 4H, Ar–H), 7.86–7.52 (m, 4H, Ar–H),
8.058–8.053 (d, 2H, Ar–H), 8.1 (s, 1H, −CH),
8.34–8.32 (d, 1H, −CH), 8.56 (s, 1H, Ar–H), 8.67–8.66
(d, 1H, Ar–H), 11.64 (s, 1H, −NH) 11.99–11.96
(d, 1H, −NH). <sup>13</sup>C NMR (100 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 56.44, 104.19, 111.69, 112.23, 119.25, 121.88,
123.53, 124.26, 127.36, 127.84, 129.21, 130.55, 133.92, 134.52, 135.35,
143.11, 143.98, 148.44, 149.62, 149.98, 151.85, 153.37, 158.43, 161.09,
175.28. Molecular mass; calculated: C<sub>25</sub>H<sub>20</sub>ClN<sub>5</sub>O<sub>2</sub>S; 489.98; found LCMS; <em>m</em>/<em>z</em> 490 (M), 491 (M + 1), 492 (M + 2).</p></section><section id="sec5.1.2.3"><h5 class="pmc_sec_title">
<em>N</em>′-((<em>E</em>)-4-chlorobenzylidene)-2-((<em>E</em>)-4-((7-chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)
Hydrazine-1-carbothiohydrazide (<strong>5c</strong>)</h5>
<p>White powder.
Yield: 96%; mp: 180–182 °C; FTIR (KBr, cm<sup>–1</sup>): 3549 (N–H), 3136 (Ar–CH), 1548 (CN), 1545
(CC), 1252 (C–O), 1166 (CS). <sup>1</sup>H
NMR (400 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>):
3.77 (s, 3H, −OCH<sub>3</sub>), 6.50 (s, 1H, Ar–H),
7.39–7.37 (d, 2H, Ar–H), 7.55–7.48 (d, 2H, Ar–H),
7.67–7.65 (d, 1H, Ar–H), 7.73 (s, 1H, Ar–H),
7.88 (s, 1H, Ar–H), 8.05­(s, 1H, Ar–H), 8.17 (s, 1H,
−CH), 8.34–8.32 (d, 1H, −CH), 8.54 (s, 1H, Ar–H),
8.67–8.65 (d, 2H, Ar–H), 11.69 (s, 1H, −NH) 12.02
(s, 1H, −NH). <sup>13</sup>C NMR (100 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 56.45 104.20, 11.86, 119.25, 121.88, 123.55,
124.27, 125.79, 127.38, 12784, 129.31, 133.67, 134.97, 135.36, 137.29,
139.34, 143.14, 148.1, 149.98, 151.86, 153.38, 158.98, 159.16, 161.09,
175.34. Molecular mass; calculated: C<sub>25</sub>H<sub>19</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>2</sub>S; 524.42; found LCMS; <em>m</em>/<em>z</em> 524 (M), 525 (M + 1), 526 (M + 2).</p></section><section id="sec5.1.2.4"><h5 class="pmc_sec_title">
<em>N</em>′-((<em>E</em>)-4-((7-Chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)-2-((<em>E</em>)-4-fluorobenzylidene) Hydrazine-1-carbothiohydrazide (<strong>5d</strong>)</h5>
<p>Buff white powder. Yield: 96%; mp: 184–186
°C; FTIR (KBr, cm<sup>–1</sup>): 3290 (N–H), 3048
(Ar–CH), 1551 (CN), 1522 (CC), 1232 (C–O),
1127 (CS). <sup>1</sup>H NMR (400 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 3.77 (s, 3H, −OCH<sub>3</sub>), 6.51–6.50
(d, 1H, Ar–H), 7.29–7.25 (m, 2H, Ar–H), 7.39–7.37
(d, 1H, Ar–H), 7.57 (s, 1H, Ar–H), 7.68–7.65
(m, 1H, Ar–H), 7.73 (s, 1H, Ar–H), 7.84 (s, 1H, Ar–H),
7.91 (s, 1H, Ar–H), 8.06–8.05 (d, 1H, Ar–H),
8.16 (s, 1H, −CH), 8.34–8.32 (d, 1H, −CH), 8.54
(s, 1H, Ar–H), 8.67–8.66 (d, 1H, Ar–H), 11.65
(s, 1H, −NH) 11.98–11.93 (d, 1H, −NH). <sup>13</sup>C NMR (100 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 56.45, 104.20, 111.23, 116.19, 116.41, 119.25, 121.28, 123.68,
123.84, 124.28, 127.38, 127.85, 131.16, 134.57, 135.36, 141.77, 143.12,
148.33, 149.99, 151.86, 153.39, 161.09, 161.88, 163.38, 175.29. Molecular
mass; calculated: C<sub>25</sub>H<sub>19</sub>ClFN<sub>5</sub>O<sub>2</sub>S; 507.97; found LCMS; <em>m</em>/<em>z</em> 508 (M), 509 (M + 1), 510 (M + 2).</p></section><section id="sec5.1.2.5"><h5 class="pmc_sec_title">
<em>N</em>′-((<em>E</em>)-4-Bromobenzylidene)-2-((<em>E</em>)-4-((7-chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)
Hydrazine-1-carbothiohydrazide (<strong>5e</strong>)</h5>
<p>White powder.
Yield: 94%; mp: 196–198 °C; FTIR (KBr, cm<sup>–1</sup>): 3445 (N–H), 3136 (Ar–CH), 1545 (CN), 1563
(CC), 1211 (C–O), 1165 (CS). <sup>1</sup>H
NMR (400 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>):
3.81 (s, 3H, −OCH<sub>3</sub>), 6.54–6.53 (d, 1H, Ar–H),
7.44.7.42 (d, 1H, Ar–H), 7.62 (s, 1H, Ar–H), 7.72–7.66
(m, 4H, Ar–H), 7.84 (s, 2H, Ar–H), 8.10–8.09
(d, 1H, Ar–H), 8.21 (s, 1H, −CH), 8.38–8.36 (d,
1H, −CH), 8.56 (s, 1H, Ar–H), 8.71–8.70 (d, 1H,
Ar–H), 11.72 (s, 1H, −NH) 12.08–12.03 (d, 1H,
−NH). <sup>13</sup>C NMR (100 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 56.02, 103.73, 111.54, 112.09, 118.81, 119.89,
121.72, 123.10, 123.35, 123.80, 126.89, 127.44, 129.01, 131.77, 133.46,
134.90, 142.46, 142.68, 147.49, 147.68, 149.57, 151.41, 152.92, 160.64,
174.90. Molecular mass; calculated: C<sub>25</sub>H<sub>19</sub>BrClN<sub>5</sub>O<sub>2</sub>S; 568.87; found LCMS; <em>m</em>/<em>z</em> 568.42 (M), 569 (M + 1), 570 (M + 2).</p></section><section id="sec5.1.2.6"><h5 class="pmc_sec_title">
<em>N</em>′-((<em>E</em>)-4-((7-Chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)-2-((<em>E</em>)-4-formylbenzylidene) Hydrazine-1-carbothiohydrazide (<strong>5f</strong>)</h5>
<p>Yellow powder. Yield: 95%; mp: 188–190
°C; FTIR (KBr, cm<sup>–1</sup>): 3516 (N–H), 3139
(Ar–CH), 1692 (CO), 1547 (CN), 1527 (CC),
1211 (C–O), 1166 (CS). <sup>1</sup>H NMR (400 MHz,
δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 3.82 (s, 3H, −OCH<sub>3</sub>), 6.55–6.54 (d, 1H, Ar–H), 7.48–7.42
(d, 2H, Ar–H), 7.51 (s, 1H, Ar–H), 7.77–7.62
(m, 3H, Ar–H), 8.05–7.98 (d, 2H, Ar–H), 8.105–8.101
(d, 1H, Ar–H), 8.12 (s, 1H, −CH), 8.38–8.36 (d,
1H, −CH), 8.71–8.70 (d, 2H, Ar–H), 10.05 (s,
1H, −CHO), 11.85 (s, 1H, −NH) 12.18 (s, 1H, −NH). <sup>13</sup>C NMR (100 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 55.98, 103.71, 103.78, 112.19, 118.81, 121.68, 123.17,
123.83, 126.90, 127.46, 129.89, 133.62, 134.90, 136.70, 139.71, 142.03,
142.74, 1447.89, 148.44, 149.57, 151.43, 152.88, 152.97, 160.63, 170.07,
192.66. Molecular mass; calculated: C<sub>26</sub>H<sub>20</sub>ClN<sub>5</sub>O<sub>3</sub>S; 517.99; found LCMS; <em>m</em>/<em>z</em> 518 (M), 519 (M + 1), 520 (M + 2).</p></section><section id="sec5.1.2.7"><h5 class="pmc_sec_title">
<em>N</em>′-((<em>E</em>)-4-((7-Chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)-2-(thiophen-2-ylmethylene)
Hydrazine-1-carbothiohydrazide (<strong>5g</strong>)</h5>
<p>Buff white
powder. Yield: 65%; mp: 182–184 °C; FTIR (KBr, cm<sup>–1</sup>): 3498 (N–H), 3132 (Ar–CH), 1563 (CN),
1506 (CC), 1211 (C–O), 1167 (CS), 856 (C–S). <sup>1</sup>H NMR (400 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 3.82 (s, 3H, −OCH<sub>3</sub>), 6.54–6.53
(d, 1H, Ar–H), 7.15 (m, 1H, Ar–H), 7.72–7.41
(m, 5H, Ar–H), 8.107–8.102 (d, 1H, Ar–H), 8.20
(s, 1H, −CH), 8.38–8.36 (d, 1H, −CH), 8.71–8.70
(d, 2H, Ar–H), 8.78 (s, 1H, Ar–H), 11.63 (s, 1H, −NH)
12.00 (s, 1H, −NH). <sup>13</sup>C NMR (100 MHz, δ, ppm,
DMSO-<em>d</em>
<sub>6</sub>): 56.07, 103.73, 112.13, 113.51,
118.81, 121.27, 123.05, 123.82, 126.91, 127.44, 127.90, 129.15, 130.95,
133.38, 134.90, 138.68, 142.65, 144.14, 149.57, 151.41, 152.93, 160.65,
174.39. Molecular mass; calculated: C<sub>23</sub>H<sub>18</sub>ClN<sub>5</sub>O<sub>2</sub>S<sub>2</sub>; 496.00; found LCMS; <em>m</em>/<em>z</em> 496 (M), 497 (M + 1), 498 (M + 2).</p></section><section id="sec5.1.2.8"><h5 class="pmc_sec_title">
<em>N</em>′-((<em>E</em>)-4-((7-Chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)-2-(pyridin-4-ylmethylene)
Hydrazine-1-carbothiohydrazide (<strong>5h</strong>)</h5>
<p>Pale-yellow
powder. Yield: 87%; mp: 118–120 °C; FTIR (KBr, cm<sup>–1</sup>): 3553 (N–H), 3077 (Ar–CH), 1545 (CN),
1506 (CC), 1184 (C–O), 1134 (CS). <sup>1</sup>H NMR (400 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 3.82 (s, 3H, −OCH<sub>3</sub>), 6.55–6.54 (d, 1H,
Ar–H), 7.44–7.42 (d, 2H, Ar–H), 7.77–7.69
(m, 3H, Ar–H), 7.86–7.80 (m, 1H, Ar–H), 8.105–8.10
(d, 1H, Ar–H), 8.21 (s, 1H, −CH), 8.38–8.36 (d,
1H, −CH), 8.59 (s. 1H, Ar–H), 8.71–8.68 (m, 3H,
Ar–H), 11.96 (s, 1H, −NH), 12.22 (s, 1H, −NH). <sup>13</sup>C NMR (100 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 56.05, 103.75, 111.19, 112.26, 118.81, 121.10, 122.14,
123.13, 123.81, 126.91, 127.46, 134.91, 141.27, 142.78, 143.05, 146.08,
148.62, 149.58, 150.20, 151.09, 151.44, 152.92, 160.63, 175.23. Molecular
mass; calculated: C<sub>24</sub>H<sub>19</sub>ClN<sub>6</sub>O<sub>2</sub>S; 490.97; found LCMS; <em>m</em>/<em>z</em> 491 (M), 492 (M + 1), 493 (M + 2).</p></section><section id="sec5.1.2.9"><h5 class="pmc_sec_title">
<em>N</em>′-((<em>E</em>)-4-((7-Chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)-2-((5-(hydroxymethyl)­furan-2-yl)­methylene)
Hydrazine-1-carbothiohydrazide (<strong>5i</strong>)</h5>
<p>Pale-yellow
powder. Yield: 92%; mp: 110–112 °C; FTIR (KBr, cm<sup>–1</sup>): 3554 (N–H), 3476–3262 (O–H),
3090 (Ar–CH), 1549 (CN), 1506 (CC), 1248 (C–O),
1144 (CS). <sup>1</sup>H NMR (400 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 3.81 (s, 3H, −OCH<sub>3</sub>), 4.48–4.46
(d, 2H, −CH<sub>2</sub>), 5.43 (s, 1H, Ar–H), 6.54–6.47
(m, 1H, Ar–H), 6.87 (s, 1H, Ar–H), 7.44–7.40
(m, 1H, Ar–H), 7.56 (s, 1H, Ar–H), 7.72–7.69
(d, 2H, Ar–H), 7.76 (s, 1H, Ar–H), 8.104–8.10
(d, 1H, Ar–H), 8.19 (s, 1H, −CH), 8.38–8.36 (d,
1H, −CH), 8.44 (s, 1H, Ar–H), 8.71–8.68 (d, 1H,
Ar–H), 11.57 (s, 1H, −NH) 12.01 (s, 1H, −NH). <sup>13</sup>C NMR (100 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 55.76, 56.04, 103.73, 109.41, 112.11, 115.24, 118.82, 121.33,
123.06, 123.83, 126.91, 127.45, 133.62, 134.90, 138.91, 142.37, 142.67,
148.36, 149.58, 151.41, 152.94, 158.03, 160.65, 174.48. Molecular
mass; calculated: C<sub>24</sub>H<sub>20</sub>ClN<sub>5</sub>O<sub>4</sub>S; 509.97; found LCMS; <em>m</em>/<em>z</em> 510 (M), 511 (M + 1), 512 (M + 2).</p></section><section id="sec5.1.2.10"><h5 class="pmc_sec_title">
<em>N</em>′-((<em>E</em>)-4-((7-Chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)-2-((<em>Z</em>)-4-nitrobenzylidene) Hydrazine-1-carbothiohydrazide (<strong>5j</strong>)</h5>
<p>Yellow powder. Yield: 83%; mp: 228–230
°C; FTIR (KBr, cm<sup>–1</sup>): 3353 (N–H), 3120
(Ar–CH), 1544 (CN), 1507 (CC), 1251 (C–O),
1165 (CS). <sup>1</sup>H NMR (400 MHz, δ, ppm, DMSO-<em>d</em>6: 3.18 (s, 3H, −OCH<sub>3</sub>), 6.53–6.52
(s, 1H, Ar–H), 7.43–7.40 (s, 1H, Ar–H), 7.49
(s, 1H, Ar–H), 7.61–7.59 (d, 1H, Ar–H), 7.70–7.68
(m, 1H, Ar–H), 7.76 (s, 1H, Ar–H), 8.08–7.99
(m, 1H, Ar–H), 8.16–8.14 (m, 1H, Ar–H), 8.23
(s, 1H, CH), 8.30–8.28 (s, 1H, CH), 8.37–8.35 (m, 3H,
Ar–H), 8.70–8.69 (d, 1HAr–H), 11.92 (s, 1H, NH),
12.20 (s, 1H, NH). <sup>13</sup>C NMR (100 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 56.50, 104.22, 119.26, 122.42, 122.54, 123.57,
124.28, 124.44, 127.39, 127.86, 128.61, 131.62, 133.27, 135.37, 137.81,
141.16, 143.25, 146.46, 148.22, 150.00, 151.88, 153.38, 155.31, 161.09,
175.63. Molecular mass; calculated: C<sub>25</sub>H<sub>19</sub>ClN<sub>6</sub>O<sub>4</sub>S; 534.98; found LCMS; <em>m</em>/<em>z</em> 535.56 (M), 536 (M + 1), 537 (M + 2).</p></section><section id="sec5.1.2.11"><h5 class="pmc_sec_title">
<em>N</em>′-((<em>E</em>)-4-((7-Chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)-2-((<em>E</em>)-4-(dimethylamino)­benzylidene) Hydrazine-1-carbothiohydrazide
(<strong>5k</strong>)</h5>
<p>Buff white powder. Yield: 76%; mp: 208–210
°C; FTIR (KBr, cm<sup>–1</sup>): 3327 (N–H), 3104
(Ar–CH), 2930 (−CH<sub>3</sub>), 1563 (CN),
1519 (CC), 1207 (C–O), 1182 (CS). <sup>1</sup>H NMR (400 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 2.95 (s, 3H, −CH<sub>3</sub>), 3.43 (s, 6H,–OCH<sub>3</sub>), 3.79 (s, 2H, Ar–H), 6.52 (s, 1H, Ar–H), 6.74–6.72
(s, 1H, Ar–H), 7.39–7.37 (s, 2H, Ar–H), 7.57
(s, 1H, Ar–H), 7.68–7.66 (s, 1H, Ar–H), 7.75
(s, 1H, Ar–H), 8.07 (s, 1H, Ar–H), 8.18 (s, 1H, −CH),
8.36–8.34 (d, 1H, −CH), 8.43 (s, 1H, −CH), 8.69–8.64
(m, 1H, Ar–H), 11.51–11.41 (d, 1H, −NH), 11.83–11.72
(d, 1H, −NH). <sup>13</sup>C NMR (100 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 39.28, 56.57, 104.18, 111.48, 112.13, 119.27,
121.75, 123.51, 124.26, 127.34, 127.84, 129.08, 129.46, 134.18, 134.21,
135.36, 142.98, 143.00, 145.27, 147.83, 149.99, 151.85, 152.00, 153.35,
161.12, 174.41, 133.79. Molecular mass; calculated: C<sub>27</sub>H<sub>25</sub>ClN<sub>6</sub>O<sub>2</sub>S; 533.05; found LCMS; <em>m</em>/<em>z</em> 533.64 (M), 534 (M + 1), 535 (M + 2).</p></section><section id="sec5.1.2.12"><h5 class="pmc_sec_title">
<em>N</em>′-((<em>E</em>)-4-((7-chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)-2-((<em>Z</em>)-3,4-dichlorobenzylidene) Hydrazine-1-carbothiohydrazide
(<strong>5l</strong>)</h5>
<p>Buff white powder. Yield: 86%; mp: 200–202
°C; FTIR (KBr, cm<sup>–1</sup>): 3287 (N–H), 3168
(Ar–CH), 1548 (CN), 1506 (CC), 1206 (C–O),
1158 (CS). <sup>1</sup>H NMR (400 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 3.79 (s, 3H, −OCH<sub>3</sub>), 5.05
(s, 1H, Ar–H), 6.53–6.49 (m, 3H, Ar–H), 7.42–7.31
(m, 3H, Ar–H), 7.75–7.65 (m, 2H, Ar–H), 7.85
(s, 1H, Ar–H), 8.060 (s, 1H, Ar–H), 8.32 (s, 1H, −CH),
8.36–8.34 (d, 1H, −CH), 9.99 (s, 1H, −NH), 11.53
(s, 1H, −NH). <sup>13</sup>C NMR (100 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 56.64, 104.17, 111.28, 119.28, 122.29, 123.24,
123.53, 124.25, 127.29, 127.87, 128.31, 131.83, 133.35, 134.10, 135.31,
141.73, 142.90, 143.21, 150.01, 151.87, 151.94, 153.35, 161.14, 176.64,
175.50. Molecular mass; calculated: C<sub>25</sub>H<sub>18</sub>Cl<sub>3</sub>N<sub>5</sub>O<sub>2</sub>S; 558.86; found LCMS; <em>m</em>/<em>z</em> 558.51 (M), 559 (M + 1), 560 (M + 2).</p></section><section id="sec5.1.2.13"><h5 class="pmc_sec_title">
<em>N</em>′-((<em>E</em>)-(5-Bromothiophen-2-yl)­methylene)-2-((<em>E</em>)-4-((7-chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)
Hydrazine-1-carbothiohydrazide (<strong>5m</strong>)</h5>
<p>Orange powder.
Yield: 85%; mp: 198–200 °C; FTIR (KBr, cm<sup>–1</sup>): 3306 (N–H), 3063 (Ar–CH), 1527 (CN), 1502
(CC), 1207 (C–O), 1165 (CS). <sup>1</sup>H
NMR (400 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>):
3.80 (s, 3H, OCH<sub>3</sub>), 6.53–6.52 (s, 1H, Ar–H),
7.30–7.25 (s, 1H, Ar–H), 7.41–7.38 (m, 1H, Ar–H),
7.47 (s, 1H, Ar–H), 7.55 (s, 1H, Ar–H), 7.73–7.601
(m, 2H, Ar–H), 8.076 (s, 1H, Ar–H), 8.22 (s, 1H, CH),
8.36–8.33 (d, 1H, CH), 8.69–8.68 (d, 2H, Ar–H),
11.76 (s, 1H, NH), 12.06 (s, 1H, NH). <sup>13</sup>C NMR (100 MHz,
δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 56.48, 104.19,
112.21, 115.36, 119.27, 121.79, 123.50, 124.25, 127.33, 127.88, 130.65,
131.71, 133.83, 135.35, 142.75, 143.16, 148.54, 150.02, 151.87, 153.35,
161.09, 174.92, 175.37. Molecular mass; calculated: C<sub>23</sub>H<sub>17</sub>BrClN<sub>5</sub>O<sub>2</sub>S<sub>2</sub>; 574.90;
found LCMS; <em>m</em>/<em>z</em> 574.47 (M), 575
(M + 1), 576 (M + 2).</p></section><section id="sec5.1.2.14"><h5 class="pmc_sec_title">
<em>N</em>′-((<em>E</em>)-4-((7-Chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)-2-((3-methylthiophen-2-yl)­methylene)
Hydrazine-1-carbothiohydrazide (<strong>5n</strong>)</h5>
<p>Brown powder.
Yield: 88%; mp: 186–188 °C; FTIR (KBr, cm<sup>–1</sup>): 3265 (N–H), 3040 (Ar–CH), 2964 (−CH<sub>3</sub>), 1566 (CN), 1506 (CC), 1207 (C–O), 1165
(CS). <sup>1</sup>H NMR (400 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 3.36 (s, 3H, CH<sub>3</sub>), 3.81 (s, 3H,
OCH<sub>3</sub>), 6.53–6.52 (d, 1H, Ar–H), 7.42–7.40
(d, 2H, Ar–H), 7.49 (s, 1H, Ar–H), 7.69–7.60
(m, 2H, Ar–H), 7.76 (s, 1H, Ar–H), 8.08 (s, 1H, Ar–H),
8.23 (s, 1H, CH), 8.36–8.34 (d, 1H, CH), 8.70–8.69 (d,
2H, Ar–H), 11.77 (s, 1H, NH), 12.08 (s, 1H, NH). <sup>13</sup>C NMR (100 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 19.65, 56.48, 104.21, 111.84, 112.12, 116.34, 119.28, 121.35,
123.54, 124.25, 127.34, 127.89, 135.35, 138.44, 139.87, 143.16, 143.36,
149.22, 150.02, 151.88, 153.36, 161.09, 174.79, 175.38. Molecular
mass; calculated: C<sub>24</sub>H<sub>20</sub>ClN<sub>5</sub>O<sub>2</sub>S<sub>2</sub>; 510.03; found LCMS; <em>m</em>/<em>z</em> 510.56 (M), 511 (M + 1), 512 (M + 2).</p></section><section id="sec5.1.2.15"><h5 class="pmc_sec_title">
<em>N</em>′-((<em>E</em>)-4-((7-Chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)-2-((5-nitrofuran-2-yl)­methylene)
Hydrazine-1-carbothiohydrazide (<strong>5o</strong>)</h5>
<p>Brown powder.
Yield: 92%; mp: 172–174 °C; FTIR (KBr, cm<sup>–1</sup>): 3620 (N–H), 3112 (Ar–CH), 1559 (CN), 1504
(CC), 1246 (C–O), 1165 (CS). <sup>1</sup>H
NMR (400 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>):
3.81 (s, 3H, OCH<sub>3</sub>), 6.52–6.51 (d, 1H, Ar–H),
7.24 (s, 1H, Ar–H), 7.42–7.40 (d, 2H, Ar–H),
7.56 (s, 1H, Ar–H), 7.68–7.66 (s, 1H, Ar–H),
7.80 (s, 1H, Ar–H), 8.07 (s, 1H, Ar–H), 8.23 (s, 1H,
CH), 8.36–8.33 (d, 1H, CH), 8.57 (s, 1H, Ar–H), 8.69–8.68
(s, 1H, Ar–H), 11.98 (s, 1H, NH), 12.29 (s, 1H, NH). <sup>13</sup>C NMR (100 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 56.51, 104.20, 111.27, 112.60, 115.28, 119.26, 121.95, 123.52,
124.24, 127.35, 127.87, 133.58, 135.35, 136.56, 143.33, 147.62, 150.01,
151.87, 152.32, 153.35, 161.07, 175.44, 159.74. Molecular mass; calculated:
C<sub>23</sub>H<sub>17</sub>ClN<sub>6</sub>O<sub>5</sub>S; 524.94;
found LCMS; <em>m</em>/<em>z</em> 525.59 (M), 526
(M + 1), 527 (M + 2).</p></section><section id="sec5.1.2.16"><h5 class="pmc_sec_title">
<em>N</em>′-((<em>E</em>)-4-((7-Chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)-2-((4-methylthiazol-5-yl)­methylene)
Hydrazine-1-carbothiohydrazide (<strong>5p</strong>)</h5>
<p>Pale-yellow
powder. Yield: 92%; mp: 218–220 °C; FTIR (KBr, cm<sup>–1</sup>): 3272 (N–H), 3111 (Ar–CH), 2925 (−CH<sub>3</sub>), 1566 (CN), 1502 (CC), 1207 (C–O),
1164 (CS). <sup>1</sup>H NMR (400 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 3.22 (s, 3H, −CH<sub>3</sub>), 3.90
(s, 3H, OCH<sub>3</sub>), 6.54–6.53 (s, 1H, Ar–H), 7.50–7.35
(m, 2H, Ar–H), 7.70–7.61 (m, 2H, Ar–H), 7.75–7.74
(d, 1H, Ar–H), 8.091 (s, 1H, Ar–H), 8.22 (s, 1H, CH),
8.37–8.35 (d, 1H, Ar–H), 8.70–8.69 (s, 2H, Ar–H),
11.77 (s, 1H, NH), 12.08 (s, 1H, NH). <sup>13</sup>C NMR (100 MHz,
δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 31.13, 56.49,
104.21, 109.87, 112.37, 119.28, 121.27, 123.55, 124.27, 127.36, 127.90,
133.86, 134.17, 135.35, 141.09, 143.16, 150.44, 151.89, 153.38, 156.59,
161.09, 173.68, 175.38. Molecular mass; calculated: C<sub>23</sub>H<sub>19</sub>ClN<sub>6</sub>O<sub>2</sub>S<sub>2</sub>; 511.02;
found LCMS; <em>m</em>/<em>z</em> 511.52 (M), 512
(M + 1), 513 (M + 2).</p></section><section id="sec5.1.2.17"><h5 class="pmc_sec_title">N′-((<em>Z</em>)-4-(Chloromethyl)­benzylidene)-2-((<em>E</em>)-4-((7-chloroquinolin-4-yl)­oxy)-3-methoxybenzylidene)
Hydrazine-1-carbothiohydrazide (<strong>5q</strong>)</h5>
<p>Brown powder.
Yield: 92%; mp: 222–224 °C; FTIR (KBr, cm<sup>–1</sup>): 3536 (N–H), 3086 (Ar–CH), 2989 (−CH<sub>2</sub>), 1553 (CN), 1501 (CC), 1259 (C–O), 1167
(CS). <sup>1</sup>H NMR (400 MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>: 3.82 (s, 3H, OCH<sub>3</sub>), 4.46 (s, 2H,
CH<sub>2</sub>), 6.63–6.62 (d, 1H, Ar–H), 7.82–7.44
(m, 11H, Ar–H), 8.13 (s, 1H, CH), 8.36–8.33 (d, 1H,
CH), 8.75–8.39 (d, 2H, Ar–H). <sup>13</sup>C NMR (100
MHz, δ, ppm, DMSO-<em>d</em>
<sub>6</sub>): 56.60, 104.35,
109.71, 112.73, 119.19, 121.59, 123.63, 124.50, 127.68, 128.14, 130.07,
131.54, 132.81, 133.17, 135.35, 138.26, 141.15, 143.34, 146.38, 150.44,
151.89, 153.38, 156.59, 161.09, 173.68, 175.38. Molecular mass; calculated:
C<sub>26</sub>H<sub>21</sub>Cl<sub>2</sub>N<sub>5</sub>O<sub>2</sub>S; 538.45; found LCMS; <em>m</em>/<em>z</em> 538.56
(M), 539 (M + 1), 540 (M + 2).</p></section></section></section><section id="sec5.2"><h3 class="pmc_sec_title">Computational Studies</h3>
<section id="sec5.2.1"><h4 class="pmc_sec_title">Molecular
Docking</h4>
<p>Schrodinger’s Glide Extra
Precision (XP) mode was employed to accurately predict the binding
interactions and positions within the binding pocket of α-amylase
and α-glucosidase, and aldose reductase. This method enhances
accuracy and helps filter out false positive results by using advanced
scoring algorithms and providing broad flexible sampling. The top-ranked
poses are based on their key interactions, such as hydrogen bonds,
π–π stacking interactions, hydrophobic contacts,
halogen bonds, etc. The synthesized ligand
molecules were sketched using Maestro, and the ligand preparation
was conducted using the LigPrep module of Schrodinger software, along
with the energy minimization performed via the OPLS4 force field.
Torsional flexibility was incorporated to improve the binding complementarity.
For the <em>in-silico</em> studies, the crystal structures
of α-amylase (PDB ID: <a href="https://www.rcsb.org/structure/4W93" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">4W93</a>), α-glucosidase (PDB ID: <a href="https://www.rcsb.org/structure/5NN8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">5NN8</a>), and aldose reductase
(PDB ID: <a href="https://www.rcsb.org/structure/4JIR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">4JIR</a>) were obtained from the Research Collaboratory for Structural Bioinformatics
Protein Data Bank (<a href="https://www.rcsb.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.rcsb.org/</a>). The preparation of protein was processed using Protein Preparation
Wizard, which includes the removal of water molecules and native ligands,
assignment of bond orders, and p<em>K</em>
<sub>a</sub> values
predicted at pH 7 ± 2. Further, the grid generation was guided
by cocrystallized ligands, and the structures were energy-minimized
using OPLS4 for docking readiness.
</p></section><section id="sec5.2.2"><h4 class="pmc_sec_title">Molecular Dynamics Simulation</h4>
<p>Molecular dynamics (MD)
simulations were employed to evaluate the structural stability of
the protein–ligand complex (4JIR-<strong>5o</strong>) with the
binding pocket of aldose reductase under the physicochemical conditions
to provide dynamic insights and a time-resolved picture by computing
atomic movement over time. The MD simulations
were performed using the Desmond module integrated within the Schrodinger
suite for about 100 ns using the OPLS force field. The complex was
preprocessed by Protein Preparation Wizard using Maestro and further
solvated in the orthorhombic box using standard TIP3P water configuration
along with sodium ions to neutralize the system. The simulation was
conducted, and the membrane model was utilized at the standard human
body temperature of 310.15 K and pressure (1.0132 bar) conditions,
with the system volume minimized throughout. Structural stability
during the simulation was assessed using the root-mean-square deviation
(RMSD) of the protein–ligand complex. The MM/GBSA (Molecular mechanics/Generalized Born Surface Area)
calculations are used to evaluate the binding affinity of ligands
toward target macromolecules as well as to determine the ligand’s
binding free energy. The calculations were carried out for the inhibitor-associated
protein–ligand complexes using the Prime module in Schrödinger
Software, utilizing the previously established protocol.
</p></section><section id="sec5.2.3"><h4 class="pmc_sec_title">Density Functional Theory (DFT)</h4>
<p>The standard Gaussian
09W program was used for all quantum chemistry and viewing results.
The density functional theory (DFT), using the B3LYP (Lee–Yang–Parr
nonlocal correlation functionals), using a basis set of 6–31
G (d, p) and the Beck’s three-parameter hybrid functional,
was used to optimize the geometry of the synthesized molecules. With
consistent theoretical parameters, the vibrational frequencies at
the ideal structures of the investigated compounds were also computed.
Furthermore, it is possible to determine the quantum chemical parameters,
including the energy gap, hardness (η), local softness (σ),
dipole moments (μ), electrophilicity (ω), and electronegativity
(χ), can be evaluated using the following mathematical equations.<em>
,
</em>
</p>
<table class="disp-formula p" id="ueq1"><tr><td class="formula"><math id="d1e2311" display="block"><mi mathvariant="normal">Δ</mi><msub><mi>E</mi><mi>GAP</mi></msub><mo>=</mo><msub><mi>E</mi><mi>LUMO</mi></msub><mo>−</mo><msub><mi>E</mi><mi>HOMO</mi></msub></math></td></tr></table>
<table class="disp-formula p" id="ueq2"><tr><td class="formula"><math id="d1e2328" display="block"><mi>η</mi><mo>=</mo><mfrac><mrow><msub><mi>E</mi><mi>LUMO</mi></msub><mo>−</mo><msub><mi>E</mi><mi>HOMO</mi></msub></mrow><mn>2</mn></mfrac></math></td></tr></table>
<table class="disp-formula p" id="ueq3"><tr><td class="formula"><math id="d1e2344" display="block"><mi>σ</mi><mo>=</mo><mfrac><mn>1</mn><mi>η</mi></mfrac></math></td></tr></table>
<table class="disp-formula p" id="ueq4"><tr><td class="formula"><math id="d1e2353" display="block"><mi>μ</mi><mo>=</mo><mfrac><mrow><msub><mi>E</mi><mi>HOMO</mi></msub><mo>+</mo><msub><mi>E</mi><mi>LUMO</mi></msub></mrow><mn>2</mn></mfrac></math></td></tr></table>
<table class="disp-formula p" id="ueq5"><tr><td class="formula"><math id="d1e2369" display="block"><mi>ω</mi><mo>=</mo><mfrac><msup><mi>μ</mi><mn>2</mn></msup><mrow><mn>2</mn><mi>η</mi></mrow></mfrac></math></td></tr></table></section><section id="sec5.2.4"><h4 class="pmc_sec_title">ADMET Studies</h4>
<p>The pharmacokinetic
profile, like absorption,
distribution, metabolism, excretion, and toxicity of the newly synthesized
derivatives, was predicted computationally with the help of the QikProp
module within the Schrodinger software, which helps assess the properties
and evaluation of the potency of compounds.<em>
,
</em>
</p></section></section><section id="sec5.3"><h3 class="pmc_sec_title">Biological Activity</h3>
<section id="sec5.3.1"><h4 class="pmc_sec_title">Human Lysosomal Acid-α-glucosidase
(HLAG) Inhibition Assay</h4>
<p>The α-glucosidase inhibition
assay was performed according
to a previous study by Gharge et al. (2024). The assay was conducted
by combining 700 μL of phosphate buffer with 100 μL of
the test compounds (<strong>5a</strong>–<strong>5q</strong>) at varying
concentrations, along with 100 μL of yeast α-glucosidase
enzyme. One unit of the enzyme was defined as the quantity required
to release 1 μmol of <em>p</em>-nitrophenol/min from
pNPG under the specified experimental conditions. The reaction mixture
was preincubated at 37 °C for 10 min, after which 100 μL
of a 0.5 mM pNPG solution (prepared in 50 mM phosphate buffer) was
added. The enzymatic reaction was then allowed to proceed at 37 °C
for 20 min. Following incubation, the enzyme activity was assessed
by measuring the absorbance of released <em>p</em>-nitrophenol
at 405 nm using a microplate reader. Absorbance values were compared
against a blank (phosphate buffer in place of the test sample) with
acarbose serving as the standard control. The degree of α-glucosidase
inhibition was calculated and expressed as a percentage using the
standard inhibition formula.
</p>
<table class="disp-formula p" id="ueq6"><tr><td class="formula"><math id="d1e2407" display="block"><mi>inhibition</mi><mrow><mo stretchy="false">(</mo><mo>%</mo><mo stretchy="false">)</mo></mrow><mo>=</mo><mrow><mo stretchy="false">(</mo><msub><mi>A</mi><mi>control</mi></msub><mo>−</mo><msub><mi>A</mi><mi>sample</mi></msub><mo stretchy="false">)</mo></mrow><mo>/</mo><msub><mi>A</mi><mi>control</mi></msub><mo>×</mo><mn>100</mn></math></td></tr></table></section><section id="sec5.3.2"><h4 class="pmc_sec_title">Human Pancreatic α-Amylase
(HPA) Inhibition Assay</h4>
<p>The HPA assay was conducted according
to a previously reported method.
A 1% solution of soluble starch used as the substrate was prepared
in 0.02 M sodium phosphate buffer (pH 6.9) containing 6 mM NaCl. The
enzyme solution was also prepared in the same buffer. To initiate
the reaction, 500 μM of the enzyme filtrate (EF) was added and
dissolved in DMSO. The mixture was then incubated at 25 °C for
10 min. Following this, 1% soluble starch was introduced into the
reaction, which was incubated for an additional 15 min. The enzymatic
reaction was halted by the addition of dinitrosalicylic acid (DNS)
reagent, and the mixture was then subjected to boiling at 85 °C
for 5 min. After cooling, the reaction mixture was diluted with 10
mL of distilled water, and absorbance was measured at 540 nm using
a UV-spectrometer. Acarbose served as the reference standard for comparison.
</p>
<table class="disp-formula p" id="ueq7"><tr><td class="formula"><math id="d1e2442" display="block"><mi>inhibition</mi><mrow><mo stretchy="false">(</mo><mo>%</mo><mo stretchy="false">)</mo></mrow><mo>=</mo><mrow><mo stretchy="false">(</mo><msub><mi>A</mi><mi>control</mi></msub><mo>−</mo><msub><mi>A</mi><mi>sample</mi></msub><mo stretchy="false">)</mo></mrow><mo>/</mo><msub><mi>A</mi><mi>control</mi></msub><mo>×</mo><mn>100</mn></math></td></tr></table></section><section id="sec5.3.3"><h4 class="pmc_sec_title">Aldose Reductase Inhibition
Assay</h4>
<p>The aldose reductase
inhibition assay was conducted following the procedure outlined by
Ramu et al. (2014). In brief, the reaction mixture (1 mL total volume)
consisted of 0.87 mL of 100 mM sodium phosphate buffer (pH 6.2), 0.05
mL of 3 mM NADPH, 0.01 mL of recombinant human aldose reductase (0.2
IU/mL), and 0.02 mL of synthesized compounds (<strong>5a</strong>–<strong>5q</strong>) at varying concentrations. To initiate the reaction, 0.05
mL of 5 mM <span class="font-variant-small-caps">dl</span>-glyceraldehyde (substrate) was added, and the
mixture was incubated at 25 °C for 3 min. The enzyme activity
was monitored by measuring the reduction in NADPH absorbance at 340
nm over a 10 min period. Inhibition potential was assessed by comparing
the absorbance change in the test samples to that of the control.
All experiments were conducted in triplicate, and the aldose reductase
inhibitory activity was expressed as percent inhibition. The IC<sub>50</sub> values were determined by analyzing the relationship between
the percentage inhibition and the corresponding compound concentration.
</p>
<table class="disp-formula p" id="ueq8"><tr><td class="formula"><math id="d1e2485" display="block"><mi>inhibition</mi><mrow><mo stretchy="false">(</mo><mo>%</mo><mo stretchy="false">)</mo></mrow><mo>=</mo><mrow><mo stretchy="false">(</mo><msub><mi>A</mi><mi>control</mi></msub><mo>−</mo><msub><mi>A</mi><mi>sample</mi></msub><mo stretchy="false">)</mo></mrow><mo>/</mo><msub><mi>A</mi><mi>control</mi></msub><mo>×</mo><mn>100</mn></math></td></tr></table></section><section id="sec5.3.4"><h4 class="pmc_sec_title">Kinetics of Aldose Reductase
Inhibition</h4>
<p>Using a range
of substrate concentrations in comparison to IC<sub>20</sub>, IC<sub>40</sub>, and IC<sub>60</sub> inhibitory doses, the enzyme kinetics
on the inhibition of aldose reductase activity of the produced compounds
were investigated. The double reciprocal plot (Lineweaver and Burk,
1934) of the substrate concentration
and the velocity (1/V vs 1/[pNPG]) was used to determine the type
of inhibition, <em>K</em>
<sub>m</sub>, and <em>V</em>
<sub>max</sub>. Additionally, the Dixon plot was used to calculate
the inhibitory constant (<em>K</em>
<sub>i</sub>) (Dixon, 1953).<em>
,
</em>
</p></section><section id="sec5.3.5"><h4 class="pmc_sec_title">Statistical Analysis</h4>
<p>The results were presented as
the mean ± SE, and the experiments were conducted in triplicate.
The ″<em>p</em>″ value was determined using
Duncan’s multiple tests. A ″<em>p</em>″
value of less than 0.05 was deemed significant.
</p></section></section></section><section id="sec51"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="sifile1"><div class="media p"><div class="caption">
<a href="/articles/instance/12332697/bin/ao5c04663_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">ao5c04663_si_001.pdf</a><sup> (4.9MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>This work
was supported Department of Pharmaceutical
chemistry, KLE College of Pharmacy, KLE Academy of Higher Education
and Research, Belagavi.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Glossary</h2>
<h3>Abbreviations</h3>
<dl class="def-list" id="dl1">
<dt>ADMET</dt>
<dd><p>absorption,
distribution, metabolism,
excretion and toxicology</p></dd>
<dt>TLC</dt>
<dd><p>thin-layer chromatography</p></dd>
<dt>OPLS</dt>
<dd><p>optimized potential for liquid simulations</p></dd>
<dt>XP</dt>
<dd><p>extra precision</p></dd>
</dl></section><section id="notes1"><p>No data generated
or analyzed during this study</p></section><section id="notes3"><p>The Supporting Information is
available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsomega.5c04663?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acsomega.5c04663</a>.</p>
<ul id="silist" class="list" style="list-style-type:none"><li><p>Raw data from the <em>in silico</em> studies
and FTIR spectra of the synthesized compounds and spectral characterization
data of the synthesized compounds, including <sup>1</sup>H NMR, <sup>13</sup>C NMR, and LCMS data (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c04663/suppl_file/ao5c04663_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p></li></ul></section><section id="notes2"><p>N.J.P. performed
synthesis, characterization, biological activities, computational
studies, and original manuscript writing. S.G. performed original
manuscript reviewing. R.S.K. was in charge of conceptualization, methodology,
and supervision. S.D.R. performed computational studies. S.G.A. gathered
resources. L.S.W. gathered resources. R.R. was in charge of the biological
activity. R.C.M. gathered resources.</p></section><section id="notes4"><p>The authors
declare no competing financial interest.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>Low C. Y., Gan W. L., Lai S. J., Tam R. S. M., Tan J. F., Dietl S., Chuah L. H., Voelcker N., Bakhtiar A.. Critical updates
on oral insulin drug delivery systems for type 2 diabetes mellitus. J. Nanobiotechnol. 2025;23:16. doi: 10.1186/s12951-024-03062-7.</cite> [<a href="https://doi.org/10.1186/s12951-024-03062-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11737240/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39815320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Nanobiotechnol.&amp;title=Critical%20updates%20on%20oral%20insulin%20drug%20delivery%20systems%20for%20type%202%20diabetes%20mellitus&amp;author=C.%20Y.%20Low&amp;author=W.%20L.%20Gan&amp;author=S.%20J.%20Lai&amp;author=R.%20S.%20M.%20Tam&amp;author=J.%20F.%20Tan&amp;volume=23&amp;publication_year=2025&amp;pages=16&amp;pmid=39815320&amp;doi=10.1186/s12951-024-03062-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>Huneif M. A., Alshehri D. B., Alshaibari K. S., Dammaj M. Z., Mahnashi M. H., Majid S. U., Javed M. A., Ahmad S., Rashid U., Sadiq A.. Design, synthesis and
bioevaluation of new vanillin hybrid as multitarget
inhibitor of α-glucosidase, α-amylase, PTP-1B and DPP4
for the treatment of type-II diabetes. Biomed.
Pharmacother. 2022;150:113038.  doi: 10.1016/j.biopha.2022.113038.</cite> [<a href="https://doi.org/10.1016/j.biopha.2022.113038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35658208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomed.%20Pharmacother.&amp;title=Design,%20synthesis%20and%20bioevaluation%20of%20new%20vanillin%20hybrid%20as%20multitarget%20inhibitor%20of%20%CE%B1-glucosidase,%20%CE%B1-amylase,%20PTP-1B%20and%20DPP4%20for%20the%20treatment%20of%20type-II%20diabetes&amp;author=M.%20A.%20Huneif&amp;author=D.%20B.%20Alshehri&amp;author=K.%20S.%20Alshaibari&amp;author=M.%20Z.%20Dammaj&amp;author=M.%20H.%20Mahnashi&amp;volume=150&amp;publication_year=2022&amp;pages=113038&amp;pmid=35658208&amp;doi=10.1016/j.biopha.2022.113038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Kazempour-Dizaji M., Mojtabavi S., Sadri A., Ghanbarpour A., Faramarzi M. A., Navidpour L.. Arylureidoaurones: Synthesis, in
vitro α-glucosidase, and α-amylase inhibition activity. Bioorg. Chem. 2023;139:106709.  doi: 10.1016/j.bioorg.2023.106709.</cite> [<a href="https://doi.org/10.1016/j.bioorg.2023.106709" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37442042/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Chem.&amp;title=Arylureidoaurones:%20Synthesis,%20in%20vitro%20%CE%B1-glucosidase,%20and%20%CE%B1-amylase%20inhibition%20activity&amp;author=M.%20Kazempour-Dizaji&amp;author=S.%20Mojtabavi&amp;author=A.%20Sadri&amp;author=A.%20Ghanbarpour&amp;author=M.%20A.%20Faramarzi&amp;volume=139&amp;publication_year=2023&amp;pages=106709&amp;pmid=37442042&amp;doi=10.1016/j.bioorg.2023.106709&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>Forouhi N. G., Wareham N. J.. Epidemiology of diabetes. Med. 2010;38:602–606. doi: 10.1016/j.mpmed.2010.08.007.</cite> [<a href="https://doi.org/10.1016/j.mpmed.2010.08.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Epidemiology%20of%20diabetes.%20Med.&amp;author=N.%20G.%20Forouhi&amp;author=N.%20J.%20Wareham&amp;volume=38&amp;publication_year=2010&amp;pages=602-606&amp;doi=10.1016/j.mpmed.2010.08.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Hossain M. J., Al-Mamun M., Islam M. R.. Diabetes
mellitus, the fastest growing
global public health concern: Early detection should be focused. Health Sci. Rep. 2024;7:e2004.  doi: 10.1002/hsr2.2004.</cite> [<a href="https://doi.org/10.1002/hsr2.2004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10958528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38524769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Health%20Sci.%20Rep.&amp;title=Diabetes%20mellitus,%20the%20fastest%20growing%20global%20public%20health%20concern:%20Early%20detection%20should%20be%20focused&amp;author=M.%20J.%20Hossain&amp;author=M.%20Al-Mamun&amp;author=M.%20R.%20Islam&amp;volume=7&amp;publication_year=2024&amp;pages=e2004&amp;pmid=38524769&amp;doi=10.1002/hsr2.2004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Gupta S., Baweja G. S., Singh S., Irani M., Singh R., Asati V.. Integrated fragment-based
drug design and virtual screening techniques
for exploring the antidiabetic potential of thiazolidine-2, 4-diones:
design, synthesis and in vivo studies. Eur.
J. Med. Chem. 2023;261:115826.  doi: 10.1016/j.ejmech.2023.115826.</cite> [<a href="https://doi.org/10.1016/j.ejmech.2023.115826" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37793328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Med.%20Chem.&amp;title=Integrated%20fragment-based%20drug%20design%20and%20virtual%20screening%20techniques%20for%20exploring%20the%20antidiabetic%20potential%20of%20thiazolidine-2,%204-diones:%20design,%20synthesis%20and%20in%20vivo%20studies&amp;author=S.%20Gupta&amp;author=G.%20S.%20Baweja&amp;author=S.%20Singh&amp;author=M.%20Irani&amp;author=R.%20Singh&amp;volume=261&amp;publication_year=2023&amp;pages=115826&amp;pmid=37793328&amp;doi=10.1016/j.ejmech.2023.115826&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Maniruzzaman M., Kumar N., Abedin M. M., Islam M. S., Suri H. S., El-Baz A. S., Suri J. S.. Comparative approaches
for classification
of diabetes mellitus data: Machine learning paradigm. Comput. Methods Programs Biomed. 2017;152:23–34. doi: 10.1016/j.cmpb.2017.09.004.</cite> [<a href="https://doi.org/10.1016/j.cmpb.2017.09.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29054258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput.%20Methods%20Programs%20Biomed.&amp;title=Comparative%20approaches%20for%20classification%20of%20diabetes%20mellitus%20data:%20Machine%20learning%20paradigm&amp;author=M.%20Maniruzzaman&amp;author=N.%20Kumar&amp;author=M.%20M.%20Abedin&amp;author=M.%20S.%20Islam&amp;author=H.%20S.%20Suri&amp;volume=152&amp;publication_year=2017&amp;pages=23-34&amp;pmid=29054258&amp;doi=10.1016/j.cmpb.2017.09.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Yousuf H., Shamim S., Khan K. M., Chigurupati S., Hameed S., Khan M. N., Taha M., Arfeen M.. Dihydropyridines
as potential α-amylase and α-glucosidase inhibitors: synthesis,
in vitro and in silico studies. Bioorg. Chem. 2020;96:103581.  doi: 10.1016/j.bioorg.2020.103581.</cite> [<a href="https://doi.org/10.1016/j.bioorg.2020.103581" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31978686/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Chem.&amp;title=Dihydropyridines%20as%20potential%20%CE%B1-amylase%20and%20%CE%B1-glucosidase%20inhibitors:%20synthesis,%20in%20vitro%20and%20in%20silico%20studies&amp;author=H.%20Yousuf&amp;author=S.%20Shamim&amp;author=K.%20M.%20Khan&amp;author=S.%20Chigurupati&amp;author=S.%20Hameed&amp;volume=96&amp;publication_year=2020&amp;pages=103581&amp;pmid=31978686&amp;doi=10.1016/j.bioorg.2020.103581&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Güleç Ö., Türkeş C., Arslan M., Demir Y., Dincer B., Ece A., Küfrevioğlu Ö. İ., Beydemir Ş.. Novel spiroindoline
derivatives targeting
aldose reductase against diabetic complications: Bioactivity, cytotoxicity,
and molecular modeling studies. Bioorg. Chem. 2024;145:107221.  doi: 10.1016/j.bioorg.2024.107221.</cite> [<a href="https://doi.org/10.1016/j.bioorg.2024.107221" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38387398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Chem.&amp;title=Novel%20spiroindoline%20derivatives%20targeting%20aldose%20reductase%20against%20diabetic%20complications:%20Bioactivity,%20cytotoxicity,%20and%20molecular%20modeling%20studies&amp;author=%C3%96.%20G%C3%BCle%C3%A7&amp;author=C.%20T%C3%BCrke%C5%9F&amp;author=M.%20Arslan&amp;author=Y.%20Demir&amp;author=B.%20Dincer&amp;volume=145&amp;publication_year=2024&amp;pages=107221&amp;pmid=38387398&amp;doi=10.1016/j.bioorg.2024.107221&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Ranade S. D., Alegaon S. G., Khatib N. A., Gharge S., Kavalapure R. S.. Quinoline-based
Schiff bases as possible antidiabetic agents: ligand-based pharmacophore
modeling, 3D QSAR, docking, and molecular dynamics simulations study. RSC Med. Chem. 2024;15:3162–3179. doi: 10.1039/D4MD00344F.</cite> [<a href="https://doi.org/10.1039/D4MD00344F" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11322893/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39149562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Med.%20Chem.&amp;title=Quinoline-based%20Schiff%20bases%20as%20possible%20antidiabetic%20agents:%20ligand-based%20pharmacophore%20modeling,%203D%20QSAR,%20docking,%20and%20molecular%20dynamics%20simulations%20study&amp;author=S.%20D.%20Ranade&amp;author=S.%20G.%20Alegaon&amp;author=N.%20A.%20Khatib&amp;author=S.%20Gharge&amp;author=R.%20S.%20Kavalapure&amp;volume=15&amp;publication_year=2024&amp;pages=3162-3179&amp;pmid=39149562&amp;doi=10.1039/D4MD00344F&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Mandewale M. C., Patil U. C., Shedge S. V., Dappadwad U. R., Yamgar R. S.. A review on quinoline hydrazone derivatives as a new
class of potent antitubercular and anticancer agents. BJBAS. 2017;6:354–361. doi: 10.1016/j.bjbas.2017.07.005.</cite> [<a href="https://doi.org/10.1016/j.bjbas.2017.07.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BJBAS.&amp;title=A%20review%20on%20quinoline%20hydrazone%20derivatives%20as%20a%20new%20class%20of%20potent%20antitubercular%20and%20anticancer%20agents&amp;author=M.%20C.%20Mandewale&amp;author=U.%20C.%20Patil&amp;author=S.%20V.%20Shedge&amp;author=U.%20R.%20Dappadwad&amp;author=R.%20S.%20Yamgar&amp;volume=6&amp;publication_year=2017&amp;pages=354-361&amp;doi=10.1016/j.bjbas.2017.07.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Ajani O. O., Iyaye K. T., Ademosun O. T.. Recent advances in chemistry and
therapeutic potential of functionalized quinoline motifs–a
review. RSC Adv. 2022;12:18594–18614. doi: 10.1039/D2RA02896D.</cite> [<a href="https://doi.org/10.1039/D2RA02896D" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9231466/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35873320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Adv.&amp;title=Recent%20advances%20in%20chemistry%20and%20therapeutic%20potential%20of%20functionalized%20quinoline%20motifs%E2%80%93a%20review&amp;author=O.%20O.%20Ajani&amp;author=K.%20T.%20Iyaye&amp;author=O.%20T.%20Ademosun&amp;volume=12&amp;publication_year=2022&amp;pages=18594-18614&amp;pmid=35873320&amp;doi=10.1039/D2RA02896D&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Elebiju O. F., Ajani O. O., Oduselu G. O., Ogunnupebi T. A., Adebiyi E.. Recent advances in functionalized
quinoline scaffolds
and hybridsExceptional pharmacophore in therapeutic medicine. Front. Chem. 2023;10:1074331.  doi: 10.3389/fchem.2022.1074331.</cite> [<a href="https://doi.org/10.3389/fchem.2022.1074331" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9859673/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36688036/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Chem.&amp;title=Recent%20advances%20in%20functionalized%20quinoline%20scaffolds%20and%20hybrids%EE%97%B8Exceptional%20pharmacophore%20in%20therapeutic%20medicine&amp;author=O.%20F.%20Elebiju&amp;author=O.%20O.%20Ajani&amp;author=G.%20O.%20Oduselu&amp;author=T.%20A.%20Ogunnupebi&amp;author=E.%20Adebiyi&amp;volume=10&amp;publication_year=2023&amp;pages=1074331&amp;pmid=36688036&amp;doi=10.3389/fchem.2022.1074331&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Jain S., Chandra V., Jain P. K., Pathak K., Pathak D., Vaidya A.. Comprehensive review
on current developments of quinoline-based
anticancer agents. Arab. J. Chem. 2019;12:4920–4946. doi: 10.1016/j.arabjc.2016.10.009.</cite> [<a href="https://doi.org/10.1016/j.arabjc.2016.10.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Arab.%20J.%20Chem.&amp;title=Comprehensive%20review%20on%20current%20developments%20of%20quinoline-based%20anticancer%20agents&amp;author=S.%20Jain&amp;author=V.%20Chandra&amp;author=P.%20K.%20Jain&amp;author=K.%20Pathak&amp;author=D.%20Pathak&amp;volume=12&amp;publication_year=2019&amp;pages=4920-4946&amp;doi=10.1016/j.arabjc.2016.10.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>El
Shehry M. F., Ghorab M. M., Abbas S. Y., Fayed E. A., Shedid S. A., Ammar Y. A.. Quinoline derivatives bearing pyrazole
moiety: Synthesis and biological evaluation as possible antibacterial
and antifungal agents. Eur. J. Med. Chem. 2018;143:1463–1473. doi: 10.1016/j.ejmech.2017.10.046.</cite> [<a href="https://doi.org/10.1016/j.ejmech.2017.10.046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29113746/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Med.%20Chem.&amp;title=Quinoline%20derivatives%20bearing%20pyrazole%20moiety:%20Synthesis%20and%20biological%20evaluation%20as%20possible%20antibacterial%20and%20antifungal%20agents&amp;author=M.%20F.%20El%20Shehry&amp;author=M.%20M.%20Ghorab&amp;author=S.%20Y.%20Abbas&amp;author=E.%20A.%20Fayed&amp;author=S.%20A.%20Shedid&amp;volume=143&amp;publication_year=2018&amp;pages=1463-1473&amp;pmid=29113746&amp;doi=10.1016/j.ejmech.2017.10.046&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Savegnago L., Vieira A. I., Seus N., Goldani B. S., Castro M. R., Lenardão E. J., Alves D.. Synthesis and antioxidant properties
of novel quinoline–chalcogenium compounds. Tetrahedron Lett. 2013;54:40–44. doi: 10.1016/j.tetlet.2012.10.067.</cite> [<a href="https://doi.org/10.1016/j.tetlet.2012.10.067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Tetrahedron%20Lett.&amp;title=Synthesis%20and%20antioxidant%20properties%20of%20novel%20quinoline%E2%80%93chalcogenium%20compounds&amp;author=L.%20Savegnago&amp;author=A.%20I.%20Vieira&amp;author=N.%20Seus&amp;author=B.%20S.%20Goldani&amp;author=M.%20R.%20Castro&amp;volume=54&amp;publication_year=2013&amp;pages=40-44&amp;doi=10.1016/j.tetlet.2012.10.067&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Mukherjee S., Pal M.. Quinolines: a new hope against inflammation. Drug Discovery Today. 2013;18:389–398. doi: 10.1016/j.drudis.2012.11.003.</cite> [<a href="https://doi.org/10.1016/j.drudis.2012.11.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23159484/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Discovery%20Today.&amp;title=Quinolines:%20a%20new%20hope%20against%20inflammation&amp;author=S.%20Mukherjee&amp;author=M.%20Pal&amp;volume=18&amp;publication_year=2013&amp;pages=389-398&amp;pmid=23159484&amp;doi=10.1016/j.drudis.2012.11.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>Crespo I., Gimenez-Dejoz J., Porte S., Cousido-Siah A., Mitschler A., Podjarny A., Pratsinis H., Kletsas D., Pares X., Ruiz F. X., Metwally K.. Design, synthesis,
structure-activity relationships and X-ray structural studies of novel
1-oxopyrimido [4, 5-c] quinoline-2-acetic acid derivatives as selective
and potent inhibitors of human aldose reductase. Eur. J. Med. Chem. 2018;152:160–174. doi: 10.1016/j.ejmech.2018.04.015.</cite> [<a href="https://doi.org/10.1016/j.ejmech.2018.04.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29705708/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Med.%20Chem.&amp;title=Design,%20synthesis,%20structure-activity%20relationships%20and%20X-ray%20structural%20studies%20of%20novel%201-oxopyrimido%20%5B4,%205-c%5D%20quinoline-2-acetic%20acid%20derivatives%20as%20selective%20and%20potent%20inhibitors%20of%20human%20aldose%20reductase&amp;author=I.%20Crespo&amp;author=J.%20Gimenez-Dejoz&amp;author=S.%20Porte&amp;author=A.%20Cousido-Siah&amp;author=A.%20Mitschler&amp;volume=152&amp;publication_year=2018&amp;pages=160-174&amp;pmid=29705708&amp;doi=10.1016/j.ejmech.2018.04.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Taha M., Javid M. T., Imran S., Selvaraj M., Chigurupati S., Ullah H., Rahim F., Khan F., Mohammad J. I., Khan K. M.. Synthesis and study
of the α-amylase inhibitory
potential of thiadiazole quinoline derivatives. Bioorg. Chem. 2017;74:179–186. doi: 10.1016/j.bioorg.2017.08.003.</cite> [<a href="https://doi.org/10.1016/j.bioorg.2017.08.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28826047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Chem.&amp;title=Synthesis%20and%20study%20of%20the%20%CE%B1-amylase%20inhibitory%20potential%20of%20thiadiazole%20quinoline%20derivatives&amp;author=M.%20Taha&amp;author=M.%20T.%20Javid&amp;author=S.%20Imran&amp;author=M.%20Selvaraj&amp;author=S.%20Chigurupati&amp;volume=74&amp;publication_year=2017&amp;pages=179-186&amp;pmid=28826047&amp;doi=10.1016/j.bioorg.2017.08.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>Taha M., Sultan S., Imran S., Rahim F., Zaman K., Wadood A., Rehman A. U., Uddin N., Khan K. M.. Synthesis
of quinoline derivatives as diabetic II inhibitors and molecular docking
studies. Bioorg. Med. Chem. 2019;27:4081–4088. doi: 10.1016/j.bmc.2019.07.035.</cite> [<a href="https://doi.org/10.1016/j.bmc.2019.07.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31378594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Med.%20Chem.&amp;title=Synthesis%20of%20quinoline%20derivatives%20as%20diabetic%20II%20inhibitors%20and%20molecular%20docking%20studies&amp;author=M.%20Taha&amp;author=S.%20Sultan&amp;author=S.%20Imran&amp;author=F.%20Rahim&amp;author=K.%20Zaman&amp;volume=27&amp;publication_year=2019&amp;pages=4081-4088&amp;pmid=31378594&amp;doi=10.1016/j.bmc.2019.07.035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>Taha M., Ismail N. H., Imran S., Wadood A., Rahim F., Ali M., Rehman A. U.. Novel quinoline
derivatives as potent in vitro α-glucosidase
inhibitors: in silico studies and SAR predictions. MedChemComm. 2015;6:1826–1836. doi: 10.1039/C5MD00280J.</cite> [<a href="https://doi.org/10.1039/C5MD00280J" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=MedChemComm.&amp;title=Novel%20quinoline%20derivatives%20as%20potent%20in%20vitro%20%CE%B1-glucosidase%20inhibitors:%20in%20silico%20studies%20and%20SAR%20predictions&amp;author=M.%20Taha&amp;author=N.%20H.%20Ismail&amp;author=S.%20Imran&amp;author=A.%20Wadood&amp;author=F.%20Rahim&amp;volume=6&amp;publication_year=2015&amp;pages=1826-1836&amp;doi=10.1039/C5MD00280J&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Alkahtani H. M., Almehizia A. A., Al-Omar M. A., Obaidullah A. J., Zen A. A., Hassan A. S., Aboulthana W. M.. In vitro
evaluation and bioinformatics analysis of schiff bases bearing pyrazole
scaffold as bioactive agents: antioxidant, anti-diabetic, anti-alzheimer,
and anti-arthritic. Molecules. 2023;28:7125. doi: 10.3390/molecules28207125.</cite> [<a href="https://doi.org/10.3390/molecules28207125" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10609138/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37894604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules.&amp;title=In%20vitro%20evaluation%20and%20bioinformatics%20analysis%20of%20schiff%20bases%20bearing%20pyrazole%20scaffold%20as%20bioactive%20agents:%20antioxidant,%20anti-diabetic,%20anti-alzheimer,%20and%20anti-arthritic&amp;author=H.%20M.%20Alkahtani&amp;author=A.%20A.%20Almehizia&amp;author=M.%20A.%20Al-Omar&amp;author=A.%20J.%20Obaidullah&amp;author=A.%20A.%20Zen&amp;volume=28&amp;publication_year=2023&amp;pages=7125&amp;pmid=37894604&amp;doi=10.3390/molecules28207125&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>El-Feky A. M., Aboulthana W. M., El-Rashedy A. A.. Assessment of the in vitro anti-diabetic
activity with molecular dynamic simulations of limonoids isolated
from Adalia lemon peels. Sci. Rep. 2024;14:21478. doi: 10.1038/s41598-024-71198-5.</cite> [<a href="https://doi.org/10.1038/s41598-024-71198-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11401861/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39277638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Assessment%20of%20the%20in%20vitro%20anti-diabetic%20activity%20with%20molecular%20dynamic%20simulations%20of%20limonoids%20isolated%20from%20Adalia%20lemon%20peels&amp;author=A.%20M.%20El-Feky&amp;author=W.%20M.%20Aboulthana&amp;author=A.%20A.%20El-Rashedy&amp;volume=14&amp;publication_year=2024&amp;pages=21478&amp;pmid=39277638&amp;doi=10.1038/s41598-024-71198-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Shehab W. S., Elhoseni N. K. R., Aboulthana W. M., Assy M. G., Mousa S. M., El-Bassyouni G. T.. Synthesis,
molecular docking, assessment of biological
and anti-diabetic properties of benzalacetophenone derivatives. Sci. Rep. 2025;15:14159. doi: 10.1038/s41598-025-96610-6.</cite> [<a href="https://doi.org/10.1038/s41598-025-96610-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12019235/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40269055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Synthesis,%20molecular%20docking,%20assessment%20of%20biological%20and%20anti-diabetic%20properties%20of%20benzalacetophenone%20derivatives&amp;author=W.%20S.%20Shehab&amp;author=N.%20K.%20R.%20Elhoseni&amp;author=W.%20M.%20Aboulthana&amp;author=M.%20G.%20Assy&amp;author=S.%20M.%20Mousa&amp;volume=15&amp;publication_year=2025&amp;pages=14159&amp;pmid=40269055&amp;doi=10.1038/s41598-025-96610-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>El-Sayed A. F., Aboulthana W. M., Sherief M. A., El-Bassyouni G. T., Mousa S. M.. Synthesis, structural,
molecular docking, and in vitro
biological activities of Cu-doped ZnO nanomaterials. Sci. Rep. 2024;14:9027. doi: 10.1038/s41598-024-59088-2.</cite> [<a href="https://doi.org/10.1038/s41598-024-59088-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11031592/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38641640/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Synthesis,%20structural,%20molecular%20docking,%20and%20in%20vitro%20biological%20activities%20of%20Cu-doped%20ZnO%20nanomaterials&amp;author=A.%20F.%20El-Sayed&amp;author=W.%20M.%20Aboulthana&amp;author=M.%20A.%20Sherief&amp;author=G.%20T.%20El-Bassyouni&amp;author=S.%20M.%20Mousa&amp;volume=14&amp;publication_year=2024&amp;pages=9027&amp;pmid=38641640&amp;doi=10.1038/s41598-024-59088-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Hassan A. S., Morsy N. M., Aboulthana W. M., Ragab A.. In vitro enzymatic
evaluation of some pyrazolo [1, 5-a] pyrimidine derivatives: Design,
synthesis, antioxidant, anti-diabetic, anti-Alzheimer, and anti-arthritic
activities with molecular modeling simulation. Drug Dev. Res. 2023;84:3–24. doi: 10.1002/ddr.22008.</cite> [<a href="https://doi.org/10.1002/ddr.22008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36380556/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Dev.%20Res.&amp;title=In%20vitro%20enzymatic%20evaluation%20of%20some%20pyrazolo%20%5B1,%205-a%5D%20pyrimidine%20derivatives:%20Design,%20synthesis,%20antioxidant,%20anti-diabetic,%20anti-Alzheimer,%20and%20anti-arthritic%20activities%20with%20molecular%20modeling%20simulation&amp;author=A.%20S.%20Hassan&amp;author=N.%20M.%20Morsy&amp;author=W.%20M.%20Aboulthana&amp;author=A.%20Ragab&amp;volume=84&amp;publication_year=2023&amp;pages=3-24&amp;pmid=36380556&amp;doi=10.1002/ddr.22008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Almehizia A. A., Aboulthana W. M., Naglah A. M., Hassan A. S.. In vitro biological
studies and computational prediction-based analyses of pyrazolo [1,
5-a] pyrimidine derivatives. RSC advances. 2024;14:8397–408. doi: 10.1039/D4RA00423J.</cite> [<a href="https://doi.org/10.1039/D4RA00423J" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10928850/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38476172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20advances.&amp;title=In%20vitro%20biological%20studies%20and%20computational%20prediction-based%20analyses%20of%20pyrazolo%20%5B1,%205-a%5D%20pyrimidine%20derivatives&amp;author=A.%20A.%20Almehizia&amp;author=W.%20M.%20Aboulthana&amp;author=A.%20M.%20Naglah&amp;author=A.%20S.%20Hassan&amp;volume=14&amp;publication_year=2024&amp;pages=8397-408&amp;pmid=38476172&amp;doi=10.1039/D4RA00423J&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Abdelazeem N. M., Aboulthana W. M., Elshahid Z. A., El-Hussieny M., Al-Ashmawy A. A.. Synthesis
and biological (in vitro and in silico) screening
of the 4-aryl-fused pyranopyrazole derivatives as enzyme (α-amylase,
α-glucosidase, acetylcholinesterase &amp; proteinase) inhibitors
with anti-oxidant and cytotoxic activities. J. Mol. Struct. 2024;1310:138224.  doi: 10.1016/j.molstruc.2024.138224.</cite> [<a href="https://doi.org/10.1016/j.molstruc.2024.138224" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Struct.&amp;title=Synthesis%20and%20biological%20(in%20vitro%20and%20in%20silico)%20screening%20of%20the%204-aryl-fused%20pyranopyrazole%20derivatives%20as%20enzyme%20(%CE%B1-amylase,%20%CE%B1-glucosidase,%20acetylcholinesterase%20&amp;%20proteinase)%20inhibitors%20with%20anti-oxidant%20and%20cytotoxic%20activities&amp;author=N.%20M.%20Abdelazeem&amp;author=W.%20M.%20Aboulthana&amp;author=Z.%20A.%20Elshahid&amp;author=M.%20El-Hussieny&amp;author=A.%20A.%20Al-Ashmawy&amp;volume=1310&amp;publication_year=2024&amp;pages=138224&amp;doi=10.1016/j.molstruc.2024.138224&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>Naglah A. M., Almehizia A. A., Al-Wasidi A. S., Alharbi A. S., Alqarni M. H., Hassan A. S., Aboulthana W. M.. Exploring the Potential Biological
Activities of Pyrazole-Based Schiff Bases as Anti-Diabetic, Anti-Alzheimer’s,
Anti-Inflammatory, and Cytotoxic Agents: In Vitro Studies with Computational
Predictions. Pharmaceuticals. 2024;17:655. doi: 10.3390/ph17050655.</cite> [<a href="https://doi.org/10.3390/ph17050655" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11125359/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38794225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmaceuticals.&amp;title=Exploring%20the%20Potential%20Biological%20Activities%20of%20Pyrazole-Based%20Schiff%20Bases%20as%20Anti-Diabetic,%20Anti-Alzheimer%E2%80%99s,%20Anti-Inflammatory,%20and%20Cytotoxic%20Agents:%20In%20Vitro%20Studies%20with%20Computational%20Predictions&amp;author=A.%20M.%20Naglah&amp;author=A.%20A.%20Almehizia&amp;author=A.%20S.%20Al-Wasidi&amp;author=A.%20S.%20Alharbi&amp;author=M.%20H.%20Alqarni&amp;volume=17&amp;publication_year=2024&amp;pages=655&amp;pmid=38794225&amp;doi=10.3390/ph17050655&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Ragab A., Salem M. A., Ammar Y. A., Aboulthana W. M., Helal M. H., Abusaif M. S.. Explore new quinoxaline
pharmacophore
tethered sulfonamide fragments as in vitro α-glucosidase, α-amylase,
and acetylcholinesterase inhibitors with ADMET and molecular modeling
simulation. Drug Dev. Res. 2024;85:e22216.  doi: 10.1002/ddr.22216.</cite> [<a href="https://doi.org/10.1002/ddr.22216" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38831547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Dev.%20Res.&amp;title=Explore%20new%20quinoxaline%20pharmacophore%20tethered%20sulfonamide%20fragments%20as%20in%20vitro%20%CE%B1-glucosidase,%20%CE%B1-amylase,%20and%20acetylcholinesterase%20inhibitors%20with%20ADMET%20and%20molecular%20modeling%20simulation&amp;author=A.%20Ragab&amp;author=M.%20A.%20Salem&amp;author=Y.%20A.%20Ammar&amp;author=W.%20M.%20Aboulthana&amp;author=M.%20H.%20Helal&amp;volume=85&amp;publication_year=2024&amp;pages=e22216&amp;pmid=38831547&amp;doi=10.1002/ddr.22216&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>Salve P. S., Alegaon S. G.. Synthesis of new 7-chloro-4-phenoxyquinoline analogues
as potential antitubercular agents. Med. Chem.
Res. 2018;27:1–4. doi: 10.1007/s00044-017-1970-2.</cite> [<a href="https://doi.org/10.1007/s00044-017-1970-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Med.%20Chem.%20Res.&amp;title=Synthesis%20of%20new%207-chloro-4-phenoxyquinoline%20analogues%20as%20potential%20antitubercular%20agents&amp;author=P.%20S.%20Salve&amp;author=S.%20G.%20Alegaon&amp;volume=27&amp;publication_year=2018&amp;pages=1-4&amp;doi=10.1007/s00044-017-1970-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>Patil S., Alegaon S. G., Gharge S., Ranade S. D., Khatib N. A.. Molecular
hybridization, synthesis, in vitro α-glucosidase inhibition,
in vivo antidiabetic activity and computational studies of isatin
based compounds. Bioorg. Chem. 2024;153:107783.  doi: 10.1016/j.bioorg.2024.107783.</cite> [<a href="https://doi.org/10.1016/j.bioorg.2024.107783" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39255610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Chem.&amp;title=Molecular%20hybridization,%20synthesis,%20in%20vitro%20%CE%B1-glucosidase%20inhibition,%20in%20vivo%20antidiabetic%20activity%20and%20computational%20studies%20of%20isatin%20based%20compounds&amp;author=S.%20Patil&amp;author=S.%20G.%20Alegaon&amp;author=S.%20Gharge&amp;author=S.%20D.%20Ranade&amp;author=N.%20A.%20Khatib&amp;volume=153&amp;publication_year=2024&amp;pages=107783&amp;pmid=39255610&amp;doi=10.1016/j.bioorg.2024.107783&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>Kavalapure R. S., Alegaon S. G., Venkatasubramanian U., Priya A. S., Ranade S. D., Khanal P., Mishra S., Patil D., Salve P. S., Jalalpure S. S.. Design,
synthesis, and molecular docking study of some
2-((7-chloroquinolin-4-yl) amino) benzohydrazide Schiff bases as potential
Eg5 inhibitory agents. Bioorg. Chem. 2021;116:105381.  doi: 10.1016/j.bioorg.2021.105381.</cite> [<a href="https://doi.org/10.1016/j.bioorg.2021.105381" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34601297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Chem.&amp;title=Design,%20synthesis,%20and%20molecular%20docking%20study%20of%20some%202-((7-chloroquinolin-4-yl)%20amino)%20benzohydrazide%20Schiff%20bases%20as%20potential%20Eg5%20inhibitory%20agents&amp;author=R.%20S.%20Kavalapure&amp;author=S.%20G.%20Alegaon&amp;author=U.%20Venkatasubramanian&amp;author=A.%20S.%20Priya&amp;author=S.%20D.%20Ranade&amp;volume=116&amp;publication_year=2021&amp;pages=105381&amp;pmid=34601297&amp;doi=10.1016/j.bioorg.2021.105381&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34">
<cite>Gharge S., Alegaon S. G., Ranade S. D., Kavalapure R. S., Kumar B. P., Mhaske P. C.. Expression of PPAR-γ
TF by
newly synthesized thiazolidine-2, 4-diones to manage glycemic control:
Insights from in silico, in vitro and experimental pharmacology in
wistar rats. Bioorg. Chem. 2024;153:107966.  doi: 10.1016/j.bioorg.2024.107966.</cite> [<a href="https://doi.org/10.1016/j.bioorg.2024.107966" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39579552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Chem.&amp;title=Expression%20of%20PPAR-%CE%B3%20TF%20by%20newly%20synthesized%20thiazolidine-2,%204-diones%20to%20manage%20glycemic%20control:%20Insights%20from%20in%20silico,%20in%20vitro%20and%20experimental%20pharmacology%20in%20wistar%20rats&amp;author=S.%20Gharge&amp;author=S.%20G.%20Alegaon&amp;author=S.%20D.%20Ranade&amp;author=R.%20S.%20Kavalapure&amp;author=B.%20P.%20Kumar&amp;volume=153&amp;publication_year=2024&amp;pages=107966&amp;pmid=39579552&amp;doi=10.1016/j.bioorg.2024.107966&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref35">
<cite>Kahksha, Alam O., Al-Keridis L. A., Khan J., Naaz S., Alam A., Ashraf S. A., Alshammari N., Adnan M., Beg M. A.. Evaluation
of antidiabetic
effect of luteolin in STZ induced diabetic rats: molecular docking,
molecular dynamics, in vitro and in vivo studies. J. Funct. Biomater. 2023;14:126. doi: 10.3390/jfb14030126.</cite> [<a href="https://doi.org/10.3390/jfb14030126" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10053838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36976050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Funct.%20Biomater.&amp;title=Evaluation%20of%20antidiabetic%20effect%20of%20luteolin%20in%20STZ%20induced%20diabetic%20rats:%20molecular%20docking,%20molecular%20dynamics,%20in%20vitro%20and%20in%20vivo%20studies&amp;author=%20Kahksha&amp;author=O.%20Alam&amp;author=L.%20A.%20Al-Keridis&amp;author=J.%20Khan&amp;author=S.%20Naaz&amp;volume=14&amp;publication_year=2023&amp;pages=126&amp;pmid=36976050&amp;doi=10.3390/jfb14030126&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>Akinwumi I. A., Rabie A. M., Katiyar K., Ajayi A., Bello R. O., Aborode A. T., Moin A. T., Ferdoush J., Ogunyemi A., Etinosa P., Osinuga A.. In-silico
discovery of Dipeptidyl
Peptidase-4 inhibitors from African medicinal plants: Molecular docking,
ADMET, dynamics simulation, and MM-GBSA analyses. Nucleus. 2025;68:1–23. doi: 10.1007/s13237-024-00526-x.</cite> [<a href="https://doi.org/10.1007/s13237-024-00526-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleus&amp;title=In-silico%20discovery%20of%20Dipeptidyl%20Peptidase-4%20inhibitors%20from%20African%20medicinal%20plants:%20Molecular%20docking,%20ADMET,%20dynamics%20simulation,%20and%20MM-GBSA%20analyses&amp;author=I.%20A.%20Akinwumi&amp;author=A.%20M.%20Rabie&amp;author=K.%20Katiyar&amp;author=A.%20Ajayi&amp;author=R.%20O.%20Bello&amp;volume=68&amp;publication_year=2025&amp;pages=1-23&amp;doi=10.1007/s13237-024-00526-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>Ranade S. D., Alegaon S. G., Khatib N. A., Gharge S., Kavalapure R. S., Kumar B. P.. Reversal of insulin
resistance to combat type 2 Diabetes
mellitus by newer thiazolidinedione’s in fructose induced insulin
resistant rats. Eur. J. Med. Chem. 2024;280:116939.  doi: 10.1016/j.ejmech.2024.116939.</cite> [<a href="https://doi.org/10.1016/j.ejmech.2024.116939" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39396421/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Med.%20Chem.&amp;title=Reversal%20of%20insulin%20resistance%20to%20combat%20type%202%20Diabetes%20mellitus%20by%20newer%20thiazolidinedione%E2%80%99s%20in%20fructose%20induced%20insulin%20resistant%20rats&amp;author=S.%20D.%20Ranade&amp;author=S.%20G.%20Alegaon&amp;author=N.%20A.%20Khatib&amp;author=S.%20Gharge&amp;author=R.%20S.%20Kavalapure&amp;volume=280&amp;publication_year=2024&amp;pages=116939&amp;pmid=39396421&amp;doi=10.1016/j.ejmech.2024.116939&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Gharge S., Alegaon S. G., Ranade S. D., Kavalapure R. S., Kumar B. P.. Novel rhodanine–thiazole hybrids
as potential
antidiabetic agents: a structure-based drug design approach. RSC Med. Chem. 2025;16:927–944. doi: 10.1039/D4MD00689E.</cite> [<a href="https://doi.org/10.1039/D4MD00689E" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11650657/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39697245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Med.%20Chem.&amp;title=Novel%20rhodanine%E2%80%93thiazole%20hybrids%20as%20potential%20antidiabetic%20agents:%20a%20structure-based%20drug%20design%20approach&amp;author=S.%20Gharge&amp;author=S.%20G.%20Alegaon&amp;author=S.%20D.%20Ranade&amp;author=R.%20S.%20Kavalapure&amp;author=B.%20P.%20Kumar&amp;volume=16&amp;publication_year=2025&amp;pages=927-944&amp;pmid=39697245&amp;doi=10.1039/D4MD00689E&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Alegaon S. G., Venkatasubramanian U., Alagawadi K. R., Kumar D., Kavalapure R. S., Ranade S. D., Priya A. S., Jalalpure S. S.. Synthesis,
molecular docking and ADME studies of thiazole-thiazolidinedione hybrids
as antimicrobial agents. J. Biomol. Struct.
Dyn. 2022;40:6211–6227. doi: 10.1080/07391102.2021.1880479.</cite> [<a href="https://doi.org/10.1080/07391102.2021.1880479" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33538239/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20Struct.%20Dyn.&amp;title=Synthesis,%20molecular%20docking%20and%20ADME%20studies%20of%20thiazole-thiazolidinedione%20hybrids%20as%20antimicrobial%20agents&amp;author=S.%20G.%20Alegaon&amp;author=U.%20Venkatasubramanian&amp;author=K.%20R.%20Alagawadi&amp;author=D.%20Kumar&amp;author=R.%20S.%20Kavalapure&amp;volume=40&amp;publication_year=2022&amp;pages=6211-6227&amp;pmid=33538239&amp;doi=10.1080/07391102.2021.1880479&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Ranade S. D., Alegaon S. G., Venkatasubramanian U., Priya A. S., Kavalapure R. S., Chand J., Jalalpure S. S., Vinod D.. Design, synthesis,
molecular dynamics simulation, MM/GBSA studies and kinesin spindle
protein inhibitory evaluation of some 4-aminoquinoline hybrids. Comput. Biol. Chem. 2023;105:107881.  doi: 10.1016/j.compbiolchem.2023.107881.</cite> [<a href="https://doi.org/10.1016/j.compbiolchem.2023.107881" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37257398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput.%20Biol.%20Chem.&amp;title=Design,%20synthesis,%20molecular%20dynamics%20simulation,%20MM/GBSA%20studies%20and%20kinesin%20spindle%20protein%20inhibitory%20evaluation%20of%20some%204-aminoquinoline%20hybrids&amp;author=S.%20D.%20Ranade&amp;author=S.%20G.%20Alegaon&amp;author=U.%20Venkatasubramanian&amp;author=A.%20S.%20Priya&amp;author=R.%20S.%20Kavalapure&amp;volume=105&amp;publication_year=2023&amp;pages=107881&amp;pmid=37257398&amp;doi=10.1016/j.compbiolchem.2023.107881&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Gharge S., Alegaon S. G., Jadhav S., Ranade S. D., Kavalapure R. S.. Design,
synthesis, characterization and Antidiabetic evaluation of 3, 5-substituted
thiazolidinediones: Evidenced by network pharmacology, Molecular docking,
dynamic simulation, in vitro and in vivo assessment. Eur. J. Med. Chem. Rep. 2024;12:100213.  doi: 10.1016/j.ejmcr.2024.100213.</cite> [<a href="https://doi.org/10.1016/j.ejmcr.2024.100213" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Med.%20Chem.%20Rep.&amp;title=Design,%20synthesis,%20characterization%20and%20Antidiabetic%20evaluation%20of%203,%205-substituted%20thiazolidinediones:%20Evidenced%20by%20network%20pharmacology,%20Molecular%20docking,%20dynamic%20simulation,%20in%20vitro%20and%20in%20vivo%20assessment&amp;author=S.%20Gharge&amp;author=S.%20G.%20Alegaon&amp;author=S.%20Jadhav&amp;author=S.%20D.%20Ranade&amp;author=R.%20S.%20Kavalapure&amp;volume=12&amp;publication_year=2024&amp;pages=100213&amp;doi=10.1016/j.ejmcr.2024.100213&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Ramu R., Shirahatti P. S., Zameer F., Ranganatha L. V., Prasad M. N.. Inhibitory effect of banana (Musa sp. var. Nanjangud
rasa bale) flower extract and its constituents Umbelliferone and Lupeol
on α-glucosidase, aldose reductase and glycation at multiple
stages. S. Afr. J. Bot. 2014;95:54–63. doi: 10.1016/j.sajb.2014.08.001.</cite> [<a href="https://doi.org/10.1016/j.sajb.2014.08.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=S.%20Afr.%20J.%20Bot.&amp;title=Inhibitory%20effect%20of%20banana%20(Musa%20sp.%20var.%20Nanjangud%20rasa%20bale)%20flower%20extract%20and%20its%20constituents%20Umbelliferone%20and%20Lupeol%20on%20%CE%B1-glucosidase,%20aldose%20reductase%20and%20glycation%20at%20multiple%20stages&amp;author=R.%20Ramu&amp;author=P.%20S.%20Shirahatti&amp;author=F.%20Zameer&amp;author=L.%20V.%20Ranganatha&amp;author=M.%20N.%20Prasad&amp;volume=95&amp;publication_year=2014&amp;pages=54-63&amp;doi=10.1016/j.sajb.2014.08.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Nishimura C., Yamaoka T., Mizutani M., Yamashita K., Akera T., Tanimoto T.. Purification and characterization
of the recombinant human aldose reductase expressed in baculovirus
system. Biochim. Biophys. Acta. 1991;1078:171–178. doi: 10.1016/0167-4838(91)99006-E.</cite> [<a href="https://doi.org/10.1016/0167-4838(91)99006-E" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1905957/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta&amp;title=Purification%20and%20characterization%20of%20the%20recombinant%20human%20aldose%20reductase%20expressed%20in%20baculovirus%20system&amp;author=C.%20Nishimura&amp;author=T.%20Yamaoka&amp;author=M.%20Mizutani&amp;author=K.%20Yamashita&amp;author=T.%20Akera&amp;volume=1078&amp;publication_year=1991&amp;pages=171-178&amp;pmid=1905957&amp;doi=10.1016/0167-4838(91)99006-E&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref44">
<cite>Lineweaver H., Burk D.. The determination of
enzyme dissociation constants. J. Am. Chem.
Soc. 1934;56:658–666. doi: 10.1021/ja01318a036.</cite> [<a href="https://doi.org/10.1021/ja01318a036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Am.%20Chem.%20Soc.&amp;title=The%20determination%20of%20enzyme%20dissociation%20constants&amp;author=H.%20Lineweaver&amp;author=D.%20Burk&amp;volume=56&amp;publication_year=1934&amp;pages=658-666&amp;doi=10.1021/ja01318a036&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref45">
<cite>Dixon M.. The determination
of enzyme inhibitor constants. Biochem. J. 1953;55:170–171. doi: 10.1042/bj0550170.</cite> [<a href="https://doi.org/10.1042/bj0550170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1269152/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/13093635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20J.&amp;title=The%20determination%20of%20enzyme%20inhibitor%20constants&amp;author=M.%20Dixon&amp;volume=55&amp;publication_year=1953&amp;pages=170-171&amp;pmid=13093635&amp;doi=10.1042/bj0550170&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref46">
<cite>Chowtivannakul P., Srichaikul B., Talubmook C.. Antidiabetic and antioxidant activities
of seed extract from Leucaena leucocephala (Lam.) de Wit. Agric. Nat. Resour. 2016;50:357–361. doi: 10.1016/j.anres.2016.06.007.</cite> [<a href="https://doi.org/10.1016/j.anres.2016.06.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agric.%20Nat.%20Resour.&amp;title=Antidiabetic%20and%20antioxidant%20activities%20of%20seed%20extract%20from%20Leucaena%20leucocephala%20(Lam.)%20de%20Wit&amp;author=P.%20Chowtivannakul&amp;author=B.%20Srichaikul&amp;author=C.%20Talubmook&amp;volume=50&amp;publication_year=2016&amp;pages=357-361&amp;doi=10.1016/j.anres.2016.06.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12332697/bin/ao5c04663_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">ao5c04663_si_001.pdf</a><sup> (4.9MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>No data generated
or analyzed during this study</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Omega are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsomega.5c04663"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/ao5c04663.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (7.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12332697/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12332697/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332697%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332697/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12332697/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12332697/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40787348/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12332697/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40787348/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12332697/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12332697/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="aeuSnYebArtU6q7ObqlSrzRoHCje4rjM3rarPXl3mnFgiXrBwW8uaZOYE7M5NGHH">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
