
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            A Deeply Branching Lineage in Heterolobosea (Discoba) With Multiflagellated and Multigrooved Life Stages - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="4DDE475F89B71A531B475F001D8B13F5.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328842/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Journal of Eukaryotic Microbiology">
<meta name="citation_title" content="A Deeply Branching Lineage in Heterolobosea (Discoba) With Multiflagellated and Multigrooved Life Stages">
<meta name="citation_author" content="Kristina I Prokina">
<meta name="citation_author_institution" content="Ecologie Société et Evolution, CNRS, Université Paris‐Saclay, AgroParisTech, Gif‐sur‐Yvette, France">
<meta name="citation_author_institution" content="Papanin Institute for Biology of Inland Waters Russian Academy of Science, Borok, Russia">
<meta name="citation_author" content="Naoji Yubuki">
<meta name="citation_author_institution" content="Ecologie Société et Evolution, CNRS, Université Paris‐Saclay, AgroParisTech, Gif‐sur‐Yvette, France">
<meta name="citation_author_institution" content="Institut Curie, Université PSL, CNRS UAR2016, Inserm US43, Université Paris‐Saclay, Multimodal Imaging Center, Orsay, France">
<meta name="citation_author" content="Luis Javier Galindo">
<meta name="citation_author_institution" content="Ecologie Société et Evolution, CNRS, Université Paris‐Saclay, AgroParisTech, Gif‐sur‐Yvette, France">
<meta name="citation_author_institution" content="Institute of Water Research, University of Granada, Granada, Spain">
<meta name="citation_author_institution" content="Department of Ecology, University of Granada, Granada, Spain">
<meta name="citation_author" content="Guifré Torruella">
<meta name="citation_author_institution" content="Ecologie Société et Evolution, CNRS, Université Paris‐Saclay, AgroParisTech, Gif‐sur‐Yvette, France">
<meta name="citation_author_institution" content="Institut de Biologia Evolutiva, UPF‐CSIC, Barcelona, Catalonia, Spain">
<meta name="citation_author" content="Yuji Inagaki">
<meta name="citation_author_institution" content="Center for Computational Sciences, University of Tsukuba, Tsukuba, Japan">
<meta name="citation_author" content="Koji Iwamoto">
<meta name="citation_author_institution" content="Malaysia‐Japan International Institute of Technology, Universiti Teknologi Malaysia, Kuala Lumpur, Malaysia">
<meta name="citation_author" content="Purificación López‐García">
<meta name="citation_author_institution" content="Ecologie Société et Evolution, CNRS, Université Paris‐Saclay, AgroParisTech, Gif‐sur‐Yvette, France">
<meta name="citation_author" content="David Moreira">
<meta name="citation_author_institution" content="Ecologie Société et Evolution, CNRS, Université Paris‐Saclay, AgroParisTech, Gif‐sur‐Yvette, France">
<meta name="citation_publication_date" content="2025 Aug 6">
<meta name="citation_volume" content="72">
<meta name="citation_issue" content="5">
<meta name="citation_firstpage" content="e70037">
<meta name="citation_doi" content="10.1111/jeu.70037">
<meta name="citation_pmid" content="40770464">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328842/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328842/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328842/pdf/JEU-72-e70037.pdf">
<meta name="description" content="Several distantly related, deep‐branching lineages within the eukaryotic tree of life share what is known as the “typical excavate” morphology. This cell architecture includes a suspension‐feeding ventral groove, supported by complex and specific ...">
<meta name="og:title" content="A Deeply Branching Lineage in Heterolobosea (Discoba) With Multiflagellated and Multigrooved Life Stages">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Several distantly related, deep‐branching lineages within the eukaryotic tree of life share what is known as the “typical excavate” morphology. This cell architecture includes a suspension‐feeding ventral groove, supported by complex and specific ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328842/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12328842">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1111/jeu.70037"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/JEU-72-e70037.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12328842%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12328842/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12328842/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328842/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1111/jeu.70037" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">J Eukaryot Microbiol</button></div>. 2025 Aug 6;72(5):e70037. doi: <a href="https://doi.org/10.1111/jeu.70037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1111/jeu.70037</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22J%20Eukaryot%20Microbiol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22J%20Eukaryot%20Microbiol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22J%20Eukaryot%20Microbiol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22J%20Eukaryot%20Microbiol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>A Deeply Branching Lineage in Heterolobosea (Discoba) With Multiflagellated and Multigrooved Life Stages</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prokina%20KI%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Kristina I Prokina</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Kristina I Prokina</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Ecologie Société et Evolution, CNRS, Université Paris‐Saclay, AgroParisTech, Gif‐sur‐Yvette, France</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Papanin Institute for Biology of Inland Waters Russian Academy of Science, Borok, Russia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prokina%20KI%22%5BAuthor%5D" class="usa-link"><span class="name western">Kristina I Prokina</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yubuki%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Naoji Yubuki</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Naoji Yubuki</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Ecologie Société et Evolution, CNRS, Université Paris‐Saclay, AgroParisTech, Gif‐sur‐Yvette, France</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Institut Curie, Université PSL, CNRS UAR2016, Inserm US43, Université Paris‐Saclay, Multimodal Imaging Center, Orsay, France</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yubuki%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Naoji Yubuki</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Galindo%20LJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Luis Javier Galindo</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Luis Javier Galindo</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Ecologie Société et Evolution, CNRS, Université Paris‐Saclay, AgroParisTech, Gif‐sur‐Yvette, France</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>Institute of Water Research, University of Granada, Granada, Spain</div>
<div class="p">
<sup>
<sup>5</sup>
</sup>Department of Ecology, University of Granada, Granada, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Galindo%20LJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Luis Javier Galindo</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Torruella%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Guifré Torruella</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Guifré Torruella</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Ecologie Société et Evolution, CNRS, Université Paris‐Saclay, AgroParisTech, Gif‐sur‐Yvette, France</div>
<div class="p">
<sup>
<sup>6</sup>
</sup>Institut de Biologia Evolutiva, UPF‐CSIC, Barcelona, Catalonia, Spain</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Torruella%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Guifré Torruella</span></a>
</div>
</div>
<sup>1,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Inagaki%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Yuji Inagaki</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Yuji Inagaki</span></h3>
<div class="p">
<sup>
<sup>7</sup>
</sup>Center for Computational Sciences, University of Tsukuba, Tsukuba, Japan</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Inagaki%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuji Inagaki</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Iwamoto%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Koji Iwamoto</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Koji Iwamoto</span></h3>
<div class="p">
<sup>
<sup>8</sup>
</sup>Malaysia‐Japan International Institute of Technology, Universiti Teknologi Malaysia, Kuala Lumpur, Malaysia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Iwamoto%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Koji Iwamoto</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22L%C3%B3pez%E2%80%90Garc%C3%ADa%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Purificación López‐García</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Purificación López‐García</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Ecologie Société et Evolution, CNRS, Université Paris‐Saclay, AgroParisTech, Gif‐sur‐Yvette, France</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22L%C3%B3pez%E2%80%90Garc%C3%ADa%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Purificación López‐García</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moreira%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">David Moreira</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">David Moreira</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Ecologie Société et Evolution, CNRS, Université Paris‐Saclay, AgroParisTech, Gif‐sur‐Yvette, France</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moreira%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">David Moreira</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="jeu70037-aff-0001">
<sup>
<sup>1</sup>
</sup>Ecologie Société et Evolution, CNRS, Université Paris‐Saclay, AgroParisTech, Gif‐sur‐Yvette, France</div>
<div id="jeu70037-aff-0002">
<sup>
<sup>2</sup>
</sup>Papanin Institute for Biology of Inland Waters Russian Academy of Science, Borok, Russia</div>
<div id="jeu70037-aff-0003">
<sup>
<sup>3</sup>
</sup>Institut Curie, Université PSL, CNRS UAR2016, Inserm US43, Université Paris‐Saclay, Multimodal Imaging Center, Orsay, France</div>
<div id="jeu70037-aff-0004">
<sup>
<sup>4</sup>
</sup>Institute of Water Research, University of Granada, Granada, Spain</div>
<div id="jeu70037-aff-0005">
<sup>
<sup>5</sup>
</sup>Department of Ecology, University of Granada, Granada, Spain</div>
<div id="jeu70037-aff-0006">
<sup>
<sup>6</sup>
</sup>Institut de Biologia Evolutiva, UPF‐CSIC, Barcelona, Catalonia, Spain</div>
<div id="jeu70037-aff-0007">
<sup>
<sup>7</sup>
</sup>Center for Computational Sciences, University of Tsukuba, Tsukuba, Japan</div>
<div id="jeu70037-aff-0008">
<sup>
<sup>8</sup>
</sup>Malaysia‐Japan International Institute of Technology, Universiti Teknologi Malaysia, Kuala Lumpur, Malaysia</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong>
 
Kristina I. Prokina (<span>kris142014@gmail.com</span>), 
David Moreira (<span>david.moreira@universite-paris-saclay.fr</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jun 27; Received 2024 Dec 18; Accepted 2025 Jul 28; Issue date 2025 Sep-Oct.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Journal of Eukaryotic Microbiology</em> published by Wiley Periodicals LLC on behalf of International Society of Protistologists.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12328842  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40770464/" class="usa-link">40770464</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>Several distantly related, deep‐branching lineages within the eukaryotic tree of life share what is known as the “typical excavate” morphology. This cell architecture includes a suspension‐feeding ventral groove, supported by complex and specific structures of the cytoskeleton, and a vane‐bearing posterior flagellum. Within Discoba, such characteristics were found only in the deepest branch, Jakobida, also famous for having the most gene‐rich and bacterial‐like mitogenomes. Some members of the discoban lineage Heterolobosea also possess a ventral groove. However, they lost many other typical excavate characteristics, including the flagellar vane and some flagellar root elements. We have established a clonal culture of a free‐living freshwater flagellate from Malaysia, strain NY0229, described here under the provisional name <em>Multisulcus malaysiensis</em> nom. prov. Electron microscopy observations show a conspicuous ventral groove reinforced with microtubules of the flagellar apparatus and a vane on the posterior flagellum. Strain NY0229 has typical excavate biflagellated forms but also exhibits very atypical multiflagellated and multigrooved forms. The kinetid consists of two orthogonally oriented kinetosomes; both flagella of biflagellate cells display proximal and distal projections. Molecular phylogenetic analyses support a deep‐branching position of the new strain within Heterolobosea, as the sister lineage to all previously known Eutetramitia.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> electron microscopy, Heterolobosea, karyomastigont, multigene phylogeny, 
<em>Multisulcus malaysiensis</em> nom. prov, typical excavates, ultrastructure</p></section></section><section id="jeu70037-sec-0001"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Excavates are a phylogenetically diverse group of single‐celled eukaryotes distinguished by their peculiar ventral (excavated) feeding groove and its associated complex flagellar apparatus. This group encompasses several notable lineages, including both free‐living and parasitic species, with adaptations allowing them to thrive in environments ranging from aquatic habitats to animal hosts. Within this group, “typical excavates”, that is, jakobids, malawimonads, <em>Trimastix</em>, <em>Paratrimastix</em>, “<em>Carpediemonas</em>‐like organisms” and retortamonads, showcase characteristic similar cellular structures, most especially encompassing unique combinations of the complex flagellar apparatus elements, such as vane(s) on the posterior flagellum, I fiber, B fiber, C fiber, split R2, singlet root and a composite fiber (Bernard et al. <a href="#jeu70037-bib-0006" class="usa-link" aria-describedby="jeu70037-bib-0006">1997</a>; O'Kelly <a href="#jeu70037-bib-0034" class="usa-link" aria-describedby="jeu70037-bib-0034">1993</a>; O'Kelly and Nerad <a href="#jeu70037-bib-0036" class="usa-link" aria-describedby="jeu70037-bib-0036">1999</a>; O'Kelly et al. <a href="#jeu70037-bib-0035" class="usa-link" aria-describedby="jeu70037-bib-0035">1999</a>; Simpson <a href="#jeu70037-bib-0051" class="usa-link" aria-describedby="jeu70037-bib-0051">2003</a>; Simpson and Patterson <a href="#jeu70037-bib-0052" class="usa-link" aria-describedby="jeu70037-bib-0052">1999</a>).</p>
<p>Excavates are currently treated more as a convenient grouping of certain protists with similar morphological traits rather than a strict monophyletic lineage. Their phylogeny remains unresolved, and many studies suggest them to be polyphyletic (Baldauf <a href="#jeu70037-bib-0004" class="usa-link" aria-describedby="jeu70037-bib-0004">2003</a>; Burki et al. <a href="#jeu70037-bib-0010" class="usa-link" aria-describedby="jeu70037-bib-0010">2020</a>; Derelle et al. <a href="#jeu70037-bib-0015" class="usa-link" aria-describedby="jeu70037-bib-0015">2015</a>; Torruella et al. <a href="#jeu70037-bib-0054" class="usa-link" aria-describedby="jeu70037-bib-0054">2025</a>). Since typical excavates may thus belong to distinct eukaryotic lineages, which might even lie on opposite splits of the eukaryotic tree according to the likely place of its root, some scientists suggest that the typical excavate cell structure may be ancestral to all eukaryotes (Simpson <a href="#jeu70037-bib-0051" class="usa-link" aria-describedby="jeu70037-bib-0051">2003</a>; Suzuki‐Tellier et al. <a href="#jeu70037-bib-0053" class="usa-link" aria-describedby="jeu70037-bib-0053">2024</a>; Torruella et al. <a href="#jeu70037-bib-0054" class="usa-link" aria-describedby="jeu70037-bib-0054">2025</a>; Williamson et al. <a href="#jeu70037-bib-0056" class="usa-link" aria-describedby="jeu70037-bib-0056">2024</a>).</p>
<p>Among excavates, the Discoba represents a particularly intriguing clade. Discoba members exhibit distinctive discoidal mitochondrial cristae and include important model organisms (Gibson <a href="#jeu70037-bib-0018" class="usa-link" aria-describedby="jeu70037-bib-0018">2017</a>; Leander et al. <a href="#jeu70037-bib-0029" class="usa-link" aria-describedby="jeu70037-bib-0029">2017</a>; Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>). This clade encompasses different lineages with specialized characteristics and ecological roles. Only the deepest branching discoban lineage, the jakobids, exhibits a typical excavate morphology (Simpson <a href="#jeu70037-bib-0051" class="usa-link" aria-describedby="jeu70037-bib-0051">2003</a>). Another discoban lineage, the Heterolobosea, is evolutionarily important because of its unique morphological and genetic characteristics (Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>). Members of this group are single‐celled, flagellated protists known for their diverse life stages and cellular structures. They are notable for having a flexible amoeboid form at some stages of their life cycle and can transition between amoeboid, flagellate, and cyst stages (Harding et al. <a href="#jeu70037-bib-0021" class="usa-link" aria-describedby="jeu70037-bib-0021">2013</a>; Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>, <a href="#jeu70037-bib-0041" class="usa-link" aria-describedby="jeu70037-bib-0041">2025</a>). Heteroloboseans are predominantly free‐living, although some are endobionts or facultative pathogens of animals and humans (Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>). They are found in very diverse environments, and many lineages independently adapted to anoxic and/or extreme environments, such as hypersaline, acidic, and thermal habitats (Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>). They have retained some of the ancestral excavate characteristics, including the suspension‐feeding groove, the I fiber, the split of the right root R2, and ancestrally the C fiber (Hanousková et al. <a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>; Park and Simpson <a href="#jeu70037-bib-0043" class="usa-link" aria-describedby="jeu70037-bib-0043">2011</a>; Simpson <a href="#jeu70037-bib-0051" class="usa-link" aria-describedby="jeu70037-bib-0051">2003</a>). However, they are not generally recognized as typical excavates because they lack other characteristic traits such as the vane on the posterior flagellum, the B fiber, a singlet root, and a composite fiber (Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>; Simpson <a href="#jeu70037-bib-0051" class="usa-link" aria-describedby="jeu70037-bib-0051">2003</a>). Root R1 was thought to be lost in the majority of heterolobosean lineages, but it is found in the deep‐branching genera <em>Pharyngomonas</em> and <em>Dactylomonas</em>, and presumably also in <em>Creneis carolina</em> (Hanousková et al. <a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>; Pánek, Simpson, et al. <a href="#jeu70037-bib-0039" class="usa-link" aria-describedby="jeu70037-bib-0039">2014</a>; Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>; Park and Simpson <a href="#jeu70037-bib-0043" class="usa-link" aria-describedby="jeu70037-bib-0043">2011</a>). Only <em>Pharyngomonas</em> has retained the C fiber supporting the R1 root (Park and Simpson <a href="#jeu70037-bib-0043" class="usa-link" aria-describedby="jeu70037-bib-0043">2011</a>).</p>
<p>A characteristic cellular structure found in all heterolobosean lineages is the absence of classical stacked Golgi dictyosomes (Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>). Mitochondria, if present, contain flattened or discoidal cristae and are usually enveloped by endoplasmic reticulum (Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>). The heterolobosean flagellar apparatus has a characteristic “double bikont” structure in which all flagellar elements are doubled (Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>), and kinetosomes are oriented nearly parallel to each other in most lineages, except in the deep‐branching <em>Pharyngomonas</em> spp. and <em>Dactylomonas</em> spp. (Hanousková et al. <a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>; Park and Simpson <a href="#jeu70037-bib-0043" class="usa-link" aria-describedby="jeu70037-bib-0043">2011</a>).</p>
<p>Heterolobosea are known for their complex life cycles, which exhibit alternations between amoeboid, flagellated, and cyst stages (Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>). One or two life stages may be missing in some lineages, often within the same clade. For example, <em>Selenaion koniopes</em> is a typical heterolobosean amoeba with eruptive pseudopodia and cysts (Park et al. <a href="#jeu70037-bib-0042" class="usa-link" aria-describedby="jeu70037-bib-0042">2012</a>), while its sister lineage <em>Dactylomonas</em> is a typical flagellate with unknown amoeboid and cyst stages (Hanousková et al. <a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>). By contrast, all three stages of the life cycle are found in the deepest‐branching lineage, <em>Pharyngomonas</em>, suggesting that this feature may have been present in the last common ancestor of Heterolobosea (Harding et al. <a href="#jeu70037-bib-0021" class="usa-link" aria-describedby="jeu70037-bib-0021">2013</a>).</p>
<p>Heterolobosea comprise two sister subphyla, Pharyngomonada and Tetramitia (Yang et al. <a href="#jeu70037-bib-0059" class="usa-link" aria-describedby="jeu70037-bib-0059">2017</a>). The species‐poor Pharyngomonada includes the single genus <em>Pharyngomonas</em>, with halotolerant or halophilic representatives (Park and Simpson <a href="#jeu70037-bib-0043" class="usa-link" aria-describedby="jeu70037-bib-0043">2011</a>, <a href="#jeu70037-bib-0044" class="usa-link" aria-describedby="jeu70037-bib-0044">2016</a>). A rare fusion of two key genes coding for peroxisomal β‐oxidation enzymes (MFP and ACCA1) was recently identified as an autapomorphy for the subphylum Pharyngomonada (Pánek et al. <a href="#jeu70037-bib-0041" class="usa-link" aria-describedby="jeu70037-bib-0041">2025</a>). Members of the taxonomically diverse Tetramitia exhibit a wide range of morphologies, including typically tetraflagellated and amoeboid cells, but also the multiflagellated pseudociliate <em>Stephanopogon</em> spp. (Lee et al. <a href="#jeu70037-bib-0030" class="usa-link" aria-describedby="jeu70037-bib-0030">2014</a>; Patterson and Brugerolle <a href="#jeu70037-bib-0045" class="usa-link" aria-describedby="jeu70037-bib-0045">1988</a>; Yubuki and Leander <a href="#jeu70037-bib-0060" class="usa-link" aria-describedby="jeu70037-bib-0060">2008</a>), and the aggregative slime molds <em>Acrasis</em> and <em>Pocheina</em> with multicellular fruiting bodies (Brown et al. <a href="#jeu70037-bib-0008" class="usa-link" aria-describedby="jeu70037-bib-0008">2010</a>; Hohl and Hamamoto <a href="#jeu70037-bib-0022" class="usa-link" aria-describedby="jeu70037-bib-0022">1969</a>). The presence of an extra helix (17_1) in the 18S rRNA molecule is considered a synapomorphy for the subphylum Tetramitia (Nikolaev et al. <a href="#jeu70037-bib-0033" class="usa-link" aria-describedby="jeu70037-bib-0033">2004</a>). The phylogenetic relationships among the subgroups of Tetramitia remain not fully resolved due to the lack of massive sequence data for some representatives (Pánek, Simpson, et al. <a href="#jeu70037-bib-0039" class="usa-link" aria-describedby="jeu70037-bib-0039">2014</a>; Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>, <a href="#jeu70037-bib-0041" class="usa-link" aria-describedby="jeu70037-bib-0041">2025</a>). Pánek, Simpson, et al. (<a href="#jeu70037-bib-0039" class="usa-link" aria-describedby="jeu70037-bib-0039">2014</a>) divided Tetramitia into eight subgroups (I–VIII) plus the species <em>C. carolina</em>. Recently, the new class Selenaionea, uniting Selenaionida and Neovahlkampfiida, was found to be the first to diverge based on molecular and morphological data, thereby distinguishing them from the class Eutetramitia (Hanousková et al. <a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>; Pánek, Simpson, et al. <a href="#jeu70037-bib-0039" class="usa-link" aria-describedby="jeu70037-bib-0039">2014</a>; Park et al. <a href="#jeu70037-bib-0042" class="usa-link" aria-describedby="jeu70037-bib-0042">2012</a>).</p>
<p>In this study, we isolated a free‐living freshwater bacteriotrophic flagellated strain (NY0229) from Malaysia. The new strain was characterized by a typical excavate‐like biflagellated form but also exhibited unusual multiflagellated and multigrooved forms, constantly present in our culture. Unfortunately, the strain NY0229 died after 2 years of cultivation, leaving some gaps in its morphological characterization. Therefore, we do not formulate here its formal taxonomic diagnosis but provide a provisional name (<em>Multisulcus malaysiensis</em> nom. prov.) that can be used for future research. The provisional name is an unusual formulation within Zoological nomenclature (in particular) and has no status within Zoological nomenclature, nor any other code. Both 18S rRNA gene and multigene molecular phylogenetic analyses supported a very deep position of the new strain within Tetramitia (Heterolobosea), sister to Selenaionea and Eutetramitia.</p></section><section id="jeu70037-sec-0002"><h2 class="pmc_sec_title">2. Materials and Methods</h2>
<section id="jeu70037-sec-0003"><h3 class="pmc_sec_title">2.1. Sample Collection and Culture Establishment</h3>
<p>Water samples were collected from a freshwater body in the palm oil factory Seri Ulu Langat Palm Oil Mill Sdn Bhd, Batu 34, Jalan Banting, Dengkil, 43800, Dengkil, Selangor, 43800, Malaysia, on February 18, 2018. The clonal culture strain NY0229 was obtained by isolating individual cells using a glass micropipette and transferring them to Volvic mineral water (Danone, France) supplemented with 1% YT (100 mg of yeast extract and 200 mg of tryptone in 100 mL of distilled water as described in the protocol from the National Institute for Environmental Studies, Japan). The nonaxenic culture was maintained at 15°C, with the medium renewed every 2 weeks. Unfortunately, the strain NY0229 died after 2 years of cultivation.</p></section><section id="jeu70037-sec-0004"><h3 class="pmc_sec_title">2.2. Light and Electron Microscopy</h3>
<p>Light microscopy (LM) observations of strain NY0229 were conducted using an inverted Zeiss Axiovert 40 CFL microscope equipped with phase contrast objectives (20× and 40×) and an upright Zeiss Axioplan 2 microscope equipped with an oil‐immersion DIC objective (100×). Videos were recorded with a Sony α9 digital camera.</p>
<p>For scanning electron microscopy (SEM), a concentrated cell culture was mixed with 4% glutaraldehyde (resulting in a final concentration of 2%) diluted in 0.2 M sodium cacodylate buffer (pH 7.2). The mixture was deposited on a 9‐mm circular coverslip coated with poly‐<span class="font-variant-small-caps">l</span>‐lysine and fixed for 1 h. After fixation, cells were washed with 0.2 M sodium cacodylate buffer (pH 7.2) and dehydrated through a graded series of ethanol baths (30%, 50%, 70%, 80%, 90%, 100% three times for 10 min each). Subsequently, coverslips with cells were washed in 100% hexamethyldisiloxane three times for 15 min and then dried at room temperature. The dry coverslips were mounted on aluminum stubs, coated with platinum, and observed using a GeminiSEM 500 (Carl Zeiss, Germany) scanning electron microscope.</p>
<p>For transmission electron microscopy (TEM), aliquots of cell‐rich cultures were mixed with glutaraldehyde (2.5% final concentration) and paraformaldehyde (2% final concentration) diluted in 0.2 M sodium cacodylate buffer and fixed for 1 h at room temperature. After fixation, the cell pellets were collected by centrifugation for 10 min at 1000<em>g</em>, washed with 0.2 M cacodylate buffer (pH 7.2) and then fixed with 1% osmium tetroxide diluted in phosphate‐buffered saline (PBS) for 1.5 h at room temperature. Following fixation, the cells were dehydrated through a series of alcohol baths (30%, 50%, 70%, 80%, 90%, 96%, 100% twice for 10 min each) and acetone (100% twice for 10 min); the pellets were subsequently embedded in Low Viscosity resin (EM 0300 Sigma‐Aldrich). Ultrathin sections (70 nm) were prepared using a Leica EM UC6 ultramicrotome (Leica Microsystems, Germany), double‐stained with 2% uranyl acetate and 2% lead citrate, and observed using a JEM‐1400 transmission electron microscope (JEOL, Japan).</p></section><section id="jeu70037-sec-0005"><h3 class="pmc_sec_title">2.3. 
18S rRNA Gene PCR Amplification, Sequence Alignment, and Phylogenetic Analyses</h3>
<p>For DNA extraction, a 100 μL volume of cell culture was centrifuged at the maximum speed (13,400<em>g</em>) for 5 min, and the supernatant was removed. Then, 30 μL of 0.1 M Tris buffer (pH 8) were added to the pellet. The mixture was microwaved at low intensity three times for 5 s each to lyse the cells. Finally, the cell lysate was stored at −20°C. PCR amplification of the 18S rRNA gene was performed using a nested approach. In the first PCR reaction, 2 μL of the cell lysate was added to a Platinum Taq polymerase reaction mix (Promega, Lyon, France) with one of these two combinations of universal eukaryotic primers: (A) EK‐1F (5′‐CTGGTTGATCCTGCCAG‐3′) and EK‐1269R (5′‐AAGAACGGCCATGCACCAC‐3′) and (B) EK‐1F and 18S‐1379R (5′‐TGTGTACAAAGGGCAGGGAC‐3′). The resulting PCR products from primer set A were then reamplified using primer set 555F (5′‐AGTCTGGTGCCAGCAGCCGC‐3′) and 1269R, while the products from primer set B were reamplified with the primer set 18S‐42F (5′‐CTCAARGAYTAAGCCATGCA‐3′) and 1379R. The amplification steps were as follows: initial denaturing period at 95°C for 3 min, followed by 35 cycles of denaturation at 95°C for 45 s, annealing at 55°C for 45 s, extension at 72°C for 2 min, and a final extension period at 72°C for 5 min.</p>
<p>Sanger sequencing of amplification products was performed by GENEWIZ (Leipzig, Germany). We used MAFFT (Katoh et al. <a href="#jeu70037-bib-0024" class="usa-link" aria-describedby="jeu70037-bib-0024">2002</a>) with default settings to align the two overlapping sequences and obtain a consensus sequence using the cloud‐based platform Benchling (<a href="https://www" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www</a>. <a href="http://benchling.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">benchling.com</a>/). The resulting 1451 bp long 18S rRNA gene fragment sequence was used to determine the preliminary phylogenetic position of strain NY0229. Subsequent transcriptome analysis (see below) allowed us to extract the complete 18S rRNA gene sequence, 2531 bp long, which was submitted to GenBank under accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ627902" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ627902</a>.</p>
<p>We updated the 18S rRNA gene datasets created by Hanousková et al. (<a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>) and Pánek et al. <a href="#jeu70037-bib-0041" class="usa-link" aria-describedby="jeu70037-bib-0041">2025</a> by including our complete 18S rRNA gene sequence of strain NY0229, resulting in a total of 53 heterolobosean species and including jakobids and <em>Tsukubamonas</em> as outgroups. A multiple sequence alignment was prepared using MAFFT v7.427 (Katoh and Standley <a href="#jeu70037-bib-0025" class="usa-link" aria-describedby="jeu70037-bib-0025">2013</a>) and trimmed with Trimal v1.5 (Capella‐Gutiérrez et al. <a href="#jeu70037-bib-0011" class="usa-link" aria-describedby="jeu70037-bib-0011">2009</a>) by removing sites with gaps in more than 20% of the sequences (‐gt 0.8), resulting in a total of 1436 nucleotide positions. To infer Bayesian phylogenetic trees, the parallel MPI version of MrBayes v.3.2.7a (Ronquist et al. <a href="#jeu70037-bib-0048" class="usa-link" aria-describedby="jeu70037-bib-0048">2012</a>) was used with the GTR model with four gamma‐distributed among‐site rate variation categories (GTR + I + GAMMA4). To calculate posterior probabilities of individual branches, four independent Metropolis‐coupled Markov chains were run for 20 million generations and summarized after the 50% burn‐in. The convergence of log‐likelihood values and model parameters for chains was verified using a plot and convergence diagnostics provided by the MrBayes sump utility. A maximum likelihood (ML) phylogenetic tree was inferred with IQ‐TREE (Minh et al. <a href="#jeu70037-bib-0032" class="usa-link" aria-describedby="jeu70037-bib-0032">2020</a>) v.2.2.2.6 with the GTR + F + I + R5 substitution model chosen by the Bayesian Information Criterion (BIC) using the in‐built ModelFinder (Kalyaanamoorthy et al. <a href="#jeu70037-bib-0023" class="usa-link" aria-describedby="jeu70037-bib-0023">2017</a>) option and 1000 non‐parametric bootstrap pseudoreplicates for statistical support. Trees were visualized and edited with the cloud‐based platform iTOL (Letunic and Bork <a href="#jeu70037-bib-0031" class="usa-link" aria-describedby="jeu70037-bib-0031">2024</a>).</p>
<p>For the reconstruction of the secondary structure of 18S rRNA molecules, we used the RNAstructure WebServer (<a href="https://rna.urmc.rochester.edu/RNAstructureWeb/Servers/Predict1/Predict1.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://rna.urmc.rochester.edu/RNAstructureWeb/Servers/Predict1/Predict1.html</a>).</p></section><section id="jeu70037-sec-0006"><h3 class="pmc_sec_title">2.4. 
RNA‐Seq and Multigene and Mitochondrial Gene Phylogenies</h3>
<p>Cells from grown cultures were collected by centrifugation for 15 min at 15,000<em>g</em> and 10°C. Total RNA was extracted using the RNeasy mini Kit (Qiagen) following the manufacturer's protocol, including a DNAse treatment, and quantified using a Qubit fluorometer (ThermoFisher Scientific). After polyA mRNA selection, cDNA Illumina libraries were constructed, tagged, and paired‐end (2 × 150 bp) sequenced with NovaSeq 6000 S2 (Eurofins Genomics, Germany). Raw reads were submitted to the SRA NCBI database under the Bioproject PRJNA1188566. High‐quality reads, checked with FastQC v0.11.8 (Andrews <a href="#jeu70037-bib-0003" class="usa-link" aria-describedby="jeu70037-bib-0003">2010</a>), were used for de novo assembly with the default version of Spades v3.13.1 (Bankevich et al. <a href="#jeu70037-bib-0005" class="usa-link" aria-describedby="jeu70037-bib-0005">2012</a>) with ‐rna mode.</p>
<p>Prior to the multigene phylogenetic analyses, transcripts were translated using TransDecoder v5.5 (<a href="https://github.com/TransDecoder/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/TransDecoder/</a>) with the LongOrfs option. Gene completeness was investigated using BUSCO v4 (Simão et al. <a href="#jeu70037-bib-0050" class="usa-link" aria-describedby="jeu70037-bib-0050">2015</a>) with the eukaryota_db10 dataset. The peptides were included in a paneukaryotic phylogenomic dataset, as in a previous study (Torruella et al. <a href="#jeu70037-bib-0054" class="usa-link" aria-describedby="jeu70037-bib-0054">2025</a>), using BLASTp (Altschul et al. <a href="#jeu70037-bib-0002" class="usa-link" aria-describedby="jeu70037-bib-0002">1990</a>). Briefly, homologs of 351 conserved protein markers were identified using sequences from <em>Homo</em>, <em>Saprolegnia</em>, <em>Spizellomyces</em>, and <em>Diphylleia</em> as queries. Non‐orthologous sequences were removed after manually inspecting the single marker alignments with 230 taxa [with Aliview v1.26 (Larsson <a href="#jeu70037-bib-0028" class="usa-link" aria-describedby="jeu70037-bib-0028">2014</a>)] and single‐gene trees [with FigTree v1.4.3 (Rambaut <a href="#jeu70037-bib-0047" class="usa-link" aria-describedby="jeu70037-bib-0047">2016</a>)], done with MAFFT v7.427 (Katoh and Standley <a href="#jeu70037-bib-0025" class="usa-link" aria-describedby="jeu70037-bib-0025">2013</a>) L‐INS‐i with 1000 iterations, trimmed with Trimal v1.4.rev22 (Capella‐Gutiérrez et al. <a href="#jeu70037-bib-0011" class="usa-link" aria-describedby="jeu70037-bib-0011">2009</a>) with the automated1 option, and inferred with FastTree v2.1.11 (Price et al. <a href="#jeu70037-bib-0046" class="usa-link" aria-describedby="jeu70037-bib-0046">2010</a>). For the final dataset, most taxa were removed, keeping the 12 outgroup representatives from most eukaryotic supergroups, 8 metamonads as sister to 17 discobans, resulting in a dataset of 350 markers for 37 species. After realigning and trimming with the same software and parameters, the markers were concatenated with alvert.py from the barrel‐o‐monkeys (<a href="http://rogerlab.biochemistryandmolecularbiology.dal.ca/Software/Software.htm#Monkeybarrel" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://rogerlab.biochemistryandmolecularbiology.dal.ca/Software/Software.htm#Monkeybarrel</a>), yielding 112,451 amino acid positions. A ML phylogenetic tree was inferred using IQ‐TREE (Minh et al. <a href="#jeu70037-bib-0032" class="usa-link" aria-describedby="jeu70037-bib-0032">2020</a>) v1.6 under the LG + C60 + G mixture model as chosen by BIC (‐mwopt flag) with 1000 ultrafast bootstraps, and used as a guide tree for the posterior mean site frequencies (PMSF) approximation (Wang et al. <a href="#jeu70037-bib-0055" class="usa-link" aria-describedby="jeu70037-bib-0055">2018</a>) to calculate 100 non‐parametric bootstraps as statistical support.</p>
<p>We used the 14 mitochondrial protein dataset from Galindo et al. (<a href="#jeu70037-bib-0017" class="usa-link" aria-describedby="jeu70037-bib-0017">2023</a>) and updated it with the sequences Cox1 and Nad5 from the NY0229 inferred proteome. These two proteins were the only complete mitochondrial proteins that we could retrieve from our data. The dataset was aligned with MAFFT v7.427 (Katoh and Standley <a href="#jeu70037-bib-0025" class="usa-link" aria-describedby="jeu70037-bib-0025">2013</a>) and trimmed with Trimal v1.4.rev22 (Capella‐Gutiérrez et al. <a href="#jeu70037-bib-0011" class="usa-link" aria-describedby="jeu70037-bib-0011">2009</a>) using the automated1 option, resulting in a final dataset with 26 species and 4524 amino acid positions. An ML phylogenetic tree was inferred with IQ‐TREE (Minh et al. <a href="#jeu70037-bib-0032" class="usa-link" aria-describedby="jeu70037-bib-0032">2020</a>) v1.6 under the LG + C60 + F + R6 model, as chosen by BIC using the ‐TESTNEW option, with 1000 ultrafast bootstraps (‐mwopt flag). This tree was then used as a guide tree for the PMSF approximation (Wang et al. <a href="#jeu70037-bib-0055" class="usa-link" aria-describedby="jeu70037-bib-0055">2018</a>) to calculate 1000 ultra‐fast bootstraps as statistical support.</p>
<p>To check for the presence of a fusion of the peroxisomal multifunctional protein (MFP) and peroxisomal ketoacyl‐CoA thiolase (ACCA1), we used the fused sequence of <em>Pharyngomonas kirbyi</em> as a query in a BLAST search (Altschul et al. <a href="#jeu70037-bib-0002" class="usa-link" aria-describedby="jeu70037-bib-0002">1990</a>) to screen the <em>Multisulcus</em> proteome predicted from the sequenced transcriptome. Transcriptome assembly, predicted proteome, and all phylogenetic information, including sequence datasets and trees, can be found at: <a href="https://figshare.com/projects/Multisulcus_malaysiensis/220201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/projects/Multisulcus_malaysiensis/220201</a>. Transcriptome raw read sequence data have been submitted to GenBank in the form of SRA under accession number PRJNA1188566.</p></section></section><section id="jeu70037-sec-0007"><h2 class="pmc_sec_title">3. Results</h2>
<section id="jeu70037-sec-0008"><h3 class="pmc_sec_title">3.1. Light Microscopy</h3>
<p>We were able to determine some features of the cell morphology and characteristics of movement based on LM observations of living cells. However, due to the unexpected death of the culture, we could not conduct more extensive LM observations. Accordingly, the external morphology of the cells was mainly described based on fixed cells and SEM observations (see below).</p>
<p>
<em>M. malaysiensis</em> nom. prov. strain NY0229 exhibited biflagellated and multiflagellated stages, both equally represented in all our cultures. Biflagellated cells (Figure <a href="#jeu70037-fig-0001" class="usa-link">1A–G</a>) were ovoid, measuring about 7–8 μm in length, with a concave ventral side and a visible ventral groove (Figure <a href="#jeu70037-fig-0001" class="usa-link">1A–C</a>). The posterior part of the cell was elongated and sometimes pointed (Figure <a href="#jeu70037-fig-0001" class="usa-link">1A</a>). The right side of the ventral groove was more pronounced (Figure <a href="#jeu70037-fig-0001" class="usa-link">1B</a>). A single nucleus, with a central nucleolus, was situated near the flagellar bases, in the anterior part of the cell (Figure <a href="#jeu70037-fig-0001" class="usa-link">1D</a>). Some cells had irregular pseudopodia, extending from different parts of the cell (Figure <a href="#jeu70037-fig-0001" class="usa-link">1G</a>). Tetraflagellated cells with two differently originated sets of flagella were ovoid and did not have a clearly visible ventral groove (Figure <a href="#jeu70037-fig-0001" class="usa-link">1H,I</a>). Multiflagellated cells were varied in shape depending on their size; smaller cells (8–12 flagella) were generally rounded (Figure <a href="#jeu70037-fig-0001" class="usa-link">1J–L</a>), while the larger cells were usually irregular in shape (Figure <a href="#jeu70037-fig-0001" class="usa-link">1M–O</a>). Multiflagellated cells contained several nuclei located peripherally, close to the associated flagella (Figure <a href="#jeu70037-fig-0001" class="usa-link">1J–O</a>). All types of cells possessed many large round vacuoles, which were larger in multiflagellated forms (Figure <a href="#jeu70037-fig-0001" class="usa-link">1L,M</a>).</p>
<figure class="fig xbox font-sm" id="jeu70037-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8097/12328842/cacbe0f421fc/JEU-72-e70037-g001.jpg" loading="lazy" id="jats-graphic-1" height="546" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/jeu70037-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Light microscopy images of <em>Multisulcus malaysiensis</em>: (A–G) Biflagellated cells with a prominent ventral groove (A, B, D), with a pointed posterior end (A), and pseudopodia (G); (H, I) Tetraflagellated cells; (J–M) Multiflagellated cells with rounded (J–L) and irregular (M–O) cell bodies. fl, flagella; n, nucleus; ps, pseudopodia; vg, ventral groove. Scale bars: 5 μm.</p></figcaption></figure><p>The cells were always swimming and vibrating, and could be found in various parts of the water column (Video <a href="#jeu70037-supitem-0002" class="usa-link">S1</a>). The direction of movement constantly changed, rarely moving in a straight line or in a circle, sometimes exhibiting slow rotation around the cell axis. The flagella moved quickly and were poorly visible under LM. We did not observe any amoeboid or cyst stages.</p></section><section id="jeu70037-sec-0009"><h3 class="pmc_sec_title">3.2. Scanning Electron Microscopy</h3>
<p>Biflagellated cells had a non‐flattened ovoid body with a concave ventral side and a convex dorsal side (Figure <a href="#jeu70037-fig-0002" class="usa-link">2A–C</a>). They measured 5.2–8.2 μm in length (mean ± SD of 6.7 ± 0.8 μm, <em>n</em> = 38) and 2.8–5.2 μm in width (mean ± SD of 3.6 ± 0.4 μm, <em>n</em> = 38). The average ratio of cell length to cell width was 1.9 (range: 1.3–2.6). The posterior part of the cell was more elongated and pointed, often slightly curved (Figure <a href="#jeu70037-fig-0002" class="usa-link">2B,C</a>). The anterodorsal part of the cell was concave, probably due to the support of microtubules beneath the plasma membrane (Figure <a href="#jeu70037-fig-0002" class="usa-link">2A,B</a>). Observed tetraflagellated cells with a single ventral groove (Figure <a href="#jeu70037-fig-0002" class="usa-link">2D</a>) had a similar cell size to biflagellated cells, measuring 5.8–8.4 μm in length (mean ± SD of 7.3 ± 1.0 μm, <em>n</em> = 6) and 4.2–5.6 μm in width (mean ± SD of 4.9 ± 0.6 μm, <em>n</em> = 6). The average ratio of cell length to cell width for those cells was 1.5 (range: 1.0–1.7). Spherical tetraflagellated cells were also found, featuring two ventral grooves on the opposite sides of the body (Figure <a href="#jeu70037-fig-0002" class="usa-link">2M</a>). Multiflagellated cells had a more flattened shape (Figure <a href="#jeu70037-fig-0002" class="usa-link">2N,O</a>), typically rounded in outline, and measured 9.8–14.8 μm in diameter (mean ± SD of 11.5 ± 1.6 μm, <em>n</em> = 17). The largest observed multiflagellated cells reached up to 42.3 μm and were irregular in shape (not shown).</p>
<figure class="fig xbox font-sm" id="jeu70037-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8097/12328842/a37ee99d3192/JEU-72-e70037-g004.jpg" loading="lazy" id="jats-graphic-3" height="901" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/jeu70037-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Scanning electron microscopy images of <em>Multisulcus malaysiensis</em>: (A–C) “Typical excavate‐like” biflagellated cells; (D) “Typical excavate‐like” tetraflagellated cell; (E–L) Variability of flagellated vanes and projections including proximal projections on the AF (E), typical vane on the PF (F), vane and projections on the opposite sides of the PF (G), merged vane and projections on the same side of the PF (H), typical semicircular distal projections (I), rectangular distal projections (J), small distal projections (K), two distal projections on the opposite sides; (M) Rounded tetraflagellated cell; (N, O) Multiflagellated rounded cells with multiple ventral grooves. a, acronema; af, anterior flagellum; mt, microtubules under plasmalemma; pf, posterior flagellum; vg, ventral groove; white arrow, projections; black arrow, vane on the PF; black arrowhead, putative extrusomes; white asterisk, cytostome. Scale bars: (A–D, M–O) 5 μm, (E–L) 0.5 μm.</p></figcaption></figure><p>The conspicuous ventral groove extended for almost the entire cell length in biflagellated forms (Figure <a href="#jeu70037-fig-0002" class="usa-link">2A–C</a>). The ribbed inner surface of the ventral groove (Figure <a href="#jeu70037-fig-0002" class="usa-link">2B,C</a>) was probably due to the microtubules of the flagellar roots reinforcing it underneath the plasma membrane. The right side of the ventral groove was more pronounced and bent (Figure <a href="#jeu70037-fig-0002" class="usa-link">2A–D,M</a>). The margins of the ventral groove were very prominent and thickened. A small opening, measuring 0.3 ± 0.1 μm in diameter (possibly the cytostome), was sometimes visible in the posterior part of the ventral groove (Figure <a href="#jeu70037-fig-0002" class="usa-link">2A–C</a>). Multiflagellated cells exhibited multiple ventral grooves on the cell surface, with the characteristic ribbed inner surface (Figure <a href="#jeu70037-fig-0002" class="usa-link">2N,O</a>). From what can be discerned from SEM images, these ventral grooves were mainly oriented in one direction (parallel to each other and aligned with the anterior–posterior direction).</p>
<p>The flagella emerged subapically from the most anterior end of the ventral groove. The anterior flagellum (AF) was directed perpendicular to the longitudinal axis of the cell body at the site of its emergence (Figure <a href="#jeu70037-fig-0002" class="usa-link">2A–C</a>), while the posterior flagellum (PF) was usually directed backward and passed within the ventral groove (Figure <a href="#jeu70037-fig-0002" class="usa-link">2A–C,E,F</a>). The AF measured 4.3–21.4 μm in length (mean ± SD of 14.9 ± 3.8 μm, <em>n</em> = 35), making it 2.2 times longer than the cell body. The PF ranged from 7.9 to 29.2 μm in length (mean ± SD of 19.1 ± 4.6 μm, <em>n</em> = 35), 2.8 times longer than the cell body. Both flagella had a noticeable acronema, measuring 0.6–5.8 μm long (mean ± SD of 2.7 ± 0.9 μm, <em>n</em> = 77). Tetraflagellated cells with shorter flagella (4.6–4.8 μm long) were occasionally observed (Figure <a href="#jeu70037-fig-0002" class="usa-link">2D</a>), most likely indicating the process of cell division.</p>
<p>The PF had a prominent ventral vane (directed away from the cell) that measured 1.8–4.0 μm in length and 0.2–0.3 μm in width (<em>n</em> = 22) (Figure <a href="#jeu70037-fig-0002" class="usa-link">2A–C,F–H</a>). In addition to the posterior vane, both flagella bore semicircular or semioval flattened projections at the most distal (Figure <a href="#jeu70037-fig-0002" class="usa-link">2A,I–L</a>) and at the proximal (Figure <a href="#jeu70037-fig-0002" class="usa-link">2E,M</a>) parts of the axonema. The proximal and distal projections were mostly similar in shape, measuring 0.2–1.7 μm in length (mean ± SD of 0.9 ± 0.3 μm, <em>n</em> = 55) and 0.2–0.8 μm in width (<em>n</em> = 55). Length to width ratio was 1.9 (range 1.0–3.6). When located in the proximal part of the PF, such projections were merged with the vane (Figure <a href="#jeu70037-fig-0002" class="usa-link">2B,H</a>), although projections were sometimes found on the opposite side of the PF (Figure <a href="#jeu70037-fig-0002" class="usa-link">2G</a>) and, occasionally, on both sides of the distal part of the AF (Figure <a href="#jeu70037-fig-0002" class="usa-link">2L</a>). Vanes and both types of projections were also present in the tetra‐ and multiflagellated cells (Figure <a href="#jeu70037-fig-0002" class="usa-link">2M–O</a>). Occasionally, projections appeared very small and almost circular in outline (Figure <a href="#jeu70037-fig-0002" class="usa-link">2K</a>). A slight thickening in the proximal part of the AF was rarely observed (Figure <a href="#jeu70037-fig-0002" class="usa-link">2A</a>, white arrow), which probably represented folded projections. Apart from the structures described above, the flagella were naked and did not carry additional elements such as mastigonemes or scales.</p>
<p>The cell surface was usually covered with extrusome‐like structures (Figure <a href="#jeu70037-fig-0002" class="usa-link">2A–C,E,F,M–O</a>) measuring 0.1–0.4 μm in diameter (<em>n</em> = 57). They were very similar in size and shape and found on almost every cell (except in those poorly fixed), including bi‐, tetra‐, and multiflagellated forms. The cell surface also displayed some very small holes (Figure <a href="#jeu70037-fig-0002" class="usa-link">2A,B</a>), most probably fixation artifacts.</p></section><section id="jeu70037-sec-0010"><h3 class="pmc_sec_title">3.3. Transmission Electron Microscopy</h3>
<p>Thin sections of <em>M. malaysiensis</em> revealed some characteristic features, despite suboptimal fixation. The nucleus (1.14–2.26 μm in diameter, <em>n</em> = 24) usually occupied an anterior position in the cell, closely associated with the flagella, and appeared generally round or irregular (Figure <a href="#jeu70037-fig-0003" class="usa-link">3A,B</a>), likely due to fixation artifacts. A large single round nucleolus was situated in the center of the nucleus and had a diameter of at least 0.39 μm (<em>n</em> = 22). In some cells, the nucleolus had a more electron‐dense rounded area located on its periphery (Figure <a href="#jeu70037-fig-0003" class="usa-link">3A,B</a> indicated by white asterisks). This region was 0.22–0.41 μm in diameter (<em>n</em> = 12).</p>
<figure class="fig xbox font-sm" id="jeu70037-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8097/12328842/430d411c94fc/JEU-72-e70037-g002.jpg" loading="lazy" id="jats-graphic-5" height="901" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/jeu70037-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Transmission electron microscopy images of <em>Multisulcus malaysiensis</em>: (A) transverse section through the anterior part of the cell with the apical nucleus and central nucleolus, note the presence of a densely stained eccentrical area in the nucleolus indicated by a white asterisk; (B) longitudinal section of the whole cell showing apical nucleus and numerous mitochondria concentrated near the nucleus, the insets show magnified parts of the cell with dense bodies (putative extrusomes); (C) transverse section through the middle part of the cell showing numerous food vacuoles and the developed ventral groove; (D) magnified part of the cell from (A), showing the anterior part of the ventral groove reinforced with microtubules; (E) magnified part of the cell from (C) with microtubular rows supporting the margins of the ventral groove, and rarely spaced microtubules (indicated by black arrowheads) underlying the floor of the ventral groove, the inset shows a magnified part of the groove margin with double rows of microtubules (black asterisks indicating beginning and end of the external row); (F–H) orientation of basal bodies and flagella; (I) proximal part of microtubular root with Z‐shape, note cytoplasmic microtubules probably directed to support the dorsal side of the cell, black arrowheads pointing out microtubules; (J) longitudinal section of the anterior part of the cell with posteriorly directed microtubules; (K) mitochondria with discoidal cristae and dense body (putative extrusome); (L) rounded “empty” vesicles; (M) probable pinocytosis; (N) anterior part of the cell with numerous mitochondria surrounded by rough ER. af, anterior flagellum; db, dense body (putative extrusome); fv, food vacuole; mt—mitochondria; n, nucleus; nu, nucleolus; pf, posterior flagellum; v, vacuole. Scale bars: (A–C) 2 μm, (D–N) 0.5 μm.</p></figcaption></figure><p>We did not observe a conspicuous Golgi body. Numerous elongated mitochondria were mostly located in the anterior part of the cell (Figure <a href="#jeu70037-fig-0003" class="usa-link">3A–C,E,J,K,N</a>), surrounded by rough endoplasmic reticulum (Figure <a href="#jeu70037-fig-0003" class="usa-link">3E,K,N</a>). Mitochondrial cristae were discoidal in shape, rather than flattened (Figure <a href="#jeu70037-fig-0003" class="usa-link">3E,K</a>). Rarely, round electron‐dense bodies (putative extrusomes) were found in the vicinity of the plasma membrane (Figure <a href="#jeu70037-fig-0003" class="usa-link">3B,J,K</a>). These bodies had a paracrystalline appearance and a single‐layered binding membrane (Figure <a href="#jeu70037-fig-0003" class="usa-link">3B</a>, insertions). Such electron‐dense bodies were 0.11–0.4 μm in diameter (<em>n</em> = 62). One of them was always situated near the flagellar insertion (Figure <a href="#jeu70037-fig-0003" class="usa-link">3J</a>). Food vacuoles containing partially digested bacteria were located in the posterior part of the cell, often along the ventral groove (Figure <a href="#jeu70037-fig-0003" class="usa-link">3A–C</a>). Round and elongated electron‐lucent vesicles of variable size (between 0.18 and 0.91 μm, <em>n</em> = 53) occupied a peripheral position in the cell (Figure <a href="#jeu70037-fig-0003" class="usa-link">3B,G,J,L</a>). Smaller vesicles were generally round, while larger vesicles were usually more elongated.</p>
<p>In the biflagellate cells, two kinetosomes appeared oriented nearly orthogonal to each other (Figure <a href="#jeu70037-fig-0003" class="usa-link">3F–H</a>). We could not observe the kinetosomal structure, but we found microtubules of the flagellar apparatus reinforcing the ventral groove from the inside. We also observed a Z‐like orientation of microtubules in the proximal part of the cell (Figure <a href="#jeu70037-fig-0003" class="usa-link">3I</a>). The middle part was formed by 11 microtubules in a flat row, with three additional microtubules from each side. There were at least 14 microtubules located on one side of the groove (Figure <a href="#jeu70037-fig-0003" class="usa-link">3D,E</a>), and two closely adjusted parallel rows of microtubules on another side of the groove (Figure <a href="#jeu70037-fig-0003" class="usa-link">3E</a>, insertion): seven microtubules located closely to the plasma membrane, and about 20 microtubules located more internally. In addition, there were several rarely spaced microtubules lining the floor of the ventral groove (Figure <a href="#jeu70037-fig-0003" class="usa-link">3E</a>, indicated by black arrowheads). Thin sections also showed the vane on the PF (Figure <a href="#jeu70037-fig-0003" class="usa-link">3E</a>), but fine details were not discernible.</p></section><section id="jeu70037-sec-0011"><h3 class="pmc_sec_title">3.4. Molecular Phylogenetic Analyses and Reconstruction of the 18S rRNA Secondary Structure</h3>
<p>We amplified and Sanger sequenced the 18S rRNA gene of strain NY0229. The length of the resulting sequence was 1451 bp. Preliminary BLAST (Altschul et al. <a href="#jeu70037-bib-0002" class="usa-link" aria-describedby="jeu70037-bib-0002">1990</a>) searches against the nonredundant (nr) GenBank database did not help to determine the precise phylogenetic affiliation of the new strain. We therefore prepared a multiple 18S rRNA gene sequence alignment containing a wide range of eukaryotic lineages; a preliminary phylogenetic analysis revealed that the new strain was related to Heterolobosea (not shown). Based on this result, we carried out a more targeted maximum likelihood (ML) and Bayesian Inference (BI) phylogenetic analyses using a comprehensive set of heterolobosean 18S rRNA gene sequences, including sequences representing all known lineages of two subphyla, Pharyngomonada and Tetramitia, based on Hanousková et al. (<a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>) and Pánek et al. (<a href="#jeu70037-bib-0041" class="usa-link" aria-describedby="jeu70037-bib-0041">2025</a>), and rooted using <em>Tsukubamonas globosa</em> and jakobid sequences as an outgroup. For these phylogenetic analyses, we used the complete 18S rRNA gene sequence (2531 bp) extracted from transcriptome data (see below). Both phylogenetic analyses showed that <em>M. malaysiensis</em> represents a distinct deep‐branching lineage within the Tetramitia, sister to both Eutetramitia and Selenaionea, with a high statistical support (Figure <a href="#jeu70037-fig-0004" class="usa-link">4</a>).</p>
<figure class="fig xbox font-sm" id="jeu70037-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8097/12328842/6f0e04ab2963/JEU-72-e70037-g005.jpg" loading="lazy" id="jats-graphic-7" height="863" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/jeu70037-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phylogenetic tree generated from Bayesian analysis of 18S rRNA gene sequences showing the phylogenetic position of <em>Multisulcus malaysiensis</em> strain NY0229 within Heterolobosea. The tree was reconstructed using an updated version of the dataset from Hanousková et al. (<a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>) for a total of 53 species and 1436 nucleotide positions. Bayesian posterior probabilities (GTR + I + GAMMA4 model) and ML (GTR+ F + I + R5 model) bootstrap values are indicated at branches; black dots indicate values of BPP = 1.00 and ML bootstrap = 100%; dt: different topology.</p></figcaption></figure><p>Interestingly, we retrieved the monophyly of Selenaionea (Selenaionida and Neovahlkampfiida), although with a low statistical support. This topology was previously obtained by multigene phylogenetic analysis with full statistical support (Pánek et al. <a href="#jeu70037-bib-0041" class="usa-link" aria-describedby="jeu70037-bib-0041">2025</a>). Another unexpected result was the position of Creneidae, branching within Tetramitia VI, notably as a sister to Psalteriomonadidae (<em>Psalteriomonas</em>, <em>Harpagon</em>, <em>Pseudoharpagon</em>, <em>Monopylocystis</em>). This topology also had a low statistical support. In all other respects, the topology that we obtained within Tetramitia was congruent with Hanousková et al. (<a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>).</p>
<p>To verify this main result using more data, we isolated total RNA from 
<em>M. malaysiensis</em>
 and sequenced its transcriptome to carry out a multigene ML phylogenetic analysis. The resulting multigene tree was based on 350 conserved proteins and included 37 species. It robustly placed the new strain within Tetramitia with 100% support (Figure <a href="#jeu70037-fig-0005" class="usa-link">5A</a>) as an individual branch sister to the Eutetramitia clade (containing 
<em>Naegleria gruberi</em>
, <em>Percolomonas cosmopolitus</em>, and <em>Sawyeria marylandensis</em>) with 100% support. <em>P. kirbyi</em> branched as the deepest branching heterolobosean lineage with maximal support.</p>
<figure class="fig xbox font-sm" id="jeu70037-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8097/12328842/5adc7d8d32aa/JEU-72-e70037-g003.jpg" loading="lazy" id="jats-graphic-9" height="705" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/jeu70037-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) ML phylogenomic tree reconstructed from a dataset of 350 conserved proteins. The tree was reconstructed using 37 species and 112,451 amino acid positions with the LG + C60 + G model using the PMSF approximation and 100 non‐parametric bootstraps as statistical support. (B) ML phylogenomic tree based on six conserved mitochondrion‐encoded proteins from Galindo et al. (<a href="#jeu70037-bib-0017" class="usa-link" aria-describedby="jeu70037-bib-0017">2023</a>). The tree was reconstructed using 26 species and 4524 amino acid positions under the LG + C60 + F + R6 model using the PMSF approximation and 1000 ultra‐fast bootstrap replicates as statistical support.</p></figcaption></figure><p>Finally, we analyzed mitochondrial proteins. An ML phylogenetic analysis based on 14 conserved mitochondrion‐encoded proteins from Galindo et al. (<a href="#jeu70037-bib-0017" class="usa-link" aria-describedby="jeu70037-bib-0017">2023</a>) also placed 
<em>M. malaysiensis</em>
 sister to Eutetramitia (containing <em>Stachyamoeba lipophora</em>, <em>Acrasis kona</em>, <em>Pleurostomum flabellatum</em>, 
<em>N. gruberi</em>
, and 
<em>N. fowleri</em>
) with strong support (98%), whereas Pharyngomonada (containing <em>P. kirbyi</em> and Heterolobosea sp. BB2) emerged as the first heterolobosean branch (Figure <a href="#jeu70037-fig-0005" class="usa-link">5B</a>).</p>
<p>In agreement with its phylogenetic relation with Tetramitia, we found the characteristic helix 17_1 of the 18S rRNA sequence, which is a synapomorphy of Tetramitia (Harding et al. <a href="#jeu70037-bib-0021" class="usa-link" aria-describedby="jeu70037-bib-0021">2013</a>; Nikolaev et al. <a href="#jeu70037-bib-0033" class="usa-link" aria-describedby="jeu70037-bib-0033">2004</a>), in 
<em>M. malaysiensis</em>
 (Figure <a href="#jeu70037-supitem-0001" class="usa-link">S1</a>). This helix is missing in Pharyngomonada. To check if the MFP‐ACAA1 gene fusion, which was recently found to be an autapomorphy of Pharyngomonada (Pánek et al. <a href="#jeu70037-bib-0041" class="usa-link" aria-describedby="jeu70037-bib-0041">2025</a>), was present, we searched it in the <em>Multisulcus</em> proteome predicted from the sequenced transcriptome using the fused sequence of <em>P. kirbyi</em> as query in a BLAST search. We found MFP and ACAA1 as individual proteins encoded by two different transcripts. Thus, 
<em>M. malaysiensis</em>
 lacks this characteristic autapomorphy of Pharyngomonada. Altogether, our molecular phylogenetic analyses support the phylogenetic position of 
<em>M. malaysiensis</em>
 as a distinct deep‐branching lineage at the class level within Tetramitia.</p></section></section><section id="jeu70037-sec-0012"><h2 class="pmc_sec_title">4. Discussion</h2>
<section id="jeu70037-sec-0013"><h3 class="pmc_sec_title">4.1. The Ventral Groove, Flagellar Vane, and Projections</h3>
<p>
<em>M. malaysiensis</em> nom. prov. has biflagellated and multiflagellated stages in the life cycle. Remarkably, the biflagellated forms are superficially reminiscent of jakobids, the deepest branching lineage of Discoba, and other typical excavates. This is most apparent at the level of the highly developed deep ventral groove, which extends along the entire cell length, and a probable vane on the PF. Heteroloboseans lost the vane, and some flagellated lineages have even completely lost the ventral groove, for example, <em>Dactylomonas</em> spp., <em>Euplaesiobystra hypersalinica</em>, <em>Naegleria</em> spp., <em>Willaertia magna</em>, <em>Stephanopogon</em> spp., and <em>C. carolina</em> (Hanousková et al. <a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>; Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>). As for other discobid lineages, they either lost it completely, as in Euglenozoa, or highly reduced it to a shallow temporal groove, as in <em>T. globosa</em> (Gibson <a href="#jeu70037-bib-0018" class="usa-link" aria-describedby="jeu70037-bib-0018">2017</a>; Leander et al. <a href="#jeu70037-bib-0029" class="usa-link" aria-describedby="jeu70037-bib-0029">2017</a>; Yabuki et al. <a href="#jeu70037-bib-0058" class="usa-link" aria-describedby="jeu70037-bib-0058">2011</a>).</p>
<p>The main and original function of the ventral groove is facilitating suspension feeding by creating a water flow thanks to the PF undulating within it (Suzuki‐Tellier et al. <a href="#jeu70037-bib-0053" class="usa-link" aria-describedby="jeu70037-bib-0053">2024</a>). Moreover, the ventral groove is most efficient when combined with a vane‐bearing PF. It has been shown that the energy efficiency of suspension feeding is significantly reduced both by beating a vaned flagellum outside the ventral groove or by beating a naked flagellum inside the ventral groove (Suzuki‐Tellier et al. <a href="#jeu70037-bib-0053" class="usa-link" aria-describedby="jeu70037-bib-0053">2024</a>). Therefore, both structures (the ventral groove and the PF vane) usually co‐occur because of their synergetic function, leading to increased feeding efficacy.</p>
<p>Many typical excavates remain attached to the substrate during suspension feeding. This is achieved by the attachment of the AF, the distal part of the PF, the posterior end of the cell, or even with the stalk of loricated jakobids (Suzuki‐Tellier et al. <a href="#jeu70037-bib-0053" class="usa-link" aria-describedby="jeu70037-bib-0053">2024</a>). At the same time, the swimming efficiency of typical excavates is low. It is interesting to note that our strain NY0229 did not attach to the substrate during feeding and constantly swam, despite having a fully developed ventral groove. In typical excavates, the ventral groove is usually accompanied by one to three vanes on the PF. These PF vanes were previously thought to be lost in heteroloboseans (Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>). We therefore describe here the first observation of a PF vane in Heterolobosea. The presence of two oppositely directed flagellar vanes is the most common configuration in excavates and is considered to be ancestral (Simpson <a href="#jeu70037-bib-0051" class="usa-link" aria-describedby="jeu70037-bib-0051">2003</a>). The ventral vane (directed away from the cell) is the most prominent and better developed (wider and longer) in most excavates, including most “<em>Carpediemonas</em>‐like organisms,” the deepest jakobid branch Ophirinina, and some species in the deep‐branching Preaxostyla lineage Paratrimastigidae (Galindo et al. <a href="#jeu70037-bib-0017" class="usa-link" aria-describedby="jeu70037-bib-0017">2023</a>; Hampl <a href="#jeu70037-bib-0019" class="usa-link" aria-describedby="jeu70037-bib-0019">2017</a>; Kulda <a href="#jeu70037-bib-0026" class="usa-link" aria-describedby="jeu70037-bib-0026">2017</a>; Yabuki et al. <a href="#jeu70037-bib-0057" class="usa-link" aria-describedby="jeu70037-bib-0057">2018</a>). By contrast, the core jakobids have a single dorsal vane. In each excavate group, the PF vane is supported internally by striated lamella, which strongly suggests the homology of this structure (Simpson <a href="#jeu70037-bib-0051" class="usa-link" aria-describedby="jeu70037-bib-0051">2003</a>). We did not obtain good transverse sections of the <em>Multisulcus</em> vane to see if it also contains striated lamella. Obtaining this data in future studies would help to better understand its origin and evolution.</p>
<p>In addition to the PF vane, <em>Multisulcus</em> had projections located proximally and distally on both flagella, which is a unique feature of this species. We observed these projections on most flagella, although some were very small and even absent on recently divided cells with short flagella (Figure <a href="#jeu70037-fig-0002" class="usa-link">2D</a>). The proximal projection on the PF could be found on the same side as the vane, and thus both structures merged, or alternatively, the proximal projection and the vane could also be on opposite sides. This indicates that the location of the projections is not strictly determined. Similar projections, but only on the PF proximal part, were found in <em>Dactylomonas</em> (Hanousková et al. <a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>). They are supported by striated fibers similar to the striated lamella of typical excavates. The projections on <em>Dactylomonas venusta</em> are similar in size (0.5 μm wide) to the projections we observe (0.5 ± 0.15 μm wide), while <em>Dactylomonas crassa</em> has much wider projections (1–1.5 μm wide).</p>
<p>It is remarkable that very similar flagellar projections have been identified in three distantly related lineages of Heterolobosea, but are unknown outside this phylum. These projections probably have a common origin, although this hypothesis will need further testing. Since we observe both projections and PF vanes in <em>Multisulcus</em>, the idea that the projections are reduced excavate vanes (Hanousková et al. <a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>) seems unlikely. Most probably, these structures evolved independently and perform different functions.</p></section><section id="jeu70037-sec-0014"><h3 class="pmc_sec_title">4.2. Multiflagellated Life Stages and Evolution of a Karyomastigont</h3>
<p>A unique character of <em>Multisulcus</em> is the presence of multiflagellated stages. We observed multiple well‐developed feeding grooves on these multiflagellated forms, each groove being associated with its own pair of flagella. Moreover, in LM, we observed many nuclei, each located near the associated pair of flagella. The flagellar apparatus and nucleus in <em>Multisulcus</em> seem closely connected to one another, thus representing a single morpho‐functional complex, a karyomastigont. This assumption needs to be confirmed by transmission electron microscopy of thin sections of multiflagellated forms.</p>
<p>Karyomastigonts are only known in one heterolobosean species, <em>Psalteriomonas lanterna</em>, but are widespread in many metamonads, including oxymonads, trichomonads, cristamonads, and diplomonads (Adam <a href="#jeu70037-bib-0001" class="usa-link" aria-describedby="jeu70037-bib-0001">2017</a>; Broers et al. <a href="#jeu70037-bib-0007" class="usa-link" aria-describedby="jeu70037-bib-0007">1990</a>; Čepička et al. <a href="#jeu70037-bib-0012" class="usa-link" aria-describedby="jeu70037-bib-0012">2017</a>; Hampl <a href="#jeu70037-bib-0019" class="usa-link" aria-describedby="jeu70037-bib-0019">2017</a>). The evolution of these groups often involved the polymerization of a karyomastigont, sometimes reaching tens and even hundreds in a single cell (Čepička et al. <a href="#jeu70037-bib-0012" class="usa-link" aria-describedby="jeu70037-bib-0012">2017</a>). Diplomonads represent the simplest version of karyomastigont polymerization, a duplication. Interestingly, <em>Trepomonas agilis</em> has two narrow lateral grooves, one on each cell side (Eyden and Vickerman <a href="#jeu70037-bib-0016" class="usa-link" aria-describedby="jeu70037-bib-0016">1975</a>). Two ventral grooves, each associated with its own pair of flagella, have also been observed in the dividing cells of the otherwise normally biflagellated “<em>Carpediemonas</em>‐like organism” <em>Kipferlia bialata</em> (Yubuki et al. <a href="#jeu70037-bib-0061" class="usa-link" aria-describedby="jeu70037-bib-0061">2013</a>). Spheroidal “monster” cells, up to 50 μm in diameter, with several nuclei and flagellar pairs, have been observed in <em>Malawimonas jakobiformis</em>, an organism with a typical excavate morphology (O'Kelly and Nerad <a href="#jeu70037-bib-0036" class="usa-link" aria-describedby="jeu70037-bib-0036">1999</a>).</p>
<p>
<em>P. lanterna</em> cells normally have four karyomastigonts, each containing a single nucleus, four kinetosomes, four flagella, four sets of associated microtubular roots, and four fully developed feeding grooves, one on each side of a fourfolded cell (Broers et al. <a href="#jeu70037-bib-0007" class="usa-link" aria-describedby="jeu70037-bib-0007">1990</a>). This form persists during ontogenesis, unlike in <em>Multisulcus</em>, for which multiflagellated forms are as common as biflagellated ones.</p>
<p>Unfortunately, the life cycle of <em>Multisulcus</em> could not be studied in detail due to the culture death. Therefore, changes undergone by multiflagellated forms to return to the biflagellated state remain to be determined. Since we always observed both types of cells in culture, we assume that this process takes place continuously. We hypothesize that the multinucleated forms undergo cellularization, a coordinated cytokinesis that results in cleavage into individual biflagellated cells, similar to what is observed in ichthyosporeans and some other eukaryotes with uni‐ and multinucleated life cycle stages (Lamża <a href="#jeu70037-bib-0027" class="usa-link" aria-describedby="jeu70037-bib-0027">2023</a>; Olivetta and Dudin <a href="#jeu70037-bib-0037" class="usa-link" aria-describedby="jeu70037-bib-0037">2023</a>).</p></section><section id="jeu70037-sec-0015"><h3 class="pmc_sec_title">4.3. Evolution of <em>Multisulcus</em> Kinetid and Flagellar Apparatus</h3>
<p>
<em>M. malaysiensis</em> has two orthogonally oriented kinetosomes. The flagellar apparatus of most heteroloboseans consists of two kinetosomes or four kinetosomes arranged in two pairs, and has a peculiar doubled structure (Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>), in which each of the two kinetosomes or two pairs of kinetosomes is associated with broadly similar sets of microtubular and non‐microtubular elements (Brugerolle and Simpson <a href="#jeu70037-bib-0009" class="usa-link" aria-describedby="jeu70037-bib-0009">2004</a>; Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>). Moreover, the deepest heterolobosean branch, <em>Pharyngomonas</em>, displays the least developed “double bikont” flagellar apparatus: R2 and R2′ roots are identical at the base and have the same set of non‐microtubular fibers, but R2′ microtubules are much shorter and do not directly support the groove (Park and Simpson <a href="#jeu70037-bib-0043" class="usa-link" aria-describedby="jeu70037-bib-0043">2011</a>). Thus, at some point during the evolution of Heterolobosea, a duplication of the flagellar apparatus might have occurred, probably due to some cell division impairment. Curiously, the <em>Multisulcus</em>' flagellar apparatus divides repeatedly during the life cycle, which can be seen as evolutionarily associated with, or a prerequisite of the “double bikont” flagellar apparatus of Heterolobosea.</p>
<p>The orthogonal arrangement of kinetosomes is another atypical feature of heteroloboseans, as most of them have a parallel kinetosome orientation. An orthogonal orientation of kinetosomes was found only in the deepest‐branching flagellated lineages, <em>Pharyngomonas</em> and <em>Dactylomonas</em> (Hanousková et al. <a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>; Park and Simpson <a href="#jeu70037-bib-0043" class="usa-link" aria-describedby="jeu70037-bib-0043">2011</a>), but it is the most common arrangement among excavates, including the deepest branching discobid lineages jakobids and <em>T. globosa</em> (Galindo et al. <a href="#jeu70037-bib-0017" class="usa-link" aria-describedby="jeu70037-bib-0017">2023</a>; Yabuki et al. <a href="#jeu70037-bib-0058" class="usa-link" aria-describedby="jeu70037-bib-0058">2011</a>, <a href="#jeu70037-bib-0057" class="usa-link" aria-describedby="jeu70037-bib-0057">2018</a>).</p>
<p>Unfortunately, we were unable to obtain sufficient data to make comparisons for other elements of the flagellar apparatus. Future studies should help clarify many intriguing details. In particular, it would be interesting to know whether the left root R1 has been retained by <em>Multisulcus</em>; this root is present in <em>Pharyngomonas</em> and <em>Dactylomonas</em>, but was lost in Eutetramitia (Brugerolle and Simpson <a href="#jeu70037-bib-0009" class="usa-link" aria-describedby="jeu70037-bib-0009">2004</a>; Hanousková et al. <a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>; Park and Simpson <a href="#jeu70037-bib-0043" class="usa-link" aria-describedby="jeu70037-bib-0043">2011</a>; Simpson <a href="#jeu70037-bib-0051" class="usa-link" aria-describedby="jeu70037-bib-0051">2003</a>). In addition to the R1 root, <em>Pharyngomonas</em>, like typical excavates, retains a multilayered C fiber. The R1 root (and possibly the C fiber) is likely to be found in <em>Multisulcus</em>, since all three groups (<em>Multisulcus</em>, <em>Pharyngomonas</em>, and <em>Dactylomonas</em>) share many ancestral characteristics.</p></section><section id="jeu70037-sec-0016"><h3 class="pmc_sec_title">4.4. Comparison With Related Groups</h3>
<p>According to 18S rRNA and multigene phylogenetic analyses, <em>M. malaysiensis</em> occupies a distinct deep‐branching position within the phylogeny of Heterolobosea, sister to the known Tetramitia classes—Eutetramitia and Senenaionea (Figures <a href="#jeu70037-fig-0004" class="usa-link">4</a> and <a href="#jeu70037-fig-0005" class="usa-link">5</a>). Morphological studies also show features typical of heteroloboseans, such as the absence of discrete Golgi dictyosomes, discoid mitochondrial cristae, ventral groove, microtubular corset, and mitochondria surrounded by endoplasmic reticulum.</p>
<p>
<em>Multisulcus</em> shares many features with the deepest branching heterolobosean lineage, the halophilic or halotolerant Pharyngomonada (Park and Simpson <a href="#jeu70037-bib-0043" class="usa-link" aria-describedby="jeu70037-bib-0043">2011</a>, <a href="#jeu70037-bib-0044" class="usa-link" aria-describedby="jeu70037-bib-0044">2016</a>). Given the deep‐branching phylogenetic position of <em>Multisulcus</em> within Tetramitia, these characters are of evolutionary interest and can be considered ancestral. <em>Pharyngomonas</em>, like <em>Multisulcus</em>, has a ventral groove, orthogonally oriented kinetosomes, and discoid mitochondrial cristae. However, <em>Multisulcus</em> lacks the MFP‐ACAA1 gene fusion, recently found in members of Pharyngomonada (Pánek et al. <a href="#jeu70037-bib-0041" class="usa-link" aria-describedby="jeu70037-bib-0041">2025</a>), but possesses the helix 17_1 typical of the 18S rRNA gene sequences of Tetramitia (Figure <a href="#jeu70037-supitem-0001" class="usa-link">S1</a>). <em>Pharyngomonas</em> mitochondria are not surrounded by ER, although this is also observed in some representatives of Tetramitia, including <em>P. cosmopolitus</em>, <em>Stephanopogon</em> spp., <em>P. flabellatum</em>, and <em>Psalteriomonas vulgaris</em> (Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>). <em>Pharyngomonas</em> possesses a cytostome, which may also be present in <em>Multisulcus</em> (Figure <a href="#jeu70037-fig-0002" class="usa-link">2C</a>, asterisk). <em>Pharyngomonas</em> has flagellated cells with four flagella and four kinetosomes arranged in pairs, and it has amoeboid and cyst stages in its life cycle (Park and Simpson <a href="#jeu70037-bib-0043" class="usa-link" aria-describedby="jeu70037-bib-0043">2011</a>). Pharyngomonada has neither a flagellar vane nor projections (Park and Simpson <a href="#jeu70037-bib-0043" class="usa-link" aria-describedby="jeu70037-bib-0043">2011</a>).</p>
<p>Common features of <em>Multisulcus</em> and Tetramitia include the mitochondria (or mitochondria‐like organelles in anaerobic species) surrounded by endoplasmic reticulum, a specific helix 17_1 in the secondary structure of the 18S rRNA molecule, and the presence of processes on flagella in <em>Dactylomonas</em> and <em>P. lanterna</em> (Broers et al. <a href="#jeu70037-bib-0007" class="usa-link" aria-describedby="jeu70037-bib-0007">1990</a>; Hanousková et al. <a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>; Nikolaev et al. <a href="#jeu70037-bib-0033" class="usa-link" aria-describedby="jeu70037-bib-0033">2004</a>; Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>). However, Tetramitia lost the flagellar vane, and only two lineages preserved the proximal flagellar projections, <em>Dactylomonas</em> and <em>Psalteriomonas</em> (Broers et al. <a href="#jeu70037-bib-0007" class="usa-link" aria-describedby="jeu70037-bib-0007">1990</a>; Hanousková et al. <a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>), while distal projections are presumably unique for <em>Multisulcus</em>. The ventral groove of Tetramitia is less developed and even lost in some lineages (Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>).</p>
<p>Within the Tetramitia, <em>Multisulcus</em> has more similarities with the deepest‐branching lineage, Selenaionea, than with Eutetramitia. The latter have a “double bikont” structure of the flagellar apparatus with two sets of flagellar roots, and kinetosomes mostly arranged in parallel (Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>). <em>Multisulcus</em> and flagellated representatives of Selenaionea, <em>S. koniopes</em> and <em>Dactylomonas</em> spp., have similar bikinetosomal kinetids with orthogonal kinetosomal orientation. However, <em>Dactylomonas</em> completely lost the ventral groove and possesses an anterior rostrum supported by microtubules of the flagellar apparatus (Hanousková et al. <a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>). Another difference with <em>Dactylomonas</em> is its anaerobic or microaerophilic lifestyle and acristate mitochondria (Hanousková et al. <a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>).</p>
<p>Distinctive unique features of <em>M. malaysiensis</em> are the combination of a vane on the PF with distal and proximal projections on all flagella, and the presence of multinucleated stages (coenocyte) with multiple ventral grooves, which result from multiple polymerizations of the whole karyomastigont. By contrast, the polymerization of the karyomastigont in <em>P. lanterna</em> was maintained at the state of tetrakaryomastigont, whereas in diplomonads it was maintained at the state of bikaryomastigont.</p></section><section id="jeu70037-sec-0017"><h3 class="pmc_sec_title">4.5. Amoeboid Stages in the Life Cycles of <em>Multisulcus</em> and Heterolobosea</h3>
<p>Many heteroloboseans possess a three‐phase asexual life cycle that includes amoeboid, flagellated, and cyst stages. The amoeboid stage is considered to be the central, as flagellates and cysts are usually formed from amoebae (Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>). Although we observed in <em>Multisulcus</em> the formation of long and morphologically variable, but never eruptive, pseudopodia in flagellated cells, we did not observe any fully amoeboid stage under our culture conditions.</p>
<p>The flagellated or amoeboid forms of many species of Heterolobosea were only observed after several years of cultivation or under certain culture conditions. For instance, <em>P. kirbyi</em> and <em>Pseudoharpagon</em> spp. were considered exclusively flagellates for a long time until amoeboid forms were observed (Harding et al. <a href="#jeu70037-bib-0021" class="usa-link" aria-describedby="jeu70037-bib-0021">2013</a>; Pánek, Ptáčková, and Čepička <a href="#jeu70037-bib-0038" class="usa-link" aria-describedby="jeu70037-bib-0038">2014</a>). On the contrary, the amoeboflagellated <em>P. lanterna</em>, <em>Heteramoeba clara</em>, <em>W. magna</em>, and some <em>Tetramitus</em> species lost the ability to form flagellated stages after several years of cultivation (Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>). The type of bacterial prey seemed to drive the dominance of the flagellate or amoeboid stages in <em>Oramoeba fumarolia</em> cultures (De Jonckheere, Baumgartner, et al. <a href="#jeu70037-bib-0014" class="usa-link" aria-describedby="jeu70037-bib-0014">2011</a>). Likewise, adding a small amount of oxygen to an anaerobic culture of tetranucleated <em>P. lanterna</em> flagellates led to the formation of amoeboid uninucleated stages (Broers et al. <a href="#jeu70037-bib-0007" class="usa-link" aria-describedby="jeu70037-bib-0007">1990</a>). Although <em>C. carolina</em> predominantly exhibits uniflagellated amoeboid phenotypes, extremely rare elongated, fast‐swimming, multiflagellated cells have also been observed (Pánek, Simpson, et al. <a href="#jeu70037-bib-0039" class="usa-link" aria-describedby="jeu70037-bib-0039">2014</a>).</p>
<p>Although an amoeboid stage was probably lost in some heterolobosean lineages, it has been observed in each of the main Tetramitia clades (Hanousková et al. <a href="#jeu70037-bib-0020" class="usa-link" aria-describedby="jeu70037-bib-0020">2019</a>; Pánek et al. <a href="#jeu70037-bib-0040" class="usa-link" aria-describedby="jeu70037-bib-0040">2017</a>) and in the deepest‐branching heterolobosean lineage, Pharyngomonada (Park and Simpson <a href="#jeu70037-bib-0043" class="usa-link" aria-describedby="jeu70037-bib-0043">2011</a>, <a href="#jeu70037-bib-0044" class="usa-link" aria-describedby="jeu70037-bib-0044">2016</a>). This supports that the amoeboid‐to‐flagellate transformation in the life cycle evolved early in the Heterolobosea, probably before the radiation of Pharyngomonada and Tetramitia (Harding et al. <a href="#jeu70037-bib-0021" class="usa-link" aria-describedby="jeu70037-bib-0021">2013</a>; Park and Simpson <a href="#jeu70037-bib-0044" class="usa-link" aria-describedby="jeu70037-bib-0044">2016</a>). The isolation of new strains and experiments on different culture conditions (media composition, temperature, pH, etc.) will help to determine whether amoeboid stages can be induced under certain conditions or are truly absent in <em>Multisulcus</em>.</p></section><section id="jeu70037-sec-0018"><h3 class="pmc_sec_title">4.6. Taxonomic Summary</h3>
<p>Taxonomy: Eukaryota; Discoba; Heterolobosea; Tetramitia.</p>
<p>
<em>Multisulcus malaysiensis</em> Prokina et al., nom. prov.</p>
<p>Etymology: Genus name after life stage with multiple ventral grooves. Species name after the locality of the type strain, Malaysia.</p>
<p>Type locality: freshwaters of the palm oil factory Seri Ulu Langat Palm Oil Mill Sdn Bhd, Batu 34, Jalan Banting, Dengkil, 43800, Dengkil, Selangor, 43800, Malaysia.</p>
<p>Gene sequence: The complete 18S rRNA gene sequence of the strain NY0229 has the GenBank Accession Number <a href="https://www.ncbi.nlm.nih.gov/nuccore/PQ627902" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PQ627902</a>.</p></section></section><section id="sec19"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="jeu70037-supitem-0001"><div class="caption p"><p>
<strong>Figure S1:</strong> (a) Inferred secondary structures of the 18S rRNA region containing the helices 17 and 18 in <em>Pharyngomonas kirby</em>, <em>Psalteriomonas lanterna</em>, and <em>Multisulcus malaysiensis</em>. Notice the additional helix 17_1 present in <em>P. lanterna</em> and 
<em>M. malaysiensis</em>
. (b) Multiple sequence alignment of the 18S rRNA region containing the helices 17 and 18. Uppercase positions are those shown in panel a. Some additional conserved positions are shown on lowercase. The green positions in <em>P. lanterna</em> and 
<em>M. malaysiensis</em>
 sequences correspond to the additional helix 17_1.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328842/bin/JEU-72-e70037-s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">JEU-72-e70037-s002.pdf</a><sup> (597KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="jeu70037-supitem-0002"><div class="caption p"><p>
<strong>Video S1:</strong> Live biflagellated cells of <em>Multisulcus malaysiensis</em> strain NY0229. DIC × 100 oil immersion.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328842/bin/JEU-72-e70037-s001.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (11MB, mp4) </sup>
</div></div></section></section><section id="jeu70037-sec-0019" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>D.M. and P.L.‐G. were supported by grants from the European Research Council (ERC Advanced grants 787904 and 101141745). L.J.G. was funded by the Ramón y Cajal Programme (Grant RYC2022‐035282‐I, funded by MCIU/AEI/10.13039/501100011033 and FSE+). The TEM work has benefited from the Imagerie‐Gif core facility supported by l'Agence Nationale de la Recherche (ANR‐10‐INBS‐04/FranceBioImaging; ANR‐11‐IDEX‐0003‐02/Saclay Plant Sciences). The sampling trip in Malaysia was supported by the “Tree of Life” research project of the University of Tsukuba, Japan. Permit for sample collection was obtained from the Research Promotion and Co‐ordination Committee, Ministry of Economic Affairs, Malaysia [permit no. UPE 40/200/19/3548(11)].</p></section><section id="notes1"><p>


Prokina, K. I.
, Yubuki N., Galindo L. J., et al. 2025. “A Deeply Branching Lineage in Heterolobosea (Discoba) With Multiflagellated and Multigrooved Life Stages.” Journal of Eukaryotic Microbiology
72, no. 5: e70037. 10.1111/jeu.70037.
</p>
<section id="jeu70037-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="jeu70037-note-0001"><p>
<strong>Funding:</strong> D.M. and P.L.‐G. were supported by grants from the European Research Council (ERC Advanced grants 787904 and 101141745). L.J.G. was funded by the Ramón y Cajal Programme (grant RYC2022‐035282‐I, funded by MCIU/AEI/10.13039/501100011033 and FSE+).</p></div></div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Kristina I. Prokina, Email: kris142014@gmail.com.</p>
<p>David Moreira, Email: david.moreira@universite-paris-saclay.fr.</p></section><section id="jeu70037-sec-0021"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The data that support the findings of this study are openly available in Figshare at <a href="https://figshare.com/projects/Multisulcus_malaysiensis/220201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/projects/Multisulcus_malaysiensis/220201</a>.</p></section><section id="jeu70037-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="jeu70037-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="jeu70037-bib-0001">
<cite>

Adam, R. D.

2017. “Diplomonadida.” In Handbook of the Protists, edited by Archibald J. M., Simpson A. G. B., and Slamovits C. H., 2nd ed., 1219–1246. Springer.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Handbook%20of%20the%20Protists&amp;author=R.%20D.%20Adam&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0002">
<cite>

Altschul, S. F.
, Gish W., Miller W., Myers E. W., and Lipman D. J.. 1990. “Basic Local Alignment Search Tool.” Journal of Molecular Biology
215, no. 3: 403–410.
</cite> [<a href="https://doi.org/10.1016/S0022-2836(05)80360-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2231712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Molecular%20Biology&amp;title=Basic%20Local%20Alignment%20Search%20Tool&amp;author=S.%20F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.%20W.%20Myers&amp;author=D.%20J.%20Lipman&amp;volume=215&amp;issue=3&amp;publication_year=1990&amp;pages=403-410&amp;pmid=2231712&amp;doi=10.1016/S0022-2836(05)80360-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0003"><cite>

Andrews, S.

2010. “FastQC: A Quality Control Tool for High Throughput Sequence Data [Online].”
<a href="https://www.bioinformatics.babraham.ac.uk/projects/fastqc/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a>.</cite></li>
<li id="jeu70037-bib-0004">
<cite>

Baldauf, S. L.

2003. “The Deep Roots of Eukaryotes.” Science
300, no. 5626: 1703–1706.
</cite> [<a href="https://doi.org/10.1126/science.1085544" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12805537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20Deep%20Roots%20of%20Eukaryotes&amp;author=S.%20L.%20Baldauf&amp;volume=300&amp;issue=5626&amp;publication_year=2003&amp;pages=1703-1706&amp;pmid=12805537&amp;doi=10.1126/science.1085544&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0005">
<cite>

Bankevich, A.
, Nurk S., Antipov D., et al. 2012. “SPAdes: A New Genome Assembly Algorithm and Its Applications to Single‐Cell Sequencing.” Journal of Computational Biology
19, no. 5: 455–477.
</cite> [<a href="https://doi.org/10.1089/cmb.2012.0021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3342519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22506599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Computational%20Biology&amp;title=SPAdes:%20A%20New%20Genome%20Assembly%20Algorithm%20and%20Its%20Applications%20to%20Single%E2%80%90Cell%20Sequencing&amp;author=A.%20Bankevich&amp;author=S.%20Nurk&amp;author=D.%20Antipov&amp;volume=19&amp;issue=5&amp;publication_year=2012&amp;pages=455-477&amp;pmid=22506599&amp;doi=10.1089/cmb.2012.0021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0006">
<cite>

Bernard, C.
, Simpson A. G., and Patterson D. J.. 1997. “An Ultrastructural Study of a Free‐Living Retortamonad, <em>Chilomastix cuspidata</em> (Larsen &amp; Patterson, 1990) n. Comb. (Retortamonadida, Protista).” European Journal of Protistology
33, no. 3: 254–265.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Protistology&amp;title=An%20Ultrastructural%20Study%20of%20a%20Free%E2%80%90Living%20Retortamonad,%20Chilomastix%20cuspidata%20(Larsen%20&amp;%20Patterson,%201990)%20n.%20Comb.%20(Retortamonadida,%20Protista)&amp;author=C.%20Bernard&amp;author=A.%20G.%20Simpson&amp;author=D.%20J.%20Patterson&amp;volume=33&amp;issue=3&amp;publication_year=1997&amp;pages=254-265&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0007">
<cite>

Broers, C. A.
, Stumm C. K., Vogels G. D., and Brugerolle G.. 1990. “
<em>Psalteriomonas lanterna</em> Gen. Nov., sp. Nov., a Free‐Living Amoeboflagellate Isolated From Freshwater Anaerobic Sediments.” European Journal of Protistology
25, no. 4: 369–380.
</cite> [<a href="https://doi.org/10.1016/S0932-4739(11)80130-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23196051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Protistology&amp;title=Psalteriomonas%20lanterna%20Gen.%20Nov.,%20sp.%20Nov.,%20a%20Free%E2%80%90Living%20Amoeboflagellate%20Isolated%20From%20Freshwater%20Anaerobic%20Sediments&amp;author=C.%20A.%20Broers&amp;author=C.%20K.%20Stumm&amp;author=G.%20D.%20Vogels&amp;author=G.%20Brugerolle&amp;volume=25&amp;issue=4&amp;publication_year=1990&amp;pages=369-380&amp;pmid=23196051&amp;doi=10.1016/S0932-4739(11)80130-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0008">
<cite>

Brown, M. W.
, Silberman J. D., and Spiegel F. W.. 2010. “A Morphologically Simple Species of Acrasis (Heterolobosea, Excavata), <em>Acrasis helenhemmesae</em> n. sp.” Journal of Eukaryotic Microbiology
57, no. 4: 346–353.
</cite> [<a href="https://doi.org/10.1111/j.1550-7408.2010.00481.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20497285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Eukaryotic%20Microbiology&amp;title=A%20Morphologically%20Simple%20Species%20of%20Acrasis%20(Heterolobosea,%20Excavata),%20Acrasis%20helenhemmesae%20n.%20sp&amp;author=M.%20W.%20Brown&amp;author=J.%20D.%20Silberman&amp;author=F.%20W.%20Spiegel&amp;volume=57&amp;issue=4&amp;publication_year=2010&amp;pages=346-353&amp;pmid=20497285&amp;doi=10.1111/j.1550-7408.2010.00481.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0009">
<cite>

Brugerolle, G.
, and Simpson A. G.. 2004. “The Flagellar Apparatus of Heteroloboseans.” Journal of Eukaryotic Microbiology
51, no. 1: 96–107.
</cite> [<a href="https://doi.org/10.1111/j.1550-7408.2004.tb00169.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15068271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Eukaryotic%20Microbiology&amp;title=The%20Flagellar%20Apparatus%20of%20Heteroloboseans&amp;author=G.%20Brugerolle&amp;author=A.%20G.%20Simpson&amp;volume=51&amp;issue=1&amp;publication_year=2004&amp;pages=96-107&amp;pmid=15068271&amp;doi=10.1111/j.1550-7408.2004.tb00169.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0010">
<cite>

Burki, F.
, Roger A. J., Brown M. W., and Simpson A. G.. 2020. “The New Tree of Eukaryotes.” Trends in Ecology &amp; Evolution
35, no. 1: 43–55.
</cite> [<a href="https://doi.org/10.1016/j.tree.2019.08.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31606140/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Ecology%20&amp;%20Evolution&amp;title=The%20New%20Tree%20of%20Eukaryotes&amp;author=F.%20Burki&amp;author=A.%20J.%20Roger&amp;author=M.%20W.%20Brown&amp;author=A.%20G.%20Simpson&amp;volume=35&amp;issue=1&amp;publication_year=2020&amp;pages=43-55&amp;pmid=31606140&amp;doi=10.1016/j.tree.2019.08.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0011">
<cite>

Capella‐Gutiérrez, S.
, Silla‐Martínez J. M., and Gabaldón T.. 2009. “trimAl: A Tool for Automated Alignment Trimming in Large‐Scale Phylogenetic Analyses.” Bioinformatics
25, no. 15: 1972–1973.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2712344/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19505945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=trimAl:%20A%20Tool%20for%20Automated%20Alignment%20Trimming%20in%20Large%E2%80%90Scale%20Phylogenetic%20Analyses&amp;author=S.%20Capella%E2%80%90Guti%C3%A9rrez&amp;author=J.%20M.%20Silla%E2%80%90Mart%C3%ADnez&amp;author=T.%20Gabald%C3%B3n&amp;volume=25&amp;issue=15&amp;publication_year=2009&amp;pages=1972-1973&amp;pmid=19505945&amp;doi=10.1093/bioinformatics/btp348&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0012">
<cite>

Čepička, I.
, Dolan M. F., and Gile G.. 2017. “Parabasalia.” In Handbook of the Protists, edited by Archibald J. M., Simpson A. G. B., and Slamovits C. H., 2nd ed., 1175–1218. Springer.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Handbook%20of%20the%20Protists&amp;author=I.%20%C4%8Cepi%C4%8Dka&amp;author=M.%20F.%20Dolan&amp;author=G.%20Gile&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0014">
<cite>

De Jonckheere, J. F.
, Baumgartner M., Eberhardt S., Opperdoes F. R., and Stetter K. O.. 2011. “
<em>Oramoeba fumarolia</em> Gen. Nov., sp. Nov., a New Marine Heterolobosean Amoeboflagellate Growing at 54°C.” European Journal of Protistology
47, no. 1: 16–23.
</cite> [<a href="https://doi.org/10.1016/j.ejop.2010.09.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20932726/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Protistology&amp;title=Oramoeba%20fumarolia%20Gen.%20Nov.,%20sp.%20Nov.,%20a%20New%20Marine%20Heterolobosean%20Amoeboflagellate%20Growing%20at%2054%C2%B0C&amp;author=J.%20F.%20De%20Jonckheere&amp;author=M.%20Baumgartner&amp;author=S.%20Eberhardt&amp;author=F.%20R.%20Opperdoes&amp;author=K.%20O.%20Stetter&amp;volume=47&amp;issue=1&amp;publication_year=2011&amp;pages=16-23&amp;pmid=20932726&amp;doi=10.1016/j.ejop.2010.09.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0015">
<cite>

Derelle, R.
, Torruella G., Klimeš V., et al. 2015. “Bacterial Proteins Pinpoint a Single Eukaryotic Root.” Proceedings of the National Academy of Sciences of the United States of America
112, no. 7: E693–E699.
</cite> [<a href="https://doi.org/10.1073/pnas.1420657112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4343179/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25646484/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Bacterial%20Proteins%20Pinpoint%20a%20Single%20Eukaryotic%20Root&amp;author=R.%20Derelle&amp;author=G.%20Torruella&amp;author=V.%20Klime%C5%A1&amp;volume=112&amp;issue=7&amp;publication_year=2015&amp;pages=E693-E699&amp;pmid=25646484&amp;doi=10.1073/pnas.1420657112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0016">
<cite>

Eyden, B. P.
, and Vickerman K.. 1975. “Ultrastructure and Vacuolar Movements in the Free‐Living Diplomonad <em>Trepomonas agilis</em> Klebs.” Journal of Protozoology
22, no. 1: 54–66.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Protozoology&amp;title=Ultrastructure%20and%20Vacuolar%20Movements%20in%20the%20Free%E2%80%90Living%20Diplomonad%20Trepomonas%20agilis%20Klebs&amp;author=B.%20P.%20Eyden&amp;author=K.%20Vickerman&amp;volume=22&amp;issue=1&amp;publication_year=1975&amp;pages=54-66&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0017">
<cite>

Galindo, L. J.
, Prokina K., Torruella G., López‐García P., and Moreira D.. 2023. “Maturases and Group II Introns in the Mitochondrial Genomes of the Deepest Jakobid Branch.” Genome Biology and Evolution
15, no. 4: evad058.
</cite> [<a href="https://doi.org/10.1093/gbe/evad058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10139444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37029959/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology%20and%20Evolution&amp;title=Maturases%20and%20Group%20II%20Introns%20in%20the%20Mitochondrial%20Genomes%20of%20the%20Deepest%20Jakobid%20Branch&amp;author=L.%20J.%20Galindo&amp;author=K.%20Prokina&amp;author=G.%20Torruella&amp;author=P.%20L%C3%B3pez%E2%80%90Garc%C3%ADa&amp;author=D.%20Moreira&amp;volume=15&amp;issue=4&amp;publication_year=2023&amp;pages=evad058&amp;pmid=37029959&amp;doi=10.1093/gbe/evad058&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0018">
<cite>

Gibson, W.

2017. “Kinetoplastea.” In Handbook of the Protists, edited by Archibald J. M., Simpson A. G. B., and Slamovits C. H., 2nd ed., 1089–1138. Springer.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Handbook%20of%20the%20Protists&amp;author=W.%20Gibson&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0019">
<cite>

Hampl, V.

2017. “Preaxostyla.” In Handbook of the Protists, edited by Archibald J. M., Simpson A. G. B., and Slamovits C. H., 2nd ed., 1139–1174. Springer.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Handbook%20of%20the%20Protists&amp;author=V.%20Hampl&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0020">
<cite>

Hanousková, P.
, Táborský P., and Čepička I.. 2019. “
<em>Dactylomonas</em> Gen. Nov., a Novel Lineage of Heterolobosean Flagellates With Unique Ultrastructure, Closely Related to the Amoeba <em>Selenaion koniopes</em> Park, De Jonckheere &amp; Simpson, 2012.” Journal of Eukaryotic Microbiology
66, no. 1: 120–139.
</cite> [<a href="https://doi.org/10.1111/jeu.12637" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29791056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Eukaryotic%20Microbiology&amp;title=Dactylomonas%20Gen.%20Nov.,%20a%20Novel%20Lineage%20of%20Heterolobosean%20Flagellates%20With%20Unique%20Ultrastructure,%20Closely%20Related%20to%20the%20Amoeba%20Selenaion%20koniopes%20Park,%20De%20Jonckheere%20&amp;%20Simpson,%202012&amp;author=P.%20Hanouskov%C3%A1&amp;author=P.%20T%C3%A1borsk%C3%BD&amp;author=I.%20%C4%8Cepi%C4%8Dka&amp;volume=66&amp;issue=1&amp;publication_year=2019&amp;pages=120-139&amp;pmid=29791056&amp;doi=10.1111/jeu.12637&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0021">
<cite>

Harding, T.
, Brown M. W., Plotnikov A., et al. 2013. “Amoeba Stages in the Deepest Branching Heteroloboseans, Including <em>Pharyngomonas</em>: Evolutionary and Systematic Implications.” Protist
164, no. 2: 272–286.
</cite> [<a href="https://doi.org/10.1016/j.protis.2012.08.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23021907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protist&amp;title=Amoeba%20Stages%20in%20the%20Deepest%20Branching%20Heteroloboseans,%20Including%20Pharyngomonas:%20Evolutionary%20and%20Systematic%20Implications&amp;author=T.%20Harding&amp;author=M.%20W.%20Brown&amp;author=A.%20Plotnikov&amp;volume=164&amp;issue=2&amp;publication_year=2013&amp;pages=272-286&amp;pmid=23021907&amp;doi=10.1016/j.protis.2012.08.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0022">
<cite>

Hohl, H. R.
, and Hamamoto S. T.. 1969. “Ultrastructure of <em>Acrasis rosea</em>, a Cellular Slime Mold, During Development.” Journal of Protozoology
16, no. 2: 333–344.
</cite> [<a href="https://doi.org/10.1111/j.1550-7408.1969.tb02279.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/5816073/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Protozoology&amp;title=Ultrastructure%20of%20Acrasis%20rosea,%20a%20Cellular%20Slime%20Mold,%20During%20Development&amp;author=H.%20R.%20Hohl&amp;author=S.%20T.%20Hamamoto&amp;volume=16&amp;issue=2&amp;publication_year=1969&amp;pages=333-344&amp;pmid=5816073&amp;doi=10.1111/j.1550-7408.1969.tb02279.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0023">
<cite>

Kalyaanamoorthy, S.
, Minh B. Q., Wong T. K., Von Haeseler A., and Jermiin L. S.. 2017. “ModelFinder: Fast Model Selection for Accurate Phylogenetic Estimates.” Nature Methods
14, no. 6: 587–589.
</cite> [<a href="https://doi.org/10.1038/nmeth.4285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5453245/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28481363/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=ModelFinder:%20Fast%20Model%20Selection%20for%20Accurate%20Phylogenetic%20Estimates&amp;author=S.%20Kalyaanamoorthy&amp;author=B.%20Q.%20Minh&amp;author=T.%20K.%20Wong&amp;author=A.%20Von%20Haeseler&amp;author=L.%20S.%20Jermiin&amp;volume=14&amp;issue=6&amp;publication_year=2017&amp;pages=587-589&amp;pmid=28481363&amp;doi=10.1038/nmeth.4285&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0024">
<cite>

Katoh, K.
, Misawa K., Kuma K. I., and Miyata T.. 2002. “MAFFT: A Novel Method for Rapid Multiple Sequence Alignment Based on Fast Fourier Transform.” Nucleic Acids Research
30, no. 14: 3059–3066.
</cite> [<a href="https://doi.org/10.1093/nar/gkf436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC135756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12136088/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=MAFFT:%20A%20Novel%20Method%20for%20Rapid%20Multiple%20Sequence%20Alignment%20Based%20on%20Fast%20Fourier%20Transform&amp;author=K.%20Katoh&amp;author=K.%20Misawa&amp;author=K.%20I.%20Kuma&amp;author=T.%20Miyata&amp;volume=30&amp;issue=14&amp;publication_year=2002&amp;pages=3059-3066&amp;pmid=12136088&amp;doi=10.1093/nar/gkf436&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0025">
<cite>

Katoh, K.
, and Standley D. M.. 2013. “MAFFT Multiple Sequence Alignment Software Version 7: Improvements in Performance and Usability.” Molecular Biology and Evolution
30, no. 4: 772–780.
</cite> [<a href="https://doi.org/10.1093/molbev/mst010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3603318/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23329690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=MAFFT%20Multiple%20Sequence%20Alignment%20Software%20Version%207:%20Improvements%20in%20Performance%20and%20Usability&amp;author=K.%20Katoh&amp;author=D.%20M.%20Standley&amp;volume=30&amp;issue=4&amp;publication_year=2013&amp;pages=772-780&amp;pmid=23329690&amp;doi=10.1093/molbev/mst010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0026">
<cite>

Kulda, J.

2017. “Retortamonadida (With Notes on <em>Carpediemonas</em>‐Like Organisms and Caviomonadidae).” In Handbook of the Protists, edited by Archibald J. M., Simpson A. G. B., and Slamovits C. H., 2nd ed., 1247–1278. Springer.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Handbook%20of%20the%20Protists&amp;author=J.%20Kulda&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0027">
<cite>

Lamża, Ł.

2023. “Diversity of ‘Simple’ Multicellular Eukaryotes: 45 Independent Cases and Six Types of Multicellularity.” Biological Reviews
98, no. 6: 2188–2209.
</cite> [<a href="https://doi.org/10.1111/brv.13001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37475165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biological%20Reviews&amp;title=Diversity%20of%20%E2%80%98Simple%E2%80%99%20Multicellular%20Eukaryotes:%2045%20Independent%20Cases%20and%20Six%20Types%20of%20Multicellularity&amp;author=%C5%81.%20Lam%C5%BCa&amp;volume=98&amp;issue=6&amp;publication_year=2023&amp;pages=2188-2209&amp;pmid=37475165&amp;doi=10.1111/brv.13001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0028">
<cite>

Larsson, A.

2014. “AliView: A Fast and Lightweight Alignment Viewer and Editor for Large Datasets.” Bioinformatics
30, no. 22: 3276–3278.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu531" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4221126/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25095880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=AliView:%20A%20Fast%20and%20Lightweight%20Alignment%20Viewer%20and%20Editor%20for%20Large%20Datasets&amp;author=A.%20Larsson&amp;volume=30&amp;issue=22&amp;publication_year=2014&amp;pages=3276-3278&amp;pmid=25095880&amp;doi=10.1093/bioinformatics/btu531&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0029">
<cite>

Leander, B. S.
, Lax G., Karnkowska A., and Simpson A. G. B.. 2017. “Euglenida.” In Handbook of the Protists, edited by Archibald J. M., Simpson A. G. B., and Slamovits C. H., 2nd ed., 1047–1088. Springer.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Handbook%20of%20the%20Protists&amp;author=B.%20S.%20Leander&amp;author=G.%20Lax&amp;author=A.%20Karnkowska&amp;author=A.%20G.%20B.%20Simpson&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0030">
<cite>

Lee, W. J.
, Miller K., and Simpson A. G.. 2014. “Morphological and Molecular Characterization of a New Species of <em>Stephanopogon</em>, <em>Stephanopogon pattersoni</em> n. sp.” Journal of Eukaryotic Microbiology
61, no. 4: 389–398.
</cite> [<a href="https://doi.org/10.1111/jeu.12124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24830341/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Eukaryotic%20Microbiology&amp;title=Morphological%20and%20Molecular%20Characterization%20of%20a%20New%20Species%20of%20Stephanopogon,%20Stephanopogon%20pattersoni%20n.%20sp&amp;author=W.%20J.%20Lee&amp;author=K.%20Miller&amp;author=A.%20G.%20Simpson&amp;volume=61&amp;issue=4&amp;publication_year=2014&amp;pages=389-398&amp;pmid=24830341&amp;doi=10.1111/jeu.12124&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0031">
<cite>

Letunic, I.
, and Bork P.. 2024. “Interactive Tree of Life (iTOL) v6: Recent Updates to the Phylogenetic Tree Display and Annotation Tool.” Nucleic Acids Research
52, no. W1: W78–W82.
</cite> [<a href="https://doi.org/10.1093/nar/gkae268" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11223838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38613393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Interactive%20Tree%20of%20Life%20(iTOL)%20v6:%20Recent%20Updates%20to%20the%20Phylogenetic%20Tree%20Display%20and%20Annotation%20Tool&amp;author=I.%20Letunic&amp;author=P.%20Bork&amp;volume=52&amp;issue=W1&amp;publication_year=2024&amp;pages=W78-W82&amp;pmid=38613393&amp;doi=10.1093/nar/gkae268&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0032">
<cite>

Minh, B. Q.
, Schmidt H. A., Chernomor O., et al. 2020. “IQ‐TREE 2: New Models and Efficient Methods for Phylogenetic Inference in the Genomic Era.” Molecular Biology and Evolution
37, no. 5: 1530–1534.
</cite> [<a href="https://doi.org/10.1093/molbev/msaa015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7182206/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32011700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=IQ%E2%80%90TREE%202:%20New%20Models%20and%20Efficient%20Methods%20for%20Phylogenetic%20Inference%20in%20the%20Genomic%20Era&amp;author=B.%20Q.%20Minh&amp;author=H.%20A.%20Schmidt&amp;author=O.%20Chernomor&amp;volume=37&amp;issue=5&amp;publication_year=2020&amp;pages=1530-1534&amp;pmid=32011700&amp;doi=10.1093/molbev/msaa015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0033">
<cite>

Nikolaev, S. I.
, Mylnikov A. P., Berney C., et al. 2004. “Molecular Phylogenetic Analysis Places <em>Percolomonas cosmopolites</em> Within Heterolobosea: Evolutionary Implications.” Journal of Eukaryotic Microbiology
51, no. 5: 575–581.
</cite> [<a href="https://doi.org/10.1111/j.1550-7408.2004.tb00294.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15537093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Eukaryotic%20Microbiology&amp;title=Molecular%20Phylogenetic%20Analysis%20Places%20Percolomonas%20cosmopolites%20Within%20Heterolobosea:%20Evolutionary%20Implications&amp;author=S.%20I.%20Nikolaev&amp;author=A.%20P.%20Mylnikov&amp;author=C.%20Berney&amp;volume=51&amp;issue=5&amp;publication_year=2004&amp;pages=575-581&amp;pmid=15537093&amp;doi=10.1111/j.1550-7408.2004.tb00294.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0034">
<cite>

O'Kelly, C. J.

1993. “The Jakobid Flagellates: Structural Features of Jakoba, <em>Reclinomonas</em> and <em>Histiona</em> and Implications for the Early Diversification of Eukaryotes.” Journal of Eukaryotic Microbiology
40, no. 5: 627–636.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Eukaryotic%20Microbiology&amp;title=The%20Jakobid%20Flagellates:%20Structural%20Features%20of%20Jakoba,%20Reclinomonas%20and%20Histiona%20and%20Implications%20for%20the%20Early%20Diversification%20of%20Eukaryotes&amp;author=C.%20J.%20O'Kelly&amp;volume=40&amp;issue=5&amp;publication_year=1993&amp;pages=627-636&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0035">
<cite>

O'Kelly, C. J.
, Farmer M. A., and Nerad T. A.. 1999. “Ultrastructure of <em>Trimastix pyriformis</em> (Klebs) Bernard et al.: Similarities of <em>Trimastix</em> Species With Retortamonad and Jakobid Flagellates.” Protist
150, no. 2: 149–162.
</cite> [<a href="https://doi.org/10.1016/S1434-4610(99)70018-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10505415/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protist&amp;title=Ultrastructure%20of%20Trimastix%20pyriformis%20(Klebs)%20Bernard%20et%C2%A0al.:%20Similarities%20of%20Trimastix%20Species%20With%20Retortamonad%20and%20Jakobid%20Flagellates&amp;author=C.%20J.%20O'Kelly&amp;author=M.%20A.%20Farmer&amp;author=T.%20A.%20Nerad&amp;volume=150&amp;issue=2&amp;publication_year=1999&amp;pages=149-162&amp;pmid=10505415&amp;doi=10.1016/S1434-4610(99)70018-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0036">
<cite>

O'Kelly, C. J.
, and Nerad T. A.. 1999. “
<em>Malawimonas jakobiformis</em> n. Gen., n. sp. (Malawimonadidae n. Fam.): A <em>Jakoba</em>‐Like Heterotrophic Nanoflagellate With Discoidal Mitochondrial Cristae.” Journal of Eukaryotic Microbiology
46, no. 5: 522–531.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Eukaryotic%20Microbiology&amp;title=Malawimonas%20jakobiformis%20n.%20Gen.,%20n.%20sp.%20(Malawimonadidae%20n.%20Fam.):%20A%20Jakoba%E2%80%90Like%20Heterotrophic%20Nanoflagellate%20With%20Discoidal%20Mitochondrial%20Cristae&amp;author=C.%20J.%20O'Kelly&amp;author=T.%20A.%20Nerad&amp;volume=46&amp;issue=5&amp;publication_year=1999&amp;pages=522-531&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0037">
<cite>

Olivetta, M.
, and Dudin O.. 2023. “The Nuclear‐To‐Cytoplasmic Ratio Drives Cellularization in the Close Animal Relative <em>Sphaeroforma arctica</em>
.” Current Biology
33, no. 8: 1597–1605.
</cite> [<a href="https://doi.org/10.1016/j.cub.2023.03.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36996815/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Biology&amp;title=The%20Nuclear%E2%80%90To%E2%80%90Cytoplasmic%20Ratio%20Drives%20Cellularization%20in%20the%20Close%20Animal%20Relative%20Sphaeroforma%20arctica&amp;author=M.%20Olivetta&amp;author=O.%20Dudin&amp;volume=33&amp;issue=8&amp;publication_year=2023&amp;pages=1597-1605&amp;pmid=36996815&amp;doi=10.1016/j.cub.2023.03.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0038">
<cite>

Pánek, T.
, Ptáčková E., and Čepička I.. 2014. “Survey on Diversity of Marine/Saline Anaerobic Heterolobosea (Excavata: Discoba) With Description of Seven New Species.” International Journal of Systematic and Evolutionary Microbiology
64, no. Pt_7: 2280–2304.
</cite> [<a href="https://doi.org/10.1099/ijs.0.063487-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24729392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Systematic%20and%20Evolutionary%20Microbiology&amp;title=Survey%20on%20Diversity%20of%20Marine/Saline%20Anaerobic%20Heterolobosea%20(Excavata:%20Discoba)%20With%20Description%20of%20Seven%20New%20Species&amp;author=T.%20P%C3%A1nek&amp;author=E.%20Pt%C3%A1%C4%8Dkov%C3%A1&amp;author=I.%20%C4%8Cepi%C4%8Dka&amp;volume=64&amp;issue=Pt_7&amp;publication_year=2014&amp;pages=2280-2304&amp;pmid=24729392&amp;doi=10.1099/ijs.0.063487-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0039">
<cite>

Pánek, T.
, Simpson A. G., Hampl V., and Čepička I.. 2014. “
<em>Creneis carolina</em> Gen. Et sp. Nov. (Heterolobosea), a Novel Marine Anaerobic Protist With Strikingly Derived Morphology and Life Cycle.” Protist
165, no. 4: 542–567.
</cite> [<a href="https://doi.org/10.1016/j.protis.2014.05.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24999602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protist&amp;title=Creneis%20carolina%20Gen.%20Et%20sp.%20Nov.%20(Heterolobosea),%20a%20Novel%20Marine%20Anaerobic%20Protist%20With%20Strikingly%20Derived%20Morphology%20and%20Life%20Cycle&amp;author=T.%20P%C3%A1nek&amp;author=A.%20G.%20Simpson&amp;author=V.%20Hampl&amp;author=I.%20%C4%8Cepi%C4%8Dka&amp;volume=165&amp;issue=4&amp;publication_year=2014&amp;pages=542-567&amp;pmid=24999602&amp;doi=10.1016/j.protis.2014.05.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0040">
<cite>

Pánek, T.
, Simpson A. G. B., Brown M. W., and Dyer B. D.. 2017. “Heterolobosea.” In Handbook of the Protists, edited by Archibald J. M., Simpson A. G. B., and Slamovits C. H., 2nd ed., 1005–1046. Springer.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Handbook%20of%20the%20Protists&amp;author=T.%20P%C3%A1nek&amp;author=A.%20G.%20B.%20Simpson&amp;author=M.%20W.%20Brown&amp;author=B.%20D.%20Dyer&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0041">
<cite>

Pánek, T.
, Tice A. K., Corre P., et al. 2025. “An Expanded Phylogenomic Analysis of Heterolobosea Reveals the Deep Relationships, Non‐Canonical Genetic Codes, and Cryptic Flagellate Stages in the Group.” Molecular Phylogenetics and Evolution
204: 108289.
</cite> [<a href="https://doi.org/10.1016/j.ympev.2025.108289" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39826589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Phylogenetics%20and%20Evolution&amp;title=An%20Expanded%20Phylogenomic%20Analysis%20of%20Heterolobosea%20Reveals%20the%20Deep%20Relationships,%20Non%E2%80%90Canonical%20Genetic%20Codes,%20and%20Cryptic%20Flagellate%20Stages%20in%20the%20Group&amp;author=T.%20P%C3%A1nek&amp;author=A.%20K.%20Tice&amp;author=P.%20Corre&amp;volume=204&amp;publication_year=2025&amp;pages=108289&amp;pmid=39826589&amp;doi=10.1016/j.ympev.2025.108289&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0042">
<cite>

Park, J. S.
, De Jonckheere J. F., and Simpson A. G.. 2012. “Characterization of <em>Selenaion koniopes</em> n. Gen., n. sp., an Amoeba That Represents a New Major Lineage Within Heterolobosea, Isolated From the Wieliczka Salt Mine.” Journal of Eukaryotic Microbiology
59, no. 6: 601–613.
</cite> [<a href="https://doi.org/10.1111/j.1550-7408.2012.00641.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22888835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Eukaryotic%20Microbiology&amp;title=Characterization%20of%20Selenaion%20koniopes%20n.%20Gen.,%20n.%20sp.,%20an%20Amoeba%20That%20Represents%20a%20New%20Major%20Lineage%20Within%20Heterolobosea,%20Isolated%20From%20the%20Wieliczka%20Salt%20Mine&amp;author=J.%20S.%20Park&amp;author=J.%20F.%20De%20Jonckheere&amp;author=A.%20G.%20Simpson&amp;volume=59&amp;issue=6&amp;publication_year=2012&amp;pages=601-613&amp;pmid=22888835&amp;doi=10.1111/j.1550-7408.2012.00641.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0043">
<cite>

Park, J. S.
, and Simpson A. G.. 2011. “Characterization of <em>Pharyngomonas kirbyi</em> (=“<em>Macropharyngomonas halophila</em>” Nomen Nudum), a Very Deep‐Branching, Obligately Halophilic Heterolobosean Flagellate.” Protist
162, no. 5: 691–709.
</cite> [<a href="https://doi.org/10.1016/j.protis.2011.05.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21723194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protist&amp;title=Characterization%20of%20Pharyngomonas%20kirbyi%20(=%E2%80%9CMacropharyngomonas%20halophila%E2%80%9D%20Nomen%20Nudum),%20a%20Very%20Deep%E2%80%90Branching,%20Obligately%20Halophilic%20Heterolobosean%20Flagellate&amp;author=J.%20S.%20Park&amp;author=A.%20G.%20Simpson&amp;volume=162&amp;issue=5&amp;publication_year=2011&amp;pages=691-709&amp;pmid=21723194&amp;doi=10.1016/j.protis.2011.05.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0044">
<cite>

Park, J. S.
, and Simpson A. G.. 2016. “Characterization of a Deep‐Branching Heterolobosean, <em>Pharyngomonas turkanaensis</em> n. sp., Isolated From a Non‐Hypersaline Habitat, and Ultrastructural Comparison of Cysts and Amoebae Among <em>Pharyngomonas</em> Strains.” Journal of Eukaryotic Microbiology
63, no. 1: 100–111.
</cite> [<a href="https://doi.org/10.1111/jeu.12260" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26291784/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Eukaryotic%20Microbiology&amp;title=Characterization%20of%20a%20Deep%E2%80%90Branching%20Heterolobosean,%20Pharyngomonas%20turkanaensis%20n.%20sp.,%20Isolated%20From%20a%20Non%E2%80%90Hypersaline%20Habitat,%20and%20Ultrastructural%20Comparison%20of%20Cysts%20and%20Amoebae%20Among%20Pharyngomonas%20Strains&amp;author=J.%20S.%20Park&amp;author=A.%20G.%20Simpson&amp;volume=63&amp;issue=1&amp;publication_year=2016&amp;pages=100-111&amp;pmid=26291784&amp;doi=10.1111/jeu.12260&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0045">
<cite>

Patterson, D. J.
, and Brugerolle G.. 1988. “The Ultrastructural Identity of <em>Stephanopogon apogon</em> and the Relatedness of the Genus to Other Kinds of Protists.” European Journal of Protistology
23, no. 3: 279–290.
</cite> [<a href="https://doi.org/10.1016/S0932-4739(88)80045-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23195217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Protistology&amp;title=The%20Ultrastructural%20Identity%20of%20Stephanopogon%20apogon%20and%20the%20Relatedness%20of%20the%20Genus%20to%20Other%20Kinds%20of%20Protists&amp;author=D.%20J.%20Patterson&amp;author=G.%20Brugerolle&amp;volume=23&amp;issue=3&amp;publication_year=1988&amp;pages=279-290&amp;pmid=23195217&amp;doi=10.1016/S0932-4739(88)80045-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0046">
<cite>

Price, M. N.
, Dehal P. S., and Arkin A. P.. 2010. “FastTree 2–Approximately Maximum‐Likelihood Trees for Large Alignments.” PLoS One
5, no. 3: e9490.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0009490" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2835736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20224823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=FastTree%202%E2%80%93Approximately%20Maximum%E2%80%90Likelihood%20Trees%20for%20Large%20Alignments&amp;author=M.%20N.%20Price&amp;author=P.%20S.%20Dehal&amp;author=A.%20P.%20Arkin&amp;volume=5&amp;issue=3&amp;publication_year=2010&amp;pages=e9490&amp;pmid=20224823&amp;doi=10.1371/journal.pone.0009490&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0047"><cite>

Rambaut, A.

2016. “FigTree 1.4.3.”
<a href="https://tree.bio.ed.ac.uk/software/figtree" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://tree.bio.ed.ac.uk/software/figtree</a>.</cite></li>
<li id="jeu70037-bib-0048">
<cite>

Ronquist, F.
, Teslenko M., Van Der Mark P., et al. 2012. “MrBayes 3.2: Efficient Bayesian Phylogenetic Inference and Model Choice Across a Large Model Space.” Systematic Biology
61, no. 3: 539–542.
</cite> [<a href="https://doi.org/10.1093/sysbio/sys029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3329765/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22357727/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Systematic%20Biology&amp;title=MrBayes%203.2:%20Efficient%20Bayesian%20Phylogenetic%20Inference%20and%20Model%20Choice%20Across%20a%20Large%20Model%20Space&amp;author=F.%20Ronquist&amp;author=M.%20Teslenko&amp;author=P.%20Van%20Der%20Mark&amp;volume=61&amp;issue=3&amp;publication_year=2012&amp;pages=539-542&amp;pmid=22357727&amp;doi=10.1093/sysbio/sys029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0050">
<cite>

Simão, F. A.
, Waterhouse R. M., Ioannidis P., Kriventseva E. V., and Zdobnov E. M.. 2015. “BUSCO: Assessing Genome Assembly and Annotation Completeness With Single‐Copy Orthologs.” Bioinformatics
31, no. 19: 3210–3212.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv351" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26059717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=BUSCO:%20Assessing%20Genome%20Assembly%20and%20Annotation%20Completeness%20With%20Single%E2%80%90Copy%20Orthologs&amp;author=F.%20A.%20Sim%C3%A3o&amp;author=R.%20M.%20Waterhouse&amp;author=P.%20Ioannidis&amp;author=E.%20V.%20Kriventseva&amp;author=E.%20M.%20Zdobnov&amp;volume=31&amp;issue=19&amp;publication_year=2015&amp;pages=3210-3212&amp;pmid=26059717&amp;doi=10.1093/bioinformatics/btv351&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0051">
<cite>

Simpson, A. G. B.

2003. “Cytoskeletal Organization, Phylogenetic Affinities and Systematics in the Contentious Taxon Excavata (Eukaryota).” International Journal of Systematic and Evolutionary Microbiology
53, no. 6: 1759–1777.
</cite> [<a href="https://doi.org/10.1099/ijs.0.02578-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14657103/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Systematic%20and%20Evolutionary%20Microbiology&amp;title=Cytoskeletal%20Organization,%20Phylogenetic%20Affinities%20and%20Systematics%20in%20the%20Contentious%20Taxon%20Excavata%20(Eukaryota)&amp;author=A.%20G.%20B.%20Simpson&amp;volume=53&amp;issue=6&amp;publication_year=2003&amp;pages=1759-1777&amp;pmid=14657103&amp;doi=10.1099/ijs.0.02578-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0052">
<cite>

Simpson, A. G. B.
, and Patterson D. J.. 1999. “The Ultrastructure of <em>Carpediemonas membranifera</em> (Eukaryota), With Reference to the Excavate Hypothesis.” European Journal of Protistology
35, no. 4: 353–370.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Protistology&amp;title=The%20Ultrastructure%20of%20Carpediemonas%20membranifera%20(Eukaryota),%20With%20Reference%20to%20the%20Excavate%20Hypothesis&amp;author=A.%20G.%20B.%20Simpson&amp;author=D.%20J.%20Patterson&amp;volume=35&amp;issue=4&amp;publication_year=1999&amp;pages=353-370&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0053">
<cite>

Suzuki‐Tellier, S.
, Miano F., Asadzadeh S. S., Simpson A. G., and Kiørboe T.. 2024. “Foraging Mechanisms in Excavate Flagellates Shed Light on the Functional Ecology of Early Eukaryotes.” Proceedings of the National Academy of Sciences of the United States of America
121, no. 22: e2317264121.
</cite> [<a href="https://doi.org/10.1073/pnas.2317264121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11145212/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38781211/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Foraging%20Mechanisms%20in%20Excavate%20Flagellates%20Shed%20Light%20on%20the%20Functional%20Ecology%20of%20Early%20Eukaryotes&amp;author=S.%20Suzuki%E2%80%90Tellier&amp;author=F.%20Miano&amp;author=S.%20S.%20Asadzadeh&amp;author=A.%20G.%20Simpson&amp;author=T.%20Ki%C3%B8rboe&amp;volume=121&amp;issue=22&amp;publication_year=2024&amp;pages=e2317264121&amp;pmid=38781211&amp;doi=10.1073/pnas.2317264121&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0054">
<cite>

Torruella, G.
, Galindo L. J., Moreira D., and López‐García P.. 2025. “Phylogenomics of Neglected Flagellated Protists Supports a Revised Eukaryotic Tree of Life.” Current Biology
35, no. 1: 198–207.
</cite> [<a href="https://doi.org/10.1016/j.cub.2024.10.075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39642877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Biology&amp;title=Phylogenomics%20of%20Neglected%20Flagellated%20Protists%20Supports%20a%20Revised%20Eukaryotic%20Tree%20of%20Life&amp;author=G.%20Torruella&amp;author=L.%20J.%20Galindo&amp;author=D.%20Moreira&amp;author=P.%20L%C3%B3pez%E2%80%90Garc%C3%ADa&amp;volume=35&amp;issue=1&amp;publication_year=2025&amp;pages=198-207&amp;pmid=39642877&amp;doi=10.1016/j.cub.2024.10.075&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0055">
<cite>

Wang, H. C.
, Minh B. Q., Susko E., and Roger A. J.. 2018. “Modeling Site Heterogeneity With Posterior Mean Site Frequency Profiles Accelerates Accurate Phylogenomic Estimation.” Systematic Biology
67, no. 2: 216–235.
</cite> [<a href="https://doi.org/10.1093/sysbio/syx068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28950365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Systematic%20Biology&amp;title=Modeling%20Site%20Heterogeneity%20With%20Posterior%20Mean%20Site%20Frequency%20Profiles%20Accelerates%20Accurate%20Phylogenomic%20Estimation&amp;author=H.%20C.%20Wang&amp;author=B.%20Q.%20Minh&amp;author=E.%20Susko&amp;author=A.%20J.%20Roger&amp;volume=67&amp;issue=2&amp;publication_year=2018&amp;pages=216-235&amp;pmid=28950365&amp;doi=10.1093/sysbio/syx068&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0056">
<cite>

Williamson, K.
, Eme L., Baños H., et al. 2024. “A Robustly Rooted Tree of Eukaryotes Reveals Their Excavate Ancestry.” Nature
640: 974–981.</cite> [<a href="https://doi.org/10.1038/s41586-025-08709-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40074902/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A%20Robustly%20Rooted%20Tree%20of%20Eukaryotes%20Reveals%20Their%20Excavate%20Ancestry&amp;author=K.%20Williamson&amp;author=L.%20Eme&amp;author=H.%20Ba%C3%B1os&amp;volume=640&amp;publication_year=2024&amp;pages=974-981&amp;pmid=40074902&amp;doi=10.1038/s41586-025-08709-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0057">
<cite>

Yabuki, A.
, Gyaltshen Y., Heiss A. A., Fujikura K., and Kim E.. 2018. “
<em>Ophirina amphinema</em> n. Gen., n. sp., a New Deeply Branching Discobid With Phylogenetic Affinity to Jakobids.” Scientific Reports
8, no. 1: 16219.
</cite> [<a href="https://doi.org/10.1038/s41598-018-34504-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6212452/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30385814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Ophirina%20amphinema%20n.%20Gen.,%20n.%20sp.,%20a%20New%20Deeply%20Branching%20Discobid%20With%20Phylogenetic%20Affinity%20to%20Jakobids&amp;author=A.%20Yabuki&amp;author=Y.%20Gyaltshen&amp;author=A.%20A.%20Heiss&amp;author=K.%20Fujikura&amp;author=E.%20Kim&amp;volume=8&amp;issue=1&amp;publication_year=2018&amp;pages=16219&amp;pmid=30385814&amp;doi=10.1038/s41598-018-34504-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0058">
<cite>

Yabuki, A.
, Nakayama T., Yubuki N., Hashimoto T., Ishida K. I., and Inagaki Y.. 2011. “
<em>Tsukubamonas globosa</em> n. Gen., n. sp., a Novel Excavate Flagellate Possibly Holding a Key for the Early Evolution in “Discoba”.” Journal of Eukaryotic Microbiology
58, no. 4: 319–331.
</cite> [<a href="https://doi.org/10.1111/j.1550-7408.2011.00552.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21569159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Eukaryotic%20Microbiology&amp;title=Tsukubamonas%20globosa%20n.%20Gen.,%20n.%20sp.,%20a%20Novel%20Excavate%20Flagellate%20Possibly%20Holding%20a%20Key%20for%20the%20Early%20Evolution%20in%20%E2%80%9CDiscoba%E2%80%9D&amp;author=A.%20Yabuki&amp;author=T.%20Nakayama&amp;author=N.%20Yubuki&amp;author=T.%20Hashimoto&amp;author=K.%20I.%20Ishida&amp;volume=58&amp;issue=4&amp;publication_year=2011&amp;pages=319-331&amp;pmid=21569159&amp;doi=10.1111/j.1550-7408.2011.00552.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0059">
<cite>

Yang, J.
, Harding T., Kamikawa R., Simpson A. G. B., and Roger A. J.. 2017. “Mitochondrial Genome Evolution and a Novel RNA Editing System in Deep‐Branching Heteroloboseids.” Genome Biology and Evolution
9: 1161–1174.
</cite> [<a href="https://doi.org/10.1093/gbe/evx086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5421314/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28453770/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology%20and%20Evolution&amp;title=Mitochondrial%20Genome%20Evolution%20and%20a%20Novel%20RNA%20Editing%20System%20in%20Deep%E2%80%90Branching%20Heteroloboseids&amp;author=J.%20Yang&amp;author=T.%20Harding&amp;author=R.%20Kamikawa&amp;author=A.%20G.%20B.%20Simpson&amp;author=A.%20J.%20Roger&amp;volume=9&amp;publication_year=2017&amp;pages=1161-1174&amp;pmid=28453770&amp;doi=10.1093/gbe/evx086&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0060">
<cite>

Yubuki, N.
, and Leander B. S.. 2008. “Ultrastructure and Molecular Phylogeny of <em>Stephanopogon minuta</em>: An Enigmatic Microeukaryote From Marine Interstitial Environments.” European Journal of Protistology
44, no. 4: 241–253.
</cite> [<a href="https://doi.org/10.1016/j.ejop.2007.12.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18403188/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=European%20Journal%20of%20Protistology&amp;title=Ultrastructure%20and%20Molecular%20Phylogeny%20of%20Stephanopogon%20minuta:%20An%20Enigmatic%20Microeukaryote%20From%20Marine%20Interstitial%20Environments&amp;author=N.%20Yubuki&amp;author=B.%20S.%20Leander&amp;volume=44&amp;issue=4&amp;publication_year=2008&amp;pages=241-253&amp;pmid=18403188&amp;doi=10.1016/j.ejop.2007.12.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="jeu70037-bib-0061">
<cite>

Yubuki, N.
, Simpson A. G. B., and Leander B. S.. 2013. “Comprehensive Ultrastructure of <em>Kipferlia Bialata</em> Provides Evidence for Character Evolution Within the Fornicata (Excavata).” Protist
164, no. 3: 423–439.
</cite> [<a href="https://doi.org/10.1016/j.protis.2013.02.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23517666/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protist&amp;title=Comprehensive%20Ultrastructure%20of%20Kipferlia%20Bialata%20Provides%20Evidence%20for%20Character%20Evolution%20Within%20the%20Fornicata%20(Excavata)&amp;author=N.%20Yubuki&amp;author=A.%20G.%20B.%20Simpson&amp;author=B.%20S.%20Leander&amp;volume=164&amp;issue=3&amp;publication_year=2013&amp;pages=423-439&amp;pmid=23517666&amp;doi=10.1016/j.protis.2013.02.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
<strong>Figure S1:</strong> (a) Inferred secondary structures of the 18S rRNA region containing the helices 17 and 18 in <em>Pharyngomonas kirby</em>, <em>Psalteriomonas lanterna</em>, and <em>Multisulcus malaysiensis</em>. Notice the additional helix 17_1 present in <em>P. lanterna</em> and 
<em>M. malaysiensis</em>
. (b) Multiple sequence alignment of the 18S rRNA region containing the helices 17 and 18. Uppercase positions are those shown in panel a. Some additional conserved positions are shown on lowercase. The green positions in <em>P. lanterna</em> and 
<em>M. malaysiensis</em>
 sequences correspond to the additional helix 17_1.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328842/bin/JEU-72-e70037-s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">JEU-72-e70037-s002.pdf</a><sup> (597KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>
<strong>Video S1:</strong> Live biflagellated cells of <em>Multisulcus malaysiensis</em> strain NY0229. DIC × 100 oil immersion.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328842/bin/JEU-72-e70037-s001.mp4" data-ga-action="click_feat_suppl" class="usa-link">Download video file</a><sup> (11MB, mp4) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are openly available in Figshare at <a href="https://figshare.com/projects/Multisulcus_malaysiensis/220201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://figshare.com/projects/Multisulcus_malaysiensis/220201</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Journal of Eukaryotic Microbiology are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1111/jeu.70037"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/JEU-72-e70037.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12328842/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12328842/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12328842%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328842/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12328842/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12328842/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40770464/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12328842/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40770464/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12328842/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12328842/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="RoBHjR02xhU3os7OPCD4uoATZ34kKpRDIKhuYRML6LNnnoEiBSsllloEEYbmmZPE">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
