
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Chromosome‐level genome of Zoysia sinica in the intertidal zone reveals genomic insights into waterlogging stress adaptation - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="4DDE47A689B722731B47A60004841B4E.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12238702/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Chromosome‐level genome of Zoysia sinica in the intertidal zone reveals genomic insights into waterlogging stress adaptation">
<meta name="citation_author" content="Hyeonseon Park">
<meta name="citation_author_institution" content="Department of Biological Science, Chungnam National University, Daejeon, Republic of Korea">
<meta name="citation_author" content="Eunji Bae">
<meta name="citation_author_institution" content="Forest Biomaterials Research Center, National Institute of Forest Science, Jinju, Republic of Korea">
<meta name="citation_author" content="Jae Gyeong Jung">
<meta name="citation_author_institution" content="Department of Biological Science, Chungnam National University, Daejeon, Republic of Korea">
<meta name="citation_author" content="Jaewook Kim">
<meta name="citation_author_institution" content="Department of Biology Education, Korea National University of Education, Cheongju, Republic of Korea">
<meta name="citation_author" content="Bae Young Choi">
<meta name="citation_author_institution" content="School of Liberal Arts and Sciences, Korea National University of Transportation, Chungju, Republic of Korea">
<meta name="citation_author" content="Geungjoo Lee">
<meta name="citation_author_institution" content="Department of Horticulture, Chungnam National University, Daejeon, Republic of Korea">
<meta name="citation_author_institution" content="Department of Smart Agriculture Systems, Chungnam National University, Daejeon, Republic of Korea">
<meta name="citation_author" content="Changsoo Kim">
<meta name="citation_author_institution" content="Department of Crop Science, College of Life Sciences, Chungnam National University, Daejeon, Republic of Korea">
<meta name="citation_author" content="Donghwan Shim">
<meta name="citation_author_institution" content="Department of Biological Science, Chungnam National University, Daejeon, Republic of Korea">
<meta name="citation_author_institution" content="Center for Genome Engineering, Institute for Basic Science, Daejeon, Republic of Korea">
<meta name="citation_publication_date" content="2025 Jul 8">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70070">
<meta name="citation_doi" content="10.1002/tpg2.70070">
<meta name="citation_pmid" content="40629672">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12238702/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12238702/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12238702/pdf/TPG2-18-e70070.pdf">
<meta name="description" content="Zoysia sinica is a perennial grass that thrives in intertidal zones, even under extreme waterlogging stress. In this study, we present a high‐quality chromosome‐level genome assembly of Z. sinica, with a total size of 312.67 Mb. Through genome ...">
<meta name="og:title" content="Chromosome‐level genome of Zoysia sinica in the intertidal zone reveals genomic insights into waterlogging stress adaptation">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Zoysia sinica is a perennial grass that thrives in intertidal zones, even under extreme waterlogging stress. In this study, we present a high‐quality chromosome‐level genome assembly of Z. sinica, with a total size of 312.67 Mb. Through genome ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12238702/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12238702">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70070"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70070.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12238702%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12238702/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12238702/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12238702/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70070" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Jul 8;18(3):e70070. doi: <a href="https://doi.org/10.1002/tpg2.70070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70070</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Chromosome‐level genome of <em>Zoysia sinica</em> in the intertidal zone reveals genomic insights into waterlogging stress adaptation</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Hyeonseon Park</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Hyeonseon Park</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Biological Science, Chungnam National University, Daejeon, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Park%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hyeonseon Park</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bae%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Eunji Bae</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Eunji Bae</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Forest Biomaterials Research Center, National Institute of Forest Science, Jinju, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bae%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Eunji Bae</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jung%20JG%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jae Gyeong Jung</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jae Gyeong Jung</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Biological Science, Chungnam National University, Daejeon, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jung%20JG%22%5BAuthor%5D" class="usa-link"><span class="name western">Jae Gyeong Jung</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Jaewook Kim</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Jaewook Kim</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Biology Education, Korea National University of Education, Cheongju, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jaewook Kim</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Choi%20BY%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Bae Young Choi</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Bae Young Choi</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>School of Liberal Arts and Sciences, Korea National University of Transportation, Chungju, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Choi%20BY%22%5BAuthor%5D" class="usa-link"><span class="name western">Bae Young Choi</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Geungjoo Lee</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Geungjoo Lee</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Department of Horticulture, Chungnam National University, Daejeon, Republic of Korea</div>
<div class="p">
<sup>
<sup>6</sup>
</sup>Department of Smart Agriculture Systems, Chungnam National University, Daejeon, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Geungjoo Lee</span></a>
</div>
</div>
<sup>5,</sup><sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Changsoo Kim</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Changsoo Kim</span></h3>
<div class="p">
<sup>
<sup>7</sup>
</sup>Department of Crop Science, College of Life Sciences, Chungnam National University, Daejeon, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Changsoo Kim</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shim%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Donghwan Shim</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Donghwan Shim</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Biological Science, Chungnam National University, Daejeon, Republic of Korea</div>
<div class="p">
<sup>
<sup>8</sup>
</sup>Center for Genome Engineering, Institute for Basic Science, Daejeon, Republic of Korea</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shim%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Donghwan Shim</span></a>
</div>
</div>
<sup>1,</sup><sup>8,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270070-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Biological Science, Chungnam National University, Daejeon, Republic of Korea</div>
<div id="tpg270070-aff-0002">
<sup>
<sup>2</sup>
</sup>Forest Biomaterials Research Center, National Institute of Forest Science, Jinju, Republic of Korea</div>
<div id="tpg270070-aff-0003">
<sup>
<sup>3</sup>
</sup>Department of Biology Education, Korea National University of Education, Cheongju, Republic of Korea</div>
<div id="tpg270070-aff-0004">
<sup>
<sup>4</sup>
</sup>School of Liberal Arts and Sciences, Korea National University of Transportation, Chungju, Republic of Korea</div>
<div id="tpg270070-aff-0005">
<sup>
<sup>5</sup>
</sup>Department of Horticulture, Chungnam National University, Daejeon, Republic of Korea</div>
<div id="tpg270070-aff-0006">
<sup>
<sup>6</sup>
</sup>Department of Smart Agriculture Systems, Chungnam National University, Daejeon, Republic of Korea</div>
<div id="tpg270070-aff-0007">
<sup>
<sup>7</sup>
</sup>Department of Crop Science, College of Life Sciences, Chungnam National University, Daejeon, Republic of Korea</div>
<div id="tpg270070-aff-0008">
<sup>
<sup>8</sup>
</sup>Center for Genome Engineering, Institute for Basic Science, Daejeon, Republic of Korea</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Donghwan Shim, Department of Biological Science, Chungnam National University, Daejeon 34134, Republic of Korea. Email: <span>dshim104@cnu.ac.kr</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Apr 17; Received 2024 Dec 11; Accepted 2025 May 14; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12238702  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40629672/" class="usa-link">40629672</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>
<em>Zoysia sinica</em> is a perennial grass that thrives in intertidal zones, even under extreme waterlogging stress. In this study, we present a high‐quality chromosome‐level genome assembly of <em>Z. sinica</em>, with a total size of 312.67 Mb. Through genome annotation, we identified 29,551 protein‐coding genes in the <em>Z. sinica</em> genome and re‐annotated 32,925, 53,226, and 53,656 genes in the previously reported <em>Zoysia</em> genomes: <em>Zoysia japonica</em>, <em>Zoysia matrella</em>, and <em>Zoysia pacifica</em>, respectively. Genome divergence analysis indicated that the <em>Zoysia</em> species diverged relatively recently, approximately 3.63 million years ago. Comparative genomic analysis revealed an expansion of ethylene response factors and identified <em>Z. sinica</em>‐specific genes related to the response to auxin and ethylene. Transcriptome data from intertidal environments with different levels of waterlogging showed significant upregulation of nitrate transporters (<em>NRT2.2</em> and <em>NRT2.4</em>) in roots and genes involved in suberin biosynthesis in shoots. Additionally, various transporters were responsive to the intertidal environment. Our study provides insights into the intertidal adaptation of <em>Zoysia</em> species and offers a foundation for the development of stress‐tolerant cultivars.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270070-list-10001" class="list" style="list-style-type:disc">
<li><p>This study presents the first chromosome‐level genome assembly of <em>Zoysia sinica</em> using Oxford Nanopore long reads and Omni‐C data, which provides a foundational genomic resource.</p></li>
<li><p>We provided a re‐annotation of genomes for <em>Zoysia japonica</em>, <em>Zoysia matrella</em>, and <em>Zoysia pacifica</em>, enhancing gene annotation accuracy and comparative analyses.</p></li>
<li><p>Our findings offer insights into intertidal adaptation of <em>Z. sinica</em>, highlighting genes involved in nitrogen metabolism, ethylene response, and suberin biosynthesis under waterlogging stress.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270070-dl-0001">
<dt>BUSCO</dt>
<dd><p>benchmarking universal single‐copy ortholog</p></dd>
<dt>GOBP</dt>
<dd><p>gene ontology in terms of biological process</p></dd>
<dt>GSEA</dt>
<dd><p>gene set enrichment analysis</p></dd>
<dt>GWC</dt>
<dd><p>gravimetric water content</p></dd>
<dt>HIZ</dt>
<dd><p>high intertidal zone</p></dd>
<dt>IL</dt>
<dd><p>Inland</p></dd>
<dt>KEGG</dt>
<dd><p>Kyoto Encyclopedia of Genes and Genomes</p></dd>
<dt>LIZ</dt>
<dd><p>low intertidal zone</p></dd>
<dt>NES</dt>
<dd><p>normalized enrichment score</p></dd>
<dt>PCA</dt>
<dd><p>principal component analysis</p></dd>
<dt>PSG</dt>
<dd><p>positively selected gene</p></dd>
<dt>RNA‐seq</dt>
<dd><p>RNA sequencing</p></dd>
<dt>TPM</dt>
<dd><p>transcripts per million</p></dd>
<dt>WGD</dt>
<dd><p>whole‐genome duplication</p></dd>
</dl>
<section id="tpg270070-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>
<em>Zoysia sinica</em> Hance is a warm‐season turfgrass belonging to the genus <em>Zoysia</em> (<em>Zoysia</em> spp. Willd.). <em>Zoysia</em> species are perennial grasses that are commonly used in golf courses, gardens, and parks, owing to their horticultural value. Therefore, <em>Zoysia</em> spp. have been a major target of breeding due to their economically important resources. One of the major phenotypic targets of breeding in <em>Zoysia</em> is the cold tolerance phenotypes (Guo et al., <a href="#tpg270070-bib-0031" class="usa-link" aria-describedby="tpg270070-bib-0031">2012</a>; Jin et al., <a href="#tpg270070-bib-0038" class="usa-link" aria-describedby="tpg270070-bib-0038">2022</a>; Wei et al., <a href="#tpg270070-bib-0090" class="usa-link" aria-describedby="tpg270070-bib-0090">2015</a>). In the course of inbreeding the cold‐tolerant cultivar, intercross among the same <em>Zoysia</em> species and the intercross among other <em>Zoysia</em> species were both frequently utilized (Guo et al., <a href="#tpg270070-bib-0031" class="usa-link" aria-describedby="tpg270070-bib-0031">2012</a>; Jin et al., <a href="#tpg270070-bib-0038" class="usa-link" aria-describedby="tpg270070-bib-0038">2022</a>; Wei et al., <a href="#tpg270070-bib-0090" class="usa-link" aria-describedby="tpg270070-bib-0090">2015</a>). Another phenotypic target was leaf and stolon tensile strength, which is correlated with the market value of <em>Zoysia</em> species (Choi et al., <a href="#tpg270070-bib-0019" class="usa-link" aria-describedby="tpg270070-bib-0019">2017</a>; Lulli et al., <a href="#tpg270070-bib-0052" class="usa-link" aria-describedby="tpg270070-bib-0052">2011</a>). <em>Zoysia</em> species were known to suffer from herbivorous bugs and fungus; thus, resistance to the pests was the target of breeding (Anderson et al., <a href="#tpg270070-bib-0004" class="usa-link" aria-describedby="tpg270070-bib-0004">2007</a>; Obasa et al., <a href="#tpg270070-bib-0062" class="usa-link" aria-describedby="tpg270070-bib-0062">2012</a>).</p>
<p>In nature, <em>Zoysia</em> species are distributed along coastlines and islands (Loch et al., <a href="#tpg270070-bib-0051" class="usa-link" aria-describedby="tpg270070-bib-0051">2017</a>), and <em>Z. sinica</em> is a salt marsh turfgrass naturally distributed in Korea and Japan. The ecological distribution of <em>Z. sinica</em> ranges from mudflats that are inundated by seawater for 3 h per day to mesohaline waters consisting of soils with high salt content and little direct influence from seawater (Bang et al., <a href="#tpg270070-bib-0005" class="usa-link" aria-describedby="tpg270070-bib-0005">2018</a>; J.‐S. Lee et al., <a href="#tpg270070-bib-0044" class="usa-link" aria-describedby="tpg270070-bib-0044">2009</a>; S. H. Lee et al., <a href="#tpg270070-bib-0045" class="usa-link" aria-describedby="tpg270070-bib-0045">2018</a>; Numata et al., <a href="#tpg270070-bib-0061" class="usa-link" aria-describedby="tpg270070-bib-0061">1972</a>). <em>Z. sinica</em> is a species within the genus <em>Zoysia</em> that has adapted to waterlogged environments and is found in intertidal zones.</p>
<p>Although plants historically transitioned to land, recent studies have increasingly focused on their adaptations to waterlogged environments. Notably, diverse plant species in mangrove ecosystems, which thrive in coastal salt marshes and intertidal environments, have shown significant associations with ethylene response pathways and hypoxic stress response mechanisms (Liu et al., <a href="#tpg270070-bib-0049" class="usa-link" aria-describedby="tpg270070-bib-0049">2024</a>; Su et al., <a href="#tpg270070-bib-0077" class="usa-link" aria-describedby="tpg270070-bib-0077">2022</a>). Recently, research on the monocot mangrove species <em>Nypa fruticans</em> has reported that it maintains genomic stability over long periods through low mutation rates and a strong purifying selection (Wu et al., <a href="#tpg270070-bib-0091" class="usa-link" aria-describedby="tpg270070-bib-0091">2024</a>). Key genes such as ethylene response factor VII (<em>ERF‐VII</em>), wound‐induced polypeptide 3 (<em>WIP3</em>), and UDP‐glucosyl transferase 73B3 (<em>UGT73B3</em>) play crucial roles in intertidal environments. Similarly, recent studies on <em>Spartina alterniflora</em>, a member of the Zoysieae tribe within the subfamily Chloridoideae, have demonstrated its aggressive invasion in coastal ecosystems worldwide, supported by genomic studies (S. Chen et al., <a href="#tpg270070-bib-0015" class="usa-link" aria-describedby="tpg270070-bib-0015">2024</a>; Hao et al., <a href="#tpg270070-bib-0033" class="usa-link" aria-describedby="tpg270070-bib-0033">2024</a>). It was reported that the gene families related to “ion transport” and “response to salt stress” have expanded and been positively selected. Such mechanisms of plant adaptation to salt and waterlogged environments are crucial for their survival in saline and waterlogged conditions. While the <em>Zoysia</em> genus, a well‐known halophyte with high resistance to abiotic stress, has been extensively studied for its salt tolerance (W. Wang et al., <a href="#tpg270070-bib-0088" class="usa-link" aria-describedby="tpg270070-bib-0088">2022</a>; Zheng et al., <a href="#tpg270070-bib-0096" class="usa-link" aria-describedby="tpg270070-bib-0096">2022</a>), research on its waterlogging response is lacking.</p>
<p>The chromosome number of <em>Zoysia</em> species has been identified as 2<em>n</em> = 40 based on experimental evidence (Forbes, <a href="#tpg270070-bib-0026" class="usa-link" aria-describedby="tpg270070-bib-0026">1952</a>; Kole, <a href="#tpg270070-bib-0041" class="usa-link" aria-describedby="tpg270070-bib-0041">2011</a>; Loch et al., <a href="#tpg270070-bib-0051" class="usa-link" aria-describedby="tpg270070-bib-0051">2017</a>). Although the chloroplast genome <em>Z. sinica</em> has been sequenced and used in phylogenetic studies (Cheon et al., <a href="#tpg270070-bib-0018" class="usa-link" aria-describedby="tpg270070-bib-0018">2021</a>), a chromosomal‐level genome assembly has not yet been reported. <em>Z. sinica</em> was classified as being closely related to <em>Zoysia japonica</em>. In the genus <em>Zoysia</em>, the genomes of <em>Z. japonica</em>, <em>Zoysia matrella</em>, and <em>Zoysia pacifica</em> were assembled in 2016 (Tanaka et al., <a href="#tpg270070-bib-0082" class="usa-link" aria-describedby="tpg270070-bib-0082">2016</a>). The <em>Z. japonica</em> genome was assembled at the chromosomal level by scaffolding with the <em>Z. japonica</em> ‘El Toro’ linkage map of restriction site‐associated DNA markers (F. Wang et al., <a href="#tpg270070-bib-0087" class="usa-link" aria-describedby="tpg270070-bib-0087">2015</a>). Comparative genomic analyses revealed that expanded cytochrome P450 and ABA biosynthetic gene families and many duplicated genes may contribute to salt adaptation in <em>Zoysia</em> species (W. Wang et al., <a href="#tpg270070-bib-0088" class="usa-link" aria-describedby="tpg270070-bib-0088">2022</a>). To understand the differences between <em>Zoysia</em> species, the availability of a new, high‐quality genome will allow access to novel genetic variants.</p>
<p>Here, we present a high‐quality chromosome‐level genome assembly and landscape transcriptomic analysis across a gradient from terrestrial areas to different degrees of tidal inundation, examining both shoot and root tissues of <em>Z. sinica</em>. We analyzed the overall genomic features and gene family evolution of <em>Z. sinica</em> to infer the reasons for its adaptation to the intertidal zone and examined the gene‐level responses in actual environments through transcriptome analysis. We identified that genes associated with suberin biosynthesis in the shoots of <em>Z. sinica</em> and genes related to nitrate metabolism in the roots were significantly upregulated under waterlogging environments, suggesting these as key genes for adaptation. Our findings provide insights into the various strategies involved in the adaptation of salt marsh plants and offer a foundation for breeding waterlogging stress‐tolerant turfgrass varieties.</p></section><section id="tpg270070-sec-0020"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270070-sec-0030"><h3 class="pmc_sec_title">2.1. Plant materials</h3>
<p>
<em>Z. sinica</em> was collected from Gwangpo Bay, Sacheon, South Korea. Gwangpo Bay (35.03410° N, 127.97671° E), a geographically intertidal zone, is the largest residential area of <em>Z. sinica</em> in Korea. For transcriptome research, <em>Z. sinica</em> was collected from three locations with different waterlogged environments: the high intertidal zone (HIZ) and low intertidal zone (LIZ) of Gwangpo Bay, and the Forest Biomaterials Research Center, National Institute of Forest Science (Sacheon turfgrass genetic resources conservation field) as an inland (IL) site (35°6′14.54″ N, 127°58′13.45″ E), where it is maintained under the accession number ZN2289. The collected samples included shoot (comprising leaves and sheath) and root tissues to ensure comprehensive transcriptomic coverage. Immediately after collection, the samples were flash‐frozen in liquid nitrogen and stored at −80°C until further processing.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270070-sec-0040"><h4 class="pmc_sec_title">Core Ideas</h4>
<ul id="tpg270070-list-0001" class="list" style="list-style-type:disc">
<li><p>This study presents the first chromosome‐level genome assembly of <em>Zoysia sinica</em> using Oxford Nanopore long reads and Omni‐C data, which provides a foundational genomic resource.</p></li>
<li><p>We provided a re‐annotation of genomes for <em>Zoysia japonica</em>, <em>Zoysia matrella</em>, and <em>Zoysia pacifica</em>, enhancing gene annotation accuracy and comparative analyses.</p></li>
<li><p>Our findings offer insights into intertidal adaptation of <em>Z. sinica</em>, highlighting genes involved in nitrogen metabolism, ethylene response, and suberin biosynthesis under waterlogging stress.</p></li>
</ul></section></section></section><section id="tpg270070-sec-0050"><h3 class="pmc_sec_title">2.2. Library construction and sequencing</h3>
<p>Genomic DNA was extracted from the leaves using Exgene Plant SV (GeneAll Biotechnology) according to the manufacturer's protocols. To extract total RNA from the shoot and root samples, the SmartGene Plant RNA Extraction Kit was used according to the manufacturer's protocol. For Oxford Nanopore long‐read sequencing, we constructed a genomic DNA library using the ONT ligation sequencing kit (SQK‐LSK110) according to the manufacturer's protocols. Sequencing was performed using flow cells (R9.4.1) on a Minion sequencer. For genomic DNA sequencing, a paired‐end library with an insert size of 150 bp was constructed using the xGen DNA Library Prep EZ UNI Kit. For RNA sequencing, mRNA was enriched using the Poly(A) RNA Selection Kit (Lexogen). The enriched mRNA was then subjected to 150‐bp paired‐end library preparation using the xGen RNA Lib Prep kit (Integrated DNA Technologies). All Illumina libraries were sequenced on the Illumina NovaSeq 6000 platform.</p>
<p>For the preparation of Omni‐C libraries, the Dovetail Omni‐C Kit (Dovetail Genomics) was used according to the manufacturer's instructions. Approximately 300 mg of <em>Z. sinica</em> leaf tissue was homogenized to a fine powder in liquid nitrogen. The powdered tissue was then subjected to crosslinking by adding formaldehyde to a final concentration of 37% and incubated for 10 min at room temperature. Chromatin was then digested in situ with DNase I provided in the kit, and the resulting DNA fragments were end‐repaired, ligated, and purified. The ligated DNA was then subjected to proximity ligation to create long‐range interactions, followed by reverse crosslinking to release the DNA. The resulting DNA was purified and quantified using Qubit (Thermo Fisher Scientific). The purified DNA was used to construct Omni‐C libraries using the Dovetail Library Module for Illumina (Dovetail Cat. No. 21004). Libraries were indexed using the Dovetail Dual Index Primer Set for Illumina (Dovetail Cat. No. 25005) and amplified according to the manufacturer's protocol. The quality and quantity of the libraries were assessed using Qubit 4.0 Fluorometer (Invitrogen Ltd) and 4150 TapeStation (Agilent), respectively. Sequencing of the library was performed on an Illumina NovaSeq 6000 platform with paired‐end 151‐bp reads.</p></section><section id="tpg270070-sec-0060"><h3 class="pmc_sec_title">2.3. Genome estimation, assembly, and scaffolding</h3>
<p>The Illumina reads were cleaned using Trimmomatic v0.39 (Bolger et al., <a href="#tpg270070-bib-0007" class="usa-link" aria-describedby="tpg270070-bib-0007">2014</a>). The <em>k</em>‐mer frequency analysis and estimation of genome size were performed using jellyfish v2.3.0, with the <em>k</em>‐mer size option 19 (Marçais &amp; Kingsford, <a href="#tpg270070-bib-0056" class="usa-link" aria-describedby="tpg270070-bib-0056">2011</a>), and Genomescope (Vurture et al., <a href="#tpg270070-bib-0086" class="usa-link" aria-describedby="tpg270070-bib-0086">2017</a>), respectively, using Illumina reads. Nanopore reads were corrected for Illumina reads using LoRDEC, with the <em>k</em>‐mer size option 19 (Salmela &amp; Rivals, <a href="#tpg270070-bib-0072" class="usa-link" aria-describedby="tpg270070-bib-0072">2014</a>). The corrected Nanopore reads were assembled into contigs using NextDeNovo v2.5.0 with options ‐k19 and ‐w19 (Hu et al., <a href="#tpg270070-bib-0035" class="usa-link" aria-describedby="tpg270070-bib-0035">2024</a>). The assembled contigs were scaffolded into 20 chromosomes using the Omni‐C data, processed with Juicer v1.6 (Durand et al., <a href="#tpg270070-bib-0022" class="usa-link" aria-describedby="tpg270070-bib-0022">2016</a>) and the 3D‐DNA (v180922) pipeline (<a href="https://github.com/aidenlab/3d-dna" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/aidenlab/3d‐dna</a>) (Dudchenko et al., <a href="#tpg270070-bib-0020" class="usa-link" aria-describedby="tpg270070-bib-0020">2017</a>). Additionally, gaps in the genome were closed using the Nanopore reads with YAGCloser (<a href="https://github.com/merlyescalona/yagcloser" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/merlyescalona/yagcloser</a>), ensuring a more complete and accurate genome assembly. Finally, the chromosomal order was adjusted using the previously reported chromosome‐level genome of the closely related <em>Z. japonica</em> (F. Wang et al., <a href="#tpg270070-bib-0087" class="usa-link" aria-describedby="tpg270070-bib-0087">2015</a>). The Omni‐C contact map was visualized using Juicebox. Assembly statistics were calculated using Quast v5.1.0 (Gurevich et al., <a href="#tpg270070-bib-0032" class="usa-link" aria-describedby="tpg270070-bib-0032">2013</a>). To verify the genome coverage and quality of the assembled genome sequences, 4896 Poales orthologs were aligned using benchmarking universal single‐copy orthologs (BUSCOs) v5.2.2 (Manni et al., <a href="#tpg270070-bib-0054" class="usa-link" aria-describedby="tpg270070-bib-0054">2021</a>). Additionally, the completeness of the genome was assessed based on 19‐mers from Illumina reads using meryl v1.3 and merqury v1.3 3 (Rhie et al., <a href="#tpg270070-bib-0070" class="usa-link" aria-describedby="tpg270070-bib-0070">2020</a>). Cleaned Illumina reads were also mapped to the genome using BWA v0.7.17‐r1188 (H. Li, <a href="#tpg270070-bib-0046" class="usa-link" aria-describedby="tpg270070-bib-0046">2013</a>) to further support the completeness assessment.</p></section><section id="tpg270070-sec-0070"><h3 class="pmc_sec_title">2.4. Genome annotation</h3>
<p>We identified known and novel repetitive elements in the <em>Z. sinica</em> genome using RepeatModeler v2.0.4 (<a href="http://www.repeatmasker.org/RepeatModeler/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.repeatmasker.org/RepeatModeler/</a>) (Flynn et al., <a href="#tpg270070-bib-0025" class="usa-link" aria-describedby="tpg270070-bib-0025">2020</a>) with default parameters to build a database. Prediction and masking of repeat sequences were performed using RepeatMasker v4.1.5 (<a href="http://www.repeatmasker.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.repeatmasker.org/</a>). Telomeric regions of the <em>Z. sinica</em> genome were identified using the TeloExplorer module of QuarTeT (v1.2.0) with the “‐c plant” option. For centromere prediction, the CentroMiner module was utilized, incorporating transposable element annotation data to detect tandem repeat monomers. Potential centromeric repeats were filtered based on repeat period and copy number, followed by clustering to minimize redundancy. The identified regions were visualized using RIdeogram. For gene annotation, a genome with soft‐masked repeated sequences was used. The BRAKER3 pipeline (Gabriel et al., <a href="#tpg270070-bib-0029" class="usa-link" aria-describedby="tpg270070-bib-0029">2024</a>; G. Pertea &amp; Pertea, <a href="#tpg270070-bib-0065" class="usa-link" aria-describedby="tpg270070-bib-0065">2020</a>; Quinlan, <a href="#tpg270070-bib-0067" class="usa-link" aria-describedby="tpg270070-bib-0067">2014</a>) was employed to annotate the protein‐coding genes of <em>Z. sinica</em> using two different hints: short‐read RNA sequencing (RNA‐seq) data and protein homology information. PRINSEQ‐lite v0.20.4 was used to trim and filter short RNA‐seq reads with the following parameters: min len 50, min qual score 10, min qual mean 20, derep 14, trim qual left 20, and trim qual right 20 (Schmieder &amp; Edwards, <a href="#tpg270070-bib-0073" class="usa-link" aria-describedby="tpg270070-bib-0073">2011</a>). Cleaned reads were aligned to the genome using HISAT2 v2.2.1 (Kim et al., <a href="#tpg270070-bib-0040" class="usa-link" aria-describedby="tpg270070-bib-0040">2019</a>), and the resulting mapping files were supplied to the BRAKER3 pipeline along with protein sequences from 27 species within the class Liliopsida, downloaded from NCBI (Table <a href="#tpg270070-supinfo-0001" class="usa-link">S1</a>). The hints were used for training GeneMark‐ETP and AUGUSTUS to predict genes (Bruna et al., <a href="#tpg270070-bib-0010" class="usa-link" aria-describedby="tpg270070-bib-0010">2024</a>).</p>
<p>The functional annotation of protein‐coding genes was performed using EnTAP v1.1.1 (Hart et al., <a href="#tpg270070-bib-0034" class="usa-link" aria-describedby="tpg270070-bib-0034">2020</a>). Protein sequences were analyzed against the NCBI RefSeq database (O'Leary et al., <a href="#tpg270070-bib-0063" class="usa-link" aria-describedby="tpg270070-bib-0063">2015</a>) and the Uniprot database using DIAMOND (Buchfink et al., <a href="#tpg270070-bib-0011" class="usa-link" aria-describedby="tpg270070-bib-0011">2015</a>) with an <em>e</em>‐value cutoff of 1<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-1" display="inline"><mrow><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></math></span>. Additionally, genes were annotated with KEGG (Kyoto Encyclopedia of Genes and Genomes) terms and gene ontology terms using eggNOG‐mapper (Cantalapiedra et al., <a href="#tpg270070-bib-0012" class="usa-link" aria-describedby="tpg270070-bib-0012">2021</a>; The Gene Ontology Consortium, <a href="#tpg270070-bib-0084" class="usa-link" aria-describedby="tpg270070-bib-0084">2014</a>; Huerta‐Cepas et al., <a href="#tpg270070-bib-0036" class="usa-link" aria-describedby="tpg270070-bib-0036">2017</a>, <a href="#tpg270070-bib-0037" class="usa-link" aria-describedby="tpg270070-bib-0037">2018</a>; Kanehisa et al., <a href="#tpg270070-bib-0039" class="usa-link" aria-describedby="tpg270070-bib-0039">2016</a>).</p>
<p>To ensure uniformity and accurate gene prediction, re‐annotation analyses for <em>Z. japonica</em>, <em>Z. matrella</em>, and <em>Z. pacifica</em> were performed using protein sequences from 27 species within the class Liliopsida. For <em>Z. japonica</em> and <em>Z. matrella</em>, transcriptome data were additionally downloaded from NCBI (Table <a href="#tpg270070-supinfo-0001" class="usa-link">S2</a>) and integrated into the re‐annotation process. For <em>Z. pacifica</em>, re‐annotation was carried out using the BRAKER2 pipeline with GeneMark‐EP+ and AUGUSTUS, as no RNA‐seq data have been reported for this species (Brůna et al., <a href="#tpg270070-bib-0009" class="usa-link" aria-describedby="tpg270070-bib-0009">2021</a>). The genome annotation was visualized using Circos (Krzywinski et al., <a href="#tpg270070-bib-0043" class="usa-link" aria-describedby="tpg270070-bib-0043">2009</a>), and intragenomic gene synteny was detected using MCScan software with parameters –cscore 0.99 and –minspan = 30 (Tang et al., <a href="#tpg270070-bib-0083" class="usa-link" aria-describedby="tpg270070-bib-0083">2024</a>).</p></section><section id="tpg270070-sec-0080"><h3 class="pmc_sec_title">2.5. Phylogenomic analysis and divergence time estimation</h3>
<p>Genomic data for the 11 plant species were downloaded from various sources: <em>Arabidopsis thaliana</em> (TAIR10), <em>Brachypodium distachyon</em> (v3.2), <em>Oropetium thomaeum</em> (v1.0), <em>Sorghum bicolor</em> (v5.1), <em>Setaria italica</em> (v2.2), and <em>Zea mays</em> (RefGen_V4) were downloaded from Phytozome 13; <em>Oryza sativa</em> (IRGSP‐1.0) was downloaded from Ensembl (Yates et al., <a href="#tpg270070-bib-0093" class="usa-link" aria-describedby="tpg270070-bib-0093">2021</a>); <em>S. alterniflora</em> was downloaded from the National Genomics Data Center under project numbers PRJCA016599 (Hao et al., <a href="#tpg270070-bib-0033" class="usa-link" aria-describedby="tpg270070-bib-0033">2024</a>); and <em>Z. japonica</em>, <em>Z. matrella</em>, and <em>Z. pacifica</em> were downloaded from the Zoysia Genome Database (<a href="http://zoysia.kazusa.or.jp/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://zoysia.kazusa.or.jp/</a>) (Tanaka et al., <a href="#tpg270070-bib-0082" class="usa-link" aria-describedby="tpg270070-bib-0082">2016</a>). Using OrthoFinder v2.5.5 with the ‐M msa option (Emms &amp; Kelly, <a href="#tpg270070-bib-0023" class="usa-link" aria-describedby="tpg270070-bib-0023">2018</a>, <a href="#tpg270070-bib-0024" class="usa-link" aria-describedby="tpg270070-bib-0024">2019</a>), we clustered gene families for the 12 plant species. We generated a rooted species tree from a concatenated alignment of single‐copy orthologs using the STAG method in OrthoFinder. The divergence times of <em>Z. sinica</em> were inferred using this tree with MCMCtree in PAML v4.10.7, employing the JC69 model, a burn‐in of 5,000,000, a sampling frequency of 30, and 10,000,000 samples (Rannala &amp; Yang, <a href="#tpg270070-bib-0069" class="usa-link" aria-describedby="tpg270070-bib-0069">2007</a>). Calibration points were sourced from the TimeTree database (<a href="https://www.timetree.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.timetree.org</a>) using nwkit (Fukushima &amp; Pollock, <a href="#tpg270070-bib-0028" class="usa-link" aria-describedby="tpg270070-bib-0028">2023</a>). The resulting phylogenetic tree with divergence times was visualized using MCMCtreeR v1.1 (Puttick, <a href="#tpg270070-bib-0066" class="usa-link" aria-describedby="tpg270070-bib-0066">2019</a>).</p>
<p>To infer whole‐genome duplication (WGD) events, we performed protein homology analysis across seven species using DIAMOND v2.1.9 with an <em>e</em>‐value cutoff of &lt;1<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-2" display="inline"><mrow><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></math></span> (Buchfink et al., <a href="#tpg270070-bib-0011" class="usa-link" aria-describedby="tpg270070-bib-0011">2015</a>). Collinear blocks were identified using MCScanX (Y. Wang et al., <a href="#tpg270070-bib-0089" class="usa-link" aria-describedby="tpg270070-bib-0089">2012</a>), where regions containing at least five homologous genes were classified as collinear. Synonymous substitution rates (Ks) values for collinear orthologous gene pairs were calculated using the add_ka_and_ks_to_collinearity.pl script implemented in MCScanX. Visualization was conducted using Python libraries, including pandas, matplotlib, numpy, and seaborn.</p></section><section id="tpg270070-sec-0090"><h3 class="pmc_sec_title">2.6. Gene family expansion and contraction analyses</h3>
<p>We analyzed gene family evolution, including expansion and contraction, for the genomes of <em>Z. sinica</em> and 11 other plant species. All input data used the protein sequences of primary transcripts. Using the gene family clusters and phylogenetic tree topology generated by OrthoFinder, we conducted the analysis with CAFE5 v1.1 (Mendes et al., <a href="#tpg270070-bib-0057" class="usa-link" aria-describedby="tpg270070-bib-0057">2020</a>). CAFE employs a random birth and death model to estimate the size of gene families at each ancestral node. The significance of gene family expansion or contraction was determined with a cutoff <em>p</em>‐value &lt; 0.01.</p></section><section id="tpg270070-sec-0100"><h3 class="pmc_sec_title">2.7. Identification of positively selected genes (PSGs)</h3>
<p>We conducted a selection pressure analysis on the 265 single‐copy orthologous gene sets retrieved from the genomes of seven species—<em>O. sativa</em>, <em>O. thomaeum</em>, <em>S. alterniflora</em>, <em>Z. matrella</em>, <em>Z. japonica</em>, <em>Z. pacifica</em>, and <em>Z. sinica</em>—using OrthoFinder. To prevent out‐of‐frame indels, we first aligned the translated protein sequences and then used these protein alignments to generate the codon alignments, utilizing PAL2NAL (Suyama et al., <a href="#tpg270070-bib-0080" class="usa-link" aria-describedby="tpg270070-bib-0080">2006</a>) and TranslatorX (Abascal et al., <a href="#tpg270070-bib-0001" class="usa-link" aria-describedby="tpg270070-bib-0001">2010</a>). The alignments were then trimmed using Gblocks (Castresana, <a href="#tpg270070-bib-0013" class="usa-link" aria-describedby="tpg270070-bib-0013">2000</a>) and converted into PAML input format. For the branches of <em>S. alterniflora</em> and <em>Z. sinica</em> that inhabit intertidal zones, as well as the branch specific to <em>Zoysia species</em>, we identified PSGs (Álvarez‐Carretero et al., <a href="#tpg270070-bib-0003" class="usa-link" aria-describedby="tpg270070-bib-0003">2023</a>). Subsequently, the branch‐site model (setting model = 2, Nssites = 2) was employed to determine the frequency of positive selection in each lineage. Two models were fitted: the alternative model with fix_omega = 0 and omega = 2, and the null model with fix_omega = 1 and omega = 1. These models were compared using a likelihood ratio test with 1 degree of freedom. Genes with a <em>p</em>‐value below 0.05 were classified as PSGs.</p></section><section id="tpg270070-sec-0110"><h3 class="pmc_sec_title">2.8. Gravimetric water content (GWC) and porosity measurements of soil</h3>
<p>Soil samples were collected from the LIZ, HIZ, and the Forest Biomaterials Research Center, National Institute of Forest Science (Sacheon turfgrass genetic resources conservation field) as an IL site (35°6′14.54″ N, 127°58′13.45″ E). A 100‐mL core sampler was used to analyze soil physical properties. At each location, the top 1–2 cm of soil was removed, and 500 g samples were collected from a depth of 10 cm in five replicates. The samples were immediately sealed to prevent any changes in soil moisture before analysis. Approximately 500 g of soil was collected from each location. The GWC of the soil samples was determined following standard protocols (Robinson et al., <a href="#tpg270070-bib-0071" class="usa-link" aria-describedby="tpg270070-bib-0071">2008</a>). Each soil sample was first weighed to obtain fresh weight (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-3" display="inline"><mrow><msub><mi>W</mi><mi mathvariant="normal">f</mi></msub></mrow></math></span>). The samples were then dried in an oven at 70°C for 48 h to a constant weight, after which they were weighed again to determine the dry weight (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline"><mrow><msub><mi>W</mi><mi mathvariant="normal">d</mi></msub></mrow></math></span>). GWC was calculated using the following formula:
</p>
<table class="disp-formula p" id="tpg270070-disp-0001"><tr><td class="formula"><math id="jats-math-5" display="block"><mrow><mrow><mrow><mi>GWC</mi><mspace width="0.28em"></mspace></mrow><mfenced open="(" close=")"><mo>%</mo></mfenced><mo linebreak="badbreak">=</mo><mfrac><mrow><msub><mi>W</mi><mi mathvariant="normal">f</mi></msub><mo>−</mo><msub><mi>W</mi><mi mathvariant="normal">d</mi></msub></mrow><msub><mi>W</mi><mi mathvariant="normal">d</mi></msub></mfrac><mo linebreak="goodbreak">×</mo><mn>100</mn></mrow></mrow></math></td></tr></table>
<p>Soil porosity was measured using the bulk density method. The bulk density (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline"><mrow><msub><mi mathvariant="normal">ρ</mi><mi mathvariant="normal">b</mi></msub></mrow></math></span>) was calculated as the dry weight of the soil (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline"><mrow><msub><mi>W</mi><mi mathvariant="normal">d</mi></msub></mrow></math></span>) divided by its volume. The particle density (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-8" display="inline"><mrow><msub><mi mathvariant="normal">ρ</mi><mi mathvariant="normal">p</mi></msub></mrow></math></span>) was assumed to be 2.65 g/cm<sup>3</sup>, a standard value for mineral soils. Soil porosity (<em>φ</em>) was calculated using the following equation: 
</p>
<table class="disp-formula p" id="tpg270070-disp-0002"><tr><td class="formula"><math id="jats-math-9" display="block"><mrow><mrow><mrow><mi>Porosity</mi><mspace width="0.28em"></mspace></mrow><mfenced open="(" close=")"><mo>%</mo></mfenced><mo linebreak="badbreak">=</mo><mfenced separators="" open="(" close=")"><mrow><mn>1</mn><mo>−</mo><mfrac><msub><mi mathvariant="normal">ρ</mi><mi mathvariant="normal">b</mi></msub><msub><mi mathvariant="normal">ρ</mi><mi mathvariant="normal">p</mi></msub></mfrac></mrow></mfenced><mo linebreak="goodbreak">×</mo><mn>100</mn></mrow></mrow></math></td></tr></table>
<p>The GWC and porosity measurements were conducted in quintuplicate for each sample location. Statistical significance between the different locations was assessed using analysis of variance (ANOVA), followed by Tukey's post hoc test, with a <em>p</em>‐value of &lt;0.05 considered significant.</p></section><section id="tpg270070-sec-0120"><h3 class="pmc_sec_title">2.9. Gene expression analysis</h3>
<p>For transcriptome analysis, shoot and root tissues were collected from three locations: LIZ, where roots are waterlogged during low tide; HIZ, where roots are not waterlogged during low tide; and IL (Figure <a href="#tpg270070-fig-0003" class="usa-link">3A</a>). The reads were cleaned using PRINSEQ‐lite v0.20.4 (Schmieder &amp; Edwards, <a href="#tpg270070-bib-0073" class="usa-link" aria-describedby="tpg270070-bib-0073">2011</a>) and Trimmomatic v0.39 (Bolger et al., <a href="#tpg270070-bib-0007" class="usa-link" aria-describedby="tpg270070-bib-0007">2014</a>). The cleaned reads were then aligned to the <em>Z. sinica</em> genome using HISAT2 v2.2.1 (Kim et al., <a href="#tpg270070-bib-0040" class="usa-link" aria-describedby="tpg270070-bib-0040">2019</a>). Gene expression levels were quantified with StringTie v2.2.1 based on transcripts per million (TPM) (M. Pertea et al., <a href="#tpg270070-bib-0064" class="usa-link" aria-describedby="tpg270070-bib-0064">2016</a>). Principal component analysis (PCA) was performed using PtR to explore relationships between sample replicates. Differentially expressed genes (DEGs) were identified using edgeR with trimmed mean of M‐values‐normalized TPM counts (|log2(FoldChange)| &gt; 2 and <em>p</em>‐value &lt; 0.001). For GOBP (gene ontology in terms of biological process) and KEGG functional enrichment analysis, a BLASTp homology search was conducted with an <em>e</em>‐value cutoff of 1<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-10" display="inline"><mrow><msup><mi mathvariant="normal">e</mi><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></math></span> between <em>Z. sinica</em> and <em>O. sativa</em> (Altschul et al., <a href="#tpg270070-bib-0002" class="usa-link" aria-describedby="tpg270070-bib-0002">1990</a>). GOBP and KEGG functional enrichment analyses were performed using DAVID (Sherman et al., <a href="#tpg270070-bib-0076" class="usa-link" aria-describedby="tpg270070-bib-0076">2022</a>), and the significant results (<em>p</em>‐value &lt; 0.05) were visualized using TBtools (C. Chen et al., <a href="#tpg270070-bib-0014" class="usa-link" aria-describedby="tpg270070-bib-0014">2020</a>). Gene set enrichment analysis (GSEA) was performed using GSEA v4.3.3 (Mootha et al., <a href="#tpg270070-bib-0058" class="usa-link" aria-describedby="tpg270070-bib-0058">2003</a>; Subramanian et al., <a href="#tpg270070-bib-0078" class="usa-link" aria-describedby="tpg270070-bib-0078">2005</a>). Expression datasets were ranked based on the signal‐to‐noise ratio across the sampling locations (LIZ, HIZ, and IL), and enrichment scores were computed for predefined gene sets. The ES values were then normalized to generate normalized enrichment scores (NESs). The statistical significance of the enrichment was assessed using 1000 permutations with the “gene set” option to account for variability in gene set composition.</p>
<figure class="fig xbox font-sm" id="tpg270070-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed98/12238702/cbe110ee8f7d/TPG2-18-e70070-g001.jpg" loading="lazy" id="jats-graphic-5" height="564" width="800" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270070-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Transcriptome analysis of <em>Z. sinica</em> shoots and roots in intertidal environments. (A) Diagram of the geographic distribution of <em>Z. sinica</em>. (B–D) Gene ontology in terms of biological process (GOBP) results are shown in the upper subplots, and Kyoto Encyclopedia of Genes and Genomes (KEGG) enrichment analyses are shown in the lower subplots. The figure presents the enrichment analysis of upregulated differentially expressed genes (DEGs) in the low intertidal zone (LIZ) compared to the high intertidal zone (HIZ) (B) and inland (IL) (C). (D) Enrichment analysis of upregulated DEGs in HIZ compared to IL is also shown.</p></figcaption></figure></section><section id="tpg270070-sec-0130"><h3 class="pmc_sec_title">2.10. Quantitative reverse transcriptase polymerase chain reaction (qRT‐PCR)</h3>
<p>For RNA‐seq validation, 1 µg of total RNA was converted to cDNA using the SensiFAST cDNA Synthesis Kit (Meridian Bioscience) following the manufacturer's instructions. The synthesized cDNA was used for qRT‐PCR with the SensiFAST SYBR Lo‐ROX Kit (Meridian Bioscience) on the CFX Opus 96 Real‐Time PCR System (Bio‐Rad Laboratories, Inc.), using <em>HTA11</em> as the housekeeping gene, which exhibited uniform trimmed mean of M‐values‐normalized TPM values across all conditions. Primer information used in this study is listed in Table <a href="#tpg270070-supinfo-0001" class="usa-link">S3</a>. qRT‐PCR was performed under the following conditions: denaturation at 95°C for 2 min, followed by 40 cycles of 95°C for 5 s, 63°C for 10 s, and 72°C for 20 s. The Cq values obtained from qRT‐PCR were converted to ΔΔCq values for comparative analysis with RNA‐seq data (Livak &amp; Schmittgen, <a href="#tpg270070-bib-0050" class="usa-link" aria-describedby="tpg270070-bib-0050">2001</a>). Correlation analysis between qRT‐PCR and RNA‐seq was performed using Python libraries, including pandas, numpy, and scipy.stats for statistical calculations, and matplotlib and seaborn for data visualization.</p></section></section><section id="tpg270070-sec-0140"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270070-sec-0150"><h3 class="pmc_sec_title">3.1. Genome sequencing and assembly</h3>
<p>
<em>Z. sinica</em>, a species of perennial grasses, was collected from the Gwangpo Bay, an estuarine intertidal zone where freshwater and saltwater meet, and is the largest colony of <em>Z. sinica</em> in South Korea (Figure <a href="#tpg270070-fig-0001" class="usa-link">1A,B</a>). The <em>Z. sinica</em> genome size was estimated to be 328.2 Mb using a total of 19.06 Gb of Illumina reads (Table <a href="#tpg270070-supinfo-0001" class="usa-link">S4</a>; Figure <a href="#tpg270070-supinfo-0001" class="usa-link">S1</a>). The <em>k</em>‐mer (<em>k</em> = 19) distribution exhibited nearly one peak, indicating a low proportion of heterozygosity in the <em>Z. sinica</em> genome. To obtain long‐read sequences that covered at least an 80× coverage depth against the estimated genome size, we produced 27.27 Gb of long‐read data with 16 kb of N50 using the Oxford Nanopore Minion sequencer (Table <a href="#tpg270070-supinfo-0001" class="usa-link">S5</a>).</p>
<figure class="fig xbox font-sm" id="tpg270070-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed98/12238702/f4392aa405a7/TPG2-18-e70070-g005.jpg" loading="lazy" id="jats-graphic-1" height="654" width="700" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270070-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Morphological and genomic characteristics of <em>Z. sinica</em>. (A) Geographic and environmental distribution of <em>Z. sinica</em> from the intertidal zone in Gangpo Bay, Korea. (B) The appearance of <em>Z. sinica</em> growing in the low intertidal zone. (C) The Omni‐C contact map for the 20 chromosomes of <em>Z. sinica</em>. (D) Genomic feature distribution in the <em>Z. sinica</em> genome. Tracks from the outside to the inside represent protein‐coding genes, long terminal repeat (LTR) Gypsy, LTR Copia, and intragenomic synteny based on homologous genes.</p></figcaption></figure><p>The <em>Z. sinica</em> genome was assembled using a hybrid pipeline that combined Nanopore and Illumina reads. Illumina‐based corrected Nanopore long reads, providing an 83.8× coverage of the estimated genome size, were assembled using NextDenovo. We obtained a de novo assembled genome consisting of 73 contigs with a total length of 315.7 Mb (Table <a href="#tpg270070-tbl-0001" class="usa-link">1</a>). Based on valid Omni‐C chromatin interaction data (Figure <a href="#tpg270070-fig-0001" class="usa-link">1C</a>), the contig N50 value of 10.4 Mb was significantly improved to a scaffold N50 value of 18.9 Mb. Of the 315.7 Mb contig sequences, 312.7 Mb were scaffolded and anchored onto 20 chromosomes. Gap‐closing analysis further refined the assembly, closing three gaps and resulting in the final genome with 54 remaining gaps spanning a total of 27 Kb. The 20 chromosome‐level <em>Z. sinica</em> genomes were ordered into subgenome A (odd‐numbered chromosomes) and subgenome B (even‐numbered chromosomes) using the <em>Z. japonica</em> genome as a reference (Table <a href="#tpg270070-supinfo-0001" class="usa-link">S6</a>). The seven‐base telomeric repeat (CCCTAAA/TTTAGGG), which is characteristic of plants, could not be identified in the <em>Z. sinica</em> genome. However, tandem repeat monomers were identified and used to predict centromeric regions. The candidate centromeres were located at comparable positions in subgenomes A and B, suggesting structural similarity between the subgenomes (Figure <a href="#tpg270070-supinfo-0001" class="usa-link">S2</a>).</p>
<section class="tw xbox font-sm" id="tpg270070-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Major assembly features of the <em>Z. sinica</em> (2<em>n</em> = 4<em>x</em> = 40) genome.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Category</th>
<th align="left" rowspan="1" colspan="1">
<em>Z. sinica</em> genome</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Sequencing platform</td>
<td align="left" rowspan="1" colspan="1">Illumina, Oxford Nanopore</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Estimated genome size (Mb)</td>
<td align="left" rowspan="1" colspan="1">328.2</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Total contigs</td>
<td align="left" rowspan="1" colspan="1">73</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Total contig length (Mb)</td>
<td align="left" rowspan="1" colspan="1">315.7</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Contig N50 (Mb)</td>
<td align="left" rowspan="1" colspan="1">10.4</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Total scaffolds</td>
<td align="left" rowspan="1" colspan="1">20</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Total scaffold length (Mb)</td>
<td align="left" rowspan="1" colspan="1">312.67</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Scaffold N50 (Mb)</td>
<td align="left" rowspan="1" colspan="1">18.9</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">GC (%)</td>
<td align="left" rowspan="1" colspan="1">43.97</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Complete BUSCOs (%)</td>
<td align="left" rowspan="1" colspan="1">97.7</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Complete and single‐copy BUSCOs (%)</td>
<td align="left" rowspan="1" colspan="1">73.6</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Complete and duplicated BUSCOs (%)</td>
<td align="left" rowspan="1" colspan="1">24.1</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Fragmented BUSCOs (%)</td>
<td align="left" rowspan="1" colspan="1">0.5</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Missing BUSCOs (%)</td>
<td align="left" rowspan="1" colspan="1">1.8</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<em>k</em>‐mer completeness (%)</td>
<td align="left" rowspan="1" colspan="1">86.81</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Base pair QV</td>
<td align="left" rowspan="1" colspan="1">35.38</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Annotation</td>
<td align="left" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td style="padding-left:10%" align="left" rowspan="1" colspan="1">Repeat sequences (%)</td>
<td align="left" rowspan="1" colspan="1">42.15</td>
</tr>
<tr>
<td style="padding-left:10%" align="left" rowspan="1" colspan="1">Number of protein‐coding genes</td>
<td align="left" rowspan="1" colspan="1">29,551</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270070-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270070-tbl1-note-0001"><p>Abbreviations: BUSCOs, benchmarking universal single‐copy orthologs; QV, quality value.</p></div></div></section><p>For genome quality assessment, the integrity of the <em>Z. sinica</em> genome was assessed by comparing its homology with the poales BUSCO genes. Out of 4896 core poales genes, 4783 (97.7%) complete genes were found in the <em>Z. sinica</em> genome, with 3604 (73.6%) being single‐copy genes and 1179 (24.1%) being duplicated genes (Table <a href="#tpg270070-tbl-0001" class="usa-link">1</a>). Additionally, results from merqury showed that the <em>k</em>‐mer completeness was 86.81%, with a quality value of 35.38. Furthermore, Illumina reads mapping supported the high completeness of the genome, with 122,682,537 reads (97.69%) mapped, including 121,556,866 reads (96.79%) that were primary mapped. We then compared our newly assembled <em>Z. sinica</em> genome with previously reported <em>Zoysia</em> genomes including those of <em>Z. japonica</em>, <em>Z. matrella</em>, and <em>Z. pacifica</em> whose assembly sizes range from 315.7 to 563.4 Mb (Table <a href="#tpg270070-supinfo-0001" class="usa-link">S7</a>). Notably, the <em>Z. sinica</em> genome has fewer contigs and a higher contig N50, with a scaffold N50 of 18.9 Mb and scaffold coverage of 99%. Excluding the contig‐level assemblies of <em>Z. matrella</em> and <em>Z. pacifica</em>, <em>Z. sinica</em> exhibits the highest genome coverage at 95.3%. Collectively, these results suggest that the <em>Z. sinica</em> genome exhibits excellent contiguity and is well‐suited for genome annotation and further studies.</p></section><section id="tpg270070-sec-0160"><h3 class="pmc_sec_title">3.2. Genome annotation of <em>Z. sinica</em> and other <em>Zoysia</em> species</h3>
<p>To identify repetitive elements in the <em>Z. sinica</em> genome, we predicted both known and novel repetitive elements. The repetitive element prediction analysis indicated that 42.15% of the genomic sequences were annotated (Table <a href="#tpg270070-supinfo-0001" class="usa-link">S8</a>). Among the repetitive elements, retroelements and DNA transposons accounted for 23.27% and 2.62% of the genome sequence, respectively. Long terminal repeat retrotransposons were predominant because Ty1/Copia and Gypsy/DIRS1 covered 6.85% and 13.23% of the genome, respectively. In addition, 15.34% of the genome was unclassified interspersed repeats.</p>
<p>To predict the protein‐coding genes in the masked <em>Z. sinica</em> genome, we combined ab initio‐ and homology‐based prediction strategies using the RNA‐seq data of <em>Z. sinica</em> and the protein sets of class Liliopsida (Table <a href="#tpg270070-supinfo-0001" class="usa-link">S1</a>). We identified 29,551 protein‐coding genes in the <em>Z. sinica</em> genome, with an average length of 2,671 bp, 5.1 exons per gene, and an average exon length of 237 bp (Table <a href="#tpg270070-tbl-0002" class="usa-link">2</a>). The intragenomic gene synteny analysis revealed that 10 pairs among 20 chromosomes had high collinearity based on gene homology (Figure <a href="#tpg270070-fig-0001" class="usa-link">1D</a>). A 5.6 Mb duplication and a 1.6 Mb translocation between the subgenomes were identified within genomic blocks based on the synteny of at least 30 consecutive genes (Table <a href="#tpg270070-supinfo-0001" class="usa-link">S9</a>).</p>
<section class="tw xbox font-sm" id="tpg270070-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Statistics of predicted protein‐coding genes in the <em>Zoysia</em> species.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Species</th>
<th align="left" rowspan="1" colspan="1">
<em>Z. sinica</em> (in this study)</th>
<th align="left" rowspan="1" colspan="1">
<em>Z. japonica</em> (Nagirizaki, v2)</th>
<th align="left" rowspan="1" colspan="1">
<em>Z. matrella</em> (Wakaba, v2)</th>
<th align="left" rowspan="1" colspan="1">
<em>Z. pacifica</em> (Zanpa, v2)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">Protein coding genes</td>
<td align="left" rowspan="1" colspan="1">29,551</td>
<td align="left" rowspan="1" colspan="1">32,925</td>
<td align="left" rowspan="1" colspan="1">53,226</td>
<td align="left" rowspan="1" colspan="1">53,656</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Mean gene length (bp)</td>
<td align="left" rowspan="1" colspan="1">2,671</td>
<td align="left" rowspan="1" colspan="1">2,625</td>
<td align="left" rowspan="1" colspan="1">2,361</td>
<td align="left" rowspan="1" colspan="1">1,786</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Mean exon per transcript</td>
<td align="left" rowspan="1" colspan="1">5.1</td>
<td align="left" rowspan="1" colspan="1">5.4</td>
<td align="left" rowspan="1" colspan="1">4.9</td>
<td align="left" rowspan="1" colspan="1">3.5</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Mean exon length (bp)</td>
<td align="left" rowspan="1" colspan="1">237</td>
<td align="left" rowspan="1" colspan="1">225</td>
<td align="left" rowspan="1" colspan="1">230</td>
<td align="left" rowspan="1" colspan="1">244</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270070-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>To ensure more reliable analysis, we re‐annotated the genomes of <em>Zoysia</em> species (<em>Z. japonica</em>, <em>Z. pacifica</em>, and <em>Z. matrella</em>) using previously reported RNA‐seq data (Table <a href="#tpg270070-supinfo-0001" class="usa-link">S2</a>). We used 636 Gb of RNA‐seq data from <em>Z. japonica</em> and 95 Gb from <em>Z. matrella</em>, along with Liliopsida protein hints, for gene prediction and achieved high‐quality annotations. For <em>Z. pacifica</em>, high‐quality annotation was obtained using only Liliopsida protein hints. The quality of gene annotation was assessed through BUSCO analysis, showing significant improvement: <em>Z. japonica</em> improved from 53.8% to 95.8%, <em>Z. matrella</em> from 56.3% to 92.5%, and <em>Z. pacifica</em> from 53.5% to 75.7% (Figure <a href="#tpg270070-supinfo-0001" class="usa-link">S3</a>). We predicted 32,925 genes in <em>Z. japonica</em>, 53,226 genes in <em>Z. matrella</em>, and 53,656 genes in <em>Z. pacifica</em>. The protein‐coding gene characteristics of <em>Z. sinica</em>, <em>Z. japonica</em>, and <em>Z. matrella</em> were similar (Table <a href="#tpg270070-tbl-0002" class="usa-link">2</a>; Table <a href="#tpg270070-supinfo-0001" class="usa-link">S10</a>).</p></section><section id="tpg270070-sec-0170"><h3 class="pmc_sec_title">3.3. Comparative genomic analysis of the <em>Z. sinica</em> genome</h3>
<p>To understand the evolutionary and phylogenetic relationships of <em>Z. sinica</em>, we performed a comparative genomic analysis incorporating divergence time and WGD inference. We used 473,267 representative protein‐coding genes from 12 species, which were clustered into 34,222 orthogroups. The phylogenetic analysis revealed that <em>Z. sinica</em> formed a monophyletic clade with other <em>Zoysia</em> species (<em>Z. japonica</em>, <em>Z. pacifica</em>, and <em>Z. matrella</em>), with the closest relationship observed with <em>Z. pacifica</em> (Figure <a href="#tpg270070-fig-0002" class="usa-link">2A</a>). Divergence time estimation indicated that the <em>Zoysia</em> species gradually diverged around 3.63 (2.42–4.45) million years ago (Mya), and they diverged from <em>S. alterniflora</em> and <em>O. thomaeum</em> around 27.8 (21.04–33.65) Mya and 33.23 (31.48–35.85) Mya, respectively. In addition, analysis of Ks among collinear gene pairs supported these WGD and speciation events of <em>Z. sinica</em> (Figure <a href="#tpg270070-fig-0002" class="usa-link">2B</a>). The Ks values among <em>Zoysia</em> species were very low, below 0.3, supporting the divergence time estimation (Figure <a href="#tpg270070-fig-0002" class="usa-link">2A</a>), which ranked <em>S. alterniflora</em>, <em>O. thomaeum</em>, and <em>O. sativa</em> in sequence. Furthermore, the Ks values of paralogs within <em>Z. sinica</em> were intermediate between the Ks values of orthologs between <em>Z. sinica</em> and <em>O. thomaeum</em> and those between <em>Z. sinica</em> and <em>S. alterniflora</em>. Additionally, the Ks values of paralogs within <em>Z. sinica</em> were similar to the Ks values of orthologs between <em>Z. sinica</em> and other <em>Zoysia</em> species. Therefore, this suggests that the WGD event in <em>Z. sinica</em> likely occurred between 27.8 and 33.2 Mya. Together, these results demonstrate that the divergence of <em>Z. sinica</em> within the <em>Zoysia</em> genus occurred relatively recently, and these findings provide important insights into the evolutionary history and polyploidization events that have influenced the genus.</p>
<figure class="fig xbox font-sm" id="tpg270070-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed98/12238702/927ba7cd2eda/TPG2-18-e70070-g004.jpg" loading="lazy" id="jats-graphic-3" height="546" width="800" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270070-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genome and gene family evolution of <em>Z. sinica</em>. (A) The phylogenetic tree showing the divergence time of <em>Z. sinica</em> and the expansion and contraction of gene families across 12 species. (B) The Ks density distribution between <em>Z. sinica</em> and its closely related species (<em>O. sativa</em>, <em>O. thomaeum</em>, <em>S. alterniflora</em>, <em>Z. japonica</em>, <em>Z. matrella</em>, and <em>Z. pacifica</em>). (C and D) Gene ontology in terms of biological process (GOBP) enrichment analysis of the expanded gene families and <em>Z. sinica</em>‐specific genes.</p></figcaption></figure><p>To understand the genomic foundation that enabled <em>Z. sinica</em> to adapt to harsh intertidal environments, we performed a comprehensive comparative analysis that included both gene family evolution analysis and molecular evolutionary analysis to identify genes under positive selection pressure (Figure <a href="#tpg270070-fig-0002" class="usa-link">2A</a>). Through gene family evolution analysis, we found that <em>Z. sinica</em> exhibited expansion in only 66 gene families and contraction in 1362 gene families, with just nine gene families significantly expanded (<em>p</em>‐value &lt; 0.01). These significantly expanded gene families were enriched in gene ontology terms related to macromolecule biosynthetic processes and cell wall (Figure <a href="#tpg270070-fig-0002" class="usa-link">2C</a>). Among the expanded gene families in <em>Z. sinica</em>, homologs in <em>O. sativa</em> included ethylene response factors (<em>OsERF11</em> and <em>OsERF17</em>) and RNA helicases (<em>OsRH6</em> and <em>OsRH12</em>). There were 568 <em>Z. sinica</em>‐specific genes that did not belong to any of the 12 species orthogroups, and these genes were associated with response to auxin and ethylene, response to toxic substances, and peroxisomes (Figure <a href="#tpg270070-fig-0002" class="usa-link">2D</a>). In parallel, our molecular evolutionary analysis across four <em>Zoysia</em> genomes revealed that <em>Z. sinica</em> harbored six PSGs, compared to nine, six, and 25 PSGs in <em>Z. japonica</em>, <em>Z. matrella</em>, and <em>Z. pacifica</em>, respectively (Figure <a href="#tpg270070-supinfo-0001" class="usa-link">S4</a>; Table <a href="#tpg270070-supinfo-0001" class="usa-link">S11</a>). Among them, <em>Z. japonica</em> and <em>Z. pacifica</em> shared three PSGs, indicating common selective pressures acting on these species. To further investigate the genetic basis of the adaptation of <em>Z. sinica</em> to the intertidal environment, we analyzed PSGs shared between <em>Z. sinica</em> and <em>S. alterniflora</em>, another species inhabiting intertidal environments. A total of 29 PSGs were found to be common to both species, while six PSGs were specific to <em>Z. sinica</em>, with four genes overlapping in positive selection between them (Figure <a href="#tpg270070-supinfo-0001" class="usa-link">S5</a>; Table <a href="#tpg270070-supinfo-0001" class="usa-link">S12</a>). The four overlapped genes were annotated to the “NmrA‐like family,” “GDSL‐like Lipase/Acylhydrolase,” “XH domain,” and “may be involved in modulation of pathogen defense and leaf cell death,” according to EGGNOG. Among the PSGs shared by <em>Z. sinica</em> and <em>S. alterniflora</em>, several key stress response genes were identified. Genes involved in nitrogen‐use efficiency (<em>ENOD</em>), osmotic regulation (<em>SUT</em>), and salt and ABA responses (<em>AP2/EREBP</em>) were detected. In addition, genes related to fungal defense, including RING finger E3 ligase and nucleotide pyrophosphatase/phosphodiesterase (<em>PPI</em>), were also under positive selection. These results indicate that gene families associated with stress responses have undergone expansion, contraction, and positive selection in <em>Z. sinica</em>, providing genomic insights into their association with intertidal environments.</p></section><section id="tpg270070-sec-0180"><h3 class="pmc_sec_title">3.4. Transcriptome analysis of the intertidal zone under distinct waterlogging conditions</h3>
<p>We conducted a landscape transcriptomic study to examine gene expression patterns across intertidal environments with different levels of waterlogging. The physical characteristics of the soil in the intertidal environments and IL were investigated, showing significant differences in GWC among the three locations. HIZ soil had an average water content of 25.12% and LIZ soil had 22.92%, both significantly higher than the 17.66% observed in IL soil (Figure <a href="#tpg270070-supinfo-0001" class="usa-link">S6</a>; Table <a href="#tpg270070-supinfo-0001" class="usa-link">S13</a> and <a href="#tpg270070-supinfo-0001" class="usa-link">S14</a>).</p>
<p>The transcriptome data from shoot and root tissues of <em>Z. sinica</em> collected from the three locations formed clusters in PCA analysis, indicating high‐quality biological replicates (Figure <a href="#tpg270070-supinfo-0001" class="usa-link">S7</a>; Table <a href="#tpg270070-supinfo-0001" class="usa-link">S15</a>). In shoot tissues, our analysis revealed that 84 genes were upregulated in LIZ versus HIZ, 222 in LIZ versus IL, and 156 in HIZ versus IL. In root tissues, 202 genes were upregulated in LIZ versus HIZ, 147 in LIZ versus IL, and 45 in HIZ versus IL. To interpret the biological context of the DEGs identified between locations, we performed GOBP and KEGG enrichment analyses (Figure <a href="#tpg270070-fig-0003" class="usa-link">3</a>). In the LIZ location, <em>Z. sinica</em> showed significant upregulation of various genes related to nitrogen metabolism in roots compared to other locations. In shoots, genes associated with nitrogen metabolism, response to cold, carbohydrate metabolic process, and polysaccharide catabolic process were upregulated compared to IL. In the HIZ location, compared to IL, genes related to Gene Ontology (GO) biological process terms such as ethylene‐activated signaling pathway and response to cold were upregulated in the shoot.</p>
<p>Focusing on adaptation to persistent waterlogging stress, we identified DEGs uniquely upregulated in the LIZ environment across tidal cycles. We identified 42 DEGs specifically upregulated in the shoot (Figure <a href="#tpg270070-fig-0004" class="usa-link">4A</a>) and 36 DEGs in the root (Figure <a href="#tpg270070-fig-0004" class="usa-link">4B</a>) of the LIZ location. Notably, the DEGs specifically upregulated in the root tissue of LIZ included seven transporter genes (Figure <a href="#tpg270070-fig-0004" class="usa-link">4C</a>). The genes involved in suberin biosynthesis were upregulated in the shoot tissue (Figure <a href="#tpg270070-fig-0004" class="usa-link">4D</a>). In the root tissue, genes related to nitrate uptake (<em>NRT2.2</em> and <em>NRT2.4</em>), root–shoot nitrate translocation (<em>NRT2.3a</em>), nitrate reductase (<em>NR</em>), and glutamine synthesis (<em>GS</em> and <em>NADH‐GOGAT1</em>) were upregulated under LIZ condition. Furthermore, <em>EREBP</em> and <em>ERF</em> genes associated with ethylene response were upregulated in the root tissue under HIZ conditions. Our transcriptomic analysis across intertidal zones revealed distinct expression patterns, with a specific focus on genes uniquely upregulated in the consistently waterlogged LIZ environment. Notably, genes associated with suberin biosynthesis in the shoots and nitrate metabolism in the roots were significantly upregulated, suggesting they may play important roles in adaptation to persistent waterlogging environments.</p>
<figure class="fig xbox font-sm" id="tpg270070-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed98/12238702/4639d4441a96/TPG2-18-e70070-g002.jpg" loading="lazy" id="jats-graphic-7" height="558" width="703" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270070-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Gene expression patterns in shoot and root tissues of <em>Zoysia sinica</em> in intertidal environments. (A) Comparison of the number of upregulated differentially expressed genes (DEGs) in shoot tissues under low intertidal zone (LIZ) conditions (inland [IL] vs. LIZ conditions, high intertidal zone [HIZ] vs. LIZ conditions). (B) Comparison of the number of upregulated DEGs in root tissues under LIZ conditions (IL vs. LIZ conditions, HIZ vs. LIZ conditions). (C) Heatmap of the expression of transporter genes among the upregulated DEGs in root tissues under LIZ conditions. (D) Heatmap of the expression of genes related to suberin biosynthesis, nitrogen metabolism, and ethylene response in root and shoot tissues under waterlogging conditions.</p></figcaption></figure></section><section id="tpg270070-sec-0190"><h3 class="pmc_sec_title">3.5. GSEA of multiple abiotic stress gene sets and qRT‐PCR validation</h3>
<p>To further investigate how multiple environmental factors influence the <em>Z. sinica</em> transcriptomes across locations with varying degrees of waterlogging, we performed GSEA. A total of 224 genes were assigned to response to heat (GO:0009408), 524 genes to response to salt stress (GO:0009651), 362 genes to response to water deprivation (GO:0009414), and 500 genes to response to abscisic acid (GO:0009737). In shoot tissues, GSEA revealed that the ABA response gene set was significantly enriched in intertidal conditions (LIZ and HIZ) compared to IL (false discovery rate [FDR = 0.0]) (Figure <a href="#tpg270070-fig-0005" class="usa-link">5A</a>). The heat response gene set was significantly enriched in HIZ relative to the combined conditions of LIZ and IL (REST) (FDR = 0.0). In root tissues, we revealed that the ABA, salt stress, and water deprivation gene sets were significantly enriched in intertidal conditions (LIZ and HIZ) compared to IL (all FDR = 0.0) (Figure <a href="#tpg270070-fig-0005" class="usa-link">5B</a>). The heat response gene set was significantly enriched in HIZ relative to the combined conditions of LIZ and IL (REST) (FDR = 0.0). Interestingly, unlike in the shoot tissue, where response to heat gene sets were positively enriched (NES = 1.76), the root tissue exhibited a negative correlation with NES = −1.54, suggesting tissue‐specific differences in heat stress responses. Taken together, these results suggest that both the shoot and root transcriptomes of <em>Z. sinica</em> are likely influenced by multiple abiotic environmental factors, including salt stress, water deprivation, and heat, across intertidal zones with varying waterlogging levels.</p>
<figure class="fig xbox font-sm" id="tpg270070-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ed98/12238702/3c796421142a/TPG2-18-e70070-g003.jpg" loading="lazy" id="jats-graphic-9" height="529" width="800" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270070-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Gene set enrichment analysis (GSEA) of abiotic stress responses in <em>Z. sinica</em> under intertidal environments. The result shows significant enrichment of gene sets related to response to heat (GO:0009408), response to salt stress (GO:0009651), response to water deprivation (GO:0009414), and response to abscisic acid (GO:0009737) in (A) shoot and (B) root tissues. NES, normalized enrichment score.</p></figcaption></figure><p>To validate the RNA‐seq results, qRT‐PCR analysis based on the ΔΔCq method showed a high correlation with the RNA‐seq data (Figure <a href="#tpg270070-supinfo-0001" class="usa-link">S8</a>). Notably, five DEGs in the shoot exhibited a strong correlation between qRT‐PCR and RNA‐seq data (<em>R</em>
<sup>2</sup> = 0.9162, Pearson <em>r</em> = 0.96, Spearman rho = 0.98), further supporting the reliability of the transcriptomic analysis.</p></section></section><section id="tpg270070-sec-0200"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>In this study, we generated a high‐quality chromosome‐level genome of <em>Z. sinica</em>, a salt marsh grass adapted to saline and anoxic conditions. Compared to the previously reported chromosome‐level genome of <em>Z. japonica</em>, which was scaffolded using a linkage map and reached 81.8% scaffolding coverage (Tanaka et al., <a href="#tpg270070-bib-0082" class="usa-link" aria-describedby="tpg270070-bib-0082">2016</a>), the <em>Z. sinica</em> genome exhibits significantly improved contiguity and completeness, with 95.3% scaffolding coverage (Table <a href="#tpg270070-supinfo-0001" class="usa-link">S7</a>). As a result, <em>Z. sinica</em> achieved the highest genome coverage (99%) and Poales BUSCO score (95.3%) among <em>Zoysia</em> species, compared to <em>Z. japonica</em>, which had a BUSCO score of 92.9%. This high‐quality genome not only enabled reliable genomic research on the polyploid structure of <em>Zoysia</em> species but also provided an opportunity to identify marker genes involved in resilience to abiotic stresses, including waterlogging stress (Cheng et al., <a href="#tpg270070-bib-0017" class="usa-link" aria-describedby="tpg270070-bib-0017">2023</a>; Njaci et al., <a href="#tpg270070-bib-0059" class="usa-link" aria-describedby="tpg270070-bib-0059">2023</a>). Furthermore, it provides an important foundation for molecular breeding techniques such as marker‐assisted selection and gene editing.</p>
<p>To facilitate comparative genomic studies, we re‐annotated the protein‐coding genes of <em>Zoysia</em> species (<em>Z. japonica</em>, <em>Z. matrella</em>, and <em>Z. pacifica</em>) using RNA‐seq data and protein data available until recently (Table <a href="#tpg270070-supinfo-0001" class="usa-link">S2</a>). This re‐annotation was confirmed to have improved integrity, as demonstrated by BUSCO analysis, ensuring high‐quality gene annotation (Figure <a href="#tpg270070-supinfo-0001" class="usa-link">S3</a>). Compared to previous annotation results (Tanaka et al., <a href="#tpg270070-bib-0082" class="usa-link" aria-describedby="tpg270070-bib-0082">2016</a>), the number of protein‐coding genes was reduced by half in both <em>Z. japonica</em> and <em>Z. matrella</em>, while <em>Z. pacifica</em> showed a reduction of approximately 12,000 genes. This reduction is due to improvements in the accuracy and reliability of gene annotation. Previous annotations relied on limited gene expression data, which likely led to overestimating the number of genes. In contrast, our approach combined ab initio prediction, RNA‐seq data, and protein homology information, allowing us to retain only genes that were strongly supported by multiple sources of evidence (Brůna et al., <a href="#tpg270070-bib-0009" class="usa-link" aria-describedby="tpg270070-bib-0009">2021</a>; Gabriel et al., <a href="#tpg270070-bib-0029" class="usa-link" aria-describedby="tpg270070-bib-0029">2024</a>). The mean gene length in the chromosome‐level genomes of <em>Z. sinica</em> and <em>Z. japonica</em> was much longer than that in the draft genomes of <em>Z. matrella</em> and <em>Z. pacifica</em> (Table <a href="#tpg270070-tbl-0002" class="usa-link">2</a>), underscoring the importance of high‐quality genome assemblies for accurate genome annotation.</p>
<p>Comparative genomic analysis provided insights into the speciation timing of <em>Zoysia</em> species and the timing of the WGD event in <em>Z. sinica</em> (Figure <a href="#tpg270070-fig-0002" class="usa-link">2A,B</a>). It is likely that the WGD event in the common ancestor of the <em>Zoysia</em> genus occurred between its divergence from <em>S. alterniflora</em> (21.04–33.65 Mya) and <em>O. thomaeum</em> (31.48–35.85 Mya), coinciding with the speciation event from <em>S. alterniflora</em> (S. Chen et al., <a href="#tpg270070-bib-0015" class="usa-link" aria-describedby="tpg270070-bib-0015">2024</a>). Paleoclimatic evidence indicates significant environmental shifts during the Miocene epoch, such as increased aridification, expansion of open grasslands, and declining atmospheric CO₂ levels (Bouchenak‐Khelladi et al., <a href="#tpg270070-bib-0008" class="usa-link" aria-describedby="tpg270070-bib-0008">2014</a>; Gallaher et al., <a href="#tpg270070-bib-0030" class="usa-link" aria-describedby="tpg270070-bib-0030">2022</a>). These climatic factors may have created selective pressures conducive to polyploidization, potentially enhancing adaptability to fluctuating water availability, increased salinity, and nutrient‐poor soils. Additionally, it can be inferred that <em>Zoysia</em> species underwent speciation around 2.6–3.6 Mya. This period coincided with the intensification of monsoonal patterns in East Asia, leading to alternating droughts and the expansion of salt‐affected wetlands (Dunlea et al., <a href="#tpg270070-bib-0021" class="usa-link" aria-describedby="tpg270070-bib-0021">2020</a>). These climatic fluctuations likely influenced the adaptation and diversification of <em>Zoysia</em>, which is now predominantly distributed along coastal regions from temperate to tropical Southeast and Southern Asia. The inferred WGD and speciation events provide a genomic basis for understanding the geological and ecological distribution of <em>Zoysia</em> species in these dynamic environments.</p>
<p>The gene family evolutionary analysis provided evidence for the adaptation of <em>Z. sinica</em> to intertidal environments. The expansion of gene families related to ethylene signaling (<em>ERF11</em> and <em>ERF17</em>) (Tan et al., <a href="#tpg270070-bib-0081" class="usa-link" aria-describedby="tpg270070-bib-0081">2021</a>; Zhou et al., <a href="#tpg270070-bib-0097" class="usa-link" aria-describedby="tpg270070-bib-0097">2016</a>), auxin transport (<em>ABCB19</em>) (Sukumar et al., <a href="#tpg270070-bib-0079" class="usa-link" aria-describedby="tpg270070-bib-0079">2013</a>), and reactive oxygen species scavenging (<em>APX2</em> and <em>CATB</em>) (Mahmood et al., <a href="#tpg270070-bib-0053" class="usa-link" aria-describedby="tpg270070-bib-0053">2021</a>; Zhang et al., <a href="#tpg270070-bib-0095" class="usa-link" aria-describedby="tpg270070-bib-0095">2013</a>) suggests their roles in regulating growth, oxidative stress tolerance, and root system development under periodic submergence. We identified six PSGs in <em>Z. sinica</em>, including GDSL‐like lipase (<em>GELP</em>), which is associated with suberin biosynthesis in roots (Nomberg et al., <a href="#tpg270070-bib-0060" class="usa-link" aria-describedby="tpg270070-bib-0060">2022</a>; Ursache et al., <a href="#tpg270070-bib-0085" class="usa-link" aria-describedby="tpg270070-bib-0085">2021</a>), and the <em>NmrA</em>‐like family gene, which plays a role in nitrogen metabolism (Mao et al., <a href="#tpg270070-bib-0055" class="usa-link" aria-describedby="tpg270070-bib-0055">2023</a>) (Table <a href="#tpg270070-supinfo-0001" class="usa-link">S12</a>). These genes may be associated with oxygen availability in hypoxic soil environments and contribute to nitrate‐based energy metabolism. Additionally, <em>Z. sinica</em> shares several PSGs with <em>S. alterniflora</em>, including genes related to nitrogen metabolism (<em>ENOD</em>) (BI et al., <a href="#tpg270070-bib-0006" class="usa-link" aria-describedby="tpg270070-bib-0006">2009</a>) and osmotic stress tolerance (<em>AP2</em>/<em>EREBP</em>, <em>SUT</em>, and <em>HAK1</em>) (G. Chen et al., <a href="#tpg270070-bib-0016" class="usa-link" aria-describedby="tpg270070-bib-0016">2017</a>; Y. Li et al., <a href="#tpg270070-bib-0048" class="usa-link" aria-describedby="tpg270070-bib-0048">2022</a>; Xu et al., <a href="#tpg270070-bib-0092" class="usa-link" aria-describedby="tpg270070-bib-0092">2017</a>). The presence of these genes under positive selection suggests that both species have been shaped by similar evolutionary pressures to survive in environments characterized by periodic waterlogging and salt fluctuations.</p>
<p>Building upon these genomic findings, transcriptome analysis examined responses of <em>Z. sinica</em> to varying soil moisture conditions across intertidal zones. The observed upregulation of nitrate uptake‐related genes in root tissues from the LIZ environment suggests their potential involvement in adaptive responses to hypoxic conditions prevalent in intertidal environments (Figure <a href="#tpg270070-fig-0003" class="usa-link">3</a>). A similar upregulation of nitrate transporter genes under waterlogging stress has also been observed in mangrove plants, representative species adapted to intertidal zones (Su et al., <a href="#tpg270070-bib-0077" class="usa-link" aria-describedby="tpg270070-bib-0077">2022</a>). These observations, together with previous findings (G. Li et al., <a href="#tpg270070-bib-0047" class="usa-link" aria-describedby="tpg270070-bib-0047">2016</a>), suggest that the upregulation of nitrate transporters, such as <em>NRT2.2</em> and <em>NRT2.4</em> in <em>Z. sinica</em>, may contribute to adaptive responses by influencing root hydraulic properties and potentially facilitating water uptake under fluctuating moisture conditions. In shoot tissues, the upregulation of suberin biosynthesis genes may help reduce radial oxygen loss and enhance adaptation to frequent submergence by maintaining cellular homeostasis and minimizing ion imbalance (Figure <a href="#tpg270070-fig-0004" class="usa-link">4</a>) (Kreszies et al., <a href="#tpg270070-bib-0042" class="usa-link" aria-describedby="tpg270070-bib-0042">2020</a>; Ranathunge et al., <a href="#tpg270070-bib-0068" class="usa-link" aria-describedby="tpg270070-bib-0068">2011</a>). Additionally, <em>Z. sinica</em> samples were collected during low tide, exposing them to aeration and a post‐submergence state. As a result, prolonged submergence followed by re‐exposure to atmospheric oxygen may lead to a water deprivation state (Figure <a href="#tpg270070-fig-0005" class="usa-link">5</a>) (Fukao et al., <a href="#tpg270070-bib-0027" class="usa-link" aria-describedby="tpg270070-bib-0027">2010</a>; Setter et al., <a href="#tpg270070-bib-0074" class="usa-link" aria-describedby="tpg270070-bib-0074">2010</a>; Shahzad et al., <a href="#tpg270070-bib-0075" class="usa-link" aria-describedby="tpg270070-bib-0075">2016</a>; Yeung et al., <a href="#tpg270070-bib-0094" class="usa-link" aria-describedby="tpg270070-bib-0094">2018</a>). In response, <em>Z. sinica</em> may activate ABA‐mediated signaling to facilitate submergence recovery and regulate water balance. These transcriptomic responses align with the genomic findings, suggesting that genes under positive selection, such as <em>GELP</em>, <em>NmrA</em>‐like family genes, and <em>ENOD</em>, play roles in optimizing nitrogen use and maintaining structural integrity under intertidal conditions. While our transcriptome analysis focused on responses to varying soil moisture, it is important to acknowledge that intertidal zones are complex environments where other abiotic factors, such as fluctuations in salinity and temperature, are also prevalent. These factors could act as confounding co‐stressors and potentially contribute to or modulate the observed transcriptomic profiles, and their specific impacts warrant further investigation in conjunction with waterlogging stress.</p>
<p>In summary, the chromosome‐level genome of <em>Z. sinica</em> provides molecular evolutionary insights into the mechanisms of adaptation and survival in intertidal environments. This study shifts the focus from the traditionally studied salt stress tolerance in zoysiagrass to a new perspective on the mechanisms of waterlogging stress responses, establishing a foundation for future research on this abiotic stress. Additionally, our landscape transcriptome study enhances our understanding of population responses to environmental variables, offering deep genomic insights with ecological significance. While these genomic and transcriptomic analyses offer significant insights, future functional studies are crucial to experimentally confirm the roles of the identified candidate genes in the adaptation of <em>Z. sinica</em> to intertidal stresses. This research provides a foundation for the development of new abiotic stress‐tolerant varieties through molecular breeding methods.</p></section><section id="tpg270070-sec-0210"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Hyeonseon Park</strong>: Data curation; formal analysis; methodology; software; visualization; writing—original draft; writing—review and editing. <strong>Eunji Bae</strong>: Funding acquisition; investigation; resources; writing—original draft; writing—review and editing. <strong>Jae Gyeong Jung</strong>: Formal analysis; investigation; resources. <strong>Jaewook Kim</strong>: Methodology; software; writing—review and editing. <strong>Bae Young Choi</strong>: Methodology; software; visualization; writing—review and editing. <strong>Geungjoo Lee</strong>: Supervision. <strong>Changsoo Kim</strong>: Project administration. <strong>Donghwan Shim</strong>: Conceptualization; project administration; supervision.</p></section><section id="tpg270070-sec-0230"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec23"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270070-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270070-supitem-0001"><div class="caption p"><p>Figure S1. Genome size estimation of <em>Zoysia sinica</em> based on k‐mer distribution analysis; Figure S2. Chromosome Ideogram; Figure S3. BUSCO assessment of protein‐coding genes in four <em>Zoysia</em> species; Figure S4. Analysis of positively selected genes in <em>Zoysia</em> species. Figure S5. Analysis of positively selected genes in <em>Z. sinica</em> and <em>S. alterniflora</em>; Figure S6. Physical characteristics of soil in intertidal environments; Figure S7. PCA analysis of transcriptome data replicates; Figure S8. qRT‐PCR correlation analysis; Table S1. Liliopsida NCBI protein sets; Table S2. Public RNAseq data for Zoysia genus reannotation; Table S3. qRT‐PCR primer information; Table S4. The statistical information of raw and trimmed Illumina data; Table S5. The statistical information of Oxford Nanopore data; Table S6. The scaffolding statistical results of <em>Zoysia sinica</em> genome; Table S7. Statistics of the <em>Zoysia</em> genomes; Table S8. Repeat element annotation; Table S9. Macrosynteny in <em>Z.sinica</em> genome; Table S10.Zoysia genus gene annotation; Table S11. Positive‐selected genes of the <em>Zoysia</em> species; Table S12. Positive‐selected genes of <em>Z. sinica</em> and <em>S. alterniflora</em>; Table S13. Soil characteristics; Table S14. Statistics of Soil characteristics; Table S15. Statistics of RNAseq data;</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12238702/bin/TPG2-18-e70070-s001.zip" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70070-s001.zip</a><sup> (2MB, zip) </sup>
</div></div></section></section><section id="tpg270070-sec-0220" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>This research was funded by the Forest Biomaterial Research Center, National Institute of Forest Science (Project No. FE0100‐ 2025‐03), and was supported by Basic Science Research Program through the National Research Foundation of Korea (NRF) funded by the Ministry of Education (No. 2022R1I1A3072428).</p></section><section id="notes1"><p>


Park, H.
, 
Bae, E.
, 
Jung, J. G.
, 
Kim, J.
, 
Choi, B. Y.
, 
Lee, G.
, 
Kim, C.
, &amp; 
Shim, D.
 (2025). Chromosome‐level genome of <em>Zoysia sinica</em> in the intertidal zone reveals genomic insights into waterlogging stress adaptation. The Plant Genome, 18, e70070. 10.1002/tpg2.70070

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="tpg270070-note-0001"><p>Assigned to Associate Editor Manuel Spannagl.</p></div>
<div class="fn p" id="tpg270070-note-0002"><p>Hyeonseon Park and Eunji Bae contributed equally to this work.</p></div>
</div></section></section><section id="tpg270070-sec-0250"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>All sequence read files for the assembled genome have been deposited in the Sequence Read Archive database at NCBI under BioProject IDs PRJNA1149339. The Whole Genome Shotgun (WGS) project has been deposited at DDBJ/ENA/GenBank under the accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/JBGRUJ000000000" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">JBGRUJ000000000</a> and can be accessed using this number. Additionally, the gene annotation results for four <em>Zoysia</em> species (<em>Z. sinica</em>, <em>Z. japonica</em>, <em>Z. matrella</em>, and <em>Z. pacifica</em>) have been shared via Figshare (<a href="https://doi.org/10.6084/m9.figshare.27888426.v1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.6084/m9.figshare.27888426.v1</a>).</p></section><section id="tpg270070-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270070-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270070-bib-0001">
<cite>

Abascal, F.
, 
Zardoya, R.
, &amp; 
Telford, M. J.
 (2010). TranslatorX: Multiple alignment of nucleotide sequences guided by amino acid translations. Nucleic Acids Research, 38(ppl_2), W7–W13. 10.1093/nar/gkq291

</cite> [<a href="https://doi.org/10.1093/nar/gkq291" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2896173/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20435676/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=TranslatorX:%20Multiple%20alignment%20of%20nucleotide%20sequences%20guided%20by%20amino%20acid%20translations&amp;author=F.%20Abascal&amp;author=R.%20Zardoya&amp;author=M.%20J.%20Telford&amp;volume=38&amp;publication_year=2010&amp;pages=W7-W13&amp;pmid=20435676&amp;doi=10.1093/nar/gkq291&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0002">
<cite>

Altschul, S. F.
, 
Gish, W.
, 
Miller, W.
, 
Myers, E. W.
, &amp; 
Lipman, D. J.
 (1990). Basic local alignment search tool. Journal of Molecular Biology, 215(3), 403–410. 10.1016/S0022-2836(05)80360-2

</cite> [<a href="https://doi.org/10.1016/S0022-2836(05)80360-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2231712/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Molecular%20Biology&amp;title=Basic%20local%20alignment%20search%20tool&amp;author=S.%20F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.%20W.%20Myers&amp;author=D.%20J.%20Lipman&amp;volume=215&amp;issue=3&amp;publication_year=1990&amp;pages=403-410&amp;pmid=2231712&amp;doi=10.1016/S0022-2836(05)80360-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0003">
<cite>

Álvarez‐Carretero, S.
, 
Kapli, P.
, &amp; 
Yang, Z.
 (2023). Beginner's guide on the use of PAML to detect positive selection. Molecular Biology and Evolution, 40(4), Article msad041. 10.1093/molbev/msad041

</cite> [<a href="https://doi.org/10.1093/molbev/msad041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10127084/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37096789/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=Beginner's%20guide%20on%20the%20use%20of%20PAML%20to%20detect%20positive%20selection&amp;author=S.%20%C3%81lvarez%E2%80%90Carretero&amp;author=P.%20Kapli&amp;author=Z.%20Yang&amp;volume=40&amp;issue=4&amp;publication_year=2023&amp;pages=msad041&amp;pmid=37096789&amp;doi=10.1093/molbev/msad041&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0004">
<cite>

Anderson, W. F.
, 
Snook, M. E.
, &amp; 
Johnson, A. W.
 (2007). Flavonoids of zoysiagrass (<em>Zoysia</em> spp.) cultivars varying in fall armyworm (<em>Spodoptera frugiperda</em>) resistance. Journal of Agricultural and Food Chemistry, 55(5), 1853–1861. 10.1021/jf062376n

</cite> [<a href="https://doi.org/10.1021/jf062376n" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17288435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Agricultural%20and%20Food%20Chemistry&amp;title=Flavonoids%20of%20zoysiagrass%20(Zoysia%20spp.)%20cultivars%20varying%20in%20fall%20armyworm%20(Spodoptera%20frugiperda)%20resistance&amp;author=W.%20F.%20Anderson&amp;author=M.%20E.%20Snook&amp;author=A.%20W.%20Johnson&amp;volume=55&amp;issue=5&amp;publication_year=2007&amp;pages=1853-1861&amp;pmid=17288435&amp;doi=10.1021/jf062376n&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0005">
<cite>

Bang, J. H.
, 
Bae, M.‐J.
, &amp; 
Lee, E. J.
 (2018). Plant distribution along an elevational gradient in a macrotidal salt marsh on the west coast of Korea. Aquatic Botany, 147, 52–60. 10.1016/j.aquabot.2018.03.005
</cite> [<a href="https://doi.org/10.1016/j.aquabot.2018.03.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aquatic%20Botany&amp;title=Plant%20distribution%20along%20an%20elevational%20gradient%20in%20a%20macrotidal%20salt%20marsh%20on%20the%20west%20coast%20of%20Korea&amp;author=J.%20H.%20Bang&amp;author=M.%E2%80%90J.%20Bae&amp;author=E.%20J.%20Lee&amp;volume=147&amp;publication_year=2018&amp;pages=52-60&amp;doi=10.1016/j.aquabot.2018.03.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0006">
<cite>

Bi, Y.‐M.
, 
Kant, S.
, 
Clark, J.
, 
Gidda, S.
, 
Ming, F.
, 
Xu, J.
, 
Rochon, A.
, 
Shelp, B. J.
, 
Hao, L.
, 
Zhao, R.
, 
Mullen, R. T.
, 
Zhu, T.
, &amp; 
Rothstein, S. J.
 (2009). Increased nitrogen‐use efficiency in transgenic rice plants over‐expressing a nitrogen‐responsive early nodulin gene identified from rice expression profiling. Plant, Cell &amp; Environment, 32(12), 1749–1760. 10.1111/j.1365-3040.2009.02032.x
</cite> [<a href="https://doi.org/10.1111/j.1365-3040.2009.02032.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19682292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant,%20Cell%20&amp;%20Environment&amp;title=Increased%20nitrogen%E2%80%90use%20efficiency%20in%20transgenic%20rice%20plants%20over%E2%80%90expressing%20a%20nitrogen%E2%80%90responsive%20early%20nodulin%20gene%20identified%20from%20rice%20expression%20profiling&amp;author=Y.%E2%80%90M.%20Bi&amp;author=S.%20Kant&amp;author=J.%20Clark&amp;author=S.%20Gidda&amp;author=F.%20Ming&amp;volume=32&amp;issue=12&amp;publication_year=2009&amp;pages=1749-1760&amp;pmid=19682292&amp;doi=10.1111/j.1365-3040.2009.02032.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0007">
<cite>

Bolger, A. M.
, 
Lohse, M.
, &amp; 
Usadel, B.
 (2014). Trimmomatic: A flexible trimmer for Illumina sequence data. Bioinformatics, 30(15), 2114–2120. 10.1093/bioinformatics/btu170

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4103590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24695404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Trimmomatic:%20A%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;author=A.%20M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;volume=30&amp;issue=15&amp;publication_year=2014&amp;pages=2114-2120&amp;pmid=24695404&amp;doi=10.1093/bioinformatics/btu170&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0008">
<cite>

Bouchenak‐Khelladi, Y.
, 
Slingsby, J. A.
, 
Verboom, G. A.
, &amp; 
Bond, W. J.
 (2014). Diversification of C4 grasses (Poaceae) does not coincide with their ecological dominance. American Journal of Botany, 101(2), 300–307. 10.3732/ajb.1300439

</cite> [<a href="https://doi.org/10.3732/ajb.1300439" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24509796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Botany&amp;title=Diversification%20of%20C4%20grasses%20(Poaceae)%20does%20not%20coincide%20with%20their%20ecological%20dominance&amp;author=Y.%20Bouchenak%E2%80%90Khelladi&amp;author=J.%20A.%20Slingsby&amp;author=G.%20A.%20Verboom&amp;author=W.%20J.%20Bond&amp;volume=101&amp;issue=2&amp;publication_year=2014&amp;pages=300-307&amp;pmid=24509796&amp;doi=10.3732/ajb.1300439&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0009">
<cite>

Brůna, T.
, 
Hoff, K. J.
, 
Lomsadze, A.
, 
Stanke, M.
, &amp; 
Borodovsky, M.
 (2021). BRAKER2: Automatic eukaryotic genome annotation with GeneMark‐EP+ and AUGUSTUS supported by a protein database. NAR Genomics and Bioinformatics, 3(1), Article lqaa108. 10.1093/nargab/lqaa108

</cite> [<a href="https://doi.org/10.1093/nargab/lqaa108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7787252/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33575650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=NAR%20Genomics%20and%20Bioinformatics&amp;title=BRAKER2:%20Automatic%20eukaryotic%20genome%20annotation%20with%20GeneMark%E2%80%90EP+%20and%20AUGUSTUS%20supported%20by%20a%20protein%20database&amp;author=T.%20Br%C5%AFna&amp;author=K.%20J.%20Hoff&amp;author=A.%20Lomsadze&amp;author=M.%20Stanke&amp;author=M.%20Borodovsky&amp;volume=3&amp;issue=1&amp;publication_year=2021&amp;pages=lqaa108&amp;pmid=33575650&amp;doi=10.1093/nargab/lqaa108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0010">
<cite>

Bruna, T.
, 
Lomsadze, A.
, &amp; 
Borodovsky, M.
 (2024). GeneMark‐ETP significantly improves the accuracy of automatic annotation of large eukaryotic genomes. Genome Research, 34(5), 757–768. 10.1101/gr.278373.123

</cite> [<a href="https://doi.org/10.1101/gr.278373.123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11216313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38866548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=GeneMark%E2%80%90ETP%20significantly%20improves%20the%20accuracy%20of%20automatic%20annotation%20of%20large%20eukaryotic%20genomes&amp;author=T.%20Bruna&amp;author=A.%20Lomsadze&amp;author=M.%20Borodovsky&amp;volume=34&amp;issue=5&amp;publication_year=2024&amp;pages=757-768&amp;pmid=38866548&amp;doi=10.1101/gr.278373.123&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0011">
<cite>

Buchfink, B.
, 
Xie, C.
, &amp; 
Huson, D. H.
 (2015). Fast and sensitive protein alignment using DIAMOND. Nature Methods, 12(1), 59–60. 10.1038/nmeth.3176

</cite> [<a href="https://doi.org/10.1038/nmeth.3176" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25402007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Fast%20and%20sensitive%20protein%20alignment%20using%20DIAMOND&amp;author=B.%20Buchfink&amp;author=C.%20Xie&amp;author=D.%20H.%20Huson&amp;volume=12&amp;issue=1&amp;publication_year=2015&amp;pages=59-60&amp;pmid=25402007&amp;doi=10.1038/nmeth.3176&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0012">
<cite>

Cantalapiedra, C. P.
, 
Hernández‐Plaza, A.
, 
Letunic, I.
, 
Bork, P.
, &amp; 
Huerta‐Cepas, J.
 (2021). eggNOG‐mapper v2: Functional annotation, orthology assignments, and domain prediction at the metagenomic scale. Molecular Biology and Evolution, 38(12), 5825–5829. 10.1093/molbev/msab293

</cite> [<a href="https://doi.org/10.1093/molbev/msab293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8662613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34597405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=eggNOG%E2%80%90mapper%20v2:%20Functional%20annotation,%20orthology%20assignments,%20and%20domain%20prediction%20at%20the%20metagenomic%20scale&amp;author=C.%20P.%20Cantalapiedra&amp;author=A.%20Hern%C3%A1ndez%E2%80%90Plaza&amp;author=I.%20Letunic&amp;author=P.%20Bork&amp;author=J.%20Huerta%E2%80%90Cepas&amp;volume=38&amp;issue=12&amp;publication_year=2021&amp;pages=5825-5829&amp;pmid=34597405&amp;doi=10.1093/molbev/msab293&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0013">
<cite>

Castresana, J.
 (2000). Selection of conserved blocks from multiple alignments for their use in phylogenetic analysis. Molecular Biology and Evolution, 17(4), 540–552. 10.1093/oxfordjournals.molbev.a026334

</cite> [<a href="https://doi.org/10.1093/oxfordjournals.molbev.a026334" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10742046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=Selection%20of%20conserved%20blocks%20from%20multiple%20alignments%20for%20their%20use%20in%20phylogenetic%20analysis&amp;author=J.%20Castresana&amp;volume=17&amp;issue=4&amp;publication_year=2000&amp;pages=540-552&amp;pmid=10742046&amp;doi=10.1093/oxfordjournals.molbev.a026334&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0014">
<cite>

Chen, C.
, 
Chen, H.
, 
Zhang, Y.
, 
Thomas, H. R.
, 
Frank, M. H.
, 
He, Y.
, &amp; 
Xia, R.
 (2020). TBtools: An integrative toolkit developed for interactive analyses of big biological data. Molecular Plant, 13(8), 1194–1202. 10.1016/j.molp.2020.06.009

</cite> [<a href="https://doi.org/10.1016/j.molp.2020.06.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32585190/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=TBtools:%20An%20integrative%20toolkit%20developed%20for%20interactive%20analyses%20of%20big%20biological%20data&amp;author=C.%20Chen&amp;author=H.%20Chen&amp;author=Y.%20Zhang&amp;author=H.%20R.%20Thomas&amp;author=M.%20H.%20Frank&amp;volume=13&amp;issue=8&amp;publication_year=2020&amp;pages=1194-1202&amp;pmid=32585190&amp;doi=10.1016/j.molp.2020.06.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0015">
<cite>

Chen, S.
, 
Du, T.
, 
Huang, Z.
, 
He, K.
, 
Yang, M.
, 
Gao, S.
, 
Yu, T.
, 
Zhang, H.
, 
Li, X.
, 
Chen, S.
, 
Liu, C.‐M.
, &amp; 
Li, H.
 (2024). The <em>Spartina alterniflora</em> genome sequence provides insights into the salt‐tolerance mechanisms of exo‐recretohalophytes. Plant Biotechnology Journal, 22(9), 2558–2574. 10.1111/pbi.14368

</cite> [<a href="https://doi.org/10.1111/pbi.14368" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11331799/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38685729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=The%20Spartina%20alterniflora%20genome%20sequence%20provides%20insights%20into%20the%20salt%E2%80%90tolerance%20mechanisms%20of%20exo%E2%80%90recretohalophytes&amp;author=S.%20Chen&amp;author=T.%20Du&amp;author=Z.%20Huang&amp;author=K.%20He&amp;author=M.%20Yang&amp;volume=22&amp;issue=9&amp;publication_year=2024&amp;pages=2558-2574&amp;pmid=38685729&amp;doi=10.1111/pbi.14368&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0016">
<cite>

Chen, G.
, 
Liu, C.
, 
Gao, Z.
, 
Zhang, Y.
, 
Jiang, H.
, 
Zhu, L.
, 
Ren, D.
, 
Yu, L.
, 
Xu, G.
, &amp; 
Qian, Q.
 (2017). 
<em>OsHAK1</em>, a high‐affinity potassium transporter, positively regulates responses to drought stress in rice. Frontiers in Plant Science, 8, Article 1885. 10.3389/fpls.2017.01885

</cite> [<a href="https://doi.org/10.3389/fpls.2017.01885" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5671996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29163608/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=OsHAK1,%20a%20high%E2%80%90affinity%20potassium%20transporter,%20positively%20regulates%20responses%20to%20drought%20stress%20in%20rice&amp;author=G.%20Chen&amp;author=C.%20Liu&amp;author=Z.%20Gao&amp;author=Y.%20Zhang&amp;author=H.%20Jiang&amp;volume=8&amp;publication_year=2017&amp;pages=1885&amp;pmid=29163608&amp;doi=10.3389/fpls.2017.01885&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0017">
<cite>

Cheng, H.
, 
Song, X.
, 
Hu, Y.
, 
Wu, T.
, 
Yang, Q.
, 
An, Z.
, 
Feng, S.
, 
Deng, Z.
, 
Wu, W.
, 
Zeng, X.
, 
Tu, M.
, 
Wang, X.
, &amp; 
Huang, H.
 (2023). Chromosome‐level wild <em>Hevea brasiliensis</em> genome provides new tools for genomic‐assisted breeding and valuable loci to elevate rubber yield. Plant Biotechnology Journal, 21(5), 1058–1072. 10.1111/pbi.14018

</cite> [<a href="https://doi.org/10.1111/pbi.14018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10106855/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36710373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=Chromosome%E2%80%90level%20wild%20Hevea%20brasiliensis%20genome%20provides%20new%20tools%20for%20genomic%E2%80%90assisted%20breeding%20and%20valuable%20loci%20to%20elevate%20rubber%20yield&amp;author=H.%20Cheng&amp;author=X.%20Song&amp;author=Y.%20Hu&amp;author=T.%20Wu&amp;author=Q.%20Yang&amp;volume=21&amp;issue=5&amp;publication_year=2023&amp;pages=1058-1072&amp;pmid=36710373&amp;doi=10.1111/pbi.14018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0018">
<cite>

Cheon, S.‐H.
, 
Woo, M.‐A.
, 
Jo, S.
, 
Kim, Y.‐K.
, &amp; 
Kim, K.‐J.
 (2021). The chloroplast phylogenomics and systematics of <em>Zoysia</em> (Poaceae). Plants, 10(8), Article 1517. 10.3390/plants10081517

</cite> [<a href="https://doi.org/10.3390/plants10081517" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8400354/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34451562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=The%20chloroplast%20phylogenomics%20and%20systematics%20of%20Zoysia%20(Poaceae)&amp;author=S.%E2%80%90H.%20Cheon&amp;author=M.%E2%80%90A.%20Woo&amp;author=S.%20Jo&amp;author=Y.%E2%80%90K.%20Kim&amp;author=K.%E2%80%90J.%20Kim&amp;volume=10&amp;issue=8&amp;publication_year=2021&amp;pages=1517&amp;pmid=34451562&amp;doi=10.3390/plants10081517&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0019">
<cite>

Choi, J.‐S.
, 
Yang, G.‐M.
, 
Bae, E.‐J.
, 
Park, Y.‐B.
, &amp; 
Lee, K.‐S.
 (2017). Development of new hybrid zoysiagrass cultivar ‘Seah’. Weed &amp; Turfgrass Science, 6(4), 306–312.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Weed%20&amp;%20Turfgrass%20Science&amp;title=Development%20of%20new%20hybrid%20zoysiagrass%20cultivar%20%E2%80%98Seah%E2%80%99&amp;author=J.%E2%80%90S.%20Choi&amp;author=G.%E2%80%90M.%20Yang&amp;author=E.%E2%80%90J.%20Bae&amp;author=Y.%E2%80%90B.%20Park&amp;author=K.%E2%80%90S.%20Lee&amp;volume=6&amp;issue=4&amp;publication_year=2017&amp;pages=306-312&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0020">
<cite>

Dudchenko, O.
, 
Batra, S. S.
, 
Omer, A. D.
, 
Nyquist, S. K.
, 
Hoeger, M.
, 
Durand, N. C.
, 
Shamim, M. S.
, 
Machol, I.
, 
Lander, E. S.
, 
Aiden, A. P.
, &amp; 
Aiden, E. L.
 (2017). De novo assembly of the <em>Aedes aegypti</em>genome using Hi‐C yields chromosome‐length scaffolds. Science, 356(6333), 92–95. 10.1126/science.aal3327

</cite> [<a href="https://doi.org/10.1126/science.aal3327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5635820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28336562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=De%20novo%20assembly%20of%20the%20Aedes%20aegyptigenome%20using%20Hi%E2%80%90C%20yields%20chromosome%E2%80%90length%20scaffolds&amp;author=O.%20Dudchenko&amp;author=S.%20S.%20Batra&amp;author=A.%20D.%20Omer&amp;author=S.%20K.%20Nyquist&amp;author=M.%20Hoeger&amp;volume=356&amp;issue=6333&amp;publication_year=2017&amp;pages=92-95&amp;pmid=28336562&amp;doi=10.1126/science.aal3327&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0021">
<cite>

Dunlea, A. G.
, 
Giosan, L.
, &amp; 
Huang, Y.
 (2020). Pliocene expansion of C<sub>4</sub> vegetation in the Core Monsoon Zone on the Indian Peninsula. Climate of the Past, 16(6), 2533–2546. 10.5194/cp-16-2533-2020
</cite> [<a href="https://doi.org/10.5194/cp-16-2533-2020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Climate%20of%20the%20Past&amp;title=Pliocene%20expansion%20of%20C4%20vegetation%20in%20the%20Core%20Monsoon%20Zone%20on%20the%20Indian%20Peninsula&amp;author=A.%20G.%20Dunlea&amp;author=L.%20Giosan&amp;author=Y.%20Huang&amp;volume=16&amp;issue=6&amp;publication_year=2020&amp;pages=2533-2546&amp;doi=10.5194/cp-16-2533-2020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0022">
<cite>

Durand, N. C.
, 
Shamim, M. S.
, 
Machol, I.
, 
Rao, S. S. P.
, 
Huntley, M. H.
, 
Lander, E. S.
, &amp; 
Aiden, E. L.
 (2016). Juicer provides a one‐click system for analyzing loop‐resolution Hi‐C experiments. Cell Systems, 3(1), 95–98. 10.1016/j.cels.2016.07.002

</cite> [<a href="https://doi.org/10.1016/j.cels.2016.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5846465/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27467249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Systems&amp;title=Juicer%20provides%20a%20one%E2%80%90click%20system%20for%20analyzing%20loop%E2%80%90resolution%20Hi%E2%80%90C%20experiments&amp;author=N.%20C.%20Durand&amp;author=M.%20S.%20Shamim&amp;author=I.%20Machol&amp;author=S.%20S.%20P.%20Rao&amp;author=M.%20H.%20Huntley&amp;volume=3&amp;issue=1&amp;publication_year=2016&amp;pages=95-98&amp;pmid=27467249&amp;doi=10.1016/j.cels.2016.07.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0023">
<cite>

Emms, D. M.
, &amp; 
Kelly, S.
 (2018). STAG: Species tree inference from all genes. <em>bioRxiv</em>. 10.1101/267914
</cite> [<a href="https://doi.org/10.1101/267914" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="tpg270070-bib-0024">
<cite>

Emms, D. M.
, &amp; 
Kelly, S.
 (2019). OrthoFinder: Phylogenetic orthology inference for comparative genomics. Genome Biology, 20(1), Article 238. 10.1186/s13059-019-1832-y

</cite> [<a href="https://doi.org/10.1186/s13059-019-1832-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6857279/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31727128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=OrthoFinder:%20Phylogenetic%20orthology%20inference%20for%20comparative%20genomics&amp;author=D.%20M.%20Emms&amp;author=S.%20Kelly&amp;volume=20&amp;issue=1&amp;publication_year=2019&amp;pages=238&amp;pmid=31727128&amp;doi=10.1186/s13059-019-1832-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0025">
<cite>

Flynn, J. M.
, 
Hubley, R.
, 
Goubert, C.
, 
Rosen, J.
, 
Clark, A. G.
, 
Feschotte, C.
, &amp; 
Smit, A. F.
 (2020). RepeatModeler2 for automated genomic discovery of transposable element families. PNAS, 117(17), 9451–9457. 10.1073/pnas.1921046117

</cite> [<a href="https://doi.org/10.1073/pnas.1921046117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7196820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32300014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=RepeatModeler2%20for%20automated%20genomic%20discovery%20of%20transposable%20element%20families&amp;author=J.%20M.%20Flynn&amp;author=R.%20Hubley&amp;author=C.%20Goubert&amp;author=J.%20Rosen&amp;author=A.%20G.%20Clark&amp;volume=117&amp;issue=17&amp;publication_year=2020&amp;pages=9451-9457&amp;pmid=32300014&amp;doi=10.1073/pnas.1921046117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0026">
<cite>

Forbes, I., Jr.
 (1952). Chromosome numbers and hybrids in <em>Zoysia</em>
. Agronomy Journal, 44(4), 194–199. 10.2134/agronj1952.00021962004400040008x
</cite> [<a href="https://doi.org/10.2134/agronj1952.00021962004400040008x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy%20Journal&amp;title=Chromosome%20numbers%20and%20hybrids%20in%20Zoysia&amp;author=I.%20Forbes&amp;volume=44&amp;issue=4&amp;publication_year=1952&amp;pages=194-199&amp;doi=10.2134/agronj1952.00021962004400040008x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0027">
<cite>

Fukao, T.
, 
Yeung, E.
, &amp; 
Bailey‐Serres, J.
 (2010). The submergence tolerance regulator SUB1A mediates crosstalk between submergence and drought tolerance in rice. The Plant Cell, 23(1), 412–427. 10.1105/tpc.110.080325
</cite> [<a href="https://doi.org/10.1105/tpc.110.080325" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3051255/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21239643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Cell&amp;title=The%20submergence%20tolerance%20regulator%20SUB1A%20mediates%20crosstalk%20between%20submergence%20and%20drought%20tolerance%20in%20rice&amp;author=T.%20Fukao&amp;author=E.%20Yeung&amp;author=J.%20Bailey%E2%80%90Serres&amp;volume=23&amp;issue=1&amp;publication_year=2010&amp;pages=412-427&amp;pmid=21239643&amp;doi=10.1105/tpc.110.080325&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0028">
<cite>

Fukushima, K.
, &amp; 
Pollock, D. D.
 (2023). Detecting macroevolutionary genotype–phenotype associations using error‐corrected rates of protein convergence. Nature Ecology &amp; Evolution, 7(1), 155–170. 10.1038/s41559-022-01932-7

</cite> [<a href="https://doi.org/10.1038/s41559-022-01932-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9834058/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36604553/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Ecology%20&amp;%20Evolution&amp;title=Detecting%20macroevolutionary%20genotype%E2%80%93phenotype%20associations%20using%20error%E2%80%90corrected%20rates%20of%20protein%20convergence&amp;author=K.%20Fukushima&amp;author=D.%20D.%20Pollock&amp;volume=7&amp;issue=1&amp;publication_year=2023&amp;pages=155-170&amp;pmid=36604553&amp;doi=10.1038/s41559-022-01932-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0029">
<cite>

Gabriel, L.
, 
Brůna, T.
, 
Hoff, K. J.
, 
Ebel, M.
, 
Lomsadze, A.
, 
Borodovsky, M.
, &amp; 
Stanke, M.
 (2024). BRAKER3: Fully automated genome annotation using RNA‐seq and protein evidence with GeneMark‐ETP, AUGUSTUS and TSEBRA. <em>bioRxiv</em>. 10.1101/2023.06.10.544449
</cite> [<a href="https://doi.org/10.1101/2023.06.10.544449" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11216308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38866550/" class="usa-link">PubMed</a>]</li>
<li id="tpg270070-bib-0030">
<cite>

Gallaher, T. J.
, 
Peterson, P. M.
, 
Soreng, R. J.
, 
Zuloaga, F. O.
, 
Li, D.‐Z.
, 
Clark, L. G.
, 
Tyrrell, C. D.
, 
Welker, C. A. D.
, 
Kellogg, E. A.
, &amp; 
Teisher, J. K.
 (2022). Grasses through space and time: An overview of the biogeographical and macroevolutionary history of Poaceae. Journal of Systematics and Evolution, 60(3), 522–569. 10.1111/jse.12857
</cite> [<a href="https://doi.org/10.1111/jse.12857" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Systematics%20and%20Evolution&amp;title=Grasses%20through%20space%20and%20time:%20An%20overview%20of%20the%20biogeographical%20and%20macroevolutionary%20history%20of%20Poaceae&amp;author=T.%20J.%20Gallaher&amp;author=P.%20M.%20Peterson&amp;author=R.%20J.%20Soreng&amp;author=F.%20O.%20Zuloaga&amp;author=D.%E2%80%90Z.%20Li&amp;volume=60&amp;issue=3&amp;publication_year=2022&amp;pages=522-569&amp;doi=10.1111/jse.12857&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0031">
<cite>

Guo, H.‐L.
, 
Xuan, J.‐P.
, 
Liu, J.‐X.
, 
Zhang, Y.‐M.
, &amp; 
Zheng, Y.‐Q.
 (2012). Association of molecular markers with cold tolerance and green period in zoysiagrass (<em>Zoysia</em> Willd.). Breeding Science, 62(4), 320–327. 10.1270/jsbbs.62.320

</cite> [<a href="https://doi.org/10.1270/jsbbs.62.320" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3528328/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23341745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Breeding%20Science&amp;title=Association%20of%20molecular%20markers%20with%20cold%20tolerance%20and%20green%20period%20in%20zoysiagrass%20(Zoysia%20Willd.)&amp;author=H.%E2%80%90L.%20Guo&amp;author=J.%E2%80%90P.%20Xuan&amp;author=J.%E2%80%90X.%20Liu&amp;author=Y.%E2%80%90M.%20Zhang&amp;author=Y.%E2%80%90Q.%20Zheng&amp;volume=62&amp;issue=4&amp;publication_year=2012&amp;pages=320-327&amp;pmid=23341745&amp;doi=10.1270/jsbbs.62.320&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0032">
<cite>

Gurevich, A.
, 
Saveliev, V.
, 
Vyahhi, N.
, &amp; 
Tesler, G.
 (2013). QUAST: Quality assessment tool for genome assemblies. Bioinformatics, 29(8), 1072–1075. 10.1093/bioinformatics/btt086

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3624806/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23422339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=QUAST:%20Quality%20assessment%20tool%20for%20genome%20assemblies&amp;author=A.%20Gurevich&amp;author=V.%20Saveliev&amp;author=N.%20Vyahhi&amp;author=G.%20Tesler&amp;volume=29&amp;issue=8&amp;publication_year=2013&amp;pages=1072-1075&amp;pmid=23422339&amp;doi=10.1093/bioinformatics/btt086&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0033">
<cite>

Hao, Y.
, 
Wang, X.‐F.
, 
Guo, Y.
, 
Li, T.‐Y.
, 
Yang, J.
, 
Ainouche, M. L.
, 
Salmon, A.
, 
Ju, R.‐T.
, 
Wu, J.‐H.
, 
Li, L.‐F.
, &amp; 
Li, B.
 (2024). Genomic and phenotypic signatures provide insights into the wide adaptation of a global plant invader. Plant Communications, 5(4), Article 100820. 10.1016/j.xplc.2024.100820

</cite> [<a href="https://doi.org/10.1016/j.xplc.2024.100820" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11009367/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38221758/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Communications&amp;title=Genomic%20and%20phenotypic%20signatures%20provide%20insights%20into%20the%20wide%20adaptation%20of%20a%20global%20plant%20invader&amp;author=Y.%20Hao&amp;author=X.%E2%80%90F.%20Wang&amp;author=Y.%20Guo&amp;author=T.%E2%80%90Y.%20Li&amp;author=J.%20Yang&amp;volume=5&amp;issue=4&amp;publication_year=2024&amp;pages=100820&amp;pmid=38221758&amp;doi=10.1016/j.xplc.2024.100820&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0034">
<cite>

Hart, A. J.
, 
Ginzburg, S.
, 
Xu, M.
, 
Fisher, C. R.
, 
Rahmatpour, N.
, 
Mitton, J. B.
, 
Paul, R.
, &amp; 
Wegrzyn, J. L.
 (2020). EnTAP: Bringing faster and smarter functional annotation to non‐model eukaryotic transcriptomes. Molecular Ecology Resources, 20(2), 591–604. 10.1111/1755-0998.13106

</cite> [<a href="https://doi.org/10.1111/1755-0998.13106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31628884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology%20Resources&amp;title=EnTAP:%20Bringing%20faster%20and%20smarter%20functional%20annotation%20to%20non%E2%80%90model%20eukaryotic%20transcriptomes&amp;author=A.%20J.%20Hart&amp;author=S.%20Ginzburg&amp;author=M.%20Xu&amp;author=C.%20R.%20Fisher&amp;author=N.%20Rahmatpour&amp;volume=20&amp;issue=2&amp;publication_year=2020&amp;pages=591-604&amp;pmid=31628884&amp;doi=10.1111/1755-0998.13106&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0035">
<cite>

Hu, J.
, 
Wang, Z.
, 
Sun, Z.
, 
Hu, B.
, 
Ayoola, A. O.
, 
Liang, F.
, 
Li, J.
, 
Sandoval, J. R.
, 
Cooper, D. N.
, 
Ye, K.
, 
Ruan, J.
, 
Xiao, C.‐L.
, 
Wang, D.
, 
Wu, D.‐D.
, &amp; 
Wang, S.
 (2024). NextDenovo: An efficient error correction and accurate assembly tool for noisy long reads. Genome Biology, 25(1), Article 107. 10.1186/s13059-024-03252-4

</cite> [<a href="https://doi.org/10.1186/s13059-024-03252-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11046930/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38671502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=NextDenovo:%20An%20efficient%20error%20correction%20and%20accurate%20assembly%20tool%20for%20noisy%20long%20reads&amp;author=J.%20Hu&amp;author=Z.%20Wang&amp;author=Z.%20Sun&amp;author=B.%20Hu&amp;author=A.%20O.%20Ayoola&amp;volume=25&amp;issue=1&amp;publication_year=2024&amp;pages=107&amp;pmid=38671502&amp;doi=10.1186/s13059-024-03252-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0036">
<cite>

Huerta‐Cepas, J.
, 
Forslund, K.
, 
Coelho, L. P.
, 
Szklarczyk, D.
, 
Jensen, L. J.
, 
von Mering, C.
, &amp; 
Bork, P.
 (2017). Fast genome‐wide functional annotation through orthology assignment by eggNOG‐mapper. Molecular Biology and Evolution, 34(8), 2115–2122. 10.1093/molbev/msx148

</cite> [<a href="https://doi.org/10.1093/molbev/msx148" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5850834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28460117/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=Fast%20genome%E2%80%90wide%20functional%20annotation%20through%20orthology%20assignment%20by%20eggNOG%E2%80%90mapper&amp;author=J.%20Huerta%E2%80%90Cepas&amp;author=K.%20Forslund&amp;author=L.%20P.%20Coelho&amp;author=D.%20Szklarczyk&amp;author=L.%20J.%20Jensen&amp;volume=34&amp;issue=8&amp;publication_year=2017&amp;pages=2115-2122&amp;pmid=28460117&amp;doi=10.1093/molbev/msx148&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0037">
<cite>

Huerta‐Cepas, J.
, 
Szklarczyk, D.
, 
Heller, D.
, 
Hernández‐Plaza, A.
, 
Forslund, S. K.
, 
Cook, H.
, 
Mende, D. R.
, 
Letunic, I.
, 
Rattei, T.
, 
Jensen, L. J.
, 
von Mering, C.
, &amp; 
Bork, P.
 (2018). eggNOG 5.0: A hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. Nucleic Acids Research, 47(D1), D309–D314. 10.1093/nar/gky1085
</cite> [<a href="https://doi.org/10.1093/nar/gky1085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6324079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30418610/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=eggNOG%205.0:%20A%20hierarchical,%20functionally%20and%20phylogenetically%20annotated%20orthology%20resource%20based%20on%205090%20organisms%20and%202502%20viruses&amp;author=J.%20Huerta%E2%80%90Cepas&amp;author=D.%20Szklarczyk&amp;author=D.%20Heller&amp;author=A.%20Hern%C3%A1ndez%E2%80%90Plaza&amp;author=S.%20K.%20Forslund&amp;volume=47&amp;issue=D1&amp;publication_year=2018&amp;pages=D309-D314&amp;pmid=30418610&amp;doi=10.1093/nar/gky1085&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0038">
<cite>

Jin, H.‐X.
, 
Jiang, M.
, 
Yang, J.‐F.
, 
Wu, Z.‐H.
, 
Ma, L.‐L.
, 
Wang, C.‐C.
, 
Liang, C.
, 
Ning, X.‐Y.
, 
Ge, L.‐F.
, &amp; 
Chen, S.
 (2022). A survey of enhanced cold tolerance and low‐temperature‐induced anthocyanin accumulation in a novel <em>Zoysia japonica</em> biotype. Plants, 11(3), 429. 10.3390/plants11030429

</cite> [<a href="https://doi.org/10.3390/plants11030429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8839389/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35161412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=A%20survey%20of%20enhanced%20cold%20tolerance%20and%20low%E2%80%90temperature%E2%80%90induced%20anthocyanin%20accumulation%20in%20a%20novel%20Zoysia%20japonica%20biotype&amp;author=H.%E2%80%90X.%20Jin&amp;author=M.%20Jiang&amp;author=J.%E2%80%90F.%20Yang&amp;author=Z.%E2%80%90H.%20Wu&amp;author=L.%E2%80%90L.%20Ma&amp;volume=11&amp;issue=3&amp;publication_year=2022&amp;pages=429&amp;pmid=35161412&amp;doi=10.3390/plants11030429&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0039">
<cite>

Kanehisa, M.
, 
Furumichi, M.
, 
Tanabe, M.
, 
Sato, Y.
, &amp; 
Morishima, K.
 (2016). KEGG: New perspectives on genomes, pathways, diseases and drugs. Nucleic Acids Research, 45(D1), D353–D361. 10.1093/nar/gkw1092

</cite> [<a href="https://doi.org/10.1093/nar/gkw1092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5210567/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27899662/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=KEGG:%20New%20perspectives%20on%20genomes,%20pathways,%20diseases%20and%20drugs&amp;author=M.%20Kanehisa&amp;author=M.%20Furumichi&amp;author=M.%20Tanabe&amp;author=Y.%20Sato&amp;author=K.%20Morishima&amp;volume=45&amp;issue=D1&amp;publication_year=2016&amp;pages=D353-D361&amp;pmid=27899662&amp;doi=10.1093/nar/gkw1092&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0040">
<cite>

Kim, D.
, 
Paggi, J. M.
, 
Park, C.
, 
Bennett, C.
, &amp; 
Salzberg, S. L.
 (2019). Graph‐based genome alignment and genotyping with HISAT2 and HISAT‐genotype. Nature Biotechnology, 37(8), 907–915. 10.1038/s41587-019-0201-4
</cite> [<a href="https://doi.org/10.1038/s41587-019-0201-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7605509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31375807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Graph%E2%80%90based%20genome%20alignment%20and%20genotyping%20with%20HISAT2%20and%20HISAT%E2%80%90genotype&amp;author=D.%20Kim&amp;author=J.%20M.%20Paggi&amp;author=C.%20Park&amp;author=C.%20Bennett&amp;author=S.%20L.%20Salzberg&amp;volume=37&amp;issue=8&amp;publication_year=2019&amp;pages=907-915&amp;pmid=31375807&amp;doi=10.1038/s41587-019-0201-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0041">
<cite>

Kole, C.
 (2011). Wild crop relatives: Genomic and breeding resources: Temperate fruits. Springer. 10.1007/978-3-642-14228-4
</cite> [<a href="https://doi.org/10.1007/978-3-642-14228-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Wild%20crop%20relatives:%20Genomic%20and%20breeding%20resources:%20Temperate%20fruits&amp;author=C.%20Kole&amp;publication_year=2011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0042">
<cite>

Kreszies, T.
, 
Eggels, S.
, 
Kreszies, V.
, 
Osthoff, A.
, 
Shellakkutti, N.
, 
Baldauf, J. A.
, 
Zeisler‐Diehl, V. V.
, 
Hochholdinger, F.
, 
Ranathunge, K.
, &amp; 
Schreiber, L.
 (2020). Seminal roots of wild and cultivated barley differentially respond to osmotic stress in gene expression, suberization, and hydraulic conductivity. Plant, Cell &amp; Environment, 43(2), 344–357. 10.1111/pce.13675
</cite> [<a href="https://doi.org/10.1111/pce.13675" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31762057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant,%20Cell%20&amp;%20Environment&amp;title=Seminal%20roots%20of%20wild%20and%20cultivated%20barley%20differentially%20respond%20to%20osmotic%20stress%20in%20gene%20expression,%20suberization,%20and%20hydraulic%20conductivity&amp;author=T.%20Kreszies&amp;author=S.%20Eggels&amp;author=V.%20Kreszies&amp;author=A.%20Osthoff&amp;author=N.%20Shellakkutti&amp;volume=43&amp;issue=2&amp;publication_year=2020&amp;pages=344-357&amp;pmid=31762057&amp;doi=10.1111/pce.13675&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0043">
<cite>

Krzywinski, M.
, 
Schein, J.
, 
Birol, İ.
, 
Connors, J.
, 
Gascoyne, R.
, 
Horsman, D.
, 
Jones, S. J.
, &amp; 
Marra, M. A.
 (2009). Circos: An information aesthetic for comparative genomics. Genome Research, 19(9), 1639–1645. 10.1101/gr.092759.109

</cite> [<a href="https://doi.org/10.1101/gr.092759.109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2752132/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19541911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=Circos:%20An%20information%20aesthetic%20for%20comparative%20genomics&amp;author=M.%20Krzywinski&amp;author=J.%20Schein&amp;author=%C4%B0.%20Birol&amp;author=J.%20Connors&amp;author=R.%20Gascoyne&amp;volume=19&amp;issue=9&amp;publication_year=2009&amp;pages=1639-1645&amp;pmid=19541911&amp;doi=10.1101/gr.092759.109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0044">
<cite>

Lee, J.‐S.
, 
Ilm, B.‐S.
, 
Myeong, H.‐H.
, 
Park, J.‐W.
, &amp; 
Kim, H.‐S.
 (2009). Soil environment analysis and habitat of halophyte for restoration in the salt marshes of southern and western coasts of Korea. Korean Journal of Plant Resources, 22(1), 102–110. 10.7732/kjpr.2014.27.1.102
</cite> [<a href="https://doi.org/10.7732/kjpr.2014.27.1.102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Korean%20Journal%20of%20Plant%20Resources&amp;title=Soil%20environment%20analysis%20and%20habitat%20of%20halophyte%20for%20restoration%20in%20the%20salt%20marshes%20of%20southern%20and%20western%20coasts%20of%20Korea&amp;author=J.%E2%80%90S.%20Lee&amp;author=B.%E2%80%90S.%20Ilm&amp;author=H.%E2%80%90H.%20Myeong&amp;author=J.%E2%80%90W.%20Park&amp;author=H.%E2%80%90S.%20Kim&amp;volume=22&amp;issue=1&amp;publication_year=2009&amp;pages=102-110&amp;doi=10.7732/kjpr.2014.27.1.102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0045">
<cite>

Lee, S. H.
, 
Lee, J.‐S.
, &amp; 
Kim, J.‐W.
 (2018). Relationship between halophyte distribution and soil environmental factors in the west coast of South Korea. Journal of Ecology and Environment, 42, Article 2. 10.1186/s41610-017-0062-z
</cite> [<a href="https://doi.org/10.1186/s41610-017-0062-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Ecology%20and%20Environment&amp;title=Relationship%20between%20halophyte%20distribution%20and%20soil%20environmental%20factors%20in%20the%20west%20coast%20of%20South%20Korea&amp;author=S.%20H.%20Lee&amp;author=J.%E2%80%90S.%20Lee&amp;author=J.%E2%80%90W.%20Kim&amp;volume=42&amp;publication_year=2018&amp;pages=2&amp;doi=10.1186/s41610-017-0062-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0046">
<cite>

Li, H.
 (2013). Aligning sequence reads, clone sequences and assembly contigs with BWA‐MEM. arXiv. 10.48550/arXiv.1303.3997
</cite> [<a href="https://doi.org/10.48550/arXiv.1303.3997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=arXiv&amp;title=Aligning%20sequence%20reads,%20clone%20sequences%20and%20assembly%20contigs%20with%20BWA%E2%80%90MEM&amp;author=H.%20Li&amp;publication_year=2013&amp;doi=10.48550/arXiv.1303.3997&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0047">
<cite>

Li, G.
, 
Tillard, P.
, 
Gojon, A.
, &amp; 
Maurel, C.
 (2016). Dual regulation of root hydraulic conductivity and plasma membrane aquaporins by plant nitrate accumulation and high‐affinity nitrate transporter NRT2.1. Plant and Cell Physiology, 57(4), 733–742. 10.1093/pcp/pcw022

</cite> [<a href="https://doi.org/10.1093/pcp/pcw022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26823528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20and%20Cell%20Physiology&amp;title=Dual%20regulation%20of%20root%20hydraulic%20conductivity%20and%20plasma%20membrane%20aquaporins%20by%20plant%20nitrate%20accumulation%20and%20high%E2%80%90affinity%20nitrate%20transporter%20NRT2.1&amp;author=G.%20Li&amp;author=P.%20Tillard&amp;author=A.%20Gojon&amp;author=C.%20Maurel&amp;volume=57&amp;issue=4&amp;publication_year=2016&amp;pages=733-742&amp;pmid=26823528&amp;doi=10.1093/pcp/pcw022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0048">
<cite>

Li, Y.
, 
Zhou, J.
, 
Li, Z.
, 
Qiao, J.
, 
Quan, R.
, 
Wang, J.
, 
Huang, R.
, &amp; 
Qin, H.
 (2022). SALT AND ABA RESPONSE ERF1 improves seed germination and salt tolerance by repressing ABA signaling in rice. Plant Physiology, 189(2), 1110–1127. 10.1093/plphys/kiac125

</cite> [<a href="https://doi.org/10.1093/plphys/kiac125" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9157093/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35294556/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=SALT%20AND%20ABA%20RESPONSE%20ERF1%20improves%20seed%20germination%20and%20salt%20tolerance%20by%20repressing%20ABA%20signaling%20in%20rice&amp;author=Y.%20Li&amp;author=J.%20Zhou&amp;author=Z.%20Li&amp;author=J.%20Qiao&amp;author=R.%20Quan&amp;volume=189&amp;issue=2&amp;publication_year=2022&amp;pages=1110-1127&amp;pmid=35294556&amp;doi=10.1093/plphys/kiac125&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0049">
<cite>

Liu, H.
, 
An, X.
, 
Liu, X.
, 
Yang, S.
, 
Liu, Y.
, 
Wei, X.
, 
Li, X.
, 
Chen, Q.
, &amp; 
Wang, J.
 (2024). Molecular mechanism of salinity and waterlogging tolerance in mangrove <em>Kandelia obovata</em>
. Frontiers in Plant Science, 15, Article 1354249. 10.3389/fpls.2024.1354249

</cite> [<a href="https://doi.org/10.3389/fpls.2024.1354249" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10879410/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38384752/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Molecular%20mechanism%20of%20salinity%20and%20waterlogging%20tolerance%20in%20mangrove%20Kandelia%20obovata&amp;author=H.%20Liu&amp;author=X.%20An&amp;author=X.%20Liu&amp;author=S.%20Yang&amp;author=Y.%20Liu&amp;volume=15&amp;publication_year=2024&amp;pages=1354249&amp;pmid=38384752&amp;doi=10.3389/fpls.2024.1354249&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0050">
<cite>

Livak, K. J.
, &amp; 
Schmittgen, T. D.
 (2001). Analysis of relative gene expression data using real‐time quantitative PCR and the 2<sup>−ΔΔCT</sup> method. Methods, 25(4), 402–408. 10.1006/meth.2001.1262

</cite> [<a href="https://doi.org/10.1006/meth.2001.1262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11846609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=Analysis%20of%20relative%20gene%20expression%20data%20using%20real%E2%80%90time%20quantitative%20PCR%20and%20the%202%E2%88%92%CE%94%CE%94CT%20method&amp;author=K.%20J.%20Livak&amp;author=T.%20D.%20Schmittgen&amp;volume=25&amp;issue=4&amp;publication_year=2001&amp;pages=402-408&amp;pmid=11846609&amp;doi=10.1006/meth.2001.1262&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0051">
<cite>

Loch, D. S.
, 
Ebina, M.
, 
Choi, J. S.
, &amp; 
Han, L.
 (2017). Ecological implications of <em>Zoysia</em> species, distribution, and adaptation for management and use of zoysiagrasses. International Turfgrass Society Research Journal, 13(1), 11–25. 10.2134/itsrj2016.10.0857
</cite> [<a href="https://doi.org/10.2134/itsrj2016.10.0857" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Turfgrass%20Society%20Research%20Journal&amp;title=Ecological%20implications%20of%20Zoysia%20species,%20distribution,%20and%20adaptation%20for%20management%20and%20use%20of%20zoysiagrasses&amp;author=D.%20S.%20Loch&amp;author=M.%20Ebina&amp;author=J.%20S.%20Choi&amp;author=L.%20Han&amp;volume=13&amp;issue=1&amp;publication_year=2017&amp;pages=11-25&amp;doi=10.2134/itsrj2016.10.0857&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0052">
<cite>

Lulli, F.
, 
Guglielminetti, L.
, 
Grossi, N.
, 
Armeni, R.
, 
Stefanini, S.
, &amp; 
Volterrani, M.
 (2011). Physiological and morphological factors influencing leaf, rhizome and stolon tensile strength in C<sub>4</sub> turfgrass species. Functional Plant Biology, 38(11), 919–926. 10.1071/FP11070

</cite> [<a href="https://doi.org/10.1071/FP11070" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32480949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Functional%20Plant%20Biology&amp;title=Physiological%20and%20morphological%20factors%20influencing%20leaf,%20rhizome%20and%20stolon%20tensile%20strength%20in%20C4%20turfgrass%20species&amp;author=F.%20Lulli&amp;author=L.%20Guglielminetti&amp;author=N.%20Grossi&amp;author=R.%20Armeni&amp;author=S.%20Stefanini&amp;volume=38&amp;issue=11&amp;publication_year=2011&amp;pages=919-926&amp;pmid=32480949&amp;doi=10.1071/FP11070&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0053">
<cite>

Mahmood, U.
, 
Hussain, S.
, 
Hussain, S.
, 
Ali, B.
, 
Ashraf, U.
, 
Zamir, S.
, 
Al‐Robai, S. A.
, 
Alzahrani, F. O.
, 
Hano, C.
, &amp; 
El‐Esawi, M. A.
 (2021). Morpho‐physio‐biochemical and molecular responses of maize hybrids to salinity and waterlogging during stress and recovery phase. Plants, 10(7), Article 1345. 10.3390/plants10071345

</cite> [<a href="https://doi.org/10.3390/plants10071345" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8309398/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34371548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Morpho%E2%80%90physio%E2%80%90biochemical%20and%20molecular%20responses%20of%20maize%20hybrids%20to%20salinity%20and%20waterlogging%20during%20stress%20and%20recovery%20phase&amp;author=U.%20Mahmood&amp;author=S.%20Hussain&amp;author=S.%20Hussain&amp;author=B.%20Ali&amp;author=U.%20Ashraf&amp;volume=10&amp;issue=7&amp;publication_year=2021&amp;pages=1345&amp;pmid=34371548&amp;doi=10.3390/plants10071345&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0054">
<cite>

Manni, M.
, 
Berkeley, M. R.
, 
Seppey, M.
, 
Simão, F. A.
, &amp; 
Zdobnov, E. M.
 (2021). BUSCO update: Novel and streamlined workflows along with broader and deeper phylogenetic coverage for scoring of eukaryotic, prokaryotic, and viral genomes. Molecular Biology and Evolution, 38(10), 4647–4654. 10.1093/molbev/msab199

</cite> [<a href="https://doi.org/10.1093/molbev/msab199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8476166/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34320186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=BUSCO%20update:%20Novel%20and%20streamlined%20workflows%20along%20with%20broader%20and%20deeper%20phylogenetic%20coverage%20for%20scoring%20of%20eukaryotic,%20prokaryotic,%20and%20viral%20genomes&amp;author=M.%20Manni&amp;author=M.%20R.%20Berkeley&amp;author=M.%20Seppey&amp;author=F.%20A.%20Sim%C3%A3o&amp;author=E.%20M.%20Zdobnov&amp;volume=38&amp;issue=10&amp;publication_year=2021&amp;pages=4647-4654&amp;pmid=34320186&amp;doi=10.1093/molbev/msab199&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0055">
<cite>

Mao, T.
, 
Geng, Z.
, 
Zhang, Y.
, 
Xue, W.
, 
Ma, L.
, 
Yang, J.
, 
Jin, Y.
, 
Wang, S.
, 
Zhuo, S.
, 
Zhang, Y.
, 
Yu, C.
, 
Sheng, Y.
, 
Zhang, J.
, &amp; 
Zhang, H.
 (2023). Genome‐wide characterization of NmrA‐like proteins and the regulatory function of soybean GmNmrA6 in response to salt and oxidative stresses. Environmental and Experimental Botany, 213, 105447. 10.1016/j.envexpbot.2023.105447
</cite> [<a href="https://doi.org/10.1016/j.envexpbot.2023.105447" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environmental%20and%20Experimental%20Botany&amp;title=Genome%E2%80%90wide%20characterization%20of%20NmrA%E2%80%90like%20proteins%20and%20the%20regulatory%20function%20of%20soybean%20GmNmrA6%20in%20response%20to%20salt%20and%20oxidative%20stresses&amp;author=T.%20Mao&amp;author=Z.%20Geng&amp;author=Y.%20Zhang&amp;author=W.%20Xue&amp;author=L.%20Ma&amp;volume=213&amp;publication_year=2023&amp;pages=105447&amp;doi=10.1016/j.envexpbot.2023.105447&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0056">
<cite>

Marçais, G.
, &amp; 
Kingsford, C.
 (2011). A fast, lock‐free approach for efficient parallel counting of occurrences of <em>k</em>‐mers. Bioinformatics, 27(6), 764–770. 10.1093/bioinformatics/btr011

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3051319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21217122/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=A%20fast,%20lock%E2%80%90free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k%E2%80%90mers&amp;author=G.%20Mar%C3%A7ais&amp;author=C.%20Kingsford&amp;volume=27&amp;issue=6&amp;publication_year=2011&amp;pages=764-770&amp;pmid=21217122&amp;doi=10.1093/bioinformatics/btr011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0057">
<cite>

Mendes, F. K.
, 
Vanderpool, D.
, 
Fulton, B.
, &amp; 
Hahn, M. W.
 (2020). CAFE 5 models variation in evolutionary rates among gene families. Bioinformatics, 36(22–23), 5516–5518. 10.1093/bioinformatics/btaa1022
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaa1022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33325502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=CAFE%205%20models%20variation%20in%20evolutionary%20rates%20among%20gene%20families&amp;author=F.%20K.%20Mendes&amp;author=D.%20Vanderpool&amp;author=B.%20Fulton&amp;author=M.%20W.%20Hahn&amp;volume=36&amp;issue=22%E2%80%9323&amp;publication_year=2020&amp;pages=5516-5518&amp;pmid=33325502&amp;doi=10.1093/bioinformatics/btaa1022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0058">
<cite>

Mootha, V. K.
, 
Lindgren, C. M.
, 
Eriksson, K.‐F.
, 
Subramanian, A.
, 
Sihag, S.
, 
Lehar, J.
, 
Puigserver, P.
, 
Carlsson, E.
, 
Ridderstråle, M.
, 
Laurila, E.
, 
Houstis, N.
, 
Daly, M. J.
, 
Patterson, N.
, 
Mesirov, J. P.
, 
Golub, T. R.
, 
Tamayo, P.
, 
Spiegelman, B.
, 
Lander, E. S.
, 
Hirschhorn, J. N.
, … 
Groop, L. C.
 (2003). PGC‐1α‐responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes. Nature Genetics, 34(3), 267–273. 10.1038/ng1180

</cite> [<a href="https://doi.org/10.1038/ng1180" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12808457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Genetics&amp;title=PGC%E2%80%901%CE%B1%E2%80%90responsive%20genes%20involved%20in%20oxidative%20phosphorylation%20are%20coordinately%20downregulated%20in%20human%20diabetes&amp;author=V.%20K.%20Mootha&amp;author=C.%20M.%20Lindgren&amp;author=K.%E2%80%90F.%20Eriksson&amp;author=A.%20Subramanian&amp;author=S.%20Sihag&amp;volume=34&amp;issue=3&amp;publication_year=2003&amp;pages=267-273&amp;pmid=12808457&amp;doi=10.1038/ng1180&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0059">
<cite>

Njaci, I.
, 
Waweru, B.
, 
Kamal, N.
, 
Muktar, M. S.
, 
Fisher, D.
, 
Gundlach, H.
, 
Muli, C.
, 
Muthui, L.
, 
Maranga, M.
, 
Kiambi, D.
, 
Maass, B. L.
, 
Emmrich, P. M. F.
, 
Domelevo Entfellner, J.‐B.
, 
Spannagl, M.
, 
Chapman, M. A.
, 
Shorinola, O.
, &amp; 
Jones, C. S.
 (2023). Chromosome‐level genome assembly and population genomic resource to accelerate orphan crop lablab breeding. Nature Communications, 14(1), 1915. 10.1038/s41467-023-37489-7
</cite> [<a href="https://doi.org/10.1038/s41467-023-37489-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10110558/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37069152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Chromosome%E2%80%90level%20genome%20assembly%20and%20population%20genomic%20resource%20to%20accelerate%20orphan%20crop%20lablab%20breeding&amp;author=I.%20Njaci&amp;author=B.%20Waweru&amp;author=N.%20Kamal&amp;author=M.%20S.%20Muktar&amp;author=D.%20Fisher&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=1915&amp;pmid=37069152&amp;doi=10.1038/s41467-023-37489-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0060">
<cite>

Nomberg, G.
, 
Marinov, O.
, 
Arya, G. C.
, 
Manasherova, E.
, &amp; 
Cohen, H.
 (2022). The key enzymes in the suberin biosynthetic pathway in plants: An update. Plants, 11(3), Article 392. 10.3390/plants11030392

</cite> [<a href="https://doi.org/10.3390/plants11030392" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8839845/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35161373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=The%20key%20enzymes%20in%20the%20suberin%20biosynthetic%20pathway%20in%20plants:%20An%20update&amp;author=G.%20Nomberg&amp;author=O.%20Marinov&amp;author=G.%20C.%20Arya&amp;author=E.%20Manasherova&amp;author=H.%20Cohen&amp;volume=11&amp;issue=3&amp;publication_year=2022&amp;pages=392&amp;pmid=35161373&amp;doi=10.3390/plants11030392&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0061">
<cite>

Numata, M.
, 
Miyawaki, A.
, &amp; 
Itow, D.
 (1972). Natural and semi‐natural vegetation in Japan. Blumea: Biodiversity, Evolution and Biogeography of Plants, 20(2), 435–496.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Blumea:%20Biodiversity,%20Evolution%20and%20Biogeography%20of%20Plants&amp;title=Natural%20and%20semi%E2%80%90natural%20vegetation%20in%20Japan&amp;author=M.%20Numata&amp;author=A.%20Miyawaki&amp;author=D.%20Itow&amp;volume=20&amp;issue=2&amp;publication_year=1972&amp;pages=435-496&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0062">
<cite>

Obasa, K.
, 
Fry, J.
, &amp; 
Kennelly, M.
 (2012). Susceptibility of zoysiagrass germplasm to large patch caused by <em>Rhizoctonia solani</em>
. HortScience, 47(9), 1252–1256. 10.21273/HORTSCI.47.9.1252
</cite> [<a href="https://doi.org/10.21273/HORTSCI.47.9.1252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=HortScience&amp;title=Susceptibility%20of%20zoysiagrass%20germplasm%20to%20large%20patch%20caused%20by%20Rhizoctonia%20solani&amp;author=K.%20Obasa&amp;author=J.%20Fry&amp;author=M.%20Kennelly&amp;volume=47&amp;issue=9&amp;publication_year=2012&amp;pages=1252-1256&amp;doi=10.21273/HORTSCI.47.9.1252&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0063">
<cite>

O'Leary, N. A.
, 
Wright, M. W.
, 
Brister, J. R.
, 
Ciufo, S.
, 
Haddad, D.
, 
McVeigh, R.
, 
Rajput, B.
, 
Robbertse, B.
, 
Smith‐White, B.
, 
Ako‐Adjei, D.
, 
Astashyn, A.
, 
Badretdin, A.
, 
Bao, Y.
, 
Blinkova, O.
, 
Brover, V.
, 
Chetvernin, V.
, 
Choi, J.
, 
Cox, E.
, 
Ermolaeva, O.
, … 
Pruitt, K. D.
 (2015). Reference sequence (RefSeq) database at NCBI: Current status, taxonomic expansion, and functional annotation. Nucleic Acids Research, 44(D1), D733–D745. 10.1093/nar/gkv1189

</cite> [<a href="https://doi.org/10.1093/nar/gkv1189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4702849/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26553804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Reference%20sequence%20(RefSeq)%20database%20at%20NCBI:%20Current%20status,%20taxonomic%20expansion,%20and%20functional%20annotation&amp;author=N.%20A.%20O'Leary&amp;author=M.%20W.%20Wright&amp;author=J.%20R.%20Brister&amp;author=S.%20Ciufo&amp;author=D.%20Haddad&amp;volume=44&amp;issue=D1&amp;publication_year=2015&amp;pages=D733-D745&amp;pmid=26553804&amp;doi=10.1093/nar/gkv1189&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0064">
<cite>

Pertea, M.
, 
Kim, D.
, 
Pertea, G. M.
, 
Leek, J. T.
, &amp; 
Salzberg, S. L.
 (2016). Transcript‐level expression analysis of RNA‐seq experiments with HISAT, StringTie and Ballgown. Nature Protocols, 11(9), 1650–1667. 10.1038/nprot.2016.095

</cite> [<a href="https://doi.org/10.1038/nprot.2016.095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5032908/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27560171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Protocols&amp;title=Transcript%E2%80%90level%20expression%20analysis%20of%20RNA%E2%80%90seq%20experiments%20with%20HISAT,%20StringTie%20and%20Ballgown&amp;author=M.%20Pertea&amp;author=D.%20Kim&amp;author=G.%20M.%20Pertea&amp;author=J.%20T.%20Leek&amp;author=S.%20L.%20Salzberg&amp;volume=11&amp;issue=9&amp;publication_year=2016&amp;pages=1650-1667&amp;pmid=27560171&amp;doi=10.1038/nprot.2016.095&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0065">
<cite>

Pertea, G.
, &amp; 
Pertea, M.
 (2020). GFF utilities: GffRead and GffCompare [version 2; peer review: 3 approved]. F1000Research, 9, Article 304. 10.12688/f1000research.23297.2
</cite> [<a href="https://doi.org/10.12688/f1000research.23297.2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7222033/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32489650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=F1000Research&amp;title=GFF%20utilities:%20GffRead%20and%20GffCompare%20%5Bversion%202;%20peer%20review:%203%20approved%5D&amp;author=G.%20Pertea&amp;author=M.%20Pertea&amp;volume=9&amp;publication_year=2020&amp;pages=304&amp;pmid=32489650&amp;doi=10.12688/f1000research.23297.2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0066">
<cite>

Puttick, M. N.
 (2019). MCMCtreeR: Functions to prepare MCMCtree analyses and visualize posterior ages on trees. Bioinformatics, 35(24), 5321–5322. 10.1093/bioinformatics/btz554

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btz554" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31292621/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=MCMCtreeR:%20Functions%20to%20prepare%20MCMCtree%20analyses%20and%20visualize%20posterior%20ages%20on%20trees&amp;author=M.%20N.%20Puttick&amp;volume=35&amp;issue=24&amp;publication_year=2019&amp;pages=5321-5322&amp;pmid=31292621&amp;doi=10.1093/bioinformatics/btz554&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0067">
<cite>

Quinlan, A. R.
 (2014). BEDTools: The Swiss‐Army tool for genome feature analysis. Current Protocols in Bioinformatics, 47(1), 11.12.1–11.12.34. 10.1002/0471250953.bi1112s47
</cite> [<a href="https://doi.org/10.1002/0471250953.bi1112s47" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4213956/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25199790/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Protocols%20in%20Bioinformatics&amp;title=BEDTools:%20The%20Swiss%E2%80%90Army%20tool%20for%20genome%20feature%20analysis&amp;author=A.%20R.%20Quinlan&amp;volume=47&amp;issue=1&amp;publication_year=2014&amp;pages=11.12.1-11.12.34&amp;pmid=25199790&amp;doi=10.1002/0471250953.bi1112s47&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0068">
<cite>

Ranathunge, K.
, 
Lin, J.
, 
Steudle, E.
, &amp; 
Schreiber, L.
 (2011). Stagnant deoxygenated growth enhances root suberization and lignifications, but differentially affects water and NaCl permeabilities in rice (<em>Oryza sativa</em> L.) roots. Plant, Cell &amp; Environment, 34(8), 1223–1240. 10.1111/j.1365-3040.2011.02318.x
</cite> [<a href="https://doi.org/10.1111/j.1365-3040.2011.02318.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21414017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant,%20Cell%20&amp;%20Environment&amp;title=Stagnant%20deoxygenated%20growth%20enhances%20root%20suberization%20and%20lignifications,%20but%20differentially%20affects%20water%20and%20NaCl%20permeabilities%20in%20rice%20(Oryza%20sativa%20L.)%20roots&amp;author=K.%20Ranathunge&amp;author=J.%20Lin&amp;author=E.%20Steudle&amp;author=L.%20Schreiber&amp;volume=34&amp;issue=8&amp;publication_year=2011&amp;pages=1223-1240&amp;pmid=21414017&amp;doi=10.1111/j.1365-3040.2011.02318.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0069">
<cite>

Rannala, B.
, &amp; 
Yang, Z.
 (2007). Inferring speciation times under an episodic molecular clock. Systematic Biology, 56(3), 453–466. 10.1080/10635150701420643

</cite> [<a href="https://doi.org/10.1080/10635150701420643" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17558967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Systematic%20Biology&amp;title=Inferring%20speciation%20times%20under%20an%20episodic%20molecular%20clock&amp;author=B.%20Rannala&amp;author=Z.%20Yang&amp;volume=56&amp;issue=3&amp;publication_year=2007&amp;pages=453-466&amp;pmid=17558967&amp;doi=10.1080/10635150701420643&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0070">
<cite>

Rhie, A.
, 
Walenz, B. P.
, 
Koren, S.
, &amp; 
Phillippy, A. M.
 (2020). Merqury: Reference‐free quality, completeness, and phasing assessment for genome assemblies. Genome Biology, 21(1), Article 245. 10.1186/s13059-020-02134-9

</cite> [<a href="https://doi.org/10.1186/s13059-020-02134-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7488777/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32928274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Merqury:%20Reference%E2%80%90free%20quality,%20completeness,%20and%20phasing%20assessment%20for%20genome%20assemblies&amp;author=A.%20Rhie&amp;author=B.%20P.%20Walenz&amp;author=S.%20Koren&amp;author=A.%20M.%20Phillippy&amp;volume=21&amp;issue=1&amp;publication_year=2020&amp;pages=245&amp;pmid=32928274&amp;doi=10.1186/s13059-020-02134-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0071">
<cite>

Robinson, D. A.
, 
Campbell, C. S.
, 
Hopmans, J. W.
, 
Hornbuckle, B. K.
, 
Jones, S. B.
, 
Knight, R.
, 
Ogden, F.
, 
Selker, J.
, &amp; 
Wendroth, O.
 (2008). Soil moisture measurement for ecological and hydrological watershed‐scale observatories: A review. Vadose Zone Journal, 7(1), 358–389. 10.2136/vzj2007.0143
</cite> [<a href="https://doi.org/10.2136/vzj2007.0143" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Vadose%20Zone%20Journal&amp;title=Soil%20moisture%20measurement%20for%20ecological%20and%20hydrological%20watershed%E2%80%90scale%20observatories:%20A%20review&amp;author=D.%20A.%20Robinson&amp;author=C.%20S.%20Campbell&amp;author=J.%20W.%20Hopmans&amp;author=B.%20K.%20Hornbuckle&amp;author=S.%20B.%20Jones&amp;volume=7&amp;issue=1&amp;publication_year=2008&amp;pages=358-389&amp;doi=10.2136/vzj2007.0143&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0072">
<cite>

Salmela, L.
, &amp; 
Rivals, E.
 (2014). LoRDEC: Accurate and efficient long read error correction. Bioinformatics, 30(24), 3506–3514. 10.1093/bioinformatics/btu538

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu538" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4253826/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25165095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=LoRDEC:%20Accurate%20and%20efficient%20long%20read%20error%20correction&amp;author=L.%20Salmela&amp;author=E.%20Rivals&amp;volume=30&amp;issue=24&amp;publication_year=2014&amp;pages=3506-3514&amp;pmid=25165095&amp;doi=10.1093/bioinformatics/btu538&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0073">
<cite>

Schmieder, R.
, &amp; 
Edwards, R.
 (2011). Quality control and preprocessing of metagenomic datasets. Bioinformatics, 27(6), 863–864. 10.1093/bioinformatics/btr026

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3051327/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21278185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Quality%20control%20and%20preprocessing%20of%20metagenomic%20datasets&amp;author=R.%20Schmieder&amp;author=R.%20Edwards&amp;volume=27&amp;issue=6&amp;publication_year=2011&amp;pages=863-864&amp;pmid=21278185&amp;doi=10.1093/bioinformatics/btr026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0074">
<cite>

Setter, T. L.
, 
Bhekasut, P.
, &amp; 
Greenway, H.
 (2010). Desiccation of leaves after de‐submergence is one cause for intolerance to complete submergence of the rice cultivar IR 42. Functional Plant Biology, 37(11), 1096–1104. 10.1071/FP10025
</cite> [<a href="https://doi.org/10.1071/FP10025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Functional%20Plant%20Biology&amp;title=Desiccation%20of%20leaves%20after%20de%E2%80%90submergence%20is%20one%20cause%20for%20intolerance%20to%20complete%20submergence%20of%20the%20rice%20cultivar%20IR%2042&amp;author=T.%20L.%20Setter&amp;author=P.%20Bhekasut&amp;author=H.%20Greenway&amp;volume=37&amp;issue=11&amp;publication_year=2010&amp;pages=1096-1104&amp;doi=10.1071/FP10025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0075">
<cite>

Shahzad, Z.
, 
Canut, M.
, 
Tournaire‐Roux, C.
, 
Martinière, A.
, 
Boursiac, Y.
, 
Loudet, O.
, &amp; 
Maurel, C.
 (2016). A potassium‐dependent oxygen sensing pathway regulates plant root hydraulics. Cell, 167(1), 87–98. e14. 10.1016/j.cell.2016.08.068

</cite> [<a href="https://doi.org/10.1016/j.cell.2016.08.068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27641502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20potassium%E2%80%90dependent%20oxygen%20sensing%20pathway%20regulates%20plant%20root%20hydraulics&amp;author=Z.%20Shahzad&amp;author=M.%20Canut&amp;author=C.%20Tournaire%E2%80%90Roux&amp;author=A.%20Martini%C3%A8re&amp;author=Y.%20Boursiac&amp;volume=167&amp;issue=1&amp;publication_year=2016&amp;pages=87-98&amp;pmid=27641502&amp;doi=10.1016/j.cell.2016.08.068&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0076">
<cite>

Sherman, B. T.
, 
Hao, M.
, 
Qiu, J.
, 
Jiao, X.
, 
Baseler, M. W.
, 
Lane, H. C.
, 
Imamichi, T.
, &amp; 
Chang, W.
 (2022). DAVID: A web server for functional enrichment analysis and functional annotation of gene lists (2021 update). Nucleic Acids Research, 50(W1), W216–W221. 10.1093/nar/gkac194

</cite> [<a href="https://doi.org/10.1093/nar/gkac194" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9252805/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35325185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=DAVID:%20A%20web%20server%20for%20functional%20enrichment%20analysis%20and%20functional%20annotation%20of%20gene%20lists%20(2021%20update)&amp;author=B.%20T.%20Sherman&amp;author=M.%20Hao&amp;author=J.%20Qiu&amp;author=X.%20Jiao&amp;author=M.%20W.%20Baseler&amp;volume=50&amp;issue=W1&amp;publication_year=2022&amp;pages=W216-W221&amp;pmid=35325185&amp;doi=10.1093/nar/gkac194&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0077">
<cite>

Su, B.‐Y.
, 
Wang, Y.‐S.
, &amp; 
Sun, C.‐C.
 (2022). Response adaptive mechanisms of three mangrove (<em>Avicennia marina</em>, <em>Aegiceras corniculatum</em>, and <em>Bruguiera gymnorrhiza</em>) plants to waterlogging stress revealed by transcriptome analysis. Frontiers in Marine Science, 9, Article 929649. 10.3389/fmars.2022.929649
</cite> [<a href="https://doi.org/10.3389/fmars.2022.929649" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Marine%20Science&amp;title=Response%20adaptive%20mechanisms%20of%20three%20mangrove%20(Avicennia%20marina,%20Aegiceras%20corniculatum,%20and%20Bruguiera%20gymnorrhiza)%20plants%20to%20waterlogging%20stress%20revealed%20by%20transcriptome%20analysis&amp;author=B.%E2%80%90Y.%20Su&amp;author=Y.%E2%80%90S.%20Wang&amp;author=C.%E2%80%90C.%20Sun&amp;volume=9&amp;publication_year=2022&amp;pages=929649&amp;doi=10.3389/fmars.2022.929649&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0078">
<cite>

Subramanian, A.
, 
Tamayo, P.
, 
Mootha, V. K.
, 
Mukherjee, S.
, 
Ebert, B. L.
, 
Gillette, M. A.
, 
Paulovich, A.
, 
Pomeroy, S. L.
, 
Golub, T. R.
, 
Lander, E. S.
, &amp; 
Mesirov, J. P.
 (2005). Gene set enrichment analysis: A knowledge‐based approach for interpreting genome‐wide expression profiles. Proceedings of the National Academy of Sciences of the United States of America, 102(43), 15545–15550. 10.1073/pnas.0506580102

</cite> [<a href="https://doi.org/10.1073/pnas.0506580102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1239896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16199517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Gene%20set%20enrichment%20analysis:%20A%20knowledge%E2%80%90based%20approach%20for%20interpreting%20genome%E2%80%90wide%20expression%20profiles&amp;author=A.%20Subramanian&amp;author=P.%20Tamayo&amp;author=V.%20K.%20Mootha&amp;author=S.%20Mukherjee&amp;author=B.%20L.%20Ebert&amp;volume=102&amp;issue=43&amp;publication_year=2005&amp;pages=15545-15550&amp;pmid=16199517&amp;doi=10.1073/pnas.0506580102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0079">
<cite>

Sukumar, P.
, 
Maloney, G. S.
, &amp; 
Muday, G. K.
 (2013). Localized induction of the ATP‐binding cassette B19 auxin transporter enhances adventitious root formation in Arabidopsis. Plant Physiology, 162(3), 1392–1405. 10.1104/pp.113.217174

</cite> [<a href="https://doi.org/10.1104/pp.113.217174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3707546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23677937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Localized%20induction%20of%20the%20ATP%E2%80%90binding%20cassette%20B19%20auxin%20transporter%20enhances%20adventitious%20root%20formation%20in%20Arabidopsis&amp;author=P.%20Sukumar&amp;author=G.%20S.%20Maloney&amp;author=G.%20K.%20Muday&amp;volume=162&amp;issue=3&amp;publication_year=2013&amp;pages=1392-1405&amp;pmid=23677937&amp;doi=10.1104/pp.113.217174&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0080">
<cite>

Suyama, M.
, 
Torrents, D.
, &amp; 
Bork, P.
 (2006). PAL2NAL: Robust conversion of protein sequence alignments into the corresponding codon alignments. Nucleic Acids Research, 34(Suppl_2), W609–W612. 10.1093/nar/gkl315

</cite> [<a href="https://doi.org/10.1093/nar/gkl315" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1538804/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16845082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=PAL2NAL:%20Robust%20conversion%20of%20protein%20sequence%20alignments%20into%20the%20corresponding%20codon%20alignments&amp;author=M.%20Suyama&amp;author=D.%20Torrents&amp;author=P.%20Bork&amp;volume=34&amp;publication_year=2006&amp;pages=W609-W612&amp;pmid=16845082&amp;doi=10.1093/nar/gkl315&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0081">
<cite>

Tan, X.
, 
Liu, M.
, 
Du, N.
, &amp; 
Zwiazek, J. J.
 (2021). Ethylene enhances root water transport and aquaporin expression in trembling aspen (<em>Populus tremuloides</em>) exposed to root hypoxia. BMC Plant Biology, 21(1), Article 227. 10.1186/s12870-021-02995-7

</cite> [<a href="https://doi.org/10.1186/s12870-021-02995-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8140438/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34020594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Ethylene%20enhances%20root%20water%20transport%20and%20aquaporin%20expression%20in%20trembling%20aspen%20(Populus%20tremuloides)%20exposed%20to%20root%20hypoxia&amp;author=X.%20Tan&amp;author=M.%20Liu&amp;author=N.%20Du&amp;author=J.%20J.%20Zwiazek&amp;volume=21&amp;issue=1&amp;publication_year=2021&amp;pages=227&amp;pmid=34020594&amp;doi=10.1186/s12870-021-02995-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0082">
<cite>

Tanaka, H.
, 
Hirakawa, H.
, 
Kosugi, S.
, 
Nakayama, S.
, 
Ono, A.
, 
Watanabe, A.
, 
Hashiguchi, M.
, 
Gondo, T.
, 
Ishigaki, G.
, 
Muguerza, M.
, 
Shimizu, K.
, 
Sawamura, N.
, 
Inoue, T.
, 
Shigeki, Y.
, 
Ohno, N.
, 
Tabata, S.
, 
Akashi, R.
, &amp; 
Sato, S.
 (2016). Sequencing and comparative analyses of the genomes of zoysiagrasses. DNA Research, 23(2), 171–180. 10.1093/dnares/dsw006

</cite> [<a href="https://doi.org/10.1093/dnares/dsw006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4833424/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26975196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=DNA%20Research&amp;title=Sequencing%20and%20comparative%20analyses%20of%20the%20genomes%20of%20zoysiagrasses&amp;author=H.%20Tanaka&amp;author=H.%20Hirakawa&amp;author=S.%20Kosugi&amp;author=S.%20Nakayama&amp;author=A.%20Ono&amp;volume=23&amp;issue=2&amp;publication_year=2016&amp;pages=171-180&amp;pmid=26975196&amp;doi=10.1093/dnares/dsw006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0083">
<cite>

Tang, H.
, 
Krishnakumar, V.
, 
Zeng, X.
, 
Xu, Z.
, 
Taranto, A.
, 
Lomas, J. S.
, 
Zhang, Y.
, 
Huang, Y.
, 
Wang, Y.
, 
Yim, W. C.
, 
Zhang, J.
, &amp; 
Zhang, X.
 (2024). JCVI: A versatile toolkit for comparative genomics analysis. iMeta, 3(4), Article e211. 10.1002/imt2.211

</cite> [<a href="https://doi.org/10.1002/imt2.211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11316928/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39135687/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=iMeta&amp;title=JCVI:%20A%20versatile%20toolkit%20for%20comparative%20genomics%20analysis&amp;author=H.%20Tang&amp;author=V.%20Krishnakumar&amp;author=X.%20Zeng&amp;author=Z.%20Xu&amp;author=A.%20Taranto&amp;volume=3&amp;issue=4&amp;publication_year=2024&amp;pages=e211&amp;pmid=39135687&amp;doi=10.1002/imt2.211&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0084">
<cite>
The Gene Ontology Consortium
. (2014). Gene Ontology Consortium: Going forward. Nucleic Acids Research, 43(D1), D1049–D1056. 10.1093/nar/gku1179

</cite> [<a href="https://doi.org/10.1093/nar/gku1179" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4383973/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25428369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Gene%20Ontology%20Consortium:%20Going%20forward&amp;volume=43&amp;issue=D1&amp;publication_year=2014&amp;pages=D1049-D1056&amp;pmid=25428369&amp;doi=10.1093/nar/gku1179&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0085">
<cite>

Ursache, R.
, 
De Jesus Vieira Teixeira, C.
, 
Dénervaud Tendon, V.
, 
Gully, K.
, 
De Bellis, D.
, 
Schmid‐Siegert, E.
, 
Grube Andersen, T.
, 
Shekhar, V.
, 
Calderon, S.
, 
Pradervand, S.
, 
Nawrath, C.
, 
Geldner, N.
, &amp; 
Vermeer, J. E. M.
 (2021). GDSL‐domain proteins have key roles in suberin polymerization and degradation. Nature Plants, 7(3), 353–364. 10.1038/s41477-021-00862-9

</cite> [<a href="https://doi.org/10.1038/s41477-021-00862-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7610369/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33686223/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Plants&amp;title=GDSL%E2%80%90domain%20proteins%20have%20key%20roles%20in%20suberin%20polymerization%20and%20degradation&amp;author=R.%20Ursache&amp;author=C.%20De%20Jesus%20Vieira%20Teixeira&amp;author=V.%20D%C3%A9nervaud%20Tendon&amp;author=K.%20Gully&amp;author=D.%20De%20Bellis&amp;volume=7&amp;issue=3&amp;publication_year=2021&amp;pages=353-364&amp;pmid=33686223&amp;doi=10.1038/s41477-021-00862-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0086">
<cite>

Vurture, G. W.
, 
Sedlazeck, F. J.
, 
Nattestad, M.
, 
Underwood, C. J.
, 
Fang, H.
, 
Gurtowski, J.
, &amp; 
Schatz, M. C.
 (2017). GenomeScope: Fast reference‐free genome profiling from short reads. Bioinformatics, 33(14), 2202–2204. 10.1093/bioinformatics/btx153

</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5870704/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28369201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GenomeScope:%20Fast%20reference%E2%80%90free%20genome%20profiling%20from%20short%20reads&amp;author=G.%20W.%20Vurture&amp;author=F.%20J.%20Sedlazeck&amp;author=M.%20Nattestad&amp;author=C.%20J.%20Underwood&amp;author=H.%20Fang&amp;volume=33&amp;issue=14&amp;publication_year=2017&amp;pages=2202-2204&amp;pmid=28369201&amp;doi=10.1093/bioinformatics/btx153&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0087">
<cite>

Wang, F.
, 
Singh, R.
, 
Genovesi, A. D.
, 
Wai, C. M.
, 
Huang, X.
, 
Chandra, A.
, &amp; 
Yu, Q.
 (2015). Sequence‐tagged high‐density genetic maps of <em>Zoysia japonica</em> provide insights into genome evolution in Chloridoideae. The Plant Journal, 82(5), 744–757. 10.1111/tpj.12842

</cite> [<a href="https://doi.org/10.1111/tpj.12842" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25846381/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=Sequence%E2%80%90tagged%20high%E2%80%90density%20genetic%20maps%20of%20Zoysia%20japonica%20provide%20insights%20into%20genome%20evolution%20in%20Chloridoideae&amp;author=F.%20Wang&amp;author=R.%20Singh&amp;author=A.%20D.%20Genovesi&amp;author=C.%20M.%20Wai&amp;author=X.%20Huang&amp;volume=82&amp;issue=5&amp;publication_year=2015&amp;pages=744-757&amp;pmid=25846381&amp;doi=10.1111/tpj.12842&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0088">
<cite>

Wang, W.
, 
Shao, A.
, 
Xu, X.
, 
Fan, S.
, &amp; 
Fu, J.
 (2022). Comparative genomics reveals the molecular mechanism of salt adaptation for zoysiagrasses. BMC Plant Biology, 22(1), Article 355. 10.1186/s12870-022-03752-0

</cite> [<a href="https://doi.org/10.1186/s12870-022-03752-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9306052/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35864464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Comparative%20genomics%20reveals%20the%20molecular%20mechanism%20of%20salt%20adaptation%20for%20zoysiagrasses&amp;author=W.%20Wang&amp;author=A.%20Shao&amp;author=X.%20Xu&amp;author=S.%20Fan&amp;author=J.%20Fu&amp;volume=22&amp;issue=1&amp;publication_year=2022&amp;pages=355&amp;pmid=35864464&amp;doi=10.1186/s12870-022-03752-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0089">
<cite>

Wang, Y.
, 
Tang, H.
, 
DeBarry, J. D.
, 
Tan, X.
, 
Li, J.
, 
Wang, X.
, 
Lee, T.‐H.
, 
Jin, H.
, 
Marler, B.
, 
Guo, H.
, 
Kissinger, J. C.
, &amp; 
Paterson, A. H.
 (2012). MCScanX: A toolkit for detection and evolutionary analysis of gene synteny and collinearity. Nucleic Acids Research, 40(7), e49–e49. 10.1093/nar/gkr1293

</cite> [<a href="https://doi.org/10.1093/nar/gkr1293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3326336/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22217600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=MCScanX:%20A%20toolkit%20for%20detection%20and%20evolutionary%20analysis%20of%20gene%20synteny%20and%20collinearity&amp;author=Y.%20Wang&amp;author=H.%20Tang&amp;author=J.%20D.%20DeBarry&amp;author=X.%20Tan&amp;author=J.%20Li&amp;volume=40&amp;issue=7&amp;publication_year=2012&amp;pages=e49-e49&amp;pmid=22217600&amp;doi=10.1093/nar/gkr1293&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0090">
<cite>

Wei, S.
, 
Du, Z.
, 
Gao, F.
, 
Ke, X.
, 
Li, J.
, 
Liu, J.
, &amp; 
Zhou, Y.
 (2015). Global transcriptome profiles of ‘Meyer’ zoysiagrass in response to cold stress. PLoS ONE, 10(6), Article e0131153. 10.1371/journal.pone.0131153

</cite> [<a href="https://doi.org/10.1371/journal.pone.0131153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4482698/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26115186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Global%20transcriptome%20profiles%20of%20%E2%80%98Meyer%E2%80%99%20zoysiagrass%20in%20response%20to%20cold%20stress&amp;author=S.%20Wei&amp;author=Z.%20Du&amp;author=F.%20Gao&amp;author=X.%20Ke&amp;author=J.%20Li&amp;volume=10&amp;issue=6&amp;publication_year=2015&amp;pages=e0131153&amp;pmid=26115186&amp;doi=10.1371/journal.pone.0131153&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0091">
<cite>

Wu, W.
, 
Feng, X.
, 
Wang, N.
, 
Shao, S.
, 
Liu, M.
, 
Si, F.
, 
Chen, L.
, 
Jin, C.
, 
Xu, S.
, 
Guo, Z.
, 
Zhong, C.
, 
Shi, S.
, &amp; 
He, Z.
 (2024). Genomic analysis of <em>Nypa fruticans</em> elucidates its intertidal adaptations and early palm evolution. Journal of Integrative Plant Biology, 66(4), 824–843. 10.1111/jipb.13625

</cite> [<a href="https://doi.org/10.1111/jipb.13625" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38372488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Integrative%20Plant%20Biology&amp;title=Genomic%20analysis%20of%20Nypa%20fruticans%20elucidates%20its%20intertidal%20adaptations%20and%20early%20palm%20evolution&amp;author=W.%20Wu&amp;author=X.%20Feng&amp;author=N.%20Wang&amp;author=S.%20Shao&amp;author=M.%20Liu&amp;volume=66&amp;issue=4&amp;publication_year=2024&amp;pages=824-843&amp;pmid=38372488&amp;doi=10.1111/jipb.13625&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0092">
<cite>

Xu, Q.
, 
Chen, S.
, 
Yunjuan, R.
, 
Chen, S.
, &amp; 
Liesche, J.
 (2017). Regulation of sucrose transporters and phloem loading in response to environmental cues. Plant Physiology, 176(1), 930–945. 10.1104/pp.17.01088

</cite> [<a href="https://doi.org/10.1104/pp.17.01088" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5761784/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29158330/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Regulation%20of%20sucrose%20transporters%20and%20phloem%20loading%20in%20response%20to%20environmental%20cues&amp;author=Q.%20Xu&amp;author=S.%20Chen&amp;author=R.%20Yunjuan&amp;author=S.%20Chen&amp;author=J.%20Liesche&amp;volume=176&amp;issue=1&amp;publication_year=2017&amp;pages=930-945&amp;pmid=29158330&amp;doi=10.1104/pp.17.01088&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0093">
<cite>

Yates, A. D.
, 
Allen, J.
, 
Amode, R. M.
, 
Azov, A. G.
, 
Barba, M.
, 
Becerra, A.
, 
Bhai, J.
, 
Campbell, L. I.
, 
Carbajo Martinez, M.
, 
Chakiachvili, M.
, 
Chougule, K.
, 
Christensen, M.
, 
Contreras‐Moreira, B.
, 
Cuzick, A.
, 
Da Rin Fioretto, L.
, 
Davis, P.
, 
De Silva, N. H.
, 
Diamantakis, S.
, 
Dyer, S.
, … 
Flicek, P.
 (2021). Ensembl Genomes 2022: An expanding genome resource for non‐vertebrates. Nucleic Acids Research, 50(D1), D996–D1003. 10.1093/nar/gkab1007
</cite> [<a href="https://doi.org/10.1093/nar/gkab1007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34791415/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Ensembl%20Genomes%202022:%20An%20expanding%20genome%20resource%20for%20non%E2%80%90vertebrates&amp;author=A.%20D.%20Yates&amp;author=J.%20Allen&amp;author=R.%20M.%20Amode&amp;author=A.%20G.%20Azov&amp;author=M.%20Barba&amp;volume=50&amp;issue=D1&amp;publication_year=2021&amp;pages=D996-D1003&amp;pmid=34791415&amp;doi=10.1093/nar/gkab1007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0094">
<cite>

Yeung, E.
, 
van Veen, H.
, 
Vashisht, D.
, 
Sobral Paiva, A. L.
, 
Hummel, M.
, 
Rankenberg, T.
, 
Steffens, B.
, 
Steffen‐Heins, A.
, 
Sauter, M.
, 
de Vries, M.
, 
Schuurink, R. C.
, 
Bazin, J.
, 
Bailey‐Serres, J.
, 
Voesenek, L. A. C. J.
, &amp; 
Sasidharan, R.
 (2018). A stress recovery signaling network for enhanced flooding tolerance in <em>Arabidopsis thaliana</em>
. Proceedings of the National Academy of Sciences of the United States of America, 115(26), E6085–E6094. 10.1073/pnas.1803841115

</cite> [<a href="https://doi.org/10.1073/pnas.1803841115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6042063/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29891679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=A%20stress%20recovery%20signaling%20network%20for%20enhanced%20flooding%20tolerance%20in%20Arabidopsis%20thaliana&amp;author=E.%20Yeung&amp;author=H.%20van%20Veen&amp;author=D.%20Vashisht&amp;author=A.%20L.%20Sobral%20Paiva&amp;author=M.%20Hummel&amp;volume=115&amp;issue=26&amp;publication_year=2018&amp;pages=E6085-E6094&amp;pmid=29891679&amp;doi=10.1073/pnas.1803841115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0095">
<cite>

Zhang, Z.
, 
Zhang, Q.
, 
Wu, J.
, 
Zheng, X.
, 
Zheng, S.
, 
Sun, X.
, 
Qiu, Q.
, &amp; 
Lu, T.
 (2013). Gene knockout study reveals that cytosolic ascorbate peroxidase 2(OsAPX2) plays a critical role in growth and reproduction in rice under drought, salt and cold stresses. PLoS ONE, 8(2), Article e57472. 10.1371/journal.pone.0057472

</cite> [<a href="https://doi.org/10.1371/journal.pone.0057472" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3585366/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23468992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Gene%20knockout%20study%20reveals%20that%20cytosolic%20ascorbate%20peroxidase%202(OsAPX2)%20plays%20a%20critical%20role%20in%20growth%20and%20reproduction%20in%20rice%20under%20drought,%20salt%20and%20cold%20stresses&amp;author=Z.%20Zhang&amp;author=Q.%20Zhang&amp;author=J.%20Wu&amp;author=X.%20Zheng&amp;author=S.%20Zheng&amp;volume=8&amp;issue=2&amp;publication_year=2013&amp;pages=e57472&amp;pmid=23468992&amp;doi=10.1371/journal.pone.0057472&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0096">
<cite>

Zheng, Y.
, 
Zong, J.
, 
Liu, J.
, 
Wang, R.
, 
Chen, J.
, 
Guo, H.
, 
Kong, W.
, 
Liu, J.
, &amp; 
Chen, Y.
 (2022). Mining for salt‐tolerant genes from halophyte <em>Zoysia matrella</em> using FOX system and functional analysis of ZmGnTL. Frontiers in Plant Science, 13, Article 1063436. 10.3389/fpls.2022.1063436

</cite> [<a href="https://doi.org/10.3389/fpls.2022.1063436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9714509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36466287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Mining%20for%20salt%E2%80%90tolerant%20genes%20from%20halophyte%20Zoysia%20matrella%20using%20FOX%20system%20and%20functional%20analysis%20of%20ZmGnTL&amp;author=Y.%20Zheng&amp;author=J.%20Zong&amp;author=J.%20Liu&amp;author=R.%20Wang&amp;author=J.%20Chen&amp;volume=13&amp;publication_year=2022&amp;pages=1063436&amp;pmid=36466287&amp;doi=10.3389/fpls.2022.1063436&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270070-bib-0097">
<cite>

Zhou, X.
, 
Zhang, Z.‐L.
, 
Park, J.
, 
Tyler, L.
, 
Yusuke, J.
, 
Qiu, K.
, 
Nam, E. A.
, 
Lumba, S.
, 
Desveaux, D.
, 
McCourt, P.
, 
Kamiya, Y.
, &amp; 
Sun, T.‐P.
 (2016). The ERF11 transcription factor promotes internode elongation by activating gibberellin biosynthesis and signaling. Plant Physiology, 171(4), 2760–2770. 10.1104/pp.16.00154

</cite> [<a href="https://doi.org/10.1104/pp.16.00154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4972265/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27255484/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=The%20ERF11%20transcription%20factor%20promotes%20internode%20elongation%20by%20activating%20gibberellin%20biosynthesis%20and%20signaling&amp;author=X.%20Zhou&amp;author=Z.%E2%80%90L.%20Zhang&amp;author=J.%20Park&amp;author=L.%20Tyler&amp;author=J.%20Yusuke&amp;volume=171&amp;issue=4&amp;publication_year=2016&amp;pages=2760-2770&amp;pmid=27255484&amp;doi=10.1104/pp.16.00154&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Figure S1. Genome size estimation of <em>Zoysia sinica</em> based on k‐mer distribution analysis; Figure S2. Chromosome Ideogram; Figure S3. BUSCO assessment of protein‐coding genes in four <em>Zoysia</em> species; Figure S4. Analysis of positively selected genes in <em>Zoysia</em> species. Figure S5. Analysis of positively selected genes in <em>Z. sinica</em> and <em>S. alterniflora</em>; Figure S6. Physical characteristics of soil in intertidal environments; Figure S7. PCA analysis of transcriptome data replicates; Figure S8. qRT‐PCR correlation analysis; Table S1. Liliopsida NCBI protein sets; Table S2. Public RNAseq data for Zoysia genus reannotation; Table S3. qRT‐PCR primer information; Table S4. The statistical information of raw and trimmed Illumina data; Table S5. The statistical information of Oxford Nanopore data; Table S6. The scaffolding statistical results of <em>Zoysia sinica</em> genome; Table S7. Statistics of the <em>Zoysia</em> genomes; Table S8. Repeat element annotation; Table S9. Macrosynteny in <em>Z.sinica</em> genome; Table S10.Zoysia genus gene annotation; Table S11. Positive‐selected genes of the <em>Zoysia</em> species; Table S12. Positive‐selected genes of <em>Z. sinica</em> and <em>S. alterniflora</em>; Table S13. Soil characteristics; Table S14. Statistics of Soil characteristics; Table S15. Statistics of RNAseq data;</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12238702/bin/TPG2-18-e70070-s001.zip" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70070-s001.zip</a><sup> (2MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All sequence read files for the assembled genome have been deposited in the Sequence Read Archive database at NCBI under BioProject IDs PRJNA1149339. The Whole Genome Shotgun (WGS) project has been deposited at DDBJ/ENA/GenBank under the accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/JBGRUJ000000000" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">JBGRUJ000000000</a> and can be accessed using this number. Additionally, the gene annotation results for four <em>Zoysia</em> species (<em>Z. sinica</em>, <em>Z. japonica</em>, <em>Z. matrella</em>, and <em>Z. pacifica</em>) have been shared via Figshare (<a href="https://doi.org/10.6084/m9.figshare.27888426.v1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.6084/m9.figshare.27888426.v1</a>).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70070"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70070.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12238702/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12238702/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12238702%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12238702/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12238702/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12238702/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40629672/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12238702/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40629672/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12238702/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12238702/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="MzyAPp14tQnPapCTAdFHnRKLbm760Ypc2jBkDWr2KUKJuqxhmVl87mTpMZpFaBTa">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-d9849939.js"></script>
    
    

    </body>
</html>
