
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            AND-1 is a critical regulator of R-loop dynamics and a target to overcome endocrine resistance - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="FF2082FE89B753831A82FE0025ED3246.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="sciadv">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327453/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Science Advances">
<meta name="citation_title" content="AND-1 is a critical regulator of R-loop dynamics and a target to overcome endocrine resistance">
<meta name="citation_author" content="Zhuqing Li">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Medicine, George Washington University School of Medicine and Health Sciences, Washington, DC, USA.">
<meta name="citation_author_institution" content="GW Cancer Center, George Washington University, Washington, DC, USA.">
<meta name="citation_author" content="Yihan Peng">
<meta name="citation_author_institution" content="Department of Oncology, Georgetown Lombardi Comprehensive Cancer Center, Georgetown University Medical Center, Washington, DC, USA.">
<meta name="citation_author" content="Huai-Chin Chiang">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Medicine, George Washington University School of Medicine and Health Sciences, Washington, DC, USA.">
<meta name="citation_author" content="Yanjun Gao">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Medicine, George Washington University School of Medicine and Health Sciences, Washington, DC, USA.">
<meta name="citation_author_institution" content="GW Cancer Center, George Washington University, Washington, DC, USA.">
<meta name="citation_author" content="Chen Chu">
<meta name="citation_author_institution" content="Department of Cancer Biology, Dana-Farber Cancer Institute, Boston, MA, USA.">
<meta name="citation_author_institution" content="Department of Genetics, Blavatnik Institute, Harvard Medical School, Boston, MA, USA.">
<meta name="citation_author" content="Yi Zhang">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Medicine, George Washington University School of Medicine and Health Sciences, Washington, DC, USA.">
<meta name="citation_author_institution" content="GW Cancer Center, George Washington University, Washington, DC, USA.">
<meta name="citation_author" content="Muhammad Jameel Mughal">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Medicine, George Washington University School of Medicine and Health Sciences, Washington, DC, USA.">
<meta name="citation_author_institution" content="GW Cancer Center, George Washington University, Washington, DC, USA.">
<meta name="citation_author" content="Chengyu Liu">
<meta name="citation_author_institution" content="The Transgenic Core Facility, National Heart, Lung, and Blood Institute, National Institutes of Health, Bethesda, MD, USA.">
<meta name="citation_author" content="Rong Li">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Medicine, George Washington University School of Medicine and Health Sciences, Washington, DC, USA.">
<meta name="citation_author" content="Huadong Pei">
<meta name="citation_author_institution" content="Department of Oncology, Georgetown Lombardi Comprehensive Cancer Center, Georgetown University Medical Center, Washington, DC, USA.">
<meta name="citation_author" content="Wenge Zhu">
<meta name="citation_author_institution" content="Department of Biochemistry and Molecular Medicine, George Washington University School of Medicine and Health Sciences, Washington, DC, USA.">
<meta name="citation_author_institution" content="GW Cancer Center, George Washington University, Washington, DC, USA.">
<meta name="citation_publication_date" content="2025 Aug 8">
<meta name="citation_volume" content="11">
<meta name="citation_issue" content="32">
<meta name="citation_firstpage" content="eadv2453">
<meta name="citation_doi" content="10.1126/sciadv.adv2453">
<meta name="citation_pmid" content="40768568">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327453/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327453/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327453/pdf/sciadv.adv2453.pdf">
<meta name="description" content="R-loops are three-stranded DNA/RNA hybrids that are essential for various cellular pathways. However, when dysregulated, they lead to genomic instability and numerous human diseases. R-loops are tightly regulated, with RNase H1 acting as a key ...">
<meta name="og:title" content="AND-1 is a critical regulator of R-loop dynamics and a target to overcome endocrine resistance">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="R-loops are three-stranded DNA/RNA hybrids that are essential for various cellular pathways. However, when dysregulated, they lead to genomic instability and numerous human diseases. R-loops are tightly regulated, with RNase H1 acting as a key ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327453/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12327453">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1126/sciadv.adv2453"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/sciadv.adv2453.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12327453%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12327453/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12327453/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327453/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-sciadv.gif" alt="Science Advances logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Science Advances" title="Link to Science Advances" shape="default" href="http://advances.sciencemag.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Adv</button></div>. 2025 Aug 8;11(32):eadv2453. doi: <a href="https://doi.org/10.1126/sciadv.adv2453" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1126/sciadv.adv2453</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Adv%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Adv%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>AND-1 is a critical regulator of R-loop dynamics and a target to overcome endocrine resistance</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Zhuqing Li</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Zhuqing Li</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Biochemistry and Molecular Medicine, George Washington University School of Medicine and Health Sciences, Washington, DC, USA.</div>
<div class="p">
<sup><sup>2</sup></sup>GW Cancer Center, George Washington University, Washington, DC, USA.</div>
<div>Conceptualization, Data curation, Investigation, Project administration, Supervision, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhuqing Li</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>*,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yihan Peng</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yihan Peng</span></h3>
<div class="p">
<sup><sup>3</sup></sup>Department of Oncology, Georgetown Lombardi Comprehensive Cancer Center, Georgetown University Medical Center, Washington, DC, USA.</div>
<div>Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Visualization, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yihan Peng</span></a>
</div>
</div>
<sup>3,</sup><sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chiang%20HC%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Huai-Chin Chiang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Huai-Chin Chiang</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Biochemistry and Molecular Medicine, George Washington University School of Medicine and Health Sciences, Washington, DC, USA.</div>
<div>Investigation, Resources, Validation, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chiang%20HC%22%5BAuthor%5D" class="usa-link"><span class="name western">Huai-Chin Chiang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Yanjun Gao</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Yanjun Gao</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Biochemistry and Molecular Medicine, George Washington University School of Medicine and Health Sciences, Washington, DC, USA.</div>
<div class="p">
<sup><sup>2</sup></sup>GW Cancer Center, George Washington University, Washington, DC, USA.</div>
<div>Resources</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yanjun Gao</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chu%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Chen Chu</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Chen Chu</span></h3>
<div class="p">
<sup><sup>4</sup></sup>Department of Cancer Biology, Dana-Farber Cancer Institute, Boston, MA, USA.</div>
<div class="p">
<sup><sup>5</sup></sup>Department of Genetics, Blavatnik Institute, Harvard Medical School, Boston, MA, USA.</div>
<div>Data curation, Formal analysis, Investigation, Software, Validation, Visualization</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chu%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chen Chu</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Yi Zhang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Yi Zhang</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Biochemistry and Molecular Medicine, George Washington University School of Medicine and Health Sciences, Washington, DC, USA.</div>
<div class="p">
<sup><sup>2</sup></sup>GW Cancer Center, George Washington University, Washington, DC, USA.</div>
<div>Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yi Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mughal%20MJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Muhammad Jameel Mughal</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Muhammad Jameel Mughal</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Biochemistry and Molecular Medicine, George Washington University School of Medicine and Health Sciences, Washington, DC, USA.</div>
<div class="p">
<sup><sup>2</sup></sup>GW Cancer Center, George Washington University, Washington, DC, USA.</div>
<div>Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mughal%20MJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Muhammad Jameel Mughal</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Chengyu Liu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Chengyu Liu</span></h3>
<div class="p">
<sup><sup>6</sup></sup>The Transgenic Core Facility, National Heart, Lung, and Blood Institute, National Institutes of Health, Bethesda, MD, USA.</div>
<div>Methodology, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chengyu Liu</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Rong Li</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Rong Li</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Biochemistry and Molecular Medicine, George Washington University School of Medicine and Health Sciences, Washington, DC, USA.</div>
<div>Formal analysis, Funding acquisition, Investigation, Methodology, Resources, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rong Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pei%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Huadong Pei</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Huadong Pei</span></h3>
<div class="p">
<sup><sup>3</sup></sup>Department of Oncology, Georgetown Lombardi Comprehensive Cancer Center, Georgetown University Medical Center, Washington, DC, USA.</div>
<div>Conceptualization, Funding acquisition, Project administration, Resources, Supervision, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pei%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Huadong Pei</span></a>
</div>
</div>
<sup>3,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Wenge Zhu</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Wenge Zhu</span></h3>
<div class="p">
<sup><sup>1</sup></sup>Department of Biochemistry and Molecular Medicine, George Washington University School of Medicine and Health Sciences, Washington, DC, USA.</div>
<div class="p">
<sup><sup>2</sup></sup>GW Cancer Center, George Washington University, Washington, DC, USA.</div>
<div>Conceptualization, Funding acquisition, Methodology, Project administration, Resources, Supervision, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wenge Zhu</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup><sup>1</sup></sup>Department of Biochemistry and Molecular Medicine, George Washington University School of Medicine and Health Sciences, Washington, DC, USA.</div>
<div id="aff2">
<sup><sup>2</sup></sup>GW Cancer Center, George Washington University, Washington, DC, USA.</div>
<div id="aff3">
<sup><sup>3</sup></sup>Department of Oncology, Georgetown Lombardi Comprehensive Cancer Center, Georgetown University Medical Center, Washington, DC, USA.</div>
<div id="aff4">
<sup><sup>4</sup></sup>Department of Cancer Biology, Dana-Farber Cancer Institute, Boston, MA, USA.</div>
<div id="aff5">
<sup><sup>5</sup></sup>Department of Genetics, Blavatnik Institute, Harvard Medical School, Boston, MA, USA.</div>
<div id="aff6">
<sup><sup>6</sup></sup>The Transgenic Core Facility, National Heart, Lung, and Blood Institute, National Institutes of Health, Bethesda, MD, USA.</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">Corresponding author. Email: <span>zhuqing_li@gwu.edu</span> (Z.L.); <span>huadong.pei@georgetown.edu</span> (H.P.); <span>wz6812@gwu.edu</span> (W.Z.)</p>
</div>
<div class="fn" id="afn1">
<sup>†</sup><p class="display-inline">These authors contributed equally to this work.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Zhuqing Li</span></strong>: <span class="role">Conceptualization, Data curation, Investigation, Project administration, Supervision, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Yihan Peng</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Visualization, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Huai-Chin Chiang</span></strong>: <span class="role">Investigation, Resources, Validation, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Yanjun Gao</span></strong>: <span class="role">Resources</span>
</div>
<div>
<strong class="contrib"><span class="name western">Chen Chu</span></strong>: <span class="role">Data curation, Formal analysis, Investigation, Software, Validation, Visualization</span>
</div>
<div>
<strong class="contrib"><span class="name western">Yi Zhang</span></strong>: <span class="role">Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Muhammad Jameel Mughal</span></strong>: <span class="role">Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Chengyu Liu</span></strong>: <span class="role">Methodology, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Rong Li</span></strong>: <span class="role">Formal analysis, Funding acquisition, Investigation, Methodology, Resources, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Huadong Pei</span></strong>: <span class="role">Conceptualization, Funding acquisition, Project administration, Resources, Supervision, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Wenge Zhu</span></strong>: <span class="role">Conceptualization, Funding acquisition, Methodology, Project administration, Resources, Supervision, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Dec 12; Accepted 2025 Jul 9; Collection date 2025 Aug 8.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Copyright © 2025 The Authors, some rights reserved; exclusive licensee American Association for the Advancement of Science. No claim to original U.S. Government Works. Distributed under a Creative Commons Attribution NonCommercial License 4.0 (CC BY-NC).</div>
<p>This is an open-access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by-nc/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution-NonCommercial license</a>, which permits use, distribution, and reproduction in any medium, so long as the resultant use is <strong>not</strong> for commercial advantage and provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12327453  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40768568/" class="usa-link">40768568</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>R-loops are three-stranded DNA/RNA hybrids that are essential for various cellular pathways. However, when dysregulated, they lead to genomic instability and numerous human diseases. R-loops are tightly regulated, with RNase H1 acting as a key enzyme responsible for resolving DNA/RNA hybrids. Here, we identify the DNA-binding protein AND-1 as an essential factor in R-loop regulation through directly binding to R-loop structures, where it enhances the recruitment of RNase H1 and stimulates its endonuclease activity. We also provide in vivo evidence that R-loop accumulation occurs in the mammary gland tissue of AND-1–deficient mice. Furthermore, we demonstrate that inhibition of AND-1 decreases <em>ESR1</em> expression by disrupting R-loop regulation at the enhancer region of the <em>ESR1</em> gene in estrogen receptor–positive (ER<sup>+</sup>) breast cancer cells, thereby overcoming resistance to aromatase inhibitors. Collectively, our findings reveal a mechanism by which AND-1 modulates R-loop dynamics and present a promising therapeutic strategy to combat endocrine resistance in breast cancer.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p>AND-1 regulates R-loops, and its inhibition overcomes endocrine resistance by suppressing <em>ESR1</em> expression in ER<sup>+</sup> breast cancer.</p></section><section id="sec1"><h2 class="pmc_sec_title">INTRODUCTION</h2>
<p>R-loops are non–B DNA structures formed when an RNA strand invades double-stranded DNA (dsDNA) and displaces the nontemplate DNA strand. These structures have been detected across the genomes of bacteria, yeast, and higher eukaryotes throughout the cell cycle. Naturally occurring R-loops participate in various physiological processes, including prokaryotic origins of replication (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>, <a href="#R2" class="usa-link" aria-describedby="R2"><em>2</em></a>), mitochondrial DNA replication (<a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>), immunoglobulin class switching (<a href="#R4" class="usa-link" aria-describedby="R4"><em>4</em></a>), transcription (<a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>), and DNA repair (<a href="#R6" class="usa-link" aria-describedby="R6"><em>6</em></a>–<a href="#R9" class="usa-link" aria-describedby="R9"><em>9</em></a>). However, dysregulated R-loop formation or unresolved R-loops can lead to DNA double-strand breaks, chromosome translocation, gene mutations (<a href="#R10" class="usa-link" aria-describedby="R10"><em>10</em></a>), and interference with DNA replication and transcription, all of which are sources of genomic instability (<a href="#R11" class="usa-link" aria-describedby="R11"><em>11</em></a>, <a href="#R12" class="usa-link" aria-describedby="R12"><em>12</em></a>). Recent evidence suggests that aberrant R-loop homeostasis is implicated in the development of multiple human diseases, including neurological disorders, Aicardi-Goutières syndrome, nucleotide expansion diseases, various cancers, and even drug resistance (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>–<a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>). Therefore, understanding the molecular mechanisms governing R-loop regulation is crucial for identifying novel therapeutic opportunities.</p>
<p>A growing number of proteins have been implicated in the prevention or resolution of unscheduled R-loops. These include RNA biogenesis factors, ribonuclease (RNase) H enzymes, RNA:DNA (R:D) helicases, and specific DNA repair proteins such as FANCD2, BRCA1, and BRCA2 (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>–<a href="#R16" class="usa-link" aria-describedby="R16"><em>16</em></a>). Loss of these regulators often results in elevated R-loop levels and associated DNA damage in both yeast and human cells (<a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>–<a href="#R21" class="usa-link" aria-describedby="R21"><em>21</em></a>). Among them, RNase H1 is a conserved enzyme from bacteria to humans that recognizes R:D hybrids and resolves R-loops through degrading the RNA strand (<a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>). Due to its key role, exogenous expression of RNase H1 is widely used to validate the presence of R-loops at specific genomic loci. Cells that are deficient in RNase H1 exhibit impaired R-loop resolution, leading to increased genomic instability (<a href="#R23" class="usa-link" aria-describedby="R23"><em>23</em></a>). Despite its importance, the mechanisms by which RNase H1 specifically recognizes R-loops and how its endonuclease activity is regulated at these sites remain largely unknown.</p>
<p>AND-1/<em>WDHD1</em> (also known as Ctf4 in yeast) is an acidic nucleoplasmic DNA-binding protein that is essential for DNA replication and DNA damage repair (<a href="#R24" class="usa-link" aria-describedby="R24"><em>24</em></a>–<a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>). Previous studies in yeast have demonstrated that Ctf4 deletion leads to an accumulation of R:D hybrids recognized by S9.6 antibody (<a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>), suggesting a potential role of Ctf4 in R-loop resolution. However, the precise mechanism by which Ctf4 regulates R-loops remains unknown. Moreover, whether AND-1 plays a similar role in R-loop regulation in human cells has yet to be investigated.</p>
<p>Estrogen receptor–positive (ER<sup>+</sup>) breast cancer is a leading cause of cancer-related death in women (<a href="#R31" class="usa-link" aria-describedby="R31"><em>31</em></a>). ER<sup>+</sup> breast cancer cells express estrogen receptors that drive tumor proliferation upon binding to the hormone. Endocrine therapy (ET), including aromatase inhibitors (AIs) such as letrozole and selective estrogen receptor modulators (SERMs) like tamoxifen, is a commonly prescribed treatment. However, patients harboring <em>ESR1</em> mutations often develop resistance to standard ET and exhibit worse overall survival (<a href="#R31" class="usa-link" aria-describedby="R31"><em>31</em></a>, <a href="#R32" class="usa-link" aria-describedby="R32"><em>32</em></a>). These mutations, primarily located in the ligand-binding domain of <em>ESR1</em>, render the receptor constitutively active and are less sensitive to ET treatment (<a href="#R32" class="usa-link" aria-describedby="R32"><em>32</em></a>). The Y537S and D538G mutations are the most common and confer ET resistance through altered protein structure that leads to constitutive transcriptional activity and reduced affinity for ligands, including estrogen, AIs, and SERMs (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>). Thus, targeting mutant <em>ESR1</em> is a critical therapeutic strategy for overcoming endocrine resistance in breast cancer.</p>
<p>We and others have identified R-loops at a noncoding region upstream of the <em>ESR1</em> locus, where unresolved R-loops can suppress <em>ESR1</em> gene expression (<a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>, <a href="#R34" class="usa-link" aria-describedby="R34"><em>34</em></a>–<a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>). These findings provide a previously underappreciated concept to down-regulate <em>ESR1</em> expression by manipulating R-loop resolution, offering a potential strategy to overcome endocrine resistance caused by <em>ESR1</em> mutations in breast cancer. In this study, we demonstrated that AND-1 directly binds to R-loops and is critical for R-loop regulation. Mechanistically, AND-1 facilitates R-loop clearance by promoting the recruitment of RNase H1 to R-loops and enhances RNase H1 endonuclease activity in a manner dependent on its phosphorylation at the T826 site. Notably, the role of AND-1 in R-loop regulation is supported by in vivo evidence, showing increased R-loop accumulation in the mammary gland of AND-1–deficient mice. Furthermore, silencing AND-1 by small interfering RNA (siRNA) or inhibiting AND-1 using newly identified specific inhibitors (<em>E</em>)-5-(3,4-dichlorostyryl)benzo[<em>c</em>][1,2]oxaborol-1(3<em>H</em>)-ol (CH3) and bazedoxifene acetate (BZA) effectively suppressed <em>ESR1</em> expression and overcame AI resistance (AR) driven by <em>ESR1</em> mutations. Together, our study uncovers a mechanism by which AND-1 regulates R-loops via modulation of RNase H1 activity and provides preclinical evidence for a therapeutic strategy targeting <em>ESR1</em> expression through AND-1–mediated R-loop dynamics.</p></section><section id="sec2"><h2 class="pmc_sec_title">RESULTS</h2>
<section id="sec3"><h3 class="pmc_sec_title">AND-1 directly binds to R-loops and is critical for R-loop regulation</h3>
<p>To explore the molecular mechanism by which AND-1 regulates R-loops, we examined R-loop levels by immunofluorescence (IF) assay using S9.6 antibody in U2OS cells with AND-1 depletion by siRNA. Because S9.6 antibody recognizes not only R:D hybrids but also non–R-loop cellular RNAs (<a href="#R37" class="usa-link" aria-describedby="R37"><em>37</em></a>), we therefore pretreated cells with RNase A, T1, and III to remove non–R-loop RNA as described previously (<a href="#R38" class="usa-link" aria-describedby="R38"><em>38</em></a>–<a href="#R40" class="usa-link" aria-describedby="R40"><em>40</em></a>). Consistent with previous report (<a href="#R38" class="usa-link" aria-describedby="R38"><em>38</em></a>–<a href="#R40" class="usa-link" aria-describedby="R40"><em>40</em></a>), RNase A, T1, and III treatment significantly reduced the amount of cytoplasmic S9.6 IF signals (fig. S1A), confirming the effective removal of non–R-loop RNAs recognized by S9.6 antibody and validating the specificity of our IF assay. Using this IF assay, we found that knockdown of AND-1 by siRNA significantly increased the nuclear S9.6 fluorescence compared to cells treated with control siRNA (<a href="#F1" class="usa-link">Fig. 1, A to C</a>, and fig. S1B). Ectopic expression of AND-1 suppressed the increased nuclear S9.6 fluorescence caused by AND-1 depletion (<a href="#F1" class="usa-link">Fig. 1, A to C</a>, and fig. S1B), ruling out potential siRNA off-target effects. To further confirm the specificity of R-loop detection, we ectopically expressed V5-tagged RNase H1, which is known to efficiently resolve R-loops and is widely used for R-loop detection (<a href="#R20" class="usa-link" aria-describedby="R20"><em>20</em></a>, <a href="#R41" class="usa-link" aria-describedby="R41"><em>41</em></a>). As expected, exogenous expression of RNase H1 significantly reduced the S9.6 signal in cells across all three groups (<a href="#F1" class="usa-link">Fig. 1, A to C</a>), supporting the specificity of the S9.6 antibody in recognition of R-loops.</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">Fig. 1. AND-1 interacts with R-loops and its depletion leads to R:D hybrid accumulation.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12327453_sciadv.adv2453-f1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/32e9/12327453/63ee7b4e00ef/sciadv.adv2453-f1.jpg" loading="lazy" height="1018" width="746" alt="Fig. 1."></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong> to <strong>C</strong>) Images [(A) and (B)] and quantification (C) of S9.6 immunostaining in U2OS cells treated with control siRNA (Ctrl), AND-1 siRNA (siA), or siA with plasmid-mediated expression of WT AND-1 (siA + WT). Cells were also transfected with or without V5-tagged RNase H1 (V5-RH1). Scale bars, 10 μm. <em>n</em> = 50. (<strong>D</strong>) Quantification of S9.6 dot blot signals shown in fig. S1E. U2OS cells were treated with Ctrl, siA, or siA + WT as in (A), with or without 1 μM camptothecin (CPT) for 4 hours. <em>n</em> = 3. (<strong>E</strong>) Proximity ligation assay (PLA) to assess the association of AND-1 with FLAG-RNase H1<sup>D210N</sup>. U2OS cells expressing FLAG-RNase H1<sup>D210N</sup> were treated with or without 1 μM CPT for 4 hours. Anti–AND-1 and anti-FLAG antibodies were used. Scale bars, 10 μm. (<strong>F</strong>) Quantification of PLA signals from (E). <em>n</em> = 50. (<strong>G</strong>) S9.6 co-immunoprecipitation (co-IP) assay to show the interaction between R-loops and endogenous AND-1 and RNase H1 (RH1). Cells were treated with or without 1 μM CPT for 4 hours. (<strong>H</strong>) Electrophoretic mobility shift assay (EMSA) to show that AND-1 binds to different substrates including R-loop, R:D, dsDNA, and D-loop. (<strong>I</strong>) Schematic representation of AND-1 (FL) and its mutant (1-984). (<strong>J</strong>) EMSA to show that AND-1 (FL), but not AND-1 (1-984), binds to R-loop molecules. (<strong>K</strong> and <strong>L</strong>) PLA image (K) and quantification (L) in U2OS cells cotransfected with V5-tagged RNase H1<sup>D210N</sup> (V5-RH1) and FLAG tagged AND-1 (FL or 1-984). Anti-FLAG and anti-V5 antibodies were used. Scale bars, 10 μm. <em>n</em> = 50. (<strong>M</strong>) S9.6 co-IP assay to examine the interactions between R-loops and FLAG-tagged AND-1 (FL or 1-984) in U2OS cells. Data are shown as means ± SD. *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; ***<em>P</em> &lt; 0.001; ****<em>P</em> &lt; 0.0001; n.s., not significant, based on unpaired two-tailed Student’s <em>t</em> tests. IgG, immunoglobulin G.</p></figcaption></figure><p>To complement these findings, we performed dot blot analysis as described previously (fig. S1C) (<a href="#R39" class="usa-link" aria-describedby="R39"><em>39</em></a>, <a href="#R42" class="usa-link" aria-describedby="R42"><em>42</em></a>). Consistently, AND-1 knockdown increased R-loop levels, as indicated by the intensity of dot blots recognized by S9.6 antibody. This signal was eliminated upon RNase H treatment, confirming its R-loop specificity (<a href="#F1" class="usa-link">Fig. 1D</a> and fig. S1, D and E). Camptothecin (CPT) is a topoisomerase I inhibitor that can induce R-loop (<a href="#R43" class="usa-link" aria-describedby="R43"><em>43</em></a>). Consistent with this result, CPT treatment increased R-loop levels across all groups (<a href="#F1" class="usa-link">Fig. 1D</a> and fig. S1, D and E), supporting the role of AND-1 in R-loop regulation.</p>
<p>The above results indicate that AND-1 is involved in R-loop regulation. We therefore hypothesized that AND-1 might physically associate with R-loops in cells. To test this, we examined the colocalization of AND-1 with R-loops using proximity ligation assay (PLA), an assay to detect protein-protein interactions in situ (at distances of &lt;40 nm) (<a href="#R44" class="usa-link" aria-describedby="R44"><em>44</em></a>). RNase H1<sup>D210N</sup> is a catalytically inactive RNase H1 mutant that remains persistently present at R-loops and was, therefore, used as a marker for R-loop localization (<a href="#R45" class="usa-link" aria-describedby="R45"><em>45</em></a>, <a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>). We expressed FLAG-RNase H1<sup>D210N</sup> in U2OS cells and performed PLA using anti–AND-1 and anti-FLAG antibodies to detect their colocalization. No PLA signal was observed in negative controls lacking primary antibodies or stained with either one of the two antibodies (<a href="#F1" class="usa-link">Fig. 1, E and F</a>). However, PLA signal (red puncta) was detected in cells costained with both anti–AND-1 and anti-FLAG antibodies, and the signal was increased following CPT treatment, indicating an interaction between AND-1 and R-loops (<a href="#F1" class="usa-link">Fig. 1, E and F</a>). To confirm this interaction, we conducted a co-immunoprecipitation (co-IP) assay using S9.6 antibody. As shown in <a href="#F1" class="usa-link">Fig. 1G</a>, both RNase H1 and AND-1 were detected in S9.6 immunoprecipitates (IPs), and the association of both proteins with R-loops was enhanced upon CPT treatment, supporting the involvement of AND-1 in R-loop regulation.</p>
<p>To corroborate the PLA and co-IP results, we further examined the interaction between AND-1 and R-loops using chromatin immunoprecipitation (ChIP). Specifically, we analyzed the association of AND-1 with the termination pause sites of β-<em>ACTIN</em> and <em>GEMIN7</em> genes, two genomic loci known to form R-loops (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>). FLAG-RNase H1<sup>D210N</sup> was used as a positive control in these experiments. Consistent with the previous observation (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>), FLAG-RNase H1<sup>D210N</sup> was enriched at the termination pause sites of both β-<em>ACTIN</em> and <em>GEMIN7</em> genes compared with nearby control loci without R-loop formation, and these associations were reduced in cells expressing V5-tagged wild-type (WT) RNase H1 (fig. S1, F and H). Intriguingly, endogenous AND-1 was also enriched at the same termination pause sites (fig. S1, G and I). These findings strongly support the colocalization of RNase H1 and AND-1 at two R-loop–forming loci.</p>
<p>To further examine whether AND-1 directly interacts with R-loops in vitro, we generated different biotin-labeled nucleic acid substrates, including R-loop, D-loop, R:D hybrid, and dsDNA, as previously described (fig. S1J) (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>). We then performed an electrophoretic mobility shift assay (EMSA) to assess the binding of AND-1 to these substrates (<a href="#R47" class="usa-link" aria-describedby="R47"><em>47</em></a>). As shown in <a href="#F1" class="usa-link">Fig. 1H</a>, AND-1 bound to all tested substrate, but exhibited notably higher affinity for R-loop and D-loop.</p>
<p>AND-1 contains an N-terminal WD40 domain, a middle SepB domain, and a C-terminal high-mobility group (HMG) domain that is responsible for interacting with DNA directly (<a href="#R48" class="usa-link" aria-describedby="R48"><em>48</em></a>, <a href="#R49" class="usa-link" aria-describedby="R49"><em>49</em></a>). To dissect the requirement of the HMG domain for R-loop biding, we purified recombinant full-length AND-1 [AND-1 (FL)] and a truncated mutant, AND-1 (1-984), lacking the HMG domain (<a href="#F1" class="usa-link">Fig. 1I</a> and fig. S1K). EMSA revealed that the R-loop substrate was upshifted upon incubation with AND-1 (FL), but not with AND-1 (1-984) (<a href="#F1" class="usa-link">Fig. 1J</a>), indicating that the HMG domain is required for the interaction. Consistently, both PLA (<a href="#F1" class="usa-link">Fig. 1, K and L</a>, and fig. S1L) and S9.6 co-IP assays (<a href="#F1" class="usa-link">Fig. 1M</a>) confirmed the loss of interaction between AND-1 (1-984) and R-loops as compared to AND-1 (FL). Collectively, these data demonstrate that AND-1 directly associates with R-loops, and its HMG domain is essential for this interaction.</p></section><section id="sec4"><h3 class="pmc_sec_title">AND-1 interacts with RNase H1 and is required for its association with R-loops</h3>
<p>Given that AND-1 directly binds to R-loops, we assumed that it may functionally and physically interact with RNase H1. To test this, we performed co-IP assays to examine the interaction between AND-1 and RNase H1. As shown in <a href="#F2" class="usa-link">Fig. 2A</a>, endogenous RNase H1 was detected in AND-1 IPs, and this interaction was enhanced upon CPT treatment. Conversely, the interaction was suppressed by actinomycin D, an RNA polymerase inhibitor known to reduce R-loop formation (<a href="#R50" class="usa-link" aria-describedby="R50"><em>50</em></a>). In addition, we detected the interaction between AND-1 and FLAG-tagged RNase H1<sup>D210N</sup> (fig. S2A), indicating a physical association between AND-1 and RNase H1 and that the catalytic inactivation of RNase H1 does not impair its interaction with AND-1. To test whether the AND-1–R-loop interaction is limited to S-phase, we conducted a short EdU pulse (10 μM, 20 min) in U2OS cells, followed by PLA staining using anti–AND-1 and anti-S9.6 antibodies, along with EdU click. As shown in fig. S2 (B and C), PLA signals were detected in both EdU-positive and EdU-negative cells, suggesting that the interaction of AND-1 with R-loops is not restricted to the S-phase.</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Fig. 2. AND-1 interacts with RNase H1 and promotes its recruitment to R-loops.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12327453_sciadv.adv2453-f2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/32e9/12327453/0939d788546f/sciadv.adv2453-f2.jpg" loading="lazy" height="1018" width="785" alt="Fig. 2."></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Co-immunoprecipitation (co-IP) assay to show the interaction between endogenous AND-1 and RNase H1 (RH1) in U2OS cells. Cell lysates were immunoprecipitated with anti–AND-1 or control immunoglobulin G (IgG) antibodies, followed by immunoblotting for indicated proteins. Cells were treated with 1 μM CPT for 4 hours to induce R-loop formation or treated with RNA polymerase inhibitor (5 μg/ml; actinomycin D) for 6 hours to attenuate R-loop generation. Dimethyl sulfoxide (DMSO) treatment was used as control. (<strong>B</strong>) Schematic representation of AND-1 (FL) and its truncated mutants. (<strong>C</strong>) Co-IP assay using lysates of HEK293T cells expressing indicated FLAG-tagged AND-1 or its mutants. Lysates were immunoprecipitated with anti-FLAG antibody, followed by immunoblotting for indicated proteins. (<strong>D</strong> and <strong>E</strong>) In vitro co-IP assay to show the interaction between purified His–AND-1 and purified FLAG-RNase H1<sup>WT</sup> (FLAG-RH1<sup>WT</sup>) (D) or FLAG-RNase H1<sup>D210N</sup> (FLAG-RH1<sup>D210N</sup>) (E). (<strong>F</strong>) In vitro co-IP assay to show the interaction between purified FLAG-RH1<sup>WT</sup> proteins and purified His–AND-1 (FL) or its mutants. (<strong>G</strong>) PLA to show the colocalization of RH1 with R-loops in U2OS cells costained with anti-RH1 and S9.6 antibodies. Red foci indicate PLA signals. Scale bars, 10 μm. Cells were treated with control siRNA (Ctrl), AND-1 siRNA (siA), or siA followed by transfection with WT AND-1 plasmid. (<strong>H</strong>) S9.6 co-IP assay to detect the interaction between endogenous RH1 and R-loops in U2OS cells. Cells were treated with Ctrl or siA, followed by the treatment with 1 μM CPT for 4 hours or actinomycin D (5 μg/ml) for 6 hours, as in (A).</p></figcaption></figure><p>We next mapped the domain(s) of AND-1 responsible for its interaction with RNase H1 by generating various AND-1 mutants (<a href="#F2" class="usa-link">Fig. 2B</a>). FLAG-tagged AND-1 (FL) and its truncated variants were expressed in human embryonic kidney (HEK) 293T cells. As shown in <a href="#F2" class="usa-link">Fig. 2C</a>, RNase H1 was detected from FLAG IPs of AND-1 (FL) and AND-1 mutants (336-984), (1-984), and (336-1129), but not (1-336) or (984-1129), indicating that the SepB domain, located between residues 336 and 984, is critical to mediate the interaction between AND-1 and RNase H1. To examine whether this interaction is direct, we conducted an in vitro co-IP assay by using purified recombinant His-tagged AND-1 (fig. S1K) and FLAG-tagged RNase H1 proteins (fig. S2D). As shown in <a href="#F2" class="usa-link">Fig. 2 (D and E)</a>, both FLAG-RNase H1<sup>WT</sup> and FLAG-RNase H1<sup>D210N</sup> were detected from His–AND-1 IPs, confirming a direct interaction between AND-1 and RNase H1. Consistent with the cell-based data, in vitro co-IP assay using purified AND-1 and truncated AND-1 mutants (fig. S2E) also demonstrated that the SepB domain is required for AND-1’s binding to RNase H1 (<a href="#F2" class="usa-link">Fig. 2F</a>).</p>
<p>Given that AND-1 directly interacts with both R-loops and RNase H1, we next explored whether AND-1 regulates the association of RNase H1 with R-loops. Using PLA, we observed that depletion of AND-1 by siRNA significantly reduced the RNase H1:S9.6 PLA fluorescence signals, which was rescued by ectopic expression of AND-1 (<a href="#F2" class="usa-link">Fig. 2G</a> and fig. S2, F to H). Consistently, the amount of RNase H1 detected from S9.6 IPs was decreased in cells treated with siAND-1 compared to control siRNA treatment (<a href="#F2" class="usa-link">Fig. 2H</a>). To further determine whether AND-1 can promote the association of RNase H1 with R-loops in vitro, we performed streptavidin-bead pulldown assay (<a href="#R51" class="usa-link" aria-describedby="R51"><em>51</em></a>). Specifically, biotin-labeled R-loop molecules (fig. S1J) were incubated with purified recombinant RNase H1 in the presence of bovine serum albumin (BSA) or purified AND-1 protein, followed by pulldown using streptavidin-conjugated beads. As shown in fig. S2I, RNase H1 exhibited a weak association with biotin-labeled R-loops in the presence or absence of BSA; however, the interaction was markedly enhanced in the presence of AND-1. These results demonstrate that AND-1 promotes the association of RNase H1 with R-loops in vitro. Together with the cellular data, these findings strongly suggest that AND-1 facilitates the recruitment of RNase H1 to R-loops.</p></section><section id="sec5"><h3 class="pmc_sec_title">AND-1 stimulates the endonuclease activity of RNase H1 on R-loops</h3>
<p>We next explored whether AND-1 regulates RNase H1 endonuclease activity using in vitro endonuclease assays. Given that AND-1 directly binds to both R-loops and RNase H1 and enhances their association, we hypothesized that AND-1 may regulate RNase H1 endonuclease activity. To test this possibility, we purified recombinant RNase H1 and AND-1 proteins (fig. S2D) and conducted both fluorescence-based and gel-based RNase H1 activity assays. The principle of the fluorescence-based assay is that RNA cleavage by RNase H1 releases the fluorescein molecules, which can be measured with excitation at 488 nm and emission at 535 nm (<a href="#F3" class="usa-link">Fig. 3A</a>) (<a href="#R52" class="usa-link" aria-describedby="R52"><em>52</em></a>, <a href="#R53" class="usa-link" aria-describedby="R53"><em>53</em></a>). We synthesized a dual-labeled 25–base pair (bp) R:D hybrid using oligos of 5′-FAM–labeled RNA oligo and 3′-ROX–labeled DNA oligo (fig. S3A), following previously described protocols (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>). As shown in <a href="#F3" class="usa-link">Fig. 3B</a>, when the inactive R:D substrates were incubated with WT RNase H1, but not the catalytically inactive mutant RNase H1<sup>D210N</sup>, we could detect a robust increase in fluorescence in both concentration- and time-dependent manner, indicating that RNase H1 efficiently cleaves the RNA within R:D hybrids efficiently in vitro.</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Fig. 3. AND-1 stimulates the endonuclease activity of RNase H1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12327453_sciadv.adv2453-f3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/32e9/12327453/cff6a412013d/sciadv.adv2453-f3.jpg" loading="lazy" height="1017" width="726" alt="Fig. 3."></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Schematic representation of the fluorescence-based RNase H1 (RH1) activity assay. (<strong>B</strong>) Relative fluorescence intensity of R:D substrates incubated with indicated amounts of WT RH1 or RH1<sup>D210N</sup>. (<strong>C</strong>) Relative fluorescence intensity of R:D substrates incubated with 2 nM RH1 alone or premixed with 40 nM BSA or AND-1. (<strong>D</strong>) Top: Schematic representation of AND-1 (FL) and its truncated mutants. Bottom: Relative fluorescence intensity of R:D substrates incubated with 2 nM RH1 alone or premixed with 40 nM AND-1 (FL) or two mutants (1-984 and S1). (<strong>E</strong>) Gel-based assay to detect RH1 activity. FAM-labeled R:D substrates were incubated with RH1<sup>D210N</sup> or with increasing concentrations of RH1 for 20 min. The cleavage of substrates was analyzed on a 12% denaturing PAGE. Bottom: Quantification of cleaved substrates. (<strong>F</strong>) FAM-labeled R:D substrates were incubated with RH1 and either BSA or increasing concentrations of AND-1 for 20 min, followed by gel analysis as in (E). Bottom: Quantification of cleaved products. (<strong>G</strong>) Time-course analysis of RH1 activity. FAM-labeled R:D substrates were incubated with RH1 in the presence or absence of AND-1. The reactions were analyzed at indicated time points. Bottom: Quantification of cleaved products. (<strong>H</strong>) FAM-labeled R:D substrates were incubated with RH1 alone or with AND-1 (FL, 1-984, or S1) for the indicated time points. Bottom: Quantification of cleaved products shown in top panel. (<strong>I</strong>) FAM-labeled R-loop substrates (1 μM) were incubated with RH1 alone or with AND-1 (FL, 1-984, or S1) for the indicated time points. Right: The cleaved products were quantified. <em>n</em> = 3. Data are shown as means ± SD. *<em>P</em> &lt; 0.05; ***<em>P</em> &lt; 0.001; ****<em>P</em> &lt; 0.0001; n.s., not significant, compared to RH1-alone group, based on unpaired two-tailed Student’s <em>t</em> tests.</p></figcaption></figure><p>We next used this assay to test whether AND-1 influences RNase H1 endonuclease activity. As shown in <a href="#F3" class="usa-link">Fig. 3C</a>, AND-1 alone did not alter the fluorescence signal when incubated with R:D hybrids, indicating that AND-1 lacks endonuclease activity to cleave RNA. Notably, the addition of AND-1, but not BSA, to R:D hybrids in the presence of RNase H1 significantly increased the fluorescence signal compared to reactions with RNase H1 alone. These data strongly suggest that AND-1 stimulates RNase H1 endonuclease activity toward RNA cleavage in R:D hybrids.</p>
<p>To further delineate the domains of AND-1 required for this reaction, we repeated the assay using AND-1 mutants. Neither the SepB domain-deleted mutant (S1) nor the HMG domain-deleted mutant (1-984) enhanced the fluorescence signal generated by RNase H1 (<a href="#F3" class="usa-link">Fig. 3D</a>), suggesting that both HMG and the SepB domains of AND-1 are indispensable for AND-1–mediated stimulation of RNase H1 activity. The purified RNase H1 and AND-1 proteins used in these assays are shown in fig. S2 (D and E).</p>
<p>To visualize cleaved products by RNase H1, we synthesized a 25-bp R:D hybrid containing 5′-FAM–labeled RNA (fig. S3B) and performed gel-based RNase H1 activity assay (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>, <a href="#R54" class="usa-link" aria-describedby="R54"><em>54</em></a>, <a href="#R55" class="usa-link" aria-describedby="R55"><em>55</em></a>). Consistent with fluorescence-based assay results, cleaved RNA products could only be detected when the R:D substrates were incubated with WT RNase H1, but not catalytically inactive RNaseH1<sup>D210N</sup>, and the cleavage occurred in a concentration-dependent manner (<a href="#F3" class="usa-link">Fig. 3E</a>), demonstrating that RNase H1-mediated cleavage of the R:D hybrid can be effectively visualized in vitro by a gel-based assay. Using this gel-based assay, we found that AND-1 alone at 40 nM did not display any RNase activity toward the R:D hybrid. However, in the presence of RNase H1, AND-1 stimulated the cleavage of the R:D substrates in a concentration-dependent manner (<a href="#F3" class="usa-link">Fig. 3F</a>). Additionally, AND-1 accelerated the cleavage of R:D substrates by RNase H1 in a time-dependent manner (<a href="#F3" class="usa-link">Fig. 3G</a>). Compared to AND-1 (FL), AND-1 mutants (S1) or (1-984) exhibited markedly reduced capability to stimulate RNase H1 activity (<a href="#F3" class="usa-link">Fig. 3H</a>).</p>
<p>To test whether AND-1 stimulates RNase H1 endonuclease activity on R-loops, we generated an “R-loop–like” substrate using a protocol as described by Nguyen <em>et al.</em> (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>). This structure contains 30-bp dsDNA arms, a central 31-nucleotide ssDNA bubble, and a 25-bp R:D hybrid within the bubble. The RNA strand was 5′-FAM–labeled (fig. S3C). As shown in fig. S3D, RNase H1 was able to cleave the RNA in the R-loop substrates in a concentration-dependent manner. Although AND-1 alone showed no detectable activity, it significantly stimulated the activity of RNase H1 on the R-loop substrates in both concentration- and time-dependent manner (fig. S3, E and F). Moreover, RNase H1 cleaved the R-loop substrates more rapidly in the presence of AND-1 (FL), but not the S1 or 1-984 mutants (<a href="#F3" class="usa-link">Fig. 3I</a>). These in vitro data strongly demonstrate that AND-1 can stimulate the endonuclease activity of RNase H1 on R-loops and that both SepB and HMG domains are required.</p></section><section id="sec6"><h3 class="pmc_sec_title">Phosphorylation of AND-1 at T826 is important for its function in R-loop regulation</h3>
<p>Our previous studies demonstrated that AND-1 is phosphorylated by ATR at threonine-826 (T826), a modification crucial for its role in DNA damage response (<a href="#R28" class="usa-link" aria-describedby="R28"><em>28</em></a>, <a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>, <a href="#R49" class="usa-link" aria-describedby="R49"><em>49</em></a>). We therefore examined whether phosphorylation at T826 regulates AND-1’s function in R-loop dynamics. Using PLA with a phospho–AND-1 (T826) antibody (p–AND-1) and V5 antibody in U2OS cells expressing V5-RNase H1<sup>D210N</sup>, we detected distinct PLA fluorescence signals (fig. S4, A and B), indicating that phosphorylated AND-1 localizes to R-loops. Treatment with CPT not only induced phosphorylation of AND-1 at T826 (fig. S4C) but also enhanced the PLA signals (fig. S4, A and B), further supporting a role for p–AND-1 in R-loop regulation.</p>
<p>To further evaluate the functional impact of T826 phosphorylation, we coexpressed V5-RNase H1<sup>D210N</sup> with FLAG-tagged AND-1 (WT) or the phosphorylation-deficient T826A mutant in U2OS cells and performed PLA using anti-FLAG and anti-V5 antibodies. Both WT and T826A mutant AND-1 localized to the nuclei (fig. S4, D and E); however, PLA signals were only observed in cells expressing WT AND-1, but not the T826A mutant (<a href="#F4" class="usa-link">Fig. 4, A and B</a>), indicating that phosphorylation at T826 facilitates the interaction between AND-1 and R-loops. We next examined the effect of T826 phosphorylation on the direct association between AND-1 and R-loops in vitro. To achieve this objective, we purified recombinant His-tag WT AND-1 and T826A or T826E mutant AND-1 proteins and confirmed the absence of phosphorylation in the T826A mutant. The phosphomimic T826E mutant cannot be recognized by phosphor-T826 antibody (fig. S4, F and G). EMSA results revealed that WT and T826E AND-1 proteins, but not T826A mutant, were able to upshift R-loop molecules (fig. S4H). These findings indicate that T826 phosphorylation is essential for AND-1’s association with R-loop.</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Fig. 4. Phosphorylation of AND-1 at T826 is crucial for its role in R-loop regulation.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12327453_sciadv.adv2453-f4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/32e9/12327453/7b3aea25b5e1/sciadv.adv2453-f4.jpg" loading="lazy" height="1018" width="734" alt="Fig. 4."></a></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong> and <strong>B</strong>) PLA (A) and quantification (B) of U2OS cells cotransfected with V5-RNase H1<sup>D210N</sup> and FLAG–AND-1 (WT) or mutant (T826A). Anti-FLAG and anti-V5 antibodies were used. Scale bars, 10 μm. <em>n</em> = 50. Data are shown as means ± SD. ****<em>P</em> &lt; 0.0001, based on unpaired two-tailed Student’s <em>t</em> tests. (<strong>C</strong>) Co-IP assay in AND-1 KO U2OS cells transfected with FLAG-empty vector (Ctrl) and FLAG- AND-1 (WT or T826A). Cells were treated with 1 μM CPT for 4 hours or with ATR inhibitor (VE-821, 2.5 μM for 12 hours) and CPT. Cell lysates were immunoprecipitated with anti-FLAG antibody. (<strong>D</strong>) S9.6 co-IP to show the interaction between RH1 and R-loops under the treatment as (C). (<strong>E</strong>) Fluorescence-based RNase H1 (RH1) activity assay. FAM-labeled R:D substrates (1 μM) were incubated with 2 nM RNase H1 alone or in the presence of 40 nM AND-1 (RH1 + WT), phosphatase-treated AND-1 (RH1 + WT-P), or T826A AND-1 (RH1 + T826A). <em>n</em> = 3. Data are shown as means ± SD. ****<em>P</em> &lt; 0.0001; n.s., not significant, when compared with the RH1 group, based on unpaired two-tailed Student’s <em>t</em> tests. (<strong>F</strong> and <strong>G</strong>) Gel-based RH1 activity assay (F) and quantification of cleaved products (G). FAM-labeled R-loop substrates (1 μM) were incubated with RH1 in the presence or absence of 40 nM WT AND-1, WT-P, or T826A. <em>n</em> = 3. Data are shown as means ± SD. ****<em>P</em> &lt; 0.0001; n.s., not significant, based on unpaired two-tailed Student’s <em>t</em> tests. (<strong>H</strong> and <strong>I</strong>) S9.6 Immunostaining (H) and quantification (I) in U2OS cells treated with control siRNA (Ctrl), AND-1 siRNA (siA), or siA with plasmid expressing WT or T826A AND-1. Scale bars, 5 μm. <em>n</em> = 50. Data are shown as means ± SD. ***<em>P</em> &lt; 0.001; ****<em>P</em> &lt; 0.0001; n.s., not significant, based on unpaired two-tailed Student’s <em>t</em> tests.</p></figcaption></figure><p>We next explored how phosphorylation of AND-1 at T826 regulates its role in R-loops. To this end, we expressed FLAG-tagged WT AND-1 (WT) and T826A in AND-1 knockout (KO) U2OS cells (AND-1 KO), followed by treatment with CPT or cotreatment with CPT and the ATR inhibitor VE-821. Co-IP using anti-FLAG antibody (<a href="#F4" class="usa-link">Fig. 4C</a>) showed that CPT treatment enhanced the association between AND-1 and RNase H1, whereas cotreatment with VE-821, which blocked AND-1 phosphorylation at T826, diminished this CPT-mediated enhanced interaction. Additionally, the T826A mutant showed a reduced association with RNase H1 compared to WT AND-1. These data suggest that AND-1 phosphorylation at T826A is critical for its interaction with RNase H1. We therefore hypothesized that this phosphorylation may also affect the recruitment of RNase H1 to R-loops. S9.6 co-IP results (<a href="#F4" class="usa-link">Fig. 4D</a>) showed that both AND-1 and RNase H1 were detected from S9.6 IPs and CPT treatment enhanced their association, which was abrogated when treated with VE-821. Furthermore, AND-1 (T826A) was not detected in S9.6 IPs, and the amount of RNase H1 bound to R-loops was reduced in cells with AND-1 (T826A) expression. These findings indicate that T826 phosphorylation is required for AND-1 to facilitate the recruitment of RNase H1 to R-loops.</p>
<p>Given that AND-1 phosphorylation at T826 regulates the interaction between RNase H1 and R-loops, we assumed that this modification may also influence RNase H1 endonuclease activity. To test this hypothesis, we used purified WT and T826A AND-1 proteins (fig. S4, F and G) to conduct in vitro fluorescence-based and gel-based RNase H1 activity assays. As shown in <a href="#F4" class="usa-link">Fig. 4E</a>, the addition of purified WT AND-1, but not T826A AND-1, significantly increased the fluorescence signals generated by RNase H1 when incubated with R:D hybrids, indicating enhanced endonuclease activity. WT AND-1 pretreated with phosphatase (WT-P) failed to enhance fluorescence signals, confirming the requirement of phosphorylation for this effect.</p>
<p>Consistently, gel-based cleavage assays showed that RNaseH1 cleaved both R:D hybrids (fig. S4, I and J) and R-loops substrates (<a href="#F4" class="usa-link">Fig. 4, F and G</a>) more efficiently in the presence of WT AND-1, but not phosphatase-pretreated WT AND-1 or T826A mutant. These data collectively indicate that phosphorylation of AND-1 at T826 is critical for its ability to stimulate RNase H1 endonuclease activity.</p>
<p>We next examined whether phosphorylation of AND-1 at T826 is required for R-loop accumulation in cells. To this end, we exogenously expressed FLAG-tagged WT or T826A mutant AND-1 in U2OS cells with endogenous AND-1 deleted by siRNA (fig. S4K) and assessed R-loop levels using both S9.6 dot blot and S9.6 IF assays. Compared to the control treated cells, AND-1 deletion by siRNA increased R-loops levels as evidenced by elevated signals of R-loop in the dot blot assay (fig. S4, L and M) and increased fluorescence intensity in S9.6 IF staining (<a href="#F4" class="usa-link">Fig. 4, H and I</a>). The increased R-loop level by AND-1 depletion was significantly suppressed by exogenous expression of WT AND-1, but not the T826A mutant (<a href="#F4" class="usa-link">Fig. 4, H and I</a>; and fig. S4, L and M). Thus, our results demonstrate that phosphorylation of AND-1 at T826 is crucial for R-loop regulation.</p></section><section id="sec7"><h3 class="pmc_sec_title">AND-1 is a critical factor for maintaining R-loop levels in vivo</h3>
<p>We have provided strong in vitro evidence demonstrating a critical role of AND-1 in R-loop regulation. We next investigated whether AND-1 regulates R-loops in vivo by using AND-1–deficient mouse models. To this end, we generated two AND-1–deficient mouse models with C57BL/6J background: <em>Wdhd1</em> conditional KO (<em>Wdhd1</em><sup><em>loxP</em>/<em>loxP</em></sup>) mice (<a href="#F5" class="usa-link">Fig. 5A</a>) and <em>Wdhd1</em> T819A knock-in mice (<em>Wdhd1<sup>T819A</sup></em>) (<a href="#F5" class="usa-link">Fig. 5B</a>) (human T826 corresponds to T819 in mice). Specifically, to generate <em>Wdhd1<sup>loxP/loxP</sup></em> mice, the left <em>loxP</em> and the right <em>loxP</em> were inserted upstream and downstream of exon 8 of <em>Wdhd1</em> using CRISPR-Cas9 gene editing. To generate <em>Wdhd1 <sup>T819A</sup></em> mice, the <em>Wdhd1</em> T819 codon was mutated from ACC (T) to GCA (A) using CRISPR-Cas9 technology. A silent mutation from TTG (L) to CTT (L) was introduced into the donor to block a backup gRNA cleavage (fig. S5C). Both mouse models were confirmed by DNA sequencing as shown in fig. S5 (A to C)</p>
<figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">Fig. 5. AND-1 is required for R-loop regulation in AND-1–deficient mice.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12327453_sciadv.adv2453-f5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/32e9/12327453/12ad7a97e358/sciadv.adv2453-f5.jpg" loading="lazy" height="1018" width="778" alt="Fig. 5."></a></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Schematic illustration of the CRISPR-Cas9–mediated generation of the <em>Wdhd1</em>-floxed allele. Guide RNAs (gRNAs) were designed to target sites upstream and downstream of the eighth coding exon of <em>Wdhd1</em>, guiding Cas9 to create double-strand breaks (green vertical arrows). These breaks are then mended by homology-directed repair (HDR) using a donor template, which contains the two <em>loxP</em> sites and homology arms. Two pairs of screening primers as indicated are designed and used to screen the founders. (<strong>B</strong>) Schematic illustration of CRISPR-Cas9–mediated generation of the <em>Wdhd1</em> T819A allele within the exon 18, along with the locations of PCR primers (F3a and R3a) used for screening. <em>Wdhd1</em> T819 was mutated from ACC (T) to GCA (A). (<strong>C</strong> and <strong>D</strong>) Immunostaining (C) and quantification (D) of S9.6 nuclear fluorescence signal in indicated MEF cells pretreated with or without RH. All cells were pretreated with a combination of RNases A, T1, and III. Scale bars, 10 μm. <em>n</em> = 50. Data are shown as means ± SD. ****<em>P</em> &lt; 0.0001; n.s., not significant, based on unpaired two-tailed Student’s <em>t</em> tests. (<strong>E</strong> and <strong>F</strong>) S9.6 dot blot (E) and quantification (F) using genomic DNA extracted from indicated MEF cells treated with or without 1 μM CPT for 4 hours before harvest. <em>n</em> = 3. Data are shown as means ± SD. *<em>P</em> &lt; 0.05; ***<em>P</em> &lt; 0.001, based on unpaired two-tailed Student’s <em>t</em> tests. (<strong>G</strong> and <strong>H</strong>) R-loop IF images (G) and quantification of the relative R-loop intensity (H) in mouse mammary gland from 17- to 18-week-old virgin <em>Wdhd1 <sup>WT</sup></em> and <em>Wdhd1<sup>T819A</sup></em> mice. Scale bars, 50 μm and 10 μm (inset). Five animals were used in each group. Data are shown as means <kbd>±</kbd> SD. **<em>P</em> &lt; 0.01; ***<em>P</em> &lt; 0.001, based on unpaired two-tailed Student’s <em>t</em> tests.</p></figcaption></figure><p>We isolated mouse embryonic fibroblasts (MEFs) from WT (<em>Wdhd1 <sup>WT</sup></em>), <em>Wdhd1<sup>loxP/loxP</sup></em>, and <em>Wdhd1<sup>T819A</sup></em> embryonic day 13.5 (E13.5) embryos, using previously described protocols (fig. S5D) (<a href="#R56" class="usa-link" aria-describedby="R56"><em>56</em></a>). We then obtained <em>Wdhd1</em><sup>−/−</sup> MEF cells by infecting <em>Wdhd1<sup>loxP/loxP</sup></em> MEFs with adenovirus expressing Cre (Ad-CMV-Cre). As shown in fig. S5E, AND-1 protein expression was significantly decreased in <em>Wdhd1</em><sup>−/−</sup> MEF cells, and p–AND-1 was only detected in MEF <em>Wdhd1<sup>WT</sup></em> cells.</p>
<p>Using IF staining with the S9.6 antibody, we found that the R-loop level was increased in AND-1–deficient MEFs (<em>Wdhd1<sup>−/−</sup></em> and <em>Wdhd1<sup>T819A</sup></em>) as compared to WT MEFs (<em>Wdhd1 <sup>WT</sup></em>) (<a href="#F5" class="usa-link">Fig. 5, C and D</a>, left). Pretreatment of MEF cells with RNase H abolished the IF signal (<a href="#F5" class="usa-link">Fig. 5, C and D</a>, right), which confirms the specificity of the S9.6 antibody in the detection of the R-loops in MEFs. In PLA assay using anti–AND-1 and anti-S9.6 antibodies, PLA signals were observed only in <em>Wdhd1 <sup>WT</sup></em> MEFs and RNase H pretreatment significantly reduced PLA signals (fig. S5, F and G). These results suggest the absence of a role for AND-1 in R-loop regulation in <em>Wdhd1<sup>−/−</sup></em> and <em>Wdhd1<sup>T819A</sup></em> MEFs. Consistently, S9.6 dot blot results showed the increased level of R-loops in AND-1–deficient MEFs, with or without CPT treatment (<a href="#F5" class="usa-link">Fig. 5, E and F</a>).</p>
<p>Previous reports showed R-loop accumulation in BRCA1-deficient mouse mammary epithelium and in breast epithelium of human germline BRCA1 mutation carriers (<a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>, <a href="#R35" class="usa-link" aria-describedby="R35"><em>35</em></a>). To investigate R-loop formation in our model, we assessed R-loop levels in the mouse mammary epithelium of both <em>Wdhd1 <sup>WT</sup></em> and <em>Wdhd1<sup>T819A</sup></em> using IF staining (<a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>). To induce R-loop accumulation, mice of each genotype were treated with aphidicolin (APH) once per week for 10 weeks (fig. S5D) (<a href="#R57" class="usa-link" aria-describedby="R57"><em>57</em></a>). We previously reported that R-loop levels are increased in mice with deletion of <em>Brca1</em> in mammary epithelium [deletion of Brca1 in mammary epithelium (BKO)] (<a href="#R35" class="usa-link" aria-describedby="R35"><em>35</em></a>). Accordingly, BKO mammary tissues were used as a positive control to validate the specificity of R-loop IF signal. Pretreatment with RNase H abolished the nuclear staining of S9.6 antibody, confirming the specificity of S9.6 antibody for detecting R-loops in BKO mouse mammary epithelium (fig. S5, H and I). As shown in <a href="#F5" class="usa-link">Fig. 5</a> (G and H), R-loop signals were significantly increased in the mammary epithelium of <em>Wdhd1<sup>T819A</sup></em> mice compared to that in <em>Wdhd1 <sup>WT</sup></em> mice. As expected, APH treatment elevated R-loop levels in both genotypes, with a robust increase in the mammary epithelium of <em>Wdhd1<sup>T819A</sup></em> mice. These findings strongly support a critical role of AND-1 in suppressing R-loop levels in vivo.</p></section><section id="sec8"><h3 class="pmc_sec_title">AND-1 regulates <em>ESR1</em> expression by suppressing R-loops at regulatory element region of <em>ESR1</em>1 gene (<em>ESR1</em>-RE)</h3>
<p>Recent studies have uncovered a key role of R-loops in cancer propagation and the development of chemotherapy drug resistance, including in breast cancer (<a href="#R58" class="usa-link" aria-describedby="R58"><em>58</em></a>). To explore the clinical potential of targeting the AND-1–mediated R-loop suppression pathway, we examined whether AND-1 inhibition could suppress <em>ESR1</em> expression by regulating R-loops in breast cancer cells. To this end, we first validated the impact of AND-1 depletion on R-loop accumulation in MCF7 breast cancer cells. Both S9.6 dot blot and IF assays demonstrated that AND-1 depletion significantly increased R-loop levels, which could be suppressed by ectopic expression of WT AND-1, but not T826 mutant (<a href="#F6" class="usa-link">Fig. 6, A and B</a>; and fig. S6, A to C). These results indicate that AND-1, and specifically its phosphorylation at T826, is essential for R-loop regulation in MCF7 cells.</p>
<figure class="fig xbox font-sm" id="F6"><h4 class="obj_head">Fig. 6. AND-1 regulates <em>ESR1</em> expression by suppressing R-loops located within the <em>ESR1</em>-RE region.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12327453_sciadv.adv2453-f6.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/32e9/12327453/6bceb14e048d/sciadv.adv2453-f6.jpg" loading="lazy" height="1017" width="746" alt="Fig. 6."></a></p>
<div class="p text-right font-secondary"><a href="figure/F6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong> and <strong>B</strong>) S9.6 dot blot (A) and quantification (B) using genomic DNA extracted from MCF7 cells treated with control siRNA (Ctrl), AND-1 siRNA (siA), or siA with plasmid-mediated expression of the indicated AND-1 variants. (<strong>C</strong>) Schematic representation of the <em>ESR1</em> genomic locus, including <em>ESR1</em> gene (blue), <em>ESR1</em> super-enhancer region (yellow), and <em>ESR1</em> regulatory element (<em>ESR1</em>-RE; red). (<strong>D</strong>) DNA-RNA immunoprecipitation (DRIP) combined with qPCR (DRIP-qPCR) to detect R-loop enrichment at <em>ESR1</em>-RE region in MCF7 cells. Cells were transfected with V5-empty vector (Vec) or V5-RNase H1 (RH1). (<strong>E</strong>) ChIP-qPCR to detect AND-1 enrichment at <em>ESR1</em>-RE region in MCF7 cells transfected with V5-Vec or V5-RH1. (<strong>F</strong>) DRIP-qPCR to detect R-loop enrichment at <em>ESR1</em>-RE region in MCF7 cells expressing V5-Vec or V5-RH1 and treated with Ctrl or siA. (<strong>G</strong> and <strong>H</strong>) qPCR (G) and Western blot (H) to detect <em>ESR1</em> mRNA or protein levels in MCF7 cells treated with Ctrl, siA, or siA with plasmid-mediated expression of the WT AND-1 (WT) or T816A mutant (TA). (<strong>I</strong> and <strong>J</strong>) mRNA (I) and protein (J) levels of <em>ESR1</em> in MCF7 cells transfected with V5-Vec or V5-RH1 plasmid and treated with Con, siA, or siA and AND-1 (WT) or T816A mutant (TA) as in (H). (<strong>K</strong> and <strong>L</strong>) mRNA (K) and protein (L) levels of <em>ESR1</em> in MCF7 cells (WT) and MCF7 cells with depletion of <em>ESR1</em>-RE (<em>ESR1</em>-RE KO). Cells were treated as in (H). Statistical analysis for (B), (D) to (G), (I), and (K). <em>n</em> = 3. Data are shown as means ± SD. *<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; ***<em>P</em> &lt; 0.001; ****<em>P</em> &lt; 0.0001; n.s., not significant, based on unpaired two-tailed Student’s <em>t</em> tests.</p></figcaption></figure><p>Previous studies identified R-loops at a noncoding region upstream of the <em>ESR1</em> locus and the resolution of these R-loops affects <em>ESR1</em> gene expression (<a href="#F6" class="usa-link">Fig. 6C</a>) (<a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>, <a href="#R34" class="usa-link" aria-describedby="R34"><em>34</em></a>, <a href="#R35" class="usa-link" aria-describedby="R35"><em>35</em></a>, <a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>). To validate the presence of R-loops at upstream of the <em>ESR1</em> locus, we conducted DNA-RNA immunoprecipitation (DRIP) combined with quantitative polymerase chain reaction (qPCR; DRIP-qPCR) using primers specific to the <em>ESR1</em> regulatory element (<em>ESR1</em>-RE) region that was reported to contain R-loops (<a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>, <a href="#R59" class="usa-link" aria-describedby="R59"><em>59</em></a>). As shown in <a href="#F6" class="usa-link">Fig. 6D</a> and fig. S6D, R-loops were enriched at the <em>ESR1</em>-RE region. Pretreatment of DNA samples with RNase H (RH) significantly reduced the DRIP signal, confirming the presence of R-loop. Notably, in DNA samples from cells expressing V5-tagged RNase H1, R-loop signals were significantly diminished compared to the control cells. Notably, ChIP-qPCR analysis revealed that AND-1 was also enriched at this <em>ESR1</em>-RE region; however, this association was markedly reduced in cells expressing V5-tagged RNase H1 (<a href="#F6" class="usa-link">Fig. 6E</a> and fig. S6D). These findings suggest that AND-1 binds directly to R-loops present at the <em>ESR1</em>-RE region.</p>
<p>To assess the specificity of R-loop enrichment at <em>ESR1</em>-RE, we extended the DRIP-qPCR analysis to additional regions within <em>ESR1</em> locus, including the enhancer and gene body, as well as the two nearby genes, <em>PPP14CD</em> and <em>PLEKHG1</em>, which were reported to lack R-loops across their entire gene regions (<a href="#R34" class="usa-link" aria-describedby="R34"><em>34</em></a>). As shown in fig. S6 (D to F), DRIP-qPCR signals were only detected at <em>ESR1</em>-RE and at two identified R-loop–positive loci (E-P1 and G-P1) (<a href="#R34" class="usa-link" aria-describedby="R34"><em>34</em></a>), with the signal at <em>ESR1</em>-RE markedly higher than the other two loci.</p>
<p>Moreover, we found that siRNA-mediated deletion of AND-1 increased the DRIP-qPCR signal at <em>ESR1</em>-RE region in cell expressing a V5-tagged empty vector. This elevated signal was suppressed by RNase H pretreatment, confirming the R-loop specificity of the signal (<a href="#F6" class="usa-link">Fig. 6F</a> and fig. S6G). In contrast, in cells expressing V5-tagged RNase H1, the DRIP-qPCR signal at <em>ESR1</em>-RE was markedly reduced compared to the cells expressing the empty vector, and AND-1 depletion had no further effect on DRIP-qPCR signal at <em>ESR1</em>-RE in these cells. These data support a critical role for AND-1 in regulating R-loop formation at <em>ESR1</em>-RE.</p>
<p>We next tested whether AND-1 depletion specifically suppresses <em>ESR1</em> expression. As shown in <a href="#F6" class="usa-link">Fig. 6 (G and H)</a>, both <em>ESR1</em> mRNA levels and <em>ESR1</em> protein (ER-α) levels were significantly reduced in cells treated with siAND-1. This effect was rescued by exogenous expression of WT AND-1, but not T826A mutant.</p>
<p>To determine whether AND-1 regulates <em>ESR1</em> transcription through R-loop modulation at <em>ESR1</em>-RE, we knocked down AND-1 in MCF7 cells expressing either a V5-tagged empty vector or V5-tagged RNase H1. In cells expressing the empty vector, AND-1 depletion significantly reduced <em>ESR1</em> mRNA and protein levels, which were restored by exogenous expression of WT AND-1, but not T826A mutant. In contrast, in cells expressing exogenous RNase H1, <em>ESR1</em> mRNA and protein levels were elevated, and AND-1 depletion had no additional effect (<a href="#F6" class="usa-link">Fig. 6, I and J</a>). These results suggest that the suppression of <em>ESR1</em> expression following AND-1 depletion is dependent on R-loop regulation.</p>
<p>To further investigate the role of AND-1 in regulating <em>ESR1</em> expression, we used an <em>ESR1</em>-RE KO cell line, in which <em>ESR1</em>-RE region was removed by using CRISPR-Cas9 technology as we described previously (<a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>). Consistent with previous findings, <em>ESR1</em> expression was reduced in <em>ESR1</em>-RE KO cells (<a href="#F6" class="usa-link">Fig. 6, K and L</a>) (<a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>). Notably, depletion of AND-1 by siRNA reduced expression of both <em>ESR1</em> mRNA and protein levels in WT cells, but not in <em>ESR1</em>-RE KO cell lines, further indicating that AND-1 regulates <em>ESR1</em> expression by affecting R-loops accumulation at <em>ESR1</em>-RE.</p>
<p>To test whether AND-1 also regulates R-loop resolution at other R-loop–prone loci, we performed DRIP-qPCR analysis at several well-characterized R-loop–enriched regions, including <em>RPL13A</em>, <em>TRIM33</em>, <em>TPCN2</em>, and <em>SRRT</em> (<a href="#R34" class="usa-link" aria-describedby="R34"><em>34</em></a>, <a href="#R60" class="usa-link" aria-describedby="R60"><em>60</em></a>). As shown in fig. S6H, AND-1 depletion led to increased R-loop accumulation at all these loci, suggesting that AND-1’s role in R-loops is not restricted to <em>ESR1</em>-RE loci.</p></section><section id="sec9"><h3 class="pmc_sec_title">Inhibition of AND-1 overcomes resistance to AI</h3>
<p>Our above data demonstrated that inhibition of AND-1 suppresses <em>ESR1</em> expression, suggesting that targeting AND-1 may be a useful approach to overcome resistance to AIs driven by <em>ESR1</em> mutations in breast cancer cells. To test this hypothesis, we treated MCF7 cells with two newly identified AND-1 inhibitors, BZA and CH3, which specifically inhibit AND-1 by promoting its degradation (<a href="#R61" class="usa-link" aria-describedby="R61"><em>61</em></a>). In cells expressing V5-tagged empty vector, both inhibitors resulted in AND-1 protein degradation and decreased <em>ESR1</em> expression at both mRNA and protein levels. In contrast, in cells expressing V5-tagged RNase H1, the effect of the AND-1 inhibitors on <em>ESR1</em> expression was reduced. These results indicate that AND-1 inhibitors suppress <em>ESR1</em> expression through R-loop regulation (fig. S7, A and B).</p>
<p>To investigate whether targeting AND-1 could overcome AR in breast cancer cells, we knocked down AND-1 in WT and AR MCF7 cells expressing <em>ESR1</em> mutations, including Y537S, Y537N, or D538G (fig. S7C), and then treated the cells with letrozole, an AI (<a href="#R62" class="usa-link" aria-describedby="R62"><em>62</em></a>). Notably, AND-1 knockdown by siRNA decreased ER-α levels (fig. S7D) and restored the sensitivity of all three AR MCF7 cells to letrozole (fig. S7, E to G). Additionally, a strong synergistic effect was observed when combining AND-1 inhibitor (CH3) with letrozole in both WT and AR MCF7 cells (<a href="#F7" class="usa-link">Fig. 7, A to D</a>; and fig. S7, H to M). We next subcutaneously implanted AR MCF7 cells (Y537S) into nude mice to form tumors, which were then treated with a combination of CH3 and letrozole. In line with the in vitro results, the combined treatments of CH3 with letrozole exhibited a significant synergistic effect in suppressing resistant tumor growth with minimal effects on the mice’s body weight (<a href="#F7" class="usa-link">Fig. 7, E to G</a>). Compared with the control and the letrozole treatment groups, tumors in mice treated with CH3 alone or CH3 combined with letrozole displayed significantly reduced expression of AND-1 and ER-α proteins (<a href="#F7" class="usa-link">Fig. 7H</a>) and increased expression of R-loops (<a href="#F7" class="usa-link">Fig. 7, I and J</a>). Together, these results suggest that targeting AND-1 could overcome AR caused by <em>ESR1</em> mutations in ER<sup>+</sup> breast cancer cells.</p>
<figure class="fig xbox font-sm" id="F7"><h4 class="obj_head">Fig. 7. Inhibition of AND-1 overcomes resistance to AI (letrozole).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12327453_sciadv.adv2453-f7.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/32e9/12327453/a3674dcbeafa/sciadv.adv2453-f7.jpg" loading="lazy" height="1017" width="768" alt="Fig. 7."></a></p>
<div class="p text-right font-secondary"><a href="figure/F7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong> and <strong>B</strong>) Surface plots (top) and synergy matrix (bottom) showing the synergistic effect of AND-1 inhibitor (CH3) with AI (letrozole) in breast cancer cells, including WT (MCF7 WT) (A) and letrozole resistant cells (MCF7 Y537S) (B). (<strong>C</strong> and <strong>D</strong>) Synergetic effect of CH3 and Letrozole in MCF7 WT (C) and MCF7 Y537S (D) cells. Cells treated with indicated concentration of inhibitors were harvested 72 hours posttreatment. The combination index (CI) values were presented below the bars. (<strong>E</strong> and <strong>F</strong>) Tumor growth of MCF7 Y537S xenograft tumor-bearing mice intraperitoneally treated with control, letrozole (20 mg/kg), CH3 (20 mg/kg), or combination of letrozole and CH3 for 20 days, administered once every 2 days. <em>n</em> = 5. Data are shown as means ± SD. **<em>P</em> &lt; 0.01; ****<em>P</em> &lt; 0.0001; n.s., not significant. All the data were analyzed on the basis of unpaired two-tailed Student’s <em>t</em> tests. (<strong>G</strong>) Body weights of MCF7 Y537S xenograft tumor-bearing mice during treatment as in (E) and (F). (<strong>H</strong>) Immunoblot analysis of AND-1 and ER-α protein levels of tumor samples in each experiment group. Five animals were used in each group. GAPDH, glyceraldehyde-3-phosphate dehydrogenase. (<strong>I</strong> and <strong>J</strong>) S9.6 dot blot (I) and quantification (J) using genomic DNA extracted from tumor samples from mice bearing MCF7 Y537S xenograft tumors intraperitoneally treated with indicated drugs as in (E) and (F). Five animals were used in each group. Data are shown as means ± SD. ****<em>P</em> &lt; 0.0001; n.s., not significant, based on unpaired two-tailed Student’s <em>t</em> tests. (<strong>K</strong>) Schematic representation of the mechanism. AND-1 directly binds to R-loop structures, where it promotes RNase H1 recruitment to R-loops and stimulates endonuclease activity of RNase H1.</p></figcaption></figure></section></section><section id="sec10"><h2 class="pmc_sec_title">DISCUSSION</h2>
<p>R-loop dysregulation has been implicated in immune response, disease progression, and cancer development (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>). Therefore, understanding the mechanisms of regulating R-loops is particularly important. In this study, we identified AND-1 as an R-loop–interacting protein that plays a critical role in R-loop regulation, as demonstrated by studies in both cells and animal models. We further show that inhibition of AND-1, by either siRNA-mediated depletion or treatment with specific small-molecule inhibitors, can overcome <em>ESR1</em> mutation-driven resistance to the AI letrozole, through modulation of R-loop dynamics. Collectively, our study not only elucidates a previously unidentified mechanism of R-loop regulation but also provides preclinical evidence that targeting AND-1 may offer a promising therapeutic target for the treatment of endocrine-resistant breast cancer.</p>
<p>Ctf4, the homolog of AND-1 in yeast cells, was shown to be required for R-loop regulation (<a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>). However, the underlying mechanism by which it modulates R-loops remains unsolved. In this study, we uncovered a previously unidentified role for AND-1 in the early stage of R-loop regulation by modulating both the recruitment and the endonuclease activity of RNase H1 at R-loops. AND-1 contains a C-terminal HMG DNA-binding domain, which contributes to its direct interaction with R-loops, as observed both in vitro and in cell-based studies. The HMG domain usually facilitates the formation of nucleoprotein complexes on the chromatin via bending DNA (<a href="#R48" class="usa-link" aria-describedby="R48"><em>48</em></a>, <a href="#R63" class="usa-link" aria-describedby="R63"><em>63</em></a>). It is likely that AND-1 directly binds to R-loops and alters chromatin architecture at R-loop regions to enhance RNase H1 accessibility. Given that AND-1 also interacts with RNase H1, our working model (<a href="#F7" class="usa-link">Fig. 7K</a>) is that AND-1 directly binds to R-loops and promotes a favorable chromatin change that facilitates RNase H1’s recruitment to R-loops via the AND-1–RNase H1 interaction. Further structural analysis will help to elucidate the details of how AND-1 remodels R-loop–containing chromatin to facilitate the recruitment of RNase H1 to R-loops.</p>
<p>While Ctf4/AND-1 is known to function in DNA replication (<a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>, <a href="#R64" class="usa-link" aria-describedby="R64"><em>64</em></a>, <a href="#R65" class="usa-link" aria-describedby="R65"><em>65</em></a>), one question is whether the observed R-loop accumulation upon AND-1 depletion is a secondary consequence of impaired DNA replication. To address this, we performed all the experiments at 48 hours post-siRNA transfection, before the onset of detectable DNA replication defects, which only became evident at 72 hours posttransfection, as assessed by EdU incorporation and fluorescence-activated cell sorting analysis (fig. S1, M and N). Furthermore, our data show that the interaction between AND-1 and R-loops is not restricted to S-phase cells (fig. S2, B and C), supporting the idea that the role of AND-1 in R-loops is not dependent on DNA replication. Therefore, the R-loop accumulation observed upon AND-1 loss in this study is unlikely to be a consequence of replication stress. Notably, AND-1 is a replication fork-associated protein, and its accumulation at stalled replication forks is known to facilitate the recruitment of DNA repair factors to maintain genomic stability (<a href="#R49" class="usa-link" aria-describedby="R49"><em>49</em></a>, <a href="#R64" class="usa-link" aria-describedby="R64"><em>64</em></a>). Our findings extend its role by demonstrating that AND-1 contributes to genome stability through not only replication fork maintenance but also a previously unrecognized function in sensing R-loops and promoting RNase H1 activity. Collectively, these results highlight the multifaceted role of AND-1 in safeguarding genome integrity through direct interactions with distinct DNA structures.</p>
<p>Notably, we also provided in vivo evidence that R-loop levels are elevated in mammary epithelial cells of AND-1–deficient (T819A) mice, indicating a key role for AND-1 in R-loop regulation in vivo. Ctf4/AND-1 is an important factor to maintain chromosome stability (<a href="#R66" class="usa-link" aria-describedby="R66"><em>66</em></a>, <a href="#R67" class="usa-link" aria-describedby="R67"><em>67</em></a>). Therefore, future studies investigating whether R-loop accumulation due to AND-1 deficiency contributes to tumorigenesis in animal models would greatly advance our understanding of its role in cancer development.</p>
<p>ER<sup>+</sup> breast cancer, which accounts for a substantial portion of breast cancer cases, often develops resistance to AIs, resulting in treatment failure (<a href="#R68" class="usa-link" aria-describedby="R68"><em>68</em></a>). One of the major causes of ET resistance is mutations in the <em>ESR1</em> gene, which lead to its constitutive transcriptional activity and reduced ET sensitivity (<a href="#R69" class="usa-link" aria-describedby="R69"><em>69</em></a>). Targeting <em>ESR1</em> mutations remains a critical unmet clinical need, as these <em>ESR1</em> mutations are rare in those with primary breast cancer but occur in ~30% of patients who develop metastasis after AI treatment (<a href="#R32" class="usa-link" aria-describedby="R32"><em>32</em></a>). Our study demonstrates that AND-1 plays a pivotal role in controlling R-loop levels in breast cancer cells and that AND-1 depletion leads to decreased <em>ESR1</em> expression. This reduction is due to increased R-loop accumulation within the <em>ESR1</em>-RE locus, a region previously shown to form R-loop structures (<a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>, <a href="#R34" class="usa-link" aria-describedby="R34"><em>34</em></a>, <a href="#R35" class="usa-link" aria-describedby="R35"><em>35</em></a>, <a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>). Notably, we show that pharmacological inhibition of AND-1 using small molecules such as BZA and CH3 effectively suppresses <em>ESR1</em> expression and overcomes the <em>ESR1</em> mutation-driven AR in both in vitro and in vivo models. AND-1/<em>WDHD1</em> is highly expressed across multiple tumor types but exhibits extremely low expression in normal tissues (fig. S7N) (<a href="#R70" class="usa-link" aria-describedby="R70"><em>70</em></a>). Furthermore, pharmacological inhibition of AND-1 causes minimal toxicity in mice (<a href="#R61" class="usa-link" aria-describedby="R61"><em>61</em></a>), further supporting its potential as a tumor-selective therapeutic target.</p>
<p>The precise mechanism by which R-loops influence <em>ESR1</em> transcription remains to be fully elucidated. Our results highlight a regulatory role of AND-1 on <em>ESR1</em> expression by affecting R-loop resolution at the <em>ESR1</em>-RE locus, indicating a potential approach to overcome <em>ESR1</em> mutation-mediated endocrine resistance by targeting AND-1 in breast cancer. However, we must realize that AND-1’s role in R-loops is not restricted to the <em>ESR1</em>-RE, and we still do not know how AND-1 regulates R-loops as well as R-loop–mediated gene expression at a genome-wide level. In addition, the molecular mechanism of how unresolved R-loops at the <em>ESR1</em>-RE locus affects its transcription is still not clear. We are planning to answer these questions in our future studies, which we believe will provide more evidence to support anti–AND-1–mediated therapy for cancer treatment.</p>
<p>Collectively, our findings provide preclinical evidence to support the development of approaches targeting mutated <em>ESR1</em> at the transcriptional level by inhibiting AND-1 to treat ET-resistant breast cancer. Notably, BZA is a Food and Drug Administration–approved SERM for osteoporosis (<a href="#R71" class="usa-link" aria-describedby="R71"><em>71</em></a>), raising the possibility of repurposing BZA for clinical trials in AI-resistant breast cancer patients in the near future.</p></section><section id="sec11"><h2 class="pmc_sec_title">MATERIALS AND METHODS</h2>
<section id="sec12"><h3 class="pmc_sec_title">Cell culture</h3>
<p>Human U2OS (RRID: CVCL_0042), HEK293T (RRID: CVCL_0045), and WT MCF7 cells (RRID: CVCL_0031) were purchased from American Type Culture Collection. MCF7 mutant cell lines (Y537S, Y537N, and D538G) were a gift from J. Shao (Washington University, St. Louis) (<a href="#R72" class="usa-link" aria-describedby="R72"><em>72</em></a>). <em>ESR1</em>-RE KO MCF7 cells were generated as previously described (<a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>). All cells were cultured in Dulbecco’s modified Eagle’s medium supplemented with 10% fetal bovine serum at 37°C with 5% CO<sub>2</sub>. HEK293T AND-1 KO cells were generated by using CRISPR-Cas9–mediated technology. Specific guide RNAs (sgRNAs) were designed using online CRISPR Design Tool (<a href="https://portals.broadinstitute.org/gppx/crispick/public" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://portals.broadinstitute.org/gppx/crispick/public</a>), and sgRNAs with the highest score (<a href="#T1" class="usa-link">Table 1</a>) were chosen and ligated into the Lenti V2 plasmid according to the protocol described previously (<a href="#R73" class="usa-link" aria-describedby="R73"><em>73</em></a>). Cells transfected with control or sgRNA were selected in a medium with puromycin (2.5 μg/ml) for 2 to 3 weeks.</p>
<section class="tw xbox font-sm" id="T1"><h4 class="obj_head">Table 1. sgRNA/siRNA target sequences and PCR primer sequences.</h4>
<div class="caption p"><p>3′UTR, 3′ untranslated region; qRT-PCR, quantitative reverse transcription polymerase chain reaction; F-primer, forward primer; R-primer, reverse primer.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col width="%" span="1">
<col width="%" span="1">
<thead><tr>
<th valign="top" align="left" scope="col" rowspan="1" colspan="1">sgRNA/siRNA oligoes</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Target sequences</th>
</tr></thead>
<tbody>
<tr>
<td scope="row" valign="top" align="left" rowspan="1" colspan="1">AND-1 single-guide RNA (sgRNA)</td>
<td valign="top" align="center" rowspan="1" colspan="1">TGACTTAGCTGAAACATCAG</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">AND-1 siRNA-1 (sense)</td>
<td valign="top" align="center" rowspan="1" colspan="1">AAGCAGGCAUCUGCAGCAUCC</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">AND-1 siRNA-2 (sense)</td>
<td valign="top" align="center" rowspan="1" colspan="1">AGGAAAACAUGCCUGCCAC</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">AND-1 siRNA-3 (3′UTR) (sense)</td>
<td valign="top" align="center" rowspan="1" colspan="1">GCAAAGUUAUGGAAAGUAU</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Control siRNA (sense)</td>
<td valign="top" align="center" rowspan="1" colspan="1">AACGUACGCGGAAUACUUCGA</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<strong>Primers for qRT-PCR of the ChIP experiment</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">
<strong>Primer sequences</strong>
</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">β-<em>Actin</em> pause F-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">GGGACTATTTGGGGGTGTCT</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">β-<em>Actin</em> pause R-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">TCCCATAGGTGAAGGCAAAG</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">β-<em>Actin</em> control F-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">CAGTGGTGTGGTGTGATCTTG</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">β-<em>Actin</em> control R-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">GGCAAAACCCTGTATCTGTGA</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>GEMIN7</em> pause F-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">AGCTCACGCTGGTTCTTTCTT</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>GEMIN7</em> pause R-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">GAAATTCCAAAGGCGAGAGAC</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>GEMIN7</em> control F-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">GATTCTATTTGGGCCACCTATG</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>GEMIN7</em> control R-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">GGGAGGCATCTAAACCTCATC</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<strong>Primers for qRT-PCR of the DRIP experiment</strong>
</td>
<td valign="top" align="center" rowspan="1" colspan="1">
<strong>Primer sequences</strong>
</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>ESR1</em>-RE F-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">TGGACGTTAACTGCAGCCCA</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>ESR1</em>-RE R-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">TCATCCCATAATCAGGACCATCAAT</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Geneless F-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">CCTGTGTCTGACATATGATTGGTTT</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Geneless R-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">CCATTGCCTGGCTGTTATTTG</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>RPL13A</em> F-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">AGGTGCCTTGCTCACAGAGT</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>RPL13A</em> R-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">GGTTGCATTGCCCTCATTAC</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>TRIM33</em> F-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">ATGCCCAGCTTTCCCTAACT</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>TRIM33</em> R-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">GGAAAGTGGACTGCATGGTT</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>TPCN2</em> F-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">ACCGTTCAGGGAACACAGAC</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>TPCN2</em> R-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">TTAAGCAACCTGCCAAGGTC</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>SRRT</em> F-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">ACTCGCCTGCTCACTCATTT</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">
<em>SRRT</em> R-primer</td>
<td valign="top" align="center" rowspan="1" colspan="1">GACAGGAGTTCACCGGACAT</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec13"><h3 class="pmc_sec_title">Antibodies and plasmids</h3>
<p>The antibodies against AND-1 were raised against a synthetic peptide (SKKQKPLDFSTNQKLSAF) of AND-1. The antibody against AND-1 at the T826 site (p–AND-1 T826) was described previously (<a href="#R49" class="usa-link" aria-describedby="R49"><em>49</em></a>). S9.6 antibody (ENH002, RRID: AB_2687463) was from Kerafast. Antibody against RNase H1 (no. ab229078, RRID: AB_945244) was from Abcam. Antibodies against ER-α (no. 8644, RRID: AB_2617128), β-tubulin (no. 2146, RRID: AB_2210545), β-Actin (no. 8457, RRID: AB_10950489), glyceraldehyde-3-phosphate dehydrogenase (no. 5174, RRID: AB_10622025), V5-Tag (no. 13202, RRID: AB_2687461), FLAG-Tag (no. 14793, RRID: AB_2572291), His-Tag (no. 2365, RRID: AB_2115720), and glutathione <em>S</em>-transferase (GST)–Tag (no. 2622, RRID: AB_331670) were from Cell Signaling Technology. The antibody against BSA (no. B2901, RRID: AB_258533) was from MilliporeSigma.</p>
<p>The plasmid expressing pCDH-EF1-FLAG-RNase H1 (M27) under the control of an EF-1α promoter was a gift from R. Li (The George Washington University, Washington, DC). The M27 form of RNaseH1 primarily localizes to the nucleus (<a href="#R74" class="usa-link" aria-describedby="R74"><em>74</em></a>). The RNase H1 mutation (D210N) was generated using the QuikChange Site-Directed Mutagenesis Kit (no. 200518, Agilent). FLAG–AND-1 and GST–AND-1 (full-length and various fragments) were constructed as described previously (<a href="#R25" class="usa-link" aria-describedby="R25"><em>25</em></a>).</p></section><section id="sec14"><h3 class="pmc_sec_title">siRNA transfection</h3>
<p>Individual siRNAs were synthesized from Sigma-Aldrich (see <a href="#T1" class="usa-link">Table 1</a> for target sequences). siRNA transfections were performed with a final concentration of 20 nM siRNA oligonucleotide duplexes using Lipofectamine RNAiMAX (Invitrogen) according to the manufacturer’s instructions. Experiments were carried out 3 days after siRNA transfection. siRNA oligonucleotides against AND-1 are shown in <a href="#T1" class="usa-link">Table 1</a>.</p></section><section id="sec15"><h3 class="pmc_sec_title">Co-IP and Western blots</h3>
<p>Cells were lysed in NETN lysis buffer containing 50 mM tris-HCl (pH 8.0), 250 mM NaCl, 5 mM EDTA (pH 8.0), 0.5% NP-40, and 1× protease inhibitor cocktail (no. 11836170001, Roche) for 30 min on ice, followed by sonication. Benzonase nuclease (E1014, Sigma-Aldrich) was included during the co-IP assay to degrade DNA and RNA, ensuring that the observed protein-protein interactions were not bridged by nucleic acids. After centrifugation, the supernatants were collected and incubated with indicated antibodies preconjugated with protein A/G beads at 4°C overnight. Subsequently, beads were then washed three times with lysis buffer and analyzed by Western blot. For Western blot analysis, samples were boiled at 95°C for 10 min in SDS loading buffer. Then, they were subjected to electrophoresis in 8 to 12% SDS–polyacrylamide gel electrophoresis (PAGE) and transferred to the nitrocellulose membrane. The membranes were blocked with 5% nonfat milk in Tris-buffered saline with Tween 20 (TBST) for 1 hour before being incubated with primary antibody at 4°C overnight. Then, the membranes were washed three times with 0.1% TBST and incubated with horseradish peroxidase (HRP)–conjugated secondary antibody (1:5000) for 2 hours at room temperature (RT) followed by three times wash with 0.1% TBST. SuperSignal West Pico PLUS Chemiluminescent Substrate (no. 34580, Thermo Fisher Scientific) was used for protein detection. Images were acquired in the Odyssey Fc imaging system (LICORbio) with Image Studio software.</p></section><section id="sec16"><h3 class="pmc_sec_title">Chromatin immunoprecipitation</h3>
<p>ChIP assays were carried out with the SimpleChIP Enzymatic Chromatin IP Kit (no. 9003, Cell Signaling Technology) according to the manufacturer’s instruction manual. Briefly, cells were cross-linked with 1% formaldehyde for 10 min and quenched with 1× glycine. Extracted genomic DNA was sonicated to an average size of 200 bp. ChIPs were performed using ChIP-validated antibodies and ChIP-Grade Protein G Magnetic Beads. After reversal of protein-DNA cross-links, the DNA was eluted and purified. The enrichment of particular DNA sequences was analyzed by quantitative real-time polymerase chain reaction (PCR) using primers listed in <a href="#T1" class="usa-link">Table 1</a>.</p></section><section id="sec17"><h3 class="pmc_sec_title">DNA-RNA immunoprecipitation</h3>
<p>DRIP was performed with modifications from previously described (<a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>, <a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>, <a href="#R59" class="usa-link" aria-describedby="R59"><em>59</em></a>). Briefly, genomic DNA extracted from U2OS or MCF7 cells was pretreated with 1:200 dilutions of RNase T1 (Thermo Fisher Scientific, EN0541), ShortCut RNase III (New England Biolabs, M0245S), RNase A (Thermo Fisher Scientific, EN0531) at 37°C for 1 hour to reduce noise as the S9.6 antibody recognizes non–R-loop cellular RNAs (<a href="#R75" class="usa-link" aria-describedby="R75"><em>75</em></a>). The genomic DNA was sonicated to an average size of 200 bp. For immunoprecipitation, 7.5 μg of genomic DNA was incubated overnight at 4°C with 10 μl of S9.6 antibody or no antibody control in binding buffer (10 mM sodium phosphate, 140 mM NaCl, and 0.05% Triton X-100) overnight at 4°C. Dynabeads incubation, washes, and elution steps were carried out following the same procedure described for ChIP. RNase H sensitivity was performed by adding 1.7 U of RNase H (New England Biolabs, M0297S) per microgram of genomic DNA for 2 hours or overnight at 37°C before the IP step. Locus-specific DRIP signals were assessed by reverse transcription PCR using primers listed in <a href="#T1" class="usa-link">Table 1</a> and table S1.</p></section><section id="sec18"><h3 class="pmc_sec_title">S9.6 co-IP assay</h3>
<p>S9.6 co-IP was performed as described previously (<a href="#R75" class="usa-link" aria-describedby="R75"><em>75</em></a>), with modifications. Briefly, genomic DNA was extracted from non–cross-linked cells and pretreated with a combination of RNases A, T1, and III and then was sonicated to an average size of 200 bp, following the same procedures in DRIP. Genomic DNA (7.5 μg) was incubated overnight at 4°C with 10 μl of S9.6 antibody or immunoglobulin G and 20 μl of Dynabeads Protein A or Protein G (Invitrogen) in co-IP buffer overnight at 4°C. Beads were washed three times with co-IP buffer and incubated with nuclear extraction pretreated with RNase A at 4°C overnight. The beads were then washed three times with co-IP buffer and eluted for Western blot.</p></section><section id="sec19"><h3 class="pmc_sec_title">S9.6 dot blot</h3>
<p>Either U2OS or MCF7 genomic DNA was collected and sonicated to an average size of 200 bp according to the same protocols as described in DRIP. DNA fragments were diluted in TE buffer and pretreated with a combination of RNases A, T1, and III to remove single-stranded RNA/double-stranded RNA and spotted onto a positively charged nylon membrane. RNase H sensitivity treatment was performed before spotting DNA onto the membrane. Once dried, the membrane was ultraviolet (UV)–cross-linked and blocked in 5% milk/TBST at RT for 1 hour. Membranes were incubated overnight at 4°C with mouse S9.6 antibodies (Kerafast S9.6, 0.5 μg/ml) or methylene blue (NYLM-RO, Roche). Membranes were washed three times with TBST and incubated with a secondary antibody (anti-mouse HRP, 1:5000 in TBST) at RT for 2 hours before developing on a film using ECL Western Blotting Detection Reagent (Thermo Fisher Scientific, 34580).</p></section><section id="sec20"><h3 class="pmc_sec_title">EdU-488 staining versus forward scatter</h3>
<p>U2OS cells with the indicated treatment were incubated with 10 μM EdU for 30 min. After the click reaction, the cells were analyzed by flow cytometry to assess EdU incorporation. Data were collected and analyzed using an Attune Acoustic Cytometer using 488-nm excitation and a 530/30 bandpass for detection of the EdU Alexa Fluor 488 azide and 405-nm excitation and a 450/40 bandpass for detection of the FxCycle Violet fluorescence.</p></section><section id="sec21"><h3 class="pmc_sec_title">Immunofluorescence</h3>
<p>For R-loop IF using the S9.6 antibody performed in cells, samples were prepared using a previously described protocol (<a href="#R40" class="usa-link" aria-describedby="R40"><em>40</em></a>, <a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>). Briefly, U2OS and MCF7 cells were collected using trypsin and incubated with prewarmed KCl solution (75 mM) at 37°C for 12 min. Cells were pelleted (1000 rpm, 5 min, 25°C) and resuspended in freshly made, ice-cold methanol:acetic acid (3:1) on ice for 20 min. After one time wash with methanol:acetic acid (3:1), cells were spotted onto slides and incubated with blocking buffer [1× phosphate-buffered saline (PBS), 5% BSA, and 0.5% Triton X-100] for 1 hour at RT. Enzymatic treatments were done before primary antibody incubation. Cells were incubated with 1:200 dilutions of RNase T1 (Thermo Fisher Scientific, EN0541), ShortCut RNase III (New England Biolabs, M0245S), RNase A (Thermo Fisher Scientific, EN0531), and/or RNase H (New England Biolabs, M0297S) at 37°C for 1 hour (<a href="#R75" class="usa-link" aria-describedby="R75"><em>75</em></a>–<a href="#R77" class="usa-link" aria-describedby="R77"><em>77</em></a>). Samples were subsequently washed and incubated with primary S9.6 antibody (Karafast, ENH001, 1:1000) overnight at 4°C and then incubated with Alexa 488–conjugated secondary antibody for 45 min at RT. Cells were stained with 4,6-diamidino-2-phenylindole (DAPI) and mounted using Vectorshield (Vector Laboratories). R-loop intensity was quantified using the MetaMorph Microscopy Automation and Image Analysis Software 7.8 as previously described (<a href="#R35" class="usa-link" aria-describedby="R35"><em>35</em></a>).</p>
<p>For R-loop IF in breast mammary gland samples, experiments were carried out according to a previous description (<a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>). Briefly, freshly cut 3-μm formalin-fixed paraffin-embedded samples were performed with deparaffin, rehydration, and antigen unmasking. Samples were then treated with 0.2× saline sodium citrate buffer for 20 min and incubated in a staining buffer (TBST with 0.1% BSA) for 10 min with rocking. Enzymatic treatments with 1:200 dilutions of RNase T1, ShortCut RNase III, RNase A, and/or RNase H in staining buffer supplemented with 3 mM magnesium chloride were performed before incubation with the S9.6 Primary antibody (Karafast, ENH001, 1:100). The following steps of washing and secondary antibody incubation were the same as in cell R-loop IF described above. Samples were then mounted with DAPI and analyzed as described (<a href="#R35" class="usa-link" aria-describedby="R35"><em>35</em></a>, <a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>). The relative fluorescence signal of R-loop was analyzed by ImageJ.</p></section><section id="sec22"><h3 class="pmc_sec_title">In situ PLA</h3>
<p>In situ PLAs were conducted in U2OS/MCF7 cells with the Duolink In Situ Red Starter Kit Mouse/Rabbit (MilliporeSigma, DUO92101) according to the manufacturer’s instructions. Briefly, cells were fixed and incubated with the primary antibody as described in IF experiments. The cells were treated with PLA probe incubation, ligation, and amplification. Duolink In Situ Mounting Medium with DAPI was used for mounting cells. The result from a Duolink PLA experiment was viewed using a fluorescence microscope with the appropriate filters for the detection of the fluorophore.</p></section><section id="sec23"><h3 class="pmc_sec_title">Protein purification</h3>
<p>Various N-terminally 3xFAG-tagged and C-terminally His-tagged human AND-1 proteins (WT and mutants) and N-terminally 3xFAG-tagged human RNaseH1 (WT and D210N mutant) were purified from AND-1 KO HEK293T cells. Plasmids were transfected into AND-1 KO HEK293T cells. Cells were harvested 36 hours after transfection and lysed in NETN lysis buffer containing protease inhibitor cocktail for 30 min on ice, followed by sonication and clarified by centrifugation (12,000g for 10 min). The supernatants were incubated with ANTI-FLAG M2 Affinity Agarose Gel (A2220, MilliporeSigma) at 4°C overnight. The agarose gel was washed three times with NETN lysis buffer and subsequently eluted with FLAG Peptide (F3290, MilliporeSigma). AND-1 proteins were concentrated using Ultracel-100 K centrifugal filters, and RNase H1 fractions were concentrated using Ultracel-30 K centrifugal filters and stored at −80°C in small aliquots.</p></section><section id="sec24"><h3 class="pmc_sec_title">R:D hybrid substrates for EMSA</h3>
<p>The oligonucleotide sequences used to make R:D hybrids or R-loops were described previously (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>). The RNA oligo was 5′-biotin labeled. Oligos were annealed by heating to 95°C in a water bath for 5 min and slowly cooled down overnight in buffer H [90 mM tris-HCl (pH 7.5), 10 mM MgCl<sub>2</sub>, and 50 mM NaCl]. Annealed substrates were separated on a 10% native PAGE gel at 4°C. The corresponding gel band of the indicated annealed substrate was excised, purified, and stored at −20°C. Oligonucleotides used for in vitro substrates were listed in <a href="#T2" class="usa-link">Table 2</a>. For the R:D substrate, the biotin-labeled RNA oligo was annealed with oligo 4. For the dsDNA substrate, the biotin-labeled oligo 3 was annealed with oligo 4. For the R-loop substrate, the biotin-labeled RNA oligo was annealed with oligos 1 and 2. For the D-loop substrate, the biotin-labeled oligo 3 was annealed with oligos 1 and 2. EMSA was performed using the LightShift Chemiluminescent EMSA Kit (Thermo Scientific, 20148) according to the manufacturer’s instructions. Briefly, R-loop substrates containing 5′-biotin–labeled RNA were incubated with various amounts of AND-1 at 30°C for 15 min (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>). The resulting protein-substrate complexes were resolved on 6% PAGE and then transferred to Nylon Membrane. After that, the transferred DNA was UV–cross-linked and detected by chemiluminescence.</p>
<section class="tw xbox font-sm" id="T2"><h4 class="obj_head">Table 2. List of oligonucleotides used in vitro substrates.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col width="%" span="1">
<col width="%" span="1">
<thead><tr>
<th valign="top" align="left" scope="col" rowspan="1" colspan="1">Name</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Sequence</th>
</tr></thead>
<tbody>
<tr>
<td scope="row" valign="top" align="left" rowspan="1" colspan="1">Oligo 1</td>
<td valign="top" align="center" rowspan="1" colspan="1">5′GCCAGGGACGAGGTGAACCTGCAGGTGGGCGGCTACTACTTAGATGTCATCCGAGGCTTATTGGTAGAATTCGGCAGCGTCATGCGACGGC</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Oligo 2</td>
<td valign="top" align="center" rowspan="1" colspan="1">5′GCCGTCGCATGACGCTGCCGAATTCTACCACGCGATTCATACCTGTCGTGCCAGCTGCTTTGCCCACCTGCAGGTTCACCTCGTCCCTGGC</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Oligo 3</td>
<td valign="top" align="center" rowspan="1" colspan="1">(Biotin)5′GCAGCTGGCACGACAGGTATGAATC</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">Oligo 4</td>
<td valign="top" align="center" rowspan="1" colspan="1">5′GATTCATACCTGTCGTGCCAGCTGC</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">RNA oligo</td>
<td valign="top" align="center" rowspan="1" colspan="1">(Biotin)5′GCAGCUGGCACGACAGGUAUGAAUC</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">FAM-RNA oligo</td>
<td valign="top" align="center" rowspan="1" colspan="1">(FAM)5′GCAGCUGGCACGACAGGUAUGAAUC</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">ROX-DNA oligo</td>
<td valign="top" align="center" rowspan="1" colspan="1">GATTCATACCTGTCGTGCCAGCTGC3′(ROX)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec25"><h3 class="pmc_sec_title">Fluorescence-based RNase H1 activity assay</h3>
<p>The assay was preformed according to previous publications with modification (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>, <a href="#R52" class="usa-link" aria-describedby="R52"><em>52</em></a>, <a href="#R53" class="usa-link" aria-describedby="R53"><em>53</em></a>). Specifically, the assay to measure RNase H1 activity was performed in a 96 Well Black/Clear Bottom Plate (Thermo Scientific, no. 152036) and monitored by fluorescence stopped flow on SpectraMax i3x (Molecular Devices). R:D hybrid substrate was synthesized using oligos of 5′-FAM–labeled RNA oligo and 3′-ROX–labeled DNA oligo shown in <a href="#T2" class="usa-link">Table 2</a>. Then, the two strands were annealed by heating to 95°C and slowly cooling over a long period of time in annealing buffer [90 mM tris-HCl (pH 7.5), 10 mM MgCl<sub>2</sub>, and 50 mM NaCl]. Annealed substrates were separated on a 10% native PAGE in Tris-Borate-EDTA (TBE) buffer at 4°C. The gel band corresponding to the annealed substrate was excised, purified, and lastly eluted. After annealing, the FAM moiety was quenched by the ROX moiety in close proximity. Once RNase H1 cleaves the RNA strand within the R:D substrate, the FAM moiety can be detected at the excitation of 488 nm. Experiments were conducted at 30°C in reaction buffer [25 mM tris-HCl (pH 7.5), 1 mM dithiothreitol (DTT), 5 mM MgCl<sub>2</sub>, BSA (50 μg/ml), and 50 mM KCl]. The concentration of R:D substrate was 50 nM, and RNase H1 and AND-1 protein concentrations varied from 0 to 10 nM and from 0 to 40 nM, respectively. The indicated concentration of AND-1 and RNase H1 was preincubated with R:D substrate for 5 min at 30°C in a 50-μl system, followed by the addition of 50-μl 2× reaction buffer, and the fluorescence signal was monitored for 40 min using the Spectramax i3x machine with excitation at 488 nm and emission at 535 nm. Control experiments included protein mixture alone or R:D substrates alone mixed with reaction buffer. Data were analyzed and plotted using GraphPad Prism 10 software (GraphPad).</p></section><section id="sec26"><h3 class="pmc_sec_title">Gel-based RNase H1 activity assay</h3>
<p>The assay was performed according to previous publications with modification (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>, <a href="#R54" class="usa-link" aria-describedby="R54"><em>54</em></a>, <a href="#R55" class="usa-link" aria-describedby="R55"><em>55</em></a>). R:D hybrid substrates were synthesized using oligos of 5′-FAM–labeled RNA oligo and DNA oligo 3 shown in <a href="#T2" class="usa-link">Table 2</a>. For the R-loop substrates, the 5′-FAM–labeled RNA oligo was annealed with DNA oligos 1 and 2. The indicated concentration of RNaseH1 was preincubated with varying amounts of AND-1 on ice for 10 min in reaction buffer [25 mM tris-HCl (pH 7.5), 1 mM DTT, 5 mM MgCl<sub>2</sub>, BSA (50 μg/ml), and 50 mM KCl], followed by the addition of 1 μM R:D or R-loop substrates containing 5′-FAM–labeled RNA in the reaction. The reactions were incubated at 37°C for indicated time and halted by addition of stop buffer [proteinase K (0.5 mg/ml), 5 mM EDTA, and 0.5% SDS] for 10 min at 37°C. Each sample was mixed with denaturing PAGE gel loading buffer (Thermo Fisher Scientific, AM8547) and was analyzed by 12% denaturing PAGE with 7 M urea in 0.5× TBE buffer. Gels were exposed at 600 OD in the Odyssey Fc imaging system (LICORbio) with Image Studio software and analyzed by ImageJ.</p></section><section id="sec27"><h3 class="pmc_sec_title">Xenograft experiments</h3>
<p>Five- to 6-week-old female BALB/c athymic nude mice of 20 to 25 g were purchased from the Jackson Laboratory. Mice were subcutaneously injected with 4 × 10<sup>6</sup> MCF7 (Y537S) cells into the dorsal flank and were randomly divided into four groups: control (saline), letrozole (20 mg/kg), CH3 (20 mg/kg), and combo (letrozole + CH3). When the tumor volume reached about 100 mm<sup>3</sup>, mice were administered intraperitoneally with indicated drugs every 2 days for 4 weeks. Tumor size and body weight were measured every 2 days. The relative tumor volume was calculated using the formula: <em>a</em> × (<em>b</em><sup>2</sup>)/2, for which <em>a</em> and <em>b</em> represent the longest and shortest diameters, respectively (<a href="#R78" class="usa-link" aria-describedby="R78"><em>78</em></a>). All the tissue and tumor samples were obtained 3 days after the final drug administration for subsequent evaluation. All animal studies were conducted in accordance with National Institutes of Health (NIH) regulation concerning the care and use of experimental animals and with the approval of the Institutional Animal Care and Use Committees of the George Washington University (Washington, DC).</p></section><section id="sec28"><h3 class="pmc_sec_title">Generation of AND-1 conditional KO and T819 knock in mice</h3>
<p>All animal studies were conducted in accordance with NIH regulations concerning the care and use of experimental animals and with the approval of the Institutional Animal Care and Use Committees of the George Washington University (Washington, DC).</p>
<p>All mice are on a C57BL/6J background. To generate <em>Wdhd1<sup>loxP/loxP</sup></em> mice, the left <em>loxP</em> and the right <em>loxP</em> were knocked into intron 7 and intron 8 of the <em>Wdhd1</em> gene using CRISPR-Cas9 technology (Applied StemCell). Correct insertion of the loxP sites was confirmed by PCR and DNA sequencing. Floxed founder mice were backcrossed to WT mice for at least five generations. All animals were weaned between 21 to 28 days old. Genotyping of <em>Wdhd1<sup>loxP/loxP</sup></em> mice was performed by PCR using tail snip DNA and primers listed in <a href="#T3" class="usa-link">Table 3</a>. The <em>Wdhd1<sup>loxP/loxP</sup></em> allele was detected by PCR amplification of the left <em>loxP</em> site with primers F1a and R1a, yielding the PCR product of 358 bp for the floxed allele (324 bp for WT). The right <em>loxP</em> site was amplified with primers F2a and R2a, yielding a 307-bp PCR product for the floxed allele (273 bp for WT). For the above PCRs, thermocycling conditions consisted of one step of 5 min at 94°C, 35 cycles of 10 s at 94°C, 20 s at 58°C, and 30 s at 72°C, followed by 5 min at 72°C.</p>
<section class="tw xbox font-sm" id="T3"><h4 class="obj_head">Table 3. PCR primer sequences for mice genotyping.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col width="%" span="1">
<col width="%" span="1">
<thead><tr>
<th valign="top" align="left" scope="col" rowspan="1" colspan="1">Name</th>
<th valign="top" align="center" scope="col" rowspan="1" colspan="1">Primer sequences</th>
</tr></thead>
<tbody>
<tr>
<td scope="row" valign="top" align="left" rowspan="1" colspan="1">F1a</td>
<td valign="top" align="center" rowspan="1" colspan="1">CTGAGCCCTGGTCCTTCGTAAAAGC</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">R1a</td>
<td valign="top" align="center" rowspan="1" colspan="1">TCAGCTATTAGCAAATATCATGTCA</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">F2a</td>
<td valign="top" align="center" rowspan="1" colspan="1">GAGGCATGCACFAAATAATGTTCTG</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">R2a</td>
<td valign="top" align="center" rowspan="1" colspan="1">TGTGACTCGGTAGCAAATATTCATG</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">F3a</td>
<td valign="top" align="center" rowspan="1" colspan="1">GGTAGTGCACCCAGACGACTTTCAT</td>
</tr>
<tr>
<td valign="top" align="left" scope="row" rowspan="1" colspan="1">R3a</td>
<td valign="top" align="center" rowspan="1" colspan="1">CTTCAACTTGACTTCTGACCCGTGG</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>The mouse T819 is equivalent to human T826, so we generated the <em>Wdhd1 <sup>T819A</sup></em> knock-in mouse line by changing Codon 819 from ACC&gt;GCA using CRISPR-Cas9 technology (Applied StemCell). Briefly, a mixture of guide RNA (gRNA; 5′-TTCTTTCTCTTCTTCTGACTGGG-3′), spCas9 mRNA, and single-stranded oligodeoxynucleotide donor was injected into zygotes collected from C57BL/6J mice. F0 founders were identified by PCR and confirmed by Sanger sequencing using F3a and R3a primers listed in <a href="#T3" class="usa-link">Table 3</a>. They were then crossed with C57Bl/6J WT mice to produce F1 pups.</p>
<p>MEFs were derived from WT (<em>Wdhd1<sup>WT</sup></em>), <em>Wdhd1<sup>loxP/loxP</sup></em>, and <em>Wdhd1<sup>T819A</sup></em> E13.5 embryos using previously described protocols (<a href="#R56" class="usa-link" aria-describedby="R56"><em>56</em></a>). <em>Wdhd1</em><sup>−/−</sup> MEF cells were obtained by infecting <em>Wdhd1<sup>loxP/loxP</sup></em> MEFs with adenovirus expressing cre recombinase (Ad-CMV-Cre; Vector Biolaboratories, no. 1045). Following infection, clones were isolated by limiting dilution or colony picking and screened by PCR for the excision of Exon 8 (<a href="#R79" class="usa-link" aria-describedby="R79"><em>79</em></a>).</p>
<p>Seven- to 8-week-old mice (<em>Wdhd1<sup>WT</sup></em> and <em>Wdhd1<sup>T819A</sup></em>) were intraperitoneally injected with APH once a week for 10 weeks to induce R-loops in vivo. Inguinal mammary glands were isolated and fixed in 8% formaldehyde solution for 36 to ~48 hours. Next, the fixing solution was removed, and the tissue was rinsed in a washing solution containing 30% PBS and 70% ethanol, and then stored at −20°C. The tissues were embedded in paraffin and sectioned at 3-μm thickness before IF staining of R-loop, and the relative fluorescence intensity was analyzed by ImageJ.</p></section><section id="sec29"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p>Data analysis and statistics were done using GraphPad Prism 10 software. Data are represented as the means ± SD; <em>P</em> values were calculated with two-way analysis of variance (ANOVA) or unpaired two-tailed Student’s <em>t</em> tests. *<em>P</em> &lt; 0.05 was considered statistically significant.</p></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We are grateful to J. Shao for providing the MCF7 mutant cell lines (Y537S, Y537N, and D538G). Our thanks also go to R. Fernandes for sharing the Spectramax i3x machine. We would like to extend our appreciation to C. Peng and M. Jameel Mughal for valuable support throughout this work.</p>
<section id="sec30"><p><strong>Funding:</strong> This work was partially supported by funding from the National Institutes of Health (CA288337, CA258357, and CA247684 to W.Z.; CA270153 to H.P.; and CA282303 to R.L.) and a grant from the McCormick Genomic and Proteomic Center.</p></section><section id="sec31"><p><strong>Author contributions:</strong> Conceptualization: W.Z., H.P., Z.L., and Y.P. Methodology: W.Z., Y.P., and C.L. Validation: Z.L., H.-C.C., and C.C. Software: C.C. Formal analysis: Y.P., C.C., and R.L. Investigation: Z.L., Y.P., H.-C.C., and R.L. Resources: W.Z., H.P., H.-C.C., Y.G., and R.L. Data curation: Z.L., Y.P., and C.C. Writing—original draft: W.Z. and Z.L. Writing—review and editing: W.Z., H.P., Z.L., Y.P., H.-C.C., Y.Z., M.J.M., C.L., and R.L. Visualization: W.Z., Z.L., Y.P., and C.C. Supervision: W.Z., H.P., and Z.L. Project administration: W.Z., H.P., and Z.L. Funding acquisition: W.Z., H.P., and R.L.</p></section><section id="sec32"><p><strong>Competing interests:</strong> The authors declare that they have no competing interests.</p></section><section id="sec33"><p><strong>Data and materials availability:</strong> All data needed to evaluate the conclusions in the paper are present in the paper and/or the Supplementary Materials.</p></section></section><section id="sec34"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section id="sec35"><h3 class="pmc_sec_title">This PDF file includes:</h3>
<section class="sm xbox font-sm" id="supplementary-material1"><div class="caption p">
<p>Figs. S1 to S7</p>
<p>Table S1</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12327453/bin/sciadv.adv2453_sm.pdf" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adv2453_sm.pdf</a><sup> (2.5MB, pdf) </sup>
</div></div></section></section></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">REFERENCES AND NOTES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R1">
<span class="label">1.</span><cite>Carles-Kinch K., Kreuzer K. N., 
RNA-DNA hybrid formation at a bacteriophage T4 replication origin. J. Mol. Biol.
266, 
915–926 (1997).
</cite> [<a href="https://doi.org/10.1006/jmbi.1996.0844" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9086270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=RNA-DNA%20hybrid%20formation%20at%20a%20bacteriophage%20T4%20replication%20origin&amp;volume=266&amp;publication_year=1997&amp;pages=915-926&amp;pmid=9086270&amp;doi=10.1006/jmbi.1996.0844&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R2">
<span class="label">2.</span><cite>Lee D. Y., Clayton D. A., 
Properties of a primer RNA-DNA hybrid at the mouse mitochondrial DNA leading-strand origin of replication. J. Biol. Chem.
271, 
24262–24269 (1996).
</cite> [<a href="https://doi.org/10.1074/jbc.271.39.24262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8798672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Properties%20of%20a%20primer%20RNA-DNA%20hybrid%20at%20the%20mouse%20mitochondrial%20DNA%20leading-strand%20origin%20of%20replication&amp;volume=271&amp;publication_year=1996&amp;pages=24262-24269&amp;pmid=8798672&amp;doi=10.1074/jbc.271.39.24262&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R3">
<span class="label">3.</span><cite>Xu B., Clayton D. A., 
RNA-DNA hybrid formation at the human mitochondrial heavy-strand origin ceases at replication start sites: An implication for RNA-DNA hybrids serving as primers. EMBO J.
15, 
3135–3143 (1996).
</cite> [<a href="/articles/PMC450256/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8670814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20J.&amp;title=RNA-DNA%20hybrid%20formation%20at%20the%20human%20mitochondrial%20heavy-strand%20origin%20ceases%20at%20replication%20start%20sites:%20An%20implication%20for%20RNA-DNA%20hybrids%20serving%20as%20primers&amp;volume=15&amp;publication_year=1996&amp;pages=3135-3143&amp;pmid=8670814&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R4">
<span class="label">4.</span><cite>Yu K., Chedin F., Hsieh C. L., Wilson T. E., Lieber M. R., 
R-loops at immunoglobulin class switch regions in the chromosomes of stimulated B cells. Nat. Immunol.
4, 
442–451 (2003).
</cite> [<a href="https://doi.org/10.1038/ni919" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12679812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Immunol.&amp;title=R-loops%20at%20immunoglobulin%20class%20switch%20regions%20in%20the%20chromosomes%20of%20stimulated%20B%20cells&amp;volume=4&amp;publication_year=2003&amp;pages=442-451&amp;pmid=12679812&amp;doi=10.1038/ni919&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R5">
<span class="label">5.</span><cite>Skourti-Stathaki K., Kamieniarz-Gdula K., Proudfoot N. J., 
R-loops induce repressive chromatin marks over mammalian gene terminators. Nature
516, 
436–439 (2014).
</cite> [<a href="https://doi.org/10.1038/nature13787" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4272244/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25296254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=R-loops%20induce%20repressive%20chromatin%20marks%20over%20mammalian%20gene%20terminators&amp;volume=516&amp;publication_year=2014&amp;pages=436-439&amp;pmid=25296254&amp;doi=10.1038/nature13787&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R6">
<span class="label">6.</span><cite>Liu S., Hua Y., Wang J., Li L., Yuan J., Zhang B., Wang Z., Ji J., Kong D., 
RNA polymerase III is required for the repair of DNA double-strand breaks by homologous recombination. Cell
184, 
1314–1329.e10 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.01.048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33626331/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=RNA%20polymerase%20III%20is%20required%20for%20the%20repair%20of%20DNA%20double-strand%20breaks%20by%20homologous%20recombination&amp;volume=184&amp;publication_year=2021&amp;pages=1314-1329.e10&amp;pmid=33626331&amp;doi=10.1016/j.cell.2021.01.048&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R7">
<span class="label">7.</span><cite>Michelini F., Pitchiaya S., Vitelli V., Sharma S., Gioia U., Pessina F., Cabrini M., Wang Y., Capozzo I., Iannelli F., Matti V., Francia S., Shivashankar G. V., Walter N. G., d’Adda di Fagagna F., 
Damage-induced lncRNAs control the DNA damage response through interaction with DDRNAs at individual double-strand breaks. Nat. Cell Biol.
19, 
1400–1411 (2017).
</cite> [<a href="https://doi.org/10.1038/ncb3643" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5714282/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29180822/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Cell%20Biol.&amp;title=Damage-induced%20lncRNAs%20control%20the%20DNA%20damage%20response%20through%20interaction%20with%20DDRNAs%20at%20individual%20double-strand%20breaks&amp;volume=19&amp;publication_year=2017&amp;pages=1400-1411&amp;pmid=29180822&amp;doi=10.1038/ncb3643&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R8">
<span class="label">8.</span><cite>Teng Y., Yadav T., Duan M., Tan J., Xiang Y., Gao B., Xu J., Liang Z., Liu Y., Nakajima S., Shi Y., Levine A. S., Zou L., Lan L., 
ROS-induced R loops trigger a transcription-coupled but BRCA1/2-independent homologous recombination pathway through CSB. Nat. Commun.
9, 
4115 (2018).
</cite> [<a href="https://doi.org/10.1038/s41467-018-06586-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6175878/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30297739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=ROS-induced%20R%20loops%20trigger%20a%20transcription-coupled%20but%20BRCA1/2-independent%20homologous%20recombination%20pathway%20through%20CSB&amp;volume=9&amp;publication_year=2018&amp;pages=4115&amp;pmid=30297739&amp;doi=10.1038/s41467-018-06586-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R9">
<span class="label">9.</span><cite>Marnef A., Legube G., 
R-loops as Janus-faced modulators of DNA repair. Nat. Cell Biol.
23, 
305–313 (2021).
</cite> [<a href="https://doi.org/10.1038/s41556-021-00663-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33837288/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Cell%20Biol.&amp;title=R-loops%20as%20Janus-faced%20modulators%20of%20DNA%20repair&amp;volume=23&amp;publication_year=2021&amp;pages=305-313&amp;pmid=33837288&amp;doi=10.1038/s41556-021-00663-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R10">
<span class="label">10.</span><cite>Lang K. S., Hall A. N., Merrikh C. N., Ragheb M., Tabakh H., Pollock A. J., Woodward J. J., Dreifus J. E., Merrikh H., 
Replication-transcription conflicts generate R-loops that orchestrate bacterial stress survival and pathogenesis. Cell
170, 
787–799.e18 (2017).
</cite> [<a href="https://doi.org/10.1016/j.cell.2017.07.044" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5630229/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28802046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Replication-transcription%20conflicts%20generate%20R-loops%20that%20orchestrate%20bacterial%20stress%20survival%20and%20pathogenesis&amp;volume=170&amp;publication_year=2017&amp;pages=787-799.e18&amp;pmid=28802046&amp;doi=10.1016/j.cell.2017.07.044&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R11">
<span class="label">11.</span><cite>Aguilera A., Garcia-Muse T., 
R loops: From transcription byproducts to threats to genome stability. Mol. Cell
46, 
115–124 (2012).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2012.04.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22541554/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=R%20loops:%20From%20transcription%20byproducts%20to%20threats%20to%20genome%20stability&amp;volume=46&amp;publication_year=2012&amp;pages=115-124&amp;pmid=22541554&amp;doi=10.1016/j.molcel.2012.04.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R12">
<span class="label">12.</span><cite>Crossley M. P., Bocek M., Cimprich K. A., 
R-Loops as cellular regulators and genomic threats. Mol. Cell
73, 
398–411 (2019).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2019.01.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6402819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30735654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=R-Loops%20as%20cellular%20regulators%20and%20genomic%20threats&amp;volume=73&amp;publication_year=2019&amp;pages=398-411&amp;pmid=30735654&amp;doi=10.1016/j.molcel.2019.01.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R13">
<span class="label">13.</span><cite>Brickner J. R., Garzon J. L., Cimprich K. A., 
Walking a tightrope: The complex balancing act of R-loops in genome stability. Mol. Cell
82, 
2267–2297 (2022).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2022.04.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9233011/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35508167/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Walking%20a%20tightrope:%20The%20complex%20balancing%20act%20of%20R-loops%20in%20genome%20stability&amp;volume=82&amp;publication_year=2022&amp;pages=2267-2297&amp;pmid=35508167&amp;doi=10.1016/j.molcel.2022.04.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R14">
<span class="label">14.</span><cite>Richard P., Manley J. L., 
R loops and links to human disease. J. Mol. Biol.
429, 
3168–3180 (2017).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2016.08.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5478472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27600412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=R%20loops%20and%20links%20to%20human%20disease&amp;volume=429&amp;publication_year=2017&amp;pages=3168-3180&amp;pmid=27600412&amp;doi=10.1016/j.jmb.2016.08.031&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R15">
<span class="label">15.</span><cite>Chiang H. C., Zhang X., Li J., Zhao X., Chen J., Wang H. T., Jatoi I., Brenner A., Hu Y., Li R., 
BRCA1-associated R-loop affects transcription and differentiation in breast luminal epithelial cells. Nucleic Acids Res.
47, 
5086–5099 (2019).
</cite> [<a href="https://doi.org/10.1093/nar/gkz262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6547407/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30982901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=BRCA1-associated%20R-loop%20affects%20transcription%20and%20differentiation%20in%20breast%20luminal%20epithelial%20cells&amp;volume=47&amp;publication_year=2019&amp;pages=5086-5099&amp;pmid=30982901&amp;doi=10.1093/nar/gkz262&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R16">
<span class="label">16.</span><cite>Garcia-Muse T., Aguilera A., 
R loops: From physiological to pathological roles. Cell
179, 
604–618 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.08.055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31607512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=R%20loops:%20From%20physiological%20to%20pathological%20roles&amp;volume=179&amp;publication_year=2019&amp;pages=604-618&amp;pmid=31607512&amp;doi=10.1016/j.cell.2019.08.055&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R17">
<span class="label">17.</span><cite>Garcia-Rubio M. L., Perez-Calero C., Barroso S. I., Tumini E., Herrera-Moyano E., Rosado I. V., Aguilera A., 
The fanconi anemia pathway protects genome integrity from R-loops. PLOS Genet.
11, 
e1005674 (2015).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1005674" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4652862/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26584049/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genet.&amp;title=The%20fanconi%20anemia%20pathway%20protects%20genome%20integrity%20from%20R-loops&amp;volume=11&amp;publication_year=2015&amp;pages=e1005674&amp;pmid=26584049&amp;doi=10.1371/journal.pgen.1005674&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R18">
<span class="label">18.</span><cite>Huertas P., Aguilera A., 
Cotranscriptionally formed DNA:RNA hybrids mediate transcription elongation impairment and transcription-associated recombination. Mol. Cell
12, 
711–721 (2003).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2003.08.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14527416/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Cotranscriptionally%20formed%20DNA:RNA%20hybrids%20mediate%20transcription%20elongation%20impairment%20and%20transcription-associated%20recombination&amp;volume=12&amp;publication_year=2003&amp;pages=711-721&amp;pmid=14527416&amp;doi=10.1016/j.molcel.2003.08.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R19">
<span class="label">19.</span><cite>Li X., Manley J. L., 
Inactivation of the SR protein splicing factor ASF/SF2 results in genomic instability. Cell
122, 
365–378 (2005).
</cite> [<a href="https://doi.org/10.1016/j.cell.2005.06.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16096057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Inactivation%20of%20the%20SR%20protein%20splicing%20factor%20ASF/SF2%20results%20in%20genomic%20instability&amp;volume=122&amp;publication_year=2005&amp;pages=365-378&amp;pmid=16096057&amp;doi=10.1016/j.cell.2005.06.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R20">
<span class="label">20.</span><cite>Paulsen R. D., Soni D. V., Wollman R., Hahn A. T., Yee M. C., Guan A., Hesley J. A., Miller S. C., Cromwell E. F., Solow-Cordero D. E., Meyer T., Cimprich K. A., 
A genome-wide siRNA screen reveals diverse cellular processes and pathways that mediate genome stability. Mol. Cell
35, 
228–239 (2009).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2009.06.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2772893/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19647519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=A%20genome-wide%20siRNA%20screen%20reveals%20diverse%20cellular%20processes%20and%20pathways%20that%20mediate%20genome%20stability&amp;volume=35&amp;publication_year=2009&amp;pages=228-239&amp;pmid=19647519&amp;doi=10.1016/j.molcel.2009.06.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R21">
<span class="label">21.</span><cite>Sollier J., Stork C. T., Garcia-Rubio M. L., Paulsen R. D., Aguilera A., Cimprich K. A., 
Transcription-coupled nucleotide excision repair factors promote R-loop-induced genome instability. Mol. Cell
56, 
777–785 (2014).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2014.10.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4272638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25435140/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Transcription-coupled%20nucleotide%20excision%20repair%20factors%20promote%20R-loop-induced%20genome%20instability&amp;volume=56&amp;publication_year=2014&amp;pages=777-785&amp;pmid=25435140&amp;doi=10.1016/j.molcel.2014.10.020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R22">
<span class="label">22.</span><cite>Chen J. Y., Zhang X., Fu X. D., Chen L., 
R-ChIP for genome-wide mapping of R-loops by using catalytically inactive RNASEH1. Nat. Protoc.
14, 
1661–1685 (2019).
</cite> [<a href="https://doi.org/10.1038/s41596-019-0154-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6604627/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30996261/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=R-ChIP%20for%20genome-wide%20mapping%20of%20R-loops%20by%20using%20catalytically%20inactive%20RNASEH1&amp;volume=14&amp;publication_year=2019&amp;pages=1661-1685&amp;pmid=30996261&amp;doi=10.1038/s41596-019-0154-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R23">
<span class="label">23.</span><cite>Lockhart A., Pires V. B., Bento F., Kellner V., Luke-Glaser S., Yakoub G., Ulrich H. D., Luke B., 
RNase H1 and H2 Are differentially regulated to process RNA-DNA hybrids. Cell Rep.
29, 
2890–2900.e5 (2019).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2019.10.108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31775053/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&amp;title=RNase%20H1%20and%20H2%20Are%20differentially%20regulated%20to%20process%20RNA-DNA%20hybrids&amp;volume=29&amp;publication_year=2019&amp;pages=2890-2900.e5&amp;pmid=31775053&amp;doi=10.1016/j.celrep.2019.10.108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R24">
<span class="label">24.</span><cite>Jenkyn-Bedford M., Jones M. L., Baris Y., Labib K. P. M., Cannone G., Yeeles J. T. P., Deegan T. D., 
A conserved mechanism for regulating replisome disassembly in eukaryotes. Nature
600, 
743–747 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-04145-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8695382/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34700328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A%20conserved%20mechanism%20for%20regulating%20replisome%20disassembly%20in%20eukaryotes&amp;volume=600&amp;publication_year=2021&amp;pages=743-747&amp;pmid=34700328&amp;doi=10.1038/s41586-021-04145-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R25">
<span class="label">25.</span><cite>Zhu W., Ukomadu C., Jha S., Senga T., Dhar S. K., Wohlschlegel J. A., Nutt L. K., Kornbluth S., Dutta A., 
Mcm10 and And-1/CTF4 recruit DNA polymerase alpha to chromatin for initiation of DNA replication. Genes Dev.
21, 
2288–2299 (2007).
</cite> [<a href="https://doi.org/10.1101/gad.1585607" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1973143/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17761813/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=Mcm10%20and%20And-1/CTF4%20recruit%20DNA%20polymerase%20alpha%20to%20chromatin%20for%20initiation%20of%20DNA%20replication&amp;volume=21&amp;publication_year=2007&amp;pages=2288-2299&amp;pmid=17761813&amp;doi=10.1101/gad.1585607&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R26">
<span class="label">26.</span><cite>Simon A. C., Zhou J. C., Perera R. L., van Deursen F., Evrin C., Ivanova M. E., Kilkenny M. L., Renault L., Kjaer S., Matak-Vinkovic D., Labib K., Costa A., Pellegrini L., 
A Ctf4 trimer couples the CMG helicase to DNA polymerase α in the eukaryotic replisome. Nature
510, 
293–297 (2014).
</cite> [<a href="https://doi.org/10.1038/nature13234" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4059944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24805245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A%20Ctf4%20trimer%20couples%20the%20CMG%20helicase%20to%20DNA%20polymerase%20%CE%B1%20in%20the%20eukaryotic%20replisome&amp;volume=510&amp;publication_year=2014&amp;pages=293-297&amp;pmid=24805245&amp;doi=10.1038/nature13234&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R27">
<span class="label">27.</span><cite>Samora C. P., Saksouk J., Goswami P., Wade B. O., Singleton M. R., Bates P. A., Lengronne A., Costa A., Uhlmann F., 
Ctf4 links DNA replication with sister chromatid cohesion establishment by recruiting the Chl1 helicase to the replisome. Mol. Cell
63, 
371–384 (2016).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2016.05.036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4980427/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27397686/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Ctf4%20links%20DNA%20replication%20with%20sister%20chromatid%20cohesion%20establishment%20by%20recruiting%20the%20Chl1%20helicase%20to%20the%20replisome&amp;volume=63&amp;publication_year=2016&amp;pages=371-384&amp;pmid=27397686&amp;doi=10.1016/j.molcel.2016.05.036&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R28">
<span class="label">28.</span><cite>Li Y., Li Z., Wu R., Han Z., Zhu W., 
And-1 is required for homologous recombination repair by regulating DNA end resection. Nucleic Acids Res.
45, 
2531–2545 (2017).
</cite> [<a href="https://doi.org/10.1093/nar/gkw1241" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5389477/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27940557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=And-1%20is%20required%20for%20homologous%20recombination%20repair%20by%20regulating%20DNA%20end%20resection&amp;volume=45&amp;publication_year=2017&amp;pages=2531-2545&amp;pmid=27940557&amp;doi=10.1093/nar/gkw1241&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R29">
<span class="label">29.</span><cite>Hao J., de Renty C., Li Y., Xiao H., Kemp M. G., Han Z., DePamphilis M. L., Zhu W., 
And-1 coordinates with Claspin for efficient Chk1 activation in response to replication stress. EMBO J.
34, 
2096–2110 (2015).
</cite> [<a href="https://doi.org/10.15252/embj.201488016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4551354/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26082189/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20J.&amp;title=And-1%20coordinates%20with%20Claspin%20for%20efficient%20Chk1%20activation%20in%20response%20to%20replication%20stress&amp;volume=34&amp;publication_year=2015&amp;pages=2096-2110&amp;pmid=26082189&amp;doi=10.15252/embj.201488016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R30">
<span class="label">30.</span><cite>Chang E. Y., Tsai S., Aristizabal M. J., Wells J. P., Coulombe Y., Busatto F. F., Chan Y. A., Kumar A., Dan Zhu Y., Wang A. Y., Fournier L. A., Hieter P., Kobor M. S., Masson J. Y., Stirling P. C., 
MRE11-RAD50-NBS1 promotes Fanconi Anemia R-loop suppression at transcription-replication conflicts. Nat. Commun.
10, 
4265 (2019).
</cite> [<a href="https://doi.org/10.1038/s41467-019-12271-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6753070/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31537797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=MRE11-RAD50-NBS1%20promotes%20Fanconi%20Anemia%20R-loop%20suppression%20at%20transcription-replication%20conflicts&amp;volume=10&amp;publication_year=2019&amp;pages=4265&amp;pmid=31537797&amp;doi=10.1038/s41467-019-12271-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R31">
<span class="label">31.</span><cite>Dowsett M., Jones A., Johnston S. R., Jacobs S., Trunet P., Smith I. E., 
In vivo measurement of aromatase inhibition by letrozole (CGS 20267) in postmenopausal patients with breast cancer. Clin. Cancer Res.
1, 
1511–1515 (1995).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/9815951/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&amp;title=In%20vivo%20measurement%20of%20aromatase%20inhibition%20by%20letrozole%20(CGS%2020267)%20in%20postmenopausal%20patients%20with%20breast%20cancer&amp;volume=1&amp;publication_year=1995&amp;pages=1511-1515&amp;pmid=9815951&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R32">
<span class="label">32.</span><cite>Dustin D., Gu G., Fuqua S. A. W., 
ESR1 mutations in breast cancer. Cancer
125, 
3714–3728 (2019).
</cite> [<a href="https://doi.org/10.1002/cncr.32345" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6788940/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31318440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer&amp;title=ESR1%20mutations%20in%20breast%20cancer&amp;volume=125&amp;publication_year=2019&amp;pages=3714-3728&amp;pmid=31318440&amp;doi=10.1002/cncr.32345&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R33">
<span class="label">33.</span><cite>Merenbakh-Lamin K., Ben-Baruch N., Yeheskel A., Dvir A., Soussan-Gutman L., Jeselsohn R., Yelensky R., Brown M., Miller V. A., Sarid D., Rizel S., Klein B., Rubinek T., Wolf I., 
D538G mutation in estrogen receptor-α: A novel mechanism for acquired endocrine resistance in breast cancer. Cancer Res.
73, 
6856–6864 (2013).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-13-1197" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24217577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&amp;title=D538G%20mutation%20in%20estrogen%20receptor-%CE%B1:%20A%20novel%20mechanism%20for%20acquired%20endocrine%20resistance%20in%20breast%20cancer&amp;volume=73&amp;publication_year=2013&amp;pages=6856-6864&amp;pmid=24217577&amp;doi=10.1158/0008-5472.CAN-13-1197&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R34">
<span class="label">34.</span><cite>Stork C. T., Bocek M., Crossley M. P., Sollier J., Sanz L. A., Chedin F., Swigut T., Cimprich K. A., 
Co-transcriptional R-loops are the main cause of estrogen-induced DNA damage. eLife
5, 
e17548 (2016).
</cite> [<a href="https://doi.org/10.7554/eLife.17548" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5030092/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27552054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Co-transcriptional%20R-loops%20are%20the%20main%20cause%20of%20estrogen-induced%20DNA%20damage&amp;volume=5&amp;publication_year=2016&amp;pages=e17548&amp;pmid=27552054&amp;doi=10.7554/eLife.17548&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R35">
<span class="label">35.</span><cite>Zhang X., Chiang H. C., Wang Y., Zhang C., Smith S., Zhao X., Nair S. J., Michalek J., Jatoi I., Lautner M., Oliver B., Wang H., Petit A., Soler T., Brunet J., Mateo F., Angel Pujana M., Poggi E., Chaldekas K., Isaacs C., Peshkin B. N., Ochoa O., Chedin F., Theoharis C., Sun L. Z., Curiel T. J., Elledge R., Jin V. X., Hu Y., Li R., 
Attenuation of RNA polymerase II pausing mitigates BRCA1-associated R-loop accumulation and tumorigenesis. Nat. Commun.
8, 
15908 (2017).
</cite> [<a href="https://doi.org/10.1038/ncomms15908" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5490191/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28649985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Attenuation%20of%20RNA%20polymerase%20II%20pausing%20mitigates%20BRCA1-associated%20R-loop%20accumulation%20and%20tumorigenesis&amp;volume=8&amp;publication_year=2017&amp;pages=15908&amp;pmid=28649985&amp;doi=10.1038/ncomms15908&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R36">
<span class="label">36.</span><cite>Chiang H. C., Qi L., Mitra P., Huang Y., Hu Y., Li R., 
R-loop functions in <em>Brca1</em>-associated mammary tumorigenesis. Proc. Natl. Acad. Sci. U.S.A.
121, 
e2403600121 (2024).
</cite> [<a href="https://doi.org/10.1073/pnas.2403600121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11331088/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39116124/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=R-loop%20functions%20in%20Brca1-associated%20mammary%20tumorigenesis&amp;volume=121&amp;publication_year=2024&amp;pages=e2403600121&amp;pmid=39116124&amp;doi=10.1073/pnas.2403600121&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R37">
<span class="label">37.</span><cite>Boguslawski S. J., Smith D. E., Michalak M. A., Mickelson K. E., Yehle C. O., Patterson W. L., Carrico R. J., 
Characterization of monoclonal antibody to DNA·RNA and its application to immunodetection of hybrids. J. Immunol. Methods
89, 
123–130 (1986).
</cite> [<a href="https://doi.org/10.1016/0022-1759(86)90040-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2422282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Immunol.%20Methods&amp;title=Characterization%20of%20monoclonal%20antibody%20to%20DNA%C2%B7RNA%20and%20its%20application%20to%20immunodetection%20of%20hybrids&amp;volume=89&amp;publication_year=1986&amp;pages=123-130&amp;pmid=2422282&amp;doi=10.1016/0022-1759(86)90040-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R38">
<span class="label">38.</span><cite>Smolka J. A., Sanz L. A., Hartono S. R., Chedin F., 
Recognition of RNA by the S9.6 antibody creates pervasive artifacts when imaging RNA:DNA hybrids. J. Cell Biol.
220, (2021).</cite> [<a href="https://doi.org/10.1083/jcb.202004079" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8040515/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33830170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Biol.&amp;title=Recognition%20of%20RNA%20by%20the%20S9.6%20antibody%20creates%20pervasive%20artifacts%20when%20imaging%20RNA:DNA%20hybrids&amp;volume=220&amp;publication_year=2021&amp;pmid=33830170&amp;doi=10.1083/jcb.202004079&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R39">
<span class="label">39.</span><cite>Ramirez P., Crouch R. J., Cheung V. G., Grunseich C., 
R-Loop Analysis by Dot-Blot. J. Vis. Exp., 10.3791/62069 (2021).</cite> [<a href="https://doi.org/10.3791/62069" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8506650/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33554969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Vis.%20Exp.&amp;title=R-Loop%20Analysis%20by%20Dot-Blot&amp;publication_year=2021&amp;pmid=33554969&amp;doi=10.3791/62069&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R40">
<span class="label">40.</span><cite>Wan Y., Zheng X., Chen H., Guo Y., Jiang H., He X., Zhu X., Zheng Y., 
Splicing function of mitotic regulators links R-loop-mediated DNA damage to tumor cell killing. J. Cell Biol.
209, 
235–246 (2015).
</cite> [<a href="https://doi.org/10.1083/jcb.201409073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4411280/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25918225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Cell%20Biol.&amp;title=Splicing%20function%20of%20mitotic%20regulators%20links%20R-loop-mediated%20DNA%20damage%20to%20tumor%20cell%20killing&amp;volume=209&amp;publication_year=2015&amp;pages=235-246&amp;pmid=25918225&amp;doi=10.1083/jcb.201409073&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R41">
<span class="label">41.</span><cite>Stirling P. C., Chan Y. A., Minaker S. W., Aristizabal M. J., Barrett I., Sipahimalani P., Kobor M. S., Hieter P., 
R-loop-mediated genome instability in mRNA cleavage and polyadenylation mutants. Genes Dev.
26, 
163–175 (2012).
</cite> [<a href="https://doi.org/10.1101/gad.179721.111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3273840/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22279048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=R-loop-mediated%20genome%20instability%20in%20mRNA%20cleavage%20and%20polyadenylation%20mutants&amp;volume=26&amp;publication_year=2012&amp;pages=163-175&amp;pmid=22279048&amp;doi=10.1101/gad.179721.111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R42">
<span class="label">42.</span><cite>Marabitti V., Lillo G., Malacaria E., Palermo V., Sanchez M., Pichierri P., Franchitto A., 
ATM pathway activation limits R-loop-associated genomic instability in Werner syndrome cells. Nucleic Acids Res.
47, 
3485–3502 (2019).
</cite> [<a href="https://doi.org/10.1093/nar/gkz025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6468170/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30657978/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=ATM%20pathway%20activation%20limits%20R-loop-associated%20genomic%20instability%20in%20Werner%20syndrome%20cells&amp;volume=47&amp;publication_year=2019&amp;pages=3485-3502&amp;pmid=30657978&amp;doi=10.1093/nar/gkz025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R43">
<span class="label">43.</span><cite>Saha S., Pommier Y., 
R-loops, type I topoisomerases and <em>cancer</em>. NAR Cancer
5, 
zcad013 (2023).
</cite> [<a href="https://doi.org/10.1093/narcan/zcad013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9984992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37600974/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=NAR%20Cancer&amp;title=R-loops,%20type%20I%20topoisomerases%20and%20cancer&amp;volume=5&amp;publication_year=2023&amp;pages=zcad013&amp;pmid=37600974&amp;doi=10.1093/narcan/zcad013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R44">
<span class="label">44.</span><cite>Alam M. S., 
Proximity ligation assay (PLA). Curr. Protoc. Immunol.
123, 
e58 (2018).
</cite> [<a href="https://doi.org/10.1002/cpim.58" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6205916/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30238640/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Protoc.%20Immunol.&amp;title=Proximity%20ligation%20assay%20(PLA)&amp;volume=123&amp;publication_year=2018&amp;pages=e58&amp;pmid=30238640&amp;doi=10.1002/cpim.58&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R45">
<span class="label">45.</span><cite>Wu H., Lima W. F., Crooke S. T., 
Investigating the structure of human RNase H1 by site-directed mutagenesis. J. Biol. Chem.
276, 
23547–23553 (2001).
</cite> [<a href="https://doi.org/10.1074/jbc.M009676200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11319219/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Investigating%20the%20structure%20of%20human%20RNase%20H1%20by%20site-directed%20mutagenesis&amp;volume=276&amp;publication_year=2001&amp;pages=23547-23553&amp;pmid=11319219&amp;doi=10.1074/jbc.M009676200&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R46">
<span class="label">46.</span><cite>Nguyen H. D., Yadav T., Giri S., Saez B., Graubert T. A., Zou L., 
Functions of replication protein A as a sensor of R Loops and a regulator of RNaseH1. Mol. Cell
65, 
832–847.e4 (2017).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2017.01.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5507214/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28257700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Functions%20of%20replication%20protein%20A%20as%20a%20sensor%20of%20R%20Loops%20and%20a%20regulator%20of%20RNaseH1&amp;volume=65&amp;publication_year=2017&amp;pages=832-847.e4&amp;pmid=28257700&amp;doi=10.1016/j.molcel.2017.01.029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R47">
<span class="label">47.</span><cite>Hellman L. M., Fried M. G., 
Electrophoretic mobility shift assay (EMSA) for detecting protein-nucleic acid interactions. Nat. Protoc.
2, 
1849–1861 (2007).
</cite> [<a href="https://doi.org/10.1038/nprot.2007.249" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2757439/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17703195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Electrophoretic%20mobility%20shift%20assay%20(EMSA)%20for%20detecting%20protein-nucleic%20acid%20interactions&amp;volume=2&amp;publication_year=2007&amp;pages=1849-1861&amp;pmid=17703195&amp;doi=10.1038/nprot.2007.249&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R48">
<span class="label">48.</span><cite>Grosschedl R., Giese K., Pagel J., 
HMG domain proteins: Architectural elements in the assembly of nucleoprotein structures. Trends Genet.
10, 
94–100 (1994).
</cite> [<a href="https://doi.org/10.1016/0168-9525(94)90232-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8178371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Genet.&amp;title=HMG%20domain%20proteins:%20Architectural%20elements%20in%20the%20assembly%20of%20nucleoprotein%20structures&amp;volume=10&amp;publication_year=1994&amp;pages=94-100&amp;pmid=8178371&amp;doi=10.1016/0168-9525(94)90232-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R49">
<span class="label">49.</span><cite>Zhang Y., Li J., Zhou Y., Li Z., Peng C., Pei H., Zhu W., 
And-1 coordinates with the FANCM complex to regulate fanconi anemia signaling and cisplatin resistance. Cancer Res.
82, 
3249–3262 (2022).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-22-0769" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9481708/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35867033/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res.&amp;title=And-1%20coordinates%20with%20the%20FANCM%20complex%20to%20regulate%20fanconi%20anemia%20signaling%20and%20cisplatin%20resistance&amp;volume=82&amp;publication_year=2022&amp;pages=3249-3262&amp;pmid=35867033&amp;doi=10.1158/0008-5472.CAN-22-0769&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R50">
<span class="label">50.</span><cite>Crossley M. P., Song C., Bocek M. J., Choi J. H., Kousouros J. N., Sathirachinda A., Lin C., Brickner J. R., Bai G., Lans H., Vermeulen W., Abu-Remaileh M., Cimprich K. A., 
Author correction: R-loop-derived cytoplasmic RNA-DNA hybrids activate an immune response. Nature
626, 
E6 (2024).
</cite> [<a href="https://doi.org/10.1038/s41586-024-07064-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38263519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Author%20correction:%20R-loop-derived%20cytoplasmic%20RNA-DNA%20hybrids%20activate%20an%20immune%20response&amp;volume=626&amp;publication_year=2024&amp;pages=E6&amp;pmid=38263519&amp;doi=10.1038/s41586-024-07064-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R51">
<span class="label">51.</span><cite>Wu K. K., 
Analysis of protein-DNA binding by streptavidin-agarose pulldown. Methods Mol. Biol.
338, 
281–290 (2006).
</cite> [<a href="https://doi.org/10.1385/1-59745-097-9:281" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16888365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol.&amp;title=Analysis%20of%20protein-DNA%20binding%20by%20streptavidin-agarose%20pulldown&amp;volume=338&amp;publication_year=2006&amp;pages=281-290&amp;pmid=16888365&amp;doi=10.1385/1-59745-097-9:281&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R52">
<span class="label">52.</span><cite>Rizzo J., Gifford L. K., Zhang X., Gewirtz A. M., Lu P., 
Chimeric RNA-DNA molecular beacon assay for ribonuclease H activity. Mol. Cell. Probes
16, 
277–283 (2002).
</cite> [<a href="https://doi.org/10.1006/mcpr.2002.0423" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12270269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Probes&amp;title=Chimeric%20RNA-DNA%20molecular%20beacon%20assay%20for%20ribonuclease%20H%20activity&amp;volume=16&amp;publication_year=2002&amp;pages=277-283&amp;pmid=12270269&amp;doi=10.1006/mcpr.2002.0423&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R53">
<span class="label">53.</span><cite>Wahba A. S., Esmaeili A., Damha M. J., Hudson R. H., 
A single-label phenylpyrrolocytidine provides a molecular beacon-like response reporting HIV-1 RT RNase H activity. Nucleic Acids Res.
38, 
1048–1056 (2010).
</cite> [<a href="https://doi.org/10.1093/nar/gkp1022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2817455/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19933258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=A%20single-label%20phenylpyrrolocytidine%20provides%20a%20molecular%20beacon-like%20response%20reporting%20HIV-1%20RT%20RNase%20H%20activity&amp;volume=38&amp;publication_year=2010&amp;pages=1048-1056&amp;pmid=19933258&amp;doi=10.1093/nar/gkp1022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R54">
<span class="label">54.</span><cite>Kaniowski D., Kulik K., Suwara J., Ebenryter-Olbinska K., Nawrot B., 
Boron clusters as enhancers of RNase H activity in the smart strategy of gene silencing by antisense oligonucleotides. Int. J. Mol. Sci.
23, 
12190 (2022).
</cite> [<a href="https://doi.org/10.3390/ijms232012190" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9603397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36293047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Boron%20clusters%20as%20enhancers%20of%20RNase%20H%20activity%20in%20the%20smart%20strategy%20of%20gene%20silencing%20by%20antisense%20oligonucleotides&amp;volume=23&amp;publication_year=2022&amp;pages=12190&amp;pmid=36293047&amp;doi=10.3390/ijms232012190&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R55">
<span class="label">55.</span><cite>Nowotny M., Gaidamakov S. A., Crouch R. J., Yang W., 
Crystal structures of RNase H bound to an RNA/DNA hybrid: Substrate specificity and metal-dependent catalysis. Cell
121, 
1005–1016 (2005).
</cite> [<a href="https://doi.org/10.1016/j.cell.2005.04.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15989951/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Crystal%20structures%20of%20RNase%20H%20bound%20to%20an%20RNA/DNA%20hybrid:%20Substrate%20specificity%20and%20metal-dependent%20catalysis&amp;volume=121&amp;publication_year=2005&amp;pages=1005-1016&amp;pmid=15989951&amp;doi=10.1016/j.cell.2005.04.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R56">
<span class="label">56.</span><cite>Hagos E. G., Ghaleb A. M., Dalton W. B., Bialkowska A. B., Yang V. W., 
Mouse embryonic fibroblasts null for the Krüppel-like factor 4 gene are genetically unstable. Oncogene
28, 
1197–1205 (2009).
</cite> [<a href="https://doi.org/10.1038/onc.2008.465" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2667867/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19137014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Mouse%20embryonic%20fibroblasts%20null%20for%20the%20Kr%C3%BCppel-like%20factor%204%20gene%20are%20genetically%20unstable&amp;volume=28&amp;publication_year=2009&amp;pages=1197-1205&amp;pmid=19137014&amp;doi=10.1038/onc.2008.465&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R57">
<span class="label">57.</span><cite>Hamperl S., Bocek M. J., Saldivar J. C., Swigut T., Cimprich K. A., 
Transcription-replication conflict orientation modulates R-Loop levels and activates distinct DNA damage responses. Cell
170, 
774–786.e19 (2017).
</cite> [<a href="https://doi.org/10.1016/j.cell.2017.07.043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5570545/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28802045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Transcription-replication%20conflict%20orientation%20modulates%20R-Loop%20levels%20and%20activates%20distinct%20DNA%20damage%20responses&amp;volume=170&amp;publication_year=2017&amp;pages=774-786.e19&amp;pmid=28802045&amp;doi=10.1016/j.cell.2017.07.043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R58">
<span class="label">58.</span><cite>Bhatia V., Barroso S. I., Garcia-Rubio M. L., Tumini E., Herrera-Moyano E., Aguilera A., 
BRCA2 prevents R-loop accumulation and associates with TREX-2 mRNA export factor PCID2. Nature
511, 
362–365 (2014).
</cite> [<a href="https://doi.org/10.1038/nature13374" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24896180/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=BRCA2%20prevents%20R-loop%20accumulation%20and%20associates%20with%20TREX-2%20mRNA%20export%20factor%20PCID2&amp;volume=511&amp;publication_year=2014&amp;pages=362-365&amp;pmid=24896180&amp;doi=10.1038/nature13374&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R59">
<span class="label">59.</span><cite>Crossley M. P., Bocek M. J., Hamperl S., Swigut T., Cimprich K. A., 
qDRIP: A method to quantitatively assess RNA-DNA hybrid formation genome-wide. Nucleic Acids Res.
48, 
e84 (2020).
</cite> [<a href="https://doi.org/10.1093/nar/gkaa500" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7641308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32544226/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=qDRIP:%20A%20method%20to%20quantitatively%20assess%20RNA-DNA%20hybrid%20formation%20genome-wide&amp;volume=48&amp;publication_year=2020&amp;pages=e84&amp;pmid=32544226&amp;doi=10.1093/nar/gkaa500&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R60">
<span class="label">60.</span><cite>Sanz L. A., Hartono S. R., Lim Y. W., Steyaert S., Rajpurkar A., Ginno P. A., Xu X., Chedin F., 
Prevalent, dynamic, and conserved R-loop structures associate with specific epigenomic signatures in mammals. Mol. Cell
63, 
167–178 (2016).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2016.05.032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4955522/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27373332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Prevalent,%20dynamic,%20and%20conserved%20R-loop%20structures%20associate%20with%20specific%20epigenomic%20signatures%20in%20mammals&amp;volume=63&amp;publication_year=2016&amp;pages=167-178&amp;pmid=27373332&amp;doi=10.1016/j.molcel.2016.05.032&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R61">
<span class="label">61.</span><cite>Li J., Zhang Y., Sun J., Chen L., Gou W., Chen C. W., Zhou Y., Li Z., Chan D. W., Huang R., Pei H., Zheng W., Li Y., Xia M., Zhu W., 
Discovery and characterization of potent And-1 inhibitors for cancer treatment. Clin. Transl. Med.
11, 
e627 (2021).
</cite> [<a href="https://doi.org/10.1002/ctm2.627" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8684776/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34923765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Transl.%20Med.&amp;title=Discovery%20and%20characterization%20of%20potent%20And-1%20inhibitors%20for%20cancer%20treatment&amp;volume=11&amp;publication_year=2021&amp;pages=e627&amp;pmid=34923765&amp;doi=10.1002/ctm2.627&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R62">
<span class="label">62.</span><cite>Wardell S. E., Ellis M. J., Alley H. M., Eisele K., VanArsdale T., Dann S. G., Arndt K. T., Primeau T., Griffin E., Shao J., Crowder R., Lai J. P., Norris J. D., McDonnell D. P., Li S., 
Efficacy of SERD/SERM hybrid-CDK4/6 inhibitor combinations in models of endocrine therapy-resistant breast cancer. Clin. Cancer Res.
21, 
5121–5130 (2015).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-15-0360" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4644714/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25991817/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&amp;title=Efficacy%20of%20SERD/SERM%20hybrid-CDK4/6%20inhibitor%20combinations%20in%20models%20of%20endocrine%20therapy-resistant%20breast%20cancer&amp;volume=21&amp;publication_year=2015&amp;pages=5121-5130&amp;pmid=25991817&amp;doi=10.1158/1078-0432.CCR-15-0360&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R63">
<span class="label">63.</span><cite>Bustin M., 
Regulation of DNA-dependent activities by the functional motifs of the high-mobility-group chromosomal proteins. Mol. Cell. Biol.
19, 
5237–5246 (1999).
</cite> [<a href="https://doi.org/10.1128/mcb.19.8.5237" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC84367/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10409715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Biol.&amp;title=Regulation%20of%20DNA-dependent%20activities%20by%20the%20functional%20motifs%20of%20the%20high-mobility-group%20chromosomal%20proteins&amp;volume=19&amp;publication_year=1999&amp;pages=5237-5246&amp;pmid=10409715&amp;doi=10.1128/mcb.19.8.5237&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R64">
<span class="label">64.</span><cite>Villa F., Simon A. C., Ortiz Bazan M. A., Kilkenny M. L., Wirthensohn D., Wightman M., Matak-Vinkovic D., Pellegrini L., Labib K., 
Ctf4 is a hub in the eukaryotic replisome that links multiple CIP-box proteins to the CMG helicase. Mol. Cell
63, 
385–396 (2016).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2016.06.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4980431/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27397685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Ctf4%20is%20a%20hub%20in%20the%20eukaryotic%20replisome%20that%20links%20multiple%20CIP-box%20proteins%20to%20the%20CMG%20helicase&amp;volume=63&amp;publication_year=2016&amp;pages=385-396&amp;pmid=27397685&amp;doi=10.1016/j.molcel.2016.06.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R65">
<span class="label">65.</span><cite>Kang Y. H., Farina A., Bermudez V. P., Tappin I., Du F., Galal W. C., Hurwitz J., 
Interaction between human Ctf4 and the Cdc45/Mcm2-7/GINS (CMG) replicative helicase. Proc. Natl. Acad. Sci. U.S.A.
110, 
19760–19765 (2013).
</cite> [<a href="https://doi.org/10.1073/pnas.1320202110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3856782/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24255107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Interaction%20between%20human%20Ctf4%20and%20the%20Cdc45/Mcm2-7/GINS%20(CMG)%20replicative%20helicase&amp;volume=110&amp;publication_year=2013&amp;pages=19760-19765&amp;pmid=24255107&amp;doi=10.1073/pnas.1320202110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R66">
<span class="label">66.</span><cite>Kilkenny M. L., Simon A. C., Mainwaring J., Wirthensohn D., Holzer S., Pellegrini L., 
The human CTF4-orthologue AND-1 interacts with DNA polymerase α/primase via its unique C-terminal HMG box. Open Biol.
7, 
170217 (2017).
</cite> [<a href="https://doi.org/10.1098/rsob.170217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5717350/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29167311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Open%20Biol.&amp;title=The%20human%20CTF4-orthologue%20AND-1%20interacts%20with%20DNA%20polymerase%20%CE%B1/primase%20via%20its%20unique%20C-terminal%20HMG%20box&amp;volume=7&amp;publication_year=2017&amp;pages=170217&amp;pmid=29167311&amp;doi=10.1098/rsob.170217&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R67">
<span class="label">67.</span><cite>Bermudez V. P., Farina A., Tappin I., Hurwitz J., 
Influence of the human cohesion establishment factor Ctf4/AND-1 on DNA replication. J. Biol. Chem.
285, 
9493–9505 (2010).
</cite> [<a href="https://doi.org/10.1074/jbc.M109.093609" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2843200/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20089864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Influence%20of%20the%20human%20cohesion%20establishment%20factor%20Ctf4/AND-1%20on%20DNA%20replication&amp;volume=285&amp;publication_year=2010&amp;pages=9493-9505&amp;pmid=20089864&amp;doi=10.1074/jbc.M109.093609&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R68">
<span class="label">68.</span><cite>Fabian C. J., 
The what, why and how of aromatase inhibitors: Hormonal agents for treatment and prevention of breast cancer. Int. J. Clin. Pract.
61, 
2051–2063 (2007).
</cite> [<a href="https://doi.org/10.1111/j.1742-1241.2007.01587.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2228389/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17892469/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Clin.%20Pract.&amp;title=The%20what,%20why%20and%20how%20of%20aromatase%20inhibitors:%20Hormonal%20agents%20for%20treatment%20and%20prevention%20of%20breast%20cancer&amp;volume=61&amp;publication_year=2007&amp;pages=2051-2063&amp;pmid=17892469&amp;doi=10.1111/j.1742-1241.2007.01587.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R69">
<span class="label">69.</span><cite>Herzog S. K., Fuqua S. A. W., 
ESR1 mutations and therapeutic resistance in metastatic breast cancer: Progress and remaining challenges. Br. J. Cancer
126, 
174–186 (2022).
</cite> [<a href="https://doi.org/10.1038/s41416-021-01564-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8770568/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34621045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Br.%20J.%20Cancer&amp;title=ESR1%20mutations%20and%20therapeutic%20resistance%20in%20metastatic%20breast%20cancer:%20Progress%20and%20remaining%20challenges&amp;volume=126&amp;publication_year=2022&amp;pages=174-186&amp;pmid=34621045&amp;doi=10.1038/s41416-021-01564-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R70">
<span class="label">70.</span><cite>Sato N., Koinuma J., Fujita M., Hosokawa M., Ito T., Tsuchiya E., Kondo S., Nakamura Y., Daigo Y., 
Activation of WD repeat and high-mobility group box DNA binding protein 1 in pulmonary and esophageal carcinogenesis. Clin. Cancer Res.
16, 
226–239 (2010).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-09-1405" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20028748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&amp;title=Activation%20of%20WD%20repeat%20and%20high-mobility%20group%20box%20DNA%20binding%20protein%201%20in%20pulmonary%20and%20esophageal%20carcinogenesis&amp;volume=16&amp;publication_year=2010&amp;pages=226-239&amp;pmid=20028748&amp;doi=10.1158/1078-0432.CCR-09-1405&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R71">
<span class="label">71.</span><cite>Biskobing D. M., 
Update on bazedoxifene: A novel selective estrogen receptor modulator. Clin. Interv. Aging
2, 
299–303 (2007).
</cite> [<a href="/articles/PMC2685267/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18044180/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Interv.%20Aging&amp;title=Update%20on%20bazedoxifene:%20A%20novel%20selective%20estrogen%20receptor%20modulator&amp;volume=2&amp;publication_year=2007&amp;pages=299-303&amp;pmid=18044180&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R72">
<span class="label">72.</span><cite>Li S., Shen D., Shao J., Crowder R., Liu W., Prat A., He X., Liu S., Hoog J., Lu C., Ding L., Griffith O. L., Miller C., Larson D., Fulton R. S., Harrison M., Mooney T., McMichael J. F., Luo J., Tao Y., Goncalves R., Schlosberg C., Hiken J. F., Saied L., Sanchez C., Giuntoli T., Bumb C., Cooper C., Kitchens R. T., Lin A., Phommaly C., Davies S. R., Zhang J., Kavuri M. S., McEachern D., Dong Y. Y., Ma C., Pluard T., Naughton M., Bose R., Suresh R., McDowell R., Michel L., Aft R., Gillanders W., DeSchryver K., Wilson R. K., Wang S., Mills G. B., Gonzalez-Angulo A., Edwards J. R., Maher C., Perou C. M., Mardis E. R., Ellis M. J., 
Endocrine-therapy-resistant ESR1 variants revealed by genomic characterization of breast-cancer-derived xenografts. Cell Rep.
4, 
1116–1130 (2013).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2013.08.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3881975/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24055055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep.&amp;title=Endocrine-therapy-resistant%20ESR1%20variants%20revealed%20by%20genomic%20characterization%20of%20breast-cancer-derived%20xenografts&amp;volume=4&amp;publication_year=2013&amp;pages=1116-1130&amp;pmid=24055055&amp;doi=10.1016/j.celrep.2013.08.022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R73">
<span class="label">73.</span><cite>Sanjana N. E., Shalem O., Zhang F., 
Improved vectors and genome-wide libraries for CRISPR screening. Nat. Methods
11, 
783–784 (2014).
</cite> [<a href="https://doi.org/10.1038/nmeth.3047" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4486245/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25075903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=Improved%20vectors%20and%20genome-wide%20libraries%20for%20CRISPR%20screening&amp;volume=11&amp;publication_year=2014&amp;pages=783-784&amp;pmid=25075903&amp;doi=10.1038/nmeth.3047&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R74">
<span class="label">74.</span><cite>Suzuki Y., Holmes J. B., Cerritelli S. M., Sakhuja K., Minczuk M., Holt I. J., Crouch R. J., 
An upstream open reading frame and the context of the two AUG codons affect the abundance of mitochondrial and nuclear RNase H1. Mol. Cell. Biol.
30, 
5123–5134 (2010).
</cite> [<a href="https://doi.org/10.1128/MCB.00619-10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2953059/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20823270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Biol.&amp;title=An%20upstream%20open%20reading%20frame%20and%20the%20context%20of%20the%20two%20AUG%20codons%20affect%20the%20abundance%20of%20mitochondrial%20and%20nuclear%20RNase%20H1&amp;volume=30&amp;publication_year=2010&amp;pages=5123-5134&amp;pmid=20823270&amp;doi=10.1128/MCB.00619-10&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R75">
<span class="label">75.</span><cite>Li Y., Song Y., Xu W., Li Q., Wang X., Li K., Wang J., Liu Z., Velychko S., Ye R., Xia Q., Wang L., Guo R., Dong X., Zheng Z., Dai Y., Li H., Yao M., Xue Y., Scholer H. R., Sun Q., Yao H., 
R-loops coordinate with SOX2 in regulating reprogramming to pluripotency. Sci. Adv.
6, 
eaba0777 (2020).
</cite> [<a href="https://doi.org/10.1126/sciadv.aba0777" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7360481/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32704541/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Adv.&amp;title=R-loops%20coordinate%20with%20SOX2%20in%20regulating%20reprogramming%20to%20pluripotency&amp;volume=6&amp;publication_year=2020&amp;pages=eaba0777&amp;pmid=32704541&amp;doi=10.1126/sciadv.aba0777&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R76">
<span class="label">76.</span><cite>Chedin F., Hartono S. R., Sanz L. A., Vanoosthuyse V., 
Best practices for the visualization, mapping, and manipulation of R-loops. EMBO J.
40, 
e106394 (2021).
</cite> [<a href="https://doi.org/10.15252/embj.2020106394" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7883053/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33411340/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20J.&amp;title=Best%20practices%20for%20the%20visualization,%20mapping,%20and%20manipulation%20of%20R-loops&amp;volume=40&amp;publication_year=2021&amp;pages=e106394&amp;pmid=33411340&amp;doi=10.15252/embj.2020106394&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R77">
<span class="label">77.</span><cite>Abraham K. J., Khosraviani N., Chan J. N. Y., Gorthi A., Samman A., Zhao D. Y., Wang M., Bokros M., Vidya E., Ostrowski L. A., Oshidari R., Pietrobon V., Patel P. S., Algouneh A., Singhania R., Liu Y., Yerlici V. T., De Carvalho D. D., Ohh M., Dickson B. C., Hakem R., Greenblatt J. F., Lee S., Bishop A. J. R., Mekhail K., 
Nucleolar RNA polymerase II drives ribosome biogenesis. Nature
585, 
298–302 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-2497-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7486236/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32669707/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Nucleolar%20RNA%20polymerase%20II%20drives%20ribosome%20biogenesis&amp;volume=585&amp;publication_year=2020&amp;pages=298-302&amp;pmid=32669707&amp;doi=10.1038/s41586-020-2497-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R78">
<span class="label">78.</span><cite>Li Z., Zhou W., Zhang Y., Sun W., Yung M. M. H., Sun J., Li J., Chen C. W., Li Z., Meng Y., Chai J., Zhou Y., Liu S. S., Cheung A. N. Y., Ngan H. Y. S., Chan D. W., Zheng W., Zhu W., 
ERK regulates HIF1α-mediated platinum resistance by directly targeting PHD2 in ovarian cancer. Clin. Cancer Res.
25, 
5947–5960 (2019).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-18-4145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7449248/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31285371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Cancer%20Res.&amp;title=ERK%20regulates%20HIF1%CE%B1-mediated%20platinum%20resistance%20by%20directly%20targeting%20PHD2%20in%20ovarian%20cancer&amp;volume=25&amp;publication_year=2019&amp;pages=5947-5960&amp;pmid=31285371&amp;doi=10.1158/1078-0432.CCR-18-4145&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R79">
<span class="label">79.</span><cite>Rago C., Vogelstein B., Bunz F., 
Genetic knockouts and knockins in human somatic cells. Nat. Protoc.
2, 
2734–2746 (2007).
</cite> [<a href="https://doi.org/10.1038/nprot.2007.408" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18007609/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Protoc.&amp;title=Genetic%20knockouts%20and%20knockins%20in%20human%20somatic%20cells&amp;volume=2&amp;publication_year=2007&amp;pages=2734-2746&amp;pmid=18007609&amp;doi=10.1038/nprot.2007.408&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<p>Figs. S1 to S7</p>
<p>Table S1</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12327453/bin/sciadv.adv2453_sm.pdf" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adv2453_sm.pdf</a><sup> (2.5MB, pdf) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Science Advances are provided here courtesy of <strong>American Association for the Advancement of Science</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1126/sciadv.adv2453"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/sciadv.adv2453.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12327453/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12327453/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12327453%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12327453/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12327453/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12327453/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40768568/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12327453/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40768568/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12327453/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12327453/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="vq48jxXQC3gf6rVFTSaYCxr6bLJRst5bFA0l4Lcmif01x42IvNRY4gW5Hnpb0miw">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
