
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Exploring the Microbial Peptides Derived from the Human Gut Microbiota to Regulate Class B GPCRS Using an In Silico Approach - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="67D713F689B742B31C13F60023B65358.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsomega">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332626/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Omega">
<meta name="citation_title" content="Exploring the Microbial Peptides Derived from the Human Gut Microbiota to Regulate Class B GPCRS Using an In Silico Approach">
<meta name="citation_author" content="Swagath S">
<meta name="citation_author_institution" content="Department of Chemistry, School of Advanced Sciences, Vellore Institute of Technology, Vellore 632014, Tamil Nadu, India">
<meta name="citation_author" content="Prateek Nayak">
<meta name="citation_author_institution" content="Department of Biosciences, School of Bioscience and Technology, Vellore Institute of Technology, Vellore 632014, Tamil Nadu, India">
<meta name="citation_author" content="Kuntal Pal">
<meta name="citation_author_institution" content="Department of Biosciences, School of Bioscience and Technology, Vellore Institute of Technology, Vellore 632014, Tamil Nadu, India">
<meta name="citation_publication_date" content="2025 Jul 23">
<meta name="citation_volume" content="10">
<meta name="citation_issue" content="30">
<meta name="citation_firstpage" content="33270">
<meta name="citation_doi" content="10.1021/acsomega.5c03268">
<meta name="citation_pmid" content="40787310">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332626/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332626/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332626/pdf/ao5c03268.pdf">
<meta name="description" content="Class B G-protein coupled receptors (GPCRs) are significant therapeutic recipients in cardiovascular, neurological, and metabolic diseases. The human gut microbiome is a complex microbial ecology recently identified as a possible source of bioactive ...">
<meta name="og:title" content="Exploring the Microbial Peptides Derived from the Human Gut Microbiota to Regulate Class B GPCRS Using an In Silico Approach">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Class B G-protein coupled receptors (GPCRs) are significant therapeutic recipients in cardiovascular, neurological, and metabolic diseases. The human gut microbiome is a complex microbial ecology recently identified as a possible source of bioactive ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332626/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12332626">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsomega.5c03268"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/ao5c03268.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332626%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12332626/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12332626/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332626/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsomega.png" alt="ACS Omega logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Omega" title="Link to ACS Omega" shape="default" href="https://pubs.acs.org/journal/acsodf" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Omega</button></div>. 2025 Jul 23;10(30):33270–33287. doi: <a href="https://doi.org/10.1021/acsomega.5c03268" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsomega.5c03268</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Omega%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Omega%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Exploring the Microbial
Peptides Derived from the
Human Gut Microbiota to Regulate Class B GPCRS Using an In Silico
Approach</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22S%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Swagath S</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Swagath S</span></h3>
<div class="p">
<sup>†</sup>
Department
of Chemistry, School of Advanced Sciences, Vellore Institute of Technology, Vellore
632014, Tamil Nadu, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22S%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Swagath S</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nayak%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Prateek Nayak</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Prateek Nayak</span></h3>
<div class="p">
<sup>‡</sup>
Department
of Biosciences, School of Bioscience and Technology, Vellore Institute of Technology, Vellore
632014, Tamil
Nadu, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nayak%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Prateek Nayak</span></a>
</div>
</div>
<sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pal%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Kuntal Pal</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Kuntal Pal</span></h3>
<div class="p">
<sup>‡</sup>
Department
of Biosciences, School of Bioscience and Technology, Vellore Institute of Technology, Vellore
632014, Tamil
Nadu, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pal%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kuntal Pal</span></a>
</div>
</div>
<sup>‡,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>†</sup>
Department
of Chemistry, School of Advanced Sciences, Vellore Institute of Technology, Vellore
632014, Tamil Nadu, India
</div>
<div id="aff2">
<sup>‡</sup>
Department
of Biosciences, School of Bioscience and Technology, Vellore Institute of Technology, Vellore
632014, Tamil
Nadu, India
</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
Email: <span>kuntal.pal@vit.ac.in</span>.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 10; Accepted 2025 Jun 12; Revised 2025 Jun 8; Collection date 2025 Aug 5.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY-NC-ND 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12332626  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40787310/" class="usa-link">40787310</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Class B G-protein coupled receptors (GPCRs) are significant
therapeutic
recipients in cardiovascular, neurological, and metabolic diseases.
The human gut microbiome is a complex microbial ecology recently identified
as a possible source of bioactive peptides that control host physiological
functions. Candidate peptides were found using advanced bioinformatics
tools including sequence homology analysis, structure modeling, and
molecular docking. These peptides were then evaluated for their binding
affinity and potential functional regulation of the GPCR activity.
Molecular dynamics simulations offered additional insights regarding
the stability and interaction diversity of peptide-receptor complexes,
highlighting receptor conformational state of G-protein interaction.
The findings identify unique microbial peptides capable of influencing
class B GPCR function, providing important insights into microbiome–host
interactions and therapeutic potential. This study emphasizes the
gut microbiome’s previously untapped potential as a source
of GPCR modulators, opening up new avenues for microbiome-driven therapy
approaches for metabolic and endocrine disorders.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a52/12332626/e6fc34caecd7/ao5c03268_0016.jpg" loading="lazy" id="abs1" height="560" width="424" alt="graphic file with name ao5c03268_0016.jpg"></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a52/12332626/e6fc34caecd7/ao5c03268_0014.jpg" loading="lazy" id="tgr1" height="560" width="424" alt="graphic file with name ao5c03268_0014.jpg"></p></section><section id="sec1"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>The human microflora can
be interpreted as the dynamic population
of microbes colonizing with their diverse set of genes and variable
products. Various physiological and multiomics
analyses supported by in-cell studies and animal experiments establish
the interlink of gut microbiome communities in maintaining body homeostasis. The dynamics of microbiome composition in the
gut were studied extensively to correlate with dysbiosis like nonalcoholic
fatty liver disease (NAFLD), cardiovascular disorders, obesity-linked
disorders like diabetes mellitus, and neurodegenerative disarrays
with immune modulation and inflammatory responses.<em>
−

</em> Microbial diversity
varies throughout the gastrointestinal tract depending on pH, oxygen
levels, and the presence of bile and mucus. The lower gut microbiome
community represented predominantly by anaerobic bacteria belongs
to species of Actinobacteria, Bacteroidetes, Firmicutes ( and ), Actinobacteria, Proteobacteria, and Verrucomicrobia (). Noninvasive
and invasive experimental techniques like magnetically controlled
sampling capsule endoscopy (MSCE) and surgical ileostomy of intestinal
effluents map the gut microbiome composition.<em>
,
</em> Over
the period, integrating the advanced techniques of metagenomics, metatranscriptomics,
and metabolomics has identified the highly dynamic microbiome composition
across the different populations of the world.<em>
,
</em> Further,
this integrative approach also focused on identifying the interplay
between different species that might result in viable biofilm formation. The microbiome’s contribution in generating
short-chain fatty acids (SCFAs) like butyrate, propionate and acetate
from complex carbohydrates in the colon has been studied for metabolite
contribution. These microbial metabolites
were found to have a supplemental role directly or indirectly in maintaining
homeostasis for energy metabolism, glucose-lipid metabolism, fortifying
immunity, and even in the prevention of cancer.<em>
−

</em>
</p>
<p>GPCRs
constitute the most targeted class of receptors for commercially
available drugs. The seven transmembrane helices are the signature
of these membrane proteins, responsible for relaying the signals through
the membrane. GPCRs usually transduce signaling through G-protein
and arrestin pathways that induce a subsequent intracellular signaling
cascade toward the effector molecule. Signaling by GPCRs is mediated
by four distinct types of G-proteins, and their signaling is modulated
mainly by two arrestins for the vast majority of GPCRs, with two other
arrestins exclusively modulating signaling in the visual system.<em>
,
</em> Heterotrimer G-proteins consist of α, β, and γ
domains and are mainly classified into Gαs, Gαq, Gαi,
and Gα1/2 based upon the type of signaling induced by the α-domain
while functioning by switching between two nucleotide (GDP/GTP) binding
states. The expression patterns of the
Gα protein vary among different tissues. Gαs and Gαq
show ubiquitous expression, while Gαt (transducing) is expressed
specifically in rod cells. While both GPCRs and G-proteins diffuse
freely through the membrane, receptors activated by natural ligand
or drug molecules promote G-protein binding on the cytoplasmic side,
where receptors catalyze the exchange of GTP for GDP of the Gα
domain. The GTP-bound Gα domain will undergo dissociation of
the heterotrimer, making the Gα-GTP domain free to bind and
initiate different downstream pathways. Subsequently, the free receptor
will be specifically phosphorylated at serine and threonine residues
of ICL3 and the cytoplasmic tail by G-protein receptor kinases (GRKs). The phosphorylation of the receptor is critical
to initiate the arrestin signaling pathways, which either leads to
receptor internalization or other pathways like MAPK. The GPCRs were
classified into five subfamilies based on sequence homology and molecular
architecture.</p>
<p>The class B group of the GPCR family is represented
by 15 membrane-bound
peptide hormone receptors important for various physiological functions.
It constitutes a subclass of therapeutically important receptors that
communicate for glucose and amino-acid metabolism, cardiovascular
and bone development and regeneration, along with immune responses. A detailed information about class B GPCR members
regulating different metabolic pathways and diseases is highlighted
in <a href="#fig1" class="usa-link">Figure 
</a>. Several
class B -GPCR extracellular domain (ECD) structures in complex with
peptide hormones were reported earlier. These structures provide important
insight into the α-β-β-α conformation of ECD
and also characterize their role in ligand specificity.<em>
,
</em> The structures of the antagonist-bound state of several full-length
receptors highlight the insights of the inactive conformation. In contrast, these structures explain the structural
plasticity of transmembrane helices associated with different states
of the receptor but also enlighten the role of biased agonism for
specific metabolic modulations. The ligand-bound
ternary complexes of various class-B GPCRs with G-proteins such as
secretin-bound secretin receptor–Gs complex (PDB: <a href="https://www.rcsb.org/structure/7D3S" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7D3S</a>), glucagon-bound glucagon receptor in the absence and presence
of Gs/i (PDB: <a href="https://www.rcsb.org/structure/5YQZ" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">5YQZ</a>; <a href="https://www.rcsb.org/structure/6WPW" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6WPW</a>),<em>
,
</em> Glp-bound GLP1-R-Gs (PDB: <a href="https://www.rcsb.org/structure/7LCJ" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7LCJ</a>), PTH analogue and PTHrP-bound
PTH1r-Gs complex (PDB: <a href="https://www.rcsb.org/structure/6NBI" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6NBI</a>; <a href="https://www.rcsb.org/structure/8HAF" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8HAF</a>),<em>
,
</em> CRF-bound CRFR1/2 in complex with Gs (PDB: <a href="https://www.rcsb.org/structure/6PB1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6PB1</a>; <a href="https://www.rcsb.org/structure/6PB0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6PB0</a>), and PACAP27-bound PAC1R-Gs (PDB: <a href="https://www.rcsb.org/structure/8E3X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">8E3X</a>) were able to resolve the signaling state of these receptors
upon activation.</p>
<figure class="fig xbox font-sm" id="fig1"><h3 class="obj_head">1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332626_ao5c03268_0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a52/12332626/bde552315e7f/ao5c03268_0001.jpg" loading="lazy" id="gr1" height="703" width="697" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phylogenetic sequence-based analysis of significant Class
B GPCRs
and depiction of their respective metabolic roles in human physiology.</p></figcaption></figure><p>There are 21 class B GPCR peptide hormones that
function as cognate
ligands for 15 receptors. Structures of GPCRs in complex with these
peptides revealed the novel binding pockets, which can be utilized
to develop peptide-based drugs targeting these sites. These peptides consist of 40–52 amino acids with
the designated role of C-termini in ECD binding and N-termini in receptor
activation. The biochemical studies with
GLP-1R, GIPR, PTH-R1, and CRF-R2, where receptor-tethered N-termini
of peptides show activation signals in Luciferase/cAMP assays, validate
the presence of activation-specific critical residues in the N-terminal
region of peptides.<em>
−

</em> Based on this hypothesis, we want to investigate
microbiome-derived bioactive peptides’ regulatory role with
similar N-terminal amino acids in targeting class B GPCR family members.
The role of microbiome-derived peptides as antimicrobials is well
studied, with characterized targets and molecular pathways. The microbiome data mining even identified the
newer peptides mimicking the activity of antibiotics in rodent models. However, these stable peptides’ role
in activating GPCRs has not been explored yet. Identifying their possible
role in regulating these receptors might pave the way for designing
naturally stable peptide-based drugs as agonists or antagonists. The
microbiome-derived peptides with their smaller size exhibit better
structural stability and may keep them intact while reaching targeted
sites
</p>
<p>Overall, the importance of
microbiome-derived metabolites and their
assimilation in human physiology has been extensively investigated.
However, GPCR activity modulation by microbial peptides having sequence
similarity to small peptide hormones is still unknown. Here, in this
in silico work, we investigated the binding properties of microbiome
peptides to respective class B GPCRs using docking and simulation
studies. Further, we were able to identify the molecular state of
the microbiome peptide-induced receptor’s closeness to G-protein
signaling.</p></section><section id="sec2"><h2 class="pmc_sec_title">2. Methods</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">2.1. Search for Microbiome-Derived Peptides Having
Receptor-Binding Residues</h3>
<p>In order to identify the microbiome
peptide having sequence similarity to the amino acid sequences of
native peptides, an NCBI BLAST search was performed against a gut
microbial peptide sequence database. The PepFunk database (<a href="https://shiny.imetalab.ca/pepFunk/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://shiny.imetalab.ca/pepFunk/</a>) was utilized for this purpose. Based on the literature review,
N-terminal amino acid residues of ligand peptides critical for binding
and receptor activation were noted. These residues were then used
as a reference for sequence matching during the search for relevant
peptide sequences.</p>
<p>From the matched peptide sequences, residues
located near the N-terminus were selected for further analysis.<em>
,
</em> In line with previous literature on GIP-bound GIPR (PDB ID: <a href="https://www.rcsb.org/structure/7RA3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7RA3</a>), Tyr1 and Ala2
at the N-terminus of Predencin (derived from the species name “”) were identified as
critical for binding with the GIP receptor. Similarly, Asp1 and Asn2
at the N-terminus of Paraexcin (derived from the species name “”) were found
to be essential for urocortin (UCN) binding with the CRF receptor
(PDB ID: <a href="https://www.rcsb.org/structure/6BP0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6BP0</a>). The first five N-terminal amino acids of the native peptides were
subsequently used for sequence BLAST searches against the peptide
database.</p></section><section id="sec2.2"><h3 class="pmc_sec_title">2.2. Retrieval of Target Receptor Structure Coordinates</h3>
<p>The PDB structures of the active receptors with ligand peptide
and G-protein occupied ternary complexes for gastric inhibitory polypeptide
(GIP)-bound glucose-dependent insulinotropic polypeptide receptor
(GIPR) (PDB ID: <a href="https://www.rcsb.org/structure/7RA3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7RA3</a>) and the Urocortin1-bound corticotropin-releasing factor 1 receptor
(PDB ID: <a href="https://www.rcsb.org/structure/6BP0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6BP0</a>) were retrieved from the protein data bank (pdb). The visualization
software Chimera was used to examine
the respective ligand binding position. The coordinates of these binding
sites were subsequently used for docking studies. The principal residues
critical for peptide binding and receptor stimulation were identified
from the literature and documented.</p></section><section id="sec2.3"><h3 class="pmc_sec_title">2.3. Modeling of Peptides</h3>
<p>The peptide
conformation was modeled using the PEP-FOLD server<em>
,
</em> available at PEP-FOLD3. Peptide structures were modelled using PEP-FOLD3
. Each peptide sequence was submitted with default settings, generating
up to 200 structural models. The top-ranked model based on sOPEP energy
scoring was selected for downstream analyses. The server algorithm
conducts a set of 100 simulations for a 15-residue peptide, with each
simulation exploring a distinct folds of the conformational space.
The output includes a compilation of all generated models along with
detailed clustering information and the top conformation from the
five best clusters. PEP-FOLD 3 allows users to bias 3D modeling by
specifying regions to preserve during simulations. This can be achieved
either by using a prior 3D structure and selecting regions to retain,
or by accepting the server’s internal optimal predictions for
specific residues. This functionality enables iterative modeling for
refining existing structures or generating alternative conformations
that explore different topological spaces.</p></section><section id="sec2.4"><h3 class="pmc_sec_title">2.4. Molecular Docking</h3>
<p>Protein-peptide
docking is a computer-aided process used to predict the interactions
between a protein and a peptide, playing a key role in elucidating
protein function, drug design, and various molecular biology applications.
The 3D structures of the target proteins, including the GIPR (PDB
ID: <a href="https://www.rcsb.org/structure/7RA3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">7RA3</a>) and
the CRFR1 (PDB ID: <a href="https://www.rcsb.org/structure/6PB0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6PB0</a>), were retrieved, while the peptide sequence was designed and its
3D structure generated using PEP-FOLD. Energy minimization was applied
to both protein and peptide structures to resolve steric clashes and
eliminate unfavorable interactions. Proper protonation states were
assigned based on the environmental pH using tools such as PROPKA
or PDB <a href="https://www.rcsb.org/structure/2PQR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">2PQR</a>.
Multiple peptide conformations were generated through conformational
sampling to explore different potential binding modes. Docking simulations
were conducted using MdockPep, an FFT-based flexible docking tool
optimized for peptide–protein complexes. The receptor was prepared
by removing nonessential water molecules and heteroatoms. Peptide
docking was performed with the default scoring function (ITScorePeP),
and 100 docking poses were generated for each peptide. The top-ranked
pose was selected based on binding score and visual inspection for
biological relevance.<em>
,
</em> The peptide was then docked to
the protein’s active or binding site to predict the interaction.</p>
<p>The docked complexes were assessed using the scoring functions
provided by the docking software, which generally indicate the binding
affinity between the protein and the peptide. Key interactions, including
hydrophobic contacts, hydrogen bonds and salt bridges, were analyzed
to understand the protein–peptide interaction. Docked poses
were clustered to determine the most likely binding mode. Visualization
tools such as PyMOL and Chimera were employed to examine the binding
conformations and interactions in greater detail. For consistency,
docking was performed using multiple software platforms to compare
the results. In our controlled study, the docking of many microbial
peptides with two amino acid identities at the N-termini did not dock
to the target receptor.</p></section><section id="sec2.5"><h3 class="pmc_sec_title">2.5. Molecular Simulation</h3>
<p>The receptor-peptide
complex having most favorable binding energy was subjected to a 300
ns molecular dynamics (MD) simulation using Gromacs 2024, with input files prepared via Charmm-Gui. Initially,
a 100 ns simulation was carried out for both the complexes to gain
an in-depth understanding of the stability of the interaction and
of the conformational shifts; a further 300 ns simulation was carried
out.<em>
,
</em> The ligand topologies were generated using
CGENFF in Charmm-Gui, and parameters were obtained through the Charmm-Gui
Force field Generation module.<em>
−

</em> The CHARMM36 force field was
used for the simulation, and the system was first reduced in vacuum
for 5000 steps using the steepest descent technique. The protein–ligand combination was then solvated
in a cubic periodic box with a 0.5 nm buffer using the Simple Point
Charge (SPC) water model. To neutralize the system, sodium (Na<sup>+</sup>) and chloride (Cl<sup>–</sup>) ions were introduced
at 0.15 M concentrations.<em>
,
</em> Energy minimization
was conducted using the steepest descent algorithm for up to 50,000
steps, with a convergence threshold set to a maximum force of 1000
kJ/mol/nm. The simulation, consisting of 125,000 steps, was conducted
under NPT conditions (constant number of pressure, temperature and
particles) with the system maintained at 1.0 bar and 303.15 K using
the NPT ensemble in the final run, applying positional restraints
to the heavy atoms of the complex. The production MD simulations were
run for 300 ns with a 2 fs integration time step, employing the V-rescale
thermostat to maintain a temperature of 300 K and the Parrinello–Rahman
barostat to regulate the pressure at 1 bar. Particle Mesh Ewald (PME)
computes long-distance electrostatic interactions, and covalent
bonds involving hydrogen atoms were constrained following the LINCS
algorithm. The lipid bilayer was made up of cholesterol, POPC (1-palmitoyl-2-oleoyl-<em>sn</em>-glycero-3-phosphocholine), and DOPC (1,2-dioleoyl-<em>sn</em>-glycero-3-phosphocholine) molecules in a 1:1:2 ratio,
symmetrically built to mirror the receptor’s native cellular
environment. The lipid bilayer consisted of cholesterol, POPC (1-palmitoyl-2-oleoyl-<em>sn</em>-glycero-3-phosphocholine), and DOPC (1,2-dioleoyl-<em>sn</em>-glycero-3-phosphocholine) molecules in a ratio of 1:1:2,
constructed symmetrically to match the receptor’s native cellular
environment.</p>
<p>Periodic boundary conditions were applied based
on the shape and size of the system, and approximately 3000 frames
were generated per simulation. Trajectory analysis was performed using
GROMACS, narrowing down on key structural metrics, such as root-mean-square
deviation (RMSD), root-mean-square fluctuation (RMSF), radius of gyration
(<em>R</em>
<sub>g</sub>), and hydrogen bonding patterns. The
RMSD of the protein backbone atoms was monitored to assess the system’s
overall stability throughout the simulation, while RMSF provided details
about the dynamicity of residues and domain. The radius of gyration
was calculated to evaluate the compactness of the protein structure.
Additionally, specific interactions between the peptide and GPCR were
examined by calculating hydrogen bonds, hydrophobic interactions,
and binding free energy using the MM-PBSA method. Further to investigate
the downstream signaling induced by microbial peptide, the intermittent
simulation frames of microbial peptide-bound receptors were aligned
with the G-protein-occupied conformation to mark the distance deviations
between key residues important for interaction with G-protein and
were analyzed over time.</p></section><section id="sec2.6"><h3 class="pmc_sec_title">2.6. Principal Component Analysis (PCA) and Free
Energy Landscape (FEL) of Receptor-Peptide Complex</h3>
<p>(PCA),
a multivariate statistical technique, was employed to investigate
the dominant molecular motions within molecular dynamics (MD) trajectories.
To ensure that only internal conformational dynamics were analyzed,
overall translational and rotational motions were first removed via
a least-squares fitting procedure relative to a reference structure.
Subsequently, a covariance matrix of atomic positional fluctuations
was constructed to capture the correlated atomic displacements. Diagonalization
of this matrix yielded a set of eigenvectors and corresponding eigenvalues,
where the eigenvectors define the principal directions of motion and
the eigenvalues quantify the magnitude or energetic contribution of
these motions. By projection of the trajectory onto selected eigenvectors,
the temporal evolution of the principal motions was analyzed. The
GROMACS tools g_covar and g_anaeig were utilized for the computation
and visualization of the covariance matrix, eigenvectors, and eigenvalues.
</p></section><section id="sec2.7"><h3 class="pmc_sec_title">2.7. MMPBSA Analysis</h3>
<p>MMPBSA (Molecular
Mechanics Poisson–Boltzmann Surface Area) analysis is a widely
used computational approach for estimating the binding free energy
between a receptor and a ligand. This method combines molecular mechanics
with continuum solvent models to provide a detailed understanding
of the molecular interactions that contribute to the stability of
the receptor–ligand complex. Here, MMPBSA was exercised to
quantify the binding affinity of microbial peptides to GPCR, providing
insights into the key interactions that govern binding and show similarity
with natural ligand interactions. To estimate the binding affinity
of the protein–peptide complex, we performed MMPBSA calculations
using the FastDRH server.
</p>
<p>This
study implemented a comprehensive in silico pipeline that integrates
peptide sequence screening through BLASTp and PepFunk database queries,
validation of microbial origin via taxonomic annotation, molecular
docking using MdockPep, and structural evaluation through molecular
dynamics simulations with GROMACS. This workflow facilitates the systematic
identification and characterization of gut microbiota-derived peptides
with the potential to interact with host receptors. By combining sequence
homology assessment, physicochemical property profiling, and dynamic
structural analysis, the approach offers a robust framework for prioritizing
candidate bioactive peptides for functional investigation.</p></section></section><section id="sec3"><h2 class="pmc_sec_title">3. Results</h2>
<section id="sec3.1"><h3 class="pmc_sec_title">3.1. Screening of Microbiome-Derived Peptides for
Peptidomimetics</h3>
<p>Through the application of our integrative
computational workflow, we identified a subset of microbiota-derived
peptides exhibiting high sequence similarity to human endogenous peptides
along with favorable physicochemical characteristics. Several of these
candidates demonstrated strong binding affinities to the target receptor
in molecular docking studies and sustained stable interactions throughout
the 300 ns molecular dynamics simulations. These results support the
hypothesis that gut microbial peptides may modulate host signaling
pathways, highlighting their potential to function as molecular mimics
of endogenous ligands.</p>
<p>Among 2,34,852 peptides derived from
Pepfunk, two peptides were identified for class B GPCRs using Homology
sequence alignment with matching key residues at the N-terminal region
and occupying the same binding pocket as the native GPCR peptide.</p></section><section id="sec3.2"><h3 class="pmc_sec_title">3.2. Molecular Docking of Screened Potential Microbial
Peptides</h3>
<p>The docking analysis reported a number of possible
binding configurations in <a href="#fig2" class="usa-link">Figure 
</a>a, where the top-ranked confirmation displayed key
hydrogen bonds and hydrophobic contacts. In <a href="#fig2" class="usa-link">Figure 
</a>b, the peptide was observed to interact with
Arg308, His364, Val311, Tyr239, and Ile235 of the GIP receptor, which
is known to be crucial to initiate the activation of the transmembrane
section of the receptor. The outputs
derived from the docking study implied a potential mechanism of activation.</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332626_ao5c03268_0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a52/12332626/66fd4aad0cb1/ao5c03268_0002.jpg" loading="lazy" id="gr2" height="595" width="667" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Figure
illustrates the docking interaction between the GIPR and
Predencin. (a) The peptide is shown binding to the active site of
the receptor. (b) The key binding residues are highlighted within
the ligand binding pocket (deep bluemicrobial peptide-bound
GIPR; pinkmicrobial peptide Predencin). (c) The peptide alignment
between Predencin and Endogenous peptide ligand is shown in a stick
representation, while the protein is depicted in a cartoon for clarity
(Cyangastric inhibitory polypeptide). (d) Predicted hydrogen
bonds and hydrophobic interactions are indicated, providing insights
into the stabilization and orientation of the peptide within the binding
pocket. Docking scores and binding affinities were calculated using
MdockPep, validating the interaction specificity and affinity of the
complex.</p></figcaption></figure><p>This could provide a foundation for the design
of peptide-based
inhibitors or activators. The identified binding site could serve
as a target for therapeutic intervention, drug design, or biomolecular
engineering. The docking results were consistent with those of previous
studies, validating the predicted binding mode. This result adds to
the demonstration of the utility of docking approaches for the prediction
of protein–peptide interactions in homologous systems. Nonbonded
interactions were found to be formed between the N-terminal Tyr1 of
Predencin and Arg308, His361, Val311, Tyr239, and Ile236 residues
of the receptor. The docking results in <a href="#fig2" class="usa-link">Figure 
</a>d revealed that Paraexcin binds to CRF1R
with a high affinity, as indicated by a low binding energy. In <a href="#fig2" class="usa-link">Figure 
</a>c, the similarities
between the microbial peptide and endogenous ligand in terms of binding
are shown. This score suggests strong interactions between the peptide
and the protein at the binding interface, implying a stable complex
formation. The top-scoring binding poses were ranked according to
their docking energy, with pose 1 showing the highest affinity.</p>
<p>In <a href="#fig3" class="usa-link">Figure 
</a>a, a
detailed examination of the CRF1R-peptide interactions showed multiple
hydrogen bonds, hydrophobic interactions, and van der Waals forces
contributing for stability of the complex. Notably, in <a href="#fig3" class="usa-link">Figure 
</a>d, key residues such as H198,
L351, K260, and Y195 were involved in hydrogen bonding with the peptide,
while other residues provided hydrophobic support, stabilizing the
interaction further. As shown in <a href="#fig3" class="usa-link">Figure 
</a>c, the natural peptide binding site of the
receptor demonstrated a complementary fit with the microbiome-derived
peptide, allowing for optimal positioning of the peptide side chains.</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332626_ao5c03268_0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a52/12332626/165c22777420/ao5c03268_0003.jpg" loading="lazy" id="gr3" height="530" width="683" alt="3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>This figure
displays the docking analysis of the CRF1R–Paraexcin
complex. (a) The peptide is shown binding to the active site of the
receptor (Redmicrobial peptide bound to CRF1R; pale bluemicrobial
peptide Paraexcin). (b) The more focused representation of the peptide
occupying the same binding pocket as the endogenous ligand is shown.
The protein structure is represented in cartoon form (c) while the
peptide alignment of Paraexcin and Corticoliberin is depicted in stick
representation (brownendogenous Corticoliberin). The docking
results were obtained using MdockPep, and the calculated binding affinity
indicates a strong interaction between the protein and peptide. (d)
Critical residues involved in binding are labeled for clarity.</p></figcaption></figure><p>As shown in <a href="#fig3" class="usa-link">Figure 
</a>b, the peptide was observed to adopt an extended conformation,
interacting
deeply within the binding pocket of the protein. Critical residues
from the peptide, including Asn6, Phe7, Ser8, and Ala9, were found
to penetrate the core of the binding site, forming several key interactions
with the protein’s active site residues. The binding mode suggests
that these interactions play a vital role in the specificity and stabilization
of the complex.</p></section><section id="sec3.3"><h3 class="pmc_sec_title">3.3. Receptor Dynamics and Stability of PREDENCIN
Bound to GIPR</h3>
<p>The RMSD analysis interpreted that the complex
attained a stable state of equilibrium after 300 ns, with the complex
maintaining a stable conformation throughout the simulation. The RMSF
profiles revealed that loop regions and binding sites exhibited higher
flexibility, consistent with their functional roles in ligand binding.
The MD simulations provided crucial insights regarding the conformational
changes in the loop regions during ligand recognition, followed by
receptor activation and the dynamic properties of these interactions.
The analysis of hydrogen bonds and hydrophobic contacts unveiled that
certain interactions were stable throughout the simulation, while
others were transient (<a href="#fig5" class="usa-link">Figure 
</a>b,d). These findings suggest that TYR1 and
ALA2 of Predencin played a crucial role in the stability and confirming
affinity between the GIPR–peptide complex. Similarly, in CRF1R,
the GLN6 and PRO7 residues have a significant influence on the binding
and stimulation of the receptor. The RMSD of the Predencin–GIPR
complex shows that the complex gets stable after 120 ns (<a href="#fig4" class="usa-link">Figure 
</a>c).</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332626_ao5c03268_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a52/12332626/723d5a01102f/ao5c03268_0005.jpg" loading="lazy" id="gr5" height="671" width="750" alt="5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Surface area analysis and hydrogen bond interaction of
the GIPR
and Predencin complex model. (a) Surface area of the protein in the
protein–peptide complex, illustrating the accessible surface
regions involved in peptide binding and potential interaction sites.
(b) Representation of hydrogen bond interactions between aligned molecular
structures, showcasing the bonds as dotted lines between hydrogen
acceptor and donor atoms, illustrating the electrostatic attraction
that stabilizes the molecular configuration. (c) Solvent-accessible
surface area (SASA) plot of the receptor–peptide complex over
the simulation time, indicating changes in surface exposure and potential
shifts in protein–peptide interactions during the MD simulation.
(d) Hydrogen bond plot of the protein–peptide complex over
the simulation time, showing the number of hydrogen bonds involved
between the protein and peptide, providing insights into interaction
stability throughout the MD simulation.</p></figcaption></figure><figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332626_ao5c03268_0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a52/12332626/57f10c625181/ao5c03268_0004.jpg" loading="lazy" id="gr4" height="670" width="750" alt="4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Stability and dynamicity
of the GIPR and Predencin complex model.
(a) Surface mapping of Predencin within the protein–peptide
complex, highlighting the interaction interfaces and accessible surface
areas critical for binding and stability [deep bluePredencin-bound
GIPR; Pale blueG-Protein Coupled GIPR (7RA3)]. (b) Vacuum
electrostatic potential (VEP) map of the GIPR–peptide complex,
illustrating the distribution of electrostatic charges on the molecular
surface, revealing key interaction regions between the protein and
peptide. (c) Root-mean-square deviation (RMSD) of Predencin-bound
GIPR over a 300 ns simulation time, indicating structural stability
and conformational changes during the MD simulation. (d) Root-mean-square
fluctuation (RMSF) of Predencin-bound GIPR complex residues, showing
the flexibility and mobility of individual residues throughout the
MD simulation.</p></figcaption></figure><p>
<a href="#fig4" class="usa-link">Figure 
</a>b,d shows
the vacuum electrostatic potential of the receptor and root-mean-square
fluctuations of the receptor residues during the conformational shift.
The SASA plot reveals the binding pocket area available for the receptor–peptide
interaction (<a href="#fig5" class="usa-link">Figure 
</a>c). The data obtained from the MD simulations implied a potential
mechanism of receptor activation and offered a template for the design
of peptide-based drugs. The dynamic behavior observed in the orthosteric
binding site could be targeted in future drug design efforts to enhance
binding affinity and efficacy (<a href="#fig4" class="usa-link">Figures 
</a>a and a).</p>
<p>The root-mean-square
deviation (RMSD) of the CRF1R–peptide
complex was calculated over the course of a 300 ns molecular dynamics
(MD) simulation to assess the structural stability. The RMSD plot
(<a href="#fig6" class="usa-link">Figure 
</a>c) shows an
initial equilibration phase, followed by a stable plateau, suggesting
that the complex maintained a consistent conformation after the equilibration
phase. The average RMSD value of the system remained within a reasonable
range (1–2 Å), indicating that the protein and peptide
experienced no major conformational shifts during the simulation,
maintaining a stable interaction. The surface area of the peptide
that occupies the binding pocket reveals key information about the
binding and stability of the complex (<a href="#fig6" class="usa-link">Figure 
</a>a). <a href="#fig6" class="usa-link">Figure 
</a>b shows how the electrostatic charge is distributed
across the receptor, highlighting regions of positive and negative
potential that are crucial for the binding.</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332626_ao5c03268_0006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a52/12332626/634dd6575ad3/ao5c03268_0006.jpg" loading="lazy" id="gr6" height="664" width="750" alt="6"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Stability and dynamicity
of the CRF1R and Paraexcin complex model.
(a) Surface mapping of Paraexcin in the protein–peptide complex,
showcasing the interaction sites and accessible surface areas essential
for binding and stability. RedParaexcin-bound CRF1R; light
brown: G-Protein-coupled CRF1R (6PB0). (b) Vacuum electrostatic potential
(VEP) map of the CRF1R–peptide complex, depicting the distribution
of electrostatic charges on the molecular surface and identifying
key regions involved in protein–peptide interactions. (c) Root-mean-square
deviation (RMSD) of the Paraexcin-bound CRF1R over a 300 ns simulation,
reflecting structural stability and conformational shifts during the
MD simulation. (d) Root-mean-square fluctuation (RMSF) of the CRF1R–Paraexcin
complex, indicating the flexibility and mobility of individual residues
throughout the MD simulation.</p></figcaption></figure><p>To further investigate the dynamics of individual
residues, the
root-mean-square fluctuation was computed for the CRF1R–peptide
complex (<a href="#fig6" class="usa-link">Figure 
</a>d).
The RMSF analysis revealed regions of both high and low flexibility,
with residues located in the N-terminal region and the transmembrane
region, showing the least fluctuation, indicating their involvement
in stable peptide binding. In contrast, residues in flexible loops
and surface-exposed regions displayed higher RMSF values, suggesting
increased mobility. The peptide exhibited lower fluctuations in the
binding interface, further supporting the stable interaction between
the protein and peptide. The SASA plot exhibited the binding area
for the complex interaction (<a href="#fig7" class="usa-link">Figure 
</a>c).</p>
<figure class="fig xbox font-sm" id="fig7"><h4 class="obj_head">7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332626_ao5c03268_0007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a52/12332626/7c108f802ea3/ao5c03268_0007.jpg" loading="lazy" id="gr7" height="671" width="750" alt="7"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The surface area analysis and hydrogen bond interaction
of the
CRF1R and Paraexcin complex model. (a) Surface area of the protein
in the protein–peptide complex, illustrating the accessible
surface regions involved in peptide binding and potential interaction
sites. (b) Representation of hydrogen bond interactions between aligned
molecular structures, showcasing the bonds as dotted lines between
hydrogen acceptor and donor atoms, illustrating the electrostatic
attraction that stabilizes the molecular configuration. (c) Solvent-accessible
surface area (SASA) plot of the receptor–peptide complex over
the simulation time, indicating changes in surface exposure and potential
shifts in protein–peptide interactions during the MD simulation.
(d) Hydrogen bond plot of the protein–peptide complex over
the simulation time, showing the number of hydrogen bonds formed between
the protein and peptide, providing insights into interaction stability
throughout the MD simulation.</p></figcaption></figure><p>Surface mapping of the protein–peptide complex
was performed
to identify the interaction interfaces and accessible surface areas
that are critical for binding. The analysis (<a href="#fig7" class="usa-link">Figure 
</a>a) revealed that Paraexcin primarily interacts
with E209, H198, L351, K260, and Y195, forming stable contacts within
the binding pocket of the protein. Hydrophobic and electrostatic interactions
appeared to dominate the binding interface, with key residues facilitating
these contacts (<a href="#fig7" class="usa-link">Figure 
</a>b,d).</p></section><section id="sec3.4"><h3 class="pmc_sec_title">3.4. Principal Component Analysis</h3>
<p>To elucidate
the predominant collective motions of the system throughout the simulation,
principal component analysis (PCA) was conducted on the 300 ns molecular
dynamics trajectory. Prior to the analysis, the trajectory frames
were structurally aligned to a reference conformation using least-squares
fitting, effectively removing global translational and rotational
movements and thereby isolating intrinsic conformational dynamics. A covariance matrix was then constructed from
the Cartesian positional fluctuations of the backbone atoms, and its
subsequent diagonalization yielded a set of eigenvectors (principal
components, PCs) along with their associated eigenvalues, which represent
the directions and relative amplitudes of atomic motions, respectively.
The first ten principal components captured over 85% of the total
variance in atomic displacement, with PC1 and PC2 alone accounting
for approximately 56.2% of the variance (PC1:34.5%, PC2:21.7%). This
concentration of variance within the first few PCs indicates that
the majority of significant conformational transitions can be effectively
described within a low-dimensional subspace. Projection of the MD
trajectory onto the PC1–PC2 plane (<a href="#fig8" class="usa-link">Figures 
</a>a and a) revealed
distinct clustering of conformational states, indicative of transitions
between discrete metastable states sampled during the simulation.
These observations suggest the presence of conformational substates
relevant to the functional dynamics of the biomolecule. Furthermore, temporal analysis of the trajectory
projected onto PC1 (<a href="#fig8" class="usa-link">Figures 
</a>b and b) demonstrated a gradual shift
in conformational space over time, consistent with large-scale structural
transitions occurring during the simulation. Structural analysis of
conformers located at the extrema of PC1 and PC2 revealed that the
principal motions are predominantly associated with outward displacement
of transmembrane helices and enhanced loop flexibility. Such motions
are mechanistically significant and may correspond to key transitions
involved in receptor activation, ligand modulation, or allosteric
regulation.</p>
<figure class="fig xbox font-sm" id="fig8"><h4 class="obj_head">8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332626_ao5c03268_0008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a52/12332626/a76a1e165e4d/ao5c03268_0008.jpg" loading="lazy" id="gr8" height="576" width="750" alt="8"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>PCA and FEL analysis for GIPR-Predencin MD trajectories. (a) Two-dimensional
projection of the MD trajectory onto the first two principal components
(PC1 and PC2), illustrating the conformational space sampled during
the simulation. The density and spread of data points reflect the
frequency and distribution of structural fluctuations along these
dominant modes. (b) Scree plot depicting the variance explained by
individual principal components. A significant portion of the overall
motion is captured by the first few components, with PC1 and PC2 accounting
for the largest collective movements. (c) Two-dimensional free energy
landscape (FEL) plotted as a function of PC1 and PC2, identifying
thermodynamically favorable conformational states (energy minima)
and the corresponding energy barriers separating them. (d) Three-dimensional
volcano representation of the FEL, where PC1 and PC2 define the conformational
coordinates and the vertical axis corresponds to the relative free
energy (kcal/mol), providing a detailed view of the energetic landscape
and the depth of metastable basins.</p></figcaption></figure><figure class="fig xbox font-sm" id="fig9"><h4 class="obj_head">9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332626_ao5c03268_0009.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a52/12332626/701d7b9a01eb/ao5c03268_0009.jpg" loading="lazy" id="gr9" height="576" width="750" alt="9"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>PCA and FEL analysis for CR1R–Paraexcin MD trajectories
(a) Two-dimensional projection of the MD trajectory onto the first
two principal components (PC1 and PC2). The density and distribution
of points reflect the frequency of conformational sampling across
the simulation. (b) The scree plot demonstrates that the first few
principal components capture the majority of the system’s motion,
with PC1 and PC2 accounting for the most significant collective dynamics.
(c) Two-dimensional free energy landscape mapped over the first two
principal components (PC1 and PC2). Energy minima represent thermodynamically
favorable conformations sampled during the simulation, while the energy
barriers suggest transitions between dominant states. (d) Three-dimensional
volcano plot of the free energy landscape constructed using principal
components PC1 and PC2 as reaction coordinates. The vertical axis
represents the relative free energy (in kcal/mol), highlighting the
depth of conformational basins and the relative energy barriers separating
metastable states.</p></figcaption></figure></section><section id="sec3.5"><h3 class="pmc_sec_title">3.5. Free Energy Landscape</h3>
<p>To investigate
the thermodynamic stability and conformational heterogeneity of the
states identified through PCA, a free energy landscape (FEL) analysis
was conducted by projecting the molecular dynamics trajectory onto
the first two principal components (PC1 and PC2). The resulting two-dimensional
FEL plot (<a href="#fig8" class="usa-link">Figures 
</a>c and c) delineates the conformational space
sampled by the system during the simulation, with local minima corresponding
to energetically favorable, thermodynamically stable states. The landscape
revealed two dominant energy basins, referred to as State A and State
B, which are separated by a relatively shallow free energy barrier
of approximately 2.5 kcal/mol. These basins represent distinct conformational
ensembles accessed throughout the trajectory. State A, characterized
by the lowest free energy, corresponds to the predominant structural
population observed during the latter stages of the simulation, suggesting
the stabilization of a functionally relevant conformation. Conversely,
State B represents a higher-energy, metastable intermediate potentially
linked to transient structural transitions, such as those involved
in ligand ingress/egress or partial receptor activation. The 3D FEL
Volcano plot visualization (<a href="#fig8" class="usa-link">Figures 
</a>d and d) emphasizes the ruggedness
of the conformational space sampled during the trajectory.</p>
<p>In
addition to the primary basins, several minor energy wells were identified
along the PC2 axis. These local minima likely reflect conformational
substates within the broader dominant ensemble, possibly arising from
flexible regions such as intracellular loops, transmembrane segments,
or ligand-binding domains. Such microstates may play critical roles
in dynamic processes, such as signal transduction, allosteric regulation,
or effector recruitment.</p></section><section id="sec3.6"><h3 class="pmc_sec_title">3.6. MMPBSA Analysis</h3>
<section id="sec3.6.1"><h4 class="pmc_sec_title">3.6.1. Predencin-Bound GIPR</h4>
<p>The N-terminal
extracellular domain (ECD) of GIPR is crucial for ligand interaction.
According to ref , specific residues in the ECD facilitate the initial interaction
with GIP (<a href="#fig10" class="usa-link">Figure 
</a>a). Residues like Asp63, Trp74, and Trp84 are tasked with stabilizing
the GIP ligand binding. Microbial peptide’s N-terminal amino
acids form hydrogen bonds and hydrophobic interactions with GIPR,
with Tyr1 interacting with residues like Arg301, Leu215, and Asp294.
Additionally, Glu3 forms ionic bonds with Arg151, which is essential
for the ligand’s binding. Upon Predencin binding, conformational
changes are induced, particularly in TM6, which bends outward to interact
probably with G-proteins. Specific hydrophobic and polar interactions
between the transmembrane helices and the microbial peptide will stabilize
the active receptor state. For example, as shown in <a href="#fig10" class="usa-link">Figure 
</a>b, while performing comparative
residue analysis, Arg293 and Asp294 contribute to the interaction
network, helping in maintaining the active conformation of the receptor.
These residues are vital for the receptor’s activation, mediating
both the ligand binding and the downstream signaling cascade, which
includes G protein interaction and subsequent cellular effects like
insulin secretion.</p>
<figure class="fig xbox font-sm" id="fig10"><h5 class="obj_head">10.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332626_ao5c03268_0010.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a52/12332626/e4971694d546/ao5c03268_0010.jpg" loading="lazy" id="gr10" height="387" width="697" alt="10"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>MMPBSA energy calculation for the GIPR and CRF1R microbial
peptide
complexes. (a) Binding energy boxplot of the top 10 potential residues
for Predencin-bound GIPR. (b) Energy heatmap of top 30 residues for
all poses for the GIPR complex. Key residues such as Asp63, Trp74,
and Trp84 are highlighted, which are important for the binding and
regulation of the receptor. (c) Binding energy boxplot of the top
10 potential residues for Paraexcin-bound CRF1R. (d) Energy heatmap
for the top 30 residues for all poses for the Paraexcin-CRF1R complex.
Key tyrosine residues in TM1 that are involved in receptor activation
and ligand binding are highlighted. Trp154 is a conserved tryptophan
residue that is essential for GPCR activation and contributes to the
preservation of the receptor’s structural integrity.</p></figcaption></figure></section><section id="sec3.6.2"><h4 class="pmc_sec_title">3.6.2. MMPBSA AnalysisParaexcin-Bound CRF1R</h4>
<p>The corticotropin-releasing factor 1 receptor (CRF1R) bound to
urocortin (UCN) reveals that several key residues are typically involved
in receptor activation. And its activation
involves interactions between the peptide ligand (urocortin) and specific
residues within the extracellular domain (ECD) and the transmembrane
(TM) region of the receptor. According to <sup>31</sup>, the tyrosine
residues in TM1 play a role in ligand interaction and receptor activation
(<a href="#fig10" class="usa-link">Figure 
</a>c). Trp154,
this conserved tryptophan residue, is crucial for GPCR activation
and plays a role in maintaining the structural integrity of the receptor.
As shown in <a href="#fig10" class="usa-link">Figure 
</a>d, key lysine residue Lys152 is involved in receptor activation,
potentially through interactions with conserved motifs within GPCRs.
Ser3 and Leu5 residues contribute to the binding affinity and stability
of the urocortin1–CRF1R interaction. Asn150 is involved in
stabilizing the active state of the receptor and is crucial for G-protein
coupling.</p></section></section><section id="sec3.7"><h3 class="pmc_sec_title">3.7. Validation of Peptide Origin from the Human
Gut Microbiome</h3>
<p>To confirm the microbial origin of the peptides,
a multistep bioinformatics pipeline was implemented. First, the peptide
sequence was converted into the corresponding nucleotide sequence,
and then genome blast was performed against the Integrated Microbial
Genome database to get respective protein and nucleotide matches.
Hits were considered microbial in origin only if no significant homology
was detected in human or other eukaryotic proteomes, verified by additional
BLASTp searches restricted to Homo sapiens. The obtained protein match
was then validated using UNIPROT by querying in the NCBI nonredundant
(nr) protein database using BLASTp with an E-value cutoff of 1 ×
10<sup>–5</sup>, retaining hits with ≥90% sequence identity
and ≥80% query coverage. To ensure specificity to microbial
taxa, top hits were cross-referenced with taxonomic identifiers using
the NCBI Taxonomy Browser. Then, the full-length amino acid sequence
of the acquired microbial peptide-associated protein was analyzed
using ExPasy’s Peptide Cutter to
predict the cleavage patterns and assess whether the desired microbial
peptide is released from the parent protein.</p>
<p>As represented
in the infographic (<a href="#fig11" class="usa-link">Figure 
</a>), the nucleotide sequence of the microbial peptides Predencin
and Paraexcin yielded two matches when blasted against the microbial
genome database. Predencin showed a 100% match with dITP/XTP pyrophosphatase: CRIS 18C-A, and Paraexcin showed a match
with Chaperone protein HtpG [] BIOML-A13. The protein and genome match obtained were validated
in UNIPROT (ID: A0A379E6F1). Cleavage patterns of both the proteins
showed that the candidate peptides Predencin and Paraexcin were being
extracted from both the proteins. These results provide us with verification
that these peptides indeed originate from the human gut microbiome.</p>
<figure class="fig xbox font-sm" id="fig11"><h4 class="obj_head">11.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332626_ao5c03268_0011.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a52/12332626/80081de38f38/ao5c03268_0011.jpg" loading="lazy" id="gr11" height="786" width="697" alt="11"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The
above infographic represents the pipeline for validating the
microbial peptides being derived from the human gut microbiota using
microbial genome blast, protein digestion, and NCBI blast.</p></figcaption></figure></section><section id="sec3.8"><h3 class="pmc_sec_title">3.8. Bacterial Population in the Human Gut Microbiome</h3>
<p> is a bacterium commonly
found in the human oral and gut microbiome. It belongs to the Prevotella
genus, which is known for its role in breaking down complex carbohydrates
and fibers, particularly in individuals who consume plant-rich diets.
While Prevotella species are often considered beneficial for digestion
and fiber breakdown, there are also associations with inflammation
and disease when the balance of gut microbes is disrupted. Prevotella
species, including , are
more prevalent in individuals consuming high-fiber, plant-based diets,
particularly in non-Western populations. Some studies suggest that
a high prevalence of Prevotella, including , may be linked to beneficial metabolic functions, such as enhanced
fermentation of dietary fibers. However,
there is growing interest in understanding its role in inflammation
and diseases such as periodontitis (as it is found in the oral microbiome)
or conditions such as irritable bowel syndrome (IBS), where a disturbed
balance of microbial populations can lead to negative health outcomes.
Since is also found in
the oral microbiome, its transition to the gut may occur during swallowing
or other interactions between oral and gut ecosystems. The migration
of oral bacteria to the gut has been linked to systemic inflammatory
responses in some cases. is a member of the Prevotella genus,
which plays a dual role in health: contributing to fiber digestion
and potentially being involved in inflammatory processes when the
microbial balance is disrupted. The role of , specifically in gut health, continues to be an area of ongoing
research.</p>
<p> is a bacterium from the genus Parasutterella, which is commonly
found in the human gut microbiome. This
genus has gained attention because it occupies a distinct ecological
niche and is thought to play a role in maintaining gut homeostasis
and influencing host metabolism. Although relatively understudied
compared to other gut bacteria, is considered a core member of the healthy human gut microbiome. is reported to have influence in bile acid metabolism, which is
crucial for fat digestion and regulation of gut microbial communities.
It appears to be involved in energy harvesting and the regulation
of gut immune responses, although its specific metabolic pathways
are still under investigation. It is considered a marker of a healthy
gut microbiome and is typically found in balanced quantities in people
without gut disorders. Some studies have noted reduced levels of Parasutterella
species in patients with inflammatory bowel disease (IBD) and irritable
bowel syndrome (IBS). Lower levels of may indicate a disruption of
bile acid metabolism or gut immune regulation. This bacterium seems
to have a symbiotic relationship with its human host, contributing
to the modulation of gut functions. It has been observed to affect
the synthesis of short-chain fatty acids (SCFAs), pivotal for maintaining
gut integrity and providing energy to colon cells. may
also interact with other microbial communities to help maintain microbial
balance, though its interactions are still being studied. The abundance of can be influenced by diet, particularly diets that impact bile acid
profiles. High-fat diets, for instance, may affect its population
due to the bacterium’s involvement in bile acid metabolism.</p></section></section><section id="sec4"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>The N-terminal residues
of a peptide ligand play an important role
in the activation of G protein-coupled receptors (GPCRs), particularly
peptide-bound GPCRs, for a variety of reasons, including conformational
change induction, contribution to ligand binding affinity, allosteric
regulation, and stability. The peptide
database Pepfunk consists of 234,852 peptides of variable length,
and physicochemical criteria were screened for molecular docking with
receptors. The N-terminal region of microbiome-derived peptides Predencin
and Paraexcin demonstrated the conservation of critical residues and
interactions within the binding pocket of GIPR and CRF1R during molecular
docking. The role of N-terminal residues in initiating and stabilizing
receptor–ligand interactions has been established previously;
our analysis focused on the specific contacts made by this region
of the peptide within the binding pocket and their contribution to
the overall binding affinity and conformational stability of the complex.
The highest-ranked binding poses of Predencin and Paraexcin demonstrated
a favorable binding free energy, suggesting a strong interaction with
the target receptor. N-terminal residue Tyr1 interacts with residue
His361<sup>TM6</sup> via a hydrogen bond in the TM6 region, which
likely leads to additional interactions of the peptide with the receptor.
The N-terminal hydrophobic residues Ala2 and Gly4 of the peptide engage
with hydrophobic residues L386<sup>TM7</sup> in the binding pocket,
especially near TM7, which may reduce solvent exposure, contributing
to the binding stability and the overall binding energy.</p>
<p>The
molecular dynamics (MD) simulations provided critical insights
into the stability and behavior of the protein–peptide complex
over a 300 ns time. The simulation study shows the dynamic nature
of interaction between Paraexcin–CRF1R; however, the complex
remains stable through the duration of the simulation, as evidenced
by the relatively low root-mean-square deviation (RMSD) values for
both the protein and peptide, particularly in the N-terminal binding
pocket.</p>
<p>For Predencin-bound GIPR, the RMSD plot shows various
fluctuations
throughout the simulation but appears stable toward the end of the
simulation. A stable RMSD over time may indicate that the complex
has attained structural stability.</p>
<p>In this study, we applied
distance-based structural alignment to
compare two protein conformations after MD simulations. We calculated
the distances of key residues involved in the interactions between
the peptide and the GPCR, between the post-MD conformation and the
receptor conformation that activates the G-protein signaling pathway
(<a href="#fig12" class="usa-link">Figures 
</a>a and a). This approach allowed us to capture both global
and local conformational changes between the structures, providing
novel insights into the dynamic aspects of the formation of the GPCR–peptide
complex. The distance-based method was particularly useful for identifying
localized structural differences. For instance, we observed conserved
residue–residue distances in the protein’s core (transmembrane
domain), indicating that these regions remain stable even in the face
of global conformational shifts. The movement of transmembrane region
6 observed during simulation is noteworthy, given the role it plays
in receptor activation. Residues like His 361, Tyr 239, and Arg 199
in post-MD GIPR have less deviation at 220–225 ns, and similarly,
His 198, Lys 351, Leu 260, and Tyr 195 in CRF1R showed structural
similarity compared to that of corresponding residues in standard
conformation at 100–125 ns.</p>
<figure class="fig xbox font-sm" id="fig12"><h3 class="obj_head">12.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332626_ao5c03268_0012.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a52/12332626/34c649dc38fe/ao5c03268_0012.jpg" loading="lazy" id="gr12" height="455" width="750" alt="12"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig12/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Residue–residue distances showcasing
the key conformational
changes required for modulation of the receptor. (a) Distance-based
structural alignment of post-MD conformation and G-Protein-coupled
conformation (dark blue: Predencin-bound GIPR; light blue: G-Protein-coupled
receptor; green: inactive model). (b) The residue–residue distance
matrices compare the conformational shifts between the microbial peptide-bound
model and the GPCR-bound structure, highlighting regions of high conservation
and significant deviation. (c) The conformational shifts between the
microbial peptide-bound model and the inactive model are shown. (d)
Notable conformational changes are observed in the transmembrane region
6 and the extracellular ligand-binding domain, which may be related
to GPCR activation and signal transduction.</p></figcaption></figure><figure class="fig xbox font-sm" id="fig13"><h3 class="obj_head">13.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332626_ao5c03268_0013.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0a52/12332626/1abf26099d85/ao5c03268_0013.jpg" loading="lazy" id="gr13" height="457" width="750" alt="13"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig13/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Residue–residue distances showcasing the key conformational
changes required for modulation of the receptor. (a) Distance-based
structural alignment between post-MD conformation and G-protein-coupled
conformation (red: Paraexcin-bound CRF1R; brown: G-protein-coupled
receptor; cyan: inactive conformation). (b) The residue–residue
distance matrices compare the conformational changes between the Paraexcin-bound
model and the GPCR structure, emphasizing areas of strong conservation
and considerable divergence. (c) The conformational shifts between
the Paraexcin-bound model and inactive structure (PDB ID: <a href="https://www.rcsb.org/structure/4K5Y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">4K5Y</a>) are shown. (d)
Significant conformational changes are seen in the transmembrane region
6 and the extracellular ligand-binding domain, which could be connected
to GPCR activation and signal transduction.</p></figcaption></figure><p>
<a href="#fig12" class="usa-link">Figures 
</a>d and d clearly present that the
TM6 helix is moving
outward in both GPCRs and microbial peptide-bound receptors, whereas
in the inactive conformation (<a href="#fig12" class="usa-link">Figures 
</a>c and c), the TM6
helix is bent inward. The outward transition of TM6 in G protein-coupled
receptors is one of the hallmark conformational shift coupled with
receptor activation. This movement enables the receptor transition
to an active state capable of interacting with molecular transducers.
The subsiding deviation of residual distances may point to the possibility
of the receptors undergoing G-protein signaling in the presence of
the microbial peptides (<a href="#fig12" class="usa-link">Figures 
</a>b and b). The binding free
energy of the receptor–peptide complex, calculated using the
MMPBSA method, provided quantitative insights into the interaction
strength. The binding free energy was −25.3 kcal/mol, indicating
a strong interaction between the peptide and the receptor. The MMPBSA
analysis underscores the importance of hydrogen bonds and hydrophobic
interactions in stabilizing the receptor–peptide complex. The
strong binding affinity of Predencin and Paraexcin to GIPR and CRF1R
suggests their potential role as agonists in the activation of the
receptor. The key residues contributing to the binding energy were
identified, including Trp154 and Lys152 in GIPR and Trp84, Arg151,
and Arg293 in CRF1R. Modifications to these residues could lead to
enhanced or reduced binding affinity, suggesting avenues for future
peptide optimization studies.</p>
<p>PCA and FEL analyses served as
complementary approaches to uncover
the key dynamic and thermodynamic features of the system throughout
the molecular dynamics (MD) simulation. By reducing the complexity
of atomic motions into a set of orthogonal vectors, PCA effectively
highlighted the dominant collective movements that often drive critical
conformational changes related to protein function. The initial few
principal components, particularly PC1 and PC2, captured a significant
fraction of the overall motion, indicating that the essential dynamics
are largely confined to a low-dimensional subspace. This dimensionality
reduction not only simplifies interpretation but also enhances our
ability to detect biologically meaningful motions. The trajectory
projected along PC1 and PC2 revealed discrete conformational clusters,
suggesting transitions between multiple metastable states over the
simulation time scale. These findings point to the inherent flexibility
of the protein and the presence of conformational substates that may
underlie different functional roles.</p>
<p>Further structural mapping
of the motions along PC1 and PC2 showed
that these transitions are concentrated in specific regions of the
protein such as transmembrane domains 6 and 7 and flexible loops (ICL)
that are commonly associated with processes like ligand binding, allosteric
control, or conformational switching. These patterns are consistent
with known structural mechanisms, reinforcing the functional importance
of the observed dynamics. To complement these observations, the FEL
analysis provided a thermodynamic view of how the system explores
conformational space. The two-dimensional FEL, projected along the
primary principal components, identified distinct energy basins separated
by modest barriers. The presence of two dominant minima-designated
states A and B suggests that the protein alternates between stable
and metastable conformations during the simulation. State A, representing
the global free energy minimum, became increasingly populated in the
latter stages, implying stabilization of a potentially active or functionally
relevant conformation. State B, being higher in energy, may correspond
to intermediate or transition-state conformations, possibly linked
to functional motions such as ligand ingress or preactivation steps.
Additional shallow wells along the PC2 axis point to the existence
of microstates within the broader conformational ensemble, possibly
arising from local flexibility in structural motifs like intracellular
loops or helical kinks. These substates could be functionally relevant,
offering the structural plasticity required for dynamic processes
such as effector coupling or allosteric signal transmission. The three-dimensional
FEL volcano plot further highlighted the rugged topography of the
energy landscape, illustrating both the depth of the conformational
wells and the energy barriers between them. The relatively low barrier
(∼2.5 kcal/mol) separating states A and B suggests that such
transitions are energetically accessible under physiological conditions,
potentially facilitating the dynamic behavior necessary for biological
activity.</p>
<p>In summary, the combined use of PCA and FEL analysis
provides a
robust framework for interpreting the conformational and energetic
landscape of biomolecular systems. These approaches not only identify
dominant structural transitions but also offer insights into the thermodynamic
stability and accessibility of functional states. Such information
is particularly valuable in the study of complex and dynamic proteins,
including GPCRs, membrane transporters, and multidomain enzymes, where
function is intricately linked to conformational flexibility. Native
peptides that bind to G-protein-coupled receptors have significant
therapeutic potential but with limitations like low stability and
enzymatic degradation, bioavailability, short half-life, specificity,
and off-targets. Microbial peptides, those derived from bacteria,
fungi, and other microorganisms that reside in the human gut, have
been reported to show proteolytic stability, chemical stability, improved
bioavailability, and longer half-life, which suggests that microbial
peptides are interesting therapeutic candidates for peptide-based
drug developments with reduced side effects.
</p></section><section id="sec5"><h2 class="pmc_sec_title">5. Conclusion</h2>
<p>In this study, we explored
the potential of human gut microbial
peptides as mimicking modulators of secretin family receptors. The
current work highlighted the intricate interplay between the gut microflora
and host signaling pathways, demonstrating that microbial peptides
could interact with and modulate the signaling of Class B GPCRs. These
findings open new area into the structural and functional diversity
of microbial-derived peptides with their ability to influence physiological
processes, such as glucose homeostasis, appetite regulation, and hormonal
responses. Furthermore, the computational approaches employed reveal
key peptide sequences and receptor residues critical for binding and
activation, paving the way for the rational design of therapeutic
peptides. Future studies should delve deeper into in vivo functional
assays and clinical investigations to validate these interactions
and to explore their therapeutic applications. Harnessing the gut
microbiome’s molecular repertoire may open new avenues for
targeted treatments of metabolic and gastrointestinal disorders by
leveraging the specificity and tunability of GPCR signaling.</p></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>I thank Dr. Abhishek
Bandyopadhyay, an Assistant
Research Scientist at Purdue University, for critical reading of the
manuscript and helpful inputs.</p></section><section id="notes3"><p>The data supporting
the findings of this study are available from the corresponding author
upon reasonable request</p></section><section id="notes2"><p>K.P., S.S.,
and P.N. conceived and designed the study. S.S. performed all the
simulations and analyses. P.N. assisted in performing experiments
and analyzing the data. S.S., K.P., and P.N. wrote the manuscript.
All authors reviewed and approved the final manuscript.</p></section><section id="notes1"><p>The authors
declare no competing financial interest.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>Korpela K., de Vos W. M.. Early life colonization of the human gut: microbes
matter everywhere. Curr. Opin. Microbiol. 2018;44:70–78. doi: 10.1016/j.mib.2018.06.003.</cite> [<a href="https://doi.org/10.1016/j.mib.2018.06.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30086431/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Microbiol.&amp;title=Early%20life%20colonization%20of%20the%20human%20gut:%20microbes%20matter%20everywhere&amp;author=K.%20Korpela&amp;author=W.%20M.%20de%20Vos&amp;volume=44&amp;publication_year=2018&amp;pages=70-78&amp;pmid=30086431&amp;doi=10.1016/j.mib.2018.06.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>Lynch S. V., Pedersen O.. The Human Intestinal
Microbiome in Health and Disease. N. Engl. J.
Med. 2016;375:2369–2379. doi: 10.1056/NEJMra1600266.</cite> [<a href="https://doi.org/10.1056/NEJMra1600266" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27974040/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=N.%20Engl.%20J.%20Med.&amp;title=The%20Human%20Intestinal%20Microbiome%20in%20Health%20and%20Disease&amp;author=S.%20V.%20Lynch&amp;author=O.%20Pedersen&amp;volume=375&amp;publication_year=2016&amp;pages=2369-2379&amp;pmid=27974040&amp;doi=10.1056/NEJMra1600266&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Fan Y., Pedersen O.. Gut microbiota in human
metabolic health and disease. Nat. Rev. Microbiol. 2021;19:55–71. doi: 10.1038/s41579-020-0433-9.</cite> [<a href="https://doi.org/10.1038/s41579-020-0433-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32887946/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Microbiol.&amp;title=Gut%20microbiota%20in%20human%20metabolic%20health%20and%20disease&amp;author=Y.%20Fan&amp;author=O.%20Pedersen&amp;volume=19&amp;publication_year=2021&amp;pages=55-71&amp;pmid=32887946&amp;doi=10.1038/s41579-020-0433-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>de
Vos W. M., Tilg H., Van Hul M., Cani P. D.. Gut microbiome
and health: mechanistic insights. Gut. 2022;71:1020–1032. doi: 10.1136/gutjnl-2021-326789.</cite> [<a href="https://doi.org/10.1136/gutjnl-2021-326789" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8995832/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35105664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Gut%20microbiome%20and%20health:%20mechanistic%20insights&amp;author=W.%20M.%20de%20Vos&amp;author=H.%20Tilg&amp;author=M.%20Van%20Hul&amp;author=P.%20D.%20Cani&amp;volume=71&amp;publication_year=2022&amp;pages=1020-1032&amp;pmid=35105664&amp;doi=10.1136/gutjnl-2021-326789&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Cryan J. F.. et al. The Microbiota-Gut-Brain
Axis. Physiol. Rev. 2019;99:1877–2013. doi: 10.1152/physrev.00018.2018.</cite> [<a href="https://doi.org/10.1152/physrev.00018.2018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31460832/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Physiol.%20Rev.&amp;title=The%20Microbiota-Gut-Brain%20Axis&amp;author=J.%20F.%20Cryan&amp;volume=99&amp;publication_year=2019&amp;pages=1877-2013&amp;pmid=31460832&amp;doi=10.1152/physrev.00018.2018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Rajilić-Stojanović M., de Vos W. M.. The first 1000 cultured
species of the human gastrointestinal
microbiota. FEMS Microbiol. Rev. 2014;38:996–1047. doi: 10.1111/1574-6976.12075.</cite> [<a href="https://doi.org/10.1111/1574-6976.12075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4262072/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24861948/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Rev.&amp;title=The%20first%201000%20cultured%20species%20of%20the%20human%20gastrointestinal%20microbiota&amp;author=M.%20Rajili%C4%87-Stojanovi%C4%87&amp;author=W.%20M.%20de%20Vos&amp;volume=38&amp;publication_year=2014&amp;pages=996-1047&amp;pmid=24861948&amp;doi=10.1111/1574-6976.12075&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Booijink C. C. G. M.. et al. High temporal and inter-individual variation
detected
in the human ileal microbiota. Environ. Microbiol. 2010;12:3213–3227. doi: 10.1111/j.1462-2920.2010.02294.x.</cite> [<a href="https://doi.org/10.1111/j.1462-2920.2010.02294.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20626454/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&amp;title=High%20temporal%20and%20inter-individual%20variation%20detected%20in%20the%20human%20ileal%20microbiota&amp;author=C.%20C.%20G.%20M.%20Booijink&amp;volume=12&amp;publication_year=2010&amp;pages=3213-3227&amp;pmid=20626454&amp;doi=10.1111/j.1462-2920.2010.02294.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Ding Z.. et al. Novel scheme for non-invasive
gut bioinformation acquisition with
a magnetically controlled sampling capsule endoscope. Gut. 2021;70:2297–2306. doi: 10.1136/gutjnl-2020-322465.</cite> [<a href="https://doi.org/10.1136/gutjnl-2020-322465" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33452177/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Novel%20scheme%20for%20non-invasive%20gut%20bioinformation%20acquisition%20with%20a%20magnetically%20controlled%20sampling%20capsule%20endoscope&amp;author=Z.%20Ding&amp;volume=70&amp;publication_year=2021&amp;pages=2297-2306&amp;pmid=33452177&amp;doi=10.1136/gutjnl-2020-322465&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Dhakan D. B., Maji A., Sharma A. K., Saxena R., Pulikkan J., Grace T., Gomez A., Scaria J., Amato K. R., Sharma V. K.. The unique composition
of Indian gut microbiome, gene
catalogue, and associated fecal metabolome deciphered using multi-omics
approaches. Gigascience. 2019;8:giz004. doi: 10.1093/gigascience/giz004.</cite> [<a href="https://doi.org/10.1093/gigascience/giz004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6394208/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30698687/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gigascience&amp;title=The%20unique%20composition%20of%20Indian%20gut%20microbiome,%20gene%20catalogue,%20and%20associated%20fecal%20metabolome%20deciphered%20using%20multi-omics%20approaches&amp;author=D.%20B.%20Dhakan&amp;author=A.%20Maji&amp;author=A.%20K.%20Sharma&amp;author=R.%20Saxena&amp;author=J.%20Pulikkan&amp;volume=8&amp;publication_year=2019&amp;pages=giz004&amp;pmid=30698687&amp;doi=10.1093/gigascience/giz004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Ojala T., Kankuri E., Kankainen M.. Understanding human health through
metatranscriptomics. Trends Mol. Med. 2023;29:376–389. doi: 10.1016/j.molmed.2023.02.002.</cite> [<a href="https://doi.org/10.1016/j.molmed.2023.02.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36842848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Mol.%20Med.&amp;title=Understanding%20human%20health%20through%20metatranscriptomics&amp;author=T.%20Ojala&amp;author=E.%20Kankuri&amp;author=M.%20Kankainen&amp;volume=29&amp;publication_year=2023&amp;pages=376-389&amp;pmid=36842848&amp;doi=10.1016/j.molmed.2023.02.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Zoetendal E. G.. et al. The human small intestinal
microbiota is driven by rapid uptake and
conversion of simple carbohydrates. ISME J. 2012;6:1415–1426. doi: 10.1038/ismej.2011.212.</cite> [<a href="https://doi.org/10.1038/ismej.2011.212" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3379644/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22258098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ISME%20J.&amp;title=The%20human%20small%20intestinal%20microbiota%20is%20driven%20by%20rapid%20uptake%20and%20conversion%20of%20simple%20carbohydrates&amp;author=E.%20G.%20Zoetendal&amp;volume=6&amp;publication_year=2012&amp;pages=1415-1426&amp;pmid=22258098&amp;doi=10.1038/ismej.2011.212&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Louis P., Flint H. J.. Formation of propionate and butyrate
by the human colonic
microbiota. Environ. Microbiol. 2017;19:29–41. doi: 10.1111/1462-2920.13589.</cite> [<a href="https://doi.org/10.1111/1462-2920.13589" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27928878/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&amp;title=Formation%20of%20propionate%20and%20butyrate%20by%20the%20human%20colonic%20microbiota&amp;author=P.%20Louis&amp;author=H.%20J.%20Flint&amp;volume=19&amp;publication_year=2017&amp;pages=29-41&amp;pmid=27928878&amp;doi=10.1111/1462-2920.13589&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Tilg H., Adolph T. E., Gerner R. R., Moschen A. R.. The Intestinal Microbiota
in Colorectal Cancer. Cancer Cell. 2018;33:954–964. doi: 10.1016/j.ccell.2018.03.004.</cite> [<a href="https://doi.org/10.1016/j.ccell.2018.03.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29657127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=The%20Intestinal%20Microbiota%20in%20Colorectal%20Cancer&amp;author=H.%20Tilg&amp;author=T.%20E.%20Adolph&amp;author=R.%20R.%20Gerner&amp;author=A.%20R.%20Moschen&amp;volume=33&amp;publication_year=2018&amp;pages=954-964&amp;pmid=29657127&amp;doi=10.1016/j.ccell.2018.03.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Gilijamse P. W., Hartstra A. V., Levin E., Wortelboer K., Serlie M. J., Ackermans M. T., Herrema H., Nederveen A. J., Imangaliyev S., Aalvink S.. et al. Treatment with Anaerobutyricum
soehngenii: a pilot study of safety and dose-response effects on glucose
metabolism in human subjects with metabolic syndrome. npj Biofilms Microbiomes. 2020;6:16. doi: 10.1038/s41522-020-0127-0.</cite> [<a href="https://doi.org/10.1038/s41522-020-0127-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7101376/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32221294/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=npj%20Biofilms%20Microbiomes&amp;title=Treatment%20with%20Anaerobutyricum%20soehngenii:%20a%20pilot%20study%20of%20safety%20and%20dose-response%20effects%20on%20glucose%20metabolism%20in%20human%20subjects%20with%20metabolic%20syndrome&amp;author=P.%20W.%20Gilijamse&amp;author=A.%20V.%20Hartstra&amp;author=E.%20Levin&amp;author=K.%20Wortelboer&amp;author=M.%20J.%20Serlie&amp;volume=6&amp;publication_year=2020&amp;pages=16&amp;pmid=32221294&amp;doi=10.1038/s41522-020-0127-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>Sánchez-Alcoholado L., Ordóñez R., Otero A., Plaza-Andrade I., Laborda-Illanes A., Medina J. A., Ramos-Molina B., Gómez-Millán J., Queipo-Ortuño M. I.. Gut Microbiota-Mediated
Inflammation and Gut Permeability in Patients with Obesity and Colorectal
Cancer. Int. J. Mol. Sci. 2020;21:6782. doi: 10.3390/ijms21186782.</cite> [<a href="https://doi.org/10.3390/ijms21186782" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7555154/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32947866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Gut%20Microbiota-Mediated%20Inflammation%20and%20Gut%20Permeability%20in%20Patients%20with%20Obesity%20and%20Colorectal%20Cancer&amp;author=L.%20S%C3%A1nchez-Alcoholado&amp;author=R.%20Ord%C3%B3%C3%B1ez&amp;author=A.%20Otero&amp;author=I.%20Plaza-Andrade&amp;author=A.%20Laborda-Illanes&amp;volume=21&amp;publication_year=2020&amp;pages=6782&amp;pmid=32947866&amp;doi=10.3390/ijms21186782&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Kamato D., Thach L., Bernard R., Chan V., Zheng W., Kaur H., Brimble M., Osman N., Little P. J.. Structure,
Function, Pharmacology, and Therapeutic Potential of the G Protein,
Gα/q,11. Front. Cardiovasc. Med. 2015;2:14. doi: 10.3389/fcvm.2015.00014.</cite> [<a href="https://doi.org/10.3389/fcvm.2015.00014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4671355/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26664886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Cardiovasc.%20Med.&amp;title=Structure,%20Function,%20Pharmacology,%20and%20Therapeutic%20Potential%20of%20the%20G%20Protein,%20G%CE%B1/q,11&amp;author=D.%20Kamato&amp;author=L.%20Thach&amp;author=R.%20Bernard&amp;author=V.%20Chan&amp;author=W.%20Zheng&amp;volume=2&amp;publication_year=2015&amp;pages=14&amp;pmid=26664886&amp;doi=10.3389/fcvm.2015.00014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Kee T. R.. et al. The multifaceted functions of β-arrestins and their therapeutic
potential in neurodegenerative diseases. Exp.
Mol. Med. 2024;56:129–141. doi: 10.1038/s12276-023-01144-4.</cite> [<a href="https://doi.org/10.1038/s12276-023-01144-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10834518/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38212557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Exp.%20Mol.%20Med.&amp;title=The%20multifaceted%20functions%20of%20%CE%B2-arrestins%20and%20their%20therapeutic%20potential%20in%20neurodegenerative%20diseases&amp;author=T.%20R.%20Kee&amp;volume=56&amp;publication_year=2024&amp;pages=129-141&amp;pmid=38212557&amp;doi=10.1038/s12276-023-01144-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>Syrovatkina V., Alegre K. O., Dey R., Huang X.-Y.. Regulation, Signaling,
and Physiological Functions of G-Proteins. J.
Mol. Biol. 2016;428:3850–3868. doi: 10.1016/j.jmb.2016.08.002.</cite> [<a href="https://doi.org/10.1016/j.jmb.2016.08.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5023507/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27515397/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Biol.&amp;title=Regulation,%20Signaling,%20and%20Physiological%20Functions%20of%20G-Proteins&amp;author=V.%20Syrovatkina&amp;author=K.%20O.%20Alegre&amp;author=R.%20Dey&amp;author=X.-Y.%20Huang&amp;volume=428&amp;publication_year=2016&amp;pages=3850-3868&amp;pmid=27515397&amp;doi=10.1016/j.jmb.2016.08.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Gurevich V. V., Gurevich E. V.. GPCR Signaling Regulation:
The Role of GRKs and Arrestins. Front. Pharmacol. 2019;10:125. doi: 10.3389/fphar.2019.00125.</cite> [<a href="https://doi.org/10.3389/fphar.2019.00125" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6389790/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30837883/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Pharmacol&amp;title=GPCR%20Signaling%20Regulation:%20The%20Role%20of%20GRKs%20and%20Arrestins&amp;author=V.%20V.%20Gurevich&amp;author=E.%20V.%20Gurevich&amp;volume=10&amp;publication_year=2019&amp;pages=125&amp;pmid=30837883&amp;doi=10.3389/fphar.2019.00125&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>Pal K., Melcher K., Xu H. E.. Structure and mechanism for recognition
of peptide hormones by Class B G-protein-coupled receptors. Acta Pharmacol. Sin. 2012;33:300–311. doi: 10.1038/aps.2011.170.</cite> [<a href="https://doi.org/10.1038/aps.2011.170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3690506/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22266723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Pharmacol.%20Sin.&amp;title=Structure%20and%20mechanism%20for%20recognition%20of%20peptide%20hormones%20by%20Class%20B%20G-protein-coupled%20receptors&amp;author=K.%20Pal&amp;author=K.%20Melcher&amp;author=H.%20E.%20Xu&amp;volume=33&amp;publication_year=2012&amp;pages=300-311&amp;pmid=22266723&amp;doi=10.1038/aps.2011.170&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>Pioszak A. A., Xu H. E.. Molecular recognition
of parathyroid hormone by its G protein-coupled
receptor. Proc. Natl. Acad. Sci. U.S.A. 2008;105:5034–5039. doi: 10.1073/pnas.0801027105.</cite> [<a href="https://doi.org/10.1073/pnas.0801027105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2278174/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18375760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Molecular%20recognition%20of%20parathyroid%20hormone%20by%20its%20G%20protein-coupled%20receptor&amp;author=A.%20A.%20Pioszak&amp;author=H.%20E.%20Xu&amp;volume=105&amp;publication_year=2008&amp;pages=5034-5039&amp;pmid=18375760&amp;doi=10.1073/pnas.0801027105&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Pal K., Swaminathan K., Xu H. E., Pioszak A. A.. Structural basis
for hormone recognition by the Human CRFR2­{alpha} G protein-coupled
receptor. J. Biol. Chem. 2010;285:40351–40361. doi: 10.1074/jbc.M110.186072.</cite> [<a href="https://doi.org/10.1074/jbc.M110.186072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3001015/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20966082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Structural%20basis%20for%20hormone%20recognition%20by%20the%20Human%20CRFR2%C2%AD%7Balpha%7D%20G%20protein-coupled%20receptor&amp;author=K.%20Pal&amp;author=K.%20Swaminathan&amp;author=H.%20E.%20Xu&amp;author=A.%20A.%20Pioszak&amp;volume=285&amp;publication_year=2010&amp;pages=40351-40361&amp;pmid=20966082&amp;doi=10.1074/jbc.M110.186072&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Latorraca N. R., Venkatakrishnan A. J., Dror R. O.. GPCR dynamics: Structures in motion. Chem. Rev. 2017;117:139–155. doi: 10.1021/acs.chemrev.6b00177.</cite> [<a href="https://doi.org/10.1021/acs.chemrev.6b00177" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27622975/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Rev.&amp;title=GPCR%20dynamics:%20Structures%20in%20motion&amp;author=N.%20R.%20Latorraca&amp;author=A.%20J.%20Venkatakrishnan&amp;author=R.%20O.%20Dror&amp;volume=117&amp;publication_year=2017&amp;pages=139-155&amp;pmid=27622975&amp;doi=10.1021/acs.chemrev.6b00177&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>

Dror, R. O.

et al. 
Activation mechanism of the β 2-adrenergic
receptor, 2011. 
108, 18684-18689.doi: 10.1073/pnas.1110499108.</cite> [<a href="https://doi.org/10.1073/pnas.1110499108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3219117/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22031696/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dror,%20R.%20O.%20et%20al.%20Activation%20mechanism%20of%20the%20%CE%B2%202-adrenergic%20receptor,%202011.%20108,%2018684-18689.doi:%2010.1073/pnas.1110499108." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Fukuhara S.. et al. Structure of the human secretin receptor coupled to an engineered
heterotrimeric G protein. Biochem. Biophys.
Res. Commun. 2020;533:861–866. doi: 10.1016/j.bbrc.2020.08.042.</cite> [<a href="https://doi.org/10.1016/j.bbrc.2020.08.042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33008599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;title=Structure%20of%20the%20human%20secretin%20receptor%20coupled%20to%20an%20engineered%20heterotrimeric%20G%20protein&amp;author=S.%20Fukuhara&amp;volume=533&amp;publication_year=2020&amp;pages=861-866&amp;pmid=33008599&amp;doi=10.1016/j.bbrc.2020.08.042&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Qiao A.. et al. Structural basis of
Gs and Gi recognition by the human glucagon receptor. Science. 2020;367:1346–1352. doi: 10.1126/science.aaz5346.</cite> [<a href="https://doi.org/10.1126/science.aaz5346" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32193322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Structural%20basis%20of%20Gs%20and%20Gi%20recognition%20by%20the%20human%20glucagon%20receptor&amp;author=A.%20Qiao&amp;volume=367&amp;publication_year=2020&amp;pages=1346-1352&amp;pmid=32193322&amp;doi=10.1126/science.aaz5346&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Hilger D., Kumar K. K., Hu H., Pedersen M. F., O’Brien E. S., Giehm L., Jennings C., Eskici G., Inoue A., Lerch M.. et al. Structural
insights into differences in G protein activation
by family A and family B GPCRs. Science. 2020;369:eaba3373.  doi: 10.1126/science.aba3373.</cite> [<a href="https://doi.org/10.1126/science.aba3373" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7954662/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32732395/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Structural%20insights%20into%20differences%20in%20G%20protein%20activation%20by%20family%20A%20and%20family%20B%20GPCRs&amp;author=D.%20Hilger&amp;author=K.%20K.%20Kumar&amp;author=H.%20Hu&amp;author=M.%20F.%20Pedersen&amp;author=E.%20S.%20O%E2%80%99Brien&amp;volume=369&amp;publication_year=2020&amp;pages=eaba3373&amp;pmid=32732395&amp;doi=10.1126/science.aba3373&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Zhang X.. et al. Evolving cryo-EM structural approaches for GPCR drug
discovery. Structure. 2021;29:963–974e6. doi: 10.1016/j.str.2021.04.008.</cite> [<a href="https://doi.org/10.1016/j.str.2021.04.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33957078/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Evolving%20cryo-EM%20structural%20approaches%20for%20GPCR%20drug%20discovery&amp;author=X.%20Zhang&amp;volume=29&amp;publication_year=2021&amp;pages=963-974e6&amp;pmid=33957078&amp;doi=10.1016/j.str.2021.04.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>Zhao L.-H.. et al. Structure and dynamics of the active human parathyroid
hormone receptor-1. Science. 2019;364:148–153. doi: 10.1126/science.aav7942.</cite> [<a href="https://doi.org/10.1126/science.aav7942" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6929210/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30975883/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Structure%20and%20dynamics%20of%20the%20active%20human%20parathyroid%20hormone%20receptor-1&amp;author=L.-H.%20Zhao&amp;volume=364&amp;publication_year=2019&amp;pages=148-153&amp;pmid=30975883&amp;doi=10.1126/science.aav7942&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Zhao L.-H.. et al. Molecular recognition of two endogenous hormones by the human parathyroid
hormone receptor-1. Acta Pharmacol. Sin. 2023;44:1227–1237. doi: 10.1038/s41401-022-01032-z.</cite> [<a href="https://doi.org/10.1038/s41401-022-01032-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10203121/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36482086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Pharmacol.%20Sin.&amp;title=Molecular%20recognition%20of%20two%20endogenous%20hormones%20by%20the%20human%20parathyroid%20hormone%20receptor-1&amp;author=L.-H.%20Zhao&amp;volume=44&amp;publication_year=2023&amp;pages=1227-1237&amp;pmid=36482086&amp;doi=10.1038/s41401-022-01032-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>Ma S.. et al. Molecular Basis for
Hormone Recognition and Activation of Corticotropin-Releasing
Factor Receptors. Mol. Cell. 2020;77:669–680e4. doi: 10.1016/j.molcel.2020.01.013.</cite> [<a href="https://doi.org/10.1016/j.molcel.2020.01.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32004470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=Molecular%20Basis%20for%20Hormone%20Recognition%20and%20Activation%20of%20Corticotropin-Releasing%20Factor%20Receptors&amp;author=S.%20Ma&amp;volume=77&amp;publication_year=2020&amp;pages=669-680e4&amp;pmid=32004470&amp;doi=10.1016/j.molcel.2020.01.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>Piper S. J., Deganutti G., Lu J., Zhao P., Liang Y. L., Lu Y., Fletcher M. M., Hossain M. A., Christopoulos A., Reynolds C. A.. et al. Understanding VPAC receptor
family peptide
binding and selectivity. Nat. Commun. 2022;13:7013. doi: 10.1038/s41467-022-34629-3.</cite> [<a href="https://doi.org/10.1038/s41467-022-34629-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9668914/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36385145/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Understanding%20VPAC%20receptor%20family%20peptide%20binding%20and%20selectivity&amp;author=S.%20J.%20Piper&amp;author=G.%20Deganutti&amp;author=J.%20Lu&amp;author=P.%20Zhao&amp;author=Y.%20L.%20Liang&amp;volume=13&amp;publication_year=2022&amp;pages=7013&amp;pmid=36385145&amp;doi=10.1038/s41467-022-34629-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>Xiao H., Sun Q., Sun Q.. A promising small molecule
binding pocket in class
B GPCRs: expanding potential for drug development. Signal Transduction Targeted Ther. 2023;8:313. doi: 10.1038/s41392-023-01598-y.</cite> [<a href="https://doi.org/10.1038/s41392-023-01598-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10421945/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37567868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Signal%20Transduction%20Targeted%20Ther.&amp;title=A%20promising%20small%20molecule%20binding%20pocket%20in%20class%20B%20GPCRs:%20expanding%20potential%20for%20drug%20development&amp;author=H.%20Xiao&amp;author=Q.%20Sun&amp;author=Q.%20Sun&amp;volume=8&amp;publication_year=2023&amp;pages=313&amp;pmid=37567868&amp;doi=10.1038/s41392-023-01598-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34">
<cite>Zhao L.-H.. et al. Differential Requirement
of the Extracellular Domain in Activation
of Class B G Protein-coupled Receptors. J. Biol.
Chem. 2016;291:15119–15130. doi: 10.1074/jbc.M116.726620.</cite> [<a href="https://doi.org/10.1074/jbc.M116.726620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4946928/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27226600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Differential%20Requirement%20of%20the%20Extracellular%20Domain%20in%20Activation%20of%20Class%20B%20G%20Protein-coupled%20Receptors&amp;author=L.-H.%20Zhao&amp;volume=291&amp;publication_year=2016&amp;pages=15119-15130&amp;pmid=27226600&amp;doi=10.1074/jbc.M116.726620&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref35">
<cite>Fortin J.-P.. et al. Membrane-tethered ligands
are effective probes for exploring class
B1 G protein-coupled receptor function. Proc.
Natl. Acad. Sci. U.S.A. 2009;106:8049–8054. doi: 10.1073/pnas.0900149106.</cite> [<a href="https://doi.org/10.1073/pnas.0900149106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2683074/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19416829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Membrane-tethered%20ligands%20are%20effective%20probes%20for%20exploring%20class%20B1%20G%20protein-coupled%20receptor%20function&amp;author=J.-P.%20Fortin&amp;volume=106&amp;publication_year=2009&amp;pages=8049-8054&amp;pmid=19416829&amp;doi=10.1073/pnas.0900149106&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>Zhao L.-H.. et al. Differential Requirement of the Extracellular Domain in Activation
of Class B G Protein-coupled Receptors. J. Biol.
Chem. 2016;291:15119–15130. doi: 10.1074/jbc.M116.726620.</cite> [<a href="https://doi.org/10.1074/jbc.M116.726620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4946928/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27226600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Differential%20Requirement%20of%20the%20Extracellular%20Domain%20in%20Activation%20of%20Class%20B%20G%20Protein-coupled%20Receptors&amp;author=L.-H.%20Zhao&amp;volume=291&amp;publication_year=2016&amp;pages=15119-15130&amp;pmid=27226600&amp;doi=10.1074/jbc.M116.726620&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>Gallardo-Becerra L., Cervantes-Echeverría M., Cornejo-Granados F., Vazquez-Morado L. E., Ochoa-Leyva A.. Perspectives
in Searching Antimicrobial
Peptides (AMPs) Produced by the Microbiota. Microb. Ecol. 2023;87:8. doi: 10.1007/s00248-023-02313-8.</cite> [<a href="https://doi.org/10.1007/s00248-023-02313-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10689560/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38036921/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microb.%20Ecol.&amp;title=Perspectives%20in%20Searching%20Antimicrobial%20Peptides%20(AMPs)%20Produced%20by%20the%20Microbiota&amp;author=L.%20Gallardo-Becerra&amp;author=M.%20Cervantes-Echeverr%C3%ADa&amp;author=F.%20Cornejo-Granados&amp;author=L.%20E.%20Vazquez-Morado&amp;author=A.%20Ochoa-Leyva&amp;volume=87&amp;publication_year=2023&amp;pages=8&amp;pmid=38036921&amp;doi=10.1007/s00248-023-02313-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Torres M. D. T.. et al. Mining human microbiomes
reveals an untapped source
of peptide antibiotics. Cell. 2024;187:5453–5467e15. doi: 10.1016/j.cell.2024.07.027.</cite> [<a href="https://doi.org/10.1016/j.cell.2024.07.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39163860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Mining%20human%20microbiomes%20reveals%20an%20untapped%20source%20of%20peptide%20antibiotics&amp;author=M.%20D.%20T.%20Torres&amp;volume=187&amp;publication_year=2024&amp;pages=5453-5467e15&amp;pmid=39163860&amp;doi=10.1016/j.cell.2024.07.027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Wang B., Xie N., Li B.. Influence of peptide
characteristics on their stability,
intestinal transport, and in vitro bioavailability: A review. J. Food Biochem. 2019;43:e12571.  doi: 10.1111/jfbc.12571.</cite> [<a href="https://doi.org/10.1111/jfbc.12571" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31353489/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Food%20Biochem.&amp;title=Influence%20of%20peptide%20characteristics%20on%20their%20stability,%20intestinal%20transport,%20and%20in%20vitro%20bioavailability:%20A%20review&amp;author=B.%20Wang&amp;author=N.%20Xie&amp;author=B.%20Li&amp;volume=43&amp;publication_year=2019&amp;pages=e12571&amp;pmid=31353489&amp;doi=10.1111/jfbc.12571&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Sun B., Willard F. S., Feng D., Alsina-Fernandez J., Chen Q., Vieth M., Ho J. D., Showalter A. D., Stutsman C., Ding L.. et al. Structural
determinants
of dual incretin receptor agonism by tirzepatide. Proc. Natl. Acad. Sci. U.S.A. 2022;119:e2116506119.  doi: 10.1073/pnas.2116506119.</cite> [<a href="https://doi.org/10.1073/pnas.2116506119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9060465/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35333651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Structural%20determinants%20of%20dual%20incretin%20receptor%20agonism%20by%20tirzepatide&amp;author=B.%20Sun&amp;author=F.%20S.%20Willard&amp;author=D.%20Feng&amp;author=J.%20Alsina-Fernandez&amp;author=Q.%20Chen&amp;volume=119&amp;publication_year=2022&amp;pages=e2116506119&amp;pmid=35333651&amp;doi=10.1073/pnas.2116506119&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Pettersen E. F.. et al. UCSF Chimera--a visualization system for exploratory research and
analysis. J. Comput. Chem. 2004;25:1605–1612. doi: 10.1002/jcc.20084.</cite> [<a href="https://doi.org/10.1002/jcc.20084" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15264254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&amp;title=UCSF%20Chimera--a%20visualization%20system%20for%20exploratory%20research%20and%20analysis&amp;author=E.%20F.%20Pettersen&amp;volume=25&amp;publication_year=2004&amp;pages=1605-1612&amp;pmid=15264254&amp;doi=10.1002/jcc.20084&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Shen Y., Maupetit J., Derreumaux P., Tufféry P.. Improved PEP-FOLD
Approach for Peptide and Miniprotein Structure Prediction. J. Chem. Theory Comput. 2014;10:4745–4758. doi: 10.1021/ct500592m.</cite> [<a href="https://doi.org/10.1021/ct500592m" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26588162/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Theory%20Comput.&amp;title=Improved%20PEP-FOLD%20Approach%20for%20Peptide%20and%20Miniprotein%20Structure%20Prediction&amp;author=Y.%20Shen&amp;author=J.%20Maupetit&amp;author=P.%20Derreumaux&amp;author=P.%20Tuff%C3%A9ry&amp;volume=10&amp;publication_year=2014&amp;pages=4745-4758&amp;pmid=26588162&amp;doi=10.1021/ct500592m&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Thévenet P.. et al. PEP-FOLD:
an updated de novo structure prediction server for both
linear and disulfide bonded cyclic peptides. Nucleic Acids Res. 2012;40:W288–W293. doi: 10.1093/nar/gks419.</cite> [<a href="https://doi.org/10.1093/nar/gks419" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3394260/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22581768/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=PEP-FOLD:%20an%20updated%20de%20novo%20structure%20prediction%20server%20for%20both%20linear%20and%20disulfide%20bonded%20cyclic%20peptides&amp;author=P.%20Th%C3%A9venet&amp;volume=40&amp;publication_year=2012&amp;pages=W288-W293&amp;pmid=22581768&amp;doi=10.1093/nar/gks419&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref44">
<cite>Xu X., Yan C., Zou X.. MDockPeP:
An ab-initio protein-peptide docking server. J. Comput. Chem. 2018;39:2409–2413. doi: 10.1002/jcc.25555.</cite> [<a href="https://doi.org/10.1002/jcc.25555" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6226323/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30368849/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&amp;title=MDockPeP:%20An%20ab-initio%20protein-peptide%20docking%20server&amp;author=X.%20Xu&amp;author=C.%20Yan&amp;author=X.%20Zou&amp;volume=39&amp;publication_year=2018&amp;pages=2409-2413&amp;pmid=30368849&amp;doi=10.1002/jcc.25555&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref45">
<cite>Yan C., Xu X., Zou X.. Fully Blind
Docking at the Atomic Level for Protein-Peptide
Complex Structure Prediction. Structure. 2016;24:1842–1853. doi: 10.1016/j.str.2016.07.021.</cite> [<a href="https://doi.org/10.1016/j.str.2016.07.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5080282/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27642160/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Fully%20Blind%20Docking%20at%20the%20Atomic%20Level%20for%20Protein-Peptide%20Complex%20Structure%20Prediction&amp;author=C.%20Yan&amp;author=X.%20Xu&amp;author=X.%20Zou&amp;volume=24&amp;publication_year=2016&amp;pages=1842-1853&amp;pmid=27642160&amp;doi=10.1016/j.str.2016.07.021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref46">
<cite>Lemkul J. A.. From Proteins
to Perturbed Hamiltonians: A Suite of Tutorials for the GROMACS-2018
Molecular Simulation Package [Article v1.0] Living J. Comput. Mol. Sci. 2019;1:5068. doi: 10.33011/livecoms.1.1.5068.</cite> [<a href="https://doi.org/10.33011/livecoms.1.1.5068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Living%20J.%20Comput.%20Mol.%20Sci.&amp;title=From%20Proteins%20to%20Perturbed%20Hamiltonians:%20A%20Suite%20of%20Tutorials%20for%20the%20GROMACS-2018%20Molecular%20Simulation%20Package%20%5BArticle%20v1.0%5D&amp;author=J.%20A.%20Lemkul&amp;volume=1&amp;publication_year=2019&amp;pages=5068&amp;doi=10.33011/livecoms.1.1.5068&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref47">
<cite>Lee J.. et al. CHARMM-GUI Membrane
Builder for Complex Biological Membrane Simulations
with Glycolipids and Lipoglycans. J. Chem. Theory
Comput. 2019;15:775–786. doi: 10.1021/acs.jctc.8b01066.</cite> [<a href="https://doi.org/10.1021/acs.jctc.8b01066" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30525595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Theory%20Comput.&amp;title=CHARMM-GUI%20Membrane%20Builder%20for%20Complex%20Biological%20Membrane%20Simulations%20with%20Glycolipids%20and%20Lipoglycans&amp;author=J.%20Lee&amp;volume=15&amp;publication_year=2019&amp;pages=775-786&amp;pmid=30525595&amp;doi=10.1021/acs.jctc.8b01066&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref48">
<cite>Park S., Choi Y. K., Kim S., Lee J., Im W.. CHARMM-GUI
Membrane Builder for Lipid Nanoparticles with Ionizable Cationic Lipids
and PEGylated Lipids. J. Chem. Inf. Model. 2021;61:5192–5202. doi: 10.1021/acs.jcim.1c00770.</cite> [<a href="https://doi.org/10.1021/acs.jcim.1c00770" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8545881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34546048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Inf.%20Model.&amp;title=CHARMM-GUI%20Membrane%20Builder%20for%20Lipid%20Nanoparticles%20with%20Ionizable%20Cationic%20Lipids%20and%20PEGylated%20Lipids&amp;author=S.%20Park&amp;author=Y.%20K.%20Choi&amp;author=S.%20Kim&amp;author=J.%20Lee&amp;author=W.%20Im&amp;volume=61&amp;publication_year=2021&amp;pages=5192-5202&amp;pmid=34546048&amp;doi=10.1021/acs.jcim.1c00770&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref49">
<cite>Jo S., Kim T., Im W.. Automated builder and database of
protein/membrane
complexes for molecular dynamics simulations. PLoS One. 2007;2:e880.  doi: 10.1371/journal.pone.0000880.</cite> [<a href="https://doi.org/10.1371/journal.pone.0000880" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1963319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17849009/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Automated%20builder%20and%20database%20of%20protein/membrane%20complexes%20for%20molecular%20dynamics%20simulations&amp;author=S.%20Jo&amp;author=T.%20Kim&amp;author=W.%20Im&amp;volume=2&amp;publication_year=2007&amp;pages=e880&amp;pmid=17849009&amp;doi=10.1371/journal.pone.0000880&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref50">
<cite>Brooks B. R.. et al. CHARMM: the biomolecular
simulation program. J. Comput. Chem. 2009;30:1545–1614. doi: 10.1002/jcc.21287.</cite> [<a href="https://doi.org/10.1002/jcc.21287" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2810661/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19444816/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&amp;title=CHARMM:%20the%20biomolecular%20simulation%20program&amp;author=B.%20R.%20Brooks&amp;volume=30&amp;publication_year=2009&amp;pages=1545-1614&amp;pmid=19444816&amp;doi=10.1002/jcc.21287&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref51">
<cite>Kim S.. et al. CHARMM-GUI ligand reader and modeler for CHARMM force field generation
of small molecules. J. Comput. Chem. 2017;38:1879–1886. doi: 10.1002/jcc.24829.</cite> [<a href="https://doi.org/10.1002/jcc.24829" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5488718/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28497616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Comput.%20Chem.&amp;title=CHARMM-GUI%20ligand%20reader%20and%20modeler%20for%20CHARMM%20force%20field%20generation%20of%20small%20molecules&amp;author=S.%20Kim&amp;volume=38&amp;publication_year=2017&amp;pages=1879-1886&amp;pmid=28497616&amp;doi=10.1002/jcc.24829&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref52">
<cite>Sharma D., Arumugam S.. Pharmacophore-based identification and in Silico characterization
of microbial metabolites as potential modulators of Wnt signaling
pathway in colorectal cancer therapy. Mol. Diversity. 2025:1–26. doi: 10.1007/s11030-024-11103-4.</cite> [<a href="https://doi.org/10.1007/s11030-024-11103-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39921842/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Diversity&amp;title=Pharmacophore-based%20identification%20and%20in%20Silico%20characterization%20of%20microbial%20metabolites%20as%20potential%20modulators%20of%20Wnt%20signaling%20pathway%20in%20colorectal%20cancer%20therapy&amp;author=D.%20Sharma&amp;author=S.%20Arumugam&amp;publication_year=2025&amp;pages=1-26&amp;pmid=39921842&amp;doi=10.1007/s11030-024-11103-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref53">
<cite>

Wang, Z.
; 
Pan, H.
; 
Sun, H.
; 
Kang, Y.
; 
Liu, H.
; 
Cao, D.
; 
Hou, T.
. 
fastDRH:
a webserver to predict and analyze protein-ligand complexes based
on molecular docking and MM/PB­(GB)­SA computation. Briefings Bioinf.
2022, 23. 10.1093/bib/bbac201.</cite> [<a href="https://doi.org/10.1093/bib/bbac201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35580866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Briefings%20Bioinf.&amp;title=fastDRH:%20a%20webserver%20to%20predict%20and%20analyze%20protein-ligand%20complexes%20based%20on%20molecular%20docking%20and%20MM/PB%C2%AD(GB)%C2%ADSA%20computation&amp;volume=23&amp;publication_year=2022&amp;pmid=35580866&amp;doi=10.1093/bib/bbac201&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref54">
<cite>Buslaev P., Gordeliy V., Grudinin S., Gushchin I.. Principal
Component
Analysis of Lipid Molecule Conformational Changes in Molecular Dynamics
Simulations. J. Chem. Theory Comput. 2016;12:1019–1028. doi: 10.1021/acs.jctc.5b01106.</cite> [<a href="https://doi.org/10.1021/acs.jctc.5b01106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26765212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Theory%20Comput.&amp;title=Principal%20Component%20Analysis%20of%20Lipid%20Molecule%20Conformational%20Changes%20in%20Molecular%20Dynamics%20Simulations&amp;author=P.%20Buslaev&amp;author=V.%20Gordeliy&amp;author=S.%20Grudinin&amp;author=I.%20Gushchin&amp;volume=12&amp;publication_year=2016&amp;pages=1019-1028&amp;pmid=26765212&amp;doi=10.1021/acs.jctc.5b01106&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref55">
<cite>Palma J., Pierdominici-Sottile G.. On the Uses of PCA
to Characterise Molecular Dynamics
Simulations of Biological Macromolecules: Basics and Tips for an Effective
Use. ChemPhysChem. 2023;24:e202200491.  doi: 10.1002/cphc.202200491.</cite> [<a href="https://doi.org/10.1002/cphc.202200491" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36285677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ChemPhysChem&amp;title=On%20the%20Uses%20of%20PCA%20to%20Characterise%20Molecular%20Dynamics%20Simulations%20of%20Biological%20Macromolecules:%20Basics%20and%20Tips%20for%20an%20Effective%20Use&amp;author=J.%20Palma&amp;author=G.%20Pierdominici-Sottile&amp;volume=24&amp;publication_year=2023&amp;pages=e202200491&amp;pmid=36285677&amp;doi=10.1002/cphc.202200491&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref56">
<cite>Wilkins M. R.. et al. Protein identification and analysis tools in the ExPASy server. Methods Mol. Biol. 1998;112:531–552. doi: 10.1385/1-59259-584-7:531.</cite> [<a href="https://doi.org/10.1385/1-59259-584-7:531" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10027275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol.&amp;title=Protein%20identification%20and%20analysis%20tools%20in%20the%20ExPASy%20server&amp;author=M.%20R.%20Wilkins&amp;volume=112&amp;publication_year=1998&amp;pages=531-552&amp;pmid=10027275&amp;doi=10.1385/1-59259-584-7:531&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref57">
<cite>Prasoodanan
P K V., Sharma A. K., Mahajan S., Dhakan D. B., Maji A., Scaria J., Sharma V. K.. Western and non-western gut microbiomes
reveal new roles of Prevotella in carbohydrate metabolism and mouth-gut
axis. npj Biofilms Microbiomes. 2021;7:77. doi: 10.1038/s41522-021-00248-x.</cite> [<a href="https://doi.org/10.1038/s41522-021-00248-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8497558/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34620880/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=npj%20Biofilms%20Microbiomes&amp;title=Western%20and%20non-western%20gut%20microbiomes%20reveal%20new%20roles%20of%20Prevotella%20in%20carbohydrate%20metabolism%20and%20mouth-gut%20axis&amp;author=V.%20Prasoodanan%20P%20K&amp;author=A.%20K.%20Sharma&amp;author=S.%20Mahajan&amp;author=D.%20B.%20Dhakan&amp;author=A.%20Maji&amp;volume=7&amp;publication_year=2021&amp;pages=77&amp;pmid=34620880&amp;doi=10.1038/s41522-021-00248-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref58">
<cite>Hajishengallis G.. et al. Low-abundance biofilm species orchestrates
inflammatory periodontal
disease through the commensal microbiota and complement. Cell Host Microbe. 2011;10:497–506. doi: 10.1016/j.chom.2011.10.006.</cite> [<a href="https://doi.org/10.1016/j.chom.2011.10.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3221781/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22036469/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Host%20Microbe&amp;title=Low-abundance%20biofilm%20species%20orchestrates%20inflammatory%20periodontal%20disease%20through%20the%20commensal%20microbiota%20and%20complement&amp;author=G.%20Hajishengallis&amp;volume=10&amp;publication_year=2011&amp;pages=497-506&amp;pmid=22036469&amp;doi=10.1016/j.chom.2011.10.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref59">
<cite>Shin N.-R., Whon T. W., Bae J.-W. P.. Proteobacteria:
microbial signature
of dysbiosis in gut microbiota. Trends Biotechnol. 2015;33:496–503. doi: 10.1016/j.tibtech.2015.06.011.</cite> [<a href="https://doi.org/10.1016/j.tibtech.2015.06.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26210164/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Biotechnol.&amp;title=Proteobacteria:%20microbial%20signature%20of%20dysbiosis%20in%20gut%20microbiota&amp;author=N.-R.%20Shin&amp;author=T.%20W.%20Whon&amp;author=J.-W.%20P.%20Bae&amp;volume=33&amp;publication_year=2015&amp;pages=496-503&amp;pmid=26210164&amp;doi=10.1016/j.tibtech.2015.06.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref60">
<cite>Ju T., Kong J. Y., Stothard P., Willing B. P.. Defining the role
of Parasutterella, a previously uncharacterized member of the core
gut microbiota. ISME J. 2019;13:1520–1534. doi: 10.1038/s41396-019-0364-5.</cite> [<a href="https://doi.org/10.1038/s41396-019-0364-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6776049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30742017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ISME%20J.&amp;title=Defining%20the%20role%20of%20Parasutterella,%20a%20previously%20uncharacterized%20member%20of%20the%20core%20gut%20microbiota&amp;author=T.%20Ju&amp;author=J.%20Y.%20Kong&amp;author=P.%20Stothard&amp;author=B.%20P.%20Willing&amp;volume=13&amp;publication_year=2019&amp;pages=1520-1534&amp;pmid=30742017&amp;doi=10.1038/s41396-019-0364-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref61">
<cite>Chen Y.-J.. et al. Parasutterella, in association with irritable bowel
syndrome and
intestinal chronic inflammation. J. Gastroenterol.
Hepatol. 2018;33:1844–1852. doi: 10.1111/jgh.14281.</cite> [<a href="https://doi.org/10.1111/jgh.14281" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29744928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Gastroenterol.%20Hepatol.&amp;title=Parasutterella,%20in%20association%20with%20irritable%20bowel%20syndrome%20and%20intestinal%20chronic%20inflammation&amp;author=Y.-J.%20Chen&amp;volume=33&amp;publication_year=2018&amp;pages=1844-1852&amp;pmid=29744928&amp;doi=10.1111/jgh.14281&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref62">
<cite>Nagai F., Morotomi M., Sakon H., Tanaka R.. Parasutterella excrementihominis
gen. nov., sp. nov., a member of the family Alcaligenaceae isolated
from human faeces. Int. J. Syst. Evol. Microbiol. 2009;59:1793–1797. doi: 10.1099/ijs.0.002519-0.</cite> [<a href="https://doi.org/10.1099/ijs.0.002519-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19542131/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&amp;title=Parasutterella%20excrementihominis%20gen.%20nov.,%20sp.%20nov.,%20a%20member%20of%20the%20family%20Alcaligenaceae%20isolated%20from%20human%20faeces&amp;author=F.%20Nagai&amp;author=M.%20Morotomi&amp;author=H.%20Sakon&amp;author=R.%20Tanaka&amp;volume=59&amp;publication_year=2009&amp;pages=1793-1797&amp;pmid=19542131&amp;doi=10.1099/ijs.0.002519-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref63">
<cite>Danneskiold-Samsøe N. B., Andersen D., Radulescu I. D., Normann-Hansen A., Brejnrod A., Kragh M., Madsen T., Nielsen C., Josefsen K., Fretté X.. et al. A safflower oil based
high-fat/high-sucrose diet modulates the gut microbiota and liver
phospholipid profiles associated with early glucose intolerance in
the absence of tissue inflammation. Mol. Nutr.
Food Res. 2017;61:1600528. doi: 10.1002/mnfr.201600528.</cite> [<a href="https://doi.org/10.1002/mnfr.201600528" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28012235/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Nutr.%20Food%20Res.&amp;title=A%20safflower%20oil%20based%20high-fat/high-sucrose%20diet%20modulates%20the%20gut%20microbiota%20and%20liver%20phospholipid%20profiles%20associated%20with%20early%20glucose%20intolerance%20in%20the%20absence%20of%20tissue%20inflammation&amp;author=N.%20B.%20Danneskiold-Sams%C3%B8e&amp;author=D.%20Andersen&amp;author=I.%20D.%20Radulescu&amp;author=A.%20Normann-Hansen&amp;author=A.%20Brejnrod&amp;volume=61&amp;publication_year=2017&amp;pages=1600528&amp;pmid=28012235&amp;doi=10.1002/mnfr.201600528&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data supporting
the findings of this study are available from the corresponding author
upon reasonable request</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Omega are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsomega.5c03268"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/ao5c03268.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12332626/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12332626/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332626%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332626/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12332626/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12332626/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40787310/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12332626/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40787310/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12332626/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12332626/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="4sZ7ELFJQCLY6XR4QHGMWyaO9hDPueEPMGhiH1GBixIOmCPRjz3e6s3KIvTpBTLE">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
