
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Comparative Embryology and Transcriptomics of Asellus infernus, an Isopod Crustacean From Sulfidic Groundwater - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="FF20808589B71FB31A80850011025164.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326928/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Evolution &amp; Development">
<meta name="citation_title" content="Comparative Embryology and Transcriptomics of Asellus infernus, an Isopod Crustacean From Sulfidic Groundwater">
<meta name="citation_author" content="Haeli J Lomheim">
<meta name="citation_author_institution" content="Department of Natural Sciences and Mathematics, Dominican University of California, San Rafael, California, USA">
<meta name="citation_author_institution" content="Department of Biology, Georgetown University, Washington DC, Washington DC, USA">
<meta name="citation_author" content="Lizet Reyes Rodas">
<meta name="citation_author_institution" content="Department of Natural Sciences and Mathematics, Dominican University of California, San Rafael, California, USA">
<meta name="citation_author" content="Devon Price">
<meta name="citation_author_institution" content="Department of Natural Sciences and Mathematics, Dominican University of California, San Rafael, California, USA">
<meta name="citation_author" content="Serban M Sarbu">
<meta name="citation_author_institution" content="“Emil Racoviţă” Institute of Speleology, Department Cluj‐Napoca, Cluj‐Napoca, Romania">
<meta name="citation_author_institution" content="Department of Biological Sciences, California State University, Chico, California, USA">
<meta name="citation_author_institution" content="“Emil Racoviţă” Institute of Speleology, Romanian Academy, Bucharest, Romania">
<meta name="citation_author" content="Raluca I Băncilă">
<meta name="citation_author_institution" content="“Emil Racoviţă” Institute of Speleology, Department Cluj‐Napoca, Cluj‐Napoca, Romania">
<meta name="citation_author_institution" content="“Emil Racoviţă” Institute of Speleology, Romanian Academy, Bucharest, Romania">
<meta name="citation_author" content="Cody Carroll">
<meta name="citation_author_institution" content="Department of Mathematics and Statistics, Data Institute, University of San Francisco, San Francisco, California, USA">
<meta name="citation_author" content="Layla Freeborn">
<meta name="citation_author_institution" content="Research Computing, Office of Information Technology, The Center for Research Data and Digital Scholarship, University of Colorado Boulder, Boulder, Colorado, USA">
<meta name="citation_author" content="Sheri Sanders">
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, USA">
<meta name="citation_author" content="Meredith E Protas">
<meta name="citation_author_institution" content="Department of Natural Sciences and Mathematics, Dominican University of California, San Rafael, California, USA">
<meta name="citation_author_institution" content="Department of Biology, University of San Francisco, San Francisco, California, USA">
<meta name="citation_publication_date" content="2025 Aug 1">
<meta name="citation_volume" content="27">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70014">
<meta name="citation_doi" content="10.1111/ede.70014">
<meta name="citation_pmid" content="40746274">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326928/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326928/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326928/pdf/EDE-27-e70014.pdf">
<meta name="description" content="Sulfidic caves are harsh and extreme environments characterized by limited oxygen, low pH, and the presence of hydrogen sulfide. Amazingly, animals can live in sulfidic caves, one such animal being Asellus infernus, a representative of the Asellus ...">
<meta name="og:title" content="Comparative Embryology and Transcriptomics of Asellus infernus, an Isopod Crustacean From Sulfidic Groundwater">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Sulfidic caves are harsh and extreme environments characterized by limited oxygen, low pH, and the presence of hydrogen sulfide. Amazingly, animals can live in sulfidic caves, one such animal being Asellus infernus, a representative of the Asellus ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326928/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12326928">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1111/ede.70014"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/EDE-27-e70014.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12326928%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12326928/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12326928/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326928/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1111/ede.70014" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Evol Dev</button></div>. 2025 Aug 1;27(3):e70014. doi: <a href="https://doi.org/10.1111/ede.70014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1111/ede.70014</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Evol%20Dev%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Evol%20Dev%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Evol%20Dev%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Evol%20Dev%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Comparative Embryology and Transcriptomics of <em>Asellus infernus</em>, an Isopod Crustacean From Sulfidic Groundwater</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lomheim%20HJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Haeli J Lomheim</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Haeli J Lomheim</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Natural Sciences and Mathematics, Dominican University of California, San Rafael, California, USA</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Biology, Georgetown University, Washington DC, Washington DC, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lomheim%20HJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Haeli J Lomheim</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reyes%20Rodas%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Lizet Reyes Rodas</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Lizet Reyes Rodas</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Natural Sciences and Mathematics, Dominican University of California, San Rafael, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Reyes%20Rodas%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lizet Reyes Rodas</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Price%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Devon Price</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Devon Price</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Natural Sciences and Mathematics, Dominican University of California, San Rafael, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Price%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Devon Price</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sarbu%20SM%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Serban M Sarbu</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Serban M Sarbu</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>“Emil Racoviţă” Institute of Speleology, Department Cluj‐Napoca, Cluj‐Napoca, Romania</div>
<div class="p">
<sup>
<sup>4</sup>
</sup>Department of Biological Sciences, California State University, Chico, California, USA</div>
<div class="p">
<sup>
<sup>5</sup>
</sup>“Emil Racoviţă” Institute of Speleology, Romanian Academy, Bucharest, Romania</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sarbu%20SM%22%5BAuthor%5D" class="usa-link"><span class="name western">Serban M Sarbu</span></a>
</div>
</div>
<sup>3,</sup><sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22B%C4%83ncil%C4%83%20RI%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Raluca I Băncilă</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Raluca I Băncilă</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>“Emil Racoviţă” Institute of Speleology, Department Cluj‐Napoca, Cluj‐Napoca, Romania</div>
<div class="p">
<sup>
<sup>5</sup>
</sup>“Emil Racoviţă” Institute of Speleology, Romanian Academy, Bucharest, Romania</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22B%C4%83ncil%C4%83%20RI%22%5BAuthor%5D" class="usa-link"><span class="name western">Raluca I Băncilă</span></a>
</div>
</div>
<sup>3,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Carroll%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Cody Carroll</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Cody Carroll</span></h3>
<div class="p">
<sup>
<sup>6</sup>
</sup>Department of Mathematics and Statistics, Data Institute, University of San Francisco, San Francisco, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Carroll%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Cody Carroll</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Freeborn%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Layla Freeborn</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Layla Freeborn</span></h3>
<div class="p">
<sup>
<sup>7</sup>
</sup>Research Computing, Office of Information Technology, The Center for Research Data and Digital Scholarship, University of Colorado Boulder, Boulder, Colorado, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Freeborn%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Layla Freeborn</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sanders%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Sheri Sanders</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Sheri Sanders</span></h3>
<div class="p">
<sup>
<sup>8</sup>
</sup>Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sanders%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sheri Sanders</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Protas%20ME%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Meredith E Protas</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Meredith E Protas</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Natural Sciences and Mathematics, Dominican University of California, San Rafael, California, USA</div>
<div class="p">
<sup>
<sup>9</sup>
</sup>Department of Biology, University of San Francisco, San Francisco, California, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Protas%20ME%22%5BAuthor%5D" class="usa-link"><span class="name western">Meredith E Protas</span></a>
</div>
</div>
<sup>1,</sup><sup>9,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="ede70014-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Natural Sciences and Mathematics, Dominican University of California, San Rafael, California, USA</div>
<div id="ede70014-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Biology, Georgetown University, Washington DC, Washington DC, USA</div>
<div id="ede70014-aff-0003">
<sup>
<sup>3</sup>
</sup>“Emil Racoviţă” Institute of Speleology, Department Cluj‐Napoca, Cluj‐Napoca, Romania</div>
<div id="ede70014-aff-0004">
<sup>
<sup>4</sup>
</sup>Department of Biological Sciences, California State University, Chico, California, USA</div>
<div id="ede70014-aff-0005">
<sup>
<sup>5</sup>
</sup>“Emil Racoviţă” Institute of Speleology, Romanian Academy, Bucharest, Romania</div>
<div id="ede70014-aff-0006">
<sup>
<sup>6</sup>
</sup>Department of Mathematics and Statistics, Data Institute, University of San Francisco, San Francisco, California, USA</div>
<div id="ede70014-aff-0007">
<sup>
<sup>7</sup>
</sup>Research Computing, Office of Information Technology, The Center for Research Data and Digital Scholarship, University of Colorado Boulder, Boulder, Colorado, USA</div>
<div id="ede70014-aff-0008">
<sup>
<sup>8</sup>
</sup>Department of Biological Sciences, University of Notre Dame, Notre Dame, Indiana, USA</div>
<div id="ede70014-aff-0009">
<sup>
<sup>9</sup>
</sup>Department of Biology, University of San Francisco, San Francisco, California, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong> Meredith E. Protas (<span>mprotas@usfca.edu</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jun 30; Received 2023 Dec 31; Accepted 2025 Jul 1; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Evolution &amp; Development</em> published by Wiley Periodicals LLC.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12326928  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40746274/" class="usa-link">40746274</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>Sulfidic caves are harsh and extreme environments characterized by limited oxygen, low pH, and the presence of hydrogen sulfide. Amazingly, animals can live in sulfidic caves, one such animal being <em>Asellus infernus</em>, a representative of the <em>Asellus aquaticus</em> species complex, originating from Movile Cave and from old wells that represent windows of access to a sulfidic groundwater ecosystem located in southeast Romania. Little previous work has been done on lab‐reared populations of <em>A. infernus</em> as they have been historically difficult to raise in the lab. Here, we develop resources for <em>A. infernus</em>, examining questions of timing of morphological differences in cave versus surface individuals, whether the environment (lab‐bred vs. wild‐caught) influenced size characteristics, and the genes and pathways showing differential expression between cave and surface samples. We found that <em>A. infernus</em> did not develop pigmentation embryonically, and juveniles had increased body length and longer antenna II as compared to surface individuals. Furthermore, we found that some of these measures differed between wild‐caught and lab‐reared juveniles for a given population, indicating that environmental differences can also influence these size characteristics. In addition, differential expression between cave and surface samples and allele‐specific expression studies within F1 hybrids identified multiple genes, including those involved in sulfide metabolism and phototransduction. Strikingly, molecular convergence of genes involved in sulfide detoxification was observed between <em>A. infernus</em> and previous work on a fish that lives in both cave and sulfidic environments, <em>Poecilia mexicana</em>. In sum, we were able to develop embryonic and genomic tools for <em>A. infernus</em>, a model for understanding cave adaptation and adaptation to sulfidic environments.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> allele‐specific expression, cave, isopod, sulfidic water</p></section></section><section class="abstract" id="abstract2"><hr class="headless">
<p>Individuals of <em>Asellus infernus</em>, a sulfidic subterranean population of the <em>Asellus aquaticus</em> species complex from southeastern Romania (left) and surface individuals of <em>A. aquaticus</em> (right). Extreme differences in phenotype are seen comparing the subterranean and surface individuals as adults (top panels) and as embryonic samples (bottom panels).
</p>
<section class="bt xbox font-sm" id="boxed-text1"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/49af/12326928/6ba0706c9d6f/EDE-27-e70014-g006.jpg" loading="lazy" id="jats-graphic-1" height="451" width="600" alt="graphic file with name EDE-27-e70014-g006.jpg"></p></section></section><section class="abstract" id="abstract3"><h2>Summary</h2>
<ul class="list" style="list-style-type:disc">
<li><p>Development of <em>Asellus infernus</em> as a model for understanding adaptation to cave and sulfidic environments.</p></li>
<li><p>Comparative embryology analysis of <em>A. infernus</em> and a surface population of <em>Asellus aquaticus</em>.</p></li>
<li><p>Generation of cave and surface transcriptomes, differential and allele‐specific expression analysis.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list">
<dt>CAVE_kl</dt>
<dd><p>cave ecomorph from Romania, collected from Karaoban Lake</p></dd>
<dt>CAVE_mw</dt>
<dd><p>cave ecomorph from Romania, collected from drinking wells in Mangalia</p></dd>
<dt>SURF_li</dt>
<dd><p>surface ecomorph from Romania, Limanu Bridge Stream population</p></dd>
<dt>SURF_tb</dt>
<dd><p>surface ecomorph from Romania, Turkish Bath population</p></dd>
</dl>
<section id="ede70014-sec-0010"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Caves, considered extreme environments, are characterized by a lack of light, limited nutrient availability, and in some cases, extreme temperatures as compared to surface environments (Engel <a href="#ede70014-bib-0021" class="usa-link" aria-describedby="ede70014-bib-0021">2007</a>). A specific type of cave, sulfidic cave, is further characterized by high hydrogen sulfide (H<sub>2</sub>S), carbon dioxide (CO<sub>2</sub>), methane (CH<sub>4</sub>), ammonia (NH<sub>4</sub>
<sup>+</sup>), and acidity levels. In addition, the temperature can be relatively high, such as 21°C. Copious food produced in situ by chemoautotrophic bacteria thriving in the sulfidic groundwater ecosystems can support rich and diverse invertebrate communities.</p>
<p>Experimental studies of cave animals are not very numerous, but even more limited are studies of sulfidic cave‐dwelling animals. This is likely because sulfidic caves are rare, and they are typically hostile environments that are difficult to enter (Engel <a href="#ede70014-bib-0021" class="usa-link" aria-describedby="ede70014-bib-0021">2007</a>; Sarbu et al. <a href="#ede70014-bib-0076" class="usa-link" aria-describedby="ede70014-bib-0076">2024</a>). Furthermore, even if collection is successful, reproducing a given sulfidic environment in a lab setting such that the animals can be worked on is even more challenging. The majority of studies of sulfidic populations have been either morphological, taxonomic, or phylogenetic, as samples are difficult to acquire, and the species are not well suited for laboratory‐based studies (Engel <a href="#ede70014-bib-0021" class="usa-link" aria-describedby="ede70014-bib-0021">2007</a>). Despite these challenges, <em>Poecilia mexicana</em>, a cave fish that inhabits multiple environments, including surface waters, caves, and sulfidic waters, can be used to compare sulfidic cave populations versus non‐sulfidic cave populations (Tobler et al. <a href="#ede70014-bib-0084" class="usa-link" aria-describedby="ede70014-bib-0084">2018</a>). Cave populations showed less pigmentation and smaller eyes, whereas sulfidic populations showed larger heads, larger gill filaments, and differences in brain anatomy (Tobler et al. <a href="#ede70014-bib-0083" class="usa-link" aria-describedby="ede70014-bib-0083">2008</a>; Schulz‐Mirbach et al. <a href="#ede70014-bib-0078" class="usa-link" aria-describedby="ede70014-bib-0078">2016</a>). Furthermore, both sulfidic environments and cave environments were associated with fewer but larger offspring (Riesch et al. <a href="#ede70014-bib-0071" class="usa-link" aria-describedby="ede70014-bib-0071">2010</a>). In addition to morphological and life history studies, transcriptomic studies of cave and surface populations, both sulfidic and non‐sulfidic, have also been performed for <em>P. mexicana</em> (Kelley et al. <a href="#ede70014-bib-0039" class="usa-link" aria-describedby="ede70014-bib-0039">2012</a>; Passow, Henpita, et al. <a href="#ede70014-bib-0063" class="usa-link" aria-describedby="ede70014-bib-0063">2017</a>; Passow, Brown et al. <a href="#ede70014-bib-0062" class="usa-link" aria-describedby="ede70014-bib-0062">2017</a>; McGowan et al. <a href="#ede70014-bib-0056" class="usa-link" aria-describedby="ede70014-bib-0056">2019</a>). An additional genus with sulfidic populations that have been investigated is the amphipod <em>Niphargus</em> (Flot et al. <a href="#ede70014-bib-0024" class="usa-link" aria-describedby="ede70014-bib-0024">2010</a>; Fišer et al. <a href="#ede70014-bib-0022" class="usa-link" aria-describedby="ede70014-bib-0022">2015</a>).</p>
<p>Another species that has the potential to shed insight into the biology of sulfide‐adapted animals is <em>Asellus aquaticus</em>, a pan‐European freshwater isopod crustacean with both surface and cave forms or ecomorphs. This represents an emerging model for studies of eco‐evo as well as evo‐devo biology (Protas and Jeffery <a href="#ede70014-bib-0065" class="usa-link" aria-describedby="ede70014-bib-0065">2012</a>; Lafuente et al. <a href="#ede70014-bib-0042" class="usa-link" aria-describedby="ede70014-bib-0042">2021</a>; Protas et al. <a href="#ede70014-bib-0066" class="usa-link" aria-describedby="ede70014-bib-0066">2023</a>). There are multiple cave populations of the <em>A. aquaticus</em> species complex, including one known population from Movile Cave, a sulfidic cave in southeastern Romania. Much work has been done on the phylogeography of these populations (including both sulfidic and non‐sulfidic cave populations) as well as comparative morphology of adult specimens (Verovnik et al. <a href="#ede70014-bib-0088" class="usa-link" aria-describedby="ede70014-bib-0088">2004</a>; Konec et al. <a href="#ede70014-bib-0040" class="usa-link" aria-describedby="ede70014-bib-0040">2015</a>; Bakovic et al. <a href="#ede70014-bib-0001" class="usa-link" aria-describedby="ede70014-bib-0001">2021</a>), identifying at least four clades and sets of characteristics common to many cave populations, including eye loss, pigment loss, and increased relative length of antenna II (the longer of the two antennae in isopods). Laboratory studies involving breeding of different populations have mostly focused on populations from Slovenia, specifically the Old Subterranean Pivka and Subterranean Rak populations from Postojna‐Planina Cave System (PPCS), Sweden, and Croatia, all of which are non‐sulfidic caves (Protas et al. <a href="#ede70014-bib-0067" class="usa-link" aria-describedby="ede70014-bib-0067">2011</a>; Re et al. <a href="#ede70014-bib-0069" class="usa-link" aria-describedby="ede70014-bib-0069">2018</a>; Bakovic et al. <a href="#ede70014-bib-0001" class="usa-link" aria-describedby="ede70014-bib-0001">2021</a>; Lukić et al. <a href="#ede70014-bib-0050" class="usa-link" aria-describedby="ede70014-bib-0050">2024</a>). Genomic information exists from non‐sulfidic cave populations including a genetic map generated from the Old Subterranean Pivka population, a draft genome generated from the Swedish Lummelunda Cave population, and an unannotated genome recently published from a surface individual from Great Britain (Protas et al. <a href="#ede70014-bib-0067" class="usa-link" aria-describedby="ede70014-bib-0067">2011</a>; Bakovic et al. <a href="#ede70014-bib-0001" class="usa-link" aria-describedby="ede70014-bib-0001">2021</a>; Thomas and University of Oxford and Wytham Woods Genome Acquisition Lab, Natural History Museum Genome Acquisition Lab, Darwin Tree of Life Barcoding collective, Wellcome Sanger Institute Tree of Life Management, Samples and Laboratory team, Wellcome Sanger Institute Scientific Operations: Sequencing Operations, Wellcome Sanger Institute Tree of Life Core Informatics team, Tree of Life Core Informatics collective and Darwin Tree of Life Consortium <a href="#ede70014-bib-0082" class="usa-link" aria-describedby="ede70014-bib-0082">2025</a>). In addition, genomic regions responsible for eye and pigmentation phenotypes were mapped (Protas et al. <a href="#ede70014-bib-0067" class="usa-link" aria-describedby="ede70014-bib-0067">2011</a>; Bakovic et al. <a href="#ede70014-bib-0001" class="usa-link" aria-describedby="ede70014-bib-0001">2021</a>). Comparative embryological studies were done on the Subterranean Rak population (Mojaddidi et al. <a href="#ede70014-bib-0057" class="usa-link" aria-describedby="ede70014-bib-0057">2018</a>), finding that cave embryos were larger and never developed pigmentation. Transcriptomic studies were performed on the Old Subterranean Pivka and Subterranean Rak populations (Stahl et al. <a href="#ede70014-bib-0081" class="usa-link" aria-describedby="ede70014-bib-0081">2015</a>; Gross et al. <a href="#ede70014-bib-0032" class="usa-link" aria-describedby="ede70014-bib-0032">2019</a>; Lomheim et al. <a href="#ede70014-bib-0047" class="usa-link" aria-describedby="ede70014-bib-0047">2023</a>) and on the Molnár János Cave population (Pérez‐Moreno et al. <a href="#ede70014-bib-0064" class="usa-link" aria-describedby="ede70014-bib-0064">2018</a>), identifying many candidate genes involved in pigmentation and phototransduction. Though many tools were generated for different cave populations of <em>A. aquaticus</em>, the sulfidic cave population has been historically difficult to work with, and few tools have been generated.</p>
<p>A troglomorphic population of the <em>A. aquaticus</em> species complex inhabits a sulfidic groundwater ecosystem in southeastern Romania. The discovery of Movile Cave in 1986 near the town of Mangalia, close to the shore of the Black Sea, provided a subterranean window of access to this mesothermal (21°C) aquifer that occupies a surface area of 50–100 km<sup>2</sup>. Of the 51 invertebrate species that inhabit this unusually rich and diverse cave ecosystem, 37 are endemic (Sarbu et al. <a href="#ede70014-bib-0077" class="usa-link" aria-describedby="ede70014-bib-0077">2019</a>; Brad et al. <a href="#ede70014-bib-0011" class="usa-link" aria-describedby="ede70014-bib-0011">2021</a>). One of these species, originally called <em>A. aquaticus infernus</em> (Turk‐Prevorčnik et al. <a href="#ede70014-bib-0087" class="usa-link" aria-describedby="ede70014-bib-0087">1998</a>), has been found in Movile Cave as well as in several sulfidic springs and old drinking wells in the town of Mangalia. In agreement with the results by Konec et al. (<a href="#ede70014-bib-0040" class="usa-link" aria-describedby="ede70014-bib-0040">2015</a>), it was proposed by Protas et al. (<a href="#ede70014-bib-0066" class="usa-link" aria-describedby="ede70014-bib-0066">2023</a>) that the population of the Mangalia sulfidic groundwater ecosystem was a good species and taxonomic status would be changed to: <em>Asellus infernu</em>s (Turk‐Prevorčnik et al. <a href="#ede70014-bib-0087" class="usa-link" aria-describedby="ede70014-bib-0087">1998</a>). Note that we categorize <em>A. infernus</em> as a cave ecomorph because the individuals are depigmented and eyeless. However, <em>A. infernus</em> can be found in multiple subterranean environments, not all of which are caves, such as wells and upwellings from underground environments in springs or lakes. Comparative morphology studies have shown that <em>A. infernus</em> has extreme phenotypes of eye and pigment loss similar to those seen in populations from PPCS in Slovenia (Turk‐Prevorčnik et al. <a href="#ede70014-bib-0087" class="usa-link" aria-describedby="ede70014-bib-0087">1998</a>). In addition, in adults, relative antenna II length is greater in the cave population as compared to the surface population (Konec et al. <a href="#ede70014-bib-0040" class="usa-link" aria-describedby="ede70014-bib-0040">2015</a>). Furthermore, the size of wild‐caught adults of <em>A. infernus</em> was larger than the size of wild‐caught adults from a nearby surface population (Herczeg et al. <a href="#ede70014-bib-0035" class="usa-link" aria-describedby="ede70014-bib-0035">2023</a>). Despite similarities with the populations from PPCS in Slovenia in eye and pigment morphology, multiple other characters have shown differences between PPCS Slovenian populations and <em>A. infernus</em> (Konec et al. <a href="#ede70014-bib-0040" class="usa-link" aria-describedby="ede70014-bib-0040">2015</a>). Furthermore, comparative phylogeography has shown that the Romanian populations are in a separate phylogenetic group distinct from the Slovenian cave populations (Konec et al. <a href="#ede70014-bib-0040" class="usa-link" aria-describedby="ede70014-bib-0040">2015</a>). <em>A. infernus</em> has been historically difficult to acquire and work with, and little has been done that requires laboratory rearing or access to embryonic samples of these populations aside from a recent study generating F1 hybrids from this population to study the origin of an orange phenotype revealed within surface populations and within crosses between surface and cave populations (Rodas et al. <a href="#ede70014-bib-0074" class="usa-link" aria-describedby="ede70014-bib-0074">2023</a>). Before the work described here, methods of working with and raising embryos outside of the female were not developed for <em>A. infernus</em>, which is one of the first steps critical in the development of a species as an evo‐devo model. In addition, no sequencing information existed for this unique population.</p>
<p>Here, we established <em>A. infernus</em> as a model for understanding adaptations to sulfidic and cave environments, investigated the timeframe that morphological differences between cave and surface ecomorphs are established, examined the interplay between genetic and environmental inputs into morphological differences, and identified genes and pathways that might be different between this sulfidic cave population and the nearby surface population. Specifically, we asked the following questions:
</p>
<ul class="list" style="list-style-type:disc">
<li><p>Are differences in lack of pigmentation and body size between cave and surface ecomorphs established by the end of embryonic development?</p></li>
<li><p>Do wild‐caught and lab‐bred individuals show differences in morphological characteristics?</p></li>
<li><p>Are genes and pathways involved in phototransduction, pigmentation, and sulfide detoxification differentially expressed and/or show allele‐specific expression between cave and surface samples?</p></li>
</ul>
<p>To address these questions, we performed comparative embryology/juvenile analysis between cave and surface individuals (both wild‐caught and lab‐bred) of <em>A. infernus</em> and a nearby surface population. Second, we generated transcriptomes from the cave and surface populations and performed differential expression and allele‐specific expression studies to examine genes and pathways that were different in cave and surface samples.</p></section><section id="ede70014-sec-0020"><h2 class="pmc_sec_title">2. Methods</h2>
<section id="ede70014-sec-0030"><h3 class="pmc_sec_title">2.1. Animals</h3>
<p>In the following population codes, CAVE refers to the cave ecomorph and SURF to the surface ecomorph. The two‐letter code after the underscore is an acronym for the collection location. The cave ecomorph, <em>A. infernus</em>, was collected from old drinking wells within the town of Mangalia in May 2021 (individuals from both wells were pooled and abbreviated CAVE_mw) and also from Karaoban Lake in August 2015 (abbreviated CAVE_kl). The wells are located near each other (0.5 km), but Karaoban Lake is located 2.7 km away from the wells. The wells represent a habitat that is fed by sulfidic water through fractures in the bedrock. Theoretically, fauna could move from location to location, but might be limited if the conduits lack dissolved O<sub>2</sub> or are not large enough. Therefore, it is possible that there might be some cryptic diversity within the cave ecomorph. However, the samples that were used for the gene expression analyses (detailed below) were from the two wells; they are likely to be part of the same breeding population because of their close geographical location, though molecular analysis comparing individuals from different wells has not been performed. Surface individuals of <em>A. aquaticus</em> were collected from two nearby locations, Turkish Bath (July 2018; SURF_tb) and Limanu Bridge Stream (May 2021; SURF_li), both of which do not have sulfidic waters (Table <a href="#ede70014-suppl-0001" class="usa-link">S1</a>). These two locations are located at a distance of 2.1 km on the shore of the same pond and are therefore likely to be part of the same breeding population. Animals of each ecomorph were collected from multiple, but nearby, locations because individuals were not found at the previous location at different collection times. In the methods, we refer to the animals by the code above, so it is clear which populations were used for which assays, but in the “Results and Discussion,” we use cave or <em>A. infernus</em> to include both CAVE_mw and CAVE_kl and surface to include both SURF_tb or SURF_li. Note that both <em>A. infernus</em> and <em>A. aquaticus</em> are part of the <em>A. aquaticus</em> species complex (Protas et al. <a href="#ede70014-bib-0066" class="usa-link" aria-describedby="ede70014-bib-0066">2023</a>). For the populations we examined in this study, <em>A. aquaticus</em> refers to the Romanian surface population. However, there are subterranean populations of <em>A. aquaticus</em> from other locations, for example, in Slovenia.</p>
<p>We raised animals similarly to what was previously described (Protas et al. <a href="#ede70014-bib-0067" class="usa-link" aria-describedby="ede70014-bib-0067">2011</a>), but we added algae pellets instead of decaying leaves as food, as algae pellets are easier to obtain and have less potential of introducing biological contaminants. Note that these conditions are very different than the animals' natural conditions, as we gradually changed the water that the animals were collected into synthetic freshwater (Protas et al. <a href="#ede70014-bib-0067" class="usa-link" aria-describedby="ede70014-bib-0067">2011</a>) for both ecomorphs and did not keep the cave ecomorph in sulfidic water conditions. In addition, the food source of algae pellets was also very different than the natural food source, particularly of the cave ecomorph. Individuals from the parental populations, CAVE_mw, CAVE_kl, SURF_tb, and SURF_li, were raised separately in plastic containers (700 mL capacity) containing around 10–15 individuals. When juveniles from wild‐caught individuals were found within a tank, they were separated into their own tank and allowed to grow to adulthood.</p></section><section id="ede70014-sec-0040"><h3 class="pmc_sec_title">2.2. Generation of F1 Hybrids</h3>
<p>We set up F1 hybrid crosses by placing a CAVE_mw male and 2–3 SURF_li or SURF_tb females in a single container. Three F1 hybrid crosses were used for the transcriptomic samples described below, and five additional F1 hybrid broods were used for the juvenile comparisons. F1 hybrid crosses in the other direction (CAVE_mw females and SURF_li or SURF_tb males) were also set up, but all failed to produce offspring. CAVE_mw females very infrequently generate embryos in the laboratory, even with males from the same population, so the lack of successful crosses between CAVE_mw females and SURF_li males is likely not due to a barrier or incompatibility but rather to the low success of CAVE_mw females reproducing in the lab. Ovigerous SURF_li or SURF_tb females with hybrid offspring were separated, and F1 hybrid embryos (SURF_li_CAVE_mw_F1) were removed as described below.</p></section><section id="ede70014-sec-0050"><h3 class="pmc_sec_title">2.3. Comparative Embryology</h3>
<section id="ede70014-sec-0060"><h4 class="pmc_sec_title">2.3.1. Embryo Collection and Imaging</h4>
<p>We anaesthetized females with embryos in a solution containing 50 mL of synthetic freshwater (Protas et al. <a href="#ede70014-bib-0067" class="usa-link" aria-describedby="ede70014-bib-0067">2011</a>) and 20 µL of clove oil. Then, each female was washed twice in synthetic freshwater, and the embryos were removed by opening the brood pouch gently with forceps and flushing out the embryos with a glass pipette. The embryos were moved into a new Petri dish with synthetic freshwater and kept at 12°C. Embryos were monitored daily and photographed using a Leica S8 Apo Microscope. Embryos that were used for RNAseq were harvested at either 70% through embryonic development, in which the embryos have a comma shape and will shortly show eye pigmentation in the surface form, or 90% of the way through embryonic development, when eye and head pigmentation is visible and the body of the embryo is elongated (Mojaddidi et al. <a href="#ede70014-bib-0057" class="usa-link" aria-describedby="ede70014-bib-0057">2018</a>).</p>
<p>We collected embryos from gravid CAVE_mw females upon arrival in the lab or females that mated immediately after arrival to the lab and then raised the embryos in vitro as described above. SURF_li females mated during transport to the lab or immediately after arrival at the lab. We then removed the embryos and raised them in vitro. We called these sets of embryos “wild‐caught” as they were from wild‐caught females that either mated in the wild or immediately upon arrival to the lab. Our initial comparisons of cave and surface juveniles and embryonic development used these wild‐caught embryos because we did not know whether the cave animals would breed in the laboratory environment. Fortunately, we were able to raise juveniles to adulthood in the lab and generate gravid females from these lab‐reared juveniles for both CAVE_mw and SURF_li populations. We labeled juveniles from these lab‐reared embryos as “lab‐bred.” F1 hybrid embryos, generated as detailed above, could not be classified as either wild‐caught or lab‐bred as they were generated from wild‐caught cave males and either wild‐caught surface females or lab‐bred surface females around a month after the animals were brought into the lab from the wild. We examined pigmentation as a presence versus absence trait in all embryos. Ideally, we would also have examined the presence versus absence of eyes (ommatidia) in embryonic development, but this requires visualization methods that often result in embryo death, and our sample sizes were too limited to successfully view enough individuals.</p></section><section id="ede70014-sec-0070"><h4 class="pmc_sec_title">2.3.2. Comparative Juvenile Analysis</h4>
<p>For the comparative juvenile analysis, our goal was to harvest juveniles when they were first able to walk and had the appearance of miniature adults. We selected this timepoint as it is easier to flatten individuals for measurements when their appendages can be easily removed. Embryos were observed every day, and once they were able to walk, they were harvested in 100% ethanol and placed at −20°C until mounting on microscopic slides. Juveniles that were selected for mounting had at least one intact antenna II. As individuals were raised in a dish and were harvested right when they were able to walk, antennae were generally intact. To ensure that juveniles could be flattened in ethanol, all pereopods were removed under a Leica S8 Apo stereomicroscope. Some individuals also had to have their mouth parts removed to allow further flattening. After limb removal, the juvenile was washed in 50% ethanol, 100% Milli‐Q water, and 50% glycerol, and then placed on a microscope slide with the dorsal side up in 100% glycerol. On the left‐ and right‐hand sides of the juvenile, a coverslip was placed with a single drop of glycerol to create a shallow wall around the juvenile so that when a coverslip was placed over the juvenile, the third coverslip rested on top of the first two coverslips and did not crush the juvenile. Juveniles were measured using LAS Core Software (Leica, Wetzlar, Germany) on a Leica S8 Apo stereomicroscope. Antenna II length (left and right if available), head width, and body length were measured. Five groups of samples were measured CAVE_mw_wild samples, SURF_li_wild samples, CAVE_mw_lab samples, SURF_li_lab samples, and SURF_li_CAVE_mw_F1 hybrid samples (Supporting Information File <a href="#ede70014-suppl-0002" class="usa-link">1</a> and Table <a href="#ede70014-suppl-0001" class="usa-link">S2</a>). Each juvenile was photographed and measured separately by two different people using LAS Core Software. The following measures were analyzed: left and right antenna II length, body length, and head width. The measurements from the two different people were averaged. Again, pigmentation was examined as a presence versus absence trait in all embryos.</p></section><section id="ede70014-sec-0080"><h4 class="pmc_sec_title">2.3.3. Statistical Analysis</h4>
<p>First, we performed the Shapiro–Wilk test in R 4.1.3 (R Core Team <a href="#ede70014-bib-1068" class="usa-link" aria-describedby="ede70014-bib-1068">2022</a>) to assess normality of measurements. Measures which did not deviate significantly from normality included antenna II lengths (left and right), body length, and head width. For these measures, ANOVA and Tukey's Honestly Significant Difference (HSD) tests for pairwise comparison were performed for each group. To address questions of allometry, we also performed principal components analysis on the measurements of body length, head width, and left and right antennae length, followed by an ANOVA and Tukey's HSD test on the resulting first and second principal component scores across groups. The Bonferroni correction for multiple testing was performed for tests of comparative juvenile analysis.</p></section></section><section id="ede70014-sec-0090"><h3 class="pmc_sec_title">2.4. Comparative Transcriptomics</h3>
<section id="ede70014-sec-0100"><h4 class="pmc_sec_title">2.4.1. Samples</h4>
<p>One brood of CAVE_kl embryos and one brood of SURF_tb embryos were prepared for Iso‐seq to obtain full‐length transcripts with the goal of combining this sequence ultimately with Illumina sequence to obtain more complete transcriptomes (Table <a href="#ede70014-tbl-0002" class="usa-link">2</a>). RNA extraction was performed as described (Lomheim et al. <a href="#ede70014-bib-0047" class="usa-link" aria-describedby="ede70014-bib-0047">2023</a>), and the sequencing was performed on a PacBio machine by the Functional Genomics Lab, Vincent C. Coates Genomics Sequencing Laboratory, and UC Berkeley. For Illumina sequencing, samples were collected more recently, and individuals at the SURF_tb and CAVE_kl locations were not present, so nearby populations were used instead, CAVE_mw and SURF_li. Three CAVE_mw broods and three SURF_li broods at two different developmental timepoints, that is, mid‐stage (ca. 70%) through embryonic development and late stage (ca. 90%) through embryonic development, were harvested. Additionally, three broods of late‐stage F1 hybrids (SURF_li_CAVE_mw_F1) were harvested (Table <a href="#ede70014-suppl-0001" class="usa-link">S3</a>). Each of these samples consisted of a single brood of siblings. After harvesting, all broods were homogenized in 200 µL of TRIzol (Thermofisher, Waltham, Massachusetts, the United States). Sequencing libraries were prepared for each of these samples: three CAVE_mw mid‐stage, three CAVE_mw late‐stage, three SURF_li mid‐stage, three SURF_li late‐stage, and three late‐stage SURF_li_CAVE_mw_F1. RNA was extracted using the RNeasy Plus Universal Mini Kit by the Genetic Epidemiology and Genomics Lab (GEGL), UC Berkeley. PolyA selection was performed, and libraries were constructed using the Nugen Kit low‐input protocol. Sequencing was performed on a HiSeq. 4000 machine with 150 bp paired‐end reads by the Functional Genomics Lab, Vincent C. Coates Genomics Sequencing Laboratory, and UC Berkeley.</p>
<section class="tw xbox font-sm" id="ede70014-tbl-0002"><h5 class="obj_head">Table 2.</h5>
<div class="caption p"><p>Quast and BUSCO statistics for cave and surface transcriptomes.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1">
<col align="center" style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1">
<col align="center" style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000" valign="bottom">
<th align="left" valign="bottom" rowspan="1" colspan="1"></th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Surface</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Cave</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1"># contigs (≥ 0 bp)</td>
<td align="center" valign="top" rowspan="1" colspan="1">80,182</td>
<td align="center" valign="top" rowspan="1" colspan="1">88,614</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1"># contigs (≥ 1000 bp)</td>
<td align="center" valign="top" rowspan="1" colspan="1">20,989</td>
<td align="center" valign="top" rowspan="1" colspan="1">20,380</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Largest contig</td>
<td align="center" valign="top" rowspan="1" colspan="1">46,211</td>
<td align="center" valign="top" rowspan="1" colspan="1">45,558</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Total length (≥ 0 bp)</td>
<td align="center" valign="top" rowspan="1" colspan="1">82,452,877</td>
<td align="center" valign="top" rowspan="1" colspan="1">88,155,484</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Total length (≥ 1000 bp)</td>
<td align="center" valign="top" rowspan="1" colspan="1">59,944,846</td>
<td align="center" valign="top" rowspan="1" colspan="1">61,449,113</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">N50</td>
<td align="center" valign="top" rowspan="1" colspan="1">2935</td>
<td align="center" valign="top" rowspan="1" colspan="1">3088</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GC (%)</td>
<td align="center" valign="top" rowspan="1" colspan="1">37.18</td>
<td align="center" valign="top" rowspan="1" colspan="1">36.99</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1"># Ns per 100 kbp</td>
<td align="center" valign="top" rowspan="1" colspan="1">1103.3</td>
<td align="center" valign="top" rowspan="1" colspan="1">1011.18</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Complete BUSCOs (%)</td>
<td align="center" valign="top" rowspan="1" colspan="1">92.9%</td>
<td align="center" valign="top" rowspan="1" colspan="1">93.4%</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Complete and single‐copy BUSCOs (%)</td>
<td align="center" valign="top" rowspan="1" colspan="1">88.8%</td>
<td align="center" valign="top" rowspan="1" colspan="1">89.6%</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Complete and duplicated BUSCOs (%)</td>
<td align="center" valign="top" rowspan="1" colspan="1">4.1%</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.8%</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Fragmented BUSCOs (%)</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.0%</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.8%</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Missing BUSCOs (%)</td>
<td align="center" valign="top" rowspan="1" colspan="1">6.1%</td>
<td align="center" valign="top" rowspan="1" colspan="1">5.8%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ede70014-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="ede70014-sec-0110"><h4 class="pmc_sec_title">2.4.2. De novo transcriptome assembly and annotation</h4>
<p>Two transcriptomes were made, one SURF_li transcriptome and one CAVE_mw transcriptome. First, Iso‐seq sequences were run through the in‐house Iso‐seq pipeline of the Functional Genomics Lab, Vincent C. Coates Genomics Sequencing Laboratory, and UC Berkeley, which included the synthesis of CCS sequences, demultiplexing of sequences, primer polyA and concatamer removal, clustering, and polishing. The final transcripts were combined with the assemblies generated from Illumina reads. We used the same methods as in a previous study by Lomheim et al. (<a href="#ede70014-bib-0047" class="usa-link" aria-describedby="ede70014-bib-0047">2023</a>). Briefly, for the Illumina samples, the reads were trimmed with Trimmomatic using the following parameters: sliding window 4:24, headcrop 10, avgqual 30, and minlen 30. Then, the transcriptomes were generated using the de‐novo‐transcriptome‐assembly pipeline of the National Center for Genome Analysis Support (NCGAS) (<a href="https://github.com/NCGAS/de-novo-transcriptome-assembly-pipeline" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/NCGAS/de-novo-transcriptome-assembly-pipeline</a>), which incorporates multiple assemblers: SOAP version 1.03 (kmer 35, 45, 55, 65, 75, and 85; Xie et al. <a href="#ede70014-bib-0090" class="usa-link" aria-describedby="ede70014-bib-0090">2014</a>), TransAbyss version 2.0.1 (kmer 35, 45, 55, 65, 75, and 85; Robertson et al. <a href="#ede70014-bib-0073" class="usa-link" aria-describedby="ede70014-bib-0073">2010</a>), Trinity 2.11.0 using default parameters (Grabherr et al. <a href="#ede70014-bib-0029" class="usa-link" aria-describedby="ede70014-bib-0029">2011</a>), and Velvet 1.2.10 (kmer 35, 45, 55, 65, 75, and 85; Zerbino and Birney <a href="#ede70014-bib-0092" class="usa-link" aria-describedby="ede70014-bib-0092">2008</a>). All the assemblies plus the Iso‐seq sequences were run through Evidential Gene's tr2aacds pipeline (Gilbert <a href="#ede70014-bib-1029" class="usa-link" aria-describedby="ede70014-bib-1029">2013</a>). Recently, a genome was published for an individual of <em>A. aquaticus</em> from Wytham Woods, Great Britian (Thomas and University of Oxford and Wytham Woods Genome Acquisition Lab, Natural History Museum Genome Acquisition Lab, Darwin Tree of Life Barcoding collective, Wellcome Sanger Institute Tree of Life Management, Samples and Laboratory team, Wellcome Sanger Institute Scientific Operations: Sequencing Operations, Wellcome Sanger Institute Tree of Life Core Informatics team, Tree of Life Core Informatics collective and Darwin Tree of Life Consortium <a href="#ede70014-bib-0082" class="usa-link" aria-describedby="ede70014-bib-0082">2025</a>). We mapped the de novo transcriptomes to the published genome in quality control (minimap2, Li <a href="#ede70014-bib-0046" class="usa-link" aria-describedby="ede70014-bib-0046">2018</a>), but no annotation was published with this genome. As a result, once transcriptomes were generated, we identified homologous sequences to the <em>Tribolium castaneum</em> annotation from 2019 (file entitled GCF_000002335.3_Tcas5.2_protein.faa) <a href="https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/002/335/GCF_000002335.3_Tcas5.2/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp.ncbi.nlm.nih.gov/genomes/all/GCF/000/002/335/GCF_000002335.3_Tcas5.2/</a> using Blast2GO (Conesa et al. <a href="#ede70014-bib-0019" class="usa-link" aria-describedby="ede70014-bib-0019">2005</a>), allowing us to transfer annotation information from <em>Tribolium</em> to our de novo transcriptomes. Transcriptomes were evaluated using Galaxy Version 2.10.1+galaxy2 with BUSCO version 5.3.2 (Arthropoda selected; Simão et al. <a href="#ede70014-bib-0080" class="usa-link" aria-describedby="ede70014-bib-0080">2015</a>) and QUAST version 5.2.0 (eukaryote selected; Gurevich et al. <a href="#ede70014-bib-0033" class="usa-link" aria-describedby="ede70014-bib-0033">2013</a>). XSEDE resources were used (Towns et al. <a href="#ede70014-bib-0086" class="usa-link" aria-describedby="ede70014-bib-0086">2014</a>).</p></section><section id="ede70014-sec-0120"><h4 class="pmc_sec_title">2.4.3. Differential Expression of Mid‐ and Late‐Stage Embryos</h4>
<p>Differentially expressed genes between surface and cave samples at the mid‐stage and late‐stage timepoints were examined. Differential expression was assayed as previously discussed (Lomheim et al. <a href="#ede70014-bib-0047" class="usa-link" aria-describedby="ede70014-bib-0047">2023</a>). Briefly, trimmed reads were quantified against both transcriptomes with the default parameters from Kallisto (Bray et al. <a href="#ede70014-bib-0012" class="usa-link" aria-describedby="ede70014-bib-0012">2016</a>). The Kallisto output of estimated counts was combined into four matrices: all mid‐stage embryo samples mapped against CAVE_mw, all mid‐stage embryos mapped against SURF_li, all late‐stage samples mapped against CAVE_mw, and all late‐stage samples mapped against SURF_li. Differential expression analysis was performed using DEseq2 (Love et al. <a href="#ede70014-bib-0048" class="usa-link" aria-describedby="ede70014-bib-0048">2014</a>). DESeq2's shrinkage estimation and dispersion modeling provide more stable results for analyses with fewer biological replicates by using empirical Bayes estimators. These estimators pool information across genes to gain extra information and provide more accurate estimations of dispersion in cases where there might be artificially high variance due to sample size. DESeq2 also models the likelihood of data capture using a negative binomial distribution model, which also helps combat overdispersion in smaller sample sizes and heterogeneous data such as field samples of non‐model species.</p>
<p>Pairwise DESeq2 analysis was conducted on the four matrices described above using Trinity's differential expression perl wrappers (run_DE_analysis.pl and analyze_diff_expr.pl, Haas <a href="#ede70014-bib-0034" class="usa-link" aria-describedby="ede70014-bib-0034">2023</a>). This wrapper filters out transcripts with 0 CPM in any replicate, as this is ambiguously missing data, which could result from a lack of capture (technical bias) or a lack of expression (real variation). Normalization was also handled in the DESeq() function, using DESeq2's default mean of ratios method to normalize the count data and adjust for library size effects and compositional biases. Wald tests were used to evaluate the significance of the difference between log‐fold changes. Multiple test correction was done with False Discovery Rate via DESeq2's workflow. Signal and noise in the normalized, filtered data were then visualized in a principal component analysis (PCA) of count data (Figure <a href="#ede70014-suppl-0001" class="usa-link">S1</a>). The first two PCs explained 67.38% of the variation in the data and showed reasonably cohesive grouping of samples (in the context of expected noise due to variation in field samples) with distinct distances between the replicates (signal of different expression). Finally, we classified genes as significantly differentially expressed if they had a log2Fold change of &gt; 2 (minimum 4× change in expression) and FDR &lt; 0.05. We chose a moderate log fold change cutoff to guard against false positives and an FDR of 0.05 to account for our more limited statistical power. These cutoffs were further evaluated against the volcano plot to ensure we removed obviously spurious results, such as those with high fold change but no statistical support.</p>
<p>To unify the information from the four pairwise comparisons (mid‐stage to both transcriptomes and late‐stage to both transcriptomes), we needed to identify homologous loci in cave and surface transcriptomes to compare expression between the cave and surface samples. Each transcriptome had unique names as they were de novo assembled separately; there is no published annotation for this genome, and the gene name annotation transferred from Tribolium was not unique (i.e., multiple genes in our transcriptomes could have the same Tribolium ID). To establish a set of homologous sequences between the cave and surface (cave/surface pairs), a reciprocal blast technique was used. CAVE_mw and SURF_li databases were created using the web‐based platform European Galaxy NCBI BLAST+ (Galaxy Version 2.10.1+galaxy2; Camacho et al. <a href="#ede70014-bib-0015" class="usa-link" aria-describedby="ede70014-bib-0015">2009</a>; Cock et al. <a href="#ede70014-bib-0018" class="usa-link" aria-describedby="ede70014-bib-0018">2015</a>). The CAVE_mw transcriptome was then blasted against the SURF_li database, and vice versa, and we retained the longest and highest hit for each transcript in each blast run. As these are the same species, it is expected that there will be high homology between assembled transcripts. However, due to limitations of de novo assembly and sequencing a subset of transcripts in a sample, not all transcripts will have a complete assembly or a homologous pair retained. As such, transcripts under 400 bp and lacking a reciprocal hit were removed. A database of 15,516 cave/surface pairs resulted, with entries including the CAVE_mw transcript name and sequence, the differential expression results for that sequence (ref: CAVE_mw), the SURF_li transcript name and sequence, and the differential expression results for that sequence (ref: SURF_li).</p>
<p>Further filtering of possible differentially expressed genes was applied. Transcripts that had unique <em>Tribolium</em> IDs were retained as those were clearly unique sequences with clear homology to a related species. Conversely, we eliminated any transcripts that shared a <em>Tribolium</em> annotation with 9 or more other assembled transcripts in that transcriptome, rationalizing that a large number of hits to the same <em>Tribolium</em> gene were more likely to be fragments or chimeras rather than large numbers of unique paralogues. To further filter out any duplicates, we self‐blasted the transcripts against their own database and removed any expressed duplicates (identity &gt; 90%, length &gt; 150 bp, and CPM &gt; 1). This setup allowed for the use of VLOOKUP in Excel to identify which high‐confidence transcripts showed an LFC &gt; 2 and FDR &lt; 0.05 against both reference transcriptomes.</p>
<p>To perform GO enrichment, we compared this filtered set of differentially expressed transcripts against the background of the full list of genes output from DESeq2, minus any loci that shared the same Tribolium ID with 9 or more transcripts (low confidence assemblies). This was done separately for the output of mid‐stage comparisons and late‐stage comparisons, generating a mid‐stage reference list and a late‐stage reference list. Then, we used G‐profiler to analyze GO enrichment at each timepoint (Raudvere et al. <a href="#ede70014-bib-0068" class="usa-link" aria-describedby="ede70014-bib-0068">2019</a>).</p></section><section id="ede70014-sec-0130"><h4 class="pmc_sec_title">2.4.4. Allele‐Specific Expression Analysis</h4>
<p>Allele‐specific expression was performed as previously by Lomheim et al. (<a href="#ede70014-bib-0047" class="usa-link" aria-describedby="ede70014-bib-0047">2023</a>). First, cave/surface transcript pairs were generated, as described above, yielding 15,516 genes with cave/surface transcript pairs. The cave and surface sequences were combined (combined transcripts) using merge_pat_mat_fasta.pl script from ASE‐Tigar (Nariai et al. <a href="#ede70014-bib-0059" class="usa-link" aria-describedby="ede70014-bib-0059">2016</a>). This does not combine the actual sequences but retains two sequences for each gene, and the reads are mapped to each of the sequences.</p>
<p>The trimmed paired‐end reads of all SURF_li, CAVE_mw, and SURF_li_CAVE_mw_F1 samples were mapped to the combined transcripts using bowtie2, ‐X 1000 ‐k 100 –very‐sensitive, and ASE‐TIGAR (Langmead and Salzberg <a href="#ede70014-bib-0043" class="usa-link" aria-describedby="ede70014-bib-0043">2012</a>). <em>Z</em> values from ASE‐Tigar are a measure of transcript abundance. The ratio of <em>Z</em> mapped to each surface transcript to the corresponding cave transcript was compared for each sample. Genes were selected where the surface samples mapped to the surface transcript 3.3× more than the corresponding cave transcript and where the cave samples mapped to the cave transcript 3.3× more than the corresponding surface transcript. Within this subset of genes, we selected genes where all three F1 hybrids showed a bias towards one allele, mapping to either the cave transcript 3.3× more than the surface transcript, or vice versa. Transcripts that were not annotated or where the <em>Tribolium</em> ID was present in 10 or more copies in the transcriptome were removed.</p>
<p>Next, to identify fixed SNPs that were different between cave and surface transcripts, all trimmed, paired‐end reads of CAVE_mw, SURF_li, and SURF_li_CAVE_mw_F1 samples were mapped to combined transcripts using bowtie2 (default settings plus very fast end‐to‐end) and Freebayes (Garrison and Marth <a href="#ede70014-bib-0026" class="usa-link" aria-describedby="ede70014-bib-0026">2012</a>). Mapping SURF_li reads to the CAVE_mw transcripts and reciprocally mapping CAVE_mw reads to the SURF_li transcripts identified fixed SNPs between the cave and surface transcripts. All fixed SNPs were retained that were present with at least five observations.</p>
<p>For all the above genes prioritized by ASE‐Tigar, genes were further prioritized that also showed differential expression. For this subset of genes, we performed FreeBayes allele counting in the F1 hybrids using the fixed SNPs outlined above (Garrison and Marth <a href="#ede70014-bib-0026" class="usa-link" aria-describedby="ede70014-bib-0026">2012</a>). To eliminate bias from the reference used, allele counting was performed both when the cave and the surface transcripts were used as a reference. Five SNPs, if available, were selected across the span of a transcript for allele counting in the F1 hybrids. Most of the time, the location of these SNPs was the same in the cave versus the surface transcript. However, if an indel was present, the location of the SNPs might be offset in the two transcripts, and alignments were performed if it was unclear where the location of the SNPs was.</p>
<p>For each SNP, a binomial test was performed to investigate when the allele counts deviated from the predicted fraction of 0.5. To control for multiple testing, the resulting <em>p</em> values were adjusted using the Benjamini–Yekutieli procedure (Benjamini and Yekutieli <a href="#ede70014-bib-0005" class="usa-link" aria-describedby="ede70014-bib-0005">2001</a>). Any gene where at least two of the five SNPs showed adjusted <em>p</em> values &lt; 0.05 for all three F1 samples to both the cave and surface transcript was identified as having allele‐specific expression.</p></section></section></section><section id="ede70014-sec-0140"><h2 class="pmc_sec_title">3. Results</h2>
<section id="ede70014-sec-0150"><h3 class="pmc_sec_title">3.1. Comparative Embryology</h3>
<section id="ede70014-sec-0160"><h4 class="pmc_sec_title">3.1.1. 
<em>Asellus infernus</em> Embryos Were Depigmented Throughout Embryonic Development</h4>
<p>We first asked when morphological differences were established in <em>A. infernus</em> as compared to the surface form, embryonically or postembryonically. <em>A. infernus</em> adults collected from wells in Mangalia (cave) were depigmented and eyeless as compared to <em>A. aquaticus</em> adults collected from the Limanu Bridge Stream population (surface) (Figure <a href="#ede70014-fig-0001" class="usa-link">1</a>). We compared embryonic development of wild‐caught gravid cave females and wild‐caught surface females, which became gravid upon transport to the lab or immediately after arriving at the lab. Regarding pigmentation, <em>A. infernus</em> embryos never developed pigmentation, whereas surface embryos developed eye pigmentation at around 80% of the way through embryonic development (Figures <a href="#ede70014-fig-0002" class="usa-link">2</a>, <a href="#ede70014-suppl-0001" class="usa-link">S2</a>, and <a href="#ede70014-suppl-0001" class="usa-link">S3</a>) and shortly after, developed head and body pigmentation. Additionally, F1 hybrid embryos from <em>A. infernus</em> males crossed to surface females did develop eye pigmentation, similar to the surface form (Figure <a href="#ede70014-fig-0002" class="usa-link">2</a>). Furthermore, adult F1s had both head and body pigmentation and ommatidia (Figure <a href="#ede70014-fig-0001" class="usa-link">1D–F</a>). Therefore, we established that pigmentation was never seen embryonically in <em>A. infernus,</em> rather than initially being present and then lost later in development.</p>
<figure class="fig xbox font-sm" id="ede70014-fig-0001"><h5 class="obj_head">Figure 1.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/49af/12326928/d08110953d83/EDE-27-e70014-g002.jpg" loading="lazy" id="jats-graphic-3" height="453" width="600" alt="Figure 1"></p>
<div class="p text-right font-secondary"><a href="figure/ede70014-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of adult morphology between cave, F1 hybrid, and surface individuals. (A, B, C) Cave individual. (D, E, F) F1 hybrid individual. (G, H, I) Surface individual. (A, D, G) Whole body. (B, E, H) Head. (C, F, I) Profile. In A, the scale bar is 2 mm and is relevant for A, D, and G. In B, the scale bar is 200 µm and relevant for B, C, E, F, H, and I. [Color figure can be viewed at <a href="https://wileyonlinelibrary.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">wileyonlinelibrary.com</a>]</p></figcaption></figure><figure class="fig xbox font-sm" id="ede70014-fig-0002"><h5 class="obj_head">Figure 2.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/49af/12326928/75379e2da20f/EDE-27-e70014-g004.jpg" loading="lazy" id="jats-graphic-5" height="396" width="702" alt="Figure 2"></p>
<div class="p text-right font-secondary"><a href="figure/ede70014-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparative embryology between cave, F1 hybrid, and surface embryos. (A–D) Cave embryos. (E–H) F1 hybrid embryos. (I–L) Surface embryos. (A, E, I) Embryos at approximately 70% of the way through embryonic development (mid‐stage embryos). (B, F, J) Embryos at approximately 75% of the way through embryonic development. (C, G, K) Embryos at approximately 80% of the way through embryonic development. (D, H, L) Embryos at approximately 90% of the way through embryonic development. White arrow: Pigmented eye can be seen in H and L. Scale bar is 200 µM for all panels. [Color figure can be viewed at <a href="https://wileyonlinelibrary.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">wileyonlinelibrary.com</a>]</p></figcaption></figure></section><section id="ede70014-sec-0170"><h4 class="pmc_sec_title">3.1.2. Lab‐Bred <em>Asellus infernus</em> Juveniles Had Longer Body Length and Longer Antennae II Than Lab‐Bred Surface <em>Asellus aquaticus</em> Juveniles</h4>
<p>To further address the question of the timing of the establishment of morphological characteristics, we investigated size characteristics between lab‐bred <em>A. infernus</em> and surface individuals. For this comparison, we examined lab‐bred <em>A. infernus</em> and surface individuals that were raised in the same conditions to eliminate, as much as possible, any environmental variables. Specifically, we compared antenna II length, body length, and head width in lab‐bred surface and cave juveniles as soon as they were able to walk and would have been released from the brood pouch (Figure <a href="#ede70014-fig-0003" class="usa-link">3</a> and Table <a href="#ede70014-suppl-0001" class="usa-link">S4</a>). For body length, antenna II length, and head width, cave samples were longer compared to surface samples (<em>p</em> &lt; 0.0005 for all comparisons; Figure <a href="#ede70014-fig-0003" class="usa-link">3</a> and Table <a href="#ede70014-suppl-0001" class="usa-link">S4</a>). Overall, we conclude that lab‐bred cave and surface juveniles are different in body size, head width, and antenna size, and that these differences, many of which are also present in comparisons between adult cave and surface individuals, are established by the end of embryonic development.</p>
<figure class="fig xbox font-sm" id="ede70014-fig-0003"><h5 class="obj_head">Figure 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/49af/12326928/1eba116837c6/EDE-27-e70014-g005.jpg" loading="lazy" id="jats-graphic-7" height="338" width="702" alt="Figure 3"></p>
<div class="p text-right font-secondary"><a href="figure/ede70014-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of body and antenna length for the comparisons: cave lab versus surface lab, wild versus lab, and F1 versus cave and surface. “Lab” indicates juveniles from females that were raised from juveniles in the lab and bred in the lab. “Wild” indicates juveniles that were collected from wild‐caught individuals with embryos. F1 are F1 hybrids between surface females and cave males. (A) Body length of the juvenile. (B) Size of left antenna II. Black bars and asterisks are for cave lab versus surface lab comparison. Magenta bars and asterisks are for wild versus lab comparisons. Blue bars and asterisks are for F1 versus cave and lab comparisons. Table <a href="#ede70014-suppl-0001" class="usa-link">S4</a> holds <em>p</em> values of statistical tests of all possible comparisons. ANOVA and Tukey's HSD test were performed for antenna length, body length, and head width. ***<em>p</em> &lt; 0.0005, **0.0005 &lt; <em>p</em> &lt; 0.005, *0.005 &lt; <em>p</em> &lt; 0.05. <em>p</em> &gt; 0.05 was labeled NS. [Color figure can be viewed at <a href="https://wileyonlinelibrary.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">wileyonlinelibrary.com</a>]</p></figcaption></figure></section><section id="ede70014-sec-0180"><h4 class="pmc_sec_title">3.1.3. Population‐Environment Interactions (Wild‐Caught Vs. Lab‐Bred) Influenced Several Measurements' Differences Within Both Cave and Surface Samples</h4>
<p>To investigate whether environmental changes also influence these morphological differences, we compared wild‐caught and lab‐bred juveniles. For the surface wild‐caught to surface lab‐bred comparisons, two measures showed significant differences: head width and body length (<em>p</em> &lt; 0.0005 for both; Figure <a href="#ede70014-fig-0003" class="usa-link">3</a> and Table <a href="#ede70014-suppl-0001" class="usa-link">S4</a>). For the cave wild‐caught to cave lab‐bred comparisons, only body length showed a significant difference (<em>p</em> = 0.00073). To ensure that differences in these measurements were not simply reflections of distinctions in overall size, we performed PCAs on the measurements of body length, head width, and left and right antennae length (Figure <a href="#ede70014-fig-0004" class="usa-link">4</a> and Table <a href="#ede70014-suppl-0001" class="usa-link">S5</a>). The first two principal components together explained approximately 95% of the total variation, so further components were not considered. The first principal component (PC1) corresponded roughly to an average of all four body measurements and therefore can be interpreted as a general measure of overall size: large positive PC1 scores are associated simultaneously with longer body lengths, antennae, and wider heads, while more negative PC1 scores are associated with smaller measurements overall. The second principal component (PC2) corresponded to a contrast between antennae length and head and body measurements: highly positive PC2 scores are associated with longer antennae but shorter body lengths overall, and more negative PC2 scores were associated with longer bodies but shorter antennae. Figure <a href="#ede70014-fig-0004" class="usa-link">4</a> displays schematic archetypes of isopods with extreme values in each of these two directions of variation.</p>
<figure class="fig xbox font-sm" id="ede70014-fig-0004"><h5 class="obj_head">Figure 4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/49af/12326928/0c9bcfcb744d/EDE-27-e70014-g001.jpg" loading="lazy" id="jats-graphic-9" height="402" width="702" alt="Figure 4"></p>
<div class="p text-right font-secondary"><a href="figure/ede70014-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>First and second principal component scores of body measurements, which explain roughly 88% and 6.4% of total variation, respectively. Archetypal morphologies for extremes in each direction are depicted with isopod schematic diagrams, with the average for each subsample denoted by a cross. Higher first principal component scores generally correspond to larger body measurements overall and are generally held by isopods with longer body lengths, antennae, and wider heads; lower first principal component scores are associated with smaller measurements overall. Higher second principal component scores are held by isopods with longer antennae but shorter body lengths, and vice versa for isopods with lower second principal component scores. [Color figure can be viewed at <a href="https://wileyonlinelibrary.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">wileyonlinelibrary.com</a>]</p></figcaption></figure><p>We performed ANOVA using juvenile's specific combination of population and environment as a predictor for each of the two principal component scores, which are uncorrelated by construction, ensuring that the differences in contrasts between antennae relative to body measurements (i.e., PC2) are not simply due to differences in size (i.e., PC1). Pairwise comparisons across subgroups were assessed using Tukey's HSD test, and <em>p</em> values were reported after adjusting for multiple testing (Table <a href="#ede70014-tbl-0001" class="usa-link">1</a>).</p>
<section class="tw xbox font-sm" id="ede70014-tbl-0001"><h5 class="obj_head">Table 1.</h5>
<div class="caption p"><p>Pairwise differences across environments for average first and second principal component scores evaluated with ANOVA and Tukey's HSD test, with <em>p</em> values adjusted for multiple comparisons.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1">
<col align="center" style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1">
<col align="center" style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1">
<col align="center" style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1">
<col align="center" style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1">
<col align="center" style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1">
<col align="center" style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1">
<thead valign="bottom">
<tr valign="bottom">
<th align="left" valign="bottom" rowspan="1" colspan="1"></th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="center" valign="bottom" rowspan="1">1st PC Score (overall size)</th>
<th align="center" colspan="3" style="border-bottom:solid 1px #000000" valign="bottom" rowspan="1">2nd PC Score (antennae vs. body length contrast)</th>
</tr>
<tr style="border-bottom:solid 1px #000000" valign="bottom">
<th align="left" valign="bottom" rowspan="1" colspan="1">Comparison</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Diff.</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">St. Err.</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>p</em> (adj.)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Diff.</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">St. Err.</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>p</em> (adj.)</th>
</tr>
</thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Cave Wild–Cave Lab</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.3392</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.3389</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.4604</td>
<td align="center" valign="top" rowspan="1" colspan="1">−0.3885</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.1887</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>0.0166</strong>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Cave Wild–Surface Wild</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.9513</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.3415</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>≈0</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">−0.3553</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.1707</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>0.0152</strong>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Cave Wild–Surface Lab</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.9296</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.3365</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>≈0</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.2369</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.1738</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.1952</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Cave Lab–Surface Wild</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.6121</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.3123</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>≈0</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.0332</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.1873</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.9937</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Cave Lab–Surface Lab</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.5904</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.3068</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>≈0</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.6254</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.1902</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>≈0</strong>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Surface Lab–Surface Wild</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.0217</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.3096</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>2.76 × 10<sup>−5</sup>
</strong>
</td>
<td align="center" valign="top" rowspan="1" colspan="1">−0.5922</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.1723</td>
<td align="center" valign="top" rowspan="1" colspan="1">
<strong>≈0</strong>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ede70014-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="ede70014-tbl1-note-0001"><p>
<em>Note:</em> Bold indicates significant at the 0.05 level after multiple comparison correction. Italics is used for all standard errors. Significant differences in general size (PC1 score) were detected across all comparisons except for cave wild versus cave lab subgroups. Meanwhile, differences in antennae versus body length contrasts (PC2 scores) were significant for all comparisons except cave wild versus surface lab and cave lab versus surface wild subgroups. Multiple comparison corrections for significance were performed using the Bonferroni procedure.</p></div></div></section><p>Significant differences in overall size (PC1) were detected for all comparisons except between cave lab‐bred versus cave wild‐caught. A detailed look at all comparisons together suggests that cave specimens were larger overall than surface specimens. However, size differences between lab‐bred and wild‐caught subgroups were not so straightforward and depended on whether the isopod was a surface or cave‐dwelling specimen. For surface individuals, lab‐bred subjects were generally larger than their wild‐caught counterparts across the board (<em>p</em> = 0.00003), while for cave individuals, wild‐caught specimens were on average larger than lab‐bred individuals, though this difference was not significant (<em>p</em> = 0.4604).</p>
<p>Significant differences in contrasts between antennae and body length (PC2) were also found in most comparisons, except for those across cave wild versus surface lab‐bred subgroups and cave lab‐bred versus surface wild subgroups. No marginal cave versus surface trend emerged in this direction; rather, differences in PC2 scores were dependent on the interaction between surface/cave environments and lab‐bred/wild breeding. The cave lab‐bred and surface wild‐caught groups had positive PC2 scores (i.e., longer antennae and shorter body lengths, while controlling for overall size), while cave wild‐caught and surface lab‐bred groups generally had negative PC2 scores (i.e., shorter antennae and longer body lengths, while controlling for overall size). See Figure <a href="#ede70014-fig-0004" class="usa-link">4</a> for visualization of group morphologies and Table <a href="#ede70014-tbl-0001" class="usa-link">1</a> for specific pairwise comparisons and corresponding <em>p</em> values.</p></section><section id="ede70014-sec-0190"><h4 class="pmc_sec_title">3.1.4. F1 Hybrid Samples Were Intermediate and Different in Most Size Measures Between Cave and Surface Individuals</h4>
<p>Because cave and surface forms can interbreed, we had the opportunity to compare the above size characteristics in F1 hybrids with the goal of seeing whether the F1 hybrids were similar to <em>A. infernus</em>, to surface, or intermediate between the two. Our expectation was that the F1 hybrids would be intermediate between the two measures. The F1 samples were significantly different from both cave lab‐bred and wild‐caught samples for the measures of head width, body length, and antenna length (<em>p</em> &lt; 0.0005 for all measures; Figure <a href="#ede70014-fig-0003" class="usa-link">3</a> and Table <a href="#ede70014-suppl-0001" class="usa-link">S4</a>). The F1 individuals were different for both surface lab‐bred and wild‐caught samples for only antenna length (<em>p</em> = 0.0153 for surface lab and <em>p</em> &lt; 0.0005 for surface wild; Figure <a href="#ede70014-fig-0003" class="usa-link">3</a> and Table <a href="#ede70014-suppl-0001" class="usa-link">S4</a>).</p></section></section><section id="ede70014-sec-0200"><h3 class="pmc_sec_title">3.2. Transcriptomic Analysis</h3>
<section id="ede70014-sec-0210"><h4 class="pmc_sec_title">3.2.1. Different mRNA Levels Were Seen in Genes Involved in Phototransduction and Sulfide Detoxification in Cave and Surface Samples</h4>
<p>Next, we investigated the question of what genes showed expression differences between <em>A. infernus</em> and surface samples. First, we generated transcriptomes for the <em>A. infernus</em> cave and <em>A. aquaticus</em> Limanu springs surface populations (Table <a href="#ede70014-tbl-0002" class="usa-link">2</a>). With BUSCO scores of 93%, this is similar to updated transcriptomes for the Subterranean Rak population and Rakov Škocjan of <em>A. aquaticus</em> (Lomheim et al. <a href="#ede70014-bib-0047" class="usa-link" aria-describedby="ede70014-bib-0047">2023</a>) and transcriptomes generated from the Molnár János Cave population and its adjacent surface populations, Malom Lake and Soroksár from Hungary (Pérez‐Moreno et al. <a href="#ede70014-bib-0064" class="usa-link" aria-describedby="ede70014-bib-0064">2018</a>). Cave and surface transcriptomes had a similar number of transcripts from the previously examined populations (Table <a href="#ede70014-tbl-0002" class="usa-link">2</a>).</p>
<p>Recently, an unannotated genome for <em>A. aquaticus</em> became available (Thomas and University of Oxford and Wytham Woods Genome Acquisition Lab, Natural History Museum Genome Acquisition Lab, Darwin Tree of Life Barcoding collective, Wellcome Sanger Institute Tree of Life Management, Samples and Laboratory team, Wellcome Sanger Institute Scientific Operations: Sequencing Operations, Wellcome Sanger Institute Tree of Life Core Informatics team, Tree of Life Core Informatics collective and Darwin Tree of Life Consortium <a href="#ede70014-bib-0082" class="usa-link" aria-describedby="ede70014-bib-0082">2025</a>), and we mapped both the cave and surface transcriptomes to the genome. For the cave transcriptome, all 88,614 transcripts had a primary alignment. Of these, 55,950 transcripts had high‐quality, non‐chimeric primary alignments, and another 2117 had supplementary alignments across transcripts. For the surface transcriptome, all 80,182 transcripts had a primary alignment. 59,304 transcripts had high‐quality, non‐chimeric primary alignments, and another 1537 had supplementary alignments across transcripts. In all figures, tables, and files, an asterisk is placed next to any transcript that did not have a primary alignment.</p>
<p>We performed differential expression analysis on wild‐caught embryos at two different embryonic timepoints, mid‐stage and late‐stage, identifying genes that were surface‐biased (higher mRNA levels in surface samples as compared to cave samples) and cave‐biased (higher mRNA levels in cave samples as compared to surface samples) (Supporting Information Files <a href="#ede70014-suppl-0001" class="usa-link">2</a>, <a href="#ede70014-suppl-0001" class="usa-link">3</a>, <a href="#ede70014-suppl-0001" class="usa-link">4</a>, and <a href="#ede70014-suppl-0001" class="usa-link">5</a> and Figures <a href="#ede70014-suppl-0001" class="usa-link">S4</a> and <a href="#ede70014-suppl-0001" class="usa-link">S5</a>). We found 166 genes that were surface‐biased and 73 genes that were cave‐biased at the mid‐stage timepoint. At the late‐stage timepoint, we found 160 genes that were surface‐biased and 73 that were cave‐biased. We had expected that genes involved in phototransduction would show lower mRNA levels in the cave individuals as compared to the surface samples and found that this was the case for six genes at the mid‐stage timepoint and seven genes at the late‐stage timepoint; some of the genes were in common between the two timepoints (Table <a href="#ede70014-tbl-0003" class="usa-link">3</a>). Additionally, several genes involved in melanin synthesis and the diurnal clock showed significantly different mRNA levels in cave versus surface samples.</p>
<section class="tw xbox font-sm" id="ede70014-tbl-0003"><h5 class="obj_head">Table 3.</h5>
<div class="caption p"><p>Light‐interacting genes that exhibit different mRNA levels in cave versus surface samples in mid‐ and late‐stage embryos.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1">
<col align="center" style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1">
<col align="center" style="border-right:solid 1px #000000; border-bottom:solid 1px #000000" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000" valign="bottom">
<th align="left" valign="bottom" rowspan="1" colspan="1">Mid‐stage embryos</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Cave or surface biased</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Light‐interacting gene category</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/NP_001155991" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NP_001155991</a>.1rhodopsin 1/6‐like+</td>
<td align="center" valign="top" rowspan="1" colspan="1">Surface</td>
<td align="center" valign="top" rowspan="1" colspan="1">Phototransduction</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_008192672.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_008192672.1</a> serine/threonine‐protein phosphatase rdgC</td>
<td align="center" valign="top" rowspan="1" colspan="1">Surface</td>
<td align="center" valign="top" rowspan="1" colspan="1">Phototransduction</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_015835336.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_015835336.1</a> high affinity cGMP‐specific 3′,5′‐cyclic phosphodiesterase 9A</td>
<td align="center" valign="top" rowspan="1" colspan="1">Surface</td>
<td align="center" valign="top" rowspan="1" colspan="1">Phototransduction</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_015838325.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_015838325.1</a> G‐protein coupled receptor 161</td>
<td align="center" valign="top" rowspan="1" colspan="1">Surface</td>
<td align="center" valign="top" rowspan="1" colspan="1">Phototransduction</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_015834977.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_015834977.1</a> protein kinase C‐binding protein NELL1 isoform X1</td>
<td align="center" valign="top" rowspan="1" colspan="1">Surface</td>
<td align="center" valign="top" rowspan="1" colspan="1">Phototransduction</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_008197970.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_008197970.1</a> regulator of G‐protein signaling 7 isoform X2</td>
<td align="center" valign="top" rowspan="1" colspan="1">Surface</td>
<td align="center" valign="top" rowspan="1" colspan="1">Phototransduction</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/NP_001139379" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NP_001139379</a>.1dopamine N acetyltransferase isoform 2+</td>
<td align="center" valign="top" rowspan="1" colspan="1">Surface</td>
<td align="center" valign="top" rowspan="1" colspan="1">Melanin synthesis</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_015839422.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_015839422.1</a> laccase 1 isoform X1</td>
<td align="center" valign="top" rowspan="1" colspan="1">Surface</td>
<td align="center" valign="top" rowspan="1" colspan="1">Melanin synthesis</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_972873.2" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_972873.2</a> dopamine N‐acetyltransferase°</td>
<td align="center" valign="top" rowspan="1" colspan="1">Cave, surface</td>
<td align="center" valign="top" rowspan="1" colspan="1">Melanin synthesis</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_008199627.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_008199627.1</a> carotenoid isomerooxygenase</td>
<td align="center" valign="top" rowspan="1" colspan="1">Cave</td>
<td align="center" valign="top" rowspan="1" colspan="1">Carotenoid pathway</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_008201448.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_008201448.1</a> putative ferric‐chelate reductase 1</td>
<td align="center" valign="top" rowspan="1" colspan="1">Cave</td>
<td align="center" valign="top" rowspan="1" colspan="1">Heme pathway</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_967100.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_967100.1</a> retinol dehydrogenase 11</td>
<td align="center" valign="top" rowspan="1" colspan="1">Cave</td>
<td align="center" valign="top" rowspan="1" colspan="1">Retinoid pathway</td>
</tr>
<tr><td align="left" colspan="3" valign="top" rowspan="1">
<strong>Late‐stage embryos</strong>
</td></tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_015834291.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_015834291.1</a> c‐opsin isoform X1</td>
<td align="center" valign="top" rowspan="1" colspan="1">Surface</td>
<td align="center" valign="top" rowspan="1" colspan="1">Phototransduction</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/NP_001155991" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NP_001155991</a>.1rhodopsin 1/6‐like+</td>
<td align="center" valign="top" rowspan="1" colspan="1">Surface</td>
<td align="center" valign="top" rowspan="1" colspan="1">Phototransduction</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_015835000.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_015835000.1</a> transient receptor potential protein isoform X1</td>
<td align="center" valign="top" rowspan="1" colspan="1">Surface</td>
<td align="center" valign="top" rowspan="1" colspan="1">Phototransduction</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_015837026.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_015837026.1</a> transient receptor potential channel pyrexia+</td>
<td align="center" valign="top" rowspan="1" colspan="1">Surface</td>
<td align="center" valign="top" rowspan="1" colspan="1">Phototransduction</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_015835336.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_015835336.1</a> high affinity cGMP‐specific 3′,5′‐cyclic phosphodiesterase 9A</td>
<td align="center" valign="top" rowspan="1" colspan="1">Surface</td>
<td align="center" valign="top" rowspan="1" colspan="1">Phototransduction</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_015834977.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_015834977.1</a> protein kinase C‐binding protein NELL1 isoform X1</td>
<td align="center" valign="top" rowspan="1" colspan="1">Surface</td>
<td align="center" valign="top" rowspan="1" colspan="1">Phototransduction</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_008197970.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_008197970.1</a> regulator of G‐protein signaling 7 isoform X2</td>
<td align="center" valign="top" rowspan="1" colspan="1">Surface</td>
<td align="center" valign="top" rowspan="1" colspan="1">Phototransduction</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_015835617.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_015835617.1</a> period isoform X1</td>
<td align="center" valign="top" rowspan="1" colspan="1">Surface</td>
<td align="center" valign="top" rowspan="1" colspan="1">Diurnal clock</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_015835556.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_015835556.1</a> PDF receptor isoform X4</td>
<td align="center" valign="top" rowspan="1" colspan="1">Surface</td>
<td align="center" valign="top" rowspan="1" colspan="1">Diurnal clock</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_972873.2" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_972873.2</a> dopamine N‐acetyltransferase</td>
<td align="center" valign="top" rowspan="1" colspan="1">Cave</td>
<td align="center" valign="top" rowspan="1" colspan="1">Melanin synthesis</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.ncbi.nlm.nih.gov/protein/XP_967100.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XP_967100.1</a> retinol dehydrogenase 11</td>
<td align="center" valign="top" rowspan="1" colspan="1">Cave</td>
<td align="center" valign="top" rowspan="1" colspan="1">Retinoid pathway</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ede70014-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="ede70014-tbl3-note-0001"><p>
<em>Note:</em> +gene‐name also showed differential expression between the Subterranean Rak population versus Rakov Škocjan (Lomheim et al. <a href="#ede70014-bib-0047" class="usa-link" aria-describedby="ede70014-bib-0047">2023</a>). °Two paralogues of this gene were present; one was cave‐biased and the other was surface‐biased.</p></div></div></section><p>We had also predicted that some of the genes that were differentially expressed would be in response to the sulfidic environment, such as genes involved in H<sub>2</sub>S oxidation. As expected, we saw cave‐biased differential expression in sulfide:quinone oxidoreductase and in persulfide dioxygenase ETHE1 at both the mid‐stage and late‐stage timepoints (Supporting Information Files <a href="#ede70014-suppl-0001" class="usa-link">3</a> and <a href="#ede70014-suppl-0002" class="usa-link">5</a>).</p>
<p>GO enrichment analysis of differentially expressed genes did not highlight any categories involved with eyes, pigmentation, or sulfide metabolism. However, in the mid‐stage embryos, the KEGG term of “arachidonic acid metabolism” was enriched in the genes that were surface biased (KEGG:00590, <em>p</em> = 0.042). In the late‐stage embryos, the GO term “structural constituent of cuticle” was enriched in cave‐biased genes (GO:0042302, <em>p</em> &lt; 0.001). The lack of enrichment data is likely due to the comparatively small number of genes identified as showing statistically significant mRNA levels and the usage of whole embryos rather than tissue‐specific RNAseq.</p></section><section id="ede70014-sec-0220"><h4 class="pmc_sec_title">3.2.2. Genes With Cave‐Biased and Surface‐Biased Allele‐Specific Expression Were Identified Including Genes With Possible Roles in Adaptation to Sulfidic Environments</h4>
<p>To continue examining the question of what genes showed expression differences between cave and surface samples, allele‐specific expression was used. Allele‐specific expression is a method of identifying genes that might have <em>cis</em>‐regulatory changes. First, we used ASE‐Tigar to prioritize genes with allele‐specific expression (Supporting Information Files <a href="#ede70014-suppl-0002" class="usa-link">6</a> and <a href="#ede70014-suppl-0001" class="usa-link">7</a>). In our final list of genes with allele‐specific expression using FreeBayes allele counting and eliminating genes that did not also show differential expression, we identified 5 genes that were cave‐biased and 11 genes that were surface‐biased (Figure <a href="#ede70014-fig-0005" class="usa-link">5</a> and Supporting Information File <a href="#ede70014-suppl-0002" class="usa-link">8</a>).</p>
<figure class="fig xbox font-sm" id="ede70014-fig-0005"><h5 class="obj_head">Figure 5.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/49af/12326928/8ee544a46c5c/EDE-27-e70014-g003.jpg" loading="lazy" id="jats-graphic-11" height="482" width="702" alt="Figure 5"></p>
<div class="p text-right font-secondary"><a href="figure/ede70014-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genes showing allele‐specific expression. Three F1 hybrid samples were examined both in reference to the surface transcriptome and the cave transcriptome. A. Allele counting was performed using FreeBayes variant counting for five SNPs along the location of each gene, both the cave and surface versions of the transcript (shown as rows labeled as SNP1‐5 in parts B and C). C represents the cave allele, and S represents the surface allele. Here, a schematic of a hypothetical gene with cave‐biased expression is shown as an example. B. Genes that show cave‐biased allele‐specific expression are shown in red. C. Genes that show surface‐biased allele‐specific expression are in green. We performed binomial tests to show whether the ratio of surface to cave alleles deviates significantly from the expected 1:1 ratio. Multiple comparison correction was performed using the Benjamini–Yekutieli procedure. White cells denote non‐significance with <em>p</em> &gt; 0.05, light green/red cells denote significance with 0.005 &lt; <em>p</em> &lt; 0.05, plain green/red cells denote significance with 0.0005 &lt; <em>p</em> &lt; 0.005, and dark green/red cells denote significance with <em>p</em> &lt; 0.0005. Genes shown in B and C are those for which at least two SNPs showed a <em>p</em> &lt; 0.05 for all hybrid samples to both the surface and cave transcript. The code before the gene name is the <em>Tribolium castaneum</em> gene ID. An asterisk indicates a gene that did not have a high‐quality, non‐chimeric primary alignment for either the surface or cave transcript. [Color figure can be viewed at <a href="https://wileyonlinelibrary.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">wileyonlinelibrary.com</a>]</p></figcaption></figure></section></section></section><section id="ede70014-sec-0230"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>We have generated resources for a subterranean species, <em>A. infernus</em>, that has been historically difficult to work with in the lab. We addressed the following questions: when are morphological differences established between cave and surface individuals, do both genetic and environmental inputs influence phenotype differences between cave and surface individuals, and finally, what genes and pathways are responsible for phenotype evolution between cave and surface individuals? We found that multiple morphological differences were established by the end of embryonic development: lack of pigmentation, longer body length, and longer antennae II. We also found that the origin of samples, lab‐bred versus wild‐caught, influenced many of the measures, both in cave and surface samples, supporting that environmental inputs, as well as genetics, influence the size phenotypes. Furthermore, we generated genomic resources for this population, including an <em>A. infernus</em> transcriptome, a surface transcriptome, and, through differential expression analysis and allele‐specific expression analysis, detected differences in multiple genes involved in phototransduction and increased expression in genes involved in sulfide oxidation.</p>
<section id="ede70014-sec-0240"><h3 class="pmc_sec_title">4.1. Comparative Embryology</h3>
<p>First, we investigated the question of when morphological differences were established between cave and surface individuals, embryonically or post‐embryonically. Comparative embryological studies of <em>A. infernus</em> and surface samples showed that pigmentation never developed similarly to what had been seen in comparative embryology of the Subterranean Rak population from Slovenia (Mojaddidi et al. <a href="#ede70014-bib-0057" class="usa-link" aria-describedby="ede70014-bib-0057">2018</a>). Therefore, the pigmentation difference was established during embryonic development in both examined populations of the <em>A. aquaticus</em> species complex. This is similar to <em>Astyanax mexicanus</em>, where cave embryos, from an albino cave population, never developed pigmentation when examined as embryos (McCauley et al. <a href="#ede70014-bib-0055" class="usa-link" aria-describedby="ede70014-bib-0055">2004</a>). In <em>A. mexicanus</em>, though we do not know if complete loss of pigmentation is intrinsically advantageous in the cave environment, it has been seen that the gene responsible, <em>oca2</em>, shows pleiotropic effects responsible for sleep loss, which is thought to be an adaptive trait (O'Gorman et al. <a href="#ede70014-bib-0061" class="usa-link" aria-describedby="ede70014-bib-0061">2021</a>). Loss of pigmentation in <em>A. aquaticus</em> could similarly have some sort of advantageous pleiotropic effect. Support for the idea that loss of pigmentation in <em>A. aquaticus</em> is adaptive is that the same region/possibly gene is likely responsible for loss of pigmentation in multiple cave populations of the <em>A. aquaticus</em> species complex, including <em>A. infernus</em> (Re et al. <a href="#ede70014-bib-0069" class="usa-link" aria-describedby="ede70014-bib-0069">2018</a>; Rodas et al. <a href="#ede70014-bib-0074" class="usa-link" aria-describedby="ede70014-bib-0074">2023</a>; Fišer et al. <a href="#ede70014-bib-0023" class="usa-link" aria-describedby="ede70014-bib-0023">2024</a>).</p>
<p>In addition, we investigated whether size characteristics were established between cave and surface individuals embryonically or post‐embryonically. The specific traits of body length, head width, and antenna II length were all significantly different between lab‐bred <em>A. infernus</em> and surface samples. Previous work had shown that the Subterranean Rak population also had increased body length and increased antenna length (Mojaddidi et al. <a href="#ede70014-bib-0057" class="usa-link" aria-describedby="ede70014-bib-0057">2018</a>). As these cave populations are ecologically and geographically distinct, as reviewed by Protas et al. (<a href="#ede70014-bib-0066" class="usa-link" aria-describedby="ede70014-bib-0066">2023</a>), it is striking that similar size differences are present in adults but also in juveniles, suggesting that the size differences might be adaptive in the cave environment. In fact, one of the commonly described features of cave animals is smaller numbers of larger embryos (Culver <a href="#ede70014-bib-0020" class="usa-link" aria-describedby="ede70014-bib-0020">1982</a>), and <em>A. infernus</em> has previously been described as having fewer embryos per brood than the adjacent surface population (Turk‐Prevorčnik et al. <a href="#ede70014-bib-0087" class="usa-link" aria-describedby="ede70014-bib-0087">1998</a>). It could be that larger adults have larger embryos, but previous studies in the surface form of <em>A. aquaticus</em> did not see a correlation between adult size and embryo size (Ridley and Thompson <a href="#ede70014-bib-0070" class="usa-link" aria-describedby="ede70014-bib-0070">1979</a>). In addition, sulfidic environments have also been associated with larger embryo size in different species. For example, in <em>P. mexicana</em>, both cave environments and sulfidic environments housed individuals with larger embryos (Riesch et al. <a href="#ede70014-bib-0071" class="usa-link" aria-describedby="ede70014-bib-0071">2010</a>). One reason for this could be resource scarcity (Riesch et al. <a href="#ede70014-bib-0072" class="usa-link" aria-describedby="ede70014-bib-0072">2014</a>). Or, perhaps minimizing the surface area to volume ratio with larger individuals minimizes the region exposed to the toxin (pertinent to the sulfidic environment). Another possibility is that the size difference is due to genome size (nucleotypic effect) (Gregory <a href="#ede70014-bib-0031" class="usa-link" aria-describedby="ede70014-bib-0031">2001</a>). Subterranean <em>Proasellus</em> isopods have been shown to have a larger genome size than non‐subterranean <em>Proasellus</em> isopods (Lefébure et al. <a href="#ede70014-bib-0045" class="usa-link" aria-describedby="ede70014-bib-0045">2017</a>). Interestingly, though we have seen this size difference in juveniles comparing two cave/surface pairs, a recent study comparing adult individuals from six cave populations and nine surface populations for adult individuals of <em>A. aquaticus</em> did not show a general trend of larger cave adults as compared to surface adults (Herczeg et al. <a href="#ede70014-bib-0035" class="usa-link" aria-describedby="ede70014-bib-0035">2023</a>). However, <em>A. infernus</em> did have the largest adult size of all the populations examined and showed a larger size than the Turkish bath surface population (two of the populations examined here; Herczeg et al. <a href="#ede70014-bib-0035" class="usa-link" aria-describedby="ede70014-bib-0035">2023</a>). Future work could examine juveniles of other cave populations to see if there is a trend in cave versus surface juvenile size, though absent as an overall trend in adults.</p>
<p>With the cave and surface samples, we were able to compare wild and lab‐reared juveniles of both populations. Interestingly, many of the measures differed between wild and lab‐reared individuals within a population. For example, body length was statistically different between wild and lab surface populations as well as between wild and lab cave populations. Body length was higher in wild cave samples as compared to lab cave samples. Conversely, body length was lower in wild surface samples as compared to lab surface populations. In terms of allometry, lab cave and wild surface samples had longer antennae and shorter body lengths, controlling for overall size, while wild cave and lab surface samples generally had shorter antennae and longer body lengths, controlling for overall size. Therefore, there was no overall cave versus surface trend in allometry. A possible reason for the opposite trends seen for wild‐caught cave versus wild‐caught surface samples is that, as mentioned above, the wild‐caught cave samples were mostly taken from females that were already gravid when individuals were collected but the majority of the wild‐caught surface samples were taken from females that became gravid on transport to the lab which was likely a stressful and different environment. As for the lab‐bred versus wild‐caught environmental conditions, the supply of food is likely to be greater and of higher nutritional value in the lab for both cave and surface populations. In the wild, surface populations are susceptible to seasonal fluctuations in food availability. Additionally, it is unclear whether the type of food used in the laboratory populations, algae pellets, recapitulates the type of nutrition the animals would be getting in either the cave or surface environment. Furthermore, other conditions, such as water composition and light regime, could also influence juvenile size by influencing gene expression changes. Finally, our lab‐bred individuals were from a single‐generation bred in the lab; it is possible that an increased number of generations in the lab might also affect some size measures if some sort of epigenetic modifications were affecting gene expression changes. Rearing <em>A. mexicanus</em> surface fish in the dark also resulted in morphological changes (Bilandžija et al. <a href="#ede70014-bib-0008" class="usa-link" aria-describedby="ede70014-bib-0008">2020</a>). Additionally, studies of <em>A. mexicanus</em> showed many gene expression differences in population comparisons of both cave and surface populations that were wild‐caught and lab‐reared (Krishnan et al. <a href="#ede70014-bib-0041" class="usa-link" aria-describedby="ede70014-bib-0041">2020</a>). Furthermore, altering conditions that cave and surface <em>A. mexicanus</em> were housed in (light–dark cycle vs. dark rearing) resulted in many gene expression changes (Sears et al. <a href="#ede70014-bib-0079" class="usa-link" aria-describedby="ede70014-bib-0079">2020</a>). Therefore, the difference in size characters between wild and lab‐raised <em>A. aquaticus</em> juveniles is not surprising.</p>
<p>F1 hybrids were investigated to see if they were more similar in size to surface or <em>A. infernus</em> samples or were intermediate in size between the two. In an ideal world, we would have generated F1s from cave and surface individuals that were raised from embryos in the lab, and then, by our previous definition, these F1s would count as lab‐bred, and we could compare them exclusively to lab‐bred surface and cave samples. However, F1s of these populations are not straightforward to achieve, and the F1s we generated were from cave males that were recently brought from the wild and surface females, of which some were recently brought from the wild or others that were lab‐bred. These crosses were performed in the lab, mostly after they had been living in the lab for at least a month. As a result, these F1s cannot be classified by our previous definitions as wild‐caught or lab‐bred but instead are somewhere in between. Therefore, we compared the F1 measurements to both wild‐caught and lab‐bred samples to determine if the F1 measurement was significantly different for each measure. F1 hybrid juveniles were significantly different than and intermediate between cave and surface samples for antenna length. Furthermore, F1 hybrid juveniles were significantly different than cave samples for body length and head width, though not significantly different than surface samples. In <em>A. mexicanus</em>, F1 hybrids were different than both cave and surface parents for melanophore number, width of lower jaw, sclera size, and eye diameter (Wilkens <a href="#ede70014-bib-0089" class="usa-link" aria-describedby="ede70014-bib-0089">1988</a>; Ma et al. <a href="#ede70014-bib-0052" class="usa-link" aria-describedby="ede70014-bib-0052">2018</a>). Furthermore, some of these measures in <em>A. mexicanus</em> showed maternal effect and were different for F1 hybrids generated from cave females × surface males as compared to F1 hybrids generated from cave males × surface females (Ma et al. <a href="#ede70014-bib-0052" class="usa-link" aria-describedby="ede70014-bib-0052">2018</a>). We, unfortunately, were not yet able to investigate maternal effect in <em>A. aquaticus</em> as the females for all F1 hybrids were surface, and the males were cave due to the difficulty in setting up the reciprocal crosses. This is likely not an incompatibility issue; cave females do not breed robustly in the lab, so it is more difficult to get them to breed (even if with males from the same population).</p></section><section id="ede70014-sec-0250"><h3 class="pmc_sec_title">4.2. Transcriptomic Analysis</h3>
<p>We had predicted that we would see lower mRNA levels of genes involved in phototransduction in the mid‐ and late‐stage cave embryos (surface‐biased expression). Supporting our prediction, six genes within the phototransduction group were surface‐biased in the mid‐stage timepoint, and seven genes were surface‐biased at the late‐stage timepoint (some of these genes are overlapping). This resembles what was seen in the Subterranean Rak population where three genes involved in phototransduction showed significantly different mRNA levels in cave and surface samples at both timepoints (Lomheim et al. <a href="#ede70014-bib-0047" class="usa-link" aria-describedby="ede70014-bib-0047">2023</a>); two genes in particular were in common between the analyses here and those involving the Subterranean Rak population—rhodopsin 1/6‐like and transient receptor potential channel pyrexia. Additionally, differential expression in cave versus surface spiders was also seen for opsins (Gainett et al. <a href="#ede70014-bib-0025" class="usa-link" aria-describedby="ede70014-bib-0025">2020</a>). What is somewhat surprising is that no genes involved in ommochrome synthesis were differentially expressed in the cave versus surface comparison, as ommochromes are thought to be the pigments present in <em>A. aquaticus</em> (Needham and Brunet <a href="#ede70014-bib-0060" class="usa-link" aria-describedby="ede70014-bib-0060">1957</a>). In comparisons of the Subterranean Rak population and Rakov Škocjan surface (Lomheim et al. <a href="#ede70014-bib-0047" class="usa-link" aria-describedby="ede70014-bib-0047">2023</a>), a single gene in the ommochrome pathway showed significantly different mRNA levels, <em>scarlet</em> (Lomheim et al. <a href="#ede70014-bib-0047" class="usa-link" aria-describedby="ede70014-bib-0047">2023</a>). It is possible that genes involved in the ommochrome synthesis pathway are lowly expressed at the timepoints examined, and that is why we did not detect significant differential expression in this set of genes.</p>
<p>We found that genes involved in melanin synthesis showed different mRNA levels between cave and surface samples: one paralogue of <em>dopamine N‐acetyltransferase</em> was cave‐biased, another paralogue of <em>dopamine N‐acetyltransferase</em> was surface‐biased, and <em>laccase 1</em> was surface‐biased. Interestingly, in comparisons of Subterranean Rak versus Rakov Škocjan surface, two paralogues of <em>dopamine N‐acetyltransferase</em> were surface‐biased (Lomheim et al. <a href="#ede70014-bib-0047" class="usa-link" aria-describedby="ede70014-bib-0047">2023</a>). However, in the current study, the cave and surface sequences of <em>dopamine N‐acetyltransferase</em> do appear to be the orthologue of the <em>dopamine N‐acetyltransferase</em> sequences from Subterranean Rak versus Rakov Škocjan surface that showed differential expression in the opposite direction indicating that for this gene, there is likely a difference in bias in the two cave populations. Differences in melanin synthesis are not surprising as they have been seen in other cave animals and might also relate to other functions of melanin pathway genes, such as melatonin synthesis (Bilandžija et al. <a href="#ede70014-bib-0010" class="usa-link" aria-describedby="ede70014-bib-0010">2012</a>; Bilandžija et al. <a href="#ede70014-bib-0009" class="usa-link" aria-describedby="ede70014-bib-0009">2013</a>; Bilandžija et al. <a href="#ede70014-bib-0007" class="usa-link" aria-describedby="ede70014-bib-0007">2018</a>). One of the differentially expressed genes discussed above, dopamine <em>N</em>‐acetyltransferase, blasts to <em>aanat</em>, which has been previously described as targeted for inactivating mutations in cave and deep‐sea organisms (Huang et al. <a href="#ede70014-bib-0037" class="usa-link" aria-describedby="ede70014-bib-0037">2022</a>). Furthermore, <em>aanat2</em> has been disrupted in <em>A. mexicanus</em>, by CRISPR, which resulted in decreased nighttime sleep (Mack et al. <a href="#ede70014-bib-0053" class="usa-link" aria-describedby="ede70014-bib-0053">2021</a>). In addition, two genes in the diurnal clock category showed decreased mRNA levels in cave samples as compared to surface samples, <em>period</em> and <em>PDF receptor</em>. Whether these expression differences indicate a behavioral difference between cave and surface populations, such as sleep, remains to be seen. Behavioral studies of another cave population of <em>A. aquaticus</em>, from Molnár János Cave, have shown a greater likelihood of feeding and higher movement when compared to a nearby surface population (Herczeg et al. <a href="#ede70014-bib-0036" class="usa-link" aria-describedby="ede70014-bib-0036">2020</a>; Berisha et al. <a href="#ede70014-bib-0006" class="usa-link" aria-describedby="ede70014-bib-0006">2022</a>). Additionally, a previous study involving <em>A. infernus</em> reported that cave individuals fed less but moved more as compared to surface individuals (Mösslacher and Creuzé des Châtelliers <a href="#ede70014-bib-0058" class="usa-link" aria-describedby="ede70014-bib-0058">1996</a>). Further behavioral studies between <em>A. infernus</em> and surface individuals will help to understand some of these gene expression differences.</p>
<p>Differential expression analysis allowed us to investigate genes and pathways that might be involved in cave‐specific phenotypes, such as eye and pigmentation loss, but also genes and pathways that were involved in sulfidic versus non‐sulfidic environments. Differential expression experiments had previously been performed on sulfidic and non‐sulfidic populations of the fish, <em>P. mexicana</em>. Lists of genes were identified that were overexpressed and underexpressed in multiple sulfidic populations versus multiple non‐sulfidic populations (Tobler et al. <a href="#ede70014-bib-0085" class="usa-link" aria-describedby="ede70014-bib-0085">2016</a>). Several genes were in common in the list of genes overexpressed in sulfidic populations of <em>P. mexicana</em> that also showed higher mRNA levels in <em>A. infernus</em> cave samples as compared to surface samples including <em>sulfide:quinone oxidoreductase (sqrdl)</em>, <em>persulfide dioxygenase ETHE1 (ethe1)</em>, <em>acidic amino acid decarboxylase GADL1 (gadl1)</em>, and <em>carbonic anhydrase 3</em>. In addition, one gene was biased toward surface samples in <em>A. aquaticus,</em> which was more lowly expressed in sulfidic populations in <em>P. mexicana</em>, <em>adenosylhomocysteinase 2</em>. Additional studies in <em>P. mexicana</em> showed evidence for genetic changes and selection in two of these genes, <em>sqrdl</em> and <em>ethe1,</em> in multiple sulfidic populations of <em>P. mexicana</em> (Brown et al. <a href="#ede70014-bib-0013" class="usa-link" aria-describedby="ede70014-bib-0013">2018</a>; Brown et al. <a href="#ede70014-bib-0014" class="usa-link" aria-describedby="ede70014-bib-0014">2019</a>; Greenway et al. <a href="#ede70014-bib-0030" class="usa-link" aria-describedby="ede70014-bib-0030">2020</a>; Ryan et al. <a href="#ede70014-bib-0075" class="usa-link" aria-describedby="ede70014-bib-0075">2023</a>). Both genes encode key enzymes in mitochondrial sulfide oxidation. Interestingly, one of the above genes also showed cave‐biased allele‐specific expression in our study, <em>sqrdl</em>. Additionally, we also saw cave‐biased allele‐specific expression in <em>gadl1</em>; when blasted, this gene showed the top hit in crustaceans as cysteine sulfinic acid decarboxylase (<em>csad</em>). <em>Csad</em> is involved in the biosynthesis of taurine, a sulfur‐containing amino acid which was shown to protect against oxidative stress (Baliou et al. <a href="#ede70014-bib-0003" class="usa-link" aria-describedby="ede70014-bib-0003">2021</a>). Interestingly, <em>csad</em> facilitates the production of hypotaurine, a precursor to taurine, and hypotaurine and thiotaurine are found at high concentrations in hydrothermal vent‐dwelling organisms and are thought to be protective in environments of high H<sub>2</sub>S (Yancey et al. <a href="#ede70014-bib-0091" class="usa-link" aria-describedby="ede70014-bib-0091">2009</a>). The parallels between the genes identified as likely to play a role in adaptation to the sulfidic environment between other organisms discussed above and <em>A. infernus</em> support the idea that multiple sulfidic organisms (from invertebrates to vertebrates) may have evolved similar solutions to adapt to sulfidic environments. However, it is important to keep in mind that in our current study in <em>A. aquaticus</em>, the comparison is between sulfidic cave and non‐sulfidic surface with two different parameters that are not separable: sulfidic versus non‐sulfidic and cave versus surface. However, because the <em>A. aquaticus</em> species complex contains both sulfidic cave and non‐sulfidic cave populations, comparison of sulfidic versus non‐sulfidic cave populations should be possible in the future. We do not currently know of any sulfidic surface <em>A. aquaticus</em> populations, but if discovered, the additional comparison of sulfidic surface to non‐sulfidic surface could be made.</p>
<p>As previously mentioned, <em>gadl1</em>, blasting to <em>csad</em>, showed cave‐biased allele‐specific expression in F1 hybrids. This could also be a possible candidate for eye phenotype as <em>csad</em> is in the cysteine metabolism pathway, similar to a gene contributing to eye degeneration in <em>A. mexicanus</em> is <em>cystathionine ß‐synthase a</em> (<em>cbsa</em>) (Ma et al. <a href="#ede70014-bib-0051" class="usa-link" aria-describedby="ede70014-bib-0051">2020</a>). <em>Csad</em> is also a possible pigment candidate gene as CRISPR gene editing in the planthopper, <em>Nilaparvata lugens</em>, showed increased pigmentation (Chen et al. <a href="#ede70014-bib-0016" class="usa-link" aria-describedby="ede70014-bib-0016">2021</a>).</p>
<p>Multiple other genes showed surface‐biased allele‐specific expression, including <em>ER membrane protein complex subunit 1</em>, <em>rap guanine nucleotide exchange factor 2</em>, and <em>tramtrack</em>. Humans with defects in <em>ER membrane protein complex subunit 1</em> have visual impairment (Geetha et al. <a href="#ede70014-bib-0027" class="usa-link" aria-describedby="ede70014-bib-0027">2018</a>), and therefore, this could be a candidate gene for the ommatidia loss phenotype. <em>Rap guanine nucleotide exchange factor 2</em> is also known as <em>pdz‐gef,</em> which is required for photoreceptor differentiation in <em>D. melanogaster</em> (Baril et al. <a href="#ede70014-bib-0004" class="usa-link" aria-describedby="ede70014-bib-0004">2014</a>). Interestingly, allele‐specific expression analyses in <em>A. mexicanus</em> showed allele‐specific expression in <em>rap guanine exchange factor 2</em> as well (Leclercq et al. <a href="#ede70014-bib-0044" class="usa-link" aria-describedby="ede70014-bib-0044">2024</a>), though cave‐biased rather than surface‐biased. <em>Tramtrack</em> blasted to <em>bric‐a‐brac,</em> which has been associated with pigmentation variation in <em>Drosophilidae</em> (reviewed by Massey and Wittkopp (<a href="#ede70014-bib-0054" class="usa-link" aria-describedby="ede70014-bib-0054">2016</a>)). Furthermore, <em>bric‐a‐brac</em> is known to play a role in <em>D. melanogaster</em> limb segmentation (Chu et al. <a href="#ede70014-bib-0017" class="usa-link" aria-describedby="ede70014-bib-0017">2002</a>) and therefore could also be involved in appendage differences between the cave and surface forms.</p>
<p>There was no overlap in the lists of genes that showed both allele‐specific expression and differential expression with log2 fold of 2 and <em>p</em> &lt; 0.05 between the Romanian samples described here (<em>A. infernus</em> and surface) and the previously described populations of Subterranean Rak and Rakov Škocjan (Lomheim et al. <a href="#ede70014-bib-0047" class="usa-link" aria-describedby="ede70014-bib-0047">2023</a>). However, if the log2 fold were lowered to 1.8 for cave versus surface analysis, there would be one gene in common between both cave/surface pair comparisons that showed cave‐biased allele‐specific expression and differential expression: EFR3 homolog cmp44E (<em>efr3</em>). The <em>Drosophila melanogaster</em> orthologue of <em>efr3</em> has been documented to play a role in phototransduction, hypoxia, and olfaction (Kain et al. <a href="#ede70014-bib-0038" class="usa-link" aria-describedby="ede70014-bib-0038">2009</a>; Balakrishnan et al. <a href="#ede70014-bib-0002" class="usa-link" aria-describedby="ede70014-bib-0002">2018</a>; Lu et al. <a href="#ede70014-bib-0049" class="usa-link" aria-describedby="ede70014-bib-0049">2022</a>). It is striking that this gene shows allele‐specific expression in both populations, which are geographically and ecologically disparate. Examination of other populations in the <em>A. aquaticus</em> species complex would help to inform what overexpression of this gene in the cave ecomorph might accomplish and whether this expression difference could result in adaptations in multiple cave environments. The lack of overlap in genes that show allele‐specific expression, aside from <em>efr3</em>, between the two cave/surface pair comparisons could be due to the restrictive nature of how we defined allele‐specific expression, or it could be because these cave populations have come to different solutions in the evolution to their respective subterranean environments. We do know, however, that <em>A. infernus</em> likely has the same genetic mechanism for an orange phenotype masked by depigmentation (Rodas et al. <a href="#ede70014-bib-0074" class="usa-link" aria-describedby="ede70014-bib-0074">2023</a>). Therefore, it is likely that there are both genetic commonalities and differences in the genetic basis of subterranean phenotypes in <em>A. infernus</em> and the Subterranean Rak population.</p>
<p>One limitation of our allele‐specific expression studies, as previously mentioned, was that we were only able to obtain F1 hybrids with cave males and surface females and not the reciprocal cross of cave females and surface males. To address mis‐identifying genes with parent‐of‐origin effect as genes with allele‐specific expression, all our genes with allele‐specific expression also showed differential expression, which one would not expect to see with genes of parent‐of‐origin effect. Future work will prioritize investigating other laboratory living conditions for <em>A. infernus</em> to improve the survival/reproduction of individuals and hopefully generate F1 crosses with cave females × surface males.</p>
<p>Another potential limitation of our studies is that we pooled together individuals from two different wells for the differential expression, allele‐specific expression, and comparative embryology studies. It is possible that these wells have connections between them. However, it is also possible that some degree of cryptic diversity might be present. Future studies can examine the genetic structure between the different wells and other populations of <em>A. infernus</em>.</p>
<p>In sum, we have established <em>A. infernus</em> as a model for both understanding cave‐specific phenotypes and adaptation to sulfidic environments. We now have the tools for future comparative work, both between sulfidic and non‐sulfidic cave populations of the <em>A. aquaticus</em> species complex, but also comparative work between sulfidic cave species—<em>A. infernus</em> and <em>P. mexicana</em>. Our transcriptomic work has identified candidate genes of interest that could be involved in adaptation to the cave environment and/or adaptation to sulfidic environments. Of particular interest are genes that show molecular convergence between <em>A. infernus</em> and <em>P. mexicana</em>. The next step will be to test these genes functionally to investigate what advantages they could be providing.</p></section></section><section id="ede70014-sec-0270"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec27"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="ede70014-suppl-0001"><div class="caption p"><p>
<strong>SupFiles: Infernus_6_7_25. Supplementary Figure 1:</strong> Principal Count Analysis (PCA) of count data of RNA‐seq data from Asellus infernus (cave) and A. aquaticus (surface) embryos A PCA was performed on the RNA‐seq count data from Asellus infernus (cave) and A. aquaticus (surface) embryos sampled at ~70% (mid‐stage) and ~90% (late‐stage) of embryonic development. <strong>Supplementary Figure 2:</strong> Embryonic development of cave samples of A. infernus. <strong>Supplementary Figure 3:</strong> Embryonic development of surface samples. <strong>Supplementary Figure 4:</strong> Gene expression matrix of genes mapped to the cave transcriptome. <strong>Supplementary Figure 5:</strong> Gene expression matrix of genes mapped to the surface transcriptome. <strong>Supplementary Table 1:</strong> Populations collected of A. aquaticus and A. infernus. <strong>Supplementary Table 2:</strong> Samples used for hatchling analysis. <strong>Supplementary Table 3:</strong> Samples used for Illumina sequencing. <strong>Supplementary Table 4:</strong> P‐values for statistical tests comparing cave, surface, and F1 samples. <strong>Supplementary Table 5:</strong> Factor loadings and fraction of variance explained (FVE) for first two principal components.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12326928/bin/EDE-27-e70014-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">EDE-27-e70014-s001.docx</a><sup> (10.3MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="ede70014-suppl-0002"><div class="caption p"><p>Supporting Information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12326928/bin/EDE-27-e70014-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">EDE-27-e70014-s002.xlsx</a><sup> (121.4KB, xlsx) </sup>
</div></div></section></section><section id="ede70014-sec-0260" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>Thank you to Gregor Bračko, Ruxandra Niţescu, Gergely Horvath, Viktoria Hafenscher, and Gergely Balazs for animal collection. Thank you to Peter Trontelj and Žiga Fišer for advice about the animals. Thank you to Dennis Sun and Julie Cridland for advice about allele‐specific expression. Thank you to Žiga Fišer and Jean Francois Flot for the critical reading of the manuscript. Thank you to Hillary Protas for advice about statistical tests. Thank you to the Genetic Epidemiology and Genomics Lab (GEGL), UC Berkeley, and the Vincent J. Coates Genomics Sequencing Laboratory, California Institute for Quantitative Biosciences (QB3), University of California, Berkeley, for sample preparation and sequencing, respectively. Thanks also for advice and assistance from the National Center for Genome Analysis Support (NSF ABI‐1759906 to Indiana University; NSF ABI‐1759914 to the Pittsburgh Supercomputing Center). This work was supported by the National Eye Institute of the National Institutes of Health under Award Number R15EY029499 to M.E.P. The content is solely the responsibility of the authors and does not necessarily represent the official views of the National Institutes of Health. S.S. and L.F. were supported by the National Center for Genome Analysis Support (NSF ABI‐1759906 to Indiana University; NSF ABI‐1759914 to the Pittsburgh Supercomputing Center). This work used Bridges‐2 through allocation request number BIO210174 from the Extreme Science and Engineering Discovery Environment (XSEDE), supported by the National Science Foundation (grant number #1548562). This research was funded by Biodiversa+, the European Biodiversity Partnership under the 2021‐2022 BiodivProtect joint call for research proposals, co‐funded by the European Commission (GA No. 101052342) and with the funding organisations Ministry of Universities and Research (Italy), Agencia Estatal de Investigación—Fundación Biodiversidad (Spain), Fundo Regional para a Ciência e Tecnologia (Portugal), Suomen Akatemia—Ministry of the Environment (Finland), Belgian Science Policy Office (Belgium), Agence Nationale de la Recherche (France), Deutsche Forschungsgemeinschaft e.V. (Germany), Schweizerischer Nationalfonds (Grant No. 31BD30_209583, Switzerland), Fonds zur Förderung der Wissenschaftlichen Forschung (Austria), Ministry of Higher Education, Science and Innovation (Slovenia), and the Executive Agency for Higher Education, Research, Development and Innovation Funding (Romania).</p></section><section id="ede70014-sec-0290"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The data that support the findings of this study are openly available in PRJNA957305 at <a href="https://www.ncbi.nlm.nih.gov" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov</a>, reference number PRJNA957305. All sequences discussed in this report are present in the National Center for Biotechnology Information, Sequencing Reads Archive (BioProject ID: PRJNA957305).</p></section><section id="ede70014-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ede70014-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="ede70014-bib-0001">
<cite>

Bakovic, V.
, Martin Cerezo M. L., Höglund A., et al. 2021. “The Genomics of Phenotypically Differentiated <em>Asellus aquaticus</em> Cave, Surface Stream and Lake Ecotypes.” Molecular Ecology
30: 3530–3547.
</cite> [<a href="https://doi.org/10.1111/mec.15987" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34002902/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=The%20Genomics%20of%20Phenotypically%20Differentiated%20Asellus%20aquaticus%20Cave,%20Surface%20Stream%20and%20Lake%20Ecotypes&amp;author=V.%20Bakovic&amp;author=M.%20L.%20Martin%20Cerezo&amp;author=A.%20H%C3%B6glund&amp;volume=30&amp;publication_year=2021&amp;pages=3530-3547&amp;pmid=34002902&amp;doi=10.1111/mec.15987&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0002">
<cite>

Balakrishnan, S. S.
, Basu U., Shinde D., Thakur R., Jaiswal M., and Raghu P.. 2018. “Regulation of PI4P Levels by PI4KIIIα During G‐Protein‐Coupled PLC Signaling in Drosophila Photoreceptors.” Journal of Cell Science
131: jcs217257.
</cite> [<a href="https://doi.org/10.1242/jcs.217257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6104824/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29980590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cell%20Science&amp;title=Regulation%20of%20PI4P%20Levels%20by%20PI4KIII%CE%B1%20During%20G%E2%80%90Protein%E2%80%90Coupled%20PLC%20Signaling%20in%20Drosophila%20Photoreceptors&amp;author=S.%20S.%20Balakrishnan&amp;author=U.%20Basu&amp;author=D.%20Shinde&amp;author=R.%20Thakur&amp;author=M.%20Jaiswal&amp;volume=131&amp;publication_year=2018&amp;pages=jcs217257&amp;pmid=29980590&amp;doi=10.1242/jcs.217257&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0003">
<cite>

Baliou, S.
, Adamaki M., Ioannou P., et al. 2021. “Protective Role of Taurine Against Oxidative Stress.” Molecular Medicine Reports
24: 1–19.</cite> [<a href="https://doi.org/10.3892/mmr.2021.12242" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8240184/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34184084/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Medicine%20Reports&amp;title=Protective%20Role%20of%20Taurine%20Against%20Oxidative%20Stress&amp;author=S.%20Baliou&amp;author=M.%20Adamaki&amp;author=P.%20Ioannou&amp;volume=24&amp;publication_year=2021&amp;pages=1-19&amp;pmid=34184084&amp;doi=10.3892/mmr.2021.12242&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0004">
<cite>

Baril, C.
, Lefrançois M., Sahmi M., Knævelsrud H., and Therrien M.. 2014. “Apical Accumulation of the Sevenless Receptor Tyrosine Kinase During Drosophila Eye Development Is Promoted by the Small GTPase Rap1.” Genetics
197: 1237–1250.
</cite> [<a href="https://doi.org/10.1534/genetics.114.166272" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4125397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24899161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Apical%20Accumulation%20of%20the%20Sevenless%20Receptor%20Tyrosine%20Kinase%20During%20Drosophila%20Eye%20Development%20Is%20Promoted%20by%20the%20Small%20GTPase%20Rap1&amp;author=C.%20Baril&amp;author=M.%20Lefran%C3%A7ois&amp;author=M.%20Sahmi&amp;author=H.%20Kn%C3%A6velsrud&amp;author=M.%20Therrien&amp;volume=197&amp;publication_year=2014&amp;pages=1237-1250&amp;pmid=24899161&amp;doi=10.1534/genetics.114.166272&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0005">
<cite>

Benjamini, Y.
, and Yekutieli D.. 2001. “The Control of the False Discovery Rate in Multiple Testing Under Dependency.” Annals of Statistics
4: 1165–1188.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Statistics&amp;title=The%20Control%20of%20the%20False%20Discovery%20Rate%20in%20Multiple%20Testing%20Under%20Dependency&amp;author=Y.%20Benjamini&amp;author=D.%20Yekutieli&amp;volume=4&amp;publication_year=2001&amp;pages=1165-1188&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0006">
<cite>

Berisha, H.
, Horváth G., Fišer Ž., Balázs G., Fišer C., and Herczeg G.. 2022. “Sex‐Dependent Increase of Movement Activity in the Freshwater Isopod <em>Asellus aquaticus</em> Following Adaptation to a Predator‐Free Cave Habitat.” Current Zoology
69: 418–425.
</cite> [<a href="https://doi.org/10.1093/cz/zoac063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10443615/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37614916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Zoology&amp;title=Sex%E2%80%90Dependent%20Increase%20of%20Movement%20Activity%20in%20the%20Freshwater%20Isopod%20Asellus%20aquaticus%20Following%20Adaptation%20to%20a%20Predator%E2%80%90Free%20Cave%20Habitat&amp;author=H.%20Berisha&amp;author=G.%20Horv%C3%A1th&amp;author=%C5%BD.%20Fi%C5%A1er&amp;author=G.%20Bal%C3%A1zs&amp;author=C.%20Fi%C5%A1er&amp;volume=69&amp;publication_year=2022&amp;pages=418-425&amp;pmid=37614916&amp;doi=10.1093/cz/zoac063&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0007">
<cite>

Bilandžija, H.
, Abraham L., Ma L., Renner K. J., and Jeffery W. R.. 2018. “Behavioural Changes Controlled by Catecholaminergic Systems Explain Recurrent Loss of Pigmentation in Cavefish.” Proceedings of the Royal Society B: Biological Sciences
285: 20180243.</cite> [<a href="https://doi.org/10.1098/rspb.2018.0243" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5966602/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29720416/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20Royal%20Society%20B:%20Biological%20Sciences&amp;title=Behavioural%20Changes%20Controlled%20by%20Catecholaminergic%20Systems%20Explain%20Recurrent%20Loss%20of%20Pigmentation%20in%20Cavefish&amp;author=H.%20Biland%C5%BEija&amp;author=L.%20Abraham&amp;author=L.%20Ma&amp;author=K.%20J.%20Renner&amp;author=W.%20R.%20Jeffery&amp;volume=285&amp;publication_year=2018&amp;pages=20180243&amp;pmid=29720416&amp;doi=10.1098/rspb.2018.0243&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0008">
<cite>

Bilandžija, H.
, Hollifield B., Steck M., et al. 2020. “Phenotypic Plasticity as a Mechanism of Cave Colonization and Adaptation.” eLife
9: e51830.
</cite> [<a href="https://doi.org/10.7554/eLife.51830" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7173965/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32314737/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Phenotypic%20Plasticity%20as%20a%20Mechanism%20of%20Cave%20Colonization%20and%20Adaptation&amp;author=H.%20Biland%C5%BEija&amp;author=B.%20Hollifield&amp;author=M.%20Steck&amp;volume=9&amp;publication_year=2020&amp;pages=e51830&amp;pmid=32314737&amp;doi=10.7554/eLife.51830&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0009">
<cite>

Bilandžija, H.
, Ma L., Parkhurst A., and Jeffery W. R.. 2013. “A Potential Benefit of Albinism in Astyanax Cavefish: Downregulation of the Oca2 Gene Increases Tyrosine and Catecholamine Levels as an Alternative to Melanin Synthesis.” PLoS One
8: e80823.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0080823" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3840000/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24282555/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=A%20Potential%20Benefit%20of%20Albinism%20in%20Astyanax%20Cavefish:%20Downregulation%20of%20the%20Oca2%20Gene%20Increases%20Tyrosine%20and%20Catecholamine%20Levels%20as%20an%20Alternative%20to%20Melanin%20Synthesis&amp;author=H.%20Biland%C5%BEija&amp;author=L.%20Ma&amp;author=A.%20Parkhurst&amp;author=W.%20R.%20Jeffery&amp;volume=8&amp;publication_year=2013&amp;pages=e80823&amp;pmid=24282555&amp;doi=10.1371/journal.pone.0080823&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0010">
<cite>

Bilandžija, H.
, Ćetković H., and Jeffery W. R.. 2012. “Evolution of Albinism in Cave Planthoppers by a Convergent Defect in the First Step of Melanin Biosynthesis.” Evolution &amp; Development
14: 196–203.
</cite> [<a href="https://doi.org/10.1111/j.1525-142X.2012.00535.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6169799/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23017027/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evolution%20&amp;%20Development&amp;title=Evolution%20of%20Albinism%20in%20Cave%20Planthoppers%20by%20a%20Convergent%20Defect%20in%20the%20First%20Step%20of%20Melanin%20Biosynthesis&amp;author=H.%20Biland%C5%BEija&amp;author=H.%20%C4%86etkovi%C4%87&amp;author=W.%20R.%20Jeffery&amp;volume=14&amp;publication_year=2012&amp;pages=196-203&amp;pmid=23017027&amp;doi=10.1111/j.1525-142X.2012.00535.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0011">
<cite>

Brad, T.
, Iepure S., and Sarbu S.. 2021. “The Chemoautotrophically Based Movile Cave Groundwater Ecosystem, a Hotspot of Subterranean Biodiversity.” Diversity
13: 128.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Diversity&amp;title=The%20Chemoautotrophically%20Based%20Movile%20Cave%20Groundwater%20Ecosystem,%20a%20Hotspot%20of%20Subterranean%20Biodiversity&amp;author=T.%20Brad&amp;author=S.%20Iepure&amp;author=S.%20Sarbu&amp;volume=13&amp;publication_year=2021&amp;pages=128&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0012">
<cite>

Bray, N. L.
, Pimentel H., Melsted P., and Pachter L.. 2016. “Near‐Optimal Probabilistic RNA‐seq Quantification.” Nature Biotechnology
34: 525–527.</cite> [<a href="https://doi.org/10.1038/nbt.3519" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27043002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Near%E2%80%90Optimal%20Probabilistic%20RNA%E2%80%90seq%20Quantification&amp;author=N.%20L.%20Bray&amp;author=H.%20Pimentel&amp;author=P.%20Melsted&amp;author=L.%20Pachter&amp;volume=34&amp;publication_year=2016&amp;pages=525-527&amp;pmid=27043002&amp;doi=10.1038/nbt.3519&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0013">
<cite>

Brown, A. P.
, Arias‐Rodriguez L., Yee M., Tobler M., and Kelley J. L.. 2018. “Concordant Changes in Gene Expression and Nucleotides Underlie Independent Adaptation to Hydrogen‐Sulfide‐Rich Environments.” Genome Biology and Evolution
10: 2867–2881.
</cite> [<a href="https://doi.org/10.1093/gbe/evy198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6225894/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30215710/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology%20and%20Evolution&amp;title=Concordant%20Changes%20in%20Gene%20Expression%20and%20Nucleotides%20Underlie%20Independent%20Adaptation%20to%20Hydrogen%E2%80%90Sulfide%E2%80%90Rich%20Environments&amp;author=A.%20P.%20Brown&amp;author=L.%20Arias%E2%80%90Rodriguez&amp;author=M.%20Yee&amp;author=M.%20Tobler&amp;author=J.%20L.%20Kelley&amp;volume=10&amp;publication_year=2018&amp;pages=2867-2881&amp;pmid=30215710&amp;doi=10.1093/gbe/evy198&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0014">
<cite>

Brown, A. P.
, McGowan K. L., Schwarzkopf E. J., et al. 2019. “Local Ancestry Analysis Reveals Genomic Convergence in Extremophile Fishes.” Philosophical Transactions of the Royal Society, B: Biological Sciences
374: 20180240.</cite> [<a href="https://doi.org/10.1098/rstb.2018.0240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6560267/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31154969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Philosophical%20Transactions%20of%20the%20Royal%20Society,%20B:%20Biological%20Sciences&amp;title=Local%20Ancestry%20Analysis%20Reveals%20Genomic%20Convergence%20in%20Extremophile%20Fishes&amp;author=A.%20P.%20Brown&amp;author=K.%20L.%20McGowan&amp;author=E.%20J.%20Schwarzkopf&amp;volume=374&amp;publication_year=2019&amp;pages=20180240&amp;pmid=31154969&amp;doi=10.1098/rstb.2018.0240&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0015">
<cite>

Camacho, C.
, Coulouris G., Avagyan V., et al. 2009. “BLAST: Architecture and Applications.” BMC Bioinformatics
10: 421.
</cite> [<a href="https://doi.org/10.1186/1471-2105-10-421" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2803857/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20003500/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=BLAST:%20Architecture%20and%20Applications&amp;author=C.%20Camacho&amp;author=G.%20Coulouris&amp;author=V.%20Avagyan&amp;volume=10&amp;publication_year=2009&amp;pages=421&amp;pmid=20003500&amp;doi=10.1186/1471-2105-10-421&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0016">
<cite>

Chen, J. X.
, Li W. X., Lyu J., Hu Y. T., Huang G., and Zhang W. Q.. 2021. “CRISPR/Cas9‐Mediated Knockout of the NlCSAD Gene Results in Darker Cuticle Pigmentation and a Reduction in Female Fecundity in <em>Nilaparvata lugens</em> (Hemiptera: Delphacidae).” Comparative Biochemistry and Physiology. Part A, Molecular &amp; Integrative Physiology
256: 110921.</cite> [<a href="https://doi.org/10.1016/j.cbpa.2021.110921" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33610762/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comparative%20Biochemistry%20and%20Physiology.%20Part%20A,%20Molecular%20&amp;%20Integrative%20Physiology&amp;title=CRISPR/Cas9%E2%80%90Mediated%20Knockout%20of%20the%20NlCSAD%20Gene%20Results%20in%20Darker%20Cuticle%20Pigmentation%20and%20a%20Reduction%20in%20Female%20Fecundity%20in%20Nilaparvata%20lugens%20(Hemiptera:%20Delphacidae)&amp;author=J.%20X.%20Chen&amp;author=W.%20X.%20Li&amp;author=J.%20Lyu&amp;author=Y.%20T.%20Hu&amp;author=G.%20Huang&amp;volume=256&amp;publication_year=2021&amp;pages=110921&amp;pmid=33610762&amp;doi=10.1016/j.cbpa.2021.110921&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0017">
<cite>

Chu, J.
, Dong P. D. S., and Panganiban G.. 2002. “Limb Type‐Specific Regulation of Bric a Brac Contributes to Morphological Diversity.” Development
129: 695–704.
</cite> [<a href="https://doi.org/10.1242/dev.129.3.695" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11830570/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Limb%20Type%E2%80%90Specific%20Regulation%20of%20Bric%20a%20Brac%20Contributes%20to%20Morphological%20Diversity&amp;author=J.%20Chu&amp;author=P.%20D.%20S.%20Dong&amp;author=G.%20Panganiban&amp;volume=129&amp;publication_year=2002&amp;pages=695-704&amp;pmid=11830570&amp;doi=10.1242/dev.129.3.695&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0018">
<cite>

Cock, P. J. A.
, Chilton J. M., Grüning B., Johnson J. E., and Soranzo N.. 2015. “NCBI BLAST Integrated Into Galaxy.” GigaScience
4: s13742–s13747.</cite> [<a href="https://doi.org/10.1186/s13742-015-0080-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4557756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26336600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=GigaScience&amp;title=NCBI%20BLAST%20Integrated%20Into%20Galaxy&amp;author=P.%20J.%20A.%20Cock&amp;author=J.%20M.%20Chilton&amp;author=B.%20Gr%C3%BCning&amp;author=J.%20E.%20Johnson&amp;author=N.%20Soranzo&amp;volume=4&amp;publication_year=2015&amp;pages=s13742-s13747&amp;pmid=26336600&amp;doi=10.1186/s13742-015-0080-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0019">
<cite>

Conesa, A.
, Götz S., García‐Gómez J. M., Terol J., Talón M., and Robles M.. 2005. “Blast2GO: A Universal Tool for Annotation, Visualization and Analysis in Functional Genomics Research.” Bioinformatics
21: 3674–3676.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bti610" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16081474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Blast2GO:%20A%20Universal%20Tool%20for%20Annotation,%20Visualization%20and%20Analysis%20in%20Functional%20Genomics%20Research&amp;author=A.%20Conesa&amp;author=S.%20G%C3%B6tz&amp;author=J.%20M.%20Garc%C3%ADa%E2%80%90G%C3%B3mez&amp;author=J.%20Terol&amp;author=M.%20Tal%C3%B3n&amp;volume=21&amp;publication_year=2005&amp;pages=3674-3676&amp;pmid=16081474&amp;doi=10.1093/bioinformatics/bti610&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0020">
<cite>

Culver, D. C.

1982. Cave Life: Evolution and Ecology. Harvard University Press.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Cave%20Life:%20Evolution%20and%20Ecology&amp;author=D.%20C.%20Culver&amp;publication_year=1982&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0021">
<cite>

Engel, A. S.

2007. “Observations on the Biodiversity of Sulfidic Karst Habitats.” Journal of Cave and Karst Studies
69: 187–206.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cave%20and%20Karst%20Studies&amp;title=Observations%20on%20the%20Biodiversity%20of%20Sulfidic%20Karst%20Habitats&amp;author=A.%20S.%20Engel&amp;volume=69&amp;publication_year=2007&amp;pages=187-206&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0022">
<cite>

Fišer, C.
, Luštrik R., Sarbu S., Flot J. F., and Trontelj P.. 2015. “Morphological Evolution of Coexisting Amphipod Species Pairs From Sulfidic Caves Suggests Competitive Interactions and Character Displacement, but No Environmental Filtering and Convergence.” PLoS One
10: e0123535.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0123535" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4407961/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25905793/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Morphological%20Evolution%20of%20Coexisting%20Amphipod%20Species%20Pairs%20From%20Sulfidic%20Caves%20Suggests%20Competitive%20Interactions%20and%20Character%20Displacement,%20but%20No%20Environmental%20Filtering%20and%20Convergence&amp;author=C.%20Fi%C5%A1er&amp;author=R.%20Lu%C5%A1trik&amp;author=S.%20Sarbu&amp;author=J.%20F.%20Flot&amp;author=P.%20Trontelj&amp;volume=10&amp;publication_year=2015&amp;pages=e0123535&amp;pmid=25905793&amp;doi=10.1371/journal.pone.0123535&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0023">
<cite>

Fišer, Ž.
, Whitehorn H., Furness T., Trontelj P., and Protas M.. 2024. “Genetic Bias in Repeated Evolution of Pigment Loss in Cave Populations of the <em>Asellus aquaticus</em> Species Complex.” Journal of Experimental Zoology Part B: Molecular and Developmental Evolution
342: 425–436.
</cite> [<a href="https://doi.org/10.1002/jez.b.23256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12322715/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38828691/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Zoology%20Part%20B:%20Molecular%20and%20Developmental%20Evolution&amp;title=Genetic%20Bias%20in%20Repeated%20Evolution%20of%20Pigment%20Loss%20in%20Cave%20Populations%20of%20the%20Asellus%20aquaticus%20Species%20Complex&amp;author=%C5%BD.%20Fi%C5%A1er&amp;author=H.%20Whitehorn&amp;author=T.%20Furness&amp;author=P.%20Trontelj&amp;author=M.%20Protas&amp;volume=342&amp;publication_year=2024&amp;pages=425-436&amp;pmid=38828691&amp;doi=10.1002/jez.b.23256&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0024">
<cite>

Flot, J. F.
, Wörheide G., and Dattagupta S.. 2010. “Unsuspected Diversity of Niphargus Amphipods in the Chemoautotrophic Cave Ecosystem of Frasassi, Central Italy.” BMC Evolutionary Biology
10: 171.
</cite> [<a href="https://doi.org/10.1186/1471-2148-10-171" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2896373/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20534131/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Evolutionary%20Biology&amp;title=Unsuspected%20Diversity%20of%20Niphargus%20Amphipods%20in%20the%20Chemoautotrophic%20Cave%20Ecosystem%20of%20Frasassi,%20Central%20Italy&amp;author=J.%20F.%20Flot&amp;author=G.%20W%C3%B6rheide&amp;author=S.%20Dattagupta&amp;volume=10&amp;publication_year=2010&amp;pages=171&amp;pmid=20534131&amp;doi=10.1186/1471-2148-10-171&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0025">
<cite>

Gainett, G.
, Ballesteros J. A., Kanzler C. R., et al. 2020. “Systemic Paralogy and Function of Retinal Determination Network Homologs in Arachnids.” BMC Genomics
21: 811.
</cite> [<a href="https://doi.org/10.1186/s12864-020-07149-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7681978/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33225889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Systemic%20Paralogy%20and%20Function%20of%20Retinal%20Determination%20Network%20Homologs%20in%20Arachnids&amp;author=G.%20Gainett&amp;author=J.%20A.%20Ballesteros&amp;author=C.%20R.%20Kanzler&amp;volume=21&amp;publication_year=2020&amp;pages=811&amp;pmid=33225889&amp;doi=10.1186/s12864-020-07149-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0026"><cite>

Garrison, E.
, and Marth G., 2012. Haplotype‐Based Variant Detection From Short‐Read Sequencing. arXiv, 1207.</cite></li>
<li id="ede70014-bib-0027">
<cite>

Geetha, T. S.
, Lingappa L., Jain A. R., et al. 2018. “A Novel Splice Variant in EMC1 Is Associated With Cerebellar Atrophy, Visual Impairment, Psychomotor Retardation With Epilepsy.” Molecular Genetics &amp; Genomic Medicine
6: 282–287.
</cite> [<a href="https://doi.org/10.1002/mgg3.352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5902391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29271071/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Genetics%20&amp;%20Genomic%20Medicine&amp;title=A%20Novel%20Splice%20Variant%20in%20EMC1%20Is%20Associated%20With%20Cerebellar%20Atrophy,%20Visual%20Impairment,%20Psychomotor%20Retardation%20With%20Epilepsy&amp;author=T.%20S.%20Geetha&amp;author=L.%20Lingappa&amp;author=A.%20R.%20Jain&amp;volume=6&amp;publication_year=2018&amp;pages=282-287&amp;pmid=29271071&amp;doi=10.1002/mgg3.352&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-1029"><cite>

Gilbert, D.

2013. “EvidentialGene: tr2aacds, mRNA transcript assembly software.” <a href="http://arthropods.eugenes.org/EvidentialGene" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://arthropods.eugenes.org/EvidentialGene</a>.</cite></li>
<li id="ede70014-bib-0029">
<cite>

Grabherr, M. G.
, Haas B. J., Yassour M., et al. 2011. “Full‐Length Transcriptome Assembly From RNA‐Seq Data Without a Reference Genome.” Nature Biotechnology
29: 644–652.</cite> [<a href="https://doi.org/10.1038/nbt.1883" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3571712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21572440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Full%E2%80%90Length%20Transcriptome%20Assembly%20From%20RNA%E2%80%90Seq%20Data%20Without%20a%20Reference%20Genome&amp;author=M.%20G.%20Grabherr&amp;author=B.%20J.%20Haas&amp;author=M.%20Yassour&amp;volume=29&amp;publication_year=2011&amp;pages=644-652&amp;pmid=21572440&amp;doi=10.1038/nbt.1883&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0030">
<cite>

Greenway, R.
, Barts N., Henpita C., et al. 2020. “Convergent Evolution of Conserved Mitochondrial Pathways Underlies Repeated Adaptation to Extreme Environments.” Proceedings of the National Academy of Sciences
117: 16424–16430.</cite> [<a href="https://doi.org/10.1073/pnas.2004223117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7368198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32586956/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Convergent%20Evolution%20of%20Conserved%20Mitochondrial%20Pathways%20Underlies%20Repeated%20Adaptation%20to%20Extreme%20Environments&amp;author=R.%20Greenway&amp;author=N.%20Barts&amp;author=C.%20Henpita&amp;volume=117&amp;publication_year=2020&amp;pages=16424-16430&amp;pmid=32586956&amp;doi=10.1073/pnas.2004223117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0031">
<cite>

Gregory, T. R.

2001. “Coincidence, Coevolution, or Causation? DNA Content, Cell Size, and the C‐Value Enigma.” Biological Reviews of the Cambridge Philosophical Society
76: 65–101.
</cite> [<a href="https://doi.org/10.1017/s1464793100005595" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11325054/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biological%20Reviews%20of%20the%20Cambridge%20Philosophical%20Society&amp;title=Coincidence,%20Coevolution,%20or%20Causation?%20DNA%20Content,%20Cell%20Size,%20and%20the%20C%E2%80%90Value%20Enigma&amp;author=T.%20R.%20Gregory&amp;volume=76&amp;publication_year=2001&amp;pages=65-101&amp;pmid=11325054&amp;doi=10.1017/s1464793100005595&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0032">
<cite>

Gross, J. B.
, Sun D. A., Carlson B. M., Brodo‐Abo S., and Protas M. E.. 2019. “Developmental Transcriptomic Analysis of the Cave‐Dwelling Crustacean, <em>Asellus aquaticus</em>
.” Genes
11: 42.
</cite> [<a href="https://doi.org/10.3390/genes11010042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7016750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31905778/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes&amp;title=Developmental%20Transcriptomic%20Analysis%20of%20the%20Cave%E2%80%90Dwelling%20Crustacean,%20Asellus%20aquaticus&amp;author=J.%20B.%20Gross&amp;author=D.%20A.%20Sun&amp;author=B.%20M.%20Carlson&amp;author=S.%20Brodo%E2%80%90Abo&amp;author=M.%20E.%20Protas&amp;volume=11&amp;publication_year=2019&amp;pages=42&amp;pmid=31905778&amp;doi=10.3390/genes11010042&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0033">
<cite>

Gurevich, A.
, Saveliev V., Vyahhi N., and Tesler G.. 2013. “QUAST: Quality Assessment Tool for Genome Assemblies.” Bioinformatics
29: 1072–1075.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3624806/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23422339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=QUAST:%20Quality%20Assessment%20Tool%20for%20Genome%20Assemblies&amp;author=A.%20Gurevich&amp;author=V.%20Saveliev&amp;author=N.%20Vyahhi&amp;author=G.%20Tesler&amp;volume=29&amp;publication_year=2013&amp;pages=1072-1075&amp;pmid=23422339&amp;doi=10.1093/bioinformatics/btt086&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0034"><cite>

Haas, B.

2023. Trinity Differential Expression. <a href="https://github.com/trinityrnaseq/trinityrnaseq/wiki/Trinity-Differential-Expression" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/trinityrnaseq/trinityrnaseq/wiki/Trinity-Differential-Expression</a>.</cite></li>
<li id="ede70014-bib-0035">
<cite>

Herczeg, G.
, Balázs G., Biró A., Fišer Ž., Kralj‐Fišer S., and Fišer C.. 2023. “Island and Rensch's Rules Do Not Apply to Cave Vs. Surface Populations of <em>Asellus aquaticus</em>
.” Frontiers in Ecology and Evolution
11: 1155261.</cite> [<a href="https://doi.org/10.1002/ece3.8233" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8571603/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34765185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Ecology%20and%20Evolution&amp;title=Island%20and%20Rensch's%20Rules%20Do%20Not%20Apply%20to%20Cave%20Vs.%20Surface%20Populations%20of%20Asellus%20aquaticus&amp;author=G.%20Herczeg&amp;author=G.%20Bal%C3%A1zs&amp;author=A.%20Bir%C3%B3&amp;author=%C5%BD.%20Fi%C5%A1er&amp;author=S.%20Kralj%E2%80%90Fi%C5%A1er&amp;volume=11&amp;publication_year=2023&amp;pages=1155261&amp;pmid=34765185&amp;doi=10.1002/ece3.8233&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0036">
<cite>

Herczeg, G.
, Hafenscher V. P., Balázs G., Fišer Ž., Kralj‐Fišer S., and Horváth G.. 2020. “Is Foraging Innovation Lost Following Colonization of a Less Variable Environment? A Case Study in Surface‐ Vs. Cave‐Dwelling <em>Asellus aquaticus</em>
.” Ecology and Evolution
10: 5323–5331.
</cite> [<a href="https://doi.org/10.1002/ece3.6276" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7319158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32607155/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecology%20and%20Evolution&amp;title=Is%20Foraging%20Innovation%20Lost%20Following%20Colonization%20of%20a%20Less%20Variable%20Environment?%20A%20Case%20Study%20in%20Surface%E2%80%90%20Vs.%20Cave%E2%80%90Dwelling%20Asellus%20aquaticus&amp;author=G.%20Herczeg&amp;author=V.%20P.%20Hafenscher&amp;author=G.%20Bal%C3%A1zs&amp;author=%C5%BD.%20Fi%C5%A1er&amp;author=S.%20Kralj%E2%80%90Fi%C5%A1er&amp;volume=10&amp;publication_year=2020&amp;pages=5323-5331&amp;pmid=32607155&amp;doi=10.1002/ece3.6276&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0037">
<cite>

Huang, Y.
, Li J., Bian C., Li R., You X., and Shi Q.. 2022. “Evolutionary Genomics Reveals Multiple Functions of Arylalkylamine N‐Acetyltransferase in Fish.” Frontiers in Genetics
13: 820442.
</cite> [<a href="https://doi.org/10.3389/fgene.2022.820442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9160868/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35664299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Genetics&amp;title=Evolutionary%20Genomics%20Reveals%20Multiple%20Functions%20of%20Arylalkylamine%20N%E2%80%90Acetyltransferase%20in%20Fish&amp;author=Y.%20Huang&amp;author=J.%20Li&amp;author=C.%20Bian&amp;author=R.%20Li&amp;author=X.%20You&amp;volume=13&amp;publication_year=2022&amp;pages=820442&amp;pmid=35664299&amp;doi=10.3389/fgene.2022.820442&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0038">
<cite>

Kain, P.
, Chandrashekaran S., Rodrigues V., and Hasan G.. 2009. “Drosophila Mutants in Phospholipid Signaling Have Reduced Olfactory Responses as Adults and Larvae.” Journal of Neurogenetics
23: 303–312.
</cite> [<a href="https://doi.org/10.1080/01677060802372494" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19089787/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Neurogenetics&amp;title=Drosophila%20Mutants%20in%20Phospholipid%20Signaling%20Have%20Reduced%20Olfactory%20Responses%20as%20Adults%20and%20Larvae&amp;author=P.%20Kain&amp;author=S.%20Chandrashekaran&amp;author=V.%20Rodrigues&amp;author=G.%20Hasan&amp;volume=23&amp;publication_year=2009&amp;pages=303-312&amp;pmid=19089787&amp;doi=10.1080/01677060802372494&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0039">
<cite>

Kelley, J. L.
, Passow C. N., Plath M., Rodriguez L. A., Yee M. C., and Tobler M.. 2012. “Genomic Resources for a Model in Adaptation and Speciation Research: Characterization of the <em>Poecilia mexicana</em> Transcriptome.” BMC Genomics
13: 652.
</cite> [<a href="https://doi.org/10.1186/1471-2164-13-652" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3585874/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23170846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Genomic%20Resources%20for%20a%20Model%20in%20Adaptation%20and%20Speciation%20Research:%20Characterization%20of%20the%20Poecilia%20mexicana%20Transcriptome&amp;author=J.%20L.%20Kelley&amp;author=C.%20N.%20Passow&amp;author=M.%20Plath&amp;author=L.%20A.%20Rodriguez&amp;author=M.%20C.%20Yee&amp;volume=13&amp;publication_year=2012&amp;pages=652&amp;pmid=23170846&amp;doi=10.1186/1471-2164-13-652&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0040">
<cite>

Konec, M.
, Prevorčnik S., Sarbu S. M., Verovnik R., and Trontelj P.. 2015. “Parallels Between Two Geographically and Ecologically Disparate Cave Invasions by the Same Species, <em>Asellus aquaticus</em> (Isopoda, Crustacea).” Journal of Evolutionary Biology
28: 864–875.
</cite> [<a href="https://doi.org/10.1111/jeb.12610" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25728816/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Evolutionary%20Biology&amp;title=Parallels%20Between%20Two%20Geographically%20and%20Ecologically%20Disparate%20Cave%20Invasions%20by%20the%20Same%20Species,%20Asellus%20aquaticus%20(Isopoda,%20Crustacea)&amp;author=M.%20Konec&amp;author=S.%20Prevor%C4%8Dnik&amp;author=S.%20M.%20Sarbu&amp;author=R.%20Verovnik&amp;author=P.%20Trontelj&amp;volume=28&amp;publication_year=2015&amp;pages=864-875&amp;pmid=25728816&amp;doi=10.1111/jeb.12610&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0041">
<cite>

Krishnan, J.
, Persons J. L., Peuß R., et al. 2020. “Comparative Transcriptome Analysis of Wild and Lab Populations of <em>Astyanax mexicanus</em> Uncovers Differential Effects of Environment and Morphotype on Gene Expression.” Journal of Experimental Zoology Part B: Molecular and Developmental Evolution
334: 530–539.
</cite> [<a href="https://doi.org/10.1002/jez.b.22933" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32017448/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Zoology%20Part%20B:%20Molecular%20and%20Developmental%20Evolution&amp;title=Comparative%20Transcriptome%20Analysis%20of%20Wild%20and%20Lab%20Populations%20of%20Astyanax%20mexicanus%20Uncovers%20Differential%20Effects%20of%20Environment%20and%20Morphotype%20on%20Gene%20Expression&amp;author=J.%20Krishnan&amp;author=J.%20L.%20Persons&amp;author=R.%20Peu%C3%9F&amp;volume=334&amp;publication_year=2020&amp;pages=530-539&amp;pmid=32017448&amp;doi=10.1002/jez.b.22933&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0042">
<cite>

Lafuente, E.
, Lürig M. D., Rövekamp M., et al. 2021. “Building on 150 Years of Knowledge: The Freshwater Isopod <em>Asellus aquaticus</em> as an Integrative Eco‐Evolutionary Model System.” Frontiers in Ecology and Evolution
9: 748212.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Ecology%20and%20Evolution&amp;title=Building%20on%20150%20Years%20of%20Knowledge:%20The%20Freshwater%20Isopod%20Asellus%20aquaticus%20as%20an%20Integrative%20Eco%E2%80%90Evolutionary%20Model%20System&amp;author=E.%20Lafuente&amp;author=M.%20D.%20L%C3%BCrig&amp;author=M.%20R%C3%B6vekamp&amp;volume=9&amp;publication_year=2021&amp;pages=748212&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0043">
<cite>

Langmead, B.
, and Salzberg S. L.. 2012. “Fast Gapped‐Read Alignment With Bowtie 2.” Nature Methods
9: 357–359.
</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=Fast%20Gapped%E2%80%90Read%20Alignment%20With%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.%20L.%20Salzberg&amp;volume=9&amp;publication_year=2012&amp;pages=357-359&amp;pmid=22388286&amp;doi=10.1038/nmeth.1923&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0044">
<cite>

Leclercq, J.
, Torres‐Paz J., Policarpo M., Agnès F., and Rétaux S.. 2024. “Evolution of the Regulation of Developmental Gene Expression in Blind Mexican Cavefish.” Development
151: dev202610.
</cite> [<a href="https://doi.org/10.1242/dev.202610" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39007346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=Evolution%20of%20the%20Regulation%20of%20Developmental%20Gene%20Expression%20in%20Blind%20Mexican%20Cavefish&amp;author=J.%20Leclercq&amp;author=J.%20Torres%E2%80%90Paz&amp;author=M.%20Policarpo&amp;author=F.%20Agn%C3%A8s&amp;author=S.%20R%C3%A9taux&amp;volume=151&amp;publication_year=2024&amp;pages=dev202610&amp;pmid=39007346&amp;doi=10.1242/dev.202610&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0045">
<cite>

Lefébure, T.
, Morvan C., Malard F., et al. 2017. “Less Effective Selection Leads to Larger Genomes.” Genome Research
27: 1016–1028.
</cite> [<a href="https://doi.org/10.1101/gr.212589.116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5453316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28424354/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=Less%20Effective%20Selection%20Leads%20to%20Larger%20Genomes&amp;author=T.%20Lef%C3%A9bure&amp;author=C.%20Morvan&amp;author=F.%20Malard&amp;volume=27&amp;publication_year=2017&amp;pages=1016-1028&amp;pmid=28424354&amp;doi=10.1101/gr.212589.116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0046">
<cite>

Li, H.

2018. “Minimap2: Pairwise Alignment for Nucleotide Sequences.” Bioinformatics
34: 3094–3100. 10.1093/bioinformatics/bty191.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6137996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29750242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Minimap2:%20Pairwise%20Alignment%20for%20Nucleotide%20Sequences&amp;author=H.%20Li&amp;volume=34&amp;publication_year=2018&amp;pages=3094-3100&amp;pmid=29750242&amp;doi=10.1093/bioinformatics/bty191&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0047">
<cite>

Lomheim, H. J.
, Reyes Rodas L., Mulla L., et al. 2023. “Transcriptomic Analysis of Cave, Surface, and Hybrid Samples of the Isopod <em>Asellus aquaticus</em> and Identification of Chromosomal Location of Candidate Genes for Cave Phenotype Evolution.” EvoDevo
14: 9.
</cite> [<a href="https://doi.org/10.1186/s13227-023-00213-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10163715/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37149716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EvoDevo&amp;title=Transcriptomic%20Analysis%20of%20Cave,%20Surface,%20and%20Hybrid%20Samples%20of%20the%20Isopod%20Asellus%20aquaticus%20and%20Identification%20of%20Chromosomal%20Location%20of%20Candidate%20Genes%20for%20Cave%20Phenotype%20Evolution&amp;author=H.%20J.%20Lomheim&amp;author=L.%20Reyes%20Rodas&amp;author=L.%20Mulla&amp;volume=14&amp;publication_year=2023&amp;pages=9&amp;pmid=37149716&amp;doi=10.1186/s13227-023-00213-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0048">
<cite>

Love, M. I.
, Huber W., and Anders S.. 2014. “Moderated Estimation of Fold Change and Dispersion for RNA‐seq Data With DESeq2.” Genome Biology
15: 550.
</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Moderated%20Estimation%20of%20Fold%20Change%20and%20Dispersion%20for%20RNA%E2%80%90seq%20Data%20With%20DESeq2&amp;author=M.%20I.%20Love&amp;author=W.%20Huber&amp;author=S.%20Anders&amp;volume=15&amp;publication_year=2014&amp;pages=550&amp;pmid=25516281&amp;doi=10.1186/s13059-014-0550-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0049">
<cite>

Lu, J.
, Dong W., Hammond G. R., and Hong Y.. 2022. “Hypoxia Controls Plasma Membrane Targeting of Polarity Proteins by Dynamic Turnover of PI4P and PI (4, 5) P2.” Frontiers in Ecology and Evolution
11: e79582.</cite> [<a href="https://doi.org/10.7554/eLife.79582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9242647/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35678383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Ecology%20and%20Evolution&amp;title=Hypoxia%20Controls%20Plasma%20Membrane%20Targeting%20of%20Polarity%20Proteins%20by%20Dynamic%20Turnover%20of%20PI4P%20and%20PI%20(4,%205)%20P2&amp;author=J.%20Lu&amp;author=W.%20Dong&amp;author=G.%20R.%20Hammond&amp;author=Y.%20Hong&amp;volume=11&amp;publication_year=2022&amp;pages=e79582&amp;pmid=35678383&amp;doi=10.7554/eLife.79582&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0050">
<cite>

Lukić, M.
, Jovović L., Bedek J., et al. 2024. “A Practical Guide for the Husbandry of Cave and Surface Invertebrates as the First Step in Establishing New Model Organisms.” PLoS One
19, no. 4: e0300962.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0300962" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10994295/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38573919/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=A%20Practical%20Guide%20for%20the%20Husbandry%20of%20Cave%20and%20Surface%20Invertebrates%20as%20the%20First%20Step%20in%20Establishing%20New%20Model%20Organisms&amp;author=M.%20Luki%C4%87&amp;author=L.%20Jovovi%C4%87&amp;author=J.%20Bedek&amp;volume=19&amp;issue=4&amp;publication_year=2024&amp;pages=e0300962&amp;pmid=38573919&amp;doi=10.1371/journal.pone.0300962&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0051">
<cite>

Ma, L.
, Gore A. V., Castranova D., et al. 2020. “A Hypomorphic Cystathionine ß‐Synthase Gene Contributes to Cavefish Eye Loss by Disrupting Optic Vasculature.” Nature Communications
11: 2772.</cite> [<a href="https://doi.org/10.1038/s41467-020-16497-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7265459/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32487986/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=A%20Hypomorphic%20Cystathionine%20%C3%9F%E2%80%90Synthase%20Gene%20Contributes%20to%20Cavefish%20Eye%20Loss%20by%20Disrupting%20Optic%20Vasculature&amp;author=L.%20Ma&amp;author=A.%20V.%20Gore&amp;author=D.%20Castranova&amp;volume=11&amp;publication_year=2020&amp;pages=2772&amp;pmid=32487986&amp;doi=10.1038/s41467-020-16497-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0052">
<cite>

Ma, L.
, Strickler A. G., Parkhurst A., Yoshizawa M., Shi J., and Jeffery W. R.. 2018. “Maternal Genetic Effects in Astyanax Cavefish Development.” Developmental Biology
441: 209–220.
</cite> [<a href="https://doi.org/10.1016/j.ydbio.2018.07.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6148371/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30031754/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Developmental%20Biology&amp;title=Maternal%20Genetic%20Effects%20in%20Astyanax%20Cavefish%20Development&amp;author=L.%20Ma&amp;author=A.%20G.%20Strickler&amp;author=A.%20Parkhurst&amp;author=M.%20Yoshizawa&amp;author=J.%20Shi&amp;volume=441&amp;publication_year=2018&amp;pages=209-220&amp;pmid=30031754&amp;doi=10.1016/j.ydbio.2018.07.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0053">
<cite>

Mack, K. L.
, Jaggard J. B., Persons J. L., et al. 2021. “Repeated Evolution of Circadian Clock Dysregulation in Cavefish Populations.” PLoS Genetics
17: e1009642.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1009642" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8297936/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34252077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genetics&amp;title=Repeated%20Evolution%20of%20Circadian%20Clock%20Dysregulation%20in%20Cavefish%20Populations&amp;author=K.%20L.%20Mack&amp;author=J.%20B.%20Jaggard&amp;author=J.%20L.%20Persons&amp;volume=17&amp;publication_year=2021&amp;pages=e1009642&amp;pmid=34252077&amp;doi=10.1371/journal.pgen.1009642&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0054">
<cite>

Massey, J. H.
, and Wittkopp P. J.. 2016. “The Genetic Basis of Pigmentation Differences Within and Between Drosophila Species.” Current Topics in Developmental Biology
119: 27–61.
</cite> [<a href="https://doi.org/10.1016/bs.ctdb.2016.03.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5002358/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27282023/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Topics%20in%20Developmental%20Biology&amp;title=The%20Genetic%20Basis%20of%20Pigmentation%20Differences%20Within%20and%20Between%20Drosophila%20Species&amp;author=J.%20H.%20Massey&amp;author=P.%20J.%20Wittkopp&amp;volume=119&amp;publication_year=2016&amp;pages=27-61&amp;pmid=27282023&amp;doi=10.1016/bs.ctdb.2016.03.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0055">
<cite>

McCauley, D. W.
, Hixon E., and Jeffery W. R.. 2004. “Evolution of Pigment Cell Regression in the Cavefish Astyanax: A Late Step in Melanogenesis.” Evolution &amp; Development
6: 209–218.
</cite> [<a href="https://doi.org/10.1111/j.1525-142X.2004.04026.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15230961/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evolution%20&amp;%20Development&amp;title=Evolution%20of%20Pigment%20Cell%20Regression%20in%20the%20Cavefish%20Astyanax:%20A%20Late%20Step%20in%20Melanogenesis&amp;author=D.%20W.%20McCauley&amp;author=E.%20Hixon&amp;author=W.%20R.%20Jeffery&amp;volume=6&amp;publication_year=2004&amp;pages=209-218&amp;pmid=15230961&amp;doi=10.1111/j.1525-142X.2004.04026.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0056">
<cite>

McGowan, K. L.
, Passow C. N., Arias‐Rodriguez L., Tobler M., and Kelley J. L.. 2019. “Expression Analyses of Cave Mollies (<em>Poecilia mexicana</em>) Reveal Key Genes Involved in the Early Evolution of Eye Regression.” Biology Letters
15: 20190554.
</cite> [<a href="https://doi.org/10.1098/rsbl.2019.0554" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6832190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31640527/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biology%20Letters&amp;title=Expression%20Analyses%20of%20Cave%20Mollies%20(Poecilia%20mexicana)%20Reveal%20Key%20Genes%20Involved%20in%20the%20Early%20Evolution%20of%20Eye%20Regression&amp;author=K.%20L.%20McGowan&amp;author=C.%20N.%20Passow&amp;author=L.%20Arias%E2%80%90Rodriguez&amp;author=M.%20Tobler&amp;author=J.%20L.%20Kelley&amp;volume=15&amp;publication_year=2019&amp;pages=20190554&amp;pmid=31640527&amp;doi=10.1098/rsbl.2019.0554&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0057">
<cite>

Mojaddidi, H.
, Fernandez F. E., Erickson P. A., and Protas M. E.. 2018. “Embryonic Origin and Genetic Basis of Cave Associated Phenotypes in the Isopod Crustacean <em>Asellus aquaticus</em>
.” Scientific Reports
8: 16589.
</cite> [<a href="https://doi.org/10.1038/s41598-018-34405-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6224564/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30409988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Embryonic%20Origin%20and%20Genetic%20Basis%20of%20Cave%20Associated%20Phenotypes%20in%20the%20Isopod%20Crustacean%20Asellus%20aquaticus&amp;author=H.%20Mojaddidi&amp;author=F.%20E.%20Fernandez&amp;author=P.%20A.%20Erickson&amp;author=M.%20E.%20Protas&amp;volume=8&amp;publication_year=2018&amp;pages=16589&amp;pmid=30409988&amp;doi=10.1038/s41598-018-34405-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0058">
<cite>

Mösslacher, F.
, and Creuzé des Châtelliers M.. 1996. “Physiological and Behavioural Adaptations of an Epigean and a Hypogean Dwelling Population of <em>Asellus aquaticus</em> (L.) (Crustacea, Isopoda).” Archiv für Hydrobiologie
138: 187–198.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Archiv%20f%C3%BCr%20Hydrobiologie&amp;title=Physiological%20and%20Behavioural%20Adaptations%20of%20an%20Epigean%20and%20a%20Hypogean%20Dwelling%20Population%20of%20Asellus%20aquaticus%20(L.)%20(Crustacea,%20Isopoda)&amp;author=F.%20M%C3%B6sslacher&amp;author=M.%20Creuz%C3%A9%20des%20Ch%C3%A2telliers&amp;volume=138&amp;publication_year=1996&amp;pages=187-198&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0059">
<cite>

Nariai, N.
, Kojima K., Mimori T., Kawai Y., and Nagasaki M.. 2016. “A Bayesian Approach for Estimating Allele‐Specific Expression From RNA‐Seq Data With Diploid Genomes.” BMC Genomics
17: 2.
</cite> [<a href="https://doi.org/10.1186/s12864-015-2295-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4895278/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26818838/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=A%20Bayesian%20Approach%20for%20Estimating%20Allele%E2%80%90Specific%20Expression%20From%20RNA%E2%80%90Seq%20Data%20With%20Diploid%20Genomes&amp;author=N.%20Nariai&amp;author=K.%20Kojima&amp;author=T.%20Mimori&amp;author=Y.%20Kawai&amp;author=M.%20Nagasaki&amp;volume=17&amp;publication_year=2016&amp;pages=2&amp;pmid=26818838&amp;doi=10.1186/s12864-015-2295-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0060">
<cite>

Needham, A. E.
, and Brunet P.. 1957. “The Integumental Pigment of Asellus.” Comparative Biochemistry and Physiology
13: 207–209.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Comparative%20Biochemistry%20and%20Physiology&amp;title=The%20Integumental%20Pigment%20of%20Asellus&amp;author=A.%20E.%20Needham&amp;author=P.%20Brunet&amp;volume=13&amp;publication_year=1957&amp;pages=207-209&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0061">
<cite>

O'Gorman, M.
, Thakur S., Imrie G., et al. 2021. “Pleiotropic Function of the oca2 Gene Underlies the Evolution of Sleep Loss and Albinism in Cavefish.” Current Biology
31: 3694–3701.e4.
</cite> [<a href="https://doi.org/10.1016/j.cub.2021.06.077" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34293332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Biology&amp;title=Pleiotropic%20Function%20of%20the%20oca2%20Gene%20Underlies%20the%20Evolution%20of%20Sleep%20Loss%20and%20Albinism%20in%20Cavefish&amp;author=M.%20O'Gorman&amp;author=S.%20Thakur&amp;author=G.%20Imrie&amp;volume=31&amp;publication_year=2021&amp;pages=3694-3701.e4&amp;pmid=34293332&amp;doi=10.1016/j.cub.2021.06.077&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0062">
<cite>

Passow, C. N.
, Brown A. P., Arias‐Rodriguez L., et al. 2017. “Complexities of Gene Expression Patterns in Natural Populations of an Extremophile Fish (<em>Poecilia mexicana</em>, Poeciliidae).” Molecular Ecology
26: 4211–4225.
</cite> [<a href="https://doi.org/10.1111/mec.14198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5731456/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28598519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Complexities%20of%20Gene%20Expression%20Patterns%20in%20Natural%20Populations%20of%20an%20Extremophile%20Fish%20(Poecilia%20mexicana,%20Poeciliidae)&amp;author=C.%20N.%20Passow&amp;author=A.%20P.%20Brown&amp;author=L.%20Arias%E2%80%90Rodriguez&amp;volume=26&amp;publication_year=2017&amp;pages=4211-4225&amp;pmid=28598519&amp;doi=10.1111/mec.14198&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0063">
<cite>

Passow, C. N.
, Henpita C., Shaw J. H., et al. 2017. “The Roles of Plasticity and Evolutionary Change in Shaping Gene Expression Variation in Natural Populations of Extremophile Fish.” Molecular Ecology
26: 6384–6399.
</cite> [<a href="https://doi.org/10.1111/mec.14360" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28926156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=The%20Roles%20of%20Plasticity%20and%20Evolutionary%20Change%20in%20Shaping%20Gene%20Expression%20Variation%20in%20Natural%20Populations%20of%20Extremophile%20Fish&amp;author=C.%20N.%20Passow&amp;author=C.%20Henpita&amp;author=J.%20H.%20Shaw&amp;volume=26&amp;publication_year=2017&amp;pages=6384-6399&amp;pmid=28926156&amp;doi=10.1111/mec.14360&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0064">
<cite>

Pérez‐Moreno, J. L.
, Balázs G., and Bracken‐Grissom H. D.. 2018. “Transcriptomic Insights Into the Loss of Vision in Molnár János Cave's Crustaceans.” Integrative and Comparative Biology
58: 452–464.
</cite> [<a href="https://doi.org/10.1093/icb/icy071" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29931265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Integrative%20and%20Comparative%20Biology&amp;title=Transcriptomic%20Insights%20Into%20the%20Loss%20of%20Vision%20in%20Moln%C3%A1r%20J%C3%A1nos%20Cave's%20Crustaceans&amp;author=J.%20L.%20P%C3%A9rez%E2%80%90Moreno&amp;author=G.%20Bal%C3%A1zs&amp;author=H.%20D.%20Bracken%E2%80%90Grissom&amp;volume=58&amp;publication_year=2018&amp;pages=452-464&amp;pmid=29931265&amp;doi=10.1093/icb/icy071&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0065">
<cite>

Protas, M.
, and Jeffery W. R.. 2012. “Evolution and Development in Cave Animals: From Fish to Crustaceans.” WIREs Developmental Biology
1: 823–845.
</cite> [<a href="https://doi.org/10.1002/wdev.61" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3620605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23580903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=WIREs%20Developmental%20Biology&amp;title=Evolution%20and%20Development%20in%20Cave%20Animals:%20From%20Fish%20to%20Crustaceans&amp;author=M.%20Protas&amp;author=W.%20R.%20Jeffery&amp;volume=1&amp;publication_year=2012&amp;pages=823-845&amp;pmid=23580903&amp;doi=10.1002/wdev.61&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0066">
<cite>

Protas, M.
, Trontelj P., Prevorčnik S., and Fišer Ž.. 2023. “The <em>Asellus aquaticus</em> Species Complex: An Invertebrate Model in Subterranean Evolution.” In Groundwater Ecology and Evolution, 329–350. Elsevier.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Groundwater%20Ecology%20and%20Evolution&amp;author=M.%20Protas&amp;author=P.%20Trontelj&amp;author=S.%20Prevor%C4%8Dnik&amp;author=%C5%BD.%20Fi%C5%A1er&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0067">
<cite>

Protas, M. E.
, Trontelj P., and Patel N. H.. 2011. “Genetic Basis of Eye and Pigment Loss in the Cave Crustacean, <em>Asellus aquaticus</em>
.” Proceedings of the National Academy of Sciences
108: 5702–5707.</cite> [<a href="https://doi.org/10.1073/pnas.1013850108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3078413/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21422298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Genetic%20Basis%20of%20Eye%20and%20Pigment%20Loss%20in%20the%20Cave%20Crustacean,%20Asellus%20aquaticus&amp;author=M.%20E.%20Protas&amp;author=P.%20Trontelj&amp;author=N.%20H.%20Patel&amp;volume=108&amp;publication_year=2011&amp;pages=5702-5707&amp;pmid=21422298&amp;doi=10.1073/pnas.1013850108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-1068"><cite>
R Core Team
. 2022. “
<em>R: A Language and Environment for Statistical Computing</em>. R Foundation for Statistical Computing.” <a href="https://www.Rproject.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.Rproject.org/</a>.</cite></li>
<li id="ede70014-bib-0068">
<cite>

Raudvere, U.
, Kolberg L., Kuzmin I., et al. 2019. “G: Profiler: A Web Server for Functional Enrichment Analysis and Conversions of Gene Lists (2019 Update).” Nucleic Acids Research
47: W191–W198.
</cite> [<a href="https://doi.org/10.1093/nar/gkz369" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6602461/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31066453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=G:%20Profiler:%20A%20Web%20Server%20for%20Functional%20Enrichment%20Analysis%20and%20Conversions%20of%20Gene%20Lists%20(2019%20Update)&amp;author=U.%20Raudvere&amp;author=L.%20Kolberg&amp;author=I.%20Kuzmin&amp;volume=47&amp;publication_year=2019&amp;pages=W191-W198&amp;pmid=31066453&amp;doi=10.1093/nar/gkz369&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0069">
<cite>

Re, C.
, Fišer Ž., Perez J., Tacdol A., Trontelj P., and Protas M. E.. 2018. “Common Genetic Basis of Eye and Pigment Loss in Two Distinct Cave Populations of the Isopod Crustacean <em>Asellus aquaticus</em>
.” Integrative and Comparative Biology
58: 421–430.
</cite> [<a href="https://doi.org/10.1093/icb/icy028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29790967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Integrative%20and%20Comparative%20Biology&amp;title=Common%20Genetic%20Basis%20of%20Eye%20and%20Pigment%20Loss%20in%20Two%20Distinct%20Cave%20Populations%20of%20the%20Isopod%20Crustacean%20Asellus%20aquaticus&amp;author=C.%20Re&amp;author=%C5%BD.%20Fi%C5%A1er&amp;author=J.%20Perez&amp;author=A.%20Tacdol&amp;author=P.%20Trontelj&amp;volume=58&amp;publication_year=2018&amp;pages=421-430&amp;pmid=29790967&amp;doi=10.1093/icb/icy028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0070">
<cite>

Ridley, M.
, and Thompson D. J.. 1979. “Size and Mating in <em>Asellus aquaticus</em> (Crustacea: Isopoda).” Zeitschrift für Tierpsychologie
51: 380–397.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Zeitschrift%20f%C3%BCr%20Tierpsychologie&amp;title=Size%20and%20Mating%20in%20Asellus%20aquaticus%20(Crustacea:%20Isopoda)&amp;author=M.%20Ridley&amp;author=D.%20J.%20Thompson&amp;volume=51&amp;publication_year=1979&amp;pages=380-397&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0071">
<cite>

Riesch, R.
, Plath M., and Schlupp I.. 2010. “Toxic Hydrogen Sulfide and Dark Caves: Life‐History Adaptations in a Livebearing Fish (<em>Poecilia mexicana</em>, Poeciliidae).” Ecology
91: 1494–1505.
</cite> [<a href="https://doi.org/10.1890/09-1008.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20503881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecology&amp;title=Toxic%20Hydrogen%20Sulfide%20and%20Dark%20Caves:%20Life%E2%80%90History%20Adaptations%20in%20a%20Livebearing%20Fish%20(Poecilia%20mexicana,%20Poeciliidae)&amp;author=R.%20Riesch&amp;author=M.%20Plath&amp;author=I.%20Schlupp&amp;volume=91&amp;publication_year=2010&amp;pages=1494-1505&amp;pmid=20503881&amp;doi=10.1890/09-1008.1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0072">
<cite>

Riesch, R.
, Plath M., Schlupp I., Tobler M., and Brian Langerhans R.. 2014. “Colonisation of Toxic Environments Drives Predictable Life‐History Evolution in Livebearing Fishes (Poeciliidae).” Ecology Letters
17: 65–71.
</cite> [<a href="https://doi.org/10.1111/ele.12209" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24188245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecology%20Letters&amp;title=Colonisation%20of%20Toxic%20Environments%20Drives%20Predictable%20Life%E2%80%90History%20Evolution%20in%20Livebearing%20Fishes%20(Poeciliidae)&amp;author=R.%20Riesch&amp;author=M.%20Plath&amp;author=I.%20Schlupp&amp;author=M.%20Tobler&amp;author=R.%20Brian%20Langerhans&amp;volume=17&amp;publication_year=2014&amp;pages=65-71&amp;pmid=24188245&amp;doi=10.1111/ele.12209&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0073">
<cite>

Robertson, G.
, Schein J., Chiu R., et al. 2010. “De Novo Assembly and Analysis of RNA‐Seq Data.” Nature Methods
7: 909–912.
</cite> [<a href="https://doi.org/10.1038/nmeth.1517" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20935650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=De%20Novo%20Assembly%20and%20Analysis%20of%20RNA%E2%80%90Seq%20Data&amp;author=G.%20Robertson&amp;author=J.%20Schein&amp;author=R.%20Chiu&amp;volume=7&amp;publication_year=2010&amp;pages=909-912&amp;pmid=20935650&amp;doi=10.1038/nmeth.1517&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0074">
<cite>

Rodas, L. R.
, Sarbu S. M., Bancila R., Price D., Fišer Ž., and Protas M.. 2023. “Standing Genetic Variation as a Potential Mechanism of Novel Cave Phenotype Evolution in the Freshwater Isopod, <em>Asellus aquaticus</em>
.” Evolution &amp; Development
25: 137–152.
</cite> [<a href="https://doi.org/10.1111/ede.12428" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10331845/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36755467/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evolution%20&amp;%20Development&amp;title=Standing%20Genetic%20Variation%20as%20a%20Potential%20Mechanism%20of%20Novel%20Cave%20Phenotype%20Evolution%20in%20the%20Freshwater%20Isopod,%20Asellus%20aquaticus&amp;author=L.%20R.%20Rodas&amp;author=S.%20M.%20Sarbu&amp;author=R.%20Bancila&amp;author=D.%20Price&amp;author=%C5%BD.%20Fi%C5%A1er&amp;volume=25&amp;publication_year=2023&amp;pages=137-152&amp;pmid=36755467&amp;doi=10.1111/ede.12428&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0075">
<cite>

Ryan, K.
, Greenway R., Landers J., Arias‐Rodriguez L., Tobler M., and Kelley J. L.. 2023. “Selection on Standing Genetic Variation Mediates Convergent Evolution in Extremophile Fish.” Molecular Ecology
32: 5042–5054.
</cite> [<a href="https://doi.org/10.1111/mec.17081" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37548336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Selection%20on%20Standing%20Genetic%20Variation%20Mediates%20Convergent%20Evolution%20in%20Extremophile%20Fish&amp;author=K.%20Ryan&amp;author=R.%20Greenway&amp;author=J.%20Landers&amp;author=L.%20Arias%E2%80%90Rodriguez&amp;author=M.%20Tobler&amp;volume=32&amp;publication_year=2023&amp;pages=5042-5054&amp;pmid=37548336&amp;doi=10.1111/mec.17081&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0076">
<cite>

Sarbu, S. M.
, Brad T., Băncilă R. I., and Ştefan A.. 2024. “Exploring Biodiversity and Food Webs in Sulfur Cave in the Vromoner Canyon on the Greek–Albanian Border.” Diversity
16: 477.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Diversity&amp;title=Exploring%20Biodiversity%20and%20Food%20Webs%20in%20Sulfur%20Cave%20in%20the%20Vromoner%20Canyon%20on%20the%20Greek%E2%80%93Albanian%20Border&amp;author=S.%20M.%20Sarbu&amp;author=T.%20Brad&amp;author=R.%20I.%20B%C4%83ncil%C4%83&amp;author=A.%20%C5%9Etefan&amp;volume=16&amp;publication_year=2024&amp;pages=477&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0077">
<cite>

Sarbu, S. M.
, Lascu C., and Brad T.. 2019. “Dobrogea: Movile Cave.” In Cave and Karst Systems of Romania, 429–436. Springer.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Cave%20and%20Karst%20Systems%20of%20Romania&amp;author=S.%20M.%20Sarbu&amp;author=C.%20Lascu&amp;author=T.%20Brad&amp;publication_year=2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0078">
<cite>

Schulz‐Mirbach, T.
, Eifert C., Riesch R., et al. 2016. “Toxic Hydrogen Sulphide Shapes Brain Anatomy: A Comparative Study of Sulphide‐Adapted Ecotypes in the <em>Poecilia mexicana</em> Complex.” Journal of Zoology
300: 163–176.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Zoology&amp;title=Toxic%20Hydrogen%20Sulphide%20Shapes%20Brain%20Anatomy:%20A%20Comparative%20Study%20of%20Sulphide%E2%80%90Adapted%20Ecotypes%20in%20the%20Poecilia%20mexicana%20Complex&amp;author=T.%20Schulz%E2%80%90Mirbach&amp;author=C.%20Eifert&amp;author=R.%20Riesch&amp;volume=300&amp;publication_year=2016&amp;pages=163-176&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0079">
<cite>

Sears, C. R.
, Boggs T. E., and Gross J. B.. 2020. “Dark‐Rearing Uncovers Novel Gene Expression Patterns in an Obligate Cave‐Dwelling Fish.” Journal of Experimental Zoology Part B: Molecular and Developmental Evolution
334: 518–529.
</cite> [<a href="https://doi.org/10.1002/jez.b.22947" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7641996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32372488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Zoology%20Part%20B:%20Molecular%20and%20Developmental%20Evolution&amp;title=Dark%E2%80%90Rearing%20Uncovers%20Novel%20Gene%20Expression%20Patterns%20in%20an%20Obligate%20Cave%E2%80%90Dwelling%20Fish&amp;author=C.%20R.%20Sears&amp;author=T.%20E.%20Boggs&amp;author=J.%20B.%20Gross&amp;volume=334&amp;publication_year=2020&amp;pages=518-529&amp;pmid=32372488&amp;doi=10.1002/jez.b.22947&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0080">
<cite>

Simão, F. A.
, Waterhouse R. M., Ioannidis P., Kriventseva E. V., and Zdobnov E. M.. 2015. “BUSCO: Assessing Genome Assembly and Annotation Completeness With Single‐Copy Orthologs.” Bioinformatics
31: 3210–3212.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv351" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26059717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=BUSCO:%20Assessing%20Genome%20Assembly%20and%20Annotation%20Completeness%20With%20Single%E2%80%90Copy%20Orthologs&amp;author=F.%20A.%20Sim%C3%A3o&amp;author=R.%20M.%20Waterhouse&amp;author=P.%20Ioannidis&amp;author=E.%20V.%20Kriventseva&amp;author=E.%20M.%20Zdobnov&amp;volume=31&amp;publication_year=2015&amp;pages=3210-3212&amp;pmid=26059717&amp;doi=10.1093/bioinformatics/btv351&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0081">
<cite>

Stahl, B. A.
, Gross J. B., Speiser D. I., et al. 2015. “A Transcriptomic Analysis of Cave, Surface, and Hybrid Isopod Crustaceans of the Species <em>Asellus aquaticus</em>
.” PLoS One
10: e0140484.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0140484" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4604090/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26462237/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=A%20Transcriptomic%20Analysis%20of%20Cave,%20Surface,%20and%20Hybrid%20Isopod%20Crustaceans%20of%20the%20Species%20Asellus%20aquaticus&amp;author=B.%20A.%20Stahl&amp;author=J.%20B.%20Gross&amp;author=D.%20I.%20Speiser&amp;volume=10&amp;publication_year=2015&amp;pages=e0140484&amp;pmid=26462237&amp;doi=10.1371/journal.pone.0140484&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0082">
<cite>

Thomas, J.
, University of Oxford and Wytham Woods Genome Acquisition Lab, Natural History Museum Genome Acquisition Lab, Darwin Tree of Life Barcoding collective, Wellcome Sanger Institute Tree of Life Management, Samples and Laboratory team, Wellcome Sanger Institute Scientific Operations: Sequencing Operations, Wellcome Sanger Institute Tree of Life Core Informatics team, Tree of Life Core Informatics collective and Darwin Tree of Life Consortium
. 2025. “The Genome Sequence of the Pond Louse, <em>Asellus aquaticus</em> (Linnaeus, 1758).” Wellcome Open Research
10: 291.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Wellcome%20Open%20Research&amp;title=The%20Genome%20Sequence%20of%20the%20Pond%20Louse,%20Asellus%20aquaticus%20(Linnaeus,%201758)&amp;author=J.%20Thomas&amp;volume=10&amp;publication_year=2025&amp;pages=291&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0083">
<cite>

Tobler, M.
, DeWitt T. J., Schlupp I., et al. 2008. “Toxic Hydrogen Sulfide and Dark Caves: Phenotypic and Genetic Divergence Across Two Abiotic Environmental Gradients in <em>Poecilia mexicana</em>
.” Evolution
62: 2643–2659.
</cite> [<a href="https://doi.org/10.1111/j.1558-5646.2008.00466.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18637957/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evolution&amp;title=Toxic%20Hydrogen%20Sulfide%20and%20Dark%20Caves:%20Phenotypic%20and%20Genetic%20Divergence%20Across%20Two%20Abiotic%20Environmental%20Gradients%20in%20Poecilia%20mexicana&amp;author=M.%20Tobler&amp;author=T.%20J.%20DeWitt&amp;author=I.%20Schlupp&amp;volume=62&amp;publication_year=2008&amp;pages=2643-2659&amp;pmid=18637957&amp;doi=10.1111/j.1558-5646.2008.00466.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0084">
<cite>

Tobler, M.
, Kelley J. L., Plath M., and Riesch R.. 2018. “Extreme Environments and the Origins of Biodiversity: Adaptation and Speciation in Sulphide Spring Fishes.” Molecular Ecology
27: 843–859.
</cite> [<a href="https://doi.org/10.1111/mec.14497" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29368386/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Extreme%20Environments%20and%20the%20Origins%20of%20Biodiversity:%20Adaptation%20and%20Speciation%20in%20Sulphide%20Spring%20Fishes&amp;author=M.%20Tobler&amp;author=J.%20L.%20Kelley&amp;author=M.%20Plath&amp;author=R.%20Riesch&amp;volume=27&amp;publication_year=2018&amp;pages=843-859&amp;pmid=29368386&amp;doi=10.1111/mec.14497&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0085">
<cite>

Tobler, M.
, Passow C. N., Greenway R., Kelley J. L., and Shaw J. H.. 2016. “The Evolutionary Ecology of Animals Inhabiting Hydrogen Sulfide‐Rich Environments.” Annual Review of Ecology, Evolution, and Systematics
47: 239–262.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Ecology,%20Evolution,%20and%20Systematics&amp;title=The%20Evolutionary%20Ecology%20of%20Animals%20Inhabiting%20Hydrogen%20Sulfide%E2%80%90Rich%20Environments&amp;author=M.%20Tobler&amp;author=C.%20N.%20Passow&amp;author=R.%20Greenway&amp;author=J.%20L.%20Kelley&amp;author=J.%20H.%20Shaw&amp;volume=47&amp;publication_year=2016&amp;pages=239-262&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0086">
<cite>

Towns, J.
, Cockerill T., Dahan M., et al. 2014. “XSEDE: Accelerating Scientific Discovery.” Computing in Science &amp; Engineering
16: 62–74.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Computing%20in%20Science%20&amp;%20Engineering&amp;title=XSEDE:%20Accelerating%20Scientific%20Discovery&amp;author=J.%20Towns&amp;author=T.%20Cockerill&amp;author=M.%20Dahan&amp;volume=16&amp;publication_year=2014&amp;pages=62-74&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0087">
<cite>

Turk‐Prevorčnik, S.
, Blejec A., and Turk‐Prevorcnik S.. 1998. “
<em>Asellus aquaticus</em> Infernus, New Subspecies (Isopoda: Asellota: Asellidae), From Romanian Hypogean Waters.” Journal of Crustacean Biology
18: 763–773.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Crustacean%20Biology&amp;title=Asellus%20aquaticus%20Infernus,%20New%20Subspecies%20(Isopoda:%20Asellota:%20Asellidae),%20From%20Romanian%20Hypogean%20Waters&amp;author=S.%20Turk%E2%80%90Prevor%C4%8Dnik&amp;author=A.%20Blejec&amp;author=S.%20Turk%E2%80%90Prevorcnik&amp;volume=18&amp;publication_year=1998&amp;pages=763-773&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0088">
<cite>

Verovnik, R.
, Sket B., and Trontelj P.. 2004. “Phylogeography of Subterranean and Surface Populations of Water Lice <em>Asellus aquaticus</em> (Crustacea: Isopoda).” Molecular Ecology
13: 1519–1532.
</cite> [<a href="https://doi.org/10.1111/j.1365-294X.2004.02171.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15140095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Phylogeography%20of%20Subterranean%20and%20Surface%20Populations%20of%20Water%20Lice%20Asellus%20aquaticus%20(Crustacea:%20Isopoda)&amp;author=R.%20Verovnik&amp;author=B.%20Sket&amp;author=P.%20Trontelj&amp;volume=13&amp;publication_year=2004&amp;pages=1519-1532&amp;pmid=15140095&amp;doi=10.1111/j.1365-294X.2004.02171.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0089">
<cite>

Wilkens, H.

1988. “Evolution and Genetics of Epigean and Cave <em>Astyanax fasciatus</em> (Characidae, Pisces) Support for the Neutral Mutation Theory.” In Evolutionary Biology, 271–367. Springer.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20Biology&amp;author=H.%20Wilkens&amp;publication_year=1988&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0090">
<cite>

Xie, Y.
, Wu G., Tang J., et al. 2014. “SOAPdenovo‐Trans: De Novo Transcriptome Assembly With Short RNA‐Seq Reads.” Bioinformatics
30: 1660–1666.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu077" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24532719/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=SOAPdenovo%E2%80%90Trans:%20De%20Novo%20Transcriptome%20Assembly%20With%20Short%20RNA%E2%80%90Seq%20Reads&amp;author=Y.%20Xie&amp;author=G.%20Wu&amp;author=J.%20Tang&amp;volume=30&amp;publication_year=2014&amp;pages=1660-1666&amp;pmid=24532719&amp;doi=10.1093/bioinformatics/btu077&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0091">
<cite>

Yancey, P. H.
, Ishikawa J., Meyer B., Girguis P. R., and Lee R. W.. 2009. “Thiotaurine and Hypotaurine Contents in Hydrothermal‐Vent Polychaetes Without Thiotrophic Endosymbionts: Correlation With Sulfide Exposure.” Journal of Experimental Zoology Part A: Ecological Genetics and Physiology
311: 439–447.
</cite> [<a href="https://doi.org/10.1002/jez.541" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19402134/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Zoology%20Part%20A:%20Ecological%20Genetics%20and%20Physiology&amp;title=Thiotaurine%20and%20Hypotaurine%20Contents%20in%20Hydrothermal%E2%80%90Vent%20Polychaetes%20Without%20Thiotrophic%20Endosymbionts:%20Correlation%20With%20Sulfide%20Exposure&amp;author=P.%20H.%20Yancey&amp;author=J.%20Ishikawa&amp;author=B.%20Meyer&amp;author=P.%20R.%20Girguis&amp;author=R.%20W.%20Lee&amp;volume=311&amp;publication_year=2009&amp;pages=439-447&amp;pmid=19402134&amp;doi=10.1002/jez.541&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ede70014-bib-0092">
<cite>

Zerbino, D. R.
, and Birney E.. 2008. “Velvet: Algorithms for De Novo Short Read Assembly Using De Bruijn Graphs.” Genome Research
18: 821–829.
</cite> [<a href="https://doi.org/10.1101/gr.074492.107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2336801/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18349386/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=Velvet:%20Algorithms%20for%20De%20Novo%20Short%20Read%20Assembly%20Using%20De%20Bruijn%20Graphs&amp;author=D.%20R.%20Zerbino&amp;author=E.%20Birney&amp;volume=18&amp;publication_year=2008&amp;pages=821-829&amp;pmid=18349386&amp;doi=10.1101/gr.074492.107&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
<strong>SupFiles: Infernus_6_7_25. Supplementary Figure 1:</strong> Principal Count Analysis (PCA) of count data of RNA‐seq data from Asellus infernus (cave) and A. aquaticus (surface) embryos A PCA was performed on the RNA‐seq count data from Asellus infernus (cave) and A. aquaticus (surface) embryos sampled at ~70% (mid‐stage) and ~90% (late‐stage) of embryonic development. <strong>Supplementary Figure 2:</strong> Embryonic development of cave samples of A. infernus. <strong>Supplementary Figure 3:</strong> Embryonic development of surface samples. <strong>Supplementary Figure 4:</strong> Gene expression matrix of genes mapped to the cave transcriptome. <strong>Supplementary Figure 5:</strong> Gene expression matrix of genes mapped to the surface transcriptome. <strong>Supplementary Table 1:</strong> Populations collected of A. aquaticus and A. infernus. <strong>Supplementary Table 2:</strong> Samples used for hatchling analysis. <strong>Supplementary Table 3:</strong> Samples used for Illumina sequencing. <strong>Supplementary Table 4:</strong> P‐values for statistical tests comparing cave, surface, and F1 samples. <strong>Supplementary Table 5:</strong> Factor loadings and fraction of variance explained (FVE) for first two principal components.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12326928/bin/EDE-27-e70014-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">EDE-27-e70014-s001.docx</a><sup> (10.3MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Supporting Information</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12326928/bin/EDE-27-e70014-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">EDE-27-e70014-s002.xlsx</a><sup> (121.4KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support the findings of this study are openly available in PRJNA957305 at <a href="https://www.ncbi.nlm.nih.gov" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov</a>, reference number PRJNA957305. All sequences discussed in this report are present in the National Center for Biotechnology Information, Sequencing Reads Archive (BioProject ID: PRJNA957305).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Evolution &amp; Development are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1111/ede.70014"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/EDE-27-e70014.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12326928/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12326928/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12326928%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326928/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12326928/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12326928/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40746274/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12326928/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40746274/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12326928/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12326928/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="AaXYkLQqhJ404w1IwfRCRHwpYYZhaDWcDdVxG5S1CRyRy1VKqHHlqaOpyeyHNpKz">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
