
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Exoproteome Profiling Reveals Increased Secretion of Adhesins and Proteases by to Facilitate Host Colonization and Immune Modulation - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="4DDE484989B738131B4849002B813326.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsomega">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332550/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Omega">
<meta name="citation_title" content="Exoproteome Profiling Reveals Increased Secretion of Adhesins and Proteases by to Facilitate Host Colonization and Immune Modulation">
<meta name="citation_author" content="Abhijit Sarma">
<meta name="citation_author_institution" content="ICMR - Regional Medical Research Centre, Port Blair 744103, Andaman and Nicobar Islands, India">
<meta name="citation_author" content="Homen Phukan">
<meta name="citation_author_institution" content="ICMR - Regional Medical Research Centre, Port Blair 744103, Andaman and Nicobar Islands, India">
<meta name="citation_author" content="Rex Devasahayam Arokia Balaya">
<meta name="citation_author_institution" content="Center for Systems Biology and Molecular Medicine, Yenepoya Research Centre, Yenepoya (Deemed to be University), Mangalore 575018, India">
<meta name="citation_author" content="Sikha Thoduvayil">
<meta name="citation_author_institution" content="ICMR - Regional Medical Research Centre, Port Blair 744103, Andaman and Nicobar Islands, India">
<meta name="citation_author" content="Gunasekaran Dhandapani">
<meta name="citation_author_institution" content="ICMR - Regional Medical Research Centre, Port Blair 744103, Andaman and Nicobar Islands, India">
<meta name="citation_author" content="Swamidurai Arul Diana Christie">
<meta name="citation_author_institution" content="Molecular Biophysics Unit, Indian Institute of Science, Bangalore 560012, Karnataka, India">
<meta name="citation_author" content="Thottethodi Subrahmanya Keshava Prasad">
<meta name="citation_author_institution" content="Center for Systems Biology and Molecular Medicine, Yenepoya Research Centre, Yenepoya (Deemed to be University), Mangalore 575018, India">
<meta name="citation_author" content="Madathiparambil Gopalakrishnan Madanan">
<meta name="citation_author_institution" content="ICMR - Regional Medical Research Centre, Port Blair 744103, Andaman and Nicobar Islands, India">
<meta name="citation_publication_date" content="2025 Jul 27">
<meta name="citation_volume" content="10">
<meta name="citation_issue" content="30">
<meta name="citation_firstpage" content="32728">
<meta name="citation_doi" content="10.1021/acsomega.4c10983">
<meta name="citation_pmid" content="40787326">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332550/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332550/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332550/pdf/ao4c10983.pdf">
<meta name="description" content="Leptospirosis, a re-emerging zoonotic disease, is challenging human and animal health due to the lack of early and rapid diagnostic tools and effective vaccines. The exoproteome of the pathogen expressed under pathogenic conditions possesses a ...">
<meta name="og:title" content="Exoproteome Profiling Reveals Increased Secretion of Adhesins and Proteases by to Facilitate Host Colonization and Immune Modulation">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Leptospirosis, a re-emerging zoonotic disease, is challenging human and animal health due to the lack of early and rapid diagnostic tools and effective vaccines. The exoproteome of the pathogen expressed under pathogenic conditions possesses a ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332550/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12332550">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsomega.4c10983"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/ao4c10983.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332550%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12332550/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12332550/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332550/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsomega.png" alt="ACS Omega logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Omega" title="Link to ACS Omega" shape="default" href="https://pubs.acs.org/journal/acsodf" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Omega</button></div>. 2025 Jul 27;10(30):32728–32743. doi: <a href="https://doi.org/10.1021/acsomega.4c10983" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsomega.4c10983</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Omega%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Omega%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Exoproteome Profiling
Reveals Increased Secretion
of Adhesins and Proteases by to Facilitate Host Colonization and Immune Modulation</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sarma%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Abhijit Sarma</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Abhijit Sarma</span></h3>
<div class="p">
<sup>†</sup>
ICMR
- Regional Medical Research Centre, Port Blair
744103, Andaman and Nicobar Islands, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sarma%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Abhijit Sarma</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Phukan%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Homen Phukan</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Homen Phukan</span></h3>
<div class="p">
<sup>†</sup>
ICMR
- Regional Medical Research Centre, Port Blair
744103, Andaman and Nicobar Islands, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Phukan%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Homen Phukan</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Devasahayam%20Arokia%20Balaya%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Rex Devasahayam Arokia Balaya</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Rex Devasahayam Arokia Balaya</span></h3>
<div class="p">
<sup>‡</sup>
Center
for Systems Biology and Molecular Medicine, Yenepoya Research Centre, Yenepoya (Deemed to be University), Mangalore
575018, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Devasahayam%20Arokia%20Balaya%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rex Devasahayam Arokia Balaya</span></a>
</div>
</div>
<sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thoduvayil%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Sikha Thoduvayil</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Sikha Thoduvayil</span></h3>
<div class="p">
<sup>†</sup>
ICMR
- Regional Medical Research Centre, Port Blair
744103, Andaman and Nicobar Islands, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thoduvayil%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sikha Thoduvayil</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dhandapani%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Gunasekaran Dhandapani</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Gunasekaran Dhandapani</span></h3>
<div class="p">
<sup>†</sup>
ICMR
- Regional Medical Research Centre, Port Blair
744103, Andaman and Nicobar Islands, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dhandapani%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gunasekaran Dhandapani</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Christie%20SAD%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Swamidurai Arul Diana Christie</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Swamidurai Arul Diana Christie</span></h3>
<div class="p">
<sup>§</sup>
Molecular
Biophysics Unit, Indian Institute of Science, Bangalore
560012, Karnataka, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Christie%20SAD%22%5BAuthor%5D" class="usa-link"><span class="name western">Swamidurai Arul Diana Christie</span></a>
</div>
</div>
<sup>§</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prasad%20TSK%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Thottethodi Subrahmanya Keshava Prasad</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Thottethodi Subrahmanya Keshava Prasad</span></h3>
<div class="p">
<sup>‡</sup>
Center
for Systems Biology and Molecular Medicine, Yenepoya Research Centre, Yenepoya (Deemed to be University), Mangalore
575018, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prasad%20TSK%22%5BAuthor%5D" class="usa-link"><span class="name western">Thottethodi Subrahmanya Keshava Prasad</span></a>
</div>
</div>
<sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Madanan%20MG%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Madathiparambil Gopalakrishnan Madanan</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Madathiparambil Gopalakrishnan Madanan</span></h3>
<div class="p">
<sup>†</sup>
ICMR
- Regional Medical Research Centre, Port Blair
744103, Andaman and Nicobar Islands, India
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Madanan%20MG%22%5BAuthor%5D" class="usa-link"><span class="name western">Madathiparambil Gopalakrishnan Madanan</span></a>
</div>
</div>
<sup>†,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>†</sup>
ICMR
- Regional Medical Research Centre, Port Blair
744103, Andaman and Nicobar Islands, India
</div>
<div id="aff2">
<sup>‡</sup>
Center
for Systems Biology and Molecular Medicine, Yenepoya Research Centre, Yenepoya (Deemed to be University), Mangalore
575018, India
</div>
<div id="aff3">
<sup>§</sup>
Molecular
Biophysics Unit, Indian Institute of Science, Bangalore
560012, Karnataka, India
</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
Email: <span>madananmg@gmail.com</span>.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Dec 7; Accepted 2025 Mar 6; Revised 2025 Mar 1; Collection date 2025 Aug 5.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY-NC-ND 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12332550  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40787326/" class="usa-link">40787326</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Leptospirosis, a re-emerging zoonotic
disease, is challenging
human
and animal health due to the lack of early and rapid diagnostic tools
and effective vaccines. The exoproteome of the pathogen expressed under pathogenic
conditions possesses a rational diagnostic significance due to its
consistent presence in body fluids. were challenged to conditions simulating infection using physiological
temperatures and osmolarity. Using state-of-the-art extraction techniques,
efficient enrichment of nonabundant proteins, and high-resolution
LC-MS/MS, we identified 1575 exoproteins from both the pathogen surface
and culture supernatant. The results indicate a significant upregulation
of 155 exoproteins, of which 41 were predicted to have moonlighting
properties, 35 were identified as adhesins, and several proteins were
components of the type 2 secretion system (T2SS). Additionally, 10
proteins showed extracellular matrix (ECM) binding properties, out
of which 4 orthologs were found using the T2SS. The overall characteristics
of upregulated proteins show that they can help <em>Leptospira</em> establish infection, invasion, and protection from the host defense,
thereby providing new insights into the pathogen to confront the host
via an increased energy level, secretion system, and host ECM binding
molecules. Furthermore, the study suggests potential candidates for
efficient antileptospiral countermeasures.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332550_ao4c10983_0010.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2efc/12332550/8b1507f1693c/ao4c10983_0010.jpg" loading="lazy" id="abs1" height="279" width="674" alt="graphic file with name ao4c10983_0010.jpg"></a></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332550_ao4c10983_0008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2efc/12332550/8b1507f1693c/ao4c10983_0008.jpg" loading="lazy" id="tgr1" height="279" width="674" alt="graphic file with name ao4c10983_0008.jpg"></a></p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Leptospirosis
is a neglected tropical
disease of global importance
that is caused by the helically coiled motile spirochete . About 1.3 million people
are infected with leptospirosis annually, with a death toll of about
60,000. Leptospires enter the host through cuts, abrasions, or mucosal
routes and then spread to various organs via circulation. The disease
causes flu-like symptoms and, in its most severe form, can lead to
multiorgan failure. Current serological diagnostic methods include
microscopic agglutination tests (MAT) and enzyme-linked immunosorbent
assays. However, these tests can detect
infection only after significant progress. After the bacteria invade
vital organs and cause severe damage, antibiotic treatment becomes
less effective, making early diagnosis the primary challenge to detect
leptospirosis. In the case of prevention, many attempts of vaccination
using proteins, lipopolysaccharide (LPS), and heat-killed <em>Leptospira</em> were attempted, but these are inefficient or
provide protection for a small subset of serovars. The identification of new target molecules provides greater
sensitivity and specificity for diagnosis at the early stages of infection,
and the active molecules, whose function is critical for pathogen
survival, can be used to develop vaccines.
</p>
<p>
<em>Leptospira</em> can thrive in diverse environments,
enabling efficient transmission of the disease. The adaptive capacity
of <em>Leptospira</em> to sense its surroundings and produce
differential proteins and cellular components indicates its remarkable
versatility in surviving in the environment. It is well known that
many pathogenic bacteria respond to changes in temperature and pH
after invading a host by altering the production of virulence factors
and other proteins. Some bacteria, such
as and , require the environmental conditions
of a mammalian infection to produce proteins associated with virulence
and pathogenesis.<em>
,
</em> Another spirochete, , appears to be able to detect
changes in temperature during infection in vertebrates. However, it is unclear how the leptospiral proteome
changes when it migrates from its free-living natural environment
to a human host. A rise from roughly 25 °C in ambient temperatures
(in wet soil and water) to a human physiological temperature (PT)
of 37 °C during the febrile phase of infection is necessary for
subsequent hematogenous spread to target organs/tissues of the mammalian
host. Similarly, <em>Leptospira</em> is exposed to physiological change from its natural habitat in water
with an osmolarity of 60–66 mOsm to the physiological osmotic
environment of mammals (300 mOsm). This
transition of physiological conditions from a free-living to an invasive
phase necessitates significant proteome modifications, which play
a crucial role in the pathogen’s survival within the host.
Although many studies on the differential expression of proteins were
conferred to the outer membrane, limited studies were conducted on
exoproteome identification. The surface-exposed proteins constitute
the first line of contact between bacteria and host being leading
actors in this complex interplay. These
proteins, either exposed on the surface or dissolved in the extracellular
milieu, are crucial in pathogenesis and host–pathogen interactions,
making them valuable for developing effective antileptospiral targets.
The exoproteins may contain many molecules useful to combat leptospirosis. Exoproteomes in <em>Leptospira</em> are
crucial in a wide range of biological processes, such as host–pathogen
interactions, immunological evasion, and virulence.<em>
,
</em> The bacteria release these proteins into the surroundings, which
could impact the host’s immunological response and contribute
to pathogenicity. Secretory enzymes, such as lipase, hyaluronidase,
elastase, etc., are secreted by many infectious organisms along with
toxins and proteases. The culture supernatant
of has been demonstrated
to have several protease activities that can destroy the extracellular
matrix (ECM), contributing to infection and pathogenesis. Studies have shown that a 60 kDa hemolysin secretion
can induce proinflammatory cytokine production through the Toll-like
receptor 2- and 4-mediated JNK and NF-kB signaling pathways.<em>
,
</em> Researchers have utilized specific antisera to identify hemolysin
and sphingomyelinase activity in the culture fluids of various pathogenic <em>Leptospira</em> strains.<em>
,
</em> In addition, patients
with acute and chronic leptospirosis have exhibited diverse immunoreactive
proteins in their serum. Like , which employs immune-evasion
molecules like proteases to evade the host immune system and damage
the extracellular matrix, <em>Leptospira</em> may utilize
secretory proteins to enhance its survival within the host. These
secretory proteins could activate host zymogens that target host-specific
defense mechanisms, thereby suppressing the antibacterial actions
of the host. Previous investigations
have identified exoproteomes of in culture supernatants, further supporting the notion that exoproteomes
can play many crucial roles in establishing leptospiral infection. To date, an extracellular proteomic analysis
conducted by Zeng et al. has revealed 66 proteins in the culture supernatant
of serovar Lai. Subsequently,
Eshghi et al. identified a remarkable 325 exoproteins, marking the
highest number of proteins reported in such studies.<em>
,
</em> With the biological significance in pathogenesis and host–pathogen
interactions, secretory proteins hold promising potential for diagnosing
leptospiral infections. Concerning this
biomedical significance, the identification and characterization of
exoproteomes are of vital importance.</p>
<p>Difficulties in isolating
and purifying secretory proteins from
biological materials and protein-enriched culture supernatants have
hindered investigations on their use in antileptospirosis treatments.
Our previous study identified 1176 exoproteins and their posttranslational
modifications, showing that 57% of the extracellularly isolated pathogenic
proteins are involved in nutrient acquisition, cell–cell communication,
and environmental detoxification. The
exoproteome expressed under pathogenic conditions could give more
insight into the pathogen invasion mechanism and its strategies for
host defense. So, in this study, we proposed challenging leptospires
with physiological conditions similar to those of the mammalian host
using significant temperature changes (from about 25 to 37 °C)
and osmolarity (from 66 to 300 mOsm), which would mimic infection.
A high-resolution LC-MS/MS quantitative proteomics was performed on to analyze the exoproteome and gain
insight into their biological role in finding candidates for antileptospiral
measures.</p></section><section id="sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">Analysis of Exoproteome Using Tandem Mass
Tags (TMT)-Labeled
LC-MS/MS</h3>
<p>The proteome data of cellular as Triton X-114 fractions
(PXD035782; referred from Phukan et al.), wash and culture supernatant under all treatment conditions are
submitted to PRIDE available under the identification number PXD037702
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Tables S1–S10</a>). Each
type of protein of the entire Triton X-114 fractions was added together
as the cellular abundance of that protein; the protein identified
in the wash was considered as surface and the protein obtained from
the culture supernatant was called secretory. If the quantity of each
protein in the wash is &gt;1.5 times the total of the cellular and
supernatant
is sorted as surface proteins and the quantity of supernatant is &gt;1.5
times the total of cellular and wash are sorted as supernatant. Further,
if the abundance of the combined wash and supernatant is &gt;1.5 times
that of the cellular, it is considered an exoprotein. The list of
proteins obtained in each category is listed in <a href="#tbl1" class="usa-link">Table 
</a>. The LC-MS/MS analysis identified a total
of 1420 proteins at control, i.e., standard culture condition (SCC),
1401 in physiological temperature (PT), 1401 in physiological osmolarity
(PO), and 1401 in both physiological temperature and osmolarity (PTO)
treatments in the wash and 473 proteins in the supernatant (culture
condition and PT and PTO) and 472 proteins in the PO. The differential
expression of proteins in wash (listed in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Table S11</a>), supernatant (listed in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Table S12</a>), and exoproteins (listed in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Table S13</a>) was assessed between their respective
control group. The flowchart for the extraction of exoproteins (wash
and supernatant) is depicted in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Figure S1</a>.</p>
<section class="tw xbox font-sm" id="tbl1"><h4 class="obj_head">1. Number of Proteins in Wash, Supernatant,
and Exoprotein Fractions That Show &gt;1.5 Abundance against Their
Total
Triton X-114 Fractions.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
<col align="char" char="." span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">treatment fraction</th>
<th align="center" colspan="1" rowspan="1">standard culture condition (SCC)</th>
<th align="center" colspan="1" rowspan="1">37 °C (PT)</th>
<th align="center" colspan="1" rowspan="1">300 mOsm (PO)</th>
<th align="center" colspan="1" rowspan="1">37 °C + 300 mOsm (PTO)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">wash/cellular = &gt;1.5</td>
<td align="center" colspan="1" rowspan="1">1420</td>
<td align="center" colspan="1" rowspan="1">1401</td>
<td align="center" colspan="1" rowspan="1">1401</td>
<td align="center" colspan="1" rowspan="1">1401</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">supernatant/cellular = &gt;1.5</td>
<td align="center" colspan="1" rowspan="1">473</td>
<td align="center" colspan="1" rowspan="1">473</td>
<td align="center" colspan="1" rowspan="1">472</td>
<td align="center" colspan="1" rowspan="1">473</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">exoproteome = &gt;1.5</td>
<td align="center" colspan="1" rowspan="1">1575</td>
<td align="center" colspan="1" rowspan="1">1575</td>
<td align="center" colspan="1" rowspan="1">1575</td>
<td align="center" colspan="1" rowspan="1">1575</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec2.2"><h3 class="pmc_sec_title">Differential Expression of Exoproteins under
Conditions of Infection</h3>
<p>The statistical analysis shows proteins
above fold 1.2 as upregulated
and values below fold 0.83 as downregulated (<a href="#fig1" class="usa-link">Figure 
</a>A–O). Among individual fractions,
the wash containing surface proteins showed 199 (14.02%) upregulation,
while 17 (1.21%) downregulation (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Table S11</a>). In the supernatant, 15 proteins (3.17%) were found to
be upregulated and 8 (1.69%) were downregulated (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Table S12</a>). In addition, at the exoproteome level
(wash + supernatant), 155 (9.84%) were upregulated and 109 (6.92%)
were downregulated, named as differentially expressed (DE) exoproteins
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Table S13</a>). The analysis revealed
a higher percentage of upregulation in the wash, followed by the supernatant.
Additionally, 43 proteins in the wash fraction and 5 in the supernatant
fraction were exclusively found, and 4 were identified in both fractions,
which were not found in the cellular fraction (<a href="#tbl2" class="usa-link">Table 
</a>).</p>
<figure class="fig xbox font-sm" id="fig1"><h4 class="obj_head">1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332550_ao4c10983_0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2efc/12332550/fd6aab9faa03/ao4c10983_0001.jpg" loading="lazy" id="gr1" height="419" width="696" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>DE proteins of <em>Leptospira</em> under
pathogenic conditions:
The physiological changes under pathogenic conditions such as physiological
temperature (PT), physiological osmolarity (PO), and physiological
temperature and osmolarity (PTO) were induced in <em>Leptospira</em> culture, and the proteins were identified quantitatively using TMT-LC-MS/MS
approach. The scatter plot shows the relative quantification and analysis
of proteins expressed under physiological conditions (PT, PO, and
PTO) within the surface, supernatant, and exoproteome and the Venn
diagram shows the number of proteins common in various pathogenic
conditions. (A–E) accounts for the volcano plot analysis and
distribution of differentially expressed (DE) proteins in the wash
(surface) fractions; (F–J) accounts for the volcano plot analysis
and distribution of differential expression of proteins in the supernatant
(secretome) fractions; (K–O) accounts for the volcano plot
analysis and distribution of differential expression of proteins in
the exoproteome (wash + supernatant) fractions. The <em>P</em>-value obtained was transformed into −log2 and a volcano plot
was generated in GraphPad prism by plotting the log fold change column
on the <em>x</em>-axis and the −log2 of the <em>P</em>-value on the <em>y</em>-axis where −log2 <em>P</em>-value &gt;4.321 signifies that <em>P</em>-value
&lt;0.05.
Red dots represent significantly upregulated proteins, and green dots
represent significantly downregulated proteins.</p></figcaption></figure><section class="tw xbox font-sm" id="tbl2"><h4 class="obj_head">2. Exclusive Proteins Identified in the
Wash and Supernatant Fractions But Not Reported in Cellular Fractions.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">sl. no.</th>
<th align="center" colspan="1" rowspan="1">ref seq
no.</th>
<th align="center" colspan="1" rowspan="1">protein name</th>
<th align="center" colspan="1" rowspan="1">exclusively present in</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_001061755.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_001061755.1</a></td>
<td align="left" colspan="1" rowspan="1">MULTISPECIES:
endoflagellar motor switch protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000015950.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000015950.1</a></td>
<td align="left" colspan="1" rowspan="1">DUF1292 domain-containing protein</td>
<td align="left" colspan="1" rowspan="1">supernatant,
wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000601827.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000601827.1</a></td>
<td align="left" colspan="1" rowspan="1">MULTISPECIES: endoflagellar protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000642158.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000642158.1</a></td>
<td align="left" colspan="1" rowspan="1">dTDP-4-dehydrorhamnose reductase</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000418124.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000418124.1</a></td>
<td align="left" colspan="1" rowspan="1">MULTISPECIES:
XRE family transcriptional regulator</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_002145802.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_002145802.1</a></td>
<td align="left" colspan="1" rowspan="1">transposase</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000875567.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000875567.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_001969378.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_001969378.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_001010386.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_001010386.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_001216953.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_001216953.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_080011796.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_080011796.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_001051394.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_001051394.1</a></td>
<td align="left" colspan="1" rowspan="1">SGNH hydrolase</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">13</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_001973793.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_001973793.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">14</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_001247796.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_001247796.1</a></td>
<td align="left" colspan="1" rowspan="1">(2Fe-2S)-binding protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_001183065.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_001183065.1</a></td>
<td align="left" colspan="1" rowspan="1">VOC family protein</td>
<td align="left" colspan="1" rowspan="1">supernatant,
wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_001072161.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_001072161.1</a></td>
<td align="left" colspan="1" rowspan="1">Fic family protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">17</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000243193.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000243193.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">18</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000671159.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000671159.1</a></td>
<td align="left" colspan="1" rowspan="1">tRNA(guanosine(37)-<em>N</em>1)-methyltransferase TrmD</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">19</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000844347.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000844347.1</a></td>
<td align="left" colspan="1" rowspan="1">SGNH/GDSL hydrolase family protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000252841.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000252841.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">21</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_001239008.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_001239008.1</a></td>
<td align="left" colspan="1" rowspan="1">
<em>N</em>-acetyltransferase</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">22</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_002144868.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_002144868.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_002080383.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_002080383.1</a></td>
<td align="left" colspan="1" rowspan="1">histidine kinase</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">24</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_080011831.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_080011831.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">25</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000971361.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000971361.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">26</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000403598.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000403598.1</a></td>
<td align="left" colspan="1" rowspan="1">Crp/Fnr family transcriptional
regulator</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">27</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000659127.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000659127.1</a></td>
<td align="left" colspan="1" rowspan="1">copper resistance protein CopZ</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">28</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000536657.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000536657.1</a></td>
<td align="left" colspan="1" rowspan="1">MarR family transcriptional regulator</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">29</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_001188390.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_001188390.1</a></td>
<td align="left" colspan="1" rowspan="1">α/β hydrolase</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">30</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000715958.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000715958.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical
protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">31</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000041838.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000041838.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">32</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000257955.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000257955.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical
protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">33</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000187969.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000187969.1</a></td>
<td align="left" colspan="1" rowspan="1">sulfur transferase</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">34</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_001106967.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_001106967.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical
protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">35</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000579997.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000579997.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">36</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_001084748.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_001084748.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical
protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">37</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000899963.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000899963.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">38</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_080011799.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_080011799.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical
protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">39</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_001142430.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_001142430.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">40</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_002069163.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_002069163.1</a></td>
<td align="left" colspan="1" rowspan="1">toxin-antitoxin system toxin
subunit</td>
<td align="left" colspan="1" rowspan="1">supernatant, wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">41</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_080010717.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_080010717.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">42</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000372884.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000372884.1</a></td>
<td align="left" colspan="1" rowspan="1">MULTISPECIES: hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">43</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_080010610.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_080010610.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical
protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">44</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_001258003.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_001258003.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">45</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000716433.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000716433.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical
protein</td>
<td align="left" colspan="1" rowspan="1">supernatant, wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">46</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_001974956.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_001974956.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">47</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000397121.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000397121.1</a></td>
<td align="left" colspan="1" rowspan="1">MULTISPECIES: hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">48</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000908409.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000908409.1</a></td>
<td align="left" colspan="1" rowspan="1">methylamine
utilization protein</td>
<td align="left" colspan="1" rowspan="1">wash</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">49</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_002002492.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_002002492.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">supernatant</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">50</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_002072017.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_002072017.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">supernatant</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">51</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000229001.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000229001.1</a></td>
<td align="left" colspan="1" rowspan="1">low-molecular-weight phosphotyrosine
protein phosphatase</td>
<td align="left" colspan="1" rowspan="1">supernatant</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">52</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000694669.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000694669.1</a></td>
<td align="left" colspan="1" rowspan="1">hypothetical protein</td>
<td align="left" colspan="1" rowspan="1">supernatant</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">53</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/WP_000218838.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000218838.1</a></td>
<td align="left" colspan="1" rowspan="1">MULTISPECIES: ABC transporter ATP-binding protein</td>
<td align="left" colspan="1" rowspan="1">supernatant</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec2.3"><h3 class="pmc_sec_title">Pathogenicity Prediction among the Upregulated
Exoproteins</h3>
<p>The analysis using MP3-predicted 363 pathogenic
proteins and 1039
nonpathogenic proteins in the wash fraction. Similarly, 136 pathogenic
proteins and 337 nonpathogenic proteins were predicted in the supernatant
fraction. Among the 1575 exoproteomes, 450 were pathogenic, while
the remaining 1175 were nonpathogenic (<a href="#fig2" class="usa-link">Figure 
</a>A). exoproteins, 34 were pathogenic and
121 were nonpathogenic. Fraction wise, out of 54 upregulated proteins
in the wash, were pathogenic and 142 were nonpathogenic across all
treatments. Similarly, 2 upregulated proteins were pathogenic, and
13 were nonpathogenic in all treatments of supernatant (<a href="#fig2" class="usa-link">Figure 
</a>B).</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332550_ao4c10983_0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2efc/12332550/3ce596b64182/ao4c10983_0002.jpg" loading="lazy" id="gr2" height="556" width="700" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Bar chart depicting the
distribution of pathogenic proteins within
the leptospiral exoproteome: (A) MP3-predicted pathogenic proteins
identified in the wash, supernatant, and combined exoproteome; (B)
differential expression of proteins in correlation with pathogenicity
and various treatments across the wash, supernatant, and combined
exoproteins. The MP3 analysis (pathogenicity) data of these proteins
are provided in the <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Tables S11–S13</a>.</p></figcaption></figure></section><section id="sec2.4"><h3 class="pmc_sec_title">Exoproteins Involved in
the Protein Export System May Provide
Insight into Transport Strategies</h3>
<p>In silico insights into
the assignment of exoproteins were employed using the Phobius program
to predict N-terminal signal peptides and the SecretomeP (version
2.0) server to identify nonclassical protein export. Of the 1402 wash
proteins identified, 228 contained an N-terminal signal peptide, indicating
their involvement in classical protein export. Additionally, 181 proteins
were predicted to be exported through nonclassical pathways, defined
as pathways for exporting proteins lacking a classical signal peptide.
Among the 473 proteins identified in the supernatant, 31 were predicted
to have an N-terminal signal peptide associated with classical protein
export. In addition, nonclassical export was predicted to 86 proteins.
These findings highlight the variety of export channels used by distinct
groups of proteins and provide significant insight into the processes
of protein export. In total, 8.25% of the exoproteins carried an N-terminal
signal peptide, while approximately 11.87% were found to be exported
via nonclassical pathways, and the majority did not carry any signal
peptide nor secreted by nonclassical pathway.</p></section><section id="sec2.5"><h3 class="pmc_sec_title">Identification of DE Exoproteins
under In Vitro Conditions Provides
Insight into Host–Pathogen Interactions</h3>
<p>Out of 155
upregulated proteins identified in the exoproteome, 101 were found
to be upregulated under PT, 122 were upregulated under PO, and 110
were upregulated under PTO. Among them, 65 were found to be common
to all of the treatment conditions, where 20 proteins were found to
be pathogenic and 49 were found to be nonpathogenic. The upregulation
of exoproteins under physiological conditions suggests their importance
in infection (<a href="#fig3" class="usa-link">Figure 
</a>A). Within the downregulated exoproteome, 56 of these proteins were
identified as downregulated in all treatments, while 87 were identified
in each PT, 80 in PO, and 84 in PTO, representing 12 as pathogenic
and 73 as nonpathogenic (<a href="#fig3" class="usa-link">Figure 
</a>B) and listed in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Table S13</a>.</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332550_ao4c10983_0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2efc/12332550/eab269f49c6e/ao4c10983_0003.jpg" loading="lazy" id="gr3" height="759" width="700" alt="3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Heatmap depicting the exoproteomes with fold change and pathogenicity.
In this analysis, (A) out of the 155 exoproteomes, 65 proteins were
consistently upregulated under all infection conditions (PT, PO, PTO),
and the fold change of the remaining 90 proteins. Additionally, (B)
represents the 56 exoproteomes downregulated under all of the infection
conditions (PT, PO, PTO). The remaining fold change of 90 upregulated
proteins and 53 downregulated proteins is outlined in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Table S13</a>.</p></figcaption></figure></section><section id="sec2.6"><h3 class="pmc_sec_title">Adhesins as a Key Contributor among the Upregulated Exoproteome
Unveiling Molecular Mechanisms of Pathogenicity</h3>
<p>Among the
155 upregulated exoproteomes, 41 moonlighting proteins were identified
that can perform multiple biological functions with a single polypeptide
chain. Out of 41 moonlighting proteins, 31 were nonpathogenic and
10 were pathogenic, as listed in the <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Table S14</a>. Among these, 32.72% target plasminogen and fibrinogen,
while 25.45% were directed toward laminin and fibronectin, 30.6% were
adhesins, and 13.23% were involved in other cell binding activities.
Moreover, the upregulation of 10 known surface adhesins, along with
4 prominent metalloproteases, were found in the study (<a href="#fig4" class="usa-link">Figure 
</a>).</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332550_ao4c10983_0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2efc/12332550/d58e5e0c3401/ao4c10983_0004.jpg" loading="lazy" id="gr4" height="307" width="627" alt="4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Identification of the
adhesins among the DE exoproteins. (A) Functional
characterization of upregulated exoproteins involves the identification
of adhesins and proteases within the exoproteome. The pie chart (B)
inset shows various moonlighting activities among the 41 moonlight
proteins. Details of the function of proteins are listed in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Table S14</a>.</p></figcaption></figure></section><section id="sec2.7"><h3 class="pmc_sec_title">Conservation Analysis of Adhesins Orthologs among Species Types
of <em>Leptospira</em>
</h3>
<p>Protein BLAST analysis was
performed to identify orthologues of the 10 major adhesin proteins
among the pathogenic P1 and P2 types and the saprophytic S1 and S2
types within the <em>Leptospira</em> species. A query coverage
of 98–100% was considered for analyzing the distribution of
these proteins among the species. The results, depicted in the heatmap
(<a href="#fig5" class="usa-link">Figure 
</a>), reveal
that most proteins exhibit the highest conservation in the P1 and
P2 pathogenic species. LipL32 (<a href="https://www.ncbi.nlm.nih.gov/protein/WP_000736494.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000736494.1</a>) showed conservation
in P1, followed by that in P2. LipL45 (<a href="https://www.ncbi.nlm.nih.gov/protein/WP_000714033.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000714033.1</a>), zinc metalloprotease
(<a href="https://www.ncbi.nlm.nih.gov/protein/WP_000906630.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000906630.1</a>) exhibited conservation among P1, P2, S1, and S2.
The M4 family metalloprotease (<a href="https://www.ncbi.nlm.nih.gov/protein/WP_000875705.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000875705.1</a>) was conserved among
P1, while the M43 family zinc metalloprotease (<a href="https://www.ncbi.nlm.nih.gov/protein/WP_000914951.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000914951.1</a>) was
conserved among P1, P2, and S1 ( and ). The S41 family
peptidase (<a href="https://www.ncbi.nlm.nih.gov/protein/WP_000790723.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000790723.1</a>) was conserved among P1, P2, and S1 ( and ), and LIC_12730 showed conservation among P1, P2, and S2. OmpL47
(<a href="https://www.ncbi.nlm.nih.gov/protein/WP_000545443.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000545443.1</a>) and Lsa25.6 (<a href="https://www.ncbi.nlm.nih.gov/protein/WP_000672495.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000672495.1</a>) were conserved only
among the P1 type, and imelysin (<a href="https://www.ncbi.nlm.nih.gov/protein/WP_001049563.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_001049563.1</a>) demonstrated conservation
in P1, P2, and S1.</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332550_ao4c10983_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2efc/12332550/7001d2cfa3b6/ao4c10983_0005.jpg" loading="lazy" id="gr5" height="308" width="700" alt="5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Ortholog analysis of the adhesins identified among DE
exoproteins.
The heatmap illustrates 10 crucial adhesins across <em>Leptospira</em> pathogenic (P1 and P2) and saprophytic (S1 and S2) species. The
percentage identity was considered when constructing the heatmap on
proteins with top 500 blast hits with a high query coverage (98–100%).
The scale of 40–100 represents the percentage identity of proteins
among different <em>Leptospira</em> serovars, as determined
from BLAST results.</p></figcaption></figure></section><section id="sec2.8"><h3 class="pmc_sec_title">Immunoblot Analysis for
Validation of DE Exoproteins</h3>
<p>Immunoblot analysis validated
the DE exoproteins within wash fractions
among the SCC and treatment physiological temperature (PT), physiological
osmolarity (PO), and both physiological temperature and osmolarity
(PTO). The upregulation or downregulation of the proteins in PT, PO,
and PTO was evaluated based on the standard. Lipl45, Lipl32, and zinc
metalloprotease were used as representatives, and the immunoblot depicted
in <a href="#fig6" class="usa-link">Figure 
</a>A–C
shows the correlation with the MS data. The evaluation of conservation
revealed that the three selected proteins, lipoprotein LipL32, LipL45,
and zinc metalloprotease, were present in all types of <em>Leptospira</em> species, i.e., P1 pathogenic and P2 saprophytic groups. However,
in the case of LipL32, no conservation was observed in , , and , represented by lanes
7, 9, and 11, which belong to the saprophytic groups. Furthermore,
no proteins were present in , as shown in lane 12, depicted in <a href="#fig6" class="usa-link">Figure 
</a>D–F and in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Figures S5–S7</a>.</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332550_ao4c10983_0006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2efc/12332550/7269c8b8a1e8/ao4c10983_0006.jpg" loading="lazy" id="gr6" height="742" width="696" alt="6"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Surface-associated exoproteins were represented
in immunoblots.
(A) Immunoblot of LipL32, (B) immunoblot of LipL45, and (C) immunoblot
of Zinc metalloprotease. Accordingly, ImageJ analysis was also conducted
(where blue indicates the total protein, while pink indicates the
surface proteins extracted under physiological conditions). The intensity
graph was presented in a bar chart with (<em>P</em> &lt; 0.001**)
and (<em>P</em> &lt; 0.01*). Furthermore, (D–F) shows
the conservation analysis of the protein LipL32, LipL45, and zinc
metalloprotease among various species types of <em>Leptospira</em>, represented in Lane (1–12) where 1. Copenhageni Fiocruz L1–130 (P1), 2. Lai (P1), 3. Cynopteri
3522 C (P1), 4. Louisiana
LSU 1945 (P1), 5. Hebdomadis
Bovino 131 (P1), 6. Ballum Castellon 3 (P1), 7. Celledoni (P2), 8. Hurstbridge
BUT 6 (S1), 9. Semaranga
Patoc I (S1), 10 Codice
CDC (S1), 11. Javanica Sofia
874 (S1), and 12. sequentially.</p></figcaption></figure></section><section id="sec2.9"><h3 class="pmc_sec_title">Exploring Pathways in Associated with DE Exoproteins</h3>
<p>The Kyoto Encyclopedia of
Genes and Genomes (KEGG) analysis revealed a range of biological pathways
displaying differences in expression levels. These pathways encompassed
metabolism, genetic information processing, environmental information
processing, cellular processes, biochemical hierarchies, and human
diseases. The involvement of 155 upregulated exoproteins and 109 downregulated
exoproteins in various pathways, with 130 proteins as part of metabolism,
1 involved in genetic information processing, 14 associated with environmental
information processing, 12 proteins involved in cellular processes,
and 1 in human diseases, respectively, is listed in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Tables S16 and S17</a>. Considering the abundance of
peptides and the upregulated proteins, we identified 8 highly significant
pathways: glycolysis, tricarboxylic acid (TCA) cycle, protein export,
chemotaxis, ribosomal function, motility, quorum sensing, and two-component
systems listed in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Tables S15–S18</a>.</p>
<p>The gene ontology enrichment analysis of the DE exoproteins
revealed three main categories: biological processes, molecular functions,
and cellular components. The upregulated proteins strongly enriched
gene ontology terms associated with metabolic processes and cellular
components such as macromolecular complexes, cells, cytoplasm, ribosomes,
ribosomal subunits, and proton-transporting ATP synthase. On the other
hand, the biological process category showed upregulation in organic
cyclic compound binding, structural molecule activity, and heterocyclic
compound binding. In contrast, the downregulated proteins were primarily
involved in catalytic binding and ion binding in the molecular function
category and the de novo inosine 5′-monophosphate biosynthetic
process and cellular nitrogen compound metabolic process in the cellular
component category.</p></section></section><section id="sec3"><h2 class="pmc_sec_title">Discussion</h2>
<p>Exoproteomes containing
surface-associated
proteins and those from
the culture supernatant are significant due to their interactions
with the host environment.<em>
,
</em> Studying exoproteome
changes and differential expression under pathogenic conditions is
vital to understanding the host’s biology and mechanisms of
pathogenesis and the need to develop diagnostics, drugs, and vaccines
rationally against the disease. In this study, we use an efficient
exoproteome extraction method coupled with the elimination of cellular
proteins from consideration. The stepwise centrifugation reduced the
lysis of the cells, collection of the growth medium, and enrichment
of the nonabundant proteins from the protein-enriched EMJH media using
the ProteoMiner kit. Similarly, washing with phosphate-buffered saline
(PBS) containing 5 mM MgCl<sub>2</sub> without any harsh conditions
or solubilizers extracted the easily detachable surface-associated
proteins bound to the surface of <em>Leptospira</em>. The
proteins are then identified using TMT-labeled LC-MS/MS analysis,
identifying the highest number of exoproteomes (1575) to date. The
unique proteins identified in high abundance suggest an extraction
efficiency with negligible cross-contamination. Exoproteome was sorted
out using a rational method supported by robust statistical analysis.
This is characterized by the enrichment of the proteins found in the
exoproteome, which should exceed those present in the whole cell,
eliminating the possibility of proteins being released from the lysis.
Physiological conditions of infection, such as temperature, osmolarity,
and a combination of both, are applied to obtain the maximum number
of DE exoproteins. Compared to our previous study, which identified
1176 proteins in wash and supernatant, our present study reported
1402 proteins in the wash and 473 as supernatant proteins, with upregulation
of 10.77 and 1.88% in the respective wash and supernatant fractions. Across all of the physiological treatments,
the wash fractions exhibited more DE proteins than the supernatant
fractions. Further, the percentage of upregulated proteins in each
fraction was generally higher than that of downregulated proteins.
We observed a significant overall upregulation of 155 (9.84%) exoproteins
under all physiological conditions, and vice versa for downregulated
proteins.</p>
<p>The exoproteome extraction method was very efficient
in enriching
and identifying the proteins. Compared to the cellular presence, the
abundance in the exoproteome identified 1102 (69.96%) proteins unique
to the wash fractions, indicating association with the surface. Additionally,
155 (9.84%) of the proteins were found in the supernatant, suggesting
their classification as secretory proteins. About 318 (20.08%) of
the proteins were found in both fractions. Furthermore, our analysis
revealed that 3.42% of the exoproteins, consisting of 43, 5, and 4
proteins, were identified as wash, supernatant, and both, respectively,
which is not reported in the cellular fraction. The present study
identified nearly four times higher exoproteins than the 325 exoproteins
reported by Eshghi et al. These significant
differences could be attributed to the distinct identification methods
employed for surface-associated and supernatant proteins. Additionally,
our study identified 1400 proteins as abundant surface-associated
proteins, which is 20-fold higher than the number reported by Techawiwattanaboon
et al. Notably, 153 were upregulated
among these proteins under the physiological conditions of infection,
indicating that those surface proteins may be essential for leptospiral
pathogenesis.</p>
<p>Pathogenesis involves a series of interrelated
events, including
adhesion, colonization, invasion, drug resistance, and evasion of
host defenses. These are crucial for leptospiral invasion and dissemination
within the host<em>
,
</em> Our study found that a relatively
small percentage of pathogenic proteins were upregulated, most of
which were associated with the surface (i.e., 20), and only 12 pathogenic
proteins were found abundant in the medium. The limited number of
pathogenic proteins may be due to the limitation of MP3 prediction
software because many of the exoproteins were found to be immunogenic
in experimental conditions and not predicted as pathogenic by the
software. For instance, LipL45 and GroEL are antigens produced during
leptospiral infection that induce cytokine generation, although predicted
to be nonpathogenic by MP3.<em>
,
</em> In this context, we
have performed additional studies to determine the functions of these
proteins and found many to exhibit “moonlighting” properties.
Our study identified 37 moonlighting proteins (<a href="#tbl2" class="usa-link">Table 
</a>), which aligns with the 32 moonlighting
proteins reported by Eshghi et al., indicating
the potential presence of additional pathogenic proteins that contribute
to infection, invasion, and pathogenesis.</p>
<p>Among the 155 upregulated
exoproteins in our current study, 69
were consistently identified across all treatment groups (<a href="#fig4" class="usa-link">Figure 
</a>A). These proteins
showed a significant increase in expression compared to that of the
control group, with a high confidence level. Moreover, many of these
proteins have been extensively studied and are known to play critical
roles in the survival, export, and pathogenesis of <em>Leptospira</em>. Importantly, these proteins demonstrate remarkable conservation
among the pathogenic P1 strains, followed by P2 and the saprophytic
S1 and S2 species. This conservation suggests their essentiality in
survival and may contribute to leptospiral pathogenicity.</p>
<p>The
in silico analysis of 1575 exoproteins revealed that <em>Leptospira</em> primarily exports proteins using a dedicated
export mechanism. Among the 1402 and 473 proteins identified in the
wash and supernatant, 228 (16%) and 77 (16.2%) contained an N-terminal
signal sequence, suggesting classical protein export. However, 181
(12.7%) and 86 (18.18%) proteins were predicted to be exported through
a nonclassical protein export system. In the wash fractions, 32 (2.25%)
contained N-terminal signal peptide among the upregulated proteins
and 24 (1.64%) suggested nonclassical protein export without the peptide.
In the supernatant, there were 2 proteins with 4 N-terminal signal
peptides and 2 proteins with nonclassical protein export systems among
the upregulated proteins. Though the export mechanism of the remaining
993 proteins in the wash and 310 in the supernatant remains unknown,
it is reported as a part of exoproteome analysis in our study. The
exoproteome nature of these proteins is supported by the identification
of the abundance of experimentally proven exoproteins like LipL32
and GroES on the surface<em>
,
</em> and GroEL in the supernatant
fractions.
</p>
<p>Our study observed a
4-fold rise in LipL32 (LIC_11352) in the wash
fractions under simulating infection conditions that clearly shows
the surface localization of LipL32 and its upregulation under pathogenic
conditions, corroborating with earlier reports. The 4-fold rise under pathogenic conditions and earlier
reports on its interaction with TLR2 on human kidney cells, stimulation
of inflammatory response, and binding on the ECM shows this molecule will be active and available for targeting
diagnostic or vaccine during infection. Further, our conservations
analysis shows the presence of LipL32 in P1 and P2 (pathogenic) also
shows its potential in the identification of pathogenic strains, which
is supported by the use of LipL32 genes for the identification of <em>Leptospira</em> in clinical samples depicted in <a href="#fig6" class="usa-link">Figure 
</a>A and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Figure S2</a>.</p>
<p>We report
another well-known protein, LipL45 (LIC_11643), found
to be upregulated with a 1.6-fold raise in PT, a 1.8-fold raise in
PO, and a 2-fold raise in PTO in the wash fractions, which confirms
surface association along with its presence in P1, P2 (pathogenic)
and S1, S2 (saprophytic) groups of <em>Leptospira</em>. The
surface presence is accompanied by the role of a vital immunogen involved
in degranulation and heparin-binding stimulating inflammatory response, suggesting its potential to use as a vaccine
candidate for which the possibility is explored by Ugen et al., as
a DNA vaccine depicted in <a href="#fig6" class="usa-link">Figure 
</a>B and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Figure S3</a>.</p>
<p>LipL71, a major surface molecule, exhibited
a significant upregulation
of 0.3-fold, 1.9-fold, and 1.5-fold expression under PT, PO, and PTO,
respectively, that corroborated with a previous study, highlighting LipL71 as a potent inducer of antibody
responses in patients and a promising
vaccine candidate.</p>
<p>GroEL (LIC_11335), an extracellular protein
interacting with host
proteins, observed a 3.5-fold upregulation, corroborating an earlier
study that demonstrated upregulation at 300 mOsm in the culture supernatant. Further, the GroEL is a moonlighting protein,
resembling the multifunctional nature as predicted in . Another
study also shows the usefulness of GroEL in IgM ELISA assay for screening
leptospirosis cases, suggesting their
distribution among the <em>Leptospira</em> species.</p>
<p>Proteases found in the wash and exoproteome are another set of
potential zinc-dependent molecules that can degrade the extracellular
matrix, affect the host immune system, and promote the invagination
of leptospires. The M4 family metalloprotease
(LIC_10715) exhibited 0.4-fold upregulation exclusively in PT, while
the zinc metalloprotease (LIC_10857) displayed notable upregulation
with 1.3-fold in PT, 1.8-fold in PO, and 1.5-fold in PTO on the surface
depicted in <a href="#fig6" class="usa-link">Figure 
</a>B and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Figure S4</a>. Another M43
family zinc metalloprotease (LIC_13434) showed upregulation of 1.68
in PT, 1.15 in PO, and 2.4 in PTO in the exoproteome. These findings
are consistent with observations by Fraga et al., who reported similar
proteins, such as Zmp1 in and zinc-dependent metalloprotease in , both demonstrating pathogenic activities in both in vitro and in
vivo contexts.
</p>
<p>The imelysin family
(LIC_10713) is known to cleave insulin and
fibrinogen and has been found to upregulate
to 1.5-fold in PT, 2.3-fold in PO, and 1.7-fold in PTO in the supernatant.
This protein in the medium may be uniformly available in serum for
use as a diagnostic candidate. Recent studies show it can bind to
many ECM components and its potential as a diagnostic marker.
</p>
<p>We utilized the KEGG pathway analysis
to identify potential signaling
pathways among the DE proteins. Our analysis shows the upregulation
of carbohydrate metabolism molecules (glycolytic pathways and the
tricarboxylic acid (TCA) cycle, such as LIC_12090, LIC_12233, LIC_12475,
LIC_13105, LIC_12002, LIC_12573, LIC_12574, and LIC_12829). Proteins
(LIC_11241, LIC_11243, and LIC_12002) related to oxidative phosphorylation
are upregulated in the exoproteome. Protein (LIC_10583) fatty acid
metabolism showed downregulation in wash fractions listed in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Tables S15–S18</a>. The study also
displayed significant modulation of proteins associated with signal
transduction, chemotaxis (LIC_12454), flagellar assembly (LIC_11531,
LIC_11889, and LIC_11890), and quorum sensing (LIC_12540 and LIC_13384),
which may help the bacteria to survive. Interestingly, under physiological
infection conditions, we observed significant upregulation of orthologs
of the Sec-translocation system (SecA, SecD, YajC) in the exoproteome,
indicating that <em>Leptospira</em> may be actively secreting
proteins through the Sec translocase system during infection. The
Type I secretory pathway, which relies on ABC transporters ATP-binding
proteins (LIC_12684, LIC_13384, and LIC_10233) is upregulated in the
exoproteome, determining its importance in transporting various exoproteins,
including toxins, antimicrobial peptides, siderophores, and hydrolytic
enzymes. Similarly, GspG and GspC, components of the type II secretion
system, show upregulation in the exoproteome, and GspD in the wash
fraction. This finding aligns with a previous study where GspD was
identified as a secreted protein. Activating
these secretion systems primarily aids <em>Leptospira</em> in interacting with the host’s fibrinolytic system, a significant
factor in invasion mechanisms. <em>Leptospira</em> can bind
plasminogen on its surface and convert it to plasmin, enhancing its
aggressive spread. While associated with
plasmin, <em>Leptospira</em> can degrade ECM proteins and
interfere with the deposition of complement components C3b and IgG
on its surface. This is evident through
the existence and increased expression of high levels of adhesins
(36.77%) serving a dual role, i.e., moonlighting proteins. Among these,
36.7 and 2% target plasminogen and fibrinogen, while 12 and 8.2% are
directed toward laminin and fibronectin. Moreover, we observed the
upregulation of (<a href="https://www.ncbi.nlm.nih.gov/protein/WP_000678902.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000678902.1</a>) OmpL47, (<a href="https://www.ncbi.nlm.nih.gov/protein/WP_000545443.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000545443.1</a>) LIC_12730,
and (<a href="https://www.ncbi.nlm.nih.gov/protein/WP_000672495.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">WP_000672495.1</a>) Lsa25.6, in the wash (surface) fraction, which
justifies that leptospiral directness toward fibrinogen, laminin,
and fibronectins for initiating its dissemination into host.<em>
,
</em> These findings dwell on the significance of these proteins in contributing
to leptospiral invasion of host cells. Further, the upregulation of
M4 family metalloprotease (LIC_10715), M43 family zinc Metalloprotease
(LIC_13434), S41 family peptidase (LIC_12303), and zinc metalloprotease
(LIC_10857) are particularly noteworthy, as they are presumed to be
secreted effectors through the type 2 secretion system (T2SS).<em>
,
</em> A total of 36.77% of the ECM binding molecules were found to be
upregulated among the total upregulated exoproteome. The upregulation
of exoproteins capable of ECM binding and degrading proteins can help <em>Leptospira</em> degrade the ECM molecules, thereby facilitating
infection. So, with the upregulation and downregulation observed in
the exoproteome, literature survey, and considering the peptide abundance
in the corresponding fractions, we can conclude that secretion of
these proteins along with specific virulence factors through the leptospiral
secretion system may play an important role in infection, pathogenesis,
and survival of the pathogen in the host (<a href="#fig7" class="usa-link">Figure 
</a>). However, each protein has to be investigated
individually and in combination with the coexpressed proteins to find
their exact role.</p>
<figure class="fig xbox font-sm" id="fig7"><h3 class="obj_head">7.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332550_ao4c10983_0007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2efc/12332550/1b3c6d7f1288/ao4c10983_0007.jpg" loading="lazy" id="gr7" height="808" width="696" alt="7"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Pathways and network analysis of the wash, supernatant,
and exoproteome
revealed significant differential expression in various biological
processes. Upregulation was observed in pathways related to glycolysis,
tricarboxylic acid (TCA) cycle, Sec-dependent pathway, Type II secretion
pathway, chemotaxis, two-component system, quorum sensing, proteases,
and adhesins. In the visual representation, red indicates the upregulated
proteins, while black represents the peptide abundance in the individual
fractions.</p></figcaption></figure></section><section id="sec4"><h2 class="pmc_sec_title">Conclusions</h2>
<p>This study comprehensively
analyzed the
exoproteome profile of <em>Leptospira</em> under in vitro
simulated infection conditions.
A quantitative proteomic approach utilizing high-resolution LC-MS/MS
analysis identified significant exoproteins that showed differential
expression under the physiological conditions of infection. Three
proteins, namely, LipL32, LipL45, and zinc metalloprotease, showed
a notable increase in protein expression levels in the surfactome.
Additionally, LIC_10713, an imelysin family protein, was found to
be upregulated in the supernatant. Comparing pathogenicity, it becomes
apparent that nonpathogenic proteins exhibit greater diversity, closely
resembling many adhesins in their moonlighting role as significant
ECM binding proteins. The predictions from KEGG and GO analyses further
support the notion that the upregulated proteins participate in various
biological processes. Notably, the upregulation of T2SS also stands
out, strongly suggesting that the identified proteases are indeed
secreted via the T2SS pathway. Additionally, the upregulated exoproteome
under infection conditions, including GroEL, LipL45, and Peptidase
M75, represents a promising candidate for leptospiral intervention
strategies. Further research focused on individual proteins or pathways
is essential to comprehending the functions of these distinctively
expressed exoproteins fully. Nevertheless, these findings offer valuable
insights for future exoproteome-related studies using this methodology
and enhance our understanding of pathogenic mechanisms and their role
in leptospirosis transmission and pathogenesis.</p></section><section id="sec5"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="sec5.1"><h3 class="pmc_sec_title">Bacterial
Strains, Media, and Growth Conditions</h3>
<p> serogroup Icterohememorrhagiae serovar
Copenhageni strain Fiocruz L1–130 was obtained from the repository
of the ICMR-Regional Medical Research Centre, Port Blair, India, a
WHO collaborating center for diagnosis, reference, research, and training
in leptospirosis. The standard culture of <em>Leptospira</em> was maintained in EMJH (Ellinghausen McCullough Johnson Harris)
medium (Difco Laboratories, Franklin Lakes, NJ) supplemented with
1% bovine serum albumin at 29 °C until the mid log phase of growth.
</p></section><section id="sec5.2"><h3 class="pmc_sec_title">Challenging Leptospires with Physiological
Environments</h3>
<p>As described earlier, the human physiological
environment was induced
to <em>Leptospira</em>, with the following modifications. A mid log phase culture of <em>Leptospira</em> was grown in EMJH, which is used for inducing the physiological
temperature (PT) at 37 °C, inducing physiological osmolarity
(PO) at 150 mM/L NaCl and both physiological
temperature and osmolarity (PTO) for 24 h, respectively. The control
was the standard growth condition mentioned above, and each test condition
was replicated four times, each containing 25 mL of the culture.</p></section><section id="sec5.3"><h3 class="pmc_sec_title">Enrichment of the Exoproteome</h3>
<section id="sec5.3.1"><h4 class="pmc_sec_title">Extraction of Protein from
the Culture Supernatant</h4>
<p>Proteins released by leptospires
in the culture supernatant were
extracted using the protocol outlined earlier with slight modifications.<em>
,
</em> Briefly, the <em>Leptospira</em> culture was centrifuged
at 4000<em>g</em>, and the culture supernatant was collected
to isolate supernatant proteins. The culture supernatant was again
centrifuged at 6000<em>g</em> for 30 min, followed by additional
centrifugation at 12,000<em>g</em> for 30 min, to remove any
remaining <em>Leptospira</em> from the supernatant. This step
was crucial to prevent <em>Leptospira</em> from being tightly
packed and ruptured, which may release cellular proteins during pelleting.</p>
<p>The medium containing a low abundance of supernatant proteins was
enriched using the ProteoMiner protein enrichment kit (Biorad, Hercules,
CA, catalog no: 1633011) with slight modifications according to the
manufacturer’s instructions. In brief, the supernatant fractions
(i.e., media) were dialyzed against phosphate-buffered saline (PBS)
to facilitate optimal binding to the ProteoMiner slurry. The ProteoMiner
slurry, consisting of 100 μL of settled beads in the column,
was washed twice with 1 mL of PBS. The washed beads were taken from
the column, added to the dialyzed supernatant, and incubated at 4
°C overnight (&gt;8 h) with gentle shaking to facilitate optimal
binding. The ProteoMiner beads were collected by centrifugation at
2000<em>g</em> for 5 min, replaced in the column, washed with
100 μL PBS, and eluted with 2 × 20 μL buffer provided
in the kit.</p></section><section id="sec5.3.2"><h4 class="pmc_sec_title">Extraction of Surface Proteins (Surfactome)</h4>
<p>The <em>Leptospira</em> pellet from the culture was washed
thrice with
PBS containing 5 mM MgCl<sub>2</sub> with gentle mixing that allows
leaching out of surface proteins, and the washing buffer was collected
by centrifuging at 2500<em>g</em> for 5 min at 4 °C.
The collected wash was pooled and centrifuged again at 12,000<em>g</em> for 30 min to ensure the removal of any residual leptospires,
it was precipitated using chilled acetone for further solubilization
in suitable buffer and analysis.</p></section></section><section id="sec5.4"><h3 class="pmc_sec_title">Identification of Cellular
Proteins</h3>
<p>The cellular proteins
were identified from the cell pellet after fractionation using Triton
X-114. The total amount of each protein from all of the Triton X-114
fractions was considered cellular abundance. This cellular abundance
was compared with their abundance in the supernatant or surface or
combined as exoproteins. The exoproteins found &lt;1.5 times that
found as cellular are assumed to have come from the lysis of the cells
during the extraction procedures and not accounted as exoproteins.</p></section><section id="sec5.5"><h3 class="pmc_sec_title">Protein Digestion and TMT Labeling</h3>
<p>Equal amounts of
protein samples from all conditions were subjected to in-solution
digestion after reduction using 10 mM dithiothreitol (DTT) (I6125,
Sigma-Aldrich, Bangalore, India), followed by alkylation with 20 mM
iodoacetamide (D9779, Sigma-Aldrich, Bangalore, India). The proteins
were then precipitated by using ice-cold acetone. In-solution trypsin
digestion was carried out with a 1:20 enzyme-to-substrate ratio. The
resulting peptide samples were lyophilized and labeled with tandem
mass tags (TMT) according to the manufacturer’s instructions
(Thermo Scientific, Waltham, MA). The reaction was halted by adding
8 μL of 5% hydroxylamine and incubating at room temperature
(RT) for 15 min. Next, the pooled and TMT-labeled peptides were fractionated
based on basic pH reversed-phase liquid chromatography (bRPLC) using
an X Bridge BEH C18 Column (Waters Corporation, Milford, MA). The
labeled peptides were resolved using an increasing gradient of 7 mM
TEABC and 90% acetonitrile (pH 9) over a 30 min duration at a 1 mL/min
flow rate, employing an Agilent 1100 LC system. This process yielded
a total of 96 fractions, which were later combined into 12 fractions.
The peptides were concentrated by vacuum centrifugation and subjected
to a C18 StageTip (66883-U, Empore, Bellefonte, PA) cleanup before
mass spectrometry analysis.<em>
,
</em>
</p></section><section id="sec5.6"><h3 class="pmc_sec_title">LC-MS/MS Analysis</h3>
<p>The fractionated samples were dissolved
in 0.1% formic acid and analyzed using a Thermo Scientific Orbitrap
Fusion Tribrid mass spectrometer (Thermo Fisher Scientific, Bremen,
Germany) connected to an Easy-nLC-1200 nanoflow liquid chromatography
system (Thermo Scientific, Bremen, Germany). The peptides were loaded
onto a 2 cm trap column (nanoViper, 3 μm C18 Aq; Thermo Fisher
Scientific) and separated on a 15 cm analytical column (nanoViper,
75 μm silica capillary, C18 Aq) at a flow rate of 300 nL/min.
A linear gradient of 5–30% solvent B (80% acetonitrile (ACN)
in 0.1% formic acid) was applied over 100 min, followed by a stepped
gradient of 60–100% solvent B for 4 and 7 min, respectively.
An MS analysis was carried out using an Orbitrap mass analyzer in
data-dependent acquisition (DDA) mode, scanning the 400–1600 <em>m</em>/<em>z</em> mass range at a resolution of 120,000
mass. The injection time was limited to 10 ms. For the MS2 analysis,
the data were acquired in top speed mode with a duty cycle of 3 s
and subjected to higher collision energy dissociation at a normalized
collision energy of 32. The MS/MS scans were performed on an Orbitrap
mass analyzer with a resolution of 30,000 at 200 <em>m</em>/<em>z</em>, and the maximum injection time was 200 ms. The
lock mass option (<em>m</em>/<em>z</em> 445.1200025)
derived from ambient air was utilized for internal calibration.</p></section><section id="sec5.7"><h3 class="pmc_sec_title">Identification of Peptides and Proteins</h3>
<p>The mass spectrometry
data (Raw) files obtained from the analysis were searched against
the serogroup Icterohemaemorrhagiae
serovar Copenhageni strain Fiocruz L1–130, acquired from the
National Center for Biotechnology Information (NCBI) database, which
consisted of 3667 protein entries. In addition, 115 common contaminants,
including bovine serum albumin, trypsin, and keratin, were included.
Sequest HT and Mascot (versions 2.5.1; Matrix Science, London, U.K.)
through Proteome Discoverer (Version 2.4) (Thermo Scientific, Bremen,
Germany) were used to analyze the MS/MS data. Both algorithms were
configured with specific search parameters, including a maximum of
one missed cleavage, carbamidomethylation at cysteine, TMT 6-plex
Lysine, and TMT 6-plex N-terminal as fixed modifications, and oxidation
of methionine as dynamic modifications. The precursor mass error tolerance
was set at 20 ppm, and the fragment mass error tolerance was set at
0.1 Da. The reliability of the result was ensured by searching the
data against a decoy database, and peptides identified at a 1% false
discovery rate (FDR) were selected for further protein identification.
Normalization of the data based on total peptide amount was performed
using the normalization feature available in the Reporter Ions Quantifier
node of Proteome Discoverer.</p></section><section id="sec5.8"><h3 class="pmc_sec_title">Gel Electrophoresis and Immunoblotting</h3>
<p>The concentration
of protein in the supernatant and wash samples was estimated using
the Bradford protein estimation protocol, following the method described
by Kruger. Briefly, protein samples were
treated with β-mercaptoethanol and then subjected to electrophoresis
in an 8% SDS-PAGE gel. The separated proteins were transferred onto
poly­(vinylidene fluoride) (PVDF) membranes (0.2 μm). Unbound
sites on the membranes were blocked using 10% nonfat dried milk dissolved
in Tris-buffered saline (TBS; pH 8.0) containing 0.1% Tween 20 (TBS-T)
for 2 h at room temperature (RT). Following the blocking step, the
membranes were probed with monospecific antisera specifically raised
against each leptospiral protein and lipoprotein (LipL45, LipL32)
and zinc metalloprotease (1:5000), respectively (GenScript manufactured
the antibodies). The probing
was performed overnight at 4 °C with gentle shaking. After incubation,
the membranes were washed thrice for 4 min each using buffer (TBST).
Subsequently, the membranes were probed for 2 h with goat antirabbit
IgG conjugated with horseradish peroxidase (HRP) (GenScript: A01856)
(1:5000) in TBST. Followed by washing with TBST for 4 min each, as
described by Taylor et al. 2013. Visualizing
and recording were done by subjecting the membranes to a chemiluminescent
substrate and exposing them to blue-sensitive X-ray film. The quantification
of bands in the immunoblots was analyzed using ImageJ software using
the rolling disc background subtraction approach.
</p></section><section id="sec5.9"><h3 class="pmc_sec_title">Conservation of Proteins among <em>Leptospira</em> Species</h3>
<p>The conservation analysis to determine the phylogenetic
relationship
was carried out using NCBI-BlastP. Additionally, to determine the
conservation of the proteins among the <em>Leptospira</em>, 11 different species (pathogenic and saprophytic) were used to
extract total proteins for immunoblot assays. The transferred proteins
were probed with LipL45 (1:5000), LipL32 (1:5000), and zinc metalloprotease
(1:3000). Secondary antibody, goat antirabbit IgG conjugated with
HRP at a dilution of 1:5000, was utilized for probing. The bands were
developed by using a chemiluminescence substrate and recorded in a
blue-sensitive X-ray film.</p></section><section id="sec5.10"><h3 class="pmc_sec_title">Bioinformatics Analysis</h3>
<section id="sec5.10.1"><h4 class="pmc_sec_title">Prediction
of Pathogenic Proteins</h4>
<p>DE exoproteins were
classified as pathogenic or nonpathogenic using MP3, an online program
incorporating a support vector machine-hidden Markov model with better
sensitivity and specificity. Assignments
were carried out based on the following criteria: If the predictions
of both HMM and SVM (Support Vector Machine) were in agreement and
the prediction by MP3 is called “HS,” it indicated high
accuracy. In the field, when the SVM and HMM modules provided different
predictions, priority was given to the HMM. This choice was justified
by the HMM demonstrating relatively higher sensitivity, specificity,
and accuracy than SVM.</p></section><section id="sec5.10.2"><h4 class="pmc_sec_title">Prediction of Secretory and Moonlighting
Proteins</h4>
<p>We
employed various prediction methods to predict protein secretion,
considering canonical and noncanonical secretion pathways. SecretomeP
(<a href="https://services.healthtech.dtu.dk/services/SecretomeP-2.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://services.healthtech.dtu.dk/services/SecretomeP-2.0/</a>) focuses explicitly on nonclassical secretion in bacteria and helps
identify proteins secreted through signal peptide-independent pathways.
Additionally, we utilized Phobius (<a href="https://phobius.sbc.su.se/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://phobius.sbc.su.se/</a>), a comprehensive predictor that considers both transmembrane topology
and signal peptides in protein analysis. Further, the Moonprot database was used to identify the heterologous
functions of the exoproteome (<a href="http://moonlightingproteins.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://moonlightingproteins.org/</a>).
</p></section></section><section id="sec5.11"><h3 class="pmc_sec_title">KEGG Analysis</h3>
<p>Gene Ontology enrichment analysis was
used to classify the proteins that showed treatment-dependent changes
in expression levels in the supernatant and on the surface. The significance
level (<em>P</em>-value) used in this study was determined
using a false discovery rate (FDR) cutoff 0.05 (<a href="http://www.geneontology.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.geneontology.org/</a>). The pathways involved with these proteins were also interpreted
with the help of the Kyoto Encyclopedia of Genes and Genomes (KEGG).
</p></section><section id="sec5.12"><h3 class="pmc_sec_title">Statistical Analysis</h3>
<p>The proteomic
data were analyzed
to identify proteins with significant fold changes using statistical
methods performed with Microsoft Excel 2018. The analysis encompassed
various steps: data filtration (removing proteins that do not have
quantitative values), logarithmic data transformation (variances became
more constant and the data fit the normal distribution), two-step
normalization: data normalization by average and normalization by
slope, imputation of missing value, subsequent statistical test (F-test
and <em>t</em> test), fold change and <em>P</em>-value
calculation. The <em>P</em>-value was then transformed into
−log2 and volcano plots were generated to visually represent
the log fold change treatment/control fold change on the <em>x</em>-axis and −log2 on the <em>y</em>-axis by GraphPad Prism version 10. A −log2
value &gt;4.312 threshold indicated a <em>P</em>-value &lt;0.05.
Proteins with log2 fold changes &gt;1.2 were upregulated, while those
with log2 fold change &lt;0.8 were considered downregulated.<em>
−

</em> However, Bonferroni was not used for low power (i.e., few replicates)
and moderate conditions. In such cases, multiassay correction methods
with conventional thresholds may be inadequate for actual positive
detection, even if they exist.</p></section></section><section id="sec31"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="sifile1"><div class="media p"><div class="caption">
<a href="/articles/instance/12332550/bin/ao4c10983_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">ao4c10983_si_001.pdf</a><sup> (1.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="sifile2"><div class="media p"><div class="caption">
<a href="/articles/instance/12332550/bin/ao4c10983_si_002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ao4c10983_si_002.xlsx</a><sup> (7MB, xlsx) </sup>
</div></div></section></section><section id="ACK-d14e1339-autogenerated" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors
express their sincere gratitude
to the Indian
Council of Medical Research, New Delhi, India, for the financial support
provided through Grant No. Leptos/22/2013-ECD-I-2012-2400 and the
Department of Science and Technology, Science and Engineering Research
Board, New Delhi, India, for their funding support via Grant No. SR/SO/HS/0108/2012
to M.G.M. Additionally, the authors extend their appreciation to the
Indian Council of Medical Research, New Delhi, India, for awarding
Senior Research Fellowship through Grant No. Fellowship/131/2022-ECD-II-2021-8230
and ICMR-SRF 2020-0756/PROTEOMICS- BMS to A.S. and H.P., respectively,
and Department of Science and Technology, Science and Engineering
Research Board, New Delhi, India, Grant No. PDF/2017/000343 to S.A.D.C.</p></section><section id="notes-1"><p>
<strong>The proteomic
data obtained by MS analysis were deposited to the ProteomeXchange
Consortium via the Proteomics identifications database (PMID), a partner
repository, under the data set identifiers PXD035782 and PXD037702.</strong>
</p></section><section id="notes-11"><p>The Supporting Information
is available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsomega.4c10983?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acsomega.4c10983</a>.</p>
<ul id="silist" class="list" style="list-style-type:none">
<li><p>Flowchart of extraction
of exoproteins from <em>Leptospira</em> (Figure S1); representation
of immunoblot using
LipL32 (Figure S2); representation of Immunoblot using LipL45 (Figure
S3); representation of immunoblot using zinc metalloprotease (64 kDa)
(Figure S4); representation of the conservation analysis of Lipl32
among the <em>Leptospira</em> strains (Figure S5); representation
of the conservation analysis of Lipl45 among the <em>Leptospira</em> strains (Figure S6); representation of the conservation analysis
of zinc metalloprotease among the <em>Leptospira</em> strains
(Figure S7); overlapping plot comparing the unique proteins identified
in the study by Techawiwattanaboon et al. with those identified in our study (Figure S8); and bubble plot
representing the differential expression of exoproteomes, analyzed
through KEGG database (Figure S9) (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p></li>
<li><p>Total proteins identified by LC-MS/MS analysis
in the
cellular and extracellular fractions (Table S1); statistical analysis
for fold change determination in the wash fraction with respect to
the physiological temperature (Table S2); statistical analysis for
fold change determination in the wash fraction with respect to the
physiological osmolarity (Table S3); statistical analysis for fold
change determination in the wash fraction with respect to the physiological
temperature and osmolarity (Table S4); statistical analysis for fold
change determination in the supernatant fraction with respect to the
physiological temperature (Table S5); statistical analysis for fold
change determination in the supernatant fraction with respect to the
physiological osmolarity (Table S6); statistical analysis for fold
change determination in the supernatant fraction with respect to the
physiological temperature and osmolarity (Table S7); statistical analysis
for fold change determination in the exoproteome fraction with respect
to the physiological temperature (Table S8); statistical analysis
for fold change determination in the exoproteome fraction with respect
to the physiological osmolarity (Table S9); statistical analysis for
fold change determination in the exoproteome fraction with respect
to the physiological temperature and Osmolarity (Table S10); fold
change analysis of the upregulated and downregulated protein reported
in wash with respect to the three in vitro conditions (Table S11);
fold change analysis of the upregulated and downregulated protein
reported in supernatant with respect to the three in vitro conditions
(Table S12); fold change analysis of the upregulated and downregulated
protein reported in exoproteome with respect to the three in vitro
conditions (Table S13); list showing the 41 moonlighting proteins
depicted in <a href="#fig4" class="usa-link">Figure 
</a> along with the moonlighting functions (Table S14); exoproteome upregulated
and downregulated proteins along with the KEGG analysis (Table S15);
gene ontology of the upregulated exoproteome (Table S16); gene ontology
of the downregulated exoproteome (Table S17); and fold change along
with KEGG analysis for wash fraction (Table S18) (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.4c10983/suppl_file/ao4c10983_si_002.xlsx" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">XLSX</a>)</p></li>
</ul></section><section id="notes-2"><h2>∇.</h2>
<p>Cellular &amp; Structural Biology Division,
ICMR-National Institute
for Research in Reproductive &amp; Child Health, (ICMR-NIRRCH), Parel,
Mumbai 400012, India</p></section><section id="notes-3"><h2>○.</h2>
<p>Department
of Chemistry &amp; Molecular Biology, Gothenburg University,
Medicinaregatan 7 B, 41390 Göteborg, Sweden.</p></section><section id="notes-4"><h2>◆.</h2>
<p>Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester,
Minnesota 55905, United States.</p></section><section id="notes-5"><h2>∥.</h2>
<p>Department
of Cell Biology and Biochemistry, Texas Tech University
Health Science Center, 3601 Fourth Street, Lubbock, Texas 79430-0002,
United States.</p></section><section id="notes-6"><h2>⊥.</h2>
<p>Vollum
Institute, Oregon Health and Science University, Portland,
Oregon 97239-3098, United States.</p></section><section id="notes-7"><h2>#.</h2>
<p>Department
of Microbiology, Ramakrishna College of Arts and Science
for Women, Coimbatore 641044, Tamil Nadu, India.</p></section><section id="notes-8"><p>M.G.M. and
A.S. conceptualized and designed the study; A.S., H.P., R.D.A.B.,
and S.A.D.C. conducted the experiments; A.S. and H.P., performed the
statistical analysis. M.G.M., T.S.K.P., S.T., and G.D. supervised
the work; A.S. and H.P. prepared the original manuscript; A.S., S.T.,
R.D.A.B., and M.G.M. edited the final manuscript. All authors read
and approved the final manuscript.</p></section><section id="notes-10"><p>The authors
declare no competing financial interest.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>Ooteman M. C., Vago A. R., Koury M. C.. Evaluation of MAT, IgM ELISA and
PCR Methods for the Diagnosis of Human Leptospirosis. J. Microbiol. Methods. 2006;65(2):247–257. doi: 10.1016/j.mimet.2005.07.015.</cite> [<a href="https://doi.org/10.1016/j.mimet.2005.07.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16253361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Microbiol.%20Methods&amp;title=Evaluation%20of%20MAT,%20IgM%20ELISA%20and%20PCR%20Methods%20for%20the%20Diagnosis%20of%20Human%20Leptospirosis&amp;author=M.%20C.%20Ooteman&amp;author=A.%20R.%20Vago&amp;author=M.%20C.%20Koury&amp;volume=65&amp;issue=2&amp;publication_year=2006&amp;pages=247-257&amp;pmid=16253361&amp;doi=10.1016/j.mimet.2005.07.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>Teixeira A. F., Fernandes L. G. V., Cavenague M. F., Takahashi M. B., Santos J. C., Passalia F. J., Daroz B. B., Kochi L. T., Vieira M. L., Nascimento A. L. T. O.. Adjuvanted
Leptospiral Vaccines:
Challenges and Future Development of New Leptospirosis Vaccines. Vaccine. 2019;37(30):3961–3973. doi: 10.1016/j.vaccine.2019.05.087.</cite> [<a href="https://doi.org/10.1016/j.vaccine.2019.05.087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31186193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine&amp;title=Adjuvanted%20Leptospiral%20Vaccines:%20Challenges%20and%20Future%20Development%20of%20New%20Leptospirosis%20Vaccines&amp;author=A.%20F.%20Teixeira&amp;author=L.%20G.%20V.%20Fernandes&amp;author=M.%20F.%20Cavenague&amp;author=M.%20B.%20Takahashi&amp;author=J.%20C.%20Santos&amp;volume=37&amp;issue=30&amp;publication_year=2019&amp;pages=3961-3973&amp;pmid=31186193&amp;doi=10.1016/j.vaccine.2019.05.087&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Kavela S., Vyas P., CP J., Kushwaha S. K., Majumdar S. S., Faisal S. M.. Use of an Integrated
Multi-Omics Approach To Identify
Molecular Mechanisms and Critical Factors Involved in the Pathogenesis
of Leptospira. Microbiol. Spectrum. 2023;11(2):e0313522.  doi: 10.1128/spectrum.03135-22.</cite> [<a href="https://doi.org/10.1128/spectrum.03135-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10100824/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36853003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Spectrum&amp;title=Use%20of%20an%20Integrated%20Multi-Omics%20Approach%20To%20Identify%20Molecular%20Mechanisms%20and%20Critical%20Factors%20Involved%20in%20the%20Pathogenesis%20of%20Leptospira&amp;author=S.%20Kavela&amp;author=P.%20Vyas&amp;author=J.%20CP&amp;author=S.%20K.%20Kushwaha&amp;author=S.%20S.%20Majumdar&amp;volume=11&amp;issue=2&amp;publication_year=2023&amp;pages=e0313522&amp;pmid=36853003&amp;doi=10.1128/spectrum.03135-22&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>Matsunaga J., Lo M., Bulach D. M., Zuerner R. L., Adler B., Haake D. A.. Response
of to Physiologic
Osmolarity : Relevance in Signaling the Environment-to-Host
Transition. Infect. Immun. 2007;75(6):2864–2874. doi: 10.1128/IAI.01619-06.</cite> [<a href="https://doi.org/10.1128/IAI.01619-06" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1932867/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17371863/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&amp;title=Response%20of%20to%20Physiologic%20Osmolarity%E2%80%AF:%20Relevance%20in%20Signaling%20the%20Environment-to-Host%20Transition&amp;author=J.%20Matsunaga&amp;author=M.%20Lo&amp;author=D.%20M.%20Bulach&amp;author=R.%20L.%20Zuerner&amp;author=B.%20Adler&amp;volume=75&amp;issue=6&amp;publication_year=2007&amp;pages=2864-2874&amp;pmid=17371863&amp;doi=10.1128/IAI.01619-06&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Takai S., Fukunaga N., Kamisawa K., Imai Y., Sasaki Y., Tsubaki S.. Expression of Virulence-Associated Antigens of Is Regulated by Temperature and
PH. Microbiol. Immunol. 1996;40(8):591–594. doi: 10.1111/j.1348-0421.1996.tb01113.x.</cite> [<a href="https://doi.org/10.1111/j.1348-0421.1996.tb01113.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8887354/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Immunol.&amp;title=Expression%20of%20Virulence-Associated%20Antigens%20of%20Is%20Regulated%20by%20Temperature%20and%20PH&amp;author=S.%20Takai&amp;author=N.%20Fukunaga&amp;author=K.%20Kamisawa&amp;author=Y.%20Imai&amp;author=Y.%20Sasaki&amp;volume=40&amp;issue=8&amp;publication_year=1996&amp;pages=591-594&amp;pmid=8887354&amp;doi=10.1111/j.1348-0421.1996.tb01113.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Lucchini S., Liu H., Jin Q., Hinton J. C. D., Yu J.. Transcriptional Adaptation
of during Infection
of Macrophages and Epithelial Cells: Insights into the Strategies
of a Cytosolic Bacterial Pathogen. Infect. Immun. 2005;73(1):88–102. doi: 10.1128/IAI.73.1.88-102.2005.</cite> [<a href="https://doi.org/10.1128/IAI.73.1.88-102.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC538992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15618144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&amp;title=Transcriptional%20Adaptation%20of%20during%20Infection%20of%20Macrophages%20and%20Epithelial%20Cells:%20Insights%20into%20the%20Strategies%20of%20a%20Cytosolic%20Bacterial%20Pathogen&amp;author=S.%20Lucchini&amp;author=H.%20Liu&amp;author=Q.%20Jin&amp;author=J.%20C.%20D.%20Hinton&amp;author=J.%20Yu&amp;volume=73&amp;issue=1&amp;publication_year=2005&amp;pages=88-102&amp;pmid=15618144&amp;doi=10.1128/IAI.73.1.88-102.2005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Tokarz R., Anderton J. M., Katona L. I., Benach J. L.. Combined Effects
of Blood and Temperature Shift on Gene Expression as Determined by Whole Genome DNA Array. Infect. Immun. 2004;72(9):5419–5432. doi: 10.1128/IAI.72.9.5419-5432.2004.</cite> [<a href="https://doi.org/10.1128/IAI.72.9.5419-5432.2004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC517457/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15322040/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&amp;title=Combined%20Effects%20of%20Blood%20and%20Temperature%20Shift%20on%20Gene%20Expression%20as%20Determined%20by%20Whole%20Genome%20DNA%20Array&amp;author=R.%20Tokarz&amp;author=J.%20M.%20Anderton&amp;author=L.%20I.%20Katona&amp;author=J.%20L.%20Benach&amp;volume=72&amp;issue=9&amp;publication_year=2004&amp;pages=5419-5432&amp;pmid=15322040&amp;doi=10.1128/IAI.72.9.5419-5432.2004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Nally J. E., Timoney J. F., Stevenson B.. Temperature-Regulated
Protein Synthesis
by . Infect. Immun. 2001;69(1):400–404. doi: 10.1128/IAI.69.1.400-404.2001.</cite> [<a href="https://doi.org/10.1128/IAI.69.1.400-404.2001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC97896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11119530/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&amp;title=Temperature-Regulated%20Protein%20Synthesis%20by&amp;author=J.%20E.%20Nally&amp;author=J.%20F.%20Timoney&amp;author=B.%20Stevenson&amp;volume=69&amp;issue=1&amp;publication_year=2001&amp;pages=400-404&amp;pmid=11119530&amp;doi=10.1128/IAI.69.1.400-404.2001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Nally J. E., Monahan A. M., Miller I. S., Bonilla-Santiago R., Souda P., Whitelegge J. P.. Comparative
Proteomic Analysis of
Differentially Expressed Proteins in the Urine of Reservoir Hosts
of Leptospirosis. PLoS One. 2011;6(10):e26046.  doi: 10.1371/journal.pone.0026046.</cite> [<a href="https://doi.org/10.1371/journal.pone.0026046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3197145/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22043303/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Comparative%20Proteomic%20Analysis%20of%20Differentially%20Expressed%20Proteins%20in%20the%20Urine%20of%20Reservoir%20Hosts%20of%20Leptospirosis&amp;author=J.%20E.%20Nally&amp;author=A.%20M.%20Monahan&amp;author=I.%20S.%20Miller&amp;author=R.%20Bonilla-Santiago&amp;author=P.%20Souda&amp;volume=6&amp;issue=10&amp;publication_year=2011&amp;pages=e26046&amp;pmid=22043303&amp;doi=10.1371/journal.pone.0026046&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Siciliano R. A., Lippolis R., Mazzeo M. F.. Proteomics
for the Investigation
of Surface-Exposed Proteins in Probiotics. Front.
Nutr. 2019;6:52.  doi: 10.3389/fnut.2019.00052.</cite> [<a href="https://doi.org/10.3389/fnut.2019.00052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6491629/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31069232/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Nutr.&amp;title=Proteomics%20for%20the%20Investigation%20of%20Surface-Exposed%20Proteins%20in%20Probiotics&amp;author=R.%20A.%20Siciliano&amp;author=R.%20Lippolis&amp;author=M.%20F.%20Mazzeo&amp;volume=6&amp;publication_year=2019&amp;pages=52&amp;pmid=31069232&amp;doi=10.3389/fnut.2019.00052&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Nogueira S. V., Backstedt B. T., Smith A. A., Qin J.-H., Wunder E. A., Ko A., Pal U..  Enolase Is Secreted Extracellularly and
Interacts with Plasminogen. PLoS One. 2013;8(10):e78150.  doi: 10.1371/journal.pone.0078150.</cite> [<a href="https://doi.org/10.1371/journal.pone.0078150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3799732/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24205133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Enolase%20Is%20Secreted%20Extracellularly%20and%20Interacts%20with%20Plasminogen&amp;author=S.%20V.%20Nogueira&amp;author=B.%20T.%20Backstedt&amp;author=A.%20A.%20Smith&amp;author=J.-H.%20Qin&amp;author=E.%20A.%20Wunder&amp;volume=8&amp;issue=10&amp;publication_year=2013&amp;pages=e78150&amp;pmid=24205133&amp;doi=10.1371/journal.pone.0078150&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Eshghi A., Pappalardo E., Hester S., Thomas B., Pretre G., Picardeau M.. Pathogenic Exoproteins Are Primarily Involved in Heterotrophic Processes. Infect. Immun. 2015;83(8):3061–3073. doi: 10.1128/IAI.00427-15.</cite> [<a href="https://doi.org/10.1128/IAI.00427-15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4496612/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25987703/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&amp;title=Pathogenic%20Exoproteins%20Are%20Primarily%20Involved%20in%20Heterotrophic%20Processes&amp;author=A.%20Eshghi&amp;author=E.%20Pappalardo&amp;author=S.%20Hester&amp;author=B.%20Thomas&amp;author=G.%20Pretre&amp;volume=83&amp;issue=8&amp;publication_year=2015&amp;pages=3061-3073&amp;pmid=25987703&amp;doi=10.1128/IAI.00427-15&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>

Armengaud, J.
; 
Duport, C.
. 
Exoproteomics
of Pathogens. In Methods in Enzymology; Elsevier Inc., 2017; Vol. 586, pp 211–227.
</cite> [<a href="https://doi.org/10.1016/bs.mie.2016.09.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28137564/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Methods%20in%20Enzymology&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Depluverez S., Devos S., Devreese B.. The Role of
Bacterial Secretion Systems
in the Virulence of Gram-Negative Airway Pathogens Associated with
Cystic Fibrosis. Front. Microbiol. 2016;7:1336. doi: 10.3389/fmicb.2016.01336.</cite> [<a href="https://doi.org/10.3389/fmicb.2016.01336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5003817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27625638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=The%20Role%20of%20Bacterial%20Secretion%20Systems%20in%20the%20Virulence%20of%20Gram-Negative%20Airway%20Pathogens%20Associated%20with%20Cystic%20Fibrosis&amp;author=S.%20Depluverez&amp;author=S.%20Devos&amp;author=B.%20Devreese&amp;volume=7&amp;publication_year=2016&amp;pages=1336&amp;pmid=27625638&amp;doi=10.3389/fmicb.2016.01336&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>da
Silva L. B., Menezes M. C., Kitano E. S., Oliveira A. K., Abreu A. G., Souza G. O., Heinemann M. B., Isaac L., Fraga T. R., Serrano S. M. T., Barbosa A. S..  Secreted Proteases Degrade
Extracellular Matrix and Plasma Proteins From the Host. Front. Cell. Infect. Microbiol. 2018;8:92.  doi: 10.3389/fcimb.2018.00092.</cite> [<a href="https://doi.org/10.3389/fcimb.2018.00092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5881292/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29637048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Cell.%20Infect.%20Microbiol.&amp;title=Secreted%20Proteases%20Degrade%20Extracellular%20Matrix%20and%20Plasma%20Proteins%20From%20the%20Host&amp;author=L.%20B.%20da%20Silva&amp;author=M.%20C.%20Menezes&amp;author=E.%20S.%20Kitano&amp;author=A.%20K.%20Oliveira&amp;author=A.%20G.%20Abreu&amp;volume=8&amp;publication_year=2018&amp;pages=92&amp;pmid=29637048&amp;doi=10.3389/fcimb.2018.00092&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Wang H., Wu Y., Ojcius D. M., Yang X. F., Zhang C., Ding S., Lin X., Yan J.. Leptospiral Hemolysins Induce Proinflammatory Cytokines
through Toll-Like Receptor 2-and 4-Mediated JNK and NF-KB Signaling
Pathways. PLoS One. 2012;7(8):e42266.  doi: 10.1371/journal.pone.0042266.</cite> [<a href="https://doi.org/10.1371/journal.pone.0042266" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3411626/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22870312/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Leptospiral%20Hemolysins%20Induce%20Proinflammatory%20Cytokines%20through%20Toll-Like%20Receptor%202-and%204-Mediated%20JNK%20and%20NF-KB%20Signaling%20Pathways&amp;author=H.%20Wang&amp;author=Y.%20Wu&amp;author=D.%20M.%20Ojcius&amp;author=X.%20F.%20Yang&amp;author=C.%20Zhang&amp;volume=7&amp;issue=8&amp;publication_year=2012&amp;pages=e42266&amp;pmid=22870312&amp;doi=10.1371/journal.pone.0042266&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Zuerner R. L., Knudtson W., Bolin C. A., Trueba G.. Characterization of
Outer Membrane and Secreted Proteins of Serovar Pomona. Microb. Pathog. 1991;10(4):311–322. doi: 10.1016/0882-4010(91)90014-2.</cite> [<a href="https://doi.org/10.1016/0882-4010(91)90014-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1895930/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microb.%20Pathog.&amp;title=Characterization%20of%20Outer%20Membrane%20and%20Secreted%20Proteins%20of%20Serovar%20Pomona&amp;author=R.%20L.%20Zuerner&amp;author=W.%20Knudtson&amp;author=C.%20A.%20Bolin&amp;author=G.%20Trueba&amp;volume=10&amp;issue=4&amp;publication_year=1991&amp;pages=311-322&amp;pmid=1895930&amp;doi=10.1016/0882-4010(91)90014-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>Carvalho E., Barbosa A. S., Gómez R. M., Oliveira M. L. S., Romero E. C., Gonçales A. P., Morais Z. M., Vasconcellos S. A., Ho P. L.. Evaluation of the
Expression and Protective Potential of Leptospiral
Sphingomyelinases. Curr. Microbiol. 2010;60(2):134–142. doi: 10.1007/s00284-009-9519-3.</cite> [<a href="https://doi.org/10.1007/s00284-009-9519-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19826861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Microbiol.&amp;title=Evaluation%20of%20the%20Expression%20and%20Protective%20Potential%20of%20Leptospiral%20Sphingomyelinases&amp;author=E.%20Carvalho&amp;author=A.%20S.%20Barbosa&amp;author=R.%20M.%20G%C3%B3mez&amp;author=M.%20L.%20S.%20Oliveira&amp;author=E.%20C.%20Romero&amp;volume=60&amp;issue=2&amp;publication_year=2010&amp;pages=134-142&amp;pmid=19826861&amp;doi=10.1007/s00284-009-9519-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Matsunaga J., Medeiros M. A., Sanchez Y., Werneid K. F., Ko A. I.. Osmotic
Regulation of Expression of Two Extracellular Matrix-Binding Proteins
and a Haemolysin of : Differential Effects on LigA and Sph2 Extracellular Release. Microbiology. 2007;153:3390. doi: 10.1099/mic.0.2007/007948-0.</cite> [<a href="https://doi.org/10.1099/mic.0.2007/007948-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17906138/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Osmotic%20Regulation%20of%20Expression%20of%20Two%20Extracellular%20Matrix-Binding%20Proteins%20and%20a%20Haemolysin%20of%20:%20Differential%20Effects%20on%20LigA%20and%20Sph2%20Extracellular%20Release&amp;author=J.%20Matsunaga&amp;author=M.%20A.%20Medeiros&amp;author=Y.%20Sanchez&amp;author=K.%20F.%20Werneid&amp;author=A.%20I.%20Ko&amp;volume=153&amp;publication_year=2007&amp;pages=3390&amp;pmid=17906138&amp;doi=10.1099/mic.0.2007/007948-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>Natarajaseenivasan K., Vijayachari P., Sugunan A. P., Sharma S., Sehgal S. C.. Leptospiral
Proteins Expressed during Acute &amp; Convalescent Phases of Human
Leptospirosis. Indian J. Med. Res. 2004;120:151–159.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/15489552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Indian%20J.%20Med.%20Res.&amp;title=Leptospiral%20Proteins%20Expressed%20during%20Acute%20&amp;%20Convalescent%20Phases%20of%20Human%20Leptospirosis&amp;author=K.%20Natarajaseenivasan&amp;author=P.%20Vijayachari&amp;author=A.%20P.%20Sugunan&amp;author=S.%20Sharma&amp;author=S.%20C.%20Sehgal&amp;volume=120&amp;publication_year=2004&amp;pages=151-159&amp;pmid=15489552&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>Pietrocola G., Nobile G., Rindi S., Speziale P..  Manipulates Innate Immunity through Own and
Host-Expressed Proteases. Front. Cell. Infect.
Microbiol. 2017;7:166. doi: 10.3389/fcimb.2017.00166.</cite> [<a href="https://doi.org/10.3389/fcimb.2017.00166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5418230/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28529927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Cell.%20Infect.%20Microbiol.&amp;title=Manipulates%20Innate%20Immunity%20through%20Own%20and%20Host-Expressed%20Proteases&amp;author=G.%20Pietrocola&amp;author=G.%20Nobile&amp;author=S.%20Rindi&amp;author=P.%20Speziale&amp;volume=7&amp;publication_year=2017&amp;pages=166&amp;pmid=28529927&amp;doi=10.3389/fcimb.2017.00166&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Sarma A., Gunasekaran D., Rex D. A. B., Sikha T., Phukan H., Kiran K. M., Pinto S. M., Prasad T. S. K., Madanan M. G.. Extracellular
Proteome Analysis Shows the Abundance of Histidine Kinase Sensor Protein,
DNA Helicase, Putative Lipoprotein Containing Peptidase M75 Domain
and Peptidase C39 Domain Protein in Grown in EMJH Medium. Pathogens. 2021;10(7):852. doi: 10.3390/pathogens10070852.</cite> [<a href="https://doi.org/10.3390/pathogens10070852" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8308593/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34358002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pathogens&amp;title=Extracellular%20Proteome%20Analysis%20Shows%20the%20Abundance%20of%20Histidine%20Kinase%20Sensor%20Protein,%20DNA%20Helicase,%20Putative%20Lipoprotein%20Containing%20Peptidase%20M75%20Domain%20and%20Peptidase%20C39%20Domain%20Protein%20in%20Grown%20in%20EMJH%20Medium&amp;author=A.%20Sarma&amp;author=D.%20Gunasekaran&amp;author=D.%20A.%20B.%20Rex&amp;author=T.%20Sikha&amp;author=H.%20Phukan&amp;volume=10&amp;issue=7&amp;publication_year=2021&amp;pages=852&amp;pmid=34358002&amp;doi=10.3390/pathogens10070852&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Zeng L., Zhang Y., Zhu Y., Yin H., Zhuang X., Zhu W., Guo X., Qin J.. Extracellular
Proteome Analysis of Serovar Lai. OMICS. 2013;17(10):527–535. doi: 10.1089/omi.2013.0043.</cite> [<a href="https://doi.org/10.1089/omi.2013.0043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3783971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23895271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=OMICS&amp;title=Extracellular%20Proteome%20Analysis%20of%20Serovar%20Lai&amp;author=L.%20Zeng&amp;author=Y.%20Zhang&amp;author=Y.%20Zhu&amp;author=H.%20Yin&amp;author=X.%20Zhuang&amp;volume=17&amp;issue=10&amp;publication_year=2013&amp;pages=527-535&amp;pmid=23895271&amp;doi=10.1089/omi.2013.0043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Rana A., Thakur S., Bhardwaj N., Kumar D., Akhter Y.. Excavating
the Surface-Associated and Secretory Proteome of for Identifying Vaccines and Diagnostic Markers
Relevant Immunodominant Epitopes. Pathog. Dis. 2016;74(9):ftw110.  doi: 10.1093/femspd/ftw110.</cite> [<a href="https://doi.org/10.1093/femspd/ftw110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27856491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pathog.%20Dis.&amp;title=Excavating%20the%20Surface-Associated%20and%20Secretory%20Proteome%20of%20for%20Identifying%20Vaccines%20and%20Diagnostic%20Markers%20Relevant%20Immunodominant%20Epitopes&amp;author=A.%20Rana&amp;author=S.%20Thakur&amp;author=N.%20Bhardwaj&amp;author=D.%20Kumar&amp;author=Y.%20Akhter&amp;volume=74&amp;issue=9&amp;publication_year=2016&amp;pages=ftw110&amp;pmid=27856491&amp;doi=10.1093/femspd/ftw110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Phukan H., Sarma A., Rex D. A. B., Rai A. B., Prasad T. S. K., Madanan M. G.. Unique Posttranslational
Modification Sites of Acetylation,
Citrullination, Glutarylation, and Phosphorylation Are Found to Be
Specific to the Proteins Partitioned in the Triton X-114 Fractions
of Leptospira. ACS Omega. 2022;7(22):18569–18576. doi: 10.1021/acsomega.2c01245.</cite> [<a href="https://doi.org/10.1021/acsomega.2c01245" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9178745/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35694507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Omega&amp;title=Unique%20Posttranslational%20Modification%20Sites%20of%20Acetylation,%20Citrullination,%20Glutarylation,%20and%20Phosphorylation%20Are%20Found%20to%20Be%20Specific%20to%20the%20Proteins%20Partitioned%20in%20the%20Triton%20X-114%20Fractions%20of%20Leptospira&amp;author=H.%20Phukan&amp;author=A.%20Sarma&amp;author=D.%20A.%20B.%20Rex&amp;author=A.%20B.%20Rai&amp;author=T.%20S.%20K.%20Prasad&amp;volume=7&amp;issue=22&amp;publication_year=2022&amp;pages=18569-18576&amp;pmid=35694507&amp;doi=10.1021/acsomega.2c01245&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Phukan H., Sarma A., Rex D. A. B., Christie S. A. D., Sabu S. K., Hariharan S., Prasad T. S. K., Madanan M. G.. Physiological Temperature
and Osmotic Changes Drive Dynamic Proteome Alterations in the Leptospiral
Outer Membrane and Enhance Protein Export Systems. J. Proteome Res. 2023;22(11):3447–3463. doi: 10.1021/acs.jproteome.3c00295.</cite> [<a href="https://doi.org/10.1021/acs.jproteome.3c00295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37877620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res.&amp;title=Physiological%20Temperature%20and%20Osmotic%20Changes%20Drive%20Dynamic%20Proteome%20Alterations%20in%20the%20Leptospiral%20Outer%20Membrane%20and%20Enhance%20Protein%20Export%20Systems&amp;author=H.%20Phukan&amp;author=A.%20Sarma&amp;author=D.%20A.%20B.%20Rex&amp;author=S.%20A.%20D.%20Christie&amp;author=S.%20K.%20Sabu&amp;volume=22&amp;issue=11&amp;publication_year=2023&amp;pages=3447-3463&amp;pmid=37877620&amp;doi=10.1021/acs.jproteome.3c00295&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Sarma A., Gunasekaran D., Phukan H., Baby A., Hariharan S.. et al. Leptospiral Imelysin (LIC_10713) Is Secretory, Immunogenic and Binds
to Laminin, Fibronectin, and Collagen IV. Appl.
Microbiol. Biotechnol. 2023;107:4275–4289. doi: 10.1007/s00253-023-12573-6.</cite> [<a href="https://doi.org/10.1007/s00253-023-12573-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37227474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&amp;title=Leptospiral%20Imelysin%20(LIC_10713)%20Is%20Secretory,%20Immunogenic%20and%20Binds%20to%20Laminin,%20Fibronectin,%20and%20Collagen%20IV&amp;author=A.%20Sarma&amp;author=D.%20Gunasekaran&amp;author=H.%20Phukan&amp;author=A.%20Baby&amp;author=S.%20Hariharan&amp;volume=107&amp;publication_year=2023&amp;pages=4275-4289&amp;pmid=37227474&amp;doi=10.1007/s00253-023-12573-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Techawiwattanaboon T., Thaibankluay P., Kreangkaiwal C., Sathean-Anan-Kun S., Khaenam P., Makjaroen J., Pisitkun T., Patarakul K.. Surface Proteomics
and Label-Free Quantification of Serovar Pomona. PLoS Neglected Trop. Dis. 2021;15(11):e0009983.  doi: 10.1371/journal.pntd.0009983.</cite> [<a href="https://doi.org/10.1371/journal.pntd.0009983" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8659334/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34843470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Neglected%20Trop.%20Dis.&amp;title=Surface%20Proteomics%20and%20Label-Free%20Quantification%20of%20Serovar%20Pomona&amp;author=T.%20Techawiwattanaboon&amp;author=P.%20Thaibankluay&amp;author=C.%20Kreangkaiwal&amp;author=S.%20Sathean-Anan-Kun&amp;author=P.%20Khaenam&amp;volume=15&amp;issue=11&amp;publication_year=2021&amp;pages=e0009983&amp;pmid=34843470&amp;doi=10.1371/journal.pntd.0009983&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>Kassegne K., Hu W., Ojcius D. M., Sun D., Ge Y., Zhao J., Yang X. F.. Identi Fi Cation of Collagenase as
a Critical Virulence
Factor for Invasiveness and Transmission of Pathogenic Leptospira
Species. J. Infect. Dis. 2014;209:1105–1115. doi: 10.1093/infdis/jit659.</cite> [<a href="https://doi.org/10.1093/infdis/jit659" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24277745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Infect.%20Dis.&amp;title=Identi%20Fi%20Cation%20of%20Collagenase%20as%20a%20Critical%20Virulence%20Factor%20for%20Invasiveness%20and%20Transmission%20of%20Pathogenic%20Leptospira%20Species&amp;author=K.%20Kassegne&amp;author=W.%20Hu&amp;author=D.%20M.%20Ojcius&amp;author=D.%20Sun&amp;author=Y.%20Ge&amp;volume=209&amp;publication_year=2014&amp;pages=1105-1115&amp;pmid=24277745&amp;doi=10.1093/infdis/jit659&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Sun H.. The Interaction
Between Pathogens and the Host Coagulation System. Physiology. 2006;21(4):281–288. doi: 10.1152/physiol.00059.2005.</cite> [<a href="https://doi.org/10.1152/physiol.00059.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16868317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Physiology&amp;title=The%20Interaction%20Between%20Pathogens%20and%20the%20Host%20Coagulation%20System&amp;author=H.%20Sun&amp;volume=21&amp;issue=4&amp;publication_year=2006&amp;pages=281-288&amp;pmid=16868317&amp;doi=10.1152/physiol.00059.2005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>Matsunaga J., Young T. A., Barnett J. K., Barnett D., Bolin C. A., Haake D. A.. Novel 45-Kilodalton Leptospiral Protein
That Is Processed
to a 31-Kilodalton Growth-Phase-Regulated Peripheral Membrane Protein. Infect. Immun. 2002;70(1):323–334. doi: 10.1128/IAI.70.1.323-334.2002.</cite> [<a href="https://doi.org/10.1128/IAI.70.1.323-334.2002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC127625/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11748198/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&amp;title=Novel%2045-Kilodalton%20Leptospiral%20Protein%20That%20Is%20Processed%20to%20a%2031-Kilodalton%20Growth-Phase-Regulated%20Peripheral%20Membrane%20Protein&amp;author=J.%20Matsunaga&amp;author=T.%20A.%20Young&amp;author=J.%20K.%20Barnett&amp;author=D.%20Barnett&amp;author=C.%20A.%20Bolin&amp;volume=70&amp;issue=1&amp;publication_year=2002&amp;pages=323-334&amp;pmid=11748198&amp;doi=10.1128/IAI.70.1.323-334.2002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>Ho J. D., Takara L. E. M., Monaris D., Gonçalves A. P., Souza-Filho A. F., de Souza G. O., Heinemann M. B., Ho P. L., Abreu P. A. E.. GroEL Protein of the Leptospira Spp.
Interacts with Host Proteins and Induces Cytokines Secretion on Macrophages. BMC Microbiol. 2021;21(1):99. doi: 10.1186/s12866-021-02162-w.</cite> [<a href="https://doi.org/10.1186/s12866-021-02162-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8011160/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33789603/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Microbiol.&amp;title=GroEL%20Protein%20of%20the%20Leptospira%20Spp.%20Interacts%20with%20Host%20Proteins%20and%20Induces%20Cytokines%20Secretion%20on%20Macrophages&amp;author=J.%20D.%20Ho&amp;author=L.%20E.%20M.%20Takara&amp;author=D.%20Monaris&amp;author=A.%20P.%20Gon%C3%A7alves&amp;author=A.%20F.%20Souza-Filho&amp;volume=21&amp;issue=1&amp;publication_year=2021&amp;pages=99&amp;pmid=33789603&amp;doi=10.1186/s12866-021-02162-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>Eshghi A., Pinne M., Haake D. A., Zuerner R. L., Frank A., Cameron C. E.. Methylation and in Vivo Expression
of the Surface-Exposed Outer-Membrane Protein OmpL32. Microbiology. 2012;158(3):622–635. doi: 10.1099/mic.0.054767-0.</cite> [<a href="https://doi.org/10.1099/mic.0.054767-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3352116/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22174381/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Methylation%20and%20in%20Vivo%20Expression%20of%20the%20Surface-Exposed%20Outer-Membrane%20Protein%20OmpL32&amp;author=A.%20Eshghi&amp;author=M.%20Pinne&amp;author=D.%20A.%20Haake&amp;author=R.%20L.%20Zuerner&amp;author=A.%20Frank&amp;volume=158&amp;issue=3&amp;publication_year=2012&amp;pages=622-635&amp;pmid=22174381&amp;doi=10.1099/mic.0.054767-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34">
<cite>Murray G. L., Srikram A., Hoke D. E., Wunder E. A., Henry R., Lo M., Zhang K., Sermswan R. W., Ko A. I., Adler B.. Major Surface
Protein LipL32 Is Not Required for Either Acute or Chronic Infection
with . Infect. Immun. 2009;77(3):952–958. doi: 10.1128/IAI.01370-08.</cite> [<a href="https://doi.org/10.1128/IAI.01370-08" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2643616/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19103763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&amp;title=Major%20Surface%20Protein%20LipL32%20Is%20Not%20Required%20for%20Either%20Acute%20or%20Chronic%20Infection%20with&amp;author=G.%20L.%20Murray&amp;author=A.%20Srikram&amp;author=D.%20E.%20Hoke&amp;author=E.%20A.%20Wunder&amp;author=R.%20Henry&amp;volume=77&amp;issue=3&amp;publication_year=2009&amp;pages=952-958&amp;pmid=19103763&amp;doi=10.1128/IAI.01370-08&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref35">
<cite>Pinne M., Haake D. A.. LipL32 Is a Subsurface
Lipoprotein of : Presentation of New Data
and Reevaluation of Previous Studies. PLoS One. 2013;8(1):e51025.  doi: 10.1371/journal.pone.0051025.</cite> [<a href="https://doi.org/10.1371/journal.pone.0051025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3544172/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23323152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=LipL32%20Is%20a%20Subsurface%20Lipoprotein%20of%20:%20Presentation%20of%20New%20Data%20and%20Reevaluation%20of%20Previous%20Studies&amp;author=M.%20Pinne&amp;author=D.%20A.%20Haake&amp;volume=8&amp;issue=1&amp;publication_year=2013&amp;pages=e51025&amp;pmid=23323152&amp;doi=10.1371/journal.pone.0051025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>Hoke D. E., Egan S., Cullen P. A., Adler B.. LipL32 Is an Extracellular
Matrix-Interacting Protein of Leptospira Spp. and . Infect. Immun. 2008;76(5):2063–2069. doi: 10.1128/IAI.01643-07.</cite> [<a href="https://doi.org/10.1128/IAI.01643-07" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2346718/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18285490/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&amp;title=LipL32%20Is%20an%20Extracellular%20Matrix-Interacting%20Protein%20of%20Leptospira%20Spp.%20and&amp;author=D.%20E.%20Hoke&amp;author=S.%20Egan&amp;author=P.%20A.%20Cullen&amp;author=B.%20Adler&amp;volume=76&amp;issue=5&amp;publication_year=2008&amp;pages=2063-2069&amp;pmid=18285490&amp;doi=10.1128/IAI.01643-07&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>Ahmed A. A., Goris M. G. A., Meijer M. C.. Development of LipL32 Real-Time PCR
Combined with an Internal and Extraction Control for Pathogenic Leptospira
Detection. PLoS One. 2020;15(11):e0241584.  doi: 10.1371/journal.pone.0241584.</cite> [<a href="https://doi.org/10.1371/journal.pone.0241584" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7605690/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33137154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Development%20of%20LipL32%20Real-Time%20PCR%20Combined%20with%20an%20Internal%20and%20Extraction%20Control%20for%20Pathogenic%20Leptospira%20Detection&amp;author=A.%20A.%20Ahmed&amp;author=M.%20G.%20A.%20Goris&amp;author=M.%20C.%20Meijer&amp;volume=15&amp;issue=11&amp;publication_year=2020&amp;pages=e0241584&amp;pmid=33137154&amp;doi=10.1371/journal.pone.0241584&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Vieira M. L., Persson S., Lopes-ferreira M., Romero E. C., Kirchgatter K., Nascimento A. L. T. O., Herwald H.. Heparin-Binding Protein Release Is
Strongly Induced by Leptospira Species and Is a Candidate for an Early
Diagnostic Marker of Human Leptospirosis. J.
Infect. Dis. 2019;219:996–1006. doi: 10.1093/infdis/jiy589.</cite> [<a href="https://doi.org/10.1093/infdis/jiy589" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30299510/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Infect.%20Dis.&amp;title=Heparin-Binding%20Protein%20Release%20Is%20Strongly%20Induced%20by%20Leptospira%20Species%20and%20Is%20a%20Candidate%20for%20an%20Early%20Diagnostic%20Marker%20of%20Human%20Leptospirosis&amp;author=M.%20L.%20Vieira&amp;author=S.%20Persson&amp;author=M.%20Lopes-ferreira&amp;author=E.%20C.%20Romero&amp;author=K.%20Kirchgatter&amp;volume=219&amp;publication_year=2019&amp;pages=996-1006&amp;pmid=30299510&amp;doi=10.1093/infdis/jiy589&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Ugen K. E., Muthumani K., Vijayachari P., Vedhagiri K., Mallilankaraman K., Mathur P. P., Sardesai N. Y., Weiner D. B., Ugen K. E., Muthumani K.. Immunogenicity of a Novel Enhanced
Consensus DNA Vaccine Encoding the Leptospiral Protein Immunogenicity
of a Novel Enhanced Consensus DNA Vaccine Encoding the Leptospiral
Protein LipL45. Hum. Vaccines Immunother. 2015;5515:1945–1953. doi: 10.1080/21645515.2015.1047117.</cite> [<a href="https://doi.org/10.1080/21645515.2015.1047117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4635901/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26020621/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hum.%20Vaccines%20Immunother.&amp;title=Immunogenicity%20of%20a%20Novel%20Enhanced%20Consensus%20DNA%20Vaccine%20Encoding%20the%20Leptospiral%20Protein%20Immunogenicity%20of%20a%20Novel%20Enhanced%20Consensus%20DNA%20Vaccine%20Encoding%20the%20Leptospiral%20Protein%20LipL45&amp;author=K.%20E.%20Ugen&amp;author=K.%20Muthumani&amp;author=P.%20Vijayachari&amp;author=K.%20Vedhagiri&amp;author=K.%20Mallilankaraman&amp;volume=5515&amp;publication_year=2015&amp;pages=1945-1953&amp;pmid=26020621&amp;doi=10.1080/21645515.2015.1047117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Cafiero T., Toledo A..  Surface Exposed GroEL Is a Multifunctional Protein. Pathogens. 2021;10(2):226. doi: 10.3390/pathogens10020226.</cite> [<a href="https://doi.org/10.3390/pathogens10020226" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7922809/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33670728/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pathogens&amp;title=Surface%20Exposed%20GroEL%20Is%20a%20Multifunctional%20Protein&amp;author=T.%20Cafiero&amp;author=A.%20Toledo&amp;volume=10&amp;issue=2&amp;publication_year=2021&amp;pages=226&amp;pmid=33670728&amp;doi=10.3390/pathogens10020226&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Maneewatchararangsri S., Doungchawee G., Kalambaheti T., Luvira V., Soonthornworasiri N., Vattanatham P., Chaisri U., Adisakwattana P.. Evaluation
of a Genus-Specific RGroEL1–524 IgM-ELISA and Commercial ELISA
Kits during the Course of Leptospirosis in Thailand. Sci. Rep. 2021;11(1):19785.  doi: 10.1038/s41598-021-99377-8.</cite> [<a href="https://doi.org/10.1038/s41598-021-99377-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8492722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34611273/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Evaluation%20of%20a%20Genus-Specific%20RGroEL1%E2%80%93524%20IgM-ELISA%20and%20Commercial%20ELISA%20Kits%20during%20the%20Course%20of%20Leptospirosis%20in%20Thailand&amp;author=S.%20Maneewatchararangsri&amp;author=G.%20Doungchawee&amp;author=T.%20Kalambaheti&amp;author=V.%20Luvira&amp;author=N.%20Soonthornworasiri&amp;volume=11&amp;issue=1&amp;publication_year=2021&amp;pages=19785&amp;pmid=34611273&amp;doi=10.1038/s41598-021-99377-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Fraga T. R., Courrol S., Castiblanco-Valencia M. M., Hirata I. Y., Vasconcellos S. A., Juliano L., Barbosa A. S., Isaac L.. Immune Evasion
by Pathogenic Leptospira Strains : The Secretion of Proteases
That Directly Cleave Complement Proteins. J.
Infect. Dis. 2014;209(6):876–886. doi: 10.1093/infdis/jit569.</cite> [<a href="https://doi.org/10.1093/infdis/jit569" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24163418/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Infect.%20Dis.&amp;title=Immune%20Evasion%20by%20Pathogenic%20Leptospira%20Strains%E2%80%AF:%20The%20Secretion%20of%20Proteases%20That%20Directly%20Cleave%20Complement%20Proteins&amp;author=T.%20R.%20Fraga&amp;author=S.%20Courrol&amp;author=M.%20M.%20Castiblanco-Valencia&amp;author=I.%20Y.%20Hirata&amp;author=S.%20A.%20Vasconcellos&amp;volume=209&amp;issue=6&amp;publication_year=2014&amp;pages=876-886&amp;pmid=24163418&amp;doi=10.1093/infdis/jit569&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Maruyama Y., Ochiai A., Mikami B., Hashimoto W., Murata K.. Biochemical and Biophysical Research
Communications
Crystal Structure of Bacterial Cell-Surface Alginate-Binding Protein
with an M75 Peptidase Motif Q. Biochem. Biophys.
Res. Commun. 2011;405(3):411–416. doi: 10.1016/j.bbrc.2011.01.043.</cite> [<a href="https://doi.org/10.1016/j.bbrc.2011.01.043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21238429/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;title=Biochemical%20and%20Biophysical%20Research%20Communications%20Crystal%20Structure%20of%20Bacterial%20Cell-Surface%20Alginate-Binding%20Protein%20with%20an%20M75%20Peptidase%20Motif%20Q&amp;author=Y.%20Maruyama&amp;author=A.%20Ochiai&amp;author=B.%20Mikami&amp;author=W.%20Hashimoto&amp;author=K.%20Murata&amp;volume=405&amp;issue=3&amp;publication_year=2011&amp;pages=411-416&amp;pmid=21238429&amp;doi=10.1016/j.bbrc.2011.01.043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref44">
<cite>Schuler E., Marconi R.. The Leptospiral General Secretory
Protein D (GspD),
a Secretin, Elicits Complement-Independent Bactericidal Antibody against
Diverse Leptospira Species and Serovars. Vaccine:
X. 2021;7:100089.  doi: 10.1016/j.jvacx.2021.100089.</cite> [<a href="https://doi.org/10.1016/j.jvacx.2021.100089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7941034/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33733085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Vaccine:%20X&amp;title=The%20Leptospiral%20General%20Secretory%20Protein%20D%20(GspD),%20a%20Secretin,%20Elicits%20Complement-Independent%20Bactericidal%20Antibody%20against%20Diverse%20Leptospira%20Species%20and%20Serovars&amp;author=E.%20Schuler&amp;author=R.%20Marconi&amp;volume=7&amp;publication_year=2021&amp;pages=100089&amp;pmid=33733085&amp;doi=10.1016/j.jvacx.2021.100089&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref45">
<cite>Vieira M. L., Atzingen M. V., Oliveira R., Mendes R. S., Domingos R. F., Vasconcellos S. A., Nascimento A. L. T. O.. Plasminogen Binding Proteins and
Plasmin Generation on the Surface of Leptospira Spp.: The Contribution
to the Bacteria-Host Interactions. J. Biomed.
Biotechnol. 2012;2012:758513.  doi: 10.1155/2012/758513.</cite> [<a href="https://doi.org/10.1155/2012/758513" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3481863/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23118516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomed.%20Biotechnol.&amp;title=Plasminogen%20Binding%20Proteins%20and%20Plasmin%20Generation%20on%20the%20Surface%20of%20Leptospira%20Spp.:%20The%20Contribution%20to%20the%20Bacteria-Host%20Interactions&amp;author=M.%20L.%20Vieira&amp;author=M.%20V.%20Atzingen&amp;author=R.%20Oliveira&amp;author=R.%20S.%20Mendes&amp;author=R.%20F.%20Domingos&amp;volume=2012&amp;publication_year=2012&amp;pages=758513&amp;pmid=23118516&amp;doi=10.1155/2012/758513&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref46">
<cite>Cavenague M. F., Teixeira A. F., Filho A. S., Souza G. O., Vasconcellos S. A., Heinemann M. B., Nascimento A. L. T. O.. Characterization of a Novel Protein
of Exhibiting
Plasminogen, Vitronectin and Complement Binding Properties. Int. J. Med. Microbiol. 2019;309(2):116–129. doi: 10.1016/j.ijmm.2018.12.005.</cite> [<a href="https://doi.org/10.1016/j.ijmm.2018.12.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30638770/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Med.%20Microbiol.&amp;title=Characterization%20of%20a%20Novel%20Protein%20of%20Exhibiting%20Plasminogen,%20Vitronectin%20and%20Complement%20Binding%20Properties&amp;author=M.%20F.%20Cavenague&amp;author=A.%20F.%20Teixeira&amp;author=A.%20S.%20Filho&amp;author=G.%20O.%20Souza&amp;author=S.%20A.%20Vasconcellos&amp;volume=309&amp;issue=2&amp;publication_year=2019&amp;pages=116-129&amp;pmid=30638770&amp;doi=10.1016/j.ijmm.2018.12.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref47">
<cite>Vieira M. L., Atzingen M. V., Oliveira T. R., Oliveira R., Andrade D. M., Vasconcellos S. A., Nascimento A. L. T. O.. In Vitro Identification of Novel
Plasminogen-Binding Receptors of the Pathogen . PLoS One. 2010;5(6):e11259.  doi: 10.1371/journal.pone.0011259.</cite> [<a href="https://doi.org/10.1371/journal.pone.0011259" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2889836/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20582320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=In%20Vitro%20Identification%20of%20Novel%20Plasminogen-Binding%20Receptors%20of%20the%20Pathogen&amp;author=M.%20L.%20Vieira&amp;author=M.%20V.%20Atzingen&amp;author=T.%20R.%20Oliveira&amp;author=R.%20Oliveira&amp;author=D.%20M.%20Andrade&amp;volume=5&amp;issue=6&amp;publication_year=2010&amp;pages=e11259&amp;pmid=20582320&amp;doi=10.1371/journal.pone.0011259&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref48">
<cite>Daroz B. B., Fernandes L. G. V., Cavenague M. F., Kochi L. T., Passalia F. J., Takahashi M. B., Nascimento Filho E. G., Teixeira A. F., Nascimento A. L. T. O.. A
Review on Host-Leptospira Interactions: What We Know and Future Expectations. Front. Cell. Infect. Microbiol. 2021;11:777709.  doi: 10.3389/fcimb.2021.777709.</cite> [<a href="https://doi.org/10.3389/fcimb.2021.777709" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8657130/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34900757/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Cell.%20Infect.%20Microbiol.&amp;title=A%20Review%20on%20Host-Leptospira%20Interactions:%20What%20We%20Know%20and%20Future%20Expectations&amp;author=B.%20B.%20Daroz&amp;author=L.%20G.%20V.%20Fernandes&amp;author=M.%20F.%20Cavenague&amp;author=L.%20T.%20Kochi&amp;author=F.%20J.%20Passalia&amp;volume=11&amp;publication_year=2021&amp;pages=777709&amp;pmid=34900757&amp;doi=10.3389/fcimb.2021.777709&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref49">
<cite>Cianciotto N. P., White R. C.. Expanding Role of Type II Secretion
in Bacterial Pathogenesis
and Beyond. Infect. Immun. 2017;85(5):10–1128. doi: 10.1128/IAI.00014-17.</cite> [<a href="https://doi.org/10.1128/IAI.00014-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5400843/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28264910/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Infect.%20Immun.&amp;title=Expanding%20Role%20of%20Type%20II%20Secretion%20in%20Bacterial%20Pathogenesis%20and%20Beyond&amp;author=N.%20P.%20Cianciotto&amp;author=R.%20C.%20White&amp;volume=85&amp;issue=5&amp;publication_year=2017&amp;pages=10-1128&amp;pmid=28264910&amp;doi=10.1128/IAI.00014-17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref50">
<cite>Linhartová I., Bumba L., Mašn J., Basler M., Osička R., Kamanová J., Procházková K., Adkins I., HejnováHolubová J., Sadílková L., Morová J., Šebo P.. RTX Proteins: A Highly Diverse Family
Secreted Bya Common Mechanism. FEMS Microbiol.
Rev. 2010;34(6):1076–1112. doi: 10.1111/j.1574-6976.2010.00231.x.</cite> [<a href="https://doi.org/10.1111/j.1574-6976.2010.00231.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3034196/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20528947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Rev.&amp;title=RTX%20Proteins:%20A%20Highly%20Diverse%20Family%20Secreted%20Bya%20Common%20Mechanism&amp;author=I.%20Linhartov%C3%A1&amp;author=L.%20Bumba&amp;author=J.%20Ma%C5%A1n&amp;author=M.%20Basler&amp;author=R.%20Osi%C4%8Dka&amp;volume=34&amp;issue=6&amp;publication_year=2010&amp;pages=1076-1112&amp;pmid=20528947&amp;doi=10.1111/j.1574-6976.2010.00231.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref51">
<cite>Prasad T. S. K., Harsha H. C., Keerthikumar S., Sekhar N. R., Selvan L. D. N., Kumar P., Pinto S. M., Muthusamy B., Subbannayya Y., Renuse S., Chaerkady R., Mathur P. P., Ravikumar R., Pandey A.. Proteogenomic Analysis
of Candida Glabrata Using High Resolution Mass Spectrometry. J. Proteome Res. 2012;11(1):247–260. doi: 10.1021/pr200827k.</cite> [<a href="https://doi.org/10.1021/pr200827k" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22129275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res.&amp;title=Proteogenomic%20Analysis%20of%20Candida%20Glabrata%20Using%20High%20Resolution%20Mass%20Spectrometry&amp;author=T.%20S.%20K.%20Prasad&amp;author=H.%20C.%20Harsha&amp;author=S.%20Keerthikumar&amp;author=N.%20R.%20Sekhar&amp;author=L.%20D.%20N.%20Selvan&amp;volume=11&amp;issue=1&amp;publication_year=2012&amp;pages=247-260&amp;pmid=22129275&amp;doi=10.1021/pr200827k&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref52">
<cite>

Kruger, N. J.

The Bradford
Method for Protein Quantitation. In Protein
Protocols Handbook, 1996; Vol. 32, pp 15–20.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Protocols%20Handbook&amp;title=The%20Bradford%20Method%20for%20Protein%20Quantitation&amp;volume=32&amp;publication_year=1996&amp;pages=15-20&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref53">
<cite>Taylor S. C., Posch A.. The Design of a Quantitative Western Blot Experiment. BioMed Res. Int. 2014;2014:361590.  doi: 10.1155/2014/361590.</cite> [<a href="https://doi.org/10.1155/2014/361590" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3971489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24738055/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BioMed%20Res.%20Int.&amp;title=The%20Design%20of%20a%20Quantitative%20Western%20Blot%20Experiment&amp;author=S.%20C.%20Taylor&amp;author=A.%20Posch&amp;volume=2014&amp;publication_year=2014&amp;pages=361590&amp;pmid=24738055&amp;doi=10.1155/2014/361590&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref54">
<cite>Mruk D. D., Cheng C. Y.. Enhanced Chemiluminescence (ECL) for Routine Immunoblotting:
An Inexpensive Alternative to Commercially Available Kits. Spermatogenesis. 2011;1:121–122. doi: 10.4161/spmg.1.2.16606.</cite> [<a href="https://doi.org/10.4161/spmg.1.2.16606" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3271654/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22319660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Spermatogenesis&amp;title=Enhanced%20Chemiluminescence%20(ECL)%20for%20Routine%20Immunoblotting:%20An%20Inexpensive%20Alternative%20to%20Commercially%20Available%20Kits&amp;author=D.%20D.%20Mruk&amp;author=C.%20Y.%20Cheng&amp;volume=1&amp;publication_year=2011&amp;pages=121-122&amp;pmid=22319660&amp;doi=10.4161/spmg.1.2.16606&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref55">
<cite>Gupta A., Kapil R., Dhakan D. B., Sharma V. K.. MP3: A Software
Tool for the Prediction of Pathogenic Proteins in Genomic and Metagenomic
Data. PLoS One. 2014;9(4):e93907.  doi: 10.1371/journal.pone.0093907.</cite> [<a href="https://doi.org/10.1371/journal.pone.0093907" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3988012/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24736651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=MP3:%20A%20Software%20Tool%20for%20the%20Prediction%20of%20Pathogenic%20Proteins%20in%20Genomic%20and%20Metagenomic%20Data&amp;author=A.%20Gupta&amp;author=R.%20Kapil&amp;author=D.%20B.%20Dhakan&amp;author=V.%20K.%20Sharma&amp;volume=9&amp;issue=4&amp;publication_year=2014&amp;pages=e93907&amp;pmid=24736651&amp;doi=10.1371/journal.pone.0093907&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref56">
<cite>Ayalew S., Confer A. W., Hartson S. D., Canaan P. J., Payton M., Couger B.. Proteomic and Bioinformatic
Analyses of Putative Mannheimia
Haemolytica Secretome by Liquid Chromatography and Tandem Mass Spectrometry. Vet. Microbiol. 2017;203:73–80. doi: 10.1016/j.vetmic.2017.02.011.</cite> [<a href="https://doi.org/10.1016/j.vetmic.2017.02.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28619171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Vet.%20Microbiol.&amp;title=Proteomic%20and%20Bioinformatic%20Analyses%20of%20Putative%20Mannheimia%20Haemolytica%20Secretome%20by%20Liquid%20Chromatography%20and%20Tandem%20Mass%20Spectrometry&amp;author=S.%20Ayalew&amp;author=A.%20W.%20Confer&amp;author=S.%20D.%20Hartson&amp;author=P.%20J.%20Canaan&amp;author=M.%20Payton&amp;volume=203&amp;publication_year=2017&amp;pages=73-80&amp;pmid=28619171&amp;doi=10.1016/j.vetmic.2017.02.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref57">
<cite>Chen C., Liu H., Zabad S., Rivera N., Rowin E., Hassan M., Gomez De Jesus S. M., Llinás Santos P. S., Kravchenko K., Mikhova M., Ketterer S., Shen A., Shen S., Navas E., Horan B., Raudsepp J., Jeffery C.. MoonProt 3.0:
An Update of the Moonlighting Proteins Database. Nucleic Acids Res. 2021;49(D1):D368–D372. doi: 10.1093/nar/gkaa1101.</cite> [<a href="https://doi.org/10.1093/nar/gkaa1101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7778978/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33245761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=MoonProt%203.0:%20An%20Update%20of%20the%20Moonlighting%20Proteins%20Database&amp;author=C.%20Chen&amp;author=H.%20Liu&amp;author=S.%20Zabad&amp;author=N.%20Rivera&amp;author=E.%20Rowin&amp;volume=49&amp;issue=D1&amp;publication_year=2021&amp;pages=D368-D372&amp;pmid=33245761&amp;doi=10.1093/nar/gkaa1101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref58">
<cite>Chen L., Zhang Y.-H., Wang S., Zhang Y., Huang T., Cai Y.-D.. Prediction and Analysis
of Essential Genes Using the
Enrichments of Gene Ontology and KEGG Pathways. PLoS One. 2017;12(9):e0184129.  doi: 10.1371/journal.pone.0184129.</cite> [<a href="https://doi.org/10.1371/journal.pone.0184129" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5584762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28873455/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Prediction%20and%20Analysis%20of%20Essential%20Genes%20Using%20the%20Enrichments%20of%20Gene%20Ontology%20and%20KEGG%20Pathways&amp;author=L.%20Chen&amp;author=Y.-H.%20Zhang&amp;author=S.%20Wang&amp;author=Y.%20Zhang&amp;author=T.%20Huang&amp;volume=12&amp;issue=9&amp;publication_year=2017&amp;pages=e0184129&amp;pmid=28873455&amp;doi=10.1371/journal.pone.0184129&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref59">
<cite>Aguilan J. T., Kulej K., Sidoli S.. Guide for Protein Fold
Change and
P-Value Calculation for Non-Experts in Proteomics. Mol. Omics. 2020;16(6):573–582. doi: 10.1039/D0MO00087F.</cite> [<a href="https://doi.org/10.1039/D0MO00087F" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32968743/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Omics&amp;title=Guide%20for%20Protein%20Fold%20Change%20and%20P-Value%20Calculation%20for%20Non-Experts%20in%20Proteomics&amp;author=J.%20T.%20Aguilan&amp;author=K.%20Kulej&amp;author=S.%20Sidoli&amp;volume=16&amp;issue=6&amp;publication_year=2020&amp;pages=573-582&amp;pmid=32968743&amp;doi=10.1039/D0MO00087F&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref60">
<cite>Zhou X., Zhou L., Ge X., Guo X., Han J., Zhang Y., Yang H.. Quantitative Proteomic Analysis of
Porcine Intestinal Epithelial Cells Infected with Porcine Deltacoronavirus
Using ITRAQ-Coupled LC-MS/MS. J. Proteome Res. 2020;19(11):4470–4485. doi: 10.1021/acs.jproteome.0c00592.</cite> [<a href="https://doi.org/10.1021/acs.jproteome.0c00592" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33045833/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Proteome%20Res.&amp;title=Quantitative%20Proteomic%20Analysis%20of%20Porcine%20Intestinal%20Epithelial%20Cells%20Infected%20with%20Porcine%20Deltacoronavirus%20Using%20ITRAQ-Coupled%20LC-MS/MS&amp;author=X.%20Zhou&amp;author=L.%20Zhou&amp;author=X.%20Ge&amp;author=X.%20Guo&amp;author=J.%20Han&amp;volume=19&amp;issue=11&amp;publication_year=2020&amp;pages=4470-4485&amp;pmid=33045833&amp;doi=10.1021/acs.jproteome.0c00592&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref61">
<cite>Liu Y., Chen X., Pan L., Mao Z.. Differential Protein
Expression of a Streptomycin-Resistant Streptomyces Albulus Mutant
in High Yield Production of ε-Poly-l-Lysine: A Proteomics Study. RSC Adv. 2019;9(42):24092–24104. doi: 10.1039/C9RA03156A.</cite> [<a href="https://doi.org/10.1039/C9RA03156A" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9069503/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35527895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Adv.&amp;title=Differential%20Protein%20Expression%20of%20a%20Streptomycin-Resistant%20Streptomyces%20Albulus%20Mutant%20in%20High%20Yield%20Production%20of%20%CE%B5-Poly-l-Lysine:%20A%20Proteomics%20Study&amp;author=Y.%20Liu&amp;author=X.%20Chen&amp;author=L.%20Pan&amp;author=Z.%20Mao&amp;volume=9&amp;issue=42&amp;publication_year=2019&amp;pages=24092-24104&amp;pmid=35527895&amp;doi=10.1039/C9RA03156A&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref62">
<cite>Gao Z., Wang J., Bai Y., Bao J., Dai E.. Identification
and Verification of the Main Differentially Expressed Proteins in
Gastric Cancer via ITRAQ Combined with Liquid Chromatography-Mass
Spectrometry. Anal. Cell. Pathol. 2019;2019:5310684.  doi: 10.1155/2019/5310684.</cite> [<a href="https://doi.org/10.1155/2019/5310684" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6914953/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31886120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Cell.%20Pathol.&amp;title=Identification%20and%20Verification%20of%20the%20Main%20Differentially%20Expressed%20Proteins%20in%20Gastric%20Cancer%20via%20ITRAQ%20Combined%20with%20Liquid%20Chromatography-Mass%20Spectrometry&amp;author=Z.%20Gao&amp;author=J.%20Wang&amp;author=Y.%20Bai&amp;author=J.%20Bao&amp;author=E.%20Dai&amp;volume=2019&amp;publication_year=2019&amp;pages=5310684&amp;pmid=31886120&amp;doi=10.1155/2019/5310684&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12332550/bin/ao4c10983_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">ao4c10983_si_001.pdf</a><sup> (1.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12332550/bin/ao4c10983_si_002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ao4c10983_si_002.xlsx</a><sup> (7MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>
<strong>The proteomic
data obtained by MS analysis were deposited to the ProteomeXchange
Consortium via the Proteomics identifications database (PMID), a partner
repository, under the data set identifiers PXD035782 and PXD037702.</strong>
</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Omega are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsomega.4c10983"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/ao4c10983.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (9.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12332550/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12332550/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332550%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332550/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12332550/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12332550/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40787326/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12332550/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40787326/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12332550/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12332550/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="yW0omjoyWuH8IVhFoexgpOGU0dAQVkEDvTSMFElmNV3jt1NQDPek2EorIOrM327c">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
