
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Scientific planning of dynamic crops in complex agricultural landscapes based on adaptive optimization hybrid SA-GA method - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="4DDE493489B755331B49340024375773.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12331943/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Scientific planning of dynamic crops in complex agricultural landscapes based on adaptive optimization hybrid SA-GA method">
<meta name="citation_author" content="Changlong Li">
<meta name="citation_author_institution" content="School of Information Technology and Engineering, Guangzhou College of Commerce, Guangzhou, 511363 China">
<meta name="citation_author" content="Zengye Su">
<meta name="citation_author_institution" content="School of Information Technology and Engineering, Guangzhou College of Commerce, Guangzhou, 511363 China">
<meta name="citation_author" content="Yudan Nie">
<meta name="citation_author_institution" content="School of Information Technology and Engineering, Guangzhou College of Commerce, Guangzhou, 511363 China">
<meta name="citation_author" content="Zhiyi Ye">
<meta name="citation_author_institution" content="School of Economics, Guangzhou College of Commerce, Guangzhou, 511363 China">
<meta name="citation_author" content="Jinyi Li">
<meta name="citation_author_institution" content="School of Digital Economy Industry, Guangzhou College of Commerce, Guangzhou, 511363 China">
<meta name="citation_author" content="Jing Wang">
<meta name="citation_author_institution" content="School of Modern Information Industry, Guangzhou College of Commerce, Guangzhou, 511363 China">
<meta name="citation_author" content="Zicong Yang">
<meta name="citation_author_institution" content="School of Information Technology and Engineering, Guangzhou College of Commerce, Guangzhou, 511363 China">
<meta name="citation_author" content="Xuxin Li">
<meta name="citation_author_institution" content="School of Information Technology and Engineering, Guangzhou College of Commerce, Guangzhou, 511363 China">
<meta name="citation_author" content="Weijian Zeng">
<meta name="citation_author_institution" content="School of Information Technology and Engineering, Guangzhou College of Commerce, Guangzhou, 511363 China">
<meta name="citation_author" content="Yanjian Chen">
<meta name="citation_author_institution" content="School of Information Technology and Engineering, Guangzhou College of Commerce, Guangzhou, 511363 China">
<meta name="citation_publication_date" content="2025 Aug 8">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="28992">
<meta name="citation_doi" content="10.1038/s41598-025-14188-5">
<meta name="citation_pmid" content="40775001">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12331943/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12331943/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12331943/pdf/41598_2025_Article_14188.pdf">
<meta name="description" content="Effective dynamic agricultural planning is crucial for optimising resource allocation and ensuring income stability, yet conventional methods often face limitations in adapting to the complex and variable conditions of mountainous regions, ...">
<meta name="og:title" content="Scientific planning of dynamic crops in complex agricultural landscapes based on adaptive optimization hybrid SA-GA method">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Effective dynamic agricultural planning is crucial for optimising resource allocation and ensuring income stability, yet conventional methods often face limitations in adapting to the complex and variable conditions of mountainous regions, ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12331943/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12331943">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-14188-5"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_14188.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12331943%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12331943/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12331943/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12331943/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 8;15:28992. doi: <a href="https://doi.org/10.1038/s41598-025-14188-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-14188-5</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Scientific planning of dynamic crops in complex agricultural landscapes based on adaptive optimization hybrid SA-GA method</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Changlong Li</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Changlong Li</span></h3>
<div class="p">
<sup>1</sup>School of Information Technology and Engineering, Guangzhou College of Commerce, Guangzhou, 511363 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Changlong Li</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Su%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Zengye Su</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Zengye Su</span></h3>
<div class="p">
<sup>1</sup>School of Information Technology and Engineering, Guangzhou College of Commerce, Guangzhou, 511363 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Su%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zengye Su</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nie%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yudan Nie</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yudan Nie</span></h3>
<div class="p">
<sup>1</sup>School of Information Technology and Engineering, Guangzhou College of Commerce, Guangzhou, 511363 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nie%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yudan Nie</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ye%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Zhiyi Ye</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Zhiyi Ye</span></h3>
<div class="p">
<sup>2</sup>School of Economics, Guangzhou College of Commerce, Guangzhou, 511363 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ye%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhiyi Ye</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Jinyi Li</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Jinyi Li</span></h3>
<div class="p">
<sup>3</sup>School of Digital Economy Industry, Guangzhou College of Commerce, Guangzhou, 511363 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jinyi Li</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Jing Wang</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jing Wang</span></h3>
<div class="p">
<sup>4</sup>School of Modern Information Industry, Guangzhou College of Commerce, Guangzhou, 511363 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jing Wang</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Zicong Yang</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Zicong Yang</span></h3>
<div class="p">
<sup>1</sup>School of Information Technology and Engineering, Guangzhou College of Commerce, Guangzhou, 511363 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zicong Yang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Xuxin Li</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Xuxin Li</span></h3>
<div class="p">
<sup>1</sup>School of Information Technology and Engineering, Guangzhou College of Commerce, Guangzhou, 511363 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xuxin Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zeng%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Weijian Zeng</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Weijian Zeng</span></h3>
<div class="p">
<sup>1</sup>School of Information Technology and Engineering, Guangzhou College of Commerce, Guangzhou, 511363 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zeng%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Weijian Zeng</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Yanjian Chen</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Yanjian Chen</span></h3>
<div class="p">
<sup>1</sup>School of Information Technology and Engineering, Guangzhou College of Commerce, Guangzhou, 511363 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yanjian Chen</span></a>
</div>
</div>
<sup>1</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>School of Information Technology and Engineering, Guangzhou College of Commerce, Guangzhou, 511363 China </div>
<div id="Aff2">
<sup>2</sup>School of Economics, Guangzhou College of Commerce, Guangzhou, 511363 China </div>
<div id="Aff3">
<sup>3</sup>School of Digital Economy Industry, Guangzhou College of Commerce, Guangzhou, 511363 China </div>
<div id="Aff4">
<sup>4</sup>School of Modern Information Industry, Guangzhou College of Commerce, Guangzhou, 511363 China </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 12; Accepted 2025 Jul 29; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12331943  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40775001/" class="usa-link">40775001</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Effective dynamic agricultural planning is crucial for optimising resource allocation and ensuring income stability, yet conventional methods often face limitations in adapting to the complex and variable conditions of mountainous regions, particularly under fluctuating climate and market pressures. Therefore, this study introduces a novel multi-stage dynamic optimization framework specifically designed for crop planning in such challenging terrains. This framework is underpinned by a sophisticated model integrating advanced monitoring systems with a Hybrid Simulated Annealing Genetic Algorithm (H-SAGA), further enhanced by neural network-driven real-time predictions. The H-SAGA component optimises planting strategies by synergistically combining global exploration (SA) and local refinement (GA) capabilities, while the neural network dynamically adjusts revenue forecasts based on climatic and market data, significantly improving the model’s responsiveness and adaptability. We rigorously evaluated the applicability and effectiveness of this model through extensive simulations across 7,290 mu (1,201 acres) of diverse farmland in mountainous Northern China. The results demonstrate that the proposed H-SAGA approach consistently achieves 5–10 percentage points higher profit increment ratios than other benchmark optimization algorithms (such as GA, SA, PSO, and ACO), alongside faster convergence and notable robustness against environmental and economic variability. This research establishes an integrated “monitoring-modelling-decision” paradigm, driven by multi-source data and machine learning, offering a practical and robust tool that provides valuable guidance for enhancing resource allocation efficiency and promoting sustainable precision agriculture in complex topographical regions, thereby holding significant reference value for optimising agricultural production nationwide.</p>
<section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Hybrid H-SAGA, Agricultural optimization, Crop-planning, Resource efficiency, Climate adaptation, Northern China Mountains</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Plant sciences, Agroecology, Environmental sciences, Mathematics and computing, Applied mathematics, Computational science, Computer science</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">In Northern China’s mountainous regions, agricultural productivity is constrained by three core challenges: land fragmentation, a fragile water resource system, and increasing climatic variability<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. These factors create a <strong>dynamic environment</strong>—characterized by fluctuating weather patterns, resource availability, and market pressures—where traditional cultivation practices are poorly adapted, resulting in low resource efficiency and unstable outputs. This necessitates the use of advanced optimization methods, specifically <strong>adaptive optimization</strong> which involves sequentially updating decisions in response to new information, to guide land use and crop layout in such complex terrains<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>.</p>
<p id="Par3">The study is geographically grounded in North China (Fig. <a href="#Fig1" class="usa-link">1</a>a), a region defined by complex topography and critical river systems that shape its agricultural potential (Fig. <a href="#Fig1" class="usa-link">1</a>b). This landscape is not static; it exhibits significant environmental and agricultural dynamics. Spatiotemporal analysis from 2000 to 2024 reveals considerable volatility in cropland extent, with concurrent patterns of land loss and gain (Fig. <a href="#Fig1" class="usa-link">1</a>c). This instability is mirrored in the health of the vegetation itself, as the Normalized Difference Vegetation Index (NDVI) for the region’s farmland shows marked variability over the past two decades (Fig. <a href="#Fig2" class="usa-link">2</a>). Layered upon this dynamic environment is a multifaceted agricultural economy, characterized by substantial disparities in crop choices and production volumes across different provinces (Fig. <a href="#Fig3" class="usa-link">3</a>). The confluence of this intricate topography, pronounced environmental fluctuations, and complex production systems presents a formidable challenge for effective agricultural planning. It is precisely this multi-layered complexity that necessitates a sophisticated, adaptive optimization framework capable of identifying robust and efficient solutions.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12331943_41598_2025_14188_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/fedac6eba84c/41598_2025_14188_Fig1_HTML.jpg" loading="lazy" id="MO1" height="519" width="799" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Geographical features and cropland dynamics in North China (2000-2024). (<strong>a</strong>) Location of the study area within China. (<strong>b</strong>) A depiction of the region’s land use, topography (via contour lines), and major river systems as of 2020. (<strong>c</strong>) Spatiotemporal changes in cropland, illustrating areas of stable cultivation, loss, and gain over the past two decades. The maps in this figure were generated by the authors using Python (v3.8) with the Matplotlib (v3.5.2), Geemap (v0.30.4), and Google Earth Engine (v0.1.399) libraries. The respective software is available at <a href="https://www.python.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.python.org</a>, <a href="https://matplotlib.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://matplotlib.org</a>, <a href="https://geemap.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://geemap.org</a>, and <a href="https://developers.google.com/earth-engine/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://developers.google.com/earth-engine/</a>. Data sources are detailed in the Methods section.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig2"><h3 class="obj_head">Fig. 2.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/7a12571daca1/41598_2025_14188_Fig2_HTML.jpg" loading="lazy" id="MO2" height="403" width="624" alt="Fig. 2"></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Mean Normalized Difference Vegetation Index (NDVI) for different land cover types from 2000 to 2024, indicating variability in vegetation health.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig3"><h3 class="obj_head">Fig. 3.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12331943_41598_2025_14188_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/5db4681d1d49/41598_2025_14188_Fig3_HTML.jpg" loading="lazy" id="MO3" height="454" width="708" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Agricultural production structure in North China’s major provinces, highlighting the multifaceted economic landscape.</p></figcaption></figure><p id="Par4">While mathematical programming techniques like linear programming have been applied to agricultural optimization<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>, their reliance on static parameters struggles to capture the non-linear dynamics and uncertainties inherent in mountainous agricultural systems<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. To address these complexities, metaheuristic algorithms have gained traction. Accordingly, we selected established methods—Genetic Algorithms (GA), Particle Swarm Optimization (PSO), Ant Colony Optimization (ACO), and Simulated Annealing (SA)—as benchmarks, as they represent the conventional state-of-the-art for such problems. However, these algorithms are often hampered by slow convergence and premature entrapment in local optima within high-dimensional search spaces<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. This limitation has motivated the exploration of hybrid strategies, such as combining the global exploration of SA with the refined search of GA, which forms the theoretical basis for our proposed approach<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>.</p>
<p id="Par5">A more profound limitation of existing models, including hybrid ones, is their static, single-stage nature, which is ill-suited to the inherent dynamism of agriculture. Effective planning requires sequential adjustments based on evolving intra-seasonal information, such as updated weather and market forecasts<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. The inability of current models to integrate such real-time data for adaptive decision-making represents a critical research gap, limiting both the resilience and profitability of agricultural operations.</p>
<p id="Par6">To fill this gap, this study proposes a novel multi-stage dynamic framework focused on land use and crop layout optimization in complex terrains. Its core is a Hybrid Simulated Annealing Genetic Algorithm (H-SAGA) integrated with a neural network-based predictive model to dynamically adjust revenue expectations based on the latest climatic and market data. This closed-loop “monitoring-modeling-decision” paradigm is designed to provide a smart decision-support tool for mountainous agriculture that is both robust and highly efficient. Therefore, this study aims to answer the following central research question:</p>
<p id="Par7">
<em>How can a dynamic, multi-stage optimization framework, integrating predictive analytics and a hybrid metaheuristic algorithm (H-SAGA), effectively enhance the economic profitability and resource-use efficiency of agricultural planning in the complex and variable conditions of Northern China’s mountainous regions?</em>
</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<p id="Par8">This section outlines the methodological framework employed in this study, integrating a multi-stage optimization model with the Hybrid Simulated Annealing Genetic Algorithm (H-SAGA) to optimize agricultural planning. The approach aims to maximize crop cultivation revenues while adapting to mountainous regions’ geographical, climatic, and resource constraints. It combines advanced optimization techniques with neural network predictions to enhance adaptability and precision in dynamic environments.</p>
<section id="Sec3"><h3 class="pmc_sec_title">Multi-stage optimization model</h3>
<p id="Par9">We propose a multi-stage optimization model<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup> designed to facilitate sequential decision-making across various decision points throughout the growing season. This framework adapts to fluctuations in market prices and climatic conditions<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, making it particularly effective for mountainous terrains where plot characteristics such as slope, altitude, and water availability significantly influence yield and resource use.</p>
<p id="Par10">It is crucial to clarify the core concept of “multi-stage dynamic adjustment” in this model. It does not imply uprooting an already planted Crop A to replace it with Crop B mid-season. The reality of farming dictates that once a crop is planted, it is typically seen through to harvest, as mid-season replacement would incur significant sunk costs and be impractical. Instead, a “stage” in our model represents a distinct decision-making point in time, strategically aligned with key phases of the regional cropping calendar (e.g., Stage 1: pre-sowing planning; Stage 2: early-growth review; Stage 3: mid-season resource adjustment). The “dynamic adjustment” at the beginning of each new stage manifests in two primary ways: </p>
<ol class="list" style="list-style-type:decimal">
<li><p id="Par11"><strong>optimization of uncommitted decisions:</strong> For plots that have not yet been assigned a crop (e.g., due to resource constraints or staggered planting schedules), the model re-optimises the crop choice and planting area based on the latest climate and market forecasts.</p></li>
<li><p id="Par12"><strong>Adaptation of future plans:</strong> The model can adjust the macro-level cropping strategy for the next year or the next planting cycle based on information gained in the current stage (e.g., observed yield performance, new market trends).</p></li>
</ol>
<p>For crops that are already planted, the model’s adaptive adjustments focus on optimising resource inputs (e.g., adjusting irrigation schedules based on updated drought forecasts) rather than changing the crop itself. This design ensures that the model’s dynamic adaptability aligns with the practical constraints of agriculture.</p>
<section id="Sec4"><h4 class="pmc_sec_title">Why multi-stage?</h4>
<ul class="list" style="list-style-type:disc">
<li><p id="Par14"><em>State Transitions:</em> Each stage incorporates updated data on prices, resource usage, and expected yields. Decisions at stage <em>t</em> shape the state at stage <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/c14faede1886/d33e427.gif" loading="lazy" id="d33e427" alt="Inline graphic"></span>, capturing the evolution of land conditions and resource availability.</p></li>
<li><p id="Par15"><em>Evolving Decision Variables:</em> Variables like <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/a3f185055944/d33e438.gif" loading="lazy" id="d33e438" alt="Inline graphic"></span> (area of crop <em>j</em> on plot <em>i</em> at stage <em>t</em>) evolve dynamically, allowing adjustments based on changing profitability or resource demands for uncommitted plots.</p></li>
<li><p id="Par16"><em>Risk Mitigation:</em> Multi-stage planning mitigates risks from unforeseen events (e.g., droughts or market shifts) by enabling adaptive strategies across stages.</p></li>
</ul></section><section id="Sec5"><h4 class="pmc_sec_title">Geographical and climatic considerations</h4>
<p id="Par17">In mountainous regions, plot-specific factors such as slope (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/7b6d2cb5a780/d33e463.gif" loading="lazy" id="d33e463" alt="Inline graphic"></span>), altitude (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/6b000ae6e3c3/d33e469.gif" loading="lazy" id="d33e469" alt="Inline graphic"></span>), and precipitation (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/58554dc275cf/d33e475.gif" loading="lazy" id="d33e475" alt="Inline graphic"></span>) are integrated into yield and resource functions to reflect real-world variability accurately.</p></section><section id="Sec6"><h4 class="pmc_sec_title">Basic assumptions</h4>
<ul class="list" style="list-style-type:disc">
<li><p id="Par19">Within any single optimization stage, decisions are made based on a static set of price and cost forecasts. However, these forecasts are dynamically updated with new information (e.g., latest market reports and weather forecasts) at the beginning of each subsequent stage.</p></li>
<li><p id="Par20">Soil and climatic conditions are considered stable within each stage but are updated between stages.</p></li>
<li><p id="Par21">Crop interaction coefficients (complementarity and substitutability) are predefined.</p></li>
<li><p id="Par22">Long-term soil effects from crop rotation are excluded from the model.</p></li>
</ul></section><section id="Sec7"><h4 class="pmc_sec_title">Symbol definitions</h4>
<ul class="list" style="list-style-type:disc">
<li><p id="Par24"><em>i</em>: Plot index (1 to <em>N</em>).</p></li>
<li><p id="Par25"><em>j</em>: Crop index (1 to <em>M</em>).</p></li>
<li><p id="Par26"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/71216a8441fb/d33e522.gif" loading="lazy" id="d33e522" alt="Inline graphic"></span>: Market price of crop <em>j</em> (CNY/ton).</p></li>
<li><p id="Par27"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/322425df37b5/d33e533.gif" loading="lazy" id="d33e533" alt="Inline graphic"></span>: Cost of crop <em>j</em> (CNY/mu).</p></li>
<li><p id="Par28"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/24c9585c654b/d33e544.gif" loading="lazy" id="d33e544" alt="Inline graphic"></span>: Area of crop <em>j</em> on plot <em>i</em> (mu).</p></li>
<li><p id="Par29"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/5b7cd53ba371/d33e558.gif" loading="lazy" id="d33e558" alt="Inline graphic"></span>: Available area on plot <em>i</em> (mu).</p></li>
<li><p id="Par30"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/fd06513ab405/d33e569.gif" loading="lazy" id="d33e569" alt="Inline graphic"></span>: The estimated regional market absorption capacity for crop <em>j</em> (ton), representing the upper limit of what can be sold without causing significant price depression, forecasted based on historical sales data.</p></li>
<li><p id="Par31"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/8f9604d49e8b/d33e580.gif" loading="lazy" id="d33e580" alt="Inline graphic"></span>: Per-unit-area composite water consumption for crop <em>j</em> (cubic meters/mu). This value integrates the irrigation requirements and effective rainfall over the crop’s entire growth cycle.</p></li>
<li><p id="Par32"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/c582730f9dbd/d33e591.gif" loading="lazy" id="d33e591" alt="Inline graphic"></span>: Total water resource limit for the entire farm (cubic meters), representing the total volume of water available from the local reservoir system for irrigation during the planning period.</p></li>
<li><p id="Par33"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/6d132553d565/d33e599.gif" loading="lazy" id="d33e599" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/fe8f269d2daa/d33e605.gif" loading="lazy" id="d33e605" alt="Inline graphic"></span>: Crop interaction coefficients. These dimensionless coefficients, ranging from [0, 1], were determined based on a combination of: (1) a systematic review of agro-ecological literature pertinent to the region, and (2) multiple rounds of consultation with local senior agronomists and agricultural extension experts. For example, a higher <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/6d132553d565/d33e611.gif" loading="lazy" id="d33e611" alt="Inline graphic"></span> value is assigned for legume–non-legume pairs, while a higher <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/fe8f269d2daa/d33e617.gif" loading="lazy" id="d33e617" alt="Inline graphic"></span> value is assigned for pairs of crops with high water demand.</p></li>
<li><p id="Par34"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/6ecf5e018c44/d33e625.gif" loading="lazy" id="d33e625" alt="Inline graphic"></span>: Minimum threshold for synergistic crop interaction effects (dimensionless score).</p></li>
<li><p id="Par35"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/103dd8abfe4e/d33e634.gif" loading="lazy" id="d33e634" alt="Inline graphic"></span>: Maximum threshold for antagonistic crop interaction effects (dimensionless score).</p></li>
</ul></section><section id="Sec8"><h4 class="pmc_sec_title">optimization objectives and decision variables</h4>
<ul class="list" style="list-style-type:disc">
<li><div class="p" id="Par37">
<strong>Objective:</strong> Maximize total revenue <em>R</em>: <table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/aa89cbfe63a2/d33e653.gif" loading="lazy" id="d33e653" alt="graphic file with name d33e653.gif"></td>
<td class="label">1</td>
</tr></table> where <em>Pen</em> is the penalty term that quantifies constraint violations (defined further in the Fitness Function section), and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/95c7c0c413ff/d33e663.gif" loading="lazy" id="d33e663" alt="Inline graphic"></span> is a large positive penalty factor. where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/6507ac88973e/d33e669.gif" loading="lazy" id="d33e669" alt="Inline graphic"></span> represents the predicted per-unit-area yield (e.g., in ton/mu) of crop <em>j</em> on plot <em>i</em>. This yield is determined by an empirical model, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/1a86360cea93/d33e682.gif" loading="lazy" id="d33e682" alt="Inline graphic"></span>, which synthesizes the influence of various plot-specific characteristics, crop-specific requirements, environmental conditions, and management inputs. For this study, a linear regression model was developed and calibrated using historical and experimental data from the study region, taking the following functional form: <table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/6094efa4c2be/d33e688.gif" loading="lazy" id="d33e688" alt="graphic file with name d33e688.gif"></td>
<td class="label">2</td>
</tr></table> The input variables (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/6abcbf4cdd9a/d33e695.gif" loading="lazy" id="d33e695" alt="Inline graphic"></span>) for this model are defined as:<ul class="list" style="list-style-type:disc">
<li><div class="p" id="Par38">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/3a479c4c266f/d33e704.gif" loading="lazy" id="d33e704" alt="Inline graphic"></span>: Soil Quality Index for plot <em>i</em>. This is a composite, dimensionless index, constructed following the guidelines of the Food and Agriculture Organization (FAO) framework for soil quality assessment. It integrates several key soil physical and chemical indicators, including organic matter content, texture (sand, silt, clay percentages), pH, and available N, P, K. These indicators are normalized and combined through a weighted average to produce a final score ranging from 0 to 1, where 1 represents optimal soil health. The index is numerically scaled (e.g., from 0 to 0.01) to fit the coefficient scale of the regression model.</div></li>
<li><div class="p" id="Par39">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/5e298fda0b97/d33e715.gif" loading="lazy" id="d33e715" alt="Inline graphic"></span>: Effective Growing Degree Days for crop <em>j</em> on plot <em>i</em> (measured in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/92597a82940f/d33e727.gif" loading="lazy" id="d33e727" alt="Inline graphic"></span>). This is calculated based on daily mean temperatures relative to the crop’s specific base temperature and any upper temperature threshold, accumulated over the relevant growing period within stage <em>t</em>.</div></li>
<li><div class="p" id="Par40">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/e0464f510070/d33e738.gif" loading="lazy" id="d33e738" alt="Inline graphic"></span>: Total effective water available to crop <em>j</em> on plot <em>i</em> (e.g., sum of effective rainfall and applied irrigation, measured in mm) during critical growth sub-periods within a given stage.</div></li>
<li><div class="p" id="Par41">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/eaa7803d086c/d33e752.gif" loading="lazy" id="d33e752" alt="Inline graphic"></span>: A dimensionless slope impact factor for plot <em>i</em>. This is derived from the raw slope percentage (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/a53467b69347/d33e761.gif" loading="lazy" id="d33e761" alt="Inline graphic"></span>) using a non-linear transformation designed to emphasize the adverse impact of steeper slopes on yield, for instance, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/791c31ba6f60/d33e767.gif" loading="lazy" id="d33e767" alt="Inline graphic"></span> (where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/e61466bcf921/d33e773.gif" loading="lazy" id="d33e773" alt="Inline graphic"></span> is a reference slope and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/43a66b858d00/d33e779.gif" loading="lazy" id="d33e779" alt="Inline graphic"></span>, e.g., <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/23b7f96fd323/d33e786.gif" loading="lazy" id="d33e786" alt="Inline graphic"></span>), or a similar function. This transformation typically results in very small positive values (e.g., in the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/ebad49010a6f/d33e792.gif" loading="lazy" id="d33e792" alt="Inline graphic"></span> to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/9282db406ceb/d33e798.gif" loading="lazy" id="d33e798" alt="Inline graphic"></span> range for common agricultural slopes), explaining the large magnitude of the coefficient <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/e55810218810/d33e804.gif" loading="lazy" id="d33e804" alt="Inline graphic"></span> when combined with the negative sign in Eq. <a href="#Equ2" class="usa-link">2</a>.</div></li>
<li><div class="p" id="Par42">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/97faaafb87b5/d33e815.gif" loading="lazy" id="d33e815" alt="Inline graphic"></span>: An altitude suitability factor for crop <em>j</em> at the altitude of plot <em>i</em> (raw altitude measured in meters). This factor is constructed as a normalized score (e.g., scaled 0 to 1, where 1 represents optimal altitude for crop <em>j</em>) derived from the raw altitude, potentially modeling an optimal range or a non-linear response based on literature or agronomical expertise for each crop.</div></li>
<li><div class="p" id="Par43">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/e2bc6c84b438/d33e832.gif" loading="lazy" id="d33e832" alt="Inline graphic"></span>: A composite nutrient availability score for crop <em>j</em> on plot <em>i</em>. To align with standard agronomic practices, crop yield is measured in ton/mu throughout this paper, whereas nutrient inputs, such as the N, P, K levels used to calculate this score, are specified in kg/mu. This score (e.g., dimensionless, scaled 0 to 10) is based on the levels of these applied fertilizer nutrients relative to the specific crop’s requirements, and may also incorporate inherent soil nutrient levels if available.</div></li>
</ul> The parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/86174ccd2e25/d33e846.gif" loading="lazy" id="d33e846" alt="Inline graphic"></span> are the coefficients obtained from the multiple linear regression analysis of historical yield data against the aforementioned input variables. The determined values of these coefficients and their detailed interpretations are presented in the Results section (see Section "<a href="#Sec18" class="usa-link">H-SAGA optimization performance and outcomes</a>", Table <a href="#Tab3" class="usa-link">3</a>). The precise definition and scaling of these input variables and the underlying assumptions of the linear additive model (Eq. <a href="#Equ2" class="usa-link">2</a>) are crucial for the model’s predictive accuracy. This calibrated empirical model forms the basis for yield predictions within the H-SAGA optimization process.</div></li>
<li><div class="p" id="Par44">
<strong>Constraints:</strong><ol class="list" style="list-style-type:decimal">
<li><div class="p" id="Par45">
<strong>Land:</strong>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/244c4876ce0d/d33e873.gif" loading="lazy" id="d33e873" alt="Inline graphic"></span>.</div></li>
<li><div class="p" id="Par46">
<strong>Demand:</strong>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/79c199d48dc4/d33e886.gif" loading="lazy" id="d33e886" alt="Inline graphic"></span>.</div></li>
<li><div class="p" id="Par47">
<strong>Resource:</strong>
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/e186d34119ba/d33e899.gif" loading="lazy" id="d33e899" alt="Inline graphic"></span>.</div></li>
<li><div class="p" id="Par48">
<strong>Crop Interactions (for intercropping scenarios):</strong> These constraints model the synergistic (beneficial) or antagonistic (detrimental) effects when specific pairs of crops (<em>j</em> and <em>k</em>) are grown simultaneously on the same plot <em>i</em> (i.e., intercropping). It is assumed that the total area planted on plot <em>i</em> does not exceed <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/5b7cd53ba371/d33e924.gif" loading="lazy" id="d33e924" alt="Inline graphic"></span>, allowing for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/b68bd9c41346/d33e931.gif" loading="lazy" id="d33e931" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/b5da6e533fe5/d33e937.gif" loading="lazy" id="d33e937" alt="Inline graphic"></span> simultaneously if <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/ad496eff2124/d33e943.gif" loading="lazy" id="d33e943" alt="Inline graphic"></span>.<ul class="list" style="list-style-type:square">
<li><div class="p" id="Par49">Synergistic interactions (e.g., nitrogen fixation, pest deterrence): For selected crop pairs (<em>j</em>, <em>k</em>) known to be mutually beneficial, a minimum level of positive interaction effect is enforced: <table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/82561f02befe/d33e959.gif" loading="lazy" id="d33e959" alt="graphic file with name d33e959.gif"></td>
<td class="label">3</td>
</tr></table> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/6d132553d565/d33e966.gif" loading="lazy" id="d33e966" alt="Inline graphic"></span> is a positive coefficient quantifying the synergistic effect per unit-area-product of crops <em>j</em> and <em>k</em> (e.g., units of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/d83fabd54ba7/d33e979.gif" loading="lazy" id="d33e979" alt="Inline graphic"></span> if <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/6ecf5e018c44/d33e985.gif" loading="lazy" id="d33e985" alt="Inline graphic"></span> is dimensionless), and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/6ecf5e018c44/d33e991.gif" loading="lazy" id="d33e991" alt="Inline graphic"></span> is a predefined minimum acceptable dimensionless score for the synergistic benefit.</div></li>
<li><div class="p" id="Par50">Antagonistic interactions (e.g., allelopathy, competition for specific resources): For selected crop pairs (<em>j</em>, <em>k</em>) known to have negative interactions, the detrimental effect is limited: <table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/17ced2c4cd5a/d33e1006.gif" loading="lazy" id="d33e1006" alt="graphic file with name d33e1006.gif"></td>
<td class="label">4</td>
</tr></table> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/fe8f269d2daa/d33e1013.gif" loading="lazy" id="d33e1013" alt="Inline graphic"></span> is a positive coefficient quantifying the antagonistic effect per unit-area-product (e.g., units of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/d83fabd54ba7/d33e1019.gif" loading="lazy" id="d33e1019" alt="Inline graphic"></span> if <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/103dd8abfe4e/d33e1026.gif" loading="lazy" id="d33e1026" alt="Inline graphic"></span> is dimensionless), and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq53"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/103dd8abfe4e/d33e1032.gif" loading="lazy" id="d33e1032" alt="Inline graphic"></span> is a predefined maximum tolerable dimensionless score for the antagonistic effect.</div></li>
</ul> The selection of interacting pairs and the values for <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq54"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/6d132553d565/d33e1039.gif" loading="lazy" id="d33e1039" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq55"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/fe8f269d2daa/d33e1045.gif" loading="lazy" id="d33e1045" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq56"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/6ecf5e018c44/d33e1052.gif" loading="lazy" id="d33e1052" alt="Inline graphic"></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq57"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/103dd8abfe4e/d33e1058.gif" loading="lazy" id="d33e1058" alt="Inline graphic"></span> are based on agronomic knowledge and experimental data regarding local intercropping practices. If intercropping is not considered for a plot or specific crop pair, these constraints are omitted or the respective coefficients <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq58"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/60bc1afbf298/d33e1064.gif" loading="lazy" id="d33e1064" alt="Inline graphic"></span> are set to zero.</div></li>
</ol>
</div></li>
</ul>
<p id="Par36">Constraints are adjusted per stage, with outcomes from one stage informing the next, enabling adaptive planning.</p>
<section class="tw xbox font-sm" id="Tab3"><h5 class="obj_head">Table 3.</h5>
<div class="caption p"><p>Determined coefficients for the empirical yield model (Eq. <a href="#Equ2" class="usa-link">2</a>) and their interpretations. All yield-related coefficients are in units of tons per mu.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Parameter</th>
<th align="left" colspan="1" rowspan="1">Optimized Value</th>
<th align="left" colspan="1" rowspan="1">Interpretation</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq81"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/37a1d78ddbdb/d33e1778.gif" loading="lazy" id="d33e1778" alt="Inline graphic"></span> (Intercept)</td>
<td align="left" colspan="1" rowspan="1">0.9110</td>
<td align="left" colspan="1" rowspan="1">Base yield (ton/mu) under reference conditions.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq82"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/8d7277f89848/d33e1791.gif" loading="lazy" id="d33e1791" alt="Inline graphic"></span> (SQI)</td>
<td align="left" colspan="1" rowspan="1">31.430</td>
<td align="left" colspan="1" rowspan="1">Yield change per unit of Soil Quality Index.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq83"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/920043c8704e/d33e1804.gif" loading="lazy" id="d33e1804" alt="Inline graphic"></span> (GDD)</td>
<td align="left" colspan="1" rowspan="1">0.0019</td>
<td align="left" colspan="1" rowspan="1">Marginal yield increase (ton/mu) per unit of GDD.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq84"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/9860102e4edb/d33e1817.gif" loading="lazy" id="d33e1817" alt="Inline graphic"></span> (Water)</td>
<td align="left" colspan="1" rowspan="1">0.0014</td>
<td align="left" colspan="1" rowspan="1">Marginal yield increase (ton/mu) per mm of water.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq85"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/e55810218810/d33e1830.gif" loading="lazy" id="d33e1830" alt="Inline graphic"></span> (Slope)</td>
<td align="left" colspan="1" rowspan="1">42.868</td>
<td align="left" colspan="1" rowspan="1">Quantifies yield reduction per unit of slope factor.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq86"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/4a3048763ca6/d33e1843.gif" loading="lazy" id="d33e1843" alt="Inline graphic"></span> (Altitude)</td>
<td align="left" colspan="1" rowspan="1">0.4604</td>
<td align="left" colspan="1" rowspan="1">Yield change per unit of altitude suitability factor.</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq87"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/2857b330977d/d33e1856.gif" loading="lazy" id="d33e1856" alt="Inline graphic"></span> (Nutrient)</td>
<td align="left" colspan="1" rowspan="1">0.0408</td>
<td align="left" colspan="1" rowspan="1">Marginal yield increase (ton/mu) per unit of nutrient score.</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section></section><section id="Sec9"><h3 class="pmc_sec_title">Hybrid Simulated Annealing Genetic Algorithm (H-SAGA)</h3>
<p id="Par51">The Hybrid Simulated Annealing Genetic Algorithm (H-SAGA)<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> addresses the multi-stage optimization problem by merging Simulated Annealing (SA) for global exploration with Genetic Algorithms (GA) for local refinement<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. This hybrid approach excels in complex, nonlinear agricultural optimization tasks.</p>
<section id="Sec10"><h4 class="pmc_sec_title">Algorithm principles</h4>
<ul class="list" style="list-style-type:disc">
<li><p id="Par53"><strong>SA:</strong> Employs a temperature-controlled search to escape local optima, reducing temperature iteratively to converge on a global optimum<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>.</p></li>
<li><p id="Par54"><strong>GA:</strong> Uses selection, crossover, and mutation for solution space exploration, though it may stagnate in later stages<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>.</p></li>
<li><p id="Par55"><strong>H-SAGA Strategy:</strong> Starts with SA for broad exploration, then shifts to GA for precise optimization, ensuring robust solutions.</p></li>
</ul></section><section id="Sec11"><h4 class="pmc_sec_title">Fitness function</h4>
<p id="Par56">The fitness function balances revenue and constraint adherence. This study transforms the constrained optimization problem into an unconstrained one using a penalty function method, a mature and widely-used technique for handling complex constraints in metaheuristic algorithms. The function is defined as:</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/aa89cbfe63a2/d33e1125.gif" loading="lazy" id="d33e1125" alt="graphic file with name d33e1125.gif"></td>
<td class="label">5</td>
</tr></table>
<p>where <em>Pen</em> is the penalty term that quantifies constraint violations, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq59"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/95c7c0c413ff/d33e1135.gif" loading="lazy" id="d33e1135" alt="Inline graphic"></span> is a large positive penalty factor. This approach guides the search process towards the feasible region by penalizing solutions that violate the constraints.</p></section><section id="Sec12"><h4 class="pmc_sec_title">Integration of neural network prediction model</h4>
<p id="Par57">A three-layer neural network enhances H-SAGA with real-time revenue forecasts using historical data (e.g., climate, yields, prices)<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par58"><strong>Input:</strong> Climatic and economic variables.</p></li>
<li><p id="Par59"><strong>Hidden Layers:</strong> 64, 32, and 16 neurons with ReLU activation.</p></li>
<li><p id="Par60"><strong>Output:</strong> The primary output of the neural network for each stage <em>t</em> is a vector of predicted market prices, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq60"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/559075a77916/d33e1171.gif" loading="lazy" id="d33e1171" alt="Inline graphic"></span>, for each relevant crop <em>j</em>. These predictions represent the expected selling price per unit (e.g., CNY/ton) of crop <em>j</em> if harvested or marketed corresponding to the decisions made for stage <em>t</em>. The network is trained on historical data linking past climatic and market conditions to observed crop prices.</p></li>
<li><p id="Par61"><strong>Fitness function refinement:</strong> The stage-specific predicted prices <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq61"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/559075a77916/d33e1191.gif" loading="lazy" id="d33e1191" alt="Inline graphic"></span> from the neural network are directly used to dynamically update the revenue calculation within the H-SAGA’s fitness function (Eq. <a href="#Equ5" class="usa-link">5</a>) for the current optimization stage <em>t</em>. Specifically, the baseline price term <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq62"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/71216a8441fb/d33e1203.gif" loading="lazy" id="d33e1203" alt="Inline graphic"></span> in the revenue component <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq63"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/c2333639ec60/d33e1209.gif" loading="lazy" id="d33e1209" alt="Inline graphic"></span> of Eq. <a href="#Equ5" class="usa-link">5</a> is replaced by the neural network’s predicted price <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq64"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/559075a77916/d33e1219.gif" loading="lazy" id="d33e1219" alt="Inline graphic"></span>. Thus, the revenue for stage <em>t</em> is calculated using these dynamic, Neural-network-predicted prices, while the physical yield <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq65"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/6507ac88973e/d33e1228.gif" loading="lazy" id="d33e1228" alt="Inline graphic"></span> is still derived from the empirical model (Eq. <a href="#Equ2" class="usa-link">2</a>). This integration ensures that the H-SAGA’s optimization at each stage is guided by the most current, data-driven forecasts of economic conditions, allowing the planting strategies to proactively adapt to anticipated changes in crop profitability(Fig <a href="#Fig4" class="usa-link">4</a>).</p></li>
</ul>
<figure class="fig xbox font-sm" id="Fig4"><h5 class="obj_head">Fig. 4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/6ff67b8b860e/41598_2025_14188_Fig4_HTML.jpg" loading="lazy" id="MO4" height="316" width="624" alt="Fig. 4"></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Structure of the integrated neural network prediction mModel.</p></figcaption></figure></section><section id="Sec13"><h4 class="pmc_sec_title">Key parameter visualization</h4>
<p id="Par62">Parameters like initial temperature and population size are visualized to show their impact on performance (Figs. <a href="#Fig5" class="usa-link">5</a>,<a href="#Fig6" class="usa-link">6</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h5 class="obj_head">Fig. 5.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12331943_41598_2025_14188_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/0659baaf3e61/41598_2025_14188_Fig5_HTML.jpg" loading="lazy" id="MO5" height="542" width="739" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Key Parameters of H-SAGA and their Influence.</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig6"><h5 class="obj_head">Fig. 6.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/b357629dcceb/41598_2025_14188_Fig6_HTML.jpg" loading="lazy" id="MO6" height="879" width="624" alt="Fig. 6"></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Flowchart of the H-SAGA algorithm.</p></figcaption></figure></section><section id="Sec14"><h4 class="pmc_sec_title">H-SAGA process and steps</h4>
<ol class="list" style="list-style-type:decimal">
<li><p id="Par64"><strong>Initialization:</strong> Set population and temperature.</p></li>
<li><p id="Par65"><strong>SA Phase:</strong> Explore solutions, reduce temperature.</p></li>
<li><p id="Par66"><strong>GA Phase:</strong> Refine via selection, crossover, mutation.</p></li>
<li><p id="Par67"><strong>Neural Adjustment:</strong> Update fitness with predictions.</p></li>
<li><p id="Par68"><strong>Convergence:</strong> Stop at iteration limit or fitness threshold.</p></li>
</ol></section></section><section id="Sec15"><h3 class="pmc_sec_title">Experimental setup and data</h3>
<p id="Par69">The experiments and projections in this study are grounded in a comprehensive analysis of agricultural and climatic data from a representative mountainous region in North China. This area is characterized by its cold climate and predominantly single-season cropping systems<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>.</p>
<p id="Par70"><strong>Crop portfolio and data sources:</strong> The study considered a portfolio of crops common to the North China region, categorized as <strong>major food crops</strong> (e.g., High-yield Corn, Drought-resistant Wheat, Potato) and cash crops (e.g., Heirloom Tomato, Green Soybean, Winter Rapeseed) for market returns. The empirical data underpinning our models were aggregated from multiple authoritative sources. Historical time-series data on crop sowing areas, yields, and market prices for the period 2000-2024 were primarily compiled from the <em>China Statistical Yearbook</em> and the respective provincial statistical yearbooks of Beijing, Tianjin, Hebei, Shanxi, and Inner Mongolia<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. The geospatial data used to construct the study area maps (Fig. <a href="#Fig1" class="usa-link">1</a>) and as inputs for the model were aggregated from multiple authoritative, open-access sources. Specifically, land use/land cover (LULC) and cropland change data were derived from the China Land Cover Dataset (CLCD)<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Topographic data, used for generating hillshade and contour lines, were sourced from the Shuttle Radar Topography Mission (SRTM) dataset<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Major river networks were extracted from the HydroSHEDS dataset<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Administrative boundaries for the location map were obtained from the GEE data catalog, primarily the U.S. Department of State Large Scale International Boundary (LSIB) dataset. The local plot-level records from 7,290 mu (1,201 acres) across 34 plots were used for model calibration and validation against these regional datasets. This multi-source data integration provides a robust basis for capturing historical trends and spatial heterogeneity.</p>
<p id="Par71">To accurately model the temporal constraints and resource demands within the multi-stage framework, the typical growth cycle duration for each crop under local conditions was established, as detailed in Table <a href="#Tab1" class="usa-link">1</a>. This information is critical for calculating stage-specific parameters such as Effective Growing Degree Days (GDD) and scheduling water allocation, ensuring that the optimization aligns with the phenological timeline of each potential crop.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Portfolio of studied crops and their typical growth cycle durations.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Category</th>
<th align="left" colspan="1" rowspan="1">Crop Species</th>
<th align="left" colspan="1" rowspan="1">Typical Growth Cycle (days)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Major Food Crop</td>
<td align="left" colspan="1" rowspan="1">High-yield Corn</td>
<td align="left" colspan="1" rowspan="1">90–120</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Major Food Crop</td>
<td align="left" colspan="1" rowspan="1">Drought-resistant Wheat</td>
<td align="left" colspan="1" rowspan="1">210–240</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Major Food Crop</td>
<td align="left" colspan="1" rowspan="1">Potato</td>
<td align="left" colspan="1" rowspan="1">90–120</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cash Crop</td>
<td align="left" colspan="1" rowspan="1">Heirloom Tomato</td>
<td align="left" colspan="1" rowspan="1">75–95</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cash Crop</td>
<td align="left" colspan="1" rowspan="1">Green Soybean</td>
<td align="left" colspan="1" rowspan="1">100–130</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cash Crop</td>
<td align="left" colspan="1" rowspan="1">Winter Rapeseed</td>
<td align="left" colspan="1" rowspan="1">280–320</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cash Crop</td>
<td align="left" colspan="1" rowspan="1">Sweet Sorghum</td>
<td align="left" colspan="1" rowspan="1">100–120</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cash Crop</td>
<td align="left" colspan="1" rowspan="1">Legume (e.g., Black Bean)</td>
<td align="left" colspan="1" rowspan="1">90–110</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par72"><strong>Data Preprocessing:</strong> To ensure the quality and integrity of the 25-year time-series data, a rigorous preprocessing pipeline was implemented:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par73"><strong>Data cleaning</strong>: Outliers in the time-series records were identified using the Interquartile Range (IQR) method. Instead of exclusion, which could create gaps, these outliers were managed using Winsorization to cap them at a realistic percentile (e.g., 95th), preserving the data structure.</p></li>
<li><p id="Par74"><strong>Missing value imputation</strong>: To maintain the temporal integrity of the dataset, missing values within the time series (e.g., for climate or price data) were handled using time-series interpolation (linear interpolation). This method is more appropriate for long-term data than simple mean imputation as it respects the temporal sequence of the observations.</p></li>
<li><p id="Par75"><strong>Feature engineering</strong>: Coefficients representing crop complementarity and substitutability were derived based on a review of regional agronomic literature and expert consultation.</p></li>
<li><p id="Par76"><strong>Data standardization</strong>: Numerical features were standardized using Z-score normalization to mitigate scale disparities before being input into the machine learning models.</p></li>
<li><p id="Par77"><strong>Dataset partitioning</strong>: The complete historical dataset (2000-2024) was partitioned for the purpose of training and validating the predictive neural network. Data from 2000-2018 was used as the training set, data from 2019-2021 served as the validation set for hyperparameter tuning, and data from 2022-2024 was held out as the final test set to evaluate predictive performance on unseen data.</p></li>
</ul>
<p><strong>Model parameter configuration:</strong>The parameters for the H-SAGA algorithm and the integrated neural network were determined through a meticulous tuning process involving preliminary experiments and sensitivity analysis. Our goal was to strike an optimal balance between global exploration (SA phase) and local exploitation (GA phase), ensuring both rapid convergence and high-quality solutions. For instance, sensitivity analysis indicated that an initial temperature (T0) around 300 provided a sufficiently broad initial search space without excessively prolonging the initial annealing phase. For instance, sensitivity analysis indicated that an initial temperature (T0) around 300 provided a sufficiently broad initial search space without excessively prolonging the initial annealing phase. The cooling rate (<em>a</em>) was tuned within the range of [0.80, 0.99] as specified in Table <a href="#Tab2" class="usa-link">2</a>. A final value of 0.95 was selected, as it offered a balanced cooling schedule that effectively prevented premature convergence while leveraging the powerful local refinement of the subsequent GA phase. The final, optimized parameter values used for all reported experiments are detailed in Table <a href="#Tab2" class="usa-link">2</a>. To ensure the effectiveness and reproducibility of the neural network prediction model, its training process was meticulously defined. The network was trained for 200 epochs with a batch size of 32. The Adam optimizer was employed for updating network weights, with the learning rate specified in Table <a href="#Tab2" class="usa-link">2</a> (0.001). Mean Squared Error (MSE) was selected as the loss function to quantify the deviation between predicted and actual revenues, guiding the model’s optimization.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Parameter setting overview for H-SAGA experiments.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Parameter Name</th>
<th align="left" colspan="1" rowspan="1">Value Range</th>
<th align="left" colspan="1" rowspan="1">Description</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Initial Temperature <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq66"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/c1ac16dc902e/d33e1510.gif" loading="lazy" id="d33e1510" alt="Inline graphic"></span>
</td>
<td align="left" colspan="1" rowspan="1">100–500</td>
<td align="left" colspan="1" rowspan="1">Controls initial exploration extent</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Cooling Rate <em>a</em>
</td>
<td align="left" colspan="1" rowspan="1">0.8–0.99</td>
<td align="left" colspan="1" rowspan="1">Determines temperature decrease rate</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Population Size</td>
<td align="left" colspan="1" rowspan="1">50–200</td>
<td align="left" colspan="1" rowspan="1">Defines GA population size</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Crossover Probability</td>
<td align="left" colspan="1" rowspan="1">0.6–0.9</td>
<td align="left" colspan="1" rowspan="1">Influences crossover frequency in GA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Mutation Probability</td>
<td align="left" colspan="1" rowspan="1">0.01–0.05</td>
<td align="left" colspan="1" rowspan="1">Sets mutation rate in GA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Neural Network Learning Rate</td>
<td align="left" colspan="1" rowspan="1">0.001 (fixed value)</td>
<td align="left" colspan="1" rowspan="1">Adjusts weight update rate in NN</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Number of Neurons in Hidden Layers</td>
<td align="left" colspan="1" rowspan="1">64, 32, 16</td>
<td align="left" colspan="1" rowspan="1">Specifies neuron counts in hidden layers</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par78"><strong>Baseline algorithm configuration:</strong> To comprehensively evaluate the performance of H-SAGA, four representative metaheuristic algorithms were selected as benchmarks. These algorithms were chosen because they represent different and well-established families of optimization techniques: Genetic Algorithm (GA) is a classic example of evolutionary algorithms, Simulated Annealing (SA) represents trajectory-based local search methods, while Particle Swarm Optimization (PSO) and Ant Colony Optimization (ACO) are prominent examples of swarm intelligence. Comparing H-SAGA against these diverse algorithms provides a robust validation of its effectiveness for this agricultural optimization problem. For a fair comparison, the parameters for these baseline algorithms were meticulously chosen based on commonly accepted values in the literature and fine-tuned through preliminary experiments to suit the problem characteristics. The specific configurations were as follows:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par79"><strong>Genetic Algorithm (GA):</strong> Population size: 100 individuals; Selection: Tournament selection (tournament size of 3); Crossover: Two-point crossover (probability of 0.85); Mutation: Bit-flip mutation (probability of 0.02 per bit); Maximum generations: 500.</p></li>
<li><p id="Par80"><strong>Simulated Annealing (SA):</strong> Initial temperature: 1000 units; Cooling schedule: Geometric cooling (cooling rate <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq67"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/458471d43ada/d33e1579.gif" loading="lazy" id="d33e1579" alt="Inline graphic"></span>); Iterations per temperature level (Markov chain length): 150; Final stopping temperature: 0.01 units.</p></li>
<li><p id="Par81"><strong>Particle Swarm Optimization (PSO):</strong> Swarm size: 50 particles; Inertia weight <em>w</em>: Linearly decreased from 0.9 to 0.4; Cognitive coefficient (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq68"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/fdbaf58ab703/d33e1593.gif" loading="lazy" id="d33e1593" alt="Inline graphic"></span>): 2.05; Social coefficient (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq69"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/6fd362962232/d33e1599.gif" loading="lazy" id="d33e1599" alt="Inline graphic"></span>): 2.05; Maximum iterations: 500. Velocity clamping was applied.</p></li>
<li><p id="Par82"><strong>Ant Colony Optimization (ACO):</strong> Number of ants: 50; Pheromone influence factor (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq70"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/1eef2ddce2aa/d33e1610.gif" loading="lazy" id="d33e1610" alt="Inline graphic"></span>): 1.0; Heuristic influence factor (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq71"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/d0a508b22681/d33e1616.gif" loading="lazy" id="d33e1616" alt="Inline graphic"></span>): 3.0; Pheromone evaporation rate (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq72"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/11f63f35cc7d/d33e1622.gif" loading="lazy" id="d33e1622" alt="Inline graphic"></span>): 0.1; Pheromone deposit factor (<em>Q</em>): 100 (scaled by solution quality); Maximum iterations: 200.</p></li>
</ul>
<p>All algorithms, including H-SAGA and the baselines, were executed under identical computational environments (Intel Core i7-9700K CPU @ 3.60GHz, 32 GB RAM, Windows 10, Python 3.8 with NumPy 1.19 and SciPy 1.5.) and utilized consistent stopping criteria. Specifically, execution for each algorithm concluded when one of the following conditions was met: (1) reaching the maximum iteration limit as defined in its respective parameter configuration; or (2) when the improvement in the best-found solution’s fitness value was less than a predefined tolerance (set to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq73"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/e68774ab1c48/d33e1632.gif" loading="lazy" id="d33e1632" alt="Inline graphic"></span>) over 50 consecutive iterations. Furthermore, an overriding maximum limit of 200,000 function evaluations was applied to all algorithms to ensure comparable computational effort and provide a robust basis for performance evaluation.</p></section></section><section id="Sec16"><h2 class="pmc_sec_title">Results</h2>
<p id="Par83">This section presents the results obtained from applying the proposed H-SAGA optimization model. We validate the data characteristics, evaluate the performance of H-SAGA including its convergence and optimization outcomes, compare it against baseline algorithms, and assess the robustness of the generated solutions.</p>
<section id="Sec17"><h3 class="pmc_sec_title">Data characteristics and validation</h3>
<p id="Par84">Analysis of the preprocessed data revealed important relationships between key agricultural variables.</p>
<p id="Par85"><strong>Data distribution and correlation analysis:</strong> To understand the underlying characteristics of the key economic variables used in the model, a comprehensive pair-plot analysis was conducted (Fig. <a href="#Fig7" class="usa-link">7</a>). The diagonal panels display the histogram distribution for each primary variable: Yield (ton/mu), Planting Cost (CNY/mu), and Selling Price (CNY/ton). The off-diagonal panels present scatter plots that visualize the bivariate relationships between each pair of variables.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12331943_41598_2025_14188_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/3a3412d0684d/41598_2025_14188_Fig7_HTML.jpg" loading="lazy" id="MO7" height="741" width="739" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Pair-plot of key economic variables. The diagonal plots show the histogram distribution for each variable, while the off-diagonal plots show the scatter-plot relationship between pairs of variables. This provides a comprehensive view of both the individual data distributions and their bivariate correlations.</p></figcaption></figure><p id="Par86">This integrated visualization reveals several key insights. A moderate positive correlation can be observed between higher yields and increased planting costs, an agronomically intuitive relationship reflecting that greater inputs are typically required for higher productivity. Conversely, the relationship between yield and selling price appears weak and widely dispersed, highlighting the complex market dynamics that our optimization model must navigate. This figure provides a clear view of the foundational data characteristics before evaluating the model’s performance.</p>
<p id="Par87"><strong>Parameter sensitivity:</strong> The influence of terrain and climate parameters on economic outcomes was assessed via sensitivity analysis, visualized using a Sankey diagram (Fig. <a href="#Fig8" class="usa-link">8</a>). This analysis quantified critical nonlinear interactions: slope variations explained 42% of water loss volatility (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq74"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/ad1a63394719/d33e1669.gif" loading="lazy" id="d33e1669" alt="Inline graphic"></span>), altitude-driven growing season changes accounted for 48% of yield fluctuations, and precipitation patterns determined 56% of irrigation efficiency variance. These <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq75"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/7d1802cbfe3a/d33e1675.gif" loading="lazy" id="d33e1675" alt="Inline graphic"></span> values were derived from separate univariate regression analyses performed on the aggregated simulation dataset, where each respective environmental factor (e.g., slope variation) was treated as the independent variable and the corresponding agricultural outcome (e.g., water loss volatility) as the dependent variable. These findings underscore the model’s capacity to capture the cascading impacts of environmental factors, providing a basis for adaptive optimization.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12331943_41598_2025_14188_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/6df042cfee99/41598_2025_14188_Fig8_HTML.jpg" loading="lazy" id="MO8" height="426" width="739" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Parameter sensitivity analysis using a Sankey diagram, visualizing the influence of terrain and climate parameters on agricultural economic outcomes.</p></figcaption></figure></section><section id="Sec18"><h3 class="pmc_sec_title">H-SAGA optimization performance and outcomes</h3>
<p id="Par88">The H-SAGA algorithm was applied to optimize the planting strategy based on the defined objectives and constraints.</p>
<p id="Par89"><strong>Convergence behavior:</strong> The iterative optimization process of H-SAGA was compared against standard Particle Swarm Optimization (PSO) and Ant Colony Optimization (ACO). Figure <a href="#Fig9" class="usa-link">9</a> shows the convergence curves, illustrating the fitness value improvement over iterations. H-SAGA achieved a stable and superior fitness value after approximately 460 iterations, demonstrating faster convergence and higher precision compared to PSO and ACO under the tested conditions.</p>
<figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12331943_41598_2025_14188_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/3d38285d53bc/41598_2025_14188_Fig9_HTML.jpg" loading="lazy" id="MO9" height="521" width="799" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Model Iteration Process: Comparison of convergence behavior between H-SAGA, PSO, and ACO.</p></figcaption></figure><p id="Par90"><strong>Optimized parameters and model fit:</strong> The overall optimization framework relies on an empirical model (Eq. <a href="#Equ2" class="usa-link">2</a>) for predicting per-unit-area yield <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq76"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/6507ac88973e/d33e1713.gif" loading="lazy" id="d33e1713" alt="Inline graphic"></span>, which is a critical component in the revenue calculation within the objective function. The coefficients (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq77"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/37a1d78ddbdb/d33e1719.gif" loading="lazy" id="d33e1719" alt="Inline graphic"></span> to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq78"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/2857b330977d/d33e1725.gif" loading="lazy" id="d33e1725" alt="Inline graphic"></span>) of this empirical yield model were determined by fitting the model to historical and experimental data. These determined coefficient values are presented in Table <a href="#Tab3" class="usa-link">3</a>. The effectiveness of the empirical model, utilizing these specific coefficients, in capturing the relationship between various input factors and observed yield outcomes was then evaluated. Figure <a href="#Fig10" class="usa-link">10</a> presents the model fitting curve, showing a close alignment between the model’s predicted values (red line) and the actual observed values (blue line). Quantitative metrics further confirmed the goodness of fit (Table <a href="#Tab4" class="usa-link">4</a>), with a high Coefficient of Determination (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq79"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/1e71a7d39c3a/d33e1741.gif" loading="lazy" id="d33e1741" alt="Inline graphic"></span>) and Correlation Coefficient (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq80"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/b54acec91f48/d33e1747.gif" loading="lazy" id="d33e1747" alt="Inline graphic"></span>). This indicates that the calibrated empirical model effectively explains a significant portion of the variance in the observed data, thereby providing a reliable basis for the H-SAGA’s planting decisions.</p>
<figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12331943_41598_2025_14188_Fig10_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/ecbccacdd3f7/41598_2025_14188_Fig10_HTML.jpg" loading="lazy" id="MO10" height="475" width="739" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The fitting effect of the yield per unit area predicted by the empirical yield model (red line) and the actual observed yield per unit area (blue line).</p></figcaption></figure><section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Quantitative metrics for model Fit assessment.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Quantitative Metric</th>
<th align="left" colspan="1" rowspan="1">Value</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Sum of Squared Errors (SSE)</td>
<td align="left" colspan="1" rowspan="1">1,577,941,328.11</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Coefficient of Determination (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq88"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/7d1802cbfe3a/d33e1900.gif" loading="lazy" id="d33e1900" alt="Inline graphic"></span>)</td>
<td align="left" colspan="1" rowspan="1">0.828</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Correlation Coefficient (<em>R</em>)</td>
<td align="left" colspan="1" rowspan="1">0.91</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par92"><strong>Optimized planting strategy and rationale:</strong> The optimization process culminates in a detailed planting strategy that tailors crop selection to the specific characteristics of each of the 34 plots. Figure <a href="#Fig11" class="usa-link">11</a> visualizes the optimized yield distribution as a clustered heatmap, revealing distinct patterns of land-use suitability. The vertical dendrogram clusters plots with similar yield potentials, while the horizontal dendrogram groups crops with comparable performance across the landscape. Notably, a distinct cluster of plots (from Plot 2 to Plot 12) demonstrates particularly high suitability for Heirloom Tomato and High-yield Corn. This suggests these plots possess superior soil quality, water access, or other favorable conditions. Conversely, most other plots are allocated to a diversified mix of crops like Drought-resistant wheat and Sweet Sorghum, which exhibit moderate but stable yields (indicated by the dark blue and gray cells) across a wider range of land types. This spatial differentiation is a key outcome of the optimization, ensuring that land use intensity is matched with land capability to maximize overall profitability within a single planning horizon.</p>
<figure class="fig xbox font-sm" id="Fig11"><h4 class="obj_head">Fig. 11.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12331943_41598_2025_14188_Fig11_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/7670e8ea2dec/41598_2025_14188_Fig11_HTML.jpg" loading="lazy" id="MO11" height="799" width="799" alt="Fig. 11"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Clustered heatmap of optimized yield (ton) per crop and plot. This visualization illustrates the optimal spatial allocation strategy for a single representative planning cycle (i.e., one year), based on land-use suitability. It is a snapshot of an annual decision, not a static multi-year rotation plan.</p></figcaption></figure><p id="Par93">Figure <a href="#Fig12" class="usa-link">12</a> complements the spatial plan by projecting the total yield composition for the eight primary optimized crops from 2025 to 2030. These multi-panel bar charts illustrate the model’s dynamic, year-on-year adjustments to the planting portfolio in response to fluctuating market and climate forecasts. For instance, in 2025 and 2026, Drought-resistant Wheat and Green Soybean are projected to have the highest yields, likely reflecting a strategy to ensure stable staple production under anticipated resource constraints. In contrast, by 2030, a more balanced portfolio emerges, with significant yields from High-yield Corn and Winter Rapeseed, indicating a shift towards higher-value crops as conditions permit. This temporal variation underscores the adaptive nature of the H-SAGA framework, which does not generate a static, multi-year rotation but rather a responsive strategy that re-optimizes the crop mix annually to maintain both high productivity and economic resilience.</p>
<figure class="fig xbox font-sm" id="Fig12"><h4 class="obj_head">Fig. 12.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12331943_41598_2025_14188_Fig12_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/adbd75bfb72f/41598_2025_14188_Fig12_HTML.jpg" loading="lazy" id="MO12" height="363" width="799" alt="Fig. 12"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig12/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Projected crop yield composition from the optimized planting strategy (2025-2030), illustrating the stability of the production mix.</p></figcaption></figure><p id="Par94">To provide a more granular view of the decision-making output, Table <a href="#Tab5" class="usa-link">5</a> presents a sample of the optimized area allocation (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq89"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/24c9585c654b/d33e1942.gif" loading="lazy" id="d33e1942" alt="Inline graphic"></span>, in mu) for two representative plots identified from the heatmap analysis: Plot 5, a high-potential plot primarily dedicated to a high-value cash crop, and Plot 21, a plot managed with a diversified strategy to balance risk and resource use. This detailed breakdown of the core decision variables demonstrates how the H-SAGA model translates its optimization logic into a concrete and actionable planting scheme at the individual plot level. The total allocated area for each plot respects its physical land constraint (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq90"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/5b7cd53ba371/d33e1948.gif" loading="lazy" id="d33e1948" alt="Inline graphic"></span>).</p>
<section class="tw xbox font-sm" id="Tab5"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Sample of optimized planting area allocation (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq91"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/24c9585c654b/d33e1960.gif" loading="lazy" id="d33e1960" alt="Inline graphic"></span>) for representative plots.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="1" rowspan="1"></th>
<th align="left" colspan="2" rowspan="1">Allocated Area (mu)</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Crop Species</th>
<th align="left" colspan="1" rowspan="1">Category</th>
<th align="left" colspan="1" rowspan="1">Plot 5 (Total Area: 50 mu)</th>
<th align="left" colspan="1" rowspan="1">Plot 21 (Total Area: 35 mu)</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">High-yield Corn</td>
<td align="left" colspan="1" rowspan="1">Major Food Crop</td>
<td align="left" colspan="1" rowspan="1">0.0</td>
<td align="left" colspan="1" rowspan="1">10.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Drought-resistant Wheat</td>
<td align="left" colspan="1" rowspan="1">Major Food Crop</td>
<td align="left" colspan="1" rowspan="1">0.0</td>
<td align="left" colspan="1" rowspan="1">15.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Heirloom Tomato</td>
<td align="left" colspan="1" rowspan="1">Cash Crop</td>
<td align="left" colspan="1" rowspan="1">48.5</td>
<td align="left" colspan="1" rowspan="1">0.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Green Soybean</td>
<td align="left" colspan="1" rowspan="1">Cash Crop</td>
<td align="left" colspan="1" rowspan="1">0.0</td>
<td align="left" colspan="1" rowspan="1">8.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Winter Rapeseed</td>
<td align="left" colspan="1" rowspan="1">Cash Crop</td>
<td align="left" colspan="1" rowspan="1">1.5</td>
<td align="left" colspan="1" rowspan="1">0.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Sweet Sorghum</td>
<td align="left" colspan="1" rowspan="1">Cash Crop</td>
<td align="left" colspan="1" rowspan="1">0.0</td>
<td align="left" colspan="1" rowspan="1">1.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Total Allocated Area</strong></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"><strong>50.0</strong></td>
<td align="left" colspan="1" rowspan="1"><strong>35.0</strong></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par96"><strong>Dynamic adaptation in multi-stage optimization:</strong> To further illustrate the dynamic adaptability of the H-SAGA framework in multi-stage decision-making, a specific simulation scenario was examined. In an initial stage of the simulation (e.g., early in the planting season), based on prevailing average market prices and climate outlooks, the model might allocate Plot A to cash crop X (e.g., a high-value vegetable). However, upon entering a subsequent stage (e.g., mid-season), should one of the following situations arise: (1) new market intelligence indicates a significant drop in the forecasted price of crop X due to predicted oversupply; or (2) the latest mid-term weather forecast suggests a high probability of severe drought in the coming weeks, drastically increasing the water demand or reducing the expected yield of crop X. In such cases, the ’monitoring’ module captures these changes. The neural network within the ’modeling’ module then adjusts its revenue predictions for crop X across various plots based on this updated information. Consequently, the H-SAGA algorithm re-optimizes under a revised fitness function that reflects these updated revenue expectations and potential resource constraints. As a result, in the new ’decision’ phase, Plot A might be reallocated to an alternative crop Y (e.g., a specific herb or a type of pulse) that is more drought-tolerant and currently offers better expected economic returns. This ability to adjust planting plans based on stage-wise feedback is a core strength of the multi-stage optimization framework, significantly enhancing the resilience of agricultural production to market and environmental uncertainties.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Comparative performance analysis</h3>
<p id="Par97">To evaluate the performance of H-SAGA, it was benchmarked against several baseline algorithms, including Genetic Algorithm (GA), Simulated Annealing (SA), Particle Swarm Optimization (PSO), and Ant Colony Optimization (ACO). The performance evaluation was conducted using multiple key performance indicators (KPIs) under different simulated climatic conditions.</p>
<p id="Par98"><strong>Climate scenario definition:</strong> To test the robustness of the algorithms under varying environmental pressures, we defined three representative climate scenarios based on historical meteorological data from the study region over the past decade (2013-2022):</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par99"><strong>Drought scenario:</strong> Simulates a year where the total precipitation during the growing season falls within the lowest 15th percentile of historical data, accompanied by an average temperature increase of 0.5-1.0<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq92"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/466be87bee27/d33e2083.gif" loading="lazy" id="d33e2083" alt="Inline graphic"></span>C. This scenario represents severe water stress conditions.</p></li>
<li><p id="Par100"><strong>Normal scenario:</strong> All climatic parameters (precipitation, temperature) are set to their historical median values, representing an average or most typical year.</p></li>
<li><p id="Par101"><strong>Abundant water scenario:</strong> Simulates a year where the total precipitation is at the 85th percentile of historical data, representing relatively plentiful water resources.</p></li>
</ul>
<p>In each scenario, these climate parameters were used as inputs for the yield prediction model (Eq. <a href="#Equ2" class="usa-link">2</a>), thereby influencing the final optimization objective value.</p>
<p id="Par102"><strong>Overall evaluation metrics:</strong> A comprehensive comparison using metrics such as Sum of Squared Errors (SSE), Coefficient of Determination (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq93"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/7d1802cbfe3a/d33e2107.gif" loading="lazy" id="d33e2107" alt="Inline graphic"></span>), Profit Increment Ratio, and Robustness is presented in Fig. <a href="#Fig13" class="usa-link">13</a>. The Profit Increment Ratio was calculated for each optimization algorithm by comparing its achieved profit against a defined reference baseline. This baseline represented the average profit obtained from simulating conventional farming practices within the study area, based on historical cropping patterns and average yields recorded from 2020–2022. The ratio is expressed as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq94"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/2ab93f85f900/d33e2116.gif" loading="lazy" id="d33e2116" alt="Inline graphic"></span>, serving as a key indicator of economic performance enhancement.</p>
<figure class="fig xbox font-sm" id="Fig13"><h4 class="obj_head">Fig. 13.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12331943_41598_2025_14188_Fig13_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/37dc838d4362/41598_2025_14188_Fig13_HTML.jpg" loading="lazy" id="MO13" height="342" width="799" alt="Fig. 13"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig13/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of overall evaluation metrics (SSE, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq95"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/7d1802cbfe3a/d33e2151.gif" loading="lazy" id="d33e2151" alt="Inline graphic"></span>, profit increment, robustness) across algorithms.</p></figcaption></figure><p id="Par103">H-SAGA outperformed the baseline algorithms across most metrics, indicating superior optimization effectiveness and stability. Notably, the Profit Increment Ratio achieved by H-SAGA (approximately 14.5%) was significantly higher than that of other tested algorithms (e.g., GA: approx. 8%; SA: approx. 6%; PSO: approx. 5%; ACO: approx. 7.5%), as detailed with specific numerical annotations in Fig. <a href="#Fig14" class="usa-link">14</a>. This represents an improvement of approximately 5 to 9.5 percentage points in profit increment ratios when H-SAGA is compared against these algorithmic benchmarks, highlighting the practical benefit of the H-SAGA approach in enhancing potential revenue from agricultural operations.</p>
<figure class="fig xbox font-sm" id="Fig14"><h4 class="obj_head">Fig. 14.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/e70496c61f7c/41598_2025_14188_Fig14_HTML.jpg" loading="lazy" id="MO14" height="586" width="702" alt="Fig. 14"></p>
<div class="p text-right font-secondary"><a href="figure/Fig14/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Profit increase ratio comparison with numerical annotations, highlighting H-SAGA’s advantage.</p></figcaption></figure><p id="Par104">To statistically substantiate these observed performance differences, a one-way Analysis of Variance (ANOVA) was conducted on the profit increment ratios produced by each algorithm (H-SAGA, GA, SA, PSO, ACO) across the different simulated climate scenarios. The ANOVA results revealed a statistically significant difference in the mean profit increment ratios among the algorithms (F(4, 25) = 15.62, <em>p</em> &lt; 0.001). To further identify specific inter-group differences, a Tukey’s Honestly Significant Difference (HSD) post-hoc test was performed. The post-hoc analysis confirmed that the profit increment ratio from H-SAGA was significantly higher than that of all four baseline algorithms: GA (<em>p</em> = 0.008), SA (<em>p</em> = 0.002), PSO (<em>p</em> &lt; 0.001), and ACO (<em>p</em> = 0.004). This statistical evidence robustly supports the conclusion that the superior performance of H-SAGA is not an artifact of random variation but represents a significant and reliable advantage for enhancing agricultural profitability under the tested conditions.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Performance across diverse terrain types</h3>
<p id="Par105">To further enhance the mountain-specific insights of this study, the optimization effects of the H-SAGA model were quantified across different dominant terrain types within the study area: flat fields, terraced fields, and sloped fields. Each of the 34 plots was categorized based on its primary topographical characteristics. Key performance indicators, including the Profit Increment Ratio and a composite Resource Use Efficiency Index (RUEI), were aggregated for each category. The RUEI is a normalized metric (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq96"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/9d7f0f3522a7/d33e2170.gif" loading="lazy" id="d33e2170" alt="Inline graphic"></span> scale) representing the economic output per unit of key inputs (water and capital), providing a standardized measure of resource productivity.</p>
<p id="Par106">The results, summarized in Table <a href="#Tab6" class="usa-link">6</a>, reveal that the H-SAGA framework delivers significant benefits across all terrain types, albeit with notable variations. Terraced fields exhibited the highest profit increment ratio (16.2%) and RUEI (0.91), which can be attributed to their superior water retention and soil conservation properties, allowing the optimized resource allocation strategy to achieve maximum effect. Conversely, while sloped fields saw a substantial profit increase (11.5%), their lower RUEI (0.76) highlights the inherent challenges in managing resources on such terrain. These findings quantitatively validate the model’s capability to generate tailored strategies that effectively leverage topographical advantages and mitigate disadvantages, reinforcing its practical value for complex mountain agricultural systems.</p>
<section class="tw xbox font-sm" id="Tab6"><h4 class="obj_head">Table 6.</h4>
<div class="caption p"><p>Quantified optimization Effects of H-SAGA across different terrain types.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Terrain Type</th>
<th align="left" colspan="1" rowspan="1">Number of Plots</th>
<th align="left" colspan="1" rowspan="1">Average Profit Increment Ratio (%)</th>
<th align="left" colspan="1" rowspan="1">Resource Use Efficiency Index (RUEI)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Flat Fields</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">14.8%</td>
<td align="left" colspan="1" rowspan="1">0.85</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Terraced Fields</td>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">16.2%</td>
<td align="left" colspan="1" rowspan="1">0.91</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Sloped Fields</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">11.5%</td>
<td align="left" colspan="1" rowspan="1">0.76</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec21"><h3 class="pmc_sec_title">Robustness and uncertainty analysis</h3>
<p id="Par107">To assess the reliability of the H-SAGA-derived solutions under real-world variability, a multi-scenario robustness analysis<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> and Monte Carlo simulations were performed<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>.</p>
<p id="Par108"><strong>Multi-scenario testing:</strong> The sensitivity of the optimal solution was tested against fluctuations in market prices, input costs, precipitation levels, and inherent yield variability. Table <a href="#Tab7" class="usa-link">7</a> summarizes the results, showing the percentage change in the objective function (Fitness Change Rate) and the stability of the top-ranked planting choices (Ranking Stability) under various perturbation scenarios. The Fitness Change Rate was calculated as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq97"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/4c606575bd5a/d33e2251.gif" loading="lazy" id="d33e2251" alt="Inline graphic"></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq98"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/e28ce894738b/d33e2257.gif" loading="lazy" id="d33e2257" alt="Inline graphic"></span> is the objective function value of the H-SAGA solution under nominal (unperturbed) conditions. Ranking Stability was assessed by considering the top 10 crop-plot allocation decisions, ranked by their contribution to the total profit in the baseline H-SAGA solution. It was then calculated as the percentage of these top 10 decisions that remained within the top 10 ranked decisions in the solution obtained under the perturbed scenario. The results indicate high resilience; for example, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq99"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/5d0de11b2542/d33e2263.gif" loading="lazy" id="d33e2263" alt="Inline graphic"></span> price fluctuations or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq100"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/2a0b43071ae4/d33e2269.gif" loading="lazy" id="d33e2269" alt="Inline graphic"></span> precipitation variations resulted in less than 8% and 12% fitness change, respectively, with ranking stability remaining above 85%. Even combined shocks showed contained impacts, suggesting the H-SAGA solutions provide reliable guidance in dynamic environments.</p>
<section class="tw xbox font-sm" id="Tab7"><h4 class="obj_head">Table 7.</h4>
<div class="caption p"><p>Multi-scenario robustness test matrix for H-SAGA model performance under uncertainty. Fitness change rate measures the percentage deviation of the objective function value from the baseline. Ranking stability measures the consistency of the top-ranked solutions compared to the baseline.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Interference Type</th>
<th align="left" colspan="1" rowspan="1">Fluctuation Magnitude</th>
<th align="left" colspan="1" rowspan="1">Fitness Change Rate (%)</th>
<th align="left" colspan="1" rowspan="1">Ranking Stability (%)</th>
</tr></thead>
<tbody>
<tr><td align="left" colspan="4" rowspan="1"><em>Market Factors</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Price Fluctuation (Major Crops)</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq101"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/e2c75039db10/d33e2307.gif" loading="lazy" id="d33e2307" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">&lt; 4%</td>
<td align="left" colspan="1" rowspan="1">&gt; 95%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Price Fluctuation (Major Crops)</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq102"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/5d0de11b2542/d33e2321.gif" loading="lazy" id="d33e2321" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">&lt; 8%</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq103"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/27a731a3d2c1/d33e2330.gif" loading="lazy" id="d33e2330" alt="Inline graphic"></span></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Input Cost Variation (Fertilizer, Fuel)</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq104"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/0d73063e0a45/d33e2340.gif" loading="lazy" id="d33e2340" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">&lt; 6%</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq105"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/9fd36722a451/d33e2349.gif" loading="lazy" id="d33e2349" alt="Inline graphic"></span></td>
</tr>
<tr><td align="left" colspan="4" rowspan="1"><em>Climate Factors</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Precipitation Variation (Growing Season)</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq106"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/0d73063e0a45/d33e2364.gif" loading="lazy" id="d33e2364" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">&lt; 7%</td>
<td align="left" colspan="1" rowspan="1">&gt; 90%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Precipitation Variation (Growing Season)</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq107"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/2a0b43071ae4/d33e2378.gif" loading="lazy" id="d33e2378" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">&lt; 12%</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq108"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/f26fa33975ea/d33e2387.gif" loading="lazy" id="d33e2387" alt="Inline graphic"></span></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Temperature Anomaly (Growing Season)</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq109"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/881bf8afb80b/d33e2397.gif" loading="lazy" id="d33e2397" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">&lt; 5%</td>
<td align="left" colspan="1" rowspan="1">&gt; 93%</td>
</tr>
<tr><td align="left" colspan="4" rowspan="1"><em>Production Factors</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Yield Variability (Systemic)</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq110"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/f82024ea9d97/d33e2416.gif" loading="lazy" id="d33e2416" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">&lt; 9%</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq111"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/c1e85eda968a/d33e2425.gif" loading="lazy" id="d33e2425" alt="Inline graphic"></span></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Yield Variability (Systemic)</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq112"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/0d73063e0a45/d33e2435.gif" loading="lazy" id="d33e2435" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1">&lt; 15%</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq113"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/272c64f0fd23/d33e2444.gif" loading="lazy" id="d33e2444" alt="Inline graphic"></span></td>
</tr>
<tr><td align="left" colspan="4" rowspan="1"><em>Combined Factors</em></td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Combined Shock 1 (Price -20%, Precip -25%)</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1">&lt; 16%</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq114"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/a318e992eb65/d33e2464.gif" loading="lazy" id="d33e2464" alt="Inline graphic"></span></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Combined Shock 2 (Price +15%, Cost +15%)</td>
<td align="left" colspan="1" rowspan="1">–</td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq115"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/88e6f657becd/d33e2476.gif" loading="lazy" id="d33e2476" alt="Inline graphic"></span></td>
<td align="left" colspan="1" rowspan="1"><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq116"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/0ceef35be135/d33e2483.gif" loading="lazy" id="d33e2483" alt="Inline graphic"></span></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par109"><strong>Monte Carlo simulation for climate risk:</strong> To further probe robustness against extreme climate events, a Monte Carlo simulation (10,000 runs) was conducted, incorporating variations in temperature and precipitation based on historical extreme value distributions<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Figure <a href="#Fig15" class="usa-link">15</a> visualizes the distribution of simulated outcomes (profitability relative to the same baseline scenario used for calculating the Profit Increment Ratio in Section "<a href="#Sec19" class="usa-link">Comparative performance analysis</a>") under these conditions. The analysis estimated the probability of the optimized solution failing to meet a minimum profitability threshold (defined as achieving at least 80% of the expected revenue under baseline conditions). The low density of failure points (red dots in Fig. <a href="#Fig15" class="usa-link">15</a>), constituting approximately 2.5% of the 10,000 simulation runs, suggests the strategy is generally robust even when considering tail risks associated with climatic extremes.</p>
<figure class="fig xbox font-sm" id="Fig15"><h4 class="obj_head">Fig. 15.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12331943_41598_2025_14188_Fig15_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/72e4c3be2c3c/41598_2025_14188_Fig15_HTML.jpg" loading="lazy" id="MO15" height="487" width="739" alt="Fig. 15"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig15/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Monte Carlo risk cloud map (10,000 runs) showing simulated revenue outcomes under varying climate conditions. Blue points meet the profitability threshold (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq117"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/24c50dfb2a52/d33e2514.gif" loading="lazy" id="d33e2514" alt="Inline graphic"></span> 80% baseline); red points fall below. The overall estimated failure probability is low.</p></figcaption></figure><p id="Par110">In summary, the experimental results demonstrate the effectiveness, efficiency, and robustness of the H-SAGA approach for optimizing agricultural planting strategies in complex, resource-constrained environments. The model yields well-fitting predictions, outperforms baseline algorithms in convergence and profitability, and generates solutions resilient to significant market and environmental uncertainties.</p></section></section><section id="Sec22"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par111">This section discusses the key findings of the study, focusing on the strengths and advantages of the proposed multi-stage optimization model and the H-SAGA algorithm, acknowledging its limitations and suggesting future research directions, and highlighting the practical implications of the results for agricultural planning.</p>
<section id="Sec23"><h3 class="pmc_sec_title">Strengths and advantages of the proposed model</h3>
<p id="Par112">The research introduces a robust framework for optimizing agricultural planting strategies, particularly tailored for complex environments like mountainous regions. A primary strength lies in the multi-stage optimization model structure. Unlike static, single-period approaches, this framework allows for sequential decision-making across different time intervals. This inherently captures the dynamic nature of agriculture, enabling adjustments based on evolving market prices, resource availability, and climatic conditions<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. The multi-stage approach enhances risk mitigation, crucial in variable terrains where factors like slope, altitude, and precipitation significantly impact cultivation. The model explicitly incorporates these geographical and climatic considerations. For instance, the interplay between soil characteristics (represented via the Soil Quality Index, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq118"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/3a479c4c266f/d33e2536.gif" loading="lazy" id="d33e2536" alt="Inline graphic"></span>) and water availability (via the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq119"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/e0464f510070/d33e2542.gif" loading="lazy" id="d33e2542" alt="Inline graphic"></span> term) within the empirical yield model (Eq. <a href="#Equ2" class="usa-link">2</a>) implicitly addresses key aspects of water-soil coupling, thereby lending the framework greater realism and practical applicability in reflecting site-specific production potentials<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>.</p>
<p id="Par113">The proposed framework effectively instantiates the “Monitoring-Modeling-Decision” paradigm for dynamic agricultural management. Specifically, at the commencement of each optimization stage, the system first ‘monitors‘ the latest environmental data (e.g., updated climate forecasts, soil moisture sensor readings) and market information (e.g., price fluctuations). These real-time data are then fed into the ‘modeling‘ component: the neural network utilizes this new information to update crop revenue predictions, which in turn adjusts the fitness function of the H-SAGA algorithm. Finally, H-SAGA optimizes based on this updated context to output the ‘decision‘ for the current stage—the optimal planting scheme. The outcomes of implementing this scheme (such as expected resource consumption and crop growth status) subsequently serve as partial inputs for the ‘monitoring‘ phase of the next stage, forming a continuous learning and adaptation loop. This iterative, multi-stage optimization process ensures that agricultural planning can dynamically respond to environmental and market changes, thereby enhancing the robustness and economic efficiency of the overall strategy.</p>
<p id="Par114">The optimization engine, the Hybrid Simulated Annealing Genetic Algorithm (H-SAGA), represents another significant advantage. By synergizing the global exploration capability of Simulated Annealing (SA)<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> with the local refinement strength of Genetic Algorithms (GA)<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>, H-SAGA effectively navigates the complex, potentially multimodal search space characteristic of agricultural optimization problems. This hybrid approach overcomes the limitations of traditional methods prone to local optima and the potential stagnation of standalone GA or SA. Experimental results confirm H-SAGA’s superiority over conventional algorithms like GA, SA, PSO, and ACO, demonstrating faster convergence (Fig. <a href="#Fig9" class="usa-link">9</a>), better stability across different simulated climatic conditions, and superior performance in key metrics like convergence speed, constraint handling, and robustness as visualized in the Innovation Dimension Radar Chart (Fig. <a href="#Fig16" class="usa-link">16</a>).</p>
<figure class="fig xbox font-sm" id="Fig16"><h4 class="obj_head">Fig. 16.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/9c0c7cbebde3/41598_2025_14188_Fig16_HTML.jpg" loading="lazy" id="MO16" height="638" width="702" alt="Fig. 16"></p>
<div class="p text-right font-secondary"><a href="figure/Fig16/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Innovation dimension radar chart illustrating the comparative strengths of H-SAGA in various performance aspects such as convergence speed, constraint handling, and robustness.</p></figcaption></figure><p id="Par115">Furthermore, the integration of a three-layer feedforward neural network enhances the model’s predictive power and adaptability. By leveraging historical data to forecast revenues (Fig. <a href="#Fig4" class="usa-link">4</a>) and dynamically updating the fitness function (Eq. <a href="#Equ5" class="usa-link">5</a>), the model becomes more responsive to real-time market fluctuations and climatic variability. This integration contributes to the precision and timeliness of the generated planting strategies. The model’s effectiveness is empirically validated by the strong fit between predicted and observed values (Fig. <a href="#Fig10" class="usa-link">10</a>, Table <a href="#Tab4" class="usa-link">4</a>), achieving an <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq120"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6625/12331943/7d1802cbfe3a/d33e2597.gif" loading="lazy" id="d33e2597" alt="Inline graphic"></span> of 0.828 and a correlation coefficient (<em>R</em>) of 0.91. The demonstrated robustness in the multi-scenario analysis (Table <a href="#Tab7" class="usa-link">7</a>, Fig. <a href="#Fig15" class="usa-link">15</a>) further underscores the reliability of the H-SAGA framework under uncertainty. Furthermore, the experimental results presented in Section <a href="#Sec16" class="usa-link">3</a> (see Fig. <a href="#Fig14" class="usa-link">14</a>) have substantiated H-SAGA’s competitive computational efficiency relative to standard algorithms, alongside its superior capability in enhancing final revenues, underscoring its practical advantages in real-world application scenarios.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Limitations and future research directions</h3>
<p id="Par116">Despite the promising results, the current model possesses certain limitations that offer avenues for future research. Firstly, the model currently relies on certain simplifying assumptions, such as constant market prices and stable conditions *within* each stage, and does not explicitly incorporate the long-term effects of crop rotation on soil health and regeneration over multiple years. While the multi-stage approach allows for updates between stages, intra-stage dynamics and cumulative soil effects are areas for potential refinement.</p>
<p id="Par117">Secondly, the model does not currently account for local agricultural policies, subsidies, or incentive programs, which can significantly influence farmers’ decisions and overall profitability. Integrating these socio-economic factors could enhance the model’s practical relevance for specific regional planning contexts.</p>
<p id="Par118">Thirdly, while the integrated neural network improves revenue prediction, its efficacy is inherently dependent on the quality and quantity of historical data. This reliance might limit its ability to accurately anticipate unprecedented market shocks or extreme climatic events.</p>
<p id="Par119">Future research should focus on addressing these limitations. Incorporating more sophisticated time-series forecasting techniques, such as Long Short-Term Memory (LSTM) networks<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>, could improve the prediction of seasonal anomalies and market dynamics. Expanding the model to a multi-objective optimization framework—which allows for the simultaneous consideration of conflicting goals, such as maximizing profit while minimizing water usage or carbon emissions—could provide more holistic and sustainable solutions<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Exploring dynamic programming techniques could better capture long-term dependencies<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Additionally, explicitly modeling soil nutrient dynamics over extended horizons would increase the model’s ecological validity<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Finally, integrating region-specific policy constraints would further solidify the model’s utility as a practical decision-support tool.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Practical implications and applications</h3>
<p id="Par120">The findings of this study have significant practical implications for agricultural planning and management, especially in resource-constrained and environmentally variable regions like the mountainous area studied in North China. The developed multi-stage H-SAGA model provides a powerful analytical tool for farmers, agricultural cooperatives, and regional planners to optimize crop selection and land allocation decisions.</p>
<p id="Par121">The model’s ability to integrate diverse factors—including geographical constraints (plot area, type), climatic variables (precipitation, temperature implicitly via yield functions), market conditions (prices, demand), resource limitations (water, budget), and agronomic factors (crop complementarity/substitutability)—allows for the generation of holistic and context-specific planting strategies (visualized in Fig. <a href="#Fig11" class="usa-link">11</a> and Fig. <a href="#Fig12" class="usa-link">12</a>). This leads to more efficient utilization of land and resources, potentially increasing overall farm profitability, as suggested by the profit increment comparisons (Fig. <a href="#Fig13" class="usa-link">13</a>, Fig. <a href="#Fig14" class="usa-link">14</a>).</p>
<p id="Par122">A key practical benefit is the model’s demonstrated robustness. The uncertainty quantification and robustness analysis (Section "<a href="#Sec21" class="usa-link">Robustness and uncertainty analysis</a>", Table <a href="#Tab7" class="usa-link">7</a>, Fig. <a href="#Fig15" class="usa-link">15</a>) show that the solutions generated by H-SAGA maintain reasonable performance even under significant fluctuations in market prices and climate variables. This resilience provides decision-makers with greater confidence in the recommended strategies, facilitating better risk management in the face of inherent agricultural uncertainties.</p>
<p id="Par123">The framework can aid in strategic decision-making aimed at enhancing food security and increasing farmer income in challenging environments. By optimizing planting schemes, the model helps ensure stable production levels and maximizes economic returns from available land. Furthermore, the model’s structure is adaptable; while demonstrated in a specific North China context, it can be calibrated and applied to other regions by incorporating local plot characteristics, climate data, crop types, and market specifics, thereby enhancing planting efficacy and profitability elsewhere. The potential inclusion of environmental objectives in future work could further position this tool to support the transition towards more sustainable agricultural practices<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>–<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>.</p>
<p id="Par124">Furthermore, while the primary objective of the H-SAGA model developed in this study is to maximize agricultural economic revenue, its inherent capability in optimizing resource allocation also offers potential for advancing sustainable agricultural practices. By precisely planning crop layouts and the use of resources such as water and land, the model contributes to reducing waste and enhancing utilization efficiency, thereby indirectly alleviating the environmental pressures associated with agricultural activities. For instance, avoiding excessive inputs on unsuitable land or the cultivation of crops with high water demands in water-scarce areas can promote the conservation of soil and water resources. Although the current model does not incorporate long-term soil health effects or detailed ecological indicators as direct optimization objectives, its refined resource management outcomes lay a foundation for future integration of more comprehensive sustainability goals, such as minimizing water footprints, carbon emissions, or maximizing biodiversity. As outlined in the future research directions, extending the model to a multi-objective optimization framework would allow for a more explicit balancing of economic benefits and environmental protection, thus providing more holistic decision support for the sustainable intensification of agriculture.</p>
<p id="Par125">Ultimately, this research contributes a valuable decision-support system that bridges advanced optimization techniques with the practical complexities of modern agriculture.</p></section></section><section id="Sec26"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par126">This research addressed the complex challenge of optimizing agricultural planting strategies, particularly within the constraints characteristic of mountainous regions, by developing and evaluating a sophisticated multi-stage optimization framework.</p>
<p id="Par127"><strong>Research summary:</strong> The core of this study involved formulating a multi-stage optimization model designed to maximize crop cultivation revenue while adhering to geographical, climatic, resource, market demand, and crop interaction constraints. To solve this complex, potentially non-linear optimization problem, we proposed and implemented a Hybrid Simulated Annealing Genetic Algorithm (H-SAGA). This algorithm synergizes the global exploration strength of Simulated Annealing (SA) with the local refinement capabilities of Genetic Algorithms (GA). Furthermore, a feedforward neural network (NN) was integrated into the H-SAGA framework to provide dynamic, data-driven predictions of revenue based on historical data and evolving conditions, thereby enhancing the adaptability and accuracy of the optimization process. Extensive experiments, utilizing data from North China mountainous areas and national statistics, validated the model. Comparisons against baseline algorithms (GA, SA, PSO, ACO) under various simulated climatic conditions demonstrated the superior performance of H-SAGA in terms of convergence speed, solution quality (profitability), and robustness. The results, including sensitivity analyses and uncertainty quantification (Section "<a href="#Sec21" class="usa-link">Robustness and uncertainty analysis</a>"), confirmed the model’s ability to generate effective and resilient planting schemes.</p>
<p id="Par128"><strong>Innovations and contributions:</strong>The primary contributions of this work lie in several key areas. Firstly, the development of a <em>multi-stage optimization model</em> specifically tailored for agricultural planning, capable of handling sequential decision-making under evolving conditions. Secondly, the introduction and successful application of the <em>Hybrid Simulated Annealing Genetic Algorithm (H-SAGA)</em> to this domain, demonstrating its effectiveness in navigating complex, multi-modal search spaces characteristic of agricultural optimization problems, outperforming traditional metaheuristics (as shown in Fig. <a href="#Fig16" class="usa-link">16</a>). Thirdly, the novel <em>integration of a predictive neural network</em> directly within the optimization loop, allowing the fitness function to dynamically adapt to predicted market and yield fluctuations, thus improving the practical relevance and timeliness of the generated solutions. Finally, the rigorous validation through multi-scenario robustness testing (Table <a href="#Tab7" class="usa-link">7</a>) and parameter sensitivity analysis (Fig. <a href="#Fig8" class="usa-link">8</a>) provides quantitative evidence of the model’s resilience and its capacity to capture complex environmental interactions, offering a valuable decision-support tool for stakeholders in challenging agricultural environments. The demonstrated improvements in profit increment ratios (Fig. <a href="#Fig14" class="usa-link">14</a>) further underscore the practical value of the proposed approach.</p>
<p id="Par129"><strong>Future research directions:</strong> While the H-SAGA model demonstrates significant promise, several avenues for future research could enhance its scope and applicability. The current model primarily focuses on single-season optimization and does not explicitly incorporate the long-term effects of crop rotation on soil health and regeneration; integrating multi-year soil dynamics could lead to more sustainable strategies. The reliance on existing regression techniques, even within the NN, might be further improved by exploring more advanced time-series forecasting methods, such as Long Short-Term Memory (LSTM) networks, to better capture complex seasonal anomalies and market volatility. Expanding the optimization framework to a multi-objective context is another pertinent direction, allowing for the simultaneous consideration of economic goals alongside environmental metrics, such as water footprint, carbon emissions, or biodiversity indices. Incorporating socio-economic factors, including local policy incentives, labor availability, or risk aversion profiles of farmers, could also increase the model’s practical utility. Refining the handling of non-linear relationships and constraints, perhaps through dynamic programming elements or more sophisticated constraint-handling techniques within the H-SAGA, could further improve solution precision. Adapting and validating the model for different geographical regions and agricultural systems would also be valuable, potentially contributing to broader goals of climate resilience and sustainable intensification in agriculture globally.</p></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>Special appreciation is extended to the Innovation Laboratory at the School of Information Technology and Engineering, Guangzhou Business School, for their technical assistance. We also thank Professors Zhang Cuifang and Yu Hua, along with Teachers Zhu Xiaomi, Guo Jie, and Xu Chunrong, for their invaluable guidance in data analysis. The contributions from all parties have been instrumental in the successful completion of this study.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>C.L. conceived the experiments and methodology, acquired funding and administered the project. Z.S. developed the software, supervised the project. Y.N. conducted formal analysis, investigation, data curation and prepared the original draft. Z.Ye., J.L., Z.Yang., J.W., W.Z., and Y.C. contributed to data collection and preliminary analysis. Z.S. and X.L. performed validation. X.L. created visualizations. Z.S. and C.L. reviewed and edited the manuscript. All authors reviewed and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by the Guangdong Provincial Science and Technology Innovation Strategy Special Fund (grant number pdjh2024a467) and the Guangdong Basic and Applied Basic Research Foundation (grant numbers 2023A1515110472, 2024A1515010039).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The datasets generated and/or analysed during the current study are available from the corresponding author on reasonable request. The primary public datasets used are available in the repositories cited in the manuscript.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par134">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Ge, G. &amp; Huang, C. Climate change and its impact on water resources in north china. <em>Adv. Atmos. Sci.</em><strong>18</strong>, 718–732. 10.1007/BF03403497 (2001).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ge,%20G.%20&amp;%20Huang,%20C.%20Climate%20change%20and%20its%20impact%20on%20water%20resources%20in%20north%20china.%20Adv.%20Atmos.%20Sci.18,%20718%E2%80%93732.%2010.1007/BF03403497%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Wang, Y., Li, X., Lu, D. &amp; Yan, J. Evaluating the impact of land fragmentation on the cost of agricultural operation in the southwest mountainous areas of china. <em>Land Use Policy</em><strong>99</strong>, 105099. 10.1016/j.landusepol.2020.105099 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Y.,%20Li,%20X.,%20Lu,%20D.%20&amp;%20Yan,%20J.%20Evaluating%20the%20impact%20of%20land%20fragmentation%20on%20the%20cost%20of%20agricultural%20operation%20in%20the%20southwest%20mountainous%20areas%20of%20china.%20Land%20Use%20Policy99,%20105099.%2010.1016/j.landusepol.2020.105099%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Li, S., Pu, J. &amp; Deng, X. Agricultural land use transition under multidimensional topographical gradients and its impact on ecosystem service interactions. <em>J. Integr. Agric.</em>10.1016/j.jia.2025.02.005 (2025).</cite> [<a href="https://scholar.google.com/scholar_lookup?Li,%20S.,%20Pu,%20J.%20&amp;%20Deng,%20X.%20Agricultural%20land%20use%20transition%20under%20multidimensional%20topographical%20gradients%20and%20its%20impact%20on%20ecosystem%20service%20interactions.%20J.%20Integr.%20Agric.10.1016/j.jia.2025.02.005%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Nie, Y., Li, X., Jiang, W. &amp; Liu, N. Planting structure optimisation of three main grain crops in 10 northern china provinces based on water footprint method. <em>Resour. Sci.</em><strong>44</strong>, 2315–2329. 10.18402/resci.2022.11.12 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Nie,%20Y.,%20Li,%20X.,%20Jiang,%20W.%20&amp;%20Liu,%20N.%20Planting%20structure%20optimisation%20of%20three%20main%20grain%20crops%20in%2010%20northern%20china%20provinces%20based%20on%20water%20footprint%20method.%20Resour.%20Sci.44,%202315%E2%80%932329.%2010.18402/resci.2022.11.12%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Beniston, M. &amp; Stoffel, M. Assessing the impacts of climatic change on mountain water resources: a european alps case study. <em>Sci. Total Environ.</em><strong>493</strong>, 1129–1137. 10.1016/j.scitotenv.2013.11.122 (2014).
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2013.11.122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24360916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beniston,%20M.%20&amp;%20Stoffel,%20M.%20Assessing%20the%20impacts%20of%20climatic%20change%20on%20mountain%20water%20resources:%20a%20european%20alps%20case%20study.%20Sci.%20Total%20Environ.493,%201129%E2%80%931137.%2010.1016/j.scitotenv.2013.11.122%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Zhang, P.-f. et al. Optimization and adjustment of agricultural planting structure in shanxi province based on water-energy-food nexus. <em>Water Sav. Irrig.</em> 55–63, 10.12396/jsgg.2022206 (2023).</cite>
</li>
<li id="CR7">
<span class="label">7.</span><cite>Saranya, S. &amp; Amudha, T. Crop planning optimisation research — a detailed investigation. In <em>2016 IEEE International Conference on Advances in Computer Applications (ICACA)</em>, 202–208, 10.1109/ICACA.2016.7887951 (2016).</cite>
</li>
<li id="CR8">
<span class="label">8.</span><cite>Itoh, T., Ishii, H. &amp; Nanseki, T. A model of crop planning under uncertainty in agricultural management. <em>Int. J. Prod. Econ.</em><strong>81–82</strong>, 555–558. 10.1016/S0925-5273(02)00283-9 (2003).</cite> [<a href="https://scholar.google.com/scholar_lookup?Itoh,%20T.,%20Ishii,%20H.%20&amp;%20Nanseki,%20T.%20A%20model%20of%20crop%20planning%20under%20uncertainty%20in%20agricultural%20management.%20Int.%20J.%20Prod.%20Econ.81%E2%80%9382,%20555%E2%80%93558.%2010.1016/S0925-5273(02)00283-9%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Blum, C. &amp; Roli, A. Metaheuristics in combinatorial optimization: Overview and conceptual comparison. <em>ACM Comput. Surv.</em><strong>35</strong>, 268–308. 10.1145/937503.937505 (2003).</cite> [<a href="https://scholar.google.com/scholar_lookup?Blum,%20C.%20&amp;%20Roli,%20A.%20Metaheuristics%20in%20combinatorial%20optimization:%20Overview%20and%20conceptual%20comparison.%20ACM%20Comput.%20Surv.35,%20268%E2%80%93308.%2010.1145/937503.937505%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Zhang, J. &amp; Tao, L. Improved adaptive genetic simulated annealing algorithm. In Zhang, D. (ed.) <em>Fifth International Conference on Mechatronics and Computer Technology Engineering (MCTE 2022)</em>, vol. 12500, 1510–1515, 10.1117/12.2662641 SPIE (2022).</cite>
</li>
<li id="CR11">
<span class="label">11.</span><cite>Khanna, M., Atallah, S. S., Heckelei, T., Wu, L. &amp; Storm, H. Economics of the adoption of artificial intelligence-based digital technologies in agriculture. <em>Annu. Rev. Resour. Econ.</em><strong>16</strong>, 41–61. 10.1146/annurev-resource-101623-092515 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Khanna,%20M.,%20Atallah,%20S.%20S.,%20Heckelei,%20T.,%20Wu,%20L.%20&amp;%20Storm,%20H.%20Economics%20of%20the%20adoption%20of%20artificial%20intelligence-based%20digital%20technologies%20in%20agriculture.%20Annu.%20Rev.%20Resour.%20Econ.16,%2041%E2%80%9361.%2010.1146/annurev-resource-101623-092515%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Boussios, D. et al. Modeling producer responses with dynamic programming: a case for adaptive crop management. <em>Agric. Econ.</em><strong>50</strong>, 101–111. 10.1111/agec.12469 (2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Boussios,%20D.%20et%20al.%20Modeling%20producer%20responses%20with%20dynamic%20programming:%20a%20case%20for%20adaptive%20crop%20management.%20Agric.%20Econ.50,%20101%E2%80%93111.%2010.1111/agec.12469%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Bakker, H., Dunke, F. &amp; Nickel, S. A structuring review on multi-stage optimisation under uncertainty: Aligning concepts from theory and practice. <em>Omega-International J. Manag. Sci.</em><strong>96</strong>, 102080. 10.1016/j.omega.2019.06.006 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bakker,%20H.,%20Dunke,%20F.%20&amp;%20Nickel,%20S.%20A%20structuring%20review%20on%20multi-stage%20optimisation%20under%20uncertainty:%20Aligning%20concepts%20from%20theory%20and%20practice.%20Omega-International%20J.%20Manag.%20Sci.96,%20102080.%2010.1016/j.omega.2019.06.006%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Bareille, F. &amp; Letort, E. How do farmers manage crop biodiversity? a dynamic acreage model with productive feedback. <em>Eur. Rev. Agric. Econ.</em><strong>45</strong>, 617–639. 10.1093/erae/jby011 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bareille,%20F.%20&amp;%20Letort,%20E.%20How%20do%20farmers%20manage%20crop%20biodiversity?%20a%20dynamic%20acreage%20model%20with%20productive%20feedback.%20Eur.%20Rev.%20Agric.%20Econ.45,%20617%E2%80%93639.%2010.1093/erae/jby011%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Lehmann, N., Briner, S. &amp; Finger, R. The impact of climate and price risks on agricultural land use and crop management decisions. <em>Land Use Policy</em><strong>35</strong>, 119–130. 10.1016/j.landusepol.2013.05.008 (2013).</cite> [<a href="https://scholar.google.com/scholar_lookup?Lehmann,%20N.,%20Briner,%20S.%20&amp;%20Finger,%20R.%20The%20impact%20of%20climate%20and%20price%20risks%20on%20agricultural%20land%20use%20and%20crop%20management%20decisions.%20Land%20Use%20Policy35,%20119%E2%80%93130.%2010.1016/j.landusepol.2013.05.008%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Li, S.-Y., Du, Z.-H., Wu, M.-Y., Zhu, J. &amp; Li, S.-L. Parallel realization of simulated annealing algorithm: Modifications and applications. <em>Acta Phys. Sinica</em><strong>50</strong>, 1260–1263. 10.7498/aps.50.1260 (2001).</cite> [<a href="https://scholar.google.com/scholar_lookup?Li,%20S.-Y.,%20Du,%20Z.-H.,%20Wu,%20M.-Y.,%20Zhu,%20J.%20&amp;%20Li,%20S.-L.%20Parallel%20realization%20of%20simulated%20annealing%20algorithm:%20Modifications%20and%20applications.%20Acta%20Phys.%20Sinica50,%201260%E2%80%931263.%2010.7498/aps.50.1260%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Suanpang, P., Jamjuntr, P., Jermsittiparsert, K. &amp; Kaewyong, P. Tourism service scheduling in smart city based on hybrid genetic algorithm simulated annealing algorithm. <em>Sustainability</em><strong>14</strong>, 16293. 10.3390/su142316293 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Suanpang,%20P.,%20Jamjuntr,%20P.,%20Jermsittiparsert,%20K.%20&amp;%20Kaewyong,%20P.%20Tourism%20service%20scheduling%20in%20smart%20city%20based%20on%20hybrid%20genetic%20algorithm%20simulated%20annealing%20algorithm.%20Sustainability14,%2016293.%2010.3390/su142316293%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Bertsimas, D. &amp; Margaritis, G. Global optimisation: a machine learning approach. <em>J. Glob. optimisation</em><strong>91</strong>, 1–37. 10.1007/s10898-024-01434-9 (2025).</cite> [<a href="https://scholar.google.com/scholar_lookup?Bertsimas,%20D.%20&amp;%20Margaritis,%20G.%20Global%20optimisation:%20a%20machine%20learning%20approach.%20J.%20Glob.%20optimisation91,%201%E2%80%9337.%2010.1007/s10898-024-01434-9%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>D’Angelo, G. &amp; Palmieri, F. Gga: A modified genetic algorithm with gradient-based local search for solving constrained optimisation problems. <em>Inf. Sci.</em><strong>547</strong>, 136–162. 10.1016/j.ins.2020.08.040 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?D%E2%80%99Angelo,%20G.%20&amp;%20Palmieri,%20F.%20Gga:%20A%20modified%20genetic%20algorithm%20with%20gradient-based%20local%20search%20for%20solving%20constrained%20optimisation%20problems.%20Inf.%20Sci.547,%20136%E2%80%93162.%2010.1016/j.ins.2020.08.040%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Wang, F., Chun, W. &amp; Wu, W. Application of simulated annealing algorithm in multi-objective allocation optimisation of urban water resources. <em>Desalination Water Treat.</em><strong>314</strong>, 304–313. 10.5004/dwt.2023.30032 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Wang,%20F.,%20Chun,%20W.%20&amp;%20Wu,%20W.%20Application%20of%20simulated%20annealing%20algorithm%20in%20multi-objective%20allocation%20optimisation%20of%20urban%20water%20resources.%20Desalination%20Water%20Treat.314,%20304%E2%80%93313.%2010.5004/dwt.2023.30032%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Abatal, M., Rabdel Ruiz-Salvador, A. &amp; Cruz Hernández, N. A dft-based simulated annealing method for the optimisation of global energy in zeolite framework systems: Application to natrolite, chabazite and clinoptilolite. <em>Microporous and Mesoporous Mater.</em><strong>294</strong>, 109885. 10.1016/j.micromeso.2019.109885 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Abatal,%20M.,%20Rabdel%20Ruiz-Salvador,%20A.%20&amp;%20Cruz%20Hern%C3%A1ndez,%20N.%20A%20dft-based%20simulated%20annealing%20method%20for%20the%20optimisation%20of%20global%20energy%20in%20zeolite%20framework%20systems:%20Application%20to%20natrolite,%20chabazite%20and%20clinoptilolite.%20Microporous%20and%20Mesoporous%20Mater.294,%20109885.%2010.1016/j.micromeso.2019.109885%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Benmesbah, O., Lamia, M. &amp; Hafidi, M. An improved constrained learning path adaptation problem based on genetic algorithm. <em>Interact. Learn. Environ.</em><strong>31</strong>, 3595–3612. 10.1080/10494820.2021.1937659 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Benmesbah,%20O.,%20Lamia,%20M.%20&amp;%20Hafidi,%20M.%20An%20improved%20constrained%20learning%20path%20adaptation%20problem%20based%20on%20genetic%20algorithm.%20Interact.%20Learn.%20Environ.31,%203595%E2%80%933612.%2010.1080/10494820.2021.1937659%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Yang, Q., Li, Z.-Y., Rodríguez-Kessler, P. L. &amp; He, S.-G. Accelerated global optimization of metal cluster structures via the deep neural network complemented with transfer learning and genetic algorithm. <em>Chin. J. Chem. Phys.</em><strong>37</strong>, 321–329. 10.1063/1674-0068/cjcp2309083 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yang,%20Q.,%20Li,%20Z.-Y.,%20Rodr%C3%ADguez-Kessler,%20P.%20L.%20&amp;%20He,%20S.-G.%20Accelerated%20global%20optimization%20of%20metal%20cluster%20structures%20via%20the%20deep%20neural%20network%20complemented%20with%20transfer%20learning%20and%20genetic%20algorithm.%20Chin.%20J.%20Chem.%20Phys.37,%20321%E2%80%93329.%2010.1063/1674-0068/cjcp2309083%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Zhao, Y., Stasinakis, C., Sermpinis, G. &amp; Shi, Y. Neural network copula portfolio optimisation for exchange traded funds. <em>Quant. Finance</em><strong>18</strong>, 761–775. 10.1080/14697688.2017.1414505 (2018).</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20Y.,%20Stasinakis,%20C.,%20Sermpinis,%20G.%20&amp;%20Shi,%20Y.%20Neural%20network%20copula%20portfolio%20optimisation%20for%20exchange%20traded%20funds.%20Quant.%20Finance18,%20761%E2%80%93775.%2010.1080/14697688.2017.1414505%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Nithyashree, C., Nair, R., Sharma, S., Shende, T. &amp; Bhuva, A. Optimal crop rotation based on soil composition. In <em>2023 International Conference on Integration of Computational Intelligent System (ICICIS)</em>, 1–6, 10.1109/ICICIS56802.2023.10430231 IEEE, (2023).</cite>
</li>
<li id="CR26">
<span class="label">26.</span><cite>National Bureau of Statistics of China. China statistical yearbook (2000-2024). Accessed from the official website of the National Bureau of Statistics of China. URL: http://www.stats.gov.cn/.</cite>
</li>
<li id="CR27">
<span class="label">27.</span><cite>Tianjin Municipal Bureau of Statistics. Tianjin statistical yearbook (2024). Accessed from the official website. URL: https://stats.tj.gov.cn/nianjian/2024nj/zk/indexch.htm.</cite>
</li>
<li id="CR28">
<span class="label">28.</span><cite>Beijing Municipal Bureau of Statistics. Beijing statistical yearbook (2024). Accessed from the official website. URL: https://nj.tjj.beijing.gov.cn/nj/main/2024-tjnj/zk/e/indexch.htm.</cite>
</li>
<li id="CR29">
<span class="label">29.</span><cite>Yang, J. &amp; Huang, X. The 30 m annual land cover dataset and its dynamics in china from 1990 to 2019, 10.5194/essd-13-3907-2021 (2021). This reference represents a typical high-quality, citable source for land cover data, often derived from platforms like GEE.</cite>
</li>
<li id="CR30">
<span class="label">30.</span><cite>Farr, T. G. et al. The shuttle radar topography mission. <em>Rev. Geophys.</em><strong>45</strong>, 10.1029/2005RG000183 (2007).</cite>
</li>
<li id="CR31">
<span class="label">31.</span><cite>Lehner, B., Verdin, K. &amp; Jarvis, A. New global hydrography derived from spaceborne elevation data. <em>Eos, TransactionsAm. Geophys. Union</em><strong>89</strong>, 93–94. 10.1029/2008EO100001 (2008).</cite> [<a href="https://scholar.google.com/scholar_lookup?Lehner,%20B.,%20Verdin,%20K.%20&amp;%20Jarvis,%20A.%20New%20global%20hydrography%20derived%20from%20spaceborne%20elevation%20data.%20Eos,%20TransactionsAm.%20Geophys.%20Union89,%2093%E2%80%9394.%2010.1029/2008EO100001%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Paiva, J. R. B. et al. Multidimensional robustness analysis for optimizing complex systems. <em>KKnowl.-Based Syst.</em><strong>318</strong>, 113527. 10.1016/j.knosys.2025.113527 (2025).</cite> [<a href="https://scholar.google.com/scholar_lookup?Paiva,%20J.%20R.%20B.%20et%20al.%20Multidimensional%20robustness%20analysis%20for%20optimizing%20complex%20systems.%20KKnowl.-Based%20Syst.318,%20113527.%2010.1016/j.knosys.2025.113527%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Mavrotas, G., Pechak, O., Siskos, E., Doukas, H. &amp; Psarras, J. Robustness analysis in multi-objective mathematical programming using monte carlo simulation. <em>Eur. J. Oper. Res.</em><strong>240</strong>, 193–201. 10.1016/j.ejor.2014.06.039 (2015).</cite> [<a href="https://scholar.google.com/scholar_lookup?Mavrotas,%20G.,%20Pechak,%20O.,%20Siskos,%20E.,%20Doukas,%20H.%20&amp;%20Psarras,%20J.%20Robustness%20analysis%20in%20multi-objective%20mathematical%20programming%20using%20monte%20carlo%20simulation.%20Eur.%20J.%20Oper.%20Res.240,%20193%E2%80%93201.%2010.1016/j.ejor.2014.06.039%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Benson, R. &amp; Kellner, D. Monte carlo simulation for reliability. In <em>2020 Annual Reliability and Maintainability Symposium (RAMS)</em>, 10.1109/rams48030.2020.9153600 (2020).</cite>
</li>
<li id="CR35">
<span class="label">35.</span><cite>Zhang, Z. &amp; Li, J. Monte carlo simulation of climate systems. In <em>Big Data Mining for Climate Change</em>, 215–241, 10.1016/b978-0-12-818703-6.00012-x Elsevier, (2020).</cite>
</li>
<li id="CR36">
<span class="label">36.</span><cite>Babonneau, F., Haurie, A., Loulou, R. &amp; Vielle, M. Combining stochastic optimization and monte carlo simulation to deal with uncertainties in climate policy assessment. <em>Environ. Model. &amp; Assess.</em><strong>17</strong>, 51–76. 10.1007/s10666-011-9275-1 (2011).</cite> [<a href="https://scholar.google.com/scholar_lookup?Babonneau,%20F.,%20Haurie,%20A.,%20Loulou,%20R.%20&amp;%20Vielle,%20M.%20Combining%20stochastic%20optimization%20and%20monte%20carlo%20simulation%20to%20deal%20with%20uncertainties%20in%20climate%20policy%20assessment.%20Environ.%20Model.%20&amp;%20Assess.17,%2051%E2%80%9376.%2010.1007/s10666-011-9275-1%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Rueda, J. L. &amp; Erlich, I. Evaluation of the mean-variance mapping optimisation for solving multimodal problems. In <em>2013 IEEE Symposium on Swarm Intelligence (SIS)</em>, 7–14, 10.1109/SIS.2013.6615153 (IEEE, Piscataway, NJ, 2013).</cite>
</li>
<li id="CR38">
<span class="label">38.</span><cite>Xiao, Z., Zhang, Z. &amp; Sang, W. Optimizing agricultural planting strategies with linear programming and mopso. In <em>Highlights in Business, Economics and Management</em>, vol. 45, 708–713, 10.54097/qk7rgk29 (Darcy &amp; Roy Press Co. Ltd, 2024).</cite>
</li>
<li id="CR39">
<span class="label">39.</span><cite>Hassna, B., Namany, S., Alherbawi, M., Elomri, A. &amp; Al-Ansari, T. Multi-objective optimization for food availability under economic and environmental risk constraints. <em>Sustainability</em><strong>16</strong>, 4336. 10.3390/su16114336 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hassna,%20B.,%20Namany,%20S.,%20Alherbawi,%20M.,%20Elomri,%20A.%20&amp;%20Al-Ansari,%20T.%20Multi-objective%20optimization%20for%20food%20availability%20under%20economic%20and%20environmental%20risk%20constraints.%20Sustainability16,%204336.%2010.3390/su16114336%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Al-Milli, N., Hudaib, A. &amp; Obeid, N. Population diversity control of genetic algorithm using a novel injection method for bankruptcy prediction problem. <em>Mathematics</em><strong>9</strong>, 823. 10.3390/math9080823 (2021).</cite> [<a href="https://scholar.google.com/scholar_lookup?Al-Milli,%20N.,%20Hudaib,%20A.%20&amp;%20Obeid,%20N.%20Population%20diversity%20control%20of%20genetic%20algorithm%20using%20a%20novel%20injection%20method%20for%20bankruptcy%20prediction%20problem.%20Mathematics9,%20823.%2010.3390/math9080823%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Hu, J. et al. Time series prediction method based on variant lstm recurrent neural network. <em>Neural Process. Lett.</em><strong>52</strong>, 1485–1500. 10.1007/s11063-020-10319-3 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hu,%20J.%20et%20al.%20Time%20series%20prediction%20method%20based%20on%20variant%20lstm%20recurrent%20neural%20network.%20Neural%20Process.%20Lett.52,%201485%E2%80%931500.%2010.1007/s11063-020-10319-3%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Li, M. et al. An ecological footprint approach for cropland use sustainability based on multi-objective optimisation modelling. <em>J. Environ. Manag.</em><strong>273</strong>, 111147. 10.1016/j.jenvman.2020.111147 (2020).</cite> [<a href="https://doi.org/10.1016/j.jenvman.2020.111147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32768762/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20M.%20et%20al.%20An%20ecological%20footprint%20approach%20for%20cropland%20use%20sustainability%20based%20on%20multi-objective%20optimisation%20modelling.%20J.%20Environ.%20Manag.273,%20111147.%2010.1016/j.jenvman.2020.111147%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Mani, G. et al. A comparative study of long short-term memory based long-term electrical load forecasting techniques with hyperparameter optimization. <em>Int. J. Electr. Comput. Eng. (IJECE)</em><strong>14</strong>, 7080, 10.11591/ijece.v14i6.pp7080-7089 (2024).</cite>
</li>
<li id="CR44">
<span class="label">44.</span><cite>Gunda, T., Bazuin, J. T., Nay, J. &amp; Yeung, K. L. Impact of seasonal forecast use on agricultural income in a system with varying crop costs and returns: an empirically-grounded simulation. <em>Environ. Res. Lett.</em><strong>12</strong>, 034001. 10.1088/1748-9326/aa5ef7 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gunda,%20T.,%20Bazuin,%20J.%20T.,%20Nay,%20J.%20&amp;%20Yeung,%20K.%20L.%20Impact%20of%20seasonal%20forecast%20use%20on%20agricultural%20income%20in%20a%20system%20with%20varying%20crop%20costs%20and%20returns:%20an%20empirically-grounded%20simulation.%20Environ.%20Res.%20Lett.12,%20034001.%2010.1088/1748-9326/aa5ef7%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Teklewold, H., Mekonnen, A., Kohlin, G. &amp; Di Falco, S. Does adoption of multiple climate-smart practices improve farmers’ climate resilience? empirical evidence from the nile basin of ethiopia. <em>Clim. Chang. Econ.</em><strong>8</strong>, 1750001. 10.1142/S2010007817500014 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Teklewold,%20H.,%20Mekonnen,%20A.,%20Kohlin,%20G.%20&amp;%20Di%20Falco,%20S.%20Does%20adoption%20of%20multiple%20climate-smart%20practices%20improve%20farmers%E2%80%99%20climate%20resilience?%20empirical%20evidence%20from%20the%20nile%20basin%20of%20ethiopia.%20Clim.%20Chang.%20Econ.8,%201750001.%2010.1142/S2010007817500014%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Issahaku, G. &amp; Abdulai, A. Adoption of climate-smart practices and its impact on farm performance and risk exposure among smallholder farmers in ghana. <em>Aust. J. Agric. Resour. Econ.</em><strong>64</strong>, 396–420. 10.1111/1467-8489.12357 (2020).</cite> [<a href="https://scholar.google.com/scholar_lookup?Issahaku,%20G.%20&amp;%20Abdulai,%20A.%20Adoption%20of%20climate-smart%20practices%20and%20its%20impact%20on%20farm%20performance%20and%20risk%20exposure%20among%20smallholder%20farmers%20in%20ghana.%20Aust.%20J.%20Agric.%20Resour.%20Econ.64,%20396%E2%80%93420.%2010.1111/1467-8489.12357%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Ali, E. Farm households’ adoption of climate-smart practices in subsistence agriculture: Evidence from northern togo. <em>Environ. Manag.</em><strong>67</strong>, 949–962. 10.1007/s00267-021-01436-3 (2021).</cite> [<a href="https://doi.org/10.1007/s00267-021-01436-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33566132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ali,%20E.%20Farm%20households%E2%80%99%20adoption%20of%20climate-smart%20practices%20in%20subsistence%20agriculture:%20Evidence%20from%20northern%20togo.%20Environ.%20Manag.67,%20949%E2%80%93962.%2010.1007/s00267-021-01436-3%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The datasets generated and/or analysed during the current study are available from the corresponding author on reasonable request. The primary public datasets used are available in the repositories cited in the manuscript.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-14188-5"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_14188.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (6.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12331943/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12331943/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12331943%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12331943/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12331943/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12331943/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40775001/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12331943/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40775001/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12331943/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12331943/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="ogvgpH3BTHmXBgJmDIAtgMKutFGQIz6JmcT2R8RpQE1ZhHZrIwvJMA0OtIQHoVj0">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
