
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Role of Structural Modifications in Peptidomimetic Compounds as Potential Antimicrobial Agents against and : Balancing Bioavailability, Safety, and Antimicrobial Activity - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="FF20820E89B737231A820E00184AD9A4.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsomega">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332667/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Omega">
<meta name="citation_title" content="Role of Structural Modifications in Peptidomimetic Compounds as Potential Antimicrobial Agents against and : Balancing Bioavailability, Safety, and Antimicrobial Activity">
<meta name="citation_author" content="Maria Dzierżyńska">
<meta name="citation_author_institution" content="Department of Biomedical Chemistry, Faculty of Chemistry, University of Gdańsk, Wita Stwosza 63, 80-308 Gdansk, Poland">
<meta name="citation_author" content="Justyna Sawicka">
<meta name="citation_author_institution" content="Department of Molecular Biotechnology, Mossakowski Medical Research Institute Polish Academy of Science, Wita Stwosza 63, 80-308 Gdansk, Poland">
<meta name="citation_author" content="Katarzyna Łada">
<meta name="citation_author_institution" content="Department of Biomedical Chemistry, Faculty of Chemistry, University of Gdańsk, Wita Stwosza 63, 80-308 Gdansk, Poland">
<meta name="citation_author" content="Agnieszka Gajewicz-Skretna">
<meta name="citation_author_institution" content="Department of Physical Chemistry, Faculty of Chemistry, Gdańsk University of Technology, Narutowicza 11/12, 80-233 Gdansk, Poland">
<meta name="citation_author" content="Milena Deptuła">
<meta name="citation_author_institution" content="Laboratory of Tissue Engineering and Regenerative Medicine, Division of Embryology, Medical University of Gdansk, Dębinki 1, 80-211 Gdansk, Poland">
<meta name="citation_author" content="Alexey Chernobrovkin">
<meta name="citation_author_institution" content="Department of Medical Biochemistry and Biophysics, Karolinska Institutet, Solnavägen 9, 17177 Stockholm, Sweden">
<meta name="citation_author_institution" content="Pelago Bioscience AB, Scheelesväg 1, 17165 Solna, Sweden">
<meta name="citation_author" content="Aneta Pogorzelska">
<meta name="citation_author_institution" content="Department of Organic Chemistry, Faculty of Pharmacy, Medical University of Gdańsk, Hallera 107, 80-416 Gdansk, Poland">
<meta name="citation_author" content="Anders Grubb">
<meta name="citation_author_institution" content="Department of Clinical Chemistry, Skåne University Hospital, Lund University, Klinikgatan 19, 22185 Lund, Sweden">
<meta name="citation_author" content="Roman A Zubarev">
<meta name="citation_author_institution" content="Department of Medical Biochemistry and Biophysics, Karolinska Institutet, Solnavägen 9, 17177 Stockholm, Sweden">
<meta name="citation_author" content="Michał Pikuła">
<meta name="citation_author_institution" content="Laboratory of Tissue Engineering and Regenerative Medicine, Division of Embryology, Medical University of Gdansk, Dębinki 1, 80-211 Gdansk, Poland">
<meta name="citation_author" content="Franciszek Kasprzykowski">
<meta name="citation_author_institution" content="Department of Biomedical Chemistry, Faculty of Chemistry, University of Gdańsk, Wita Stwosza 63, 80-308 Gdansk, Poland">
<meta name="citation_author" content="Sylwia Rodziewicz-Motowidło">
<meta name="citation_author_institution" content="Department of Biomedical Chemistry, Faculty of Chemistry, University of Gdańsk, Wita Stwosza 63, 80-308 Gdansk, Poland">
<meta name="citation_publication_date" content="2025 Jul 22">
<meta name="citation_volume" content="10">
<meta name="citation_issue" content="30">
<meta name="citation_firstpage" content="33435">
<meta name="citation_doi" content="10.1021/acsomega.5c03775">
<meta name="citation_pmid" content="40787370">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332667/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332667/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332667/pdf/ao5c03775.pdf">
<meta name="description" content="The emergence of drug-resistant Gram-positive pathogens, particularlyand, has driven the need for novel antimicrobial agents. This study explores 21 newly synthesized peptidomimetic analogues of cystatin C N-terminal fragment, designed to enhance ...">
<meta name="og:title" content="Role of Structural Modifications in Peptidomimetic Compounds as Potential Antimicrobial Agents against and : Balancing Bioavailability, Safety, and Antimicrobial Activity">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The emergence of drug-resistant Gram-positive pathogens, particularlyand, has driven the need for novel antimicrobial agents. This study explores 21 newly synthesized peptidomimetic analogues of cystatin C N-terminal fragment, designed to enhance ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332667/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12332667">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsomega.5c03775"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/ao5c03775.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332667%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12332667/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12332667/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332667/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsomega.png" alt="ACS Omega logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Omega" title="Link to ACS Omega" shape="default" href="https://pubs.acs.org/journal/acsodf" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Omega</button></div>. 2025 Jul 22;10(30):33435–33460. doi: <a href="https://doi.org/10.1021/acsomega.5c03775" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsomega.5c03775</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Omega%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Omega%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Omega%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Role of Structural
Modifications in Peptidomimetic
Compounds as Potential Antimicrobial Agents against and : Balancing Bioavailability, Safety, and Antimicrobial
Activity</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dzier%C5%BCy%C5%84ska%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Maria Dzierżyńska</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Maria Dzierżyńska</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biomedical Chemistry, Faculty of Chemistry, University of Gdańsk, Wita Stwosza 63, 80-308
Gdansk, Poland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dzier%C5%BCy%C5%84ska%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Maria Dzierżyńska</span></a>
</div>
</div>
<sup>†,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sawicka%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Justyna Sawicka</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Justyna Sawicka</span></h3>
<div class="p">
<sup>‡</sup>
Department
of Molecular Biotechnology, Mossakowski
Medical Research Institute Polish Academy of Science, Wita Stwosza 63, 80-308
Gdansk, Poland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sawicka%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Justyna Sawicka</span></a>
</div>
</div>
<sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22%C5%81ada%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Katarzyna Łada</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Katarzyna Łada</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biomedical Chemistry, Faculty of Chemistry, University of Gdańsk, Wita Stwosza 63, 80-308
Gdansk, Poland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22%C5%81ada%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Katarzyna Łada</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gajewicz-Skretna%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Agnieszka Gajewicz-Skretna</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Agnieszka Gajewicz-Skretna</span></h3>
<div class="p">
<sup>§</sup>
Department
of Physical Chemistry, Faculty of Chemistry, Gdańsk University of Technology, Narutowicza 11/12, 80-233
Gdansk, Poland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gajewicz-Skretna%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Agnieszka Gajewicz-Skretna</span></a>
</div>
</div>
<sup>§</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deptu%C5%82a%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Milena Deptuła</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Milena Deptuła</span></h3>
<div class="p">
<sup>∥</sup>
Laboratory
of Tissue Engineering and Regenerative Medicine, Division of Embryology, Medical University of Gdansk, Dębinki 1, 80-211
Gdansk, Poland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deptu%C5%82a%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Milena Deptuła</span></a>
</div>
</div>
<sup>∥</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chernobrovkin%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Alexey Chernobrovkin</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Alexey Chernobrovkin</span></h3>
<div class="p">
<sup>⊥</sup>
Department
of Medical Biochemistry and Biophysics, Karolinska Institutet, Solnavägen 9, 17177
Stockholm, Sweden
</div>
<div class="p">
<sup>#</sup>
Pelago Bioscience
AB, Scheelesväg
1, 17165
Solna, Sweden
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chernobrovkin%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexey Chernobrovkin</span></a>
</div>
</div>
<sup>⊥,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pogorzelska%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Aneta Pogorzelska</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Aneta Pogorzelska</span></h3>
<div class="p">
<sup>∇</sup>
Department
of Organic Chemistry, Faculty of Pharmacy, Medical University of Gdańsk, Hallera 107, 80-416
Gdansk, Poland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pogorzelska%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Aneta Pogorzelska</span></a>
</div>
</div>
<sup>∇</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grubb%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Anders Grubb</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Anders Grubb</span></h3>
<div class="p">
<sup>○</sup>
Department
of Clinical Chemistry, Skåne University Hospital, Lund University, Klinikgatan 19, 22185
Lund, Sweden
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grubb%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anders Grubb</span></a>
</div>
</div>
<sup>○</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zubarev%20RA%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Roman A Zubarev</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Roman A Zubarev</span></h3>
<div class="p">
<sup>⊥</sup>
Department
of Medical Biochemistry and Biophysics, Karolinska Institutet, Solnavägen 9, 17177
Stockholm, Sweden
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zubarev%20RA%22%5BAuthor%5D" class="usa-link"><span class="name western">Roman A Zubarev</span></a>
</div>
</div>
<sup>⊥</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Piku%C5%82a%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Michał Pikuła</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Michał Pikuła</span></h3>
<div class="p">
<sup>∥</sup>
Laboratory
of Tissue Engineering and Regenerative Medicine, Division of Embryology, Medical University of Gdansk, Dębinki 1, 80-211
Gdansk, Poland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Piku%C5%82a%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michał Pikuła</span></a>
</div>
</div>
<sup>∥</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kasprzykowski%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Franciszek Kasprzykowski</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Franciszek Kasprzykowski</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biomedical Chemistry, Faculty of Chemistry, University of Gdańsk, Wita Stwosza 63, 80-308
Gdansk, Poland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kasprzykowski%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Franciszek Kasprzykowski</span></a>
</div>
</div>
<sup>†</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodziewicz-Motowid%C5%82o%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Sylwia Rodziewicz-Motowidło</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Sylwia Rodziewicz-Motowidło</span></h3>
<div class="p">
<sup>†</sup>
Department
of Biomedical Chemistry, Faculty of Chemistry, University of Gdańsk, Wita Stwosza 63, 80-308
Gdansk, Poland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rodziewicz-Motowid%C5%82o%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sylwia Rodziewicz-Motowidło</span></a>
</div>
</div>
<sup>†</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>†</sup>
Department
of Biomedical Chemistry, Faculty of Chemistry, University of Gdańsk, Wita Stwosza 63, 80-308
Gdansk, Poland
</div>
<div id="aff2">
<sup>‡</sup>
Department
of Molecular Biotechnology, Mossakowski
Medical Research Institute Polish Academy of Science, Wita Stwosza 63, 80-308
Gdansk, Poland
</div>
<div id="aff3">
<sup>§</sup>
Department
of Physical Chemistry, Faculty of Chemistry, Gdańsk University of Technology, Narutowicza 11/12, 80-233
Gdansk, Poland
</div>
<div id="aff4">
<sup>∥</sup>
Laboratory
of Tissue Engineering and Regenerative Medicine, Division of Embryology, Medical University of Gdansk, Dębinki 1, 80-211
Gdansk, Poland
</div>
<div id="aff5">
<sup>⊥</sup>
Department
of Medical Biochemistry and Biophysics, Karolinska Institutet, Solnavägen 9, 17177
Stockholm, Sweden
</div>
<div id="aff6">
<sup>#</sup>
Pelago Bioscience
AB, Scheelesväg
1, 17165
Solna, Sweden
</div>
<div id="aff7">
<sup>∇</sup>
Department
of Organic Chemistry, Faculty of Pharmacy, Medical University of Gdańsk, Hallera 107, 80-416
Gdansk, Poland
</div>
<div id="aff8">
<sup>○</sup>
Department
of Clinical Chemistry, Skåne University Hospital, Lund University, Klinikgatan 19, 22185
Lund, Sweden
</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
Email: <span>maria.dzierzynska@ug.edu.pl</span>.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 24; Accepted 2025 Jul 11; Revised 2025 Jul 9; Collection date 2025 Aug 5.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12332667  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40787370/" class="usa-link">40787370</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The emergence of drug-resistant Gram-positive pathogens,
particularlyand, has driven the need for novel antimicrobial
agents. This study explores 21 newly synthesized peptidomimetic analogues
of cystatin C <em>N</em>-terminal fragment, designed to enhance
bioactivity, solubility, and safety. These compounds were evaluated
for antimicrobial potency, cytotoxicity, pro-proliferative effects,
and pharmacokinetic properties. Key findings indicate that analogues
A-192 and A-164 exhibited the strongest antimicrobial effects against and . Most compounds were inactive against Gram-negative bacteria. Cytotoxicity
profiling identified several derivatives with low to moderate toxicity
and favorable pro-proliferative effects at specific concentrations.
Stability tests confirmed the robustness in aqueous and plasma environments.
Computational absorption, distribution, metabolism, excretion, and
toxicity (ADMET) modeling revealed low gastrointestinal absorption,
but favorable parameters for topical applications. Exploratory analyses
(principal component analysis (PCA) and two-way hierarchical cluster
analysis (2D-HCA)) linked structural featuressuch as branching,
molecular weight, and solubility, with biological activity. These
results support the potential of structurally optimized peptidomimetics
as targeted, topical therapeutics for Gram-positive infections and
provide a rationale for further preclinical development.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac4c/12332667/983f1f96324b/ao5c03775_0018.jpg" loading="lazy" id="abs1" height="558" width="583" alt="graphic file with name ao5c03775_0018.jpg"></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac4c/12332667/983f1f96324b/ao5c03775_0016.jpg" loading="lazy" id="tgr1" height="558" width="583" alt="graphic file with name ao5c03775_0016.jpg"></p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Antibiotics have revolutionized the treatment
of bacterial infections
since their discovery. However, shortly after the discovery of the
first antibiotic, Alexander Fleming warned that its extensive use
may result in bacterial resistance. Unfortunately,
it appears that his predictions align with current developments. Decades
of antibiotic misuse in medicine and overuse in agriculture and animal
farms have caused the appearance of bacterial strains resistant to
most, if not all, known antibiotics.<em>
−

</em> The increasing prevalence
of ESKAPE bacteriai.e., , , , , , and <em>Enterobacter</em> spp.is a growing concern,
as it may lead to infections that are difficult or even impossible
to treat. The emergence of the COVID-19
pandemic has substantially contributed to the prevailing circumstances.<em>
−

</em> Therefore, the development of new-generation antimicrobial compounds
is sorely needed if we wish to cope with infectious diseases. Most commonly used antibiotics were developed
more than 30 years ago. Despite great advances in biotechnology, very
few new antibacterial drugs have been introduced to common pharmaceutical
use since then. However, the rapid development
of artificial intelligence is opening new possibilities for drug discovery,
including the design of novel antibiotics, which could lead to groundbreaking
solutions in the search for effective antimicrobial agents.</p>
<p>Among the compounds with antibacterial activity, antimicrobial
peptides (AMPs) have garnered attention for their potent activity
against a wide range of pathogens. Their antimicrobial mechanism of
action is primarily associated with targeting cell membranes, due
to their cationic nature.<em>
,
</em> However, there is evidence
that peptides may act intracellularly, after the cell-penetrating
step, by targeting nucleic acids, proteins, proteases, and cell division
mechanics.<em>
−

</em>
</p>
<p>In the class of antimicrobial molecules, human
cystatin C may also
be included. It is a small protein whose main purpose seems to be
to protect tissues from excessive lysosomal cysteine proteases. Human
cystatin C also has antibacterial and antiviral effects. Combined
with the fact that it is present in all tissues and body fluids, this
makes it part of the first line of the human immune system.<em>
−


</em>
</p>
<p>In addition to peptides and proteins with antibacterial activity,
a large group is constituted by small antimicrobial peptidomimetics,
which offer an innovative approach combining the desirable properties
of the peptide design with the structural motif of natural peptides,
paving the way toward development of novel therapeutic agents. Peptidomimetics designed based on the <em>N</em>-terminal structure of cystatin C has been studied by our
group for some time.<em>
−


</em> In this article, we present twenty-one newly synthesized analogues
of one of the most potent peptidomimetic in this series, Cystapep
1 (A-20). Given its significant antimicrobial potential, we aimed
to optimize its structure to reduce the minimum inhibitory concentration
(MIC) values against methicillin-resistant and other Gram-positive bacteria, while maintaining low cytotoxicity
and enhancing solubility. Additionally, due to the skin-colonizing
nature of these bacteria, we evaluated the pro-proliferative effect
of these peptidomimetics on skin cells. Along with the experimental
data, we employed exploratory data analysis and computed pharmacokinetic
parameters, including absorption, distribution, metabolism, excretion,
and toxicity (ADMET), to aid in optimization and assessment of the
compounds’ potential for future application.</p></section><section id="sec2"><h2 class="pmc_sec_title">Materials and Methods</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">Synthesis</h3>
<p>The synthesis of A-20 peptidomimetic has
previously been described. Syntheses
of peptidomimetics were carried out by classical, in solution method
using (2<em>S</em>)-<em>N</em>
<sup>2</sup>-Boc-1,2-diamines
(specifically (2<em>S</em>)-<em>N</em>
<sup>2</sup>-Boc-1,2-diamino-3-methylbutane and ((2<em>S</em>)-<em>N</em>
<sup>2</sup>-Boc-1,2-diaminobutane)). The latter were obtained
from Boc-Val-OH and Boc-Abu-OH in accordance with the general procedure
presented in <a href="#sch1" class="usa-link">Scheme 
</a>. The amino acid was reduced to the corresponding alcohol using NaAlH<sub>2</sub>(OCH<sub>2</sub>CH<sub>2</sub>OCH<sub>3</sub>)<sub>2</sub> as the reducing agent (as described previously). The resulting alcohol was then converted to the mesylate
to facilitate nucleophilic substitution with the azide. Finally, conversion
to the amine hydrochloride was accomplished by hydrogenolysis. The corresponding diamine residues, introduced
using these reagents, are given in <a href="#tbl1" class="usa-link">Table 
</a> as −NH–CH­(iPr)–CH<sub>2</sub>–NH– and −NH–CH­(CH<sub>2</sub>–CH<sub>3</sub>)–CH<sub>2</sub>–NH–,
respectively. All amino acids were in the L form. The elongation from
lysine to homoarginine (described as Har in <a href="#tbl1" class="usa-link">Table 
</a>) in A-134, A-192, A-193, and A-194 was obtained
by using 3,5-dimethyl-1-pyrazolylformaminidium nitrate, detailed in
a previous article. The homologation
of Boc-Val-OH to Boc-βLeu-OH for A-152 synthesis was prepared
via an Arndt–Eistert synthesis using isobutyl chloroformate
and diazomethane to obtain a diazoketone, as represented in <a href="#sch2" class="usa-link">Scheme 
</a>. Wolff rearrangement
of the diazoketone to a carboxylic acid was achieved with water in
the presence of Ag<sub>2</sub>O as a catalyst.<em>
,
</em> The final product was coupled with amino acids and is described
as −NH–CH­(iPr)–CH<sub>2</sub>–CO–
in <a href="#tbl1" class="usa-link">Table 
</a>. <em>Z</em>(<em>o</em>-Me)-Arg-OH for A-210 was synthesized
using 2-methylbenzyl alcohol and triphosgene in the presence of pyridine
and then coupled with arginine. Other substrates were commercially
purchased. For all compounds, <em>N</em>,<em>N</em>′-dicyclohexylcarbodiimide (DCC) (Sigma-Aldrich, USA) was
used as a coupling agent, and hydroxybenzotriazole (HOBt) (Merck Millipore,
USA) was used as a racemization suppressor. The Boc-protecting group
was cleaved using anhydrous 4 M HCl in dioxane. In the case of A-198,
the benzyloxycarbonyl (<em>Z</em>) protecting group was removed
by hydrogenolysis, simultaneously with the reduction of the double
bond of cinnamic acid. Crude peptidomimetics were purified with a
high-performance liquid chromatography (HPLC) system equipped with
a Kromasil C8 (100 Å, 5 μm, 21.2 × 250 mm<sup>2</sup>) column using isocratic modes. The molecular masses were determined
using a liquid chromatography-mass spectrometry (LC-MS) system (Shimadzu,
Japan) and purity (above 95%) was ascertained by HPLC analysis in
full gradient mode.</p>
<figure class="fig xbox font-sm" id="sch1"><h4 class="obj_head">1. General Procedure for Preparation (2<em>S</em>)<em>-N</em>
<sup>2</sup>-Boc-1,2-diamines.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac4c/12332667/f126288cb277/ao5c03775_0014.jpg" loading="lazy" id="gr1" height="1053" width="3502" alt="1"></p>
<div class="p text-right font-secondary"><a href="figure/sch1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><section class="tw xbox font-sm" id="tbl1"><h4 class="obj_head">1. Synthesized New Compounds with Potential
Antimicrobial Activity<sup></sup>
.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">no</th>
<th align="center" colspan="1" rowspan="1">code</th>
<th align="center" colspan="1" rowspan="1">P1</th>
<th align="center" colspan="1" rowspan="1">P2</th>
<th align="center" colspan="1" rowspan="1">P3</th>
<th align="center" colspan="1" rowspan="1">P4</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">A-20</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Arg-</td>
<td align="left" colspan="1" rowspan="1">-Leu-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH(iPr)–CH<sub>2</sub>–NH–</td>
<td align="left" colspan="1" rowspan="1">–Cin</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">A-134</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Har</td>
<td align="left" colspan="1" rowspan="1">-Leu-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH(iPr)–CH<sub>2</sub>–NH–</td>
<td align="left" colspan="1" rowspan="1">–Cin</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">A-152</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Arg-</td>
<td align="left" colspan="1" rowspan="1">-Leu-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH(iPr)–CH<sub>2</sub>–CO–</td>
<td align="left" colspan="1" rowspan="1">–NH–CH<sub>2</sub>–CH<sub>2</sub>–C<sub>6</sub>H<sub>5</sub>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">A-164</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Arg-</td>
<td align="left" colspan="1" rowspan="1">-Leu-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH(iPr)–CH<sub>2</sub>–NH–</td>
<td align="left" colspan="1" rowspan="1">–PCA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">A-165</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Arg-</td>
<td align="left" colspan="1" rowspan="1">-Leu-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH(iPr)–CH<sub>2</sub>–NH–</td>
<td align="left" colspan="1" rowspan="1">–NAA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">A-174</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Lys-</td>
<td align="left" colspan="1" rowspan="1">-His-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH(iPr)–CH<sub>2</sub>–NH–</td>
<td align="left" colspan="1" rowspan="1">–Cin</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">A-176</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Arg-</td>
<td align="left" colspan="1" rowspan="1">-His-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH(iPr)–CH<sub>2</sub>–NH–</td>
<td align="left" colspan="1" rowspan="1">–PCA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1">A-179</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Arg-</td>
<td align="left" colspan="1" rowspan="1">-Arg-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH(iPr)–CH<sub>2</sub>–NH–</td>
<td align="left" colspan="1" rowspan="1">–PCA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">A-184</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Arg-</td>
<td align="left" colspan="1" rowspan="1">-Arg-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH(iPr)–CH<sub>2</sub>–NH–</td>
<td align="left" colspan="1" rowspan="1">–NAA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1">A-191</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Lys-</td>
<td align="left" colspan="1" rowspan="1">-Arg-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH(iPr)–CH<sub>2</sub>–NH–</td>
<td align="left" colspan="1" rowspan="1">–PCA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">A-192</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Har-</td>
<td align="left" colspan="1" rowspan="1">-Arg-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH(iPr)–CH<sub>2</sub>–NH–</td>
<td align="left" colspan="1" rowspan="1">–PCA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">A-193</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Har-</td>
<td align="left" colspan="1" rowspan="1">-His-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH(iPr)–CH<sub>2</sub>–NH–</td>
<td align="left" colspan="1" rowspan="1">–PCA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">A-194</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Har-</td>
<td align="left" colspan="1" rowspan="1">-Arg-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH(iPr)–CH<sub>2</sub>–NH–</td>
<td align="left" colspan="1" rowspan="1">–NAA</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">13</td>
<td align="left" colspan="1" rowspan="1">A-196</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Arg-</td>
<td align="left" colspan="1" rowspan="1">-Leu-</td>
<td align="left" colspan="1" rowspan="1">-Val-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH<sub>2</sub>–CH<sub>2</sub>–C<sub>6</sub>H<sub>5</sub>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">14</td>
<td align="left" colspan="1" rowspan="1">A-197</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Arg-</td>
<td align="left" colspan="1" rowspan="1">-Leu-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH(iPr)–CH<sub>2</sub>–NH–</td>
<td align="left" colspan="1" rowspan="1"> </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">A-198</td>
<td align="left" colspan="1" rowspan="1">Arg-</td>
<td align="left" colspan="1" rowspan="1">-Leu-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH(iPr)–CH<sub>2</sub>–NH–</td>
<td align="left" colspan="1" rowspan="1">–CO–CH<sub>2</sub>–CH<sub>2</sub>–CH<sub>2</sub>–C<sub>6</sub>H<sub>5</sub>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">A-209</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Arg-</td>
<td align="left" colspan="1" rowspan="1">-Leu-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH(iPr)–CH<sub>2</sub>–NH–</td>
<td align="left" colspan="1" rowspan="1">–CO–CH<sub>2</sub>–CHCH<sub>2</sub>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">17</td>
<td align="left" colspan="1" rowspan="1">A-210</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>(<em>o</em>-CH<sub>3</sub>)-Arg-</td>
<td align="left" colspan="1" rowspan="1">-Leu-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH(iPr)–CH<sub>2</sub>–NH–</td>
<td align="left" colspan="1" rowspan="1">–Cin</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">18</td>
<td align="left" colspan="1" rowspan="1">A-213</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Arg-</td>
<td align="left" colspan="1" rowspan="1">-Leu-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH(CH<sub>2</sub>–CH<sub>3</sub>)–CH<sub>2</sub>–NH–</td>
<td align="left" colspan="1" rowspan="1">–Cin</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">19</td>
<td align="left" colspan="1" rowspan="1">A-219</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Arg-</td>
<td align="left" colspan="1" rowspan="1">-His-</td>
<td align="left" colspan="1" rowspan="1">-Val-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH<sub>2</sub>–CH<sub>2</sub>–C<sub>6</sub>H<sub>5</sub>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">A-220</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Lys-</td>
<td align="left" colspan="1" rowspan="1">-His-</td>
<td align="left" colspan="1" rowspan="1">-Val-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH<sub>2</sub>–CH<sub>2</sub>–C<sub>6</sub>H<sub>5</sub>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">21</td>
<td align="left" colspan="1" rowspan="1">A-221</td>
<td align="left" colspan="1" rowspan="1">
<em>Z</em>-Arg-</td>
<td align="left" colspan="1" rowspan="1">-Arg-</td>
<td align="left" colspan="1" rowspan="1">-Val-</td>
<td align="left" colspan="1" rowspan="1">–NH–CH<sub>2</sub>–CH<sub>2</sub>–C<sub>6</sub>H<sub>5</sub>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t1fn1">
<sup>a</sup><p class="display-inline">P1–P4 are positions of modification.
Cin<em>trans</em>-cinnamyl moiety, PCAα-phenylcinnamyl
moiety, NAA1-naphthylacetic acid moiety.</p>
</div></div></section><figure class="fig xbox font-sm" id="sch2"><h4 class="obj_head">2. Synthesis of Boc-βLeu-OH for Compound A-152.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac4c/12332667/b135e000f5ec/ao5c03775_0015.jpg" loading="lazy" id="gr2" height="547" width="3500" alt="2"></p>
<div class="p text-right font-secondary"><a href="figure/sch2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec2.2"><h3 class="pmc_sec_title">Stability in Water and Presence in Human Plasma</h3>
<p>Blood
was drawn from healthy volunteers, and lithium/heparin holding tubes
were used to remove the clot by centrifugation. The plasma was aliquoted
in sterile conditions and incubated with the desired compounds for
24 h at 37 °C. The samples were prepared under sterile conditions
following the previously described protocol. The samples were collected at six time points (0, 1, 2, 3, 6, and
24 h) for determining stability in water solutions, and time points
of 0, 10, 20, 30, 40, 50, and 60 min, as well as of 2, 3, 6, and 24
h, for determining stability in human plasma. The analysis was performed
by HPLC using a Kromasil C8 column (5 μm, 90 Å, 1.0 mm
× 250 mm) monitored by a PDA or ELDS-LT II detector. For quantification,
peak areas were compared to a calibration curve calculated by Shimadzu
LCsolution Software (Kyoto, Japan). Graphs were prepared in GraphPad
Prism (Boston, USA).</p></section><section id="sec2.3"><h3 class="pmc_sec_title">Antimicrobial Assay</h3>
<p> (ATCC 29213), (ATCC 19615), (ATCC 27956), (ATCC 25922), (ATCC 6896), (ATCC 700603) and (ATCC
10145) reference strains, as well as, , , and clinical isolates, were used to test the antibacterial
activity of the peptidomimetics. Bacteria were aerobically cultured
for 18 h at 37 °C in plates with sterilized brain heart infusion
(BHI) broth (Sigma-Aldrich, USA) media mixed with bacteriological
agar (Sigma-Aldrich, USA) using the streak plate method. After 18
h, BHI broth was inoculated to 0.5 on the McFarland scale and diluted
to approximately 10<sup>5</sup> CFU/mL. To achieve a final stock concentration
of 1024 μg/mL, the peptidomimetics were dissolved in sterile
water, and 5–10% of absolute ethanol or dimethyl sulfoxide
(DMSO) was added. The antibacterial activities were determined by
using the standard broth microdilution method of the National Committee
for Clinical Laboratory Standards. The process involved adding serially
diluted peptidomimetics to consecutive wells of a 96-well microtiter
plate (Brand, Germany) holding media with microorganisms. The inoculated
plates treated with peptides were then placed in an incubator at 37
°C for 18 h. The MIC value was determined as the lowest concentration
of peptidomimetic at which there was no obvious growth (equal or more
than 90% of the untreated sample). To determine minimum bactericidal
concentration (MBC) values, 10 μL of medium from each well with
no visible growth was spread on BHI-agar plates and incubated at 37
°C for 18 h. The lowest concentration yielding growth in no more
than 3 colonies, corresponding to at least a 99.9% kill rate, was
taken as MBC. Each biological replicate included control wells with
the addition of water, DMSO, and ethanol as respective solvent controls.
All experiments were conducted with consistent adherence to growth
and sterility controls and were performed at least in technical triplicate
with three biological repeats. MICs were converted to a color scale
and displayed by using GraphPad Prism 8 (Boston, USA).</p></section><section id="sec2.4"><h3 class="pmc_sec_title">Cell Culture Conditions</h3>
<p>In our study, we used two distinct
cell types: immortalized human HaCaT keratinocytes (DKFZ, Heidelberg,
Germany) and a human dermal fibroblast cell line (46BR.1N). Both HaCaT
and 46BR.1N cells were cultured in Dulbecco’s modified Eagle’s
medium (DMEM) (Sigma-Aldrich, USA), containing 4500 μg/mL glucose,
584 μg/mL <span class="font-variant-small-caps">l</span>-glutamine, sodium pyruvate, and sodium
bicarbonate, supplemented with 10% fetal bovine serum (FBS), 100 units/mL
penicillin, and 100 μg/mL streptomycin (Sigma-Aldrich, USA).
Cultures were maintained in a humidified atmosphere with 5% CO<sub>2</sub> at 37 °C in 25 cm<sup>2</sup> culture flasks with the
medium replenished every 2 days. The condition of the cells was monitored
using optical microscopy.</p></section><section id="sec2.5"><h3 class="pmc_sec_title">Cytotoxicity Test</h3>
<p>Cell death in immortalized human
keratinocytes (HaCaT) and fibroblasts (46BR.1N) was measured by quantifying
the lactate dehydrogenase (LDH) activity using a commercial kit (Roche,
Switzerland). Cells were seeded in 96-well plates at a density of
5000 cells per well in DMEM high glucose medium supplemented with
10% FBS. After 24 h of incubation, the medium was replaced with serum-free
medium containing various concentrations of peptidomimetics (50, 100,
and 150 μg/mL). After 48 h, supernatants were collected for
LDH analysis. Plate readings were taken spectrophotometrically using
a 450 nm wavelength. Cell death measurements were normalized to an
untreated control (0%), with 1% Triton X-100 detergent used as a positive
control to induce maximum LDH release, indicating maximal cytotoxicity.
Statistical significance was determined with a Mann–Whitney <em>U</em> test with a statistical significance <em>p</em> ≤ 0.05. Analysis was performed with Statistica software (Krakow,
Poland) and graphs were prepared in GraphPad Prism (Boston, USA).</p></section><section id="sec2.6"><h3 class="pmc_sec_title">Cell Proliferation Assay</h3>
<p>Cells were seeded into 96-well
plates at a density of 5000 cells per well in a medium supplemented
with 10% fetal bovine serum (FBS). After 24 h of incubation, the medium
was replaced with serum-free HG-DMEM containing appropriate concentrations
of peptidomimetics. All solutions used in the experiments were prepared
with double-distilled water or with the addition of ethanol under
sterile conditions to ensure compatibility with the conditions of
the bacteriological assay. For 72 h of incubation, half of the medium
was replaced with fresh medium after 48 h, and cells were stimulated
with peptides for a second time. The XTT cell proliferation assay
was then performed according to the manufacturer’s instructions
(Roche, Switzerland). Cells were treated with the peptides for either
48 or 72 h, followed by the addition of XTT reagent, and the plates
were then incubated at 37 °C in a 5% CO<sub>2</sub> environment
for 4 h. Plate readings were taken using a standard plate reader at
a wavelength of 490 nm (OD = 490 nm). Cell proliferation levels were
normalized to the untreated control (100%). Statistical significance
was determined with a Mann–Whitney <em>U</em> test
with statistical significance <em>p</em> ≤ 0.05. Analysis
was performed with Statistica software, and graphs were prepared in
GraphPad Prism.</p></section><section id="sec2.7"><h3 class="pmc_sec_title">ADMET Prediction</h3>
<p>Absorption, distribution, metabolism,
excretion, and toxicity (ADMET) properties of synthesized compounds
were assessed by utilizing advanced online servers: SwissADME and pkCSM.
</p></section><section id="sec2.8"><h3 class="pmc_sec_title">Exploratory Data Analysis</h3>
<p>Exploratory data analyses
(i.e., principal component analysis (PCA) and two-way hierarchical
cluster analysis (2D-HCA)) were performed using R software (v4.0.3)
with the following packages: factoextra, caret, and ggplot2. The additional descriptors were calculated using Dragon-7
software, while more complex electronic
descriptors associated with the 3D conformation of the molecule were
computed using the MOPAC2016 program. Prior to these calculations, the geometries of all peptidomimetic
derivatives were preoptimized using the PM7 semiempirical Hamiltonian. This method was chosen for its balance between
accuracy and computational efficiency, ensuring reliable results with
minimal computational cost. A detailed
list of all calculated molecular descriptors can be found in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03775/suppl_file/ao5c03775_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>.</p></section></section><section id="sec3"><h2 class="pmc_sec_title">Results</h2>
<section id="sec3.1"><h3 class="pmc_sec_title">Design and Synthesis</h3>
<p>Based on our previous studies<em>
−


</em> we designed and synthesized a series of twenty-one compounds based
on the <em>N</em>-terminal fragment of human cystatin C (RLVG).
The scheme of conversion from Boc-Aa-OH to <em>N</em>
<sup>2</sup>-Boc-1,2-diamines, which is a core of these compounds (<a href="#fig1" class="usa-link">Figure 
</a>, P3) is represented
in <a href="#sch1" class="usa-link">Scheme 
</a>. This method
was used to synthesize 17 compounds (A-20, A-134, A-164, A-165, A-174,
A-176, A-179, A-184, A-191, A-192, A-193, A-194, A-197, A-198, A-209,
A-210, and A-213). For elongation from lysine to homoarginine (described
as Har in <a href="#tbl1" class="usa-link">Table 
</a>)
in A-134, A-192, A-193, and A-194, we used 3,5-dimethyl-1-pyrazolylformaminidium
nitrate, which was then placed in position P1 (<a href="#fig1" class="usa-link">Figure 
</a>). For preparation of Boc-βLeu-OH an
Arndt–Eistert synthesis was used, as shown in <a href="#sch2" class="usa-link">Scheme 
</a>, and this moiety was then
placed in position P3 in analogue A-152. In some cases (A-152, A-196,
A-219, A-220, and A-221) phenethylamine in position P4 was used to
mimic the structure of a cinnamic acid residue. To obtain more sterically
hindered analogues in position P4 α-phenylcinnamic acid (named
PCA in <a href="#tbl1" class="usa-link">Table 
</a>) and
1-naphthylacetic acid (named NAA in <a href="#tbl1" class="usa-link">Table 
</a>) were used (for A-164, A-165, A-176, A-179,
A-184, A-191, A-192, A-193, and A-194). To enhance the potential solubility
of compounds at position P2, basic residues such as arginine and histidine
were incorporated. All peptidomimetics were purified using a HPLC
system to at least 95% of purity and characterized by mass spectrometry
(<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03775/suppl_file/ao5c03775_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S2</a>).</p>
<figure class="fig xbox font-sm" id="fig1"><h4 class="obj_head">1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332667_ao5c03775_0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac4c/12332667/516e84feb18b/ao5c03775_0001.jpg" loading="lazy" id="gr3" height="319" width="627" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Structure of A-20, the
molecule upon which all analogues were based,
showing positions of modifications.</p></figcaption></figure></section><section id="sec3.2"><h3 class="pmc_sec_title">Hydrophobicity Parameters</h3>
<p>Hydrophobicity of antibiotics
influences their ability to attach to and penetrate bacterial cell
membranes, directly affecting their antimicrobial potency. In addition,
hydrophobicity affects the compound’s solubility, tissue distribution,
and interaction with molecular targets, playing a pivotal role in
determining pharmacokinetics and the ability to localize at infection
sites.</p>
<p>Reverse-phase chromatography was employed to assess the
hydrophobicity characteristics of the peptidomimetics. Hydrophobicity
was quantified by determining the acetonitrile content in the mobile
phase at the retention time of a peptidyl derivative (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03775/suppl_file/ao5c03775_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S2</a>). The least hydrophobic compound (40%
acetonitrile) was A-198, which aligns with the calculations of hydrophobicity
by SwissADME and pkCSM (<a href="#tbl2" class="usa-link">Table 
</a>). Next in the hydrophobicity sequence were A-197, A-220,
A-219, A-174, and A-221, eluting with 42.5–43.9% of acetonitrile.
Then there were followed by A-191, A-194, A-184, A-192, A-193, A-209,
and A-179, with the acetonitrile percentage of 44.8–49.8. The
compounds A-176, A-20, A-152, A-134, and A-210 needed over 50% of
acetonitrile to be eluted from the column. The compounds with the
poorest solubility and most hydrophobic properties were A-196, A-164,
and A-165, all of which needed more than 60% of acetonitrile to be
eluted from the column.</p>
<section class="tw xbox font-sm" id="tbl2"><h4 class="obj_head">2. ADMET Parameters Calculated by Algorithms
SwissADME and pkCSMY for All 22 Compounds.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">descriptor</th>
<th align="center" colspan="1" rowspan="1">A-20</th>
<th align="center" colspan="1" rowspan="1">A-134</th>
<th align="center" colspan="1" rowspan="1">A-152</th>
<th align="center" colspan="1" rowspan="1">A-164</th>
<th align="center" colspan="1" rowspan="1">A-165</th>
<th align="center" colspan="1" rowspan="1">A-174</th>
<th align="center" colspan="1" rowspan="1">A-176</th>
<th align="center" colspan="1" rowspan="1">A-179</th>
<th align="center" colspan="1" rowspan="1">A-184</th>
<th align="center" colspan="1" rowspan="1">A-191</th>
<th align="center" colspan="1" rowspan="1">A-192</th>
<th align="center" colspan="1" rowspan="1">A-193</th>
</tr></thead>
<tbody>
<tr><td colspan="13" align="center" rowspan="1">Adsorption</td></tr>
<tr><td colspan="13" align="left" rowspan="1">lipophilicity</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Log <em>P</em>
<sub>o/w</sub> (iLOGP)</td>
<td align="left" colspan="1" rowspan="1">2.87</td>
<td align="left" colspan="1" rowspan="1">4.52</td>
<td align="left" colspan="1" rowspan="1">3.19</td>
<td align="left" colspan="1" rowspan="1">3.99</td>
<td align="left" colspan="1" rowspan="1">4.49</td>
<td align="left" colspan="1" rowspan="1">3.74</td>
<td align="left" colspan="1" rowspan="1">–2.61</td>
<td align="left" colspan="1" rowspan="1">2.84</td>
<td align="left" colspan="1" rowspan="1">1.53</td>
<td align="left" colspan="1" rowspan="1">3.85</td>
<td align="left" colspan="1" rowspan="1">3.02</td>
<td align="left" colspan="1" rowspan="1">3.30</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Log <em>P</em>
<sub>o/w</sub> (XLOGP3)</td>
<td align="left" colspan="1" rowspan="1">4.23</td>
<td align="left" colspan="1" rowspan="1">4.58</td>
<td align="left" colspan="1" rowspan="1">4.03</td>
<td align="left" colspan="1" rowspan="1">5.86</td>
<td align="left" colspan="1" rowspan="1">4.99</td>
<td align="left" colspan="1" rowspan="1">3.26</td>
<td align="left" colspan="1" rowspan="1">4.38</td>
<td align="left" colspan="1" rowspan="1">3.72</td>
<td align="left" colspan="1" rowspan="1">2.85</td>
<td align="left" colspan="1" rowspan="1">4.31</td>
<td align="left" colspan="1" rowspan="1">4.08</td>
<td align="left" colspan="1" rowspan="1">4.66</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Log <em>P</em>
<sub>o/w</sub> (WLOGP)</td>
<td align="left" colspan="1" rowspan="1">2.79</td>
<td align="left" colspan="1" rowspan="1">3.18</td>
<td align="left" colspan="1" rowspan="1">2.81</td>
<td align="left" colspan="1" rowspan="1">4.31</td>
<td align="left" colspan="1" rowspan="1">3.58</td>
<td align="left" colspan="1" rowspan="1">2.57</td>
<td align="left" colspan="1" rowspan="1">3.23</td>
<td align="left" colspan="1" rowspan="1">2.53</td>
<td align="left" colspan="1" rowspan="1">1.79</td>
<td align="left" colspan="1" rowspan="1">3.40</td>
<td align="left" colspan="1" rowspan="1">2.92</td>
<td align="left" colspan="1" rowspan="1">3.62</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Log <em>P</em>
<sub>o/w</sub> (MLOGP)</td>
<td align="left" colspan="1" rowspan="1">1.85</td>
<td align="left" colspan="1" rowspan="1">2.03</td>
<td align="left" colspan="1" rowspan="1">1.92</td>
<td align="left" colspan="1" rowspan="1">2.70</td>
<td align="left" colspan="1" rowspan="1">2.28</td>
<td align="left" colspan="1" rowspan="1">0.54</td>
<td align="left" colspan="1" rowspan="1">1.06</td>
<td align="left" colspan="1" rowspan="1">1.62</td>
<td align="left" colspan="1" rowspan="1">1.20</td>
<td align="left" colspan="1" rowspan="1">1.95</td>
<td align="left" colspan="1" rowspan="1">1.79</td>
<td align="left" colspan="1" rowspan="1">1.23</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Log <em>P</em>
<sub>o/w</sub> (SILICOS-IT)</td>
<td align="left" colspan="1" rowspan="1">4.12</td>
<td align="left" colspan="1" rowspan="1">4.54</td>
<td align="left" colspan="1" rowspan="1">4.30</td>
<td align="left" colspan="1" rowspan="1">5.47</td>
<td align="left" colspan="1" rowspan="1">4.94</td>
<td align="left" colspan="1" rowspan="1">4.50</td>
<td align="left" colspan="1" rowspan="1">4.64</td>
<td align="left" colspan="1" rowspan="1">3.75</td>
<td align="left" colspan="1" rowspan="1">3.23</td>
<td align="left" colspan="1" rowspan="1">4.82</td>
<td align="left" colspan="1" rowspan="1">4.18</td>
<td align="left" colspan="1" rowspan="1">5.18</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">consensus Log <em>P</em>
<sub>o/w</sub>
</td>
<td align="left" colspan="1" rowspan="1">3.17</td>
<td align="left" colspan="1" rowspan="1">3.77</td>
<td align="left" colspan="1" rowspan="1">3.25</td>
<td align="left" colspan="1" rowspan="1">4.47</td>
<td align="left" colspan="1" rowspan="1">4.06</td>
<td align="left" colspan="1" rowspan="1">2.92</td>
<td align="left" colspan="1" rowspan="1">2.14</td>
<td align="left" colspan="1" rowspan="1">2.89</td>
<td align="left" colspan="1" rowspan="1">2.12</td>
<td align="left" colspan="1" rowspan="1">3.66</td>
<td align="left" colspan="1" rowspan="1">3.20</td>
<td align="left" colspan="1" rowspan="1">3.60</td>
</tr>
<tr><td colspan="13" align="left" rowspan="1">water solubility</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Log <em>S</em> (ESOL)</td>
<td align="left" colspan="1" rowspan="1">–5.06</td>
<td align="left" colspan="1" rowspan="1">–5.29</td>
<td align="left" colspan="1" rowspan="1">–4.88</td>
<td align="left" colspan="1" rowspan="1">–6.55</td>
<td align="left" colspan="1" rowspan="1">–5.82</td>
<td align="left" colspan="1" rowspan="1">–4.57</td>
<td align="left" colspan="1" rowspan="1">–5.83</td>
<td align="left" colspan="1" rowspan="1">–5.26</td>
<td align="left" colspan="1" rowspan="1">–4.53</td>
<td align="left" colspan="1" rowspan="1">–5.53</td>
<td align="left" colspan="1" rowspan="1">–5.50</td>
<td align="left" colspan="1" rowspan="1">–6.02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">solubility [μg/mL]</td>
<td align="left" colspan="1" rowspan="1">5.59</td>
<td align="left" colspan="1" rowspan="1">3.31</td>
<td align="left" colspan="1" rowspan="1">8.48</td>
<td align="left" colspan="1" rowspan="1">0.20</td>
<td align="left" colspan="1" rowspan="1">1.02</td>
<td align="left" colspan="1" rowspan="1">17.2</td>
<td align="left" colspan="1" rowspan="1">1.10</td>
<td align="left" colspan="1" rowspan="1">4.16</td>
<td align="left" colspan="1" rowspan="1">21.4</td>
<td align="left" colspan="1" rowspan="1">2.14</td>
<td align="left" colspan="1" rowspan="1">2.42</td>
<td align="left" colspan="1" rowspan="1">
<span class="text-strike">4</span>0.71</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">class</td>
<td align="left" colspan="1" rowspan="1">moderately
soluble</td>
<td align="left" colspan="1" rowspan="1">moderately soluble</td>
<td align="left" colspan="1" rowspan="1">moderately
soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">moderately soluble</td>
<td align="left" colspan="1" rowspan="1">moderately soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">moderately soluble</td>
<td align="left" colspan="1" rowspan="1">moderately soluble</td>
<td align="left" colspan="1" rowspan="1">moderately soluble</td>
<td align="left" colspan="1" rowspan="1">moderately soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Log <em>S</em> (Ali)</td>
<td align="left" colspan="1" rowspan="1">–7.88</td>
<td align="left" colspan="1" rowspan="1">–8.24</td>
<td align="left" colspan="1" rowspan="1">–7.67</td>
<td align="left" colspan="1" rowspan="1">–9.57</td>
<td align="left" colspan="1" rowspan="1">–8.67</td>
<td align="left" colspan="1" rowspan="1">–6.72</td>
<td align="left" colspan="1" rowspan="1">–8.64</td>
<td align="left" colspan="1" rowspan="1">–8.65</td>
<td align="left" colspan="1" rowspan="1">–7.75</td>
<td align="left" colspan="1" rowspan="1">–8.51</td>
<td align="left" colspan="1" rowspan="1">–9.02</td>
<td align="left" colspan="1" rowspan="1">–8.93</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">solubility [μg/mL]</td>
<td align="left" colspan="1" rowspan="1">8.40 × 10<sup>–3</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.72 ×
10<sup>–3</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.36 × 10<sup>–2</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.91 × 10<sup>–4</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.45
× 10<sup>–3</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.20 × 10<sup>–1</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.70 × 10<sup>–3</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.69
× 10<sup>–3</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.28 × 10<sup>–2</sup>
</td>
<td align="left" colspan="1" rowspan="1">2.26 × 10<sup>–3</sup>
</td>
<td align="left" colspan="1" rowspan="1">7.29
× 10<sup>–4</sup>
</td>
<td align="left" colspan="1" rowspan="1">8.87 × 10<sup>–4</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">class</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly
soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly
soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Log <em>S</em> (SILICOS-IT)</td>
<td align="left" colspan="1" rowspan="1">–8.04</td>
<td align="left" colspan="1" rowspan="1">–8.79</td>
<td align="left" colspan="1" rowspan="1">–9.12</td>
<td align="left" colspan="1" rowspan="1">–10.81</td>
<td align="left" colspan="1" rowspan="1">–10.34</td>
<td align="left" colspan="1" rowspan="1">–9.20</td>
<td align="left" colspan="1" rowspan="1">–10.64</td>
<td align="left" colspan="1" rowspan="1">–10.52</td>
<td align="left" colspan="1" rowspan="1">–10.05</td>
<td align="left" colspan="1" rowspan="1">–10.82</td>
<td align="left" colspan="1" rowspan="1">–10.90</td>
<td align="left" colspan="1" rowspan="1">–11.69</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">solubility [μg/mL]</td>
<td align="left" colspan="1" rowspan="1">2.52 × 10<sup>–3</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.06 × 10<sup>–3</sup>
</td>
<td align="left" colspan="1" rowspan="1">4.86
× 10<sup>–4</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.11 × 10<sup>–5</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.09 × 10<sup>–5</sup>
</td>
<td align="left" colspan="1" rowspan="1">4.00
× 10<sup>–4</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.67 × 10<sup>–5</sup>
</td>
<td align="left" colspan="1" rowspan="1">2.31 × 10<sup>–5</sup>
</td>
<td align="left" colspan="1" rowspan="1">6.43
× 10<sup>–5</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.11 × 10<sup>–5</sup>
</td>
<td align="left" colspan="1" rowspan="1">9.75 × 10<sup>–6</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.55
× 10<sup>–6</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">cass</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly
soluble</td>
<td align="left" colspan="1" rowspan="1">insoluble</td>
<td align="left" colspan="1" rowspan="1">insoluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">insoluble</td>
<td align="left" colspan="1" rowspan="1">insoluble</td>
<td align="left" colspan="1" rowspan="1">insoluble</td>
<td align="left" colspan="1" rowspan="1">insoluble</td>
<td align="left" colspan="1" rowspan="1">insoluble</td>
<td align="left" colspan="1" rowspan="1">insoluble</td>
</tr>
<tr><td colspan="13" align="left" rowspan="1">pharmacokinetics</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">GI absorption</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">P-glycoprotein substrate</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Log <em>K</em>
<sub>p</sub> (skin
permeation)
[cm/s]</td>
<td align="left" colspan="1" rowspan="1">–7.18</td>
<td align="left" colspan="1" rowspan="1">–7.07</td>
<td align="left" colspan="1" rowspan="1">–7.33</td>
<td align="left" colspan="1" rowspan="1">–6.48</td>
<td align="left" colspan="1" rowspan="1">–6.87</td>
<td align="left" colspan="1" rowspan="1">–7.84</td>
<td align="left" colspan="1" rowspan="1">–7.68</td>
<td align="left" colspan="1" rowspan="1">–8.26</td>
<td align="left" colspan="1" rowspan="1">–8.65</td>
<td align="left" colspan="1" rowspan="1">–7.67</td>
<td align="left" colspan="1" rowspan="1">–8.09</td>
<td align="left" colspan="1" rowspan="1">–7.57</td>
</tr>
<tr><td colspan="13" align="center" rowspan="1">Distribution</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">BBB permeant</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">VDss (human) [log L/kg]</td>
<td align="left" colspan="1" rowspan="1">–0.226</td>
<td align="left" colspan="1" rowspan="1">–0.198</td>
<td align="left" colspan="1" rowspan="1">–0.267</td>
<td align="left" colspan="1" rowspan="1">–0.436</td>
<td align="left" colspan="1" rowspan="1">0.106</td>
<td align="left" colspan="1" rowspan="1">–0.237</td>
<td align="left" colspan="1" rowspan="1">–0.116</td>
<td align="left" colspan="1" rowspan="1">–0.244</td>
<td align="left" colspan="1" rowspan="1">–0.246</td>
<td align="left" colspan="1" rowspan="1">–0.251</td>
<td align="left" colspan="1" rowspan="1">–0.205</td>
<td align="left" colspan="1" rowspan="1">–0.089</td>
</tr>
<tr><td colspan="13" align="center" rowspan="1">Metabolism</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">CYP2D6 substrate</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CYP3A4 substrate</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CYP1A2 inhibitor</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CYP2C19 inhibitor</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CYP2C9 inhibitor</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CYP2D6 inhibitor</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CYP3A4 inhibitor</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
<tr><td colspan="13" align="center" rowspan="1">Excretion</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">total clearance [log mL/min/kg]</td>
<td align="left" colspan="1" rowspan="1">0.094</td>
<td align="left" colspan="1" rowspan="1">0.116</td>
<td align="left" colspan="1" rowspan="1">0.193</td>
<td align="left" colspan="1" rowspan="1">–0.36</td>
<td align="left" colspan="1" rowspan="1">0.293</td>
<td align="left" colspan="1" rowspan="1">0.577</td>
<td align="left" colspan="1" rowspan="1">0.204</td>
<td align="left" colspan="1" rowspan="1">–0.056</td>
<td align="left" colspan="1" rowspan="1">0.157</td>
<td align="left" colspan="1" rowspan="1">0.127</td>
<td align="left" colspan="1" rowspan="1">–0.065</td>
<td align="left" colspan="1" rowspan="1">0.206</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">renal OCT2 substrate</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">yes</td>
</tr>
<tr><td colspan="13" align="center" rowspan="1">Toxicity</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">AMES toxicity</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">hepatotoxicity</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">skin sensitization</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
</tbody>
</table></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">descriptor</th>
<th align="center" colspan="1" rowspan="1">A-194</th>
<th align="center" colspan="1" rowspan="1">A-196</th>
<th align="center" colspan="1" rowspan="1">A-197</th>
<th align="center" colspan="1" rowspan="1">A-198</th>
<th align="center" colspan="1" rowspan="1">A-209</th>
<th align="center" colspan="1" rowspan="1">A-210</th>
<th align="center" colspan="1" rowspan="1">A-213</th>
<th align="center" colspan="1" rowspan="1">A-219</th>
<th align="center" colspan="1" rowspan="1">A-220</th>
<th align="center" colspan="1" rowspan="1">A-221</th>
</tr></thead>
<tbody>
<tr><td colspan="11" align="center" rowspan="1">Adsorption</td></tr>
<tr><td colspan="11" align="left" rowspan="1">lipophilicity</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Log <em>P</em>
<sub>o/w</sub> (iLOGP)</td>
<td align="left" colspan="1" rowspan="1">2.86</td>
<td align="left" colspan="1" rowspan="1">2.72</td>
<td align="left" colspan="1" rowspan="1">2.77</td>
<td align="left" colspan="1" rowspan="1">2.67</td>
<td align="left" colspan="1" rowspan="1">3.54</td>
<td align="left" colspan="1" rowspan="1">4.04</td>
<td align="left" colspan="1" rowspan="1">3.95</td>
<td align="left" colspan="1" rowspan="1">2.08</td>
<td align="left" colspan="1" rowspan="1">2.9</td>
<td align="left" colspan="1" rowspan="1">1.74</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Log <em>P</em>
<sub>o/w</sub> (XLOGP3)</td>
<td align="left" colspan="1" rowspan="1">3.27</td>
<td align="left" colspan="1" rowspan="1">4.10</td>
<td align="left" colspan="1" rowspan="1">1.89</td>
<td align="left" colspan="1" rowspan="1">1.88</td>
<td align="left" colspan="1" rowspan="1">2.89</td>
<td align="left" colspan="1" rowspan="1">4.20</td>
<td align="left" colspan="1" rowspan="1">3.79</td>
<td align="left" colspan="1" rowspan="1">2.54</td>
<td align="left" colspan="1" rowspan="1">3.13</td>
<td align="left" colspan="1" rowspan="1">1.96</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Log <em>P</em>
<sub>o/w</sub> (WLOGP)</td>
<td align="left" colspan="1" rowspan="1">2.26</td>
<td align="left" colspan="1" rowspan="1">2.42</td>
<td align="left" colspan="1" rowspan="1">1.02</td>
<td align="left" colspan="1" rowspan="1">1.00</td>
<td align="left" colspan="1" rowspan="1">1.76</td>
<td align="left" colspan="1" rowspan="1">2.79</td>
<td align="left" colspan="1" rowspan="1">2.54</td>
<td align="left" colspan="1" rowspan="1">1.34</td>
<td align="left" colspan="1" rowspan="1">2.21</td>
<td align="left" colspan="1" rowspan="1">0.64</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Log <em>P</em>
<sub>o/w</sub> (MLOGP)</td>
<td align="left" colspan="1" rowspan="1">1.47</td>
<td align="left" colspan="1" rowspan="1">1.74</td>
<td align="left" colspan="1" rowspan="1">0.83</td>
<td align="left" colspan="1" rowspan="1">0.99</td>
<td align="left" colspan="1" rowspan="1">1.13</td>
<td align="left" colspan="1" rowspan="1">1.55</td>
<td align="left" colspan="1" rowspan="1">1.67</td>
<td align="left" colspan="1" rowspan="1">0.09</td>
<td align="left" colspan="1" rowspan="1">0.43</td>
<td align="left" colspan="1" rowspan="1">0.66</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Log <em>P</em>
<sub>o/w</sub> (SILICOS-IT)</td>
<td align="left" colspan="1" rowspan="1">3.23</td>
<td align="left" colspan="1" rowspan="1">3.87</td>
<td align="left" colspan="1" rowspan="1">1.83</td>
<td align="left" colspan="1" rowspan="1">1.69</td>
<td align="left" colspan="1" rowspan="1">3.17</td>
<td align="left" colspan="1" rowspan="1">4.24</td>
<td align="left" colspan="1" rowspan="1">3.86</td>
<td align="left" colspan="1" rowspan="1">3.19</td>
<td align="left" colspan="1" rowspan="1">4.25</td>
<td align="left" colspan="1" rowspan="1">2.16</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">consensus Log <em>P</em>
<sub>o/w</sub>
</td>
<td align="left" colspan="1" rowspan="1">2.62</td>
<td align="left" colspan="1" rowspan="1">2.97</td>
<td align="left" colspan="1" rowspan="1">1.67</td>
<td align="left" colspan="1" rowspan="1">1.84</td>
<td align="left" colspan="1" rowspan="1">2.50</td>
<td align="left" colspan="1" rowspan="1">3.36</td>
<td align="left" colspan="1" rowspan="1">3.26</td>
<td align="left" colspan="1" rowspan="1">1.85</td>
<td align="left" colspan="1" rowspan="1">2.59</td>
<td align="left" colspan="1" rowspan="1">1.43</td>
</tr>
<tr><td colspan="11" align="left" rowspan="1">water solubility</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">Log <em>S</em> (ESOL)</td>
<td align="left" colspan="1" rowspan="1">–4.79</td>
<td align="left" colspan="1" rowspan="1">–4.90</td>
<td align="left" colspan="1" rowspan="1">–2.97</td>
<td align="left" colspan="1" rowspan="1">–2.95</td>
<td align="left" colspan="1" rowspan="1">–3.74</td>
<td align="left" colspan="1" rowspan="1">–5.15</td>
<td align="left" colspan="1" rowspan="1">–4.70</td>
<td align="left" colspan="1" rowspan="1">–4.14</td>
<td align="left" colspan="1" rowspan="1">–4.42</td>
<td align="left" colspan="1" rowspan="1">–3.61</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">solubility [μg/mL]</td>
<td align="left" colspan="1" rowspan="1">1.16</td>
<td align="left" colspan="1" rowspan="1">7.78</td>
<td align="left" colspan="1" rowspan="1">543</td>
<td align="left" colspan="1" rowspan="1">564</td>
<td align="left" colspan="1" rowspan="1">104</td>
<td align="left" colspan="1" rowspan="1">4.72</td>
<td align="left" colspan="1" rowspan="1">12.5</td>
<td align="left" colspan="1" rowspan="1">46.9</td>
<td align="left" colspan="1" rowspan="1">23.8</td>
<td align="left" colspan="1" rowspan="1">163</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">class</td>
<td align="left" colspan="1" rowspan="1">moderately soluble</td>
<td align="left" colspan="1" rowspan="1">moderately soluble</td>
<td align="left" colspan="1" rowspan="1">soluble</td>
<td align="left" colspan="1" rowspan="1">soluble</td>
<td align="left" colspan="1" rowspan="1">soluble</td>
<td align="left" colspan="1" rowspan="1">moderately soluble</td>
<td align="left" colspan="1" rowspan="1">moderately
soluble</td>
<td align="left" colspan="1" rowspan="1">moderately soluble</td>
<td align="left" colspan="1" rowspan="1">moderately
soluble</td>
<td align="left" colspan="1" rowspan="1">soluble</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Log <em>S</em> (Ali)</td>
<td align="left" colspan="1" rowspan="1">–8.81</td>
<td align="left" colspan="1" rowspan="1">–7.74</td>
<td align="left" colspan="1" rowspan="1">–5.39</td>
<td align="left" colspan="1" rowspan="1">–5.18</td>
<td align="left" colspan="1" rowspan="1">6.49</td>
<td align="left" colspan="1" rowspan="1">–8.04</td>
<td align="left" colspan="1" rowspan="1">–7.42</td>
<td align="left" colspan="1" rowspan="1">–6.73</td>
<td align="left" colspan="1" rowspan="1">–6.59</td>
<td align="left" colspan="1" rowspan="1">–6.82</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">solubility [μg/mL]</td>
<td align="left" colspan="1" rowspan="1">4.71 × 10<sup>–3</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.12 × 10<sup>–2</sup>
</td>
<td align="left" colspan="1" rowspan="1">2.08</td>
<td align="left" colspan="1" rowspan="1">3.31</td>
<td align="left" colspan="1" rowspan="1">1.86 × 10<sup>–1</sup>
</td>
<td align="left" colspan="1" rowspan="1">6.05 × 10<sup>–3</sup>
</td>
<td align="left" colspan="1" rowspan="1">2.35 ×
10<sup>–2</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.21 × 10<sup>–1</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.61 × 10<sup>–1</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.00
× 10<sup>–1</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">class</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">moderately
soluble</td>
<td align="left" colspan="1" rowspan="1">moderately soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly
soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Log <em>S</em> (SILICOS-IT)</td>
<td align="left" colspan="1" rowspan="1">–10.05</td>
<td align="left" colspan="1" rowspan="1">–8.73</td>
<td align="left" colspan="1" rowspan="1">–5.60</td>
<td align="left" colspan="1" rowspan="1">–6.27</td>
<td align="left" colspan="1" rowspan="1">–6.73</td>
<td align="left" colspan="1" rowspan="1">–8.49</td>
<td align="left" colspan="1" rowspan="1">–8.39</td>
<td align="left" colspan="1" rowspan="1">–9.23</td>
<td align="left" colspan="1" rowspan="1">–9.53</td>
<td align="left" colspan="1" rowspan="1">–8.45</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">solubility [μg/mL]</td>
<td align="left" colspan="1" rowspan="1">6.43 × 10<sup>–5</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.15 ×
10<sup>–3</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.26</td>
<td align="left" colspan="1" rowspan="1">2.72 ×
10<sup>–1</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.07 × 10<sup>–1</sup>
</td>
<td align="left" colspan="1" rowspan="1">2.17 × 10<sup>–3</sup>
</td>
<td align="left" colspan="1" rowspan="1">2.53
× 10<sup>–3</sup>
</td>
<td align="left" colspan="1" rowspan="1">3.80 × 10<sup>–4</sup>
</td>
<td align="left" colspan="1" rowspan="1">1.83 × 10<sup>–4</sup>
</td>
<td align="left" colspan="1" rowspan="1">2.39
× 10<sup>–3</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">cass</td>
<td align="left" colspan="1" rowspan="1">insoluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">moderately
soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly soluble</td>
<td align="left" colspan="1" rowspan="1">poorly
soluble</td>
<td align="left" colspan="1" rowspan="1">poorly solule</td>
<td align="left" colspan="1" rowspan="1">poorly solubly</td>
</tr>
<tr><td colspan="11" align="left" rowspan="1">pharmacokinetics</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">GI absorption</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
<td align="left" colspan="1" rowspan="1">low</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">P-glycoprotein
substrate</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Log <em>K</em>
<sub>p</sub> (skin permeation)
[cm/s]</td>
<td align="left" colspan="1" rowspan="1">–8.35</td>
<td align="left" colspan="1" rowspan="1">–7.19</td>
<td align="left" colspan="1" rowspan="1">–8.04</td>
<td align="left" colspan="1" rowspan="1">–8.04</td>
<td align="left" colspan="1" rowspan="1">–7.75</td>
<td align="left" colspan="1" rowspan="1">–7.38</td>
<td align="left" colspan="1" rowspan="1">–7.40</td>
<td align="left" colspan="1" rowspan="1">–8.45</td>
<td align="left" colspan="1" rowspan="1">–7.86</td>
<td align="left" colspan="1" rowspan="1">–8.98</td>
</tr>
<tr><td colspan="11" align="center" rowspan="1">Distribution</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">BBB permeant</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">VDss (human) [log L/kg]</td>
<td align="left" colspan="1" rowspan="1">–0.243</td>
<td align="left" colspan="1" rowspan="1">–0.623</td>
<td align="left" colspan="1" rowspan="1">–0.695</td>
<td align="left" colspan="1" rowspan="1">0.482</td>
<td align="left" colspan="1" rowspan="1">–0.757</td>
<td align="left" colspan="1" rowspan="1">–0.222</td>
<td align="left" colspan="1" rowspan="1">–0.205</td>
<td align="left" colspan="1" rowspan="1">–0.309</td>
<td align="left" colspan="1" rowspan="1">–0.473</td>
<td align="left" colspan="1" rowspan="1">–0.532</td>
</tr>
<tr><td colspan="11" align="center" rowspan="1">Metabolism</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">CYP2D6 substrate</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CYP3A4 substrate</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CYP1A2 inhibitor</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CYP2C19 inhibitor</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CYP2C9 inhibitor</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CYP2D6 inhibitor</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CYP3A4 inhibitor</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
<tr><td colspan="11" align="center" rowspan="1">Excretion</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">total clearance [log mL/min/kg]</td>
<td align="left" colspan="1" rowspan="1">0.023</td>
<td align="left" colspan="1" rowspan="1">0.006</td>
<td align="left" colspan="1" rowspan="1">–0.018</td>
<td align="left" colspan="1" rowspan="1">0.458</td>
<td align="left" colspan="1" rowspan="1">0.106</td>
<td align="left" colspan="1" rowspan="1">–0.024</td>
<td align="left" colspan="1" rowspan="1">0.265</td>
<td align="left" colspan="1" rowspan="1">0.212</td>
<td align="left" colspan="1" rowspan="1">0.422</td>
<td align="left" colspan="1" rowspan="1">–0.346</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">renal OCT2 substrate</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
<tr><td colspan="11" align="center" rowspan="1">Toxicity</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">AMES
toxicity</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">hepatotoxicity</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
<td align="left" colspan="1" rowspan="1">yes</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">skin sensitization</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
<td align="left" colspan="1" rowspan="1">no</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec3.3"><h3 class="pmc_sec_title">Stability in Water and Presence in Human Plasma</h3>
<p>Stability
in water and human plasma is crucial for peptides and peptidomimetics,
as it influences their bioavailability, half-life, and therapeutic
efficacy. High stability ensures that these molecules resist enzymatic
degradation and hydrolysis, enabling sustained biological activity
and improved pharmacokinetic profiles. For stability determination,
peptidomimetics were incubated both in water and in human plasma.
The samples were collected at 0, 1, 2, 3, 6, and 24 h. The peptidomimetics
were stable in water for at least 24 h (<a href="#fig2" class="usa-link">Figure 
</a>pink), and almost all peptidomimetics
were stable in human plasma for at least 24 h (<a href="#fig2" class="usa-link">Figure 
</a>green). Only A-196, A-219, A-220,
and A-221 showed slight disappearance from plasma, yet still being
85–90% present. Exemplary chromatograms can be found in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03775/suppl_file/ao5c03775_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S1</a>.</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332667_ao5c03775_0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac4c/12332667/0fcc274f770c/ao5c03775_0002.jpg" loading="lazy" id="gr4" height="298" width="627" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Stability of peptidomimetic in water after
24 h of incubation and
detection of the quantity presence of peptidomimetic in human plasma
during 24 h of incubation.</p></figcaption></figure></section><section id="sec3.4"><h3 class="pmc_sec_title">Antimicrobial Activity</h3>
<p>
<a href="#fig3" class="usa-link">Figure 
</a> shows a heatmap of some MIC peptidomimetic
values across different bacterial strains; a full list of MIC and
MBC values is provided in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03775/suppl_file/ao5c03775_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Information</a> (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03775/suppl_file/ao5c03775_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S3</a>).</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac4c/12332667/9b6ae00ea2ff/ao5c03775_0003.jpg" loading="lazy" id="gr5" height="816" width="661" alt="3"></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Heatmap of minimum inhibitory
concentration (MIC) values for peptidomimetics
across different bacterial strains. The color scale indicates the
MIC values in μg/mL, with red representing lower MIC values
(more potent inhibition) and blue representing higher MIC values (less
potent inhibition).</p></figcaption></figure><p>From the heatmap it is easily seen that compounds
A-192 and A-164
display the most promising antibacterial properties, making them strong
candidates for further investigation. A-192 was most potent against strain, but was also potent against , and . A-164 was active
for both and , , and . Next active compound
was A-191, the activity of which was mostly on and . A-210 seemed to
be active against , while
A-213 was active against .
In contrast, the activity of the tested compounds against Gram-negative
bacteria, including , , , and , was largely absent,
with MIC values exceeding 512 μg/mL in most cases. Some compounds,
such as A-220 and A-221, showed little to no activity across all tested
bacterial strains, indicating their limited therapeutic potential.</p></section><section id="sec3.5"><h3 class="pmc_sec_title">ADMET Studies</h3>
<p>Computed absorption, distribution, metabolism,
excretion, and toxicity (ADMET) predictions were calculated for potential
drugs to evaluate their pharmacokinetic and toxicological properties
before experimental testing. These computational models help identify
candidates with favorable drug-like characteristics, reducing costs
and time in drug development by filtering out compounds likely to
exhibit poor bioavailability or adverse effects. A web tool called
SwissADME was used for evaluating the pharmacokinetic properties of
small molecules. The second tool used was pkCSM, which has a wider
spectrum of parameters and completes the missing data. The ADMET properties
of all compounds are shown in <a href="#tbl2" class="usa-link">Table 
</a>.</p></section><section id="sec3.6"><h3 class="pmc_sec_title">Cytotoxicity Assay of Peptidomimetics</h3>
<p>An LDH assay
was performed to quantify cytotoxicity induced by potential drug candidates,
as increased LDH levels in the extracellular environment indicate
cell damage or death.</p>
<p>The cytotoxicity of the tested compounds
was assessed within 48 h of incubation with cells (keratinocytes and
fibroblasts) and compared to the 1% Triton X-100 control. The results
are shown in <a href="#fig4" class="usa-link">Figure 
</a>. The cytotoxicity of A-20 was previously reported. Compounds A-174 and A-197 showed no cytotoxicity. Compound
A-191 showed a high cytotoxic effect only to fibroblasts at the highest
tested concentration. At lower concentrations, this effect faded away.
Compound A-196 displayed a slight cytotoxic effect (∼20%) for
keratinocytes at a concentration of 150 μg/mL. Compound A-198
showed mild cell toxicity (∼20%) specifically toward fibroblasts.
Compound A-213 exerted cytotoxicity (∼10%) at a concentration
of 150 μg/mL. A-193 seemed to be cytotoxic only at a concentration
of 150 μg/mL, and it was more cytotoxic to keratinocytes than
to fibroblasts at this concentration. Both A-219 and A-221 exhibited
slight cytotoxicity toward fibroblasts across all concentrations,
while demonstrating no adverse effects on keratinocytes. A-194, A-209,
and A-220 displayed mild toxicity (∼10–15%) toward both
cell lines. However, A-194 exhibited a notable increase to over 40–50%
toxicity at a concentration of 150 μg/mL. A-134 and A-152 exerted
cytotoxicity at every studied concentration, although not exceeding
50%. A-176 had a negative effect on both keratinocytes (over 50%)
and fibroblasts, and this effect became stronger with increasing concentration.
A-210 seemed to have no negative effect at 50 μg/mL concentration,
but a cytotoxic effect, which was stronger toward keratinocytes, appeared
at 100 μg/mL. Both A-164 demonstrated cytotoxic effects across
all of the tested concentrations. The cytotoxic effect of A-165 was
stronger with increasing concentration. A-179 was cytotoxic to keratinocytes
at all concentrations; however, this effect was less pronounced in
fibroblasts. A-192 seemed to have a more negative effect on keratinocytes
than on fibroblasts at all concentrations (∼20% cytotoxicity).
A-184 seemed to have a negative effect on both cell lines, and its
toxic effect was over 60%.</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332667_ao5c03775_0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac4c/12332667/246b030d54ad/ao5c03775_0004.jpg" loading="lazy" id="gr6" height="1066" width="668" alt="4"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332667_ao5c03775_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac4c/12332667/22fd5ce3ce76/ao5c03775_0005.jpg" loading="lazy" id="gr7" height="1071" width="668" alt="4"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332667_ao5c03775_0006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac4c/12332667/d0e176ec3558/ao5c03775_0006.jpg" loading="lazy" id="gr8" height="828" width="668" alt="4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Results of LDH assays on both keratinocytes
and fibroblasts in
48 h of incubation. <em>p</em> &lt; 0.05 (<em>U</em> Mann–Whitney, <em>n</em> = 3).</p></figcaption></figure></section><section id="sec3.7"><h3 class="pmc_sec_title">Cell Proliferation Assay</h3>
<p>XTT tests on keratinocytes
and fibroblasts were conducted to assess the influence of potential
drug candidates on cells proliferation. These cell types are representative
of skin tissue, and their response to the compounds provides insights
into possible adverse effects, such as the inhibition of proliferation,
that could impact skin integrity or wound healing during therapeutic
application.</p>
<p>A viability test for A-20 has been previously reported. Cell proliferation assay was conducted for all
compounds for 48 and 72 h of incubation, and the graphs are shown
in <a href="#fig5" class="usa-link">Figure 
</a>. Compounds
exerting a pro-proliferating effect at lower concentrations followed
by an antiproliferative effect at higher concentrations were A-134,
A-152, A-213, and A-221. A-134 showed a pro-proliferative effect for
both keratinocytes and fibroblasts at concentrations of 0.01, 0.1,
1.0, 10, and 25 μg/mL, and this effect was stronger after 72
h of incubation. At a concentration of 50 μg/mL, the proliferation
was limited to approximately 75% and at both 100 and 150 μg/mL
concentrations, it showed an antiproliferative effect. A-152 showed
a pro-proliferative effect at concentrations ranging from 0.01 to
50 μg/mL, though only after 72 h of incubation. After 48 h,
it did not exert a pro-proliferative effect. For both cell lines,
concentrations of 100 and 150 μg/mL A-152 had a negative impact
on cell viability. A-213 showed a pro-proliferative effect at concentrations
of 0.01, 0.1, 1.0, and 10 μg/mL and was more prominent for keratinocytes
after 72 h of incubation and fibroblasts at 48 h, as well as at 72
h. The concentration of 25 μg/mL was around the control, and
after that, the proliferation dropped to around 25% at 150 μg/mL.
The viability of cells under the influence of A-221 was the best after
48 h for fibroblasts and after 72 h, it was at the level of control.
High concentrations (100–150 μg/mL) seemed to decrease
proliferation. A-219 had the best pro-proliferative effect on both
cell lines. The concentrations of 0.01 and 0.1 μg/mL were at
the level of control; however, at higher concentrations the effect
started to be more prominent (around 20% higher than control). Peptides
A-174, A-197, A-198, A-209, and A-220 did not exert any negative impact
on the viability of the studied cell lines. Peptide A-174 seemed to
have a subtle pro-proliferative effect, at all concentrations for
both tested cell lines. A-197 had no effect on both cell lines after
48 h, though after 72 h an antiproliferative effect was seen at higher
concentrations. A similar effect was observed for A-198, where a small
pro-proliferative effect was observed after 48 h of incubation. A-209
exerted neither positive nor negative effects on cells after 48 h
incubation, but a mild negative effect was observed in 72 h; however,
this effect was constant starting from 25 μg/mL. A-220 exerted
a pro-proliferative effect only after 72 h and only for keratinocytes
and at concentrations ranging from 0.01 to 50 μg/mL, then began
to be cytotoxic. On other cell lines, it had no effect or only a slight
antiproliferative effect. Peptidomimetics A-193, A-194, and A-196
had a similar impact on the tested cell lines. For A-193, after 48
h, an antiproliferative effect was observed, and it reached over 50%
at a concentration of 150 μg/mL. After 72 h, it had a slight
pro-proliferative effect on cells at a concentration of 50 μg/mL,
though this diminished at 100–150 μg/mL. The effect was
stronger for keratinocytes. A-194 had no negative impact on cell viability
at a concentration of 50 μg/mL, however, an antiproliferative
effect started at 100–150 μg/mL after 48 and 72 h of
incubation. A pro-proliferative effect was seen after 72 h on both
keratinocytes and fibroblasts; however, this effect was stronger for
keratinocytes. Peptide A-196 appeared to have the same effect on both
cell lines. At lower concentrations (0.01–10 μg/mL),
it showed a pro-proliferative effect of 5–20%, while at higher
concentrations (150 μg/mL), it showed an antiproliferative effect
of up to 50%. Peptide A-179 exerted negative impact over 48 h for
keratinocytes and fibroblasts. This was observed at all concentrations,
and it gradually increased with concentration, reaching about a 50%
decrease in cell viability in comparison to the control. The effect
was less prominent after 72 h. Peptide A-184 had the same effect.
Peptides A-164, A-165, A-176, A-191, A-192 and A-210 showed a negative
impact on the proliferation of skin cells. In the case of A-164 the
negative impact started at 0.01 μg/mL. Although this impact
was seen at every level of concentration, it seemed not to exceed
50%. Only incubation of fibroblasts for 48 h at 50, 100, and 150 μg/mL
seemed to have the most negative impact, dropping the viability to
a bit over 20%. A-165 had almost the same effect. Over 48 h of incubation
seemed to do no harm to cells at concentrations not exceeding 50 μg/mL.
A slight addition of A-176 to cell culture caused their proliferation
to stop, however it did not exceed 50%, except for fibroblasts incubated
for 48 h with concentrations of 100 or 150 μg/mL of this peptide.
A-191 seemed to limit proliferation in every concentration. At peptide
concentrations of 100 and 150 μg/mL it showed proliferation
of 20% or less, relative to the control. A-192 showed the same pattern
as A-191 and limited the proliferation to 25% in comparison to the
control. A-210 seemed to have little negative impact on both cell
lines over 48 h, though after 72 h the impact started to be negative.
Only keratinocytes showed a percentage drop in proliferation to less
than 50% at 72 h.</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332667_ao5c03775_0007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac4c/12332667/f9a80f139268/ao5c03775_0007.jpg" loading="lazy" id="gr9" height="903" width="671" alt="5"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332667_ao5c03775_0008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac4c/12332667/9e5141348e0b/ao5c03775_0008.jpg" loading="lazy" id="gr10" height="898" width="671" alt="5"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332667_ao5c03775_0009.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac4c/12332667/9c4c8a6de081/ao5c03775_0009.jpg" loading="lazy" id="gr11" height="896" width="671" alt="5"></a></p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332667_ao5c03775_0010.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac4c/12332667/756f442dc2e6/ao5c03775_0010.jpg" loading="lazy" id="gr12" height="597" width="671" alt="5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Results of XTT assays on both keratinocytes and fibroblasts
in
48 and 72 h incubation. <em>p</em> &lt; 0.05 (<em>U</em> Mann–Whitney, <em>n</em> = 3).</p></figcaption></figure></section><section id="sec3.8"><h3 class="pmc_sec_title">Exploratory Data Analysis</h3>
<p>The initial data set for
exploratory data analysis consisted of 169 parameters, including 16
ADME-related parameters and 153 molecular descriptors (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03775/suppl_file/ao5c03775_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Table S1</a>). Analyzing and interpreting such a
large and complex set of variables can pose significant challenges,
particularly in terms of readability and drawing meaningful conclusions.
To address this issue, the first step in the analysis was to apply
dimensionality reduction techniques to simplify the data set while
retaining the most relevant features. Dimensionality reduction is
a process that aims to minimize the number of variables while retaining
the most important features that contribute to the overall variability
of the data. This can be accomplished through two primary approaches:
feature selection and feature extraction. Feature selection focuses
on identifying and retaining only the most relevant features by filtering
out redundant or less informative features. On the other hand, feature
extraction generates new, meaningful features from the original dataset,
often combining or transforming existing variables to capture the
underlying structure of the data. A common technique that supports
both approaches is principal component analysis (PCA), which is widely
used to reduce the dimensionality of the data. In this study, PCA
was performed iteratively to systematically reduce the number of variables.
At each iteration, only those parameters identified as both statistically
significant and biologically relevant were retained. Through this
iterative process, the initial pool of 169 variables was reduced to
a more manageable subset of 15 critical features. These variables
included: topological polar surface area (TPSA); solvent accessible
surface area (area); synthetic accessibility; molecular volume (volume);
first Zagreb index by valence vertex degrees (ZM1 V); molecular weight
(MW); Bertz branching index (BBI); Narumi simple topological index
(SNar); Dragon branching index (DBI); overall modified Zagreb index
of order 0 and 1 by valence vertex degrees (ON0 V, ON1 V); skin permeation
(Log <em>K</em>
<sub>p</sub>); water solubility (Log <em>S</em>.ESOL); chemical potential (chem.potential) and the energy
difference between highest occupied molecular orbital and lowest unoccupied
molecular orbital (GAP). These were then used in the next steps of
exploratory data analysis, including PCA for a detailed variance exploration
and 2D-HCA.</p>
<p>PCA was conducted to provide a comprehensive understanding
of which structural features and/or physicochemical properties of
the peptidomimetic derivatives are the most influential in the original
dataset and to explore the relationship between: (i) objects (i.e.,
peptidomimetic derivatives), (ii) variables (i.e., structural and
physicochemical properties), and (iii) objects and variables. PCA
is a powerful unsupervised machine learning technique that identifies
interpretable patterns and trends in data that may elude human observation,
while also reducing the complexity inherent in high-dimensional data
sets. This is accomplished by identifying orthogonal projections that
maximize the variance within the data, termed principal components
(PCs). These PCs are derived as linear combinations of the original
variables (in this case, molecular descriptors encoding the structure
and physicochemical properties of the peptidomimetic derivatives).
The aim is for the first principal component (PC1) to explain as much
variance in the original data as possible, with subsequent principal
components accounting for progressively less variance.</p>
<p>Additionally,
to thoroughly explore the relationship between the
analyzed peptidomimetic derivatives and their biological activity,
this study also conducted a two-way hierarchical cluster analysis
(2D-HCA). This approach integrates a dendrogram with a heatmap to
illustrate the clustering of compounds based on their properties and
the grouping of physicochemical properties. 2D-HCA, acting as a pattern
recognition algorithm, relies on distance metrics to quantify the
similarity between variables/observations. Herein, Euclidean distance,
representing the shortest distance between two points in <em>n</em>-dimensional property space, served as the similarity measure, and
Ward linkage, employed to minimize variance within clusters, functioned
as the agglomeration method, facilitating the evaluation of similarity
patterns in the data.</p>
<p>The first two principal components collectively
account for 81.30%
(68.25 + 13.05%) of the total variance in the data, clearly demonstrating
the clustering of peptidomimetic derivatives based on their physicochemical
properties and biological activity. While slight variations may arise
depending on antibacterial activity (here: against and ) or the cytotoxic activity (here: keratinocytes and fibroblasts),
the overarching conclusion remains consistent. <a href="#fig6" class="usa-link">Figure 
</a>a,b illustrates that inactive peptidomimetic
derivatives are situated on the left side of the biplot. Moving along
the first principal component (<em>X</em>-axis), compounds
progress from inactivity to moderate activity and finally to high
activity against Gram-positive bacteria ( and ).</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332667_ao5c03775_0011.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac4c/12332667/09907df803a6/ao5c03775_0011.jpg" loading="lazy" id="gr13" height="629" width="696" alt="6"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>PCA biplot: (a) antibacterial
activity against ; (b) antibacterial activity against ; (c) cytotoxicity assay in human keratinocyte
cell line; (d) cytotoxicity on human fibroblast cells. The PCA biplot
simultaneously displays the principal component (PC) values of the
peptidomimetic derivatives (represented as data points) and the loadings
of the explanatory variables (represented as vectors). The length
and transparency of the vector represent the variance explained by
the variable and its contribution to the PCs, respectively. Variables
with the greatest influence on a particular PC are shown as the longest
vectors with the most intense coloring. The angles between the vectors
indicate correlations: a small angle indicates a strong positive correlation
(e.g., MW, BBI, SNar, and volume), while a large angle indicates a
weak negative correlation (e.g., Log <em>K</em>
<sub>p</sub> and Log <em>S</em>.ESOL). Data points located
close to each other in the two-dimensional PC space have similar scores,
meaning that they have similar values for the structural and physicochemical
properties represented by the variables. Log <em>S</em>.ESOL: water solubility; TPSA: topological polar surface area [Å<sup>2</sup>]; MW: molecular weight [g/mol]; BBI: Bertz branching index;
SNar: Narumi simple topological index [log function]; Log <em>K</em>
<sub>p</sub>: skin permeation [cm/s]; volume: molecular
volume [Å<sup>3</sup>]. For plot (a) and (b)antibacterial
activity is marked as “+++” for values &lt;16 μg/mL,
“++” for values 32–64 μg/mL, “+”
for values 128–256 μg/mL and “–”
for values ≥512 μg/mL. For plot (c) and (d)cytotoxicity
assay is marked as “+++” for cytotoxic effect starting
at 50 μg/mL, “++” for cytotoxic effect starting
at 100 μg/mL, “+” for cytotoxic effect starting
at 150 μg/mL and “–” for no cytotoxic effect
in any of the studied concentrations.</p></figcaption></figure><p>Analyzing the results of PCA conducted on cytotoxic
activity (specifically,
keratinocytes and fibroblasts) yields similar findings (<a href="#fig6" class="usa-link">Figure 
</a>c,d). Peptidomimetic derivatives,
when projected onto the PCs, reveal an interesting pattern: those
with the lowest cytotoxicity (&gt;150 μg/mL) are notably distant
from other derivatives and predominantly situated on the left side
of the biplot, exemplified by A-197, A-198, A-209, and A-220. Further
examination of the biplot (<a href="#fig6" class="usa-link">Figure 
</a>c,d) shows that peptidomimetic derivatives with the
highest cytotoxic activity (i.e., having cytotoxicity effect starting
at 50 μg/mL for keratinocytes/fibroblasts) are predominantly
located on the right side of the biplot, as exemplified by A-164 and
A-184. To better understand the mechanistic relationships between
the biological activity of peptidomimetic derivatives and their structural
and physicochemical properties, an analysis of normalized factor loadings
was performed (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03775/suppl_file/ao5c03775_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S2</a>). These loadings
quantify the correlation between the original variables and the PCs,
providing a measure of how strongly each variable contributes to a
given PC. Following Malinowski’s rule, only loadings with absolute
values equal to or greater than 0.7 are considered statistically significant
and meaningful for interpretation. As shown in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03775/suppl_file/ao5c03775_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S2</a>, the variables that contribute most significantly
to PC1 include MW, volume, BBI, SNar, synthetic accessibility, ZM1
V, ON0 V, and ON1 V, as well as DBI, Log <em>S</em>.ESOL, and area. Among them, all variables except Log <em>S</em>.ESOL are positively correlated with PC1. This implies
that peptidomimetic derivatives with low PC1 scores (located on the
left side of the PCA biplot) have low values for these variables,
whereas derivatives with high PC1 scores (located on the right side)
are characterized by higher values for the same variables. Conversely,
Log <em>S</em>.ESOL, which has a negative loading for
PC1, indicates that derivatives located on the left side of the biplot
are associated with relatively high water solubility. Moving along
the <em>X</em>-axis (PC1), the water solubility of these derivatives
decreases as their PC1 values increase. Thus, the positioning of peptidomimetic
derivatives along PC1 provides important insights into their structural
and physicochemical properties. Specifically, inactive derivatives
located on the left side of the biplot typically exhibit low molecular
weight, branching, surface area, and volume combined with high water
solubility. Conversely, the right-hand peptidomimetic derivatives
show the opposite trend, with higher values for these parameters and
lower solubility. The interpretation of PC2 is less straightforward.
PC2 is strongly negatively correlated with Log <em>K</em>
<sub>p</sub> and positively correlated with TPSA. As shown in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03775/suppl_file/ao5c03775_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Figure S2</a>, derivatives with varying antibacterial
or cytotoxic activities are distributed across both low and high PC2
values. This relatively low discrimination between peptidomimetic
derivatives with different biological activities along PC2 can be
attributed to the limited variance (13%) explained by this component.
Consequently, PC2 contributes less to the variance and provides limited
discrimination between derivatives with different biological activities,
underscoring the dominant role of PC1 in capturing variance related
to molecular weight, branching, surface area, volume, and water solubility.</p>
<p>Due to the slightly different classification of biological activity
employed in the cell viability assay, intriguing observations arise
from interpreting the PCA biplot presented in <a href="#fig7" class="usa-link">Figure 
</a>a,b. As the PCA analysis was conducted within
the same space of structural features and physicochemical properties
of peptidomimetic derivatives as previous PCA analyses, the positioning
of data points (i.e., compounds) in the space defined by the first
two PCs remained essentially unchanged. Notably, the marker color
changes, indicating the biological activity of the analyzed derivatives.
Upon closer examination of <a href="#fig7" class="usa-link">Figure 
</a>a,b, it becomes evident that the majority of the studied
peptidomimetic derivatives classified within the categories “0,”
“+”, “++”, or “+++” are
situated on the left side of the plot, while those categorized as
“–” are positioned on the right side of the PCA
biplot. It is noteworthy that, in derivatives exhibiting biological
activity in the cell viability assay, a discernible upward trend in
the intensity of biological activity is observable when progressing
toward higher PC1 values (along the <em>X</em>-axis), indicating
reduced water solubility and increased size (i.e., molecular weight,
branching, and volume). Furthermore, it is notable that most of the
analyzed peptide derivatives that demonstrated high antibacterial
and cytotoxic activity were found to be inactive or have negative
influence in the cell viability assay, falling into the class labeled
“–”. Examples of such compounds include compounds
A-164, A-165, A-176, A-179, A-184, and A-210. This observation was
further confirmed by two-way hierarchical cluster analysis (2D-HCA).</p>
<figure class="fig xbox font-sm" id="fig7"><h4 class="obj_head">7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332667_ao5c03775_0012.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac4c/12332667/ab689c0deef7/ao5c03775_0012.jpg" loading="lazy" id="gr14" height="304" width="696" alt="7"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>PCA biplot:
(a) cytotoxicity assay in human keratinocyte cell line
and (b) cytotoxicity on human fibroblast cells. The PCA biplot simultaneously
displays the principal component (PC) values of the peptidomimetic
derivatives (represented as data points) and the loadings of the explanatory
variables (represented as vectors). The length and transparency of
the vector represent the variance explained by the variable and its
contribution to the PCs, respectively. Variables with the greatest
influence on a particular PC are shown as the longest vectors with
the most intense coloring. The angles between the vectors indicate
correlations: a small angle indicates a strong positive correlation
(e.g., MW, BBI, SNar, and volume), while a large angle indicates a
weak negative correlation (e.g., Log <em>K</em>
<sub>p</sub> and Log <em>S</em>.ESOL). Data points located
close to each other in the two-dimensional PC space have similar scores,
meaning that they have similar values for the structural and physicochemical
properties represented by the variables. Log <em>S</em>.ESOL: water solubility; TPSA: topological polar surface area [Å<sup>2</sup>]; MW: molecular weight [g/mol]; BBI: Bertz branching index;
SNar: Narumi simple topological index [log function]; Log <em>K</em>
<sub>p</sub>: skin permeation [cm/s]; volume: Molecular
volume [Å<sup>3</sup>]. For plots (a) and (b), viability is marked
as effect at concentration of 10 μg/mL: “+++”
for strong effect, “++” for medium effect, “+”
for low effect, “0”for no effect, and “–”
negative effect.</p></figcaption></figure><p>The two-way hierarchical cluster analysis (2D-HCA)
in <a href="#fig8" class="usa-link">Figure 
</a> highlights
the relationships
between the structural and physicochemical properties of peptidomimetic
derivatives and their biological activity. Peptidomimetics with similar
chemical characteristics cluster together, suggesting that shared
structural features result in comparable bioactivity. Higher antibacterial
activity appears linked to a large topological polar surface area
(TPSA) and specific topological indices. Branching indices (BBI, DBI)
influence activity, with highly branched compounds interacting differently
with bacterial or human cells. Additionally, a larger energy gap (GAP)
correlates with lower biological activity, while high log <em>P</em> (Log <em>S</em>.ESOL) and low chemical
potential may increase cytotoxicity due to enhanced membrane permeability.</p>
<figure class="fig xbox font-sm" id="fig8"><h4 class="obj_head">8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12332667_ao5c03775_0013.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ac4c/12332667/21ee1cdb598b/ao5c03775_0013.jpg" loading="lazy" id="gr15" height="460" width="700" alt="8"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Two-way
hierarchical cluster analysis (2D-HCA) with peptidomimetic
derivatives on the <em>Y</em> axis and their structural features
and physicochemical properties on the X-axis. In the dendrograms,
branches reflect the level of similarity among variables/observations:
shorter branches denote higher similarity, while longer ones indicate
greater dissimilarity. TPSA: topological polar surface area [Å<sup>2</sup>]; area: solvent accessible surface area [Å<sup>2</sup>]; synthetic accessibility; volume: molecular volume [Å<sup>3</sup>]; ZM1 V: first Zagreb index by valence vertex degrees; MW:
molecular weight [g/mol]; BBI: Bertz branching index; SNar: Narumi
simple topological index [log function]; DBI: Dragon branching index;
ON0 V and ON1 V: overall modified Zagreb index of order 0 and 1 by
valence vertex degrees; Log <em>K</em>
<sub>p</sub>:
skin permeation [cm/s]; Log <em>S</em>.ESOL: water
solubility; chem.potential: chemical potential [eV] and GAP: the energy
difference between the highest occupied molecular orbital and the
lowest unoccupied molecular orbital [eV].</p></figcaption></figure><p>A 2D-HCA in <a href="#fig8" class="usa-link">Figure 
</a> confirmed that A-197 and A-198, which boast the highest
water solubility
and the slightest degree of branching, are notably distant from other
peptidomimetic derivatives. This suggests that they differ significantly
from the rest of the analyzed compounds. Following closely behind
in terms of dissimilarity is A-221, attributed to its remarkably low
skin permeation (Log <em>K</em>
<sub>p</sub>) value.
The most potent compound should ideally exhibit high activity in the
first two columns of <a href="#fig8" class="usa-link">Figure 
</a> (indicated by red color), low cytotoxicity in the next two
columns (indicated by blue color), and pro-proliferative effects in
the viability test (indicated as pale pink). A-191 appears to be the
most active against bacteria with low cytotoxicity and a slight negative
impact on cell line proliferation. A-213, A-196, and A-174 show low
antimicrobial activity, no cytotoxic effect, and a pro-proliferative
impact on the cell lines. Interestingly, A-196 and A-174 share a highly
similar physicochemical profile despite differences in biological
activity (they exhibit the most similar profile in the gray color
range but differ significantly in the red and blue color ranges),
implying that minor structural modifications can significantly alter
bioactivity. Moreover, A-197 and A-198 stand out due to their pronounced
deviation in several physicochemical parameters from the rest of the
data set (mainly white color), hinting at unique structural characteristics
that may influence their biological effects.</p></section></section><section id="sec4"><h2 class="pmc_sec_title">Discussion</h2>
<p>Peptidomimetics, which mimic natural peptides,
are gaining attention
in antimicrobial research as alternatives to traditional antibiotics.
With the rise of multidrug-resistant bacteria, researchers are exploring
new sources, including host defense peptides from multicellular organisms.
Antimicrobial peptides are promising compounds for combating pathogenic
Gram-positive bacteria.<em>
−



</em> While antimicrobial peptides offer high specificity, potency, and
safety,<em>
,
</em> their instability in protease-rich environments
is a drawback.<em>
−

</em>
</p>
<p>In this work, we have designed twenty-one potentially
active antimicrobial
peptidomimetics based on our previous findings.<em>
−


</em> Using various organic synthesis
strategies, we obtained different analogues with subtle structural
differences to study their impact on activity. Notably, A-20 is active
against multiple strains
and , as previously reported
in our studies.<em>
,
</em>
</p>
<p>In order to achieve good
solubility of these compounds during the
bacterial susceptibility assessment, the stock solutions were prepared
with additives. This was performed to prevent precipitation of the
peptidomimetic product during the antimicrobial assay. The additives
were 5% of ethanol or 10% of DMSO. Both these solvents have been dermatologically
investigated<em>
−


</em> and approved for topical administration by the FDA.</p>
<p>The most
potent compounds against were
A-192 and A-164, followed by A-165 and A-191. All four compounds
seemed to be active against most of the Gram-positive bacteria studied
in this paper. All these compounds include a more sterically hindered
moiety in the P4 position than A-20. A-191 and A-192 both contain
arginine in the P2 position, differing only in that A-191 has lysine
in the P1 position while A-192 has homoarginine. Additional active
compounds were A-134, A-176, A-179, A-193, and A-210. Interestingly,
despite the minor structural differences between A-192 and A-193,
their antimicrobial activity differs significantly. The key distinction
lies in the P2 position, where A-192 contains arginine and A-193 has
histidine. Despite maintaining the net charge of the compound, these
changes influence the overall activity. A study found that substituting
arginine with histidine can sometimes maintain certain functions when
replacing arginine, though it often leads to reduced efficacy or altered
activity due to the unique properties and roles of arginine in various
biological processes.<em>
,
</em> This might occur due to the environmental
conditions of the bacterial medium. BHI broth maintains a pH of 7.4,
where arginine remains in its cationic form, while histidine loses
its cationic charge and is almost deprotonated under these conditions.</p>
<p>Compounds A-191 and A-192 exhibited the lowest MIC values against in this study, with A-164 following closely
behind. These three compounds share a phenylcinnamic acid residue
at the P4 position. Five compounds also exhibited high activity against , with A-165, A-176, A-179, A-193, and
A-194 showing activity at two- to three-fold higher concentrations.
Once again, all of these compounds contain sterically hindered moieties.
A-165 and A-194 share a naphthylacetic acid moiety in the P4 position,
while the others contain a phenylcinnamic acid residue in the P4 position.
A-184 and A-210 can also be considered active. A-184, in addition
to the naphthylacetic acid moiety, possesses two consecutive arginine
residues. When analyzing the activity against , it is evident that A-192 and A-164 exhibited the lowest MIC values
for this strain. Also showing significant activity were A-210, A-179,
A-165, and A-134. A-179, which contains a phenylcinnamic acid residue
in the P4 position, is similar to A-164, with the exception that it
has arginine in the P2 position. Compound A-164 also appeared to be
potent against , with A-210
showing activity at a concentration one-fold higher. A-210 exhibited
activity against , which
also demonstrated a high susceptibility to A-164. Additionally, A-191
and A-193 showed antimicrobial activity against this strain. The next
bacterial strain investigated in this study was . The most active compound was found to
be A-164, followed by A-194. Other compounds exhibiting activity included
A-134, A-165, A-192, A-193 and A-210. A sterically hindered moiety
is also present in A-165, A-192 and A-193. A-134 features homoarginine
at the P1 position. Compound A-210 differs from the original compound
only by the presence of a methyl group on the aromatic ring in the
ortho position, of the P1 position. It is hard to determine without
structural and ligand-donor studies why such slight changes may have
such an impact on the activity of a particular molecule. It is well-known
in literature that, e.g., addition or removal of a methyl group from
the backbone of cyclic peptides can drastically affect the impact
on binding affinity, while methylation
of an amide bond, which disrupts the potential for hydrogen bonding,
can change the activity. seemed to be more susceptible to the
studied compounds. The only ones with no effect were A-197, A-198,
A-209, A-219 and A-220. All of these compounds have very drastic changes
in the structure, relative to A-20. Both A-197 and A-198 lack one
of the aromatic rings, A-197 is missing the whole cinnamoyl moiety
(P4 position), while A-198 lacks the whole carboxyloxycarbonyl moiety
(P1 position). A-209 is characterized by the absence of an aromatic
ring within the cinnamoyl moiety at the P4 position. In contrast,
both A-219 and A-220 lack the 1,2-diamine functional group at the
P3 position, instead featuring a valine residue in this region. Among
the examined peptidomimetics, the most active compound against was A-213. It differs from A-20 in lacking
a CH<sub>3</sub> group in 1,2-diamine, resulting in reduced branching
(from isopropyl to an ethyl substituent). In addition to A-213, A-194
also exhibited activity against , with a potency similar to A-20. A-194 contains a sterically hindered
moiety in the P4 position and features homoarginine instead of arginine
in the P1 position. Additionally, it has arginine in the P2 position
instead of leucine. Currently, the infections caused by are treated with chlorhexidine and silver
nanoparticles and some AMPs targeting
this bacterium have also been identified.
</p>
<p>Despite some peptidomimetics showing MIC values as low as
256 μg/mL,
this concentration was still too high to be considered indicative
of meaningful activity against any of the studied Gram-negative bacterial
(, , , used as a representative of Gram-negative
bacteria) strains. The difference in activity between Gram-negative
and Gram-positive bacteria may be attributed to distinct structural
and compositional differences. Antimicrobial compounds that exhibit
selective activity against Gram-positive bacteria, such as vancomycin, bacitracin, and
daptomycin, function primarily by targeting
structural components of a Gram-positive cell wall. These bacteria
possess a thick peptidoglycan layer, enriched with teichoic acids,
which readily attracts the cationic nature of antimicrobial peptides.
In contrast, Gram-negative bacteria are inherently resistant to many
of these agents due to their additional outer membrane, which serves
as a permeability barrier and lacks the high-density peptidoglycan
structure found in Gram-positive bacteria. Given the cationic nature
of our peptidomimetics, it is reasonable to hypothesize that their
antimicrobial activity is associated with membrane disruption. However,
our previous attempts to confirm this mechanism were unsuccessful. To gain deeper insights into the mode of action
of these peptidomimetics, more advanced microbiological models should
be employed.</p>
<p>Given that peptides are prone to degradation by
proteolytic enzymes<em>
,
</em> structural modifications are
usually introduced to enhance their
stability in enzyme-rich fluids. In our
research stability of peptidomimetics in aqueous solutions was assessed,
as peptides often tend to precipitate or degrade in solution. All synthesized peptidomimetics demonstrated
stability in water, and the majority exhibited resistance to proteolytic
degradation in human plasma. Notably, among all the peptidomimetics
tested, A-196, A-219, A-220, and A-221those with the fewest
modifications compared to natural peptidesshowed signs of
degradation after 24 h of incubation.</p>
<p>The most potent compounds
with the highest antimicrobial activity
in this study were A-164, A-165, A-191, and A-192. Both A-191 and
A-192 contain arginine in the P2 position, an insertion intended to
enhance solubility compared to that of A-164. A-164 exhibits a higher
hydrophobicity, as indicated by the percentage of acetonitrile required
for its elution from the column. Computational ADMET predictions classified
A-164 as either poorly soluble or insoluble, depending on the algorithm
applied. A-165 appears to be even more hydrophobic than A-164, as
evidenced by its longer elution time. However, ADMET predictions classified
it as either moderately soluble or insoluble. A-164 and A-165 have
the highest calculated lipophilicity due to additional aromatic rings
at the P4 position. A-192 is more hydrophobic than A-191, with a moderate
hydrophobicity level (% AcCN is 48.7, while the value for A-20 is
53.3). This shift in hydrophobicity relative to that of A-164 is attributed
to the substitution of leucine with arginine at the P2 position. Additionally,
the presence of an extra −CH<sub>2</sub>– group in the
P1 position (arginine to homoarginine) and the incorporation of an
additional aromatic ring (cinnamoyl moiety to phenylcinnamoyl moiety)
relative to A-20 would be expected to increase hydrophobicity. However,
certain intrinsic molecular interactions may influence shifts in solubility
in ways not solely dictated by structural modifications.<em>
−

</em> Alternatively, the substitution of leucine with arginine alone may
be sufficient to enhance the molecule’s solubility, despite
the aforementioned modifications. The hydrophobicity of the compounds
appears to be an important factor in their activity, which might be
linked to their mechanism of action. The least hydrophobic compounds,
namely, A-197, A-198 (lacking certain aromatic rings), A-220, and
A-219 (based on both theoretical and experimental data), seem to be
completely inactive, as evident from the PCA analysis. In contrast,
the most active compounds are positioned on opposite sides of the
graph, exhibiting poor solubility but high activity. It is readily
noticed that the additional charge (from +1 to +2) in compounds like
A-174 (histidine in place of leucine) and A-176 (arginine in place
of leucine) decreased the lipophilicity in comparison to A-164.</p>
<p>An additional verification of experimental data and additional
information was retrieved by the ADMET theoretical tool. The computational
approach is widely known and utilized.<em>
−



</em> In addition to the aforementioned lipophilicity,
parameters such as water solubility, gastrointestinal (GI) absorption,
P-glycoprotein substrate, skin permeation, BBB permeation, potential
of cytochrome inhibition/substrate enzymes, total clearance, renal
OCT2 substrate, toxicity, LD50, hepatotoxicity, and skin sensitization
were calculated for the investigated molecules.</p>
<p>The gastrointestinal
(GI) absorption of the presented compounds
was low, and oral administration is not a suitable administration
method. All the presented compounds were likely to be substrates for
P-gp, which is not so uncommon in antibiotics, e.g., clarithromycin
is a substrate for both cytochrome p4503A4 and P-gp. However, these properties can be improved by putting the
compound into a formulation, e.g., into lipids, administrating it with a P-glycoprotein ATPase inhibitor
(e.g., curcumin), or adding a probiotic
which secretes such an inhibitor and boosts absorption. All of the studied compounds seemed to have
low skin permeability. While many antibiotics exhibit poor skin permeation
values,<em>
,
</em> their permeability can be altered by choosing
the right formulation or by the addition of a penetration enhancer.<em>
,−


</em> Low skin permeation poses a challenge for topical administration
when ulcer treatment. Moreover, none of the studied compounds were
predicted to pass the BBB, which is not uncommon, as penicillin’s
penetration is as low as 1% in an uninflamed organism. However, certain antibiotics, such as ampicillin
and amoxicillin, can cross the barrier even in absence of inflammation. The next calculated parameter was VD. Almost
all the compounds in this study are considered to be distributed primarily
in plasma rather than in tissue, with only A-198 and A-165 exceeding
negative values, and three compounds (A-134, A-176, and A-193) potentially
exhibiting moderate distribution. The VD values for vancomycin range
from −0.398 to 0 log L/kg, and for gentamicin, they range from −0.699 to −0.432
log L/kg. The results showed that
none of the designed compounds are predicted to be substrates of CYP2D6;
however, all were likely to be substrates of CYP3A4. None of the compounds
were predicted to inhibit the CYP1A2, CYP2C19, and CYP2D6 isoforms,
while only two (A-174 and A-220) are likely inhibitors of the CYP3A4
isoform. Nine compounds (A-174, A-179, A-191, A-192, A-194, A-196,
A-209, A-210, and A-213) are unlikely to inhibit the CYP2C9 isoform.
The metabolism of antimicrobial compounds by CYP enzymes can result
in their conversion into active or inactive metabolites, potentially
altering their efficacy. Inhibition of CYP enzymes can reduce the
metabolism of coadministered drugs, thus increasing plasma concentration
and posing a higher risk of toxicity or adverse effects. Vancomycin
is an antibiotic that is neither a substrate nor an inhibitor of CYP
enzymes, and is primarily cleared by renal function.
</p>
<p>Six compounds (A-164, A-179, A-192, A-197, A-210,
and A-221) are
predicted to be slowly eliminated from the body in total drug clearance,
while compounds A-174, A-198, and A-220 exhibit the highest rates
of elimination. This implies that these compounds may require more
frequent administration or formulation for controlled release. None
of the listed compounds are likely substrates for OCT2, which is important
when administered with other drugs. Additionally, none of the compounds
in this study exhibited mutagenic potential, which might indicate
that they are potentially noncarcinogenic. Almost all of the compounds
are predicted to be potentially hepatotoxic, with the exception of
A-197 and A-198. Many well-known antibiotics, such as amoxicillin
and tetracyclines, also present this side effect.<em>
,
</em> Skin sensitization is another critical safety consideration for
products applied dermally, as it can lead to allergic dermatitis.
None of the tested compounds appeared to potentially irritate the
skin, which is crucial for topical administration.</p>
<p>In this study,
LDH tests were employed to evaluate the membrane
integrity of cells treated with various peptidomimetic compounds,
allowing for a comprehensive analysis of their safety profiles. The
results from the LDH assays revealed differential cytotoxic effects,
underscoring the importance of this assay in the early stages of peptidomimetic
drug development. The cytotoxicity levels were studies in concentrations
of 50, 100, and 150 μg/mL, which are mostly above the level
of MIC values. A-174, A-191, A-197, and A-213 seem to exert no or
minimal cytotoxicity toward both cell lines. Only A-191, whose MIC
value for of 16 μg/mL
exerts little cytotoxicity at higher concentrations, though this fades
with decreasing concentration. Compound A-164, whose MIC value is
against is 8 μg/mL
and shows a high cell toxicity profile at concentrations 6-fold higher
than its MIC value. It is not uncommon for antibiotics to possess
cytotoxic properties, for example, it happens in the case of fluoroquinolones
and aminoglycosides and in the case of
topical ointments containing polymixin B, bacitracin, neomycin, and
gentamicin. The results showed that A-192,
which seems to be the most potent antimicrobial compound, shows high
cytotoxicity toward keratinocytes, though its cytotoxicity toward
fibroblasts seems to be much lower. Keratinocytes are cells that are
primarily found in the epidermis, whereas fibroblasts are hidden in
connective tissue. While both cell types can exhibit sensitivity to
chemicals, keratinocytes are generally more fragile than fibroblasts
and more responsive to topical chemical exposures. A-165, A-219, and A-221 are cytotoxic only for fibroblasts
and exert a noncytotoxic effect on keratinocytes. A-220 demonstrated
consistent cytotoxicity at a level of 10% over all of the concentrations.
Compounds A-193, A-194, A-196, A-198, and A-209 become toxic to both
cell lines at concentrations of 150 μg/mL. A-210 begins to be
cytotoxic at a concentration of 100 μg/mL. The cytotoxicity
of A-152 is similar to that of A-20. A-134, A-176, A-179, and A-184
are toxic at all concentrations. The incubation time of the cell with
peptidomimetic was prolonged to 48 h due to the long-term use in the
potential treatment of wounds with the applied antimicrobial compound.
The topical administration of peptidomimetics can avoid strong side
effects such as nephrotoxicity or ototoxicity; however, the cytotoxicity
to skin cells is very important. The administration of antimicrobial
agents for a limited duration, despite their cytotoxic potential,
can be clinically advantageous, particularly when targeting multidrug-resistant
pathogens such as MRSA. Notable examples of toxic antibiotics utilized
in clinical settings include polymyxin E (colistin),<em>
,
</em> chloramphenicol, vancomycin, and linezolid.<em>
,
</em> These agents
are associated with significant adverse effects, including nephrotoxicity,
neurotoxicity, aplastic anemia, and bone marrow suppression. While
chloramphenicol is rarely employed due to its hematological toxicity,
it remains a viable option in the management of life-threatening infections,
such as bacterial meningitis, in cases where alternative treatments
are unavailable.<em>
,
</em>
</p>
<p>Peptides and peptidomimetics
are capable of exerting dual actions,
such as both antimicrobial and antiviral effects (as in the case of
A-20) or antimicrobial and pro-proliferative
effects. The safety profile of A-20
has previously been reported, demonstrating
no impact on the proliferation of primary human keratinocytes at concentrations
up to 50 μg/mL, and no induction of inflammatory responses or
allergic reactions. The cell proliferation assay was specifically
designed to evaluate whether the compounds could independently stimulate
cell proliferation. To eliminate background and accurately assess
the direct effects of the tested compounds, serum-free conditions
were employed throughout the experiment with FBS serving as a positive
control.</p>
<p>Many peptidomimetics exhibit dose-dependent effects,
with antiproliferative
activity at higher concentrations and pro-proliferative activity at
lower concentrations, as seen with A-134. Some peptidomimetics, such
as A-152, require extended exposure times to exert their effects,
with notable activity observed after only 72 h. Among the tested compounds,
A-219 exhibited the strongest pro-proliferative effect on both cell
lines across all concentrations, demonstrating a dose-dependent relationship.
A-174, although showing no significant impact on cells, exhibited
a subtle pro-proliferative effect, while A-213 displayed a positive
impact after 72 h at low concentrations. Compounds A-164, A-165, A-176,
A-179, A-184, A-191, A-192, A-209, and A-210 demonstrated antiproliferative
effects across nearly all concentrations, with increased activity
at higher concentrations. A-193 and A-194 displayed an antiproliferative
effect at high concentrations; however, after 72 h, a strong pro-proliferative
effect was observed in fibroblasts. Their antimicrobial efficacy was
noted at concentrations between 32 and 64 μg/mL, which falls
outside the pro-proliferative range, suggesting a safer profile for
wound application. A-196 showed a slight pro-proliferative effect
at low concentrations but became cytotoxic at higher concentrations.
An antiproliferative effect was noted for A-197 and A-198 only at
high concentrations after 72 h of incubation in both cell lines, while
no such effect was observed at 48 h. A-220 exhibited a slight pro-proliferative
effect on keratinocytes after 72 h at low concentrations, while A-221
exhibited a pro-proliferative effect on fibroblasts after 48 h at
low concentrations but displayed mild toxicity at higher concentrations.</p>
<p>Compounds with the most potent antimicrobial activity, such as
A-164, A-165, and A-192 (considering only activity against ), exhibited antiproliferative effects on
the tested cell lines. However, their impact on cell proliferation
decreases to 50% at 50 μg/mL and remains at this level up to
the maximum tested concentration of 150 μg/mL. Other compounds
with notable antibacterial activities include A-176, A-179, A-191,
and A-193. A-191 nearly completely inhibits proliferation at higher
concentrations, whereas A-179 progressively suppresses proliferation
as its concentration increases. A-176 and A-193 exhibit similar behavior
to A-164, A-165, and A-192 in terms of their effects on proliferation,
with a decline in cell viability as the concentration of peptidomimetic
increases. The final groups of compounds with antimicrobial activity
to be considered include A-184, A-194, A-210, and A-213. For A-184
and A-210, proliferation decreases with increasing concentrations
of peptidomimetics, reaching 50% at the highest tested concentration.
A-213 shows antiproliferative effects at high concentrations, but
at lower concentrations, it stimulates proliferation by 20–30%.
A-194 displays pro-proliferative effects at low concentrations, but
at concentrations exceeding 100 μg/mL, it demonstrates antiproliferative
properties.</p>
<p>Many empirical and computational SAR methods are
being applied
to investigate a series of compounds in order to establish which structural
(chemical structure, secondary structure, and flexibility) or physicochemical
(hydrophobicity and net charge) properties are fundamental to their
biological activity. In this case, a PCA analysis was used. This allowed
us to discover how structural features and physicochemical properties
are responsible for the molecular biological activity. The biological
results had to be simplified in order to be analyzed. The antimicrobial
activity of the peptidomimetics was calculated only by two bacterial
strains, namely and divided into four classes. A biological
assay was gathered in 4 classes for cytotoxicity and 5 classes for
the proliferation assay, distinguishing only cytotoxicity and pro-proliferative
effects for both cell lines and without distinguishing the time over
which the incubation was conducted. The features exerting the most
significant influence on the separation of peptidomimetic derivatives
are water solubility, molecular weight (MW), degree of branching (BBI
and SNar), and the compound’s volume. Inactive compounds (such
as A-197 and A-198) exhibit notably higher water solubility and lower
molecular mass, branching, and volume compared to active compounds
(e.g., A-192 and A-193). This observation aligns with the general
trend indicating that reduced water solubility and increased size
(i.e., molecular weight, branching, and volume) correlate with heightened
efficacy against and . Furthermore, it is important to note
that most of the analyzed peptidomimetic derivatives, despite showing
strong antibacterial activity, exerted some cytotoxic activity at
high concentrations. Examples of these compounds include compounds
A-164, A-165, A-176, A-179, A-184, and A-210. This finding was further
validated by a two-way hierarchical cluster analysis (2D-HCA), which
confirmed these findings. Intriguingly, certain peptidomimetic derivatives,
such as A-176, A-179, A-184, A-193, and A-194, exhibited greater antibacterial
activity against than upon exposure. In other words, demonstrated heightened sensitivity to
these studied derivatives. The variance in responses to peptidomimetic
derivative exposure is hard to analyze when the molecular target,
as well as the mechanism of action of these compounds, remains unknown.
Moreover, slight disparities in sensitivity were observed in the cytotoxic
activity and viability tests. The human keratinocyte cell line exhibited
greater sensitivity in both scenarios than did human fibroblast cells.</p></section><section id="sec5"><h2 class="pmc_sec_title">Conclusions</h2>
<p>This study explores the potential of structurally
modified peptidomimetic
analogues of cystatin C as promising antimicrobial agents against
Gram-positive pathogens, particularly, and . Through rational
design and synthesis of novel compounds, we demonstrate how subtle
modifications in molecular architectureincluding changes in
charge, hydrophobicity, and steric hindrancecan significantly
influence antimicrobial efficacy and cytocompatibility. Two key compounds
exhibited potent antimicrobial activity with acceptable cytotoxicity
levels, while others showed selective effects depending on concentration
and exposure time. However, challenges such as solubility issues and
limited efficacy against Gram-negative bacteria underscore the need
for further optimization of these compounds. Importantly, most derivatives
were stable in aqueous solutions and human plasma, with low predicted
mutagenic and skin sensitization potential. ADMET predictions also
supported the potential for topical use due to low systemic absorption
and favorable physicochemical properties. The theoretical studies
revealed a correlation between structural features, offering insight
into key parameters driving efficacy. These findings lay a strong
foundation for further preclinical development and formulation strategies
aimed at localized treatment of drug-resistant Gram-positive infections,
particularly in wound or skin-associated contexts. Overall, our results
demonstrate that peptidomimetic optimization guided by both experimental
and computational approaches can yield lead compounds with therapeutic
potential and encourage continued exploration of peptide-inspired
compounds in antimicrobial drug development. Future studies should
focus on investigating the molecular mechanism underlying the antimicrobial
activity of these compounds.</p></section><section id="sec22"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="sifile1"><div class="media p"><div class="caption">
<a href="/articles/instance/12332667/bin/ao5c03775_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">ao5c03775_si_001.pdf</a><sup> (894.7KB, pdf) </sup>
</div></div></section></section><section id="ACK-d14e2732-autogenerated" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>This work was supported by the grant PRELUDIUM BIS 2 “A
new class of antimicrobial compounds. Design, synthesis and study
of the mechanism of action.” no. UMO-2020/39/O/NZ7/00430 (National
Science Centre, Poland).</p></section><section id="notes-2"><p>The Supporting Information
is available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsomega.5c03775?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acsomega.5c03775</a>.</p>
<ul id="silist" class="list" style="list-style-type:none"><li><p>Symbols and definitions of all ADME-related parameters
and molecular descriptors used for exploratory data analysis (Table
S1); exemplary HPLC analysis of the stability of peptidomimetics (Figure
S1); plot of the normalized factor loadings (Figure S2); % of acetonitrile
required for elution from the column and theoretical and experimental
masses measured by mass spectrometry of synthesized compounds (Table
S2); results of microbiological assay for all 22 compounds for eight
Gram-positive and four Gram-negative bacteria (Table S3) (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsomega.5c03775/suppl_file/ao5c03775_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p></li></ul></section><section id="notes-1"><p>Maria Dzierżyńska:
Conceptualization, methodology, formal analysis, investigation, data
curation, supervision, writingoriginal draft, writingreview
and editing, and visualization. Justyna Sawicka: Investigation, formal
analysis, and visualization. Katarzyna Łada: Investigation.
Agnieszka Gajewicz-Skretna: Investigation, formal analysis, visualization,
and writingoriginal draft. Milena Deptuła: Investigation.
Alexey Chernobrovkin: Investigation. Aneta Pogorzelska: Investigation.
Anders Grubb: Supervision. Roman A. Zubarev: Supervision and writingreview
and editing. Michał Pikuła: Funding acquisition. Franciszek
Kasprzykowski: Supervision. Sylwia Rodziewicz-Motowidło: Funding
acquisition, writingreview and editing, and supervision.</p></section><section id="NOTES-d14e2749-autogenerated"><p>The
authors declare no
competing financial interest.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1"><cite>

Fleming, A.

PenicillinThe
Nobel Lecture. NobelPrize.org. 2025. <a href="https://www.nobelprize.org/uploads/2018/06/fleming-lecture.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nobelprize.org/uploads/2018/06/fleming-lecture.pdf</a> (accessed May 31, 2025).</cite></li>
<li id="ref2">
<cite>Abejew A. A., Wubetu G. Y., Fenta T. G.. Relationship
between Antibiotic Consumption
and Resistance: A Systematic Review. Can. J.
Infect. Dis. Med. Microbiol. 2024;2024:9958678.  doi: 10.1155/2024/9958678.</cite> [<a href="https://doi.org/10.1155/2024/9958678" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10932619/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38476862/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Can.%20J.%20Infect.%20Dis.%20Med.%20Microbiol.&amp;title=Relationship%20between%20Antibiotic%20Consumption%20and%20Resistance:%20A%20Systematic%20Review&amp;author=A.%20A.%20Abejew&amp;author=G.%20Y.%20Wubetu&amp;author=T.%20G.%20Fenta&amp;volume=2024&amp;publication_year=2024&amp;pages=9958678&amp;pmid=38476862&amp;doi=10.1155/2024/9958678&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3">
<cite>Moriel D. G., Piccioli D., Raso M. M., Pizza M.. The Overlooked Bacterial
Pandemic. Semin. Immunopathol. 2024;45(4–6):481–491. doi: 10.1007/s00281-023-00997-1.</cite> [<a href="https://doi.org/10.1007/s00281-023-00997-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11136754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38078911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Semin.%20Immunopathol.&amp;title=The%20Overlooked%20Bacterial%20Pandemic&amp;author=D.%20G.%20Moriel&amp;author=D.%20Piccioli&amp;author=M.%20M.%20Raso&amp;author=M.%20Pizza&amp;volume=45&amp;issue=4%E2%80%936&amp;publication_year=2024&amp;pages=481-491&amp;pmid=38078911&amp;doi=10.1007/s00281-023-00997-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref4">
<cite>Bonomo R. A., Perez F., Hujer A. M., Hujer K. M., Vila A. J.. The Real
Crisis in Antimicrobial Resistance: Failure to Anticipate and Respond. Clin. Infect. Dis. 2024;78(6):1429–1433. doi: 10.1093/cid/ciad758.</cite> [<a href="https://doi.org/10.1093/cid/ciad758" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38289748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Infect.%20Dis.&amp;title=The%20Real%20Crisis%20in%20Antimicrobial%20Resistance:%20Failure%20to%20Anticipate%20and%20Respond&amp;author=R.%20A.%20Bonomo&amp;author=F.%20Perez&amp;author=A.%20M.%20Hujer&amp;author=K.%20M.%20Hujer&amp;author=A.%20J.%20Vila&amp;volume=78&amp;issue=6&amp;publication_year=2024&amp;pages=1429-1433&amp;pmid=38289748&amp;doi=10.1093/cid/ciad758&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Daruka L., Czikkely M. S., Szili P., Farkas Z., Balogh D., Grézal G., Maharramov E., Vu T.-H., Sipos L., Juhász S., Dunai A., Daraba A., Számel M., Sári T., Stirling T., Vásárhelyi B. M., Ari E., Christodoulou C., Manczinger M., Enyedi M. Z., Jaksa G., Kovács K., van Houte S., Pursey E., Pintér L., Haracska L., Kintses B., Papp B., Pál C.. ESKAPE Pathogens
Rapidly Develop Resistance against Antibiotics in Development in Vitro. Nat. Microbiol. 2025;10(2):313–331. doi: 10.1038/s41564-024-01891-8.</cite> [<a href="https://doi.org/10.1038/s41564-024-01891-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11790497/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39805953/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Microbiol.&amp;title=ESKAPE%20Pathogens%20Rapidly%20Develop%20Resistance%20against%20Antibiotics%20in%20Development%20in%20Vitro&amp;author=L.%20Daruka&amp;author=M.%20S.%20Czikkely&amp;author=P.%20Szili&amp;author=Z.%20Farkas&amp;author=D.%20Balogh&amp;volume=10&amp;issue=2&amp;publication_year=2025&amp;pages=313-331&amp;pmid=39805953&amp;doi=10.1038/s41564-024-01891-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Langford B. J., Soucy J. P. R., Leung V., So M., Kwan A. T. H., Portnoff J. S., Bertagnolio S., Raybardhan S., MacFadden D. R., Daneman N.. Antibiotic Resistance
Associated
with the COVID-19 Pandemic: A Systematic Review and Meta-Analysis. Clin. Microbiol. Infect. 2023;29(3):302–309. doi: 10.1016/j.cmi.2022.12.006.</cite> [<a href="https://doi.org/10.1016/j.cmi.2022.12.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9733301/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36509377/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Microbiol.%20Infect.&amp;title=Antibiotic%20Resistance%20Associated%20with%20the%20COVID-19%20Pandemic:%20A%20Systematic%20Review%20and%20Meta-Analysis&amp;author=B.%20J.%20Langford&amp;author=J.%20P.%20R.%20Soucy&amp;author=V.%20Leung&amp;author=M.%20So&amp;author=A.%20T.%20H.%20Kwan&amp;volume=29&amp;issue=3&amp;publication_year=2023&amp;pages=302-309&amp;pmid=36509377&amp;doi=10.1016/j.cmi.2022.12.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Langford B. J., So M., Simeonova M., Leung V., Lo J., Kan T., Raybardhan S., Sapin M. E., Mponponsuo K., Farrell A., Leung E., Soucy J. R., Cassini A., MacFadden D., Macfadden D., Daneman N.. Antimicrobial Resistance
in Patients with COVID-19: A Systematic Review and Meta-Analysis. Lancet Microbe. 2023;4(3):e179–e191. doi: 10.1016/S2666-5247(22)00355-X.</cite> [<a href="https://doi.org/10.1016/S2666-5247(22)00355-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9889096/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36736332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lancet%20Microbe&amp;title=Antimicrobial%20Resistance%20in%20Patients%20with%20COVID-19:%20A%20Systematic%20Review%20and%20Meta-Analysis&amp;author=B.%20J.%20Langford&amp;author=M.%20So&amp;author=M.%20Simeonova&amp;author=V.%20Leung&amp;author=J.%20Lo&amp;volume=4&amp;issue=3&amp;publication_year=2023&amp;pages=e179-e191&amp;pmid=36736332&amp;doi=10.1016/S2666-5247(22)00355-X&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Massarine N. C. M., de Souza G. H. d. A., Nunes I. B., Salomé T. M., Barbosa M. d. S., Faccin I., Rossato L., Simionatto S.. How Did COVID-19
Impact the Antimicrobial Consumption and Bacterial Resistance Profiles
in Brazil? Antibiotics. 2023;12(9):1374.  doi: 10.3390/antibiotics12091374.</cite> [<a href="https://doi.org/10.3390/antibiotics12091374" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10526034/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37760671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antibiotics&amp;title=How%20Did%20COVID-19%20Impact%20the%20Antimicrobial%20Consumption%20and%20Bacterial%20Resistance%20Profiles%20in%20Brazil?&amp;author=N.%20C.%20M.%20Massarine&amp;author=G.%20H.%20d.%20A.%20de%20Souza&amp;author=I.%20B.%20Nunes&amp;author=T.%20M.%20Salom%C3%A9&amp;author=M.%20d.%20S.%20Barbosa&amp;volume=12&amp;issue=9&amp;publication_year=2023&amp;pages=1374&amp;pmid=37760671&amp;doi=10.3390/antibiotics12091374&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>
WHO . Global Antimicrobial
Resistance and Use Surveillance System (GLASS); World Health Organization: Geneva, 2025. <a href="https://www.who.int/publications/i/item/9789240108127" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.who.int/publications/i/item/9789240108127</a> (accessed May 31, 2025).</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Global%20Antimicrobial%20Resistance%20and%20Use%20Surveillance%20System%20(GLASS)&amp;publication_year=2025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>
WHO . Global Antimicrobial
Resistance and Use Surveillance System (GLASS) Report 2022; World Health Organization: Geneva, 2022. <a href="https://www.who.int/publications/i/item/9789240062702https://www.who.int/publications/i/item/9789240062702" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.who.int/publications/i/item/9789240062702https://www.who.int/publications/i/item/9789240062702</a> (accessed May 31, 2025).</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Global%20Antimicrobial%20Resistance%20and%20Use%20Surveillance%20System%20(GLASS)%20Report%202022&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Ji S., An F., Zhang T., Lou M., Guo J., Liu K., Zhu Y., Wu J., Wu R.. Antimicrobial Peptides: An Alternative
to Traditional Antibiotics. Eur. J. Med. Chem. 2024;265:116072.  doi: 10.1016/j.ejmech.2023.116072.</cite> [<a href="https://doi.org/10.1016/j.ejmech.2023.116072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38147812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Med.%20Chem.&amp;title=Antimicrobial%20Peptides:%20An%20Alternative%20to%20Traditional%20Antibiotics&amp;author=S.%20Ji&amp;author=F.%20An&amp;author=T.%20Zhang&amp;author=M.%20Lou&amp;author=J.%20Guo&amp;volume=265&amp;publication_year=2024&amp;pages=116072&amp;pmid=38147812&amp;doi=10.1016/j.ejmech.2023.116072&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Erriah P., Puan S. L., Yahaya N. M., Wan Ahmad Kamil W. N. I., Amin Nordin S., Muhamad A., Sabri S.. Harnessing Bacterial
Antimicrobial Peptides: A Comprehensive Review on Properties, Mechanisms,
Applications, and Challenges in Combating Antimicrobial Resistance. J. Appl. Microbiol. 2025;136(3):lxae290.  doi: 10.1093/jambio/lxae290.</cite> [<a href="https://doi.org/10.1093/jambio/lxae290" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40036746/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Appl.%20Microbiol.&amp;title=Harnessing%20Bacterial%20Antimicrobial%20Peptides:%20A%20Comprehensive%20Review%20on%20Properties,%20Mechanisms,%20Applications,%20and%20Challenges%20in%20Combating%20Antimicrobial%20Resistance&amp;author=P.%20Erriah&amp;author=S.%20L.%20Puan&amp;author=N.%20M.%20Yahaya&amp;author=W.%20N.%20I.%20Wan%20Ahmad%20Kamil&amp;author=S.%20Amin%20Nordin&amp;volume=136&amp;issue=3&amp;publication_year=2025&amp;pages=lxae290&amp;pmid=40036746&amp;doi=10.1093/jambio/lxae290&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Sneideris T., Erkamp N. A., Ausserwöger H., Saar K. L., Welsh T. J., Qian D., Katsuya-Gaviria K., Johncock M. L. L. Y., Krainer G., Borodavka A., Knowles T. P. J.. Targeting Nucleic
Acid Phase Transitions as a Mechanism of Action for Antimicrobial
Peptides. Nat. Commun. 2023;14(1):7170.  doi: 10.1038/s41467-023-42374-4.</cite> [<a href="https://doi.org/10.1038/s41467-023-42374-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10630377/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37935659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Targeting%20Nucleic%20Acid%20Phase%20Transitions%20as%20a%20Mechanism%20of%20Action%20for%20Antimicrobial%20Peptides&amp;author=T.%20Sneideris&amp;author=N.%20A.%20Erkamp&amp;author=H.%20Ausserw%C3%B6ger&amp;author=K.%20L.%20Saar&amp;author=T.%20J.%20Welsh&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=7170&amp;pmid=37935659&amp;doi=10.1038/s41467-023-42374-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Chen N., Jiang C.. Antimicrobial Peptides:
Structure, Mechanism, and Modification. Eur.
J. Med. Chem. 2023;255(3):115377.  doi: 10.1016/j.ejmech.2023.115377.</cite> [<a href="https://doi.org/10.1016/j.ejmech.2023.115377" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37099837/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Med.%20Chem.&amp;title=Antimicrobial%20Peptides:%20Structure,%20Mechanism,%20and%20Modification&amp;author=N.%20Chen&amp;author=C.%20Jiang&amp;volume=255&amp;issue=3&amp;publication_year=2023&amp;pages=115377&amp;pmid=37099837&amp;doi=10.1016/j.ejmech.2023.115377&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>Björck L., Grubb A., Kjellén L.. Cystatin C,
a Human Proteinase Inhibitor,
Blocks Replication of Herpes Simplex Virus. J. Virol. 1990;64(2):941–943. doi: 10.1128/jvi.64.2.941-943.1990.</cite> [<a href="https://doi.org/10.1128/jvi.64.2.941-943.1990" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC249195/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2153254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Virol.&amp;title=Cystatin%20C,%20a%20Human%20Proteinase%20Inhibitor,%20Blocks%20Replication%20of%20Herpes%20Simplex%20Virus&amp;author=L.%20Bj%C3%B6rck&amp;author=A.%20Grubb&amp;author=L.%20Kjell%C3%A9n&amp;volume=64&amp;issue=2&amp;publication_year=1990&amp;pages=941-943&amp;pmid=2153254&amp;doi=10.1128/jvi.64.2.941-943.1990&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>Zi M., Xu Y.. Involvement of Cystatin C in Immunity and Apoptosis. Immunol. Lett. 2018;196(1):80–90. doi: 10.1016/j.imlet.2018.01.006.</cite> [<a href="https://doi.org/10.1016/j.imlet.2018.01.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7112947/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29355583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunol.%20Lett.&amp;title=Involvement%20of%20Cystatin%20C%20in%20Immunity%20and%20Apoptosis&amp;author=M.%20Zi&amp;author=Y.%20Xu&amp;volume=196&amp;issue=1&amp;publication_year=2018&amp;pages=80-90&amp;pmid=29355583&amp;doi=10.1016/j.imlet.2018.01.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Blancas-Luciano B. E., Becker-Fauser I., Zamora-Chimal J., Delgado-Domínguez J., Ruíz-Remigio A., Leyva-Huerta E. R., Portilla-Robertson J., Fernández-Presas A. M.. Antimicrobial
and Anti-Inflammatory
Activity of Cystatin C on Human Gingival Fibroblast Incubated with . PeerJ. 2022;10:e14232.  doi: 10.7717/peerj.14232.</cite> [<a href="https://doi.org/10.7717/peerj.14232" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9615962/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36312752/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=Antimicrobial%20and%20Anti-Inflammatory%20Activity%20of%20Cystatin%20C%20on%20Human%20Gingival%20Fibroblast%20Incubated%20with&amp;author=B.%20E.%20Blancas-Luciano&amp;author=I.%20Becker-Fauser&amp;author=J.%20Zamora-Chimal&amp;author=J.%20Delgado-Dom%C3%ADnguez&amp;author=A.%20Ru%C3%ADz-Remigio&amp;volume=10&amp;publication_year=2022&amp;pages=e14232&amp;pmid=36312752&amp;doi=10.7717/peerj.14232&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>Blancas-Luciano B. E., Becker-Fauser I., Zamora-Chimal J., Jiménez-García L., Lara-Martínez R., Pérez-Torres A., del Pliego M. G., Aguirre-Benítez E. L., Fernández-Presas A. M.. Cystatin
C: Immunoregulation Role in Macrophages Infected with . PeerJ. 2024;12(4):e17252.  doi: 10.7717/peerj.17252.</cite> [<a href="https://doi.org/10.7717/peerj.17252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11067906/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38708345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=Cystatin%20C:%20Immunoregulation%20Role%20in%20Macrophages%20Infected%20with&amp;author=B.%20E.%20Blancas-Luciano&amp;author=I.%20Becker-Fauser&amp;author=J.%20Zamora-Chimal&amp;author=L.%20Jim%C3%A9nez-Garc%C3%ADa&amp;author=R.%20Lara-Mart%C3%ADnez&amp;volume=12&amp;issue=4&amp;publication_year=2024&amp;pages=e17252&amp;pmid=38708345&amp;doi=10.7717/peerj.17252&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Teng P., Shao H., Huang B., Xie J., Cui S., Wang K., Cai J.. Small Molecular Mimetics of Antimicrobial
Peptides as a Promising Therapy To Combat Bacterial Resistance. J. Med. Chem. 2023;66(4):2211–2234. doi: 10.1021/acs.jmedchem.2c00757.</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.2c00757" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36739538/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.&amp;title=Small%20Molecular%20Mimetics%20of%20Antimicrobial%20Peptides%20as%20a%20Promising%20Therapy%20To%20Combat%20Bacterial%20Resistance&amp;author=P.%20Teng&amp;author=H.%20Shao&amp;author=B.%20Huang&amp;author=J.%20Xie&amp;author=S.%20Cui&amp;volume=66&amp;issue=4&amp;publication_year=2023&amp;pages=2211-2234&amp;pmid=36739538&amp;doi=10.1021/acs.jmedchem.2c00757&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>Kasprzykowski F., Schalén C., Kasprzykowska R., Jastrzebska B., Grubb A.. Synthesis and Antibacterial
Properties of Peptidyl Derivatives and
Cyclopeptides Structurally Based upon the Inhibitory Centre of Human
Cystatin C. Dissociation of Antiproteolytic and Antibacterial Effects. APMIS. 2000;108(7–8):473–481. doi: 10.1034/j.1600-0463.2000.01087-8473.x.</cite> [<a href="https://doi.org/10.1034/j.1600-0463.2000.01087-8473.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11167542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=APMIS&amp;title=Synthesis%20and%20Antibacterial%20Properties%20of%20Peptidyl%20Derivatives%20and%20Cyclopeptides%20Structurally%20Based%20upon%20the%20Inhibitory%20Centre%20of%20Human%20Cystatin%20C.%20Dissociation%20of%20Antiproteolytic%20and%20Antibacterial%20Effects&amp;author=F.%20Kasprzykowski&amp;author=C.%20Schal%C3%A9n&amp;author=R.%20Kasprzykowska&amp;author=B.%20Jastrzebska&amp;author=A.%20Grubb&amp;volume=108&amp;issue=7%E2%80%938&amp;publication_year=2000&amp;pages=473-481&amp;pmid=11167542&amp;doi=10.1034/j.1600-0463.2000.01087-8473.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>Jasir A., Kasprzykowski F., Kasprzykowska R., Lindström V., Schalén C., Grubb A.. New Antimicrobial Cystatin C-Based
Peptide Active against Gram-Positive Bacterial Pathogens, Including
Methicillin-Resistant and Multiresistant Coagulase-Negative Staphylococci. APMIS. 2003;111(11):1004–1010. doi: 10.1111/j.1600-0463.2003.t01-1-apm1111110.x.</cite> [<a href="https://doi.org/10.1111/j.1600-0463.2003.t01-1-apm1111110.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14629266/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=APMIS&amp;title=New%20Antimicrobial%20Cystatin%20C-Based%20Peptide%20Active%20against%20Gram-Positive%20Bacterial%20Pathogens,%20Including%20Methicillin-Resistant%20and%20Multiresistant%20Coagulase-Negative%20Staphylococci&amp;author=A.%20Jasir&amp;author=F.%20Kasprzykowski&amp;author=R.%20Kasprzykowska&amp;author=V.%20Lindstr%C3%B6m&amp;author=C.%20Schal%C3%A9n&amp;volume=111&amp;issue=11&amp;publication_year=2003&amp;pages=1004-1010&amp;pmid=14629266&amp;doi=10.1111/j.1600-0463.2003.t01-1-apm1111110.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Pikuła M., Smużyńska M., Krzystyniak A., Zieliński M., Langa P., Deptuła M., Schumacher A., Łata J., Cichorek M., Grubb A., Trzonkowski P., Kasprzykowski F., Rodziewicz-Motowidło S.. Cystatin C
Peptidomimetic Derivative with Antimicrobial Properties as a Potential
Compound against Wound Infections. Bioorg. Med.
Chem. 2017;25(4):1431–1439. doi: 10.1016/j.bmc.2017.01.004.</cite> [<a href="https://doi.org/10.1016/j.bmc.2017.01.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28110818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Med.%20Chem.&amp;title=Cystatin%20C%20Peptidomimetic%20Derivative%20with%20Antimicrobial%20Properties%20as%20a%20Potential%20Compound%20against%20Wound%20Infections&amp;author=M.%20Piku%C5%82a&amp;author=M.%20Smu%C5%BCy%C5%84ska&amp;author=A.%20Krzystyniak&amp;author=M.%20Zieli%C5%84ski&amp;author=P.%20Langa&amp;volume=25&amp;issue=4&amp;publication_year=2017&amp;pages=1431-1439&amp;pmid=28110818&amp;doi=10.1016/j.bmc.2017.01.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Dzierżyńska M., Sikorska E., Pogorzelska A., Mulkiewicz E., Sawicka J., Wyrzykowski D., Małuch I., Grubb A., Kasprzykowski F., Rodziewicz-Motowidło S.. Synthesis
and SAR Studies of Antibacterial Peptidyl Derivatives Based upon the
Binding Site of Human Cystatin C. Protein Pept.
Lett. 2019;26(6):423–434. doi: 10.2174/0929866526666190311162716.</cite> [<a href="https://doi.org/10.2174/0929866526666190311162716" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30864495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Pept.%20Lett.&amp;title=Synthesis%20and%20SAR%20Studies%20of%20Antibacterial%20Peptidyl%20Derivatives%20Based%20upon%20the%20Binding%20Site%20of%20Human%20Cystatin%20C&amp;author=M.%20Dzier%C5%BCy%C5%84ska&amp;author=E.%20Sikorska&amp;author=A.%20Pogorzelska&amp;author=E.%20Mulkiewicz&amp;author=J.%20Sawicka&amp;volume=26&amp;issue=6&amp;publication_year=2019&amp;pages=423-434&amp;pmid=30864495&amp;doi=10.2174/0929866526666190311162716&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Juszczyk P., Kasprzykowska R., Kołodziejczyk A.
S.. Simple and Efficient
Synthesis of Chiral Amino Alcohols with an Amino Acid-Based Skeleton. Lett. Pept. Sci. 2003;10(2):79–82. doi: 10.1023/B:LIPS.0000032366.28662.1d.</cite> [<a href="https://doi.org/10.1023/B:LIPS.0000032366.28662.1d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lett.%20Pept.%20Sci.&amp;title=Simple%20and%20Efficient%20Synthesis%20of%20Chiral%20Amino%20Alcohols%20with%20an%20Amino%20Acid-Based%20Skeleton&amp;author=P.%20Juszczyk&amp;author=R.%20Kasprzykowska&amp;author=A.%20S.%20Ko%C5%82odziejczyk&amp;volume=10&amp;issue=2&amp;publication_year=2003&amp;pages=79-82&amp;doi=10.1023/B:LIPS.0000032366.28662.1d&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Podlech J., Seebach D.. The Arndt–Eistert
Reaction in Peptide Chemistry:
A Facile Access to Homopeptides. Angew. Chem.,
Int. Ed. 1995;34(4):471–472. doi: 10.1002/anie.199504711.</cite> [<a href="https://doi.org/10.1002/anie.199504711" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.,%20Int.%20Ed.&amp;title=The%20Arndt%E2%80%93Eistert%20Reaction%20in%20Peptide%20Chemistry:%20A%20Facile%20Access%20to%20Homopeptides&amp;author=J.%20Podlech&amp;author=D.%20Seebach&amp;volume=34&amp;issue=4&amp;publication_year=1995&amp;pages=471-472&amp;doi=10.1002/anie.199504711&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Bachmann W. E., Struve W. S.. The Arndt-Eistert Synthesis. Org. React. 2004;1:38–62. doi: 10.1002/0471264180.or001.02.</cite> [<a href="https://doi.org/10.1002/0471264180.or001.02" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Org.%20React.&amp;title=The%20Arndt-Eistert%20Synthesis&amp;author=W.%20E.%20Bachmann&amp;author=W.%20S.%20Struve&amp;volume=1&amp;publication_year=2004&amp;pages=38-62&amp;doi=10.1002/0471264180.or001.02&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>Dzierżyńska M., Sawicka J., Deptuła M., Sosnowski P., Sass P., Peplińska B., Pietralik-Molińska Z., Fularczyk M., Kasprzykowski F., Zieliński J., Kozak M., Sachadyn P., Pikuła M., Rodziewicz-Motowidło S.. Release Systems Based on Self-Assembling
RADA16-I Hydrogels with a Signal Sequence Which Improves Wound Healing
Processes. Sci. Rep. 2023;13(1):6273.  doi: 10.1038/s41598-023-33464-w.</cite> [<a href="https://doi.org/10.1038/s41598-023-33464-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10113214/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37072464/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Release%20Systems%20Based%20on%20Self-Assembling%20RADA16-I%20Hydrogels%20with%20a%20Signal%20Sequence%20Which%20Improves%20Wound%20Healing%20Processes&amp;author=M.%20Dzier%C5%BCy%C5%84ska&amp;author=J.%20Sawicka&amp;author=M.%20Deptu%C5%82a&amp;author=P.%20Sosnowski&amp;author=P.%20Sass&amp;volume=13&amp;issue=1&amp;publication_year=2023&amp;pages=6273&amp;pmid=37072464&amp;doi=10.1038/s41598-023-33464-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Daina A., Michielin O., Zoete V.. SwissADME: A Free Web Tool to Evaluate
Pharmacokinetics, Drug-Likeness and Medicinal Chemistry Friendliness
of Small Molecules. Sci. Rep. 2017;7:42717.  doi: 10.1038/srep42717.</cite> [<a href="https://doi.org/10.1038/srep42717" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5335600/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28256516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=SwissADME:%20A%20Free%20Web%20Tool%20to%20Evaluate%20Pharmacokinetics,%20Drug-Likeness%20and%20Medicinal%20Chemistry%20Friendliness%20of%20Small%20Molecules&amp;author=A.%20Daina&amp;author=O.%20Michielin&amp;author=V.%20Zoete&amp;volume=7&amp;publication_year=2017&amp;pages=42717&amp;pmid=28256516&amp;doi=10.1038/srep42717&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>Pires D. E. V., Blundell T. L., Ascher D. B.. PkCSM:
Predicting Small-Molecule
Pharmacokinetic and Toxicity Properties Using Graph-Based Signatures. J. Med. Chem. 2015;58(9):4066–4072. doi: 10.1021/acs.jmedchem.5b00104.</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.5b00104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4434528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25860834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.&amp;title=PkCSM:%20Predicting%20Small-Molecule%20Pharmacokinetic%20and%20Toxicity%20Properties%20Using%20Graph-Based%20Signatures&amp;author=D.%20E.%20V.%20Pires&amp;author=T.%20L.%20Blundell&amp;author=D.%20B.%20Ascher&amp;volume=58&amp;issue=9&amp;publication_year=2015&amp;pages=4066-4072&amp;pmid=25860834&amp;doi=10.1021/acs.jmedchem.5b00104&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>

Kassambara, A.
; 
Mundt, F.
. 
factoextra: Extract
and Visualize the Results of Multivariate Data Analyses
CRAN: Contributed Packages, 10.32614/CRAN.package.factoextra.</cite> [<a href="https://doi.org/10.32614/CRAN.package.factoextra" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="ref31">
<cite>

Kuhn, M.

Building Predictive
Models in R Using the caret Package
J. Stat.
Software, 1
26
10.18637/jss.v028.i05.</cite> [<a href="https://doi.org/10.18637/jss.v028.i05" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Stat.%20Software&amp;title=Building%20Predictive%20Models%20in%20R%20Using%20the%20caret%20Package&amp;pages=1-26&amp;doi=10.18637/jss.v028.i05&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>

Wickham, H.

Ggplot2: Elegant Graphics
for Data Analysis; Springer-Verlag: New York, 2016. <a href="https://ggplot2.tidyverse.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ggplot2.tidyverse.org</a>.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Ggplot2:%20Elegant%20Graphics%20for%20Data%20Analysis&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>
Talete srl . Dragon Software
for Molecular Descriptor Calculation; Talete
srl: Milano, Italy, 2014. <a href="http://talete.mi.it" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://talete.mi.it</a> (accessed Aug 29, 2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Dragon%20Software%20for%20Molecular%20Descriptor%20Calculation&amp;publication_year=2014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34"><cite>

Stewart, J. J. P.

Stewart
Computational Chemistry, MOPAC.2016, 2016. <a href="http://openmopac.net/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://openmopac.net/</a> (accessed
Aug 29, 2024).</cite></li>
<li id="ref35">
<cite>Stewart J. J. P.. Optimization
of Parameters for Semiempirical Methods VI: More Modifications to
the NDDO Approximations and Re-Optimization of Parameters. J. Mol. Model. 2013;19(1):1–32. doi: 10.1007/s00894-012-1667-x.</cite> [<a href="https://doi.org/10.1007/s00894-012-1667-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3536963/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23187683/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Model&amp;title=Optimization%20of%20Parameters%20for%20Semiempirical%20Methods%20VI:%20More%20Modifications%20to%20the%20NDDO%20Approximations%20and%20Re-Optimization%20of%20Parameters&amp;author=J.%20J.%20P.%20Stewart&amp;volume=19&amp;issue=1&amp;publication_year=2013&amp;pages=1-32&amp;pmid=23187683&amp;doi=10.1007/s00894-012-1667-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>Puzyn T., Suzuki N., Haranczyk M., Rak J.. Calculation of Quantum-Mechanical
Descriptors for QSPR at the DFT Level: Is It Necessary? J. Chem. Inf. Model. 2008;48(6):1174–1180. doi: 10.1021/ci800021p.</cite> [<a href="https://doi.org/10.1021/ci800021p" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18510372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Inf.%20Model.&amp;title=Calculation%20of%20Quantum-Mechanical%20Descriptors%20for%20QSPR%20at%20the%20DFT%20Level:%20Is%20It%20Necessary?&amp;author=T.%20Puzyn&amp;author=N.%20Suzuki&amp;author=M.%20Haranczyk&amp;author=J.%20Rak&amp;volume=48&amp;issue=6&amp;publication_year=2008&amp;pages=1174-1180&amp;pmid=18510372&amp;doi=10.1021/ci800021p&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>Hilchie A. L., Wuerth K., Hancock R. E. W.. Immune
Modulation by Multifaceted
Cationic Host Defense (Antimicrobial) Peptides. Nat. Chem. Biol. 2013;9(12):761–768. doi: 10.1038/nchembio.1393.</cite> [<a href="https://doi.org/10.1038/nchembio.1393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24231617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Chem.%20Biol.&amp;title=Immune%20Modulation%20by%20Multifaceted%20Cationic%20Host%20Defense%20(Antimicrobial)%20Peptides&amp;author=A.%20L.%20Hilchie&amp;author=K.%20Wuerth&amp;author=R.%20E.%20W.%20Hancock&amp;volume=9&amp;issue=12&amp;publication_year=2013&amp;pages=761-768&amp;pmid=24231617&amp;doi=10.1038/nchembio.1393&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Imura Y., Choda N., Matsuzaki K.. Magainin 2
in Action: Distinct Modes
of Membrane Permeabilization in Living Bacterial and Mammalian Cells. Biophys. J. 2008;95(12):5757–5765. doi: 10.1529/biophysj.108.133488.</cite> [<a href="https://doi.org/10.1529/biophysj.108.133488" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2599864/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18835901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys.%20J.&amp;title=Magainin%202%20in%20Action:%20Distinct%20Modes%20of%20Membrane%20Permeabilization%20in%20Living%20Bacterial%20and%20Mammalian%20Cells&amp;author=Y.%20Imura&amp;author=N.%20Choda&amp;author=K.%20Matsuzaki&amp;volume=95&amp;issue=12&amp;publication_year=2008&amp;pages=5757-5765&amp;pmid=18835901&amp;doi=10.1529/biophysj.108.133488&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Memariani H., Memariani M., Shahidi-Dadras M., Nasiri S., Akhavan M. M., Moravvej H.. Melittin:
From Honeybees to Superbugs. Appl. Microbiol.
Biotechnol. 2019;103(8):3265–3276. doi: 10.1007/s00253-019-09698-y.</cite> [<a href="https://doi.org/10.1007/s00253-019-09698-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30824944/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&amp;title=Melittin:%20From%20Honeybees%20to%20Superbugs&amp;author=H.%20Memariani&amp;author=M.%20Memariani&amp;author=M.%20Shahidi-Dadras&amp;author=S.%20Nasiri&amp;author=M.%20M.%20Akhavan&amp;volume=103&amp;issue=8&amp;publication_year=2019&amp;pages=3265-3276&amp;pmid=30824944&amp;doi=10.1007/s00253-019-09698-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Selsted M. E., Novotny M. J., Morris W. L., Tang Y. Q., Smith W., Cullor J. S.. Indolicidin, a Novel Bactericidal
Tridecapeptide Amide
from Neutrophils. J. Biol. Chem. 1992;267(7):4292–4295. doi: 10.1016/S0021-9258(18)42830-X.</cite> [<a href="https://doi.org/10.1016/S0021-9258(18)42830-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1537821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biol.%20Chem.&amp;title=Indolicidin,%20a%20Novel%20Bactericidal%20Tridecapeptide%20Amide%20from%20Neutrophils&amp;author=M.%20E.%20Selsted&amp;author=M.%20J.%20Novotny&amp;author=W.%20L.%20Morris&amp;author=Y.%20Q.%20Tang&amp;author=W.%20Smith&amp;volume=267&amp;issue=7&amp;publication_year=1992&amp;pages=4292-4295&amp;pmid=1537821&amp;doi=10.1016/S0021-9258(18)42830-X&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>Jones E. M., Smart A., Bloomberg G., Burgess L., Millar M. R.. Lactoferricin,
a New Antimicrobial Peptide. J. Appl. Bacteriol. 1994;77(2):208–214. doi: 10.1111/j.1365-2672.1994.tb03065.x.</cite> [<a href="https://doi.org/10.1111/j.1365-2672.1994.tb03065.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7961192/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Appl.%20Bacteriol.&amp;title=Lactoferricin,%20a%20New%20Antimicrobial%20Peptide&amp;author=E.%20M.%20Jones&amp;author=A.%20Smart&amp;author=G.%20Bloomberg&amp;author=L.%20Burgess&amp;author=M.%20R.%20Millar&amp;volume=77&amp;issue=2&amp;publication_year=1994&amp;pages=208-214&amp;pmid=7961192&amp;doi=10.1111/j.1365-2672.1994.tb03065.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Wang L., Wang N., Zhang W., Cheng X., Yan Z., Shao G., Wang X., Wang R., Fu C.. Therapeutic
Peptides: Current Applications and Future Directions. Signal Transduction Targeted Ther. 2022;7(1):48.  doi: 10.1038/s41392-022-00904-4.</cite> [<a href="https://doi.org/10.1038/s41392-022-00904-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8844085/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35165272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Signal%20Transduction%20Targeted%20Ther.&amp;title=Therapeutic%20Peptides:%20Current%20Applications%20and%20Future%20Directions&amp;author=L.%20Wang&amp;author=N.%20Wang&amp;author=W.%20Zhang&amp;author=X.%20Cheng&amp;author=Z.%20Yan&amp;volume=7&amp;issue=1&amp;publication_year=2022&amp;pages=48&amp;pmid=35165272&amp;doi=10.1038/s41392-022-00904-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Müller D.. Smart PeptidesInnovative
Anwendungen in Biotechnologie Und Pharmazie. BIOspektrum. 2021;27(7):776–779. doi: 10.1007/s12268-021-1673-6.</cite> [<a href="https://doi.org/10.1007/s12268-021-1673-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BIOspektrum&amp;title=Smart%20Peptides%EE%97%B8Innovative%20Anwendungen%20in%20Biotechnologie%20Und%20Pharmazie&amp;author=D.%20M%C3%BCller&amp;volume=27&amp;issue=7&amp;publication_year=2021&amp;pages=776-779&amp;doi=10.1007/s12268-021-1673-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref44">
<cite>Arenas I., Villegas E., Walls O., Barrios H., Rodríguez R., Corzo G.. Antimicrobial Activity and Stability
of Short and Long Based Arachnid
Synthetic Peptides in the Presence of Commercial Antibiotics. Molecules. 2016;21(2):225.  doi: 10.3390/molecules21020225.</cite> [<a href="https://doi.org/10.3390/molecules21020225" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6273820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26901176/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Antimicrobial%20Activity%20and%20Stability%20of%20Short%20and%20Long%20Based%20Arachnid%20Synthetic%20Peptides%20in%20the%20Presence%20of%20Commercial%20Antibiotics&amp;author=I.%20Arenas&amp;author=E.%20Villegas&amp;author=O.%20Walls&amp;author=H.%20Barrios&amp;author=R.%20Rodr%C3%ADguez&amp;volume=21&amp;issue=2&amp;publication_year=2016&amp;pages=225&amp;pmid=26901176&amp;doi=10.3390/molecules21020225&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref45">
<cite>Shao C., Zhu Y., Lai Z., Tan P., Shan A.. Antimicrobial Peptides
with Protease Stability: Progress and Perspective. Future Med. Chem. 2019;11(16):2047–2050. doi: 10.4155/fmc-2019-0167.</cite> [<a href="https://doi.org/10.4155/fmc-2019-0167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31538527/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Future%20Med.%20Chem.&amp;title=Antimicrobial%20Peptides%20with%20Protease%20Stability:%20Progress%20and%20Perspective&amp;author=C.%20Shao&amp;author=Y.%20Zhu&amp;author=Z.%20Lai&amp;author=P.%20Tan&amp;author=A.%20Shan&amp;volume=11&amp;issue=16&amp;publication_year=2019&amp;pages=2047-2050&amp;pmid=31538527&amp;doi=10.4155/fmc-2019-0167&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref46">
<cite>Kumar P., Kizhakkedathu J., Straus S.. Antimicrobial Peptides: Diversity,
Mechanism of Action and Strategies to Improve the Activity and Biocompatibility
In Vivo. Biomolecules. 2018;8(1):4.  doi: 10.3390/biom8010004.</cite> [<a href="https://doi.org/10.3390/biom8010004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5871973/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29351202/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomolecules&amp;title=Antimicrobial%20Peptides:%20Diversity,%20Mechanism%20of%20Action%20and%20Strategies%20to%20Improve%20the%20Activity%20and%20Biocompatibility%20In%20Vivo&amp;author=P.%20Kumar&amp;author=J.%20Kizhakkedathu&amp;author=S.%20Straus&amp;volume=8&amp;issue=1&amp;publication_year=2018&amp;pages=4&amp;pmid=29351202&amp;doi=10.3390/biom8010004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref47">
<cite>Kant V., Jangir B. L., Kumar V.. Gross and
Histopathological Effects
of Dimethyl Sulfoxide on Wound Healing in Rats. Wound Med. 2020;30:100194.  doi: 10.1016/j.wndm.2020.100194.</cite> [<a href="https://doi.org/10.1016/j.wndm.2020.100194" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Wound%20Med.&amp;title=Gross%20and%20Histopathological%20Effects%20of%20Dimethyl%20Sulfoxide%20on%20Wound%20Healing%20in%20Rats&amp;author=V.%20Kant&amp;author=B.%20L.%20Jangir&amp;author=V.%20Kumar&amp;volume=30&amp;publication_year=2020&amp;pages=100194&amp;doi=10.1016/j.wndm.2020.100194&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref48">
<cite>Karim M., Boikess R. S., Schwartz R. A., Cohen P. J.. Dimethyl Sulfoxide
(DMSO): A Solvent That May Solve Selected Cutaneous Clinical Challenges. Arch. Dermatol. Res. 2023;315(6):1465–1472. doi: 10.1007/s00403-022-02494-1.</cite> [<a href="https://doi.org/10.1007/s00403-022-02494-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36459193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Arch.%20Dermatol.%20Res.&amp;title=Dimethyl%20Sulfoxide%20(DMSO):%20A%20Solvent%20That%20May%20Solve%20Selected%20Cutaneous%20Clinical%20Challenges&amp;author=M.%20Karim&amp;author=R.%20S.%20Boikess&amp;author=R.%20A.%20Schwartz&amp;author=P.%20J.%20Cohen&amp;volume=315&amp;issue=6&amp;publication_year=2023&amp;pages=1465-1472&amp;pmid=36459193&amp;doi=10.1007/s00403-022-02494-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref49">
<cite>Gupta R., Badhe Y., Rai B., Mitragotri S.. Molecular
Mechanism of the Skin Permeation Enhancing Effect of Ethanol: A Molecular
Dynamics Study. RSC Adv. 2020;10(21):12234–12248. doi: 10.1039/D0RA01692F.</cite> [<a href="https://doi.org/10.1039/D0RA01692F" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9050718/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35497613/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Adv.&amp;title=Molecular%20Mechanism%20of%20the%20Skin%20Permeation%20Enhancing%20Effect%20of%20Ethanol:%20A%20Molecular%20Dynamics%20Study&amp;author=R.%20Gupta&amp;author=Y.%20Badhe&amp;author=B.%20Rai&amp;author=S.%20Mitragotri&amp;volume=10&amp;issue=21&amp;publication_year=2020&amp;pages=12234-12248&amp;pmid=35497613&amp;doi=10.1039/D0RA01692F&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref50">
<cite>Zhang Y., Wang J., Qu F., Zhang Y., Su G., Zhao Y.. Hair Growth Promotion
Effect of Cedrol Cream and Its Dermatopharmacokinetics. RSC Adv. 2018;8(73):42170–42178. doi: 10.1039/C8RA08667B.</cite> [<a href="https://doi.org/10.1039/C8RA08667B" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9092075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35558774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Adv.&amp;title=Hair%20Growth%20Promotion%20Effect%20of%20Cedrol%20Cream%20and%20Its%20Dermatopharmacokinetics&amp;author=Y.%20Zhang&amp;author=J.%20Wang&amp;author=F.%20Qu&amp;author=Y.%20Zhang&amp;author=G.%20Su&amp;volume=8&amp;issue=73&amp;publication_year=2018&amp;pages=42170-42178&amp;pmid=35558774&amp;doi=10.1039/C8RA08667B&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref51">
<cite>Bacalum M., Janosi L., Zorila F., Tepes A. M., Ionescu C., Bogdan E., Hadade N., Craciun L., Grosu I., Turcu I., Radu M.. Modulating
Short Tryptophan- and
Arginine-Rich Peptides Activity by Substitution with Histidine. Biochim. Biophys. Acta, Gen. Subj. 2017;1861(7):1844–1854. doi: 10.1016/j.bbagen.2017.03.024.</cite> [<a href="https://doi.org/10.1016/j.bbagen.2017.03.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28372989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim.%20Biophys.%20Acta,%20Gen.%20Subj.&amp;title=Modulating%20Short%20Tryptophan-%20and%20Arginine-Rich%20Peptides%20Activity%20by%20Substitution%20with%20Histidine&amp;author=M.%20Bacalum&amp;author=L.%20Janosi&amp;author=F.%20Zorila&amp;author=A.%20M.%20Tepes&amp;author=C.%20Ionescu&amp;volume=1861&amp;issue=7&amp;publication_year=2017&amp;pages=1844-1854&amp;pmid=28372989&amp;doi=10.1016/j.bbagen.2017.03.024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref52">
<cite>Muller L., Jackson S. N., Woods A. S.. Histidine,
the Less Interactive Cousin
of Arginine. Eur. J. Mass Spectrom. 2019;25(2):212–218. doi: 10.1177/1469066718791793.</cite> [<a href="https://doi.org/10.1177/1469066718791793" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8269955/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31018697/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Mass%20Spectrom.&amp;title=Histidine,%20the%20Less%20Interactive%20Cousin%20of%20Arginine&amp;author=L.%20Muller&amp;author=S.%20N.%20Jackson&amp;author=A.%20S.%20Woods&amp;volume=25&amp;issue=2&amp;publication_year=2019&amp;pages=212-218&amp;pmid=31018697&amp;doi=10.1177/1469066718791793&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref53">
<cite>Koay Y. C., Richardson N. L., Zaiter S. S., Kho J., Nguyen S. Y., Tran D. H., Lee K. W., Buckton L. K., McAlpine S. R.. Hitting
a Moving Target: How Does an N-Methyl Group Impact Biological Activity? ChemMedChem. 2016;11(8):881–892. doi: 10.1002/cmdc.201500572.</cite> [<a href="https://doi.org/10.1002/cmdc.201500572" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26805515/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ChemMedChem&amp;title=Hitting%20a%20Moving%20Target:%20How%20Does%20an%20N-Methyl%20Group%20Impact%20Biological%20Activity?&amp;author=Y.%20C.%20Koay&amp;author=N.%20L.%20Richardson&amp;author=S.%20S.%20Zaiter&amp;author=J.%20Kho&amp;author=S.%20Y.%20Nguyen&amp;volume=11&amp;issue=8&amp;publication_year=2016&amp;pages=881-892&amp;pmid=26805515&amp;doi=10.1002/cmdc.201500572&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref54">
<cite>Song H., Burton A. J., Shirran S. L., Fahrig-Kamarauskaitė J., Kaspar H., Muir T. W., Künzler M., Naismith J. H.. Engineering of a Peptide α-N-Methyltransferase
to Methylate Non-Proteinogenic Amino Acids. Angew. Chem., Int. Ed. 2021;60(26):14319–14323. doi: 10.1002/anie.202100818.</cite> [<a href="https://doi.org/10.1002/anie.202100818" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8251615/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33856715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angew.%20Chem.,%20Int.%20Ed.&amp;title=Engineering%20of%20a%20Peptide%20%CE%B1-N-Methyltransferase%20to%20Methylate%20Non-Proteinogenic%20Amino%20Acids&amp;author=H.%20Song&amp;author=A.%20J.%20Burton&amp;author=S.%20L.%20Shirran&amp;author=J.%20Fahrig-Kamarauskait%C4%97&amp;author=H.%20Kaspar&amp;volume=60&amp;issue=26&amp;publication_year=2021&amp;pages=14319-14323&amp;pmid=33856715&amp;doi=10.1002/anie.202100818&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref55">
<cite>Mohammed
Ghilan A. K., Alharbi N. S., Khaled J. M., Kadaikunnan S., Alobaidi A. S.. Virulence Factors Analysis and Determination of the
Suitable Chemical Agent to Inhibit Growth and Biofilm Formation. J. King Saud
Univ., Sci. 2023;35(8):102892.  doi: 10.1016/j.jksus.2023.102892.</cite> [<a href="https://doi.org/10.1016/j.jksus.2023.102892" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20King%20Saud%20Univ.,%20Sci.&amp;title=Virulence%20Factors%20Analysis%20and%20Determination%20of%20the%20Suitable%20Chemical%20Agent%20to%20Inhibit%20Growth%20and%20Biofilm%20Formation&amp;author=A.%20K.%20Mohammed%20Ghilan&amp;author=N.%20S.%20Alharbi&amp;author=J.%20M.%20Khaled&amp;author=S.%20Kadaikunnan&amp;author=A.%20S.%20Alobaidi&amp;volume=35&amp;issue=8&amp;publication_year=2023&amp;pages=102892&amp;doi=10.1016/j.jksus.2023.102892&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref56">
<cite>Ying X., Xue G., Sun P., Gan Z., Fan Z., Liu B., Han Y., Yang J., Zhang J., Lu A.. Antimicrobial Peptides
Targeting : Current
Research on Design, Screening and Efficacy. Curr. Microbiol. 2024;81(1):18.  doi: 10.1007/s00284-023-03540-5.</cite> [<a href="https://doi.org/10.1007/s00284-023-03540-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38007405/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Microbiol.&amp;title=Antimicrobial%20Peptides%20Targeting%20:%20Current%20Research%20on%20Design,%20Screening%20and%20Efficacy&amp;author=X.%20Ying&amp;author=G.%20Xue&amp;author=P.%20Sun&amp;author=Z.%20Gan&amp;author=Z.%20Fan&amp;volume=81&amp;issue=1&amp;publication_year=2024&amp;pages=18&amp;pmid=38007405&amp;doi=10.1007/s00284-023-03540-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref57">
<cite>Lahouati M., Brousse X., Bientz L., Chadefaux G., Dubois V., Cazanave C., Xuereb F.. Vancomycin versus Linezolid
for Treatment of Staphylococcal-Associated Central Nervous System
Infections. BMC Infect. Dis. 2025;25(1):446.  doi: 10.1186/s12879-025-10834-5.</cite> [<a href="https://doi.org/10.1186/s12879-025-10834-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11959708/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40165119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Infect.%20Dis.&amp;title=Vancomycin%20versus%20Linezolid%20for%20Treatment%20of%20Staphylococcal-Associated%20Central%20Nervous%20System%20Infections&amp;author=M.%20Lahouati&amp;author=X.%20Brousse&amp;author=L.%20Bientz&amp;author=G.%20Chadefaux&amp;author=V.%20Dubois&amp;volume=25&amp;issue=1&amp;publication_year=2025&amp;pages=446&amp;pmid=40165119&amp;doi=10.1186/s12879-025-10834-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref58">
<cite>Zhang F., Liu F., Sheng X., Liu Q., Cui L., Cao Z., Hu T., Li D., Dai M.. Bacitracin-Resistant Induced in Chicken Gut and
in Vitro under Bacitracin Exposure. Microb.
Pathog. 2024;191:106666.  doi: 10.1016/j.micpath.2024.106666.</cite> [<a href="https://doi.org/10.1016/j.micpath.2024.106666" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38685360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microb.%20Pathog.&amp;title=Bacitracin-Resistant%20Induced%20in%20Chicken%20Gut%20and%20in%20Vitro%20under%20Bacitracin%20Exposure&amp;author=F.%20Zhang&amp;author=F.%20Liu&amp;author=X.%20Sheng&amp;author=Q.%20Liu&amp;author=L.%20Cui&amp;volume=191&amp;publication_year=2024&amp;pages=106666&amp;pmid=38685360&amp;doi=10.1016/j.micpath.2024.106666&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref59">
<cite>Canette A., Boudjemaa R., Deschamps J., Steenkeste K., Marlière C., Briandet R., Fontaine-Aupart M.-P.. Real-Time
Multimodal Imaging of Daptomycin Action on the Cell Wall of Adherent . BMC
Res. Notes. 2025;18(1):54.  doi: 10.1186/s13104-025-07130-5.</cite> [<a href="https://doi.org/10.1186/s13104-025-07130-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11796211/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39910624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Res.%20Notes&amp;title=Real-Time%20Multimodal%20Imaging%20of%20Daptomycin%20Action%20on%20the%20Cell%20Wall%20of%20Adherent&amp;author=A.%20Canette&amp;author=R.%20Boudjemaa&amp;author=J.%20Deschamps&amp;author=K.%20Steenkeste&amp;author=C.%20Marli%C3%A8re&amp;volume=18&amp;issue=1&amp;publication_year=2025&amp;pages=54&amp;pmid=39910624&amp;doi=10.1186/s13104-025-07130-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref60">
<cite>Wang L., Wang N., Zhang W., Cheng X., Yan Z., Shao G., Wang X., Wang R., Fu C.. Therapeutic
Peptides: Current Applications and Future Directions. Signal Transduction Targeted Ther. 2022;7(1):48.  doi: 10.1038/s41392-022-00904-4.</cite> [<a href="https://doi.org/10.1038/s41392-022-00904-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8844085/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35165272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Signal%20Transduction%20Targeted%20Ther.&amp;title=Therapeutic%20Peptides:%20Current%20Applications%20and%20Future%20Directions&amp;author=L.%20Wang&amp;author=N.%20Wang&amp;author=W.%20Zhang&amp;author=X.%20Cheng&amp;author=Z.%20Yan&amp;volume=7&amp;issue=1&amp;publication_year=2022&amp;pages=48&amp;pmid=35165272&amp;doi=10.1038/s41392-022-00904-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref61">
<cite>Lee M. F., Poh C. L.. Strategies to Improve
the Physicochemical Properties
of Peptide-Based Drugs. Pharm. Res. 2023;40(3):617–632. doi: 10.1007/s11095-023-03486-0.</cite> [<a href="https://doi.org/10.1007/s11095-023-03486-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36869247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharm.%20Res.&amp;title=Strategies%20to%20Improve%20the%20Physicochemical%20Properties%20of%20Peptide-Based%20Drugs&amp;author=M.%20F.%20Lee&amp;author=C.%20L.%20Poh&amp;volume=40&amp;issue=3&amp;publication_year=2023&amp;pages=617-632&amp;pmid=36869247&amp;doi=10.1007/s11095-023-03486-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref62">
<cite>Lai Z., Yuan X., Chen H., Zhu Y., Dong N., Shan A.. Strategies Employed in the Design
of Antimicrobial Peptides with
Enhanced Proteolytic Stability. Biotechnol.
Adv. 2022;59:107962.  doi: 10.1016/j.biotechadv.2022.107962.</cite> [<a href="https://doi.org/10.1016/j.biotechadv.2022.107962" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35452776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnol.%20Adv.&amp;title=Strategies%20Employed%20in%20the%20Design%20of%20Antimicrobial%20Peptides%20with%20Enhanced%20Proteolytic%20Stability&amp;author=Z.%20Lai&amp;author=X.%20Yuan&amp;author=H.%20Chen&amp;author=Y.%20Zhu&amp;author=N.%20Dong&amp;volume=59&amp;publication_year=2022&amp;pages=107962&amp;pmid=35452776&amp;doi=10.1016/j.biotechadv.2022.107962&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref63">
<cite>Bell L. N.. Peptide
Stability in Solids and Solutions. Biotechnol.
Prog. 1997;13(4):342–346. doi: 10.1021/bp970057y.</cite> [<a href="https://doi.org/10.1021/bp970057y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnol.%20Prog.&amp;title=Peptide%20Stability%20in%20Solids%20and%20Solutions&amp;author=L.%20N.%20Bell&amp;volume=13&amp;issue=4&amp;publication_year=1997&amp;pages=342-346&amp;doi=10.1021/bp970057y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref64">
<cite>Wellner N., Marsh J. T., Savage A. W. J., Halford N. G., Shewry P. R., Mills E. N. C., Belton P. S.. Comparison of Repetitive Sequences
Derived from High Molecular Weight Subunits of Wheat Glutenin, an
Elastomeric Plant Protein. Biomacromolecules. 2006;7(4):1096–1103. doi: 10.1021/bm050893t.</cite> [<a href="https://doi.org/10.1021/bm050893t" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16602726/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomacromolecules&amp;title=Comparison%20of%20Repetitive%20Sequences%20Derived%20from%20High%20Molecular%20Weight%20Subunits%20of%20Wheat%20Glutenin,%20an%20Elastomeric%20Plant%20Protein&amp;author=N.%20Wellner&amp;author=J.%20T.%20Marsh&amp;author=A.%20W.%20J.%20Savage&amp;author=N.%20G.%20Halford&amp;author=P.%20R.%20Shewry&amp;volume=7&amp;issue=4&amp;publication_year=2006&amp;pages=1096-1103&amp;pmid=16602726&amp;doi=10.1021/bm050893t&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref65">
<cite>Sarma R., Wong K. Y., Lynch G. C., Pettitt B. M.. Peptide Solubility
Limits: Backbone and Side-Chain Interactions. J. Phys. Chem. B. 2018;122(13):3528–3539. doi: 10.1021/acs.jpcb.7b10734.</cite> [<a href="https://doi.org/10.1021/acs.jpcb.7b10734" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5909690/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29384681/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Phys.%20Chem.%20B&amp;title=Peptide%20Solubility%20Limits:%20Backbone%20and%20Side-Chain%20Interactions&amp;author=R.%20Sarma&amp;author=K.%20Y.%20Wong&amp;author=G.%20C.%20Lynch&amp;author=B.%20M.%20Pettitt&amp;volume=122&amp;issue=13&amp;publication_year=2018&amp;pages=3528-3539&amp;pmid=29384681&amp;doi=10.1021/acs.jpcb.7b10734&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref66">
<cite>Ou-uchi S.-k., Yang J.-Y., Lee J.-S., Murakawa Y., Narita M.. Only Weak
Dependence of the Protected Peptides Solubility in Organic Solvents
on Their Amino Acid Sequence. Polym. J. 1996;28(12):1033–1038. doi: 10.1295/polymj.28.1033.</cite> [<a href="https://doi.org/10.1295/polymj.28.1033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Polym.%20J.&amp;title=Only%20Weak%20Dependence%20of%20the%20Protected%20Peptides%20Solubility%20in%20Organic%20Solvents%20on%20Their%20Amino%20Acid%20Sequence&amp;author=S.-k.%20Ou-uchi&amp;author=J.-Y.%20Yang&amp;author=J.-S.%20Lee&amp;author=Y.%20Murakawa&amp;author=M.%20Narita&amp;volume=28&amp;issue=12&amp;publication_year=1996&amp;pages=1033-1038&amp;doi=10.1295/polymj.28.1033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref67">
<cite>Nadaraia N. S., Amiranashvili L. S., Merlani M., Kakhabrishvili M. L., Barbakadze N. N., Geronikaki A., Petrou A., Poroikov V., Ciric A., Glamoclija J., Sokovic M.. Novel Antimicrobial
Agents’ Discovery among the Steroid Derivatives. Steroids. 2019;144:52–65. doi: 10.1016/j.steroids.2019.02.012.</cite> [<a href="https://doi.org/10.1016/j.steroids.2019.02.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30776376/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Steroids&amp;title=Novel%20Antimicrobial%20Agents%E2%80%99%20Discovery%20among%20the%20Steroid%20Derivatives&amp;author=N.%20S.%20Nadaraia&amp;author=L.%20S.%20Amiranashvili&amp;author=M.%20Merlani&amp;author=M.%20L.%20Kakhabrishvili&amp;author=N.%20N.%20Barbakadze&amp;volume=144&amp;publication_year=2019&amp;pages=52-65&amp;pmid=30776376&amp;doi=10.1016/j.steroids.2019.02.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref68">
<cite>Depa N., Erothu H.. Anti-bacterial activity
and in silico studies of perillyl-4H-pyrantriazole
derivatives. Rasayan J. Chem. 2022;15(3):1668–1677. doi: 10.31788/RJC.2022.1536869.</cite> [<a href="https://doi.org/10.31788/RJC.2022.1536869" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rasayan%20J.%20Chem.&amp;title=Anti-bacterial%20activity%20and%20in%20silico%20studies%20of%20perillyl-4H-pyrantriazole%20derivatives&amp;author=N.%20Depa&amp;author=H.%20Erothu&amp;volume=15&amp;issue=3&amp;publication_year=2022&amp;pages=1668-1677&amp;doi=10.31788/RJC.2022.1536869&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref69">
<cite>Hassan A., Askar A., Nossier E., Naglah A., Moustafa G., Al-Omar M.. Antibacterial Evaluation, In Silico Characters and
Molecular Docking of Schiff Bases Derived from 5-Aminopyrazoles. Molecules. 2019;24(17):3130.  doi: 10.3390/molecules24173130.</cite> [<a href="https://doi.org/10.3390/molecules24173130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6749370/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31466322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Antibacterial%20Evaluation,%20In%20Silico%20Characters%20and%20Molecular%20Docking%20of%20Schiff%20Bases%20Derived%20from%205-Aminopyrazoles&amp;author=A.%20Hassan&amp;author=A.%20Askar&amp;author=E.%20Nossier&amp;author=A.%20Naglah&amp;author=G.%20Moustafa&amp;volume=24&amp;issue=17&amp;publication_year=2019&amp;pages=3130&amp;pmid=31466322&amp;doi=10.3390/molecules24173130&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref70">
<cite>Merlani M., Nadaraia N., Amiranashvili L., Petrou A., Geronikaki A., Ciric A., Glamoclija J., Carevic T., Sokovic M.. Antimicrobial
Activity of Some Steroidal Hydrazones. Molecules. 2023;28(3):1167.  doi: 10.3390/molecules28031167.</cite> [<a href="https://doi.org/10.3390/molecules28031167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9920613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36770834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Antimicrobial%20Activity%20of%20Some%20Steroidal%20Hydrazones&amp;author=M.%20Merlani&amp;author=N.%20Nadaraia&amp;author=L.%20Amiranashvili&amp;author=A.%20Petrou&amp;author=A.%20Geronikaki&amp;volume=28&amp;issue=3&amp;publication_year=2023&amp;pages=1167&amp;pmid=36770834&amp;doi=10.3390/molecules28031167&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref71">
<cite>Manjunatha K. S., Satyanarayan N. D., Harishkumar S.. Antimicrobial and in silico admet
screening of novel (E)-N-(2-(1H-indol-3-yl-amino) vinyl)-3-(1-methyl-1H-indol-3-yl)-3-phenylpropanamide
derivatives. Int. J. Pharm. Pharm. Sci. 2016;8(10):251. doi: 10.22159/ijpps.2016v8i10.13957.</cite> [<a href="https://doi.org/10.22159/ijpps.2016v8i10.13957" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Pharm.%20Pharm.%20Sci.&amp;title=Antimicrobial%20and%20in%20silico%20admet%20screening%20of%20novel%20(E)-N-(2-(1H-indol-3-yl-amino)%20vinyl)-3-(1-methyl-1H-indol-3-yl)-3-phenylpropanamide%20derivatives&amp;author=K.%20S.%20Manjunatha&amp;author=N.%20D.%20Satyanarayan&amp;author=S.%20Harishkumar&amp;volume=8&amp;issue=10&amp;publication_year=2016&amp;pages=251&amp;doi=10.22159/ijpps.2016v8i10.13957&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref72">
<cite>Mordi I. R., Chan B. K., Yanez N. D., Palmer C. N. A., Lang C. C., Chalmers J. D.. Genetic and Pharmacological
Relationship between P-Glycoprotein
and Increased Cardiovascular Risk Associated with Clarithromycin Prescription:
An Epidemiological and Genomic Population-Based Cohort Study in Scotland,
UK. PLoS Med. 2020;17(11):e1003372.  doi: 10.1371/journal.pmed.1003372.</cite> [<a href="https://doi.org/10.1371/journal.pmed.1003372" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7682888/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33226983/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Med.&amp;title=Genetic%20and%20Pharmacological%20Relationship%20between%20P-Glycoprotein%20and%20Increased%20Cardiovascular%20Risk%20Associated%20with%20Clarithromycin%20Prescription:%20An%20Epidemiological%20and%20Genomic%20Population-Based%20Cohort%20Study%20in%20Scotland,%20UK&amp;author=I.%20R.%20Mordi&amp;author=B.%20K.%20Chan&amp;author=N.%20D.%20Yanez&amp;author=C.%20N.%20A.%20Palmer&amp;author=C.%20C.%20Lang&amp;volume=17&amp;issue=11&amp;publication_year=2020&amp;pages=e1003372&amp;pmid=33226983&amp;doi=10.1371/journal.pmed.1003372&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref73">
<cite>Constantinides P. P., Wasan K. M.. Lipid Formulation
Strategies for Enhancing Intestinal
Transport and Absorption of P-Glycoprotein (P-Gp) Substrate Drugs:
In Vitro/In Vivo Case Studies. J. Pharm. Sci. 2007;96(2):235–248. doi: 10.1002/jps.20780.</cite> [<a href="https://doi.org/10.1002/jps.20780" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17051593/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Pharm.%20Sci.&amp;title=Lipid%20Formulation%20Strategies%20for%20Enhancing%20Intestinal%20Transport%20and%20Absorption%20of%20P-Glycoprotein%20(P-Gp)%20Substrate%20Drugs:%20In%20Vitro/In%20Vivo%20Case%20Studies&amp;author=P.%20P.%20Constantinides&amp;author=K.%20M.%20Wasan&amp;volume=96&amp;issue=2&amp;publication_year=2007&amp;pages=235-248&amp;pmid=17051593&amp;doi=10.1002/jps.20780&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref74">
<cite>Lee C. K., Ki S. H., Choi J. S.. Effects
of Oral Curcumin on the Pharmacokinetics
of Intravenous and Oral Etoposide in Rats: Possible Role of Intestinal
CYP3A and P-Gp Inhibition by Curcumin. Biopharm.
Drug Dispos. 2011;32(4):245–251. doi: 10.1002/bdd.754.</cite> [<a href="https://doi.org/10.1002/bdd.754" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21506134/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biopharm.%20Drug%20Dispos.&amp;title=Effects%20of%20Oral%20Curcumin%20on%20the%20Pharmacokinetics%20of%20Intravenous%20and%20Oral%20Etoposide%20in%20Rats:%20Possible%20Role%20of%20Intestinal%20CYP3A%20and%20P-Gp%20Inhibition%20by%20Curcumin&amp;author=C.%20K.%20Lee&amp;author=S.%20H.%20Ki&amp;author=J.%20S.%20Choi&amp;volume=32&amp;issue=4&amp;publication_year=2011&amp;pages=245-251&amp;pmid=21506134&amp;doi=10.1002/bdd.754&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref75">
<cite>Kyaw T. S., Zhang C., Sandy M., Trepka K., Zhang S., Ramirez Hernandez L. A., Ramirez L., Goh J. J. N., Yu K., Dimassa V., Bess E. N., Brockert J. G., Dumlao D. S., Bisanz J. E., Turnbaugh P. J.. Human Gut Actinobacteria Boost Drug
Absorption by Secreting P-Glycoprotein ATPase Inhibitors. iScience. 2024;27(6):110122.  doi: 10.1016/j.isci.2024.110122.</cite> [<a href="https://doi.org/10.1016/j.isci.2024.110122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11214321/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38947502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=iScience&amp;title=Human%20Gut%20Actinobacteria%20Boost%20Drug%20Absorption%20by%20Secreting%20P-Glycoprotein%20ATPase%20Inhibitors&amp;author=T.%20S.%20Kyaw&amp;author=C.%20Zhang&amp;author=M.%20Sandy&amp;author=K.%20Trepka&amp;author=S.%20Zhang&amp;volume=27&amp;issue=6&amp;publication_year=2024&amp;pages=110122&amp;pmid=38947502&amp;doi=10.1016/j.isci.2024.110122&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref76">
<cite>Torambe P. V., Chabukswar A. R., Dhamane S. P., Jagdale S. C.. Microemulgel
Delivery
of Ciprofloxacin Hydrochloride. Int. J. Drug
Delivery Technol. 2023;13(4):1222–1228. doi: 10.25258/ijddt.13.4.17.</cite> [<a href="https://doi.org/10.25258/ijddt.13.4.17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Drug%20Delivery%20Technol.&amp;title=Microemulgel%20Delivery%20of%20Ciprofloxacin%20Hydrochloride&amp;author=P.%20V.%20Torambe&amp;author=A.%20R.%20Chabukswar&amp;author=S.%20P.%20Dhamane&amp;author=S.%20C.%20Jagdale&amp;volume=13&amp;issue=4&amp;publication_year=2023&amp;pages=1222-1228&amp;doi=10.25258/ijddt.13.4.17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref77">
<cite>

Brayfield, A.

Martindale: The Complete
Drug Reference, 38th ed.; Pharmaceutical Press, 2014.</cite> [<a href="https://doi.org/10.7748/en.22.5.12.s13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25185909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Martindale:%20The%20Complete%20Drug%20Reference&amp;publication_year=2014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref78">
<cite>Alhasso B., Ghori M. U., Rout S., Conway B. R.. Development
of a
Nanoemulgel for the Topical Application of Mupirocin. Pharmaceutics. 2023;15(10):2387.  doi: 10.3390/pharmaceutics15102387.</cite> [<a href="https://doi.org/10.3390/pharmaceutics15102387" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10610056/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37896147/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmaceutics&amp;title=Development%20of%20a%20Nanoemulgel%20for%20the%20Topical%20Application%20of%20Mupirocin&amp;author=B.%20Alhasso&amp;author=M.%20U.%20Ghori&amp;author=S.%20Rout&amp;author=B.%20R.%20Conway&amp;volume=15&amp;issue=10&amp;publication_year=2023&amp;pages=2387&amp;pmid=37896147&amp;doi=10.3390/pharmaceutics15102387&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref79">
<cite>Rahimpour Y., Javadzadeh Y., Hamishehkar H.. Solid Lipid Microparticles for Enhanced
Dermal Delivery of Tetracycline HCl. Colloids
Surf., B. 2016;145:14–20. doi: 10.1016/j.colsurfb.2016.04.034.</cite> [<a href="https://doi.org/10.1016/j.colsurfb.2016.04.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27131093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Colloids%20Surf.,%20B&amp;title=Solid%20Lipid%20Microparticles%20for%20Enhanced%20Dermal%20Delivery%20of%20Tetracycline%20HCl&amp;author=Y.%20Rahimpour&amp;author=Y.%20Javadzadeh&amp;author=H.%20Hamishehkar&amp;volume=145&amp;publication_year=2016&amp;pages=14-20&amp;pmid=27131093&amp;doi=10.1016/j.colsurfb.2016.04.034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref80">
<cite>Sulistiawati, Saka Dwipayanti K., Azhar M., Rahman L., Pakki E., Himawan A., Permana A. D.. Enhanced Skin Localization of Metronidazole Using Solid
Lipid Microparticles Incorporated into Polymeric Hydrogels for Potential
Improved of Rosacea Treatment: An Ex Vivo Proof of Concept Investigation. Int. J. Pharm. 2022;628:122327.  doi: 10.1016/j.ijpharm.2022.122327.</cite> [<a href="https://doi.org/10.1016/j.ijpharm.2022.122327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36280218/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Pharm.&amp;title=Enhanced%20Skin%20Localization%20of%20Metronidazole%20Using%20Solid%20Lipid%20Microparticles%20Incorporated%20into%20Polymeric%20Hydrogels%20for%20Potential%20Improved%20of%20Rosacea%20Treatment:%20An%20Ex%20Vivo%20Proof%20of%20Concept%20Investigation&amp;author=%20Sulistiawati&amp;author=K.%20Saka%20Dwipayanti&amp;author=M.%20Azhar&amp;author=L.%20Rahman&amp;author=E.%20Pakki&amp;volume=628&amp;publication_year=2022&amp;pages=122327&amp;pmid=36280218&amp;doi=10.1016/j.ijpharm.2022.122327&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref81">
<cite>Dahlizar S., Futaki M., Okada A., Yatomi C., Todo H., Sugibayashi K.. Combined Use of N-Palmitoyl-Glycine-Histidine
Gel and
Several Penetration Enhancers on the Skin Permeation and Concentration
of Metronidazole. Pharmaceutics. 2018;10(4):163.  doi: 10.3390/pharmaceutics10040163.</cite> [<a href="https://doi.org/10.3390/pharmaceutics10040163" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6321412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30241371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmaceutics&amp;title=Combined%20Use%20of%20N-Palmitoyl-Glycine-Histidine%20Gel%20and%20Several%20Penetration%20Enhancers%20on%20the%20Skin%20Permeation%20and%20Concentration%20of%20Metronidazole&amp;author=S.%20Dahlizar&amp;author=M.%20Futaki&amp;author=A.%20Okada&amp;author=C.%20Yatomi&amp;author=H.%20Todo&amp;volume=10&amp;issue=4&amp;publication_year=2018&amp;pages=163&amp;pmid=30241371&amp;doi=10.3390/pharmaceutics10040163&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref82">
<cite>Kearney B. P., Aweeka F. T.. The penetration
of anti-infectives into the central
nervous system. Neurol. Clin. 1999;17(4):883–900. doi: 10.1016/S0733-8619(05)70171-7.</cite> [<a href="https://doi.org/10.1016/S0733-8619(05)70171-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10517933/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neurol.%20Clin.&amp;title=The%20penetration%20of%20anti-infectives%20into%20the%20central%20nervous%20system&amp;author=B.%20P.%20Kearney&amp;author=F.%20T.%20Aweeka&amp;volume=17&amp;issue=4&amp;publication_year=1999&amp;pages=883-900&amp;pmid=10517933&amp;doi=10.1016/S0733-8619(05)70171-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref83">
<cite>Clumeck N., Thys J. P., Vanhoof R., Vanderlinden M. P., Butzler J. P., Yourassowsky E.. Amoxicillin
Entry into Human Cerebrospinal
Fluid: Comparison with Ampicillin. Antimicrob.
Agents Chemother. 1978;14(4):531–532. doi: 10.1128/AAC.14.4.531.</cite> [<a href="https://doi.org/10.1128/AAC.14.4.531" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC352502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/102244/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antimicrob.%20Agents%20Chemother.&amp;title=Amoxicillin%20Entry%20into%20Human%20Cerebrospinal%20Fluid:%20Comparison%20with%20Ampicillin&amp;author=N.%20Clumeck&amp;author=J.%20P.%20Thys&amp;author=R.%20Vanhoof&amp;author=M.%20P.%20Vanderlinden&amp;author=J.%20P.%20Butzler&amp;volume=14&amp;issue=4&amp;publication_year=1978&amp;pages=531-532&amp;pmid=102244&amp;doi=10.1128/AAC.14.4.531&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref84">
<cite>Rybak M., Lomaestro B., Rotschafer J. C., Moellering R., Craig W., Billeter M., Dalovisio J. R., Levine D. P.. Therapeutic Monitoring of Vancomycin in Adult Patients:
A Consensus Review of the American Society of Health-System Pharmacists,
the Infectious Diseases Society of America, and the Society of Infectious
Diseases Pharmacists. Am. J. HealthSyst.
Pharm. 2009;66(1):82–98. doi: 10.2146/ajhp080434.</cite> [<a href="https://doi.org/10.2146/ajhp080434" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19106348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Health%EE%97%B8Syst.%20Pharm.&amp;title=Therapeutic%20Monitoring%20of%20Vancomycin%20in%20Adult%20Patients:%20A%20Consensus%20Review%20of%20the%20American%20Society%20of%20Health-System%20Pharmacists,%20the%20Infectious%20Diseases%20Society%20of%20America,%20and%20the%20Society%20of%20Infectious%20Diseases%20Pharmacists&amp;author=M.%20Rybak&amp;author=B.%20Lomaestro&amp;author=J.%20C.%20Rotschafer&amp;author=R.%20Moellering&amp;author=W.%20Craig&amp;volume=66&amp;issue=1&amp;publication_year=2009&amp;pages=82-98&amp;pmid=19106348&amp;doi=10.2146/ajhp080434&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref85">
<cite>Triggs E., Charles B.. Pharmacokinetics and Therapeutic Drug Monitoring of
Gentamicin in the Elderly. Clin. Pharmacokinet. 1999;37(4):331–341. doi: 10.2165/00003088-199937040-00004.</cite> [<a href="https://doi.org/10.2165/00003088-199937040-00004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10554048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin.%20Pharmacokinet.&amp;title=Pharmacokinetics%20and%20Therapeutic%20Drug%20Monitoring%20of%20Gentamicin%20in%20the%20Elderly&amp;author=E.%20Triggs&amp;author=B.%20Charles&amp;volume=37&amp;issue=4&amp;publication_year=1999&amp;pages=331-341&amp;pmid=10554048&amp;doi=10.2165/00003088-199937040-00004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref86">
<cite>Zaric R. Z., Milovanovic J., Rosic N., Milovanovic D., Zecevic D. R., Folic M., Jankovic S.. Pharmacokinetics of
Vancomycin in Patients with Different Renal Function Levels. Open Med. 2018;13(1):512–519. doi: 10.1515/med-2018-0068.</cite> [<a href="https://doi.org/10.1515/med-2018-0068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6227840/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30426090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Open%20Med.&amp;title=Pharmacokinetics%20of%20Vancomycin%20in%20Patients%20with%20Different%20Renal%20Function%20Levels&amp;author=R.%20Z.%20Zaric&amp;author=J.%20Milovanovic&amp;author=N.%20Rosic&amp;author=D.%20Milovanovic&amp;author=D.%20R.%20Zecevic&amp;volume=13&amp;issue=1&amp;publication_year=2018&amp;pages=512-519&amp;pmid=30426090&amp;doi=10.1515/med-2018-0068&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref87">
<cite>Fontana R. J., Shakil A. O., Greenson J. K., Boyd I., Lee W. M.. Acute Liver
Failure Due to Amoxicillin and Amoxicillin/Clavulanate. Dig. Dis. Sci. 2005;50(10):1785–1790. doi: 10.1007/s10620-005-2938-5.</cite> [<a href="https://doi.org/10.1007/s10620-005-2938-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16187174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dig.%20Dis.%20Sci.&amp;title=Acute%20Liver%20Failure%20Due%20to%20Amoxicillin%20and%20Amoxicillin/Clavulanate&amp;author=R.%20J.%20Fontana&amp;author=A.%20O.%20Shakil&amp;author=J.%20K.%20Greenson&amp;author=I.%20Boyd&amp;author=W.%20M.%20Lee&amp;volume=50&amp;issue=10&amp;publication_year=2005&amp;pages=1785-1790&amp;pmid=16187174&amp;doi=10.1007/s10620-005-2938-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref88">
<cite>Heaton P. C., Fenwick S. R., Brewer D. E.. Association
between Tetracycline
or Doxycycline and Hepatotoxicity: A Population Based Case-Control
Study. J. Clin. Pharm. Ther. 2007;32(5):483–487. doi: 10.1111/j.1365-2710.2007.00853.x.</cite> [<a href="https://doi.org/10.1111/j.1365-2710.2007.00853.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17875115/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Pharm.%20Ther.&amp;title=Association%20between%20Tetracycline%20or%20Doxycycline%20and%20Hepatotoxicity:%20A%20Population%20Based%20Case-Control%20Study&amp;author=P.%20C.%20Heaton&amp;author=S.%20R.%20Fenwick&amp;author=D.%20E.%20Brewer&amp;volume=32&amp;issue=5&amp;publication_year=2007&amp;pages=483-487&amp;pmid=17875115&amp;doi=10.1111/j.1365-2710.2007.00853.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref89">
<cite>Leonardi A., Papa V., Fregona I., Russo P., Franchis G. D., Milazzo G.. In Vitro Effects of
Fluoroquinolone and Aminoglycoside
Antibiotics on Human Keratocytes. Cornea. 2006;25(1):85–90. doi: 10.1097/01.ico.0000164782.10667.ed.</cite> [<a href="https://doi.org/10.1097/01.ico.0000164782.10667.ed" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16331047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cornea&amp;title=In%20Vitro%20Effects%20of%20Fluoroquinolone%20and%20Aminoglycoside%20Antibiotics%20on%20Human%20Keratocytes&amp;author=A.%20Leonardi&amp;author=V.%20Papa&amp;author=I.%20Fregona&amp;author=P.%20Russo&amp;author=G.%20D.%20Franchis&amp;volume=25&amp;issue=1&amp;publication_year=2006&amp;pages=85-90&amp;pmid=16331047&amp;doi=10.1097/01.ico.0000164782.10667.ed&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref90">
<cite>Cooper M. L., Boyce S. T., Hansbrough J. F., Foreman T. J., Frank D. H.. Cytotoxicity
to Cultured Human Keratinocytes of Topical Antimicrobial Agents. J. Surg. Res. 1990;48(3):190–195. doi: 10.1016/0022-4804(90)90212-K.</cite> [<a href="https://doi.org/10.1016/0022-4804(90)90212-K" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2314091/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Surg.%20Res.&amp;title=Cytotoxicity%20to%20Cultured%20Human%20Keratinocytes%20of%20Topical%20Antimicrobial%20Agents&amp;author=M.%20L.%20Cooper&amp;author=S.%20T.%20Boyce&amp;author=J.%20F.%20Hansbrough&amp;author=T.%20J.%20Foreman&amp;author=D.%20H.%20Frank&amp;volume=48&amp;issue=3&amp;publication_year=1990&amp;pages=190-195&amp;pmid=2314091&amp;doi=10.1016/0022-4804(90)90212-K&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref91">
<cite>Avalos A., Haza A. I., Mateo D., Morales P.. Interactions of Manufactured
Silver Nanoparticles of Different Sizes with Normal Human Dermal Fibroblasts. Int. Wound J. 2016;13(1):101–109. doi: 10.1111/iwj.12244.</cite> [<a href="https://doi.org/10.1111/iwj.12244" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7949580/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24612846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20Wound%20J.&amp;title=Interactions%20of%20Manufactured%20Silver%20Nanoparticles%20of%20Different%20Sizes%20with%20Normal%20Human%20Dermal%20Fibroblasts&amp;author=A.%20Avalos&amp;author=A.%20I.%20Haza&amp;author=D.%20Mateo&amp;author=P.%20Morales&amp;volume=13&amp;issue=1&amp;publication_year=2016&amp;pages=101-109&amp;pmid=24612846&amp;doi=10.1111/iwj.12244&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref92">
<cite>Lim L. M., Ly N., Anderson D., Yang J. C., Macander L., Jarkowski A., Forrest A., Bulitta J. B., Tsuji B. T.. Resurgence of Colistin:
A Review of Resistance, Toxicity, Pharmacodynamics, and Dosing. Pharmacotherapy. 2010;30(12):1279–1291. doi: 10.1592/phco.30.12.1279.</cite> [<a href="https://doi.org/10.1592/phco.30.12.1279" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4410713/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21114395/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacotherapy&amp;title=Resurgence%20of%20Colistin:%20A%20Review%20of%20Resistance,%20Toxicity,%20Pharmacodynamics,%20and%20Dosing&amp;author=L.%20M.%20Lim&amp;author=N.%20Ly&amp;author=D.%20Anderson&amp;author=J.%20C.%20Yang&amp;author=L.%20Macander&amp;volume=30&amp;issue=12&amp;publication_year=2010&amp;pages=1279-1291&amp;pmid=21114395&amp;doi=10.1592/phco.30.12.1279&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref93">
<cite>Ordooei
Javan A., Shokouhi S., Sahraei Z.. A Review on Colistin
Nephrotoxicity. Eur. J. Clin. Pharmacol. 2015;71(7):801–810. doi: 10.1007/s00228-015-1865-4.</cite> [<a href="https://doi.org/10.1007/s00228-015-1865-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26008213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Clin.%20Pharmacol.&amp;title=A%20Review%20on%20Colistin%20Nephrotoxicity&amp;author=A.%20Ordooei%20Javan&amp;author=S.%20Shokouhi&amp;author=Z.%20Sahraei&amp;volume=71&amp;issue=7&amp;publication_year=2015&amp;pages=801-810&amp;pmid=26008213&amp;doi=10.1007/s00228-015-1865-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref94">
<cite>Nagar J., Siddiqui M.. An Updated Review on Pharmacology and Toxicities Related
to Chloramphenicol. Asian J. Pharm. Res. Dev. 2020;8(4):104–109. doi: 10.22270/ajprd.v8i4.671.</cite> [<a href="https://doi.org/10.22270/ajprd.v8i4.671" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Asian%20J.%20Pharm.%20Res.%20Dev.&amp;title=An%20Updated%20Review%20on%20Pharmacology%20and%20Toxicities%20Related%20to%20Chloramphenicol&amp;author=J.%20Nagar&amp;author=M.%20Siddiqui&amp;volume=8&amp;issue=4&amp;publication_year=2020&amp;pages=104-109&amp;doi=10.22270/ajprd.v8i4.671&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref95">
<cite>Bamgbola O.. Review of
Vancomycin-Induced Renal Toxicity: An Update. Ther. Adv. Endocrinol. Metab. 2016;7(3):136–147. doi: 10.1177/2042018816638223.</cite> [<a href="https://doi.org/10.1177/2042018816638223" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4892398/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27293542/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ther.%20Adv.%20Endocrinol.%20Metab.&amp;title=Review%20of%20Vancomycin-Induced%20Renal%20Toxicity:%20An%20Update&amp;author=O.%20Bamgbola&amp;volume=7&amp;issue=3&amp;publication_year=2016&amp;pages=136-147&amp;pmid=27293542&amp;doi=10.1177/2042018816638223&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref96">
<cite>Tobias P. E., Varughese C. A., Hanson A. P., Gurnani P. K.. A Case of Linezolid
Induced Toxicity. J. Pharm. Pract. 2020;33(2):222–225. doi: 10.1177/0897190018782787.</cite> [<a href="https://doi.org/10.1177/0897190018782787" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29911459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Pharm.%20Pract.&amp;title=A%20Case%20of%20Linezolid%20Induced%20Toxicity&amp;author=P.%20E.%20Tobias&amp;author=C.%20A.%20Varughese&amp;author=A.%20P.%20Hanson&amp;author=P.%20K.%20Gurnani&amp;volume=33&amp;issue=2&amp;publication_year=2020&amp;pages=222-225&amp;pmid=29911459&amp;doi=10.1177/0897190018782787&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref97">
<cite>Vinh D. C., Rubinstein E.. Linezolid:
A Review of Safety and Tolerability. J. Infect. 2009;59(Suppl. 1):S59–S74. doi: 10.1016/S0163-4453(09)60009-8.</cite> [<a href="https://doi.org/10.1016/S0163-4453(09)60009-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19766891/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Infect.&amp;title=Linezolid:%20A%20Review%20of%20Safety%20and%20Tolerability&amp;author=D.%20C.%20Vinh&amp;author=E.%20Rubinstein&amp;volume=59&amp;issue=Suppl.%201&amp;publication_year=2009&amp;pages=S59-S74&amp;pmid=19766891&amp;doi=10.1016/S0163-4453(09)60009-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref98">
<cite>Balbi H. J.. Chloramphenicol:
A Review. Pediatr. Rev. 2004;25(8):284–288. doi: 10.1542/pir.25-8-284.</cite> [<a href="https://doi.org/10.1542/pir.25-8-284" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15286274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr.%20Rev.&amp;title=Chloramphenicol:%20A%20Review&amp;author=H.%20J.%20Balbi&amp;volume=25&amp;issue=8&amp;publication_year=2004&amp;pages=284-288&amp;pmid=15286274&amp;doi=10.1542/pir.25-8-284&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref99"><cite>

Grubb, A.
; 
Jasir, A.
; 
Schalen, C.
; 
Kasprzykowski, F.
; 
Kasprzykowska, R.
. 
Antimicrobial
Compounds. US7691805B2, 2010.</cite></li>
<li id="ref100">
<cite>Di Grazia A., Cappiello F., Imanishi A., Mastrofrancesco A., Picardo M., Paus R., Mangoni M. L.. The Frog Skin-Derived
Antimicrobial Peptide Esculentin-1a(1–21)­NH2 Promotes the Migration
of Human HaCaT Keratinocytes in an EGF Receptor-Dependent Manner:
A Novel Promoter of Human Skin Wound Healing? PLoS One. 2015;10(6):e0128663.  doi: 10.1371/journal.pone.0128663.</cite> [<a href="https://doi.org/10.1371/journal.pone.0128663" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4466536/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26068861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=The%20Frog%20Skin-Derived%20Antimicrobial%20Peptide%20Esculentin-1a(1%E2%80%9321)%C2%ADNH2%20Promotes%20the%20Migration%20of%20Human%20HaCaT%20Keratinocytes%20in%20an%20EGF%20Receptor-Dependent%20Manner:%20A%20Novel%20Promoter%20of%20Human%20Skin%20Wound%20Healing?&amp;author=A.%20Di%20Grazia&amp;author=F.%20Cappiello&amp;author=A.%20Imanishi&amp;author=A.%20Mastrofrancesco&amp;author=M.%20Picardo&amp;volume=10&amp;issue=6&amp;publication_year=2015&amp;pages=e0128663&amp;pmid=26068861&amp;doi=10.1371/journal.pone.0128663&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adc93_" lang="en" class="data-citations"><h3 class="pmc_sec_title">Data Citations</h3>
<ol class="ref-list font-sm"><li id="_dbx_ref_cit30">
<cite>

Kassambara, A.
; 
Mundt, F.
. 
factoextra: Extract
and Visualize the Results of Multivariate Data Analyses
CRAN: Contributed Packages, 10.32614/CRAN.package.factoextra.</cite> [<a href="https://doi.org/10.32614/CRAN.package.factoextra" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li></ol></section><section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12332667/bin/ao5c03775_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">ao5c03775_si_001.pdf</a><sup> (894.7KB, pdf) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Omega are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsomega.5c03775"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/ao5c03775.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (11.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12332667/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12332667/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12332667%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12332667/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12332667/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12332667/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40787370/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12332667/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40787370/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12332667/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12332667/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="ZpvomJzPOc7es4y0iFb5RTZ8RFQPVRPMivJgjciBJo3MTDtHpxEzsLzeuFgVunqB">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
