
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-a68b4900.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-0a3f24ce.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Epidrug screening identifies type I PRMT inhibitors as modulators of lysosomal exocytosis and drug sensitivity in cancers - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="FF20810489B72B131A810400350A840B.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="cddis">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334744/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Cell Death &amp; Disease">
<meta name="citation_title" content="Epidrug screening identifies type I PRMT inhibitors as modulators of lysosomal exocytosis and drug sensitivity in cancers">
<meta name="citation_author" content="Baris Sergi">
<meta name="citation_author_institution" content="Graduate School of Health Sciences, Koc University, Istanbul, Türkiye">
<meta name="citation_author" content="Neslihan Yuksel-Catal">
<meta name="citation_author_institution" content="Graduate School of Health Sciences, Koc University, Istanbul, Türkiye">
<meta name="citation_author" content="Selahattin Can Ozcan">
<meta name="citation_author_institution" content="Research Center for Translational Medicine (KUTTAM), Koc University, Istanbul, Türkiye">
<meta name="citation_author" content="Hamzah Syed">
<meta name="citation_author_institution" content="Research Center for Translational Medicine (KUTTAM), Koc University, Istanbul, Türkiye">
<meta name="citation_author" content="Umamaheswar Duvvuri">
<meta name="citation_author_institution" content="NYU Grossman School of Medicine, NYU Langone Health, New York, NY USA">
<meta name="citation_author" content="Kirill Kiselyov">
<meta name="citation_author_institution" content="Department of Biological Sciences, University of Pittsburgh, Pittsburgh, PA USA">
<meta name="citation_author" content="Ceyda Acilan">
<meta name="citation_author_institution" content="Research Center for Translational Medicine (KUTTAM), Koc University, Istanbul, Türkiye">
<meta name="citation_author_institution" content="School of Medicine, Koc University, Istanbul, Türkiye">
<meta name="citation_publication_date" content="2025 Aug 8">
<meta name="citation_volume" content="16">
<meta name="citation_issue" content="1">
<meta name="citation_firstpage" content="600">
<meta name="citation_doi" content="10.1038/s41419-025-07900-w">
<meta name="citation_pmid" content="40781072">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334744/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334744/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334744/pdf/41419_2025_Article_7900.pdf">
<meta name="description" content="Epigenetic changes drive gene expression alterations, contributing to oncogenesis and drug resistance. Lysosomes play a key role in cell signaling and sequestering toxins, including chemotherapeutic agents, which are then expelled through lysosomal ...">
<meta name="og:title" content="Epidrug screening identifies type I PRMT inhibitors as modulators of lysosomal exocytosis and drug sensitivity in cancers">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Epigenetic changes drive gene expression alterations, contributing to oncogenesis and drug resistance. Lysosomes play a key role in cell signaling and sequestering toxins, including chemotherapeutic agents, which are then expelled through lysosomal ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334744/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-testid="header" data-header >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            
                <a class="ncbi-header__logo-container" href="https://www.ncbi.nlm.nih.gov/">
                    <img alt="
                                  NCBI home page
                              "
                         class="ncbi-header__logo-image"
                         src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg" />
                </a>
            

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                


    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true"    >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                


    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true"    data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-testid="searchPanel"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      aria-describedby="search-field-desktop-navigation-help-text"
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only"
                           data-testid="label"
                           for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           data-testid="textInput"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    data-testid="button"
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  role="search">
                <label class="usa-sr-only" for="search-field">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="search" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
    

    Dashboard

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
    

    Publications

    
</a>

                        </li>
                    
                        <li class="usa-nav__primary-item">
                            








<a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
    

    Account settings

    
</a>

                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        <a class="usa-link" href="https://www.ncbi.nlm.nih.gov/pmc/advanced/" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12334744">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41419-025-07900-w"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41419_2025_Article_7900.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12334744%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12334744/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12334744/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334744/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-cddis.gif" alt="Cell Death &amp; Disease logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Cell Death &amp; Disease" title="Link to Cell Death &amp; Disease" shape="default" href="http://www.nature.com/cddis/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Cell Death Dis</button></div>. 2025 Aug 8;16(1):600. doi: <a href="https://doi.org/10.1038/s41419-025-07900-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41419-025-07900-w</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Cell%20Death%20Dis%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cell%20Death%20Dis%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Cell%20Death%20Dis%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Cell%20Death%20Dis%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Epidrug screening identifies type I PRMT inhibitors as modulators of lysosomal exocytosis and drug sensitivity in cancers</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sergi%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Baris Sergi</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Baris Sergi</span></h3>
<div class="p">
<sup>1</sup>Graduate School of Health Sciences, Koc University, Istanbul, Türkiye </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sergi%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Baris Sergi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yuksel-Catal%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Neslihan Yuksel-Catal</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Neslihan Yuksel-Catal</span></h3>
<div class="p">
<sup>1</sup>Graduate School of Health Sciences, Koc University, Istanbul, Türkiye </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yuksel-Catal%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Neslihan Yuksel-Catal</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ozcan%20SC%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Selahattin Can Ozcan</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Selahattin Can Ozcan</span></h3>
<div class="p">
<sup>2</sup>Research Center for Translational Medicine (KUTTAM), Koc University, Istanbul, Türkiye </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ozcan%20SC%22%5BAuthor%5D" class="usa-link"><span class="name western">Selahattin Can Ozcan</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Syed%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Hamzah Syed</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Hamzah Syed</span></h3>
<div class="p">
<sup>2</sup>Research Center for Translational Medicine (KUTTAM), Koc University, Istanbul, Türkiye </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Syed%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hamzah Syed</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Duvvuri%20U%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Umamaheswar Duvvuri</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Umamaheswar Duvvuri</span></h3>
<div class="p">
<sup>3</sup>NYU Grossman School of Medicine, NYU Langone Health, New York, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Duvvuri%20U%22%5BAuthor%5D" class="usa-link"><span class="name western">Umamaheswar Duvvuri</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kiselyov%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Kirill Kiselyov</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Kirill Kiselyov</span></h3>
<div class="p">
<sup>4</sup>Department of Biological Sciences, University of Pittsburgh, Pittsburgh, PA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kiselyov%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kirill Kiselyov</span></a>
</div>
</div>
<sup>4,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Acilan%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Ceyda Acilan</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Ceyda Acilan</span></h3>
<div class="p">
<sup>2</sup>Research Center for Translational Medicine (KUTTAM), Koc University, Istanbul, Türkiye </div>
<div class="p">
<sup>5</sup>School of Medicine, Koc University, Istanbul, Türkiye </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Acilan%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Ceyda Acilan</span></a>
</div>
</div>
<sup>2,</sup><sup>5,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Graduate School of Health Sciences, Koc University, Istanbul, Türkiye </div>
<div id="Aff2">
<sup>2</sup>Research Center for Translational Medicine (KUTTAM), Koc University, Istanbul, Türkiye </div>
<div id="Aff3">
<sup>3</sup>NYU Grossman School of Medicine, NYU Langone Health, New York, NY USA </div>
<div id="Aff4">
<sup>4</sup>Department of Biological Sciences, University of Pittsburgh, Pittsburgh, PA USA </div>
<div id="Aff5">
<sup>5</sup>School of Medicine, Koc University, Istanbul, Türkiye </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Aug 31; Revised 2025 May 14; Accepted 2025 Jul 17; Collection date 2025 Dec.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12334744  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40781072/" class="usa-link">40781072</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Epigenetic changes drive gene expression alterations, contributing to oncogenesis and drug resistance. Lysosomes play a key role in cell signaling and sequestering toxins, including chemotherapeutic agents, which are then expelled through lysosomal exocytosis—a process linked to drug resistance. However, the epigenetic regulation of lysosomal exocytosis is poorly understood. We hypothesize that epigenetic modifier drugs (epidrugs) inhibiting this exocytosis could serve as potential cancer therapeutics. To explore this, we screened more than 150 epidrugs targeting various epigenetic proteins for their combined cytotoxic effects with cisplatin, their impact on lysosomal exocytosis, and lysosomal biogenesis. Two type I PRMT inhibitors, MS023 and GSK3368715, showed synergy with cisplatin, reduced cell viability, and inhibited lysosomal exocytosis without altering lysosomal biogenesis gene expression. RNA-seq analysis revealed differentially expressed genes involved in vesicular trafficking and lysosome dynamics, suggesting novel regulatory mechanisms. These inhibitors also synergized with other lysosome-sequestered drugs, indicating a broader application in overcoming drug resistance. Analysis of patient data further linked lower type I PRMT levels to better responses, highlighting their potential as combination therapy candidates to enhance chemotherapy efficacy and improve cancer survival rates.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Targeted therapies, Cancer epigenetics</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Tumor resistance to chemotherapy is a growing concern, reducing the effectiveness of cytotoxic drugs across various cancer types and treatments. This resistance, whether innate or acquired, presents significant challenges in drug development and cancer therapy, highlighting the need for innovative strategies [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. Epigenetic regulation drives gene expression changes that promote oncogenic transformation or drug resistance [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. Small molecule inhibitors of these epigenetic factors (epidrugs) can induce growth arrest, cell differentiation, or apoptosis [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. To this end, epidrugs offer a promising solution by altering the epigenetic profile of cancer cells, making them more responsive to conventional therapies [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Several epigenetic inhibitors have been FDA-approved for treating various cancers, supported by evidence of their efficacy in adult cardiovascular, neurodegenerative, inflammatory, and specific pediatric lysosomal storage diseases [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>–<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>].</p>
<p id="Par3">One resistance mechanism in cancer cells is the sequestration of chemotherapeutic drugs within lysosomes [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. Lysosomes, with their high acidity, degrade and absorb nutrients and damaged cellular components and sequester toxins, along with toxic metals [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>–<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. This lysosomal sequestration process is documented for many cytotoxic drugs, including platinum compounds like cisplatin, carboplatin, and oxaliplatin [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. Especially for organic weak base drugs since the acidity of the lysosomal lumen attracts these drugs naturally [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>, <a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. Following sequestration, lysosomal exocytosis expels drugs, where lysosomal membranes fuse with the plasma membrane to release contents into the extracellular space, reducing intracellular chemotherapeutic agent concentrations and diminishing their cytotoxic effects [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>, <a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. This process is regulated by molecular motors and cytoskeletal elements, involving complex molecular machinery such as SNAREs, trafficking proteins, cytoskeleton, molecular motors, adaptors, and calcium channels, which remove chemotherapeutic agents and modulate the tumor microenvironment, promoting tumor growth and progression [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. Lysosomal enzyme release during exocytosis degrades extracellular matrix components, promoting cancer cell invasion and metastasis [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>–<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>].</p>
<p id="Par4">Previous research revealed that blocking lysosomal exocytosis increases the toxicity and intracellular accumulation of transition metals and platinum compounds [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>–<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. Thus, manipulating lysosomal flux has the potential to enhance cancer treatments [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>, <a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>, <a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. Understanding the epigenetic regulators of lysosomal exocytosis could develop interventions that potentiate existing chemotherapeutic agents. Research on epigenetic regulation of lysosomal exocytosis is limited. An epigenetic rheostat, including HDAC2 and TFEB, regulates lysosomal and autophagic pathways [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. HDAC10 depletion sensitizes cells to doxorubicin by inhibiting drug efflux via lysosomal exocytosis [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. Reduced expression of Sirtuin 1 (SIRT1) inhibits lysosomal function, promoting breast cancer aggressiveness and survival [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. These studies highlight the relationship between epigenetic modulation and lysosomal dynamics, suggesting targeted epigenetic interventions could disrupt lysosome-mediated drug resistance mechanisms. Our hypothesis focuses on reversing drug resistance by enhancing the cytotoxic activity of chemotherapeutic agents through inhibiting lysosomal exocytosis with epigenetic regulators.</p>
<p id="Par5">This study reports a novel mechanism of epigenetic regulation impacting lysosomal exocytosis and its implications for cancer therapeutics. We focused on epidrugs due to the increasingly recognized role of epigenetics in cancer biology and the emerging understanding of lysosomal exocytosis in cancer progression and therapy resistance. Using an epigenetic modifier drug library, we screened for drugs that significantly inhibit lysosomal exocytosis, revealing a poorly understood intersection of these two fundamental biological processes.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Cell culture</h3>
<p id="Par6">Du145 (ATCC, HTB-81) and H1299 (ATCC, CRL-5803) cells were maintained in RPMI-1640 (Gibco,11875093), supplemented with 10% FBS (Biowest, S1810) and 1% Pen/Strep (Biowest, L0022) at 37 °C and 5% CO<sub>2</sub> humidified incubator. Panc1 (ATCC, CRL-1469) and HEK293T (ATCC, CRL-3216) cells were maintained in DMEM (Gibco, 11965118), supplemented with 10% FBS and 1% Pen/Strep at 37 °C and 5% CO<sub>2</sub> humidified incubator. The rest of the cells were maintained at the ATCC’s recommended medium and culturing conditions. Cells were harvested using 0.05% Trypsin-EDTA (Gibco, 25300-054), passaged, or medium refreshed every 3-4 days, depending on the confluency.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Epigenetic modifier drug (Epidrug) library screening</h3>
<p id="Par7">The epidrug library containing 175 epigenetic modifier inhibitors was applied (Cayman, 11076) on Du145, Panc1, and H1299 cells. For epidrug and cisplatin combination cell viability screening, 1 × 10<sup>3</sup> cells were seeded on 96-well plates, and the next day, cells were pre-treated (primed) with epidrugs at a final concentration of 5 µM for 72 h. Following priming, cells were subjected to combinatorial treatment with epidrugs and cisplatin at an indicated concentration for a further 72 h (combination). After treatment incubations were done, screenings proceeded to Sulforhodamine B (SRB) cell viability assay, as indicated in the materials and methods section.</p>
<p id="Par8">For epidrug library screening with β-Hex lysosomal exocytosis or Lysotracker Green (LTG) staining assays, Du145 cells were seeded on 12-well plates (2 × 10<sup>5</sup> cell/well) and 96 well plates (4 × 10<sup>3</sup> cell/well), respectively. The next day, cells were treated with epidrugs (at a constant final concentration of 5 μM) for 72 h. Following epidrug treatment, β-Hex lysosomal exocytosis or Lysotracker Green (LTG) staining assays were performed as indicated in the materials and methods section.</p></section><section id="Sec5"><h3 class="pmc_sec_title">β-Hex lysosomal exocytosis assay</h3>
<p id="Par9">β-Hex activity was assayed with regular buffer (10 mM HEPES pH 7.4, 150 mM NaCl, 5 mM KCl, 1 mM CaCl<sub>2</sub>, 1 mM MgCl<sub>2</sub>, 2 g/L glucose). 2 × 10<sup>5</sup> cells were seeded on 12-well plates. The next day, cells were washed once with regular buffer, and cells were incubated with a regular buffer with (stimulated) or without (basal) CuCl<sub>2</sub> (Sigma, 222011) for 1 h. Buffer on top of cells was collected and further incubated with 3 mM 4-nitrophenyl-N-acetyl-β-D-glucosaminide substrate (Sigma, N9376) for 1 h at 37 °C. The addition of borate buffer stopped reactions, and the absorbance was measured at 405 nm. To determine the total cellular content of β-Hex, cells were lysed with 1% Triton X-100 in PBS, and the cell extract was subjected to the same enzyme activity reaction. Enzyme activity was determined via colorimetric reading as the amount of 4-nitrophenol produced at the end of the reaction, and the activity directly gave the lysosomal exocytosis rate of cells. Absorbance was calibrated with standards made up of 4-nitrophenol (Sigma, N7660) in 0.1 M citrate buffer. To determine the total % of β-hex secreted, the following formula was used: β-Hex secreted, % total = [(Abs<sub>405</sub> of regular buffer sample)/(Abs<sub>405</sub> of regular buffer sample + Abs<sub>405</sub> of lysate sample)].</p></section><section id="Sec6"><h3 class="pmc_sec_title">LysoTracker Green (LTG) staining assay</h3>
<p id="Par10">To assess changes in lysosomal content following epidrug treatment, LysoTracker Green (LTG) staining was performed. Du145, Panc1, and H1299 cells were seeded in 96-well plates at a density of 4 × 10³ cells/well. The following day, cells were treated with the indicated drugs. At the experimental endpoint, cells were washed once with PBS (Biowest, L0615), and culture medium containing 100 nM LysoTracker Green (Thermo, L7526) was added. Cells were incubated at 37 °C for 1 h in the presence of the dye. After incubation, cells were washed five times with PBS to remove excess dye. Then, fluorescence was measured using a Synergy H1 Hybrid reader (BioTek). Subsequently, cells were lysed with 25 μl of 0.2% SDS, and total protein content was quantified using the BCA Protein Assay Kit (Thermo, 23225). The measured fluorescence values were blank-subtracted, and the resulting relative fluorescence units (RFU) were normalized to the total protein content (μg) to obtain RFU/μg protein.</p>
<p id="Par11">For flow cytometry analysis, cells were stained with 100 nM LysoTracker Green as described above. Following the 1-h incubation, cells were washed twice with PBS, then detached using 0.25% Trypsin-EDTA and resuspended in PBS containing 2% FBS to maintain cell viability. Samples were analyzed using a Beckman Coulter CytoFLEX flow cytometer, with a minimum of 10,000 events recorded per sample. Median fluorescence intensity (MFI) values were calculated for each sample to assess changes in lysosomal content and normalization to control samples were applied. Population shifts toward increased fluorescence following epidrug treatments were visualized using FlowJo software (version 10.10.0).</p></section><section id="Sec7"><h3 class="pmc_sec_title">Sulforhodamine B (SRB) cell viability assay</h3>
<p id="Par12">Prior to drug exposure, cells were seeded in 96-well plates at an indicated density. After treatment incubations, the cells were fixed with a final concentration of 10% (w/v) TCA (Sigma, T6399) for 1 h at 4 °C, washed with deionized water, and allowed to air dry at room temperature. 50 μl of 0.4% (w/v) SRB (Santa Cruz, sc-253615) dye was used to stain the cells for 30 min, followed by washing with 1% acetic acid. The SRB dye was dissolved using 150 μl of a 10 mM Tris base solution (Sigma, T1503), and absorbances were measured at 564 nm using a microplate reader (Synergy H1 Hybrid reader, BioTek). Colorimetric measurements were read at 564 nm wavelength. Viabilities were calculated as follows; % Viability = [100 x (Sample Abs-blank)/(Non-treated control Abs-blank)].</p></section><section id="Sec8"><h3 class="pmc_sec_title">Crystal violet colony formation assay</h3>
<p id="Par13">0.5–2 × 10<sup>3</sup> cells were seeded on 12-well plates overnight, and then cells were treated, with the medium being refreshed every 72 h after treatments were done. Colony formation was expected to occur within 1–3 weeks. After this period, the medium was aspirated, and the wells were washed with PBS (Biowest, L0616). The cells were then fixed with cold methanol for 15–20 min at −20 °C and then washed twice with PBS. Subsequently, the cells were stained with 0.05% (w/v) crystal violet for 30–60 min, washed with dH2O, and dried, and the stained colonies were visualized. The percentage of formed colonies was analyzed using ImageJ software.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Calculation of Combination Index (CI)</h3>
<p id="Par14">The “ComBenefit” software was used to calculate the synergy index values for two-drug combinations. SRB cell viability assay was used to obtain the dose-response curves for each drug treatment alone and in combination, and the ComBenefit program calculated the synergistic interactions and CI values by the BLISS scoring model. Effect is calculated by [Effect (<em>a</em> + <em>b</em>) = <em>E</em> (<em>a</em>) + <em>E</em> (<em>b</em>) – <em>E</em> (<em>a</em>) <em>E</em> (<em>b</em>)]. The results were interpreted according to the program guidelines. According to the ComBenefit synergy scoring, a negative score indicates antagonism, whereas a positive score indicates synergy. Additionally, increasing absolute scores further potentiates either synergy or antagonism at combined doses of two drugs, stated as an increase of color intensity in surface plots showing cell viability and synergy status.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Western blotting</h3>
<p id="Par15">Cells were lysed in RIPA buffer (EcoTech, RIPA-100) containing PMSF (Merck, 10837091001) and phosSTOP (Merck, 4906845001) according to the manufacturer’s instructions. Protein concentrations were measured using the BCA Protein Assay Kit (Thermo Fisher, 2322). Proteins were denatured at 95 °C for 10 min in Laemmli buffer (Bio-Rad, 1610747) with DTT (Sigma, D0632) and separated on SDS-PAGE using Mini-PROTEAN TGX Stain-Free Gels (Bio-Rad, 4568086). Following transfer to PVDF (Bio-Rad, 1620177), membranes were blocked in 5% BSA (Sigma, A3733) and incubated overnight at 4 °C with primary antibodies: anti-aDMA (1:5000) (CST, 13522), anti-sDMA (1:2500) (CST, 13222), anti-PRMT1 (1:1000) (Abclonal, A1055), anti-PRMT6 (1:1000) (Abclonal, A5085), anti- FLAG (1:2000) (Thermo, #F3165), and anti-GAPDH (1:5000) (Abcam, AB9485). Membranes were washed with TBS-T, followed by incubation with secondary antibody (Abcam, ab205718) for 1.5 h at room temperature. Proteins were detected using Immobilon Forte western HRP substrate (EMD Millipore, WBLUF0500) and visualized on the Licor Odyssey FC Imaging System.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Real-time quantitative PCR (RT-qPCR)</h3>
<p id="Par16">Total RNA was isolated using the NucleoSpin RNA II kit (Zymo, R1051) following the manufacturer’s instructions. One microgram of total RNA was reverse transcribed using M-MLV Reverse Transcriptase (Invitrogen, 28025-013) cDNA synthesis kit following the manufacturer’s protocol. RT-qPCR was performed using a commercially available LightCycler 480 SYBR Green I master mix (QIAGEN, 1076717) and the designed primers. The reaction mixture included the master mix, cDNA template, and target gene-specific primers (Supplementary Table <a href="#MOESM1" class="usa-link">1</a>) in a final volume of 20 μl. Real-time PCR amplification was performed on LightCycler 480 Instrument II (Roche) using the following cycling conditions: initial denaturation at 95 °C for 10 min, followed by 40 cycles of denaturation at 95 °C for 15 s, annealing and extension at 72 °C for 1 min. The target gene expressions were normalized to GAPDH housekeeping gene expression, and fold changes were calculated with the 2<sup>(−ΔΔCt)</sup> method. To confirm amplification specificity, the PCR products were subjected to a melting curve analysis and subsequent agarose gel electrophoresis if necessary. Statistical analysis was performed using Student’s t-test to compare the expression levels between different groups or conditions with at least three biological replicates.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Cloning PRMT1 and PRMT6 into pLJC2 lentiviral overexpression vector</h3>
<p id="Par17">To generate FLAG-tagged PRMT1 and PRMT6 overexpression vectors for subsequent ChIP-qPCR experiments using a FLAG antibody, we amplified the coding regions of PRMT1 and PRMT6 from commercially available overexpression vectors (DNASU, #HsCD00950899 for PRMT1 and #HsCD00962806 for PRMT6) using gene-specific primers designed for subcloning (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). The amplification was performed with the Phusion DNA Polymerase kit (NEB, #E0553L), following the manufacturer’s instructions. The resulting PCR products were digested with PacI (NEB, #R0547) and NotI (NEB, #R0189) to create compatible sticky ends. Concurrently, the pLJC2-GPT2-3XFLAG vector (Addgene, #163447) was digested with the same enzymes to remove the GPT2 insert and prepare the vector for ligation. Both the digested insert fragments and the vector backbone were separated on a 1% agarose gel, and the desired DNA bands were purified using the NucleoSpin Gel and PCR Clean-up Kit (Macherey-Nagel, #740609.50), as per the manufacturer’s protocol. The purified inserts were then ligated into the prepared pLJC2-3XFLAG vector using T4 DNA Ligase (Thermo Fisher Scientific, #EL0013), following the supplier’s recommended protocol. The ligation mixtures were transformed into <em>Escherichia coli</em> Stbl3 competent cells, and transformants were selected on LB agar plates containing the appropriate antibiotic. Positive colonies were screened, and plasmid DNA was isolated using a plasmid DNA purification kit (Macherey-Nagel, #740588.50). The confirmed plasmids were subsequently used to produce viral particles for transduction experiments.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Generation of PRMT1/6-FLAG overexpressing cells</h3>
<p id="Par18">Early-passage HEK293T cells were seeded at a density of 3.5 × 10⁶ cells per 100 mm dish. The following day, the culture medium was replaced with an antibiotic-free medium. For lentiviral production, a transfection mixture was prepared by combining 2.25 μg psPAX2 (Addgene, #12260), 250 ng VSVG (Addgene, #8454), and 2.5 μg of the plasmid of interest in 200 μl Opti-MEM (Thermo Fisher, #31985062). Separately, 200 μl Opti-MEM was mixed with 15 μl polyethyleneimine (PEI) (Sigma, #408727). The two solutions were combined and incubated at room temperature for 25 min to allow complex formation. The transfection mixture was then vortexed briefly and then added dropwise on top of HEK293T cells and incubated for 18–24 h, after which the medium was replaced with fresh complete medium. At 48 and 72 h post transfection, the conditioned medium containing lentiviral particles was collected and filtered through a 0.45 μm PVDF filter. To concentrate the viral particles, the filtered supernatant was mixed with PEG8000 (Sigma, #25322) and incubated at 4 °C for 24 h. The mixture was then centrifuged at 3000 × <em>g</em> for 30 min, and the resulting pellet was resuspended in PBS. The concentrated viral particles were aliquoted and stored at −80 °C until further use.</p>
<p id="Par19">For viral transduction, Du145, Panc1, and H1299 cells were seeded at a density of 5 × 10⁴ cells per well in a 6-well plate. Then, 30 μl of the concentrated viral suspension was added to each well in the presence of 8 μg/ml protamine sulfate. The next day, the medium was replaced with fresh complete medium, and the cells were allowed to grow for an additional 24 h. At 48 h post transduction, the selection process was initiated by replacing the medium with one containing 2 μg/ml puromycin. The medium was refreshed every 48 h, maintaining puromycin selection until complete death of non-transduced control cells.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Chromatin Immunoprecipitation qPCR (ChIP-qPCR)</h3>
<p id="Par20">PRMT1/6-FLAG overexpressing Du145, Panc1, and H1299 cells were pelleted as 3 × 10<sup>6</sup> cells per sample. Subsequently, crosslinking was carried out with 1% formaldehyde (Sigma Aldrich, #818708) at room temperature for 10 min. The reaction was stopped by adding glycine (Sigma Aldrich, #G7126) to a final concentration of 125 mM for 5 min. The cell suspension was then centrifuged at 1000 × <em>g</em> for 5 min at 4 °C. After centrifugation, the pellet was washed twice with cold PBS and resuspended in ChIP Lysis Buffer (50 mM HEPES, 1 mM EDTA, 140 mM NaCl, 1% Triton X-100, 0.1% sodium deoxycholate, 1% SDS, 1 mM PMSF, and a Protease Inhibitor Cocktail). The samples were kept on ice for 15 min. Chromatin was subsequently released by sonication using the Bioruptor Plus sonicator (Diagenode), and the resulting lysate was centrifuged at 10,000 × <em>g</em> for 10 min at 4 °C. The supernatant containing sheared chromatin fragments (100–500 bp) was collected. To avoid non-specific binding, the chromatin preparation was incubated with prewashed Dynabeads® (Thermo Fisher, Protein G magnetic beads, # 88848) for 30 min at 4 °C (pre-clearing step). Antibodies against anti-mouse FLAG (Thermo, #F3165) and anti-mouse IgG (Sigma Aldrich, # 10400 C) as a control were incubated with magnetic beads in PBST for 1 h at room temperature. Then, the magnetic bead-antibody complexes were combined with the pre-cleared chromatin and incubated overnight. The magnetic beads were then washed successively with ChIP Lysis Buffer, low-salt buffer (0.1% SDS, 1% Triton X-100, 2 mM EDTA, 20 mM Tris-HCl, 140 mM NaCl, pH 8.1), high-salt buffer (0.1% SDS, 1% Triton X-100, 2 mM EDTA, 20 mM Tris-HCl, 500 mM NaCl, pH 8.1), LiCl-containing buffer (0.25 M LiCl, 1% IGEPAL-CA 630, 1% sodium deoxycholate, 1 mM EDTA, 10 mM Tris-HCl, pH 8.1), and finally with Tris-EDTA solution in order, twice for 5 min in each round. DNA elution was carried out using an elution buffer containing 1% SDS (Bio-Rad, #1610418) and 0.1 M NaHCO<sub>3</sub> (Merck, #S6014). Reverse crosslinking was then performed at 37 °C with RNase A (0.3 mg/ml) (Thermo Fisher Scientific, #12091039) and 5 M NaCl (Merck, #S9888) for 30 min, followed by incubation with proteinase K (0.3 mg/ml) (Thermo Fisher Scientific, #EO0492) at 65 °C for 1 h. DNA was purified using the ChIP DNA Clean &amp; Concentrator kit (Zymo Research, #D5205). ChIP-qPCR primers were designed using UCSC Genome Browser (<a href="https://genome.ucsc.edu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://genome.ucsc.edu/</a>) and Ensembl Genome Browser (<a href="https://www.ensembl.org/index.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ensembl.org/index.html</a>), and their sequences are provided in Supplementary Table <a href="#MOESM1" class="usa-link">2</a>. Quantitative PCR (qPCR) was performed using the LightCycler 480 system instrument (Roche) to quantify the immunoprecipitated DNA, and the results of the ChIP-qPCR were analyzed using the “% input” method.</p></section><section id="Sec15"><h3 class="pmc_sec_title">RNA sequencing and analysis</h3>
<p id="Par21">Total RNA was isolated with the Qiagen RNeasy Mini Kit following the manufacturer’s instructions. BGI Group conducted poly-A capture, library preparation, and next-generation sequencing as a contract research service. Sequencing was done with BGIseq, which generated 20 million reads/sample and 150 bp paired-end reads. The sequencing data was stored securely and analyzed on the Koc University high-performance computing cluster (KUACC). The resulting sequencing data were analyzed with a standard RNA-seq pipeline. First, the quality of the reads was assessed using FastQC software. FastQC provided information on the quality score across reads and the per base sequence content and identified adapter contamination. If contamination was present or there were low-quality regions, then TrimGalore was used for trimming. The sequencing data were then mapped to the most recent human reference genome (NCBI GRCh38) with the software HiSAT2. All mitochondrial, mis-spliced genes and duplicates were identified and removed using RSeQC and Picard Tools. Performing this detailed quality control procedure ensured the success of our study in identifying differentially expressed genes. Once the sequencing reads were aligned, the total read counts for each gene were quantified using featureCounts, part of the Subreads package. The results were then analyzed using the R package DESeq2 to identify differentially expressed genes. The significant results from the gene expression analysis were annotated using the biomart database. The final step was performing Gene Set Enrichment Analysis to understand which pathways or gene networks the differentially expressed genes were implicated in. GEO accession number of our RNA-seq analysis is GSE277490.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par22">All data were analyzed in Prism 8 (GraphPad Software, Inc.), and statistical tests were applied as described in the figure legends. Representation of error bars, sample sizes, and <em>p</em> values with asterisks were also indicated for each experiment at the figure legends.</p></section></section><section id="Sec17"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec18"><h3 class="pmc_sec_title">Cell line panel-based analysis confirms the negative correlation between lysosomal exocytosis and lysosomal volume</h3>
<p id="Par23">Previously published data indicate that chemotherapeutic drugs are sequestered in lysosomes and expelled through lysosomal exocytosis, with the efficiency of this mechanism depending on lysosomal volume and exocytosis rate. Although epigenetic regulation of lysosomal flux is not well understood, the use of epigenetic drug libraries and precise assays presents an opportunity to explore this new translational axis.</p>
<p id="Par24">To select the optimal cell line for studying lysosomal exocytosis, it was necessary to identify a cell line that yielded satisfactory absorbance values within the linear range for screening changes in exocytosis via β-Hexosaminidase (β-Hex) assay. Additionally, the ideal cell line should exhibit a higher exocytosis rate upon stimulation with copper, which is a common method used to enhance exocytosis by releasing it from lysosomes (Fig. <a href="#Fig1" class="usa-link">1A</a>). After evaluating 19 cell lines, Du145 emerged as the most suitable choice due to its superior performance in the exocytosis assay, alongside Panc1, MIA-PaCa-2, H1299 and U373 (Fig. <a href="#Fig1" class="usa-link">1B</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Screening and correlation analysis of lysosomal exocytosis and lysosomal volume across multiple cancer cell lines.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334744_41419_2025_7900_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9c8a/12334744/775b3b9aca53/41419_2025_7900_Fig1_HTML.jpg" loading="lazy" id="d33e643" height="466" width="790" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic representation of basal and CuCl₂-stimulated lysosomal exocytosis assays (Created with BioRender.com). <strong>B</strong> Lysosomal exocytosis rates (% total of β-hexosaminidase secretion) under basal and CuCl₂-stimulated conditions were measured across a panel of cancer cell lines. Correlation analysis between total lysosomal volume (Lysotracker fluorescence intensity) and lysosomal exocytosis under (<strong>C</strong>) basal and (<strong>D</strong>) stimulated conditions. Each numbered data point represents an individual cell line as follows: <strong>1</strong>. MIA-PaCa-2 (PAAD), <strong>2</strong>. HeLa (CESC), <strong>3</strong>. U2OS (SARC), <strong>4</strong>. U373 (GBM), <strong>5</strong>. PC3 (PRAD), <strong>6</strong>. Du145 (PRAD), <strong>7</strong>. 22Rv1 (PRAD), <strong>8</strong>. LNCaP (PRAD), <strong>9</strong>. H1299 (LUAD), <strong>10</strong>. A549 (LUAD), <strong>11</strong>. HepG2 (LIHC), <strong>12</strong>. Huh7 (LIHC), <strong>13</strong>. MCF-7 (BRCA), <strong>14</strong>. MDA-MB-231 (BRCA), <strong>15</strong>. A2780 (OV), <strong>16</strong>. Kuramochi (HGSOC), <strong>17</strong>. OSCC40 (HNSC), <strong>18</strong>. Panc1 (PAAD), <strong>19</strong>. RPE-1 Cell lines without complete data sets were excluded from correlation plots. Pearson’s correlation (<em>r</em>) and <em>R</em>² values are indicated on the plots along with corresponding p-values. Shaded areas with dashed line indicate 95% confidence intervals.</p></figcaption></figure><p id="Par25">To better understand the relationship between lysosomal exocytosis and total lysosomal volume, we hypothesized that inhibiting lysosomal exocytosis would lead to an accumulation of lysosomal content, resulting in increased lysosomal volume. Correlation analysis revealed an inverse relationship between total lysosomal volume and exocytosis, with a modest but near-significant correlation at basal levels (<em>R</em>² = 0.39, <em>p</em> = 0.052) and a statistically significant correlation under stimulated conditions (<em>R</em>² = 0.52, <em>p</em> = 0.019) (Fig. <a href="#Fig1" class="usa-link">1</a>C, D). Given the biological variability in lysosomal regulation, these findings suggest a consistent association between lysosomal volume and exocytosis, providing a relevant framework for interpreting drug-induced effects.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Epigenetic drug library screening on Du145 cells identified epidrugs affecting the lysosomal exocytosis, lysosomal volume, and cell viability in combination with cisplatin</h3>
<p id="Par26">To systematically evaluate epidrugs that enhance the efficacy of cisplatin, with a particular focus on those that may operate through lysosomal sequestration and exocytosis, we conducted three separate screens using a drug library comprising 175 small molecules that target diverse epigenetic ‘writer’ (such as Histone Methyl-, Histone Acetyl-, and DNA Methyltransferases), ‘eraser’ (including Histone-Deacetylase and -Demethylases), and ‘reader’ (containing bromodomains) proteins. It is noteworthy that 33 of these compounds are currently undergoing clinical trials, and 13 have been granted FDA approval. Our approach involved measuring the basal and stimulated exocytosis rates of various cell lines.</p>
<p id="Par27">The four cell lines with robust activity were evaluated, and the Du145 cells were selected for further screening since their exocytic activity fell within the linear range in both the basal and stimulated states. To assess cytotoxicity in Du145 cells, a mild dose of cisplatin was utilized, ensuring that approximately 70% of the cells remained viable. Additionally, a consistent dose of epigenetic drugs (5 µM) was administered, as these drugs typically lack cytotoxicity and instead prime cells for subsequent therapies by reprogramming cancer cells to a more normal state. Given the large-scale nature of the epidrug library screening, a single-dose strategy was employed to ensure feasibility. While this approach may limit insights into dose-dependent effects, it provided a practical means to identify potential hits for further investigation. Cells were pre-treated with epigenetic drugs for 72 h and then co-treated with cisplatin for a further 72 h (Fig. <a href="#Fig2" class="usa-link">2A</a>). Figure <a href="#Fig2" class="usa-link">2B</a> indicates that the majority of tested epigenetic drugs did not significantly impact cell viability (green dots, representing each epidrug aligned on the <em>x</em>-axis), suggesting their potential use in sensitizing cancer cells to cisplatin without causing harmful effects on cell viability. A total number of 14 epigenetic drugs that showed decreased cell viability when combined with cisplatin (red dots) were further investigated in cell viability assays with various dose combinations.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. High-content screening identifies epidrugs that synergize with cisplatin and modulate lysosomal exocytosis and lysosomal volume in Du145 cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334744_41419_2025_7900_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9c8a/12334744/0bec10ebfb8c/41419_2025_7900_Fig2_HTML.jpg" loading="lazy" id="d33e700" height="1007" width="790" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic representation of the screening procedure used to identify epidrugs synergizing with cisplatin in a viability assay. Du145 cells were seeded on 96-well plates and pre-treated (primed) with epidrugs (5 µM) for 72 h, followed by combined treatment with epidrugs (5 µM) and cisplatin (5 µM) for an additional 72 h (created with BioRender.com). <strong>B</strong> Scatter plot representing cell viability results from the epidrug library screening, with each dot indicating an individual epidrug. Drugs were classified based on their ability to synergize with cisplatin (red) compared to epidrug-only treatments (green). Key compounds are indicated in blue and purple. <strong>C</strong> Schematic overview of the lysosomal exocytosis and lysosomal volume screening assay (Created with BioRender.com). <strong>D</strong> Screening results showing the effects of epidrugs (5 µM, 72 h) on lysosomal exocytosis activity under basal and stimulated conditions, quantified as extracellular and intracellular β-hexosaminidase activity and represented in log<sub>2</sub>-normalized values. <strong>E</strong> Lysosomal volume changes upon epidrug treatment (5 µM, 72 h), measured by Lysotracker staining. Total lysosomal volume was quantified at basal and stimulated conditions, and values are normalized to the untreated control.</p></figcaption></figure><p id="Par28">A comparable methodology was employed for the follow-up screens, in which cells were pre-treated with 5 µM of epidrugs for 72 h, after which their lysosomal exocytosis and volume were assessed via β-Hex and LysoTracker staining assays, respectively (Fig. <a href="#Fig2" class="usa-link">2C</a>). The motivation for examining lysosomal exocytosis and volume assays in both basal and stimulated conditions was rooted in the hypothesis that we would not diminish the basal level of exocytosis in the cells since the exocytosis activity plays a vital role in fundamental cellular processes. We posited that the inhibition of this lysosomal activity, crucial for numerous cellular processes and whose deficiency would result in lethal consequences, could not be accurately replicated in the experimental setup. As shown in Fig. <a href="#Fig2" class="usa-link">2D</a>, various drugs showed potent and specific effects on lysosomal flux components. Among these, four epidrugs, type I PRMT inhibitors GS3368715 and MS023, a lysine acetyltransferase inhibitor known as CPTH6, and a SIRT1/2 inhibitor called Cambinol, distinguished themselves for their inhibitory effect on lysosomal exocytosis. These four epidrugs also resulted in a considerable accumulation of lysosomal volume (Fig. <a href="#Fig2" class="usa-link">2E</a>), which is consistent with the buildup of lysosomes due to deficiencies in lysosomal exocytosis, as would be expected based on the inverse correlation between exocytosis and lysosome volume in our initial analysis.</p>
<p id="Par29">To assess whether the findings are cell line-specific or a broader phenomenon, the initial epidrug library screening in Du145 cells was repeated in Panc1 and H1299 cells, followed by cell viability assessment after priming and co-treatment with cisplatin. Consistent with Du145 results, the Type I PRMT inhibitors MS023 and GSK3368715 showed synergy with cisplatin (Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>), suggesting a generalizable effect across cell lines.</p>
<p id="Par30">Based on our findings from three separate screens, we have identified 14 potential epidrugs that may influence lysosomal exocytosis, lysosomal volume, and cisplatin efficacy (Fig. <a href="#Fig3" class="usa-link">3A</a>). Through pre-treatment of Du145 cells with these 14 epidrugs in combination with cisplatin at different doses, we discovered that only 2 of the 4 epidrugs that inhibited lysosomal exocytosis also demonstrated synergy with cisplatin (Fig. <a href="#Fig3" class="usa-link">3</a>B, C). The remaining 10 epidrugs, although they did not inhibit exocytosis, did synergize with cisplatin (Fig. <a href="#Fig3" class="usa-link">3D</a>). In detail, 6 of them are HDAC inhibitors, 2 of them are p300 inhibitors, and the remaining 2 are PRMT5 inhibitors (the synergy scoring for these drugs is presented in detail in Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>). Following these screens, we selected the type I PRMT inhibitors “MS023” and “GSK3368715” for further study as they both inhibit exocytosis and demonstrate synergy with cisplatin.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Effect of selected epidrugs from the library on cell viability in combination with cisplatin.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334744_41419_2025_7900_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9c8a/12334744/58a368f71688/41419_2025_7900_Fig3_HTML.jpg" loading="lazy" id="d33e756" height="676" width="790" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Venn diagram summarizing the results of the epidrug library screening performed on Du145 cells, identifying epidrugs that inhibit lysosomal exocytosis and/or show synergistic effects with cisplatin. <em>Surface plots represent combined drug response (epidrug + cisplatin) in viability assays:</em>
<strong>B</strong> two epidrugs (CPTH6, Cambinol) that decreased lysosomal exocytosis but did not synergize with cisplatin, <strong>C</strong> two epidrugs (MS023, GSK3368715) that both inhibited exocytosis and exhibited synergy with cisplatin, and <strong>D</strong> ten epidrugs that synergized with cisplatin without affecting lysosomal exocytosis. Color gradients indicate the intensity of synergy and antagonism, while cell viabilities are shown on the <em>y</em>-axis for combined doses of two drugs.</p></figcaption></figure></section><section id="Sec20"><h3 class="pmc_sec_title">Type I PRMT inhibitors MS023 and GSK3368715 alter their target cellular processes on Du145, Panc1, and H1299 cells</h3>
<p id="Par31">Protein arginine methyltransferases (PRMTs) are enzymes responsible for the methylation of arginine residues on specific target proteins, a key post-translational modification impacting numerous cellular functions [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Type I PRMTs, encompassing PRMT1, -3, -4, -6, and -8, predominantly generate asymmetric dimethylarginine (aDMA) marks on the target protein. This modification plays a significant role in regulating gene expression, signal transduction, RNA processing, and protein-protein interactions [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>, <a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. Through the methylation of histones and various nuclear proteins, Type I PRMTs influence chromatin architecture and accessibility, thereby contributing to the epigenetic regulation of gene transcription [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>, <a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Both epidrugs identified as hits in the drug library screens conducted in our study, MS023 and GSK3368715, are defined as drugs that inhibit Type I PRMTs at different inhibition concentrations [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>, <a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>] (Supplementary Table <a href="#MOESM1" class="usa-link">3</a>). To better understand the properties of these epidrugs, we examined their effects on lysosomal exocytosis, total lysosomal volume, arginine methylation of target proteins, and the expression of lysosomal genes. The epidrugs were tested on three cell lines, Du145, Panc1, and H1299, which were among the most suitable cell lines identified in previous experiments. The β-Hex lysosomal exocytosis assay conducted on three cell lines showed that the treatment with MS023 and GSK3368715 for 72 h inhibited lysosomal exocytosis in the stimulated state (Fig. <a href="#Fig4" class="usa-link">4A</a>). Additionally, the ratio of exocytosis at the stimulated and basal conditions (stimulated/basal), which is referred to as induced exocytosis, was reduced post-exposure to epidrugs, indicating that these hit epidrugs alter the way cells release the lysosomal content (Fig. <a href="#Fig4" class="usa-link">4B</a>). Moreover, brief exposure to Type I PRMT inhibitors did not affect lysosomal exocytosis rates, confirming that the reduction in exocytosis required transcriptomic priming of the cells rather than being a direct response to the inhibitor (Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. MS023 and GSK3368715 inhibit lysosomal exocytosis, alter lysosomal volume, and reduce arginine methylation in Du145, Panc1, and H1299 cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334744_41419_2025_7900_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9c8a/12334744/72a481b1818a/41419_2025_7900_Fig4_HTML.jpg" loading="lazy" id="d33e838" height="1000" width="648" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Cells were treated with type I PRMT inhibitors MS023 and GSK3368715 (5 μM, 72 h). <strong>A</strong> Lysosomal exocytosis was quantified by measuring β-hexosaminidase secretion at basal and stimulated conditions. <strong>B</strong> Fold induction of exocytosis upon stimulation was calculated (stimulated/basal). Total lysosomal volume after treatment was evaluated using Lysotracker Green staining and quantified by fluorescence intensity measurement (<strong>C</strong>) and flow cytometry analysis (<strong>D</strong>, <strong>E</strong>). Representative flow cytometry histograms are shown in (<strong>E</strong>). <strong>F</strong> Western blot analysis shows the impact of PRMT inhibitors on asymmetric dimethylarginine (aDMA) and symmetric dimethylarginine (sDMA) modifications, as well as the expression levels of PRMT1 and PRMT6 proteins. GAPDH was used as a loading control. Error bars represent the standard deviation from at least three biological replicates (*<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, Student’s <em>t</em> test; n.s. not significant).</p></figcaption></figure><p id="Par32">Lysotracker staining assays demonstrated that Type I PRMT inhibitors led to an increase in the total lysosomal volume in Du145 and Panc1 cells (Fig. <a href="#Fig4" class="usa-link">4C</a>). Complementing the fluorescence intensity measurements from Lysotracker staining, flow cytometry experiments showed a rightward shift in the cell population within the treatment groups, further confirming the increase in lysosomal volume (Fig. <a href="#Fig4" class="usa-link">4D, E</a>). Given that lysosomal alterations can sometimes be secondary to cytotoxic effects, we assessed the viability of cells treated with these inhibitors. Cell viability assays in all three cell lines revealed that both inhibitors had IC<sub>50</sub> values exceeding 40 μM at 72 h, far above the 5 μM concentration used in this study (Supplementary Figure <a href="#MOESM1" class="usa-link">4</a>), indicating that the observed lysosomal changes were unlikely a consequence of acute cytotoxicity.</p>
<p id="Par33">To assess the impact of the drugs on the stability of type I PRMTs, particularly PRMT1 and PRMT6 -the main targets of MS023 and GSK3368715- we blotted for these proteins. Our findings showed similar levels of protein expression, indicating that the inhibition does not affect protein expression but instead directly influences protein activity (Fig. <a href="#Fig4" class="usa-link">4F</a>). Since type I PRMTs mediate asymmetric dimethylation of arginine residues on target proteins, we also analyzed arginine methylation levels to evaluate target specificity and inhibition efficacy. As expected, treatment with both drugs resulted in decreased levels of asymmetric dimethyl arginine (aDMA) and increased levels of symmetric dimethyl arginine (sDMA) as a compensatory mechanism, confirming effective inhibition of the intended targets (Fig. <a href="#Fig4" class="usa-link">4F</a>).</p>
<p id="Par34">Lysosomal biogenesis encompasses not only changes in lysosomal volume but also the expression of lysosomal genes. To investigate this, we examined the expression levels of several lysosomal genes, including the master regulators, TFEB and MiTF. Our panel of lysosomal genes included various lysosomal enzymes, transmembrane proteins, ATP pumps, ABC transporters, and proteins involved in lysosomal exocytosis. After treating Du145, Panc1, and H1299 cells with MS023 and GSK3368715, we observed that the expression of most lysosomal genes in the panel did not change significantly (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>). A more detailed analysis revealed that treatment with these inhibitors affected specific genes involved in endolysosomal function, such as TRPML1 (a key regulator of lysosomal exocytosis and biogenesis encoded by MCOLN1 [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]), TMEM205 (a transmembrane protein associated with cisplatin resistance [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>, <a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]), and ABCA3 (a lipid transporter involved in lysosomal function [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>, <a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]). Interestingly, while TRPML1 is part of the CLEAR lysosomal biogenesis network, MS023 did not broadly impact other components of this network, including the structural protein LAMP1, the enzyme palmitoyl-protein thioesterase 1 (PPT1), or genes encoding various cathepsin lysosomal proteases (Supplementary Figure <a href="#MOESM1" class="usa-link">5</a>). We conclude that type I PRMT inhibitors selectively affect components of the trafficking machinery related to lysosomal exocytosis without broadly disrupting the entire lysosomal biogenesis network.</p>
<p id="Par35">We further investigated whether epidrugs exhibit synergistic effects with other chemotherapeutic agents, beyond cisplatin, that are known to undergo lysosomal sequestration and exocytosis, which limits their efficacy. Sunitinib, a receptor tyrosine kinase inhibitor, and Doxorubicin, a topoisomerase II inhibitor, are among the drugs that experience lysosomal sequestration [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>–<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. We combined the inhibitors MS023 and GSK3368715 with Sunitinib, Doxorubicin, and platinum-based drugs including cisplatin, carboplatin, and oxaliplatin on Du145, Panc1, and H1299 cell lines. The Combenefit analysis demonstrated that both epidrugs exhibited synergistic effects with these chemotherapeutic agents across various concentrations in all cell lines (Fig. <a href="#Fig5" class="usa-link">5</a>) (the synergy scoring for these drugs is presented in detail in Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a>). Notably, similar to the need for reduced exocytosis rates, the synergy between epidrugs and agents like cisplatin and sunitinib was only observed when cells were pre-treated with the epidrugs for 72 h; co-treatment without priming did not show a significant synergy (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Type I PRMT inhibitors synergize with platinum-based agents and chemotherapeutics known to undergo lysosomal sequestration.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334744_41419_2025_7900_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9c8a/12334744/6c175fefdac4/41419_2025_7900_Fig5_HTML.jpg" loading="lazy" id="d33e919" height="755" width="790" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Surface plots illustrate the cell viability (y-axis) and synergistic interactions between type I PRMT inhibitors (MS023 and GSK3368715) and various chemotherapeutics (cisplatin, carboplatin, oxaliplatin, sunitinib, and doxorubicin) in (<strong>A</strong>) Du145, <strong>B</strong> Panc1, and <strong>C</strong> H1299 cell lines. Cells were first primed with epidrugs (MS023 or GSK3368715) for 72 h, followed by combined treatment with the indicated drugs at specified concentrations for an additional 72 h. Cell viability was assessed, and synergy analysis was performed using the Bliss model via Combenefit software, where the deepest shade of blue indicates the highest level of synergy.</p></figcaption></figure><p id="Par36">The colony formation assay performed on Du145 and Panc1 cells further confirmed this synergy by showing a reduction in the cells’ colony-forming capacity after the combination treatment (Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a>). These results cumulatively suggest that the inhibition of lysosomal exocytosis by epidrugs enhances the cytotoxicity of the chemotherapeutic agents.</p></section><section id="Sec21"><h3 class="pmc_sec_title">RNA sequencing analysis on MS023-treated cells reveals differentially expressed genes and enriched gene sets</h3>
<p id="Par37">To explore the synergistic effects of type I PRMT inhibitors with chemotherapy agents and their influence on lysosome volume and exocytosis, we performed RNA-seq analysis on Du145 cells treated with MS023. Initially, the expression changes of the top up- and down-regulated genes within the DEGs were validated through qPCR experiments (Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>). Gene ontology analysis confirmed the previously known roles of PRMTs, such as their involvement in DNA repair, methylation, and epigenetic modifications (Fig. <a href="#Fig6" class="usa-link">6A</a>), supporting the robustness of our experimental approach. The analysis also revealed the effects of MS023 treatment on lysosomal flux, showing a negative enrichment in genes associated with lysosomal hydrolases, lipases, peptidases, and catalytic activity (Fig. <a href="#Fig6" class="usa-link">6A, B</a>). Moreover, GO Molecular Function analysis identified significant changes in cellular components, particularly in vesicular transport, coated vesicles, and vesicle membranes (Fig. <a href="#Fig6" class="usa-link">6C</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Gene Set Enrichment Analysis (GSEA) identifies enriched pathways upon MS023 treatment in Du145 cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334744_41419_2025_7900_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9c8a/12334744/7a952bcee409/41419_2025_7900_Fig6_HTML.jpg" loading="lazy" id="d33e958" height="549" width="790" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>RNA-seq data from MS023-treated Du145 cells were analyzed using GSEA to identify significantly enriched pathways. Positively (left panel) and negatively (right panel) enriched gene sets are shown, categorized by (<strong>A</strong>) Gene Ontology, Biological Process “c5.go.bp.v2023.1.Hs.symbols.gmt”, (<strong>B)</strong> Gene Ontology, Cellular Components “c5.go.cc.v2023.1.Hs.symbols.gmt”, and (<strong>C)</strong> Gene Ontology, Molecular Function “c5.go.mf.v2023.1.Hs.symbols.gmt” datasets in the MsigDB database.</p></figcaption></figure><p id="Par38">Notably, there was an enrichment of autophagosomes, endolysosomes, and vacuoles following MS023 treatment, aligning with its impact on lysosomal exocytosis and the observed increase in lysosomal content. The enrichment of coated vesicles, particularly in clathrin-coated vesicle formation (NES scores &gt;2), speculatively suggests increased endocytosis and transport of coated particles from the trans-Golgi network to the lysosomes, contributing to the observed increase in lysosomal content during epidrug treatment experiments.</p>
<p id="Par39">In addition to GSEA analyses, we examined existing gene sets in the Molecular Signature Database. Based on the intracellular effects of MS023 reported in the literature and its observed impact on lysosomal functions, we generated six main combined gene sets: Exocytosis-, Lysosome-, Sequestering-, Methyltransferase-, DNA Repair-, and RNA Processing-related combined gene sets. Supplementary Table <a href="#MOESM1" class="usa-link">4</a> provides a summary of the composition of each combined dataset. We then overlapped the DEGs from our RNA-seq analysis with these combined gene sets to determine which DEGs might be involved in these functions, potentially explaining the reduction in lysosomal exocytosis or the observed drug synergy. Indeed, several genes were identified, and the top up-regulated and down-regulated genes (|logFC| &gt; 1) are highlighted in the volcano plots (Fig. <a href="#Fig7" class="usa-link">7</a>).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7. RNA-seq analysis identifies differentially expressed genes within combined gene sets.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334744_41419_2025_7900_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9c8a/12334744/2292bc89f60e/41419_2025_7900_Fig7_HTML.jpg" loading="lazy" id="d33e1004" height="866" width="790" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> The volcano plot from the RNA-seq analysis shows the differentially expressed genes following MS023 treatment (<em>p</em>-val &lt; 0.05; |log(FC)| &gt; 1). Gene sets from the GSEA Molecular Signature Database were manually selected and categorized into six distinct “combined gene sets” based on their functional relevance (details given in Supplementary Table <a href="#MOESM1" class="usa-link">4</a>): (<strong>B</strong>) lysosomal exocytosis-related gene sets, (<strong>C</strong>) lysosome-related gene sets, (<strong>D</strong>) sequestration-related gene sets, (<strong>E</strong>) methyltransferase-related gene sets, (<strong>F</strong>) DNA repair-related gene sets, and (<strong>G</strong>) RNA processing-related gene sets. The differentially expressed genes within each of these combined gene sets are marked, and volcano plots were generated, with grey vertical lines indicating the |logFC| &gt; 1 cutoff and the blue horizontal line representing p-val &lt; 0.05.</p></figcaption></figure><p id="Par40">To investigate whether type I PRMTs contribute to the regulation of DEGs identified in Fig. <a href="#Fig7" class="usa-link">7</a>, we retrieved ChIP-seq data from publicly available datasets on the ChIP Atlas platform (Fig. <a href="#Fig8" class="usa-link">8A</a>). Given data availability, we focused on PRMT1, PRMT4, and PRMT6 (based on data availability only for those Type I PRMTs on ChIP-Atlas platform) occupancy at the promoter regions of genes exhibiting significant up- or downregulation in our RNA-seq data (|logFC| &gt; 1) (Fig. <a href="#Fig8" class="usa-link">8B, C</a>). This analysis provided insights into PRMT-mediated methyltransferase activity and its potential role in transcriptional regulation.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8. Integrative analysis of RNA-seq and ChIP-seq data reveals type I PRMT occupancy at target gene promoters, validated by ChIP-qPCR.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334744_41419_2025_7900_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9c8a/12334744/b83190e0bb3c/41419_2025_7900_Fig8_HTML.jpg" loading="lazy" id="d33e1041" height="683" width="790" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic overview illustrating how promoter regions (–5 kb to +100 bp relative to transcription start sites) of selected target genes from combined gene sets were analyzed for occupancy by PRMT1, PRMT4/CARM1, and PRMT6 using publicly available ChIP-seq datasets from the ChIP-Atlas database (Created with BioRender.com). <strong>B</strong>, <strong>C</strong> Differentially expressed genes identified by RNA-seq (upregulated in red and downregulated in blue) following MS023 treatment were categorized based on their associated combined gene sets: 1. Exocytosis, 2. Lysosome, 3. Sequestering, 4. Methylation, 5. DNA Repair, and 6. RNA Processing. Bar plots indicate logFC expression values, and bubble plots represent PRMT occupancy at promoter regions. Bubble size corresponds to average ChIP-seq enrichment scores, and bubble color intensity reflects the number of ChIP-seq datasets confirming the occupancy peaks. <strong>D</strong> ChIP-qPCR validation confirming PRMT1 and PRMT6 occupancy at selected gene promoters. Du145 cells expressing PRMT6-FLAG or PRMT1-FLAG were used to measure promoter occupancy. Error bars indicate standard error of the mean (SEM) from three independent biological replicates done in duplicate (*<em>p</em> &lt; 0.05, Mann–Whitney <em>U</em> test).</p></figcaption></figure><p id="Par41">To further validate these findings, we conducted ChIP-qPCR experiments in Du145 cells to assess PRMT1 and PRMT6 occupancy at the promoter regions of selected DEGs. PRMT1 and PRMT6 were cloned into FLAG-tagged lentiviral vectors to generate stable overexpression models, which were confirmed via Western blot (Supplementary Fig. <a href="#MOESM1" class="usa-link">10</a>). ChIP-qPCR using gene-specific primers (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>) demonstrated PRMT binding at the promoter regions of most selected genes (Fig. <a href="#Fig8" class="usa-link">8D</a>), in agreement with the public dataset analysis. Similar ChIP-qPCR experiments on Panc1 and H1299 cells confirmed PRMT occupancy at the promoter regions of key DEGs in these cell lines as well (Supplementary Fig. <a href="#MOESM1" class="usa-link">11</a>). These observations strongly support the hypothesis that type I PRMTs directly regulate the expression of specific genes involved in lysosomal functions and related pathways, highlighting the potential of type I PRMT inhibitors to modulate epigenetic marks and alter gene expression, ultimately impacting cellular processes such as lysosomal exocytosis.</p>
<p id="Par42">To further validate our findings and assess the clinical relevance of PRMT-1, -4, -6, and -8 in chemotherapy resistance, we analyzed patient data from the Cancer Treatment Response Gene Signature Database (CTR-DB), a publicly available resource. This analysis aimed to determine whether the expression levels of type I PRMTs could serve as predictive biomarkers for chemotherapy response, particularly for treatments known to involve lysosomal sequestration. We specifically compared the expression levels of these PRMTs between patients who responded to therapy and those who did not, to establish a connection between our in vitro results and actual patient outcomes.</p>
<p id="Par43">The patient cohort consisted of individuals diagnosed with a specific type of cancer, divided into two groups based on their response to standard chemotherapy. Responders were defined as patients who exhibited a significant reduction in tumor size or disease stabilization, while non-responders showed no improvement or disease progression despite treatment during treatment [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. Using this classification, we focused on the patient groups who take specific chemotherapy drugs which attributed to the lysosomal sequestration and exocytosis events, including cisplatin [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>, <a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>], carboplatin [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>], Imatinib [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>, <a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>], Sorafenib [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>], and Bevacizumab [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. Our analysis did not find any significant association between PRMT-3, -4, or -8 expression and chemotherapy resistance. However, we observed that PRMT1 and PRMT6 expression levels were significantly higher in the non-responder group compared to the responder group under the specified treatment regimens, consistent with our hypothesis (Fig. <a href="#Fig9" class="usa-link">9</a>). The correlation between low PRMT1 and PRMT6 expression and better therapeutic response reinforces our in vitro findings, supporting the idea that inhibiting type I PRMTs can reduce lysosomal exocytosis-mediated drug tolerance and enhance chemotherapy efficacy.</p>
<figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9. Correlation of PRMT1 and PRMT6 expression levels with patient response to chemotherapy.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12334744_41419_2025_7900_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9c8a/12334744/d18e779e5305/41419_2025_7900_Fig9_HTML.jpg" loading="lazy" id="d33e1105" height="437" width="790" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Expression levels of (<strong>A</strong>) PRMT1 and (<strong>B)</strong> PRMT6 in patients classified as responsive (red) or non-responsive (blue) to various chemotherapy regimens. Patient data were retrieved from the Cancer Treatment Response Gene Signature Database (CTR-DB). Box plots illustrate differences in PRMT1 or PRMT6 expression levels between the two groups, indicating generally higher PRMT expression in non-responsive patients. The specific cancer types, chemotherapy regimens, sample numbers (N), fold-change values (LogFC), statistical significance (<em>p</em>-values), and area under the curve (AUC) values are indicated for each comparison.</p></figcaption></figure></section></section><section id="Sec22"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par44">This study aimed to explore the intersection of epigenetic regulation and lysosomal exocytosis in cancer therapy, particularly in overcoming chemotherapeutic resistance. Our findings underscore the significance of lysosomes in sequestering and expelling chemotherapeutic drugs, a process that contributes to drug resistance. By targeting the epigenetic regulation of lysosomal exocytosis, we sought to enhance the therapeutic efficacy of chemotherapy.</p>
<p id="Par45">We implemented a comprehensive screening process that utilized an epigenetic drug library in order to pinpoint epidrugs that modulate lysosomal exocytosis. The identification of specific epidrugs that inhibit lysosomal exocytosis without broadly affecting lysosomal biogenesis genes underscores the specificity and clinical utility of our findings. We narrowed our analysis to two epidrugs, MS023 and GSK3368715, which target the same epigenetic modifiers, type I PRMTs, emerging as potential molecules that synergize with cisplatin while reducing exocytosis. Notably, the efficacy of the drug combination varied depending on the chemotherapeutic agent used. RNA-seq analysis revealed that these epidrugs affected genes involved in several pathways, including lysosomal function and DNA repair. This suggests that drugs like cisplatin, which cause DNA damage and are sequestered to lysosomes, can synergize through both mechanisms, while Sunitinib, acting through a different pathway, shows less synergy, potentially due to its reliance solely on reduced exocytosis. The observed synergy highlights the potential of using epigenetic drugs to enhance the effectiveness of DNA-damaging agents in cancer therapy by simultaneously targeting multiple pathways, including lysosomal exocytosis.</p>
<p id="Par46">Type I PRMTs regulate critical cellular processes, including DNA damage response, repair, RNA processing, and splicing [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>, <a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]. PRMT1 and PRMT6 influence DNA repair by methylating key proteins [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>, <a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>], while PRMT4 modulates RNA splicing under stress [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. PRMT8, primarily studied in the nervous system, is involved in RNA processing [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>]. Targeting these enzymes with inhibitors is a promising therapeutic strategy, particularly in cancer. Inhibition of PRMT1 in triple-negative breast cancer (TNBC) cells triggers an interferon response, leading to cell death [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>], and disrupts chromatin recruitment, enhancing cisplatin efficacy [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]. Our results indicate that, in addition to the previously discussed mechanisms, Type I PRMT inhibitors decrease lysosomal exocytosis—a novel finding that, to our knowledge, is being described for the first time. This discovery further enhances the potential of these inhibitors in improving chemotherapy efficacy.</p>
<p id="Par47">Our RNA-seq analysis suggested potential PRMT-mediated regulation of lysosomal genes. Although publicly available ChIP-seq datasets provided initial insights, they had limitations, as they originated from different cell lines than those used in our study. To address this, we performed ChIP-qPCR experiments, confirming PRMT1 and PRMT6 occupancy at the promoters of several key genes identified in our RNA-seq analysis. These results support the direct regulation of these genes by type I PRMTs. Moreover, the correlation between low PRMT1 and PRMT6 expression and better therapy response in patient data underscores the relevance of our in vitro results. This supports the idea that type I PRMT inhibitors can mitigate lysosomal exocytosis-mediated drug resistance, potentially enhancing the effectiveness of chemotherapeutic agents, as shown in our cell line studies. While this correlation underscores the potential role of PRMT1 and PRMT6 in influencing chemotherapy response, we recognize that these data alone are not sufficient to establish those PRMTs as definitive predictive biomarkers. Further investigation is warranted to validate these observations and determine their potential clinical relevance.</p>
<p id="Par48">The discovery of MS023 and GSK3368715 as promising candidates for combination therapy, identified through a comprehensive epigenetic drug library, opens new avenues for epigenetic drug research in cancer treatment. However, several key questions remain. The precise mechanisms by which these and other epigenetic drugs influence lysosomal exocytosis and biogenesis need further exploration. In this study, we focused on the TRPML1/TFEB axis as the paradigmal lysosomal regulation pathway. As discussed above, inhibition of type I PRMTs only partially affects this pathway, suggesting the involvement of other regulatory circuits. In line with this, other pathways, including RAS-MAPK/ERK and AMPK, appear to regulate the lysosomal system [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>, <a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]. Our gene expression analysis revealed differential regulation of key lysosomal hydrolases that influence lysosomal exocytosis and cell signaling. NEU1, a known negative regulator of lysosomal exocytosis [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>], was upregulated, while NAGLU and GALNS, involved in glycosaminoglycan degradation [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>–<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>], were downregulated. The resulting accumulation of glycosaminoglycans may disrupt cellular homeostasis and impact signaling pathways. These findings suggest that type I PRMT inhibition may indirectly affect the EGFR-RAS-MAPK signaling axis via lysosomal hydrolase activity, ultimately influencing lysosomal exocytosis and drug sensitivity in cancer cells.</p>
<p id="Par49">Future investigations should clarify whether PRMT1 inhibition counteracts drug-induced lysosomal exocytosis through MAPK-dependent mechanisms and how these pathways interact with epigenetic regulation. Additionally, determining the cancer types most susceptible to this modulation will be essential for translating these findings into therapeutic strategies. Overall, our results establish a mechanistic framework for targeting lysosomal exocytosis through epigenetic interventions as a means to enhance drug sensitivity in cancer cells.</p></section><section id="Sec23"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12334744/bin/41419_2025_7900_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Merged Supplemental Materials</a><sup> (4.6MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors gratefully acknowledge the use of the services and facilities of the Koc University Research Center for Translational Medicine (KUTTAM), funded by the Presidency of Turkey, Head of Strategy and Budget. The following figures were created using BioRender.com and licensed for publication (Fig. <a href="#Fig1" class="usa-link">1</a>: OP278AR62J, Fig. <a href="#Fig2" class="usa-link">2</a>: DK278ARE9Z, Fig. <a href="#Fig8" class="usa-link">8</a>: CB28031ZWV).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>BS conducted the experiments and drafted the manuscript. NY and SCO contributed to the experimental work and provided a critical review of the manuscript. HS performed the bioinformatic analyses. UD critically reviewed the manuscript and served as a scientific advisor to the project. KK was an official contributor to the ICGEB-supported project and acted as co-corresponding author. CAA conceived the ICGEB project that supported this study and contributed to manuscript preparation. All authors reviewed and approved the final version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This study was funded by the International Centre for Genetic Engineering and Biotechnology (ICGEB) under grant number CRP/TUR20-01.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The RNA-seq data generated and analyzed in this study are publicly available in the Gene Expression Omnibus (GEO) under the accession number GSE277490. No other datasets were generated during the current study.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par50">The authors declare no competing interests.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Ethics approval and consent to participate</h2>
<p id="Par51">This study did not involve human participants, and therefore did not require ethical approval or consent to participate. All methods were performed in accordance with the relevant guidelines and regulations.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p>Edited by Dr. Angelo Peschiaroli</p></div>
<div class="fn p" id="fn2"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Kirill Kiselyov, Email: kiselyov@pitt.edu.</p>
<p>Ceyda Acilan, Email: cayhan@ku.edu.tr.</p></section><section id="sec24"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41419-025-07900-w.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Ketkar M, Dutt S. Epigenetic regulation towards acquired drug resistance in cancer. Subcell Biochem. 2022;100:473–502.
</cite> [<a href="https://doi.org/10.1007/978-3-031-07634-3_14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36301503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ketkar%20M,%20Dutt%20S.%20Epigenetic%20regulation%20towards%20acquired%20drug%20resistance%20in%20cancer.%20Subcell%20Biochem.%202022;100:473%E2%80%93502." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Wang N, Ma T, Yu B. Targeting epigenetic regulators to overcome drug resistance in cancers. Signal Transduct Target Ther. 2023;8:69.
</cite> [<a href="https://doi.org/10.1038/s41392-023-01341-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9935618/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36797239/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20N,%20Ma%20T,%20Yu%20B.%20Targeting%20epigenetic%20regulators%20to%20overcome%20drug%20resistance%20in%20cancers.%20Signal%20Transduct%20Target%20Ther.%202023;8:69." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Guo L, Lee YT, Zhou Y, Huang Y. Targeting epigenetic regulatory machinery to overcome cancer therapy resistance. Semin Cancer Biol. 2022;83:487–502.
</cite> [<a href="https://doi.org/10.1016/j.semcancer.2020.12.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8257754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33421619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20L,%20Lee%20YT,%20Zhou%20Y,%20Huang%20Y.%20Targeting%20epigenetic%20regulatory%20machinery%20to%20overcome%20cancer%20therapy%20resistance.%20Semin%20Cancer%20Biol.%202022;83:487%E2%80%93502." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Cheng Y, He C, Wang M, Ma X, Mo F, Yang S, et al. Targeting epigenetic regulators for cancer therapy: mechanisms and advances in clinical trials. Signal Transduct Target Ther. 2019;4:62.
</cite> [<a href="https://doi.org/10.1038/s41392-019-0095-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6915746/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31871779/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng%20Y,%20He%20C,%20Wang%20M,%20Ma%20X,%20Mo%20F,%20Yang%20S,%20et%20al.%20Targeting%20epigenetic%20regulators%20for%20cancer%20therapy:%20mechanisms%20and%20advances%20in%20clinical%20trials.%20Signal%20Transduct%20Target%20Ther.%202019;4:62." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Lu Y, Chan YT, Tan HY, Li S, Wang N, Feng Y. Epigenetic regulation in human cancer: the potential role of epi-drug in cancer therapy. Mol Cancer. 2020;19:79.
</cite> [<a href="https://doi.org/10.1186/s12943-020-01197-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7184703/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32340605/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu%20Y,%20Chan%20YT,%20Tan%20HY,%20Li%20S,%20Wang%20N,%20Feng%20Y.%20Epigenetic%20regulation%20in%20human%20cancer:%20the%20potential%20role%20of%20epi-drug%20in%20cancer%20therapy.%20Mol%20Cancer.%202020;19:79." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Tao F, Zhang Z. Editorial: Epigenetic drugs and therapeutic resistance for epithelial malignancies. Front Pharm. 2023;14:1208518.</cite> [<a href="https://doi.org/10.3389/fphar.2023.1208518" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10203627/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37229257/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tao%20F,%20Zhang%20Z.%20Editorial:%20Epigenetic%20drugs%20and%20therapeutic%20resistance%20for%20epithelial%20malignancies.%20Front%20Pharm.%202023;14:1208518." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Gladkova MG, Leidmaa E, Anderzhanova EA. Epidrugs in the therapy of central nervous system disorders: a way to drive on? Cells. 2023;12:1464.</cite> [<a href="https://doi.org/10.3390/cells12111464" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10253154/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37296584/" class="usa-link">PubMed</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Wong M, Gertz B, Chestnut BA, Martin LJ. Mitochondrial DNMT3A and DNA methylation in skeletal muscle and CNS of transgenic mouse models of ALS. Front Cell Neurosci. 2013;7:279.
</cite> [<a href="https://doi.org/10.3389/fncel.2013.00279" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3872319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24399935/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wong%20M,%20Gertz%20B,%20Chestnut%20BA,%20Martin%20LJ.%20Mitochondrial%20DNMT3A%20and%20DNA%20methylation%20in%20skeletal%20muscle%20and%20CNS%20of%20transgenic%20mouse%20models%20of%20ALS.%20Front%20Cell%20Neurosci.%202013;7:279." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Maceyka M, Milstien S, Spiegel S. The potential of histone deacetylase inhibitors in Niemann - Pick type C disease. FEBS J. 2013;280:6367–72.
</cite> [<a href="https://doi.org/10.1111/febs.12505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3857996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23992240/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maceyka%20M,%20Milstien%20S,%20Spiegel%20S.%20The%20potential%20of%20histone%20deacetylase%20inhibitors%20in%20Niemann%20-%20Pick%20type%20C%20disease.%20FEBS%20J.%202013;280:6367%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Pipalia NH, Cosner CC, Huang A, Chatterjee A, Bourbon P, Farley N, et al. Histone deacetylase inhibitor treatment dramatically reduces cholesterol accumulation in Niemann-Pick type C1 mutant human fibroblasts. Proc Natl Acad Sci USA. 2011;108:5620–5.
</cite> [<a href="https://doi.org/10.1073/pnas.1014890108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3078401/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21436030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pipalia%20NH,%20Cosner%20CC,%20Huang%20A,%20Chatterjee%20A,%20Bourbon%20P,%20Farley%20N,%20et%20al.%20Histone%20deacetylase%20inhibitor%20treatment%20dramatically%20reduces%20cholesterol%20accumulation%20in%20Niemann-Pick%20type%20C1%20mutant%20human%20fibroblasts.%20Proc%20Natl%20Acad%20Sci%20USA.%202011;108:5620%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Wehrmann ZT, Hulett TW, Huegel KL, Vaughan KT, Wiest O, Helquist P, et al. Quantitative comparison of the efficacy of various compounds in lowering intracellular cholesterol levels in Niemann-Pick type C fibroblasts. PLoS One. 2012;7:e48561.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0048561" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3483179/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23144769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wehrmann%20ZT,%20Hulett%20TW,%20Huegel%20KL,%20Vaughan%20KT,%20Wiest%20O,%20Helquist%20P,%20et%20al.%20Quantitative%20comparison%20of%20the%20efficacy%20of%20various%20compounds%20in%20lowering%20intracellular%20cholesterol%20levels%20in%20Niemann-Pick%20type%20C%20fibroblasts.%20PLoS%20One.%202012;7:e48561." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Lu J, Yang C, Chen M, Ye DY, Lonser RR, Brady RO, et al. Histone deacetylase inhibitors prevent the degradation and restore the activity of glucocerebrosidase in Gaucher disease. Proc Natl Acad Sci USA. 2011;108:21200–5.
</cite> [<a href="https://doi.org/10.1073/pnas.1119181109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3248545/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22160715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu%20J,%20Yang%20C,%20Chen%20M,%20Ye%20DY,%20Lonser%20RR,%20Brady%20RO,%20et%20al.%20Histone%20deacetylase%20inhibitors%20prevent%20the%20degradation%20and%20restore%20the%20activity%20of%20glucocerebrosidase%20in%20Gaucher%20disease.%20Proc%20Natl%20Acad%20Sci%20USA.%202011;108:21200%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Yang C, Rahimpour S, Lu J, Pacak K, Ikejiri B, Brady RO, et al. Histone deacetylase inhibitors increase glucocerebrosidase activity in Gaucher disease by modulation of molecular chaperones. Proc Natl Acad Sci USA. 2013;110:966–71.
</cite> [<a href="https://doi.org/10.1073/pnas.1221046110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3549125/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23277556/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20C,%20Rahimpour%20S,%20Lu%20J,%20Pacak%20K,%20Ikejiri%20B,%20Brady%20RO,%20et%20al.%20Histone%20deacetylase%20inhibitors%20increase%20glucocerebrosidase%20activity%20in%20Gaucher%20disease%20by%20modulation%20of%20molecular%20chaperones.%20Proc%20Natl%20Acad%20Sci%20USA.%202013;110:966%E2%80%9371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Hrabeta J, Belhajova M, Subrtova H, Merlos Rodrigo MA, Heger Z, Eckschlager T. Drug sequestration in lysosomes as one of the mechanisms of chemoresistance of cancer cells and the possibilities of its inhibition. Int J Mol Sci. 2020;21:4392.</cite> [<a href="https://doi.org/10.3390/ijms21124392" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7352242/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32575682/" class="usa-link">PubMed</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Halaby R. Influence of lysosomal sequestration on multidrug resistance in cancer cells. Cancer Drug Resist. 2019;2:31–42.
</cite> [<a href="https://doi.org/10.20517/cdr.2018.23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9019175/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35582144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Halaby%20R.%20Influence%20of%20lysosomal%20sequestration%20on%20multidrug%20resistance%20in%20cancer%20cells.%20Cancer%20Drug%20Resist.%202019;2:31%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Zhong D, Wang R, Zhang H, Wang M, Zhang X, Chen H. Induction of lysosomal exocytosis and biogenesis via TRPML1 activation for the treatment of uranium-induced nephrotoxicity. Nat Commun. 2023;14:3997.
</cite> [<a href="https://doi.org/10.1038/s41467-023-39716-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10326073/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37414766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhong%20D,%20Wang%20R,%20Zhang%20H,%20Wang%20M,%20Zhang%20X,%20Chen%20H.%20Induction%20of%20lysosomal%20exocytosis%20and%20biogenesis%20via%20TRPML1%20activation%20for%20the%20treatment%20of%20uranium-induced%20nephrotoxicity.%20Nat%20Commun.%202023;14:3997." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Havelaar AC, de Gast IL, Snijders S, Beerens CE, Mancini GM, Verheijen FW. Characterization of a heavy metal ion transporter in the lysosomal membrane. FEBS Lett. 1998;436:223–7.
</cite> [<a href="https://doi.org/10.1016/s0014-5793(98)01133-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9781683/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Havelaar%20AC,%20de%20Gast%20IL,%20Snijders%20S,%20Beerens%20CE,%20Mancini%20GM,%20Verheijen%20FW.%20Characterization%20of%20a%20heavy%20metal%20ion%20transporter%20in%20the%20lysosomal%20membrane.%20FEBS%20Lett.%201998;436:223%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Kim JH, Matsubara T, Lee J, Fenollar-Ferrer C, Han K, Kim D, et al. Lysosomal SLC46A3 modulates hepatic cytosolic copper homeostasis. Nat Commun. 2021;12:290.
</cite> [<a href="https://doi.org/10.1038/s41467-020-20461-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7804329/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33436590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20JH,%20Matsubara%20T,%20Lee%20J,%20Fenollar-Ferrer%20C,%20Han%20K,%20Kim%20D,%20et%20al.%20Lysosomal%20SLC46A3%20modulates%20hepatic%20cytosolic%20copper%20homeostasis.%20Nat%20Commun.%202021;12:290." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Kiselyov K, Colletti GA, Terwilliger A, Ketchum K, Lyons CW, Quinn J, et al. TRPML: transporters of metals in lysosomes essential for cell survival?. Cell Calcium. 2011;50:288–94.
</cite> [<a href="https://doi.org/10.1016/j.ceca.2011.04.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3164942/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21621258/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kiselyov%20K,%20Colletti%20GA,%20Terwilliger%20A,%20Ketchum%20K,%20Lyons%20CW,%20Quinn%20J,%20et%20al.%20TRPML:%20transporters%20of%20metals%20in%20lysosomes%20essential%20for%20cell%20survival?.%20Cell%20Calcium.%202011;50:288%E2%80%9394." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Kalayda GV, Wagner CH, Buss I, Reedijk J, Jaehde U. Altered localisation of the copper efflux transporters ATP7A and ATP7B associated with cisplatin resistance in human ovarian carcinoma cells. BMC Cancer. 2008;8:175.
</cite> [<a href="https://doi.org/10.1186/1471-2407-8-175" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2442842/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18565219/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kalayda%20GV,%20Wagner%20CH,%20Buss%20I,%20Reedijk%20J,%20Jaehde%20U.%20Altered%20localisation%20of%20the%20copper%20efflux%20transporters%20ATP7A%20and%20ATP7B%20associated%20with%20cisplatin%20resistance%20in%20human%20ovarian%20carcinoma%20cells.%20BMC%20Cancer.%202008;8:175." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Samimi G, Katano K, Holzer AK, Safaei R, Howell SB. Modulation of the cellular pharmacology of cisplatin and its analogs by the copper exporters ATP7A and ATP7B. Mol Pharm. 2004;66:25–32.</cite> [<a href="https://doi.org/10.1124/mol.66.1.25" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15213293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Samimi%20G,%20Katano%20K,%20Holzer%20AK,%20Safaei%20R,%20Howell%20SB.%20Modulation%20of%20the%20cellular%20pharmacology%20of%20cisplatin%20and%20its%20analogs%20by%20the%20copper%20exporters%20ATP7A%20and%20ATP7B.%20Mol%20Pharm.%202004;66:25%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Zhitomirsky B, Assaraf YG. Lysosomal sequestration of hydrophobic weak base chemotherapeutics triggers lysosomal biogenesis and lysosome-dependent cancer multidrug resistance. Oncotarget. 2015;6:1143–56.
</cite> [<a href="https://doi.org/10.18632/oncotarget.2732" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4359223/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25544758/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhitomirsky%20B,%20Assaraf%20YG.%20Lysosomal%20sequestration%20of%20hydrophobic%20weak%20base%20chemotherapeutics%20triggers%20lysosomal%20biogenesis%20and%20lysosome-dependent%20cancer%20multidrug%20resistance.%20Oncotarget.%202015;6:1143%E2%80%9356." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Stark M, Silva TFD, Levin G, Machuqueiro M, Assaraf YG. The lysosomotropic activity of hydrophobic weak base drugs is mediated via their intercalation into the lysosomal membrane. Cells. 2020;9:1082.</cite> [<a href="https://doi.org/10.3390/cells9051082" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7290590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32349204/" class="usa-link">PubMed</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Zhitomirsky B, Assaraf YG. Lysosomal accumulation of anticancer drugs triggers lysosomal exocytosis. Oncotarget. 2017;8:45117–32.
</cite> [<a href="https://doi.org/10.18632/oncotarget.15155" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5542171/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28187461/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhitomirsky%20B,%20Assaraf%20YG.%20Lysosomal%20accumulation%20of%20anticancer%20drugs%20triggers%20lysosomal%20exocytosis.%20Oncotarget.%202017;8:45117%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Tancini B, Buratta S, Delo F, Sagini K, Chiaradia E, Pellegrino RM, et al. Lysosomal exocytosis: the extracellular role of an intracellular organelle. Membranes. 2020;10:406.</cite> [<a href="https://doi.org/10.3390/membranes10120406" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7764620/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33316913/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Machado E, White-Gilbertson S, van de Vlekkert D, Janke L, Moshiach S, Campos Y, et al. Regulated lysosomal exocytosis mediates cancer progression. Sci Adv. 2015;1:e1500603.
</cite> [<a href="https://doi.org/10.1126/sciadv.1500603" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4730843/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26824057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Machado%20E,%20White-Gilbertson%20S,%20van%20de%20Vlekkert%20D,%20Janke%20L,%20Moshiach%20S,%20Campos%20Y,%20et%20al.%20Regulated%20lysosomal%20exocytosis%20mediates%20cancer%20progression.%20Sci%20Adv.%202015;1:e1500603." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Machado ER, Annunziata I, van de Vlekkert D, Grosveld GC, d’Azzo A. Lysosomes and Cancer Progression: A Malignant Liaison. Front Cell Dev Biol. 2021;9:642494.
</cite> [<a href="https://doi.org/10.3389/fcell.2021.642494" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7952443/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33718382/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Machado%20ER,%20Annunziata%20I,%20van%20de%20Vlekkert%20D,%20Grosveld%20GC,%20d%E2%80%99Azzo%20A.%20Lysosomes%20and%20Cancer%20Progression:%20A%20Malignant%20Liaison.%20Front%20Cell%20Dev%20Biol.%202021;9:642494." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Shi Q, Shen Q, Liu Y, Shi Y, Huang W, Wang X, et al. Increased glucose metabolism in TAMs fuels O-GlcNAcylation of lysosomal Cathepsin B to promote cancer metastasis and chemoresistance. Cancer Cell. 2022;40:1207–22.e10.
</cite> [<a href="https://doi.org/10.1016/j.ccell.2022.08.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36084651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shi%20Q,%20Shen%20Q,%20Liu%20Y,%20Shi%20Y,%20Huang%20W,%20Wang%20X,%20et%20al.%20Increased%20glucose%20metabolism%20in%20TAMs%20fuels%20O-GlcNAcylation%20of%20lysosomal%20Cathepsin%20B%20to%20promote%20cancer%20metastasis%20and%20chemoresistance.%20Cancer%20Cell.%202022;40:1207%E2%80%9322.e10." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Kos J, Mitrovic A, Perisic Nanut M, Pislar A. Lysosomal peptidases-intriguing roles in cancer progression and neurodegeneration. FEBS Open Bio. 2022;12:708–38.
</cite> [<a href="https://doi.org/10.1002/2211-5463.13372" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8972049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35067006/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kos%20J,%20Mitrovic%20A,%20Perisic%20Nanut%20M,%20Pislar%20A.%20Lysosomal%20peptidases-intriguing%20roles%20in%20cancer%20progression%20and%20neurodegeneration.%20FEBS%20Open%20Bio.%202022;12:708%E2%80%9338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Kundu ST, Grzeskowiak CL, Fradette JJ, Gibson LA, Rodriguez LB, Creighton CJ, et al. TMEM106B drives lung cancer metastasis by inducing TFEB-dependent lysosome synthesis and secretion of cathepsins. Nat Commun. 2018;9:2731.
</cite> [<a href="https://doi.org/10.1038/s41467-018-05013-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6048095/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30013069/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kundu%20ST,%20Grzeskowiak%20CL,%20Fradette%20JJ,%20Gibson%20LA,%20Rodriguez%20LB,%20Creighton%20CJ,%20et%20al.%20TMEM106B%20drives%20lung%20cancer%20metastasis%20by%20inducing%20TFEB-dependent%20lysosome%20synthesis%20and%20secretion%20of%20cathepsins.%20Nat%20Commun.%202018;9:2731." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Pena K, Coblenz J, Kiselyov K. Brief exposure to copper activates lysosomal exocytosis. Cell Calcium. 2015;57:257–62.
</cite> [<a href="https://doi.org/10.1016/j.ceca.2015.01.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4363267/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25620123/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pena%20K,%20Coblenz%20J,%20Kiselyov%20K.%20Brief%20exposure%20to%20copper%20activates%20lysosomal%20exocytosis.%20Cell%20Calcium.%202015;57:257%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Vyas A, Gomez-Casal R, Cruz-Rangel S, Villanueva H, Sikora AG, Rajagopalan P, et al. Lysosomal inhibition sensitizes TMEM16A-expressing cancer cells to chemotherapy. Proc Natl Acad Sci USA. 2022;119:e2100670119.
</cite> [<a href="https://doi.org/10.1073/pnas.2100670119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8944912/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35286200/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vyas%20A,%20Gomez-Casal%20R,%20Cruz-Rangel%20S,%20Villanueva%20H,%20Sikora%20AG,%20Rajagopalan%20P,%20et%20al.%20Lysosomal%20inhibition%20sensitizes%20TMEM16A-expressing%20cancer%20cells%20to%20chemotherapy.%20Proc%20Natl%20Acad%20Sci%20USA.%202022;119:e2100670119." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Kim B, Kim G, Kim H, Song YS, Jung J. Modulation of cisplatin sensitivity through TRPML1-mediated lysosomal exocytosis in ovarian cancer cells: a comprehensive metabolomic approach. Cells. 2024;13:115.</cite> [<a href="https://doi.org/10.3390/cells13020115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10814698/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38247807/" class="usa-link">PubMed</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Merlos Rodrigo MA, Buchtelova H, de Los Rios V, Casal JI, Eckschlager T, Hrabeta J, et al. Proteomic signature of neuroblastoma cells UKF-NB-4 reveals key role of lysosomal sequestration and the proteasome complex in acquiring chemoresistance to cisplatin. J Proteome Res. 2019;18:1255–63.
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.8b00867" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30592607/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Merlos%20Rodrigo%20MA,%20Buchtelova%20H,%20de%20Los%20Rios%20V,%20Casal%20JI,%20Eckschlager%20T,%20Hrabeta%20J,%20et%20al.%20Proteomic%20signature%20of%20neuroblastoma%20cells%20UKF-NB-4%20reveals%20key%20role%20of%20lysosomal%20sequestration%20and%20the%20proteasome%20complex%20in%20acquiring%20chemoresistance%20to%20cisplatin.%20J%20Proteome%20Res.%202019;18:1255%E2%80%9363." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Yanes RE, Tarn D, Hwang AA, Ferris DP, Sherman SP, Thomas CR, et al. Involvement of lysosomal exocytosis in the excretion of mesoporous silica nanoparticles and enhancement of the drug delivery effect by exocytosis inhibition. Small. 2013;9:697–704.
</cite> [<a href="https://doi.org/10.1002/smll.201201811" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3767416/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23152124/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yanes%20RE,%20Tarn%20D,%20Hwang%20AA,%20Ferris%20DP,%20Sherman%20SP,%20Thomas%20CR,%20et%20al.%20Involvement%20of%20lysosomal%20exocytosis%20in%20the%20excretion%20of%20mesoporous%20silica%20nanoparticles%20and%20enhancement%20of%20the%20drug%20delivery%20effect%20by%20exocytosis%20inhibition.%20Small.%202013;9:697%E2%80%93704." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Ridinger J, Koeneke E, Kolbinger FR, Koerholz K, Mahboobi S, Hellweg L, et al. Dual role of HDAC10 in lysosomal exocytosis and DNA repair promotes neuroblastoma chemoresistance. Sci Rep. 2018;8:10039.
</cite> [<a href="https://doi.org/10.1038/s41598-018-28265-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6030077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29968769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ridinger%20J,%20Koeneke%20E,%20Kolbinger%20FR,%20Koerholz%20K,%20Mahboobi%20S,%20Hellweg%20L,%20et%20al.%20Dual%20role%20of%20HDAC10%20in%20lysosomal%20exocytosis%20and%20DNA%20repair%20promotes%20neuroblastoma%20chemoresistance.%20Sci%20Rep.%202018;8:10039." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Annunziata I, van de Vlekkert D, Wolf E, Finkelstein D, Neale G, Machado E, et al. MYC competes with MiT/TFE in regulating lysosomal biogenesis and autophagy through an epigenetic rheostat. Nat Commun. 2019;10:3623.
</cite> [<a href="https://doi.org/10.1038/s41467-019-11568-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6689058/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31399583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Annunziata%20I,%20van%20de%20Vlekkert%20D,%20Wolf%20E,%20Finkelstein%20D,%20Neale%20G,%20Machado%20E,%20et%20al.%20MYC%20competes%20with%20MiT/TFE%20in%20regulating%20lysosomal%20biogenesis%20and%20autophagy%20through%20an%20epigenetic%20rheostat.%20Nat%20Commun.%202019;10:3623." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Latifkar A, Ling L, Hingorani A, Johansen E, Clement A, Zhang X, et al. Loss of sirtuin 1 alters the secretome of breast cancer cells by impairing lysosomal integrity. Dev Cell. 2019;49:393–408.e7.
</cite> [<a href="https://doi.org/10.1016/j.devcel.2019.03.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6519475/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30982660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Latifkar%20A,%20Ling%20L,%20Hingorani%20A,%20Johansen%20E,%20Clement%20A,%20Zhang%20X,%20et%20al.%20Loss%20of%20sirtuin%201%20alters%20the%20secretome%20of%20breast%20cancer%20cells%20by%20impairing%20lysosomal%20integrity.%20Dev%20Cell.%202019;49:393%E2%80%93408.e7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Hwang JW, Cho Y, Bae GU, Kim SN, Kim YK. Protein arginine methyltransferases: promising targets for cancer therapy. Exp Mol Med. 2021;53:788–808.
</cite> [<a href="https://doi.org/10.1038/s12276-021-00613-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8178397/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34006904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hwang%20JW,%20Cho%20Y,%20Bae%20GU,%20Kim%20SN,%20Kim%20YK.%20Protein%20arginine%20methyltransferases:%20promising%20targets%20for%20cancer%20therapy.%20Exp%20Mol%20Med.%202021;53:788%E2%80%93808." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Wu K, Niu C, Liu H, Fu L. Research progress on PRMTs involved in epigenetic modification and tumour signalling pathway regulation. Int J Oncol. 2023;62:62.</cite> [<a href="https://doi.org/10.3892/ijo.2023.5510" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10147102/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37026519/" class="usa-link">PubMed</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Wei HH, Fan XJ, Hu Y, Tian XX, Guo M, Mao MW, et al. A systematic survey of PRMT interactomes reveals the key roles of arginine methylation in the global control of RNA splicing and translation. Sci Bull. 2021;66:1342–57.</cite> [<a href="https://doi.org/10.1016/j.scib.2021.01.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36654156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wei%20HH,%20Fan%20XJ,%20Hu%20Y,%20Tian%20XX,%20Guo%20M,%20Mao%20MW,%20et%20al.%20A%20systematic%20survey%20of%20PRMT%20interactomes%20reveals%20the%20key%20roles%20of%20arginine%20methylation%20in%20the%20global%20control%20of%20RNA%20splicing%20and%20translation.%20Sci%20Bull.%202021;66:1342%E2%80%9357." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Grypari IM, Logotheti S, Zolota V, Troncoso P, Efstathiou E, Bravou V, et al. The protein arginine methyltransferases (PRMTs) PRMT1 and CARM1 as candidate epigenetic drivers in prostate cancer progression. Medicine. 2021;100:e27094.
</cite> [<a href="https://doi.org/10.1097/MD.0000000000027094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8428700/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34516499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grypari%20IM,%20Logotheti%20S,%20Zolota%20V,%20Troncoso%20P,%20Efstathiou%20E,%20Bravou%20V,%20et%20al.%20The%20protein%20arginine%20methyltransferases%20(PRMTs)%20PRMT1%20and%20CARM1%20as%20candidate%20epigenetic%20drivers%20in%20prostate%20cancer%20progression.%20Medicine.%202021;100:e27094." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Nicholson TB, Chen T, Richard S. The physiological and pathophysiological role of PRMT1-mediated protein arginine methylation. Pharmacol Res. 2009;60:466–74.
</cite> [<a href="https://doi.org/10.1016/j.phrs.2009.07.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19643181/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nicholson%20TB,%20Chen%20T,%20Richard%20S.%20The%20physiological%20and%20pathophysiological%20role%20of%20PRMT1-mediated%20protein%20arginine%20methylation.%20Pharmacol%20Res.%202009;60:466%E2%80%9374." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Fedoriw A, Rajapurkar SR, O’Brien S, Gerhart SV, Mitchell LH, Adams ND, et al. Anti-tumor Activity of the Type I PRMT Inhibitor, GSK3368715, Synergizes with PRMT5 Inhibition through MTAP Loss. Cancer Cell. 2019;36:100–14.e25.
</cite> [<a href="https://doi.org/10.1016/j.ccell.2019.05.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31257072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fedoriw%20A,%20Rajapurkar%20SR,%20O%E2%80%99Brien%20S,%20Gerhart%20SV,%20Mitchell%20LH,%20Adams%20ND,%20et%20al.%20Anti-tumor%20Activity%20of%20the%20Type%20I%20PRMT%20Inhibitor,%20GSK3368715,%20Synergizes%20with%20PRMT5%20Inhibition%20through%20MTAP%20Loss.%20Cancer%20Cell.%202019;36:100%E2%80%9314.e25." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Eram MS, Shen Y, Szewczyk M, Wu H, Senisterra G, Li F, et al. A potent, selective, and cell-active inhibitor of human type I protein arginine methyltransferases. ACS Chem Biol. 2016;11:772–81.
</cite> [<a href="https://doi.org/10.1021/acschembio.5b00839" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4798913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26598975/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eram%20MS,%20Shen%20Y,%20Szewczyk%20M,%20Wu%20H,%20Senisterra%20G,%20Li%20F,%20et%20al.%20A%20potent,%20selective,%20and%20cell-active%20inhibitor%20of%20human%20type%20I%20protein%20arginine%20methyltransferases.%20ACS%20Chem%20Biol.%202016;11:772%E2%80%9381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Di Paola S, Medina DL. TRPML1-/TFEB-dependent regulation of lysosomal exocytosis. Methods Mol Biol. 2019;1925:143–4.
</cite> [<a href="https://doi.org/10.1007/978-1-4939-9018-4_12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30674023/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Di%20Paola%20S,%20Medina%20DL.%20TRPML1-/TFEB-dependent%20regulation%20of%20lysosomal%20exocytosis.%20Methods%20Mol%20Biol.%202019;1925:143%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Shen DW, Ma J, Okabe M, Zhang G, Xia D, Gottesman MM. Elevated expression of TMEM205, a hypothetical membrane protein, is associated with cisplatin resistance. J Cell Physiol. 2010;225:822–8.
</cite> [<a href="https://doi.org/10.1002/jcp.22287" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2971691/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20589834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen%20DW,%20Ma%20J,%20Okabe%20M,%20Zhang%20G,%20Xia%20D,%20Gottesman%20MM.%20Elevated%20expression%20of%20TMEM205,%20a%20hypothetical%20membrane%20protein,%20is%20associated%20with%20cisplatin%20resistance.%20J%20Cell%20Physiol.%202010;225:822%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Calo CA, Smith BQ, Dorayappan KDP, Saini U, Lightfoot M, Wagner V, et al. Aberrant expression of TMEM205 signaling promotes platinum resistance in ovarian cancer: An implication for the antitumor potential of DAP compound. Gynecol Oncol. 2022;164:136–45.
</cite> [<a href="https://doi.org/10.1016/j.ygyno.2021.10.076" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34756749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Calo%20CA,%20Smith%20BQ,%20Dorayappan%20KDP,%20Saini%20U,%20Lightfoot%20M,%20Wagner%20V,%20et%20al.%20Aberrant%20expression%20of%20TMEM205%20signaling%20promotes%20platinum%20resistance%20in%20ovarian%20cancer:%20An%20implication%20for%20the%20antitumor%20potential%20of%20DAP%20compound.%20Gynecol%20Oncol.%202022;164:136%E2%80%9345." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Chapuy B, Koch R, Radunski U, Corsham S, Cheong N, Inagaki N, et al. Intracellular ABC transporter A3 confers multidrug resistance in leukemia cells by lysosomal drug sequestration. Leukemia. 2008;22:1576–86.
</cite> [<a href="https://doi.org/10.1038/leu.2008.103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18463677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chapuy%20B,%20Koch%20R,%20Radunski%20U,%20Corsham%20S,%20Cheong%20N,%20Inagaki%20N,%20et%20al.%20Intracellular%20ABC%20transporter%20A3%20confers%20multidrug%20resistance%20in%20leukemia%20cells%20by%20lysosomal%20drug%20sequestration.%20Leukemia.%202008;22:1576%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Beers MF, Mulugeta S. The biology of the ABCA3 lipid transporter in lung health and disease. Cell Tissue Res. 2017;367:481–93.
</cite> [<a href="https://doi.org/10.1007/s00441-016-2554-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5321817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28025703/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beers%20MF,%20Mulugeta%20S.%20The%20biology%20of%20the%20ABCA3%20lipid%20transporter%20in%20lung%20health%20and%20disease.%20Cell%20Tissue%20Res.%202017;367:481%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Gotink KJ, Broxterman HJ, Labots M, de Haas RR, Dekker H, Honeywell RJ, et al. Lysosomal sequestration of sunitinib: a novel mechanism of drug resistance. Clin Cancer Res. 2011;17:7337–46.
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-11-1667" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4461037/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21980135/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gotink%20KJ,%20Broxterman%20HJ,%20Labots%20M,%20de%20Haas%20RR,%20Dekker%20H,%20Honeywell%20RJ,%20et%20al.%20Lysosomal%20sequestration%20of%20sunitinib:%20a%20novel%20mechanism%20of%20drug%20resistance.%20Clin%20Cancer%20Res.%202011;17:7337%E2%80%9346." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Azijli K, Gotink KJ, Verheul HM. The potential role of lysosomal sequestration in sunitinib resistance of renal cell cancer. J Kidney Cancer VHL. 2015;2:195–203.
</cite> [<a href="https://doi.org/10.15586/jkcvhl.2015.44" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5345522/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28326274/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Azijli%20K,%20Gotink%20KJ,%20Verheul%20HM.%20The%20potential%20role%20of%20lysosomal%20sequestration%20in%20sunitinib%20resistance%20of%20renal%20cell%20cancer.%20J%20Kidney%20Cancer%20VHL.%202015;2:195%E2%80%93203." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Guo B, Tam A, Santi SA, Parissenti AM. Role of autophagy and lysosomal drug sequestration in acquired resistance to doxorubicin in MCF-7 cells. BMC Cancer. 2016;16:762.
</cite> [<a href="https://doi.org/10.1186/s12885-016-2790-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5043608/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27687594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20B,%20Tam%20A,%20Santi%20SA,%20Parissenti%20AM.%20Role%20of%20autophagy%20and%20lysosomal%20drug%20sequestration%20in%20acquired%20resistance%20to%20doxorubicin%20in%20MCF-7%20cells.%20BMC%20Cancer.%202016;16:762." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Herlevsen M, Oxford G, Owens CR, Conaway M, Theodorescu D. Depletion of major vault protein increases doxorubicin sensitivity and nuclear accumulation and disrupts its sequestration in lysosomes. Mol Cancer Ther. 2007;6:1804–13.
</cite> [<a href="https://doi.org/10.1158/1535-7163.MCT-06-0372" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17575109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Herlevsen%20M,%20Oxford%20G,%20Owens%20CR,%20Conaway%20M,%20Theodorescu%20D.%20Depletion%20of%20major%20vault%20protein%20increases%20doxorubicin%20sensitivity%20and%20nuclear%20accumulation%20and%20disrupts%20its%20sequestration%20in%20lysosomes.%20Mol%20Cancer%20Ther.%202007;6:1804%E2%80%9313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Liu Z, Liu J, Liu X, Wang X, Xie Q, Zhang X, et al. CTR-DB, an omnibus for patient-derived gene expression signatures correlated with cancer drug response. Nucleic Acids Res. 2022;50:D1184–D99.
</cite> [<a href="https://doi.org/10.1093/nar/gkab860" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34570230/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20Z,%20Liu%20J,%20Liu%20X,%20Wang%20X,%20Xie%20Q,%20Zhang%20X,%20et%20al.%20CTR-DB,%20an%20omnibus%20for%20patient-derived%20gene%20expression%20signatures%20correlated%20with%20cancer%20drug%20response.%20Nucleic%20Acids%20Res.%202022;50:D1184%E2%80%93D99." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Circu M, Cardelli J, Barr MP, O’Byrne K, Mills G, El-Osta H. Modulating lysosomal function through lysosome membrane permeabilization or autophagy suppression restores sensitivity to cisplatin in refractory non-small-cell lung cancer cells. PLoS One. 2017;12:e0184922.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0184922" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5612465/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28945807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Circu%20M,%20Cardelli%20J,%20Barr%20MP,%20O%E2%80%99Byrne%20K,%20Mills%20G,%20El-Osta%20H.%20Modulating%20lysosomal%20function%20through%20lysosome%20membrane%20permeabilization%20or%20autophagy%20suppression%20restores%20sensitivity%20to%20cisplatin%20in%20refractory%20non-small-cell%20lung%20cancer%20cells.%20PLoS%20One.%202017;12:e0184922." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Toriyama K, Takano N, Kokuba H, Kazama H, Moriya S, Hiramoto M, et al. Azithromycin enhances the cytotoxicity of DNA-damaging drugs via lysosomal membrane permeabilization in lung cancer cells. Cancer Sci. 2021;112:3324–37.
</cite> [<a href="https://doi.org/10.1111/cas.14992" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8353917/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34051014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Toriyama%20K,%20Takano%20N,%20Kokuba%20H,%20Kazama%20H,%20Moriya%20S,%20Hiramoto%20M,%20et%20al.%20Azithromycin%20enhances%20the%20cytotoxicity%20of%20DNA-damaging%20drugs%20via%20lysosomal%20membrane%20permeabilization%20in%20lung%20cancer%20cells.%20Cancer%20Sci.%202021;112:3324%E2%80%9337." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Burger H, den Dekker AT, Segeletz S, Boersma AW, de Bruijn P, Debiec-Rychter M, et al. Lysosomal Sequestration Determines Intracellular Imatinib Levels. Mol Pharm. 2015;88:477–87.</cite> [<a href="https://doi.org/10.1124/mol.114.097451" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26108972/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Burger%20H,%20den%20Dekker%20AT,%20Segeletz%20S,%20Boersma%20AW,%20de%20Bruijn%20P,%20Debiec-Rychter%20M,%20et%20al.%20Lysosomal%20Sequestration%20Determines%20Intracellular%20Imatinib%20Levels.%20Mol%20Pharm.%202015;88:477%E2%80%9387." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Chapuy B, Panse M, Radunski U, Koch R, Wenzel D, Inagaki N, et al. ABC transporter A3 facilitates lysosomal sequestration of imatinib and modulates susceptibility of chronic myeloid leukemia cell lines to this drug. Haematologica. 2009;94:1528–36.
</cite> [<a href="https://doi.org/10.3324/haematol.2009.008631" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2770963/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19880777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chapuy%20B,%20Panse%20M,%20Radunski%20U,%20Koch%20R,%20Wenzel%20D,%20Inagaki%20N,%20et%20al.%20ABC%20transporter%20A3%20facilitates%20lysosomal%20sequestration%20of%20imatinib%20and%20modulates%20susceptibility%20of%20chronic%20myeloid%20leukemia%20cell%20lines%20to%20this%20drug.%20Haematologica.%202009;94:1528%E2%80%9336." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Colombo F, Trombetta E, Cetrangolo P, Maggioni M, Razini P, De Santis F, et al. Giant Lysosomes as a Chemotherapy Resistance Mechanism in Hepatocellular Carcinoma Cells. PLoS One. 2014;9:e114787.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0114787" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4262459/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25493932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Colombo%20F,%20Trombetta%20E,%20Cetrangolo%20P,%20Maggioni%20M,%20Razini%20P,%20De%20Santis%20F,%20et%20al.%20Giant%20Lysosomes%20as%20a%20Chemotherapy%20Resistance%20Mechanism%20in%20Hepatocellular%20Carcinoma%20Cells.%20PLoS%20One.%202014;9:e114787." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Karpinska A, Magiera G, Kwapiszewska K, Holyst R. Cellular uptake of bevacizumab in cervical and breast cancer cells revealed by single-molecule spectroscopy. J Phys Chem Lett. 2023;14:1272–8.
</cite> [<a href="https://doi.org/10.1021/acs.jpclett.2c03590" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9923738/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36719904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karpinska%20A,%20Magiera%20G,%20Kwapiszewska%20K,%20Holyst%20R.%20Cellular%20uptake%20of%20bevacizumab%20in%20cervical%20and%20breast%20cancer%20cells%20revealed%20by%20single-molecule%20spectroscopy.%20J%20Phys%20Chem%20Lett.%202023;14:1272%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Wei H, Mundade R, Lange KC, Lu T. Protein arginine methylation of non-histone proteins and its role in diseases. Cell Cycle. 2014;13:32–41.
</cite> [<a href="https://doi.org/10.4161/cc.27353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3925732/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24296620/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wei%20H,%20Mundade%20R,%20Lange%20KC,%20Lu%20T.%20Protein%20arginine%20methylation%20of%20non-histone%20proteins%20and%20its%20role%20in%20diseases.%20Cell%20Cycle.%202014;13:32%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Brobbey C, Liu L, Yin S, Gan W. The role of protein arginine methyltransferases in DNA damage response. Int J Mol Sci. 2022;23:9780.</cite> [<a href="https://doi.org/10.3390/ijms23179780" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9456308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36077176/" class="usa-link">PubMed</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Yu Z, Chen T, Hebert J, Li E, Richard S. A mouse PRMT1 null allele defines an essential role for arginine methylation in genome maintenance and cell proliferation. Mol Cell Biol. 2009;29:2982–96.
</cite> [<a href="https://doi.org/10.1128/MCB.00042-09" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2681996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19289494/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu%20Z,%20Chen%20T,%20Hebert%20J,%20Li%20E,%20Richard%20S.%20A%20mouse%20PRMT1%20null%20allele%20defines%20an%20essential%20role%20for%20arginine%20methylation%20in%20genome%20maintenance%20and%20cell%20proliferation.%20Mol%20Cell%20Biol.%202009;29:2982%E2%80%9396." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>El-Andaloussi N, Valovka T, Toueille M, Steinacher R, Focke F, Gehrig P, et al. Arginine methylation regulates DNA polymerase beta. Mol Cell. 2006;22:51–62.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2006.02.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16600869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?El-Andaloussi%20N,%20Valovka%20T,%20Toueille%20M,%20Steinacher%20R,%20Focke%20F,%20Gehrig%20P,%20et%20al.%20Arginine%20methylation%20regulates%20DNA%20polymerase%20beta.%20Mol%20Cell.%202006;22:51%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Lee YH, Bedford MT, Stallcup MR. Regulated recruitment of tumor suppressor BRCA1 to the p21 gene by coactivator methylation. Genes Dev. 2011;25:176–88.
</cite> [<a href="https://doi.org/10.1101/gad.1975811" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3022263/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21245169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20YH,%20Bedford%20MT,%20Stallcup%20MR.%20Regulated%20recruitment%20of%20tumor%20suppressor%20BRCA1%20to%20the%20p21%20gene%20by%20coactivator%20methylation.%20Genes%20Dev.%202011;25:176%E2%80%9388." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Dong R, Li X, Lai KO. Activity and Function of the PRMT8 Protein Arginine Methyltransferase in Neurons. Life. 2021;11:1132.</cite> [<a href="https://doi.org/10.3390/life11111132" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8621972/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34833008/" class="usa-link">PubMed</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Wu Q, Nie DY, Ba-Alawi W, Ji Y, Zhang Z, Cruickshank J, et al. PRMT inhibition induces a viral mimicry response in triple-negative breast cancer. Nat Chem Biol. 2022;18:821–30.
</cite> [<a href="https://doi.org/10.1038/s41589-022-01024-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9337992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35578032/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20Q,%20Nie%20DY,%20Ba-Alawi%20W,%20Ji%20Y,%20Zhang%20Z,%20Cruickshank%20J,%20et%20al.%20PRMT%20inhibition%20induces%20a%20viral%20mimicry%20response%20in%20triple-negative%20breast%20cancer.%20Nat%20Chem%20Biol.%202022;18:821%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Musiani D, Giambruno R, Massignani E, Ippolito MR, Maniaci M, Jammula S, et al. PRMT1 Is recruited via DNA-PK to chromatin where it sustains the senescence-associated secretory phenotype in response to cisplatin. Cell Rep. 2020;30:1208–22.e9.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2019.12.061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31995759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Musiani%20D,%20Giambruno%20R,%20Massignani%20E,%20Ippolito%20MR,%20Maniaci%20M,%20Jammula%20S,%20et%20al.%20PRMT1%20Is%20recruited%20via%20DNA-PK%20to%20chromatin%20where%20it%20sustains%20the%20senescence-associated%20secretory%20phenotype%20in%20response%20to%20cisplatin.%20Cell%20Rep.%202020;30:1208%E2%80%9322.e9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Takemasu S, Nigorikawa K, Yamada M, Tsurumi G, Kofuji S, Takasuga S, et al. Phosphorylation of TMEM55B by Erk/MAPK regulates lysosomal positioning. J Biochem. 2019;166:175–85.
</cite> [<a href="https://doi.org/10.1093/jb/mvz026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31329883/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Takemasu%20S,%20Nigorikawa%20K,%20Yamada%20M,%20Tsurumi%20G,%20Kofuji%20S,%20Takasuga%20S,%20et%20al.%20Phosphorylation%20of%20TMEM55B%20by%20Erk/MAPK%20regulates%20lysosomal%20positioning.%20J%20Biochem.%202019;166:175%E2%80%9385." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Malik N, Ferreira BI, Hollstein PE, Curtis SD, Trefts E, Weiser Novak S, et al. Induction of lysosomal and mitochondrial biogenesis by AMPK phosphorylation of FNIP1. Science. 2023;380:eabj5559.
</cite> [<a href="https://doi.org/10.1126/science.abj5559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10794112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37079666/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Malik%20N,%20Ferreira%20BI,%20Hollstein%20PE,%20Curtis%20SD,%20Trefts%20E,%20Weiser%20Novak%20S,%20et%20al.%20Induction%20of%20lysosomal%20and%20mitochondrial%20biogenesis%20by%20AMPK%20phosphorylation%20of%20FNIP1.%20Science.%202023;380:eabj5559." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Yogalingam G, Bonten EJ, van de Vlekkert D, Hu H, Moshiach S, Connell SA, et al. Neuraminidase 1 is a negative regulator of lysosomal exocytosis. Dev Cell. 2008;15:74–86.
</cite> [<a href="https://doi.org/10.1016/j.devcel.2008.05.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2664108/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18606142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yogalingam%20G,%20Bonten%20EJ,%20van%20de%20Vlekkert%20D,%20Hu%20H,%20Moshiach%20S,%20Connell%20SA,%20et%20al.%20Neuraminidase%201%20is%20a%20negative%20regulator%20of%20lysosomal%20exocytosis.%20Dev%20Cell.%202008;15:74%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Coutinho MF, Lacerda L, Alves S. Glycosaminoglycan storage disorders: a review. Biochem Res Int. 2012;2012:471325.
</cite> [<a href="https://doi.org/10.1155/2012/471325" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3195295/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22013531/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Coutinho%20MF,%20Lacerda%20L,%20Alves%20S.%20Glycosaminoglycan%20storage%20disorders:%20a%20review.%20Biochem%20Res%20Int.%202012;2012:471325." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Birrane G, Dassier AL, Romashko A, Lundberg D, Holmes K, Cottle T, et al. Structural characterization of the alpha-N-acetylglucosaminidase, a key enzyme in the pathogenesis of Sanfilippo syndrome B. J Struct Biol. 2019;205:65–71.
</cite> [<a href="https://doi.org/10.1016/j.jsb.2019.02.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30802506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Birrane%20G,%20Dassier%20AL,%20Romashko%20A,%20Lundberg%20D,%20Holmes%20K,%20Cottle%20T,%20et%20al.%20Structural%20characterization%20of%20the%20alpha-N-acetylglucosaminidase,%20a%20key%20enzyme%20in%20the%20pathogenesis%20of%20Sanfilippo%20syndrome%20B.%20J%20Struct%20Biol.%202019;205:65%E2%80%9371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Leal AF, Benincore-Florez E, Rintz E, Herreno-Pachon AM, Celik B, Ago Y, et al. Mucopolysaccharidoses: cellular consequences of glycosaminoglycans accumulation and potential targets. Int J Mol Sci. 2022;24:477.</cite> [<a href="https://doi.org/10.3390/ijms24010477" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9820209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36613919/" class="usa-link">PubMed</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12334744/bin/41419_2025_7900_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Merged Supplemental Materials</a><sup> (4.6MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The RNA-seq data generated and analyzed in this study are publicly available in the Gene Expression Omnibus (GEO) under the accession number GSE277490. No other datasets were generated during the current study.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Cell Death &amp; Disease are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41419-025-07900-w"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41419_2025_Article_7900.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.5 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12334744/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12334744/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12334744%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12334744/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12334744/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12334744/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40781072/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12334744/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40781072/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12334744/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12334744/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="qS8c0smmcmxnSIq3z3Jr2WcuCVnnXOcQ5yu8J5B3kLxmECwAhyBPMelgpPq8FM0N">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Web Policies

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    FOIA

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS Vulnerability Disclosure

    
</a>

                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Help

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Accessibility

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            








<a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    Careers

    
</a>

                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NLM

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
    

    NIH

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    HHS

    
</a>

                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            








<a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
    

    USA.gov

    
</a>

                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-370d5dd6.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-917ba005.js"></script>
    
    

    </body>
</html>
